<!DOCTYPE html>

<html>

<head>

<meta charset="utf-8" />
<meta name="generator" content="pandoc" />
<meta http-equiv="X-UA-Compatible" content="IE=EDGE" />


<meta name="author" content="Belinda Cornes" />

<meta name="date" content="2022-07-02" />

<title>QTL Analysis - binary [ICI vs PBS] (conditioning on eoi vs. ici peaks)</title>

<script src="site_libs/header-attrs-2.3/header-attrs.js"></script>
<script src="site_libs/jquery-1.11.3/jquery.min.js"></script>
<meta name="viewport" content="width=device-width, initial-scale=1" />
<link href="site_libs/bootstrap-3.3.5/css/spacelab.min.css" rel="stylesheet" />
<script src="site_libs/bootstrap-3.3.5/js/bootstrap.min.js"></script>
<script src="site_libs/bootstrap-3.3.5/shim/html5shiv.min.js"></script>
<script src="site_libs/bootstrap-3.3.5/shim/respond.min.js"></script>
<script src="site_libs/navigation-1.1/tabsets.js"></script>
<link href="site_libs/highlightjs-9.12.0/textmate.css" rel="stylesheet" />
<script src="site_libs/highlightjs-9.12.0/highlight.js"></script>
<script src="site_libs/kePrint-0.0.1/kePrint.js"></script>
<link href="site_libs/lightable-0.0.1/lightable.css" rel="stylesheet" />
<link href="site_libs/font-awesome-5.1.0/css/all.css" rel="stylesheet" />
<link href="site_libs/font-awesome-5.1.0/css/v4-shims.css" rel="stylesheet" />

<link rel="icon" href="https://github.com/workflowr/workflowr-assets/raw/master/img/reproducible.png">
<!-- Add a small amount of space between sections. -->
<style type="text/css">
div.section {
  padding-top: 12px;
}
</style>



<style type="text/css">code{white-space: pre;}</style>
<style type="text/css">
  pre:not([class]) {
    background-color: white;
  }
</style>
<script type="text/javascript">
if (window.hljs) {
  hljs.configure({languages: []});
  hljs.initHighlightingOnLoad();
  if (document.readyState && document.readyState === "complete") {
    window.setTimeout(function() { hljs.initHighlighting(); }, 0);
  }
}
</script>



<style type="text/css">
h1 {
  font-size: 34px;
}
h1.title {
  font-size: 38px;
}
h2 {
  font-size: 30px;
}
h3 {
  font-size: 24px;
}
h4 {
  font-size: 18px;
}
h5 {
  font-size: 16px;
}
h6 {
  font-size: 12px;
}
.table th:not([align]) {
  text-align: left;
}
</style>




<style type = "text/css">
.main-container {
  max-width: 940px;
  margin-left: auto;
  margin-right: auto;
}
code {
  color: inherit;
  background-color: rgba(0, 0, 0, 0.04);
}
img {
  max-width:100%;
}
.tabbed-pane {
  padding-top: 12px;
}
.html-widget {
  margin-bottom: 20px;
}
button.code-folding-btn:focus {
  outline: none;
}
summary {
  display: list-item;
}
</style>


<style type="text/css">
/* padding for bootstrap navbar */
body {
  padding-top: 51px;
  padding-bottom: 40px;
}
/* offset scroll position for anchor links (for fixed navbar)  */
.section h1 {
  padding-top: 56px;
  margin-top: -56px;
}
.section h2 {
  padding-top: 56px;
  margin-top: -56px;
}
.section h3 {
  padding-top: 56px;
  margin-top: -56px;
}
.section h4 {
  padding-top: 56px;
  margin-top: -56px;
}
.section h5 {
  padding-top: 56px;
  margin-top: -56px;
}
.section h6 {
  padding-top: 56px;
  margin-top: -56px;
}
.dropdown-submenu {
  position: relative;
}
.dropdown-submenu>.dropdown-menu {
  top: 0;
  left: 100%;
  margin-top: -6px;
  margin-left: -1px;
  border-radius: 0 6px 6px 6px;
}
.dropdown-submenu:hover>.dropdown-menu {
  display: block;
}
.dropdown-submenu>a:after {
  display: block;
  content: " ";
  float: right;
  width: 0;
  height: 0;
  border-color: transparent;
  border-style: solid;
  border-width: 5px 0 5px 5px;
  border-left-color: #cccccc;
  margin-top: 5px;
  margin-right: -10px;
}
.dropdown-submenu:hover>a:after {
  border-left-color: #ffffff;
}
.dropdown-submenu.pull-left {
  float: none;
}
.dropdown-submenu.pull-left>.dropdown-menu {
  left: -100%;
  margin-left: 10px;
  border-radius: 6px 0 6px 6px;
}
</style>

<script>
// manage active state of menu based on current page
$(document).ready(function () {
  // active menu anchor
  href = window.location.pathname
  href = href.substr(href.lastIndexOf('/') + 1)
  if (href === "")
    href = "index.html";
  var menuAnchor = $('a[href="' + href + '"]');

  // mark it active
  menuAnchor.parent().addClass('active');

  // if it's got a parent navbar menu mark it active as well
  menuAnchor.closest('li.dropdown').addClass('active');
});
</script>

<!-- tabsets -->

<style type="text/css">
.tabset-dropdown > .nav-tabs {
  display: inline-table;
  max-height: 500px;
  min-height: 44px;
  overflow-y: auto;
  background: white;
  border: 1px solid #ddd;
  border-radius: 4px;
}

.tabset-dropdown > .nav-tabs > li.active:before {
  content: "";
  font-family: 'Glyphicons Halflings';
  display: inline-block;
  padding: 10px;
  border-right: 1px solid #ddd;
}

.tabset-dropdown > .nav-tabs.nav-tabs-open > li.active:before {
  content: "&#xe258;";
  border: none;
}

.tabset-dropdown > .nav-tabs.nav-tabs-open:before {
  content: "";
  font-family: 'Glyphicons Halflings';
  display: inline-block;
  padding: 10px;
  border-right: 1px solid #ddd;
}

.tabset-dropdown > .nav-tabs > li.active {
  display: block;
}

.tabset-dropdown > .nav-tabs > li > a,
.tabset-dropdown > .nav-tabs > li > a:focus,
.tabset-dropdown > .nav-tabs > li > a:hover {
  border: none;
  display: inline-block;
  border-radius: 4px;
  background-color: transparent;
}

.tabset-dropdown > .nav-tabs.nav-tabs-open > li {
  display: block;
  float: none;
}

.tabset-dropdown > .nav-tabs > li {
  display: none;
}
</style>

<!-- code folding -->




</head>

<body>


<div class="container-fluid main-container">




<div class="navbar navbar-default  navbar-fixed-top" role="navigation">
  <div class="container">
    <div class="navbar-header">
      <button type="button" class="navbar-toggle collapsed" data-toggle="collapse" data-target="#navbar">
        <span class="icon-bar"></span>
        <span class="icon-bar"></span>
        <span class="icon-bar"></span>
      </button>
      <a class="navbar-brand" href="index.html">Serreze T1D Workflow</a>
    </div>
    <div id="navbar" class="navbar-collapse collapse">
      <ul class="nav navbar-nav">
        <li>
  <a href="index.html">Home</a>
</li>
<li>
  <a href="about.html">About</a>
</li>
<li>
  <a href="license.html">License</a>
</li>
      </ul>
      <ul class="nav navbar-nav navbar-right">
        <li>
  <a href="https://github.com/TheJacksonLaboratory/Serreze-T1D_Workflow">
    <span class="fa fa-github"></span>
     
    Source code
  </a>
</li>
      </ul>
    </div><!--/.nav-collapse -->
  </div><!--/.container -->
</div><!--/.navbar -->

<div class="fluid-row" id="header">



<h1 class="title toc-ignore">QTL Analysis - binary [ICI vs PBS]
(conditioning on eoi vs. ici peaks)</h1>
<h4 class="author">Belinda Cornes</h4>
<h4 class="date">2022-07-02</h4>

</div>

<div id="TOC">
<ul>
<li><a href="#loading-data">Loading Data</a></li>
<li><a
href="#conditionaing-on-eoi-vs.-ici-chr-3-peak-unchs008487-chr-4-peak-unc8250659">Conditionaing
on eoi vs. ici chr 3 peak (UNCHS008487) &amp; chr 4 peak
(UNC8250659)</a>
<ul>
<li><a href="#genome-wide-scan">Genome-wide scan</a></li>
<li><a href="#genome-wide-scan-1">Genome-wide scan</a></li>
<li><a href="#lod-peaks">LOD peaks</a>
<ul>
<li><a href="#centimorgan-cm">Centimorgan (cM)</a></li>
<li><a href="#megabase-mb">Megabase (MB)</a></li>
</ul></li>
<li><a href="#qtl-effects">QTL effects</a></li>
<li><a href="#rqtl">R/qtl</a>
<ul>
<li><a href="#scanone">scanone</a></li>
</ul></li>
</ul></li>
<li><a
href="#conditionaing-on-eoi-vs.-ici-chr-3-peak-unchs008487-chr-10-peak-unc18240977">Conditionaing
on eoi vs. ici chr 3 peak (UNCHS008487) &amp; chr 10 peak
(UNC18240977)</a>
<ul>
<li><a href="#genome-wide-scan-2">Genome-wide scan</a></li>
<li><a href="#lod-peaks-1">LOD peaks</a>
<ul>
<li><a href="#centimorgan-cm-1">Centimorgan (cM)</a></li>
<li><a href="#megabase-mb-1">Megabase (MB)</a></li>
</ul></li>
<li><a href="#qtl-effects-1">QTL effects</a></li>
<li><a href="#rqtl-1">R/qtl</a>
<ul>
<li><a href="#scanone-1">scanone</a></li>
</ul></li>
</ul></li>
<li><a
href="#conditionaing-on-eoi-vs.-ici-chr-4-peak-unchs008487-chr-10-peak-unc18240977">Conditionaing
on eoi vs. ici chr 4 peak (UNCHS008487) &amp; chr 10 peak
(UNC18240977)</a>
<ul>
<li><a href="#genome-wide-scan-3">Genome-wide scan</a></li>
<li><a href="#lod-peaks-2">LOD peaks</a>
<ul>
<li><a href="#centimorgan-cm-2">Centimorgan (cM)</a></li>
<li><a href="#megabase-mb-2">Megabase (MB)</a></li>
</ul></li>
<li><a href="#qtl-effects-2">QTL effects</a></li>
<li><a href="#rqtl-2">R/qtl</a>
<ul>
<li><a href="#scanone-2">scanone</a></li>
</ul></li>
</ul></li>
</ul>
</div>

<p>
<button type="button" class="btn btn-default btn-workflowr btn-workflowr-report" data-toggle="collapse" data-target="#workflowr-report">
<span class="glyphicon glyphicon-list" aria-hidden="true"></span>
workflowr <span class="glyphicon glyphicon-exclamation-sign text-danger"
aria-hidden="true"></span>
</button>
</p>
<div id="workflowr-report" class="collapse">
<ul class="nav nav-tabs">
<li class="active">
<a data-toggle="tab" href="#summary">Summary</a>
</li>
<li>
<a data-toggle="tab" href="#checks"> Checks <span
class="glyphicon glyphicon-exclamation-sign text-danger"
aria-hidden="true"></span> </a>
</li>
<li>
<a data-toggle="tab" href="#versions">Past versions</a>
</li>
</ul>
<div class="tab-content">
<div id="summary" class="tab-pane fade in active">
<p>
<strong>Last updated:</strong> 2022-07-02
</p>
<p>
<strong>Checks:</strong> <span
class="glyphicon glyphicon-ok text-success" aria-hidden="true"></span> 5
<span class="glyphicon glyphicon-exclamation-sign text-danger"
aria-hidden="true"></span> 2
</p>
<p>
<strong>Knit directory:</strong> <code>Serreze-T1D_Workflow/</code>
<span class="glyphicon glyphicon-question-sign" aria-hidden="true"
title="This is the local directory in which the code in this file was executed.">
</span>
</p>
<p>
This reproducible <a href="http://rmarkdown.rstudio.com">R Markdown</a>
analysis was created with <a
  href="https://github.com/jdblischak/workflowr">workflowr</a> (version
1.6.2). The <em>Checks</em> tab describes the reproducibility checks
that were applied when the results were created. The <em>Past
versions</em> tab lists the development history.
</p>
<hr>
</div>
<div id="checks" class="tab-pane fade">
<div id="workflowr-checks" class="panel-group">
<div class="panel panel-default">
<div class="panel-heading">
<p class="panel-title">
<a data-toggle="collapse" data-parent="#workflowr-checks" href="#strongRMarkdownfilestronguncommittedchanges">
<span class="glyphicon glyphicon-exclamation-sign text-danger"
aria-hidden="true"></span> <strong>R Markdown file:</strong> uncommitted
changes </a>
</p>
</div>
<div id="strongRMarkdownfilestronguncommittedchanges"
class="panel-collapse collapse">
<div class="panel-body">
<p>The R Markdown is untracked by Git. To know which version of the R
Markdown file created these results, you’ll want to first commit it to
the Git repo. If you’re still working on the analysis, you can ignore
this warning. When you’re finished, you can run
<code>wflow_publish</code> to commit the R Markdown file and build the
HTML.</p>
</div>
</div>
</div>
<div class="panel panel-default">
<div class="panel-heading">
<p class="panel-title">
<a data-toggle="collapse" data-parent="#workflowr-checks" href="#strongEnvironmentstrongempty">
<span class="glyphicon glyphicon-ok text-success"
aria-hidden="true"></span> <strong>Environment:</strong> empty </a>
</p>
</div>
<div id="strongEnvironmentstrongempty" class="panel-collapse collapse">
<div class="panel-body">
<p>Great job! The global environment was empty. Objects defined in the
global environment can affect the analysis in your R Markdown file in
unknown ways. For reproduciblity it’s best to always run the code in an
empty environment.</p>
</div>
</div>
</div>
<div class="panel panel-default">
<div class="panel-heading">
<p class="panel-title">
<a data-toggle="collapse" data-parent="#workflowr-checks" href="#strongSeedstrongcodesetseed20220210code">
<span class="glyphicon glyphicon-ok text-success"
aria-hidden="true"></span> <strong>Seed:</strong>
<code>set.seed(20220210)</code> </a>
</p>
</div>
<div id="strongSeedstrongcodesetseed20220210code"
class="panel-collapse collapse">
<div class="panel-body">
<p>The command <code>set.seed(20220210)</code> was run prior to running
the code in the R Markdown file. Setting a seed ensures that any results
that rely on randomness, e.g. subsampling or permutations, are
reproducible.</p>
</div>
</div>
</div>
<div class="panel panel-default">
<div class="panel-heading">
<p class="panel-title">
<a data-toggle="collapse" data-parent="#workflowr-checks" href="#strongSessioninformationstrongrecorded">
<span class="glyphicon glyphicon-ok text-success"
aria-hidden="true"></span> <strong>Session information:</strong>
recorded </a>
</p>
</div>
<div id="strongSessioninformationstrongrecorded"
class="panel-collapse collapse">
<div class="panel-body">
<p>Great job! Recording the operating system, R version, and package
versions is critical for reproducibility.</p>
</div>
</div>
</div>
<div class="panel panel-default">
<div class="panel-heading">
<p class="panel-title">
<a data-toggle="collapse" data-parent="#workflowr-checks" href="#strongCachestrongnone">
<span class="glyphicon glyphicon-ok text-success"
aria-hidden="true"></span> <strong>Cache:</strong> none </a>
</p>
</div>
<div id="strongCachestrongnone" class="panel-collapse collapse">
<div class="panel-body">
<p>Nice! There were no cached chunks for this analysis, so you can be
confident that you successfully produced the results during this
run.</p>
</div>
</div>
</div>
<div class="panel panel-default">
<div class="panel-heading">
<p class="panel-title">
<a data-toggle="collapse" data-parent="#workflowr-checks" href="#strongFilepathsstrongabsolute">
<span class="glyphicon glyphicon-exclamation-sign text-danger"
aria-hidden="true"></span> <strong>File paths:</strong> absolute </a>
</p>
</div>
<div id="strongFilepathsstrongabsolute" class="panel-collapse collapse">
<div class="panel-body">
<p>
Using absolute paths to the files within your workflowr project makes it
difficult for you and others to run your code on a different machine.
Change the absolute path(s) below to the suggested relative path(s) to
make your code more reproducible.
</p>
<table class="table table-condensed table-hover">
<thead>
<tr>
<th style="text-align:left;">
absolute
</th>
<th style="text-align:left;">
relative
</th>
</tr>
</thead>
<tbody>
<tr>
<td style="text-align:left;">
/Users/corneb/Documents/MyJax/CS/Projects/Serreze/qc/workflowr/Serreze-T1D_Workflow
</td>
<td style="text-align:left;">
.
</td>
</tr>
</tbody>
</table>
</div>
</div>
</div>
<div class="panel panel-default">
<div class="panel-heading">
<p class="panel-title">
<a data-toggle="collapse" data-parent="#workflowr-checks" href="#strongRepositoryversionstrongahrefhttpsgithubcomTheJacksonLaboratorySerrezeT1DWorkflowtreea8c2d1a11f223361b54952ce8455e8c17017c914targetblanka8c2d1aa">
<span class="glyphicon glyphicon-ok text-success"
aria-hidden="true"></span> <strong>Repository version:</strong>
<a href="https://github.com/TheJacksonLaboratory/Serreze-T1D_Workflow/tree/a8c2d1a11f223361b54952ce8455e8c17017c914" target="_blank">a8c2d1a</a>
</a>
</p>
</div>
<div
id="strongRepositoryversionstrongahrefhttpsgithubcomTheJacksonLaboratorySerrezeT1DWorkflowtreea8c2d1a11f223361b54952ce8455e8c17017c914targetblanka8c2d1aa"
class="panel-collapse collapse">
<div class="panel-body">
<p>
Great! You are using Git for version control. Tracking code development
and connecting the code version to the results is critical for
reproducibility.
</p>
<p>
The results in this page were generated with repository version
<a href="https://github.com/TheJacksonLaboratory/Serreze-T1D_Workflow/tree/a8c2d1a11f223361b54952ce8455e8c17017c914" target="_blank">a8c2d1a</a>.
See the <em>Past versions</em> tab to see a history of the changes made
to the R Markdown and HTML files.
</p>
<p>
Note that you need to be careful to ensure that all relevant files for
the analysis have been committed to Git prior to generating the results
(you can use <code>wflow_publish</code> or
<code>wflow_git_commit</code>). workflowr only checks the R Markdown
file, but you know if there are other scripts or data files that it
depends on. Below is the status of the Git repository when the results
were generated:
</p>
<pre><code>
Ignored files:
    Ignored:    .DS_Store
    Ignored:    analysis/.DS_Store
    Ignored:    data/.DS_Store

Untracked files:
    Untracked:  analysis/3.1_phenotype.qc_corrected_5.batches_0.Rmd
    Untracked:  analysis/3.1_phenotype.qc_corrected_5.batches_52.Rmd
    Untracked:  analysis/3.1_phenotype.qc_corrected_5.batches_mis_0.Rmd
    Untracked:  analysis/3.1_phenotype.qc_corrected_5.batches_mis_52.Rmd
    Untracked:  analysis/3.1_phenotype.qc_corrected_5.batches_mis_vo.Rmd
    Untracked:  analysis/3.1_phenotype.qc_corrected_5.batches_vo.Rmd
    Untracked:  analysis/3.1_phenotype.qc_corrected_ici-early.vs.pbs.ici-late_5.batches.Rmd
    Untracked:  analysis/3.1_phenotype.qc_corrected_ici-early.vs.pbs.ici-late_5.batches_mis.Rmd
    Untracked:  analysis/4.1.1_qtl.analysis_binary_ici-early.vs.pbs.ici-late_5.batches.Rmd
    Untracked:  analysis/4.1.1_qtl.analysis_binary_ici-early.vs.pbs.ici-late_5.batches_mis.Rmd
    Untracked:  analysis/4.1.1_qtl.analysis_binary_ici-early.vs.pbs.ici-late_snpsqc_5.batches.Rmd
    Untracked:  analysis/4.1.1_qtl.analysis_binary_ici-early.vs.pbs.ici-late_snpsqc_5.batches_mis.Rmd
    Untracked:  analysis/4.1.1_qtl.analysis_binary_ici-early.vs.pbs.ici-late_snpsqc_dis_no-x_updated_5.batches.Rmd
    Untracked:  analysis/4.1.1_qtl.analysis_binary_ici-early.vs.pbs.ici-late_snpsqc_dis_no-x_updated_5.batches_mis.Rmd
    Untracked:  analysis/4.1.1_qtl.analysis_binary_ici.vs.eoi_snpsqc_dis_no-x_updated_5.batches_11.Rmd
    Untracked:  analysis/4.1.1_qtl.analysis_binary_ici.vs.eoi_snpsqc_dis_no-x_updated_5.batches_mis_11.Rmd
    Untracked:  analysis/4.1.1_qtl.analysis_binary_ici.vs.pbs_5.batches_conditional.Rmd.R
    Untracked:  analysis/4.1.1_qtl.analysis_binary_ici.vs.pbs_5.batches_conditional_1-peak.Rmd
    Untracked:  analysis/4.1.1_qtl.analysis_binary_ici.vs.pbs_5.batches_conditional_1-peak.Rmd.R
    Untracked:  analysis/4.1.1_qtl.analysis_binary_ici.vs.pbs_5.batches_conditional_2-peaks.Rmd
    Untracked:  analysis/4.1.1_qtl.analysis_binary_ici.vs.pbs_5.batches_conditional_2-peaks.Rmd.R
    Untracked:  analysis/4.1.1_qtl.analysis_binary_ici.vs.pbs_5.batches_conditional_3-peaks.Rmd
    Untracked:  analysis/4.1.1_qtl.analysis_binary_ici.vs.pbs_5.batches_conditional_3-peaks.Rmd.R
    Untracked:  analysis/4.1.1_qtl.analysis_binary_ici.vs.pbs_5.batches_mis_conditional.Rmd.R
    Untracked:  analysis/4.1.1_qtl.analysis_binary_ici.vs.pbs_5.batches_mis_conditional_1-peak.Rmd
    Untracked:  analysis/4.1.1_qtl.analysis_binary_ici.vs.pbs_5.batches_mis_conditional_1-peak.Rmd.R
    Untracked:  analysis/4.1.1_qtl.analysis_binary_ici.vs.pbs_5.batches_mis_conditional_2-peaks.Rmd
    Untracked:  analysis/4.1.1_qtl.analysis_binary_ici.vs.pbs_5.batches_mis_conditional_2-peaks.Rmd.R
    Untracked:  analysis/4.1.1_qtl.analysis_binary_ici.vs.pbs_5.batches_mis_conditional_3-peaks.Rmd
    Untracked:  analysis/4.1.1_qtl.analysis_binary_ici.vs.pbs_5.batches_mis_conditional_3-peaks.Rmd.R
    Untracked:  analysis/4.1.1_qtl.analysis_binary_ici.vs.pbs_snpsqc_5.batches_1.Rmd
    Untracked:  analysis/4.1.1_qtl.analysis_binary_ici.vs.pbs_snpsqc_5.batches_conditional.Rmd.R
    Untracked:  analysis/4.1.1_qtl.analysis_binary_ici.vs.pbs_snpsqc_5.batches_conditional_1-peak.Rmd
    Untracked:  analysis/4.1.1_qtl.analysis_binary_ici.vs.pbs_snpsqc_5.batches_conditional_1-peak.Rmd.R
    Untracked:  analysis/4.1.1_qtl.analysis_binary_ici.vs.pbs_snpsqc_5.batches_conditional_2-peaks.Rmd
    Untracked:  analysis/4.1.1_qtl.analysis_binary_ici.vs.pbs_snpsqc_5.batches_conditional_2-peaks.Rmd.R
    Untracked:  analysis/4.1.1_qtl.analysis_binary_ici.vs.pbs_snpsqc_5.batches_conditional_3-peaks.Rmd
    Untracked:  analysis/4.1.1_qtl.analysis_binary_ici.vs.pbs_snpsqc_5.batches_conditional_3-peaks.Rmd.R
    Untracked:  analysis/4.1.1_qtl.analysis_binary_ici.vs.pbs_snpsqc_5.batches_mis_1.Rmd
    Untracked:  analysis/4.1.1_qtl.analysis_binary_ici.vs.pbs_snpsqc_5.batches_mis_conditional.Rmd.R
    Untracked:  analysis/4.1.1_qtl.analysis_binary_ici.vs.pbs_snpsqc_5.batches_mis_conditional_1-peak.Rmd
    Untracked:  analysis/4.1.1_qtl.analysis_binary_ici.vs.pbs_snpsqc_5.batches_mis_conditional_1-peak.Rmd.R
    Untracked:  analysis/4.1.1_qtl.analysis_binary_ici.vs.pbs_snpsqc_5.batches_mis_conditional_2-peaks.Rmd
    Untracked:  analysis/4.1.1_qtl.analysis_binary_ici.vs.pbs_snpsqc_5.batches_mis_conditional_2-peaks.Rmd.R
    Untracked:  analysis/4.1.1_qtl.analysis_binary_ici.vs.pbs_snpsqc_5.batches_mis_conditional_3-peaks.Rmd
    Untracked:  analysis/4.1.1_qtl.analysis_binary_ici.vs.pbs_snpsqc_5.batches_mis_conditional_3-peaks.Rmd.R
    Untracked:  analysis/4.1.1_qtl.analysis_binary_ici.vs.pbs_snpsqc_dis_no-x_updated_5.batches_1.Rmd
    Untracked:  analysis/4.1.1_qtl.analysis_binary_ici.vs.pbs_snpsqc_dis_no-x_updated_5.batches_11.Rmd
    Untracked:  analysis/4.1.1_qtl.analysis_binary_ici.vs.pbs_snpsqc_dis_no-x_updated_5.batches_conditional.Rmd.R
    Untracked:  analysis/4.1.1_qtl.analysis_binary_ici.vs.pbs_snpsqc_dis_no-x_updated_5.batches_conditional_1-peak.Rmd
    Untracked:  analysis/4.1.1_qtl.analysis_binary_ici.vs.pbs_snpsqc_dis_no-x_updated_5.batches_conditional_1-peak.Rmd.R
    Untracked:  analysis/4.1.1_qtl.analysis_binary_ici.vs.pbs_snpsqc_dis_no-x_updated_5.batches_conditional_2-peaks.Rmd
    Untracked:  analysis/4.1.1_qtl.analysis_binary_ici.vs.pbs_snpsqc_dis_no-x_updated_5.batches_conditional_2-peaks.Rmd.R
    Untracked:  analysis/4.1.1_qtl.analysis_binary_ici.vs.pbs_snpsqc_dis_no-x_updated_5.batches_conditional_3-peaks.Rmd
    Untracked:  analysis/4.1.1_qtl.analysis_binary_ici.vs.pbs_snpsqc_dis_no-x_updated_5.batches_conditional_3-peaks.Rmd.R
    Untracked:  analysis/4.1.1_qtl.analysis_binary_ici.vs.pbs_snpsqc_dis_no-x_updated_5.batches_mis_11.Rmd
    Untracked:  analysis/4.1.1_qtl.analysis_binary_ici.vs.pbs_snpsqc_dis_no-x_updated_5.batches_mis_conditional.Rmd.R
    Untracked:  analysis/4.1.1_qtl.analysis_binary_ici.vs.pbs_snpsqc_dis_no-x_updated_5.batches_mis_conditional_1-peak.Rmd
    Untracked:  analysis/4.1.1_qtl.analysis_binary_ici.vs.pbs_snpsqc_dis_no-x_updated_5.batches_mis_conditional_1-peak.Rmd.R
    Untracked:  analysis/4.1.1_qtl.analysis_binary_ici.vs.pbs_snpsqc_dis_no-x_updated_5.batches_mis_conditional_1.peak.Rmd.R
    Untracked:  analysis/4.1.1_qtl.analysis_binary_ici.vs.pbs_snpsqc_dis_no-x_updated_5.batches_mis_conditional_2-peaks.Rmd
    Untracked:  analysis/4.1.1_qtl.analysis_binary_ici.vs.pbs_snpsqc_dis_no-x_updated_5.batches_mis_conditional_2-peaks.Rmd.R
    Untracked:  analysis/4.1.1_qtl.analysis_binary_ici.vs.pbs_snpsqc_dis_no-x_updated_5.batches_mis_conditional_3-peaks.Rmd
    Untracked:  analysis/4.1.1_qtl.analysis_binary_ici.vs.pbs_snpsqc_dis_no-x_updated_5.batches_mis_conditional_3-peaks.Rmd.R
    Untracked:  analysis/4.1.2_qtl.analysis_cont_age_ici-early.vs.pbs.ici-late_pheno.corrected.cleaned_5.batches.Rmd
    Untracked:  analysis/4.1.2_qtl.analysis_cont_age_ici-early.vs.pbs.ici-late_pheno.corrected.cleaned_5.batches_mis.Rmd
    Untracked:  analysis/4.1.2_qtl.analysis_cont_age_ici-early.vs.pbs.ici-late_snpsqc_pheno.corrected.cleaned_5.batches.Rmd
    Untracked:  analysis/4.1.2_qtl.analysis_cont_age_ici-early.vs.pbs.ici-late_snpsqc_pheno.corrected.cleaned_5.batches_mis.Rmd
    Untracked:  analysis/4.1.2_qtl.analysis_cont_age_ici-early.vs.pbs.ici-late_snpsqc_pheno.corrected.cleaned_dis_no-xk_5.batches.Rmd
    Untracked:  analysis/4.1.2_qtl.analysis_cont_age_ici-early.vs.pbs.ici-late_snpsqc_pheno.corrected.cleaned_dis_no-xk_5.batches_mis.Rmd
    Untracked:  analysis/4.1.2_qtl.analysis_cont_age_ici.vs.eoi_pheno.corrected.cleaned_5.batches_0.Rmd
    Untracked:  analysis/4.1.2_qtl.analysis_cont_age_ici.vs.eoi_pheno.corrected.cleaned_5.batches_52.Rmd
    Untracked:  analysis/4.1.2_qtl.analysis_cont_age_ici.vs.eoi_pheno.corrected.cleaned_5.batches_changed.Rmd
    Untracked:  analysis/4.1.2_qtl.analysis_cont_age_ici.vs.eoi_pheno.corrected.cleaned_5.batches_mis_0.Rmd
    Untracked:  analysis/4.1.2_qtl.analysis_cont_age_ici.vs.eoi_pheno.corrected.cleaned_5.batches_mis_52.Rmd
    Untracked:  analysis/4.1.2_qtl.analysis_cont_age_ici.vs.eoi_pheno.corrected.cleaned_5.batches_mis_changed.Rmd
    Untracked:  analysis/4.1.2_qtl.analysis_cont_age_ici.vs.eoi_pheno.corrected.cleaned_scanone_5.batches.Rmd
    Untracked:  analysis/4.1.2_qtl.analysis_cont_age_ici.vs.eoi_pheno.corrected.cleaned_scanone_5.batches_mis.Rmd
    Untracked:  analysis/4.1.2_qtl.analysis_cont_age_ici.vs.eoi_snpsqc_pheno.corrected.cleaned_5.batches_0.Rmd
    Untracked:  analysis/4.1.2_qtl.analysis_cont_age_ici.vs.eoi_snpsqc_pheno.corrected.cleaned_5.batches_52.Rmd
    Untracked:  analysis/4.1.2_qtl.analysis_cont_age_ici.vs.eoi_snpsqc_pheno.corrected.cleaned_5.batches_mis_0.Rmd
    Untracked:  analysis/4.1.2_qtl.analysis_cont_age_ici.vs.eoi_snpsqc_pheno.corrected.cleaned_5.batches_mis_52.Rmd
    Untracked:  analysis/4.1.2_qtl.analysis_cont_age_ici.vs.eoi_snpsqc_pheno.corrected.cleaned_5.batches_oops.Rmd
    Untracked:  analysis/4.1.2_qtl.analysis_cont_age_ici.vs.eoi_snpsqc_pheno.corrected.cleaned_dis_no-xk_5.batches_0.Rmd
    Untracked:  analysis/4.1.2_qtl.analysis_cont_age_ici.vs.eoi_snpsqc_pheno.corrected.cleaned_dis_no-xk_5.batches_52.Rmd
    Untracked:  analysis/4.1.2_qtl.analysis_cont_age_ici.vs.eoi_snpsqc_pheno.corrected.cleaned_dis_no-xk_5.batches_mis_0.Rmd
    Untracked:  analysis/4.1.2_qtl.analysis_cont_age_ici.vs.eoi_snpsqc_pheno.corrected.cleaned_dis_no-xk_5.batches_mis_52.Rmd
    Untracked:  analysis/4.1.2_qtl.analysis_cont_age_ici.vs.eoi_snpsqc_pheno.corrected.cleaned_dis_v2.Rmd.R
    Untracked:  analysis/4.1.2_qtl.analysis_cont_age_ici.vs.eoi_snpsqc_pheno.corrected.cleaned_test.with.4.Rmd
    Untracked:  analysis/4.1.2_qtl.analysis_cont_age_ici.vs.eoi_snpsqc_pheno.corrected.cleaned_test.with.4_miss.Rmd
    Untracked:  analysis/4.1.2_qtl.analysis_cont_age_ici.vs.eoi_snpsqc_pheno.corrected.cleaned_test.with.5.Rmd
    Untracked:  analysis/4.1.2_qtl.analysis_cont_age_ici.vs.eoi_snpsqc_pheno.corrected.cleaned_test.with.5_0.Rmd
    Untracked:  analysis/4.1.2_qtl.analysis_cont_age_ici.vs.pbs_pheno.corrected.cleaned_5.batches_conditional.Rmd.R
    Untracked:  analysis/4.1.2_qtl.analysis_cont_age_ici.vs.pbs_pheno.corrected.cleaned_5.batches_mis_conditional.Rmd.R
    Untracked:  analysis/4.1.2_qtl.analysis_cont_age_ici.vs.pbs_snpsqc_pheno.corrected.cleaned_5.batches_conditional.Rmd.R
    Untracked:  analysis/4.1.2_qtl.analysis_cont_age_ici.vs.pbs_snpsqc_pheno.corrected.cleaned_5.batches_mis_conditional.Rmd.R
    Untracked:  analysis/4.1.2_qtl.analysis_cont_age_ici.vs.pbs_snpsqc_pheno.corrected.cleaned_dis_no-xk_5.batches_conditional.Rmd.R
    Untracked:  analysis/4.1.2_qtl.analysis_cont_age_ici.vs.pbs_snpsqc_pheno.corrected.cleaned_dis_no-xk_5.batches_mis_conditional.Rmd.R
    Untracked:  analysis/4.1.2_qtl.analysis_cont_rz.age_ici-early.vs.pbs.ici-late_pheno.corrected.cleaned_5.batches.Rmd
    Untracked:  analysis/4.1.2_qtl.analysis_cont_rz.age_ici-early.vs.pbs.ici-late_pheno.corrected.cleaned_5.batches_mis.Rmd
    Untracked:  analysis/4.1.2_qtl.analysis_cont_rz.age_ici-early.vs.pbs.ici-late_snpsqc_pheno.corrected.cleaned_5.batches.Rmd
    Untracked:  analysis/4.1.2_qtl.analysis_cont_rz.age_ici-early.vs.pbs.ici-late_snpsqc_pheno.corrected.cleaned_5.batches_mis.Rmd
    Untracked:  analysis/4.1.2_qtl.analysis_cont_rz.age_ici-early.vs.pbs.ici-late_snpsqc_pheno.corrected.cleaned_dis_no-xk_5.batches.Rmd
    Untracked:  analysis/4.1.2_qtl.analysis_cont_rz.age_ici-early.vs.pbs.ici-late_snpsqc_pheno.corrected.cleaned_dis_no-xk_5.batches_mis.Rmd
    Untracked:  analysis/4.1.2_qtl.analysis_cont_rz.age_ici.vs.eoi_pheno.corrected.cleaned_5.batches_0.Rmd
    Untracked:  analysis/4.1.2_qtl.analysis_cont_rz.age_ici.vs.eoi_pheno.corrected.cleaned_5.batches_52.Rmd
    Untracked:  analysis/4.1.2_qtl.analysis_cont_rz.age_ici.vs.eoi_pheno.corrected.cleaned_5.batches_mis_0.Rmd
    Untracked:  analysis/4.1.2_qtl.analysis_cont_rz.age_ici.vs.eoi_pheno.corrected.cleaned_5.batches_mis_52.Rmd
    Untracked:  analysis/4.1.2_qtl.analysis_cont_rz.age_ici.vs.eoi_pheno.corrected.cleaned_5.batches_mis_vo.Rmd
    Untracked:  analysis/4.1.2_qtl.analysis_cont_rz.age_ici.vs.eoi_pheno.corrected.cleaned_5.batches_mis_vo1.Rmd
    Untracked:  analysis/4.1.2_qtl.analysis_cont_rz.age_ici.vs.eoi_pheno.corrected.cleaned_5.batches_vo.Rmd
    Untracked:  analysis/4.1.2_qtl.analysis_cont_rz.age_ici.vs.eoi_pheno.corrected.cleaned_5.batches_vo1.Rmd
    Untracked:  analysis/4.1.2_qtl.analysis_cont_rz.age_ici.vs.eoi_snpsqc_pheno.corrected.cleaned_5.batches_0.Rmd
    Untracked:  analysis/4.1.2_qtl.analysis_cont_rz.age_ici.vs.eoi_snpsqc_pheno.corrected.cleaned_5.batches_52.Rmd
    Untracked:  analysis/4.1.2_qtl.analysis_cont_rz.age_ici.vs.eoi_snpsqc_pheno.corrected.cleaned_5.batches_mis_0.Rmd
    Untracked:  analysis/4.1.2_qtl.analysis_cont_rz.age_ici.vs.eoi_snpsqc_pheno.corrected.cleaned_5.batches_mis_52.Rmd
    Untracked:  analysis/4.1.2_qtl.analysis_cont_rz.age_ici.vs.eoi_snpsqc_pheno.corrected.cleaned_dis_no-xk_5.batches_0.Rmd
    Untracked:  analysis/4.1.2_qtl.analysis_cont_rz.age_ici.vs.eoi_snpsqc_pheno.corrected.cleaned_dis_no-xk_5.batches_52.Rmd
    Untracked:  analysis/4.1.2_qtl.analysis_cont_rz.age_ici.vs.eoi_snpsqc_pheno.corrected.cleaned_dis_no-xk_5.batches_mis_0.Rmd
    Untracked:  analysis/4.1.2_qtl.analysis_cont_rz.age_ici.vs.eoi_snpsqc_pheno.corrected.cleaned_dis_no-xk_5.batches_mis_52.Rmd
    Untracked:  analysis/4.1.2_qtl.analysis_cont_rz.age_ici.vs.pbs_pheno.corrected.cleaned_5.batches_conditional.Rmd.R
    Untracked:  analysis/4.1.2_qtl.analysis_cont_rz.age_ici.vs.pbs_pheno.corrected.cleaned_5.batches_mis_conditional.Rmd.R
    Untracked:  analysis/4.1.2_qtl.analysis_cont_rz.age_ici.vs.pbs_snpsqc_pheno.corrected.cleaned_5.batches_conditional.Rmd.R
    Untracked:  analysis/4.1.2_qtl.analysis_cont_rz.age_ici.vs.pbs_snpsqc_pheno.corrected.cleaned_5.batches_mis_conditional.Rmd.R
    Untracked:  analysis/4.1.2_qtl.analysis_cont_rz.age_ici.vs.pbs_snpsqc_pheno.corrected.cleaned_dis_no-xk_5.batches_conditional.Rmd.R
    Untracked:  analysis/4.1.2_qtl.analysis_cont_rz.age_ici.vs.pbs_snpsqc_pheno.corrected.cleaned_dis_no-xk_5.batches_mis_conditional.Rmd.R
    Untracked:  analysis/4.1.2_qtl.analysis_cont_rz.age_ici.vs.pbs_snpsqc_pheno.corrected.cleaned_dis_no-xk_5.batches_mis_conditional_1-peak.Rmd.R
    Untracked:  analysis/genotype.frequencies_ici-early.vs.pbs.ici-late_5.batches.Rmd
    Untracked:  analysis/genotype.frequencies_ici-early.vs.pbs.ici-late_5.batches_mis.Rmd
    Untracked:  analysis/genotype.frequencies_ici.vs.eoi_5.batches_0.Rmd
    Untracked:  analysis/genotype.frequencies_ici.vs.eoi_5.batches_52.Rmd
    Untracked:  analysis/genotype.frequencies_ici.vs.eoi_5.batches_mis_0.Rmd
    Untracked:  analysis/genotype.frequencies_ici.vs.eoi_5.batches_mis_52.Rmd
    Untracked:  analysis/index_5.batches_additional_vo.Rmd
    Untracked:  analysis/power.analysis_ici.vs.eoi_5.batches.Rmd
    Untracked:  analysis/power.analysis_ici.vs.eoi_5.batches.Rmd.R
    Untracked:  analysis/power.analysis_ici.vs.eoi_5.batches_mis.Rmd
    Untracked:  analysis/power.analysis_ici.vs.eoi_5.batches_mis.Rmd.R
    Untracked:  analysis/power.analysis_ici.vs.pbs_5.batches.Rmd
    Untracked:  analysis/power.analysis_ici.vs.pbs_5.batches.Rmd.R
    Untracked:  analysis/power.analysis_ici.vs.pbs_5.batches_mis.Rmd
    Untracked:  analysis/power.analysis_ici.vs.pbs_5.batches_mis.Rmd.R
    Untracked:  data/GM_covar.csv
    Untracked:  data/GM_covar_BC312.csv
    Untracked:  data/bad_markers_all_4.batches.RData
    Untracked:  data/bad_markers_all_5.batches.RData
    Untracked:  data/blup_sub_chr10_lod.drop-1.5_5.batches.csv
    Untracked:  data/blup_sub_chr3_lod.drop-1.5.csv
    Untracked:  data/blup_sub_chr3_lod.drop-1.5_5.batches.csv
    Untracked:  data/blup_sub_chr4_lod.drop-1.5.csv
    Untracked:  data/blup_sub_chr4_lod.drop-1.5_5.batches.csv
    Untracked:  data/covar_cleaned_ici.vs.eoi.csv
    Untracked:  data/covar_cleaned_ici.vs.pbs.csv
    Untracked:  data/covar_corrected.cleaned_ici-early.vs.pbs.ici-late_5.batches.csv
    Untracked:  data/covar_corrected.cleaned_ici-early.vs.pbs.ici-late_5.batches_mis.csv
    Untracked:  data/covar_corrected.cleaned_ici-early.vs.pbs_5.batches.csv
    Untracked:  data/covar_corrected.cleaned_ici-early.vs.pbs_5.batches_0.csv
    Untracked:  data/covar_corrected.cleaned_ici-early.vs.pbs_5.batches_52.csv
    Untracked:  data/covar_corrected.cleaned_ici-early.vs.pbs_5.batches_mis.csv
    Untracked:  data/covar_corrected.cleaned_ici-early.vs.pbs_5.batches_mis_0.csv
    Untracked:  data/covar_corrected.cleaned_ici-early.vs.pbs_5.batches_mis_52.csv
    Untracked:  data/covar_corrected.cleaned_ici.vs.eoi.csv
    Untracked:  data/covar_corrected.cleaned_ici.vs.eoi1.csv
    Untracked:  data/covar_corrected.cleaned_ici.vs.eoi_5.batches.csv
    Untracked:  data/covar_corrected.cleaned_ici.vs.eoi_5.batches_0.csv
    Untracked:  data/covar_corrected.cleaned_ici.vs.eoi_5.batches_52.csv
    Untracked:  data/covar_corrected.cleaned_ici.vs.eoi_5.batches_mis.csv
    Untracked:  data/covar_corrected.cleaned_ici.vs.eoi_5.batches_mis_0.csv
    Untracked:  data/covar_corrected.cleaned_ici.vs.eoi_5.batches_mis_52.csv
    Untracked:  data/covar_corrected.cleaned_ici.vs.pbs.csv
    Untracked:  data/covar_corrected.cleaned_ici.vs.pbs1.csv
    Untracked:  data/covar_corrected.cleaned_ici.vs.pbs_5.batches.csv
    Untracked:  data/covar_corrected.cleaned_ici.vs.pbs_5.batches_0.csv
    Untracked:  data/covar_corrected.cleaned_ici.vs.pbs_5.batches_52.csv
    Untracked:  data/covar_corrected.cleaned_ici.vs.pbs_5.batches_mis.csv
    Untracked:  data/covar_corrected.cleaned_ici.vs.pbs_5.batches_mis_0.csv
    Untracked:  data/covar_corrected.cleaned_ici.vs.pbs_5.batches_mis_52.csv
    Untracked:  data/covar_corrected_ici-early.vs.pbs.ici-late_5.batches.csv
    Untracked:  data/covar_corrected_ici-early.vs.pbs.ici-late_5.batches_mis.csv
    Untracked:  data/covar_corrected_ici-early.vs.pbs_5.batches.csv
    Untracked:  data/covar_corrected_ici-early.vs.pbs_5.batches_0.csv
    Untracked:  data/covar_corrected_ici-early.vs.pbs_5.batches_52.csv
    Untracked:  data/covar_corrected_ici-early.vs.pbs_5.batches_mis.csv
    Untracked:  data/covar_corrected_ici-early.vs.pbs_5.batches_mis_0.csv
    Untracked:  data/covar_corrected_ici-early.vs.pbs_5.batches_mis_52.csv
    Untracked:  data/covar_corrected_ici.vs.eoi.csv
    Untracked:  data/covar_corrected_ici.vs.eoi1.csv
    Untracked:  data/covar_corrected_ici.vs.eoi_5.batches.csv
    Untracked:  data/covar_corrected_ici.vs.eoi_5.batches_0.csv
    Untracked:  data/covar_corrected_ici.vs.eoi_5.batches_52.csv
    Untracked:  data/covar_corrected_ici.vs.eoi_5.batches_mis.csv
    Untracked:  data/covar_corrected_ici.vs.eoi_5.batches_mis_0.csv
    Untracked:  data/covar_corrected_ici.vs.eoi_5.batches_mis_52.csv
    Untracked:  data/covar_corrected_ici.vs.pbs.csv
    Untracked:  data/covar_corrected_ici.vs.pbs1.csv
    Untracked:  data/covar_corrected_ici.vs.pbs_5.batches.csv
    Untracked:  data/covar_corrected_ici.vs.pbs_5.batches_0.csv
    Untracked:  data/covar_corrected_ici.vs.pbs_5.batches_52.csv
    Untracked:  data/covar_corrected_ici.vs.pbs_5.batches_mis.csv
    Untracked:  data/covar_corrected_ici.vs.pbs_5.batches_mis_0.csv
    Untracked:  data/covar_corrected_ici.vs.pbs_5.batches_mis_52.csv
    Untracked:  data/e.RData
    Untracked:  data/e_BC312.RData
    Untracked:  data/e_snpg_samqc_4.batches.RData
    Untracked:  data/e_snpg_samqc_4.batches_bc.RData
    Untracked:  data/e_snpg_samqc_5.batches.RData
    Untracked:  data/errors_ind_4.batches.RData
    Untracked:  data/errors_ind_4.batches_bc.RData
    Untracked:  data/errors_ind_5.batches.RData
    Untracked:  data/files.to.sync.txt
    Untracked:  data/fitqtl_chr3.peak_chr4.peak_additive.txt
    Untracked:  data/fitqtl_chr3.peak_chr4.peak_interacting.txt
    Untracked:  data/fitqtl_chr3.peak_chr4.peak_sex_additive.txt
    Untracked:  data/fitqtl_chr3.peak_chr4.peak_sex_interacting.txt
    Untracked:  data/g2blup_effects.csv
    Untracked:  data/g2blup_effects.xlsx
    Untracked:  data/genes_chr10_lod.drop-1.5_5.batches.csv
    Untracked:  data/genes_chr3_lod.drop-1.5.csv
    Untracked:  data/genes_chr3_lod.drop-1.5_5.batches.csv
    Untracked:  data/genes_chr4_lod.drop-1.5.csv
    Untracked:  data/genetic_map.csv
    Untracked:  data/genetic_map_BC312.csv
    Untracked:  data/genotype_errors_marker_4.batches.RData
    Untracked:  data/genotype_errors_marker_5.batches.RData
    Untracked:  data/genotype_freq_marker_4.batches.RData
    Untracked:  data/genotype_freq_marker_5.batches.RData
    Untracked:  data/gm_allqc_4.batches.RData
    Untracked:  data/gm_allqc_5.batches.RData
    Untracked:  data/gm_allqc_5.batches_mis.RData
    Untracked:  data/gm_samqc_3.batches.RData
    Untracked:  data/gm_samqc_4.batches.RData
    Untracked:  data/gm_samqc_4.batches_bc.RData
    Untracked:  data/gm_samqc_5.batches.RData
    Untracked:  data/gm_serreze.192.RData
    Untracked:  data/gm_serreze.BC312.RData
    Untracked:  data/ici-early.vs.pbs.ici-late_age.of.onset-additive.covariates_blup_sub_chr-6_peak.marker-UNC11006703_lod.drop-1.5_snpsqc_dis_no-x_updated_5.batches.csv
    Untracked:  data/ici-early.vs.pbs.ici-late_age.of.onset-additive.covariates_blup_sub_chr-6_peak.marker-UNC11006703_lod.drop-1.5_snpsqc_dis_no-x_updated_5.batches_mis.csv
    Untracked:  data/ici-early.vs.pbs.ici-late_age.of.onset-additive.covariates_blup_sub_chr-6_peak.marker-cr27snv203_lod.drop-1.5_5.batches_mis.csv
    Untracked:  data/ici-early.vs.pbs.ici-late_age.of.onset-additive.covariates_genes_chr-6_peak.marker-UNC11006703_lod.drop-1.5_snpsqc_dis_no-x_updated_5.batches.csv
    Untracked:  data/ici-early.vs.pbs.ici-late_age.of.onset-additive.covariates_genes_chr-6_peak.marker-UNC11006703_lod.drop-1.5_snpsqc_dis_no-x_updated_5.batches_mis.csv
    Untracked:  data/ici-early.vs.pbs.ici-late_age.of.onset-additive.covariates_genes_chr-6_peak.marker-cr27snv203_lod.drop-1.5_5.batches_mis.csv
    Untracked:  data/ici-early.vs.pbs.ici-late_age.of.onset-interactive.covariate_blup_sub_chr-1_peak.marker-ICR134_lod.drop-1.5_5.batches_mis.csv
    Untracked:  data/ici-early.vs.pbs.ici-late_age.of.onset-interactive.covariate_blup_sub_chr-1_peak.marker-ICR4119_lod.drop-1.5_5.batches_mis.csv
    Untracked:  data/ici-early.vs.pbs.ici-late_age.of.onset-interactive.covariate_blup_sub_chr-1_peak.marker-JAX00262446_lod.drop-1.5_5.batches_mis.csv
    Untracked:  data/ici-early.vs.pbs.ici-late_age.of.onset-interactive.covariate_blup_sub_chr-1_peak.marker-UNCHS001938_lod.drop-1.5_5.batches_mis.csv
    Untracked:  data/ici-early.vs.pbs.ici-late_age.of.onset-interactive.covariate_blup_sub_chr-6_peak.marker-UNC11008761_lod.drop-1.5_5.batches_mis.csv
    Untracked:  data/ici-early.vs.pbs.ici-late_age.of.onset-interactive.covariate_blup_sub_chr-6_peak.marker-UNCHS016990_lod.drop-1.5_5.batches_mis.csv
    Untracked:  data/ici-early.vs.pbs.ici-late_age.of.onset-interactive.covariate_genes_chr-1_peak.marker-ICR134_lod.drop-1.5_5.batches_mis.csv
    Untracked:  data/ici-early.vs.pbs.ici-late_age.of.onset-interactive.covariate_genes_chr-1_peak.marker-ICR4119_lod.drop-1.5_5.batches_mis.csv
    Untracked:  data/ici-early.vs.pbs.ici-late_age.of.onset-interactive.covariate_genes_chr-1_peak.marker-JAX00262446_lod.drop-1.5_5.batches_mis.csv
    Untracked:  data/ici-early.vs.pbs.ici-late_age.of.onset-interactive.covariate_genes_chr-1_peak.marker-UNCHS001938_lod.drop-1.5_5.batches_mis.csv
    Untracked:  data/ici-early.vs.pbs.ici-late_age.of.onset-interactive.covariate_genes_chr-6_peak.marker-UNC11008761_lod.drop-1.5_5.batches_mis.csv
    Untracked:  data/ici-early.vs.pbs.ici-late_age.of.onset-interactive.covariate_genes_chr-6_peak.marker-UNCHS016990_lod.drop-1.5_5.batches_mis.csv
    Untracked:  data/ici-early.vs.pbs.ici-late_age.of.onset-no.covariates_blup_sub_chr-14_peak.marker-UNC24056202_lod.drop-1.5_5.batches_mis.csv
    Untracked:  data/ici-early.vs.pbs.ici-late_age.of.onset-no.covariates_blup_sub_chr-14_peak.marker-UNC24056202_lod.drop-1.5_snpsqc_5.batches_mis.csv
    Untracked:  data/ici-early.vs.pbs.ici-late_age.of.onset-no.covariates_blup_sub_chr-17_peak.marker-UNCJPD006614_lod.drop-1.5_snpsqc_5.batches_mis.csv
    Untracked:  data/ici-early.vs.pbs.ici-late_age.of.onset-no.covariates_blup_sub_chr-18_peak.marker-UNC28655293_lod.drop-1.5_5.batches_mis.csv
    Untracked:  data/ici-early.vs.pbs.ici-late_age.of.onset-no.covariates_blup_sub_chr-18_peak.marker-UNCHS045343_lod.drop-1.5_snpsqc_5.batches_mis.csv
    Untracked:  data/ici-early.vs.pbs.ici-late_age.of.onset-no.covariates_blup_sub_chr-5_peak.marker-UNC10044126_lod.drop-1.5_5.batches_mis.csv
    Untracked:  data/ici-early.vs.pbs.ici-late_age.of.onset-no.covariates_blup_sub_chr-5_peak.marker-UNC8675939_lod.drop-1.5_snpsqc_5.batches_mis.csv
    Untracked:  data/ici-early.vs.pbs.ici-late_age.of.onset-no.covariates_blup_sub_chr-8_peak.marker-JAX00667121_lod.drop-1.5_5.batches_mis.csv
    Untracked:  data/ici-early.vs.pbs.ici-late_age.of.onset-no.covariates_blup_sub_chr-8_peak.marker-UNC15524531_lod.drop-1.5_snpsqc_5.batches_mis.csv
    Untracked:  data/ici-early.vs.pbs.ici-late_age.of.onset-no.covariates_blup_sub_chr-X_peak.marker-UNCHS048314_lod.drop-1.5_snpsqc_5.batches_mis.csv
    Untracked:  data/ici-early.vs.pbs.ici-late_age.of.onset-no.covariates_genes_chr-14_peak.marker-UNC24056202_lod.drop-1.5_5.batches_mis.csv
    Untracked:  data/ici-early.vs.pbs.ici-late_age.of.onset-no.covariates_genes_chr-14_peak.marker-UNC24056202_lod.drop-1.5_snpsqc_5.batches_mis.csv
    Untracked:  data/ici-early.vs.pbs.ici-late_age.of.onset-no.covariates_genes_chr-17_peak.marker-UNCJPD006614_lod.drop-1.5_snpsqc_5.batches_mis.csv
    Untracked:  data/ici-early.vs.pbs.ici-late_age.of.onset-no.covariates_genes_chr-18_peak.marker-UNC28655293_lod.drop-1.5_5.batches_mis.csv
    Untracked:  data/ici-early.vs.pbs.ici-late_age.of.onset-no.covariates_genes_chr-18_peak.marker-UNCHS045343_lod.drop-1.5_snpsqc_5.batches_mis.csv
    Untracked:  data/ici-early.vs.pbs.ici-late_age.of.onset-no.covariates_genes_chr-5_peak.marker-UNC10044126_lod.drop-1.5_5.batches_mis.csv
    Untracked:  data/ici-early.vs.pbs.ici-late_age.of.onset-no.covariates_genes_chr-5_peak.marker-UNC8675939_lod.drop-1.5_snpsqc_5.batches_mis.csv
    Untracked:  data/ici-early.vs.pbs.ici-late_age.of.onset-no.covariates_genes_chr-8_peak.marker-JAX00667121_lod.drop-1.5_5.batches_mis.csv
    Untracked:  data/ici-early.vs.pbs.ici-late_age.of.onset-no.covariates_genes_chr-8_peak.marker-UNC15524531_lod.drop-1.5_snpsqc_5.batches_mis.csv
    Untracked:  data/ici-early.vs.pbs.ici-late_age.of.onset-no.covariates_genes_chr-X_peak.marker-UNCHS048314_lod.drop-1.5_snpsqc_5.batches_mis.csv
    Untracked:  data/ici-early.vs.pbs.ici-late_blup.full_chr-10_5.batches.csv
    Untracked:  data/ici-early.vs.pbs.ici-late_blup.full_chr-10_5.batches_mis.csv
    Untracked:  data/ici-early.vs.pbs.ici-late_blup.full_chr-10_snpsqc_5.batches.csv
    Untracked:  data/ici-early.vs.pbs.ici-late_blup.full_chr-10_snpsqc_5.batches_mis.csv
    Untracked:  data/ici-early.vs.pbs.ici-late_blup.full_chr-10_snpsqc_dis_no-x_updated_5.batches.csv
    Untracked:  data/ici-early.vs.pbs.ici-late_blup.full_chr-10_snpsqc_dis_no-x_updated_5.batches_mis.csv
    Untracked:  data/ici-early.vs.pbs.ici-late_blup.full_chr-11_5.batches.csv
    Untracked:  data/ici-early.vs.pbs.ici-late_blup.full_chr-11_5.batches_mis.csv
    Untracked:  data/ici-early.vs.pbs.ici-late_blup.full_chr-11_snpsqc_5.batches.csv
    Untracked:  data/ici-early.vs.pbs.ici-late_blup.full_chr-11_snpsqc_5.batches_mis.csv
    Untracked:  data/ici-early.vs.pbs.ici-late_blup.full_chr-11_snpsqc_dis_no-x_updated_5.batches.csv
    Untracked:  data/ici-early.vs.pbs.ici-late_blup.full_chr-11_snpsqc_dis_no-x_updated_5.batches_mis.csv
    Untracked:  data/ici-early.vs.pbs.ici-late_blup.full_chr-12_5.batches.csv
    Untracked:  data/ici-early.vs.pbs.ici-late_blup.full_chr-12_5.batches_mis.csv
    Untracked:  data/ici-early.vs.pbs.ici-late_blup.full_chr-12_snpsqc_5.batches.csv
    Untracked:  data/ici-early.vs.pbs.ici-late_blup.full_chr-12_snpsqc_5.batches_mis.csv
    Untracked:  data/ici-early.vs.pbs.ici-late_blup.full_chr-12_snpsqc_dis_no-x_updated_5.batches.csv
    Untracked:  data/ici-early.vs.pbs.ici-late_blup.full_chr-12_snpsqc_dis_no-x_updated_5.batches_mis.csv
    Untracked:  data/ici-early.vs.pbs.ici-late_blup.full_chr-13_5.batches.csv
    Untracked:  data/ici-early.vs.pbs.ici-late_blup.full_chr-13_5.batches_mis.csv
    Untracked:  data/ici-early.vs.pbs.ici-late_blup.full_chr-13_snpsqc_5.batches.csv
    Untracked:  data/ici-early.vs.pbs.ici-late_blup.full_chr-13_snpsqc_5.batches_mis.csv
    Untracked:  data/ici-early.vs.pbs.ici-late_blup.full_chr-13_snpsqc_dis_no-x_updated_5.batches.csv
    Untracked:  data/ici-early.vs.pbs.ici-late_blup.full_chr-13_snpsqc_dis_no-x_updated_5.batches_mis.csv
    Untracked:  data/ici-early.vs.pbs.ici-late_blup.full_chr-14_5.batches.csv
    Untracked:  data/ici-early.vs.pbs.ici-late_blup.full_chr-14_5.batches_mis.csv
    Untracked:  data/ici-early.vs.pbs.ici-late_blup.full_chr-14_snpsqc_5.batches.csv
    Untracked:  data/ici-early.vs.pbs.ici-late_blup.full_chr-14_snpsqc_5.batches_mis.csv
    Untracked:  data/ici-early.vs.pbs.ici-late_blup.full_chr-14_snpsqc_dis_no-x_updated_5.batches.csv
    Untracked:  data/ici-early.vs.pbs.ici-late_blup.full_chr-14_snpsqc_dis_no-x_updated_5.batches_mis.csv
    Untracked:  data/ici-early.vs.pbs.ici-late_blup.full_chr-15_5.batches.csv
    Untracked:  data/ici-early.vs.pbs.ici-late_blup.full_chr-15_5.batches_mis.csv
    Untracked:  data/ici-early.vs.pbs.ici-late_blup.full_chr-15_snpsqc_5.batches.csv
    Untracked:  data/ici-early.vs.pbs.ici-late_blup.full_chr-15_snpsqc_5.batches_mis.csv
    Untracked:  data/ici-early.vs.pbs.ici-late_blup.full_chr-15_snpsqc_dis_no-x_updated_5.batches.csv
    Untracked:  data/ici-early.vs.pbs.ici-late_blup.full_chr-15_snpsqc_dis_no-x_updated_5.batches_mis.csv
    Untracked:  data/ici-early.vs.pbs.ici-late_blup.full_chr-16_5.batches.csv
    Untracked:  data/ici-early.vs.pbs.ici-late_blup.full_chr-16_5.batches_mis.csv
    Untracked:  data/ici-early.vs.pbs.ici-late_blup.full_chr-16_snpsqc_5.batches.csv
    Untracked:  data/ici-early.vs.pbs.ici-late_blup.full_chr-16_snpsqc_5.batches_mis.csv
    Untracked:  data/ici-early.vs.pbs.ici-late_blup.full_chr-16_snpsqc_dis_no-x_updated_5.batches.csv
    Untracked:  data/ici-early.vs.pbs.ici-late_blup.full_chr-16_snpsqc_dis_no-x_updated_5.batches_mis.csv
    Untracked:  data/ici-early.vs.pbs.ici-late_blup.full_chr-17_5.batches.csv
    Untracked:  data/ici-early.vs.pbs.ici-late_blup.full_chr-17_5.batches_mis.csv
    Untracked:  data/ici-early.vs.pbs.ici-late_blup.full_chr-17_snpsqc_5.batches.csv
    Untracked:  data/ici-early.vs.pbs.ici-late_blup.full_chr-17_snpsqc_5.batches_mis.csv
    Untracked:  data/ici-early.vs.pbs.ici-late_blup.full_chr-17_snpsqc_dis_no-x_updated_5.batches.csv
    Untracked:  data/ici-early.vs.pbs.ici-late_blup.full_chr-17_snpsqc_dis_no-x_updated_5.batches_mis.csv
    Untracked:  data/ici-early.vs.pbs.ici-late_blup.full_chr-18_5.batches.csv
    Untracked:  data/ici-early.vs.pbs.ici-late_blup.full_chr-18_5.batches_mis.csv
    Untracked:  data/ici-early.vs.pbs.ici-late_blup.full_chr-18_snpsqc_5.batches.csv
    Untracked:  data/ici-early.vs.pbs.ici-late_blup.full_chr-18_snpsqc_5.batches_mis.csv
    Untracked:  data/ici-early.vs.pbs.ici-late_blup.full_chr-18_snpsqc_dis_no-x_updated_5.batches.csv
    Untracked:  data/ici-early.vs.pbs.ici-late_blup.full_chr-18_snpsqc_dis_no-x_updated_5.batches_mis.csv
    Untracked:  data/ici-early.vs.pbs.ici-late_blup.full_chr-19_5.batches.csv
    Untracked:  data/ici-early.vs.pbs.ici-late_blup.full_chr-19_5.batches_mis.csv
    Untracked:  data/ici-early.vs.pbs.ici-late_blup.full_chr-19_snpsqc_5.batches.csv
    Untracked:  data/ici-early.vs.pbs.ici-late_blup.full_chr-19_snpsqc_5.batches_mis.csv
    Untracked:  data/ici-early.vs.pbs.ici-late_blup.full_chr-19_snpsqc_dis_no-x_updated_5.batches.csv
    Untracked:  data/ici-early.vs.pbs.ici-late_blup.full_chr-19_snpsqc_dis_no-x_updated_5.batches_mis.csv
    Untracked:  data/ici-early.vs.pbs.ici-late_blup.full_chr-1_5.batches.csv
    Untracked:  data/ici-early.vs.pbs.ici-late_blup.full_chr-1_5.batches_mis.csv
    Untracked:  data/ici-early.vs.pbs.ici-late_blup.full_chr-1_snpsqc_5.batches.csv
    Untracked:  data/ici-early.vs.pbs.ici-late_blup.full_chr-1_snpsqc_5.batches_mis.csv
    Untracked:  data/ici-early.vs.pbs.ici-late_blup.full_chr-1_snpsqc_dis_no-x_updated_5.batches.csv
    Untracked:  data/ici-early.vs.pbs.ici-late_blup.full_chr-1_snpsqc_dis_no-x_updated_5.batches_mis.csv
    Untracked:  data/ici-early.vs.pbs.ici-late_blup.full_chr-2_5.batches.csv
    Untracked:  data/ici-early.vs.pbs.ici-late_blup.full_chr-2_5.batches_mis.csv
    Untracked:  data/ici-early.vs.pbs.ici-late_blup.full_chr-2_snpsqc_5.batches.csv
    Untracked:  data/ici-early.vs.pbs.ici-late_blup.full_chr-2_snpsqc_5.batches_mis.csv
    Untracked:  data/ici-early.vs.pbs.ici-late_blup.full_chr-2_snpsqc_dis_no-x_updated_5.batches.csv
    Untracked:  data/ici-early.vs.pbs.ici-late_blup.full_chr-2_snpsqc_dis_no-x_updated_5.batches_mis.csv
    Untracked:  data/ici-early.vs.pbs.ici-late_blup.full_chr-3_5.batches.csv
    Untracked:  data/ici-early.vs.pbs.ici-late_blup.full_chr-3_5.batches_mis.csv
    Untracked:  data/ici-early.vs.pbs.ici-late_blup.full_chr-3_snpsqc_5.batches.csv
    Untracked:  data/ici-early.vs.pbs.ici-late_blup.full_chr-3_snpsqc_5.batches_mis.csv
    Untracked:  data/ici-early.vs.pbs.ici-late_blup.full_chr-3_snpsqc_dis_no-x_updated_5.batches.csv
    Untracked:  data/ici-early.vs.pbs.ici-late_blup.full_chr-3_snpsqc_dis_no-x_updated_5.batches_mis.csv
    Untracked:  data/ici-early.vs.pbs.ici-late_blup.full_chr-4_5.batches.csv
    Untracked:  data/ici-early.vs.pbs.ici-late_blup.full_chr-4_5.batches_mis.csv
    Untracked:  data/ici-early.vs.pbs.ici-late_blup.full_chr-4_snpsqc_5.batches.csv
    Untracked:  data/ici-early.vs.pbs.ici-late_blup.full_chr-4_snpsqc_5.batches_mis.csv
    Untracked:  data/ici-early.vs.pbs.ici-late_blup.full_chr-4_snpsqc_dis_no-x_updated_5.batches.csv
    Untracked:  data/ici-early.vs.pbs.ici-late_blup.full_chr-4_snpsqc_dis_no-x_updated_5.batches_mis.csv
    Untracked:  data/ici-early.vs.pbs.ici-late_blup.full_chr-5_5.batches.csv
    Untracked:  data/ici-early.vs.pbs.ici-late_blup.full_chr-5_5.batches_mis.csv
    Untracked:  data/ici-early.vs.pbs.ici-late_blup.full_chr-5_snpsqc_5.batches.csv
    Untracked:  data/ici-early.vs.pbs.ici-late_blup.full_chr-5_snpsqc_5.batches_mis.csv
    Untracked:  data/ici-early.vs.pbs.ici-late_blup.full_chr-5_snpsqc_dis_no-x_updated_5.batches.csv
    Untracked:  data/ici-early.vs.pbs.ici-late_blup.full_chr-5_snpsqc_dis_no-x_updated_5.batches_mis.csv
    Untracked:  data/ici-early.vs.pbs.ici-late_blup.full_chr-6_5.batches.csv
    Untracked:  data/ici-early.vs.pbs.ici-late_blup.full_chr-6_5.batches_mis.csv
    Untracked:  data/ici-early.vs.pbs.ici-late_blup.full_chr-6_snpsqc_5.batches.csv
    Untracked:  data/ici-early.vs.pbs.ici-late_blup.full_chr-6_snpsqc_5.batches_mis.csv
    Untracked:  data/ici-early.vs.pbs.ici-late_blup.full_chr-6_snpsqc_dis_no-x_updated_5.batches.csv
    Untracked:  data/ici-early.vs.pbs.ici-late_blup.full_chr-6_snpsqc_dis_no-x_updated_5.batches_mis.csv
    Untracked:  data/ici-early.vs.pbs.ici-late_blup.full_chr-7_5.batches.csv
    Untracked:  data/ici-early.vs.pbs.ici-late_blup.full_chr-7_5.batches_mis.csv
    Untracked:  data/ici-early.vs.pbs.ici-late_blup.full_chr-7_snpsqc_5.batches.csv
    Untracked:  data/ici-early.vs.pbs.ici-late_blup.full_chr-7_snpsqc_5.batches_mis.csv
    Untracked:  data/ici-early.vs.pbs.ici-late_blup.full_chr-7_snpsqc_dis_no-x_updated_5.batches.csv
    Untracked:  data/ici-early.vs.pbs.ici-late_blup.full_chr-7_snpsqc_dis_no-x_updated_5.batches_mis.csv
    Untracked:  data/ici-early.vs.pbs.ici-late_blup.full_chr-8_5.batches.csv
    Untracked:  data/ici-early.vs.pbs.ici-late_blup.full_chr-8_5.batches_mis.csv
    Untracked:  data/ici-early.vs.pbs.ici-late_blup.full_chr-8_snpsqc_5.batches.csv
    Untracked:  data/ici-early.vs.pbs.ici-late_blup.full_chr-8_snpsqc_5.batches_mis.csv
    Untracked:  data/ici-early.vs.pbs.ici-late_blup.full_chr-8_snpsqc_dis_no-x_updated_5.batches.csv
    Untracked:  data/ici-early.vs.pbs.ici-late_blup.full_chr-8_snpsqc_dis_no-x_updated_5.batches_mis.csv
    Untracked:  data/ici-early.vs.pbs.ici-late_blup.full_chr-9_5.batches.csv
    Untracked:  data/ici-early.vs.pbs.ici-late_blup.full_chr-9_5.batches_mis.csv
    Untracked:  data/ici-early.vs.pbs.ici-late_blup.full_chr-9_snpsqc_5.batches.csv
    Untracked:  data/ici-early.vs.pbs.ici-late_blup.full_chr-9_snpsqc_5.batches_mis.csv
    Untracked:  data/ici-early.vs.pbs.ici-late_blup.full_chr-9_snpsqc_dis_no-x_updated_5.batches.csv
    Untracked:  data/ici-early.vs.pbs.ici-late_blup.full_chr-9_snpsqc_dis_no-x_updated_5.batches_mis.csv
    Untracked:  data/ici-early.vs.pbs.ici-late_blup.full_chr-X_5.batches.csv
    Untracked:  data/ici-early.vs.pbs.ici-late_blup.full_chr-X_5.batches_mis.csv
    Untracked:  data/ici-early.vs.pbs.ici-late_blup.full_chr-X_snpsqc_5.batches.csv
    Untracked:  data/ici-early.vs.pbs.ici-late_blup.full_chr-X_snpsqc_5.batches_mis.csv
    Untracked:  data/ici-early.vs.pbs.ici-late_blup.full_chr-X_snpsqc_dis_no-x_updated_5.batches.csv
    Untracked:  data/ici-early.vs.pbs.ici-late_blup.full_chr-X_snpsqc_dis_no-x_updated_5.batches_mis.csv
    Untracked:  data/ici-early.vs.pbs.ici-late_blup.qc_chr-10_5.batches.csv
    Untracked:  data/ici-early.vs.pbs.ici-late_blup.qc_chr-10_5.batches_mis.csv
    Untracked:  data/ici-early.vs.pbs.ici-late_blup.qc_chr-10_snpsqc_5.batches.csv
    Untracked:  data/ici-early.vs.pbs.ici-late_blup.qc_chr-10_snpsqc_5.batches_mis.csv
    Untracked:  data/ici-early.vs.pbs.ici-late_blup.qc_chr-10_snpsqc_dis_no-x_updated_5.batches.csv
    Untracked:  data/ici-early.vs.pbs.ici-late_blup.qc_chr-10_snpsqc_dis_no-x_updated_5.batches_mis.csv
    Untracked:  data/ici-early.vs.pbs.ici-late_blup.qc_chr-11_5.batches.csv
    Untracked:  data/ici-early.vs.pbs.ici-late_blup.qc_chr-11_5.batches_mis.csv
    Untracked:  data/ici-early.vs.pbs.ici-late_blup.qc_chr-11_snpsqc_5.batches.csv
    Untracked:  data/ici-early.vs.pbs.ici-late_blup.qc_chr-11_snpsqc_5.batches_mis.csv
    Untracked:  data/ici-early.vs.pbs.ici-late_blup.qc_chr-11_snpsqc_dis_no-x_updated_5.batches.csv
    Untracked:  data/ici-early.vs.pbs.ici-late_blup.qc_chr-11_snpsqc_dis_no-x_updated_5.batches_mis.csv
    Untracked:  data/ici-early.vs.pbs.ici-late_blup.qc_chr-12_5.batches.csv
    Untracked:  data/ici-early.vs.pbs.ici-late_blup.qc_chr-12_5.batches_mis.csv
    Untracked:  data/ici-early.vs.pbs.ici-late_blup.qc_chr-12_snpsqc_5.batches.csv
    Untracked:  data/ici-early.vs.pbs.ici-late_blup.qc_chr-12_snpsqc_5.batches_mis.csv
    Untracked:  data/ici-early.vs.pbs.ici-late_blup.qc_chr-12_snpsqc_dis_no-x_updated_5.batches.csv
    Untracked:  data/ici-early.vs.pbs.ici-late_blup.qc_chr-12_snpsqc_dis_no-x_updated_5.batches_mis.csv
    Untracked:  data/ici-early.vs.pbs.ici-late_blup.qc_chr-13_5.batches.csv
    Untracked:  data/ici-early.vs.pbs.ici-late_blup.qc_chr-13_5.batches_mis.csv
    Untracked:  data/ici-early.vs.pbs.ici-late_blup.qc_chr-13_snpsqc_5.batches.csv
    Untracked:  data/ici-early.vs.pbs.ici-late_blup.qc_chr-13_snpsqc_5.batches_mis.csv
    Untracked:  data/ici-early.vs.pbs.ici-late_blup.qc_chr-13_snpsqc_dis_no-x_updated_5.batches.csv
    Untracked:  data/ici-early.vs.pbs.ici-late_blup.qc_chr-13_snpsqc_dis_no-x_updated_5.batches_mis.csv
    Untracked:  data/ici-early.vs.pbs.ici-late_blup.qc_chr-14_5.batches.csv
    Untracked:  data/ici-early.vs.pbs.ici-late_blup.qc_chr-14_5.batches_mis.csv
    Untracked:  data/ici-early.vs.pbs.ici-late_blup.qc_chr-14_snpsqc_5.batches.csv
    Untracked:  data/ici-early.vs.pbs.ici-late_blup.qc_chr-14_snpsqc_5.batches_mis.csv
    Untracked:  data/ici-early.vs.pbs.ici-late_blup.qc_chr-14_snpsqc_dis_no-x_updated_5.batches.csv
    Untracked:  data/ici-early.vs.pbs.ici-late_blup.qc_chr-14_snpsqc_dis_no-x_updated_5.batches_mis.csv
    Untracked:  data/ici-early.vs.pbs.ici-late_blup.qc_chr-15_5.batches.csv
    Untracked:  data/ici-early.vs.pbs.ici-late_blup.qc_chr-15_5.batches_mis.csv
    Untracked:  data/ici-early.vs.pbs.ici-late_blup.qc_chr-15_snpsqc_5.batches.csv
    Untracked:  data/ici-early.vs.pbs.ici-late_blup.qc_chr-15_snpsqc_5.batches_mis.csv
    Untracked:  data/ici-early.vs.pbs.ici-late_blup.qc_chr-16_5.batches.csv
    Untracked:  data/ici-early.vs.pbs.ici-late_blup.qc_chr-16_5.batches_mis.csv
    Untracked:  data/ici-early.vs.pbs.ici-late_blup.qc_chr-16_snpsqc_5.batches.csv
    Untracked:  data/ici-early.vs.pbs.ici-late_blup.qc_chr-16_snpsqc_5.batches_mis.csv
    Untracked:  data/ici-early.vs.pbs.ici-late_blup.qc_chr-16_snpsqc_dis_no-x_updated_5.batches.csv
    Untracked:  data/ici-early.vs.pbs.ici-late_blup.qc_chr-16_snpsqc_dis_no-x_updated_5.batches_mis.csv
    Untracked:  data/ici-early.vs.pbs.ici-late_blup.qc_chr-17_5.batches.csv
    Untracked:  data/ici-early.vs.pbs.ici-late_blup.qc_chr-17_5.batches_mis.csv
    Untracked:  data/ici-early.vs.pbs.ici-late_blup.qc_chr-17_snpsqc_5.batches.csv
    Untracked:  data/ici-early.vs.pbs.ici-late_blup.qc_chr-17_snpsqc_5.batches_mis.csv
    Untracked:  data/ici-early.vs.pbs.ici-late_blup.qc_chr-18_5.batches.csv
    Untracked:  data/ici-early.vs.pbs.ici-late_blup.qc_chr-18_5.batches_mis.csv
    Untracked:  data/ici-early.vs.pbs.ici-late_blup.qc_chr-18_snpsqc_5.batches.csv
    Untracked:  data/ici-early.vs.pbs.ici-late_blup.qc_chr-18_snpsqc_5.batches_mis.csv
    Untracked:  data/ici-early.vs.pbs.ici-late_blup.qc_chr-18_snpsqc_dis_no-x_updated_5.batches.csv
    Untracked:  data/ici-early.vs.pbs.ici-late_blup.qc_chr-18_snpsqc_dis_no-x_updated_5.batches_mis.csv
    Untracked:  data/ici-early.vs.pbs.ici-late_blup.qc_chr-19_5.batches.csv
    Untracked:  data/ici-early.vs.pbs.ici-late_blup.qc_chr-19_5.batches_mis.csv
    Untracked:  data/ici-early.vs.pbs.ici-late_blup.qc_chr-19_snpsqc_5.batches.csv
    Untracked:  data/ici-early.vs.pbs.ici-late_blup.qc_chr-19_snpsqc_5.batches_mis.csv
    Untracked:  data/ici-early.vs.pbs.ici-late_blup.qc_chr-19_snpsqc_dis_no-x_updated_5.batches.csv
    Untracked:  data/ici-early.vs.pbs.ici-late_blup.qc_chr-19_snpsqc_dis_no-x_updated_5.batches_mis.csv
    Untracked:  data/ici-early.vs.pbs.ici-late_blup.qc_chr-1_5.batches.csv
    Untracked:  data/ici-early.vs.pbs.ici-late_blup.qc_chr-1_5.batches_mis.csv
    Untracked:  data/ici-early.vs.pbs.ici-late_blup.qc_chr-1_snpsqc_5.batches.csv
    Untracked:  data/ici-early.vs.pbs.ici-late_blup.qc_chr-1_snpsqc_5.batches_mis.csv
    Untracked:  data/ici-early.vs.pbs.ici-late_blup.qc_chr-1_snpsqc_dis_no-x_updated_5.batches.csv
    Untracked:  data/ici-early.vs.pbs.ici-late_blup.qc_chr-1_snpsqc_dis_no-x_updated_5.batches_mis.csv
    Untracked:  data/ici-early.vs.pbs.ici-late_blup.qc_chr-2_5.batches.csv
    Untracked:  data/ici-early.vs.pbs.ici-late_blup.qc_chr-2_5.batches_mis.csv
    Untracked:  data/ici-early.vs.pbs.ici-late_blup.qc_chr-2_snpsqc_5.batches.csv
    Untracked:  data/ici-early.vs.pbs.ici-late_blup.qc_chr-2_snpsqc_5.batches_mis.csv
    Untracked:  data/ici-early.vs.pbs.ici-late_blup.qc_chr-2_snpsqc_dis_no-x_updated_5.batches.csv
    Untracked:  data/ici-early.vs.pbs.ici-late_blup.qc_chr-2_snpsqc_dis_no-x_updated_5.batches_mis.csv
    Untracked:  data/ici-early.vs.pbs.ici-late_blup.qc_chr-3_5.batches.csv
    Untracked:  data/ici-early.vs.pbs.ici-late_blup.qc_chr-3_5.batches_mis.csv
    Untracked:  data/ici-early.vs.pbs.ici-late_blup.qc_chr-3_snpsqc_5.batches.csv
    Untracked:  data/ici-early.vs.pbs.ici-late_blup.qc_chr-3_snpsqc_5.batches_mis.csv
    Untracked:  data/ici-early.vs.pbs.ici-late_blup.qc_chr-3_snpsqc_dis_no-x_updated_5.batches.csv
    Untracked:  data/ici-early.vs.pbs.ici-late_blup.qc_chr-3_snpsqc_dis_no-x_updated_5.batches_mis.csv
    Untracked:  data/ici-early.vs.pbs.ici-late_blup.qc_chr-4_5.batches.csv
    Untracked:  data/ici-early.vs.pbs.ici-late_blup.qc_chr-4_5.batches_mis.csv
    Untracked:  data/ici-early.vs.pbs.ici-late_blup.qc_chr-4_snpsqc_5.batches.csv
    Untracked:  data/ici-early.vs.pbs.ici-late_blup.qc_chr-4_snpsqc_5.batches_mis.csv
    Untracked:  data/ici-early.vs.pbs.ici-late_blup.qc_chr-4_snpsqc_dis_no-x_updated_5.batches.csv
    Untracked:  data/ici-early.vs.pbs.ici-late_blup.qc_chr-4_snpsqc_dis_no-x_updated_5.batches_mis.csv
    Untracked:  data/ici-early.vs.pbs.ici-late_blup.qc_chr-5_5.batches.csv
    Untracked:  data/ici-early.vs.pbs.ici-late_blup.qc_chr-5_5.batches_mis.csv
    Untracked:  data/ici-early.vs.pbs.ici-late_blup.qc_chr-5_snpsqc_5.batches.csv
    Untracked:  data/ici-early.vs.pbs.ici-late_blup.qc_chr-5_snpsqc_5.batches_mis.csv
    Untracked:  data/ici-early.vs.pbs.ici-late_blup.qc_chr-5_snpsqc_dis_no-x_updated_5.batches.csv
    Untracked:  data/ici-early.vs.pbs.ici-late_blup.qc_chr-5_snpsqc_dis_no-x_updated_5.batches_mis.csv
    Untracked:  data/ici-early.vs.pbs.ici-late_blup.qc_chr-6_5.batches.csv
    Untracked:  data/ici-early.vs.pbs.ici-late_blup.qc_chr-6_5.batches_mis.csv
    Untracked:  data/ici-early.vs.pbs.ici-late_blup.qc_chr-6_snpsqc_5.batches.csv
    Untracked:  data/ici-early.vs.pbs.ici-late_blup.qc_chr-6_snpsqc_5.batches_mis.csv
    Untracked:  data/ici-early.vs.pbs.ici-late_blup.qc_chr-6_snpsqc_dis_no-x_updated_5.batches.csv
    Untracked:  data/ici-early.vs.pbs.ici-late_blup.qc_chr-6_snpsqc_dis_no-x_updated_5.batches_mis.csv
    Untracked:  data/ici-early.vs.pbs.ici-late_blup.qc_chr-7_5.batches.csv
    Untracked:  data/ici-early.vs.pbs.ici-late_blup.qc_chr-7_5.batches_mis.csv
    Untracked:  data/ici-early.vs.pbs.ici-late_blup.qc_chr-7_snpsqc_5.batches.csv
    Untracked:  data/ici-early.vs.pbs.ici-late_blup.qc_chr-7_snpsqc_5.batches_mis.csv
    Untracked:  data/ici-early.vs.pbs.ici-late_blup.qc_chr-7_snpsqc_dis_no-x_updated_5.batches.csv
    Untracked:  data/ici-early.vs.pbs.ici-late_blup.qc_chr-7_snpsqc_dis_no-x_updated_5.batches_mis.csv
    Untracked:  data/ici-early.vs.pbs.ici-late_blup.qc_chr-8_5.batches.csv
    Untracked:  data/ici-early.vs.pbs.ici-late_blup.qc_chr-8_5.batches_mis.csv
    Untracked:  data/ici-early.vs.pbs.ici-late_blup.qc_chr-8_snpsqc_5.batches.csv
    Untracked:  data/ici-early.vs.pbs.ici-late_blup.qc_chr-8_snpsqc_5.batches_mis.csv
    Untracked:  data/ici-early.vs.pbs.ici-late_blup.qc_chr-8_snpsqc_dis_no-x_updated_5.batches.csv
    Untracked:  data/ici-early.vs.pbs.ici-late_blup.qc_chr-8_snpsqc_dis_no-x_updated_5.batches_mis.csv
    Untracked:  data/ici-early.vs.pbs.ici-late_blup.qc_chr-9_5.batches.csv
    Untracked:  data/ici-early.vs.pbs.ici-late_blup.qc_chr-9_5.batches_mis.csv
    Untracked:  data/ici-early.vs.pbs.ici-late_blup.qc_chr-9_snpsqc_5.batches.csv
    Untracked:  data/ici-early.vs.pbs.ici-late_blup.qc_chr-9_snpsqc_5.batches_mis.csv
    Untracked:  data/ici-early.vs.pbs.ici-late_blup.qc_chr-9_snpsqc_dis_no-x_updated_5.batches.csv
    Untracked:  data/ici-early.vs.pbs.ici-late_blup.qc_chr-9_snpsqc_dis_no-x_updated_5.batches_mis.csv
    Untracked:  data/ici-early.vs.pbs.ici-late_blup.qc_chr-X_5.batches.csv
    Untracked:  data/ici-early.vs.pbs.ici-late_blup.qc_chr-X_5.batches_mis.csv
    Untracked:  data/ici-early.vs.pbs.ici-late_blup.qc_chr-X_snpsqc_5.batches.csv
    Untracked:  data/ici-early.vs.pbs.ici-late_blup.qc_chr-X_snpsqc_5.batches_mis.csv
    Untracked:  data/ici-early.vs.pbs.ici-late_blup.qc_chr-X_snpsqc_dis_no-x_updated_5.batches.csv
    Untracked:  data/ici-early.vs.pbs.ici-late_blup.qc_chr-X_snpsqc_dis_no-x_updated_5.batches_mis.csv
    Untracked:  data/ici-early.vs.pbs.ici-late_blup_sub_chr-18_peak.marker-UNCHS045343_lod.drop-1.5_snpsqc_5.batches_mis.csv
    Untracked:  data/ici-early.vs.pbs.ici-late_genes_chr-18_peak.marker-UNCHS045343_lod.drop-1.5_snpsqc_5.batches_mis.csv
    Untracked:  data/ici-early.vs.pbs.ici-late_gm_qtl_5.batches.csv
    Untracked:  data/ici-early.vs.pbs.ici-late_gm_qtl_5.batches_mis.csv
    Untracked:  data/ici-early.vs.pbs.ici-late_gm_qtl_snpsqc_5.batches.csv
    Untracked:  data/ici-early.vs.pbs.ici-late_gm_qtl_snpsqc_5.batches_mis.csv
    Untracked:  data/ici-early.vs.pbs.ici-late_gm_qtl_snpsqc_dis_no-x_updated_5.batches.csv
    Untracked:  data/ici-early.vs.pbs.ici-late_gm_qtl_snpsqc_dis_no-x_updated_5.batches_mis.csv
    Untracked:  data/ici-early.vs.pbs.ici-late_marker.freq_low.geno.freq.removed_geno.ratio_5.batches.csv
    Untracked:  data/ici-early.vs.pbs.ici-late_marker.freq_low.geno.freq.removed_geno.ratio_5.batches_mis.csv
    Untracked:  data/ici-early.vs.pbs.ici-late_marker.freq_low.geno.freq.removed_sample.outliers.removed_geno.ratio_5.batches.csv
    Untracked:  data/ici-early.vs.pbs.ici-late_marker.freq_low.geno.freq.removed_sample.outliers.removed_geno.ratio_5.batches_mis.csv
    Untracked:  data/ici-early.vs.pbs.ici-late_marker.freq_low.probs.freq.removed_geno.ratio_5.batches.csv
    Untracked:  data/ici-early.vs.pbs.ici-late_marker.freq_low.probs.freq.removed_geno.ratio_5.batches_mis.csv
    Untracked:  data/ici-early.vs.pbs.ici-late_marker.freq_low.probs.freq.removed_sample.outliers.removed_geno.ratio_5.batches.csv
    Untracked:  data/ici-early.vs.pbs.ici-late_marker.freq_low.probs.freq.removed_sample.outliers.removed_geno.ratio_5.batches_mis.csv
    Untracked:  data/ici-early.vs.pbs.ici-late_scanone_5.batches.Rdata
    Untracked:  data/ici-early.vs.pbs.ici-late_scanone_5.batches_mis.Rdata
    Untracked:  data/ici-early.vs.pbs.ici-late_scanone_snpsqc_5.batches.Rdata
    Untracked:  data/ici-early.vs.pbs.ici-late_scanone_snpsqc_5.batches_mis.Rdata
    Untracked:  data/ici-early.vs.pbs.ici-late_scanone_snpsqc_dis_no-x_updated_5.batches.Rdata
    Untracked:  data/ici-early.vs.pbs.ici-late_scanone_snpsqc_dis_no-x_updated_5.batches_mis.Rdata
    Untracked:  data/ici-early.vs.pbs_age.of.onset-additive.covariates_blup_sub_chr-7_peak.marker-UNC13388811_lod.drop-1.5_snpsqc_5.batches.csv
    Untracked:  data/ici-early.vs.pbs_age.of.onset-additive.covariates_genes_chr-7_peak.marker-UNC13388811_lod.drop-1.5_snpsqc_5.batches.csv
    Untracked:  data/ici-early.vs.pbs_age.of.onset-no.covariates_blup_sub_chr-14_peak.marker-UNC24056202_lod.drop-1.5_snpsqc_5.batches_mis.csv
    Untracked:  data/ici-early.vs.pbs_age.of.onset-no.covariates_blup_sub_chr-17_peak.marker-UNCJPD006614_lod.drop-1.5_snpsqc_5.batches_mis.csv
    Untracked:  data/ici-early.vs.pbs_age.of.onset-no.covariates_blup_sub_chr-18_peak.marker-UNCHS045343_lod.drop-1.5_snpsqc_5.batches_mis.csv
    Untracked:  data/ici-early.vs.pbs_age.of.onset-no.covariates_blup_sub_chr-8_peak.marker-UNC15524531_lod.drop-1.5_snpsqc_5.batches_mis.csv
    Untracked:  data/ici-early.vs.pbs_age.of.onset-no.covariates_blup_sub_chr-X_peak.marker-UNCHS048314_lod.drop-1.5_snpsqc_5.batches_mis.csv
    Untracked:  data/ici-early.vs.pbs_age.of.onset-no.covariates_genes_chr-14_peak.marker-UNC24056202_lod.drop-1.5_snpsqc_5.batches_mis.csv
    Untracked:  data/ici-early.vs.pbs_age.of.onset-no.covariates_genes_chr-17_peak.marker-UNCJPD006614_lod.drop-1.5_snpsqc_5.batches_mis.csv
    Untracked:  data/ici-early.vs.pbs_age.of.onset-no.covariates_genes_chr-18_peak.marker-UNCHS045343_lod.drop-1.5_snpsqc_5.batches_mis.csv
    Untracked:  data/ici-early.vs.pbs_age.of.onset-no.covariates_genes_chr-8_peak.marker-UNC15524531_lod.drop-1.5_snpsqc_5.batches_mis.csv
    Untracked:  data/ici-early.vs.pbs_age.of.onset-no.covariates_genes_chr-X_peak.marker-UNCHS048314_lod.drop-1.5_snpsqc_5.batches_mis.csv
    Untracked:  data/ici-early.vs.pbs_blup.full_chr-10_5.batches.csv
    Untracked:  data/ici-early.vs.pbs_blup.full_chr-10_5.batches_mis.csv
    Untracked:  data/ici-early.vs.pbs_blup.full_chr-10_snpsqc_5.batches.csv
    Untracked:  data/ici-early.vs.pbs_blup.full_chr-10_snpsqc_5.batches_mis.csv
    Untracked:  data/ici-early.vs.pbs_blup.full_chr-10_snpsqc_dis_no-x_updated_5.batches.csv
    Untracked:  data/ici-early.vs.pbs_blup.full_chr-10_snpsqc_dis_no-x_updated_5.batches_mis.csv
    Untracked:  data/ici-early.vs.pbs_blup.full_chr-11_5.batches.csv
    Untracked:  data/ici-early.vs.pbs_blup.full_chr-11_5.batches_mis.csv
    Untracked:  data/ici-early.vs.pbs_blup.full_chr-11_snpsqc_5.batches.csv
    Untracked:  data/ici-early.vs.pbs_blup.full_chr-11_snpsqc_5.batches_mis.csv
    Untracked:  data/ici-early.vs.pbs_blup.full_chr-11_snpsqc_dis_no-x_updated_5.batches.csv
    Untracked:  data/ici-early.vs.pbs_blup.full_chr-11_snpsqc_dis_no-x_updated_5.batches_mis.csv
    Untracked:  data/ici-early.vs.pbs_blup.full_chr-12_5.batches.csv
    Untracked:  data/ici-early.vs.pbs_blup.full_chr-12_5.batches_mis.csv
    Untracked:  data/ici-early.vs.pbs_blup.full_chr-12_snpsqc_5.batches.csv
    Untracked:  data/ici-early.vs.pbs_blup.full_chr-12_snpsqc_5.batches_mis.csv
    Untracked:  data/ici-early.vs.pbs_blup.full_chr-12_snpsqc_dis_no-x_updated_5.batches.csv
    Untracked:  data/ici-early.vs.pbs_blup.full_chr-12_snpsqc_dis_no-x_updated_5.batches_mis.csv
    Untracked:  data/ici-early.vs.pbs_blup.full_chr-13_5.batches.csv
    Untracked:  data/ici-early.vs.pbs_blup.full_chr-13_5.batches_mis.csv
    Untracked:  data/ici-early.vs.pbs_blup.full_chr-13_snpsqc_5.batches.csv
    Untracked:  data/ici-early.vs.pbs_blup.full_chr-13_snpsqc_5.batches_mis.csv
    Untracked:  data/ici-early.vs.pbs_blup.full_chr-13_snpsqc_dis_no-x_updated_5.batches.csv
    Untracked:  data/ici-early.vs.pbs_blup.full_chr-13_snpsqc_dis_no-x_updated_5.batches_mis.csv
    Untracked:  data/ici-early.vs.pbs_blup.full_chr-14_5.batches.csv
    Untracked:  data/ici-early.vs.pbs_blup.full_chr-14_5.batches_mis.csv
    Untracked:  data/ici-early.vs.pbs_blup.full_chr-14_snpsqc_5.batches.csv
    Untracked:  data/ici-early.vs.pbs_blup.full_chr-14_snpsqc_5.batches_mis.csv
    Untracked:  data/ici-early.vs.pbs_blup.full_chr-14_snpsqc_dis_no-x_updated_5.batches.csv
    Untracked:  data/ici-early.vs.pbs_blup.full_chr-14_snpsqc_dis_no-x_updated_5.batches_mis.csv
    Untracked:  data/ici-early.vs.pbs_blup.full_chr-15_5.batches.csv
    Untracked:  data/ici-early.vs.pbs_blup.full_chr-15_5.batches_mis.csv
    Untracked:  data/ici-early.vs.pbs_blup.full_chr-15_snpsqc_5.batches.csv
    Untracked:  data/ici-early.vs.pbs_blup.full_chr-15_snpsqc_5.batches_mis.csv
    Untracked:  data/ici-early.vs.pbs_blup.full_chr-15_snpsqc_dis_no-x_updated_5.batches.csv
    Untracked:  data/ici-early.vs.pbs_blup.full_chr-15_snpsqc_dis_no-x_updated_5.batches_mis.csv
    Untracked:  data/ici-early.vs.pbs_blup.full_chr-16_5.batches.csv
    Untracked:  data/ici-early.vs.pbs_blup.full_chr-16_5.batches_mis.csv
    Untracked:  data/ici-early.vs.pbs_blup.full_chr-16_snpsqc_5.batches.csv
    Untracked:  data/ici-early.vs.pbs_blup.full_chr-16_snpsqc_5.batches_mis.csv
    Untracked:  data/ici-early.vs.pbs_blup.full_chr-16_snpsqc_dis_no-x_updated_5.batches.csv
    Untracked:  data/ici-early.vs.pbs_blup.full_chr-16_snpsqc_dis_no-x_updated_5.batches_mis.csv
    Untracked:  data/ici-early.vs.pbs_blup.full_chr-17_5.batches.csv
    Untracked:  data/ici-early.vs.pbs_blup.full_chr-17_5.batches_mis.csv
    Untracked:  data/ici-early.vs.pbs_blup.full_chr-17_snpsqc_5.batches.csv
    Untracked:  data/ici-early.vs.pbs_blup.full_chr-17_snpsqc_5.batches_mis.csv
    Untracked:  data/ici-early.vs.pbs_blup.full_chr-17_snpsqc_dis_no-x_updated_5.batches.csv
    Untracked:  data/ici-early.vs.pbs_blup.full_chr-17_snpsqc_dis_no-x_updated_5.batches_mis.csv
    Untracked:  data/ici-early.vs.pbs_blup.full_chr-18_5.batches.csv
    Untracked:  data/ici-early.vs.pbs_blup.full_chr-18_5.batches_mis.csv
    Untracked:  data/ici-early.vs.pbs_blup.full_chr-18_snpsqc_5.batches.csv
    Untracked:  data/ici-early.vs.pbs_blup.full_chr-18_snpsqc_5.batches_mis.csv
    Untracked:  data/ici-early.vs.pbs_blup.full_chr-18_snpsqc_dis_no-x_updated_5.batches.csv
    Untracked:  data/ici-early.vs.pbs_blup.full_chr-18_snpsqc_dis_no-x_updated_5.batches_mis.csv
    Untracked:  data/ici-early.vs.pbs_blup.full_chr-19_5.batches.csv
    Untracked:  data/ici-early.vs.pbs_blup.full_chr-19_5.batches_mis.csv
    Untracked:  data/ici-early.vs.pbs_blup.full_chr-19_snpsqc_5.batches.csv
    Untracked:  data/ici-early.vs.pbs_blup.full_chr-19_snpsqc_5.batches_mis.csv
    Untracked:  data/ici-early.vs.pbs_blup.full_chr-19_snpsqc_dis_no-x_updated_5.batches.csv
    Untracked:  data/ici-early.vs.pbs_blup.full_chr-19_snpsqc_dis_no-x_updated_5.batches_mis.csv
    Untracked:  data/ici-early.vs.pbs_blup.full_chr-1_5.batches.csv
    Untracked:  data/ici-early.vs.pbs_blup.full_chr-1_5.batches_mis.csv
    Untracked:  data/ici-early.vs.pbs_blup.full_chr-1_snpsqc_5.batches.csv
    Untracked:  data/ici-early.vs.pbs_blup.full_chr-1_snpsqc_5.batches_mis.csv
    Untracked:  data/ici-early.vs.pbs_blup.full_chr-1_snpsqc_dis_no-x_updated_5.batches.csv
    Untracked:  data/ici-early.vs.pbs_blup.full_chr-1_snpsqc_dis_no-x_updated_5.batches_mis.csv
    Untracked:  data/ici-early.vs.pbs_blup.full_chr-2_5.batches.csv
    Untracked:  data/ici-early.vs.pbs_blup.full_chr-2_5.batches_mis.csv
    Untracked:  data/ici-early.vs.pbs_blup.full_chr-2_snpsqc_5.batches.csv
    Untracked:  data/ici-early.vs.pbs_blup.full_chr-2_snpsqc_5.batches_mis.csv
    Untracked:  data/ici-early.vs.pbs_blup.full_chr-2_snpsqc_dis_no-x_updated_5.batches.csv
    Untracked:  data/ici-early.vs.pbs_blup.full_chr-2_snpsqc_dis_no-x_updated_5.batches_mis.csv
    Untracked:  data/ici-early.vs.pbs_blup.full_chr-3_5.batches.csv
    Untracked:  data/ici-early.vs.pbs_blup.full_chr-3_5.batches_mis.csv
    Untracked:  data/ici-early.vs.pbs_blup.full_chr-3_snpsqc_5.batches.csv
    Untracked:  data/ici-early.vs.pbs_blup.full_chr-3_snpsqc_5.batches_mis.csv
    Untracked:  data/ici-early.vs.pbs_blup.full_chr-3_snpsqc_dis_no-x_updated_5.batches.csv
    Untracked:  data/ici-early.vs.pbs_blup.full_chr-3_snpsqc_dis_no-x_updated_5.batches_mis.csv
    Untracked:  data/ici-early.vs.pbs_blup.full_chr-4_5.batches.csv
    Untracked:  data/ici-early.vs.pbs_blup.full_chr-4_5.batches_mis.csv
    Untracked:  data/ici-early.vs.pbs_blup.full_chr-4_snpsqc_5.batches.csv
    Untracked:  data/ici-early.vs.pbs_blup.full_chr-4_snpsqc_5.batches_mis.csv
    Untracked:  data/ici-early.vs.pbs_blup.full_chr-4_snpsqc_dis_no-x_updated_5.batches.csv
    Untracked:  data/ici-early.vs.pbs_blup.full_chr-4_snpsqc_dis_no-x_updated_5.batches_mis.csv
    Untracked:  data/ici-early.vs.pbs_blup.full_chr-5_5.batches.csv
    Untracked:  data/ici-early.vs.pbs_blup.full_chr-5_5.batches_mis.csv
    Untracked:  data/ici-early.vs.pbs_blup.full_chr-5_snpsqc_5.batches.csv
    Untracked:  data/ici-early.vs.pbs_blup.full_chr-5_snpsqc_5.batches_mis.csv
    Untracked:  data/ici-early.vs.pbs_blup.full_chr-5_snpsqc_dis_no-x_updated_5.batches.csv
    Untracked:  data/ici-early.vs.pbs_blup.full_chr-5_snpsqc_dis_no-x_updated_5.batches_mis.csv
    Untracked:  data/ici-early.vs.pbs_blup.full_chr-6_5.batches.csv
    Untracked:  data/ici-early.vs.pbs_blup.full_chr-6_5.batches_mis.csv
    Untracked:  data/ici-early.vs.pbs_blup.full_chr-6_snpsqc_5.batches.csv
    Untracked:  data/ici-early.vs.pbs_blup.full_chr-6_snpsqc_5.batches_mis.csv
    Untracked:  data/ici-early.vs.pbs_blup.full_chr-6_snpsqc_dis_no-x_updated_5.batches.csv
    Untracked:  data/ici-early.vs.pbs_blup.full_chr-6_snpsqc_dis_no-x_updated_5.batches_mis.csv
    Untracked:  data/ici-early.vs.pbs_blup.full_chr-7_5.batches.csv
    Untracked:  data/ici-early.vs.pbs_blup.full_chr-7_5.batches_mis.csv
    Untracked:  data/ici-early.vs.pbs_blup.full_chr-7_snpsqc_5.batches.csv
    Untracked:  data/ici-early.vs.pbs_blup.full_chr-7_snpsqc_5.batches_mis.csv
    Untracked:  data/ici-early.vs.pbs_blup.full_chr-7_snpsqc_dis_no-x_updated_5.batches.csv
    Untracked:  data/ici-early.vs.pbs_blup.full_chr-7_snpsqc_dis_no-x_updated_5.batches_mis.csv
    Untracked:  data/ici-early.vs.pbs_blup.full_chr-8_5.batches.csv
    Untracked:  data/ici-early.vs.pbs_blup.full_chr-8_5.batches_mis.csv
    Untracked:  data/ici-early.vs.pbs_blup.full_chr-8_snpsqc_5.batches.csv
    Untracked:  data/ici-early.vs.pbs_blup.full_chr-8_snpsqc_5.batches_mis.csv
    Untracked:  data/ici-early.vs.pbs_blup.full_chr-8_snpsqc_dis_no-x_updated_5.batches.csv
    Untracked:  data/ici-early.vs.pbs_blup.full_chr-8_snpsqc_dis_no-x_updated_5.batches_mis.csv
    Untracked:  data/ici-early.vs.pbs_blup.full_chr-9_5.batches.csv
    Untracked:  data/ici-early.vs.pbs_blup.full_chr-9_5.batches_mis.csv
    Untracked:  data/ici-early.vs.pbs_blup.full_chr-9_snpsqc_5.batches.csv
    Untracked:  data/ici-early.vs.pbs_blup.full_chr-9_snpsqc_5.batches_mis.csv
    Untracked:  data/ici-early.vs.pbs_blup.full_chr-9_snpsqc_dis_no-x_updated_5.batches.csv
    Untracked:  data/ici-early.vs.pbs_blup.full_chr-9_snpsqc_dis_no-x_updated_5.batches_mis.csv
    Untracked:  data/ici-early.vs.pbs_blup.full_chr-X_5.batches.csv
    Untracked:  data/ici-early.vs.pbs_blup.full_chr-X_5.batches_mis.csv
    Untracked:  data/ici-early.vs.pbs_blup.full_chr-X_snpsqc_5.batches.csv
    Untracked:  data/ici-early.vs.pbs_blup.full_chr-X_snpsqc_5.batches_mis.csv
    Untracked:  data/ici-early.vs.pbs_blup.full_chr-X_snpsqc_dis_no-x_updated_5.batches.csv
    Untracked:  data/ici-early.vs.pbs_blup.full_chr-X_snpsqc_dis_no-x_updated_5.batches_mis.csv
    Untracked:  data/ici-early.vs.pbs_blup.qc_chr-10_5.batches.csv
    Untracked:  data/ici-early.vs.pbs_blup.qc_chr-10_5.batches_mis.csv
    Untracked:  data/ici-early.vs.pbs_blup.qc_chr-10_snpsqc_5.batches.csv
    Untracked:  data/ici-early.vs.pbs_blup.qc_chr-10_snpsqc_5.batches_mis.csv
    Untracked:  data/ici-early.vs.pbs_blup.qc_chr-10_snpsqc_dis_no-x_updated_5.batches.csv
    Untracked:  data/ici-early.vs.pbs_blup.qc_chr-10_snpsqc_dis_no-x_updated_5.batches_mis.csv
    Untracked:  data/ici-early.vs.pbs_blup.qc_chr-11_5.batches.csv
    Untracked:  data/ici-early.vs.pbs_blup.qc_chr-11_5.batches_mis.csv
    Untracked:  data/ici-early.vs.pbs_blup.qc_chr-11_snpsqc_5.batches.csv
    Untracked:  data/ici-early.vs.pbs_blup.qc_chr-11_snpsqc_5.batches_mis.csv
    Untracked:  data/ici-early.vs.pbs_blup.qc_chr-11_snpsqc_dis_no-x_updated_5.batches.csv
    Untracked:  data/ici-early.vs.pbs_blup.qc_chr-11_snpsqc_dis_no-x_updated_5.batches_mis.csv
    Untracked:  data/ici-early.vs.pbs_blup.qc_chr-12_5.batches.csv
    Untracked:  data/ici-early.vs.pbs_blup.qc_chr-12_5.batches_mis.csv
    Untracked:  data/ici-early.vs.pbs_blup.qc_chr-12_snpsqc_5.batches.csv
    Untracked:  data/ici-early.vs.pbs_blup.qc_chr-12_snpsqc_5.batches_mis.csv
    Untracked:  data/ici-early.vs.pbs_blup.qc_chr-12_snpsqc_dis_no-x_updated_5.batches.csv
    Untracked:  data/ici-early.vs.pbs_blup.qc_chr-12_snpsqc_dis_no-x_updated_5.batches_mis.csv
    Untracked:  data/ici-early.vs.pbs_blup.qc_chr-13_5.batches.csv
    Untracked:  data/ici-early.vs.pbs_blup.qc_chr-13_5.batches_mis.csv
    Untracked:  data/ici-early.vs.pbs_blup.qc_chr-13_snpsqc_5.batches.csv
    Untracked:  data/ici-early.vs.pbs_blup.qc_chr-13_snpsqc_5.batches_mis.csv
    Untracked:  data/ici-early.vs.pbs_blup.qc_chr-13_snpsqc_dis_no-x_updated_5.batches.csv
    Untracked:  data/ici-early.vs.pbs_blup.qc_chr-13_snpsqc_dis_no-x_updated_5.batches_mis.csv
    Untracked:  data/ici-early.vs.pbs_blup.qc_chr-14_5.batches.csv
    Untracked:  data/ici-early.vs.pbs_blup.qc_chr-14_5.batches_mis.csv
    Untracked:  data/ici-early.vs.pbs_blup.qc_chr-14_snpsqc_5.batches.csv
    Untracked:  data/ici-early.vs.pbs_blup.qc_chr-14_snpsqc_5.batches_mis.csv
    Untracked:  data/ici-early.vs.pbs_blup.qc_chr-14_snpsqc_dis_no-x_updated_5.batches.csv
    Untracked:  data/ici-early.vs.pbs_blup.qc_chr-14_snpsqc_dis_no-x_updated_5.batches_mis.csv
    Untracked:  data/ici-early.vs.pbs_blup.qc_chr-15_5.batches.csv
    Untracked:  data/ici-early.vs.pbs_blup.qc_chr-15_5.batches_mis.csv
    Untracked:  data/ici-early.vs.pbs_blup.qc_chr-15_snpsqc_5.batches.csv
    Untracked:  data/ici-early.vs.pbs_blup.qc_chr-15_snpsqc_5.batches_mis.csv
    Untracked:  data/ici-early.vs.pbs_blup.qc_chr-16_5.batches.csv
    Untracked:  data/ici-early.vs.pbs_blup.qc_chr-16_5.batches_mis.csv
    Untracked:  data/ici-early.vs.pbs_blup.qc_chr-16_snpsqc_5.batches.csv
    Untracked:  data/ici-early.vs.pbs_blup.qc_chr-16_snpsqc_5.batches_mis.csv
    Untracked:  data/ici-early.vs.pbs_blup.qc_chr-16_snpsqc_dis_no-x_updated_5.batches.csv
    Untracked:  data/ici-early.vs.pbs_blup.qc_chr-16_snpsqc_dis_no-x_updated_5.batches_mis.csv
    Untracked:  data/ici-early.vs.pbs_blup.qc_chr-17_5.batches.csv
    Untracked:  data/ici-early.vs.pbs_blup.qc_chr-17_5.batches_mis.csv
    Untracked:  data/ici-early.vs.pbs_blup.qc_chr-17_snpsqc_5.batches.csv
    Untracked:  data/ici-early.vs.pbs_blup.qc_chr-17_snpsqc_5.batches_mis.csv
    Untracked:  data/ici-early.vs.pbs_blup.qc_chr-18_5.batches.csv
    Untracked:  data/ici-early.vs.pbs_blup.qc_chr-18_5.batches_mis.csv
    Untracked:  data/ici-early.vs.pbs_blup.qc_chr-18_snpsqc_5.batches.csv
    Untracked:  data/ici-early.vs.pbs_blup.qc_chr-18_snpsqc_5.batches_mis.csv
    Untracked:  data/ici-early.vs.pbs_blup.qc_chr-18_snpsqc_dis_no-x_updated_5.batches.csv
    Untracked:  data/ici-early.vs.pbs_blup.qc_chr-18_snpsqc_dis_no-x_updated_5.batches_mis.csv
    Untracked:  data/ici-early.vs.pbs_blup.qc_chr-19_5.batches.csv
    Untracked:  data/ici-early.vs.pbs_blup.qc_chr-19_5.batches_mis.csv
    Untracked:  data/ici-early.vs.pbs_blup.qc_chr-19_snpsqc_5.batches.csv
    Untracked:  data/ici-early.vs.pbs_blup.qc_chr-19_snpsqc_5.batches_mis.csv
    Untracked:  data/ici-early.vs.pbs_blup.qc_chr-19_snpsqc_dis_no-x_updated_5.batches.csv
    Untracked:  data/ici-early.vs.pbs_blup.qc_chr-19_snpsqc_dis_no-x_updated_5.batches_mis.csv
    Untracked:  data/ici-early.vs.pbs_blup.qc_chr-1_5.batches.csv
    Untracked:  data/ici-early.vs.pbs_blup.qc_chr-1_5.batches_mis.csv
    Untracked:  data/ici-early.vs.pbs_blup.qc_chr-1_snpsqc_5.batches.csv
    Untracked:  data/ici-early.vs.pbs_blup.qc_chr-1_snpsqc_5.batches_mis.csv
    Untracked:  data/ici-early.vs.pbs_blup.qc_chr-1_snpsqc_dis_no-x_updated_5.batches.csv
    Untracked:  data/ici-early.vs.pbs_blup.qc_chr-1_snpsqc_dis_no-x_updated_5.batches_mis.csv
    Untracked:  data/ici-early.vs.pbs_blup.qc_chr-2_5.batches.csv
    Untracked:  data/ici-early.vs.pbs_blup.qc_chr-2_5.batches_mis.csv
    Untracked:  data/ici-early.vs.pbs_blup.qc_chr-2_snpsqc_5.batches.csv
    Untracked:  data/ici-early.vs.pbs_blup.qc_chr-2_snpsqc_5.batches_mis.csv
    Untracked:  data/ici-early.vs.pbs_blup.qc_chr-2_snpsqc_dis_no-x_updated_5.batches.csv
    Untracked:  data/ici-early.vs.pbs_blup.qc_chr-2_snpsqc_dis_no-x_updated_5.batches_mis.csv
    Untracked:  data/ici-early.vs.pbs_blup.qc_chr-3_5.batches.csv
    Untracked:  data/ici-early.vs.pbs_blup.qc_chr-3_5.batches_mis.csv
    Untracked:  data/ici-early.vs.pbs_blup.qc_chr-3_snpsqc_5.batches.csv
    Untracked:  data/ici-early.vs.pbs_blup.qc_chr-3_snpsqc_5.batches_mis.csv
    Untracked:  data/ici-early.vs.pbs_blup.qc_chr-3_snpsqc_dis_no-x_updated_5.batches.csv
    Untracked:  data/ici-early.vs.pbs_blup.qc_chr-3_snpsqc_dis_no-x_updated_5.batches_mis.csv
    Untracked:  data/ici-early.vs.pbs_blup.qc_chr-4_5.batches.csv
    Untracked:  data/ici-early.vs.pbs_blup.qc_chr-4_5.batches_mis.csv
    Untracked:  data/ici-early.vs.pbs_blup.qc_chr-4_snpsqc_5.batches.csv
    Untracked:  data/ici-early.vs.pbs_blup.qc_chr-4_snpsqc_5.batches_mis.csv
    Untracked:  data/ici-early.vs.pbs_blup.qc_chr-4_snpsqc_dis_no-x_updated_5.batches.csv
    Untracked:  data/ici-early.vs.pbs_blup.qc_chr-4_snpsqc_dis_no-x_updated_5.batches_mis.csv
    Untracked:  data/ici-early.vs.pbs_blup.qc_chr-5_5.batches.csv
    Untracked:  data/ici-early.vs.pbs_blup.qc_chr-5_5.batches_mis.csv
    Untracked:  data/ici-early.vs.pbs_blup.qc_chr-5_snpsqc_5.batches.csv
    Untracked:  data/ici-early.vs.pbs_blup.qc_chr-5_snpsqc_5.batches_mis.csv
    Untracked:  data/ici-early.vs.pbs_blup.qc_chr-5_snpsqc_dis_no-x_updated_5.batches.csv
    Untracked:  data/ici-early.vs.pbs_blup.qc_chr-5_snpsqc_dis_no-x_updated_5.batches_mis.csv
    Untracked:  data/ici-early.vs.pbs_blup.qc_chr-6_5.batches.csv
    Untracked:  data/ici-early.vs.pbs_blup.qc_chr-6_5.batches_mis.csv
    Untracked:  data/ici-early.vs.pbs_blup.qc_chr-6_snpsqc_5.batches.csv
    Untracked:  data/ici-early.vs.pbs_blup.qc_chr-6_snpsqc_5.batches_mis.csv
    Untracked:  data/ici-early.vs.pbs_blup.qc_chr-6_snpsqc_dis_no-x_updated_5.batches.csv
    Untracked:  data/ici-early.vs.pbs_blup.qc_chr-6_snpsqc_dis_no-x_updated_5.batches_mis.csv
    Untracked:  data/ici-early.vs.pbs_blup.qc_chr-7_5.batches.csv
    Untracked:  data/ici-early.vs.pbs_blup.qc_chr-7_5.batches_mis.csv
    Untracked:  data/ici-early.vs.pbs_blup.qc_chr-7_snpsqc_5.batches.csv
    Untracked:  data/ici-early.vs.pbs_blup.qc_chr-7_snpsqc_5.batches_mis.csv
    Untracked:  data/ici-early.vs.pbs_blup.qc_chr-7_snpsqc_dis_no-x_updated_5.batches.csv
    Untracked:  data/ici-early.vs.pbs_blup.qc_chr-7_snpsqc_dis_no-x_updated_5.batches_mis.csv
    Untracked:  data/ici-early.vs.pbs_blup.qc_chr-8_5.batches.csv
    Untracked:  data/ici-early.vs.pbs_blup.qc_chr-8_5.batches_mis.csv
    Untracked:  data/ici-early.vs.pbs_blup.qc_chr-8_snpsqc_5.batches.csv
    Untracked:  data/ici-early.vs.pbs_blup.qc_chr-8_snpsqc_5.batches_mis.csv
    Untracked:  data/ici-early.vs.pbs_blup.qc_chr-8_snpsqc_dis_no-x_updated_5.batches.csv
    Untracked:  data/ici-early.vs.pbs_blup.qc_chr-8_snpsqc_dis_no-x_updated_5.batches_mis.csv
    Untracked:  data/ici-early.vs.pbs_blup.qc_chr-9_5.batches.csv
    Untracked:  data/ici-early.vs.pbs_blup.qc_chr-9_5.batches_mis.csv
    Untracked:  data/ici-early.vs.pbs_blup.qc_chr-9_snpsqc_5.batches.csv
    Untracked:  data/ici-early.vs.pbs_blup.qc_chr-9_snpsqc_5.batches_mis.csv
    Untracked:  data/ici-early.vs.pbs_blup.qc_chr-9_snpsqc_dis_no-x_updated_5.batches.csv
    Untracked:  data/ici-early.vs.pbs_blup.qc_chr-9_snpsqc_dis_no-x_updated_5.batches_mis.csv
    Untracked:  data/ici-early.vs.pbs_blup.qc_chr-X_5.batches.csv
    Untracked:  data/ici-early.vs.pbs_blup.qc_chr-X_5.batches_mis.csv
    Untracked:  data/ici-early.vs.pbs_blup.qc_chr-X_snpsqc_5.batches.csv
    Untracked:  data/ici-early.vs.pbs_blup.qc_chr-X_snpsqc_5.batches_mis.csv
    Untracked:  data/ici-early.vs.pbs_blup.qc_chr-X_snpsqc_dis_no-x_updated_5.batches.csv
    Untracked:  data/ici-early.vs.pbs_blup.qc_chr-X_snpsqc_dis_no-x_updated_5.batches_mis.csv
    Untracked:  data/ici-early.vs.pbs_blup_sub_chr-18_peak.marker-UNCHS045343_lod.drop-1.5_snpsqc_5.batches_mis.csv
    Untracked:  data/ici-early.vs.pbs_genes_chr-18_peak.marker-UNCHS045343_lod.drop-1.5_snpsqc_5.batches_mis.csv
    Untracked:  data/ici-early.vs.pbs_gm_qtl_5.batches.csv
    Untracked:  data/ici-early.vs.pbs_gm_qtl_5.batches_mis.csv
    Untracked:  data/ici-early.vs.pbs_gm_qtl_snpsqc_5.batches.csv
    Untracked:  data/ici-early.vs.pbs_gm_qtl_snpsqc_5.batches_mis.csv
    Untracked:  data/ici-early.vs.pbs_gm_qtl_snpsqc_dis_no-x_updated_5.batches.csv
    Untracked:  data/ici-early.vs.pbs_gm_qtl_snpsqc_dis_no-x_updated_5.batches_mis.csv
    Untracked:  data/ici-early.vs.pbs_marker.freq_low.geno.freq.removed_geno.ratio_5.batches.csv
    Untracked:  data/ici-early.vs.pbs_marker.freq_low.geno.freq.removed_geno.ratio_5.batches_mis.csv
    Untracked:  data/ici-early.vs.pbs_marker.freq_low.geno.freq.removed_sample.outliers.removed_geno.ratio_5.batches.csv
    Untracked:  data/ici-early.vs.pbs_marker.freq_low.geno.freq.removed_sample.outliers.removed_geno.ratio_5.batches_mis.csv
    Untracked:  data/ici-early.vs.pbs_marker.freq_low.probs.freq.removed_geno.ratio_5.batches.csv
    Untracked:  data/ici-early.vs.pbs_marker.freq_low.probs.freq.removed_geno.ratio_5.batches_mis.csv
    Untracked:  data/ici-early.vs.pbs_marker.freq_low.probs.freq.removed_sample.outliers.removed_geno.ratio_5.batches.csv
    Untracked:  data/ici-early.vs.pbs_marker.freq_low.probs.freq.removed_sample.outliers.removed_geno.ratio_5.batches_mis.csv
    Untracked:  data/ici-early.vs.pbs_rz.age-additive.covariates_blup_sub_chr-7_peak.marker-UNC13239362_lod.drop-1.5_snpsqc_5.batches.csv
    Untracked:  data/ici-early.vs.pbs_rz.age-additive.covariates_blup_sub_chr-7_peak.marker-UNC13239362_lod.drop-1.5_snpsqc_5.batches_mis.csv
    Untracked:  data/ici-early.vs.pbs_rz.age-additive.covariates_blup_sub_chr-7_peak.marker-UNC13239362_lod.drop-1.5_snpsqc_dis_no-x_updated_5.batches.csv
    Untracked:  data/ici-early.vs.pbs_rz.age-additive.covariates_blup_sub_chr-7_peak.marker-UNC13239362_lod.drop-1.5_snpsqc_dis_no-x_updated_5.batches_mis.csv
    Untracked:  data/ici-early.vs.pbs_rz.age-additive.covariates_genes_chr-7_peak.marker-UNC13239362_lod.drop-1.5_snpsqc_5.batches.csv
    Untracked:  data/ici-early.vs.pbs_rz.age-additive.covariates_genes_chr-7_peak.marker-UNC13239362_lod.drop-1.5_snpsqc_5.batches_mis.csv
    Untracked:  data/ici-early.vs.pbs_rz.age-additive.covariates_genes_chr-7_peak.marker-UNC13239362_lod.drop-1.5_snpsqc_dis_no-x_updated_5.batches.csv
    Untracked:  data/ici-early.vs.pbs_rz.age-additive.covariates_genes_chr-7_peak.marker-UNC13239362_lod.drop-1.5_snpsqc_dis_no-x_updated_5.batches_mis.csv
    Untracked:  data/ici-early.vs.pbs_scanone_5.batches.Rdata
    Untracked:  data/ici-early.vs.pbs_scanone_5.batches_mis.Rdata
    Untracked:  data/ici-early.vs.pbs_scanone_snpsqc_5.batches.Rdata
    Untracked:  data/ici-early.vs.pbs_scanone_snpsqc_5.batches_mis.Rdata
    Untracked:  data/ici-early.vs.pbs_scanone_snpsqc_dis_no-x_updated_5.batches.Rdata
    Untracked:  data/ici-early.vs.pbs_scanone_snpsqc_dis_no-x_updated_5.batches_mis.Rdata
    Untracked:  data/ici.vs.eoi_age.of.onset-no.covariates_blup_sub_chr-10_peak.marker-JAX00020646_lod.drop-1.5_5.batches_0.csv
    Untracked:  data/ici.vs.eoi_age.of.onset-no.covariates_blup_sub_chr-10_peak.marker-JAX00292499_lod.drop-1.5_5.batches_52.csv
    Untracked:  data/ici.vs.eoi_age.of.onset-no.covariates_blup_sub_chr-10_peak.marker-JAX00292927_lod.drop-1.5_5.batches_0.csv
    Untracked:  data/ici.vs.eoi_age.of.onset-no.covariates_blup_sub_chr-10_peak.marker-JAX00292927_lod.drop-1.5_5.batches_52.csv
    Untracked:  data/ici.vs.eoi_age.of.onset-no.covariates_blup_sub_chr-10_peak.marker-JAX00294019_lod.drop-1.5_snpsqc_dis_no-x_updated_5.batches_0.csv
    Untracked:  data/ici.vs.eoi_age.of.onset-no.covariates_blup_sub_chr-10_peak.marker-UNC18216614_lod.drop-1.5_5.batches_52.csv
    Untracked:  data/ici.vs.eoi_age.of.onset-no.covariates_blup_sub_chr-10_peak.marker-UNC18240977_lod.drop-1.5_snpsqc_dis_no-x_updated_5.batches_52.csv
    Untracked:  data/ici.vs.eoi_age.of.onset-no.covariates_blup_sub_chr-10_peak.marker-UNC18311938_lod.drop-1.5_5.batches_0.csv
    Untracked:  data/ici.vs.eoi_age.of.onset-no.covariates_blup_sub_chr-10_peak.marker-UNC18311938_lod.drop-1.5_5.batches_52.csv
    Untracked:  data/ici.vs.eoi_age.of.onset-no.covariates_blup_sub_chr-10_peak.marker-UNC18311938_lod.drop-1.5_snpsqc_5.batches_52.csv
    Untracked:  data/ici.vs.eoi_age.of.onset-no.covariates_blup_sub_chr-10_peak.marker-UNC18311938_lod.drop-1.5_snpsqc_5.batches_mis_52.csv
    Untracked:  data/ici.vs.eoi_age.of.onset-no.covariates_blup_sub_chr-10_peak.marker-UNC18343181_lod.drop-1.5_5.batches_0.csv
    Untracked:  data/ici.vs.eoi_age.of.onset-no.covariates_blup_sub_chr-10_peak.marker-UNC18363544_lod.drop-1.5_5.batches_0.csv
    Untracked:  data/ici.vs.eoi_age.of.onset-no.covariates_blup_sub_chr-10_peak.marker-UNC18363544_lod.drop-1.5_5.batches_52.csv
    Untracked:  data/ici.vs.eoi_age.of.onset-no.covariates_blup_sub_chr-10_peak.marker-UNC18363544_lod.drop-1.5_snpsqc_5.batches_mis_0.csv
    Untracked:  data/ici.vs.eoi_age.of.onset-no.covariates_blup_sub_chr-10_peak.marker-UNC18376338_lod.drop-1.5_snpsqc_5.batches_0.csv
    Untracked:  data/ici.vs.eoi_age.of.onset-no.covariates_blup_sub_chr-10_peak.marker-UNCHS028236_lod.drop-1.5_snpsqc_5.batches_mis_0.csv
    Untracked:  data/ici.vs.eoi_age.of.onset-no.covariates_blup_sub_chr-10_peak.marker-UNCHS028236_lod.drop-1.5_snpsqc_5.batches_mis_52.csv
    Untracked:  data/ici.vs.eoi_age.of.onset-no.covariates_blup_sub_chr-10_peak.marker-UNCHS028536_lod.drop-1.5_snpsqc_dis_no-x_updated_5.batches_mis_0.csv
    Untracked:  data/ici.vs.eoi_age.of.onset-no.covariates_blup_sub_chr-10_peak.marker-UNCHS028536_lod.drop-1.5_snpsqc_dis_no-x_updated_5.batches_mis_52.csv
    Untracked:  data/ici.vs.eoi_age.of.onset-no.covariates_blup_sub_chr-11_peak.marker-UNC19970181_lod.drop-1.5_snpsqc_5.batches_mis_0.csv
    Untracked:  data/ici.vs.eoi_age.of.onset-no.covariates_blup_sub_chr-11_peak.marker-UNC19970181_lod.drop-1.5_snpsqc_5.batches_mis_52.csv
    Untracked:  data/ici.vs.eoi_age.of.onset-no.covariates_blup_sub_chr-11_peak.marker-UNC20090524_lod.drop-1.5_snpsqc_5.batches_52.csv
    Untracked:  data/ici.vs.eoi_age.of.onset-no.covariates_blup_sub_chr-12_peak.marker-ICR499_lod.drop-1.5_snpsqc_5.batches_mis_0.csv
    Untracked:  data/ici.vs.eoi_age.of.onset-no.covariates_blup_sub_chr-12_peak.marker-ICR499_lod.drop-1.5_snpsqc_5.batches_mis_52.csv
    Untracked:  data/ici.vs.eoi_age.of.onset-no.covariates_blup_sub_chr-12_peak.marker-ICR499_lod.drop-1.5_snpsqc_dis_no-x_updated_5.batches_mis_0.csv
    Untracked:  data/ici.vs.eoi_age.of.onset-no.covariates_blup_sub_chr-13_peak.marker-UNCHS036773_lod.drop-1.5_snpsqc_5.batches_mis_0.csv
    Untracked:  data/ici.vs.eoi_age.of.onset-no.covariates_blup_sub_chr-13_peak.marker-UNCHS036773_lod.drop-1.5_snpsqc_5.batches_mis_52.csv
    Untracked:  data/ici.vs.eoi_age.of.onset-no.covariates_blup_sub_chr-14_peak.marker-UNC24056202_lod.drop-1.5_snpsqc_5.batches_mis_0.csv
    Untracked:  data/ici.vs.eoi_age.of.onset-no.covariates_blup_sub_chr-14_peak.marker-UNC24056202_lod.drop-1.5_snpsqc_5.batches_mis_52.csv
    Untracked:  data/ici.vs.eoi_age.of.onset-no.covariates_blup_sub_chr-15_peak.marker-UNC26070435_lod.drop-1.5_snpsqc_dis_no-x_updated_5.batches_mis_0.csv
    Untracked:  data/ici.vs.eoi_age.of.onset-no.covariates_blup_sub_chr-15_peak.marker-UNC26070435_lod.drop-1.5_snpsqc_dis_no-x_updated_5.batches_mis_52.csv
    Untracked:  data/ici.vs.eoi_age.of.onset-no.covariates_blup_sub_chr-17_peak.marker-UNCHS044241_lod.drop-1.5_snpsqc_5.batches_mis_0.csv
    Untracked:  data/ici.vs.eoi_age.of.onset-no.covariates_blup_sub_chr-17_peak.marker-UNCHS044241_lod.drop-1.5_snpsqc_5.batches_mis_52.csv
    Untracked:  data/ici.vs.eoi_age.of.onset-no.covariates_blup_sub_chr-17_peak.marker-UNCJPD006614_lod.drop-1.5_snpsqc_dis_no-x_updated_5.batches_mis_0.csv
    Untracked:  data/ici.vs.eoi_age.of.onset-no.covariates_blup_sub_chr-17_peak.marker-UNCrs47191360_lod.drop-1.5_snpsqc_dis_no-x_updated_5.batches_mis_52.csv
    Untracked:  data/ici.vs.eoi_age.of.onset-no.covariates_blup_sub_chr-18_peak.marker-UNCHS045343_lod.drop-1.5_snpsqc_5.batches_mis_0.csv
    Untracked:  data/ici.vs.eoi_age.of.onset-no.covariates_blup_sub_chr-18_peak.marker-UNCHS045343_lod.drop-1.5_snpsqc_5.batches_mis_52.csv
    Untracked:  data/ici.vs.eoi_age.of.onset-no.covariates_blup_sub_chr-2_peak.marker-UNC4609527_lod.drop-1.5_snpsqc_dis_no-x_updated_5.batches_mis_0.csv
    Untracked:  data/ici.vs.eoi_age.of.onset-no.covariates_blup_sub_chr-2_peak.marker-UNC4609527_lod.drop-1.5_snpsqc_dis_no-x_updated_5.batches_mis_52.csv
    Untracked:  data/ici.vs.eoi_age.of.onset-no.covariates_blup_sub_chr-2_peak.marker-UNCHS008007_lod.drop-1.5_5.batches_mis.csv
    Untracked:  data/ici.vs.eoi_age.of.onset-no.covariates_blup_sub_chr-2_peak.marker-UNCHS008007_lod.drop-1.5_5.batches_mis_0.csv
    Untracked:  data/ici.vs.eoi_age.of.onset-no.covariates_blup_sub_chr-3_peak.marker-ICR1338_lod.drop-1.5_5.batches_0.csv
    Untracked:  data/ici.vs.eoi_age.of.onset-no.covariates_blup_sub_chr-3_peak.marker-ICR1338_lod.drop-1.5_5.batches_52.csv
    Untracked:  data/ici.vs.eoi_age.of.onset-no.covariates_blup_sub_chr-3_peak.marker-ICR1338_lod.drop-1.5_5.batches_mis_0.csv
    Untracked:  data/ici.vs.eoi_age.of.onset-no.covariates_blup_sub_chr-3_peak.marker-ICR5263_lod.drop-1.5_5.batches_mis.csv
    Untracked:  data/ici.vs.eoi_age.of.onset-no.covariates_blup_sub_chr-3_peak.marker-ICR5263_lod.drop-1.5_5.batches_mis_0.csv
    Untracked:  data/ici.vs.eoi_age.of.onset-no.covariates_blup_sub_chr-3_peak.marker-JAX00105649_lod.drop-1.5_5.batches_mis.csv
    Untracked:  data/ici.vs.eoi_age.of.onset-no.covariates_blup_sub_chr-3_peak.marker-JAX00105649_lod.drop-1.5_5.batches_mis_0.csv
    Untracked:  data/ici.vs.eoi_age.of.onset-no.covariates_blup_sub_chr-3_peak.marker-JAX00106210_lod.drop-1.5_5.batches_mis_0.csv
    Untracked:  data/ici.vs.eoi_age.of.onset-no.covariates_blup_sub_chr-3_peak.marker-JAX00106957_lod.drop-1.5_5.batches_0.csv
    Untracked:  data/ici.vs.eoi_age.of.onset-no.covariates_blup_sub_chr-3_peak.marker-JAX00106957_lod.drop-1.5_5.batches_52.csv
    Untracked:  data/ici.vs.eoi_age.of.onset-no.covariates_blup_sub_chr-3_peak.marker-JAX00106957_lod.drop-1.5_5.batches_mis_0.csv
    Untracked:  data/ici.vs.eoi_age.of.onset-no.covariates_blup_sub_chr-3_peak.marker-JAX00107193_lod.drop-1.5_5.batches_mis_0.csv
    Untracked:  data/ici.vs.eoi_age.of.onset-no.covariates_blup_sub_chr-3_peak.marker-JAX00220631_lod.drop-1.5_5.batches_0.csv
    Untracked:  data/ici.vs.eoi_age.of.onset-no.covariates_blup_sub_chr-3_peak.marker-JAX00220631_lod.drop-1.5_5.batches_mis_0.csv
    Untracked:  data/ici.vs.eoi_age.of.onset-no.covariates_blup_sub_chr-3_peak.marker-JAX00514662_lod.drop-1.5_5.batches_0.csv
    Untracked:  data/ici.vs.eoi_age.of.onset-no.covariates_blup_sub_chr-3_peak.marker-JAX00514662_lod.drop-1.5_5.batches_52.csv
    Untracked:  data/ici.vs.eoi_age.of.onset-no.covariates_blup_sub_chr-3_peak.marker-JAX00516812_lod.drop-1.5_5.batches_mis.csv
    Untracked:  data/ici.vs.eoi_age.of.onset-no.covariates_blup_sub_chr-3_peak.marker-JAX00516812_lod.drop-1.5_5.batches_mis_0.csv
    Untracked:  data/ici.vs.eoi_age.of.onset-no.covariates_blup_sub_chr-3_peak.marker-JAX00517674_lod.drop-1.5_5.batches_mis.csv
    Untracked:  data/ici.vs.eoi_age.of.onset-no.covariates_blup_sub_chr-3_peak.marker-JAX00517674_lod.drop-1.5_5.batches_mis_0.csv
    Untracked:  data/ici.vs.eoi_age.of.onset-no.covariates_blup_sub_chr-3_peak.marker-JAX00518911r_lod.drop-1.5_5.batches_mis.csv
    Untracked:  data/ici.vs.eoi_age.of.onset-no.covariates_blup_sub_chr-3_peak.marker-JAX00518911r_lod.drop-1.5_5.batches_mis_0.csv
    Untracked:  data/ici.vs.eoi_age.of.onset-no.covariates_blup_sub_chr-3_peak.marker-JAX00522922_lod.drop-1.5_5.batches_mis_0.csv
    Untracked:  data/ici.vs.eoi_age.of.onset-no.covariates_blup_sub_chr-3_peak.marker-JAX00524205_lod.drop-1.5_5.batches_mis_0.csv
    Untracked:  data/ici.vs.eoi_age.of.onset-no.covariates_blup_sub_chr-3_peak.marker-JAX00529707r_lod.drop-1.5_5.batches_0.csv
    Untracked:  data/ici.vs.eoi_age.of.onset-no.covariates_blup_sub_chr-3_peak.marker-JAX00529707r_lod.drop-1.5_5.batches_52.csv
    Untracked:  data/ici.vs.eoi_age.of.onset-no.covariates_blup_sub_chr-3_peak.marker-JAX00535764_lod.drop-1.5_5.batches_52.csv
    Untracked:  data/ici.vs.eoi_age.of.onset-no.covariates_blup_sub_chr-3_peak.marker-UNC4809586_lod.drop-1.5_5.batches_mis.csv
    Untracked:  data/ici.vs.eoi_age.of.onset-no.covariates_blup_sub_chr-3_peak.marker-UNC4809586_lod.drop-1.5_5.batches_mis_0.csv
    Untracked:  data/ici.vs.eoi_age.of.onset-no.covariates_blup_sub_chr-3_peak.marker-UNC4813186_lod.drop-1.5_5.batches_0.csv
    Untracked:  data/ici.vs.eoi_age.of.onset-no.covariates_blup_sub_chr-3_peak.marker-UNC4813186_lod.drop-1.5_5.batches_52.csv
    Untracked:  data/ici.vs.eoi_age.of.onset-no.covariates_blup_sub_chr-3_peak.marker-UNC4813186_lod.drop-1.5_5.batches_mis.csv
    Untracked:  data/ici.vs.eoi_age.of.onset-no.covariates_blup_sub_chr-3_peak.marker-UNC4813186_lod.drop-1.5_5.batches_mis_0.csv
    Untracked:  data/ici.vs.eoi_age.of.onset-no.covariates_blup_sub_chr-3_peak.marker-UNC4820379_lod.drop-1.5_5.batches_mis.csv
    Untracked:  data/ici.vs.eoi_age.of.onset-no.covariates_blup_sub_chr-3_peak.marker-UNC4820379_lod.drop-1.5_5.batches_mis_0.csv
    Untracked:  data/ici.vs.eoi_age.of.onset-no.covariates_blup_sub_chr-3_peak.marker-UNC4836550_lod.drop-1.5_5.batches_0.csv
    Untracked:  data/ici.vs.eoi_age.of.onset-no.covariates_blup_sub_chr-3_peak.marker-UNC4836550_lod.drop-1.5_5.batches_52.csv
    Untracked:  data/ici.vs.eoi_age.of.onset-no.covariates_blup_sub_chr-3_peak.marker-UNC4836550_lod.drop-1.5_5.batches_mis_0.csv
    Untracked:  data/ici.vs.eoi_age.of.onset-no.covariates_blup_sub_chr-3_peak.marker-UNC4875653_lod.drop-1.5_5.batches_mis.csv
    Untracked:  data/ici.vs.eoi_age.of.onset-no.covariates_blup_sub_chr-3_peak.marker-UNC4875653_lod.drop-1.5_5.batches_mis_0.csv
    Untracked:  data/ici.vs.eoi_age.of.onset-no.covariates_blup_sub_chr-3_peak.marker-UNC4901083_lod.drop-1.5_5.batches_mis.csv
    Untracked:  data/ici.vs.eoi_age.of.onset-no.covariates_blup_sub_chr-3_peak.marker-UNC4901083_lod.drop-1.5_5.batches_mis_0.csv
    Untracked:  data/ici.vs.eoi_age.of.onset-no.covariates_blup_sub_chr-3_peak.marker-UNC4909519_lod.drop-1.5_5.batches_mis.csv
    Untracked:  data/ici.vs.eoi_age.of.onset-no.covariates_blup_sub_chr-3_peak.marker-UNC4909519_lod.drop-1.5_5.batches_mis_0.csv
    Untracked:  data/ici.vs.eoi_age.of.onset-no.covariates_blup_sub_chr-3_peak.marker-UNC4912039_lod.drop-1.5_5.batches_0.csv
    Untracked:  data/ici.vs.eoi_age.of.onset-no.covariates_blup_sub_chr-3_peak.marker-UNC4912039_lod.drop-1.5_5.batches_mis_0.csv
    Untracked:  data/ici.vs.eoi_age.of.onset-no.covariates_blup_sub_chr-3_peak.marker-UNC4912288_lod.drop-1.5_5.batches_mis.csv
    Untracked:  data/ici.vs.eoi_age.of.onset-no.covariates_blup_sub_chr-3_peak.marker-UNC4912288_lod.drop-1.5_5.batches_mis_0.csv
    Untracked:  data/ici.vs.eoi_age.of.onset-no.covariates_blup_sub_chr-3_peak.marker-UNC4915446_lod.drop-1.5_5.batches_mis.csv
    Untracked:  data/ici.vs.eoi_age.of.onset-no.covariates_blup_sub_chr-3_peak.marker-UNC4915446_lod.drop-1.5_5.batches_mis_0.csv
    Untracked:  data/ici.vs.eoi_age.of.onset-no.covariates_blup_sub_chr-3_peak.marker-UNC4923398_lod.drop-1.5_5.batches_mis.csv
    Untracked:  data/ici.vs.eoi_age.of.onset-no.covariates_blup_sub_chr-3_peak.marker-UNC4923398_lod.drop-1.5_5.batches_mis_0.csv
    Untracked:  data/ici.vs.eoi_age.of.onset-no.covariates_blup_sub_chr-3_peak.marker-UNC4924377_lod.drop-1.5_5.batches_0.csv
    Untracked:  data/ici.vs.eoi_age.of.onset-no.covariates_blup_sub_chr-3_peak.marker-UNC4924377_lod.drop-1.5_5.batches_52.csv
    Untracked:  data/ici.vs.eoi_age.of.onset-no.covariates_blup_sub_chr-3_peak.marker-UNC4924377_lod.drop-1.5_5.batches_mis_0.csv
    Untracked:  data/ici.vs.eoi_age.of.onset-no.covariates_blup_sub_chr-3_peak.marker-UNC4938565_lod.drop-1.5_5.batches_mis.csv
    Untracked:  data/ici.vs.eoi_age.of.onset-no.covariates_blup_sub_chr-3_peak.marker-UNC4938565_lod.drop-1.5_5.batches_mis_0.csv
    Untracked:  data/ici.vs.eoi_age.of.onset-no.covariates_blup_sub_chr-3_peak.marker-UNC4945186_lod.drop-1.5_5.batches_mis.csv
    Untracked:  data/ici.vs.eoi_age.of.onset-no.covariates_blup_sub_chr-3_peak.marker-UNC4945186_lod.drop-1.5_5.batches_mis_0.csv
    Untracked:  data/ici.vs.eoi_age.of.onset-no.covariates_blup_sub_chr-3_peak.marker-UNC4947492_lod.drop-1.5_5.batches_mis.csv
    Untracked:  data/ici.vs.eoi_age.of.onset-no.covariates_blup_sub_chr-3_peak.marker-UNC4947492_lod.drop-1.5_5.batches_mis_0.csv
    Untracked:  data/ici.vs.eoi_age.of.onset-no.covariates_blup_sub_chr-3_peak.marker-UNC4956618_lod.drop-1.5_5.batches_0.csv
    Untracked:  data/ici.vs.eoi_age.of.onset-no.covariates_blup_sub_chr-3_peak.marker-UNC4956618_lod.drop-1.5_5.batches_mis_0.csv
    Untracked:  data/ici.vs.eoi_age.of.onset-no.covariates_blup_sub_chr-3_peak.marker-UNC4957168_lod.drop-1.5_5.batches_52.csv
    Untracked:  data/ici.vs.eoi_age.of.onset-no.covariates_blup_sub_chr-3_peak.marker-UNC4957859_lod.drop-1.5_5.batches_mis.csv
    Untracked:  data/ici.vs.eoi_age.of.onset-no.covariates_blup_sub_chr-3_peak.marker-UNC4957859_lod.drop-1.5_5.batches_mis_0.csv
    Untracked:  data/ici.vs.eoi_age.of.onset-no.covariates_blup_sub_chr-3_peak.marker-UNC4959266_lod.drop-1.5_5.batches_mis.csv
    Untracked:  data/ici.vs.eoi_age.of.onset-no.covariates_blup_sub_chr-3_peak.marker-UNC4959266_lod.drop-1.5_5.batches_mis_0.csv
    Untracked:  data/ici.vs.eoi_age.of.onset-no.covariates_blup_sub_chr-3_peak.marker-UNC4960612_lod.drop-1.5_5.batches_mis.csv
    Untracked:  data/ici.vs.eoi_age.of.onset-no.covariates_blup_sub_chr-3_peak.marker-UNC4960612_lod.drop-1.5_5.batches_mis_0.csv
    Untracked:  data/ici.vs.eoi_age.of.onset-no.covariates_blup_sub_chr-3_peak.marker-UNC4961542_lod.drop-1.5_5.batches_mis.csv
    Untracked:  data/ici.vs.eoi_age.of.onset-no.covariates_blup_sub_chr-3_peak.marker-UNC4961542_lod.drop-1.5_5.batches_mis_0.csv
    Untracked:  data/ici.vs.eoi_age.of.onset-no.covariates_blup_sub_chr-3_peak.marker-UNC4979914_lod.drop-1.5_5.batches_mis.csv
    Untracked:  data/ici.vs.eoi_age.of.onset-no.covariates_blup_sub_chr-3_peak.marker-UNC4979914_lod.drop-1.5_5.batches_mis_0.csv
    Untracked:  data/ici.vs.eoi_age.of.onset-no.covariates_blup_sub_chr-3_peak.marker-UNC4990111_lod.drop-1.5_5.batches_mis.csv
    Untracked:  data/ici.vs.eoi_age.of.onset-no.covariates_blup_sub_chr-3_peak.marker-UNC4990111_lod.drop-1.5_5.batches_mis_0.csv
    Untracked:  data/ici.vs.eoi_age.of.onset-no.covariates_blup_sub_chr-3_peak.marker-UNC5008656_lod.drop-1.5_5.batches_0.csv
    Untracked:  data/ici.vs.eoi_age.of.onset-no.covariates_blup_sub_chr-3_peak.marker-UNC5008656_lod.drop-1.5_5.batches_52.csv
    Untracked:  data/ici.vs.eoi_age.of.onset-no.covariates_blup_sub_chr-3_peak.marker-UNC5008656_lod.drop-1.5_5.batches_mis.csv
    Untracked:  data/ici.vs.eoi_age.of.onset-no.covariates_blup_sub_chr-3_peak.marker-UNC5008656_lod.drop-1.5_5.batches_mis_0.csv
    Untracked:  data/ici.vs.eoi_age.of.onset-no.covariates_blup_sub_chr-3_peak.marker-UNC5046545_lod.drop-1.5_5.batches_mis_0.csv
    Untracked:  data/ici.vs.eoi_age.of.onset-no.covariates_blup_sub_chr-3_peak.marker-UNC5055259_lod.drop-1.5_5.batches_mis_0.csv
    Untracked:  data/ici.vs.eoi_age.of.onset-no.covariates_blup_sub_chr-3_peak.marker-UNC5062524_lod.drop-1.5_5.batches_mis_0.csv
    Untracked:  data/ici.vs.eoi_age.of.onset-no.covariates_blup_sub_chr-3_peak.marker-UNC5068215_lod.drop-1.5_5.batches_mis_0.csv
    Untracked:  data/ici.vs.eoi_age.of.onset-no.covariates_blup_sub_chr-3_peak.marker-UNC5076935_lod.drop-1.5_5.batches_mis_0.csv
    Untracked:  data/ici.vs.eoi_age.of.onset-no.covariates_blup_sub_chr-3_peak.marker-UNC5082757_lod.drop-1.5_5.batches_mis_0.csv
    Untracked:  data/ici.vs.eoi_age.of.onset-no.covariates_blup_sub_chr-3_peak.marker-UNC5083168_lod.drop-1.5_5.batches_0.csv
    Untracked:  data/ici.vs.eoi_age.of.onset-no.covariates_blup_sub_chr-3_peak.marker-UNC5083168_lod.drop-1.5_5.batches_52.csv
    Untracked:  data/ici.vs.eoi_age.of.onset-no.covariates_blup_sub_chr-3_peak.marker-UNC5083168_lod.drop-1.5_5.batches_mis_0.csv
    Untracked:  data/ici.vs.eoi_age.of.onset-no.covariates_blup_sub_chr-3_peak.marker-UNC5173293_lod.drop-1.5_5.batches_mis_0.csv
    Untracked:  data/ici.vs.eoi_age.of.onset-no.covariates_blup_sub_chr-3_peak.marker-UNC5192582_lod.drop-1.5_5.batches_0.csv
    Untracked:  data/ici.vs.eoi_age.of.onset-no.covariates_blup_sub_chr-3_peak.marker-UNC5192582_lod.drop-1.5_5.batches_52.csv
    Untracked:  data/ici.vs.eoi_age.of.onset-no.covariates_blup_sub_chr-3_peak.marker-UNC5192582_lod.drop-1.5_5.batches_mis_0.csv
    Untracked:  data/ici.vs.eoi_age.of.onset-no.covariates_blup_sub_chr-3_peak.marker-UNC5193970_lod.drop-1.5_5.batches_mis_0.csv
    Untracked:  data/ici.vs.eoi_age.of.onset-no.covariates_blup_sub_chr-3_peak.marker-UNC5213888_lod.drop-1.5_snpsqc_5.batches_mis_0.csv
    Untracked:  data/ici.vs.eoi_age.of.onset-no.covariates_blup_sub_chr-3_peak.marker-UNC5213888_lod.drop-1.5_snpsqc_5.batches_mis_52.csv
    Untracked:  data/ici.vs.eoi_age.of.onset-no.covariates_blup_sub_chr-3_peak.marker-UNC5257268_lod.drop-1.5_5.batches_mis_0.csv
    Untracked:  data/ici.vs.eoi_age.of.onset-no.covariates_blup_sub_chr-3_peak.marker-UNC5262472_lod.drop-1.5_5.batches_0.csv
    Untracked:  data/ici.vs.eoi_age.of.onset-no.covariates_blup_sub_chr-3_peak.marker-UNC5262472_lod.drop-1.5_5.batches_52.csv
    Untracked:  data/ici.vs.eoi_age.of.onset-no.covariates_blup_sub_chr-3_peak.marker-UNC5262472_lod.drop-1.5_5.batches_mis_0.csv
    Untracked:  data/ici.vs.eoi_age.of.onset-no.covariates_blup_sub_chr-3_peak.marker-UNC5264696_lod.drop-1.5_5.batches_mis_0.csv
    Untracked:  data/ici.vs.eoi_age.of.onset-no.covariates_blup_sub_chr-3_peak.marker-UNC5267813_lod.drop-1.5_5.batches_mis_0.csv
    Untracked:  data/ici.vs.eoi_age.of.onset-no.covariates_blup_sub_chr-3_peak.marker-UNC5270225_lod.drop-1.5_5.batches_mis_0.csv
    Untracked:  data/ici.vs.eoi_age.of.onset-no.covariates_blup_sub_chr-3_peak.marker-UNC5277061_lod.drop-1.5_5.batches_mis_0.csv
    Untracked:  data/ici.vs.eoi_age.of.onset-no.covariates_blup_sub_chr-3_peak.marker-UNC5278229_lod.drop-1.5_5.batches_0.csv
    Untracked:  data/ici.vs.eoi_age.of.onset-no.covariates_blup_sub_chr-3_peak.marker-UNC5278229_lod.drop-1.5_5.batches_mis_0.csv
    Untracked:  data/ici.vs.eoi_age.of.onset-no.covariates_blup_sub_chr-3_peak.marker-UNC5282259_lod.drop-1.5_5.batches_mis_0.csv
    Untracked:  data/ici.vs.eoi_age.of.onset-no.covariates_blup_sub_chr-3_peak.marker-UNC5287977_lod.drop-1.5_5.batches_mis_0.csv
    Untracked:  data/ici.vs.eoi_age.of.onset-no.covariates_blup_sub_chr-3_peak.marker-UNC5301460_lod.drop-1.5_5.batches_mis_0.csv
    Untracked:  data/ici.vs.eoi_age.of.onset-no.covariates_blup_sub_chr-3_peak.marker-UNC5309136_lod.drop-1.5_5.batches_mis_0.csv
    Untracked:  data/ici.vs.eoi_age.of.onset-no.covariates_blup_sub_chr-3_peak.marker-UNC5321559_lod.drop-1.5_5.batches_mis_0.csv
    Untracked:  data/ici.vs.eoi_age.of.onset-no.covariates_blup_sub_chr-3_peak.marker-UNC5322455_lod.drop-1.5_5.batches_mis_0.csv
    Untracked:  data/ici.vs.eoi_age.of.onset-no.covariates_blup_sub_chr-3_peak.marker-UNC5329171_lod.drop-1.5_5.batches_mis_0.csv
    Untracked:  data/ici.vs.eoi_age.of.onset-no.covariates_blup_sub_chr-3_peak.marker-UNC5360055_lod.drop-1.5_5.batches_52.csv
    Untracked:  data/ici.vs.eoi_age.of.onset-no.covariates_blup_sub_chr-3_peak.marker-UNC5454839_lod.drop-1.5_5.batches_0.csv
    Untracked:  data/ici.vs.eoi_age.of.onset-no.covariates_blup_sub_chr-3_peak.marker-UNC5454839_lod.drop-1.5_5.batches_52.csv
    Untracked:  data/ici.vs.eoi_age.of.onset-no.covariates_blup_sub_chr-3_peak.marker-UNC5454839_lod.drop-1.5_5.batches_mis_0.csv
    Untracked:  data/ici.vs.eoi_age.of.onset-no.covariates_blup_sub_chr-3_peak.marker-UNC5645844_lod.drop-1.5_5.batches_0.csv
    Untracked:  data/ici.vs.eoi_age.of.onset-no.covariates_blup_sub_chr-3_peak.marker-UNC5645844_lod.drop-1.5_5.batches_52.csv
    Untracked:  data/ici.vs.eoi_age.of.onset-no.covariates_blup_sub_chr-3_peak.marker-UNC5645844_lod.drop-1.5_5.batches_mis_0.csv
    Untracked:  data/ici.vs.eoi_age.of.onset-no.covariates_blup_sub_chr-3_peak.marker-UNC5778977_lod.drop-1.5_5.batches_0.csv
    Untracked:  data/ici.vs.eoi_age.of.onset-no.covariates_blup_sub_chr-3_peak.marker-UNC5808246_lod.drop-1.5_5.batches_52.csv
    Untracked:  data/ici.vs.eoi_age.of.onset-no.covariates_blup_sub_chr-3_peak.marker-UNC5916997_lod.drop-1.5_5.batches_52.csv
    Untracked:  data/ici.vs.eoi_age.of.onset-no.covariates_blup_sub_chr-3_peak.marker-UNCHS008182_lod.drop-1.5_5.batches_0.csv
    Untracked:  data/ici.vs.eoi_age.of.onset-no.covariates_blup_sub_chr-3_peak.marker-UNCHS008182_lod.drop-1.5_5.batches_52.csv
    Untracked:  data/ici.vs.eoi_age.of.onset-no.covariates_blup_sub_chr-3_peak.marker-UNCHS008182_lod.drop-1.5_5.batches_mis_0.csv
    Untracked:  data/ici.vs.eoi_age.of.onset-no.covariates_blup_sub_chr-3_peak.marker-UNCHS008193_lod.drop-1.5_5.batches_mis.csv
    Untracked:  data/ici.vs.eoi_age.of.onset-no.covariates_blup_sub_chr-3_peak.marker-UNCHS008193_lod.drop-1.5_5.batches_mis_0.csv
    Untracked:  data/ici.vs.eoi_age.of.onset-no.covariates_blup_sub_chr-3_peak.marker-UNCHS008209_lod.drop-1.5_5.batches_mis.csv
    Untracked:  data/ici.vs.eoi_age.of.onset-no.covariates_blup_sub_chr-3_peak.marker-UNCHS008209_lod.drop-1.5_5.batches_mis_0.csv
    Untracked:  data/ici.vs.eoi_age.of.onset-no.covariates_blup_sub_chr-3_peak.marker-UNCHS008215_lod.drop-1.5_5.batches_mis.csv
    Untracked:  data/ici.vs.eoi_age.of.onset-no.covariates_blup_sub_chr-3_peak.marker-UNCHS008215_lod.drop-1.5_5.batches_mis_0.csv
    Untracked:  data/ici.vs.eoi_age.of.onset-no.covariates_blup_sub_chr-3_peak.marker-UNCHS008219_lod.drop-1.5_5.batches_mis.csv
    Untracked:  data/ici.vs.eoi_age.of.onset-no.covariates_blup_sub_chr-3_peak.marker-UNCHS008219_lod.drop-1.5_5.batches_mis_0.csv
    Untracked:  data/ici.vs.eoi_age.of.onset-no.covariates_blup_sub_chr-3_peak.marker-UNCHS008229_lod.drop-1.5_5.batches_mis.csv
    Untracked:  data/ici.vs.eoi_age.of.onset-no.covariates_blup_sub_chr-3_peak.marker-UNCHS008229_lod.drop-1.5_5.batches_mis_0.csv
    Untracked:  data/ici.vs.eoi_age.of.onset-no.covariates_blup_sub_chr-3_peak.marker-UNCHS008234_lod.drop-1.5_5.batches_mis.csv
    Untracked:  data/ici.vs.eoi_age.of.onset-no.covariates_blup_sub_chr-3_peak.marker-UNCHS008234_lod.drop-1.5_5.batches_mis_0.csv
    Untracked:  data/ici.vs.eoi_age.of.onset-no.covariates_blup_sub_chr-3_peak.marker-UNCHS008242_lod.drop-1.5_5.batches_52.csv
    Untracked:  data/ici.vs.eoi_age.of.onset-no.covariates_blup_sub_chr-3_peak.marker-UNCHS008260_lod.drop-1.5_5.batches_mis.csv
    Untracked:  data/ici.vs.eoi_age.of.onset-no.covariates_blup_sub_chr-3_peak.marker-UNCHS008260_lod.drop-1.5_5.batches_mis_0.csv
    Untracked:  data/ici.vs.eoi_age.of.onset-no.covariates_blup_sub_chr-3_peak.marker-UNCHS008271_lod.drop-1.5_5.batches_mis.csv
    Untracked:  data/ici.vs.eoi_age.of.onset-no.covariates_blup_sub_chr-3_peak.marker-UNCHS008271_lod.drop-1.5_5.batches_mis_0.csv
    Untracked:  data/ici.vs.eoi_age.of.onset-no.covariates_blup_sub_chr-3_peak.marker-UNCHS008281_lod.drop-1.5_5.batches_mis.csv
    Untracked:  data/ici.vs.eoi_age.of.onset-no.covariates_blup_sub_chr-3_peak.marker-UNCHS008281_lod.drop-1.5_5.batches_mis_0.csv
    Untracked:  data/ici.vs.eoi_age.of.onset-no.covariates_blup_sub_chr-3_peak.marker-UNCHS008286_lod.drop-1.5_5.batches_0.csv
    Untracked:  data/ici.vs.eoi_age.of.onset-no.covariates_blup_sub_chr-3_peak.marker-UNCHS008286_lod.drop-1.5_5.batches_52.csv
    Untracked:  data/ici.vs.eoi_age.of.onset-no.covariates_blup_sub_chr-3_peak.marker-UNCHS008286_lod.drop-1.5_5.batches_mis_0.csv
    Untracked:  data/ici.vs.eoi_age.of.onset-no.covariates_blup_sub_chr-3_peak.marker-UNCHS008338_lod.drop-1.5_5.batches_mis.csv
    Untracked:  data/ici.vs.eoi_age.of.onset-no.covariates_blup_sub_chr-3_peak.marker-UNCHS008338_lod.drop-1.5_5.batches_mis_0.csv
    Untracked:  data/ici.vs.eoi_age.of.onset-no.covariates_blup_sub_chr-3_peak.marker-UNCHS008363_lod.drop-1.5_5.batches_mis.csv
    Untracked:  data/ici.vs.eoi_age.of.onset-no.covariates_blup_sub_chr-3_peak.marker-UNCHS008363_lod.drop-1.5_5.batches_mis_0.csv
    Untracked:  data/ici.vs.eoi_age.of.onset-no.covariates_blup_sub_chr-3_peak.marker-UNCHS008379_lod.drop-1.5_5.batches_mis.csv
    Untracked:  data/ici.vs.eoi_age.of.onset-no.covariates_blup_sub_chr-3_peak.marker-UNCHS008379_lod.drop-1.5_5.batches_mis_0.csv
    Untracked:  data/ici.vs.eoi_age.of.onset-no.covariates_blup_sub_chr-3_peak.marker-UNCHS008409_lod.drop-1.5_5.batches_mis_0.csv
    Untracked:  data/ici.vs.eoi_age.of.onset-no.covariates_blup_sub_chr-3_peak.marker-UNCHS008432_lod.drop-1.5_5.batches_mis_0.csv
    Untracked:  data/ici.vs.eoi_age.of.onset-no.covariates_blup_sub_chr-3_peak.marker-UNCHS008487_lod.drop-1.5_snpsqc_5.batches_0.csv
    Untracked:  data/ici.vs.eoi_age.of.onset-no.covariates_blup_sub_chr-3_peak.marker-UNCHS008487_lod.drop-1.5_snpsqc_5.batches_52.csv
    Untracked:  data/ici.vs.eoi_age.of.onset-no.covariates_blup_sub_chr-3_peak.marker-UNCHS008487_lod.drop-1.5_snpsqc_dis_no-x_updated_5.batches_0.csv
    Untracked:  data/ici.vs.eoi_age.of.onset-no.covariates_blup_sub_chr-3_peak.marker-UNCHS008487_lod.drop-1.5_snpsqc_dis_no-x_updated_5.batches_52.csv
    Untracked:  data/ici.vs.eoi_age.of.onset-no.covariates_blup_sub_chr-3_peak.marker-UNCHS008487_lod.drop-1.5_snpsqc_dis_no-x_updated_5.batches_mis_0.csv
    Untracked:  data/ici.vs.eoi_age.of.onset-no.covariates_blup_sub_chr-3_peak.marker-UNCHS008487_lod.drop-1.5_snpsqc_dis_no-x_updated_5.batches_mis_52.csv
    Untracked:  data/ici.vs.eoi_age.of.onset-no.covariates_blup_sub_chr-3_peak.marker-UNCHS008511_lod.drop-1.5_snpsqc_5.batches_mis_0.csv
    Untracked:  data/ici.vs.eoi_age.of.onset-no.covariates_blup_sub_chr-3_peak.marker-UNCHS008511_lod.drop-1.5_snpsqc_5.batches_mis_52.csv
    Untracked:  data/ici.vs.eoi_age.of.onset-no.covariates_blup_sub_chr-3_peak.marker-UNCHS008609_lod.drop-1.5_5.batches_mis_0.csv
    Untracked:  data/ici.vs.eoi_age.of.onset-no.covariates_blup_sub_chr-3_peak.marker-UNCHS008613_lod.drop-1.5_5.batches_0.csv
    Untracked:  data/ici.vs.eoi_age.of.onset-no.covariates_blup_sub_chr-3_peak.marker-UNCHS008613_lod.drop-1.5_5.batches_52.csv
    Untracked:  data/ici.vs.eoi_age.of.onset-no.covariates_blup_sub_chr-3_peak.marker-UNCHS008614_lod.drop-1.5_5.batches_mis_0.csv
    Untracked:  data/ici.vs.eoi_age.of.onset-no.covariates_blup_sub_chr-3_peak.marker-UNCHS008627_lod.drop-1.5_5.batches_mis_0.csv
    Untracked:  data/ici.vs.eoi_age.of.onset-no.covariates_blup_sub_chr-3_peak.marker-UNCHS008659_lod.drop-1.5_5.batches_52.csv
    Untracked:  data/ici.vs.eoi_age.of.onset-no.covariates_blup_sub_chr-3_peak.marker-UNCHS008725_lod.drop-1.5_5.batches_mis_0.csv
    Untracked:  data/ici.vs.eoi_age.of.onset-no.covariates_blup_sub_chr-3_peak.marker-UNCHS008815_lod.drop-1.5_5.batches_0.csv
    Untracked:  data/ici.vs.eoi_age.of.onset-no.covariates_blup_sub_chr-3_peak.marker-UNCHS008815_lod.drop-1.5_5.batches_52.csv
    Untracked:  data/ici.vs.eoi_age.of.onset-no.covariates_blup_sub_chr-3_peak.marker-UNCHS008815_lod.drop-1.5_5.batches_mis_0.csv
    Untracked:  data/ici.vs.eoi_age.of.onset-no.covariates_blup_sub_chr-3_peak.marker-UNCHS009066_lod.drop-1.5_5.batches_52.csv
    Untracked:  data/ici.vs.eoi_age.of.onset-no.covariates_blup_sub_chr-3_peak.marker-UNCHS009067_lod.drop-1.5_5.batches_0.csv
    Untracked:  data/ici.vs.eoi_age.of.onset-no.covariates_blup_sub_chr-3_peak.marker-UNCHS009068_lod.drop-1.5_5.batches_mis_0.csv
    Untracked:  data/ici.vs.eoi_age.of.onset-no.covariates_blup_sub_chr-3_peak.marker-UNCHS009112_lod.drop-1.5_5.batches_52.csv
    Untracked:  data/ici.vs.eoi_age.of.onset-no.covariates_blup_sub_chr-3_peak.marker-UNCHS009224_lod.drop-1.5_5.batches_0.csv
    Untracked:  data/ici.vs.eoi_age.of.onset-no.covariates_blup_sub_chr-3_peak.marker-UNCHS009224_lod.drop-1.5_5.batches_52.csv
    Untracked:  data/ici.vs.eoi_age.of.onset-no.covariates_blup_sub_chr-3_peak.marker-UNCHS009383_lod.drop-1.5_5.batches_0.csv
    Untracked:  data/ici.vs.eoi_age.of.onset-no.covariates_blup_sub_chr-3_peak.marker-UNCHS009383_lod.drop-1.5_5.batches_52.csv
    Untracked:  data/ici.vs.eoi_age.of.onset-no.covariates_blup_sub_chr-3_peak.marker-UNCHS009693_lod.drop-1.5_5.batches_52.csv
    Untracked:  data/ici.vs.eoi_age.of.onset-no.covariates_blup_sub_chr-3_peak.marker-UNCHS009821_lod.drop-1.5_5.batches_52.csv
    Untracked:  data/ici.vs.eoi_age.of.onset-no.covariates_blup_sub_chr-3_peak.marker-UNCJPD001170_lod.drop-1.5_5.batches_mis.csv
    Untracked:  data/ici.vs.eoi_age.of.onset-no.covariates_blup_sub_chr-3_peak.marker-UNCJPD001170_lod.drop-1.5_5.batches_mis_0.csv
    Untracked:  data/ici.vs.eoi_age.of.onset-no.covariates_blup_sub_chr-3_peak.marker-UNCJPD001198_lod.drop-1.5_5.batches_mis.csv
    Untracked:  data/ici.vs.eoi_age.of.onset-no.covariates_blup_sub_chr-3_peak.marker-UNCJPD001198_lod.drop-1.5_5.batches_mis_0.csv
    Untracked:  data/ici.vs.eoi_age.of.onset-no.covariates_blup_sub_chr-3_peak.marker-UNCJPD001276_lod.drop-1.5_5.batches_mis_0.csv
    Untracked:  data/ici.vs.eoi_age.of.onset-no.covariates_blup_sub_chr-3_peak.marker-sanger2496q_lod.drop-1.5_5.batches_mis.csv
    Untracked:  data/ici.vs.eoi_age.of.onset-no.covariates_blup_sub_chr-3_peak.marker-sanger2496q_lod.drop-1.5_5.batches_mis_0.csv
    Untracked:  data/ici.vs.eoi_age.of.onset-no.covariates_blup_sub_chr-4_peak.marker-UNC8250659_lod.drop-1.5_snpsqc_dis_no-x_updated_5.batches_52.csv
    Untracked:  data/ici.vs.eoi_age.of.onset-no.covariates_blup_sub_chr-4_peak.marker-UNC8439633_lod.drop-1.5_snpsqc_5.batches_52.csv
    Untracked:  data/ici.vs.eoi_age.of.onset-no.covariates_blup_sub_chr-4_peak.marker-UNC8439633_lod.drop-1.5_snpsqc_5.batches_mis_52.csv
    Untracked:  data/ici.vs.eoi_age.of.onset-no.covariates_blup_sub_chr-4_peak.marker-UNCHS012955_lod.drop-1.5_snpsqc_dis_no-x_updated_5.batches_mis_52.csv
    Untracked:  data/ici.vs.eoi_age.of.onset-no.covariates_blup_sub_chr-6_peak.marker-UNC11108920_lod.drop-1.5_snpsqc_5.batches.csv
    Untracked:  data/ici.vs.eoi_age.of.onset-no.covariates_blup_sub_chr-6_peak.marker-UNC11108920_lod.drop-1.5_snpsqc_5.batches_mis.csv
    Untracked:  data/ici.vs.eoi_age.of.onset-no.covariates_blup_sub_chr-7_peak.marker-UNCHS020066_lod.drop-1.5_snpsqc_5.batches_mis_0.csv
    Untracked:  data/ici.vs.eoi_age.of.onset-no.covariates_blup_sub_chr-7_peak.marker-UNCHS020066_lod.drop-1.5_snpsqc_5.batches_mis_52.csv
    Untracked:  data/ici.vs.eoi_age.of.onset-no.covariates_blup_sub_chr-7_peak.marker-UNCHS020066_lod.drop-1.5_snpsqc_dis_no-x_updated_5.batches_mis_0.csv
    Untracked:  data/ici.vs.eoi_age.of.onset-no.covariates_blup_sub_chr-7_peak.marker-UNCHS020066_lod.drop-1.5_snpsqc_dis_no-x_updated_5.batches_mis_52.csv
    Untracked:  data/ici.vs.eoi_age.of.onset-no.covariates_blup_sub_chr-8_peak.marker-UNC15524531_lod.drop-1.5_snpsqc_5.batches_mis_0.csv
    Untracked:  data/ici.vs.eoi_age.of.onset-no.covariates_blup_sub_chr-8_peak.marker-UNC15524531_lod.drop-1.5_snpsqc_5.batches_mis_52.csv
    Untracked:  data/ici.vs.eoi_age.of.onset-no.covariates_blup_sub_chr-8_peak.marker-UNC15524531_lod.drop-1.5_snpsqc_dis_no-x_updated_5.batches_mis_0.csv
    Untracked:  data/ici.vs.eoi_age.of.onset-no.covariates_blup_sub_chr-8_peak.marker-UNC15524531_lod.drop-1.5_snpsqc_dis_no-x_updated_5.batches_mis_52.csv
    Untracked:  data/ici.vs.eoi_age.of.onset-no.covariates_blup_sub_chr-9_peak.marker-UNC17203597_lod.drop-1.5_snpsqc_5.batches_mis_0.csv
    Untracked:  data/ici.vs.eoi_age.of.onset-no.covariates_blup_sub_chr-9_peak.marker-UNC17203597_lod.drop-1.5_snpsqc_5.batches_mis_52.csv
    Untracked:  data/ici.vs.eoi_age.of.onset-no.covariates_blup_sub_chr-X_peak.marker-UNCHS048314_lod.drop-1.5_snpsqc_5.batches_mis_0.csv
    Untracked:  data/ici.vs.eoi_age.of.onset-no.covariates_blup_sub_chr-X_peak.marker-UNCHS048314_lod.drop-1.5_snpsqc_5.batches_mis_52.csv
    Untracked:  data/ici.vs.eoi_age.of.onset-no.covariates_blup_sub_chr-X_peak.marker-UNCHS048314_lod.drop-1.5_snpsqc_dis_no-x_updated_5.batches_mis_0.csv
    Untracked:  data/ici.vs.eoi_age.of.onset-no.covariates_blup_sub_chr-X_peak.marker-UNCHS048314_lod.drop-1.5_snpsqc_dis_no-x_updated_5.batches_mis_52.csv
    Untracked:  data/ici.vs.eoi_age.of.onset-no.covariates_genes_chr-10_peak.marker-JAX00020646_lod.drop-1.5_5.batches_0.csv
    Untracked:  data/ici.vs.eoi_age.of.onset-no.covariates_genes_chr-10_peak.marker-JAX00292499_lod.drop-1.5_5.batches_52.csv
    Untracked:  data/ici.vs.eoi_age.of.onset-no.covariates_genes_chr-10_peak.marker-JAX00292927_lod.drop-1.5_5.batches_0.csv
    Untracked:  data/ici.vs.eoi_age.of.onset-no.covariates_genes_chr-10_peak.marker-JAX00292927_lod.drop-1.5_5.batches_52.csv
    Untracked:  data/ici.vs.eoi_age.of.onset-no.covariates_genes_chr-10_peak.marker-JAX00294019_lod.drop-1.5_snpsqc_dis_no-x_updated_5.batches_0.csv
    Untracked:  data/ici.vs.eoi_age.of.onset-no.covariates_genes_chr-10_peak.marker-UNC18216614_lod.drop-1.5_5.batches_52.csv
    Untracked:  data/ici.vs.eoi_age.of.onset-no.covariates_genes_chr-10_peak.marker-UNC18240977_lod.drop-1.5_snpsqc_dis_no-x_updated_5.batches_52.csv
    Untracked:  data/ici.vs.eoi_age.of.onset-no.covariates_genes_chr-10_peak.marker-UNC18311938_lod.drop-1.5_5.batches_0.csv
    Untracked:  data/ici.vs.eoi_age.of.onset-no.covariates_genes_chr-10_peak.marker-UNC18311938_lod.drop-1.5_5.batches_52.csv
    Untracked:  data/ici.vs.eoi_age.of.onset-no.covariates_genes_chr-10_peak.marker-UNC18311938_lod.drop-1.5_snpsqc_5.batches_52.csv
    Untracked:  data/ici.vs.eoi_age.of.onset-no.covariates_genes_chr-10_peak.marker-UNC18311938_lod.drop-1.5_snpsqc_5.batches_mis_52.csv
    Untracked:  data/ici.vs.eoi_age.of.onset-no.covariates_genes_chr-10_peak.marker-UNC18343181_lod.drop-1.5_5.batches_0.csv
    Untracked:  data/ici.vs.eoi_age.of.onset-no.covariates_genes_chr-10_peak.marker-UNC18363544_lod.drop-1.5_5.batches_0.csv
    Untracked:  data/ici.vs.eoi_age.of.onset-no.covariates_genes_chr-10_peak.marker-UNC18363544_lod.drop-1.5_5.batches_52.csv
    Untracked:  data/ici.vs.eoi_age.of.onset-no.covariates_genes_chr-10_peak.marker-UNC18363544_lod.drop-1.5_snpsqc_5.batches_mis_0.csv
    Untracked:  data/ici.vs.eoi_age.of.onset-no.covariates_genes_chr-10_peak.marker-UNC18376338_lod.drop-1.5_snpsqc_5.batches_0.csv
    Untracked:  data/ici.vs.eoi_age.of.onset-no.covariates_genes_chr-10_peak.marker-UNCHS028236_lod.drop-1.5_snpsqc_5.batches_mis_0.csv
    Untracked:  data/ici.vs.eoi_age.of.onset-no.covariates_genes_chr-10_peak.marker-UNCHS028236_lod.drop-1.5_snpsqc_5.batches_mis_52.csv
    Untracked:  data/ici.vs.eoi_age.of.onset-no.covariates_genes_chr-10_peak.marker-UNCHS028536_lod.drop-1.5_snpsqc_dis_no-x_updated_5.batches_mis_0.csv
    Untracked:  data/ici.vs.eoi_age.of.onset-no.covariates_genes_chr-10_peak.marker-UNCHS028536_lod.drop-1.5_snpsqc_dis_no-x_updated_5.batches_mis_52.csv
    Untracked:  data/ici.vs.eoi_age.of.onset-no.covariates_genes_chr-11_peak.marker-UNC19970181_lod.drop-1.5_snpsqc_5.batches_mis_0.csv
    Untracked:  data/ici.vs.eoi_age.of.onset-no.covariates_genes_chr-11_peak.marker-UNC19970181_lod.drop-1.5_snpsqc_5.batches_mis_52.csv
    Untracked:  data/ici.vs.eoi_age.of.onset-no.covariates_genes_chr-11_peak.marker-UNC20090524_lod.drop-1.5_snpsqc_5.batches_52.csv
    Untracked:  data/ici.vs.eoi_age.of.onset-no.covariates_genes_chr-12_peak.marker-ICR499_lod.drop-1.5_snpsqc_5.batches_mis_0.csv
    Untracked:  data/ici.vs.eoi_age.of.onset-no.covariates_genes_chr-12_peak.marker-ICR499_lod.drop-1.5_snpsqc_5.batches_mis_52.csv
    Untracked:  data/ici.vs.eoi_age.of.onset-no.covariates_genes_chr-12_peak.marker-ICR499_lod.drop-1.5_snpsqc_dis_no-x_updated_5.batches_mis_0.csv
    Untracked:  data/ici.vs.eoi_age.of.onset-no.covariates_genes_chr-13_peak.marker-UNCHS036773_lod.drop-1.5_snpsqc_5.batches_mis_0.csv
    Untracked:  data/ici.vs.eoi_age.of.onset-no.covariates_genes_chr-13_peak.marker-UNCHS036773_lod.drop-1.5_snpsqc_5.batches_mis_52.csv
    Untracked:  data/ici.vs.eoi_age.of.onset-no.covariates_genes_chr-14_peak.marker-UNC24056202_lod.drop-1.5_snpsqc_5.batches_mis_0.csv
    Untracked:  data/ici.vs.eoi_age.of.onset-no.covariates_genes_chr-14_peak.marker-UNC24056202_lod.drop-1.5_snpsqc_5.batches_mis_52.csv
    Untracked:  data/ici.vs.eoi_age.of.onset-no.covariates_genes_chr-15_peak.marker-UNC26070435_lod.drop-1.5_snpsqc_dis_no-x_updated_5.batches_mis_0.csv
    Untracked:  data/ici.vs.eoi_age.of.onset-no.covariates_genes_chr-15_peak.marker-UNC26070435_lod.drop-1.5_snpsqc_dis_no-x_updated_5.batches_mis_52.csv
    Untracked:  data/ici.vs.eoi_age.of.onset-no.covariates_genes_chr-17_peak.marker-UNCHS044241_lod.drop-1.5_snpsqc_5.batches_mis_0.csv
    Untracked:  data/ici.vs.eoi_age.of.onset-no.covariates_genes_chr-17_peak.marker-UNCHS044241_lod.drop-1.5_snpsqc_5.batches_mis_52.csv
    Untracked:  data/ici.vs.eoi_age.of.onset-no.covariates_genes_chr-17_peak.marker-UNCJPD006614_lod.drop-1.5_snpsqc_dis_no-x_updated_5.batches_mis_0.csv
    Untracked:  data/ici.vs.eoi_age.of.onset-no.covariates_genes_chr-17_peak.marker-UNCrs47191360_lod.drop-1.5_snpsqc_dis_no-x_updated_5.batches_mis_52.csv
    Untracked:  data/ici.vs.eoi_age.of.onset-no.covariates_genes_chr-18_peak.marker-UNCHS045343_lod.drop-1.5_snpsqc_5.batches_mis_0.csv
    Untracked:  data/ici.vs.eoi_age.of.onset-no.covariates_genes_chr-18_peak.marker-UNCHS045343_lod.drop-1.5_snpsqc_5.batches_mis_52.csv
    Untracked:  data/ici.vs.eoi_age.of.onset-no.covariates_genes_chr-2_peak.marker-UNC4609527_lod.drop-1.5_snpsqc_dis_no-x_updated_5.batches_mis_0.csv
    Untracked:  data/ici.vs.eoi_age.of.onset-no.covariates_genes_chr-2_peak.marker-UNC4609527_lod.drop-1.5_snpsqc_dis_no-x_updated_5.batches_mis_52.csv
    Untracked:  data/ici.vs.eoi_age.of.onset-no.covariates_genes_chr-2_peak.marker-UNCHS008007_lod.drop-1.5_5.batches_mis.csv
    Untracked:  data/ici.vs.eoi_age.of.onset-no.covariates_genes_chr-2_peak.marker-UNCHS008007_lod.drop-1.5_5.batches_mis_0.csv
    Untracked:  data/ici.vs.eoi_age.of.onset-no.covariates_genes_chr-3_peak.marker-ICR1338_lod.drop-1.5_5.batches_0.csv
    Untracked:  data/ici.vs.eoi_age.of.onset-no.covariates_genes_chr-3_peak.marker-ICR1338_lod.drop-1.5_5.batches_52.csv
    Untracked:  data/ici.vs.eoi_age.of.onset-no.covariates_genes_chr-3_peak.marker-ICR1338_lod.drop-1.5_5.batches_mis_0.csv
    Untracked:  data/ici.vs.eoi_age.of.onset-no.covariates_genes_chr-3_peak.marker-ICR5263_lod.drop-1.5_5.batches_mis.csv
    Untracked:  data/ici.vs.eoi_age.of.onset-no.covariates_genes_chr-3_peak.marker-ICR5263_lod.drop-1.5_5.batches_mis_0.csv
    Untracked:  data/ici.vs.eoi_age.of.onset-no.covariates_genes_chr-3_peak.marker-JAX00105649_lod.drop-1.5_5.batches_mis.csv
    Untracked:  data/ici.vs.eoi_age.of.onset-no.covariates_genes_chr-3_peak.marker-JAX00105649_lod.drop-1.5_5.batches_mis_0.csv
    Untracked:  data/ici.vs.eoi_age.of.onset-no.covariates_genes_chr-3_peak.marker-JAX00106210_lod.drop-1.5_5.batches_mis_0.csv
    Untracked:  data/ici.vs.eoi_age.of.onset-no.covariates_genes_chr-3_peak.marker-JAX00106957_lod.drop-1.5_5.batches_0.csv
    Untracked:  data/ici.vs.eoi_age.of.onset-no.covariates_genes_chr-3_peak.marker-JAX00106957_lod.drop-1.5_5.batches_52.csv
    Untracked:  data/ici.vs.eoi_age.of.onset-no.covariates_genes_chr-3_peak.marker-JAX00106957_lod.drop-1.5_5.batches_mis_0.csv
    Untracked:  data/ici.vs.eoi_age.of.onset-no.covariates_genes_chr-3_peak.marker-JAX00107193_lod.drop-1.5_5.batches_mis_0.csv
    Untracked:  data/ici.vs.eoi_age.of.onset-no.covariates_genes_chr-3_peak.marker-JAX00220631_lod.drop-1.5_5.batches_0.csv
    Untracked:  data/ici.vs.eoi_age.of.onset-no.covariates_genes_chr-3_peak.marker-JAX00220631_lod.drop-1.5_5.batches_mis_0.csv
    Untracked:  data/ici.vs.eoi_age.of.onset-no.covariates_genes_chr-3_peak.marker-JAX00514662_lod.drop-1.5_5.batches_0.csv
    Untracked:  data/ici.vs.eoi_age.of.onset-no.covariates_genes_chr-3_peak.marker-JAX00514662_lod.drop-1.5_5.batches_52.csv
    Untracked:  data/ici.vs.eoi_age.of.onset-no.covariates_genes_chr-3_peak.marker-JAX00516812_lod.drop-1.5_5.batches_mis.csv
    Untracked:  data/ici.vs.eoi_age.of.onset-no.covariates_genes_chr-3_peak.marker-JAX00516812_lod.drop-1.5_5.batches_mis_0.csv
    Untracked:  data/ici.vs.eoi_age.of.onset-no.covariates_genes_chr-3_peak.marker-JAX00517674_lod.drop-1.5_5.batches_mis.csv
    Untracked:  data/ici.vs.eoi_age.of.onset-no.covariates_genes_chr-3_peak.marker-JAX00517674_lod.drop-1.5_5.batches_mis_0.csv
    Untracked:  data/ici.vs.eoi_age.of.onset-no.covariates_genes_chr-3_peak.marker-JAX00518911r_lod.drop-1.5_5.batches_mis.csv
    Untracked:  data/ici.vs.eoi_age.of.onset-no.covariates_genes_chr-3_peak.marker-JAX00518911r_lod.drop-1.5_5.batches_mis_0.csv
    Untracked:  data/ici.vs.eoi_age.of.onset-no.covariates_genes_chr-3_peak.marker-JAX00522922_lod.drop-1.5_5.batches_mis_0.csv
    Untracked:  data/ici.vs.eoi_age.of.onset-no.covariates_genes_chr-3_peak.marker-JAX00524205_lod.drop-1.5_5.batches_mis_0.csv
    Untracked:  data/ici.vs.eoi_age.of.onset-no.covariates_genes_chr-3_peak.marker-JAX00529707r_lod.drop-1.5_5.batches_0.csv
    Untracked:  data/ici.vs.eoi_age.of.onset-no.covariates_genes_chr-3_peak.marker-JAX00529707r_lod.drop-1.5_5.batches_52.csv
    Untracked:  data/ici.vs.eoi_age.of.onset-no.covariates_genes_chr-3_peak.marker-JAX00535764_lod.drop-1.5_5.batches_52.csv
    Untracked:  data/ici.vs.eoi_age.of.onset-no.covariates_genes_chr-3_peak.marker-UNC4809586_lod.drop-1.5_5.batches_mis.csv
    Untracked:  data/ici.vs.eoi_age.of.onset-no.covariates_genes_chr-3_peak.marker-UNC4809586_lod.drop-1.5_5.batches_mis_0.csv
    Untracked:  data/ici.vs.eoi_age.of.onset-no.covariates_genes_chr-3_peak.marker-UNC4813186_lod.drop-1.5_5.batches_0.csv
    Untracked:  data/ici.vs.eoi_age.of.onset-no.covariates_genes_chr-3_peak.marker-UNC4813186_lod.drop-1.5_5.batches_52.csv
    Untracked:  data/ici.vs.eoi_age.of.onset-no.covariates_genes_chr-3_peak.marker-UNC4813186_lod.drop-1.5_5.batches_mis.csv
    Untracked:  data/ici.vs.eoi_age.of.onset-no.covariates_genes_chr-3_peak.marker-UNC4813186_lod.drop-1.5_5.batches_mis_0.csv
    Untracked:  data/ici.vs.eoi_age.of.onset-no.covariates_genes_chr-3_peak.marker-UNC4820379_lod.drop-1.5_5.batches_mis.csv
    Untracked:  data/ici.vs.eoi_age.of.onset-no.covariates_genes_chr-3_peak.marker-UNC4820379_lod.drop-1.5_5.batches_mis_0.csv
    Untracked:  data/ici.vs.eoi_age.of.onset-no.covariates_genes_chr-3_peak.marker-UNC4836550_lod.drop-1.5_5.batches_0.csv
    Untracked:  data/ici.vs.eoi_age.of.onset-no.covariates_genes_chr-3_peak.marker-UNC4836550_lod.drop-1.5_5.batches_52.csv
    Untracked:  data/ici.vs.eoi_age.of.onset-no.covariates_genes_chr-3_peak.marker-UNC4836550_lod.drop-1.5_5.batches_mis_0.csv
    Untracked:  data/ici.vs.eoi_age.of.onset-no.covariates_genes_chr-3_peak.marker-UNC4875653_lod.drop-1.5_5.batches_mis.csv
    Untracked:  data/ici.vs.eoi_age.of.onset-no.covariates_genes_chr-3_peak.marker-UNC4875653_lod.drop-1.5_5.batches_mis_0.csv
    Untracked:  data/ici.vs.eoi_age.of.onset-no.covariates_genes_chr-3_peak.marker-UNC4901083_lod.drop-1.5_5.batches_mis.csv
    Untracked:  data/ici.vs.eoi_age.of.onset-no.covariates_genes_chr-3_peak.marker-UNC4901083_lod.drop-1.5_5.batches_mis_0.csv
    Untracked:  data/ici.vs.eoi_age.of.onset-no.covariates_genes_chr-3_peak.marker-UNC4909519_lod.drop-1.5_5.batches_mis.csv
    Untracked:  data/ici.vs.eoi_age.of.onset-no.covariates_genes_chr-3_peak.marker-UNC4909519_lod.drop-1.5_5.batches_mis_0.csv
    Untracked:  data/ici.vs.eoi_age.of.onset-no.covariates_genes_chr-3_peak.marker-UNC4912039_lod.drop-1.5_5.batches_0.csv
    Untracked:  data/ici.vs.eoi_age.of.onset-no.covariates_genes_chr-3_peak.marker-UNC4912039_lod.drop-1.5_5.batches_mis_0.csv
    Untracked:  data/ici.vs.eoi_age.of.onset-no.covariates_genes_chr-3_peak.marker-UNC4912288_lod.drop-1.5_5.batches_mis.csv
    Untracked:  data/ici.vs.eoi_age.of.onset-no.covariates_genes_chr-3_peak.marker-UNC4912288_lod.drop-1.5_5.batches_mis_0.csv
    Untracked:  data/ici.vs.eoi_age.of.onset-no.covariates_genes_chr-3_peak.marker-UNC4915446_lod.drop-1.5_5.batches_mis.csv
    Untracked:  data/ici.vs.eoi_age.of.onset-no.covariates_genes_chr-3_peak.marker-UNC4915446_lod.drop-1.5_5.batches_mis_0.csv
    Untracked:  data/ici.vs.eoi_age.of.onset-no.covariates_genes_chr-3_peak.marker-UNC4923398_lod.drop-1.5_5.batches_mis.csv
    Untracked:  data/ici.vs.eoi_age.of.onset-no.covariates_genes_chr-3_peak.marker-UNC4923398_lod.drop-1.5_5.batches_mis_0.csv
    Untracked:  data/ici.vs.eoi_age.of.onset-no.covariates_genes_chr-3_peak.marker-UNC4924377_lod.drop-1.5_5.batches_0.csv
    Untracked:  data/ici.vs.eoi_age.of.onset-no.covariates_genes_chr-3_peak.marker-UNC4924377_lod.drop-1.5_5.batches_52.csv
    Untracked:  data/ici.vs.eoi_age.of.onset-no.covariates_genes_chr-3_peak.marker-UNC4924377_lod.drop-1.5_5.batches_mis_0.csv
    Untracked:  data/ici.vs.eoi_age.of.onset-no.covariates_genes_chr-3_peak.marker-UNC4938565_lod.drop-1.5_5.batches_mis.csv
    Untracked:  data/ici.vs.eoi_age.of.onset-no.covariates_genes_chr-3_peak.marker-UNC4938565_lod.drop-1.5_5.batches_mis_0.csv
    Untracked:  data/ici.vs.eoi_age.of.onset-no.covariates_genes_chr-3_peak.marker-UNC4945186_lod.drop-1.5_5.batches_mis.csv
    Untracked:  data/ici.vs.eoi_age.of.onset-no.covariates_genes_chr-3_peak.marker-UNC4945186_lod.drop-1.5_5.batches_mis_0.csv
    Untracked:  data/ici.vs.eoi_age.of.onset-no.covariates_genes_chr-3_peak.marker-UNC4947492_lod.drop-1.5_5.batches_mis.csv
    Untracked:  data/ici.vs.eoi_age.of.onset-no.covariates_genes_chr-3_peak.marker-UNC4947492_lod.drop-1.5_5.batches_mis_0.csv
    Untracked:  data/ici.vs.eoi_age.of.onset-no.covariates_genes_chr-3_peak.marker-UNC4956618_lod.drop-1.5_5.batches_0.csv
    Untracked:  data/ici.vs.eoi_age.of.onset-no.covariates_genes_chr-3_peak.marker-UNC4956618_lod.drop-1.5_5.batches_mis_0.csv
    Untracked:  data/ici.vs.eoi_age.of.onset-no.covariates_genes_chr-3_peak.marker-UNC4957168_lod.drop-1.5_5.batches_52.csv
    Untracked:  data/ici.vs.eoi_age.of.onset-no.covariates_genes_chr-3_peak.marker-UNC4957859_lod.drop-1.5_5.batches_mis.csv
    Untracked:  data/ici.vs.eoi_age.of.onset-no.covariates_genes_chr-3_peak.marker-UNC4957859_lod.drop-1.5_5.batches_mis_0.csv
    Untracked:  data/ici.vs.eoi_age.of.onset-no.covariates_genes_chr-3_peak.marker-UNC4959266_lod.drop-1.5_5.batches_mis.csv
    Untracked:  data/ici.vs.eoi_age.of.onset-no.covariates_genes_chr-3_peak.marker-UNC4959266_lod.drop-1.5_5.batches_mis_0.csv
    Untracked:  data/ici.vs.eoi_age.of.onset-no.covariates_genes_chr-3_peak.marker-UNC4960612_lod.drop-1.5_5.batches_mis.csv
    Untracked:  data/ici.vs.eoi_age.of.onset-no.covariates_genes_chr-3_peak.marker-UNC4960612_lod.drop-1.5_5.batches_mis_0.csv
    Untracked:  data/ici.vs.eoi_age.of.onset-no.covariates_genes_chr-3_peak.marker-UNC4961542_lod.drop-1.5_5.batches_mis.csv
    Untracked:  data/ici.vs.eoi_age.of.onset-no.covariates_genes_chr-3_peak.marker-UNC4961542_lod.drop-1.5_5.batches_mis_0.csv
    Untracked:  data/ici.vs.eoi_age.of.onset-no.covariates_genes_chr-3_peak.marker-UNC4979914_lod.drop-1.5_5.batches_mis.csv
    Untracked:  data/ici.vs.eoi_age.of.onset-no.covariates_genes_chr-3_peak.marker-UNC4979914_lod.drop-1.5_5.batches_mis_0.csv
    Untracked:  data/ici.vs.eoi_age.of.onset-no.covariates_genes_chr-3_peak.marker-UNC4990111_lod.drop-1.5_5.batches_mis.csv
    Untracked:  data/ici.vs.eoi_age.of.onset-no.covariates_genes_chr-3_peak.marker-UNC4990111_lod.drop-1.5_5.batches_mis_0.csv
    Untracked:  data/ici.vs.eoi_age.of.onset-no.covariates_genes_chr-3_peak.marker-UNC5008656_lod.drop-1.5_5.batches_0.csv
    Untracked:  data/ici.vs.eoi_age.of.onset-no.covariates_genes_chr-3_peak.marker-UNC5008656_lod.drop-1.5_5.batches_52.csv
    Untracked:  data/ici.vs.eoi_age.of.onset-no.covariates_genes_chr-3_peak.marker-UNC5008656_lod.drop-1.5_5.batches_mis.csv
    Untracked:  data/ici.vs.eoi_age.of.onset-no.covariates_genes_chr-3_peak.marker-UNC5008656_lod.drop-1.5_5.batches_mis_0.csv
    Untracked:  data/ici.vs.eoi_age.of.onset-no.covariates_genes_chr-3_peak.marker-UNC5046545_lod.drop-1.5_5.batches_mis_0.csv
    Untracked:  data/ici.vs.eoi_age.of.onset-no.covariates_genes_chr-3_peak.marker-UNC5055259_lod.drop-1.5_5.batches_mis_0.csv
    Untracked:  data/ici.vs.eoi_age.of.onset-no.covariates_genes_chr-3_peak.marker-UNC5062524_lod.drop-1.5_5.batches_mis_0.csv
    Untracked:  data/ici.vs.eoi_age.of.onset-no.covariates_genes_chr-3_peak.marker-UNC5068215_lod.drop-1.5_5.batches_mis_0.csv
    Untracked:  data/ici.vs.eoi_age.of.onset-no.covariates_genes_chr-3_peak.marker-UNC5076935_lod.drop-1.5_5.batches_mis_0.csv
    Untracked:  data/ici.vs.eoi_age.of.onset-no.covariates_genes_chr-3_peak.marker-UNC5082757_lod.drop-1.5_5.batches_mis_0.csv
    Untracked:  data/ici.vs.eoi_age.of.onset-no.covariates_genes_chr-3_peak.marker-UNC5083168_lod.drop-1.5_5.batches_0.csv
    Untracked:  data/ici.vs.eoi_age.of.onset-no.covariates_genes_chr-3_peak.marker-UNC5083168_lod.drop-1.5_5.batches_52.csv
    Untracked:  data/ici.vs.eoi_age.of.onset-no.covariates_genes_chr-3_peak.marker-UNC5083168_lod.drop-1.5_5.batches_mis_0.csv
    Untracked:  data/ici.vs.eoi_age.of.onset-no.covariates_genes_chr-3_peak.marker-UNC5173293_lod.drop-1.5_5.batches_mis_0.csv
    Untracked:  data/ici.vs.eoi_age.of.onset-no.covariates_genes_chr-3_peak.marker-UNC5192582_lod.drop-1.5_5.batches_0.csv
    Untracked:  data/ici.vs.eoi_age.of.onset-no.covariates_genes_chr-3_peak.marker-UNC5192582_lod.drop-1.5_5.batches_52.csv
    Untracked:  data/ici.vs.eoi_age.of.onset-no.covariates_genes_chr-3_peak.marker-UNC5192582_lod.drop-1.5_5.batches_mis_0.csv
    Untracked:  data/ici.vs.eoi_age.of.onset-no.covariates_genes_chr-3_peak.marker-UNC5193970_lod.drop-1.5_5.batches_mis_0.csv
    Untracked:  data/ici.vs.eoi_age.of.onset-no.covariates_genes_chr-3_peak.marker-UNC5213888_lod.drop-1.5_snpsqc_5.batches_mis_0.csv
    Untracked:  data/ici.vs.eoi_age.of.onset-no.covariates_genes_chr-3_peak.marker-UNC5213888_lod.drop-1.5_snpsqc_5.batches_mis_52.csv
    Untracked:  data/ici.vs.eoi_age.of.onset-no.covariates_genes_chr-3_peak.marker-UNC5257268_lod.drop-1.5_5.batches_mis_0.csv
    Untracked:  data/ici.vs.eoi_age.of.onset-no.covariates_genes_chr-3_peak.marker-UNC5262472_lod.drop-1.5_5.batches_0.csv
    Untracked:  data/ici.vs.eoi_age.of.onset-no.covariates_genes_chr-3_peak.marker-UNC5262472_lod.drop-1.5_5.batches_52.csv
    Untracked:  data/ici.vs.eoi_age.of.onset-no.covariates_genes_chr-3_peak.marker-UNC5262472_lod.drop-1.5_5.batches_mis_0.csv
    Untracked:  data/ici.vs.eoi_age.of.onset-no.covariates_genes_chr-3_peak.marker-UNC5264696_lod.drop-1.5_5.batches_mis_0.csv
    Untracked:  data/ici.vs.eoi_age.of.onset-no.covariates_genes_chr-3_peak.marker-UNC5267813_lod.drop-1.5_5.batches_mis_0.csv
    Untracked:  data/ici.vs.eoi_age.of.onset-no.covariates_genes_chr-3_peak.marker-UNC5270225_lod.drop-1.5_5.batches_mis_0.csv
    Untracked:  data/ici.vs.eoi_age.of.onset-no.covariates_genes_chr-3_peak.marker-UNC5277061_lod.drop-1.5_5.batches_mis_0.csv
    Untracked:  data/ici.vs.eoi_age.of.onset-no.covariates_genes_chr-3_peak.marker-UNC5278229_lod.drop-1.5_5.batches_0.csv
    Untracked:  data/ici.vs.eoi_age.of.onset-no.covariates_genes_chr-3_peak.marker-UNC5278229_lod.drop-1.5_5.batches_mis_0.csv
    Untracked:  data/ici.vs.eoi_age.of.onset-no.covariates_genes_chr-3_peak.marker-UNC5282259_lod.drop-1.5_5.batches_mis_0.csv
    Untracked:  data/ici.vs.eoi_age.of.onset-no.covariates_genes_chr-3_peak.marker-UNC5287977_lod.drop-1.5_5.batches_mis_0.csv
    Untracked:  data/ici.vs.eoi_age.of.onset-no.covariates_genes_chr-3_peak.marker-UNC5301460_lod.drop-1.5_5.batches_mis_0.csv
    Untracked:  data/ici.vs.eoi_age.of.onset-no.covariates_genes_chr-3_peak.marker-UNC5309136_lod.drop-1.5_5.batches_mis_0.csv
    Untracked:  data/ici.vs.eoi_age.of.onset-no.covariates_genes_chr-3_peak.marker-UNC5321559_lod.drop-1.5_5.batches_mis_0.csv
    Untracked:  data/ici.vs.eoi_age.of.onset-no.covariates_genes_chr-3_peak.marker-UNC5322455_lod.drop-1.5_5.batches_mis_0.csv
    Untracked:  data/ici.vs.eoi_age.of.onset-no.covariates_genes_chr-3_peak.marker-UNC5329171_lod.drop-1.5_5.batches_mis_0.csv
    Untracked:  data/ici.vs.eoi_age.of.onset-no.covariates_genes_chr-3_peak.marker-UNC5360055_lod.drop-1.5_5.batches_52.csv
    Untracked:  data/ici.vs.eoi_age.of.onset-no.covariates_genes_chr-3_peak.marker-UNC5454839_lod.drop-1.5_5.batches_0.csv
    Untracked:  data/ici.vs.eoi_age.of.onset-no.covariates_genes_chr-3_peak.marker-UNC5454839_lod.drop-1.5_5.batches_52.csv
    Untracked:  data/ici.vs.eoi_age.of.onset-no.covariates_genes_chr-3_peak.marker-UNC5454839_lod.drop-1.5_5.batches_mis_0.csv
    Untracked:  data/ici.vs.eoi_age.of.onset-no.covariates_genes_chr-3_peak.marker-UNC5645844_lod.drop-1.5_5.batches_0.csv
    Untracked:  data/ici.vs.eoi_age.of.onset-no.covariates_genes_chr-3_peak.marker-UNC5645844_lod.drop-1.5_5.batches_52.csv
    Untracked:  data/ici.vs.eoi_age.of.onset-no.covariates_genes_chr-3_peak.marker-UNC5645844_lod.drop-1.5_5.batches_mis_0.csv
    Untracked:  data/ici.vs.eoi_age.of.onset-no.covariates_genes_chr-3_peak.marker-UNC5778977_lod.drop-1.5_5.batches_0.csv
    Untracked:  data/ici.vs.eoi_age.of.onset-no.covariates_genes_chr-3_peak.marker-UNC5808246_lod.drop-1.5_5.batches_52.csv
    Untracked:  data/ici.vs.eoi_age.of.onset-no.covariates_genes_chr-3_peak.marker-UNC5916997_lod.drop-1.5_5.batches_52.csv
    Untracked:  data/ici.vs.eoi_age.of.onset-no.covariates_genes_chr-3_peak.marker-UNCHS008182_lod.drop-1.5_5.batches_0.csv
    Untracked:  data/ici.vs.eoi_age.of.onset-no.covariates_genes_chr-3_peak.marker-UNCHS008182_lod.drop-1.5_5.batches_52.csv
    Untracked:  data/ici.vs.eoi_age.of.onset-no.covariates_genes_chr-3_peak.marker-UNCHS008182_lod.drop-1.5_5.batches_mis_0.csv
    Untracked:  data/ici.vs.eoi_age.of.onset-no.covariates_genes_chr-3_peak.marker-UNCHS008193_lod.drop-1.5_5.batches_mis.csv
    Untracked:  data/ici.vs.eoi_age.of.onset-no.covariates_genes_chr-3_peak.marker-UNCHS008193_lod.drop-1.5_5.batches_mis_0.csv
    Untracked:  data/ici.vs.eoi_age.of.onset-no.covariates_genes_chr-3_peak.marker-UNCHS008209_lod.drop-1.5_5.batches_mis.csv
    Untracked:  data/ici.vs.eoi_age.of.onset-no.covariates_genes_chr-3_peak.marker-UNCHS008209_lod.drop-1.5_5.batches_mis_0.csv
    Untracked:  data/ici.vs.eoi_age.of.onset-no.covariates_genes_chr-3_peak.marker-UNCHS008215_lod.drop-1.5_5.batches_mis.csv
    Untracked:  data/ici.vs.eoi_age.of.onset-no.covariates_genes_chr-3_peak.marker-UNCHS008215_lod.drop-1.5_5.batches_mis_0.csv
    Untracked:  data/ici.vs.eoi_age.of.onset-no.covariates_genes_chr-3_peak.marker-UNCHS008219_lod.drop-1.5_5.batches_mis.csv
    Untracked:  data/ici.vs.eoi_age.of.onset-no.covariates_genes_chr-3_peak.marker-UNCHS008219_lod.drop-1.5_5.batches_mis_0.csv
    Untracked:  data/ici.vs.eoi_age.of.onset-no.covariates_genes_chr-3_peak.marker-UNCHS008229_lod.drop-1.5_5.batches_mis.csv
    Untracked:  data/ici.vs.eoi_age.of.onset-no.covariates_genes_chr-3_peak.marker-UNCHS008229_lod.drop-1.5_5.batches_mis_0.csv
    Untracked:  data/ici.vs.eoi_age.of.onset-no.covariates_genes_chr-3_peak.marker-UNCHS008234_lod.drop-1.5_5.batches_mis.csv
    Untracked:  data/ici.vs.eoi_age.of.onset-no.covariates_genes_chr-3_peak.marker-UNCHS008234_lod.drop-1.5_5.batches_mis_0.csv
    Untracked:  data/ici.vs.eoi_age.of.onset-no.covariates_genes_chr-3_peak.marker-UNCHS008242_lod.drop-1.5_5.batches_52.csv
    Untracked:  data/ici.vs.eoi_age.of.onset-no.covariates_genes_chr-3_peak.marker-UNCHS008260_lod.drop-1.5_5.batches_mis.csv
    Untracked:  data/ici.vs.eoi_age.of.onset-no.covariates_genes_chr-3_peak.marker-UNCHS008260_lod.drop-1.5_5.batches_mis_0.csv
    Untracked:  data/ici.vs.eoi_age.of.onset-no.covariates_genes_chr-3_peak.marker-UNCHS008271_lod.drop-1.5_5.batches_mis.csv
    Untracked:  data/ici.vs.eoi_age.of.onset-no.covariates_genes_chr-3_peak.marker-UNCHS008271_lod.drop-1.5_5.batches_mis_0.csv
    Untracked:  data/ici.vs.eoi_age.of.onset-no.covariates_genes_chr-3_peak.marker-UNCHS008281_lod.drop-1.5_5.batches_mis.csv
    Untracked:  data/ici.vs.eoi_age.of.onset-no.covariates_genes_chr-3_peak.marker-UNCHS008281_lod.drop-1.5_5.batches_mis_0.csv
    Untracked:  data/ici.vs.eoi_age.of.onset-no.covariates_genes_chr-3_peak.marker-UNCHS008286_lod.drop-1.5_5.batches_0.csv
    Untracked:  data/ici.vs.eoi_age.of.onset-no.covariates_genes_chr-3_peak.marker-UNCHS008286_lod.drop-1.5_5.batches_52.csv
    Untracked:  data/ici.vs.eoi_age.of.onset-no.covariates_genes_chr-3_peak.marker-UNCHS008286_lod.drop-1.5_5.batches_mis_0.csv
    Untracked:  data/ici.vs.eoi_age.of.onset-no.covariates_genes_chr-3_peak.marker-UNCHS008338_lod.drop-1.5_5.batches_mis.csv
    Untracked:  data/ici.vs.eoi_age.of.onset-no.covariates_genes_chr-3_peak.marker-UNCHS008338_lod.drop-1.5_5.batches_mis_0.csv
    Untracked:  data/ici.vs.eoi_age.of.onset-no.covariates_genes_chr-3_peak.marker-UNCHS008363_lod.drop-1.5_5.batches_mis.csv
    Untracked:  data/ici.vs.eoi_age.of.onset-no.covariates_genes_chr-3_peak.marker-UNCHS008363_lod.drop-1.5_5.batches_mis_0.csv
    Untracked:  data/ici.vs.eoi_age.of.onset-no.covariates_genes_chr-3_peak.marker-UNCHS008379_lod.drop-1.5_5.batches_mis.csv
    Untracked:  data/ici.vs.eoi_age.of.onset-no.covariates_genes_chr-3_peak.marker-UNCHS008379_lod.drop-1.5_5.batches_mis_0.csv
    Untracked:  data/ici.vs.eoi_age.of.onset-no.covariates_genes_chr-3_peak.marker-UNCHS008409_lod.drop-1.5_5.batches_mis_0.csv
    Untracked:  data/ici.vs.eoi_age.of.onset-no.covariates_genes_chr-3_peak.marker-UNCHS008432_lod.drop-1.5_5.batches_mis_0.csv
    Untracked:  data/ici.vs.eoi_age.of.onset-no.covariates_genes_chr-3_peak.marker-UNCHS008487_lod.drop-1.5_snpsqc_5.batches_0.csv
    Untracked:  data/ici.vs.eoi_age.of.onset-no.covariates_genes_chr-3_peak.marker-UNCHS008487_lod.drop-1.5_snpsqc_5.batches_52.csv
    Untracked:  data/ici.vs.eoi_age.of.onset-no.covariates_genes_chr-3_peak.marker-UNCHS008487_lod.drop-1.5_snpsqc_dis_no-x_updated_5.batches_0.csv
    Untracked:  data/ici.vs.eoi_age.of.onset-no.covariates_genes_chr-3_peak.marker-UNCHS008487_lod.drop-1.5_snpsqc_dis_no-x_updated_5.batches_52.csv
    Untracked:  data/ici.vs.eoi_age.of.onset-no.covariates_genes_chr-3_peak.marker-UNCHS008487_lod.drop-1.5_snpsqc_dis_no-x_updated_5.batches_mis_0.csv
    Untracked:  data/ici.vs.eoi_age.of.onset-no.covariates_genes_chr-3_peak.marker-UNCHS008487_lod.drop-1.5_snpsqc_dis_no-x_updated_5.batches_mis_52.csv
    Untracked:  data/ici.vs.eoi_age.of.onset-no.covariates_genes_chr-3_peak.marker-UNCHS008511_lod.drop-1.5_snpsqc_5.batches_mis_0.csv
    Untracked:  data/ici.vs.eoi_age.of.onset-no.covariates_genes_chr-3_peak.marker-UNCHS008511_lod.drop-1.5_snpsqc_5.batches_mis_52.csv
    Untracked:  data/ici.vs.eoi_age.of.onset-no.covariates_genes_chr-3_peak.marker-UNCHS008609_lod.drop-1.5_5.batches_mis_0.csv
    Untracked:  data/ici.vs.eoi_age.of.onset-no.covariates_genes_chr-3_peak.marker-UNCHS008613_lod.drop-1.5_5.batches_0.csv
    Untracked:  data/ici.vs.eoi_age.of.onset-no.covariates_genes_chr-3_peak.marker-UNCHS008613_lod.drop-1.5_5.batches_52.csv
    Untracked:  data/ici.vs.eoi_age.of.onset-no.covariates_genes_chr-3_peak.marker-UNCHS008614_lod.drop-1.5_5.batches_mis_0.csv
    Untracked:  data/ici.vs.eoi_age.of.onset-no.covariates_genes_chr-3_peak.marker-UNCHS008627_lod.drop-1.5_5.batches_mis_0.csv
    Untracked:  data/ici.vs.eoi_age.of.onset-no.covariates_genes_chr-3_peak.marker-UNCHS008659_lod.drop-1.5_5.batches_52.csv
    Untracked:  data/ici.vs.eoi_age.of.onset-no.covariates_genes_chr-3_peak.marker-UNCHS008725_lod.drop-1.5_5.batches_mis_0.csv
    Untracked:  data/ici.vs.eoi_age.of.onset-no.covariates_genes_chr-3_peak.marker-UNCHS008815_lod.drop-1.5_5.batches_0.csv
    Untracked:  data/ici.vs.eoi_age.of.onset-no.covariates_genes_chr-3_peak.marker-UNCHS008815_lod.drop-1.5_5.batches_52.csv
    Untracked:  data/ici.vs.eoi_age.of.onset-no.covariates_genes_chr-3_peak.marker-UNCHS008815_lod.drop-1.5_5.batches_mis_0.csv
    Untracked:  data/ici.vs.eoi_age.of.onset-no.covariates_genes_chr-3_peak.marker-UNCHS009066_lod.drop-1.5_5.batches_52.csv
    Untracked:  data/ici.vs.eoi_age.of.onset-no.covariates_genes_chr-3_peak.marker-UNCHS009067_lod.drop-1.5_5.batches_0.csv
    Untracked:  data/ici.vs.eoi_age.of.onset-no.covariates_genes_chr-3_peak.marker-UNCHS009068_lod.drop-1.5_5.batches_mis_0.csv
    Untracked:  data/ici.vs.eoi_age.of.onset-no.covariates_genes_chr-3_peak.marker-UNCHS009112_lod.drop-1.5_5.batches_52.csv
    Untracked:  data/ici.vs.eoi_age.of.onset-no.covariates_genes_chr-3_peak.marker-UNCHS009224_lod.drop-1.5_5.batches_0.csv
    Untracked:  data/ici.vs.eoi_age.of.onset-no.covariates_genes_chr-3_peak.marker-UNCHS009224_lod.drop-1.5_5.batches_52.csv
    Untracked:  data/ici.vs.eoi_age.of.onset-no.covariates_genes_chr-3_peak.marker-UNCHS009383_lod.drop-1.5_5.batches_0.csv
    Untracked:  data/ici.vs.eoi_age.of.onset-no.covariates_genes_chr-3_peak.marker-UNCHS009383_lod.drop-1.5_5.batches_52.csv
    Untracked:  data/ici.vs.eoi_age.of.onset-no.covariates_genes_chr-3_peak.marker-UNCHS009693_lod.drop-1.5_5.batches_52.csv
    Untracked:  data/ici.vs.eoi_age.of.onset-no.covariates_genes_chr-3_peak.marker-UNCHS009821_lod.drop-1.5_5.batches_52.csv
    Untracked:  data/ici.vs.eoi_age.of.onset-no.covariates_genes_chr-3_peak.marker-UNCJPD001170_lod.drop-1.5_5.batches_mis.csv
    Untracked:  data/ici.vs.eoi_age.of.onset-no.covariates_genes_chr-3_peak.marker-UNCJPD001170_lod.drop-1.5_5.batches_mis_0.csv
    Untracked:  data/ici.vs.eoi_age.of.onset-no.covariates_genes_chr-3_peak.marker-UNCJPD001198_lod.drop-1.5_5.batches_mis.csv
    Untracked:  data/ici.vs.eoi_age.of.onset-no.covariates_genes_chr-3_peak.marker-UNCJPD001198_lod.drop-1.5_5.batches_mis_0.csv
    Untracked:  data/ici.vs.eoi_age.of.onset-no.covariates_genes_chr-3_peak.marker-UNCJPD001276_lod.drop-1.5_5.batches_mis_0.csv
    Untracked:  data/ici.vs.eoi_age.of.onset-no.covariates_genes_chr-3_peak.marker-sanger2496q_lod.drop-1.5_5.batches_mis.csv
    Untracked:  data/ici.vs.eoi_age.of.onset-no.covariates_genes_chr-3_peak.marker-sanger2496q_lod.drop-1.5_5.batches_mis_0.csv
    Untracked:  data/ici.vs.eoi_age.of.onset-no.covariates_genes_chr-4_peak.marker-UNC8250659_lod.drop-1.5_snpsqc_dis_no-x_updated_5.batches_52.csv
    Untracked:  data/ici.vs.eoi_age.of.onset-no.covariates_genes_chr-4_peak.marker-UNC8439633_lod.drop-1.5_snpsqc_5.batches_52.csv
    Untracked:  data/ici.vs.eoi_age.of.onset-no.covariates_genes_chr-4_peak.marker-UNC8439633_lod.drop-1.5_snpsqc_5.batches_mis_52.csv
    Untracked:  data/ici.vs.eoi_age.of.onset-no.covariates_genes_chr-4_peak.marker-UNCHS012955_lod.drop-1.5_snpsqc_dis_no-x_updated_5.batches_mis_52.csv
    Untracked:  data/ici.vs.eoi_age.of.onset-no.covariates_genes_chr-6_peak.marker-UNC11108920_lod.drop-1.5_snpsqc_5.batches.csv
    Untracked:  data/ici.vs.eoi_age.of.onset-no.covariates_genes_chr-6_peak.marker-UNC11108920_lod.drop-1.5_snpsqc_5.batches_mis.csv
    Untracked:  data/ici.vs.eoi_age.of.onset-no.covariates_genes_chr-7_peak.marker-UNCHS020066_lod.drop-1.5_snpsqc_5.batches_mis_0.csv
    Untracked:  data/ici.vs.eoi_age.of.onset-no.covariates_genes_chr-7_peak.marker-UNCHS020066_lod.drop-1.5_snpsqc_5.batches_mis_52.csv
    Untracked:  data/ici.vs.eoi_age.of.onset-no.covariates_genes_chr-7_peak.marker-UNCHS020066_lod.drop-1.5_snpsqc_dis_no-x_updated_5.batches_mis_0.csv
    Untracked:  data/ici.vs.eoi_age.of.onset-no.covariates_genes_chr-7_peak.marker-UNCHS020066_lod.drop-1.5_snpsqc_dis_no-x_updated_5.batches_mis_52.csv
    Untracked:  data/ici.vs.eoi_age.of.onset-no.covariates_genes_chr-8_peak.marker-UNC15524531_lod.drop-1.5_snpsqc_5.batches_mis_0.csv
    Untracked:  data/ici.vs.eoi_age.of.onset-no.covariates_genes_chr-8_peak.marker-UNC15524531_lod.drop-1.5_snpsqc_5.batches_mis_52.csv
    Untracked:  data/ici.vs.eoi_age.of.onset-no.covariates_genes_chr-8_peak.marker-UNC15524531_lod.drop-1.5_snpsqc_dis_no-x_updated_5.batches_mis_0.csv
    Untracked:  data/ici.vs.eoi_age.of.onset-no.covariates_genes_chr-8_peak.marker-UNC15524531_lod.drop-1.5_snpsqc_dis_no-x_updated_5.batches_mis_52.csv
    Untracked:  data/ici.vs.eoi_age.of.onset-no.covariates_genes_chr-9_peak.marker-UNC17203597_lod.drop-1.5_snpsqc_5.batches_mis_0.csv
    Untracked:  data/ici.vs.eoi_age.of.onset-no.covariates_genes_chr-9_peak.marker-UNC17203597_lod.drop-1.5_snpsqc_5.batches_mis_52.csv
    Untracked:  data/ici.vs.eoi_age.of.onset-no.covariates_genes_chr-X_peak.marker-UNCHS048314_lod.drop-1.5_snpsqc_5.batches_mis_0.csv
    Untracked:  data/ici.vs.eoi_age.of.onset-no.covariates_genes_chr-X_peak.marker-UNCHS048314_lod.drop-1.5_snpsqc_5.batches_mis_52.csv
    Untracked:  data/ici.vs.eoi_age.of.onset-no.covariates_genes_chr-X_peak.marker-UNCHS048314_lod.drop-1.5_snpsqc_dis_no-x_updated_5.batches_mis_0.csv
    Untracked:  data/ici.vs.eoi_age.of.onset-no.covariates_genes_chr-X_peak.marker-UNCHS048314_lod.drop-1.5_snpsqc_dis_no-x_updated_5.batches_mis_52.csv
    Untracked:  data/ici.vs.eoi_blup.full_chr-10_snpsqc_5.batches.csv
    Untracked:  data/ici.vs.eoi_blup.full_chr-10_snpsqc_5.batches_mis.csv
    Untracked:  data/ici.vs.eoi_blup.full_chr-11_snpsqc_5.batches.csv
    Untracked:  data/ici.vs.eoi_blup.full_chr-11_snpsqc_5.batches_mis.csv
    Untracked:  data/ici.vs.eoi_blup.full_chr-12_snpsqc_5.batches.csv
    Untracked:  data/ici.vs.eoi_blup.full_chr-12_snpsqc_5.batches_mis.csv
    Untracked:  data/ici.vs.eoi_blup.full_chr-13_snpsqc_5.batches.csv
    Untracked:  data/ici.vs.eoi_blup.full_chr-13_snpsqc_5.batches_mis.csv
    Untracked:  data/ici.vs.eoi_blup.full_chr-14_snpsqc_5.batches.csv
    Untracked:  data/ici.vs.eoi_blup.full_chr-14_snpsqc_5.batches_mis.csv
    Untracked:  data/ici.vs.eoi_blup.full_chr-15_snpsqc_5.batches.csv
    Untracked:  data/ici.vs.eoi_blup.full_chr-15_snpsqc_5.batches_mis.csv
    Untracked:  data/ici.vs.eoi_blup.full_chr-16_snpsqc_5.batches.csv
    Untracked:  data/ici.vs.eoi_blup.full_chr-16_snpsqc_5.batches_mis.csv
    Untracked:  data/ici.vs.eoi_blup.full_chr-17_snpsqc_5.batches.csv
    Untracked:  data/ici.vs.eoi_blup.full_chr-17_snpsqc_5.batches_mis.csv
    Untracked:  data/ici.vs.eoi_blup.full_chr-18_snpsqc_5.batches.csv
    Untracked:  data/ici.vs.eoi_blup.full_chr-18_snpsqc_5.batches_mis.csv
    Untracked:  data/ici.vs.eoi_blup.full_chr-19_snpsqc_5.batches.csv
    Untracked:  data/ici.vs.eoi_blup.full_chr-19_snpsqc_5.batches_mis.csv
    Untracked:  data/ici.vs.eoi_blup.full_chr-1_snpsqc_5.batches.csv
    Untracked:  data/ici.vs.eoi_blup.full_chr-1_snpsqc_5.batches_mis.csv
    Untracked:  data/ici.vs.eoi_blup.full_chr-2_snpsqc_5.batches.csv
    Untracked:  data/ici.vs.eoi_blup.full_chr-2_snpsqc_5.batches_mis.csv
    Untracked:  data/ici.vs.eoi_blup.full_chr-3_snpsqc_5.batches.csv
    Untracked:  data/ici.vs.eoi_blup.full_chr-3_snpsqc_5.batches_mis.csv
    Untracked:  data/ici.vs.eoi_blup.full_chr-4_snpsqc_5.batches.csv
    Untracked:  data/ici.vs.eoi_blup.full_chr-4_snpsqc_5.batches_mis.csv
    Untracked:  data/ici.vs.eoi_blup.full_chr-5_snpsqc_5.batches.csv
    Untracked:  data/ici.vs.eoi_blup.full_chr-5_snpsqc_5.batches_mis.csv
    Untracked:  data/ici.vs.eoi_blup.full_chr-6_snpsqc_5.batches.csv
    Untracked:  data/ici.vs.eoi_blup.full_chr-6_snpsqc_5.batches_mis.csv
    Untracked:  data/ici.vs.eoi_blup.full_chr-7_snpsqc_5.batches.csv
    Untracked:  data/ici.vs.eoi_blup.full_chr-7_snpsqc_5.batches_mis.csv
    Untracked:  data/ici.vs.eoi_blup.full_chr-8_snpsqc_5.batches.csv
    Untracked:  data/ici.vs.eoi_blup.full_chr-8_snpsqc_5.batches_mis.csv
    Untracked:  data/ici.vs.eoi_blup.full_chr-9_snpsqc_5.batches.csv
    Untracked:  data/ici.vs.eoi_blup.full_chr-9_snpsqc_5.batches_mis.csv
    Untracked:  data/ici.vs.eoi_blup.full_chr-X_snpsqc_5.batches.csv
    Untracked:  data/ici.vs.eoi_blup.full_chr-X_snpsqc_5.batches_mis.csv
    Untracked:  data/ici.vs.eoi_blup.qc_chr-10_5.batches.csv
    Untracked:  data/ici.vs.eoi_blup.qc_chr-10_5.batches_mis.csv
    Untracked:  data/ici.vs.eoi_blup.qc_chr-10_lod.drop-1.5_snpsqc_5.batches.csv
    Untracked:  data/ici.vs.eoi_blup.qc_chr-10_snpsqc_5.batches.csv
    Untracked:  data/ici.vs.eoi_blup.qc_chr-10_snpsqc_5.batches_mis.csv
    Untracked:  data/ici.vs.eoi_blup.qc_chr-10_snpsqc_dis_no-x_updated_5.batches.csv
    Untracked:  data/ici.vs.eoi_blup.qc_chr-10_snpsqc_dis_no-x_updated_5.batches_mis.csv
    Untracked:  data/ici.vs.eoi_blup.qc_chr-11_5.batches.csv
    Untracked:  data/ici.vs.eoi_blup.qc_chr-11_5.batches_mis.csv
    Untracked:  data/ici.vs.eoi_blup.qc_chr-11_lod.drop-1.5_snpsqc_5.batches.csv
    Untracked:  data/ici.vs.eoi_blup.qc_chr-11_snpsqc_5.batches.csv
    Untracked:  data/ici.vs.eoi_blup.qc_chr-11_snpsqc_5.batches_mis.csv
    Untracked:  data/ici.vs.eoi_blup.qc_chr-11_snpsqc_dis_no-x_updated_5.batches.csv
    Untracked:  data/ici.vs.eoi_blup.qc_chr-11_snpsqc_dis_no-x_updated_5.batches_mis.csv
    Untracked:  data/ici.vs.eoi_blup.qc_chr-12_5.batches.csv
    Untracked:  data/ici.vs.eoi_blup.qc_chr-12_5.batches_mis.csv
    Untracked:  data/ici.vs.eoi_blup.qc_chr-12_lod.drop-1.5_snpsqc_5.batches.csv
    Untracked:  data/ici.vs.eoi_blup.qc_chr-12_snpsqc_5.batches.csv
    Untracked:  data/ici.vs.eoi_blup.qc_chr-12_snpsqc_5.batches_mis.csv
    Untracked:  data/ici.vs.eoi_blup.qc_chr-12_snpsqc_dis_no-x_updated_5.batches.csv
    Untracked:  data/ici.vs.eoi_blup.qc_chr-12_snpsqc_dis_no-x_updated_5.batches_mis.csv
    Untracked:  data/ici.vs.eoi_blup.qc_chr-13_5.batches.csv
    Untracked:  data/ici.vs.eoi_blup.qc_chr-13_5.batches_mis.csv
    Untracked:  data/ici.vs.eoi_blup.qc_chr-13_lod.drop-1.5_snpsqc_5.batches.csv
    Untracked:  data/ici.vs.eoi_blup.qc_chr-13_snpsqc_5.batches.csv
    Untracked:  data/ici.vs.eoi_blup.qc_chr-13_snpsqc_5.batches_mis.csv
    Untracked:  data/ici.vs.eoi_blup.qc_chr-13_snpsqc_dis_no-x_updated_5.batches.csv
    Untracked:  data/ici.vs.eoi_blup.qc_chr-13_snpsqc_dis_no-x_updated_5.batches_mis.csv
    Untracked:  data/ici.vs.eoi_blup.qc_chr-14_5.batches.csv
    Untracked:  data/ici.vs.eoi_blup.qc_chr-14_5.batches_mis.csv
    Untracked:  data/ici.vs.eoi_blup.qc_chr-14_snpsqc_5.batches.csv
    Untracked:  data/ici.vs.eoi_blup.qc_chr-14_snpsqc_5.batches_mis.csv
    Untracked:  data/ici.vs.eoi_blup.qc_chr-14_snpsqc_dis_no-x_updated_5.batches.csv
    Untracked:  data/ici.vs.eoi_blup.qc_chr-14_snpsqc_dis_no-x_updated_5.batches_mis.csv
    Untracked:  data/ici.vs.eoi_blup.qc_chr-15_5.batches.csv
    Untracked:  data/ici.vs.eoi_blup.qc_chr-15_5.batches_mis.csv
    Untracked:  data/ici.vs.eoi_blup.qc_chr-15_snpsqc_5.batches.csv
    Untracked:  data/ici.vs.eoi_blup.qc_chr-15_snpsqc_5.batches_mis.csv
    Untracked:  data/ici.vs.eoi_blup.qc_chr-15_snpsqc_dis_no-x_updated_5.batches_mis.csv
    Untracked:  data/ici.vs.eoi_blup.qc_chr-16_5.batches.csv
    Untracked:  data/ici.vs.eoi_blup.qc_chr-16_5.batches_mis.csv
    Untracked:  data/ici.vs.eoi_blup.qc_chr-16_snpsqc_5.batches.csv
    Untracked:  data/ici.vs.eoi_blup.qc_chr-16_snpsqc_5.batches_mis.csv
    Untracked:  data/ici.vs.eoi_blup.qc_chr-16_snpsqc_dis_no-x_updated_5.batches.csv
    Untracked:  data/ici.vs.eoi_blup.qc_chr-16_snpsqc_dis_no-x_updated_5.batches_mis.csv
    Untracked:  data/ici.vs.eoi_blup.qc_chr-17_5.batches.csv
    Untracked:  data/ici.vs.eoi_blup.qc_chr-17_5.batches_mis.csv
    Untracked:  data/ici.vs.eoi_blup.qc_chr-17_snpsqc_5.batches.csv
    Untracked:  data/ici.vs.eoi_blup.qc_chr-17_snpsqc_5.batches_mis.csv
    Untracked:  data/ici.vs.eoi_blup.qc_chr-17_snpsqc_dis_no-x_updated_5.batches.csv
    Untracked:  data/ici.vs.eoi_blup.qc_chr-17_snpsqc_dis_no-x_updated_5.batches_mis.csv
    Untracked:  data/ici.vs.eoi_blup.qc_chr-18_5.batches.csv
    Untracked:  data/ici.vs.eoi_blup.qc_chr-18_5.batches_mis.csv
    Untracked:  data/ici.vs.eoi_blup.qc_chr-18_snpsqc_5.batches.csv
    Untracked:  data/ici.vs.eoi_blup.qc_chr-18_snpsqc_5.batches_mis.csv
    Untracked:  data/ici.vs.eoi_blup.qc_chr-18_snpsqc_dis_no-x_updated_5.batches.csv
    Untracked:  data/ici.vs.eoi_blup.qc_chr-18_snpsqc_dis_no-x_updated_5.batches_mis.csv
    Untracked:  data/ici.vs.eoi_blup.qc_chr-19_5.batches.csv
    Untracked:  data/ici.vs.eoi_blup.qc_chr-19_5.batches_mis.csv
    Untracked:  data/ici.vs.eoi_blup.qc_chr-19_snpsqc_5.batches.csv
    Untracked:  data/ici.vs.eoi_blup.qc_chr-19_snpsqc_5.batches_mis.csv
    Untracked:  data/ici.vs.eoi_blup.qc_chr-19_snpsqc_dis_no-x_updated_5.batches.csv
    Untracked:  data/ici.vs.eoi_blup.qc_chr-19_snpsqc_dis_no-x_updated_5.batches_mis.csv
    Untracked:  data/ici.vs.eoi_blup.qc_chr-1_5.batches.csv
    Untracked:  data/ici.vs.eoi_blup.qc_chr-1_5.batches_mis.csv
    Untracked:  data/ici.vs.eoi_blup.qc_chr-1_lod.drop-1.5_snpsqc_5.batches.csv
    Untracked:  data/ici.vs.eoi_blup.qc_chr-1_snpsqc_5.batches.csv
    Untracked:  data/ici.vs.eoi_blup.qc_chr-1_snpsqc_5.batches_mis.csv
    Untracked:  data/ici.vs.eoi_blup.qc_chr-1_snpsqc_dis_no-x_updated_5.batches.csv
    Untracked:  data/ici.vs.eoi_blup.qc_chr-1_snpsqc_dis_no-x_updated_5.batches_mis.csv
    Untracked:  data/ici.vs.eoi_blup.qc_chr-2_5.batches.csv
    Untracked:  data/ici.vs.eoi_blup.qc_chr-2_5.batches_mis.csv
    Untracked:  data/ici.vs.eoi_blup.qc_chr-2_lod.drop-1.5_snpsqc_5.batches.csv
    Untracked:  data/ici.vs.eoi_blup.qc_chr-2_snpsqc_5.batches.csv
    Untracked:  data/ici.vs.eoi_blup.qc_chr-2_snpsqc_5.batches_mis.csv
    Untracked:  data/ici.vs.eoi_blup.qc_chr-2_snpsqc_dis_no-x_updated_5.batches.csv
    Untracked:  data/ici.vs.eoi_blup.qc_chr-2_snpsqc_dis_no-x_updated_5.batches_mis.csv
    Untracked:  data/ici.vs.eoi_blup.qc_chr-3_5.batches.csv
    Untracked:  data/ici.vs.eoi_blup.qc_chr-3_5.batches_mis.csv
    Untracked:  data/ici.vs.eoi_blup.qc_chr-3_lod.drop-1.5_snpsqc_5.batches.csv
    Untracked:  data/ici.vs.eoi_blup.qc_chr-3_snpsqc_5.batches.csv
    Untracked:  data/ici.vs.eoi_blup.qc_chr-3_snpsqc_5.batches_mis.csv
    Untracked:  data/ici.vs.eoi_blup.qc_chr-3_snpsqc_dis_no-x_updated_5.batches.csv
    Untracked:  data/ici.vs.eoi_blup.qc_chr-3_snpsqc_dis_no-x_updated_5.batches_mis.csv
    Untracked:  data/ici.vs.eoi_blup.qc_chr-4_5.batches.csv
    Untracked:  data/ici.vs.eoi_blup.qc_chr-4_5.batches_mis.csv
    Untracked:  data/ici.vs.eoi_blup.qc_chr-4_lod.drop-1.5_snpsqc_5.batches.csv
    Untracked:  data/ici.vs.eoi_blup.qc_chr-4_snpsqc_5.batches.csv
    Untracked:  data/ici.vs.eoi_blup.qc_chr-4_snpsqc_5.batches_mis.csv
    Untracked:  data/ici.vs.eoi_blup.qc_chr-4_snpsqc_dis_no-x_updated_5.batches.csv
    Untracked:  data/ici.vs.eoi_blup.qc_chr-4_snpsqc_dis_no-x_updated_5.batches_mis.csv
    Untracked:  data/ici.vs.eoi_blup.qc_chr-5_5.batches.csv
    Untracked:  data/ici.vs.eoi_blup.qc_chr-5_5.batches_mis.csv
    Untracked:  data/ici.vs.eoi_blup.qc_chr-5_lod.drop-1.5_snpsqc_5.batches.csv
    Untracked:  data/ici.vs.eoi_blup.qc_chr-5_snpsqc_5.batches.csv
    Untracked:  data/ici.vs.eoi_blup.qc_chr-5_snpsqc_5.batches_mis.csv
    Untracked:  data/ici.vs.eoi_blup.qc_chr-5_snpsqc_dis_no-x_updated_5.batches.csv
    Untracked:  data/ici.vs.eoi_blup.qc_chr-5_snpsqc_dis_no-x_updated_5.batches_mis.csv
    Untracked:  data/ici.vs.eoi_blup.qc_chr-6_5.batches.csv
    Untracked:  data/ici.vs.eoi_blup.qc_chr-6_5.batches_mis.csv
    Untracked:  data/ici.vs.eoi_blup.qc_chr-6_lod.drop-1.5_snpsqc_5.batches.csv
    Untracked:  data/ici.vs.eoi_blup.qc_chr-6_snpsqc_5.batches.csv
    Untracked:  data/ici.vs.eoi_blup.qc_chr-6_snpsqc_5.batches_mis.csv
    Untracked:  data/ici.vs.eoi_blup.qc_chr-6_snpsqc_dis_no-x_updated_5.batches.csv
    Untracked:  data/ici.vs.eoi_blup.qc_chr-6_snpsqc_dis_no-x_updated_5.batches_mis.csv
    Untracked:  data/ici.vs.eoi_blup.qc_chr-7_5.batches.csv
    Untracked:  data/ici.vs.eoi_blup.qc_chr-7_5.batches_mis.csv
    Untracked:  data/ici.vs.eoi_blup.qc_chr-7_lod.drop-1.5_snpsqc_5.batches.csv
    Untracked:  data/ici.vs.eoi_blup.qc_chr-7_snpsqc_5.batches.csv
    Untracked:  data/ici.vs.eoi_blup.qc_chr-7_snpsqc_5.batches_mis.csv
    Untracked:  data/ici.vs.eoi_blup.qc_chr-7_snpsqc_dis_no-x_updated_5.batches.csv
    Untracked:  data/ici.vs.eoi_blup.qc_chr-7_snpsqc_dis_no-x_updated_5.batches_mis.csv
    Untracked:  data/ici.vs.eoi_blup.qc_chr-8_5.batches.csv
    Untracked:  data/ici.vs.eoi_blup.qc_chr-8_5.batches_mis.csv
    Untracked:  data/ici.vs.eoi_blup.qc_chr-8_lod.drop-1.5_snpsqc_5.batches.csv
    Untracked:  data/ici.vs.eoi_blup.qc_chr-8_snpsqc_5.batches.csv
    Untracked:  data/ici.vs.eoi_blup.qc_chr-8_snpsqc_5.batches_mis.csv
    Untracked:  data/ici.vs.eoi_blup.qc_chr-8_snpsqc_dis_no-x_updated_5.batches.csv
    Untracked:  data/ici.vs.eoi_blup.qc_chr-8_snpsqc_dis_no-x_updated_5.batches_mis.csv
    Untracked:  data/ici.vs.eoi_blup.qc_chr-9_5.batches.csv
    Untracked:  data/ici.vs.eoi_blup.qc_chr-9_5.batches_mis.csv
    Untracked:  data/ici.vs.eoi_blup.qc_chr-9_lod.drop-1.5_snpsqc_5.batches.csv
    Untracked:  data/ici.vs.eoi_blup.qc_chr-9_snpsqc_5.batches.csv
    Untracked:  data/ici.vs.eoi_blup.qc_chr-9_snpsqc_5.batches_mis.csv
    Untracked:  data/ici.vs.eoi_blup.qc_chr-9_snpsqc_dis_no-x_updated_5.batches.csv
    Untracked:  data/ici.vs.eoi_blup.qc_chr-9_snpsqc_dis_no-x_updated_5.batches_mis.csv
    Untracked:  data/ici.vs.eoi_blup.qc_chr-X_5.batches.csv
    Untracked:  data/ici.vs.eoi_blup.qc_chr-X_5.batches_mis.csv
    Untracked:  data/ici.vs.eoi_blup.qc_chr-X_snpsqc_5.batches.csv
    Untracked:  data/ici.vs.eoi_blup.qc_chr-X_snpsqc_5.batches_mis.csv
    Untracked:  data/ici.vs.eoi_blup.qc_chr-X_snpsqc_dis_no-x_updated_5.batches.csv
    Untracked:  data/ici.vs.eoi_blup.qc_chr-X_snpsqc_dis_no-x_updated_5.batches_mis.csv
    Untracked:  data/ici.vs.eoi_blup_sub_chr-10_peak.marker-ICR4295_lod.drop-1.5_5.batches_mis.csv
    Untracked:  data/ici.vs.eoi_blup_sub_chr-10_peak.marker-JAX00292499_lod.drop-1.5_5.batches.csv
    Untracked:  data/ici.vs.eoi_blup_sub_chr-10_peak.marker-JAX00292927_lod.drop-1.5_5.batches.csv
    Untracked:  data/ici.vs.eoi_blup_sub_chr-10_peak.marker-JAX00292927_lod.drop-1.5_snpsqc_5.batches.csv
    Untracked:  data/ici.vs.eoi_blup_sub_chr-10_peak.marker-JAX00292927_lod.drop-1.5_snpsqc_5.batches_mis.csv
    Untracked:  data/ici.vs.eoi_blup_sub_chr-10_peak.marker-UNC18154330_lod.drop-1.5_5.batches.csv
    Untracked:  data/ici.vs.eoi_blup_sub_chr-10_peak.marker-UNC18195765_lod.drop-1.5_5.batches.csv
    Untracked:  data/ici.vs.eoi_blup_sub_chr-10_peak.marker-UNC18216614_lod.drop-1.5_5.batches.csv
    Untracked:  data/ici.vs.eoi_blup_sub_chr-10_peak.marker-UNC18240977_lod.drop-1.5_snpsqc_dis_no-x_updated_5.batches.csv
    Untracked:  data/ici.vs.eoi_blup_sub_chr-10_peak.marker-UNC18311938_lod.drop-1.5_5.batches.csv
    Untracked:  data/ici.vs.eoi_blup_sub_chr-10_peak.marker-UNC18856953_lod.drop-1.5_5.batches_mis.csv
    Untracked:  data/ici.vs.eoi_blup_sub_chr-10_peak.marker-UNCHS028236_lod.drop-1.5_snpsqc_5.batches_mis.csv
    Untracked:  data/ici.vs.eoi_blup_sub_chr-10_peak.marker-UNCHS028536_lod.drop-1.5_snpsqc_dis_no-x_updated_5.batches_mis.csv
    Untracked:  data/ici.vs.eoi_blup_sub_chr-10_peak.marker-UNCJPD004316_lod.drop-1.5_5.batches_mis.csv
    Untracked:  data/ici.vs.eoi_blup_sub_chr-10_peak.marker-UNCJPD009532_lod.drop-1.5_5.batches_mis.csv
    Untracked:  data/ici.vs.eoi_blup_sub_chr-11_peak.marker-UNC19155926_lod.drop-1.5_5.batches_mis.csv
    Untracked:  data/ici.vs.eoi_blup_sub_chr-11_peak.marker-UNC19481366_lod.drop-1.5_snpsqc_5.batches_mis.csv
    Untracked:  data/ici.vs.eoi_blup_sub_chr-11_peak.marker-UNC19970181_lod.drop-1.5_5.batches_mis.csv
    Untracked:  data/ici.vs.eoi_blup_sub_chr-11_peak.marker-UNC19970181_lod.drop-1.5_snpsqc_5.batches_mis.csv
    Untracked:  data/ici.vs.eoi_blup_sub_chr-11_peak.marker-UNCJPD004792_lod.drop-1.5_5.batches_mis.csv
    Untracked:  data/ici.vs.eoi_blup_sub_chr-12_peak.marker-ICR4497_lod.drop-1.5_5.batches_mis.csv
    Untracked:  data/ici.vs.eoi_blup_sub_chr-12_peak.marker-ICR499_lod.drop-1.5_snpsqc_5.batches_mis.csv
    Untracked:  data/ici.vs.eoi_blup_sub_chr-13_peak.marker-UNCHS036773_lod.drop-1.5_snpsqc_5.batches_mis.csv
    Untracked:  data/ici.vs.eoi_blup_sub_chr-13_peak.marker-UNCHS036964_lod.drop-1.5_5.batches_mis.csv
    Untracked:  data/ici.vs.eoi_blup_sub_chr-14_peak.marker-UNC24056202_lod.drop-1.5_5.batches_mis.csv
    Untracked:  data/ici.vs.eoi_blup_sub_chr-14_peak.marker-UNC24056202_lod.drop-1.5_snpsqc_5.batches_mis.csv
    Untracked:  data/ici.vs.eoi_blup_sub_chr-15_peak.marker-UNC26070435_lod.drop-1.5_snpsqc_dis_no-x_updated_5.batches_mis.csv
    Untracked:  data/ici.vs.eoi_blup_sub_chr-17_peak.marker-UNCHS044241_lod.drop-1.5_5.batches_mis.csv
    Untracked:  data/ici.vs.eoi_blup_sub_chr-17_peak.marker-UNCHS044241_lod.drop-1.5_snpsqc_5.batches_mis.csv
    Untracked:  data/ici.vs.eoi_blup_sub_chr-17_peak.marker-UNCrs47191360_lod.drop-1.5_5.batches_mis.csv
    Untracked:  data/ici.vs.eoi_blup_sub_chr-17_peak.marker-UNCrs47191360_lod.drop-1.5_snpsqc_dis_no-x_updated_5.batches_mis.csv
    Untracked:  data/ici.vs.eoi_blup_sub_chr-18_peak.marker-UNCHS045343_lod.drop-1.5_snpsqc_5.batches_mis.csv
    Untracked:  data/ici.vs.eoi_blup_sub_chr-18_peak.marker-UNCHS045344_lod.drop-1.5_5.batches_mis.csv
    Untracked:  data/ici.vs.eoi_blup_sub_chr-19_peak.marker-UNCJPD007087_lod.drop-1.5_5.batches_mis.csv
    Untracked:  data/ici.vs.eoi_blup_sub_chr-1_peak.marker-ICR010_lod.drop-1.5_5.batches.csv
    Untracked:  data/ici.vs.eoi_blup_sub_chr-1_peak.marker-ICR010_lod.drop-1.5_5.batches_mis.csv
    Untracked:  data/ici.vs.eoi_blup_sub_chr-2_peak.marker-UNC4609527_lod.drop-1.5_snpsqc_dis_no-x_updated_5.batches_mis.csv
    Untracked:  data/ici.vs.eoi_blup_sub_chr-2_peak.marker-UNC4609660_lod.drop-1.5_5.batches_mis.csv
    Untracked:  data/ici.vs.eoi_blup_sub_chr-3_peak.marker-B6_rs31740628_lod.drop-1.5_5.batches_mis.csv
    Untracked:  data/ici.vs.eoi_blup_sub_chr-3_peak.marker-ICR1269_lod.drop-1.5_5.batches_mis.csv
    Untracked:  data/ici.vs.eoi_blup_sub_chr-3_peak.marker-ICR1338_lod.drop-1.5_5.batches.csv
    Untracked:  data/ici.vs.eoi_blup_sub_chr-3_peak.marker-ICR5325_lod.drop-1.5_5.batches_mis.csv
    Untracked:  data/ici.vs.eoi_blup_sub_chr-3_peak.marker-ICR5347_lod.drop-1.5_5.batches.csv
    Untracked:  data/ici.vs.eoi_blup_sub_chr-3_peak.marker-ICR5348_lod.drop-1.5_5.batches_mis.csv
    Untracked:  data/ici.vs.eoi_blup_sub_chr-3_peak.marker-JAX00104971_lod.drop-1.5_5.batches_mis.csv
    Untracked:  data/ici.vs.eoi_blup_sub_chr-3_peak.marker-JAX00105059_lod.drop-1.5_5.batches.csv
    Untracked:  data/ici.vs.eoi_blup_sub_chr-3_peak.marker-JAX00105203_lod.drop-1.5_5.batches_mis.csv
    Untracked:  data/ici.vs.eoi_blup_sub_chr-3_peak.marker-JAX00106957_lod.drop-1.5_5.batches.csv
    Untracked:  data/ici.vs.eoi_blup_sub_chr-3_peak.marker-JAX00106957_lod.drop-1.5_5.batches_mis.csv
    Untracked:  data/ici.vs.eoi_blup_sub_chr-3_peak.marker-JAX00107930_lod.drop-1.5_5.batches_mis.csv
    Untracked:  data/ici.vs.eoi_blup_sub_chr-3_peak.marker-JAX00107980_lod.drop-1.5_5.batches_mis.csv
    Untracked:  data/ici.vs.eoi_blup_sub_chr-3_peak.marker-JAX00108068_lod.drop-1.5_5.batches_mis.csv
    Untracked:  data/ici.vs.eoi_blup_sub_chr-3_peak.marker-JAX00108114_lod.drop-1.5_5.batches_mis.csv
    Untracked:  data/ici.vs.eoi_blup_sub_chr-3_peak.marker-JAX00109750_lod.drop-1.5_5.batches_mis.csv
    Untracked:  data/ici.vs.eoi_blup_sub_chr-3_peak.marker-JAX00189245_lod.drop-1.5_5.batches_mis.csv
    Untracked:  data/ici.vs.eoi_blup_sub_chr-3_peak.marker-JAX00514662_lod.drop-1.5_5.batches.csv
    Untracked:  data/ici.vs.eoi_blup_sub_chr-3_peak.marker-JAX00517218_lod.drop-1.5_5.batches_mis.csv
    Untracked:  data/ici.vs.eoi_blup_sub_chr-3_peak.marker-JAX00517674_lod.drop-1.5_5.batches_mis.csv
    Untracked:  data/ici.vs.eoi_blup_sub_chr-3_peak.marker-JAX00517906_lod.drop-1.5_5.batches_mis.csv
    Untracked:  data/ici.vs.eoi_blup_sub_chr-3_peak.marker-JAX00518418_lod.drop-1.5_5.batches_mis.csv
    Untracked:  data/ici.vs.eoi_blup_sub_chr-3_peak.marker-JAX00518684_lod.drop-1.5_5.batches_mis.csv
    Untracked:  data/ici.vs.eoi_blup_sub_chr-3_peak.marker-JAX00518911r_lod.drop-1.5_5.batches_mis.csv
    Untracked:  data/ici.vs.eoi_blup_sub_chr-3_peak.marker-JAX00520122_lod.drop-1.5_5.batches_mis.csv
    Untracked:  data/ici.vs.eoi_blup_sub_chr-3_peak.marker-JAX00524717_lod.drop-1.5_5.batches_mis.csv
    Untracked:  data/ici.vs.eoi_blup_sub_chr-3_peak.marker-JAX00524828_lod.drop-1.5_5.batches_mis.csv
    Untracked:  data/ici.vs.eoi_blup_sub_chr-3_peak.marker-JAX00525380_lod.drop-1.5_5.batches_mis.csv
    Untracked:  data/ici.vs.eoi_blup_sub_chr-3_peak.marker-JAX00525709_lod.drop-1.5_5.batches_mis.csv
    Untracked:  data/ici.vs.eoi_blup_sub_chr-3_peak.marker-JAX00527291_lod.drop-1.5_5.batches_mis.csv
    Untracked:  data/ici.vs.eoi_blup_sub_chr-3_peak.marker-JAX00527587_lod.drop-1.5_5.batches_mis.csv
    Untracked:  data/ici.vs.eoi_blup_sub_chr-3_peak.marker-JAX00529707_lod.drop-1.5_5.batches_mis.csv
    Untracked:  data/ici.vs.eoi_blup_sub_chr-3_peak.marker-JAX00529707r_lod.drop-1.5_5.batches.csv
    Untracked:  data/ici.vs.eoi_blup_sub_chr-3_peak.marker-JAX00531601_lod.drop-1.5_5.batches_mis.csv
    Untracked:  data/ici.vs.eoi_blup_sub_chr-3_peak.marker-JAX00532122_lod.drop-1.5_5.batches_mis.csv
    Untracked:  data/ici.vs.eoi_blup_sub_chr-3_peak.marker-UNC4673471_lod.drop-1.5_5.batches_mis.csv
    Untracked:  data/ici.vs.eoi_blup_sub_chr-3_peak.marker-UNC4681411_lod.drop-1.5_5.batches_mis.csv
    Untracked:  data/ici.vs.eoi_blup_sub_chr-3_peak.marker-UNC4793677_lod.drop-1.5_5.batches_mis.csv
    Untracked:  data/ici.vs.eoi_blup_sub_chr-3_peak.marker-UNC4799475_lod.drop-1.5_5.batches_mis.csv
    Untracked:  data/ici.vs.eoi_blup_sub_chr-3_peak.marker-UNC4809586_lod.drop-1.5_5.batches_mis.csv
    Untracked:  data/ici.vs.eoi_blup_sub_chr-3_peak.marker-UNC4813186_lod.drop-1.5_5.batches.csv
    Untracked:  data/ici.vs.eoi_blup_sub_chr-3_peak.marker-UNC4813186_lod.drop-1.5_5.batches_mis.csv
    Untracked:  data/ici.vs.eoi_blup_sub_chr-3_peak.marker-UNC4820379_lod.drop-1.5_5.batches_mis.csv
    Untracked:  data/ici.vs.eoi_blup_sub_chr-3_peak.marker-UNC4875653_lod.drop-1.5_5.batches_mis.csv
    Untracked:  data/ici.vs.eoi_blup_sub_chr-3_peak.marker-UNC4909830_lod.drop-1.5_5.batches_mis.csv
    Untracked:  data/ici.vs.eoi_blup_sub_chr-3_peak.marker-UNC4912288_lod.drop-1.5_5.batches_mis.csv
    Untracked:  data/ici.vs.eoi_blup_sub_chr-3_peak.marker-UNC4914130_lod.drop-1.5_5.batches_mis.csv
    Untracked:  data/ici.vs.eoi_blup_sub_chr-3_peak.marker-UNC4923398_lod.drop-1.5_5.batches_mis.csv
    Untracked:  data/ici.vs.eoi_blup_sub_chr-3_peak.marker-UNC4924377_lod.drop-1.5_5.batches.csv
    Untracked:  data/ici.vs.eoi_blup_sub_chr-3_peak.marker-UNC4945186_lod.drop-1.5_5.batches_mis.csv
    Untracked:  data/ici.vs.eoi_blup_sub_chr-3_peak.marker-UNC4947492_lod.drop-1.5_5.batches_mis.csv
    Untracked:  data/ici.vs.eoi_blup_sub_chr-3_peak.marker-UNC4957168_lod.drop-1.5_5.batches.csv
    Untracked:  data/ici.vs.eoi_blup_sub_chr-3_peak.marker-UNC4957859_lod.drop-1.5_5.batches_mis.csv
    Untracked:  data/ici.vs.eoi_blup_sub_chr-3_peak.marker-UNC4964062_lod.drop-1.5_5.batches_mis.csv
    Untracked:  data/ici.vs.eoi_blup_sub_chr-3_peak.marker-UNC4979914_lod.drop-1.5_5.batches_mis.csv
    Untracked:  data/ici.vs.eoi_blup_sub_chr-3_peak.marker-UNC4985064_lod.drop-1.5_5.batches_mis.csv
    Untracked:  data/ici.vs.eoi_blup_sub_chr-3_peak.marker-UNC4990111_lod.drop-1.5_5.batches_mis.csv
    Untracked:  data/ici.vs.eoi_blup_sub_chr-3_peak.marker-UNC4991565_lod.drop-1.5_5.batches_mis.csv
    Untracked:  data/ici.vs.eoi_blup_sub_chr-3_peak.marker-UNC5001723_lod.drop-1.5_5.batches_mis.csv
    Untracked:  data/ici.vs.eoi_blup_sub_chr-3_peak.marker-UNC5008656_lod.drop-1.5_5.batches.csv
    Untracked:  data/ici.vs.eoi_blup_sub_chr-3_peak.marker-UNC5008656_lod.drop-1.5_5.batches_mis.csv
    Untracked:  data/ici.vs.eoi_blup_sub_chr-3_peak.marker-UNC5042361_lod.drop-1.5_5.batches_mis.csv
    Untracked:  data/ici.vs.eoi_blup_sub_chr-3_peak.marker-UNC5055259_lod.drop-1.5_5.batches_mis.csv
    Untracked:  data/ici.vs.eoi_blup_sub_chr-3_peak.marker-UNC5076935_lod.drop-1.5_5.batches_mis.csv
    Untracked:  data/ici.vs.eoi_blup_sub_chr-3_peak.marker-UNC5082757_lod.drop-1.5_5.batches_mis.csv
    Untracked:  data/ici.vs.eoi_blup_sub_chr-3_peak.marker-UNC5083168_lod.drop-1.5_5.batches.csv
    Untracked:  data/ici.vs.eoi_blup_sub_chr-3_peak.marker-UNC5173293_lod.drop-1.5_5.batches_mis.csv
    Untracked:  data/ici.vs.eoi_blup_sub_chr-3_peak.marker-UNC5182782_lod.drop-1.5_5.batches_mis.csv
    Untracked:  data/ici.vs.eoi_blup_sub_chr-3_peak.marker-UNC5192582_lod.drop-1.5_5.batches.csv
    Untracked:  data/ici.vs.eoi_blup_sub_chr-3_peak.marker-UNC5193970_lod.drop-1.5_5.batches_mis.csv
    Untracked:  data/ici.vs.eoi_blup_sub_chr-3_peak.marker-UNC5213888_lod.drop-1.5_snpsqc_5.batches_mis.csv
    Untracked:  data/ici.vs.eoi_blup_sub_chr-3_peak.marker-UNC5240484_lod.drop-1.5_5.batches_mis.csv
    Untracked:  data/ici.vs.eoi_blup_sub_chr-3_peak.marker-UNC5256922_lod.drop-1.5_5.batches_mis.csv
    Untracked:  data/ici.vs.eoi_blup_sub_chr-3_peak.marker-UNC5266664_lod.drop-1.5_5.batches_mis.csv
    Untracked:  data/ici.vs.eoi_blup_sub_chr-3_peak.marker-UNC5266711_lod.drop-1.5_5.batches.csv
    Untracked:  data/ici.vs.eoi_blup_sub_chr-3_peak.marker-UNC5267813_lod.drop-1.5_5.batches_mis.csv
    Untracked:  data/ici.vs.eoi_blup_sub_chr-3_peak.marker-UNC5274554_lod.drop-1.5_5.batches_mis.csv
    Untracked:  data/ici.vs.eoi_blup_sub_chr-3_peak.marker-UNC5277061_lod.drop-1.5_5.batches_mis.csv
    Untracked:  data/ici.vs.eoi_blup_sub_chr-3_peak.marker-UNC5282259_lod.drop-1.5_5.batches_mis.csv
    Untracked:  data/ici.vs.eoi_blup_sub_chr-3_peak.marker-UNC5287977_lod.drop-1.5_5.batches_mis.csv
    Untracked:  data/ici.vs.eoi_blup_sub_chr-3_peak.marker-UNC5301460_lod.drop-1.5_5.batches_mis.csv
    Untracked:  data/ici.vs.eoi_blup_sub_chr-3_peak.marker-UNC5321559_lod.drop-1.5_5.batches_mis.csv
    Untracked:  data/ici.vs.eoi_blup_sub_chr-3_peak.marker-UNC5322455_lod.drop-1.5_5.batches_mis.csv
    Untracked:  data/ici.vs.eoi_blup_sub_chr-3_peak.marker-UNC5328596_lod.drop-1.5_5.batches_mis.csv
    Untracked:  data/ici.vs.eoi_blup_sub_chr-3_peak.marker-UNC5350384_lod.drop-1.5_5.batches_mis.csv
    Untracked:  data/ici.vs.eoi_blup_sub_chr-3_peak.marker-UNC5354724_lod.drop-1.5_5.batches_mis.csv
    Untracked:  data/ici.vs.eoi_blup_sub_chr-3_peak.marker-UNC5375208_lod.drop-1.5_5.batches_mis.csv
    Untracked:  data/ici.vs.eoi_blup_sub_chr-3_peak.marker-UNC5384072_lod.drop-1.5_5.batches_mis.csv
    Untracked:  data/ici.vs.eoi_blup_sub_chr-3_peak.marker-UNC5400314_lod.drop-1.5_5.batches_mis.csv
    Untracked:  data/ici.vs.eoi_blup_sub_chr-3_peak.marker-UNC5433945_lod.drop-1.5_5.batches_mis.csv
    Untracked:  data/ici.vs.eoi_blup_sub_chr-3_peak.marker-UNC5435874_lod.drop-1.5_5.batches_mis.csv
    Untracked:  data/ici.vs.eoi_blup_sub_chr-3_peak.marker-UNC5454839_lod.drop-1.5_5.batches.csv
    Untracked:  data/ici.vs.eoi_blup_sub_chr-3_peak.marker-UNC5454839_lod.drop-1.5_5.batches_mis.csv
    Untracked:  data/ici.vs.eoi_blup_sub_chr-3_peak.marker-UNC5466669_lod.drop-1.5_5.batches_mis.csv
    Untracked:  data/ici.vs.eoi_blup_sub_chr-3_peak.marker-UNC5517972_lod.drop-1.5_5.batches_mis.csv
    Untracked:  data/ici.vs.eoi_blup_sub_chr-3_peak.marker-UNC5522257_lod.drop-1.5_5.batches_mis.csv
    Untracked:  data/ici.vs.eoi_blup_sub_chr-3_peak.marker-UNC5523265_lod.drop-1.5_5.batches_mis.csv
    Untracked:  data/ici.vs.eoi_blup_sub_chr-3_peak.marker-UNC5532424_lod.drop-1.5_5.batches_mis.csv
    Untracked:  data/ici.vs.eoi_blup_sub_chr-3_peak.marker-UNC5553882_lod.drop-1.5_5.batches_mis.csv
    Untracked:  data/ici.vs.eoi_blup_sub_chr-3_peak.marker-UNC5572051_lod.drop-1.5_5.batches_mis.csv
    Untracked:  data/ici.vs.eoi_blup_sub_chr-3_peak.marker-UNC5582753_lod.drop-1.5_5.batches.csv
    Untracked:  data/ici.vs.eoi_blup_sub_chr-3_peak.marker-UNC5589260_lod.drop-1.5_5.batches_mis.csv
    Untracked:  data/ici.vs.eoi_blup_sub_chr-3_peak.marker-UNC5595555_lod.drop-1.5_5.batches_mis.csv
    Untracked:  data/ici.vs.eoi_blup_sub_chr-3_peak.marker-UNC5623239_lod.drop-1.5_5.batches_mis.csv
    Untracked:  data/ici.vs.eoi_blup_sub_chr-3_peak.marker-UNC5639830_lod.drop-1.5_5.batches_mis.csv
    Untracked:  data/ici.vs.eoi_blup_sub_chr-3_peak.marker-UNC5645844_lod.drop-1.5_5.batches.csv
    Untracked:  data/ici.vs.eoi_blup_sub_chr-3_peak.marker-UNC5645844_lod.drop-1.5_5.batches_mis.csv
    Untracked:  data/ici.vs.eoi_blup_sub_chr-3_peak.marker-UNC5680234_lod.drop-1.5_5.batches_mis.csv
    Untracked:  data/ici.vs.eoi_blup_sub_chr-3_peak.marker-UNC5682314_lod.drop-1.5_5.batches_mis.csv
    Untracked:  data/ici.vs.eoi_blup_sub_chr-3_peak.marker-UNC5725356_lod.drop-1.5_5.batches_mis.csv
    Untracked:  data/ici.vs.eoi_blup_sub_chr-3_peak.marker-UNC5727351_lod.drop-1.5_5.batches_mis.csv
    Untracked:  data/ici.vs.eoi_blup_sub_chr-3_peak.marker-UNC5734420_lod.drop-1.5_5.batches_mis.csv
    Untracked:  data/ici.vs.eoi_blup_sub_chr-3_peak.marker-UNC5743257_lod.drop-1.5_5.batches_mis.csv
    Untracked:  data/ici.vs.eoi_blup_sub_chr-3_peak.marker-UNC5751567_lod.drop-1.5_5.batches_mis.csv
    Untracked:  data/ici.vs.eoi_blup_sub_chr-3_peak.marker-UNC5752623_lod.drop-1.5_5.batches_mis.csv
    Untracked:  data/ici.vs.eoi_blup_sub_chr-3_peak.marker-UNC5776974_lod.drop-1.5_5.batches_mis.csv
    Untracked:  data/ici.vs.eoi_blup_sub_chr-3_peak.marker-UNC5778977_lod.drop-1.5_5.batches_mis.csv
    Untracked:  data/ici.vs.eoi_blup_sub_chr-3_peak.marker-UNC5788507_lod.drop-1.5_5.batches_mis.csv
    Untracked:  data/ici.vs.eoi_blup_sub_chr-3_peak.marker-UNC5792468_lod.drop-1.5_5.batches_mis.csv
    Untracked:  data/ici.vs.eoi_blup_sub_chr-3_peak.marker-UNC5794598_lod.drop-1.5_5.batches_mis.csv
    Untracked:  data/ici.vs.eoi_blup_sub_chr-3_peak.marker-UNC5795561_lod.drop-1.5_5.batches_mis.csv
    Untracked:  data/ici.vs.eoi_blup_sub_chr-3_peak.marker-UNC5802298_lod.drop-1.5_5.batches_mis.csv
    Untracked:  data/ici.vs.eoi_blup_sub_chr-3_peak.marker-UNC5806628_lod.drop-1.5_5.batches_mis.csv
    Untracked:  data/ici.vs.eoi_blup_sub_chr-3_peak.marker-UNC5808246_lod.drop-1.5_5.batches.csv
    Untracked:  data/ici.vs.eoi_blup_sub_chr-3_peak.marker-UNC5817478_lod.drop-1.5_5.batches_mis.csv
    Untracked:  data/ici.vs.eoi_blup_sub_chr-3_peak.marker-UNC5836999_lod.drop-1.5_5.batches_mis.csv
    Untracked:  data/ici.vs.eoi_blup_sub_chr-3_peak.marker-UNC5841501_lod.drop-1.5_5.batches_mis.csv
    Untracked:  data/ici.vs.eoi_blup_sub_chr-3_peak.marker-UNC5847927_lod.drop-1.5_5.batches_mis.csv
    Untracked:  data/ici.vs.eoi_blup_sub_chr-3_peak.marker-UNC5966417_lod.drop-1.5_5.batches_mis.csv
    Untracked:  data/ici.vs.eoi_blup_sub_chr-3_peak.marker-UNCHS008129_lod.drop-1.5_5.batches_mis.csv
    Untracked:  data/ici.vs.eoi_blup_sub_chr-3_peak.marker-UNCHS008173_lod.drop-1.5_5.batches_mis.csv
    Untracked:  data/ici.vs.eoi_blup_sub_chr-3_peak.marker-UNCHS008182_lod.drop-1.5_5.batches.csv
    Untracked:  data/ici.vs.eoi_blup_sub_chr-3_peak.marker-UNCHS008193_lod.drop-1.5_5.batches_mis.csv
    Untracked:  data/ici.vs.eoi_blup_sub_chr-3_peak.marker-UNCHS008209_lod.drop-1.5_5.batches_mis.csv
    Untracked:  data/ici.vs.eoi_blup_sub_chr-3_peak.marker-UNCHS008215_lod.drop-1.5_5.batches_mis.csv
    Untracked:  data/ici.vs.eoi_blup_sub_chr-3_peak.marker-UNCHS008219_lod.drop-1.5_5.batches_mis.csv
    Untracked:  data/ici.vs.eoi_blup_sub_chr-3_peak.marker-UNCHS008231_lod.drop-1.5_5.batches_mis.csv
    Untracked:  data/ici.vs.eoi_blup_sub_chr-3_peak.marker-UNCHS008235_lod.drop-1.5_5.batches_mis.csv
    Untracked:  data/ici.vs.eoi_blup_sub_chr-3_peak.marker-UNCHS008242_lod.drop-1.5_5.batches.csv
    Untracked:  data/ici.vs.eoi_blup_sub_chr-3_peak.marker-UNCHS008253_lod.drop-1.5_5.batches_mis.csv
    Untracked:  data/ici.vs.eoi_blup_sub_chr-3_peak.marker-UNCHS008256_lod.drop-1.5_5.batches_mis.csv
    Untracked:  data/ici.vs.eoi_blup_sub_chr-3_peak.marker-UNCHS008259_lod.drop-1.5_5.batches_mis.csv
    Untracked:  data/ici.vs.eoi_blup_sub_chr-3_peak.marker-UNCHS008271_lod.drop-1.5_5.batches_mis.csv
    Untracked:  data/ici.vs.eoi_blup_sub_chr-3_peak.marker-UNCHS008286_lod.drop-1.5_5.batches.csv
    Untracked:  data/ici.vs.eoi_blup_sub_chr-3_peak.marker-UNCHS008286_lod.drop-1.5_5.batches_mis.csv
    Untracked:  data/ici.vs.eoi_blup_sub_chr-3_peak.marker-UNCHS008311_lod.drop-1.5_5.batches_mis.csv
    Untracked:  data/ici.vs.eoi_blup_sub_chr-3_peak.marker-UNCHS008317_lod.drop-1.5_5.batches_mis.csv
    Untracked:  data/ici.vs.eoi_blup_sub_chr-3_peak.marker-UNCHS008372_lod.drop-1.5_5.batches_mis.csv
    Untracked:  data/ici.vs.eoi_blup_sub_chr-3_peak.marker-UNCHS008379_lod.drop-1.5_5.batches_mis.csv
    Untracked:  data/ici.vs.eoi_blup_sub_chr-3_peak.marker-UNCHS008410_lod.drop-1.5_5.batches_mis.csv
    Untracked:  data/ici.vs.eoi_blup_sub_chr-3_peak.marker-UNCHS008415_lod.drop-1.5_5.batches_mis.csv
    Untracked:  data/ici.vs.eoi_blup_sub_chr-3_peak.marker-UNCHS008431_lod.drop-1.5_5.batches_mis.csv
    Untracked:  data/ici.vs.eoi_blup_sub_chr-3_peak.marker-UNCHS008487_lod.drop-1.5_snpsqc_5.batches.csv
    Untracked:  data/ici.vs.eoi_blup_sub_chr-3_peak.marker-UNCHS008487_lod.drop-1.5_snpsqc_dis_no-x_updated_5.batches.csv
    Untracked:  data/ici.vs.eoi_blup_sub_chr-3_peak.marker-UNCHS008487_lod.drop-1.5_snpsqc_dis_no-x_updated_5.batches_mis.csv
    Untracked:  data/ici.vs.eoi_blup_sub_chr-3_peak.marker-UNCHS008511_lod.drop-1.5_snpsqc_5.batches_mis.csv
    Untracked:  data/ici.vs.eoi_blup_sub_chr-3_peak.marker-UNCHS008609_lod.drop-1.5_5.batches_mis.csv
    Untracked:  data/ici.vs.eoi_blup_sub_chr-3_peak.marker-UNCHS008614_lod.drop-1.5_5.batches.csv
    Untracked:  data/ici.vs.eoi_blup_sub_chr-3_peak.marker-UNCHS008649_lod.drop-1.5_5.batches_mis.csv
    Untracked:  data/ici.vs.eoi_blup_sub_chr-3_peak.marker-UNCHS008651_lod.drop-1.5_5.batches.csv
    Untracked:  data/ici.vs.eoi_blup_sub_chr-3_peak.marker-UNCHS008656_lod.drop-1.5_5.batches_mis.csv
    Untracked:  data/ici.vs.eoi_blup_sub_chr-3_peak.marker-UNCHS008723_lod.drop-1.5_5.batches_mis.csv
    Untracked:  data/ici.vs.eoi_blup_sub_chr-3_peak.marker-UNCHS008802_lod.drop-1.5_5.batches_mis.csv
    Untracked:  data/ici.vs.eoi_blup_sub_chr-3_peak.marker-UNCHS008803_lod.drop-1.5_5.batches_mis.csv
    Untracked:  data/ici.vs.eoi_blup_sub_chr-3_peak.marker-UNCHS008808_lod.drop-1.5_5.batches_mis.csv
    Untracked:  data/ici.vs.eoi_blup_sub_chr-3_peak.marker-UNCHS008815_lod.drop-1.5_5.batches.csv
    Untracked:  data/ici.vs.eoi_blup_sub_chr-3_peak.marker-UNCHS008815_lod.drop-1.5_5.batches_mis.csv
    Untracked:  data/ici.vs.eoi_blup_sub_chr-3_peak.marker-UNCHS008834_lod.drop-1.5_5.batches_mis.csv
    Untracked:  data/ici.vs.eoi_blup_sub_chr-3_peak.marker-UNCHS008838_lod.drop-1.5_5.batches_mis.csv
    Untracked:  data/ici.vs.eoi_blup_sub_chr-3_peak.marker-UNCHS008914_lod.drop-1.5_5.batches_mis.csv
    Untracked:  data/ici.vs.eoi_blup_sub_chr-3_peak.marker-UNCHS008928_lod.drop-1.5_5.batches_mis.csv
    Untracked:  data/ici.vs.eoi_blup_sub_chr-3_peak.marker-UNCHS008948_lod.drop-1.5_5.batches_mis.csv
    Untracked:  data/ici.vs.eoi_blup_sub_chr-3_peak.marker-UNCHS008959_lod.drop-1.5_5.batches_mis.csv
    Untracked:  data/ici.vs.eoi_blup_sub_chr-3_peak.marker-UNCHS008973_lod.drop-1.5_5.batches_mis.csv
    Untracked:  data/ici.vs.eoi_blup_sub_chr-3_peak.marker-UNCHS008987_lod.drop-1.5_5.batches_mis.csv
    Untracked:  data/ici.vs.eoi_blup_sub_chr-3_peak.marker-UNCHS009001_lod.drop-1.5_5.batches_mis.csv
    Untracked:  data/ici.vs.eoi_blup_sub_chr-3_peak.marker-UNCHS009025_lod.drop-1.5_5.batches_mis.csv
    Untracked:  data/ici.vs.eoi_blup_sub_chr-3_peak.marker-UNCHS009034_lod.drop-1.5_5.batches_mis.csv
    Untracked:  data/ici.vs.eoi_blup_sub_chr-3_peak.marker-UNCHS009087_lod.drop-1.5_5.batches_mis.csv
    Untracked:  data/ici.vs.eoi_blup_sub_chr-3_peak.marker-UNCHS009109_lod.drop-1.5_5.batches_mis.csv
    Untracked:  data/ici.vs.eoi_blup_sub_chr-3_peak.marker-UNCHS009142_lod.drop-1.5_5.batches_mis.csv
    Untracked:  data/ici.vs.eoi_blup_sub_chr-3_peak.marker-UNCHS009207_lod.drop-1.5_5.batches_mis.csv
    Untracked:  data/ici.vs.eoi_blup_sub_chr-3_peak.marker-UNCHS009216_lod.drop-1.5_5.batches_mis.csv
    Untracked:  data/ici.vs.eoi_blup_sub_chr-3_peak.marker-UNCHS009224_lod.drop-1.5_5.batches.csv
    Untracked:  data/ici.vs.eoi_blup_sub_chr-3_peak.marker-UNCHS009306_lod.drop-1.5_5.batches_mis.csv
    Untracked:  data/ici.vs.eoi_blup_sub_chr-3_peak.marker-UNCHS009316_lod.drop-1.5_5.batches_mis.csv
    Untracked:  data/ici.vs.eoi_blup_sub_chr-3_peak.marker-UNCHS009364_lod.drop-1.5_5.batches_mis.csv
    Untracked:  data/ici.vs.eoi_blup_sub_chr-3_peak.marker-UNCHS009377_lod.drop-1.5_5.batches_mis.csv
    Untracked:  data/ici.vs.eoi_blup_sub_chr-3_peak.marker-UNCHS009382_lod.drop-1.5_5.batches_mis.csv
    Untracked:  data/ici.vs.eoi_blup_sub_chr-3_peak.marker-UNCHS009383_lod.drop-1.5_5.batches.csv
    Untracked:  data/ici.vs.eoi_blup_sub_chr-3_peak.marker-UNCHS009387_lod.drop-1.5_5.batches_mis.csv
    Untracked:  data/ici.vs.eoi_blup_sub_chr-3_peak.marker-UNCHS009395_lod.drop-1.5_5.batches_mis.csv
    Untracked:  data/ici.vs.eoi_blup_sub_chr-3_peak.marker-UNCHS009413_lod.drop-1.5_5.batches_mis.csv
    Untracked:  data/ici.vs.eoi_blup_sub_chr-3_peak.marker-UNCHS009428_lod.drop-1.5_5.batches_mis.csv
    Untracked:  data/ici.vs.eoi_blup_sub_chr-3_peak.marker-UNCHS009463_lod.drop-1.5_5.batches_mis.csv
    Untracked:  data/ici.vs.eoi_blup_sub_chr-3_peak.marker-UNCHS009503_lod.drop-1.5_5.batches_mis.csv
    Untracked:  data/ici.vs.eoi_blup_sub_chr-3_peak.marker-UNCHS009532_lod.drop-1.5_5.batches_mis.csv
    Untracked:  data/ici.vs.eoi_blup_sub_chr-3_peak.marker-UNCHS009541_lod.drop-1.5_5.batches_mis.csv
    Untracked:  data/ici.vs.eoi_blup_sub_chr-3_peak.marker-UNCHS009560_lod.drop-1.5_5.batches_mis.csv
    Untracked:  data/ici.vs.eoi_blup_sub_chr-3_peak.marker-UNCHS009574_lod.drop-1.5_5.batches_mis.csv
    Untracked:  data/ici.vs.eoi_blup_sub_chr-3_peak.marker-UNCJPD001170_lod.drop-1.5_5.batches_mis.csv
    Untracked:  data/ici.vs.eoi_blup_sub_chr-3_peak.marker-UNCJPD001198_lod.drop-1.5_5.batches_mis.csv
    Untracked:  data/ici.vs.eoi_blup_sub_chr-3_peak.marker-UNCJPD001276_lod.drop-1.5_5.batches_mis.csv
    Untracked:  data/ici.vs.eoi_blup_sub_chr-3_peak.marker-UNCJPD001315_lod.drop-1.5_5.batches_mis.csv
    Untracked:  data/ici.vs.eoi_blup_sub_chr-3_peak.marker-UNCJPD001355_lod.drop-1.5_5.batches_mis.csv
    Untracked:  data/ici.vs.eoi_blup_sub_chr-3_peak.marker-UNCJPD001373_lod.drop-1.5_5.batches_mis.csv
    Untracked:  data/ici.vs.eoi_blup_sub_chr-3_peak.marker-UNCJPD001382_lod.drop-1.5_5.batches_mis.csv
    Untracked:  data/ici.vs.eoi_blup_sub_chr-4_peak.marker-UNC6851205_lod.drop-1.5_5.batches_mis.csv
    Untracked:  data/ici.vs.eoi_blup_sub_chr-4_peak.marker-UNC8250659_lod.drop-1.5_snpsqc_dis_no-x_updated_5.batches.csv
    Untracked:  data/ici.vs.eoi_blup_sub_chr-4_peak.marker-UNC8299912_lod.drop-1.5_5.batches_mis.csv
    Untracked:  data/ici.vs.eoi_blup_sub_chr-4_peak.marker-UNC8300499_lod.drop-1.5_5.batches.csv
    Untracked:  data/ici.vs.eoi_blup_sub_chr-4_peak.marker-UNC8381981_lod.drop-1.5_5.batches_mis.csv
    Untracked:  data/ici.vs.eoi_blup_sub_chr-4_peak.marker-UNC8436434_lod.drop-1.5_5.batches_mis.csv
    Untracked:  data/ici.vs.eoi_blup_sub_chr-4_peak.marker-UNC8439633_lod.drop-1.5_5.batches_mis.csv
    Untracked:  data/ici.vs.eoi_blup_sub_chr-4_peak.marker-UNC8530763_lod.drop-1.5_5.batches_mis.csv
    Untracked:  data/ici.vs.eoi_blup_sub_chr-4_peak.marker-UNCHS012948_lod.drop-1.5_snpsqc_dis_no-x_updated_5.batches_mis.csv
    Untracked:  data/ici.vs.eoi_blup_sub_chr-4_peak.marker-UNCHS012976_lod.drop-1.5_5.batches_mis.csv
    Untracked:  data/ici.vs.eoi_blup_sub_chr-4_peak.marker-UNCHS013118_lod.drop-1.5_5.batches.csv
    Untracked:  data/ici.vs.eoi_blup_sub_chr-4_peak.marker-UNCHS013118_lod.drop-1.5_snpsqc_5.batches.csv
    Untracked:  data/ici.vs.eoi_blup_sub_chr-4_peak.marker-UNCHS013118_lod.drop-1.5_snpsqc_5.batches_mis.csv
    Untracked:  data/ici.vs.eoi_blup_sub_chr-4_peak.marker-UNCHS013130_lod.drop-1.5_5.batches_mis.csv
    Untracked:  data/ici.vs.eoi_blup_sub_chr-4_peak.marker-UNCHS013139_lod.drop-1.5_5.batches_mis.csv
    Untracked:  data/ici.vs.eoi_blup_sub_chr-4_peak.marker-UNCHS013304_lod.drop-1.5_5.batches.csv
    Untracked:  data/ici.vs.eoi_blup_sub_chr-5_peak.marker-JAX00590770_lod.drop-1.5_5.batches_mis.csv
    Untracked:  data/ici.vs.eoi_blup_sub_chr-5_peak.marker-UNC10044126_lod.drop-1.5_5.batches_mis.csv
    Untracked:  data/ici.vs.eoi_blup_sub_chr-7_peak.marker-ICR1830_lod.drop-1.5_5.batches_mis.csv
    Untracked:  data/ici.vs.eoi_blup_sub_chr-7_peak.marker-UNC13170794_lod.drop-1.5_5.batches_mis.csv
    Untracked:  data/ici.vs.eoi_blup_sub_chr-7_peak.marker-UNCHS019480_lod.drop-1.5_5.batches.csv
    Untracked:  data/ici.vs.eoi_blup_sub_chr-7_peak.marker-UNCHS020066_lod.drop-1.5_snpsqc_5.batches_mis.csv
    Untracked:  data/ici.vs.eoi_blup_sub_chr-7_peak.marker-UNCHS020066_lod.drop-1.5_snpsqc_dis_no-x_updated_5.batches_mis.csv
    Untracked:  data/ici.vs.eoi_blup_sub_chr-7_peak.marker-cr31snv87_lod.drop-1.5_5.batches_mis.csv
    Untracked:  data/ici.vs.eoi_blup_sub_chr-8_peak.marker-JAX00667121_lod.drop-1.5_5.batches_mis.csv
    Untracked:  data/ici.vs.eoi_blup_sub_chr-8_peak.marker-UNC15430711_lod.drop-1.5_5.batches_mis.csv
    Untracked:  data/ici.vs.eoi_blup_sub_chr-8_peak.marker-UNC15524531_lod.drop-1.5_5.batches_mis.csv
    Untracked:  data/ici.vs.eoi_blup_sub_chr-8_peak.marker-UNC15524531_lod.drop-1.5_snpsqc_5.batches_mis.csv
    Untracked:  data/ici.vs.eoi_blup_sub_chr-8_peak.marker-UNC15524531_lod.drop-1.5_snpsqc_dis_no-x_updated_5.batches_mis.csv
    Untracked:  data/ici.vs.eoi_blup_sub_chr-9_peak.marker-UNC17203329_lod.drop-1.5_5.batches_mis.csv
    Untracked:  data/ici.vs.eoi_blup_sub_chr-9_peak.marker-UNC17203597_lod.drop-1.5_snpsqc_5.batches_mis.csv
    Untracked:  data/ici.vs.eoi_blup_sub_chr-9_peak.marker-UNCHS024593_lod.drop-1.5_5.batches_mis.csv
    Untracked:  data/ici.vs.eoi_blup_sub_chr-X_peak.marker-UNC30627130_lod.drop-1.5_5.batches_mis.csv
    Untracked:  data/ici.vs.eoi_blup_sub_chr-X_peak.marker-UNCHS048313_lod.drop-1.5_5.batches_mis.csv
    Untracked:  data/ici.vs.eoi_blup_sub_chr-X_peak.marker-UNCHS048314_lod.drop-1.5_snpsqc_5.batches_mis.csv
    Untracked:  data/ici.vs.eoi_blup_sub_chr-X_peak.marker-UNCHS048314_lod.drop-1.5_snpsqc_dis_no-x_updated_5.batches_mis.csv
    Untracked:  data/ici.vs.eoi_fitqtl_peaks_additive_5.batches.txt
    Untracked:  data/ici.vs.eoi_fitqtl_peaks_additive_5.batches_mis.txt
    Untracked:  data/ici.vs.eoi_fitqtl_peaks_additive_snpsqc_5.batches.txt
    Untracked:  data/ici.vs.eoi_fitqtl_peaks_additive_snpsqc_dis_no-x_updated_5.batches.txt
    Untracked:  data/ici.vs.eoi_fitqtl_peaks_additive_snpsqc_dis_no-x_updated_5.batches_mis.txt
    Untracked:  data/ici.vs.eoi_fitqtl_peaks_interacting_5.batches.txt
    Untracked:  data/ici.vs.eoi_fitqtl_peaks_interacting_5.batches_mis.txt
    Untracked:  data/ici.vs.eoi_fitqtl_peaks_interacting_snpsqc_5.batches.txt
    Untracked:  data/ici.vs.eoi_fitqtl_peaks_interacting_snpsqc_dis_no-x_updated_5.batches.txt
    Untracked:  data/ici.vs.eoi_fitqtl_peaks_interacting_snpsqc_dis_no-x_updated_5.batches_mis.txt
    Untracked:  data/ici.vs.eoi_fitqtl_peaks_sex_additive_5.batches.txt
    Untracked:  data/ici.vs.eoi_fitqtl_peaks_sex_additive_5.batches_mis.txt
    Untracked:  data/ici.vs.eoi_fitqtl_peaks_sex_additive_snpsqc_5.batches.txt
    Untracked:  data/ici.vs.eoi_fitqtl_peaks_sex_additive_snpsqc_dis_no-x_updated_5.batches.txt
    Untracked:  data/ici.vs.eoi_fitqtl_peaks_sex_additive_snpsqc_dis_no-x_updated_5.batches_mis.txt
    Untracked:  data/ici.vs.eoi_fitqtl_peaks_sex_interacting_5.batches.txt
    Untracked:  data/ici.vs.eoi_fitqtl_peaks_sex_interacting_5.batches_mis.txt
    Untracked:  data/ici.vs.eoi_fitqtl_peaks_sex_interacting_snpsqc_5.batches.txt
    Untracked:  data/ici.vs.eoi_fitqtl_peaks_sex_interacting_snpsqc_dis_no-x_updated_5.batches.txt
    Untracked:  data/ici.vs.eoi_fitqtl_peaks_sex_interacting_snpsqc_dis_no-x_updated_5.batches_mis.mis.txt
    Untracked:  data/ici.vs.eoi_genes_chr-10_peak.marker-ICR4295_lod.drop-1.5_5.batches_mis.csv
    Untracked:  data/ici.vs.eoi_genes_chr-10_peak.marker-JAX00292499_lod.drop-1.5_5.batches.csv
    Untracked:  data/ici.vs.eoi_genes_chr-10_peak.marker-JAX00292927_lod.drop-1.5_5.batches.csv
    Untracked:  data/ici.vs.eoi_genes_chr-10_peak.marker-JAX00292927_lod.drop-1.5_snpsqc_5.batches.csv
    Untracked:  data/ici.vs.eoi_genes_chr-10_peak.marker-JAX00292927_lod.drop-1.5_snpsqc_5.batches_mis.csv
    Untracked:  data/ici.vs.eoi_genes_chr-10_peak.marker-UNC18154330_lod.drop-1.5_5.batches.csv
    Untracked:  data/ici.vs.eoi_genes_chr-10_peak.marker-UNC18195765_lod.drop-1.5_5.batches.csv
    Untracked:  data/ici.vs.eoi_genes_chr-10_peak.marker-UNC18216614_lod.drop-1.5_5.batches.csv
    Untracked:  data/ici.vs.eoi_genes_chr-10_peak.marker-UNC18240977_lod.drop-1.5_snpsqc_dis_no-x_updated_5.batches.csv
    Untracked:  data/ici.vs.eoi_genes_chr-10_peak.marker-UNC18311938_lod.drop-1.5_5.batches.csv
    Untracked:  data/ici.vs.eoi_genes_chr-10_peak.marker-UNC18856953_lod.drop-1.5_5.batches_mis.csv
    Untracked:  data/ici.vs.eoi_genes_chr-10_peak.marker-UNCHS028236_lod.drop-1.5_snpsqc_5.batches_mis.csv
    Untracked:  data/ici.vs.eoi_genes_chr-10_peak.marker-UNCHS028536_lod.drop-1.5_snpsqc_dis_no-x_updated_5.batches_mis.csv
    Untracked:  data/ici.vs.eoi_genes_chr-10_peak.marker-UNCJPD004316_lod.drop-1.5_5.batches_mis.csv
    Untracked:  data/ici.vs.eoi_genes_chr-10_peak.marker-UNCJPD009532_lod.drop-1.5_5.batches_mis.csv
    Untracked:  data/ici.vs.eoi_genes_chr-11_peak.marker-UNC19155926_lod.drop-1.5_5.batches_mis.csv
    Untracked:  data/ici.vs.eoi_genes_chr-11_peak.marker-UNC19481366_lod.drop-1.5_snpsqc_5.batches_mis.csv
    Untracked:  data/ici.vs.eoi_genes_chr-11_peak.marker-UNC19970181_lod.drop-1.5_5.batches_mis.csv
    Untracked:  data/ici.vs.eoi_genes_chr-11_peak.marker-UNC19970181_lod.drop-1.5_snpsqc_5.batches_mis.csv
    Untracked:  data/ici.vs.eoi_genes_chr-11_peak.marker-UNCJPD004792_lod.drop-1.5_5.batches_mis.csv
    Untracked:  data/ici.vs.eoi_genes_chr-12_peak.marker-ICR4497_lod.drop-1.5_5.batches_mis.csv
    Untracked:  data/ici.vs.eoi_genes_chr-12_peak.marker-ICR499_lod.drop-1.5_snpsqc_5.batches_mis.csv
    Untracked:  data/ici.vs.eoi_genes_chr-13_peak.marker-UNCHS036773_lod.drop-1.5_snpsqc_5.batches_mis.csv
    Untracked:  data/ici.vs.eoi_genes_chr-13_peak.marker-UNCHS036964_lod.drop-1.5_5.batches_mis.csv
    Untracked:  data/ici.vs.eoi_genes_chr-14_peak.marker-UNC24056202_lod.drop-1.5_5.batches_mis.csv
    Untracked:  data/ici.vs.eoi_genes_chr-14_peak.marker-UNC24056202_lod.drop-1.5_snpsqc_5.batches_mis.csv
    Untracked:  data/ici.vs.eoi_genes_chr-15_peak.marker-UNC26070435_lod.drop-1.5_snpsqc_dis_no-x_updated_5.batches_mis.csv
    Untracked:  data/ici.vs.eoi_genes_chr-17_peak.marker-UNCHS044241_lod.drop-1.5_5.batches_mis.csv
    Untracked:  data/ici.vs.eoi_genes_chr-17_peak.marker-UNCHS044241_lod.drop-1.5_snpsqc_5.batches_mis.csv
    Untracked:  data/ici.vs.eoi_genes_chr-17_peak.marker-UNCrs47191360_lod.drop-1.5_5.batches_mis.csv
    Untracked:  data/ici.vs.eoi_genes_chr-17_peak.marker-UNCrs47191360_lod.drop-1.5_snpsqc_dis_no-x_updated_5.batches_mis.csv
    Untracked:  data/ici.vs.eoi_genes_chr-18_peak.marker-UNCHS045343_lod.drop-1.5_snpsqc_5.batches_mis.csv
    Untracked:  data/ici.vs.eoi_genes_chr-18_peak.marker-UNCHS045344_lod.drop-1.5_5.batches_mis.csv
    Untracked:  data/ici.vs.eoi_genes_chr-19_peak.marker-UNCJPD007087_lod.drop-1.5_5.batches_mis.csv
    Untracked:  data/ici.vs.eoi_genes_chr-1_peak.marker-ICR010_lod.drop-1.5_5.batches.csv
    Untracked:  data/ici.vs.eoi_genes_chr-1_peak.marker-ICR010_lod.drop-1.5_5.batches_mis.csv
    Untracked:  data/ici.vs.eoi_genes_chr-2_peak.marker-UNC4609527_lod.drop-1.5_snpsqc_dis_no-x_updated_5.batches_mis.csv
    Untracked:  data/ici.vs.eoi_genes_chr-2_peak.marker-UNC4609660_lod.drop-1.5_5.batches_mis.csv
    Untracked:  data/ici.vs.eoi_genes_chr-3_peak.marker-B6_rs31740628_lod.drop-1.5_5.batches_mis.csv
    Untracked:  data/ici.vs.eoi_genes_chr-3_peak.marker-ICR1269_lod.drop-1.5_5.batches_mis.csv
    Untracked:  data/ici.vs.eoi_genes_chr-3_peak.marker-ICR1338_lod.drop-1.5_5.batches.csv
    Untracked:  data/ici.vs.eoi_genes_chr-3_peak.marker-ICR5325_lod.drop-1.5_5.batches_mis.csv
    Untracked:  data/ici.vs.eoi_genes_chr-3_peak.marker-ICR5347_lod.drop-1.5_5.batches.csv
    Untracked:  data/ici.vs.eoi_genes_chr-3_peak.marker-ICR5348_lod.drop-1.5_5.batches_mis.csv
    Untracked:  data/ici.vs.eoi_genes_chr-3_peak.marker-JAX00104971_lod.drop-1.5_5.batches_mis.csv
    Untracked:  data/ici.vs.eoi_genes_chr-3_peak.marker-JAX00105059_lod.drop-1.5_5.batches.csv
    Untracked:  data/ici.vs.eoi_genes_chr-3_peak.marker-JAX00105203_lod.drop-1.5_5.batches_mis.csv
    Untracked:  data/ici.vs.eoi_genes_chr-3_peak.marker-JAX00106957_lod.drop-1.5_5.batches.csv
    Untracked:  data/ici.vs.eoi_genes_chr-3_peak.marker-JAX00106957_lod.drop-1.5_5.batches_mis.csv
    Untracked:  data/ici.vs.eoi_genes_chr-3_peak.marker-JAX00107930_lod.drop-1.5_5.batches_mis.csv
    Untracked:  data/ici.vs.eoi_genes_chr-3_peak.marker-JAX00107980_lod.drop-1.5_5.batches_mis.csv
    Untracked:  data/ici.vs.eoi_genes_chr-3_peak.marker-JAX00108068_lod.drop-1.5_5.batches_mis.csv
    Untracked:  data/ici.vs.eoi_genes_chr-3_peak.marker-JAX00108114_lod.drop-1.5_5.batches_mis.csv
    Untracked:  data/ici.vs.eoi_genes_chr-3_peak.marker-JAX00109750_lod.drop-1.5_5.batches_mis.csv
    Untracked:  data/ici.vs.eoi_genes_chr-3_peak.marker-JAX00189245_lod.drop-1.5_5.batches_mis.csv
    Untracked:  data/ici.vs.eoi_genes_chr-3_peak.marker-JAX00514662_lod.drop-1.5_5.batches.csv
    Untracked:  data/ici.vs.eoi_genes_chr-3_peak.marker-JAX00517218_lod.drop-1.5_5.batches_mis.csv
    Untracked:  data/ici.vs.eoi_genes_chr-3_peak.marker-JAX00517674_lod.drop-1.5_5.batches_mis.csv
    Untracked:  data/ici.vs.eoi_genes_chr-3_peak.marker-JAX00517906_lod.drop-1.5_5.batches_mis.csv
    Untracked:  data/ici.vs.eoi_genes_chr-3_peak.marker-JAX00518418_lod.drop-1.5_5.batches_mis.csv
    Untracked:  data/ici.vs.eoi_genes_chr-3_peak.marker-JAX00518684_lod.drop-1.5_5.batches_mis.csv
    Untracked:  data/ici.vs.eoi_genes_chr-3_peak.marker-JAX00518911r_lod.drop-1.5_5.batches_mis.csv
    Untracked:  data/ici.vs.eoi_genes_chr-3_peak.marker-JAX00520122_lod.drop-1.5_5.batches_mis.csv
    Untracked:  data/ici.vs.eoi_genes_chr-3_peak.marker-JAX00524717_lod.drop-1.5_5.batches_mis.csv
    Untracked:  data/ici.vs.eoi_genes_chr-3_peak.marker-JAX00524828_lod.drop-1.5_5.batches_mis.csv
    Untracked:  data/ici.vs.eoi_genes_chr-3_peak.marker-JAX00525380_lod.drop-1.5_5.batches_mis.csv
    Untracked:  data/ici.vs.eoi_genes_chr-3_peak.marker-JAX00525709_lod.drop-1.5_5.batches_mis.csv
    Untracked:  data/ici.vs.eoi_genes_chr-3_peak.marker-JAX00527291_lod.drop-1.5_5.batches_mis.csv
    Untracked:  data/ici.vs.eoi_genes_chr-3_peak.marker-JAX00527587_lod.drop-1.5_5.batches_mis.csv
    Untracked:  data/ici.vs.eoi_genes_chr-3_peak.marker-JAX00529707_lod.drop-1.5_5.batches_mis.csv
    Untracked:  data/ici.vs.eoi_genes_chr-3_peak.marker-JAX00529707r_lod.drop-1.5_5.batches.csv
    Untracked:  data/ici.vs.eoi_genes_chr-3_peak.marker-JAX00531601_lod.drop-1.5_5.batches_mis.csv
    Untracked:  data/ici.vs.eoi_genes_chr-3_peak.marker-JAX00532122_lod.drop-1.5_5.batches_mis.csv
    Untracked:  data/ici.vs.eoi_genes_chr-3_peak.marker-UNC4673471_lod.drop-1.5_5.batches_mis.csv
    Untracked:  data/ici.vs.eoi_genes_chr-3_peak.marker-UNC4681411_lod.drop-1.5_5.batches_mis.csv
    Untracked:  data/ici.vs.eoi_genes_chr-3_peak.marker-UNC4793677_lod.drop-1.5_5.batches_mis.csv
    Untracked:  data/ici.vs.eoi_genes_chr-3_peak.marker-UNC4799475_lod.drop-1.5_5.batches_mis.csv
    Untracked:  data/ici.vs.eoi_genes_chr-3_peak.marker-UNC4809586_lod.drop-1.5_5.batches_mis.csv
    Untracked:  data/ici.vs.eoi_genes_chr-3_peak.marker-UNC4813186_lod.drop-1.5_5.batches.csv
    Untracked:  data/ici.vs.eoi_genes_chr-3_peak.marker-UNC4813186_lod.drop-1.5_5.batches_mis.csv
    Untracked:  data/ici.vs.eoi_genes_chr-3_peak.marker-UNC4820379_lod.drop-1.5_5.batches_mis.csv
    Untracked:  data/ici.vs.eoi_genes_chr-3_peak.marker-UNC4875653_lod.drop-1.5_5.batches_mis.csv
    Untracked:  data/ici.vs.eoi_genes_chr-3_peak.marker-UNC4909830_lod.drop-1.5_5.batches_mis.csv
    Untracked:  data/ici.vs.eoi_genes_chr-3_peak.marker-UNC4912288_lod.drop-1.5_5.batches_mis.csv
    Untracked:  data/ici.vs.eoi_genes_chr-3_peak.marker-UNC4914130_lod.drop-1.5_5.batches_mis.csv
    Untracked:  data/ici.vs.eoi_genes_chr-3_peak.marker-UNC4923398_lod.drop-1.5_5.batches_mis.csv
    Untracked:  data/ici.vs.eoi_genes_chr-3_peak.marker-UNC4924377_lod.drop-1.5_5.batches.csv
    Untracked:  data/ici.vs.eoi_genes_chr-3_peak.marker-UNC4945186_lod.drop-1.5_5.batches_mis.csv
    Untracked:  data/ici.vs.eoi_genes_chr-3_peak.marker-UNC4947492_lod.drop-1.5_5.batches_mis.csv
    Untracked:  data/ici.vs.eoi_genes_chr-3_peak.marker-UNC4957168_lod.drop-1.5_5.batches.csv
    Untracked:  data/ici.vs.eoi_genes_chr-3_peak.marker-UNC4957859_lod.drop-1.5_5.batches_mis.csv
    Untracked:  data/ici.vs.eoi_genes_chr-3_peak.marker-UNC4964062_lod.drop-1.5_5.batches_mis.csv
    Untracked:  data/ici.vs.eoi_genes_chr-3_peak.marker-UNC4979914_lod.drop-1.5_5.batches_mis.csv
    Untracked:  data/ici.vs.eoi_genes_chr-3_peak.marker-UNC4985064_lod.drop-1.5_5.batches_mis.csv
    Untracked:  data/ici.vs.eoi_genes_chr-3_peak.marker-UNC4990111_lod.drop-1.5_5.batches_mis.csv
    Untracked:  data/ici.vs.eoi_genes_chr-3_peak.marker-UNC4991565_lod.drop-1.5_5.batches_mis.csv
    Untracked:  data/ici.vs.eoi_genes_chr-3_peak.marker-UNC5001723_lod.drop-1.5_5.batches_mis.csv
    Untracked:  data/ici.vs.eoi_genes_chr-3_peak.marker-UNC5008656_lod.drop-1.5_5.batches.csv
    Untracked:  data/ici.vs.eoi_genes_chr-3_peak.marker-UNC5008656_lod.drop-1.5_5.batches_mis.csv
    Untracked:  data/ici.vs.eoi_genes_chr-3_peak.marker-UNC5042361_lod.drop-1.5_5.batches_mis.csv
    Untracked:  data/ici.vs.eoi_genes_chr-3_peak.marker-UNC5055259_lod.drop-1.5_5.batches_mis.csv
    Untracked:  data/ici.vs.eoi_genes_chr-3_peak.marker-UNC5076935_lod.drop-1.5_5.batches_mis.csv
    Untracked:  data/ici.vs.eoi_genes_chr-3_peak.marker-UNC5082757_lod.drop-1.5_5.batches_mis.csv
    Untracked:  data/ici.vs.eoi_genes_chr-3_peak.marker-UNC5083168_lod.drop-1.5_5.batches.csv
    Untracked:  data/ici.vs.eoi_genes_chr-3_peak.marker-UNC5173293_lod.drop-1.5_5.batches_mis.csv
    Untracked:  data/ici.vs.eoi_genes_chr-3_peak.marker-UNC5182782_lod.drop-1.5_5.batches_mis.csv
    Untracked:  data/ici.vs.eoi_genes_chr-3_peak.marker-UNC5192582_lod.drop-1.5_5.batches.csv
    Untracked:  data/ici.vs.eoi_genes_chr-3_peak.marker-UNC5193970_lod.drop-1.5_5.batches_mis.csv
    Untracked:  data/ici.vs.eoi_genes_chr-3_peak.marker-UNC5213888_lod.drop-1.5_snpsqc_5.batches_mis.csv
    Untracked:  data/ici.vs.eoi_genes_chr-3_peak.marker-UNC5240484_lod.drop-1.5_5.batches_mis.csv
    Untracked:  data/ici.vs.eoi_genes_chr-3_peak.marker-UNC5256922_lod.drop-1.5_5.batches_mis.csv
    Untracked:  data/ici.vs.eoi_genes_chr-3_peak.marker-UNC5266664_lod.drop-1.5_5.batches_mis.csv
    Untracked:  data/ici.vs.eoi_genes_chr-3_peak.marker-UNC5266711_lod.drop-1.5_5.batches.csv
    Untracked:  data/ici.vs.eoi_genes_chr-3_peak.marker-UNC5267813_lod.drop-1.5_5.batches_mis.csv
    Untracked:  data/ici.vs.eoi_genes_chr-3_peak.marker-UNC5274554_lod.drop-1.5_5.batches_mis.csv
    Untracked:  data/ici.vs.eoi_genes_chr-3_peak.marker-UNC5277061_lod.drop-1.5_5.batches_mis.csv
    Untracked:  data/ici.vs.eoi_genes_chr-3_peak.marker-UNC5282259_lod.drop-1.5_5.batches_mis.csv
    Untracked:  data/ici.vs.eoi_genes_chr-3_peak.marker-UNC5287977_lod.drop-1.5_5.batches_mis.csv
    Untracked:  data/ici.vs.eoi_genes_chr-3_peak.marker-UNC5301460_lod.drop-1.5_5.batches_mis.csv
    Untracked:  data/ici.vs.eoi_genes_chr-3_peak.marker-UNC5321559_lod.drop-1.5_5.batches_mis.csv
    Untracked:  data/ici.vs.eoi_genes_chr-3_peak.marker-UNC5322455_lod.drop-1.5_5.batches_mis.csv
    Untracked:  data/ici.vs.eoi_genes_chr-3_peak.marker-UNC5328596_lod.drop-1.5_5.batches_mis.csv
    Untracked:  data/ici.vs.eoi_genes_chr-3_peak.marker-UNC5350384_lod.drop-1.5_5.batches_mis.csv
    Untracked:  data/ici.vs.eoi_genes_chr-3_peak.marker-UNC5354724_lod.drop-1.5_5.batches_mis.csv
    Untracked:  data/ici.vs.eoi_genes_chr-3_peak.marker-UNC5375208_lod.drop-1.5_5.batches_mis.csv
    Untracked:  data/ici.vs.eoi_genes_chr-3_peak.marker-UNC5384072_lod.drop-1.5_5.batches_mis.csv
    Untracked:  data/ici.vs.eoi_genes_chr-3_peak.marker-UNC5400314_lod.drop-1.5_5.batches_mis.csv
    Untracked:  data/ici.vs.eoi_genes_chr-3_peak.marker-UNC5433945_lod.drop-1.5_5.batches_mis.csv
    Untracked:  data/ici.vs.eoi_genes_chr-3_peak.marker-UNC5435874_lod.drop-1.5_5.batches_mis.csv
    Untracked:  data/ici.vs.eoi_genes_chr-3_peak.marker-UNC5454839_lod.drop-1.5_5.batches.csv
    Untracked:  data/ici.vs.eoi_genes_chr-3_peak.marker-UNC5454839_lod.drop-1.5_5.batches_mis.csv
    Untracked:  data/ici.vs.eoi_genes_chr-3_peak.marker-UNC5466669_lod.drop-1.5_5.batches_mis.csv
    Untracked:  data/ici.vs.eoi_genes_chr-3_peak.marker-UNC5517972_lod.drop-1.5_5.batches_mis.csv
    Untracked:  data/ici.vs.eoi_genes_chr-3_peak.marker-UNC5522257_lod.drop-1.5_5.batches_mis.csv
    Untracked:  data/ici.vs.eoi_genes_chr-3_peak.marker-UNC5523265_lod.drop-1.5_5.batches_mis.csv
    Untracked:  data/ici.vs.eoi_genes_chr-3_peak.marker-UNC5532424_lod.drop-1.5_5.batches_mis.csv
    Untracked:  data/ici.vs.eoi_genes_chr-3_peak.marker-UNC5553882_lod.drop-1.5_5.batches_mis.csv
    Untracked:  data/ici.vs.eoi_genes_chr-3_peak.marker-UNC5572051_lod.drop-1.5_5.batches_mis.csv
    Untracked:  data/ici.vs.eoi_genes_chr-3_peak.marker-UNC5582753_lod.drop-1.5_5.batches.csv
    Untracked:  data/ici.vs.eoi_genes_chr-3_peak.marker-UNC5589260_lod.drop-1.5_5.batches_mis.csv
    Untracked:  data/ici.vs.eoi_genes_chr-3_peak.marker-UNC5595555_lod.drop-1.5_5.batches_mis.csv
    Untracked:  data/ici.vs.eoi_genes_chr-3_peak.marker-UNC5623239_lod.drop-1.5_5.batches_mis.csv
    Untracked:  data/ici.vs.eoi_genes_chr-3_peak.marker-UNC5639830_lod.drop-1.5_5.batches_mis.csv
    Untracked:  data/ici.vs.eoi_genes_chr-3_peak.marker-UNC5645844_lod.drop-1.5_5.batches.csv
    Untracked:  data/ici.vs.eoi_genes_chr-3_peak.marker-UNC5645844_lod.drop-1.5_5.batches_mis.csv
    Untracked:  data/ici.vs.eoi_genes_chr-3_peak.marker-UNC5680234_lod.drop-1.5_5.batches_mis.csv
    Untracked:  data/ici.vs.eoi_genes_chr-3_peak.marker-UNC5682314_lod.drop-1.5_5.batches_mis.csv
    Untracked:  data/ici.vs.eoi_genes_chr-3_peak.marker-UNC5725356_lod.drop-1.5_5.batches_mis.csv
    Untracked:  data/ici.vs.eoi_genes_chr-3_peak.marker-UNC5727351_lod.drop-1.5_5.batches_mis.csv
    Untracked:  data/ici.vs.eoi_genes_chr-3_peak.marker-UNC5734420_lod.drop-1.5_5.batches_mis.csv
    Untracked:  data/ici.vs.eoi_genes_chr-3_peak.marker-UNC5743257_lod.drop-1.5_5.batches_mis.csv
    Untracked:  data/ici.vs.eoi_genes_chr-3_peak.marker-UNC5751567_lod.drop-1.5_5.batches_mis.csv
    Untracked:  data/ici.vs.eoi_genes_chr-3_peak.marker-UNC5752623_lod.drop-1.5_5.batches_mis.csv
    Untracked:  data/ici.vs.eoi_genes_chr-3_peak.marker-UNC5776974_lod.drop-1.5_5.batches_mis.csv
    Untracked:  data/ici.vs.eoi_genes_chr-3_peak.marker-UNC5778977_lod.drop-1.5_5.batches_mis.csv
    Untracked:  data/ici.vs.eoi_genes_chr-3_peak.marker-UNC5788507_lod.drop-1.5_5.batches_mis.csv
    Untracked:  data/ici.vs.eoi_genes_chr-3_peak.marker-UNC5792468_lod.drop-1.5_5.batches_mis.csv
    Untracked:  data/ici.vs.eoi_genes_chr-3_peak.marker-UNC5794598_lod.drop-1.5_5.batches_mis.csv
    Untracked:  data/ici.vs.eoi_genes_chr-3_peak.marker-UNC5795561_lod.drop-1.5_5.batches_mis.csv
    Untracked:  data/ici.vs.eoi_genes_chr-3_peak.marker-UNC5802298_lod.drop-1.5_5.batches_mis.csv
    Untracked:  data/ici.vs.eoi_genes_chr-3_peak.marker-UNC5806628_lod.drop-1.5_5.batches_mis.csv
    Untracked:  data/ici.vs.eoi_genes_chr-3_peak.marker-UNC5808246_lod.drop-1.5_5.batches.csv
    Untracked:  data/ici.vs.eoi_genes_chr-3_peak.marker-UNC5817478_lod.drop-1.5_5.batches_mis.csv
    Untracked:  data/ici.vs.eoi_genes_chr-3_peak.marker-UNC5836999_lod.drop-1.5_5.batches_mis.csv
    Untracked:  data/ici.vs.eoi_genes_chr-3_peak.marker-UNC5841501_lod.drop-1.5_5.batches_mis.csv
    Untracked:  data/ici.vs.eoi_genes_chr-3_peak.marker-UNC5847927_lod.drop-1.5_5.batches_mis.csv
    Untracked:  data/ici.vs.eoi_genes_chr-3_peak.marker-UNC5966417_lod.drop-1.5_5.batches_mis.csv
    Untracked:  data/ici.vs.eoi_genes_chr-3_peak.marker-UNCHS008129_lod.drop-1.5_5.batches_mis.csv
    Untracked:  data/ici.vs.eoi_genes_chr-3_peak.marker-UNCHS008173_lod.drop-1.5_5.batches_mis.csv
    Untracked:  data/ici.vs.eoi_genes_chr-3_peak.marker-UNCHS008182_lod.drop-1.5_5.batches.csv
    Untracked:  data/ici.vs.eoi_genes_chr-3_peak.marker-UNCHS008193_lod.drop-1.5_5.batches_mis.csv
    Untracked:  data/ici.vs.eoi_genes_chr-3_peak.marker-UNCHS008209_lod.drop-1.5_5.batches_mis.csv
    Untracked:  data/ici.vs.eoi_genes_chr-3_peak.marker-UNCHS008215_lod.drop-1.5_5.batches_mis.csv
    Untracked:  data/ici.vs.eoi_genes_chr-3_peak.marker-UNCHS008219_lod.drop-1.5_5.batches_mis.csv
    Untracked:  data/ici.vs.eoi_genes_chr-3_peak.marker-UNCHS008231_lod.drop-1.5_5.batches_mis.csv
    Untracked:  data/ici.vs.eoi_genes_chr-3_peak.marker-UNCHS008235_lod.drop-1.5_5.batches_mis.csv
    Untracked:  data/ici.vs.eoi_genes_chr-3_peak.marker-UNCHS008242_lod.drop-1.5_5.batches.csv
    Untracked:  data/ici.vs.eoi_genes_chr-3_peak.marker-UNCHS008253_lod.drop-1.5_5.batches_mis.csv
    Untracked:  data/ici.vs.eoi_genes_chr-3_peak.marker-UNCHS008256_lod.drop-1.5_5.batches_mis.csv
    Untracked:  data/ici.vs.eoi_genes_chr-3_peak.marker-UNCHS008259_lod.drop-1.5_5.batches_mis.csv
    Untracked:  data/ici.vs.eoi_genes_chr-3_peak.marker-UNCHS008271_lod.drop-1.5_5.batches_mis.csv
    Untracked:  data/ici.vs.eoi_genes_chr-3_peak.marker-UNCHS008286_lod.drop-1.5_5.batches.csv
    Untracked:  data/ici.vs.eoi_genes_chr-3_peak.marker-UNCHS008286_lod.drop-1.5_5.batches_mis.csv
    Untracked:  data/ici.vs.eoi_genes_chr-3_peak.marker-UNCHS008311_lod.drop-1.5_5.batches_mis.csv
    Untracked:  data/ici.vs.eoi_genes_chr-3_peak.marker-UNCHS008317_lod.drop-1.5_5.batches_mis.csv
    Untracked:  data/ici.vs.eoi_genes_chr-3_peak.marker-UNCHS008372_lod.drop-1.5_5.batches_mis.csv
    Untracked:  data/ici.vs.eoi_genes_chr-3_peak.marker-UNCHS008379_lod.drop-1.5_5.batches_mis.csv
    Untracked:  data/ici.vs.eoi_genes_chr-3_peak.marker-UNCHS008410_lod.drop-1.5_5.batches_mis.csv
    Untracked:  data/ici.vs.eoi_genes_chr-3_peak.marker-UNCHS008415_lod.drop-1.5_5.batches_mis.csv
    Untracked:  data/ici.vs.eoi_genes_chr-3_peak.marker-UNCHS008431_lod.drop-1.5_5.batches_mis.csv
    Untracked:  data/ici.vs.eoi_genes_chr-3_peak.marker-UNCHS008487_lod.drop-1.5_snpsqc_5.batches.csv
    Untracked:  data/ici.vs.eoi_genes_chr-3_peak.marker-UNCHS008487_lod.drop-1.5_snpsqc_dis_no-x_updated_5.batches.csv
    Untracked:  data/ici.vs.eoi_genes_chr-3_peak.marker-UNCHS008487_lod.drop-1.5_snpsqc_dis_no-x_updated_5.batches_mis.csv
    Untracked:  data/ici.vs.eoi_genes_chr-3_peak.marker-UNCHS008511_lod.drop-1.5_snpsqc_5.batches_mis.csv
    Untracked:  data/ici.vs.eoi_genes_chr-3_peak.marker-UNCHS008609_lod.drop-1.5_5.batches_mis.csv
    Untracked:  data/ici.vs.eoi_genes_chr-3_peak.marker-UNCHS008614_lod.drop-1.5_5.batches.csv
    Untracked:  data/ici.vs.eoi_genes_chr-3_peak.marker-UNCHS008649_lod.drop-1.5_5.batches_mis.csv
    Untracked:  data/ici.vs.eoi_genes_chr-3_peak.marker-UNCHS008651_lod.drop-1.5_5.batches.csv
    Untracked:  data/ici.vs.eoi_genes_chr-3_peak.marker-UNCHS008656_lod.drop-1.5_5.batches_mis.csv
    Untracked:  data/ici.vs.eoi_genes_chr-3_peak.marker-UNCHS008723_lod.drop-1.5_5.batches_mis.csv
    Untracked:  data/ici.vs.eoi_genes_chr-3_peak.marker-UNCHS008802_lod.drop-1.5_5.batches_mis.csv
    Untracked:  data/ici.vs.eoi_genes_chr-3_peak.marker-UNCHS008803_lod.drop-1.5_5.batches_mis.csv
    Untracked:  data/ici.vs.eoi_genes_chr-3_peak.marker-UNCHS008808_lod.drop-1.5_5.batches_mis.csv
    Untracked:  data/ici.vs.eoi_genes_chr-3_peak.marker-UNCHS008815_lod.drop-1.5_5.batches.csv
    Untracked:  data/ici.vs.eoi_genes_chr-3_peak.marker-UNCHS008815_lod.drop-1.5_5.batches_mis.csv
    Untracked:  data/ici.vs.eoi_genes_chr-3_peak.marker-UNCHS008834_lod.drop-1.5_5.batches_mis.csv
    Untracked:  data/ici.vs.eoi_genes_chr-3_peak.marker-UNCHS008838_lod.drop-1.5_5.batches_mis.csv
    Untracked:  data/ici.vs.eoi_genes_chr-3_peak.marker-UNCHS008914_lod.drop-1.5_5.batches_mis.csv
    Untracked:  data/ici.vs.eoi_genes_chr-3_peak.marker-UNCHS008928_lod.drop-1.5_5.batches_mis.csv
    Untracked:  data/ici.vs.eoi_genes_chr-3_peak.marker-UNCHS008948_lod.drop-1.5_5.batches_mis.csv
    Untracked:  data/ici.vs.eoi_genes_chr-3_peak.marker-UNCHS008959_lod.drop-1.5_5.batches_mis.csv
    Untracked:  data/ici.vs.eoi_genes_chr-3_peak.marker-UNCHS008973_lod.drop-1.5_5.batches_mis.csv
    Untracked:  data/ici.vs.eoi_genes_chr-3_peak.marker-UNCHS008987_lod.drop-1.5_5.batches_mis.csv
    Untracked:  data/ici.vs.eoi_genes_chr-3_peak.marker-UNCHS009001_lod.drop-1.5_5.batches_mis.csv
    Untracked:  data/ici.vs.eoi_genes_chr-3_peak.marker-UNCHS009025_lod.drop-1.5_5.batches_mis.csv
    Untracked:  data/ici.vs.eoi_genes_chr-3_peak.marker-UNCHS009034_lod.drop-1.5_5.batches_mis.csv
    Untracked:  data/ici.vs.eoi_genes_chr-3_peak.marker-UNCHS009087_lod.drop-1.5_5.batches_mis.csv
    Untracked:  data/ici.vs.eoi_genes_chr-3_peak.marker-UNCHS009109_lod.drop-1.5_5.batches_mis.csv
    Untracked:  data/ici.vs.eoi_genes_chr-3_peak.marker-UNCHS009142_lod.drop-1.5_5.batches_mis.csv
    Untracked:  data/ici.vs.eoi_genes_chr-3_peak.marker-UNCHS009207_lod.drop-1.5_5.batches_mis.csv
    Untracked:  data/ici.vs.eoi_genes_chr-3_peak.marker-UNCHS009216_lod.drop-1.5_5.batches_mis.csv
    Untracked:  data/ici.vs.eoi_genes_chr-3_peak.marker-UNCHS009224_lod.drop-1.5_5.batches.csv
    Untracked:  data/ici.vs.eoi_genes_chr-3_peak.marker-UNCHS009306_lod.drop-1.5_5.batches_mis.csv
    Untracked:  data/ici.vs.eoi_genes_chr-3_peak.marker-UNCHS009316_lod.drop-1.5_5.batches_mis.csv
    Untracked:  data/ici.vs.eoi_genes_chr-3_peak.marker-UNCHS009364_lod.drop-1.5_5.batches_mis.csv
    Untracked:  data/ici.vs.eoi_genes_chr-3_peak.marker-UNCHS009377_lod.drop-1.5_5.batches_mis.csv
    Untracked:  data/ici.vs.eoi_genes_chr-3_peak.marker-UNCHS009382_lod.drop-1.5_5.batches_mis.csv
    Untracked:  data/ici.vs.eoi_genes_chr-3_peak.marker-UNCHS009383_lod.drop-1.5_5.batches.csv
    Untracked:  data/ici.vs.eoi_genes_chr-3_peak.marker-UNCHS009387_lod.drop-1.5_5.batches_mis.csv
    Untracked:  data/ici.vs.eoi_genes_chr-3_peak.marker-UNCHS009395_lod.drop-1.5_5.batches_mis.csv
    Untracked:  data/ici.vs.eoi_genes_chr-3_peak.marker-UNCHS009413_lod.drop-1.5_5.batches_mis.csv
    Untracked:  data/ici.vs.eoi_genes_chr-3_peak.marker-UNCHS009428_lod.drop-1.5_5.batches_mis.csv
    Untracked:  data/ici.vs.eoi_genes_chr-3_peak.marker-UNCHS009463_lod.drop-1.5_5.batches_mis.csv
    Untracked:  data/ici.vs.eoi_genes_chr-3_peak.marker-UNCHS009503_lod.drop-1.5_5.batches_mis.csv
    Untracked:  data/ici.vs.eoi_genes_chr-3_peak.marker-UNCHS009532_lod.drop-1.5_5.batches_mis.csv
    Untracked:  data/ici.vs.eoi_genes_chr-3_peak.marker-UNCHS009541_lod.drop-1.5_5.batches_mis.csv
    Untracked:  data/ici.vs.eoi_genes_chr-3_peak.marker-UNCHS009560_lod.drop-1.5_5.batches_mis.csv
    Untracked:  data/ici.vs.eoi_genes_chr-3_peak.marker-UNCHS009574_lod.drop-1.5_5.batches_mis.csv
    Untracked:  data/ici.vs.eoi_genes_chr-3_peak.marker-UNCJPD001170_lod.drop-1.5_5.batches_mis.csv
    Untracked:  data/ici.vs.eoi_genes_chr-3_peak.marker-UNCJPD001198_lod.drop-1.5_5.batches_mis.csv
    Untracked:  data/ici.vs.eoi_genes_chr-3_peak.marker-UNCJPD001276_lod.drop-1.5_5.batches_mis.csv
    Untracked:  data/ici.vs.eoi_genes_chr-3_peak.marker-UNCJPD001315_lod.drop-1.5_5.batches_mis.csv
    Untracked:  data/ici.vs.eoi_genes_chr-3_peak.marker-UNCJPD001355_lod.drop-1.5_5.batches_mis.csv
    Untracked:  data/ici.vs.eoi_genes_chr-3_peak.marker-UNCJPD001373_lod.drop-1.5_5.batches_mis.csv
    Untracked:  data/ici.vs.eoi_genes_chr-3_peak.marker-UNCJPD001382_lod.drop-1.5_5.batches_mis.csv
    Untracked:  data/ici.vs.eoi_genes_chr-4_peak.marker-UNC6851205_lod.drop-1.5_5.batches_mis.csv
    Untracked:  data/ici.vs.eoi_genes_chr-4_peak.marker-UNC8250659_lod.drop-1.5_snpsqc_dis_no-x_updated_5.batches.csv
    Untracked:  data/ici.vs.eoi_genes_chr-4_peak.marker-UNC8299912_lod.drop-1.5_5.batches_mis.csv
    Untracked:  data/ici.vs.eoi_genes_chr-4_peak.marker-UNC8300499_lod.drop-1.5_5.batches.csv
    Untracked:  data/ici.vs.eoi_genes_chr-4_peak.marker-UNC8381981_lod.drop-1.5_5.batches_mis.csv
    Untracked:  data/ici.vs.eoi_genes_chr-4_peak.marker-UNC8436434_lod.drop-1.5_5.batches_mis.csv
    Untracked:  data/ici.vs.eoi_genes_chr-4_peak.marker-UNC8439633_lod.drop-1.5_5.batches_mis.csv
    Untracked:  data/ici.vs.eoi_genes_chr-4_peak.marker-UNC8530763_lod.drop-1.5_5.batches_mis.csv
    Untracked:  data/ici.vs.eoi_genes_chr-4_peak.marker-UNCHS012948_lod.drop-1.5_snpsqc_dis_no-x_updated_5.batches_mis.csv
    Untracked:  data/ici.vs.eoi_genes_chr-4_peak.marker-UNCHS012976_lod.drop-1.5_5.batches_mis.csv
    Untracked:  data/ici.vs.eoi_genes_chr-4_peak.marker-UNCHS013118_lod.drop-1.5_5.batches.csv
    Untracked:  data/ici.vs.eoi_genes_chr-4_peak.marker-UNCHS013118_lod.drop-1.5_snpsqc_5.batches.csv
    Untracked:  data/ici.vs.eoi_genes_chr-4_peak.marker-UNCHS013118_lod.drop-1.5_snpsqc_5.batches_mis.csv
    Untracked:  data/ici.vs.eoi_genes_chr-4_peak.marker-UNCHS013130_lod.drop-1.5_5.batches_mis.csv
    Untracked:  data/ici.vs.eoi_genes_chr-4_peak.marker-UNCHS013139_lod.drop-1.5_5.batches_mis.csv
    Untracked:  data/ici.vs.eoi_genes_chr-4_peak.marker-UNCHS013304_lod.drop-1.5_5.batches.csv
    Untracked:  data/ici.vs.eoi_genes_chr-5_peak.marker-JAX00590770_lod.drop-1.5_5.batches_mis.csv
    Untracked:  data/ici.vs.eoi_genes_chr-5_peak.marker-UNC10044126_lod.drop-1.5_5.batches_mis.csv
    Untracked:  data/ici.vs.eoi_genes_chr-7_peak.marker-ICR1830_lod.drop-1.5_5.batches_mis.csv
    Untracked:  data/ici.vs.eoi_genes_chr-7_peak.marker-UNC13170794_lod.drop-1.5_5.batches_mis.csv
    Untracked:  data/ici.vs.eoi_genes_chr-7_peak.marker-UNCHS019480_lod.drop-1.5_5.batches.csv
    Untracked:  data/ici.vs.eoi_genes_chr-7_peak.marker-UNCHS020066_lod.drop-1.5_snpsqc_5.batches_mis.csv
    Untracked:  data/ici.vs.eoi_genes_chr-7_peak.marker-UNCHS020066_lod.drop-1.5_snpsqc_dis_no-x_updated_5.batches_mis.csv
    Untracked:  data/ici.vs.eoi_genes_chr-7_peak.marker-cr31snv87_lod.drop-1.5_5.batches_mis.csv
    Untracked:  data/ici.vs.eoi_genes_chr-8_peak.marker-JAX00667121_lod.drop-1.5_5.batches_mis.csv
    Untracked:  data/ici.vs.eoi_genes_chr-8_peak.marker-UNC15430711_lod.drop-1.5_5.batches_mis.csv
    Untracked:  data/ici.vs.eoi_genes_chr-8_peak.marker-UNC15524531_lod.drop-1.5_5.batches_mis.csv
    Untracked:  data/ici.vs.eoi_genes_chr-8_peak.marker-UNC15524531_lod.drop-1.5_snpsqc_5.batches_mis.csv
    Untracked:  data/ici.vs.eoi_genes_chr-8_peak.marker-UNC15524531_lod.drop-1.5_snpsqc_dis_no-x_updated_5.batches_mis.csv
    Untracked:  data/ici.vs.eoi_genes_chr-9_peak.marker-UNC17203329_lod.drop-1.5_5.batches_mis.csv
    Untracked:  data/ici.vs.eoi_genes_chr-9_peak.marker-UNC17203597_lod.drop-1.5_snpsqc_5.batches_mis.csv
    Untracked:  data/ici.vs.eoi_genes_chr-9_peak.marker-UNCHS024593_lod.drop-1.5_5.batches_mis.csv
    Untracked:  data/ici.vs.eoi_genes_chr-X_peak.marker-UNC30627130_lod.drop-1.5_5.batches_mis.csv
    Untracked:  data/ici.vs.eoi_genes_chr-X_peak.marker-UNCHS048313_lod.drop-1.5_5.batches_mis.csv
    Untracked:  data/ici.vs.eoi_genes_chr-X_peak.marker-UNCHS048314_lod.drop-1.5_snpsqc_5.batches_mis.csv
    Untracked:  data/ici.vs.eoi_genes_chr-X_peak.marker-UNCHS048314_lod.drop-1.5_snpsqc_dis_no-x_updated_5.batches_mis.csv
    Untracked:  data/ici.vs.eoi_gm_qtl_5.batches.csv
    Untracked:  data/ici.vs.eoi_gm_qtl_5.batches_mis.csv
    Untracked:  data/ici.vs.eoi_gm_qtl_snpsqc_5.batches.csv
    Untracked:  data/ici.vs.eoi_gm_qtl_snpsqc_5.batches_mis.csv
    Untracked:  data/ici.vs.eoi_gm_qtl_snpsqc_dis_no-x_updated_5.batches.csv
    Untracked:  data/ici.vs.eoi_gm_qtl_snpsqc_dis_no-x_updated_5.batches_mis.csv
    Untracked:  data/ici.vs.eoi_marker.freq_low.geno.freq.removed_geno.ratio.csv
    Untracked:  data/ici.vs.eoi_marker.freq_low.geno.freq.removed_geno.ratio_5.batches.csv
    Untracked:  data/ici.vs.eoi_marker.freq_low.geno.freq.removed_geno.ratio_5.batches_0.csv
    Untracked:  data/ici.vs.eoi_marker.freq_low.geno.freq.removed_geno.ratio_5.batches_52.csv
    Untracked:  data/ici.vs.eoi_marker.freq_low.geno.freq.removed_geno.ratio_5.batches_mis.csv
    Untracked:  data/ici.vs.eoi_marker.freq_low.geno.freq.removed_geno.ratio_5.batches_mis_0.csv
    Untracked:  data/ici.vs.eoi_marker.freq_low.geno.freq.removed_geno.ratio_5.batches_mis_52.csv
    Untracked:  data/ici.vs.eoi_marker.freq_low.geno.freq.removed_sample.outliers.removed_geno.ratio.csv
    Untracked:  data/ici.vs.eoi_marker.freq_low.geno.freq.removed_sample.outliers.removed_geno.ratio_5.batches.csv
    Untracked:  data/ici.vs.eoi_marker.freq_low.geno.freq.removed_sample.outliers.removed_geno.ratio_5.batches_0.csv
    Untracked:  data/ici.vs.eoi_marker.freq_low.geno.freq.removed_sample.outliers.removed_geno.ratio_5.batches_52.csv
    Untracked:  data/ici.vs.eoi_marker.freq_low.geno.freq.removed_sample.outliers.removed_geno.ratio_5.batches_mis.csv
    Untracked:  data/ici.vs.eoi_marker.freq_low.geno.freq.removed_sample.outliers.removed_geno.ratio_5.batches_mis_0.csv
    Untracked:  data/ici.vs.eoi_marker.freq_low.geno.freq.removed_sample.outliers.removed_geno.ratio_5.batches_mis_52.csv
    Untracked:  data/ici.vs.eoi_marker.freq_low.probs.freq.removed_geno.ratio.csv
    Untracked:  data/ici.vs.eoi_marker.freq_low.probs.freq.removed_geno.ratio_5.batches.csv
    Untracked:  data/ici.vs.eoi_marker.freq_low.probs.freq.removed_geno.ratio_5.batches_0.csv
    Untracked:  data/ici.vs.eoi_marker.freq_low.probs.freq.removed_geno.ratio_5.batches_52.csv
    Untracked:  data/ici.vs.eoi_marker.freq_low.probs.freq.removed_geno.ratio_5.batches_mis.csv
    Untracked:  data/ici.vs.eoi_marker.freq_low.probs.freq.removed_geno.ratio_5.batches_mis_0.csv
    Untracked:  data/ici.vs.eoi_marker.freq_low.probs.freq.removed_geno.ratio_5.batches_mis_52.csv
    Untracked:  data/ici.vs.eoi_marker.freq_low.probs.freq.removed_sample.outliers.removed_geno.ratio.csv
    Untracked:  data/ici.vs.eoi_marker.freq_low.probs.freq.removed_sample.outliers.removed_geno.ratio_5.batches.csv
    Untracked:  data/ici.vs.eoi_marker.freq_low.probs.freq.removed_sample.outliers.removed_geno.ratio_5.batches_0.csv
    Untracked:  data/ici.vs.eoi_marker.freq_low.probs.freq.removed_sample.outliers.removed_geno.ratio_5.batches_52.csv
    Untracked:  data/ici.vs.eoi_marker.freq_low.probs.freq.removed_sample.outliers.removed_geno.ratio_5.batches_mis.csv
    Untracked:  data/ici.vs.eoi_marker.freq_low.probs.freq.removed_sample.outliers.removed_geno.ratio_5.batches_mis_0.csv
    Untracked:  data/ici.vs.eoi_marker.freq_low.probs.freq.removed_sample.outliers.removed_geno.ratio_5.batches_mis_52.csv
    Untracked:  data/ici.vs.eoi_rz.age-additive.covariates_blup_sub_chr-7_peak.marker-UNC13239362_lod.drop-1.5_snpsqc_5.batches.csv
    Untracked:  data/ici.vs.eoi_rz.age-additive.covariates_genes_chr-7_peak.marker-UNC13239362_lod.drop-1.5_snpsqc_5.batches.csv
    Untracked:  data/ici.vs.eoi_rz.age-no.covariates_blup_sub_chr-10_peak.marker-JAX00292927_lod.drop-1.5_5.batches_0.csv
    Untracked:  data/ici.vs.eoi_rz.age-no.covariates_blup_sub_chr-10_peak.marker-UNC18311938_lod.drop-1.5_5.batches_0.csv
    Untracked:  data/ici.vs.eoi_rz.age-no.covariates_blup_sub_chr-10_peak.marker-UNC18343181_lod.drop-1.5_5.batches_0.csv
    Untracked:  data/ici.vs.eoi_rz.age-no.covariates_blup_sub_chr-10_peak.marker-UNC18376338_lod.drop-1.5_5.batches_0.csv
    Untracked:  data/ici.vs.eoi_rz.age-no.covariates_blup_sub_chr-10_peak.marker-UNC18376338_lod.drop-1.5_snpsqc_5.batches_0.csv
    Untracked:  data/ici.vs.eoi_rz.age-no.covariates_blup_sub_chr-10_peak.marker-UNC18376338_lod.drop-1.5_snpsqc_5.batches_mis_0.csv
    Untracked:  data/ici.vs.eoi_rz.age-no.covariates_blup_sub_chr-10_peak.marker-UNC18382395_lod.drop-1.5_snpsqc_dis_no-x_updated_5.batches_0.csv
    Untracked:  data/ici.vs.eoi_rz.age-no.covariates_blup_sub_chr-10_peak.marker-UNCHS028236_lod.drop-1.5_snpsqc_5.batches_mis_0.csv
    Untracked:  data/ici.vs.eoi_rz.age-no.covariates_blup_sub_chr-10_peak.marker-UNCHS028236_lod.drop-1.5_snpsqc_5.batches_mis_52.csv
    Untracked:  data/ici.vs.eoi_rz.age-no.covariates_blup_sub_chr-10_peak.marker-UNCHS028536_lod.drop-1.5_snpsqc_dis_no-x_updated_5.batches_mis_0.csv
    Untracked:  data/ici.vs.eoi_rz.age-no.covariates_blup_sub_chr-10_peak.marker-UNCHS029024_lod.drop-1.5_5.batches_0.csv
    Untracked:  data/ici.vs.eoi_rz.age-no.covariates_blup_sub_chr-11_peak.marker-UNC19970181_lod.drop-1.5_snpsqc_5.batches_mis_0.csv
    Untracked:  data/ici.vs.eoi_rz.age-no.covariates_blup_sub_chr-11_peak.marker-UNC19970181_lod.drop-1.5_snpsqc_5.batches_mis_52.csv
    Untracked:  data/ici.vs.eoi_rz.age-no.covariates_blup_sub_chr-12_peak.marker-ICR499_lod.drop-1.5_snpsqc_5.batches_mis_0.csv
    Untracked:  data/ici.vs.eoi_rz.age-no.covariates_blup_sub_chr-12_peak.marker-ICR499_lod.drop-1.5_snpsqc_5.batches_mis_52.csv
    Untracked:  data/ici.vs.eoi_rz.age-no.covariates_blup_sub_chr-13_peak.marker-UNCHS036773_lod.drop-1.5_snpsqc_5.batches_mis_0.csv
    Untracked:  data/ici.vs.eoi_rz.age-no.covariates_blup_sub_chr-13_peak.marker-UNCHS036773_lod.drop-1.5_snpsqc_5.batches_mis_52.csv
    Untracked:  data/ici.vs.eoi_rz.age-no.covariates_blup_sub_chr-14_peak.marker-UNC24056202_lod.drop-1.5_snpsqc_5.batches_mis_0.csv
    Untracked:  data/ici.vs.eoi_rz.age-no.covariates_blup_sub_chr-14_peak.marker-UNC24056202_lod.drop-1.5_snpsqc_5.batches_mis_52.csv
    Untracked:  data/ici.vs.eoi_rz.age-no.covariates_blup_sub_chr-15_peak.marker-UNC26070435_lod.drop-1.5_snpsqc_dis_no-x_updated_5.batches_mis_0.csv
    Untracked:  data/ici.vs.eoi_rz.age-no.covariates_blup_sub_chr-15_peak.marker-UNC26070435_lod.drop-1.5_snpsqc_dis_no-x_updated_5.batches_mis_52.csv
    Untracked:  data/ici.vs.eoi_rz.age-no.covariates_blup_sub_chr-17_peak.marker-UNCHS044241_lod.drop-1.5_snpsqc_5.batches_mis_0.csv
    Untracked:  data/ici.vs.eoi_rz.age-no.covariates_blup_sub_chr-17_peak.marker-UNCHS044241_lod.drop-1.5_snpsqc_5.batches_mis_52.csv
    Untracked:  data/ici.vs.eoi_rz.age-no.covariates_blup_sub_chr-17_peak.marker-UNCHS044241_lod.drop-1.5_snpsqc_dis_no-x_updated_5.batches_mis_0.csv
    Untracked:  data/ici.vs.eoi_rz.age-no.covariates_blup_sub_chr-17_peak.marker-UNCrs47191360_lod.drop-1.5_snpsqc_dis_no-x_updated_5.batches_mis_52.csv
    Untracked:  data/ici.vs.eoi_rz.age-no.covariates_blup_sub_chr-18_peak.marker-UNCHS045343_lod.drop-1.5_snpsqc_5.batches_mis_0.csv
    Untracked:  data/ici.vs.eoi_rz.age-no.covariates_blup_sub_chr-18_peak.marker-UNCHS045343_lod.drop-1.5_snpsqc_5.batches_mis_52.csv
    Untracked:  data/ici.vs.eoi_rz.age-no.covariates_blup_sub_chr-1_peak.marker-UNC228500_lod.drop-1.5_snpsqc_dis_no-x_updated_5.batches_52.csv
    Untracked:  data/ici.vs.eoi_rz.age-no.covariates_blup_sub_chr-1_peak.marker-UNC228500_lod.drop-1.5_snpsqc_dis_no-x_updated_5.batches_mis_52.csv
    Untracked:  data/ici.vs.eoi_rz.age-no.covariates_blup_sub_chr-2_peak.marker-UNC4609527_lod.drop-1.5_snpsqc_dis_no-x_updated_5.batches_mis_0.csv
    Untracked:  data/ici.vs.eoi_rz.age-no.covariates_blup_sub_chr-2_peak.marker-UNC4609527_lod.drop-1.5_snpsqc_dis_no-x_updated_5.batches_mis_52.csv
    Untracked:  data/ici.vs.eoi_rz.age-no.covariates_blup_sub_chr-3_peak.marker-ICR1338_lod.drop-1.5_5.batches_0.csv
    Untracked:  data/ici.vs.eoi_rz.age-no.covariates_blup_sub_chr-3_peak.marker-ICR1338_lod.drop-1.5_5.batches_52.csv
    Untracked:  data/ici.vs.eoi_rz.age-no.covariates_blup_sub_chr-3_peak.marker-ICR2047_lod.drop-1.5_5.batches_52.csv
    Untracked:  data/ici.vs.eoi_rz.age-no.covariates_blup_sub_chr-3_peak.marker-JAX00106957_lod.drop-1.5_5.batches_0.csv
    Untracked:  data/ici.vs.eoi_rz.age-no.covariates_blup_sub_chr-3_peak.marker-JAX00112499_lod.drop-1.5_5.batches_52.csv
    Untracked:  data/ici.vs.eoi_rz.age-no.covariates_blup_sub_chr-3_peak.marker-JAX00219554_lod.drop-1.5_5.batches_0.csv
    Untracked:  data/ici.vs.eoi_rz.age-no.covariates_blup_sub_chr-3_peak.marker-JAX00220631_lod.drop-1.5_5.batches_0.csv
    Untracked:  data/ici.vs.eoi_rz.age-no.covariates_blup_sub_chr-3_peak.marker-JAX00514662_lod.drop-1.5_5.batches_52.csv
    Untracked:  data/ici.vs.eoi_rz.age-no.covariates_blup_sub_chr-3_peak.marker-JAX00529707_lod.drop-1.5_5.batches_52.csv
    Untracked:  data/ici.vs.eoi_rz.age-no.covariates_blup_sub_chr-3_peak.marker-UNC4813186_lod.drop-1.5_5.batches_0.csv
    Untracked:  data/ici.vs.eoi_rz.age-no.covariates_blup_sub_chr-3_peak.marker-UNC4813186_lod.drop-1.5_5.batches_52.csv
    Untracked:  data/ici.vs.eoi_rz.age-no.covariates_blup_sub_chr-3_peak.marker-UNC4836550_lod.drop-1.5_5.batches_52.csv
    Untracked:  data/ici.vs.eoi_rz.age-no.covariates_blup_sub_chr-3_peak.marker-UNC4837069_lod.drop-1.5_5.batches_0.csv
    Untracked:  data/ici.vs.eoi_rz.age-no.covariates_blup_sub_chr-3_peak.marker-UNC4912039_lod.drop-1.5_5.batches_0.csv
    Untracked:  data/ici.vs.eoi_rz.age-no.covariates_blup_sub_chr-3_peak.marker-UNC4912039_lod.drop-1.5_5.batches_52.csv
    Untracked:  data/ici.vs.eoi_rz.age-no.covariates_blup_sub_chr-3_peak.marker-UNC4924377_lod.drop-1.5_5.batches_0.csv
    Untracked:  data/ici.vs.eoi_rz.age-no.covariates_blup_sub_chr-3_peak.marker-UNC4924377_lod.drop-1.5_5.batches_52.csv
    Untracked:  data/ici.vs.eoi_rz.age-no.covariates_blup_sub_chr-3_peak.marker-UNC4956618_lod.drop-1.5_5.batches_52.csv
    Untracked:  data/ici.vs.eoi_rz.age-no.covariates_blup_sub_chr-3_peak.marker-UNC4957168_lod.drop-1.5_5.batches_0.csv
    Untracked:  data/ici.vs.eoi_rz.age-no.covariates_blup_sub_chr-3_peak.marker-UNC5006948_lod.drop-1.5_5.batches_52.csv
    Untracked:  data/ici.vs.eoi_rz.age-no.covariates_blup_sub_chr-3_peak.marker-UNC5008656_lod.drop-1.5_5.batches_0.csv
    Untracked:  data/ici.vs.eoi_rz.age-no.covariates_blup_sub_chr-3_peak.marker-UNC5083168_lod.drop-1.5_5.batches_0.csv
    Untracked:  data/ici.vs.eoi_rz.age-no.covariates_blup_sub_chr-3_peak.marker-UNC5083168_lod.drop-1.5_5.batches_52.csv
    Untracked:  data/ici.vs.eoi_rz.age-no.covariates_blup_sub_chr-3_peak.marker-UNC5192582_lod.drop-1.5_5.batches_0.csv
    Untracked:  data/ici.vs.eoi_rz.age-no.covariates_blup_sub_chr-3_peak.marker-UNC5213888_lod.drop-1.5_snpsqc_5.batches_mis_0.csv
    Untracked:  data/ici.vs.eoi_rz.age-no.covariates_blup_sub_chr-3_peak.marker-UNC5213888_lod.drop-1.5_snpsqc_5.batches_mis_52.csv
    Untracked:  data/ici.vs.eoi_rz.age-no.covariates_blup_sub_chr-3_peak.marker-UNC5262472_lod.drop-1.5_5.batches_0.csv
    Untracked:  data/ici.vs.eoi_rz.age-no.covariates_blup_sub_chr-3_peak.marker-UNC5262472_lod.drop-1.5_5.batches_52.csv
    Untracked:  data/ici.vs.eoi_rz.age-no.covariates_blup_sub_chr-3_peak.marker-UNC5266664_lod.drop-1.5_5.batches_0.csv
    Untracked:  data/ici.vs.eoi_rz.age-no.covariates_blup_sub_chr-3_peak.marker-UNC5300878_lod.drop-1.5_5.batches_52.csv
    Untracked:  data/ici.vs.eoi_rz.age-no.covariates_blup_sub_chr-3_peak.marker-UNC5360055_lod.drop-1.5_5.batches_52.csv
    Untracked:  data/ici.vs.eoi_rz.age-no.covariates_blup_sub_chr-3_peak.marker-UNC5454839_lod.drop-1.5_5.batches_0.csv
    Untracked:  data/ici.vs.eoi_rz.age-no.covariates_blup_sub_chr-3_peak.marker-UNC5454839_lod.drop-1.5_5.batches_52.csv
    Untracked:  data/ici.vs.eoi_rz.age-no.covariates_blup_sub_chr-3_peak.marker-UNC5645844_lod.drop-1.5_5.batches_0.csv
    Untracked:  data/ici.vs.eoi_rz.age-no.covariates_blup_sub_chr-3_peak.marker-UNC5645844_lod.drop-1.5_5.batches_52.csv
    Untracked:  data/ici.vs.eoi_rz.age-no.covariates_blup_sub_chr-3_peak.marker-UNC5679155_lod.drop-1.5_5.batches_52.csv
    Untracked:  data/ici.vs.eoi_rz.age-no.covariates_blup_sub_chr-3_peak.marker-UNC5808246_lod.drop-1.5_5.batches_52.csv
    Untracked:  data/ici.vs.eoi_rz.age-no.covariates_blup_sub_chr-3_peak.marker-UNC5945286_lod.drop-1.5_5.batches_52.csv
    Untracked:  data/ici.vs.eoi_rz.age-no.covariates_blup_sub_chr-3_peak.marker-UNC6227106_lod.drop-1.5_5.batches_52.csv
    Untracked:  data/ici.vs.eoi_rz.age-no.covariates_blup_sub_chr-3_peak.marker-UNCHS008182_lod.drop-1.5_5.batches_52.csv
    Untracked:  data/ici.vs.eoi_rz.age-no.covariates_blup_sub_chr-3_peak.marker-UNCHS008286_lod.drop-1.5_5.batches_0.csv
    Untracked:  data/ici.vs.eoi_rz.age-no.covariates_blup_sub_chr-3_peak.marker-UNCHS008286_lod.drop-1.5_5.batches_52.csv
    Untracked:  data/ici.vs.eoi_rz.age-no.covariates_blup_sub_chr-3_peak.marker-UNCHS008487_lod.drop-1.5_snpsqc_5.batches_0.csv
    Untracked:  data/ici.vs.eoi_rz.age-no.covariates_blup_sub_chr-3_peak.marker-UNCHS008487_lod.drop-1.5_snpsqc_5.batches_52.csv
    Untracked:  data/ici.vs.eoi_rz.age-no.covariates_blup_sub_chr-3_peak.marker-UNCHS008487_lod.drop-1.5_snpsqc_dis_no-x_updated_5.batches_0.csv
    Untracked:  data/ici.vs.eoi_rz.age-no.covariates_blup_sub_chr-3_peak.marker-UNCHS008487_lod.drop-1.5_snpsqc_dis_no-x_updated_5.batches_52.csv
    Untracked:  data/ici.vs.eoi_rz.age-no.covariates_blup_sub_chr-3_peak.marker-UNCHS008487_lod.drop-1.5_snpsqc_dis_no-x_updated_5.batches_mis_0.csv
    Untracked:  data/ici.vs.eoi_rz.age-no.covariates_blup_sub_chr-3_peak.marker-UNCHS008487_lod.drop-1.5_snpsqc_dis_no-x_updated_5.batches_mis_52.csv
    Untracked:  data/ici.vs.eoi_rz.age-no.covariates_blup_sub_chr-3_peak.marker-UNCHS008511_lod.drop-1.5_snpsqc_5.batches_mis_0.csv
    Untracked:  data/ici.vs.eoi_rz.age-no.covariates_blup_sub_chr-3_peak.marker-UNCHS008511_lod.drop-1.5_snpsqc_5.batches_mis_52.csv
    Untracked:  data/ici.vs.eoi_rz.age-no.covariates_blup_sub_chr-3_peak.marker-UNCHS008595_lod.drop-1.5_5.batches_52.csv
    Untracked:  data/ici.vs.eoi_rz.age-no.covariates_blup_sub_chr-3_peak.marker-UNCHS008613_lod.drop-1.5_5.batches_0.csv
    Untracked:  data/ici.vs.eoi_rz.age-no.covariates_blup_sub_chr-3_peak.marker-UNCHS008613_lod.drop-1.5_5.batches_52.csv
    Untracked:  data/ici.vs.eoi_rz.age-no.covariates_blup_sub_chr-3_peak.marker-UNCHS008815_lod.drop-1.5_5.batches_0.csv
    Untracked:  data/ici.vs.eoi_rz.age-no.covariates_blup_sub_chr-3_peak.marker-UNCHS008815_lod.drop-1.5_5.batches_52.csv
    Untracked:  data/ici.vs.eoi_rz.age-no.covariates_blup_sub_chr-3_peak.marker-UNCHS009067_lod.drop-1.5_5.batches_0.csv
    Untracked:  data/ici.vs.eoi_rz.age-no.covariates_blup_sub_chr-3_peak.marker-UNCHS009383_lod.drop-1.5_5.batches_52.csv
    Untracked:  data/ici.vs.eoi_rz.age-no.covariates_blup_sub_chr-3_peak.marker-UNCHS009693_lod.drop-1.5_5.batches_52.csv
    Untracked:  data/ici.vs.eoi_rz.age-no.covariates_blup_sub_chr-3_peak.marker-UNCJPD001246_lod.drop-1.5_5.batches_52.csv
    Untracked:  data/ici.vs.eoi_rz.age-no.covariates_blup_sub_chr-3_peak.marker-UNCJPD001341_lod.drop-1.5_5.batches_52.csv
    Untracked:  data/ici.vs.eoi_rz.age-no.covariates_blup_sub_chr-3_peak.marker-sanger2623_lod.drop-1.5_5.batches_52.csv
    Untracked:  data/ici.vs.eoi_rz.age-no.covariates_blup_sub_chr-4_peak.marker-UNCHS013125_lod.drop-1.5_snpsqc_dis_no-x_updated_5.batches_52.csv
    Untracked:  data/ici.vs.eoi_rz.age-no.covariates_blup_sub_chr-4_peak.marker-UNCHS013125_lod.drop-1.5_snpsqc_dis_no-x_updated_5.batches_mis_52.csv
    Untracked:  data/ici.vs.eoi_rz.age-no.covariates_blup_sub_chr-6_peak.marker-UNCHS018170_lod.drop-1.5_snpsqc_5.batches_0.csv
    Untracked:  data/ici.vs.eoi_rz.age-no.covariates_blup_sub_chr-6_peak.marker-UNCHS018231_lod.drop-1.5_snpsqc_5.batches_mis_0.csv
    Untracked:  data/ici.vs.eoi_rz.age-no.covariates_blup_sub_chr-7_peak.marker-UNCHS020066_lod.drop-1.5_snpsqc_5.batches_mis_0.csv
    Untracked:  data/ici.vs.eoi_rz.age-no.covariates_blup_sub_chr-7_peak.marker-UNCHS020066_lod.drop-1.5_snpsqc_5.batches_mis_52.csv
    Untracked:  data/ici.vs.eoi_rz.age-no.covariates_blup_sub_chr-7_peak.marker-UNCHS020066_lod.drop-1.5_snpsqc_dis_no-x_updated_5.batches_mis_0.csv
    Untracked:  data/ici.vs.eoi_rz.age-no.covariates_blup_sub_chr-7_peak.marker-UNCHS020066_lod.drop-1.5_snpsqc_dis_no-x_updated_5.batches_mis_52.csv
    Untracked:  data/ici.vs.eoi_rz.age-no.covariates_blup_sub_chr-8_peak.marker-UNC15524531_lod.drop-1.5_snpsqc_5.batches_mis_0.csv
    Untracked:  data/ici.vs.eoi_rz.age-no.covariates_blup_sub_chr-8_peak.marker-UNC15524531_lod.drop-1.5_snpsqc_5.batches_mis_52.csv
    Untracked:  data/ici.vs.eoi_rz.age-no.covariates_blup_sub_chr-8_peak.marker-UNC15524531_lod.drop-1.5_snpsqc_dis_no-x_updated_5.batches_mis_0.csv
    Untracked:  data/ici.vs.eoi_rz.age-no.covariates_blup_sub_chr-8_peak.marker-UNC15524531_lod.drop-1.5_snpsqc_dis_no-x_updated_5.batches_mis_52.csv
    Untracked:  data/ici.vs.eoi_rz.age-no.covariates_blup_sub_chr-9_peak.marker-UNC17203597_lod.drop-1.5_snpsqc_5.batches_mis_0.csv
    Untracked:  data/ici.vs.eoi_rz.age-no.covariates_blup_sub_chr-9_peak.marker-UNC17203597_lod.drop-1.5_snpsqc_5.batches_mis_52.csv
    Untracked:  data/ici.vs.eoi_rz.age-no.covariates_blup_sub_chr-X_peak.marker-UNCHS048314_lod.drop-1.5_snpsqc_5.batches_mis_0.csv
    Untracked:  data/ici.vs.eoi_rz.age-no.covariates_blup_sub_chr-X_peak.marker-UNCHS048314_lod.drop-1.5_snpsqc_5.batches_mis_52.csv
    Untracked:  data/ici.vs.eoi_rz.age-no.covariates_blup_sub_chr-X_peak.marker-UNCHS048314_lod.drop-1.5_snpsqc_dis_no-x_updated_5.batches_mis_0.csv
    Untracked:  data/ici.vs.eoi_rz.age-no.covariates_blup_sub_chr-X_peak.marker-UNCHS048314_lod.drop-1.5_snpsqc_dis_no-x_updated_5.batches_mis_52.csv
    Untracked:  data/ici.vs.eoi_rz.age-no.covariates_genes_chr-10_peak.marker-JAX00292927_lod.drop-1.5_5.batches_0.csv
    Untracked:  data/ici.vs.eoi_rz.age-no.covariates_genes_chr-10_peak.marker-UNC18311938_lod.drop-1.5_5.batches_0.csv
    Untracked:  data/ici.vs.eoi_rz.age-no.covariates_genes_chr-10_peak.marker-UNC18343181_lod.drop-1.5_5.batches_0.csv
    Untracked:  data/ici.vs.eoi_rz.age-no.covariates_genes_chr-10_peak.marker-UNC18376338_lod.drop-1.5_5.batches_0.csv
    Untracked:  data/ici.vs.eoi_rz.age-no.covariates_genes_chr-10_peak.marker-UNC18376338_lod.drop-1.5_snpsqc_5.batches_0.csv
    Untracked:  data/ici.vs.eoi_rz.age-no.covariates_genes_chr-10_peak.marker-UNC18376338_lod.drop-1.5_snpsqc_5.batches_mis_0.csv
    Untracked:  data/ici.vs.eoi_rz.age-no.covariates_genes_chr-10_peak.marker-UNC18382395_lod.drop-1.5_snpsqc_dis_no-x_updated_5.batches_0.csv
    Untracked:  data/ici.vs.eoi_rz.age-no.covariates_genes_chr-10_peak.marker-UNCHS028236_lod.drop-1.5_snpsqc_5.batches_mis_0.csv
    Untracked:  data/ici.vs.eoi_rz.age-no.covariates_genes_chr-10_peak.marker-UNCHS028236_lod.drop-1.5_snpsqc_5.batches_mis_52.csv
    Untracked:  data/ici.vs.eoi_rz.age-no.covariates_genes_chr-10_peak.marker-UNCHS028536_lod.drop-1.5_snpsqc_dis_no-x_updated_5.batches_mis_0.csv
    Untracked:  data/ici.vs.eoi_rz.age-no.covariates_genes_chr-10_peak.marker-UNCHS029024_lod.drop-1.5_5.batches_0.csv
    Untracked:  data/ici.vs.eoi_rz.age-no.covariates_genes_chr-11_peak.marker-UNC19970181_lod.drop-1.5_snpsqc_5.batches_mis_0.csv
    Untracked:  data/ici.vs.eoi_rz.age-no.covariates_genes_chr-11_peak.marker-UNC19970181_lod.drop-1.5_snpsqc_5.batches_mis_52.csv
    Untracked:  data/ici.vs.eoi_rz.age-no.covariates_genes_chr-12_peak.marker-ICR499_lod.drop-1.5_snpsqc_5.batches_mis_0.csv
    Untracked:  data/ici.vs.eoi_rz.age-no.covariates_genes_chr-12_peak.marker-ICR499_lod.drop-1.5_snpsqc_5.batches_mis_52.csv
    Untracked:  data/ici.vs.eoi_rz.age-no.covariates_genes_chr-13_peak.marker-UNCHS036773_lod.drop-1.5_snpsqc_5.batches_mis_0.csv
    Untracked:  data/ici.vs.eoi_rz.age-no.covariates_genes_chr-13_peak.marker-UNCHS036773_lod.drop-1.5_snpsqc_5.batches_mis_52.csv
    Untracked:  data/ici.vs.eoi_rz.age-no.covariates_genes_chr-14_peak.marker-UNC24056202_lod.drop-1.5_snpsqc_5.batches_mis_0.csv
    Untracked:  data/ici.vs.eoi_rz.age-no.covariates_genes_chr-14_peak.marker-UNC24056202_lod.drop-1.5_snpsqc_5.batches_mis_52.csv
    Untracked:  data/ici.vs.eoi_rz.age-no.covariates_genes_chr-15_peak.marker-UNC26070435_lod.drop-1.5_snpsqc_dis_no-x_updated_5.batches_mis_0.csv
    Untracked:  data/ici.vs.eoi_rz.age-no.covariates_genes_chr-15_peak.marker-UNC26070435_lod.drop-1.5_snpsqc_dis_no-x_updated_5.batches_mis_52.csv
    Untracked:  data/ici.vs.eoi_rz.age-no.covariates_genes_chr-17_peak.marker-UNCHS044241_lod.drop-1.5_snpsqc_5.batches_mis_0.csv
    Untracked:  data/ici.vs.eoi_rz.age-no.covariates_genes_chr-17_peak.marker-UNCHS044241_lod.drop-1.5_snpsqc_5.batches_mis_52.csv
    Untracked:  data/ici.vs.eoi_rz.age-no.covariates_genes_chr-17_peak.marker-UNCHS044241_lod.drop-1.5_snpsqc_dis_no-x_updated_5.batches_mis_0.csv
    Untracked:  data/ici.vs.eoi_rz.age-no.covariates_genes_chr-17_peak.marker-UNCrs47191360_lod.drop-1.5_snpsqc_dis_no-x_updated_5.batches_mis_52.csv
    Untracked:  data/ici.vs.eoi_rz.age-no.covariates_genes_chr-18_peak.marker-UNCHS045343_lod.drop-1.5_snpsqc_5.batches_mis_0.csv
    Untracked:  data/ici.vs.eoi_rz.age-no.covariates_genes_chr-18_peak.marker-UNCHS045343_lod.drop-1.5_snpsqc_5.batches_mis_52.csv
    Untracked:  data/ici.vs.eoi_rz.age-no.covariates_genes_chr-1_peak.marker-UNC228500_lod.drop-1.5_snpsqc_dis_no-x_updated_5.batches_52.csv
    Untracked:  data/ici.vs.eoi_rz.age-no.covariates_genes_chr-1_peak.marker-UNC228500_lod.drop-1.5_snpsqc_dis_no-x_updated_5.batches_mis_52.csv
    Untracked:  data/ici.vs.eoi_rz.age-no.covariates_genes_chr-2_peak.marker-UNC4609527_lod.drop-1.5_snpsqc_dis_no-x_updated_5.batches_mis_0.csv
    Untracked:  data/ici.vs.eoi_rz.age-no.covariates_genes_chr-2_peak.marker-UNC4609527_lod.drop-1.5_snpsqc_dis_no-x_updated_5.batches_mis_52.csv
    Untracked:  data/ici.vs.eoi_rz.age-no.covariates_genes_chr-3_peak.marker-ICR1338_lod.drop-1.5_5.batches_0.csv
    Untracked:  data/ici.vs.eoi_rz.age-no.covariates_genes_chr-3_peak.marker-ICR1338_lod.drop-1.5_5.batches_52.csv
    Untracked:  data/ici.vs.eoi_rz.age-no.covariates_genes_chr-3_peak.marker-ICR2047_lod.drop-1.5_5.batches_52.csv
    Untracked:  data/ici.vs.eoi_rz.age-no.covariates_genes_chr-3_peak.marker-JAX00106957_lod.drop-1.5_5.batches_0.csv
    Untracked:  data/ici.vs.eoi_rz.age-no.covariates_genes_chr-3_peak.marker-JAX00112499_lod.drop-1.5_5.batches_52.csv
    Untracked:  data/ici.vs.eoi_rz.age-no.covariates_genes_chr-3_peak.marker-JAX00219554_lod.drop-1.5_5.batches_0.csv
    Untracked:  data/ici.vs.eoi_rz.age-no.covariates_genes_chr-3_peak.marker-JAX00220631_lod.drop-1.5_5.batches_0.csv
    Untracked:  data/ici.vs.eoi_rz.age-no.covariates_genes_chr-3_peak.marker-JAX00514662_lod.drop-1.5_5.batches_52.csv
    Untracked:  data/ici.vs.eoi_rz.age-no.covariates_genes_chr-3_peak.marker-JAX00529707_lod.drop-1.5_5.batches_52.csv
    Untracked:  data/ici.vs.eoi_rz.age-no.covariates_genes_chr-3_peak.marker-UNC4813186_lod.drop-1.5_5.batches_0.csv
    Untracked:  data/ici.vs.eoi_rz.age-no.covariates_genes_chr-3_peak.marker-UNC4813186_lod.drop-1.5_5.batches_52.csv
    Untracked:  data/ici.vs.eoi_rz.age-no.covariates_genes_chr-3_peak.marker-UNC4836550_lod.drop-1.5_5.batches_52.csv
    Untracked:  data/ici.vs.eoi_rz.age-no.covariates_genes_chr-3_peak.marker-UNC4837069_lod.drop-1.5_5.batches_0.csv
    Untracked:  data/ici.vs.eoi_rz.age-no.covariates_genes_chr-3_peak.marker-UNC4912039_lod.drop-1.5_5.batches_0.csv
    Untracked:  data/ici.vs.eoi_rz.age-no.covariates_genes_chr-3_peak.marker-UNC4912039_lod.drop-1.5_5.batches_52.csv
    Untracked:  data/ici.vs.eoi_rz.age-no.covariates_genes_chr-3_peak.marker-UNC4924377_lod.drop-1.5_5.batches_0.csv
    Untracked:  data/ici.vs.eoi_rz.age-no.covariates_genes_chr-3_peak.marker-UNC4924377_lod.drop-1.5_5.batches_52.csv
    Untracked:  data/ici.vs.eoi_rz.age-no.covariates_genes_chr-3_peak.marker-UNC4956618_lod.drop-1.5_5.batches_52.csv
    Untracked:  data/ici.vs.eoi_rz.age-no.covariates_genes_chr-3_peak.marker-UNC4957168_lod.drop-1.5_5.batches_0.csv
    Untracked:  data/ici.vs.eoi_rz.age-no.covariates_genes_chr-3_peak.marker-UNC5006948_lod.drop-1.5_5.batches_52.csv
    Untracked:  data/ici.vs.eoi_rz.age-no.covariates_genes_chr-3_peak.marker-UNC5008656_lod.drop-1.5_5.batches_0.csv
    Untracked:  data/ici.vs.eoi_rz.age-no.covariates_genes_chr-3_peak.marker-UNC5083168_lod.drop-1.5_5.batches_0.csv
    Untracked:  data/ici.vs.eoi_rz.age-no.covariates_genes_chr-3_peak.marker-UNC5083168_lod.drop-1.5_5.batches_52.csv
    Untracked:  data/ici.vs.eoi_rz.age-no.covariates_genes_chr-3_peak.marker-UNC5192582_lod.drop-1.5_5.batches_0.csv
    Untracked:  data/ici.vs.eoi_rz.age-no.covariates_genes_chr-3_peak.marker-UNC5213888_lod.drop-1.5_snpsqc_5.batches_mis_0.csv
    Untracked:  data/ici.vs.eoi_rz.age-no.covariates_genes_chr-3_peak.marker-UNC5213888_lod.drop-1.5_snpsqc_5.batches_mis_52.csv
    Untracked:  data/ici.vs.eoi_rz.age-no.covariates_genes_chr-3_peak.marker-UNC5262472_lod.drop-1.5_5.batches_0.csv
    Untracked:  data/ici.vs.eoi_rz.age-no.covariates_genes_chr-3_peak.marker-UNC5262472_lod.drop-1.5_5.batches_52.csv
    Untracked:  data/ici.vs.eoi_rz.age-no.covariates_genes_chr-3_peak.marker-UNC5266664_lod.drop-1.5_5.batches_0.csv
    Untracked:  data/ici.vs.eoi_rz.age-no.covariates_genes_chr-3_peak.marker-UNC5300878_lod.drop-1.5_5.batches_52.csv
    Untracked:  data/ici.vs.eoi_rz.age-no.covariates_genes_chr-3_peak.marker-UNC5360055_lod.drop-1.5_5.batches_52.csv
    Untracked:  data/ici.vs.eoi_rz.age-no.covariates_genes_chr-3_peak.marker-UNC5454839_lod.drop-1.5_5.batches_0.csv
    Untracked:  data/ici.vs.eoi_rz.age-no.covariates_genes_chr-3_peak.marker-UNC5454839_lod.drop-1.5_5.batches_52.csv
    Untracked:  data/ici.vs.eoi_rz.age-no.covariates_genes_chr-3_peak.marker-UNC5645844_lod.drop-1.5_5.batches_0.csv
    Untracked:  data/ici.vs.eoi_rz.age-no.covariates_genes_chr-3_peak.marker-UNC5645844_lod.drop-1.5_5.batches_52.csv
    Untracked:  data/ici.vs.eoi_rz.age-no.covariates_genes_chr-3_peak.marker-UNC5679155_lod.drop-1.5_5.batches_52.csv
    Untracked:  data/ici.vs.eoi_rz.age-no.covariates_genes_chr-3_peak.marker-UNC5808246_lod.drop-1.5_5.batches_52.csv
    Untracked:  data/ici.vs.eoi_rz.age-no.covariates_genes_chr-3_peak.marker-UNC5945286_lod.drop-1.5_5.batches_52.csv
    Untracked:  data/ici.vs.eoi_rz.age-no.covariates_genes_chr-3_peak.marker-UNC6227106_lod.drop-1.5_5.batches_52.csv
    Untracked:  data/ici.vs.eoi_rz.age-no.covariates_genes_chr-3_peak.marker-UNCHS008182_lod.drop-1.5_5.batches_52.csv
    Untracked:  data/ici.vs.eoi_rz.age-no.covariates_genes_chr-3_peak.marker-UNCHS008286_lod.drop-1.5_5.batches_0.csv
    Untracked:  data/ici.vs.eoi_rz.age-no.covariates_genes_chr-3_peak.marker-UNCHS008286_lod.drop-1.5_5.batches_52.csv
    Untracked:  data/ici.vs.eoi_rz.age-no.covariates_genes_chr-3_peak.marker-UNCHS008487_lod.drop-1.5_snpsqc_5.batches_0.csv
    Untracked:  data/ici.vs.eoi_rz.age-no.covariates_genes_chr-3_peak.marker-UNCHS008487_lod.drop-1.5_snpsqc_5.batches_52.csv
    Untracked:  data/ici.vs.eoi_rz.age-no.covariates_genes_chr-3_peak.marker-UNCHS008487_lod.drop-1.5_snpsqc_dis_no-x_updated_5.batches_0.csv
    Untracked:  data/ici.vs.eoi_rz.age-no.covariates_genes_chr-3_peak.marker-UNCHS008487_lod.drop-1.5_snpsqc_dis_no-x_updated_5.batches_52.csv
    Untracked:  data/ici.vs.eoi_rz.age-no.covariates_genes_chr-3_peak.marker-UNCHS008487_lod.drop-1.5_snpsqc_dis_no-x_updated_5.batches_mis_0.csv
    Untracked:  data/ici.vs.eoi_rz.age-no.covariates_genes_chr-3_peak.marker-UNCHS008487_lod.drop-1.5_snpsqc_dis_no-x_updated_5.batches_mis_52.csv
    Untracked:  data/ici.vs.eoi_rz.age-no.covariates_genes_chr-3_peak.marker-UNCHS008511_lod.drop-1.5_snpsqc_5.batches_mis_0.csv
    Untracked:  data/ici.vs.eoi_rz.age-no.covariates_genes_chr-3_peak.marker-UNCHS008511_lod.drop-1.5_snpsqc_5.batches_mis_52.csv
    Untracked:  data/ici.vs.eoi_rz.age-no.covariates_genes_chr-3_peak.marker-UNCHS008595_lod.drop-1.5_5.batches_52.csv
    Untracked:  data/ici.vs.eoi_rz.age-no.covariates_genes_chr-3_peak.marker-UNCHS008613_lod.drop-1.5_5.batches_0.csv
    Untracked:  data/ici.vs.eoi_rz.age-no.covariates_genes_chr-3_peak.marker-UNCHS008613_lod.drop-1.5_5.batches_52.csv
    Untracked:  data/ici.vs.eoi_rz.age-no.covariates_genes_chr-3_peak.marker-UNCHS008815_lod.drop-1.5_5.batches_0.csv
    Untracked:  data/ici.vs.eoi_rz.age-no.covariates_genes_chr-3_peak.marker-UNCHS008815_lod.drop-1.5_5.batches_52.csv
    Untracked:  data/ici.vs.eoi_rz.age-no.covariates_genes_chr-3_peak.marker-UNCHS009067_lod.drop-1.5_5.batches_0.csv
    Untracked:  data/ici.vs.eoi_rz.age-no.covariates_genes_chr-3_peak.marker-UNCHS009383_lod.drop-1.5_5.batches_52.csv
    Untracked:  data/ici.vs.eoi_rz.age-no.covariates_genes_chr-3_peak.marker-UNCHS009693_lod.drop-1.5_5.batches_52.csv
    Untracked:  data/ici.vs.eoi_rz.age-no.covariates_genes_chr-3_peak.marker-UNCJPD001246_lod.drop-1.5_5.batches_52.csv
    Untracked:  data/ici.vs.eoi_rz.age-no.covariates_genes_chr-3_peak.marker-UNCJPD001341_lod.drop-1.5_5.batches_52.csv
    Untracked:  data/ici.vs.eoi_rz.age-no.covariates_genes_chr-3_peak.marker-sanger2623_lod.drop-1.5_5.batches_52.csv
    Untracked:  data/ici.vs.eoi_rz.age-no.covariates_genes_chr-4_peak.marker-UNCHS013125_lod.drop-1.5_snpsqc_dis_no-x_updated_5.batches_52.csv
    Untracked:  data/ici.vs.eoi_rz.age-no.covariates_genes_chr-4_peak.marker-UNCHS013125_lod.drop-1.5_snpsqc_dis_no-x_updated_5.batches_mis_52.csv
    Untracked:  data/ici.vs.eoi_rz.age-no.covariates_genes_chr-6_peak.marker-UNCHS018170_lod.drop-1.5_snpsqc_5.batches_0.csv
    Untracked:  data/ici.vs.eoi_rz.age-no.covariates_genes_chr-6_peak.marker-UNCHS018231_lod.drop-1.5_snpsqc_5.batches_mis_0.csv
    Untracked:  data/ici.vs.eoi_rz.age-no.covariates_genes_chr-7_peak.marker-UNCHS020066_lod.drop-1.5_snpsqc_5.batches_mis_0.csv
    Untracked:  data/ici.vs.eoi_rz.age-no.covariates_genes_chr-7_peak.marker-UNCHS020066_lod.drop-1.5_snpsqc_5.batches_mis_52.csv
    Untracked:  data/ici.vs.eoi_rz.age-no.covariates_genes_chr-7_peak.marker-UNCHS020066_lod.drop-1.5_snpsqc_dis_no-x_updated_5.batches_mis_0.csv
    Untracked:  data/ici.vs.eoi_rz.age-no.covariates_genes_chr-7_peak.marker-UNCHS020066_lod.drop-1.5_snpsqc_dis_no-x_updated_5.batches_mis_52.csv
    Untracked:  data/ici.vs.eoi_rz.age-no.covariates_genes_chr-8_peak.marker-UNC15524531_lod.drop-1.5_snpsqc_5.batches_mis_0.csv
    Untracked:  data/ici.vs.eoi_rz.age-no.covariates_genes_chr-8_peak.marker-UNC15524531_lod.drop-1.5_snpsqc_5.batches_mis_52.csv
    Untracked:  data/ici.vs.eoi_rz.age-no.covariates_genes_chr-8_peak.marker-UNC15524531_lod.drop-1.5_snpsqc_dis_no-x_updated_5.batches_mis_0.csv
    Untracked:  data/ici.vs.eoi_rz.age-no.covariates_genes_chr-8_peak.marker-UNC15524531_lod.drop-1.5_snpsqc_dis_no-x_updated_5.batches_mis_52.csv
    Untracked:  data/ici.vs.eoi_rz.age-no.covariates_genes_chr-9_peak.marker-UNC17203597_lod.drop-1.5_snpsqc_5.batches_mis_0.csv
    Untracked:  data/ici.vs.eoi_rz.age-no.covariates_genes_chr-9_peak.marker-UNC17203597_lod.drop-1.5_snpsqc_5.batches_mis_52.csv
    Untracked:  data/ici.vs.eoi_rz.age-no.covariates_genes_chr-X_peak.marker-UNCHS048314_lod.drop-1.5_snpsqc_5.batches_mis_0.csv
    Untracked:  data/ici.vs.eoi_rz.age-no.covariates_genes_chr-X_peak.marker-UNCHS048314_lod.drop-1.5_snpsqc_5.batches_mis_52.csv
    Untracked:  data/ici.vs.eoi_rz.age-no.covariates_genes_chr-X_peak.marker-UNCHS048314_lod.drop-1.5_snpsqc_dis_no-x_updated_5.batches_mis_0.csv
    Untracked:  data/ici.vs.eoi_rz.age-no.covariates_genes_chr-X_peak.marker-UNCHS048314_lod.drop-1.5_snpsqc_dis_no-x_updated_5.batches_mis_52.csv
    Untracked:  data/ici.vs.eoi_sample.genos_marker.freq_low.geno.freq.removed.csv
    Untracked:  data/ici.vs.eoi_sample.genos_marker.freq_low.geno.freq.removed_5.batches.csv
    Untracked:  data/ici.vs.eoi_sample.genos_marker.freq_low.geno.freq.removed_5.batches_mis.csv
    Untracked:  data/ici.vs.eoi_sample.genos_marker.freq_low.geno.freq.removed_sample.outliers.removed.csv
    Untracked:  data/ici.vs.eoi_sample.genos_marker.freq_low.geno.freq.removed_sample.outliers.removed_5.batches.csv
    Untracked:  data/ici.vs.eoi_sample.genos_marker.freq_low.geno.freq.removed_sample.outliers.removed_5.batches_mis.csv
    Untracked:  data/ici.vs.eoi_sample.genos_marker.freq_low.probs.freq.removed.csv
    Untracked:  data/ici.vs.eoi_sample.genos_marker.freq_low.probs.freq.removed_5.batches.csv
    Untracked:  data/ici.vs.eoi_sample.genos_marker.freq_low.probs.freq.removed_5.batches_mis.csv
    Untracked:  data/ici.vs.eoi_sample.genos_marker.freq_low.probs.freq.removed_sample.outliers.removed.csv
    Untracked:  data/ici.vs.eoi_sample.genos_marker.freq_low.probs.freq.removed_sample.outliers.removed_5.batches.csv
    Untracked:  data/ici.vs.eoi_sample.genos_marker.freq_low.probs.freq.removed_sample.outliers.removed_5.batches_mis.csv
    Untracked:  data/ici.vs.eoi_scanone_5.batches.Rdata
    Untracked:  data/ici.vs.eoi_scanone_5.batches_mis.Rdata
    Untracked:  data/ici.vs.eoi_scanone_snpsqc_5.batches.Rdata
    Untracked:  data/ici.vs.eoi_scanone_snpsqc_5.batches_mis.Rdata
    Untracked:  data/ici.vs.eoi_scanone_snpsqc_dis_no-x_updated_5.batches.Rdata
    Untracked:  data/ici.vs.pbs_age.of.onset-no.covariates_blup_sub_chr-14_peak.marker-UNC24056202_lod.drop-1.5_5.batches_mis.csv
    Untracked:  data/ici.vs.pbs_age.of.onset-no.covariates_blup_sub_chr-14_peak.marker-UNC24056202_lod.drop-1.5_snpsqc_5.batches_mis.csv
    Untracked:  data/ici.vs.pbs_age.of.onset-no.covariates_blup_sub_chr-17_peak.marker-UNCJPD006614_lod.drop-1.5_snpsqc_5.batches_mis.csv
    Untracked:  data/ici.vs.pbs_age.of.onset-no.covariates_blup_sub_chr-18_peak.marker-UNC28655293_lod.drop-1.5_5.batches_mis.csv
    Untracked:  data/ici.vs.pbs_age.of.onset-no.covariates_blup_sub_chr-18_peak.marker-UNCHS045343_lod.drop-1.5_snpsqc_5.batches_mis.csv
    Untracked:  data/ici.vs.pbs_age.of.onset-no.covariates_blup_sub_chr-5_peak.marker-UNC10044126_lod.drop-1.5_5.batches_mis.csv
    Untracked:  data/ici.vs.pbs_age.of.onset-no.covariates_blup_sub_chr-5_peak.marker-UNC8675939_lod.drop-1.5_snpsqc_5.batches_mis.csv
    Untracked:  data/ici.vs.pbs_age.of.onset-no.covariates_blup_sub_chr-8_peak.marker-JAX00667121_lod.drop-1.5_5.batches_mis.csv
    Untracked:  data/ici.vs.pbs_age.of.onset-no.covariates_blup_sub_chr-8_peak.marker-UNC15524531_lod.drop-1.5_snpsqc_5.batches_mis.csv
    Untracked:  data/ici.vs.pbs_age.of.onset-no.covariates_blup_sub_chr-X_peak.marker-UNCHS048314_lod.drop-1.5_snpsqc_5.batches_mis.csv
    Untracked:  data/ici.vs.pbs_age.of.onset-no.covariates_genes_chr-14_peak.marker-UNC24056202_lod.drop-1.5_5.batches_mis.csv
    Untracked:  data/ici.vs.pbs_age.of.onset-no.covariates_genes_chr-14_peak.marker-UNC24056202_lod.drop-1.5_snpsqc_5.batches_mis.csv
    Untracked:  data/ici.vs.pbs_age.of.onset-no.covariates_genes_chr-17_peak.marker-UNCJPD006614_lod.drop-1.5_snpsqc_5.batches_mis.csv
    Untracked:  data/ici.vs.pbs_age.of.onset-no.covariates_genes_chr-18_peak.marker-UNC28655293_lod.drop-1.5_5.batches_mis.csv
    Untracked:  data/ici.vs.pbs_age.of.onset-no.covariates_genes_chr-18_peak.marker-UNCHS045343_lod.drop-1.5_snpsqc_5.batches_mis.csv
    Untracked:  data/ici.vs.pbs_age.of.onset-no.covariates_genes_chr-5_peak.marker-UNC10044126_lod.drop-1.5_5.batches_mis.csv
    Untracked:  data/ici.vs.pbs_age.of.onset-no.covariates_genes_chr-5_peak.marker-UNC8675939_lod.drop-1.5_snpsqc_5.batches_mis.csv
    Untracked:  data/ici.vs.pbs_age.of.onset-no.covariates_genes_chr-8_peak.marker-JAX00667121_lod.drop-1.5_5.batches_mis.csv
    Untracked:  data/ici.vs.pbs_age.of.onset-no.covariates_genes_chr-8_peak.marker-UNC15524531_lod.drop-1.5_snpsqc_5.batches_mis.csv
    Untracked:  data/ici.vs.pbs_age.of.onset-no.covariates_genes_chr-X_peak.marker-UNCHS048314_lod.drop-1.5_snpsqc_5.batches_mis.csv
    Untracked:  data/ici.vs.pbs_bad.markers_snpsqc_5.batches_mis.csv
    Untracked:  data/ici.vs.pbs_blup.full_chr-10_lod.drop-1.5_5.batches.csv
    Untracked:  data/ici.vs.pbs_blup.full_chr-10_snpsqc_5.batches.csv
    Untracked:  data/ici.vs.pbs_blup.full_chr-10_snpsqc_5.batches_mis.csv
    Untracked:  data/ici.vs.pbs_blup.full_chr-10_snpsqc_dis_no-x_updated_5.batches.csv
    Untracked:  data/ici.vs.pbs_blup.full_chr-10_snpsqc_dis_no-x_updated_5.batches_mis.csv
    Untracked:  data/ici.vs.pbs_blup.full_chr-11_lod.drop-1.5_5.batches.csv
    Untracked:  data/ici.vs.pbs_blup.full_chr-11_snpsqc_5.batches.csv
    Untracked:  data/ici.vs.pbs_blup.full_chr-11_snpsqc_5.batches_mis.csv
    Untracked:  data/ici.vs.pbs_blup.full_chr-11_snpsqc_dis_no-x_updated_5.batches.csv
    Untracked:  data/ici.vs.pbs_blup.full_chr-11_snpsqc_dis_no-x_updated_5.batches_mis.csv
    Untracked:  data/ici.vs.pbs_blup.full_chr-12_lod.drop-1.5_5.batches.csv
    Untracked:  data/ici.vs.pbs_blup.full_chr-12_snpsqc_5.batches.csv
    Untracked:  data/ici.vs.pbs_blup.full_chr-12_snpsqc_5.batches_mis.csv
    Untracked:  data/ici.vs.pbs_blup.full_chr-12_snpsqc_dis_no-x_updated_5.batches.csv
    Untracked:  data/ici.vs.pbs_blup.full_chr-12_snpsqc_dis_no-x_updated_5.batches_mis.csv
    Untracked:  data/ici.vs.pbs_blup.full_chr-13_lod.drop-1.5_5.batches.csv
    Untracked:  data/ici.vs.pbs_blup.full_chr-13_snpsqc_5.batches.csv
    Untracked:  data/ici.vs.pbs_blup.full_chr-13_snpsqc_5.batches_mis.csv
    Untracked:  data/ici.vs.pbs_blup.full_chr-13_snpsqc_dis_no-x_updated_5.batches.csv
    Untracked:  data/ici.vs.pbs_blup.full_chr-13_snpsqc_dis_no-x_updated_5.batches_mis.csv
    Untracked:  data/ici.vs.pbs_blup.full_chr-14_lod.drop-1.5_5.batches.csv
    Untracked:  data/ici.vs.pbs_blup.full_chr-14_snpsqc_5.batches.csv
    Untracked:  data/ici.vs.pbs_blup.full_chr-14_snpsqc_5.batches_mis.csv
    Untracked:  data/ici.vs.pbs_blup.full_chr-14_snpsqc_dis_no-x_updated_5.batches.csv
    Untracked:  data/ici.vs.pbs_blup.full_chr-14_snpsqc_dis_no-x_updated_5.batches_mis.csv
    Untracked:  data/ici.vs.pbs_blup.full_chr-15_lod.drop-1.5_5.batches.csv
    Untracked:  data/ici.vs.pbs_blup.full_chr-15_snpsqc_5.batches.csv
    Untracked:  data/ici.vs.pbs_blup.full_chr-15_snpsqc_5.batches_mis.csv
    Untracked:  data/ici.vs.pbs_blup.full_chr-15_snpsqc_dis_no-x_updated_5.batches.csv
    Untracked:  data/ici.vs.pbs_blup.full_chr-15_snpsqc_dis_no-x_updated_5.batches_mis.csv
    Untracked:  data/ici.vs.pbs_blup.full_chr-16_lod.drop-1.5_5.batches.csv
    Untracked:  data/ici.vs.pbs_blup.full_chr-16_snpsqc_5.batches.csv
    Untracked:  data/ici.vs.pbs_blup.full_chr-16_snpsqc_5.batches_mis.csv
    Untracked:  data/ici.vs.pbs_blup.full_chr-16_snpsqc_dis_no-x_updated_5.batches.csv
    Untracked:  data/ici.vs.pbs_blup.full_chr-16_snpsqc_dis_no-x_updated_5.batches_mis.csv
    Untracked:  data/ici.vs.pbs_blup.full_chr-17_lod.drop-1.5_5.batches.csv
    Untracked:  data/ici.vs.pbs_blup.full_chr-17_snpsqc_5.batches.csv
    Untracked:  data/ici.vs.pbs_blup.full_chr-17_snpsqc_5.batches_mis.csv
    Untracked:  data/ici.vs.pbs_blup.full_chr-17_snpsqc_dis_no-x_updated_5.batches.csv
    Untracked:  data/ici.vs.pbs_blup.full_chr-17_snpsqc_dis_no-x_updated_5.batches_mis.csv
    Untracked:  data/ici.vs.pbs_blup.full_chr-18_lod.drop-1.5_5.batches.csv
    Untracked:  data/ici.vs.pbs_blup.full_chr-18_snpsqc_5.batches.csv
    Untracked:  data/ici.vs.pbs_blup.full_chr-18_snpsqc_5.batches_mis.csv
    Untracked:  data/ici.vs.pbs_blup.full_chr-18_snpsqc_dis_no-x_updated_5.batches.csv
    Untracked:  data/ici.vs.pbs_blup.full_chr-18_snpsqc_dis_no-x_updated_5.batches_mis.csv
    Untracked:  data/ici.vs.pbs_blup.full_chr-19_lod.drop-1.5_5.batches.csv
    Untracked:  data/ici.vs.pbs_blup.full_chr-19_snpsqc_5.batches.csv
    Untracked:  data/ici.vs.pbs_blup.full_chr-19_snpsqc_5.batches_mis.csv
    Untracked:  data/ici.vs.pbs_blup.full_chr-19_snpsqc_dis_no-x_updated_5.batches.csv
    Untracked:  data/ici.vs.pbs_blup.full_chr-19_snpsqc_dis_no-x_updated_5.batches_mis.csv
    Untracked:  data/ici.vs.pbs_blup.full_chr-1_lod.drop-1.5_5.batches.csv
    Untracked:  data/ici.vs.pbs_blup.full_chr-1_snpsqc_5.batches.csv
    Untracked:  data/ici.vs.pbs_blup.full_chr-1_snpsqc_5.batches_mis.csv
    Untracked:  data/ici.vs.pbs_blup.full_chr-1_snpsqc_dis_no-x_updated_5.batches.csv
    Untracked:  data/ici.vs.pbs_blup.full_chr-1_snpsqc_dis_no-x_updated_5.batches_mis.csv
    Untracked:  data/ici.vs.pbs_blup.full_chr-2_lod.drop-1.5_5.batches.csv
    Untracked:  data/ici.vs.pbs_blup.full_chr-2_snpsqc_5.batches.csv
    Untracked:  data/ici.vs.pbs_blup.full_chr-2_snpsqc_5.batches_mis.csv
    Untracked:  data/ici.vs.pbs_blup.full_chr-2_snpsqc_dis_no-x_updated_5.batches.csv
    Untracked:  data/ici.vs.pbs_blup.full_chr-2_snpsqc_dis_no-x_updated_5.batches_mis.csv
    Untracked:  data/ici.vs.pbs_blup.full_chr-3_lod.drop-1.5_5.batches.csv
    Untracked:  data/ici.vs.pbs_blup.full_chr-3_snpsqc_5.batches.csv
    Untracked:  data/ici.vs.pbs_blup.full_chr-3_snpsqc_5.batches_mis.csv
    Untracked:  data/ici.vs.pbs_blup.full_chr-3_snpsqc_dis_no-x_updated_5.batches.csv
    Untracked:  data/ici.vs.pbs_blup.full_chr-3_snpsqc_dis_no-x_updated_5.batches_mis.csv
    Untracked:  data/ici.vs.pbs_blup.full_chr-4_lod.drop-1.5_5.batches.csv
    Untracked:  data/ici.vs.pbs_blup.full_chr-4_snpsqc_5.batches.csv
    Untracked:  data/ici.vs.pbs_blup.full_chr-4_snpsqc_5.batches_mis.csv
    Untracked:  data/ici.vs.pbs_blup.full_chr-4_snpsqc_dis_no-x_updated_5.batches.csv
    Untracked:  data/ici.vs.pbs_blup.full_chr-4_snpsqc_dis_no-x_updated_5.batches_mis.csv
    Untracked:  data/ici.vs.pbs_blup.full_chr-5_lod.drop-1.5_5.batches.csv
    Untracked:  data/ici.vs.pbs_blup.full_chr-5_snpsqc_5.batches.csv
    Untracked:  data/ici.vs.pbs_blup.full_chr-5_snpsqc_5.batches_mis.csv
    Untracked:  data/ici.vs.pbs_blup.full_chr-5_snpsqc_dis_no-x_updated_5.batches.csv
    Untracked:  data/ici.vs.pbs_blup.full_chr-5_snpsqc_dis_no-x_updated_5.batches_mis.csv
    Untracked:  data/ici.vs.pbs_blup.full_chr-6_lod.drop-1.5_5.batches.csv
    Untracked:  data/ici.vs.pbs_blup.full_chr-6_snpsqc_5.batches.csv
    Untracked:  data/ici.vs.pbs_blup.full_chr-6_snpsqc_5.batches_mis.csv
    Untracked:  data/ici.vs.pbs_blup.full_chr-6_snpsqc_dis_no-x_updated_5.batches.csv
    Untracked:  data/ici.vs.pbs_blup.full_chr-6_snpsqc_dis_no-x_updated_5.batches_mis.csv
    Untracked:  data/ici.vs.pbs_blup.full_chr-7_lod.drop-1.5_5.batches.csv
    Untracked:  data/ici.vs.pbs_blup.full_chr-7_snpsqc_5.batches.csv
    Untracked:  data/ici.vs.pbs_blup.full_chr-7_snpsqc_5.batches_mis.csv
    Untracked:  data/ici.vs.pbs_blup.full_chr-7_snpsqc_dis_no-x_updated_5.batches.csv
    Untracked:  data/ici.vs.pbs_blup.full_chr-7_snpsqc_dis_no-x_updated_5.batches_mis.csv
    Untracked:  data/ici.vs.pbs_blup.full_chr-8_lod.drop-1.5_5.batches.csv
    Untracked:  data/ici.vs.pbs_blup.full_chr-8_snpsqc_5.batches.csv
    Untracked:  data/ici.vs.pbs_blup.full_chr-8_snpsqc_5.batches_mis.csv
    Untracked:  data/ici.vs.pbs_blup.full_chr-8_snpsqc_dis_no-x_updated_5.batches.csv
    Untracked:  data/ici.vs.pbs_blup.full_chr-8_snpsqc_dis_no-x_updated_5.batches_mis.csv
    Untracked:  data/ici.vs.pbs_blup.full_chr-9_lod.drop-1.5_5.batches.csv
    Untracked:  data/ici.vs.pbs_blup.full_chr-9_snpsqc_5.batches.csv
    Untracked:  data/ici.vs.pbs_blup.full_chr-9_snpsqc_5.batches_mis.csv
    Untracked:  data/ici.vs.pbs_blup.full_chr-9_snpsqc_dis_no-x_updated_5.batches.csv
    Untracked:  data/ici.vs.pbs_blup.full_chr-9_snpsqc_dis_no-x_updated_5.batches_mis.csv
    Untracked:  data/ici.vs.pbs_blup.full_chr-X_lod.drop-1.5_5.batches.csv
    Untracked:  data/ici.vs.pbs_blup.full_chr-X_snpsqc_5.batches.csv
    Untracked:  data/ici.vs.pbs_blup.full_chr-X_snpsqc_5.batches_mis.csv
    Untracked:  data/ici.vs.pbs_blup.full_chr-X_snpsqc_dis_no-x_updated_5.batches.csv
    Untracked:  data/ici.vs.pbs_blup.full_chr-X_snpsqc_dis_no-x_updated_5.batches_mis.csv
    Untracked:  data/ici.vs.pbs_blup.qc_chr-10_5.batches.csv
    Untracked:  data/ici.vs.pbs_blup.qc_chr-10_5.batches_mis.csv
    Untracked:  data/ici.vs.pbs_blup.qc_chr-10_lod.drop-1.5_5.batches.csv
    Untracked:  data/ici.vs.pbs_blup.qc_chr-10_lod.drop-1.5_5.batches_mis.csv
    Untracked:  data/ici.vs.pbs_blup.qc_chr-10_snpsqc_5.batches.csv
    Untracked:  data/ici.vs.pbs_blup.qc_chr-10_snpsqc_5.batches_mis.csv
    Untracked:  data/ici.vs.pbs_blup.qc_chr-10_snpsqc_dis_no-x_updated_5.batches.csv
    Untracked:  data/ici.vs.pbs_blup.qc_chr-10_snpsqc_dis_no-x_updated_5.batches_mis.csv
    Untracked:  data/ici.vs.pbs_blup.qc_chr-11_5.batches.csv
    Untracked:  data/ici.vs.pbs_blup.qc_chr-11_5.batches_mis.csv
    Untracked:  data/ici.vs.pbs_blup.qc_chr-11_lod.drop-1.5_5.batches.csv
    Untracked:  data/ici.vs.pbs_blup.qc_chr-11_lod.drop-1.5_5.batches_mis.csv
    Untracked:  data/ici.vs.pbs_blup.qc_chr-11_snpsqc_5.batches.csv
    Untracked:  data/ici.vs.pbs_blup.qc_chr-11_snpsqc_5.batches_mis.csv
    Untracked:  data/ici.vs.pbs_blup.qc_chr-11_snpsqc_dis_no-x_updated_5.batches.csv
    Untracked:  data/ici.vs.pbs_blup.qc_chr-11_snpsqc_dis_no-x_updated_5.batches_mis.csv
    Untracked:  data/ici.vs.pbs_blup.qc_chr-12_5.batches.csv
    Untracked:  data/ici.vs.pbs_blup.qc_chr-12_5.batches_mis.csv
    Untracked:  data/ici.vs.pbs_blup.qc_chr-12_lod.drop-1.5_5.batches.csv
    Untracked:  data/ici.vs.pbs_blup.qc_chr-12_lod.drop-1.5_5.batches_mis.csv
    Untracked:  data/ici.vs.pbs_blup.qc_chr-12_snpsqc_5.batches.csv
    Untracked:  data/ici.vs.pbs_blup.qc_chr-12_snpsqc_5.batches_mis.csv
    Untracked:  data/ici.vs.pbs_blup.qc_chr-12_snpsqc_dis_no-x_updated_5.batches.csv
    Untracked:  data/ici.vs.pbs_blup.qc_chr-12_snpsqc_dis_no-x_updated_5.batches_mis.csv
    Untracked:  data/ici.vs.pbs_blup.qc_chr-13_5.batches.csv
    Untracked:  data/ici.vs.pbs_blup.qc_chr-13_5.batches_mis.csv
    Untracked:  data/ici.vs.pbs_blup.qc_chr-13_lod.drop-1.5_5.batches.csv
    Untracked:  data/ici.vs.pbs_blup.qc_chr-13_lod.drop-1.5_5.batches_mis.csv
    Untracked:  data/ici.vs.pbs_blup.qc_chr-13_snpsqc_5.batches.csv
    Untracked:  data/ici.vs.pbs_blup.qc_chr-13_snpsqc_5.batches_mis.csv
    Untracked:  data/ici.vs.pbs_blup.qc_chr-13_snpsqc_dis_no-x_updated_5.batches.csv
    Untracked:  data/ici.vs.pbs_blup.qc_chr-13_snpsqc_dis_no-x_updated_5.batches_mis.csv
    Untracked:  data/ici.vs.pbs_blup.qc_chr-14_5.batches.csv
    Untracked:  data/ici.vs.pbs_blup.qc_chr-14_5.batches_mis.csv
    Untracked:  data/ici.vs.pbs_blup.qc_chr-14_lod.drop-1.5_5.batches.csv
    Untracked:  data/ici.vs.pbs_blup.qc_chr-14_snpsqc_5.batches.csv
    Untracked:  data/ici.vs.pbs_blup.qc_chr-14_snpsqc_5.batches_mis.csv
    Untracked:  data/ici.vs.pbs_blup.qc_chr-14_snpsqc_dis_no-x_updated_5.batches.csv
    Untracked:  data/ici.vs.pbs_blup.qc_chr-14_snpsqc_dis_no-x_updated_5.batches_mis.csv
    Untracked:  data/ici.vs.pbs_blup.qc_chr-15_5.batches.csv
    Untracked:  data/ici.vs.pbs_blup.qc_chr-15_5.batches_mis.csv
    Untracked:  data/ici.vs.pbs_blup.qc_chr-15_lod.drop-1.5_5.batches.csv
    Untracked:  data/ici.vs.pbs_blup.qc_chr-15_snpsqc_5.batches.csv
    Untracked:  data/ici.vs.pbs_blup.qc_chr-15_snpsqc_5.batches_mis.csv
    Untracked:  data/ici.vs.pbs_blup.qc_chr-16_5.batches.csv
    Untracked:  data/ici.vs.pbs_blup.qc_chr-16_5.batches_mis.csv
    Untracked:  data/ici.vs.pbs_blup.qc_chr-16_lod.drop-1.5_5.batches.csv
    Untracked:  data/ici.vs.pbs_blup.qc_chr-16_snpsqc_5.batches.csv
    Untracked:  data/ici.vs.pbs_blup.qc_chr-16_snpsqc_5.batches_mis.csv
    Untracked:  data/ici.vs.pbs_blup.qc_chr-16_snpsqc_dis_no-x_updated_5.batches.csv
    Untracked:  data/ici.vs.pbs_blup.qc_chr-16_snpsqc_dis_no-x_updated_5.batches_mis.csv
    Untracked:  data/ici.vs.pbs_blup.qc_chr-17_5.batches.csv
    Untracked:  data/ici.vs.pbs_blup.qc_chr-17_5.batches_mis.csv
    Untracked:  data/ici.vs.pbs_blup.qc_chr-17_lod.drop-1.5_5.batches.csv
    Untracked:  data/ici.vs.pbs_blup.qc_chr-17_snpsqc_5.batches.csv
    Untracked:  data/ici.vs.pbs_blup.qc_chr-17_snpsqc_5.batches_mis.csv
    Untracked:  data/ici.vs.pbs_blup.qc_chr-18_5.batches.csv
    Untracked:  data/ici.vs.pbs_blup.qc_chr-18_5.batches_mis.csv
    Untracked:  data/ici.vs.pbs_blup.qc_chr-18_lod.drop-1.5_5.batches.csv
    Untracked:  data/ici.vs.pbs_blup.qc_chr-18_snpsqc_5.batches.csv
    Untracked:  data/ici.vs.pbs_blup.qc_chr-18_snpsqc_5.batches_mis.csv
    Untracked:  data/ici.vs.pbs_blup.qc_chr-18_snpsqc_dis_no-x_updated_5.batches.csv
    Untracked:  data/ici.vs.pbs_blup.qc_chr-18_snpsqc_dis_no-x_updated_5.batches_mis.csv
    Untracked:  data/ici.vs.pbs_blup.qc_chr-19_5.batches.csv
    Untracked:  data/ici.vs.pbs_blup.qc_chr-19_5.batches_mis.csv
    Untracked:  data/ici.vs.pbs_blup.qc_chr-19_lod.drop-1.5_5.batches.csv
    Untracked:  data/ici.vs.pbs_blup.qc_chr-19_snpsqc_5.batches.csv
    Untracked:  data/ici.vs.pbs_blup.qc_chr-19_snpsqc_5.batches_mis.csv
    Untracked:  data/ici.vs.pbs_blup.qc_chr-19_snpsqc_dis_no-x_updated_5.batches.csv
    Untracked:  data/ici.vs.pbs_blup.qc_chr-19_snpsqc_dis_no-x_updated_5.batches_mis.csv
    Untracked:  data/ici.vs.pbs_blup.qc_chr-1_5.batches.csv
    Untracked:  data/ici.vs.pbs_blup.qc_chr-1_5.batches_mis.csv
    Untracked:  data/ici.vs.pbs_blup.qc_chr-1_lod.drop-1.5_5.batches.csv
    Untracked:  data/ici.vs.pbs_blup.qc_chr-1_lod.drop-1.5_5.batches_mis.csv
    Untracked:  data/ici.vs.pbs_blup.qc_chr-1_snpsqc_5.batches.csv
    Untracked:  data/ici.vs.pbs_blup.qc_chr-1_snpsqc_5.batches_mis.csv
    Untracked:  data/ici.vs.pbs_blup.qc_chr-1_snpsqc_dis_no-x_updated_5.batches.csv
    Untracked:  data/ici.vs.pbs_blup.qc_chr-1_snpsqc_dis_no-x_updated_5.batches_mis.csv
    Untracked:  data/ici.vs.pbs_blup.qc_chr-2_5.batches.csv
    Untracked:  data/ici.vs.pbs_blup.qc_chr-2_5.batches_mis.csv
    Untracked:  data/ici.vs.pbs_blup.qc_chr-2_lod.drop-1.5_5.batches.csv
    Untracked:  data/ici.vs.pbs_blup.qc_chr-2_lod.drop-1.5_5.batches_mis.csv
    Untracked:  data/ici.vs.pbs_blup.qc_chr-2_snpsqc_5.batches.csv
    Untracked:  data/ici.vs.pbs_blup.qc_chr-2_snpsqc_5.batches_mis.csv
    Untracked:  data/ici.vs.pbs_blup.qc_chr-2_snpsqc_dis_no-x_updated_5.batches.csv
    Untracked:  data/ici.vs.pbs_blup.qc_chr-2_snpsqc_dis_no-x_updated_5.batches_mis.csv
    Untracked:  data/ici.vs.pbs_blup.qc_chr-3_5.batches.csv
    Untracked:  data/ici.vs.pbs_blup.qc_chr-3_5.batches_mis.csv
    Untracked:  data/ici.vs.pbs_blup.qc_chr-3_lod.drop-1.5_5.batches.csv
    Untracked:  data/ici.vs.pbs_blup.qc_chr-3_lod.drop-1.5_5.batches_mis.csv
    Untracked:  data/ici.vs.pbs_blup.qc_chr-3_snpsqc_5.batches.csv
    Untracked:  data/ici.vs.pbs_blup.qc_chr-3_snpsqc_5.batches_mis.csv
    Untracked:  data/ici.vs.pbs_blup.qc_chr-3_snpsqc_dis_no-x_updated_5.batches.csv
    Untracked:  data/ici.vs.pbs_blup.qc_chr-3_snpsqc_dis_no-x_updated_5.batches_mis.csv
    Untracked:  data/ici.vs.pbs_blup.qc_chr-4_5.batches.csv
    Untracked:  data/ici.vs.pbs_blup.qc_chr-4_5.batches_mis.csv
    Untracked:  data/ici.vs.pbs_blup.qc_chr-4_lod.drop-1.5_5.batches.csv
    Untracked:  data/ici.vs.pbs_blup.qc_chr-4_lod.drop-1.5_5.batches_mis.csv
    Untracked:  data/ici.vs.pbs_blup.qc_chr-4_snpsqc_5.batches.csv
    Untracked:  data/ici.vs.pbs_blup.qc_chr-4_snpsqc_5.batches_mis.csv
    Untracked:  data/ici.vs.pbs_blup.qc_chr-4_snpsqc_dis_no-x_updated_5.batches.csv
    Untracked:  data/ici.vs.pbs_blup.qc_chr-4_snpsqc_dis_no-x_updated_5.batches_mis.csv
    Untracked:  data/ici.vs.pbs_blup.qc_chr-5_5.batches.csv
    Untracked:  data/ici.vs.pbs_blup.qc_chr-5_5.batches_mis.csv
    Untracked:  data/ici.vs.pbs_blup.qc_chr-5_lod.drop-1.5_5.batches.csv
    Untracked:  data/ici.vs.pbs_blup.qc_chr-5_lod.drop-1.5_5.batches_mis.csv
    Untracked:  data/ici.vs.pbs_blup.qc_chr-5_snpsqc_5.batches.csv
    Untracked:  data/ici.vs.pbs_blup.qc_chr-5_snpsqc_5.batches_mis.csv
    Untracked:  data/ici.vs.pbs_blup.qc_chr-5_snpsqc_dis_no-x_updated_5.batches.csv
    Untracked:  data/ici.vs.pbs_blup.qc_chr-5_snpsqc_dis_no-x_updated_5.batches_mis.csv
    Untracked:  data/ici.vs.pbs_blup.qc_chr-6_5.batches.csv
    Untracked:  data/ici.vs.pbs_blup.qc_chr-6_5.batches_mis.csv
    Untracked:  data/ici.vs.pbs_blup.qc_chr-6_lod.drop-1.5_5.batches.csv
    Untracked:  data/ici.vs.pbs_blup.qc_chr-6_lod.drop-1.5_5.batches_mis.csv
    Untracked:  data/ici.vs.pbs_blup.qc_chr-6_snpsqc_5.batches.csv
    Untracked:  data/ici.vs.pbs_blup.qc_chr-6_snpsqc_5.batches_mis.csv
    Untracked:  data/ici.vs.pbs_blup.qc_chr-6_snpsqc_dis_no-x_updated_5.batches.csv
    Untracked:  data/ici.vs.pbs_blup.qc_chr-6_snpsqc_dis_no-x_updated_5.batches_mis.csv
    Untracked:  data/ici.vs.pbs_blup.qc_chr-7_5.batches.csv
    Untracked:  data/ici.vs.pbs_blup.qc_chr-7_5.batches_mis.csv
    Untracked:  data/ici.vs.pbs_blup.qc_chr-7_lod.drop-1.5_5.batches.csv
    Untracked:  data/ici.vs.pbs_blup.qc_chr-7_lod.drop-1.5_5.batches_mis.csv
    Untracked:  data/ici.vs.pbs_blup.qc_chr-7_snpsqc_5.batches.csv
    Untracked:  data/ici.vs.pbs_blup.qc_chr-7_snpsqc_5.batches_mis.csv
    Untracked:  data/ici.vs.pbs_blup.qc_chr-7_snpsqc_dis_no-x_updated_5.batches.csv
    Untracked:  data/ici.vs.pbs_blup.qc_chr-7_snpsqc_dis_no-x_updated_5.batches_mis.csv
    Untracked:  data/ici.vs.pbs_blup.qc_chr-8_5.batches.csv
    Untracked:  data/ici.vs.pbs_blup.qc_chr-8_5.batches_mis.csv
    Untracked:  data/ici.vs.pbs_blup.qc_chr-8_lod.drop-1.5_5.batches.csv
    Untracked:  data/ici.vs.pbs_blup.qc_chr-8_lod.drop-1.5_5.batches_mis.csv
    Untracked:  data/ici.vs.pbs_blup.qc_chr-8_snpsqc_5.batches.csv
    Untracked:  data/ici.vs.pbs_blup.qc_chr-8_snpsqc_5.batches_mis.csv
    Untracked:  data/ici.vs.pbs_blup.qc_chr-8_snpsqc_dis_no-x_updated_5.batches.csv
    Untracked:  data/ici.vs.pbs_blup.qc_chr-8_snpsqc_dis_no-x_updated_5.batches_mis.csv
    Untracked:  data/ici.vs.pbs_blup.qc_chr-9_5.batches.csv
    Untracked:  data/ici.vs.pbs_blup.qc_chr-9_5.batches_mis.csv
    Untracked:  data/ici.vs.pbs_blup.qc_chr-9_lod.drop-1.5_5.batches.csv
    Untracked:  data/ici.vs.pbs_blup.qc_chr-9_lod.drop-1.5_5.batches_mis.csv
    Untracked:  data/ici.vs.pbs_blup.qc_chr-9_snpsqc_5.batches.csv
    Untracked:  data/ici.vs.pbs_blup.qc_chr-9_snpsqc_5.batches_mis.csv
    Untracked:  data/ici.vs.pbs_blup.qc_chr-9_snpsqc_dis_no-x_updated_5.batches.csv
    Untracked:  data/ici.vs.pbs_blup.qc_chr-9_snpsqc_dis_no-x_updated_5.batches_mis.csv
    Untracked:  data/ici.vs.pbs_blup.qc_chr-X_5.batches.csv
    Untracked:  data/ici.vs.pbs_blup.qc_chr-X_5.batches_mis.csv
    Untracked:  data/ici.vs.pbs_blup.qc_chr-X_lod.drop-1.5_5.batches.csv
    Untracked:  data/ici.vs.pbs_blup.qc_chr-X_snpsqc_5.batches.csv
    Untracked:  data/ici.vs.pbs_blup.qc_chr-X_snpsqc_5.batches_mis.csv
    Untracked:  data/ici.vs.pbs_blup.qc_chr-X_snpsqc_dis_no-x_updated_5.batches.csv
    Untracked:  data/ici.vs.pbs_blup.qc_chr-X_snpsqc_dis_no-x_updated_5.batches_mis.csv
    Untracked:  data/ici.vs.pbs_blup_chr-10_lod.drop-1.5_5.batches.csv
    Untracked:  data/ici.vs.pbs_blup_chr-10_lod.drop-1.5_snpsqc_5.batches.csv
    Untracked:  data/ici.vs.pbs_blup_chr-10_lod.drop-1.5_snpsqc_5.batches_mis.csv
    Untracked:  data/ici.vs.pbs_blup_chr-10_lod.drop-1.5_snpsqc_dis_no-x_updated_5.batches.csv
    Untracked:  data/ici.vs.pbs_blup_chr-10_lod.drop-1.5_snpsqc_dis_no-x_updated_5.batches_mis.csv
    Untracked:  data/ici.vs.pbs_blup_chr-11_lod.drop-1.5_5.batches.csv
    Untracked:  data/ici.vs.pbs_blup_chr-11_lod.drop-1.5_snpsqc_5.batches.csv
    Untracked:  data/ici.vs.pbs_blup_chr-11_lod.drop-1.5_snpsqc_5.batches_mis.csv
    Untracked:  data/ici.vs.pbs_blup_chr-11_lod.drop-1.5_snpsqc_dis_no-x_updated_5.batches.csv
    Untracked:  data/ici.vs.pbs_blup_chr-11_lod.drop-1.5_snpsqc_dis_no-x_updated_5.batches_mis.csv
    Untracked:  data/ici.vs.pbs_blup_chr-12_lod.drop-1.5_5.batches.csv
    Untracked:  data/ici.vs.pbs_blup_chr-12_lod.drop-1.5_snpsqc_5.batches.csv
    Untracked:  data/ici.vs.pbs_blup_chr-12_lod.drop-1.5_snpsqc_5.batches_mis.csv
    Untracked:  data/ici.vs.pbs_blup_chr-12_lod.drop-1.5_snpsqc_dis_no-x_updated_5.batches.csv
    Untracked:  data/ici.vs.pbs_blup_chr-12_lod.drop-1.5_snpsqc_dis_no-x_updated_5.batches_mis.csv
    Untracked:  data/ici.vs.pbs_blup_chr-13_lod.drop-1.5_5.batches.csv
    Untracked:  data/ici.vs.pbs_blup_chr-13_lod.drop-1.5_snpsqc_5.batches.csv
    Untracked:  data/ici.vs.pbs_blup_chr-13_lod.drop-1.5_snpsqc_5.batches_mis.csv
    Untracked:  data/ici.vs.pbs_blup_chr-13_lod.drop-1.5_snpsqc_dis_no-x_updated_5.batches.csv
    Untracked:  data/ici.vs.pbs_blup_chr-13_lod.drop-1.5_snpsqc_dis_no-x_updated_5.batches_mis.csv
    Untracked:  data/ici.vs.pbs_blup_chr-14_lod.drop-1.5_5.batches.csv
    Untracked:  data/ici.vs.pbs_blup_chr-14_lod.drop-1.5_snpsqc_5.batches.csv
    Untracked:  data/ici.vs.pbs_blup_chr-14_lod.drop-1.5_snpsqc_5.batches_mis.csv
    Untracked:  data/ici.vs.pbs_blup_chr-14_lod.drop-1.5_snpsqc_dis_no-x_updated_5.batches.csv
    Untracked:  data/ici.vs.pbs_blup_chr-14_lod.drop-1.5_snpsqc_dis_no-x_updated_5.batches_mis.csv
    Untracked:  data/ici.vs.pbs_blup_chr-15_lod.drop-1.5_5.batches.csv
    Untracked:  data/ici.vs.pbs_blup_chr-15_lod.drop-1.5_snpsqc_5.batches.csv
    Untracked:  data/ici.vs.pbs_blup_chr-15_lod.drop-1.5_snpsqc_5.batches_mis.csv
    Untracked:  data/ici.vs.pbs_blup_chr-16_lod.drop-1.5_5.batches.csv
    Untracked:  data/ici.vs.pbs_blup_chr-16_lod.drop-1.5_snpsqc_5.batches.csv
    Untracked:  data/ici.vs.pbs_blup_chr-16_lod.drop-1.5_snpsqc_5.batches_mis.csv
    Untracked:  data/ici.vs.pbs_blup_chr-16_lod.drop-1.5_snpsqc_dis_no-x_updated_5.batches.csv
    Untracked:  data/ici.vs.pbs_blup_chr-16_lod.drop-1.5_snpsqc_dis_no-x_updated_5.batches_mis.csv
    Untracked:  data/ici.vs.pbs_blup_chr-17_lod.drop-1.5_5.batches.csv
    Untracked:  data/ici.vs.pbs_blup_chr-17_lod.drop-1.5_snpsqc_5.batches.csv
    Untracked:  data/ici.vs.pbs_blup_chr-17_lod.drop-1.5_snpsqc_5.batches_mis.csv
    Untracked:  data/ici.vs.pbs_blup_chr-18_lod.drop-1.5_5.batches.csv
    Untracked:  data/ici.vs.pbs_blup_chr-18_lod.drop-1.5_snpsqc_5.batches.csv
    Untracked:  data/ici.vs.pbs_blup_chr-18_lod.drop-1.5_snpsqc_5.batches_mis.csv
    Untracked:  data/ici.vs.pbs_blup_chr-18_lod.drop-1.5_snpsqc_dis_no-x_updated_5.batches.csv
    Untracked:  data/ici.vs.pbs_blup_chr-18_lod.drop-1.5_snpsqc_dis_no-x_updated_5.batches_mis.csv
    Untracked:  data/ici.vs.pbs_blup_chr-19_lod.drop-1.5_5.batches.csv
    Untracked:  data/ici.vs.pbs_blup_chr-19_lod.drop-1.5_snpsqc_5.batches.csv
    Untracked:  data/ici.vs.pbs_blup_chr-19_lod.drop-1.5_snpsqc_5.batches_mis.csv
    Untracked:  data/ici.vs.pbs_blup_chr-19_lod.drop-1.5_snpsqc_dis_no-x_updated_5.batches.csv
    Untracked:  data/ici.vs.pbs_blup_chr-19_lod.drop-1.5_snpsqc_dis_no-x_updated_5.batches_mis.csv
    Untracked:  data/ici.vs.pbs_blup_chr-1_lod.drop-1.5_5.batches.csv
    Untracked:  data/ici.vs.pbs_blup_chr-1_lod.drop-1.5_snpsqc_5.batches.csv
    Untracked:  data/ici.vs.pbs_blup_chr-1_lod.drop-1.5_snpsqc_5.batches_mis.csv
    Untracked:  data/ici.vs.pbs_blup_chr-1_lod.drop-1.5_snpsqc_dis_no-x_updated_5.batches.csv
    Untracked:  data/ici.vs.pbs_blup_chr-1_lod.drop-1.5_snpsqc_dis_no-x_updated_5.batches_mis.csv
    Untracked:  data/ici.vs.pbs_blup_chr-2_lod.drop-1.5_5.batches.csv
    Untracked:  data/ici.vs.pbs_blup_chr-2_lod.drop-1.5_snpsqc_5.batches.csv
    Untracked:  data/ici.vs.pbs_blup_chr-2_lod.drop-1.5_snpsqc_5.batches_mis.csv
    Untracked:  data/ici.vs.pbs_blup_chr-2_lod.drop-1.5_snpsqc_dis_no-x_updated_5.batches.csv
    Untracked:  data/ici.vs.pbs_blup_chr-2_lod.drop-1.5_snpsqc_dis_no-x_updated_5.batches_mis.csv
    Untracked:  data/ici.vs.pbs_blup_chr-3_lod.drop-1.5_5.batches.csv
    Untracked:  data/ici.vs.pbs_blup_chr-3_lod.drop-1.5_snpsqc_5.batches.csv
    Untracked:  data/ici.vs.pbs_blup_chr-3_lod.drop-1.5_snpsqc_5.batches_mis.csv
    Untracked:  data/ici.vs.pbs_blup_chr-3_lod.drop-1.5_snpsqc_dis_no-x_updated_5.batches.csv
    Untracked:  data/ici.vs.pbs_blup_chr-3_lod.drop-1.5_snpsqc_dis_no-x_updated_5.batches_mis.csv
    Untracked:  data/ici.vs.pbs_blup_chr-4_lod.drop-1.5_5.batches.csv
    Untracked:  data/ici.vs.pbs_blup_chr-4_lod.drop-1.5_snpsqc_5.batches.csv
    Untracked:  data/ici.vs.pbs_blup_chr-4_lod.drop-1.5_snpsqc_5.batches_mis.csv
    Untracked:  data/ici.vs.pbs_blup_chr-4_lod.drop-1.5_snpsqc_dis_no-x_updated_5.batches.csv
    Untracked:  data/ici.vs.pbs_blup_chr-4_lod.drop-1.5_snpsqc_dis_no-x_updated_5.batches_mis.csv
    Untracked:  data/ici.vs.pbs_blup_chr-5_lod.drop-1.5_5.batches.csv
    Untracked:  data/ici.vs.pbs_blup_chr-5_lod.drop-1.5_snpsqc_5.batches.csv
    Untracked:  data/ici.vs.pbs_blup_chr-5_lod.drop-1.5_snpsqc_5.batches_mis.csv
    Untracked:  data/ici.vs.pbs_blup_chr-5_lod.drop-1.5_snpsqc_dis_no-x_updated_5.batches.csv
    Untracked:  data/ici.vs.pbs_blup_chr-5_lod.drop-1.5_snpsqc_dis_no-x_updated_5.batches_mis.csv
    Untracked:  data/ici.vs.pbs_blup_chr-6_lod.drop-1.5_5.batches.csv
    Untracked:  data/ici.vs.pbs_blup_chr-6_lod.drop-1.5_snpsqc_5.batches.csv
    Untracked:  data/ici.vs.pbs_blup_chr-6_lod.drop-1.5_snpsqc_5.batches_mis.csv
    Untracked:  data/ici.vs.pbs_blup_chr-6_lod.drop-1.5_snpsqc_dis_no-x_updated_5.batches.csv
    Untracked:  data/ici.vs.pbs_blup_chr-6_lod.drop-1.5_snpsqc_dis_no-x_updated_5.batches_mis.csv
    Untracked:  data/ici.vs.pbs_blup_chr-7_lod.drop-1.5_5.batches.csv
    Untracked:  data/ici.vs.pbs_blup_chr-7_lod.drop-1.5_snpsqc_5.batches.csv
    Untracked:  data/ici.vs.pbs_blup_chr-7_lod.drop-1.5_snpsqc_5.batches_mis.csv
    Untracked:  data/ici.vs.pbs_blup_chr-7_lod.drop-1.5_snpsqc_dis_no-x_updated_5.batches.csv
    Untracked:  data/ici.vs.pbs_blup_chr-7_lod.drop-1.5_snpsqc_dis_no-x_updated_5.batches_mis.csv
    Untracked:  data/ici.vs.pbs_blup_chr-8_lod.drop-1.5_5.batches.csv
    Untracked:  data/ici.vs.pbs_blup_chr-8_lod.drop-1.5_snpsqc_5.batches.csv
    Untracked:  data/ici.vs.pbs_blup_chr-8_lod.drop-1.5_snpsqc_5.batches_mis.csv
    Untracked:  data/ici.vs.pbs_blup_chr-8_lod.drop-1.5_snpsqc_dis_no-x_updated_5.batches.csv
    Untracked:  data/ici.vs.pbs_blup_chr-8_lod.drop-1.5_snpsqc_dis_no-x_updated_5.batches_mis.csv
    Untracked:  data/ici.vs.pbs_blup_chr-9_lod.drop-1.5_5.batches.csv
    Untracked:  data/ici.vs.pbs_blup_chr-9_lod.drop-1.5_snpsqc_5.batches.csv
    Untracked:  data/ici.vs.pbs_blup_chr-9_lod.drop-1.5_snpsqc_5.batches_mis.csv
    Untracked:  data/ici.vs.pbs_blup_chr-9_lod.drop-1.5_snpsqc_dis_no-x_updated_5.batches.csv
    Untracked:  data/ici.vs.pbs_blup_chr-9_lod.drop-1.5_snpsqc_dis_no-x_updated_5.batches_mis.csv
    Untracked:  data/ici.vs.pbs_blup_chr-X_lod.drop-1.5_5.batches.csv
    Untracked:  data/ici.vs.pbs_blup_chr-X_lod.drop-1.5_snpsqc_5.batches.csv
    Untracked:  data/ici.vs.pbs_blup_chr-X_lod.drop-1.5_snpsqc_5.batches_mis.csv
    Untracked:  data/ici.vs.pbs_blup_chr-X_lod.drop-1.5_snpsqc_dis_no-x_updated_5.batches.csv
    Untracked:  data/ici.vs.pbs_blup_chr-X_lod.drop-1.5_snpsqc_dis_no-x_updated_5.batches_mis.csv
    Untracked:  data/ici.vs.pbs_blup_sub_chr-13_peak.marker-UNC22384241_lod.drop-1.5_snpsqc_snpsqc_5.batches_peaks.csv
    Untracked:  data/ici.vs.pbs_blup_sub_chr-18_peak.marker-UNCHS045343_lod.drop-1.5_snpsqc_5.batches_mis.csv
    Untracked:  data/ici.vs.pbs_blup_sub_chr-18_peak.marker-UNCHS045343_lod.drop-1.5_snpsqc_5.batches_mis_conditional_1-peak-chr10.csv
    Untracked:  data/ici.vs.pbs_blup_sub_chr-18_peak.marker-UNCHS045343_lod.drop-1.5_snpsqc_5.batches_mis_conditional_1-peak-chr3.csv
    Untracked:  data/ici.vs.pbs_blup_sub_chr-18_peak.marker-UNCHS045343_lod.drop-1.5_snpsqc_5.batches_mis_conditional_1-peak-chr4.csv
    Untracked:  data/ici.vs.pbs_blup_sub_chr-19_peak.marker-UNC29893228_lod.drop-1.5_snpsqc_snpsqc_5.batches_peaks.csv
    Untracked:  data/ici.vs.pbs_blup_sub_chr-19_peak.marker-UNC29990033_lod.drop-1.5_snpsqc_dis_no-x_updated_5.batches_peaks.csv
    Untracked:  data/ici.vs.pbs_blup_sub_chr-19_peak.marker-UNCHS047192_lod.drop-1.5_snpsqc_dis_no-x_updated_5.batches_mis_peaks.csv
    Untracked:  data/ici.vs.pbs_blup_sub_chr-8_peak.marker-UNCHS023353_lod.drop-1.5_snpsqc_dis_no-x_updated_5.batches_mis_peaks.csv
    Untracked:  data/ici.vs.pbs_blup_sub_chr-8_peak.marker-UNCHS023353_lod.drop-1.5_snpsqc_dis_no-x_updated_5.batches_peaks.csv
    Untracked:  data/ici.vs.pbs_blup_sub_chr-8_peak.marker-UNCHS023364_lod.drop-1.5_snpsqc_snpsqc_5.batches_peaks.csv
    Untracked:  data/ici.vs.pbs_fitqtl_peaks_additive_snpsqc_dis_no-x_updated_5.batches_mis_peaks.txt
    Untracked:  data/ici.vs.pbs_fitqtl_peaks_additive_snpsqc_dis_no-x_updated_5.batches_peaks.txt
    Untracked:  data/ici.vs.pbs_fitqtl_peaks_additive_snpsqc_snpsqc_5.batches_peaks.txt
    Untracked:  data/ici.vs.pbs_fitqtl_peaks_interacting_snpsqc_dis_no-x_updated_5.batches_mis_peaks.txt
    Untracked:  data/ici.vs.pbs_fitqtl_peaks_interacting_snpsqc_dis_no-x_updated_5.batches_peaks.txt
    Untracked:  data/ici.vs.pbs_fitqtl_peaks_interacting_snpsqc_snpsqc_5.batches_peaks.txt
    Untracked:  data/ici.vs.pbs_fitqtl_peaks_sex_additive_snpsqc_dis_no-x_updated_5.batches_mis_peaks.txt
    Untracked:  data/ici.vs.pbs_fitqtl_peaks_sex_additive_snpsqc_dis_no-x_updated_5.batches_peaks.txt
    Untracked:  data/ici.vs.pbs_fitqtl_peaks_sex_additive_snpsqc_snpsqc_5.batches_peaks.txt
    Untracked:  data/ici.vs.pbs_fitqtl_peaks_sex_interacting_snpsqc_dis_no-x_updated_5.batches_mis_peaks.txt
    Untracked:  data/ici.vs.pbs_fitqtl_peaks_sex_interacting_snpsqc_dis_no-x_updated_5.batches_peaks.txt
    Untracked:  data/ici.vs.pbs_fitqtl_peaks_sex_interacting_snpsqc_snpsqc_5.batches_peaks.txt
    Untracked:  data/ici.vs.pbs_genes_chr-13_peak.marker-UNC22384241_lod.drop-1.5_snpsqc_snpsqc_5.batches_peaks.csv
    Untracked:  data/ici.vs.pbs_genes_chr-18_peak.marker-UNCHS045343_lod.drop-1.5_snpsqc_5.batches_mis.csv
    Untracked:  data/ici.vs.pbs_genes_chr-18_peak.marker-UNCHS045343_lod.drop-1.5_snpsqc_5.batches_mis_conditional_1-peak-chr10.csv
    Untracked:  data/ici.vs.pbs_genes_chr-18_peak.marker-UNCHS045343_lod.drop-1.5_snpsqc_5.batches_mis_conditional_1-peak-chr3.csv
    Untracked:  data/ici.vs.pbs_genes_chr-18_peak.marker-UNCHS045343_lod.drop-1.5_snpsqc_5.batches_mis_conditional_1-peak-chr4.csv
    Untracked:  data/ici.vs.pbs_genes_chr-19_peak.marker-UNC29893228_lod.drop-1.5_snpsqc_snpsqc_5.batches_peaks.csv
    Untracked:  data/ici.vs.pbs_genes_chr-19_peak.marker-UNC29990033_lod.drop-1.5_snpsqc_dis_no-x_updated_5.batches_peaks.csv
    Untracked:  data/ici.vs.pbs_genes_chr-19_peak.marker-UNCHS047192_lod.drop-1.5_snpsqc_dis_no-x_updated_5.batches_mis_peaks.csv
    Untracked:  data/ici.vs.pbs_genes_chr-8_peak.marker-UNCHS023353_lod.drop-1.5_snpsqc_dis_no-x_updated_5.batches_mis_peaks.csv
    Untracked:  data/ici.vs.pbs_genes_chr-8_peak.marker-UNCHS023353_lod.drop-1.5_snpsqc_dis_no-x_updated_5.batches_peaks.csv
    Untracked:  data/ici.vs.pbs_genes_chr-8_peak.marker-UNCHS023364_lod.drop-1.5_snpsqc_snpsqc_5.batches_peaks.csv
    Untracked:  data/ici.vs.pbs_gm_qtl_5.batches.csv
    Untracked:  data/ici.vs.pbs_gm_qtl_5.batches_conditional_1-peak-chr10.csv
    Untracked:  data/ici.vs.pbs_gm_qtl_5.batches_conditional_1-peak-chr3.csv
    Untracked:  data/ici.vs.pbs_gm_qtl_5.batches_conditional_1-peak-chr4.csv
    Untracked:  data/ici.vs.pbs_gm_qtl_5.batches_conditional_2-peaks-chr3-10.csv
    Untracked:  data/ici.vs.pbs_gm_qtl_5.batches_conditional_2-peaks-chr3-4.csv
    Untracked:  data/ici.vs.pbs_gm_qtl_5.batches_conditional_2-peaks-chr4-10.csv
    Untracked:  data/ici.vs.pbs_gm_qtl_5.batches_mis.csv
    Untracked:  data/ici.vs.pbs_gm_qtl_5.batches_mis_conditional_1-peak-chr10.csv
    Untracked:  data/ici.vs.pbs_gm_qtl_5.batches_mis_conditional_1-peak-chr3.csv
    Untracked:  data/ici.vs.pbs_gm_qtl_5.batches_mis_conditional_1-peak-chr4.csv
    Untracked:  data/ici.vs.pbs_gm_qtl_snpsqc_5.batches.csv
    Untracked:  data/ici.vs.pbs_gm_qtl_snpsqc_5.batches_mis.csv
    Untracked:  data/ici.vs.pbs_gm_qtl_snpsqc_5.batches_mis_conditional_1-peak-chr10.csv
    Untracked:  data/ici.vs.pbs_gm_qtl_snpsqc_5.batches_mis_conditional_1-peak-chr3.csv
    Untracked:  data/ici.vs.pbs_gm_qtl_snpsqc_5.batches_mis_conditional_1-peak-chr4.csv
    Untracked:  data/ici.vs.pbs_gm_qtl_snpsqc_5.batches_peaks.csv
    Untracked:  data/ici.vs.pbs_gm_qtl_snpsqc_dis_no-x_updated_5.batches.csv
    Untracked:  data/ici.vs.pbs_gm_qtl_snpsqc_dis_no-x_updated_5.batches_mis.csv
    Untracked:  data/ici.vs.pbs_gm_qtl_snpsqc_dis_no-x_updated_5.batches_mis_conditional_1-peak-chr3.csv
    Untracked:  data/ici.vs.pbs_gm_qtl_snpsqc_dis_no-x_updated_5.batches_mis_conditional_3-peaks-chr3-4-10.csv
    Untracked:  data/ici.vs.pbs_gm_qtl_snpsqc_dis_no-x_updated_5.batches_mis_peaks.csv
    Untracked:  data/ici.vs.pbs_gm_qtl_snpsqc_dis_no-x_updated_5.batches_peaks.csv
    Untracked:  data/ici.vs.pbs_marker.freq_low.geno.freq.removed_geno.ratio.csv
    Untracked:  data/ici.vs.pbs_marker.freq_low.geno.freq.removed_geno.ratio_5.batches.csv
    Untracked:  data/ici.vs.pbs_marker.freq_low.geno.freq.removed_geno.ratio_5.batches_mis.csv
    Untracked:  data/ici.vs.pbs_marker.freq_low.geno.freq.removed_sample.outliers.removed_geno.ratio.csv
    Untracked:  data/ici.vs.pbs_marker.freq_low.geno.freq.removed_sample.outliers.removed_geno.ratio_5.batches.csv
    Untracked:  data/ici.vs.pbs_marker.freq_low.geno.freq.removed_sample.outliers.removed_geno.ratio_5.batches_mis.csv
    Untracked:  data/ici.vs.pbs_marker.freq_low.geno.freq.removed_sample.outliers.removed_geno.ratiov_5.batches.csv
    Untracked:  data/ici.vs.pbs_marker.freq_low.geno.freq.removed_sample.outliers.removed_geno.ratiov_5.batches_mis.csv
    Untracked:  data/ici.vs.pbs_marker.freq_low.probs.freq.removed_geno.ratio.csv
    Untracked:  data/ici.vs.pbs_marker.freq_low.probs.freq.removed_geno.ratio_5.batches.csv
    Untracked:  data/ici.vs.pbs_marker.freq_low.probs.freq.removed_geno.ratio_5.batches_mis.csv
    Untracked:  data/ici.vs.pbs_marker.freq_low.probs.freq.removed_sample.outliers.removed_geno.ratio.csv
    Untracked:  data/ici.vs.pbs_marker.freq_low.probs.freq.removed_sample.outliers.removed_geno.ratio_5.batches.csv
    Untracked:  data/ici.vs.pbs_marker.freq_low.probs.freq.removed_sample.outliers.removed_geno.ratio_5.batches_mis.csv
    Untracked:  data/ici.vs.pbs_rz.age-additive.covariates_blup_sub_chr-7_peak.marker-JAX00153527_lod.drop-1.5_5.batches.csv
    Untracked:  data/ici.vs.pbs_rz.age-additive.covariates_blup_sub_chr-7_peak.marker-JAX00153527_lod.drop-1.5_snpsqc_5.batches.csv
    Untracked:  data/ici.vs.pbs_rz.age-additive.covariates_blup_sub_chr-7_peak.marker-UNCHS020443_lod.drop-1.5_snpsqc_dis_no-x_updated_5.batches.csv
    Untracked:  data/ici.vs.pbs_rz.age-additive.covariates_blup_sub_chr-7_peak.marker-UNCHS020443_lod.drop-1.5_snpsqc_dis_no-x_updated_5.batches_mis.csv
    Untracked:  data/ici.vs.pbs_rz.age-additive.covariates_blup_sub_chr-7_peak.marker-UNCHS020486_lod.drop-1.5_snpsqc_5.batches_mis.csv
    Untracked:  data/ici.vs.pbs_rz.age-additive.covariates_genes_chr-7_peak.marker-JAX00153527_lod.drop-1.5_5.batches.csv
    Untracked:  data/ici.vs.pbs_rz.age-additive.covariates_genes_chr-7_peak.marker-JAX00153527_lod.drop-1.5_snpsqc_5.batches.csv
    Untracked:  data/ici.vs.pbs_rz.age-additive.covariates_genes_chr-7_peak.marker-UNCHS020443_lod.drop-1.5_snpsqc_dis_no-x_updated_5.batches.csv
    Untracked:  data/ici.vs.pbs_rz.age-additive.covariates_genes_chr-7_peak.marker-UNCHS020443_lod.drop-1.5_snpsqc_dis_no-x_updated_5.batches_mis.csv
    Untracked:  data/ici.vs.pbs_rz.age-additive.covariates_genes_chr-7_peak.marker-UNCHS020486_lod.drop-1.5_snpsqc_5.batches_mis.csv
    Untracked:  data/ici.vs.pbs_sample.genos_marker.freq_low.geno.freq.removed.csv
    Untracked:  data/ici.vs.pbs_sample.genos_marker.freq_low.geno.freq.removed_5.batches.csv
    Untracked:  data/ici.vs.pbs_sample.genos_marker.freq_low.geno.freq.removed_5.batches_mis.csv
    Untracked:  data/ici.vs.pbs_sample.genos_marker.freq_low.geno.freq.removed_sample.outliers.removed.csv
    Untracked:  data/ici.vs.pbs_sample.genos_marker.freq_low.geno.freq.removed_sample.outliers.removed_5.batches.csv
    Untracked:  data/ici.vs.pbs_sample.genos_marker.freq_low.geno.freq.removed_sample.outliers.removed_5.batches_mis.csv
    Untracked:  data/ici.vs.pbs_sample.genos_marker.freq_low.probs.freq.removed.csv
    Untracked:  data/ici.vs.pbs_sample.genos_marker.freq_low.probs.freq.removed_5.batches.csv
    Untracked:  data/ici.vs.pbs_sample.genos_marker.freq_low.probs.freq.removed_5.batches_mis.csv
    Untracked:  data/ici.vs.pbs_sample.genos_marker.freq_low.probs.freq.removed_sample.outliers.removed.csv
    Untracked:  data/ici.vs.pbs_sample.genos_marker.freq_low.probs.freq.removed_sample.outliers.removed_5.batches.csv
    Untracked:  data/ici.vs.pbs_sample.genos_marker.freq_low.probs.freq.removed_sample.outliers.removed_5.batches_mis.csv
    Untracked:  data/ici.vs.pbs_scanone_5.batches.Rdata
    Untracked:  data/ici.vs.pbs_scanone_5.batches_mis.Rdata
    Untracked:  data/ici.vs.pbs_scanone_snpsqc_5.batches.Rdata
    Untracked:  data/ici.vs.pbs_scanone_snpsqc_5.batches_mis.Rdata
    Untracked:  data/ici.vs.pbs_scanone_snpsqc_dis_no-x_updated_5.batches.Rdata
    Untracked:  data/ici.vs.pbs_scanone_snpsqc_dis_no-x_updated_5.batches_mis.Rdata
    Untracked:  data/mean.differences_age.of.onset_ici.vs.pbs_sample.genos_marker.freq_low.geno.freq.removed_5.batches.csv
    Untracked:  data/mean.differences_age.of.onset_ici.vs.pbs_sample.genos_marker.freq_low.geno.freq.removed_5.batches_mis.csv
    Untracked:  data/mean.differences_age.of.onset_ici.vs.pbs_sample.genos_marker.freq_low.geno.freq.removed_sample.outliers.removed_5.batches.csv
    Untracked:  data/mean.differences_age.of.onset_ici.vs.pbs_sample.genos_marker.freq_low.geno.freq.removed_sample.outliers.removed_5.batches_mis.csv
    Untracked:  data/mean.differences_group_ici.vs.pbs_sample.genos_marker.freq_low.geno.freq.removed_5.batches.csv
    Untracked:  data/mean.differences_group_ici.vs.pbs_sample.genos_marker.freq_low.geno.freq.removed_5.batches_mis.csv
    Untracked:  data/mean.differences_group_ici.vs.pbs_sample.genos_marker.freq_low.geno.freq.removed_sample.outliers.removed_5.batches.csv
    Untracked:  data/mean.differences_group_ici.vs.pbs_sample.genos_marker.freq_low.geno.freq.removed_sample.outliers.removed_5.batches_mis.csv
    Untracked:  data/percent_missing_id_3.batches.RData
    Untracked:  data/percent_missing_id_4.batches.RData
    Untracked:  data/percent_missing_id_4.batches_bc.RData
    Untracked:  data/percent_missing_id_5.batches.RData
    Untracked:  data/percent_missing_marker_4.batches.RData
    Untracked:  data/percent_missing_marker_5.batches.RData
    Untracked:  data/pheno.csv
    Untracked:  data/pheno_BC312.csv
    Untracked:  data/physical_map.csv
    Untracked:  data/physical_map_BC312.csv
    Untracked:  data/qc_info_bad_sample_3.batches.RData
    Untracked:  data/qc_info_bad_sample_4.batches.RData
    Untracked:  data/qc_info_bad_sample_4.batches_bc.RData
    Untracked:  data/qc_info_bad_sample_5.batches.RData
    Untracked:  data/remaining.markers_geno.freq.xlsx
    Untracked:  data/sample_geno.csv
    Untracked:  data/sample_geno_AHB_BC312.csv
    Untracked:  data/sample_geno_bc.csv
    Untracked:  data/sample_geno_bc_BC312.csv
    Untracked:  data/serreze_probs.rds
    Untracked:  data/serreze_probs_BC312.rds
    Untracked:  data/serreze_probs_allqc.rds
    Untracked:  data/serreze_probs_allqc_5.batches.rds
    Untracked:  data/serreze_probs_allqc_5.batches_mis.rds
    Untracked:  data/summary.cg_3.batches.RData
    Untracked:  data/summary.cg_4.batches.RData
    Untracked:  data/summary.cg_4.batches_bc.RData
    Untracked:  data/summary.cg_5.batches.RData
    Untracked:  output/Percent_missing_genotype_data_5.batches.pdf
    Untracked:  output/Percent_missing_genotype_data_per_marker_5.batches.pdf
    Untracked:  output/Proportion_matching_genotypes_before_removal_of_bad_samples_5.batches.pdf
    Untracked:  output/genotype_error_marker_5.batches.pdf
    Untracked:  output/genotype_frequency_marker_5.batches.pdf

Unstaged changes:
    Modified:   analysis/4.1.1_qtl.analysis_binary_ici-early.vs.pbs_5.batches.Rmd
    Modified:   analysis/4.1.1_qtl.analysis_binary_ici-early.vs.pbs_5.batches_mis.Rmd
    Modified:   analysis/4.1.1_qtl.analysis_binary_ici-early.vs.pbs_snpsqc_5.batches.Rmd
    Modified:   analysis/4.1.1_qtl.analysis_binary_ici-early.vs.pbs_snpsqc_5.batches_mis.Rmd
    Modified:   analysis/4.1.1_qtl.analysis_binary_ici-early.vs.pbs_snpsqc_dis_no-x_updated_5.batches.Rmd
    Modified:   analysis/4.1.1_qtl.analysis_binary_ici-early.vs.pbs_snpsqc_dis_no-x_updated_5.batches_mis.Rmd
    Modified:   analysis/4.1.1_qtl.analysis_binary_ici.vs.eoi_snpsqc_dis_no-x_updated.Rmd
    Modified:   analysis/4.1.1_qtl.analysis_binary_ici.vs.pbs_snpsqc_dis_no-x_updated.Rmd
    Modified:   analysis/4.1.2_qtl.analysis_cont_age_ici-early.vs.pbs_pheno.corrected.cleaned_5.batches.Rmd
    Modified:   analysis/4.1.2_qtl.analysis_cont_age_ici-early.vs.pbs_pheno.corrected.cleaned_5.batches_mis.Rmd
    Modified:   analysis/4.1.2_qtl.analysis_cont_age_ici-early.vs.pbs_snpsqc_pheno.corrected.cleaned_5.batches.Rmd
    Modified:   analysis/4.1.2_qtl.analysis_cont_age_ici-early.vs.pbs_snpsqc_pheno.corrected.cleaned_5.batches_mis.Rmd
    Modified:   analysis/4.1.2_qtl.analysis_cont_age_ici-early.vs.pbs_snpsqc_pheno.corrected.cleaned_dis_no-xk_5.batches.Rmd
    Modified:   analysis/4.1.2_qtl.analysis_cont_age_ici-early.vs.pbs_snpsqc_pheno.corrected.cleaned_dis_no-xk_5.batches_mis.Rmd
    Modified:   analysis/4.1.2_qtl.analysis_cont_age_ici.vs.eoi_pheno.corrected.cleaned_5.batches.Rmd
    Modified:   analysis/4.1.2_qtl.analysis_cont_age_ici.vs.eoi_pheno.corrected.cleaned_5.batches_mis.Rmd
    Modified:   analysis/4.1.2_qtl.analysis_cont_age_ici.vs.eoi_snpsqc_pheno.corrected.cleaned_5.batches.Rmd
    Modified:   analysis/4.1.2_qtl.analysis_cont_age_ici.vs.eoi_snpsqc_pheno.corrected.cleaned_5.batches_mis.Rmd
    Modified:   analysis/4.1.2_qtl.analysis_cont_age_ici.vs.eoi_snpsqc_pheno.corrected.cleaned_dis_no-xk_5.batches.Rmd
    Modified:   analysis/4.1.2_qtl.analysis_cont_age_ici.vs.eoi_snpsqc_pheno.corrected.cleaned_dis_no-xk_5.batches_mis.Rmd
    Modified:   analysis/4.1.2_qtl.analysis_cont_age_ici.vs.pbs_pheno.corrected.cleaned_5.batches.Rmd
    Modified:   analysis/4.1.2_qtl.analysis_cont_age_ici.vs.pbs_pheno.corrected.cleaned_5.batches_mis.Rmd
    Modified:   analysis/4.1.2_qtl.analysis_cont_age_ici.vs.pbs_snpsqc_pheno.corrected.cleaned_5.batches.Rmd
    Modified:   analysis/4.1.2_qtl.analysis_cont_age_ici.vs.pbs_snpsqc_pheno.corrected.cleaned_5.batches_mis.Rmd
    Modified:   analysis/4.1.2_qtl.analysis_cont_age_ici.vs.pbs_snpsqc_pheno.corrected.cleaned_dis_no-xk_5.batches.Rmd
    Modified:   analysis/4.1.2_qtl.analysis_cont_age_ici.vs.pbs_snpsqc_pheno.corrected.cleaned_dis_no-xk_5.batches_mis.Rmd
    Modified:   analysis/4.1.2_qtl.analysis_cont_rz.age_ici-early.vs.pbs_pheno.corrected.cleaned_5.batches.Rmd
    Modified:   analysis/4.1.2_qtl.analysis_cont_rz.age_ici-early.vs.pbs_pheno.corrected.cleaned_5.batches_mis.Rmd
    Modified:   analysis/4.1.2_qtl.analysis_cont_rz.age_ici-early.vs.pbs_snpsqc_pheno.corrected.cleaned_5.batches.Rmd
    Modified:   analysis/4.1.2_qtl.analysis_cont_rz.age_ici-early.vs.pbs_snpsqc_pheno.corrected.cleaned_5.batches_mis.Rmd
    Modified:   analysis/4.1.2_qtl.analysis_cont_rz.age_ici-early.vs.pbs_snpsqc_pheno.corrected.cleaned_dis_no-xk_5.batches.Rmd
    Modified:   analysis/4.1.2_qtl.analysis_cont_rz.age_ici-early.vs.pbs_snpsqc_pheno.corrected.cleaned_dis_no-xk_5.batches_mis.Rmd
    Modified:   analysis/4.1.2_qtl.analysis_cont_rz.age_ici.vs.eoi_pheno.corrected.cleaned_5.batches.Rmd
    Modified:   analysis/4.1.2_qtl.analysis_cont_rz.age_ici.vs.eoi_pheno.corrected.cleaned_5.batches_mis.Rmd
    Modified:   analysis/4.1.2_qtl.analysis_cont_rz.age_ici.vs.eoi_snpsqc_pheno.corrected.cleaned_5.batches.Rmd
    Modified:   analysis/4.1.2_qtl.analysis_cont_rz.age_ici.vs.eoi_snpsqc_pheno.corrected.cleaned_5.batches_mis.Rmd
    Modified:   analysis/4.1.2_qtl.analysis_cont_rz.age_ici.vs.eoi_snpsqc_pheno.corrected.cleaned_dis_no-xk_5.batches.Rmd
    Modified:   analysis/4.1.2_qtl.analysis_cont_rz.age_ici.vs.eoi_snpsqc_pheno.corrected.cleaned_dis_no-xk_5.batches_mis.Rmd
    Modified:   analysis/4.1.2_qtl.analysis_cont_rz.age_ici.vs.pbs_pheno.corrected.cleaned_5.batches.Rmd
    Modified:   analysis/4.1.2_qtl.analysis_cont_rz.age_ici.vs.pbs_pheno.corrected.cleaned_5.batches_mis.Rmd
    Modified:   analysis/4.1.2_qtl.analysis_cont_rz.age_ici.vs.pbs_snpsqc_pheno.corrected.cleaned_5.batches.Rmd
    Modified:   analysis/4.1.2_qtl.analysis_cont_rz.age_ici.vs.pbs_snpsqc_pheno.corrected.cleaned_5.batches_mis.Rmd
    Modified:   analysis/4.1.2_qtl.analysis_cont_rz.age_ici.vs.pbs_snpsqc_pheno.corrected.cleaned_dis_no-xk_5.batches.Rmd
    Modified:   analysis/4.1.2_qtl.analysis_cont_rz.age_ici.vs.pbs_snpsqc_pheno.corrected.cleaned_dis_no-xk_5.batches_mis.Rmd
    Modified:   analysis/genotype.frequencies_ici.vs.eoi_5.batches.Rmd
    Modified:   analysis/genotype.frequencies_ici.vs.eoi_5.batches_mis.Rmd
    Modified:   analysis/genotype.frequencies_ici.vs.pbs_5.batches.Rmd
    Modified:   analysis/genotype.frequencies_ici.vs.pbs_5.batches_mis.Rmd
    Modified:   analysis/index_5.batches.Rmd
    Modified:   analysis/index_5.batches_additional.Rmd

</code></pre>
<p>
Note that any generated files, e.g. HTML, png, CSS, etc., are not
included in this status report because it is ok for generated content to
have uncommitted changes.
</p>
</div>
</div>
</div>
</div>
<hr>
</div>
<div id="versions" class="tab-pane fade">
<p>
There are no past versions. Publish this analysis with
<code>wflow_publish()</code> to start tracking its development.
</p>
<hr>
</div>
</div>
</div>
<div id="loading-data" class="section level2">
<h2>Loading Data</h2>
<p>We will load the data and subset indivials out that are in the groups
of interest. We will create a binary phenotype from this (PBS ==0, ICI
== 1).</p>
<pre class="r"><code>load(&quot;data/gm_allqc_5.batches_mis.RData&quot;)

#gm_allqc
gm=gm_allqc
gm</code></pre>
<pre><code>Object of class cross2 (crosstype &quot;bc&quot;)

Total individuals               308
No. genotyped individuals       308
No. phenotyped individuals      308
No. with both geno &amp; pheno      308

No. phenotypes                    1
No. covariates                    6
No. phenotype covariates          0

No. chromosomes                  20
Total markers                131356

No. markers by chr:
   1    2    3    4    5    6    7    8    9   10   11   12   13   14   15   16 
9956 9987 7848 7586 7609 7736 7399 6458 6713 6385 7143 6110 6082 5966 5346 5015 
  17   18   19    X 
5080 4605 3562 4770 </code></pre>
<pre class="r"><code>#pr &lt;- readRDS(&quot;data/serreze_probs_allqc_5.batches_mis.rds&quot;)
#pr &lt;- readRDS(&quot;data/serreze_probs.rds&quot;)

##extracting animals with ici and pbs group status
miceinfo &lt;- gm$covar[gm$covar$group == &quot;PBS&quot; | gm$covar$group == &quot;ICI&quot;,]
table(miceinfo$group)</code></pre>
<pre><code>
ICI PBS 
104  34 </code></pre>
<pre class="r"><code>mice.ids &lt;- rownames(miceinfo)

gm &lt;- gm[mice.ids]
gm</code></pre>
<pre><code>Object of class cross2 (crosstype &quot;bc&quot;)

Total individuals               138
No. genotyped individuals       138
No. phenotyped individuals      138
No. with both geno &amp; pheno      138

No. phenotypes                    1
No. covariates                    6
No. phenotype covariates          0

No. chromosomes                  20
Total markers                131356

No. markers by chr:
   1    2    3    4    5    6    7    8    9   10   11   12   13   14   15   16 
9956 9987 7848 7586 7609 7736 7399 6458 6713 6385 7143 6110 6082 5966 5346 5015 
  17   18   19    X 
5080 4605 3562 4770 </code></pre>
<pre class="r"><code>#pr.qc &lt;- pr
#for (i in 1:20){pr.qc[[i]] = pr.qc[[i]][mice.ids,,]}

#bin_pheno &lt;- NULL
#bin_pheno$PBS &lt;- ifelse(gm$covar$group == &quot;PBS&quot;, 1, 0)
#bin_pheno$ICI &lt;- ifelse(gm$covar$group == &quot;ICI&quot;, 1, 0)
#bin_pheno &lt;- as.data.frame(bin_pheno)
#rownames(bin_pheno) &lt;- rownames(gm$covar)

gm$covar$ICI.vs.PBS &lt;- ifelse(gm$covar$group == &quot;PBS&quot;, 0, 1)
gm.full &lt;- gm

mar.covar &lt;- pull_markers(gm, c(&quot;UNCHS008487&quot;, &quot;UNC8250659&quot;, &quot;UNC18240977&quot;))
mar.covar.g &lt;- do.call(&quot;cbind&quot;, mar.covar$geno)

covars &lt;- merge(gm$covar, mar.covar.g, by=&#39;row.names&#39;, sort=F)
table(covars$group)</code></pre>
<pre><code>
ICI PBS 
104  34 </code></pre>
<pre class="r"><code>rownames(covars) &lt;- covars$Row.names
covars &lt;- covars[-1]

##removing problmetic marker

gm &lt;- drop_markers(gm, &quot;UNCHS013106&quot;)

markers &lt;- marker_names(gm)
gmapdf &lt;- read.csv(&quot;/Users/corneb/Documents/MyJax/CS/Projects/Serreze/haplotype.reconstruction/output_5.batches/genetic_map.csv&quot;)
pmapdf &lt;- read.csv(&quot;/Users/corneb/Documents/MyJax/CS/Projects/Serreze/haplotype.reconstruction/output_5.batches/physical_map.csv&quot;)
#mapdf &lt;- merge(gmapdf,pmapdf, by=c(&quot;marker&quot;,&quot;chr&quot;), all=T)
#rownames(mapdf) &lt;- mapdf$marker
#mapdf &lt;- mapdf[markers,]
#names(mapdf) &lt;- c(&#39;marker&#39;,&#39;chr&#39;,&#39;gmapdf&#39;,&#39;pmapdf&#39;)
#mapdfnd &lt;- mapdf[!duplicated(mapdf[c(2:3)]),]

pr.qc &lt;- calc_genoprob(gm)

gm</code></pre>
<pre><code>Object of class cross2 (crosstype &quot;bc&quot;)

Total individuals               138
No. genotyped individuals       138
No. phenotyped individuals      138
No. with both geno &amp; pheno      138

No. phenotypes                    1
No. covariates                    7
No. phenotype covariates          0

No. chromosomes                  20
Total markers                131355

No. markers by chr:
   1    2    3    4    5    6    7    8    9   10   11   12   13   14   15   16 
9956 9987 7848 7585 7609 7736 7399 6458 6713 6385 7143 6110 6082 5966 5346 5015 
  17   18   19    X 
5080 4605 3562 4770 </code></pre>
</div>
<div
id="conditionaing-on-eoi-vs.-ici-chr-3-peak-unchs008487-chr-4-peak-unc8250659"
class="section level1">
<h1>Conditionaing on eoi vs. ici chr 3 peak (UNCHS008487) &amp; chr 4
peak (UNC8250659)</h1>
<div id="genome-wide-scan" class="section level2">
<h2>Genome-wide scan</h2>
</div>
<div id="genome-wide-scan-1" class="section level2">
<h2>Genome-wide scan</h2>
<pre class="r"><code>Xcovar &lt;- get_x_covar(gm)
addcovar = model.matrix(~UNCHS008487+UNC8250659, data = covars)[,-1]

#K &lt;- calc_kinship(pr.qc, type = &quot;loco&quot;)
#heatmap(K[[1]])
#K.overall &lt;- calc_kinship(pr.qc, type = &quot;overall&quot;)
#heatmap(K.overall)
kinship &lt;- calc_kinship(pr.qc)
heatmap(kinship)</code></pre>
<p><img src="figure/4.1.1_qtl.analysis_binary_ici.vs.pbs_5.batches_mis_conditional_2-peaks.Rmd/permutation-1.png" width="912" style="display: block; margin: auto;" /></p>
<pre class="r"><code>#operm &lt;- scan1perm(pr.qc, gm$covar$phenos, Xcovar=Xcovar, n_perm=2000)
#operm &lt;- scan1perm(pr.qc, gm$covar$phenos, addcovar = addcovar, n_perm=2000)
#operm &lt;- scan1perm(pr.qc, gm$covar$phenos, n_perm=2000)
operm &lt;- scan1perm(pr.qc, gm$covar[&quot;ICI.vs.PBS&quot;], model=&quot;binary&quot;, n_perm=10, perm_Xsp=TRUE, chr_lengths=chr_lengths(gm$gmap), addcovar = addcovar)

summary_table&lt;-data.frame(unclass(summary(operm, alpha=c(0.01,  0.05, 0.1))))
names(summary_table) &lt;- c(&quot;autosomes&quot;,&quot;X&quot;)
summary_table$significance.level &lt;- rownames(summary_table)

rownames(summary_table) &lt;- NULL

summary_table[c(3,1:2)] %&gt;%
  kable(escape = F,align = c(&quot;ccc&quot;)) %&gt;%
  kable_styling(&quot;striped&quot;, full_width = T) %&gt;%
  column_spec(1, bold=TRUE)</code></pre>
<table class="table table-striped" style="margin-left: auto; margin-right: auto;">
<thead>
<tr>
<th style="text-align:center;">
significance.level
</th>
<th style="text-align:center;">
autosomes
</th>
<th style="text-align:center;">
X
</th>
</tr>
</thead>
<tbody>
<tr>
<td style="text-align:center;font-weight: bold;">
0.01
</td>
<td style="text-align:center;">
4.538200
</td>
<td style="text-align:center;">
5.289082
</td>
</tr>
<tr>
<td style="text-align:center;font-weight: bold;">
0.05
</td>
<td style="text-align:center;">
4.034056
</td>
<td style="text-align:center;">
4.476800
</td>
</tr>
<tr>
<td style="text-align:center;font-weight: bold;">
0.1
</td>
<td style="text-align:center;">
3.402310
</td>
<td style="text-align:center;">
3.414586
</td>
</tr>
</tbody>
</table>
<p>The figures below show QTL maps for each phenotype</p>
<pre class="r"><code>out &lt;- scan1(pr.qc, gm$covar[&quot;ICI.vs.PBS&quot;], Xcovar=Xcovar, model=&quot;binary&quot;, addcovar = addcovar)

summary_table&lt;-data.frame(unclass(summary(operm, alpha=c(0.01,  0.05, 0.1))))

plot_lod&lt;-function(out,map){
  for (i in 1:dim(out)[2]){
    #png(filename=paste0(&quot;/Users/chenm/Documents/qtl/Jai/&quot;,colnames(out)[i],  &quot;_lod.png&quot;))
    
    ymx &lt;- maxlod(out) # overall maximum LOD score
    plot(out, map, lodcolumn=i, col=&quot;slateblue&quot;, ylim=c(0, ymx+0.5))
    ##legend(&quot;topright&quot;, lwd=2, colnames(out)[i], bg=&quot;gray90&quot;)
    title(main = paste0(colnames(out)[i], &quot; [positions in cM]&quot;))
    add_threshold(map,  summary(operm,alpha=0.1), col = &#39;purple&#39;)
    add_threshold(map,  summary(operm, alpha=0.05), col = &#39;red&#39;)
    add_threshold(map,  summary(operm, alpha=0.01), col = &#39;blue&#39;)

    #par(mar=c(5.1, 6.1, 1.1, 1.1))
    ymx &lt;- 14 # overall maximum LOD score
    plot(out, map, lodcolumn=i, col=&quot;slateblue&quot;, ylim=c(0, ymx+0.5))
    ##legend(&quot;topright&quot;, lwd=2, colnames(out)[i], bg=&quot;gray90&quot;)
    title(main = paste0(colnames(out)[i], &quot; [positions in cM] \n(using same scale as eoi  vs. ici for easier comparison)&quot;))
    add_threshold(map,  summary(operm, alpha=0.1), col = &#39;purple&#39;)
    add_threshold(map,  summary(operm, alpha=0.05), col = &#39;red&#39;)
    add_threshold(map,  summary(operm, alpha=0.01), col = &#39;blue&#39;)
    #for (j in 1: dim(summary_table)[1]){
    #  abline(h=summary_table[j, i],col=&quot;red&quot;)
    #  text(x=400, y =summary_table[j, i]+0.12, labels = paste(&quot;p=&quot;, row.names(summary_table)[j]))
    #}
    #dev.off()
  }
}

plot_lod(out,gm$gmap)</code></pre>
<p><img src="figure/4.1.1_qtl.analysis_binary_ici.vs.pbs_5.batches_mis_conditional_2-peaks.Rmd/Genome-wide%20scan-1.png" width="912" style="display: block; margin: auto;" /><img src="figure/4.1.1_qtl.analysis_binary_ici.vs.pbs_5.batches_mis_conditional_2-peaks.Rmd/Genome-wide%20scan-2.png" width="912" style="display: block; margin: auto;" /></p>
</div>
<div id="lod-peaks" class="section level2">
<h2>LOD peaks</h2>
<p>The table below shows QTL peaks associated with the phenotype. We use
the 95% threshold from the permutations to find peaks.</p>
<div id="centimorgan-cm" class="section level3">
<h3>Centimorgan (cM)</h3>
<pre class="r"><code>peaks &lt;- find_peaks(out, gm$gmap, threshold=summary(operm,alpha=0.05)$A, thresholdX = summary(operm,alpha=0.05)$X, peakdrop=3, drop=1.5)

if(nrow(peaks) &gt;0){
peaks$marker &lt;- find_marker(gm$gmap, chr=peaks$chr,pos=peaks$pos)
names(peaks)[2] &lt;- c(&quot;phenotype&quot;)
peaks &lt;- peaks[-1]

rownames(peaks) &lt;- NULL
print(kable(peaks, escape = F, align = c(&quot;cccccccc&quot;), &quot;html&quot;) 
  %&gt;% kable_styling(&quot;striped&quot;, full_width = T)%&gt;%
  column_spec(1, bold=TRUE)
  )

#plot only peak chromosomes

plot_lod_chr&lt;-function(out,map,chrom){
  for (i in 1:dim(out)[2]){
    #png(filename=paste0(&quot;/Users/chenm/Documents/qtl/Jai/&quot;,colnames(out)[i],  &quot;_lod.png&quot;))
    
    #par(mar=c(5.1, 6.1, 1.1, 1.1))
    ymx &lt;- maxlod(out) # overall maximum LOD score
    plot(out, map, chr = chrom, lodcolumn=i, col=&quot;slateblue&quot;, ylim=c(0, ymx+0.5))
    #legend(&quot;topright&quot;, lwd=2, colnames(out)[i], bg=&quot;gray90&quot;)
    title(main = paste0(colnames(out)[i], &quot; - chr&quot;, chrom, &quot; [positions in cM]&quot;))
    add_threshold(map,  summary(operm,alpha=0.1), col = &#39;purple&#39;)
    add_threshold(map,  summary(operm, alpha=0.05), col = &#39;red&#39;)
    add_threshold(map,  summary(operm, alpha=0.01), col = &#39;blue&#39;)
    #for (j in 1: dim(summary_table)[1]){
    #  abline(h=summary_table[j, i],col=&quot;red&quot;)
    #  text(x=400, y =summary_table[j, i]+0.12, labels = paste(&quot;p=&quot;, row.names(summary_table)[j]))
    #}
    #dev.off()

    
    ymx &lt;- 14
    plot(out, map, chr = chrom, lodcolumn=i, col=&quot;slateblue&quot;, ylim=c(0, ymx+0.5))
    #legend(&quot;topright&quot;, lwd=2, colnames(out)[i], bg=&quot;gray90&quot;)
    title(main = paste0(colnames(out)[i], &quot; - chr&quot;, chrom, &quot; [positions in cM]\n(using same scale as eoi vs. ici for easier comparison)&quot;))
    add_threshold(map,  summary(operm,alpha=0.1), col = &#39;purple&#39;)
    add_threshold(map,  summary(operm, alpha=0.05), col = &#39;red&#39;)
    add_threshold(map,  summary(operm, alpha=0.01), col = &#39;blue&#39;)

  }
}


for(i in unique(peaks$chr)){
#for (i in 1:nrow(peaks)){
  #plot_lod_chr(out,gm$gmap, peaks$chr[i])
  plot_lod_chr(out,gm$gmap, i)
}

} else {
  print(paste0(&quot;There are no peaks that have a LOD that reaches suggestive (p&lt;0.05) level of &quot;,summary(operm,alpha=0.05)$A, &quot; [autosomes]/&quot;,summary(operm,alpha=0.05)$X, &quot; [x-chromosome]&quot;))
}</code></pre>
<p>[1] “There are no peaks that have a LOD that reaches suggestive
(p&lt;0.05) level of 4.03405558593242 [autosomes]/4.47679966592167
[x-chromosome]”</p>
</div>
<div id="megabase-mb" class="section level3">
<h3>Megabase (MB)</h3>
<pre class="r"><code>print(&quot;peaks in MB positions&quot;)</code></pre>
<p>[1] “peaks in MB positions”</p>
<pre class="r"><code>peaks_mba &lt;- find_peaks(out, gm$pmap, threshold=summary(operm,alpha=0.05)$A, thresholdX = summary(operm,alpha=0.05)$X, peakdrop=3, drop=1.5)

if(nrow(peaks) &gt;0){
peaks_mba$marker &lt;- find_marker(gm$pmap, chr=peaks_mba$chr,pos=peaks_mba$pos)
names(peaks_mba)[2] &lt;- c(&quot;phenotype&quot;)
peaks_mba &lt;- peaks_mba[-1]

#peaks_mbl &lt;- list()
##corresponding info in Mb
#for(i in 1:nrow(peaks)){
#  #lodindex &lt;- peaks$lodindex[i]
#  phenotype &lt;- peaks$phenotype[i]
#  chr &lt;- as.character(peaks$chr[i])
#  lod &lt;- peaks$lod[i]
#  mark &lt;- peaks$marker[i]
#  pos &lt;- mapdf[mapdf$marker==mark,]$pmapdf
#  ci_lo &lt;- mapdfnd$pmapdf[which(mapdfnd$gmapdf == peaks$ci_lo[i] &amp; mapdfnd$chr == peaks$chr[i])]
#  ci_hi &lt;- mapdfnd$pmapdf[which(mapdfnd$gmapdf == peaks$ci_hi[i] &amp; mapdfnd$chr == peaks$chr[i])]
#  peaks_mb=as.data.frame(cbind(phenotype, chr, pos, lod, ci_lo, ci_hi, mark))
#  names(peaks_mb)[7] &lt;- c(&quot;marker&quot;)
#  peaks_mbl[[i]] &lt;- peaks_mb
#}
#peaks_mba2 &lt;- do.call(rbind, peaks_mbl)
#peaks_mba2 &lt;- as.data.frame(peaks_mba)
#peaks_mba[,c(&quot;chr&quot;, &quot;pos&quot;, &quot;lod&quot;, &quot;ci_lo&quot;, &quot;ci_hi&quot;)] &lt;- sapply(peaks_mba[,c(&quot;chr&quot;, &quot;pos&quot;, &quot;lod&quot;, &quot;ci_lo&quot;, &quot;ci_hi&quot;)], as.numeric)

rownames(peaks_mba) &lt;- NULL
print(kable(peaks_mba, escape = F, align = c(&quot;cccccccc&quot;), &quot;html&quot;) 
  %&gt;% kable_styling(&quot;striped&quot;, full_width = T)%&gt;%
  column_spec(1, bold=TRUE)
  )

plot_lod_chr_mb&lt;-function(out,map,chrom){
  for (i in 1:dim(out)[2]){
    #png(filename=paste0(&quot;/Users/chenm/Documents/qtl/Jai/&quot;,colnames(out)[i],  &quot;_lod.png&quot;))
    
    #par(mar=c(5.1, 6.1, 1.1, 1.1))
    ymx &lt;- maxlod(out) # overall maximum LOD score
    plot(out, map, chr = chrom, lodcolumn=i, col=&quot;slateblue&quot;, ylim=c(0, ymx+0.5))
    #legend(&quot;topright&quot;, lwd=2, colnames(out)[i], bg=&quot;gray90&quot;)
    title(main = paste0(colnames(out)[i], &quot; - chr&quot;, chrom, &quot; [positions in MB]&quot;))
    add_threshold(map,  summary(operm,alpha=0.1), col = &#39;purple&#39;)
    add_threshold(map,  summary(operm, alpha=0.05), col = &#39;red&#39;)
    add_threshold(map,  summary(operm, alpha=0.01), col = &#39;blue&#39;)
    #for (j in 1: dim(summary_table)[1]){
    #  abline(h=summary_table[j, i],col=&quot;red&quot;)
    #  text(x=400, y =summary_table[j, i]+0.12, labels = paste(&quot;p=&quot;, row.names(summary_table)[j]))
    #}
    #dev.off()

    ymx &lt;- 14
    plot(out, map, chr = chrom, lodcolumn=i, col=&quot;slateblue&quot;, ylim=c(0, ymx+0.5))
    #legend(&quot;topright&quot;, lwd=2, colnames(out)[i], bg=&quot;gray90&quot;)
    title(main = paste0(colnames(out)[i], &quot; - chr&quot;, chrom, &quot; [positions in MB]\n(using same scale as eoi vs. ici for easier comparison)&quot;))
    add_threshold(map,  summary(operm,alpha=0.1), col = &#39;purple&#39;)
    add_threshold(map,  summary(operm, alpha=0.05), col = &#39;red&#39;)
    add_threshold(map,  summary(operm, alpha=0.01), col = &#39;blue&#39;)


  }
}

for(i in unique(peaks_mba$chr)){
#for (i in 1:nrow(peaks_mba)){
  #plot_lod_chr_mb(out,gm$pmap, peaks_mba$chr[i])
  plot_lod_chr_mb(out,gm$pmap,i)
}

} else {
  print(paste0(&quot;There are no peaks that have a LOD that reaches suggestive (p&lt;0.05) level of &quot;,summary(operm,alpha=0.05)$A, &quot; [autosomes]/&quot;,summary(operm,alpha=0.05)$X, &quot; [x-chromosome]&quot;))
}</code></pre>
<p>[1] “There are no peaks that have a LOD that reaches suggestive
(p&lt;0.05) level of 4.03405558593242 [autosomes]/4.47679966592167
[x-chromosome]”</p>
</div>
</div>
<div id="qtl-effects" class="section level2">
<h2>QTL effects</h2>
<p>For each peak LOD location we give a list of gene</p>
<pre class="r"><code>query_variants &lt;- create_variant_query_func(&quot;/Users/corneb/Documents/MyJax/CS/Projects/support.files/qtl2/cc_variants.sqlite&quot;)
query_genes &lt;- create_gene_query_func(&quot;/Users/corneb/Documents/MyJax/CS/Projects/support.files/qtl2/mouse_genes_mgi.sqlite&quot;)

if(nrow(peaks) &gt;0){
for (i in 1:nrow(peaks)){
#for (i in 1:1){
  #Plot 1
  g &lt;- maxmarg(pr.qc, gm$gmap, chr=peaks$chr[i], pos=peaks$pos[i], return_char=TRUE)
  #png(filename=paste0(&quot;/Users/chenm/Documents/qtl/Jai/&quot;,&quot;qtl_effect_&quot;, i, &quot;.png&quot;))
  #par(mar=c(4.1, 4.1, 1.5, 0.6))
  plot_pxg(g, gm$covar[,peaks$phenotype[i]], ylab=peaks$phenotype[i], sort=FALSE)
  title(main = paste0(&quot;chr: &quot;, chr=peaks$chr[i],&quot;; pos: &quot;, peaks$pos[i], &quot;cM /&quot;,peaks_mba$pos[i],&quot;MB\n(&quot;,peaks$phenotype[i],&quot; )&quot;), line=0.2)
  ##dev.off()

  chr = peaks$chr[i]

# Plot 2
  pr_sub &lt;- pull_genoprobint(pr.qc, gm$gmap, chr, c(peaks$ci_lo[i], peaks$ci_hi[i]))
  #coeff &lt;- scan1coef(pr[,chr], cross$pheno[,peaks$lodcolumn[i]], addcovar = addcovar)
  #coeff &lt;- scan1coef(pr[,chr], cross$pheno[,peaks$lodcolumn[i]], Xcovar=Xcovar)
  #coeff &lt;- scan1coef(pr.qc[,chr], gm$covar[peaks$lodcolumn[i]], model=&quot;binary&quot;)
  #coeff_sub &lt;- scan1coef(pr_sub[,chr], gm$covar[peaks$lodcolumn[i]], model=&quot;binary&quot;)
  blup &lt;- scan1blup(pr.qc[,chr], gm$covar[peaks$phenotype[i]], addcovar = addcovar)
  blup_sub &lt;- scan1blup(pr_sub[,chr], gm$covar[peaks$phenotype[i]], addcovar = addcovar)

  write.csv(as.data.frame(blup_sub), paste0(&quot;data/ici.vs.pbs_blup_sub_chr-&quot;,chr,&quot;_peak.marker-&quot;,peaks$marker[i],&quot;_lod.drop-1.5_5.batches_mis_conditional_2-peaks-chr3-4.csv&quot;), quote=F)

  #plot_coef(coeff, 
  #     gm$gmap, columns=1:2,
  #     bgcolor=&quot;gray95&quot;, legend=&quot;bottomleft&quot;, 
  #     main = paste(&quot;chr&quot;, chr=peaks$chr[i],&quot;; pos: &quot;, peaks$pos[i], &quot;cM /&quot;,peaks_mba$pos[i],&quot;MB\n(&quot;,peaks$lodcolumn[i],&quot; [scan1coeff; positions in cM])&quot;)
  #     )

  #plot_coef(coeff_sub, 
  #     gm$gmap, columns=1:2,
  #     bgcolor=&quot;gray95&quot;, legend=&quot;bottomleft&quot;, 
  #     main = paste(&quot;chr&quot;, chr=peaks$chr[i],&quot;; pos: &quot;, peaks$pos[i], &quot;cM /&quot;,peaks_mba$pos[i],&quot;MB\n(&quot;,peaks$lodcolumn[i],&quot;; 1.5 LOD drop interval [scan1coeff; positions in cM] ) &quot;)
  #     )


  plot_coef(blup, 
       gm$gmap, columns=1:2,
       bgcolor=&quot;gray95&quot;, legend=&quot;bottomleft&quot;, 
       main = paste0(&quot;chr: &quot;, chr=peaks$chr[i], &quot;; pos: &quot;, peaks$pos[i], &quot;cM / &quot;,peaks_mba$pos[i],&quot;MB\n(&quot;,peaks$phenotype[i],&quot; [scan1blup; positions in cM])&quot;)
       )

  plot_coef(blup_sub, 
       gm$gmap, columns=1:2,
       bgcolor=&quot;gray95&quot;, legend=&quot;bottomleft&quot;, 
       main = paste0(&quot;chr: &quot;, chr=peaks$chr[i],&quot;; pos: &quot;, peaks$pos[i], &quot;cM /&quot;,peaks_mba$pos[i],&quot;MB\n(&quot;,peaks$phenotype[i],&quot;; 1.5 LOD drop interval [scan1blup; positions in cM])&quot;)
       )


 # Plot 3
  #c2effB &lt;- scan1coef(pr.qc[,chr], gm$covar[peaks$lodcolumn[i]], model=&quot;binary&quot;, contrasts=cbind(a=c(-1, 0), d=c(0, -1)))
  #c2effBb &lt;- scan1blup(pr.qc[,chr], gm$covar[peaks$lodcolumn[i]], contrasts=cbind(a=c(-1, 0), d=c(0, -1)))
  ##c2effB &lt;- scan1coef(pr[,chr], cross$pheno[,peaks$lodcolumn[i]], addcovar = addcovar, contrasts=cbind(mu=c(1,1,1), a=c(-1, 0, 1), d=c(0, 1, 0)))
  ##c2effB &lt;- scan1coef(pr[,chr], cross$pheno[,peaks$lodcolumn[i]],Xcovar=Xcovar, contrasts=cbind(mu=c(1,1,1), a=c(-1, 0, 1), d=c(0, 1, 0)))
  #plot(c2effB, gm$gmap[chr], columns=1:2,
  #     bgcolor=&quot;gray95&quot;, legend=&quot;bottomleft&quot;, 
  #     main = paste(&quot;chr&quot;, chr=peaks$chr[i], &quot;pos&quot;, peaks$pos[i], &quot;(&quot;,peaks$lodcolumn[i],&quot;)&quot;)
  #     )
  #plot(c2effBb, gm$gmap[chr], columns=1:2,
  #     bgcolor=&quot;gray95&quot;, legend=&quot;bottomleft&quot;, 
  #     main = paste(&quot;chr&quot;, chr=peaks$chr[i], &quot;pos&quot;, peaks$pos[i], &quot;(&quot;,peaks$lodcolumn[i],&quot;)&quot;)
  #     )
  ##last_coef &lt;- unclass(c2effB)[nrow(c2effB),2:3] # last two coefficients
  ##for(t in seq(along=last_coef))
  ##  axis(side=4, at=last_coef[t], names(last_coef)[t], tick=FALSE)


  #Table 1
  chr = peaks_mba$chr[i]
  start=as.numeric(peaks_mba$ci_lo[i])
  end=as.numeric(peaks_mba$ci_hi[i])

  genesgss = query_genes(chr, start, end)

  write.csv(genesgss, file=paste0(&quot;data/ici.vs.pbs_genes_chr-&quot;,chr,&quot;_peak.marker-&quot;,peaks$marker[i],&quot;_lod.drop-1.5_5.batches_mis_conditional_2-peaks-chr3-4.csv&quot;), quote=F)

  rownames(genesgss) &lt;- NULL
  genesgss$strand_old = genesgss$strand
  genesgss$strand[genesgss$strand==&quot;+&quot;] &lt;- &quot;positive&quot;
  genesgss$strand[genesgss$strand==&quot;-&quot;] &lt;- &quot;negative&quot;

  #genesgss &lt;- 
  #table &lt;- 
  #genesgss[,c(&quot;chr&quot;,&quot;type&quot;,&quot;start&quot;,&quot;stop&quot;,&quot;strand&quot;,&quot;ID&quot;,&quot;Name&quot;,&quot;Dbxref&quot;,&quot;gene_id&quot;,&quot;mgi_type&quot;,&quot;description&quot;)] %&gt;%
  #kable(escape = F,align = c(&quot;ccccccccccc&quot;)) %&gt;%
  #kable_styling(&quot;striped&quot;, full_width = T) #%&gt;% 
  #cat #%&gt;%
  #column_spec(1, bold=TRUE)
#
  #print(kable(genesgss[,c(&quot;chr&quot;,&quot;type&quot;,&quot;start&quot;,&quot;stop&quot;,&quot;strand&quot;,&quot;ID&quot;,&quot;Name&quot;,&quot;Dbxref&quot;,&quot;gene_id&quot;,&quot;mgi_type&quot;,&quot;description&quot;)], escape = F,align = c(&quot;ccccccccccc&quot;)))

  print(kable(genesgss[,c(&quot;chr&quot;,&quot;type&quot;,&quot;start&quot;,&quot;stop&quot;,&quot;strand&quot;,&quot;ID&quot;,&quot;Name&quot;,&quot;Dbxref&quot;,&quot;gene_id&quot;,&quot;mgi_type&quot;,&quot;description&quot;)], &quot;html&quot;) %&gt;% kable_styling(&quot;striped&quot;, full_width = T))

  #table
  

}

} else {
  print(paste0(&quot;There are no peaks that have a LOD that reaches suggestive (p&lt;0.05) level of &quot;,summary(operm,alpha=0.05)$A, &quot; [autosomes]/&quot;,summary(operm,alpha=0.05)$X, &quot; [x-chromosome]&quot;))
}</code></pre>
<p>[1] “There are no peaks that have a LOD that reaches suggestive
(p&lt;0.05) level of 4.03405558593242 [autosomes]/4.47679966592167
[x-chromosome]”</p>
</div>
<div id="rqtl" class="section level2">
<h2>R/qtl</h2>
<div id="scanone" class="section level3">
<h3>scanone</h3>
<pre class="r"><code>gm</code></pre>
<pre><code>Object of class cross2 (crosstype &quot;bc&quot;)

Total individuals               138
No. genotyped individuals       138
No. phenotyped individuals      138
No. with both geno &amp; pheno      138

No. phenotypes                    1
No. covariates                    7
No. phenotype covariates          0

No. chromosomes                  20
Total markers                131355

No. markers by chr:
   1    2    3    4    5    6    7    8    9   10   11   12   13   14   15   16 
9956 9987 7848 7585 7609 7736 7399 6458 6713 6385 7143 6110 6082 5966 5346 5015 
  17   18   19    X 
5080 4605 3562 4770 </code></pre>
<pre class="r"><code>#detach(&quot;package:qtl2&quot;, unload=TRUE)
#library(qtl)

cross &lt;- qtl::read.cross(&quot;csv&quot;, file = &quot;data/ici.vs.pbs_gm_qtl_5.batches_mis_conditional_2-peaks-chr3-4.csv&quot;,alleles=c(&quot;A&quot;,&quot;B&quot;))</code></pre>
<pre><code> --Read the following data:
     138  individuals
     131355  markers
     5  phenotypes
 --Cross type: bc </code></pre>
<pre class="r"><code>cross &lt;- qtl::jittermap(cross)

summary(cross)</code></pre>
<pre><code>    Backcross

    No. individuals:    138 

    No. phenotypes:     5 
    Percent phenotyped: 100 100 100 100 100 

    No. chromosomes:    20 
        Autosomes:      1 2 3 4 5 6 7 8 9 10 11 12 13 14 15 16 17 18 19 
        X chr:          X 

    Total markers:      131355 
    No. markers:        9956 9987 7848 7585 7609 7736 7399 6458 6713 6385 7143 
                        6110 6082 5966 5346 5015 5080 4605 3562 4770 
    Percent genotyped:  99.5 
    Genotypes (%):    
          Autosomes:    AA:88.1  AB:11.9 
       X chromosome:    AA:93.9  AB:6.1  </code></pre>
<pre class="r"><code>cross.probs &lt;- qtl::calc.genoprob(cross)

print(&quot;method == hk&quot;)</code></pre>
<pre><code>[1] &quot;method == hk&quot;</code></pre>
<pre class="r"><code>add.covars = qtl::pull.pheno(cross.probs, c(&quot;UNCHS008487&quot;,&quot;UNC8250659&quot;))

scanone.hk &lt;-qtl::scanone(cross.probs, pheno.col=&quot;ICI.vs.PBS&quot; , model=&quot;binary&quot;, method=&quot;hk&quot;, addcovar = add.covars)
operm.hk &lt;- qtl::scanone(cross.probs, method = &quot;hk&quot;, pheno.col=&quot;ICI.vs.PBS&quot;, n.perm = 10, perm.Xsp = TRUE, model=&quot;binary&quot;, verbose=FALSE, addcovar = add.covars)
plot(operm.hk)</code></pre>
<p><img src="figure/4.1.1_qtl.analysis_binary_ici.vs.pbs_5.batches_mis_conditional_2-peaks.Rmd/scanone-1.png" width="912" style="display: block; margin: auto;" /></p>
<pre class="r"><code>print(summary(operm.hk, alpha=c(0.01,  0.05, 0.1)))</code></pre>
<pre><code>Autosome LOD thresholds (10 permutations)
     lod
1%  4.21
5%  3.88
10% 3.47

X chromosome LOD thresholds (182 permutations)
     lod
1%  3.17
5%  3.17
10% 3.16</code></pre>
<pre class="r"><code>#plot(scanone.hk, bandcol = &quot;grey90&quot;,lty=1, cex=1, col = &quot;steelblue&quot;)  
#qtl::add.threshold(scanone.hk,  perms= operm.hk, alpha=0.01, col = &#39;blue&#39;)
#qtl::add.threshold(scanone.hk,  perms= operm.hk, alpha=0.05, col = &#39;red&#39;)
#qtl::add.threshold(scanone.hk,  perms= operm.hk, alpha=0.1, col = &#39;purple&#39;)

ymx &lt;- maxlod(out) # overall maximum LOD score
plot(scanone.hk, bandcol = &quot;grey90&quot;,lty=1, cex=1, col = &quot;slateblue&quot;, ylim=c(0, ymx+0.5))
title(main = paste0(colnames(out), &quot; [positions in cM]&quot;)) 
qtl::add.threshold(scanone.hk,  perms= operm.hk, alpha=0.01, col = &#39;blue&#39;)
qtl::add.threshold(scanone.hk,  perms= operm.hk, alpha=0.05, col = &#39;red&#39;)
qtl::add.threshold(scanone.hk,  perms= operm.hk, alpha=0.1, col = &#39;purple&#39;)</code></pre>
<p><img src="figure/4.1.1_qtl.analysis_binary_ici.vs.pbs_5.batches_mis_conditional_2-peaks.Rmd/scanone-2.png" width="912" style="display: block; margin: auto;" /></p>
<pre class="r"><code>ymx &lt;- 14
plot(scanone.hk, bandcol = &quot;grey90&quot;,lty=1, cex=1, col = &quot;slateblue&quot;, ylim=c(0, ymx+0.5))
title(main = paste0(colnames(out), &quot; [positions in cM]\n(using same scale as ici vs. eoi for easier comparison)&quot;))
qtl::add.threshold(scanone.hk,  perms= operm.hk, alpha=0.01, col = &#39;blue&#39;)
qtl::add.threshold(scanone.hk,  perms= operm.hk, alpha=0.05, col = &#39;red&#39;)
qtl::add.threshold(scanone.hk,  perms= operm.hk, alpha=0.1, col = &#39;purple&#39;)</code></pre>
<p><img src="figure/4.1.1_qtl.analysis_binary_ici.vs.pbs_5.batches_mis_conditional_2-peaks.Rmd/scanone-3.png" width="912" style="display: block; margin: auto;" /></p>
<pre class="r"><code>print(as.data.frame(summary(scanone.hk, perms=operm.hk, pvalues=TRUE, format=&quot;allpeaks&quot;)))</code></pre>
<pre><code>            chr       pos      lod      pval
UNC1197721    1 47.673577 1.868502 1.0000000
UNCHS005266   2 40.908610 1.873368 1.0000000
UNC6045859    3 50.620447 1.870152 1.0000000
JAX00123711   4 53.542073 1.799165 1.0000000
JAX00574407   5  6.213413 3.388862 0.1052020
UNCHS017906   6 37.001766 1.868162 1.0000000
JAX00233659   7 87.998157 1.870072 1.0000000
JAX00667121   8 24.443849 2.382541 0.7192282
UNC17203329   9 68.519175 1.323467 1.0000000
JAX00302150  10 74.547218 2.532706 0.6196652
UNC20281687  11 67.380801 1.859331 1.0000000
UNCHS034188  12 38.836978 1.870151 1.0000000
JAX00350973  13  5.743505 1.378528 1.0000000
JAX00380911  14 30.154517 2.381660 0.7192282
UNC25275535  15 11.968141 1.402063 1.0000000
UNC26790882  16 30.832641 2.019242 0.8169482
UNCHS045288  17 60.699043 2.034920 0.8169482
JAX00450853  18  2.913076 2.621475 0.6196652
UNCHS047190  19  8.712408 1.804318 1.0000000
XiB2          X 45.699951 1.870087 0.9986399</code></pre>
<pre class="r"><code>print(&quot;all peaks with a p-value less or equal to 0.05 (suggestive)&quot;)</code></pre>
<pre><code>[1] &quot;all peaks with a p-value less or equal to 0.05 (suggestive)&quot;</code></pre>
<pre class="r"><code>print(as.data.frame(summary(scanone.hk, perms=operm.hk, alpha=0.05, pvalues=TRUE, format=&quot;allpeaks&quot;)))</code></pre>
<pre><code>[1] chr pos lod
&lt;0 rows&gt; (or 0-length row.names)</code></pre>
<pre class="r"><code>#print(&quot;method == ehk&quot;)

#scanone.ehk &lt;-qtl::scanone(cross.probs, pheno.col=&quot;ICI.vs.PBS&quot; , model=&quot;binary&quot;, method=&quot;ehk&quot;)
#operm.ehk &lt;- qtl::scanone(cross.probs, method = &quot;ehk&quot;, pheno.col=&quot;ICI.vs.PBS&quot;, n.perm = 10, perm.Xsp = TRUE, model=&quot;binary&quot;, verbose=FALSE)
#plot(operm.ehk)
#print(summary(operm.ehk, alpha=c(0.01,  0.05, 0.1)))

#plot(scanone.ehk, bandcol = &quot;grey90&quot;,lty=1, cex=1, col = &quot;steelblue&quot;)  
#qtl::add.threshold(scanone.ehk,  perms= operm.ehk, alpha=0.01, col = &#39;blue&#39;)
#qtl::add.threshold(scanone.ehk,  perms= operm.ehk, alpha=0.05, col = &#39;red&#39;)
#qtl::add.threshold(scanone.ehk,  perms= operm.ehk, alpha=0.1, col = &#39;purple&#39;)

#print(as.data.frame(summary(scanone.ehk)))
#print(as.data.frame(summary(scanone.ehk, perms=operm.ehk, alpha=0.05, pvalues=TRUE, format=&quot;allpeaks&quot;)))</code></pre>
</div>
</div>
</div>
<div
id="conditionaing-on-eoi-vs.-ici-chr-3-peak-unchs008487-chr-10-peak-unc18240977"
class="section level1">
<h1>Conditionaing on eoi vs. ici chr 3 peak (UNCHS008487) &amp; chr 10
peak (UNC18240977)</h1>
<div id="genome-wide-scan-2" class="section level2">
<h2>Genome-wide scan</h2>
<pre class="r"><code>Xcovar &lt;- get_x_covar(gm)
addcovar = model.matrix(~UNCHS008487+UNC18240977, data = covars)[,-1]

#K &lt;- calc_kinship(pr.qc, type = &quot;loco&quot;)
#heatmap(K[[1]])
#K.overall &lt;- calc_kinship(pr.qc, type = &quot;overall&quot;)
#heatmap(K.overall)
kinship &lt;- calc_kinship(pr.qc)
heatmap(kinship)</code></pre>
<p><img src="figure/4.1.1_qtl.analysis_binary_ici.vs.pbs_5.batches_mis_conditional_2-peaks.Rmd/permutation2-1.png" width="912" style="display: block; margin: auto;" /></p>
<pre class="r"><code>#operm &lt;- scan1perm(pr.qc, gm$covar$phenos, Xcovar=Xcovar, n_perm=2000)
#operm &lt;- scan1perm(pr.qc, gm$covar$phenos, addcovar = addcovar, n_perm=2000)
#operm &lt;- scan1perm(pr.qc, gm$covar$phenos, n_perm=2000)
operm &lt;- scan1perm(pr.qc, gm$covar[&quot;ICI.vs.PBS&quot;], model=&quot;binary&quot;, n_perm=10, perm_Xsp=TRUE, chr_lengths=chr_lengths(gm$gmap), addcovar = addcovar)

summary_table&lt;-data.frame(unclass(summary(operm, alpha=c(0.01,  0.05, 0.1))))
names(summary_table) &lt;- c(&quot;autosomes&quot;,&quot;X&quot;)
summary_table$significance.level &lt;- rownames(summary_table)

rownames(summary_table) &lt;- NULL

summary_table[c(3,1:2)] %&gt;%
  kable(escape = F,align = c(&quot;ccc&quot;)) %&gt;%
  kable_styling(&quot;striped&quot;, full_width = T) %&gt;%
  column_spec(1, bold=TRUE)</code></pre>
<table class="table table-striped" style="margin-left: auto; margin-right: auto;">
<thead>
<tr>
<th style="text-align:center;">
significance.level
</th>
<th style="text-align:center;">
autosomes
</th>
<th style="text-align:center;">
X
</th>
</tr>
</thead>
<tbody>
<tr>
<td style="text-align:center;font-weight: bold;">
0.01
</td>
<td style="text-align:center;">
3.764568
</td>
<td style="text-align:center;">
3.519636
</td>
</tr>
<tr>
<td style="text-align:center;font-weight: bold;">
0.05
</td>
<td style="text-align:center;">
3.758664
</td>
<td style="text-align:center;">
3.355931
</td>
</tr>
<tr>
<td style="text-align:center;font-weight: bold;">
0.1
</td>
<td style="text-align:center;">
3.751265
</td>
<td style="text-align:center;">
3.141856
</td>
</tr>
</tbody>
</table>
<p>The figures below show QTL maps for each phenotype</p>
<pre class="r"><code>out &lt;- scan1(pr.qc, gm$covar[&quot;ICI.vs.PBS&quot;], Xcovar=Xcovar, model=&quot;binary&quot;, addcovar = addcovar)

summary_table&lt;-data.frame(unclass(summary(operm, alpha=c(0.01,  0.05, 0.1))))

plot_lod&lt;-function(out,map){
  for (i in 1:dim(out)[2]){
    #png(filename=paste0(&quot;/Users/chenm/Documents/qtl/Jai/&quot;,colnames(out)[i],  &quot;_lod.png&quot;))
    
    ymx &lt;- maxlod(out) # overall maximum LOD score
    plot(out, map, lodcolumn=i, col=&quot;slateblue&quot;, ylim=c(0, ymx+0.5))
    ##legend(&quot;topright&quot;, lwd=2, colnames(out)[i], bg=&quot;gray90&quot;)
    title(main = paste0(colnames(out)[i], &quot; [positions in cM]&quot;))
    add_threshold(map,  summary(operm,alpha=0.1), col = &#39;purple&#39;)
    add_threshold(map,  summary(operm, alpha=0.05), col = &#39;red&#39;)
    add_threshold(map,  summary(operm, alpha=0.01), col = &#39;blue&#39;)

    #par(mar=c(5.1, 6.1, 1.1, 1.1))
    ymx &lt;- 14 # overall maximum LOD score
    plot(out, map, lodcolumn=i, col=&quot;slateblue&quot;, ylim=c(0, ymx+0.5))
    ##legend(&quot;topright&quot;, lwd=2, colnames(out)[i], bg=&quot;gray90&quot;)
    title(main = paste0(colnames(out)[i], &quot; [positions in cM] \n(using same scale as eoi  vs. ici for easier comparison)&quot;))
    add_threshold(map,  summary(operm, alpha=0.1), col = &#39;purple&#39;)
    add_threshold(map,  summary(operm, alpha=0.05), col = &#39;red&#39;)
    add_threshold(map,  summary(operm, alpha=0.01), col = &#39;blue&#39;)
    #for (j in 1: dim(summary_table)[1]){
    #  abline(h=summary_table[j, i],col=&quot;red&quot;)
    #  text(x=400, y =summary_table[j, i]+0.12, labels = paste(&quot;p=&quot;, row.names(summary_table)[j]))
    #}
    #dev.off()
  }
}

plot_lod(out,gm$gmap)</code></pre>
<p><img src="figure/4.1.1_qtl.analysis_binary_ici.vs.pbs_5.batches_mis_conditional_2-peaks.Rmd/Genome-wide%20scan2-1.png" width="912" style="display: block; margin: auto;" /><img src="figure/4.1.1_qtl.analysis_binary_ici.vs.pbs_5.batches_mis_conditional_2-peaks.Rmd/Genome-wide%20scan2-2.png" width="912" style="display: block; margin: auto;" /></p>
</div>
<div id="lod-peaks-1" class="section level2">
<h2>LOD peaks</h2>
<p>The table below shows QTL peaks associated with the phenotype. We use
the 95% threshold from the permutations to find peaks.</p>
<div id="centimorgan-cm-1" class="section level3">
<h3>Centimorgan (cM)</h3>
<pre class="r"><code>peaks &lt;- find_peaks(out, gm$gmap, threshold=summary(operm,alpha=0.05)$A, thresholdX = summary(operm,alpha=0.05)$X, peakdrop=3, drop=1.5)

if(nrow(peaks) &gt;0){
peaks$marker &lt;- find_marker(gm$gmap, chr=peaks$chr,pos=peaks$pos)
names(peaks)[2] &lt;- c(&quot;phenotype&quot;)
peaks &lt;- peaks[-1]

rownames(peaks) &lt;- NULL
print(kable(peaks, escape = F, align = c(&quot;cccccccc&quot;), &quot;html&quot;) 
  %&gt;% kable_styling(&quot;striped&quot;, full_width = T)%&gt;%
  column_spec(1, bold=TRUE)
  )

#plot only peak chromosomes

plot_lod_chr&lt;-function(out,map,chrom){
  for (i in 1:dim(out)[2]){
    #png(filename=paste0(&quot;/Users/chenm/Documents/qtl/Jai/&quot;,colnames(out)[i],  &quot;_lod.png&quot;))
    
    #par(mar=c(5.1, 6.1, 1.1, 1.1))
    ymx &lt;- maxlod(out) # overall maximum LOD score
    plot(out, map, chr = chrom, lodcolumn=i, col=&quot;slateblue&quot;, ylim=c(0, ymx+0.5))
    #legend(&quot;topright&quot;, lwd=2, colnames(out)[i], bg=&quot;gray90&quot;)
    title(main = paste0(colnames(out)[i], &quot; - chr&quot;, chrom, &quot; [positions in cM]&quot;))
    add_threshold(map,  summary(operm,alpha=0.1), col = &#39;purple&#39;)
    add_threshold(map,  summary(operm, alpha=0.05), col = &#39;red&#39;)
    add_threshold(map,  summary(operm, alpha=0.01), col = &#39;blue&#39;)
    #for (j in 1: dim(summary_table)[1]){
    #  abline(h=summary_table[j, i],col=&quot;red&quot;)
    #  text(x=400, y =summary_table[j, i]+0.12, labels = paste(&quot;p=&quot;, row.names(summary_table)[j]))
    #}
    #dev.off()

    
    ymx &lt;- 14
    plot(out, map, chr = chrom, lodcolumn=i, col=&quot;slateblue&quot;, ylim=c(0, ymx+0.5))
    #legend(&quot;topright&quot;, lwd=2, colnames(out)[i], bg=&quot;gray90&quot;)
    title(main = paste0(colnames(out)[i], &quot; - chr&quot;, chrom, &quot; [positions in cM]\n(using same scale as eoi vs. ici for easier comparison)&quot;))
    add_threshold(map,  summary(operm,alpha=0.1), col = &#39;purple&#39;)
    add_threshold(map,  summary(operm, alpha=0.05), col = &#39;red&#39;)
    add_threshold(map,  summary(operm, alpha=0.01), col = &#39;blue&#39;)

  }
}


for(i in unique(peaks$chr)){
#for (i in 1:nrow(peaks)){
  #plot_lod_chr(out,gm$gmap, peaks$chr[i])
  plot_lod_chr(out,gm$gmap, i)
}

} else {
  print(paste0(&quot;There are no peaks that have a LOD that reaches suggestive (p&lt;0.05) level of &quot;,summary(operm,alpha=0.05)$A, &quot; [autosomes]/&quot;,summary(operm,alpha=0.05)$X, &quot; [x-chromosome]&quot;))
}</code></pre>
<p>[1] “There are no peaks that have a LOD that reaches suggestive
(p&lt;0.05) level of 3.75866362627187 [autosomes]/3.3559311891303
[x-chromosome]”</p>
</div>
<div id="megabase-mb-1" class="section level3">
<h3>Megabase (MB)</h3>
<pre class="r"><code>print(&quot;peaks in MB positions&quot;)</code></pre>
<p>[1] “peaks in MB positions”</p>
<pre class="r"><code>peaks_mba &lt;- find_peaks(out, gm$pmap, threshold=summary(operm,alpha=0.05)$A, thresholdX = summary(operm,alpha=0.05)$X, peakdrop=3, drop=1.5)

if(nrow(peaks) &gt;0){
peaks_mba$marker &lt;- find_marker(gm$pmap, chr=peaks_mba$chr,pos=peaks_mba$pos)
names(peaks_mba)[2] &lt;- c(&quot;phenotype&quot;)
peaks_mba &lt;- peaks_mba[-1]

#peaks_mbl &lt;- list()
##corresponding info in Mb
#for(i in 1:nrow(peaks)){
#  #lodindex &lt;- peaks$lodindex[i]
#  phenotype &lt;- peaks$phenotype[i]
#  chr &lt;- as.character(peaks$chr[i])
#  lod &lt;- peaks$lod[i]
#  mark &lt;- peaks$marker[i]
#  pos &lt;- mapdf[mapdf$marker==mark,]$pmapdf
#  ci_lo &lt;- mapdfnd$pmapdf[which(mapdfnd$gmapdf == peaks$ci_lo[i] &amp; mapdfnd$chr == peaks$chr[i])]
#  ci_hi &lt;- mapdfnd$pmapdf[which(mapdfnd$gmapdf == peaks$ci_hi[i] &amp; mapdfnd$chr == peaks$chr[i])]
#  peaks_mb=as.data.frame(cbind(phenotype, chr, pos, lod, ci_lo, ci_hi, mark))
#  names(peaks_mb)[7] &lt;- c(&quot;marker&quot;)
#  peaks_mbl[[i]] &lt;- peaks_mb
#}
#peaks_mba2 &lt;- do.call(rbind, peaks_mbl)
#peaks_mba2 &lt;- as.data.frame(peaks_mba)
#peaks_mba[,c(&quot;chr&quot;, &quot;pos&quot;, &quot;lod&quot;, &quot;ci_lo&quot;, &quot;ci_hi&quot;)] &lt;- sapply(peaks_mba[,c(&quot;chr&quot;, &quot;pos&quot;, &quot;lod&quot;, &quot;ci_lo&quot;, &quot;ci_hi&quot;)], as.numeric)

rownames(peaks_mba) &lt;- NULL
print(kable(peaks_mba, escape = F, align = c(&quot;cccccccc&quot;), &quot;html&quot;) 
  %&gt;% kable_styling(&quot;striped&quot;, full_width = T)%&gt;%
  column_spec(1, bold=TRUE)
  )

plot_lod_chr_mb&lt;-function(out,map,chrom){
  for (i in 1:dim(out)[2]){
    #png(filename=paste0(&quot;/Users/chenm/Documents/qtl/Jai/&quot;,colnames(out)[i],  &quot;_lod.png&quot;))
    
    #par(mar=c(5.1, 6.1, 1.1, 1.1))
    ymx &lt;- maxlod(out) # overall maximum LOD score
    plot(out, map, chr = chrom, lodcolumn=i, col=&quot;slateblue&quot;, ylim=c(0, ymx+0.5))
    #legend(&quot;topright&quot;, lwd=2, colnames(out)[i], bg=&quot;gray90&quot;)
    title(main = paste0(colnames(out)[i], &quot; - chr&quot;, chrom, &quot; [positions in MB]&quot;))
    add_threshold(map,  summary(operm,alpha=0.1), col = &#39;purple&#39;)
    add_threshold(map,  summary(operm, alpha=0.05), col = &#39;red&#39;)
    add_threshold(map,  summary(operm, alpha=0.01), col = &#39;blue&#39;)
    #for (j in 1: dim(summary_table)[1]){
    #  abline(h=summary_table[j, i],col=&quot;red&quot;)
    #  text(x=400, y =summary_table[j, i]+0.12, labels = paste(&quot;p=&quot;, row.names(summary_table)[j]))
    #}
    #dev.off()

    ymx &lt;- 14
    plot(out, map, chr = chrom, lodcolumn=i, col=&quot;slateblue&quot;, ylim=c(0, ymx+0.5))
    #legend(&quot;topright&quot;, lwd=2, colnames(out)[i], bg=&quot;gray90&quot;)
    title(main = paste0(colnames(out)[i], &quot; - chr&quot;, chrom, &quot; [positions in MB]\n(using same scale as eoi vs. ici for easier comparison)&quot;))
    add_threshold(map,  summary(operm,alpha=0.1), col = &#39;purple&#39;)
    add_threshold(map,  summary(operm, alpha=0.05), col = &#39;red&#39;)
    add_threshold(map,  summary(operm, alpha=0.01), col = &#39;blue&#39;)


  }
}

for(i in unique(peaks_mba$chr)){
#for (i in 1:nrow(peaks_mba)){
  #plot_lod_chr_mb(out,gm$pmap, peaks_mba$chr[i])
  plot_lod_chr_mb(out,gm$pmap,i)
}

} else {
  print(paste0(&quot;There are no peaks that have a LOD that reaches suggestive (p&lt;0.05) level of &quot;,summary(operm,alpha=0.05)$A, &quot; [autosomes]/&quot;,summary(operm,alpha=0.05)$X, &quot; [x-chromosome]&quot;))
}</code></pre>
<p>[1] “There are no peaks that have a LOD that reaches suggestive
(p&lt;0.05) level of 3.75866362627187 [autosomes]/3.3559311891303
[x-chromosome]”</p>
</div>
</div>
<div id="qtl-effects-1" class="section level2">
<h2>QTL effects</h2>
<p>For each peak LOD location we give a list of gene</p>
<pre class="r"><code>query_variants &lt;- create_variant_query_func(&quot;/Users/corneb/Documents/MyJax/CS/Projects/support.files/qtl2/cc_variants.sqlite&quot;)
query_genes &lt;- create_gene_query_func(&quot;/Users/corneb/Documents/MyJax/CS/Projects/support.files/qtl2/mouse_genes_mgi.sqlite&quot;)

if(nrow(peaks) &gt;0){
for (i in 1:nrow(peaks)){
#for (i in 1:1){
  #Plot 1
  g &lt;- maxmarg(pr.qc, gm$gmap, chr=peaks$chr[i], pos=peaks$pos[i], return_char=TRUE)
  #png(filename=paste0(&quot;/Users/chenm/Documents/qtl/Jai/&quot;,&quot;qtl_effect_&quot;, i, &quot;.png&quot;))
  #par(mar=c(4.1, 4.1, 1.5, 0.6))
  plot_pxg(g, gm$covar[,peaks$phenotype[i]], ylab=peaks$phenotype[i], sort=FALSE)
  title(main = paste0(&quot;chr: &quot;, chr=peaks$chr[i],&quot;; pos: &quot;, peaks$pos[i], &quot;cM /&quot;,peaks_mba$pos[i],&quot;MB\n(&quot;,peaks$phenotype[i],&quot; )&quot;), line=0.2)
  ##dev.off()

  chr = peaks$chr[i]

# Plot 2
  pr_sub &lt;- pull_genoprobint(pr.qc, gm$gmap, chr, c(peaks$ci_lo[i], peaks$ci_hi[i]))
  #coeff &lt;- scan1coef(pr[,chr], cross$pheno[,peaks$lodcolumn[i]], addcovar = addcovar)
  #coeff &lt;- scan1coef(pr[,chr], cross$pheno[,peaks$lodcolumn[i]], Xcovar=Xcovar)
  #coeff &lt;- scan1coef(pr.qc[,chr], gm$covar[peaks$lodcolumn[i]], model=&quot;binary&quot;)
  #coeff_sub &lt;- scan1coef(pr_sub[,chr], gm$covar[peaks$lodcolumn[i]], model=&quot;binary&quot;)
  blup &lt;- scan1blup(pr.qc[,chr], gm$covar[peaks$phenotype[i]], addcovar = addcovar)
  blup_sub &lt;- scan1blup(pr_sub[,chr], gm$covar[peaks$phenotype[i]], addcovar = addcovar)

  write.csv(as.data.frame(blup_sub), paste0(&quot;data/ici.vs.pbs_blup_sub_chr-&quot;,chr,&quot;_peak.marker-&quot;,peaks$marker[i],&quot;_lod.drop-1.5_5.batches_mis_conditional_2-peaks-chr3-10.csv&quot;), quote=F)

  #plot_coef(coeff, 
  #     gm$gmap, columns=1:2,
  #     bgcolor=&quot;gray95&quot;, legend=&quot;bottomleft&quot;, 
  #     main = paste(&quot;chr&quot;, chr=peaks$chr[i],&quot;; pos: &quot;, peaks$pos[i], &quot;cM /&quot;,peaks_mba$pos[i],&quot;MB\n(&quot;,peaks$lodcolumn[i],&quot; [scan1coeff; positions in cM])&quot;)
  #     )

  #plot_coef(coeff_sub, 
  #     gm$gmap, columns=1:2,
  #     bgcolor=&quot;gray95&quot;, legend=&quot;bottomleft&quot;, 
  #     main = paste(&quot;chr&quot;, chr=peaks$chr[i],&quot;; pos: &quot;, peaks$pos[i], &quot;cM /&quot;,peaks_mba$pos[i],&quot;MB\n(&quot;,peaks$lodcolumn[i],&quot;; 1.5 LOD drop interval [scan1coeff; positions in cM] ) &quot;)
  #     )


  plot_coef(blup, 
       gm$gmap, columns=1:2,
       bgcolor=&quot;gray95&quot;, legend=&quot;bottomleft&quot;, 
       main = paste0(&quot;chr: &quot;, chr=peaks$chr[i], &quot;; pos: &quot;, peaks$pos[i], &quot;cM / &quot;,peaks_mba$pos[i],&quot;MB\n(&quot;,peaks$phenotype[i],&quot; [scan1blup; positions in cM])&quot;)
       )

  plot_coef(blup_sub, 
       gm$gmap, columns=1:2,
       bgcolor=&quot;gray95&quot;, legend=&quot;bottomleft&quot;, 
       main = paste0(&quot;chr: &quot;, chr=peaks$chr[i],&quot;; pos: &quot;, peaks$pos[i], &quot;cM /&quot;,peaks_mba$pos[i],&quot;MB\n(&quot;,peaks$phenotype[i],&quot;; 1.5 LOD drop interval [scan1blup; positions in cM])&quot;)
       )


 # Plot 3
  #c2effB &lt;- scan1coef(pr.qc[,chr], gm$covar[peaks$lodcolumn[i]], model=&quot;binary&quot;, contrasts=cbind(a=c(-1, 0), d=c(0, -1)))
  #c2effBb &lt;- scan1blup(pr.qc[,chr], gm$covar[peaks$lodcolumn[i]], contrasts=cbind(a=c(-1, 0), d=c(0, -1)))
  ##c2effB &lt;- scan1coef(pr[,chr], cross$pheno[,peaks$lodcolumn[i]], addcovar = addcovar, contrasts=cbind(mu=c(1,1,1), a=c(-1, 0, 1), d=c(0, 1, 0)))
  ##c2effB &lt;- scan1coef(pr[,chr], cross$pheno[,peaks$lodcolumn[i]],Xcovar=Xcovar, contrasts=cbind(mu=c(1,1,1), a=c(-1, 0, 1), d=c(0, 1, 0)))
  #plot(c2effB, gm$gmap[chr], columns=1:2,
  #     bgcolor=&quot;gray95&quot;, legend=&quot;bottomleft&quot;, 
  #     main = paste(&quot;chr&quot;, chr=peaks$chr[i], &quot;pos&quot;, peaks$pos[i], &quot;(&quot;,peaks$lodcolumn[i],&quot;)&quot;)
  #     )
  #plot(c2effBb, gm$gmap[chr], columns=1:2,
  #     bgcolor=&quot;gray95&quot;, legend=&quot;bottomleft&quot;, 
  #     main = paste(&quot;chr&quot;, chr=peaks$chr[i], &quot;pos&quot;, peaks$pos[i], &quot;(&quot;,peaks$lodcolumn[i],&quot;)&quot;)
  #     )
  ##last_coef &lt;- unclass(c2effB)[nrow(c2effB),2:3] # last two coefficients
  ##for(t in seq(along=last_coef))
  ##  axis(side=4, at=last_coef[t], names(last_coef)[t], tick=FALSE)


  #Table 1
  chr = peaks_mba$chr[i]
  start=as.numeric(peaks_mba$ci_lo[i])
  end=as.numeric(peaks_mba$ci_hi[i])

  genesgss = query_genes(chr, start, end)

  write.csv(genesgss, file=paste0(&quot;data/ici.vs.pbs_genes_chr-&quot;,chr,&quot;_peak.marker-&quot;,peaks$marker[i],&quot;_lod.drop-1.5_5.batches_mis_conditional_2-peaks-chr3-10.csv&quot;), quote=F)

  rownames(genesgss) &lt;- NULL
  genesgss$strand_old = genesgss$strand
  genesgss$strand[genesgss$strand==&quot;+&quot;] &lt;- &quot;positive&quot;
  genesgss$strand[genesgss$strand==&quot;-&quot;] &lt;- &quot;negative&quot;

  #genesgss &lt;- 
  #table &lt;- 
  #genesgss[,c(&quot;chr&quot;,&quot;type&quot;,&quot;start&quot;,&quot;stop&quot;,&quot;strand&quot;,&quot;ID&quot;,&quot;Name&quot;,&quot;Dbxref&quot;,&quot;gene_id&quot;,&quot;mgi_type&quot;,&quot;description&quot;)] %&gt;%
  #kable(escape = F,align = c(&quot;ccccccccccc&quot;)) %&gt;%
  #kable_styling(&quot;striped&quot;, full_width = T) #%&gt;% 
  #cat #%&gt;%
  #column_spec(1, bold=TRUE)
#
  #print(kable(genesgss[,c(&quot;chr&quot;,&quot;type&quot;,&quot;start&quot;,&quot;stop&quot;,&quot;strand&quot;,&quot;ID&quot;,&quot;Name&quot;,&quot;Dbxref&quot;,&quot;gene_id&quot;,&quot;mgi_type&quot;,&quot;description&quot;)], escape = F,align = c(&quot;ccccccccccc&quot;)))

  print(kable(genesgss[,c(&quot;chr&quot;,&quot;type&quot;,&quot;start&quot;,&quot;stop&quot;,&quot;strand&quot;,&quot;ID&quot;,&quot;Name&quot;,&quot;Dbxref&quot;,&quot;gene_id&quot;,&quot;mgi_type&quot;,&quot;description&quot;)], &quot;html&quot;) %&gt;% kable_styling(&quot;striped&quot;, full_width = T))

  #table
  

}

} else {
  print(paste0(&quot;There are no peaks that have a LOD that reaches suggestive (p&lt;0.05) level of &quot;,summary(operm,alpha=0.05)$A, &quot; [autosomes]/&quot;,summary(operm,alpha=0.05)$X, &quot; [x-chromosome]&quot;))
}</code></pre>
<p>[1] “There are no peaks that have a LOD that reaches suggestive
(p&lt;0.05) level of 3.75866362627187 [autosomes]/3.3559311891303
[x-chromosome]”</p>
</div>
<div id="rqtl-1" class="section level2">
<h2>R/qtl</h2>
<div id="scanone-1" class="section level3">
<h3>scanone</h3>
<pre class="r"><code>gm</code></pre>
<pre><code>Object of class cross2 (crosstype &quot;bc&quot;)

Total individuals               138
No. genotyped individuals       138
No. phenotyped individuals      138
No. with both geno &amp; pheno      138

No. phenotypes                    1
No. covariates                    7
No. phenotype covariates          0

No. chromosomes                  20
Total markers                131355

No. markers by chr:
   1    2    3    4    5    6    7    8    9   10   11   12   13   14   15   16 
9956 9987 7848 7585 7609 7736 7399 6458 6713 6385 7143 6110 6082 5966 5346 5015 
  17   18   19    X 
5080 4605 3562 4770 </code></pre>
<pre class="r"><code>#detach(&quot;package:qtl2&quot;, unload=TRUE)
#library(qtl)

cross &lt;- qtl::read.cross(&quot;csv&quot;, file = &quot;data/ici.vs.pbs_gm_qtl_5.batches_mis_conditional_2-peaks-chr3-10.csv&quot;,alleles=c(&quot;A&quot;,&quot;B&quot;))</code></pre>
<pre><code> --Read the following data:
     138  individuals
     131355  markers
     5  phenotypes
 --Cross type: bc </code></pre>
<pre class="r"><code>cross &lt;- qtl::jittermap(cross)

summary(cross)</code></pre>
<pre><code>    Backcross

    No. individuals:    138 

    No. phenotypes:     5 
    Percent phenotyped: 100 100 100 100 100 

    No. chromosomes:    20 
        Autosomes:      1 2 3 4 5 6 7 8 9 10 11 12 13 14 15 16 17 18 19 
        X chr:          X 

    Total markers:      131355 
    No. markers:        9956 9987 7848 7585 7609 7736 7399 6458 6713 6385 7143 
                        6110 6082 5966 5346 5015 5080 4605 3562 4770 
    Percent genotyped:  99.5 
    Genotypes (%):    
          Autosomes:    AA:88.1  AB:11.9 
       X chromosome:    AA:93.9  AB:6.1  </code></pre>
<pre class="r"><code>cross.probs &lt;- qtl::calc.genoprob(cross)

print(&quot;method == hk&quot;)</code></pre>
<pre><code>[1] &quot;method == hk&quot;</code></pre>
<pre class="r"><code>add.covars = qtl::pull.pheno(cross.probs, c(&quot;UNCHS008487&quot;,&quot;UNC18240977&quot;))

scanone.hk &lt;-qtl::scanone(cross.probs, pheno.col=&quot;ICI.vs.PBS&quot; , model=&quot;binary&quot;, method=&quot;hk&quot;, addcovar = add.covars)
operm.hk &lt;- qtl::scanone(cross.probs, method = &quot;hk&quot;, pheno.col=&quot;ICI.vs.PBS&quot;, n.perm = 10, perm.Xsp = TRUE, model=&quot;binary&quot;, verbose=FALSE, addcovar = add.covars)
plot(operm.hk)</code></pre>
<p><img src="figure/4.1.1_qtl.analysis_binary_ici.vs.pbs_5.batches_mis_conditional_2-peaks.Rmd/scanone2-1.png" width="912" style="display: block; margin: auto;" /></p>
<pre class="r"><code>print(summary(operm.hk, alpha=c(0.01,  0.05, 0.1)))</code></pre>
<pre><code>Autosome LOD thresholds (10 permutations)
     lod
1%  3.87
5%  3.76
10% 3.62

X chromosome LOD thresholds (182 permutations)
     lod
1%  3.64
5%  3.56
10% 3.45</code></pre>
<pre class="r"><code>#plot(scanone.hk, bandcol = &quot;grey90&quot;,lty=1, cex=1, col = &quot;steelblue&quot;)  
#qtl::add.threshold(scanone.hk,  perms= operm.hk, alpha=0.01, col = &#39;blue&#39;)
#qtl::add.threshold(scanone.hk,  perms= operm.hk, alpha=0.05, col = &#39;red&#39;)
#qtl::add.threshold(scanone.hk,  perms= operm.hk, alpha=0.1, col = &#39;purple&#39;)

ymx &lt;- maxlod(out) # overall maximum LOD score
plot(scanone.hk, bandcol = &quot;grey90&quot;,lty=1, cex=1, col = &quot;slateblue&quot;, ylim=c(0, ymx+0.5))
title(main = paste0(colnames(out), &quot; [positions in cM]&quot;)) 
qtl::add.threshold(scanone.hk,  perms= operm.hk, alpha=0.01, col = &#39;blue&#39;)
qtl::add.threshold(scanone.hk,  perms= operm.hk, alpha=0.05, col = &#39;red&#39;)
qtl::add.threshold(scanone.hk,  perms= operm.hk, alpha=0.1, col = &#39;purple&#39;)</code></pre>
<p><img src="figure/4.1.1_qtl.analysis_binary_ici.vs.pbs_5.batches_mis_conditional_2-peaks.Rmd/scanone2-2.png" width="912" style="display: block; margin: auto;" /></p>
<pre class="r"><code>ymx &lt;- 14
plot(scanone.hk, bandcol = &quot;grey90&quot;,lty=1, cex=1, col = &quot;slateblue&quot;, ylim=c(0, ymx+0.5))
title(main = paste0(colnames(out), &quot; [positions in cM]\n(using same scale as ici vs. eoi for easier comparison)&quot;))
qtl::add.threshold(scanone.hk,  perms= operm.hk, alpha=0.01, col = &#39;blue&#39;)
qtl::add.threshold(scanone.hk,  perms= operm.hk, alpha=0.05, col = &#39;red&#39;)
qtl::add.threshold(scanone.hk,  perms= operm.hk, alpha=0.1, col = &#39;purple&#39;)</code></pre>
<p><img src="figure/4.1.1_qtl.analysis_binary_ici.vs.pbs_5.batches_mis_conditional_2-peaks.Rmd/scanone2-3.png" width="912" style="display: block; margin: auto;" /></p>
<pre class="r"><code>print(as.data.frame(summary(scanone.hk, perms=operm.hk, pvalues=TRUE, format=&quot;allpeaks&quot;)))</code></pre>
<pre><code>            chr       pos      lod      pval
UNC1197721    1 47.673577 1.868502 1.0000000
UNCHS005266   2 40.908610 1.873368 1.0000000
UNC6045859    3 50.620447 1.870152 1.0000000
JAX00123711   4 53.542073 1.799165 1.0000000
JAX00574407   5  6.213413 3.388862 0.2097615
UNCHS017906   6 37.001766 1.868162 1.0000000
JAX00233659   7 87.998157 1.870072 1.0000000
JAX00667121   8 24.443849 2.382541 0.9118988
UNC17203329   9 68.519175 1.323467 1.0000000
JAX00302150  10 74.547218 2.532706 0.7192282
UNC20281687  11 67.380801 1.859331 1.0000000
UNCHS034188  12 38.836978 1.870151 1.0000000
JAX00350973  13  5.743505 1.378528 1.0000000
JAX00380911  14 30.154517 2.381660 0.9118988
UNC25275535  15 11.968141 1.402063 1.0000000
UNC26790882  16 30.832641 2.019242 1.0000000
UNCHS045288  17 60.699043 2.034920 1.0000000
JAX00450853  18  2.913076 2.621475 0.7192282
UNCHS047190  19  8.712408 1.804318 1.0000000
XiB2          X 45.699951 1.870087 0.9992566</code></pre>
<pre class="r"><code>print(&quot;all peaks with a p-value less or equal to 0.05 (suggestive)&quot;)</code></pre>
<pre><code>[1] &quot;all peaks with a p-value less or equal to 0.05 (suggestive)&quot;</code></pre>
<pre class="r"><code>print(as.data.frame(summary(scanone.hk, perms=operm.hk, alpha=0.05, pvalues=TRUE, format=&quot;allpeaks&quot;)))</code></pre>
<pre><code>[1] chr pos lod
&lt;0 rows&gt; (or 0-length row.names)</code></pre>
<pre class="r"><code>#print(&quot;method == ehk&quot;)

#scanone.ehk &lt;-qtl::scanone(cross.probs, pheno.col=&quot;ICI.vs.PBS&quot; , model=&quot;binary&quot;, method=&quot;ehk&quot;)
#operm.ehk &lt;- qtl::scanone(cross.probs, method = &quot;ehk&quot;, pheno.col=&quot;ICI.vs.PBS&quot;, n.perm = 10, perm.Xsp = TRUE, model=&quot;binary&quot;, verbose=FALSE)
#plot(operm.ehk)
#print(summary(operm.ehk, alpha=c(0.01,  0.05, 0.1)))

#plot(scanone.ehk, bandcol = &quot;grey90&quot;,lty=1, cex=1, col = &quot;steelblue&quot;)  
#qtl::add.threshold(scanone.ehk,  perms= operm.ehk, alpha=0.01, col = &#39;blue&#39;)
#qtl::add.threshold(scanone.ehk,  perms= operm.ehk, alpha=0.05, col = &#39;red&#39;)
#qtl::add.threshold(scanone.ehk,  perms= operm.ehk, alpha=0.1, col = &#39;purple&#39;)

#print(as.data.frame(summary(scanone.ehk)))
#print(as.data.frame(summary(scanone.ehk, perms=operm.ehk, alpha=0.05, pvalues=TRUE, format=&quot;allpeaks&quot;)))</code></pre>
</div>
</div>
</div>
<div
id="conditionaing-on-eoi-vs.-ici-chr-4-peak-unchs008487-chr-10-peak-unc18240977"
class="section level1">
<h1>Conditionaing on eoi vs. ici chr 4 peak (UNCHS008487) &amp; chr 10
peak (UNC18240977)</h1>
<div id="genome-wide-scan-3" class="section level2">
<h2>Genome-wide scan</h2>
<pre class="r"><code>Xcovar &lt;- get_x_covar(gm)
addcovar = model.matrix(~UNC8250659+UNC18240977, data = covars)[,-1]

#K &lt;- calc_kinship(pr.qc, type = &quot;loco&quot;)
#heatmap(K[[1]])
#K.overall &lt;- calc_kinship(pr.qc, type = &quot;overall&quot;)
#heatmap(K.overall)
kinship &lt;- calc_kinship(pr.qc)
heatmap(kinship)</code></pre>
<p><img src="figure/4.1.1_qtl.analysis_binary_ici.vs.pbs_5.batches_mis_conditional_2-peaks.Rmd/permutation3-1.png" width="912" style="display: block; margin: auto;" /></p>
<pre class="r"><code>#operm &lt;- scan1perm(pr.qc, gm$covar$phenos, Xcovar=Xcovar, n_perm=2000)
#operm &lt;- scan1perm(pr.qc, gm$covar$phenos, addcovar = addcovar, n_perm=2000)
#operm &lt;- scan1perm(pr.qc, gm$covar$phenos, n_perm=2000)
operm &lt;- scan1perm(pr.qc, gm$covar[&quot;ICI.vs.PBS&quot;], model=&quot;binary&quot;, n_perm=10, perm_Xsp=TRUE, chr_lengths=chr_lengths(gm$gmap), addcovar = addcovar)

summary_table&lt;-data.frame(unclass(summary(operm, alpha=c(0.01,  0.05, 0.1))))
names(summary_table) &lt;- c(&quot;autosomes&quot;,&quot;X&quot;)
summary_table$significance.level &lt;- rownames(summary_table)

rownames(summary_table) &lt;- NULL

summary_table[c(3,1:2)] %&gt;%
  kable(escape = F,align = c(&quot;ccc&quot;)) %&gt;%
  kable_styling(&quot;striped&quot;, full_width = T) %&gt;%
  column_spec(1, bold=TRUE)</code></pre>
<table class="table table-striped" style="margin-left: auto; margin-right: auto;">
<thead>
<tr>
<th style="text-align:center;">
significance.level
</th>
<th style="text-align:center;">
autosomes
</th>
<th style="text-align:center;">
X
</th>
</tr>
</thead>
<tbody>
<tr>
<td style="text-align:center;font-weight: bold;">
0.01
</td>
<td style="text-align:center;">
3.086790
</td>
<td style="text-align:center;">
4.197360
</td>
</tr>
<tr>
<td style="text-align:center;font-weight: bold;">
0.05
</td>
<td style="text-align:center;">
3.058033
</td>
<td style="text-align:center;">
3.823126
</td>
</tr>
<tr>
<td style="text-align:center;font-weight: bold;">
0.1
</td>
<td style="text-align:center;">
3.021996
</td>
<td style="text-align:center;">
3.333744
</td>
</tr>
</tbody>
</table>
<p>The figures below show QTL maps for each phenotype</p>
<pre class="r"><code>out &lt;- scan1(pr.qc, gm$covar[&quot;ICI.vs.PBS&quot;], Xcovar=Xcovar, model=&quot;binary&quot;, addcovar = addcovar)

summary_table&lt;-data.frame(unclass(summary(operm, alpha=c(0.01,  0.05, 0.1))))

plot_lod&lt;-function(out,map){
  for (i in 1:dim(out)[2]){
    #png(filename=paste0(&quot;/Users/chenm/Documents/qtl/Jai/&quot;,colnames(out)[i],  &quot;_lod.png&quot;))
    
    ymx &lt;- maxlod(out) # overall maximum LOD score
    plot(out, map, lodcolumn=i, col=&quot;slateblue&quot;, ylim=c(0, ymx+0.5))
    ##legend(&quot;topright&quot;, lwd=2, colnames(out)[i], bg=&quot;gray90&quot;)
    title(main = paste0(colnames(out)[i], &quot; [positions in cM]&quot;))
    add_threshold(map,  summary(operm,alpha=0.1), col = &#39;purple&#39;)
    add_threshold(map,  summary(operm, alpha=0.05), col = &#39;red&#39;)
    add_threshold(map,  summary(operm, alpha=0.01), col = &#39;blue&#39;)

    #par(mar=c(5.1, 6.1, 1.1, 1.1))
    ymx &lt;- 14 # overall maximum LOD score
    plot(out, map, lodcolumn=i, col=&quot;slateblue&quot;, ylim=c(0, ymx+0.5))
    ##legend(&quot;topright&quot;, lwd=2, colnames(out)[i], bg=&quot;gray90&quot;)
    title(main = paste0(colnames(out)[i], &quot; [positions in cM] \n(using same scale as eoi  vs. ici for easier comparison)&quot;))
    add_threshold(map,  summary(operm, alpha=0.1), col = &#39;purple&#39;)
    add_threshold(map,  summary(operm, alpha=0.05), col = &#39;red&#39;)
    add_threshold(map,  summary(operm, alpha=0.01), col = &#39;blue&#39;)
    #for (j in 1: dim(summary_table)[1]){
    #  abline(h=summary_table[j, i],col=&quot;red&quot;)
    #  text(x=400, y =summary_table[j, i]+0.12, labels = paste(&quot;p=&quot;, row.names(summary_table)[j]))
    #}
    #dev.off()
  }
}

plot_lod(out,gm$gmap)</code></pre>
<p><img src="figure/4.1.1_qtl.analysis_binary_ici.vs.pbs_5.batches_mis_conditional_2-peaks.Rmd/Genome-wide%20scan3-1.png" width="912" style="display: block; margin: auto;" /><img src="figure/4.1.1_qtl.analysis_binary_ici.vs.pbs_5.batches_mis_conditional_2-peaks.Rmd/Genome-wide%20scan3-2.png" width="912" style="display: block; margin: auto;" /></p>
</div>
<div id="lod-peaks-2" class="section level2">
<h2>LOD peaks</h2>
<p>The table below shows QTL peaks associated with the phenotype. We use
the 95% threshold from the permutations to find peaks.</p>
<div id="centimorgan-cm-2" class="section level3">
<h3>Centimorgan (cM)</h3>
<pre class="r"><code>peaks &lt;- find_peaks(out, gm$gmap, threshold=summary(operm,alpha=0.05)$A, thresholdX = summary(operm,alpha=0.05)$X, peakdrop=3, drop=1.5)

if(nrow(peaks) &gt;0){
peaks$marker &lt;- find_marker(gm$gmap, chr=peaks$chr,pos=peaks$pos)
names(peaks)[2] &lt;- c(&quot;phenotype&quot;)
peaks &lt;- peaks[-1]

rownames(peaks) &lt;- NULL
print(kable(peaks, escape = F, align = c(&quot;cccccccc&quot;), &quot;html&quot;) 
  %&gt;% kable_styling(&quot;striped&quot;, full_width = T)%&gt;%
  column_spec(1, bold=TRUE)
  )

#plot only peak chromosomes

plot_lod_chr&lt;-function(out,map,chrom){
  for (i in 1:dim(out)[2]){
    #png(filename=paste0(&quot;/Users/chenm/Documents/qtl/Jai/&quot;,colnames(out)[i],  &quot;_lod.png&quot;))
    
    #par(mar=c(5.1, 6.1, 1.1, 1.1))
    ymx &lt;- maxlod(out) # overall maximum LOD score
    plot(out, map, chr = chrom, lodcolumn=i, col=&quot;slateblue&quot;, ylim=c(0, ymx+0.5))
    #legend(&quot;topright&quot;, lwd=2, colnames(out)[i], bg=&quot;gray90&quot;)
    title(main = paste0(colnames(out)[i], &quot; - chr&quot;, chrom, &quot; [positions in cM]&quot;))
    add_threshold(map,  summary(operm,alpha=0.1), col = &#39;purple&#39;)
    add_threshold(map,  summary(operm, alpha=0.05), col = &#39;red&#39;)
    add_threshold(map,  summary(operm, alpha=0.01), col = &#39;blue&#39;)
    #for (j in 1: dim(summary_table)[1]){
    #  abline(h=summary_table[j, i],col=&quot;red&quot;)
    #  text(x=400, y =summary_table[j, i]+0.12, labels = paste(&quot;p=&quot;, row.names(summary_table)[j]))
    #}
    #dev.off()

    
    ymx &lt;- 14
    plot(out, map, chr = chrom, lodcolumn=i, col=&quot;slateblue&quot;, ylim=c(0, ymx+0.5))
    #legend(&quot;topright&quot;, lwd=2, colnames(out)[i], bg=&quot;gray90&quot;)
    title(main = paste0(colnames(out)[i], &quot; - chr&quot;, chrom, &quot; [positions in cM]\n(using same scale as eoi vs. ici for easier comparison)&quot;))
    add_threshold(map,  summary(operm,alpha=0.1), col = &#39;purple&#39;)
    add_threshold(map,  summary(operm, alpha=0.05), col = &#39;red&#39;)
    add_threshold(map,  summary(operm, alpha=0.01), col = &#39;blue&#39;)

  }
}


for(i in unique(peaks$chr)){
#for (i in 1:nrow(peaks)){
  #plot_lod_chr(out,gm$gmap, peaks$chr[i])
  plot_lod_chr(out,gm$gmap, i)
}

} else {
  print(paste0(&quot;There are no peaks that have a LOD that reaches suggestive (p&lt;0.05) level of &quot;,summary(operm,alpha=0.05)$A, &quot; [autosomes]/&quot;,summary(operm,alpha=0.05)$X, &quot; [x-chromosome]&quot;))
}</code></pre>
<table class="table table-striped" style="margin-left: auto; margin-right: auto;">
<thead>
<tr>
<th style="text-align:center;">
phenotype
</th>
<th style="text-align:center;">
chr
</th>
<th style="text-align:center;">
pos
</th>
<th style="text-align:center;">
lod
</th>
<th style="text-align:center;">
ci_lo
</th>
<th style="text-align:center;">
ci_hi
</th>
<th style="text-align:center;">
marker
</th>
</tr>
</thead>
<tbody>
<tr>
<td style="text-align:center;font-weight: bold;">
ICI.vs.PBS
</td>
<td style="text-align:center;">
4
</td>
<td style="text-align:center;">
49.777
</td>
<td style="text-align:center;">
3.070548
</td>
<td style="text-align:center;">
49.776
</td>
<td style="text-align:center;">
77.787
</td>
<td style="text-align:center;">
UNCHS012304
</td>
</tr>
</tbody>
</table>
<p><img src="figure/4.1.1_qtl.analysis_binary_ici.vs.pbs_5.batches_mis_conditional_2-peaks.Rmd/LOD%20peaks3-1.png" width="912" style="display: block; margin: auto;" /><img src="figure/4.1.1_qtl.analysis_binary_ici.vs.pbs_5.batches_mis_conditional_2-peaks.Rmd/LOD%20peaks3-2.png" width="912" style="display: block; margin: auto;" /></p>
</div>
<div id="megabase-mb-2" class="section level3">
<h3>Megabase (MB)</h3>
<pre class="r"><code>print(&quot;peaks in MB positions&quot;)</code></pre>
<p>[1] “peaks in MB positions”</p>
<pre class="r"><code>peaks_mba &lt;- find_peaks(out, gm$pmap, threshold=summary(operm,alpha=0.05)$A, thresholdX = summary(operm,alpha=0.05)$X, peakdrop=3, drop=1.5)

if(nrow(peaks) &gt;0){
peaks_mba$marker &lt;- find_marker(gm$pmap, chr=peaks_mba$chr,pos=peaks_mba$pos)
names(peaks_mba)[2] &lt;- c(&quot;phenotype&quot;)
peaks_mba &lt;- peaks_mba[-1]

#peaks_mbl &lt;- list()
##corresponding info in Mb
#for(i in 1:nrow(peaks)){
#  #lodindex &lt;- peaks$lodindex[i]
#  phenotype &lt;- peaks$phenotype[i]
#  chr &lt;- as.character(peaks$chr[i])
#  lod &lt;- peaks$lod[i]
#  mark &lt;- peaks$marker[i]
#  pos &lt;- mapdf[mapdf$marker==mark,]$pmapdf
#  ci_lo &lt;- mapdfnd$pmapdf[which(mapdfnd$gmapdf == peaks$ci_lo[i] &amp; mapdfnd$chr == peaks$chr[i])]
#  ci_hi &lt;- mapdfnd$pmapdf[which(mapdfnd$gmapdf == peaks$ci_hi[i] &amp; mapdfnd$chr == peaks$chr[i])]
#  peaks_mb=as.data.frame(cbind(phenotype, chr, pos, lod, ci_lo, ci_hi, mark))
#  names(peaks_mb)[7] &lt;- c(&quot;marker&quot;)
#  peaks_mbl[[i]] &lt;- peaks_mb
#}
#peaks_mba2 &lt;- do.call(rbind, peaks_mbl)
#peaks_mba2 &lt;- as.data.frame(peaks_mba)
#peaks_mba[,c(&quot;chr&quot;, &quot;pos&quot;, &quot;lod&quot;, &quot;ci_lo&quot;, &quot;ci_hi&quot;)] &lt;- sapply(peaks_mba[,c(&quot;chr&quot;, &quot;pos&quot;, &quot;lod&quot;, &quot;ci_lo&quot;, &quot;ci_hi&quot;)], as.numeric)

rownames(peaks_mba) &lt;- NULL
print(kable(peaks_mba, escape = F, align = c(&quot;cccccccc&quot;), &quot;html&quot;) 
  %&gt;% kable_styling(&quot;striped&quot;, full_width = T)%&gt;%
  column_spec(1, bold=TRUE)
  )

plot_lod_chr_mb&lt;-function(out,map,chrom){
  for (i in 1:dim(out)[2]){
    #png(filename=paste0(&quot;/Users/chenm/Documents/qtl/Jai/&quot;,colnames(out)[i],  &quot;_lod.png&quot;))
    
    #par(mar=c(5.1, 6.1, 1.1, 1.1))
    ymx &lt;- maxlod(out) # overall maximum LOD score
    plot(out, map, chr = chrom, lodcolumn=i, col=&quot;slateblue&quot;, ylim=c(0, ymx+0.5))
    #legend(&quot;topright&quot;, lwd=2, colnames(out)[i], bg=&quot;gray90&quot;)
    title(main = paste0(colnames(out)[i], &quot; - chr&quot;, chrom, &quot; [positions in MB]&quot;))
    add_threshold(map,  summary(operm,alpha=0.1), col = &#39;purple&#39;)
    add_threshold(map,  summary(operm, alpha=0.05), col = &#39;red&#39;)
    add_threshold(map,  summary(operm, alpha=0.01), col = &#39;blue&#39;)
    #for (j in 1: dim(summary_table)[1]){
    #  abline(h=summary_table[j, i],col=&quot;red&quot;)
    #  text(x=400, y =summary_table[j, i]+0.12, labels = paste(&quot;p=&quot;, row.names(summary_table)[j]))
    #}
    #dev.off()

    ymx &lt;- 14
    plot(out, map, chr = chrom, lodcolumn=i, col=&quot;slateblue&quot;, ylim=c(0, ymx+0.5))
    #legend(&quot;topright&quot;, lwd=2, colnames(out)[i], bg=&quot;gray90&quot;)
    title(main = paste0(colnames(out)[i], &quot; - chr&quot;, chrom, &quot; [positions in MB]\n(using same scale as eoi vs. ici for easier comparison)&quot;))
    add_threshold(map,  summary(operm,alpha=0.1), col = &#39;purple&#39;)
    add_threshold(map,  summary(operm, alpha=0.05), col = &#39;red&#39;)
    add_threshold(map,  summary(operm, alpha=0.01), col = &#39;blue&#39;)


  }
}

for(i in unique(peaks_mba$chr)){
#for (i in 1:nrow(peaks_mba)){
  #plot_lod_chr_mb(out,gm$pmap, peaks_mba$chr[i])
  plot_lod_chr_mb(out,gm$pmap,i)
}

} else {
  print(paste0(&quot;There are no peaks that have a LOD that reaches suggestive (p&lt;0.05) level of &quot;,summary(operm,alpha=0.05)$A, &quot; [autosomes]/&quot;,summary(operm,alpha=0.05)$X, &quot; [x-chromosome]&quot;))
}</code></pre>
<table class="table table-striped" style="margin-left: auto; margin-right: auto;">
<thead>
<tr>
<th style="text-align:center;">
phenotype
</th>
<th style="text-align:center;">
chr
</th>
<th style="text-align:center;">
pos
</th>
<th style="text-align:center;">
lod
</th>
<th style="text-align:center;">
ci_lo
</th>
<th style="text-align:center;">
ci_hi
</th>
<th style="text-align:center;">
marker
</th>
</tr>
</thead>
<tbody>
<tr>
<td style="text-align:center;font-weight: bold;">
ICI.vs.PBS
</td>
<td style="text-align:center;">
4
</td>
<td style="text-align:center;">
106.7137
</td>
<td style="text-align:center;">
3.070548
</td>
<td style="text-align:center;">
106.7113
</td>
<td style="text-align:center;">
143.751
</td>
<td style="text-align:center;">
UNCHS012304
</td>
</tr>
</tbody>
</table>
<p><img src="figure/4.1.1_qtl.analysis_binary_ici.vs.pbs_5.batches_mis_conditional_2-peaks.Rmd/LOD%20peaks_mba3-1.png" width="912" style="display: block; margin: auto;" /><img src="figure/4.1.1_qtl.analysis_binary_ici.vs.pbs_5.batches_mis_conditional_2-peaks.Rmd/LOD%20peaks_mba3-2.png" width="912" style="display: block; margin: auto;" /></p>
</div>
</div>
<div id="qtl-effects-2" class="section level2">
<h2>QTL effects</h2>
<p>For each peak LOD location we give a list of gene</p>
<pre class="r"><code>query_variants &lt;- create_variant_query_func(&quot;/Users/corneb/Documents/MyJax/CS/Projects/support.files/qtl2/cc_variants.sqlite&quot;)
query_genes &lt;- create_gene_query_func(&quot;/Users/corneb/Documents/MyJax/CS/Projects/support.files/qtl2/mouse_genes_mgi.sqlite&quot;)

if(nrow(peaks) &gt;0){
for (i in 1:nrow(peaks)){
#for (i in 1:1){
  #Plot 1
  g &lt;- maxmarg(pr.qc, gm$gmap, chr=peaks$chr[i], pos=peaks$pos[i], return_char=TRUE)
  #png(filename=paste0(&quot;/Users/chenm/Documents/qtl/Jai/&quot;,&quot;qtl_effect_&quot;, i, &quot;.png&quot;))
  #par(mar=c(4.1, 4.1, 1.5, 0.6))
  plot_pxg(g, gm$covar[,peaks$phenotype[i]], ylab=peaks$phenotype[i], sort=FALSE)
  title(main = paste0(&quot;chr: &quot;, chr=peaks$chr[i],&quot;; pos: &quot;, peaks$pos[i], &quot;cM /&quot;,peaks_mba$pos[i],&quot;MB\n(&quot;,peaks$phenotype[i],&quot; )&quot;), line=0.2)
  ##dev.off()

  chr = peaks$chr[i]

# Plot 2
  pr_sub &lt;- pull_genoprobint(pr.qc, gm$gmap, chr, c(peaks$ci_lo[i], peaks$ci_hi[i]))
  #coeff &lt;- scan1coef(pr[,chr], cross$pheno[,peaks$lodcolumn[i]], addcovar = addcovar)
  #coeff &lt;- scan1coef(pr[,chr], cross$pheno[,peaks$lodcolumn[i]], Xcovar=Xcovar)
  #coeff &lt;- scan1coef(pr.qc[,chr], gm$covar[peaks$lodcolumn[i]], model=&quot;binary&quot;)
  #coeff_sub &lt;- scan1coef(pr_sub[,chr], gm$covar[peaks$lodcolumn[i]], model=&quot;binary&quot;)
  blup &lt;- scan1blup(pr.qc[,chr], gm$covar[peaks$phenotype[i]], addcovar = addcovar)
  blup_sub &lt;- scan1blup(pr_sub[,chr], gm$covar[peaks$phenotype[i]], addcovar = addcovar)

  write.csv(as.data.frame(blup_sub), paste0(&quot;data/ici.vs.pbs_blup_sub_chr-&quot;,chr,&quot;_peak.marker-&quot;,peaks$marker[i],&quot;_lod.drop-1.5_5.batches_mis_conditional_2-peaks-chr4-10.csv&quot;), quote=F)

  #plot_coef(coeff, 
  #     gm$gmap, columns=1:2,
  #     bgcolor=&quot;gray95&quot;, legend=&quot;bottomleft&quot;, 
  #     main = paste(&quot;chr&quot;, chr=peaks$chr[i],&quot;; pos: &quot;, peaks$pos[i], &quot;cM /&quot;,peaks_mba$pos[i],&quot;MB\n(&quot;,peaks$lodcolumn[i],&quot; [scan1coeff; positions in cM])&quot;)
  #     )

  #plot_coef(coeff_sub, 
  #     gm$gmap, columns=1:2,
  #     bgcolor=&quot;gray95&quot;, legend=&quot;bottomleft&quot;, 
  #     main = paste(&quot;chr&quot;, chr=peaks$chr[i],&quot;; pos: &quot;, peaks$pos[i], &quot;cM /&quot;,peaks_mba$pos[i],&quot;MB\n(&quot;,peaks$lodcolumn[i],&quot;; 1.5 LOD drop interval [scan1coeff; positions in cM] ) &quot;)
  #     )


  plot_coef(blup, 
       gm$gmap, columns=1:2,
       bgcolor=&quot;gray95&quot;, legend=&quot;bottomleft&quot;, 
       main = paste0(&quot;chr: &quot;, chr=peaks$chr[i], &quot;; pos: &quot;, peaks$pos[i], &quot;cM / &quot;,peaks_mba$pos[i],&quot;MB\n(&quot;,peaks$phenotype[i],&quot; [scan1blup; positions in cM])&quot;)
       )

  plot_coef(blup_sub, 
       gm$gmap, columns=1:2,
       bgcolor=&quot;gray95&quot;, legend=&quot;bottomleft&quot;, 
       main = paste0(&quot;chr: &quot;, chr=peaks$chr[i],&quot;; pos: &quot;, peaks$pos[i], &quot;cM /&quot;,peaks_mba$pos[i],&quot;MB\n(&quot;,peaks$phenotype[i],&quot;; 1.5 LOD drop interval [scan1blup; positions in cM])&quot;)
       )


 # Plot 3
  #c2effB &lt;- scan1coef(pr.qc[,chr], gm$covar[peaks$lodcolumn[i]], model=&quot;binary&quot;, contrasts=cbind(a=c(-1, 0), d=c(0, -1)))
  #c2effBb &lt;- scan1blup(pr.qc[,chr], gm$covar[peaks$lodcolumn[i]], contrasts=cbind(a=c(-1, 0), d=c(0, -1)))
  ##c2effB &lt;- scan1coef(pr[,chr], cross$pheno[,peaks$lodcolumn[i]], addcovar = addcovar, contrasts=cbind(mu=c(1,1,1), a=c(-1, 0, 1), d=c(0, 1, 0)))
  ##c2effB &lt;- scan1coef(pr[,chr], cross$pheno[,peaks$lodcolumn[i]],Xcovar=Xcovar, contrasts=cbind(mu=c(1,1,1), a=c(-1, 0, 1), d=c(0, 1, 0)))
  #plot(c2effB, gm$gmap[chr], columns=1:2,
  #     bgcolor=&quot;gray95&quot;, legend=&quot;bottomleft&quot;, 
  #     main = paste(&quot;chr&quot;, chr=peaks$chr[i], &quot;pos&quot;, peaks$pos[i], &quot;(&quot;,peaks$lodcolumn[i],&quot;)&quot;)
  #     )
  #plot(c2effBb, gm$gmap[chr], columns=1:2,
  #     bgcolor=&quot;gray95&quot;, legend=&quot;bottomleft&quot;, 
  #     main = paste(&quot;chr&quot;, chr=peaks$chr[i], &quot;pos&quot;, peaks$pos[i], &quot;(&quot;,peaks$lodcolumn[i],&quot;)&quot;)
  #     )
  ##last_coef &lt;- unclass(c2effB)[nrow(c2effB),2:3] # last two coefficients
  ##for(t in seq(along=last_coef))
  ##  axis(side=4, at=last_coef[t], names(last_coef)[t], tick=FALSE)


  #Table 1
  chr = peaks_mba$chr[i]
  start=as.numeric(peaks_mba$ci_lo[i])
  end=as.numeric(peaks_mba$ci_hi[i])

  genesgss = query_genes(chr, start, end)

  write.csv(genesgss, file=paste0(&quot;data/ici.vs.pbs_genes_chr-&quot;,chr,&quot;_peak.marker-&quot;,peaks$marker[i],&quot;_lod.drop-1.5_5.batches_mis_conditional_2-peaks-chr4-10.csv&quot;), quote=F)

  rownames(genesgss) &lt;- NULL
  genesgss$strand_old = genesgss$strand
  genesgss$strand[genesgss$strand==&quot;+&quot;] &lt;- &quot;positive&quot;
  genesgss$strand[genesgss$strand==&quot;-&quot;] &lt;- &quot;negative&quot;

  #genesgss &lt;- 
  #table &lt;- 
  #genesgss[,c(&quot;chr&quot;,&quot;type&quot;,&quot;start&quot;,&quot;stop&quot;,&quot;strand&quot;,&quot;ID&quot;,&quot;Name&quot;,&quot;Dbxref&quot;,&quot;gene_id&quot;,&quot;mgi_type&quot;,&quot;description&quot;)] %&gt;%
  #kable(escape = F,align = c(&quot;ccccccccccc&quot;)) %&gt;%
  #kable_styling(&quot;striped&quot;, full_width = T) #%&gt;% 
  #cat #%&gt;%
  #column_spec(1, bold=TRUE)
#
  #print(kable(genesgss[,c(&quot;chr&quot;,&quot;type&quot;,&quot;start&quot;,&quot;stop&quot;,&quot;strand&quot;,&quot;ID&quot;,&quot;Name&quot;,&quot;Dbxref&quot;,&quot;gene_id&quot;,&quot;mgi_type&quot;,&quot;description&quot;)], escape = F,align = c(&quot;ccccccccccc&quot;)))

  print(kable(genesgss[,c(&quot;chr&quot;,&quot;type&quot;,&quot;start&quot;,&quot;stop&quot;,&quot;strand&quot;,&quot;ID&quot;,&quot;Name&quot;,&quot;Dbxref&quot;,&quot;gene_id&quot;,&quot;mgi_type&quot;,&quot;description&quot;)], &quot;html&quot;) %&gt;% kable_styling(&quot;striped&quot;, full_width = T))

  #table
  

}

} else {
  print(paste0(&quot;There are no peaks that have a LOD that reaches suggestive (p&lt;0.05) level of &quot;,summary(operm,alpha=0.05)$A, &quot; [autosomes]/&quot;,summary(operm,alpha=0.05)$X, &quot; [x-chromosome]&quot;))
}</code></pre>
<img src="figure/4.1.1_qtl.analysis_binary_ici.vs.pbs_5.batches_mis_conditional_2-peaks.Rmd/QTL%20effects3-1.png" width="912" style="display: block; margin: auto;" /><img src="figure/4.1.1_qtl.analysis_binary_ici.vs.pbs_5.batches_mis_conditional_2-peaks.Rmd/QTL%20effects3-2.png" width="912" style="display: block; margin: auto;" /><img src="figure/4.1.1_qtl.analysis_binary_ici.vs.pbs_5.batches_mis_conditional_2-peaks.Rmd/QTL%20effects3-3.png" width="912" style="display: block; margin: auto;" />
<table class="table table-striped" style="margin-left: auto; margin-right: auto;">
<thead>
<tr>
<th style="text-align:left;">
chr
</th>
<th style="text-align:left;">
type
</th>
<th style="text-align:right;">
start
</th>
<th style="text-align:right;">
stop
</th>
<th style="text-align:left;">
strand
</th>
<th style="text-align:left;">
ID
</th>
<th style="text-align:left;">
Name
</th>
<th style="text-align:left;">
Dbxref
</th>
<th style="text-align:left;">
gene_id
</th>
<th style="text-align:left;">
mgi_type
</th>
<th style="text-align:left;">
description
</th>
</tr>
</thead>
<tbody>
<tr>
<td style="text-align:left;">
4
</td>
<td style="text-align:left;">
gene
</td>
<td style="text-align:right;">
106.6804
</td>
<td style="text-align:right;">
106.7366
</td>
<td style="text-align:left;">
negative
</td>
<td style="text-align:left;">
MGI_C57BL6J_2685873
</td>
<td style="text-align:left;">
Mroh7
</td>
<td style="text-align:left;">
NCBI_Gene:381538,ENSEMBL:ENSMUSG00000047502
</td>
<td style="text-align:left;">
MGI:2685873
</td>
<td style="text-align:left;">
protein coding gene
</td>
<td style="text-align:left;">
maestro heat-like repeat family member 7
</td>
</tr>
<tr>
<td style="text-align:left;">
4
</td>
<td style="text-align:left;">
gene
</td>
<td style="text-align:right;">
106.7339
</td>
<td style="text-align:right;">
106.7483
</td>
<td style="text-align:left;">
positive
</td>
<td style="text-align:left;">
MGI_C57BL6J_2657115
</td>
<td style="text-align:left;">
Fam151a
</td>
<td style="text-align:left;">
NCBI_Gene:230579,ENSEMBL:ENSMUSG00000034871
</td>
<td style="text-align:left;">
MGI:2657115
</td>
<td style="text-align:left;">
protein coding gene
</td>
<td style="text-align:left;">
family with sequence simliarity 151, member A
</td>
</tr>
<tr>
<td style="text-align:left;">
4
</td>
<td style="text-align:left;">
gene
</td>
<td style="text-align:right;">
106.7446
</td>
<td style="text-align:right;">
106.8061
</td>
<td style="text-align:left;">
negative
</td>
<td style="text-align:left;">
MGI_C57BL6J_1913736
</td>
<td style="text-align:left;">
Acot11
</td>
<td style="text-align:left;">
NCBI_Gene:329910,ENSEMBL:ENSMUSG00000034853
</td>
<td style="text-align:left;">
MGI:1913736
</td>
<td style="text-align:left;">
protein coding gene
</td>
<td style="text-align:left;">
acyl-CoA thioesterase 11
</td>
</tr>
<tr>
<td style="text-align:left;">
4
</td>
<td style="text-align:left;">
pseudogene
</td>
<td style="text-align:right;">
106.7748
</td>
<td style="text-align:right;">
106.7750
</td>
<td style="text-align:left;">
positive
</td>
<td style="text-align:left;">
MGI_C57BL6J_3651452
</td>
<td style="text-align:left;">
Gm12745
</td>
<td style="text-align:left;">
ENSEMBL:ENSMUSG00000081110
</td>
<td style="text-align:left;">
MGI:3651452
</td>
<td style="text-align:left;">
pseudogene
</td>
<td style="text-align:left;">
predicted gene 12745
</td>
</tr>
<tr>
<td style="text-align:left;">
4
</td>
<td style="text-align:left;">
gene
</td>
<td style="text-align:right;">
106.8149
</td>
<td style="text-align:right;">
106.8283
</td>
<td style="text-align:left;">
negative
</td>
<td style="text-align:left;">
MGI_C57BL6J_5590224
</td>
<td style="text-align:left;">
Gm31065
</td>
<td style="text-align:left;">
NCBI_Gene:102633170
</td>
<td style="text-align:left;">
MGI:5590224
</td>
<td style="text-align:left;">
lncRNA gene
</td>
<td style="text-align:left;">
predicted gene, 31065
</td>
</tr>
<tr>
<td style="text-align:left;">
4
</td>
<td style="text-align:left;">
gene
</td>
<td style="text-align:right;">
106.8393
</td>
<td style="text-align:right;">
106.8402
</td>
<td style="text-align:left;">
negative
</td>
<td style="text-align:left;">
MGI_C57BL6J_5590284
</td>
<td style="text-align:left;">
Gm31125
</td>
<td style="text-align:left;">
NCBI_Gene:102633257
</td>
<td style="text-align:left;">
MGI:5590284
</td>
<td style="text-align:left;">
lncRNA gene
</td>
<td style="text-align:left;">
predicted gene, 31125
</td>
</tr>
<tr>
<td style="text-align:left;">
4
</td>
<td style="text-align:left;">
gene
</td>
<td style="text-align:right;">
106.8418
</td>
<td style="text-align:right;">
106.8479
</td>
<td style="text-align:left;">
negative
</td>
<td style="text-align:left;">
MGI_C57BL6J_3652135
</td>
<td style="text-align:left;">
Gm12746
</td>
<td style="text-align:left;">
ENSEMBL:ENSMUSG00000086940
</td>
<td style="text-align:left;">
MGI:3652135
</td>
<td style="text-align:left;">
lncRNA gene
</td>
<td style="text-align:left;">
predicted gene 12746
</td>
</tr>
<tr>
<td style="text-align:left;">
4
</td>
<td style="text-align:left;">
gene
</td>
<td style="text-align:right;">
106.9002
</td>
<td style="text-align:right;">
106.9129
</td>
<td style="text-align:left;">
negative
</td>
<td style="text-align:left;">
MGI_C57BL6J_5826502
</td>
<td style="text-align:left;">
Gm46865
</td>
<td style="text-align:left;">
NCBI_Gene:108168970
</td>
<td style="text-align:left;">
MGI:5826502
</td>
<td style="text-align:left;">
lncRNA gene
</td>
<td style="text-align:left;">
predicted gene, 46865
</td>
</tr>
<tr>
<td style="text-align:left;">
4
</td>
<td style="text-align:left;">
gene
</td>
<td style="text-align:right;">
106.9107
</td>
<td style="text-align:right;">
107.0497
</td>
<td style="text-align:left;">
positive
</td>
<td style="text-align:left;">
MGI_C57BL6J_1919725
</td>
<td style="text-align:left;">
Ssbp3
</td>
<td style="text-align:left;">
NCBI_Gene:72475,ENSEMBL:ENSMUSG00000061887
</td>
<td style="text-align:left;">
MGI:1919725
</td>
<td style="text-align:left;">
protein coding gene
</td>
<td style="text-align:left;">
single-stranded DNA binding protein 3
</td>
</tr>
<tr>
<td style="text-align:left;">
4
</td>
<td style="text-align:left;">
gene
</td>
<td style="text-align:right;">
107.0179
</td>
<td style="text-align:right;">
107.0225
</td>
<td style="text-align:left;">
negative
</td>
<td style="text-align:left;">
MGI_C57BL6J_3650114
</td>
<td style="text-align:left;">
Gm12786
</td>
<td style="text-align:left;">
ENSEMBL:ENSMUSG00000085581
</td>
<td style="text-align:left;">
MGI:3650114
</td>
<td style="text-align:left;">
lncRNA gene
</td>
<td style="text-align:left;">
predicted gene 12786
</td>
</tr>
<tr>
<td style="text-align:left;">
4
</td>
<td style="text-align:left;">
gene
</td>
<td style="text-align:right;">
107.0559
</td>
<td style="text-align:right;">
107.0669
</td>
<td style="text-align:left;">
negative
</td>
<td style="text-align:left;">
MGI_C57BL6J_1926268
</td>
<td style="text-align:left;">
Mrpl37
</td>
<td style="text-align:left;">
NCBI_Gene:56280,ENSEMBL:ENSMUSG00000028622
</td>
<td style="text-align:left;">
MGI:1926268
</td>
<td style="text-align:left;">
protein coding gene
</td>
<td style="text-align:left;">
mitochondrial ribosomal protein L37
</td>
</tr>
<tr>
<td style="text-align:left;">
4
</td>
<td style="text-align:left;">
gene
</td>
<td style="text-align:right;">
107.0669
</td>
<td style="text-align:right;">
107.0883
</td>
<td style="text-align:left;">
positive
</td>
<td style="text-align:left;">
MGI_C57BL6J_1919657
</td>
<td style="text-align:left;">
Cyb5rl
</td>
<td style="text-align:left;">
NCBI_Gene:230582,ENSEMBL:ENSMUSG00000028621
</td>
<td style="text-align:left;">
MGI:1919657
</td>
<td style="text-align:left;">
protein coding gene
</td>
<td style="text-align:left;">
cytochrome b5 reductase-like
</td>
</tr>
<tr>
<td style="text-align:left;">
4
</td>
<td style="text-align:left;">
gene
</td>
<td style="text-align:right;">
107.0969
</td>
<td style="text-align:right;">
107.1131
</td>
<td style="text-align:left;">
positive
</td>
<td style="text-align:left;">
MGI_C57BL6J_3045328
</td>
<td style="text-align:left;">
Cdcp2
</td>
<td style="text-align:left;">
NCBI_Gene:242603,ENSEMBL:ENSMUSG00000047636
</td>
<td style="text-align:left;">
MGI:3045328
</td>
<td style="text-align:left;">
protein coding gene
</td>
<td style="text-align:left;">
CUB domain containing protein 2
</td>
</tr>
<tr>
<td style="text-align:left;">
4
</td>
<td style="text-align:left;">
gene
</td>
<td style="text-align:right;">
107.1266
</td>
<td style="text-align:right;">
107.1363
</td>
<td style="text-align:left;">
positive
</td>
<td style="text-align:left;">
MGI_C57BL6J_3649683
</td>
<td style="text-align:left;">
Gm12802
</td>
<td style="text-align:left;">
ENSEMBL:ENSMUSG00000085304
</td>
<td style="text-align:left;">
MGI:3649683
</td>
<td style="text-align:left;">
lncRNA gene
</td>
<td style="text-align:left;">
predicted gene 12802
</td>
</tr>
<tr>
<td style="text-align:left;">
4
</td>
<td style="text-align:left;">
gene
</td>
<td style="text-align:right;">
107.1342
</td>
<td style="text-align:right;">
107.1791
</td>
<td style="text-align:left;">
negative
</td>
<td style="text-align:left;">
MGI_C57BL6J_1913776
</td>
<td style="text-align:left;">
Tceanc2
</td>
<td style="text-align:left;">
NCBI_Gene:66526,ENSEMBL:ENSMUSG00000028619
</td>
<td style="text-align:left;">
MGI:1913776
</td>
<td style="text-align:left;">
protein coding gene
</td>
<td style="text-align:left;">
transcription elongation factor A (SII) N-terminal and central domain
containing 2
</td>
</tr>
<tr>
<td style="text-align:left;">
4
</td>
<td style="text-align:left;">
gene
</td>
<td style="text-align:right;">
107.1784
</td>
<td style="text-align:right;">
107.2010
</td>
<td style="text-align:left;">
positive
</td>
<td style="text-align:left;">
MGI_C57BL6J_1929278
</td>
<td style="text-align:left;">
Tmem59
</td>
<td style="text-align:left;">
NCBI_Gene:56374,ENSEMBL:ENSMUSG00000028618
</td>
<td style="text-align:left;">
MGI:1929278
</td>
<td style="text-align:left;">
protein coding gene
</td>
<td style="text-align:left;">
transmembrane protein 59
</td>
</tr>
<tr>
<td style="text-align:left;">
4
</td>
<td style="text-align:left;">
gene
</td>
<td style="text-align:right;">
107.2091
</td>
<td style="text-align:right;">
107.2211
</td>
<td style="text-align:left;">
positive
</td>
<td style="text-align:left;">
MGI_C57BL6J_3652166
</td>
<td style="text-align:left;">
Ldlrad1
</td>
<td style="text-align:left;">
NCBI_Gene:546840,ENSEMBL:ENSMUSG00000070877
</td>
<td style="text-align:left;">
MGI:3652166
</td>
<td style="text-align:left;">
protein coding gene
</td>
<td style="text-align:left;">
low density lipoprotein receptor class A domain containing 1
</td>
</tr>
<tr>
<td style="text-align:left;">
4
</td>
<td style="text-align:left;">
gene
</td>
<td style="text-align:right;">
107.2335
</td>
<td style="text-align:right;">
107.2535
</td>
<td style="text-align:left;">
negative
</td>
<td style="text-align:left;">
MGI_C57BL6J_1925059
</td>
<td style="text-align:left;">
Lrrc42
</td>
<td style="text-align:left;">
NCBI_Gene:77809,ENSEMBL:ENSMUSG00000028617
</td>
<td style="text-align:left;">
MGI:1925059
</td>
<td style="text-align:left;">
protein coding gene
</td>
<td style="text-align:left;">
leucine rich repeat containing 42
</td>
</tr>
<tr>
<td style="text-align:left;">
4
</td>
<td style="text-align:left;">
gene
</td>
<td style="text-align:right;">
107.2536
</td>
<td style="text-align:right;">
107.2845
</td>
<td style="text-align:left;">
positive
</td>
<td style="text-align:left;">
MGI_C57BL6J_1920188
</td>
<td style="text-align:left;">
Hspb11
</td>
<td style="text-align:left;">
NCBI_Gene:72938,ENSEMBL:ENSMUSG00000063172
</td>
<td style="text-align:left;">
MGI:1920188
</td>
<td style="text-align:left;">
protein coding gene
</td>
<td style="text-align:left;">
heat shock protein family B (small), member 11
</td>
</tr>
<tr>
<td style="text-align:left;">
4
</td>
<td style="text-align:left;">
pseudogene
</td>
<td style="text-align:right;">
107.2704
</td>
<td style="text-align:right;">
107.2709
</td>
<td style="text-align:left;">
positive
</td>
<td style="text-align:left;">
MGI_C57BL6J_3651240
</td>
<td style="text-align:left;">
Gm12850
</td>
<td style="text-align:left;">
ENSEMBL:ENSMUSG00000082941
</td>
<td style="text-align:left;">
MGI:3651240
</td>
<td style="text-align:left;">
pseudogene
</td>
<td style="text-align:left;">
predicted gene 12850
</td>
</tr>
<tr>
<td style="text-align:left;">
4
</td>
<td style="text-align:left;">
gene
</td>
<td style="text-align:right;">
107.2915
</td>
<td style="text-align:right;">
107.3072
</td>
<td style="text-align:left;">
negative
</td>
<td style="text-align:left;">
MGI_C57BL6J_94896
</td>
<td style="text-align:left;">
Dio1
</td>
<td style="text-align:left;">
NCBI_Gene:13370,ENSEMBL:ENSMUSG00000034785
</td>
<td style="text-align:left;">
MGI:94896
</td>
<td style="text-align:left;">
protein coding gene
</td>
<td style="text-align:left;">
deiodinase, iodothyronine, type I
</td>
</tr>
<tr>
<td style="text-align:left;">
4
</td>
<td style="text-align:left;">
gene
</td>
<td style="text-align:right;">
107.3142
</td>
<td style="text-align:right;">
107.3598
</td>
<td style="text-align:left;">
positive
</td>
<td style="text-align:left;">
MGI_C57BL6J_1915532
</td>
<td style="text-align:left;">
Yipf1
</td>
<td style="text-align:left;">
NCBI_Gene:230584,ENSEMBL:ENSMUSG00000057375
</td>
<td style="text-align:left;">
MGI:1915532
</td>
<td style="text-align:left;">
protein coding gene
</td>
<td style="text-align:left;">
Yip1 domain family, member 1
</td>
</tr>
<tr>
<td style="text-align:left;">
4
</td>
<td style="text-align:left;">
pseudogene
</td>
<td style="text-align:right;">
107.3268
</td>
<td style="text-align:right;">
107.3285
</td>
<td style="text-align:left;">
negative
</td>
<td style="text-align:left;">
MGI_C57BL6J_3649408
</td>
<td style="text-align:left;">
Gm12870
</td>
<td style="text-align:left;">
NCBI_Gene:652920,ENSEMBL:ENSMUSG00000082279
</td>
<td style="text-align:left;">
MGI:3649408
</td>
<td style="text-align:left;">
pseudogene
</td>
<td style="text-align:left;">
predicted gene 12870
</td>
</tr>
<tr>
<td style="text-align:left;">
4
</td>
<td style="text-align:left;">
gene
</td>
<td style="text-align:right;">
107.3490
</td>
<td style="text-align:right;">
107.3514
</td>
<td style="text-align:left;">
negative
</td>
<td style="text-align:left;">
MGI_C57BL6J_3649409
</td>
<td style="text-align:left;">
Gm12869
</td>
<td style="text-align:left;">
ENSEMBL:ENSMUSG00000086041
</td>
<td style="text-align:left;">
MGI:3649409
</td>
<td style="text-align:left;">
lncRNA gene
</td>
<td style="text-align:left;">
predicted gene 12869
</td>
</tr>
<tr>
<td style="text-align:left;">
4
</td>
<td style="text-align:left;">
gene
</td>
<td style="text-align:right;">
107.3678
</td>
<td style="text-align:right;">
107.4163
</td>
<td style="text-align:left;">
positive
</td>
<td style="text-align:left;">
MGI_C57BL6J_1920037
</td>
<td style="text-align:left;">
Ndc1
</td>
<td style="text-align:left;">
NCBI_Gene:72787,ENSEMBL:ENSMUSG00000028614
</td>
<td style="text-align:left;">
MGI:1920037
</td>
<td style="text-align:left;">
protein coding gene
</td>
<td style="text-align:left;">
NDC1 transmembrane nucleoporin
</td>
</tr>
<tr>
<td style="text-align:left;">
4
</td>
<td style="text-align:left;">
gene
</td>
<td style="text-align:right;">
107.4266
</td>
<td style="text-align:right;">
107.6351
</td>
<td style="text-align:left;">
positive
</td>
<td style="text-align:left;">
MGI_C57BL6J_2386723
</td>
<td style="text-align:left;">
Glis1
</td>
<td style="text-align:left;">
NCBI_Gene:230587,ENSEMBL:ENSMUSG00000034762
</td>
<td style="text-align:left;">
MGI:2386723
</td>
<td style="text-align:left;">
protein coding gene
</td>
<td style="text-align:left;">
GLIS family zinc finger 1
</td>
</tr>
<tr>
<td style="text-align:left;">
4
</td>
<td style="text-align:left;">
gene
</td>
<td style="text-align:right;">
107.4338
</td>
<td style="text-align:right;">
107.4345
</td>
<td style="text-align:left;">
negative
</td>
<td style="text-align:left;">
MGI_C57BL6J_5826503
</td>
<td style="text-align:left;">
Gm46866
</td>
<td style="text-align:left;">
NCBI_Gene:108168971
</td>
<td style="text-align:left;">
MGI:5826503
</td>
<td style="text-align:left;">
lncRNA gene
</td>
<td style="text-align:left;">
predicted gene, 46866
</td>
</tr>
<tr>
<td style="text-align:left;">
4
</td>
<td style="text-align:left;">
gene
</td>
<td style="text-align:right;">
107.4346
</td>
<td style="text-align:right;">
107.4347
</td>
<td style="text-align:left;">
positive
</td>
<td style="text-align:left;">
MGI_C57BL6J_5454581
</td>
<td style="text-align:left;">
Gm24804
</td>
<td style="text-align:left;">
ENSEMBL:ENSMUSG00000092723
</td>
<td style="text-align:left;">
MGI:5454581
</td>
<td style="text-align:left;">
miRNA gene
</td>
<td style="text-align:left;">
predicted gene, 24804
</td>
</tr>
<tr>
<td style="text-align:left;">
4
</td>
<td style="text-align:left;">
gene
</td>
<td style="text-align:right;">
107.5918
</td>
<td style="text-align:right;">
107.6013
</td>
<td style="text-align:left;">
negative
</td>
<td style="text-align:left;">
MGI_C57BL6J_1925456
</td>
<td style="text-align:left;">
4930552P06Rik
</td>
<td style="text-align:left;">
NCBI_Gene:102633643
</td>
<td style="text-align:left;">
MGI:1925456
</td>
<td style="text-align:left;">
lncRNA gene
</td>
<td style="text-align:left;">
RIKEN cDNA 4930552P06 gene
</td>
</tr>
<tr>
<td style="text-align:left;">
4
</td>
<td style="text-align:left;">
gene
</td>
<td style="text-align:right;">
107.6763
</td>
<td style="text-align:right;">
107.6843
</td>
<td style="text-align:left;">
negative
</td>
<td style="text-align:left;">
MGI_C57BL6J_1927125
</td>
<td style="text-align:left;">
Dmrtb1
</td>
<td style="text-align:left;">
NCBI_Gene:56296,ENSEMBL:ENSMUSG00000028610
</td>
<td style="text-align:left;">
MGI:1927125
</td>
<td style="text-align:left;">
protein coding gene
</td>
<td style="text-align:left;">
DMRT-like family B with proline-rich C-terminal, 1
</td>
</tr>
<tr>
<td style="text-align:left;">
4
</td>
<td style="text-align:left;">
pseudogene
</td>
<td style="text-align:right;">
107.6881
</td>
<td style="text-align:right;">
107.6887
</td>
<td style="text-align:left;">
positive
</td>
<td style="text-align:left;">
MGI_C57BL6J_3650162
</td>
<td style="text-align:left;">
Gm12899
</td>
<td style="text-align:left;">
NCBI_Gene:100417388,ENSEMBL:ENSMUSG00000081916
</td>
<td style="text-align:left;">
MGI:3650162
</td>
<td style="text-align:left;">
pseudogene
</td>
<td style="text-align:left;">
predicted gene 12899
</td>
</tr>
<tr>
<td style="text-align:left;">
4
</td>
<td style="text-align:left;">
gene
</td>
<td style="text-align:right;">
107.6896
</td>
<td style="text-align:right;">
107.7590
</td>
<td style="text-align:left;">
positive
</td>
<td style="text-align:left;">
MGI_C57BL6J_1920570
</td>
<td style="text-align:left;">
1700047F07Rik
</td>
<td style="text-align:left;">
NCBI_Gene:102633724,ENSEMBL:ENSMUSG00000085549
</td>
<td style="text-align:left;">
MGI:1920570
</td>
<td style="text-align:left;">
lncRNA gene
</td>
<td style="text-align:left;">
RIKEN cDNA 1700047F07 gene
</td>
</tr>
<tr>
<td style="text-align:left;">
4
</td>
<td style="text-align:left;">
gene
</td>
<td style="text-align:right;">
107.6978
</td>
<td style="text-align:right;">
107.6980
</td>
<td style="text-align:left;">
negative
</td>
<td style="text-align:left;">
MGI_C57BL6J_5454900
</td>
<td style="text-align:left;">
Gm25123
</td>
<td style="text-align:left;">
ENSEMBL:ENSMUSG00000094267
</td>
<td style="text-align:left;">
MGI:5454900
</td>
<td style="text-align:left;">
miRNA gene
</td>
<td style="text-align:left;">
predicted gene, 25123
</td>
</tr>
<tr>
<td style="text-align:left;">
4
</td>
<td style="text-align:left;">
gene
</td>
<td style="text-align:right;">
107.6982
</td>
<td style="text-align:right;">
107.7365
</td>
<td style="text-align:left;">
negative
</td>
<td style="text-align:left;">
MGI_C57BL6J_5590841
</td>
<td style="text-align:left;">
Gm31682
</td>
<td style="text-align:left;">
NCBI_Gene:102633989
</td>
<td style="text-align:left;">
MGI:5590841
</td>
<td style="text-align:left;">
lncRNA gene
</td>
<td style="text-align:left;">
predicted gene, 31682
</td>
</tr>
<tr>
<td style="text-align:left;">
4
</td>
<td style="text-align:left;">
pseudogene
</td>
<td style="text-align:right;">
107.7594
</td>
<td style="text-align:right;">
107.7605
</td>
<td style="text-align:left;">
negative
</td>
<td style="text-align:left;">
MGI_C57BL6J_3650884
</td>
<td style="text-align:left;">
Gm12906
</td>
<td style="text-align:left;">
NCBI_Gene:433743,ENSEMBL:ENSMUSG00000084874
</td>
<td style="text-align:left;">
MGI:3650884
</td>
<td style="text-align:left;">
pseudogene
</td>
<td style="text-align:left;">
predicted gene 12906
</td>
</tr>
<tr>
<td style="text-align:left;">
4
</td>
<td style="text-align:left;">
gene
</td>
<td style="text-align:right;">
107.7595
</td>
<td style="text-align:right;">
107.7710
</td>
<td style="text-align:left;">
positive
</td>
<td style="text-align:left;">
MGI_C57BL6J_3650882
</td>
<td style="text-align:left;">
Gm12907
</td>
<td style="text-align:left;">
NCBI_Gene:102634068,ENSEMBL:ENSMUSG00000087195
</td>
<td style="text-align:left;">
MGI:3650882
</td>
<td style="text-align:left;">
lncRNA gene
</td>
<td style="text-align:left;">
predicted gene 12907
</td>
</tr>
<tr>
<td style="text-align:left;">
4
</td>
<td style="text-align:left;">
gene
</td>
<td style="text-align:right;">
107.7837
</td>
<td style="text-align:right;">
107.7899
</td>
<td style="text-align:left;">
positive
</td>
<td style="text-align:left;">
MGI_C57BL6J_1914798
</td>
<td style="text-align:left;">
4933424M12Rik
</td>
<td style="text-align:left;">
NCBI_Gene:67548,ENSEMBL:ENSMUSG00000087289
</td>
<td style="text-align:left;">
MGI:1914798
</td>
<td style="text-align:left;">
lncRNA gene
</td>
<td style="text-align:left;">
RIKEN cDNA 4933424M12 gene
</td>
</tr>
<tr>
<td style="text-align:left;">
4
</td>
<td style="text-align:left;">
gene
</td>
<td style="text-align:right;">
107.7931
</td>
<td style="text-align:right;">
107.8022
</td>
<td style="text-align:left;">
negative
</td>
<td style="text-align:left;">
MGI_C57BL6J_3651133
</td>
<td style="text-align:left;">
Lrp8os3
</td>
<td style="text-align:left;">
NCBI_Gene:105244644,ENSEMBL:ENSMUSG00000087200
</td>
<td style="text-align:left;">
MGI:3651133
</td>
<td style="text-align:left;">
bidirectional promoter lncRNA gene
</td>
<td style="text-align:left;">
low density lipoprotein receptor-related protein 8, apolipoprotein e
receptor, opposite strand 3
</td>
</tr>
<tr>
<td style="text-align:left;">
4
</td>
<td style="text-align:left;">
gene
</td>
<td style="text-align:right;">
107.8019
</td>
<td style="text-align:right;">
107.8768
</td>
<td style="text-align:left;">
positive
</td>
<td style="text-align:left;">
MGI_C57BL6J_1340044
</td>
<td style="text-align:left;">
Lrp8
</td>
<td style="text-align:left;">
NCBI_Gene:16975,ENSEMBL:ENSMUSG00000028613
</td>
<td style="text-align:left;">
MGI:1340044
</td>
<td style="text-align:left;">
protein coding gene
</td>
<td style="text-align:left;">
low density lipoprotein receptor-related protein 8, apolipoprotein e
receptor
</td>
</tr>
<tr>
<td style="text-align:left;">
4
</td>
<td style="text-align:left;">
gene
</td>
<td style="text-align:right;">
107.8044
</td>
<td style="text-align:right;">
107.8232
</td>
<td style="text-align:left;">
negative
</td>
<td style="text-align:left;">
MGI_C57BL6J_3588233
</td>
<td style="text-align:left;">
Lrp8os2
</td>
<td style="text-align:left;">
NCBI_Gene:619295,ENSEMBL:ENSMUSG00000073779
</td>
<td style="text-align:left;">
MGI:3588233
</td>
<td style="text-align:left;">
antisense lncRNA gene
</td>
<td style="text-align:left;">
low density lipoprotein receptor-related protein 8, apolipoprotein e
receptor, opposite strand 2
</td>
</tr>
<tr>
<td style="text-align:left;">
4
</td>
<td style="text-align:left;">
gene
</td>
<td style="text-align:right;">
107.8279
</td>
<td style="text-align:right;">
107.8308
</td>
<td style="text-align:left;">
negative
</td>
<td style="text-align:left;">
MGI_C57BL6J_1925359
</td>
<td style="text-align:left;">
Lrp8os1
</td>
<td style="text-align:left;">
ENSEMBL:ENSMUSG00000028612
</td>
<td style="text-align:left;">
MGI:1925359
</td>
<td style="text-align:left;">
antisense lncRNA gene
</td>
<td style="text-align:left;">
low density lipoprotein receptor-related protein 8, apolipoprotein e
receptor, opposite strand 1
</td>
</tr>
<tr>
<td style="text-align:left;">
4
</td>
<td style="text-align:left;">
gene
</td>
<td style="text-align:right;">
107.8798
</td>
<td style="text-align:right;">
107.8874
</td>
<td style="text-align:left;">
positive
</td>
<td style="text-align:left;">
MGI_C57BL6J_1330312
</td>
<td style="text-align:left;">
Magoh
</td>
<td style="text-align:left;">
NCBI_Gene:17149,ENSEMBL:ENSMUSG00000028609
</td>
<td style="text-align:left;">
MGI:1330312
</td>
<td style="text-align:left;">
protein coding gene
</td>
<td style="text-align:left;">
mago homolog, exon junction complex core component
</td>
</tr>
<tr>
<td style="text-align:left;">
4
</td>
<td style="text-align:left;">
gene
</td>
<td style="text-align:right;">
107.8898
</td>
<td style="text-align:right;">
107.8994
</td>
<td style="text-align:left;">
positive
</td>
<td style="text-align:left;">
MGI_C57BL6J_1921348
</td>
<td style="text-align:left;">
Czib
</td>
<td style="text-align:left;">
NCBI_Gene:74098,ENSEMBL:ENSMUSG00000028608
</td>
<td style="text-align:left;">
MGI:1921348
</td>
<td style="text-align:left;">
protein coding gene
</td>
<td style="text-align:left;">
CXXC motif containing zinc binding protein
</td>
</tr>
<tr>
<td style="text-align:left;">
4
</td>
<td style="text-align:left;">
gene
</td>
<td style="text-align:right;">
107.9040
</td>
<td style="text-align:right;">
107.9236
</td>
<td style="text-align:left;">
negative
</td>
<td style="text-align:left;">
MGI_C57BL6J_109176
</td>
<td style="text-align:left;">
Cpt2
</td>
<td style="text-align:left;">
NCBI_Gene:12896,ENSEMBL:ENSMUSG00000028607
</td>
<td style="text-align:left;">
MGI:109176
</td>
<td style="text-align:left;">
protein coding gene
</td>
<td style="text-align:left;">
carnitine palmitoyltransferase 2
</td>
</tr>
<tr>
<td style="text-align:left;">
4
</td>
<td style="text-align:left;">
gene
</td>
<td style="text-align:right;">
107.9375
</td>
<td style="text-align:right;">
107.9474
</td>
<td style="text-align:left;">
positive
</td>
<td style="text-align:left;">
MGI_C57BL6J_5623106
</td>
<td style="text-align:left;">
Gm40221
</td>
<td style="text-align:left;">
NCBI_Gene:105244646
</td>
<td style="text-align:left;">
MGI:5623106
</td>
<td style="text-align:left;">
lncRNA gene
</td>
<td style="text-align:left;">
predicted gene, 40221
</td>
</tr>
<tr>
<td style="text-align:left;">
4
</td>
<td style="text-align:left;">
gene
</td>
<td style="text-align:right;">
107.9539
</td>
<td style="text-align:right;">
107.9607
</td>
<td style="text-align:left;">
positive
</td>
<td style="text-align:left;">
MGI_C57BL6J_1924473
</td>
<td style="text-align:left;">
4930407G08Rik
</td>
<td style="text-align:left;">
NCBI_Gene:77223,ENSEMBL:ENSMUSG00000086713
</td>
<td style="text-align:left;">
MGI:1924473
</td>
<td style="text-align:left;">
lncRNA gene
</td>
<td style="text-align:left;">
RIKEN cDNA 4930407G08 gene
</td>
</tr>
<tr>
<td style="text-align:left;">
4
</td>
<td style="text-align:left;">
gene
</td>
<td style="text-align:right;">
107.9683
</td>
<td style="text-align:right;">
108.0135
</td>
<td style="text-align:left;">
positive
</td>
<td style="text-align:left;">
MGI_C57BL6J_2444087
</td>
<td style="text-align:left;">
Slc1a7
</td>
<td style="text-align:left;">
NCBI_Gene:242607,ENSEMBL:ENSMUSG00000008932
</td>
<td style="text-align:left;">
MGI:2444087
</td>
<td style="text-align:left;">
protein coding gene
</td>
<td style="text-align:left;">
solute carrier family 1 (glutamate transporter), member 7
</td>
</tr>
<tr>
<td style="text-align:left;">
4
</td>
<td style="text-align:left;">
gene
</td>
<td style="text-align:right;">
108.0148
</td>
<td style="text-align:right;">
108.0964
</td>
<td style="text-align:left;">
negative
</td>
<td style="text-align:left;">
MGI_C57BL6J_2674939
</td>
<td style="text-align:left;">
Podn
</td>
<td style="text-align:left;">
NCBI_Gene:242608,ENSEMBL:ENSMUSG00000028600
</td>
<td style="text-align:left;">
MGI:2674939
</td>
<td style="text-align:left;">
protein coding gene
</td>
<td style="text-align:left;">
podocan
</td>
</tr>
<tr>
<td style="text-align:left;">
4
</td>
<td style="text-align:left;">
gene
</td>
<td style="text-align:right;">
108.0417
</td>
<td style="text-align:right;">
108.0418
</td>
<td style="text-align:left;">
negative
</td>
<td style="text-align:left;">
MGI_C57BL6J_5453131
</td>
<td style="text-align:left;">
Gm23354
</td>
<td style="text-align:left;">
ENSEMBL:ENSMUSG00000089051
</td>
<td style="text-align:left;">
MGI:5453131
</td>
<td style="text-align:left;">
snRNA gene
</td>
<td style="text-align:left;">
predicted gene, 23354
</td>
</tr>
<tr>
<td style="text-align:left;">
4
</td>
<td style="text-align:left;">
gene
</td>
<td style="text-align:right;">
108.0438
</td>
<td style="text-align:right;">
108.1450
</td>
<td style="text-align:left;">
negative
</td>
<td style="text-align:left;">
MGI_C57BL6J_98254
</td>
<td style="text-align:left;">
Scp2
</td>
<td style="text-align:left;">
NCBI_Gene:20280,ENSEMBL:ENSMUSG00000028603
</td>
<td style="text-align:left;">
MGI:98254
</td>
<td style="text-align:left;">
protein coding gene
</td>
<td style="text-align:left;">
sterol carrier protein 2, liver
</td>
</tr>
<tr>
<td style="text-align:left;">
4
</td>
<td style="text-align:left;">
gene
</td>
<td style="text-align:right;">
108.0874
</td>
<td style="text-align:right;">
108.0874
</td>
<td style="text-align:left;">
positive
</td>
<td style="text-align:left;">
MGI_C57BL6J_5531386
</td>
<td style="text-align:left;">
Mir6397
</td>
<td style="text-align:left;">
miRBase:MI0021933,NCBI_Gene:102465213,ENSEMBL:ENSMUSG00000098980
</td>
<td style="text-align:left;">
MGI:5531386
</td>
<td style="text-align:left;">
miRNA gene
</td>
<td style="text-align:left;">
microRNA 6397
</td>
</tr>
<tr>
<td style="text-align:left;">
4
</td>
<td style="text-align:left;">
gene
</td>
<td style="text-align:right;">
108.1654
</td>
<td style="text-align:right;">
108.1793
</td>
<td style="text-align:left;">
positive
</td>
<td style="text-align:left;">
MGI_C57BL6J_1289238
</td>
<td style="text-align:left;">
Echdc2
</td>
<td style="text-align:left;">
NCBI_Gene:52430,ENSEMBL:ENSMUSG00000028601
</td>
<td style="text-align:left;">
MGI:1289238
</td>
<td style="text-align:left;">
protein coding gene
</td>
<td style="text-align:left;">
enoyl Coenzyme A hydratase domain containing 2
</td>
</tr>
<tr>
<td style="text-align:left;">
4
</td>
<td style="text-align:left;">
gene
</td>
<td style="text-align:right;">
108.1802
</td>
<td style="text-align:right;">
108.2182
</td>
<td style="text-align:left;">
negative
</td>
<td style="text-align:left;">
MGI_C57BL6J_2446208
</td>
<td style="text-align:left;">
Zyg11a
</td>
<td style="text-align:left;">
NCBI_Gene:230590,ENSEMBL:ENSMUSG00000034645
</td>
<td style="text-align:left;">
MGI:2446208
</td>
<td style="text-align:left;">
protein coding gene
</td>
<td style="text-align:left;">
zyg-11 family member A, cell cycle regulator
</td>
</tr>
<tr>
<td style="text-align:left;">
4
</td>
<td style="text-align:left;">
gene
</td>
<td style="text-align:right;">
108.2180
</td>
<td style="text-align:right;">
108.2205
</td>
<td style="text-align:left;">
positive
</td>
<td style="text-align:left;">
MGI_C57BL6J_5623107
</td>
<td style="text-align:left;">
Gm40222
</td>
<td style="text-align:left;">
NCBI_Gene:105244647
</td>
<td style="text-align:left;">
MGI:5623107
</td>
<td style="text-align:left;">
lncRNA gene
</td>
<td style="text-align:left;">
predicted gene, 40222
</td>
</tr>
<tr>
<td style="text-align:left;">
4
</td>
<td style="text-align:left;">
gene
</td>
<td style="text-align:right;">
108.2278
</td>
<td style="text-align:right;">
108.3011
</td>
<td style="text-align:left;">
negative
</td>
<td style="text-align:left;">
MGI_C57BL6J_2685277
</td>
<td style="text-align:left;">
Zyg11b
</td>
<td style="text-align:left;">
NCBI_Gene:414872,ENSEMBL:ENSMUSG00000034636
</td>
<td style="text-align:left;">
MGI:2685277
</td>
<td style="text-align:left;">
protein coding gene
</td>
<td style="text-align:left;">
zyg-ll family member B, cell cycle regulator
</td>
</tr>
<tr>
<td style="text-align:left;">
4
</td>
<td style="text-align:left;">
pseudogene
</td>
<td style="text-align:right;">
108.3104
</td>
<td style="text-align:right;">
108.3112
</td>
<td style="text-align:left;">
negative
</td>
<td style="text-align:left;">
MGI_C57BL6J_3651646
</td>
<td style="text-align:left;">
Gm12742
</td>
<td style="text-align:left;">
ENSEMBL:ENSMUSG00000083957
</td>
<td style="text-align:left;">
MGI:3651646
</td>
<td style="text-align:left;">
pseudogene
</td>
<td style="text-align:left;">
predicted gene 12742
</td>
</tr>
<tr>
<td style="text-align:left;">
4
</td>
<td style="text-align:left;">
pseudogene
</td>
<td style="text-align:right;">
108.3223
</td>
<td style="text-align:right;">
108.3226
</td>
<td style="text-align:left;">
positive
</td>
<td style="text-align:left;">
MGI_C57BL6J_3651641
</td>
<td style="text-align:left;">
Gm12740
</td>
<td style="text-align:left;">
ENSEMBL:ENSMUSG00000081734
</td>
<td style="text-align:left;">
MGI:3651641
</td>
<td style="text-align:left;">
pseudogene
</td>
<td style="text-align:left;">
predicted gene 12740
</td>
</tr>
<tr>
<td style="text-align:left;">
4
</td>
<td style="text-align:left;">
gene
</td>
<td style="text-align:right;">
108.3281
</td>
<td style="text-align:right;">
108.3415
</td>
<td style="text-align:left;">
positive
</td>
<td style="text-align:left;">
MGI_C57BL6J_1917143
</td>
<td style="text-align:left;">
Coa7
</td>
<td style="text-align:left;">
NCBI_Gene:69893,ENSEMBL:ENSMUSG00000048351
</td>
<td style="text-align:left;">
MGI:1917143
</td>
<td style="text-align:left;">
protein coding gene
</td>
<td style="text-align:left;">
cytochrome c oxidase assembly factor 7
</td>
</tr>
<tr>
<td style="text-align:left;">
4
</td>
<td style="text-align:left;">
gene
</td>
<td style="text-align:right;">
108.3455
</td>
<td style="text-align:right;">
108.3512
</td>
<td style="text-align:left;">
negative
</td>
<td style="text-align:left;">
MGI_C57BL6J_5826526
</td>
<td style="text-align:left;">
Gm46889
</td>
<td style="text-align:left;">
NCBI_Gene:108169005
</td>
<td style="text-align:left;">
MGI:5826526
</td>
<td style="text-align:left;">
lncRNA gene
</td>
<td style="text-align:left;">
predicted gene, 46889
</td>
</tr>
<tr>
<td style="text-align:left;">
4
</td>
<td style="text-align:left;">
gene
</td>
<td style="text-align:right;">
108.3678
</td>
<td style="text-align:right;">
108.3858
</td>
<td style="text-align:left;">
negative
</td>
<td style="text-align:left;">
MGI_C57BL6J_3651644
</td>
<td style="text-align:left;">
Shisal2a
</td>
<td style="text-align:left;">
NCBI_Gene:545667,ENSEMBL:ENSMUSG00000059816
</td>
<td style="text-align:left;">
MGI:3651644
</td>
<td style="text-align:left;">
protein coding gene
</td>
<td style="text-align:left;">
shisa like 2A
</td>
</tr>
<tr>
<td style="text-align:left;">
4
</td>
<td style="text-align:left;">
gene
</td>
<td style="text-align:right;">
108.3833
</td>
<td style="text-align:right;">
108.4066
</td>
<td style="text-align:left;">
positive
</td>
<td style="text-align:left;">
MGI_C57BL6J_5591456
</td>
<td style="text-align:left;">
Gm32297
</td>
<td style="text-align:left;">
NCBI_Gene:105244648
</td>
<td style="text-align:left;">
MGI:5591456
</td>
<td style="text-align:left;">
lncRNA gene
</td>
<td style="text-align:left;">
predicted gene, 32297
</td>
</tr>
<tr>
<td style="text-align:left;">
4
</td>
<td style="text-align:left;">
gene
</td>
<td style="text-align:right;">
108.4002
</td>
<td style="text-align:right;">
108.4070
</td>
<td style="text-align:left;">
negative
</td>
<td style="text-align:left;">
MGI_C57BL6J_1914555
</td>
<td style="text-align:left;">
Gpx7
</td>
<td style="text-align:left;">
NCBI_Gene:67305,ENSEMBL:ENSMUSG00000028597
</td>
<td style="text-align:left;">
MGI:1914555
</td>
<td style="text-align:left;">
protein coding gene
</td>
<td style="text-align:left;">
glutathione peroxidase 7
</td>
</tr>
<tr>
<td style="text-align:left;">
4
</td>
<td style="text-align:left;">
gene
</td>
<td style="text-align:right;">
108.4306
</td>
<td style="text-align:right;">
108.4542
</td>
<td style="text-align:left;">
positive
</td>
<td style="text-align:left;">
MGI_C57BL6J_5623108
</td>
<td style="text-align:left;">
Gm40223
</td>
<td style="text-align:left;">
NCBI_Gene:105244650
</td>
<td style="text-align:left;">
MGI:5623108
</td>
<td style="text-align:left;">
lncRNA gene
</td>
<td style="text-align:left;">
predicted gene, 40223
</td>
</tr>
<tr>
<td style="text-align:left;">
4
</td>
<td style="text-align:left;">
pseudogene
</td>
<td style="text-align:right;">
108.4507
</td>
<td style="text-align:right;">
108.4514
</td>
<td style="text-align:left;">
negative
</td>
<td style="text-align:left;">
MGI_C57BL6J_3651217
</td>
<td style="text-align:left;">
Gm12738
</td>
<td style="text-align:left;">
NCBI_Gene:665728,ENSEMBL:ENSMUSG00000083255
</td>
<td style="text-align:left;">
MGI:3651217
</td>
<td style="text-align:left;">
pseudogene
</td>
<td style="text-align:left;">
predicted gene 12738
</td>
</tr>
<tr>
<td style="text-align:left;">
4
</td>
<td style="text-align:left;">
gene
</td>
<td style="text-align:right;">
108.4594
</td>
<td style="text-align:right;">
108.5607
</td>
<td style="text-align:left;">
positive
</td>
<td style="text-align:left;">
MGI_C57BL6J_2445126
</td>
<td style="text-align:left;">
Tut4
</td>
<td style="text-align:left;">
NCBI_Gene:230594,ENSEMBL:ENSMUSG00000034610
</td>
<td style="text-align:left;">
MGI:2445126
</td>
<td style="text-align:left;">
protein coding gene
</td>
<td style="text-align:left;">
terminal uridylyl transferase 4
</td>
</tr>
<tr>
<td style="text-align:left;">
4
</td>
<td style="text-align:left;">
gene
</td>
<td style="text-align:right;">
108.4669
</td>
<td style="text-align:right;">
108.4671
</td>
<td style="text-align:left;">
negative
</td>
<td style="text-align:left;">
MGI_C57BL6J_5454744
</td>
<td style="text-align:left;">
Gm24967
</td>
<td style="text-align:left;">
ENSEMBL:ENSMUSG00000095361
</td>
<td style="text-align:left;">
MGI:5454744
</td>
<td style="text-align:left;">
snRNA gene
</td>
<td style="text-align:left;">
predicted gene, 24967
</td>
</tr>
<tr>
<td style="text-align:left;">
4
</td>
<td style="text-align:left;">
pseudogene
</td>
<td style="text-align:right;">
108.4841
</td>
<td style="text-align:right;">
108.4849
</td>
<td style="text-align:left;">
negative
</td>
<td style="text-align:left;">
MGI_C57BL6J_3651018
</td>
<td style="text-align:left;">
Gm12739
</td>
<td style="text-align:left;">
NCBI_Gene:105244651,ENSEMBL:ENSMUSG00000083783
</td>
<td style="text-align:left;">
MGI:3651018
</td>
<td style="text-align:left;">
pseudogene
</td>
<td style="text-align:left;">
predicted gene 12739
</td>
</tr>
<tr>
<td style="text-align:left;">
4
</td>
<td style="text-align:left;">
gene
</td>
<td style="text-align:right;">
108.4980
</td>
<td style="text-align:right;">
108.4981
</td>
<td style="text-align:left;">
positive
</td>
<td style="text-align:left;">
MGI_C57BL6J_5452915
</td>
<td style="text-align:left;">
Gm23138
</td>
<td style="text-align:left;">
ENSEMBL:ENSMUSG00000065299
</td>
<td style="text-align:left;">
MGI:5452915
</td>
<td style="text-align:left;">
snRNA gene
</td>
<td style="text-align:left;">
predicted gene, 23138
</td>
</tr>
<tr>
<td style="text-align:left;">
4
</td>
<td style="text-align:left;">
gene
</td>
<td style="text-align:right;">
108.5138
</td>
<td style="text-align:right;">
108.5168
</td>
<td style="text-align:left;">
positive
</td>
<td style="text-align:left;">
MGI_C57BL6J_3651021
</td>
<td style="text-align:left;">
Gm12737
</td>
<td style="text-align:left;">
ENSEMBL:ENSMUSG00000085093
</td>
<td style="text-align:left;">
MGI:3651021
</td>
<td style="text-align:left;">
lncRNA gene
</td>
<td style="text-align:left;">
predicted gene 12737
</td>
</tr>
<tr>
<td style="text-align:left;">
4
</td>
<td style="text-align:left;">
gene
</td>
<td style="text-align:right;">
108.5219
</td>
<td style="text-align:right;">
108.5220
</td>
<td style="text-align:left;">
negative
</td>
<td style="text-align:left;">
MGI_C57BL6J_5455604
</td>
<td style="text-align:left;">
Gm25827
</td>
<td style="text-align:left;">
ENSEMBL:ENSMUSG00000087714
</td>
<td style="text-align:left;">
MGI:5455604
</td>
<td style="text-align:left;">
snRNA gene
</td>
<td style="text-align:left;">
predicted gene, 25827
</td>
</tr>
<tr>
<td style="text-align:left;">
4
</td>
<td style="text-align:left;">
gene
</td>
<td style="text-align:right;">
108.5375
</td>
<td style="text-align:right;">
108.5376
</td>
<td style="text-align:left;">
negative
</td>
<td style="text-align:left;">
MGI_C57BL6J_5452122
</td>
<td style="text-align:left;">
Gm22345
</td>
<td style="text-align:left;">
ENSEMBL:ENSMUSG00000084593
</td>
<td style="text-align:left;">
MGI:5452122
</td>
<td style="text-align:left;">
snRNA gene
</td>
<td style="text-align:left;">
predicted gene, 22345
</td>
</tr>
<tr>
<td style="text-align:left;">
4
</td>
<td style="text-align:left;">
gene
</td>
<td style="text-align:right;">
108.5632
</td>
<td style="text-align:right;">
108.5793
</td>
<td style="text-align:left;">
negative
</td>
<td style="text-align:left;">
MGI_C57BL6J_1916962
</td>
<td style="text-align:left;">
Prpf38a
</td>
<td style="text-align:left;">
NCBI_Gene:230596,ENSEMBL:ENSMUSG00000063800
</td>
<td style="text-align:left;">
MGI:1916962
</td>
<td style="text-align:left;">
protein coding gene
</td>
<td style="text-align:left;">
PRP38 pre-mRNA processing factor 38 (yeast) domain containing A
</td>
</tr>
<tr>
<td style="text-align:left;">
4
</td>
<td style="text-align:left;">
gene
</td>
<td style="text-align:right;">
108.5756
</td>
<td style="text-align:right;">
108.6160
</td>
<td style="text-align:left;">
positive
</td>
<td style="text-align:left;">
MGI_C57BL6J_1328337
</td>
<td style="text-align:left;">
Orc1
</td>
<td style="text-align:left;">
NCBI_Gene:18392,ENSEMBL:ENSMUSG00000028587
</td>
<td style="text-align:left;">
MGI:1328337
</td>
<td style="text-align:left;">
protein coding gene
</td>
<td style="text-align:left;">
origin recognition complex, subunit 1
</td>
</tr>
<tr>
<td style="text-align:left;">
4
</td>
<td style="text-align:left;">
gene
</td>
<td style="text-align:right;">
108.5924
</td>
<td style="text-align:right;">
108.6198
</td>
<td style="text-align:left;">
negative
</td>
<td style="text-align:left;">
MGI_C57BL6J_3651449
</td>
<td style="text-align:left;">
Gm12743
</td>
<td style="text-align:left;">
ENSEMBL:ENSMUSG00000084968
</td>
<td style="text-align:left;">
MGI:3651449
</td>
<td style="text-align:left;">
lncRNA gene
</td>
<td style="text-align:left;">
predicted gene 12743
</td>
</tr>
<tr>
<td style="text-align:left;">
4
</td>
<td style="text-align:left;">
gene
</td>
<td style="text-align:right;">
108.6198
</td>
<td style="text-align:right;">
108.6355
</td>
<td style="text-align:left;">
positive
</td>
<td style="text-align:left;">
MGI_C57BL6J_2443076
</td>
<td style="text-align:left;">
Cc2d1b
</td>
<td style="text-align:left;">
NCBI_Gene:319965,ENSEMBL:ENSMUSG00000028582
</td>
<td style="text-align:left;">
MGI:2443076
</td>
<td style="text-align:left;">
protein coding gene
</td>
<td style="text-align:left;">
coiled-coil and C2 domain containing 1B
</td>
</tr>
<tr>
<td style="text-align:left;">
4
</td>
<td style="text-align:left;">
gene
</td>
<td style="text-align:right;">
108.6229
</td>
<td style="text-align:right;">
108.6252
</td>
<td style="text-align:left;">
negative
</td>
<td style="text-align:left;">
MGI_C57BL6J_5313040
</td>
<td style="text-align:left;">
Gm20731
</td>
<td style="text-align:left;">
ENSEMBL:ENSMUSG00000102912
</td>
<td style="text-align:left;">
MGI:5313040
</td>
<td style="text-align:left;">
protein coding gene
</td>
<td style="text-align:left;">
predicted gene, 20731
</td>
</tr>
<tr>
<td style="text-align:left;">
4
</td>
<td style="text-align:left;">
gene
</td>
<td style="text-align:right;">
108.6338
</td>
<td style="text-align:right;">
108.6374
</td>
<td style="text-align:left;">
negative
</td>
<td style="text-align:left;">
MGI_C57BL6J_4936988
</td>
<td style="text-align:left;">
Gm17354
</td>
<td style="text-align:left;">
ENSEMBL:ENSMUSG00000091985
</td>
<td style="text-align:left;">
MGI:4936988
</td>
<td style="text-align:left;">
lncRNA gene
</td>
<td style="text-align:left;">
predicted gene, 17354
</td>
</tr>
<tr>
<td style="text-align:left;">
4
</td>
<td style="text-align:left;">
gene
</td>
<td style="text-align:right;">
108.6375
</td>
<td style="text-align:right;">
108.7822
</td>
<td style="text-align:left;">
negative
</td>
<td style="text-align:left;">
MGI_C57BL6J_2652838
</td>
<td style="text-align:left;">
Zfyve9
</td>
<td style="text-align:left;">
NCBI_Gene:230597,ENSEMBL:ENSMUSG00000034557
</td>
<td style="text-align:left;">
MGI:2652838
</td>
<td style="text-align:left;">
protein coding gene
</td>
<td style="text-align:left;">
zinc finger, FYVE domain containing 9
</td>
</tr>
<tr>
<td style="text-align:left;">
4
</td>
<td style="text-align:left;">
gene
</td>
<td style="text-align:right;">
108.7196
</td>
<td style="text-align:right;">
108.7819
</td>
<td style="text-align:left;">
positive
</td>
<td style="text-align:left;">
MGI_C57BL6J_1920383
</td>
<td style="text-align:left;">
3110021N24Rik
</td>
<td style="text-align:left;">
ENSEMBL:ENSMUSG00000094958
</td>
<td style="text-align:left;">
MGI:1920383
</td>
<td style="text-align:left;">
protein coding gene
</td>
<td style="text-align:left;">
RIKEN cDNA 3110021N24 gene
</td>
</tr>
<tr>
<td style="text-align:left;">
4
</td>
<td style="text-align:left;">
pseudogene
</td>
<td style="text-align:right;">
108.7317
</td>
<td style="text-align:right;">
108.7324
</td>
<td style="text-align:left;">
positive
</td>
<td style="text-align:left;">
MGI_C57BL6J_3651642
</td>
<td style="text-align:left;">
Gm12741
</td>
<td style="text-align:left;">
NCBI_Gene:100040326,ENSEMBL:ENSMUSG00000082374
</td>
<td style="text-align:left;">
MGI:3651642
</td>
<td style="text-align:left;">
pseudogene
</td>
<td style="text-align:left;">
predicted gene 12741
</td>
</tr>
<tr>
<td style="text-align:left;">
4
</td>
<td style="text-align:left;">
gene
</td>
<td style="text-align:right;">
108.8143
</td>
<td style="text-align:right;">
108.8343
</td>
<td style="text-align:left;">
negative
</td>
<td style="text-align:left;">
MGI_C57BL6J_1915312
</td>
<td style="text-align:left;">
Btf3l4
</td>
<td style="text-align:left;">
NCBI_Gene:70533,ENSEMBL:ENSMUSG00000028568
</td>
<td style="text-align:left;">
MGI:1915312
</td>
<td style="text-align:left;">
protein coding gene
</td>
<td style="text-align:left;">
basic transcription factor 3-like 4
</td>
</tr>
<tr>
<td style="text-align:left;">
4
</td>
<td style="text-align:left;">
gene
</td>
<td style="text-align:right;">
108.8346
</td>
<td style="text-align:right;">
108.8621
</td>
<td style="text-align:left;">
positive
</td>
<td style="text-align:left;">
MGI_C57BL6J_1913323
</td>
<td style="text-align:left;">
Txndc12
</td>
<td style="text-align:left;">
NCBI_Gene:66073,ENSEMBL:ENSMUSG00000028567
</td>
<td style="text-align:left;">
MGI:1913323
</td>
<td style="text-align:left;">
protein coding gene
</td>
<td style="text-align:left;">
thioredoxin domain containing 12 (endoplasmic reticulum)
</td>
</tr>
<tr>
<td style="text-align:left;">
4
</td>
<td style="text-align:left;">
gene
</td>
<td style="text-align:right;">
108.8474
</td>
<td style="text-align:right;">
108.8486
</td>
<td style="text-align:left;">
negative
</td>
<td style="text-align:left;">
MGI_C57BL6J_3704235
</td>
<td style="text-align:left;">
A730015C16Rik
</td>
<td style="text-align:left;">
NA
</td>
<td style="text-align:left;">
NA
</td>
<td style="text-align:left;">
lncRNA gene
</td>
<td style="text-align:left;">
RIKEN cDNA A730015C16 gene
</td>
</tr>
<tr>
<td style="text-align:left;">
4
</td>
<td style="text-align:left;">
gene
</td>
<td style="text-align:right;">
108.8478
</td>
<td style="text-align:right;">
108.8494
</td>
<td style="text-align:left;">
positive
</td>
<td style="text-align:left;">
MGI_C57BL6J_1923547
</td>
<td style="text-align:left;">
Kti12
</td>
<td style="text-align:left;">
NCBI_Gene:100087,ENSEMBL:ENSMUSG00000073775
</td>
<td style="text-align:left;">
MGI:1923547
</td>
<td style="text-align:left;">
protein coding gene
</td>
<td style="text-align:left;">
KTI12 homolog, chromatin associated
</td>
</tr>
<tr>
<td style="text-align:left;">
4
</td>
<td style="text-align:left;">
gene
</td>
<td style="text-align:right;">
108.8509
</td>
<td style="text-align:right;">
108.8520
</td>
<td style="text-align:left;">
positive
</td>
<td style="text-align:left;">
MGI_C57BL6J_1924896
</td>
<td style="text-align:left;">
C030043A13Rik
</td>
<td style="text-align:left;">
NA
</td>
<td style="text-align:left;">
NA
</td>
<td style="text-align:left;">
unclassified gene
</td>
<td style="text-align:left;">
RIKEN cDNA C030043A13 gene
</td>
</tr>
<tr>
<td style="text-align:left;">
4
</td>
<td style="text-align:left;">
gene
</td>
<td style="text-align:right;">
108.8791
</td>
<td style="text-align:right;">
108.9433
</td>
<td style="text-align:left;">
positive
</td>
<td style="text-align:left;">
MGI_C57BL6J_1917158
</td>
<td style="text-align:left;">
Rab3b
</td>
<td style="text-align:left;">
NCBI_Gene:69908,ENSEMBL:ENSMUSG00000003411
</td>
<td style="text-align:left;">
MGI:1917158
</td>
<td style="text-align:left;">
protein coding gene
</td>
<td style="text-align:left;">
RAB3B, member RAS oncogene family
</td>
</tr>
<tr>
<td style="text-align:left;">
4
</td>
<td style="text-align:left;">
pseudogene
</td>
<td style="text-align:right;">
108.9336
</td>
<td style="text-align:right;">
108.9338
</td>
<td style="text-align:left;">
negative
</td>
<td style="text-align:left;">
MGI_C57BL6J_3650723
</td>
<td style="text-align:left;">
Gm12736
</td>
<td style="text-align:left;">
ENSEMBL:ENSMUSG00000082699
</td>
<td style="text-align:left;">
MGI:3650723
</td>
<td style="text-align:left;">
pseudogene
</td>
<td style="text-align:left;">
predicted gene 12736
</td>
</tr>
<tr>
<td style="text-align:left;">
4
</td>
<td style="text-align:left;">
gene
</td>
<td style="text-align:right;">
108.9428
</td>
<td style="text-align:right;">
108.9721
</td>
<td style="text-align:left;">
negative
</td>
<td style="text-align:left;">
MGI_C57BL6J_1924423
</td>
<td style="text-align:left;">
8030443G20Rik
</td>
<td style="text-align:left;">
NCBI_Gene:77173,ENSEMBL:ENSMUSG00000086483
</td>
<td style="text-align:left;">
MGI:1924423
</td>
<td style="text-align:left;">
lncRNA gene
</td>
<td style="text-align:left;">
RIKEN cDNA 8030443G20 gene
</td>
</tr>
<tr>
<td style="text-align:left;">
4
</td>
<td style="text-align:left;">
gene
</td>
<td style="text-align:right;">
108.9986
</td>
<td style="text-align:right;">
108.9987
</td>
<td style="text-align:left;">
negative
</td>
<td style="text-align:left;">
MGI_C57BL6J_5453366
</td>
<td style="text-align:left;">
Gm23589
</td>
<td style="text-align:left;">
ENSEMBL:ENSMUSG00000088285
</td>
<td style="text-align:left;">
MGI:5453366
</td>
<td style="text-align:left;">
snRNA gene
</td>
<td style="text-align:left;">
predicted gene, 23589
</td>
</tr>
<tr>
<td style="text-align:left;">
4
</td>
<td style="text-align:left;">
gene
</td>
<td style="text-align:right;">
109.0006
</td>
<td style="text-align:right;">
109.0618
</td>
<td style="text-align:left;">
positive
</td>
<td style="text-align:left;">
MGI_C57BL6J_1201386
</td>
<td style="text-align:left;">
Nrd1
</td>
<td style="text-align:left;">
NCBI_Gene:230598,ENSEMBL:ENSMUSG00000053510
</td>
<td style="text-align:left;">
MGI:1201386
</td>
<td style="text-align:left;">
protein coding gene
</td>
<td style="text-align:left;">
nardilysin, N-arginine dibasic convertase, NRD convertase 1
</td>
</tr>
<tr>
<td style="text-align:left;">
4
</td>
<td style="text-align:left;">
gene
</td>
<td style="text-align:right;">
109.0177
</td>
<td style="text-align:right;">
109.0177
</td>
<td style="text-align:left;">
positive
</td>
<td style="text-align:left;">
MGI_C57BL6J_3691609
</td>
<td style="text-align:left;">
Mir761
</td>
<td style="text-align:left;">
miRBase:MI0004306,NCBI_Gene:791075,ENSEMBL:ENSMUSG00000076444
</td>
<td style="text-align:left;">
MGI:3691609
</td>
<td style="text-align:left;">
miRNA gene
</td>
<td style="text-align:left;">
microRNA 761
</td>
</tr>
<tr>
<td style="text-align:left;">
4
</td>
<td style="text-align:left;">
gene
</td>
<td style="text-align:right;">
109.0611
</td>
<td style="text-align:right;">
109.2023
</td>
<td style="text-align:left;">
negative
</td>
<td style="text-align:left;">
MGI_C57BL6J_1923784
</td>
<td style="text-align:left;">
Osbpl9
</td>
<td style="text-align:left;">
NCBI_Gene:100273,ENSEMBL:ENSMUSG00000028559
</td>
<td style="text-align:left;">
MGI:1923784
</td>
<td style="text-align:left;">
protein coding gene
</td>
<td style="text-align:left;">
oxysterol binding protein-like 9
</td>
</tr>
<tr>
<td style="text-align:left;">
4
</td>
<td style="text-align:left;">
gene
</td>
<td style="text-align:right;">
109.0698
</td>
<td style="text-align:right;">
109.0702
</td>
<td style="text-align:left;">
negative
</td>
<td style="text-align:left;">
MGI_C57BL6J_1925486
</td>
<td style="text-align:left;">
5330417P21Rik
</td>
<td style="text-align:left;">
NA
</td>
<td style="text-align:left;">
NA
</td>
<td style="text-align:left;">
unclassified gene
</td>
<td style="text-align:left;">
RIKEN cDNA 5330417P21 gene
</td>
</tr>
<tr>
<td style="text-align:left;">
4
</td>
<td style="text-align:left;">
gene
</td>
<td style="text-align:right;">
109.0785
</td>
<td style="text-align:right;">
109.0812
</td>
<td style="text-align:left;">
positive
</td>
<td style="text-align:left;">
MGI_C57BL6J_5591740
</td>
<td style="text-align:left;">
Gm32581
</td>
<td style="text-align:left;">
NCBI_Gene:102635178
</td>
<td style="text-align:left;">
MGI:5591740
</td>
<td style="text-align:left;">
lncRNA gene
</td>
<td style="text-align:left;">
predicted gene, 32581
</td>
</tr>
<tr>
<td style="text-align:left;">
4
</td>
<td style="text-align:left;">
gene
</td>
<td style="text-align:right;">
109.1030
</td>
<td style="text-align:right;">
109.1035
</td>
<td style="text-align:left;">
negative
</td>
<td style="text-align:left;">
MGI_C57BL6J_1924670
</td>
<td style="text-align:left;">
C030032G21Rik
</td>
<td style="text-align:left;">
NA
</td>
<td style="text-align:left;">
NA
</td>
<td style="text-align:left;">
unclassified gene
</td>
<td style="text-align:left;">
RIKEN cDNA C030032G21 gene
</td>
</tr>
<tr>
<td style="text-align:left;">
4
</td>
<td style="text-align:left;">
gene
</td>
<td style="text-align:right;">
109.1609
</td>
<td style="text-align:right;">
109.1612
</td>
<td style="text-align:left;">
negative
</td>
<td style="text-align:left;">
MGI_C57BL6J_5453520
</td>
<td style="text-align:left;">
Gm23743
</td>
<td style="text-align:left;">
ENSEMBL:ENSMUSG00000088014
</td>
<td style="text-align:left;">
MGI:5453520
</td>
<td style="text-align:left;">
unclassified non-coding RNA gene
</td>
<td style="text-align:left;">
predicted gene, 23743
</td>
</tr>
<tr>
<td style="text-align:left;">
4
</td>
<td style="text-align:left;">
gene
</td>
<td style="text-align:right;">
109.2022
</td>
<td style="text-align:right;">
109.2085
</td>
<td style="text-align:left;">
positive
</td>
<td style="text-align:left;">
MGI_C57BL6J_5826504
</td>
<td style="text-align:left;">
Gm46867
</td>
<td style="text-align:left;">
NCBI_Gene:108168972
</td>
<td style="text-align:left;">
MGI:5826504
</td>
<td style="text-align:left;">
lncRNA gene
</td>
<td style="text-align:left;">
predicted gene, 46867
</td>
</tr>
<tr>
<td style="text-align:left;">
4
</td>
<td style="text-align:left;">
gene
</td>
<td style="text-align:right;">
109.2184
</td>
<td style="text-align:right;">
109.2551
</td>
<td style="text-align:left;">
positive
</td>
<td style="text-align:left;">
MGI_C57BL6J_2140435
</td>
<td style="text-align:left;">
Calr4
</td>
<td style="text-align:left;">
NCBI_Gene:108802,ENSEMBL:ENSMUSG00000028558
</td>
<td style="text-align:left;">
MGI:2140435
</td>
<td style="text-align:left;">
protein coding gene
</td>
<td style="text-align:left;">
calreticulin 4
</td>
</tr>
<tr>
<td style="text-align:left;">
4
</td>
<td style="text-align:left;">
gene
</td>
<td style="text-align:right;">
109.2241
</td>
<td style="text-align:right;">
109.2801
</td>
<td style="text-align:left;">
negative
</td>
<td style="text-align:left;">
MGI_C57BL6J_5623110
</td>
<td style="text-align:left;">
Gm40225
</td>
<td style="text-align:left;">
NCBI_Gene:105244653
</td>
<td style="text-align:left;">
MGI:5623110
</td>
<td style="text-align:left;">
lncRNA gene
</td>
<td style="text-align:left;">
predicted gene, 40225
</td>
</tr>
<tr>
<td style="text-align:left;">
4
</td>
<td style="text-align:left;">
gene
</td>
<td style="text-align:right;">
109.2803
</td>
<td style="text-align:right;">
109.3878
</td>
<td style="text-align:left;">
positive
</td>
<td style="text-align:left;">
MGI_C57BL6J_104583
</td>
<td style="text-align:left;">
Eps15
</td>
<td style="text-align:left;">
NCBI_Gene:13858,ENSEMBL:ENSMUSG00000028552
</td>
<td style="text-align:left;">
MGI:104583
</td>
<td style="text-align:left;">
protein coding gene
</td>
<td style="text-align:left;">
epidermal growth factor receptor pathway substrate 15
</td>
</tr>
<tr>
<td style="text-align:left;">
4
</td>
<td style="text-align:left;">
gene
</td>
<td style="text-align:right;">
109.3554
</td>
<td style="text-align:right;">
109.3600
</td>
<td style="text-align:left;">
negative
</td>
<td style="text-align:left;">
MGI_C57BL6J_5623109
</td>
<td style="text-align:left;">
Gm40224
</td>
<td style="text-align:left;">
NCBI_Gene:105244652
</td>
<td style="text-align:left;">
MGI:5623109
</td>
<td style="text-align:left;">
lncRNA gene
</td>
<td style="text-align:left;">
predicted gene, 40224
</td>
</tr>
<tr>
<td style="text-align:left;">
4
</td>
<td style="text-align:left;">
pseudogene
</td>
<td style="text-align:right;">
109.3737
</td>
<td style="text-align:right;">
109.3749
</td>
<td style="text-align:left;">
negative
</td>
<td style="text-align:left;">
MGI_C57BL6J_3652196
</td>
<td style="text-align:left;">
Gm12749
</td>
<td style="text-align:left;">
NCBI_Gene:666144,ENSEMBL:ENSMUSG00000083512
</td>
<td style="text-align:left;">
MGI:3652196
</td>
<td style="text-align:left;">
pseudogene
</td>
<td style="text-align:left;">
predicted gene 12749
</td>
</tr>
<tr>
<td style="text-align:left;">
4
</td>
<td style="text-align:left;">
gene
</td>
<td style="text-align:right;">
109.3941
</td>
<td style="text-align:right;">
109.4447
</td>
<td style="text-align:left;">
positive
</td>
<td style="text-align:left;">
MGI_C57BL6J_2444350
</td>
<td style="text-align:left;">
Ttc39a
</td>
<td style="text-align:left;">
NCBI_Gene:230603,ENSEMBL:ENSMUSG00000028555
</td>
<td style="text-align:left;">
MGI:2444350
</td>
<td style="text-align:left;">
protein coding gene
</td>
<td style="text-align:left;">
tetratricopeptide repeat domain 39A
</td>
</tr>
<tr>
<td style="text-align:left;">
4
</td>
<td style="text-align:left;">
gene
</td>
<td style="text-align:right;">
109.4023
</td>
<td style="text-align:right;">
109.4063
</td>
<td style="text-align:left;">
negative
</td>
<td style="text-align:left;">
MGI_C57BL6J_3651956
</td>
<td style="text-align:left;">
Ttc39aos1
</td>
<td style="text-align:left;">
NCBI_Gene:102635290,ENSEMBL:ENSMUSG00000085873
</td>
<td style="text-align:left;">
MGI:3651956
</td>
<td style="text-align:left;">
antisense lncRNA gene
</td>
<td style="text-align:left;">
Ttc39a opposite strand RNA 1
</td>
</tr>
<tr>
<td style="text-align:left;">
4
</td>
<td style="text-align:left;">
gene
</td>
<td style="text-align:right;">
109.4511
</td>
<td style="text-align:right;">
109.4847
</td>
<td style="text-align:left;">
negative
</td>
<td style="text-align:left;">
MGI_C57BL6J_1352759
</td>
<td style="text-align:left;">
Rnf11
</td>
<td style="text-align:left;">
NCBI_Gene:29864,ENSEMBL:ENSMUSG00000028557
</td>
<td style="text-align:left;">
MGI:1352759
</td>
<td style="text-align:left;">
protein coding gene
</td>
<td style="text-align:left;">
ring finger protein 11
</td>
</tr>
<tr>
<td style="text-align:left;">
4
</td>
<td style="text-align:left;">
gene
</td>
<td style="text-align:right;">
109.5053
</td>
<td style="text-align:right;">
109.5313
</td>
<td style="text-align:left;">
negative
</td>
<td style="text-align:left;">
MGI_C57BL6J_1914896
</td>
<td style="text-align:left;">
4930522H14Rik
</td>
<td style="text-align:left;">
NCBI_Gene:67646,ENSEMBL:ENSMUSG00000060491
</td>
<td style="text-align:left;">
MGI:1914896
</td>
<td style="text-align:left;">
protein coding gene
</td>
<td style="text-align:left;">
RIKEN cDNA 4930522H14 gene
</td>
</tr>
<tr>
<td style="text-align:left;">
4
</td>
<td style="text-align:left;">
gene
</td>
<td style="text-align:right;">
109.5370
</td>
<td style="text-align:right;">
109.5372
</td>
<td style="text-align:left;">
positive
</td>
<td style="text-align:left;">
MGI_C57BL6J_5452274
</td>
<td style="text-align:left;">
Gm22497
</td>
<td style="text-align:left;">
ENSEMBL:ENSMUSG00000077687
</td>
<td style="text-align:left;">
MGI:5452274
</td>
<td style="text-align:left;">
snoRNA gene
</td>
<td style="text-align:left;">
predicted gene, 22497
</td>
</tr>
<tr>
<td style="text-align:left;">
4
</td>
<td style="text-align:left;">
gene
</td>
<td style="text-align:right;">
109.5510
</td>
<td style="text-align:right;">
109.5540
</td>
<td style="text-align:left;">
positive
</td>
<td style="text-align:left;">
MGI_C57BL6J_5623111
</td>
<td style="text-align:left;">
Gm40226
</td>
<td style="text-align:left;">
NCBI_Gene:105244654
</td>
<td style="text-align:left;">
MGI:5623111
</td>
<td style="text-align:left;">
lncRNA gene
</td>
<td style="text-align:left;">
predicted gene, 40226
</td>
</tr>
<tr>
<td style="text-align:left;">
4
</td>
<td style="text-align:left;">
gene
</td>
<td style="text-align:right;">
109.5692
</td>
<td style="text-align:right;">
109.6568
</td>
<td style="text-align:left;">
positive
</td>
<td style="text-align:left;">
MGI_C57BL6J_3650795
</td>
<td style="text-align:left;">
Gm12811
</td>
<td style="text-align:left;">
ENSEMBL:ENSMUSG00000087641
</td>
<td style="text-align:left;">
MGI:3650795
</td>
<td style="text-align:left;">
lncRNA gene
</td>
<td style="text-align:left;">
predicted gene 12811
</td>
</tr>
<tr>
<td style="text-align:left;">
4
</td>
<td style="text-align:left;">
pseudogene
</td>
<td style="text-align:right;">
109.5994
</td>
<td style="text-align:right;">
109.6000
</td>
<td style="text-align:left;">
negative
</td>
<td style="text-align:left;">
MGI_C57BL6J_5011297
</td>
<td style="text-align:left;">
Gm19112
</td>
<td style="text-align:left;">
NCBI_Gene:100418274
</td>
<td style="text-align:left;">
MGI:5011297
</td>
<td style="text-align:left;">
pseudogene
</td>
<td style="text-align:left;">
predicted gene, 19112
</td>
</tr>
<tr>
<td style="text-align:left;">
4
</td>
<td style="text-align:left;">
pseudogene
</td>
<td style="text-align:right;">
109.6296
</td>
<td style="text-align:right;">
109.6309
</td>
<td style="text-align:left;">
positive
</td>
<td style="text-align:left;">
MGI_C57BL6J_3651634
</td>
<td style="text-align:left;">
Gm12803
</td>
<td style="text-align:left;">
NCBI_Gene:100418450,ENSEMBL:ENSMUSG00000082771
</td>
<td style="text-align:left;">
MGI:3651634
</td>
<td style="text-align:left;">
pseudogene
</td>
<td style="text-align:left;">
predicted gene 12803
</td>
</tr>
<tr>
<td style="text-align:left;">
4
</td>
<td style="text-align:left;">
gene
</td>
<td style="text-align:right;">
109.6309
</td>
<td style="text-align:right;">
109.6491
</td>
<td style="text-align:left;">
negative
</td>
<td style="text-align:left;">
MGI_C57BL6J_5623112
</td>
<td style="text-align:left;">
Gm40227
</td>
<td style="text-align:left;">
NCBI_Gene:105244655
</td>
<td style="text-align:left;">
MGI:5623112
</td>
<td style="text-align:left;">
lncRNA gene
</td>
<td style="text-align:left;">
predicted gene, 40227
</td>
</tr>
<tr>
<td style="text-align:left;">
4
</td>
<td style="text-align:left;">
gene
</td>
<td style="text-align:right;">
109.6515
</td>
<td style="text-align:right;">
109.6579
</td>
<td style="text-align:left;">
negative
</td>
<td style="text-align:left;">
MGI_C57BL6J_2442649
</td>
<td style="text-align:left;">
9630013D21Rik
</td>
<td style="text-align:left;">
NCBI_Gene:319743,ENSEMBL:ENSMUSG00000059027
</td>
<td style="text-align:left;">
MGI:2442649
</td>
<td style="text-align:left;">
lncRNA gene
</td>
<td style="text-align:left;">
RIKEN cDNA 9630013D21 gene
</td>
</tr>
<tr>
<td style="text-align:left;">
4
</td>
<td style="text-align:left;">
gene
</td>
<td style="text-align:right;">
109.6609
</td>
<td style="text-align:right;">
109.6672
</td>
<td style="text-align:left;">
negative
</td>
<td style="text-align:left;">
MGI_C57BL6J_105388
</td>
<td style="text-align:left;">
Cdkn2c
</td>
<td style="text-align:left;">
NCBI_Gene:12580,ENSEMBL:ENSMUSG00000028551
</td>
<td style="text-align:left;">
MGI:105388
</td>
<td style="text-align:left;">
protein coding gene
</td>
<td style="text-align:left;">
cyclin dependent kinase inhibitor 2C
</td>
</tr>
<tr>
<td style="text-align:left;">
4
</td>
<td style="text-align:left;">
gene
</td>
<td style="text-align:right;">
109.6669
</td>
<td style="text-align:right;">
109.6682
</td>
<td style="text-align:left;">
negative
</td>
<td style="text-align:left;">
MGI_C57BL6J_3826683
</td>
<td style="text-align:left;">
Gm2703
</td>
<td style="text-align:left;">
NA
</td>
<td style="text-align:left;">
NA
</td>
<td style="text-align:left;">
protein coding gene
</td>
<td style="text-align:left;">
predicted gene 2703
</td>
</tr>
<tr>
<td style="text-align:left;">
4
</td>
<td style="text-align:left;">
gene
</td>
<td style="text-align:right;">
109.6766
</td>
<td style="text-align:right;">
109.9640
</td>
<td style="text-align:left;">
positive
</td>
<td style="text-align:left;">
MGI_C57BL6J_109419
</td>
<td style="text-align:left;">
Faf1
</td>
<td style="text-align:left;">
NCBI_Gene:14084,ENSEMBL:ENSMUSG00000010517
</td>
<td style="text-align:left;">
MGI:109419
</td>
<td style="text-align:left;">
protein coding gene
</td>
<td style="text-align:left;">
Fas-associated factor 1
</td>
</tr>
<tr>
<td style="text-align:left;">
4
</td>
<td style="text-align:left;">
pseudogene
</td>
<td style="text-align:right;">
109.7544
</td>
<td style="text-align:right;">
109.7548
</td>
<td style="text-align:left;">
negative
</td>
<td style="text-align:left;">
MGI_C57BL6J_3650597
</td>
<td style="text-align:left;">
Gm12808
</td>
<td style="text-align:left;">
ENSEMBL:ENSMUSG00000083995
</td>
<td style="text-align:left;">
MGI:3650597
</td>
<td style="text-align:left;">
pseudogene
</td>
<td style="text-align:left;">
predicted gene 12808
</td>
</tr>
<tr>
<td style="text-align:left;">
4
</td>
<td style="text-align:left;">
gene
</td>
<td style="text-align:right;">
109.9732
</td>
<td style="text-align:right;">
109.9778
</td>
<td style="text-align:left;">
negative
</td>
<td style="text-align:left;">
MGI_C57BL6J_1924703
</td>
<td style="text-align:left;">
Dmrta2os
</td>
<td style="text-align:left;">
NCBI_Gene:77453,ENSEMBL:ENSMUSG00000085256
</td>
<td style="text-align:left;">
MGI:1924703
</td>
<td style="text-align:left;">
antisense lncRNA gene
</td>
<td style="text-align:left;">
doublesex and mab-3 related transcription factor like family A2,
opposite strand
</td>
</tr>
<tr>
<td style="text-align:left;">
4
</td>
<td style="text-align:left;">
gene
</td>
<td style="text-align:right;">
109.9778
</td>
<td style="text-align:right;">
109.9837
</td>
<td style="text-align:left;">
positive
</td>
<td style="text-align:left;">
MGI_C57BL6J_2653629
</td>
<td style="text-align:left;">
Dmrta2
</td>
<td style="text-align:left;">
NCBI_Gene:242620,ENSEMBL:ENSMUSG00000047143
</td>
<td style="text-align:left;">
MGI:2653629
</td>
<td style="text-align:left;">
protein coding gene
</td>
<td style="text-align:left;">
doublesex and mab-3 related transcription factor like family A2
</td>
</tr>
<tr>
<td style="text-align:left;">
4
</td>
<td style="text-align:left;">
gene
</td>
<td style="text-align:right;">
110.1527
</td>
<td style="text-align:right;">
110.1567
</td>
<td style="text-align:left;">
positive
</td>
<td style="text-align:left;">
MGI_C57BL6J_5592229
</td>
<td style="text-align:left;">
Gm33070
</td>
<td style="text-align:left;">
NCBI_Gene:102635827
</td>
<td style="text-align:left;">
MGI:5592229
</td>
<td style="text-align:left;">
lncRNA gene
</td>
<td style="text-align:left;">
predicted gene, 33070
</td>
</tr>
<tr>
<td style="text-align:left;">
4
</td>
<td style="text-align:left;">
gene
</td>
<td style="text-align:right;">
110.2037
</td>
<td style="text-align:right;">
110.3519
</td>
<td style="text-align:left;">
negative
</td>
<td style="text-align:left;">
MGI_C57BL6J_107427
</td>
<td style="text-align:left;">
Elavl4
</td>
<td style="text-align:left;">
NCBI_Gene:15572,ENSEMBL:ENSMUSG00000028546
</td>
<td style="text-align:left;">
MGI:107427
</td>
<td style="text-align:left;">
protein coding gene
</td>
<td style="text-align:left;">
ELAV like RNA binding protein 4
</td>
</tr>
<tr>
<td style="text-align:left;">
4
</td>
<td style="text-align:left;">
gene
</td>
<td style="text-align:right;">
110.3976
</td>
<td style="text-align:right;">
111.6643
</td>
<td style="text-align:left;">
positive
</td>
<td style="text-align:left;">
MGI_C57BL6J_1918244
</td>
<td style="text-align:left;">
Agbl4
</td>
<td style="text-align:left;">
NCBI_Gene:78933,ENSEMBL:ENSMUSG00000061298
</td>
<td style="text-align:left;">
MGI:1918244
</td>
<td style="text-align:left;">
protein coding gene
</td>
<td style="text-align:left;">
ATP/GTP binding protein-like 4
</td>
</tr>
<tr>
<td style="text-align:left;">
4
</td>
<td style="text-align:left;">
gene
</td>
<td style="text-align:right;">
110.4348
</td>
<td style="text-align:right;">
110.4360
</td>
<td style="text-align:left;">
positive
</td>
<td style="text-align:left;">
MGI_C57BL6J_1924569
</td>
<td style="text-align:left;">
C030007D22Rik
</td>
<td style="text-align:left;">
NA
</td>
<td style="text-align:left;">
NA
</td>
<td style="text-align:left;">
unclassified gene
</td>
<td style="text-align:left;">
RIKEN cDNA C030007D22 gene
</td>
</tr>
<tr>
<td style="text-align:left;">
4
</td>
<td style="text-align:left;">
pseudogene
</td>
<td style="text-align:right;">
110.5100
</td>
<td style="text-align:right;">
110.5103
</td>
<td style="text-align:left;">
positive
</td>
<td style="text-align:left;">
MGI_C57BL6J_3651080
</td>
<td style="text-align:left;">
Gm12806
</td>
<td style="text-align:left;">
ENSEMBL:ENSMUSG00000081813
</td>
<td style="text-align:left;">
MGI:3651080
</td>
<td style="text-align:left;">
pseudogene
</td>
<td style="text-align:left;">
predicted gene 12806
</td>
</tr>
<tr>
<td style="text-align:left;">
4
</td>
<td style="text-align:left;">
pseudogene
</td>
<td style="text-align:right;">
110.5128
</td>
<td style="text-align:right;">
110.5143
</td>
<td style="text-align:left;">
negative
</td>
<td style="text-align:left;">
MGI_C57BL6J_3650599
</td>
<td style="text-align:left;">
Gm12807
</td>
<td style="text-align:left;">
NCBI_Gene:102635892,ENSEMBL:ENSMUSG00000082348
</td>
<td style="text-align:left;">
MGI:3650599
</td>
<td style="text-align:left;">
pseudogene
</td>
<td style="text-align:left;">
predicted gene 12807
</td>
</tr>
<tr>
<td style="text-align:left;">
4
</td>
<td style="text-align:left;">
pseudogene
</td>
<td style="text-align:right;">
111.2901
</td>
<td style="text-align:right;">
111.2911
</td>
<td style="text-align:left;">
positive
</td>
<td style="text-align:left;">
MGI_C57BL6J_3651639
</td>
<td style="text-align:left;">
Gm12804
</td>
<td style="text-align:left;">
NCBI_Gene:100417659,ENSEMBL:ENSMUSG00000081068
</td>
<td style="text-align:left;">
MGI:3651639
</td>
<td style="text-align:left;">
pseudogene
</td>
<td style="text-align:left;">
predicted gene 12804
</td>
</tr>
<tr>
<td style="text-align:left;">
4
</td>
<td style="text-align:left;">
pseudogene
</td>
<td style="text-align:right;">
111.3195
</td>
<td style="text-align:right;">
111.3206
</td>
<td style="text-align:left;">
negative
</td>
<td style="text-align:left;">
MGI_C57BL6J_3650124
</td>
<td style="text-align:left;">
Gm12805
</td>
<td style="text-align:left;">
NCBI_Gene:100417245,ENSEMBL:ENSMUSG00000083904
</td>
<td style="text-align:left;">
MGI:3650124
</td>
<td style="text-align:left;">
pseudogene
</td>
<td style="text-align:left;">
predicted gene 12805
</td>
</tr>
<tr>
<td style="text-align:left;">
4
</td>
<td style="text-align:left;">
gene
</td>
<td style="text-align:right;">
111.4138
</td>
<td style="text-align:right;">
111.4603
</td>
<td style="text-align:left;">
positive
</td>
<td style="text-align:left;">
MGI_C57BL6J_1914871
</td>
<td style="text-align:left;">
Bend5
</td>
<td style="text-align:left;">
NCBI_Gene:67621,ENSEMBL:ENSMUSG00000028545
</td>
<td style="text-align:left;">
MGI:1914871
</td>
<td style="text-align:left;">
protein coding gene
</td>
<td style="text-align:left;">
BEN domain containing 5
</td>
</tr>
<tr>
<td style="text-align:left;">
4
</td>
<td style="text-align:left;">
gene
</td>
<td style="text-align:right;">
111.4278
</td>
<td style="text-align:right;">
111.4279
</td>
<td style="text-align:left;">
negative
</td>
<td style="text-align:left;">
MGI_C57BL6J_4422056
</td>
<td style="text-align:left;">
n-R5s191
</td>
<td style="text-align:left;">
ENSEMBL:ENSMUSG00000084673
</td>
<td style="text-align:left;">
MGI:4422056
</td>
<td style="text-align:left;">
rRNA gene
</td>
<td style="text-align:left;">
nuclear encoded rRNA 5S 191
</td>
</tr>
<tr>
<td style="text-align:left;">
4
</td>
<td style="text-align:left;">
gene
</td>
<td style="text-align:right;">
111.6956
</td>
<td style="text-align:right;">
111.7240
</td>
<td style="text-align:left;">
negative
</td>
<td style="text-align:left;">
MGI_C57BL6J_5592351
</td>
<td style="text-align:left;">
Gm33192
</td>
<td style="text-align:left;">
NCBI_Gene:102635996
</td>
<td style="text-align:left;">
MGI:5592351
</td>
<td style="text-align:left;">
lncRNA gene
</td>
<td style="text-align:left;">
predicted gene, 33192
</td>
</tr>
<tr>
<td style="text-align:left;">
4
</td>
<td style="text-align:left;">
gene
</td>
<td style="text-align:right;">
111.7199
</td>
<td style="text-align:right;">
111.8292
</td>
<td style="text-align:left;">
positive
</td>
<td style="text-align:left;">
MGI_C57BL6J_1915196
</td>
<td style="text-align:left;">
Spata6
</td>
<td style="text-align:left;">
NCBI_Gene:67946,ENSEMBL:ENSMUSG00000034401
</td>
<td style="text-align:left;">
MGI:1915196
</td>
<td style="text-align:left;">
protein coding gene
</td>
<td style="text-align:left;">
spermatogenesis associated 6
</td>
</tr>
<tr>
<td style="text-align:left;">
4
</td>
<td style="text-align:left;">
pseudogene
</td>
<td style="text-align:right;">
111.7906
</td>
<td style="text-align:right;">
111.7917
</td>
<td style="text-align:left;">
negative
</td>
<td style="text-align:left;">
MGI_C57BL6J_3650787
</td>
<td style="text-align:left;">
Gm12961
</td>
<td style="text-align:left;">
NCBI_Gene:100417901,ENSEMBL:ENSMUSG00000082686
</td>
<td style="text-align:left;">
MGI:3650787
</td>
<td style="text-align:left;">
pseudogene
</td>
<td style="text-align:left;">
predicted gene 12961
</td>
</tr>
<tr>
<td style="text-align:left;">
4
</td>
<td style="text-align:left;">
pseudogene
</td>
<td style="text-align:right;">
111.8070
</td>
<td style="text-align:right;">
111.8085
</td>
<td style="text-align:left;">
negative
</td>
<td style="text-align:left;">
MGI_C57BL6J_3651525
</td>
<td style="text-align:left;">
Gm12960
</td>
<td style="text-align:left;">
NCBI_Gene:105244657,ENSEMBL:ENSMUSG00000100514
</td>
<td style="text-align:left;">
MGI:3651525
</td>
<td style="text-align:left;">
pseudogene
</td>
<td style="text-align:left;">
predicted gene 12960
</td>
</tr>
<tr>
<td style="text-align:left;">
4
</td>
<td style="text-align:left;">
gene
</td>
<td style="text-align:right;">
111.8754
</td>
<td style="text-align:right;">
111.9029
</td>
<td style="text-align:left;">
negative
</td>
<td style="text-align:left;">
MGI_C57BL6J_2140201
</td>
<td style="text-align:left;">
Slc5a9
</td>
<td style="text-align:left;">
NCBI_Gene:230612,ENSEMBL:ENSMUSG00000028544
</td>
<td style="text-align:left;">
MGI:2140201
</td>
<td style="text-align:left;">
protein coding gene
</td>
<td style="text-align:left;">
solute carrier family 5 (sodium/glucose cotransporter), member 9
</td>
</tr>
<tr>
<td style="text-align:left;">
4
</td>
<td style="text-align:left;">
gene
</td>
<td style="text-align:right;">
111.9194
</td>
<td style="text-align:right;">
111.9504
</td>
<td style="text-align:left;">
positive
</td>
<td style="text-align:left;">
MGI_C57BL6J_3651523
</td>
<td style="text-align:left;">
Skint8
</td>
<td style="text-align:left;">
NCBI_Gene:639774,ENSEMBL:ENSMUSG00000078599
</td>
<td style="text-align:left;">
MGI:3651523
</td>
<td style="text-align:left;">
protein coding gene
</td>
<td style="text-align:left;">
selection and upkeep of intraepithelial T cells 8
</td>
</tr>
<tr>
<td style="text-align:left;">
4
</td>
<td style="text-align:left;">
gene
</td>
<td style="text-align:right;">
111.9729
</td>
<td style="text-align:right;">
111.9883
</td>
<td style="text-align:left;">
positive
</td>
<td style="text-align:left;">
MGI_C57BL6J_3041190
</td>
<td style="text-align:left;">
Skint7
</td>
<td style="text-align:left;">
NCBI_Gene:328505,ENSEMBL:ENSMUSG00000049214
</td>
<td style="text-align:left;">
MGI:3041190
</td>
<td style="text-align:left;">
protein coding gene
</td>
<td style="text-align:left;">
selection and upkeep of intraepithelial T cells 7
</td>
</tr>
<tr>
<td style="text-align:left;">
4
</td>
<td style="text-align:left;">
gene
</td>
<td style="text-align:right;">
111.9887
</td>
<td style="text-align:right;">
111.9890
</td>
<td style="text-align:left;">
negative
</td>
<td style="text-align:left;">
MGI_C57BL6J_5455621
</td>
<td style="text-align:left;">
Gm25844
</td>
<td style="text-align:left;">
ENSEMBL:ENSMUSG00000084536
</td>
<td style="text-align:left;">
MGI:5455621
</td>
<td style="text-align:left;">
unclassified non-coding RNA gene
</td>
<td style="text-align:left;">
predicted gene, 25844
</td>
</tr>
<tr>
<td style="text-align:left;">
4
</td>
<td style="text-align:left;">
gene
</td>
<td style="text-align:right;">
112.0063
</td>
<td style="text-align:right;">
112.0295
</td>
<td style="text-align:left;">
positive
</td>
<td style="text-align:left;">
MGI_C57BL6J_3649627
</td>
<td style="text-align:left;">
Skint1
</td>
<td style="text-align:left;">
NCBI_Gene:639781,ENSEMBL:ENSMUSG00000089773
</td>
<td style="text-align:left;">
MGI:3649627
</td>
<td style="text-align:left;">
protein coding gene
</td>
<td style="text-align:left;">
selection and upkeep of intraepithelial T cells 1
</td>
</tr>
<tr>
<td style="text-align:left;">
4
</td>
<td style="text-align:left;">
gene
</td>
<td style="text-align:right;">
112.0720
</td>
<td style="text-align:right;">
112.1681
</td>
<td style="text-align:left;">
positive
</td>
<td style="text-align:left;">
MGI_C57BL6J_2444425
</td>
<td style="text-align:left;">
Skint4
</td>
<td style="text-align:left;">
NCBI_Gene:320640,ENSEMBL:ENSMUSG00000055960
</td>
<td style="text-align:left;">
MGI:2444425
</td>
<td style="text-align:left;">
protein coding gene
</td>
<td style="text-align:left;">
selection and upkeep of intraepithelial T cells 4
</td>
</tr>
<tr>
<td style="text-align:left;">
4
</td>
<td style="text-align:left;">
pseudogene
</td>
<td style="text-align:right;">
112.1210
</td>
<td style="text-align:right;">
112.1213
</td>
<td style="text-align:left;">
negative
</td>
<td style="text-align:left;">
MGI_C57BL6J_3652018
</td>
<td style="text-align:left;">
Gm12820
</td>
<td style="text-align:left;">
ENSEMBL:ENSMUSG00000084326
</td>
<td style="text-align:left;">
MGI:3652018
</td>
<td style="text-align:left;">
pseudogene
</td>
<td style="text-align:left;">
predicted gene 12820
</td>
</tr>
<tr>
<td style="text-align:left;">
4
</td>
<td style="text-align:left;">
pseudogene
</td>
<td style="text-align:right;">
112.1461
</td>
<td style="text-align:right;">
112.1462
</td>
<td style="text-align:left;">
negative
</td>
<td style="text-align:left;">
MGI_C57BL6J_3652016
</td>
<td style="text-align:left;">
Gm12815
</td>
<td style="text-align:left;">
ENSEMBL:ENSMUSG00000082638
</td>
<td style="text-align:left;">
MGI:3652016
</td>
<td style="text-align:left;">
pseudogene
</td>
<td style="text-align:left;">
predicted gene 12815
</td>
</tr>
<tr>
<td style="text-align:left;">
4
</td>
<td style="text-align:left;">
gene
</td>
<td style="text-align:right;">
112.2322
</td>
<td style="text-align:right;">
112.3005
</td>
<td style="text-align:left;">
positive
</td>
<td style="text-align:left;">
MGI_C57BL6J_3045331
</td>
<td style="text-align:left;">
Skint3
</td>
<td style="text-align:left;">
NCBI_Gene:195564,ENSEMBL:ENSMUSG00000070868
</td>
<td style="text-align:left;">
MGI:3045331
</td>
<td style="text-align:left;">
protein coding gene
</td>
<td style="text-align:left;">
selection and upkeep of intraepithelial T cells 3
</td>
</tr>
<tr>
<td style="text-align:left;">
4
</td>
<td style="text-align:left;">
gene
</td>
<td style="text-align:right;">
112.3860
</td>
<td style="text-align:right;">
112.4340
</td>
<td style="text-align:left;">
negative
</td>
<td style="text-align:left;">
MGI_C57BL6J_3045341
</td>
<td style="text-align:left;">
Skint9
</td>
<td style="text-align:left;">
NCBI_Gene:329918,ENSEMBL:ENSMUSG00000049972
</td>
<td style="text-align:left;">
MGI:3045341
</td>
<td style="text-align:left;">
protein coding gene
</td>
<td style="text-align:left;">
selection and upkeep of intraepithelial T cells 9
</td>
</tr>
<tr>
<td style="text-align:left;">
4
</td>
<td style="text-align:left;">
pseudogene
</td>
<td style="text-align:right;">
112.4738
</td>
<td style="text-align:right;">
112.5215
</td>
<td style="text-align:left;">
positive
</td>
<td style="text-align:left;">
MGI_C57BL6J_3652017
</td>
<td style="text-align:left;">
Gm12819
</td>
<td style="text-align:left;">
NCBI_Gene:100504555,ENSEMBL:ENSMUSG00000081166
</td>
<td style="text-align:left;">
MGI:3652017
</td>
<td style="text-align:left;">
pseudogene
</td>
<td style="text-align:left;">
predicted gene 12819
</td>
</tr>
<tr>
<td style="text-align:left;">
4
</td>
<td style="text-align:left;">
pseudogene
</td>
<td style="text-align:right;">
112.5371
</td>
<td style="text-align:right;">
112.5377
</td>
<td style="text-align:left;">
positive
</td>
<td style="text-align:left;">
MGI_C57BL6J_3649625
</td>
<td style="text-align:left;">
Gm12821
</td>
<td style="text-align:left;">
NCBI_Gene:102636060,ENSEMBL:ENSMUSG00000083599
</td>
<td style="text-align:left;">
MGI:3649625
</td>
<td style="text-align:left;">
pseudogene
</td>
<td style="text-align:left;">
predicted gene 12821
</td>
</tr>
<tr>
<td style="text-align:left;">
4
</td>
<td style="text-align:left;">
gene
</td>
<td style="text-align:right;">
112.5572
</td>
<td style="text-align:right;">
112.6522
</td>
<td style="text-align:left;">
positive
</td>
<td style="text-align:left;">
MGI_C57BL6J_3649629
</td>
<td style="text-align:left;">
Skint2
</td>
<td style="text-align:left;">
NCBI_Gene:329919,ENSEMBL:ENSMUSG00000034359
</td>
<td style="text-align:left;">
MGI:3649629
</td>
<td style="text-align:left;">
protein coding gene
</td>
<td style="text-align:left;">
selection and upkeep of intraepithelial T cells 2
</td>
</tr>
<tr>
<td style="text-align:left;">
4
</td>
<td style="text-align:left;">
pseudogene
</td>
<td style="text-align:right;">
112.5597
</td>
<td style="text-align:right;">
112.5933
</td>
<td style="text-align:left;">
positive
</td>
<td style="text-align:left;">
MGI_C57BL6J_3651208
</td>
<td style="text-align:left;">
Gm12814
</td>
<td style="text-align:left;">
NCBI_Gene:630887,ENSEMBL:ENSMUSG00000083753
</td>
<td style="text-align:left;">
MGI:3651208
</td>
<td style="text-align:left;">
pseudogene
</td>
<td style="text-align:left;">
predicted gene 12814
</td>
</tr>
<tr>
<td style="text-align:left;">
4
</td>
<td style="text-align:left;">
gene
</td>
<td style="text-align:right;">
112.5908
</td>
<td style="text-align:right;">
112.5909
</td>
<td style="text-align:left;">
negative
</td>
<td style="text-align:left;">
MGI_C57BL6J_5453876
</td>
<td style="text-align:left;">
Gm24099
</td>
<td style="text-align:left;">
ENSEMBL:ENSMUSG00000096564
</td>
<td style="text-align:left;">
MGI:5453876
</td>
<td style="text-align:left;">
snRNA gene
</td>
<td style="text-align:left;">
predicted gene, 24099
</td>
</tr>
<tr>
<td style="text-align:left;">
4
</td>
<td style="text-align:left;">
pseudogene
</td>
<td style="text-align:right;">
112.7051
</td>
<td style="text-align:right;">
112.7062
</td>
<td style="text-align:left;">
positive
</td>
<td style="text-align:left;">
MGI_C57BL6J_3649633
</td>
<td style="text-align:left;">
Gm12817
</td>
<td style="text-align:left;">
NCBI_Gene:100040581,ENSEMBL:ENSMUSG00000081066
</td>
<td style="text-align:left;">
MGI:3649633
</td>
<td style="text-align:left;">
pseudogene
</td>
<td style="text-align:left;">
predicted gene 12817
</td>
</tr>
<tr>
<td style="text-align:left;">
4
</td>
<td style="text-align:left;">
gene
</td>
<td style="text-align:right;">
112.7108
</td>
<td style="text-align:right;">
112.7749
</td>
<td style="text-align:left;">
negative
</td>
<td style="text-align:left;">
MGI_C57BL6J_2685416
</td>
<td style="text-align:left;">
Skint10
</td>
<td style="text-align:left;">
NCBI_Gene:230613,ENSEMBL:ENSMUSG00000048766
</td>
<td style="text-align:left;">
MGI:2685416
</td>
<td style="text-align:left;">
protein coding gene
</td>
<td style="text-align:left;">
selection and upkeep of intraepithelial T cells 10
</td>
</tr>
<tr>
<td style="text-align:left;">
4
</td>
<td style="text-align:left;">
pseudogene
</td>
<td style="text-align:right;">
112.7721
</td>
<td style="text-align:right;">
112.7733
</td>
<td style="text-align:left;">
positive
</td>
<td style="text-align:left;">
MGI_C57BL6J_5010585
</td>
<td style="text-align:left;">
Gm18400
</td>
<td style="text-align:left;">
NCBI_Gene:100417108
</td>
<td style="text-align:left;">
MGI:5010585
</td>
<td style="text-align:left;">
pseudogene
</td>
<td style="text-align:left;">
predicted gene, 18400
</td>
</tr>
<tr>
<td style="text-align:left;">
4
</td>
<td style="text-align:left;">
gene
</td>
<td style="text-align:right;">
112.8045
</td>
<td style="text-align:right;">
113.2870
</td>
<td style="text-align:left;">
negative
</td>
<td style="text-align:left;">
MGI_C57BL6J_3649262
</td>
<td style="text-align:left;">
Skint6
</td>
<td style="text-align:left;">
NCBI_Gene:230622,ENSEMBL:ENSMUSG00000087194
</td>
<td style="text-align:left;">
MGI:3649262
</td>
<td style="text-align:left;">
protein coding gene
</td>
<td style="text-align:left;">
selection and upkeep of intraepithelial T cells 6
</td>
</tr>
<tr>
<td style="text-align:left;">
4
</td>
<td style="text-align:left;">
gene
</td>
<td style="text-align:right;">
112.9237
</td>
<td style="text-align:right;">
112.9453
</td>
<td style="text-align:left;">
positive
</td>
<td style="text-align:left;">
MGI_C57BL6J_3651665
</td>
<td style="text-align:left;">
Gm12823
</td>
<td style="text-align:left;">
ENSEMBL:ENSMUSG00000085878
</td>
<td style="text-align:left;">
MGI:3651665
</td>
<td style="text-align:left;">
lncRNA gene
</td>
<td style="text-align:left;">
predicted gene 12823
</td>
</tr>
<tr>
<td style="text-align:left;">
4
</td>
<td style="text-align:left;">
pseudogene
</td>
<td style="text-align:right;">
112.9550
</td>
<td style="text-align:right;">
112.9563
</td>
<td style="text-align:left;">
negative
</td>
<td style="text-align:left;">
MGI_C57BL6J_3649628
</td>
<td style="text-align:left;">
Gm12816
</td>
<td style="text-align:left;">
NCBI_Gene:433745,ENSEMBL:ENSMUSG00000083011
</td>
<td style="text-align:left;">
MGI:3649628
</td>
<td style="text-align:left;">
pseudogene
</td>
<td style="text-align:left;">
predicted gene 12816
</td>
</tr>
<tr>
<td style="text-align:left;">
4
</td>
<td style="text-align:left;">
pseudogene
</td>
<td style="text-align:right;">
112.9785
</td>
<td style="text-align:right;">
112.9798
</td>
<td style="text-align:left;">
negative
</td>
<td style="text-align:left;">
MGI_C57BL6J_3649634
</td>
<td style="text-align:left;">
Gm12818
</td>
<td style="text-align:left;">
NCBI_Gene:100534383,ENSEMBL:ENSMUSG00000082678
</td>
<td style="text-align:left;">
MGI:3649634
</td>
<td style="text-align:left;">
pseudogene
</td>
<td style="text-align:left;">
predicted gene 12818
</td>
</tr>
<tr>
<td style="text-align:left;">
4
</td>
<td style="text-align:left;">
pseudogene
</td>
<td style="text-align:right;">
113.3682
</td>
<td style="text-align:right;">
113.3945
</td>
<td style="text-align:left;">
positive
</td>
<td style="text-align:left;">
MGI_C57BL6J_3650999
</td>
<td style="text-align:left;">
Gm12813
</td>
<td style="text-align:left;">
NCBI_Gene:100040607,ENSEMBL:ENSMUSG00000083384
</td>
<td style="text-align:left;">
MGI:3650999
</td>
<td style="text-align:left;">
pseudogene
</td>
<td style="text-align:left;">
predicted gene 12813
</td>
</tr>
<tr>
<td style="text-align:left;">
4
</td>
<td style="text-align:left;">
gene
</td>
<td style="text-align:right;">
113.4779
</td>
<td style="text-align:right;">
113.9995
</td>
<td style="text-align:left;">
negative
</td>
<td style="text-align:left;">
MGI_C57BL6J_3650151
</td>
<td style="text-align:left;">
Skint5
</td>
<td style="text-align:left;">
NCBI_Gene:242627,ENSEMBL:ENSMUSG00000078598
</td>
<td style="text-align:left;">
MGI:3650151
</td>
<td style="text-align:left;">
protein coding gene
</td>
<td style="text-align:left;">
selection and upkeep of intraepithelial T cells 5
</td>
</tr>
<tr>
<td style="text-align:left;">
4
</td>
<td style="text-align:left;">
pseudogene
</td>
<td style="text-align:right;">
114.0963
</td>
<td style="text-align:right;">
114.0974
</td>
<td style="text-align:left;">
negative
</td>
<td style="text-align:left;">
MGI_C57BL6J_3650153
</td>
<td style="text-align:left;">
Gm12825
</td>
<td style="text-align:left;">
NCBI_Gene:100418129,ENSEMBL:ENSMUSG00000080696
</td>
<td style="text-align:left;">
MGI:3650153
</td>
<td style="text-align:left;">
pseudogene
</td>
<td style="text-align:left;">
predicted gene 12825
</td>
</tr>
<tr>
<td style="text-align:left;">
4
</td>
<td style="text-align:left;">
gene
</td>
<td style="text-align:right;">
114.1634
</td>
<td style="text-align:right;">
114.2450
</td>
<td style="text-align:left;">
positive
</td>
<td style="text-align:left;">
MGI_C57BL6J_2685415
</td>
<td style="text-align:left;">
Skint11
</td>
<td style="text-align:left;">
NCBI_Gene:230623,ENSEMBL:ENSMUSG00000057977
</td>
<td style="text-align:left;">
MGI:2685415
</td>
<td style="text-align:left;">
protein coding gene
</td>
<td style="text-align:left;">
selection and upkeep of intraepithelial T cells 11
</td>
</tr>
<tr>
<td style="text-align:left;">
4
</td>
<td style="text-align:left;">
pseudogene
</td>
<td style="text-align:right;">
114.1651
</td>
<td style="text-align:right;">
114.1659
</td>
<td style="text-align:left;">
negative
</td>
<td style="text-align:left;">
MGI_C57BL6J_3783183
</td>
<td style="text-align:left;">
Gm15741
</td>
<td style="text-align:left;">
NCBI_Gene:100417107,ENSEMBL:ENSMUSG00000082825
</td>
<td style="text-align:left;">
MGI:3783183
</td>
<td style="text-align:left;">
pseudogene
</td>
<td style="text-align:left;">
predicted gene 15741
</td>
</tr>
<tr>
<td style="text-align:left;">
4
</td>
<td style="text-align:left;">
gene
</td>
<td style="text-align:right;">
114.3299
</td>
<td style="text-align:right;">
114.3466
</td>
<td style="text-align:left;">
negative
</td>
<td style="text-align:left;">
MGI_C57BL6J_5623113
</td>
<td style="text-align:left;">
Gm40228
</td>
<td style="text-align:left;">
NCBI_Gene:105244658
</td>
<td style="text-align:left;">
MGI:5623113
</td>
<td style="text-align:left;">
lncRNA gene
</td>
<td style="text-align:left;">
predicted gene, 40228
</td>
</tr>
<tr>
<td style="text-align:left;">
4
</td>
<td style="text-align:left;">
gene
</td>
<td style="text-align:right;">
114.4062
</td>
<td style="text-align:right;">
114.6151
</td>
<td style="text-align:left;">
positive
</td>
<td style="text-align:left;">
MGI_C57BL6J_3650152
</td>
<td style="text-align:left;">
Trabd2b
</td>
<td style="text-align:left;">
NCBI_Gene:666048,ENSEMBL:ENSMUSG00000070867
</td>
<td style="text-align:left;">
MGI:3650152
</td>
<td style="text-align:left;">
protein coding gene
</td>
<td style="text-align:left;">
TraB domain containing 2B
</td>
</tr>
<tr>
<td style="text-align:left;">
4
</td>
<td style="text-align:left;">
gene
</td>
<td style="text-align:right;">
114.4140
</td>
<td style="text-align:right;">
114.4149
</td>
<td style="text-align:left;">
negative
</td>
<td style="text-align:left;">
MGI_C57BL6J_5592869
</td>
<td style="text-align:left;">
Gm33710
</td>
<td style="text-align:left;">
NCBI_Gene:102636714
</td>
<td style="text-align:left;">
MGI:5592869
</td>
<td style="text-align:left;">
lncRNA gene
</td>
<td style="text-align:left;">
predicted gene, 33710
</td>
</tr>
<tr>
<td style="text-align:left;">
4
</td>
<td style="text-align:left;">
gene
</td>
<td style="text-align:right;">
114.5769
</td>
<td style="text-align:right;">
114.5922
</td>
<td style="text-align:left;">
negative
</td>
<td style="text-align:left;">
MGI_C57BL6J_5592814
</td>
<td style="text-align:left;">
Gm33655
</td>
<td style="text-align:left;">
NCBI_Gene:102636647
</td>
<td style="text-align:left;">
MGI:5592814
</td>
<td style="text-align:left;">
lncRNA gene
</td>
<td style="text-align:left;">
predicted gene, 33655
</td>
</tr>
<tr>
<td style="text-align:left;">
4
</td>
<td style="text-align:left;">
gene
</td>
<td style="text-align:right;">
114.6727
</td>
<td style="text-align:right;">
114.6733
</td>
<td style="text-align:left;">
negative
</td>
<td style="text-align:left;">
MGI_C57BL6J_5579570
</td>
<td style="text-align:left;">
Gm28864
</td>
<td style="text-align:left;">
ENSEMBL:ENSMUSG00000100998
</td>
<td style="text-align:left;">
MGI:5579570
</td>
<td style="text-align:left;">
lncRNA gene
</td>
<td style="text-align:left;">
predicted gene 28864
</td>
</tr>
<tr>
<td style="text-align:left;">
4
</td>
<td style="text-align:left;">
gene
</td>
<td style="text-align:right;">
114.6803
</td>
<td style="text-align:right;">
114.6876
</td>
<td style="text-align:left;">
positive
</td>
<td style="text-align:left;">
MGI_C57BL6J_3650003
</td>
<td style="text-align:left;">
Gm12829
</td>
<td style="text-align:left;">
NCBI_Gene:102636500,ENSEMBL:ENSMUSG00000085870
</td>
<td style="text-align:left;">
MGI:3650003
</td>
<td style="text-align:left;">
lncRNA gene
</td>
<td style="text-align:left;">
predicted gene 12829
</td>
</tr>
<tr>
<td style="text-align:left;">
4
</td>
<td style="text-align:left;">
gene
</td>
<td style="text-align:right;">
114.8161
</td>
<td style="text-align:right;">
114.9029
</td>
<td style="text-align:left;">
negative
</td>
<td style="text-align:left;">
MGI_C57BL6J_1918862
</td>
<td style="text-align:left;">
9130410C08Rik
</td>
<td style="text-align:left;">
NCBI_Gene:105244659,ENSEMBL:ENSMUSG00000085323
</td>
<td style="text-align:left;">
MGI:1918862
</td>
<td style="text-align:left;">
lncRNA gene
</td>
<td style="text-align:left;">
RIKEN cDNA 9130410C08 gene
</td>
</tr>
<tr>
<td style="text-align:left;">
4
</td>
<td style="text-align:left;">
gene
</td>
<td style="text-align:right;">
114.8217
</td>
<td style="text-align:right;">
114.8562
</td>
<td style="text-align:left;">
positive
</td>
<td style="text-align:left;">
MGI_C57BL6J_3649998
</td>
<td style="text-align:left;">
Gm12830
</td>
<td style="text-align:left;">
NCBI_Gene:433746,ENSEMBL:ENSMUSG00000055198
</td>
<td style="text-align:left;">
MGI:3649998
</td>
<td style="text-align:left;">
protein coding gene
</td>
<td style="text-align:left;">
predicted gene 12830
</td>
</tr>
<tr>
<td style="text-align:left;">
4
</td>
<td style="text-align:left;">
gene
</td>
<td style="text-align:right;">
114.8450
</td>
<td style="text-align:right;">
114.8451
</td>
<td style="text-align:left;">
negative
</td>
<td style="text-align:left;">
MGI_C57BL6J_5453007
</td>
<td style="text-align:left;">
Gm23230
</td>
<td style="text-align:left;">
ENSEMBL:ENSMUSG00000087923
</td>
<td style="text-align:left;">
MGI:5453007
</td>
<td style="text-align:left;">
snRNA gene
</td>
<td style="text-align:left;">
predicted gene, 23230
</td>
</tr>
<tr>
<td style="text-align:left;">
4
</td>
<td style="text-align:left;">
gene
</td>
<td style="text-align:right;">
114.9063
</td>
<td style="text-align:right;">
114.9089
</td>
<td style="text-align:left;">
negative
</td>
<td style="text-align:left;">
MGI_C57BL6J_1347471
</td>
<td style="text-align:left;">
Foxd2
</td>
<td style="text-align:left;">
NCBI_Gene:17301,ENSEMBL:ENSMUSG00000055210
</td>
<td style="text-align:left;">
MGI:1347471
</td>
<td style="text-align:left;">
protein coding gene
</td>
<td style="text-align:left;">
forkhead box D2
</td>
</tr>
<tr>
<td style="text-align:left;">
4
</td>
<td style="text-align:left;">
gene
</td>
<td style="text-align:right;">
114.9093
</td>
<td style="text-align:right;">
114.9220
</td>
<td style="text-align:left;">
positive
</td>
<td style="text-align:left;">
MGI_C57BL6J_2444065
</td>
<td style="text-align:left;">
Foxd2os
</td>
<td style="text-align:left;">
NCBI_Gene:100040736,ENSEMBL:ENSMUSG00000085399
</td>
<td style="text-align:left;">
MGI:2444065
</td>
<td style="text-align:left;">
lncRNA gene
</td>
<td style="text-align:left;">
forkhead box D2, opposite strand
</td>
</tr>
<tr>
<td style="text-align:left;">
4
</td>
<td style="text-align:left;">
gene
</td>
<td style="text-align:right;">
114.9251
</td>
<td style="text-align:right;">
114.9260
</td>
<td style="text-align:left;">
negative
</td>
<td style="text-align:left;">
MGI_C57BL6J_1353569
</td>
<td style="text-align:left;">
Foxe3
</td>
<td style="text-align:left;">
NCBI_Gene:30923,ENSEMBL:ENSMUSG00000044518
</td>
<td style="text-align:left;">
MGI:1353569
</td>
<td style="text-align:left;">
protein coding gene
</td>
<td style="text-align:left;">
forkhead box E3
</td>
</tr>
<tr>
<td style="text-align:left;">
4
</td>
<td style="text-align:left;">
gene
</td>
<td style="text-align:right;">
114.9468
</td>
<td style="text-align:right;">
114.9508
</td>
<td style="text-align:left;">
negative
</td>
<td style="text-align:left;">
MGI_C57BL6J_5623114
</td>
<td style="text-align:left;">
Gm40229
</td>
<td style="text-align:left;">
NCBI_Gene:105244660
</td>
<td style="text-align:left;">
MGI:5623114
</td>
<td style="text-align:left;">
lncRNA gene
</td>
<td style="text-align:left;">
predicted gene, 40229
</td>
</tr>
<tr>
<td style="text-align:left;">
4
</td>
<td style="text-align:left;">
gene
</td>
<td style="text-align:right;">
114.9593
</td>
<td style="text-align:right;">
114.9872
</td>
<td style="text-align:left;">
negative
</td>
<td style="text-align:left;">
MGI_C57BL6J_1913838
</td>
<td style="text-align:left;">
Cmpk1
</td>
<td style="text-align:left;">
NCBI_Gene:66588,ENSEMBL:ENSMUSG00000028719
</td>
<td style="text-align:left;">
MGI:1913838
</td>
<td style="text-align:left;">
protein coding gene
</td>
<td style="text-align:left;">
cytidine monophosphate (UMP-CMP) kinase 1
</td>
</tr>
<tr>
<td style="text-align:left;">
4
</td>
<td style="text-align:left;">
gene
</td>
<td style="text-align:right;">
114.9880
</td>
<td style="text-align:right;">
114.9881
</td>
<td style="text-align:left;">
positive
</td>
<td style="text-align:left;">
MGI_C57BL6J_5453822
</td>
<td style="text-align:left;">
Gm24045
</td>
<td style="text-align:left;">
ENSEMBL:ENSMUSG00000065047
</td>
<td style="text-align:left;">
MGI:5453822
</td>
<td style="text-align:left;">
unclassified non-coding RNA gene
</td>
<td style="text-align:left;">
predicted gene, 24045
</td>
</tr>
<tr>
<td style="text-align:left;">
4
</td>
<td style="text-align:left;">
gene
</td>
<td style="text-align:right;">
115.0001
</td>
<td style="text-align:right;">
115.0432
</td>
<td style="text-align:left;">
positive
</td>
<td style="text-align:left;">
MGI_C57BL6J_107477
</td>
<td style="text-align:left;">
Stil
</td>
<td style="text-align:left;">
NCBI_Gene:20460,ENSEMBL:ENSMUSG00000028718
</td>
<td style="text-align:left;">
MGI:107477
</td>
<td style="text-align:left;">
protein coding gene
</td>
<td style="text-align:left;">
Scl/Tal1 interrupting locus
</td>
</tr>
<tr>
<td style="text-align:left;">
4
</td>
<td style="text-align:left;">
gene
</td>
<td style="text-align:right;">
115.0564
</td>
<td style="text-align:right;">
115.0718
</td>
<td style="text-align:left;">
positive
</td>
<td style="text-align:left;">
MGI_C57BL6J_98480
</td>
<td style="text-align:left;">
Tal1
</td>
<td style="text-align:left;">
NCBI_Gene:21349,ENSEMBL:ENSMUSG00000028717
</td>
<td style="text-align:left;">
MGI:98480
</td>
<td style="text-align:left;">
protein coding gene
</td>
<td style="text-align:left;">
T cell acute lymphocytic leukemia 1
</td>
</tr>
<tr>
<td style="text-align:left;">
4
</td>
<td style="text-align:left;">
gene
</td>
<td style="text-align:right;">
115.0576
</td>
<td style="text-align:right;">
115.0582
</td>
<td style="text-align:left;">
negative
</td>
<td style="text-align:left;">
MGI_C57BL6J_5592945
</td>
<td style="text-align:left;">
Gm33786
</td>
<td style="text-align:left;">
NCBI_Gene:102636820
</td>
<td style="text-align:left;">
MGI:5592945
</td>
<td style="text-align:left;">
lncRNA gene
</td>
<td style="text-align:left;">
predicted gene, 33786
</td>
</tr>
<tr>
<td style="text-align:left;">
4
</td>
<td style="text-align:left;">
gene
</td>
<td style="text-align:right;">
115.0887
</td>
<td style="text-align:right;">
115.0939
</td>
<td style="text-align:left;">
positive
</td>
<td style="text-align:left;">
MGI_C57BL6J_1914432
</td>
<td style="text-align:left;">
Pdzk1ip1
</td>
<td style="text-align:left;">
NCBI_Gene:67182,ENSEMBL:ENSMUSG00000028716
</td>
<td style="text-align:left;">
MGI:1914432
</td>
<td style="text-align:left;">
protein coding gene
</td>
<td style="text-align:left;">
PDZK1 interacting protein 1
</td>
</tr>
<tr>
<td style="text-align:left;">
4
</td>
<td style="text-align:left;">
gene
</td>
<td style="text-align:right;">
115.1063
</td>
<td style="text-align:right;">
115.1343
</td>
<td style="text-align:left;">
negative
</td>
<td style="text-align:left;">
MGI_C57BL6J_1932403
</td>
<td style="text-align:left;">
Cyp4x1
</td>
<td style="text-align:left;">
NCBI_Gene:81906,ENSEMBL:ENSMUSG00000047155
</td>
<td style="text-align:left;">
MGI:1932403
</td>
<td style="text-align:left;">
protein coding gene
</td>
<td style="text-align:left;">
cytochrome P450, family 4, subfamily x, polypeptide 1
</td>
</tr>
<tr>
<td style="text-align:left;">
4
</td>
<td style="text-align:left;">
gene
</td>
<td style="text-align:right;">
115.1338
</td>
<td style="text-align:right;">
115.1379
</td>
<td style="text-align:left;">
positive
</td>
<td style="text-align:left;">
MGI_C57BL6J_3649427
</td>
<td style="text-align:left;">
Cyp4x1os
</td>
<td style="text-align:left;">
NCBI_Gene:105244662,ENSEMBL:ENSMUSG00000086896
</td>
<td style="text-align:left;">
MGI:3649427
</td>
<td style="text-align:left;">
antisense lncRNA gene
</td>
<td style="text-align:left;">
cytochrome P450, family 4, subfamily x, polypeptide 1, opposite strand
</td>
</tr>
<tr>
<td style="text-align:left;">
4
</td>
<td style="text-align:left;">
pseudogene
</td>
<td style="text-align:right;">
115.1386
</td>
<td style="text-align:right;">
115.1565
</td>
<td style="text-align:left;">
negative
</td>
<td style="text-align:left;">
MGI_C57BL6J_1932405
</td>
<td style="text-align:left;">
Cyp4a28-ps
</td>
<td style="text-align:left;">
NCBI_Gene:547220,ENSEMBL:ENSMUSG00000081823
</td>
<td style="text-align:left;">
MGI:1932405
</td>
<td style="text-align:left;">
pseudogene
</td>
<td style="text-align:left;">
cytochrome P450, family 4, subfamily a, polypeptide 28, pseudogene
</td>
</tr>
<tr>
<td style="text-align:left;">
4
</td>
<td style="text-align:left;">
pseudogene
</td>
<td style="text-align:right;">
115.1758
</td>
<td style="text-align:right;">
115.1768
</td>
<td style="text-align:left;">
positive
</td>
<td style="text-align:left;">
MGI_C57BL6J_3651920
</td>
<td style="text-align:left;">
Cyp4a29-ps1
</td>
<td style="text-align:left;">
ENSEMBL:ENSMUSG00000082889
</td>
<td style="text-align:left;">
MGI:3651920
</td>
<td style="text-align:left;">
pseudogene
</td>
<td style="text-align:left;">
cytochrome P450, family 4, subfamily a, polypeptide 29, pseudogene 1
</td>
</tr>
<tr>
<td style="text-align:left;">
4
</td>
<td style="text-align:left;">
pseudogene
</td>
<td style="text-align:right;">
115.2388
</td>
<td style="text-align:right;">
115.2416
</td>
<td style="text-align:left;">
negative
</td>
<td style="text-align:left;">
MGI_C57BL6J_3649428
</td>
<td style="text-align:left;">
Gm12834
</td>
<td style="text-align:left;">
NCBI_Gene:384042,ENSEMBL:ENSMUSG00000081138
</td>
<td style="text-align:left;">
MGI:3649428
</td>
<td style="text-align:left;">
pseudogene
</td>
<td style="text-align:left;">
predicted gene 12834
</td>
</tr>
<tr>
<td style="text-align:left;">
4
</td>
<td style="text-align:left;">
gene
</td>
<td style="text-align:right;">
115.2421
</td>
<td style="text-align:right;">
115.2546
</td>
<td style="text-align:left;">
positive
</td>
<td style="text-align:left;">
MGI_C57BL6J_3717143
</td>
<td style="text-align:left;">
Cyp4a29
</td>
<td style="text-align:left;">
NCBI_Gene:230639,ENSEMBL:ENSMUSG00000083138
</td>
<td style="text-align:left;">
MGI:3717143
</td>
<td style="text-align:left;">
protein coding gene
</td>
<td style="text-align:left;">
cytochrome P450, family 4, subfamily a, polypeptide 29
</td>
</tr>
<tr>
<td style="text-align:left;">
4
</td>
<td style="text-align:left;">
gene
</td>
<td style="text-align:right;">
115.2990
</td>
<td style="text-align:right;">
115.3328
</td>
<td style="text-align:left;">
positive
</td>
<td style="text-align:left;">
MGI_C57BL6J_88612
</td>
<td style="text-align:left;">
Cyp4a12a
</td>
<td style="text-align:left;">
NCBI_Gene:277753,ENSEMBL:ENSMUSG00000066071
</td>
<td style="text-align:left;">
MGI:88612
</td>
<td style="text-align:left;">
protein coding gene
</td>
<td style="text-align:left;">
cytochrome P450, family 4, subfamily a, polypeptide 12a
</td>
</tr>
<tr>
<td style="text-align:left;">
4
</td>
<td style="text-align:left;">
pseudogene
</td>
<td style="text-align:right;">
115.3438
</td>
<td style="text-align:right;">
115.3631
</td>
<td style="text-align:left;">
positive
</td>
<td style="text-align:left;">
MGI_C57BL6J_3649426
</td>
<td style="text-align:left;">
Cyp4a30-ps
</td>
<td style="text-align:left;">
NCBI_Gene:546842,ENSEMBL:ENSMUSG00000083190
</td>
<td style="text-align:left;">
MGI:3649426
</td>
<td style="text-align:left;">
pseudogene
</td>
<td style="text-align:left;">
cytochrome P450, family 4, subfamily a, member 30, pseudogene
</td>
</tr>
<tr>
<td style="text-align:left;">
4
</td>
<td style="text-align:left;">
gene
</td>
<td style="text-align:right;">
115.4116
</td>
<td style="text-align:right;">
115.4390
</td>
<td style="text-align:left;">
positive
</td>
<td style="text-align:left;">
MGI_C57BL6J_3611747
</td>
<td style="text-align:left;">
Cyp4a12b
</td>
<td style="text-align:left;">
NCBI_Gene:13118,ENSEMBL:ENSMUSG00000078597
</td>
<td style="text-align:left;">
MGI:3611747
</td>
<td style="text-align:left;">
protein coding gene
</td>
<td style="text-align:left;">
cytochrome P450, family 4, subfamily a, polypeptide 12B
</td>
</tr>
<tr>
<td style="text-align:left;">
4
</td>
<td style="text-align:left;">
pseudogene
</td>
<td style="text-align:right;">
115.4161
</td>
<td style="text-align:right;">
115.4165
</td>
<td style="text-align:left;">
negative
</td>
<td style="text-align:left;">
MGI_C57BL6J_3649425
</td>
<td style="text-align:left;">
Gm12837
</td>
<td style="text-align:left;">
ENSEMBL:ENSMUSG00000084415
</td>
<td style="text-align:left;">
MGI:3649425
</td>
<td style="text-align:left;">
pseudogene
</td>
<td style="text-align:left;">
predicted gene 12837
</td>
</tr>
<tr>
<td style="text-align:left;">
4
</td>
<td style="text-align:left;">
gene
</td>
<td style="text-align:right;">
115.4525
</td>
<td style="text-align:right;">
115.4714
</td>
<td style="text-align:left;">
positive
</td>
<td style="text-align:left;">
MGI_C57BL6J_3717145
</td>
<td style="text-align:left;">
Cyp4a30b
</td>
<td style="text-align:left;">
NCBI_Gene:435802,ENSEMBL:ENSMUSG00000084346
</td>
<td style="text-align:left;">
MGI:3717145
</td>
<td style="text-align:left;">
protein coding gene
</td>
<td style="text-align:left;">
cytochrome P450, family 4, subfamily a, polypeptide 30b
</td>
</tr>
<tr>
<td style="text-align:left;">
4
</td>
<td style="text-align:left;">
pseudogene
</td>
<td style="text-align:right;">
115.4538
</td>
<td style="text-align:right;">
115.4541
</td>
<td style="text-align:left;">
negative
</td>
<td style="text-align:left;">
MGI_C57BL6J_3649431
</td>
<td style="text-align:left;">
Gm12838
</td>
<td style="text-align:left;">
ENSEMBL:ENSMUSG00000080142
</td>
<td style="text-align:left;">
MGI:3649431
</td>
<td style="text-align:left;">
pseudogene
</td>
<td style="text-align:left;">
predicted gene 12838
</td>
</tr>
<tr>
<td style="text-align:left;">
4
</td>
<td style="text-align:left;">
pseudogene
</td>
<td style="text-align:right;">
115.4795
</td>
<td style="text-align:right;">
115.4813
</td>
<td style="text-align:left;">
positive
</td>
<td style="text-align:left;">
MGI_C57BL6J_3651919
</td>
<td style="text-align:left;">
Gm12833
</td>
<td style="text-align:left;">
NCBI_Gene:230641,ENSEMBL:ENSMUSG00000086724
</td>
<td style="text-align:left;">
MGI:3651919
</td>
<td style="text-align:left;">
pseudogene
</td>
<td style="text-align:left;">
predicted gene 12833
</td>
</tr>
<tr>
<td style="text-align:left;">
4
</td>
<td style="text-align:left;">
gene
</td>
<td style="text-align:right;">
115.4862
</td>
<td style="text-align:right;">
115.4962
</td>
<td style="text-align:left;">
negative
</td>
<td style="text-align:left;">
MGI_C57BL6J_1096550
</td>
<td style="text-align:left;">
Cyp4a14
</td>
<td style="text-align:left;">
NCBI_Gene:13119,ENSEMBL:ENSMUSG00000028715
</td>
<td style="text-align:left;">
MGI:1096550
</td>
<td style="text-align:left;">
protein coding gene
</td>
<td style="text-align:left;">
cytochrome P450, family 4, subfamily a, polypeptide 14
</td>
</tr>
<tr>
<td style="text-align:left;">
4
</td>
<td style="text-align:left;">
gene
</td>
<td style="text-align:right;">
115.5183
</td>
<td style="text-align:right;">
115.5336
</td>
<td style="text-align:left;">
positive
</td>
<td style="text-align:left;">
MGI_C57BL6J_88611
</td>
<td style="text-align:left;">
Cyp4a10
</td>
<td style="text-align:left;">
NCBI_Gene:13117,ENSEMBL:ENSMUSG00000066072
</td>
<td style="text-align:left;">
MGI:88611
</td>
<td style="text-align:left;">
protein coding gene
</td>
<td style="text-align:left;">
cytochrome P450, family 4, subfamily a, polypeptide 10
</td>
</tr>
<tr>
<td style="text-align:left;">
4
</td>
<td style="text-align:left;">
pseudogene
</td>
<td style="text-align:right;">
115.5434
</td>
<td style="text-align:right;">
115.5451
</td>
<td style="text-align:left;">
negative
</td>
<td style="text-align:left;">
MGI_C57BL6J_3651921
</td>
<td style="text-align:left;">
Cyp4b1-ps1
</td>
<td style="text-align:left;">
NCBI_Gene:100418216,ENSEMBL:ENSMUSG00000081006
</td>
<td style="text-align:left;">
MGI:3651921
</td>
<td style="text-align:left;">
pseudogene
</td>
<td style="text-align:left;">
cytochrome P450, family 4, subfamily b, polypeptide 1, pseudogene 1
</td>
</tr>
<tr>
<td style="text-align:left;">
4
</td>
<td style="text-align:left;">
gene
</td>
<td style="text-align:right;">
115.5636
</td>
<td style="text-align:right;">
115.5790
</td>
<td style="text-align:left;">
positive
</td>
<td style="text-align:left;">
MGI_C57BL6J_3028580
</td>
<td style="text-align:left;">
Cyp4a31
</td>
<td style="text-align:left;">
NCBI_Gene:666168,ENSEMBL:ENSMUSG00000028712
</td>
<td style="text-align:left;">
MGI:3028580
</td>
<td style="text-align:left;">
protein coding gene
</td>
<td style="text-align:left;">
cytochrome P450, family 4, subfamily a, polypeptide 31
</td>
</tr>
<tr>
<td style="text-align:left;">
4
</td>
<td style="text-align:left;">
pseudogene
</td>
<td style="text-align:right;">
115.5820
</td>
<td style="text-align:right;">
115.5831
</td>
<td style="text-align:left;">
negative
</td>
<td style="text-align:left;">
MGI_C57BL6J_3649236
</td>
<td style="text-align:left;">
Cyp4b1-ps2
</td>
<td style="text-align:left;">
NCBI_Gene:631037,ENSEMBL:ENSMUSG00000083457
</td>
<td style="text-align:left;">
MGI:3649236
</td>
<td style="text-align:left;">
pseudogene
</td>
<td style="text-align:left;">
cytochrome P450, family 4, subfamily b, polypeptide 1, pseudogene 2
</td>
</tr>
<tr>
<td style="text-align:left;">
4
</td>
<td style="text-align:left;">
gene
</td>
<td style="text-align:right;">
115.6009
</td>
<td style="text-align:right;">
115.6225
</td>
<td style="text-align:left;">
positive
</td>
<td style="text-align:left;">
MGI_C57BL6J_3717148
</td>
<td style="text-align:left;">
Cyp4a32
</td>
<td style="text-align:left;">
NCBI_Gene:100040843,ENSEMBL:ENSMUSG00000063929
</td>
<td style="text-align:left;">
MGI:3717148
</td>
<td style="text-align:left;">
protein coding gene
</td>
<td style="text-align:left;">
cytochrome P450, family 4, subfamily a, polypeptide 32
</td>
</tr>
<tr>
<td style="text-align:left;">
4
</td>
<td style="text-align:left;">
gene
</td>
<td style="text-align:right;">
115.6247
</td>
<td style="text-align:right;">
115.6477
</td>
<td style="text-align:left;">
negative
</td>
<td style="text-align:left;">
MGI_C57BL6J_103225
</td>
<td style="text-align:left;">
Cyp4b1
</td>
<td style="text-align:left;">
NCBI_Gene:13120,ENSEMBL:ENSMUSG00000028713
</td>
<td style="text-align:left;">
MGI:103225
</td>
<td style="text-align:left;">
protein coding gene
</td>
<td style="text-align:left;">
cytochrome P450, family 4, subfamily b, polypeptide 1
</td>
</tr>
<tr>
<td style="text-align:left;">
4
</td>
<td style="text-align:left;">
pseudogene
</td>
<td style="text-align:right;">
115.6524
</td>
<td style="text-align:right;">
115.6535
</td>
<td style="text-align:left;">
positive
</td>
<td style="text-align:left;">
MGI_C57BL6J_3650820
</td>
<td style="text-align:left;">
Gm12848
</td>
<td style="text-align:left;">
NCBI_Gene:546843,ENSEMBL:ENSMUSG00000082043
</td>
<td style="text-align:left;">
MGI:3650820
</td>
<td style="text-align:left;">
pseudogene
</td>
<td style="text-align:left;">
predicted gene 12848
</td>
</tr>
<tr>
<td style="text-align:left;">
4
</td>
<td style="text-align:left;">
pseudogene
</td>
<td style="text-align:right;">
115.7073
</td>
<td style="text-align:right;">
115.7083
</td>
<td style="text-align:left;">
positive
</td>
<td style="text-align:left;">
MGI_C57BL6J_3650623
</td>
<td style="text-align:left;">
Gm12849
</td>
<td style="text-align:left;">
ENSEMBL:ENSMUSG00000081172
</td>
<td style="text-align:left;">
MGI:3650623
</td>
<td style="text-align:left;">
pseudogene
</td>
<td style="text-align:left;">
predicted gene 12849
</td>
</tr>
<tr>
<td style="text-align:left;">
4
</td>
<td style="text-align:left;">
gene
</td>
<td style="text-align:right;">
115.7377
</td>
<td style="text-align:right;">
115.7773
</td>
<td style="text-align:left;">
positive
</td>
<td style="text-align:left;">
MGI_C57BL6J_2442397
</td>
<td style="text-align:left;">
Efcab14
</td>
<td style="text-align:left;">
NCBI_Gene:230648,ENSEMBL:ENSMUSG00000034210
</td>
<td style="text-align:left;">
MGI:2442397
</td>
<td style="text-align:left;">
protein coding gene
</td>
<td style="text-align:left;">
EF-hand calcium binding domain 14
</td>
</tr>
<tr>
<td style="text-align:left;">
4
</td>
<td style="text-align:left;">
gene
</td>
<td style="text-align:right;">
115.7798
</td>
<td style="text-align:right;">
115.7827
</td>
<td style="text-align:left;">
negative
</td>
<td style="text-align:left;">
MGI_C57BL6J_1922423
</td>
<td style="text-align:left;">
Tex38
</td>
<td style="text-align:left;">
NCBI_Gene:75173,ENSEMBL:ENSMUSG00000044556
</td>
<td style="text-align:left;">
MGI:1922423
</td>
<td style="text-align:left;">
protein coding gene
</td>
<td style="text-align:left;">
testis expressed 38
</td>
</tr>
<tr>
<td style="text-align:left;">
4
</td>
<td style="text-align:left;">
gene
</td>
<td style="text-align:right;">
115.7848
</td>
<td style="text-align:right;">
115.8188
</td>
<td style="text-align:left;">
positive
</td>
<td style="text-align:left;">
MGI_C57BL6J_2180560
</td>
<td style="text-align:left;">
Atpaf1
</td>
<td style="text-align:left;">
NCBI_Gene:230649,ENSEMBL:ENSMUSG00000028710
</td>
<td style="text-align:left;">
MGI:2180560
</td>
<td style="text-align:left;">
protein coding gene
</td>
<td style="text-align:left;">
ATP synthase mitochondrial F1 complex assembly factor 1
</td>
</tr>
<tr>
<td style="text-align:left;">
4
</td>
<td style="text-align:left;">
gene
</td>
<td style="text-align:right;">
115.8189
</td>
<td style="text-align:right;">
115.8227
</td>
<td style="text-align:left;">
positive
</td>
<td style="text-align:left;">
MGI_C57BL6J_3650822
</td>
<td style="text-align:left;">
Gm12847
</td>
<td style="text-align:left;">
ENSEMBL:ENSMUSG00000085249
</td>
<td style="text-align:left;">
MGI:3650822
</td>
<td style="text-align:left;">
lncRNA gene
</td>
<td style="text-align:left;">
predicted gene 12847
</td>
</tr>
<tr>
<td style="text-align:left;">
4
</td>
<td style="text-align:left;">
gene
</td>
<td style="text-align:right;">
115.8279
</td>
<td style="text-align:right;">
115.8362
</td>
<td style="text-align:left;">
positive
</td>
<td style="text-align:left;">
MGI_C57BL6J_2140623
</td>
<td style="text-align:left;">
Mob3c
</td>
<td style="text-align:left;">
NCBI_Gene:100465,ENSEMBL:ENSMUSG00000028709
</td>
<td style="text-align:left;">
MGI:2140623
</td>
<td style="text-align:left;">
protein coding gene
</td>
<td style="text-align:left;">
MOB kinase activator 3C
</td>
</tr>
<tr>
<td style="text-align:left;">
4
</td>
<td style="text-align:left;">
gene
</td>
<td style="text-align:right;">
115.8390
</td>
<td style="text-align:right;">
115.8793
</td>
<td style="text-align:left;">
positive
</td>
<td style="text-align:left;">
MGI_C57BL6J_894316
</td>
<td style="text-align:left;">
Mknk1
</td>
<td style="text-align:left;">
NCBI_Gene:17346,ENSEMBL:ENSMUSG00000028708
</td>
<td style="text-align:left;">
MGI:894316
</td>
<td style="text-align:left;">
protein coding gene
</td>
<td style="text-align:left;">
MAP kinase-interacting serine/threonine kinase 1
</td>
</tr>
<tr>
<td style="text-align:left;">
4
</td>
<td style="text-align:left;">
gene
</td>
<td style="text-align:right;">
115.8783
</td>
<td style="text-align:right;">
115.8786
</td>
<td style="text-align:left;">
negative
</td>
<td style="text-align:left;">
MGI_C57BL6J_5580140
</td>
<td style="text-align:left;">
Gm29434
</td>
<td style="text-align:left;">
ENSEMBL:ENSMUSG00000100081
</td>
<td style="text-align:left;">
MGI:5580140
</td>
<td style="text-align:left;">
lncRNA gene
</td>
<td style="text-align:left;">
predicted gene 29434
</td>
</tr>
<tr>
<td style="text-align:left;">
4
</td>
<td style="text-align:left;">
gene
</td>
<td style="text-align:right;">
115.8844
</td>
<td style="text-align:right;">
115.8895
</td>
<td style="text-align:left;">
positive
</td>
<td style="text-align:left;">
MGI_C57BL6J_3614952
</td>
<td style="text-align:left;">
Kncn
</td>
<td style="text-align:left;">
NCBI_Gene:654462,ENSEMBL:ENSMUSG00000073774
</td>
<td style="text-align:left;">
MGI:3614952
</td>
<td style="text-align:left;">
protein coding gene
</td>
<td style="text-align:left;">
kinocilin
</td>
</tr>
<tr>
<td style="text-align:left;">
4
</td>
<td style="text-align:left;">
gene
</td>
<td style="text-align:right;">
115.8907
</td>
<td style="text-align:right;">
115.8992
</td>
<td style="text-align:left;">
positive
</td>
<td style="text-align:left;">
MGI_C57BL6J_1923487
</td>
<td style="text-align:left;">
6430628N08Rik
</td>
<td style="text-align:left;">
NCBI_Gene:76237,ENSEMBL:ENSMUSG00000034185
</td>
<td style="text-align:left;">
MGI:1923487
</td>
<td style="text-align:left;">
protein coding gene
</td>
<td style="text-align:left;">
RIKEN cDNA 6430628N08 gene
</td>
</tr>
<tr>
<td style="text-align:left;">
4
</td>
<td style="text-align:left;">
gene
</td>
<td style="text-align:right;">
115.9151
</td>
<td style="text-align:right;">
115.9406
</td>
<td style="text-align:left;">
negative
</td>
<td style="text-align:left;">
MGI_C57BL6J_2153518
</td>
<td style="text-align:left;">
Dmbx1
</td>
<td style="text-align:left;">
NCBI_Gene:140477,ENSEMBL:ENSMUSG00000028707
</td>
<td style="text-align:left;">
MGI:2153518
</td>
<td style="text-align:left;">
protein coding gene
</td>
<td style="text-align:left;">
diencephalon/mesencephalon homeobox 1
</td>
</tr>
<tr>
<td style="text-align:left;">
4
</td>
<td style="text-align:left;">
gene
</td>
<td style="text-align:right;">
115.9671
</td>
<td style="text-align:right;">
115.9709
</td>
<td style="text-align:left;">
negative
</td>
<td style="text-align:left;">
MGI_C57BL6J_5621377
</td>
<td style="text-align:left;">
Gm38492
</td>
<td style="text-align:left;">
NCBI_Gene:102637048
</td>
<td style="text-align:left;">
MGI:5621377
</td>
<td style="text-align:left;">
lncRNA gene
</td>
<td style="text-align:left;">
predicted gene, 38492
</td>
</tr>
<tr>
<td style="text-align:left;">
4
</td>
<td style="text-align:left;">
gene
</td>
<td style="text-align:right;">
115.9671
</td>
<td style="text-align:right;">
116.0187
</td>
<td style="text-align:left;">
negative
</td>
<td style="text-align:left;">
MGI_C57BL6J_109609
</td>
<td style="text-align:left;">
Faah
</td>
<td style="text-align:left;">
NCBI_Gene:14073,ENSEMBL:ENSMUSG00000034171
</td>
<td style="text-align:left;">
MGI:109609
</td>
<td style="text-align:left;">
protein coding gene
</td>
<td style="text-align:left;">
fatty acid amide hydrolase
</td>
</tr>
<tr>
<td style="text-align:left;">
4
</td>
<td style="text-align:left;">
gene
</td>
<td style="text-align:right;">
116.0318
</td>
<td style="text-align:right;">
116.0539
</td>
<td style="text-align:left;">
negative
</td>
<td style="text-align:left;">
MGI_C57BL6J_1919431
</td>
<td style="text-align:left;">
Nsun4
</td>
<td style="text-align:left;">
NCBI_Gene:72181,ENSEMBL:ENSMUSG00000028706
</td>
<td style="text-align:left;">
MGI:1919431
</td>
<td style="text-align:left;">
protein coding gene
</td>
<td style="text-align:left;">
NOL1/NOP2/Sun domain family, member 4
</td>
</tr>
<tr>
<td style="text-align:left;">
4
</td>
<td style="text-align:left;">
gene
</td>
<td style="text-align:right;">
116.0670
</td>
<td style="text-align:right;">
116.0751
</td>
<td style="text-align:left;">
negative
</td>
<td style="text-align:left;">
MGI_C57BL6J_1913826
</td>
<td style="text-align:left;">
Uqcrh
</td>
<td style="text-align:left;">
NCBI_Gene:66576,ENSEMBL:ENSMUSG00000063882
</td>
<td style="text-align:left;">
MGI:1913826
</td>
<td style="text-align:left;">
protein coding gene
</td>
<td style="text-align:left;">
ubiquinol-cytochrome c reductase hinge protein
</td>
</tr>
<tr>
<td style="text-align:left;">
4
</td>
<td style="text-align:left;">
pseudogene
</td>
<td style="text-align:right;">
116.0672
</td>
<td style="text-align:right;">
116.0677
</td>
<td style="text-align:left;">
positive
</td>
<td style="text-align:left;">
MGI_C57BL6J_3650410
</td>
<td style="text-align:left;">
Gm12854
</td>
<td style="text-align:left;">
NCBI_Gene:102637129,ENSEMBL:ENSMUSG00000078238
</td>
<td style="text-align:left;">
MGI:3650410
</td>
<td style="text-align:left;">
pseudogene
</td>
<td style="text-align:left;">
predicted gene 12854
</td>
</tr>
<tr>
<td style="text-align:left;">
4
</td>
<td style="text-align:left;">
gene
</td>
<td style="text-align:right;">
116.0753
</td>
<td style="text-align:right;">
116.0971
</td>
<td style="text-align:left;">
positive
</td>
<td style="text-align:left;">
MGI_C57BL6J_2441984
</td>
<td style="text-align:left;">
Lrrc41
</td>
<td style="text-align:left;">
NCBI_Gene:230654,ENSEMBL:ENSMUSG00000028703
</td>
<td style="text-align:left;">
MGI:2441984
</td>
<td style="text-align:left;">
protein coding gene
</td>
<td style="text-align:left;">
leucine rich repeat containing 41
</td>
</tr>
<tr>
<td style="text-align:left;">
4
</td>
<td style="text-align:left;">
gene
</td>
<td style="text-align:right;">
116.0943
</td>
<td style="text-align:right;">
116.1237
</td>
<td style="text-align:left;">
negative
</td>
<td style="text-align:left;">
MGI_C57BL6J_894697
</td>
<td style="text-align:left;">
Rad54l
</td>
<td style="text-align:left;">
NCBI_Gene:19366,ENSEMBL:ENSMUSG00000028702
</td>
<td style="text-align:left;">
MGI:894697
</td>
<td style="text-align:left;">
protein coding gene
</td>
<td style="text-align:left;">
RAD54 like (S. cerevisiae)
</td>
</tr>
<tr>
<td style="text-align:left;">
4
</td>
<td style="text-align:left;">
gene
</td>
<td style="text-align:right;">
116.1238
</td>
<td style="text-align:right;">
116.1388
</td>
<td style="text-align:left;">
positive
</td>
<td style="text-align:left;">
MGI_C57BL6J_1924036
</td>
<td style="text-align:left;">
2510003B16Rik
</td>
<td style="text-align:left;">
NCBI_Gene:76786
</td>
<td style="text-align:left;">
MGI:1924036
</td>
<td style="text-align:left;">
lncRNA gene
</td>
<td style="text-align:left;">
RIKEN cDNA 2510003B16 gene
</td>
</tr>
<tr>
<td style="text-align:left;">
4
</td>
<td style="text-align:left;">
gene
</td>
<td style="text-align:right;">
116.1238
</td>
<td style="text-align:right;">
116.1598
</td>
<td style="text-align:left;">
positive
</td>
<td style="text-align:left;">
MGI_C57BL6J_1915523
</td>
<td style="text-align:left;">
Pomgnt1
</td>
<td style="text-align:left;">
NCBI_Gene:68273,ENSEMBL:ENSMUSG00000028700
</td>
<td style="text-align:left;">
MGI:1915523
</td>
<td style="text-align:left;">
protein coding gene
</td>
<td style="text-align:left;">
protein O-linked mannose beta 1,2-N-acetylglucosaminyltransferase
</td>
</tr>
<tr>
<td style="text-align:left;">
4
</td>
<td style="text-align:left;">
gene
</td>
<td style="text-align:right;">
116.1324
</td>
<td style="text-align:right;">
116.1513
</td>
<td style="text-align:left;">
negative
</td>
<td style="text-align:left;">
MGI_C57BL6J_1915325
</td>
<td style="text-align:left;">
Lurap1
</td>
<td style="text-align:left;">
NCBI_Gene:68075,ENSEMBL:ENSMUSG00000028701
</td>
<td style="text-align:left;">
MGI:1915325
</td>
<td style="text-align:left;">
protein coding gene
</td>
<td style="text-align:left;">
leucine rich adaptor protein 1
</td>
</tr>
<tr>
<td style="text-align:left;">
4
</td>
<td style="text-align:left;">
gene
</td>
<td style="text-align:right;">
116.1619
</td>
<td style="text-align:right;">
116.1676
</td>
<td style="text-align:left;">
negative
</td>
<td style="text-align:left;">
MGI_C57BL6J_1914055
</td>
<td style="text-align:left;">
Tspan1
</td>
<td style="text-align:left;">
NCBI_Gene:66805,ENSEMBL:ENSMUSG00000028699
</td>
<td style="text-align:left;">
MGI:1914055
</td>
<td style="text-align:left;">
protein coding gene
</td>
<td style="text-align:left;">
tetraspanin 1
</td>
</tr>
<tr>
<td style="text-align:left;">
4
</td>
<td style="text-align:left;">
gene
</td>
<td style="text-align:right;">
116.1734
</td>
<td style="text-align:right;">
116.1743
</td>
<td style="text-align:left;">
positive
</td>
<td style="text-align:left;">
MGI_C57BL6J_1914573
</td>
<td style="text-align:left;">
1700042G07Rik
</td>
<td style="text-align:left;">
NCBI_Gene:67323,ENSEMBL:ENSMUSG00000078593
</td>
<td style="text-align:left;">
MGI:1914573
</td>
<td style="text-align:left;">
protein coding gene
</td>
<td style="text-align:left;">
RIKEN cDNA 1700042G07 gene
</td>
</tr>
<tr>
<td style="text-align:left;">
4
</td>
<td style="text-align:left;">
gene
</td>
<td style="text-align:right;">
116.1734
</td>
<td style="text-align:right;">
116.2998
</td>
<td style="text-align:left;">
positive
</td>
<td style="text-align:left;">
MGI_C57BL6J_6121526
</td>
<td style="text-align:left;">
Gm49337
</td>
<td style="text-align:left;">
ENSEMBL:ENSMUSG00000111410
</td>
<td style="text-align:left;">
MGI:6121526
</td>
<td style="text-align:left;">
protein coding gene
</td>
<td style="text-align:left;">
predicted gene, 49337
</td>
</tr>
<tr>
<td style="text-align:left;">
4
</td>
<td style="text-align:left;">
gene
</td>
<td style="text-align:right;">
116.1762
</td>
<td style="text-align:right;">
116.1777
</td>
<td style="text-align:left;">
positive
</td>
<td style="text-align:left;">
MGI_C57BL6J_3649505
</td>
<td style="text-align:left;">
Gm12951
</td>
<td style="text-align:left;">
ENSEMBL:ENSMUSG00000085920
</td>
<td style="text-align:left;">
MGI:3649505
</td>
<td style="text-align:left;">
lncRNA gene
</td>
<td style="text-align:left;">
predicted gene 12951
</td>
</tr>
<tr>
<td style="text-align:left;">
4
</td>
<td style="text-align:left;">
pseudogene
</td>
<td style="text-align:right;">
116.2004
</td>
<td style="text-align:right;">
116.2010
</td>
<td style="text-align:left;">
negative
</td>
<td style="text-align:left;">
MGI_C57BL6J_3649506
</td>
<td style="text-align:left;">
Llph-ps1
</td>
<td style="text-align:left;">
NCBI_Gene:433748,ENSEMBL:ENSMUSG00000081459
</td>
<td style="text-align:left;">
MGI:3649506
</td>
<td style="text-align:left;">
pseudogene
</td>
<td style="text-align:left;">
LLP homolog, pseudogene 1
</td>
</tr>
<tr>
<td style="text-align:left;">
4
</td>
<td style="text-align:left;">
gene
</td>
<td style="text-align:right;">
116.2214
</td>
<td style="text-align:right;">
116.3031
</td>
<td style="text-align:left;">
positive
</td>
<td style="text-align:left;">
MGI_C57BL6J_109277
</td>
<td style="text-align:left;">
Pik3r3
</td>
<td style="text-align:left;">
NCBI_Gene:18710,ENSEMBL:ENSMUSG00000028698
</td>
<td style="text-align:left;">
MGI:109277
</td>
<td style="text-align:left;">
protein coding gene
</td>
<td style="text-align:left;">
phosphoinositide-3-kinase regulatory subunit 3
</td>
</tr>
<tr>
<td style="text-align:left;">
4
</td>
<td style="text-align:left;">
pseudogene
</td>
<td style="text-align:right;">
116.2500
</td>
<td style="text-align:right;">
116.2511
</td>
<td style="text-align:left;">
negative
</td>
<td style="text-align:left;">
MGI_C57BL6J_5010720
</td>
<td style="text-align:left;">
Gm18535
</td>
<td style="text-align:left;">
NCBI_Gene:100417330
</td>
<td style="text-align:left;">
MGI:5010720
</td>
<td style="text-align:left;">
pseudogene
</td>
<td style="text-align:left;">
predicted gene, 18535
</td>
</tr>
<tr>
<td style="text-align:left;">
4
</td>
<td style="text-align:left;">
gene
</td>
<td style="text-align:right;">
116.3068
</td>
<td style="text-align:right;">
116.4643
</td>
<td style="text-align:left;">
negative
</td>
<td style="text-align:left;">
MGI_C57BL6J_894676
</td>
<td style="text-align:left;">
Mast2
</td>
<td style="text-align:left;">
NCBI_Gene:17776,ENSEMBL:ENSMUSG00000003810
</td>
<td style="text-align:left;">
MGI:894676
</td>
<td style="text-align:left;">
protein coding gene
</td>
<td style="text-align:left;">
microtubule associated serine/threonine kinase 2
</td>
</tr>
<tr>
<td style="text-align:left;">
4
</td>
<td style="text-align:left;">
gene
</td>
<td style="text-align:right;">
116.3823
</td>
<td style="text-align:right;">
116.3857
</td>
<td style="text-align:left;">
negative
</td>
<td style="text-align:left;">
MGI_C57BL6J_2444793
</td>
<td style="text-align:left;">
A630078A22Rik
</td>
<td style="text-align:left;">
NA
</td>
<td style="text-align:left;">
NA
</td>
<td style="text-align:left;">
unclassified gene
</td>
<td style="text-align:left;">
RIKEN cDNA A630078A22 gene
</td>
</tr>
<tr>
<td style="text-align:left;">
4
</td>
<td style="text-align:left;">
pseudogene
</td>
<td style="text-align:right;">
116.4488
</td>
<td style="text-align:right;">
116.4492
</td>
<td style="text-align:left;">
positive
</td>
<td style="text-align:left;">
MGI_C57BL6J_3649504
</td>
<td style="text-align:left;">
Gm12950
</td>
<td style="text-align:left;">
ENSEMBL:ENSMUSG00000082592
</td>
<td style="text-align:left;">
MGI:3649504
</td>
<td style="text-align:left;">
pseudogene
</td>
<td style="text-align:left;">
predicted gene 12950
</td>
</tr>
<tr>
<td style="text-align:left;">
4
</td>
<td style="text-align:left;">
gene
</td>
<td style="text-align:right;">
116.5075
</td>
<td style="text-align:right;">
116.5426
</td>
<td style="text-align:left;">
positive
</td>
<td style="text-align:left;">
MGI_C57BL6J_96581
</td>
<td style="text-align:left;">
Ipp
</td>
<td style="text-align:left;">
NCBI_Gene:16351,ENSEMBL:ENSMUSG00000028696
</td>
<td style="text-align:left;">
MGI:96581
</td>
<td style="text-align:left;">
protein coding gene
</td>
<td style="text-align:left;">
IAP promoted placental gene
</td>
</tr>
<tr>
<td style="text-align:left;">
4
</td>
<td style="text-align:left;">
gene
</td>
<td style="text-align:right;">
116.5515
</td>
<td style="text-align:right;">
116.5570
</td>
<td style="text-align:left;">
negative
</td>
<td style="text-align:left;">
MGI_C57BL6J_3045357
</td>
<td style="text-align:left;">
Tmem69
</td>
<td style="text-align:left;">
NCBI_Gene:230657,ENSEMBL:ENSMUSG00000055900
</td>
<td style="text-align:left;">
MGI:3045357
</td>
<td style="text-align:left;">
protein coding gene
</td>
<td style="text-align:left;">
transmembrane protein 69
</td>
</tr>
<tr>
<td style="text-align:left;">
4
</td>
<td style="text-align:left;">
gene
</td>
<td style="text-align:right;">
116.5517
</td>
<td style="text-align:right;">
116.5526
</td>
<td style="text-align:left;">
positive
</td>
<td style="text-align:left;">
MGI_C57BL6J_3649497
</td>
<td style="text-align:left;">
Gm12953
</td>
<td style="text-align:left;">
ENSEMBL:ENSMUSG00000087299
</td>
<td style="text-align:left;">
MGI:3649497
</td>
<td style="text-align:left;">
lncRNA gene
</td>
<td style="text-align:left;">
predicted gene 12953
</td>
</tr>
<tr>
<td style="text-align:left;">
4
</td>
<td style="text-align:left;">
gene
</td>
<td style="text-align:right;">
116.5572
</td>
<td style="text-align:right;">
116.5939
</td>
<td style="text-align:left;">
positive
</td>
<td style="text-align:left;">
MGI_C57BL6J_1924360
</td>
<td style="text-align:left;">
Gpbp1l1
</td>
<td style="text-align:left;">
NCBI_Gene:77110,ENSEMBL:ENSMUSG00000034042
</td>
<td style="text-align:left;">
MGI:1924360
</td>
<td style="text-align:left;">
protein coding gene
</td>
<td style="text-align:left;">
GC-rich promoter binding protein 1-like 1
</td>
</tr>
<tr>
<td style="text-align:left;">
4
</td>
<td style="text-align:left;">
gene
</td>
<td style="text-align:right;">
116.5897
</td>
<td style="text-align:right;">
116.5976
</td>
<td style="text-align:left;">
negative
</td>
<td style="text-align:left;">
MGI_C57BL6J_3612454
</td>
<td style="text-align:left;">
C530005A16Rik
</td>
<td style="text-align:left;">
NCBI_Gene:654318,ENSEMBL:ENSMUSG00000085408
</td>
<td style="text-align:left;">
MGI:3612454
</td>
<td style="text-align:left;">
lncRNA gene
</td>
<td style="text-align:left;">
RIKEN cDNA C530005A16 gene
</td>
</tr>
<tr>
<td style="text-align:left;">
4
</td>
<td style="text-align:left;">
gene
</td>
<td style="text-align:right;">
116.5967
</td>
<td style="text-align:right;">
116.6003
</td>
<td style="text-align:left;">
positive
</td>
<td style="text-align:left;">
MGI_C57BL6J_1915667
</td>
<td style="text-align:left;">
Ccdc17
</td>
<td style="text-align:left;">
NCBI_Gene:622665,ENSEMBL:ENSMUSG00000034035
</td>
<td style="text-align:left;">
MGI:1915667
</td>
<td style="text-align:left;">
protein coding gene
</td>
<td style="text-align:left;">
coiled-coil domain containing 17
</td>
</tr>
<tr>
<td style="text-align:left;">
4
</td>
<td style="text-align:left;">
gene
</td>
<td style="text-align:right;">
116.6011
</td>
<td style="text-align:right;">
116.6280
</td>
<td style="text-align:left;">
negative
</td>
<td style="text-align:left;">
MGI_C57BL6J_1355328
</td>
<td style="text-align:left;">
Nasp
</td>
<td style="text-align:left;">
NCBI_Gene:50927,ENSEMBL:ENSMUSG00000028693
</td>
<td style="text-align:left;">
MGI:1355328
</td>
<td style="text-align:left;">
protein coding gene
</td>
<td style="text-align:left;">
nuclear autoantigenic sperm protein (histone-binding)
</td>
</tr>
<tr>
<td style="text-align:left;">
4
</td>
<td style="text-align:left;">
gene
</td>
<td style="text-align:right;">
116.6268
</td>
<td style="text-align:right;">
116.6287
</td>
<td style="text-align:left;">
positive
</td>
<td style="text-align:left;">
MGI_C57BL6J_2441802
</td>
<td style="text-align:left;">
A430091L06Rik
</td>
<td style="text-align:left;">
NA
</td>
<td style="text-align:left;">
NA
</td>
<td style="text-align:left;">
unclassified gene
</td>
<td style="text-align:left;">
RIKEN cDNA A430091L06 gene
</td>
</tr>
<tr>
<td style="text-align:left;">
4
</td>
<td style="text-align:left;">
gene
</td>
<td style="text-align:right;">
116.6365
</td>
<td style="text-align:right;">
116.6517
</td>
<td style="text-align:left;">
negative
</td>
<td style="text-align:left;">
MGI_C57BL6J_1929955
</td>
<td style="text-align:left;">
Akr1a1
</td>
<td style="text-align:left;">
NCBI_Gene:58810,ENSEMBL:ENSMUSG00000028692
</td>
<td style="text-align:left;">
MGI:1929955
</td>
<td style="text-align:left;">
protein coding gene
</td>
<td style="text-align:left;">
aldo-keto reductase family 1, member A1 (aldehyde reductase)
</td>
</tr>
<tr>
<td style="text-align:left;">
4
</td>
<td style="text-align:left;">
gene
</td>
<td style="text-align:right;">
116.6742
</td>
<td style="text-align:right;">
116.6743
</td>
<td style="text-align:left;">
positive
</td>
<td style="text-align:left;">
MGI_C57BL6J_5454914
</td>
<td style="text-align:left;">
Gm25137
</td>
<td style="text-align:left;">
ENSEMBL:ENSMUSG00000065185
</td>
<td style="text-align:left;">
MGI:5454914
</td>
<td style="text-align:left;">
snRNA gene
</td>
<td style="text-align:left;">
predicted gene, 25137
</td>
</tr>
<tr>
<td style="text-align:left;">
4
</td>
<td style="text-align:left;">
gene
</td>
<td style="text-align:right;">
116.6855
</td>
<td style="text-align:right;">
116.7008
</td>
<td style="text-align:left;">
positive
</td>
<td style="text-align:left;">
MGI_C57BL6J_99523
</td>
<td style="text-align:left;">
Prdx1
</td>
<td style="text-align:left;">
NCBI_Gene:18477,ENSEMBL:ENSMUSG00000028691
</td>
<td style="text-align:left;">
MGI:99523
</td>
<td style="text-align:left;">
protein coding gene
</td>
<td style="text-align:left;">
peroxiredoxin 1
</td>
</tr>
<tr>
<td style="text-align:left;">
4
</td>
<td style="text-align:left;">
gene
</td>
<td style="text-align:right;">
116.6971
</td>
<td style="text-align:right;">
116.6975
</td>
<td style="text-align:left;">
positive
</td>
<td style="text-align:left;">
MGI_C57BL6J_1924686
</td>
<td style="text-align:left;">
9530001N24Rik
</td>
<td style="text-align:left;">
NA
</td>
<td style="text-align:left;">
NA
</td>
<td style="text-align:left;">
unclassified gene
</td>
<td style="text-align:left;">
RIKEN cDNA 9530001N24 gene
</td>
</tr>
<tr>
<td style="text-align:left;">
4
</td>
<td style="text-align:left;">
gene
</td>
<td style="text-align:right;">
116.7023
</td>
<td style="text-align:right;">
116.7084
</td>
<td style="text-align:left;">
negative
</td>
<td style="text-align:left;">
MGI_C57BL6J_1914346
</td>
<td style="text-align:left;">
Mmachc
</td>
<td style="text-align:left;">
NCBI_Gene:67096,ENSEMBL:ENSMUSG00000028690
</td>
<td style="text-align:left;">
MGI:1914346
</td>
<td style="text-align:left;">
protein coding gene
</td>
<td style="text-align:left;">
methylmalonic aciduria cblC type, with homocystinuria
</td>
</tr>
<tr>
<td style="text-align:left;">
4
</td>
<td style="text-align:left;">
gene
</td>
<td style="text-align:right;">
116.7081
</td>
<td style="text-align:right;">
116.7084
</td>
<td style="text-align:left;">
positive
</td>
<td style="text-align:left;">
MGI_C57BL6J_1925287
</td>
<td style="text-align:left;">
4930565M07Rik
</td>
<td style="text-align:left;">
NA
</td>
<td style="text-align:left;">
NA
</td>
<td style="text-align:left;">
unclassified gene
</td>
<td style="text-align:left;">
RIKEN cDNA 4930565M07 gene
</td>
</tr>
<tr>
<td style="text-align:left;">
4
</td>
<td style="text-align:left;">
gene
</td>
<td style="text-align:right;">
116.7085
</td>
<td style="text-align:right;">
116.7512
</td>
<td style="text-align:left;">
positive
</td>
<td style="text-align:left;">
MGI_C57BL6J_1915644
</td>
<td style="text-align:left;">
Ccdc163
</td>
<td style="text-align:left;">
NCBI_Gene:68394,ENSEMBL:ENSMUSG00000028689
</td>
<td style="text-align:left;">
MGI:1915644
</td>
<td style="text-align:left;">
protein coding gene
</td>
<td style="text-align:left;">
coiled-coil domain containing 163
</td>
</tr>
<tr>
<td style="text-align:left;">
4
</td>
<td style="text-align:left;">
gene
</td>
<td style="text-align:right;">
116.7209
</td>
<td style="text-align:right;">
116.8060
</td>
<td style="text-align:left;">
positive
</td>
<td style="text-align:left;">
MGI_C57BL6J_2385204
</td>
<td style="text-align:left;">
Tesk2
</td>
<td style="text-align:left;">
NCBI_Gene:230661,ENSEMBL:ENSMUSG00000033985
</td>
<td style="text-align:left;">
MGI:2385204
</td>
<td style="text-align:left;">
protein coding gene
</td>
<td style="text-align:left;">
testis-specific kinase 2
</td>
</tr>
<tr>
<td style="text-align:left;">
4
</td>
<td style="text-align:left;">
gene
</td>
<td style="text-align:right;">
116.7259
</td>
<td style="text-align:right;">
116.7260
</td>
<td style="text-align:left;">
positive
</td>
<td style="text-align:left;">
MGI_C57BL6J_5531070
</td>
<td style="text-align:left;">
Gm27688
</td>
<td style="text-align:left;">
ENSEMBL:ENSMUSG00000098491
</td>
<td style="text-align:left;">
MGI:5531070
</td>
<td style="text-align:left;">
unclassified non-coding RNA gene
</td>
<td style="text-align:left;">
predicted gene, 27688
</td>
</tr>
<tr>
<td style="text-align:left;">
4
</td>
<td style="text-align:left;">
gene
</td>
<td style="text-align:right;">
116.7943
</td>
<td style="text-align:right;">
116.8076
</td>
<td style="text-align:left;">
negative
</td>
<td style="text-align:left;">
MGI_C57BL6J_1915526
</td>
<td style="text-align:left;">
Toe1
</td>
<td style="text-align:left;">
NCBI_Gene:68276,ENSEMBL:ENSMUSG00000028688
</td>
<td style="text-align:left;">
MGI:1915526
</td>
<td style="text-align:left;">
protein coding gene
</td>
<td style="text-align:left;">
target of EGR1, member 1 (nuclear)
</td>
</tr>
<tr>
<td style="text-align:left;">
4
</td>
<td style="text-align:left;">
gene
</td>
<td style="text-align:right;">
116.8077
</td>
<td style="text-align:right;">
116.8194
</td>
<td style="text-align:left;">
positive
</td>
<td style="text-align:left;">
MGI_C57BL6J_1917853
</td>
<td style="text-align:left;">
Mutyh
</td>
<td style="text-align:left;">
NCBI_Gene:70603,ENSEMBL:ENSMUSG00000028687
</td>
<td style="text-align:left;">
MGI:1917853
</td>
<td style="text-align:left;">
protein coding gene
</td>
<td style="text-align:left;">
mutY DNA glycosylase
</td>
</tr>
<tr>
<td style="text-align:left;">
4
</td>
<td style="text-align:left;">
gene
</td>
<td style="text-align:right;">
116.8199
</td>
<td style="text-align:right;">
116.8217
</td>
<td style="text-align:left;">
negative
</td>
<td style="text-align:left;">
MGI_C57BL6J_2444646
</td>
<td style="text-align:left;">
Hpdl
</td>
<td style="text-align:left;">
NCBI_Gene:242642,ENSEMBL:ENSMUSG00000043155
</td>
<td style="text-align:left;">
MGI:2444646
</td>
<td style="text-align:left;">
protein coding gene
</td>
<td style="text-align:left;">
4-hydroxyphenylpyruvate dioxygenase-like
</td>
</tr>
<tr>
<td style="text-align:left;">
4
</td>
<td style="text-align:left;">
gene
</td>
<td style="text-align:right;">
116.8243
</td>
<td style="text-align:right;">
116.8306
</td>
<td style="text-align:left;">
positive
</td>
<td style="text-align:left;">
MGI_C57BL6J_3650481
</td>
<td style="text-align:left;">
Gm12996
</td>
<td style="text-align:left;">
NCBI_Gene:100502973,ENSEMBL:ENSMUSG00000086417
</td>
<td style="text-align:left;">
MGI:3650481
</td>
<td style="text-align:left;">
lncRNA gene
</td>
<td style="text-align:left;">
predicted gene 12996
</td>
</tr>
<tr>
<td style="text-align:left;">
4
</td>
<td style="text-align:left;">
pseudogene
</td>
<td style="text-align:right;">
116.8399
</td>
<td style="text-align:right;">
116.8402
</td>
<td style="text-align:left;">
negative
</td>
<td style="text-align:left;">
MGI_C57BL6J_3650480
</td>
<td style="text-align:left;">
Rpl36-ps8
</td>
<td style="text-align:left;">
NCBI_Gene:625336,ENSEMBL:ENSMUSG00000075390
</td>
<td style="text-align:left;">
MGI:3650480
</td>
<td style="text-align:left;">
pseudogene
</td>
<td style="text-align:left;">
ribosomal protein L36, pseudogene 8
</td>
</tr>
<tr>
<td style="text-align:left;">
4
</td>
<td style="text-align:left;">
pseudogene
</td>
<td style="text-align:right;">
116.8527
</td>
<td style="text-align:right;">
116.8531
</td>
<td style="text-align:left;">
positive
</td>
<td style="text-align:left;">
MGI_C57BL6J_3650479
</td>
<td style="text-align:left;">
Gm12994
</td>
<td style="text-align:left;">
ENSEMBL:ENSMUSG00000082841
</td>
<td style="text-align:left;">
MGI:3650479
</td>
<td style="text-align:left;">
pseudogene
</td>
<td style="text-align:left;">
predicted gene 12994
</td>
</tr>
<tr>
<td style="text-align:left;">
4
</td>
<td style="text-align:left;">
pseudogene
</td>
<td style="text-align:right;">
116.8746
</td>
<td style="text-align:right;">
116.8755
</td>
<td style="text-align:left;">
negative
</td>
<td style="text-align:left;">
MGI_C57BL6J_3650477
</td>
<td style="text-align:left;">
Gm12993
</td>
<td style="text-align:left;">
ENSEMBL:ENSMUSG00000082063
</td>
<td style="text-align:left;">
MGI:3650477
</td>
<td style="text-align:left;">
pseudogene
</td>
<td style="text-align:left;">
predicted gene 12993
</td>
</tr>
<tr>
<td style="text-align:left;">
4
</td>
<td style="text-align:left;">
gene
</td>
<td style="text-align:right;">
116.8774
</td>
<td style="text-align:right;">
116.9893
</td>
<td style="text-align:left;">
positive
</td>
<td style="text-align:left;">
MGI_C57BL6J_1921714
</td>
<td style="text-align:left;">
Zswim5
</td>
<td style="text-align:left;">
NCBI_Gene:74464,ENSEMBL:ENSMUSG00000033948
</td>
<td style="text-align:left;">
MGI:1921714
</td>
<td style="text-align:left;">
protein coding gene
</td>
<td style="text-align:left;">
zinc finger SWIM-type containing 5
</td>
</tr>
<tr>
<td style="text-align:left;">
4
</td>
<td style="text-align:left;">
gene
</td>
<td style="text-align:right;">
116.9046
</td>
<td style="text-align:right;">
116.9047
</td>
<td style="text-align:left;">
positive
</td>
<td style="text-align:left;">
MGI_C57BL6J_5456132
</td>
<td style="text-align:left;">
Gm26355
</td>
<td style="text-align:left;">
ENSEMBL:ENSMUSG00000088845
</td>
<td style="text-align:left;">
MGI:5456132
</td>
<td style="text-align:left;">
snRNA gene
</td>
<td style="text-align:left;">
predicted gene, 26355
</td>
</tr>
<tr>
<td style="text-align:left;">
4
</td>
<td style="text-align:left;">
gene
</td>
<td style="text-align:right;">
116.9152
</td>
<td style="text-align:right;">
116.9155
</td>
<td style="text-align:left;">
positive
</td>
<td style="text-align:left;">
MGI_C57BL6J_5455212
</td>
<td style="text-align:left;">
Gm25435
</td>
<td style="text-align:left;">
ENSEMBL:ENSMUSG00000088902
</td>
<td style="text-align:left;">
MGI:5455212
</td>
<td style="text-align:left;">
unclassified non-coding RNA gene
</td>
<td style="text-align:left;">
predicted gene, 25435
</td>
</tr>
<tr>
<td style="text-align:left;">
4
</td>
<td style="text-align:left;">
gene
</td>
<td style="text-align:right;">
116.9224
</td>
<td style="text-align:right;">
116.9225
</td>
<td style="text-align:left;">
negative
</td>
<td style="text-align:left;">
MGI_C57BL6J_5451869
</td>
<td style="text-align:left;">
Gm22092
</td>
<td style="text-align:left;">
ENSEMBL:ENSMUSG00000095805
</td>
<td style="text-align:left;">
MGI:5451869
</td>
<td style="text-align:left;">
snRNA gene
</td>
<td style="text-align:left;">
predicted gene, 22092
</td>
</tr>
<tr>
<td style="text-align:left;">
4
</td>
<td style="text-align:left;">
pseudogene
</td>
<td style="text-align:right;">
116.9457
</td>
<td style="text-align:right;">
116.9465
</td>
<td style="text-align:left;">
negative
</td>
<td style="text-align:left;">
MGI_C57BL6J_3651168
</td>
<td style="text-align:left;">
Gm12998
</td>
<td style="text-align:left;">
NCBI_Gene:100417429,ENSEMBL:ENSMUSG00000082207
</td>
<td style="text-align:left;">
MGI:3651168
</td>
<td style="text-align:left;">
pseudogene
</td>
<td style="text-align:left;">
predicted gene 12998
</td>
</tr>
<tr>
<td style="text-align:left;">
4
</td>
<td style="text-align:left;">
gene
</td>
<td style="text-align:right;">
116.9900
</td>
<td style="text-align:right;">
116.9944
</td>
<td style="text-align:left;">
negative
</td>
<td style="text-align:left;">
MGI_C57BL6J_98916
</td>
<td style="text-align:left;">
Urod
</td>
<td style="text-align:left;">
NCBI_Gene:22275,ENSEMBL:ENSMUSG00000028684
</td>
<td style="text-align:left;">
MGI:98916
</td>
<td style="text-align:left;">
protein coding gene
</td>
<td style="text-align:left;">
uroporphyrinogen decarboxylase
</td>
</tr>
<tr>
<td style="text-align:left;">
4
</td>
<td style="text-align:left;">
gene
</td>
<td style="text-align:right;">
116.9953
</td>
<td style="text-align:right;">
117.0053
</td>
<td style="text-align:left;">
positive
</td>
<td style="text-align:left;">
MGI_C57BL6J_1923858
</td>
<td style="text-align:left;">
Hectd3
</td>
<td style="text-align:left;">
NCBI_Gene:76608,ENSEMBL:ENSMUSG00000046861
</td>
<td style="text-align:left;">
MGI:1923858
</td>
<td style="text-align:left;">
protein coding gene
</td>
<td style="text-align:left;">
HECT domain E3 ubiquitin protein ligase 3
</td>
</tr>
<tr>
<td style="text-align:left;">
4
</td>
<td style="text-align:left;">
gene
</td>
<td style="text-align:right;">
117.0022
</td>
<td style="text-align:right;">
117.0030
</td>
<td style="text-align:left;">
negative
</td>
<td style="text-align:left;">
MGI_C57BL6J_1919485
</td>
<td style="text-align:left;">
1700021J08Rik
</td>
<td style="text-align:left;">
ENSEMBL:ENSMUSG00000086890
</td>
<td style="text-align:left;">
MGI:1919485
</td>
<td style="text-align:left;">
lncRNA gene
</td>
<td style="text-align:left;">
RIKEN cDNA 1700021J08 gene
</td>
</tr>
<tr>
<td style="text-align:left;">
4
</td>
<td style="text-align:left;">
pseudogene
</td>
<td style="text-align:right;">
117.0151
</td>
<td style="text-align:right;">
117.0158
</td>
<td style="text-align:left;">
negative
</td>
<td style="text-align:left;">
MGI_C57BL6J_3650026
</td>
<td style="text-align:left;">
Gm12997
</td>
<td style="text-align:left;">
NCBI_Gene:100416056,ENSEMBL:ENSMUSG00000082249
</td>
<td style="text-align:left;">
MGI:3650026
</td>
<td style="text-align:left;">
pseudogene
</td>
<td style="text-align:left;">
predicted gene 12997
</td>
</tr>
<tr>
<td style="text-align:left;">
4
</td>
<td style="text-align:left;">
gene
</td>
<td style="text-align:right;">
117.0194
</td>
<td style="text-align:right;">
117.0873
</td>
<td style="text-align:left;">
positive
</td>
<td style="text-align:left;">
MGI_C57BL6J_1313286
</td>
<td style="text-align:left;">
Eif2b3
</td>
<td style="text-align:left;">
NCBI_Gene:108067,ENSEMBL:ENSMUSG00000028683
</td>
<td style="text-align:left;">
MGI:1313286
</td>
<td style="text-align:left;">
protein coding gene
</td>
<td style="text-align:left;">
eukaryotic translation initiation factor 2B, subunit 3
</td>
</tr>
<tr>
<td style="text-align:left;">
4
</td>
<td style="text-align:left;">
gene
</td>
<td style="text-align:right;">
117.0272
</td>
<td style="text-align:right;">
117.0273
</td>
<td style="text-align:left;">
positive
</td>
<td style="text-align:left;">
MGI_C57BL6J_5452112
</td>
<td style="text-align:left;">
Gm22335
</td>
<td style="text-align:left;">
ENSEMBL:ENSMUSG00000088349
</td>
<td style="text-align:left;">
MGI:5452112
</td>
<td style="text-align:left;">
snRNA gene
</td>
<td style="text-align:left;">
predicted gene, 22335
</td>
</tr>
<tr>
<td style="text-align:left;">
4
</td>
<td style="text-align:left;">
gene
</td>
<td style="text-align:right;">
117.0958
</td>
<td style="text-align:right;">
117.1161
</td>
<td style="text-align:left;">
positive
</td>
<td style="text-align:left;">
MGI_C57BL6J_1095405
</td>
<td style="text-align:left;">
Ptch2
</td>
<td style="text-align:left;">
NCBI_Gene:19207,ENSEMBL:ENSMUSG00000028681
</td>
<td style="text-align:left;">
MGI:1095405
</td>
<td style="text-align:left;">
protein coding gene
</td>
<td style="text-align:left;">
patched 2
</td>
</tr>
<tr>
<td style="text-align:left;">
4
</td>
<td style="text-align:left;">
gene
</td>
<td style="text-align:right;">
117.1192
</td>
<td style="text-align:right;">
117.1257
</td>
<td style="text-align:left;">
negative
</td>
<td style="text-align:left;">
MGI_C57BL6J_1925861
</td>
<td style="text-align:left;">
Btbd19
</td>
<td style="text-align:left;">
NCBI_Gene:78611,ENSEMBL:ENSMUSG00000073771
</td>
<td style="text-align:left;">
MGI:1925861
</td>
<td style="text-align:left;">
protein coding gene
</td>
<td style="text-align:left;">
BTB (POZ) domain containing 19
</td>
</tr>
<tr>
<td style="text-align:left;">
4
</td>
<td style="text-align:left;">
gene
</td>
<td style="text-align:right;">
117.1245
</td>
<td style="text-align:right;">
117.1287
</td>
<td style="text-align:left;">
positive
</td>
<td style="text-align:left;">
MGI_C57BL6J_3045358
</td>
<td style="text-align:left;">
Tctex1d4
</td>
<td style="text-align:left;">
NCBI_Gene:242646,ENSEMBL:ENSMUSG00000047671
</td>
<td style="text-align:left;">
MGI:3045358
</td>
<td style="text-align:left;">
protein coding gene
</td>
<td style="text-align:left;">
Tctex1 domain containing 4
</td>
</tr>
<tr>
<td style="text-align:left;">
4
</td>
<td style="text-align:left;">
gene
</td>
<td style="text-align:right;">
117.1287
</td>
<td style="text-align:right;">
117.1340
</td>
<td style="text-align:left;">
negative
</td>
<td style="text-align:left;">
MGI_C57BL6J_109604
</td>
<td style="text-align:left;">
Plk3
</td>
<td style="text-align:left;">
NCBI_Gene:12795,ENSEMBL:ENSMUSG00000028680
</td>
<td style="text-align:left;">
MGI:109604
</td>
<td style="text-align:left;">
protein coding gene
</td>
<td style="text-align:left;">
polo like kinase 3
</td>
</tr>
<tr>
<td style="text-align:left;">
4
</td>
<td style="text-align:left;">
pseudogene
</td>
<td style="text-align:right;">
117.1453
</td>
<td style="text-align:right;">
117.1486
</td>
<td style="text-align:left;">
positive
</td>
<td style="text-align:left;">
MGI_C57BL6J_3663793
</td>
<td style="text-align:left;">
Best4-ps
</td>
<td style="text-align:left;">
NCBI_Gene:230664,ENSEMBL:ENSMUSG00000033872
</td>
<td style="text-align:left;">
MGI:3663793
</td>
<td style="text-align:left;">
pseudogene
</td>
<td style="text-align:left;">
bestrophin 4, pseudogene
</td>
</tr>
<tr>
<td style="text-align:left;">
4
</td>
<td style="text-align:left;">
pseudogene
</td>
<td style="text-align:right;">
117.1489
</td>
<td style="text-align:right;">
117.1493
</td>
<td style="text-align:left;">
positive
</td>
<td style="text-align:left;">
MGI_C57BL6J_3651767
</td>
<td style="text-align:left;">
Gm13015
</td>
<td style="text-align:left;">
NCBI_Gene:625405,ENSEMBL:ENSMUSG00000070834
</td>
<td style="text-align:left;">
MGI:3651767
</td>
<td style="text-align:left;">
pseudogene
</td>
<td style="text-align:left;">
predicted gene 13015
</td>
</tr>
<tr>
<td style="text-align:left;">
4
</td>
<td style="text-align:left;">
gene
</td>
<td style="text-align:right;">
117.1538
</td>
<td style="text-align:right;">
117.1562
</td>
<td style="text-align:left;">
negative
</td>
<td style="text-align:left;">
MGI_C57BL6J_98166
</td>
<td style="text-align:left;">
Rps8
</td>
<td style="text-align:left;">
NCBI_Gene:20116,ENSEMBL:ENSMUSG00000047675
</td>
<td style="text-align:left;">
MGI:98166
</td>
<td style="text-align:left;">
protein coding gene
</td>
<td style="text-align:left;">
ribosomal protein S8
</td>
</tr>
<tr>
<td style="text-align:left;">
4
</td>
<td style="text-align:left;">
gene
</td>
<td style="text-align:right;">
117.1541
</td>
<td style="text-align:right;">
117.1542
</td>
<td style="text-align:left;">
negative
</td>
<td style="text-align:left;">
MGI_C57BL6J_5452757
</td>
<td style="text-align:left;">
Gm22980
</td>
<td style="text-align:left;">
ENSEMBL:ENSMUSG00000064542
</td>
<td style="text-align:left;">
MGI:5452757
</td>
<td style="text-align:left;">
snoRNA gene
</td>
<td style="text-align:left;">
predicted gene, 22980
</td>
</tr>
<tr>
<td style="text-align:left;">
4
</td>
<td style="text-align:left;">
gene
</td>
<td style="text-align:right;">
117.1545
</td>
<td style="text-align:right;">
117.1546
</td>
<td style="text-align:left;">
negative
</td>
<td style="text-align:left;">
MGI_C57BL6J_3819531
</td>
<td style="text-align:left;">
Snord38a
</td>
<td style="text-align:left;">
NCBI_Gene:100217424,ENSEMBL:ENSMUSG00000065680
</td>
<td style="text-align:left;">
MGI:3819531
</td>
<td style="text-align:left;">
snoRNA gene
</td>
<td style="text-align:left;">
small nucleolar RNA, C/D box 38A
</td>
</tr>
<tr>
<td style="text-align:left;">
4
</td>
<td style="text-align:left;">
gene
</td>
<td style="text-align:right;">
117.1552
</td>
<td style="text-align:right;">
117.1553
</td>
<td style="text-align:left;">
negative
</td>
<td style="text-align:left;">
MGI_C57BL6J_5456107
</td>
<td style="text-align:left;">
Gm26330
</td>
<td style="text-align:left;">
ENSEMBL:ENSMUSG00000064751
</td>
<td style="text-align:left;">
MGI:5456107
</td>
<td style="text-align:left;">
snoRNA gene
</td>
<td style="text-align:left;">
predicted gene, 26330
</td>
</tr>
<tr>
<td style="text-align:left;">
4
</td>
<td style="text-align:left;">
gene
</td>
<td style="text-align:right;">
117.1558
</td>
<td style="text-align:right;">
117.1558
</td>
<td style="text-align:left;">
negative
</td>
<td style="text-align:left;">
MGI_C57BL6J_3819543
</td>
<td style="text-align:left;">
Snord55
</td>
<td style="text-align:left;">
NCBI_Gene:100216533,ENSEMBL:ENSMUSG00000092680
</td>
<td style="text-align:left;">
MGI:3819543
</td>
<td style="text-align:left;">
snoRNA gene
</td>
<td style="text-align:left;">
small nucleolar RNA, C/D box 55
</td>
</tr>
<tr>
<td style="text-align:left;">
4
</td>
<td style="text-align:left;">
gene
</td>
<td style="text-align:right;">
117.1583
</td>
<td style="text-align:right;">
117.1584
</td>
<td style="text-align:left;">
negative
</td>
<td style="text-align:left;">
MGI_C57BL6J_5452175
</td>
<td style="text-align:left;">
Gm22398
</td>
<td style="text-align:left;">
ENSEMBL:ENSMUSG00000095209
</td>
<td style="text-align:left;">
MGI:5452175
</td>
<td style="text-align:left;">
miRNA gene
</td>
<td style="text-align:left;">
predicted gene, 22398
</td>
</tr>
<tr>
<td style="text-align:left;">
4
</td>
<td style="text-align:left;">
gene
</td>
<td style="text-align:right;">
117.1596
</td>
<td style="text-align:right;">
117.1826
</td>
<td style="text-align:left;">
negative
</td>
<td style="text-align:left;">
MGI_C57BL6J_1921054
</td>
<td style="text-align:left;">
Kif2c
</td>
<td style="text-align:left;">
NCBI_Gene:73804,ENSEMBL:ENSMUSG00000028678
</td>
<td style="text-align:left;">
MGI:1921054
</td>
<td style="text-align:left;">
protein coding gene
</td>
<td style="text-align:left;">
kinesin family member 2C
</td>
</tr>
<tr>
<td style="text-align:left;">
4
</td>
<td style="text-align:left;">
gene
</td>
<td style="text-align:right;">
117.1899
</td>
<td style="text-align:right;">
117.1901
</td>
<td style="text-align:left;">
positive
</td>
<td style="text-align:left;">
MGI_C57BL6J_5454876
</td>
<td style="text-align:left;">
Gm25099
</td>
<td style="text-align:left;">
ENSEMBL:ENSMUSG00000095676
</td>
<td style="text-align:left;">
MGI:5454876
</td>
<td style="text-align:left;">
snRNA gene
</td>
<td style="text-align:left;">
predicted gene, 25099
</td>
</tr>
<tr>
<td style="text-align:left;">
4
</td>
<td style="text-align:left;">
gene
</td>
<td style="text-align:right;">
117.2025
</td>
<td style="text-align:right;">
117.2026
</td>
<td style="text-align:left;">
positive
</td>
<td style="text-align:left;">
MGI_C57BL6J_5452920
</td>
<td style="text-align:left;">
Gm23143
</td>
<td style="text-align:left;">
ENSEMBL:ENSMUSG00000094405
</td>
<td style="text-align:left;">
MGI:5452920
</td>
<td style="text-align:left;">
snRNA gene
</td>
<td style="text-align:left;">
predicted gene, 23143
</td>
</tr>
<tr>
<td style="text-align:left;">
4
</td>
<td style="text-align:left;">
gene
</td>
<td style="text-align:right;">
117.2050
</td>
<td style="text-align:right;">
117.2054
</td>
<td style="text-align:left;">
positive
</td>
<td style="text-align:left;">
MGI_C57BL6J_3629729
</td>
<td style="text-align:left;">
Gt(pU21)107Imeg
</td>
<td style="text-align:left;">
NA
</td>
<td style="text-align:left;">
NA
</td>
<td style="text-align:left;">
unclassified gene
</td>
<td style="text-align:left;">
gene trap 107%2c Institute of Molecular Embryology and Genetics
</td>
</tr>
<tr>
<td style="text-align:left;">
4
</td>
<td style="text-align:left;">
gene
</td>
<td style="text-align:right;">
117.2108
</td>
<td style="text-align:right;">
117.2110
</td>
<td style="text-align:left;">
positive
</td>
<td style="text-align:left;">
MGI_C57BL6J_5453064
</td>
<td style="text-align:left;">
Gm23287
</td>
<td style="text-align:left;">
ENSEMBL:ENSMUSG00000096280
</td>
<td style="text-align:left;">
MGI:5453064
</td>
<td style="text-align:left;">
snRNA gene
</td>
<td style="text-align:left;">
predicted gene, 23287
</td>
</tr>
<tr>
<td style="text-align:left;">
4
</td>
<td style="text-align:left;">
gene
</td>
<td style="text-align:right;">
117.2133
</td>
<td style="text-align:right;">
117.2521
</td>
<td style="text-align:left;">
negative
</td>
<td style="text-align:left;">
MGI_C57BL6J_2686507
</td>
<td style="text-align:left;">
Armh1
</td>
<td style="text-align:left;">
NCBI_Gene:381544,ENSEMBL:ENSMUSG00000060268
</td>
<td style="text-align:left;">
MGI:2686507
</td>
<td style="text-align:left;">
protein coding gene
</td>
<td style="text-align:left;">
armadillo-like helical domain containing 1
</td>
</tr>
<tr>
<td style="text-align:left;">
4
</td>
<td style="text-align:left;">
gene
</td>
<td style="text-align:right;">
117.2133
</td>
<td style="text-align:right;">
117.2138
</td>
<td style="text-align:left;">
negative
</td>
<td style="text-align:left;">
MGI_C57BL6J_1923635
</td>
<td style="text-align:left;">
1700012C08Rik
</td>
<td style="text-align:left;">
ENSEMBL:ENSMUSG00000086482
</td>
<td style="text-align:left;">
MGI:1923635
</td>
<td style="text-align:left;">
lncRNA gene
</td>
<td style="text-align:left;">
RIKEN cDNA 1700012C08 gene
</td>
</tr>
<tr>
<td style="text-align:left;">
4
</td>
<td style="text-align:left;">
gene
</td>
<td style="text-align:right;">
117.2520
</td>
<td style="text-align:right;">
117.2686
</td>
<td style="text-align:left;">
positive
</td>
<td style="text-align:left;">
MGI_C57BL6J_1916027
</td>
<td style="text-align:left;">
Tmem53
</td>
<td style="text-align:left;">
NCBI_Gene:68777,ENSEMBL:ENSMUSG00000048772
</td>
<td style="text-align:left;">
MGI:1916027
</td>
<td style="text-align:left;">
protein coding gene
</td>
<td style="text-align:left;">
transmembrane protein 53
</td>
</tr>
<tr>
<td style="text-align:left;">
4
</td>
<td style="text-align:left;">
gene
</td>
<td style="text-align:right;">
117.2715
</td>
<td style="text-align:right;">
117.4971
</td>
<td style="text-align:left;">
negative
</td>
<td style="text-align:left;">
MGI_C57BL6J_1913993
</td>
<td style="text-align:left;">
Rnf220
</td>
<td style="text-align:left;">
NCBI_Gene:66743,ENSEMBL:ENSMUSG00000028677
</td>
<td style="text-align:left;">
MGI:1913993
</td>
<td style="text-align:left;">
protein coding gene
</td>
<td style="text-align:left;">
ring finger protein 220
</td>
</tr>
<tr>
<td style="text-align:left;">
4
</td>
<td style="text-align:left;">
gene
</td>
<td style="text-align:right;">
117.2894
</td>
<td style="text-align:right;">
117.3054
</td>
<td style="text-align:left;">
positive
</td>
<td style="text-align:left;">
MGI_C57BL6J_3651051
</td>
<td style="text-align:left;">
Gm12843
</td>
<td style="text-align:left;">
NCBI_Gene:102637763,ENSEMBL:ENSMUSG00000085749
</td>
<td style="text-align:left;">
MGI:3651051
</td>
<td style="text-align:left;">
lncRNA gene
</td>
<td style="text-align:left;">
predicted gene 12843
</td>
</tr>
<tr>
<td style="text-align:left;">
4
</td>
<td style="text-align:left;">
gene
</td>
<td style="text-align:right;">
117.3117
</td>
<td style="text-align:right;">
117.3194
</td>
<td style="text-align:left;">
negative
</td>
<td style="text-align:left;">
MGI_C57BL6J_5623116
</td>
<td style="text-align:left;">
Gm40231
</td>
<td style="text-align:left;">
NCBI_Gene:105244664
</td>
<td style="text-align:left;">
MGI:5623116
</td>
<td style="text-align:left;">
lncRNA gene
</td>
<td style="text-align:left;">
predicted gene, 40231
</td>
</tr>
<tr>
<td style="text-align:left;">
4
</td>
<td style="text-align:left;">
gene
</td>
<td style="text-align:right;">
117.3281
</td>
<td style="text-align:right;">
117.3290
</td>
<td style="text-align:left;">
negative
</td>
<td style="text-align:left;">
MGI_C57BL6J_1924582
</td>
<td style="text-align:left;">
C030012D19Rik
</td>
<td style="text-align:left;">
NA
</td>
<td style="text-align:left;">
NA
</td>
<td style="text-align:left;">
unclassified gene
</td>
<td style="text-align:left;">
RIKEN cDNA C030012D19 gene
</td>
</tr>
<tr>
<td style="text-align:left;">
4
</td>
<td style="text-align:left;">
gene
</td>
<td style="text-align:right;">
117.4417
</td>
<td style="text-align:right;">
117.4550
</td>
<td style="text-align:left;">
positive
</td>
<td style="text-align:left;">
MGI_C57BL6J_3650145
</td>
<td style="text-align:left;">
Gm12828
</td>
<td style="text-align:left;">
NCBI_Gene:108168973,ENSEMBL:ENSMUSG00000086876
</td>
<td style="text-align:left;">
MGI:3650145
</td>
<td style="text-align:left;">
lncRNA gene
</td>
<td style="text-align:left;">
predicted gene 12828
</td>
</tr>
<tr>
<td style="text-align:left;">
4
</td>
<td style="text-align:left;">
gene
</td>
<td style="text-align:right;">
117.4959
</td>
<td style="text-align:right;">
117.5093
</td>
<td style="text-align:left;">
positive
</td>
<td style="text-align:left;">
MGI_C57BL6J_3651358
</td>
<td style="text-align:left;">
Gm12827
</td>
<td style="text-align:left;">
NCBI_Gene:102637578,ENSEMBL:ENSMUSG00000087228
</td>
<td style="text-align:left;">
MGI:3651358
</td>
<td style="text-align:left;">
lncRNA gene
</td>
<td style="text-align:left;">
predicted gene 12827
</td>
</tr>
<tr>
<td style="text-align:left;">
4
</td>
<td style="text-align:left;">
gene
</td>
<td style="text-align:right;">
117.5094
</td>
<td style="text-align:right;">
117.5196
</td>
<td style="text-align:left;">
positive
</td>
<td style="text-align:left;">
MGI_C57BL6J_5593828
</td>
<td style="text-align:left;">
Gm34669
</td>
<td style="text-align:left;">
NCBI_Gene:102637993
</td>
<td style="text-align:left;">
MGI:5593828
</td>
<td style="text-align:left;">
lncRNA gene
</td>
<td style="text-align:left;">
predicted gene, 34669
</td>
</tr>
<tr>
<td style="text-align:left;">
4
</td>
<td style="text-align:left;">
pseudogene
</td>
<td style="text-align:right;">
117.5138
</td>
<td style="text-align:right;">
117.5142
</td>
<td style="text-align:left;">
negative
</td>
<td style="text-align:left;">
MGI_C57BL6J_3650150
</td>
<td style="text-align:left;">
Gm12826
</td>
<td style="text-align:left;">
ENSEMBL:ENSMUSG00000083446
</td>
<td style="text-align:left;">
MGI:3650150
</td>
<td style="text-align:left;">
pseudogene
</td>
<td style="text-align:left;">
predicted gene 12826
</td>
</tr>
<tr>
<td style="text-align:left;">
4
</td>
<td style="text-align:left;">
gene
</td>
<td style="text-align:right;">
117.5503
</td>
<td style="text-align:right;">
117.6743
</td>
<td style="text-align:left;">
positive
</td>
<td style="text-align:left;">
MGI_C57BL6J_2153887
</td>
<td style="text-align:left;">
Eri3
</td>
<td style="text-align:left;">
NCBI_Gene:140546,ENSEMBL:ENSMUSG00000033423
</td>
<td style="text-align:left;">
MGI:2153887
</td>
<td style="text-align:left;">
protein coding gene
</td>
<td style="text-align:left;">
exoribonuclease 3
</td>
</tr>
<tr>
<td style="text-align:left;">
4
</td>
<td style="text-align:left;">
gene
</td>
<td style="text-align:right;">
117.5701
</td>
<td style="text-align:right;">
117.5702
</td>
<td style="text-align:left;">
positive
</td>
<td style="text-align:left;">
MGI_C57BL6J_5455336
</td>
<td style="text-align:left;">
Gm25559
</td>
<td style="text-align:left;">
ENSEMBL:ENSMUSG00000087807
</td>
<td style="text-align:left;">
MGI:5455336
</td>
<td style="text-align:left;">
miRNA gene
</td>
<td style="text-align:left;">
predicted gene, 25559
</td>
</tr>
<tr>
<td style="text-align:left;">
4
</td>
<td style="text-align:left;">
pseudogene
</td>
<td style="text-align:right;">
117.5988
</td>
<td style="text-align:right;">
117.5994
</td>
<td style="text-align:left;">
positive
</td>
<td style="text-align:left;">
MGI_C57BL6J_5011377
</td>
<td style="text-align:left;">
Gm19192
</td>
<td style="text-align:left;">
NCBI_Gene:100418411
</td>
<td style="text-align:left;">
MGI:5011377
</td>
<td style="text-align:left;">
pseudogene
</td>
<td style="text-align:left;">
predicted gene, 19192
</td>
</tr>
<tr>
<td style="text-align:left;">
4
</td>
<td style="text-align:left;">
gene
</td>
<td style="text-align:right;">
117.6747
</td>
<td style="text-align:right;">
117.6823
</td>
<td style="text-align:left;">
negative
</td>
<td style="text-align:left;">
MGI_C57BL6J_1913483
</td>
<td style="text-align:left;">
Dmap1
</td>
<td style="text-align:left;">
NCBI_Gene:66233,ENSEMBL:ENSMUSG00000009640
</td>
<td style="text-align:left;">
MGI:1913483
</td>
<td style="text-align:left;">
protein coding gene
</td>
<td style="text-align:left;">
DNA methyltransferase 1-associated protein 1
</td>
</tr>
<tr>
<td style="text-align:left;">
4
</td>
<td style="text-align:left;">
gene
</td>
<td style="text-align:right;">
117.6991
</td>
<td style="text-align:right;">
117.7040
</td>
<td style="text-align:left;">
positive
</td>
<td style="text-align:left;">
MGI_C57BL6J_5593905
</td>
<td style="text-align:left;">
Gm34746
</td>
<td style="text-align:left;">
NCBI_Gene:102638104
</td>
<td style="text-align:left;">
MGI:5593905
</td>
<td style="text-align:left;">
lncRNA gene
</td>
<td style="text-align:left;">
predicted gene, 34746
</td>
</tr>
<tr>
<td style="text-align:left;">
4
</td>
<td style="text-align:left;">
gene
</td>
<td style="text-align:right;">
117.7002
</td>
<td style="text-align:right;">
117.7009
</td>
<td style="text-align:left;">
positive
</td>
<td style="text-align:left;">
MGI_C57BL6J_3650917
</td>
<td style="text-align:left;">
Gm12840
</td>
<td style="text-align:left;">
ENSEMBL:ENSMUSG00000086320
</td>
<td style="text-align:left;">
MGI:3650917
</td>
<td style="text-align:left;">
lncRNA gene
</td>
<td style="text-align:left;">
predicted gene 12840
</td>
</tr>
<tr>
<td style="text-align:left;">
4
</td>
<td style="text-align:left;">
gene
</td>
<td style="text-align:right;">
117.7244
</td>
<td style="text-align:right;">
117.7269
</td>
<td style="text-align:left;">
positive
</td>
<td style="text-align:left;">
MGI_C57BL6J_3651666
</td>
<td style="text-align:left;">
Gm12845
</td>
<td style="text-align:left;">
NCBI_Gene:100416740,ENSEMBL:ENSMUSG00000108127
</td>
<td style="text-align:left;">
MGI:3651666
</td>
<td style="text-align:left;">
protein coding gene
</td>
<td style="text-align:left;">
predicted gene 12845
</td>
</tr>
<tr>
<td style="text-align:left;">
4
</td>
<td style="text-align:left;">
pseudogene
</td>
<td style="text-align:right;">
117.7281
</td>
<td style="text-align:right;">
117.7296
</td>
<td style="text-align:left;">
negative
</td>
<td style="text-align:left;">
MGI_C57BL6J_3651053
</td>
<td style="text-align:left;">
Gm12844
</td>
<td style="text-align:left;">
NCBI_Gene:433749,ENSEMBL:ENSMUSG00000084260
</td>
<td style="text-align:left;">
MGI:3651053
</td>
<td style="text-align:left;">
pseudogene
</td>
<td style="text-align:left;">
predicted gene 12844
</td>
</tr>
<tr>
<td style="text-align:left;">
4
</td>
<td style="text-align:left;">
gene
</td>
<td style="text-align:right;">
117.7567
</td>
<td style="text-align:right;">
117.7657
</td>
<td style="text-align:left;">
negative
</td>
<td style="text-align:left;">
MGI_C57BL6J_2181068
</td>
<td style="text-align:left;">
Klf17
</td>
<td style="text-align:left;">
NCBI_Gene:75753,ENSEMBL:ENSMUSG00000048626
</td>
<td style="text-align:left;">
MGI:2181068
</td>
<td style="text-align:left;">
protein coding gene
</td>
<td style="text-align:left;">
Kruppel-like factor 17
</td>
</tr>
<tr>
<td style="text-align:left;">
4
</td>
<td style="text-align:left;">
gene
</td>
<td style="text-align:right;">
117.7817
</td>
<td style="text-align:right;">
117.7871
</td>
<td style="text-align:left;">
negative
</td>
<td style="text-align:left;">
MGI_C57BL6J_5623117
</td>
<td style="text-align:left;">
Gm40232
</td>
<td style="text-align:left;">
NCBI_Gene:105244666
</td>
<td style="text-align:left;">
MGI:5623117
</td>
<td style="text-align:left;">
lncRNA gene
</td>
<td style="text-align:left;">
predicted gene, 40232
</td>
</tr>
<tr>
<td style="text-align:left;">
4
</td>
<td style="text-align:left;">
pseudogene
</td>
<td style="text-align:right;">
117.8250
</td>
<td style="text-align:right;">
117.8257
</td>
<td style="text-align:left;">
negative
</td>
<td style="text-align:left;">
MGI_C57BL6J_3651050
</td>
<td style="text-align:left;">
Gm12842
</td>
<td style="text-align:left;">
NCBI_Gene:606540,ENSEMBL:ENSMUSG00000083507
</td>
<td style="text-align:left;">
MGI:3651050
</td>
<td style="text-align:left;">
pseudogene
</td>
<td style="text-align:left;">
predicted gene 12842
</td>
</tr>
<tr>
<td style="text-align:left;">
4
</td>
<td style="text-align:left;">
gene
</td>
<td style="text-align:right;">
117.8345
</td>
<td style="text-align:right;">
117.8752
</td>
<td style="text-align:left;">
positive
</td>
<td style="text-align:left;">
MGI_C57BL6J_95760
</td>
<td style="text-align:left;">
Slc6a9
</td>
<td style="text-align:left;">
NCBI_Gene:14664,ENSEMBL:ENSMUSG00000028542
</td>
<td style="text-align:left;">
MGI:95760
</td>
<td style="text-align:left;">
protein coding gene
</td>
<td style="text-align:left;">
solute carrier family 6 (neurotransmitter transporter, glycine), member
9
</td>
</tr>
<tr>
<td style="text-align:left;">
4
</td>
<td style="text-align:left;">
gene
</td>
<td style="text-align:right;">
117.8353
</td>
<td style="text-align:right;">
117.8686
</td>
<td style="text-align:left;">
negative
</td>
<td style="text-align:left;">
MGI_C57BL6J_4937941
</td>
<td style="text-align:left;">
Gm17114
</td>
<td style="text-align:left;">
ENSEMBL:ENSMUSG00000091237
</td>
<td style="text-align:left;">
MGI:4937941
</td>
<td style="text-align:left;">
lncRNA gene
</td>
<td style="text-align:left;">
predicted gene 17114
</td>
</tr>
<tr>
<td style="text-align:left;">
4
</td>
<td style="text-align:left;">
gene
</td>
<td style="text-align:right;">
117.8665
</td>
<td style="text-align:right;">
117.8727
</td>
<td style="text-align:left;">
negative
</td>
<td style="text-align:left;">
MGI_C57BL6J_2685874
</td>
<td style="text-align:left;">
Ccdc24
</td>
<td style="text-align:left;">
NCBI_Gene:381546,ENSEMBL:ENSMUSG00000078588
</td>
<td style="text-align:left;">
MGI:2685874
</td>
<td style="text-align:left;">
protein coding gene
</td>
<td style="text-align:left;">
coiled-coil domain containing 24
</td>
</tr>
<tr>
<td style="text-align:left;">
4
</td>
<td style="text-align:left;">
gene
</td>
<td style="text-align:right;">
117.8693
</td>
<td style="text-align:right;">
117.8835
</td>
<td style="text-align:left;">
negative
</td>
<td style="text-align:left;">
MGI_C57BL6J_1858493
</td>
<td style="text-align:left;">
B4galt2
</td>
<td style="text-align:left;">
NCBI_Gene:53418,ENSEMBL:ENSMUSG00000028541
</td>
<td style="text-align:left;">
MGI:1858493
</td>
<td style="text-align:left;">
protein coding gene
</td>
<td style="text-align:left;">
UDP-Gal:betaGlcNAc beta 1,4- galactosyltransferase, polypeptide 2
</td>
</tr>
<tr>
<td style="text-align:left;">
4
</td>
<td style="text-align:left;">
gene
</td>
<td style="text-align:right;">
117.8745
</td>
<td style="text-align:right;">
117.8793
</td>
<td style="text-align:left;">
positive
</td>
<td style="text-align:left;">
MGI_C57BL6J_5623118
</td>
<td style="text-align:left;">
Gm40233
</td>
<td style="text-align:left;">
NCBI_Gene:105244667
</td>
<td style="text-align:left;">
MGI:5623118
</td>
<td style="text-align:left;">
lncRNA gene
</td>
<td style="text-align:left;">
predicted gene, 40233
</td>
</tr>
<tr>
<td style="text-align:left;">
4
</td>
<td style="text-align:left;">
gene
</td>
<td style="text-align:right;">
117.8836
</td>
<td style="text-align:right;">
117.8862
</td>
<td style="text-align:left;">
positive
</td>
<td style="text-align:left;">
MGI_C57BL6J_3651048
</td>
<td style="text-align:left;">
Gm12841
</td>
<td style="text-align:left;">
NCBI_Gene:108168974,ENSEMBL:ENSMUSG00000085278
</td>
<td style="text-align:left;">
MGI:3651048
</td>
<td style="text-align:left;">
lncRNA gene
</td>
<td style="text-align:left;">
predicted gene 12841
</td>
</tr>
<tr>
<td style="text-align:left;">
4
</td>
<td style="text-align:left;">
gene
</td>
<td style="text-align:right;">
117.8843
</td>
<td style="text-align:right;">
117.8873
</td>
<td style="text-align:left;">
negative
</td>
<td style="text-align:left;">
MGI_C57BL6J_1890510
</td>
<td style="text-align:left;">
Atp6v0b
</td>
<td style="text-align:left;">
NCBI_Gene:114143,ENSEMBL:ENSMUSG00000033379
</td>
<td style="text-align:left;">
MGI:1890510
</td>
<td style="text-align:left;">
protein coding gene
</td>
<td style="text-align:left;">
ATPase, H+ transporting, lysosomal V0 subunit B
</td>
</tr>
<tr>
<td style="text-align:left;">
4
</td>
<td style="text-align:left;">
gene
</td>
<td style="text-align:right;">
117.8886
</td>
<td style="text-align:right;">
117.8920
</td>
<td style="text-align:left;">
negative
</td>
<td style="text-align:left;">
MGI_C57BL6J_1914978
</td>
<td style="text-align:left;">
Dph2
</td>
<td style="text-align:left;">
NCBI_Gene:67728,ENSEMBL:ENSMUSG00000028540
</td>
<td style="text-align:left;">
MGI:1914978
</td>
<td style="text-align:left;">
protein coding gene
</td>
<td style="text-align:left;">
DPH2 homolog
</td>
</tr>
<tr>
<td style="text-align:left;">
4
</td>
<td style="text-align:left;">
gene
</td>
<td style="text-align:right;">
117.8945
</td>
<td style="text-align:right;">
117.9152
</td>
<td style="text-align:left;">
negative
</td>
<td style="text-align:left;">
MGI_C57BL6J_2385205
</td>
<td style="text-align:left;">
Ipo13
</td>
<td style="text-align:left;">
NCBI_Gene:230673,ENSEMBL:ENSMUSG00000033365
</td>
<td style="text-align:left;">
MGI:2385205
</td>
<td style="text-align:left;">
protein coding gene
</td>
<td style="text-align:left;">
importin 13
</td>
</tr>
<tr>
<td style="text-align:left;">
4
</td>
<td style="text-align:left;">
gene
</td>
<td style="text-align:right;">
117.9259
</td>
<td style="text-align:right;">
117.9308
</td>
<td style="text-align:left;">
negative
</td>
<td style="text-align:left;">
MGI_C57BL6J_1333791
</td>
<td style="text-align:left;">
Artn
</td>
<td style="text-align:left;">
NCBI_Gene:11876,ENSEMBL:ENSMUSG00000028539
</td>
<td style="text-align:left;">
MGI:1333791
</td>
<td style="text-align:left;">
protein coding gene
</td>
<td style="text-align:left;">
artemin
</td>
</tr>
<tr>
<td style="text-align:left;">
4
</td>
<td style="text-align:left;">
gene
</td>
<td style="text-align:right;">
117.9322
</td>
<td style="text-align:right;">
118.1349
</td>
<td style="text-align:left;">
negative
</td>
<td style="text-align:left;">
MGI_C57BL6J_1316659
</td>
<td style="text-align:left;">
St3gal3
</td>
<td style="text-align:left;">
NCBI_Gene:20441,ENSEMBL:ENSMUSG00000028538
</td>
<td style="text-align:left;">
MGI:1316659
</td>
<td style="text-align:left;">
protein coding gene
</td>
<td style="text-align:left;">
ST3 beta-galactoside alpha-2,3-sialyltransferase 3
</td>
</tr>
<tr>
<td style="text-align:left;">
4
</td>
<td style="text-align:left;">
gene
</td>
<td style="text-align:right;">
117.9724
</td>
<td style="text-align:right;">
117.9953
</td>
<td style="text-align:left;">
positive
</td>
<td style="text-align:left;">
MGI_C57BL6J_1925865
</td>
<td style="text-align:left;">
9530034E10Rik
</td>
<td style="text-align:left;">
NCBI_Gene:78615,ENSEMBL:ENSMUSG00000086554
</td>
<td style="text-align:left;">
MGI:1925865
</td>
<td style="text-align:left;">
lncRNA gene
</td>
<td style="text-align:left;">
RIKEN cDNA 9530034E10 gene
</td>
</tr>
<tr>
<td style="text-align:left;">
4
</td>
<td style="text-align:left;">
gene
</td>
<td style="text-align:right;">
118.1370
</td>
<td style="text-align:right;">
118.1800
</td>
<td style="text-align:left;">
negative
</td>
<td style="text-align:left;">
MGI_C57BL6J_2446210
</td>
<td style="text-align:left;">
Kdm4a
</td>
<td style="text-align:left;">
NCBI_Gene:230674,ENSEMBL:ENSMUSG00000033326
</td>
<td style="text-align:left;">
MGI:2446210
</td>
<td style="text-align:left;">
protein coding gene
</td>
<td style="text-align:left;">
lysine (K)-specific demethylase 4A
</td>
</tr>
<tr>
<td style="text-align:left;">
4
</td>
<td style="text-align:left;">
gene
</td>
<td style="text-align:right;">
118.1641
</td>
<td style="text-align:right;">
118.1642
</td>
<td style="text-align:left;">
positive
</td>
<td style="text-align:left;">
MGI_C57BL6J_5453159
</td>
<td style="text-align:left;">
Gm23382
</td>
<td style="text-align:left;">
ENSEMBL:ENSMUSG00000088303
</td>
<td style="text-align:left;">
MGI:5453159
</td>
<td style="text-align:left;">
snoRNA gene
</td>
<td style="text-align:left;">
predicted gene, 23382
</td>
</tr>
<tr>
<td style="text-align:left;">
4
</td>
<td style="text-align:left;">
gene
</td>
<td style="text-align:right;">
118.2082
</td>
<td style="text-align:right;">
118.2914
</td>
<td style="text-align:left;">
negative
</td>
<td style="text-align:left;">
MGI_C57BL6J_102695
</td>
<td style="text-align:left;">
Ptprf
</td>
<td style="text-align:left;">
NCBI_Gene:19268,ENSEMBL:ENSMUSG00000033295
</td>
<td style="text-align:left;">
MGI:102695
</td>
<td style="text-align:left;">
protein coding gene
</td>
<td style="text-align:left;">
protein tyrosine phosphatase, receptor type, F
</td>
</tr>
<tr>
<td style="text-align:left;">
4
</td>
<td style="text-align:left;">
gene
</td>
<td style="text-align:right;">
118.2105
</td>
<td style="text-align:right;">
118.2106
</td>
<td style="text-align:left;">
negative
</td>
<td style="text-align:left;">
MGI_C57BL6J_5531012
</td>
<td style="text-align:left;">
Mir7226
</td>
<td style="text-align:left;">
miRBase:MI0023721,NCBI_Gene:102466820,ENSEMBL:ENSMUSG00000098596
</td>
<td style="text-align:left;">
MGI:5531012
</td>
<td style="text-align:left;">
miRNA gene
</td>
<td style="text-align:left;">
microRNA 7226
</td>
</tr>
<tr>
<td style="text-align:left;">
4
</td>
<td style="text-align:left;">
gene
</td>
<td style="text-align:right;">
118.2523
</td>
<td style="text-align:right;">
118.2575
</td>
<td style="text-align:left;">
negative
</td>
<td style="text-align:left;">
MGI_C57BL6J_5623120
</td>
<td style="text-align:left;">
Gm40235
</td>
<td style="text-align:left;">
NCBI_Gene:105244669
</td>
<td style="text-align:left;">
MGI:5623120
</td>
<td style="text-align:left;">
lncRNA gene
</td>
<td style="text-align:left;">
predicted gene, 40235
</td>
</tr>
<tr>
<td style="text-align:left;">
4
</td>
<td style="text-align:left;">
pseudogene
</td>
<td style="text-align:right;">
118.3600
</td>
<td style="text-align:right;">
118.3627
</td>
<td style="text-align:left;">
positive
</td>
<td style="text-align:left;">
MGI_C57BL6J_1915430
</td>
<td style="text-align:left;">
Hyi
</td>
<td style="text-align:left;">
NCBI_Gene:68180,ENSEMBL:ENSMUSG00000006395
</td>
<td style="text-align:left;">
MGI:1915430
</td>
<td style="text-align:left;">
polymorphic pseudogene
</td>
<td style="text-align:left;">
hydroxypyruvate isomerase (putative)
</td>
</tr>
<tr>
<td style="text-align:left;">
4
</td>
<td style="text-align:left;">
gene
</td>
<td style="text-align:right;">
118.3627
</td>
<td style="text-align:right;">
118.4093
</td>
<td style="text-align:left;">
negative
</td>
<td style="text-align:left;">
MGI_C57BL6J_3033336
</td>
<td style="text-align:left;">
Szt2
</td>
<td style="text-align:left;">
NCBI_Gene:230676,ENSEMBL:ENSMUSG00000033253
</td>
<td style="text-align:left;">
MGI:3033336
</td>
<td style="text-align:left;">
protein coding gene
</td>
<td style="text-align:left;">
SZT2 subunit of KICSTOR complex
</td>
</tr>
<tr>
<td style="text-align:left;">
4
</td>
<td style="text-align:left;">
gene
</td>
<td style="text-align:right;">
118.4093
</td>
<td style="text-align:right;">
118.4158
</td>
<td style="text-align:left;">
positive
</td>
<td style="text-align:left;">
MGI_C57BL6J_1915269
</td>
<td style="text-align:left;">
Med8
</td>
<td style="text-align:left;">
NCBI_Gene:80509,ENSEMBL:ENSMUSG00000006392
</td>
<td style="text-align:left;">
MGI:1915269
</td>
<td style="text-align:left;">
protein coding gene
</td>
<td style="text-align:left;">
mediator complex subunit 8
</td>
</tr>
<tr>
<td style="text-align:left;">
4
</td>
<td style="text-align:left;">
gene
</td>
<td style="text-align:right;">
118.4281
</td>
<td style="text-align:right;">
118.4330
</td>
<td style="text-align:left;">
positive
</td>
<td style="text-align:left;">
MGI_C57BL6J_1858959
</td>
<td style="text-align:left;">
Elovl1
</td>
<td style="text-align:left;">
NCBI_Gene:54325,ENSEMBL:ENSMUSG00000006390
</td>
<td style="text-align:left;">
MGI:1858959
</td>
<td style="text-align:left;">
protein coding gene
</td>
<td style="text-align:left;">
elongation of very long chain fatty acids (FEN1/Elo2, SUR4/Elo3,
yeast)-like 1
</td>
</tr>
<tr>
<td style="text-align:left;">
4
</td>
<td style="text-align:left;">
gene
</td>
<td style="text-align:right;">
118.4329
</td>
<td style="text-align:right;">
118.4374
</td>
<td style="text-align:left;">
negative
</td>
<td style="text-align:left;">
MGI_C57BL6J_1859866
</td>
<td style="text-align:left;">
Cdc20
</td>
<td style="text-align:left;">
NCBI_Gene:107995,ENSEMBL:ENSMUSG00000006398
</td>
<td style="text-align:left;">
MGI:1859866
</td>
<td style="text-align:left;">
protein coding gene
</td>
<td style="text-align:left;">
cell division cycle 20
</td>
</tr>
<tr>
<td style="text-align:left;">
4
</td>
<td style="text-align:left;">
pseudogene
</td>
<td style="text-align:right;">
118.4410
</td>
<td style="text-align:right;">
118.4412
</td>
<td style="text-align:left;">
negative
</td>
<td style="text-align:left;">
MGI_C57BL6J_3651259
</td>
<td style="text-align:left;">
Gm12858
</td>
<td style="text-align:left;">
ENSEMBL:ENSMUSG00000081921
</td>
<td style="text-align:left;">
MGI:3651259
</td>
<td style="text-align:left;">
pseudogene
</td>
<td style="text-align:left;">
predicted gene 12858
</td>
</tr>
<tr>
<td style="text-align:left;">
4
</td>
<td style="text-align:left;">
gene
</td>
<td style="text-align:right;">
118.4424
</td>
<td style="text-align:right;">
118.4575
</td>
<td style="text-align:left;">
negative
</td>
<td style="text-align:left;">
MGI_C57BL6J_97076
</td>
<td style="text-align:left;">
Mpl
</td>
<td style="text-align:left;">
NCBI_Gene:17480,ENSEMBL:ENSMUSG00000006389
</td>
<td style="text-align:left;">
MGI:97076
</td>
<td style="text-align:left;">
protein coding gene
</td>
<td style="text-align:left;">
myeloproliferative leukemia virus oncogene
</td>
</tr>
<tr>
<td style="text-align:left;">
4
</td>
<td style="text-align:left;">
gene
</td>
<td style="text-align:right;">
118.4712
</td>
<td style="text-align:right;">
118.4901
</td>
<td style="text-align:left;">
negative
</td>
<td style="text-align:left;">
MGI_C57BL6J_99906
</td>
<td style="text-align:left;">
Tie1
</td>
<td style="text-align:left;">
NCBI_Gene:21846,ENSEMBL:ENSMUSG00000033191
</td>
<td style="text-align:left;">
MGI:99906
</td>
<td style="text-align:left;">
protein coding gene
</td>
<td style="text-align:left;">
tyrosine kinase with immunoglobulin-like and EGF-like domains 1
</td>
</tr>
<tr>
<td style="text-align:left;">
4
</td>
<td style="text-align:left;">
pseudogene
</td>
<td style="text-align:right;">
118.4907
</td>
<td style="text-align:right;">
118.4976
</td>
<td style="text-align:left;">
negative
</td>
<td style="text-align:left;">
MGI_C57BL6J_3651649
</td>
<td style="text-align:left;">
Gm12859
</td>
<td style="text-align:left;">
NCBI_Gene:545675,ENSEMBL:ENSMUSG00000080857
</td>
<td style="text-align:left;">
MGI:3651649
</td>
<td style="text-align:left;">
pseudogene
</td>
<td style="text-align:left;">
predicted gene 12859
</td>
</tr>
<tr>
<td style="text-align:left;">
4
</td>
<td style="text-align:left;">
pseudogene
</td>
<td style="text-align:right;">
118.5220
</td>
<td style="text-align:right;">
118.5227
</td>
<td style="text-align:left;">
positive
</td>
<td style="text-align:left;">
MGI_C57BL6J_3652230
</td>
<td style="text-align:left;">
Gm12857
</td>
<td style="text-align:left;">
NCBI_Gene:666596,ENSEMBL:ENSMUSG00000081233
</td>
<td style="text-align:left;">
MGI:3652230
</td>
<td style="text-align:left;">
pseudogene
</td>
<td style="text-align:left;">
predicted gene 12857
</td>
</tr>
<tr>
<td style="text-align:left;">
4
</td>
<td style="text-align:left;">
gene
</td>
<td style="text-align:right;">
118.5270
</td>
<td style="text-align:right;">
118.5302
</td>
<td style="text-align:left;">
positive
</td>
<td style="text-align:left;">
MGI_C57BL6J_1913701
</td>
<td style="text-align:left;">
2610528J11Rik
</td>
<td style="text-align:left;">
NCBI_Gene:66451,ENSEMBL:ENSMUSG00000028536
</td>
<td style="text-align:left;">
MGI:1913701
</td>
<td style="text-align:left;">
protein coding gene
</td>
<td style="text-align:left;">
RIKEN cDNA 2610528J11 gene
</td>
</tr>
<tr>
<td style="text-align:left;">
4
</td>
<td style="text-align:left;">
gene
</td>
<td style="text-align:right;">
118.5409
</td>
<td style="text-align:right;">
118.5450
</td>
<td style="text-align:left;">
negative
</td>
<td style="text-align:left;">
MGI_C57BL6J_1923409
</td>
<td style="text-align:left;">
Tmem125
</td>
<td style="text-align:left;">
NCBI_Gene:230678,ENSEMBL:ENSMUSG00000050854
</td>
<td style="text-align:left;">
MGI:1923409
</td>
<td style="text-align:left;">
protein coding gene
</td>
<td style="text-align:left;">
transmembrane protein 125
</td>
</tr>
<tr>
<td style="text-align:left;">
4
</td>
<td style="text-align:left;">
gene
</td>
<td style="text-align:right;">
118.5471
</td>
<td style="text-align:right;">
118.5483
</td>
<td style="text-align:left;">
positive
</td>
<td style="text-align:left;">
MGI_C57BL6J_5623122
</td>
<td style="text-align:left;">
Gm40237
</td>
<td style="text-align:left;">
NCBI_Gene:105244671
</td>
<td style="text-align:left;">
MGI:5623122
</td>
<td style="text-align:left;">
lncRNA gene
</td>
<td style="text-align:left;">
predicted gene, 40237
</td>
</tr>
<tr>
<td style="text-align:left;">
4
</td>
<td style="text-align:left;">
gene
</td>
<td style="text-align:right;">
118.5546
</td>
<td style="text-align:right;">
118.6208
</td>
<td style="text-align:left;">
negative
</td>
<td style="text-align:left;">
MGI_C57BL6J_2686209
</td>
<td style="text-align:left;">
Cfap57
</td>
<td style="text-align:left;">
NCBI_Gene:68625,ENSEMBL:ENSMUSG00000028730
</td>
<td style="text-align:left;">
MGI:2686209
</td>
<td style="text-align:left;">
protein coding gene
</td>
<td style="text-align:left;">
cilia and flagella associated protein 57
</td>
</tr>
<tr>
<td style="text-align:left;">
4
</td>
<td style="text-align:left;">
gene
</td>
<td style="text-align:right;">
118.6208
</td>
<td style="text-align:right;">
118.6278
</td>
<td style="text-align:left;">
positive
</td>
<td style="text-align:left;">
MGI_C57BL6J_1916322
</td>
<td style="text-align:left;">
Ebna1bp2
</td>
<td style="text-align:left;">
NCBI_Gene:69072,ENSEMBL:ENSMUSG00000028729
</td>
<td style="text-align:left;">
MGI:1916322
</td>
<td style="text-align:left;">
protein coding gene
</td>
<td style="text-align:left;">
EBNA1 binding protein 2
</td>
</tr>
<tr>
<td style="text-align:left;">
4
</td>
<td style="text-align:left;">
gene
</td>
<td style="text-align:right;">
118.6264
</td>
<td style="text-align:right;">
118.6319
</td>
<td style="text-align:left;">
positive
</td>
<td style="text-align:left;">
MGI_C57BL6J_1277206
</td>
<td style="text-align:left;">
D4Ertd617e
</td>
<td style="text-align:left;">
NCBI_Gene:100041290
</td>
<td style="text-align:left;">
MGI:1277206
</td>
<td style="text-align:left;">
lncRNA gene
</td>
<td style="text-align:left;">
DNA segment, Chr 4, ERATO Doi 617, expressed
</td>
</tr>
<tr>
<td style="text-align:left;">
4
</td>
<td style="text-align:left;">
gene
</td>
<td style="text-align:right;">
118.6310
</td>
<td style="text-align:right;">
118.6378
</td>
<td style="text-align:left;">
negative
</td>
<td style="text-align:left;">
MGI_C57BL6J_3650185
</td>
<td style="text-align:left;">
Gm12853
</td>
<td style="text-align:left;">
NCBI_Gene:102638941,ENSEMBL:ENSMUSG00000085292
</td>
<td style="text-align:left;">
MGI:3650185
</td>
<td style="text-align:left;">
lncRNA gene
</td>
<td style="text-align:left;">
predicted gene 12853
</td>
</tr>
<tr>
<td style="text-align:left;">
4
</td>
<td style="text-align:left;">
gene
</td>
<td style="text-align:right;">
118.6590
</td>
<td style="text-align:right;">
118.6680
</td>
<td style="text-align:left;">
positive
</td>
<td style="text-align:left;">
MGI_C57BL6J_1333887
</td>
<td style="text-align:left;">
Olfr62
</td>
<td style="text-align:left;">
NCBI_Gene:18363,ENSEMBL:ENSMUSG00000043698
</td>
<td style="text-align:left;">
MGI:1333887
</td>
<td style="text-align:left;">
protein coding gene
</td>
<td style="text-align:left;">
olfactory receptor 62
</td>
</tr>
<tr>
<td style="text-align:left;">
4
</td>
<td style="text-align:left;">
pseudogene
</td>
<td style="text-align:right;">
118.6726
</td>
<td style="text-align:right;">
118.6735
</td>
<td style="text-align:left;">
negative
</td>
<td style="text-align:left;">
MGI_C57BL6J_3031177
</td>
<td style="text-align:left;">
Olfr1343-ps1
</td>
<td style="text-align:left;">
NCBI_Gene:258707,ENSEMBL:ENSMUSG00000086521
</td>
<td style="text-align:left;">
MGI:3031177
</td>
<td style="text-align:left;">
pseudogene
</td>
<td style="text-align:left;">
olfactory receptor 1343, pseudogene 1
</td>
</tr>
<tr>
<td style="text-align:left;">
4
</td>
<td style="text-align:left;">
gene
</td>
<td style="text-align:right;">
118.6875
</td>
<td style="text-align:right;">
118.6930
</td>
<td style="text-align:left;">
negative
</td>
<td style="text-align:left;">
MGI_C57BL6J_3031176
</td>
<td style="text-align:left;">
Olfr1342
</td>
<td style="text-align:left;">
NCBI_Gene:258708,ENSEMBL:ENSMUSG00000043383
</td>
<td style="text-align:left;">
MGI:3031176
</td>
<td style="text-align:left;">
protein coding gene
</td>
<td style="text-align:left;">
olfactory receptor 1342
</td>
</tr>
<tr>
<td style="text-align:left;">
4
</td>
<td style="text-align:left;">
gene
</td>
<td style="text-align:right;">
118.7035
</td>
<td style="text-align:right;">
118.7134
</td>
<td style="text-align:left;">
positive
</td>
<td style="text-align:left;">
MGI_C57BL6J_3031175
</td>
<td style="text-align:left;">
Olfr1341
</td>
<td style="text-align:left;">
NCBI_Gene:258852,ENSEMBL:ENSMUSG00000046790
</td>
<td style="text-align:left;">
MGI:3031175
</td>
<td style="text-align:left;">
protein coding gene
</td>
<td style="text-align:left;">
olfactory receptor 1341
</td>
</tr>
<tr>
<td style="text-align:left;">
4
</td>
<td style="text-align:left;">
gene
</td>
<td style="text-align:right;">
118.7201
</td>
<td style="text-align:right;">
118.7294
</td>
<td style="text-align:left;">
positive
</td>
<td style="text-align:left;">
MGI_C57BL6J_3031174
</td>
<td style="text-align:left;">
Olfr1340
</td>
<td style="text-align:left;">
NCBI_Gene:258301,ENSEMBL:ENSMUSG00000070821
</td>
<td style="text-align:left;">
MGI:3031174
</td>
<td style="text-align:left;">
protein coding gene
</td>
<td style="text-align:left;">
olfactory receptor 1340
</td>
</tr>
<tr>
<td style="text-align:left;">
4
</td>
<td style="text-align:left;">
pseudogene
</td>
<td style="text-align:right;">
118.7228
</td>
<td style="text-align:right;">
118.7255
</td>
<td style="text-align:left;">
negative
</td>
<td style="text-align:left;">
MGI_C57BL6J_5439393
</td>
<td style="text-align:left;">
Gm21942
</td>
<td style="text-align:left;">
NCBI_Gene:257848
</td>
<td style="text-align:left;">
MGI:5439393
</td>
<td style="text-align:left;">
pseudogene
</td>
<td style="text-align:left;">
predicted gene, 21942
</td>
</tr>
<tr>
<td style="text-align:left;">
4
</td>
<td style="text-align:left;">
gene
</td>
<td style="text-align:right;">
118.7322
</td>
<td style="text-align:right;">
118.7369
</td>
<td style="text-align:left;">
positive
</td>
<td style="text-align:left;">
MGI_C57BL6J_3031173
</td>
<td style="text-align:left;">
Olfr1339
</td>
<td style="text-align:left;">
NCBI_Gene:258851,ENSEMBL:ENSMUSG00000070820
</td>
<td style="text-align:left;">
MGI:3031173
</td>
<td style="text-align:left;">
protein coding gene
</td>
<td style="text-align:left;">
olfactory receptor 1339
</td>
</tr>
<tr>
<td style="text-align:left;">
4
</td>
<td style="text-align:left;">
gene
</td>
<td style="text-align:right;">
118.7520
</td>
<td style="text-align:right;">
118.7587
</td>
<td style="text-align:left;">
negative
</td>
<td style="text-align:left;">
MGI_C57BL6J_3031172
</td>
<td style="text-align:left;">
Olfr1338
</td>
<td style="text-align:left;">
NCBI_Gene:258259,ENSEMBL:ENSMUSG00000095218
</td>
<td style="text-align:left;">
MGI:3031172
</td>
<td style="text-align:left;">
protein coding gene
</td>
<td style="text-align:left;">
olfactory receptor 1338
</td>
</tr>
<tr>
<td style="text-align:left;">
4
</td>
<td style="text-align:left;">
gene
</td>
<td style="text-align:right;">
118.7816
</td>
<td style="text-align:right;">
118.7826
</td>
<td style="text-align:left;">
negative
</td>
<td style="text-align:left;">
MGI_C57BL6J_3031171
</td>
<td style="text-align:left;">
Olfr1337
</td>
<td style="text-align:left;">
NCBI_Gene:258306,ENSEMBL:ENSMUSG00000111159
</td>
<td style="text-align:left;">
MGI:3031171
</td>
<td style="text-align:left;">
protein coding gene
</td>
<td style="text-align:left;">
olfactory receptor 1337
</td>
</tr>
<tr>
<td style="text-align:left;">
4
</td>
<td style="text-align:left;">
pseudogene
</td>
<td style="text-align:right;">
118.7910
</td>
<td style="text-align:right;">
118.7919
</td>
<td style="text-align:left;">
negative
</td>
<td style="text-align:left;">
MGI_C57BL6J_3649662
</td>
<td style="text-align:left;">
Gm12855
</td>
<td style="text-align:left;">
NCBI_Gene:257847,ENSEMBL:ENSMUSG00000081474
</td>
<td style="text-align:left;">
MGI:3649662
</td>
<td style="text-align:left;">
pseudogene
</td>
<td style="text-align:left;">
predicted gene 12855
</td>
</tr>
<tr>
<td style="text-align:left;">
4
</td>
<td style="text-align:left;">
gene
</td>
<td style="text-align:right;">
118.8088
</td>
<td style="text-align:right;">
118.8099
</td>
<td style="text-align:left;">
negative
</td>
<td style="text-align:left;">
MGI_C57BL6J_3031168
</td>
<td style="text-align:left;">
Olfr1335
</td>
<td style="text-align:left;">
NCBI_Gene:435804,ENSEMBL:ENSMUSG00000066061
</td>
<td style="text-align:left;">
MGI:3031168
</td>
<td style="text-align:left;">
protein coding gene
</td>
<td style="text-align:left;">
olfactory receptor 1335
</td>
</tr>
<tr>
<td style="text-align:left;">
4
</td>
<td style="text-align:left;">
gene
</td>
<td style="text-align:right;">
118.8277
</td>
<td style="text-align:right;">
118.8363
</td>
<td style="text-align:left;">
negative
</td>
<td style="text-align:left;">
MGI_C57BL6J_3031167
</td>
<td style="text-align:left;">
Olfr1333
</td>
<td style="text-align:left;">
NCBI_Gene:258265,ENSEMBL:ENSMUSG00000110947
</td>
<td style="text-align:left;">
MGI:3031167
</td>
<td style="text-align:left;">
protein coding gene
</td>
<td style="text-align:left;">
olfactory receptor 1333
</td>
</tr>
<tr>
<td style="text-align:left;">
4
</td>
<td style="text-align:left;">
pseudogene
</td>
<td style="text-align:right;">
118.8541
</td>
<td style="text-align:right;">
118.8550
</td>
<td style="text-align:left;">
positive
</td>
<td style="text-align:left;">
MGI_C57BL6J_3031166
</td>
<td style="text-align:left;">
Olfr1332-ps1
</td>
<td style="text-align:left;">
NCBI_Gene:257952,ENSEMBL:ENSMUSG00000083336
</td>
<td style="text-align:left;">
MGI:3031166
</td>
<td style="text-align:left;">
pseudogene
</td>
<td style="text-align:left;">
olfactory receptor 1332, pseudogene 1
</td>
</tr>
<tr>
<td style="text-align:left;">
4
</td>
<td style="text-align:left;">
gene
</td>
<td style="text-align:right;">
118.8646
</td>
<td style="text-align:right;">
118.8717
</td>
<td style="text-align:left;">
positive
</td>
<td style="text-align:left;">
MGI_C57BL6J_3031165
</td>
<td style="text-align:left;">
Olfr1331
</td>
<td style="text-align:left;">
NCBI_Gene:258159,ENSEMBL:ENSMUSG00000073769
</td>
<td style="text-align:left;">
MGI:3031165
</td>
<td style="text-align:left;">
protein coding gene
</td>
<td style="text-align:left;">
olfactory receptor 1331
</td>
</tr>
<tr>
<td style="text-align:left;">
4
</td>
<td style="text-align:left;">
gene
</td>
<td style="text-align:right;">
118.8886
</td>
<td style="text-align:right;">
118.8996
</td>
<td style="text-align:left;">
positive
</td>
<td style="text-align:left;">
MGI_C57BL6J_3031164
</td>
<td style="text-align:left;">
Olfr1330
</td>
<td style="text-align:left;">
NCBI_Gene:258331,ENSEMBL:ENSMUSG00000073768
</td>
<td style="text-align:left;">
MGI:3031164
</td>
<td style="text-align:left;">
protein coding gene
</td>
<td style="text-align:left;">
olfactory receptor 1330
</td>
</tr>
<tr>
<td style="text-align:left;">
4
</td>
<td style="text-align:left;">
gene
</td>
<td style="text-align:right;">
118.8980
</td>
<td style="text-align:right;">
118.9974
</td>
<td style="text-align:left;">
negative
</td>
<td style="text-align:left;">
MGI_C57BL6J_3651214
</td>
<td style="text-align:left;">
Gm12865
</td>
<td style="text-align:left;">
NCBI_Gene:105244672,ENSEMBL:ENSMUSG00000087238
</td>
<td style="text-align:left;">
MGI:3651214
</td>
<td style="text-align:left;">
lncRNA gene
</td>
<td style="text-align:left;">
predicted gene 12865
</td>
</tr>
<tr>
<td style="text-align:left;">
4
</td>
<td style="text-align:left;">
gene
</td>
<td style="text-align:right;">
118.9165
</td>
<td style="text-align:right;">
118.9175
</td>
<td style="text-align:left;">
negative
</td>
<td style="text-align:left;">
MGI_C57BL6J_3031163
</td>
<td style="text-align:left;">
Olfr1329
</td>
<td style="text-align:left;">
NCBI_Gene:258214,ENSEMBL:ENSMUSG00000096705
</td>
<td style="text-align:left;">
MGI:3031163
</td>
<td style="text-align:left;">
protein coding gene
</td>
<td style="text-align:left;">
olfactory receptor 1329
</td>
</tr>
<tr>
<td style="text-align:left;">
4
</td>
<td style="text-align:left;">
gene
</td>
<td style="text-align:right;">
118.9301
</td>
<td style="text-align:right;">
118.9386
</td>
<td style="text-align:left;">
negative
</td>
<td style="text-align:left;">
MGI_C57BL6J_3031162
</td>
<td style="text-align:left;">
Olfr1328
</td>
<td style="text-align:left;">
NCBI_Gene:258394,ENSEMBL:ENSMUSG00000111259
</td>
<td style="text-align:left;">
MGI:3031162
</td>
<td style="text-align:left;">
protein coding gene
</td>
<td style="text-align:left;">
olfactory receptor 1328
</td>
</tr>
<tr>
<td style="text-align:left;">
4
</td>
<td style="text-align:left;">
pseudogene
</td>
<td style="text-align:right;">
118.9421
</td>
<td style="text-align:right;">
118.9434
</td>
<td style="text-align:left;">
negative
</td>
<td style="text-align:left;">
MGI_C57BL6J_3649664
</td>
<td style="text-align:left;">
Gm12856
</td>
<td style="text-align:left;">
NCBI_Gene:100415963,ENSEMBL:ENSMUSG00000084133
</td>
<td style="text-align:left;">
MGI:3649664
</td>
<td style="text-align:left;">
pseudogene
</td>
<td style="text-align:left;">
predicted gene 12856
</td>
</tr>
<tr>
<td style="text-align:left;">
4
</td>
<td style="text-align:left;">
gene
</td>
<td style="text-align:right;">
118.9543
</td>
<td style="text-align:right;">
118.9549
</td>
<td style="text-align:left;">
negative
</td>
<td style="text-align:left;">
MGI_C57BL6J_3651212
</td>
<td style="text-align:left;">
Gm12863
</td>
<td style="text-align:left;">
NCBI_Gene:100038571,ENSEMBL:ENSMUSG00000073765
</td>
<td style="text-align:left;">
MGI:3651212
</td>
<td style="text-align:left;">
lncRNA gene
</td>
<td style="text-align:left;">
predicted gene 12863
</td>
</tr>
<tr>
<td style="text-align:left;">
4
</td>
<td style="text-align:left;">
gene
</td>
<td style="text-align:right;">
118.9616
</td>
<td style="text-align:right;">
118.9689
</td>
<td style="text-align:left;">
positive
</td>
<td style="text-align:left;">
MGI_C57BL6J_2140628
</td>
<td style="text-align:left;">
Lao1
</td>
<td style="text-align:left;">
NCBI_Gene:100470,ENSEMBL:ENSMUSG00000024903
</td>
<td style="text-align:left;">
MGI:2140628
</td>
<td style="text-align:left;">
protein coding gene
</td>
<td style="text-align:left;">
L-amino acid oxidase 1
</td>
</tr>
<tr>
<td style="text-align:left;">
4
</td>
<td style="text-align:left;">
gene
</td>
<td style="text-align:right;">
118.9713
</td>
<td style="text-align:right;">
118.9995
</td>
<td style="text-align:left;">
positive
</td>
<td style="text-align:left;">
MGI_C57BL6J_3651891
</td>
<td style="text-align:left;">
Gm12861
</td>
<td style="text-align:left;">
ENSEMBL:ENSMUSG00000085895
</td>
<td style="text-align:left;">
MGI:3651891
</td>
<td style="text-align:left;">
lncRNA gene
</td>
<td style="text-align:left;">
predicted gene 12861
</td>
</tr>
<tr>
<td style="text-align:left;">
4
</td>
<td style="text-align:left;">
gene
</td>
<td style="text-align:right;">
118.9803
</td>
<td style="text-align:right;">
118.9804
</td>
<td style="text-align:left;">
positive
</td>
<td style="text-align:left;">
MGI_C57BL6J_5454129
</td>
<td style="text-align:left;">
Gm24352
</td>
<td style="text-align:left;">
ENSEMBL:ENSMUSG00000089591
</td>
<td style="text-align:left;">
MGI:5454129
</td>
<td style="text-align:left;">
snoRNA gene
</td>
<td style="text-align:left;">
predicted gene, 24352
</td>
</tr>
<tr>
<td style="text-align:left;">
4
</td>
<td style="text-align:left;">
pseudogene
</td>
<td style="text-align:right;">
118.9870
</td>
<td style="text-align:right;">
118.9882
</td>
<td style="text-align:left;">
positive
</td>
<td style="text-align:left;">
MGI_C57BL6J_3649273
</td>
<td style="text-align:left;">
Gm12864
</td>
<td style="text-align:left;">
NCBI_Gene:666703,ENSEMBL:ENSMUSG00000082143
</td>
<td style="text-align:left;">
MGI:3649273
</td>
<td style="text-align:left;">
pseudogene
</td>
<td style="text-align:left;">
predicted gene 12864
</td>
</tr>
<tr>
<td style="text-align:left;">
4
</td>
<td style="text-align:left;">
gene
</td>
<td style="text-align:right;">
118.9997
</td>
<td style="text-align:right;">
119.0005
</td>
<td style="text-align:left;">
negative
</td>
<td style="text-align:left;">
MGI_C57BL6J_3651209
</td>
<td style="text-align:left;">
Gm12862
</td>
<td style="text-align:left;">
ENSEMBL:ENSMUSG00000087464
</td>
<td style="text-align:left;">
MGI:3651209
</td>
<td style="text-align:left;">
lncRNA gene
</td>
<td style="text-align:left;">
predicted gene 12862
</td>
</tr>
<tr>
<td style="text-align:left;">
4
</td>
<td style="text-align:left;">
gene
</td>
<td style="text-align:right;">
119.0507
</td>
<td style="text-align:right;">
119.0620
</td>
<td style="text-align:left;">
positive
</td>
<td style="text-align:left;">
MGI_C57BL6J_5428673
</td>
<td style="text-align:left;">
Lincred2
</td>
<td style="text-align:left;">
NCBI_Gene:100996932
</td>
<td style="text-align:left;">
MGI:5428673
</td>
<td style="text-align:left;">
lncRNA gene
</td>
<td style="text-align:left;">
long intergenic non-protein coding RNA of erythroid differentiation 2
</td>
</tr>
<tr>
<td style="text-align:left;">
4
</td>
<td style="text-align:left;">
gene
</td>
<td style="text-align:right;">
119.0548
</td>
<td style="text-align:right;">
119.0691
</td>
<td style="text-align:left;">
negative
</td>
<td style="text-align:left;">
MGI_C57BL6J_3649462
</td>
<td style="text-align:left;">
Gm12866
</td>
<td style="text-align:left;">
NCBI_Gene:433751,ENSEMBL:ENSMUSG00000066060
</td>
<td style="text-align:left;">
MGI:3649462
</td>
<td style="text-align:left;">
lncRNA gene
</td>
<td style="text-align:left;">
predicted gene 12866
</td>
</tr>
<tr>
<td style="text-align:left;">
4
</td>
<td style="text-align:left;">
gene
</td>
<td style="text-align:right;">
119.0730
</td>
<td style="text-align:right;">
119.0844
</td>
<td style="text-align:left;">
positive
</td>
<td style="text-align:left;">
MGI_C57BL6J_5594669
</td>
<td style="text-align:left;">
Gm35510
</td>
<td style="text-align:left;">
NCBI_Gene:102639122
</td>
<td style="text-align:left;">
MGI:5594669
</td>
<td style="text-align:left;">
lncRNA gene
</td>
<td style="text-align:left;">
predicted gene, 35510
</td>
</tr>
<tr>
<td style="text-align:left;">
4
</td>
<td style="text-align:left;">
gene
</td>
<td style="text-align:right;">
119.1087
</td>
<td style="text-align:right;">
119.1380
</td>
<td style="text-align:left;">
positive
</td>
<td style="text-align:left;">
MGI_C57BL6J_95755
</td>
<td style="text-align:left;">
Slc2a1
</td>
<td style="text-align:left;">
NCBI_Gene:20525,ENSEMBL:ENSMUSG00000028645
</td>
<td style="text-align:left;">
MGI:95755
</td>
<td style="text-align:left;">
protein coding gene
</td>
<td style="text-align:left;">
solute carrier family 2 (facilitated glucose transporter), member 1
</td>
</tr>
<tr>
<td style="text-align:left;">
4
</td>
<td style="text-align:left;">
gene
</td>
<td style="text-align:right;">
119.1298
</td>
<td style="text-align:right;">
119.1298
</td>
<td style="text-align:left;">
positive
</td>
<td style="text-align:left;">
MGI_C57BL6J_3837121
</td>
<td style="text-align:left;">
Mir1957a
</td>
<td style="text-align:left;">
miRBase:MI0009954,ENSEMBL:ENSMUSG00000088552
</td>
<td style="text-align:left;">
MGI:3837121
</td>
<td style="text-align:left;">
miRNA gene
</td>
<td style="text-align:left;">
microRNA 1957a
</td>
</tr>
<tr>
<td style="text-align:left;">
4
</td>
<td style="text-align:left;">
gene
</td>
<td style="text-align:right;">
119.1303
</td>
<td style="text-align:right;">
119.1310
</td>
<td style="text-align:left;">
negative
</td>
<td style="text-align:left;">
MGI_C57BL6J_3651003
</td>
<td style="text-align:left;">
Gm12867
</td>
<td style="text-align:left;">
ENSEMBL:ENSMUSG00000086683
</td>
<td style="text-align:left;">
MGI:3651003
</td>
<td style="text-align:left;">
lncRNA gene
</td>
<td style="text-align:left;">
predicted gene 12867
</td>
</tr>
<tr>
<td style="text-align:left;">
4
</td>
<td style="text-align:left;">
gene
</td>
<td style="text-align:right;">
119.1337
</td>
<td style="text-align:right;">
119.1388
</td>
<td style="text-align:left;">
negative
</td>
<td style="text-align:left;">
MGI_C57BL6J_3651004
</td>
<td style="text-align:left;">
Gm12868
</td>
<td style="text-align:left;">
ENSEMBL:ENSMUSG00000085505
</td>
<td style="text-align:left;">
MGI:3651004
</td>
<td style="text-align:left;">
lncRNA gene
</td>
<td style="text-align:left;">
predicted gene 12868
</td>
</tr>
<tr>
<td style="text-align:left;">
4
</td>
<td style="text-align:left;">
gene
</td>
<td style="text-align:right;">
119.1695
</td>
<td style="text-align:right;">
119.1742
</td>
<td style="text-align:left;">
negative
</td>
<td style="text-align:left;">
MGI_C57BL6J_3041163
</td>
<td style="text-align:left;">
Zfp691
</td>
<td style="text-align:left;">
NCBI_Gene:195522,ENSEMBL:ENSMUSG00000045268
</td>
<td style="text-align:left;">
MGI:3041163
</td>
<td style="text-align:left;">
protein coding gene
</td>
<td style="text-align:left;">
zinc finger protein 691
</td>
</tr>
<tr>
<td style="text-align:left;">
4
</td>
<td style="text-align:left;">
gene
</td>
<td style="text-align:right;">
119.1755
</td>
<td style="text-align:right;">
119.1900
</td>
<td style="text-align:left;">
negative
</td>
<td style="text-align:left;">
MGI_C57BL6J_1349816
</td>
<td style="text-align:left;">
Ermap
</td>
<td style="text-align:left;">
NCBI_Gene:27028,ENSEMBL:ENSMUSG00000028644
</td>
<td style="text-align:left;">
MGI:1349816
</td>
<td style="text-align:left;">
protein coding gene
</td>
<td style="text-align:left;">
erythroblast membrane-associated protein
</td>
</tr>
<tr>
<td style="text-align:left;">
4
</td>
<td style="text-align:left;">
gene
</td>
<td style="text-align:right;">
119.1953
</td>
<td style="text-align:right;">
119.2013
</td>
<td style="text-align:left;">
positive
</td>
<td style="text-align:left;">
MGI_C57BL6J_1916466
</td>
<td style="text-align:left;">
Svbp
</td>
<td style="text-align:left;">
NCBI_Gene:69216,ENSEMBL:ENSMUSG00000028643
</td>
<td style="text-align:left;">
MGI:1916466
</td>
<td style="text-align:left;">
protein coding gene
</td>
<td style="text-align:left;">
small vasohibin binding protein
</td>
</tr>
<tr>
<td style="text-align:left;">
4
</td>
<td style="text-align:left;">
gene
</td>
<td style="text-align:right;">
119.2036
</td>
<td style="text-align:right;">
119.2328
</td>
<td style="text-align:left;">
negative
</td>
<td style="text-align:left;">
MGI_C57BL6J_2140466
</td>
<td style="text-align:left;">
AU022252
</td>
<td style="text-align:left;">
NCBI_Gene:230696,ENSEMBL:ENSMUSG00000078584
</td>
<td style="text-align:left;">
MGI:2140466
</td>
<td style="text-align:left;">
protein coding gene
</td>
<td style="text-align:left;">
expressed sequence AU022252
</td>
</tr>
<tr>
<td style="text-align:left;">
4
</td>
<td style="text-align:left;">
gene
</td>
<td style="text-align:right;">
119.2050
</td>
<td style="text-align:right;">
119.2186
</td>
<td style="text-align:left;">
negative
</td>
<td style="text-align:left;">
MGI_C57BL6J_1922430
</td>
<td style="text-align:left;">
Tmem269
</td>
<td style="text-align:left;">
NCBI_Gene:75180,ENSEMBL:ENSMUSG00000028642
</td>
<td style="text-align:left;">
MGI:1922430
</td>
<td style="text-align:left;">
protein coding gene
</td>
<td style="text-align:left;">
transmembrane protein 269
</td>
</tr>
<tr>
<td style="text-align:left;">
4
</td>
<td style="text-align:left;">
gene
</td>
<td style="text-align:right;">
119.2232
</td>
<td style="text-align:right;">
119.2248
</td>
<td style="text-align:left;">
positive
</td>
<td style="text-align:left;">
MGI_C57BL6J_3650325
</td>
<td style="text-align:left;">
Gm12898
</td>
<td style="text-align:left;">
ENSEMBL:ENSMUSG00000085626
</td>
<td style="text-align:left;">
MGI:3650325
</td>
<td style="text-align:left;">
lncRNA gene
</td>
<td style="text-align:left;">
predicted gene 12898
</td>
</tr>
<tr>
<td style="text-align:left;">
4
</td>
<td style="text-align:left;">
gene
</td>
<td style="text-align:right;">
119.2329
</td>
<td style="text-align:right;">
119.2490
</td>
<td style="text-align:left;">
positive
</td>
<td style="text-align:left;">
MGI_C57BL6J_1888921
</td>
<td style="text-align:left;">
P3h1
</td>
<td style="text-align:left;">
NCBI_Gene:56401,ENSEMBL:ENSMUSG00000028641
</td>
<td style="text-align:left;">
MGI:1888921
</td>
<td style="text-align:left;">
protein coding gene
</td>
<td style="text-align:left;">
prolyl 3-hydroxylase 1
</td>
</tr>
<tr>
<td style="text-align:left;">
4
</td>
<td style="text-align:left;">
gene
</td>
<td style="text-align:right;">
119.2525
</td>
<td style="text-align:right;">
119.2570
</td>
<td style="text-align:left;">
negative
</td>
<td style="text-align:left;">
MGI_C57BL6J_3651776
</td>
<td style="text-align:left;">
Gm12927
</td>
<td style="text-align:left;">
NCBI_Gene:102639193,ENSEMBL:ENSMUSG00000085203
</td>
<td style="text-align:left;">
MGI:3651776
</td>
<td style="text-align:left;">
lncRNA gene
</td>
<td style="text-align:left;">
predicted gene 12927
</td>
</tr>
<tr>
<td style="text-align:left;">
4
</td>
<td style="text-align:left;">
gene
</td>
<td style="text-align:right;">
119.2554
</td>
<td style="text-align:right;">
119.2624
</td>
<td style="text-align:left;">
positive
</td>
<td style="text-align:left;">
MGI_C57BL6J_3033992
</td>
<td style="text-align:left;">
Cldn19
</td>
<td style="text-align:left;">
NCBI_Gene:242653,ENSEMBL:ENSMUSG00000066058
</td>
<td style="text-align:left;">
MGI:3033992
</td>
<td style="text-align:left;">
protein coding gene
</td>
<td style="text-align:left;">
claudin 19
</td>
</tr>
<tr>
<td style="text-align:left;">
4
</td>
<td style="text-align:left;">
gene
</td>
<td style="text-align:right;">
119.2773
</td>
<td style="text-align:right;">
119.2947
</td>
<td style="text-align:left;">
negative
</td>
<td style="text-align:left;">
MGI_C57BL6J_99146
</td>
<td style="text-align:left;">
Ybx1
</td>
<td style="text-align:left;">
NCBI_Gene:22608,ENSEMBL:ENSMUSG00000028639
</td>
<td style="text-align:left;">
MGI:99146
</td>
<td style="text-align:left;">
protein coding gene
</td>
<td style="text-align:left;">
Y box protein 1
</td>
</tr>
<tr>
<td style="text-align:left;">
4
</td>
<td style="text-align:left;">
gene
</td>
<td style="text-align:right;">
119.2941
</td>
<td style="text-align:right;">
119.2951
</td>
<td style="text-align:left;">
positive
</td>
<td style="text-align:left;">
MGI_C57BL6J_1920791
</td>
<td style="text-align:left;">
1700102N10Rik
</td>
<td style="text-align:left;">
NA
</td>
<td style="text-align:left;">
NA
</td>
<td style="text-align:left;">
unclassified non-coding RNA gene
</td>
<td style="text-align:left;">
RIKEN cDNA 1700102N10 gene
</td>
</tr>
<tr>
<td style="text-align:left;">
4
</td>
<td style="text-align:left;">
gene
</td>
<td style="text-align:right;">
119.2950
</td>
<td style="text-align:right;">
119.2955
</td>
<td style="text-align:left;">
negative
</td>
<td style="text-align:left;">
MGI_C57BL6J_5611045
</td>
<td style="text-align:left;">
Gm37817
</td>
<td style="text-align:left;">
ENSEMBL:ENSMUSG00000102819
</td>
<td style="text-align:left;">
MGI:5611045
</td>
<td style="text-align:left;">
unclassified gene
</td>
<td style="text-align:left;">
predicted gene, 37817
</td>
</tr>
<tr>
<td style="text-align:left;">
4
</td>
<td style="text-align:left;">
gene
</td>
<td style="text-align:right;">
119.3000
</td>
<td style="text-align:right;">
119.3205
</td>
<td style="text-align:left;">
negative
</td>
<td style="text-align:left;">
MGI_C57BL6J_106499
</td>
<td style="text-align:left;">
Ppih
</td>
<td style="text-align:left;">
NCBI_Gene:66101,ENSEMBL:ENSMUSG00000060288
</td>
<td style="text-align:left;">
MGI:106499
</td>
<td style="text-align:left;">
protein coding gene
</td>
<td style="text-align:left;">
peptidyl prolyl isomerase H
</td>
</tr>
<tr>
<td style="text-align:left;">
4
</td>
<td style="text-align:left;">
gene
</td>
<td style="text-align:right;">
119.3225
</td>
<td style="text-align:right;">
119.4155
</td>
<td style="text-align:left;">
negative
</td>
<td style="text-align:left;">
MGI_C57BL6J_1920582
</td>
<td style="text-align:left;">
Ccdc30
</td>
<td style="text-align:left;">
NCBI_Gene:73332,ENSEMBL:ENSMUSG00000028637
</td>
<td style="text-align:left;">
MGI:1920582
</td>
<td style="text-align:left;">
protein coding gene
</td>
<td style="text-align:left;">
coiled-coil domain containing 30
</td>
</tr>
<tr>
<td style="text-align:left;">
4
</td>
<td style="text-align:left;">
pseudogene
</td>
<td style="text-align:right;">
119.3339
</td>
<td style="text-align:right;">
119.3355
</td>
<td style="text-align:left;">
negative
</td>
<td style="text-align:left;">
MGI_C57BL6J_96761
</td>
<td style="text-align:left;">
Ldha-ps2
</td>
<td style="text-align:left;">
NCBI_Gene:16830,ENSEMBL:ENSMUSG00000083836
</td>
<td style="text-align:left;">
MGI:96761
</td>
<td style="text-align:left;">
pseudogene
</td>
<td style="text-align:left;">
lactate dehydrogenase A, pseudogene 2
</td>
</tr>
<tr>
<td style="text-align:left;">
4
</td>
<td style="text-align:left;">
gene
</td>
<td style="text-align:right;">
119.3935
</td>
<td style="text-align:right;">
119.3999
</td>
<td style="text-align:left;">
positive
</td>
<td style="text-align:left;">
MGI_C57BL6J_6324757
</td>
<td style="text-align:left;">
Gm50484
</td>
<td style="text-align:left;">
ENSEMBL:ENSMUSG00000118403
</td>
<td style="text-align:left;">
MGI:6324757
</td>
<td style="text-align:left;">
lncRNA gene
</td>
<td style="text-align:left;">
predicted gene, 50484
</td>
</tr>
<tr>
<td style="text-align:left;">
4
</td>
<td style="text-align:left;">
gene
</td>
<td style="text-align:right;">
119.4185
</td>
<td style="text-align:right;">
119.4224
</td>
<td style="text-align:left;">
negative
</td>
<td style="text-align:left;">
MGI_C57BL6J_1915237
</td>
<td style="text-align:left;">
Ppcs
</td>
<td style="text-align:left;">
NCBI_Gene:106564,ENSEMBL:ENSMUSG00000028636
</td>
<td style="text-align:left;">
MGI:1915237
</td>
<td style="text-align:left;">
protein coding gene
</td>
<td style="text-align:left;">
phosphopantothenoylcysteine synthetase
</td>
</tr>
<tr>
<td style="text-align:left;">
4
</td>
<td style="text-align:left;">
gene
</td>
<td style="text-align:right;">
119.4227
</td>
<td style="text-align:right;">
119.4674
</td>
<td style="text-align:left;">
positive
</td>
<td style="text-align:left;">
MGI_C57BL6J_2140259
</td>
<td style="text-align:left;">
Zmynd12
</td>
<td style="text-align:left;">
NCBI_Gene:332934,ENSEMBL:ENSMUSG00000070806
</td>
<td style="text-align:left;">
MGI:2140259
</td>
<td style="text-align:left;">
protein coding gene
</td>
<td style="text-align:left;">
zinc finger, MYND domain containing 12
</td>
</tr>
<tr>
<td style="text-align:left;">
4
</td>
<td style="text-align:left;">
gene
</td>
<td style="text-align:right;">
119.4653
</td>
<td style="text-align:right;">
119.4926
</td>
<td style="text-align:left;">
negative
</td>
<td style="text-align:left;">
MGI_C57BL6J_3040686
</td>
<td style="text-align:left;">
Rimkla
</td>
<td style="text-align:left;">
NCBI_Gene:194237,ENSEMBL:ENSMUSG00000048899
</td>
<td style="text-align:left;">
MGI:3040686
</td>
<td style="text-align:left;">
protein coding gene
</td>
<td style="text-align:left;">
ribosomal modification protein rimK-like family member A
</td>
</tr>
<tr>
<td style="text-align:left;">
4
</td>
<td style="text-align:left;">
pseudogene
</td>
<td style="text-align:right;">
119.5019
</td>
<td style="text-align:right;">
119.5025
</td>
<td style="text-align:left;">
negative
</td>
<td style="text-align:left;">
MGI_C57BL6J_3652229
</td>
<td style="text-align:left;">
Gm12954
</td>
<td style="text-align:left;">
ENSEMBL:ENSMUSG00000082009
</td>
<td style="text-align:left;">
MGI:3652229
</td>
<td style="text-align:left;">
pseudogene
</td>
<td style="text-align:left;">
predicted gene 12954
</td>
</tr>
<tr>
<td style="text-align:left;">
4
</td>
<td style="text-align:left;">
pseudogene
</td>
<td style="text-align:right;">
119.5096
</td>
<td style="text-align:right;">
119.5177
</td>
<td style="text-align:left;">
negative
</td>
<td style="text-align:left;">
MGI_C57BL6J_3652227
</td>
<td style="text-align:left;">
Gm12956
</td>
<td style="text-align:left;">
NCBI_Gene:102640085,ENSEMBL:ENSMUSG00000082814
</td>
<td style="text-align:left;">
MGI:3652227
</td>
<td style="text-align:left;">
pseudogene
</td>
<td style="text-align:left;">
predicted gene 12956
</td>
</tr>
<tr>
<td style="text-align:left;">
4
</td>
<td style="text-align:left;">
pseudogene
</td>
<td style="text-align:right;">
119.5191
</td>
<td style="text-align:right;">
119.5201
</td>
<td style="text-align:left;">
negative
</td>
<td style="text-align:left;">
MGI_C57BL6J_3652228
</td>
<td style="text-align:left;">
Gm12955
</td>
<td style="text-align:left;">
NCBI_Gene:666766,ENSEMBL:ENSMUSG00000080773
</td>
<td style="text-align:left;">
MGI:3652228
</td>
<td style="text-align:left;">
pseudogene
</td>
<td style="text-align:left;">
predicted gene 12955
</td>
</tr>
<tr>
<td style="text-align:left;">
4
</td>
<td style="text-align:left;">
pseudogene
</td>
<td style="text-align:right;">
119.5209
</td>
<td style="text-align:right;">
119.5215
</td>
<td style="text-align:left;">
positive
</td>
<td style="text-align:left;">
MGI_C57BL6J_3649304
</td>
<td style="text-align:left;">
Gm12957
</td>
<td style="text-align:left;">
NCBI_Gene:384052,ENSEMBL:ENSMUSG00000084379
</td>
<td style="text-align:left;">
MGI:3649304
</td>
<td style="text-align:left;">
pseudogene
</td>
<td style="text-align:left;">
predicted gene 12957
</td>
</tr>
<tr>
<td style="text-align:left;">
4
</td>
<td style="text-align:left;">
pseudogene
</td>
<td style="text-align:right;">
119.5279
</td>
<td style="text-align:right;">
119.5282
</td>
<td style="text-align:left;">
positive
</td>
<td style="text-align:left;">
MGI_C57BL6J_97991
</td>
<td style="text-align:left;">
Rnu6-ps2
</td>
<td style="text-align:left;">
NCBI_Gene:19864,ENSEMBL:ENSMUSG00000094494
</td>
<td style="text-align:left;">
MGI:97991
</td>
<td style="text-align:left;">
pseudogene
</td>
<td style="text-align:left;">
U6 small nuclear RNA, pseudogene 2
</td>
</tr>
<tr>
<td style="text-align:left;">
4
</td>
<td style="text-align:left;">
gene
</td>
<td style="text-align:right;">
119.5303
</td>
<td style="text-align:right;">
119.5395
</td>
<td style="text-align:left;">
negative
</td>
<td style="text-align:left;">
MGI_C57BL6J_3035485
</td>
<td style="text-align:left;">
Frg2f1
</td>
<td style="text-align:left;">
NCBI_Gene:433752,ENSEMBL:ENSMUSG00000087385
</td>
<td style="text-align:left;">
MGI:3035485
</td>
<td style="text-align:left;">
protein coding gene
</td>
<td style="text-align:left;">
FSHD region gene 2 family member 1
</td>
</tr>
<tr>
<td style="text-align:left;">
4
</td>
<td style="text-align:left;">
gene
</td>
<td style="text-align:right;">
119.5370
</td>
<td style="text-align:right;">
119.6291
</td>
<td style="text-align:left;">
positive
</td>
<td style="text-align:left;">
MGI_C57BL6J_2443432
</td>
<td style="text-align:left;">
Foxj3
</td>
<td style="text-align:left;">
NCBI_Gene:230700,ENSEMBL:ENSMUSG00000032998
</td>
<td style="text-align:left;">
MGI:2443432
</td>
<td style="text-align:left;">
protein coding gene
</td>
<td style="text-align:left;">
forkhead box J3
</td>
</tr>
<tr>
<td style="text-align:left;">
4
</td>
<td style="text-align:left;">
pseudogene
</td>
<td style="text-align:right;">
119.5537
</td>
<td style="text-align:right;">
119.5541
</td>
<td style="text-align:left;">
negative
</td>
<td style="text-align:left;">
MGI_C57BL6J_3649444
</td>
<td style="text-align:left;">
Gm12959
</td>
<td style="text-align:left;">
ENSEMBL:ENSMUSG00000081678
</td>
<td style="text-align:left;">
MGI:3649444
</td>
<td style="text-align:left;">
pseudogene
</td>
<td style="text-align:left;">
predicted gene 12959
</td>
</tr>
<tr>
<td style="text-align:left;">
4
</td>
<td style="text-align:left;">
gene
</td>
<td style="text-align:right;">
119.6377
</td>
<td style="text-align:right;">
119.6395
</td>
<td style="text-align:left;">
positive
</td>
<td style="text-align:left;">
MGI_C57BL6J_102738
</td>
<td style="text-align:left;">
Guca2a
</td>
<td style="text-align:left;">
NCBI_Gene:14915,ENSEMBL:ENSMUSG00000023247
</td>
<td style="text-align:left;">
MGI:102738
</td>
<td style="text-align:left;">
protein coding gene
</td>
<td style="text-align:left;">
guanylate cyclase activator 2a (guanylin)
</td>
</tr>
<tr>
<td style="text-align:left;">
4
</td>
<td style="text-align:left;">
gene
</td>
<td style="text-align:right;">
119.6566
</td>
<td style="text-align:right;">
119.6590
</td>
<td style="text-align:left;">
negative
</td>
<td style="text-align:left;">
MGI_C57BL6J_1270851
</td>
<td style="text-align:left;">
Guca2b
</td>
<td style="text-align:left;">
NCBI_Gene:14916,ENSEMBL:ENSMUSG00000032978
</td>
<td style="text-align:left;">
MGI:1270851
</td>
<td style="text-align:left;">
protein coding gene
</td>
<td style="text-align:left;">
guanylate cyclase activator 2b (retina)
</td>
</tr>
<tr>
<td style="text-align:left;">
4
</td>
<td style="text-align:left;">
gene
</td>
<td style="text-align:right;">
119.6590
</td>
<td style="text-align:right;">
119.6638
</td>
<td style="text-align:left;">
positive
</td>
<td style="text-align:left;">
MGI_C57BL6J_5594920
</td>
<td style="text-align:left;">
Gm35761
</td>
<td style="text-align:left;">
NCBI_Gene:102639449
</td>
<td style="text-align:left;">
MGI:5594920
</td>
<td style="text-align:left;">
lncRNA gene
</td>
<td style="text-align:left;">
predicted gene, 35761
</td>
</tr>
<tr>
<td style="text-align:left;">
4
</td>
<td style="text-align:left;">
gene
</td>
<td style="text-align:right;">
119.6957
</td>
<td style="text-align:right;">
119.7241
</td>
<td style="text-align:left;">
negative
</td>
<td style="text-align:left;">
MGI_C57BL6J_5595116
</td>
<td style="text-align:left;">
Gm35957
</td>
<td style="text-align:left;">
NCBI_Gene:102639707
</td>
<td style="text-align:left;">
MGI:5595116
</td>
<td style="text-align:left;">
lncRNA gene
</td>
<td style="text-align:left;">
predicted gene, 35957
</td>
</tr>
<tr>
<td style="text-align:left;">
4
</td>
<td style="text-align:left;">
gene
</td>
<td style="text-align:right;">
119.7338
</td>
<td style="text-align:right;">
120.1380
</td>
<td style="text-align:left;">
positive
</td>
<td style="text-align:left;">
MGI_C57BL6J_106589
</td>
<td style="text-align:left;">
Hivep3
</td>
<td style="text-align:left;">
NCBI_Gene:16656,ENSEMBL:ENSMUSG00000028634
</td>
<td style="text-align:left;">
MGI:106589
</td>
<td style="text-align:left;">
protein coding gene
</td>
<td style="text-align:left;">
human immunodeficiency virus type I enhancer binding protein 3
</td>
</tr>
<tr>
<td style="text-align:left;">
4
</td>
<td style="text-align:left;">
gene
</td>
<td style="text-align:right;">
120.0046
</td>
<td style="text-align:right;">
120.0495
</td>
<td style="text-align:left;">
negative
</td>
<td style="text-align:left;">
MGI_C57BL6J_5621517
</td>
<td style="text-align:left;">
Gm38632
</td>
<td style="text-align:left;">
NCBI_Gene:102642778
</td>
<td style="text-align:left;">
MGI:5621517
</td>
<td style="text-align:left;">
lncRNA gene
</td>
<td style="text-align:left;">
predicted gene, 38632
</td>
</tr>
<tr>
<td style="text-align:left;">
4
</td>
<td style="text-align:left;">
gene
</td>
<td style="text-align:right;">
120.0658
</td>
<td style="text-align:right;">
120.0747
</td>
<td style="text-align:left;">
negative
</td>
<td style="text-align:left;">
MGI_C57BL6J_5595374
</td>
<td style="text-align:left;">
Gm36215
</td>
<td style="text-align:left;">
NCBI_Gene:102640050
</td>
<td style="text-align:left;">
MGI:5595374
</td>
<td style="text-align:left;">
lncRNA gene
</td>
<td style="text-align:left;">
predicted gene, 36215
</td>
</tr>
<tr>
<td style="text-align:left;">
4
</td>
<td style="text-align:left;">
gene
</td>
<td style="text-align:right;">
120.1542
</td>
<td style="text-align:right;">
120.2622
</td>
<td style="text-align:left;">
negative
</td>
<td style="text-align:left;">
MGI_C57BL6J_5595293
</td>
<td style="text-align:left;">
Gm36134
</td>
<td style="text-align:left;">
NCBI_Gene:102639939
</td>
<td style="text-align:left;">
MGI:5595293
</td>
<td style="text-align:left;">
lncRNA gene
</td>
<td style="text-align:left;">
predicted gene, 36134
</td>
</tr>
<tr>
<td style="text-align:left;">
4
</td>
<td style="text-align:left;">
gene
</td>
<td style="text-align:right;">
120.1612
</td>
<td style="text-align:right;">
120.1674
</td>
<td style="text-align:left;">
positive
</td>
<td style="text-align:left;">
MGI_C57BL6J_95284
</td>
<td style="text-align:left;">
Edn2
</td>
<td style="text-align:left;">
NCBI_Gene:13615,ENSEMBL:ENSMUSG00000028635
</td>
<td style="text-align:left;">
MGI:95284
</td>
<td style="text-align:left;">
protein coding gene
</td>
<td style="text-align:left;">
endothelin 2
</td>
</tr>
<tr>
<td style="text-align:left;">
4
</td>
<td style="text-align:left;">
gene
</td>
<td style="text-align:right;">
120.1762
</td>
<td style="text-align:right;">
120.1813
</td>
<td style="text-align:left;">
positive
</td>
<td style="text-align:left;">
MGI_C57BL6J_5595232
</td>
<td style="text-align:left;">
Gm36073
</td>
<td style="text-align:left;">
NCBI_Gene:102639859
</td>
<td style="text-align:left;">
MGI:5595232
</td>
<td style="text-align:left;">
lncRNA gene
</td>
<td style="text-align:left;">
predicted gene, 36073
</td>
</tr>
<tr>
<td style="text-align:left;">
4
</td>
<td style="text-align:left;">
gene
</td>
<td style="text-align:right;">
120.1908
</td>
<td style="text-align:right;">
120.2003
</td>
<td style="text-align:left;">
positive
</td>
<td style="text-align:left;">
MGI_C57BL6J_5595172
</td>
<td style="text-align:left;">
Gm36013
</td>
<td style="text-align:left;">
NCBI_Gene:102639782
</td>
<td style="text-align:left;">
MGI:5595172
</td>
<td style="text-align:left;">
lncRNA gene
</td>
<td style="text-align:left;">
predicted gene, 36013
</td>
</tr>
<tr>
<td style="text-align:left;">
4
</td>
<td style="text-align:left;">
gene
</td>
<td style="text-align:right;">
120.2671
</td>
<td style="text-align:right;">
120.2873
</td>
<td style="text-align:left;">
negative
</td>
<td style="text-align:left;">
MGI_C57BL6J_2676586
</td>
<td style="text-align:left;">
Foxo6
</td>
<td style="text-align:left;">
NCBI_Gene:329934,ENSEMBL:ENSMUSG00000052135
</td>
<td style="text-align:left;">
MGI:2676586
</td>
<td style="text-align:left;">
protein coding gene
</td>
<td style="text-align:left;">
forkhead box O6
</td>
</tr>
<tr>
<td style="text-align:left;">
4
</td>
<td style="text-align:left;">
gene
</td>
<td style="text-align:right;">
120.2911
</td>
<td style="text-align:right;">
120.3039
</td>
<td style="text-align:left;">
positive
</td>
<td style="text-align:left;">
MGI_C57BL6J_3028036
</td>
<td style="text-align:left;">
Foxo6os
</td>
<td style="text-align:left;">
NCBI_Gene:402730,ENSEMBL:ENSMUSG00000084929
</td>
<td style="text-align:left;">
MGI:3028036
</td>
<td style="text-align:left;">
antisense lncRNA gene
</td>
<td style="text-align:left;">
forkhead box O6, opposite strand
</td>
</tr>
<tr>
<td style="text-align:left;">
4
</td>
<td style="text-align:left;">
gene
</td>
<td style="text-align:right;">
120.3097
</td>
<td style="text-align:right;">
120.3129
</td>
<td style="text-align:left;">
negative
</td>
<td style="text-align:left;">
MGI_C57BL6J_5595658
</td>
<td style="text-align:left;">
Gm36499
</td>
<td style="text-align:left;">
NCBI_Gene:102640438
</td>
<td style="text-align:left;">
MGI:5595658
</td>
<td style="text-align:left;">
lncRNA gene
</td>
<td style="text-align:left;">
predicted gene, 36499
</td>
</tr>
<tr>
<td style="text-align:left;">
4
</td>
<td style="text-align:left;">
gene
</td>
<td style="text-align:right;">
120.3191
</td>
<td style="text-align:right;">
120.3972
</td>
<td style="text-align:left;">
negative
</td>
<td style="text-align:left;">
MGI_C57BL6J_5595733
</td>
<td style="text-align:left;">
Gm36574
</td>
<td style="text-align:left;">
NCBI_Gene:102640537
</td>
<td style="text-align:left;">
MGI:5595733
</td>
<td style="text-align:left;">
lncRNA gene
</td>
<td style="text-align:left;">
predicted gene, 36574
</td>
</tr>
<tr>
<td style="text-align:left;">
4
</td>
<td style="text-align:left;">
gene
</td>
<td style="text-align:right;">
120.4053
</td>
<td style="text-align:right;">
120.5302
</td>
<td style="text-align:left;">
positive
</td>
<td style="text-align:left;">
MGI_C57BL6J_1352762
</td>
<td style="text-align:left;">
Scmh1
</td>
<td style="text-align:left;">
NCBI_Gene:29871,ENSEMBL:ENSMUSG00000000085
</td>
<td style="text-align:left;">
MGI:1352762
</td>
<td style="text-align:left;">
protein coding gene
</td>
<td style="text-align:left;">
sex comb on midleg homolog 1
</td>
</tr>
<tr>
<td style="text-align:left;">
4
</td>
<td style="text-align:left;">
gene
</td>
<td style="text-align:right;">
120.5321
</td>
<td style="text-align:right;">
120.5367
</td>
<td style="text-align:left;">
positive
</td>
<td style="text-align:left;">
MGI_C57BL6J_3045330
</td>
<td style="text-align:left;">
Slfnl1
</td>
<td style="text-align:left;">
NCBI_Gene:194219,ENSEMBL:ENSMUSG00000047518
</td>
<td style="text-align:left;">
MGI:3045330
</td>
<td style="text-align:left;">
protein coding gene
</td>
<td style="text-align:left;">
schlafen like 1
</td>
</tr>
<tr>
<td style="text-align:left;">
4
</td>
<td style="text-align:left;">
gene
</td>
<td style="text-align:right;">
120.5399
</td>
<td style="text-align:right;">
120.5703
</td>
<td style="text-align:left;">
negative
</td>
<td style="text-align:left;">
MGI_C57BL6J_1858304
</td>
<td style="text-align:left;">
Ctps
</td>
<td style="text-align:left;">
NCBI_Gene:51797,ENSEMBL:ENSMUSG00000028633
</td>
<td style="text-align:left;">
MGI:1858304
</td>
<td style="text-align:left;">
protein coding gene
</td>
<td style="text-align:left;">
cytidine 5’-triphosphate synthase
</td>
</tr>
<tr>
<td style="text-align:left;">
4
</td>
<td style="text-align:left;">
gene
</td>
<td style="text-align:right;">
120.5887
</td>
<td style="text-align:right;">
120.6097
</td>
<td style="text-align:left;">
positive
</td>
<td style="text-align:left;">
MGI_C57BL6J_3648581
</td>
<td style="text-align:left;">
Gm8439
</td>
<td style="text-align:left;">
NCBI_Gene:667063,ENSEMBL:ENSMUSG00000091297
</td>
<td style="text-align:left;">
MGI:3648581
</td>
<td style="text-align:left;">
protein coding gene
</td>
<td style="text-align:left;">
predicted gene 8439
</td>
</tr>
<tr>
<td style="text-align:left;">
4
</td>
<td style="text-align:left;">
gene
</td>
<td style="text-align:right;">
120.6158
</td>
<td style="text-align:right;">
120.6227
</td>
<td style="text-align:left;">
positive
</td>
<td style="text-align:left;">
MGI_C57BL6J_3651651
</td>
<td style="text-align:left;">
Gm12860
</td>
<td style="text-align:left;">
ENSEMBL:ENSMUSG00000086187
</td>
<td style="text-align:left;">
MGI:3651651
</td>
<td style="text-align:left;">
lncRNA gene
</td>
<td style="text-align:left;">
predicted gene 12860
</td>
</tr>
<tr>
<td style="text-align:left;">
4
</td>
<td style="text-align:left;">
gene
</td>
<td style="text-align:right;">
120.6442
</td>
<td style="text-align:right;">
120.6481
</td>
<td style="text-align:left;">
negative
</td>
<td style="text-align:left;">
MGI_C57BL6J_5791415
</td>
<td style="text-align:left;">
Gm45579
</td>
<td style="text-align:left;">
ENSEMBL:ENSMUSG00000110389
</td>
<td style="text-align:left;">
MGI:5791415
</td>
<td style="text-align:left;">
lncRNA gene
</td>
<td style="text-align:left;">
predicted gene 45579
</td>
</tr>
<tr>
<td style="text-align:left;">
4
</td>
<td style="text-align:left;">
gene
</td>
<td style="text-align:right;">
120.6596
</td>
<td style="text-align:right;">
120.6633
</td>
<td style="text-align:left;">
positive
</td>
<td style="text-align:left;">
MGI_C57BL6J_5589050
</td>
<td style="text-align:left;">
Gm29891
</td>
<td style="text-align:left;">
NCBI_Gene:102631594
</td>
<td style="text-align:left;">
MGI:5589050
</td>
<td style="text-align:left;">
lncRNA gene
</td>
<td style="text-align:left;">
predicted gene, 29891
</td>
</tr>
<tr>
<td style="text-align:left;">
4
</td>
<td style="text-align:left;">
gene
</td>
<td style="text-align:right;">
120.6666
</td>
<td style="text-align:right;">
120.6678
</td>
<td style="text-align:left;">
positive
</td>
<td style="text-align:left;">
MGI_C57BL6J_1861694
</td>
<td style="text-align:left;">
Cited4
</td>
<td style="text-align:left;">
NCBI_Gene:56222,ENSEMBL:ENSMUSG00000070803
</td>
<td style="text-align:left;">
MGI:1861694
</td>
<td style="text-align:left;">
protein coding gene
</td>
<td style="text-align:left;">
Cbp/p300-interacting transactivator, with Glu/Asp-rich carboxy-terminal
domain, 4
</td>
</tr>
<tr>
<td style="text-align:left;">
4
</td>
<td style="text-align:left;">
gene
</td>
<td style="text-align:right;">
120.6961
</td>
<td style="text-align:right;">
120.7486
</td>
<td style="text-align:left;">
negative
</td>
<td style="text-align:left;">
MGI_C57BL6J_1926803
</td>
<td style="text-align:left;">
Kcnq4
</td>
<td style="text-align:left;">
NCBI_Gene:60613,ENSEMBL:ENSMUSG00000028631
</td>
<td style="text-align:left;">
MGI:1926803
</td>
<td style="text-align:left;">
protein coding gene
</td>
<td style="text-align:left;">
potassium voltage-gated channel, subfamily Q, member 4
</td>
</tr>
<tr>
<td style="text-align:left;">
4
</td>
<td style="text-align:left;">
gene
</td>
<td style="text-align:right;">
120.7574
</td>
<td style="text-align:right;">
120.8316
</td>
<td style="text-align:left;">
negative
</td>
<td style="text-align:left;">
MGI_C57BL6J_107901
</td>
<td style="text-align:left;">
Nfyc
</td>
<td style="text-align:left;">
NCBI_Gene:18046,ENSEMBL:ENSMUSG00000032897
</td>
<td style="text-align:left;">
MGI:107901
</td>
<td style="text-align:left;">
protein coding gene
</td>
<td style="text-align:left;">
nuclear transcription factor-Y gamma
</td>
</tr>
<tr>
<td style="text-align:left;">
4
</td>
<td style="text-align:left;">
gene
</td>
<td style="text-align:right;">
120.7695
</td>
<td style="text-align:right;">
120.7696
</td>
<td style="text-align:left;">
negative
</td>
<td style="text-align:left;">
MGI_C57BL6J_2676909
</td>
<td style="text-align:left;">
Mir30c-1
</td>
<td style="text-align:left;">
miRBase:MI0000547,NCBI_Gene:387227,ENSEMBL:ENSMUSG00000065490
</td>
<td style="text-align:left;">
MGI:2676909
</td>
<td style="text-align:left;">
miRNA gene
</td>
<td style="text-align:left;">
microRNA 30c-1
</td>
</tr>
<tr>
<td style="text-align:left;">
4
</td>
<td style="text-align:left;">
gene
</td>
<td style="text-align:right;">
120.7726
</td>
<td style="text-align:right;">
120.7727
</td>
<td style="text-align:left;">
positive
</td>
<td style="text-align:left;">
MGI_C57BL6J_5562761
</td>
<td style="text-align:left;">
Mir30f
</td>
<td style="text-align:left;">
miRBase:MI0021961,NCBI_Gene:102466650,ENSEMBL:ENSMUSG00000105585
</td>
<td style="text-align:left;">
MGI:5562761
</td>
<td style="text-align:left;">
miRNA gene
</td>
<td style="text-align:left;">
microRNA 30f
</td>
</tr>
<tr>
<td style="text-align:left;">
4
</td>
<td style="text-align:left;">
gene
</td>
<td style="text-align:right;">
120.7726
</td>
<td style="text-align:right;">
120.7727
</td>
<td style="text-align:left;">
negative
</td>
<td style="text-align:left;">
MGI_C57BL6J_3619329
</td>
<td style="text-align:left;">
Mir30e
</td>
<td style="text-align:left;">
miRBase:MI0000259,NCBI_Gene:723836,ENSEMBL:ENSMUSG00000065409
</td>
<td style="text-align:left;">
MGI:3619329
</td>
<td style="text-align:left;">
miRNA gene
</td>
<td style="text-align:left;">
microRNA 30e
</td>
</tr>
<tr>
<td style="text-align:left;">
4
</td>
<td style="text-align:left;">
gene
</td>
<td style="text-align:right;">
120.8375
</td>
<td style="text-align:right;">
120.8376
</td>
<td style="text-align:left;">
positive
</td>
<td style="text-align:left;">
MGI_C57BL6J_5453713
</td>
<td style="text-align:left;">
Gm23936
</td>
<td style="text-align:left;">
ENSEMBL:ENSMUSG00000096876
</td>
<td style="text-align:left;">
MGI:5453713
</td>
<td style="text-align:left;">
miRNA gene
</td>
<td style="text-align:left;">
predicted gene, 23936
</td>
</tr>
<tr>
<td style="text-align:left;">
4
</td>
<td style="text-align:left;">
gene
</td>
<td style="text-align:right;">
120.8548
</td>
<td style="text-align:right;">
120.8966
</td>
<td style="text-align:left;">
positive
</td>
<td style="text-align:left;">
MGI_C57BL6J_2443331
</td>
<td style="text-align:left;">
Rims3
</td>
<td style="text-align:left;">
NCBI_Gene:242662,ENSEMBL:ENSMUSG00000032890
</td>
<td style="text-align:left;">
MGI:2443331
</td>
<td style="text-align:left;">
protein coding gene
</td>
<td style="text-align:left;">
regulating synaptic membrane exocytosis 3
</td>
</tr>
<tr>
<td style="text-align:left;">
4
</td>
<td style="text-align:left;">
pseudogene
</td>
<td style="text-align:right;">
120.9011
</td>
<td style="text-align:right;">
120.9028
</td>
<td style="text-align:left;">
positive
</td>
<td style="text-align:left;">
MGI_C57BL6J_3649219
</td>
<td style="text-align:left;">
Gm12871
</td>
<td style="text-align:left;">
NCBI_Gene:102631670,ENSEMBL:ENSMUSG00000082200
</td>
<td style="text-align:left;">
MGI:3649219
</td>
<td style="text-align:left;">
pseudogene
</td>
<td style="text-align:left;">
predicted gene 12871
</td>
</tr>
<tr>
<td style="text-align:left;">
4
</td>
<td style="text-align:left;">
gene
</td>
<td style="text-align:right;">
120.9035
</td>
<td style="text-align:right;">
120.9036
</td>
<td style="text-align:left;">
positive
</td>
<td style="text-align:left;">
MGI_C57BL6J_5454455
</td>
<td style="text-align:left;">
Gm24678
</td>
<td style="text-align:left;">
ENSEMBL:ENSMUSG00000089381
</td>
<td style="text-align:left;">
MGI:5454455
</td>
<td style="text-align:left;">
snRNA gene
</td>
<td style="text-align:left;">
predicted gene, 24678
</td>
</tr>
<tr>
<td style="text-align:left;">
4
</td>
<td style="text-align:left;">
gene
</td>
<td style="text-align:right;">
120.9212
</td>
<td style="text-align:right;">
120.9250
</td>
<td style="text-align:left;">
negative
</td>
<td style="text-align:left;">
MGI_C57BL6J_1920422
</td>
<td style="text-align:left;">
Exo5
</td>
<td style="text-align:left;">
NCBI_Gene:73172,ENSEMBL:ENSMUSG00000028629
</td>
<td style="text-align:left;">
MGI:1920422
</td>
<td style="text-align:left;">
protein coding gene
</td>
<td style="text-align:left;">
exonuclease 5
</td>
</tr>
<tr>
<td style="text-align:left;">
4
</td>
<td style="text-align:left;">
gene
</td>
<td style="text-align:right;">
120.9301
</td>
<td style="text-align:right;">
120.9534
</td>
<td style="text-align:left;">
negative
</td>
<td style="text-align:left;">
MGI_C57BL6J_107794
</td>
<td style="text-align:left;">
Zfp69
</td>
<td style="text-align:left;">
NCBI_Gene:381549,ENSEMBL:ENSMUSG00000064141
</td>
<td style="text-align:left;">
MGI:107794
</td>
<td style="text-align:left;">
protein coding gene
</td>
<td style="text-align:left;">
zinc finger protein 69
</td>
</tr>
<tr>
<td style="text-align:left;">
4
</td>
<td style="text-align:left;">
gene
</td>
<td style="text-align:right;">
120.9683
</td>
<td style="text-align:right;">
121.0173
</td>
<td style="text-align:left;">
negative
</td>
<td style="text-align:left;">
MGI_C57BL6J_1917030
</td>
<td style="text-align:left;">
Smap2
</td>
<td style="text-align:left;">
NCBI_Gene:69780,ENSEMBL:ENSMUSG00000032870
</td>
<td style="text-align:left;">
MGI:1917030
</td>
<td style="text-align:left;">
protein coding gene
</td>
<td style="text-align:left;">
small ArfGAP 2
</td>
</tr>
<tr>
<td style="text-align:left;">
4
</td>
<td style="text-align:left;">
gene
</td>
<td style="text-align:right;">
120.9733
</td>
<td style="text-align:right;">
120.9733
</td>
<td style="text-align:left;">
negative
</td>
<td style="text-align:left;">
MGI_C57BL6J_5562762
</td>
<td style="text-align:left;">
Mir7015
</td>
<td style="text-align:left;">
miRBase:MI0022864,NCBI_Gene:102465614,ENSEMBL:ENSMUSG00000106436
</td>
<td style="text-align:left;">
MGI:5562762
</td>
<td style="text-align:left;">
miRNA gene
</td>
<td style="text-align:left;">
microRNA 7015
</td>
</tr>
<tr>
<td style="text-align:left;">
4
</td>
<td style="text-align:left;">
gene
</td>
<td style="text-align:right;">
121.0394
</td>
<td style="text-align:right;">
121.0553
</td>
<td style="text-align:left;">
positive
</td>
<td style="text-align:left;">
MGI_C57BL6J_88466
</td>
<td style="text-align:left;">
Col9a2
</td>
<td style="text-align:left;">
NCBI_Gene:12840,ENSEMBL:ENSMUSG00000028626
</td>
<td style="text-align:left;">
MGI:88466
</td>
<td style="text-align:left;">
protein coding gene
</td>
<td style="text-align:left;">
collagen, type IX, alpha 2
</td>
</tr>
<tr>
<td style="text-align:left;">
4
</td>
<td style="text-align:left;">
gene
</td>
<td style="text-align:right;">
121.0592
</td>
<td style="text-align:right;">
121.0982
</td>
<td style="text-align:left;">
negative
</td>
<td style="text-align:left;">
MGI_C57BL6J_1890508
</td>
<td style="text-align:left;">
Zmpste24
</td>
<td style="text-align:left;">
NCBI_Gene:230709,ENSEMBL:ENSMUSG00000043207
</td>
<td style="text-align:left;">
MGI:1890508
</td>
<td style="text-align:left;">
protein coding gene
</td>
<td style="text-align:left;">
zinc metallopeptidase, STE24
</td>
</tr>
<tr>
<td style="text-align:left;">
4
</td>
<td style="text-align:left;">
gene
</td>
<td style="text-align:right;">
121.0758
</td>
<td style="text-align:right;">
121.0759
</td>
<td style="text-align:left;">
negative
</td>
<td style="text-align:left;">
MGI_C57BL6J_5455963
</td>
<td style="text-align:left;">
Gm26186
</td>
<td style="text-align:left;">
ENSEMBL:ENSMUSG00000088314
</td>
<td style="text-align:left;">
MGI:5455963
</td>
<td style="text-align:left;">
snoRNA gene
</td>
<td style="text-align:left;">
predicted gene, 26186
</td>
</tr>
<tr>
<td style="text-align:left;">
4
</td>
<td style="text-align:left;">
gene
</td>
<td style="text-align:right;">
121.0840
</td>
<td style="text-align:right;">
121.0841
</td>
<td style="text-align:left;">
negative
</td>
<td style="text-align:left;">
MGI_C57BL6J_5453317
</td>
<td style="text-align:left;">
Gm23540
</td>
<td style="text-align:left;">
ENSEMBL:ENSMUSG00000088168
</td>
<td style="text-align:left;">
MGI:5453317
</td>
<td style="text-align:left;">
snRNA gene
</td>
<td style="text-align:left;">
predicted gene, 23540
</td>
</tr>
<tr>
<td style="text-align:left;">
4
</td>
<td style="text-align:left;">
gene
</td>
<td style="text-align:right;">
121.1057
</td>
<td style="text-align:right;">
121.1092
</td>
<td style="text-align:left;">
negative
</td>
<td style="text-align:left;">
MGI_C57BL6J_1916719
</td>
<td style="text-align:left;">
Tmco2
</td>
<td style="text-align:left;">
NCBI_Gene:69469,ENSEMBL:ENSMUSG00000078577
</td>
<td style="text-align:left;">
MGI:1916719
</td>
<td style="text-align:left;">
protein coding gene
</td>
<td style="text-align:left;">
transmembrane and coiled-coil domains 2
</td>
</tr>
<tr>
<td style="text-align:left;">
4
</td>
<td style="text-align:left;">
pseudogene
</td>
<td style="text-align:right;">
121.1148
</td>
<td style="text-align:right;">
121.1153
</td>
<td style="text-align:left;">
negative
</td>
<td style="text-align:left;">
MGI_C57BL6J_3651897
</td>
<td style="text-align:left;">
Gm12882
</td>
<td style="text-align:left;">
NCBI_Gene:100042174,ENSEMBL:ENSMUSG00000081423
</td>
<td style="text-align:left;">
MGI:3651897
</td>
<td style="text-align:left;">
pseudogene
</td>
<td style="text-align:left;">
predicted gene 12882
</td>
</tr>
<tr>
<td style="text-align:left;">
4
</td>
<td style="text-align:left;">
pseudogene
</td>
<td style="text-align:right;">
121.1209
</td>
<td style="text-align:right;">
121.1302
</td>
<td style="text-align:left;">
negative
</td>
<td style="text-align:left;">
MGI_C57BL6J_3651892
</td>
<td style="text-align:left;">
Gm12881
</td>
<td style="text-align:left;">
NCBI_Gene:102640296,ENSEMBL:ENSMUSG00000082891
</td>
<td style="text-align:left;">
MGI:3651892
</td>
<td style="text-align:left;">
pseudogene
</td>
<td style="text-align:left;">
predicted gene 12881
</td>
</tr>
<tr>
<td style="text-align:left;">
4
</td>
<td style="text-align:left;">
pseudogene
</td>
<td style="text-align:right;">
121.1301
</td>
<td style="text-align:right;">
121.1313
</td>
<td style="text-align:left;">
positive
</td>
<td style="text-align:left;">
MGI_C57BL6J_3649632
</td>
<td style="text-align:left;">
Gm12879
</td>
<td style="text-align:left;">
ENSEMBL:ENSMUSG00000081305
</td>
<td style="text-align:left;">
MGI:3649632
</td>
<td style="text-align:left;">
pseudogene
</td>
<td style="text-align:left;">
predicted gene 12879
</td>
</tr>
<tr>
<td style="text-align:left;">
4
</td>
<td style="text-align:left;">
gene
</td>
<td style="text-align:right;">
121.1347
</td>
<td style="text-align:right;">
121.1350
</td>
<td style="text-align:left;">
positive
</td>
<td style="text-align:left;">
MGI_C57BL6J_5452759
</td>
<td style="text-align:left;">
Gm22982
</td>
<td style="text-align:left;">
ENSEMBL:ENSMUSG00000064547
</td>
<td style="text-align:left;">
MGI:5452759
</td>
<td style="text-align:left;">
unclassified non-coding RNA gene
</td>
<td style="text-align:left;">
predicted gene, 22982
</td>
</tr>
<tr>
<td style="text-align:left;">
4
</td>
<td style="text-align:left;">
gene
</td>
<td style="text-align:right;">
121.1454
</td>
<td style="text-align:right;">
121.2151
</td>
<td style="text-align:left;">
negative
</td>
<td style="text-align:left;">
MGI_C57BL6J_1924705
</td>
<td style="text-align:left;">
Rlf
</td>
<td style="text-align:left;">
NCBI_Gene:109263,ENSEMBL:ENSMUSG00000049878
</td>
<td style="text-align:left;">
MGI:1924705
</td>
<td style="text-align:left;">
protein coding gene
</td>
<td style="text-align:left;">
rearranged L-myc fusion sequence
</td>
</tr>
<tr>
<td style="text-align:left;">
4
</td>
<td style="text-align:left;">
pseudogene
</td>
<td style="text-align:right;">
121.2271
</td>
<td style="text-align:right;">
121.2275
</td>
<td style="text-align:left;">
negative
</td>
<td style="text-align:left;">
MGI_C57BL6J_3652134
</td>
<td style="text-align:left;">
Gm12890
</td>
<td style="text-align:left;">
ENSEMBL:ENSMUSG00000080706
</td>
<td style="text-align:left;">
MGI:3652134
</td>
<td style="text-align:left;">
pseudogene
</td>
<td style="text-align:left;">
predicted gene 12890
</td>
</tr>
<tr>
<td style="text-align:left;">
4
</td>
<td style="text-align:left;">
gene
</td>
<td style="text-align:right;">
121.2309
</td>
<td style="text-align:right;">
121.2392
</td>
<td style="text-align:left;">
positive
</td>
<td style="text-align:left;">
MGI_C57BL6J_5589261
</td>
<td style="text-align:left;">
Gm30102
</td>
<td style="text-align:left;">
NCBI_Gene:102631880
</td>
<td style="text-align:left;">
MGI:5589261
</td>
<td style="text-align:left;">
lncRNA gene
</td>
<td style="text-align:left;">
predicted gene, 30102
</td>
</tr>
<tr>
<td style="text-align:left;">
4
</td>
<td style="text-align:left;">
gene
</td>
<td style="text-align:right;">
121.2357
</td>
<td style="text-align:right;">
121.2415
</td>
<td style="text-align:left;">
negative
</td>
<td style="text-align:left;">
MGI_C57BL6J_5623124
</td>
<td style="text-align:left;">
Gm40239
</td>
<td style="text-align:left;">
NCBI_Gene:105244675
</td>
<td style="text-align:left;">
MGI:5623124
</td>
<td style="text-align:left;">
lncRNA gene
</td>
<td style="text-align:left;">
predicted gene, 40239
</td>
</tr>
<tr>
<td style="text-align:left;">
4
</td>
<td style="text-align:left;">
gene
</td>
<td style="text-align:right;">
121.3163
</td>
<td style="text-align:right;">
121.3249
</td>
<td style="text-align:left;">
negative
</td>
<td style="text-align:left;">
MGI_C57BL6J_3652130
</td>
<td style="text-align:left;">
Gm12888
</td>
<td style="text-align:left;">
NCBI_Gene:545677,ENSEMBL:ENSMUSG00000073764
</td>
<td style="text-align:left;">
MGI:3652130
</td>
<td style="text-align:left;">
protein coding gene
</td>
<td style="text-align:left;">
predicted gene 12888
</td>
</tr>
<tr>
<td style="text-align:left;">
4
</td>
<td style="text-align:left;">
pseudogene
</td>
<td style="text-align:right;">
121.3798
</td>
<td style="text-align:right;">
121.3800
</td>
<td style="text-align:left;">
positive
</td>
<td style="text-align:left;">
MGI_C57BL6J_3652132
</td>
<td style="text-align:left;">
Gm12889
</td>
<td style="text-align:left;">
ENSEMBL:ENSMUSG00000083941
</td>
<td style="text-align:left;">
MGI:3652132
</td>
<td style="text-align:left;">
pseudogene
</td>
<td style="text-align:left;">
predicted gene 12889
</td>
</tr>
<tr>
<td style="text-align:left;">
4
</td>
<td style="text-align:left;">
pseudogene
</td>
<td style="text-align:right;">
121.3806
</td>
<td style="text-align:right;">
121.3809
</td>
<td style="text-align:left;">
positive
</td>
<td style="text-align:left;">
MGI_C57BL6J_3651896
</td>
<td style="text-align:left;">
Gm12885
</td>
<td style="text-align:left;">
ENSEMBL:ENSMUSG00000083901
</td>
<td style="text-align:left;">
MGI:3651896
</td>
<td style="text-align:left;">
pseudogene
</td>
<td style="text-align:left;">
predicted gene 12885
</td>
</tr>
<tr>
<td style="text-align:left;">
4
</td>
<td style="text-align:left;">
pseudogene
</td>
<td style="text-align:right;">
121.3814
</td>
<td style="text-align:right;">
121.3818
</td>
<td style="text-align:left;">
positive
</td>
<td style="text-align:left;">
MGI_C57BL6J_3651895
</td>
<td style="text-align:left;">
Gm12884
</td>
<td style="text-align:left;">
ENSEMBL:ENSMUSG00000081465
</td>
<td style="text-align:left;">
MGI:3651895
</td>
<td style="text-align:left;">
pseudogene
</td>
<td style="text-align:left;">
predicted gene 12884
</td>
</tr>
<tr>
<td style="text-align:left;">
4
</td>
<td style="text-align:left;">
pseudogene
</td>
<td style="text-align:right;">
121.3930
</td>
<td style="text-align:right;">
121.3966
</td>
<td style="text-align:left;">
positive
</td>
<td style="text-align:left;">
MGI_C57BL6J_3651898
</td>
<td style="text-align:left;">
Gm12883
</td>
<td style="text-align:left;">
ENSEMBL:ENSMUSG00000081667
</td>
<td style="text-align:left;">
MGI:3651898
</td>
<td style="text-align:left;">
pseudogene
</td>
<td style="text-align:left;">
predicted gene 12883
</td>
</tr>
<tr>
<td style="text-align:left;">
4
</td>
<td style="text-align:left;">
gene
</td>
<td style="text-align:right;">
121.3942
</td>
<td style="text-align:right;">
121.3966
</td>
<td style="text-align:left;">
negative
</td>
<td style="text-align:left;">
MGI_C57BL6J_3648197
</td>
<td style="text-align:left;">
Gm8359
</td>
<td style="text-align:left;">
NCBI_Gene:666914
</td>
<td style="text-align:left;">
MGI:3648197
</td>
<td style="text-align:left;">
protein coding gene
</td>
<td style="text-align:left;">
predicted gene 8359
</td>
</tr>
<tr>
<td style="text-align:left;">
4
</td>
<td style="text-align:left;">
gene
</td>
<td style="text-align:right;">
121.4147
</td>
<td style="text-align:right;">
121.4231
</td>
<td style="text-align:left;">
negative
</td>
<td style="text-align:left;">
MGI_C57BL6J_3651888
</td>
<td style="text-align:left;">
Gm12886
</td>
<td style="text-align:left;">
NCBI_Gene:666921,ENSEMBL:ENSMUSG00000078576
</td>
<td style="text-align:left;">
MGI:3651888
</td>
<td style="text-align:left;">
protein coding gene
</td>
<td style="text-align:left;">
predicted gene 12886
</td>
</tr>
<tr>
<td style="text-align:left;">
4
</td>
<td style="text-align:left;">
pseudogene
</td>
<td style="text-align:right;">
121.4768
</td>
<td style="text-align:right;">
121.5828
</td>
<td style="text-align:left;">
negative
</td>
<td style="text-align:left;">
MGI_C57BL6J_3651893
</td>
<td style="text-align:left;">
Gm12880
</td>
<td style="text-align:left;">
NCBI_Gene:100504652,ENSEMBL:ENSMUSG00000082821
</td>
<td style="text-align:left;">
MGI:3651893
</td>
<td style="text-align:left;">
pseudogene
</td>
<td style="text-align:left;">
predicted gene 12880
</td>
</tr>
<tr>
<td style="text-align:left;">
4
</td>
<td style="text-align:left;">
gene
</td>
<td style="text-align:right;">
121.5095
</td>
<td style="text-align:right;">
121.5096
</td>
<td style="text-align:left;">
positive
</td>
<td style="text-align:left;">
MGI_C57BL6J_5455939
</td>
<td style="text-align:left;">
Gm26162
</td>
<td style="text-align:left;">
ENSEMBL:ENSMUSG00000088418
</td>
<td style="text-align:left;">
MGI:5455939
</td>
<td style="text-align:left;">
snoRNA gene
</td>
<td style="text-align:left;">
predicted gene, 26162
</td>
</tr>
<tr>
<td style="text-align:left;">
4
</td>
<td style="text-align:left;">
gene
</td>
<td style="text-align:right;">
121.6140
</td>
<td style="text-align:right;">
121.6221
</td>
<td style="text-align:left;">
negative
</td>
<td style="text-align:left;">
MGI_C57BL6J_3652131
</td>
<td style="text-align:left;">
Gm12887
</td>
<td style="text-align:left;">
NCBI_Gene:666927,ENSEMBL:ENSMUSG00000078575
</td>
<td style="text-align:left;">
MGI:3652131
</td>
<td style="text-align:left;">
protein coding gene
</td>
<td style="text-align:left;">
predicted gene 12887
</td>
</tr>
<tr>
<td style="text-align:left;">
4
</td>
<td style="text-align:left;">
pseudogene
</td>
<td style="text-align:right;">
121.6414
</td>
<td style="text-align:right;">
121.6418
</td>
<td style="text-align:left;">
positive
</td>
<td style="text-align:left;">
MGI_C57BL6J_3649631
</td>
<td style="text-align:left;">
Gm12878
</td>
<td style="text-align:left;">
ENSEMBL:ENSMUSG00000083615
</td>
<td style="text-align:left;">
MGI:3649631
</td>
<td style="text-align:left;">
pseudogene
</td>
<td style="text-align:left;">
predicted gene 12878
</td>
</tr>
<tr>
<td style="text-align:left;">
4
</td>
<td style="text-align:left;">
gene
</td>
<td style="text-align:right;">
121.9079
</td>
<td style="text-align:right;">
121.9208
</td>
<td style="text-align:left;">
negative
</td>
<td style="text-align:left;">
MGI_C57BL6J_5434759
</td>
<td style="text-align:left;">
Gm21404
</td>
<td style="text-align:left;">
NCBI_Gene:100862019
</td>
<td style="text-align:left;">
MGI:5434759
</td>
<td style="text-align:left;">
protein coding gene
</td>
<td style="text-align:left;">
predicted gene, 21404
</td>
</tr>
<tr>
<td style="text-align:left;">
4
</td>
<td style="text-align:left;">
pseudogene
</td>
<td style="text-align:right;">
122.0081
</td>
<td style="text-align:right;">
122.0096
</td>
<td style="text-align:left;">
positive
</td>
<td style="text-align:left;">
MGI_C57BL6J_3649649
</td>
<td style="text-align:left;">
Gm12892
</td>
<td style="text-align:left;">
NCBI_Gene:666937,ENSEMBL:ENSMUSG00000083679
</td>
<td style="text-align:left;">
MGI:3649649
</td>
<td style="text-align:left;">
pseudogene
</td>
<td style="text-align:left;">
predicted gene 12892
</td>
</tr>
<tr>
<td style="text-align:left;">
4
</td>
<td style="text-align:left;">
pseudogene
</td>
<td style="text-align:right;">
122.1228
</td>
<td style="text-align:right;">
122.1231
</td>
<td style="text-align:left;">
positive
</td>
<td style="text-align:left;">
MGI_C57BL6J_3650312
</td>
<td style="text-align:left;">
Gm12896
</td>
<td style="text-align:left;">
ENSEMBL:ENSMUSG00000083558
</td>
<td style="text-align:left;">
MGI:3650312
</td>
<td style="text-align:left;">
pseudogene
</td>
<td style="text-align:left;">
predicted gene 12896
</td>
</tr>
<tr>
<td style="text-align:left;">
4
</td>
<td style="text-align:left;">
pseudogene
</td>
<td style="text-align:right;">
122.1877
</td>
<td style="text-align:right;">
122.2067
</td>
<td style="text-align:left;">
positive
</td>
<td style="text-align:left;">
MGI_C57BL6J_3649906
</td>
<td style="text-align:left;">
Gm12897
</td>
<td style="text-align:left;">
NCBI_Gene:102632107,ENSEMBL:ENSMUSG00000083949
</td>
<td style="text-align:left;">
MGI:3649906
</td>
<td style="text-align:left;">
pseudogene
</td>
<td style="text-align:left;">
predicted gene 12897
</td>
</tr>
<tr>
<td style="text-align:left;">
4
</td>
<td style="text-align:left;">
pseudogene
</td>
<td style="text-align:right;">
122.2350
</td>
<td style="text-align:right;">
122.2359
</td>
<td style="text-align:left;">
positive
</td>
<td style="text-align:left;">
MGI_C57BL6J_3652102
</td>
<td style="text-align:left;">
Gm12893
</td>
<td style="text-align:left;">
ENSEMBL:ENSMUSG00000083867
</td>
<td style="text-align:left;">
MGI:3652102
</td>
<td style="text-align:left;">
pseudogene
</td>
<td style="text-align:left;">
predicted gene 12893
</td>
</tr>
<tr>
<td style="text-align:left;">
4
</td>
<td style="text-align:left;">
gene
</td>
<td style="text-align:right;">
122.3009
</td>
<td style="text-align:right;">
122.3138
</td>
<td style="text-align:left;">
negative
</td>
<td style="text-align:left;">
MGI_C57BL6J_3649909
</td>
<td style="text-align:left;">
Gm10573
</td>
<td style="text-align:left;">
NCBI_Gene:666931
</td>
<td style="text-align:left;">
MGI:3649909
</td>
<td style="text-align:left;">
protein coding gene
</td>
<td style="text-align:left;">
predicted gene 10573
</td>
</tr>
<tr>
<td style="text-align:left;">
4
</td>
<td style="text-align:left;">
pseudogene
</td>
<td style="text-align:right;">
122.4552
</td>
<td style="text-align:right;">
122.4554
</td>
<td style="text-align:left;">
positive
</td>
<td style="text-align:left;">
MGI_C57BL6J_3649907
</td>
<td style="text-align:left;">
Gm12895
</td>
<td style="text-align:left;">
ENSEMBL:ENSMUSG00000083365
</td>
<td style="text-align:left;">
MGI:3649907
</td>
<td style="text-align:left;">
pseudogene
</td>
<td style="text-align:left;">
predicted gene 12895
</td>
</tr>
<tr>
<td style="text-align:left;">
4
</td>
<td style="text-align:left;">
gene
</td>
<td style="text-align:right;">
122.5202
</td>
<td style="text-align:right;">
122.5392
</td>
<td style="text-align:left;">
positive
</td>
<td style="text-align:left;">
MGI_C57BL6J_5589564
</td>
<td style="text-align:left;">
Gm30405
</td>
<td style="text-align:left;">
NCBI_Gene:102632293
</td>
<td style="text-align:left;">
MGI:5589564
</td>
<td style="text-align:left;">
lncRNA gene
</td>
<td style="text-align:left;">
predicted gene, 30405
</td>
</tr>
<tr>
<td style="text-align:left;">
4
</td>
<td style="text-align:left;">
pseudogene
</td>
<td style="text-align:right;">
122.5827
</td>
<td style="text-align:right;">
122.6049
</td>
<td style="text-align:left;">
negative
</td>
<td style="text-align:left;">
MGI_C57BL6J_3650567
</td>
<td style="text-align:left;">
Gm12877
</td>
<td style="text-align:left;">
ENSEMBL:ENSMUSG00000081377
</td>
<td style="text-align:left;">
MGI:3650567
</td>
<td style="text-align:left;">
pseudogene
</td>
<td style="text-align:left;">
predicted gene 12877
</td>
</tr>
<tr>
<td style="text-align:left;">
4
</td>
<td style="text-align:left;">
pseudogene
</td>
<td style="text-align:right;">
122.5930
</td>
<td style="text-align:right;">
122.5941
</td>
<td style="text-align:left;">
positive
</td>
<td style="text-align:left;">
MGI_C57BL6J_3651901
</td>
<td style="text-align:left;">
Gm12874
</td>
<td style="text-align:left;">
NCBI_Gene:100504420,ENSEMBL:ENSMUSG00000084196
</td>
<td style="text-align:left;">
MGI:3651901
</td>
<td style="text-align:left;">
pseudogene
</td>
<td style="text-align:left;">
predicted gene 12874
</td>
</tr>
<tr>
<td style="text-align:left;">
4
</td>
<td style="text-align:left;">
pseudogene
</td>
<td style="text-align:right;">
122.6403
</td>
<td style="text-align:right;">
122.6483
</td>
<td style="text-align:left;">
negative
</td>
<td style="text-align:left;">
MGI_C57BL6J_3649744
</td>
<td style="text-align:left;">
Gm12875
</td>
<td style="text-align:left;">
ENSEMBL:ENSMUSG00000080871
</td>
<td style="text-align:left;">
MGI:3649744
</td>
<td style="text-align:left;">
pseudogene
</td>
<td style="text-align:left;">
predicted gene 12875
</td>
</tr>
<tr>
<td style="text-align:left;">
4
</td>
<td style="text-align:left;">
pseudogene
</td>
<td style="text-align:right;">
122.6609
</td>
<td style="text-align:right;">
122.6773
</td>
<td style="text-align:left;">
negative
</td>
<td style="text-align:left;">
MGI_C57BL6J_3651904
</td>
<td style="text-align:left;">
Gm12876
</td>
<td style="text-align:left;">
ENSEMBL:ENSMUSG00000083109
</td>
<td style="text-align:left;">
MGI:3651904
</td>
<td style="text-align:left;">
pseudogene
</td>
<td style="text-align:left;">
predicted gene 12876
</td>
</tr>
<tr>
<td style="text-align:left;">
4
</td>
<td style="text-align:left;">
gene
</td>
<td style="text-align:right;">
122.6893
</td>
<td style="text-align:right;">
122.7051
</td>
<td style="text-align:left;">
positive
</td>
<td style="text-align:left;">
MGI_C57BL6J_1924682
</td>
<td style="text-align:left;">
9530002B09Rik
</td>
<td style="text-align:left;">
NCBI_Gene:77432,ENSEMBL:ENSMUSG00000023263
</td>
<td style="text-align:left;">
MGI:1924682
</td>
<td style="text-align:left;">
protein coding gene
</td>
<td style="text-align:left;">
RIKEN cDNA 9530002B09 gene
</td>
</tr>
<tr>
<td style="text-align:left;">
4
</td>
<td style="text-align:left;">
gene
</td>
<td style="text-align:right;">
122.7265
</td>
<td style="text-align:right;">
122.7395
</td>
<td style="text-align:left;">
negative
</td>
<td style="text-align:left;">
MGI_C57BL6J_5589707
</td>
<td style="text-align:left;">
Gm30548
</td>
<td style="text-align:left;">
NCBI_Gene:102632489
</td>
<td style="text-align:left;">
MGI:5589707
</td>
<td style="text-align:left;">
lncRNA gene
</td>
<td style="text-align:left;">
predicted gene, 30548
</td>
</tr>
<tr>
<td style="text-align:left;">
4
</td>
<td style="text-align:left;">
pseudogene
</td>
<td style="text-align:right;">
122.7303
</td>
<td style="text-align:right;">
122.7347
</td>
<td style="text-align:left;">
negative
</td>
<td style="text-align:left;">
MGI_C57BL6J_3649389
</td>
<td style="text-align:left;">
Gm12872
</td>
<td style="text-align:left;">
ENSEMBL:ENSMUSG00000084082
</td>
<td style="text-align:left;">
MGI:3649389
</td>
<td style="text-align:left;">
pseudogene
</td>
<td style="text-align:left;">
predicted gene 12872
</td>
</tr>
<tr>
<td style="text-align:left;">
4
</td>
<td style="text-align:left;">
pseudogene
</td>
<td style="text-align:right;">
122.7715
</td>
<td style="text-align:right;">
122.7938
</td>
<td style="text-align:left;">
negative
</td>
<td style="text-align:left;">
MGI_C57BL6J_3651900
</td>
<td style="text-align:left;">
Gm12873
</td>
<td style="text-align:left;">
NCBI_Gene:102632555,ENSEMBL:ENSMUSG00000083494
</td>
<td style="text-align:left;">
MGI:3651900
</td>
<td style="text-align:left;">
pseudogene
</td>
<td style="text-align:left;">
predicted gene 12873
</td>
</tr>
<tr>
<td style="text-align:left;">
4
</td>
<td style="text-align:left;">
gene
</td>
<td style="text-align:right;">
122.8361
</td>
<td style="text-align:right;">
122.8592
</td>
<td style="text-align:left;">
positive
</td>
<td style="text-align:left;">
MGI_C57BL6J_1298204
</td>
<td style="text-align:left;">
Ppt1
</td>
<td style="text-align:left;">
NCBI_Gene:19063,ENSEMBL:ENSMUSG00000028657
</td>
<td style="text-align:left;">
MGI:1298204
</td>
<td style="text-align:left;">
protein coding gene
</td>
<td style="text-align:left;">
palmitoyl-protein thioesterase 1
</td>
</tr>
<tr>
<td style="text-align:left;">
4
</td>
<td style="text-align:left;">
gene
</td>
<td style="text-align:right;">
122.8590
</td>
<td style="text-align:right;">
122.8861
</td>
<td style="text-align:left;">
negative
</td>
<td style="text-align:left;">
MGI_C57BL6J_88262
</td>
<td style="text-align:left;">
Cap1
</td>
<td style="text-align:left;">
NCBI_Gene:12331,ENSEMBL:ENSMUSG00000028656
</td>
<td style="text-align:left;">
MGI:88262
</td>
<td style="text-align:left;">
protein coding gene
</td>
<td style="text-align:left;">
CAP, adenylate cyclase-associated protein 1 (yeast)
</td>
</tr>
<tr>
<td style="text-align:left;">
4
</td>
<td style="text-align:left;">
pseudogene
</td>
<td style="text-align:right;">
122.9258
</td>
<td style="text-align:right;">
122.9262
</td>
<td style="text-align:left;">
positive
</td>
<td style="text-align:left;">
MGI_C57BL6J_3649646
</td>
<td style="text-align:left;">
Gm12891
</td>
<td style="text-align:left;">
NCBI_Gene:102632907,ENSEMBL:ENSMUSG00000080929
</td>
<td style="text-align:left;">
MGI:3649646
</td>
<td style="text-align:left;">
pseudogene
</td>
<td style="text-align:left;">
predicted gene 12891
</td>
</tr>
<tr>
<td style="text-align:left;">
4
</td>
<td style="text-align:left;">
gene
</td>
<td style="text-align:right;">
122.9468
</td>
<td style="text-align:right;">
122.9612
</td>
<td style="text-align:left;">
negative
</td>
<td style="text-align:left;">
MGI_C57BL6J_1923824
</td>
<td style="text-align:left;">
Mfsd2a
</td>
<td style="text-align:left;">
NCBI_Gene:76574,ENSEMBL:ENSMUSG00000028655
</td>
<td style="text-align:left;">
MGI:1923824
</td>
<td style="text-align:left;">
protein coding gene
</td>
<td style="text-align:left;">
major facilitator superfamily domain containing 2A
</td>
</tr>
<tr>
<td style="text-align:left;">
4
</td>
<td style="text-align:left;">
gene
</td>
<td style="text-align:right;">
122.9613
</td>
<td style="text-align:right;">
122.9635
</td>
<td style="text-align:left;">
positive
</td>
<td style="text-align:left;">
MGI_C57BL6J_1918347
</td>
<td style="text-align:left;">
4933421A08Rik
</td>
<td style="text-align:left;">
ENSEMBL:ENSMUSG00000086443
</td>
<td style="text-align:left;">
MGI:1918347
</td>
<td style="text-align:left;">
lncRNA gene
</td>
<td style="text-align:left;">
RIKEN cDNA 4933421A08 gene
</td>
</tr>
<tr>
<td style="text-align:left;">
4
</td>
<td style="text-align:left;">
gene
</td>
<td style="text-align:right;">
122.9956
</td>
<td style="text-align:right;">
123.0025
</td>
<td style="text-align:left;">
positive
</td>
<td style="text-align:left;">
MGI_C57BL6J_96799
</td>
<td style="text-align:left;">
Mycl
</td>
<td style="text-align:left;">
NCBI_Gene:16918,ENSEMBL:ENSMUSG00000028654
</td>
<td style="text-align:left;">
MGI:96799
</td>
<td style="text-align:left;">
protein coding gene
</td>
<td style="text-align:left;">
v-myc avian myelocytomatosis viral oncogene lung carcinoma derived
</td>
</tr>
<tr>
<td style="text-align:left;">
4
</td>
<td style="text-align:left;">
gene
</td>
<td style="text-align:right;">
123.0165
</td>
<td style="text-align:right;">
123.0549
</td>
<td style="text-align:left;">
positive
</td>
<td style="text-align:left;">
MGI_C57BL6J_1914216
</td>
<td style="text-align:left;">
Trit1
</td>
<td style="text-align:left;">
NCBI_Gene:66966,ENSEMBL:ENSMUSG00000028653
</td>
<td style="text-align:left;">
MGI:1914216
</td>
<td style="text-align:left;">
protein coding gene
</td>
<td style="text-align:left;">
tRNA isopentenyltransferase 1
</td>
</tr>
<tr>
<td style="text-align:left;">
4
</td>
<td style="text-align:left;">
gene
</td>
<td style="text-align:right;">
123.0812
</td>
<td style="text-align:right;">
123.0894
</td>
<td style="text-align:left;">
positive
</td>
<td style="text-align:left;">
MGI_C57BL6J_5826505
</td>
<td style="text-align:left;">
Gm46868
</td>
<td style="text-align:left;">
NCBI_Gene:108168975
</td>
<td style="text-align:left;">
MGI:5826505
</td>
<td style="text-align:left;">
lncRNA gene
</td>
<td style="text-align:left;">
predicted gene, 46868
</td>
</tr>
<tr>
<td style="text-align:left;">
4
</td>
<td style="text-align:left;">
gene
</td>
<td style="text-align:right;">
123.0812
</td>
<td style="text-align:right;">
123.0940
</td>
<td style="text-align:left;">
negative
</td>
<td style="text-align:left;">
MGI_C57BL6J_3651539
</td>
<td style="text-align:left;">
Gm12923
</td>
<td style="text-align:left;">
NCBI_Gene:105244676,ENSEMBL:ENSMUSG00000086153
</td>
<td style="text-align:left;">
MGI:3651539
</td>
<td style="text-align:left;">
lncRNA gene
</td>
<td style="text-align:left;">
predicted gene 12923
</td>
</tr>
<tr>
<td style="text-align:left;">
4
</td>
<td style="text-align:left;">
gene
</td>
<td style="text-align:right;">
123.1043
</td>
<td style="text-align:right;">
123.1321
</td>
<td style="text-align:left;">
positive
</td>
<td style="text-align:left;">
MGI_C57BL6J_107335
</td>
<td style="text-align:left;">
Bmp8b
</td>
<td style="text-align:left;">
NCBI_Gene:12164,ENSEMBL:ENSMUSG00000002384
</td>
<td style="text-align:left;">
MGI:107335
</td>
<td style="text-align:left;">
protein coding gene
</td>
<td style="text-align:left;">
bone morphogenetic protein 8b
</td>
</tr>
<tr>
<td style="text-align:left;">
4
</td>
<td style="text-align:left;">
gene
</td>
<td style="text-align:right;">
123.1133
</td>
<td style="text-align:right;">
123.1168
</td>
<td style="text-align:left;">
negative
</td>
<td style="text-align:left;">
MGI_C57BL6J_4936878
</td>
<td style="text-align:left;">
Gm17244
</td>
<td style="text-align:left;">
ENSEMBL:ENSMUSG00000102997
</td>
<td style="text-align:left;">
MGI:4936878
</td>
<td style="text-align:left;">
unclassified gene
</td>
<td style="text-align:left;">
predicted gene, 17244
</td>
</tr>
<tr>
<td style="text-align:left;">
4
</td>
<td style="text-align:left;">
gene
</td>
<td style="text-align:right;">
123.1162
</td>
<td style="text-align:right;">
123.1180
</td>
<td style="text-align:left;">
positive
</td>
<td style="text-align:left;">
MGI_C57BL6J_2664115
</td>
<td style="text-align:left;">
Oxct2b
</td>
<td style="text-align:left;">
NCBI_Gene:353371,ENSEMBL:ENSMUSG00000076438
</td>
<td style="text-align:left;">
MGI:2664115
</td>
<td style="text-align:left;">
protein coding gene
</td>
<td style="text-align:left;">
3-oxoacid CoA transferase 2B
</td>
</tr>
<tr>
<td style="text-align:left;">
4
</td>
<td style="text-align:left;">
gene
</td>
<td style="text-align:right;">
123.1271
</td>
<td style="text-align:right;">
123.1400
</td>
<td style="text-align:left;">
negative
</td>
<td style="text-align:left;">
MGI_C57BL6J_1917118
</td>
<td style="text-align:left;">
Ppie
</td>
<td style="text-align:left;">
NCBI_Gene:56031,ENSEMBL:ENSMUSG00000028651
</td>
<td style="text-align:left;">
MGI:1917118
</td>
<td style="text-align:left;">
protein coding gene
</td>
<td style="text-align:left;">
peptidylprolyl isomerase E (cyclophilin E)
</td>
</tr>
<tr>
<td style="text-align:left;">
4
</td>
<td style="text-align:left;">
gene
</td>
<td style="text-align:right;">
123.1832
</td>
<td style="text-align:right;">
123.1947
</td>
<td style="text-align:left;">
positive
</td>
<td style="text-align:left;">
MGI_C57BL6J_2157521
</td>
<td style="text-align:left;">
Hpcal4
</td>
<td style="text-align:left;">
NCBI_Gene:170638,ENSEMBL:ENSMUSG00000046093
</td>
<td style="text-align:left;">
MGI:2157521
</td>
<td style="text-align:left;">
protein coding gene
</td>
<td style="text-align:left;">
hippocalcin-like 4
</td>
</tr>
<tr>
<td style="text-align:left;">
4
</td>
<td style="text-align:left;">
gene
</td>
<td style="text-align:right;">
123.2012
</td>
<td style="text-align:right;">
123.2253
</td>
<td style="text-align:left;">
positive
</td>
<td style="text-align:left;">
MGI_C57BL6J_2155700
</td>
<td style="text-align:left;">
Nt5c1a
</td>
<td style="text-align:left;">
NCBI_Gene:230718,ENSEMBL:ENSMUSG00000054958
</td>
<td style="text-align:left;">
MGI:2155700
</td>
<td style="text-align:left;">
protein coding gene
</td>
<td style="text-align:left;">
5’-nucleotidase, cytosolic IA
</td>
</tr>
<tr>
<td style="text-align:left;">
4
</td>
<td style="text-align:left;">
gene
</td>
<td style="text-align:right;">
123.2336
</td>
<td style="text-align:right;">
123.2499
</td>
<td style="text-align:left;">
positive
</td>
<td style="text-align:left;">
MGI_C57BL6J_1860511
</td>
<td style="text-align:left;">
Heyl
</td>
<td style="text-align:left;">
NCBI_Gene:56198,ENSEMBL:ENSMUSG00000032744
</td>
<td style="text-align:left;">
MGI:1860511
</td>
<td style="text-align:left;">
protein coding gene
</td>
<td style="text-align:left;">
hairy/enhancer-of-split related with YRPW motif-like
</td>
</tr>
<tr>
<td style="text-align:left;">
4
</td>
<td style="text-align:left;">
pseudogene
</td>
<td style="text-align:right;">
123.2580
</td>
<td style="text-align:right;">
123.2589
</td>
<td style="text-align:left;">
positive
</td>
<td style="text-align:left;">
MGI_C57BL6J_3650176
</td>
<td style="text-align:left;">
Gm12901
</td>
<td style="text-align:left;">
NCBI_Gene:194197,ENSEMBL:ENSMUSG00000083720
</td>
<td style="text-align:left;">
MGI:3650176
</td>
<td style="text-align:left;">
pseudogene
</td>
<td style="text-align:left;">
predicted gene 12901
</td>
</tr>
<tr>
<td style="text-align:left;">
4
</td>
<td style="text-align:left;">
gene
</td>
<td style="text-align:right;">
123.2624
</td>
<td style="text-align:right;">
123.2989
</td>
<td style="text-align:left;">
positive
</td>
<td style="text-align:left;">
MGI_C57BL6J_2385206
</td>
<td style="text-align:left;">
Pabpc4
</td>
<td style="text-align:left;">
NCBI_Gene:230721,ENSEMBL:ENSMUSG00000011257
</td>
<td style="text-align:left;">
MGI:2385206
</td>
<td style="text-align:left;">
protein coding gene
</td>
<td style="text-align:left;">
poly(A) binding protein, cytoplasmic 4
</td>
</tr>
<tr>
<td style="text-align:left;">
4
</td>
<td style="text-align:left;">
pseudogene
</td>
<td style="text-align:right;">
123.2749
</td>
<td style="text-align:right;">
123.2755
</td>
<td style="text-align:left;">
negative
</td>
<td style="text-align:left;">
MGI_C57BL6J_3650387
</td>
<td style="text-align:left;">
Gm12900
</td>
<td style="text-align:left;">
NCBI_Gene:622334,ENSEMBL:ENSMUSG00000082634
</td>
<td style="text-align:left;">
MGI:3650387
</td>
<td style="text-align:left;">
pseudogene
</td>
<td style="text-align:left;">
predicted gene 12900
</td>
</tr>
<tr>
<td style="text-align:left;">
4
</td>
<td style="text-align:left;">
gene
</td>
<td style="text-align:right;">
123.2913
</td>
<td style="text-align:right;">
123.2915
</td>
<td style="text-align:left;">
positive
</td>
<td style="text-align:left;">
MGI_C57BL6J_5455565
</td>
<td style="text-align:left;">
Gm25788
</td>
<td style="text-align:left;">
ENSEMBL:ENSMUSG00000064655
</td>
<td style="text-align:left;">
MGI:5455565
</td>
<td style="text-align:left;">
snoRNA gene
</td>
<td style="text-align:left;">
predicted gene, 25788
</td>
</tr>
<tr>
<td style="text-align:left;">
4
</td>
<td style="text-align:left;">
gene
</td>
<td style="text-align:right;">
123.2932
</td>
<td style="text-align:right;">
123.2933
</td>
<td style="text-align:left;">
positive
</td>
<td style="text-align:left;">
MGI_C57BL6J_5451931
</td>
<td style="text-align:left;">
Gm22154
</td>
<td style="text-align:left;">
ENSEMBL:ENSMUSG00000065778
</td>
<td style="text-align:left;">
MGI:5451931
</td>
<td style="text-align:left;">
snoRNA gene
</td>
<td style="text-align:left;">
predicted gene, 22154
</td>
</tr>
<tr>
<td style="text-align:left;">
4
</td>
<td style="text-align:left;">
gene
</td>
<td style="text-align:right;">
123.3126
</td>
<td style="text-align:right;">
123.3433
</td>
<td style="text-align:left;">
negative
</td>
<td style="text-align:left;">
MGI_C57BL6J_104515
</td>
<td style="text-align:left;">
Bmp8a
</td>
<td style="text-align:left;">
NCBI_Gene:12163,ENSEMBL:ENSMUSG00000032726
</td>
<td style="text-align:left;">
MGI:104515
</td>
<td style="text-align:left;">
protein coding gene
</td>
<td style="text-align:left;">
bone morphogenetic protein 8a
</td>
</tr>
<tr>
<td style="text-align:left;">
4
</td>
<td style="text-align:left;">
gene
</td>
<td style="text-align:right;">
123.3219
</td>
<td style="text-align:right;">
123.3237
</td>
<td style="text-align:left;">
negative
</td>
<td style="text-align:left;">
MGI_C57BL6J_1891061
</td>
<td style="text-align:left;">
Oxct2a
</td>
<td style="text-align:left;">
NCBI_Gene:64059,ENSEMBL:ENSMUSG00000076436
</td>
<td style="text-align:left;">
MGI:1891061
</td>
<td style="text-align:left;">
protein coding gene
</td>
<td style="text-align:left;">
3-oxoacid CoA transferase 2A
</td>
</tr>
<tr>
<td style="text-align:left;">
4
</td>
<td style="text-align:left;">
gene
</td>
<td style="text-align:right;">
123.3496
</td>
<td style="text-align:right;">
123.6844
</td>
<td style="text-align:left;">
negative
</td>
<td style="text-align:left;">
MGI_C57BL6J_108559
</td>
<td style="text-align:left;">
Macf1
</td>
<td style="text-align:left;">
NCBI_Gene:11426,ENSEMBL:ENSMUSG00000028649
</td>
<td style="text-align:left;">
MGI:108559
</td>
<td style="text-align:left;">
protein coding gene
</td>
<td style="text-align:left;">
microtubule-actin crosslinking factor 1
</td>
</tr>
<tr>
<td style="text-align:left;">
4
</td>
<td style="text-align:left;">
gene
</td>
<td style="text-align:right;">
123.4036
</td>
<td style="text-align:right;">
123.4119
</td>
<td style="text-align:left;">
negative
</td>
<td style="text-align:left;">
MGI_C57BL6J_3045367
</td>
<td style="text-align:left;">
D830031N03Rik
</td>
<td style="text-align:left;">
NCBI_Gene:442834
</td>
<td style="text-align:left;">
MGI:3045367
</td>
<td style="text-align:left;">
protein coding gene
</td>
<td style="text-align:left;">
RIKEN cDNA D830031N03 gene
</td>
</tr>
<tr>
<td style="text-align:left;">
4
</td>
<td style="text-align:left;">
pseudogene
</td>
<td style="text-align:right;">
123.4352
</td>
<td style="text-align:right;">
123.4364
</td>
<td style="text-align:left;">
positive
</td>
<td style="text-align:left;">
MGI_C57BL6J_3651538
</td>
<td style="text-align:left;">
Gm12924
</td>
<td style="text-align:left;">
NCBI_Gene:102633376,ENSEMBL:ENSMUSG00000081528
</td>
<td style="text-align:left;">
MGI:3651538
</td>
<td style="text-align:left;">
pseudogene
</td>
<td style="text-align:left;">
predicted gene 12924
</td>
</tr>
<tr>
<td style="text-align:left;">
4
</td>
<td style="text-align:left;">
gene
</td>
<td style="text-align:right;">
123.4902
</td>
<td style="text-align:right;">
123.4903
</td>
<td style="text-align:left;">
positive
</td>
<td style="text-align:left;">
MGI_C57BL6J_5453325
</td>
<td style="text-align:left;">
Gm23548
</td>
<td style="text-align:left;">
ENSEMBL:ENSMUSG00000089588
</td>
<td style="text-align:left;">
MGI:5453325
</td>
<td style="text-align:left;">
unclassified non-coding RNA gene
</td>
<td style="text-align:left;">
predicted gene, 23548
</td>
</tr>
<tr>
<td style="text-align:left;">
4
</td>
<td style="text-align:left;">
gene
</td>
<td style="text-align:right;">
123.5504
</td>
<td style="text-align:right;">
123.5570
</td>
<td style="text-align:left;">
positive
</td>
<td style="text-align:left;">
MGI_C57BL6J_5590315
</td>
<td style="text-align:left;">
Gm31156
</td>
<td style="text-align:left;">
NCBI_Gene:102633296
</td>
<td style="text-align:left;">
MGI:5590315
</td>
<td style="text-align:left;">
lncRNA gene
</td>
<td style="text-align:left;">
predicted gene, 31156
</td>
</tr>
<tr>
<td style="text-align:left;">
4
</td>
<td style="text-align:left;">
gene
</td>
<td style="text-align:right;">
123.6164
</td>
<td style="text-align:right;">
123.6165
</td>
<td style="text-align:left;">
positive
</td>
<td style="text-align:left;">
MGI_C57BL6J_5531039
</td>
<td style="text-align:left;">
Mir6398
</td>
<td style="text-align:left;">
miRBase:MI0021934,NCBI_Gene:102465214,ENSEMBL:ENSMUSG00000099005
</td>
<td style="text-align:left;">
MGI:5531039
</td>
<td style="text-align:left;">
miRNA gene
</td>
<td style="text-align:left;">
microRNA 6398
</td>
</tr>
<tr>
<td style="text-align:left;">
4
</td>
<td style="text-align:left;">
pseudogene
</td>
<td style="text-align:right;">
123.6303
</td>
<td style="text-align:right;">
123.6307
</td>
<td style="text-align:left;">
positive
</td>
<td style="text-align:left;">
MGI_C57BL6J_3651535
</td>
<td style="text-align:left;">
Gm12926
</td>
<td style="text-align:left;">
ENSEMBL:ENSMUSG00000081585
</td>
<td style="text-align:left;">
MGI:3651535
</td>
<td style="text-align:left;">
pseudogene
</td>
<td style="text-align:left;">
predicted gene 12926
</td>
</tr>
<tr>
<td style="text-align:left;">
4
</td>
<td style="text-align:left;">
pseudogene
</td>
<td style="text-align:right;">
123.6446
</td>
<td style="text-align:right;">
123.6479
</td>
<td style="text-align:left;">
negative
</td>
<td style="text-align:left;">
MGI_C57BL6J_5594400
</td>
<td style="text-align:left;">
Gm35241
</td>
<td style="text-align:left;">
NCBI_Gene:102638752
</td>
<td style="text-align:left;">
MGI:5594400
</td>
<td style="text-align:left;">
pseudogene
</td>
<td style="text-align:left;">
predicted gene, 35241
</td>
</tr>
<tr>
<td style="text-align:left;">
4
</td>
<td style="text-align:left;">
gene
</td>
<td style="text-align:right;">
123.6659
</td>
<td style="text-align:right;">
123.6712
</td>
<td style="text-align:left;">
positive
</td>
<td style="text-align:left;">
MGI_C57BL6J_3651536
</td>
<td style="text-align:left;">
Gm12925
</td>
<td style="text-align:left;">
NCBI_Gene:102633214,ENSEMBL:ENSMUSG00000087307
</td>
<td style="text-align:left;">
MGI:3651536
</td>
<td style="text-align:left;">
lncRNA gene
</td>
<td style="text-align:left;">
predicted gene 12925
</td>
</tr>
<tr>
<td style="text-align:left;">
4
</td>
<td style="text-align:left;">
pseudogene
</td>
<td style="text-align:right;">
123.7092
</td>
<td style="text-align:right;">
123.7101
</td>
<td style="text-align:left;">
negative
</td>
<td style="text-align:left;">
MGI_C57BL6J_3651537
</td>
<td style="text-align:left;">
Gm12922
</td>
<td style="text-align:left;">
NCBI_Gene:100039110,ENSEMBL:ENSMUSG00000081170
</td>
<td style="text-align:left;">
MGI:3651537
</td>
<td style="text-align:left;">
pseudogene
</td>
<td style="text-align:left;">
predicted gene 12922
</td>
</tr>
<tr>
<td style="text-align:left;">
4
</td>
<td style="text-align:left;">
gene
</td>
<td style="text-align:right;">
123.7127
</td>
<td style="text-align:right;">
123.7182
</td>
<td style="text-align:left;">
negative
</td>
<td style="text-align:left;">
MGI_C57BL6J_1890889
</td>
<td style="text-align:left;">
Ndufs5
</td>
<td style="text-align:left;">
NCBI_Gene:595136,ENSEMBL:ENSMUSG00000028648
</td>
<td style="text-align:left;">
MGI:1890889
</td>
<td style="text-align:left;">
protein coding gene
</td>
<td style="text-align:left;">
NADH:ubiquinone oxidoreductase core subunit S5
</td>
</tr>
<tr>
<td style="text-align:left;">
4
</td>
<td style="text-align:left;">
gene
</td>
<td style="text-align:right;">
123.7346
</td>
<td style="text-align:right;">
123.7503
</td>
<td style="text-align:left;">
negative
</td>
<td style="text-align:left;">
MGI_C57BL6J_1915300
</td>
<td style="text-align:left;">
Akirin1
</td>
<td style="text-align:left;">
NCBI_Gene:68050,ENSEMBL:ENSMUSG00000023075
</td>
<td style="text-align:left;">
MGI:1915300
</td>
<td style="text-align:left;">
protein coding gene
</td>
<td style="text-align:left;">
akirin 1
</td>
</tr>
<tr>
<td style="text-align:left;">
4
</td>
<td style="text-align:left;">
gene
</td>
<td style="text-align:right;">
123.7879
</td>
<td style="text-align:right;">
123.8302
</td>
<td style="text-align:left;">
positive
</td>
<td style="text-align:left;">
MGI_C57BL6J_3608413
</td>
<td style="text-align:left;">
Rhbdl2
</td>
<td style="text-align:left;">
NCBI_Gene:230726,ENSEMBL:ENSMUSG00000043333
</td>
<td style="text-align:left;">
MGI:3608413
</td>
<td style="text-align:left;">
protein coding gene
</td>
<td style="text-align:left;">
rhomboid like 2
</td>
</tr>
<tr>
<td style="text-align:left;">
4
</td>
<td style="text-align:left;">
gene
</td>
<td style="text-align:right;">
123.8336
</td>
<td style="text-align:right;">
123.8595
</td>
<td style="text-align:left;">
negative
</td>
<td style="text-align:left;">
MGI_C57BL6J_1925891
</td>
<td style="text-align:left;">
1700121C08Rik
</td>
<td style="text-align:left;">
NCBI_Gene:105244677,ENSEMBL:ENSMUSG00000086275
</td>
<td style="text-align:left;">
MGI:1925891
</td>
<td style="text-align:left;">
lncRNA gene
</td>
<td style="text-align:left;">
RIKEN cDNA 1700121C08 gene
</td>
</tr>
<tr>
<td style="text-align:left;">
4
</td>
<td style="text-align:left;">
pseudogene
</td>
<td style="text-align:right;">
123.8381
</td>
<td style="text-align:right;">
123.8395
</td>
<td style="text-align:left;">
positive
</td>
<td style="text-align:left;">
MGI_C57BL6J_3650403
</td>
<td style="text-align:left;">
Gm12904
</td>
<td style="text-align:left;">
NCBI_Gene:667064,ENSEMBL:ENSMUSG00000082310
</td>
<td style="text-align:left;">
MGI:3650403
</td>
<td style="text-align:left;">
pseudogene
</td>
<td style="text-align:left;">
predicted gene 12904
</td>
</tr>
<tr>
<td style="text-align:left;">
4
</td>
<td style="text-align:left;">
pseudogene
</td>
<td style="text-align:right;">
123.8460
</td>
<td style="text-align:right;">
123.8495
</td>
<td style="text-align:left;">
negative
</td>
<td style="text-align:left;">
MGI_C57BL6J_5826497
</td>
<td style="text-align:left;">
Gm46860
</td>
<td style="text-align:left;">
NCBI_Gene:108168958
</td>
<td style="text-align:left;">
MGI:5826497
</td>
<td style="text-align:left;">
pseudogene
</td>
<td style="text-align:left;">
predicted gene, 46860
</td>
</tr>
<tr>
<td style="text-align:left;">
4
</td>
<td style="text-align:left;">
gene
</td>
<td style="text-align:right;">
123.8597
</td>
<td style="text-align:right;">
123.8632
</td>
<td style="text-align:left;">
positive
</td>
<td style="text-align:left;">
MGI_C57BL6J_3613760
</td>
<td style="text-align:left;">
4933427I04Rik
</td>
<td style="text-align:left;">
ENSEMBL:ENSMUSG00000073761
</td>
<td style="text-align:left;">
MGI:3613760
</td>
<td style="text-align:left;">
protein coding gene
</td>
<td style="text-align:left;">
Riken cDNA 4933427I04 gene
</td>
</tr>
<tr>
<td style="text-align:left;">
4
</td>
<td style="text-align:left;">
gene
</td>
<td style="text-align:right;">
123.8772
</td>
<td style="text-align:right;">
123.8806
</td>
<td style="text-align:left;">
positive
</td>
<td style="text-align:left;">
MGI_C57BL6J_5590468
</td>
<td style="text-align:left;">
Gm31309
</td>
<td style="text-align:left;">
NCBI_Gene:102633498
</td>
<td style="text-align:left;">
MGI:5590468
</td>
<td style="text-align:left;">
protein coding gene
</td>
<td style="text-align:left;">
predicted gene, 31309
</td>
</tr>
<tr>
<td style="text-align:left;">
4
</td>
<td style="text-align:left;">
gene
</td>
<td style="text-align:right;">
123.8953
</td>
<td style="text-align:right;">
123.8966
</td>
<td style="text-align:left;">
negative
</td>
<td style="text-align:left;">
MGI_C57BL6J_3642540
</td>
<td style="text-align:left;">
Gm10572
</td>
<td style="text-align:left;">
NA
</td>
<td style="text-align:left;">
NA
</td>
<td style="text-align:left;">
unclassified gene
</td>
<td style="text-align:left;">
predicted gene 10572
</td>
</tr>
<tr>
<td style="text-align:left;">
4
</td>
<td style="text-align:left;">
gene
</td>
<td style="text-align:right;">
123.8979
</td>
<td style="text-align:right;">
123.9012
</td>
<td style="text-align:left;">
negative
</td>
<td style="text-align:left;">
MGI_C57BL6J_5623125
</td>
<td style="text-align:left;">
Gm40240
</td>
<td style="text-align:left;">
NCBI_Gene:105244678
</td>
<td style="text-align:left;">
MGI:5623125
</td>
<td style="text-align:left;">
lncRNA gene
</td>
<td style="text-align:left;">
predicted gene, 40240
</td>
</tr>
<tr>
<td style="text-align:left;">
4
</td>
<td style="text-align:left;">
gene
</td>
<td style="text-align:right;">
123.8986
</td>
<td style="text-align:right;">
123.9048
</td>
<td style="text-align:left;">
negative
</td>
<td style="text-align:left;">
MGI_C57BL6J_5477100
</td>
<td style="text-align:left;">
Gm26606
</td>
<td style="text-align:left;">
ENSEMBL:ENSMUSG00000097337
</td>
<td style="text-align:left;">
MGI:5477100
</td>
<td style="text-align:left;">
lncRNA gene
</td>
<td style="text-align:left;">
predicted gene, 26606
</td>
</tr>
<tr>
<td style="text-align:left;">
4
</td>
<td style="text-align:left;">
pseudogene
</td>
<td style="text-align:right;">
123.9021
</td>
<td style="text-align:right;">
123.9050
</td>
<td style="text-align:left;">
negative
</td>
<td style="text-align:left;">
MGI_C57BL6J_3045257
</td>
<td style="text-align:left;">
D130007C19Rik
</td>
<td style="text-align:left;">
NCBI_Gene:442805,ENSEMBL:ENSMUSG00000054304
</td>
<td style="text-align:left;">
MGI:3045257
</td>
<td style="text-align:left;">
pseudogene
</td>
<td style="text-align:left;">
RIKEN cDNA D130007C19 gene
</td>
</tr>
<tr>
<td style="text-align:left;">
4
</td>
<td style="text-align:left;">
pseudogene
</td>
<td style="text-align:right;">
123.9024
</td>
<td style="text-align:right;">
123.9030
</td>
<td style="text-align:left;">
positive
</td>
<td style="text-align:left;">
MGI_C57BL6J_3650402
</td>
<td style="text-align:left;">
Gm12903
</td>
<td style="text-align:left;">
ENSEMBL:ENSMUSG00000080993
</td>
<td style="text-align:left;">
MGI:3650402
</td>
<td style="text-align:left;">
pseudogene
</td>
<td style="text-align:left;">
predicted gene 12903
</td>
</tr>
<tr>
<td style="text-align:left;">
4
</td>
<td style="text-align:left;">
gene
</td>
<td style="text-align:right;">
123.9048
</td>
<td style="text-align:right;">
123.9123
</td>
<td style="text-align:left;">
positive
</td>
<td style="text-align:left;">
MGI_C57BL6J_1891750
</td>
<td style="text-align:left;">
Mycbp
</td>
<td style="text-align:left;">
NCBI_Gene:56309,ENSEMBL:ENSMUSG00000028647
</td>
<td style="text-align:left;">
MGI:1891750
</td>
<td style="text-align:left;">
protein coding gene
</td>
<td style="text-align:left;">
MYC binding protein
</td>
</tr>
<tr>
<td style="text-align:left;">
4
</td>
<td style="text-align:left;">
gene
</td>
<td style="text-align:right;">
123.9051
</td>
<td style="text-align:right;">
123.9180
</td>
<td style="text-align:left;">
negative
</td>
<td style="text-align:left;">
MGI_C57BL6J_3702581
</td>
<td style="text-align:left;">
Gm12905
</td>
<td style="text-align:left;">
ENSEMBL:ENSMUSG00000085875
</td>
<td style="text-align:left;">
MGI:3702581
</td>
<td style="text-align:left;">
lncRNA gene
</td>
<td style="text-align:left;">
predicted gene 12905
</td>
</tr>
<tr>
<td style="text-align:left;">
4
</td>
<td style="text-align:left;">
gene
</td>
<td style="text-align:right;">
123.9153
</td>
<td style="text-align:right;">
123.9180
</td>
<td style="text-align:left;">
negative
</td>
<td style="text-align:left;">
MGI_C57BL6J_1922470
</td>
<td style="text-align:left;">
4930535I16Rik
</td>
<td style="text-align:left;">
NA
</td>
<td style="text-align:left;">
NA
</td>
<td style="text-align:left;">
protein coding gene
</td>
<td style="text-align:left;">
RIKEN cDNA 4930535I16 gene
</td>
</tr>
<tr>
<td style="text-align:left;">
4
</td>
<td style="text-align:left;">
gene
</td>
<td style="text-align:right;">
123.9174
</td>
<td style="text-align:right;">
123.9370
</td>
<td style="text-align:left;">
positive
</td>
<td style="text-align:left;">
MGI_C57BL6J_1858751
</td>
<td style="text-align:left;">
Rragc
</td>
<td style="text-align:left;">
NCBI_Gene:54170,ENSEMBL:ENSMUSG00000028646
</td>
<td style="text-align:left;">
MGI:1858751
</td>
<td style="text-align:left;">
protein coding gene
</td>
<td style="text-align:left;">
Ras-related GTP binding C
</td>
</tr>
<tr>
<td style="text-align:left;">
4
</td>
<td style="text-align:left;">
pseudogene
</td>
<td style="text-align:right;">
123.9262
</td>
<td style="text-align:right;">
123.9266
</td>
<td style="text-align:left;">
positive
</td>
<td style="text-align:left;">
MGI_C57BL6J_3650399
</td>
<td style="text-align:left;">
Gm12902
</td>
<td style="text-align:left;">
ENSEMBL:ENSMUSG00000082515
</td>
<td style="text-align:left;">
MGI:3650399
</td>
<td style="text-align:left;">
pseudogene
</td>
<td style="text-align:left;">
predicted gene 12902
</td>
</tr>
<tr>
<td style="text-align:left;">
4
</td>
<td style="text-align:left;">
gene
</td>
<td style="text-align:right;">
124.0540
</td>
<td style="text-align:right;">
124.0701
</td>
<td style="text-align:left;">
positive
</td>
<td style="text-align:left;">
MGI_C57BL6J_5623126
</td>
<td style="text-align:left;">
Gm40241
</td>
<td style="text-align:left;">
NCBI_Gene:105244679
</td>
<td style="text-align:left;">
MGI:5623126
</td>
<td style="text-align:left;">
lncRNA gene
</td>
<td style="text-align:left;">
predicted gene, 40241
</td>
</tr>
<tr>
<td style="text-align:left;">
4
</td>
<td style="text-align:left;">
gene
</td>
<td style="text-align:right;">
124.2714
</td>
<td style="text-align:right;">
124.2724
</td>
<td style="text-align:left;">
negative
</td>
<td style="text-align:left;">
MGI_C57BL6J_5623127
</td>
<td style="text-align:left;">
Gm40242
</td>
<td style="text-align:left;">
NCBI_Gene:105244680
</td>
<td style="text-align:left;">
MGI:5623127
</td>
<td style="text-align:left;">
lncRNA gene
</td>
<td style="text-align:left;">
predicted gene, 40242
</td>
</tr>
<tr>
<td style="text-align:left;">
4
</td>
<td style="text-align:left;">
gene
</td>
<td style="text-align:right;">
124.2778
</td>
<td style="text-align:right;">
124.2792
</td>
<td style="text-align:left;">
negative
</td>
<td style="text-align:left;">
MGI_C57BL6J_5610895
</td>
<td style="text-align:left;">
Gm37667
</td>
<td style="text-align:left;">
ENSEMBL:ENSMUSG00000103541
</td>
<td style="text-align:left;">
MGI:5610895
</td>
<td style="text-align:left;">
unclassified gene
</td>
<td style="text-align:left;">
predicted gene, 37667
</td>
</tr>
<tr>
<td style="text-align:left;">
4
</td>
<td style="text-align:left;">
gene
</td>
<td style="text-align:right;">
124.2931
</td>
<td style="text-align:right;">
124.2999
</td>
<td style="text-align:left;">
negative
</td>
<td style="text-align:left;">
MGI_C57BL6J_3649992
</td>
<td style="text-align:left;">
Gm12916
</td>
<td style="text-align:left;">
NCBI_Gene:102633918,ENSEMBL:ENSMUSG00000086087
</td>
<td style="text-align:left;">
MGI:3649992
</td>
<td style="text-align:left;">
lncRNA gene
</td>
<td style="text-align:left;">
predicted gene 12916
</td>
</tr>
<tr>
<td style="text-align:left;">
4
</td>
<td style="text-align:left;">
gene
</td>
<td style="text-align:right;">
124.3028
</td>
<td style="text-align:right;">
124.3327
</td>
<td style="text-align:left;">
negative
</td>
<td style="text-align:left;">
MGI_C57BL6J_1923660
</td>
<td style="text-align:left;">
1700021L23Rik
</td>
<td style="text-align:left;">
NCBI_Gene:76410,ENSEMBL:ENSMUSG00000086955
</td>
<td style="text-align:left;">
MGI:1923660
</td>
<td style="text-align:left;">
lncRNA gene
</td>
<td style="text-align:left;">
RIKEN cDNA 1700021L23 gene
</td>
</tr>
<tr>
<td style="text-align:left;">
4
</td>
<td style="text-align:left;">
gene
</td>
<td style="text-align:right;">
124.3263
</td>
<td style="text-align:right;">
124.3325
</td>
<td style="text-align:left;">
negative
</td>
<td style="text-align:left;">
MGI_C57BL6J_1918570
</td>
<td style="text-align:left;">
4933435F18Rik
</td>
<td style="text-align:left;">
ENSEMBL:ENSMUSG00000086978
</td>
<td style="text-align:left;">
MGI:1918570
</td>
<td style="text-align:left;">
lncRNA gene
</td>
<td style="text-align:left;">
RIKEN cDNA 4933435F18 gene
</td>
</tr>
<tr>
<td style="text-align:left;">
4
</td>
<td style="text-align:left;">
gene
</td>
<td style="text-align:right;">
124.3356
</td>
<td style="text-align:right;">
124.3357
</td>
<td style="text-align:left;">
negative
</td>
<td style="text-align:left;">
MGI_C57BL6J_5452760
</td>
<td style="text-align:left;">
Gm22983
</td>
<td style="text-align:left;">
ENSEMBL:ENSMUSG00000088933
</td>
<td style="text-align:left;">
MGI:5452760
</td>
<td style="text-align:left;">
snRNA gene
</td>
<td style="text-align:left;">
predicted gene, 22983
</td>
</tr>
<tr>
<td style="text-align:left;">
4
</td>
<td style="text-align:left;">
gene
</td>
<td style="text-align:right;">
124.4144
</td>
<td style="text-align:right;">
124.4860
</td>
<td style="text-align:left;">
negative
</td>
<td style="text-align:left;">
MGI_C57BL6J_1925710
</td>
<td style="text-align:left;">
1700057H15Rik
</td>
<td style="text-align:left;">
NCBI_Gene:78460,ENSEMBL:ENSMUSG00000084757
</td>
<td style="text-align:left;">
MGI:1925710
</td>
<td style="text-align:left;">
lncRNA gene
</td>
<td style="text-align:left;">
RIKEN cDNA 1700057H15 gene
</td>
</tr>
<tr>
<td style="text-align:left;">
4
</td>
<td style="text-align:left;">
gene
</td>
<td style="text-align:right;">
124.4514
</td>
<td style="text-align:right;">
124.4564
</td>
<td style="text-align:left;">
positive
</td>
<td style="text-align:left;">
MGI_C57BL6J_5590989
</td>
<td style="text-align:left;">
Gm31830
</td>
<td style="text-align:left;">
NCBI_Gene:102634185
</td>
<td style="text-align:left;">
MGI:5590989
</td>
<td style="text-align:left;">
lncRNA gene
</td>
<td style="text-align:left;">
predicted gene, 31830
</td>
</tr>
<tr>
<td style="text-align:left;">
4
</td>
<td style="text-align:left;">
gene
</td>
<td style="text-align:right;">
124.4653
</td>
<td style="text-align:right;">
124.4681
</td>
<td style="text-align:left;">
negative
</td>
<td style="text-align:left;">
MGI_C57BL6J_5590928
</td>
<td style="text-align:left;">
Gm31769
</td>
<td style="text-align:left;">
NCBI_Gene:102634107
</td>
<td style="text-align:left;">
MGI:5590928
</td>
<td style="text-align:left;">
lncRNA gene
</td>
<td style="text-align:left;">
predicted gene, 31769
</td>
</tr>
<tr>
<td style="text-align:left;">
4
</td>
<td style="text-align:left;">
gene
</td>
<td style="text-align:right;">
124.5229
</td>
<td style="text-align:right;">
124.6575
</td>
<td style="text-align:left;">
negative
</td>
<td style="text-align:left;">
MGI_C57BL6J_3780334
</td>
<td style="text-align:left;">
Gm2164
</td>
<td style="text-align:left;">
NCBI_Gene:100039332,ENSEMBL:ENSMUSG00000098760
</td>
<td style="text-align:left;">
MGI:3780334
</td>
<td style="text-align:left;">
lncRNA gene
</td>
<td style="text-align:left;">
predicted gene 2164
</td>
</tr>
<tr>
<td style="text-align:left;">
4
</td>
<td style="text-align:left;">
gene
</td>
<td style="text-align:right;">
124.5692
</td>
<td style="text-align:right;">
124.5755
</td>
<td style="text-align:left;">
positive
</td>
<td style="text-align:left;">
MGI_C57BL6J_1918352
</td>
<td style="text-align:left;">
4933407E24Rik
</td>
<td style="text-align:left;">
NCBI_Gene:108809,ENSEMBL:ENSMUSG00000073759
</td>
<td style="text-align:left;">
MGI:1918352
</td>
<td style="text-align:left;">
lncRNA gene
</td>
<td style="text-align:left;">
RIKEN cDNA 4933407E24 gene
</td>
</tr>
<tr>
<td style="text-align:left;">
4
</td>
<td style="text-align:left;">
gene
</td>
<td style="text-align:right;">
124.6099
</td>
<td style="text-align:right;">
124.6108
</td>
<td style="text-align:left;">
negative
</td>
<td style="text-align:left;">
MGI_C57BL6J_1917499
</td>
<td style="text-align:left;">
3100002H09Rik
</td>
<td style="text-align:left;">
NA
</td>
<td style="text-align:left;">
NA
</td>
<td style="text-align:left;">
lncRNA gene
</td>
<td style="text-align:left;">
RIKEN cDNA 3100002H09 gene
</td>
</tr>
<tr>
<td style="text-align:left;">
4
</td>
<td style="text-align:left;">
gene
</td>
<td style="text-align:right;">
124.6399
</td>
<td style="text-align:right;">
124.6408
</td>
<td style="text-align:left;">
positive
</td>
<td style="text-align:left;">
MGI_C57BL6J_1915732
</td>
<td style="text-align:left;">
1110007E10Rik
</td>
<td style="text-align:left;">
NA
</td>
<td style="text-align:left;">
NA
</td>
<td style="text-align:left;">
unclassified gene
</td>
<td style="text-align:left;">
RIKEN cDNA 1110007E10 gene
</td>
</tr>
<tr>
<td style="text-align:left;">
4
</td>
<td style="text-align:left;">
gene
</td>
<td style="text-align:right;">
124.6568
</td>
<td style="text-align:right;">
124.6607
</td>
<td style="text-align:left;">
positive
</td>
<td style="text-align:left;">
MGI_C57BL6J_101896
</td>
<td style="text-align:left;">
Pou3f1
</td>
<td style="text-align:left;">
NCBI_Gene:18991,ENSEMBL:ENSMUSG00000090125
</td>
<td style="text-align:left;">
MGI:101896
</td>
<td style="text-align:left;">
protein coding gene
</td>
<td style="text-align:left;">
POU domain, class 3, transcription factor 1
</td>
</tr>
<tr>
<td style="text-align:left;">
4
</td>
<td style="text-align:left;">
gene
</td>
<td style="text-align:right;">
124.6643
</td>
<td style="text-align:right;">
124.6687
</td>
<td style="text-align:left;">
negative
</td>
<td style="text-align:left;">
MGI_C57BL6J_5621430
</td>
<td style="text-align:left;">
Gm38545
</td>
<td style="text-align:left;">
NCBI_Gene:102641270
</td>
<td style="text-align:left;">
MGI:5621430
</td>
<td style="text-align:left;">
lncRNA gene
</td>
<td style="text-align:left;">
predicted gene, 38545
</td>
</tr>
<tr>
<td style="text-align:left;">
4
</td>
<td style="text-align:left;">
gene
</td>
<td style="text-align:right;">
124.6782
</td>
<td style="text-align:right;">
124.6936
</td>
<td style="text-align:left;">
negative
</td>
<td style="text-align:left;">
MGI_C57BL6J_1914455
</td>
<td style="text-align:left;">
Utp11
</td>
<td style="text-align:left;">
NCBI_Gene:67205,ENSEMBL:ENSMUSG00000028907
</td>
<td style="text-align:left;">
MGI:1914455
</td>
<td style="text-align:left;">
protein coding gene
</td>
<td style="text-align:left;">
UTP11 small subunit processome component
</td>
</tr>
<tr>
<td style="text-align:left;">
4
</td>
<td style="text-align:left;">
gene
</td>
<td style="text-align:right;">
124.6960
</td>
<td style="text-align:right;">
124.7086
</td>
<td style="text-align:left;">
positive
</td>
<td style="text-align:left;">
MGI_C57BL6J_1341092
</td>
<td style="text-align:left;">
Fhl3
</td>
<td style="text-align:left;">
NCBI_Gene:14201,ENSEMBL:ENSMUSG00000032643
</td>
<td style="text-align:left;">
MGI:1341092
</td>
<td style="text-align:left;">
protein coding gene
</td>
<td style="text-align:left;">
four and a half LIM domains 3
</td>
</tr>
<tr>
<td style="text-align:left;">
4
</td>
<td style="text-align:left;">
gene
</td>
<td style="text-align:right;">
124.6975
</td>
<td style="text-align:right;">
124.6976
</td>
<td style="text-align:left;">
negative
</td>
<td style="text-align:left;">
MGI_C57BL6J_5454257
</td>
<td style="text-align:left;">
Gm24480
</td>
<td style="text-align:left;">
ENSEMBL:ENSMUSG00000088067
</td>
<td style="text-align:left;">
MGI:5454257
</td>
<td style="text-align:left;">
snRNA gene
</td>
<td style="text-align:left;">
predicted gene, 24480
</td>
</tr>
<tr>
<td style="text-align:left;">
4
</td>
<td style="text-align:left;">
gene
</td>
<td style="text-align:right;">
124.7148
</td>
<td style="text-align:right;">
124.7325
</td>
<td style="text-align:left;">
positive
</td>
<td style="text-align:left;">
MGI_C57BL6J_1922312
</td>
<td style="text-align:left;">
Sf3a3
</td>
<td style="text-align:left;">
NCBI_Gene:75062,ENSEMBL:ENSMUSG00000028902
</td>
<td style="text-align:left;">
MGI:1922312
</td>
<td style="text-align:left;">
protein coding gene
</td>
<td style="text-align:left;">
splicing factor 3a, subunit 3
</td>
</tr>
<tr>
<td style="text-align:left;">
4
</td>
<td style="text-align:left;">
pseudogene
</td>
<td style="text-align:right;">
124.7190
</td>
<td style="text-align:right;">
124.7196
</td>
<td style="text-align:left;">
positive
</td>
<td style="text-align:left;">
MGI_C57BL6J_3652006
</td>
<td style="text-align:left;">
Gm12917
</td>
<td style="text-align:left;">
NCBI_Gene:664991,ENSEMBL:ENSMUSG00000083104
</td>
<td style="text-align:left;">
MGI:3652006
</td>
<td style="text-align:left;">
pseudogene
</td>
<td style="text-align:left;">
predicted gene 12917
</td>
</tr>
<tr>
<td style="text-align:left;">
4
</td>
<td style="text-align:left;">
gene
</td>
<td style="text-align:right;">
124.7317
</td>
<td style="text-align:right;">
124.7318
</td>
<td style="text-align:left;">
positive
</td>
<td style="text-align:left;">
MGI_C57BL6J_3629652
</td>
<td style="text-align:left;">
Mir697
</td>
<td style="text-align:left;">
miRBase:MI0004681,NCBI_Gene:735277,ENSEMBL:ENSMUSG00000076271
</td>
<td style="text-align:left;">
MGI:3629652
</td>
<td style="text-align:left;">
miRNA gene
</td>
<td style="text-align:left;">
microRNA 697
</td>
</tr>
<tr>
<td style="text-align:left;">
4
</td>
<td style="text-align:left;">
gene
</td>
<td style="text-align:right;">
124.7353
</td>
<td style="text-align:right;">
124.7514
</td>
<td style="text-align:left;">
negative
</td>
<td style="text-align:left;">
MGI_C57BL6J_3650273
</td>
<td style="text-align:left;">
Gm12915
</td>
<td style="text-align:left;">
NCBI_Gene:102634646,ENSEMBL:ENSMUSG00000086589
</td>
<td style="text-align:left;">
MGI:3650273
</td>
<td style="text-align:left;">
lncRNA gene
</td>
<td style="text-align:left;">
predicted gene 12915
</td>
</tr>
<tr>
<td style="text-align:left;">
4
</td>
<td style="text-align:left;">
gene
</td>
<td style="text-align:right;">
124.7358
</td>
<td style="text-align:right;">
124.7359
</td>
<td style="text-align:left;">
positive
</td>
<td style="text-align:left;">
MGI_C57BL6J_5454498
</td>
<td style="text-align:left;">
Gm24721
</td>
<td style="text-align:left;">
ENSEMBL:ENSMUSG00000096411
</td>
<td style="text-align:left;">
MGI:5454498
</td>
<td style="text-align:left;">
miRNA gene
</td>
<td style="text-align:left;">
predicted gene, 24721
</td>
</tr>
<tr>
<td style="text-align:left;">
4
</td>
<td style="text-align:left;">
gene
</td>
<td style="text-align:right;">
124.7374
</td>
<td style="text-align:right;">
124.7401
</td>
<td style="text-align:left;">
positive
</td>
<td style="text-align:left;">
MGI_C57BL6J_5591276
</td>
<td style="text-align:left;">
Gm32117
</td>
<td style="text-align:left;">
NCBI_Gene:102634569
</td>
<td style="text-align:left;">
MGI:5591276
</td>
<td style="text-align:left;">
lncRNA gene
</td>
<td style="text-align:left;">
predicted gene, 32117
</td>
</tr>
<tr>
<td style="text-align:left;">
4
</td>
<td style="text-align:left;">
gene
</td>
<td style="text-align:right;">
124.7418
</td>
<td style="text-align:right;">
124.8015
</td>
<td style="text-align:left;">
positive
</td>
<td style="text-align:left;">
MGI_C57BL6J_103257
</td>
<td style="text-align:left;">
Inpp5b
</td>
<td style="text-align:left;">
NCBI_Gene:16330,ENSEMBL:ENSMUSG00000028894
</td>
<td style="text-align:left;">
MGI:103257
</td>
<td style="text-align:left;">
protein coding gene
</td>
<td style="text-align:left;">
inositol polyphosphate-5-phosphatase B
</td>
</tr>
<tr>
<td style="text-align:left;">
4
</td>
<td style="text-align:left;">
gene
</td>
<td style="text-align:right;">
124.7438
</td>
<td style="text-align:right;">
124.7439
</td>
<td style="text-align:left;">
positive
</td>
<td style="text-align:left;">
MGI_C57BL6J_3629653
</td>
<td style="text-align:left;">
Mir698
</td>
<td style="text-align:left;">
miRBase:MI0004682,NCBI_Gene:735263,ENSEMBL:ENSMUSG00000076119
</td>
<td style="text-align:left;">
MGI:3629653
</td>
<td style="text-align:left;">
miRNA gene
</td>
<td style="text-align:left;">
microRNA 698
</td>
</tr>
<tr>
<td style="text-align:left;">
4
</td>
<td style="text-align:left;">
gene
</td>
<td style="text-align:right;">
124.8021
</td>
<td style="text-align:right;">
124.8498
</td>
<td style="text-align:left;">
positive
</td>
<td style="text-align:left;">
MGI_C57BL6J_101786
</td>
<td style="text-align:left;">
Mtf1
</td>
<td style="text-align:left;">
NCBI_Gene:17764,ENSEMBL:ENSMUSG00000028890
</td>
<td style="text-align:left;">
MGI:101786
</td>
<td style="text-align:left;">
protein coding gene
</td>
<td style="text-align:left;">
metal response element binding transcription factor 1
</td>
</tr>
<tr>
<td style="text-align:left;">
4
</td>
<td style="text-align:left;">
gene
</td>
<td style="text-align:right;">
124.8208
</td>
<td style="text-align:right;">
124.8209
</td>
<td style="text-align:left;">
positive
</td>
<td style="text-align:left;">
MGI_C57BL6J_4422057
</td>
<td style="text-align:left;">
n-R5s192
</td>
<td style="text-align:left;">
ENSEMBL:ENSMUSG00000084526
</td>
<td style="text-align:left;">
MGI:4422057
</td>
<td style="text-align:left;">
rRNA gene
</td>
<td style="text-align:left;">
nuclear encoded rRNA 5S 192
</td>
</tr>
<tr>
<td style="text-align:left;">
4
</td>
<td style="text-align:left;">
gene
</td>
<td style="text-align:right;">
124.8491
</td>
<td style="text-align:right;">
124.8512
</td>
<td style="text-align:left;">
negative
</td>
<td style="text-align:left;">
MGI_C57BL6J_1916170
</td>
<td style="text-align:left;">
1110065P20Rik
</td>
<td style="text-align:left;">
NCBI_Gene:68920,ENSEMBL:ENSMUSG00000078570
</td>
<td style="text-align:left;">
MGI:1916170
</td>
<td style="text-align:left;">
protein coding gene
</td>
<td style="text-align:left;">
RIKEN cDNA 1110065P20 gene
</td>
</tr>
<tr>
<td style="text-align:left;">
4
</td>
<td style="text-align:left;">
gene
</td>
<td style="text-align:right;">
124.8507
</td>
<td style="text-align:right;">
124.8552
</td>
<td style="text-align:left;">
positive
</td>
<td style="text-align:left;">
MGI_C57BL6J_2387201
</td>
<td style="text-align:left;">
Yrdc
</td>
<td style="text-align:left;">
NCBI_Gene:230734,ENSEMBL:ENSMUSG00000028889
</td>
<td style="text-align:left;">
MGI:2387201
</td>
<td style="text-align:left;">
protein coding gene
</td>
<td style="text-align:left;">
yrdC domain containing (E.coli)
</td>
</tr>
<tr>
<td style="text-align:left;">
4
</td>
<td style="text-align:left;">
gene
</td>
<td style="text-align:right;">
124.8552
</td>
<td style="text-align:right;">
124.8809
</td>
<td style="text-align:left;">
negative
</td>
<td style="text-align:left;">
MGI_C57BL6J_2684896
</td>
<td style="text-align:left;">
Maneal
</td>
<td style="text-align:left;">
NCBI_Gene:215090,ENSEMBL:ENSMUSG00000042763
</td>
<td style="text-align:left;">
MGI:2684896
</td>
<td style="text-align:left;">
protein coding gene
</td>
<td style="text-align:left;">
mannosidase, endo-alpha-like
</td>
</tr>
<tr>
<td style="text-align:left;">
4
</td>
<td style="text-align:left;">
gene
</td>
<td style="text-align:right;">
124.8809
</td>
<td style="text-align:right;">
124.9178
</td>
<td style="text-align:left;">
positive
</td>
<td style="text-align:left;">
MGI_C57BL6J_3586824
</td>
<td style="text-align:left;">
Epha10
</td>
<td style="text-align:left;">
NCBI_Gene:230735,ENSEMBL:ENSMUSG00000028876
</td>
<td style="text-align:left;">
MGI:3586824
</td>
<td style="text-align:left;">
protein coding gene
</td>
<td style="text-align:left;">
Eph receptor A10
</td>
</tr>
<tr>
<td style="text-align:left;">
4
</td>
<td style="text-align:left;">
gene
</td>
<td style="text-align:right;">
124.8909
</td>
<td style="text-align:right;">
124.8948
</td>
<td style="text-align:left;">
negative
</td>
<td style="text-align:left;">
MGI_C57BL6J_1920878
</td>
<td style="text-align:left;">
1700125G02Rik
</td>
<td style="text-align:left;">
NCBI_Gene:73628,ENSEMBL:ENSMUSG00000085263
</td>
<td style="text-align:left;">
MGI:1920878
</td>
<td style="text-align:left;">
lncRNA gene
</td>
<td style="text-align:left;">
RIKEN cDNA 1700125G02 gene
</td>
</tr>
<tr>
<td style="text-align:left;">
4
</td>
<td style="text-align:left;">
gene
</td>
<td style="text-align:right;">
124.9185
</td>
<td style="text-align:right;">
124.9393
</td>
<td style="text-align:left;">
negative
</td>
<td style="text-align:left;">
MGI_C57BL6J_1196274
</td>
<td style="text-align:left;">
Cdca8
</td>
<td style="text-align:left;">
NCBI_Gene:52276,ENSEMBL:ENSMUSG00000028873
</td>
<td style="text-align:left;">
MGI:1196274
</td>
<td style="text-align:left;">
protein coding gene
</td>
<td style="text-align:left;">
cell division cycle associated 8
</td>
</tr>
<tr>
<td style="text-align:left;">
4
</td>
<td style="text-align:left;">
gene
</td>
<td style="text-align:right;">
124.9370
</td>
<td style="text-align:right;">
124.9447
</td>
<td style="text-align:left;">
positive
</td>
<td style="text-align:left;">
MGI_C57BL6J_3041172
</td>
<td style="text-align:left;">
9930104L06Rik
</td>
<td style="text-align:left;">
NCBI_Gene:194268,ENSEMBL:ENSMUSG00000044730
</td>
<td style="text-align:left;">
MGI:3041172
</td>
<td style="text-align:left;">
protein coding gene
</td>
<td style="text-align:left;">
RIKEN cDNA 9930104L06 gene
</td>
</tr>
<tr>
<td style="text-align:left;">
4
</td>
<td style="text-align:left;">
gene
</td>
<td style="text-align:right;">
124.9457
</td>
<td style="text-align:right;">
124.9628
</td>
<td style="text-align:left;">
positive
</td>
<td style="text-align:left;">
MGI_C57BL6J_5623128
</td>
<td style="text-align:left;">
Gm40243
</td>
<td style="text-align:left;">
NCBI_Gene:105244681
</td>
<td style="text-align:left;">
MGI:5623128
</td>
<td style="text-align:left;">
lncRNA gene
</td>
<td style="text-align:left;">
predicted gene, 40243
</td>
</tr>
<tr>
<td style="text-align:left;">
4
</td>
<td style="text-align:left;">
gene
</td>
<td style="text-align:right;">
124.9576
</td>
<td style="text-align:right;">
124.9870
</td>
<td style="text-align:left;">
negative
</td>
<td style="text-align:left;">
MGI_C57BL6J_3650529
</td>
<td style="text-align:left;">
Gm12930
</td>
<td style="text-align:left;">
ENSEMBL:ENSMUSG00000087011
</td>
<td style="text-align:left;">
MGI:3650529
</td>
<td style="text-align:left;">
lncRNA gene
</td>
<td style="text-align:left;">
predicted gene 12930
</td>
</tr>
<tr>
<td style="text-align:left;">
4
</td>
<td style="text-align:left;">
gene
</td>
<td style="text-align:right;">
124.9676
</td>
<td style="text-align:right;">
124.9746
</td>
<td style="text-align:left;">
negative
</td>
<td style="text-align:left;">
MGI_C57BL6J_5591530
</td>
<td style="text-align:left;">
Gm32371
</td>
<td style="text-align:left;">
NCBI_Gene:102634896
</td>
<td style="text-align:left;">
MGI:5591530
</td>
<td style="text-align:left;">
lncRNA gene
</td>
<td style="text-align:left;">
predicted gene, 32371
</td>
</tr>
<tr>
<td style="text-align:left;">
4
</td>
<td style="text-align:left;">
gene
</td>
<td style="text-align:right;">
124.9864
</td>
<td style="text-align:right;">
125.0091
</td>
<td style="text-align:left;">
positive
</td>
<td style="text-align:left;">
MGI_C57BL6J_2183426
</td>
<td style="text-align:left;">
Rspo1
</td>
<td style="text-align:left;">
NCBI_Gene:192199,ENSEMBL:ENSMUSG00000028871
</td>
<td style="text-align:left;">
MGI:2183426
</td>
<td style="text-align:left;">
protein coding gene
</td>
<td style="text-align:left;">
R-spondin 1
</td>
</tr>
<tr>
<td style="text-align:left;">
4
</td>
<td style="text-align:left;">
gene
</td>
<td style="text-align:right;">
125.0166
</td>
<td style="text-align:right;">
125.0554
</td>
<td style="text-align:left;">
positive
</td>
<td style="text-align:left;">
MGI_C57BL6J_2385207
</td>
<td style="text-align:left;">
Gnl2
</td>
<td style="text-align:left;">
NCBI_Gene:230737,ENSEMBL:ENSMUSG00000028869
</td>
<td style="text-align:left;">
MGI:2385207
</td>
<td style="text-align:left;">
protein coding gene
</td>
<td style="text-align:left;">
guanine nucleotide binding protein-like 2 (nucleolar)
</td>
</tr>
<tr>
<td style="text-align:left;">
4
</td>
<td style="text-align:left;">
gene
</td>
<td style="text-align:right;">
125.0177
</td>
<td style="text-align:right;">
125.0210
</td>
<td style="text-align:left;">
positive
</td>
<td style="text-align:left;">
MGI_C57BL6J_5621357
</td>
<td style="text-align:left;">
Gm38472
</td>
<td style="text-align:left;">
NCBI_Gene:102635036
</td>
<td style="text-align:left;">
MGI:5621357
</td>
<td style="text-align:left;">
lncRNA gene
</td>
<td style="text-align:left;">
predicted gene, 38472
</td>
</tr>
<tr>
<td style="text-align:left;">
4
</td>
<td style="text-align:left;">
pseudogene
</td>
<td style="text-align:right;">
125.0195
</td>
<td style="text-align:right;">
125.0198
</td>
<td style="text-align:left;">
positive
</td>
<td style="text-align:left;">
MGI_C57BL6J_3650713
</td>
<td style="text-align:left;">
Gm12931
</td>
<td style="text-align:left;">
ENSEMBL:ENSMUSG00000080934
</td>
<td style="text-align:left;">
MGI:3650713
</td>
<td style="text-align:left;">
pseudogene
</td>
<td style="text-align:left;">
predicted gene 12931
</td>
</tr>
<tr>
<td style="text-align:left;">
4
</td>
<td style="text-align:left;">
gene
</td>
<td style="text-align:right;">
125.0553
</td>
<td style="text-align:right;">
125.0657
</td>
<td style="text-align:left;">
negative
</td>
<td style="text-align:left;">
MGI_C57BL6J_1922813
</td>
<td style="text-align:left;">
Dnali1
</td>
<td style="text-align:left;">
NCBI_Gene:75563,ENSEMBL:ENSMUSG00000042707
</td>
<td style="text-align:left;">
MGI:1922813
</td>
<td style="text-align:left;">
protein coding gene
</td>
<td style="text-align:left;">
dynein, axonemal, light intermediate polypeptide 1
</td>
</tr>
<tr>
<td style="text-align:left;">
4
</td>
<td style="text-align:left;">
gene
</td>
<td style="text-align:right;">
125.0667
</td>
<td style="text-align:right;">
125.0741
</td>
<td style="text-align:left;">
positive
</td>
<td style="text-align:left;">
MGI_C57BL6J_2156003
</td>
<td style="text-align:left;">
Snip1
</td>
<td style="text-align:left;">
NCBI_Gene:76793,ENSEMBL:ENSMUSG00000050213
</td>
<td style="text-align:left;">
MGI:2156003
</td>
<td style="text-align:left;">
protein coding gene
</td>
<td style="text-align:left;">
Smad nuclear interacting protein 1
</td>
</tr>
<tr>
<td style="text-align:left;">
4
</td>
<td style="text-align:left;">
pseudogene
</td>
<td style="text-align:right;">
125.0755
</td>
<td style="text-align:right;">
125.0765
</td>
<td style="text-align:left;">
positive
</td>
<td style="text-align:left;">
MGI_C57BL6J_5010665
</td>
<td style="text-align:left;">
Gm18480
</td>
<td style="text-align:left;">
NCBI_Gene:100417248
</td>
<td style="text-align:left;">
MGI:5010665
</td>
<td style="text-align:left;">
pseudogene
</td>
<td style="text-align:left;">
predicted gene, 18480
</td>
</tr>
<tr>
<td style="text-align:left;">
4
</td>
<td style="text-align:left;">
gene
</td>
<td style="text-align:right;">
125.0829
</td>
<td style="text-align:right;">
125.0849
</td>
<td style="text-align:left;">
negative
</td>
<td style="text-align:left;">
MGI_C57BL6J_3651151
</td>
<td style="text-align:left;">
Gm12932
</td>
<td style="text-align:left;">
ENSEMBL:ENSMUSG00000086342
</td>
<td style="text-align:left;">
MGI:3651151
</td>
<td style="text-align:left;">
lncRNA gene
</td>
<td style="text-align:left;">
predicted gene 12932
</td>
</tr>
<tr>
<td style="text-align:left;">
4
</td>
<td style="text-align:left;">
gene
</td>
<td style="text-align:right;">
125.0851
</td>
<td style="text-align:right;">
125.1132
</td>
<td style="text-align:left;">
positive
</td>
<td style="text-align:left;">
MGI_C57BL6J_1917338
</td>
<td style="text-align:left;">
Meaf6
</td>
<td style="text-align:left;">
NCBI_Gene:70088,ENSEMBL:ENSMUSG00000028863
</td>
<td style="text-align:left;">
MGI:1917338
</td>
<td style="text-align:left;">
protein coding gene
</td>
<td style="text-align:left;">
MYST/Esa1-associated factor 6
</td>
</tr>
<tr>
<td style="text-align:left;">
4
</td>
<td style="text-align:left;">
gene
</td>
<td style="text-align:right;">
125.1184
</td>
<td style="text-align:right;">
125.1279
</td>
<td style="text-align:left;">
negative
</td>
<td style="text-align:left;">
MGI_C57BL6J_2385891
</td>
<td style="text-align:left;">
Zc3h12a
</td>
<td style="text-align:left;">
NCBI_Gene:230738,ENSEMBL:ENSMUSG00000042677
</td>
<td style="text-align:left;">
MGI:2385891
</td>
<td style="text-align:left;">
protein coding gene
</td>
<td style="text-align:left;">
zinc finger CCCH type containing 12A
</td>
</tr>
<tr>
<td style="text-align:left;">
4
</td>
<td style="text-align:left;">
gene
</td>
<td style="text-align:right;">
125.1375
</td>
<td style="text-align:right;">
125.1477
</td>
<td style="text-align:left;">
negative
</td>
<td style="text-align:left;">
MGI_C57BL6J_5623131
</td>
<td style="text-align:left;">
Gm40246
</td>
<td style="text-align:left;">
NCBI_Gene:105244684
</td>
<td style="text-align:left;">
MGI:5623131
</td>
<td style="text-align:left;">
lncRNA gene
</td>
<td style="text-align:left;">
predicted gene, 40246
</td>
</tr>
<tr>
<td style="text-align:left;">
4
</td>
<td style="text-align:left;">
gene
</td>
<td style="text-align:right;">
125.1452
</td>
<td style="text-align:right;">
125.2904
</td>
<td style="text-align:left;">
positive
</td>
<td style="text-align:left;">
MGI_C57BL6J_5591770
</td>
<td style="text-align:left;">
Gm32611
</td>
<td style="text-align:left;">
NCBI_Gene:102635216
</td>
<td style="text-align:left;">
MGI:5591770
</td>
<td style="text-align:left;">
protein coding gene
</td>
<td style="text-align:left;">
predicted gene, 32611
</td>
</tr>
<tr>
<td style="text-align:left;">
4
</td>
<td style="text-align:left;">
gene
</td>
<td style="text-align:right;">
125.1500
</td>
<td style="text-align:right;">
125.1871
</td>
<td style="text-align:left;">
positive
</td>
<td style="text-align:left;">
MGI_C57BL6J_5623130
</td>
<td style="text-align:left;">
Gm40245
</td>
<td style="text-align:left;">
NCBI_Gene:105244683
</td>
<td style="text-align:left;">
MGI:5623130
</td>
<td style="text-align:left;">
lncRNA gene
</td>
<td style="text-align:left;">
predicted gene, 40245
</td>
</tr>
<tr>
<td style="text-align:left;">
4
</td>
<td style="text-align:left;">
gene
</td>
<td style="text-align:right;">
125.2184
</td>
<td style="text-align:right;">
125.2199
</td>
<td style="text-align:left;">
positive
</td>
<td style="text-align:left;">
MGI_C57BL6J_5623129
</td>
<td style="text-align:left;">
Gm40244
</td>
<td style="text-align:left;">
NCBI_Gene:105244682
</td>
<td style="text-align:left;">
MGI:5623129
</td>
<td style="text-align:left;">
lncRNA gene
</td>
<td style="text-align:left;">
predicted gene, 40244
</td>
</tr>
<tr>
<td style="text-align:left;">
4
</td>
<td style="text-align:left;">
gene
</td>
<td style="text-align:right;">
125.2286
</td>
<td style="text-align:right;">
125.2421
</td>
<td style="text-align:left;">
positive
</td>
<td style="text-align:left;">
MGI_C57BL6J_1920510
</td>
<td style="text-align:left;">
1700041M05Rik
</td>
<td style="text-align:left;">
ENSEMBL:ENSMUSG00000087010
</td>
<td style="text-align:left;">
MGI:1920510
</td>
<td style="text-align:left;">
lncRNA gene
</td>
<td style="text-align:left;">
RIKEN cDNA 1700041M05 gene
</td>
</tr>
<tr>
<td style="text-align:left;">
4
</td>
<td style="text-align:left;">
gene
</td>
<td style="text-align:right;">
125.2905
</td>
<td style="text-align:right;">
125.3015
</td>
<td style="text-align:left;">
positive
</td>
<td style="text-align:left;">
MGI_C57BL6J_5826506
</td>
<td style="text-align:left;">
Gm46869
</td>
<td style="text-align:left;">
NCBI_Gene:108168977
</td>
<td style="text-align:left;">
MGI:5826506
</td>
<td style="text-align:left;">
lncRNA gene
</td>
<td style="text-align:left;">
predicted gene, 46869
</td>
</tr>
<tr>
<td style="text-align:left;">
4
</td>
<td style="text-align:left;">
gene
</td>
<td style="text-align:right;">
125.3123
</td>
<td style="text-align:right;">
125.3135
</td>
<td style="text-align:left;">
negative
</td>
<td style="text-align:left;">
MGI_C57BL6J_5591855
</td>
<td style="text-align:left;">
Gm32696
</td>
<td style="text-align:left;">
NCBI_Gene:102635326
</td>
<td style="text-align:left;">
MGI:5591855
</td>
<td style="text-align:left;">
lncRNA gene
</td>
<td style="text-align:left;">
predicted gene, 32696
</td>
</tr>
<tr>
<td style="text-align:left;">
4
</td>
<td style="text-align:left;">
gene
</td>
<td style="text-align:right;">
125.3817
</td>
<td style="text-align:right;">
125.3945
</td>
<td style="text-align:left;">
positive
</td>
<td style="text-align:left;">
MGI_C57BL6J_5623132
</td>
<td style="text-align:left;">
Gm40247
</td>
<td style="text-align:left;">
NCBI_Gene:105244685
</td>
<td style="text-align:left;">
MGI:5623132
</td>
<td style="text-align:left;">
lncRNA gene
</td>
<td style="text-align:left;">
predicted gene, 40247
</td>
</tr>
<tr>
<td style="text-align:left;">
4
</td>
<td style="text-align:left;">
gene
</td>
<td style="text-align:right;">
125.4511
</td>
<td style="text-align:right;">
125.4772
</td>
<td style="text-align:left;">
positive
</td>
<td style="text-align:left;">
MGI_C57BL6J_5591915
</td>
<td style="text-align:left;">
Gm32756
</td>
<td style="text-align:left;">
NCBI_Gene:102635406
</td>
<td style="text-align:left;">
MGI:5591915
</td>
<td style="text-align:left;">
lncRNA gene
</td>
<td style="text-align:left;">
predicted gene, 32756
</td>
</tr>
<tr>
<td style="text-align:left;">
4
</td>
<td style="text-align:left;">
gene
</td>
<td style="text-align:right;">
125.4902
</td>
<td style="text-align:right;">
125.7142
</td>
<td style="text-align:left;">
positive
</td>
<td style="text-align:left;">
MGI_C57BL6J_95816
</td>
<td style="text-align:left;">
Grik3
</td>
<td style="text-align:left;">
NCBI_Gene:14807,ENSEMBL:ENSMUSG00000001985
</td>
<td style="text-align:left;">
MGI:95816
</td>
<td style="text-align:left;">
protein coding gene
</td>
<td style="text-align:left;">
glutamate receptor, ionotropic, kainate 3
</td>
</tr>
<tr>
<td style="text-align:left;">
4
</td>
<td style="text-align:left;">
pseudogene
</td>
<td style="text-align:right;">
125.5041
</td>
<td style="text-align:right;">
125.5050
</td>
<td style="text-align:left;">
positive
</td>
<td style="text-align:left;">
MGI_C57BL6J_95590
</td>
<td style="text-align:left;">
Ftl2-ps
</td>
<td style="text-align:left;">
NCBI_Gene:14337,ENSEMBL:ENSMUSG00000082062
</td>
<td style="text-align:left;">
MGI:95590
</td>
<td style="text-align:left;">
pseudogene
</td>
<td style="text-align:left;">
ferritin light polypeptide 2, pseudogene
</td>
</tr>
<tr>
<td style="text-align:left;">
4
</td>
<td style="text-align:left;">
gene
</td>
<td style="text-align:right;">
125.5047
</td>
<td style="text-align:right;">
125.5049
</td>
<td style="text-align:left;">
positive
</td>
<td style="text-align:left;">
MGI_C57BL6J_3629654
</td>
<td style="text-align:left;">
Mir692-2
</td>
<td style="text-align:left;">
NCBI_Gene:735288,ENSEMBL:ENSMUSG00000094403
</td>
<td style="text-align:left;">
MGI:3629654
</td>
<td style="text-align:left;">
miRNA gene
</td>
<td style="text-align:left;">
microRNA 692-2
</td>
</tr>
<tr>
<td style="text-align:left;">
4
</td>
<td style="text-align:left;">
gene
</td>
<td style="text-align:right;">
125.5502
</td>
<td style="text-align:right;">
125.5561
</td>
<td style="text-align:left;">
negative
</td>
<td style="text-align:left;">
MGI_C57BL6J_5591966
</td>
<td style="text-align:left;">
Gm32807
</td>
<td style="text-align:left;">
NCBI_Gene:102635479
</td>
<td style="text-align:left;">
MGI:5591966
</td>
<td style="text-align:left;">
lncRNA gene
</td>
<td style="text-align:left;">
predicted gene, 32807
</td>
</tr>
<tr>
<td style="text-align:left;">
4
</td>
<td style="text-align:left;">
gene
</td>
<td style="text-align:right;">
125.8297
</td>
<td style="text-align:right;">
125.8368
</td>
<td style="text-align:left;">
negative
</td>
<td style="text-align:left;">
MGI_C57BL6J_5623134
</td>
<td style="text-align:left;">
Gm40249
</td>
<td style="text-align:left;">
NCBI_Gene:105244687
</td>
<td style="text-align:left;">
MGI:5623134
</td>
<td style="text-align:left;">
lncRNA gene
</td>
<td style="text-align:left;">
predicted gene, 40249
</td>
</tr>
<tr>
<td style="text-align:left;">
4
</td>
<td style="text-align:left;">
gene
</td>
<td style="text-align:right;">
125.8879
</td>
<td style="text-align:right;">
125.9224
</td>
<td style="text-align:left;">
negative
</td>
<td style="text-align:left;">
MGI_C57BL6J_1919645
</td>
<td style="text-align:left;">
2610028E06Rik
</td>
<td style="text-align:left;">
NCBI_Gene:72395,ENSEMBL:ENSMUSG00000085562
</td>
<td style="text-align:left;">
MGI:1919645
</td>
<td style="text-align:left;">
lncRNA gene
</td>
<td style="text-align:left;">
RIKEN cDNA 2610028E06 gene
</td>
</tr>
<tr>
<td style="text-align:left;">
4
</td>
<td style="text-align:left;">
gene
</td>
<td style="text-align:right;">
126.0112
</td>
<td style="text-align:right;">
126.0245
</td>
<td style="text-align:left;">
negative
</td>
<td style="text-align:left;">
MGI_C57BL6J_5592067
</td>
<td style="text-align:left;">
Gm32908
</td>
<td style="text-align:left;">
NCBI_Gene:102635619
</td>
<td style="text-align:left;">
MGI:5592067
</td>
<td style="text-align:left;">
lncRNA gene
</td>
<td style="text-align:left;">
predicted gene, 32908
</td>
</tr>
<tr>
<td style="text-align:left;">
4
</td>
<td style="text-align:left;">
gene
</td>
<td style="text-align:right;">
126.0234
</td>
<td style="text-align:right;">
126.0450
</td>
<td style="text-align:left;">
positive
</td>
<td style="text-align:left;">
MGI_C57BL6J_1339755
</td>
<td style="text-align:left;">
Csf3r
</td>
<td style="text-align:left;">
NCBI_Gene:12986,ENSEMBL:ENSMUSG00000028859
</td>
<td style="text-align:left;">
MGI:1339755
</td>
<td style="text-align:left;">
protein coding gene
</td>
<td style="text-align:left;">
colony stimulating factor 3 receptor (granulocyte)
</td>
</tr>
<tr>
<td style="text-align:left;">
4
</td>
<td style="text-align:left;">
gene
</td>
<td style="text-align:right;">
126.0469
</td>
<td style="text-align:right;">
126.0555
</td>
<td style="text-align:left;">
positive
</td>
<td style="text-align:left;">
MGI_C57BL6J_1913657
</td>
<td style="text-align:left;">
Mrps15
</td>
<td style="text-align:left;">
NCBI_Gene:66407,ENSEMBL:ENSMUSG00000028861
</td>
<td style="text-align:left;">
MGI:1913657
</td>
<td style="text-align:left;">
protein coding gene
</td>
<td style="text-align:left;">
mitochondrial ribosomal protein S15
</td>
</tr>
<tr>
<td style="text-align:left;">
4
</td>
<td style="text-align:left;">
gene
</td>
<td style="text-align:right;">
126.0560
</td>
<td style="text-align:right;">
126.0922
</td>
<td style="text-align:left;">
positive
</td>
<td style="text-align:left;">
MGI_C57BL6J_1916308
</td>
<td style="text-align:left;">
Oscp1
</td>
<td style="text-align:left;">
NCBI_Gene:230751,ENSEMBL:ENSMUSG00000042616
</td>
<td style="text-align:left;">
MGI:1916308
</td>
<td style="text-align:left;">
protein coding gene
</td>
<td style="text-align:left;">
organic solute carrier partner 1
</td>
</tr>
<tr>
<td style="text-align:left;">
4
</td>
<td style="text-align:left;">
pseudogene
</td>
<td style="text-align:right;">
126.0698
</td>
<td style="text-align:right;">
126.0703
</td>
<td style="text-align:left;">
negative
</td>
<td style="text-align:left;">
MGI_C57BL6J_3650282
</td>
<td style="text-align:left;">
Rpl28-ps3
</td>
<td style="text-align:left;">
NCBI_Gene:100039645,ENSEMBL:ENSMUSG00000079942
</td>
<td style="text-align:left;">
MGI:3650282
</td>
<td style="text-align:left;">
pseudogene
</td>
<td style="text-align:left;">
ribosomal protein L28, pseudogene 3
</td>
</tr>
<tr>
<td style="text-align:left;">
4
</td>
<td style="text-align:left;">
gene
</td>
<td style="text-align:right;">
126.0966
</td>
<td style="text-align:right;">
126.0986
</td>
<td style="text-align:left;">
positive
</td>
<td style="text-align:left;">
MGI_C57BL6J_2151045
</td>
<td style="text-align:left;">
Lsm10
</td>
<td style="text-align:left;">
NCBI_Gene:116748,ENSEMBL:ENSMUSG00000050188
</td>
<td style="text-align:left;">
MGI:2151045
</td>
<td style="text-align:left;">
protein coding gene
</td>
<td style="text-align:left;">
U7 snRNP-specific Sm-like protein LSM10
</td>
</tr>
<tr>
<td style="text-align:left;">
4
</td>
<td style="text-align:left;">
gene
</td>
<td style="text-align:right;">
126.1005
</td>
<td style="text-align:right;">
126.1047
</td>
<td style="text-align:left;">
negative
</td>
<td style="text-align:left;">
MGI_C57BL6J_1924911
</td>
<td style="text-align:left;">
1520401O13Rik
</td>
<td style="text-align:left;">
NA
</td>
<td style="text-align:left;">
NA
</td>
<td style="text-align:left;">
unclassified gene
</td>
<td style="text-align:left;">
RIKEN cDNA 1520401O13 gene
</td>
</tr>
<tr>
<td style="text-align:left;">
4
</td>
<td style="text-align:left;">
gene
</td>
<td style="text-align:right;">
126.1039
</td>
<td style="text-align:right;">
126.1410
</td>
<td style="text-align:left;">
positive
</td>
<td style="text-align:left;">
MGI_C57BL6J_1921428
</td>
<td style="text-align:left;">
Stk40
</td>
<td style="text-align:left;">
NCBI_Gene:74178,ENSEMBL:ENSMUSG00000042608
</td>
<td style="text-align:left;">
MGI:1921428
</td>
<td style="text-align:left;">
protein coding gene
</td>
<td style="text-align:left;">
serine/threonine kinase 40
</td>
</tr>
<tr>
<td style="text-align:left;">
4
</td>
<td style="text-align:left;">
gene
</td>
<td style="text-align:right;">
126.1457
</td>
<td style="text-align:right;">
126.1499
</td>
<td style="text-align:left;">
positive
</td>
<td style="text-align:left;">
MGI_C57BL6J_1922063
</td>
<td style="text-align:left;">
Eva1b
</td>
<td style="text-align:left;">
NCBI_Gene:230752,ENSEMBL:ENSMUSG00000050212
</td>
<td style="text-align:left;">
MGI:1922063
</td>
<td style="text-align:left;">
protein coding gene
</td>
<td style="text-align:left;">
eva-1 homolog B (C. elegans)
</td>
</tr>
<tr>
<td style="text-align:left;">
4
</td>
<td style="text-align:left;">
gene
</td>
<td style="text-align:right;">
126.1460
</td>
<td style="text-align:right;">
126.1477
</td>
<td style="text-align:left;">
positive
</td>
<td style="text-align:left;">
MGI_C57BL6J_3650946
</td>
<td style="text-align:left;">
Gm12946
</td>
<td style="text-align:left;">
ENSEMBL:ENSMUSG00000086961
</td>
<td style="text-align:left;">
MGI:3650946
</td>
<td style="text-align:left;">
lncRNA gene
</td>
<td style="text-align:left;">
predicted gene 12946
</td>
</tr>
<tr>
<td style="text-align:left;">
4
</td>
<td style="text-align:left;">
gene
</td>
<td style="text-align:right;">
126.1506
</td>
<td style="text-align:right;">
126.1640
</td>
<td style="text-align:left;">
negative
</td>
<td style="text-align:left;">
MGI_C57BL6J_1914188
</td>
<td style="text-align:left;">
Sh3d21
</td>
<td style="text-align:left;">
NCBI_Gene:66938,ENSEMBL:ENSMUSG00000073758
</td>
<td style="text-align:left;">
MGI:1914188
</td>
<td style="text-align:left;">
protein coding gene
</td>
<td style="text-align:left;">
SH3 domain containing 21
</td>
</tr>
<tr>
<td style="text-align:left;">
4
</td>
<td style="text-align:left;">
gene
</td>
<td style="text-align:right;">
126.1641
</td>
<td style="text-align:right;">
126.2028
</td>
<td style="text-align:left;">
negative
</td>
<td style="text-align:left;">
MGI_C57BL6J_2442637
</td>
<td style="text-align:left;">
Thrap3
</td>
<td style="text-align:left;">
NCBI_Gene:230753,ENSEMBL:ENSMUSG00000043962
</td>
<td style="text-align:left;">
MGI:2442637
</td>
<td style="text-align:left;">
protein coding gene
</td>
<td style="text-align:left;">
thyroid hormone receptor associated protein 3
</td>
</tr>
<tr>
<td style="text-align:left;">
4
</td>
<td style="text-align:left;">
gene
</td>
<td style="text-align:right;">
126.2012
</td>
<td style="text-align:right;">
126.2013
</td>
<td style="text-align:left;">
negative
</td>
<td style="text-align:left;">
MGI_C57BL6J_5455100
</td>
<td style="text-align:left;">
Gm25323
</td>
<td style="text-align:left;">
ENSEMBL:ENSMUSG00000084504
</td>
<td style="text-align:left;">
MGI:5455100
</td>
<td style="text-align:left;">
snRNA gene
</td>
<td style="text-align:left;">
predicted gene, 25323
</td>
</tr>
<tr>
<td style="text-align:left;">
4
</td>
<td style="text-align:left;">
gene
</td>
<td style="text-align:right;">
126.2029
</td>
<td style="text-align:right;">
126.2039
</td>
<td style="text-align:left;">
positive
</td>
<td style="text-align:left;">
MGI_C57BL6J_1925109
</td>
<td style="text-align:left;">
D730050C22Rik
</td>
<td style="text-align:left;">
NA
</td>
<td style="text-align:left;">
NA
</td>
<td style="text-align:left;">
unclassified gene
</td>
<td style="text-align:left;">
RIKEN cDNA D730050C22 gene
</td>
</tr>
<tr>
<td style="text-align:left;">
4
</td>
<td style="text-align:left;">
gene
</td>
<td style="text-align:right;">
126.2322
</td>
<td style="text-align:right;">
126.2565
</td>
<td style="text-align:left;">
negative
</td>
<td style="text-align:left;">
MGI_C57BL6J_2384297
</td>
<td style="text-align:left;">
Map7d1
</td>
<td style="text-align:left;">
NCBI_Gene:245877,ENSEMBL:ENSMUSG00000028849
</td>
<td style="text-align:left;">
MGI:2384297
</td>
<td style="text-align:left;">
protein coding gene
</td>
<td style="text-align:left;">
MAP7 domain containing 1
</td>
</tr>
<tr>
<td style="text-align:left;">
4
</td>
<td style="text-align:left;">
gene
</td>
<td style="text-align:right;">
126.2623
</td>
<td style="text-align:right;">
126.2759
</td>
<td style="text-align:left;">
positive
</td>
<td style="text-align:left;">
MGI_C57BL6J_1351486
</td>
<td style="text-align:left;">
Trappc3
</td>
<td style="text-align:left;">
NCBI_Gene:27096,ENSEMBL:ENSMUSG00000028847
</td>
<td style="text-align:left;">
MGI:1351486
</td>
<td style="text-align:left;">
protein coding gene
</td>
<td style="text-align:left;">
trafficking protein particle complex 3
</td>
</tr>
<tr>
<td style="text-align:left;">
4
</td>
<td style="text-align:left;">
gene
</td>
<td style="text-align:right;">
126.2868
</td>
<td style="text-align:right;">
126.3143
</td>
<td style="text-align:left;">
positive
</td>
<td style="text-align:left;">
MGI_C57BL6J_88464
</td>
<td style="text-align:left;">
Col8a2
</td>
<td style="text-align:left;">
NCBI_Gene:329941,ENSEMBL:ENSMUSG00000056174
</td>
<td style="text-align:left;">
MGI:88464
</td>
<td style="text-align:left;">
protein coding gene
</td>
<td style="text-align:left;">
collagen, type VIII, alpha 2
</td>
</tr>
<tr>
<td style="text-align:left;">
4
</td>
<td style="text-align:left;">
gene
</td>
<td style="text-align:right;">
126.3160
</td>
<td style="text-align:right;">
126.3217
</td>
<td style="text-align:left;">
negative
</td>
<td style="text-align:left;">
MGI_C57BL6J_2140364
</td>
<td style="text-align:left;">
Adprhl2
</td>
<td style="text-align:left;">
NCBI_Gene:100206,ENSEMBL:ENSMUSG00000042558
</td>
<td style="text-align:left;">
MGI:2140364
</td>
<td style="text-align:left;">
protein coding gene
</td>
<td style="text-align:left;">
ADP-ribosylhydrolase like 2
</td>
</tr>
<tr>
<td style="text-align:left;">
4
</td>
<td style="text-align:left;">
gene
</td>
<td style="text-align:right;">
126.3221
</td>
<td style="text-align:right;">
126.3258
</td>
<td style="text-align:left;">
negative
</td>
<td style="text-align:left;">
MGI_C57BL6J_1346335
</td>
<td style="text-align:left;">
Tekt2
</td>
<td style="text-align:left;">
NCBI_Gene:24084,ENSEMBL:ENSMUSG00000028845
</td>
<td style="text-align:left;">
MGI:1346335
</td>
<td style="text-align:left;">
protein coding gene
</td>
<td style="text-align:left;">
tektin 2
</td>
</tr>
<tr>
<td style="text-align:left;">
4
</td>
<td style="text-align:left;">
gene
</td>
<td style="text-align:right;">
126.3236
</td>
<td style="text-align:right;">
126.3252
</td>
<td style="text-align:left;">
positive
</td>
<td style="text-align:left;">
MGI_C57BL6J_3704237
</td>
<td style="text-align:left;">
Gm12945
</td>
<td style="text-align:left;">
ENSEMBL:ENSMUSG00000087394
</td>
<td style="text-align:left;">
MGI:3704237
</td>
<td style="text-align:left;">
lncRNA gene
</td>
<td style="text-align:left;">
predicted gene 12945
</td>
</tr>
<tr>
<td style="text-align:left;">
4
</td>
<td style="text-align:left;">
gene
</td>
<td style="text-align:right;">
126.3270
</td>
<td style="text-align:right;">
126.4297
</td>
<td style="text-align:left;">
negative
</td>
<td style="text-align:left;">
MGI_C57BL6J_2446634
</td>
<td style="text-align:left;">
Ago3
</td>
<td style="text-align:left;">
NCBI_Gene:214150,ENSEMBL:ENSMUSG00000028842
</td>
<td style="text-align:left;">
MGI:2446634
</td>
<td style="text-align:left;">
protein coding gene
</td>
<td style="text-align:left;">
argonaute RISC catalytic subunit 3
</td>
</tr>
<tr>
<td style="text-align:left;">
4
</td>
<td style="text-align:left;">
gene
</td>
<td style="text-align:right;">
126.4350
</td>
<td style="text-align:right;">
126.4687
</td>
<td style="text-align:left;">
negative
</td>
<td style="text-align:left;">
MGI_C57BL6J_2446630
</td>
<td style="text-align:left;">
Ago1
</td>
<td style="text-align:left;">
NCBI_Gene:236511,ENSEMBL:ENSMUSG00000041530
</td>
<td style="text-align:left;">
MGI:2446630
</td>
<td style="text-align:left;">
protein coding gene
</td>
<td style="text-align:left;">
argonaute RISC catalytic subunit 1
</td>
</tr>
<tr>
<td style="text-align:left;">
4
</td>
<td style="text-align:left;">
pseudogene
</td>
<td style="text-align:right;">
126.4763
</td>
<td style="text-align:right;">
126.4767
</td>
<td style="text-align:left;">
negative
</td>
<td style="text-align:left;">
MGI_C57BL6J_3652021
</td>
<td style="text-align:left;">
Gm12929
</td>
<td style="text-align:left;">
ENSEMBL:ENSMUSG00000084099
</td>
<td style="text-align:left;">
MGI:3652021
</td>
<td style="text-align:left;">
pseudogene
</td>
<td style="text-align:left;">
predicted gene 12929
</td>
</tr>
<tr>
<td style="text-align:left;">
4
</td>
<td style="text-align:left;">
pseudogene
</td>
<td style="text-align:right;">
126.4767
</td>
<td style="text-align:right;">
126.4773
</td>
<td style="text-align:left;">
negative
</td>
<td style="text-align:left;">
MGI_C57BL6J_3652022
</td>
<td style="text-align:left;">
Gm12928
</td>
<td style="text-align:left;">
ENSEMBL:ENSMUSG00000082381
</td>
<td style="text-align:left;">
MGI:3652022
</td>
<td style="text-align:left;">
pseudogene
</td>
<td style="text-align:left;">
predicted gene 12928
</td>
</tr>
<tr>
<td style="text-align:left;">
4
</td>
<td style="text-align:left;">
gene
</td>
<td style="text-align:right;">
126.4895
</td>
<td style="text-align:right;">
126.5336
</td>
<td style="text-align:left;">
negative
</td>
<td style="text-align:left;">
MGI_C57BL6J_1924100
</td>
<td style="text-align:left;">
Ago4
</td>
<td style="text-align:left;">
NCBI_Gene:76850,ENSEMBL:ENSMUSG00000042500
</td>
<td style="text-align:left;">
MGI:1924100
</td>
<td style="text-align:left;">
protein coding gene
</td>
<td style="text-align:left;">
argonaute RISC catalytic subunit 4
</td>
</tr>
<tr>
<td style="text-align:left;">
4
</td>
<td style="text-align:left;">
pseudogene
</td>
<td style="text-align:right;">
126.5217
</td>
<td style="text-align:right;">
126.5221
</td>
<td style="text-align:left;">
positive
</td>
<td style="text-align:left;">
MGI_C57BL6J_3651149
</td>
<td style="text-align:left;">
Gm12935
</td>
<td style="text-align:left;">
ENSEMBL:ENSMUSG00000083461
</td>
<td style="text-align:left;">
MGI:3651149
</td>
<td style="text-align:left;">
pseudogene
</td>
<td style="text-align:left;">
predicted gene 12935
</td>
</tr>
<tr>
<td style="text-align:left;">
4
</td>
<td style="text-align:left;">
pseudogene
</td>
<td style="text-align:right;">
126.5243
</td>
<td style="text-align:right;">
126.5247
</td>
<td style="text-align:left;">
negative
</td>
<td style="text-align:left;">
MGI_C57BL6J_3651150
</td>
<td style="text-align:left;">
Gm12934
</td>
<td style="text-align:left;">
ENSEMBL:ENSMUSG00000082888
</td>
<td style="text-align:left;">
MGI:3651150
</td>
<td style="text-align:left;">
pseudogene
</td>
<td style="text-align:left;">
predicted gene 12934
</td>
</tr>
<tr>
<td style="text-align:left;">
4
</td>
<td style="text-align:left;">
gene
</td>
<td style="text-align:right;">
126.5569
</td>
<td style="text-align:right;">
126.5945
</td>
<td style="text-align:left;">
positive
</td>
<td style="text-align:left;">
MGI_C57BL6J_2445153
</td>
<td style="text-align:left;">
Clspn
</td>
<td style="text-align:left;">
NCBI_Gene:269582,ENSEMBL:ENSMUSG00000042489
</td>
<td style="text-align:left;">
MGI:2445153
</td>
<td style="text-align:left;">
protein coding gene
</td>
<td style="text-align:left;">
claspin
</td>
</tr>
<tr>
<td style="text-align:left;">
4
</td>
<td style="text-align:left;">
gene
</td>
<td style="text-align:right;">
126.5599
</td>
<td style="text-align:right;">
126.5599
</td>
<td style="text-align:left;">
positive
</td>
<td style="text-align:left;">
MGI_C57BL6J_5530942
</td>
<td style="text-align:left;">
Mir7119
</td>
<td style="text-align:left;">
miRBase:MI0022970,NCBI_Gene:102465673,ENSEMBL:ENSMUSG00000098831
</td>
<td style="text-align:left;">
MGI:5530942
</td>
<td style="text-align:left;">
miRNA gene
</td>
<td style="text-align:left;">
microRNA 7119
</td>
</tr>
<tr>
<td style="text-align:left;">
4
</td>
<td style="text-align:left;">
pseudogene
</td>
<td style="text-align:right;">
126.5844
</td>
<td style="text-align:right;">
126.5851
</td>
<td style="text-align:left;">
negative
</td>
<td style="text-align:left;">
MGI_C57BL6J_3651145
</td>
<td style="text-align:left;">
Gm12936
</td>
<td style="text-align:left;">
NCBI_Gene:102635893,ENSEMBL:ENSMUSG00000081319
</td>
<td style="text-align:left;">
MGI:3651145
</td>
<td style="text-align:left;">
pseudogene
</td>
<td style="text-align:left;">
predicted gene 12936
</td>
</tr>
<tr>
<td style="text-align:left;">
4
</td>
<td style="text-align:left;">
gene
</td>
<td style="text-align:right;">
126.6098
</td>
<td style="text-align:right;">
126.6144
</td>
<td style="text-align:left;">
positive
</td>
<td style="text-align:left;">
MGI_C57BL6J_3609248
</td>
<td style="text-align:left;">
5730409E04Rik
</td>
<td style="text-align:left;">
NCBI_Gene:230757,ENSEMBL:ENSMUSG00000073755
</td>
<td style="text-align:left;">
MGI:3609248
</td>
<td style="text-align:left;">
protein coding gene
</td>
<td style="text-align:left;">
RIKEN cDNA 5730409E04Rik gene
</td>
</tr>
<tr>
<td style="text-align:left;">
4
</td>
<td style="text-align:left;">
gene
</td>
<td style="text-align:right;">
126.6144
</td>
<td style="text-align:right;">
126.6148
</td>
<td style="text-align:left;">
negative
</td>
<td style="text-align:left;">
MGI_C57BL6J_5623135
</td>
<td style="text-align:left;">
Gm40250
</td>
<td style="text-align:left;">
NCBI_Gene:105244688
</td>
<td style="text-align:left;">
MGI:5623135
</td>
<td style="text-align:left;">
lncRNA gene
</td>
<td style="text-align:left;">
predicted gene, 40250
</td>
</tr>
<tr>
<td style="text-align:left;">
4
</td>
<td style="text-align:left;">
gene
</td>
<td style="text-align:right;">
126.6164
</td>
<td style="text-align:right;">
126.6203
</td>
<td style="text-align:left;">
negative
</td>
<td style="text-align:left;">
MGI_C57BL6J_5826507
</td>
<td style="text-align:left;">
Gm46870
</td>
<td style="text-align:left;">
NCBI_Gene:108168978
</td>
<td style="text-align:left;">
MGI:5826507
</td>
<td style="text-align:left;">
lncRNA gene
</td>
<td style="text-align:left;">
predicted gene, 46870
</td>
</tr>
<tr>
<td style="text-align:left;">
4
</td>
<td style="text-align:left;">
pseudogene
</td>
<td style="text-align:right;">
126.6652
</td>
<td style="text-align:right;">
126.6662
</td>
<td style="text-align:left;">
positive
</td>
<td style="text-align:left;">
MGI_C57BL6J_3651147
</td>
<td style="text-align:left;">
Gm12933
</td>
<td style="text-align:left;">
NCBI_Gene:620877,ENSEMBL:ENSMUSG00000081210
</td>
<td style="text-align:left;">
MGI:3651147
</td>
<td style="text-align:left;">
pseudogene
</td>
<td style="text-align:left;">
predicted gene 12933
</td>
</tr>
<tr>
<td style="text-align:left;">
4
</td>
<td style="text-align:left;">
gene
</td>
<td style="text-align:right;">
126.6776
</td>
<td style="text-align:right;">
126.7097
</td>
<td style="text-align:left;">
positive
</td>
<td style="text-align:left;">
MGI_C57BL6J_1347045
</td>
<td style="text-align:left;">
Psmb2
</td>
<td style="text-align:left;">
NCBI_Gene:26445,ENSEMBL:ENSMUSG00000028837
</td>
<td style="text-align:left;">
MGI:1347045
</td>
<td style="text-align:left;">
protein coding gene
</td>
<td style="text-align:left;">
proteasome (prosome, macropain) subunit, beta type 2
</td>
</tr>
<tr>
<td style="text-align:left;">
4
</td>
<td style="text-align:left;">
gene
</td>
<td style="text-align:right;">
126.6995
</td>
<td style="text-align:right;">
126.7040
</td>
<td style="text-align:left;">
positive
</td>
<td style="text-align:left;">
MGI_C57BL6J_2442195
</td>
<td style="text-align:left;">
E330021A06Rik
</td>
<td style="text-align:left;">
NA
</td>
<td style="text-align:left;">
NA
</td>
<td style="text-align:left;">
unclassified gene
</td>
<td style="text-align:left;">
RIKEN cDNA E330021A06 gene
</td>
</tr>
<tr>
<td style="text-align:left;">
4
</td>
<td style="text-align:left;">
gene
</td>
<td style="text-align:right;">
126.7160
</td>
<td style="text-align:right;">
126.7363
</td>
<td style="text-align:left;">
negative
</td>
<td style="text-align:left;">
MGI_C57BL6J_2679630
</td>
<td style="text-align:left;">
Tfap2e
</td>
<td style="text-align:left;">
NCBI_Gene:332937,ENSEMBL:ENSMUSG00000042477
</td>
<td style="text-align:left;">
MGI:2679630
</td>
<td style="text-align:left;">
protein coding gene
</td>
<td style="text-align:left;">
transcription factor AP-2, epsilon
</td>
</tr>
<tr>
<td style="text-align:left;">
4
</td>
<td style="text-align:left;">
gene
</td>
<td style="text-align:right;">
126.7438
</td>
<td style="text-align:right;">
126.7536
</td>
<td style="text-align:left;">
negative
</td>
<td style="text-align:left;">
MGI_C57BL6J_1347351
</td>
<td style="text-align:left;">
Ncdn
</td>
<td style="text-align:left;">
NCBI_Gene:26562,ENSEMBL:ENSMUSG00000028833
</td>
<td style="text-align:left;">
MGI:1347351
</td>
<td style="text-align:left;">
protein coding gene
</td>
<td style="text-align:left;">
neurochondrin
</td>
</tr>
<tr>
<td style="text-align:left;">
4
</td>
<td style="text-align:left;">
gene
</td>
<td style="text-align:right;">
126.7535
</td>
<td style="text-align:right;">
126.8701
</td>
<td style="text-align:left;">
positive
</td>
<td style="text-align:left;">
MGI_C57BL6J_2140475
</td>
<td style="text-align:left;">
AU040320
</td>
<td style="text-align:left;">
NCBI_Gene:100317,ENSEMBL:ENSMUSG00000028830
</td>
<td style="text-align:left;">
MGI:2140475
</td>
<td style="text-align:left;">
protein coding gene
</td>
<td style="text-align:left;">
expressed sequence AU040320
</td>
</tr>
<tr>
<td style="text-align:left;">
4
</td>
<td style="text-align:left;">
pseudogene
</td>
<td style="text-align:right;">
126.7955
</td>
<td style="text-align:right;">
126.7964
</td>
<td style="text-align:left;">
negative
</td>
<td style="text-align:left;">
MGI_C57BL6J_3650044
</td>
<td style="text-align:left;">
Gm12937
</td>
<td style="text-align:left;">
ENSEMBL:ENSMUSG00000082498
</td>
<td style="text-align:left;">
MGI:3650044
</td>
<td style="text-align:left;">
pseudogene
</td>
<td style="text-align:left;">
predicted gene 12937
</td>
</tr>
<tr>
<td style="text-align:left;">
4
</td>
<td style="text-align:left;">
gene
</td>
<td style="text-align:right;">
126.8180
</td>
<td style="text-align:right;">
126.8181
</td>
<td style="text-align:left;">
negative
</td>
<td style="text-align:left;">
MGI_C57BL6J_5453938
</td>
<td style="text-align:left;">
Gm24161
</td>
<td style="text-align:left;">
ENSEMBL:ENSMUSG00000077780
</td>
<td style="text-align:left;">
MGI:5453938
</td>
<td style="text-align:left;">
snoRNA gene
</td>
<td style="text-align:left;">
predicted gene, 24161
</td>
</tr>
<tr>
<td style="text-align:left;">
4
</td>
<td style="text-align:left;">
gene
</td>
<td style="text-align:right;">
126.8610
</td>
<td style="text-align:right;">
126.8611
</td>
<td style="text-align:left;">
negative
</td>
<td style="text-align:left;">
MGI_C57BL6J_5455381
</td>
<td style="text-align:left;">
Gm25604
</td>
<td style="text-align:left;">
ENSEMBL:ENSMUSG00000088984
</td>
<td style="text-align:left;">
MGI:5455381
</td>
<td style="text-align:left;">
snoRNA gene
</td>
<td style="text-align:left;">
predicted gene, 25604
</td>
</tr>
<tr>
<td style="text-align:left;">
4
</td>
<td style="text-align:left;">
gene
</td>
<td style="text-align:right;">
126.8618
</td>
<td style="text-align:right;">
126.9688
</td>
<td style="text-align:left;">
negative
</td>
<td style="text-align:left;">
MGI_C57BL6J_1915035
</td>
<td style="text-align:left;">
Zmym4
</td>
<td style="text-align:left;">
NCBI_Gene:67785,ENSEMBL:ENSMUSG00000042446
</td>
<td style="text-align:left;">
MGI:1915035
</td>
<td style="text-align:left;">
protein coding gene
</td>
<td style="text-align:left;">
zinc finger, MYM-type 4
</td>
</tr>
<tr>
<td style="text-align:left;">
4
</td>
<td style="text-align:left;">
gene
</td>
<td style="text-align:right;">
126.9681
</td>
<td style="text-align:right;">
126.9694
</td>
<td style="text-align:left;">
positive
</td>
<td style="text-align:left;">
MGI_C57BL6J_1918435
</td>
<td style="text-align:left;">
4933424C08Rik
</td>
<td style="text-align:left;">
NA
</td>
<td style="text-align:left;">
NA
</td>
<td style="text-align:left;">
unclassified gene
</td>
<td style="text-align:left;">
RIKEN cDNA 4933424C08 gene
</td>
</tr>
<tr>
<td style="text-align:left;">
4
</td>
<td style="text-align:left;">
pseudogene
</td>
<td style="text-align:right;">
126.9834
</td>
<td style="text-align:right;">
126.9836
</td>
<td style="text-align:left;">
positive
</td>
<td style="text-align:left;">
MGI_C57BL6J_3652092
</td>
<td style="text-align:left;">
Gm12939
</td>
<td style="text-align:left;">
ENSEMBL:ENSMUSG00000081609
</td>
<td style="text-align:left;">
MGI:3652092
</td>
<td style="text-align:left;">
pseudogene
</td>
<td style="text-align:left;">
predicted gene 12939
</td>
</tr>
<tr>
<td style="text-align:left;">
4
</td>
<td style="text-align:left;">
gene
</td>
<td style="text-align:right;">
127.0206
</td>
<td style="text-align:right;">
127.0511
</td>
<td style="text-align:left;">
positive
</td>
<td style="text-align:left;">
MGI_C57BL6J_1918764
</td>
<td style="text-align:left;">
Sfpq
</td>
<td style="text-align:left;">
NCBI_Gene:71514,ENSEMBL:ENSMUSG00000028820
</td>
<td style="text-align:left;">
MGI:1918764
</td>
<td style="text-align:left;">
protein coding gene
</td>
<td style="text-align:left;">
splicing factor proline/glutamine rich (polypyrimidine tract binding
protein associated)
</td>
</tr>
<tr>
<td style="text-align:left;">
4
</td>
<td style="text-align:left;">
gene
</td>
<td style="text-align:right;">
127.0337
</td>
<td style="text-align:right;">
127.0494
</td>
<td style="text-align:left;">
positive
</td>
<td style="text-align:left;">
MGI_C57BL6J_3702626
</td>
<td style="text-align:left;">
Gm12940
</td>
<td style="text-align:left;">
ENSEMBL:ENSMUSG00000085334
</td>
<td style="text-align:left;">
MGI:3702626
</td>
<td style="text-align:left;">
lncRNA gene
</td>
<td style="text-align:left;">
predicted gene 12940
</td>
</tr>
<tr>
<td style="text-align:left;">
4
</td>
<td style="text-align:left;">
pseudogene
</td>
<td style="text-align:right;">
127.0464
</td>
<td style="text-align:right;">
127.0466
</td>
<td style="text-align:left;">
positive
</td>
<td style="text-align:left;">
MGI_C57BL6J_3650501
</td>
<td style="text-align:left;">
Gm12941
</td>
<td style="text-align:left;">
ENSEMBL:ENSMUSG00000084088
</td>
<td style="text-align:left;">
MGI:3650501
</td>
<td style="text-align:left;">
pseudogene
</td>
<td style="text-align:left;">
predicted gene 12941
</td>
</tr>
<tr>
<td style="text-align:left;">
4
</td>
<td style="text-align:left;">
gene
</td>
<td style="text-align:right;">
127.0471
</td>
<td style="text-align:right;">
127.0612
</td>
<td style="text-align:left;">
negative
</td>
<td style="text-align:left;">
MGI_C57BL6J_1915560
</td>
<td style="text-align:left;">
Zmym1
</td>
<td style="text-align:left;">
NCBI_Gene:68310,ENSEMBL:ENSMUSG00000043872
</td>
<td style="text-align:left;">
MGI:1915560
</td>
<td style="text-align:left;">
protein coding gene
</td>
<td style="text-align:left;">
zinc finger, MYM domain containing 1
</td>
</tr>
<tr>
<td style="text-align:left;">
4
</td>
<td style="text-align:left;">
pseudogene
</td>
<td style="text-align:right;">
127.0559
</td>
<td style="text-align:right;">
127.0567
</td>
<td style="text-align:left;">
positive
</td>
<td style="text-align:left;">
MGI_C57BL6J_3650498
</td>
<td style="text-align:left;">
Gm12944
</td>
<td style="text-align:left;">
NCBI_Gene:102635997,ENSEMBL:ENSMUSG00000084149
</td>
<td style="text-align:left;">
MGI:3650498
</td>
<td style="text-align:left;">
pseudogene
</td>
<td style="text-align:left;">
predicted gene 12944
</td>
</tr>
<tr>
<td style="text-align:left;">
4
</td>
<td style="text-align:left;">
gene
</td>
<td style="text-align:right;">
127.0774
</td>
<td style="text-align:right;">
127.1244
</td>
<td style="text-align:left;">
positive
</td>
<td style="text-align:left;">
MGI_C57BL6J_106505
</td>
<td style="text-align:left;">
Zmym6
</td>
<td style="text-align:left;">
NCBI_Gene:100177,ENSEMBL:ENSMUSG00000042408
</td>
<td style="text-align:left;">
MGI:106505
</td>
<td style="text-align:left;">
protein coding gene
</td>
<td style="text-align:left;">
zinc finger, MYM-type 6
</td>
</tr>
<tr>
<td style="text-align:left;">
4
</td>
<td style="text-align:left;">
gene
</td>
<td style="text-align:right;">
127.1085
</td>
<td style="text-align:right;">
127.1086
</td>
<td style="text-align:left;">
negative
</td>
<td style="text-align:left;">
MGI_C57BL6J_5455377
</td>
<td style="text-align:left;">
Gm25600
</td>
<td style="text-align:left;">
ENSEMBL:ENSMUSG00000088982
</td>
<td style="text-align:left;">
MGI:5455377
</td>
<td style="text-align:left;">
snRNA gene
</td>
<td style="text-align:left;">
predicted gene, 25600
</td>
</tr>
<tr>
<td style="text-align:left;">
4
</td>
<td style="text-align:left;">
gene
</td>
<td style="text-align:right;">
127.1237
</td>
<td style="text-align:right;">
127.1297
</td>
<td style="text-align:left;">
positive
</td>
<td style="text-align:left;">
MGI_C57BL6J_3758095
</td>
<td style="text-align:left;">
Tmem35b
</td>
<td style="text-align:left;">
NCBI_Gene:100039968,ENSEMBL:ENSMUSG00000070737
</td>
<td style="text-align:left;">
MGI:3758095
</td>
<td style="text-align:left;">
protein coding gene
</td>
<td style="text-align:left;">
transmembrane protein 35B
</td>
</tr>
<tr>
<td style="text-align:left;">
4
</td>
<td style="text-align:left;">
pseudogene
</td>
<td style="text-align:right;">
127.1412
</td>
<td style="text-align:right;">
127.1415
</td>
<td style="text-align:left;">
negative
</td>
<td style="text-align:left;">
MGI_C57BL6J_3650497
</td>
<td style="text-align:left;">
Gm12943
</td>
<td style="text-align:left;">
NCBI_Gene:621588,ENSEMBL:ENSMUSG00000082220
</td>
<td style="text-align:left;">
MGI:3650497
</td>
<td style="text-align:left;">
pseudogene
</td>
<td style="text-align:left;">
predicted gene 12943
</td>
</tr>
<tr>
<td style="text-align:left;">
4
</td>
<td style="text-align:left;">
gene
</td>
<td style="text-align:right;">
127.1685
</td>
<td style="text-align:right;">
127.2370
</td>
<td style="text-align:left;">
positive
</td>
<td style="text-align:left;">
MGI_C57BL6J_3039563
</td>
<td style="text-align:left;">
Dlgap3
</td>
<td style="text-align:left;">
NCBI_Gene:242667,ENSEMBL:ENSMUSG00000042388
</td>
<td style="text-align:left;">
MGI:3039563
</td>
<td style="text-align:left;">
protein coding gene
</td>
<td style="text-align:left;">
DLG associated protein 3
</td>
</tr>
<tr>
<td style="text-align:left;">
4
</td>
<td style="text-align:left;">
gene
</td>
<td style="text-align:right;">
127.2438
</td>
<td style="text-align:right;">
127.2478
</td>
<td style="text-align:left;">
positive
</td>
<td style="text-align:left;">
MGI_C57BL6J_1933141
</td>
<td style="text-align:left;">
Smim12
</td>
<td style="text-align:left;">
NCBI_Gene:80284,ENSEMBL:ENSMUSG00000042380
</td>
<td style="text-align:left;">
MGI:1933141
</td>
<td style="text-align:left;">
protein coding gene
</td>
<td style="text-align:left;">
small integral membrane protein 12
</td>
</tr>
<tr>
<td style="text-align:left;">
4
</td>
<td style="text-align:left;">
gene
</td>
<td style="text-align:right;">
127.2487
</td>
<td style="text-align:right;">
127.3134
</td>
<td style="text-align:left;">
positive
</td>
<td style="text-align:left;">
MGI_C57BL6J_5592428
</td>
<td style="text-align:left;">
Gm33269
</td>
<td style="text-align:left;">
NCBI_Gene:102636104
</td>
<td style="text-align:left;">
MGI:5592428
</td>
<td style="text-align:left;">
lncRNA gene
</td>
<td style="text-align:left;">
predicted gene, 33269
</td>
</tr>
<tr>
<td style="text-align:left;">
4
</td>
<td style="text-align:left;">
gene
</td>
<td style="text-align:right;">
127.2847
</td>
<td style="text-align:right;">
127.2848
</td>
<td style="text-align:left;">
negative
</td>
<td style="text-align:left;">
MGI_C57BL6J_5451998
</td>
<td style="text-align:left;">
Gm22221
</td>
<td style="text-align:left;">
ENSEMBL:ENSMUSG00000087987
</td>
<td style="text-align:left;">
MGI:5451998
</td>
<td style="text-align:left;">
snRNA gene
</td>
<td style="text-align:left;">
predicted gene, 22221
</td>
</tr>
<tr>
<td style="text-align:left;">
4
</td>
<td style="text-align:left;">
pseudogene
</td>
<td style="text-align:right;">
127.2968
</td>
<td style="text-align:right;">
127.2975
</td>
<td style="text-align:left;">
negative
</td>
<td style="text-align:left;">
MGI_C57BL6J_3649530
</td>
<td style="text-align:left;">
Gm12948
</td>
<td style="text-align:left;">
NCBI_Gene:100417246,ENSEMBL:ENSMUSG00000082047
</td>
<td style="text-align:left;">
MGI:3649530
</td>
<td style="text-align:left;">
pseudogene
</td>
<td style="text-align:left;">
predicted gene 12948
</td>
</tr>
<tr>
<td style="text-align:left;">
4
</td>
<td style="text-align:left;">
gene
</td>
<td style="text-align:right;">
127.3114
</td>
<td style="text-align:right;">
127.3140
</td>
<td style="text-align:left;">
negative
</td>
<td style="text-align:left;">
MGI_C57BL6J_95715
</td>
<td style="text-align:left;">
Gja4
</td>
<td style="text-align:left;">
NCBI_Gene:14612,ENSEMBL:ENSMUSG00000050234
</td>
<td style="text-align:left;">
MGI:95715
</td>
<td style="text-align:left;">
protein coding gene
</td>
<td style="text-align:left;">
gap junction protein, alpha 4
</td>
</tr>
<tr>
<td style="text-align:left;">
4
</td>
<td style="text-align:left;">
gene
</td>
<td style="text-align:right;">
127.3252
</td>
<td style="text-align:right;">
127.3308
</td>
<td style="text-align:left;">
negative
</td>
<td style="text-align:left;">
MGI_C57BL6J_95721
</td>
<td style="text-align:left;">
Gjb3
</td>
<td style="text-align:left;">
NCBI_Gene:14620,ENSEMBL:ENSMUSG00000042367
</td>
<td style="text-align:left;">
MGI:95721
</td>
<td style="text-align:left;">
protein coding gene
</td>
<td style="text-align:left;">
gap junction protein, beta 3
</td>
</tr>
<tr>
<td style="text-align:left;">
4
</td>
<td style="text-align:left;">
gene
</td>
<td style="text-align:right;">
127.3292
</td>
<td style="text-align:right;">
127.3433
</td>
<td style="text-align:left;">
positive
</td>
<td style="text-align:left;">
MGI_C57BL6J_5623136
</td>
<td style="text-align:left;">
Gm40251
</td>
<td style="text-align:left;">
NCBI_Gene:105244689
</td>
<td style="text-align:left;">
MGI:5623136
</td>
<td style="text-align:left;">
lncRNA gene
</td>
<td style="text-align:left;">
predicted gene, 40251
</td>
</tr>
<tr>
<td style="text-align:left;">
4
</td>
<td style="text-align:left;">
gene
</td>
<td style="text-align:right;">
127.3410
</td>
<td style="text-align:right;">
127.3541
</td>
<td style="text-align:left;">
negative
</td>
<td style="text-align:left;">
MGI_C57BL6J_95722
</td>
<td style="text-align:left;">
Gjb4
</td>
<td style="text-align:left;">
NCBI_Gene:14621,ENSEMBL:ENSMUSG00000046623
</td>
<td style="text-align:left;">
MGI:95722
</td>
<td style="text-align:left;">
protein coding gene
</td>
<td style="text-align:left;">
gap junction protein, beta 4
</td>
</tr>
<tr>
<td style="text-align:left;">
4
</td>
<td style="text-align:left;">
gene
</td>
<td style="text-align:right;">
127.3548
</td>
<td style="text-align:right;">
127.3582
</td>
<td style="text-align:left;">
negative
</td>
<td style="text-align:left;">
MGI_C57BL6J_95723
</td>
<td style="text-align:left;">
Gjb5
</td>
<td style="text-align:left;">
NCBI_Gene:14622,ENSEMBL:ENSMUSG00000042357
</td>
<td style="text-align:left;">
MGI:95723
</td>
<td style="text-align:left;">
protein coding gene
</td>
<td style="text-align:left;">
gap junction protein, beta 5
</td>
</tr>
<tr>
<td style="text-align:left;">
4
</td>
<td style="text-align:left;">
pseudogene
</td>
<td style="text-align:right;">
127.4083
</td>
<td style="text-align:right;">
127.4097
</td>
<td style="text-align:left;">
negative
</td>
<td style="text-align:left;">
MGI_C57BL6J_3649527
</td>
<td style="text-align:left;">
Gm12947
</td>
<td style="text-align:left;">
NCBI_Gene:667296,ENSEMBL:ENSMUSG00000079998
</td>
<td style="text-align:left;">
MGI:3649527
</td>
<td style="text-align:left;">
pseudogene
</td>
<td style="text-align:left;">
predicted gene 12947
</td>
</tr>
<tr>
<td style="text-align:left;">
4
</td>
<td style="text-align:left;">
gene
</td>
<td style="text-align:right;">
127.4171
</td>
<td style="text-align:right;">
127.4248
</td>
<td style="text-align:left;">
positive
</td>
<td style="text-align:left;">
MGI_C57BL6J_5623137
</td>
<td style="text-align:left;">
Gm40252
</td>
<td style="text-align:left;">
NCBI_Gene:105244690
</td>
<td style="text-align:left;">
MGI:5623137
</td>
<td style="text-align:left;">
lncRNA gene
</td>
<td style="text-align:left;">
predicted gene, 40252
</td>
</tr>
<tr>
<td style="text-align:left;">
4
</td>
<td style="text-align:left;">
gene
</td>
<td style="text-align:right;">
127.4290
</td>
<td style="text-align:right;">
127.4299
</td>
<td style="text-align:left;">
positive
</td>
<td style="text-align:left;">
MGI_C57BL6J_5623138
</td>
<td style="text-align:left;">
Gm40253
</td>
<td style="text-align:left;">
NA
</td>
<td style="text-align:left;">
NA
</td>
<td style="text-align:left;">
lncRNA gene
</td>
<td style="text-align:left;">
predicted gene%2c 40253
</td>
</tr>
<tr>
<td style="text-align:left;">
4
</td>
<td style="text-align:left;">
gene
</td>
<td style="text-align:right;">
127.4748
</td>
<td style="text-align:right;">
127.5329
</td>
<td style="text-align:left;">
positive
</td>
<td style="text-align:left;">
MGI_C57BL6J_1920738
</td>
<td style="text-align:left;">
1700080G11Rik
</td>
<td style="text-align:left;">
NCBI_Gene:73488,ENSEMBL:ENSMUSG00000086771
</td>
<td style="text-align:left;">
MGI:1920738
</td>
<td style="text-align:left;">
lncRNA gene
</td>
<td style="text-align:left;">
RIKEN cDNA 1700080G11 gene
</td>
</tr>
<tr>
<td style="text-align:left;">
4
</td>
<td style="text-align:left;">
gene
</td>
<td style="text-align:right;">
127.4748
</td>
<td style="text-align:right;">
127.4768
</td>
<td style="text-align:left;">
positive
</td>
<td style="text-align:left;">
MGI_C57BL6J_1925335
</td>
<td style="text-align:left;">
4930471C06Rik
</td>
<td style="text-align:left;">
ENSEMBL:ENSMUSG00000086138
</td>
<td style="text-align:left;">
MGI:1925335
</td>
<td style="text-align:left;">
lncRNA gene
</td>
<td style="text-align:left;">
RIKEN cDNA 4930471C06 gene
</td>
</tr>
<tr>
<td style="text-align:left;">
4
</td>
<td style="text-align:left;">
gene
</td>
<td style="text-align:right;">
127.5451
</td>
<td style="text-align:right;">
127.5705
</td>
<td style="text-align:left;">
positive
</td>
<td style="text-align:left;">
MGI_C57BL6J_2444470
</td>
<td style="text-align:left;">
A630031M04Rik
</td>
<td style="text-align:left;">
ENSEMBL:ENSMUSG00000087470
</td>
<td style="text-align:left;">
MGI:2444470
</td>
<td style="text-align:left;">
lncRNA gene
</td>
<td style="text-align:left;">
RIKEN cDNA A630031M04 gene
</td>
</tr>
<tr>
<td style="text-align:left;">
4
</td>
<td style="text-align:left;">
gene
</td>
<td style="text-align:right;">
127.6187
</td>
<td style="text-align:right;">
127.6199
</td>
<td style="text-align:left;">
positive
</td>
<td style="text-align:left;">
MGI_C57BL6J_5623140
</td>
<td style="text-align:left;">
Gm40255
</td>
<td style="text-align:left;">
NCBI_Gene:105244693
</td>
<td style="text-align:left;">
MGI:5623140
</td>
<td style="text-align:left;">
lncRNA gene
</td>
<td style="text-align:left;">
predicted gene, 40255
</td>
</tr>
<tr>
<td style="text-align:left;">
4
</td>
<td style="text-align:left;">
gene
</td>
<td style="text-align:right;">
127.8106
</td>
<td style="text-align:right;">
127.8122
</td>
<td style="text-align:left;">
negative
</td>
<td style="text-align:left;">
MGI_C57BL6J_1920846
</td>
<td style="text-align:left;">
1700112K13Rik
</td>
<td style="text-align:left;">
ENSEMBL:ENSMUSG00000087510
</td>
<td style="text-align:left;">
MGI:1920846
</td>
<td style="text-align:left;">
lncRNA gene
</td>
<td style="text-align:left;">
RIKEN cDNA 1700112K13 gene
</td>
</tr>
<tr>
<td style="text-align:left;">
4
</td>
<td style="text-align:left;">
gene
</td>
<td style="text-align:right;">
127.9139
</td>
<td style="text-align:right;">
127.9164
</td>
<td style="text-align:left;">
positive
</td>
<td style="text-align:left;">
MGI_C57BL6J_5623141
</td>
<td style="text-align:left;">
Gm40256
</td>
<td style="text-align:left;">
NCBI_Gene:105244694
</td>
<td style="text-align:left;">
MGI:5623141
</td>
<td style="text-align:left;">
lncRNA gene
</td>
<td style="text-align:left;">
predicted gene, 40256
</td>
</tr>
<tr>
<td style="text-align:left;">
4
</td>
<td style="text-align:left;">
gene
</td>
<td style="text-align:right;">
127.9276
</td>
<td style="text-align:right;">
127.9881
</td>
<td style="text-align:left;">
negative
</td>
<td style="text-align:left;">
MGI_C57BL6J_3616080
</td>
<td style="text-align:left;">
CK137956
</td>
<td style="text-align:left;">
NCBI_Gene:635169,ENSEMBL:ENSMUSG00000028813
</td>
<td style="text-align:left;">
MGI:3616080
</td>
<td style="text-align:left;">
protein coding gene
</td>
<td style="text-align:left;">
cDNA sequence CK137956
</td>
</tr>
<tr>
<td style="text-align:left;">
4
</td>
<td style="text-align:left;">
gene
</td>
<td style="text-align:right;">
127.9716
</td>
<td style="text-align:right;">
127.9724
</td>
<td style="text-align:left;">
positive
</td>
<td style="text-align:left;">
MGI_C57BL6J_5621456
</td>
<td style="text-align:left;">
Gm38571
</td>
<td style="text-align:left;">
NCBI_Gene:102641725
</td>
<td style="text-align:left;">
MGI:5621456
</td>
<td style="text-align:left;">
lncRNA gene
</td>
<td style="text-align:left;">
predicted gene, 38571
</td>
</tr>
<tr>
<td style="text-align:left;">
4
</td>
<td style="text-align:left;">
gene
</td>
<td style="text-align:right;">
127.9878
</td>
<td style="text-align:right;">
128.5677
</td>
<td style="text-align:left;">
positive
</td>
<td style="text-align:left;">
MGI_C57BL6J_2386401
</td>
<td style="text-align:left;">
Csmd2
</td>
<td style="text-align:left;">
NCBI_Gene:329942,ENSEMBL:ENSMUSG00000028804
</td>
<td style="text-align:left;">
MGI:2386401
</td>
<td style="text-align:left;">
protein coding gene
</td>
<td style="text-align:left;">
CUB and Sushi multiple domains 2
</td>
</tr>
<tr>
<td style="text-align:left;">
4
</td>
<td style="text-align:left;">
gene
</td>
<td style="text-align:right;">
128.1322
</td>
<td style="text-align:right;">
128.1670
</td>
<td style="text-align:left;">
negative
</td>
<td style="text-align:left;">
MGI_C57BL6J_1922239
</td>
<td style="text-align:left;">
Csmd2os
</td>
<td style="text-align:left;">
NCBI_Gene:74989,ENSEMBL:ENSMUSG00000087459
</td>
<td style="text-align:left;">
MGI:1922239
</td>
<td style="text-align:left;">
antisense lncRNA gene
</td>
<td style="text-align:left;">
CUB and Sushi multiple domains 2, opposite strand
</td>
</tr>
<tr>
<td style="text-align:left;">
4
</td>
<td style="text-align:left;">
gene
</td>
<td style="text-align:right;">
128.2602
</td>
<td style="text-align:right;">
128.2610
</td>
<td style="text-align:left;">
negative
</td>
<td style="text-align:left;">
MGI_C57BL6J_1916567
</td>
<td style="text-align:left;">
Hmgb4
</td>
<td style="text-align:left;">
NCBI_Gene:69317,ENSEMBL:ENSMUSG00000048686
</td>
<td style="text-align:left;">
MGI:1916567
</td>
<td style="text-align:left;">
protein coding gene
</td>
<td style="text-align:left;">
high-mobility group box 4
</td>
</tr>
<tr>
<td style="text-align:left;">
4
</td>
<td style="text-align:left;">
gene
</td>
<td style="text-align:right;">
128.2610
</td>
<td style="text-align:right;">
128.2630
</td>
<td style="text-align:left;">
positive
</td>
<td style="text-align:left;">
MGI_C57BL6J_1922492
</td>
<td style="text-align:left;">
Hmgb4os
</td>
<td style="text-align:left;">
NCBI_Gene:75242,ENSEMBL:ENSMUSG00000085717
</td>
<td style="text-align:left;">
MGI:1922492
</td>
<td style="text-align:left;">
antisense lncRNA gene
</td>
<td style="text-align:left;">
high-mobility group box 4, opposite strand
</td>
</tr>
<tr>
<td style="text-align:left;">
4
</td>
<td style="text-align:left;">
gene
</td>
<td style="text-align:right;">
128.5062
</td>
<td style="text-align:right;">
128.5334
</td>
<td style="text-align:left;">
negative
</td>
<td style="text-align:left;">
MGI_C57BL6J_5592815
</td>
<td style="text-align:left;">
Gm33656
</td>
<td style="text-align:left;">
NCBI_Gene:102636648
</td>
<td style="text-align:left;">
MGI:5592815
</td>
<td style="text-align:left;">
lncRNA gene
</td>
<td style="text-align:left;">
predicted gene, 33656
</td>
</tr>
<tr>
<td style="text-align:left;">
4
</td>
<td style="text-align:left;">
gene
</td>
<td style="text-align:right;">
128.5787
</td>
<td style="text-align:right;">
128.6102
</td>
<td style="text-align:left;">
negative
</td>
<td style="text-align:left;">
MGI_C57BL6J_2679268
</td>
<td style="text-align:left;">
Zscan20
</td>
<td style="text-align:left;">
NCBI_Gene:269585,ENSEMBL:ENSMUSG00000061894
</td>
<td style="text-align:left;">
MGI:2679268
</td>
<td style="text-align:left;">
protein coding gene
</td>
<td style="text-align:left;">
zinc finger and SCAN domains 20
</td>
</tr>
<tr>
<td style="text-align:left;">
4
</td>
<td style="text-align:left;">
gene
</td>
<td style="text-align:right;">
128.6154
</td>
<td style="text-align:right;">
128.6186
</td>
<td style="text-align:left;">
negative
</td>
<td style="text-align:left;">
MGI_C57BL6J_3045221
</td>
<td style="text-align:left;">
Tlr12
</td>
<td style="text-align:left;">
NCBI_Gene:384059,ENSEMBL:ENSMUSG00000062545
</td>
<td style="text-align:left;">
MGI:3045221
</td>
<td style="text-align:left;">
protein coding gene
</td>
<td style="text-align:left;">
toll-like receptor 12
</td>
</tr>
<tr>
<td style="text-align:left;">
4
</td>
<td style="text-align:left;">
gene
</td>
<td style="text-align:right;">
128.6372
</td>
<td style="text-align:right;">
128.6392
</td>
<td style="text-align:left;">
positive
</td>
<td style="text-align:left;">
MGI_C57BL6J_3652007
</td>
<td style="text-align:left;">
Gm12958
</td>
<td style="text-align:left;">
NCBI_Gene:105244695,ENSEMBL:ENSMUSG00000084891
</td>
<td style="text-align:left;">
MGI:3652007
</td>
<td style="text-align:left;">
lncRNA gene
</td>
<td style="text-align:left;">
predicted gene 12958
</td>
</tr>
<tr>
<td style="text-align:left;">
4
</td>
<td style="text-align:left;">
gene
</td>
<td style="text-align:right;">
128.6547
</td>
<td style="text-align:right;">
128.7529
</td>
<td style="text-align:left;">
positive
</td>
<td style="text-align:left;">
MGI_C57BL6J_1860454
</td>
<td style="text-align:left;">
Phc2
</td>
<td style="text-align:left;">
NCBI_Gene:54383,ENSEMBL:ENSMUSG00000028796
</td>
<td style="text-align:left;">
MGI:1860454
</td>
<td style="text-align:left;">
protein coding gene
</td>
<td style="text-align:left;">
polyhomeotic 2
</td>
</tr>
<tr>
<td style="text-align:left;">
4
</td>
<td style="text-align:left;">
gene
</td>
<td style="text-align:right;">
128.7213
</td>
<td style="text-align:right;">
128.7275
</td>
<td style="text-align:left;">
negative
</td>
<td style="text-align:left;">
MGI_C57BL6J_5592871
</td>
<td style="text-align:left;">
Gm33712
</td>
<td style="text-align:left;">
NA
</td>
<td style="text-align:left;">
NA
</td>
<td style="text-align:left;">
unclassified non-coding RNA gene
</td>
<td style="text-align:left;">
predicted gene%2c 33712
</td>
</tr>
<tr>
<td style="text-align:left;">
4
</td>
<td style="text-align:left;">
gene
</td>
<td style="text-align:right;">
128.7554
</td>
<td style="text-align:right;">
128.7702
</td>
<td style="text-align:left;">
positive
</td>
<td style="text-align:left;">
MGI_C57BL6J_2685279
</td>
<td style="text-align:left;">
A3galt2
</td>
<td style="text-align:left;">
NCBI_Gene:215493,ENSEMBL:ENSMUSG00000028794
</td>
<td style="text-align:left;">
MGI:2685279
</td>
<td style="text-align:left;">
protein coding gene
</td>
<td style="text-align:left;">
alpha 1,3-galactosyltransferase 2 (isoglobotriaosylceramide synthase)
</td>
</tr>
<tr>
<td style="text-align:left;">
4
</td>
<td style="text-align:left;">
gene
</td>
<td style="text-align:right;">
128.7731
</td>
<td style="text-align:right;">
128.8061
</td>
<td style="text-align:left;">
negative
</td>
<td style="text-align:left;">
MGI_C57BL6J_2652839
</td>
<td style="text-align:left;">
Zfp362
</td>
<td style="text-align:left;">
NCBI_Gene:230761,ENSEMBL:ENSMUSG00000028799
</td>
<td style="text-align:left;">
MGI:2652839
</td>
<td style="text-align:left;">
protein coding gene
</td>
<td style="text-align:left;">
zinc finger protein 362
</td>
</tr>
<tr>
<td style="text-align:left;">
4
</td>
<td style="text-align:left;">
gene
</td>
<td style="text-align:right;">
128.8462
</td>
<td style="text-align:right;">
128.8737
</td>
<td style="text-align:left;">
positive
</td>
<td style="text-align:left;">
MGI_C57BL6J_3649345
</td>
<td style="text-align:left;">
Gm12968
</td>
<td style="text-align:left;">
ENSEMBL:ENSMUSG00000085701
</td>
<td style="text-align:left;">
MGI:3649345
</td>
<td style="text-align:left;">
lncRNA gene
</td>
<td style="text-align:left;">
predicted gene 12968
</td>
</tr>
<tr>
<td style="text-align:left;">
4
</td>
<td style="text-align:left;">
pseudogene
</td>
<td style="text-align:right;">
128.8680
</td>
<td style="text-align:right;">
128.8694
</td>
<td style="text-align:left;">
negative
</td>
<td style="text-align:left;">
MGI_C57BL6J_3650134
</td>
<td style="text-align:left;">
Gm12969
</td>
<td style="text-align:left;">
NCBI_Gene:667354,ENSEMBL:ENSMUSG00000084011
</td>
<td style="text-align:left;">
MGI:3650134
</td>
<td style="text-align:left;">
pseudogene
</td>
<td style="text-align:left;">
predicted gene 12969
</td>
</tr>
<tr>
<td style="text-align:left;">
4
</td>
<td style="text-align:left;">
gene
</td>
<td style="text-align:right;">
128.8720
</td>
<td style="text-align:right;">
128.8774
</td>
<td style="text-align:left;">
negative
</td>
<td style="text-align:left;">
MGI_C57BL6J_5592973
</td>
<td style="text-align:left;">
Gm33814
</td>
<td style="text-align:left;">
NCBI_Gene:102636861
</td>
<td style="text-align:left;">
MGI:5592973
</td>
<td style="text-align:left;">
lncRNA gene
</td>
<td style="text-align:left;">
predicted gene, 33814
</td>
</tr>
<tr>
<td style="text-align:left;">
4
</td>
<td style="text-align:left;">
gene
</td>
<td style="text-align:right;">
128.8776
</td>
<td style="text-align:right;">
128.8823
</td>
<td style="text-align:left;">
positive
</td>
<td style="text-align:left;">
MGI_C57BL6J_5592919
</td>
<td style="text-align:left;">
Gm33760
</td>
<td style="text-align:left;">
NA
</td>
<td style="text-align:left;">
NA
</td>
<td style="text-align:left;">
unclassified non-coding RNA gene
</td>
<td style="text-align:left;">
predicted gene%2c 33760
</td>
</tr>
<tr>
<td style="text-align:left;">
4
</td>
<td style="text-align:left;">
gene
</td>
<td style="text-align:right;">
128.8817
</td>
<td style="text-align:right;">
128.9113
</td>
<td style="text-align:left;">
positive
</td>
<td style="text-align:left;">
MGI_C57BL6J_1914775
</td>
<td style="text-align:left;">
Trim62
</td>
<td style="text-align:left;">
NCBI_Gene:67525,ENSEMBL:ENSMUSG00000041000
</td>
<td style="text-align:left;">
MGI:1914775
</td>
<td style="text-align:left;">
protein coding gene
</td>
<td style="text-align:left;">
tripartite motif-containing 62
</td>
</tr>
<tr>
<td style="text-align:left;">
4
</td>
<td style="text-align:left;">
gene
</td>
<td style="text-align:right;">
128.9173
</td>
<td style="text-align:right;">
128.9223
</td>
<td style="text-align:left;">
positive
</td>
<td style="text-align:left;">
MGI_C57BL6J_5593060
</td>
<td style="text-align:left;">
Gm33901
</td>
<td style="text-align:left;">
NCBI_Gene:102636978
</td>
<td style="text-align:left;">
MGI:5593060
</td>
<td style="text-align:left;">
lncRNA gene
</td>
<td style="text-align:left;">
predicted gene, 33901
</td>
</tr>
<tr>
<td style="text-align:left;">
4
</td>
<td style="text-align:left;">
gene
</td>
<td style="text-align:right;">
128.9302
</td>
<td style="text-align:right;">
128.9625
</td>
<td style="text-align:left;">
negative
</td>
<td style="text-align:left;">
MGI_C57BL6J_2442093
</td>
<td style="text-align:left;">
Azin2
</td>
<td style="text-align:left;">
NCBI_Gene:242669,ENSEMBL:ENSMUSG00000028789
</td>
<td style="text-align:left;">
MGI:2442093
</td>
<td style="text-align:left;">
protein coding gene
</td>
<td style="text-align:left;">
antizyme inhibitor 2
</td>
</tr>
<tr>
<td style="text-align:left;">
4
</td>
<td style="text-align:left;">
gene
</td>
<td style="text-align:right;">
128.9360
</td>
<td style="text-align:right;">
128.9402
</td>
<td style="text-align:left;">
positive
</td>
<td style="text-align:left;">
MGI_C57BL6J_3801803
</td>
<td style="text-align:left;">
Gm15904
</td>
<td style="text-align:left;">
NCBI_Gene:105244696,ENSEMBL:ENSMUSG00000085961
</td>
<td style="text-align:left;">
MGI:3801803
</td>
<td style="text-align:left;">
lncRNA gene
</td>
<td style="text-align:left;">
predicted gene 15904
</td>
</tr>
<tr>
<td style="text-align:left;">
4
</td>
<td style="text-align:left;">
gene
</td>
<td style="text-align:right;">
128.9920
</td>
<td style="text-align:right;">
129.0115
</td>
<td style="text-align:left;">
positive
</td>
<td style="text-align:left;">
MGI_C57BL6J_87978
</td>
<td style="text-align:left;">
Ak2
</td>
<td style="text-align:left;">
NCBI_Gene:11637,ENSEMBL:ENSMUSG00000028792
</td>
<td style="text-align:left;">
MGI:87978
</td>
<td style="text-align:left;">
protein coding gene
</td>
<td style="text-align:left;">
adenylate kinase 2
</td>
</tr>
<tr>
<td style="text-align:left;">
4
</td>
<td style="text-align:left;">
gene
</td>
<td style="text-align:right;">
129.0501
</td>
<td style="text-align:right;">
129.0581
</td>
<td style="text-align:left;">
negative
</td>
<td style="text-align:left;">
MGI_C57BL6J_1914599
</td>
<td style="text-align:left;">
1700086P04Rik
</td>
<td style="text-align:left;">
NCBI_Gene:67349,ENSEMBL:ENSMUSG00000085085
</td>
<td style="text-align:left;">
MGI:1914599
</td>
<td style="text-align:left;">
lncRNA gene
</td>
<td style="text-align:left;">
RIKEN cDNA 1700086P04 gene
</td>
</tr>
<tr>
<td style="text-align:left;">
4
</td>
<td style="text-align:left;">
gene
</td>
<td style="text-align:right;">
129.0581
</td>
<td style="text-align:right;">
129.0859
</td>
<td style="text-align:left;">
positive
</td>
<td style="text-align:left;">
MGI_C57BL6J_1922484
</td>
<td style="text-align:left;">
Rnf19b
</td>
<td style="text-align:left;">
NCBI_Gene:75234,ENSEMBL:ENSMUSG00000028793
</td>
<td style="text-align:left;">
MGI:1922484
</td>
<td style="text-align:left;">
protein coding gene
</td>
<td style="text-align:left;">
ring finger protein 19B
</td>
</tr>
<tr>
<td style="text-align:left;">
4
</td>
<td style="text-align:left;">
gene
</td>
<td style="text-align:right;">
129.1055
</td>
<td style="text-align:right;">
129.1116
</td>
<td style="text-align:left;">
positive
</td>
<td style="text-align:left;">
MGI_C57BL6J_1913510
</td>
<td style="text-align:left;">
Tmem54
</td>
<td style="text-align:left;">
NCBI_Gene:66260,ENSEMBL:ENSMUSG00000028786
</td>
<td style="text-align:left;">
MGI:1913510
</td>
<td style="text-align:left;">
protein coding gene
</td>
<td style="text-align:left;">
transmembrane protein 54
</td>
</tr>
<tr>
<td style="text-align:left;">
4
</td>
<td style="text-align:left;">
gene
</td>
<td style="text-align:right;">
129.1116
</td>
<td style="text-align:right;">
129.1221
</td>
<td style="text-align:left;">
negative
</td>
<td style="text-align:left;">
MGI_C57BL6J_1336200
</td>
<td style="text-align:left;">
Hpca
</td>
<td style="text-align:left;">
NCBI_Gene:15444,ENSEMBL:ENSMUSG00000028785
</td>
<td style="text-align:left;">
MGI:1336200
</td>
<td style="text-align:left;">
protein coding gene
</td>
<td style="text-align:left;">
hippocalcin
</td>
</tr>
<tr>
<td style="text-align:left;">
4
</td>
<td style="text-align:left;">
gene
</td>
<td style="text-align:right;">
129.1351
</td>
<td style="text-align:right;">
129.1369
</td>
<td style="text-align:left;">
positive
</td>
<td style="text-align:left;">
MGI_C57BL6J_3801906
</td>
<td style="text-align:left;">
Gm15906
</td>
<td style="text-align:left;">
ENSEMBL:ENSMUSG00000087416
</td>
<td style="text-align:left;">
MGI:3801906
</td>
<td style="text-align:left;">
lncRNA gene
</td>
<td style="text-align:left;">
predicted gene 15906
</td>
</tr>
<tr>
<td style="text-align:left;">
4
</td>
<td style="text-align:left;">
gene
</td>
<td style="text-align:right;">
129.1369
</td>
<td style="text-align:right;">
129.1446
</td>
<td style="text-align:left;">
positive
</td>
<td style="text-align:left;">
MGI_C57BL6J_1917614
</td>
<td style="text-align:left;">
Fndc5
</td>
<td style="text-align:left;">
NCBI_Gene:384061,ENSEMBL:ENSMUSG00000001334
</td>
<td style="text-align:left;">
MGI:1917614
</td>
<td style="text-align:left;">
protein coding gene
</td>
<td style="text-align:left;">
fibronectin type III domain containing 5
</td>
</tr>
<tr>
<td style="text-align:left;">
4
</td>
<td style="text-align:left;">
gene
</td>
<td style="text-align:right;">
129.1480
</td>
<td style="text-align:right;">
129.1897
</td>
<td style="text-align:left;">
negative
</td>
<td style="text-align:left;">
MGI_C57BL6J_1921898
</td>
<td style="text-align:left;">
S100pbp
</td>
<td style="text-align:left;">
NCBI_Gene:74648,ENSEMBL:ENSMUSG00000040928
</td>
<td style="text-align:left;">
MGI:1921898
</td>
<td style="text-align:left;">
protein coding gene
</td>
<td style="text-align:left;">
S100P binding protein
</td>
</tr>
<tr>
<td style="text-align:left;">
4
</td>
<td style="text-align:left;">
gene
</td>
<td style="text-align:right;">
129.1898
</td>
<td style="text-align:right;">
129.2196
</td>
<td style="text-align:left;">
positive
</td>
<td style="text-align:left;">
MGI_C57BL6J_2147627
</td>
<td style="text-align:left;">
Yars
</td>
<td style="text-align:left;">
NCBI_Gene:107271,ENSEMBL:ENSMUSG00000028811
</td>
<td style="text-align:left;">
MGI:2147627
</td>
<td style="text-align:left;">
protein coding gene
</td>
<td style="text-align:left;">
tyrosyl-tRNA synthetase
</td>
</tr>
<tr>
<td style="text-align:left;">
4
</td>
<td style="text-align:left;">
gene
</td>
<td style="text-align:right;">
129.2196
</td>
<td style="text-align:right;">
129.2614
</td>
<td style="text-align:left;">
negative
</td>
<td style="text-align:left;">
MGI_C57BL6J_2140651
</td>
<td style="text-align:left;">
C77080
</td>
<td style="text-align:left;">
NCBI_Gene:97130,ENSEMBL:ENSMUSG00000050390
</td>
<td style="text-align:left;">
MGI:2140651
</td>
<td style="text-align:left;">
protein coding gene
</td>
<td style="text-align:left;">
expressed sequence C77080
</td>
</tr>
<tr>
<td style="text-align:left;">
4
</td>
<td style="text-align:left;">
gene
</td>
<td style="text-align:right;">
129.2394
</td>
<td style="text-align:right;">
129.2868
</td>
<td style="text-align:left;">
positive
</td>
<td style="text-align:left;">
MGI_C57BL6J_3650499
</td>
<td style="text-align:left;">
Gm12976
</td>
<td style="text-align:left;">
NCBI_Gene:102637131,ENSEMBL:ENSMUSG00000087575
</td>
<td style="text-align:left;">
MGI:3650499
</td>
<td style="text-align:left;">
lncRNA gene
</td>
<td style="text-align:left;">
predicted gene 12976
</td>
</tr>
<tr>
<td style="text-align:left;">
4
</td>
<td style="text-align:left;">
gene
</td>
<td style="text-align:right;">
129.2829
</td>
<td style="text-align:right;">
129.2829
</td>
<td style="text-align:left;">
negative
</td>
<td style="text-align:left;">
MGI_C57BL6J_5452771
</td>
<td style="text-align:left;">
Gm22994
</td>
<td style="text-align:left;">
ENSEMBL:ENSMUSG00000093163
</td>
<td style="text-align:left;">
MGI:5452771
</td>
<td style="text-align:left;">
miRNA gene
</td>
<td style="text-align:left;">
predicted gene, 22994
</td>
</tr>
<tr>
<td style="text-align:left;">
4
</td>
<td style="text-align:left;">
gene
</td>
<td style="text-align:right;">
129.2875
</td>
<td style="text-align:right;">
129.3094
</td>
<td style="text-align:left;">
positive
</td>
<td style="text-align:left;">
MGI_C57BL6J_1916078
</td>
<td style="text-align:left;">
Sync
</td>
<td style="text-align:left;">
NCBI_Gene:68828,ENSEMBL:ENSMUSG00000001333
</td>
<td style="text-align:left;">
MGI:1916078
</td>
<td style="text-align:left;">
protein coding gene
</td>
<td style="text-align:left;">
syncoilin
</td>
</tr>
<tr>
<td style="text-align:left;">
4
</td>
<td style="text-align:left;">
gene
</td>
<td style="text-align:right;">
129.3071
</td>
<td style="text-align:right;">
129.3354
</td>
<td style="text-align:left;">
negative
</td>
<td style="text-align:left;">
MGI_C57BL6J_1194912
</td>
<td style="text-align:left;">
Rbbp4
</td>
<td style="text-align:left;">
NCBI_Gene:19646,ENSEMBL:ENSMUSG00000057236
</td>
<td style="text-align:left;">
MGI:1194912
</td>
<td style="text-align:left;">
protein coding gene
</td>
<td style="text-align:left;">
retinoblastoma binding protein 4, chromatin remodeling factor
</td>
</tr>
<tr>
<td style="text-align:left;">
4
</td>
<td style="text-align:left;">
gene
</td>
<td style="text-align:right;">
129.3091
</td>
<td style="text-align:right;">
129.3184
</td>
<td style="text-align:left;">
positive
</td>
<td style="text-align:left;">
MGI_C57BL6J_5477216
</td>
<td style="text-align:left;">
Gm26722
</td>
<td style="text-align:left;">
ENSEMBL:ENSMUSG00000096944
</td>
<td style="text-align:left;">
MGI:5477216
</td>
<td style="text-align:left;">
lncRNA gene
</td>
<td style="text-align:left;">
predicted gene, 26722
</td>
</tr>
<tr>
<td style="text-align:left;">
4
</td>
<td style="text-align:left;">
gene
</td>
<td style="text-align:right;">
129.3356
</td>
<td style="text-align:right;">
129.3545
</td>
<td style="text-align:left;">
positive
</td>
<td style="text-align:left;">
MGI_C57BL6J_1914356
</td>
<td style="text-align:left;">
Zbtb8os
</td>
<td style="text-align:left;">
NCBI_Gene:67106,ENSEMBL:ENSMUSG00000057572
</td>
<td style="text-align:left;">
MGI:1914356
</td>
<td style="text-align:left;">
protein coding gene
</td>
<td style="text-align:left;">
zinc finger and BTB domain containing 8 opposite strand
</td>
</tr>
<tr>
<td style="text-align:left;">
4
</td>
<td style="text-align:left;">
gene
</td>
<td style="text-align:right;">
129.3536
</td>
<td style="text-align:right;">
129.3781
</td>
<td style="text-align:left;">
negative
</td>
<td style="text-align:left;">
MGI_C57BL6J_1920930
</td>
<td style="text-align:left;">
Zbtb8a
</td>
<td style="text-align:left;">
NCBI_Gene:73680,ENSEMBL:ENSMUSG00000028807
</td>
<td style="text-align:left;">
MGI:1920930
</td>
<td style="text-align:left;">
protein coding gene
</td>
<td style="text-align:left;">
zinc finger and BTB domain containing 8a
</td>
</tr>
<tr>
<td style="text-align:left;">
4
</td>
<td style="text-align:left;">
pseudogene
</td>
<td style="text-align:right;">
129.3798
</td>
<td style="text-align:right;">
129.3802
</td>
<td style="text-align:left;">
positive
</td>
<td style="text-align:left;">
MGI_C57BL6J_3651320
</td>
<td style="text-align:left;">
Gm12980
</td>
<td style="text-align:left;">
ENSEMBL:ENSMUSG00000082468
</td>
<td style="text-align:left;">
MGI:3651320
</td>
<td style="text-align:left;">
pseudogene
</td>
<td style="text-align:left;">
predicted gene 12980
</td>
</tr>
<tr>
<td style="text-align:left;">
4
</td>
<td style="text-align:left;">
gene
</td>
<td style="text-align:right;">
129.3825
</td>
<td style="text-align:right;">
129.4075
</td>
<td style="text-align:left;">
positive
</td>
<td style="text-align:left;">
MGI_C57BL6J_2444986
</td>
<td style="text-align:left;">
C330020E22Rik
</td>
<td style="text-align:left;">
NCBI_Gene:320896
</td>
<td style="text-align:left;">
MGI:2444986
</td>
<td style="text-align:left;">
lncRNA gene
</td>
<td style="text-align:left;">
RIKEN cDNA C330020E22 gene
</td>
</tr>
<tr>
<td style="text-align:left;">
4
</td>
<td style="text-align:left;">
gene
</td>
<td style="text-align:right;">
129.4258
</td>
<td style="text-align:right;">
129.4468
</td>
<td style="text-align:left;">
negative
</td>
<td style="text-align:left;">
MGI_C57BL6J_2387181
</td>
<td style="text-align:left;">
Zbtb8b
</td>
<td style="text-align:left;">
NCBI_Gene:215627,ENSEMBL:ENSMUSG00000048485
</td>
<td style="text-align:left;">
MGI:2387181
</td>
<td style="text-align:left;">
protein coding gene
</td>
<td style="text-align:left;">
zinc finger and BTB domain containing 8b
</td>
</tr>
<tr>
<td style="text-align:left;">
4
</td>
<td style="text-align:left;">
gene
</td>
<td style="text-align:right;">
129.4584
</td>
<td style="text-align:right;">
129.4585
</td>
<td style="text-align:left;">
positive
</td>
<td style="text-align:left;">
MGI_C57BL6J_5454398
</td>
<td style="text-align:left;">
Gm24621
</td>
<td style="text-align:left;">
ENSEMBL:ENSMUSG00000064613
</td>
<td style="text-align:left;">
MGI:5454398
</td>
<td style="text-align:left;">
snRNA gene
</td>
<td style="text-align:left;">
predicted gene, 24621
</td>
</tr>
<tr>
<td style="text-align:left;">
4
</td>
<td style="text-align:left;">
gene
</td>
<td style="text-align:right;">
129.4602
</td>
<td style="text-align:right;">
129.4615
</td>
<td style="text-align:left;">
negative
</td>
<td style="text-align:left;">
MGI_C57BL6J_1918074
</td>
<td style="text-align:left;">
4921504P20Rik
</td>
<td style="text-align:left;">
NA
</td>
<td style="text-align:left;">
NA
</td>
<td style="text-align:left;">
unclassified gene
</td>
<td style="text-align:left;">
RIKEN cDNA 4921504P20 gene
</td>
</tr>
<tr>
<td style="text-align:left;">
4
</td>
<td style="text-align:left;">
gene
</td>
<td style="text-align:right;">
129.4616
</td>
<td style="text-align:right;">
129.4885
</td>
<td style="text-align:left;">
positive
</td>
<td style="text-align:left;">
MGI_C57BL6J_1913466
</td>
<td style="text-align:left;">
Bsdc1
</td>
<td style="text-align:left;">
NCBI_Gene:100383,ENSEMBL:ENSMUSG00000040859
</td>
<td style="text-align:left;">
MGI:1913466
</td>
<td style="text-align:left;">
protein coding gene
</td>
<td style="text-align:left;">
BSD domain containing 1
</td>
</tr>
<tr>
<td style="text-align:left;">
4
</td>
<td style="text-align:left;">
gene
</td>
<td style="text-align:right;">
129.4890
</td>
<td style="text-align:right;">
129.4910
</td>
<td style="text-align:left;">
negative
</td>
<td style="text-align:left;">
MGI_C57BL6J_1929914
</td>
<td style="text-align:left;">
Tssk3
</td>
<td style="text-align:left;">
NCBI_Gene:58864,ENSEMBL:ENSMUSG00000000411
</td>
<td style="text-align:left;">
MGI:1929914
</td>
<td style="text-align:left;">
protein coding gene
</td>
<td style="text-align:left;">
testis-specific serine kinase 3
</td>
</tr>
<tr>
<td style="text-align:left;">
4
</td>
<td style="text-align:left;">
gene
</td>
<td style="text-align:right;">
129.4912
</td>
<td style="text-align:right;">
129.4920
</td>
<td style="text-align:left;">
positive
</td>
<td style="text-align:left;">
MGI_C57BL6J_1915483
</td>
<td style="text-align:left;">
Fam229a
</td>
<td style="text-align:left;">
NCBI_Gene:68233,ENSEMBL:ENSMUSG00000078554
</td>
<td style="text-align:left;">
MGI:1915483
</td>
<td style="text-align:left;">
protein coding gene
</td>
<td style="text-align:left;">
family with sequence similarity 229, member A
</td>
</tr>
<tr>
<td style="text-align:left;">
4
</td>
<td style="text-align:left;">
gene
</td>
<td style="text-align:right;">
129.4920
</td>
<td style="text-align:right;">
129.5097
</td>
<td style="text-align:left;">
negative
</td>
<td style="text-align:left;">
MGI_C57BL6J_3651303
</td>
<td style="text-align:left;">
Gm12979
</td>
<td style="text-align:left;">
ENSEMBL:ENSMUSG00000086336
</td>
<td style="text-align:left;">
MGI:3651303
</td>
<td style="text-align:left;">
lncRNA gene
</td>
<td style="text-align:left;">
predicted gene 12979
</td>
</tr>
<tr>
<td style="text-align:left;">
4
</td>
<td style="text-align:left;">
gene
</td>
<td style="text-align:right;">
129.5136
</td>
<td style="text-align:right;">
129.5160
</td>
<td style="text-align:left;">
positive
</td>
<td style="text-align:left;">
MGI_C57BL6J_97143
</td>
<td style="text-align:left;">
Marcksl1
</td>
<td style="text-align:left;">
NCBI_Gene:17357,ENSEMBL:ENSMUSG00000047945
</td>
<td style="text-align:left;">
MGI:97143
</td>
<td style="text-align:left;">
protein coding gene
</td>
<td style="text-align:left;">
MARCKS-like 1
</td>
</tr>
<tr>
<td style="text-align:left;">
4
</td>
<td style="text-align:left;">
gene
</td>
<td style="text-align:right;">
129.5161
</td>
<td style="text-align:right;">
129.5427
</td>
<td style="text-align:left;">
negative
</td>
<td style="text-align:left;">
MGI_C57BL6J_108086
</td>
<td style="text-align:left;">
Hdac1
</td>
<td style="text-align:left;">
NCBI_Gene:433759,ENSEMBL:ENSMUSG00000028800
</td>
<td style="text-align:left;">
MGI:108086
</td>
<td style="text-align:left;">
protein coding gene
</td>
<td style="text-align:left;">
histone deacetylase 1
</td>
</tr>
<tr>
<td style="text-align:left;">
4
</td>
<td style="text-align:left;">
gene
</td>
<td style="text-align:right;">
129.5483
</td>
<td style="text-align:right;">
129.5736
</td>
<td style="text-align:left;">
negative
</td>
<td style="text-align:left;">
MGI_C57BL6J_96756
</td>
<td style="text-align:left;">
Lck
</td>
<td style="text-align:left;">
NCBI_Gene:16818,ENSEMBL:ENSMUSG00000000409
</td>
<td style="text-align:left;">
MGI:96756
</td>
<td style="text-align:left;">
protein coding gene
</td>
<td style="text-align:left;">
lymphocyte protein tyrosine kinase
</td>
</tr>
<tr>
<td style="text-align:left;">
4
</td>
<td style="text-align:left;">
gene
</td>
<td style="text-align:right;">
129.5577
</td>
<td style="text-align:right;">
129.5578
</td>
<td style="text-align:left;">
negative
</td>
<td style="text-align:left;">
MGI_C57BL6J_5530949
</td>
<td style="text-align:left;">
Mir8119
</td>
<td style="text-align:left;">
miRBase:MI0026051,NCBI_Gene:102465997,ENSEMBL:ENSMUSG00000099070
</td>
<td style="text-align:left;">
MGI:5530949
</td>
<td style="text-align:left;">
miRNA gene
</td>
<td style="text-align:left;">
microRNA 8119
</td>
</tr>
<tr>
<td style="text-align:left;">
4
</td>
<td style="text-align:left;">
gene
</td>
<td style="text-align:right;">
129.5768
</td>
<td style="text-align:right;">
129.5786
</td>
<td style="text-align:left;">
negative
</td>
<td style="text-align:left;">
MGI_C57BL6J_2668032
</td>
<td style="text-align:left;">
Fam167b
</td>
<td style="text-align:left;">
NCBI_Gene:230766,ENSEMBL:ENSMUSG00000050493
</td>
<td style="text-align:left;">
MGI:2668032
</td>
<td style="text-align:left;">
protein coding gene
</td>
<td style="text-align:left;">
family with sequence similarity 167, member B
</td>
</tr>
<tr>
<td style="text-align:left;">
4
</td>
<td style="text-align:left;">
gene
</td>
<td style="text-align:right;">
129.5920
</td>
<td style="text-align:right;">
129.6006
</td>
<td style="text-align:left;">
negative
</td>
<td style="text-align:left;">
MGI_C57BL6J_1860763
</td>
<td style="text-align:left;">
Eif3i
</td>
<td style="text-align:left;">
NCBI_Gene:54709,ENSEMBL:ENSMUSG00000028798
</td>
<td style="text-align:left;">
MGI:1860763
</td>
<td style="text-align:left;">
protein coding gene
</td>
<td style="text-align:left;">
eukaryotic translation initiation factor 3, subunit I
</td>
</tr>
<tr>
<td style="text-align:left;">
4
</td>
<td style="text-align:left;">
gene
</td>
<td style="text-align:right;">
129.6007
</td>
<td style="text-align:right;">
129.6241
</td>
<td style="text-align:left;">
positive
</td>
<td style="text-align:left;">
MGI_C57BL6J_1924049
</td>
<td style="text-align:left;">
Tmem234
</td>
<td style="text-align:left;">
NCBI_Gene:76799,ENSEMBL:ENSMUSG00000028797
</td>
<td style="text-align:left;">
MGI:1924049
</td>
<td style="text-align:left;">
protein coding gene
</td>
<td style="text-align:left;">
transmembrane protein 234
</td>
</tr>
<tr>
<td style="text-align:left;">
4
</td>
<td style="text-align:left;">
gene
</td>
<td style="text-align:right;">
129.6083
</td>
<td style="text-align:right;">
129.6145
</td>
<td style="text-align:left;">
negative
</td>
<td style="text-align:left;">
MGI_C57BL6J_2686212
</td>
<td style="text-align:left;">
Dcdc2b
</td>
<td style="text-align:left;">
NCBI_Gene:100504491,ENSEMBL:ENSMUSG00000078552
</td>
<td style="text-align:left;">
MGI:2686212
</td>
<td style="text-align:left;">
protein coding gene
</td>
<td style="text-align:left;">
doublecortin domain containing 2b
</td>
</tr>
<tr>
<td style="text-align:left;">
4
</td>
<td style="text-align:left;">
gene
</td>
<td style="text-align:right;">
129.6151
</td>
<td style="text-align:right;">
129.6191
</td>
<td style="text-align:left;">
negative
</td>
<td style="text-align:left;">
MGI_C57BL6J_2446212
</td>
<td style="text-align:left;">
Iqcc
</td>
<td style="text-align:left;">
NCBI_Gene:230767,ENSEMBL:ENSMUSG00000040795
</td>
<td style="text-align:left;">
MGI:2446212
</td>
<td style="text-align:left;">
protein coding gene
</td>
<td style="text-align:left;">
IQ motif containing C
</td>
</tr>
<tr>
<td style="text-align:left;">
4
</td>
<td style="text-align:left;">
gene
</td>
<td style="text-align:right;">
129.6193
</td>
<td style="text-align:right;">
129.6239
</td>
<td style="text-align:left;">
negative
</td>
<td style="text-align:left;">
MGI_C57BL6J_1913514
</td>
<td style="text-align:left;">
Ccdc28b
</td>
<td style="text-align:left;">
NCBI_Gene:66264,ENSEMBL:ENSMUSG00000028795
</td>
<td style="text-align:left;">
MGI:1913514
</td>
<td style="text-align:left;">
protein coding gene
</td>
<td style="text-align:left;">
coiled coil domain containing 28B
</td>
</tr>
<tr>
<td style="text-align:left;">
4
</td>
<td style="text-align:left;">
gene
</td>
<td style="text-align:right;">
129.6231
</td>
<td style="text-align:right;">
129.6241
</td>
<td style="text-align:left;">
positive
</td>
<td style="text-align:left;">
MGI_C57BL6J_5593464
</td>
<td style="text-align:left;">
Gm34305
</td>
<td style="text-align:left;">
NCBI_Gene:102637517
</td>
<td style="text-align:left;">
MGI:5593464
</td>
<td style="text-align:left;">
lncRNA gene
</td>
<td style="text-align:left;">
predicted gene, 34305
</td>
</tr>
<tr>
<td style="text-align:left;">
4
</td>
<td style="text-align:left;">
gene
</td>
<td style="text-align:right;">
129.6243
</td>
<td style="text-align:right;">
129.6251
</td>
<td style="text-align:left;">
negative
</td>
<td style="text-align:left;">
MGI_C57BL6J_1920034
</td>
<td style="text-align:left;">
2810452K05Rik
</td>
<td style="text-align:left;">
NA
</td>
<td style="text-align:left;">
NA
</td>
<td style="text-align:left;">
unclassified gene
</td>
<td style="text-align:left;">
RIKEN cDNA 2810452K05 gene
</td>
</tr>
<tr>
<td style="text-align:left;">
4
</td>
<td style="text-align:left;">
gene
</td>
<td style="text-align:right;">
129.6261
</td>
<td style="text-align:right;">
129.6411
</td>
<td style="text-align:left;">
negative
</td>
<td style="text-align:left;">
MGI_C57BL6J_105968
</td>
<td style="text-align:left;">
Txlna
</td>
<td style="text-align:left;">
NCBI_Gene:109658,ENSEMBL:ENSMUSG00000053841
</td>
<td style="text-align:left;">
MGI:105968
</td>
<td style="text-align:left;">
protein coding gene
</td>
<td style="text-align:left;">
taxilin alpha
</td>
</tr>
<tr>
<td style="text-align:left;">
4
</td>
<td style="text-align:left;">
gene
</td>
<td style="text-align:right;">
129.6440
</td>
<td style="text-align:right;">
129.6728
</td>
<td style="text-align:left;">
negative
</td>
<td style="text-align:left;">
MGI_C57BL6J_1100836
</td>
<td style="text-align:left;">
Kpna6
</td>
<td style="text-align:left;">
NCBI_Gene:16650,ENSEMBL:ENSMUSG00000003731
</td>
<td style="text-align:left;">
MGI:1100836
</td>
<td style="text-align:left;">
protein coding gene
</td>
<td style="text-align:left;">
karyopherin (importin) alpha 6
</td>
</tr>
<tr>
<td style="text-align:left;">
4
</td>
<td style="text-align:left;">
gene
</td>
<td style="text-align:right;">
129.6764
</td>
<td style="text-align:right;">
129.6968
</td>
<td style="text-align:left;">
negative
</td>
<td style="text-align:left;">
MGI_C57BL6J_2682939
</td>
<td style="text-align:left;">
Tmem39b
</td>
<td style="text-align:left;">
NCBI_Gene:230770,ENSEMBL:ENSMUSG00000053730
</td>
<td style="text-align:left;">
MGI:2682939
</td>
<td style="text-align:left;">
protein coding gene
</td>
<td style="text-align:left;">
transmembrane protein 39b
</td>
</tr>
<tr>
<td style="text-align:left;">
4
</td>
<td style="text-align:left;">
gene
</td>
<td style="text-align:right;">
129.6845
</td>
<td style="text-align:right;">
129.6846
</td>
<td style="text-align:left;">
negative
</td>
<td style="text-align:left;">
MGI_C57BL6J_5531376
</td>
<td style="text-align:left;">
Mir7016
</td>
<td style="text-align:left;">
miRBase:MI0022865,NCBI_Gene:102465615,ENSEMBL:ENSMUSG00000098422
</td>
<td style="text-align:left;">
MGI:5531376
</td>
<td style="text-align:left;">
miRNA gene
</td>
<td style="text-align:left;">
microRNA 7016
</td>
</tr>
<tr>
<td style="text-align:left;">
4
</td>
<td style="text-align:left;">
gene
</td>
<td style="text-align:right;">
129.7024
</td>
<td style="text-align:right;">
129.7428
</td>
<td style="text-align:left;">
negative
</td>
<td style="text-align:left;">
MGI_C57BL6J_893579
</td>
<td style="text-align:left;">
Khdrbs1
</td>
<td style="text-align:left;">
NCBI_Gene:20218,ENSEMBL:ENSMUSG00000028790
</td>
<td style="text-align:left;">
MGI:893579
</td>
<td style="text-align:left;">
protein coding gene
</td>
<td style="text-align:left;">
KH domain containing, RNA binding, signal transduction associated 1
</td>
</tr>
<tr>
<td style="text-align:left;">
4
</td>
<td style="text-align:left;">
pseudogene
</td>
<td style="text-align:right;">
129.7616
</td>
<td style="text-align:right;">
129.7619
</td>
<td style="text-align:left;">
negative
</td>
<td style="text-align:left;">
MGI_C57BL6J_3652042
</td>
<td style="text-align:left;">
Gm12967
</td>
<td style="text-align:left;">
NCBI_Gene:100040532,ENSEMBL:ENSMUSG00000080944
</td>
<td style="text-align:left;">
MGI:3652042
</td>
<td style="text-align:left;">
pseudogene
</td>
<td style="text-align:left;">
predicted gene 12967
</td>
</tr>
<tr>
<td style="text-align:left;">
4
</td>
<td style="text-align:left;">
gene
</td>
<td style="text-align:right;">
129.7623
</td>
<td style="text-align:right;">
129.7624
</td>
<td style="text-align:left;">
negative
</td>
<td style="text-align:left;">
MGI_C57BL6J_5531399
</td>
<td style="text-align:left;">
Gm28017
</td>
<td style="text-align:left;">
ENSEMBL:ENSMUSG00000099322
</td>
<td style="text-align:left;">
MGI:5531399
</td>
<td style="text-align:left;">
miRNA gene
</td>
<td style="text-align:left;">
predicted gene, 28017
</td>
</tr>
<tr>
<td style="text-align:left;">
4
</td>
<td style="text-align:left;">
gene
</td>
<td style="text-align:right;">
129.8112
</td>
<td style="text-align:right;">
129.8500
</td>
<td style="text-align:left;">
positive
</td>
<td style="text-align:left;">
MGI_C57BL6J_1277117
</td>
<td style="text-align:left;">
Ptp4a2
</td>
<td style="text-align:left;">
NCBI_Gene:19244,ENSEMBL:ENSMUSG00000028788
</td>
<td style="text-align:left;">
MGI:1277117
</td>
<td style="text-align:left;">
protein coding gene
</td>
<td style="text-align:left;">
protein tyrosine phosphatase 4a2
</td>
</tr>
<tr>
<td style="text-align:left;">
4
</td>
<td style="text-align:left;">
gene
</td>
<td style="text-align:right;">
129.8207
</td>
<td style="text-align:right;">
129.8221
</td>
<td style="text-align:left;">
positive
</td>
<td style="text-align:left;">
MGI_C57BL6J_1923343
</td>
<td style="text-align:left;">
5830469G19Rik
</td>
<td style="text-align:left;">
NA
</td>
<td style="text-align:left;">
NA
</td>
<td style="text-align:left;">
unclassified gene
</td>
<td style="text-align:left;">
RIKEN cDNA 5830469G19 gene
</td>
</tr>
<tr>
<td style="text-align:left;">
4
</td>
<td style="text-align:left;">
gene
</td>
<td style="text-align:right;">
129.8525
</td>
<td style="text-align:right;">
129.8537
</td>
<td style="text-align:left;">
negative
</td>
<td style="text-align:left;">
MGI_C57BL6J_1921548
</td>
<td style="text-align:left;">
1700108I11Rik
</td>
<td style="text-align:left;">
NCBI_Gene:105244698
</td>
<td style="text-align:left;">
MGI:1921548
</td>
<td style="text-align:left;">
lncRNA gene
</td>
<td style="text-align:left;">
RIKEN cDNA 1700108I11 gene
</td>
</tr>
<tr>
<td style="text-align:left;">
4
</td>
<td style="text-align:left;">
pseudogene
</td>
<td style="text-align:right;">
129.8551
</td>
<td style="text-align:right;">
129.8567
</td>
<td style="text-align:left;">
positive
</td>
<td style="text-align:left;">
MGI_C57BL6J_3708701
</td>
<td style="text-align:left;">
Gm10571
</td>
<td style="text-align:left;">
ENSEMBL:ENSMUSG00000073753
</td>
<td style="text-align:left;">
MGI:3708701
</td>
<td style="text-align:left;">
pseudogene
</td>
<td style="text-align:left;">
predicted gene 10571
</td>
</tr>
<tr>
<td style="text-align:left;">
4
</td>
<td style="text-align:left;">
gene
</td>
<td style="text-align:right;">
129.8956
</td>
<td style="text-align:right;">
129.8957
</td>
<td style="text-align:left;">
positive
</td>
<td style="text-align:left;">
MGI_C57BL6J_5453622
</td>
<td style="text-align:left;">
Gm23845
</td>
<td style="text-align:left;">
ENSEMBL:ENSMUSG00000065768
</td>
<td style="text-align:left;">
MGI:5453622
</td>
<td style="text-align:left;">
snRNA gene
</td>
<td style="text-align:left;">
predicted gene, 23845
</td>
</tr>
<tr>
<td style="text-align:left;">
4
</td>
<td style="text-align:left;">
pseudogene
</td>
<td style="text-align:right;">
129.8989
</td>
<td style="text-align:right;">
129.9004
</td>
<td style="text-align:left;">
positive
</td>
<td style="text-align:left;">
MGI_C57BL6J_3652043
</td>
<td style="text-align:left;">
Gm12966
</td>
<td style="text-align:left;">
NCBI_Gene:622469,ENSEMBL:ENSMUSG00000070729
</td>
<td style="text-align:left;">
MGI:3652043
</td>
<td style="text-align:left;">
pseudogene
</td>
<td style="text-align:left;">
predicted gene 12966
</td>
</tr>
<tr>
<td style="text-align:left;">
4
</td>
<td style="text-align:left;">
gene
</td>
<td style="text-align:right;">
129.9062
</td>
<td style="text-align:right;">
129.9198
</td>
<td style="text-align:left;">
positive
</td>
<td style="text-align:left;">
MGI_C57BL6J_2442924
</td>
<td style="text-align:left;">
E330017L17Rik
</td>
<td style="text-align:left;">
NCBI_Gene:319894,ENSEMBL:ENSMUSG00000086369
</td>
<td style="text-align:left;">
MGI:2442924
</td>
<td style="text-align:left;">
lncRNA gene
</td>
<td style="text-align:left;">
RIKEN cDNA E330017L17 gene
</td>
</tr>
<tr>
<td style="text-align:left;">
4
</td>
<td style="text-align:left;">
gene
</td>
<td style="text-align:right;">
129.9288
</td>
<td style="text-align:right;">
129.9571
</td>
<td style="text-align:left;">
positive
</td>
<td style="text-align:left;">
MGI_C57BL6J_3652045
</td>
<td style="text-align:left;">
Spocd1
</td>
<td style="text-align:left;">
NCBI_Gene:622480,ENSEMBL:ENSMUSG00000028784
</td>
<td style="text-align:left;">
MGI:3652045
</td>
<td style="text-align:left;">
protein coding gene
</td>
<td style="text-align:left;">
SPOC domain containing 1
</td>
</tr>
<tr>
<td style="text-align:left;">
4
</td>
<td style="text-align:left;">
gene
</td>
<td style="text-align:right;">
129.9604
</td>
<td style="text-align:right;">
129.9651
</td>
<td style="text-align:left;">
positive
</td>
<td style="text-align:left;">
MGI_C57BL6J_1919475
</td>
<td style="text-align:left;">
1700003M07Rik
</td>
<td style="text-align:left;">
NCBI_Gene:72225,ENSEMBL:ENSMUSG00000085389
</td>
<td style="text-align:left;">
MGI:1919475
</td>
<td style="text-align:left;">
lncRNA gene
</td>
<td style="text-align:left;">
RIKEN cDNA 1700003M07 gene
</td>
</tr>
<tr>
<td style="text-align:left;">
4
</td>
<td style="text-align:left;">
gene
</td>
<td style="text-align:right;">
129.9849
</td>
<td style="text-align:right;">
130.0226
</td>
<td style="text-align:left;">
positive
</td>
<td style="text-align:left;">
MGI_C57BL6J_2451244
</td>
<td style="text-align:left;">
Adgrb2
</td>
<td style="text-align:left;">
NCBI_Gene:230775,ENSEMBL:ENSMUSG00000028782
</td>
<td style="text-align:left;">
MGI:2451244
</td>
<td style="text-align:left;">
protein coding gene
</td>
<td style="text-align:left;">
adhesion G protein-coupled receptor B2
</td>
</tr>
<tr>
<td style="text-align:left;">
4
</td>
<td style="text-align:left;">
gene
</td>
<td style="text-align:right;">
130.0230
</td>
<td style="text-align:right;">
130.0497
</td>
<td style="text-align:left;">
negative
</td>
<td style="text-align:left;">
MGI_C57BL6J_3651569
</td>
<td style="text-align:left;">
Gm12963
</td>
<td style="text-align:left;">
NCBI_Gene:102640839,ENSEMBL:ENSMUSG00000085517
</td>
<td style="text-align:left;">
MGI:3651569
</td>
<td style="text-align:left;">
lncRNA gene
</td>
<td style="text-align:left;">
predicted gene 12963
</td>
</tr>
<tr>
<td style="text-align:left;">
4
</td>
<td style="text-align:left;">
gene
</td>
<td style="text-align:right;">
130.0474
</td>
<td style="text-align:right;">
130.0993
</td>
<td style="text-align:left;">
positive
</td>
<td style="text-align:left;">
MGI_C57BL6J_1095396
</td>
<td style="text-align:left;">
Col16a1
</td>
<td style="text-align:left;">
NCBI_Gene:107581,ENSEMBL:ENSMUSG00000040690
</td>
<td style="text-align:left;">
MGI:1095396
</td>
<td style="text-align:left;">
protein coding gene
</td>
<td style="text-align:left;">
collagen, type XVI, alpha 1
</td>
</tr>
<tr>
<td style="text-align:left;">
4
</td>
<td style="text-align:left;">
gene
</td>
<td style="text-align:right;">
130.0762
</td>
<td style="text-align:right;">
130.0795
</td>
<td style="text-align:left;">
negative
</td>
<td style="text-align:left;">
MGI_C57BL6J_5826508
</td>
<td style="text-align:left;">
Gm46871
</td>
<td style="text-align:left;">
NCBI_Gene:108168980
</td>
<td style="text-align:left;">
MGI:5826508
</td>
<td style="text-align:left;">
lncRNA gene
</td>
<td style="text-align:left;">
predicted gene, 46871
</td>
</tr>
<tr>
<td style="text-align:left;">
4
</td>
<td style="text-align:left;">
gene
</td>
<td style="text-align:right;">
130.1026
</td>
<td style="text-align:right;">
130.1281
</td>
<td style="text-align:left;">
positive
</td>
<td style="text-align:left;">
MGI_C57BL6J_1915148
</td>
<td style="text-align:left;">
Pef1
</td>
<td style="text-align:left;">
NCBI_Gene:67898,ENSEMBL:ENSMUSG00000028779
</td>
<td style="text-align:left;">
MGI:1915148
</td>
<td style="text-align:left;">
protein coding gene
</td>
<td style="text-align:left;">
penta-EF hand domain containing 1
</td>
</tr>
<tr>
<td style="text-align:left;">
4
</td>
<td style="text-align:left;">
gene
</td>
<td style="text-align:right;">
130.1055
</td>
<td style="text-align:right;">
130.1083
</td>
<td style="text-align:left;">
negative
</td>
<td style="text-align:left;">
MGI_C57BL6J_5623143
</td>
<td style="text-align:left;">
Gm40258
</td>
<td style="text-align:left;">
NCBI_Gene:105244699
</td>
<td style="text-align:left;">
MGI:5623143
</td>
<td style="text-align:left;">
lncRNA gene
</td>
<td style="text-align:left;">
predicted gene, 40258
</td>
</tr>
<tr>
<td style="text-align:left;">
4
</td>
<td style="text-align:left;">
gene
</td>
<td style="text-align:right;">
130.1302
</td>
<td style="text-align:right;">
130.1397
</td>
<td style="text-align:left;">
negative
</td>
<td style="text-align:left;">
MGI_C57BL6J_2385650
</td>
<td style="text-align:left;">
Hcrtr1
</td>
<td style="text-align:left;">
NCBI_Gene:230777,ENSEMBL:ENSMUSG00000028778
</td>
<td style="text-align:left;">
MGI:2385650
</td>
<td style="text-align:left;">
protein coding gene
</td>
<td style="text-align:left;">
hypocretin (orexin) receptor 1
</td>
</tr>
<tr>
<td style="text-align:left;">
4
</td>
<td style="text-align:left;">
gene
</td>
<td style="text-align:right;">
130.1440
</td>
<td style="text-align:right;">
130.1489
</td>
<td style="text-align:left;">
positive
</td>
<td style="text-align:left;">
MGI_C57BL6J_5623144
</td>
<td style="text-align:left;">
Gm40259
</td>
<td style="text-align:left;">
NCBI_Gene:105244700
</td>
<td style="text-align:left;">
MGI:5623144
</td>
<td style="text-align:left;">
lncRNA gene
</td>
<td style="text-align:left;">
predicted gene, 40259
</td>
</tr>
<tr>
<td style="text-align:left;">
4
</td>
<td style="text-align:left;">
gene
</td>
<td style="text-align:right;">
130.1645
</td>
<td style="text-align:right;">
130.1751
</td>
<td style="text-align:left;">
negative
</td>
<td style="text-align:left;">
MGI_C57BL6J_2137617
</td>
<td style="text-align:left;">
Tinagl1
</td>
<td style="text-align:left;">
NCBI_Gene:94242,ENSEMBL:ENSMUSG00000028776
</td>
<td style="text-align:left;">
MGI:2137617
</td>
<td style="text-align:left;">
protein coding gene
</td>
<td style="text-align:left;">
tubulointerstitial nephritis antigen-like 1
</td>
</tr>
<tr>
<td style="text-align:left;">
4
</td>
<td style="text-align:left;">
gene
</td>
<td style="text-align:right;">
130.2090
</td>
<td style="text-align:right;">
130.2224
</td>
<td style="text-align:left;">
negative
</td>
<td style="text-align:left;">
MGI_C57BL6J_2685699
</td>
<td style="text-align:left;">
Ldc1
</td>
<td style="text-align:left;">
NCBI_Gene:332942,ENSEMBL:ENSMUSG00000023120
</td>
<td style="text-align:left;">
MGI:2685699
</td>
<td style="text-align:left;">
protein coding gene
</td>
<td style="text-align:left;">
leucine decarboxylase 1
</td>
</tr>
<tr>
<td style="text-align:left;">
4
</td>
<td style="text-align:left;">
gene
</td>
<td style="text-align:right;">
130.2366
</td>
<td style="text-align:right;">
130.2491
</td>
<td style="text-align:left;">
positive
</td>
<td style="text-align:left;">
MGI_C57BL6J_5826361
</td>
<td style="text-align:left;">
Gm46724
</td>
<td style="text-align:left;">
NCBI_Gene:108168712
</td>
<td style="text-align:left;">
MGI:5826361
</td>
<td style="text-align:left;">
lncRNA gene
</td>
<td style="text-align:left;">
predicted gene, 46724
</td>
</tr>
<tr>
<td style="text-align:left;">
4
</td>
<td style="text-align:left;">
gene
</td>
<td style="text-align:right;">
130.2535
</td>
<td style="text-align:right;">
130.2792
</td>
<td style="text-align:left;">
negative
</td>
<td style="text-align:left;">
MGI_C57BL6J_1919132
</td>
<td style="text-align:left;">
Serinc2
</td>
<td style="text-align:left;">
NCBI_Gene:230779,ENSEMBL:ENSMUSG00000023232
</td>
<td style="text-align:left;">
MGI:1919132
</td>
<td style="text-align:left;">
protein coding gene
</td>
<td style="text-align:left;">
serine incorporator 2
</td>
</tr>
<tr>
<td style="text-align:left;">
4
</td>
<td style="text-align:left;">
gene
</td>
<td style="text-align:right;">
130.2757
</td>
<td style="text-align:right;">
130.2809
</td>
<td style="text-align:left;">
positive
</td>
<td style="text-align:left;">
MGI_C57BL6J_5594116
</td>
<td style="text-align:left;">
Gm34957
</td>
<td style="text-align:left;">
NCBI_Gene:102638376
</td>
<td style="text-align:left;">
MGI:5594116
</td>
<td style="text-align:left;">
lncRNA gene
</td>
<td style="text-align:left;">
predicted gene, 34957
</td>
</tr>
<tr>
<td style="text-align:left;">
4
</td>
<td style="text-align:left;">
gene
</td>
<td style="text-align:right;">
130.2912
</td>
<td style="text-align:right;">
130.3087
</td>
<td style="text-align:left;">
negative
</td>
<td style="text-align:left;">
MGI_C57BL6J_3642427
</td>
<td style="text-align:left;">
Gm10570
</td>
<td style="text-align:left;">
NCBI_Gene:102638454,ENSEMBL:ENSMUSG00000073752
</td>
<td style="text-align:left;">
MGI:3642427
</td>
<td style="text-align:left;">
lncRNA gene
</td>
<td style="text-align:left;">
predicted gene 10570
</td>
</tr>
<tr>
<td style="text-align:left;">
4
</td>
<td style="text-align:left;">
gene
</td>
<td style="text-align:right;">
130.3086
</td>
<td style="text-align:right;">
130.3155
</td>
<td style="text-align:left;">
positive
</td>
<td style="text-align:left;">
MGI_C57BL6J_95476
</td>
<td style="text-align:left;">
Fabp3
</td>
<td style="text-align:left;">
NCBI_Gene:14077,ENSEMBL:ENSMUSG00000028773
</td>
<td style="text-align:left;">
MGI:95476
</td>
<td style="text-align:left;">
protein coding gene
</td>
<td style="text-align:left;">
fatty acid binding protein 3, muscle and heart
</td>
</tr>
<tr>
<td style="text-align:left;">
4
</td>
<td style="text-align:left;">
gene
</td>
<td style="text-align:right;">
130.3154
</td>
<td style="text-align:right;">
130.3600
</td>
<td style="text-align:left;">
negative
</td>
<td style="text-align:left;">
MGI_C57BL6J_1919955
</td>
<td style="text-align:left;">
Zcchc17
</td>
<td style="text-align:left;">
NCBI_Gene:619605,ENSEMBL:ENSMUSG00000028772
</td>
<td style="text-align:left;">
MGI:1919955
</td>
<td style="text-align:left;">
protein coding gene
</td>
<td style="text-align:left;">
zinc finger, CCHC domain containing 17
</td>
</tr>
<tr>
<td style="text-align:left;">
4
</td>
<td style="text-align:left;">
gene
</td>
<td style="text-align:right;">
130.3177
</td>
<td style="text-align:right;">
130.3333
</td>
<td style="text-align:left;">
positive
</td>
<td style="text-align:left;">
MGI_C57BL6J_5594233
</td>
<td style="text-align:left;">
Gm35074
</td>
<td style="text-align:left;">
NCBI_Gene:102638533
</td>
<td style="text-align:left;">
MGI:5594233
</td>
<td style="text-align:left;">
lncRNA gene
</td>
<td style="text-align:left;">
predicted gene, 35074
</td>
</tr>
<tr>
<td style="text-align:left;">
4
</td>
<td style="text-align:left;">
gene
</td>
<td style="text-align:right;">
130.3601
</td>
<td style="text-align:right;">
130.3900
</td>
<td style="text-align:left;">
positive
</td>
<td style="text-align:left;">
MGI_C57BL6J_1913835
</td>
<td style="text-align:left;">
Snrnp40
</td>
<td style="text-align:left;">
NCBI_Gene:66585,ENSEMBL:ENSMUSG00000074088
</td>
<td style="text-align:left;">
MGI:1913835
</td>
<td style="text-align:left;">
protein coding gene
</td>
<td style="text-align:left;">
small nuclear ribonucleoprotein 40 (U5)
</td>
</tr>
<tr>
<td style="text-align:left;">
4
</td>
<td style="text-align:left;">
gene
</td>
<td style="text-align:right;">
130.3710
</td>
<td style="text-align:right;">
130.3720
</td>
<td style="text-align:left;">
negative
</td>
<td style="text-align:left;">
MGI_C57BL6J_1922417
</td>
<td style="text-align:left;">
4930527F18Rik
</td>
<td style="text-align:left;">
NA
</td>
<td style="text-align:left;">
NA
</td>
<td style="text-align:left;">
unclassified gene
</td>
<td style="text-align:left;">
RIKEN cDNA 4930527F18 gene
</td>
</tr>
<tr>
<td style="text-align:left;">
4
</td>
<td style="text-align:left;">
gene
</td>
<td style="text-align:right;">
130.3799
</td>
<td style="text-align:right;">
130.3808
</td>
<td style="text-align:left;">
positive
</td>
<td style="text-align:left;">
MGI_C57BL6J_1920110
</td>
<td style="text-align:left;">
2900017G11Rik
</td>
<td style="text-align:left;">
NA
</td>
<td style="text-align:left;">
NA
</td>
<td style="text-align:left;">
unclassified gene
</td>
<td style="text-align:left;">
RIKEN cDNA 2900017G11 gene
</td>
</tr>
<tr>
<td style="text-align:left;">
4
</td>
<td style="text-align:left;">
gene
</td>
<td style="text-align:right;">
130.5301
</td>
<td style="text-align:right;">
130.5742
</td>
<td style="text-align:left;">
negative
</td>
<td style="text-align:left;">
MGI_C57BL6J_1914399
</td>
<td style="text-align:left;">
Nkain1
</td>
<td style="text-align:left;">
NCBI_Gene:67149,ENSEMBL:ENSMUSG00000078532
</td>
<td style="text-align:left;">
MGI:1914399
</td>
<td style="text-align:left;">
protein coding gene
</td>
<td style="text-align:left;">
Na+/K+ transporting ATPase interacting 1
</td>
</tr>
<tr>
<td style="text-align:left;">
4
</td>
<td style="text-align:left;">
gene
</td>
<td style="text-align:right;">
130.5723
</td>
<td style="text-align:right;">
130.5731
</td>
<td style="text-align:left;">
negative
</td>
<td style="text-align:left;">
MGI_C57BL6J_1916206
</td>
<td style="text-align:left;">
1500006G06Rik
</td>
<td style="text-align:left;">
NA
</td>
<td style="text-align:left;">
NA
</td>
<td style="text-align:left;">
unclassified gene
</td>
<td style="text-align:left;">
RIKEN cDNA 1500006G06 gene
</td>
</tr>
<tr>
<td style="text-align:left;">
4
</td>
<td style="text-align:left;">
gene
</td>
<td style="text-align:right;">
130.6633
</td>
<td style="text-align:right;">
130.7816
</td>
<td style="text-align:left;">
positive
</td>
<td style="text-align:left;">
MGI_C57BL6J_1931749
</td>
<td style="text-align:left;">
Pum1
</td>
<td style="text-align:left;">
NCBI_Gene:80912,ENSEMBL:ENSMUSG00000028580
</td>
<td style="text-align:left;">
MGI:1931749
</td>
<td style="text-align:left;">
protein coding gene
</td>
<td style="text-align:left;">
pumilio RNA-binding family member 1
</td>
</tr>
<tr>
<td style="text-align:left;">
4
</td>
<td style="text-align:left;">
gene
</td>
<td style="text-align:right;">
130.7097
</td>
<td style="text-align:right;">
130.7105
</td>
<td style="text-align:left;">
positive
</td>
<td style="text-align:left;">
MGI_C57BL6J_1919888
</td>
<td style="text-align:left;">
2810017D21Rik
</td>
<td style="text-align:left;">
NA
</td>
<td style="text-align:left;">
NA
</td>
<td style="text-align:left;">
unclassified gene
</td>
<td style="text-align:left;">
RIKEN cDNA 2810017D21 gene
</td>
</tr>
<tr>
<td style="text-align:left;">
4
</td>
<td style="text-align:left;">
gene
</td>
<td style="text-align:right;">
130.7496
</td>
<td style="text-align:right;">
130.7497
</td>
<td style="text-align:left;">
positive
</td>
<td style="text-align:left;">
MGI_C57BL6J_3819558
</td>
<td style="text-align:left;">
Snord85
</td>
<td style="text-align:left;">
NCBI_Gene:100217460,ENSEMBL:ENSMUSG00000065196
</td>
<td style="text-align:left;">
MGI:3819558
</td>
<td style="text-align:left;">
snoRNA gene
</td>
<td style="text-align:left;">
small nucleolar RNA, C/D box 85
</td>
</tr>
<tr>
<td style="text-align:left;">
4
</td>
<td style="text-align:left;">
gene
</td>
<td style="text-align:right;">
130.7586
</td>
<td style="text-align:right;">
130.7624
</td>
<td style="text-align:left;">
positive
</td>
<td style="text-align:left;">
MGI_C57BL6J_3650041
</td>
<td style="text-align:left;">
Gm12971
</td>
<td style="text-align:left;">
ENSEMBL:ENSMUSG00000087449
</td>
<td style="text-align:left;">
MGI:3650041
</td>
<td style="text-align:left;">
lncRNA gene
</td>
<td style="text-align:left;">
predicted gene 12971
</td>
</tr>
<tr>
<td style="text-align:left;">
4
</td>
<td style="text-align:left;">
gene
</td>
<td style="text-align:right;">
130.7663
</td>
<td style="text-align:right;">
130.7664
</td>
<td style="text-align:left;">
positive
</td>
<td style="text-align:left;">
MGI_C57BL6J_5453868
</td>
<td style="text-align:left;">
Gm24091
</td>
<td style="text-align:left;">
ENSEMBL:ENSMUSG00000093793
</td>
<td style="text-align:left;">
MGI:5453868
</td>
<td style="text-align:left;">
snoRNA gene
</td>
<td style="text-align:left;">
predicted gene, 24091
</td>
</tr>
<tr>
<td style="text-align:left;">
4
</td>
<td style="text-align:left;">
gene
</td>
<td style="text-align:right;">
130.7748
</td>
<td style="text-align:right;">
130.7792
</td>
<td style="text-align:left;">
negative
</td>
<td style="text-align:left;">
MGI_C57BL6J_5477010
</td>
<td style="text-align:left;">
Gm26516
</td>
<td style="text-align:left;">
ENSEMBL:ENSMUSG00000097158
</td>
<td style="text-align:left;">
MGI:5477010
</td>
<td style="text-align:left;">
lncRNA gene
</td>
<td style="text-align:left;">
predicted gene, 26516
</td>
</tr>
<tr>
<td style="text-align:left;">
4
</td>
<td style="text-align:left;">
gene
</td>
<td style="text-align:right;">
130.7757
</td>
<td style="text-align:right;">
130.7758
</td>
<td style="text-align:left;">
positive
</td>
<td style="text-align:left;">
MGI_C57BL6J_5453488
</td>
<td style="text-align:left;">
Gm23711
</td>
<td style="text-align:left;">
ENSEMBL:ENSMUSG00000095119
</td>
<td style="text-align:left;">
MGI:5453488
</td>
<td style="text-align:left;">
snoRNA gene
</td>
<td style="text-align:left;">
predicted gene, 23711
</td>
</tr>
<tr>
<td style="text-align:left;">
4
</td>
<td style="text-align:left;">
gene
</td>
<td style="text-align:right;">
130.7845
</td>
<td style="text-align:right;">
130.7880
</td>
<td style="text-align:left;">
positive
</td>
<td style="text-align:left;">
MGI_C57BL6J_3650043
</td>
<td style="text-align:left;">
Gm12972
</td>
<td style="text-align:left;">
NCBI_Gene:108168710,ENSEMBL:ENSMUSG00000086548
</td>
<td style="text-align:left;">
MGI:3650043
</td>
<td style="text-align:left;">
lncRNA gene
</td>
<td style="text-align:left;">
predicted gene 12972
</td>
</tr>
<tr>
<td style="text-align:left;">
4
</td>
<td style="text-align:left;">
gene
</td>
<td style="text-align:right;">
130.7925
</td>
<td style="text-align:right;">
130.8263
</td>
<td style="text-align:left;">
positive
</td>
<td style="text-align:left;">
MGI_C57BL6J_1349163
</td>
<td style="text-align:left;">
Sdc3
</td>
<td style="text-align:left;">
NCBI_Gene:20970,ENSEMBL:ENSMUSG00000025743
</td>
<td style="text-align:left;">
MGI:1349163
</td>
<td style="text-align:left;">
protein coding gene
</td>
<td style="text-align:left;">
syndecan 3
</td>
</tr>
<tr>
<td style="text-align:left;">
4
</td>
<td style="text-align:left;">
gene
</td>
<td style="text-align:right;">
130.8116
</td>
<td style="text-align:right;">
130.8123
</td>
<td style="text-align:left;">
negative
</td>
<td style="text-align:left;">
MGI_C57BL6J_3650040
</td>
<td style="text-align:left;">
Gm12970
</td>
<td style="text-align:left;">
ENSEMBL:ENSMUSG00000087451
</td>
<td style="text-align:left;">
MGI:3650040
</td>
<td style="text-align:left;">
lncRNA gene
</td>
<td style="text-align:left;">
predicted gene 12970
</td>
</tr>
<tr>
<td style="text-align:left;">
4
</td>
<td style="text-align:left;">
gene
</td>
<td style="text-align:right;">
130.8244
</td>
<td style="text-align:right;">
130.8263
</td>
<td style="text-align:left;">
negative
</td>
<td style="text-align:left;">
MGI_C57BL6J_5477210
</td>
<td style="text-align:left;">
Gm26716
</td>
<td style="text-align:left;">
ENSEMBL:ENSMUSG00000097224
</td>
<td style="text-align:left;">
MGI:5477210
</td>
<td style="text-align:left;">
lncRNA gene
</td>
<td style="text-align:left;">
predicted gene, 26716
</td>
</tr>
<tr>
<td style="text-align:left;">
4
</td>
<td style="text-align:left;">
gene
</td>
<td style="text-align:right;">
130.8278
</td>
<td style="text-align:right;">
130.8342
</td>
<td style="text-align:left;">
negative
</td>
<td style="text-align:left;">
MGI_C57BL6J_5625201
</td>
<td style="text-align:left;">
Gm42316
</td>
<td style="text-align:left;">
NCBI_Gene:105247171
</td>
<td style="text-align:left;">
MGI:5625201
</td>
<td style="text-align:left;">
lncRNA gene
</td>
<td style="text-align:left;">
predicted gene, 42316
</td>
</tr>
<tr>
<td style="text-align:left;">
4
</td>
<td style="text-align:left;">
gene
</td>
<td style="text-align:right;">
130.8578
</td>
<td style="text-align:right;">
130.8579
</td>
<td style="text-align:left;">
negative
</td>
<td style="text-align:left;">
MGI_C57BL6J_5452437
</td>
<td style="text-align:left;">
Gm22660
</td>
<td style="text-align:left;">
ENSEMBL:ENSMUSG00000077381
</td>
<td style="text-align:left;">
MGI:5452437
</td>
<td style="text-align:left;">
snoRNA gene
</td>
<td style="text-align:left;">
predicted gene, 22660
</td>
</tr>
<tr>
<td style="text-align:left;">
4
</td>
<td style="text-align:left;">
gene
</td>
<td style="text-align:right;">
130.8710
</td>
<td style="text-align:right;">
130.8729
</td>
<td style="text-align:left;">
positive
</td>
<td style="text-align:left;">
MGI_C57BL6J_5594453
</td>
<td style="text-align:left;">
Gm35294
</td>
<td style="text-align:left;">
NCBI_Gene:102638819
</td>
<td style="text-align:left;">
MGI:5594453
</td>
<td style="text-align:left;">
lncRNA gene
</td>
<td style="text-align:left;">
predicted gene, 35294
</td>
</tr>
<tr>
<td style="text-align:left;">
4
</td>
<td style="text-align:left;">
gene
</td>
<td style="text-align:right;">
130.9061
</td>
<td style="text-align:right;">
130.9146
</td>
<td style="text-align:left;">
negative
</td>
<td style="text-align:left;">
MGI_C57BL6J_5621484
</td>
<td style="text-align:left;">
Gm38599
</td>
<td style="text-align:left;">
NCBI_Gene:102642196
</td>
<td style="text-align:left;">
MGI:5621484
</td>
<td style="text-align:left;">
lncRNA gene
</td>
<td style="text-align:left;">
predicted gene, 38599
</td>
</tr>
<tr>
<td style="text-align:left;">
4
</td>
<td style="text-align:left;">
gene
</td>
<td style="text-align:right;">
130.9129
</td>
<td style="text-align:right;">
130.9361
</td>
<td style="text-align:left;">
positive
</td>
<td style="text-align:left;">
MGI_C57BL6J_108046
</td>
<td style="text-align:left;">
Laptm5
</td>
<td style="text-align:left;">
NCBI_Gene:16792,ENSEMBL:ENSMUSG00000028581
</td>
<td style="text-align:left;">
MGI:108046
</td>
<td style="text-align:left;">
protein coding gene
</td>
<td style="text-align:left;">
lysosomal-associated protein transmembrane 5
</td>
</tr>
<tr>
<td style="text-align:left;">
4
</td>
<td style="text-align:left;">
gene
</td>
<td style="text-align:right;">
130.9444
</td>
<td style="text-align:right;">
130.9555
</td>
<td style="text-align:left;">
positive
</td>
<td style="text-align:left;">
MGI_C57BL6J_106591
</td>
<td style="text-align:left;">
Matn1
</td>
<td style="text-align:left;">
NCBI_Gene:17180,ENSEMBL:ENSMUSG00000040533
</td>
<td style="text-align:left;">
MGI:106591
</td>
<td style="text-align:left;">
protein coding gene
</td>
<td style="text-align:left;">
matrilin 1, cartilage matrix protein
</td>
</tr>
<tr>
<td style="text-align:left;">
4
</td>
<td style="text-align:left;">
gene
</td>
<td style="text-align:right;">
130.9747
</td>
<td style="text-align:right;">
131.0013
</td>
<td style="text-align:left;">
positive
</td>
<td style="text-align:left;">
MGI_C57BL6J_3650042
</td>
<td style="text-align:left;">
Gm12973
</td>
<td style="text-align:left;">
NCBI_Gene:108168713,ENSEMBL:ENSMUSG00000085804
</td>
<td style="text-align:left;">
MGI:3650042
</td>
<td style="text-align:left;">
lncRNA gene
</td>
<td style="text-align:left;">
predicted gene 12973
</td>
</tr>
<tr>
<td style="text-align:left;">
4
</td>
<td style="text-align:left;">
gene
</td>
<td style="text-align:right;">
131.0008
</td>
<td style="text-align:right;">
131.0105
</td>
<td style="text-align:left;">
negative
</td>
<td style="text-align:left;">
MGI_C57BL6J_1925085
</td>
<td style="text-align:left;">
A930031H19Rik
</td>
<td style="text-align:left;">
NCBI_Gene:77835,ENSEMBL:ENSMUSG00000086262
</td>
<td style="text-align:left;">
MGI:1925085
</td>
<td style="text-align:left;">
lncRNA gene
</td>
<td style="text-align:left;">
RIKEN cDNA A930031H19 gene
</td>
</tr>
<tr>
<td style="text-align:left;">
4
</td>
<td style="text-align:left;">
gene
</td>
<td style="text-align:right;">
131.0639
</td>
<td style="text-align:right;">
131.0656
</td>
<td style="text-align:left;">
negative
</td>
<td style="text-align:left;">
MGI_C57BL6J_5625202
</td>
<td style="text-align:left;">
Gm42317
</td>
<td style="text-align:left;">
NCBI_Gene:105247172
</td>
<td style="text-align:left;">
MGI:5625202
</td>
<td style="text-align:left;">
lncRNA gene
</td>
<td style="text-align:left;">
predicted gene, 42317
</td>
</tr>
<tr>
<td style="text-align:left;">
4
</td>
<td style="text-align:left;">
gene
</td>
<td style="text-align:right;">
131.0757
</td>
<td style="text-align:right;">
131.0803
</td>
<td style="text-align:left;">
negative
</td>
<td style="text-align:left;">
MGI_C57BL6J_5594641
</td>
<td style="text-align:left;">
Gm35482
</td>
<td style="text-align:left;">
NCBI_Gene:102639082
</td>
<td style="text-align:left;">
MGI:5594641
</td>
<td style="text-align:left;">
lncRNA gene
</td>
<td style="text-align:left;">
predicted gene, 35482
</td>
</tr>
<tr>
<td style="text-align:left;">
4
</td>
<td style="text-align:left;">
gene
</td>
<td style="text-align:right;">
131.1563
</td>
<td style="text-align:right;">
131.1627
</td>
<td style="text-align:left;">
positive
</td>
<td style="text-align:left;">
MGI_C57BL6J_5625203
</td>
<td style="text-align:left;">
Gm42318
</td>
<td style="text-align:left;">
NCBI_Gene:105247173
</td>
<td style="text-align:left;">
MGI:5625203
</td>
<td style="text-align:left;">
lncRNA gene
</td>
<td style="text-align:left;">
predicted gene, 42318
</td>
</tr>
<tr>
<td style="text-align:left;">
4
</td>
<td style="text-align:left;">
gene
</td>
<td style="text-align:right;">
131.2393
</td>
<td style="text-align:right;">
131.2394
</td>
<td style="text-align:left;">
positive
</td>
<td style="text-align:left;">
MGI_C57BL6J_5455038
</td>
<td style="text-align:left;">
Gm25261
</td>
<td style="text-align:left;">
ENSEMBL:ENSMUSG00000088041
</td>
<td style="text-align:left;">
MGI:5455038
</td>
<td style="text-align:left;">
snoRNA gene
</td>
<td style="text-align:left;">
predicted gene, 25261
</td>
</tr>
<tr>
<td style="text-align:left;">
4
</td>
<td style="text-align:left;">
gene
</td>
<td style="text-align:right;">
131.4091
</td>
<td style="text-align:right;">
131.4283
</td>
<td style="text-align:left;">
positive
</td>
<td style="text-align:left;">
MGI_C57BL6J_5594793
</td>
<td style="text-align:left;">
Gm35634
</td>
<td style="text-align:left;">
NCBI_Gene:102639287
</td>
<td style="text-align:left;">
MGI:5594793
</td>
<td style="text-align:left;">
lncRNA gene
</td>
<td style="text-align:left;">
predicted gene, 35634
</td>
</tr>
<tr>
<td style="text-align:left;">
4
</td>
<td style="text-align:left;">
gene
</td>
<td style="text-align:right;">
131.4563
</td>
<td style="text-align:right;">
131.4826
</td>
<td style="text-align:left;">
positive
</td>
<td style="text-align:left;">
MGI_C57BL6J_5826360
</td>
<td style="text-align:left;">
Gm46723
</td>
<td style="text-align:left;">
NCBI_Gene:108168711
</td>
<td style="text-align:left;">
MGI:5826360
</td>
<td style="text-align:left;">
lncRNA gene
</td>
<td style="text-align:left;">
predicted gene, 46723
</td>
</tr>
<tr>
<td style="text-align:left;">
4
</td>
<td style="text-align:left;">
gene
</td>
<td style="text-align:right;">
131.4584
</td>
<td style="text-align:right;">
131.4855
</td>
<td style="text-align:left;">
negative
</td>
<td style="text-align:left;">
MGI_C57BL6J_5594938
</td>
<td style="text-align:left;">
Gm35779
</td>
<td style="text-align:left;">
NCBI_Gene:102639475
</td>
<td style="text-align:left;">
MGI:5594938
</td>
<td style="text-align:left;">
lncRNA gene
</td>
<td style="text-align:left;">
predicted gene, 35779
</td>
</tr>
<tr>
<td style="text-align:left;">
4
</td>
<td style="text-align:left;">
gene
</td>
<td style="text-align:right;">
131.4872
</td>
<td style="text-align:right;">
131.5055
</td>
<td style="text-align:left;">
positive
</td>
<td style="text-align:left;">
MGI_C57BL6J_2685677
</td>
<td style="text-align:left;">
Gm831
</td>
<td style="text-align:left;">
NCBI_Gene:329950,ENSEMBL:ENSMUSG00000078531
</td>
<td style="text-align:left;">
MGI:2685677
</td>
<td style="text-align:left;">
lncRNA gene
</td>
<td style="text-align:left;">
predicted gene 831
</td>
</tr>
<tr>
<td style="text-align:left;">
4
</td>
<td style="text-align:left;">
gene
</td>
<td style="text-align:right;">
131.6174
</td>
<td style="text-align:right;">
131.6351
</td>
<td style="text-align:left;">
positive
</td>
<td style="text-align:left;">
MGI_C57BL6J_5625205
</td>
<td style="text-align:left;">
Gm42320
</td>
<td style="text-align:left;">
NCBI_Gene:105247175
</td>
<td style="text-align:left;">
MGI:5625205
</td>
<td style="text-align:left;">
lncRNA gene
</td>
<td style="text-align:left;">
predicted gene, 42320
</td>
</tr>
<tr>
<td style="text-align:left;">
4
</td>
<td style="text-align:left;">
gene
</td>
<td style="text-align:right;">
131.6470
</td>
<td style="text-align:right;">
131.6778
</td>
<td style="text-align:left;">
negative
</td>
<td style="text-align:left;">
MGI_C57BL6J_3650901
</td>
<td style="text-align:left;">
Gm12962
</td>
<td style="text-align:left;">
ENSEMBL:ENSMUSG00000087171
</td>
<td style="text-align:left;">
MGI:3650901
</td>
<td style="text-align:left;">
lncRNA gene
</td>
<td style="text-align:left;">
predicted gene 12962
</td>
</tr>
<tr>
<td style="text-align:left;">
4
</td>
<td style="text-align:left;">
gene
</td>
<td style="text-align:right;">
131.6471
</td>
<td style="text-align:right;">
131.7105
</td>
<td style="text-align:left;">
negative
</td>
<td style="text-align:left;">
MGI_C57BL6J_5625206
</td>
<td style="text-align:left;">
Gm42321
</td>
<td style="text-align:left;">
NCBI_Gene:105247176
</td>
<td style="text-align:left;">
MGI:5625206
</td>
<td style="text-align:left;">
lncRNA gene
</td>
<td style="text-align:left;">
predicted gene, 42321
</td>
</tr>
<tr>
<td style="text-align:left;">
4
</td>
<td style="text-align:left;">
gene
</td>
<td style="text-align:right;">
131.6946
</td>
<td style="text-align:right;">
131.7105
</td>
<td style="text-align:left;">
negative
</td>
<td style="text-align:left;">
MGI_C57BL6J_3802056
</td>
<td style="text-align:left;">
Gm16080
</td>
<td style="text-align:left;">
ENSEMBL:ENSMUSG00000086856
</td>
<td style="text-align:left;">
MGI:3802056
</td>
<td style="text-align:left;">
lncRNA gene
</td>
<td style="text-align:left;">
predicted gene 16080
</td>
</tr>
<tr>
<td style="text-align:left;">
4
</td>
<td style="text-align:left;">
gene
</td>
<td style="text-align:right;">
131.7685
</td>
<td style="text-align:right;">
131.8383
</td>
<td style="text-align:left;">
negative
</td>
<td style="text-align:left;">
MGI_C57BL6J_1321151
</td>
<td style="text-align:left;">
Ptpru
</td>
<td style="text-align:left;">
NCBI_Gene:19273,ENSEMBL:ENSMUSG00000028909
</td>
<td style="text-align:left;">
MGI:1321151
</td>
<td style="text-align:left;">
protein coding gene
</td>
<td style="text-align:left;">
protein tyrosine phosphatase, receptor type, U
</td>
</tr>
<tr>
<td style="text-align:left;">
4
</td>
<td style="text-align:left;">
gene
</td>
<td style="text-align:right;">
131.8434
</td>
<td style="text-align:right;">
131.8678
</td>
<td style="text-align:left;">
positive
</td>
<td style="text-align:left;">
MGI_C57BL6J_1349441
</td>
<td style="text-align:left;">
Mecr
</td>
<td style="text-align:left;">
NCBI_Gene:26922,ENSEMBL:ENSMUSG00000028910
</td>
<td style="text-align:left;">
MGI:1349441
</td>
<td style="text-align:left;">
protein coding gene
</td>
<td style="text-align:left;">
mitochondrial trans-2-enoyl-CoA reductase
</td>
</tr>
<tr>
<td style="text-align:left;">
4
</td>
<td style="text-align:left;">
gene
</td>
<td style="text-align:right;">
131.8736
</td>
<td style="text-align:right;">
131.9017
</td>
<td style="text-align:left;">
positive
</td>
<td style="text-align:left;">
MGI_C57BL6J_1890577
</td>
<td style="text-align:left;">
Srsf4
</td>
<td style="text-align:left;">
NCBI_Gene:57317,ENSEMBL:ENSMUSG00000028911
</td>
<td style="text-align:left;">
MGI:1890577
</td>
<td style="text-align:left;">
protein coding gene
</td>
<td style="text-align:left;">
serine/arginine-rich splicing factor 4
</td>
</tr>
<tr>
<td style="text-align:left;">
4
</td>
<td style="text-align:left;">
gene
</td>
<td style="text-align:right;">
131.8786
</td>
<td style="text-align:right;">
131.8832
</td>
<td style="text-align:left;">
positive
</td>
<td style="text-align:left;">
MGI_C57BL6J_2442442
</td>
<td style="text-align:left;">
A930004J17Rik
</td>
<td style="text-align:left;">
ENSEMBL:ENSMUSG00000102602
</td>
<td style="text-align:left;">
MGI:2442442
</td>
<td style="text-align:left;">
unclassified gene
</td>
<td style="text-align:left;">
RIKEN cDNA A930004J17 gene
</td>
</tr>
<tr>
<td style="text-align:left;">
4
</td>
<td style="text-align:left;">
gene
</td>
<td style="text-align:right;">
131.8879
</td>
<td style="text-align:right;">
131.8894
</td>
<td style="text-align:left;">
positive
</td>
<td style="text-align:left;">
MGI_C57BL6J_1918083
</td>
<td style="text-align:left;">
4733401A01Rik
</td>
<td style="text-align:left;">
NA
</td>
<td style="text-align:left;">
NA
</td>
<td style="text-align:left;">
unclassified gene
</td>
<td style="text-align:left;">
RIKEN cDNA 4733401A01 gene
</td>
</tr>
<tr>
<td style="text-align:left;">
4
</td>
<td style="text-align:left;">
gene
</td>
<td style="text-align:right;">
131.8995
</td>
<td style="text-align:right;">
131.9200
</td>
<td style="text-align:left;">
negative
</td>
<td style="text-align:left;">
MGI_C57BL6J_3702644
</td>
<td style="text-align:left;">
Gm12992
</td>
<td style="text-align:left;">
NCBI_Gene:545681,ENSEMBL:ENSMUSG00000085667
</td>
<td style="text-align:left;">
MGI:3702644
</td>
<td style="text-align:left;">
lncRNA gene
</td>
<td style="text-align:left;">
predicted gene 12992
</td>
</tr>
<tr>
<td style="text-align:left;">
4
</td>
<td style="text-align:left;">
pseudogene
</td>
<td style="text-align:right;">
131.9027
</td>
<td style="text-align:right;">
131.9033
</td>
<td style="text-align:left;">
negative
</td>
<td style="text-align:left;">
MGI_C57BL6J_3705382
</td>
<td style="text-align:left;">
Gm12978
</td>
<td style="text-align:left;">
NCBI_Gene:100418326,ENSEMBL:ENSMUSG00000083013
</td>
<td style="text-align:left;">
MGI:3705382
</td>
<td style="text-align:left;">
pseudogene
</td>
<td style="text-align:left;">
predicted gene 12978
</td>
</tr>
<tr>
<td style="text-align:left;">
4
</td>
<td style="text-align:left;">
gene
</td>
<td style="text-align:right;">
131.9200
</td>
<td style="text-align:right;">
131.9240
</td>
<td style="text-align:left;">
positive
</td>
<td style="text-align:left;">
MGI_C57BL6J_3646343
</td>
<td style="text-align:left;">
Tmem200b
</td>
<td style="text-align:left;">
NCBI_Gene:623230,ENSEMBL:ENSMUSG00000070720
</td>
<td style="text-align:left;">
MGI:3646343
</td>
<td style="text-align:left;">
protein coding gene
</td>
<td style="text-align:left;">
transmembrane protein 200B
</td>
</tr>
<tr>
<td style="text-align:left;">
4
</td>
<td style="text-align:left;">
gene
</td>
<td style="text-align:right;">
131.9208
</td>
<td style="text-align:right;">
131.9215
</td>
<td style="text-align:left;">
negative
</td>
<td style="text-align:left;">
MGI_C57BL6J_6324745
</td>
<td style="text-align:left;">
Gm50475
</td>
<td style="text-align:left;">
ENSEMBL:ENSMUSG00000118394
</td>
<td style="text-align:left;">
MGI:6324745
</td>
<td style="text-align:left;">
lncRNA gene
</td>
<td style="text-align:left;">
predicted gene, 50475
</td>
</tr>
<tr>
<td style="text-align:left;">
4
</td>
<td style="text-align:left;">
gene
</td>
<td style="text-align:right;">
131.9234
</td>
<td style="text-align:right;">
132.0756
</td>
<td style="text-align:left;">
negative
</td>
<td style="text-align:left;">
MGI_C57BL6J_95401
</td>
<td style="text-align:left;">
Epb41
</td>
<td style="text-align:left;">
NCBI_Gene:269587,ENSEMBL:ENSMUSG00000028906
</td>
<td style="text-align:left;">
MGI:95401
</td>
<td style="text-align:left;">
protein coding gene
</td>
<td style="text-align:left;">
erythrocyte membrane protein band 4.1
</td>
</tr>
<tr>
<td style="text-align:left;">
4
</td>
<td style="text-align:left;">
gene
</td>
<td style="text-align:right;">
131.9922
</td>
<td style="text-align:right;">
131.9923
</td>
<td style="text-align:left;">
negative
</td>
<td style="text-align:left;">
MGI_C57BL6J_5530827
</td>
<td style="text-align:left;">
Gm27445
</td>
<td style="text-align:left;">
ENSEMBL:ENSMUSG00000098577
</td>
<td style="text-align:left;">
MGI:5530827
</td>
<td style="text-align:left;">
miRNA gene
</td>
<td style="text-align:left;">
predicted gene, 27445
</td>
</tr>
<tr>
<td style="text-align:left;">
4
</td>
<td style="text-align:left;">
gene
</td>
<td style="text-align:right;">
132.0482
</td>
<td style="text-align:right;">
132.0490
</td>
<td style="text-align:left;">
positive
</td>
<td style="text-align:left;">
MGI_C57BL6J_3649474
</td>
<td style="text-align:left;">
Gm13063
</td>
<td style="text-align:left;">
ENSEMBL:ENSMUSG00000084869
</td>
<td style="text-align:left;">
MGI:3649474
</td>
<td style="text-align:left;">
lncRNA gene
</td>
<td style="text-align:left;">
predicted gene 13063
</td>
</tr>
<tr>
<td style="text-align:left;">
4
</td>
<td style="text-align:left;">
gene
</td>
<td style="text-align:right;">
132.0495
</td>
<td style="text-align:right;">
132.0498
</td>
<td style="text-align:left;">
positive
</td>
<td style="text-align:left;">
MGI_C57BL6J_5595316
</td>
<td style="text-align:left;">
Gm36157
</td>
<td style="text-align:left;">
NCBI_Gene:102639968
</td>
<td style="text-align:left;">
MGI:5595316
</td>
<td style="text-align:left;">
lncRNA gene
</td>
<td style="text-align:left;">
predicted gene, 36157
</td>
</tr>
<tr>
<td style="text-align:left;">
4
</td>
<td style="text-align:left;">
gene
</td>
<td style="text-align:right;">
132.0747
</td>
<td style="text-align:right;">
132.0770
</td>
<td style="text-align:left;">
positive
</td>
<td style="text-align:left;">
MGI_C57BL6J_3641670
</td>
<td style="text-align:left;">
Gm10300
</td>
<td style="text-align:left;">
ENSEMBL:ENSMUSG00000070717
</td>
<td style="text-align:left;">
MGI:3641670
</td>
<td style="text-align:left;">
protein coding gene
</td>
<td style="text-align:left;">
predicted gene 10300
</td>
</tr>
<tr>
<td style="text-align:left;">
4
</td>
<td style="text-align:left;">
pseudogene
</td>
<td style="text-align:right;">
132.0939
</td>
<td style="text-align:right;">
132.0943
</td>
<td style="text-align:left;">
positive
</td>
<td style="text-align:left;">
MGI_C57BL6J_3650812
</td>
<td style="text-align:left;">
Gm13214
</td>
<td style="text-align:left;">
ENSEMBL:ENSMUSG00000083368
</td>
<td style="text-align:left;">
MGI:3650812
</td>
<td style="text-align:left;">
pseudogene
</td>
<td style="text-align:left;">
predicted gene 13214
</td>
</tr>
<tr>
<td style="text-align:left;">
4
</td>
<td style="text-align:left;">
gene
</td>
<td style="text-align:right;">
132.1048
</td>
<td style="text-align:right;">
132.1094
</td>
<td style="text-align:left;">
negative
</td>
<td style="text-align:left;">
MGI_C57BL6J_5610939
</td>
<td style="text-align:left;">
Gm37711
</td>
<td style="text-align:left;">
ENSEMBL:ENSMUSG00000102796
</td>
<td style="text-align:left;">
MGI:5610939
</td>
<td style="text-align:left;">
unclassified gene
</td>
<td style="text-align:left;">
predicted gene, 37711
</td>
</tr>
<tr>
<td style="text-align:left;">
4
</td>
<td style="text-align:left;">
gene
</td>
<td style="text-align:right;">
132.1077
</td>
<td style="text-align:right;">
132.1445
</td>
<td style="text-align:left;">
negative
</td>
<td style="text-align:left;">
MGI_C57BL6J_97438
</td>
<td style="text-align:left;">
Oprd1
</td>
<td style="text-align:left;">
NCBI_Gene:18386,ENSEMBL:ENSMUSG00000050511
</td>
<td style="text-align:left;">
MGI:97438
</td>
<td style="text-align:left;">
protein coding gene
</td>
<td style="text-align:left;">
opioid receptor, delta 1
</td>
</tr>
<tr>
<td style="text-align:left;">
4
</td>
<td style="text-align:left;">
gene
</td>
<td style="text-align:right;">
132.1253
</td>
<td style="text-align:right;">
132.1253
</td>
<td style="text-align:left;">
positive
</td>
<td style="text-align:left;">
MGI_C57BL6J_4413695
</td>
<td style="text-align:left;">
n-TAcgc3
</td>
<td style="text-align:left;">
NCBI_Gene:102467236
</td>
<td style="text-align:left;">
MGI:4413695
</td>
<td style="text-align:left;">
tRNA gene
</td>
<td style="text-align:left;">
nuclear encoded tRNA alanine 3 (anticodon CGC)
</td>
</tr>
<tr>
<td style="text-align:left;">
4
</td>
<td style="text-align:left;">
pseudogene
</td>
<td style="text-align:right;">
132.1484
</td>
<td style="text-align:right;">
132.1489
</td>
<td style="text-align:left;">
negative
</td>
<td style="text-align:left;">
MGI_C57BL6J_3650813
</td>
<td style="text-align:left;">
Gm13215
</td>
<td style="text-align:left;">
NCBI_Gene:664894,ENSEMBL:ENSMUSG00000081394
</td>
<td style="text-align:left;">
MGI:3650813
</td>
<td style="text-align:left;">
pseudogene
</td>
<td style="text-align:left;">
predicted gene 13215
</td>
</tr>
<tr>
<td style="text-align:left;">
4
</td>
<td style="text-align:left;">
gene
</td>
<td style="text-align:right;">
132.1849
</td>
<td style="text-align:right;">
132.2123
</td>
<td style="text-align:left;">
negative
</td>
<td style="text-align:left;">
MGI_C57BL6J_2444233
</td>
<td style="text-align:left;">
Ythdf2
</td>
<td style="text-align:left;">
NCBI_Gene:213541,ENSEMBL:ENSMUSG00000040025
</td>
<td style="text-align:left;">
MGI:2444233
</td>
<td style="text-align:left;">
protein coding gene
</td>
<td style="text-align:left;">
YTH N6-methyladenosine RNA binding protein 2
</td>
</tr>
<tr>
<td style="text-align:left;">
4
</td>
<td style="text-align:left;">
gene
</td>
<td style="text-align:right;">
132.2184
</td>
<td style="text-align:right;">
132.2185
</td>
<td style="text-align:left;">
negative
</td>
<td style="text-align:left;">
MGI_C57BL6J_5451810
</td>
<td style="text-align:left;">
Gm22033
</td>
<td style="text-align:left;">
ENSEMBL:ENSMUSG00000080597
</td>
<td style="text-align:left;">
MGI:5451810
</td>
<td style="text-align:left;">
miRNA gene
</td>
<td style="text-align:left;">
predicted gene, 22033
</td>
</tr>
<tr>
<td style="text-align:left;">
4
</td>
<td style="text-align:left;">
pseudogene
</td>
<td style="text-align:right;">
132.2199
</td>
<td style="text-align:right;">
132.2206
</td>
<td style="text-align:left;">
negative
</td>
<td style="text-align:left;">
MGI_C57BL6J_3652187
</td>
<td style="text-align:left;">
Rps15a-ps4
</td>
<td style="text-align:left;">
NCBI_Gene:664903,ENSEMBL:ENSMUSG00000083757
</td>
<td style="text-align:left;">
MGI:3652187
</td>
<td style="text-align:left;">
pseudogene
</td>
<td style="text-align:left;">
ribosomal protein S15A, pseudogene 4
</td>
</tr>
<tr>
<td style="text-align:left;">
4
</td>
<td style="text-align:left;">
gene
</td>
<td style="text-align:right;">
132.2210
</td>
<td style="text-align:right;">
132.2616
</td>
<td style="text-align:left;">
negative
</td>
<td style="text-align:left;">
MGI_C57BL6J_2135604
</td>
<td style="text-align:left;">
Gmeb1
</td>
<td style="text-align:left;">
NCBI_Gene:56809,ENSEMBL:ENSMUSG00000028901
</td>
<td style="text-align:left;">
MGI:2135604
</td>
<td style="text-align:left;">
protein coding gene
</td>
<td style="text-align:left;">
glucocorticoid modulatory element binding protein 1
</td>
</tr>
<tr>
<td style="text-align:left;">
4
</td>
<td style="text-align:left;">
pseudogene
</td>
<td style="text-align:right;">
132.2447
</td>
<td style="text-align:right;">
132.2460
</td>
<td style="text-align:left;">
negative
</td>
<td style="text-align:left;">
MGI_C57BL6J_3651953
</td>
<td style="text-align:left;">
Gm13252
</td>
<td style="text-align:left;">
NCBI_Gene:105247177,ENSEMBL:ENSMUSG00000084105
</td>
<td style="text-align:left;">
MGI:3651953
</td>
<td style="text-align:left;">
pseudogene
</td>
<td style="text-align:left;">
predicted gene 13252
</td>
</tr>
<tr>
<td style="text-align:left;">
4
</td>
<td style="text-align:left;">
gene
</td>
<td style="text-align:right;">
132.2605
</td>
<td style="text-align:right;">
132.2688
</td>
<td style="text-align:left;">
positive
</td>
<td style="text-align:left;">
MGI_C57BL6J_5579580
</td>
<td style="text-align:left;">
Gm28874
</td>
<td style="text-align:left;">
NCBI_Gene:102640216,ENSEMBL:ENSMUSG00000100813
</td>
<td style="text-align:left;">
MGI:5579580
</td>
<td style="text-align:left;">
lncRNA gene
</td>
<td style="text-align:left;">
predicted gene 28874
</td>
</tr>
<tr>
<td style="text-align:left;">
4
</td>
<td style="text-align:left;">
gene
</td>
<td style="text-align:right;">
132.2701
</td>
<td style="text-align:right;">
132.2702
</td>
<td style="text-align:left;">
positive
</td>
<td style="text-align:left;">
MGI_C57BL6J_5579578
</td>
<td style="text-align:left;">
Gm28872
</td>
<td style="text-align:left;">
ENSEMBL:ENSMUSG00000099937
</td>
<td style="text-align:left;">
MGI:5579578
</td>
<td style="text-align:left;">
lncRNA gene
</td>
<td style="text-align:left;">
predicted gene 28872
</td>
</tr>
<tr>
<td style="text-align:left;">
4
</td>
<td style="text-align:left;">
gene
</td>
<td style="text-align:right;">
132.2701
</td>
<td style="text-align:right;">
132.2702
</td>
<td style="text-align:left;">
negative
</td>
<td style="text-align:left;">
MGI_C57BL6J_2148804
</td>
<td style="text-align:left;">
Rnu11
</td>
<td style="text-align:left;">
NCBI_Gene:353373,ENSEMBL:ENSMUSG00000077323
</td>
<td style="text-align:left;">
MGI:2148804
</td>
<td style="text-align:left;">
snRNA gene
</td>
<td style="text-align:left;">
U11 small nuclear RNA
</td>
</tr>
<tr>
<td style="text-align:left;">
4
</td>
<td style="text-align:left;">
gene
</td>
<td style="text-align:right;">
132.2741
</td>
<td style="text-align:right;">
132.2743
</td>
<td style="text-align:left;">
negative
</td>
<td style="text-align:left;">
MGI_C57BL6J_3629726
</td>
<td style="text-align:left;">
Gt(pU21)115Imeg
</td>
<td style="text-align:left;">
NA
</td>
<td style="text-align:left;">
NA
</td>
<td style="text-align:left;">
unclassified gene
</td>
<td style="text-align:left;">
gene trap 115%2c Institute of Molecular Embryology and Genetics
</td>
</tr>
<tr>
<td style="text-align:left;">
4
</td>
<td style="text-align:left;">
gene
</td>
<td style="text-align:right;">
132.2744
</td>
<td style="text-align:right;">
132.2958
</td>
<td style="text-align:left;">
positive
</td>
<td style="text-align:left;">
MGI_C57BL6J_1913714
</td>
<td style="text-align:left;">
Taf12
</td>
<td style="text-align:left;">
NCBI_Gene:66464,ENSEMBL:ENSMUSG00000028899
</td>
<td style="text-align:left;">
MGI:1913714
</td>
<td style="text-align:left;">
protein coding gene
</td>
<td style="text-align:left;">
TATA-box binding protein associated factor 12
</td>
</tr>
<tr>
<td style="text-align:left;">
4
</td>
<td style="text-align:left;">
gene
</td>
<td style="text-align:right;">
132.3004
</td>
<td style="text-align:right;">
132.3034
</td>
<td style="text-align:left;">
negative
</td>
<td style="text-align:left;">
MGI_C57BL6J_2441753
</td>
<td style="text-align:left;">
Rab42
</td>
<td style="text-align:left;">
NCBI_Gene:242681,ENSEMBL:ENSMUSG00000089687
</td>
<td style="text-align:left;">
MGI:2441753
</td>
<td style="text-align:left;">
protein coding gene
</td>
<td style="text-align:left;">
RAB42, member RAS oncogene family
</td>
</tr>
<tr>
<td style="text-align:left;">
4
</td>
<td style="text-align:left;">
gene
</td>
<td style="text-align:right;">
132.3086
</td>
<td style="text-align:right;">
132.3110
</td>
<td style="text-align:left;">
positive
</td>
<td style="text-align:left;">
MGI_C57BL6J_1916721
</td>
<td style="text-align:left;">
Snhg12
</td>
<td style="text-align:left;">
NCBI_Gene:100039864,ENSEMBL:ENSMUSG00000086290
</td>
<td style="text-align:left;">
MGI:1916721
</td>
<td style="text-align:left;">
lncRNA gene
</td>
<td style="text-align:left;">
small nucleolar RNA host gene 12
</td>
</tr>
<tr>
<td style="text-align:left;">
4
</td>
<td style="text-align:left;">
gene
</td>
<td style="text-align:right;">
132.3095
</td>
<td style="text-align:right;">
132.3096
</td>
<td style="text-align:left;">
positive
</td>
<td style="text-align:left;">
MGI_C57BL6J_4361141
</td>
<td style="text-align:left;">
Snora16a
</td>
<td style="text-align:left;">
NCBI_Gene:100310813,ENSEMBL:ENSMUSG00000065097,ENSEMBL:ENSMUSG00000105911
</td>
<td style="text-align:left;">
MGI:4361141
</td>
<td style="text-align:left;">
snoRNA gene
</td>
<td style="text-align:left;">
small nucleolar RNA, H/ACA box 16A
</td>
</tr>
<tr>
<td style="text-align:left;">
4
</td>
<td style="text-align:left;">
gene
</td>
<td style="text-align:right;">
132.3099
</td>
<td style="text-align:right;">
132.3101
</td>
<td style="text-align:left;">
positive
</td>
<td style="text-align:left;">
MGI_C57BL6J_3819506
</td>
<td style="text-align:left;">
Snora44
</td>
<td style="text-align:left;">
NCBI_Gene:100217418,ENSEMBL:ENSMUSG00000064604
</td>
<td style="text-align:left;">
MGI:3819506
</td>
<td style="text-align:left;">
snoRNA gene
</td>
<td style="text-align:left;">
small nucleolar RNA, H/ACA box 44
</td>
</tr>
<tr>
<td style="text-align:left;">
4
</td>
<td style="text-align:left;">
gene
</td>
<td style="text-align:right;">
132.3102
</td>
<td style="text-align:right;">
132.3104
</td>
<td style="text-align:left;">
positive
</td>
<td style="text-align:left;">
MGI_C57BL6J_3819510
</td>
<td style="text-align:left;">
Snora61
</td>
<td style="text-align:left;">
NCBI_Gene:100217440,ENSEMBL:ENSMUSG00000064949
</td>
<td style="text-align:left;">
MGI:3819510
</td>
<td style="text-align:left;">
snoRNA gene
</td>
<td style="text-align:left;">
small nucleolar RNA, H/ACA box 61
</td>
</tr>
<tr>
<td style="text-align:left;">
4
</td>
<td style="text-align:left;">
gene
</td>
<td style="text-align:right;">
132.3107
</td>
<td style="text-align:right;">
132.3108
</td>
<td style="text-align:left;">
positive
</td>
<td style="text-align:left;">
MGI_C57BL6J_3819570
</td>
<td style="text-align:left;">
Snord99
</td>
<td style="text-align:left;">
NCBI_Gene:100217437,ENSEMBL:ENSMUSG00000080615
</td>
<td style="text-align:left;">
MGI:3819570
</td>
<td style="text-align:left;">
snoRNA gene
</td>
<td style="text-align:left;">
small nucleolar RNA, C/D box 99
</td>
</tr>
<tr>
<td style="text-align:left;">
4
</td>
<td style="text-align:left;">
gene
</td>
<td style="text-align:right;">
132.3118
</td>
<td style="text-align:right;">
132.3295
</td>
<td style="text-align:left;">
negative
</td>
<td style="text-align:left;">
MGI_C57BL6J_1919037
</td>
<td style="text-align:left;">
Trnau1ap
</td>
<td style="text-align:left;">
NCBI_Gene:71787,ENSEMBL:ENSMUSG00000028898
</td>
<td style="text-align:left;">
MGI:1919037
</td>
<td style="text-align:left;">
protein coding gene
</td>
<td style="text-align:left;">
tRNA selenocysteine 1 associated protein 1
</td>
</tr>
<tr>
<td style="text-align:left;">
4
</td>
<td style="text-align:left;">
gene
</td>
<td style="text-align:right;">
132.3319
</td>
<td style="text-align:right;">
132.3536
</td>
<td style="text-align:left;">
negative
</td>
<td style="text-align:left;">
MGI_C57BL6J_1913989
</td>
<td style="text-align:left;">
Rcc1
</td>
<td style="text-align:left;">
NCBI_Gene:100088,ENSEMBL:ENSMUSG00000028896
</td>
<td style="text-align:left;">
MGI:1913989
</td>
<td style="text-align:left;">
protein coding gene
</td>
<td style="text-align:left;">
regulator of chromosome condensation 1
</td>
</tr>
<tr>
<td style="text-align:left;">
4
</td>
<td style="text-align:left;">
gene
</td>
<td style="text-align:right;">
132.3480
</td>
<td style="text-align:right;">
132.3537
</td>
<td style="text-align:left;">
negative
</td>
<td style="text-align:left;">
MGI_C57BL6J_2684817
</td>
<td style="text-align:left;">
Snhg3
</td>
<td style="text-align:left;">
NCBI_Gene:399101,ENSEMBL:ENSMUSG00000085241
</td>
<td style="text-align:left;">
MGI:2684817
</td>
<td style="text-align:left;">
lncRNA gene
</td>
<td style="text-align:left;">
small nucleolar RNA host gene 3
</td>
</tr>
<tr>
<td style="text-align:left;">
4
</td>
<td style="text-align:left;">
gene
</td>
<td style="text-align:right;">
132.3518
</td>
<td style="text-align:right;">
132.3534
</td>
<td style="text-align:left;">
positive
</td>
<td style="text-align:left;">
MGI_C57BL6J_4936934
</td>
<td style="text-align:left;">
Gm17300
</td>
<td style="text-align:left;">
ENSEMBL:ENSMUSG00000091021
</td>
<td style="text-align:left;">
MGI:4936934
</td>
<td style="text-align:left;">
lncRNA gene
</td>
<td style="text-align:left;">
predicted gene, 17300
</td>
</tr>
<tr>
<td style="text-align:left;">
4
</td>
<td style="text-align:left;">
gene
</td>
<td style="text-align:right;">
132.3523
</td>
<td style="text-align:right;">
132.3525
</td>
<td style="text-align:left;">
negative
</td>
<td style="text-align:left;">
MGI_C57BL6J_4360049
</td>
<td style="text-align:left;">
Snora73b
</td>
<td style="text-align:left;">
NCBI_Gene:100306945,ENSEMBL:ENSMUSG00000065353
</td>
<td style="text-align:left;">
MGI:4360049
</td>
<td style="text-align:left;">
snoRNA gene
</td>
<td style="text-align:left;">
small nucleolar RNA, H/ACA box 73b
</td>
</tr>
<tr>
<td style="text-align:left;">
4
</td>
<td style="text-align:left;">
gene
</td>
<td style="text-align:right;">
132.3528
</td>
<td style="text-align:right;">
132.3530
</td>
<td style="text-align:left;">
negative
</td>
<td style="text-align:left;">
MGI_C57BL6J_4360046
</td>
<td style="text-align:left;">
Snora73a
</td>
<td style="text-align:left;">
NCBI_Gene:100306944,ENSEMBL:ENSMUSG00000064387
</td>
<td style="text-align:left;">
MGI:4360046
</td>
<td style="text-align:left;">
snoRNA gene
</td>
<td style="text-align:left;">
small nucleolar RNA, H/ACA box 73a
</td>
</tr>
<tr>
<td style="text-align:left;">
4
</td>
<td style="text-align:left;">
gene
</td>
<td style="text-align:right;">
132.3559
</td>
<td style="text-align:right;">
132.4225
</td>
<td style="text-align:left;">
negative
</td>
<td style="text-align:left;">
MGI_C57BL6J_2140327
</td>
<td style="text-align:left;">
Phactr4
</td>
<td style="text-align:left;">
NCBI_Gene:100169,ENSEMBL:ENSMUSG00000066043
</td>
<td style="text-align:left;">
MGI:2140327
</td>
<td style="text-align:left;">
protein coding gene
</td>
<td style="text-align:left;">
phosphatase and actin regulator 4
</td>
</tr>
<tr>
<td style="text-align:left;">
4
</td>
<td style="text-align:left;">
gene
</td>
<td style="text-align:right;">
132.4083
</td>
<td style="text-align:right;">
132.4084
</td>
<td style="text-align:left;">
positive
</td>
<td style="text-align:left;">
MGI_C57BL6J_5455484
</td>
<td style="text-align:left;">
Gm25707
</td>
<td style="text-align:left;">
ENSEMBL:ENSMUSG00000096350
</td>
<td style="text-align:left;">
MGI:5455484
</td>
<td style="text-align:left;">
miRNA gene
</td>
<td style="text-align:left;">
predicted gene, 25707
</td>
</tr>
<tr>
<td style="text-align:left;">
4
</td>
<td style="text-align:left;">
gene
</td>
<td style="text-align:right;">
132.4420
</td>
<td style="text-align:right;">
132.4421
</td>
<td style="text-align:left;">
positive
</td>
<td style="text-align:left;">
MGI_C57BL6J_5452832
</td>
<td style="text-align:left;">
Gm23055
</td>
<td style="text-align:left;">
ENSEMBL:ENSMUSG00000064739
</td>
<td style="text-align:left;">
MGI:5452832
</td>
<td style="text-align:left;">
snRNA gene
</td>
<td style="text-align:left;">
predicted gene, 23055
</td>
</tr>
<tr>
<td style="text-align:left;">
4
</td>
<td style="text-align:left;">
gene
</td>
<td style="text-align:right;">
132.4587
</td>
<td style="text-align:right;">
132.4639
</td>
<td style="text-align:left;">
negative
</td>
<td style="text-align:left;">
MGI_C57BL6J_1914469
</td>
<td style="text-align:left;">
Med18
</td>
<td style="text-align:left;">
NCBI_Gene:67219,ENSEMBL:ENSMUSG00000066042
</td>
<td style="text-align:left;">
MGI:1914469
</td>
<td style="text-align:left;">
protein coding gene
</td>
<td style="text-align:left;">
mediator complex subunit 18
</td>
</tr>
<tr>
<td style="text-align:left;">
4
</td>
<td style="text-align:left;">
gene
</td>
<td style="text-align:right;">
132.4920
</td>
<td style="text-align:right;">
132.5105
</td>
<td style="text-align:left;">
negative
</td>
<td style="text-align:left;">
MGI_C57BL6J_2651874
</td>
<td style="text-align:left;">
Sesn2
</td>
<td style="text-align:left;">
NCBI_Gene:230784,ENSEMBL:ENSMUSG00000028893
</td>
<td style="text-align:left;">
MGI:2651874
</td>
<td style="text-align:left;">
protein coding gene
</td>
<td style="text-align:left;">
sestrin 2
</td>
</tr>
<tr>
<td style="text-align:left;">
4
</td>
<td style="text-align:left;">
gene
</td>
<td style="text-align:right;">
132.5036
</td>
<td style="text-align:right;">
132.5096
</td>
<td style="text-align:left;">
negative
</td>
<td style="text-align:left;">
MGI_C57BL6J_3651528
</td>
<td style="text-align:left;">
Gm12981
</td>
<td style="text-align:left;">
ENSEMBL:ENSMUSG00000086997
</td>
<td style="text-align:left;">
MGI:3651528
</td>
<td style="text-align:left;">
lncRNA gene
</td>
<td style="text-align:left;">
predicted gene 12981
</td>
</tr>
<tr>
<td style="text-align:left;">
4
</td>
<td style="text-align:left;">
gene
</td>
<td style="text-align:right;">
132.5160
</td>
<td style="text-align:right;">
132.5161
</td>
<td style="text-align:left;">
positive
</td>
<td style="text-align:left;">
MGI_C57BL6J_4413681
</td>
<td style="text-align:left;">
n-TAagc8
</td>
<td style="text-align:left;">
NCBI_Gene:102467230
</td>
<td style="text-align:left;">
MGI:4413681
</td>
<td style="text-align:left;">
tRNA gene
</td>
<td style="text-align:left;">
nuclear encoded tRNA alanine 8 (anticodon AGC)
</td>
</tr>
<tr>
<td style="text-align:left;">
4
</td>
<td style="text-align:left;">
gene
</td>
<td style="text-align:right;">
132.5306
</td>
<td style="text-align:right;">
132.5354
</td>
<td style="text-align:left;">
negative
</td>
<td style="text-align:left;">
MGI_C57BL6J_1196457
</td>
<td style="text-align:left;">
Atpif1
</td>
<td style="text-align:left;">
NCBI_Gene:11983,ENSEMBL:ENSMUSG00000054428
</td>
<td style="text-align:left;">
MGI:1196457
</td>
<td style="text-align:left;">
protein coding gene
</td>
<td style="text-align:left;">
ATPase inhibitory factor 1
</td>
</tr>
<tr>
<td style="text-align:left;">
4
</td>
<td style="text-align:left;">
gene
</td>
<td style="text-align:right;">
132.5327
</td>
<td style="text-align:right;">
132.5336
</td>
<td style="text-align:left;">
positive
</td>
<td style="text-align:left;">
MGI_C57BL6J_3651167
</td>
<td style="text-align:left;">
Gm12999
</td>
<td style="text-align:left;">
ENSEMBL:ENSMUSG00000087352
</td>
<td style="text-align:left;">
MGI:3651167
</td>
<td style="text-align:left;">
lncRNA gene
</td>
<td style="text-align:left;">
predicted gene 12999
</td>
</tr>
<tr>
<td style="text-align:left;">
4
</td>
<td style="text-align:left;">
gene
</td>
<td style="text-align:right;">
132.5356
</td>
<td style="text-align:right;">
132.5537
</td>
<td style="text-align:left;">
positive
</td>
<td style="text-align:left;">
MGI_C57BL6J_1915848
</td>
<td style="text-align:left;">
Dnajc8
</td>
<td style="text-align:left;">
NCBI_Gene:68598,ENSEMBL:ENSMUSG00000054405
</td>
<td style="text-align:left;">
MGI:1915848
</td>
<td style="text-align:left;">
protein coding gene
</td>
<td style="text-align:left;">
DnaJ heat shock protein family (Hsp40) member C8
</td>
</tr>
<tr>
<td style="text-align:left;">
4
</td>
<td style="text-align:left;">
gene
</td>
<td style="text-align:right;">
132.5471
</td>
<td style="text-align:right;">
132.5472
</td>
<td style="text-align:left;">
negative
</td>
<td style="text-align:left;">
MGI_C57BL6J_5454539
</td>
<td style="text-align:left;">
Gm24762
</td>
<td style="text-align:left;">
ENSEMBL:ENSMUSG00000065623
</td>
<td style="text-align:left;">
MGI:5454539
</td>
<td style="text-align:left;">
snRNA gene
</td>
<td style="text-align:left;">
predicted gene, 24762
</td>
</tr>
<tr>
<td style="text-align:left;">
4
</td>
<td style="text-align:left;">
gene
</td>
<td style="text-align:right;">
132.5641
</td>
<td style="text-align:right;">
132.5827
</td>
<td style="text-align:left;">
positive
</td>
<td style="text-align:left;">
MGI_C57BL6J_106066
</td>
<td style="text-align:left;">
Ptafr
</td>
<td style="text-align:left;">
NCBI_Gene:19204,ENSEMBL:ENSMUSG00000056529
</td>
<td style="text-align:left;">
MGI:106066
</td>
<td style="text-align:left;">
protein coding gene
</td>
<td style="text-align:left;">
platelet-activating factor receptor
</td>
</tr>
<tr>
<td style="text-align:left;">
4
</td>
<td style="text-align:left;">
gene
</td>
<td style="text-align:right;">
132.6390
</td>
<td style="text-align:right;">
132.7248
</td>
<td style="text-align:left;">
positive
</td>
<td style="text-align:left;">
MGI_C57BL6J_109339
</td>
<td style="text-align:left;">
Eya3
</td>
<td style="text-align:left;">
NCBI_Gene:14050,ENSEMBL:ENSMUSG00000028886
</td>
<td style="text-align:left;">
MGI:109339
</td>
<td style="text-align:left;">
protein coding gene
</td>
<td style="text-align:left;">
EYA transcriptional coactivator and phosphatase 3
</td>
</tr>
<tr>
<td style="text-align:left;">
4
</td>
<td style="text-align:left;">
gene
</td>
<td style="text-align:right;">
132.6391
</td>
<td style="text-align:right;">
132.6403
</td>
<td style="text-align:left;">
positive
</td>
<td style="text-align:left;">
MGI_C57BL6J_1923270
</td>
<td style="text-align:left;">
5830409B07Rik
</td>
<td style="text-align:left;">
NA
</td>
<td style="text-align:left;">
NA
</td>
<td style="text-align:left;">
unclassified gene
</td>
<td style="text-align:left;">
RIKEN cDNA 5830409B07 gene
</td>
</tr>
<tr>
<td style="text-align:left;">
4
</td>
<td style="text-align:left;">
gene
</td>
<td style="text-align:right;">
132.7249
</td>
<td style="text-align:right;">
132.7326
</td>
<td style="text-align:left;">
negative
</td>
<td style="text-align:left;">
MGI_C57BL6J_2685877
</td>
<td style="text-align:left;">
Xkr8
</td>
<td style="text-align:left;">
NCBI_Gene:381560,ENSEMBL:ENSMUSG00000037752
</td>
<td style="text-align:left;">
MGI:2685877
</td>
<td style="text-align:left;">
protein coding gene
</td>
<td style="text-align:left;">
X-linked Kx blood group related 8
</td>
</tr>
<tr>
<td style="text-align:left;">
4
</td>
<td style="text-align:left;">
gene
</td>
<td style="text-align:right;">
132.7330
</td>
<td style="text-align:right;">
132.7573
</td>
<td style="text-align:left;">
negative
</td>
<td style="text-align:left;">
MGI_C57BL6J_1916022
</td>
<td style="text-align:left;">
Smpdl3b
</td>
<td style="text-align:left;">
NCBI_Gene:100340,ENSEMBL:ENSMUSG00000028885
</td>
<td style="text-align:left;">
MGI:1916022
</td>
<td style="text-align:left;">
protein coding gene
</td>
<td style="text-align:left;">
sphingomyelin phosphodiesterase, acid-like 3B
</td>
</tr>
<tr>
<td style="text-align:left;">
4
</td>
<td style="text-align:left;">
gene
</td>
<td style="text-align:right;">
132.7683
</td>
<td style="text-align:right;">
132.7788
</td>
<td style="text-align:left;">
positive
</td>
<td style="text-align:left;">
MGI_C57BL6J_1339939
</td>
<td style="text-align:left;">
Rpa2
</td>
<td style="text-align:left;">
NCBI_Gene:19891,ENSEMBL:ENSMUSG00000028884
</td>
<td style="text-align:left;">
MGI:1339939
</td>
<td style="text-align:left;">
protein coding gene
</td>
<td style="text-align:left;">
replication protein A2
</td>
</tr>
<tr>
<td style="text-align:left;">
4
</td>
<td style="text-align:left;">
gene
</td>
<td style="text-align:right;">
132.7818
</td>
<td style="text-align:right;">
132.7964
</td>
<td style="text-align:left;">
negative
</td>
<td style="text-align:left;">
MGI_C57BL6J_2446213
</td>
<td style="text-align:left;">
Themis2
</td>
<td style="text-align:left;">
NCBI_Gene:230787,ENSEMBL:ENSMUSG00000037731
</td>
<td style="text-align:left;">
MGI:2446213
</td>
<td style="text-align:left;">
protein coding gene
</td>
<td style="text-align:left;">
thymocyte selection associated family member 2
</td>
</tr>
<tr>
<td style="text-align:left;">
4
</td>
<td style="text-align:left;">
pseudogene
</td>
<td style="text-align:right;">
132.8020
</td>
<td style="text-align:right;">
132.8023
</td>
<td style="text-align:left;">
negative
</td>
<td style="text-align:left;">
MGI_C57BL6J_3651775
</td>
<td style="text-align:left;">
Gm13022
</td>
<td style="text-align:left;">
ENSEMBL:ENSMUSG00000081647
</td>
<td style="text-align:left;">
MGI:3651775
</td>
<td style="text-align:left;">
pseudogene
</td>
<td style="text-align:left;">
predicted gene 13022
</td>
</tr>
<tr>
<td style="text-align:left;">
4
</td>
<td style="text-align:left;">
gene
</td>
<td style="text-align:right;">
132.8269
</td>
<td style="text-align:right;">
132.8432
</td>
<td style="text-align:left;">
negative
</td>
<td style="text-align:left;">
MGI_C57BL6J_2140494
</td>
<td style="text-align:left;">
Ppp1r8
</td>
<td style="text-align:left;">
NCBI_Gene:100336,ENSEMBL:ENSMUSG00000028882
</td>
<td style="text-align:left;">
MGI:2140494
</td>
<td style="text-align:left;">
protein coding gene
</td>
<td style="text-align:left;">
protein phosphatase 1, regulatory subunit 8
</td>
</tr>
<tr>
<td style="text-align:left;">
4
</td>
<td style="text-align:left;">
gene
</td>
<td style="text-align:right;">
132.8384
</td>
<td style="text-align:right;">
132.8385
</td>
<td style="text-align:left;">
negative
</td>
<td style="text-align:left;">
MGI_C57BL6J_5452544
</td>
<td style="text-align:left;">
Gm22767
</td>
<td style="text-align:left;">
ENSEMBL:ENSMUSG00000088990
</td>
<td style="text-align:left;">
MGI:5452544
</td>
<td style="text-align:left;">
snoRNA gene
</td>
<td style="text-align:left;">
predicted gene, 22767
</td>
</tr>
<tr>
<td style="text-align:left;">
4
</td>
<td style="text-align:left;">
gene
</td>
<td style="text-align:right;">
132.8535
</td>
<td style="text-align:right;">
132.8845
</td>
<td style="text-align:left;">
negative
</td>
<td style="text-align:left;">
MGI_C57BL6J_1931027
</td>
<td style="text-align:left;">
Stx12
</td>
<td style="text-align:left;">
NCBI_Gene:100226,ENSEMBL:ENSMUSG00000028879
</td>
<td style="text-align:left;">
MGI:1931027
</td>
<td style="text-align:left;">
protein coding gene
</td>
<td style="text-align:left;">
syntaxin 12
</td>
</tr>
<tr>
<td style="text-align:left;">
4
</td>
<td style="text-align:left;">
gene
</td>
<td style="text-align:right;">
132.8822
</td>
<td style="text-align:right;">
132.8833
</td>
<td style="text-align:left;">
negative
</td>
<td style="text-align:left;">
MGI_C57BL6J_1924875
</td>
<td style="text-align:left;">
C530007A02Rik
</td>
<td style="text-align:left;">
NA
</td>
<td style="text-align:left;">
NA
</td>
<td style="text-align:left;">
unclassified gene
</td>
<td style="text-align:left;">
RIKEN cDNA C530007A02 gene
</td>
</tr>
<tr>
<td style="text-align:left;">
4
</td>
<td style="text-align:left;">
pseudogene
</td>
<td style="text-align:right;">
132.8848
</td>
<td style="text-align:right;">
132.8866
</td>
<td style="text-align:left;">
positive
</td>
<td style="text-align:left;">
MGI_C57BL6J_3649305
</td>
<td style="text-align:left;">
Gm13033
</td>
<td style="text-align:left;">
NCBI_Gene:100038703,ENSEMBL:ENSMUSG00000083816
</td>
<td style="text-align:left;">
MGI:3649305
</td>
<td style="text-align:left;">
pseudogene
</td>
<td style="text-align:left;">
predicted gene 13033
</td>
</tr>
<tr>
<td style="text-align:left;">
4
</td>
<td style="text-align:left;">
gene
</td>
<td style="text-align:right;">
132.8992
</td>
<td style="text-align:right;">
132.9231
</td>
<td style="text-align:left;">
negative
</td>
<td style="text-align:left;">
MGI_C57BL6J_2385211
</td>
<td style="text-align:left;">
Fam76a
</td>
<td style="text-align:left;">
NCBI_Gene:230789,ENSEMBL:ENSMUSG00000028878
</td>
<td style="text-align:left;">
MGI:2385211
</td>
<td style="text-align:left;">
protein coding gene
</td>
<td style="text-align:left;">
family with sequence similarity 76, member A
</td>
</tr>
<tr>
<td style="text-align:left;">
4
</td>
<td style="text-align:left;">
gene
</td>
<td style="text-align:right;">
132.9312
</td>
<td style="text-align:right;">
132.9313
</td>
<td style="text-align:left;">
negative
</td>
<td style="text-align:left;">
MGI_C57BL6J_5454690
</td>
<td style="text-align:left;">
Gm24913
</td>
<td style="text-align:left;">
ENSEMBL:ENSMUSG00000087911
</td>
<td style="text-align:left;">
MGI:5454690
</td>
<td style="text-align:left;">
snRNA gene
</td>
<td style="text-align:left;">
predicted gene, 24913
</td>
</tr>
<tr>
<td style="text-align:left;">
4
</td>
<td style="text-align:left;">
gene
</td>
<td style="text-align:right;">
132.9484
</td>
<td style="text-align:right;">
132.9607
</td>
<td style="text-align:left;">
negative
</td>
<td style="text-align:left;">
MGI_C57BL6J_5595677
</td>
<td style="text-align:left;">
Gm36518
</td>
<td style="text-align:left;">
NCBI_Gene:102640462
</td>
<td style="text-align:left;">
MGI:5595677
</td>
<td style="text-align:left;">
lncRNA gene
</td>
<td style="text-align:left;">
predicted gene, 36518
</td>
</tr>
<tr>
<td style="text-align:left;">
4
</td>
<td style="text-align:left;">
gene
</td>
<td style="text-align:right;">
132.9741
</td>
<td style="text-align:right;">
133.0019
</td>
<td style="text-align:left;">
positive
</td>
<td style="text-align:left;">
MGI_C57BL6J_95527
</td>
<td style="text-align:left;">
Fgr
</td>
<td style="text-align:left;">
NCBI_Gene:14191,ENSEMBL:ENSMUSG00000028874
</td>
<td style="text-align:left;">
MGI:95527
</td>
<td style="text-align:left;">
protein coding gene
</td>
<td style="text-align:left;">
FGR proto-oncogene, Src family tyrosine kinase
</td>
</tr>
<tr>
<td style="text-align:left;">
4
</td>
<td style="text-align:left;">
gene
</td>
<td style="text-align:right;">
133.0027
</td>
<td style="text-align:right;">
133.0082
</td>
<td style="text-align:left;">
negative
</td>
<td style="text-align:left;">
MGI_C57BL6J_5595758
</td>
<td style="text-align:left;">
Gm36599
</td>
<td style="text-align:left;">
NCBI_Gene:102640565
</td>
<td style="text-align:left;">
MGI:5595758
</td>
<td style="text-align:left;">
lncRNA gene
</td>
<td style="text-align:left;">
predicted gene, 36599
</td>
</tr>
<tr>
<td style="text-align:left;">
4
</td>
<td style="text-align:left;">
gene
</td>
<td style="text-align:right;">
133.0113
</td>
<td style="text-align:right;">
133.0781
</td>
<td style="text-align:left;">
positive
</td>
<td style="text-align:left;">
MGI_C57BL6J_2444218
</td>
<td style="text-align:left;">
Ahdc1
</td>
<td style="text-align:left;">
NCBI_Gene:230793,ENSEMBL:ENSMUSG00000037692
</td>
<td style="text-align:left;">
MGI:2444218
</td>
<td style="text-align:left;">
protein coding gene
</td>
<td style="text-align:left;">
AT hook, DNA binding motif, containing 1
</td>
</tr>
<tr>
<td style="text-align:left;">
4
</td>
<td style="text-align:left;">
gene
</td>
<td style="text-align:right;">
133.0834
</td>
<td style="text-align:right;">
133.0891
</td>
<td style="text-align:left;">
positive
</td>
<td style="text-align:left;">
MGI_C57BL6J_5477109
</td>
<td style="text-align:left;">
Gm26615
</td>
<td style="text-align:left;">
NCBI_Gene:102640635,ENSEMBL:ENSMUSG00000097088
</td>
<td style="text-align:left;">
MGI:5477109
</td>
<td style="text-align:left;">
lncRNA gene
</td>
<td style="text-align:left;">
predicted gene, 26615
</td>
</tr>
<tr>
<td style="text-align:left;">
4
</td>
<td style="text-align:left;">
gene
</td>
<td style="text-align:right;">
133.1145
</td>
<td style="text-align:right;">
133.1146
</td>
<td style="text-align:left;">
negative
</td>
<td style="text-align:left;">
MGI_C57BL6J_4413870
</td>
<td style="text-align:left;">
n-TGgcc1
</td>
<td style="text-align:left;">
NCBI_Gene:102467429
</td>
<td style="text-align:left;">
MGI:4413870
</td>
<td style="text-align:left;">
tRNA gene
</td>
<td style="text-align:left;">
nuclear encoded tRNA glycine 1 (anticodon GCC)
</td>
</tr>
<tr>
<td style="text-align:left;">
4
</td>
<td style="text-align:left;">
gene
</td>
<td style="text-align:right;">
133.1305
</td>
<td style="text-align:right;">
133.1998
</td>
<td style="text-align:left;">
positive
</td>
<td style="text-align:left;">
MGI_C57BL6J_1098641
</td>
<td style="text-align:left;">
Wasf2
</td>
<td style="text-align:left;">
NCBI_Gene:242687,ENSEMBL:ENSMUSG00000028868
</td>
<td style="text-align:left;">
MGI:1098641
</td>
<td style="text-align:left;">
protein coding gene
</td>
<td style="text-align:left;">
WAS protein family, member 2
</td>
</tr>
<tr>
<td style="text-align:left;">
4
</td>
<td style="text-align:left;">
gene
</td>
<td style="text-align:right;">
133.1714
</td>
<td style="text-align:right;">
133.1717
</td>
<td style="text-align:left;">
negative
</td>
<td style="text-align:left;">
MGI_C57BL6J_5454413
</td>
<td style="text-align:left;">
Gm24636
</td>
<td style="text-align:left;">
ENSEMBL:ENSMUSG00000089298
</td>
<td style="text-align:left;">
MGI:5454413
</td>
<td style="text-align:left;">
unclassified non-coding RNA gene
</td>
<td style="text-align:left;">
predicted gene, 24636
</td>
</tr>
<tr>
<td style="text-align:left;">
4
</td>
<td style="text-align:left;">
gene
</td>
<td style="text-align:right;">
133.1957
</td>
<td style="text-align:right;">
133.1957
</td>
<td style="text-align:left;">
positive
</td>
<td style="text-align:left;">
MGI_C57BL6J_5531327
</td>
<td style="text-align:left;">
Mir7017
</td>
<td style="text-align:left;">
miRBase:MI0022866,NCBI_Gene:102466215,ENSEMBL:ENSMUSG00000099202
</td>
<td style="text-align:left;">
MGI:5531327
</td>
<td style="text-align:left;">
miRNA gene
</td>
<td style="text-align:left;">
microRNA 7017
</td>
</tr>
<tr>
<td style="text-align:left;">
4
</td>
<td style="text-align:left;">
gene
</td>
<td style="text-align:right;">
133.2093
</td>
<td style="text-align:right;">
133.2125
</td>
<td style="text-align:left;">
negative
</td>
<td style="text-align:left;">
MGI_C57BL6J_101908
</td>
<td style="text-align:left;">
Gpr3
</td>
<td style="text-align:left;">
NCBI_Gene:14748,ENSEMBL:ENSMUSG00000049649
</td>
<td style="text-align:left;">
MGI:101908
</td>
<td style="text-align:left;">
protein coding gene
</td>
<td style="text-align:left;">
G-protein coupled receptor 3
</td>
</tr>
<tr>
<td style="text-align:left;">
4
</td>
<td style="text-align:left;">
gene
</td>
<td style="text-align:right;">
133.2121
</td>
<td style="text-align:right;">
133.2246
</td>
<td style="text-align:left;">
positive
</td>
<td style="text-align:left;">
MGI_C57BL6J_1916905
</td>
<td style="text-align:left;">
Cd164l2
</td>
<td style="text-align:left;">
NCBI_Gene:69655,ENSEMBL:ENSMUSG00000028865
</td>
<td style="text-align:left;">
MGI:1916905
</td>
<td style="text-align:left;">
protein coding gene
</td>
<td style="text-align:left;">
CD164 sialomucin-like 2
</td>
</tr>
<tr>
<td style="text-align:left;">
4
</td>
<td style="text-align:left;">
pseudogene
</td>
<td style="text-align:right;">
133.2315
</td>
<td style="text-align:right;">
133.2393
</td>
<td style="text-align:left;">
positive
</td>
<td style="text-align:left;">
MGI_C57BL6J_3510254
</td>
<td style="text-align:left;">
Fcnc-ps
</td>
<td style="text-align:left;">
NCBI_Gene:493918
</td>
<td style="text-align:left;">
MGI:3510254
</td>
<td style="text-align:left;">
pseudogene
</td>
<td style="text-align:left;">
ficolin C pseudogene
</td>
</tr>
<tr>
<td style="text-align:left;">
4
</td>
<td style="text-align:left;">
gene
</td>
<td style="text-align:right;">
133.2397
</td>
<td style="text-align:right;">
133.2529
</td>
<td style="text-align:left;">
positive
</td>
<td style="text-align:left;">
MGI_C57BL6J_1855691
</td>
<td style="text-align:left;">
Map3k6
</td>
<td style="text-align:left;">
NCBI_Gene:53608,ENSEMBL:ENSMUSG00000028862
</td>
<td style="text-align:left;">
MGI:1855691
</td>
<td style="text-align:left;">
protein coding gene
</td>
<td style="text-align:left;">
mitogen-activated protein kinase kinase kinase 6
</td>
</tr>
<tr>
<td style="text-align:left;">
4
</td>
<td style="text-align:left;">
gene
</td>
<td style="text-align:right;">
133.2531
</td>
<td style="text-align:right;">
133.2631
</td>
<td style="text-align:left;">
negative
</td>
<td style="text-align:left;">
MGI_C57BL6J_1933365
</td>
<td style="text-align:left;">
Sytl1
</td>
<td style="text-align:left;">
NCBI_Gene:269589,ENSEMBL:ENSMUSG00000028860
</td>
<td style="text-align:left;">
MGI:1933365
</td>
<td style="text-align:left;">
protein coding gene
</td>
<td style="text-align:left;">
synaptotagmin-like 1
</td>
</tr>
<tr>
<td style="text-align:left;">
4
</td>
<td style="text-align:left;">
gene
</td>
<td style="text-align:right;">
133.2660
</td>
<td style="text-align:right;">
133.2778
</td>
<td style="text-align:left;">
negative
</td>
<td style="text-align:left;">
MGI_C57BL6J_1098568
</td>
<td style="text-align:left;">
Tmem222
</td>
<td style="text-align:left;">
NCBI_Gene:52174,ENSEMBL:ENSMUSG00000028857
</td>
<td style="text-align:left;">
MGI:1098568
</td>
<td style="text-align:left;">
protein coding gene
</td>
<td style="text-align:left;">
transmembrane protein 222
</td>
</tr>
<tr>
<td style="text-align:left;">
4
</td>
<td style="text-align:left;">
pseudogene
</td>
<td style="text-align:right;">
133.2809
</td>
<td style="text-align:right;">
133.2813
</td>
<td style="text-align:left;">
negative
</td>
<td style="text-align:left;">
MGI_C57BL6J_3700962
</td>
<td style="text-align:left;">
Gm13259
</td>
<td style="text-align:left;">
ENSEMBL:ENSMUSG00000083241
</td>
<td style="text-align:left;">
MGI:3700962
</td>
<td style="text-align:left;">
pseudogene
</td>
<td style="text-align:left;">
predicted gene 13259
</td>
</tr>
<tr>
<td style="text-align:left;">
4
</td>
<td style="text-align:left;">
gene
</td>
<td style="text-align:right;">
133.2925
</td>
<td style="text-align:right;">
133.3535
</td>
<td style="text-align:left;">
negative
</td>
<td style="text-align:left;">
MGI_C57BL6J_2685541
</td>
<td style="text-align:left;">
Wdtc1
</td>
<td style="text-align:left;">
NCBI_Gene:230796,ENSEMBL:ENSMUSG00000037622
</td>
<td style="text-align:left;">
MGI:2685541
</td>
<td style="text-align:left;">
protein coding gene
</td>
<td style="text-align:left;">
WD and tetratricopeptide repeats 1
</td>
</tr>
<tr>
<td style="text-align:left;">
4
</td>
<td style="text-align:left;">
gene
</td>
<td style="text-align:right;">
133.2925
</td>
<td style="text-align:right;">
133.2928
</td>
<td style="text-align:left;">
positive
</td>
<td style="text-align:left;">
MGI_C57BL6J_3781803
</td>
<td style="text-align:left;">
Gm3627
</td>
<td style="text-align:left;">
NA
</td>
<td style="text-align:left;">
NA
</td>
<td style="text-align:left;">
protein coding gene
</td>
<td style="text-align:left;">
predicted gene 3627
</td>
</tr>
<tr>
<td style="text-align:left;">
4
</td>
<td style="text-align:left;">
gene
</td>
<td style="text-align:right;">
133.3697
</td>
<td style="text-align:right;">
133.4237
</td>
<td style="text-align:left;">
positive
</td>
<td style="text-align:left;">
MGI_C57BL6J_102462
</td>
<td style="text-align:left;">
Slc9a1
</td>
<td style="text-align:left;">
NCBI_Gene:20544,ENSEMBL:ENSMUSG00000028854
</td>
<td style="text-align:left;">
MGI:102462
</td>
<td style="text-align:left;">
protein coding gene
</td>
<td style="text-align:left;">
solute carrier family 9 (sodium/hydrogen exchanger), member 1
</td>
</tr>
<tr>
<td style="text-align:left;">
4
</td>
<td style="text-align:left;">
gene
</td>
<td style="text-align:right;">
133.3698
</td>
<td style="text-align:right;">
133.3699
</td>
<td style="text-align:left;">
positive
</td>
<td style="text-align:left;">
MGI_C57BL6J_4950447
</td>
<td style="text-align:left;">
Mir5122
</td>
<td style="text-align:left;">
miRBase:MI0018031,NCBI_Gene:100628620,ENSEMBL:ENSMUSG00000092745
</td>
<td style="text-align:left;">
MGI:4950447
</td>
<td style="text-align:left;">
miRNA gene
</td>
<td style="text-align:left;">
microRNA 5122
</td>
</tr>
<tr>
<td style="text-align:left;">
4
</td>
<td style="text-align:left;">
gene
</td>
<td style="text-align:right;">
133.4327
</td>
<td style="text-align:right;">
133.4349
</td>
<td style="text-align:left;">
negative
</td>
<td style="text-align:left;">
MGI_C57BL6J_1920802
</td>
<td style="text-align:left;">
1700091J24Rik
</td>
<td style="text-align:left;">
NCBI_Gene:105247178
</td>
<td style="text-align:left;">
MGI:1920802
</td>
<td style="text-align:left;">
lncRNA gene
</td>
<td style="text-align:left;">
RIKEN cDNA 1700091J24 gene
</td>
</tr>
<tr>
<td style="text-align:left;">
4
</td>
<td style="text-align:left;">
gene
</td>
<td style="text-align:right;">
133.4371
</td>
<td style="text-align:right;">
133.4404
</td>
<td style="text-align:left;">
negative
</td>
<td style="text-align:left;">
MGI_C57BL6J_5625207
</td>
<td style="text-align:left;">
Gm42322
</td>
<td style="text-align:left;">
NCBI_Gene:105247179
</td>
<td style="text-align:left;">
MGI:5625207
</td>
<td style="text-align:left;">
lncRNA gene
</td>
<td style="text-align:left;">
predicted gene, 42322
</td>
</tr>
<tr>
<td style="text-align:left;">
4
</td>
<td style="text-align:left;">
gene
</td>
<td style="text-align:right;">
133.4794
</td>
<td style="text-align:right;">
133.4852
</td>
<td style="text-align:left;">
negative
</td>
<td style="text-align:left;">
MGI_C57BL6J_3651057
</td>
<td style="text-align:left;">
Gm13257
</td>
<td style="text-align:left;">
NCBI_Gene:102640829,ENSEMBL:ENSMUSG00000086850
</td>
<td style="text-align:left;">
MGI:3651057
</td>
<td style="text-align:left;">
lncRNA gene
</td>
<td style="text-align:left;">
predicted gene 13257
</td>
</tr>
<tr>
<td style="text-align:left;">
4
</td>
<td style="text-align:left;">
gene
</td>
<td style="text-align:right;">
133.4801
</td>
<td style="text-align:right;">
133.4879
</td>
<td style="text-align:left;">
positive
</td>
<td style="text-align:left;">
MGI_C57BL6J_2140500
</td>
<td style="text-align:left;">
Tent5b
</td>
<td style="text-align:left;">
NCBI_Gene:100342,ENSEMBL:ENSMUSG00000046694
</td>
<td style="text-align:left;">
MGI:2140500
</td>
<td style="text-align:left;">
protein coding gene
</td>
<td style="text-align:left;">
terminal nucleotidyltransferase 5B
</td>
</tr>
<tr>
<td style="text-align:left;">
4
</td>
<td style="text-align:left;">
gene
</td>
<td style="text-align:right;">
133.4911
</td>
<td style="text-align:right;">
133.4992
</td>
<td style="text-align:left;">
negative
</td>
<td style="text-align:left;">
MGI_C57BL6J_1916789
</td>
<td style="text-align:left;">
Trnp1
</td>
<td style="text-align:left;">
NCBI_Gene:69539,ENSEMBL:ENSMUSG00000056596
</td>
<td style="text-align:left;">
MGI:1916789
</td>
<td style="text-align:left;">
protein coding gene
</td>
<td style="text-align:left;">
TMF1-regulated nuclear protein 1
</td>
</tr>
<tr>
<td style="text-align:left;">
4
</td>
<td style="text-align:left;">
gene
</td>
<td style="text-align:right;">
133.5129
</td>
<td style="text-align:right;">
133.5223
</td>
<td style="text-align:left;">
negative
</td>
<td style="text-align:left;">
MGI_C57BL6J_5625208
</td>
<td style="text-align:left;">
Gm42323
</td>
<td style="text-align:left;">
NCBI_Gene:105247180
</td>
<td style="text-align:left;">
MGI:5625208
</td>
<td style="text-align:left;">
protein coding gene
</td>
<td style="text-align:left;">
predicted gene, 42323
</td>
</tr>
<tr>
<td style="text-align:left;">
4
</td>
<td style="text-align:left;">
gene
</td>
<td style="text-align:right;">
133.5189
</td>
<td style="text-align:right;">
133.5308
</td>
<td style="text-align:left;">
positive
</td>
<td style="text-align:left;">
MGI_C57BL6J_1916323
</td>
<td style="text-align:left;">
Kdf1
</td>
<td style="text-align:left;">
NCBI_Gene:69073,ENSEMBL:ENSMUSG00000037600
</td>
<td style="text-align:left;">
MGI:1916323
</td>
<td style="text-align:left;">
protein coding gene
</td>
<td style="text-align:left;">
keratinocyte differentiation factor 1
</td>
</tr>
<tr>
<td style="text-align:left;">
4
</td>
<td style="text-align:left;">
gene
</td>
<td style="text-align:right;">
133.5325
</td>
<td style="text-align:right;">
133.5460
</td>
<td style="text-align:left;">
negative
</td>
<td style="text-align:left;">
MGI_C57BL6J_106014
</td>
<td style="text-align:left;">
Nudc
</td>
<td style="text-align:left;">
NCBI_Gene:18221,ENSEMBL:ENSMUSG00000028851
</td>
<td style="text-align:left;">
MGI:106014
</td>
<td style="text-align:left;">
protein coding gene
</td>
<td style="text-align:left;">
nudC nuclear distribution protein
</td>
</tr>
<tr>
<td style="text-align:left;">
4
</td>
<td style="text-align:left;">
gene
</td>
<td style="text-align:right;">
133.5534
</td>
<td style="text-align:right;">
133.5567
</td>
<td style="text-align:left;">
positive
</td>
<td style="text-align:left;">
MGI_C57BL6J_1346344
</td>
<td style="text-align:left;">
Nr0b2
</td>
<td style="text-align:left;">
NCBI_Gene:23957,ENSEMBL:ENSMUSG00000037583
</td>
<td style="text-align:left;">
MGI:1346344
</td>
<td style="text-align:left;">
protein coding gene
</td>
<td style="text-align:left;">
nuclear receptor subfamily 0, group B, member 2
</td>
</tr>
<tr>
<td style="text-align:left;">
4
</td>
<td style="text-align:left;">
gene
</td>
<td style="text-align:right;">
133.5588
</td>
<td style="text-align:right;">
133.5589
</td>
<td style="text-align:left;">
negative
</td>
<td style="text-align:left;">
MGI_C57BL6J_5452935
</td>
<td style="text-align:left;">
Gm23158
</td>
<td style="text-align:left;">
ENSEMBL:ENSMUSG00000093886
</td>
<td style="text-align:left;">
MGI:5452935
</td>
<td style="text-align:left;">
miRNA gene
</td>
<td style="text-align:left;">
predicted gene, 23158
</td>
</tr>
<tr>
<td style="text-align:left;">
4
</td>
<td style="text-align:left;">
gene
</td>
<td style="text-align:right;">
133.5747
</td>
<td style="text-align:right;">
133.5842
</td>
<td style="text-align:left;">
positive
</td>
<td style="text-align:left;">
MGI_C57BL6J_2442492
</td>
<td style="text-align:left;">
Gpatch3
</td>
<td style="text-align:left;">
NCBI_Gene:242691,ENSEMBL:ENSMUSG00000028850
</td>
<td style="text-align:left;">
MGI:2442492
</td>
<td style="text-align:left;">
protein coding gene
</td>
<td style="text-align:left;">
G patch domain containing 3
</td>
</tr>
<tr>
<td style="text-align:left;">
4
</td>
<td style="text-align:left;">
gene
</td>
<td style="text-align:right;">
133.5844
</td>
<td style="text-align:right;">
133.5917
</td>
<td style="text-align:left;">
positive
</td>
<td style="text-align:left;">
MGI_C57BL6J_2140368
</td>
<td style="text-align:left;">
Gpn2
</td>
<td style="text-align:left;">
NCBI_Gene:100210,ENSEMBL:ENSMUSG00000028848
</td>
<td style="text-align:left;">
MGI:2140368
</td>
<td style="text-align:left;">
protein coding gene
</td>
<td style="text-align:left;">
GPN-loop GTPase 2
</td>
</tr>
<tr>
<td style="text-align:left;">
4
</td>
<td style="text-align:left;">
gene
</td>
<td style="text-align:right;">
133.5949
</td>
<td style="text-align:right;">
133.5960
</td>
<td style="text-align:left;">
negative
</td>
<td style="text-align:left;">
MGI_C57BL6J_5625209
</td>
<td style="text-align:left;">
Gm42324
</td>
<td style="text-align:left;">
NCBI_Gene:105247181
</td>
<td style="text-align:left;">
MGI:5625209
</td>
<td style="text-align:left;">
lncRNA gene
</td>
<td style="text-align:left;">
predicted gene, 42324
</td>
</tr>
<tr>
<td style="text-align:left;">
4
</td>
<td style="text-align:left;">
gene
</td>
<td style="text-align:right;">
133.6006
</td>
<td style="text-align:right;">
133.6022
</td>
<td style="text-align:left;">
negative
</td>
<td style="text-align:left;">
MGI_C57BL6J_1891831
</td>
<td style="text-align:left;">
Sfn
</td>
<td style="text-align:left;">
NCBI_Gene:55948,ENSEMBL:ENSMUSG00000047281
</td>
<td style="text-align:left;">
MGI:1891831
</td>
<td style="text-align:left;">
protein coding gene
</td>
<td style="text-align:left;">
stratifin
</td>
</tr>
<tr>
<td style="text-align:left;">
4
</td>
<td style="text-align:left;">
gene
</td>
<td style="text-align:right;">
133.6047
</td>
<td style="text-align:right;">
133.6502
</td>
<td style="text-align:left;">
negative
</td>
<td style="text-align:left;">
MGI_C57BL6J_3527792
</td>
<td style="text-align:left;">
Zdhhc18
</td>
<td style="text-align:left;">
NCBI_Gene:503610,ENSEMBL:ENSMUSG00000037553
</td>
<td style="text-align:left;">
MGI:3527792
</td>
<td style="text-align:left;">
protein coding gene
</td>
<td style="text-align:left;">
zinc finger, DHHC domain containing 18
</td>
</tr>
<tr>
<td style="text-align:left;">
4
</td>
<td style="text-align:left;">
pseudogene
</td>
<td style="text-align:right;">
133.6381
</td>
<td style="text-align:right;">
133.6506
</td>
<td style="text-align:left;">
negative
</td>
<td style="text-align:left;">
MGI_C57BL6J_3651606
</td>
<td style="text-align:left;">
Gm13213
</td>
<td style="text-align:left;">
NCBI_Gene:545683,ENSEMBL:ENSMUSG00000083885
</td>
<td style="text-align:left;">
MGI:3651606
</td>
<td style="text-align:left;">
pseudogene
</td>
<td style="text-align:left;">
predicted gene 13213
</td>
</tr>
<tr>
<td style="text-align:left;">
4
</td>
<td style="text-align:left;">
gene
</td>
<td style="text-align:right;">
133.6540
</td>
<td style="text-align:right;">
133.6543
</td>
<td style="text-align:left;">
positive
</td>
<td style="text-align:left;">
MGI_C57BL6J_5452439
</td>
<td style="text-align:left;">
Gm22662
</td>
<td style="text-align:left;">
ENSEMBL:ENSMUSG00000093031
</td>
<td style="text-align:left;">
MGI:5452439
</td>
<td style="text-align:left;">
unclassified non-coding RNA gene
</td>
<td style="text-align:left;">
predicted gene, 22662
</td>
</tr>
<tr>
<td style="text-align:left;">
4
</td>
<td style="text-align:left;">
gene
</td>
<td style="text-align:right;">
133.6599
</td>
<td style="text-align:right;">
133.6732
</td>
<td style="text-align:left;">
negative
</td>
<td style="text-align:left;">
MGI_C57BL6J_2442480
</td>
<td style="text-align:left;">
Pigv
</td>
<td style="text-align:left;">
NCBI_Gene:230801,ENSEMBL:ENSMUSG00000043257
</td>
<td style="text-align:left;">
MGI:2442480
</td>
<td style="text-align:left;">
protein coding gene
</td>
<td style="text-align:left;">
phosphatidylinositol glycan anchor biosynthesis, class V
</td>
</tr>
<tr>
<td style="text-align:left;">
4
</td>
<td style="text-align:left;">
gene
</td>
<td style="text-align:right;">
133.6727
</td>
<td style="text-align:right;">
133.6732
</td>
<td style="text-align:left;">
positive
</td>
<td style="text-align:left;">
MGI_C57BL6J_1920565
</td>
<td style="text-align:left;">
1700041L08Rik
</td>
<td style="text-align:left;">
NA
</td>
<td style="text-align:left;">
NA
</td>
<td style="text-align:left;">
unclassified gene
</td>
<td style="text-align:left;">
RIKEN cDNA 1700041L08 gene
</td>
</tr>
<tr>
<td style="text-align:left;">
4
</td>
<td style="text-align:left;">
gene
</td>
<td style="text-align:right;">
133.6790
</td>
<td style="text-align:right;">
133.7568
</td>
<td style="text-align:left;">
negative
</td>
<td style="text-align:left;">
MGI_C57BL6J_1935147
</td>
<td style="text-align:left;">
Arid1a
</td>
<td style="text-align:left;">
NCBI_Gene:93760,ENSEMBL:ENSMUSG00000007880
</td>
<td style="text-align:left;">
MGI:1935147
</td>
<td style="text-align:left;">
protein coding gene
</td>
<td style="text-align:left;">
AT rich interactive domain 1A (SWI-like)
</td>
</tr>
<tr>
<td style="text-align:left;">
4
</td>
<td style="text-align:left;">
gene
</td>
<td style="text-align:right;">
133.7171
</td>
<td style="text-align:right;">
133.7171
</td>
<td style="text-align:left;">
negative
</td>
<td style="text-align:left;">
MGI_C57BL6J_5562763
</td>
<td style="text-align:left;">
Mir7227
</td>
<td style="text-align:left;">
miRBase:MI0023722,NCBI_Gene:102465707,ENSEMBL:ENSMUSG00000104566
</td>
<td style="text-align:left;">
MGI:5562763
</td>
<td style="text-align:left;">
miRNA gene
</td>
<td style="text-align:left;">
microRNA 7227
</td>
</tr>
<tr>
<td style="text-align:left;">
4
</td>
<td style="text-align:left;">
gene
</td>
<td style="text-align:right;">
133.7681
</td>
<td style="text-align:right;">
133.7754
</td>
<td style="text-align:left;">
negative
</td>
<td style="text-align:left;">
MGI_C57BL6J_5589178
</td>
<td style="text-align:left;">
Gm30019
</td>
<td style="text-align:left;">
NCBI_Gene:102631762
</td>
<td style="text-align:left;">
MGI:5589178
</td>
<td style="text-align:left;">
lncRNA gene
</td>
<td style="text-align:left;">
predicted gene, 30019
</td>
</tr>
<tr>
<td style="text-align:left;">
4
</td>
<td style="text-align:left;">
gene
</td>
<td style="text-align:right;">
133.7750
</td>
<td style="text-align:right;">
133.7813
</td>
<td style="text-align:left;">
positive
</td>
<td style="text-align:left;">
MGI_C57BL6J_5589101
</td>
<td style="text-align:left;">
Gm29942
</td>
<td style="text-align:left;">
NCBI_Gene:102631659
</td>
<td style="text-align:left;">
MGI:5589101
</td>
<td style="text-align:left;">
lncRNA gene
</td>
<td style="text-align:left;">
predicted gene, 29942
</td>
</tr>
<tr>
<td style="text-align:left;">
4
</td>
<td style="text-align:left;">
gene
</td>
<td style="text-align:right;">
133.7920
</td>
<td style="text-align:right;">
133.8215
</td>
<td style="text-align:left;">
negative
</td>
<td style="text-align:left;">
MGI_C57BL6J_5593455
</td>
<td style="text-align:left;">
Gm34296
</td>
<td style="text-align:left;">
NCBI_Gene:102637505
</td>
<td style="text-align:left;">
MGI:5593455
</td>
<td style="text-align:left;">
lncRNA gene
</td>
<td style="text-align:left;">
predicted gene, 34296
</td>
</tr>
<tr>
<td style="text-align:left;">
4
</td>
<td style="text-align:left;">
gene
</td>
<td style="text-align:right;">
133.7976
</td>
<td style="text-align:right;">
133.7977
</td>
<td style="text-align:left;">
negative
</td>
<td style="text-align:left;">
MGI_C57BL6J_5455047
</td>
<td style="text-align:left;">
Gm25270
</td>
<td style="text-align:left;">
ENSEMBL:ENSMUSG00000084555
</td>
<td style="text-align:left;">
MGI:5455047
</td>
<td style="text-align:left;">
snRNA gene
</td>
<td style="text-align:left;">
predicted gene, 25270
</td>
</tr>
<tr>
<td style="text-align:left;">
4
</td>
<td style="text-align:left;">
gene
</td>
<td style="text-align:right;">
133.8152
</td>
<td style="text-align:right;">
133.8216
</td>
<td style="text-align:left;">
positive
</td>
<td style="text-align:left;">
MGI_C57BL6J_3650037
</td>
<td style="text-align:left;">
Gm12974
</td>
<td style="text-align:left;">
NCBI_Gene:105247182,ENSEMBL:ENSMUSG00000084789
</td>
<td style="text-align:left;">
MGI:3650037
</td>
<td style="text-align:left;">
lncRNA gene
</td>
<td style="text-align:left;">
predicted gene 12974
</td>
</tr>
<tr>
<td style="text-align:left;">
4
</td>
<td style="text-align:left;">
gene
</td>
<td style="text-align:right;">
133.8473
</td>
<td style="text-align:right;">
133.8878
</td>
<td style="text-align:left;">
negative
</td>
<td style="text-align:left;">
MGI_C57BL6J_104558
</td>
<td style="text-align:left;">
Rps6ka1
</td>
<td style="text-align:left;">
NCBI_Gene:20111,ENSEMBL:ENSMUSG00000003644
</td>
<td style="text-align:left;">
MGI:104558
</td>
<td style="text-align:left;">
protein coding gene
</td>
<td style="text-align:left;">
ribosomal protein S6 kinase polypeptide 1
</td>
</tr>
<tr>
<td style="text-align:left;">
4
</td>
<td style="text-align:left;">
gene
</td>
<td style="text-align:right;">
133.8716
</td>
<td style="text-align:right;">
133.8772
</td>
<td style="text-align:left;">
positive
</td>
<td style="text-align:left;">
MGI_C57BL6J_3650699
</td>
<td style="text-align:left;">
Gm12977
</td>
<td style="text-align:left;">
NCBI_Gene:102631939,ENSEMBL:ENSMUSG00000085009
</td>
<td style="text-align:left;">
MGI:3650699
</td>
<td style="text-align:left;">
lncRNA gene
</td>
<td style="text-align:left;">
predicted gene 12977
</td>
</tr>
<tr>
<td style="text-align:left;">
4
</td>
<td style="text-align:left;">
pseudogene
</td>
<td style="text-align:right;">
133.8829
</td>
<td style="text-align:right;">
133.8831
</td>
<td style="text-align:left;">
negative
</td>
<td style="text-align:left;">
MGI_C57BL6J_3651383
</td>
<td style="text-align:left;">
Gm12985
</td>
<td style="text-align:left;">
ENSEMBL:ENSMUSG00000080946
</td>
<td style="text-align:left;">
MGI:3651383
</td>
<td style="text-align:left;">
pseudogene
</td>
<td style="text-align:left;">
predicted gene 12985
</td>
</tr>
<tr>
<td style="text-align:left;">
4
</td>
<td style="text-align:left;">
gene
</td>
<td style="text-align:right;">
133.8834
</td>
<td style="text-align:right;">
133.8835
</td>
<td style="text-align:left;">
negative
</td>
<td style="text-align:left;">
MGI_C57BL6J_5452300
</td>
<td style="text-align:left;">
Gm22523
</td>
<td style="text-align:left;">
ENSEMBL:ENSMUSG00000089321
</td>
<td style="text-align:left;">
MGI:5452300
</td>
<td style="text-align:left;">
snRNA gene
</td>
<td style="text-align:left;">
predicted gene, 22523
</td>
</tr>
<tr>
<td style="text-align:left;">
4
</td>
<td style="text-align:left;">
gene
</td>
<td style="text-align:right;">
133.9110
</td>
<td style="text-align:right;">
133.9114
</td>
<td style="text-align:left;">
negative
</td>
<td style="text-align:left;">
MGI_C57BL6J_1921148
</td>
<td style="text-align:left;">
4930429E23Rik
</td>
<td style="text-align:left;">
NA
</td>
<td style="text-align:left;">
NA
</td>
<td style="text-align:left;">
unclassified gene
</td>
<td style="text-align:left;">
RIKEN cDNA 4930429E23 gene
</td>
</tr>
<tr>
<td style="text-align:left;">
4
</td>
<td style="text-align:left;">
gene
</td>
<td style="text-align:right;">
133.9647
</td>
<td style="text-align:right;">
133.9686
</td>
<td style="text-align:left;">
negative
</td>
<td style="text-align:left;">
MGI_C57BL6J_96136
</td>
<td style="text-align:left;">
Hmgn2
</td>
<td style="text-align:left;">
NCBI_Gene:15331,ENSEMBL:ENSMUSG00000003038
</td>
<td style="text-align:left;">
MGI:96136
</td>
<td style="text-align:left;">
protein coding gene
</td>
<td style="text-align:left;">
high mobility group nucleosomal binding domain 2
</td>
</tr>
<tr>
<td style="text-align:left;">
4
</td>
<td style="text-align:left;">
gene
</td>
<td style="text-align:right;">
133.9690
</td>
<td style="text-align:right;">
134.0009
</td>
<td style="text-align:left;">
negative
</td>
<td style="text-align:left;">
MGI_C57BL6J_1914672
</td>
<td style="text-align:left;">
Dhdds
</td>
<td style="text-align:left;">
NCBI_Gene:67422,ENSEMBL:ENSMUSG00000012117
</td>
<td style="text-align:left;">
MGI:1914672
</td>
<td style="text-align:left;">
protein coding gene
</td>
<td style="text-align:left;">
dehydrodolichyl diphosphate synthase
</td>
</tr>
<tr>
<td style="text-align:left;">
4
</td>
<td style="text-align:left;">
gene
</td>
<td style="text-align:right;">
134.0033
</td>
<td style="text-align:right;">
134.0192
</td>
<td style="text-align:left;">
negative
</td>
<td style="text-align:left;">
MGI_C57BL6J_1890546
</td>
<td style="text-align:left;">
Lin28a
</td>
<td style="text-align:left;">
NCBI_Gene:83557,ENSEMBL:ENSMUSG00000050966
</td>
<td style="text-align:left;">
MGI:1890546
</td>
<td style="text-align:left;">
protein coding gene
</td>
<td style="text-align:left;">
lin-28 homolog A (C. elegans)
</td>
</tr>
<tr>
<td style="text-align:left;">
4
</td>
<td style="text-align:left;">
gene
</td>
<td style="text-align:right;">
134.0034
</td>
<td style="text-align:right;">
134.0049
</td>
<td style="text-align:left;">
positive
</td>
<td style="text-align:left;">
MGI_C57BL6J_3710625
</td>
<td style="text-align:left;">
Gm10299
</td>
<td style="text-align:left;">
NA
</td>
<td style="text-align:left;">
NA
</td>
<td style="text-align:left;">
unclassified gene
</td>
<td style="text-align:left;">
predicted gene 10299
</td>
</tr>
<tr>
<td style="text-align:left;">
4
</td>
<td style="text-align:left;">
gene
</td>
<td style="text-align:right;">
134.0181
</td>
<td style="text-align:right;">
134.0199
</td>
<td style="text-align:left;">
positive
</td>
<td style="text-align:left;">
MGI_C57BL6J_3650259
</td>
<td style="text-align:left;">
Gm13061
</td>
<td style="text-align:left;">
ENSEMBL:ENSMUSG00000087288
</td>
<td style="text-align:left;">
MGI:3650259
</td>
<td style="text-align:left;">
lncRNA gene
</td>
<td style="text-align:left;">
predicted gene 13061
</td>
</tr>
<tr>
<td style="text-align:left;">
4
</td>
<td style="text-align:left;">
gene
</td>
<td style="text-align:right;">
134.0512
</td>
<td style="text-align:right;">
134.0590
</td>
<td style="text-align:left;">
positive
</td>
<td style="text-align:left;">
MGI_C57BL6J_3650254
</td>
<td style="text-align:left;">
Zfp683
</td>
<td style="text-align:left;">
NCBI_Gene:100503878,ENSEMBL:ENSMUSG00000049410
</td>
<td style="text-align:left;">
MGI:3650254
</td>
<td style="text-align:left;">
protein coding gene
</td>
<td style="text-align:left;">
zinc finger protein 683
</td>
</tr>
<tr>
<td style="text-align:left;">
4
</td>
<td style="text-align:left;">
gene
</td>
<td style="text-align:right;">
134.0607
</td>
<td style="text-align:right;">
134.0925
</td>
<td style="text-align:left;">
positive
</td>
<td style="text-align:left;">
MGI_C57BL6J_1334463
</td>
<td style="text-align:left;">
Crybg2
</td>
<td style="text-align:left;">
NCBI_Gene:230806,ENSEMBL:ENSMUSG00000012123
</td>
<td style="text-align:left;">
MGI:1334463
</td>
<td style="text-align:left;">
protein coding gene
</td>
<td style="text-align:left;">
crystallin beta-gamma domain containing 2
</td>
</tr>
<tr>
<td style="text-align:left;">
4
</td>
<td style="text-align:left;">
gene
</td>
<td style="text-align:right;">
134.0824
</td>
<td style="text-align:right;">
134.0951
</td>
<td style="text-align:left;">
negative
</td>
<td style="text-align:left;">
MGI_C57BL6J_1346088
</td>
<td style="text-align:left;">
Cd52
</td>
<td style="text-align:left;">
NCBI_Gene:23833,ENSEMBL:ENSMUSG00000000682
</td>
<td style="text-align:left;">
MGI:1346088
</td>
<td style="text-align:left;">
protein coding gene
</td>
<td style="text-align:left;">
CD52 antigen
</td>
</tr>
<tr>
<td style="text-align:left;">
4
</td>
<td style="text-align:left;">
gene
</td>
<td style="text-align:right;">
134.1025
</td>
<td style="text-align:right;">
134.1276
</td>
<td style="text-align:left;">
positive
</td>
<td style="text-align:left;">
MGI_C57BL6J_1914836
</td>
<td style="text-align:left;">
Ubxn11
</td>
<td style="text-align:left;">
NCBI_Gene:67586,ENSEMBL:ENSMUSG00000012126
</td>
<td style="text-align:left;">
MGI:1914836
</td>
<td style="text-align:left;">
protein coding gene
</td>
<td style="text-align:left;">
UBX domain protein 11
</td>
</tr>
<tr>
<td style="text-align:left;">
4
</td>
<td style="text-align:left;">
gene
</td>
<td style="text-align:right;">
134.1274
</td>
<td style="text-align:right;">
134.1288
</td>
<td style="text-align:left;">
negative
</td>
<td style="text-align:left;">
MGI_C57BL6J_1920973
</td>
<td style="text-align:left;">
Sh3bgrl3
</td>
<td style="text-align:left;">
NCBI_Gene:73723,ENSEMBL:ENSMUSG00000028843
</td>
<td style="text-align:left;">
MGI:1920973
</td>
<td style="text-align:left;">
protein coding gene
</td>
<td style="text-align:left;">
SH3 domain binding glutamic acid-rich protein-like 3
</td>
</tr>
<tr>
<td style="text-align:left;">
4
</td>
<td style="text-align:left;">
gene
</td>
<td style="text-align:right;">
134.1299
</td>
<td style="text-align:right;">
134.1871
</td>
<td style="text-align:left;">
negative
</td>
<td style="text-align:left;">
MGI_C57BL6J_1917262
</td>
<td style="text-align:left;">
Cep85
</td>
<td style="text-align:left;">
NCBI_Gene:70012,ENSEMBL:ENSMUSG00000037443
</td>
<td style="text-align:left;">
MGI:1917262
</td>
<td style="text-align:left;">
protein coding gene
</td>
<td style="text-align:left;">
centrosomal protein 85
</td>
</tr>
<tr>
<td style="text-align:left;">
4
</td>
<td style="text-align:left;">
gene
</td>
<td style="text-align:right;">
134.1600
</td>
<td style="text-align:right;">
134.1641
</td>
<td style="text-align:left;">
positive
</td>
<td style="text-align:left;">
MGI_C57BL6J_5826531
</td>
<td style="text-align:left;">
Gm46894
</td>
<td style="text-align:left;">
NCBI_Gene:108169013
</td>
<td style="text-align:left;">
MGI:5826531
</td>
<td style="text-align:left;">
protein coding gene
</td>
<td style="text-align:left;">
predicted gene, 46894
</td>
</tr>
<tr>
<td style="text-align:left;">
4
</td>
<td style="text-align:left;">
gene
</td>
<td style="text-align:right;">
134.1678
</td>
<td style="text-align:right;">
134.1679
</td>
<td style="text-align:left;">
negative
</td>
<td style="text-align:left;">
MGI_C57BL6J_5531001
</td>
<td style="text-align:left;">
Gm27619
</td>
<td style="text-align:left;">
ENSEMBL:ENSMUSG00000098693
</td>
<td style="text-align:left;">
MGI:5531001
</td>
<td style="text-align:left;">
snoRNA gene
</td>
<td style="text-align:left;">
predicted gene, 27619
</td>
</tr>
<tr>
<td style="text-align:left;">
4
</td>
<td style="text-align:left;">
gene
</td>
<td style="text-align:right;">
134.1908
</td>
<td style="text-align:right;">
134.2030
</td>
<td style="text-align:left;">
negative
</td>
<td style="text-align:left;">
MGI_C57BL6J_3702974
</td>
<td style="text-align:left;">
Gm7534
</td>
<td style="text-align:left;">
NCBI_Gene:665186,ENSEMBL:ENSMUSG00000073747
</td>
<td style="text-align:left;">
MGI:3702974
</td>
<td style="text-align:left;">
protein coding gene
</td>
<td style="text-align:left;">
predicted gene 7534
</td>
</tr>
<tr>
<td style="text-align:left;">
4
</td>
<td style="text-align:left;">
pseudogene
</td>
<td style="text-align:right;">
134.1998
</td>
<td style="text-align:right;">
134.2002
</td>
<td style="text-align:left;">
negative
</td>
<td style="text-align:left;">
MGI_C57BL6J_3651062
</td>
<td style="text-align:left;">
Gm13131
</td>
<td style="text-align:left;">
ENSEMBL:ENSMUSG00000081772
</td>
<td style="text-align:left;">
MGI:3651062
</td>
<td style="text-align:left;">
pseudogene
</td>
<td style="text-align:left;">
predicted gene 13131
</td>
</tr>
<tr>
<td style="text-align:left;">
4
</td>
<td style="text-align:left;">
gene
</td>
<td style="text-align:right;">
134.2120
</td>
<td style="text-align:right;">
134.2274
</td>
<td style="text-align:left;">
negative
</td>
<td style="text-align:left;">
MGI_C57BL6J_3043288
</td>
<td style="text-align:left;">
Catsper4
</td>
<td style="text-align:left;">
NCBI_Gene:329954,ENSEMBL:ENSMUSG00000048003
</td>
<td style="text-align:left;">
MGI:3043288
</td>
<td style="text-align:left;">
protein coding gene
</td>
<td style="text-align:left;">
cation channel, sperm associated 4
</td>
</tr>
<tr>
<td style="text-align:left;">
4
</td>
<td style="text-align:left;">
gene
</td>
<td style="text-align:right;">
134.2280
</td>
<td style="text-align:right;">
134.2384
</td>
<td style="text-align:left;">
negative
</td>
<td style="text-align:left;">
MGI_C57BL6J_2670958
</td>
<td style="text-align:left;">
Cnksr1
</td>
<td style="text-align:left;">
NCBI_Gene:194231,ENSEMBL:ENSMUSG00000028841
</td>
<td style="text-align:left;">
MGI:2670958
</td>
<td style="text-align:left;">
protein coding gene
</td>
<td style="text-align:left;">
connector enhancer of kinase suppressor of Ras 1
</td>
</tr>
<tr>
<td style="text-align:left;">
4
</td>
<td style="text-align:left;">
gene
</td>
<td style="text-align:right;">
134.2433
</td>
<td style="text-align:right;">
134.2456
</td>
<td style="text-align:left;">
negative
</td>
<td style="text-align:left;">
MGI_C57BL6J_1915290
</td>
<td style="text-align:left;">
Zfp593
</td>
<td style="text-align:left;">
NCBI_Gene:68040,ENSEMBL:ENSMUSG00000028840
</td>
<td style="text-align:left;">
MGI:1915290
</td>
<td style="text-align:left;">
protein coding gene
</td>
<td style="text-align:left;">
zinc finger protein 593
</td>
</tr>
<tr>
<td style="text-align:left;">
4
</td>
<td style="text-align:left;">
gene
</td>
<td style="text-align:right;">
134.2437
</td>
<td style="text-align:right;">
134.2459
</td>
<td style="text-align:left;">
positive
</td>
<td style="text-align:left;">
MGI_C57BL6J_3528958
</td>
<td style="text-align:left;">
E130218I03Rik
</td>
<td style="text-align:left;">
NCBI_Gene:77490,ENSEMBL:ENSMUSG00000086322
</td>
<td style="text-align:left;">
MGI:3528958
</td>
<td style="text-align:left;">
lncRNA gene
</td>
<td style="text-align:left;">
RIKEN cDNA E130218I03 gene
</td>
</tr>
<tr>
<td style="text-align:left;">
4
</td>
<td style="text-align:left;">
gene
</td>
<td style="text-align:right;">
134.2469
</td>
<td style="text-align:right;">
134.2500
</td>
<td style="text-align:left;">
positive
</td>
<td style="text-align:left;">
MGI_C57BL6J_5589350
</td>
<td style="text-align:left;">
Gm30191
</td>
<td style="text-align:left;">
ENSEMBL:ENSMUSG00000108398
</td>
<td style="text-align:left;">
MGI:5589350
</td>
<td style="text-align:left;">
protein coding gene
</td>
<td style="text-align:left;">
predicted gene, 30191
</td>
</tr>
<tr>
<td style="text-align:left;">
4
</td>
<td style="text-align:left;">
gene
</td>
<td style="text-align:right;">
134.2511
</td>
<td style="text-align:right;">
134.2662
</td>
<td style="text-align:left;">
negative
</td>
<td style="text-align:left;">
MGI_C57BL6J_1919940
</td>
<td style="text-align:left;">
Grrp1
</td>
<td style="text-align:left;">
NCBI_Gene:72690,ENSEMBL:ENSMUSG00000050105
</td>
<td style="text-align:left;">
MGI:1919940
</td>
<td style="text-align:left;">
protein coding gene
</td>
<td style="text-align:left;">
glycine/arginine rich protein 1
</td>
</tr>
<tr>
<td style="text-align:left;">
4
</td>
<td style="text-align:left;">
gene
</td>
<td style="text-align:right;">
134.2558
</td>
<td style="text-align:right;">
134.2657
</td>
<td style="text-align:left;">
positive
</td>
<td style="text-align:left;">
MGI_C57BL6J_5625210
</td>
<td style="text-align:left;">
Gm42325
</td>
<td style="text-align:left;">
NCBI_Gene:105247183
</td>
<td style="text-align:left;">
MGI:5625210
</td>
<td style="text-align:left;">
lncRNA gene
</td>
<td style="text-align:left;">
predicted gene, 42325
</td>
</tr>
<tr>
<td style="text-align:left;">
4
</td>
<td style="text-align:left;">
gene
</td>
<td style="text-align:right;">
134.2750
</td>
<td style="text-align:right;">
134.2879
</td>
<td style="text-align:left;">
negative
</td>
<td style="text-align:left;">
MGI_C57BL6J_2385213
</td>
<td style="text-align:left;">
Pdik1l
</td>
<td style="text-align:left;">
NCBI_Gene:230809,ENSEMBL:ENSMUSG00000050890
</td>
<td style="text-align:left;">
MGI:2385213
</td>
<td style="text-align:left;">
protein coding gene
</td>
<td style="text-align:left;">
PDLIM1 interacting kinase 1 like
</td>
</tr>
<tr>
<td style="text-align:left;">
4
</td>
<td style="text-align:left;">
gene
</td>
<td style="text-align:right;">
134.3151
</td>
<td style="text-align:right;">
134.3296
</td>
<td style="text-align:left;">
positive
</td>
<td style="text-align:left;">
MGI_C57BL6J_2447992
</td>
<td style="text-align:left;">
Trim63
</td>
<td style="text-align:left;">
NCBI_Gene:433766,ENSEMBL:ENSMUSG00000028834
</td>
<td style="text-align:left;">
MGI:2447992
</td>
<td style="text-align:left;">
protein coding gene
</td>
<td style="text-align:left;">
tripartite motif-containing 63
</td>
</tr>
<tr>
<td style="text-align:left;">
4
</td>
<td style="text-align:left;">
pseudogene
</td>
<td style="text-align:right;">
134.3355
</td>
<td style="text-align:right;">
134.3358
</td>
<td style="text-align:left;">
negative
</td>
<td style="text-align:left;">
MGI_C57BL6J_3652269
</td>
<td style="text-align:left;">
Gm13195
</td>
<td style="text-align:left;">
ENSEMBL:ENSMUSG00000083065
</td>
<td style="text-align:left;">
MGI:3652269
</td>
<td style="text-align:left;">
pseudogene
</td>
<td style="text-align:left;">
predicted gene 13195
</td>
</tr>
<tr>
<td style="text-align:left;">
4
</td>
<td style="text-align:left;">
gene
</td>
<td style="text-align:right;">
134.3430
</td>
<td style="text-align:right;">
134.3545
</td>
<td style="text-align:left;">
positive
</td>
<td style="text-align:left;">
MGI_C57BL6J_106637
</td>
<td style="text-align:left;">
Slc30a2
</td>
<td style="text-align:left;">
NCBI_Gene:230810,ENSEMBL:ENSMUSG00000028836
</td>
<td style="text-align:left;">
MGI:106637
</td>
<td style="text-align:left;">
protein coding gene
</td>
<td style="text-align:left;">
solute carrier family 30 (zinc transporter), member 2
</td>
</tr>
<tr>
<td style="text-align:left;">
4
</td>
<td style="text-align:left;">
gene
</td>
<td style="text-align:right;">
134.3564
</td>
<td style="text-align:right;">
134.3838
</td>
<td style="text-align:left;">
negative
</td>
<td style="text-align:left;">
MGI_C57BL6J_1888742
</td>
<td style="text-align:left;">
Extl1
</td>
<td style="text-align:left;">
NCBI_Gene:56219,ENSEMBL:ENSMUSG00000028838
</td>
<td style="text-align:left;">
MGI:1888742
</td>
<td style="text-align:left;">
protein coding gene
</td>
<td style="text-align:left;">
exostoses (multiple)-like 1
</td>
</tr>
<tr>
<td style="text-align:left;">
4
</td>
<td style="text-align:left;">
gene
</td>
<td style="text-align:right;">
134.3849
</td>
<td style="text-align:right;">
134.3870
</td>
<td style="text-align:left;">
positive
</td>
<td style="text-align:left;">
MGI_C57BL6J_3779501
</td>
<td style="text-align:left;">
Gm5589
</td>
<td style="text-align:left;">
NA
</td>
<td style="text-align:left;">
NA
</td>
<td style="text-align:left;">
unclassified gene
</td>
<td style="text-align:left;">
predicted gene 5589
</td>
</tr>
<tr>
<td style="text-align:left;">
4
</td>
<td style="text-align:left;">
gene
</td>
<td style="text-align:right;">
134.3963
</td>
<td style="text-align:right;">
134.4274
</td>
<td style="text-align:left;">
positive
</td>
<td style="text-align:left;">
MGI_C57BL6J_2140321
</td>
<td style="text-align:left;">
Pafah2
</td>
<td style="text-align:left;">
NCBI_Gene:100163,ENSEMBL:ENSMUSG00000037366
</td>
<td style="text-align:left;">
MGI:2140321
</td>
<td style="text-align:left;">
protein coding gene
</td>
<td style="text-align:left;">
platelet-activating factor acetylhydrolase 2
</td>
</tr>
<tr>
<td style="text-align:left;">
4
</td>
<td style="text-align:left;">
gene
</td>
<td style="text-align:right;">
134.4337
</td>
<td style="text-align:right;">
134.4483
</td>
<td style="text-align:left;">
negative
</td>
<td style="text-align:left;">
MGI_C57BL6J_5589612
</td>
<td style="text-align:left;">
Gm30453
</td>
<td style="text-align:left;">
NCBI_Gene:102632361
</td>
<td style="text-align:left;">
MGI:5589612
</td>
<td style="text-align:left;">
lncRNA gene
</td>
<td style="text-align:left;">
predicted gene, 30453
</td>
</tr>
<tr>
<td style="text-align:left;">
4
</td>
<td style="text-align:left;">
gene
</td>
<td style="text-align:right;">
134.4488
</td>
<td style="text-align:right;">
134.4502
</td>
<td style="text-align:left;">
negative
</td>
<td style="text-align:left;">
MGI_C57BL6J_1916659
</td>
<td style="text-align:left;">
1700021N21Rik
</td>
<td style="text-align:left;">
NCBI_Gene:69409,ENSEMBL:ENSMUSG00000087343
</td>
<td style="text-align:left;">
MGI:1916659
</td>
<td style="text-align:left;">
lncRNA gene
</td>
<td style="text-align:left;">
RIKEN cDNA 1700021N21 gene
</td>
</tr>
<tr>
<td style="text-align:left;">
4
</td>
<td style="text-align:left;">
gene
</td>
<td style="text-align:right;">
134.4683
</td>
<td style="text-align:right;">
134.4738
</td>
<td style="text-align:left;">
positive
</td>
<td style="text-align:left;">
MGI_C57BL6J_96739
</td>
<td style="text-align:left;">
Stmn1
</td>
<td style="text-align:left;">
NCBI_Gene:16765,ENSEMBL:ENSMUSG00000028832
</td>
<td style="text-align:left;">
MGI:96739
</td>
<td style="text-align:left;">
protein coding gene
</td>
<td style="text-align:left;">
stathmin 1
</td>
</tr>
<tr>
<td style="text-align:left;">
4
</td>
<td style="text-align:left;">
gene
</td>
<td style="text-align:right;">
134.4684
</td>
<td style="text-align:right;">
134.4685
</td>
<td style="text-align:left;">
positive
</td>
<td style="text-align:left;">
MGI_C57BL6J_5453411
</td>
<td style="text-align:left;">
Gm23634
</td>
<td style="text-align:left;">
ENSEMBL:ENSMUSG00000092812
</td>
<td style="text-align:left;">
MGI:5453411
</td>
<td style="text-align:left;">
miRNA gene
</td>
<td style="text-align:left;">
predicted gene, 23634
</td>
</tr>
<tr>
<td style="text-align:left;">
4
</td>
<td style="text-align:left;">
pseudogene
</td>
<td style="text-align:right;">
134.4932
</td>
<td style="text-align:right;">
134.4940
</td>
<td style="text-align:left;">
positive
</td>
<td style="text-align:left;">
MGI_C57BL6J_3650425
</td>
<td style="text-align:left;">
Gm13250
</td>
<td style="text-align:left;">
NCBI_Gene:100416260,ENSEMBL:ENSMUSG00000083001
</td>
<td style="text-align:left;">
MGI:3650425
</td>
<td style="text-align:left;">
pseudogene
</td>
<td style="text-align:left;">
predicted gene 13250
</td>
</tr>
<tr>
<td style="text-align:left;">
4
</td>
<td style="text-align:left;">
gene
</td>
<td style="text-align:right;">
134.4967
</td>
<td style="text-align:right;">
134.5102
</td>
<td style="text-align:left;">
positive
</td>
<td style="text-align:left;">
MGI_C57BL6J_1919154
</td>
<td style="text-align:left;">
Paqr7
</td>
<td style="text-align:left;">
NCBI_Gene:71904,ENSEMBL:ENSMUSG00000037348
</td>
<td style="text-align:left;">
MGI:1919154
</td>
<td style="text-align:left;">
protein coding gene
</td>
<td style="text-align:left;">
progestin and adipoQ receptor family member VII
</td>
</tr>
<tr>
<td style="text-align:left;">
4
</td>
<td style="text-align:left;">
gene
</td>
<td style="text-align:right;">
134.5110
</td>
<td style="text-align:right;">
134.5239
</td>
<td style="text-align:left;">
positive
</td>
<td style="text-align:left;">
MGI_C57BL6J_1917135
</td>
<td style="text-align:left;">
Aunip
</td>
<td style="text-align:left;">
NCBI_Gene:69885,ENSEMBL:ENSMUSG00000078521
</td>
<td style="text-align:left;">
MGI:1917135
</td>
<td style="text-align:left;">
protein coding gene
</td>
<td style="text-align:left;">
aurora kinase A and ninein interacting protein
</td>
</tr>
<tr>
<td style="text-align:left;">
4
</td>
<td style="text-align:left;">
gene
</td>
<td style="text-align:right;">
134.5256
</td>
<td style="text-align:right;">
134.5354
</td>
<td style="text-align:left;">
negative
</td>
<td style="text-align:left;">
MGI_C57BL6J_1924074
</td>
<td style="text-align:left;">
Mtfr1l
</td>
<td style="text-align:left;">
NCBI_Gene:76824,ENSEMBL:ENSMUSG00000046671
</td>
<td style="text-align:left;">
MGI:1924074
</td>
<td style="text-align:left;">
protein coding gene
</td>
<td style="text-align:left;">
mitochondrial fission regulator 1-like
</td>
</tr>
<tr>
<td style="text-align:left;">
4
</td>
<td style="text-align:left;">
gene
</td>
<td style="text-align:right;">
134.5379
</td>
<td style="text-align:right;">
134.5522
</td>
<td style="text-align:left;">
negative
</td>
<td style="text-align:left;">
MGI_C57BL6J_2151208
</td>
<td style="text-align:left;">
Selenon
</td>
<td style="text-align:left;">
NCBI_Gene:74777,ENSEMBL:ENSMUSG00000050989
</td>
<td style="text-align:left;">
MGI:2151208
</td>
<td style="text-align:left;">
protein coding gene
</td>
<td style="text-align:left;">
selenoprotein N
</td>
</tr>
<tr>
<td style="text-align:left;">
4
</td>
<td style="text-align:left;">
gene
</td>
<td style="text-align:right;">
134.5617
</td>
<td style="text-align:right;">
134.7043
</td>
<td style="text-align:left;">
negative
</td>
<td style="text-align:left;">
MGI_C57BL6J_2446214
</td>
<td style="text-align:left;">
Man1c1
</td>
<td style="text-align:left;">
NCBI_Gene:230815,ENSEMBL:ENSMUSG00000037306
</td>
<td style="text-align:left;">
MGI:2446214
</td>
<td style="text-align:left;">
protein coding gene
</td>
<td style="text-align:left;">
mannosidase, alpha, class 1C, member 1
</td>
</tr>
<tr>
<td style="text-align:left;">
4
</td>
<td style="text-align:left;">
gene
</td>
<td style="text-align:right;">
134.5677
</td>
<td style="text-align:right;">
134.5679
</td>
<td style="text-align:left;">
negative
</td>
<td style="text-align:left;">
MGI_C57BL6J_5531374
</td>
<td style="text-align:left;">
Mir6403
</td>
<td style="text-align:left;">
miRBase:MI0021939,NCBI_Gene:102465216,ENSEMBL:ENSMUSG00000099289
</td>
<td style="text-align:left;">
MGI:5531374
</td>
<td style="text-align:left;">
miRNA gene
</td>
<td style="text-align:left;">
microRNA 6403
</td>
</tr>
<tr>
<td style="text-align:left;">
4
</td>
<td style="text-align:left;">
gene
</td>
<td style="text-align:right;">
134.5914
</td>
<td style="text-align:right;">
134.5950
</td>
<td style="text-align:left;">
positive
</td>
<td style="text-align:left;">
MGI_C57BL6J_5625211
</td>
<td style="text-align:left;">
Gm42326
</td>
<td style="text-align:left;">
NCBI_Gene:105247184
</td>
<td style="text-align:left;">
MGI:5625211
</td>
<td style="text-align:left;">
lncRNA gene
</td>
<td style="text-align:left;">
predicted gene, 42326
</td>
</tr>
<tr>
<td style="text-align:left;">
4
</td>
<td style="text-align:left;">
gene
</td>
<td style="text-align:right;">
134.7416
</td>
<td style="text-align:right;">
134.7680
</td>
<td style="text-align:left;">
negative
</td>
<td style="text-align:left;">
MGI_C57BL6J_2140175
</td>
<td style="text-align:left;">
Ldlrap1
</td>
<td style="text-align:left;">
NCBI_Gene:100017,ENSEMBL:ENSMUSG00000037295
</td>
<td style="text-align:left;">
MGI:2140175
</td>
<td style="text-align:left;">
protein coding gene
</td>
<td style="text-align:left;">
low density lipoprotein receptor adaptor protein 1
</td>
</tr>
<tr>
<td style="text-align:left;">
4
</td>
<td style="text-align:left;">
gene
</td>
<td style="text-align:right;">
134.8028
</td>
<td style="text-align:right;">
134.8536
</td>
<td style="text-align:left;">
negative
</td>
<td style="text-align:left;">
MGI_C57BL6J_1913396
</td>
<td style="text-align:left;">
Maco1
</td>
<td style="text-align:left;">
NCBI_Gene:66146,ENSEMBL:ENSMUSG00000028826
</td>
<td style="text-align:left;">
MGI:1913396
</td>
<td style="text-align:left;">
protein coding gene
</td>
<td style="text-align:left;">
macoilin 1
</td>
</tr>
<tr>
<td style="text-align:left;">
4
</td>
<td style="text-align:left;">
gene
</td>
<td style="text-align:right;">
134.8645
</td>
<td style="text-align:right;">
134.8962
</td>
<td style="text-align:left;">
positive
</td>
<td style="text-align:left;">
MGI_C57BL6J_1202882
</td>
<td style="text-align:left;">
Rhd
</td>
<td style="text-align:left;">
NCBI_Gene:19746,ENSEMBL:ENSMUSG00000028825
</td>
<td style="text-align:left;">
MGI:1202882
</td>
<td style="text-align:left;">
protein coding gene
</td>
<td style="text-align:left;">
Rh blood group, D antigen
</td>
</tr>
<tr>
<td style="text-align:left;">
4
</td>
<td style="text-align:left;">
gene
</td>
<td style="text-align:right;">
134.8978
</td>
<td style="text-align:right;">
134.9150
</td>
<td style="text-align:left;">
negative
</td>
<td style="text-align:left;">
MGI_C57BL6J_1919067
</td>
<td style="text-align:left;">
Tmem50a
</td>
<td style="text-align:left;">
NCBI_Gene:71817,ENSEMBL:ENSMUSG00000028822
</td>
<td style="text-align:left;">
MGI:1919067
</td>
<td style="text-align:left;">
protein coding gene
</td>
<td style="text-align:left;">
transmembrane protein 50A
</td>
</tr>
<tr>
<td style="text-align:left;">
4
</td>
<td style="text-align:left;">
gene
</td>
<td style="text-align:right;">
134.9236
</td>
<td style="text-align:right;">
134.9277
</td>
<td style="text-align:left;">
positive
</td>
<td style="text-align:left;">
MGI_C57BL6J_106498
</td>
<td style="text-align:left;">
Rsrp1
</td>
<td style="text-align:left;">
NCBI_Gene:27981,ENSEMBL:ENSMUSG00000037266
</td>
<td style="text-align:left;">
MGI:106498
</td>
<td style="text-align:left;">
protein coding gene
</td>
<td style="text-align:left;">
arginine/serine rich protein 1
</td>
</tr>
<tr>
<td style="text-align:left;">
4
</td>
<td style="text-align:left;">
gene
</td>
<td style="text-align:right;">
134.9309
</td>
<td style="text-align:right;">
134.9375
</td>
<td style="text-align:left;">
positive
</td>
<td style="text-align:left;">
MGI_C57BL6J_1915842
</td>
<td style="text-align:left;">
Syf2
</td>
<td style="text-align:left;">
NCBI_Gene:68592,ENSEMBL:ENSMUSG00000028821
</td>
<td style="text-align:left;">
MGI:1915842
</td>
<td style="text-align:left;">
protein coding gene
</td>
<td style="text-align:left;">
SYF2 homolog, RNA splicing factor (S. cerevisiae)
</td>
</tr>
<tr>
<td style="text-align:left;">
4
</td>
<td style="text-align:left;">
gene
</td>
<td style="text-align:right;">
135.0437
</td>
<td style="text-align:right;">
135.0514
</td>
<td style="text-align:left;">
negative
</td>
<td style="text-align:left;">
MGI_C57BL6J_5625212
</td>
<td style="text-align:left;">
Gm42327
</td>
<td style="text-align:left;">
NCBI_Gene:105247185
</td>
<td style="text-align:left;">
MGI:5625212
</td>
<td style="text-align:left;">
lncRNA gene
</td>
<td style="text-align:left;">
predicted gene, 42327
</td>
</tr>
<tr>
<td style="text-align:left;">
4
</td>
<td style="text-align:left;">
gene
</td>
<td style="text-align:right;">
135.1182
</td>
<td style="text-align:right;">
135.1204
</td>
<td style="text-align:left;">
negative
</td>
<td style="text-align:left;">
MGI_C57BL6J_3801903
</td>
<td style="text-align:left;">
Gm16225
</td>
<td style="text-align:left;">
ENSEMBL:ENSMUSG00000086884
</td>
<td style="text-align:left;">
MGI:3801903
</td>
<td style="text-align:left;">
lncRNA gene
</td>
<td style="text-align:left;">
predicted gene 16225
</td>
</tr>
<tr>
<td style="text-align:left;">
4
</td>
<td style="text-align:left;">
gene
</td>
<td style="text-align:right;">
135.1206
</td>
<td style="text-align:right;">
135.1780
</td>
<td style="text-align:left;">
positive
</td>
<td style="text-align:left;">
MGI_C57BL6J_102672
</td>
<td style="text-align:left;">
Runx3
</td>
<td style="text-align:left;">
NCBI_Gene:12399,ENSEMBL:ENSMUSG00000070691
</td>
<td style="text-align:left;">
MGI:102672
</td>
<td style="text-align:left;">
protein coding gene
</td>
<td style="text-align:left;">
runt related transcription factor 3
</td>
</tr>
<tr>
<td style="text-align:left;">
4
</td>
<td style="text-align:left;">
gene
</td>
<td style="text-align:right;">
135.1434
</td>
<td style="text-align:right;">
135.1443
</td>
<td style="text-align:left;">
negative
</td>
<td style="text-align:left;">
MGI_C57BL6J_3801902
</td>
<td style="text-align:left;">
Gm16224
</td>
<td style="text-align:left;">
ENSEMBL:ENSMUSG00000084986
</td>
<td style="text-align:left;">
MGI:3801902
</td>
<td style="text-align:left;">
lncRNA gene
</td>
<td style="text-align:left;">
predicted gene 16224
</td>
</tr>
<tr>
<td style="text-align:left;">
4
</td>
<td style="text-align:left;">
gene
</td>
<td style="text-align:right;">
135.2140
</td>
<td style="text-align:right;">
135.2728
</td>
<td style="text-align:left;">
negative
</td>
<td style="text-align:left;">
MGI_C57BL6J_1352754
</td>
<td style="text-align:left;">
Clic4
</td>
<td style="text-align:left;">
NCBI_Gene:29876,ENSEMBL:ENSMUSG00000037242
</td>
<td style="text-align:left;">
MGI:1352754
</td>
<td style="text-align:left;">
protein coding gene
</td>
<td style="text-align:left;">
chloride intracellular channel 4 (mitochondrial)
</td>
</tr>
<tr>
<td style="text-align:left;">
4
</td>
<td style="text-align:left;">
gene
</td>
<td style="text-align:right;">
135.2831
</td>
<td style="text-align:right;">
135.2891
</td>
<td style="text-align:left;">
positive
</td>
<td style="text-align:left;">
MGI_C57BL6J_5589926
</td>
<td style="text-align:left;">
Gm30767
</td>
<td style="text-align:left;">
NCBI_Gene:102632783
</td>
<td style="text-align:left;">
MGI:5589926
</td>
<td style="text-align:left;">
lncRNA gene
</td>
<td style="text-align:left;">
predicted gene, 30767
</td>
</tr>
<tr>
<td style="text-align:left;">
4
</td>
<td style="text-align:left;">
pseudogene
</td>
<td style="text-align:right;">
135.2903
</td>
<td style="text-align:right;">
135.2907
</td>
<td style="text-align:left;">
negative
</td>
<td style="text-align:left;">
MGI_C57BL6J_3651378
</td>
<td style="text-align:left;">
Gm12983
</td>
<td style="text-align:left;">
NCBI_Gene:676072,ENSEMBL:ENSMUSG00000087270
</td>
<td style="text-align:left;">
MGI:3651378
</td>
<td style="text-align:left;">
pseudogene
</td>
<td style="text-align:left;">
predicted gene 12983
</td>
</tr>
<tr>
<td style="text-align:left;">
4
</td>
<td style="text-align:left;">
gene
</td>
<td style="text-align:right;">
135.2966
</td>
<td style="text-align:right;">
135.2994
</td>
<td style="text-align:left;">
negative
</td>
<td style="text-align:left;">
MGI_C57BL6J_5589860
</td>
<td style="text-align:left;">
Gm30701
</td>
<td style="text-align:left;">
NCBI_Gene:102632698
</td>
<td style="text-align:left;">
MGI:5589860
</td>
<td style="text-align:left;">
lncRNA gene
</td>
<td style="text-align:left;">
predicted gene, 30701
</td>
</tr>
<tr>
<td style="text-align:left;">
4
</td>
<td style="text-align:left;">
gene
</td>
<td style="text-align:right;">
135.3073
</td>
<td style="text-align:right;">
135.3095
</td>
<td style="text-align:left;">
positive
</td>
<td style="text-align:left;">
MGI_C57BL6J_3651384
</td>
<td style="text-align:left;">
Gm12984
</td>
<td style="text-align:left;">
ENSEMBL:ENSMUSG00000085002
</td>
<td style="text-align:left;">
MGI:3651384
</td>
<td style="text-align:left;">
lncRNA gene
</td>
<td style="text-align:left;">
predicted gene 12984
</td>
</tr>
<tr>
<td style="text-align:left;">
4
</td>
<td style="text-align:left;">
pseudogene
</td>
<td style="text-align:right;">
135.3078
</td>
<td style="text-align:right;">
135.3080
</td>
<td style="text-align:left;">
negative
</td>
<td style="text-align:left;">
MGI_C57BL6J_3651381
</td>
<td style="text-align:left;">
Gm12982
</td>
<td style="text-align:left;">
ENSEMBL:ENSMUSG00000082833
</td>
<td style="text-align:left;">
MGI:3651381
</td>
<td style="text-align:left;">
pseudogene
</td>
<td style="text-align:left;">
predicted gene 12982
</td>
</tr>
<tr>
<td style="text-align:left;">
4
</td>
<td style="text-align:left;">
gene
</td>
<td style="text-align:right;">
135.3205
</td>
<td style="text-align:right;">
135.3533
</td>
<td style="text-align:left;">
negative
</td>
<td style="text-align:left;">
MGI_C57BL6J_1858303
</td>
<td style="text-align:left;">
Srrm1
</td>
<td style="text-align:left;">
NCBI_Gene:51796,ENSEMBL:ENSMUSG00000028809
</td>
<td style="text-align:left;">
MGI:1858303
</td>
<td style="text-align:left;">
protein coding gene
</td>
<td style="text-align:left;">
serine/arginine repetitive matrix 1
</td>
</tr>
<tr>
<td style="text-align:left;">
4
</td>
<td style="text-align:left;">
gene
</td>
<td style="text-align:right;">
135.3696
</td>
<td style="text-align:right;">
135.3983
</td>
<td style="text-align:left;">
negative
</td>
<td style="text-align:left;">
MGI_C57BL6J_2444888
</td>
<td style="text-align:left;">
Ncmap
</td>
<td style="text-align:left;">
NCBI_Gene:230822,ENSEMBL:ENSMUSG00000043924
</td>
<td style="text-align:left;">
MGI:2444888
</td>
<td style="text-align:left;">
protein coding gene
</td>
<td style="text-align:left;">
noncompact myelin associated protein
</td>
</tr>
<tr>
<td style="text-align:left;">
4
</td>
<td style="text-align:left;">
pseudogene
</td>
<td style="text-align:right;">
135.3772
</td>
<td style="text-align:right;">
135.3772
</td>
<td style="text-align:left;">
positive
</td>
<td style="text-align:left;">
MGI_C57BL6J_3651750
</td>
<td style="text-align:left;">
Gm12991
</td>
<td style="text-align:left;">
ENSEMBL:ENSMUSG00000084032
</td>
<td style="text-align:left;">
MGI:3651750
</td>
<td style="text-align:left;">
pseudogene
</td>
<td style="text-align:left;">
predicted gene 12991
</td>
</tr>
<tr>
<td style="text-align:left;">
4
</td>
<td style="text-align:left;">
pseudogene
</td>
<td style="text-align:right;">
135.4032
</td>
<td style="text-align:right;">
135.4037
</td>
<td style="text-align:left;">
negative
</td>
<td style="text-align:left;">
MGI_C57BL6J_3651107
</td>
<td style="text-align:left;">
Gm12990
</td>
<td style="text-align:left;">
ENSEMBL:ENSMUSG00000081136
</td>
<td style="text-align:left;">
MGI:3651107
</td>
<td style="text-align:left;">
pseudogene
</td>
<td style="text-align:left;">
predicted gene 12990
</td>
</tr>
<tr>
<td style="text-align:left;">
4
</td>
<td style="text-align:left;">
gene
</td>
<td style="text-align:right;">
135.4123
</td>
<td style="text-align:right;">
135.4339
</td>
<td style="text-align:left;">
negative
</td>
<td style="text-align:left;">
MGI_C57BL6J_1858220
</td>
<td style="text-align:left;">
Rcan3
</td>
<td style="text-align:left;">
NCBI_Gene:53902,ENSEMBL:ENSMUSG00000059713
</td>
<td style="text-align:left;">
MGI:1858220
</td>
<td style="text-align:left;">
protein coding gene
</td>
<td style="text-align:left;">
regulator of calcineurin 3
</td>
</tr>
<tr>
<td style="text-align:left;">
4
</td>
<td style="text-align:left;">
gene
</td>
<td style="text-align:right;">
135.4166
</td>
<td style="text-align:right;">
135.4166
</td>
<td style="text-align:left;">
negative
</td>
<td style="text-align:left;">
MGI_C57BL6J_3629655
</td>
<td style="text-align:left;">
Mir700
</td>
<td style="text-align:left;">
miRBase:MI0004684,NCBI_Gene:735285,ENSEMBL:ENSMUSG00000076123
</td>
<td style="text-align:left;">
MGI:3629655
</td>
<td style="text-align:left;">
miRNA gene
</td>
<td style="text-align:left;">
microRNA 700
</td>
</tr>
<tr>
<td style="text-align:left;">
4
</td>
<td style="text-align:left;">
gene
</td>
<td style="text-align:right;">
135.4207
</td>
<td style="text-align:right;">
135.4278
</td>
<td style="text-align:left;">
positive
</td>
<td style="text-align:left;">
MGI_C57BL6J_5589981
</td>
<td style="text-align:left;">
Gm30822
</td>
<td style="text-align:left;">
NCBI_Gene:102632859
</td>
<td style="text-align:left;">
MGI:5589981
</td>
<td style="text-align:left;">
lncRNA gene
</td>
<td style="text-align:left;">
predicted gene, 30822
</td>
</tr>
<tr>
<td style="text-align:left;">
4
</td>
<td style="text-align:left;">
gene
</td>
<td style="text-align:right;">
135.4301
</td>
<td style="text-align:right;">
135.4308
</td>
<td style="text-align:left;">
negative
</td>
<td style="text-align:left;">
MGI_C57BL6J_3801882
</td>
<td style="text-align:left;">
Gm15979
</td>
<td style="text-align:left;">
ENSEMBL:ENSMUSG00000087016
</td>
<td style="text-align:left;">
MGI:3801882
</td>
<td style="text-align:left;">
lncRNA gene
</td>
<td style="text-align:left;">
predicted gene 15979
</td>
</tr>
<tr>
<td style="text-align:left;">
4
</td>
<td style="text-align:left;">
gene
</td>
<td style="text-align:right;">
135.4454
</td>
<td style="text-align:right;">
135.4950
</td>
<td style="text-align:left;">
negative
</td>
<td style="text-align:left;">
MGI_C57BL6J_1921802
</td>
<td style="text-align:left;">
Nipal3
</td>
<td style="text-align:left;">
NCBI_Gene:74552,ENSEMBL:ENSMUSG00000028803
</td>
<td style="text-align:left;">
MGI:1921802
</td>
<td style="text-align:left;">
protein coding gene
</td>
<td style="text-align:left;">
NIPA-like domain containing 3
</td>
</tr>
<tr>
<td style="text-align:left;">
4
</td>
<td style="text-align:left;">
gene
</td>
<td style="text-align:right;">
135.4509
</td>
<td style="text-align:right;">
135.4512
</td>
<td style="text-align:left;">
positive
</td>
<td style="text-align:left;">
MGI_C57BL6J_5610498
</td>
<td style="text-align:left;">
Gm37270
</td>
<td style="text-align:left;">
ENSEMBL:ENSMUSG00000103955
</td>
<td style="text-align:left;">
MGI:5610498
</td>
<td style="text-align:left;">
unclassified gene
</td>
<td style="text-align:left;">
predicted gene, 37270
</td>
</tr>
<tr>
<td style="text-align:left;">
4
</td>
<td style="text-align:left;">
gene
</td>
<td style="text-align:right;">
135.4948
</td>
<td style="text-align:right;">
135.5378
</td>
<td style="text-align:left;">
positive
</td>
<td style="text-align:left;">
MGI_C57BL6J_1926056
</td>
<td style="text-align:left;">
Stpg1
</td>
<td style="text-align:left;">
NCBI_Gene:78806,ENSEMBL:ENSMUSG00000028801
</td>
<td style="text-align:left;">
MGI:1926056
</td>
<td style="text-align:left;">
protein coding gene
</td>
<td style="text-align:left;">
sperm tail PG rich repeat containing 1
</td>
</tr>
<tr>
<td style="text-align:left;">
4
</td>
<td style="text-align:left;">
gene
</td>
<td style="text-align:right;">
135.5124
</td>
<td style="text-align:right;">
135.5125
</td>
<td style="text-align:left;">
negative
</td>
<td style="text-align:left;">
MGI_C57BL6J_5455094
</td>
<td style="text-align:left;">
Gm25317
</td>
<td style="text-align:left;">
ENSEMBL:ENSMUSG00000080474
</td>
<td style="text-align:left;">
MGI:5455094
</td>
<td style="text-align:left;">
miRNA gene
</td>
<td style="text-align:left;">
predicted gene, 25317
</td>
</tr>
<tr>
<td style="text-align:left;">
4
</td>
<td style="text-align:left;">
gene
</td>
<td style="text-align:right;">
135.5419
</td>
<td style="text-align:right;">
135.5736
</td>
<td style="text-align:left;">
negative
</td>
<td style="text-align:left;">
MGI_C57BL6J_2655333
</td>
<td style="text-align:left;">
Grhl3
</td>
<td style="text-align:left;">
NCBI_Gene:230824,ENSEMBL:ENSMUSG00000037188
</td>
<td style="text-align:left;">
MGI:2655333
</td>
<td style="text-align:left;">
protein coding gene
</td>
<td style="text-align:left;">
grainyhead like transcription factor 3
</td>
</tr>
<tr>
<td style="text-align:left;">
4
</td>
<td style="text-align:left;">
gene
</td>
<td style="text-align:right;">
135.5736
</td>
<td style="text-align:right;">
135.5802
</td>
<td style="text-align:left;">
positive
</td>
<td style="text-align:left;">
MGI_C57BL6J_5590047
</td>
<td style="text-align:left;">
Gm30888
</td>
<td style="text-align:left;">
NCBI_Gene:102632942
</td>
<td style="text-align:left;">
MGI:5590047
</td>
<td style="text-align:left;">
lncRNA gene
</td>
<td style="text-align:left;">
predicted gene, 30888
</td>
</tr>
<tr>
<td style="text-align:left;">
4
</td>
<td style="text-align:left;">
gene
</td>
<td style="text-align:right;">
135.5837
</td>
<td style="text-align:right;">
135.5838
</td>
<td style="text-align:left;">
positive
</td>
<td style="text-align:left;">
MGI_C57BL6J_5452883
</td>
<td style="text-align:left;">
Gm23106
</td>
<td style="text-align:left;">
ENSEMBL:ENSMUSG00000088179
</td>
<td style="text-align:left;">
MGI:5452883
</td>
<td style="text-align:left;">
snRNA gene
</td>
<td style="text-align:left;">
predicted gene, 23106
</td>
</tr>
<tr>
<td style="text-align:left;">
4
</td>
<td style="text-align:left;">
gene
</td>
<td style="text-align:right;">
135.6267
</td>
<td style="text-align:right;">
135.6302
</td>
<td style="text-align:left;">
positive
</td>
<td style="text-align:left;">
MGI_C57BL6J_1913597
</td>
<td style="text-align:left;">
1700029M20Rik
</td>
<td style="text-align:left;">
NCBI_Gene:73937,ENSEMBL:ENSMUSG00000086788
</td>
<td style="text-align:left;">
MGI:1913597
</td>
<td style="text-align:left;">
lncRNA gene
</td>
<td style="text-align:left;">
RIKEN cDNA 1700029M20 gene
</td>
</tr>
<tr>
<td style="text-align:left;">
4
</td>
<td style="text-align:left;">
gene
</td>
<td style="text-align:right;">
135.6503
</td>
<td style="text-align:right;">
135.6540
</td>
<td style="text-align:left;">
negative
</td>
<td style="text-align:left;">
MGI_C57BL6J_5590100
</td>
<td style="text-align:left;">
Gm30941
</td>
<td style="text-align:left;">
NCBI_Gene:102633011
</td>
<td style="text-align:left;">
MGI:5590100
</td>
<td style="text-align:left;">
lncRNA gene
</td>
<td style="text-align:left;">
predicted gene, 30941
</td>
</tr>
<tr>
<td style="text-align:left;">
4
</td>
<td style="text-align:left;">
gene
</td>
<td style="text-align:right;">
135.6863
</td>
<td style="text-align:right;">
135.7082
</td>
<td style="text-align:left;">
positive
</td>
<td style="text-align:left;">
MGI_C57BL6J_2429859
</td>
<td style="text-align:left;">
Ifnlr1
</td>
<td style="text-align:left;">
NCBI_Gene:242700,ENSEMBL:ENSMUSG00000062157
</td>
<td style="text-align:left;">
MGI:2429859
</td>
<td style="text-align:left;">
protein coding gene
</td>
<td style="text-align:left;">
interferon lambda receptor 1
</td>
</tr>
<tr>
<td style="text-align:left;">
4
</td>
<td style="text-align:left;">
pseudogene
</td>
<td style="text-align:right;">
135.7157
</td>
<td style="text-align:right;">
135.7167
</td>
<td style="text-align:left;">
negative
</td>
<td style="text-align:left;">
MGI_C57BL6J_3652245
</td>
<td style="text-align:left;">
Gm12989
</td>
<td style="text-align:left;">
NCBI_Gene:100418217,ENSEMBL:ENSMUSG00000083678
</td>
<td style="text-align:left;">
MGI:3652245
</td>
<td style="text-align:left;">
pseudogene
</td>
<td style="text-align:left;">
predicted gene 12989
</td>
</tr>
<tr>
<td style="text-align:left;">
4
</td>
<td style="text-align:left;">
gene
</td>
<td style="text-align:right;">
135.7282
</td>
<td style="text-align:right;">
135.7554
</td>
<td style="text-align:left;">
positive
</td>
<td style="text-align:left;">
MGI_C57BL6J_2663588
</td>
<td style="text-align:left;">
Il22ra1
</td>
<td style="text-align:left;">
NCBI_Gene:230828,ENSEMBL:ENSMUSG00000037157
</td>
<td style="text-align:left;">
MGI:2663588
</td>
<td style="text-align:left;">
protein coding gene
</td>
<td style="text-align:left;">
interleukin 22 receptor, alpha 1
</td>
</tr>
<tr>
<td style="text-align:left;">
4
</td>
<td style="text-align:left;">
pseudogene
</td>
<td style="text-align:right;">
135.7556
</td>
<td style="text-align:right;">
135.7568
</td>
<td style="text-align:left;">
negative
</td>
<td style="text-align:left;">
MGI_C57BL6J_3650876
</td>
<td style="text-align:left;">
Gm12988
</td>
<td style="text-align:left;">
NCBI_Gene:665334,ENSEMBL:ENSMUSG00000082743
</td>
<td style="text-align:left;">
MGI:3650876
</td>
<td style="text-align:left;">
pseudogene
</td>
<td style="text-align:left;">
predicted gene 12988
</td>
</tr>
<tr>
<td style="text-align:left;">
4
</td>
<td style="text-align:left;">
gene
</td>
<td style="text-align:right;">
135.7588
</td>
<td style="text-align:right;">
135.8156
</td>
<td style="text-align:left;">
positive
</td>
<td style="text-align:left;">
MGI_C57BL6J_2685280
</td>
<td style="text-align:left;">
Myom3
</td>
<td style="text-align:left;">
NCBI_Gene:242702,ENSEMBL:ENSMUSG00000037139
</td>
<td style="text-align:left;">
MGI:2685280
</td>
<td style="text-align:left;">
protein coding gene
</td>
<td style="text-align:left;">
myomesin family, member 3
</td>
</tr>
<tr>
<td style="text-align:left;">
4
</td>
<td style="text-align:left;">
pseudogene
</td>
<td style="text-align:right;">
135.8537
</td>
<td style="text-align:right;">
135.8542
</td>
<td style="text-align:left;">
positive
</td>
<td style="text-align:left;">
MGI_C57BL6J_3650940
</td>
<td style="text-align:left;">
Gm13000
</td>
<td style="text-align:left;">
ENSEMBL:ENSMUSG00000081672
</td>
<td style="text-align:left;">
MGI:3650940
</td>
<td style="text-align:left;">
pseudogene
</td>
<td style="text-align:left;">
predicted gene 13000
</td>
</tr>
<tr>
<td style="text-align:left;">
4
</td>
<td style="text-align:left;">
gene
</td>
<td style="text-align:right;">
135.8557
</td>
<td style="text-align:right;">
135.8699
</td>
<td style="text-align:left;">
positive
</td>
<td style="text-align:left;">
MGI_C57BL6J_1333805
</td>
<td style="text-align:left;">
Srsf10
</td>
<td style="text-align:left;">
NCBI_Gene:14105,ENSEMBL:ENSMUSG00000028676
</td>
<td style="text-align:left;">
MGI:1333805
</td>
<td style="text-align:left;">
protein coding gene
</td>
<td style="text-align:left;">
serine/arginine-rich splicing factor 10
</td>
</tr>
<tr>
<td style="text-align:left;">
4
</td>
<td style="text-align:left;">
gene
</td>
<td style="text-align:right;">
135.8709
</td>
<td style="text-align:right;">
135.8739
</td>
<td style="text-align:left;">
negative
</td>
<td style="text-align:left;">
MGI_C57BL6J_106512
</td>
<td style="text-align:left;">
Pnrc2
</td>
<td style="text-align:left;">
NCBI_Gene:52830,ENSEMBL:ENSMUSG00000028675
</td>
<td style="text-align:left;">
MGI:106512
</td>
<td style="text-align:left;">
protein coding gene
</td>
<td style="text-align:left;">
proline-rich nuclear receptor coactivator 2
</td>
</tr>
<tr>
<td style="text-align:left;">
4
</td>
<td style="text-align:left;">
pseudogene
</td>
<td style="text-align:right;">
135.8913
</td>
<td style="text-align:right;">
135.8915
</td>
<td style="text-align:left;">
negative
</td>
<td style="text-align:left;">
MGI_C57BL6J_3651355
</td>
<td style="text-align:left;">
Gm13006
</td>
<td style="text-align:left;">
ENSEMBL:ENSMUSG00000083430
</td>
<td style="text-align:left;">
MGI:3651355
</td>
<td style="text-align:left;">
pseudogene
</td>
<td style="text-align:left;">
predicted gene 13006
</td>
</tr>
<tr>
<td style="text-align:left;">
4
</td>
<td style="text-align:left;">
gene
</td>
<td style="text-align:right;">
135.8952
</td>
<td style="text-align:right;">
135.9202
</td>
<td style="text-align:left;">
positive
</td>
<td style="text-align:left;">
MGI_C57BL6J_104650
</td>
<td style="text-align:left;">
Cnr2
</td>
<td style="text-align:left;">
NCBI_Gene:12802,ENSEMBL:ENSMUSG00000062585
</td>
<td style="text-align:left;">
MGI:104650
</td>
<td style="text-align:left;">
protein coding gene
</td>
<td style="text-align:left;">
cannabinoid receptor 2 (macrophage)
</td>
</tr>
<tr>
<td style="text-align:left;">
4
</td>
<td style="text-align:left;">
gene
</td>
<td style="text-align:right;">
135.9207
</td>
<td style="text-align:right;">
135.9403
</td>
<td style="text-align:left;">
positive
</td>
<td style="text-align:left;">
MGI_C57BL6J_95593
</td>
<td style="text-align:left;">
Fuca1
</td>
<td style="text-align:left;">
NCBI_Gene:71665,ENSEMBL:ENSMUSG00000028673
</td>
<td style="text-align:left;">
MGI:95593
</td>
<td style="text-align:left;">
protein coding gene
</td>
<td style="text-align:left;">
fucosidase, alpha-L- 1, tissue
</td>
</tr>
<tr>
<td style="text-align:left;">
4
</td>
<td style="text-align:left;">
gene
</td>
<td style="text-align:right;">
135.9464
</td>
<td style="text-align:right;">
135.9626
</td>
<td style="text-align:left;">
positive
</td>
<td style="text-align:left;">
MGI_C57BL6J_96158
</td>
<td style="text-align:left;">
Hmgcl
</td>
<td style="text-align:left;">
NCBI_Gene:15356,ENSEMBL:ENSMUSG00000028672
</td>
<td style="text-align:left;">
MGI:96158
</td>
<td style="text-align:left;">
protein coding gene
</td>
<td style="text-align:left;">
3-hydroxy-3-methylglutaryl-Coenzyme A lyase
</td>
</tr>
<tr>
<td style="text-align:left;">
4
</td>
<td style="text-align:left;">
gene
</td>
<td style="text-align:right;">
135.9637
</td>
<td style="text-align:right;">
135.9682
</td>
<td style="text-align:left;">
positive
</td>
<td style="text-align:left;">
MGI_C57BL6J_1921496
</td>
<td style="text-align:left;">
Gale
</td>
<td style="text-align:left;">
NCBI_Gene:74246,ENSEMBL:ENSMUSG00000028671
</td>
<td style="text-align:left;">
MGI:1921496
</td>
<td style="text-align:left;">
protein coding gene
</td>
<td style="text-align:left;">
galactose-4-epimerase, UDP
</td>
</tr>
<tr>
<td style="text-align:left;">
4
</td>
<td style="text-align:left;">
gene
</td>
<td style="text-align:right;">
135.9682
</td>
<td style="text-align:right;">
135.9726
</td>
<td style="text-align:left;">
negative
</td>
<td style="text-align:left;">
MGI_C57BL6J_1347000
</td>
<td style="text-align:left;">
Lypla2
</td>
<td style="text-align:left;">
NCBI_Gene:26394,ENSEMBL:ENSMUSG00000028670
</td>
<td style="text-align:left;">
MGI:1347000
</td>
<td style="text-align:left;">
protein coding gene
</td>
<td style="text-align:left;">
lysophospholipase 2
</td>
</tr>
<tr>
<td style="text-align:left;">
4
</td>
<td style="text-align:left;">
gene
</td>
<td style="text-align:right;">
135.9755
</td>
<td style="text-align:right;">
135.9873
</td>
<td style="text-align:left;">
negative
</td>
<td style="text-align:left;">
MGI_C57BL6J_1913443
</td>
<td style="text-align:left;">
Pithd1
</td>
<td style="text-align:left;">
NCBI_Gene:66193,ENSEMBL:ENSMUSG00000028669
</td>
<td style="text-align:left;">
MGI:1913443
</td>
<td style="text-align:left;">
protein coding gene
</td>
<td style="text-align:left;">
PITH (C-terminal proteasome-interacting domain of thioredoxin-like)
domain containing 1
</td>
</tr>
<tr>
<td style="text-align:left;">
4
</td>
<td style="text-align:left;">
pseudogene
</td>
<td style="text-align:right;">
135.9975
</td>
<td style="text-align:right;">
135.9979
</td>
<td style="text-align:left;">
positive
</td>
<td style="text-align:left;">
MGI_C57BL6J_3649722
</td>
<td style="text-align:left;">
Gm13007
</td>
<td style="text-align:left;">
NCBI_Gene:669445,ENSEMBL:ENSMUSG00000083852
</td>
<td style="text-align:left;">
MGI:3649722
</td>
<td style="text-align:left;">
pseudogene
</td>
<td style="text-align:left;">
predicted gene 13007
</td>
</tr>
<tr>
<td style="text-align:left;">
4
</td>
<td style="text-align:left;">
gene
</td>
<td style="text-align:right;">
136.0034
</td>
<td style="text-align:right;">
136.0218
</td>
<td style="text-align:left;">
negative
</td>
<td style="text-align:left;">
MGI_C57BL6J_1351315
</td>
<td style="text-align:left;">
Eloa
</td>
<td style="text-align:left;">
NCBI_Gene:27224,ENSEMBL:ENSMUSG00000028668
</td>
<td style="text-align:left;">
MGI:1351315
</td>
<td style="text-align:left;">
protein coding gene
</td>
<td style="text-align:left;">
elongin A
</td>
</tr>
<tr>
<td style="text-align:left;">
4
</td>
<td style="text-align:left;">
gene
</td>
<td style="text-align:right;">
136.0036
</td>
<td style="text-align:right;">
136.0109
</td>
<td style="text-align:left;">
positive
</td>
<td style="text-align:left;">
MGI_C57BL6J_3702670
</td>
<td style="text-align:left;">
Gm13008
</td>
<td style="text-align:left;">
ENSEMBL:ENSMUSG00000085714
</td>
<td style="text-align:left;">
MGI:3702670
</td>
<td style="text-align:left;">
lncRNA gene
</td>
<td style="text-align:left;">
predicted gene 13008
</td>
</tr>
<tr>
<td style="text-align:left;">
4
</td>
<td style="text-align:left;">
gene
</td>
<td style="text-align:right;">
136.0181
</td>
<td style="text-align:right;">
136.0185
</td>
<td style="text-align:left;">
negative
</td>
<td style="text-align:left;">
MGI_C57BL6J_5454139
</td>
<td style="text-align:left;">
Gm24362
</td>
<td style="text-align:left;">
ENSEMBL:ENSMUSG00000097995
</td>
<td style="text-align:left;">
MGI:5454139
</td>
<td style="text-align:left;">
lncRNA gene
</td>
<td style="text-align:left;">
predicted gene, 24362
</td>
</tr>
<tr>
<td style="text-align:left;">
4
</td>
<td style="text-align:left;">
gene
</td>
<td style="text-align:right;">
136.0283
</td>
<td style="text-align:right;">
136.0534
</td>
<td style="text-align:left;">
negative
</td>
<td style="text-align:left;">
MGI_C57BL6J_1914275
</td>
<td style="text-align:left;">
Rpl11
</td>
<td style="text-align:left;">
NCBI_Gene:67025,ENSEMBL:ENSMUSG00000059291
</td>
<td style="text-align:left;">
MGI:1914275
</td>
<td style="text-align:left;">
protein coding gene
</td>
<td style="text-align:left;">
ribosomal protein L11
</td>
</tr>
<tr>
<td style="text-align:left;">
4
</td>
<td style="text-align:left;">
pseudogene
</td>
<td style="text-align:right;">
136.0745
</td>
<td style="text-align:right;">
136.0750
</td>
<td style="text-align:left;">
negative
</td>
<td style="text-align:left;">
MGI_C57BL6J_3651086
</td>
<td style="text-align:left;">
Gm13009
</td>
<td style="text-align:left;">
ENSEMBL:ENSMUSG00000082877
</td>
<td style="text-align:left;">
MGI:3651086
</td>
<td style="text-align:left;">
pseudogene
</td>
<td style="text-align:left;">
predicted gene 13009
</td>
</tr>
<tr>
<td style="text-align:left;">
4
</td>
<td style="text-align:left;">
gene
</td>
<td style="text-align:right;">
136.1375
</td>
<td style="text-align:right;">
136.1376
</td>
<td style="text-align:left;">
positive
</td>
<td style="text-align:left;">
MGI_C57BL6J_5455778
</td>
<td style="text-align:left;">
Gm26001
</td>
<td style="text-align:left;">
ENSEMBL:ENSMUSG00000088614
</td>
<td style="text-align:left;">
MGI:5455778
</td>
<td style="text-align:left;">
snRNA gene
</td>
<td style="text-align:left;">
predicted gene, 26001
</td>
</tr>
<tr>
<td style="text-align:left;">
4
</td>
<td style="text-align:left;">
gene
</td>
<td style="text-align:right;">
136.1386
</td>
<td style="text-align:right;">
136.1434
</td>
<td style="text-align:left;">
negative
</td>
<td style="text-align:left;">
MGI_C57BL6J_5625214
</td>
<td style="text-align:left;">
Gm42329
</td>
<td style="text-align:left;">
NCBI_Gene:105247187
</td>
<td style="text-align:left;">
MGI:5625214
</td>
<td style="text-align:left;">
lncRNA gene
</td>
<td style="text-align:left;">
predicted gene, 42329
</td>
</tr>
<tr>
<td style="text-align:left;">
4
</td>
<td style="text-align:left;">
gene
</td>
<td style="text-align:right;">
136.1435
</td>
<td style="text-align:right;">
136.1458
</td>
<td style="text-align:left;">
positive
</td>
<td style="text-align:left;">
MGI_C57BL6J_96398
</td>
<td style="text-align:left;">
Id3
</td>
<td style="text-align:left;">
NCBI_Gene:15903,ENSEMBL:ENSMUSG00000007872
</td>
<td style="text-align:left;">
MGI:96398
</td>
<td style="text-align:left;">
protein coding gene
</td>
<td style="text-align:left;">
inhibitor of DNA binding 3
</td>
</tr>
<tr>
<td style="text-align:left;">
4
</td>
<td style="text-align:left;">
gene
</td>
<td style="text-align:right;">
136.1723
</td>
<td style="text-align:right;">
136.1961
</td>
<td style="text-align:left;">
positive
</td>
<td style="text-align:left;">
MGI_C57BL6J_1096341
</td>
<td style="text-align:left;">
E2f2
</td>
<td style="text-align:left;">
NCBI_Gene:242705,ENSEMBL:ENSMUSG00000018983
</td>
<td style="text-align:left;">
MGI:1096341
</td>
<td style="text-align:left;">
protein coding gene
</td>
<td style="text-align:left;">
E2F transcription factor 2
</td>
</tr>
<tr>
<td style="text-align:left;">
4
</td>
<td style="text-align:left;">
gene
</td>
<td style="text-align:right;">
136.2063
</td>
<td style="text-align:right;">
136.2466
</td>
<td style="text-align:left;">
positive
</td>
<td style="text-align:left;">
MGI_C57BL6J_2684986
</td>
<td style="text-align:left;">
Asap3
</td>
<td style="text-align:left;">
NCBI_Gene:230837,ENSEMBL:ENSMUSG00000036995
</td>
<td style="text-align:left;">
MGI:2684986
</td>
<td style="text-align:left;">
protein coding gene
</td>
<td style="text-align:left;">
ArfGAP with SH3 domain, ankyrin repeat and PH domain 3
</td>
</tr>
<tr>
<td style="text-align:left;">
4
</td>
<td style="text-align:left;">
gene
</td>
<td style="text-align:right;">
136.2145
</td>
<td style="text-align:right;">
136.2166
</td>
<td style="text-align:left;">
positive
</td>
<td style="text-align:left;">
MGI_C57BL6J_2441962
</td>
<td style="text-align:left;">
D230019A03Rik
</td>
<td style="text-align:left;">
NA
</td>
<td style="text-align:left;">
NA
</td>
<td style="text-align:left;">
unclassified gene
</td>
<td style="text-align:left;">
RIKEN cDNA D230019A03 gene
</td>
</tr>
<tr>
<td style="text-align:left;">
4
</td>
<td style="text-align:left;">
gene
</td>
<td style="text-align:right;">
136.2477
</td>
<td style="text-align:right;">
136.2749
</td>
<td style="text-align:left;">
positive
</td>
<td style="text-align:left;">
MGI_C57BL6J_1196908
</td>
<td style="text-align:left;">
Tcea3
</td>
<td style="text-align:left;">
NCBI_Gene:21401,ENSEMBL:ENSMUSG00000001604
</td>
<td style="text-align:left;">
MGI:1196908
</td>
<td style="text-align:left;">
protein coding gene
</td>
<td style="text-align:left;">
transcription elongation factor A (SII), 3
</td>
</tr>
<tr>
<td style="text-align:left;">
4
</td>
<td style="text-align:left;">
gene
</td>
<td style="text-align:right;">
136.2842
</td>
<td style="text-align:right;">
136.2939
</td>
<td style="text-align:left;">
positive
</td>
<td style="text-align:left;">
MGI_C57BL6J_99192
</td>
<td style="text-align:left;">
Zfp46
</td>
<td style="text-align:left;">
NCBI_Gene:22704,ENSEMBL:ENSMUSG00000051351
</td>
<td style="text-align:left;">
MGI:99192
</td>
<td style="text-align:left;">
protein coding gene
</td>
<td style="text-align:left;">
zinc finger protein 46
</td>
</tr>
<tr>
<td style="text-align:left;">
4
</td>
<td style="text-align:left;">
pseudogene
</td>
<td style="text-align:right;">
136.3074
</td>
<td style="text-align:right;">
136.3079
</td>
<td style="text-align:left;">
negative
</td>
<td style="text-align:left;">
MGI_C57BL6J_3650319
</td>
<td style="text-align:left;">
Gm13013
</td>
<td style="text-align:left;">
ENSEMBL:ENSMUSG00000081330
</td>
<td style="text-align:left;">
MGI:3650319
</td>
<td style="text-align:left;">
pseudogene
</td>
<td style="text-align:left;">
predicted gene 13013
</td>
</tr>
<tr>
<td style="text-align:left;">
4
</td>
<td style="text-align:left;">
gene
</td>
<td style="text-align:right;">
136.3109
</td>
<td style="text-align:right;">
136.3594
</td>
<td style="text-align:left;">
positive
</td>
<td style="text-align:left;">
MGI_C57BL6J_1891692
</td>
<td style="text-align:left;">
Hnrnpr
</td>
<td style="text-align:left;">
NCBI_Gene:74326,ENSEMBL:ENSMUSG00000066037
</td>
<td style="text-align:left;">
MGI:1891692
</td>
<td style="text-align:left;">
protein coding gene
</td>
<td style="text-align:left;">
heterogeneous nuclear ribonucleoprotein R
</td>
</tr>
<tr>
<td style="text-align:left;">
4
</td>
<td style="text-align:left;">
gene
</td>
<td style="text-align:right;">
136.3478
</td>
<td style="text-align:right;">
136.3494
</td>
<td style="text-align:left;">
negative
</td>
<td style="text-align:left;">
MGI_C57BL6J_5504137
</td>
<td style="text-align:left;">
Gm27022
</td>
<td style="text-align:left;">
ENSEMBL:ENSMUSG00000098155
</td>
<td style="text-align:left;">
MGI:5504137
</td>
<td style="text-align:left;">
lncRNA gene
</td>
<td style="text-align:left;">
predicted gene, 27022
</td>
</tr>
<tr>
<td style="text-align:left;">
4
</td>
<td style="text-align:left;">
gene
</td>
<td style="text-align:right;">
136.3503
</td>
<td style="text-align:right;">
136.3594
</td>
<td style="text-align:left;">
positive
</td>
<td style="text-align:left;">
MGI_C57BL6J_1918848
</td>
<td style="text-align:left;">
9130020K20Rik
</td>
<td style="text-align:left;">
NCBI_Gene:105247188
</td>
<td style="text-align:left;">
MGI:1918848
</td>
<td style="text-align:left;">
lncRNA gene
</td>
<td style="text-align:left;">
RIKEN cDNA 9130020K20 gene
</td>
</tr>
<tr>
<td style="text-align:left;">
4
</td>
<td style="text-align:left;">
gene
</td>
<td style="text-align:right;">
136.3558
</td>
<td style="text-align:right;">
136.3594
</td>
<td style="text-align:left;">
positive
</td>
<td style="text-align:left;">
MGI_C57BL6J_4937022
</td>
<td style="text-align:left;">
Gm17388
</td>
<td style="text-align:left;">
NA
</td>
<td style="text-align:left;">
NA
</td>
<td style="text-align:left;">
lncRNA gene
</td>
<td style="text-align:left;">
predicted gene%2c 17388
</td>
</tr>
<tr>
<td style="text-align:left;">
4
</td>
<td style="text-align:left;">
gene
</td>
<td style="text-align:right;">
136.3875
</td>
<td style="text-align:right;">
136.3890
</td>
<td style="text-align:left;">
positive
</td>
<td style="text-align:left;">
MGI_C57BL6J_5590454
</td>
<td style="text-align:left;">
Gm31295
</td>
<td style="text-align:left;">
NCBI_Gene:102633481
</td>
<td style="text-align:left;">
MGI:5590454
</td>
<td style="text-align:left;">
lncRNA gene
</td>
<td style="text-align:left;">
predicted gene, 31295
</td>
</tr>
<tr>
<td style="text-align:left;">
4
</td>
<td style="text-align:left;">
gene
</td>
<td style="text-align:right;">
136.4235
</td>
<td style="text-align:right;">
136.4444
</td>
<td style="text-align:left;">
positive
</td>
<td style="text-align:left;">
MGI_C57BL6J_96276
</td>
<td style="text-align:left;">
Htr1d
</td>
<td style="text-align:left;">
NCBI_Gene:15552,ENSEMBL:ENSMUSG00000070687
</td>
<td style="text-align:left;">
MGI:96276
</td>
<td style="text-align:left;">
protein coding gene
</td>
<td style="text-align:left;">
5-hydroxytryptamine (serotonin) receptor 1D
</td>
</tr>
<tr>
<td style="text-align:left;">
4
</td>
<td style="text-align:left;">
gene
</td>
<td style="text-align:right;">
136.4623
</td>
<td style="text-align:right;">
136.4640
</td>
<td style="text-align:left;">
positive
</td>
<td style="text-align:left;">
MGI_C57BL6J_2443447
</td>
<td style="text-align:left;">
6030445D17Rik
</td>
<td style="text-align:left;">
ENSEMBL:ENSMUSG00000066178
</td>
<td style="text-align:left;">
MGI:2443447
</td>
<td style="text-align:left;">
protein coding gene
</td>
<td style="text-align:left;">
RIKEN cDNA 6030445D17 gene
</td>
</tr>
<tr>
<td style="text-align:left;">
4
</td>
<td style="text-align:left;">
gene
</td>
<td style="text-align:right;">
136.4697
</td>
<td style="text-align:right;">
136.5548
</td>
<td style="text-align:left;">
positive
</td>
<td style="text-align:left;">
MGI_C57BL6J_107629
</td>
<td style="text-align:left;">
Luzp1
</td>
<td style="text-align:left;">
NCBI_Gene:269593,ENSEMBL:ENSMUSG00000001089
</td>
<td style="text-align:left;">
MGI:107629
</td>
<td style="text-align:left;">
protein coding gene
</td>
<td style="text-align:left;">
leucine zipper protein 1
</td>
</tr>
<tr>
<td style="text-align:left;">
4
</td>
<td style="text-align:left;">
gene
</td>
<td style="text-align:right;">
136.5505
</td>
<td style="text-align:right;">
136.6028
</td>
<td style="text-align:left;">
negative
</td>
<td style="text-align:left;">
MGI_C57BL6J_1196256
</td>
<td style="text-align:left;">
Kdm1a
</td>
<td style="text-align:left;">
NCBI_Gene:99982,ENSEMBL:ENSMUSG00000036940
</td>
<td style="text-align:left;">
MGI:1196256
</td>
<td style="text-align:left;">
protein coding gene
</td>
<td style="text-align:left;">
lysine (K)-specific demethylase 1A
</td>
</tr>
<tr>
<td style="text-align:left;">
4
</td>
<td style="text-align:left;">
pseudogene
</td>
<td style="text-align:right;">
136.5842
</td>
<td style="text-align:right;">
136.5847
</td>
<td style="text-align:left;">
negative
</td>
<td style="text-align:left;">
MGI_C57BL6J_3649751
</td>
<td style="text-align:left;">
Gm12987
</td>
<td style="text-align:left;">
ENSEMBL:ENSMUSG00000082711
</td>
<td style="text-align:left;">
MGI:3649751
</td>
<td style="text-align:left;">
pseudogene
</td>
<td style="text-align:left;">
predicted gene 12987
</td>
</tr>
<tr>
<td style="text-align:left;">
4
</td>
<td style="text-align:left;">
gene
</td>
<td style="text-align:right;">
136.6069
</td>
<td style="text-align:right;">
136.6133
</td>
<td style="text-align:left;">
positive
</td>
<td style="text-align:left;">
MGI_C57BL6J_1914913
</td>
<td style="text-align:left;">
Tex46
</td>
<td style="text-align:left;">
NCBI_Gene:67663,ENSEMBL:ENSMUSG00000036921
</td>
<td style="text-align:left;">
MGI:1914913
</td>
<td style="text-align:left;">
protein coding gene
</td>
<td style="text-align:left;">
testis expressed 46
</td>
</tr>
<tr>
<td style="text-align:left;">
4
</td>
<td style="text-align:left;">
pseudogene
</td>
<td style="text-align:right;">
136.6088
</td>
<td style="text-align:right;">
136.6099
</td>
<td style="text-align:left;">
negative
</td>
<td style="text-align:left;">
MGI_C57BL6J_3651584
</td>
<td style="text-align:left;">
Gm12986
</td>
<td style="text-align:left;">
NCBI_Gene:433770,ENSEMBL:ENSMUSG00000083054
</td>
<td style="text-align:left;">
MGI:3651584
</td>
<td style="text-align:left;">
pseudogene
</td>
<td style="text-align:left;">
predicted gene 12986
</td>
</tr>
<tr>
<td style="text-align:left;">
4
</td>
<td style="text-align:left;">
gene
</td>
<td style="text-align:right;">
136.6098
</td>
<td style="text-align:right;">
136.6099
</td>
<td style="text-align:left;">
negative
</td>
<td style="text-align:left;">
MGI_C57BL6J_5451957
</td>
<td style="text-align:left;">
Gm22180
</td>
<td style="text-align:left;">
ENSEMBL:ENSMUSG00000092747
</td>
<td style="text-align:left;">
MGI:5451957
</td>
<td style="text-align:left;">
miRNA gene
</td>
<td style="text-align:left;">
predicted gene, 22180
</td>
</tr>
<tr>
<td style="text-align:left;">
4
</td>
<td style="text-align:left;">
gene
</td>
<td style="text-align:right;">
136.6226
</td>
<td style="text-align:right;">
136.6398
</td>
<td style="text-align:left;">
positive
</td>
<td style="text-align:left;">
MGI_C57BL6J_2448566
</td>
<td style="text-align:left;">
Lactbl1
</td>
<td style="text-align:left;">
NCBI_Gene:242707,ENSEMBL:ENSMUSG00000070683
</td>
<td style="text-align:left;">
MGI:2448566
</td>
<td style="text-align:left;">
protein coding gene
</td>
<td style="text-align:left;">
lactamase, beta-like 1
</td>
</tr>
<tr>
<td style="text-align:left;">
4
</td>
<td style="text-align:left;">
gene
</td>
<td style="text-align:right;">
136.6396
</td>
<td style="text-align:right;">
136.6421
</td>
<td style="text-align:left;">
positive
</td>
<td style="text-align:left;">
MGI_C57BL6J_5610811
</td>
<td style="text-align:left;">
Gm37583
</td>
<td style="text-align:left;">
ENSEMBL:ENSMUSG00000103119
</td>
<td style="text-align:left;">
MGI:5610811
</td>
<td style="text-align:left;">
unclassified gene
</td>
<td style="text-align:left;">
predicted gene, 37583
</td>
</tr>
<tr>
<td style="text-align:left;">
4
</td>
<td style="text-align:left;">
gene
</td>
<td style="text-align:right;">
136.6475
</td>
<td style="text-align:right;">
136.8360
</td>
<td style="text-align:left;">
negative
</td>
<td style="text-align:left;">
MGI_C57BL6J_99611
</td>
<td style="text-align:left;">
Ephb2
</td>
<td style="text-align:left;">
NCBI_Gene:13844,ENSEMBL:ENSMUSG00000028664
</td>
<td style="text-align:left;">
MGI:99611
</td>
<td style="text-align:left;">
protein coding gene
</td>
<td style="text-align:left;">
Eph receptor B2
</td>
</tr>
<tr>
<td style="text-align:left;">
4
</td>
<td style="text-align:left;">
gene
</td>
<td style="text-align:right;">
136.8360
</td>
<td style="text-align:right;">
136.8421
</td>
<td style="text-align:left;">
positive
</td>
<td style="text-align:left;">
MGI_C57BL6J_5590695
</td>
<td style="text-align:left;">
Gm31536
</td>
<td style="text-align:left;">
NCBI_Gene:102633796
</td>
<td style="text-align:left;">
MGI:5590695
</td>
<td style="text-align:left;">
lncRNA gene
</td>
<td style="text-align:left;">
predicted gene, 31536
</td>
</tr>
<tr>
<td style="text-align:left;">
4
</td>
<td style="text-align:left;">
gene
</td>
<td style="text-align:right;">
136.8801
</td>
<td style="text-align:right;">
136.8862
</td>
<td style="text-align:left;">
negative
</td>
<td style="text-align:left;">
MGI_C57BL6J_88224
</td>
<td style="text-align:left;">
C1qb
</td>
<td style="text-align:left;">
NCBI_Gene:12260,ENSEMBL:ENSMUSG00000036905
</td>
<td style="text-align:left;">
MGI:88224
</td>
<td style="text-align:left;">
protein coding gene
</td>
<td style="text-align:left;">
complement component 1, q subcomponent, beta polypeptide
</td>
</tr>
<tr>
<td style="text-align:left;">
4
</td>
<td style="text-align:left;">
gene
</td>
<td style="text-align:right;">
136.8898
</td>
<td style="text-align:right;">
136.8931
</td>
<td style="text-align:left;">
negative
</td>
<td style="text-align:left;">
MGI_C57BL6J_88225
</td>
<td style="text-align:left;">
C1qc
</td>
<td style="text-align:left;">
NCBI_Gene:12262,ENSEMBL:ENSMUSG00000036896
</td>
<td style="text-align:left;">
MGI:88225
</td>
<td style="text-align:left;">
protein coding gene
</td>
<td style="text-align:left;">
complement component 1, q subcomponent, C chain
</td>
</tr>
<tr>
<td style="text-align:left;">
4
</td>
<td style="text-align:left;">
gene
</td>
<td style="text-align:right;">
136.8959
</td>
<td style="text-align:right;">
136.8988
</td>
<td style="text-align:left;">
negative
</td>
<td style="text-align:left;">
MGI_C57BL6J_88223
</td>
<td style="text-align:left;">
C1qa
</td>
<td style="text-align:left;">
NCBI_Gene:12259,ENSEMBL:ENSMUSG00000036887
</td>
<td style="text-align:left;">
MGI:88223
</td>
<td style="text-align:left;">
protein coding gene
</td>
<td style="text-align:left;">
complement component 1, q subcomponent, alpha polypeptide
</td>
</tr>
<tr>
<td style="text-align:left;">
4
</td>
<td style="text-align:left;">
gene
</td>
<td style="text-align:right;">
136.9294
</td>
<td style="text-align:right;">
136.9568
</td>
<td style="text-align:left;">
negative
</td>
<td style="text-align:left;">
MGI_C57BL6J_109378
</td>
<td style="text-align:left;">
Epha8
</td>
<td style="text-align:left;">
NCBI_Gene:13842,ENSEMBL:ENSMUSG00000028661
</td>
<td style="text-align:left;">
MGI:109378
</td>
<td style="text-align:left;">
protein coding gene
</td>
<td style="text-align:left;">
Eph receptor A8
</td>
</tr>
<tr>
<td style="text-align:left;">
4
</td>
<td style="text-align:left;">
gene
</td>
<td style="text-align:right;">
136.9797
</td>
<td style="text-align:right;">
137.0494
</td>
<td style="text-align:left;">
negative
</td>
<td style="text-align:left;">
MGI_C57BL6J_2682254
</td>
<td style="text-align:left;">
Zbtb40
</td>
<td style="text-align:left;">
NCBI_Gene:230848,ENSEMBL:ENSMUSG00000060862
</td>
<td style="text-align:left;">
MGI:2682254
</td>
<td style="text-align:left;">
protein coding gene
</td>
<td style="text-align:left;">
zinc finger and BTB domain containing 40
</td>
</tr>
<tr>
<td style="text-align:left;">
4
</td>
<td style="text-align:left;">
gene
</td>
<td style="text-align:right;">
137.0257
</td>
<td style="text-align:right;">
137.0258
</td>
<td style="text-align:left;">
positive
</td>
<td style="text-align:left;">
MGI_C57BL6J_5453611
</td>
<td style="text-align:left;">
Gm23834
</td>
<td style="text-align:left;">
ENSEMBL:ENSMUSG00000077624
</td>
<td style="text-align:left;">
MGI:5453611
</td>
<td style="text-align:left;">
snoRNA gene
</td>
<td style="text-align:left;">
predicted gene, 23834
</td>
</tr>
<tr>
<td style="text-align:left;">
4
</td>
<td style="text-align:left;">
gene
</td>
<td style="text-align:right;">
137.0766
</td>
<td style="text-align:right;">
137.0951
</td>
<td style="text-align:left;">
positive
</td>
<td style="text-align:left;">
MGI_C57BL6J_5590807
</td>
<td style="text-align:left;">
Gm31648
</td>
<td style="text-align:left;">
NCBI_Gene:102633949
</td>
<td style="text-align:left;">
MGI:5590807
</td>
<td style="text-align:left;">
lncRNA gene
</td>
<td style="text-align:left;">
predicted gene, 31648
</td>
</tr>
<tr>
<td style="text-align:left;">
4
</td>
<td style="text-align:left;">
gene
</td>
<td style="text-align:right;">
137.1167
</td>
<td style="text-align:right;">
137.1183
</td>
<td style="text-align:left;">
negative
</td>
<td style="text-align:left;">
MGI_C57BL6J_3702655
</td>
<td style="text-align:left;">
Gm13001
</td>
<td style="text-align:left;">
NCBI_Gene:102634021,ENSEMBL:ENSMUSG00000087417
</td>
<td style="text-align:left;">
MGI:3702655
</td>
<td style="text-align:left;">
lncRNA gene
</td>
<td style="text-align:left;">
predicted gene 13001
</td>
</tr>
<tr>
<td style="text-align:left;">
4
</td>
<td style="text-align:left;">
gene
</td>
<td style="text-align:right;">
137.1167
</td>
<td style="text-align:right;">
137.1176
</td>
<td style="text-align:left;">
negative
</td>
<td style="text-align:left;">
MGI_C57BL6J_1921513
</td>
<td style="text-align:left;">
1700037C06Rik
</td>
<td style="text-align:left;">
ENSEMBL:ENSMUSG00000085542
</td>
<td style="text-align:left;">
MGI:1921513
</td>
<td style="text-align:left;">
lncRNA gene
</td>
<td style="text-align:left;">
RIKEN cDNA 1700037C06 gene
</td>
</tr>
<tr>
<td style="text-align:left;">
4
</td>
<td style="text-align:left;">
gene
</td>
<td style="text-align:right;">
137.1395
</td>
<td style="text-align:right;">
137.1425
</td>
<td style="text-align:left;">
negative
</td>
<td style="text-align:left;">
MGI_C57BL6J_5590921
</td>
<td style="text-align:left;">
Gm31762
</td>
<td style="text-align:left;">
NCBI_Gene:102634099
</td>
<td style="text-align:left;">
MGI:5590921
</td>
<td style="text-align:left;">
lncRNA gene
</td>
<td style="text-align:left;">
predicted gene, 31762
</td>
</tr>
<tr>
<td style="text-align:left;">
4
</td>
<td style="text-align:left;">
gene
</td>
<td style="text-align:right;">
137.1591
</td>
<td style="text-align:right;">
137.1651
</td>
<td style="text-align:left;">
negative
</td>
<td style="text-align:left;">
MGI_C57BL6J_3650099
</td>
<td style="text-align:left;">
Gm13003
</td>
<td style="text-align:left;">
NCBI_Gene:100503595,ENSEMBL:ENSMUSG00000085833
</td>
<td style="text-align:left;">
MGI:3650099
</td>
<td style="text-align:left;">
lncRNA gene
</td>
<td style="text-align:left;">
predicted gene 13003
</td>
</tr>
<tr>
<td style="text-align:left;">
4
</td>
<td style="text-align:left;">
gene
</td>
<td style="text-align:right;">
137.1713
</td>
<td style="text-align:right;">
137.1781
</td>
<td style="text-align:left;">
negative
</td>
<td style="text-align:left;">
MGI_C57BL6J_5826509
</td>
<td style="text-align:left;">
Gm46872
</td>
<td style="text-align:left;">
NCBI_Gene:108168981
</td>
<td style="text-align:left;">
MGI:5826509
</td>
<td style="text-align:left;">
lncRNA gene
</td>
<td style="text-align:left;">
predicted gene, 46872
</td>
</tr>
<tr>
<td style="text-align:left;">
4
</td>
<td style="text-align:left;">
gene
</td>
<td style="text-align:right;">
137.1840
</td>
<td style="text-align:right;">
137.1841
</td>
<td style="text-align:left;">
positive
</td>
<td style="text-align:left;">
MGI_C57BL6J_5455176
</td>
<td style="text-align:left;">
Gm25399
</td>
<td style="text-align:left;">
ENSEMBL:ENSMUSG00000093294
</td>
<td style="text-align:left;">
MGI:5455176
</td>
<td style="text-align:left;">
miRNA gene
</td>
<td style="text-align:left;">
predicted gene, 25399
</td>
</tr>
<tr>
<td style="text-align:left;">
4
</td>
<td style="text-align:left;">
gene
</td>
<td style="text-align:right;">
137.1841
</td>
<td style="text-align:right;">
137.1841
</td>
<td style="text-align:left;">
positive
</td>
<td style="text-align:left;">
MGI_C57BL6J_5453040
</td>
<td style="text-align:left;">
Gm23263
</td>
<td style="text-align:left;">
ENSEMBL:ENSMUSG00000092848
</td>
<td style="text-align:left;">
MGI:5453040
</td>
<td style="text-align:left;">
miRNA gene
</td>
<td style="text-align:left;">
predicted gene, 23263
</td>
</tr>
<tr>
<td style="text-align:left;">
4
</td>
<td style="text-align:left;">
gene
</td>
<td style="text-align:right;">
137.1841
</td>
<td style="text-align:right;">
137.2528
</td>
<td style="text-align:left;">
negative
</td>
<td style="text-align:left;">
MGI_C57BL6J_5590979
</td>
<td style="text-align:left;">
Gm31820
</td>
<td style="text-align:left;">
NCBI_Gene:102634171
</td>
<td style="text-align:left;">
MGI:5590979
</td>
<td style="text-align:left;">
lncRNA gene
</td>
<td style="text-align:left;">
predicted gene, 31820
</td>
</tr>
<tr>
<td style="text-align:left;">
4
</td>
<td style="text-align:left;">
pseudogene
</td>
<td style="text-align:right;">
137.2190
</td>
<td style="text-align:right;">
137.2204
</td>
<td style="text-align:left;">
negative
</td>
<td style="text-align:left;">
MGI_C57BL6J_3650102
</td>
<td style="text-align:left;">
Gm13005
</td>
<td style="text-align:left;">
NCBI_Gene:100047000,ENSEMBL:ENSMUSG00000080806
</td>
<td style="text-align:left;">
MGI:3650102
</td>
<td style="text-align:left;">
pseudogene
</td>
<td style="text-align:left;">
predicted gene 13005
</td>
</tr>
<tr>
<td style="text-align:left;">
4
</td>
<td style="text-align:left;">
pseudogene
</td>
<td style="text-align:right;">
137.2226
</td>
<td style="text-align:right;">
137.2233
</td>
<td style="text-align:left;">
negative
</td>
<td style="text-align:left;">
MGI_C57BL6J_3650101
</td>
<td style="text-align:left;">
Gm13002
</td>
<td style="text-align:left;">
ENSEMBL:ENSMUSG00000082149
</td>
<td style="text-align:left;">
MGI:3650101
</td>
<td style="text-align:left;">
pseudogene
</td>
<td style="text-align:left;">
predicted gene 13002
</td>
</tr>
<tr>
<td style="text-align:left;">
4
</td>
<td style="text-align:left;">
gene
</td>
<td style="text-align:right;">
137.2775
</td>
<td style="text-align:right;">
137.2997
</td>
<td style="text-align:left;">
positive
</td>
<td style="text-align:left;">
MGI_C57BL6J_98957
</td>
<td style="text-align:left;">
Wnt4
</td>
<td style="text-align:left;">
NCBI_Gene:22417,ENSEMBL:ENSMUSG00000036856
</td>
<td style="text-align:left;">
MGI:98957
</td>
<td style="text-align:left;">
protein coding gene
</td>
<td style="text-align:left;">
wingless-type MMTV integration site family, member 4
</td>
</tr>
<tr>
<td style="text-align:left;">
4
</td>
<td style="text-align:left;">
gene
</td>
<td style="text-align:right;">
137.3197
</td>
<td style="text-align:right;">
137.3578
</td>
<td style="text-align:left;">
negative
</td>
<td style="text-align:left;">
MGI_C57BL6J_106211
</td>
<td style="text-align:left;">
Cdc42
</td>
<td style="text-align:left;">
NCBI_Gene:12540,ENSEMBL:ENSMUSG00000006699
</td>
<td style="text-align:left;">
MGI:106211
</td>
<td style="text-align:left;">
protein coding gene
</td>
<td style="text-align:left;">
cell division cycle 42
</td>
</tr>
<tr>
<td style="text-align:left;">
4
</td>
<td style="text-align:left;">
gene
</td>
<td style="text-align:right;">
137.3256
</td>
<td style="text-align:right;">
137.3256
</td>
<td style="text-align:left;">
positive
</td>
<td style="text-align:left;">
MGI_C57BL6J_5455549
</td>
<td style="text-align:left;">
Gm25772
</td>
<td style="text-align:left;">
ENSEMBL:ENSMUSG00000089199
</td>
<td style="text-align:left;">
MGI:5455549
</td>
<td style="text-align:left;">
snRNA gene
</td>
<td style="text-align:left;">
predicted gene, 25772
</td>
</tr>
<tr>
<td style="text-align:left;">
4
</td>
<td style="text-align:left;">
gene
</td>
<td style="text-align:right;">
137.3364
</td>
<td style="text-align:right;">
137.3401
</td>
<td style="text-align:left;">
negative
</td>
<td style="text-align:left;">
MGI_C57BL6J_2441797
</td>
<td style="text-align:left;">
A430061O12Rik
</td>
<td style="text-align:left;">
NA
</td>
<td style="text-align:left;">
NA
</td>
<td style="text-align:left;">
unclassified gene
</td>
<td style="text-align:left;">
RIKEN cDNA A430061O12 gene
</td>
</tr>
<tr>
<td style="text-align:left;">
4
</td>
<td style="text-align:left;">
gene
</td>
<td style="text-align:right;">
137.3873
</td>
<td style="text-align:right;">
137.3884
</td>
<td style="text-align:left;">
negative
</td>
<td style="text-align:left;">
MGI_C57BL6J_1917225
</td>
<td style="text-align:left;">
2810405F17Rik
</td>
<td style="text-align:left;">
ENSEMBL:ENSMUSG00000101493
</td>
<td style="text-align:left;">
MGI:1917225
</td>
<td style="text-align:left;">
lncRNA gene
</td>
<td style="text-align:left;">
RIKEN cDNA 2810405F17 gene
</td>
</tr>
<tr>
<td style="text-align:left;">
4
</td>
<td style="text-align:left;">
pseudogene
</td>
<td style="text-align:right;">
137.3953
</td>
<td style="text-align:right;">
137.3957
</td>
<td style="text-align:left;">
positive
</td>
<td style="text-align:left;">
MGI_C57BL6J_3650100
</td>
<td style="text-align:left;">
Rpl31-ps10
</td>
<td style="text-align:left;">
NCBI_Gene:665533,ENSEMBL:ENSMUSG00000070667
</td>
<td style="text-align:left;">
MGI:3650100
</td>
<td style="text-align:left;">
pseudogene
</td>
<td style="text-align:left;">
ribosomal protein L31, pseudogene 10
</td>
</tr>
<tr>
<td style="text-align:left;">
4
</td>
<td style="text-align:left;">
gene
</td>
<td style="text-align:right;">
137.4016
</td>
<td style="text-align:right;">
137.4098
</td>
<td style="text-align:left;">
negative
</td>
<td style="text-align:left;">
MGI_C57BL6J_3651647
</td>
<td style="text-align:left;">
Cela3a
</td>
<td style="text-align:left;">
NCBI_Gene:242711,ENSEMBL:ENSMUSG00000078520
</td>
<td style="text-align:left;">
MGI:3651647
</td>
<td style="text-align:left;">
protein coding gene
</td>
<td style="text-align:left;">
chymotrypsin-like elastase family, member 3A
</td>
</tr>
<tr>
<td style="text-align:left;">
4
</td>
<td style="text-align:left;">
gene
</td>
<td style="text-align:right;">
137.4210
</td>
<td style="text-align:right;">
137.4305
</td>
<td style="text-align:left;">
negative
</td>
<td style="text-align:left;">
MGI_C57BL6J_1915118
</td>
<td style="text-align:left;">
Cela3b
</td>
<td style="text-align:left;">
NCBI_Gene:67868,ENSEMBL:ENSMUSG00000023433
</td>
<td style="text-align:left;">
MGI:1915118
</td>
<td style="text-align:left;">
protein coding gene
</td>
<td style="text-align:left;">
chymotrypsin-like elastase family, member 3B
</td>
</tr>
<tr>
<td style="text-align:left;">
4
</td>
<td style="text-align:left;">
gene
</td>
<td style="text-align:right;">
137.4366
</td>
<td style="text-align:right;">
137.4671
</td>
<td style="text-align:left;">
negative
</td>
<td style="text-align:left;">
MGI_C57BL6J_3651180
</td>
<td style="text-align:left;">
Gm13010
</td>
<td style="text-align:left;">
NCBI_Gene:105247191,ENSEMBL:ENSMUSG00000086845
</td>
<td style="text-align:left;">
MGI:3651180
</td>
<td style="text-align:left;">
lncRNA gene
</td>
<td style="text-align:left;">
predicted gene 13010
</td>
</tr>
<tr>
<td style="text-align:left;">
4
</td>
<td style="text-align:left;">
gene
</td>
<td style="text-align:right;">
137.4533
</td>
<td style="text-align:right;">
137.4555
</td>
<td style="text-align:left;">
positive
</td>
<td style="text-align:left;">
MGI_C57BL6J_1916630
</td>
<td style="text-align:left;">
1700013G24Rik
</td>
<td style="text-align:left;">
NCBI_Gene:69380,ENSEMBL:ENSMUSG00000041399
</td>
<td style="text-align:left;">
MGI:1916630
</td>
<td style="text-align:left;">
protein coding gene
</td>
<td style="text-align:left;">
RIKEN cDNA 1700013G24 gene
</td>
</tr>
<tr>
<td style="text-align:left;">
4
</td>
<td style="text-align:left;">
gene
</td>
<td style="text-align:right;">
137.4688
</td>
<td style="text-align:right;">
137.5706
</td>
<td style="text-align:left;">
positive
</td>
<td style="text-align:left;">
MGI_C57BL6J_96257
</td>
<td style="text-align:left;">
Hspg2
</td>
<td style="text-align:left;">
NCBI_Gene:15530,ENSEMBL:ENSMUSG00000028763
</td>
<td style="text-align:left;">
MGI:96257
</td>
<td style="text-align:left;">
protein coding gene
</td>
<td style="text-align:left;">
perlecan (heparan sulfate proteoglycan 2)
</td>
</tr>
<tr>
<td style="text-align:left;">
4
</td>
<td style="text-align:left;">
gene
</td>
<td style="text-align:right;">
137.5383
</td>
<td style="text-align:right;">
137.5384
</td>
<td style="text-align:left;">
positive
</td>
<td style="text-align:left;">
MGI_C57BL6J_5530988
</td>
<td style="text-align:left;">
Mir7018
</td>
<td style="text-align:left;">
miRBase:MI0022867,NCBI_Gene:102465616,ENSEMBL:ENSMUSG00000099134
</td>
<td style="text-align:left;">
MGI:5530988
</td>
<td style="text-align:left;">
miRNA gene
</td>
<td style="text-align:left;">
microRNA 7018
</td>
</tr>
<tr>
<td style="text-align:left;">
4
</td>
<td style="text-align:left;">
gene
</td>
<td style="text-align:right;">
137.5707
</td>
<td style="text-align:right;">
137.5824
</td>
<td style="text-align:left;">
negative
</td>
<td style="text-align:left;">
MGI_C57BL6J_3588210
</td>
<td style="text-align:left;">
Ldlrad2
</td>
<td style="text-align:left;">
NCBI_Gene:435811,ENSEMBL:ENSMUSG00000094035
</td>
<td style="text-align:left;">
MGI:3588210
</td>
<td style="text-align:left;">
protein coding gene
</td>
<td style="text-align:left;">
low density lipoprotein receptor class A domain containing 2
</td>
</tr>
<tr>
<td style="text-align:left;">
4
</td>
<td style="text-align:left;">
gene
</td>
<td style="text-align:right;">
137.5938
</td>
<td style="text-align:right;">
137.6585
</td>
<td style="text-align:left;">
positive
</td>
<td style="text-align:left;">
MGI_C57BL6J_2158502
</td>
<td style="text-align:left;">
Usp48
</td>
<td style="text-align:left;">
NCBI_Gene:170707,ENSEMBL:ENSMUSG00000043411
</td>
<td style="text-align:left;">
MGI:2158502
</td>
<td style="text-align:left;">
protein coding gene
</td>
<td style="text-align:left;">
ubiquitin specific peptidase 48
</td>
</tr>
<tr>
<td style="text-align:left;">
4
</td>
<td style="text-align:left;">
gene
</td>
<td style="text-align:right;">
137.6647
</td>
<td style="text-align:right;">
137.7299
</td>
<td style="text-align:left;">
positive
</td>
<td style="text-align:left;">
MGI_C57BL6J_109338
</td>
<td style="text-align:left;">
Rap1gap
</td>
<td style="text-align:left;">
NCBI_Gene:110351,ENSEMBL:ENSMUSG00000041351
</td>
<td style="text-align:left;">
MGI:109338
</td>
<td style="text-align:left;">
protein coding gene
</td>
<td style="text-align:left;">
Rap1 GTPase-activating protein
</td>
</tr>
<tr>
<td style="text-align:left;">
4
</td>
<td style="text-align:left;">
gene
</td>
<td style="text-align:right;">
137.6970
</td>
<td style="text-align:right;">
137.7074
</td>
<td style="text-align:left;">
negative
</td>
<td style="text-align:left;">
MGI_C57BL6J_1917058
</td>
<td style="text-align:left;">
Rap1gapos
</td>
<td style="text-align:left;">
NCBI_Gene:69808,ENSEMBL:ENSMUSG00000087476
</td>
<td style="text-align:left;">
MGI:1917058
</td>
<td style="text-align:left;">
antisense lncRNA gene
</td>
<td style="text-align:left;">
RAP1 GTPase activating protein, opposite strand
</td>
</tr>
<tr>
<td style="text-align:left;">
4
</td>
<td style="text-align:left;">
gene
</td>
<td style="text-align:right;">
137.7417
</td>
<td style="text-align:right;">
137.7964
</td>
<td style="text-align:left;">
negative
</td>
<td style="text-align:left;">
MGI_C57BL6J_87983
</td>
<td style="text-align:left;">
Alpl
</td>
<td style="text-align:left;">
NCBI_Gene:11647,ENSEMBL:ENSMUSG00000028766
</td>
<td style="text-align:left;">
MGI:87983
</td>
<td style="text-align:left;">
protein coding gene
</td>
<td style="text-align:left;">
alkaline phosphatase, liver/bone/kidney
</td>
</tr>
<tr>
<td style="text-align:left;">
4
</td>
<td style="text-align:left;">
gene
</td>
<td style="text-align:right;">
137.8063
</td>
<td style="text-align:right;">
137.8120
</td>
<td style="text-align:left;">
negative
</td>
<td style="text-align:left;">
MGI_C57BL6J_5591238
</td>
<td style="text-align:left;">
Gm32079
</td>
<td style="text-align:left;">
NCBI_Gene:102634516
</td>
<td style="text-align:left;">
MGI:5591238
</td>
<td style="text-align:left;">
lncRNA gene
</td>
<td style="text-align:left;">
predicted gene, 32079
</td>
</tr>
<tr>
<td style="text-align:left;">
4
</td>
<td style="text-align:left;">
gene
</td>
<td style="text-align:right;">
137.8622
</td>
<td style="text-align:right;">
137.9652
</td>
<td style="text-align:left;">
positive
</td>
<td style="text-align:left;">
MGI_C57BL6J_1101357
</td>
<td style="text-align:left;">
Ece1
</td>
<td style="text-align:left;">
NCBI_Gene:230857,ENSEMBL:ENSMUSG00000057530
</td>
<td style="text-align:left;">
MGI:1101357
</td>
<td style="text-align:left;">
protein coding gene
</td>
<td style="text-align:left;">
endothelin converting enzyme 1
</td>
</tr>
<tr>
<td style="text-align:left;">
4
</td>
<td style="text-align:left;">
gene
</td>
<td style="text-align:right;">
137.8820
</td>
<td style="text-align:right;">
137.8839
</td>
<td style="text-align:left;">
negative
</td>
<td style="text-align:left;">
MGI_C57BL6J_3652270
</td>
<td style="text-align:left;">
Gm13012
</td>
<td style="text-align:left;">
NCBI_Gene:102634683,ENSEMBL:ENSMUSG00000085823
</td>
<td style="text-align:left;">
MGI:3652270
</td>
<td style="text-align:left;">
lncRNA gene
</td>
<td style="text-align:left;">
predicted gene 13012
</td>
</tr>
<tr>
<td style="text-align:left;">
4
</td>
<td style="text-align:left;">
gene
</td>
<td style="text-align:right;">
137.9154
</td>
<td style="text-align:right;">
137.9155
</td>
<td style="text-align:left;">
negative
</td>
<td style="text-align:left;">
MGI_C57BL6J_1921862
</td>
<td style="text-align:left;">
4833410I11Rik
</td>
<td style="text-align:left;">
NA
</td>
<td style="text-align:left;">
NA
</td>
<td style="text-align:left;">
unclassified gene
</td>
<td style="text-align:left;">
RIKEN cDNA 4833410I11 gene
</td>
</tr>
<tr>
<td style="text-align:left;">
4
</td>
<td style="text-align:left;">
gene
</td>
<td style="text-align:right;">
137.9918
</td>
<td style="text-align:right;">
137.9923
</td>
<td style="text-align:left;">
positive
</td>
<td style="text-align:left;">
MGI_C57BL6J_3035105
</td>
<td style="text-align:left;">
BE691133
</td>
<td style="text-align:left;">
NA
</td>
<td style="text-align:left;">
NA
</td>
<td style="text-align:left;">
unclassified gene
</td>
<td style="text-align:left;">
expressed sequence BE691133
</td>
</tr>
<tr>
<td style="text-align:left;">
4
</td>
<td style="text-align:left;">
gene
</td>
<td style="text-align:right;">
137.9930
</td>
<td style="text-align:right;">
138.2085
</td>
<td style="text-align:left;">
positive
</td>
<td style="text-align:left;">
MGI_C57BL6J_1923935
</td>
<td style="text-align:left;">
Eif4g3
</td>
<td style="text-align:left;">
NCBI_Gene:230861,ENSEMBL:ENSMUSG00000028760
</td>
<td style="text-align:left;">
MGI:1923935
</td>
<td style="text-align:left;">
protein coding gene
</td>
<td style="text-align:left;">
eukaryotic translation initiation factor 4 gamma, 3
</td>
</tr>
<tr>
<td style="text-align:left;">
4
</td>
<td style="text-align:left;">
gene
</td>
<td style="text-align:right;">
137.9942
</td>
<td style="text-align:right;">
138.0279
</td>
<td style="text-align:left;">
positive
</td>
<td style="text-align:left;">
MGI_C57BL6J_5591553
</td>
<td style="text-align:left;">
Gm32394
</td>
<td style="text-align:left;">
NCBI_Gene:102634927
</td>
<td style="text-align:left;">
MGI:5591553
</td>
<td style="text-align:left;">
lncRNA gene
</td>
<td style="text-align:left;">
predicted gene, 32394
</td>
</tr>
<tr>
<td style="text-align:left;">
4
</td>
<td style="text-align:left;">
gene
</td>
<td style="text-align:right;">
138.0140
</td>
<td style="text-align:right;">
138.0184
</td>
<td style="text-align:left;">
positive
</td>
<td style="text-align:left;">
MGI_C57BL6J_3611235
</td>
<td style="text-align:left;">
8030494B02Rik
</td>
<td style="text-align:left;">
NA
</td>
<td style="text-align:left;">
NA
</td>
<td style="text-align:left;">
unclassified non-coding RNA gene
</td>
<td style="text-align:left;">
Riken cDNA 8030494B02 gene
</td>
</tr>
<tr>
<td style="text-align:left;">
4
</td>
<td style="text-align:left;">
gene
</td>
<td style="text-align:right;">
138.1376
</td>
<td style="text-align:right;">
138.1385
</td>
<td style="text-align:left;">
negative
</td>
<td style="text-align:left;">
MGI_C57BL6J_1920813
</td>
<td style="text-align:left;">
1700095J12Rik
</td>
<td style="text-align:left;">
NCBI_Gene:73563,ENSEMBL:ENSMUSG00000084804
</td>
<td style="text-align:left;">
MGI:1920813
</td>
<td style="text-align:left;">
lncRNA gene
</td>
<td style="text-align:left;">
RIKEN cDNA 1700095J12 gene
</td>
</tr>
<tr>
<td style="text-align:left;">
4
</td>
<td style="text-align:left;">
gene
</td>
<td style="text-align:right;">
138.1632
</td>
<td style="text-align:right;">
138.1633
</td>
<td style="text-align:left;">
positive
</td>
<td style="text-align:left;">
MGI_C57BL6J_5531049
</td>
<td style="text-align:left;">
Mir6399
</td>
<td style="text-align:left;">
miRBase:MI0021935,NCBI_Gene:102466907,ENSEMBL:ENSMUSG00000098513
</td>
<td style="text-align:left;">
MGI:5531049
</td>
<td style="text-align:left;">
miRNA gene
</td>
<td style="text-align:left;">
microRNA 6399
</td>
</tr>
<tr>
<td style="text-align:left;">
4
</td>
<td style="text-align:left;">
gene
</td>
<td style="text-align:right;">
138.2157
</td>
<td style="text-align:right;">
138.2162
</td>
<td style="text-align:left;">
positive
</td>
<td style="text-align:left;">
MGI_C57BL6J_1916844
</td>
<td style="text-align:left;">
2310026L22Rik
</td>
<td style="text-align:left;">
ENSEMBL:ENSMUSG00000078519
</td>
<td style="text-align:left;">
MGI:1916844
</td>
<td style="text-align:left;">
lncRNA gene
</td>
<td style="text-align:left;">
RIKEN cDNA 2310026L22 gene
</td>
</tr>
<tr>
<td style="text-align:left;">
4
</td>
<td style="text-align:left;">
gene
</td>
<td style="text-align:right;">
138.2163
</td>
<td style="text-align:right;">
138.2447
</td>
<td style="text-align:left;">
positive
</td>
<td style="text-align:left;">
MGI_C57BL6J_109369
</td>
<td style="text-align:left;">
Hp1bp3
</td>
<td style="text-align:left;">
NCBI_Gene:15441,ENSEMBL:ENSMUSG00000028759
</td>
<td style="text-align:left;">
MGI:109369
</td>
<td style="text-align:left;">
protein coding gene
</td>
<td style="text-align:left;">
heterochromatin protein 1, binding protein 3
</td>
</tr>
<tr>
<td style="text-align:left;">
4
</td>
<td style="text-align:left;">
gene
</td>
<td style="text-align:right;">
138.2504
</td>
<td style="text-align:right;">
138.2613
</td>
<td style="text-align:left;">
positive
</td>
<td style="text-align:left;">
MGI_C57BL6J_2446215
</td>
<td style="text-align:left;">
Sh2d5
</td>
<td style="text-align:left;">
NCBI_Gene:230863,ENSEMBL:ENSMUSG00000045349
</td>
<td style="text-align:left;">
MGI:2446215
</td>
<td style="text-align:left;">
protein coding gene
</td>
<td style="text-align:left;">
SH2 domain containing 5
</td>
</tr>
<tr>
<td style="text-align:left;">
4
</td>
<td style="text-align:left;">
gene
</td>
<td style="text-align:right;">
138.2504
</td>
<td style="text-align:right;">
138.3020
</td>
<td style="text-align:left;">
positive
</td>
<td style="text-align:left;">
MGI_C57BL6J_1098229
</td>
<td style="text-align:left;">
Kif17
</td>
<td style="text-align:left;">
NCBI_Gene:16559,ENSEMBL:ENSMUSG00000028758
</td>
<td style="text-align:left;">
MGI:1098229
</td>
<td style="text-align:left;">
protein coding gene
</td>
<td style="text-align:left;">
kinesin family member 17
</td>
</tr>
<tr>
<td style="text-align:left;">
4
</td>
<td style="text-align:left;">
gene
</td>
<td style="text-align:right;">
138.3047
</td>
<td style="text-align:right;">
138.3126
</td>
<td style="text-align:left;">
positive
</td>
<td style="text-align:left;">
MGI_C57BL6J_1194508
</td>
<td style="text-align:left;">
Ddost
</td>
<td style="text-align:left;">
NCBI_Gene:13200,ENSEMBL:ENSMUSG00000028757
</td>
<td style="text-align:left;">
MGI:1194508
</td>
<td style="text-align:left;">
protein coding gene
</td>
<td style="text-align:left;">
dolichyl-di-phosphooligosaccharide-protein glycotransferase
</td>
</tr>
<tr>
<td style="text-align:left;">
4
</td>
<td style="text-align:left;">
gene
</td>
<td style="text-align:right;">
138.3134
</td>
<td style="text-align:right;">
138.3263
</td>
<td style="text-align:left;">
negative
</td>
<td style="text-align:left;">
MGI_C57BL6J_1916193
</td>
<td style="text-align:left;">
Pink1
</td>
<td style="text-align:left;">
NCBI_Gene:68943,ENSEMBL:ENSMUSG00000028756
</td>
<td style="text-align:left;">
MGI:1916193
</td>
<td style="text-align:left;">
protein coding gene
</td>
<td style="text-align:left;">
PTEN induced putative kinase 1
</td>
</tr>
<tr>
<td style="text-align:left;">
4
</td>
<td style="text-align:left;">
gene
</td>
<td style="text-align:right;">
138.3161
</td>
<td style="text-align:right;">
138.3162
</td>
<td style="text-align:left;">
negative
</td>
<td style="text-align:left;">
MGI_C57BL6J_5531329
</td>
<td style="text-align:left;">
Mir7019
</td>
<td style="text-align:left;">
miRBase:MI0022868,NCBI_Gene:102466788,ENSEMBL:ENSMUSG00000099200
</td>
<td style="text-align:left;">
MGI:5531329
</td>
<td style="text-align:left;">
miRNA gene
</td>
<td style="text-align:left;">
microRNA 7019
</td>
</tr>
<tr>
<td style="text-align:left;">
4
</td>
<td style="text-align:left;">
gene
</td>
<td style="text-align:right;">
138.3260
</td>
<td style="text-align:right;">
138.3261
</td>
<td style="text-align:left;">
negative
</td>
<td style="text-align:left;">
MGI_C57BL6J_5531028
</td>
<td style="text-align:left;">
Gm27646
</td>
<td style="text-align:left;">
ENSEMBL:ENSMUSG00000098364
</td>
<td style="text-align:left;">
MGI:5531028
</td>
<td style="text-align:left;">
miRNA gene
</td>
<td style="text-align:left;">
predicted gene, 27646
</td>
</tr>
<tr>
<td style="text-align:left;">
4
</td>
<td style="text-align:left;">
gene
</td>
<td style="text-align:right;">
138.3384
</td>
<td style="text-align:right;">
138.3683
</td>
<td style="text-align:left;">
negative
</td>
<td style="text-align:left;">
MGI_C57BL6J_1919519
</td>
<td style="text-align:left;">
Cda
</td>
<td style="text-align:left;">
NCBI_Gene:72269,ENSEMBL:ENSMUSG00000028755
</td>
<td style="text-align:left;">
MGI:1919519
</td>
<td style="text-align:left;">
protein coding gene
</td>
<td style="text-align:left;">
cytidine deaminase
</td>
</tr>
<tr>
<td style="text-align:left;">
4
</td>
<td style="text-align:left;">
gene
</td>
<td style="text-align:right;">
138.3941
</td>
<td style="text-align:right;">
138.3965
</td>
<td style="text-align:left;">
negative
</td>
<td style="text-align:left;">
MGI_C57BL6J_3651622
</td>
<td style="text-align:left;">
Fam43b
</td>
<td style="text-align:left;">
NCBI_Gene:625638,ENSEMBL:ENSMUSG00000078235
</td>
<td style="text-align:left;">
MGI:3651622
</td>
<td style="text-align:left;">
protein coding gene
</td>
<td style="text-align:left;">
family with sequence similarity 43, member B
</td>
</tr>
<tr>
<td style="text-align:left;">
4
</td>
<td style="text-align:left;">
gene
</td>
<td style="text-align:right;">
138.3952
</td>
<td style="text-align:right;">
138.3977
</td>
<td style="text-align:left;">
positive
</td>
<td style="text-align:left;">
MGI_C57BL6J_1931024
</td>
<td style="text-align:left;">
AB041806
</td>
<td style="text-align:left;">
ENSEMBL:ENSMUSG00000046109
</td>
<td style="text-align:left;">
MGI:1931024
</td>
<td style="text-align:left;">
lncRNA gene
</td>
<td style="text-align:left;">
hypothetical protein, MNCb-2457
</td>
</tr>
<tr>
<td style="text-align:left;">
4
</td>
<td style="text-align:left;">
pseudogene
</td>
<td style="text-align:right;">
138.4257
</td>
<td style="text-align:right;">
138.4260
</td>
<td style="text-align:left;">
positive
</td>
<td style="text-align:left;">
MGI_C57BL6J_3649580
</td>
<td style="text-align:left;">
Rpl38-ps1
</td>
<td style="text-align:left;">
NCBI_Gene:625646,ENSEMBL:ENSMUSG00000083326
</td>
<td style="text-align:left;">
MGI:3649580
</td>
<td style="text-align:left;">
pseudogene
</td>
<td style="text-align:left;">
ribosomal protein L38, pseudogene 1
</td>
</tr>
<tr>
<td style="text-align:left;">
4
</td>
<td style="text-align:left;">
gene
</td>
<td style="text-align:right;">
138.4347
</td>
<td style="text-align:right;">
138.4423
</td>
<td style="text-align:left;">
positive
</td>
<td style="text-align:left;">
MGI_C57BL6J_1915600
</td>
<td style="text-align:left;">
Mul1
</td>
<td style="text-align:left;">
NCBI_Gene:68350,ENSEMBL:ENSMUSG00000041241
</td>
<td style="text-align:left;">
MGI:1915600
</td>
<td style="text-align:left;">
protein coding gene
</td>
<td style="text-align:left;">
mitochondrial ubiquitin ligase activator of NFKB 1
</td>
</tr>
<tr>
<td style="text-align:left;">
4
</td>
<td style="text-align:left;">
gene
</td>
<td style="text-align:right;">
138.4543
</td>
<td style="text-align:right;">
138.4601
</td>
<td style="text-align:left;">
positive
</td>
<td style="text-align:left;">
MGI_C57BL6J_1913509
</td>
<td style="text-align:left;">
Camk2n1
</td>
<td style="text-align:left;">
NCBI_Gene:66259,ENSEMBL:ENSMUSG00000046447
</td>
<td style="text-align:left;">
MGI:1913509
</td>
<td style="text-align:left;">
protein coding gene
</td>
<td style="text-align:left;">
calcium/calmodulin-dependent protein kinase II inhibitor 1
</td>
</tr>
<tr>
<td style="text-align:left;">
4
</td>
<td style="text-align:left;">
gene
</td>
<td style="text-align:right;">
138.4976
</td>
<td style="text-align:right;">
138.5285
</td>
<td style="text-align:left;">
positive
</td>
<td style="text-align:left;">
MGI_C57BL6J_5591708
</td>
<td style="text-align:left;">
Gm32549
</td>
<td style="text-align:left;">
NCBI_Gene:102635131
</td>
<td style="text-align:left;">
MGI:5591708
</td>
<td style="text-align:left;">
lncRNA gene
</td>
<td style="text-align:left;">
predicted gene, 32549
</td>
</tr>
<tr>
<td style="text-align:left;">
4
</td>
<td style="text-align:left;">
gene
</td>
<td style="text-align:right;">
138.5642
</td>
<td style="text-align:right;">
138.6359
</td>
<td style="text-align:left;">
negative
</td>
<td style="text-align:left;">
MGI_C57BL6J_1922968
</td>
<td style="text-align:left;">
Vwa5b1
</td>
<td style="text-align:left;">
NCBI_Gene:75718,ENSEMBL:ENSMUSG00000028753
</td>
<td style="text-align:left;">
MGI:1922968
</td>
<td style="text-align:left;">
protein coding gene
</td>
<td style="text-align:left;">
von Willebrand factor A domain containing 5B1
</td>
</tr>
<tr>
<td style="text-align:left;">
4
</td>
<td style="text-align:left;">
gene
</td>
<td style="text-align:right;">
138.5876
</td>
<td style="text-align:right;">
138.5877
</td>
<td style="text-align:left;">
negative
</td>
<td style="text-align:left;">
MGI_C57BL6J_5452738
</td>
<td style="text-align:left;">
Gm22961
</td>
<td style="text-align:left;">
ENSEMBL:ENSMUSG00000088589
</td>
<td style="text-align:left;">
MGI:5452738
</td>
<td style="text-align:left;">
snRNA gene
</td>
<td style="text-align:left;">
predicted gene, 22961
</td>
</tr>
<tr>
<td style="text-align:left;">
4
</td>
<td style="text-align:left;">
gene
</td>
<td style="text-align:right;">
138.6390
</td>
<td style="text-align:right;">
138.6576
</td>
<td style="text-align:left;">
positive
</td>
<td style="text-align:left;">
MGI_C57BL6J_5591821
</td>
<td style="text-align:left;">
Gm32662
</td>
<td style="text-align:left;">
NCBI_Gene:102635281
</td>
<td style="text-align:left;">
MGI:5591821
</td>
<td style="text-align:left;">
lncRNA gene
</td>
<td style="text-align:left;">
predicted gene, 32662
</td>
</tr>
<tr>
<td style="text-align:left;">
4
</td>
<td style="text-align:left;">
gene
</td>
<td style="text-align:right;">
138.6775
</td>
<td style="text-align:right;">
138.7117
</td>
<td style="text-align:left;">
positive
</td>
<td style="text-align:left;">
MGI_C57BL6J_5591929
</td>
<td style="text-align:left;">
Gm32770
</td>
<td style="text-align:left;">
NCBI_Gene:102635431
</td>
<td style="text-align:left;">
MGI:5591929
</td>
<td style="text-align:left;">
lncRNA gene
</td>
<td style="text-align:left;">
predicted gene, 32770
</td>
</tr>
<tr>
<td style="text-align:left;">
4
</td>
<td style="text-align:left;">
gene
</td>
<td style="text-align:right;">
138.6778
</td>
<td style="text-align:right;">
138.6782
</td>
<td style="text-align:left;">
positive
</td>
<td style="text-align:left;">
MGI_C57BL6J_1914908
</td>
<td style="text-align:left;">
4930563F15Rik
</td>
<td style="text-align:left;">
NA
</td>
<td style="text-align:left;">
NA
</td>
<td style="text-align:left;">
unclassified gene
</td>
<td style="text-align:left;">
RIKEN cDNA 4930563F15 gene
</td>
</tr>
<tr>
<td style="text-align:left;">
4
</td>
<td style="text-align:left;">
gene
</td>
<td style="text-align:right;">
138.7098
</td>
<td style="text-align:right;">
138.7384
</td>
<td style="text-align:left;">
negative
</td>
<td style="text-align:left;">
MGI_C57BL6J_2443123
</td>
<td style="text-align:left;">
Ubxn10
</td>
<td style="text-align:left;">
NCBI_Gene:212190,ENSEMBL:ENSMUSG00000043621
</td>
<td style="text-align:left;">
MGI:2443123
</td>
<td style="text-align:left;">
protein coding gene
</td>
<td style="text-align:left;">
UBX domain protein 10
</td>
</tr>
<tr>
<td style="text-align:left;">
4
</td>
<td style="text-align:left;">
gene
</td>
<td style="text-align:right;">
138.7248
</td>
<td style="text-align:right;">
138.7461
</td>
<td style="text-align:left;">
positive
</td>
<td style="text-align:left;">
MGI_C57BL6J_106638
</td>
<td style="text-align:left;">
Pla2g2c
</td>
<td style="text-align:left;">
NCBI_Gene:18781,ENSEMBL:ENSMUSG00000028750
</td>
<td style="text-align:left;">
MGI:106638
</td>
<td style="text-align:left;">
protein coding gene
</td>
<td style="text-align:left;">
phospholipase A2, group IIC
</td>
</tr>
<tr>
<td style="text-align:left;">
4
</td>
<td style="text-align:left;">
gene
</td>
<td style="text-align:right;">
138.7505
</td>
<td style="text-align:right;">
138.7576
</td>
<td style="text-align:left;">
negative
</td>
<td style="text-align:left;">
MGI_C57BL6J_1349661
</td>
<td style="text-align:left;">
Pla2g2f
</td>
<td style="text-align:left;">
NCBI_Gene:26971,ENSEMBL:ENSMUSG00000028749
</td>
<td style="text-align:left;">
MGI:1349661
</td>
<td style="text-align:left;">
protein coding gene
</td>
<td style="text-align:left;">
phospholipase A2, group IIF
</td>
</tr>
<tr>
<td style="text-align:left;">
4
</td>
<td style="text-align:left;">
gene
</td>
<td style="text-align:right;">
138.7614
</td>
<td style="text-align:right;">
138.7697
</td>
<td style="text-align:left;">
negative
</td>
<td style="text-align:left;">
MGI_C57BL6J_5625217
</td>
<td style="text-align:left;">
Gm42332
</td>
<td style="text-align:left;">
NCBI_Gene:105247193
</td>
<td style="text-align:left;">
MGI:5625217
</td>
<td style="text-align:left;">
lncRNA gene
</td>
<td style="text-align:left;">
predicted gene, 42332
</td>
</tr>
<tr>
<td style="text-align:left;">
4
</td>
<td style="text-align:left;">
gene
</td>
<td style="text-align:right;">
138.7725
</td>
<td style="text-align:right;">
138.7802
</td>
<td style="text-align:left;">
negative
</td>
<td style="text-align:left;">
MGI_C57BL6J_5592030
</td>
<td style="text-align:left;">
Gm32871
</td>
<td style="text-align:left;">
NCBI_Gene:102635577
</td>
<td style="text-align:left;">
MGI:5592030
</td>
<td style="text-align:left;">
lncRNA gene
</td>
<td style="text-align:left;">
predicted gene, 32871
</td>
</tr>
<tr>
<td style="text-align:left;">
4
</td>
<td style="text-align:left;">
gene
</td>
<td style="text-align:right;">
138.7757
</td>
<td style="text-align:right;">
138.7821
</td>
<td style="text-align:left;">
positive
</td>
<td style="text-align:left;">
MGI_C57BL6J_1341796
</td>
<td style="text-align:left;">
Pla2g2d
</td>
<td style="text-align:left;">
NCBI_Gene:18782,ENSEMBL:ENSMUSG00000041202
</td>
<td style="text-align:left;">
MGI:1341796
</td>
<td style="text-align:left;">
protein coding gene
</td>
<td style="text-align:left;">
phospholipase A2, group IID
</td>
</tr>
<tr>
<td style="text-align:left;">
4
</td>
<td style="text-align:left;">
gene
</td>
<td style="text-align:right;">
138.7992
</td>
<td style="text-align:right;">
138.8635
</td>
<td style="text-align:left;">
negative
</td>
<td style="text-align:left;">
MGI_C57BL6J_101899
</td>
<td style="text-align:left;">
Pla2g5
</td>
<td style="text-align:left;">
NCBI_Gene:18784,ENSEMBL:ENSMUSG00000041193
</td>
<td style="text-align:left;">
MGI:101899
</td>
<td style="text-align:left;">
protein coding gene
</td>
<td style="text-align:left;">
phospholipase A2, group V
</td>
</tr>
<tr>
<td style="text-align:left;">
4
</td>
<td style="text-align:left;">
gene
</td>
<td style="text-align:right;">
138.8274
</td>
<td style="text-align:right;">
138.8275
</td>
<td style="text-align:left;">
negative
</td>
<td style="text-align:left;">
MGI_C57BL6J_5455057
</td>
<td style="text-align:left;">
Gm25280
</td>
<td style="text-align:left;">
ENSEMBL:ENSMUSG00000087894
</td>
<td style="text-align:left;">
MGI:5455057
</td>
<td style="text-align:left;">
snRNA gene
</td>
<td style="text-align:left;">
predicted gene, 25280
</td>
</tr>
<tr>
<td style="text-align:left;">
4
</td>
<td style="text-align:left;">
pseudogene
</td>
<td style="text-align:right;">
138.8319
</td>
<td style="text-align:right;">
138.8352
</td>
<td style="text-align:left;">
positive
</td>
<td style="text-align:left;">
MGI_C57BL6J_104642
</td>
<td style="text-align:left;">
Pla2g2a
</td>
<td style="text-align:left;">
NCBI_Gene:18780,ENSEMBL:ENSMUSG00000058908
</td>
<td style="text-align:left;">
MGI:104642
</td>
<td style="text-align:left;">
polymorphic pseudogene
</td>
<td style="text-align:left;">
phospholipase A2, group IIA (platelets, synovial fluid)
</td>
</tr>
<tr>
<td style="text-align:left;">
4
</td>
<td style="text-align:left;">
gene
</td>
<td style="text-align:right;">
138.8635
</td>
<td style="text-align:right;">
138.8686
</td>
<td style="text-align:left;">
positive
</td>
<td style="text-align:left;">
MGI_C57BL6J_5592089
</td>
<td style="text-align:left;">
Gm32930
</td>
<td style="text-align:left;">
NCBI_Gene:102635643
</td>
<td style="text-align:left;">
MGI:5592089
</td>
<td style="text-align:left;">
lncRNA gene
</td>
<td style="text-align:left;">
predicted gene, 32930
</td>
</tr>
<tr>
<td style="text-align:left;">
4
</td>
<td style="text-align:left;">
gene
</td>
<td style="text-align:right;">
138.8710
</td>
<td style="text-align:right;">
138.8740
</td>
<td style="text-align:left;">
negative
</td>
<td style="text-align:left;">
MGI_C57BL6J_3651765
</td>
<td style="text-align:left;">
Gm13030
</td>
<td style="text-align:left;">
NCBI_Gene:105734733,ENSEMBL:ENSMUSG00000078518
</td>
<td style="text-align:left;">
MGI:3651765
</td>
<td style="text-align:left;">
protein coding gene
</td>
<td style="text-align:left;">
predicted gene 13030
</td>
</tr>
<tr>
<td style="text-align:left;">
4
</td>
<td style="text-align:left;">
gene
</td>
<td style="text-align:right;">
138.8779
</td>
<td style="text-align:right;">
138.8828
</td>
<td style="text-align:left;">
positive
</td>
<td style="text-align:left;">
MGI_C57BL6J_1349660
</td>
<td style="text-align:left;">
Pla2g2e
</td>
<td style="text-align:left;">
NCBI_Gene:26970,ENSEMBL:ENSMUSG00000028751
</td>
<td style="text-align:left;">
MGI:1349660
</td>
<td style="text-align:left;">
protein coding gene
</td>
<td style="text-align:left;">
phospholipase A2, group IIE
</td>
</tr>
<tr>
<td style="text-align:left;">
4
</td>
<td style="text-align:left;">
gene
</td>
<td style="text-align:right;">
138.8911
</td>
<td style="text-align:right;">
138.9140
</td>
<td style="text-align:left;">
negative
</td>
<td style="text-align:left;">
MGI_C57BL6J_1920412
</td>
<td style="text-align:left;">
Otud3
</td>
<td style="text-align:left;">
NCBI_Gene:73162,ENSEMBL:ENSMUSG00000041161
</td>
<td style="text-align:left;">
MGI:1920412
</td>
<td style="text-align:left;">
protein coding gene
</td>
<td style="text-align:left;">
OTU domain containing 3
</td>
</tr>
<tr>
<td style="text-align:left;">
4
</td>
<td style="text-align:left;">
pseudogene
</td>
<td style="text-align:right;">
138.9352
</td>
<td style="text-align:right;">
138.9357
</td>
<td style="text-align:left;">
negative
</td>
<td style="text-align:left;">
MGI_C57BL6J_5010282
</td>
<td style="text-align:left;">
Gm18097
</td>
<td style="text-align:left;">
NCBI_Gene:100416403
</td>
<td style="text-align:left;">
MGI:5010282
</td>
<td style="text-align:left;">
pseudogene
</td>
<td style="text-align:left;">
predicted gene, 18097
</td>
</tr>
<tr>
<td style="text-align:left;">
4
</td>
<td style="text-align:left;">
gene
</td>
<td style="text-align:right;">
138.9364
</td>
<td style="text-align:right;">
138.9386
</td>
<td style="text-align:left;">
negative
</td>
<td style="text-align:left;">
MGI_C57BL6J_5592170
</td>
<td style="text-align:left;">
Gm33011
</td>
<td style="text-align:left;">
NCBI_Gene:102635752
</td>
<td style="text-align:left;">
MGI:5592170
</td>
<td style="text-align:left;">
lncRNA gene
</td>
<td style="text-align:left;">
predicted gene, 33011
</td>
</tr>
<tr>
<td style="text-align:left;">
4
</td>
<td style="text-align:left;">
gene
</td>
<td style="text-align:right;">
138.9619
</td>
<td style="text-align:right;">
138.9647
</td>
<td style="text-align:left;">
negative
</td>
<td style="text-align:left;">
MGI_C57BL6J_5625218
</td>
<td style="text-align:left;">
Gm42333
</td>
<td style="text-align:left;">
NCBI_Gene:105247194
</td>
<td style="text-align:left;">
MGI:5625218
</td>
<td style="text-align:left;">
lncRNA gene
</td>
<td style="text-align:left;">
predicted gene, 42333
</td>
</tr>
<tr>
<td style="text-align:left;">
4
</td>
<td style="text-align:left;">
gene
</td>
<td style="text-align:right;">
138.9671
</td>
<td style="text-align:right;">
138.9684
</td>
<td style="text-align:left;">
positive
</td>
<td style="text-align:left;">
MGI_C57BL6J_1914075
</td>
<td style="text-align:left;">
Rnf186
</td>
<td style="text-align:left;">
NCBI_Gene:66825,ENSEMBL:ENSMUSG00000070661
</td>
<td style="text-align:left;">
MGI:1914075
</td>
<td style="text-align:left;">
protein coding gene
</td>
<td style="text-align:left;">
ring finger protein 186
</td>
</tr>
<tr>
<td style="text-align:left;">
4
</td>
<td style="text-align:left;">
gene
</td>
<td style="text-align:right;">
138.9721
</td>
<td style="text-align:right;">
139.0592
</td>
<td style="text-align:left;">
positive
</td>
<td style="text-align:left;">
MGI_C57BL6J_1924306
</td>
<td style="text-align:left;">
Tmco4
</td>
<td style="text-align:left;">
NCBI_Gene:77056,ENSEMBL:ENSMUSG00000041143
</td>
<td style="text-align:left;">
MGI:1924306
</td>
<td style="text-align:left;">
protein coding gene
</td>
<td style="text-align:left;">
transmembrane and coiled-coil domains 4
</td>
</tr>
<tr>
<td style="text-align:left;">
4
</td>
<td style="text-align:left;">
gene
</td>
<td style="text-align:right;">
139.0032
</td>
<td style="text-align:right;">
139.0196
</td>
<td style="text-align:left;">
negative
</td>
<td style="text-align:left;">
MGI_C57BL6J_5592373
</td>
<td style="text-align:left;">
Gm33214
</td>
<td style="text-align:left;">
NCBI_Gene:102636025
</td>
<td style="text-align:left;">
MGI:5592373
</td>
<td style="text-align:left;">
lncRNA gene
</td>
<td style="text-align:left;">
predicted gene, 33214
</td>
</tr>
<tr>
<td style="text-align:left;">
4
</td>
<td style="text-align:left;">
gene
</td>
<td style="text-align:right;">
139.0597
</td>
<td style="text-align:right;">
139.0756
</td>
<td style="text-align:left;">
negative
</td>
<td style="text-align:left;">
MGI_C57BL6J_1196627
</td>
<td style="text-align:left;">
Htr6
</td>
<td style="text-align:left;">
NCBI_Gene:15565,ENSEMBL:ENSMUSG00000028747
</td>
<td style="text-align:left;">
MGI:1196627
</td>
<td style="text-align:left;">
protein coding gene
</td>
<td style="text-align:left;">
5-hydroxytryptamine (serotonin) receptor 6
</td>
</tr>
<tr>
<td style="text-align:left;">
4
</td>
<td style="text-align:left;">
gene
</td>
<td style="text-align:right;">
139.0749
</td>
<td style="text-align:right;">
139.0772
</td>
<td style="text-align:left;">
positive
</td>
<td style="text-align:left;">
MGI_C57BL6J_5791369
</td>
<td style="text-align:left;">
Gm45533
</td>
<td style="text-align:left;">
NCBI_Gene:108168960,ENSEMBL:ENSMUSG00000110296
</td>
<td style="text-align:left;">
MGI:5791369
</td>
<td style="text-align:left;">
lncRNA gene
</td>
<td style="text-align:left;">
predicted gene 45533
</td>
</tr>
<tr>
<td style="text-align:left;">
4
</td>
<td style="text-align:left;">
gene
</td>
<td style="text-align:right;">
139.0823
</td>
<td style="text-align:right;">
139.0930
</td>
<td style="text-align:left;">
negative
</td>
<td style="text-align:left;">
MGI_C57BL6J_104591
</td>
<td style="text-align:left;">
Nbl1
</td>
<td style="text-align:left;">
NCBI_Gene:17965,ENSEMBL:ENSMUSG00000041120
</td>
<td style="text-align:left;">
MGI:104591
</td>
<td style="text-align:left;">
protein coding gene
</td>
<td style="text-align:left;">
NBL1, DAN family BMP antagonist
</td>
</tr>
<tr>
<td style="text-align:left;">
4
</td>
<td style="text-align:left;">
gene
</td>
<td style="text-align:right;">
139.1018
</td>
<td style="text-align:right;">
139.1311
</td>
<td style="text-align:left;">
negative
</td>
<td style="text-align:left;">
MGI_C57BL6J_1913628
</td>
<td style="text-align:left;">
Micos10
</td>
<td style="text-align:left;">
NCBI_Gene:433771,ENSEMBL:ENSMUSG00000050608
</td>
<td style="text-align:left;">
MGI:1913628
</td>
<td style="text-align:left;">
protein coding gene
</td>
<td style="text-align:left;">
mitochondrial contact site and cristae organizing system subunit 10
</td>
</tr>
<tr>
<td style="text-align:left;">
4
</td>
<td style="text-align:left;">
gene
</td>
<td style="text-align:right;">
139.1341
</td>
<td style="text-align:right;">
139.1653
</td>
<td style="text-align:left;">
negative
</td>
<td style="text-align:left;">
MGI_C57BL6J_5592463
</td>
<td style="text-align:left;">
Gm33304
</td>
<td style="text-align:left;">
NCBI_Gene:102636159
</td>
<td style="text-align:left;">
MGI:5592463
</td>
<td style="text-align:left;">
lncRNA gene
</td>
<td style="text-align:left;">
predicted gene, 33304
</td>
</tr>
<tr>
<td style="text-align:left;">
4
</td>
<td style="text-align:left;">
pseudogene
</td>
<td style="text-align:right;">
139.1502
</td>
<td style="text-align:right;">
139.1508
</td>
<td style="text-align:left;">
positive
</td>
<td style="text-align:left;">
MGI_C57BL6J_1935164
</td>
<td style="text-align:left;">
Hspe1-ps4
</td>
<td style="text-align:left;">
NCBI_Gene:93751,ENSEMBL:ENSMUSG00000083238
</td>
<td style="text-align:left;">
MGI:1935164
</td>
<td style="text-align:left;">
pseudogene
</td>
<td style="text-align:left;">
heat shock protein 1 (chaperonin 10), pseudogene 4
</td>
</tr>
<tr>
<td style="text-align:left;">
4
</td>
<td style="text-align:left;">
gene
</td>
<td style="text-align:right;">
139.1754
</td>
<td style="text-align:right;">
139.1807
</td>
<td style="text-align:left;">
negative
</td>
<td style="text-align:left;">
MGI_C57BL6J_3650322
</td>
<td style="text-align:left;">
Gm16287
</td>
<td style="text-align:left;">
NCBI_Gene:100038595,ENSEMBL:ENSMUSG00000073739
</td>
<td style="text-align:left;">
MGI:3650322
</td>
<td style="text-align:left;">
lncRNA gene
</td>
<td style="text-align:left;">
predicted gene 16287
</td>
</tr>
<tr>
<td style="text-align:left;">
4
</td>
<td style="text-align:left;">
gene
</td>
<td style="text-align:right;">
139.1922
</td>
<td style="text-align:right;">
139.1927
</td>
<td style="text-align:left;">
negative
</td>
<td style="text-align:left;">
MGI_C57BL6J_1925704
</td>
<td style="text-align:left;">
9530077C14Rik
</td>
<td style="text-align:left;">
NA
</td>
<td style="text-align:left;">
NA
</td>
<td style="text-align:left;">
unclassified gene
</td>
<td style="text-align:left;">
RIKEN cDNA 9530077C14 gene
</td>
</tr>
<tr>
<td style="text-align:left;">
4
</td>
<td style="text-align:left;">
gene
</td>
<td style="text-align:right;">
139.1929
</td>
<td style="text-align:right;">
139.2918
</td>
<td style="text-align:left;">
positive
</td>
<td style="text-align:left;">
MGI_C57BL6J_104652
</td>
<td style="text-align:left;">
Capzb
</td>
<td style="text-align:left;">
NCBI_Gene:12345,ENSEMBL:ENSMUSG00000028745
</td>
<td style="text-align:left;">
MGI:104652
</td>
<td style="text-align:left;">
protein coding gene
</td>
<td style="text-align:left;">
capping protein (actin filament) muscle Z-line, beta
</td>
</tr>
<tr>
<td style="text-align:left;">
4
</td>
<td style="text-align:left;">
gene
</td>
<td style="text-align:right;">
139.2940
</td>
<td style="text-align:right;">
139.3107
</td>
<td style="text-align:left;">
negative
</td>
<td style="text-align:left;">
MGI_C57BL6J_2384837
</td>
<td style="text-align:left;">
Pqlc2
</td>
<td style="text-align:left;">
NCBI_Gene:212555,ENSEMBL:ENSMUSG00000028744
</td>
<td style="text-align:left;">
MGI:2384837
</td>
<td style="text-align:left;">
protein coding gene
</td>
<td style="text-align:left;">
PQ loop repeat containing 2
</td>
</tr>
<tr>
<td style="text-align:left;">
4
</td>
<td style="text-align:left;">
gene
</td>
<td style="text-align:right;">
139.3107
</td>
<td style="text-align:right;">
139.3188
</td>
<td style="text-align:left;">
positive
</td>
<td style="text-align:left;">
MGI_C57BL6J_107796
</td>
<td style="text-align:left;">
Akr7a5
</td>
<td style="text-align:left;">
NCBI_Gene:110198,ENSEMBL:ENSMUSG00000028743
</td>
<td style="text-align:left;">
MGI:107796
</td>
<td style="text-align:left;">
protein coding gene
</td>
<td style="text-align:left;">
aldo-keto reductase family 7, member A5 (aflatoxin aldehyde reductase)
</td>
</tr>
<tr>
<td style="text-align:left;">
4
</td>
<td style="text-align:left;">
gene
</td>
<td style="text-align:right;">
139.3382
</td>
<td style="text-align:right;">
139.3383
</td>
<td style="text-align:left;">
positive
</td>
<td style="text-align:left;">
MGI_C57BL6J_5455509
</td>
<td style="text-align:left;">
Gm25732
</td>
<td style="text-align:left;">
ENSEMBL:ENSMUSG00000092909
</td>
<td style="text-align:left;">
MGI:5455509
</td>
<td style="text-align:left;">
miRNA gene
</td>
<td style="text-align:left;">
predicted gene, 25732
</td>
</tr>
<tr>
<td style="text-align:left;">
4
</td>
<td style="text-align:left;">
gene
</td>
<td style="text-align:right;">
139.3383
</td>
<td style="text-align:right;">
139.3384
</td>
<td style="text-align:left;">
positive
</td>
<td style="text-align:left;">
MGI_C57BL6J_5454022
</td>
<td style="text-align:left;">
Gm24245
</td>
<td style="text-align:left;">
ENSEMBL:ENSMUSG00000087943
</td>
<td style="text-align:left;">
MGI:5454022
</td>
<td style="text-align:left;">
miRNA gene
</td>
<td style="text-align:left;">
predicted gene, 24245
</td>
</tr>
<tr>
<td style="text-align:left;">
4
</td>
<td style="text-align:left;">
pseudogene
</td>
<td style="text-align:right;">
139.3422
</td>
<td style="text-align:right;">
139.3433
</td>
<td style="text-align:left;">
positive
</td>
<td style="text-align:left;">
MGI_C57BL6J_3652310
</td>
<td style="text-align:left;">
Gm13019
</td>
<td style="text-align:left;">
ENSEMBL:ENSMUSG00000083404
</td>
<td style="text-align:left;">
MGI:3652310
</td>
<td style="text-align:left;">
pseudogene
</td>
<td style="text-align:left;">
predicted gene 13019
</td>
</tr>
<tr>
<td style="text-align:left;">
4
</td>
<td style="text-align:left;">
pseudogene
</td>
<td style="text-align:right;">
139.3444
</td>
<td style="text-align:right;">
139.3463
</td>
<td style="text-align:left;">
negative
</td>
<td style="text-align:left;">
MGI_C57BL6J_1100505
</td>
<td style="text-align:left;">
Ccnd3-ps
</td>
<td style="text-align:left;">
NCBI_Gene:626000,ENSEMBL:ENSMUSG00000080708
</td>
<td style="text-align:left;">
MGI:1100505
</td>
<td style="text-align:left;">
pseudogene
</td>
<td style="text-align:left;">
cyclin D3, pseudogene
</td>
</tr>
<tr>
<td style="text-align:left;">
4
</td>
<td style="text-align:left;">
gene
</td>
<td style="text-align:right;">
139.3474
</td>
<td style="text-align:right;">
139.3526
</td>
<td style="text-align:left;">
negative
</td>
<td style="text-align:left;">
MGI_C57BL6J_1917152
</td>
<td style="text-align:left;">
Mrto4
</td>
<td style="text-align:left;">
NCBI_Gene:69902,ENSEMBL:ENSMUSG00000028741
</td>
<td style="text-align:left;">
MGI:1917152
</td>
<td style="text-align:left;">
protein coding gene
</td>
<td style="text-align:left;">
mRNA turnover 4, ribosome maturation factor
</td>
</tr>
<tr>
<td style="text-align:left;">
4
</td>
<td style="text-align:left;">
gene
</td>
<td style="text-align:right;">
139.3526
</td>
<td style="text-align:right;">
139.3787
</td>
<td style="text-align:left;">
positive
</td>
<td style="text-align:left;">
MGI_C57BL6J_2443696
</td>
<td style="text-align:left;">
Emc1
</td>
<td style="text-align:left;">
NCBI_Gene:230866,ENSEMBL:ENSMUSG00000078517
</td>
<td style="text-align:left;">
MGI:2443696
</td>
<td style="text-align:left;">
protein coding gene
</td>
<td style="text-align:left;">
ER membrane protein complex subunit 1
</td>
</tr>
<tr>
<td style="text-align:left;">
4
</td>
<td style="text-align:left;">
gene
</td>
<td style="text-align:right;">
139.3526
</td>
<td style="text-align:right;">
139.4896
</td>
<td style="text-align:left;">
positive
</td>
<td style="text-align:left;">
MGI_C57BL6J_1916366
</td>
<td style="text-align:left;">
Ubr4
</td>
<td style="text-align:left;">
NCBI_Gene:69116,ENSEMBL:ENSMUSG00000066036
</td>
<td style="text-align:left;">
MGI:1916366
</td>
<td style="text-align:left;">
protein coding gene
</td>
<td style="text-align:left;">
ubiquitin protein ligase E3 component n-recognin 4
</td>
</tr>
<tr>
<td style="text-align:left;">
4
</td>
<td style="text-align:left;">
gene
</td>
<td style="text-align:right;">
139.3765
</td>
<td style="text-align:right;">
139.3804
</td>
<td style="text-align:left;">
negative
</td>
<td style="text-align:left;">
MGI_C57BL6J_1919807
</td>
<td style="text-align:left;">
2700016F22Rik
</td>
<td style="text-align:left;">
ENSEMBL:ENSMUSG00000097731
</td>
<td style="text-align:left;">
MGI:1919807
</td>
<td style="text-align:left;">
lncRNA gene
</td>
<td style="text-align:left;">
RIKEN cDNA 2700016F22 gene
</td>
</tr>
<tr>
<td style="text-align:left;">
4
</td>
<td style="text-align:left;">
gene
</td>
<td style="text-align:right;">
139.4305
</td>
<td style="text-align:right;">
139.4306
</td>
<td style="text-align:left;">
negative
</td>
<td style="text-align:left;">
MGI_C57BL6J_4414084
</td>
<td style="text-align:left;">
n-TVaac1
</td>
<td style="text-align:left;">
NCBI_Gene:102467443
</td>
<td style="text-align:left;">
MGI:4414084
</td>
<td style="text-align:left;">
tRNA gene
</td>
<td style="text-align:left;">
nuclear encoded tRNA valine 1 (anticodon AAC)
</td>
</tr>
<tr>
<td style="text-align:left;">
4
</td>
<td style="text-align:left;">
gene
</td>
<td style="text-align:right;">
139.5305
</td>
<td style="text-align:right;">
139.6204
</td>
<td style="text-align:left;">
positive
</td>
<td style="text-align:left;">
MGI_C57BL6J_2140675
</td>
<td style="text-align:left;">
Iffo2
</td>
<td style="text-align:left;">
NCBI_Gene:212632,ENSEMBL:ENSMUSG00000041025
</td>
<td style="text-align:left;">
MGI:2140675
</td>
<td style="text-align:left;">
protein coding gene
</td>
<td style="text-align:left;">
intermediate filament family orphan 2
</td>
</tr>
<tr>
<td style="text-align:left;">
4
</td>
<td style="text-align:left;">
gene
</td>
<td style="text-align:right;">
139.5889
</td>
<td style="text-align:right;">
139.5890
</td>
<td style="text-align:left;">
negative
</td>
<td style="text-align:left;">
MGI_C57BL6J_5455348
</td>
<td style="text-align:left;">
Gm25571
</td>
<td style="text-align:left;">
ENSEMBL:ENSMUSG00000094359
</td>
<td style="text-align:left;">
MGI:5455348
</td>
<td style="text-align:left;">
miRNA gene
</td>
<td style="text-align:left;">
predicted gene, 25571
</td>
</tr>
<tr>
<td style="text-align:left;">
4
</td>
<td style="text-align:left;">
gene
</td>
<td style="text-align:right;">
139.6032
</td>
<td style="text-align:right;">
139.6627
</td>
<td style="text-align:left;">
positive
</td>
<td style="text-align:left;">
MGI_C57BL6J_5439438
</td>
<td style="text-align:left;">
Gm21969
</td>
<td style="text-align:left;">
ENSEMBL:ENSMUSG00000094439
</td>
<td style="text-align:left;">
MGI:5439438
</td>
<td style="text-align:left;">
protein coding gene
</td>
<td style="text-align:left;">
predicted gene 21969
</td>
</tr>
<tr>
<td style="text-align:left;">
4
</td>
<td style="text-align:left;">
gene
</td>
<td style="text-align:right;">
139.6229
</td>
<td style="text-align:right;">
139.6497
</td>
<td style="text-align:left;">
positive
</td>
<td style="text-align:left;">
MGI_C57BL6J_2443883
</td>
<td style="text-align:left;">
Aldh4a1
</td>
<td style="text-align:left;">
NCBI_Gene:212647,ENSEMBL:ENSMUSG00000028737
</td>
<td style="text-align:left;">
MGI:2443883
</td>
<td style="text-align:left;">
protein coding gene
</td>
<td style="text-align:left;">
aldehyde dehydrogenase 4 family, member A1
</td>
</tr>
<tr>
<td style="text-align:left;">
4
</td>
<td style="text-align:left;">
gene
</td>
<td style="text-align:right;">
139.6440
</td>
<td style="text-align:right;">
139.6441
</td>
<td style="text-align:left;">
positive
</td>
<td style="text-align:left;">
MGI_C57BL6J_5531141
</td>
<td style="text-align:left;">
Mir7020
</td>
<td style="text-align:left;">
miRBase:MI0022869,NCBI_Gene:102465617,ENSEMBL:ENSMUSG00000098569
</td>
<td style="text-align:left;">
MGI:5531141
</td>
<td style="text-align:left;">
miRNA gene
</td>
<td style="text-align:left;">
microRNA 7020
</td>
</tr>
<tr>
<td style="text-align:left;">
4
</td>
<td style="text-align:left;">
gene
</td>
<td style="text-align:right;">
139.6535
</td>
<td style="text-align:right;">
139.6703
</td>
<td style="text-align:left;">
positive
</td>
<td style="text-align:left;">
MGI_C57BL6J_1933546
</td>
<td style="text-align:left;">
Tas1r2
</td>
<td style="text-align:left;">
NCBI_Gene:83770,ENSEMBL:ENSMUSG00000028738
</td>
<td style="text-align:left;">
MGI:1933546
</td>
<td style="text-align:left;">
protein coding gene
</td>
<td style="text-align:left;">
taste receptor, type 1, member 2
</td>
</tr>
<tr>
<td style="text-align:left;">
4
</td>
<td style="text-align:left;">
gene
</td>
<td style="text-align:right;">
139.7350
</td>
<td style="text-align:right;">
139.7357
</td>
<td style="text-align:left;">
positive
</td>
<td style="text-align:left;">
MGI_C57BL6J_5625219
</td>
<td style="text-align:left;">
Gm42334
</td>
<td style="text-align:left;">
NCBI_Gene:105247195
</td>
<td style="text-align:left;">
MGI:5625219
</td>
<td style="text-align:left;">
lncRNA gene
</td>
<td style="text-align:left;">
predicted gene, 42334
</td>
</tr>
<tr>
<td style="text-align:left;">
4
</td>
<td style="text-align:left;">
gene
</td>
<td style="text-align:right;">
139.7371
</td>
<td style="text-align:right;">
139.8335
</td>
<td style="text-align:left;">
negative
</td>
<td style="text-align:left;">
MGI_C57BL6J_97491
</td>
<td style="text-align:left;">
Pax7
</td>
<td style="text-align:left;">
NCBI_Gene:18509,ENSEMBL:ENSMUSG00000028736
</td>
<td style="text-align:left;">
MGI:97491
</td>
<td style="text-align:left;">
protein coding gene
</td>
<td style="text-align:left;">
paired box 7
</td>
</tr>
<tr>
<td style="text-align:left;">
4
</td>
<td style="text-align:left;">
gene
</td>
<td style="text-align:right;">
139.8331
</td>
<td style="text-align:right;">
139.8529
</td>
<td style="text-align:left;">
positive
</td>
<td style="text-align:left;">
MGI_C57BL6J_5826510
</td>
<td style="text-align:left;">
Gm46873
</td>
<td style="text-align:left;">
NCBI_Gene:108168982
</td>
<td style="text-align:left;">
MGI:5826510
</td>
<td style="text-align:left;">
lncRNA gene
</td>
<td style="text-align:left;">
predicted gene, 46873
</td>
</tr>
<tr>
<td style="text-align:left;">
4
</td>
<td style="text-align:left;">
gene
</td>
<td style="text-align:right;">
139.9105
</td>
<td style="text-align:right;">
139.9344
</td>
<td style="text-align:left;">
positive
</td>
<td style="text-align:left;">
MGI_C57BL6J_2686513
</td>
<td style="text-align:left;">
Gm1667
</td>
<td style="text-align:left;">
NCBI_Gene:102636380,ENSEMBL:ENSMUSG00000087646
</td>
<td style="text-align:left;">
MGI:2686513
</td>
<td style="text-align:left;">
lncRNA gene
</td>
<td style="text-align:left;">
predicted gene 1667
</td>
</tr>
<tr>
<td style="text-align:left;">
4
</td>
<td style="text-align:left;">
gene
</td>
<td style="text-align:right;">
139.9233
</td>
<td style="text-align:right;">
139.9332
</td>
<td style="text-align:left;">
positive
</td>
<td style="text-align:left;">
MGI_C57BL6J_3651504
</td>
<td style="text-align:left;">
Gm13028
</td>
<td style="text-align:left;">
ENSEMBL:ENSMUSG00000087628
</td>
<td style="text-align:left;">
MGI:3651504
</td>
<td style="text-align:left;">
lncRNA gene
</td>
<td style="text-align:left;">
predicted gene 13028
</td>
</tr>
<tr>
<td style="text-align:left;">
4
</td>
<td style="text-align:left;">
gene
</td>
<td style="text-align:right;">
139.9261
</td>
<td style="text-align:right;">
139.9435
</td>
<td style="text-align:left;">
negative
</td>
<td style="text-align:left;">
MGI_C57BL6J_1918485
</td>
<td style="text-align:left;">
4933427I22Rik
</td>
<td style="text-align:left;">
NCBI_Gene:71235,ENSEMBL:ENSMUSG00000085928
</td>
<td style="text-align:left;">
MGI:1918485
</td>
<td style="text-align:left;">
lncRNA gene
</td>
<td style="text-align:left;">
RIKEN cDNA 4933427I22 gene
</td>
</tr>
<tr>
<td style="text-align:left;">
4
</td>
<td style="text-align:left;">
gene
</td>
<td style="text-align:right;">
139.9602
</td>
<td style="text-align:right;">
139.9680
</td>
<td style="text-align:left;">
negative
</td>
<td style="text-align:left;">
MGI_C57BL6J_2444612
</td>
<td style="text-align:left;">
Klhdc7a
</td>
<td style="text-align:left;">
NCBI_Gene:242721,ENSEMBL:ENSMUSG00000078234
</td>
<td style="text-align:left;">
MGI:2444612
</td>
<td style="text-align:left;">
protein coding gene
</td>
<td style="text-align:left;">
kelch domain containing 7A
</td>
</tr>
<tr>
<td style="text-align:left;">
4
</td>
<td style="text-align:left;">
gene
</td>
<td style="text-align:right;">
139.9675
</td>
<td style="text-align:right;">
139.9676
</td>
<td style="text-align:left;">
negative
</td>
<td style="text-align:left;">
MGI_C57BL6J_4358926
</td>
<td style="text-align:left;">
Mir2139
</td>
<td style="text-align:left;">
miRBase:MI0010752,NCBI_Gene:100316727,ENSEMBL:ENSMUSG00000089294
</td>
<td style="text-align:left;">
MGI:4358926
</td>
<td style="text-align:left;">
miRNA gene
</td>
<td style="text-align:left;">
microRNA 2139
</td>
</tr>
<tr>
<td style="text-align:left;">
4
</td>
<td style="text-align:left;">
gene
</td>
<td style="text-align:right;">
140.0268
</td>
<td style="text-align:right;">
140.2469
</td>
<td style="text-align:left;">
negative
</td>
<td style="text-align:left;">
MGI_C57BL6J_2681842
</td>
<td style="text-align:left;">
Igsf21
</td>
<td style="text-align:left;">
NCBI_Gene:230868,ENSEMBL:ENSMUSG00000040972
</td>
<td style="text-align:left;">
MGI:2681842
</td>
<td style="text-align:left;">
protein coding gene
</td>
<td style="text-align:left;">
immunoglobulin superfamily, member 21
</td>
</tr>
<tr>
<td style="text-align:left;">
4
</td>
<td style="text-align:left;">
gene
</td>
<td style="text-align:right;">
140.1086
</td>
<td style="text-align:right;">
140.1450
</td>
<td style="text-align:left;">
positive
</td>
<td style="text-align:left;">
MGI_C57BL6J_3651764
</td>
<td style="text-align:left;">
Gm13029
</td>
<td style="text-align:left;">
NCBI_Gene:102636673,ENSEMBL:ENSMUSG00000086164
</td>
<td style="text-align:left;">
MGI:3651764
</td>
<td style="text-align:left;">
lncRNA gene
</td>
<td style="text-align:left;">
predicted gene 13029
</td>
</tr>
<tr>
<td style="text-align:left;">
4
</td>
<td style="text-align:left;">
gene
</td>
<td style="text-align:right;">
140.1319
</td>
<td style="text-align:right;">
140.1334
</td>
<td style="text-align:left;">
positive
</td>
<td style="text-align:left;">
MGI_C57BL6J_3651912
</td>
<td style="text-align:left;">
Gm13027
</td>
<td style="text-align:left;">
ENSEMBL:ENSMUSG00000085968
</td>
<td style="text-align:left;">
MGI:3651912
</td>
<td style="text-align:left;">
lncRNA gene
</td>
<td style="text-align:left;">
predicted gene 13027
</td>
</tr>
<tr>
<td style="text-align:left;">
4
</td>
<td style="text-align:left;">
gene
</td>
<td style="text-align:right;">
140.2185
</td>
<td style="text-align:right;">
140.2225
</td>
<td style="text-align:left;">
positive
</td>
<td style="text-align:left;">
MGI_C57BL6J_5592701
</td>
<td style="text-align:left;">
Gm33542
</td>
<td style="text-align:left;">
NCBI_Gene:102636490
</td>
<td style="text-align:left;">
MGI:5592701
</td>
<td style="text-align:left;">
lncRNA gene
</td>
<td style="text-align:left;">
predicted gene, 33542
</td>
</tr>
<tr>
<td style="text-align:left;">
4
</td>
<td style="text-align:left;">
gene
</td>
<td style="text-align:right;">
140.3207
</td>
<td style="text-align:right;">
140.3233
</td>
<td style="text-align:left;">
negative
</td>
<td style="text-align:left;">
MGI_C57BL6J_3642724
</td>
<td style="text-align:left;">
Gm9867
</td>
<td style="text-align:left;">
ENSEMBL:ENSMUSG00000103107
</td>
<td style="text-align:left;">
MGI:3642724
</td>
<td style="text-align:left;">
lncRNA gene
</td>
<td style="text-align:left;">
predicted gene 9867
</td>
</tr>
<tr>
<td style="text-align:left;">
4
</td>
<td style="text-align:left;">
gene
</td>
<td style="text-align:right;">
140.3234
</td>
<td style="text-align:right;">
140.3305
</td>
<td style="text-align:left;">
positive
</td>
<td style="text-align:left;">
MGI_C57BL6J_5593054
</td>
<td style="text-align:left;">
Gm33895
</td>
<td style="text-align:left;">
NCBI_Gene:102636971
</td>
<td style="text-align:left;">
MGI:5593054
</td>
<td style="text-align:left;">
lncRNA gene
</td>
<td style="text-align:left;">
predicted gene, 33895
</td>
</tr>
<tr>
<td style="text-align:left;">
4
</td>
<td style="text-align:left;">
gene
</td>
<td style="text-align:right;">
140.3554
</td>
<td style="text-align:right;">
140.3619
</td>
<td style="text-align:left;">
positive
</td>
<td style="text-align:left;">
MGI_C57BL6J_5593233
</td>
<td style="text-align:left;">
Gm34074
</td>
<td style="text-align:left;">
NCBI_Gene:102637201
</td>
<td style="text-align:left;">
MGI:5593233
</td>
<td style="text-align:left;">
lncRNA gene
</td>
<td style="text-align:left;">
predicted gene, 34074
</td>
</tr>
<tr>
<td style="text-align:left;">
4
</td>
<td style="text-align:left;">
gene
</td>
<td style="text-align:right;">
140.3805
</td>
<td style="text-align:right;">
140.3864
</td>
<td style="text-align:left;">
positive
</td>
<td style="text-align:left;">
MGI_C57BL6J_3650527
</td>
<td style="text-align:left;">
Gm13016
</td>
<td style="text-align:left;">
NCBI_Gene:329970,ENSEMBL:ENSMUSG00000087084
</td>
<td style="text-align:left;">
MGI:3650527
</td>
<td style="text-align:left;">
lncRNA gene
</td>
<td style="text-align:left;">
predicted gene 13016
</td>
</tr>
<tr>
<td style="text-align:left;">
4
</td>
<td style="text-align:left;">
gene
</td>
<td style="text-align:right;">
140.3904
</td>
<td style="text-align:right;">
140.3969
</td>
<td style="text-align:left;">
positive
</td>
<td style="text-align:left;">
MGI_C57BL6J_3650528
</td>
<td style="text-align:left;">
Gm13017
</td>
<td style="text-align:left;">
NCBI_Gene:102637278,ENSEMBL:ENSMUSG00000086960
</td>
<td style="text-align:left;">
MGI:3650528
</td>
<td style="text-align:left;">
lncRNA gene
</td>
<td style="text-align:left;">
predicted gene 13017
</td>
</tr>
<tr>
<td style="text-align:left;">
4
</td>
<td style="text-align:left;">
gene
</td>
<td style="text-align:right;">
140.4225
</td>
<td style="text-align:right;">
140.4319
</td>
<td style="text-align:left;">
positive
</td>
<td style="text-align:left;">
MGI_C57BL6J_3649637
</td>
<td style="text-align:left;">
Gm13021
</td>
<td style="text-align:left;">
NCBI_Gene:102637389,ENSEMBL:ENSMUSG00000086437
</td>
<td style="text-align:left;">
MGI:3649637
</td>
<td style="text-align:left;">
lncRNA gene
</td>
<td style="text-align:left;">
predicted gene 13021
</td>
</tr>
<tr>
<td style="text-align:left;">
4
</td>
<td style="text-align:left;">
gene
</td>
<td style="text-align:right;">
140.4380
</td>
<td style="text-align:right;">
140.4613
</td>
<td style="text-align:left;">
positive
</td>
<td style="text-align:left;">
MGI_C57BL6J_3650881
</td>
<td style="text-align:left;">
Gm13026
</td>
<td style="text-align:left;">
ENSEMBL:ENSMUSG00000085047
</td>
<td style="text-align:left;">
MGI:3650881
</td>
<td style="text-align:left;">
lncRNA gene
</td>
<td style="text-align:left;">
predicted gene 13026
</td>
</tr>
<tr>
<td style="text-align:left;">
4
</td>
<td style="text-align:left;">
pseudogene
</td>
<td style="text-align:right;">
140.4458
</td>
<td style="text-align:right;">
140.4462
</td>
<td style="text-align:left;">
negative
</td>
<td style="text-align:left;">
MGI_C57BL6J_6324716
</td>
<td style="text-align:left;">
Gm50453
</td>
<td style="text-align:left;">
ENSEMBL:ENSMUSG00000118416
</td>
<td style="text-align:left;">
MGI:6324716
</td>
<td style="text-align:left;">
pseudogene
</td>
<td style="text-align:left;">
predicted gene, 50453
</td>
</tr>
<tr>
<td style="text-align:left;">
4
</td>
<td style="text-align:left;">
gene
</td>
<td style="text-align:right;">
140.5145
</td>
<td style="text-align:right;">
140.6660
</td>
<td style="text-align:left;">
negative
</td>
<td style="text-align:left;">
MGI_C57BL6J_1920004
</td>
<td style="text-align:left;">
Arhgef10l
</td>
<td style="text-align:left;">
NCBI_Gene:72754,ENSEMBL:ENSMUSG00000040964
</td>
<td style="text-align:left;">
MGI:1920004
</td>
<td style="text-align:left;">
protein coding gene
</td>
<td style="text-align:left;">
Rho guanine nucleotide exchange factor (GEF) 10-like
</td>
</tr>
<tr>
<td style="text-align:left;">
4
</td>
<td style="text-align:left;">
gene
</td>
<td style="text-align:right;">
140.6051
</td>
<td style="text-align:right;">
140.6066
</td>
<td style="text-align:left;">
positive
</td>
<td style="text-align:left;">
MGI_C57BL6J_5625220
</td>
<td style="text-align:left;">
Gm42335
</td>
<td style="text-align:left;">
NCBI_Gene:105247196
</td>
<td style="text-align:left;">
MGI:5625220
</td>
<td style="text-align:left;">
lncRNA gene
</td>
<td style="text-align:left;">
predicted gene, 42335
</td>
</tr>
<tr>
<td style="text-align:left;">
4
</td>
<td style="text-align:left;">
gene
</td>
<td style="text-align:right;">
140.6797
</td>
<td style="text-align:right;">
140.6863
</td>
<td style="text-align:left;">
positive
</td>
<td style="text-align:left;">
MGI_C57BL6J_3650883
</td>
<td style="text-align:left;">
Gm13025
</td>
<td style="text-align:left;">
NCBI_Gene:102637546,ENSEMBL:ENSMUSG00000086159
</td>
<td style="text-align:left;">
MGI:3650883
</td>
<td style="text-align:left;">
lncRNA gene
</td>
<td style="text-align:left;">
predicted gene 13025
</td>
</tr>
<tr>
<td style="text-align:left;">
4
</td>
<td style="text-align:left;">
gene
</td>
<td style="text-align:right;">
140.6927
</td>
<td style="text-align:right;">
140.6928
</td>
<td style="text-align:left;">
positive
</td>
<td style="text-align:left;">
MGI_C57BL6J_5452822
</td>
<td style="text-align:left;">
Gm23045
</td>
<td style="text-align:left;">
ENSEMBL:ENSMUSG00000095878
</td>
<td style="text-align:left;">
MGI:5452822
</td>
<td style="text-align:left;">
miRNA gene
</td>
<td style="text-align:left;">
predicted gene, 23045
</td>
</tr>
<tr>
<td style="text-align:left;">
4
</td>
<td style="text-align:left;">
gene
</td>
<td style="text-align:right;">
140.7005
</td>
<td style="text-align:right;">
140.7232
</td>
<td style="text-align:left;">
positive
</td>
<td style="text-align:left;">
MGI_C57BL6J_1919784
</td>
<td style="text-align:left;">
Rcc2
</td>
<td style="text-align:left;">
NCBI_Gene:108911,ENSEMBL:ENSMUSG00000040945
</td>
<td style="text-align:left;">
MGI:1919784
</td>
<td style="text-align:left;">
protein coding gene
</td>
<td style="text-align:left;">
regulator of chromosome condensation 2
</td>
</tr>
<tr>
<td style="text-align:left;">
4
</td>
<td style="text-align:left;">
gene
</td>
<td style="text-align:right;">
140.7170
</td>
<td style="text-align:right;">
140.7171
</td>
<td style="text-align:left;">
positive
</td>
<td style="text-align:left;">
MGI_C57BL6J_5455728
</td>
<td style="text-align:left;">
Gm25951
</td>
<td style="text-align:left;">
ENSEMBL:ENSMUSG00000093071
</td>
<td style="text-align:left;">
MGI:5455728
</td>
<td style="text-align:left;">
miRNA gene
</td>
<td style="text-align:left;">
predicted gene, 25951
</td>
</tr>
<tr>
<td style="text-align:left;">
4
</td>
<td style="text-align:left;">
gene
</td>
<td style="text-align:right;">
140.7274
</td>
<td style="text-align:right;">
140.7426
</td>
<td style="text-align:left;">
negative
</td>
<td style="text-align:left;">
MGI_C57BL6J_2655198
</td>
<td style="text-align:left;">
Padi6
</td>
<td style="text-align:left;">
NCBI_Gene:242726,ENSEMBL:ENSMUSG00000040935
</td>
<td style="text-align:left;">
MGI:2655198
</td>
<td style="text-align:left;">
protein coding gene
</td>
<td style="text-align:left;">
peptidyl arginine deiminase, type VI
</td>
</tr>
<tr>
<td style="text-align:left;">
4
</td>
<td style="text-align:left;">
gene
</td>
<td style="text-align:right;">
140.7455
</td>
<td style="text-align:right;">
140.7742
</td>
<td style="text-align:left;">
negative
</td>
<td style="text-align:left;">
MGI_C57BL6J_1338898
</td>
<td style="text-align:left;">
Padi4
</td>
<td style="text-align:left;">
NCBI_Gene:18602,ENSEMBL:ENSMUSG00000025330
</td>
<td style="text-align:left;">
MGI:1338898
</td>
<td style="text-align:left;">
protein coding gene
</td>
<td style="text-align:left;">
peptidyl arginine deiminase, type IV
</td>
</tr>
<tr>
<td style="text-align:left;">
4
</td>
<td style="text-align:left;">
gene
</td>
<td style="text-align:right;">
140.7854
</td>
<td style="text-align:right;">
140.8106
</td>
<td style="text-align:left;">
negative
</td>
<td style="text-align:left;">
MGI_C57BL6J_1338891
</td>
<td style="text-align:left;">
Padi3
</td>
<td style="text-align:left;">
NCBI_Gene:18601,ENSEMBL:ENSMUSG00000025328
</td>
<td style="text-align:left;">
MGI:1338891
</td>
<td style="text-align:left;">
protein coding gene
</td>
<td style="text-align:left;">
peptidyl arginine deiminase, type III
</td>
</tr>
<tr>
<td style="text-align:left;">
4
</td>
<td style="text-align:left;">
gene
</td>
<td style="text-align:right;">
140.8110
</td>
<td style="text-align:right;">
140.8175
</td>
<td style="text-align:left;">
positive
</td>
<td style="text-align:left;">
MGI_C57BL6J_3702685
</td>
<td style="text-align:left;">
Gm13032
</td>
<td style="text-align:left;">
NCBI_Gene:100049161,ENSEMBL:ENSMUSG00000085918
</td>
<td style="text-align:left;">
MGI:3702685
</td>
<td style="text-align:left;">
lncRNA gene
</td>
<td style="text-align:left;">
predicted gene 13032
</td>
</tr>
<tr>
<td style="text-align:left;">
4
</td>
<td style="text-align:left;">
gene
</td>
<td style="text-align:right;">
140.8130
</td>
<td style="text-align:right;">
140.8458
</td>
<td style="text-align:left;">
negative
</td>
<td style="text-align:left;">
MGI_C57BL6J_1338893
</td>
<td style="text-align:left;">
Padi1
</td>
<td style="text-align:left;">
NCBI_Gene:18599,ENSEMBL:ENSMUSG00000025329
</td>
<td style="text-align:left;">
MGI:1338893
</td>
<td style="text-align:left;">
protein coding gene
</td>
<td style="text-align:left;">
peptidyl arginine deiminase, type I
</td>
</tr>
<tr>
<td style="text-align:left;">
4
</td>
<td style="text-align:left;">
gene
</td>
<td style="text-align:right;">
140.8727
</td>
<td style="text-align:right;">
140.8781
</td>
<td style="text-align:left;">
positive
</td>
<td style="text-align:left;">
MGI_C57BL6J_1922378
</td>
<td style="text-align:left;">
4930515B02Rik
</td>
<td style="text-align:left;">
NCBI_Gene:75128,ENSEMBL:ENSMUSG00000087045
</td>
<td style="text-align:left;">
MGI:1922378
</td>
<td style="text-align:left;">
lncRNA gene
</td>
<td style="text-align:left;">
RIKEN cDNA 4930515B02 gene
</td>
</tr>
<tr>
<td style="text-align:left;">
4
</td>
<td style="text-align:left;">
gene
</td>
<td style="text-align:right;">
140.9063
</td>
<td style="text-align:right;">
140.9526
</td>
<td style="text-align:left;">
positive
</td>
<td style="text-align:left;">
MGI_C57BL6J_1338892
</td>
<td style="text-align:left;">
Padi2
</td>
<td style="text-align:left;">
NCBI_Gene:18600,ENSEMBL:ENSMUSG00000028927
</td>
<td style="text-align:left;">
MGI:1338892
</td>
<td style="text-align:left;">
protein coding gene
</td>
<td style="text-align:left;">
peptidyl arginine deiminase, type II
</td>
</tr>
<tr>
<td style="text-align:left;">
4
</td>
<td style="text-align:left;">
gene
</td>
<td style="text-align:right;">
140.9090
</td>
<td style="text-align:right;">
140.9103
</td>
<td style="text-align:left;">
negative
</td>
<td style="text-align:left;">
MGI_C57BL6J_5826356
</td>
<td style="text-align:left;">
Gm46719
</td>
<td style="text-align:left;">
NCBI_Gene:108168688
</td>
<td style="text-align:left;">
MGI:5826356
</td>
<td style="text-align:left;">
lncRNA gene
</td>
<td style="text-align:left;">
predicted gene, 46719
</td>
</tr>
<tr>
<td style="text-align:left;">
4
</td>
<td style="text-align:left;">
gene
</td>
<td style="text-align:right;">
140.9187
</td>
<td style="text-align:right;">
140.9188
</td>
<td style="text-align:left;">
positive
</td>
<td style="text-align:left;">
MGI_C57BL6J_5456003
</td>
<td style="text-align:left;">
Gm26226
</td>
<td style="text-align:left;">
ENSEMBL:ENSMUSG00000064443
</td>
<td style="text-align:left;">
MGI:5456003
</td>
<td style="text-align:left;">
snRNA gene
</td>
<td style="text-align:left;">
predicted gene, 26226
</td>
</tr>
<tr>
<td style="text-align:left;">
4
</td>
<td style="text-align:left;">
gene
</td>
<td style="text-align:right;">
140.9272
</td>
<td style="text-align:right;">
140.9321
</td>
<td style="text-align:left;">
negative
</td>
<td style="text-align:left;">
MGI_C57BL6J_5593534
</td>
<td style="text-align:left;">
Gm34375
</td>
<td style="text-align:left;">
NCBI_Gene:102637611
</td>
<td style="text-align:left;">
MGI:5593534
</td>
<td style="text-align:left;">
lncRNA gene
</td>
<td style="text-align:left;">
predicted gene, 34375
</td>
</tr>
<tr>
<td style="text-align:left;">
4
</td>
<td style="text-align:left;">
gene
</td>
<td style="text-align:right;">
140.9477
</td>
<td style="text-align:right;">
140.9579
</td>
<td style="text-align:left;">
negative
</td>
<td style="text-align:left;">
MGI_C57BL6J_3702680
</td>
<td style="text-align:left;">
Gm13031
</td>
<td style="text-align:left;">
NCBI_Gene:100126227,ENSEMBL:ENSMUSG00000087698
</td>
<td style="text-align:left;">
MGI:3702680
</td>
<td style="text-align:left;">
lncRNA gene
</td>
<td style="text-align:left;">
predicted gene 13031
</td>
</tr>
<tr>
<td style="text-align:left;">
4
</td>
<td style="text-align:left;">
gene
</td>
<td style="text-align:right;">
140.9604
</td>
<td style="text-align:right;">
140.9613
</td>
<td style="text-align:left;">
negative
</td>
<td style="text-align:left;">
MGI_C57BL6J_5593590
</td>
<td style="text-align:left;">
Gm34431
</td>
<td style="text-align:left;">
NCBI_Gene:102637683
</td>
<td style="text-align:left;">
MGI:5593590
</td>
<td style="text-align:left;">
lncRNA gene
</td>
<td style="text-align:left;">
predicted gene, 34431
</td>
</tr>
<tr>
<td style="text-align:left;">
4
</td>
<td style="text-align:left;">
gene
</td>
<td style="text-align:right;">
140.9612
</td>
<td style="text-align:right;">
140.9792
</td>
<td style="text-align:left;">
positive
</td>
<td style="text-align:left;">
MGI_C57BL6J_1914930
</td>
<td style="text-align:left;">
Sdhb
</td>
<td style="text-align:left;">
NCBI_Gene:67680,ENSEMBL:ENSMUSG00000009863
</td>
<td style="text-align:left;">
MGI:1914930
</td>
<td style="text-align:left;">
protein coding gene
</td>
<td style="text-align:left;">
succinate dehydrogenase complex, subunit B, iron sulfur (Ip)
</td>
</tr>
<tr>
<td style="text-align:left;">
4
</td>
<td style="text-align:left;">
gene
</td>
<td style="text-align:right;">
140.9869
</td>
<td style="text-align:right;">
141.0077
</td>
<td style="text-align:left;">
positive
</td>
<td style="text-align:left;">
MGI_C57BL6J_1922022
</td>
<td style="text-align:left;">
Atp13a2
</td>
<td style="text-align:left;">
NCBI_Gene:74772,ENSEMBL:ENSMUSG00000036622
</td>
<td style="text-align:left;">
MGI:1922022
</td>
<td style="text-align:left;">
protein coding gene
</td>
<td style="text-align:left;">
ATPase type 13A2
</td>
</tr>
<tr>
<td style="text-align:left;">
4
</td>
<td style="text-align:left;">
gene
</td>
<td style="text-align:right;">
141.0104
</td>
<td style="text-align:right;">
141.0160
</td>
<td style="text-align:left;">
positive
</td>
<td style="text-align:left;">
MGI_C57BL6J_99559
</td>
<td style="text-align:left;">
Mfap2
</td>
<td style="text-align:left;">
NCBI_Gene:17150,ENSEMBL:ENSMUSG00000060572
</td>
<td style="text-align:left;">
MGI:99559
</td>
<td style="text-align:left;">
protein coding gene
</td>
<td style="text-align:left;">
microfibrillar-associated protein 2
</td>
</tr>
<tr>
<td style="text-align:left;">
4
</td>
<td style="text-align:left;">
gene
</td>
<td style="text-align:right;">
141.0166
</td>
<td style="text-align:right;">
141.0606
</td>
<td style="text-align:left;">
negative
</td>
<td style="text-align:left;">
MGI_C57BL6J_3529431
</td>
<td style="text-align:left;">
Crocc
</td>
<td style="text-align:left;">
NCBI_Gene:230872,ENSEMBL:ENSMUSG00000040860
</td>
<td style="text-align:left;">
MGI:3529431
</td>
<td style="text-align:left;">
protein coding gene
</td>
<td style="text-align:left;">
ciliary rootlet coiled-coil, rootletin
</td>
</tr>
<tr>
<td style="text-align:left;">
4
</td>
<td style="text-align:left;">
gene
</td>
<td style="text-align:right;">
141.0665
</td>
<td style="text-align:right;">
141.0784
</td>
<td style="text-align:left;">
negative
</td>
<td style="text-align:left;">
MGI_C57BL6J_1913397
</td>
<td style="text-align:left;">
Necap2
</td>
<td style="text-align:left;">
NCBI_Gene:66147,ENSEMBL:ENSMUSG00000028923
</td>
<td style="text-align:left;">
MGI:1913397
</td>
<td style="text-align:left;">
protein coding gene
</td>
<td style="text-align:left;">
NECAP endocytosis associated 2
</td>
</tr>
<tr>
<td style="text-align:left;">
4
</td>
<td style="text-align:left;">
pseudogene
</td>
<td style="text-align:right;">
141.0848
</td>
<td style="text-align:right;">
141.0857
</td>
<td style="text-align:left;">
positive
</td>
<td style="text-align:left;">
MGI_C57BL6J_3651114
</td>
<td style="text-align:left;">
Gm13047
</td>
<td style="text-align:left;">
NCBI_Gene:638991,ENSEMBL:ENSMUSG00000082246
</td>
<td style="text-align:left;">
MGI:3651114
</td>
<td style="text-align:left;">
pseudogene
</td>
<td style="text-align:left;">
predicted gene 13047
</td>
</tr>
<tr>
<td style="text-align:left;">
4
</td>
<td style="text-align:left;">
gene
</td>
<td style="text-align:right;">
141.0883
</td>
<td style="text-align:right;">
141.1128
</td>
<td style="text-align:left;">
positive
</td>
<td style="text-align:left;">
MGI_C57BL6J_3607787
</td>
<td style="text-align:left;">
Spata21
</td>
<td style="text-align:left;">
NCBI_Gene:329972,ENSEMBL:ENSMUSG00000045004
</td>
<td style="text-align:left;">
MGI:3607787
</td>
<td style="text-align:left;">
protein coding gene
</td>
<td style="text-align:left;">
spermatogenesis associated 21
</td>
</tr>
<tr>
<td style="text-align:left;">
4
</td>
<td style="text-align:left;">
gene
</td>
<td style="text-align:right;">
141.1130
</td>
<td style="text-align:right;">
141.1398
</td>
<td style="text-align:left;">
negative
</td>
<td style="text-align:left;">
MGI_C57BL6J_1098672
</td>
<td style="text-align:left;">
Szrd1
</td>
<td style="text-align:left;">
NCBI_Gene:213491,ENSEMBL:ENSMUSG00000040842
</td>
<td style="text-align:left;">
MGI:1098672
</td>
<td style="text-align:left;">
protein coding gene
</td>
<td style="text-align:left;">
SUZ RNA binding domain containing 1
</td>
</tr>
<tr>
<td style="text-align:left;">
4
</td>
<td style="text-align:left;">
gene
</td>
<td style="text-align:right;">
141.1157
</td>
<td style="text-align:right;">
141.1181
</td>
<td style="text-align:left;">
positive
</td>
<td style="text-align:left;">
MGI_C57BL6J_3704238
</td>
<td style="text-align:left;">
4921514A10Rik
</td>
<td style="text-align:left;">
ENSEMBL:ENSMUSG00000097620
</td>
<td style="text-align:left;">
MGI:3704238
</td>
<td style="text-align:left;">
lncRNA gene
</td>
<td style="text-align:left;">
RIKEN cDNA 4921514A10 gene
</td>
</tr>
<tr>
<td style="text-align:left;">
4
</td>
<td style="text-align:left;">
gene
</td>
<td style="text-align:right;">
141.1479
</td>
<td style="text-align:right;">
141.2041
</td>
<td style="text-align:left;">
positive
</td>
<td style="text-align:left;">
MGI_C57BL6J_1924992
</td>
<td style="text-align:left;">
Fbxo42
</td>
<td style="text-align:left;">
NCBI_Gene:213499,ENSEMBL:ENSMUSG00000028920
</td>
<td style="text-align:left;">
MGI:1924992
</td>
<td style="text-align:left;">
protein coding gene
</td>
<td style="text-align:left;">
F-box protein 42
</td>
</tr>
<tr>
<td style="text-align:left;">
4
</td>
<td style="text-align:left;">
gene
</td>
<td style="text-align:right;">
141.2137
</td>
<td style="text-align:right;">
141.2268
</td>
<td style="text-align:left;">
positive
</td>
<td style="text-align:left;">
MGI_C57BL6J_1923416
</td>
<td style="text-align:left;">
Cplane2
</td>
<td style="text-align:left;">
NCBI_Gene:76166,ENSEMBL:ENSMUSG00000073733
</td>
<td style="text-align:left;">
MGI:1923416
</td>
<td style="text-align:left;">
protein coding gene
</td>
<td style="text-align:left;">
ciliogenesis and planar polarity effector 2
</td>
</tr>
<tr>
<td style="text-align:left;">
4
</td>
<td style="text-align:left;">
gene
</td>
<td style="text-align:right;">
141.2382
</td>
<td style="text-align:right;">
141.2395
</td>
<td style="text-align:left;">
negative
</td>
<td style="text-align:left;">
MGI_C57BL6J_3641643
</td>
<td style="text-align:left;">
C630004L07Rik
</td>
<td style="text-align:left;">
ENSEMBL:ENSMUSG00000103366
</td>
<td style="text-align:left;">
MGI:3641643
</td>
<td style="text-align:left;">
unclassified gene
</td>
<td style="text-align:left;">
RIKEN cDNA C630004L07 gene
</td>
</tr>
<tr>
<td style="text-align:left;">
4
</td>
<td style="text-align:left;">
gene
</td>
<td style="text-align:right;">
141.2395
</td>
<td style="text-align:right;">
141.2579
</td>
<td style="text-align:left;">
positive
</td>
<td style="text-align:left;">
MGI_C57BL6J_1925912
</td>
<td style="text-align:left;">
Arhgef19
</td>
<td style="text-align:left;">
NCBI_Gene:213649,ENSEMBL:ENSMUSG00000028919
</td>
<td style="text-align:left;">
MGI:1925912
</td>
<td style="text-align:left;">
protein coding gene
</td>
<td style="text-align:left;">
Rho guanine nucleotide exchange factor (GEF) 19
</td>
</tr>
<tr>
<td style="text-align:left;">
4
</td>
<td style="text-align:left;">
gene
</td>
<td style="text-align:right;">
141.2618
</td>
<td style="text-align:right;">
141.2660
</td>
<td style="text-align:left;">
negative
</td>
<td style="text-align:left;">
MGI_C57BL6J_3649999
</td>
<td style="text-align:left;">
Gm13055
</td>
<td style="text-align:left;">
ENSEMBL:ENSMUSG00000085774
</td>
<td style="text-align:left;">
MGI:3649999
</td>
<td style="text-align:left;">
lncRNA gene
</td>
<td style="text-align:left;">
predicted gene 13055
</td>
</tr>
<tr>
<td style="text-align:left;">
4
</td>
<td style="text-align:left;">
gene
</td>
<td style="text-align:right;">
141.2784
</td>
<td style="text-align:right;">
141.2798
</td>
<td style="text-align:left;">
positive
</td>
<td style="text-align:left;">
MGI_C57BL6J_3650725
</td>
<td style="text-align:left;">
Gm13056
</td>
<td style="text-align:left;">
NCBI_Gene:100503810,ENSEMBL:ENSMUSG00000085395
</td>
<td style="text-align:left;">
MGI:3650725
</td>
<td style="text-align:left;">
lncRNA gene
</td>
<td style="text-align:left;">
predicted gene 13056
</td>
</tr>
<tr>
<td style="text-align:left;">
4
</td>
<td style="text-align:left;">
gene
</td>
<td style="text-align:right;">
141.3012
</td>
<td style="text-align:right;">
141.3294
</td>
<td style="text-align:left;">
positive
</td>
<td style="text-align:left;">
MGI_C57BL6J_95278
</td>
<td style="text-align:left;">
Epha2
</td>
<td style="text-align:left;">
NCBI_Gene:13836,ENSEMBL:ENSMUSG00000006445
</td>
<td style="text-align:left;">
MGI:95278
</td>
<td style="text-align:left;">
protein coding gene
</td>
<td style="text-align:left;">
Eph receptor A2
</td>
</tr>
<tr>
<td style="text-align:left;">
4
</td>
<td style="text-align:left;">
gene
</td>
<td style="text-align:right;">
141.3462
</td>
<td style="text-align:right;">
141.3462
</td>
<td style="text-align:left;">
positive
</td>
<td style="text-align:left;">
MGI_C57BL6J_5455467
</td>
<td style="text-align:left;">
Gm25690
</td>
<td style="text-align:left;">
ENSEMBL:ENSMUSG00000093180
</td>
<td style="text-align:left;">
MGI:5455467
</td>
<td style="text-align:left;">
miRNA gene
</td>
<td style="text-align:left;">
predicted gene, 25690
</td>
</tr>
<tr>
<td style="text-align:left;">
4
</td>
<td style="text-align:left;">
gene
</td>
<td style="text-align:right;">
141.3478
</td>
<td style="text-align:right;">
141.3509
</td>
<td style="text-align:left;">
positive
</td>
<td style="text-align:left;">
MGI_C57BL6J_3651959
</td>
<td style="text-align:left;">
Gm13074
</td>
<td style="text-align:left;">
ENSEMBL:ENSMUSG00000085836
</td>
<td style="text-align:left;">
MGI:3651959
</td>
<td style="text-align:left;">
lncRNA gene
</td>
<td style="text-align:left;">
predicted gene 13074
</td>
</tr>
<tr>
<td style="text-align:left;">
4
</td>
<td style="text-align:left;">
pseudogene
</td>
<td style="text-align:right;">
141.3569
</td>
<td style="text-align:right;">
141.3571
</td>
<td style="text-align:left;">
negative
</td>
<td style="text-align:left;">
MGI_C57BL6J_3651957
</td>
<td style="text-align:left;">
Gm13076
</td>
<td style="text-align:left;">
NCBI_Gene:102637915,ENSEMBL:ENSMUSG00000084238
</td>
<td style="text-align:left;">
MGI:3651957
</td>
<td style="text-align:left;">
pseudogene
</td>
<td style="text-align:left;">
predicted gene 13076
</td>
</tr>
<tr>
<td style="text-align:left;">
4
</td>
<td style="text-align:left;">
gene
</td>
<td style="text-align:right;">
141.3682
</td>
<td style="text-align:right;">
141.3842
</td>
<td style="text-align:left;">
positive
</td>
<td style="text-align:left;">
MGI_C57BL6J_2685539
</td>
<td style="text-align:left;">
Fam131c
</td>
<td style="text-align:left;">
NCBI_Gene:277743,ENSEMBL:ENSMUSG00000006218
</td>
<td style="text-align:left;">
MGI:2685539
</td>
<td style="text-align:left;">
protein coding gene
</td>
<td style="text-align:left;">
family with sequence similarity 131, member C
</td>
</tr>
<tr>
<td style="text-align:left;">
4
</td>
<td style="text-align:left;">
gene
</td>
<td style="text-align:right;">
141.3846
</td>
<td style="text-align:right;">
141.3987
</td>
<td style="text-align:left;">
negative
</td>
<td style="text-align:left;">
MGI_C57BL6J_1329026
</td>
<td style="text-align:left;">
Clcnka
</td>
<td style="text-align:left;">
NCBI_Gene:12733,ENSEMBL:ENSMUSG00000033770
</td>
<td style="text-align:left;">
MGI:1329026
</td>
<td style="text-align:left;">
protein coding gene
</td>
<td style="text-align:left;">
chloride channel, voltage-sensitive Ka
</td>
</tr>
<tr>
<td style="text-align:left;">
4
</td>
<td style="text-align:left;">
gene
</td>
<td style="text-align:right;">
141.3988
</td>
<td style="text-align:right;">
141.4112
</td>
<td style="text-align:left;">
positive
</td>
<td style="text-align:left;">
MGI_C57BL6J_3651958
</td>
<td style="text-align:left;">
Gm13075
</td>
<td style="text-align:left;">
ENSEMBL:ENSMUSG00000086061
</td>
<td style="text-align:left;">
MGI:3651958
</td>
<td style="text-align:left;">
lncRNA gene
</td>
<td style="text-align:left;">
predicted gene 13075
</td>
</tr>
<tr>
<td style="text-align:left;">
4
</td>
<td style="text-align:left;">
gene
</td>
<td style="text-align:right;">
141.3998
</td>
<td style="text-align:right;">
141.4006
</td>
<td style="text-align:left;">
negative
</td>
<td style="text-align:left;">
MGI_C57BL6J_5593852
</td>
<td style="text-align:left;">
Gm34693
</td>
<td style="text-align:left;">
NCBI_Gene:102638025
</td>
<td style="text-align:left;">
MGI:5593852
</td>
<td style="text-align:left;">
lncRNA gene
</td>
<td style="text-align:left;">
predicted gene, 34693
</td>
</tr>
<tr>
<td style="text-align:left;">
4
</td>
<td style="text-align:left;">
gene
</td>
<td style="text-align:right;">
141.4044
</td>
<td style="text-align:right;">
141.4160
</td>
<td style="text-align:left;">
negative
</td>
<td style="text-align:left;">
MGI_C57BL6J_1930643
</td>
<td style="text-align:left;">
Clcnkb
</td>
<td style="text-align:left;">
NCBI_Gene:56365,ENSEMBL:ENSMUSG00000006216
</td>
<td style="text-align:left;">
MGI:1930643
</td>
<td style="text-align:left;">
protein coding gene
</td>
<td style="text-align:left;">
chloride channel, voltage-sensitive Kb
</td>
</tr>
<tr>
<td style="text-align:left;">
4
</td>
<td style="text-align:left;">
gene
</td>
<td style="text-align:right;">
141.4208
</td>
<td style="text-align:right;">
141.4253
</td>
<td style="text-align:left;">
positive
</td>
<td style="text-align:left;">
MGI_C57BL6J_1352494
</td>
<td style="text-align:left;">
Hspb7
</td>
<td style="text-align:left;">
NCBI_Gene:29818,ENSEMBL:ENSMUSG00000006221
</td>
<td style="text-align:left;">
MGI:1352494
</td>
<td style="text-align:left;">
protein coding gene
</td>
<td style="text-align:left;">
heat shock protein family, member 7 (cardiovascular)
</td>
</tr>
<tr>
<td style="text-align:left;">
4
</td>
<td style="text-align:left;">
gene
</td>
<td style="text-align:right;">
141.4327
</td>
<td style="text-align:right;">
141.4361
</td>
<td style="text-align:left;">
negative
</td>
<td style="text-align:left;">
MGI_C57BL6J_2685540
</td>
<td style="text-align:left;">
Srarp
</td>
<td style="text-align:left;">
NCBI_Gene:277744,ENSEMBL:ENSMUSG00000070637
</td>
<td style="text-align:left;">
MGI:2685540
</td>
<td style="text-align:left;">
protein coding gene
</td>
<td style="text-align:left;">
steroid receptor associated and regulated protein
</td>
</tr>
<tr>
<td style="text-align:left;">
4
</td>
<td style="text-align:left;">
gene
</td>
<td style="text-align:right;">
141.4440
</td>
<td style="text-align:right;">
141.4447
</td>
<td style="text-align:left;">
negative
</td>
<td style="text-align:left;">
MGI_C57BL6J_5625222
</td>
<td style="text-align:left;">
Gm42337
</td>
<td style="text-align:left;">
NCBI_Gene:105247198
</td>
<td style="text-align:left;">
MGI:5625222
</td>
<td style="text-align:left;">
lncRNA gene
</td>
<td style="text-align:left;">
predicted gene, 42337
</td>
</tr>
<tr>
<td style="text-align:left;">
4
</td>
<td style="text-align:left;">
gene
</td>
<td style="text-align:right;">
141.4447
</td>
<td style="text-align:right;">
141.4679
</td>
<td style="text-align:left;">
positive
</td>
<td style="text-align:left;">
MGI_C57BL6J_107410
</td>
<td style="text-align:left;">
Zbtb17
</td>
<td style="text-align:left;">
NCBI_Gene:22642,ENSEMBL:ENSMUSG00000006215
</td>
<td style="text-align:left;">
MGI:107410
</td>
<td style="text-align:left;">
protein coding gene
</td>
<td style="text-align:left;">
zinc finger and BTB domain containing 17
</td>
</tr>
<tr>
<td style="text-align:left;">
4
</td>
<td style="text-align:left;">
gene
</td>
<td style="text-align:right;">
141.4679
</td>
<td style="text-align:right;">
141.5386
</td>
<td style="text-align:left;">
negative
</td>
<td style="text-align:left;">
MGI_C57BL6J_1891706
</td>
<td style="text-align:left;">
Spen
</td>
<td style="text-align:left;">
NCBI_Gene:56381,ENSEMBL:ENSMUSG00000040761
</td>
<td style="text-align:left;">
MGI:1891706
</td>
<td style="text-align:left;">
protein coding gene
</td>
<td style="text-align:left;">
spen family transcription repressor
</td>
</tr>
<tr>
<td style="text-align:left;">
4
</td>
<td style="text-align:left;">
gene
</td>
<td style="text-align:right;">
141.5355
</td>
<td style="text-align:right;">
141.5374
</td>
<td style="text-align:left;">
positive
</td>
<td style="text-align:left;">
MGI_C57BL6J_3782299
</td>
<td style="text-align:left;">
Gm4123
</td>
<td style="text-align:left;">
NCBI_Gene:100042951
</td>
<td style="text-align:left;">
MGI:3782299
</td>
<td style="text-align:left;">
unclassified gene
</td>
<td style="text-align:left;">
predicted gene 4123
</td>
</tr>
<tr>
<td style="text-align:left;">
4
</td>
<td style="text-align:left;">
gene
</td>
<td style="text-align:right;">
141.5392
</td>
<td style="text-align:right;">
141.5436
</td>
<td style="text-align:left;">
positive
</td>
<td style="text-align:left;">
MGI_C57BL6J_2443748
</td>
<td style="text-align:left;">
B830004H01Rik
</td>
<td style="text-align:left;">
NA
</td>
<td style="text-align:left;">
NA
</td>
<td style="text-align:left;">
unclassified non-coding RNA gene
</td>
<td style="text-align:left;">
RIKEN cDNA B830004H01 gene
</td>
</tr>
<tr>
<td style="text-align:left;">
4
</td>
<td style="text-align:left;">
gene
</td>
<td style="text-align:right;">
141.5462
</td>
<td style="text-align:right;">
141.5483
</td>
<td style="text-align:left;">
positive
</td>
<td style="text-align:left;">
MGI_C57BL6J_2444063
</td>
<td style="text-align:left;">
B330016D10Rik
</td>
<td style="text-align:left;">
NCBI_Gene:320456,ENSEMBL:ENSMUSG00000048406
</td>
<td style="text-align:left;">
MGI:2444063
</td>
<td style="text-align:left;">
lncRNA gene
</td>
<td style="text-align:left;">
RIKEN cDNA B330016D10 gene
</td>
</tr>
<tr>
<td style="text-align:left;">
4
</td>
<td style="text-align:left;">
gene
</td>
<td style="text-align:right;">
141.5761
</td>
<td style="text-align:right;">
141.6061
</td>
<td style="text-align:left;">
negative
</td>
<td style="text-align:left;">
MGI_C57BL6J_1921452
</td>
<td style="text-align:left;">
Fblim1
</td>
<td style="text-align:left;">
NCBI_Gene:74202,ENSEMBL:ENSMUSG00000006219
</td>
<td style="text-align:left;">
MGI:1921452
</td>
<td style="text-align:left;">
protein coding gene
</td>
<td style="text-align:left;">
filamin binding LIM protein 1
</td>
</tr>
<tr>
<td style="text-align:left;">
4
</td>
<td style="text-align:left;">
gene
</td>
<td style="text-align:right;">
141.6134
</td>
<td style="text-align:right;">
141.6186
</td>
<td style="text-align:left;">
negative
</td>
<td style="text-align:left;">
MGI_C57BL6J_2384869
</td>
<td style="text-align:left;">
Tmem82
</td>
<td style="text-align:left;">
NCBI_Gene:213989,ENSEMBL:ENSMUSG00000043085
</td>
<td style="text-align:left;">
MGI:2384869
</td>
<td style="text-align:left;">
protein coding gene
</td>
<td style="text-align:left;">
transmembrane protein 82
</td>
</tr>
<tr>
<td style="text-align:left;">
4
</td>
<td style="text-align:left;">
gene
</td>
<td style="text-align:right;">
141.6140
</td>
<td style="text-align:right;">
141.6156
</td>
<td style="text-align:left;">
positive
</td>
<td style="text-align:left;">
MGI_C57BL6J_2140323
</td>
<td style="text-align:left;">
AI507597
</td>
<td style="text-align:left;">
NCBI_Gene:100165,ENSEMBL:ENSMUSG00000073731
</td>
<td style="text-align:left;">
MGI:2140323
</td>
<td style="text-align:left;">
lncRNA gene
</td>
<td style="text-align:left;">
expressed sequence AI507597
</td>
</tr>
<tr>
<td style="text-align:left;">
4
</td>
<td style="text-align:left;">
gene
</td>
<td style="text-align:right;">
141.6188
</td>
<td style="text-align:right;">
141.6240
</td>
<td style="text-align:left;">
negative
</td>
<td style="text-align:left;">
MGI_C57BL6J_2686215
</td>
<td style="text-align:left;">
Slc25a34
</td>
<td style="text-align:left;">
NCBI_Gene:384071,ENSEMBL:ENSMUSG00000040740
</td>
<td style="text-align:left;">
MGI:2686215
</td>
<td style="text-align:left;">
protein coding gene
</td>
<td style="text-align:left;">
solute carrier family 25, member 34
</td>
</tr>
<tr>
<td style="text-align:left;">
4
</td>
<td style="text-align:left;">
gene
</td>
<td style="text-align:right;">
141.6257
</td>
<td style="text-align:right;">
141.6649
</td>
<td style="text-align:left;">
negative
</td>
<td style="text-align:left;">
MGI_C57BL6J_1916832
</td>
<td style="text-align:left;">
Plekhm2
</td>
<td style="text-align:left;">
NCBI_Gene:69582,ENSEMBL:ENSMUSG00000028917
</td>
<td style="text-align:left;">
MGI:1916832
</td>
<td style="text-align:left;">
protein coding gene
</td>
<td style="text-align:left;">
pleckstrin homology domain containing, family M (with RUN domain) member
2
</td>
</tr>
<tr>
<td style="text-align:left;">
4
</td>
<td style="text-align:left;">
gene
</td>
<td style="text-align:right;">
141.6521
</td>
<td style="text-align:right;">
141.6549
</td>
<td style="text-align:left;">
positive
</td>
<td style="text-align:left;">
MGI_C57BL6J_1921207
</td>
<td style="text-align:left;">
4930451G21Rik
</td>
<td style="text-align:left;">
NA
</td>
<td style="text-align:left;">
NA
</td>
<td style="text-align:left;">
unclassified gene
</td>
<td style="text-align:left;">
RIKEN cDNA 4930451G21 gene
</td>
</tr>
<tr>
<td style="text-align:left;">
4
</td>
<td style="text-align:left;">
gene
</td>
<td style="text-align:right;">
141.6775
</td>
<td style="text-align:right;">
141.7234
</td>
<td style="text-align:left;">
negative
</td>
<td style="text-align:left;">
MGI_C57BL6J_1917244
</td>
<td style="text-align:left;">
Ddi2
</td>
<td style="text-align:left;">
NCBI_Gene:68817,ENSEMBL:ENSMUSG00000078515
</td>
<td style="text-align:left;">
MGI:1917244
</td>
<td style="text-align:left;">
protein coding gene
</td>
<td style="text-align:left;">
DNA-damage inducible protein 2
</td>
</tr>
<tr>
<td style="text-align:left;">
4
</td>
<td style="text-align:left;">
gene
</td>
<td style="text-align:right;">
141.6836
</td>
<td style="text-align:right;">
141.6857
</td>
<td style="text-align:left;">
negative
</td>
<td style="text-align:left;">
MGI_C57BL6J_3526447
</td>
<td style="text-align:left;">
Rsc1a1
</td>
<td style="text-align:left;">
NCBI_Gene:69994,ENSEMBL:ENSMUSG00000040715
</td>
<td style="text-align:left;">
MGI:3526447
</td>
<td style="text-align:left;">
protein coding gene
</td>
<td style="text-align:left;">
regulatory solute carrier protein, family 1, member 1
</td>
</tr>
<tr>
<td style="text-align:left;">
4
</td>
<td style="text-align:left;">
gene
</td>
<td style="text-align:right;">
141.7295
</td>
<td style="text-align:right;">
141.7438
</td>
<td style="text-align:left;">
positive
</td>
<td style="text-align:left;">
MGI_C57BL6J_5594022
</td>
<td style="text-align:left;">
Gm34863
</td>
<td style="text-align:left;">
NCBI_Gene:102638257
</td>
<td style="text-align:left;">
MGI:5594022
</td>
<td style="text-align:left;">
lncRNA gene
</td>
<td style="text-align:left;">
predicted gene, 34863
</td>
</tr>
<tr>
<td style="text-align:left;">
4
</td>
<td style="text-align:left;">
gene
</td>
<td style="text-align:right;">
141.7467
</td>
<td style="text-align:right;">
141.7593
</td>
<td style="text-align:left;">
positive
</td>
<td style="text-align:left;">
MGI_C57BL6J_1923236
</td>
<td style="text-align:left;">
Agmat
</td>
<td style="text-align:left;">
NCBI_Gene:75986,ENSEMBL:ENSMUSG00000040706
</td>
<td style="text-align:left;">
MGI:1923236
</td>
<td style="text-align:left;">
protein coding gene
</td>
<td style="text-align:left;">
agmatine ureohydrolase (agmatinase)
</td>
</tr>
<tr>
<td style="text-align:left;">
4
</td>
<td style="text-align:left;">
gene
</td>
<td style="text-align:right;">
141.7602
</td>
<td style="text-align:right;">
141.7909
</td>
<td style="text-align:left;">
negative
</td>
<td style="text-align:left;">
MGI_C57BL6J_2442146
</td>
<td style="text-align:left;">
Dnajc16
</td>
<td style="text-align:left;">
NCBI_Gene:214063,ENSEMBL:ENSMUSG00000040697
</td>
<td style="text-align:left;">
MGI:2442146
</td>
<td style="text-align:left;">
protein coding gene
</td>
<td style="text-align:left;">
DnaJ heat shock protein family (Hsp40) member C16
</td>
</tr>
<tr>
<td style="text-align:left;">
4
</td>
<td style="text-align:left;">
gene
</td>
<td style="text-align:right;">
141.7936
</td>
<td style="text-align:right;">
141.8160
</td>
<td style="text-align:left;">
positive
</td>
<td style="text-align:left;">
MGI_C57BL6J_1277950
</td>
<td style="text-align:left;">
Casp9
</td>
<td style="text-align:left;">
NCBI_Gene:12371,ENSEMBL:ENSMUSG00000028914
</td>
<td style="text-align:left;">
MGI:1277950
</td>
<td style="text-align:left;">
protein coding gene
</td>
<td style="text-align:left;">
caspase 9
</td>
</tr>
<tr>
<td style="text-align:left;">
4
</td>
<td style="text-align:left;">
gene
</td>
<td style="text-align:right;">
141.8150
</td>
<td style="text-align:right;">
141.8262
</td>
<td style="text-align:left;">
negative
</td>
<td style="text-align:left;">
MGI_C57BL6J_95316
</td>
<td style="text-align:left;">
Cela2a
</td>
<td style="text-align:left;">
NCBI_Gene:13706,ENSEMBL:ENSMUSG00000058579
</td>
<td style="text-align:left;">
MGI:95316
</td>
<td style="text-align:left;">
protein coding gene
</td>
<td style="text-align:left;">
chymotrypsin-like elastase family, member 2A
</td>
</tr>
<tr>
<td style="text-align:left;">
4
</td>
<td style="text-align:left;">
gene
</td>
<td style="text-align:right;">
141.8150
</td>
<td style="text-align:right;">
141.8464
</td>
<td style="text-align:left;">
negative
</td>
<td style="text-align:left;">
MGI_C57BL6J_1923951
</td>
<td style="text-align:left;">
Ctrc
</td>
<td style="text-align:left;">
NCBI_Gene:76701,ENSEMBL:ENSMUSG00000062478
</td>
<td style="text-align:left;">
MGI:1923951
</td>
<td style="text-align:left;">
protein coding gene
</td>
<td style="text-align:left;">
chymotrypsin C (caldecrin)
</td>
</tr>
<tr>
<td style="text-align:left;">
4
</td>
<td style="text-align:left;">
gene
</td>
<td style="text-align:right;">
141.8443
</td>
<td style="text-align:right;">
141.8470
</td>
<td style="text-align:left;">
positive
</td>
<td style="text-align:left;">
MGI_C57BL6J_1916389
</td>
<td style="text-align:left;">
Ctrcos
</td>
<td style="text-align:left;">
NCBI_Gene:69139,ENSEMBL:ENSMUSG00000086818
</td>
<td style="text-align:left;">
MGI:1916389
</td>
<td style="text-align:left;">
antisense lncRNA gene
</td>
<td style="text-align:left;">
chymotrypsin C (caldecrin), opposite strand
</td>
</tr>
<tr>
<td style="text-align:left;">
4
</td>
<td style="text-align:left;">
gene
</td>
<td style="text-align:right;">
141.8483
</td>
<td style="text-align:right;">
141.8585
</td>
<td style="text-align:left;">
positive
</td>
<td style="text-align:left;">
MGI_C57BL6J_5594193
</td>
<td style="text-align:left;">
Gm35034
</td>
<td style="text-align:left;">
NCBI_Gene:108168984
</td>
<td style="text-align:left;">
MGI:5594193
</td>
<td style="text-align:left;">
lncRNA gene
</td>
<td style="text-align:left;">
predicted gene, 35034
</td>
</tr>
<tr>
<td style="text-align:left;">
4
</td>
<td style="text-align:left;">
gene
</td>
<td style="text-align:right;">
141.8581
</td>
<td style="text-align:right;">
141.8749
</td>
<td style="text-align:left;">
negative
</td>
<td style="text-align:left;">
MGI_C57BL6J_106504
</td>
<td style="text-align:left;">
Efhd2
</td>
<td style="text-align:left;">
NCBI_Gene:27984,ENSEMBL:ENSMUSG00000040659
</td>
<td style="text-align:left;">
MGI:106504
</td>
<td style="text-align:left;">
protein coding gene
</td>
<td style="text-align:left;">
EF hand domain containing 2
</td>
</tr>
<tr>
<td style="text-align:left;">
4
</td>
<td style="text-align:left;">
gene
</td>
<td style="text-align:right;">
141.8757
</td>
<td style="text-align:right;">
141.8770
</td>
<td style="text-align:left;">
positive
</td>
<td style="text-align:left;">
MGI_C57BL6J_5580073
</td>
<td style="text-align:left;">
Gm29367
</td>
<td style="text-align:left;">
ENSEMBL:ENSMUSG00000100396
</td>
<td style="text-align:left;">
MGI:5580073
</td>
<td style="text-align:left;">
lncRNA gene
</td>
<td style="text-align:left;">
predicted gene 29367
</td>
</tr>
<tr>
<td style="text-align:left;">
4
</td>
<td style="text-align:left;">
gene
</td>
<td style="text-align:right;">
141.8797
</td>
<td style="text-align:right;">
141.8856
</td>
<td style="text-align:left;">
negative
</td>
<td style="text-align:left;">
MGI_C57BL6J_5826511
</td>
<td style="text-align:left;">
Gm46874
</td>
<td style="text-align:left;">
NCBI_Gene:108168983
</td>
<td style="text-align:left;">
MGI:5826511
</td>
<td style="text-align:left;">
lncRNA gene
</td>
<td style="text-align:left;">
predicted gene, 46874
</td>
</tr>
<tr>
<td style="text-align:left;">
4
</td>
<td style="text-align:left;">
gene
</td>
<td style="text-align:right;">
141.8813
</td>
<td style="text-align:right;">
141.9003
</td>
<td style="text-align:left;">
positive
</td>
<td style="text-align:left;">
MGI_C57BL6J_3651870
</td>
<td style="text-align:left;">
Fhad1os2
</td>
<td style="text-align:left;">
NCBI_Gene:102638933,ENSEMBL:ENSMUSG00000085922
</td>
<td style="text-align:left;">
MGI:3651870
</td>
<td style="text-align:left;">
antisense lncRNA gene
</td>
<td style="text-align:left;">
forkhead-associated (FHA) phosphopeptide binding domain 1, opposite
strand 2
</td>
</tr>
<tr>
<td style="text-align:left;">
4
</td>
<td style="text-align:left;">
gene
</td>
<td style="text-align:right;">
141.8904
</td>
<td style="text-align:right;">
142.0151
</td>
<td style="text-align:left;">
negative
</td>
<td style="text-align:left;">
MGI_C57BL6J_1920323
</td>
<td style="text-align:left;">
Fhad1
</td>
<td style="text-align:left;">
NCBI_Gene:329977,ENSEMBL:ENSMUSG00000051435
</td>
<td style="text-align:left;">
MGI:1920323
</td>
<td style="text-align:left;">
protein coding gene
</td>
<td style="text-align:left;">
forkhead-associated (FHA) phosphopeptide binding domain 1
</td>
</tr>
<tr>
<td style="text-align:left;">
4
</td>
<td style="text-align:left;">
gene
</td>
<td style="text-align:right;">
141.9099
</td>
<td style="text-align:right;">
141.9159
</td>
<td style="text-align:left;">
positive
</td>
<td style="text-align:left;">
MGI_C57BL6J_5625223
</td>
<td style="text-align:left;">
Gm42338
</td>
<td style="text-align:left;">
NCBI_Gene:105247199
</td>
<td style="text-align:left;">
MGI:5625223
</td>
<td style="text-align:left;">
lncRNA gene
</td>
<td style="text-align:left;">
predicted gene, 42338
</td>
</tr>
<tr>
<td style="text-align:left;">
4
</td>
<td style="text-align:left;">
gene
</td>
<td style="text-align:right;">
141.9209
</td>
<td style="text-align:right;">
141.9328
</td>
<td style="text-align:left;">
positive
</td>
<td style="text-align:left;">
MGI_C57BL6J_5594454
</td>
<td style="text-align:left;">
Gm35295
</td>
<td style="text-align:left;">
NCBI_Gene:102638820
</td>
<td style="text-align:left;">
MGI:5594454
</td>
<td style="text-align:left;">
lncRNA gene
</td>
<td style="text-align:left;">
predicted gene, 35295
</td>
</tr>
<tr>
<td style="text-align:left;">
4
</td>
<td style="text-align:left;">
pseudogene
</td>
<td style="text-align:right;">
141.9768
</td>
<td style="text-align:right;">
141.9773
</td>
<td style="text-align:left;">
positive
</td>
<td style="text-align:left;">
MGI_C57BL6J_3650255
</td>
<td style="text-align:left;">
Gm13059
</td>
<td style="text-align:left;">
ENSEMBL:ENSMUSG00000081079
</td>
<td style="text-align:left;">
MGI:3650255
</td>
<td style="text-align:left;">
pseudogene
</td>
<td style="text-align:left;">
predicted gene 13059
</td>
</tr>
<tr>
<td style="text-align:left;">
4
</td>
<td style="text-align:left;">
gene
</td>
<td style="text-align:right;">
141.9830
</td>
<td style="text-align:right;">
141.9868
</td>
<td style="text-align:left;">
positive
</td>
<td style="text-align:left;">
MGI_C57BL6J_3649995
</td>
<td style="text-align:left;">
Fhad1os1
</td>
<td style="text-align:left;">
NCBI_Gene:545691,ENSEMBL:ENSMUSG00000085424
</td>
<td style="text-align:left;">
MGI:3649995
</td>
<td style="text-align:left;">
antisense lncRNA gene
</td>
<td style="text-align:left;">
forkhead-associated (FHA) phosphopeptide binding domain 1, opposite
strand 1
</td>
</tr>
<tr>
<td style="text-align:left;">
4
</td>
<td style="text-align:left;">
gene
</td>
<td style="text-align:right;">
142.0179
</td>
<td style="text-align:right;">
142.0290
</td>
<td style="text-align:left;">
positive
</td>
<td style="text-align:left;">
MGI_C57BL6J_1926167
</td>
<td style="text-align:left;">
4930455G09Rik
</td>
<td style="text-align:left;">
NCBI_Gene:78917,ENSEMBL:ENSMUSG00000085761
</td>
<td style="text-align:left;">
MGI:1926167
</td>
<td style="text-align:left;">
lncRNA gene
</td>
<td style="text-align:left;">
RIKEN cDNA 4930455G09 gene
</td>
</tr>
<tr>
<td style="text-align:left;">
4
</td>
<td style="text-align:left;">
gene
</td>
<td style="text-align:right;">
142.0310
</td>
<td style="text-align:right;">
142.0846
</td>
<td style="text-align:left;">
negative
</td>
<td style="text-align:left;">
MGI_C57BL6J_2384874
</td>
<td style="text-align:left;">
Tmem51
</td>
<td style="text-align:left;">
NCBI_Gene:214359,ENSEMBL:ENSMUSG00000040616
</td>
<td style="text-align:left;">
MGI:2384874
</td>
<td style="text-align:left;">
protein coding gene
</td>
<td style="text-align:left;">
transmembrane protein 51
</td>
</tr>
<tr>
<td style="text-align:left;">
4
</td>
<td style="text-align:left;">
gene
</td>
<td style="text-align:right;">
142.0429
</td>
<td style="text-align:right;">
142.0486
</td>
<td style="text-align:left;">
negative
</td>
<td style="text-align:left;">
MGI_C57BL6J_3651989
</td>
<td style="text-align:left;">
Gm13053
</td>
<td style="text-align:left;">
ENSEMBL:ENSMUSG00000085574
</td>
<td style="text-align:left;">
MGI:3651989
</td>
<td style="text-align:left;">
lncRNA gene
</td>
<td style="text-align:left;">
predicted gene 13053
</td>
</tr>
<tr>
<td style="text-align:left;">
4
</td>
<td style="text-align:left;">
gene
</td>
<td style="text-align:right;">
142.0840
</td>
<td style="text-align:right;">
142.0881
</td>
<td style="text-align:left;">
positive
</td>
<td style="text-align:left;">
MGI_C57BL6J_3642733
</td>
<td style="text-align:left;">
Tmem51os1
</td>
<td style="text-align:left;">
NCBI_Gene:100038693,ENSEMBL:ENSMUSG00000073728
</td>
<td style="text-align:left;">
MGI:3642733
</td>
<td style="text-align:left;">
antisense lncRNA gene
</td>
<td style="text-align:left;">
Tmem51 opposite strand 1
</td>
</tr>
<tr>
<td style="text-align:left;">
4
</td>
<td style="text-align:left;">
gene
</td>
<td style="text-align:right;">
142.1024
</td>
<td style="text-align:right;">
142.2394
</td>
<td style="text-align:left;">
negative
</td>
<td style="text-align:left;">
MGI_C57BL6J_1918779
</td>
<td style="text-align:left;">
Kazn
</td>
<td style="text-align:left;">
NCBI_Gene:71529,ENSEMBL:ENSMUSG00000040606
</td>
<td style="text-align:left;">
MGI:1918779
</td>
<td style="text-align:left;">
protein coding gene
</td>
<td style="text-align:left;">
kazrin, periplakin interacting protein
</td>
</tr>
<tr>
<td style="text-align:left;">
4
</td>
<td style="text-align:left;">
gene
</td>
<td style="text-align:right;">
142.1267
</td>
<td style="text-align:right;">
142.1282
</td>
<td style="text-align:left;">
positive
</td>
<td style="text-align:left;">
MGI_C57BL6J_5594740
</td>
<td style="text-align:left;">
Gm35581
</td>
<td style="text-align:left;">
NCBI_Gene:102639222
</td>
<td style="text-align:left;">
MGI:5594740
</td>
<td style="text-align:left;">
lncRNA gene
</td>
<td style="text-align:left;">
predicted gene, 35581
</td>
</tr>
<tr>
<td style="text-align:left;">
4
</td>
<td style="text-align:left;">
gene
</td>
<td style="text-align:right;">
142.1806
</td>
<td style="text-align:right;">
142.1841
</td>
<td style="text-align:left;">
positive
</td>
<td style="text-align:left;">
MGI_C57BL6J_3649476
</td>
<td style="text-align:left;">
Gm13062
</td>
<td style="text-align:left;">
ENSEMBL:ENSMUSG00000087525
</td>
<td style="text-align:left;">
MGI:3649476
</td>
<td style="text-align:left;">
lncRNA gene
</td>
<td style="text-align:left;">
predicted gene 13062
</td>
</tr>
<tr>
<td style="text-align:left;">
4
</td>
<td style="text-align:left;">
gene
</td>
<td style="text-align:right;">
142.2394
</td>
<td style="text-align:right;">
142.2426
</td>
<td style="text-align:left;">
positive
</td>
<td style="text-align:left;">
MGI_C57BL6J_5594690
</td>
<td style="text-align:left;">
Gm35531
</td>
<td style="text-align:left;">
NCBI_Gene:102639151
</td>
<td style="text-align:left;">
MGI:5594690
</td>
<td style="text-align:left;">
lncRNA gene
</td>
<td style="text-align:left;">
predicted gene, 35531
</td>
</tr>
<tr>
<td style="text-align:left;">
4
</td>
<td style="text-align:left;">
gene
</td>
<td style="text-align:right;">
142.2557
</td>
<td style="text-align:right;">
142.2561
</td>
<td style="text-align:left;">
negative
</td>
<td style="text-align:left;">
MGI_C57BL6J_1920742
</td>
<td style="text-align:left;">
1700085G17Rik
</td>
<td style="text-align:left;">
NA
</td>
<td style="text-align:left;">
NA
</td>
<td style="text-align:left;">
unclassified gene
</td>
<td style="text-align:left;">
RIKEN cDNA 1700085G17 gene
</td>
</tr>
<tr>
<td style="text-align:left;">
4
</td>
<td style="text-align:left;">
gene
</td>
<td style="text-align:right;">
142.3078
</td>
<td style="text-align:right;">
142.3465
</td>
<td style="text-align:left;">
negative
</td>
<td style="text-align:left;">
MGI_C57BL6J_3651990
</td>
<td style="text-align:left;">
Gm13052
</td>
<td style="text-align:left;">
ENSEMBL:ENSMUSG00000085816
</td>
<td style="text-align:left;">
MGI:3651990
</td>
<td style="text-align:left;">
lncRNA gene
</td>
<td style="text-align:left;">
predicted gene 13052
</td>
</tr>
<tr>
<td style="text-align:left;">
4
</td>
<td style="text-align:left;">
gene
</td>
<td style="text-align:right;">
142.3487
</td>
<td style="text-align:right;">
142.3610
</td>
<td style="text-align:left;">
positive
</td>
<td style="text-align:left;">
MGI_C57BL6J_5826512
</td>
<td style="text-align:left;">
Gm46875
</td>
<td style="text-align:left;">
NCBI_Gene:108168985
</td>
<td style="text-align:left;">
MGI:5826512
</td>
<td style="text-align:left;">
lncRNA gene
</td>
<td style="text-align:left;">
predicted gene, 46875
</td>
</tr>
<tr>
<td style="text-align:left;">
4
</td>
<td style="text-align:left;">
gene
</td>
<td style="text-align:right;">
142.4071
</td>
<td style="text-align:right;">
142.4143
</td>
<td style="text-align:left;">
negative
</td>
<td style="text-align:left;">
MGI_C57BL6J_5625224
</td>
<td style="text-align:left;">
Gm42339
</td>
<td style="text-align:left;">
NCBI_Gene:105247200
</td>
<td style="text-align:left;">
MGI:5625224
</td>
<td style="text-align:left;">
lncRNA gene
</td>
<td style="text-align:left;">
predicted gene, 42339
</td>
</tr>
<tr>
<td style="text-align:left;">
4
</td>
<td style="text-align:left;">
gene
</td>
<td style="text-align:right;">
142.4217
</td>
<td style="text-align:right;">
142.4230
</td>
<td style="text-align:left;">
negative
</td>
<td style="text-align:left;">
MGI_C57BL6J_3665284
</td>
<td style="text-align:left;">
C030029H13Rik
</td>
<td style="text-align:left;">
NA
</td>
<td style="text-align:left;">
NA
</td>
<td style="text-align:left;">
unclassified gene
</td>
<td style="text-align:left;">
Riken cDNA C030029H13 gene
</td>
</tr>
<tr>
<td style="text-align:left;">
4
</td>
<td style="text-align:left;">
gene
</td>
<td style="text-align:right;">
142.4956
</td>
<td style="text-align:right;">
142.4996
</td>
<td style="text-align:left;">
positive
</td>
<td style="text-align:left;">
MGI_C57BL6J_5594911
</td>
<td style="text-align:left;">
Gm35752
</td>
<td style="text-align:left;">
NCBI_Gene:102639438
</td>
<td style="text-align:left;">
MGI:5594911
</td>
<td style="text-align:left;">
lncRNA gene
</td>
<td style="text-align:left;">
predicted gene, 35752
</td>
</tr>
<tr>
<td style="text-align:left;">
4
</td>
<td style="text-align:left;">
gene
</td>
<td style="text-align:right;">
142.5984
</td>
<td style="text-align:right;">
143.0748
</td>
<td style="text-align:left;">
negative
</td>
<td style="text-align:left;">
MGI_C57BL6J_1917975
</td>
<td style="text-align:left;">
6330411D24Rik
</td>
<td style="text-align:left;">
ENSEMBL:ENSMUSG00000110067
</td>
<td style="text-align:left;">
MGI:1917975
</td>
<td style="text-align:left;">
lncRNA gene
</td>
<td style="text-align:left;">
RIKEN cDNA 6330411D24 gene
</td>
</tr>
<tr>
<td style="text-align:left;">
4
</td>
<td style="text-align:left;">
gene
</td>
<td style="text-align:right;">
142.6317
</td>
<td style="text-align:right;">
142.6354
</td>
<td style="text-align:left;">
positive
</td>
<td style="text-align:left;">
MGI_C57BL6J_5625227
</td>
<td style="text-align:left;">
Gm42342
</td>
<td style="text-align:left;">
NCBI_Gene:105247203
</td>
<td style="text-align:left;">
MGI:5625227
</td>
<td style="text-align:left;">
lncRNA gene
</td>
<td style="text-align:left;">
predicted gene, 42342
</td>
</tr>
<tr>
<td style="text-align:left;">
4
</td>
<td style="text-align:left;">
gene
</td>
<td style="text-align:right;">
142.6418
</td>
<td style="text-align:right;">
142.6631
</td>
<td style="text-align:left;">
positive
</td>
<td style="text-align:left;">
MGI_C57BL6J_5625225
</td>
<td style="text-align:left;">
Gm42340
</td>
<td style="text-align:left;">
NCBI_Gene:105247201
</td>
<td style="text-align:left;">
MGI:5625225
</td>
<td style="text-align:left;">
lncRNA gene
</td>
<td style="text-align:left;">
predicted gene, 42340
</td>
</tr>
<tr>
<td style="text-align:left;">
4
</td>
<td style="text-align:left;">
gene
</td>
<td style="text-align:right;">
142.6434
</td>
<td style="text-align:right;">
142.6476
</td>
<td style="text-align:left;">
negative
</td>
<td style="text-align:left;">
MGI_C57BL6J_5625226
</td>
<td style="text-align:left;">
Gm42341
</td>
<td style="text-align:left;">
NCBI_Gene:105247202
</td>
<td style="text-align:left;">
MGI:5625226
</td>
<td style="text-align:left;">
lncRNA gene
</td>
<td style="text-align:left;">
predicted gene, 42341
</td>
</tr>
<tr>
<td style="text-align:left;">
4
</td>
<td style="text-align:left;">
gene
</td>
<td style="text-align:right;">
142.7908
</td>
<td style="text-align:right;">
142.7910
</td>
<td style="text-align:left;">
positive
</td>
<td style="text-align:left;">
MGI_C57BL6J_5610852
</td>
<td style="text-align:left;">
Gm37624
</td>
<td style="text-align:left;">
ENSEMBL:ENSMUSG00000103678
</td>
<td style="text-align:left;">
MGI:5610852
</td>
<td style="text-align:left;">
unclassified gene
</td>
<td style="text-align:left;">
predicted gene, 37624
</td>
</tr>
<tr>
<td style="text-align:left;">
4
</td>
<td style="text-align:left;">
gene
</td>
<td style="text-align:right;">
143.1027
</td>
<td style="text-align:right;">
143.1063
</td>
<td style="text-align:left;">
negative
</td>
<td style="text-align:left;">
MGI_C57BL6J_1926077
</td>
<td style="text-align:left;">
5830426C09Rik
</td>
<td style="text-align:left;">
NA
</td>
<td style="text-align:left;">
NA
</td>
<td style="text-align:left;">
unclassified gene
</td>
<td style="text-align:left;">
RIKEN cDNA 5830426C09 gene
</td>
</tr>
<tr>
<td style="text-align:left;">
4
</td>
<td style="text-align:left;">
gene
</td>
<td style="text-align:right;">
143.1074
</td>
<td style="text-align:right;">
143.2130
</td>
<td style="text-align:left;">
negative
</td>
<td style="text-align:left;">
MGI_C57BL6J_107628
</td>
<td style="text-align:left;">
Prdm2
</td>
<td style="text-align:left;">
NCBI_Gene:110593,ENSEMBL:ENSMUSG00000057637
</td>
<td style="text-align:left;">
MGI:107628
</td>
<td style="text-align:left;">
protein coding gene
</td>
<td style="text-align:left;">
PR domain containing 2, with ZNF domain
</td>
</tr>
<tr>
<td style="text-align:left;">
4
</td>
<td style="text-align:left;">
gene
</td>
<td style="text-align:right;">
143.1361
</td>
<td style="text-align:right;">
143.1362
</td>
<td style="text-align:left;">
negative
</td>
<td style="text-align:left;">
MGI_C57BL6J_5562764
</td>
<td style="text-align:left;">
Mir7021
</td>
<td style="text-align:left;">
miRBase:MI0022870,NCBI_Gene:102465618,ENSEMBL:ENSMUSG00000106597
</td>
<td style="text-align:left;">
MGI:5562764
</td>
<td style="text-align:left;">
miRNA gene
</td>
<td style="text-align:left;">
microRNA 7021
</td>
</tr>
<tr>
<td style="text-align:left;">
4
</td>
<td style="text-align:left;">
pseudogene
</td>
<td style="text-align:right;">
143.1526
</td>
<td style="text-align:right;">
143.1530
</td>
<td style="text-align:left;">
positive
</td>
<td style="text-align:left;">
MGI_C57BL6J_3650684
</td>
<td style="text-align:left;">
Gm13038
</td>
<td style="text-align:left;">
ENSEMBL:ENSMUSG00000081838
</td>
<td style="text-align:left;">
MGI:3650684
</td>
<td style="text-align:left;">
pseudogene
</td>
<td style="text-align:left;">
predicted gene 13038
</td>
</tr>
<tr>
<td style="text-align:left;">
4
</td>
<td style="text-align:left;">
pseudogene
</td>
<td style="text-align:right;">
143.1666
</td>
<td style="text-align:right;">
143.1674
</td>
<td style="text-align:left;">
positive
</td>
<td style="text-align:left;">
MGI_C57BL6J_3649997
</td>
<td style="text-align:left;">
Gm13039
</td>
<td style="text-align:left;">
NCBI_Gene:100384872,ENSEMBL:ENSMUSG00000083826
</td>
<td style="text-align:left;">
MGI:3649997
</td>
<td style="text-align:left;">
pseudogene
</td>
<td style="text-align:left;">
predicted gene 13039
</td>
</tr>
<tr>
<td style="text-align:left;">
4
</td>
<td style="text-align:left;">
gene
</td>
<td style="text-align:right;">
143.2041
</td>
<td style="text-align:right;">
143.2042
</td>
<td style="text-align:left;">
positive
</td>
<td style="text-align:left;">
MGI_C57BL6J_5451816
</td>
<td style="text-align:left;">
Gm22039
</td>
<td style="text-align:left;">
ENSEMBL:ENSMUSG00000087903
</td>
<td style="text-align:left;">
MGI:5451816
</td>
<td style="text-align:left;">
snoRNA gene
</td>
<td style="text-align:left;">
predicted gene, 22039
</td>
</tr>
<tr>
<td style="text-align:left;">
4
</td>
<td style="text-align:left;">
gene
</td>
<td style="text-align:right;">
143.2120
</td>
<td style="text-align:right;">
143.2150
</td>
<td style="text-align:left;">
positive
</td>
<td style="text-align:left;">
MGI_C57BL6J_3646960
</td>
<td style="text-align:left;">
Gm6687
</td>
<td style="text-align:left;">
NA
</td>
<td style="text-align:left;">
NA
</td>
<td style="text-align:left;">
protein coding gene
</td>
<td style="text-align:left;">
predicted gene 6687
</td>
</tr>
<tr>
<td style="text-align:left;">
4
</td>
<td style="text-align:left;">
pseudogene
</td>
<td style="text-align:right;">
143.2420
</td>
<td style="text-align:right;">
143.2434
</td>
<td style="text-align:left;">
negative
</td>
<td style="text-align:left;">
MGI_C57BL6J_3650664
</td>
<td style="text-align:left;">
Eef2-ps2
</td>
<td style="text-align:left;">
NCBI_Gene:433776,ENSEMBL:ENSMUSG00000082575
</td>
<td style="text-align:left;">
MGI:3650664
</td>
<td style="text-align:left;">
pseudogene
</td>
<td style="text-align:left;">
eukaryotic translation elongation factor 2, pseudogene 2
</td>
</tr>
<tr>
<td style="text-align:left;">
4
</td>
<td style="text-align:left;">
gene
</td>
<td style="text-align:right;">
143.2674
</td>
<td style="text-align:right;">
143.2996
</td>
<td style="text-align:left;">
negative
</td>
<td style="text-align:left;">
MGI_C57BL6J_103098
</td>
<td style="text-align:left;">
Pdpn
</td>
<td style="text-align:left;">
NCBI_Gene:14726,ENSEMBL:ENSMUSG00000028583
</td>
<td style="text-align:left;">
MGI:103098
</td>
<td style="text-align:left;">
protein coding gene
</td>
<td style="text-align:left;">
podoplanin
</td>
</tr>
<tr>
<td style="text-align:left;">
4
</td>
<td style="text-align:left;">
gene
</td>
<td style="text-align:right;">
143.3404
</td>
<td style="text-align:right;">
143.3407
</td>
<td style="text-align:left;">
positive
</td>
<td style="text-align:left;">
MGI_C57BL6J_5456090
</td>
<td style="text-align:left;">
Gm26313
</td>
<td style="text-align:left;">
ENSEMBL:ENSMUSG00000088395
</td>
<td style="text-align:left;">
MGI:5456090
</td>
<td style="text-align:left;">
unclassified non-coding RNA gene
</td>
<td style="text-align:left;">
predicted gene, 26313
</td>
</tr>
<tr>
<td style="text-align:left;">
4
</td>
<td style="text-align:left;">
gene
</td>
<td style="text-align:right;">
143.3465
</td>
<td style="text-align:right;">
143.3710
</td>
<td style="text-align:left;">
positive
</td>
<td style="text-align:left;">
MGI_C57BL6J_2442845
</td>
<td style="text-align:left;">
Lrrc38
</td>
<td style="text-align:left;">
NCBI_Gene:242735,ENSEMBL:ENSMUSG00000028584
</td>
<td style="text-align:left;">
MGI:2442845
</td>
<td style="text-align:left;">
protein coding gene
</td>
<td style="text-align:left;">
leucine rich repeat containing 38
</td>
</tr>
<tr>
<td style="text-align:left;">
4
</td>
<td style="text-align:left;">
pseudogene
</td>
<td style="text-align:right;">
143.3895
</td>
<td style="text-align:right;">
143.3905
</td>
<td style="text-align:left;">
positive
</td>
<td style="text-align:left;">
MGI_C57BL6J_3651262
</td>
<td style="text-align:left;">
Anp32b-ps1
</td>
<td style="text-align:left;">
NCBI_Gene:621961,ENSEMBL:ENSMUSG00000081792
</td>
<td style="text-align:left;">
MGI:3651262
</td>
<td style="text-align:left;">
pseudogene
</td>
<td style="text-align:left;">
Bacidic (leucine-rich) nuclear phosphoprotein 32 family, member B,
pseudogene 1
</td>
</tr>
<tr>
<td style="text-align:left;">
4
</td>
<td style="text-align:left;">
gene
</td>
<td style="text-align:right;">
143.3944
</td>
<td style="text-align:right;">
143.4002
</td>
<td style="text-align:left;">
positive
</td>
<td style="text-align:left;">
MGI_C57BL6J_1890541
</td>
<td style="text-align:left;">
Pramel1
</td>
<td style="text-align:left;">
NCBI_Gene:83491,ENSEMBL:ENSMUSG00000041805
</td>
<td style="text-align:left;">
MGI:1890541
</td>
<td style="text-align:left;">
protein coding gene
</td>
<td style="text-align:left;">
preferentially expressed antigen in melanoma-like 1
</td>
</tr>
<tr>
<td style="text-align:left;">
4
</td>
<td style="text-align:left;">
pseudogene
</td>
<td style="text-align:right;">
143.4023
</td>
<td style="text-align:right;">
143.4034
</td>
<td style="text-align:left;">
positive
</td>
<td style="text-align:left;">
MGI_C57BL6J_3649502
</td>
<td style="text-align:left;">
Gm13045
</td>
<td style="text-align:left;">
ENSEMBL:ENSMUSG00000082407
</td>
<td style="text-align:left;">
MGI:3649502
</td>
<td style="text-align:left;">
pseudogene
</td>
<td style="text-align:left;">
predicted gene 13045
</td>
</tr>
<tr>
<td style="text-align:left;">
4
</td>
<td style="text-align:left;">
gene
</td>
<td style="text-align:right;">
143.4123
</td>
<td style="text-align:right;">
143.4211
</td>
<td style="text-align:left;">
positive
</td>
<td style="text-align:left;">
MGI_C57BL6J_2140473
</td>
<td style="text-align:left;">
Pramef8
</td>
<td style="text-align:left;">
NCBI_Gene:242736,ENSEMBL:ENSMUSG00000046862
</td>
<td style="text-align:left;">
MGI:2140473
</td>
<td style="text-align:left;">
protein coding gene
</td>
<td style="text-align:left;">
PRAME family member 8
</td>
</tr>
<tr>
<td style="text-align:left;">
4
</td>
<td style="text-align:left;">
gene
</td>
<td style="text-align:right;">
143.4372
</td>
<td style="text-align:right;">
143.4503
</td>
<td style="text-align:left;">
negative
</td>
<td style="text-align:left;">
MGI_C57BL6J_2684051
</td>
<td style="text-align:left;">
Oog4
</td>
<td style="text-align:left;">
NCBI_Gene:242737,ENSEMBL:ENSMUSG00000047799
</td>
<td style="text-align:left;">
MGI:2684051
</td>
<td style="text-align:left;">
protein coding gene
</td>
<td style="text-align:left;">
oogenesin 4
</td>
</tr>
<tr>
<td style="text-align:left;">
4
</td>
<td style="text-align:left;">
pseudogene
</td>
<td style="text-align:right;">
143.4568
</td>
<td style="text-align:right;">
143.4606
</td>
<td style="text-align:left;">
negative
</td>
<td style="text-align:left;">
MGI_C57BL6J_3651155
</td>
<td style="text-align:left;">
Gm13042
</td>
<td style="text-align:left;">
NCBI_Gene:666015,ENSEMBL:ENSMUSG00000083930
</td>
<td style="text-align:left;">
MGI:3651155
</td>
<td style="text-align:left;">
pseudogene
</td>
<td style="text-align:left;">
predicted gene 13042
</td>
</tr>
<tr>
<td style="text-align:left;">
4
</td>
<td style="text-align:left;">
pseudogene
</td>
<td style="text-align:right;">
143.4643
</td>
<td style="text-align:right;">
143.4648
</td>
<td style="text-align:left;">
negative
</td>
<td style="text-align:left;">
MGI_C57BL6J_3649503
</td>
<td style="text-align:left;">
Gm13046
</td>
<td style="text-align:left;">
ENSEMBL:ENSMUSG00000084059
</td>
<td style="text-align:left;">
MGI:3649503
</td>
<td style="text-align:left;">
pseudogene
</td>
<td style="text-align:left;">
predicted gene 13046
</td>
</tr>
<tr>
<td style="text-align:left;">
4
</td>
<td style="text-align:left;">
pseudogene
</td>
<td style="text-align:right;">
143.4994
</td>
<td style="text-align:right;">
143.5025
</td>
<td style="text-align:left;">
positive
</td>
<td style="text-align:left;">
MGI_C57BL6J_3649499
</td>
<td style="text-align:left;">
Gm13041
</td>
<td style="text-align:left;">
NCBI_Gene:626700,ENSEMBL:ENSMUSG00000083403
</td>
<td style="text-align:left;">
MGI:3649499
</td>
<td style="text-align:left;">
pseudogene
</td>
<td style="text-align:left;">
predicted gene 13041
</td>
</tr>
<tr>
<td style="text-align:left;">
4
</td>
<td style="text-align:left;">
gene
</td>
<td style="text-align:right;">
143.5113
</td>
<td style="text-align:right;">
143.5178
</td>
<td style="text-align:left;">
positive
</td>
<td style="text-align:left;">
MGI_C57BL6J_3649500
</td>
<td style="text-align:left;">
Gm13043
</td>
<td style="text-align:left;">
NCBI_Gene:545693,ENSEMBL:ENSMUSG00000095409
</td>
<td style="text-align:left;">
MGI:3649500
</td>
<td style="text-align:left;">
protein coding gene
</td>
<td style="text-align:left;">
predicted gene 13043
</td>
</tr>
<tr>
<td style="text-align:left;">
4
</td>
<td style="text-align:left;">
gene
</td>
<td style="text-align:right;">
143.5360
</td>
<td style="text-align:right;">
143.5425
</td>
<td style="text-align:left;">
positive
</td>
<td style="text-align:left;">
MGI_C57BL6J_3649498
</td>
<td style="text-align:left;">
Gm13040
</td>
<td style="text-align:left;">
NCBI_Gene:100040854,ENSEMBL:ENSMUSG00000070616
</td>
<td style="text-align:left;">
MGI:3649498
</td>
<td style="text-align:left;">
protein coding gene
</td>
<td style="text-align:left;">
predicted gene 13040
</td>
</tr>
<tr>
<td style="text-align:left;">
4
</td>
<td style="text-align:left;">
gene
</td>
<td style="text-align:right;">
143.5517
</td>
<td style="text-align:right;">
143.5582
</td>
<td style="text-align:left;">
positive
</td>
<td style="text-align:left;">
MGI_C57BL6J_3649690
</td>
<td style="text-align:left;">
Gm13057
</td>
<td style="text-align:left;">
NCBI_Gene:100040861,ENSEMBL:ENSMUSG00000096154
</td>
<td style="text-align:left;">
MGI:3649690
</td>
<td style="text-align:left;">
protein coding gene
</td>
<td style="text-align:left;">
predicted gene 13057
</td>
</tr>
<tr>
<td style="text-align:left;">
4
</td>
<td style="text-align:left;">
gene
</td>
<td style="text-align:right;">
143.5517
</td>
<td style="text-align:right;">
143.5738
</td>
<td style="text-align:left;">
positive
</td>
<td style="text-align:left;">
MGI_C57BL6J_3525148
</td>
<td style="text-align:left;">
BC080695
</td>
<td style="text-align:left;">
NCBI_Gene:329986,ENSEMBL:ENSMUSG00000070618
</td>
<td style="text-align:left;">
MGI:3525148
</td>
<td style="text-align:left;">
protein coding gene
</td>
<td style="text-align:left;">
cDNA sequence BC080695
</td>
</tr>
<tr>
<td style="text-align:left;">
4
</td>
<td style="text-align:left;">
pseudogene
</td>
<td style="text-align:right;">
143.5950
</td>
<td style="text-align:right;">
143.6010
</td>
<td style="text-align:left;">
negative
</td>
<td style="text-align:left;">
MGI_C57BL6J_3650002
</td>
<td style="text-align:left;">
Gm13058
</td>
<td style="text-align:left;">
NCBI_Gene:666049,ENSEMBL:ENSMUSG00000081523
</td>
<td style="text-align:left;">
MGI:3650002
</td>
<td style="text-align:left;">
pseudogene
</td>
<td style="text-align:left;">
predicted gene 13058
</td>
</tr>
<tr>
<td style="text-align:left;">
4
</td>
<td style="text-align:left;">
pseudogene
</td>
<td style="text-align:right;">
143.6116
</td>
<td style="text-align:right;">
143.6122
</td>
<td style="text-align:left;">
negative
</td>
<td style="text-align:left;">
MGI_C57BL6J_3650235
</td>
<td style="text-align:left;">
Gm13080
</td>
<td style="text-align:left;">
NCBI_Gene:100044633,ENSEMBL:ENSMUSG00000044377
</td>
<td style="text-align:left;">
MGI:3650235
</td>
<td style="text-align:left;">
pseudogene
</td>
<td style="text-align:left;">
predicted gene 13080
</td>
</tr>
<tr>
<td style="text-align:left;">
4
</td>
<td style="text-align:left;">
gene
</td>
<td style="text-align:right;">
143.6150
</td>
<td style="text-align:right;">
143.6187
</td>
<td style="text-align:left;">
positive
</td>
<td style="text-align:left;">
MGI_C57BL6J_3650232
</td>
<td style="text-align:left;">
Gm13083
</td>
<td style="text-align:left;">
NCBI_Gene:279185,ENSEMBL:ENSMUSG00000066688
</td>
<td style="text-align:left;">
MGI:3650232
</td>
<td style="text-align:left;">
protein coding gene
</td>
<td style="text-align:left;">
predicted gene 13083
</td>
</tr>
<tr>
<td style="text-align:left;">
4
</td>
<td style="text-align:left;">
gene
</td>
<td style="text-align:right;">
143.6538
</td>
<td style="text-align:right;">
143.6606
</td>
<td style="text-align:left;">
negative
</td>
<td style="text-align:left;">
MGI_C57BL6J_3649971
</td>
<td style="text-align:left;">
Gm13088
</td>
<td style="text-align:left;">
NCBI_Gene:277668,ENSEMBL:ENSMUSG00000078513
</td>
<td style="text-align:left;">
MGI:3649971
</td>
<td style="text-align:left;">
protein coding gene
</td>
<td style="text-align:left;">
predicted gene 13088
</td>
</tr>
<tr>
<td style="text-align:left;">
4
</td>
<td style="text-align:left;">
pseudogene
</td>
<td style="text-align:right;">
143.6631
</td>
<td style="text-align:right;">
143.6638
</td>
<td style="text-align:left;">
negative
</td>
<td style="text-align:left;">
MGI_C57BL6J_3650236
</td>
<td style="text-align:left;">
Gm13085
</td>
<td style="text-align:left;">
ENSEMBL:ENSMUSG00000084351
</td>
<td style="text-align:left;">
MGI:3650236
</td>
<td style="text-align:left;">
pseudogene
</td>
<td style="text-align:left;">
predicted gene 13085
</td>
</tr>
<tr>
<td style="text-align:left;">
4
</td>
<td style="text-align:left;">
pseudogene
</td>
<td style="text-align:right;">
143.6711
</td>
<td style="text-align:right;">
143.6736
</td>
<td style="text-align:left;">
positive
</td>
<td style="text-align:left;">
MGI_C57BL6J_3650233
</td>
<td style="text-align:left;">
Gm13082
</td>
<td style="text-align:left;">
NCBI_Gene:100040924,ENSEMBL:ENSMUSG00000081227
</td>
<td style="text-align:left;">
MGI:3650233
</td>
<td style="text-align:left;">
pseudogene
</td>
<td style="text-align:left;">
predicted gene 13082
</td>
</tr>
<tr>
<td style="text-align:left;">
4
</td>
<td style="text-align:left;">
pseudogene
</td>
<td style="text-align:right;">
143.6819
</td>
<td style="text-align:right;">
143.6853
</td>
<td style="text-align:left;">
positive
</td>
<td style="text-align:left;">
MGI_C57BL6J_5826498
</td>
<td style="text-align:left;">
Gm46861
</td>
<td style="text-align:left;">
NCBI_Gene:108168961
</td>
<td style="text-align:left;">
MGI:5826498
</td>
<td style="text-align:left;">
pseudogene
</td>
<td style="text-align:left;">
predicted gene, 46861
</td>
</tr>
<tr>
<td style="text-align:left;">
4
</td>
<td style="text-align:left;">
gene
</td>
<td style="text-align:right;">
143.6965
</td>
<td style="text-align:right;">
143.7027
</td>
<td style="text-align:left;">
negative
</td>
<td style="text-align:left;">
MGI_C57BL6J_3649972
</td>
<td style="text-align:left;">
Gm13089
</td>
<td style="text-align:left;">
NCBI_Gene:277667,ENSEMBL:ENSMUSG00000070617
</td>
<td style="text-align:left;">
MGI:3649972
</td>
<td style="text-align:left;">
protein coding gene
</td>
<td style="text-align:left;">
predicted gene 13089
</td>
</tr>
<tr>
<td style="text-align:left;">
4
</td>
<td style="text-align:left;">
gene
</td>
<td style="text-align:right;">
143.7195
</td>
<td style="text-align:right;">
143.7292
</td>
<td style="text-align:left;">
positive
</td>
<td style="text-align:left;">
MGI_C57BL6J_3650231
</td>
<td style="text-align:left;">
Gm13078
</td>
<td style="text-align:left;">
NCBI_Gene:277666,ENSEMBL:ENSMUSG00000046435
</td>
<td style="text-align:left;">
MGI:3650231
</td>
<td style="text-align:left;">
protein coding gene
</td>
<td style="text-align:left;">
predicted gene 13078
</td>
</tr>
<tr>
<td style="text-align:left;">
4
</td>
<td style="text-align:left;">
pseudogene
</td>
<td style="text-align:right;">
143.7353
</td>
<td style="text-align:right;">
143.7356
</td>
<td style="text-align:left;">
negative
</td>
<td style="text-align:left;">
MGI_C57BL6J_3649970
</td>
<td style="text-align:left;">
Gm13087
</td>
<td style="text-align:left;">
ENSEMBL:ENSMUSG00000082717
</td>
<td style="text-align:left;">
MGI:3649970
</td>
<td style="text-align:left;">
pseudogene
</td>
<td style="text-align:left;">
predicted gene 13087
</td>
</tr>
<tr>
<td style="text-align:left;">
4
</td>
<td style="text-align:left;">
pseudogene
</td>
<td style="text-align:right;">
143.7458
</td>
<td style="text-align:right;">
143.7514
</td>
<td style="text-align:left;">
positive
</td>
<td style="text-align:left;">
MGI_C57BL6J_3650234
</td>
<td style="text-align:left;">
Gm13081
</td>
<td style="text-align:left;">
ENSEMBL:ENSMUSG00000082030
</td>
<td style="text-align:left;">
MGI:3650234
</td>
<td style="text-align:left;">
pseudogene
</td>
<td style="text-align:left;">
predicted gene 13081
</td>
</tr>
</tbody>
</table>
</div>
<div id="rqtl-2" class="section level2">
<h2>R/qtl</h2>
<div id="scanone-2" class="section level3">
<h3>scanone</h3>
<pre class="r"><code>gm</code></pre>
<pre><code>Object of class cross2 (crosstype &quot;bc&quot;)

Total individuals               138
No. genotyped individuals       138
No. phenotyped individuals      138
No. with both geno &amp; pheno      138

No. phenotypes                    1
No. covariates                    7
No. phenotype covariates          0

No. chromosomes                  20
Total markers                131355

No. markers by chr:
   1    2    3    4    5    6    7    8    9   10   11   12   13   14   15   16 
9956 9987 7848 7585 7609 7736 7399 6458 6713 6385 7143 6110 6082 5966 5346 5015 
  17   18   19    X 
5080 4605 3562 4770 </code></pre>
<pre class="r"><code>#detach(&quot;package:qtl2&quot;, unload=TRUE)
#library(qtl)

cross &lt;- qtl::read.cross(&quot;csv&quot;, file = &quot;data/ici.vs.pbs_gm_qtl_5.batches_mis_conditional_2-peaks-chr4-10.csv&quot;,alleles=c(&quot;A&quot;,&quot;B&quot;))</code></pre>
<pre><code> --Read the following data:
     138  individuals
     131355  markers
     5  phenotypes
 --Cross type: bc </code></pre>
<pre class="r"><code>cross &lt;- qtl::jittermap(cross)

summary(cross)</code></pre>
<pre><code>    Backcross

    No. individuals:    138 

    No. phenotypes:     5 
    Percent phenotyped: 100 100 100 100 100 

    No. chromosomes:    20 
        Autosomes:      1 2 3 4 5 6 7 8 9 10 11 12 13 14 15 16 17 18 19 
        X chr:          X 

    Total markers:      131355 
    No. markers:        9956 9987 7848 7585 7609 7736 7399 6458 6713 6385 7143 
                        6110 6082 5966 5346 5015 5080 4605 3562 4770 
    Percent genotyped:  99.5 
    Genotypes (%):    
          Autosomes:    AA:88.1  AB:11.9 
       X chromosome:    AA:93.9  AB:6.1  </code></pre>
<pre class="r"><code>cross.probs &lt;- qtl::calc.genoprob(cross)

print(&quot;method == hk&quot;)</code></pre>
<pre><code>[1] &quot;method == hk&quot;</code></pre>
<pre class="r"><code>add.covars = qtl::pull.pheno(cross.probs, c(&quot;UNC8250659&quot;,&quot;UNC18240977&quot;))

scanone.hk &lt;-qtl::scanone(cross.probs, pheno.col=&quot;ICI.vs.PBS&quot; , model=&quot;binary&quot;, method=&quot;hk&quot;, addcovar = add.covars)
operm.hk &lt;- qtl::scanone(cross.probs, method = &quot;hk&quot;, pheno.col=&quot;ICI.vs.PBS&quot;, n.perm = 10, perm.Xsp = TRUE, model=&quot;binary&quot;, verbose=FALSE, addcovar = add.covars)
plot(operm.hk)</code></pre>
<p><img src="figure/4.1.1_qtl.analysis_binary_ici.vs.pbs_5.batches_mis_conditional_2-peaks.Rmd/scanone3-1.png" width="912" style="display: block; margin: auto;" /></p>
<pre class="r"><code>print(summary(operm.hk, alpha=c(0.01,  0.05, 0.1)))</code></pre>
<pre><code>Autosome LOD thresholds (10 permutations)
     lod
1%  3.33
5%  3.28
10% 3.21

X chromosome LOD thresholds (182 permutations)
     lod
1%  3.39
5%  3.29
10% 3.17</code></pre>
<pre class="r"><code>#plot(scanone.hk, bandcol = &quot;grey90&quot;,lty=1, cex=1, col = &quot;steelblue&quot;)  
#qtl::add.threshold(scanone.hk,  perms= operm.hk, alpha=0.01, col = &#39;blue&#39;)
#qtl::add.threshold(scanone.hk,  perms= operm.hk, alpha=0.05, col = &#39;red&#39;)
#qtl::add.threshold(scanone.hk,  perms= operm.hk, alpha=0.1, col = &#39;purple&#39;)

ymx &lt;- maxlod(out) # overall maximum LOD score
plot(scanone.hk, bandcol = &quot;grey90&quot;,lty=1, cex=1, col = &quot;slateblue&quot;, ylim=c(0, ymx+0.5))
title(main = paste0(colnames(out), &quot; [positions in cM]&quot;)) 
qtl::add.threshold(scanone.hk,  perms= operm.hk, alpha=0.01, col = &#39;blue&#39;)
qtl::add.threshold(scanone.hk,  perms= operm.hk, alpha=0.05, col = &#39;red&#39;)
qtl::add.threshold(scanone.hk,  perms= operm.hk, alpha=0.1, col = &#39;purple&#39;)</code></pre>
<p><img src="figure/4.1.1_qtl.analysis_binary_ici.vs.pbs_5.batches_mis_conditional_2-peaks.Rmd/scanone3-2.png" width="912" style="display: block; margin: auto;" /></p>
<pre class="r"><code>ymx &lt;- 14
plot(scanone.hk, bandcol = &quot;grey90&quot;,lty=1, cex=1, col = &quot;slateblue&quot;, ylim=c(0, ymx+0.5))
title(main = paste0(colnames(out), &quot; [positions in cM]\n(using same scale as ici vs. eoi for easier comparison)&quot;))
qtl::add.threshold(scanone.hk,  perms= operm.hk, alpha=0.01, col = &#39;blue&#39;)
qtl::add.threshold(scanone.hk,  perms= operm.hk, alpha=0.05, col = &#39;red&#39;)
qtl::add.threshold(scanone.hk,  perms= operm.hk, alpha=0.1, col = &#39;purple&#39;)</code></pre>
<p><img src="figure/4.1.1_qtl.analysis_binary_ici.vs.pbs_5.batches_mis_conditional_2-peaks.Rmd/scanone3-3.png" width="912" style="display: block; margin: auto;" /></p>
<pre class="r"><code>print(as.data.frame(summary(scanone.hk, perms=operm.hk, pvalues=TRUE, format=&quot;allpeaks&quot;)))</code></pre>
<pre><code>            chr       pos      lod      pval
UNC1197721    1 47.673577 1.868502 1.0000000
UNCHS005266   2 40.908610 1.873368 1.0000000
UNC6045859    3 50.620447 1.870152 1.0000000
JAX00123711   4 53.542073 1.799165 1.0000000
JAX00574407   5  6.213413 3.388862 0.0000000
UNCHS017906   6 37.001766 1.868162 1.0000000
JAX00233659   7 87.998157 1.870072 1.0000000
JAX00667121   8 24.443849 2.382541 0.6196652
UNC17203329   9 68.519175 1.323467 1.0000000
JAX00302150  10 74.547218 2.532706 0.5187088
UNC20281687  11 67.380801 1.859331 1.0000000
UNCHS034188  12 38.836978 1.870151 1.0000000
JAX00350973  13  5.743505 1.378528 1.0000000
JAX00380911  14 30.154517 2.381660 0.6196652
UNC25275535  15 11.968141 1.402063 1.0000000
UNC26790882  16 30.832641 2.019242 0.9118988
UNCHS045288  17 60.699043 2.034920 0.9118988
JAX00450853  18  2.913076 2.621475 0.5187088
UNCHS047190  19  8.712408 1.804318 1.0000000
XiB2          X 45.699951 1.870087 0.9996602</code></pre>
<pre class="r"><code>print(&quot;all peaks with a p-value less or equal to 0.05 (suggestive)&quot;)</code></pre>
<pre><code>[1] &quot;all peaks with a p-value less or equal to 0.05 (suggestive)&quot;</code></pre>
<pre class="r"><code>print(as.data.frame(summary(scanone.hk, perms=operm.hk, alpha=0.05, pvalues=TRUE, format=&quot;allpeaks&quot;)))</code></pre>
<pre><code>            chr      pos      lod pval
JAX00574407   5 6.213413 3.388862    0</code></pre>
<pre class="r"><code>#print(&quot;method == ehk&quot;)

#scanone.ehk &lt;-qtl::scanone(cross.probs, pheno.col=&quot;ICI.vs.PBS&quot; , model=&quot;binary&quot;, method=&quot;ehk&quot;)
#operm.ehk &lt;- qtl::scanone(cross.probs, method = &quot;ehk&quot;, pheno.col=&quot;ICI.vs.PBS&quot;, n.perm = 10, perm.Xsp = TRUE, model=&quot;binary&quot;, verbose=FALSE)
#plot(operm.ehk)
#print(summary(operm.ehk, alpha=c(0.01,  0.05, 0.1)))

#plot(scanone.ehk, bandcol = &quot;grey90&quot;,lty=1, cex=1, col = &quot;steelblue&quot;)  
#qtl::add.threshold(scanone.ehk,  perms= operm.ehk, alpha=0.01, col = &#39;blue&#39;)
#qtl::add.threshold(scanone.ehk,  perms= operm.ehk, alpha=0.05, col = &#39;red&#39;)
#qtl::add.threshold(scanone.ehk,  perms= operm.ehk, alpha=0.1, col = &#39;purple&#39;)

#print(as.data.frame(summary(scanone.ehk)))
#print(as.data.frame(summary(scanone.ehk, perms=operm.ehk, alpha=0.05, pvalues=TRUE, format=&quot;allpeaks&quot;)))</code></pre>
<br>
<p>
<button type="button" class="btn btn-default btn-workflowr btn-workflowr-sessioninfo" data-toggle="collapse" data-target="#workflowr-sessioninfo" style="display: block;">
<span class="glyphicon glyphicon-wrench" aria-hidden="true"></span>
Session information
</button>
</p>
<div id="workflowr-sessioninfo" class="collapse">
<pre><code>R version 3.5.1 (2018-07-02)
Platform: x86_64-apple-darwin15.6.0 (64-bit)
Running under: macOS  10.15.7

Matrix products: default
BLAS: /Library/Frameworks/R.framework/Versions/3.5/Resources/lib/libRblas.0.dylib
LAPACK: /Library/Frameworks/R.framework/Versions/3.5/Resources/lib/libRlapack.dylib

locale:
[1] en_AU.UTF-8/en_AU.UTF-8/en_AU.UTF-8/C/en_AU.UTF-8/en_AU.UTF-8

attached base packages:
[1] stats     graphics  grDevices utils     datasets  methods   base     

other attached packages:
 [1] abind_1.4-5       qtl2_0.22         reshape2_1.4.3    ggplot2_3.3.6    
 [5] tibble_3.1.7      psych_2.2.5       readxl_1.4.0      cluster_2.0.7-1  
 [9] dplyr_1.0.9       optparse_1.6.6    rhdf5_2.26.2      mclust_5.4.5     
[13] tidyr_1.2.0       data.table_1.14.2 knitr_1.29        kableExtra_1.3.4 
[17] workflowr_1.6.2  

loaded via a namespace (and not attached):
 [1] httr_1.4.3        bit64_0.9-7       viridisLite_0.3.0 assertthat_0.2.1 
 [5] highr_0.8         blob_1.2.1        cellranger_1.1.0  yaml_2.2.1       
 [9] gdtools_0.2.1     pillar_1.7.0      RSQLite_2.2.0     backports_1.1.5  
[13] lattice_0.20-35   glue_1.6.2        digest_0.6.25     promises_1.1.0   
[17] rvest_1.0.2       colorspace_1.4-1  htmltools_0.5.2   httpuv_1.5.2     
[21] plyr_1.8.6        pkgconfig_2.0.3   purrr_0.3.4       scales_1.1.1     
[25] webshot_0.5.3     svglite_1.2.3     qtl_1.46-2        getopt_1.20.3    
[29] later_1.0.0       git2r_0.26.1      generics_0.0.2    ellipsis_0.3.2   
[33] withr_2.5.0       cli_3.3.0         mnormt_1.5-6      magrittr_2.0.3   
[37] crayon_1.5.1      memoise_1.1.0     evaluate_0.14     fs_1.3.2         
[41] fansi_0.4.1       nlme_3.1-137      xml2_1.3.2        tools_3.5.1      
[45] lifecycle_1.0.1   stringr_1.4.0     Rhdf5lib_1.4.3    munsell_0.5.0    
[49] compiler_3.5.1    systemfonts_0.1.1 rlang_1.0.3       grid_3.5.1       
[53] rstudioapi_0.13   rmarkdown_2.3     gtable_0.3.0      DBI_1.1.0        
[57] R6_2.4.1          fastmap_1.1.0     bit_1.1-15.2      utf8_1.1.4       
[61] rprojroot_1.3-2   stringi_1.4.6     parallel_3.5.1    Rcpp_1.0.4.6     
[65] vctrs_0.4.1       tidyselect_1.1.2  xfun_0.15        </code></pre>
</div>
</div>
</div>
</div>


<!-- Adjust MathJax settings so that all math formulae are shown using
TeX fonts only; see
http://docs.mathjax.org/en/latest/configuration.html.  This will make
the presentation more consistent at the cost of the webpage sometimes
taking slightly longer to load. Note that this only works because the
footer is added to webpages before the MathJax javascript. -->
<script type="text/x-mathjax-config">
  MathJax.Hub.Config({
    "HTML-CSS": { availableFonts: ["TeX"] }
  });
</script>





</div>

<script>

// add bootstrap table styles to pandoc tables
function bootstrapStylePandocTables() {
  $('tr.header').parent('thead').parent('table').addClass('table table-condensed');
}
$(document).ready(function () {
  bootstrapStylePandocTables();
});


</script>

<!-- tabsets -->

<script>
$(document).ready(function () {
  window.buildTabsets("TOC");
});

$(document).ready(function () {
  $('.tabset-dropdown > .nav-tabs > li').click(function () {
    $(this).parent().toggleClass('nav-tabs-open')
  });
});
</script>

<!-- code folding -->


<!-- dynamically load mathjax for compatibility with self-contained -->
<script>
  (function () {
    var script = document.createElement("script");
    script.type = "text/javascript";
    script.src  = "https://mathjax.rstudio.com/latest/MathJax.js?config=TeX-AMS-MML_HTMLorMML";
    document.getElementsByTagName("head")[0].appendChild(script);
  })();
</script>

</body>
</html>
