<!DOCTYPE html>

<html>

<head>

<meta charset="utf-8" />
<meta name="generator" content="pandoc" />
<meta http-equiv="X-UA-Compatible" content="IE=EDGE" />


<meta name="author" content="Belinda Cornes" />

<meta name="date" content="2022-04-14" />

<title>QTL Analysis - age of onset [ICI vs EOI] (corrected phenotype with outliers removed if any)</title>

<script src="site_libs/header-attrs-2.1/header-attrs.js"></script>
<script src="site_libs/jquery-1.11.3/jquery.min.js"></script>
<meta name="viewport" content="width=device-width, initial-scale=1" />
<link href="site_libs/bootstrap-3.3.5/css/spacelab.min.css" rel="stylesheet" />
<script src="site_libs/bootstrap-3.3.5/js/bootstrap.min.js"></script>
<script src="site_libs/bootstrap-3.3.5/shim/html5shiv.min.js"></script>
<script src="site_libs/bootstrap-3.3.5/shim/respond.min.js"></script>
<script src="site_libs/navigation-1.1/tabsets.js"></script>
<link href="site_libs/highlightjs-9.12.0/textmate.css" rel="stylesheet" />
<script src="site_libs/highlightjs-9.12.0/highlight.js"></script>
<script src="site_libs/kePrint-0.0.1/kePrint.js"></script>
<link href="site_libs/font-awesome-5.1.0/css/all.css" rel="stylesheet" />
<link href="site_libs/font-awesome-5.1.0/css/v4-shims.css" rel="stylesheet" />

<link rel="icon" href="https://github.com/workflowr/workflowr-assets/raw/master/img/reproducible.png">
<!-- Add a small amount of space between sections. -->
<style type="text/css">
div.section {
  padding-top: 12px;
}
</style>



<style type="text/css">code{white-space: pre;}</style>
<style type="text/css">
  pre:not([class]) {
    background-color: white;
  }
</style>
<script type="text/javascript">
if (window.hljs) {
  hljs.configure({languages: []});
  hljs.initHighlightingOnLoad();
  if (document.readyState && document.readyState === "complete") {
    window.setTimeout(function() { hljs.initHighlighting(); }, 0);
  }
}
</script>



<style type="text/css">
h1 {
  font-size: 34px;
}
h1.title {
  font-size: 38px;
}
h2 {
  font-size: 30px;
}
h3 {
  font-size: 24px;
}
h4 {
  font-size: 18px;
}
h5 {
  font-size: 16px;
}
h6 {
  font-size: 12px;
}
.table th:not([align]) {
  text-align: left;
}
</style>




<style type = "text/css">
.main-container {
  max-width: 940px;
  margin-left: auto;
  margin-right: auto;
}
code {
  color: inherit;
  background-color: rgba(0, 0, 0, 0.04);
}
img {
  max-width:100%;
}
.tabbed-pane {
  padding-top: 12px;
}
.html-widget {
  margin-bottom: 20px;
}
button.code-folding-btn:focus {
  outline: none;
}
summary {
  display: list-item;
}
</style>


<style type="text/css">
/* padding for bootstrap navbar */
body {
  padding-top: 51px;
  padding-bottom: 40px;
}
/* offset scroll position for anchor links (for fixed navbar)  */
.section h1 {
  padding-top: 56px;
  margin-top: -56px;
}
.section h2 {
  padding-top: 56px;
  margin-top: -56px;
}
.section h3 {
  padding-top: 56px;
  margin-top: -56px;
}
.section h4 {
  padding-top: 56px;
  margin-top: -56px;
}
.section h5 {
  padding-top: 56px;
  margin-top: -56px;
}
.section h6 {
  padding-top: 56px;
  margin-top: -56px;
}
.dropdown-submenu {
  position: relative;
}
.dropdown-submenu>.dropdown-menu {
  top: 0;
  left: 100%;
  margin-top: -6px;
  margin-left: -1px;
  border-radius: 0 6px 6px 6px;
}
.dropdown-submenu:hover>.dropdown-menu {
  display: block;
}
.dropdown-submenu>a:after {
  display: block;
  content: " ";
  float: right;
  width: 0;
  height: 0;
  border-color: transparent;
  border-style: solid;
  border-width: 5px 0 5px 5px;
  border-left-color: #cccccc;
  margin-top: 5px;
  margin-right: -10px;
}
.dropdown-submenu:hover>a:after {
  border-left-color: #ffffff;
}
.dropdown-submenu.pull-left {
  float: none;
}
.dropdown-submenu.pull-left>.dropdown-menu {
  left: -100%;
  margin-left: 10px;
  border-radius: 6px 0 6px 6px;
}
</style>

<script>
// manage active state of menu based on current page
$(document).ready(function () {
  // active menu anchor
  href = window.location.pathname
  href = href.substr(href.lastIndexOf('/') + 1)
  if (href === "")
    href = "index.html";
  var menuAnchor = $('a[href="' + href + '"]');

  // mark it active
  menuAnchor.parent().addClass('active');

  // if it's got a parent navbar menu mark it active as well
  menuAnchor.closest('li.dropdown').addClass('active');
});
</script>

<!-- tabsets -->

<style type="text/css">
.tabset-dropdown > .nav-tabs {
  display: inline-table;
  max-height: 500px;
  min-height: 44px;
  overflow-y: auto;
  background: white;
  border: 1px solid #ddd;
  border-radius: 4px;
}

.tabset-dropdown > .nav-tabs > li.active:before {
  content: "";
  font-family: 'Glyphicons Halflings';
  display: inline-block;
  padding: 10px;
  border-right: 1px solid #ddd;
}

.tabset-dropdown > .nav-tabs.nav-tabs-open > li.active:before {
  content: "&#xe258;";
  border: none;
}

.tabset-dropdown > .nav-tabs.nav-tabs-open:before {
  content: "";
  font-family: 'Glyphicons Halflings';
  display: inline-block;
  padding: 10px;
  border-right: 1px solid #ddd;
}

.tabset-dropdown > .nav-tabs > li.active {
  display: block;
}

.tabset-dropdown > .nav-tabs > li > a,
.tabset-dropdown > .nav-tabs > li > a:focus,
.tabset-dropdown > .nav-tabs > li > a:hover {
  border: none;
  display: inline-block;
  border-radius: 4px;
  background-color: transparent;
}

.tabset-dropdown > .nav-tabs.nav-tabs-open > li {
  display: block;
  float: none;
}

.tabset-dropdown > .nav-tabs > li {
  display: none;
}
</style>

<!-- code folding -->




</head>

<body>


<div class="container-fluid main-container">




<div class="navbar navbar-default  navbar-fixed-top" role="navigation">
  <div class="container">
    <div class="navbar-header">
      <button type="button" class="navbar-toggle collapsed" data-toggle="collapse" data-target="#navbar">
        <span class="icon-bar"></span>
        <span class="icon-bar"></span>
        <span class="icon-bar"></span>
      </button>
      <a class="navbar-brand" href="index.html">Serreze T1D Workflow</a>
    </div>
    <div id="navbar" class="navbar-collapse collapse">
      <ul class="nav navbar-nav">
        <li>
  <a href="index.html">Home</a>
</li>
<li>
  <a href="about.html">About</a>
</li>
<li>
  <a href="license.html">License</a>
</li>
      </ul>
      <ul class="nav navbar-nav navbar-right">
        <li>
  <a href="https://github.com/TheJacksonLaboratory/Serreze-T1D_Workflow">
    <span class="fa fa-github"></span>
     
    Source code
  </a>
</li>
      </ul>
    </div><!--/.nav-collapse -->
  </div><!--/.container -->
</div><!--/.navbar -->

<div class="fluid-row" id="header">



<h1 class="title toc-ignore">QTL Analysis - age of onset [ICI vs EOI] (corrected phenotype with outliers removed if any)</h1>
<h4 class="author">Belinda Cornes</h4>
<h4 class="date">2022-04-14</h4>

</div>

<div id="TOC">
<ul>
<li><a href="#loading-data">Loading Data</a></li>
<li><a href="#preparing-necessary-info">Preparing necessary info</a>
<ul>
<li><a href="#no-additive-covariates">No additive covariates</a>
<ul>
<li><a href="#significance-thresholds">Significance thresholds</a></li>
<li><a href="#genome-wide-scan">Genome-wide scan</a></li>
<li><a href="#lod-peaks">LOD peaks</a></li>
<li><a href="#qtl-effects">QTL effects</a></li>
</ul></li>
<li><a href="#additive-covariates">Additive covariates</a>
<ul>
<li><a href="#significance-thresholds-1">Significance thresholds</a></li>
<li><a href="#genome-wide-scan-1">Genome-wide scan</a></li>
<li><a href="#lod-peaks-1">LOD peaks</a></li>
<li><a href="#qtl-effects-1">QTL effects</a></li>
</ul></li>
<li><a href="#interactive-term">Interactive term</a>
<ul>
<li><a href="#significance-thresholds-2">Significance thresholds</a></li>
<li><a href="#genome-wide-scan-2">Genome-wide scan</a></li>
<li><a href="#lod-peaks-2">LOD peaks</a></li>
<li><a href="#qtl-effects-2">QTL effects</a></li>
</ul></li>
</ul></li>
</ul>
</div>

<p>
<button type="button" class="btn btn-default btn-workflowr btn-workflowr-report" data-toggle="collapse" data-target="#workflowr-report">
<span class="glyphicon glyphicon-list" aria-hidden="true"></span> workflowr <span class="glyphicon glyphicon-exclamation-sign text-danger" aria-hidden="true"></span>
</button>
</p>
<div id="workflowr-report" class="collapse">
<ul class="nav nav-tabs">
<li class="active">
<a data-toggle="tab" href="#summary">Summary</a>
</li>
<li>
<a data-toggle="tab" href="#checks"> Checks <span class="glyphicon glyphicon-exclamation-sign text-danger" aria-hidden="true"></span> </a>
</li>
<li>
<a data-toggle="tab" href="#versions">Past versions</a>
</li>
</ul>
<div class="tab-content">
<div id="summary" class="tab-pane fade in active">
<p>
<strong>Last updated:</strong> 2022-04-14
</p>
<p>
<strong>Checks:</strong> <span class="glyphicon glyphicon-ok text-success" aria-hidden="true"></span> 5 <span class="glyphicon glyphicon-exclamation-sign text-danger" aria-hidden="true"></span> 2
</p>
<p>
<strong>Knit directory:</strong> <code>Serreze-T1D_Workflow/</code> <span class="glyphicon glyphicon-question-sign" aria-hidden="true" title="This is the local directory in which the code in this file was executed."> </span>
</p>
<p>
This reproducible <a href="http://rmarkdown.rstudio.com">R Markdown</a> analysis was created with <a
  href="https://github.com/jdblischak/workflowr">workflowr</a> (version 1.6.2). The <em>Checks</em> tab describes the reproducibility checks that were applied when the results were created. The <em>Past versions</em> tab lists the development history.
</p>
<hr>
</div>
<div id="checks" class="tab-pane fade">
<div id="workflowr-checks" class="panel-group">
<div class="panel panel-default">
<div class="panel-heading">
<p class="panel-title">
<a data-toggle="collapse" data-parent="#workflowr-checks" href="#strongRMarkdownfilestronguncommittedchanges"> <span class="glyphicon glyphicon-exclamation-sign text-danger" aria-hidden="true"></span> <strong>R Markdown file:</strong> uncommitted changes </a>
</p>
</div>
<div id="strongRMarkdownfilestronguncommittedchanges" class="panel-collapse collapse">
<div class="panel-body">
<p>The R Markdown is untracked by Git. To know which version of the R Markdown file created these results, you’ll want to first commit it to the Git repo. If you’re still working on the analysis, you can ignore this warning. When you’re finished, you can run <code>wflow_publish</code> to commit the R Markdown file and build the HTML.</p>
</div>
</div>
</div>
<div class="panel panel-default">
<div class="panel-heading">
<p class="panel-title">
<a data-toggle="collapse" data-parent="#workflowr-checks" href="#strongEnvironmentstrongempty"> <span class="glyphicon glyphicon-ok text-success" aria-hidden="true"></span> <strong>Environment:</strong> empty </a>
</p>
</div>
<div id="strongEnvironmentstrongempty" class="panel-collapse collapse">
<div class="panel-body">
<p>Great job! The global environment was empty. Objects defined in the global environment can affect the analysis in your R Markdown file in unknown ways. For reproduciblity it’s best to always run the code in an empty environment.</p>
</div>
</div>
</div>
<div class="panel panel-default">
<div class="panel-heading">
<p class="panel-title">
<a data-toggle="collapse" data-parent="#workflowr-checks" href="#strongSeedstrongcodesetseed20220210code"> <span class="glyphicon glyphicon-ok text-success" aria-hidden="true"></span> <strong>Seed:</strong> <code>set.seed(20220210)</code> </a>
</p>
</div>
<div id="strongSeedstrongcodesetseed20220210code" class="panel-collapse collapse">
<div class="panel-body">
<p>The command <code>set.seed(20220210)</code> was run prior to running the code in the R Markdown file. Setting a seed ensures that any results that rely on randomness, e.g. subsampling or permutations, are reproducible.</p>
</div>
</div>
</div>
<div class="panel panel-default">
<div class="panel-heading">
<p class="panel-title">
<a data-toggle="collapse" data-parent="#workflowr-checks" href="#strongSessioninformationstrongrecorded"> <span class="glyphicon glyphicon-ok text-success" aria-hidden="true"></span> <strong>Session information:</strong> recorded </a>
</p>
</div>
<div id="strongSessioninformationstrongrecorded" class="panel-collapse collapse">
<div class="panel-body">
<p>Great job! Recording the operating system, R version, and package versions is critical for reproducibility.</p>
</div>
</div>
</div>
<div class="panel panel-default">
<div class="panel-heading">
<p class="panel-title">
<a data-toggle="collapse" data-parent="#workflowr-checks" href="#strongCachestrongnone"> <span class="glyphicon glyphicon-ok text-success" aria-hidden="true"></span> <strong>Cache:</strong> none </a>
</p>
</div>
<div id="strongCachestrongnone" class="panel-collapse collapse">
<div class="panel-body">
<p>Nice! There were no cached chunks for this analysis, so you can be confident that you successfully produced the results during this run.</p>
</div>
</div>
</div>
<div class="panel panel-default">
<div class="panel-heading">
<p class="panel-title">
<a data-toggle="collapse" data-parent="#workflowr-checks" href="#strongFilepathsstrongabsolute"> <span class="glyphicon glyphicon-exclamation-sign text-danger" aria-hidden="true"></span> <strong>File paths:</strong> absolute </a>
</p>
</div>
<div id="strongFilepathsstrongabsolute" class="panel-collapse collapse">
<div class="panel-body">
<p>
Using absolute paths to the files within your workflowr project makes it difficult for you and others to run your code on a different machine. Change the absolute path(s) below to the suggested relative path(s) to make your code more reproducible.
</p>
<table class="table table-condensed table-hover">
<thead>
<tr>
<th style="text-align:left;">
absolute
</th>
<th style="text-align:left;">
relative
</th>
</tr>
</thead>
<tbody>
<tr>
<td style="text-align:left;">
/Users/corneb/Documents/MyJax/CS/Projects/Serreze/qc/workflowr/Serreze-T1D_Workflow
</td>
<td style="text-align:left;">
.
</td>
</tr>
</tbody>
</table>
</div>
</div>
</div>
<div class="panel panel-default">
<div class="panel-heading">
<p class="panel-title">
<a data-toggle="collapse" data-parent="#workflowr-checks" href="#strongRepositoryversionstrongahrefhttpsgithubcomTheJacksonLaboratorySerrezeT1DWorkflowtreea8c2d1a11f223361b54952ce8455e8c17017c914targetblanka8c2d1aa"> <span class="glyphicon glyphicon-ok text-success" aria-hidden="true"></span> <strong>Repository version:</strong> <a href="https://github.com/TheJacksonLaboratory/Serreze-T1D_Workflow/tree/a8c2d1a11f223361b54952ce8455e8c17017c914" target="_blank">a8c2d1a</a> </a>
</p>
</div>
<div id="strongRepositoryversionstrongahrefhttpsgithubcomTheJacksonLaboratorySerrezeT1DWorkflowtreea8c2d1a11f223361b54952ce8455e8c17017c914targetblanka8c2d1aa" class="panel-collapse collapse">
<div class="panel-body">
<p>
Great! You are using Git for version control. Tracking code development and connecting the code version to the results is critical for reproducibility.
</p>
<p>
The results in this page were generated with repository version <a href="https://github.com/TheJacksonLaboratory/Serreze-T1D_Workflow/tree/a8c2d1a11f223361b54952ce8455e8c17017c914" target="_blank">a8c2d1a</a>. See the <em>Past versions</em> tab to see a history of the changes made to the R Markdown and HTML files.
</p>
<p>
Note that you need to be careful to ensure that all relevant files for the analysis have been committed to Git prior to generating the results (you can use <code>wflow_publish</code> or <code>wflow_git_commit</code>). workflowr only checks the R Markdown file, but you know if there are other scripts or data files that it depends on. Below is the status of the Git repository when the results were generated:
</p>
<pre><code>
Ignored files:
    Ignored:    .DS_Store
    Ignored:    analysis/.DS_Store
    Ignored:    data/.DS_Store

Untracked files:
    Untracked:  analysis/3.1_phenotype.qc_corrected_5.batches_0.Rmd
    Untracked:  analysis/3.1_phenotype.qc_corrected_5.batches_52.Rmd
    Untracked:  analysis/3.1_phenotype.qc_corrected_5.batches_mis_0.Rmd
    Untracked:  analysis/3.1_phenotype.qc_corrected_5.batches_mis_52.Rmd
    Untracked:  analysis/3.1_phenotype.qc_corrected_5.batches_mis_vo.Rmd
    Untracked:  analysis/3.1_phenotype.qc_corrected_5.batches_vo.Rmd
    Untracked:  analysis/4.1.1_qtl.analysis_binary_ici.vs.eoi_snpsqc_dis_no-x_updated_5.batches_11.Rmd
    Untracked:  analysis/4.1.1_qtl.analysis_binary_ici.vs.eoi_snpsqc_dis_no-x_updated_5.batches_mis_11.Rmd
    Untracked:  analysis/4.1.1_qtl.analysis_binary_ici.vs.pbs_snpsqc_5.batches_1.Rmd
    Untracked:  analysis/4.1.1_qtl.analysis_binary_ici.vs.pbs_snpsqc_5.batches_mis_1.Rmd
    Untracked:  analysis/4.1.1_qtl.analysis_binary_ici.vs.pbs_snpsqc_dis_no-x_updated_5.batches_1.Rmd
    Untracked:  analysis/4.1.1_qtl.analysis_binary_ici.vs.pbs_snpsqc_dis_no-x_updated_5.batches_11.Rmd
    Untracked:  analysis/4.1.1_qtl.analysis_binary_ici.vs.pbs_snpsqc_dis_no-x_updated_5.batches_mis_11.Rmd
    Untracked:  analysis/4.1.2_qtl.analysis_cont_age_ici.vs.eoi_pheno.corrected.cleaned_5.batches_0.Rmd
    Untracked:  analysis/4.1.2_qtl.analysis_cont_age_ici.vs.eoi_pheno.corrected.cleaned_5.batches_52.Rmd
    Untracked:  analysis/4.1.2_qtl.analysis_cont_age_ici.vs.eoi_pheno.corrected.cleaned_5.batches_changed.Rmd
    Untracked:  analysis/4.1.2_qtl.analysis_cont_age_ici.vs.eoi_pheno.corrected.cleaned_5.batches_mis_0.Rmd
    Untracked:  analysis/4.1.2_qtl.analysis_cont_age_ici.vs.eoi_pheno.corrected.cleaned_5.batches_mis_52.Rmd
    Untracked:  analysis/4.1.2_qtl.analysis_cont_age_ici.vs.eoi_pheno.corrected.cleaned_5.batches_mis_changed.Rmd
    Untracked:  analysis/4.1.2_qtl.analysis_cont_age_ici.vs.eoi_pheno.corrected.cleaned_scanone_5.batches.Rmd
    Untracked:  analysis/4.1.2_qtl.analysis_cont_age_ici.vs.eoi_pheno.corrected.cleaned_scanone_5.batches_mis.Rmd
    Untracked:  analysis/4.1.2_qtl.analysis_cont_age_ici.vs.eoi_snpsqc_pheno.corrected.cleaned_5.batches_0.Rmd
    Untracked:  analysis/4.1.2_qtl.analysis_cont_age_ici.vs.eoi_snpsqc_pheno.corrected.cleaned_5.batches_52.Rmd
    Untracked:  analysis/4.1.2_qtl.analysis_cont_age_ici.vs.eoi_snpsqc_pheno.corrected.cleaned_5.batches_mis_0.Rmd
    Untracked:  analysis/4.1.2_qtl.analysis_cont_age_ici.vs.eoi_snpsqc_pheno.corrected.cleaned_5.batches_mis_52.Rmd
    Untracked:  analysis/4.1.2_qtl.analysis_cont_age_ici.vs.eoi_snpsqc_pheno.corrected.cleaned_5.batches_oops.Rmd
    Untracked:  analysis/4.1.2_qtl.analysis_cont_age_ici.vs.eoi_snpsqc_pheno.corrected.cleaned_dis.Rmd.R
    Untracked:  analysis/4.1.2_qtl.analysis_cont_age_ici.vs.eoi_snpsqc_pheno.corrected.cleaned_dis_no-xk_5.batches_0.Rmd
    Untracked:  analysis/4.1.2_qtl.analysis_cont_age_ici.vs.eoi_snpsqc_pheno.corrected.cleaned_dis_no-xk_5.batches_52.Rmd
    Untracked:  analysis/4.1.2_qtl.analysis_cont_age_ici.vs.eoi_snpsqc_pheno.corrected.cleaned_dis_no-xk_5.batches_mis_0.Rmd
    Untracked:  analysis/4.1.2_qtl.analysis_cont_age_ici.vs.eoi_snpsqc_pheno.corrected.cleaned_dis_no-xk_5.batches_mis_52.Rmd
    Untracked:  analysis/4.1.2_qtl.analysis_cont_age_ici.vs.eoi_snpsqc_pheno.corrected.cleaned_test.with.4.Rmd
    Untracked:  analysis/4.1.2_qtl.analysis_cont_age_ici.vs.eoi_snpsqc_pheno.corrected.cleaned_test.with.4.Rmd.R
    Untracked:  analysis/4.1.2_qtl.analysis_cont_age_ici.vs.eoi_snpsqc_pheno.corrected.cleaned_test.with.4_miss.Rmd
    Untracked:  analysis/4.1.2_qtl.analysis_cont_age_ici.vs.eoi_snpsqc_pheno.corrected.cleaned_test.with.4_miss.Rmd.R
    Untracked:  analysis/4.1.2_qtl.analysis_cont_age_ici.vs.eoi_snpsqc_pheno.corrected.cleaned_test.with.5.Rmd
    Untracked:  analysis/4.1.2_qtl.analysis_cont_age_ici.vs.eoi_snpsqc_pheno.corrected.cleaned_test.with.5.Rmd.R
    Untracked:  analysis/4.1.2_qtl.analysis_cont_age_ici.vs.eoi_snpsqc_pheno.corrected.cleaned_test.with.5_0.Rmd
    Untracked:  analysis/4.1.2_qtl.analysis_cont_age_ici.vs.eoi_snpsqc_pheno.corrected.cleaned_test.with.5_0.Rmd.R
    Untracked:  analysis/4.1.2_qtl.analysis_cont_rz.age_ici.vs.eoi_pheno.corrected.cleaned_5.batches_0.Rmd
    Untracked:  analysis/4.1.2_qtl.analysis_cont_rz.age_ici.vs.eoi_pheno.corrected.cleaned_5.batches_52.Rmd
    Untracked:  analysis/4.1.2_qtl.analysis_cont_rz.age_ici.vs.eoi_pheno.corrected.cleaned_5.batches_mis_0.Rmd
    Untracked:  analysis/4.1.2_qtl.analysis_cont_rz.age_ici.vs.eoi_pheno.corrected.cleaned_5.batches_mis_52.Rmd
    Untracked:  analysis/4.1.2_qtl.analysis_cont_rz.age_ici.vs.eoi_pheno.corrected.cleaned_5.batches_mis_vo.Rmd
    Untracked:  analysis/4.1.2_qtl.analysis_cont_rz.age_ici.vs.eoi_pheno.corrected.cleaned_5.batches_mis_vo1.Rmd
    Untracked:  analysis/4.1.2_qtl.analysis_cont_rz.age_ici.vs.eoi_pheno.corrected.cleaned_5.batches_vo.Rmd
    Untracked:  analysis/4.1.2_qtl.analysis_cont_rz.age_ici.vs.eoi_pheno.corrected.cleaned_5.batches_vo1.Rmd
    Untracked:  analysis/4.1.2_qtl.analysis_cont_rz.age_ici.vs.eoi_snpsqc_pheno.corrected.cleaned_5.batches_0.Rmd
    Untracked:  analysis/4.1.2_qtl.analysis_cont_rz.age_ici.vs.eoi_snpsqc_pheno.corrected.cleaned_5.batches_52.Rmd
    Untracked:  analysis/4.1.2_qtl.analysis_cont_rz.age_ici.vs.eoi_snpsqc_pheno.corrected.cleaned_5.batches_mis_0.Rmd
    Untracked:  analysis/4.1.2_qtl.analysis_cont_rz.age_ici.vs.eoi_snpsqc_pheno.corrected.cleaned_5.batches_mis_52.Rmd
    Untracked:  analysis/4.1.2_qtl.analysis_cont_rz.age_ici.vs.eoi_snpsqc_pheno.corrected.cleaned_dis_no-xk_5.batches_0.Rmd
    Untracked:  analysis/4.1.2_qtl.analysis_cont_rz.age_ici.vs.eoi_snpsqc_pheno.corrected.cleaned_dis_no-xk_5.batches_52.Rmd
    Untracked:  analysis/4.1.2_qtl.analysis_cont_rz.age_ici.vs.eoi_snpsqc_pheno.corrected.cleaned_dis_no-xk_5.batches_mis_0.Rmd
    Untracked:  analysis/4.1.2_qtl.analysis_cont_rz.age_ici.vs.eoi_snpsqc_pheno.corrected.cleaned_dis_no-xk_5.batches_mis_52.Rmd
    Untracked:  analysis/genotype.frequencies_ici.vs.eoi_5.batches_0.Rmd
    Untracked:  analysis/genotype.frequencies_ici.vs.eoi_5.batches_52.Rmd
    Untracked:  analysis/genotype.frequencies_ici.vs.eoi_5.batches_mis_0.Rmd
    Untracked:  analysis/genotype.frequencies_ici.vs.eoi_5.batches_mis_52.Rmd
    Untracked:  analysis/index_5.batches_additional_vo.Rmd
    Untracked:  data/GM_covar.csv
    Untracked:  data/GM_covar_BC312.csv
    Untracked:  data/bad_markers_all_4.batches.RData
    Untracked:  data/bad_markers_all_5.batches.RData
    Untracked:  data/blup_sub_chr10_lod.drop-1.5_5.batches.csv
    Untracked:  data/blup_sub_chr3_lod.drop-1.5.csv
    Untracked:  data/blup_sub_chr3_lod.drop-1.5_5.batches.csv
    Untracked:  data/blup_sub_chr4_lod.drop-1.5.csv
    Untracked:  data/blup_sub_chr4_lod.drop-1.5_5.batches.csv
    Untracked:  data/covar_cleaned_ici.vs.eoi.csv
    Untracked:  data/covar_cleaned_ici.vs.pbs.csv
    Untracked:  data/covar_corrected.cleaned_ici-early.vs.pbs_5.batches.csv
    Untracked:  data/covar_corrected.cleaned_ici-early.vs.pbs_5.batches_0.csv
    Untracked:  data/covar_corrected.cleaned_ici-early.vs.pbs_5.batches_52.csv
    Untracked:  data/covar_corrected.cleaned_ici-early.vs.pbs_5.batches_mis.csv
    Untracked:  data/covar_corrected.cleaned_ici-early.vs.pbs_5.batches_mis_0.csv
    Untracked:  data/covar_corrected.cleaned_ici-early.vs.pbs_5.batches_mis_52.csv
    Untracked:  data/covar_corrected.cleaned_ici.vs.eoi.csv
    Untracked:  data/covar_corrected.cleaned_ici.vs.eoi1.csv
    Untracked:  data/covar_corrected.cleaned_ici.vs.eoi_5.batches.csv
    Untracked:  data/covar_corrected.cleaned_ici.vs.eoi_5.batches_0.csv
    Untracked:  data/covar_corrected.cleaned_ici.vs.eoi_5.batches_52.csv
    Untracked:  data/covar_corrected.cleaned_ici.vs.eoi_5.batches_mis.csv
    Untracked:  data/covar_corrected.cleaned_ici.vs.eoi_5.batches_mis_0.csv
    Untracked:  data/covar_corrected.cleaned_ici.vs.eoi_5.batches_mis_52.csv
    Untracked:  data/covar_corrected.cleaned_ici.vs.pbs.csv
    Untracked:  data/covar_corrected.cleaned_ici.vs.pbs1.csv
    Untracked:  data/covar_corrected.cleaned_ici.vs.pbs_5.batches.csv
    Untracked:  data/covar_corrected.cleaned_ici.vs.pbs_5.batches_0.csv
    Untracked:  data/covar_corrected.cleaned_ici.vs.pbs_5.batches_52.csv
    Untracked:  data/covar_corrected.cleaned_ici.vs.pbs_5.batches_mis.csv
    Untracked:  data/covar_corrected.cleaned_ici.vs.pbs_5.batches_mis_0.csv
    Untracked:  data/covar_corrected.cleaned_ici.vs.pbs_5.batches_mis_52.csv
    Untracked:  data/covar_corrected_ici-early.vs.pbs_5.batches.csv
    Untracked:  data/covar_corrected_ici-early.vs.pbs_5.batches_0.csv
    Untracked:  data/covar_corrected_ici-early.vs.pbs_5.batches_52.csv
    Untracked:  data/covar_corrected_ici-early.vs.pbs_5.batches_mis.csv
    Untracked:  data/covar_corrected_ici-early.vs.pbs_5.batches_mis_0.csv
    Untracked:  data/covar_corrected_ici-early.vs.pbs_5.batches_mis_52.csv
    Untracked:  data/covar_corrected_ici.vs.eoi.csv
    Untracked:  data/covar_corrected_ici.vs.eoi1.csv
    Untracked:  data/covar_corrected_ici.vs.eoi_5.batches.csv
    Untracked:  data/covar_corrected_ici.vs.eoi_5.batches_0.csv
    Untracked:  data/covar_corrected_ici.vs.eoi_5.batches_52.csv
    Untracked:  data/covar_corrected_ici.vs.eoi_5.batches_mis.csv
    Untracked:  data/covar_corrected_ici.vs.eoi_5.batches_mis_0.csv
    Untracked:  data/covar_corrected_ici.vs.eoi_5.batches_mis_52.csv
    Untracked:  data/covar_corrected_ici.vs.pbs.csv
    Untracked:  data/covar_corrected_ici.vs.pbs1.csv
    Untracked:  data/covar_corrected_ici.vs.pbs_5.batches.csv
    Untracked:  data/covar_corrected_ici.vs.pbs_5.batches_0.csv
    Untracked:  data/covar_corrected_ici.vs.pbs_5.batches_52.csv
    Untracked:  data/covar_corrected_ici.vs.pbs_5.batches_mis.csv
    Untracked:  data/covar_corrected_ici.vs.pbs_5.batches_mis_0.csv
    Untracked:  data/covar_corrected_ici.vs.pbs_5.batches_mis_52.csv
    Untracked:  data/e.RData
    Untracked:  data/e_BC312.RData
    Untracked:  data/e_snpg_samqc_4.batches.RData
    Untracked:  data/e_snpg_samqc_4.batches_bc.RData
    Untracked:  data/e_snpg_samqc_5.batches.RData
    Untracked:  data/errors_ind_4.batches.RData
    Untracked:  data/errors_ind_4.batches_bc.RData
    Untracked:  data/errors_ind_5.batches.RData
    Untracked:  data/files.to.sync.txt
    Untracked:  data/fitqtl_chr3.peak_chr4.peak_additive.txt
    Untracked:  data/fitqtl_chr3.peak_chr4.peak_interacting.txt
    Untracked:  data/fitqtl_chr3.peak_chr4.peak_sex_additive.txt
    Untracked:  data/fitqtl_chr3.peak_chr4.peak_sex_interacting.txt
    Untracked:  data/g2blup_effects.csv
    Untracked:  data/g2blup_effects.xlsx
    Untracked:  data/genes_chr10_lod.drop-1.5_5.batches.csv
    Untracked:  data/genes_chr3_lod.drop-1.5.csv
    Untracked:  data/genes_chr3_lod.drop-1.5_5.batches.csv
    Untracked:  data/genes_chr4_lod.drop-1.5.csv
    Untracked:  data/genetic_map.csv
    Untracked:  data/genetic_map_BC312.csv
    Untracked:  data/genotype_errors_marker_4.batches.RData
    Untracked:  data/genotype_errors_marker_5.batches.RData
    Untracked:  data/genotype_freq_marker_4.batches.RData
    Untracked:  data/genotype_freq_marker_5.batches.RData
    Untracked:  data/gm_allqc_4.batches.RData
    Untracked:  data/gm_allqc_5.batches.RData
    Untracked:  data/gm_allqc_5.batches_mis.RData
    Untracked:  data/gm_samqc_3.batches.RData
    Untracked:  data/gm_samqc_4.batches.RData
    Untracked:  data/gm_samqc_4.batches_bc.RData
    Untracked:  data/gm_samqc_5.batches.RData
    Untracked:  data/gm_serreze.192.RData
    Untracked:  data/gm_serreze.BC312.RData
    Untracked:  data/ici-early.vs.pbs_age.of.onset-additive.covariates_blup_sub_chr-7_peak.marker-UNC13388811_lod.drop-1.5_snpsqc_5.batches.csv
    Untracked:  data/ici-early.vs.pbs_age.of.onset-additive.covariates_genes_chr-7_peak.marker-UNC13388811_lod.drop-1.5_snpsqc_5.batches.csv
    Untracked:  data/ici-early.vs.pbs_age.of.onset-no.covariates_blup_sub_chr-14_peak.marker-UNC24056202_lod.drop-1.5_snpsqc_5.batches_mis.csv
    Untracked:  data/ici-early.vs.pbs_age.of.onset-no.covariates_blup_sub_chr-17_peak.marker-UNCJPD006614_lod.drop-1.5_snpsqc_5.batches_mis.csv
    Untracked:  data/ici-early.vs.pbs_age.of.onset-no.covariates_blup_sub_chr-18_peak.marker-UNCHS045343_lod.drop-1.5_snpsqc_5.batches_mis.csv
    Untracked:  data/ici-early.vs.pbs_age.of.onset-no.covariates_blup_sub_chr-8_peak.marker-UNC15524531_lod.drop-1.5_snpsqc_5.batches_mis.csv
    Untracked:  data/ici-early.vs.pbs_age.of.onset-no.covariates_blup_sub_chr-X_peak.marker-UNCHS048314_lod.drop-1.5_snpsqc_5.batches_mis.csv
    Untracked:  data/ici-early.vs.pbs_age.of.onset-no.covariates_genes_chr-14_peak.marker-UNC24056202_lod.drop-1.5_snpsqc_5.batches_mis.csv
    Untracked:  data/ici-early.vs.pbs_age.of.onset-no.covariates_genes_chr-17_peak.marker-UNCJPD006614_lod.drop-1.5_snpsqc_5.batches_mis.csv
    Untracked:  data/ici-early.vs.pbs_age.of.onset-no.covariates_genes_chr-18_peak.marker-UNCHS045343_lod.drop-1.5_snpsqc_5.batches_mis.csv
    Untracked:  data/ici-early.vs.pbs_age.of.onset-no.covariates_genes_chr-8_peak.marker-UNC15524531_lod.drop-1.5_snpsqc_5.batches_mis.csv
    Untracked:  data/ici-early.vs.pbs_age.of.onset-no.covariates_genes_chr-X_peak.marker-UNCHS048314_lod.drop-1.5_snpsqc_5.batches_mis.csv
    Untracked:  data/ici-early.vs.pbs_blup.full_chr-10_5.batches.csv
    Untracked:  data/ici-early.vs.pbs_blup.full_chr-10_5.batches_mis.csv
    Untracked:  data/ici-early.vs.pbs_blup.full_chr-10_snpsqc_5.batches.csv
    Untracked:  data/ici-early.vs.pbs_blup.full_chr-10_snpsqc_5.batches_mis.csv
    Untracked:  data/ici-early.vs.pbs_blup.full_chr-10_snpsqc_dis_no-x_updated_5.batches.csv
    Untracked:  data/ici-early.vs.pbs_blup.full_chr-10_snpsqc_dis_no-x_updated_5.batches_mis.csv
    Untracked:  data/ici-early.vs.pbs_blup.full_chr-11_5.batches.csv
    Untracked:  data/ici-early.vs.pbs_blup.full_chr-11_5.batches_mis.csv
    Untracked:  data/ici-early.vs.pbs_blup.full_chr-11_snpsqc_5.batches.csv
    Untracked:  data/ici-early.vs.pbs_blup.full_chr-11_snpsqc_5.batches_mis.csv
    Untracked:  data/ici-early.vs.pbs_blup.full_chr-11_snpsqc_dis_no-x_updated_5.batches.csv
    Untracked:  data/ici-early.vs.pbs_blup.full_chr-11_snpsqc_dis_no-x_updated_5.batches_mis.csv
    Untracked:  data/ici-early.vs.pbs_blup.full_chr-12_5.batches.csv
    Untracked:  data/ici-early.vs.pbs_blup.full_chr-12_5.batches_mis.csv
    Untracked:  data/ici-early.vs.pbs_blup.full_chr-12_snpsqc_5.batches.csv
    Untracked:  data/ici-early.vs.pbs_blup.full_chr-12_snpsqc_5.batches_mis.csv
    Untracked:  data/ici-early.vs.pbs_blup.full_chr-12_snpsqc_dis_no-x_updated_5.batches.csv
    Untracked:  data/ici-early.vs.pbs_blup.full_chr-12_snpsqc_dis_no-x_updated_5.batches_mis.csv
    Untracked:  data/ici-early.vs.pbs_blup.full_chr-13_5.batches.csv
    Untracked:  data/ici-early.vs.pbs_blup.full_chr-13_5.batches_mis.csv
    Untracked:  data/ici-early.vs.pbs_blup.full_chr-13_snpsqc_5.batches.csv
    Untracked:  data/ici-early.vs.pbs_blup.full_chr-13_snpsqc_5.batches_mis.csv
    Untracked:  data/ici-early.vs.pbs_blup.full_chr-13_snpsqc_dis_no-x_updated_5.batches.csv
    Untracked:  data/ici-early.vs.pbs_blup.full_chr-13_snpsqc_dis_no-x_updated_5.batches_mis.csv
    Untracked:  data/ici-early.vs.pbs_blup.full_chr-14_5.batches.csv
    Untracked:  data/ici-early.vs.pbs_blup.full_chr-14_5.batches_mis.csv
    Untracked:  data/ici-early.vs.pbs_blup.full_chr-14_snpsqc_5.batches.csv
    Untracked:  data/ici-early.vs.pbs_blup.full_chr-14_snpsqc_5.batches_mis.csv
    Untracked:  data/ici-early.vs.pbs_blup.full_chr-14_snpsqc_dis_no-x_updated_5.batches.csv
    Untracked:  data/ici-early.vs.pbs_blup.full_chr-14_snpsqc_dis_no-x_updated_5.batches_mis.csv
    Untracked:  data/ici-early.vs.pbs_blup.full_chr-15_5.batches.csv
    Untracked:  data/ici-early.vs.pbs_blup.full_chr-15_5.batches_mis.csv
    Untracked:  data/ici-early.vs.pbs_blup.full_chr-15_snpsqc_5.batches.csv
    Untracked:  data/ici-early.vs.pbs_blup.full_chr-15_snpsqc_5.batches_mis.csv
    Untracked:  data/ici-early.vs.pbs_blup.full_chr-15_snpsqc_dis_no-x_updated_5.batches.csv
    Untracked:  data/ici-early.vs.pbs_blup.full_chr-15_snpsqc_dis_no-x_updated_5.batches_mis.csv
    Untracked:  data/ici-early.vs.pbs_blup.full_chr-16_5.batches.csv
    Untracked:  data/ici-early.vs.pbs_blup.full_chr-16_5.batches_mis.csv
    Untracked:  data/ici-early.vs.pbs_blup.full_chr-16_snpsqc_5.batches.csv
    Untracked:  data/ici-early.vs.pbs_blup.full_chr-16_snpsqc_5.batches_mis.csv
    Untracked:  data/ici-early.vs.pbs_blup.full_chr-16_snpsqc_dis_no-x_updated_5.batches.csv
    Untracked:  data/ici-early.vs.pbs_blup.full_chr-16_snpsqc_dis_no-x_updated_5.batches_mis.csv
    Untracked:  data/ici-early.vs.pbs_blup.full_chr-17_5.batches.csv
    Untracked:  data/ici-early.vs.pbs_blup.full_chr-17_5.batches_mis.csv
    Untracked:  data/ici-early.vs.pbs_blup.full_chr-17_snpsqc_5.batches.csv
    Untracked:  data/ici-early.vs.pbs_blup.full_chr-17_snpsqc_5.batches_mis.csv
    Untracked:  data/ici-early.vs.pbs_blup.full_chr-17_snpsqc_dis_no-x_updated_5.batches.csv
    Untracked:  data/ici-early.vs.pbs_blup.full_chr-17_snpsqc_dis_no-x_updated_5.batches_mis.csv
    Untracked:  data/ici-early.vs.pbs_blup.full_chr-18_5.batches.csv
    Untracked:  data/ici-early.vs.pbs_blup.full_chr-18_5.batches_mis.csv
    Untracked:  data/ici-early.vs.pbs_blup.full_chr-18_snpsqc_5.batches.csv
    Untracked:  data/ici-early.vs.pbs_blup.full_chr-18_snpsqc_5.batches_mis.csv
    Untracked:  data/ici-early.vs.pbs_blup.full_chr-18_snpsqc_dis_no-x_updated_5.batches.csv
    Untracked:  data/ici-early.vs.pbs_blup.full_chr-18_snpsqc_dis_no-x_updated_5.batches_mis.csv
    Untracked:  data/ici-early.vs.pbs_blup.full_chr-19_5.batches.csv
    Untracked:  data/ici-early.vs.pbs_blup.full_chr-19_5.batches_mis.csv
    Untracked:  data/ici-early.vs.pbs_blup.full_chr-19_snpsqc_5.batches.csv
    Untracked:  data/ici-early.vs.pbs_blup.full_chr-19_snpsqc_5.batches_mis.csv
    Untracked:  data/ici-early.vs.pbs_blup.full_chr-19_snpsqc_dis_no-x_updated_5.batches.csv
    Untracked:  data/ici-early.vs.pbs_blup.full_chr-19_snpsqc_dis_no-x_updated_5.batches_mis.csv
    Untracked:  data/ici-early.vs.pbs_blup.full_chr-1_5.batches.csv
    Untracked:  data/ici-early.vs.pbs_blup.full_chr-1_5.batches_mis.csv
    Untracked:  data/ici-early.vs.pbs_blup.full_chr-1_snpsqc_5.batches.csv
    Untracked:  data/ici-early.vs.pbs_blup.full_chr-1_snpsqc_5.batches_mis.csv
    Untracked:  data/ici-early.vs.pbs_blup.full_chr-1_snpsqc_dis_no-x_updated_5.batches.csv
    Untracked:  data/ici-early.vs.pbs_blup.full_chr-1_snpsqc_dis_no-x_updated_5.batches_mis.csv
    Untracked:  data/ici-early.vs.pbs_blup.full_chr-2_5.batches.csv
    Untracked:  data/ici-early.vs.pbs_blup.full_chr-2_5.batches_mis.csv
    Untracked:  data/ici-early.vs.pbs_blup.full_chr-2_snpsqc_5.batches.csv
    Untracked:  data/ici-early.vs.pbs_blup.full_chr-2_snpsqc_5.batches_mis.csv
    Untracked:  data/ici-early.vs.pbs_blup.full_chr-2_snpsqc_dis_no-x_updated_5.batches.csv
    Untracked:  data/ici-early.vs.pbs_blup.full_chr-2_snpsqc_dis_no-x_updated_5.batches_mis.csv
    Untracked:  data/ici-early.vs.pbs_blup.full_chr-3_5.batches.csv
    Untracked:  data/ici-early.vs.pbs_blup.full_chr-3_5.batches_mis.csv
    Untracked:  data/ici-early.vs.pbs_blup.full_chr-3_snpsqc_5.batches.csv
    Untracked:  data/ici-early.vs.pbs_blup.full_chr-3_snpsqc_5.batches_mis.csv
    Untracked:  data/ici-early.vs.pbs_blup.full_chr-3_snpsqc_dis_no-x_updated_5.batches.csv
    Untracked:  data/ici-early.vs.pbs_blup.full_chr-3_snpsqc_dis_no-x_updated_5.batches_mis.csv
    Untracked:  data/ici-early.vs.pbs_blup.full_chr-4_5.batches.csv
    Untracked:  data/ici-early.vs.pbs_blup.full_chr-4_5.batches_mis.csv
    Untracked:  data/ici-early.vs.pbs_blup.full_chr-4_snpsqc_5.batches.csv
    Untracked:  data/ici-early.vs.pbs_blup.full_chr-4_snpsqc_5.batches_mis.csv
    Untracked:  data/ici-early.vs.pbs_blup.full_chr-4_snpsqc_dis_no-x_updated_5.batches.csv
    Untracked:  data/ici-early.vs.pbs_blup.full_chr-4_snpsqc_dis_no-x_updated_5.batches_mis.csv
    Untracked:  data/ici-early.vs.pbs_blup.full_chr-5_5.batches.csv
    Untracked:  data/ici-early.vs.pbs_blup.full_chr-5_5.batches_mis.csv
    Untracked:  data/ici-early.vs.pbs_blup.full_chr-5_snpsqc_5.batches.csv
    Untracked:  data/ici-early.vs.pbs_blup.full_chr-5_snpsqc_5.batches_mis.csv
    Untracked:  data/ici-early.vs.pbs_blup.full_chr-5_snpsqc_dis_no-x_updated_5.batches.csv
    Untracked:  data/ici-early.vs.pbs_blup.full_chr-5_snpsqc_dis_no-x_updated_5.batches_mis.csv
    Untracked:  data/ici-early.vs.pbs_blup.full_chr-6_5.batches.csv
    Untracked:  data/ici-early.vs.pbs_blup.full_chr-6_5.batches_mis.csv
    Untracked:  data/ici-early.vs.pbs_blup.full_chr-6_snpsqc_5.batches.csv
    Untracked:  data/ici-early.vs.pbs_blup.full_chr-6_snpsqc_5.batches_mis.csv
    Untracked:  data/ici-early.vs.pbs_blup.full_chr-6_snpsqc_dis_no-x_updated_5.batches.csv
    Untracked:  data/ici-early.vs.pbs_blup.full_chr-6_snpsqc_dis_no-x_updated_5.batches_mis.csv
    Untracked:  data/ici-early.vs.pbs_blup.full_chr-7_5.batches.csv
    Untracked:  data/ici-early.vs.pbs_blup.full_chr-7_5.batches_mis.csv
    Untracked:  data/ici-early.vs.pbs_blup.full_chr-7_snpsqc_5.batches.csv
    Untracked:  data/ici-early.vs.pbs_blup.full_chr-7_snpsqc_5.batches_mis.csv
    Untracked:  data/ici-early.vs.pbs_blup.full_chr-7_snpsqc_dis_no-x_updated_5.batches.csv
    Untracked:  data/ici-early.vs.pbs_blup.full_chr-7_snpsqc_dis_no-x_updated_5.batches_mis.csv
    Untracked:  data/ici-early.vs.pbs_blup.full_chr-8_5.batches.csv
    Untracked:  data/ici-early.vs.pbs_blup.full_chr-8_5.batches_mis.csv
    Untracked:  data/ici-early.vs.pbs_blup.full_chr-8_snpsqc_5.batches.csv
    Untracked:  data/ici-early.vs.pbs_blup.full_chr-8_snpsqc_5.batches_mis.csv
    Untracked:  data/ici-early.vs.pbs_blup.full_chr-8_snpsqc_dis_no-x_updated_5.batches.csv
    Untracked:  data/ici-early.vs.pbs_blup.full_chr-8_snpsqc_dis_no-x_updated_5.batches_mis.csv
    Untracked:  data/ici-early.vs.pbs_blup.full_chr-9_5.batches.csv
    Untracked:  data/ici-early.vs.pbs_blup.full_chr-9_5.batches_mis.csv
    Untracked:  data/ici-early.vs.pbs_blup.full_chr-9_snpsqc_5.batches.csv
    Untracked:  data/ici-early.vs.pbs_blup.full_chr-9_snpsqc_5.batches_mis.csv
    Untracked:  data/ici-early.vs.pbs_blup.full_chr-9_snpsqc_dis_no-x_updated_5.batches.csv
    Untracked:  data/ici-early.vs.pbs_blup.full_chr-9_snpsqc_dis_no-x_updated_5.batches_mis.csv
    Untracked:  data/ici-early.vs.pbs_blup.full_chr-X_5.batches.csv
    Untracked:  data/ici-early.vs.pbs_blup.full_chr-X_5.batches_mis.csv
    Untracked:  data/ici-early.vs.pbs_blup.full_chr-X_snpsqc_5.batches.csv
    Untracked:  data/ici-early.vs.pbs_blup.full_chr-X_snpsqc_5.batches_mis.csv
    Untracked:  data/ici-early.vs.pbs_blup.full_chr-X_snpsqc_dis_no-x_updated_5.batches.csv
    Untracked:  data/ici-early.vs.pbs_blup.full_chr-X_snpsqc_dis_no-x_updated_5.batches_mis.csv
    Untracked:  data/ici-early.vs.pbs_blup.qc_chr-10_5.batches.csv
    Untracked:  data/ici-early.vs.pbs_blup.qc_chr-10_5.batches_mis.csv
    Untracked:  data/ici-early.vs.pbs_blup.qc_chr-10_snpsqc_5.batches.csv
    Untracked:  data/ici-early.vs.pbs_blup.qc_chr-10_snpsqc_5.batches_mis.csv
    Untracked:  data/ici-early.vs.pbs_blup.qc_chr-10_snpsqc_dis_no-x_updated_5.batches.csv
    Untracked:  data/ici-early.vs.pbs_blup.qc_chr-10_snpsqc_dis_no-x_updated_5.batches_mis.csv
    Untracked:  data/ici-early.vs.pbs_blup.qc_chr-11_5.batches.csv
    Untracked:  data/ici-early.vs.pbs_blup.qc_chr-11_5.batches_mis.csv
    Untracked:  data/ici-early.vs.pbs_blup.qc_chr-11_snpsqc_5.batches.csv
    Untracked:  data/ici-early.vs.pbs_blup.qc_chr-11_snpsqc_5.batches_mis.csv
    Untracked:  data/ici-early.vs.pbs_blup.qc_chr-11_snpsqc_dis_no-x_updated_5.batches.csv
    Untracked:  data/ici-early.vs.pbs_blup.qc_chr-11_snpsqc_dis_no-x_updated_5.batches_mis.csv
    Untracked:  data/ici-early.vs.pbs_blup.qc_chr-12_5.batches.csv
    Untracked:  data/ici-early.vs.pbs_blup.qc_chr-12_5.batches_mis.csv
    Untracked:  data/ici-early.vs.pbs_blup.qc_chr-12_snpsqc_5.batches.csv
    Untracked:  data/ici-early.vs.pbs_blup.qc_chr-12_snpsqc_5.batches_mis.csv
    Untracked:  data/ici-early.vs.pbs_blup.qc_chr-12_snpsqc_dis_no-x_updated_5.batches.csv
    Untracked:  data/ici-early.vs.pbs_blup.qc_chr-12_snpsqc_dis_no-x_updated_5.batches_mis.csv
    Untracked:  data/ici-early.vs.pbs_blup.qc_chr-13_5.batches.csv
    Untracked:  data/ici-early.vs.pbs_blup.qc_chr-13_5.batches_mis.csv
    Untracked:  data/ici-early.vs.pbs_blup.qc_chr-13_snpsqc_5.batches.csv
    Untracked:  data/ici-early.vs.pbs_blup.qc_chr-13_snpsqc_5.batches_mis.csv
    Untracked:  data/ici-early.vs.pbs_blup.qc_chr-13_snpsqc_dis_no-x_updated_5.batches.csv
    Untracked:  data/ici-early.vs.pbs_blup.qc_chr-13_snpsqc_dis_no-x_updated_5.batches_mis.csv
    Untracked:  data/ici-early.vs.pbs_blup.qc_chr-14_5.batches.csv
    Untracked:  data/ici-early.vs.pbs_blup.qc_chr-14_5.batches_mis.csv
    Untracked:  data/ici-early.vs.pbs_blup.qc_chr-14_snpsqc_5.batches.csv
    Untracked:  data/ici-early.vs.pbs_blup.qc_chr-14_snpsqc_5.batches_mis.csv
    Untracked:  data/ici-early.vs.pbs_blup.qc_chr-14_snpsqc_dis_no-x_updated_5.batches.csv
    Untracked:  data/ici-early.vs.pbs_blup.qc_chr-14_snpsqc_dis_no-x_updated_5.batches_mis.csv
    Untracked:  data/ici-early.vs.pbs_blup.qc_chr-15_5.batches.csv
    Untracked:  data/ici-early.vs.pbs_blup.qc_chr-15_5.batches_mis.csv
    Untracked:  data/ici-early.vs.pbs_blup.qc_chr-15_snpsqc_5.batches.csv
    Untracked:  data/ici-early.vs.pbs_blup.qc_chr-15_snpsqc_5.batches_mis.csv
    Untracked:  data/ici-early.vs.pbs_blup.qc_chr-16_5.batches.csv
    Untracked:  data/ici-early.vs.pbs_blup.qc_chr-16_5.batches_mis.csv
    Untracked:  data/ici-early.vs.pbs_blup.qc_chr-16_snpsqc_5.batches.csv
    Untracked:  data/ici-early.vs.pbs_blup.qc_chr-16_snpsqc_5.batches_mis.csv
    Untracked:  data/ici-early.vs.pbs_blup.qc_chr-16_snpsqc_dis_no-x_updated_5.batches.csv
    Untracked:  data/ici-early.vs.pbs_blup.qc_chr-16_snpsqc_dis_no-x_updated_5.batches_mis.csv
    Untracked:  data/ici-early.vs.pbs_blup.qc_chr-17_5.batches.csv
    Untracked:  data/ici-early.vs.pbs_blup.qc_chr-17_5.batches_mis.csv
    Untracked:  data/ici-early.vs.pbs_blup.qc_chr-17_snpsqc_5.batches.csv
    Untracked:  data/ici-early.vs.pbs_blup.qc_chr-17_snpsqc_5.batches_mis.csv
    Untracked:  data/ici-early.vs.pbs_blup.qc_chr-18_5.batches.csv
    Untracked:  data/ici-early.vs.pbs_blup.qc_chr-18_5.batches_mis.csv
    Untracked:  data/ici-early.vs.pbs_blup.qc_chr-18_snpsqc_5.batches.csv
    Untracked:  data/ici-early.vs.pbs_blup.qc_chr-18_snpsqc_5.batches_mis.csv
    Untracked:  data/ici-early.vs.pbs_blup.qc_chr-18_snpsqc_dis_no-x_updated_5.batches.csv
    Untracked:  data/ici-early.vs.pbs_blup.qc_chr-18_snpsqc_dis_no-x_updated_5.batches_mis.csv
    Untracked:  data/ici-early.vs.pbs_blup.qc_chr-19_5.batches.csv
    Untracked:  data/ici-early.vs.pbs_blup.qc_chr-19_5.batches_mis.csv
    Untracked:  data/ici-early.vs.pbs_blup.qc_chr-19_snpsqc_5.batches.csv
    Untracked:  data/ici-early.vs.pbs_blup.qc_chr-19_snpsqc_5.batches_mis.csv
    Untracked:  data/ici-early.vs.pbs_blup.qc_chr-19_snpsqc_dis_no-x_updated_5.batches.csv
    Untracked:  data/ici-early.vs.pbs_blup.qc_chr-19_snpsqc_dis_no-x_updated_5.batches_mis.csv
    Untracked:  data/ici-early.vs.pbs_blup.qc_chr-1_5.batches.csv
    Untracked:  data/ici-early.vs.pbs_blup.qc_chr-1_5.batches_mis.csv
    Untracked:  data/ici-early.vs.pbs_blup.qc_chr-1_snpsqc_5.batches.csv
    Untracked:  data/ici-early.vs.pbs_blup.qc_chr-1_snpsqc_5.batches_mis.csv
    Untracked:  data/ici-early.vs.pbs_blup.qc_chr-1_snpsqc_dis_no-x_updated_5.batches.csv
    Untracked:  data/ici-early.vs.pbs_blup.qc_chr-1_snpsqc_dis_no-x_updated_5.batches_mis.csv
    Untracked:  data/ici-early.vs.pbs_blup.qc_chr-2_5.batches.csv
    Untracked:  data/ici-early.vs.pbs_blup.qc_chr-2_5.batches_mis.csv
    Untracked:  data/ici-early.vs.pbs_blup.qc_chr-2_snpsqc_5.batches.csv
    Untracked:  data/ici-early.vs.pbs_blup.qc_chr-2_snpsqc_5.batches_mis.csv
    Untracked:  data/ici-early.vs.pbs_blup.qc_chr-2_snpsqc_dis_no-x_updated_5.batches.csv
    Untracked:  data/ici-early.vs.pbs_blup.qc_chr-2_snpsqc_dis_no-x_updated_5.batches_mis.csv
    Untracked:  data/ici-early.vs.pbs_blup.qc_chr-3_5.batches.csv
    Untracked:  data/ici-early.vs.pbs_blup.qc_chr-3_5.batches_mis.csv
    Untracked:  data/ici-early.vs.pbs_blup.qc_chr-3_snpsqc_5.batches.csv
    Untracked:  data/ici-early.vs.pbs_blup.qc_chr-3_snpsqc_5.batches_mis.csv
    Untracked:  data/ici-early.vs.pbs_blup.qc_chr-3_snpsqc_dis_no-x_updated_5.batches.csv
    Untracked:  data/ici-early.vs.pbs_blup.qc_chr-3_snpsqc_dis_no-x_updated_5.batches_mis.csv
    Untracked:  data/ici-early.vs.pbs_blup.qc_chr-4_5.batches.csv
    Untracked:  data/ici-early.vs.pbs_blup.qc_chr-4_5.batches_mis.csv
    Untracked:  data/ici-early.vs.pbs_blup.qc_chr-4_snpsqc_5.batches.csv
    Untracked:  data/ici-early.vs.pbs_blup.qc_chr-4_snpsqc_5.batches_mis.csv
    Untracked:  data/ici-early.vs.pbs_blup.qc_chr-4_snpsqc_dis_no-x_updated_5.batches.csv
    Untracked:  data/ici-early.vs.pbs_blup.qc_chr-4_snpsqc_dis_no-x_updated_5.batches_mis.csv
    Untracked:  data/ici-early.vs.pbs_blup.qc_chr-5_5.batches.csv
    Untracked:  data/ici-early.vs.pbs_blup.qc_chr-5_5.batches_mis.csv
    Untracked:  data/ici-early.vs.pbs_blup.qc_chr-5_snpsqc_5.batches.csv
    Untracked:  data/ici-early.vs.pbs_blup.qc_chr-5_snpsqc_5.batches_mis.csv
    Untracked:  data/ici-early.vs.pbs_blup.qc_chr-5_snpsqc_dis_no-x_updated_5.batches.csv
    Untracked:  data/ici-early.vs.pbs_blup.qc_chr-5_snpsqc_dis_no-x_updated_5.batches_mis.csv
    Untracked:  data/ici-early.vs.pbs_blup.qc_chr-6_5.batches.csv
    Untracked:  data/ici-early.vs.pbs_blup.qc_chr-6_5.batches_mis.csv
    Untracked:  data/ici-early.vs.pbs_blup.qc_chr-6_snpsqc_5.batches.csv
    Untracked:  data/ici-early.vs.pbs_blup.qc_chr-6_snpsqc_5.batches_mis.csv
    Untracked:  data/ici-early.vs.pbs_blup.qc_chr-6_snpsqc_dis_no-x_updated_5.batches.csv
    Untracked:  data/ici-early.vs.pbs_blup.qc_chr-6_snpsqc_dis_no-x_updated_5.batches_mis.csv
    Untracked:  data/ici-early.vs.pbs_blup.qc_chr-7_5.batches.csv
    Untracked:  data/ici-early.vs.pbs_blup.qc_chr-7_5.batches_mis.csv
    Untracked:  data/ici-early.vs.pbs_blup.qc_chr-7_snpsqc_5.batches.csv
    Untracked:  data/ici-early.vs.pbs_blup.qc_chr-7_snpsqc_5.batches_mis.csv
    Untracked:  data/ici-early.vs.pbs_blup.qc_chr-7_snpsqc_dis_no-x_updated_5.batches.csv
    Untracked:  data/ici-early.vs.pbs_blup.qc_chr-7_snpsqc_dis_no-x_updated_5.batches_mis.csv
    Untracked:  data/ici-early.vs.pbs_blup.qc_chr-8_5.batches.csv
    Untracked:  data/ici-early.vs.pbs_blup.qc_chr-8_5.batches_mis.csv
    Untracked:  data/ici-early.vs.pbs_blup.qc_chr-8_snpsqc_5.batches.csv
    Untracked:  data/ici-early.vs.pbs_blup.qc_chr-8_snpsqc_5.batches_mis.csv
    Untracked:  data/ici-early.vs.pbs_blup.qc_chr-8_snpsqc_dis_no-x_updated_5.batches.csv
    Untracked:  data/ici-early.vs.pbs_blup.qc_chr-8_snpsqc_dis_no-x_updated_5.batches_mis.csv
    Untracked:  data/ici-early.vs.pbs_blup.qc_chr-9_5.batches.csv
    Untracked:  data/ici-early.vs.pbs_blup.qc_chr-9_5.batches_mis.csv
    Untracked:  data/ici-early.vs.pbs_blup.qc_chr-9_snpsqc_5.batches.csv
    Untracked:  data/ici-early.vs.pbs_blup.qc_chr-9_snpsqc_5.batches_mis.csv
    Untracked:  data/ici-early.vs.pbs_blup.qc_chr-9_snpsqc_dis_no-x_updated_5.batches.csv
    Untracked:  data/ici-early.vs.pbs_blup.qc_chr-9_snpsqc_dis_no-x_updated_5.batches_mis.csv
    Untracked:  data/ici-early.vs.pbs_blup.qc_chr-X_5.batches.csv
    Untracked:  data/ici-early.vs.pbs_blup.qc_chr-X_5.batches_mis.csv
    Untracked:  data/ici-early.vs.pbs_blup.qc_chr-X_snpsqc_5.batches.csv
    Untracked:  data/ici-early.vs.pbs_blup.qc_chr-X_snpsqc_5.batches_mis.csv
    Untracked:  data/ici-early.vs.pbs_blup.qc_chr-X_snpsqc_dis_no-x_updated_5.batches.csv
    Untracked:  data/ici-early.vs.pbs_blup.qc_chr-X_snpsqc_dis_no-x_updated_5.batches_mis.csv
    Untracked:  data/ici-early.vs.pbs_blup_sub_chr-18_peak.marker-UNCHS045343_lod.drop-1.5_snpsqc_5.batches_mis.csv
    Untracked:  data/ici-early.vs.pbs_genes_chr-18_peak.marker-UNCHS045343_lod.drop-1.5_snpsqc_5.batches_mis.csv
    Untracked:  data/ici-early.vs.pbs_gm_qtl_5.batches.csv
    Untracked:  data/ici-early.vs.pbs_gm_qtl_5.batches_mis.csv
    Untracked:  data/ici-early.vs.pbs_gm_qtl_snpsqc_5.batches.csv
    Untracked:  data/ici-early.vs.pbs_gm_qtl_snpsqc_5.batches_mis.csv
    Untracked:  data/ici-early.vs.pbs_gm_qtl_snpsqc_dis_no-x_updated_5.batches.csv
    Untracked:  data/ici-early.vs.pbs_gm_qtl_snpsqc_dis_no-x_updated_5.batches_mis.csv
    Untracked:  data/ici-early.vs.pbs_marker.freq_low.geno.freq.removed_geno.ratio_5.batches.csv
    Untracked:  data/ici-early.vs.pbs_marker.freq_low.geno.freq.removed_geno.ratio_5.batches_mis.csv
    Untracked:  data/ici-early.vs.pbs_marker.freq_low.geno.freq.removed_sample.outliers.removed_geno.ratio_5.batches.csv
    Untracked:  data/ici-early.vs.pbs_marker.freq_low.geno.freq.removed_sample.outliers.removed_geno.ratio_5.batches_mis.csv
    Untracked:  data/ici-early.vs.pbs_marker.freq_low.probs.freq.removed_geno.ratio_5.batches.csv
    Untracked:  data/ici-early.vs.pbs_marker.freq_low.probs.freq.removed_geno.ratio_5.batches_mis.csv
    Untracked:  data/ici-early.vs.pbs_marker.freq_low.probs.freq.removed_sample.outliers.removed_geno.ratio_5.batches.csv
    Untracked:  data/ici-early.vs.pbs_marker.freq_low.probs.freq.removed_sample.outliers.removed_geno.ratio_5.batches_mis.csv
    Untracked:  data/ici-early.vs.pbs_rz.age-additive.covariates_blup_sub_chr-7_peak.marker-UNC13239362_lod.drop-1.5_snpsqc_5.batches.csv
    Untracked:  data/ici-early.vs.pbs_rz.age-additive.covariates_blup_sub_chr-7_peak.marker-UNC13239362_lod.drop-1.5_snpsqc_5.batches_mis.csv
    Untracked:  data/ici-early.vs.pbs_rz.age-additive.covariates_blup_sub_chr-7_peak.marker-UNC13239362_lod.drop-1.5_snpsqc_dis_no-x_updated_5.batches.csv
    Untracked:  data/ici-early.vs.pbs_rz.age-additive.covariates_blup_sub_chr-7_peak.marker-UNC13239362_lod.drop-1.5_snpsqc_dis_no-x_updated_5.batches_mis.csv
    Untracked:  data/ici-early.vs.pbs_rz.age-additive.covariates_genes_chr-7_peak.marker-UNC13239362_lod.drop-1.5_snpsqc_5.batches.csv
    Untracked:  data/ici-early.vs.pbs_rz.age-additive.covariates_genes_chr-7_peak.marker-UNC13239362_lod.drop-1.5_snpsqc_5.batches_mis.csv
    Untracked:  data/ici-early.vs.pbs_rz.age-additive.covariates_genes_chr-7_peak.marker-UNC13239362_lod.drop-1.5_snpsqc_dis_no-x_updated_5.batches.csv
    Untracked:  data/ici-early.vs.pbs_rz.age-additive.covariates_genes_chr-7_peak.marker-UNC13239362_lod.drop-1.5_snpsqc_dis_no-x_updated_5.batches_mis.csv
    Untracked:  data/ici-early.vs.pbs_scanone_5.batches.Rdata
    Untracked:  data/ici-early.vs.pbs_scanone_5.batches_mis.Rdata
    Untracked:  data/ici-early.vs.pbs_scanone_snpsqc_5.batches.Rdata
    Untracked:  data/ici-early.vs.pbs_scanone_snpsqc_5.batches_mis.Rdata
    Untracked:  data/ici-early.vs.pbs_scanone_snpsqc_dis_no-x_updated_5.batches.Rdata
    Untracked:  data/ici-early.vs.pbs_scanone_snpsqc_dis_no-x_updated_5.batches_mis.Rdata
    Untracked:  data/ici.vs.eoi_age.of.onset-no.covariates_blup_sub_chr-10_peak.marker-JAX00020646_lod.drop-1.5_5.batches_0.csv
    Untracked:  data/ici.vs.eoi_age.of.onset-no.covariates_blup_sub_chr-10_peak.marker-JAX00292499_lod.drop-1.5_5.batches_52.csv
    Untracked:  data/ici.vs.eoi_age.of.onset-no.covariates_blup_sub_chr-10_peak.marker-JAX00292927_lod.drop-1.5_5.batches_0.csv
    Untracked:  data/ici.vs.eoi_age.of.onset-no.covariates_blup_sub_chr-10_peak.marker-JAX00292927_lod.drop-1.5_5.batches_52.csv
    Untracked:  data/ici.vs.eoi_age.of.onset-no.covariates_blup_sub_chr-10_peak.marker-JAX00294019_lod.drop-1.5_snpsqc_dis_no-x_updated_5.batches_0.csv
    Untracked:  data/ici.vs.eoi_age.of.onset-no.covariates_blup_sub_chr-10_peak.marker-UNC18216614_lod.drop-1.5_5.batches_52.csv
    Untracked:  data/ici.vs.eoi_age.of.onset-no.covariates_blup_sub_chr-10_peak.marker-UNC18240977_lod.drop-1.5_snpsqc_dis_no-x_updated_5.batches_52.csv
    Untracked:  data/ici.vs.eoi_age.of.onset-no.covariates_blup_sub_chr-10_peak.marker-UNC18311938_lod.drop-1.5_5.batches_0.csv
    Untracked:  data/ici.vs.eoi_age.of.onset-no.covariates_blup_sub_chr-10_peak.marker-UNC18311938_lod.drop-1.5_5.batches_52.csv
    Untracked:  data/ici.vs.eoi_age.of.onset-no.covariates_blup_sub_chr-10_peak.marker-UNC18311938_lod.drop-1.5_snpsqc_5.batches_52.csv
    Untracked:  data/ici.vs.eoi_age.of.onset-no.covariates_blup_sub_chr-10_peak.marker-UNC18311938_lod.drop-1.5_snpsqc_5.batches_mis_52.csv
    Untracked:  data/ici.vs.eoi_age.of.onset-no.covariates_blup_sub_chr-10_peak.marker-UNC18343181_lod.drop-1.5_5.batches_0.csv
    Untracked:  data/ici.vs.eoi_age.of.onset-no.covariates_blup_sub_chr-10_peak.marker-UNC18363544_lod.drop-1.5_5.batches_0.csv
    Untracked:  data/ici.vs.eoi_age.of.onset-no.covariates_blup_sub_chr-10_peak.marker-UNC18363544_lod.drop-1.5_5.batches_52.csv
    Untracked:  data/ici.vs.eoi_age.of.onset-no.covariates_blup_sub_chr-10_peak.marker-UNC18363544_lod.drop-1.5_snpsqc_5.batches_mis_0.csv
    Untracked:  data/ici.vs.eoi_age.of.onset-no.covariates_blup_sub_chr-10_peak.marker-UNC18376338_lod.drop-1.5_snpsqc_5.batches_0.csv
    Untracked:  data/ici.vs.eoi_age.of.onset-no.covariates_blup_sub_chr-10_peak.marker-UNCHS028236_lod.drop-1.5_snpsqc_5.batches_mis_0.csv
    Untracked:  data/ici.vs.eoi_age.of.onset-no.covariates_blup_sub_chr-10_peak.marker-UNCHS028236_lod.drop-1.5_snpsqc_5.batches_mis_52.csv
    Untracked:  data/ici.vs.eoi_age.of.onset-no.covariates_blup_sub_chr-10_peak.marker-UNCHS028536_lod.drop-1.5_snpsqc_dis_no-x_updated_5.batches_mis_0.csv
    Untracked:  data/ici.vs.eoi_age.of.onset-no.covariates_blup_sub_chr-10_peak.marker-UNCHS028536_lod.drop-1.5_snpsqc_dis_no-x_updated_5.batches_mis_52.csv
    Untracked:  data/ici.vs.eoi_age.of.onset-no.covariates_blup_sub_chr-11_peak.marker-UNC19970181_lod.drop-1.5_snpsqc_5.batches_mis_0.csv
    Untracked:  data/ici.vs.eoi_age.of.onset-no.covariates_blup_sub_chr-11_peak.marker-UNC19970181_lod.drop-1.5_snpsqc_5.batches_mis_52.csv
    Untracked:  data/ici.vs.eoi_age.of.onset-no.covariates_blup_sub_chr-11_peak.marker-UNC20090524_lod.drop-1.5_snpsqc_5.batches_52.csv
    Untracked:  data/ici.vs.eoi_age.of.onset-no.covariates_blup_sub_chr-12_peak.marker-ICR499_lod.drop-1.5_snpsqc_5.batches_mis_0.csv
    Untracked:  data/ici.vs.eoi_age.of.onset-no.covariates_blup_sub_chr-12_peak.marker-ICR499_lod.drop-1.5_snpsqc_5.batches_mis_52.csv
    Untracked:  data/ici.vs.eoi_age.of.onset-no.covariates_blup_sub_chr-12_peak.marker-ICR499_lod.drop-1.5_snpsqc_dis_no-x_updated_5.batches_mis_0.csv
    Untracked:  data/ici.vs.eoi_age.of.onset-no.covariates_blup_sub_chr-13_peak.marker-UNCHS036773_lod.drop-1.5_snpsqc_5.batches_mis_0.csv
    Untracked:  data/ici.vs.eoi_age.of.onset-no.covariates_blup_sub_chr-13_peak.marker-UNCHS036773_lod.drop-1.5_snpsqc_5.batches_mis_52.csv
    Untracked:  data/ici.vs.eoi_age.of.onset-no.covariates_blup_sub_chr-14_peak.marker-UNC24056202_lod.drop-1.5_snpsqc_5.batches_mis_0.csv
    Untracked:  data/ici.vs.eoi_age.of.onset-no.covariates_blup_sub_chr-14_peak.marker-UNC24056202_lod.drop-1.5_snpsqc_5.batches_mis_52.csv
    Untracked:  data/ici.vs.eoi_age.of.onset-no.covariates_blup_sub_chr-15_peak.marker-UNC26070435_lod.drop-1.5_snpsqc_dis_no-x_updated_5.batches_mis_0.csv
    Untracked:  data/ici.vs.eoi_age.of.onset-no.covariates_blup_sub_chr-15_peak.marker-UNC26070435_lod.drop-1.5_snpsqc_dis_no-x_updated_5.batches_mis_52.csv
    Untracked:  data/ici.vs.eoi_age.of.onset-no.covariates_blup_sub_chr-17_peak.marker-UNCHS044241_lod.drop-1.5_snpsqc_5.batches_mis_0.csv
    Untracked:  data/ici.vs.eoi_age.of.onset-no.covariates_blup_sub_chr-17_peak.marker-UNCHS044241_lod.drop-1.5_snpsqc_5.batches_mis_52.csv
    Untracked:  data/ici.vs.eoi_age.of.onset-no.covariates_blup_sub_chr-17_peak.marker-UNCJPD006614_lod.drop-1.5_snpsqc_dis_no-x_updated_5.batches_mis_0.csv
    Untracked:  data/ici.vs.eoi_age.of.onset-no.covariates_blup_sub_chr-17_peak.marker-UNCrs47191360_lod.drop-1.5_snpsqc_dis_no-x_updated_5.batches_mis_52.csv
    Untracked:  data/ici.vs.eoi_age.of.onset-no.covariates_blup_sub_chr-18_peak.marker-UNCHS045343_lod.drop-1.5_snpsqc_5.batches_mis_0.csv
    Untracked:  data/ici.vs.eoi_age.of.onset-no.covariates_blup_sub_chr-18_peak.marker-UNCHS045343_lod.drop-1.5_snpsqc_5.batches_mis_52.csv
    Untracked:  data/ici.vs.eoi_age.of.onset-no.covariates_blup_sub_chr-2_peak.marker-UNC4609527_lod.drop-1.5_snpsqc_dis_no-x_updated_5.batches_mis_0.csv
    Untracked:  data/ici.vs.eoi_age.of.onset-no.covariates_blup_sub_chr-2_peak.marker-UNC4609527_lod.drop-1.5_snpsqc_dis_no-x_updated_5.batches_mis_52.csv
    Untracked:  data/ici.vs.eoi_age.of.onset-no.covariates_blup_sub_chr-2_peak.marker-UNCHS008007_lod.drop-1.5_5.batches_mis.csv
    Untracked:  data/ici.vs.eoi_age.of.onset-no.covariates_blup_sub_chr-2_peak.marker-UNCHS008007_lod.drop-1.5_5.batches_mis_0.csv
    Untracked:  data/ici.vs.eoi_age.of.onset-no.covariates_blup_sub_chr-3_peak.marker-ICR1338_lod.drop-1.5_5.batches_0.csv
    Untracked:  data/ici.vs.eoi_age.of.onset-no.covariates_blup_sub_chr-3_peak.marker-ICR1338_lod.drop-1.5_5.batches_52.csv
    Untracked:  data/ici.vs.eoi_age.of.onset-no.covariates_blup_sub_chr-3_peak.marker-ICR1338_lod.drop-1.5_5.batches_mis_0.csv
    Untracked:  data/ici.vs.eoi_age.of.onset-no.covariates_blup_sub_chr-3_peak.marker-ICR5263_lod.drop-1.5_5.batches_mis.csv
    Untracked:  data/ici.vs.eoi_age.of.onset-no.covariates_blup_sub_chr-3_peak.marker-ICR5263_lod.drop-1.5_5.batches_mis_0.csv
    Untracked:  data/ici.vs.eoi_age.of.onset-no.covariates_blup_sub_chr-3_peak.marker-JAX00105649_lod.drop-1.5_5.batches_mis.csv
    Untracked:  data/ici.vs.eoi_age.of.onset-no.covariates_blup_sub_chr-3_peak.marker-JAX00105649_lod.drop-1.5_5.batches_mis_0.csv
    Untracked:  data/ici.vs.eoi_age.of.onset-no.covariates_blup_sub_chr-3_peak.marker-JAX00106210_lod.drop-1.5_5.batches_mis_0.csv
    Untracked:  data/ici.vs.eoi_age.of.onset-no.covariates_blup_sub_chr-3_peak.marker-JAX00106957_lod.drop-1.5_5.batches_0.csv
    Untracked:  data/ici.vs.eoi_age.of.onset-no.covariates_blup_sub_chr-3_peak.marker-JAX00106957_lod.drop-1.5_5.batches_52.csv
    Untracked:  data/ici.vs.eoi_age.of.onset-no.covariates_blup_sub_chr-3_peak.marker-JAX00106957_lod.drop-1.5_5.batches_mis_0.csv
    Untracked:  data/ici.vs.eoi_age.of.onset-no.covariates_blup_sub_chr-3_peak.marker-JAX00107193_lod.drop-1.5_5.batches_mis_0.csv
    Untracked:  data/ici.vs.eoi_age.of.onset-no.covariates_blup_sub_chr-3_peak.marker-JAX00220631_lod.drop-1.5_5.batches_0.csv
    Untracked:  data/ici.vs.eoi_age.of.onset-no.covariates_blup_sub_chr-3_peak.marker-JAX00220631_lod.drop-1.5_5.batches_mis_0.csv
    Untracked:  data/ici.vs.eoi_age.of.onset-no.covariates_blup_sub_chr-3_peak.marker-JAX00514662_lod.drop-1.5_5.batches_0.csv
    Untracked:  data/ici.vs.eoi_age.of.onset-no.covariates_blup_sub_chr-3_peak.marker-JAX00514662_lod.drop-1.5_5.batches_52.csv
    Untracked:  data/ici.vs.eoi_age.of.onset-no.covariates_blup_sub_chr-3_peak.marker-JAX00516812_lod.drop-1.5_5.batches_mis.csv
    Untracked:  data/ici.vs.eoi_age.of.onset-no.covariates_blup_sub_chr-3_peak.marker-JAX00516812_lod.drop-1.5_5.batches_mis_0.csv
    Untracked:  data/ici.vs.eoi_age.of.onset-no.covariates_blup_sub_chr-3_peak.marker-JAX00517674_lod.drop-1.5_5.batches_mis.csv
    Untracked:  data/ici.vs.eoi_age.of.onset-no.covariates_blup_sub_chr-3_peak.marker-JAX00517674_lod.drop-1.5_5.batches_mis_0.csv
    Untracked:  data/ici.vs.eoi_age.of.onset-no.covariates_blup_sub_chr-3_peak.marker-JAX00518911r_lod.drop-1.5_5.batches_mis.csv
    Untracked:  data/ici.vs.eoi_age.of.onset-no.covariates_blup_sub_chr-3_peak.marker-JAX00518911r_lod.drop-1.5_5.batches_mis_0.csv
    Untracked:  data/ici.vs.eoi_age.of.onset-no.covariates_blup_sub_chr-3_peak.marker-JAX00522922_lod.drop-1.5_5.batches_mis_0.csv
    Untracked:  data/ici.vs.eoi_age.of.onset-no.covariates_blup_sub_chr-3_peak.marker-JAX00524205_lod.drop-1.5_5.batches_mis_0.csv
    Untracked:  data/ici.vs.eoi_age.of.onset-no.covariates_blup_sub_chr-3_peak.marker-JAX00529707r_lod.drop-1.5_5.batches_0.csv
    Untracked:  data/ici.vs.eoi_age.of.onset-no.covariates_blup_sub_chr-3_peak.marker-JAX00529707r_lod.drop-1.5_5.batches_52.csv
    Untracked:  data/ici.vs.eoi_age.of.onset-no.covariates_blup_sub_chr-3_peak.marker-JAX00535764_lod.drop-1.5_5.batches_52.csv
    Untracked:  data/ici.vs.eoi_age.of.onset-no.covariates_blup_sub_chr-3_peak.marker-UNC4809586_lod.drop-1.5_5.batches_mis.csv
    Untracked:  data/ici.vs.eoi_age.of.onset-no.covariates_blup_sub_chr-3_peak.marker-UNC4809586_lod.drop-1.5_5.batches_mis_0.csv
    Untracked:  data/ici.vs.eoi_age.of.onset-no.covariates_blup_sub_chr-3_peak.marker-UNC4813186_lod.drop-1.5_5.batches_0.csv
    Untracked:  data/ici.vs.eoi_age.of.onset-no.covariates_blup_sub_chr-3_peak.marker-UNC4813186_lod.drop-1.5_5.batches_52.csv
    Untracked:  data/ici.vs.eoi_age.of.onset-no.covariates_blup_sub_chr-3_peak.marker-UNC4813186_lod.drop-1.5_5.batches_mis.csv
    Untracked:  data/ici.vs.eoi_age.of.onset-no.covariates_blup_sub_chr-3_peak.marker-UNC4813186_lod.drop-1.5_5.batches_mis_0.csv
    Untracked:  data/ici.vs.eoi_age.of.onset-no.covariates_blup_sub_chr-3_peak.marker-UNC4820379_lod.drop-1.5_5.batches_mis.csv
    Untracked:  data/ici.vs.eoi_age.of.onset-no.covariates_blup_sub_chr-3_peak.marker-UNC4820379_lod.drop-1.5_5.batches_mis_0.csv
    Untracked:  data/ici.vs.eoi_age.of.onset-no.covariates_blup_sub_chr-3_peak.marker-UNC4836550_lod.drop-1.5_5.batches_0.csv
    Untracked:  data/ici.vs.eoi_age.of.onset-no.covariates_blup_sub_chr-3_peak.marker-UNC4836550_lod.drop-1.5_5.batches_52.csv
    Untracked:  data/ici.vs.eoi_age.of.onset-no.covariates_blup_sub_chr-3_peak.marker-UNC4836550_lod.drop-1.5_5.batches_mis_0.csv
    Untracked:  data/ici.vs.eoi_age.of.onset-no.covariates_blup_sub_chr-3_peak.marker-UNC4875653_lod.drop-1.5_5.batches_mis.csv
    Untracked:  data/ici.vs.eoi_age.of.onset-no.covariates_blup_sub_chr-3_peak.marker-UNC4875653_lod.drop-1.5_5.batches_mis_0.csv
    Untracked:  data/ici.vs.eoi_age.of.onset-no.covariates_blup_sub_chr-3_peak.marker-UNC4901083_lod.drop-1.5_5.batches_mis.csv
    Untracked:  data/ici.vs.eoi_age.of.onset-no.covariates_blup_sub_chr-3_peak.marker-UNC4901083_lod.drop-1.5_5.batches_mis_0.csv
    Untracked:  data/ici.vs.eoi_age.of.onset-no.covariates_blup_sub_chr-3_peak.marker-UNC4909519_lod.drop-1.5_5.batches_mis.csv
    Untracked:  data/ici.vs.eoi_age.of.onset-no.covariates_blup_sub_chr-3_peak.marker-UNC4909519_lod.drop-1.5_5.batches_mis_0.csv
    Untracked:  data/ici.vs.eoi_age.of.onset-no.covariates_blup_sub_chr-3_peak.marker-UNC4912039_lod.drop-1.5_5.batches_0.csv
    Untracked:  data/ici.vs.eoi_age.of.onset-no.covariates_blup_sub_chr-3_peak.marker-UNC4912039_lod.drop-1.5_5.batches_mis_0.csv
    Untracked:  data/ici.vs.eoi_age.of.onset-no.covariates_blup_sub_chr-3_peak.marker-UNC4912288_lod.drop-1.5_5.batches_mis.csv
    Untracked:  data/ici.vs.eoi_age.of.onset-no.covariates_blup_sub_chr-3_peak.marker-UNC4912288_lod.drop-1.5_5.batches_mis_0.csv
    Untracked:  data/ici.vs.eoi_age.of.onset-no.covariates_blup_sub_chr-3_peak.marker-UNC4915446_lod.drop-1.5_5.batches_mis.csv
    Untracked:  data/ici.vs.eoi_age.of.onset-no.covariates_blup_sub_chr-3_peak.marker-UNC4915446_lod.drop-1.5_5.batches_mis_0.csv
    Untracked:  data/ici.vs.eoi_age.of.onset-no.covariates_blup_sub_chr-3_peak.marker-UNC4923398_lod.drop-1.5_5.batches_mis.csv
    Untracked:  data/ici.vs.eoi_age.of.onset-no.covariates_blup_sub_chr-3_peak.marker-UNC4923398_lod.drop-1.5_5.batches_mis_0.csv
    Untracked:  data/ici.vs.eoi_age.of.onset-no.covariates_blup_sub_chr-3_peak.marker-UNC4924377_lod.drop-1.5_5.batches_0.csv
    Untracked:  data/ici.vs.eoi_age.of.onset-no.covariates_blup_sub_chr-3_peak.marker-UNC4924377_lod.drop-1.5_5.batches_52.csv
    Untracked:  data/ici.vs.eoi_age.of.onset-no.covariates_blup_sub_chr-3_peak.marker-UNC4924377_lod.drop-1.5_5.batches_mis_0.csv
    Untracked:  data/ici.vs.eoi_age.of.onset-no.covariates_blup_sub_chr-3_peak.marker-UNC4938565_lod.drop-1.5_5.batches_mis.csv
    Untracked:  data/ici.vs.eoi_age.of.onset-no.covariates_blup_sub_chr-3_peak.marker-UNC4938565_lod.drop-1.5_5.batches_mis_0.csv
    Untracked:  data/ici.vs.eoi_age.of.onset-no.covariates_blup_sub_chr-3_peak.marker-UNC4945186_lod.drop-1.5_5.batches_mis.csv
    Untracked:  data/ici.vs.eoi_age.of.onset-no.covariates_blup_sub_chr-3_peak.marker-UNC4945186_lod.drop-1.5_5.batches_mis_0.csv
    Untracked:  data/ici.vs.eoi_age.of.onset-no.covariates_blup_sub_chr-3_peak.marker-UNC4947492_lod.drop-1.5_5.batches_mis.csv
    Untracked:  data/ici.vs.eoi_age.of.onset-no.covariates_blup_sub_chr-3_peak.marker-UNC4947492_lod.drop-1.5_5.batches_mis_0.csv
    Untracked:  data/ici.vs.eoi_age.of.onset-no.covariates_blup_sub_chr-3_peak.marker-UNC4956618_lod.drop-1.5_5.batches_0.csv
    Untracked:  data/ici.vs.eoi_age.of.onset-no.covariates_blup_sub_chr-3_peak.marker-UNC4956618_lod.drop-1.5_5.batches_mis_0.csv
    Untracked:  data/ici.vs.eoi_age.of.onset-no.covariates_blup_sub_chr-3_peak.marker-UNC4957168_lod.drop-1.5_5.batches_52.csv
    Untracked:  data/ici.vs.eoi_age.of.onset-no.covariates_blup_sub_chr-3_peak.marker-UNC4957859_lod.drop-1.5_5.batches_mis.csv
    Untracked:  data/ici.vs.eoi_age.of.onset-no.covariates_blup_sub_chr-3_peak.marker-UNC4957859_lod.drop-1.5_5.batches_mis_0.csv
    Untracked:  data/ici.vs.eoi_age.of.onset-no.covariates_blup_sub_chr-3_peak.marker-UNC4959266_lod.drop-1.5_5.batches_mis.csv
    Untracked:  data/ici.vs.eoi_age.of.onset-no.covariates_blup_sub_chr-3_peak.marker-UNC4959266_lod.drop-1.5_5.batches_mis_0.csv
    Untracked:  data/ici.vs.eoi_age.of.onset-no.covariates_blup_sub_chr-3_peak.marker-UNC4960612_lod.drop-1.5_5.batches_mis.csv
    Untracked:  data/ici.vs.eoi_age.of.onset-no.covariates_blup_sub_chr-3_peak.marker-UNC4960612_lod.drop-1.5_5.batches_mis_0.csv
    Untracked:  data/ici.vs.eoi_age.of.onset-no.covariates_blup_sub_chr-3_peak.marker-UNC4961542_lod.drop-1.5_5.batches_mis.csv
    Untracked:  data/ici.vs.eoi_age.of.onset-no.covariates_blup_sub_chr-3_peak.marker-UNC4961542_lod.drop-1.5_5.batches_mis_0.csv
    Untracked:  data/ici.vs.eoi_age.of.onset-no.covariates_blup_sub_chr-3_peak.marker-UNC4979914_lod.drop-1.5_5.batches_mis.csv
    Untracked:  data/ici.vs.eoi_age.of.onset-no.covariates_blup_sub_chr-3_peak.marker-UNC4979914_lod.drop-1.5_5.batches_mis_0.csv
    Untracked:  data/ici.vs.eoi_age.of.onset-no.covariates_blup_sub_chr-3_peak.marker-UNC4990111_lod.drop-1.5_5.batches_mis.csv
    Untracked:  data/ici.vs.eoi_age.of.onset-no.covariates_blup_sub_chr-3_peak.marker-UNC4990111_lod.drop-1.5_5.batches_mis_0.csv
    Untracked:  data/ici.vs.eoi_age.of.onset-no.covariates_blup_sub_chr-3_peak.marker-UNC5008656_lod.drop-1.5_5.batches_0.csv
    Untracked:  data/ici.vs.eoi_age.of.onset-no.covariates_blup_sub_chr-3_peak.marker-UNC5008656_lod.drop-1.5_5.batches_52.csv
    Untracked:  data/ici.vs.eoi_age.of.onset-no.covariates_blup_sub_chr-3_peak.marker-UNC5008656_lod.drop-1.5_5.batches_mis.csv
    Untracked:  data/ici.vs.eoi_age.of.onset-no.covariates_blup_sub_chr-3_peak.marker-UNC5008656_lod.drop-1.5_5.batches_mis_0.csv
    Untracked:  data/ici.vs.eoi_age.of.onset-no.covariates_blup_sub_chr-3_peak.marker-UNC5046545_lod.drop-1.5_5.batches_mis_0.csv
    Untracked:  data/ici.vs.eoi_age.of.onset-no.covariates_blup_sub_chr-3_peak.marker-UNC5055259_lod.drop-1.5_5.batches_mis_0.csv
    Untracked:  data/ici.vs.eoi_age.of.onset-no.covariates_blup_sub_chr-3_peak.marker-UNC5062524_lod.drop-1.5_5.batches_mis_0.csv
    Untracked:  data/ici.vs.eoi_age.of.onset-no.covariates_blup_sub_chr-3_peak.marker-UNC5068215_lod.drop-1.5_5.batches_mis_0.csv
    Untracked:  data/ici.vs.eoi_age.of.onset-no.covariates_blup_sub_chr-3_peak.marker-UNC5076935_lod.drop-1.5_5.batches_mis_0.csv
    Untracked:  data/ici.vs.eoi_age.of.onset-no.covariates_blup_sub_chr-3_peak.marker-UNC5082757_lod.drop-1.5_5.batches_mis_0.csv
    Untracked:  data/ici.vs.eoi_age.of.onset-no.covariates_blup_sub_chr-3_peak.marker-UNC5083168_lod.drop-1.5_5.batches_0.csv
    Untracked:  data/ici.vs.eoi_age.of.onset-no.covariates_blup_sub_chr-3_peak.marker-UNC5083168_lod.drop-1.5_5.batches_52.csv
    Untracked:  data/ici.vs.eoi_age.of.onset-no.covariates_blup_sub_chr-3_peak.marker-UNC5083168_lod.drop-1.5_5.batches_mis_0.csv
    Untracked:  data/ici.vs.eoi_age.of.onset-no.covariates_blup_sub_chr-3_peak.marker-UNC5173293_lod.drop-1.5_5.batches_mis_0.csv
    Untracked:  data/ici.vs.eoi_age.of.onset-no.covariates_blup_sub_chr-3_peak.marker-UNC5192582_lod.drop-1.5_5.batches_0.csv
    Untracked:  data/ici.vs.eoi_age.of.onset-no.covariates_blup_sub_chr-3_peak.marker-UNC5192582_lod.drop-1.5_5.batches_52.csv
    Untracked:  data/ici.vs.eoi_age.of.onset-no.covariates_blup_sub_chr-3_peak.marker-UNC5192582_lod.drop-1.5_5.batches_mis_0.csv
    Untracked:  data/ici.vs.eoi_age.of.onset-no.covariates_blup_sub_chr-3_peak.marker-UNC5193970_lod.drop-1.5_5.batches_mis_0.csv
    Untracked:  data/ici.vs.eoi_age.of.onset-no.covariates_blup_sub_chr-3_peak.marker-UNC5213888_lod.drop-1.5_snpsqc_5.batches_mis_0.csv
    Untracked:  data/ici.vs.eoi_age.of.onset-no.covariates_blup_sub_chr-3_peak.marker-UNC5213888_lod.drop-1.5_snpsqc_5.batches_mis_52.csv
    Untracked:  data/ici.vs.eoi_age.of.onset-no.covariates_blup_sub_chr-3_peak.marker-UNC5257268_lod.drop-1.5_5.batches_mis_0.csv
    Untracked:  data/ici.vs.eoi_age.of.onset-no.covariates_blup_sub_chr-3_peak.marker-UNC5262472_lod.drop-1.5_5.batches_0.csv
    Untracked:  data/ici.vs.eoi_age.of.onset-no.covariates_blup_sub_chr-3_peak.marker-UNC5262472_lod.drop-1.5_5.batches_52.csv
    Untracked:  data/ici.vs.eoi_age.of.onset-no.covariates_blup_sub_chr-3_peak.marker-UNC5262472_lod.drop-1.5_5.batches_mis_0.csv
    Untracked:  data/ici.vs.eoi_age.of.onset-no.covariates_blup_sub_chr-3_peak.marker-UNC5264696_lod.drop-1.5_5.batches_mis_0.csv
    Untracked:  data/ici.vs.eoi_age.of.onset-no.covariates_blup_sub_chr-3_peak.marker-UNC5267813_lod.drop-1.5_5.batches_mis_0.csv
    Untracked:  data/ici.vs.eoi_age.of.onset-no.covariates_blup_sub_chr-3_peak.marker-UNC5270225_lod.drop-1.5_5.batches_mis_0.csv
    Untracked:  data/ici.vs.eoi_age.of.onset-no.covariates_blup_sub_chr-3_peak.marker-UNC5277061_lod.drop-1.5_5.batches_mis_0.csv
    Untracked:  data/ici.vs.eoi_age.of.onset-no.covariates_blup_sub_chr-3_peak.marker-UNC5278229_lod.drop-1.5_5.batches_0.csv
    Untracked:  data/ici.vs.eoi_age.of.onset-no.covariates_blup_sub_chr-3_peak.marker-UNC5278229_lod.drop-1.5_5.batches_mis_0.csv
    Untracked:  data/ici.vs.eoi_age.of.onset-no.covariates_blup_sub_chr-3_peak.marker-UNC5282259_lod.drop-1.5_5.batches_mis_0.csv
    Untracked:  data/ici.vs.eoi_age.of.onset-no.covariates_blup_sub_chr-3_peak.marker-UNC5287977_lod.drop-1.5_5.batches_mis_0.csv
    Untracked:  data/ici.vs.eoi_age.of.onset-no.covariates_blup_sub_chr-3_peak.marker-UNC5301460_lod.drop-1.5_5.batches_mis_0.csv
    Untracked:  data/ici.vs.eoi_age.of.onset-no.covariates_blup_sub_chr-3_peak.marker-UNC5309136_lod.drop-1.5_5.batches_mis_0.csv
    Untracked:  data/ici.vs.eoi_age.of.onset-no.covariates_blup_sub_chr-3_peak.marker-UNC5321559_lod.drop-1.5_5.batches_mis_0.csv
    Untracked:  data/ici.vs.eoi_age.of.onset-no.covariates_blup_sub_chr-3_peak.marker-UNC5322455_lod.drop-1.5_5.batches_mis_0.csv
    Untracked:  data/ici.vs.eoi_age.of.onset-no.covariates_blup_sub_chr-3_peak.marker-UNC5329171_lod.drop-1.5_5.batches_mis_0.csv
    Untracked:  data/ici.vs.eoi_age.of.onset-no.covariates_blup_sub_chr-3_peak.marker-UNC5360055_lod.drop-1.5_5.batches_52.csv
    Untracked:  data/ici.vs.eoi_age.of.onset-no.covariates_blup_sub_chr-3_peak.marker-UNC5454839_lod.drop-1.5_5.batches_0.csv
    Untracked:  data/ici.vs.eoi_age.of.onset-no.covariates_blup_sub_chr-3_peak.marker-UNC5454839_lod.drop-1.5_5.batches_52.csv
    Untracked:  data/ici.vs.eoi_age.of.onset-no.covariates_blup_sub_chr-3_peak.marker-UNC5454839_lod.drop-1.5_5.batches_mis_0.csv
    Untracked:  data/ici.vs.eoi_age.of.onset-no.covariates_blup_sub_chr-3_peak.marker-UNC5645844_lod.drop-1.5_5.batches_0.csv
    Untracked:  data/ici.vs.eoi_age.of.onset-no.covariates_blup_sub_chr-3_peak.marker-UNC5645844_lod.drop-1.5_5.batches_52.csv
    Untracked:  data/ici.vs.eoi_age.of.onset-no.covariates_blup_sub_chr-3_peak.marker-UNC5645844_lod.drop-1.5_5.batches_mis_0.csv
    Untracked:  data/ici.vs.eoi_age.of.onset-no.covariates_blup_sub_chr-3_peak.marker-UNC5778977_lod.drop-1.5_5.batches_0.csv
    Untracked:  data/ici.vs.eoi_age.of.onset-no.covariates_blup_sub_chr-3_peak.marker-UNC5808246_lod.drop-1.5_5.batches_52.csv
    Untracked:  data/ici.vs.eoi_age.of.onset-no.covariates_blup_sub_chr-3_peak.marker-UNC5916997_lod.drop-1.5_5.batches_52.csv
    Untracked:  data/ici.vs.eoi_age.of.onset-no.covariates_blup_sub_chr-3_peak.marker-UNCHS008182_lod.drop-1.5_5.batches_0.csv
    Untracked:  data/ici.vs.eoi_age.of.onset-no.covariates_blup_sub_chr-3_peak.marker-UNCHS008182_lod.drop-1.5_5.batches_52.csv
    Untracked:  data/ici.vs.eoi_age.of.onset-no.covariates_blup_sub_chr-3_peak.marker-UNCHS008182_lod.drop-1.5_5.batches_mis_0.csv
    Untracked:  data/ici.vs.eoi_age.of.onset-no.covariates_blup_sub_chr-3_peak.marker-UNCHS008193_lod.drop-1.5_5.batches_mis.csv
    Untracked:  data/ici.vs.eoi_age.of.onset-no.covariates_blup_sub_chr-3_peak.marker-UNCHS008193_lod.drop-1.5_5.batches_mis_0.csv
    Untracked:  data/ici.vs.eoi_age.of.onset-no.covariates_blup_sub_chr-3_peak.marker-UNCHS008209_lod.drop-1.5_5.batches_mis.csv
    Untracked:  data/ici.vs.eoi_age.of.onset-no.covariates_blup_sub_chr-3_peak.marker-UNCHS008209_lod.drop-1.5_5.batches_mis_0.csv
    Untracked:  data/ici.vs.eoi_age.of.onset-no.covariates_blup_sub_chr-3_peak.marker-UNCHS008215_lod.drop-1.5_5.batches_mis.csv
    Untracked:  data/ici.vs.eoi_age.of.onset-no.covariates_blup_sub_chr-3_peak.marker-UNCHS008215_lod.drop-1.5_5.batches_mis_0.csv
    Untracked:  data/ici.vs.eoi_age.of.onset-no.covariates_blup_sub_chr-3_peak.marker-UNCHS008219_lod.drop-1.5_5.batches_mis.csv
    Untracked:  data/ici.vs.eoi_age.of.onset-no.covariates_blup_sub_chr-3_peak.marker-UNCHS008219_lod.drop-1.5_5.batches_mis_0.csv
    Untracked:  data/ici.vs.eoi_age.of.onset-no.covariates_blup_sub_chr-3_peak.marker-UNCHS008229_lod.drop-1.5_5.batches_mis.csv
    Untracked:  data/ici.vs.eoi_age.of.onset-no.covariates_blup_sub_chr-3_peak.marker-UNCHS008229_lod.drop-1.5_5.batches_mis_0.csv
    Untracked:  data/ici.vs.eoi_age.of.onset-no.covariates_blup_sub_chr-3_peak.marker-UNCHS008234_lod.drop-1.5_5.batches_mis.csv
    Untracked:  data/ici.vs.eoi_age.of.onset-no.covariates_blup_sub_chr-3_peak.marker-UNCHS008234_lod.drop-1.5_5.batches_mis_0.csv
    Untracked:  data/ici.vs.eoi_age.of.onset-no.covariates_blup_sub_chr-3_peak.marker-UNCHS008242_lod.drop-1.5_5.batches_52.csv
    Untracked:  data/ici.vs.eoi_age.of.onset-no.covariates_blup_sub_chr-3_peak.marker-UNCHS008260_lod.drop-1.5_5.batches_mis.csv
    Untracked:  data/ici.vs.eoi_age.of.onset-no.covariates_blup_sub_chr-3_peak.marker-UNCHS008260_lod.drop-1.5_5.batches_mis_0.csv
    Untracked:  data/ici.vs.eoi_age.of.onset-no.covariates_blup_sub_chr-3_peak.marker-UNCHS008271_lod.drop-1.5_5.batches_mis.csv
    Untracked:  data/ici.vs.eoi_age.of.onset-no.covariates_blup_sub_chr-3_peak.marker-UNCHS008271_lod.drop-1.5_5.batches_mis_0.csv
    Untracked:  data/ici.vs.eoi_age.of.onset-no.covariates_blup_sub_chr-3_peak.marker-UNCHS008281_lod.drop-1.5_5.batches_mis.csv
    Untracked:  data/ici.vs.eoi_age.of.onset-no.covariates_blup_sub_chr-3_peak.marker-UNCHS008281_lod.drop-1.5_5.batches_mis_0.csv
    Untracked:  data/ici.vs.eoi_age.of.onset-no.covariates_blup_sub_chr-3_peak.marker-UNCHS008286_lod.drop-1.5_5.batches_0.csv
    Untracked:  data/ici.vs.eoi_age.of.onset-no.covariates_blup_sub_chr-3_peak.marker-UNCHS008286_lod.drop-1.5_5.batches_52.csv
    Untracked:  data/ici.vs.eoi_age.of.onset-no.covariates_blup_sub_chr-3_peak.marker-UNCHS008286_lod.drop-1.5_5.batches_mis_0.csv
    Untracked:  data/ici.vs.eoi_age.of.onset-no.covariates_blup_sub_chr-3_peak.marker-UNCHS008338_lod.drop-1.5_5.batches_mis.csv
    Untracked:  data/ici.vs.eoi_age.of.onset-no.covariates_blup_sub_chr-3_peak.marker-UNCHS008338_lod.drop-1.5_5.batches_mis_0.csv
    Untracked:  data/ici.vs.eoi_age.of.onset-no.covariates_blup_sub_chr-3_peak.marker-UNCHS008363_lod.drop-1.5_5.batches_mis.csv
    Untracked:  data/ici.vs.eoi_age.of.onset-no.covariates_blup_sub_chr-3_peak.marker-UNCHS008363_lod.drop-1.5_5.batches_mis_0.csv
    Untracked:  data/ici.vs.eoi_age.of.onset-no.covariates_blup_sub_chr-3_peak.marker-UNCHS008379_lod.drop-1.5_5.batches_mis.csv
    Untracked:  data/ici.vs.eoi_age.of.onset-no.covariates_blup_sub_chr-3_peak.marker-UNCHS008379_lod.drop-1.5_5.batches_mis_0.csv
    Untracked:  data/ici.vs.eoi_age.of.onset-no.covariates_blup_sub_chr-3_peak.marker-UNCHS008409_lod.drop-1.5_5.batches_mis_0.csv
    Untracked:  data/ici.vs.eoi_age.of.onset-no.covariates_blup_sub_chr-3_peak.marker-UNCHS008432_lod.drop-1.5_5.batches_mis_0.csv
    Untracked:  data/ici.vs.eoi_age.of.onset-no.covariates_blup_sub_chr-3_peak.marker-UNCHS008487_lod.drop-1.5_snpsqc_5.batches_0.csv
    Untracked:  data/ici.vs.eoi_age.of.onset-no.covariates_blup_sub_chr-3_peak.marker-UNCHS008487_lod.drop-1.5_snpsqc_5.batches_52.csv
    Untracked:  data/ici.vs.eoi_age.of.onset-no.covariates_blup_sub_chr-3_peak.marker-UNCHS008487_lod.drop-1.5_snpsqc_dis_no-x_updated_5.batches_0.csv
    Untracked:  data/ici.vs.eoi_age.of.onset-no.covariates_blup_sub_chr-3_peak.marker-UNCHS008487_lod.drop-1.5_snpsqc_dis_no-x_updated_5.batches_52.csv
    Untracked:  data/ici.vs.eoi_age.of.onset-no.covariates_blup_sub_chr-3_peak.marker-UNCHS008487_lod.drop-1.5_snpsqc_dis_no-x_updated_5.batches_mis_0.csv
    Untracked:  data/ici.vs.eoi_age.of.onset-no.covariates_blup_sub_chr-3_peak.marker-UNCHS008487_lod.drop-1.5_snpsqc_dis_no-x_updated_5.batches_mis_52.csv
    Untracked:  data/ici.vs.eoi_age.of.onset-no.covariates_blup_sub_chr-3_peak.marker-UNCHS008511_lod.drop-1.5_snpsqc_5.batches_mis_0.csv
    Untracked:  data/ici.vs.eoi_age.of.onset-no.covariates_blup_sub_chr-3_peak.marker-UNCHS008511_lod.drop-1.5_snpsqc_5.batches_mis_52.csv
    Untracked:  data/ici.vs.eoi_age.of.onset-no.covariates_blup_sub_chr-3_peak.marker-UNCHS008609_lod.drop-1.5_5.batches_mis_0.csv
    Untracked:  data/ici.vs.eoi_age.of.onset-no.covariates_blup_sub_chr-3_peak.marker-UNCHS008613_lod.drop-1.5_5.batches_0.csv
    Untracked:  data/ici.vs.eoi_age.of.onset-no.covariates_blup_sub_chr-3_peak.marker-UNCHS008613_lod.drop-1.5_5.batches_52.csv
    Untracked:  data/ici.vs.eoi_age.of.onset-no.covariates_blup_sub_chr-3_peak.marker-UNCHS008614_lod.drop-1.5_5.batches_mis_0.csv
    Untracked:  data/ici.vs.eoi_age.of.onset-no.covariates_blup_sub_chr-3_peak.marker-UNCHS008627_lod.drop-1.5_5.batches_mis_0.csv
    Untracked:  data/ici.vs.eoi_age.of.onset-no.covariates_blup_sub_chr-3_peak.marker-UNCHS008659_lod.drop-1.5_5.batches_52.csv
    Untracked:  data/ici.vs.eoi_age.of.onset-no.covariates_blup_sub_chr-3_peak.marker-UNCHS008725_lod.drop-1.5_5.batches_mis_0.csv
    Untracked:  data/ici.vs.eoi_age.of.onset-no.covariates_blup_sub_chr-3_peak.marker-UNCHS008815_lod.drop-1.5_5.batches_0.csv
    Untracked:  data/ici.vs.eoi_age.of.onset-no.covariates_blup_sub_chr-3_peak.marker-UNCHS008815_lod.drop-1.5_5.batches_52.csv
    Untracked:  data/ici.vs.eoi_age.of.onset-no.covariates_blup_sub_chr-3_peak.marker-UNCHS008815_lod.drop-1.5_5.batches_mis_0.csv
    Untracked:  data/ici.vs.eoi_age.of.onset-no.covariates_blup_sub_chr-3_peak.marker-UNCHS009066_lod.drop-1.5_5.batches_52.csv
    Untracked:  data/ici.vs.eoi_age.of.onset-no.covariates_blup_sub_chr-3_peak.marker-UNCHS009067_lod.drop-1.5_5.batches_0.csv
    Untracked:  data/ici.vs.eoi_age.of.onset-no.covariates_blup_sub_chr-3_peak.marker-UNCHS009068_lod.drop-1.5_5.batches_mis_0.csv
    Untracked:  data/ici.vs.eoi_age.of.onset-no.covariates_blup_sub_chr-3_peak.marker-UNCHS009112_lod.drop-1.5_5.batches_52.csv
    Untracked:  data/ici.vs.eoi_age.of.onset-no.covariates_blup_sub_chr-3_peak.marker-UNCHS009224_lod.drop-1.5_5.batches_0.csv
    Untracked:  data/ici.vs.eoi_age.of.onset-no.covariates_blup_sub_chr-3_peak.marker-UNCHS009224_lod.drop-1.5_5.batches_52.csv
    Untracked:  data/ici.vs.eoi_age.of.onset-no.covariates_blup_sub_chr-3_peak.marker-UNCHS009383_lod.drop-1.5_5.batches_0.csv
    Untracked:  data/ici.vs.eoi_age.of.onset-no.covariates_blup_sub_chr-3_peak.marker-UNCHS009383_lod.drop-1.5_5.batches_52.csv
    Untracked:  data/ici.vs.eoi_age.of.onset-no.covariates_blup_sub_chr-3_peak.marker-UNCHS009693_lod.drop-1.5_5.batches_52.csv
    Untracked:  data/ici.vs.eoi_age.of.onset-no.covariates_blup_sub_chr-3_peak.marker-UNCHS009821_lod.drop-1.5_5.batches_52.csv
    Untracked:  data/ici.vs.eoi_age.of.onset-no.covariates_blup_sub_chr-3_peak.marker-UNCJPD001170_lod.drop-1.5_5.batches_mis.csv
    Untracked:  data/ici.vs.eoi_age.of.onset-no.covariates_blup_sub_chr-3_peak.marker-UNCJPD001170_lod.drop-1.5_5.batches_mis_0.csv
    Untracked:  data/ici.vs.eoi_age.of.onset-no.covariates_blup_sub_chr-3_peak.marker-UNCJPD001198_lod.drop-1.5_5.batches_mis.csv
    Untracked:  data/ici.vs.eoi_age.of.onset-no.covariates_blup_sub_chr-3_peak.marker-UNCJPD001198_lod.drop-1.5_5.batches_mis_0.csv
    Untracked:  data/ici.vs.eoi_age.of.onset-no.covariates_blup_sub_chr-3_peak.marker-UNCJPD001276_lod.drop-1.5_5.batches_mis_0.csv
    Untracked:  data/ici.vs.eoi_age.of.onset-no.covariates_blup_sub_chr-3_peak.marker-sanger2496q_lod.drop-1.5_5.batches_mis.csv
    Untracked:  data/ici.vs.eoi_age.of.onset-no.covariates_blup_sub_chr-3_peak.marker-sanger2496q_lod.drop-1.5_5.batches_mis_0.csv
    Untracked:  data/ici.vs.eoi_age.of.onset-no.covariates_blup_sub_chr-4_peak.marker-UNC8250659_lod.drop-1.5_snpsqc_dis_no-x_updated_5.batches_52.csv
    Untracked:  data/ici.vs.eoi_age.of.onset-no.covariates_blup_sub_chr-4_peak.marker-UNC8439633_lod.drop-1.5_snpsqc_5.batches_52.csv
    Untracked:  data/ici.vs.eoi_age.of.onset-no.covariates_blup_sub_chr-4_peak.marker-UNC8439633_lod.drop-1.5_snpsqc_5.batches_mis_52.csv
    Untracked:  data/ici.vs.eoi_age.of.onset-no.covariates_blup_sub_chr-4_peak.marker-UNCHS012955_lod.drop-1.5_snpsqc_dis_no-x_updated_5.batches_mis_52.csv
    Untracked:  data/ici.vs.eoi_age.of.onset-no.covariates_blup_sub_chr-6_peak.marker-UNC11108920_lod.drop-1.5_snpsqc_5.batches.csv
    Untracked:  data/ici.vs.eoi_age.of.onset-no.covariates_blup_sub_chr-6_peak.marker-UNC11108920_lod.drop-1.5_snpsqc_5.batches_mis.csv
    Untracked:  data/ici.vs.eoi_age.of.onset-no.covariates_blup_sub_chr-7_peak.marker-UNCHS020066_lod.drop-1.5_snpsqc_5.batches_mis_0.csv
    Untracked:  data/ici.vs.eoi_age.of.onset-no.covariates_blup_sub_chr-7_peak.marker-UNCHS020066_lod.drop-1.5_snpsqc_5.batches_mis_52.csv
    Untracked:  data/ici.vs.eoi_age.of.onset-no.covariates_blup_sub_chr-7_peak.marker-UNCHS020066_lod.drop-1.5_snpsqc_dis_no-x_updated_5.batches_mis_0.csv
    Untracked:  data/ici.vs.eoi_age.of.onset-no.covariates_blup_sub_chr-7_peak.marker-UNCHS020066_lod.drop-1.5_snpsqc_dis_no-x_updated_5.batches_mis_52.csv
    Untracked:  data/ici.vs.eoi_age.of.onset-no.covariates_blup_sub_chr-8_peak.marker-UNC15524531_lod.drop-1.5_snpsqc_5.batches_mis_0.csv
    Untracked:  data/ici.vs.eoi_age.of.onset-no.covariates_blup_sub_chr-8_peak.marker-UNC15524531_lod.drop-1.5_snpsqc_5.batches_mis_52.csv
    Untracked:  data/ici.vs.eoi_age.of.onset-no.covariates_blup_sub_chr-8_peak.marker-UNC15524531_lod.drop-1.5_snpsqc_dis_no-x_updated_5.batches_mis_0.csv
    Untracked:  data/ici.vs.eoi_age.of.onset-no.covariates_blup_sub_chr-8_peak.marker-UNC15524531_lod.drop-1.5_snpsqc_dis_no-x_updated_5.batches_mis_52.csv
    Untracked:  data/ici.vs.eoi_age.of.onset-no.covariates_blup_sub_chr-9_peak.marker-UNC17203597_lod.drop-1.5_snpsqc_5.batches_mis_0.csv
    Untracked:  data/ici.vs.eoi_age.of.onset-no.covariates_blup_sub_chr-9_peak.marker-UNC17203597_lod.drop-1.5_snpsqc_5.batches_mis_52.csv
    Untracked:  data/ici.vs.eoi_age.of.onset-no.covariates_blup_sub_chr-X_peak.marker-UNCHS048314_lod.drop-1.5_snpsqc_5.batches_mis_0.csv
    Untracked:  data/ici.vs.eoi_age.of.onset-no.covariates_blup_sub_chr-X_peak.marker-UNCHS048314_lod.drop-1.5_snpsqc_5.batches_mis_52.csv
    Untracked:  data/ici.vs.eoi_age.of.onset-no.covariates_blup_sub_chr-X_peak.marker-UNCHS048314_lod.drop-1.5_snpsqc_dis_no-x_updated_5.batches_mis_0.csv
    Untracked:  data/ici.vs.eoi_age.of.onset-no.covariates_blup_sub_chr-X_peak.marker-UNCHS048314_lod.drop-1.5_snpsqc_dis_no-x_updated_5.batches_mis_52.csv
    Untracked:  data/ici.vs.eoi_age.of.onset-no.covariates_genes_chr-10_peak.marker-JAX00020646_lod.drop-1.5_5.batches_0.csv
    Untracked:  data/ici.vs.eoi_age.of.onset-no.covariates_genes_chr-10_peak.marker-JAX00292499_lod.drop-1.5_5.batches_52.csv
    Untracked:  data/ici.vs.eoi_age.of.onset-no.covariates_genes_chr-10_peak.marker-JAX00292927_lod.drop-1.5_5.batches_0.csv
    Untracked:  data/ici.vs.eoi_age.of.onset-no.covariates_genes_chr-10_peak.marker-JAX00292927_lod.drop-1.5_5.batches_52.csv
    Untracked:  data/ici.vs.eoi_age.of.onset-no.covariates_genes_chr-10_peak.marker-JAX00294019_lod.drop-1.5_snpsqc_dis_no-x_updated_5.batches_0.csv
    Untracked:  data/ici.vs.eoi_age.of.onset-no.covariates_genes_chr-10_peak.marker-UNC18216614_lod.drop-1.5_5.batches_52.csv
    Untracked:  data/ici.vs.eoi_age.of.onset-no.covariates_genes_chr-10_peak.marker-UNC18240977_lod.drop-1.5_snpsqc_dis_no-x_updated_5.batches_52.csv
    Untracked:  data/ici.vs.eoi_age.of.onset-no.covariates_genes_chr-10_peak.marker-UNC18311938_lod.drop-1.5_5.batches_0.csv
    Untracked:  data/ici.vs.eoi_age.of.onset-no.covariates_genes_chr-10_peak.marker-UNC18311938_lod.drop-1.5_5.batches_52.csv
    Untracked:  data/ici.vs.eoi_age.of.onset-no.covariates_genes_chr-10_peak.marker-UNC18311938_lod.drop-1.5_snpsqc_5.batches_52.csv
    Untracked:  data/ici.vs.eoi_age.of.onset-no.covariates_genes_chr-10_peak.marker-UNC18311938_lod.drop-1.5_snpsqc_5.batches_mis_52.csv
    Untracked:  data/ici.vs.eoi_age.of.onset-no.covariates_genes_chr-10_peak.marker-UNC18343181_lod.drop-1.5_5.batches_0.csv
    Untracked:  data/ici.vs.eoi_age.of.onset-no.covariates_genes_chr-10_peak.marker-UNC18363544_lod.drop-1.5_5.batches_0.csv
    Untracked:  data/ici.vs.eoi_age.of.onset-no.covariates_genes_chr-10_peak.marker-UNC18363544_lod.drop-1.5_5.batches_52.csv
    Untracked:  data/ici.vs.eoi_age.of.onset-no.covariates_genes_chr-10_peak.marker-UNC18363544_lod.drop-1.5_snpsqc_5.batches_mis_0.csv
    Untracked:  data/ici.vs.eoi_age.of.onset-no.covariates_genes_chr-10_peak.marker-UNC18376338_lod.drop-1.5_snpsqc_5.batches_0.csv
    Untracked:  data/ici.vs.eoi_age.of.onset-no.covariates_genes_chr-10_peak.marker-UNCHS028236_lod.drop-1.5_snpsqc_5.batches_mis_0.csv
    Untracked:  data/ici.vs.eoi_age.of.onset-no.covariates_genes_chr-10_peak.marker-UNCHS028236_lod.drop-1.5_snpsqc_5.batches_mis_52.csv
    Untracked:  data/ici.vs.eoi_age.of.onset-no.covariates_genes_chr-10_peak.marker-UNCHS028536_lod.drop-1.5_snpsqc_dis_no-x_updated_5.batches_mis_0.csv
    Untracked:  data/ici.vs.eoi_age.of.onset-no.covariates_genes_chr-10_peak.marker-UNCHS028536_lod.drop-1.5_snpsqc_dis_no-x_updated_5.batches_mis_52.csv
    Untracked:  data/ici.vs.eoi_age.of.onset-no.covariates_genes_chr-11_peak.marker-UNC19970181_lod.drop-1.5_snpsqc_5.batches_mis_0.csv
    Untracked:  data/ici.vs.eoi_age.of.onset-no.covariates_genes_chr-11_peak.marker-UNC19970181_lod.drop-1.5_snpsqc_5.batches_mis_52.csv
    Untracked:  data/ici.vs.eoi_age.of.onset-no.covariates_genes_chr-11_peak.marker-UNC20090524_lod.drop-1.5_snpsqc_5.batches_52.csv
    Untracked:  data/ici.vs.eoi_age.of.onset-no.covariates_genes_chr-12_peak.marker-ICR499_lod.drop-1.5_snpsqc_5.batches_mis_0.csv
    Untracked:  data/ici.vs.eoi_age.of.onset-no.covariates_genes_chr-12_peak.marker-ICR499_lod.drop-1.5_snpsqc_5.batches_mis_52.csv
    Untracked:  data/ici.vs.eoi_age.of.onset-no.covariates_genes_chr-12_peak.marker-ICR499_lod.drop-1.5_snpsqc_dis_no-x_updated_5.batches_mis_0.csv
    Untracked:  data/ici.vs.eoi_age.of.onset-no.covariates_genes_chr-13_peak.marker-UNCHS036773_lod.drop-1.5_snpsqc_5.batches_mis_0.csv
    Untracked:  data/ici.vs.eoi_age.of.onset-no.covariates_genes_chr-13_peak.marker-UNCHS036773_lod.drop-1.5_snpsqc_5.batches_mis_52.csv
    Untracked:  data/ici.vs.eoi_age.of.onset-no.covariates_genes_chr-14_peak.marker-UNC24056202_lod.drop-1.5_snpsqc_5.batches_mis_0.csv
    Untracked:  data/ici.vs.eoi_age.of.onset-no.covariates_genes_chr-14_peak.marker-UNC24056202_lod.drop-1.5_snpsqc_5.batches_mis_52.csv
    Untracked:  data/ici.vs.eoi_age.of.onset-no.covariates_genes_chr-15_peak.marker-UNC26070435_lod.drop-1.5_snpsqc_dis_no-x_updated_5.batches_mis_0.csv
    Untracked:  data/ici.vs.eoi_age.of.onset-no.covariates_genes_chr-15_peak.marker-UNC26070435_lod.drop-1.5_snpsqc_dis_no-x_updated_5.batches_mis_52.csv
    Untracked:  data/ici.vs.eoi_age.of.onset-no.covariates_genes_chr-17_peak.marker-UNCHS044241_lod.drop-1.5_snpsqc_5.batches_mis_0.csv
    Untracked:  data/ici.vs.eoi_age.of.onset-no.covariates_genes_chr-17_peak.marker-UNCHS044241_lod.drop-1.5_snpsqc_5.batches_mis_52.csv
    Untracked:  data/ici.vs.eoi_age.of.onset-no.covariates_genes_chr-17_peak.marker-UNCJPD006614_lod.drop-1.5_snpsqc_dis_no-x_updated_5.batches_mis_0.csv
    Untracked:  data/ici.vs.eoi_age.of.onset-no.covariates_genes_chr-17_peak.marker-UNCrs47191360_lod.drop-1.5_snpsqc_dis_no-x_updated_5.batches_mis_52.csv
    Untracked:  data/ici.vs.eoi_age.of.onset-no.covariates_genes_chr-18_peak.marker-UNCHS045343_lod.drop-1.5_snpsqc_5.batches_mis_0.csv
    Untracked:  data/ici.vs.eoi_age.of.onset-no.covariates_genes_chr-18_peak.marker-UNCHS045343_lod.drop-1.5_snpsqc_5.batches_mis_52.csv
    Untracked:  data/ici.vs.eoi_age.of.onset-no.covariates_genes_chr-2_peak.marker-UNC4609527_lod.drop-1.5_snpsqc_dis_no-x_updated_5.batches_mis_0.csv
    Untracked:  data/ici.vs.eoi_age.of.onset-no.covariates_genes_chr-2_peak.marker-UNC4609527_lod.drop-1.5_snpsqc_dis_no-x_updated_5.batches_mis_52.csv
    Untracked:  data/ici.vs.eoi_age.of.onset-no.covariates_genes_chr-2_peak.marker-UNCHS008007_lod.drop-1.5_5.batches_mis.csv
    Untracked:  data/ici.vs.eoi_age.of.onset-no.covariates_genes_chr-2_peak.marker-UNCHS008007_lod.drop-1.5_5.batches_mis_0.csv
    Untracked:  data/ici.vs.eoi_age.of.onset-no.covariates_genes_chr-3_peak.marker-ICR1338_lod.drop-1.5_5.batches_0.csv
    Untracked:  data/ici.vs.eoi_age.of.onset-no.covariates_genes_chr-3_peak.marker-ICR1338_lod.drop-1.5_5.batches_52.csv
    Untracked:  data/ici.vs.eoi_age.of.onset-no.covariates_genes_chr-3_peak.marker-ICR1338_lod.drop-1.5_5.batches_mis_0.csv
    Untracked:  data/ici.vs.eoi_age.of.onset-no.covariates_genes_chr-3_peak.marker-ICR5263_lod.drop-1.5_5.batches_mis.csv
    Untracked:  data/ici.vs.eoi_age.of.onset-no.covariates_genes_chr-3_peak.marker-ICR5263_lod.drop-1.5_5.batches_mis_0.csv
    Untracked:  data/ici.vs.eoi_age.of.onset-no.covariates_genes_chr-3_peak.marker-JAX00105649_lod.drop-1.5_5.batches_mis.csv
    Untracked:  data/ici.vs.eoi_age.of.onset-no.covariates_genes_chr-3_peak.marker-JAX00105649_lod.drop-1.5_5.batches_mis_0.csv
    Untracked:  data/ici.vs.eoi_age.of.onset-no.covariates_genes_chr-3_peak.marker-JAX00106210_lod.drop-1.5_5.batches_mis_0.csv
    Untracked:  data/ici.vs.eoi_age.of.onset-no.covariates_genes_chr-3_peak.marker-JAX00106957_lod.drop-1.5_5.batches_0.csv
    Untracked:  data/ici.vs.eoi_age.of.onset-no.covariates_genes_chr-3_peak.marker-JAX00106957_lod.drop-1.5_5.batches_52.csv
    Untracked:  data/ici.vs.eoi_age.of.onset-no.covariates_genes_chr-3_peak.marker-JAX00106957_lod.drop-1.5_5.batches_mis_0.csv
    Untracked:  data/ici.vs.eoi_age.of.onset-no.covariates_genes_chr-3_peak.marker-JAX00107193_lod.drop-1.5_5.batches_mis_0.csv
    Untracked:  data/ici.vs.eoi_age.of.onset-no.covariates_genes_chr-3_peak.marker-JAX00220631_lod.drop-1.5_5.batches_0.csv
    Untracked:  data/ici.vs.eoi_age.of.onset-no.covariates_genes_chr-3_peak.marker-JAX00220631_lod.drop-1.5_5.batches_mis_0.csv
    Untracked:  data/ici.vs.eoi_age.of.onset-no.covariates_genes_chr-3_peak.marker-JAX00514662_lod.drop-1.5_5.batches_0.csv
    Untracked:  data/ici.vs.eoi_age.of.onset-no.covariates_genes_chr-3_peak.marker-JAX00514662_lod.drop-1.5_5.batches_52.csv
    Untracked:  data/ici.vs.eoi_age.of.onset-no.covariates_genes_chr-3_peak.marker-JAX00516812_lod.drop-1.5_5.batches_mis.csv
    Untracked:  data/ici.vs.eoi_age.of.onset-no.covariates_genes_chr-3_peak.marker-JAX00516812_lod.drop-1.5_5.batches_mis_0.csv
    Untracked:  data/ici.vs.eoi_age.of.onset-no.covariates_genes_chr-3_peak.marker-JAX00517674_lod.drop-1.5_5.batches_mis.csv
    Untracked:  data/ici.vs.eoi_age.of.onset-no.covariates_genes_chr-3_peak.marker-JAX00517674_lod.drop-1.5_5.batches_mis_0.csv
    Untracked:  data/ici.vs.eoi_age.of.onset-no.covariates_genes_chr-3_peak.marker-JAX00518911r_lod.drop-1.5_5.batches_mis.csv
    Untracked:  data/ici.vs.eoi_age.of.onset-no.covariates_genes_chr-3_peak.marker-JAX00518911r_lod.drop-1.5_5.batches_mis_0.csv
    Untracked:  data/ici.vs.eoi_age.of.onset-no.covariates_genes_chr-3_peak.marker-JAX00522922_lod.drop-1.5_5.batches_mis_0.csv
    Untracked:  data/ici.vs.eoi_age.of.onset-no.covariates_genes_chr-3_peak.marker-JAX00524205_lod.drop-1.5_5.batches_mis_0.csv
    Untracked:  data/ici.vs.eoi_age.of.onset-no.covariates_genes_chr-3_peak.marker-JAX00529707r_lod.drop-1.5_5.batches_0.csv
    Untracked:  data/ici.vs.eoi_age.of.onset-no.covariates_genes_chr-3_peak.marker-JAX00529707r_lod.drop-1.5_5.batches_52.csv
    Untracked:  data/ici.vs.eoi_age.of.onset-no.covariates_genes_chr-3_peak.marker-JAX00535764_lod.drop-1.5_5.batches_52.csv
    Untracked:  data/ici.vs.eoi_age.of.onset-no.covariates_genes_chr-3_peak.marker-UNC4809586_lod.drop-1.5_5.batches_mis.csv
    Untracked:  data/ici.vs.eoi_age.of.onset-no.covariates_genes_chr-3_peak.marker-UNC4809586_lod.drop-1.5_5.batches_mis_0.csv
    Untracked:  data/ici.vs.eoi_age.of.onset-no.covariates_genes_chr-3_peak.marker-UNC4813186_lod.drop-1.5_5.batches_0.csv
    Untracked:  data/ici.vs.eoi_age.of.onset-no.covariates_genes_chr-3_peak.marker-UNC4813186_lod.drop-1.5_5.batches_52.csv
    Untracked:  data/ici.vs.eoi_age.of.onset-no.covariates_genes_chr-3_peak.marker-UNC4813186_lod.drop-1.5_5.batches_mis.csv
    Untracked:  data/ici.vs.eoi_age.of.onset-no.covariates_genes_chr-3_peak.marker-UNC4813186_lod.drop-1.5_5.batches_mis_0.csv
    Untracked:  data/ici.vs.eoi_age.of.onset-no.covariates_genes_chr-3_peak.marker-UNC4820379_lod.drop-1.5_5.batches_mis.csv
    Untracked:  data/ici.vs.eoi_age.of.onset-no.covariates_genes_chr-3_peak.marker-UNC4820379_lod.drop-1.5_5.batches_mis_0.csv
    Untracked:  data/ici.vs.eoi_age.of.onset-no.covariates_genes_chr-3_peak.marker-UNC4836550_lod.drop-1.5_5.batches_0.csv
    Untracked:  data/ici.vs.eoi_age.of.onset-no.covariates_genes_chr-3_peak.marker-UNC4836550_lod.drop-1.5_5.batches_52.csv
    Untracked:  data/ici.vs.eoi_age.of.onset-no.covariates_genes_chr-3_peak.marker-UNC4836550_lod.drop-1.5_5.batches_mis_0.csv
    Untracked:  data/ici.vs.eoi_age.of.onset-no.covariates_genes_chr-3_peak.marker-UNC4875653_lod.drop-1.5_5.batches_mis.csv
    Untracked:  data/ici.vs.eoi_age.of.onset-no.covariates_genes_chr-3_peak.marker-UNC4875653_lod.drop-1.5_5.batches_mis_0.csv
    Untracked:  data/ici.vs.eoi_age.of.onset-no.covariates_genes_chr-3_peak.marker-UNC4901083_lod.drop-1.5_5.batches_mis.csv
    Untracked:  data/ici.vs.eoi_age.of.onset-no.covariates_genes_chr-3_peak.marker-UNC4901083_lod.drop-1.5_5.batches_mis_0.csv
    Untracked:  data/ici.vs.eoi_age.of.onset-no.covariates_genes_chr-3_peak.marker-UNC4909519_lod.drop-1.5_5.batches_mis.csv
    Untracked:  data/ici.vs.eoi_age.of.onset-no.covariates_genes_chr-3_peak.marker-UNC4909519_lod.drop-1.5_5.batches_mis_0.csv
    Untracked:  data/ici.vs.eoi_age.of.onset-no.covariates_genes_chr-3_peak.marker-UNC4912039_lod.drop-1.5_5.batches_0.csv
    Untracked:  data/ici.vs.eoi_age.of.onset-no.covariates_genes_chr-3_peak.marker-UNC4912039_lod.drop-1.5_5.batches_mis_0.csv
    Untracked:  data/ici.vs.eoi_age.of.onset-no.covariates_genes_chr-3_peak.marker-UNC4912288_lod.drop-1.5_5.batches_mis.csv
    Untracked:  data/ici.vs.eoi_age.of.onset-no.covariates_genes_chr-3_peak.marker-UNC4912288_lod.drop-1.5_5.batches_mis_0.csv
    Untracked:  data/ici.vs.eoi_age.of.onset-no.covariates_genes_chr-3_peak.marker-UNC4915446_lod.drop-1.5_5.batches_mis.csv
    Untracked:  data/ici.vs.eoi_age.of.onset-no.covariates_genes_chr-3_peak.marker-UNC4915446_lod.drop-1.5_5.batches_mis_0.csv
    Untracked:  data/ici.vs.eoi_age.of.onset-no.covariates_genes_chr-3_peak.marker-UNC4923398_lod.drop-1.5_5.batches_mis.csv
    Untracked:  data/ici.vs.eoi_age.of.onset-no.covariates_genes_chr-3_peak.marker-UNC4923398_lod.drop-1.5_5.batches_mis_0.csv
    Untracked:  data/ici.vs.eoi_age.of.onset-no.covariates_genes_chr-3_peak.marker-UNC4924377_lod.drop-1.5_5.batches_0.csv
    Untracked:  data/ici.vs.eoi_age.of.onset-no.covariates_genes_chr-3_peak.marker-UNC4924377_lod.drop-1.5_5.batches_52.csv
    Untracked:  data/ici.vs.eoi_age.of.onset-no.covariates_genes_chr-3_peak.marker-UNC4924377_lod.drop-1.5_5.batches_mis_0.csv
    Untracked:  data/ici.vs.eoi_age.of.onset-no.covariates_genes_chr-3_peak.marker-UNC4938565_lod.drop-1.5_5.batches_mis.csv
    Untracked:  data/ici.vs.eoi_age.of.onset-no.covariates_genes_chr-3_peak.marker-UNC4938565_lod.drop-1.5_5.batches_mis_0.csv
    Untracked:  data/ici.vs.eoi_age.of.onset-no.covariates_genes_chr-3_peak.marker-UNC4945186_lod.drop-1.5_5.batches_mis.csv
    Untracked:  data/ici.vs.eoi_age.of.onset-no.covariates_genes_chr-3_peak.marker-UNC4945186_lod.drop-1.5_5.batches_mis_0.csv
    Untracked:  data/ici.vs.eoi_age.of.onset-no.covariates_genes_chr-3_peak.marker-UNC4947492_lod.drop-1.5_5.batches_mis.csv
    Untracked:  data/ici.vs.eoi_age.of.onset-no.covariates_genes_chr-3_peak.marker-UNC4947492_lod.drop-1.5_5.batches_mis_0.csv
    Untracked:  data/ici.vs.eoi_age.of.onset-no.covariates_genes_chr-3_peak.marker-UNC4956618_lod.drop-1.5_5.batches_0.csv
    Untracked:  data/ici.vs.eoi_age.of.onset-no.covariates_genes_chr-3_peak.marker-UNC4956618_lod.drop-1.5_5.batches_mis_0.csv
    Untracked:  data/ici.vs.eoi_age.of.onset-no.covariates_genes_chr-3_peak.marker-UNC4957168_lod.drop-1.5_5.batches_52.csv
    Untracked:  data/ici.vs.eoi_age.of.onset-no.covariates_genes_chr-3_peak.marker-UNC4957859_lod.drop-1.5_5.batches_mis.csv
    Untracked:  data/ici.vs.eoi_age.of.onset-no.covariates_genes_chr-3_peak.marker-UNC4957859_lod.drop-1.5_5.batches_mis_0.csv
    Untracked:  data/ici.vs.eoi_age.of.onset-no.covariates_genes_chr-3_peak.marker-UNC4959266_lod.drop-1.5_5.batches_mis.csv
    Untracked:  data/ici.vs.eoi_age.of.onset-no.covariates_genes_chr-3_peak.marker-UNC4959266_lod.drop-1.5_5.batches_mis_0.csv
    Untracked:  data/ici.vs.eoi_age.of.onset-no.covariates_genes_chr-3_peak.marker-UNC4960612_lod.drop-1.5_5.batches_mis.csv
    Untracked:  data/ici.vs.eoi_age.of.onset-no.covariates_genes_chr-3_peak.marker-UNC4960612_lod.drop-1.5_5.batches_mis_0.csv
    Untracked:  data/ici.vs.eoi_age.of.onset-no.covariates_genes_chr-3_peak.marker-UNC4961542_lod.drop-1.5_5.batches_mis.csv
    Untracked:  data/ici.vs.eoi_age.of.onset-no.covariates_genes_chr-3_peak.marker-UNC4961542_lod.drop-1.5_5.batches_mis_0.csv
    Untracked:  data/ici.vs.eoi_age.of.onset-no.covariates_genes_chr-3_peak.marker-UNC4979914_lod.drop-1.5_5.batches_mis.csv
    Untracked:  data/ici.vs.eoi_age.of.onset-no.covariates_genes_chr-3_peak.marker-UNC4979914_lod.drop-1.5_5.batches_mis_0.csv
    Untracked:  data/ici.vs.eoi_age.of.onset-no.covariates_genes_chr-3_peak.marker-UNC4990111_lod.drop-1.5_5.batches_mis.csv
    Untracked:  data/ici.vs.eoi_age.of.onset-no.covariates_genes_chr-3_peak.marker-UNC4990111_lod.drop-1.5_5.batches_mis_0.csv
    Untracked:  data/ici.vs.eoi_age.of.onset-no.covariates_genes_chr-3_peak.marker-UNC5008656_lod.drop-1.5_5.batches_0.csv
    Untracked:  data/ici.vs.eoi_age.of.onset-no.covariates_genes_chr-3_peak.marker-UNC5008656_lod.drop-1.5_5.batches_52.csv
    Untracked:  data/ici.vs.eoi_age.of.onset-no.covariates_genes_chr-3_peak.marker-UNC5008656_lod.drop-1.5_5.batches_mis.csv
    Untracked:  data/ici.vs.eoi_age.of.onset-no.covariates_genes_chr-3_peak.marker-UNC5008656_lod.drop-1.5_5.batches_mis_0.csv
    Untracked:  data/ici.vs.eoi_age.of.onset-no.covariates_genes_chr-3_peak.marker-UNC5046545_lod.drop-1.5_5.batches_mis_0.csv
    Untracked:  data/ici.vs.eoi_age.of.onset-no.covariates_genes_chr-3_peak.marker-UNC5055259_lod.drop-1.5_5.batches_mis_0.csv
    Untracked:  data/ici.vs.eoi_age.of.onset-no.covariates_genes_chr-3_peak.marker-UNC5062524_lod.drop-1.5_5.batches_mis_0.csv
    Untracked:  data/ici.vs.eoi_age.of.onset-no.covariates_genes_chr-3_peak.marker-UNC5068215_lod.drop-1.5_5.batches_mis_0.csv
    Untracked:  data/ici.vs.eoi_age.of.onset-no.covariates_genes_chr-3_peak.marker-UNC5076935_lod.drop-1.5_5.batches_mis_0.csv
    Untracked:  data/ici.vs.eoi_age.of.onset-no.covariates_genes_chr-3_peak.marker-UNC5082757_lod.drop-1.5_5.batches_mis_0.csv
    Untracked:  data/ici.vs.eoi_age.of.onset-no.covariates_genes_chr-3_peak.marker-UNC5083168_lod.drop-1.5_5.batches_0.csv
    Untracked:  data/ici.vs.eoi_age.of.onset-no.covariates_genes_chr-3_peak.marker-UNC5083168_lod.drop-1.5_5.batches_52.csv
    Untracked:  data/ici.vs.eoi_age.of.onset-no.covariates_genes_chr-3_peak.marker-UNC5083168_lod.drop-1.5_5.batches_mis_0.csv
    Untracked:  data/ici.vs.eoi_age.of.onset-no.covariates_genes_chr-3_peak.marker-UNC5173293_lod.drop-1.5_5.batches_mis_0.csv
    Untracked:  data/ici.vs.eoi_age.of.onset-no.covariates_genes_chr-3_peak.marker-UNC5192582_lod.drop-1.5_5.batches_0.csv
    Untracked:  data/ici.vs.eoi_age.of.onset-no.covariates_genes_chr-3_peak.marker-UNC5192582_lod.drop-1.5_5.batches_52.csv
    Untracked:  data/ici.vs.eoi_age.of.onset-no.covariates_genes_chr-3_peak.marker-UNC5192582_lod.drop-1.5_5.batches_mis_0.csv
    Untracked:  data/ici.vs.eoi_age.of.onset-no.covariates_genes_chr-3_peak.marker-UNC5193970_lod.drop-1.5_5.batches_mis_0.csv
    Untracked:  data/ici.vs.eoi_age.of.onset-no.covariates_genes_chr-3_peak.marker-UNC5213888_lod.drop-1.5_snpsqc_5.batches_mis_0.csv
    Untracked:  data/ici.vs.eoi_age.of.onset-no.covariates_genes_chr-3_peak.marker-UNC5213888_lod.drop-1.5_snpsqc_5.batches_mis_52.csv
    Untracked:  data/ici.vs.eoi_age.of.onset-no.covariates_genes_chr-3_peak.marker-UNC5257268_lod.drop-1.5_5.batches_mis_0.csv
    Untracked:  data/ici.vs.eoi_age.of.onset-no.covariates_genes_chr-3_peak.marker-UNC5262472_lod.drop-1.5_5.batches_0.csv
    Untracked:  data/ici.vs.eoi_age.of.onset-no.covariates_genes_chr-3_peak.marker-UNC5262472_lod.drop-1.5_5.batches_52.csv
    Untracked:  data/ici.vs.eoi_age.of.onset-no.covariates_genes_chr-3_peak.marker-UNC5262472_lod.drop-1.5_5.batches_mis_0.csv
    Untracked:  data/ici.vs.eoi_age.of.onset-no.covariates_genes_chr-3_peak.marker-UNC5264696_lod.drop-1.5_5.batches_mis_0.csv
    Untracked:  data/ici.vs.eoi_age.of.onset-no.covariates_genes_chr-3_peak.marker-UNC5267813_lod.drop-1.5_5.batches_mis_0.csv
    Untracked:  data/ici.vs.eoi_age.of.onset-no.covariates_genes_chr-3_peak.marker-UNC5270225_lod.drop-1.5_5.batches_mis_0.csv
    Untracked:  data/ici.vs.eoi_age.of.onset-no.covariates_genes_chr-3_peak.marker-UNC5277061_lod.drop-1.5_5.batches_mis_0.csv
    Untracked:  data/ici.vs.eoi_age.of.onset-no.covariates_genes_chr-3_peak.marker-UNC5278229_lod.drop-1.5_5.batches_0.csv
    Untracked:  data/ici.vs.eoi_age.of.onset-no.covariates_genes_chr-3_peak.marker-UNC5278229_lod.drop-1.5_5.batches_mis_0.csv
    Untracked:  data/ici.vs.eoi_age.of.onset-no.covariates_genes_chr-3_peak.marker-UNC5282259_lod.drop-1.5_5.batches_mis_0.csv
    Untracked:  data/ici.vs.eoi_age.of.onset-no.covariates_genes_chr-3_peak.marker-UNC5287977_lod.drop-1.5_5.batches_mis_0.csv
    Untracked:  data/ici.vs.eoi_age.of.onset-no.covariates_genes_chr-3_peak.marker-UNC5301460_lod.drop-1.5_5.batches_mis_0.csv
    Untracked:  data/ici.vs.eoi_age.of.onset-no.covariates_genes_chr-3_peak.marker-UNC5309136_lod.drop-1.5_5.batches_mis_0.csv
    Untracked:  data/ici.vs.eoi_age.of.onset-no.covariates_genes_chr-3_peak.marker-UNC5321559_lod.drop-1.5_5.batches_mis_0.csv
    Untracked:  data/ici.vs.eoi_age.of.onset-no.covariates_genes_chr-3_peak.marker-UNC5322455_lod.drop-1.5_5.batches_mis_0.csv
    Untracked:  data/ici.vs.eoi_age.of.onset-no.covariates_genes_chr-3_peak.marker-UNC5329171_lod.drop-1.5_5.batches_mis_0.csv
    Untracked:  data/ici.vs.eoi_age.of.onset-no.covariates_genes_chr-3_peak.marker-UNC5360055_lod.drop-1.5_5.batches_52.csv
    Untracked:  data/ici.vs.eoi_age.of.onset-no.covariates_genes_chr-3_peak.marker-UNC5454839_lod.drop-1.5_5.batches_0.csv
    Untracked:  data/ici.vs.eoi_age.of.onset-no.covariates_genes_chr-3_peak.marker-UNC5454839_lod.drop-1.5_5.batches_52.csv
    Untracked:  data/ici.vs.eoi_age.of.onset-no.covariates_genes_chr-3_peak.marker-UNC5454839_lod.drop-1.5_5.batches_mis_0.csv
    Untracked:  data/ici.vs.eoi_age.of.onset-no.covariates_genes_chr-3_peak.marker-UNC5645844_lod.drop-1.5_5.batches_0.csv
    Untracked:  data/ici.vs.eoi_age.of.onset-no.covariates_genes_chr-3_peak.marker-UNC5645844_lod.drop-1.5_5.batches_52.csv
    Untracked:  data/ici.vs.eoi_age.of.onset-no.covariates_genes_chr-3_peak.marker-UNC5645844_lod.drop-1.5_5.batches_mis_0.csv
    Untracked:  data/ici.vs.eoi_age.of.onset-no.covariates_genes_chr-3_peak.marker-UNC5778977_lod.drop-1.5_5.batches_0.csv
    Untracked:  data/ici.vs.eoi_age.of.onset-no.covariates_genes_chr-3_peak.marker-UNC5808246_lod.drop-1.5_5.batches_52.csv
    Untracked:  data/ici.vs.eoi_age.of.onset-no.covariates_genes_chr-3_peak.marker-UNC5916997_lod.drop-1.5_5.batches_52.csv
    Untracked:  data/ici.vs.eoi_age.of.onset-no.covariates_genes_chr-3_peak.marker-UNCHS008182_lod.drop-1.5_5.batches_0.csv
    Untracked:  data/ici.vs.eoi_age.of.onset-no.covariates_genes_chr-3_peak.marker-UNCHS008182_lod.drop-1.5_5.batches_52.csv
    Untracked:  data/ici.vs.eoi_age.of.onset-no.covariates_genes_chr-3_peak.marker-UNCHS008182_lod.drop-1.5_5.batches_mis_0.csv
    Untracked:  data/ici.vs.eoi_age.of.onset-no.covariates_genes_chr-3_peak.marker-UNCHS008193_lod.drop-1.5_5.batches_mis.csv
    Untracked:  data/ici.vs.eoi_age.of.onset-no.covariates_genes_chr-3_peak.marker-UNCHS008193_lod.drop-1.5_5.batches_mis_0.csv
    Untracked:  data/ici.vs.eoi_age.of.onset-no.covariates_genes_chr-3_peak.marker-UNCHS008209_lod.drop-1.5_5.batches_mis.csv
    Untracked:  data/ici.vs.eoi_age.of.onset-no.covariates_genes_chr-3_peak.marker-UNCHS008209_lod.drop-1.5_5.batches_mis_0.csv
    Untracked:  data/ici.vs.eoi_age.of.onset-no.covariates_genes_chr-3_peak.marker-UNCHS008215_lod.drop-1.5_5.batches_mis.csv
    Untracked:  data/ici.vs.eoi_age.of.onset-no.covariates_genes_chr-3_peak.marker-UNCHS008215_lod.drop-1.5_5.batches_mis_0.csv
    Untracked:  data/ici.vs.eoi_age.of.onset-no.covariates_genes_chr-3_peak.marker-UNCHS008219_lod.drop-1.5_5.batches_mis.csv
    Untracked:  data/ici.vs.eoi_age.of.onset-no.covariates_genes_chr-3_peak.marker-UNCHS008219_lod.drop-1.5_5.batches_mis_0.csv
    Untracked:  data/ici.vs.eoi_age.of.onset-no.covariates_genes_chr-3_peak.marker-UNCHS008229_lod.drop-1.5_5.batches_mis.csv
    Untracked:  data/ici.vs.eoi_age.of.onset-no.covariates_genes_chr-3_peak.marker-UNCHS008229_lod.drop-1.5_5.batches_mis_0.csv
    Untracked:  data/ici.vs.eoi_age.of.onset-no.covariates_genes_chr-3_peak.marker-UNCHS008234_lod.drop-1.5_5.batches_mis.csv
    Untracked:  data/ici.vs.eoi_age.of.onset-no.covariates_genes_chr-3_peak.marker-UNCHS008234_lod.drop-1.5_5.batches_mis_0.csv
    Untracked:  data/ici.vs.eoi_age.of.onset-no.covariates_genes_chr-3_peak.marker-UNCHS008242_lod.drop-1.5_5.batches_52.csv
    Untracked:  data/ici.vs.eoi_age.of.onset-no.covariates_genes_chr-3_peak.marker-UNCHS008260_lod.drop-1.5_5.batches_mis.csv
    Untracked:  data/ici.vs.eoi_age.of.onset-no.covariates_genes_chr-3_peak.marker-UNCHS008260_lod.drop-1.5_5.batches_mis_0.csv
    Untracked:  data/ici.vs.eoi_age.of.onset-no.covariates_genes_chr-3_peak.marker-UNCHS008271_lod.drop-1.5_5.batches_mis.csv
    Untracked:  data/ici.vs.eoi_age.of.onset-no.covariates_genes_chr-3_peak.marker-UNCHS008271_lod.drop-1.5_5.batches_mis_0.csv
    Untracked:  data/ici.vs.eoi_age.of.onset-no.covariates_genes_chr-3_peak.marker-UNCHS008281_lod.drop-1.5_5.batches_mis.csv
    Untracked:  data/ici.vs.eoi_age.of.onset-no.covariates_genes_chr-3_peak.marker-UNCHS008281_lod.drop-1.5_5.batches_mis_0.csv
    Untracked:  data/ici.vs.eoi_age.of.onset-no.covariates_genes_chr-3_peak.marker-UNCHS008286_lod.drop-1.5_5.batches_0.csv
    Untracked:  data/ici.vs.eoi_age.of.onset-no.covariates_genes_chr-3_peak.marker-UNCHS008286_lod.drop-1.5_5.batches_52.csv
    Untracked:  data/ici.vs.eoi_age.of.onset-no.covariates_genes_chr-3_peak.marker-UNCHS008286_lod.drop-1.5_5.batches_mis_0.csv
    Untracked:  data/ici.vs.eoi_age.of.onset-no.covariates_genes_chr-3_peak.marker-UNCHS008338_lod.drop-1.5_5.batches_mis.csv
    Untracked:  data/ici.vs.eoi_age.of.onset-no.covariates_genes_chr-3_peak.marker-UNCHS008338_lod.drop-1.5_5.batches_mis_0.csv
    Untracked:  data/ici.vs.eoi_age.of.onset-no.covariates_genes_chr-3_peak.marker-UNCHS008363_lod.drop-1.5_5.batches_mis.csv
    Untracked:  data/ici.vs.eoi_age.of.onset-no.covariates_genes_chr-3_peak.marker-UNCHS008363_lod.drop-1.5_5.batches_mis_0.csv
    Untracked:  data/ici.vs.eoi_age.of.onset-no.covariates_genes_chr-3_peak.marker-UNCHS008379_lod.drop-1.5_5.batches_mis.csv
    Untracked:  data/ici.vs.eoi_age.of.onset-no.covariates_genes_chr-3_peak.marker-UNCHS008379_lod.drop-1.5_5.batches_mis_0.csv
    Untracked:  data/ici.vs.eoi_age.of.onset-no.covariates_genes_chr-3_peak.marker-UNCHS008409_lod.drop-1.5_5.batches_mis_0.csv
    Untracked:  data/ici.vs.eoi_age.of.onset-no.covariates_genes_chr-3_peak.marker-UNCHS008432_lod.drop-1.5_5.batches_mis_0.csv
    Untracked:  data/ici.vs.eoi_age.of.onset-no.covariates_genes_chr-3_peak.marker-UNCHS008487_lod.drop-1.5_snpsqc_5.batches_0.csv
    Untracked:  data/ici.vs.eoi_age.of.onset-no.covariates_genes_chr-3_peak.marker-UNCHS008487_lod.drop-1.5_snpsqc_5.batches_52.csv
    Untracked:  data/ici.vs.eoi_age.of.onset-no.covariates_genes_chr-3_peak.marker-UNCHS008487_lod.drop-1.5_snpsqc_dis_no-x_updated_5.batches_0.csv
    Untracked:  data/ici.vs.eoi_age.of.onset-no.covariates_genes_chr-3_peak.marker-UNCHS008487_lod.drop-1.5_snpsqc_dis_no-x_updated_5.batches_52.csv
    Untracked:  data/ici.vs.eoi_age.of.onset-no.covariates_genes_chr-3_peak.marker-UNCHS008487_lod.drop-1.5_snpsqc_dis_no-x_updated_5.batches_mis_0.csv
    Untracked:  data/ici.vs.eoi_age.of.onset-no.covariates_genes_chr-3_peak.marker-UNCHS008487_lod.drop-1.5_snpsqc_dis_no-x_updated_5.batches_mis_52.csv
    Untracked:  data/ici.vs.eoi_age.of.onset-no.covariates_genes_chr-3_peak.marker-UNCHS008511_lod.drop-1.5_snpsqc_5.batches_mis_0.csv
    Untracked:  data/ici.vs.eoi_age.of.onset-no.covariates_genes_chr-3_peak.marker-UNCHS008511_lod.drop-1.5_snpsqc_5.batches_mis_52.csv
    Untracked:  data/ici.vs.eoi_age.of.onset-no.covariates_genes_chr-3_peak.marker-UNCHS008609_lod.drop-1.5_5.batches_mis_0.csv
    Untracked:  data/ici.vs.eoi_age.of.onset-no.covariates_genes_chr-3_peak.marker-UNCHS008613_lod.drop-1.5_5.batches_0.csv
    Untracked:  data/ici.vs.eoi_age.of.onset-no.covariates_genes_chr-3_peak.marker-UNCHS008613_lod.drop-1.5_5.batches_52.csv
    Untracked:  data/ici.vs.eoi_age.of.onset-no.covariates_genes_chr-3_peak.marker-UNCHS008614_lod.drop-1.5_5.batches_mis_0.csv
    Untracked:  data/ici.vs.eoi_age.of.onset-no.covariates_genes_chr-3_peak.marker-UNCHS008627_lod.drop-1.5_5.batches_mis_0.csv
    Untracked:  data/ici.vs.eoi_age.of.onset-no.covariates_genes_chr-3_peak.marker-UNCHS008659_lod.drop-1.5_5.batches_52.csv
    Untracked:  data/ici.vs.eoi_age.of.onset-no.covariates_genes_chr-3_peak.marker-UNCHS008725_lod.drop-1.5_5.batches_mis_0.csv
    Untracked:  data/ici.vs.eoi_age.of.onset-no.covariates_genes_chr-3_peak.marker-UNCHS008815_lod.drop-1.5_5.batches_0.csv
    Untracked:  data/ici.vs.eoi_age.of.onset-no.covariates_genes_chr-3_peak.marker-UNCHS008815_lod.drop-1.5_5.batches_52.csv
    Untracked:  data/ici.vs.eoi_age.of.onset-no.covariates_genes_chr-3_peak.marker-UNCHS008815_lod.drop-1.5_5.batches_mis_0.csv
    Untracked:  data/ici.vs.eoi_age.of.onset-no.covariates_genes_chr-3_peak.marker-UNCHS009066_lod.drop-1.5_5.batches_52.csv
    Untracked:  data/ici.vs.eoi_age.of.onset-no.covariates_genes_chr-3_peak.marker-UNCHS009067_lod.drop-1.5_5.batches_0.csv
    Untracked:  data/ici.vs.eoi_age.of.onset-no.covariates_genes_chr-3_peak.marker-UNCHS009068_lod.drop-1.5_5.batches_mis_0.csv
    Untracked:  data/ici.vs.eoi_age.of.onset-no.covariates_genes_chr-3_peak.marker-UNCHS009112_lod.drop-1.5_5.batches_52.csv
    Untracked:  data/ici.vs.eoi_age.of.onset-no.covariates_genes_chr-3_peak.marker-UNCHS009224_lod.drop-1.5_5.batches_0.csv
    Untracked:  data/ici.vs.eoi_age.of.onset-no.covariates_genes_chr-3_peak.marker-UNCHS009224_lod.drop-1.5_5.batches_52.csv
    Untracked:  data/ici.vs.eoi_age.of.onset-no.covariates_genes_chr-3_peak.marker-UNCHS009383_lod.drop-1.5_5.batches_0.csv
    Untracked:  data/ici.vs.eoi_age.of.onset-no.covariates_genes_chr-3_peak.marker-UNCHS009383_lod.drop-1.5_5.batches_52.csv
    Untracked:  data/ici.vs.eoi_age.of.onset-no.covariates_genes_chr-3_peak.marker-UNCHS009693_lod.drop-1.5_5.batches_52.csv
    Untracked:  data/ici.vs.eoi_age.of.onset-no.covariates_genes_chr-3_peak.marker-UNCHS009821_lod.drop-1.5_5.batches_52.csv
    Untracked:  data/ici.vs.eoi_age.of.onset-no.covariates_genes_chr-3_peak.marker-UNCJPD001170_lod.drop-1.5_5.batches_mis.csv
    Untracked:  data/ici.vs.eoi_age.of.onset-no.covariates_genes_chr-3_peak.marker-UNCJPD001170_lod.drop-1.5_5.batches_mis_0.csv
    Untracked:  data/ici.vs.eoi_age.of.onset-no.covariates_genes_chr-3_peak.marker-UNCJPD001198_lod.drop-1.5_5.batches_mis.csv
    Untracked:  data/ici.vs.eoi_age.of.onset-no.covariates_genes_chr-3_peak.marker-UNCJPD001198_lod.drop-1.5_5.batches_mis_0.csv
    Untracked:  data/ici.vs.eoi_age.of.onset-no.covariates_genes_chr-3_peak.marker-UNCJPD001276_lod.drop-1.5_5.batches_mis_0.csv
    Untracked:  data/ici.vs.eoi_age.of.onset-no.covariates_genes_chr-3_peak.marker-sanger2496q_lod.drop-1.5_5.batches_mis.csv
    Untracked:  data/ici.vs.eoi_age.of.onset-no.covariates_genes_chr-3_peak.marker-sanger2496q_lod.drop-1.5_5.batches_mis_0.csv
    Untracked:  data/ici.vs.eoi_age.of.onset-no.covariates_genes_chr-4_peak.marker-UNC8250659_lod.drop-1.5_snpsqc_dis_no-x_updated_5.batches_52.csv
    Untracked:  data/ici.vs.eoi_age.of.onset-no.covariates_genes_chr-4_peak.marker-UNC8439633_lod.drop-1.5_snpsqc_5.batches_52.csv
    Untracked:  data/ici.vs.eoi_age.of.onset-no.covariates_genes_chr-4_peak.marker-UNC8439633_lod.drop-1.5_snpsqc_5.batches_mis_52.csv
    Untracked:  data/ici.vs.eoi_age.of.onset-no.covariates_genes_chr-4_peak.marker-UNCHS012955_lod.drop-1.5_snpsqc_dis_no-x_updated_5.batches_mis_52.csv
    Untracked:  data/ici.vs.eoi_age.of.onset-no.covariates_genes_chr-6_peak.marker-UNC11108920_lod.drop-1.5_snpsqc_5.batches.csv
    Untracked:  data/ici.vs.eoi_age.of.onset-no.covariates_genes_chr-6_peak.marker-UNC11108920_lod.drop-1.5_snpsqc_5.batches_mis.csv
    Untracked:  data/ici.vs.eoi_age.of.onset-no.covariates_genes_chr-7_peak.marker-UNCHS020066_lod.drop-1.5_snpsqc_5.batches_mis_0.csv
    Untracked:  data/ici.vs.eoi_age.of.onset-no.covariates_genes_chr-7_peak.marker-UNCHS020066_lod.drop-1.5_snpsqc_5.batches_mis_52.csv
    Untracked:  data/ici.vs.eoi_age.of.onset-no.covariates_genes_chr-7_peak.marker-UNCHS020066_lod.drop-1.5_snpsqc_dis_no-x_updated_5.batches_mis_0.csv
    Untracked:  data/ici.vs.eoi_age.of.onset-no.covariates_genes_chr-7_peak.marker-UNCHS020066_lod.drop-1.5_snpsqc_dis_no-x_updated_5.batches_mis_52.csv
    Untracked:  data/ici.vs.eoi_age.of.onset-no.covariates_genes_chr-8_peak.marker-UNC15524531_lod.drop-1.5_snpsqc_5.batches_mis_0.csv
    Untracked:  data/ici.vs.eoi_age.of.onset-no.covariates_genes_chr-8_peak.marker-UNC15524531_lod.drop-1.5_snpsqc_5.batches_mis_52.csv
    Untracked:  data/ici.vs.eoi_age.of.onset-no.covariates_genes_chr-8_peak.marker-UNC15524531_lod.drop-1.5_snpsqc_dis_no-x_updated_5.batches_mis_0.csv
    Untracked:  data/ici.vs.eoi_age.of.onset-no.covariates_genes_chr-8_peak.marker-UNC15524531_lod.drop-1.5_snpsqc_dis_no-x_updated_5.batches_mis_52.csv
    Untracked:  data/ici.vs.eoi_age.of.onset-no.covariates_genes_chr-9_peak.marker-UNC17203597_lod.drop-1.5_snpsqc_5.batches_mis_0.csv
    Untracked:  data/ici.vs.eoi_age.of.onset-no.covariates_genes_chr-9_peak.marker-UNC17203597_lod.drop-1.5_snpsqc_5.batches_mis_52.csv
    Untracked:  data/ici.vs.eoi_age.of.onset-no.covariates_genes_chr-X_peak.marker-UNCHS048314_lod.drop-1.5_snpsqc_5.batches_mis_0.csv
    Untracked:  data/ici.vs.eoi_age.of.onset-no.covariates_genes_chr-X_peak.marker-UNCHS048314_lod.drop-1.5_snpsqc_5.batches_mis_52.csv
    Untracked:  data/ici.vs.eoi_age.of.onset-no.covariates_genes_chr-X_peak.marker-UNCHS048314_lod.drop-1.5_snpsqc_dis_no-x_updated_5.batches_mis_0.csv
    Untracked:  data/ici.vs.eoi_age.of.onset-no.covariates_genes_chr-X_peak.marker-UNCHS048314_lod.drop-1.5_snpsqc_dis_no-x_updated_5.batches_mis_52.csv
    Untracked:  data/ici.vs.eoi_blup.full_chr-10_snpsqc_5.batches.csv
    Untracked:  data/ici.vs.eoi_blup.full_chr-10_snpsqc_5.batches_mis.csv
    Untracked:  data/ici.vs.eoi_blup.full_chr-11_snpsqc_5.batches.csv
    Untracked:  data/ici.vs.eoi_blup.full_chr-11_snpsqc_5.batches_mis.csv
    Untracked:  data/ici.vs.eoi_blup.full_chr-12_snpsqc_5.batches.csv
    Untracked:  data/ici.vs.eoi_blup.full_chr-12_snpsqc_5.batches_mis.csv
    Untracked:  data/ici.vs.eoi_blup.full_chr-13_snpsqc_5.batches.csv
    Untracked:  data/ici.vs.eoi_blup.full_chr-13_snpsqc_5.batches_mis.csv
    Untracked:  data/ici.vs.eoi_blup.full_chr-14_snpsqc_5.batches.csv
    Untracked:  data/ici.vs.eoi_blup.full_chr-14_snpsqc_5.batches_mis.csv
    Untracked:  data/ici.vs.eoi_blup.full_chr-15_snpsqc_5.batches.csv
    Untracked:  data/ici.vs.eoi_blup.full_chr-15_snpsqc_5.batches_mis.csv
    Untracked:  data/ici.vs.eoi_blup.full_chr-16_snpsqc_5.batches.csv
    Untracked:  data/ici.vs.eoi_blup.full_chr-16_snpsqc_5.batches_mis.csv
    Untracked:  data/ici.vs.eoi_blup.full_chr-17_snpsqc_5.batches.csv
    Untracked:  data/ici.vs.eoi_blup.full_chr-17_snpsqc_5.batches_mis.csv
    Untracked:  data/ici.vs.eoi_blup.full_chr-18_snpsqc_5.batches.csv
    Untracked:  data/ici.vs.eoi_blup.full_chr-18_snpsqc_5.batches_mis.csv
    Untracked:  data/ici.vs.eoi_blup.full_chr-19_snpsqc_5.batches.csv
    Untracked:  data/ici.vs.eoi_blup.full_chr-19_snpsqc_5.batches_mis.csv
    Untracked:  data/ici.vs.eoi_blup.full_chr-1_snpsqc_5.batches.csv
    Untracked:  data/ici.vs.eoi_blup.full_chr-1_snpsqc_5.batches_mis.csv
    Untracked:  data/ici.vs.eoi_blup.full_chr-2_snpsqc_5.batches.csv
    Untracked:  data/ici.vs.eoi_blup.full_chr-2_snpsqc_5.batches_mis.csv
    Untracked:  data/ici.vs.eoi_blup.full_chr-3_snpsqc_5.batches.csv
    Untracked:  data/ici.vs.eoi_blup.full_chr-3_snpsqc_5.batches_mis.csv
    Untracked:  data/ici.vs.eoi_blup.full_chr-4_snpsqc_5.batches.csv
    Untracked:  data/ici.vs.eoi_blup.full_chr-4_snpsqc_5.batches_mis.csv
    Untracked:  data/ici.vs.eoi_blup.full_chr-5_snpsqc_5.batches.csv
    Untracked:  data/ici.vs.eoi_blup.full_chr-5_snpsqc_5.batches_mis.csv
    Untracked:  data/ici.vs.eoi_blup.full_chr-6_snpsqc_5.batches.csv
    Untracked:  data/ici.vs.eoi_blup.full_chr-6_snpsqc_5.batches_mis.csv
    Untracked:  data/ici.vs.eoi_blup.full_chr-7_snpsqc_5.batches.csv
    Untracked:  data/ici.vs.eoi_blup.full_chr-7_snpsqc_5.batches_mis.csv
    Untracked:  data/ici.vs.eoi_blup.full_chr-8_snpsqc_5.batches.csv
    Untracked:  data/ici.vs.eoi_blup.full_chr-8_snpsqc_5.batches_mis.csv
    Untracked:  data/ici.vs.eoi_blup.full_chr-9_snpsqc_5.batches.csv
    Untracked:  data/ici.vs.eoi_blup.full_chr-9_snpsqc_5.batches_mis.csv
    Untracked:  data/ici.vs.eoi_blup.full_chr-X_snpsqc_5.batches.csv
    Untracked:  data/ici.vs.eoi_blup.full_chr-X_snpsqc_5.batches_mis.csv
    Untracked:  data/ici.vs.eoi_blup.qc_chr-10_5.batches.csv
    Untracked:  data/ici.vs.eoi_blup.qc_chr-10_5.batches_mis.csv
    Untracked:  data/ici.vs.eoi_blup.qc_chr-10_lod.drop-1.5_snpsqc_5.batches.csv
    Untracked:  data/ici.vs.eoi_blup.qc_chr-10_snpsqc_5.batches.csv
    Untracked:  data/ici.vs.eoi_blup.qc_chr-10_snpsqc_5.batches_mis.csv
    Untracked:  data/ici.vs.eoi_blup.qc_chr-10_snpsqc_dis_no-x_updated_5.batches.csv
    Untracked:  data/ici.vs.eoi_blup.qc_chr-10_snpsqc_dis_no-x_updated_5.batches_mis.csv
    Untracked:  data/ici.vs.eoi_blup.qc_chr-11_5.batches.csv
    Untracked:  data/ici.vs.eoi_blup.qc_chr-11_5.batches_mis.csv
    Untracked:  data/ici.vs.eoi_blup.qc_chr-11_lod.drop-1.5_snpsqc_5.batches.csv
    Untracked:  data/ici.vs.eoi_blup.qc_chr-11_snpsqc_5.batches.csv
    Untracked:  data/ici.vs.eoi_blup.qc_chr-11_snpsqc_5.batches_mis.csv
    Untracked:  data/ici.vs.eoi_blup.qc_chr-11_snpsqc_dis_no-x_updated_5.batches.csv
    Untracked:  data/ici.vs.eoi_blup.qc_chr-11_snpsqc_dis_no-x_updated_5.batches_mis.csv
    Untracked:  data/ici.vs.eoi_blup.qc_chr-12_5.batches.csv
    Untracked:  data/ici.vs.eoi_blup.qc_chr-12_5.batches_mis.csv
    Untracked:  data/ici.vs.eoi_blup.qc_chr-12_lod.drop-1.5_snpsqc_5.batches.csv
    Untracked:  data/ici.vs.eoi_blup.qc_chr-12_snpsqc_5.batches.csv
    Untracked:  data/ici.vs.eoi_blup.qc_chr-12_snpsqc_5.batches_mis.csv
    Untracked:  data/ici.vs.eoi_blup.qc_chr-12_snpsqc_dis_no-x_updated_5.batches.csv
    Untracked:  data/ici.vs.eoi_blup.qc_chr-12_snpsqc_dis_no-x_updated_5.batches_mis.csv
    Untracked:  data/ici.vs.eoi_blup.qc_chr-13_5.batches.csv
    Untracked:  data/ici.vs.eoi_blup.qc_chr-13_5.batches_mis.csv
    Untracked:  data/ici.vs.eoi_blup.qc_chr-13_lod.drop-1.5_snpsqc_5.batches.csv
    Untracked:  data/ici.vs.eoi_blup.qc_chr-13_snpsqc_5.batches.csv
    Untracked:  data/ici.vs.eoi_blup.qc_chr-13_snpsqc_5.batches_mis.csv
    Untracked:  data/ici.vs.eoi_blup.qc_chr-13_snpsqc_dis_no-x_updated_5.batches.csv
    Untracked:  data/ici.vs.eoi_blup.qc_chr-13_snpsqc_dis_no-x_updated_5.batches_mis.csv
    Untracked:  data/ici.vs.eoi_blup.qc_chr-14_5.batches.csv
    Untracked:  data/ici.vs.eoi_blup.qc_chr-14_5.batches_mis.csv
    Untracked:  data/ici.vs.eoi_blup.qc_chr-14_snpsqc_5.batches.csv
    Untracked:  data/ici.vs.eoi_blup.qc_chr-14_snpsqc_5.batches_mis.csv
    Untracked:  data/ici.vs.eoi_blup.qc_chr-14_snpsqc_dis_no-x_updated_5.batches.csv
    Untracked:  data/ici.vs.eoi_blup.qc_chr-14_snpsqc_dis_no-x_updated_5.batches_mis.csv
    Untracked:  data/ici.vs.eoi_blup.qc_chr-15_5.batches.csv
    Untracked:  data/ici.vs.eoi_blup.qc_chr-15_5.batches_mis.csv
    Untracked:  data/ici.vs.eoi_blup.qc_chr-15_snpsqc_5.batches.csv
    Untracked:  data/ici.vs.eoi_blup.qc_chr-15_snpsqc_5.batches_mis.csv
    Untracked:  data/ici.vs.eoi_blup.qc_chr-15_snpsqc_dis_no-x_updated_5.batches_mis.csv
    Untracked:  data/ici.vs.eoi_blup.qc_chr-16_5.batches.csv
    Untracked:  data/ici.vs.eoi_blup.qc_chr-16_5.batches_mis.csv
    Untracked:  data/ici.vs.eoi_blup.qc_chr-16_snpsqc_5.batches.csv
    Untracked:  data/ici.vs.eoi_blup.qc_chr-16_snpsqc_5.batches_mis.csv
    Untracked:  data/ici.vs.eoi_blup.qc_chr-16_snpsqc_dis_no-x_updated_5.batches.csv
    Untracked:  data/ici.vs.eoi_blup.qc_chr-16_snpsqc_dis_no-x_updated_5.batches_mis.csv
    Untracked:  data/ici.vs.eoi_blup.qc_chr-17_5.batches.csv
    Untracked:  data/ici.vs.eoi_blup.qc_chr-17_5.batches_mis.csv
    Untracked:  data/ici.vs.eoi_blup.qc_chr-17_snpsqc_5.batches.csv
    Untracked:  data/ici.vs.eoi_blup.qc_chr-17_snpsqc_5.batches_mis.csv
    Untracked:  data/ici.vs.eoi_blup.qc_chr-17_snpsqc_dis_no-x_updated_5.batches.csv
    Untracked:  data/ici.vs.eoi_blup.qc_chr-17_snpsqc_dis_no-x_updated_5.batches_mis.csv
    Untracked:  data/ici.vs.eoi_blup.qc_chr-18_5.batches.csv
    Untracked:  data/ici.vs.eoi_blup.qc_chr-18_5.batches_mis.csv
    Untracked:  data/ici.vs.eoi_blup.qc_chr-18_snpsqc_5.batches.csv
    Untracked:  data/ici.vs.eoi_blup.qc_chr-18_snpsqc_5.batches_mis.csv
    Untracked:  data/ici.vs.eoi_blup.qc_chr-18_snpsqc_dis_no-x_updated_5.batches.csv
    Untracked:  data/ici.vs.eoi_blup.qc_chr-18_snpsqc_dis_no-x_updated_5.batches_mis.csv
    Untracked:  data/ici.vs.eoi_blup.qc_chr-19_5.batches.csv
    Untracked:  data/ici.vs.eoi_blup.qc_chr-19_5.batches_mis.csv
    Untracked:  data/ici.vs.eoi_blup.qc_chr-19_snpsqc_5.batches.csv
    Untracked:  data/ici.vs.eoi_blup.qc_chr-19_snpsqc_5.batches_mis.csv
    Untracked:  data/ici.vs.eoi_blup.qc_chr-19_snpsqc_dis_no-x_updated_5.batches.csv
    Untracked:  data/ici.vs.eoi_blup.qc_chr-19_snpsqc_dis_no-x_updated_5.batches_mis.csv
    Untracked:  data/ici.vs.eoi_blup.qc_chr-1_5.batches.csv
    Untracked:  data/ici.vs.eoi_blup.qc_chr-1_5.batches_mis.csv
    Untracked:  data/ici.vs.eoi_blup.qc_chr-1_lod.drop-1.5_snpsqc_5.batches.csv
    Untracked:  data/ici.vs.eoi_blup.qc_chr-1_snpsqc_5.batches.csv
    Untracked:  data/ici.vs.eoi_blup.qc_chr-1_snpsqc_5.batches_mis.csv
    Untracked:  data/ici.vs.eoi_blup.qc_chr-1_snpsqc_dis_no-x_updated_5.batches.csv
    Untracked:  data/ici.vs.eoi_blup.qc_chr-1_snpsqc_dis_no-x_updated_5.batches_mis.csv
    Untracked:  data/ici.vs.eoi_blup.qc_chr-2_5.batches.csv
    Untracked:  data/ici.vs.eoi_blup.qc_chr-2_5.batches_mis.csv
    Untracked:  data/ici.vs.eoi_blup.qc_chr-2_lod.drop-1.5_snpsqc_5.batches.csv
    Untracked:  data/ici.vs.eoi_blup.qc_chr-2_snpsqc_5.batches.csv
    Untracked:  data/ici.vs.eoi_blup.qc_chr-2_snpsqc_5.batches_mis.csv
    Untracked:  data/ici.vs.eoi_blup.qc_chr-2_snpsqc_dis_no-x_updated_5.batches.csv
    Untracked:  data/ici.vs.eoi_blup.qc_chr-2_snpsqc_dis_no-x_updated_5.batches_mis.csv
    Untracked:  data/ici.vs.eoi_blup.qc_chr-3_5.batches.csv
    Untracked:  data/ici.vs.eoi_blup.qc_chr-3_5.batches_mis.csv
    Untracked:  data/ici.vs.eoi_blup.qc_chr-3_lod.drop-1.5_snpsqc_5.batches.csv
    Untracked:  data/ici.vs.eoi_blup.qc_chr-3_snpsqc_5.batches.csv
    Untracked:  data/ici.vs.eoi_blup.qc_chr-3_snpsqc_5.batches_mis.csv
    Untracked:  data/ici.vs.eoi_blup.qc_chr-3_snpsqc_dis_no-x_updated_5.batches.csv
    Untracked:  data/ici.vs.eoi_blup.qc_chr-3_snpsqc_dis_no-x_updated_5.batches_mis.csv
    Untracked:  data/ici.vs.eoi_blup.qc_chr-4_5.batches.csv
    Untracked:  data/ici.vs.eoi_blup.qc_chr-4_5.batches_mis.csv
    Untracked:  data/ici.vs.eoi_blup.qc_chr-4_lod.drop-1.5_snpsqc_5.batches.csv
    Untracked:  data/ici.vs.eoi_blup.qc_chr-4_snpsqc_5.batches.csv
    Untracked:  data/ici.vs.eoi_blup.qc_chr-4_snpsqc_5.batches_mis.csv
    Untracked:  data/ici.vs.eoi_blup.qc_chr-4_snpsqc_dis_no-x_updated_5.batches.csv
    Untracked:  data/ici.vs.eoi_blup.qc_chr-4_snpsqc_dis_no-x_updated_5.batches_mis.csv
    Untracked:  data/ici.vs.eoi_blup.qc_chr-5_5.batches.csv
    Untracked:  data/ici.vs.eoi_blup.qc_chr-5_5.batches_mis.csv
    Untracked:  data/ici.vs.eoi_blup.qc_chr-5_lod.drop-1.5_snpsqc_5.batches.csv
    Untracked:  data/ici.vs.eoi_blup.qc_chr-5_snpsqc_5.batches.csv
    Untracked:  data/ici.vs.eoi_blup.qc_chr-5_snpsqc_5.batches_mis.csv
    Untracked:  data/ici.vs.eoi_blup.qc_chr-5_snpsqc_dis_no-x_updated_5.batches.csv
    Untracked:  data/ici.vs.eoi_blup.qc_chr-5_snpsqc_dis_no-x_updated_5.batches_mis.csv
    Untracked:  data/ici.vs.eoi_blup.qc_chr-6_5.batches.csv
    Untracked:  data/ici.vs.eoi_blup.qc_chr-6_5.batches_mis.csv
    Untracked:  data/ici.vs.eoi_blup.qc_chr-6_lod.drop-1.5_snpsqc_5.batches.csv
    Untracked:  data/ici.vs.eoi_blup.qc_chr-6_snpsqc_5.batches.csv
    Untracked:  data/ici.vs.eoi_blup.qc_chr-6_snpsqc_5.batches_mis.csv
    Untracked:  data/ici.vs.eoi_blup.qc_chr-6_snpsqc_dis_no-x_updated_5.batches.csv
    Untracked:  data/ici.vs.eoi_blup.qc_chr-6_snpsqc_dis_no-x_updated_5.batches_mis.csv
    Untracked:  data/ici.vs.eoi_blup.qc_chr-7_5.batches.csv
    Untracked:  data/ici.vs.eoi_blup.qc_chr-7_5.batches_mis.csv
    Untracked:  data/ici.vs.eoi_blup.qc_chr-7_lod.drop-1.5_snpsqc_5.batches.csv
    Untracked:  data/ici.vs.eoi_blup.qc_chr-7_snpsqc_5.batches.csv
    Untracked:  data/ici.vs.eoi_blup.qc_chr-7_snpsqc_5.batches_mis.csv
    Untracked:  data/ici.vs.eoi_blup.qc_chr-7_snpsqc_dis_no-x_updated_5.batches.csv
    Untracked:  data/ici.vs.eoi_blup.qc_chr-7_snpsqc_dis_no-x_updated_5.batches_mis.csv
    Untracked:  data/ici.vs.eoi_blup.qc_chr-8_5.batches.csv
    Untracked:  data/ici.vs.eoi_blup.qc_chr-8_5.batches_mis.csv
    Untracked:  data/ici.vs.eoi_blup.qc_chr-8_lod.drop-1.5_snpsqc_5.batches.csv
    Untracked:  data/ici.vs.eoi_blup.qc_chr-8_snpsqc_5.batches.csv
    Untracked:  data/ici.vs.eoi_blup.qc_chr-8_snpsqc_5.batches_mis.csv
    Untracked:  data/ici.vs.eoi_blup.qc_chr-8_snpsqc_dis_no-x_updated_5.batches.csv
    Untracked:  data/ici.vs.eoi_blup.qc_chr-8_snpsqc_dis_no-x_updated_5.batches_mis.csv
    Untracked:  data/ici.vs.eoi_blup.qc_chr-9_5.batches.csv
    Untracked:  data/ici.vs.eoi_blup.qc_chr-9_5.batches_mis.csv
    Untracked:  data/ici.vs.eoi_blup.qc_chr-9_lod.drop-1.5_snpsqc_5.batches.csv
    Untracked:  data/ici.vs.eoi_blup.qc_chr-9_snpsqc_5.batches.csv
    Untracked:  data/ici.vs.eoi_blup.qc_chr-9_snpsqc_5.batches_mis.csv
    Untracked:  data/ici.vs.eoi_blup.qc_chr-9_snpsqc_dis_no-x_updated_5.batches.csv
    Untracked:  data/ici.vs.eoi_blup.qc_chr-9_snpsqc_dis_no-x_updated_5.batches_mis.csv
    Untracked:  data/ici.vs.eoi_blup.qc_chr-X_5.batches.csv
    Untracked:  data/ici.vs.eoi_blup.qc_chr-X_5.batches_mis.csv
    Untracked:  data/ici.vs.eoi_blup.qc_chr-X_snpsqc_5.batches.csv
    Untracked:  data/ici.vs.eoi_blup.qc_chr-X_snpsqc_5.batches_mis.csv
    Untracked:  data/ici.vs.eoi_blup.qc_chr-X_snpsqc_dis_no-x_updated_5.batches.csv
    Untracked:  data/ici.vs.eoi_blup.qc_chr-X_snpsqc_dis_no-x_updated_5.batches_mis.csv
    Untracked:  data/ici.vs.eoi_blup_sub_chr-10_peak.marker-ICR4295_lod.drop-1.5_5.batches_mis.csv
    Untracked:  data/ici.vs.eoi_blup_sub_chr-10_peak.marker-JAX00292499_lod.drop-1.5_5.batches.csv
    Untracked:  data/ici.vs.eoi_blup_sub_chr-10_peak.marker-JAX00292927_lod.drop-1.5_5.batches.csv
    Untracked:  data/ici.vs.eoi_blup_sub_chr-10_peak.marker-JAX00292927_lod.drop-1.5_snpsqc_5.batches.csv
    Untracked:  data/ici.vs.eoi_blup_sub_chr-10_peak.marker-JAX00292927_lod.drop-1.5_snpsqc_5.batches_mis.csv
    Untracked:  data/ici.vs.eoi_blup_sub_chr-10_peak.marker-UNC18154330_lod.drop-1.5_5.batches.csv
    Untracked:  data/ici.vs.eoi_blup_sub_chr-10_peak.marker-UNC18195765_lod.drop-1.5_5.batches.csv
    Untracked:  data/ici.vs.eoi_blup_sub_chr-10_peak.marker-UNC18216614_lod.drop-1.5_5.batches.csv
    Untracked:  data/ici.vs.eoi_blup_sub_chr-10_peak.marker-UNC18240977_lod.drop-1.5_snpsqc_dis_no-x_updated_5.batches.csv
    Untracked:  data/ici.vs.eoi_blup_sub_chr-10_peak.marker-UNC18311938_lod.drop-1.5_5.batches.csv
    Untracked:  data/ici.vs.eoi_blup_sub_chr-10_peak.marker-UNC18856953_lod.drop-1.5_5.batches_mis.csv
    Untracked:  data/ici.vs.eoi_blup_sub_chr-10_peak.marker-UNCHS028236_lod.drop-1.5_snpsqc_5.batches_mis.csv
    Untracked:  data/ici.vs.eoi_blup_sub_chr-10_peak.marker-UNCHS028536_lod.drop-1.5_snpsqc_dis_no-x_updated_5.batches_mis.csv
    Untracked:  data/ici.vs.eoi_blup_sub_chr-10_peak.marker-UNCJPD004316_lod.drop-1.5_5.batches_mis.csv
    Untracked:  data/ici.vs.eoi_blup_sub_chr-10_peak.marker-UNCJPD009532_lod.drop-1.5_5.batches_mis.csv
    Untracked:  data/ici.vs.eoi_blup_sub_chr-11_peak.marker-UNC19155926_lod.drop-1.5_5.batches_mis.csv
    Untracked:  data/ici.vs.eoi_blup_sub_chr-11_peak.marker-UNC19481366_lod.drop-1.5_snpsqc_5.batches_mis.csv
    Untracked:  data/ici.vs.eoi_blup_sub_chr-11_peak.marker-UNC19970181_lod.drop-1.5_5.batches_mis.csv
    Untracked:  data/ici.vs.eoi_blup_sub_chr-11_peak.marker-UNC19970181_lod.drop-1.5_snpsqc_5.batches_mis.csv
    Untracked:  data/ici.vs.eoi_blup_sub_chr-11_peak.marker-UNCJPD004792_lod.drop-1.5_5.batches_mis.csv
    Untracked:  data/ici.vs.eoi_blup_sub_chr-12_peak.marker-ICR4497_lod.drop-1.5_5.batches_mis.csv
    Untracked:  data/ici.vs.eoi_blup_sub_chr-12_peak.marker-ICR499_lod.drop-1.5_snpsqc_5.batches_mis.csv
    Untracked:  data/ici.vs.eoi_blup_sub_chr-13_peak.marker-UNCHS036773_lod.drop-1.5_snpsqc_5.batches_mis.csv
    Untracked:  data/ici.vs.eoi_blup_sub_chr-13_peak.marker-UNCHS036964_lod.drop-1.5_5.batches_mis.csv
    Untracked:  data/ici.vs.eoi_blup_sub_chr-14_peak.marker-UNC24056202_lod.drop-1.5_5.batches_mis.csv
    Untracked:  data/ici.vs.eoi_blup_sub_chr-14_peak.marker-UNC24056202_lod.drop-1.5_snpsqc_5.batches_mis.csv
    Untracked:  data/ici.vs.eoi_blup_sub_chr-15_peak.marker-UNC26070435_lod.drop-1.5_snpsqc_dis_no-x_updated_5.batches_mis.csv
    Untracked:  data/ici.vs.eoi_blup_sub_chr-17_peak.marker-UNCHS044241_lod.drop-1.5_5.batches_mis.csv
    Untracked:  data/ici.vs.eoi_blup_sub_chr-17_peak.marker-UNCHS044241_lod.drop-1.5_snpsqc_5.batches_mis.csv
    Untracked:  data/ici.vs.eoi_blup_sub_chr-17_peak.marker-UNCrs47191360_lod.drop-1.5_5.batches_mis.csv
    Untracked:  data/ici.vs.eoi_blup_sub_chr-17_peak.marker-UNCrs47191360_lod.drop-1.5_snpsqc_dis_no-x_updated_5.batches_mis.csv
    Untracked:  data/ici.vs.eoi_blup_sub_chr-18_peak.marker-UNCHS045343_lod.drop-1.5_snpsqc_5.batches_mis.csv
    Untracked:  data/ici.vs.eoi_blup_sub_chr-18_peak.marker-UNCHS045344_lod.drop-1.5_5.batches_mis.csv
    Untracked:  data/ici.vs.eoi_blup_sub_chr-19_peak.marker-UNCJPD007087_lod.drop-1.5_5.batches_mis.csv
    Untracked:  data/ici.vs.eoi_blup_sub_chr-1_peak.marker-ICR010_lod.drop-1.5_5.batches.csv
    Untracked:  data/ici.vs.eoi_blup_sub_chr-1_peak.marker-ICR010_lod.drop-1.5_5.batches_mis.csv
    Untracked:  data/ici.vs.eoi_blup_sub_chr-2_peak.marker-UNC4609527_lod.drop-1.5_snpsqc_dis_no-x_updated_5.batches_mis.csv
    Untracked:  data/ici.vs.eoi_blup_sub_chr-2_peak.marker-UNC4609660_lod.drop-1.5_5.batches_mis.csv
    Untracked:  data/ici.vs.eoi_blup_sub_chr-3_peak.marker-B6_rs31740628_lod.drop-1.5_5.batches_mis.csv
    Untracked:  data/ici.vs.eoi_blup_sub_chr-3_peak.marker-ICR1269_lod.drop-1.5_5.batches_mis.csv
    Untracked:  data/ici.vs.eoi_blup_sub_chr-3_peak.marker-ICR1338_lod.drop-1.5_5.batches.csv
    Untracked:  data/ici.vs.eoi_blup_sub_chr-3_peak.marker-ICR5325_lod.drop-1.5_5.batches_mis.csv
    Untracked:  data/ici.vs.eoi_blup_sub_chr-3_peak.marker-ICR5347_lod.drop-1.5_5.batches.csv
    Untracked:  data/ici.vs.eoi_blup_sub_chr-3_peak.marker-ICR5348_lod.drop-1.5_5.batches_mis.csv
    Untracked:  data/ici.vs.eoi_blup_sub_chr-3_peak.marker-JAX00104971_lod.drop-1.5_5.batches_mis.csv
    Untracked:  data/ici.vs.eoi_blup_sub_chr-3_peak.marker-JAX00105059_lod.drop-1.5_5.batches.csv
    Untracked:  data/ici.vs.eoi_blup_sub_chr-3_peak.marker-JAX00105203_lod.drop-1.5_5.batches_mis.csv
    Untracked:  data/ici.vs.eoi_blup_sub_chr-3_peak.marker-JAX00106957_lod.drop-1.5_5.batches.csv
    Untracked:  data/ici.vs.eoi_blup_sub_chr-3_peak.marker-JAX00106957_lod.drop-1.5_5.batches_mis.csv
    Untracked:  data/ici.vs.eoi_blup_sub_chr-3_peak.marker-JAX00107930_lod.drop-1.5_5.batches_mis.csv
    Untracked:  data/ici.vs.eoi_blup_sub_chr-3_peak.marker-JAX00107980_lod.drop-1.5_5.batches_mis.csv
    Untracked:  data/ici.vs.eoi_blup_sub_chr-3_peak.marker-JAX00108068_lod.drop-1.5_5.batches_mis.csv
    Untracked:  data/ici.vs.eoi_blup_sub_chr-3_peak.marker-JAX00108114_lod.drop-1.5_5.batches_mis.csv
    Untracked:  data/ici.vs.eoi_blup_sub_chr-3_peak.marker-JAX00109750_lod.drop-1.5_5.batches_mis.csv
    Untracked:  data/ici.vs.eoi_blup_sub_chr-3_peak.marker-JAX00189245_lod.drop-1.5_5.batches_mis.csv
    Untracked:  data/ici.vs.eoi_blup_sub_chr-3_peak.marker-JAX00514662_lod.drop-1.5_5.batches.csv
    Untracked:  data/ici.vs.eoi_blup_sub_chr-3_peak.marker-JAX00517218_lod.drop-1.5_5.batches_mis.csv
    Untracked:  data/ici.vs.eoi_blup_sub_chr-3_peak.marker-JAX00517674_lod.drop-1.5_5.batches_mis.csv
    Untracked:  data/ici.vs.eoi_blup_sub_chr-3_peak.marker-JAX00517906_lod.drop-1.5_5.batches_mis.csv
    Untracked:  data/ici.vs.eoi_blup_sub_chr-3_peak.marker-JAX00518418_lod.drop-1.5_5.batches_mis.csv
    Untracked:  data/ici.vs.eoi_blup_sub_chr-3_peak.marker-JAX00518684_lod.drop-1.5_5.batches_mis.csv
    Untracked:  data/ici.vs.eoi_blup_sub_chr-3_peak.marker-JAX00518911r_lod.drop-1.5_5.batches_mis.csv
    Untracked:  data/ici.vs.eoi_blup_sub_chr-3_peak.marker-JAX00520122_lod.drop-1.5_5.batches_mis.csv
    Untracked:  data/ici.vs.eoi_blup_sub_chr-3_peak.marker-JAX00524717_lod.drop-1.5_5.batches_mis.csv
    Untracked:  data/ici.vs.eoi_blup_sub_chr-3_peak.marker-JAX00524828_lod.drop-1.5_5.batches_mis.csv
    Untracked:  data/ici.vs.eoi_blup_sub_chr-3_peak.marker-JAX00525380_lod.drop-1.5_5.batches_mis.csv
    Untracked:  data/ici.vs.eoi_blup_sub_chr-3_peak.marker-JAX00525709_lod.drop-1.5_5.batches_mis.csv
    Untracked:  data/ici.vs.eoi_blup_sub_chr-3_peak.marker-JAX00527291_lod.drop-1.5_5.batches_mis.csv
    Untracked:  data/ici.vs.eoi_blup_sub_chr-3_peak.marker-JAX00527587_lod.drop-1.5_5.batches_mis.csv
    Untracked:  data/ici.vs.eoi_blup_sub_chr-3_peak.marker-JAX00529707_lod.drop-1.5_5.batches_mis.csv
    Untracked:  data/ici.vs.eoi_blup_sub_chr-3_peak.marker-JAX00529707r_lod.drop-1.5_5.batches.csv
    Untracked:  data/ici.vs.eoi_blup_sub_chr-3_peak.marker-JAX00531601_lod.drop-1.5_5.batches_mis.csv
    Untracked:  data/ici.vs.eoi_blup_sub_chr-3_peak.marker-JAX00532122_lod.drop-1.5_5.batches_mis.csv
    Untracked:  data/ici.vs.eoi_blup_sub_chr-3_peak.marker-UNC4673471_lod.drop-1.5_5.batches_mis.csv
    Untracked:  data/ici.vs.eoi_blup_sub_chr-3_peak.marker-UNC4681411_lod.drop-1.5_5.batches_mis.csv
    Untracked:  data/ici.vs.eoi_blup_sub_chr-3_peak.marker-UNC4793677_lod.drop-1.5_5.batches_mis.csv
    Untracked:  data/ici.vs.eoi_blup_sub_chr-3_peak.marker-UNC4799475_lod.drop-1.5_5.batches_mis.csv
    Untracked:  data/ici.vs.eoi_blup_sub_chr-3_peak.marker-UNC4809586_lod.drop-1.5_5.batches_mis.csv
    Untracked:  data/ici.vs.eoi_blup_sub_chr-3_peak.marker-UNC4813186_lod.drop-1.5_5.batches.csv
    Untracked:  data/ici.vs.eoi_blup_sub_chr-3_peak.marker-UNC4813186_lod.drop-1.5_5.batches_mis.csv
    Untracked:  data/ici.vs.eoi_blup_sub_chr-3_peak.marker-UNC4820379_lod.drop-1.5_5.batches_mis.csv
    Untracked:  data/ici.vs.eoi_blup_sub_chr-3_peak.marker-UNC4875653_lod.drop-1.5_5.batches_mis.csv
    Untracked:  data/ici.vs.eoi_blup_sub_chr-3_peak.marker-UNC4909830_lod.drop-1.5_5.batches_mis.csv
    Untracked:  data/ici.vs.eoi_blup_sub_chr-3_peak.marker-UNC4912288_lod.drop-1.5_5.batches_mis.csv
    Untracked:  data/ici.vs.eoi_blup_sub_chr-3_peak.marker-UNC4914130_lod.drop-1.5_5.batches_mis.csv
    Untracked:  data/ici.vs.eoi_blup_sub_chr-3_peak.marker-UNC4923398_lod.drop-1.5_5.batches_mis.csv
    Untracked:  data/ici.vs.eoi_blup_sub_chr-3_peak.marker-UNC4924377_lod.drop-1.5_5.batches.csv
    Untracked:  data/ici.vs.eoi_blup_sub_chr-3_peak.marker-UNC4945186_lod.drop-1.5_5.batches_mis.csv
    Untracked:  data/ici.vs.eoi_blup_sub_chr-3_peak.marker-UNC4947492_lod.drop-1.5_5.batches_mis.csv
    Untracked:  data/ici.vs.eoi_blup_sub_chr-3_peak.marker-UNC4957168_lod.drop-1.5_5.batches.csv
    Untracked:  data/ici.vs.eoi_blup_sub_chr-3_peak.marker-UNC4957859_lod.drop-1.5_5.batches_mis.csv
    Untracked:  data/ici.vs.eoi_blup_sub_chr-3_peak.marker-UNC4964062_lod.drop-1.5_5.batches_mis.csv
    Untracked:  data/ici.vs.eoi_blup_sub_chr-3_peak.marker-UNC4979914_lod.drop-1.5_5.batches_mis.csv
    Untracked:  data/ici.vs.eoi_blup_sub_chr-3_peak.marker-UNC4985064_lod.drop-1.5_5.batches_mis.csv
    Untracked:  data/ici.vs.eoi_blup_sub_chr-3_peak.marker-UNC4990111_lod.drop-1.5_5.batches_mis.csv
    Untracked:  data/ici.vs.eoi_blup_sub_chr-3_peak.marker-UNC4991565_lod.drop-1.5_5.batches_mis.csv
    Untracked:  data/ici.vs.eoi_blup_sub_chr-3_peak.marker-UNC5001723_lod.drop-1.5_5.batches_mis.csv
    Untracked:  data/ici.vs.eoi_blup_sub_chr-3_peak.marker-UNC5008656_lod.drop-1.5_5.batches.csv
    Untracked:  data/ici.vs.eoi_blup_sub_chr-3_peak.marker-UNC5008656_lod.drop-1.5_5.batches_mis.csv
    Untracked:  data/ici.vs.eoi_blup_sub_chr-3_peak.marker-UNC5042361_lod.drop-1.5_5.batches_mis.csv
    Untracked:  data/ici.vs.eoi_blup_sub_chr-3_peak.marker-UNC5055259_lod.drop-1.5_5.batches_mis.csv
    Untracked:  data/ici.vs.eoi_blup_sub_chr-3_peak.marker-UNC5076935_lod.drop-1.5_5.batches_mis.csv
    Untracked:  data/ici.vs.eoi_blup_sub_chr-3_peak.marker-UNC5082757_lod.drop-1.5_5.batches_mis.csv
    Untracked:  data/ici.vs.eoi_blup_sub_chr-3_peak.marker-UNC5083168_lod.drop-1.5_5.batches.csv
    Untracked:  data/ici.vs.eoi_blup_sub_chr-3_peak.marker-UNC5173293_lod.drop-1.5_5.batches_mis.csv
    Untracked:  data/ici.vs.eoi_blup_sub_chr-3_peak.marker-UNC5182782_lod.drop-1.5_5.batches_mis.csv
    Untracked:  data/ici.vs.eoi_blup_sub_chr-3_peak.marker-UNC5192582_lod.drop-1.5_5.batches.csv
    Untracked:  data/ici.vs.eoi_blup_sub_chr-3_peak.marker-UNC5193970_lod.drop-1.5_5.batches_mis.csv
    Untracked:  data/ici.vs.eoi_blup_sub_chr-3_peak.marker-UNC5213888_lod.drop-1.5_snpsqc_5.batches_mis.csv
    Untracked:  data/ici.vs.eoi_blup_sub_chr-3_peak.marker-UNC5240484_lod.drop-1.5_5.batches_mis.csv
    Untracked:  data/ici.vs.eoi_blup_sub_chr-3_peak.marker-UNC5256922_lod.drop-1.5_5.batches_mis.csv
    Untracked:  data/ici.vs.eoi_blup_sub_chr-3_peak.marker-UNC5266664_lod.drop-1.5_5.batches_mis.csv
    Untracked:  data/ici.vs.eoi_blup_sub_chr-3_peak.marker-UNC5266711_lod.drop-1.5_5.batches.csv
    Untracked:  data/ici.vs.eoi_blup_sub_chr-3_peak.marker-UNC5267813_lod.drop-1.5_5.batches_mis.csv
    Untracked:  data/ici.vs.eoi_blup_sub_chr-3_peak.marker-UNC5274554_lod.drop-1.5_5.batches_mis.csv
    Untracked:  data/ici.vs.eoi_blup_sub_chr-3_peak.marker-UNC5277061_lod.drop-1.5_5.batches_mis.csv
    Untracked:  data/ici.vs.eoi_blup_sub_chr-3_peak.marker-UNC5282259_lod.drop-1.5_5.batches_mis.csv
    Untracked:  data/ici.vs.eoi_blup_sub_chr-3_peak.marker-UNC5287977_lod.drop-1.5_5.batches_mis.csv
    Untracked:  data/ici.vs.eoi_blup_sub_chr-3_peak.marker-UNC5301460_lod.drop-1.5_5.batches_mis.csv
    Untracked:  data/ici.vs.eoi_blup_sub_chr-3_peak.marker-UNC5321559_lod.drop-1.5_5.batches_mis.csv
    Untracked:  data/ici.vs.eoi_blup_sub_chr-3_peak.marker-UNC5322455_lod.drop-1.5_5.batches_mis.csv
    Untracked:  data/ici.vs.eoi_blup_sub_chr-3_peak.marker-UNC5328596_lod.drop-1.5_5.batches_mis.csv
    Untracked:  data/ici.vs.eoi_blup_sub_chr-3_peak.marker-UNC5350384_lod.drop-1.5_5.batches_mis.csv
    Untracked:  data/ici.vs.eoi_blup_sub_chr-3_peak.marker-UNC5354724_lod.drop-1.5_5.batches_mis.csv
    Untracked:  data/ici.vs.eoi_blup_sub_chr-3_peak.marker-UNC5375208_lod.drop-1.5_5.batches_mis.csv
    Untracked:  data/ici.vs.eoi_blup_sub_chr-3_peak.marker-UNC5384072_lod.drop-1.5_5.batches_mis.csv
    Untracked:  data/ici.vs.eoi_blup_sub_chr-3_peak.marker-UNC5400314_lod.drop-1.5_5.batches_mis.csv
    Untracked:  data/ici.vs.eoi_blup_sub_chr-3_peak.marker-UNC5433945_lod.drop-1.5_5.batches_mis.csv
    Untracked:  data/ici.vs.eoi_blup_sub_chr-3_peak.marker-UNC5435874_lod.drop-1.5_5.batches_mis.csv
    Untracked:  data/ici.vs.eoi_blup_sub_chr-3_peak.marker-UNC5454839_lod.drop-1.5_5.batches.csv
    Untracked:  data/ici.vs.eoi_blup_sub_chr-3_peak.marker-UNC5454839_lod.drop-1.5_5.batches_mis.csv
    Untracked:  data/ici.vs.eoi_blup_sub_chr-3_peak.marker-UNC5466669_lod.drop-1.5_5.batches_mis.csv
    Untracked:  data/ici.vs.eoi_blup_sub_chr-3_peak.marker-UNC5517972_lod.drop-1.5_5.batches_mis.csv
    Untracked:  data/ici.vs.eoi_blup_sub_chr-3_peak.marker-UNC5522257_lod.drop-1.5_5.batches_mis.csv
    Untracked:  data/ici.vs.eoi_blup_sub_chr-3_peak.marker-UNC5523265_lod.drop-1.5_5.batches_mis.csv
    Untracked:  data/ici.vs.eoi_blup_sub_chr-3_peak.marker-UNC5532424_lod.drop-1.5_5.batches_mis.csv
    Untracked:  data/ici.vs.eoi_blup_sub_chr-3_peak.marker-UNC5553882_lod.drop-1.5_5.batches_mis.csv
    Untracked:  data/ici.vs.eoi_blup_sub_chr-3_peak.marker-UNC5572051_lod.drop-1.5_5.batches_mis.csv
    Untracked:  data/ici.vs.eoi_blup_sub_chr-3_peak.marker-UNC5582753_lod.drop-1.5_5.batches.csv
    Untracked:  data/ici.vs.eoi_blup_sub_chr-3_peak.marker-UNC5589260_lod.drop-1.5_5.batches_mis.csv
    Untracked:  data/ici.vs.eoi_blup_sub_chr-3_peak.marker-UNC5595555_lod.drop-1.5_5.batches_mis.csv
    Untracked:  data/ici.vs.eoi_blup_sub_chr-3_peak.marker-UNC5623239_lod.drop-1.5_5.batches_mis.csv
    Untracked:  data/ici.vs.eoi_blup_sub_chr-3_peak.marker-UNC5639830_lod.drop-1.5_5.batches_mis.csv
    Untracked:  data/ici.vs.eoi_blup_sub_chr-3_peak.marker-UNC5645844_lod.drop-1.5_5.batches.csv
    Untracked:  data/ici.vs.eoi_blup_sub_chr-3_peak.marker-UNC5645844_lod.drop-1.5_5.batches_mis.csv
    Untracked:  data/ici.vs.eoi_blup_sub_chr-3_peak.marker-UNC5680234_lod.drop-1.5_5.batches_mis.csv
    Untracked:  data/ici.vs.eoi_blup_sub_chr-3_peak.marker-UNC5682314_lod.drop-1.5_5.batches_mis.csv
    Untracked:  data/ici.vs.eoi_blup_sub_chr-3_peak.marker-UNC5725356_lod.drop-1.5_5.batches_mis.csv
    Untracked:  data/ici.vs.eoi_blup_sub_chr-3_peak.marker-UNC5727351_lod.drop-1.5_5.batches_mis.csv
    Untracked:  data/ici.vs.eoi_blup_sub_chr-3_peak.marker-UNC5734420_lod.drop-1.5_5.batches_mis.csv
    Untracked:  data/ici.vs.eoi_blup_sub_chr-3_peak.marker-UNC5743257_lod.drop-1.5_5.batches_mis.csv
    Untracked:  data/ici.vs.eoi_blup_sub_chr-3_peak.marker-UNC5751567_lod.drop-1.5_5.batches_mis.csv
    Untracked:  data/ici.vs.eoi_blup_sub_chr-3_peak.marker-UNC5752623_lod.drop-1.5_5.batches_mis.csv
    Untracked:  data/ici.vs.eoi_blup_sub_chr-3_peak.marker-UNC5776974_lod.drop-1.5_5.batches_mis.csv
    Untracked:  data/ici.vs.eoi_blup_sub_chr-3_peak.marker-UNC5778977_lod.drop-1.5_5.batches_mis.csv
    Untracked:  data/ici.vs.eoi_blup_sub_chr-3_peak.marker-UNC5788507_lod.drop-1.5_5.batches_mis.csv
    Untracked:  data/ici.vs.eoi_blup_sub_chr-3_peak.marker-UNC5792468_lod.drop-1.5_5.batches_mis.csv
    Untracked:  data/ici.vs.eoi_blup_sub_chr-3_peak.marker-UNC5794598_lod.drop-1.5_5.batches_mis.csv
    Untracked:  data/ici.vs.eoi_blup_sub_chr-3_peak.marker-UNC5795561_lod.drop-1.5_5.batches_mis.csv
    Untracked:  data/ici.vs.eoi_blup_sub_chr-3_peak.marker-UNC5802298_lod.drop-1.5_5.batches_mis.csv
    Untracked:  data/ici.vs.eoi_blup_sub_chr-3_peak.marker-UNC5806628_lod.drop-1.5_5.batches_mis.csv
    Untracked:  data/ici.vs.eoi_blup_sub_chr-3_peak.marker-UNC5808246_lod.drop-1.5_5.batches.csv
    Untracked:  data/ici.vs.eoi_blup_sub_chr-3_peak.marker-UNC5817478_lod.drop-1.5_5.batches_mis.csv
    Untracked:  data/ici.vs.eoi_blup_sub_chr-3_peak.marker-UNC5836999_lod.drop-1.5_5.batches_mis.csv
    Untracked:  data/ici.vs.eoi_blup_sub_chr-3_peak.marker-UNC5841501_lod.drop-1.5_5.batches_mis.csv
    Untracked:  data/ici.vs.eoi_blup_sub_chr-3_peak.marker-UNC5847927_lod.drop-1.5_5.batches_mis.csv
    Untracked:  data/ici.vs.eoi_blup_sub_chr-3_peak.marker-UNC5966417_lod.drop-1.5_5.batches_mis.csv
    Untracked:  data/ici.vs.eoi_blup_sub_chr-3_peak.marker-UNCHS008129_lod.drop-1.5_5.batches_mis.csv
    Untracked:  data/ici.vs.eoi_blup_sub_chr-3_peak.marker-UNCHS008173_lod.drop-1.5_5.batches_mis.csv
    Untracked:  data/ici.vs.eoi_blup_sub_chr-3_peak.marker-UNCHS008182_lod.drop-1.5_5.batches.csv
    Untracked:  data/ici.vs.eoi_blup_sub_chr-3_peak.marker-UNCHS008193_lod.drop-1.5_5.batches_mis.csv
    Untracked:  data/ici.vs.eoi_blup_sub_chr-3_peak.marker-UNCHS008209_lod.drop-1.5_5.batches_mis.csv
    Untracked:  data/ici.vs.eoi_blup_sub_chr-3_peak.marker-UNCHS008215_lod.drop-1.5_5.batches_mis.csv
    Untracked:  data/ici.vs.eoi_blup_sub_chr-3_peak.marker-UNCHS008219_lod.drop-1.5_5.batches_mis.csv
    Untracked:  data/ici.vs.eoi_blup_sub_chr-3_peak.marker-UNCHS008231_lod.drop-1.5_5.batches_mis.csv
    Untracked:  data/ici.vs.eoi_blup_sub_chr-3_peak.marker-UNCHS008235_lod.drop-1.5_5.batches_mis.csv
    Untracked:  data/ici.vs.eoi_blup_sub_chr-3_peak.marker-UNCHS008242_lod.drop-1.5_5.batches.csv
    Untracked:  data/ici.vs.eoi_blup_sub_chr-3_peak.marker-UNCHS008253_lod.drop-1.5_5.batches_mis.csv
    Untracked:  data/ici.vs.eoi_blup_sub_chr-3_peak.marker-UNCHS008256_lod.drop-1.5_5.batches_mis.csv
    Untracked:  data/ici.vs.eoi_blup_sub_chr-3_peak.marker-UNCHS008259_lod.drop-1.5_5.batches_mis.csv
    Untracked:  data/ici.vs.eoi_blup_sub_chr-3_peak.marker-UNCHS008271_lod.drop-1.5_5.batches_mis.csv
    Untracked:  data/ici.vs.eoi_blup_sub_chr-3_peak.marker-UNCHS008286_lod.drop-1.5_5.batches.csv
    Untracked:  data/ici.vs.eoi_blup_sub_chr-3_peak.marker-UNCHS008286_lod.drop-1.5_5.batches_mis.csv
    Untracked:  data/ici.vs.eoi_blup_sub_chr-3_peak.marker-UNCHS008311_lod.drop-1.5_5.batches_mis.csv
    Untracked:  data/ici.vs.eoi_blup_sub_chr-3_peak.marker-UNCHS008317_lod.drop-1.5_5.batches_mis.csv
    Untracked:  data/ici.vs.eoi_blup_sub_chr-3_peak.marker-UNCHS008372_lod.drop-1.5_5.batches_mis.csv
    Untracked:  data/ici.vs.eoi_blup_sub_chr-3_peak.marker-UNCHS008379_lod.drop-1.5_5.batches_mis.csv
    Untracked:  data/ici.vs.eoi_blup_sub_chr-3_peak.marker-UNCHS008410_lod.drop-1.5_5.batches_mis.csv
    Untracked:  data/ici.vs.eoi_blup_sub_chr-3_peak.marker-UNCHS008415_lod.drop-1.5_5.batches_mis.csv
    Untracked:  data/ici.vs.eoi_blup_sub_chr-3_peak.marker-UNCHS008431_lod.drop-1.5_5.batches_mis.csv
    Untracked:  data/ici.vs.eoi_blup_sub_chr-3_peak.marker-UNCHS008487_lod.drop-1.5_snpsqc_5.batches.csv
    Untracked:  data/ici.vs.eoi_blup_sub_chr-3_peak.marker-UNCHS008487_lod.drop-1.5_snpsqc_dis_no-x_updated_5.batches.csv
    Untracked:  data/ici.vs.eoi_blup_sub_chr-3_peak.marker-UNCHS008487_lod.drop-1.5_snpsqc_dis_no-x_updated_5.batches_mis.csv
    Untracked:  data/ici.vs.eoi_blup_sub_chr-3_peak.marker-UNCHS008511_lod.drop-1.5_snpsqc_5.batches_mis.csv
    Untracked:  data/ici.vs.eoi_blup_sub_chr-3_peak.marker-UNCHS008609_lod.drop-1.5_5.batches_mis.csv
    Untracked:  data/ici.vs.eoi_blup_sub_chr-3_peak.marker-UNCHS008614_lod.drop-1.5_5.batches.csv
    Untracked:  data/ici.vs.eoi_blup_sub_chr-3_peak.marker-UNCHS008649_lod.drop-1.5_5.batches_mis.csv
    Untracked:  data/ici.vs.eoi_blup_sub_chr-3_peak.marker-UNCHS008651_lod.drop-1.5_5.batches.csv
    Untracked:  data/ici.vs.eoi_blup_sub_chr-3_peak.marker-UNCHS008656_lod.drop-1.5_5.batches_mis.csv
    Untracked:  data/ici.vs.eoi_blup_sub_chr-3_peak.marker-UNCHS008723_lod.drop-1.5_5.batches_mis.csv
    Untracked:  data/ici.vs.eoi_blup_sub_chr-3_peak.marker-UNCHS008802_lod.drop-1.5_5.batches_mis.csv
    Untracked:  data/ici.vs.eoi_blup_sub_chr-3_peak.marker-UNCHS008803_lod.drop-1.5_5.batches_mis.csv
    Untracked:  data/ici.vs.eoi_blup_sub_chr-3_peak.marker-UNCHS008808_lod.drop-1.5_5.batches_mis.csv
    Untracked:  data/ici.vs.eoi_blup_sub_chr-3_peak.marker-UNCHS008815_lod.drop-1.5_5.batches.csv
    Untracked:  data/ici.vs.eoi_blup_sub_chr-3_peak.marker-UNCHS008815_lod.drop-1.5_5.batches_mis.csv
    Untracked:  data/ici.vs.eoi_blup_sub_chr-3_peak.marker-UNCHS008834_lod.drop-1.5_5.batches_mis.csv
    Untracked:  data/ici.vs.eoi_blup_sub_chr-3_peak.marker-UNCHS008838_lod.drop-1.5_5.batches_mis.csv
    Untracked:  data/ici.vs.eoi_blup_sub_chr-3_peak.marker-UNCHS008914_lod.drop-1.5_5.batches_mis.csv
    Untracked:  data/ici.vs.eoi_blup_sub_chr-3_peak.marker-UNCHS008928_lod.drop-1.5_5.batches_mis.csv
    Untracked:  data/ici.vs.eoi_blup_sub_chr-3_peak.marker-UNCHS008948_lod.drop-1.5_5.batches_mis.csv
    Untracked:  data/ici.vs.eoi_blup_sub_chr-3_peak.marker-UNCHS008959_lod.drop-1.5_5.batches_mis.csv
    Untracked:  data/ici.vs.eoi_blup_sub_chr-3_peak.marker-UNCHS008973_lod.drop-1.5_5.batches_mis.csv
    Untracked:  data/ici.vs.eoi_blup_sub_chr-3_peak.marker-UNCHS008987_lod.drop-1.5_5.batches_mis.csv
    Untracked:  data/ici.vs.eoi_blup_sub_chr-3_peak.marker-UNCHS009001_lod.drop-1.5_5.batches_mis.csv
    Untracked:  data/ici.vs.eoi_blup_sub_chr-3_peak.marker-UNCHS009025_lod.drop-1.5_5.batches_mis.csv
    Untracked:  data/ici.vs.eoi_blup_sub_chr-3_peak.marker-UNCHS009034_lod.drop-1.5_5.batches_mis.csv
    Untracked:  data/ici.vs.eoi_blup_sub_chr-3_peak.marker-UNCHS009087_lod.drop-1.5_5.batches_mis.csv
    Untracked:  data/ici.vs.eoi_blup_sub_chr-3_peak.marker-UNCHS009109_lod.drop-1.5_5.batches_mis.csv
    Untracked:  data/ici.vs.eoi_blup_sub_chr-3_peak.marker-UNCHS009142_lod.drop-1.5_5.batches_mis.csv
    Untracked:  data/ici.vs.eoi_blup_sub_chr-3_peak.marker-UNCHS009207_lod.drop-1.5_5.batches_mis.csv
    Untracked:  data/ici.vs.eoi_blup_sub_chr-3_peak.marker-UNCHS009216_lod.drop-1.5_5.batches_mis.csv
    Untracked:  data/ici.vs.eoi_blup_sub_chr-3_peak.marker-UNCHS009224_lod.drop-1.5_5.batches.csv
    Untracked:  data/ici.vs.eoi_blup_sub_chr-3_peak.marker-UNCHS009306_lod.drop-1.5_5.batches_mis.csv
    Untracked:  data/ici.vs.eoi_blup_sub_chr-3_peak.marker-UNCHS009316_lod.drop-1.5_5.batches_mis.csv
    Untracked:  data/ici.vs.eoi_blup_sub_chr-3_peak.marker-UNCHS009364_lod.drop-1.5_5.batches_mis.csv
    Untracked:  data/ici.vs.eoi_blup_sub_chr-3_peak.marker-UNCHS009377_lod.drop-1.5_5.batches_mis.csv
    Untracked:  data/ici.vs.eoi_blup_sub_chr-3_peak.marker-UNCHS009382_lod.drop-1.5_5.batches_mis.csv
    Untracked:  data/ici.vs.eoi_blup_sub_chr-3_peak.marker-UNCHS009383_lod.drop-1.5_5.batches.csv
    Untracked:  data/ici.vs.eoi_blup_sub_chr-3_peak.marker-UNCHS009387_lod.drop-1.5_5.batches_mis.csv
    Untracked:  data/ici.vs.eoi_blup_sub_chr-3_peak.marker-UNCHS009395_lod.drop-1.5_5.batches_mis.csv
    Untracked:  data/ici.vs.eoi_blup_sub_chr-3_peak.marker-UNCHS009413_lod.drop-1.5_5.batches_mis.csv
    Untracked:  data/ici.vs.eoi_blup_sub_chr-3_peak.marker-UNCHS009428_lod.drop-1.5_5.batches_mis.csv
    Untracked:  data/ici.vs.eoi_blup_sub_chr-3_peak.marker-UNCHS009463_lod.drop-1.5_5.batches_mis.csv
    Untracked:  data/ici.vs.eoi_blup_sub_chr-3_peak.marker-UNCHS009503_lod.drop-1.5_5.batches_mis.csv
    Untracked:  data/ici.vs.eoi_blup_sub_chr-3_peak.marker-UNCHS009532_lod.drop-1.5_5.batches_mis.csv
    Untracked:  data/ici.vs.eoi_blup_sub_chr-3_peak.marker-UNCHS009541_lod.drop-1.5_5.batches_mis.csv
    Untracked:  data/ici.vs.eoi_blup_sub_chr-3_peak.marker-UNCHS009560_lod.drop-1.5_5.batches_mis.csv
    Untracked:  data/ici.vs.eoi_blup_sub_chr-3_peak.marker-UNCHS009574_lod.drop-1.5_5.batches_mis.csv
    Untracked:  data/ici.vs.eoi_blup_sub_chr-3_peak.marker-UNCJPD001170_lod.drop-1.5_5.batches_mis.csv
    Untracked:  data/ici.vs.eoi_blup_sub_chr-3_peak.marker-UNCJPD001198_lod.drop-1.5_5.batches_mis.csv
    Untracked:  data/ici.vs.eoi_blup_sub_chr-3_peak.marker-UNCJPD001276_lod.drop-1.5_5.batches_mis.csv
    Untracked:  data/ici.vs.eoi_blup_sub_chr-3_peak.marker-UNCJPD001315_lod.drop-1.5_5.batches_mis.csv
    Untracked:  data/ici.vs.eoi_blup_sub_chr-3_peak.marker-UNCJPD001355_lod.drop-1.5_5.batches_mis.csv
    Untracked:  data/ici.vs.eoi_blup_sub_chr-3_peak.marker-UNCJPD001373_lod.drop-1.5_5.batches_mis.csv
    Untracked:  data/ici.vs.eoi_blup_sub_chr-3_peak.marker-UNCJPD001382_lod.drop-1.5_5.batches_mis.csv
    Untracked:  data/ici.vs.eoi_blup_sub_chr-4_peak.marker-UNC6851205_lod.drop-1.5_5.batches_mis.csv
    Untracked:  data/ici.vs.eoi_blup_sub_chr-4_peak.marker-UNC8250659_lod.drop-1.5_snpsqc_dis_no-x_updated_5.batches.csv
    Untracked:  data/ici.vs.eoi_blup_sub_chr-4_peak.marker-UNC8299912_lod.drop-1.5_5.batches_mis.csv
    Untracked:  data/ici.vs.eoi_blup_sub_chr-4_peak.marker-UNC8300499_lod.drop-1.5_5.batches.csv
    Untracked:  data/ici.vs.eoi_blup_sub_chr-4_peak.marker-UNC8381981_lod.drop-1.5_5.batches_mis.csv
    Untracked:  data/ici.vs.eoi_blup_sub_chr-4_peak.marker-UNC8436434_lod.drop-1.5_5.batches_mis.csv
    Untracked:  data/ici.vs.eoi_blup_sub_chr-4_peak.marker-UNC8439633_lod.drop-1.5_5.batches_mis.csv
    Untracked:  data/ici.vs.eoi_blup_sub_chr-4_peak.marker-UNC8530763_lod.drop-1.5_5.batches_mis.csv
    Untracked:  data/ici.vs.eoi_blup_sub_chr-4_peak.marker-UNCHS012948_lod.drop-1.5_snpsqc_dis_no-x_updated_5.batches_mis.csv
    Untracked:  data/ici.vs.eoi_blup_sub_chr-4_peak.marker-UNCHS012976_lod.drop-1.5_5.batches_mis.csv
    Untracked:  data/ici.vs.eoi_blup_sub_chr-4_peak.marker-UNCHS013118_lod.drop-1.5_5.batches.csv
    Untracked:  data/ici.vs.eoi_blup_sub_chr-4_peak.marker-UNCHS013118_lod.drop-1.5_snpsqc_5.batches.csv
    Untracked:  data/ici.vs.eoi_blup_sub_chr-4_peak.marker-UNCHS013118_lod.drop-1.5_snpsqc_5.batches_mis.csv
    Untracked:  data/ici.vs.eoi_blup_sub_chr-4_peak.marker-UNCHS013130_lod.drop-1.5_5.batches_mis.csv
    Untracked:  data/ici.vs.eoi_blup_sub_chr-4_peak.marker-UNCHS013139_lod.drop-1.5_5.batches_mis.csv
    Untracked:  data/ici.vs.eoi_blup_sub_chr-4_peak.marker-UNCHS013304_lod.drop-1.5_5.batches.csv
    Untracked:  data/ici.vs.eoi_blup_sub_chr-5_peak.marker-JAX00590770_lod.drop-1.5_5.batches_mis.csv
    Untracked:  data/ici.vs.eoi_blup_sub_chr-5_peak.marker-UNC10044126_lod.drop-1.5_5.batches_mis.csv
    Untracked:  data/ici.vs.eoi_blup_sub_chr-7_peak.marker-ICR1830_lod.drop-1.5_5.batches_mis.csv
    Untracked:  data/ici.vs.eoi_blup_sub_chr-7_peak.marker-UNC13170794_lod.drop-1.5_5.batches_mis.csv
    Untracked:  data/ici.vs.eoi_blup_sub_chr-7_peak.marker-UNCHS019480_lod.drop-1.5_5.batches.csv
    Untracked:  data/ici.vs.eoi_blup_sub_chr-7_peak.marker-UNCHS020066_lod.drop-1.5_snpsqc_5.batches_mis.csv
    Untracked:  data/ici.vs.eoi_blup_sub_chr-7_peak.marker-UNCHS020066_lod.drop-1.5_snpsqc_dis_no-x_updated_5.batches_mis.csv
    Untracked:  data/ici.vs.eoi_blup_sub_chr-7_peak.marker-cr31snv87_lod.drop-1.5_5.batches_mis.csv
    Untracked:  data/ici.vs.eoi_blup_sub_chr-8_peak.marker-JAX00667121_lod.drop-1.5_5.batches_mis.csv
    Untracked:  data/ici.vs.eoi_blup_sub_chr-8_peak.marker-UNC15430711_lod.drop-1.5_5.batches_mis.csv
    Untracked:  data/ici.vs.eoi_blup_sub_chr-8_peak.marker-UNC15524531_lod.drop-1.5_5.batches_mis.csv
    Untracked:  data/ici.vs.eoi_blup_sub_chr-8_peak.marker-UNC15524531_lod.drop-1.5_snpsqc_5.batches_mis.csv
    Untracked:  data/ici.vs.eoi_blup_sub_chr-8_peak.marker-UNC15524531_lod.drop-1.5_snpsqc_dis_no-x_updated_5.batches_mis.csv
    Untracked:  data/ici.vs.eoi_blup_sub_chr-9_peak.marker-UNC17203329_lod.drop-1.5_5.batches_mis.csv
    Untracked:  data/ici.vs.eoi_blup_sub_chr-9_peak.marker-UNC17203597_lod.drop-1.5_snpsqc_5.batches_mis.csv
    Untracked:  data/ici.vs.eoi_blup_sub_chr-9_peak.marker-UNCHS024593_lod.drop-1.5_5.batches_mis.csv
    Untracked:  data/ici.vs.eoi_blup_sub_chr-X_peak.marker-UNC30627130_lod.drop-1.5_5.batches_mis.csv
    Untracked:  data/ici.vs.eoi_blup_sub_chr-X_peak.marker-UNCHS048313_lod.drop-1.5_5.batches_mis.csv
    Untracked:  data/ici.vs.eoi_blup_sub_chr-X_peak.marker-UNCHS048314_lod.drop-1.5_snpsqc_5.batches_mis.csv
    Untracked:  data/ici.vs.eoi_blup_sub_chr-X_peak.marker-UNCHS048314_lod.drop-1.5_snpsqc_dis_no-x_updated_5.batches_mis.csv
    Untracked:  data/ici.vs.eoi_fitqtl_peaks_additive_5.batches.txt
    Untracked:  data/ici.vs.eoi_fitqtl_peaks_additive_5.batches_mis.txt
    Untracked:  data/ici.vs.eoi_fitqtl_peaks_additive_snpsqc_5.batches.txt
    Untracked:  data/ici.vs.eoi_fitqtl_peaks_additive_snpsqc_dis_no-x_updated_5.batches.txt
    Untracked:  data/ici.vs.eoi_fitqtl_peaks_additive_snpsqc_dis_no-x_updated_5.batches_mis.txt
    Untracked:  data/ici.vs.eoi_fitqtl_peaks_interacting_5.batches.txt
    Untracked:  data/ici.vs.eoi_fitqtl_peaks_interacting_5.batches_mis.txt
    Untracked:  data/ici.vs.eoi_fitqtl_peaks_interacting_snpsqc_5.batches.txt
    Untracked:  data/ici.vs.eoi_fitqtl_peaks_interacting_snpsqc_dis_no-x_updated_5.batches.txt
    Untracked:  data/ici.vs.eoi_fitqtl_peaks_interacting_snpsqc_dis_no-x_updated_5.batches_mis.txt
    Untracked:  data/ici.vs.eoi_fitqtl_peaks_sex_additive_5.batches.txt
    Untracked:  data/ici.vs.eoi_fitqtl_peaks_sex_additive_5.batches_mis.txt
    Untracked:  data/ici.vs.eoi_fitqtl_peaks_sex_additive_snpsqc_5.batches.txt
    Untracked:  data/ici.vs.eoi_fitqtl_peaks_sex_additive_snpsqc_dis_no-x_updated_5.batches.txt
    Untracked:  data/ici.vs.eoi_fitqtl_peaks_sex_additive_snpsqc_dis_no-x_updated_5.batches_mis.txt
    Untracked:  data/ici.vs.eoi_fitqtl_peaks_sex_interacting_5.batches.txt
    Untracked:  data/ici.vs.eoi_fitqtl_peaks_sex_interacting_5.batches_mis.txt
    Untracked:  data/ici.vs.eoi_fitqtl_peaks_sex_interacting_snpsqc_5.batches.txt
    Untracked:  data/ici.vs.eoi_fitqtl_peaks_sex_interacting_snpsqc_dis_no-x_updated_5.batches.txt
    Untracked:  data/ici.vs.eoi_fitqtl_peaks_sex_interacting_snpsqc_dis_no-x_updated_5.batches_mis.mis.txt
    Untracked:  data/ici.vs.eoi_genes_chr-10_peak.marker-ICR4295_lod.drop-1.5_5.batches_mis.csv
    Untracked:  data/ici.vs.eoi_genes_chr-10_peak.marker-JAX00292499_lod.drop-1.5_5.batches.csv
    Untracked:  data/ici.vs.eoi_genes_chr-10_peak.marker-JAX00292927_lod.drop-1.5_5.batches.csv
    Untracked:  data/ici.vs.eoi_genes_chr-10_peak.marker-JAX00292927_lod.drop-1.5_snpsqc_5.batches.csv
    Untracked:  data/ici.vs.eoi_genes_chr-10_peak.marker-JAX00292927_lod.drop-1.5_snpsqc_5.batches_mis.csv
    Untracked:  data/ici.vs.eoi_genes_chr-10_peak.marker-UNC18154330_lod.drop-1.5_5.batches.csv
    Untracked:  data/ici.vs.eoi_genes_chr-10_peak.marker-UNC18195765_lod.drop-1.5_5.batches.csv
    Untracked:  data/ici.vs.eoi_genes_chr-10_peak.marker-UNC18216614_lod.drop-1.5_5.batches.csv
    Untracked:  data/ici.vs.eoi_genes_chr-10_peak.marker-UNC18240977_lod.drop-1.5_snpsqc_dis_no-x_updated_5.batches.csv
    Untracked:  data/ici.vs.eoi_genes_chr-10_peak.marker-UNC18311938_lod.drop-1.5_5.batches.csv
    Untracked:  data/ici.vs.eoi_genes_chr-10_peak.marker-UNC18856953_lod.drop-1.5_5.batches_mis.csv
    Untracked:  data/ici.vs.eoi_genes_chr-10_peak.marker-UNCHS028236_lod.drop-1.5_snpsqc_5.batches_mis.csv
    Untracked:  data/ici.vs.eoi_genes_chr-10_peak.marker-UNCHS028536_lod.drop-1.5_snpsqc_dis_no-x_updated_5.batches_mis.csv
    Untracked:  data/ici.vs.eoi_genes_chr-10_peak.marker-UNCJPD004316_lod.drop-1.5_5.batches_mis.csv
    Untracked:  data/ici.vs.eoi_genes_chr-10_peak.marker-UNCJPD009532_lod.drop-1.5_5.batches_mis.csv
    Untracked:  data/ici.vs.eoi_genes_chr-11_peak.marker-UNC19155926_lod.drop-1.5_5.batches_mis.csv
    Untracked:  data/ici.vs.eoi_genes_chr-11_peak.marker-UNC19481366_lod.drop-1.5_snpsqc_5.batches_mis.csv
    Untracked:  data/ici.vs.eoi_genes_chr-11_peak.marker-UNC19970181_lod.drop-1.5_5.batches_mis.csv
    Untracked:  data/ici.vs.eoi_genes_chr-11_peak.marker-UNC19970181_lod.drop-1.5_snpsqc_5.batches_mis.csv
    Untracked:  data/ici.vs.eoi_genes_chr-11_peak.marker-UNCJPD004792_lod.drop-1.5_5.batches_mis.csv
    Untracked:  data/ici.vs.eoi_genes_chr-12_peak.marker-ICR4497_lod.drop-1.5_5.batches_mis.csv
    Untracked:  data/ici.vs.eoi_genes_chr-12_peak.marker-ICR499_lod.drop-1.5_snpsqc_5.batches_mis.csv
    Untracked:  data/ici.vs.eoi_genes_chr-13_peak.marker-UNCHS036773_lod.drop-1.5_snpsqc_5.batches_mis.csv
    Untracked:  data/ici.vs.eoi_genes_chr-13_peak.marker-UNCHS036964_lod.drop-1.5_5.batches_mis.csv
    Untracked:  data/ici.vs.eoi_genes_chr-14_peak.marker-UNC24056202_lod.drop-1.5_5.batches_mis.csv
    Untracked:  data/ici.vs.eoi_genes_chr-14_peak.marker-UNC24056202_lod.drop-1.5_snpsqc_5.batches_mis.csv
    Untracked:  data/ici.vs.eoi_genes_chr-15_peak.marker-UNC26070435_lod.drop-1.5_snpsqc_dis_no-x_updated_5.batches_mis.csv
    Untracked:  data/ici.vs.eoi_genes_chr-17_peak.marker-UNCHS044241_lod.drop-1.5_5.batches_mis.csv
    Untracked:  data/ici.vs.eoi_genes_chr-17_peak.marker-UNCHS044241_lod.drop-1.5_snpsqc_5.batches_mis.csv
    Untracked:  data/ici.vs.eoi_genes_chr-17_peak.marker-UNCrs47191360_lod.drop-1.5_5.batches_mis.csv
    Untracked:  data/ici.vs.eoi_genes_chr-17_peak.marker-UNCrs47191360_lod.drop-1.5_snpsqc_dis_no-x_updated_5.batches_mis.csv
    Untracked:  data/ici.vs.eoi_genes_chr-18_peak.marker-UNCHS045343_lod.drop-1.5_snpsqc_5.batches_mis.csv
    Untracked:  data/ici.vs.eoi_genes_chr-18_peak.marker-UNCHS045344_lod.drop-1.5_5.batches_mis.csv
    Untracked:  data/ici.vs.eoi_genes_chr-19_peak.marker-UNCJPD007087_lod.drop-1.5_5.batches_mis.csv
    Untracked:  data/ici.vs.eoi_genes_chr-1_peak.marker-ICR010_lod.drop-1.5_5.batches.csv
    Untracked:  data/ici.vs.eoi_genes_chr-1_peak.marker-ICR010_lod.drop-1.5_5.batches_mis.csv
    Untracked:  data/ici.vs.eoi_genes_chr-2_peak.marker-UNC4609527_lod.drop-1.5_snpsqc_dis_no-x_updated_5.batches_mis.csv
    Untracked:  data/ici.vs.eoi_genes_chr-2_peak.marker-UNC4609660_lod.drop-1.5_5.batches_mis.csv
    Untracked:  data/ici.vs.eoi_genes_chr-3_peak.marker-B6_rs31740628_lod.drop-1.5_5.batches_mis.csv
    Untracked:  data/ici.vs.eoi_genes_chr-3_peak.marker-ICR1269_lod.drop-1.5_5.batches_mis.csv
    Untracked:  data/ici.vs.eoi_genes_chr-3_peak.marker-ICR1338_lod.drop-1.5_5.batches.csv
    Untracked:  data/ici.vs.eoi_genes_chr-3_peak.marker-ICR5325_lod.drop-1.5_5.batches_mis.csv
    Untracked:  data/ici.vs.eoi_genes_chr-3_peak.marker-ICR5347_lod.drop-1.5_5.batches.csv
    Untracked:  data/ici.vs.eoi_genes_chr-3_peak.marker-ICR5348_lod.drop-1.5_5.batches_mis.csv
    Untracked:  data/ici.vs.eoi_genes_chr-3_peak.marker-JAX00104971_lod.drop-1.5_5.batches_mis.csv
    Untracked:  data/ici.vs.eoi_genes_chr-3_peak.marker-JAX00105059_lod.drop-1.5_5.batches.csv
    Untracked:  data/ici.vs.eoi_genes_chr-3_peak.marker-JAX00105203_lod.drop-1.5_5.batches_mis.csv
    Untracked:  data/ici.vs.eoi_genes_chr-3_peak.marker-JAX00106957_lod.drop-1.5_5.batches.csv
    Untracked:  data/ici.vs.eoi_genes_chr-3_peak.marker-JAX00106957_lod.drop-1.5_5.batches_mis.csv
    Untracked:  data/ici.vs.eoi_genes_chr-3_peak.marker-JAX00107930_lod.drop-1.5_5.batches_mis.csv
    Untracked:  data/ici.vs.eoi_genes_chr-3_peak.marker-JAX00107980_lod.drop-1.5_5.batches_mis.csv
    Untracked:  data/ici.vs.eoi_genes_chr-3_peak.marker-JAX00108068_lod.drop-1.5_5.batches_mis.csv
    Untracked:  data/ici.vs.eoi_genes_chr-3_peak.marker-JAX00108114_lod.drop-1.5_5.batches_mis.csv
    Untracked:  data/ici.vs.eoi_genes_chr-3_peak.marker-JAX00109750_lod.drop-1.5_5.batches_mis.csv
    Untracked:  data/ici.vs.eoi_genes_chr-3_peak.marker-JAX00189245_lod.drop-1.5_5.batches_mis.csv
    Untracked:  data/ici.vs.eoi_genes_chr-3_peak.marker-JAX00514662_lod.drop-1.5_5.batches.csv
    Untracked:  data/ici.vs.eoi_genes_chr-3_peak.marker-JAX00517218_lod.drop-1.5_5.batches_mis.csv
    Untracked:  data/ici.vs.eoi_genes_chr-3_peak.marker-JAX00517674_lod.drop-1.5_5.batches_mis.csv
    Untracked:  data/ici.vs.eoi_genes_chr-3_peak.marker-JAX00517906_lod.drop-1.5_5.batches_mis.csv
    Untracked:  data/ici.vs.eoi_genes_chr-3_peak.marker-JAX00518418_lod.drop-1.5_5.batches_mis.csv
    Untracked:  data/ici.vs.eoi_genes_chr-3_peak.marker-JAX00518684_lod.drop-1.5_5.batches_mis.csv
    Untracked:  data/ici.vs.eoi_genes_chr-3_peak.marker-JAX00518911r_lod.drop-1.5_5.batches_mis.csv
    Untracked:  data/ici.vs.eoi_genes_chr-3_peak.marker-JAX00520122_lod.drop-1.5_5.batches_mis.csv
    Untracked:  data/ici.vs.eoi_genes_chr-3_peak.marker-JAX00524717_lod.drop-1.5_5.batches_mis.csv
    Untracked:  data/ici.vs.eoi_genes_chr-3_peak.marker-JAX00524828_lod.drop-1.5_5.batches_mis.csv
    Untracked:  data/ici.vs.eoi_genes_chr-3_peak.marker-JAX00525380_lod.drop-1.5_5.batches_mis.csv
    Untracked:  data/ici.vs.eoi_genes_chr-3_peak.marker-JAX00525709_lod.drop-1.5_5.batches_mis.csv
    Untracked:  data/ici.vs.eoi_genes_chr-3_peak.marker-JAX00527291_lod.drop-1.5_5.batches_mis.csv
    Untracked:  data/ici.vs.eoi_genes_chr-3_peak.marker-JAX00527587_lod.drop-1.5_5.batches_mis.csv
    Untracked:  data/ici.vs.eoi_genes_chr-3_peak.marker-JAX00529707_lod.drop-1.5_5.batches_mis.csv
    Untracked:  data/ici.vs.eoi_genes_chr-3_peak.marker-JAX00529707r_lod.drop-1.5_5.batches.csv
    Untracked:  data/ici.vs.eoi_genes_chr-3_peak.marker-JAX00531601_lod.drop-1.5_5.batches_mis.csv
    Untracked:  data/ici.vs.eoi_genes_chr-3_peak.marker-JAX00532122_lod.drop-1.5_5.batches_mis.csv
    Untracked:  data/ici.vs.eoi_genes_chr-3_peak.marker-UNC4673471_lod.drop-1.5_5.batches_mis.csv
    Untracked:  data/ici.vs.eoi_genes_chr-3_peak.marker-UNC4681411_lod.drop-1.5_5.batches_mis.csv
    Untracked:  data/ici.vs.eoi_genes_chr-3_peak.marker-UNC4793677_lod.drop-1.5_5.batches_mis.csv
    Untracked:  data/ici.vs.eoi_genes_chr-3_peak.marker-UNC4799475_lod.drop-1.5_5.batches_mis.csv
    Untracked:  data/ici.vs.eoi_genes_chr-3_peak.marker-UNC4809586_lod.drop-1.5_5.batches_mis.csv
    Untracked:  data/ici.vs.eoi_genes_chr-3_peak.marker-UNC4813186_lod.drop-1.5_5.batches.csv
    Untracked:  data/ici.vs.eoi_genes_chr-3_peak.marker-UNC4813186_lod.drop-1.5_5.batches_mis.csv
    Untracked:  data/ici.vs.eoi_genes_chr-3_peak.marker-UNC4820379_lod.drop-1.5_5.batches_mis.csv
    Untracked:  data/ici.vs.eoi_genes_chr-3_peak.marker-UNC4875653_lod.drop-1.5_5.batches_mis.csv
    Untracked:  data/ici.vs.eoi_genes_chr-3_peak.marker-UNC4909830_lod.drop-1.5_5.batches_mis.csv
    Untracked:  data/ici.vs.eoi_genes_chr-3_peak.marker-UNC4912288_lod.drop-1.5_5.batches_mis.csv
    Untracked:  data/ici.vs.eoi_genes_chr-3_peak.marker-UNC4914130_lod.drop-1.5_5.batches_mis.csv
    Untracked:  data/ici.vs.eoi_genes_chr-3_peak.marker-UNC4923398_lod.drop-1.5_5.batches_mis.csv
    Untracked:  data/ici.vs.eoi_genes_chr-3_peak.marker-UNC4924377_lod.drop-1.5_5.batches.csv
    Untracked:  data/ici.vs.eoi_genes_chr-3_peak.marker-UNC4945186_lod.drop-1.5_5.batches_mis.csv
    Untracked:  data/ici.vs.eoi_genes_chr-3_peak.marker-UNC4947492_lod.drop-1.5_5.batches_mis.csv
    Untracked:  data/ici.vs.eoi_genes_chr-3_peak.marker-UNC4957168_lod.drop-1.5_5.batches.csv
    Untracked:  data/ici.vs.eoi_genes_chr-3_peak.marker-UNC4957859_lod.drop-1.5_5.batches_mis.csv
    Untracked:  data/ici.vs.eoi_genes_chr-3_peak.marker-UNC4964062_lod.drop-1.5_5.batches_mis.csv
    Untracked:  data/ici.vs.eoi_genes_chr-3_peak.marker-UNC4979914_lod.drop-1.5_5.batches_mis.csv
    Untracked:  data/ici.vs.eoi_genes_chr-3_peak.marker-UNC4985064_lod.drop-1.5_5.batches_mis.csv
    Untracked:  data/ici.vs.eoi_genes_chr-3_peak.marker-UNC4990111_lod.drop-1.5_5.batches_mis.csv
    Untracked:  data/ici.vs.eoi_genes_chr-3_peak.marker-UNC4991565_lod.drop-1.5_5.batches_mis.csv
    Untracked:  data/ici.vs.eoi_genes_chr-3_peak.marker-UNC5001723_lod.drop-1.5_5.batches_mis.csv
    Untracked:  data/ici.vs.eoi_genes_chr-3_peak.marker-UNC5008656_lod.drop-1.5_5.batches.csv
    Untracked:  data/ici.vs.eoi_genes_chr-3_peak.marker-UNC5008656_lod.drop-1.5_5.batches_mis.csv
    Untracked:  data/ici.vs.eoi_genes_chr-3_peak.marker-UNC5042361_lod.drop-1.5_5.batches_mis.csv
    Untracked:  data/ici.vs.eoi_genes_chr-3_peak.marker-UNC5055259_lod.drop-1.5_5.batches_mis.csv
    Untracked:  data/ici.vs.eoi_genes_chr-3_peak.marker-UNC5076935_lod.drop-1.5_5.batches_mis.csv
    Untracked:  data/ici.vs.eoi_genes_chr-3_peak.marker-UNC5082757_lod.drop-1.5_5.batches_mis.csv
    Untracked:  data/ici.vs.eoi_genes_chr-3_peak.marker-UNC5083168_lod.drop-1.5_5.batches.csv
    Untracked:  data/ici.vs.eoi_genes_chr-3_peak.marker-UNC5173293_lod.drop-1.5_5.batches_mis.csv
    Untracked:  data/ici.vs.eoi_genes_chr-3_peak.marker-UNC5182782_lod.drop-1.5_5.batches_mis.csv
    Untracked:  data/ici.vs.eoi_genes_chr-3_peak.marker-UNC5192582_lod.drop-1.5_5.batches.csv
    Untracked:  data/ici.vs.eoi_genes_chr-3_peak.marker-UNC5193970_lod.drop-1.5_5.batches_mis.csv
    Untracked:  data/ici.vs.eoi_genes_chr-3_peak.marker-UNC5213888_lod.drop-1.5_snpsqc_5.batches_mis.csv
    Untracked:  data/ici.vs.eoi_genes_chr-3_peak.marker-UNC5240484_lod.drop-1.5_5.batches_mis.csv
    Untracked:  data/ici.vs.eoi_genes_chr-3_peak.marker-UNC5256922_lod.drop-1.5_5.batches_mis.csv
    Untracked:  data/ici.vs.eoi_genes_chr-3_peak.marker-UNC5266664_lod.drop-1.5_5.batches_mis.csv
    Untracked:  data/ici.vs.eoi_genes_chr-3_peak.marker-UNC5266711_lod.drop-1.5_5.batches.csv
    Untracked:  data/ici.vs.eoi_genes_chr-3_peak.marker-UNC5267813_lod.drop-1.5_5.batches_mis.csv
    Untracked:  data/ici.vs.eoi_genes_chr-3_peak.marker-UNC5274554_lod.drop-1.5_5.batches_mis.csv
    Untracked:  data/ici.vs.eoi_genes_chr-3_peak.marker-UNC5277061_lod.drop-1.5_5.batches_mis.csv
    Untracked:  data/ici.vs.eoi_genes_chr-3_peak.marker-UNC5282259_lod.drop-1.5_5.batches_mis.csv
    Untracked:  data/ici.vs.eoi_genes_chr-3_peak.marker-UNC5287977_lod.drop-1.5_5.batches_mis.csv
    Untracked:  data/ici.vs.eoi_genes_chr-3_peak.marker-UNC5301460_lod.drop-1.5_5.batches_mis.csv
    Untracked:  data/ici.vs.eoi_genes_chr-3_peak.marker-UNC5321559_lod.drop-1.5_5.batches_mis.csv
    Untracked:  data/ici.vs.eoi_genes_chr-3_peak.marker-UNC5322455_lod.drop-1.5_5.batches_mis.csv
    Untracked:  data/ici.vs.eoi_genes_chr-3_peak.marker-UNC5328596_lod.drop-1.5_5.batches_mis.csv
    Untracked:  data/ici.vs.eoi_genes_chr-3_peak.marker-UNC5350384_lod.drop-1.5_5.batches_mis.csv
    Untracked:  data/ici.vs.eoi_genes_chr-3_peak.marker-UNC5354724_lod.drop-1.5_5.batches_mis.csv
    Untracked:  data/ici.vs.eoi_genes_chr-3_peak.marker-UNC5375208_lod.drop-1.5_5.batches_mis.csv
    Untracked:  data/ici.vs.eoi_genes_chr-3_peak.marker-UNC5384072_lod.drop-1.5_5.batches_mis.csv
    Untracked:  data/ici.vs.eoi_genes_chr-3_peak.marker-UNC5400314_lod.drop-1.5_5.batches_mis.csv
    Untracked:  data/ici.vs.eoi_genes_chr-3_peak.marker-UNC5433945_lod.drop-1.5_5.batches_mis.csv
    Untracked:  data/ici.vs.eoi_genes_chr-3_peak.marker-UNC5435874_lod.drop-1.5_5.batches_mis.csv
    Untracked:  data/ici.vs.eoi_genes_chr-3_peak.marker-UNC5454839_lod.drop-1.5_5.batches.csv
    Untracked:  data/ici.vs.eoi_genes_chr-3_peak.marker-UNC5454839_lod.drop-1.5_5.batches_mis.csv
    Untracked:  data/ici.vs.eoi_genes_chr-3_peak.marker-UNC5466669_lod.drop-1.5_5.batches_mis.csv
    Untracked:  data/ici.vs.eoi_genes_chr-3_peak.marker-UNC5517972_lod.drop-1.5_5.batches_mis.csv
    Untracked:  data/ici.vs.eoi_genes_chr-3_peak.marker-UNC5522257_lod.drop-1.5_5.batches_mis.csv
    Untracked:  data/ici.vs.eoi_genes_chr-3_peak.marker-UNC5523265_lod.drop-1.5_5.batches_mis.csv
    Untracked:  data/ici.vs.eoi_genes_chr-3_peak.marker-UNC5532424_lod.drop-1.5_5.batches_mis.csv
    Untracked:  data/ici.vs.eoi_genes_chr-3_peak.marker-UNC5553882_lod.drop-1.5_5.batches_mis.csv
    Untracked:  data/ici.vs.eoi_genes_chr-3_peak.marker-UNC5572051_lod.drop-1.5_5.batches_mis.csv
    Untracked:  data/ici.vs.eoi_genes_chr-3_peak.marker-UNC5582753_lod.drop-1.5_5.batches.csv
    Untracked:  data/ici.vs.eoi_genes_chr-3_peak.marker-UNC5589260_lod.drop-1.5_5.batches_mis.csv
    Untracked:  data/ici.vs.eoi_genes_chr-3_peak.marker-UNC5595555_lod.drop-1.5_5.batches_mis.csv
    Untracked:  data/ici.vs.eoi_genes_chr-3_peak.marker-UNC5623239_lod.drop-1.5_5.batches_mis.csv
    Untracked:  data/ici.vs.eoi_genes_chr-3_peak.marker-UNC5639830_lod.drop-1.5_5.batches_mis.csv
    Untracked:  data/ici.vs.eoi_genes_chr-3_peak.marker-UNC5645844_lod.drop-1.5_5.batches.csv
    Untracked:  data/ici.vs.eoi_genes_chr-3_peak.marker-UNC5645844_lod.drop-1.5_5.batches_mis.csv
    Untracked:  data/ici.vs.eoi_genes_chr-3_peak.marker-UNC5680234_lod.drop-1.5_5.batches_mis.csv
    Untracked:  data/ici.vs.eoi_genes_chr-3_peak.marker-UNC5682314_lod.drop-1.5_5.batches_mis.csv
    Untracked:  data/ici.vs.eoi_genes_chr-3_peak.marker-UNC5725356_lod.drop-1.5_5.batches_mis.csv
    Untracked:  data/ici.vs.eoi_genes_chr-3_peak.marker-UNC5727351_lod.drop-1.5_5.batches_mis.csv
    Untracked:  data/ici.vs.eoi_genes_chr-3_peak.marker-UNC5734420_lod.drop-1.5_5.batches_mis.csv
    Untracked:  data/ici.vs.eoi_genes_chr-3_peak.marker-UNC5743257_lod.drop-1.5_5.batches_mis.csv
    Untracked:  data/ici.vs.eoi_genes_chr-3_peak.marker-UNC5751567_lod.drop-1.5_5.batches_mis.csv
    Untracked:  data/ici.vs.eoi_genes_chr-3_peak.marker-UNC5752623_lod.drop-1.5_5.batches_mis.csv
    Untracked:  data/ici.vs.eoi_genes_chr-3_peak.marker-UNC5776974_lod.drop-1.5_5.batches_mis.csv
    Untracked:  data/ici.vs.eoi_genes_chr-3_peak.marker-UNC5778977_lod.drop-1.5_5.batches_mis.csv
    Untracked:  data/ici.vs.eoi_genes_chr-3_peak.marker-UNC5788507_lod.drop-1.5_5.batches_mis.csv
    Untracked:  data/ici.vs.eoi_genes_chr-3_peak.marker-UNC5792468_lod.drop-1.5_5.batches_mis.csv
    Untracked:  data/ici.vs.eoi_genes_chr-3_peak.marker-UNC5794598_lod.drop-1.5_5.batches_mis.csv
    Untracked:  data/ici.vs.eoi_genes_chr-3_peak.marker-UNC5795561_lod.drop-1.5_5.batches_mis.csv
    Untracked:  data/ici.vs.eoi_genes_chr-3_peak.marker-UNC5802298_lod.drop-1.5_5.batches_mis.csv
    Untracked:  data/ici.vs.eoi_genes_chr-3_peak.marker-UNC5806628_lod.drop-1.5_5.batches_mis.csv
    Untracked:  data/ici.vs.eoi_genes_chr-3_peak.marker-UNC5808246_lod.drop-1.5_5.batches.csv
    Untracked:  data/ici.vs.eoi_genes_chr-3_peak.marker-UNC5817478_lod.drop-1.5_5.batches_mis.csv
    Untracked:  data/ici.vs.eoi_genes_chr-3_peak.marker-UNC5836999_lod.drop-1.5_5.batches_mis.csv
    Untracked:  data/ici.vs.eoi_genes_chr-3_peak.marker-UNC5841501_lod.drop-1.5_5.batches_mis.csv
    Untracked:  data/ici.vs.eoi_genes_chr-3_peak.marker-UNC5847927_lod.drop-1.5_5.batches_mis.csv
    Untracked:  data/ici.vs.eoi_genes_chr-3_peak.marker-UNC5966417_lod.drop-1.5_5.batches_mis.csv
    Untracked:  data/ici.vs.eoi_genes_chr-3_peak.marker-UNCHS008129_lod.drop-1.5_5.batches_mis.csv
    Untracked:  data/ici.vs.eoi_genes_chr-3_peak.marker-UNCHS008173_lod.drop-1.5_5.batches_mis.csv
    Untracked:  data/ici.vs.eoi_genes_chr-3_peak.marker-UNCHS008182_lod.drop-1.5_5.batches.csv
    Untracked:  data/ici.vs.eoi_genes_chr-3_peak.marker-UNCHS008193_lod.drop-1.5_5.batches_mis.csv
    Untracked:  data/ici.vs.eoi_genes_chr-3_peak.marker-UNCHS008209_lod.drop-1.5_5.batches_mis.csv
    Untracked:  data/ici.vs.eoi_genes_chr-3_peak.marker-UNCHS008215_lod.drop-1.5_5.batches_mis.csv
    Untracked:  data/ici.vs.eoi_genes_chr-3_peak.marker-UNCHS008219_lod.drop-1.5_5.batches_mis.csv
    Untracked:  data/ici.vs.eoi_genes_chr-3_peak.marker-UNCHS008231_lod.drop-1.5_5.batches_mis.csv
    Untracked:  data/ici.vs.eoi_genes_chr-3_peak.marker-UNCHS008235_lod.drop-1.5_5.batches_mis.csv
    Untracked:  data/ici.vs.eoi_genes_chr-3_peak.marker-UNCHS008242_lod.drop-1.5_5.batches.csv
    Untracked:  data/ici.vs.eoi_genes_chr-3_peak.marker-UNCHS008253_lod.drop-1.5_5.batches_mis.csv
    Untracked:  data/ici.vs.eoi_genes_chr-3_peak.marker-UNCHS008256_lod.drop-1.5_5.batches_mis.csv
    Untracked:  data/ici.vs.eoi_genes_chr-3_peak.marker-UNCHS008259_lod.drop-1.5_5.batches_mis.csv
    Untracked:  data/ici.vs.eoi_genes_chr-3_peak.marker-UNCHS008271_lod.drop-1.5_5.batches_mis.csv
    Untracked:  data/ici.vs.eoi_genes_chr-3_peak.marker-UNCHS008286_lod.drop-1.5_5.batches.csv
    Untracked:  data/ici.vs.eoi_genes_chr-3_peak.marker-UNCHS008286_lod.drop-1.5_5.batches_mis.csv
    Untracked:  data/ici.vs.eoi_genes_chr-3_peak.marker-UNCHS008311_lod.drop-1.5_5.batches_mis.csv
    Untracked:  data/ici.vs.eoi_genes_chr-3_peak.marker-UNCHS008317_lod.drop-1.5_5.batches_mis.csv
    Untracked:  data/ici.vs.eoi_genes_chr-3_peak.marker-UNCHS008372_lod.drop-1.5_5.batches_mis.csv
    Untracked:  data/ici.vs.eoi_genes_chr-3_peak.marker-UNCHS008379_lod.drop-1.5_5.batches_mis.csv
    Untracked:  data/ici.vs.eoi_genes_chr-3_peak.marker-UNCHS008410_lod.drop-1.5_5.batches_mis.csv
    Untracked:  data/ici.vs.eoi_genes_chr-3_peak.marker-UNCHS008415_lod.drop-1.5_5.batches_mis.csv
    Untracked:  data/ici.vs.eoi_genes_chr-3_peak.marker-UNCHS008431_lod.drop-1.5_5.batches_mis.csv
    Untracked:  data/ici.vs.eoi_genes_chr-3_peak.marker-UNCHS008487_lod.drop-1.5_snpsqc_5.batches.csv
    Untracked:  data/ici.vs.eoi_genes_chr-3_peak.marker-UNCHS008487_lod.drop-1.5_snpsqc_dis_no-x_updated_5.batches.csv
    Untracked:  data/ici.vs.eoi_genes_chr-3_peak.marker-UNCHS008487_lod.drop-1.5_snpsqc_dis_no-x_updated_5.batches_mis.csv
    Untracked:  data/ici.vs.eoi_genes_chr-3_peak.marker-UNCHS008511_lod.drop-1.5_snpsqc_5.batches_mis.csv
    Untracked:  data/ici.vs.eoi_genes_chr-3_peak.marker-UNCHS008609_lod.drop-1.5_5.batches_mis.csv
    Untracked:  data/ici.vs.eoi_genes_chr-3_peak.marker-UNCHS008614_lod.drop-1.5_5.batches.csv
    Untracked:  data/ici.vs.eoi_genes_chr-3_peak.marker-UNCHS008649_lod.drop-1.5_5.batches_mis.csv
    Untracked:  data/ici.vs.eoi_genes_chr-3_peak.marker-UNCHS008651_lod.drop-1.5_5.batches.csv
    Untracked:  data/ici.vs.eoi_genes_chr-3_peak.marker-UNCHS008656_lod.drop-1.5_5.batches_mis.csv
    Untracked:  data/ici.vs.eoi_genes_chr-3_peak.marker-UNCHS008723_lod.drop-1.5_5.batches_mis.csv
    Untracked:  data/ici.vs.eoi_genes_chr-3_peak.marker-UNCHS008802_lod.drop-1.5_5.batches_mis.csv
    Untracked:  data/ici.vs.eoi_genes_chr-3_peak.marker-UNCHS008803_lod.drop-1.5_5.batches_mis.csv
    Untracked:  data/ici.vs.eoi_genes_chr-3_peak.marker-UNCHS008808_lod.drop-1.5_5.batches_mis.csv
    Untracked:  data/ici.vs.eoi_genes_chr-3_peak.marker-UNCHS008815_lod.drop-1.5_5.batches.csv
    Untracked:  data/ici.vs.eoi_genes_chr-3_peak.marker-UNCHS008815_lod.drop-1.5_5.batches_mis.csv
    Untracked:  data/ici.vs.eoi_genes_chr-3_peak.marker-UNCHS008834_lod.drop-1.5_5.batches_mis.csv
    Untracked:  data/ici.vs.eoi_genes_chr-3_peak.marker-UNCHS008838_lod.drop-1.5_5.batches_mis.csv
    Untracked:  data/ici.vs.eoi_genes_chr-3_peak.marker-UNCHS008914_lod.drop-1.5_5.batches_mis.csv
    Untracked:  data/ici.vs.eoi_genes_chr-3_peak.marker-UNCHS008928_lod.drop-1.5_5.batches_mis.csv
    Untracked:  data/ici.vs.eoi_genes_chr-3_peak.marker-UNCHS008948_lod.drop-1.5_5.batches_mis.csv
    Untracked:  data/ici.vs.eoi_genes_chr-3_peak.marker-UNCHS008959_lod.drop-1.5_5.batches_mis.csv
    Untracked:  data/ici.vs.eoi_genes_chr-3_peak.marker-UNCHS008973_lod.drop-1.5_5.batches_mis.csv
    Untracked:  data/ici.vs.eoi_genes_chr-3_peak.marker-UNCHS008987_lod.drop-1.5_5.batches_mis.csv
    Untracked:  data/ici.vs.eoi_genes_chr-3_peak.marker-UNCHS009001_lod.drop-1.5_5.batches_mis.csv
    Untracked:  data/ici.vs.eoi_genes_chr-3_peak.marker-UNCHS009025_lod.drop-1.5_5.batches_mis.csv
    Untracked:  data/ici.vs.eoi_genes_chr-3_peak.marker-UNCHS009034_lod.drop-1.5_5.batches_mis.csv
    Untracked:  data/ici.vs.eoi_genes_chr-3_peak.marker-UNCHS009087_lod.drop-1.5_5.batches_mis.csv
    Untracked:  data/ici.vs.eoi_genes_chr-3_peak.marker-UNCHS009109_lod.drop-1.5_5.batches_mis.csv
    Untracked:  data/ici.vs.eoi_genes_chr-3_peak.marker-UNCHS009142_lod.drop-1.5_5.batches_mis.csv
    Untracked:  data/ici.vs.eoi_genes_chr-3_peak.marker-UNCHS009207_lod.drop-1.5_5.batches_mis.csv
    Untracked:  data/ici.vs.eoi_genes_chr-3_peak.marker-UNCHS009216_lod.drop-1.5_5.batches_mis.csv
    Untracked:  data/ici.vs.eoi_genes_chr-3_peak.marker-UNCHS009224_lod.drop-1.5_5.batches.csv
    Untracked:  data/ici.vs.eoi_genes_chr-3_peak.marker-UNCHS009306_lod.drop-1.5_5.batches_mis.csv
    Untracked:  data/ici.vs.eoi_genes_chr-3_peak.marker-UNCHS009316_lod.drop-1.5_5.batches_mis.csv
    Untracked:  data/ici.vs.eoi_genes_chr-3_peak.marker-UNCHS009364_lod.drop-1.5_5.batches_mis.csv
    Untracked:  data/ici.vs.eoi_genes_chr-3_peak.marker-UNCHS009377_lod.drop-1.5_5.batches_mis.csv
    Untracked:  data/ici.vs.eoi_genes_chr-3_peak.marker-UNCHS009382_lod.drop-1.5_5.batches_mis.csv
    Untracked:  data/ici.vs.eoi_genes_chr-3_peak.marker-UNCHS009383_lod.drop-1.5_5.batches.csv
    Untracked:  data/ici.vs.eoi_genes_chr-3_peak.marker-UNCHS009387_lod.drop-1.5_5.batches_mis.csv
    Untracked:  data/ici.vs.eoi_genes_chr-3_peak.marker-UNCHS009395_lod.drop-1.5_5.batches_mis.csv
    Untracked:  data/ici.vs.eoi_genes_chr-3_peak.marker-UNCHS009413_lod.drop-1.5_5.batches_mis.csv
    Untracked:  data/ici.vs.eoi_genes_chr-3_peak.marker-UNCHS009428_lod.drop-1.5_5.batches_mis.csv
    Untracked:  data/ici.vs.eoi_genes_chr-3_peak.marker-UNCHS009463_lod.drop-1.5_5.batches_mis.csv
    Untracked:  data/ici.vs.eoi_genes_chr-3_peak.marker-UNCHS009503_lod.drop-1.5_5.batches_mis.csv
    Untracked:  data/ici.vs.eoi_genes_chr-3_peak.marker-UNCHS009532_lod.drop-1.5_5.batches_mis.csv
    Untracked:  data/ici.vs.eoi_genes_chr-3_peak.marker-UNCHS009541_lod.drop-1.5_5.batches_mis.csv
    Untracked:  data/ici.vs.eoi_genes_chr-3_peak.marker-UNCHS009560_lod.drop-1.5_5.batches_mis.csv
    Untracked:  data/ici.vs.eoi_genes_chr-3_peak.marker-UNCHS009574_lod.drop-1.5_5.batches_mis.csv
    Untracked:  data/ici.vs.eoi_genes_chr-3_peak.marker-UNCJPD001170_lod.drop-1.5_5.batches_mis.csv
    Untracked:  data/ici.vs.eoi_genes_chr-3_peak.marker-UNCJPD001198_lod.drop-1.5_5.batches_mis.csv
    Untracked:  data/ici.vs.eoi_genes_chr-3_peak.marker-UNCJPD001276_lod.drop-1.5_5.batches_mis.csv
    Untracked:  data/ici.vs.eoi_genes_chr-3_peak.marker-UNCJPD001315_lod.drop-1.5_5.batches_mis.csv
    Untracked:  data/ici.vs.eoi_genes_chr-3_peak.marker-UNCJPD001355_lod.drop-1.5_5.batches_mis.csv
    Untracked:  data/ici.vs.eoi_genes_chr-3_peak.marker-UNCJPD001373_lod.drop-1.5_5.batches_mis.csv
    Untracked:  data/ici.vs.eoi_genes_chr-3_peak.marker-UNCJPD001382_lod.drop-1.5_5.batches_mis.csv
    Untracked:  data/ici.vs.eoi_genes_chr-4_peak.marker-UNC6851205_lod.drop-1.5_5.batches_mis.csv
    Untracked:  data/ici.vs.eoi_genes_chr-4_peak.marker-UNC8250659_lod.drop-1.5_snpsqc_dis_no-x_updated_5.batches.csv
    Untracked:  data/ici.vs.eoi_genes_chr-4_peak.marker-UNC8299912_lod.drop-1.5_5.batches_mis.csv
    Untracked:  data/ici.vs.eoi_genes_chr-4_peak.marker-UNC8300499_lod.drop-1.5_5.batches.csv
    Untracked:  data/ici.vs.eoi_genes_chr-4_peak.marker-UNC8381981_lod.drop-1.5_5.batches_mis.csv
    Untracked:  data/ici.vs.eoi_genes_chr-4_peak.marker-UNC8436434_lod.drop-1.5_5.batches_mis.csv
    Untracked:  data/ici.vs.eoi_genes_chr-4_peak.marker-UNC8439633_lod.drop-1.5_5.batches_mis.csv
    Untracked:  data/ici.vs.eoi_genes_chr-4_peak.marker-UNC8530763_lod.drop-1.5_5.batches_mis.csv
    Untracked:  data/ici.vs.eoi_genes_chr-4_peak.marker-UNCHS012948_lod.drop-1.5_snpsqc_dis_no-x_updated_5.batches_mis.csv
    Untracked:  data/ici.vs.eoi_genes_chr-4_peak.marker-UNCHS012976_lod.drop-1.5_5.batches_mis.csv
    Untracked:  data/ici.vs.eoi_genes_chr-4_peak.marker-UNCHS013118_lod.drop-1.5_5.batches.csv
    Untracked:  data/ici.vs.eoi_genes_chr-4_peak.marker-UNCHS013118_lod.drop-1.5_snpsqc_5.batches.csv
    Untracked:  data/ici.vs.eoi_genes_chr-4_peak.marker-UNCHS013118_lod.drop-1.5_snpsqc_5.batches_mis.csv
    Untracked:  data/ici.vs.eoi_genes_chr-4_peak.marker-UNCHS013130_lod.drop-1.5_5.batches_mis.csv
    Untracked:  data/ici.vs.eoi_genes_chr-4_peak.marker-UNCHS013139_lod.drop-1.5_5.batches_mis.csv
    Untracked:  data/ici.vs.eoi_genes_chr-4_peak.marker-UNCHS013304_lod.drop-1.5_5.batches.csv
    Untracked:  data/ici.vs.eoi_genes_chr-5_peak.marker-JAX00590770_lod.drop-1.5_5.batches_mis.csv
    Untracked:  data/ici.vs.eoi_genes_chr-5_peak.marker-UNC10044126_lod.drop-1.5_5.batches_mis.csv
    Untracked:  data/ici.vs.eoi_genes_chr-7_peak.marker-ICR1830_lod.drop-1.5_5.batches_mis.csv
    Untracked:  data/ici.vs.eoi_genes_chr-7_peak.marker-UNC13170794_lod.drop-1.5_5.batches_mis.csv
    Untracked:  data/ici.vs.eoi_genes_chr-7_peak.marker-UNCHS019480_lod.drop-1.5_5.batches.csv
    Untracked:  data/ici.vs.eoi_genes_chr-7_peak.marker-UNCHS020066_lod.drop-1.5_snpsqc_5.batches_mis.csv
    Untracked:  data/ici.vs.eoi_genes_chr-7_peak.marker-UNCHS020066_lod.drop-1.5_snpsqc_dis_no-x_updated_5.batches_mis.csv
    Untracked:  data/ici.vs.eoi_genes_chr-7_peak.marker-cr31snv87_lod.drop-1.5_5.batches_mis.csv
    Untracked:  data/ici.vs.eoi_genes_chr-8_peak.marker-JAX00667121_lod.drop-1.5_5.batches_mis.csv
    Untracked:  data/ici.vs.eoi_genes_chr-8_peak.marker-UNC15430711_lod.drop-1.5_5.batches_mis.csv
    Untracked:  data/ici.vs.eoi_genes_chr-8_peak.marker-UNC15524531_lod.drop-1.5_5.batches_mis.csv
    Untracked:  data/ici.vs.eoi_genes_chr-8_peak.marker-UNC15524531_lod.drop-1.5_snpsqc_5.batches_mis.csv
    Untracked:  data/ici.vs.eoi_genes_chr-8_peak.marker-UNC15524531_lod.drop-1.5_snpsqc_dis_no-x_updated_5.batches_mis.csv
    Untracked:  data/ici.vs.eoi_genes_chr-9_peak.marker-UNC17203329_lod.drop-1.5_5.batches_mis.csv
    Untracked:  data/ici.vs.eoi_genes_chr-9_peak.marker-UNC17203597_lod.drop-1.5_snpsqc_5.batches_mis.csv
    Untracked:  data/ici.vs.eoi_genes_chr-9_peak.marker-UNCHS024593_lod.drop-1.5_5.batches_mis.csv
    Untracked:  data/ici.vs.eoi_genes_chr-X_peak.marker-UNC30627130_lod.drop-1.5_5.batches_mis.csv
    Untracked:  data/ici.vs.eoi_genes_chr-X_peak.marker-UNCHS048313_lod.drop-1.5_5.batches_mis.csv
    Untracked:  data/ici.vs.eoi_genes_chr-X_peak.marker-UNCHS048314_lod.drop-1.5_snpsqc_5.batches_mis.csv
    Untracked:  data/ici.vs.eoi_genes_chr-X_peak.marker-UNCHS048314_lod.drop-1.5_snpsqc_dis_no-x_updated_5.batches_mis.csv
    Untracked:  data/ici.vs.eoi_gm_qtl_5.batches.csv
    Untracked:  data/ici.vs.eoi_gm_qtl_5.batches_mis.csv
    Untracked:  data/ici.vs.eoi_gm_qtl_snpsqc_5.batches.csv
    Untracked:  data/ici.vs.eoi_gm_qtl_snpsqc_5.batches_mis.csv
    Untracked:  data/ici.vs.eoi_gm_qtl_snpsqc_dis_no-x_updated_5.batches.csv
    Untracked:  data/ici.vs.eoi_gm_qtl_snpsqc_dis_no-x_updated_5.batches_mis.csv
    Untracked:  data/ici.vs.eoi_marker.freq_low.geno.freq.removed_geno.ratio.csv
    Untracked:  data/ici.vs.eoi_marker.freq_low.geno.freq.removed_geno.ratio_5.batches.csv
    Untracked:  data/ici.vs.eoi_marker.freq_low.geno.freq.removed_geno.ratio_5.batches_0.csv
    Untracked:  data/ici.vs.eoi_marker.freq_low.geno.freq.removed_geno.ratio_5.batches_52.csv
    Untracked:  data/ici.vs.eoi_marker.freq_low.geno.freq.removed_geno.ratio_5.batches_mis.csv
    Untracked:  data/ici.vs.eoi_marker.freq_low.geno.freq.removed_geno.ratio_5.batches_mis_0.csv
    Untracked:  data/ici.vs.eoi_marker.freq_low.geno.freq.removed_geno.ratio_5.batches_mis_52.csv
    Untracked:  data/ici.vs.eoi_marker.freq_low.geno.freq.removed_sample.outliers.removed_geno.ratio.csv
    Untracked:  data/ici.vs.eoi_marker.freq_low.geno.freq.removed_sample.outliers.removed_geno.ratio_5.batches.csv
    Untracked:  data/ici.vs.eoi_marker.freq_low.geno.freq.removed_sample.outliers.removed_geno.ratio_5.batches_0.csv
    Untracked:  data/ici.vs.eoi_marker.freq_low.geno.freq.removed_sample.outliers.removed_geno.ratio_5.batches_52.csv
    Untracked:  data/ici.vs.eoi_marker.freq_low.geno.freq.removed_sample.outliers.removed_geno.ratio_5.batches_mis.csv
    Untracked:  data/ici.vs.eoi_marker.freq_low.geno.freq.removed_sample.outliers.removed_geno.ratio_5.batches_mis_0.csv
    Untracked:  data/ici.vs.eoi_marker.freq_low.geno.freq.removed_sample.outliers.removed_geno.ratio_5.batches_mis_52.csv
    Untracked:  data/ici.vs.eoi_marker.freq_low.probs.freq.removed_geno.ratio.csv
    Untracked:  data/ici.vs.eoi_marker.freq_low.probs.freq.removed_geno.ratio_5.batches.csv
    Untracked:  data/ici.vs.eoi_marker.freq_low.probs.freq.removed_geno.ratio_5.batches_0.csv
    Untracked:  data/ici.vs.eoi_marker.freq_low.probs.freq.removed_geno.ratio_5.batches_52.csv
    Untracked:  data/ici.vs.eoi_marker.freq_low.probs.freq.removed_geno.ratio_5.batches_mis.csv
    Untracked:  data/ici.vs.eoi_marker.freq_low.probs.freq.removed_geno.ratio_5.batches_mis_0.csv
    Untracked:  data/ici.vs.eoi_marker.freq_low.probs.freq.removed_geno.ratio_5.batches_mis_52.csv
    Untracked:  data/ici.vs.eoi_marker.freq_low.probs.freq.removed_sample.outliers.removed_geno.ratio.csv
    Untracked:  data/ici.vs.eoi_marker.freq_low.probs.freq.removed_sample.outliers.removed_geno.ratio_5.batches.csv
    Untracked:  data/ici.vs.eoi_marker.freq_low.probs.freq.removed_sample.outliers.removed_geno.ratio_5.batches_0.csv
    Untracked:  data/ici.vs.eoi_marker.freq_low.probs.freq.removed_sample.outliers.removed_geno.ratio_5.batches_52.csv
    Untracked:  data/ici.vs.eoi_marker.freq_low.probs.freq.removed_sample.outliers.removed_geno.ratio_5.batches_mis.csv
    Untracked:  data/ici.vs.eoi_marker.freq_low.probs.freq.removed_sample.outliers.removed_geno.ratio_5.batches_mis_0.csv
    Untracked:  data/ici.vs.eoi_marker.freq_low.probs.freq.removed_sample.outliers.removed_geno.ratio_5.batches_mis_52.csv
    Untracked:  data/ici.vs.eoi_rz.age-additive.covariates_blup_sub_chr-7_peak.marker-UNC13239362_lod.drop-1.5_snpsqc_5.batches.csv
    Untracked:  data/ici.vs.eoi_rz.age-additive.covariates_genes_chr-7_peak.marker-UNC13239362_lod.drop-1.5_snpsqc_5.batches.csv
    Untracked:  data/ici.vs.eoi_rz.age-no.covariates_blup_sub_chr-10_peak.marker-JAX00292927_lod.drop-1.5_5.batches_0.csv
    Untracked:  data/ici.vs.eoi_rz.age-no.covariates_blup_sub_chr-10_peak.marker-UNC18311938_lod.drop-1.5_5.batches_0.csv
    Untracked:  data/ici.vs.eoi_rz.age-no.covariates_blup_sub_chr-10_peak.marker-UNC18343181_lod.drop-1.5_5.batches_0.csv
    Untracked:  data/ici.vs.eoi_rz.age-no.covariates_blup_sub_chr-10_peak.marker-UNC18376338_lod.drop-1.5_5.batches_0.csv
    Untracked:  data/ici.vs.eoi_rz.age-no.covariates_blup_sub_chr-10_peak.marker-UNC18376338_lod.drop-1.5_snpsqc_5.batches_0.csv
    Untracked:  data/ici.vs.eoi_rz.age-no.covariates_blup_sub_chr-10_peak.marker-UNC18376338_lod.drop-1.5_snpsqc_5.batches_mis_0.csv
    Untracked:  data/ici.vs.eoi_rz.age-no.covariates_blup_sub_chr-10_peak.marker-UNC18382395_lod.drop-1.5_snpsqc_dis_no-x_updated_5.batches_0.csv
    Untracked:  data/ici.vs.eoi_rz.age-no.covariates_blup_sub_chr-10_peak.marker-UNCHS028236_lod.drop-1.5_snpsqc_5.batches_mis_0.csv
    Untracked:  data/ici.vs.eoi_rz.age-no.covariates_blup_sub_chr-10_peak.marker-UNCHS028236_lod.drop-1.5_snpsqc_5.batches_mis_52.csv
    Untracked:  data/ici.vs.eoi_rz.age-no.covariates_blup_sub_chr-10_peak.marker-UNCHS028536_lod.drop-1.5_snpsqc_dis_no-x_updated_5.batches_mis_0.csv
    Untracked:  data/ici.vs.eoi_rz.age-no.covariates_blup_sub_chr-10_peak.marker-UNCHS029024_lod.drop-1.5_5.batches_0.csv
    Untracked:  data/ici.vs.eoi_rz.age-no.covariates_blup_sub_chr-11_peak.marker-UNC19970181_lod.drop-1.5_snpsqc_5.batches_mis_0.csv
    Untracked:  data/ici.vs.eoi_rz.age-no.covariates_blup_sub_chr-11_peak.marker-UNC19970181_lod.drop-1.5_snpsqc_5.batches_mis_52.csv
    Untracked:  data/ici.vs.eoi_rz.age-no.covariates_blup_sub_chr-12_peak.marker-ICR499_lod.drop-1.5_snpsqc_5.batches_mis_0.csv
    Untracked:  data/ici.vs.eoi_rz.age-no.covariates_blup_sub_chr-12_peak.marker-ICR499_lod.drop-1.5_snpsqc_5.batches_mis_52.csv
    Untracked:  data/ici.vs.eoi_rz.age-no.covariates_blup_sub_chr-13_peak.marker-UNCHS036773_lod.drop-1.5_snpsqc_5.batches_mis_0.csv
    Untracked:  data/ici.vs.eoi_rz.age-no.covariates_blup_sub_chr-13_peak.marker-UNCHS036773_lod.drop-1.5_snpsqc_5.batches_mis_52.csv
    Untracked:  data/ici.vs.eoi_rz.age-no.covariates_blup_sub_chr-14_peak.marker-UNC24056202_lod.drop-1.5_snpsqc_5.batches_mis_0.csv
    Untracked:  data/ici.vs.eoi_rz.age-no.covariates_blup_sub_chr-14_peak.marker-UNC24056202_lod.drop-1.5_snpsqc_5.batches_mis_52.csv
    Untracked:  data/ici.vs.eoi_rz.age-no.covariates_blup_sub_chr-15_peak.marker-UNC26070435_lod.drop-1.5_snpsqc_dis_no-x_updated_5.batches_mis_0.csv
    Untracked:  data/ici.vs.eoi_rz.age-no.covariates_blup_sub_chr-15_peak.marker-UNC26070435_lod.drop-1.5_snpsqc_dis_no-x_updated_5.batches_mis_52.csv
    Untracked:  data/ici.vs.eoi_rz.age-no.covariates_blup_sub_chr-17_peak.marker-UNCHS044241_lod.drop-1.5_snpsqc_5.batches_mis_0.csv
    Untracked:  data/ici.vs.eoi_rz.age-no.covariates_blup_sub_chr-17_peak.marker-UNCHS044241_lod.drop-1.5_snpsqc_5.batches_mis_52.csv
    Untracked:  data/ici.vs.eoi_rz.age-no.covariates_blup_sub_chr-17_peak.marker-UNCHS044241_lod.drop-1.5_snpsqc_dis_no-x_updated_5.batches_mis_0.csv
    Untracked:  data/ici.vs.eoi_rz.age-no.covariates_blup_sub_chr-17_peak.marker-UNCrs47191360_lod.drop-1.5_snpsqc_dis_no-x_updated_5.batches_mis_52.csv
    Untracked:  data/ici.vs.eoi_rz.age-no.covariates_blup_sub_chr-18_peak.marker-UNCHS045343_lod.drop-1.5_snpsqc_5.batches_mis_0.csv
    Untracked:  data/ici.vs.eoi_rz.age-no.covariates_blup_sub_chr-18_peak.marker-UNCHS045343_lod.drop-1.5_snpsqc_5.batches_mis_52.csv
    Untracked:  data/ici.vs.eoi_rz.age-no.covariates_blup_sub_chr-1_peak.marker-UNC228500_lod.drop-1.5_snpsqc_dis_no-x_updated_5.batches_52.csv
    Untracked:  data/ici.vs.eoi_rz.age-no.covariates_blup_sub_chr-1_peak.marker-UNC228500_lod.drop-1.5_snpsqc_dis_no-x_updated_5.batches_mis_52.csv
    Untracked:  data/ici.vs.eoi_rz.age-no.covariates_blup_sub_chr-2_peak.marker-UNC4609527_lod.drop-1.5_snpsqc_dis_no-x_updated_5.batches_mis_0.csv
    Untracked:  data/ici.vs.eoi_rz.age-no.covariates_blup_sub_chr-2_peak.marker-UNC4609527_lod.drop-1.5_snpsqc_dis_no-x_updated_5.batches_mis_52.csv
    Untracked:  data/ici.vs.eoi_rz.age-no.covariates_blup_sub_chr-3_peak.marker-ICR1338_lod.drop-1.5_5.batches_0.csv
    Untracked:  data/ici.vs.eoi_rz.age-no.covariates_blup_sub_chr-3_peak.marker-ICR1338_lod.drop-1.5_5.batches_52.csv
    Untracked:  data/ici.vs.eoi_rz.age-no.covariates_blup_sub_chr-3_peak.marker-ICR2047_lod.drop-1.5_5.batches_52.csv
    Untracked:  data/ici.vs.eoi_rz.age-no.covariates_blup_sub_chr-3_peak.marker-JAX00106957_lod.drop-1.5_5.batches_0.csv
    Untracked:  data/ici.vs.eoi_rz.age-no.covariates_blup_sub_chr-3_peak.marker-JAX00112499_lod.drop-1.5_5.batches_52.csv
    Untracked:  data/ici.vs.eoi_rz.age-no.covariates_blup_sub_chr-3_peak.marker-JAX00219554_lod.drop-1.5_5.batches_0.csv
    Untracked:  data/ici.vs.eoi_rz.age-no.covariates_blup_sub_chr-3_peak.marker-JAX00220631_lod.drop-1.5_5.batches_0.csv
    Untracked:  data/ici.vs.eoi_rz.age-no.covariates_blup_sub_chr-3_peak.marker-JAX00514662_lod.drop-1.5_5.batches_52.csv
    Untracked:  data/ici.vs.eoi_rz.age-no.covariates_blup_sub_chr-3_peak.marker-JAX00529707_lod.drop-1.5_5.batches_52.csv
    Untracked:  data/ici.vs.eoi_rz.age-no.covariates_blup_sub_chr-3_peak.marker-UNC4813186_lod.drop-1.5_5.batches_0.csv
    Untracked:  data/ici.vs.eoi_rz.age-no.covariates_blup_sub_chr-3_peak.marker-UNC4813186_lod.drop-1.5_5.batches_52.csv
    Untracked:  data/ici.vs.eoi_rz.age-no.covariates_blup_sub_chr-3_peak.marker-UNC4836550_lod.drop-1.5_5.batches_52.csv
    Untracked:  data/ici.vs.eoi_rz.age-no.covariates_blup_sub_chr-3_peak.marker-UNC4837069_lod.drop-1.5_5.batches_0.csv
    Untracked:  data/ici.vs.eoi_rz.age-no.covariates_blup_sub_chr-3_peak.marker-UNC4912039_lod.drop-1.5_5.batches_0.csv
    Untracked:  data/ici.vs.eoi_rz.age-no.covariates_blup_sub_chr-3_peak.marker-UNC4912039_lod.drop-1.5_5.batches_52.csv
    Untracked:  data/ici.vs.eoi_rz.age-no.covariates_blup_sub_chr-3_peak.marker-UNC4924377_lod.drop-1.5_5.batches_0.csv
    Untracked:  data/ici.vs.eoi_rz.age-no.covariates_blup_sub_chr-3_peak.marker-UNC4924377_lod.drop-1.5_5.batches_52.csv
    Untracked:  data/ici.vs.eoi_rz.age-no.covariates_blup_sub_chr-3_peak.marker-UNC4956618_lod.drop-1.5_5.batches_52.csv
    Untracked:  data/ici.vs.eoi_rz.age-no.covariates_blup_sub_chr-3_peak.marker-UNC4957168_lod.drop-1.5_5.batches_0.csv
    Untracked:  data/ici.vs.eoi_rz.age-no.covariates_blup_sub_chr-3_peak.marker-UNC5006948_lod.drop-1.5_5.batches_52.csv
    Untracked:  data/ici.vs.eoi_rz.age-no.covariates_blup_sub_chr-3_peak.marker-UNC5008656_lod.drop-1.5_5.batches_0.csv
    Untracked:  data/ici.vs.eoi_rz.age-no.covariates_blup_sub_chr-3_peak.marker-UNC5083168_lod.drop-1.5_5.batches_0.csv
    Untracked:  data/ici.vs.eoi_rz.age-no.covariates_blup_sub_chr-3_peak.marker-UNC5083168_lod.drop-1.5_5.batches_52.csv
    Untracked:  data/ici.vs.eoi_rz.age-no.covariates_blup_sub_chr-3_peak.marker-UNC5192582_lod.drop-1.5_5.batches_0.csv
    Untracked:  data/ici.vs.eoi_rz.age-no.covariates_blup_sub_chr-3_peak.marker-UNC5213888_lod.drop-1.5_snpsqc_5.batches_mis_0.csv
    Untracked:  data/ici.vs.eoi_rz.age-no.covariates_blup_sub_chr-3_peak.marker-UNC5213888_lod.drop-1.5_snpsqc_5.batches_mis_52.csv
    Untracked:  data/ici.vs.eoi_rz.age-no.covariates_blup_sub_chr-3_peak.marker-UNC5262472_lod.drop-1.5_5.batches_0.csv
    Untracked:  data/ici.vs.eoi_rz.age-no.covariates_blup_sub_chr-3_peak.marker-UNC5262472_lod.drop-1.5_5.batches_52.csv
    Untracked:  data/ici.vs.eoi_rz.age-no.covariates_blup_sub_chr-3_peak.marker-UNC5266664_lod.drop-1.5_5.batches_0.csv
    Untracked:  data/ici.vs.eoi_rz.age-no.covariates_blup_sub_chr-3_peak.marker-UNC5300878_lod.drop-1.5_5.batches_52.csv
    Untracked:  data/ici.vs.eoi_rz.age-no.covariates_blup_sub_chr-3_peak.marker-UNC5360055_lod.drop-1.5_5.batches_52.csv
    Untracked:  data/ici.vs.eoi_rz.age-no.covariates_blup_sub_chr-3_peak.marker-UNC5454839_lod.drop-1.5_5.batches_0.csv
    Untracked:  data/ici.vs.eoi_rz.age-no.covariates_blup_sub_chr-3_peak.marker-UNC5454839_lod.drop-1.5_5.batches_52.csv
    Untracked:  data/ici.vs.eoi_rz.age-no.covariates_blup_sub_chr-3_peak.marker-UNC5645844_lod.drop-1.5_5.batches_0.csv
    Untracked:  data/ici.vs.eoi_rz.age-no.covariates_blup_sub_chr-3_peak.marker-UNC5645844_lod.drop-1.5_5.batches_52.csv
    Untracked:  data/ici.vs.eoi_rz.age-no.covariates_blup_sub_chr-3_peak.marker-UNC5679155_lod.drop-1.5_5.batches_52.csv
    Untracked:  data/ici.vs.eoi_rz.age-no.covariates_blup_sub_chr-3_peak.marker-UNC5808246_lod.drop-1.5_5.batches_52.csv
    Untracked:  data/ici.vs.eoi_rz.age-no.covariates_blup_sub_chr-3_peak.marker-UNC5945286_lod.drop-1.5_5.batches_52.csv
    Untracked:  data/ici.vs.eoi_rz.age-no.covariates_blup_sub_chr-3_peak.marker-UNC6227106_lod.drop-1.5_5.batches_52.csv
    Untracked:  data/ici.vs.eoi_rz.age-no.covariates_blup_sub_chr-3_peak.marker-UNCHS008182_lod.drop-1.5_5.batches_52.csv
    Untracked:  data/ici.vs.eoi_rz.age-no.covariates_blup_sub_chr-3_peak.marker-UNCHS008286_lod.drop-1.5_5.batches_0.csv
    Untracked:  data/ici.vs.eoi_rz.age-no.covariates_blup_sub_chr-3_peak.marker-UNCHS008286_lod.drop-1.5_5.batches_52.csv
    Untracked:  data/ici.vs.eoi_rz.age-no.covariates_blup_sub_chr-3_peak.marker-UNCHS008487_lod.drop-1.5_snpsqc_5.batches_0.csv
    Untracked:  data/ici.vs.eoi_rz.age-no.covariates_blup_sub_chr-3_peak.marker-UNCHS008487_lod.drop-1.5_snpsqc_5.batches_52.csv
    Untracked:  data/ici.vs.eoi_rz.age-no.covariates_blup_sub_chr-3_peak.marker-UNCHS008487_lod.drop-1.5_snpsqc_dis_no-x_updated_5.batches_0.csv
    Untracked:  data/ici.vs.eoi_rz.age-no.covariates_blup_sub_chr-3_peak.marker-UNCHS008487_lod.drop-1.5_snpsqc_dis_no-x_updated_5.batches_52.csv
    Untracked:  data/ici.vs.eoi_rz.age-no.covariates_blup_sub_chr-3_peak.marker-UNCHS008487_lod.drop-1.5_snpsqc_dis_no-x_updated_5.batches_mis_0.csv
    Untracked:  data/ici.vs.eoi_rz.age-no.covariates_blup_sub_chr-3_peak.marker-UNCHS008487_lod.drop-1.5_snpsqc_dis_no-x_updated_5.batches_mis_52.csv
    Untracked:  data/ici.vs.eoi_rz.age-no.covariates_blup_sub_chr-3_peak.marker-UNCHS008511_lod.drop-1.5_snpsqc_5.batches_mis_0.csv
    Untracked:  data/ici.vs.eoi_rz.age-no.covariates_blup_sub_chr-3_peak.marker-UNCHS008511_lod.drop-1.5_snpsqc_5.batches_mis_52.csv
    Untracked:  data/ici.vs.eoi_rz.age-no.covariates_blup_sub_chr-3_peak.marker-UNCHS008595_lod.drop-1.5_5.batches_52.csv
    Untracked:  data/ici.vs.eoi_rz.age-no.covariates_blup_sub_chr-3_peak.marker-UNCHS008613_lod.drop-1.5_5.batches_0.csv
    Untracked:  data/ici.vs.eoi_rz.age-no.covariates_blup_sub_chr-3_peak.marker-UNCHS008613_lod.drop-1.5_5.batches_52.csv
    Untracked:  data/ici.vs.eoi_rz.age-no.covariates_blup_sub_chr-3_peak.marker-UNCHS008815_lod.drop-1.5_5.batches_0.csv
    Untracked:  data/ici.vs.eoi_rz.age-no.covariates_blup_sub_chr-3_peak.marker-UNCHS008815_lod.drop-1.5_5.batches_52.csv
    Untracked:  data/ici.vs.eoi_rz.age-no.covariates_blup_sub_chr-3_peak.marker-UNCHS009067_lod.drop-1.5_5.batches_0.csv
    Untracked:  data/ici.vs.eoi_rz.age-no.covariates_blup_sub_chr-3_peak.marker-UNCHS009383_lod.drop-1.5_5.batches_52.csv
    Untracked:  data/ici.vs.eoi_rz.age-no.covariates_blup_sub_chr-3_peak.marker-UNCHS009693_lod.drop-1.5_5.batches_52.csv
    Untracked:  data/ici.vs.eoi_rz.age-no.covariates_blup_sub_chr-3_peak.marker-UNCJPD001246_lod.drop-1.5_5.batches_52.csv
    Untracked:  data/ici.vs.eoi_rz.age-no.covariates_blup_sub_chr-3_peak.marker-UNCJPD001341_lod.drop-1.5_5.batches_52.csv
    Untracked:  data/ici.vs.eoi_rz.age-no.covariates_blup_sub_chr-3_peak.marker-sanger2623_lod.drop-1.5_5.batches_52.csv
    Untracked:  data/ici.vs.eoi_rz.age-no.covariates_blup_sub_chr-4_peak.marker-UNCHS013125_lod.drop-1.5_snpsqc_dis_no-x_updated_5.batches_52.csv
    Untracked:  data/ici.vs.eoi_rz.age-no.covariates_blup_sub_chr-4_peak.marker-UNCHS013125_lod.drop-1.5_snpsqc_dis_no-x_updated_5.batches_mis_52.csv
    Untracked:  data/ici.vs.eoi_rz.age-no.covariates_blup_sub_chr-6_peak.marker-UNCHS018170_lod.drop-1.5_snpsqc_5.batches_0.csv
    Untracked:  data/ici.vs.eoi_rz.age-no.covariates_blup_sub_chr-6_peak.marker-UNCHS018231_lod.drop-1.5_snpsqc_5.batches_mis_0.csv
    Untracked:  data/ici.vs.eoi_rz.age-no.covariates_blup_sub_chr-7_peak.marker-UNCHS020066_lod.drop-1.5_snpsqc_5.batches_mis_0.csv
    Untracked:  data/ici.vs.eoi_rz.age-no.covariates_blup_sub_chr-7_peak.marker-UNCHS020066_lod.drop-1.5_snpsqc_5.batches_mis_52.csv
    Untracked:  data/ici.vs.eoi_rz.age-no.covariates_blup_sub_chr-7_peak.marker-UNCHS020066_lod.drop-1.5_snpsqc_dis_no-x_updated_5.batches_mis_0.csv
    Untracked:  data/ici.vs.eoi_rz.age-no.covariates_blup_sub_chr-7_peak.marker-UNCHS020066_lod.drop-1.5_snpsqc_dis_no-x_updated_5.batches_mis_52.csv
    Untracked:  data/ici.vs.eoi_rz.age-no.covariates_blup_sub_chr-8_peak.marker-UNC15524531_lod.drop-1.5_snpsqc_5.batches_mis_0.csv
    Untracked:  data/ici.vs.eoi_rz.age-no.covariates_blup_sub_chr-8_peak.marker-UNC15524531_lod.drop-1.5_snpsqc_5.batches_mis_52.csv
    Untracked:  data/ici.vs.eoi_rz.age-no.covariates_blup_sub_chr-8_peak.marker-UNC15524531_lod.drop-1.5_snpsqc_dis_no-x_updated_5.batches_mis_0.csv
    Untracked:  data/ici.vs.eoi_rz.age-no.covariates_blup_sub_chr-8_peak.marker-UNC15524531_lod.drop-1.5_snpsqc_dis_no-x_updated_5.batches_mis_52.csv
    Untracked:  data/ici.vs.eoi_rz.age-no.covariates_blup_sub_chr-9_peak.marker-UNC17203597_lod.drop-1.5_snpsqc_5.batches_mis_0.csv
    Untracked:  data/ici.vs.eoi_rz.age-no.covariates_blup_sub_chr-9_peak.marker-UNC17203597_lod.drop-1.5_snpsqc_5.batches_mis_52.csv
    Untracked:  data/ici.vs.eoi_rz.age-no.covariates_blup_sub_chr-X_peak.marker-UNCHS048314_lod.drop-1.5_snpsqc_5.batches_mis_0.csv
    Untracked:  data/ici.vs.eoi_rz.age-no.covariates_blup_sub_chr-X_peak.marker-UNCHS048314_lod.drop-1.5_snpsqc_5.batches_mis_52.csv
    Untracked:  data/ici.vs.eoi_rz.age-no.covariates_blup_sub_chr-X_peak.marker-UNCHS048314_lod.drop-1.5_snpsqc_dis_no-x_updated_5.batches_mis_0.csv
    Untracked:  data/ici.vs.eoi_rz.age-no.covariates_blup_sub_chr-X_peak.marker-UNCHS048314_lod.drop-1.5_snpsqc_dis_no-x_updated_5.batches_mis_52.csv
    Untracked:  data/ici.vs.eoi_rz.age-no.covariates_genes_chr-10_peak.marker-JAX00292927_lod.drop-1.5_5.batches_0.csv
    Untracked:  data/ici.vs.eoi_rz.age-no.covariates_genes_chr-10_peak.marker-UNC18311938_lod.drop-1.5_5.batches_0.csv
    Untracked:  data/ici.vs.eoi_rz.age-no.covariates_genes_chr-10_peak.marker-UNC18343181_lod.drop-1.5_5.batches_0.csv
    Untracked:  data/ici.vs.eoi_rz.age-no.covariates_genes_chr-10_peak.marker-UNC18376338_lod.drop-1.5_5.batches_0.csv
    Untracked:  data/ici.vs.eoi_rz.age-no.covariates_genes_chr-10_peak.marker-UNC18376338_lod.drop-1.5_snpsqc_5.batches_0.csv
    Untracked:  data/ici.vs.eoi_rz.age-no.covariates_genes_chr-10_peak.marker-UNC18376338_lod.drop-1.5_snpsqc_5.batches_mis_0.csv
    Untracked:  data/ici.vs.eoi_rz.age-no.covariates_genes_chr-10_peak.marker-UNC18382395_lod.drop-1.5_snpsqc_dis_no-x_updated_5.batches_0.csv
    Untracked:  data/ici.vs.eoi_rz.age-no.covariates_genes_chr-10_peak.marker-UNCHS028236_lod.drop-1.5_snpsqc_5.batches_mis_0.csv
    Untracked:  data/ici.vs.eoi_rz.age-no.covariates_genes_chr-10_peak.marker-UNCHS028236_lod.drop-1.5_snpsqc_5.batches_mis_52.csv
    Untracked:  data/ici.vs.eoi_rz.age-no.covariates_genes_chr-10_peak.marker-UNCHS028536_lod.drop-1.5_snpsqc_dis_no-x_updated_5.batches_mis_0.csv
    Untracked:  data/ici.vs.eoi_rz.age-no.covariates_genes_chr-10_peak.marker-UNCHS029024_lod.drop-1.5_5.batches_0.csv
    Untracked:  data/ici.vs.eoi_rz.age-no.covariates_genes_chr-11_peak.marker-UNC19970181_lod.drop-1.5_snpsqc_5.batches_mis_0.csv
    Untracked:  data/ici.vs.eoi_rz.age-no.covariates_genes_chr-11_peak.marker-UNC19970181_lod.drop-1.5_snpsqc_5.batches_mis_52.csv
    Untracked:  data/ici.vs.eoi_rz.age-no.covariates_genes_chr-12_peak.marker-ICR499_lod.drop-1.5_snpsqc_5.batches_mis_0.csv
    Untracked:  data/ici.vs.eoi_rz.age-no.covariates_genes_chr-12_peak.marker-ICR499_lod.drop-1.5_snpsqc_5.batches_mis_52.csv
    Untracked:  data/ici.vs.eoi_rz.age-no.covariates_genes_chr-13_peak.marker-UNCHS036773_lod.drop-1.5_snpsqc_5.batches_mis_0.csv
    Untracked:  data/ici.vs.eoi_rz.age-no.covariates_genes_chr-13_peak.marker-UNCHS036773_lod.drop-1.5_snpsqc_5.batches_mis_52.csv
    Untracked:  data/ici.vs.eoi_rz.age-no.covariates_genes_chr-14_peak.marker-UNC24056202_lod.drop-1.5_snpsqc_5.batches_mis_0.csv
    Untracked:  data/ici.vs.eoi_rz.age-no.covariates_genes_chr-14_peak.marker-UNC24056202_lod.drop-1.5_snpsqc_5.batches_mis_52.csv
    Untracked:  data/ici.vs.eoi_rz.age-no.covariates_genes_chr-15_peak.marker-UNC26070435_lod.drop-1.5_snpsqc_dis_no-x_updated_5.batches_mis_0.csv
    Untracked:  data/ici.vs.eoi_rz.age-no.covariates_genes_chr-15_peak.marker-UNC26070435_lod.drop-1.5_snpsqc_dis_no-x_updated_5.batches_mis_52.csv
    Untracked:  data/ici.vs.eoi_rz.age-no.covariates_genes_chr-17_peak.marker-UNCHS044241_lod.drop-1.5_snpsqc_5.batches_mis_0.csv
    Untracked:  data/ici.vs.eoi_rz.age-no.covariates_genes_chr-17_peak.marker-UNCHS044241_lod.drop-1.5_snpsqc_5.batches_mis_52.csv
    Untracked:  data/ici.vs.eoi_rz.age-no.covariates_genes_chr-17_peak.marker-UNCHS044241_lod.drop-1.5_snpsqc_dis_no-x_updated_5.batches_mis_0.csv
    Untracked:  data/ici.vs.eoi_rz.age-no.covariates_genes_chr-17_peak.marker-UNCrs47191360_lod.drop-1.5_snpsqc_dis_no-x_updated_5.batches_mis_52.csv
    Untracked:  data/ici.vs.eoi_rz.age-no.covariates_genes_chr-18_peak.marker-UNCHS045343_lod.drop-1.5_snpsqc_5.batches_mis_0.csv
    Untracked:  data/ici.vs.eoi_rz.age-no.covariates_genes_chr-18_peak.marker-UNCHS045343_lod.drop-1.5_snpsqc_5.batches_mis_52.csv
    Untracked:  data/ici.vs.eoi_rz.age-no.covariates_genes_chr-1_peak.marker-UNC228500_lod.drop-1.5_snpsqc_dis_no-x_updated_5.batches_52.csv
    Untracked:  data/ici.vs.eoi_rz.age-no.covariates_genes_chr-1_peak.marker-UNC228500_lod.drop-1.5_snpsqc_dis_no-x_updated_5.batches_mis_52.csv
    Untracked:  data/ici.vs.eoi_rz.age-no.covariates_genes_chr-2_peak.marker-UNC4609527_lod.drop-1.5_snpsqc_dis_no-x_updated_5.batches_mis_0.csv
    Untracked:  data/ici.vs.eoi_rz.age-no.covariates_genes_chr-2_peak.marker-UNC4609527_lod.drop-1.5_snpsqc_dis_no-x_updated_5.batches_mis_52.csv
    Untracked:  data/ici.vs.eoi_rz.age-no.covariates_genes_chr-3_peak.marker-ICR1338_lod.drop-1.5_5.batches_0.csv
    Untracked:  data/ici.vs.eoi_rz.age-no.covariates_genes_chr-3_peak.marker-ICR1338_lod.drop-1.5_5.batches_52.csv
    Untracked:  data/ici.vs.eoi_rz.age-no.covariates_genes_chr-3_peak.marker-ICR2047_lod.drop-1.5_5.batches_52.csv
    Untracked:  data/ici.vs.eoi_rz.age-no.covariates_genes_chr-3_peak.marker-JAX00106957_lod.drop-1.5_5.batches_0.csv
    Untracked:  data/ici.vs.eoi_rz.age-no.covariates_genes_chr-3_peak.marker-JAX00112499_lod.drop-1.5_5.batches_52.csv
    Untracked:  data/ici.vs.eoi_rz.age-no.covariates_genes_chr-3_peak.marker-JAX00219554_lod.drop-1.5_5.batches_0.csv
    Untracked:  data/ici.vs.eoi_rz.age-no.covariates_genes_chr-3_peak.marker-JAX00220631_lod.drop-1.5_5.batches_0.csv
    Untracked:  data/ici.vs.eoi_rz.age-no.covariates_genes_chr-3_peak.marker-JAX00514662_lod.drop-1.5_5.batches_52.csv
    Untracked:  data/ici.vs.eoi_rz.age-no.covariates_genes_chr-3_peak.marker-JAX00529707_lod.drop-1.5_5.batches_52.csv
    Untracked:  data/ici.vs.eoi_rz.age-no.covariates_genes_chr-3_peak.marker-UNC4813186_lod.drop-1.5_5.batches_0.csv
    Untracked:  data/ici.vs.eoi_rz.age-no.covariates_genes_chr-3_peak.marker-UNC4813186_lod.drop-1.5_5.batches_52.csv
    Untracked:  data/ici.vs.eoi_rz.age-no.covariates_genes_chr-3_peak.marker-UNC4836550_lod.drop-1.5_5.batches_52.csv
    Untracked:  data/ici.vs.eoi_rz.age-no.covariates_genes_chr-3_peak.marker-UNC4837069_lod.drop-1.5_5.batches_0.csv
    Untracked:  data/ici.vs.eoi_rz.age-no.covariates_genes_chr-3_peak.marker-UNC4912039_lod.drop-1.5_5.batches_0.csv
    Untracked:  data/ici.vs.eoi_rz.age-no.covariates_genes_chr-3_peak.marker-UNC4912039_lod.drop-1.5_5.batches_52.csv
    Untracked:  data/ici.vs.eoi_rz.age-no.covariates_genes_chr-3_peak.marker-UNC4924377_lod.drop-1.5_5.batches_0.csv
    Untracked:  data/ici.vs.eoi_rz.age-no.covariates_genes_chr-3_peak.marker-UNC4924377_lod.drop-1.5_5.batches_52.csv
    Untracked:  data/ici.vs.eoi_rz.age-no.covariates_genes_chr-3_peak.marker-UNC4956618_lod.drop-1.5_5.batches_52.csv
    Untracked:  data/ici.vs.eoi_rz.age-no.covariates_genes_chr-3_peak.marker-UNC4957168_lod.drop-1.5_5.batches_0.csv
    Untracked:  data/ici.vs.eoi_rz.age-no.covariates_genes_chr-3_peak.marker-UNC5006948_lod.drop-1.5_5.batches_52.csv
    Untracked:  data/ici.vs.eoi_rz.age-no.covariates_genes_chr-3_peak.marker-UNC5008656_lod.drop-1.5_5.batches_0.csv
    Untracked:  data/ici.vs.eoi_rz.age-no.covariates_genes_chr-3_peak.marker-UNC5083168_lod.drop-1.5_5.batches_0.csv
    Untracked:  data/ici.vs.eoi_rz.age-no.covariates_genes_chr-3_peak.marker-UNC5083168_lod.drop-1.5_5.batches_52.csv
    Untracked:  data/ici.vs.eoi_rz.age-no.covariates_genes_chr-3_peak.marker-UNC5192582_lod.drop-1.5_5.batches_0.csv
    Untracked:  data/ici.vs.eoi_rz.age-no.covariates_genes_chr-3_peak.marker-UNC5213888_lod.drop-1.5_snpsqc_5.batches_mis_0.csv
    Untracked:  data/ici.vs.eoi_rz.age-no.covariates_genes_chr-3_peak.marker-UNC5213888_lod.drop-1.5_snpsqc_5.batches_mis_52.csv
    Untracked:  data/ici.vs.eoi_rz.age-no.covariates_genes_chr-3_peak.marker-UNC5262472_lod.drop-1.5_5.batches_0.csv
    Untracked:  data/ici.vs.eoi_rz.age-no.covariates_genes_chr-3_peak.marker-UNC5262472_lod.drop-1.5_5.batches_52.csv
    Untracked:  data/ici.vs.eoi_rz.age-no.covariates_genes_chr-3_peak.marker-UNC5266664_lod.drop-1.5_5.batches_0.csv
    Untracked:  data/ici.vs.eoi_rz.age-no.covariates_genes_chr-3_peak.marker-UNC5300878_lod.drop-1.5_5.batches_52.csv
    Untracked:  data/ici.vs.eoi_rz.age-no.covariates_genes_chr-3_peak.marker-UNC5360055_lod.drop-1.5_5.batches_52.csv
    Untracked:  data/ici.vs.eoi_rz.age-no.covariates_genes_chr-3_peak.marker-UNC5454839_lod.drop-1.5_5.batches_0.csv
    Untracked:  data/ici.vs.eoi_rz.age-no.covariates_genes_chr-3_peak.marker-UNC5454839_lod.drop-1.5_5.batches_52.csv
    Untracked:  data/ici.vs.eoi_rz.age-no.covariates_genes_chr-3_peak.marker-UNC5645844_lod.drop-1.5_5.batches_0.csv
    Untracked:  data/ici.vs.eoi_rz.age-no.covariates_genes_chr-3_peak.marker-UNC5645844_lod.drop-1.5_5.batches_52.csv
    Untracked:  data/ici.vs.eoi_rz.age-no.covariates_genes_chr-3_peak.marker-UNC5679155_lod.drop-1.5_5.batches_52.csv
    Untracked:  data/ici.vs.eoi_rz.age-no.covariates_genes_chr-3_peak.marker-UNC5808246_lod.drop-1.5_5.batches_52.csv
    Untracked:  data/ici.vs.eoi_rz.age-no.covariates_genes_chr-3_peak.marker-UNC5945286_lod.drop-1.5_5.batches_52.csv
    Untracked:  data/ici.vs.eoi_rz.age-no.covariates_genes_chr-3_peak.marker-UNC6227106_lod.drop-1.5_5.batches_52.csv
    Untracked:  data/ici.vs.eoi_rz.age-no.covariates_genes_chr-3_peak.marker-UNCHS008182_lod.drop-1.5_5.batches_52.csv
    Untracked:  data/ici.vs.eoi_rz.age-no.covariates_genes_chr-3_peak.marker-UNCHS008286_lod.drop-1.5_5.batches_0.csv
    Untracked:  data/ici.vs.eoi_rz.age-no.covariates_genes_chr-3_peak.marker-UNCHS008286_lod.drop-1.5_5.batches_52.csv
    Untracked:  data/ici.vs.eoi_rz.age-no.covariates_genes_chr-3_peak.marker-UNCHS008487_lod.drop-1.5_snpsqc_5.batches_0.csv
    Untracked:  data/ici.vs.eoi_rz.age-no.covariates_genes_chr-3_peak.marker-UNCHS008487_lod.drop-1.5_snpsqc_5.batches_52.csv
    Untracked:  data/ici.vs.eoi_rz.age-no.covariates_genes_chr-3_peak.marker-UNCHS008487_lod.drop-1.5_snpsqc_dis_no-x_updated_5.batches_0.csv
    Untracked:  data/ici.vs.eoi_rz.age-no.covariates_genes_chr-3_peak.marker-UNCHS008487_lod.drop-1.5_snpsqc_dis_no-x_updated_5.batches_52.csv
    Untracked:  data/ici.vs.eoi_rz.age-no.covariates_genes_chr-3_peak.marker-UNCHS008487_lod.drop-1.5_snpsqc_dis_no-x_updated_5.batches_mis_0.csv
    Untracked:  data/ici.vs.eoi_rz.age-no.covariates_genes_chr-3_peak.marker-UNCHS008487_lod.drop-1.5_snpsqc_dis_no-x_updated_5.batches_mis_52.csv
    Untracked:  data/ici.vs.eoi_rz.age-no.covariates_genes_chr-3_peak.marker-UNCHS008511_lod.drop-1.5_snpsqc_5.batches_mis_0.csv
    Untracked:  data/ici.vs.eoi_rz.age-no.covariates_genes_chr-3_peak.marker-UNCHS008511_lod.drop-1.5_snpsqc_5.batches_mis_52.csv
    Untracked:  data/ici.vs.eoi_rz.age-no.covariates_genes_chr-3_peak.marker-UNCHS008595_lod.drop-1.5_5.batches_52.csv
    Untracked:  data/ici.vs.eoi_rz.age-no.covariates_genes_chr-3_peak.marker-UNCHS008613_lod.drop-1.5_5.batches_0.csv
    Untracked:  data/ici.vs.eoi_rz.age-no.covariates_genes_chr-3_peak.marker-UNCHS008613_lod.drop-1.5_5.batches_52.csv
    Untracked:  data/ici.vs.eoi_rz.age-no.covariates_genes_chr-3_peak.marker-UNCHS008815_lod.drop-1.5_5.batches_0.csv
    Untracked:  data/ici.vs.eoi_rz.age-no.covariates_genes_chr-3_peak.marker-UNCHS008815_lod.drop-1.5_5.batches_52.csv
    Untracked:  data/ici.vs.eoi_rz.age-no.covariates_genes_chr-3_peak.marker-UNCHS009067_lod.drop-1.5_5.batches_0.csv
    Untracked:  data/ici.vs.eoi_rz.age-no.covariates_genes_chr-3_peak.marker-UNCHS009383_lod.drop-1.5_5.batches_52.csv
    Untracked:  data/ici.vs.eoi_rz.age-no.covariates_genes_chr-3_peak.marker-UNCHS009693_lod.drop-1.5_5.batches_52.csv
    Untracked:  data/ici.vs.eoi_rz.age-no.covariates_genes_chr-3_peak.marker-UNCJPD001246_lod.drop-1.5_5.batches_52.csv
    Untracked:  data/ici.vs.eoi_rz.age-no.covariates_genes_chr-3_peak.marker-UNCJPD001341_lod.drop-1.5_5.batches_52.csv
    Untracked:  data/ici.vs.eoi_rz.age-no.covariates_genes_chr-3_peak.marker-sanger2623_lod.drop-1.5_5.batches_52.csv
    Untracked:  data/ici.vs.eoi_rz.age-no.covariates_genes_chr-4_peak.marker-UNCHS013125_lod.drop-1.5_snpsqc_dis_no-x_updated_5.batches_52.csv
    Untracked:  data/ici.vs.eoi_rz.age-no.covariates_genes_chr-4_peak.marker-UNCHS013125_lod.drop-1.5_snpsqc_dis_no-x_updated_5.batches_mis_52.csv
    Untracked:  data/ici.vs.eoi_rz.age-no.covariates_genes_chr-6_peak.marker-UNCHS018170_lod.drop-1.5_snpsqc_5.batches_0.csv
    Untracked:  data/ici.vs.eoi_rz.age-no.covariates_genes_chr-6_peak.marker-UNCHS018231_lod.drop-1.5_snpsqc_5.batches_mis_0.csv
    Untracked:  data/ici.vs.eoi_rz.age-no.covariates_genes_chr-7_peak.marker-UNCHS020066_lod.drop-1.5_snpsqc_5.batches_mis_0.csv
    Untracked:  data/ici.vs.eoi_rz.age-no.covariates_genes_chr-7_peak.marker-UNCHS020066_lod.drop-1.5_snpsqc_5.batches_mis_52.csv
    Untracked:  data/ici.vs.eoi_rz.age-no.covariates_genes_chr-7_peak.marker-UNCHS020066_lod.drop-1.5_snpsqc_dis_no-x_updated_5.batches_mis_0.csv
    Untracked:  data/ici.vs.eoi_rz.age-no.covariates_genes_chr-7_peak.marker-UNCHS020066_lod.drop-1.5_snpsqc_dis_no-x_updated_5.batches_mis_52.csv
    Untracked:  data/ici.vs.eoi_rz.age-no.covariates_genes_chr-8_peak.marker-UNC15524531_lod.drop-1.5_snpsqc_5.batches_mis_0.csv
    Untracked:  data/ici.vs.eoi_rz.age-no.covariates_genes_chr-8_peak.marker-UNC15524531_lod.drop-1.5_snpsqc_5.batches_mis_52.csv
    Untracked:  data/ici.vs.eoi_rz.age-no.covariates_genes_chr-8_peak.marker-UNC15524531_lod.drop-1.5_snpsqc_dis_no-x_updated_5.batches_mis_0.csv
    Untracked:  data/ici.vs.eoi_rz.age-no.covariates_genes_chr-8_peak.marker-UNC15524531_lod.drop-1.5_snpsqc_dis_no-x_updated_5.batches_mis_52.csv
    Untracked:  data/ici.vs.eoi_rz.age-no.covariates_genes_chr-9_peak.marker-UNC17203597_lod.drop-1.5_snpsqc_5.batches_mis_0.csv
    Untracked:  data/ici.vs.eoi_rz.age-no.covariates_genes_chr-9_peak.marker-UNC17203597_lod.drop-1.5_snpsqc_5.batches_mis_52.csv
    Untracked:  data/ici.vs.eoi_rz.age-no.covariates_genes_chr-X_peak.marker-UNCHS048314_lod.drop-1.5_snpsqc_5.batches_mis_0.csv
    Untracked:  data/ici.vs.eoi_rz.age-no.covariates_genes_chr-X_peak.marker-UNCHS048314_lod.drop-1.5_snpsqc_5.batches_mis_52.csv
    Untracked:  data/ici.vs.eoi_rz.age-no.covariates_genes_chr-X_peak.marker-UNCHS048314_lod.drop-1.5_snpsqc_dis_no-x_updated_5.batches_mis_0.csv
    Untracked:  data/ici.vs.eoi_rz.age-no.covariates_genes_chr-X_peak.marker-UNCHS048314_lod.drop-1.5_snpsqc_dis_no-x_updated_5.batches_mis_52.csv
    Untracked:  data/ici.vs.eoi_sample.genos_marker.freq_low.geno.freq.removed.csv
    Untracked:  data/ici.vs.eoi_sample.genos_marker.freq_low.geno.freq.removed_sample.outliers.removed.csv
    Untracked:  data/ici.vs.eoi_sample.genos_marker.freq_low.probs.freq.removed.csv
    Untracked:  data/ici.vs.eoi_sample.genos_marker.freq_low.probs.freq.removed_sample.outliers.removed.csv
    Untracked:  data/ici.vs.eoi_scanone_5.batches.Rdata
    Untracked:  data/ici.vs.eoi_scanone_5.batches_mis.Rdata
    Untracked:  data/ici.vs.eoi_scanone_snpsqc_5.batches.Rdata
    Untracked:  data/ici.vs.eoi_scanone_snpsqc_5.batches_mis.Rdata
    Untracked:  data/ici.vs.eoi_scanone_snpsqc_dis_no-x_updated_5.batches.Rdata
    Untracked:  data/ici.vs.pbs_age.of.onset-no.covariates_blup_sub_chr-14_peak.marker-UNC24056202_lod.drop-1.5_5.batches_mis.csv
    Untracked:  data/ici.vs.pbs_age.of.onset-no.covariates_blup_sub_chr-14_peak.marker-UNC24056202_lod.drop-1.5_snpsqc_5.batches_mis.csv
    Untracked:  data/ici.vs.pbs_age.of.onset-no.covariates_blup_sub_chr-17_peak.marker-UNCJPD006614_lod.drop-1.5_snpsqc_5.batches_mis.csv
    Untracked:  data/ici.vs.pbs_age.of.onset-no.covariates_blup_sub_chr-18_peak.marker-UNC28655293_lod.drop-1.5_5.batches_mis.csv
    Untracked:  data/ici.vs.pbs_age.of.onset-no.covariates_blup_sub_chr-18_peak.marker-UNCHS045343_lod.drop-1.5_snpsqc_5.batches_mis.csv
    Untracked:  data/ici.vs.pbs_age.of.onset-no.covariates_blup_sub_chr-5_peak.marker-UNC10044126_lod.drop-1.5_5.batches_mis.csv
    Untracked:  data/ici.vs.pbs_age.of.onset-no.covariates_blup_sub_chr-5_peak.marker-UNC8675939_lod.drop-1.5_snpsqc_5.batches_mis.csv
    Untracked:  data/ici.vs.pbs_age.of.onset-no.covariates_blup_sub_chr-8_peak.marker-JAX00667121_lod.drop-1.5_5.batches_mis.csv
    Untracked:  data/ici.vs.pbs_age.of.onset-no.covariates_blup_sub_chr-8_peak.marker-UNC15524531_lod.drop-1.5_snpsqc_5.batches_mis.csv
    Untracked:  data/ici.vs.pbs_age.of.onset-no.covariates_blup_sub_chr-X_peak.marker-UNCHS048314_lod.drop-1.5_snpsqc_5.batches_mis.csv
    Untracked:  data/ici.vs.pbs_age.of.onset-no.covariates_genes_chr-14_peak.marker-UNC24056202_lod.drop-1.5_5.batches_mis.csv
    Untracked:  data/ici.vs.pbs_age.of.onset-no.covariates_genes_chr-14_peak.marker-UNC24056202_lod.drop-1.5_snpsqc_5.batches_mis.csv
    Untracked:  data/ici.vs.pbs_age.of.onset-no.covariates_genes_chr-17_peak.marker-UNCJPD006614_lod.drop-1.5_snpsqc_5.batches_mis.csv
    Untracked:  data/ici.vs.pbs_age.of.onset-no.covariates_genes_chr-18_peak.marker-UNC28655293_lod.drop-1.5_5.batches_mis.csv
    Untracked:  data/ici.vs.pbs_age.of.onset-no.covariates_genes_chr-18_peak.marker-UNCHS045343_lod.drop-1.5_snpsqc_5.batches_mis.csv
    Untracked:  data/ici.vs.pbs_age.of.onset-no.covariates_genes_chr-5_peak.marker-UNC10044126_lod.drop-1.5_5.batches_mis.csv
    Untracked:  data/ici.vs.pbs_age.of.onset-no.covariates_genes_chr-5_peak.marker-UNC8675939_lod.drop-1.5_snpsqc_5.batches_mis.csv
    Untracked:  data/ici.vs.pbs_age.of.onset-no.covariates_genes_chr-8_peak.marker-JAX00667121_lod.drop-1.5_5.batches_mis.csv
    Untracked:  data/ici.vs.pbs_age.of.onset-no.covariates_genes_chr-8_peak.marker-UNC15524531_lod.drop-1.5_snpsqc_5.batches_mis.csv
    Untracked:  data/ici.vs.pbs_age.of.onset-no.covariates_genes_chr-X_peak.marker-UNCHS048314_lod.drop-1.5_snpsqc_5.batches_mis.csv
    Untracked:  data/ici.vs.pbs_bad.markers_snpsqc_5.batches_mis.csv
    Untracked:  data/ici.vs.pbs_blup.full_chr-10_lod.drop-1.5_5.batches.csv
    Untracked:  data/ici.vs.pbs_blup.full_chr-10_snpsqc_5.batches.csv
    Untracked:  data/ici.vs.pbs_blup.full_chr-10_snpsqc_5.batches_mis.csv
    Untracked:  data/ici.vs.pbs_blup.full_chr-10_snpsqc_dis_no-x_updated_5.batches.csv
    Untracked:  data/ici.vs.pbs_blup.full_chr-10_snpsqc_dis_no-x_updated_5.batches_mis.csv
    Untracked:  data/ici.vs.pbs_blup.full_chr-11_lod.drop-1.5_5.batches.csv
    Untracked:  data/ici.vs.pbs_blup.full_chr-11_snpsqc_5.batches.csv
    Untracked:  data/ici.vs.pbs_blup.full_chr-11_snpsqc_5.batches_mis.csv
    Untracked:  data/ici.vs.pbs_blup.full_chr-11_snpsqc_dis_no-x_updated_5.batches.csv
    Untracked:  data/ici.vs.pbs_blup.full_chr-11_snpsqc_dis_no-x_updated_5.batches_mis.csv
    Untracked:  data/ici.vs.pbs_blup.full_chr-12_lod.drop-1.5_5.batches.csv
    Untracked:  data/ici.vs.pbs_blup.full_chr-12_snpsqc_5.batches.csv
    Untracked:  data/ici.vs.pbs_blup.full_chr-12_snpsqc_5.batches_mis.csv
    Untracked:  data/ici.vs.pbs_blup.full_chr-12_snpsqc_dis_no-x_updated_5.batches.csv
    Untracked:  data/ici.vs.pbs_blup.full_chr-12_snpsqc_dis_no-x_updated_5.batches_mis.csv
    Untracked:  data/ici.vs.pbs_blup.full_chr-13_lod.drop-1.5_5.batches.csv
    Untracked:  data/ici.vs.pbs_blup.full_chr-13_snpsqc_5.batches.csv
    Untracked:  data/ici.vs.pbs_blup.full_chr-13_snpsqc_5.batches_mis.csv
    Untracked:  data/ici.vs.pbs_blup.full_chr-13_snpsqc_dis_no-x_updated_5.batches.csv
    Untracked:  data/ici.vs.pbs_blup.full_chr-13_snpsqc_dis_no-x_updated_5.batches_mis.csv
    Untracked:  data/ici.vs.pbs_blup.full_chr-14_lod.drop-1.5_5.batches.csv
    Untracked:  data/ici.vs.pbs_blup.full_chr-14_snpsqc_5.batches.csv
    Untracked:  data/ici.vs.pbs_blup.full_chr-14_snpsqc_5.batches_mis.csv
    Untracked:  data/ici.vs.pbs_blup.full_chr-14_snpsqc_dis_no-x_updated_5.batches.csv
    Untracked:  data/ici.vs.pbs_blup.full_chr-14_snpsqc_dis_no-x_updated_5.batches_mis.csv
    Untracked:  data/ici.vs.pbs_blup.full_chr-15_lod.drop-1.5_5.batches.csv
    Untracked:  data/ici.vs.pbs_blup.full_chr-15_snpsqc_5.batches.csv
    Untracked:  data/ici.vs.pbs_blup.full_chr-15_snpsqc_5.batches_mis.csv
    Untracked:  data/ici.vs.pbs_blup.full_chr-15_snpsqc_dis_no-x_updated_5.batches.csv
    Untracked:  data/ici.vs.pbs_blup.full_chr-15_snpsqc_dis_no-x_updated_5.batches_mis.csv
    Untracked:  data/ici.vs.pbs_blup.full_chr-16_lod.drop-1.5_5.batches.csv
    Untracked:  data/ici.vs.pbs_blup.full_chr-16_snpsqc_5.batches.csv
    Untracked:  data/ici.vs.pbs_blup.full_chr-16_snpsqc_5.batches_mis.csv
    Untracked:  data/ici.vs.pbs_blup.full_chr-16_snpsqc_dis_no-x_updated_5.batches.csv
    Untracked:  data/ici.vs.pbs_blup.full_chr-16_snpsqc_dis_no-x_updated_5.batches_mis.csv
    Untracked:  data/ici.vs.pbs_blup.full_chr-17_lod.drop-1.5_5.batches.csv
    Untracked:  data/ici.vs.pbs_blup.full_chr-17_snpsqc_5.batches.csv
    Untracked:  data/ici.vs.pbs_blup.full_chr-17_snpsqc_5.batches_mis.csv
    Untracked:  data/ici.vs.pbs_blup.full_chr-17_snpsqc_dis_no-x_updated_5.batches.csv
    Untracked:  data/ici.vs.pbs_blup.full_chr-17_snpsqc_dis_no-x_updated_5.batches_mis.csv
    Untracked:  data/ici.vs.pbs_blup.full_chr-18_lod.drop-1.5_5.batches.csv
    Untracked:  data/ici.vs.pbs_blup.full_chr-18_snpsqc_5.batches.csv
    Untracked:  data/ici.vs.pbs_blup.full_chr-18_snpsqc_5.batches_mis.csv
    Untracked:  data/ici.vs.pbs_blup.full_chr-18_snpsqc_dis_no-x_updated_5.batches.csv
    Untracked:  data/ici.vs.pbs_blup.full_chr-18_snpsqc_dis_no-x_updated_5.batches_mis.csv
    Untracked:  data/ici.vs.pbs_blup.full_chr-19_lod.drop-1.5_5.batches.csv
    Untracked:  data/ici.vs.pbs_blup.full_chr-19_snpsqc_5.batches.csv
    Untracked:  data/ici.vs.pbs_blup.full_chr-19_snpsqc_5.batches_mis.csv
    Untracked:  data/ici.vs.pbs_blup.full_chr-19_snpsqc_dis_no-x_updated_5.batches.csv
    Untracked:  data/ici.vs.pbs_blup.full_chr-19_snpsqc_dis_no-x_updated_5.batches_mis.csv
    Untracked:  data/ici.vs.pbs_blup.full_chr-1_lod.drop-1.5_5.batches.csv
    Untracked:  data/ici.vs.pbs_blup.full_chr-1_snpsqc_5.batches.csv
    Untracked:  data/ici.vs.pbs_blup.full_chr-1_snpsqc_5.batches_mis.csv
    Untracked:  data/ici.vs.pbs_blup.full_chr-1_snpsqc_dis_no-x_updated_5.batches.csv
    Untracked:  data/ici.vs.pbs_blup.full_chr-1_snpsqc_dis_no-x_updated_5.batches_mis.csv
    Untracked:  data/ici.vs.pbs_blup.full_chr-2_lod.drop-1.5_5.batches.csv
    Untracked:  data/ici.vs.pbs_blup.full_chr-2_snpsqc_5.batches.csv
    Untracked:  data/ici.vs.pbs_blup.full_chr-2_snpsqc_5.batches_mis.csv
    Untracked:  data/ici.vs.pbs_blup.full_chr-2_snpsqc_dis_no-x_updated_5.batches.csv
    Untracked:  data/ici.vs.pbs_blup.full_chr-2_snpsqc_dis_no-x_updated_5.batches_mis.csv
    Untracked:  data/ici.vs.pbs_blup.full_chr-3_lod.drop-1.5_5.batches.csv
    Untracked:  data/ici.vs.pbs_blup.full_chr-3_snpsqc_5.batches.csv
    Untracked:  data/ici.vs.pbs_blup.full_chr-3_snpsqc_5.batches_mis.csv
    Untracked:  data/ici.vs.pbs_blup.full_chr-3_snpsqc_dis_no-x_updated_5.batches.csv
    Untracked:  data/ici.vs.pbs_blup.full_chr-3_snpsqc_dis_no-x_updated_5.batches_mis.csv
    Untracked:  data/ici.vs.pbs_blup.full_chr-4_lod.drop-1.5_5.batches.csv
    Untracked:  data/ici.vs.pbs_blup.full_chr-4_snpsqc_5.batches.csv
    Untracked:  data/ici.vs.pbs_blup.full_chr-4_snpsqc_5.batches_mis.csv
    Untracked:  data/ici.vs.pbs_blup.full_chr-4_snpsqc_dis_no-x_updated_5.batches.csv
    Untracked:  data/ici.vs.pbs_blup.full_chr-4_snpsqc_dis_no-x_updated_5.batches_mis.csv
    Untracked:  data/ici.vs.pbs_blup.full_chr-5_lod.drop-1.5_5.batches.csv
    Untracked:  data/ici.vs.pbs_blup.full_chr-5_snpsqc_5.batches.csv
    Untracked:  data/ici.vs.pbs_blup.full_chr-5_snpsqc_5.batches_mis.csv
    Untracked:  data/ici.vs.pbs_blup.full_chr-5_snpsqc_dis_no-x_updated_5.batches.csv
    Untracked:  data/ici.vs.pbs_blup.full_chr-5_snpsqc_dis_no-x_updated_5.batches_mis.csv
    Untracked:  data/ici.vs.pbs_blup.full_chr-6_lod.drop-1.5_5.batches.csv
    Untracked:  data/ici.vs.pbs_blup.full_chr-6_snpsqc_5.batches.csv
    Untracked:  data/ici.vs.pbs_blup.full_chr-6_snpsqc_5.batches_mis.csv
    Untracked:  data/ici.vs.pbs_blup.full_chr-6_snpsqc_dis_no-x_updated_5.batches.csv
    Untracked:  data/ici.vs.pbs_blup.full_chr-6_snpsqc_dis_no-x_updated_5.batches_mis.csv
    Untracked:  data/ici.vs.pbs_blup.full_chr-7_lod.drop-1.5_5.batches.csv
    Untracked:  data/ici.vs.pbs_blup.full_chr-7_snpsqc_5.batches.csv
    Untracked:  data/ici.vs.pbs_blup.full_chr-7_snpsqc_5.batches_mis.csv
    Untracked:  data/ici.vs.pbs_blup.full_chr-7_snpsqc_dis_no-x_updated_5.batches.csv
    Untracked:  data/ici.vs.pbs_blup.full_chr-7_snpsqc_dis_no-x_updated_5.batches_mis.csv
    Untracked:  data/ici.vs.pbs_blup.full_chr-8_lod.drop-1.5_5.batches.csv
    Untracked:  data/ici.vs.pbs_blup.full_chr-8_snpsqc_5.batches.csv
    Untracked:  data/ici.vs.pbs_blup.full_chr-8_snpsqc_5.batches_mis.csv
    Untracked:  data/ici.vs.pbs_blup.full_chr-8_snpsqc_dis_no-x_updated_5.batches.csv
    Untracked:  data/ici.vs.pbs_blup.full_chr-8_snpsqc_dis_no-x_updated_5.batches_mis.csv
    Untracked:  data/ici.vs.pbs_blup.full_chr-9_lod.drop-1.5_5.batches.csv
    Untracked:  data/ici.vs.pbs_blup.full_chr-9_snpsqc_5.batches.csv
    Untracked:  data/ici.vs.pbs_blup.full_chr-9_snpsqc_5.batches_mis.csv
    Untracked:  data/ici.vs.pbs_blup.full_chr-9_snpsqc_dis_no-x_updated_5.batches.csv
    Untracked:  data/ici.vs.pbs_blup.full_chr-9_snpsqc_dis_no-x_updated_5.batches_mis.csv
    Untracked:  data/ici.vs.pbs_blup.full_chr-X_lod.drop-1.5_5.batches.csv
    Untracked:  data/ici.vs.pbs_blup.full_chr-X_snpsqc_5.batches.csv
    Untracked:  data/ici.vs.pbs_blup.full_chr-X_snpsqc_5.batches_mis.csv
    Untracked:  data/ici.vs.pbs_blup.full_chr-X_snpsqc_dis_no-x_updated_5.batches.csv
    Untracked:  data/ici.vs.pbs_blup.full_chr-X_snpsqc_dis_no-x_updated_5.batches_mis.csv
    Untracked:  data/ici.vs.pbs_blup.qc_chr-10_5.batches.csv
    Untracked:  data/ici.vs.pbs_blup.qc_chr-10_5.batches_mis.csv
    Untracked:  data/ici.vs.pbs_blup.qc_chr-10_lod.drop-1.5_5.batches.csv
    Untracked:  data/ici.vs.pbs_blup.qc_chr-10_lod.drop-1.5_5.batches_mis.csv
    Untracked:  data/ici.vs.pbs_blup.qc_chr-10_snpsqc_5.batches.csv
    Untracked:  data/ici.vs.pbs_blup.qc_chr-10_snpsqc_5.batches_mis.csv
    Untracked:  data/ici.vs.pbs_blup.qc_chr-10_snpsqc_dis_no-x_updated_5.batches.csv
    Untracked:  data/ici.vs.pbs_blup.qc_chr-10_snpsqc_dis_no-x_updated_5.batches_mis.csv
    Untracked:  data/ici.vs.pbs_blup.qc_chr-11_5.batches.csv
    Untracked:  data/ici.vs.pbs_blup.qc_chr-11_5.batches_mis.csv
    Untracked:  data/ici.vs.pbs_blup.qc_chr-11_lod.drop-1.5_5.batches.csv
    Untracked:  data/ici.vs.pbs_blup.qc_chr-11_lod.drop-1.5_5.batches_mis.csv
    Untracked:  data/ici.vs.pbs_blup.qc_chr-11_snpsqc_5.batches.csv
    Untracked:  data/ici.vs.pbs_blup.qc_chr-11_snpsqc_5.batches_mis.csv
    Untracked:  data/ici.vs.pbs_blup.qc_chr-11_snpsqc_dis_no-x_updated_5.batches.csv
    Untracked:  data/ici.vs.pbs_blup.qc_chr-11_snpsqc_dis_no-x_updated_5.batches_mis.csv
    Untracked:  data/ici.vs.pbs_blup.qc_chr-12_5.batches.csv
    Untracked:  data/ici.vs.pbs_blup.qc_chr-12_5.batches_mis.csv
    Untracked:  data/ici.vs.pbs_blup.qc_chr-12_lod.drop-1.5_5.batches.csv
    Untracked:  data/ici.vs.pbs_blup.qc_chr-12_lod.drop-1.5_5.batches_mis.csv
    Untracked:  data/ici.vs.pbs_blup.qc_chr-12_snpsqc_5.batches.csv
    Untracked:  data/ici.vs.pbs_blup.qc_chr-12_snpsqc_5.batches_mis.csv
    Untracked:  data/ici.vs.pbs_blup.qc_chr-12_snpsqc_dis_no-x_updated_5.batches.csv
    Untracked:  data/ici.vs.pbs_blup.qc_chr-12_snpsqc_dis_no-x_updated_5.batches_mis.csv
    Untracked:  data/ici.vs.pbs_blup.qc_chr-13_5.batches.csv
    Untracked:  data/ici.vs.pbs_blup.qc_chr-13_5.batches_mis.csv
    Untracked:  data/ici.vs.pbs_blup.qc_chr-13_lod.drop-1.5_5.batches.csv
    Untracked:  data/ici.vs.pbs_blup.qc_chr-13_lod.drop-1.5_5.batches_mis.csv
    Untracked:  data/ici.vs.pbs_blup.qc_chr-13_snpsqc_5.batches.csv
    Untracked:  data/ici.vs.pbs_blup.qc_chr-13_snpsqc_5.batches_mis.csv
    Untracked:  data/ici.vs.pbs_blup.qc_chr-13_snpsqc_dis_no-x_updated_5.batches.csv
    Untracked:  data/ici.vs.pbs_blup.qc_chr-13_snpsqc_dis_no-x_updated_5.batches_mis.csv
    Untracked:  data/ici.vs.pbs_blup.qc_chr-14_5.batches.csv
    Untracked:  data/ici.vs.pbs_blup.qc_chr-14_5.batches_mis.csv
    Untracked:  data/ici.vs.pbs_blup.qc_chr-14_lod.drop-1.5_5.batches.csv
    Untracked:  data/ici.vs.pbs_blup.qc_chr-14_snpsqc_5.batches.csv
    Untracked:  data/ici.vs.pbs_blup.qc_chr-14_snpsqc_5.batches_mis.csv
    Untracked:  data/ici.vs.pbs_blup.qc_chr-14_snpsqc_dis_no-x_updated_5.batches.csv
    Untracked:  data/ici.vs.pbs_blup.qc_chr-14_snpsqc_dis_no-x_updated_5.batches_mis.csv
    Untracked:  data/ici.vs.pbs_blup.qc_chr-15_5.batches.csv
    Untracked:  data/ici.vs.pbs_blup.qc_chr-15_5.batches_mis.csv
    Untracked:  data/ici.vs.pbs_blup.qc_chr-15_lod.drop-1.5_5.batches.csv
    Untracked:  data/ici.vs.pbs_blup.qc_chr-15_snpsqc_5.batches.csv
    Untracked:  data/ici.vs.pbs_blup.qc_chr-15_snpsqc_5.batches_mis.csv
    Untracked:  data/ici.vs.pbs_blup.qc_chr-16_5.batches.csv
    Untracked:  data/ici.vs.pbs_blup.qc_chr-16_5.batches_mis.csv
    Untracked:  data/ici.vs.pbs_blup.qc_chr-16_lod.drop-1.5_5.batches.csv
    Untracked:  data/ici.vs.pbs_blup.qc_chr-16_snpsqc_5.batches.csv
    Untracked:  data/ici.vs.pbs_blup.qc_chr-16_snpsqc_5.batches_mis.csv
    Untracked:  data/ici.vs.pbs_blup.qc_chr-16_snpsqc_dis_no-x_updated_5.batches.csv
    Untracked:  data/ici.vs.pbs_blup.qc_chr-16_snpsqc_dis_no-x_updated_5.batches_mis.csv
    Untracked:  data/ici.vs.pbs_blup.qc_chr-17_5.batches.csv
    Untracked:  data/ici.vs.pbs_blup.qc_chr-17_5.batches_mis.csv
    Untracked:  data/ici.vs.pbs_blup.qc_chr-17_lod.drop-1.5_5.batches.csv
    Untracked:  data/ici.vs.pbs_blup.qc_chr-17_snpsqc_5.batches.csv
    Untracked:  data/ici.vs.pbs_blup.qc_chr-17_snpsqc_5.batches_mis.csv
    Untracked:  data/ici.vs.pbs_blup.qc_chr-18_5.batches.csv
    Untracked:  data/ici.vs.pbs_blup.qc_chr-18_5.batches_mis.csv
    Untracked:  data/ici.vs.pbs_blup.qc_chr-18_lod.drop-1.5_5.batches.csv
    Untracked:  data/ici.vs.pbs_blup.qc_chr-18_snpsqc_5.batches.csv
    Untracked:  data/ici.vs.pbs_blup.qc_chr-18_snpsqc_5.batches_mis.csv
    Untracked:  data/ici.vs.pbs_blup.qc_chr-18_snpsqc_dis_no-x_updated_5.batches.csv
    Untracked:  data/ici.vs.pbs_blup.qc_chr-18_snpsqc_dis_no-x_updated_5.batches_mis.csv
    Untracked:  data/ici.vs.pbs_blup.qc_chr-19_5.batches.csv
    Untracked:  data/ici.vs.pbs_blup.qc_chr-19_5.batches_mis.csv
    Untracked:  data/ici.vs.pbs_blup.qc_chr-19_lod.drop-1.5_5.batches.csv
    Untracked:  data/ici.vs.pbs_blup.qc_chr-19_snpsqc_5.batches.csv
    Untracked:  data/ici.vs.pbs_blup.qc_chr-19_snpsqc_5.batches_mis.csv
    Untracked:  data/ici.vs.pbs_blup.qc_chr-19_snpsqc_dis_no-x_updated_5.batches.csv
    Untracked:  data/ici.vs.pbs_blup.qc_chr-19_snpsqc_dis_no-x_updated_5.batches_mis.csv
    Untracked:  data/ici.vs.pbs_blup.qc_chr-1_5.batches.csv
    Untracked:  data/ici.vs.pbs_blup.qc_chr-1_5.batches_mis.csv
    Untracked:  data/ici.vs.pbs_blup.qc_chr-1_lod.drop-1.5_5.batches.csv
    Untracked:  data/ici.vs.pbs_blup.qc_chr-1_lod.drop-1.5_5.batches_mis.csv
    Untracked:  data/ici.vs.pbs_blup.qc_chr-1_snpsqc_5.batches.csv
    Untracked:  data/ici.vs.pbs_blup.qc_chr-1_snpsqc_5.batches_mis.csv
    Untracked:  data/ici.vs.pbs_blup.qc_chr-1_snpsqc_dis_no-x_updated_5.batches.csv
    Untracked:  data/ici.vs.pbs_blup.qc_chr-1_snpsqc_dis_no-x_updated_5.batches_mis.csv
    Untracked:  data/ici.vs.pbs_blup.qc_chr-2_5.batches.csv
    Untracked:  data/ici.vs.pbs_blup.qc_chr-2_5.batches_mis.csv
    Untracked:  data/ici.vs.pbs_blup.qc_chr-2_lod.drop-1.5_5.batches.csv
    Untracked:  data/ici.vs.pbs_blup.qc_chr-2_lod.drop-1.5_5.batches_mis.csv
    Untracked:  data/ici.vs.pbs_blup.qc_chr-2_snpsqc_5.batches.csv
    Untracked:  data/ici.vs.pbs_blup.qc_chr-2_snpsqc_5.batches_mis.csv
    Untracked:  data/ici.vs.pbs_blup.qc_chr-2_snpsqc_dis_no-x_updated_5.batches.csv
    Untracked:  data/ici.vs.pbs_blup.qc_chr-2_snpsqc_dis_no-x_updated_5.batches_mis.csv
    Untracked:  data/ici.vs.pbs_blup.qc_chr-3_5.batches.csv
    Untracked:  data/ici.vs.pbs_blup.qc_chr-3_5.batches_mis.csv
    Untracked:  data/ici.vs.pbs_blup.qc_chr-3_lod.drop-1.5_5.batches.csv
    Untracked:  data/ici.vs.pbs_blup.qc_chr-3_lod.drop-1.5_5.batches_mis.csv
    Untracked:  data/ici.vs.pbs_blup.qc_chr-3_snpsqc_5.batches.csv
    Untracked:  data/ici.vs.pbs_blup.qc_chr-3_snpsqc_5.batches_mis.csv
    Untracked:  data/ici.vs.pbs_blup.qc_chr-3_snpsqc_dis_no-x_updated_5.batches.csv
    Untracked:  data/ici.vs.pbs_blup.qc_chr-3_snpsqc_dis_no-x_updated_5.batches_mis.csv
    Untracked:  data/ici.vs.pbs_blup.qc_chr-4_5.batches.csv
    Untracked:  data/ici.vs.pbs_blup.qc_chr-4_5.batches_mis.csv
    Untracked:  data/ici.vs.pbs_blup.qc_chr-4_lod.drop-1.5_5.batches.csv
    Untracked:  data/ici.vs.pbs_blup.qc_chr-4_lod.drop-1.5_5.batches_mis.csv
    Untracked:  data/ici.vs.pbs_blup.qc_chr-4_snpsqc_5.batches.csv
    Untracked:  data/ici.vs.pbs_blup.qc_chr-4_snpsqc_5.batches_mis.csv
    Untracked:  data/ici.vs.pbs_blup.qc_chr-4_snpsqc_dis_no-x_updated_5.batches.csv
    Untracked:  data/ici.vs.pbs_blup.qc_chr-4_snpsqc_dis_no-x_updated_5.batches_mis.csv
    Untracked:  data/ici.vs.pbs_blup.qc_chr-5_5.batches.csv
    Untracked:  data/ici.vs.pbs_blup.qc_chr-5_5.batches_mis.csv
    Untracked:  data/ici.vs.pbs_blup.qc_chr-5_lod.drop-1.5_5.batches.csv
    Untracked:  data/ici.vs.pbs_blup.qc_chr-5_lod.drop-1.5_5.batches_mis.csv
    Untracked:  data/ici.vs.pbs_blup.qc_chr-5_snpsqc_5.batches.csv
    Untracked:  data/ici.vs.pbs_blup.qc_chr-5_snpsqc_5.batches_mis.csv
    Untracked:  data/ici.vs.pbs_blup.qc_chr-5_snpsqc_dis_no-x_updated_5.batches.csv
    Untracked:  data/ici.vs.pbs_blup.qc_chr-5_snpsqc_dis_no-x_updated_5.batches_mis.csv
    Untracked:  data/ici.vs.pbs_blup.qc_chr-6_5.batches.csv
    Untracked:  data/ici.vs.pbs_blup.qc_chr-6_5.batches_mis.csv
    Untracked:  data/ici.vs.pbs_blup.qc_chr-6_lod.drop-1.5_5.batches.csv
    Untracked:  data/ici.vs.pbs_blup.qc_chr-6_lod.drop-1.5_5.batches_mis.csv
    Untracked:  data/ici.vs.pbs_blup.qc_chr-6_snpsqc_5.batches.csv
    Untracked:  data/ici.vs.pbs_blup.qc_chr-6_snpsqc_5.batches_mis.csv
    Untracked:  data/ici.vs.pbs_blup.qc_chr-6_snpsqc_dis_no-x_updated_5.batches.csv
    Untracked:  data/ici.vs.pbs_blup.qc_chr-6_snpsqc_dis_no-x_updated_5.batches_mis.csv
    Untracked:  data/ici.vs.pbs_blup.qc_chr-7_5.batches.csv
    Untracked:  data/ici.vs.pbs_blup.qc_chr-7_5.batches_mis.csv
    Untracked:  data/ici.vs.pbs_blup.qc_chr-7_lod.drop-1.5_5.batches.csv
    Untracked:  data/ici.vs.pbs_blup.qc_chr-7_lod.drop-1.5_5.batches_mis.csv
    Untracked:  data/ici.vs.pbs_blup.qc_chr-7_snpsqc_5.batches.csv
    Untracked:  data/ici.vs.pbs_blup.qc_chr-7_snpsqc_5.batches_mis.csv
    Untracked:  data/ici.vs.pbs_blup.qc_chr-7_snpsqc_dis_no-x_updated_5.batches.csv
    Untracked:  data/ici.vs.pbs_blup.qc_chr-7_snpsqc_dis_no-x_updated_5.batches_mis.csv
    Untracked:  data/ici.vs.pbs_blup.qc_chr-8_5.batches.csv
    Untracked:  data/ici.vs.pbs_blup.qc_chr-8_5.batches_mis.csv
    Untracked:  data/ici.vs.pbs_blup.qc_chr-8_lod.drop-1.5_5.batches.csv
    Untracked:  data/ici.vs.pbs_blup.qc_chr-8_lod.drop-1.5_5.batches_mis.csv
    Untracked:  data/ici.vs.pbs_blup.qc_chr-8_snpsqc_5.batches.csv
    Untracked:  data/ici.vs.pbs_blup.qc_chr-8_snpsqc_5.batches_mis.csv
    Untracked:  data/ici.vs.pbs_blup.qc_chr-8_snpsqc_dis_no-x_updated_5.batches.csv
    Untracked:  data/ici.vs.pbs_blup.qc_chr-8_snpsqc_dis_no-x_updated_5.batches_mis.csv
    Untracked:  data/ici.vs.pbs_blup.qc_chr-9_5.batches.csv
    Untracked:  data/ici.vs.pbs_blup.qc_chr-9_5.batches_mis.csv
    Untracked:  data/ici.vs.pbs_blup.qc_chr-9_lod.drop-1.5_5.batches.csv
    Untracked:  data/ici.vs.pbs_blup.qc_chr-9_lod.drop-1.5_5.batches_mis.csv
    Untracked:  data/ici.vs.pbs_blup.qc_chr-9_snpsqc_5.batches.csv
    Untracked:  data/ici.vs.pbs_blup.qc_chr-9_snpsqc_5.batches_mis.csv
    Untracked:  data/ici.vs.pbs_blup.qc_chr-9_snpsqc_dis_no-x_updated_5.batches.csv
    Untracked:  data/ici.vs.pbs_blup.qc_chr-9_snpsqc_dis_no-x_updated_5.batches_mis.csv
    Untracked:  data/ici.vs.pbs_blup.qc_chr-X_5.batches.csv
    Untracked:  data/ici.vs.pbs_blup.qc_chr-X_5.batches_mis.csv
    Untracked:  data/ici.vs.pbs_blup.qc_chr-X_lod.drop-1.5_5.batches.csv
    Untracked:  data/ici.vs.pbs_blup.qc_chr-X_snpsqc_5.batches.csv
    Untracked:  data/ici.vs.pbs_blup.qc_chr-X_snpsqc_5.batches_mis.csv
    Untracked:  data/ici.vs.pbs_blup.qc_chr-X_snpsqc_dis_no-x_updated_5.batches.csv
    Untracked:  data/ici.vs.pbs_blup.qc_chr-X_snpsqc_dis_no-x_updated_5.batches_mis.csv
    Untracked:  data/ici.vs.pbs_blup_chr-10_lod.drop-1.5_5.batches.csv
    Untracked:  data/ici.vs.pbs_blup_chr-10_lod.drop-1.5_snpsqc_5.batches.csv
    Untracked:  data/ici.vs.pbs_blup_chr-10_lod.drop-1.5_snpsqc_5.batches_mis.csv
    Untracked:  data/ici.vs.pbs_blup_chr-10_lod.drop-1.5_snpsqc_dis_no-x_updated_5.batches.csv
    Untracked:  data/ici.vs.pbs_blup_chr-10_lod.drop-1.5_snpsqc_dis_no-x_updated_5.batches_mis.csv
    Untracked:  data/ici.vs.pbs_blup_chr-11_lod.drop-1.5_5.batches.csv
    Untracked:  data/ici.vs.pbs_blup_chr-11_lod.drop-1.5_snpsqc_5.batches.csv
    Untracked:  data/ici.vs.pbs_blup_chr-11_lod.drop-1.5_snpsqc_5.batches_mis.csv
    Untracked:  data/ici.vs.pbs_blup_chr-11_lod.drop-1.5_snpsqc_dis_no-x_updated_5.batches.csv
    Untracked:  data/ici.vs.pbs_blup_chr-11_lod.drop-1.5_snpsqc_dis_no-x_updated_5.batches_mis.csv
    Untracked:  data/ici.vs.pbs_blup_chr-12_lod.drop-1.5_5.batches.csv
    Untracked:  data/ici.vs.pbs_blup_chr-12_lod.drop-1.5_snpsqc_5.batches.csv
    Untracked:  data/ici.vs.pbs_blup_chr-12_lod.drop-1.5_snpsqc_5.batches_mis.csv
    Untracked:  data/ici.vs.pbs_blup_chr-12_lod.drop-1.5_snpsqc_dis_no-x_updated_5.batches.csv
    Untracked:  data/ici.vs.pbs_blup_chr-12_lod.drop-1.5_snpsqc_dis_no-x_updated_5.batches_mis.csv
    Untracked:  data/ici.vs.pbs_blup_chr-13_lod.drop-1.5_5.batches.csv
    Untracked:  data/ici.vs.pbs_blup_chr-13_lod.drop-1.5_snpsqc_5.batches.csv
    Untracked:  data/ici.vs.pbs_blup_chr-13_lod.drop-1.5_snpsqc_5.batches_mis.csv
    Untracked:  data/ici.vs.pbs_blup_chr-13_lod.drop-1.5_snpsqc_dis_no-x_updated_5.batches.csv
    Untracked:  data/ici.vs.pbs_blup_chr-13_lod.drop-1.5_snpsqc_dis_no-x_updated_5.batches_mis.csv
    Untracked:  data/ici.vs.pbs_blup_chr-14_lod.drop-1.5_5.batches.csv
    Untracked:  data/ici.vs.pbs_blup_chr-14_lod.drop-1.5_snpsqc_5.batches.csv
    Untracked:  data/ici.vs.pbs_blup_chr-14_lod.drop-1.5_snpsqc_5.batches_mis.csv
    Untracked:  data/ici.vs.pbs_blup_chr-14_lod.drop-1.5_snpsqc_dis_no-x_updated_5.batches.csv
    Untracked:  data/ici.vs.pbs_blup_chr-14_lod.drop-1.5_snpsqc_dis_no-x_updated_5.batches_mis.csv
    Untracked:  data/ici.vs.pbs_blup_chr-15_lod.drop-1.5_5.batches.csv
    Untracked:  data/ici.vs.pbs_blup_chr-15_lod.drop-1.5_snpsqc_5.batches.csv
    Untracked:  data/ici.vs.pbs_blup_chr-15_lod.drop-1.5_snpsqc_5.batches_mis.csv
    Untracked:  data/ici.vs.pbs_blup_chr-16_lod.drop-1.5_5.batches.csv
    Untracked:  data/ici.vs.pbs_blup_chr-16_lod.drop-1.5_snpsqc_5.batches.csv
    Untracked:  data/ici.vs.pbs_blup_chr-16_lod.drop-1.5_snpsqc_5.batches_mis.csv
    Untracked:  data/ici.vs.pbs_blup_chr-16_lod.drop-1.5_snpsqc_dis_no-x_updated_5.batches.csv
    Untracked:  data/ici.vs.pbs_blup_chr-16_lod.drop-1.5_snpsqc_dis_no-x_updated_5.batches_mis.csv
    Untracked:  data/ici.vs.pbs_blup_chr-17_lod.drop-1.5_5.batches.csv
    Untracked:  data/ici.vs.pbs_blup_chr-17_lod.drop-1.5_snpsqc_5.batches.csv
    Untracked:  data/ici.vs.pbs_blup_chr-17_lod.drop-1.5_snpsqc_5.batches_mis.csv
    Untracked:  data/ici.vs.pbs_blup_chr-18_lod.drop-1.5_5.batches.csv
    Untracked:  data/ici.vs.pbs_blup_chr-18_lod.drop-1.5_snpsqc_5.batches.csv
    Untracked:  data/ici.vs.pbs_blup_chr-18_lod.drop-1.5_snpsqc_5.batches_mis.csv
    Untracked:  data/ici.vs.pbs_blup_chr-18_lod.drop-1.5_snpsqc_dis_no-x_updated_5.batches.csv
    Untracked:  data/ici.vs.pbs_blup_chr-18_lod.drop-1.5_snpsqc_dis_no-x_updated_5.batches_mis.csv
    Untracked:  data/ici.vs.pbs_blup_chr-19_lod.drop-1.5_5.batches.csv
    Untracked:  data/ici.vs.pbs_blup_chr-19_lod.drop-1.5_snpsqc_5.batches.csv
    Untracked:  data/ici.vs.pbs_blup_chr-19_lod.drop-1.5_snpsqc_5.batches_mis.csv
    Untracked:  data/ici.vs.pbs_blup_chr-19_lod.drop-1.5_snpsqc_dis_no-x_updated_5.batches.csv
    Untracked:  data/ici.vs.pbs_blup_chr-19_lod.drop-1.5_snpsqc_dis_no-x_updated_5.batches_mis.csv
    Untracked:  data/ici.vs.pbs_blup_chr-1_lod.drop-1.5_5.batches.csv
    Untracked:  data/ici.vs.pbs_blup_chr-1_lod.drop-1.5_snpsqc_5.batches.csv
    Untracked:  data/ici.vs.pbs_blup_chr-1_lod.drop-1.5_snpsqc_5.batches_mis.csv
    Untracked:  data/ici.vs.pbs_blup_chr-1_lod.drop-1.5_snpsqc_dis_no-x_updated_5.batches.csv
    Untracked:  data/ici.vs.pbs_blup_chr-1_lod.drop-1.5_snpsqc_dis_no-x_updated_5.batches_mis.csv
    Untracked:  data/ici.vs.pbs_blup_chr-2_lod.drop-1.5_5.batches.csv
    Untracked:  data/ici.vs.pbs_blup_chr-2_lod.drop-1.5_snpsqc_5.batches.csv
    Untracked:  data/ici.vs.pbs_blup_chr-2_lod.drop-1.5_snpsqc_5.batches_mis.csv
    Untracked:  data/ici.vs.pbs_blup_chr-2_lod.drop-1.5_snpsqc_dis_no-x_updated_5.batches.csv
    Untracked:  data/ici.vs.pbs_blup_chr-2_lod.drop-1.5_snpsqc_dis_no-x_updated_5.batches_mis.csv
    Untracked:  data/ici.vs.pbs_blup_chr-3_lod.drop-1.5_5.batches.csv
    Untracked:  data/ici.vs.pbs_blup_chr-3_lod.drop-1.5_snpsqc_5.batches.csv
    Untracked:  data/ici.vs.pbs_blup_chr-3_lod.drop-1.5_snpsqc_5.batches_mis.csv
    Untracked:  data/ici.vs.pbs_blup_chr-3_lod.drop-1.5_snpsqc_dis_no-x_updated_5.batches.csv
    Untracked:  data/ici.vs.pbs_blup_chr-3_lod.drop-1.5_snpsqc_dis_no-x_updated_5.batches_mis.csv
    Untracked:  data/ici.vs.pbs_blup_chr-4_lod.drop-1.5_5.batches.csv
    Untracked:  data/ici.vs.pbs_blup_chr-4_lod.drop-1.5_snpsqc_5.batches.csv
    Untracked:  data/ici.vs.pbs_blup_chr-4_lod.drop-1.5_snpsqc_5.batches_mis.csv
    Untracked:  data/ici.vs.pbs_blup_chr-4_lod.drop-1.5_snpsqc_dis_no-x_updated_5.batches.csv
    Untracked:  data/ici.vs.pbs_blup_chr-4_lod.drop-1.5_snpsqc_dis_no-x_updated_5.batches_mis.csv
    Untracked:  data/ici.vs.pbs_blup_chr-5_lod.drop-1.5_5.batches.csv
    Untracked:  data/ici.vs.pbs_blup_chr-5_lod.drop-1.5_snpsqc_5.batches.csv
    Untracked:  data/ici.vs.pbs_blup_chr-5_lod.drop-1.5_snpsqc_5.batches_mis.csv
    Untracked:  data/ici.vs.pbs_blup_chr-5_lod.drop-1.5_snpsqc_dis_no-x_updated_5.batches.csv
    Untracked:  data/ici.vs.pbs_blup_chr-5_lod.drop-1.5_snpsqc_dis_no-x_updated_5.batches_mis.csv
    Untracked:  data/ici.vs.pbs_blup_chr-6_lod.drop-1.5_5.batches.csv
    Untracked:  data/ici.vs.pbs_blup_chr-6_lod.drop-1.5_snpsqc_5.batches.csv
    Untracked:  data/ici.vs.pbs_blup_chr-6_lod.drop-1.5_snpsqc_5.batches_mis.csv
    Untracked:  data/ici.vs.pbs_blup_chr-6_lod.drop-1.5_snpsqc_dis_no-x_updated_5.batches.csv
    Untracked:  data/ici.vs.pbs_blup_chr-6_lod.drop-1.5_snpsqc_dis_no-x_updated_5.batches_mis.csv
    Untracked:  data/ici.vs.pbs_blup_chr-7_lod.drop-1.5_5.batches.csv
    Untracked:  data/ici.vs.pbs_blup_chr-7_lod.drop-1.5_snpsqc_5.batches.csv
    Untracked:  data/ici.vs.pbs_blup_chr-7_lod.drop-1.5_snpsqc_5.batches_mis.csv
    Untracked:  data/ici.vs.pbs_blup_chr-7_lod.drop-1.5_snpsqc_dis_no-x_updated_5.batches.csv
    Untracked:  data/ici.vs.pbs_blup_chr-7_lod.drop-1.5_snpsqc_dis_no-x_updated_5.batches_mis.csv
    Untracked:  data/ici.vs.pbs_blup_chr-8_lod.drop-1.5_5.batches.csv
    Untracked:  data/ici.vs.pbs_blup_chr-8_lod.drop-1.5_snpsqc_5.batches.csv
    Untracked:  data/ici.vs.pbs_blup_chr-8_lod.drop-1.5_snpsqc_5.batches_mis.csv
    Untracked:  data/ici.vs.pbs_blup_chr-8_lod.drop-1.5_snpsqc_dis_no-x_updated_5.batches.csv
    Untracked:  data/ici.vs.pbs_blup_chr-8_lod.drop-1.5_snpsqc_dis_no-x_updated_5.batches_mis.csv
    Untracked:  data/ici.vs.pbs_blup_chr-9_lod.drop-1.5_5.batches.csv
    Untracked:  data/ici.vs.pbs_blup_chr-9_lod.drop-1.5_snpsqc_5.batches.csv
    Untracked:  data/ici.vs.pbs_blup_chr-9_lod.drop-1.5_snpsqc_5.batches_mis.csv
    Untracked:  data/ici.vs.pbs_blup_chr-9_lod.drop-1.5_snpsqc_dis_no-x_updated_5.batches.csv
    Untracked:  data/ici.vs.pbs_blup_chr-9_lod.drop-1.5_snpsqc_dis_no-x_updated_5.batches_mis.csv
    Untracked:  data/ici.vs.pbs_blup_chr-X_lod.drop-1.5_5.batches.csv
    Untracked:  data/ici.vs.pbs_blup_chr-X_lod.drop-1.5_snpsqc_5.batches.csv
    Untracked:  data/ici.vs.pbs_blup_chr-X_lod.drop-1.5_snpsqc_5.batches_mis.csv
    Untracked:  data/ici.vs.pbs_blup_chr-X_lod.drop-1.5_snpsqc_dis_no-x_updated_5.batches.csv
    Untracked:  data/ici.vs.pbs_blup_chr-X_lod.drop-1.5_snpsqc_dis_no-x_updated_5.batches_mis.csv
    Untracked:  data/ici.vs.pbs_blup_sub_chr-13_peak.marker-UNC22384241_lod.drop-1.5_snpsqc_snpsqc_5.batches_peaks.csv
    Untracked:  data/ici.vs.pbs_blup_sub_chr-18_peak.marker-UNCHS045343_lod.drop-1.5_snpsqc_5.batches_mis.csv
    Untracked:  data/ici.vs.pbs_blup_sub_chr-19_peak.marker-UNC29893228_lod.drop-1.5_snpsqc_snpsqc_5.batches_peaks.csv
    Untracked:  data/ici.vs.pbs_blup_sub_chr-19_peak.marker-UNC29990033_lod.drop-1.5_snpsqc_dis_no-x_updated_5.batches_peaks.csv
    Untracked:  data/ici.vs.pbs_blup_sub_chr-19_peak.marker-UNCHS047192_lod.drop-1.5_snpsqc_dis_no-x_updated_5.batches_mis_peaks.csv
    Untracked:  data/ici.vs.pbs_blup_sub_chr-8_peak.marker-UNCHS023353_lod.drop-1.5_snpsqc_dis_no-x_updated_5.batches_mis_peaks.csv
    Untracked:  data/ici.vs.pbs_blup_sub_chr-8_peak.marker-UNCHS023353_lod.drop-1.5_snpsqc_dis_no-x_updated_5.batches_peaks.csv
    Untracked:  data/ici.vs.pbs_blup_sub_chr-8_peak.marker-UNCHS023364_lod.drop-1.5_snpsqc_snpsqc_5.batches_peaks.csv
    Untracked:  data/ici.vs.pbs_fitqtl_peaks_additive_snpsqc_dis_no-x_updated_5.batches_mis_peaks.txt
    Untracked:  data/ici.vs.pbs_fitqtl_peaks_additive_snpsqc_dis_no-x_updated_5.batches_peaks.txt
    Untracked:  data/ici.vs.pbs_fitqtl_peaks_additive_snpsqc_snpsqc_5.batches_peaks.txt
    Untracked:  data/ici.vs.pbs_fitqtl_peaks_interacting_snpsqc_dis_no-x_updated_5.batches_mis_peaks.txt
    Untracked:  data/ici.vs.pbs_fitqtl_peaks_interacting_snpsqc_dis_no-x_updated_5.batches_peaks.txt
    Untracked:  data/ici.vs.pbs_fitqtl_peaks_interacting_snpsqc_snpsqc_5.batches_peaks.txt
    Untracked:  data/ici.vs.pbs_fitqtl_peaks_sex_additive_snpsqc_dis_no-x_updated_5.batches_mis_peaks.txt
    Untracked:  data/ici.vs.pbs_fitqtl_peaks_sex_additive_snpsqc_dis_no-x_updated_5.batches_peaks.txt
    Untracked:  data/ici.vs.pbs_fitqtl_peaks_sex_additive_snpsqc_snpsqc_5.batches_peaks.txt
    Untracked:  data/ici.vs.pbs_fitqtl_peaks_sex_interacting_snpsqc_dis_no-x_updated_5.batches_mis_peaks.txt
    Untracked:  data/ici.vs.pbs_fitqtl_peaks_sex_interacting_snpsqc_dis_no-x_updated_5.batches_peaks.txt
    Untracked:  data/ici.vs.pbs_fitqtl_peaks_sex_interacting_snpsqc_snpsqc_5.batches_peaks.txt
    Untracked:  data/ici.vs.pbs_genes_chr-13_peak.marker-UNC22384241_lod.drop-1.5_snpsqc_snpsqc_5.batches_peaks.csv
    Untracked:  data/ici.vs.pbs_genes_chr-18_peak.marker-UNCHS045343_lod.drop-1.5_snpsqc_5.batches_mis.csv
    Untracked:  data/ici.vs.pbs_genes_chr-19_peak.marker-UNC29893228_lod.drop-1.5_snpsqc_snpsqc_5.batches_peaks.csv
    Untracked:  data/ici.vs.pbs_genes_chr-19_peak.marker-UNC29990033_lod.drop-1.5_snpsqc_dis_no-x_updated_5.batches_peaks.csv
    Untracked:  data/ici.vs.pbs_genes_chr-19_peak.marker-UNCHS047192_lod.drop-1.5_snpsqc_dis_no-x_updated_5.batches_mis_peaks.csv
    Untracked:  data/ici.vs.pbs_genes_chr-8_peak.marker-UNCHS023353_lod.drop-1.5_snpsqc_dis_no-x_updated_5.batches_mis_peaks.csv
    Untracked:  data/ici.vs.pbs_genes_chr-8_peak.marker-UNCHS023353_lod.drop-1.5_snpsqc_dis_no-x_updated_5.batches_peaks.csv
    Untracked:  data/ici.vs.pbs_genes_chr-8_peak.marker-UNCHS023364_lod.drop-1.5_snpsqc_snpsqc_5.batches_peaks.csv
    Untracked:  data/ici.vs.pbs_gm_qtl_5.batches.csv
    Untracked:  data/ici.vs.pbs_gm_qtl_5.batches_mis.csv
    Untracked:  data/ici.vs.pbs_gm_qtl_snpsqc_5.batches.csv
    Untracked:  data/ici.vs.pbs_gm_qtl_snpsqc_5.batches_mis.csv
    Untracked:  data/ici.vs.pbs_gm_qtl_snpsqc_5.batches_peaks.csv
    Untracked:  data/ici.vs.pbs_gm_qtl_snpsqc_dis_no-x_updated_5.batches.csv
    Untracked:  data/ici.vs.pbs_gm_qtl_snpsqc_dis_no-x_updated_5.batches_mis.csv
    Untracked:  data/ici.vs.pbs_gm_qtl_snpsqc_dis_no-x_updated_5.batches_mis_peaks.csv
    Untracked:  data/ici.vs.pbs_gm_qtl_snpsqc_dis_no-x_updated_5.batches_peaks.csv
    Untracked:  data/ici.vs.pbs_marker.freq_low.geno.freq.removed_geno.ratio.csv
    Untracked:  data/ici.vs.pbs_marker.freq_low.geno.freq.removed_geno.ratio_5.batches.csv
    Untracked:  data/ici.vs.pbs_marker.freq_low.geno.freq.removed_geno.ratio_5.batches_mis.csv
    Untracked:  data/ici.vs.pbs_marker.freq_low.geno.freq.removed_sample.outliers.removed_geno.ratio.csv
    Untracked:  data/ici.vs.pbs_marker.freq_low.geno.freq.removed_sample.outliers.removed_geno.ratio_5.batches.csv
    Untracked:  data/ici.vs.pbs_marker.freq_low.geno.freq.removed_sample.outliers.removed_geno.ratio_5.batches_mis.csv
    Untracked:  data/ici.vs.pbs_marker.freq_low.probs.freq.removed_geno.ratio.csv
    Untracked:  data/ici.vs.pbs_marker.freq_low.probs.freq.removed_geno.ratio_5.batches.csv
    Untracked:  data/ici.vs.pbs_marker.freq_low.probs.freq.removed_geno.ratio_5.batches_mis.csv
    Untracked:  data/ici.vs.pbs_marker.freq_low.probs.freq.removed_sample.outliers.removed_geno.ratio.csv
    Untracked:  data/ici.vs.pbs_marker.freq_low.probs.freq.removed_sample.outliers.removed_geno.ratio_5.batches.csv
    Untracked:  data/ici.vs.pbs_marker.freq_low.probs.freq.removed_sample.outliers.removed_geno.ratio_5.batches_mis.csv
    Untracked:  data/ici.vs.pbs_rz.age-additive.covariates_blup_sub_chr-7_peak.marker-JAX00153527_lod.drop-1.5_5.batches.csv
    Untracked:  data/ici.vs.pbs_rz.age-additive.covariates_blup_sub_chr-7_peak.marker-JAX00153527_lod.drop-1.5_snpsqc_5.batches.csv
    Untracked:  data/ici.vs.pbs_rz.age-additive.covariates_blup_sub_chr-7_peak.marker-UNCHS020443_lod.drop-1.5_snpsqc_dis_no-x_updated_5.batches.csv
    Untracked:  data/ici.vs.pbs_rz.age-additive.covariates_blup_sub_chr-7_peak.marker-UNCHS020443_lod.drop-1.5_snpsqc_dis_no-x_updated_5.batches_mis.csv
    Untracked:  data/ici.vs.pbs_rz.age-additive.covariates_blup_sub_chr-7_peak.marker-UNCHS020486_lod.drop-1.5_snpsqc_5.batches_mis.csv
    Untracked:  data/ici.vs.pbs_rz.age-additive.covariates_genes_chr-7_peak.marker-JAX00153527_lod.drop-1.5_5.batches.csv
    Untracked:  data/ici.vs.pbs_rz.age-additive.covariates_genes_chr-7_peak.marker-JAX00153527_lod.drop-1.5_snpsqc_5.batches.csv
    Untracked:  data/ici.vs.pbs_rz.age-additive.covariates_genes_chr-7_peak.marker-UNCHS020443_lod.drop-1.5_snpsqc_dis_no-x_updated_5.batches.csv
    Untracked:  data/ici.vs.pbs_rz.age-additive.covariates_genes_chr-7_peak.marker-UNCHS020443_lod.drop-1.5_snpsqc_dis_no-x_updated_5.batches_mis.csv
    Untracked:  data/ici.vs.pbs_rz.age-additive.covariates_genes_chr-7_peak.marker-UNCHS020486_lod.drop-1.5_snpsqc_5.batches_mis.csv
    Untracked:  data/ici.vs.pbs_sample.genos_marker.freq_low.geno.freq.removed.csv
    Untracked:  data/ici.vs.pbs_sample.genos_marker.freq_low.geno.freq.removed_sample.outliers.removed.csv
    Untracked:  data/ici.vs.pbs_sample.genos_marker.freq_low.probs.freq.removed.csv
    Untracked:  data/ici.vs.pbs_sample.genos_marker.freq_low.probs.freq.removed_sample.outliers.removed.csv
    Untracked:  data/ici.vs.pbs_scanone_5.batches.Rdata
    Untracked:  data/ici.vs.pbs_scanone_5.batches_mis.Rdata
    Untracked:  data/ici.vs.pbs_scanone_snpsqc_5.batches.Rdata
    Untracked:  data/ici.vs.pbs_scanone_snpsqc_5.batches_mis.Rdata
    Untracked:  data/ici.vs.pbs_scanone_snpsqc_dis_no-x_updated_5.batches.Rdata
    Untracked:  data/ici.vs.pbs_scanone_snpsqc_dis_no-x_updated_5.batches_mis.Rdata
    Untracked:  data/percent_missing_id_3.batches.RData
    Untracked:  data/percent_missing_id_4.batches.RData
    Untracked:  data/percent_missing_id_4.batches_bc.RData
    Untracked:  data/percent_missing_id_5.batches.RData
    Untracked:  data/percent_missing_marker_4.batches.RData
    Untracked:  data/percent_missing_marker_5.batches.RData
    Untracked:  data/pheno.csv
    Untracked:  data/pheno_BC312.csv
    Untracked:  data/physical_map.csv
    Untracked:  data/physical_map_BC312.csv
    Untracked:  data/qc_info_bad_sample_3.batches.RData
    Untracked:  data/qc_info_bad_sample_4.batches.RData
    Untracked:  data/qc_info_bad_sample_4.batches_bc.RData
    Untracked:  data/qc_info_bad_sample_5.batches.RData
    Untracked:  data/remaining.markers_geno.freq.xlsx
    Untracked:  data/sample_geno.csv
    Untracked:  data/sample_geno_AHB_BC312.csv
    Untracked:  data/sample_geno_bc.csv
    Untracked:  data/sample_geno_bc_BC312.csv
    Untracked:  data/serreze_probs.rds
    Untracked:  data/serreze_probs_BC312.rds
    Untracked:  data/serreze_probs_allqc.rds
    Untracked:  data/serreze_probs_allqc_5.batches.rds
    Untracked:  data/serreze_probs_allqc_5.batches_mis.rds
    Untracked:  data/summary.cg_3.batches.RData
    Untracked:  data/summary.cg_4.batches.RData
    Untracked:  data/summary.cg_4.batches_bc.RData
    Untracked:  data/summary.cg_5.batches.RData
    Untracked:  output/Percent_missing_genotype_data_5.batches.pdf
    Untracked:  output/Percent_missing_genotype_data_per_marker_5.batches.pdf
    Untracked:  output/Proportion_matching_genotypes_before_removal_of_bad_samples_5.batches.pdf
    Untracked:  output/genotype_error_marker_5.batches.pdf
    Untracked:  output/genotype_frequency_marker_5.batches.pdf

Unstaged changes:
    Modified:   analysis/4.1.1_qtl.analysis_binary_ici.vs.eoi_snpsqc_dis_no-x_updated.Rmd
    Modified:   analysis/4.1.1_qtl.analysis_binary_ici.vs.pbs_snpsqc_dis_no-x_updated.Rmd
    Modified:   analysis/4.1.2_qtl.analysis_cont_age_ici-early.vs.pbs_pheno.corrected.cleaned_5.batches.Rmd
    Modified:   analysis/4.1.2_qtl.analysis_cont_age_ici-early.vs.pbs_pheno.corrected.cleaned_5.batches_mis.Rmd
    Modified:   analysis/4.1.2_qtl.analysis_cont_age_ici-early.vs.pbs_snpsqc_pheno.corrected.cleaned_5.batches.Rmd
    Modified:   analysis/4.1.2_qtl.analysis_cont_age_ici-early.vs.pbs_snpsqc_pheno.corrected.cleaned_5.batches_mis.Rmd
    Modified:   analysis/4.1.2_qtl.analysis_cont_age_ici-early.vs.pbs_snpsqc_pheno.corrected.cleaned_dis_no-xk_5.batches.Rmd
    Modified:   analysis/4.1.2_qtl.analysis_cont_age_ici-early.vs.pbs_snpsqc_pheno.corrected.cleaned_dis_no-xk_5.batches_mis.Rmd
    Modified:   analysis/4.1.2_qtl.analysis_cont_age_ici.vs.eoi_pheno.corrected.cleaned_5.batches.Rmd
    Modified:   analysis/4.1.2_qtl.analysis_cont_age_ici.vs.eoi_pheno.corrected.cleaned_5.batches_mis.Rmd
    Modified:   analysis/4.1.2_qtl.analysis_cont_age_ici.vs.eoi_snpsqc_pheno.corrected.cleaned_5.batches.Rmd
    Modified:   analysis/4.1.2_qtl.analysis_cont_age_ici.vs.eoi_snpsqc_pheno.corrected.cleaned_5.batches_mis.Rmd
    Modified:   analysis/4.1.2_qtl.analysis_cont_age_ici.vs.eoi_snpsqc_pheno.corrected.cleaned_dis_no-xk_5.batches.Rmd
    Modified:   analysis/4.1.2_qtl.analysis_cont_age_ici.vs.eoi_snpsqc_pheno.corrected.cleaned_dis_no-xk_5.batches_mis.Rmd
    Modified:   analysis/4.1.2_qtl.analysis_cont_age_ici.vs.pbs_pheno.corrected.cleaned_5.batches.Rmd
    Modified:   analysis/4.1.2_qtl.analysis_cont_age_ici.vs.pbs_pheno.corrected.cleaned_5.batches_mis.Rmd
    Modified:   analysis/4.1.2_qtl.analysis_cont_age_ici.vs.pbs_snpsqc_pheno.corrected.cleaned_5.batches.Rmd
    Modified:   analysis/4.1.2_qtl.analysis_cont_age_ici.vs.pbs_snpsqc_pheno.corrected.cleaned_5.batches_mis.Rmd
    Modified:   analysis/4.1.2_qtl.analysis_cont_age_ici.vs.pbs_snpsqc_pheno.corrected.cleaned_dis_no-xk_5.batches.Rmd
    Modified:   analysis/4.1.2_qtl.analysis_cont_age_ici.vs.pbs_snpsqc_pheno.corrected.cleaned_dis_no-xk_5.batches_mis.Rmd
    Modified:   analysis/4.1.2_qtl.analysis_cont_rz.age_ici-early.vs.pbs_pheno.corrected.cleaned_5.batches.Rmd
    Modified:   analysis/4.1.2_qtl.analysis_cont_rz.age_ici-early.vs.pbs_pheno.corrected.cleaned_5.batches_mis.Rmd
    Modified:   analysis/4.1.2_qtl.analysis_cont_rz.age_ici-early.vs.pbs_snpsqc_pheno.corrected.cleaned_5.batches.Rmd
    Modified:   analysis/4.1.2_qtl.analysis_cont_rz.age_ici-early.vs.pbs_snpsqc_pheno.corrected.cleaned_5.batches_mis.Rmd
    Modified:   analysis/4.1.2_qtl.analysis_cont_rz.age_ici-early.vs.pbs_snpsqc_pheno.corrected.cleaned_dis_no-xk_5.batches.Rmd
    Modified:   analysis/4.1.2_qtl.analysis_cont_rz.age_ici-early.vs.pbs_snpsqc_pheno.corrected.cleaned_dis_no-xk_5.batches_mis.Rmd
    Modified:   analysis/4.1.2_qtl.analysis_cont_rz.age_ici.vs.eoi_pheno.corrected.cleaned_5.batches.Rmd
    Modified:   analysis/4.1.2_qtl.analysis_cont_rz.age_ici.vs.eoi_pheno.corrected.cleaned_5.batches_mis.Rmd
    Modified:   analysis/4.1.2_qtl.analysis_cont_rz.age_ici.vs.eoi_snpsqc_pheno.corrected.cleaned_5.batches.Rmd
    Modified:   analysis/4.1.2_qtl.analysis_cont_rz.age_ici.vs.eoi_snpsqc_pheno.corrected.cleaned_5.batches_mis.Rmd
    Modified:   analysis/4.1.2_qtl.analysis_cont_rz.age_ici.vs.eoi_snpsqc_pheno.corrected.cleaned_dis_no-xk_5.batches.Rmd
    Modified:   analysis/4.1.2_qtl.analysis_cont_rz.age_ici.vs.eoi_snpsqc_pheno.corrected.cleaned_dis_no-xk_5.batches_mis.Rmd
    Modified:   analysis/4.1.2_qtl.analysis_cont_rz.age_ici.vs.pbs_pheno.corrected.cleaned_5.batches.Rmd
    Modified:   analysis/4.1.2_qtl.analysis_cont_rz.age_ici.vs.pbs_pheno.corrected.cleaned_5.batches_mis.Rmd
    Modified:   analysis/4.1.2_qtl.analysis_cont_rz.age_ici.vs.pbs_snpsqc_pheno.corrected.cleaned_5.batches.Rmd
    Modified:   analysis/4.1.2_qtl.analysis_cont_rz.age_ici.vs.pbs_snpsqc_pheno.corrected.cleaned_5.batches_mis.Rmd
    Modified:   analysis/4.1.2_qtl.analysis_cont_rz.age_ici.vs.pbs_snpsqc_pheno.corrected.cleaned_dis_no-xk_5.batches.Rmd
    Modified:   analysis/4.1.2_qtl.analysis_cont_rz.age_ici.vs.pbs_snpsqc_pheno.corrected.cleaned_dis_no-xk_5.batches_mis.Rmd
    Modified:   analysis/index_5.batches.Rmd
    Modified:   analysis/index_5.batches_additional.Rmd

</code></pre>
<p>
Note that any generated files, e.g. HTML, png, CSS, etc., are not included in this status report because it is ok for generated content to have uncommitted changes.
</p>
</div>
</div>
</div>
</div>
<hr>
</div>
<div id="versions" class="tab-pane fade">
<p>
There are no past versions. Publish this analysis with <code>wflow_publish()</code> to start tracking its development.
</p>
<hr>
</div>
</div>
</div>
<div id="loading-data" class="section level1">
<h1>Loading Data</h1>
<pre class="r"><code>load(&quot;data/gm_allqc_5.batches.RData&quot;)

#gm_allqc
gm=gm_allqc
gm</code></pre>
<pre><code>Object of class cross2 (crosstype &quot;bc&quot;)

Total individuals              308
No. genotyped individuals      308
No. phenotyped individuals     308
No. with both geno &amp; pheno     308

No. phenotypes                   1
No. covariates                   6
No. phenotype covariates         0

No. chromosomes                 20
Total markers                34537

No. markers by chr:
   1    2    3    4    5    6    7    8    9   10   11   12   13   14   15   16 
2643 2629 1857 1890 1774 1941 1672 1627 1878 1176 1871 1300 1549 1578 1257  935 
  17   18   19    X 
 501  913 1014 4532 </code></pre>
<pre class="r"><code>#pr &lt;- readRDS(&quot;data/serreze_probs_allqc.rds&quot;)
#pr &lt;- readRDS(&quot;data/serreze_probs.rds&quot;)

##extracting animals with ici and eoi group status
miceinfo &lt;- gm$covar[gm$covar$group == &quot;EOI&quot; | gm$covar$group == &quot;ICI&quot;,]
table(miceinfo$group)</code></pre>
<pre><code>
EOI ICI 
164 104 </code></pre>
<pre class="r"><code>mice.ids &lt;- rownames(miceinfo)

gm &lt;- gm[mice.ids]
gm</code></pre>
<pre><code>Object of class cross2 (crosstype &quot;bc&quot;)

Total individuals              268
No. genotyped individuals      268
No. phenotyped individuals     268
No. with both geno &amp; pheno     268

No. phenotypes                   1
No. covariates                   6
No. phenotype covariates         0

No. chromosomes                 20
Total markers                34537

No. markers by chr:
   1    2    3    4    5    6    7    8    9   10   11   12   13   14   15   16 
2643 2629 1857 1890 1774 1941 1672 1627 1878 1176 1871 1300 1549 1578 1257  935 
  17   18   19    X 
 501  913 1014 4532 </code></pre>
<pre class="r"><code>table(gm$covar$group)</code></pre>
<pre><code>
EOI ICI 
164 104 </code></pre>
<pre class="r"><code>gm.full &lt;- gm

covars &lt;- read_csv(&quot;data/covar_corrected.cleaned_ici.vs.eoi_5.batches_0.csv&quot;)
#removing any missing info
missing_ids &lt;- covars[which(covars$&quot;age.of.onset&quot;==&quot;&quot;),]$Mouse.ID
length(missing_ids)</code></pre>
<pre><code>[1] 2</code></pre>
<pre class="r"><code>missing_ids</code></pre>
<pre><code>[1] &quot;NG00526-ICI-LATE&quot; &quot;NG00522-ICI-LATE&quot;</code></pre>
<pre class="r"><code>if(length(missing_ids) != 0){
gm &lt;- gm[paste0(&quot;-&quot;,as.character(missing_ids)),]
gm
table(gm$covar$group)

#keeping only mouseids in covars
covars &lt;- covars[gm$covar$id,]
table(covars$group)
}</code></pre>
<pre><code>
EOI ICI 
164 102 </code></pre>
<pre class="r"><code>#covars[covars$age.of.onset==0,]$age.of.onset &lt;- &#39;&#39;

##removing problmetic marker
gm &lt;- drop_markers(gm, &quot;UNCHS013106&quot;)

query_variants &lt;- create_variant_query_func(&quot;/Users/corneb/Documents/MyJax/CS/Projects/support.files/qtl2/cc_variants.sqlite&quot;)
query_genes &lt;- create_gene_query_func(&quot;/Users/corneb/Documents/MyJax/CS/Projects/support.files/qtl2/mouse_genes_mgi.sqlite&quot;)


##dropping monomorphic markers within the dataset

g &lt;- do.call(&quot;cbind&quot;, gm$geno)
#g[1,1:ncol(g)]

gf_mar &lt;- t(apply(g, 2, function(a) table(factor(a, 1:2))/sum(a != 0)))
#gn_mar &lt;- t(apply(g, 2, function(a) table(factor(a, 1:2))))

gf_mar &lt;- gf_mar[gf_mar[,2] != &quot;NaN&quot;,]

count &lt;- rowSums(gf_mar &lt;=0.05)
low_freq_df &lt;- merge(as.data.frame(gf_mar),as.data.frame(count), by=&quot;row.names&quot;,all=T)
low_freq_df[is.na(low_freq_df)] &lt;- &#39;&#39;
low_freq_df &lt;- low_freq_df[low_freq_df$count == 1,]
rownames(low_freq_df) &lt;- low_freq_df$Row.names

low_freq &lt;- find_markerpos(gm, rownames(low_freq_df))
low_freq$id &lt;- rownames(low_freq)

nrow(low_freq)</code></pre>
<pre><code>[1] 8398</code></pre>
<pre class="r"><code>low_freq_bad &lt;- merge(low_freq,low_freq_df, by=&quot;row.names&quot;,all=T)
names(low_freq_bad)[1] &lt;- c(&quot;marker&quot;)

gf_mar &lt;- gf_mar[gf_mar[,2] != &quot;NaN&quot;,]
MAF &lt;- apply(gf_mar, 1, function(x) min(x))
MAF &lt;- as.data.frame(MAF)
MAF$index &lt;- 1:nrow(gf_mar)
gf_mar_maf &lt;- merge(gf_mar,as.data.frame(MAF), by=&quot;row.names&quot;)
gf_mar_maf &lt;- gf_mar_maf[order(gf_mar_maf$index),]

gfmar &lt;- NULL
gfmar$gfmar_mar_0 &lt;- sum(gf_mar_maf$MAF==0)
gfmar$gfmar_mar_1 &lt;- sum(gf_mar_maf$MAF&lt; 0.01)
gfmar$gfmar_mar_5 &lt;- sum(gf_mar_maf$MAF&lt; 0.05)
gfmar$gfmar_mar_10 &lt;- sum(gf_mar_maf$MAF&lt; 0.10)
gfmar$gfmar_mar_15 &lt;- sum(gf_mar_maf$MAF&lt; 0.15)
gfmar$gfmar_mar_25 &lt;- sum(gf_mar_maf$MAF&lt; 0.25)
gfmar$gfmar_mar_50 &lt;- sum(gf_mar_maf$MAF&lt; 0.50)
gfmar$total_snps &lt;- nrow(as.data.frame(gf_mar_maf))

gfmar &lt;- t(as.data.frame(gfmar))
gfmar &lt;- as.data.frame(gfmar)
gfmar$count &lt;- gfmar$V1

gfmar[c(2)] %&gt;%
  kable(escape = F,align = c(&quot;ccccccccc&quot;),linesep =&quot;\\hline&quot;) %&gt;%
  kable_styling(full_width = F) %&gt;%
  kable_styling(&quot;striped&quot;, full_width = F)  %&gt;%
  row_spec(8 ,bold=T,color= &quot;white&quot;,background = &quot;black&quot;)</code></pre>
<table class="table table table-striped" style="width: auto !important; margin-left: auto; margin-right: auto; width: auto !important; margin-left: auto; margin-right: auto;">
<thead>
<tr>
<th style="text-align:left;">
</th>
<th style="text-align:center;">
count
</th>
</tr>
</thead>
<tbody>
<tr>
<td style="text-align:left;">
gfmar_mar_0
</td>
<td style="text-align:center;">
3816
</td>
</tr>
<tr>
<td style="text-align:left;">
gfmar_mar_1
</td>
<td style="text-align:center;">
4125
</td>
</tr>
<tr>
<td style="text-align:left;">
gfmar_mar_5
</td>
<td style="text-align:center;">
8397
</td>
</tr>
<tr>
<td style="text-align:left;">
gfmar_mar_10
</td>
<td style="text-align:center;">
8545
</td>
</tr>
<tr>
<td style="text-align:left;">
gfmar_mar_15
</td>
<td style="text-align:center;">
8618
</td>
</tr>
<tr>
<td style="text-align:left;">
gfmar_mar_25
</td>
<td style="text-align:center;">
9026
</td>
</tr>
<tr>
<td style="text-align:left;">
gfmar_mar_50
</td>
<td style="text-align:center;">
33434
</td>
</tr>
<tr>
<td style="text-align:left;font-weight: bold;color: white !important;background-color: black !important;">
total_snps
</td>
<td style="text-align:center;font-weight: bold;color: white !important;background-color: black !important;">
34537
</td>
</tr>
</tbody>
</table>
<pre class="r"><code>gm_qc &lt;- drop_markers(gm, low_freq_bad$marker)
gm_qc &lt;- drop_nullmarkers(gm_qc)
 
gm = gm_qc
gm</code></pre>
<pre><code>Object of class cross2 (crosstype &quot;bc&quot;)

Total individuals              266
No. genotyped individuals      266
No. phenotyped individuals     266
No. with both geno &amp; pheno     266

No. phenotypes                   1
No. covariates                   6
No. phenotype covariates         0

No. chromosomes                 20
Total markers                26139

No. markers by chr:
   1    2    3    4    5    6    7    8    9   10   11   12   13   14   15   16 
2336 2342 1589 1661 1535 1681 1450 1438 1654  947 1651 1107 1359 1396  945  728 
  17   18   19    X 
 303  747  903  367 </code></pre>
<pre class="r"><code>markers &lt;- marker_names(gm)
gmapdf &lt;- read.csv(&quot;/Users/corneb/Documents/MyJax/CS/Projects/Serreze/haplotype.reconstruction/output_5.batches/genetic_map.csv&quot;)
pmapdf &lt;- read.csv(&quot;/Users/corneb/Documents/MyJax/CS/Projects/Serreze/haplotype.reconstruction/output_5.batches/physical_map.csv&quot;)
#mapdf &lt;- merge(gmapdf,pmapdf, by=c(&quot;marker&quot;,&quot;chr&quot;), all=T)
#rownames(mapdf) &lt;- mapdf$marker
#mapdf &lt;- mapdf[markers,]
#names(mapdf) &lt;- c(&#39;marker&#39;,&#39;chr&#39;,&#39;gmapdf&#39;,&#39;pmapdf&#39;)
#mapdfnd &lt;- mapdf[!duplicated(mapdf[c(2:3)]),]

pr.qc &lt;- calc_genoprob(gm)</code></pre>
</div>
<div id="preparing-necessary-info" class="section level1">
<h1>Preparing necessary info</h1>
<pre class="r"><code>covars$age.of.onset &lt;- as.numeric(covars$age.of.onset)
Xcovar &lt;- get_x_covar(gm)
addcovar = model.matrix(~ICI.vs.EOI, data = covars)[,-1]
#addcovar.i = model.matrix(~ICI.vs.EOI, data = covars)[,-1]

K &lt;- calc_kinship(pr.qc, type = &quot;loco&quot;)
#heatmap(K[[1]])
K.overall &lt;- calc_kinship(pr.qc, type = &quot;overall&quot;)
#heatmap(K.overall)
kinship &lt;- calc_kinship(pr.qc)
heatmap(kinship)</code></pre>
<p><img src="figure/4.1.2_qtl.analysis_cont_age_ici.vs.eoi_snpsqc_pheno.corrected.cleaned_5.batches_0.Rmd/permutation%20prelim-1.png" width="912" style="display: block; margin: auto;" /></p>
<div id="no-additive-covariates" class="section level2">
<h2>No additive covariates</h2>
<div id="significance-thresholds" class="section level3">
<h3>Significance thresholds</h3>
<pre class="r"><code>operm &lt;- scan1perm(pr.qc, covars[&quot;age.of.onset&quot;], model=&quot;normal&quot;, n_perm=10, perm_Xsp=TRUE, chr_lengths=chr_lengths(gm$gmap))

summary_table&lt;-data.frame(unclass(summary(operm, alpha=c(0.01,  0.05, 0.1))))
names(summary_table) &lt;- c(&quot;autosomes&quot;,&quot;X&quot;)
summary_table$significance.level &lt;- rownames(summary_table)

rownames(summary_table) &lt;- NULL

summary_table[c(3,1:2)] %&gt;%
  kable(escape = F,align = c(&quot;ccc&quot;)) %&gt;%
  kable_styling(&quot;striped&quot;, full_width = T) %&gt;%
  column_spec(1, bold=TRUE)</code></pre>
<table class="table table-striped" style="margin-left: auto; margin-right: auto;">
<thead>
<tr>
<th style="text-align:center;">
significance.level
</th>
<th style="text-align:center;">
autosomes
</th>
<th style="text-align:center;">
X
</th>
</tr>
</thead>
<tbody>
<tr>
<td style="text-align:center;font-weight: bold;">
0.01
</td>
<td style="text-align:center;">
2.950006
</td>
<td style="text-align:center;">
2.504684
</td>
</tr>
<tr>
<td style="text-align:center;font-weight: bold;">
0.05
</td>
<td style="text-align:center;">
2.771835
</td>
<td style="text-align:center;">
2.378893
</td>
</tr>
<tr>
<td style="text-align:center;font-weight: bold;">
0.1
</td>
<td style="text-align:center;">
2.548565
</td>
<td style="text-align:center;">
2.214400
</td>
</tr>
</tbody>
</table>
</div>
<div id="genome-wide-scan" class="section level3">
<h3>Genome-wide scan</h3>
<pre class="r"><code>plot_lod&lt;-function(out,map){
  for (i in 1:dim(out)[2]){
#par(mar=c(5.1, 6.1, 1.1, 1.1))
    ymx &lt;- maxlod(out) # overall maximum LOD score
    plot(out, map, lodcolumn=i, col=&quot;slateblue&quot;, ylim=c(0, ymx+0.5))
    #legend(&quot;topright&quot;, lwd=2, colnames(out)[i], bg=&quot;gray90&quot;)
    title(main = paste0(colnames(out)[i], &quot; - ICI vs. EOI (no covariates) [positions in cM]&quot;))
    add_threshold(map,  summary(operm,alpha=0.1), col = &#39;purple&#39;)
    add_threshold(map,  summary(operm, alpha=0.05), col = &#39;red&#39;)
    add_threshold(map,  summary(operm, alpha=0.01), col = &#39;blue&#39;)

  }
}


print(&quot;with no kinship&quot;)</code></pre>
<p>[1] “with no kinship”</p>
<pre class="r"><code>outf &lt;- scan1(pr.qc, covars[&quot;age.of.onset&quot;], Xcovar=Xcovar, model=&quot;normal&quot;)

plot_lod(outf,gm$gmap)</code></pre>
<p><img src="figure/4.1.2_qtl.analysis_cont_age_ici.vs.eoi_snpsqc_pheno.corrected.cleaned_5.batches_0.Rmd/scanone-1.png" width="912" style="display: block; margin: auto;" /></p>
<pre class="r"><code>print(&quot;with normal kinship&quot;)</code></pre>
<p>[1] “with normal kinship”</p>
<pre class="r"><code>outk &lt;- scan1(pr.qc, covars[&quot;age.of.onset&quot;], Xcovar=Xcovar, model=&quot;normal&quot;, kinship=kinship)

plot_lod(outk,gm$gmap)</code></pre>
<p><img src="figure/4.1.2_qtl.analysis_cont_age_ici.vs.eoi_snpsqc_pheno.corrected.cleaned_5.batches_0.Rmd/scanone-2.png" width="912" style="display: block; margin: auto;" /></p>
<pre class="r"><code>print(&quot;with loco kinship&quot;)</code></pre>
<p>[1] “with loco kinship”</p>
<pre class="r"><code>out &lt;- scan1(pr.qc, covars[&quot;age.of.onset&quot;], Xcovar=Xcovar, model=&quot;normal&quot;, kinship=K)

plot_lod(out,gm$gmap)</code></pre>
<p><img src="figure/4.1.2_qtl.analysis_cont_age_ici.vs.eoi_snpsqc_pheno.corrected.cleaned_5.batches_0.Rmd/scanone-3.png" width="912" style="display: block; margin: auto;" /></p>
<pre class="r"><code>print(paste0(&quot;number of samples in analysis = &quot;, as.data.frame(attributes(out)$sample_size)[1,]))</code></pre>
<p>[1] “number of samples in analysis = 266”</p>
</div>
<div id="lod-peaks" class="section level3">
<h3>LOD peaks</h3>
<div id="centimorgan-cm" class="section level4">
<h4>Centimorgan (cM)</h4>
<pre class="r"><code>peaks&lt;-find_peaks(out, gm$gmap, threshold=summary(operm,alpha=0.05)$A, thresholdX = summary(operm,alpha=0.05)$X, peakdrop=3, drop=1.5)

if(nrow(peaks) &gt; 0){
peaks$marker &lt;- find_marker(gm$gmap, chr=peaks$chr,pos=peaks$pos)
names(peaks)[2] &lt;- c(&quot;phenotype&quot;)
peaks &lt;- peaks[-1]

rownames(peaks) &lt;- NULL
print(kable(peaks, escape = F, align = c(&quot;cccccccc&quot;), &quot;html&quot;) 
  %&gt;% kable_styling(&quot;striped&quot;, full_width = T)%&gt;%
  column_spec(1, bold=TRUE)
  )

#plot only peak chromosomes

plot_lod_chr&lt;-function(out,map,chrom){
  for (i in 1:dim(out)[2]){
    #png(filename=paste0(&quot;/Users/chenm/Documents/qtl/Jai/&quot;,colnames(out)[i],  &quot;_lod.png&quot;))
    
    #par(mar=c(5.1, 6.1, 1.1, 1.1))
    ymx &lt;- maxlod(out) # overall maximum LOD score
    plot(out, map, chr = chrom, lodcolumn=i, col=&quot;slateblue&quot;, ylim=c(0, ymx+0.5))
    #legend(&quot;topright&quot;, lwd=2, colnames(out)[i], bg=&quot;gray90&quot;)
    title(main = paste0(colnames(out)[i], &quot; - chr&quot;, chrom, &quot; - ICI vs. EOI (no covariates) [positions in cM]&quot;))
    add_threshold(map,  summary(operm,alpha=0.1), col = &#39;purple&#39;)
    add_threshold(map,  summary(operm, alpha=0.05), col = &#39;red&#39;)
    add_threshold(map,  summary(operm, alpha=0.01), col = &#39;blue&#39;)
    #for (j in 1: dim(summary_table)[1]){
    #  abline(h=summary_table[j, i],col=&quot;red&quot;)
    #  text(x=400, y =summary_table[j, i]+0.12, labels = paste(&quot;p=&quot;, row.names(summary_table)[j]))
    #}
    #dev.off()

  }
}


} else {
  print(paste0(&quot;There are no peaks that have a LOD that reaches suggestive (p&lt;0.05) level of &quot;,summary(operm,alpha=0.05)$A, &quot; [autosomes]/&quot;,summary(operm,alpha=0.05)$X, &quot; [x-chromosome]&quot;))
}</code></pre>
<table class="table table-striped" style="margin-left: auto; margin-right: auto;">
<thead>
<tr>
<th style="text-align:center;">
phenotype
</th>
<th style="text-align:center;">
chr
</th>
<th style="text-align:center;">
pos
</th>
<th style="text-align:center;">
lod
</th>
<th style="text-align:center;">
ci_lo
</th>
<th style="text-align:center;">
ci_hi
</th>
<th style="text-align:center;">
marker
</th>
</tr>
</thead>
<tbody>
<tr>
<td style="text-align:center;font-weight: bold;">
age.of.onset
</td>
<td style="text-align:center;">
3
</td>
<td style="text-align:center;">
18.749
</td>
<td style="text-align:center;">
8.846727
</td>
<td style="text-align:center;">
16.581
</td>
<td style="text-align:center;">
25.884
</td>
<td style="text-align:center;">
UNCHS008487
</td>
</tr>
<tr>
<td style="text-align:center;font-weight: bold;">
age.of.onset
</td>
<td style="text-align:center;">
10
</td>
<td style="text-align:center;">
43.651
</td>
<td style="text-align:center;">
3.862031
</td>
<td style="text-align:center;">
36.745
</td>
<td style="text-align:center;">
46.549
</td>
<td style="text-align:center;">
UNC18376338
</td>
</tr>
</tbody>
</table>
<pre class="r"><code>if(nrow(peaks) &lt; 50 &amp; nrow(peaks) &gt; 0){

for(i in unique(peaks$chr)){
#for (i in 1:nrow(peaks)){
  #plot_lod_chr(out,gm$gmap, peaks$chr[i])
  plot_lod_chr(out,gm$gmap, i)
}

} else {
  print(paste0(&quot;There are too many peaks (&quot;,nrow(peaks),&quot; peaks) that have a LOD that reaches suggestive (p&lt;0.05) level of &quot;,summary(operm,alpha=0.05)$A, &quot; [autosomes]/&quot;,summary(operm,alpha=0.05)$X, &quot; [x-chromosome]&quot;))
}</code></pre>
<p><img src="figure/4.1.2_qtl.analysis_cont_age_ici.vs.eoi_snpsqc_pheno.corrected.cleaned_5.batches_0.Rmd/LOD%20peaks-1.png" width="912" style="display: block; margin: auto;" /><img src="figure/4.1.2_qtl.analysis_cont_age_ici.vs.eoi_snpsqc_pheno.corrected.cleaned_5.batches_0.Rmd/LOD%20peaks-2.png" width="912" style="display: block; margin: auto;" /></p>
</div>
<div id="megabase-mb" class="section level4">
<h4>Megabase (MB)</h4>
<pre class="r"><code>peaks_mba&lt;-find_peaks(out, gm$pmap, threshold=summary(operm,alpha=0.05)$A, thresholdX = summary(operm,alpha=0.05)$X, peakdrop=3, drop=1.5)
if(nrow(peaks) &gt; 0){
peaks_mba$marker &lt;- find_marker(gm$pmap, chr=peaks_mba$chr,pos=peaks_mba$pos)
names(peaks_mba)[2] &lt;- c(&quot;phenotype&quot;)
peaks_mba &lt;- peaks_mba[-1]

rownames(peaks_mba) &lt;- NULL
print(kable(peaks_mba, escape = F, align = c(&quot;cccccccc&quot;), &quot;html&quot;) 
  %&gt;% kable_styling(&quot;striped&quot;, full_width = T)%&gt;%
  column_spec(1, bold=TRUE)
  )

  plot_lod_chr_mb&lt;-function(out,map,chrom){
  for (i in 1:dim(out)[2]){
    #png(filename=paste0(&quot;/Users/chenm/Documents/qtl/Jai/&quot;,colnames(out)[i],  &quot;_lod.png&quot;))
    
    #par(mar=c(5.1, 6.1, 1.1, 1.1))
    ymx &lt;- maxlod(out) # overall maximum LOD score
    plot(out, map, chr = chrom, lodcolumn=i, col=&quot;slateblue&quot;, ylim=c(0, ymx+0.5))
    #legend(&quot;topright&quot;, lwd=2, colnames(out)[i], bg=&quot;gray90&quot;)
    title(main = paste0(colnames(out)[i], &quot; - chr&quot;, chrom, &quot; - ICI vs. EOI (no covariates) [positions in cM]&quot;))
    add_threshold(map,  summary(operm,alpha=0.1), col = &#39;purple&#39;)
    add_threshold(map,  summary(operm, alpha=0.05), col = &#39;red&#39;)
    add_threshold(map,  summary(operm, alpha=0.01), col = &#39;blue&#39;)
    #for (j in 1: dim(summary_table)[1]){
    #  abline(h=summary_table[j, i],col=&quot;red&quot;)
    #  text(x=400, y =summary_table[j, i]+0.12, labels = paste(&quot;p=&quot;, row.names(summary_table)[j]))
    #}
    #dev.off()
  }
}

} else {
  print(paste0(&quot;There are no peaks that have a LOD that reaches suggestive (p&lt;0.05) level of &quot;,summary(operm,alpha=0.05)$A, &quot; [autosomes]/&quot;,summary(operm,alpha=0.05)$X, &quot; [x-chromosome]&quot;))
}</code></pre>
<table class="table table-striped" style="margin-left: auto; margin-right: auto;">
<thead>
<tr>
<th style="text-align:center;">
phenotype
</th>
<th style="text-align:center;">
chr
</th>
<th style="text-align:center;">
pos
</th>
<th style="text-align:center;">
lod
</th>
<th style="text-align:center;">
ci_lo
</th>
<th style="text-align:center;">
ci_hi
</th>
<th style="text-align:center;">
marker
</th>
</tr>
</thead>
<tbody>
<tr>
<td style="text-align:center;font-weight: bold;">
age.of.onset
</td>
<td style="text-align:center;">
3
</td>
<td style="text-align:center;">
38.95539
</td>
<td style="text-align:center;">
8.846727
</td>
<td style="text-align:center;">
34.11194
</td>
<td style="text-align:center;">
54.90525
</td>
<td style="text-align:center;">
UNCHS008487
</td>
</tr>
<tr>
<td style="text-align:center;font-weight: bold;">
age.of.onset
</td>
<td style="text-align:center;">
10
</td>
<td style="text-align:center;">
87.54037
</td>
<td style="text-align:center;">
3.862031
</td>
<td style="text-align:center;">
70.77844
</td>
<td style="text-align:center;">
91.91713
</td>
<td style="text-align:center;">
UNC18376338
</td>
</tr>
</tbody>
</table>
<pre class="r"><code>if(nrow(peaks_mba) &lt; 50 &amp; nrow(peaks_mba) &gt; 0){

for(i in unique(peaks_mba$chr)){
#for (i in 1:nrow(peaks_mba)){
  #plot_lod_chr_mb(out,gm$pmap, peaks_mba$chr[i])
  plot_lod_chr_mb(out,gm$pmap,i)
}

} else {
  print(paste0(&quot;There are too many peaks (&quot;,nrow(peaks_mba),&quot; peaks) that have a LOD that reaches suggestive (p&lt;0.05) level of &quot;,summary(operm,alpha=0.05)$A, &quot; [autosomes]/&quot;,summary(operm,alpha=0.05)$X, &quot; [x-chromosome]&quot;))
}</code></pre>
<p><img src="figure/4.1.2_qtl.analysis_cont_age_ici.vs.eoi_snpsqc_pheno.corrected.cleaned_5.batches_0.Rmd/LOD%20peaks_mba-1.png" width="912" style="display: block; margin: auto;" /><img src="figure/4.1.2_qtl.analysis_cont_age_ici.vs.eoi_snpsqc_pheno.corrected.cleaned_5.batches_0.Rmd/LOD%20peaks_mba-2.png" width="912" style="display: block; margin: auto;" /></p>
</div>
</div>
<div id="qtl-effects" class="section level3">
<h3>QTL effects</h3>
<p>For each peak LOD location we give a list of gene</p>
<pre class="r"><code>if(nrow(peaks) &lt; 50 &amp; nrow(peaks) &gt; 0){
for (i in 1:nrow(peaks)){
#for (i in 1:1){
  #Plot 1
  g &lt;- maxmarg(pr.qc, gm$gmap, chr=peaks$chr[i], pos=peaks$pos[i], return_char=TRUE)
  #png(filename=paste0(&quot;/Users/chenm/Documents/qtl/Jai/&quot;,&quot;qtl_effect_&quot;, i, &quot;.png&quot;))
  #par(mar=c(4.1, 4.1, 1.5, 0.6))
  plot_pxg(g, covars[,peaks$phenotype[i]], ylab=peaks$phenotype[i], sort=FALSE)
  title(main = paste0(&quot;chr: &quot;, chr=peaks$chr[i],&quot;; pos: &quot;, peaks$pos[i], &quot;cM /&quot;,peaks_mba$pos[i],&quot;MB\n(&quot;,peaks$phenotype[i],&quot; )&quot;), line=0.2)
  ##dev.off()

  chr = peaks$chr[i]

# Plot 2
  pr_sub &lt;- pull_genoprobint(pr.qc, gm$gmap, chr, c(peaks$ci_lo[i], peaks$ci_hi[i]))
  #coeff &lt;- scan1coef(pr[,chr], cross$pheno[,peaks$lodcolumn[i]], addcovar = addcovar)
  #coeff &lt;- scan1coef(pr[,chr], cross$pheno[,peaks$lodcolumn[i]], Xcovar=Xcovar)
  #coeff &lt;- scan1coef(pr.qc[,chr], gm$covar[peaks$lodcolumn[i]], model=&quot;binary&quot;)
  #coeff_sub &lt;- scan1coef(pr_sub[,chr], gm$covar[peaks$lodcolumn[i]], model=&quot;binary&quot;)
  blup &lt;- scan1blup(pr.qc[,chr], covars[peaks$phenotype[i]])
  blup_sub &lt;- scan1blup(pr_sub[,chr], covars[peaks$phenotype[i]])

  write.csv(as.data.frame(blup_sub), paste0(&quot;data/ici.vs.eoi_&quot;,peaks$phenotype[i],&quot;-no.covariates_blup_sub_chr-&quot;,chr,&quot;_peak.marker-&quot;,peaks$marker[i],&quot;_lod.drop-1.5_snpsqc_5.batches_0.csv&quot;), quote=F)

  #plot_coef(coeff, 
  #     gm$gmap, columns=1:2,
  #     bgcolor=&quot;gray95&quot;, legend=&quot;bottomleft&quot;, 
  #     main = paste(&quot;chr&quot;, chr=peaks$chr[i],&quot;; pos: &quot;, peaks$pos[i], &quot;cM /&quot;,peaks_mba$pos[i],&quot;MB\n(&quot;,peaks$lodcolumn[i],&quot; [scan1coeff; positions in cM] )&quot;)
  #     )

  #plot_coef(coeff_sub, 
  #     gm$gmap, columns=1:2,
  #     bgcolor=&quot;gray95&quot;, legend=&quot;bottomleft&quot;, 
  #     main = paste(&quot;chr&quot;, chr=peaks$chr[i],&quot;; pos: &quot;, peaks$pos[i], &quot;cM /&quot;,peaks_mba$pos[i],&quot;MB\n(&quot;,peaks$lodcolumn[i],&quot;; 1.5 LOD drop interval [scan1coeff; positions in cM] ) &quot;)
  #     )


  plot_coef(blup, 
       gm$gmap, columns=1:2,
       bgcolor=&quot;gray95&quot;, legend=&quot;bottomleft&quot;, 
       main = paste0(&quot;chr: &quot;, chr=peaks$chr[i],&quot;; pos: &quot;, peaks$pos[i], &quot;cM /&quot;,peaks_mba$pos[i],&quot;MB\n(&quot;,peaks$phenotype[i],&quot; [scan1blup; positions in cM] )&quot;)
       )

  plot_coef(blup_sub, 
       gm$gmap, columns=1:2,
       bgcolor=&quot;gray95&quot;, legend=&quot;bottomleft&quot;, 
       main = paste0(&quot;chr: &quot;, chr=peaks$chr[i],&quot;; pos: &quot;, peaks$pos[i], &quot;cM /&quot;,peaks_mba$pos[i],&quot;MB\n(&quot;,peaks$phenotype[i],&quot;; 1.5 LOD drop interval [scan1blup; positions in cM] )&quot;)
       )


 # Plot 3
  #c2effB &lt;- scan1coef(pr.qc[,chr], gm$covar[peaks$lodcolumn[i]], model=&quot;binary&quot;, contrasts=cbind(a=c(-1, 0), d=c(0, -1)))
  #c2effBb &lt;- scan1blup(pr.qc[,chr], gm$covar[peaks$lodcolumn[i]], contrasts=cbind(a=c(-1, 0), d=c(0, -1)))
  ##c2effB &lt;- scan1coef(pr[,chr], cross$pheno[,peaks$lodcolumn[i]], addcovar = addcovar, contrasts=cbind(mu=c(1,1,1), a=c(-1, 0, 1), d=c(0, 1, 0)))
  ##c2effB &lt;- scan1coef(pr[,chr], cross$pheno[,peaks$lodcolumn[i]],Xcovar=Xcovar, contrasts=cbind(mu=c(1,1,1), a=c(-1, 0, 1), d=c(0, 1, 0)))
  #plot(c2effB, gm$gmap[chr], columns=1:2,
  #     bgcolor=&quot;gray95&quot;, legend=&quot;bottomleft&quot;, 
  #     main = paste(&quot;chr&quot;, chr=peaks$chr[i], &quot;pos&quot;, peaks$pos[i], &quot;(&quot;,peaks$lodcolumn[i],&quot;)&quot;)
  #     )
  #plot(c2effBb, gm$gmap[chr], columns=1:2,
  #     bgcolor=&quot;gray95&quot;, legend=&quot;bottomleft&quot;, 
  #     main = paste(&quot;chr&quot;, chr=peaks$chr[i], &quot;pos&quot;, peaks$pos[i], &quot;(&quot;,peaks$lodcolumn[i],&quot;)&quot;)
  #     )
  ##last_coef &lt;- unclass(c2effB)[nrow(c2effB),2:3] # last two coefficients
  ##for(t in seq(along=last_coef))
  ##  axis(side=4, at=last_coef[t], names(last_coef)[t], tick=FALSE)


  #Table 1
  chr = peaks_mba$chr[i]
  start=as.numeric(peaks_mba$ci_lo[i])
  end=as.numeric(peaks_mba$ci_hi[i])

  genesgss = query_genes(chr, start, end)

  write.csv(genesgss, file=paste0(&quot;data/ici.vs.eoi_&quot;,peaks$phenotype[i],&quot;-no.covariates_genes_chr-&quot;,chr,&quot;_peak.marker-&quot;,peaks$marker[i],&quot;_lod.drop-1.5_snpsqc_5.batches_0.csv&quot;), quote=F)

  rownames(genesgss) &lt;- NULL
  genesgss$strand_old = genesgss$strand
  genesgss$strand[genesgss$strand==&quot;+&quot;] &lt;- &quot;positive&quot;
  genesgss$strand[genesgss$strand==&quot;-&quot;] &lt;- &quot;negative&quot;

  #genesgss &lt;- 
  #table &lt;- 
  #genesgss[,c(&quot;chr&quot;,&quot;type&quot;,&quot;start&quot;,&quot;stop&quot;,&quot;strand&quot;,&quot;ID&quot;,&quot;Name&quot;,&quot;Dbxref&quot;,&quot;gene_id&quot;,&quot;mgi_type&quot;,&quot;description&quot;)] %&gt;%
  #kable(escape = F,align = c(&quot;ccccccccccc&quot;)) %&gt;%
  #kable_styling(&quot;striped&quot;, full_width = T) #%&gt;% 
  #cat #%&gt;%
  #column_spec(1, bold=TRUE)
#
  #print(kable(genesgss[,c(&quot;chr&quot;,&quot;type&quot;,&quot;start&quot;,&quot;stop&quot;,&quot;strand&quot;,&quot;ID&quot;,&quot;Name&quot;,&quot;Dbxref&quot;,&quot;gene_id&quot;,&quot;mgi_type&quot;,&quot;description&quot;)], escape = F,align = c(&quot;ccccccccccc&quot;)))

  print(kable(genesgss[,c(&quot;chr&quot;,&quot;type&quot;,&quot;start&quot;,&quot;stop&quot;,&quot;strand&quot;,&quot;ID&quot;,&quot;Name&quot;,&quot;Dbxref&quot;,&quot;gene_id&quot;,&quot;mgi_type&quot;,&quot;description&quot;)], &quot;html&quot;) %&gt;% kable_styling(&quot;striped&quot;, full_width = T))

  #table
  

}

} else {
  print(paste0(&quot;There are too many peaks (&quot;,nrow(peaks_mba),&quot; peaks) that have a LOD that reaches suggestive (p&lt;0.05) level of &quot;,summary(operm,alpha=0.05)$A, &quot; [autosomes]/&quot;,summary(operm,alpha=0.05)$X, &quot; [x-chromosome]&quot;))
}</code></pre>
<img src="figure/4.1.2_qtl.analysis_cont_age_ici.vs.eoi_snpsqc_pheno.corrected.cleaned_5.batches_0.Rmd/QTL%20effects-1.png" width="912" style="display: block; margin: auto;" /><img src="figure/4.1.2_qtl.analysis_cont_age_ici.vs.eoi_snpsqc_pheno.corrected.cleaned_5.batches_0.Rmd/QTL%20effects-2.png" width="912" style="display: block; margin: auto;" /><img src="figure/4.1.2_qtl.analysis_cont_age_ici.vs.eoi_snpsqc_pheno.corrected.cleaned_5.batches_0.Rmd/QTL%20effects-3.png" width="912" style="display: block; margin: auto;" />
<table class="table table-striped" style="margin-left: auto; margin-right: auto;">
<thead>
<tr>
<th style="text-align:left;">
chr
</th>
<th style="text-align:left;">
type
</th>
<th style="text-align:right;">
start
</th>
<th style="text-align:right;">
stop
</th>
<th style="text-align:left;">
strand
</th>
<th style="text-align:left;">
ID
</th>
<th style="text-align:left;">
Name
</th>
<th style="text-align:left;">
Dbxref
</th>
<th style="text-align:left;">
gene_id
</th>
<th style="text-align:left;">
mgi_type
</th>
<th style="text-align:left;">
description
</th>
</tr>
</thead>
<tbody>
<tr>
<td style="text-align:left;">
3
</td>
<td style="text-align:left;">
gene
</td>
<td style="text-align:right;">
34.10427
</td>
<td style="text-align:right;">
34.68262
</td>
<td style="text-align:left;">
positive
</td>
<td style="text-align:left;">
MGI_C57BL6J_2444112
</td>
<td style="text-align:left;">
Sox2ot
</td>
<td style="text-align:left;">
NCBI_Gene:320478,ENSEMBL:ENSMUSG00000105265
</td>
<td style="text-align:left;">
MGI:2444112
</td>
<td style="text-align:left;">
lncRNA gene
</td>
<td style="text-align:left;">
SOX2 overlapping transcript (non-protein coding)
</td>
</tr>
<tr>
<td style="text-align:left;">
3
</td>
<td style="text-align:left;">
pseudogene
</td>
<td style="text-align:right;">
34.18331
</td>
<td style="text-align:right;">
34.18419
</td>
<td style="text-align:left;">
positive
</td>
<td style="text-align:left;">
MGI_C57BL6J_3647478
</td>
<td style="text-align:left;">
Gm5708
</td>
<td style="text-align:left;">
NCBI_Gene:435727,ENSEMBL:ENSMUSG00000105739
</td>
<td style="text-align:left;">
MGI:3647478
</td>
<td style="text-align:left;">
pseudogene
</td>
<td style="text-align:left;">
predicted gene 5708
</td>
</tr>
<tr>
<td style="text-align:left;">
3
</td>
<td style="text-align:left;">
gene
</td>
<td style="text-align:right;">
34.28133
</td>
<td style="text-align:right;">
34.35199
</td>
<td style="text-align:left;">
negative
</td>
<td style="text-align:left;">
MGI_C57BL6J_5621390
</td>
<td style="text-align:left;">
Gm38505
</td>
<td style="text-align:left;">
NCBI_Gene:102638008,ENSEMBL:ENSMUSG00000092196
</td>
<td style="text-align:left;">
MGI:5621390
</td>
<td style="text-align:left;">
lncRNA gene
</td>
<td style="text-align:left;">
predicted gene, 38505
</td>
</tr>
<tr>
<td style="text-align:left;">
3
</td>
<td style="text-align:left;">
gene
</td>
<td style="text-align:right;">
34.41544
</td>
<td style="text-align:right;">
34.41816
</td>
<td style="text-align:left;">
positive
</td>
<td style="text-align:left;">
MGI_C57BL6J_5593758
</td>
<td style="text-align:left;">
Gm34599
</td>
<td style="text-align:left;">
NCBI_Gene:102637903,ENSEMBL:ENSMUSG00000105577
</td>
<td style="text-align:left;">
MGI:5593758
</td>
<td style="text-align:left;">
lncRNA gene
</td>
<td style="text-align:left;">
predicted gene, 34599
</td>
</tr>
<tr>
<td style="text-align:left;">
3
</td>
<td style="text-align:left;">
gene
</td>
<td style="text-align:right;">
34.42737
</td>
<td style="text-align:right;">
34.43510
</td>
<td style="text-align:left;">
negative
</td>
<td style="text-align:left;">
MGI_C57BL6J_5141980
</td>
<td style="text-align:left;">
Gm20515
</td>
<td style="text-align:left;">
ENSEMBL:ENSMUSG00000092242
</td>
<td style="text-align:left;">
MGI:5141980
</td>
<td style="text-align:left;">
lncRNA gene
</td>
<td style="text-align:left;">
predicted gene 20515
</td>
</tr>
<tr>
<td style="text-align:left;">
3
</td>
<td style="text-align:left;">
gene
</td>
<td style="text-align:right;">
34.48143
</td>
<td style="text-align:right;">
34.48221
</td>
<td style="text-align:left;">
negative
</td>
<td style="text-align:left;">
MGI_C57BL6J_5579841
</td>
<td style="text-align:left;">
Gm29135
</td>
<td style="text-align:left;">
ENSEMBL:ENSMUSG00000099437
</td>
<td style="text-align:left;">
MGI:5579841
</td>
<td style="text-align:left;">
lncRNA gene
</td>
<td style="text-align:left;">
predicted gene 29135
</td>
</tr>
<tr>
<td style="text-align:left;">
3
</td>
<td style="text-align:left;">
gene
</td>
<td style="text-align:right;">
34.56034
</td>
<td style="text-align:right;">
34.56055
</td>
<td style="text-align:left;">
positive
</td>
<td style="text-align:left;">
MGI_C57BL6J_5530785
</td>
<td style="text-align:left;">
Gm27403
</td>
<td style="text-align:left;">
ENSEMBL:ENSMUSG00000098795
</td>
<td style="text-align:left;">
MGI:5530785
</td>
<td style="text-align:left;">
unclassified non-coding RNA gene
</td>
<td style="text-align:left;">
predicted gene, 27403
</td>
</tr>
<tr>
<td style="text-align:left;">
3
</td>
<td style="text-align:left;">
gene
</td>
<td style="text-align:right;">
34.56526
</td>
<td style="text-align:right;">
34.57220
</td>
<td style="text-align:left;">
negative
</td>
<td style="text-align:left;">
MGI_C57BL6J_5662832
</td>
<td style="text-align:left;">
Gm42695
</td>
<td style="text-align:left;">
ENSEMBL:ENSMUSG00000106525
</td>
<td style="text-align:left;">
MGI:5662832
</td>
<td style="text-align:left;">
lncRNA gene
</td>
<td style="text-align:left;">
predicted gene 42695
</td>
</tr>
<tr>
<td style="text-align:left;">
3
</td>
<td style="text-align:left;">
gene
</td>
<td style="text-align:right;">
34.59782
</td>
<td style="text-align:right;">
34.60488
</td>
<td style="text-align:left;">
negative
</td>
<td style="text-align:left;">
MGI_C57BL6J_5625090
</td>
<td style="text-align:left;">
Gm42205
</td>
<td style="text-align:left;">
NCBI_Gene:105247023,ENSEMBL:ENSMUSG00000105222
</td>
<td style="text-align:left;">
MGI:5625090
</td>
<td style="text-align:left;">
lncRNA gene
</td>
<td style="text-align:left;">
predicted gene, 42205
</td>
</tr>
<tr>
<td style="text-align:left;">
3
</td>
<td style="text-align:left;">
gene
</td>
<td style="text-align:right;">
34.63846
</td>
<td style="text-align:right;">
34.63854
</td>
<td style="text-align:left;">
positive
</td>
<td style="text-align:left;">
MGI_C57BL6J_3811407
</td>
<td style="text-align:left;">
Mir1897
</td>
<td style="text-align:left;">
miRBase:MI0008314,NCBI_Gene:100316679,ENSEMBL:ENSMUSG00000104797
</td>
<td style="text-align:left;">
MGI:3811407
</td>
<td style="text-align:left;">
miRNA gene
</td>
<td style="text-align:left;">
microRNA 1897
</td>
</tr>
<tr>
<td style="text-align:left;">
3
</td>
<td style="text-align:left;">
gene
</td>
<td style="text-align:right;">
34.64326
</td>
<td style="text-align:right;">
34.64610
</td>
<td style="text-align:left;">
positive
</td>
<td style="text-align:left;">
MGI_C57BL6J_5662829
</td>
<td style="text-align:left;">
Gm42692
</td>
<td style="text-align:left;">
ENSEMBL:ENSMUSG00000104901
</td>
<td style="text-align:left;">
MGI:5662829
</td>
<td style="text-align:left;">
unclassified gene
</td>
<td style="text-align:left;">
predicted gene 42692
</td>
</tr>
<tr>
<td style="text-align:left;">
3
</td>
<td style="text-align:left;">
gene
</td>
<td style="text-align:right;">
34.64999
</td>
<td style="text-align:right;">
34.65246
</td>
<td style="text-align:left;">
positive
</td>
<td style="text-align:left;">
MGI_C57BL6J_98364
</td>
<td style="text-align:left;">
Sox2
</td>
<td style="text-align:left;">
NCBI_Gene:20674,ENSEMBL:ENSMUSG00000074637
</td>
<td style="text-align:left;">
MGI:98364
</td>
<td style="text-align:left;">
protein coding gene
</td>
<td style="text-align:left;">
SRY (sex determining region Y)-box 2
</td>
</tr>
<tr>
<td style="text-align:left;">
3
</td>
<td style="text-align:left;">
gene
</td>
<td style="text-align:right;">
34.65318
</td>
<td style="text-align:right;">
34.65328
</td>
<td style="text-align:left;">
positive
</td>
<td style="text-align:left;">
MGI_C57BL6J_5530993
</td>
<td style="text-align:left;">
Gm27611
</td>
<td style="text-align:left;">
ENSEMBL:ENSMUSG00000098254
</td>
<td style="text-align:left;">
MGI:5530993
</td>
<td style="text-align:left;">
unclassified non-coding RNA gene
</td>
<td style="text-align:left;">
predicted gene, 27611
</td>
</tr>
<tr>
<td style="text-align:left;">
3
</td>
<td style="text-align:left;">
gene
</td>
<td style="text-align:right;">
34.66366
</td>
<td style="text-align:right;">
34.66429
</td>
<td style="text-align:left;">
negative
</td>
<td style="text-align:left;">
MGI_C57BL6J_5662830
</td>
<td style="text-align:left;">
Gm42693
</td>
<td style="text-align:left;">
ENSEMBL:ENSMUSG00000105284
</td>
<td style="text-align:left;">
MGI:5662830
</td>
<td style="text-align:left;">
unclassified gene
</td>
<td style="text-align:left;">
predicted gene 42693
</td>
</tr>
<tr>
<td style="text-align:left;">
3
</td>
<td style="text-align:left;">
gene
</td>
<td style="text-align:right;">
34.68263
</td>
<td style="text-align:right;">
34.68275
</td>
<td style="text-align:left;">
positive
</td>
<td style="text-align:left;">
MGI_C57BL6J_5663343
</td>
<td style="text-align:left;">
Gm43206
</td>
<td style="text-align:left;">
ENSEMBL:ENSMUSG00000105101
</td>
<td style="text-align:left;">
MGI:5663343
</td>
<td style="text-align:left;">
unclassified gene
</td>
<td style="text-align:left;">
predicted gene 43206
</td>
</tr>
<tr>
<td style="text-align:left;">
3
</td>
<td style="text-align:left;">
gene
</td>
<td style="text-align:right;">
34.68938
</td>
<td style="text-align:right;">
34.69292
</td>
<td style="text-align:left;">
positive
</td>
<td style="text-align:left;">
MGI_C57BL6J_5594100
</td>
<td style="text-align:left;">
Gm34941
</td>
<td style="text-align:left;">
NCBI_Gene:102638356
</td>
<td style="text-align:left;">
MGI:5594100
</td>
<td style="text-align:left;">
lncRNA gene
</td>
<td style="text-align:left;">
predicted gene, 34941
</td>
</tr>
<tr>
<td style="text-align:left;">
3
</td>
<td style="text-align:left;">
gene
</td>
<td style="text-align:right;">
34.69964
</td>
<td style="text-align:right;">
34.71666
</td>
<td style="text-align:left;">
negative
</td>
<td style="text-align:left;">
MGI_C57BL6J_3781322
</td>
<td style="text-align:left;">
Gm3143
</td>
<td style="text-align:left;">
NCBI_Gene:100041108,ENSEMBL:ENSMUSG00000105153
</td>
<td style="text-align:left;">
MGI:3781322
</td>
<td style="text-align:left;">
lncRNA gene
</td>
<td style="text-align:left;">
predicted gene 3143
</td>
</tr>
<tr>
<td style="text-align:left;">
3
</td>
<td style="text-align:left;">
gene
</td>
<td style="text-align:right;">
34.77208
</td>
<td style="text-align:right;">
34.78235
</td>
<td style="text-align:left;">
positive
</td>
<td style="text-align:left;">
MGI_C57BL6J_5621394
</td>
<td style="text-align:left;">
Gm38509
</td>
<td style="text-align:left;">
NCBI_Gene:102638436,ENSEMBL:ENSMUSG00000106519
</td>
<td style="text-align:left;">
MGI:5621394
</td>
<td style="text-align:left;">
lncRNA gene
</td>
<td style="text-align:left;">
predicted gene, 38509
</td>
</tr>
<tr>
<td style="text-align:left;">
3
</td>
<td style="text-align:left;">
gene
</td>
<td style="text-align:right;">
34.82656
</td>
<td style="text-align:right;">
34.87928
</td>
<td style="text-align:left;">
positive
</td>
<td style="text-align:left;">
MGI_C57BL6J_5434743
</td>
<td style="text-align:left;">
Gm21388
</td>
<td style="text-align:left;">
NCBI_Gene:100861998,ENSEMBL:ENSMUSG00000114832
</td>
<td style="text-align:left;">
MGI:5434743
</td>
<td style="text-align:left;">
lncRNA gene
</td>
<td style="text-align:left;">
predicted gene, 21388
</td>
</tr>
<tr>
<td style="text-align:left;">
3
</td>
<td style="text-align:left;">
gene
</td>
<td style="text-align:right;">
34.92254
</td>
<td style="text-align:right;">
34.92265
</td>
<td style="text-align:left;">
negative
</td>
<td style="text-align:left;">
MGI_C57BL6J_5531088
</td>
<td style="text-align:left;">
Mir6378
</td>
<td style="text-align:left;">
miRBase:MI0021909,NCBI_Gene:102466155,ENSEMBL:ENSMUSG00000098862
</td>
<td style="text-align:left;">
MGI:5531088
</td>
<td style="text-align:left;">
miRNA gene
</td>
<td style="text-align:left;">
microRNA 6378
</td>
</tr>
<tr>
<td style="text-align:left;">
3
</td>
<td style="text-align:left;">
gene
</td>
<td style="text-align:right;">
34.95023
</td>
<td style="text-align:right;">
34.95953
</td>
<td style="text-align:left;">
positive
</td>
<td style="text-align:left;">
MGI_C57BL6J_5594414
</td>
<td style="text-align:left;">
Gm35255
</td>
<td style="text-align:left;">
NCBI_Gene:102638771
</td>
<td style="text-align:left;">
MGI:5594414
</td>
<td style="text-align:left;">
lncRNA gene
</td>
<td style="text-align:left;">
predicted gene, 35255
</td>
</tr>
<tr>
<td style="text-align:left;">
3
</td>
<td style="text-align:left;">
gene
</td>
<td style="text-align:right;">
35.25646
</td>
<td style="text-align:right;">
35.27613
</td>
<td style="text-align:left;">
negative
</td>
<td style="text-align:left;">
MGI_C57BL6J_1923648
</td>
<td style="text-align:left;">
1700017M07Rik
</td>
<td style="text-align:left;">
NCBI_Gene:76398,ENSEMBL:ENSMUSG00000105651
</td>
<td style="text-align:left;">
MGI:1923648
</td>
<td style="text-align:left;">
lncRNA gene
</td>
<td style="text-align:left;">
RIKEN cDNA 1700017M07 gene
</td>
</tr>
<tr>
<td style="text-align:left;">
3
</td>
<td style="text-align:left;">
gene
</td>
<td style="text-align:right;">
35.31844
</td>
<td style="text-align:right;">
35.33011
</td>
<td style="text-align:left;">
positive
</td>
<td style="text-align:left;">
MGI_C57BL6J_5594584
</td>
<td style="text-align:left;">
Gm35425
</td>
<td style="text-align:left;">
NCBI_Gene:102639001
</td>
<td style="text-align:left;">
MGI:5594584
</td>
<td style="text-align:left;">
lncRNA gene
</td>
<td style="text-align:left;">
predicted gene, 35425
</td>
</tr>
<tr>
<td style="text-align:left;">
3
</td>
<td style="text-align:left;">
gene
</td>
<td style="text-align:right;">
35.32561
</td>
<td style="text-align:right;">
35.32570
</td>
<td style="text-align:left;">
positive
</td>
<td style="text-align:left;">
MGI_C57BL6J_5455219
</td>
<td style="text-align:left;">
Gm25442
</td>
<td style="text-align:left;">
ENSEMBL:ENSMUSG00000093499
</td>
<td style="text-align:left;">
MGI:5455219
</td>
<td style="text-align:left;">
miRNA gene
</td>
<td style="text-align:left;">
predicted gene, 25442
</td>
</tr>
<tr>
<td style="text-align:left;">
3
</td>
<td style="text-align:left;">
gene
</td>
<td style="text-align:right;">
35.33770
</td>
<td style="text-align:right;">
35.40595
</td>
<td style="text-align:left;">
negative
</td>
<td style="text-align:left;">
MGI_C57BL6J_5621397
</td>
<td style="text-align:left;">
Gm38512
</td>
<td style="text-align:left;">
NCBI_Gene:102638848
</td>
<td style="text-align:left;">
MGI:5621397
</td>
<td style="text-align:left;">
lncRNA gene
</td>
<td style="text-align:left;">
predicted gene, 38512
</td>
</tr>
<tr>
<td style="text-align:left;">
3
</td>
<td style="text-align:left;">
gene
</td>
<td style="text-align:right;">
35.41383
</td>
<td style="text-align:right;">
35.41400
</td>
<td style="text-align:left;">
positive
</td>
<td style="text-align:left;">
MGI_C57BL6J_5663215
</td>
<td style="text-align:left;">
Gm43078
</td>
<td style="text-align:left;">
ENSEMBL:ENSMUSG00000106512
</td>
<td style="text-align:left;">
MGI:5663215
</td>
<td style="text-align:left;">
unclassified gene
</td>
<td style="text-align:left;">
predicted gene 43078
</td>
</tr>
<tr>
<td style="text-align:left;">
3
</td>
<td style="text-align:left;">
pseudogene
</td>
<td style="text-align:right;">
35.53688
</td>
<td style="text-align:right;">
35.53780
</td>
<td style="text-align:left;">
negative
</td>
<td style="text-align:left;">
MGI_C57BL6J_3643628
</td>
<td style="text-align:left;">
Gm7733
</td>
<td style="text-align:left;">
NCBI_Gene:665655,ENSEMBL:ENSMUSG00000106496
</td>
<td style="text-align:left;">
MGI:3643628
</td>
<td style="text-align:left;">
pseudogene
</td>
<td style="text-align:left;">
predicted gene 7733
</td>
</tr>
<tr>
<td style="text-align:left;">
3
</td>
<td style="text-align:left;">
pseudogene
</td>
<td style="text-align:right;">
35.54589
</td>
<td style="text-align:right;">
35.54633
</td>
<td style="text-align:left;">
positive
</td>
<td style="text-align:left;">
MGI_C57BL6J_1347022
</td>
<td style="text-align:left;">
Ube2l3-ps1
</td>
<td style="text-align:left;">
ENSEMBL:ENSMUSG00000105040
</td>
<td style="text-align:left;">
MGI:1347022
</td>
<td style="text-align:left;">
pseudogene
</td>
<td style="text-align:left;">
ubiquitin-conjugating enzyme E2L 3, pseudogene 1
</td>
</tr>
<tr>
<td style="text-align:left;">
3
</td>
<td style="text-align:left;">
gene
</td>
<td style="text-align:right;">
35.59715
</td>
<td style="text-align:right;">
35.66616
</td>
<td style="text-align:left;">
negative
</td>
<td style="text-align:left;">
MGI_C57BL6J_3779620
</td>
<td style="text-align:left;">
Gm6639
</td>
<td style="text-align:left;">
NCBI_Gene:625963,ENSEMBL:ENSMUSG00000105662
</td>
<td style="text-align:left;">
MGI:3779620
</td>
<td style="text-align:left;">
lncRNA gene
</td>
<td style="text-align:left;">
predicted gene 6639
</td>
</tr>
<tr>
<td style="text-align:left;">
3
</td>
<td style="text-align:left;">
gene
</td>
<td style="text-align:right;">
35.73537
</td>
<td style="text-align:right;">
35.73548
</td>
<td style="text-align:left;">
positive
</td>
<td style="text-align:left;">
MGI_C57BL6J_5455473
</td>
<td style="text-align:left;">
Gm25696
</td>
<td style="text-align:left;">
ENSEMBL:ENSMUSG00000064834
</td>
<td style="text-align:left;">
MGI:5455473
</td>
<td style="text-align:left;">
snRNA gene
</td>
<td style="text-align:left;">
predicted gene, 25696
</td>
</tr>
<tr>
<td style="text-align:left;">
3
</td>
<td style="text-align:left;">
gene
</td>
<td style="text-align:right;">
35.75406
</td>
<td style="text-align:right;">
35.85628
</td>
<td style="text-align:left;">
positive
</td>
<td style="text-align:left;">
MGI_C57BL6J_1923545
</td>
<td style="text-align:left;">
Atp11b
</td>
<td style="text-align:left;">
NCBI_Gene:76295,ENSEMBL:ENSMUSG00000037400
</td>
<td style="text-align:left;">
MGI:1923545
</td>
<td style="text-align:left;">
protein coding gene
</td>
<td style="text-align:left;">
ATPase, class VI, type 11B
</td>
</tr>
<tr>
<td style="text-align:left;">
3
</td>
<td style="text-align:left;">
gene
</td>
<td style="text-align:right;">
35.84534
</td>
<td style="text-align:right;">
35.84590
</td>
<td style="text-align:left;">
positive
</td>
<td style="text-align:left;">
MGI_C57BL6J_5663340
</td>
<td style="text-align:left;">
Gm43203
</td>
<td style="text-align:left;">
ENSEMBL:ENSMUSG00000104897
</td>
<td style="text-align:left;">
MGI:5663340
</td>
<td style="text-align:left;">
unclassified gene
</td>
<td style="text-align:left;">
predicted gene 43203
</td>
</tr>
<tr>
<td style="text-align:left;">
3
</td>
<td style="text-align:left;">
gene
</td>
<td style="text-align:right;">
35.86162
</td>
<td style="text-align:right;">
35.86212
</td>
<td style="text-align:left;">
positive
</td>
<td style="text-align:left;">
MGI_C57BL6J_1915850
</td>
<td style="text-align:left;">
1110021P09Rik
</td>
<td style="text-align:left;">
NA
</td>
<td style="text-align:left;">
NA
</td>
<td style="text-align:left;">
unclassified gene
</td>
<td style="text-align:left;">
RIKEN cDNA 1110021P09 gene
</td>
</tr>
<tr>
<td style="text-align:left;">
3
</td>
<td style="text-align:left;">
pseudogene
</td>
<td style="text-align:right;">
35.86386
</td>
<td style="text-align:right;">
35.86507
</td>
<td style="text-align:left;">
positive
</td>
<td style="text-align:left;">
MGI_C57BL6J_3647088
</td>
<td style="text-align:left;">
Gm7741
</td>
<td style="text-align:left;">
NCBI_Gene:665667,ENSEMBL:ENSMUSG00000097713
</td>
<td style="text-align:left;">
MGI:3647088
</td>
<td style="text-align:left;">
pseudogene
</td>
<td style="text-align:left;">
predicted gene 7741
</td>
</tr>
<tr>
<td style="text-align:left;">
3
</td>
<td style="text-align:left;">
pseudogene
</td>
<td style="text-align:right;">
35.87565
</td>
<td style="text-align:right;">
35.88335
</td>
<td style="text-align:left;">
negative
</td>
<td style="text-align:left;">
MGI_C57BL6J_5295701
</td>
<td style="text-align:left;">
Gm20595
</td>
<td style="text-align:left;">
NCBI_Gene:100463514
</td>
<td style="text-align:left;">
MGI:5295701
</td>
<td style="text-align:left;">
pseudogene
</td>
<td style="text-align:left;">
predicted gene, 20595
</td>
</tr>
<tr>
<td style="text-align:left;">
3
</td>
<td style="text-align:left;">
gene
</td>
<td style="text-align:right;">
35.89211
</td>
<td style="text-align:right;">
35.93744
</td>
<td style="text-align:left;">
negative
</td>
<td style="text-align:left;">
MGI_C57BL6J_2150386
</td>
<td style="text-align:left;">
Dcun1d1
</td>
<td style="text-align:left;">
NCBI_Gene:114893,ENSEMBL:ENSMUSG00000027708
</td>
<td style="text-align:left;">
MGI:2150386
</td>
<td style="text-align:left;">
protein coding gene
</td>
<td style="text-align:left;">
DCN1, defective in cullin neddylation 1, domain containing 1 (S. cerevisiae)
</td>
</tr>
<tr>
<td style="text-align:left;">
3
</td>
<td style="text-align:left;">
gene
</td>
<td style="text-align:right;">
35.93304
</td>
<td style="text-align:right;">
35.99084
</td>
<td style="text-align:left;">
positive
</td>
<td style="text-align:left;">
MGI_C57BL6J_3802149
</td>
<td style="text-align:left;">
Gm15952
</td>
<td style="text-align:left;">
NCBI_Gene:102639351,ENSEMBL:ENSMUSG00000085655
</td>
<td style="text-align:left;">
MGI:3802149
</td>
<td style="text-align:left;">
lncRNA gene
</td>
<td style="text-align:left;">
predicted gene 15952
</td>
</tr>
<tr>
<td style="text-align:left;">
3
</td>
<td style="text-align:left;">
gene
</td>
<td style="text-align:right;">
35.95544
</td>
<td style="text-align:right;">
35.95626
</td>
<td style="text-align:left;">
negative
</td>
<td style="text-align:left;">
MGI_C57BL6J_1925753
</td>
<td style="text-align:left;">
8430422M14Rik
</td>
<td style="text-align:left;">
ENSEMBL:ENSMUSG00000105095
</td>
<td style="text-align:left;">
MGI:1925753
</td>
<td style="text-align:left;">
unclassified gene
</td>
<td style="text-align:left;">
RIKEN cDNA 8430422M14 gene
</td>
</tr>
<tr>
<td style="text-align:left;">
3
</td>
<td style="text-align:left;">
gene
</td>
<td style="text-align:right;">
35.95629
</td>
<td style="text-align:right;">
36.00068
</td>
<td style="text-align:left;">
negative
</td>
<td style="text-align:left;">
MGI_C57BL6J_1919289
</td>
<td style="text-align:left;">
Mccc1
</td>
<td style="text-align:left;">
NCBI_Gene:72039,ENSEMBL:ENSMUSG00000027709
</td>
<td style="text-align:left;">
MGI:1919289
</td>
<td style="text-align:left;">
protein coding gene
</td>
<td style="text-align:left;">
methylcrotonoyl-Coenzyme A carboxylase 1 (alpha)
</td>
</tr>
<tr>
<td style="text-align:left;">
3
</td>
<td style="text-align:left;">
gene
</td>
<td style="text-align:right;">
35.96938
</td>
<td style="text-align:right;">
35.97203
</td>
<td style="text-align:left;">
negative
</td>
<td style="text-align:left;">
MGI_C57BL6J_5663217
</td>
<td style="text-align:left;">
Gm43080
</td>
<td style="text-align:left;">
ENSEMBL:ENSMUSG00000105556
</td>
<td style="text-align:left;">
MGI:5663217
</td>
<td style="text-align:left;">
unclassified gene
</td>
<td style="text-align:left;">
predicted gene 43080
</td>
</tr>
<tr>
<td style="text-align:left;">
3
</td>
<td style="text-align:left;">
pseudogene
</td>
<td style="text-align:right;">
35.97377
</td>
<td style="text-align:right;">
35.97409
</td>
<td style="text-align:left;">
positive
</td>
<td style="text-align:left;">
MGI_C57BL6J_3704213
</td>
<td style="text-align:left;">
B230207O21Rik
</td>
<td style="text-align:left;">
ENSEMBL:ENSMUSG00000106096
</td>
<td style="text-align:left;">
MGI:3704213
</td>
<td style="text-align:left;">
pseudogene
</td>
<td style="text-align:left;">
RIKEN cDNA B230207O21 gene
</td>
</tr>
<tr>
<td style="text-align:left;">
3
</td>
<td style="text-align:left;">
gene
</td>
<td style="text-align:right;">
35.99983
</td>
<td style="text-align:right;">
36.02511
</td>
<td style="text-align:left;">
positive
</td>
<td style="text-align:left;">
MGI_C57BL6J_3590669
</td>
<td style="text-align:left;">
Mccc1os
</td>
<td style="text-align:left;">
NCBI_Gene:654424,ENSEMBL:ENSMUSG00000086392
</td>
<td style="text-align:left;">
MGI:3590669
</td>
<td style="text-align:left;">
antisense lncRNA gene
</td>
<td style="text-align:left;">
methylcrotonoyl-Coenzyme A carboxylase 1 (alpha), opposite strand
</td>
</tr>
<tr>
<td style="text-align:left;">
3
</td>
<td style="text-align:left;">
gene
</td>
<td style="text-align:right;">
36.00724
</td>
<td style="text-align:right;">
36.05355
</td>
<td style="text-align:left;">
negative
</td>
<td style="text-align:left;">
MGI_C57BL6J_2681852
</td>
<td style="text-align:left;">
Ccdc144b
</td>
<td style="text-align:left;">
NCBI_Gene:241943,ENSEMBL:ENSMUSG00000047696
</td>
<td style="text-align:left;">
MGI:2681852
</td>
<td style="text-align:left;">
protein coding gene
</td>
<td style="text-align:left;">
coiled-coil domain containing 144B
</td>
</tr>
<tr>
<td style="text-align:left;">
3
</td>
<td style="text-align:left;">
gene
</td>
<td style="text-align:right;">
36.06598
</td>
<td style="text-align:right;">
36.09286
</td>
<td style="text-align:left;">
positive
</td>
<td style="text-align:left;">
MGI_C57BL6J_1914272
</td>
<td style="text-align:left;">
Acad9
</td>
<td style="text-align:left;">
NCBI_Gene:229211,ENSEMBL:ENSMUSG00000027710
</td>
<td style="text-align:left;">
MGI:1914272
</td>
<td style="text-align:left;">
protein coding gene
</td>
<td style="text-align:left;">
acyl-Coenzyme A dehydrogenase family, member 9
</td>
</tr>
<tr>
<td style="text-align:left;">
3
</td>
<td style="text-align:left;">
gene
</td>
<td style="text-align:right;">
36.08886
</td>
<td style="text-align:right;">
36.09145
</td>
<td style="text-align:left;">
positive
</td>
<td style="text-align:left;">
MGI_C57BL6J_5663216
</td>
<td style="text-align:left;">
Gm43079
</td>
<td style="text-align:left;">
ENSEMBL:ENSMUSG00000105055
</td>
<td style="text-align:left;">
MGI:5663216
</td>
<td style="text-align:left;">
lncRNA gene
</td>
<td style="text-align:left;">
predicted gene 43079
</td>
</tr>
<tr>
<td style="text-align:left;">
3
</td>
<td style="text-align:left;">
gene
</td>
<td style="text-align:right;">
36.09718
</td>
<td style="text-align:right;">
36.09914
</td>
<td style="text-align:left;">
positive
</td>
<td style="text-align:left;">
MGI_C57BL6J_5663342
</td>
<td style="text-align:left;">
Gm43205
</td>
<td style="text-align:left;">
ENSEMBL:ENSMUSG00000105419
</td>
<td style="text-align:left;">
MGI:5663342
</td>
<td style="text-align:left;">
unclassified gene
</td>
<td style="text-align:left;">
predicted gene 43205
</td>
</tr>
<tr>
<td style="text-align:left;">
3
</td>
<td style="text-align:left;">
gene
</td>
<td style="text-align:right;">
36.15102
</td>
<td style="text-align:right;">
36.17034
</td>
<td style="text-align:left;">
positive
</td>
<td style="text-align:left;">
MGI_C57BL6J_1098769
</td>
<td style="text-align:left;">
Zfp267
</td>
<td style="text-align:left;">
NCBI_Gene:241944,ENSEMBL:ENSMUSG00000033883
</td>
<td style="text-align:left;">
MGI:1098769
</td>
<td style="text-align:left;">
protein coding gene
</td>
<td style="text-align:left;">
zinc finger protein 267
</td>
</tr>
<tr>
<td style="text-align:left;">
3
</td>
<td style="text-align:left;">
gene
</td>
<td style="text-align:right;">
36.17942
</td>
<td style="text-align:right;">
36.22231
</td>
<td style="text-align:left;">
negative
</td>
<td style="text-align:left;">
MGI_C57BL6J_2677633
</td>
<td style="text-align:left;">
Qrfpr
</td>
<td style="text-align:left;">
NCBI_Gene:229214,ENSEMBL:ENSMUSG00000058400
</td>
<td style="text-align:left;">
MGI:2677633
</td>
<td style="text-align:left;">
protein coding gene
</td>
<td style="text-align:left;">
pyroglutamylated RFamide peptide receptor
</td>
</tr>
<tr>
<td style="text-align:left;">
3
</td>
<td style="text-align:left;">
gene
</td>
<td style="text-align:right;">
36.20329
</td>
<td style="text-align:right;">
36.20562
</td>
<td style="text-align:left;">
positive
</td>
<td style="text-align:left;">
MGI_C57BL6J_5663392
</td>
<td style="text-align:left;">
Gm43255
</td>
<td style="text-align:left;">
ENSEMBL:ENSMUSG00000104568
</td>
<td style="text-align:left;">
MGI:5663392
</td>
<td style="text-align:left;">
lncRNA gene
</td>
<td style="text-align:left;">
predicted gene 43255
</td>
</tr>
<tr>
<td style="text-align:left;">
3
</td>
<td style="text-align:left;">
pseudogene
</td>
<td style="text-align:right;">
36.21381
</td>
<td style="text-align:right;">
36.21476
</td>
<td style="text-align:left;">
negative
</td>
<td style="text-align:left;">
MGI_C57BL6J_5011078
</td>
<td style="text-align:left;">
Gm18893
</td>
<td style="text-align:left;">
NCBI_Gene:100417907,ENSEMBL:ENSMUSG00000106180
</td>
<td style="text-align:left;">
MGI:5011078
</td>
<td style="text-align:left;">
pseudogene
</td>
<td style="text-align:left;">
predicted gene, 18893
</td>
</tr>
<tr>
<td style="text-align:left;">
3
</td>
<td style="text-align:left;">
gene
</td>
<td style="text-align:right;">
36.27233
</td>
<td style="text-align:right;">
36.27896
</td>
<td style="text-align:left;">
negative
</td>
<td style="text-align:left;">
MGI_C57BL6J_5625093
</td>
<td style="text-align:left;">
Gm42208
</td>
<td style="text-align:left;">
NCBI_Gene:105247026,ENSEMBL:ENSMUSG00000106140
</td>
<td style="text-align:left;">
MGI:5625093
</td>
<td style="text-align:left;">
lncRNA gene
</td>
<td style="text-align:left;">
predicted gene, 42208
</td>
</tr>
<tr>
<td style="text-align:left;">
3
</td>
<td style="text-align:left;">
gene
</td>
<td style="text-align:right;">
36.35422
</td>
<td style="text-align:right;">
36.40651
</td>
<td style="text-align:left;">
negative
</td>
<td style="text-align:left;">
MGI_C57BL6J_5625094
</td>
<td style="text-align:left;">
Gm42209
</td>
<td style="text-align:left;">
NCBI_Gene:105247027
</td>
<td style="text-align:left;">
MGI:5625094
</td>
<td style="text-align:left;">
lncRNA gene
</td>
<td style="text-align:left;">
predicted gene, 42209
</td>
</tr>
<tr>
<td style="text-align:left;">
3
</td>
<td style="text-align:left;">
pseudogene
</td>
<td style="text-align:right;">
36.39470
</td>
<td style="text-align:right;">
36.39520
</td>
<td style="text-align:left;">
negative
</td>
<td style="text-align:left;">
MGI_C57BL6J_5010020
</td>
<td style="text-align:left;">
Gm17835
</td>
<td style="text-align:left;">
NCBI_Gene:100415952,ENSEMBL:ENSMUSG00000105281
</td>
<td style="text-align:left;">
MGI:5010020
</td>
<td style="text-align:left;">
pseudogene
</td>
<td style="text-align:left;">
predicted gene, 17835
</td>
</tr>
<tr>
<td style="text-align:left;">
3
</td>
<td style="text-align:left;">
pseudogene
</td>
<td style="text-align:right;">
36.42158
</td>
<td style="text-align:right;">
36.42212
</td>
<td style="text-align:left;">
positive
</td>
<td style="text-align:left;">
MGI_C57BL6J_5010072
</td>
<td style="text-align:left;">
Gm17887
</td>
<td style="text-align:left;">
NCBI_Gene:100416041,ENSEMBL:ENSMUSG00000105792
</td>
<td style="text-align:left;">
MGI:5010072
</td>
<td style="text-align:left;">
pseudogene
</td>
<td style="text-align:left;">
predicted gene, 17887
</td>
</tr>
<tr>
<td style="text-align:left;">
3
</td>
<td style="text-align:left;">
gene
</td>
<td style="text-align:right;">
36.44892
</td>
<td style="text-align:right;">
36.47589
</td>
<td style="text-align:left;">
negative
</td>
<td style="text-align:left;">
MGI_C57BL6J_106008
</td>
<td style="text-align:left;">
Anxa5
</td>
<td style="text-align:left;">
NCBI_Gene:11747,ENSEMBL:ENSMUSG00000027712
</td>
<td style="text-align:left;">
MGI:106008
</td>
<td style="text-align:left;">
protein coding gene
</td>
<td style="text-align:left;">
annexin A5
</td>
</tr>
<tr>
<td style="text-align:left;">
3
</td>
<td style="text-align:left;">
gene
</td>
<td style="text-align:right;">
36.47546
</td>
<td style="text-align:right;">
36.47552
</td>
<td style="text-align:left;">
negative
</td>
<td style="text-align:left;">
MGI_C57BL6J_5530854
</td>
<td style="text-align:left;">
Mir7009
</td>
<td style="text-align:left;">
miRBase:MI0022858,NCBI_Gene:102466786,ENSEMBL:ENSMUSG00000098808
</td>
<td style="text-align:left;">
MGI:5530854
</td>
<td style="text-align:left;">
miRNA gene
</td>
<td style="text-align:left;">
microRNA 7009
</td>
</tr>
<tr>
<td style="text-align:left;">
3
</td>
<td style="text-align:left;">
gene
</td>
<td style="text-align:right;">
36.47594
</td>
<td style="text-align:right;">
36.48230
</td>
<td style="text-align:left;">
positive
</td>
<td style="text-align:left;">
MGI_C57BL6J_1919532
</td>
<td style="text-align:left;">
1810062G17Rik
</td>
<td style="text-align:left;">
NCBI_Gene:72282,ENSEMBL:ENSMUSG00000027713
</td>
<td style="text-align:left;">
MGI:1919532
</td>
<td style="text-align:left;">
protein coding gene
</td>
<td style="text-align:left;">
RIKEN cDNA 1810062G17 gene
</td>
</tr>
<tr>
<td style="text-align:left;">
3
</td>
<td style="text-align:left;">
gene
</td>
<td style="text-align:right;">
36.49980
</td>
<td style="text-align:right;">
36.50681
</td>
<td style="text-align:left;">
negative
</td>
<td style="text-align:left;">
MGI_C57BL6J_3650340
</td>
<td style="text-align:left;">
Gm11548
</td>
<td style="text-align:left;">
NCBI_Gene:329623,ENSEMBL:ENSMUSG00000085320
</td>
<td style="text-align:left;">
MGI:3650340
</td>
<td style="text-align:left;">
lncRNA gene
</td>
<td style="text-align:left;">
predicted gene 11548
</td>
</tr>
<tr>
<td style="text-align:left;">
3
</td>
<td style="text-align:left;">
gene
</td>
<td style="text-align:right;">
36.51506
</td>
<td style="text-align:right;">
36.52181
</td>
<td style="text-align:left;">
negative
</td>
<td style="text-align:left;">
MGI_C57BL6J_3650339
</td>
<td style="text-align:left;">
Gm11549
</td>
<td style="text-align:left;">
NCBI_Gene:100503068,ENSEMBL:ENSMUSG00000085007
</td>
<td style="text-align:left;">
MGI:3650339
</td>
<td style="text-align:left;">
lincRNA gene
</td>
<td style="text-align:left;">
predicted gene 11549
</td>
</tr>
<tr>
<td style="text-align:left;">
3
</td>
<td style="text-align:left;">
gene
</td>
<td style="text-align:right;">
36.52922
</td>
<td style="text-align:right;">
36.53233
</td>
<td style="text-align:left;">
negative
</td>
<td style="text-align:left;">
MGI_C57BL6J_5663674
</td>
<td style="text-align:left;">
Gm43537
</td>
<td style="text-align:left;">
ENSEMBL:ENSMUSG00000105022
</td>
<td style="text-align:left;">
MGI:5663674
</td>
<td style="text-align:left;">
unclassified gene
</td>
<td style="text-align:left;">
predicted gene 43537
</td>
</tr>
<tr>
<td style="text-align:left;">
3
</td>
<td style="text-align:left;">
pseudogene
</td>
<td style="text-align:right;">
36.53917
</td>
<td style="text-align:right;">
36.54010
</td>
<td style="text-align:left;">
negative
</td>
<td style="text-align:left;">
MGI_C57BL6J_3650891
</td>
<td style="text-align:left;">
Gm11550
</td>
<td style="text-align:left;">
NCBI_Gene:665709,ENSEMBL:ENSMUSG00000083388
</td>
<td style="text-align:left;">
MGI:3650891
</td>
<td style="text-align:left;">
pseudogene
</td>
<td style="text-align:left;">
predicted gene 11550
</td>
</tr>
<tr>
<td style="text-align:left;">
3
</td>
<td style="text-align:left;">
gene
</td>
<td style="text-align:right;">
36.55261
</td>
<td style="text-align:right;">
36.56573
</td>
<td style="text-align:left;">
positive
</td>
<td style="text-align:left;">
MGI_C57BL6J_1355319
</td>
<td style="text-align:left;">
Exosc9
</td>
<td style="text-align:left;">
NCBI_Gene:50911,ENSEMBL:ENSMUSG00000027714
</td>
<td style="text-align:left;">
MGI:1355319
</td>
<td style="text-align:left;">
protein coding gene
</td>
<td style="text-align:left;">
exosome component 9
</td>
</tr>
<tr>
<td style="text-align:left;">
3
</td>
<td style="text-align:left;">
gene
</td>
<td style="text-align:right;">
36.56486
</td>
<td style="text-align:right;">
36.57303
</td>
<td style="text-align:left;">
negative
</td>
<td style="text-align:left;">
MGI_C57BL6J_108069
</td>
<td style="text-align:left;">
Ccna2
</td>
<td style="text-align:left;">
NCBI_Gene:12428,ENSEMBL:ENSMUSG00000027715
</td>
<td style="text-align:left;">
MGI:108069
</td>
<td style="text-align:left;">
protein coding gene
</td>
<td style="text-align:left;">
cyclin A2
</td>
</tr>
<tr>
<td style="text-align:left;">
3
</td>
<td style="text-align:left;">
gene
</td>
<td style="text-align:right;">
36.57314
</td>
<td style="text-align:right;">
36.61349
</td>
<td style="text-align:left;">
negative
</td>
<td style="text-align:left;">
MGI_C57BL6J_1918742
</td>
<td style="text-align:left;">
Bbs7
</td>
<td style="text-align:left;">
NCBI_Gene:71492,ENSEMBL:ENSMUSG00000037325
</td>
<td style="text-align:left;">
MGI:1918742
</td>
<td style="text-align:left;">
protein coding gene
</td>
<td style="text-align:left;">
Bardet-Biedl syndrome 7 (human)
</td>
</tr>
<tr>
<td style="text-align:left;">
3
</td>
<td style="text-align:left;">
gene
</td>
<td style="text-align:right;">
36.62048
</td>
<td style="text-align:right;">
36.69022
</td>
<td style="text-align:left;">
negative
</td>
<td style="text-align:left;">
MGI_C57BL6J_109526
</td>
<td style="text-align:left;">
Trpc3
</td>
<td style="text-align:left;">
NCBI_Gene:22065,ENSEMBL:ENSMUSG00000027716
</td>
<td style="text-align:left;">
MGI:109526
</td>
<td style="text-align:left;">
protein coding gene
</td>
<td style="text-align:left;">
transient receptor potential cation channel, subfamily C, member 3
</td>
</tr>
<tr>
<td style="text-align:left;">
3
</td>
<td style="text-align:left;">
gene
</td>
<td style="text-align:right;">
36.65407
</td>
<td style="text-align:right;">
36.65501
</td>
<td style="text-align:left;">
negative
</td>
<td style="text-align:left;">
MGI_C57BL6J_5595015
</td>
<td style="text-align:left;">
Gm35856
</td>
<td style="text-align:left;">
NCBI_Gene:102639576
</td>
<td style="text-align:left;">
MGI:5595015
</td>
<td style="text-align:left;">
lncRNA gene
</td>
<td style="text-align:left;">
predicted gene, 35856
</td>
</tr>
<tr>
<td style="text-align:left;">
3
</td>
<td style="text-align:left;">
pseudogene
</td>
<td style="text-align:right;">
36.69480
</td>
<td style="text-align:right;">
36.69505
</td>
<td style="text-align:left;">
negative
</td>
<td style="text-align:left;">
MGI_C57BL6J_3651607
</td>
<td style="text-align:left;">
Gm12377
</td>
<td style="text-align:left;">
ENSEMBL:ENSMUSG00000082548
</td>
<td style="text-align:left;">
MGI:3651607
</td>
<td style="text-align:left;">
pseudogene
</td>
<td style="text-align:left;">
predicted gene 12377
</td>
</tr>
<tr>
<td style="text-align:left;">
3
</td>
<td style="text-align:left;">
pseudogene
</td>
<td style="text-align:right;">
36.82435
</td>
<td style="text-align:right;">
36.82506
</td>
<td style="text-align:left;">
negative
</td>
<td style="text-align:left;">
MGI_C57BL6J_3651608
</td>
<td style="text-align:left;">
Gm12376
</td>
<td style="text-align:left;">
NCBI_Gene:100417988,ENSEMBL:ENSMUSG00000084027
</td>
<td style="text-align:left;">
MGI:3651608
</td>
<td style="text-align:left;">
pseudogene
</td>
<td style="text-align:left;">
predicted gene 12376
</td>
</tr>
<tr>
<td style="text-align:left;">
3
</td>
<td style="text-align:left;">
gene
</td>
<td style="text-align:right;">
36.86306
</td>
<td style="text-align:right;">
37.05303
</td>
<td style="text-align:left;">
positive
</td>
<td style="text-align:left;">
MGI_C57BL6J_2444631
</td>
<td style="text-align:left;">
4932438A13Rik
</td>
<td style="text-align:left;">
NCBI_Gene:229227,ENSEMBL:ENSMUSG00000037270
</td>
<td style="text-align:left;">
MGI:2444631
</td>
<td style="text-align:left;">
protein coding gene
</td>
<td style="text-align:left;">
RIKEN cDNA 4932438A13 gene
</td>
</tr>
<tr>
<td style="text-align:left;">
3
</td>
<td style="text-align:left;">
gene
</td>
<td style="text-align:right;">
37.05221
</td>
<td style="text-align:right;">
37.06003
</td>
<td style="text-align:left;">
negative
</td>
<td style="text-align:left;">
MGI_C57BL6J_3650531
</td>
<td style="text-align:left;">
Gm12531
</td>
<td style="text-align:left;">
ENSEMBL:ENSMUSG00000086090
</td>
<td style="text-align:left;">
MGI:3650531
</td>
<td style="text-align:left;">
lncRNA gene
</td>
<td style="text-align:left;">
predicted gene 12531
</td>
</tr>
<tr>
<td style="text-align:left;">
3
</td>
<td style="text-align:left;">
gene
</td>
<td style="text-align:right;">
37.06309
</td>
<td style="text-align:right;">
37.12193
</td>
<td style="text-align:left;">
positive
</td>
<td style="text-align:left;">
MGI_C57BL6J_103258
</td>
<td style="text-align:left;">
Adad1
</td>
<td style="text-align:left;">
NCBI_Gene:21744,ENSEMBL:ENSMUSG00000027719
</td>
<td style="text-align:left;">
MGI:103258
</td>
<td style="text-align:left;">
protein coding gene
</td>
<td style="text-align:left;">
adenosine deaminase domain containing 1 (testis specific)
</td>
</tr>
<tr>
<td style="text-align:left;">
3
</td>
<td style="text-align:left;">
pseudogene
</td>
<td style="text-align:right;">
37.07756
</td>
<td style="text-align:right;">
37.07891
</td>
<td style="text-align:left;">
positive
</td>
<td style="text-align:left;">
MGI_C57BL6J_3651489
</td>
<td style="text-align:left;">
Gm12582
</td>
<td style="text-align:left;">
NCBI_Gene:652987,ENSEMBL:ENSMUSG00000081705
</td>
<td style="text-align:left;">
MGI:3651489
</td>
<td style="text-align:left;">
pseudogene
</td>
<td style="text-align:left;">
predicted gene 12582
</td>
</tr>
<tr>
<td style="text-align:left;">
3
</td>
<td style="text-align:left;">
pseudogene
</td>
<td style="text-align:right;">
37.10808
</td>
<td style="text-align:right;">
37.10933
</td>
<td style="text-align:left;">
positive
</td>
<td style="text-align:left;">
MGI_C57BL6J_3651485
</td>
<td style="text-align:left;">
Gm12583
</td>
<td style="text-align:left;">
NCBI_Gene:100416060,ENSEMBL:ENSMUSG00000082878
</td>
<td style="text-align:left;">
MGI:3651485
</td>
<td style="text-align:left;">
pseudogene
</td>
<td style="text-align:left;">
predicted gene 12583
</td>
</tr>
<tr>
<td style="text-align:left;">
3
</td>
<td style="text-align:left;">
pseudogene
</td>
<td style="text-align:right;">
37.11673
</td>
<td style="text-align:right;">
37.11784
</td>
<td style="text-align:left;">
positive
</td>
<td style="text-align:left;">
MGI_C57BL6J_3651486
</td>
<td style="text-align:left;">
Gm12584
</td>
<td style="text-align:left;">
ENSEMBL:ENSMUSG00000083857
</td>
<td style="text-align:left;">
MGI:3651486
</td>
<td style="text-align:left;">
pseudogene
</td>
<td style="text-align:left;">
predicted gene 12584
</td>
</tr>
<tr>
<td style="text-align:left;">
3
</td>
<td style="text-align:left;">
gene
</td>
<td style="text-align:right;">
37.12052
</td>
<td style="text-align:right;">
37.12596
</td>
<td style="text-align:left;">
negative
</td>
<td style="text-align:left;">
MGI_C57BL6J_96548
</td>
<td style="text-align:left;">
Il2
</td>
<td style="text-align:left;">
NCBI_Gene:16183,ENSEMBL:ENSMUSG00000027720
</td>
<td style="text-align:left;">
MGI:96548
</td>
<td style="text-align:left;">
protein coding gene
</td>
<td style="text-align:left;">
interleukin 2
</td>
</tr>
<tr>
<td style="text-align:left;">
3
</td>
<td style="text-align:left;">
gene
</td>
<td style="text-align:right;">
37.16972
</td>
<td style="text-align:right;">
37.17665
</td>
<td style="text-align:left;">
negative
</td>
<td style="text-align:left;">
MGI_C57BL6J_5595159
</td>
<td style="text-align:left;">
Gm36000
</td>
<td style="text-align:left;">
NCBI_Gene:102639768
</td>
<td style="text-align:left;">
MGI:5595159
</td>
<td style="text-align:left;">
lncRNA gene
</td>
<td style="text-align:left;">
predicted gene, 36000
</td>
</tr>
<tr>
<td style="text-align:left;">
3
</td>
<td style="text-align:left;">
gene
</td>
<td style="text-align:right;">
37.18218
</td>
<td style="text-align:right;">
37.18860
</td>
<td style="text-align:left;">
positive
</td>
<td style="text-align:left;">
MGI_C57BL6J_3650618
</td>
<td style="text-align:left;">
Gm12532
</td>
<td style="text-align:left;">
NCBI_Gene:102639848,ENSEMBL:ENSMUSG00000084754
</td>
<td style="text-align:left;">
MGI:3650618
</td>
<td style="text-align:left;">
lncRNA gene
</td>
<td style="text-align:left;">
predicted gene 12532
</td>
</tr>
<tr>
<td style="text-align:left;">
3
</td>
<td style="text-align:left;">
gene
</td>
<td style="text-align:right;">
37.22276
</td>
<td style="text-align:right;">
37.23264
</td>
<td style="text-align:left;">
negative
</td>
<td style="text-align:left;">
MGI_C57BL6J_1890474
</td>
<td style="text-align:left;">
Il21
</td>
<td style="text-align:left;">
NCBI_Gene:60505,ENSEMBL:ENSMUSG00000027718
</td>
<td style="text-align:left;">
MGI:1890474
</td>
<td style="text-align:left;">
protein coding gene
</td>
<td style="text-align:left;">
interleukin 21
</td>
</tr>
<tr>
<td style="text-align:left;">
3
</td>
<td style="text-align:left;">
gene
</td>
<td style="text-align:right;">
37.22538
</td>
<td style="text-align:right;">
37.22711
</td>
<td style="text-align:left;">
positive
</td>
<td style="text-align:left;">
MGI_C57BL6J_5595327
</td>
<td style="text-align:left;">
Gm36168
</td>
<td style="text-align:left;">
NCBI_Gene:102639985
</td>
<td style="text-align:left;">
MGI:5595327
</td>
<td style="text-align:left;">
lncRNA gene
</td>
<td style="text-align:left;">
predicted gene, 36168
</td>
</tr>
<tr>
<td style="text-align:left;">
3
</td>
<td style="text-align:left;">
gene
</td>
<td style="text-align:right;">
37.23133
</td>
<td style="text-align:right;">
37.23247
</td>
<td style="text-align:left;">
positive
</td>
<td style="text-align:left;">
MGI_C57BL6J_3650303
</td>
<td style="text-align:left;">
Gm12534
</td>
<td style="text-align:left;">
ENSEMBL:ENSMUSG00000086517
</td>
<td style="text-align:left;">
MGI:3650303
</td>
<td style="text-align:left;">
lncRNA gene
</td>
<td style="text-align:left;">
predicted gene 12534
</td>
</tr>
<tr>
<td style="text-align:left;">
3
</td>
<td style="text-align:left;">
gene
</td>
<td style="text-align:right;">
37.23287
</td>
<td style="text-align:right;">
37.23334
</td>
<td style="text-align:left;">
positive
</td>
<td style="text-align:left;">
MGI_C57BL6J_5663959
</td>
<td style="text-align:left;">
Gm43822
</td>
<td style="text-align:left;">
ENSEMBL:ENSMUSG00000104998
</td>
<td style="text-align:left;">
MGI:5663959
</td>
<td style="text-align:left;">
unclassified gene
</td>
<td style="text-align:left;">
predicted gene 43822
</td>
</tr>
<tr>
<td style="text-align:left;">
3
</td>
<td style="text-align:left;">
gene
</td>
<td style="text-align:right;">
37.25678
</td>
<td style="text-align:right;">
37.25689
</td>
<td style="text-align:left;">
negative
</td>
<td style="text-align:left;">
MGI_C57BL6J_5453946
</td>
<td style="text-align:left;">
Gm24169
</td>
<td style="text-align:left;">
ENSEMBL:ENSMUSG00000094007
</td>
<td style="text-align:left;">
MGI:5453946
</td>
<td style="text-align:left;">
miRNA gene
</td>
<td style="text-align:left;">
predicted gene, 24169
</td>
</tr>
<tr>
<td style="text-align:left;">
3
</td>
<td style="text-align:left;">
gene
</td>
<td style="text-align:right;">
37.30775
</td>
<td style="text-align:right;">
37.31271
</td>
<td style="text-align:left;">
negative
</td>
<td style="text-align:left;">
MGI_C57BL6J_2677454
</td>
<td style="text-align:left;">
Cetn4
</td>
<td style="text-align:left;">
NCBI_Gene:207175,ENSEMBL:ENSMUSG00000045031
</td>
<td style="text-align:left;">
MGI:2677454
</td>
<td style="text-align:left;">
protein coding gene
</td>
<td style="text-align:left;">
centrin 4
</td>
</tr>
<tr>
<td style="text-align:left;">
3
</td>
<td style="text-align:left;">
gene
</td>
<td style="text-align:right;">
37.31249
</td>
<td style="text-align:right;">
37.32145
</td>
<td style="text-align:left;">
positive
</td>
<td style="text-align:left;">
MGI_C57BL6J_2686651
</td>
<td style="text-align:left;">
Bbs12
</td>
<td style="text-align:left;">
NCBI_Gene:241950,ENSEMBL:ENSMUSG00000051444
</td>
<td style="text-align:left;">
MGI:2686651
</td>
<td style="text-align:left;">
protein coding gene
</td>
<td style="text-align:left;">
Bardet-Biedl syndrome 12 (human)
</td>
</tr>
<tr>
<td style="text-align:left;">
3
</td>
<td style="text-align:left;">
gene
</td>
<td style="text-align:right;">
37.31252
</td>
<td style="text-align:right;">
37.40487
</td>
<td style="text-align:left;">
positive
</td>
<td style="text-align:left;">
MGI_C57BL6J_5663576
</td>
<td style="text-align:left;">
Gm43439
</td>
<td style="text-align:left;">
ENSEMBL:ENSMUSG00000106407
</td>
<td style="text-align:left;">
MGI:5663576
</td>
<td style="text-align:left;">
lncRNA gene
</td>
<td style="text-align:left;">
predicted gene 43439
</td>
</tr>
<tr>
<td style="text-align:left;">
3
</td>
<td style="text-align:left;">
pseudogene
</td>
<td style="text-align:right;">
37.31253
</td>
<td style="text-align:right;">
37.32919
</td>
<td style="text-align:left;">
positive
</td>
<td style="text-align:left;">
MGI_C57BL6J_3651849
</td>
<td style="text-align:left;">
Gm12540
</td>
<td style="text-align:left;">
NCBI_Gene:102640067,ENSEMBL:ENSMUSG00000083353
</td>
<td style="text-align:left;">
MGI:3651849
</td>
<td style="text-align:left;">
pseudogene
</td>
<td style="text-align:left;">
predicted gene 12540
</td>
</tr>
<tr>
<td style="text-align:left;">
3
</td>
<td style="text-align:left;">
gene
</td>
<td style="text-align:right;">
37.33329
</td>
<td style="text-align:right;">
37.35103
</td>
<td style="text-align:left;">
negative
</td>
<td style="text-align:left;">
MGI_C57BL6J_3649376
</td>
<td style="text-align:left;">
Fgf2os
</td>
<td style="text-align:left;">
NCBI_Gene:329626,ENSEMBL:ENSMUSG00000054779
</td>
<td style="text-align:left;">
MGI:3649376
</td>
<td style="text-align:left;">
antisense lncRNA gene
</td>
<td style="text-align:left;">
fibroblast growth factor 2, opposite strand
</td>
</tr>
<tr>
<td style="text-align:left;">
3
</td>
<td style="text-align:left;">
gene
</td>
<td style="text-align:right;">
37.33902
</td>
<td style="text-align:right;">
37.34152
</td>
<td style="text-align:left;">
positive
</td>
<td style="text-align:left;">
MGI_C57BL6J_5663625
</td>
<td style="text-align:left;">
Gm43488
</td>
<td style="text-align:left;">
ENSEMBL:ENSMUSG00000105063
</td>
<td style="text-align:left;">
MGI:5663625
</td>
<td style="text-align:left;">
unclassified gene
</td>
<td style="text-align:left;">
predicted gene 43488
</td>
</tr>
<tr>
<td style="text-align:left;">
3
</td>
<td style="text-align:left;">
gene
</td>
<td style="text-align:right;">
37.34835
</td>
<td style="text-align:right;">
37.41011
</td>
<td style="text-align:left;">
positive
</td>
<td style="text-align:left;">
MGI_C57BL6J_95516
</td>
<td style="text-align:left;">
Fgf2
</td>
<td style="text-align:left;">
NCBI_Gene:14173,ENSEMBL:ENSMUSG00000037225
</td>
<td style="text-align:left;">
MGI:95516
</td>
<td style="text-align:left;">
protein coding gene
</td>
<td style="text-align:left;">
fibroblast growth factor 2
</td>
</tr>
<tr>
<td style="text-align:left;">
3
</td>
<td style="text-align:left;">
gene
</td>
<td style="text-align:right;">
37.39221
</td>
<td style="text-align:right;">
37.39510
</td>
<td style="text-align:left;">
negative
</td>
<td style="text-align:left;">
MGI_C57BL6J_5663060
</td>
<td style="text-align:left;">
Gm42923
</td>
<td style="text-align:left;">
ENSEMBL:ENSMUSG00000105134
</td>
<td style="text-align:left;">
MGI:5663060
</td>
<td style="text-align:left;">
unclassified gene
</td>
<td style="text-align:left;">
predicted gene 42923
</td>
</tr>
<tr>
<td style="text-align:left;">
3
</td>
<td style="text-align:left;">
gene
</td>
<td style="text-align:right;">
37.40491
</td>
<td style="text-align:right;">
37.42021
</td>
<td style="text-align:left;">
negative
</td>
<td style="text-align:left;">
MGI_C57BL6J_2387618
</td>
<td style="text-align:left;">
Nudt6
</td>
<td style="text-align:left;">
NCBI_Gene:229228,ENSEMBL:ENSMUSG00000050174
</td>
<td style="text-align:left;">
MGI:2387618
</td>
<td style="text-align:left;">
protein coding gene
</td>
<td style="text-align:left;">
nudix (nucleoside diphosphate linked moiety X)-type motif 6
</td>
</tr>
<tr>
<td style="text-align:left;">
3
</td>
<td style="text-align:left;">
gene
</td>
<td style="text-align:right;">
37.41990
</td>
<td style="text-align:right;">
37.57910
</td>
<td style="text-align:left;">
positive
</td>
<td style="text-align:left;">
MGI_C57BL6J_1927170
</td>
<td style="text-align:left;">
Spata5
</td>
<td style="text-align:left;">
NCBI_Gene:57815,ENSEMBL:ENSMUSG00000027722
</td>
<td style="text-align:left;">
MGI:1927170
</td>
<td style="text-align:left;">
protein coding gene
</td>
<td style="text-align:left;">
spermatogenesis associated 5
</td>
</tr>
<tr>
<td style="text-align:left;">
3
</td>
<td style="text-align:left;">
gene
</td>
<td style="text-align:right;">
37.48883
</td>
<td style="text-align:right;">
37.49479
</td>
<td style="text-align:left;">
positive
</td>
<td style="text-align:left;">
MGI_C57BL6J_5663621
</td>
<td style="text-align:left;">
Gm43484
</td>
<td style="text-align:left;">
ENSEMBL:ENSMUSG00000106166
</td>
<td style="text-align:left;">
MGI:5663621
</td>
<td style="text-align:left;">
unclassified gene
</td>
<td style="text-align:left;">
predicted gene 43484
</td>
</tr>
<tr>
<td style="text-align:left;">
3
</td>
<td style="text-align:left;">
gene
</td>
<td style="text-align:right;">
37.54188
</td>
<td style="text-align:right;">
37.60239
</td>
<td style="text-align:left;">
negative
</td>
<td style="text-align:left;">
MGI_C57BL6J_5595571
</td>
<td style="text-align:left;">
Gm36412
</td>
<td style="text-align:left;">
NCBI_Gene:102640314
</td>
<td style="text-align:left;">
MGI:5595571
</td>
<td style="text-align:left;">
lncRNA gene
</td>
<td style="text-align:left;">
predicted gene, 36412
</td>
</tr>
<tr>
<td style="text-align:left;">
3
</td>
<td style="text-align:left;">
pseudogene
</td>
<td style="text-align:right;">
37.55618
</td>
<td style="text-align:right;">
37.55650
</td>
<td style="text-align:left;">
negative
</td>
<td style="text-align:left;">
MGI_C57BL6J_3652207
</td>
<td style="text-align:left;">
Gm12564
</td>
<td style="text-align:left;">
ENSEMBL:ENSMUSG00000080723
</td>
<td style="text-align:left;">
MGI:3652207
</td>
<td style="text-align:left;">
pseudogene
</td>
<td style="text-align:left;">
predicted gene 12564
</td>
</tr>
<tr>
<td style="text-align:left;">
3
</td>
<td style="text-align:left;">
pseudogene
</td>
<td style="text-align:right;">
37.59942
</td>
<td style="text-align:right;">
37.60007
</td>
<td style="text-align:left;">
negative
</td>
<td style="text-align:left;">
MGI_C57BL6J_3652206
</td>
<td style="text-align:left;">
Gm12565
</td>
<td style="text-align:left;">
NCBI_Gene:665778,ENSEMBL:ENSMUSG00000081093
</td>
<td style="text-align:left;">
MGI:3652206
</td>
<td style="text-align:left;">
pseudogene
</td>
<td style="text-align:left;">
predicted gene 12565
</td>
</tr>
<tr>
<td style="text-align:left;">
3
</td>
<td style="text-align:left;">
gene
</td>
<td style="text-align:right;">
37.60989
</td>
<td style="text-align:right;">
37.61528
</td>
<td style="text-align:left;">
positive
</td>
<td style="text-align:left;">
MGI_C57BL6J_1923197
</td>
<td style="text-align:left;">
4930594O21Rik
</td>
<td style="text-align:left;">
NCBI_Gene:75947,ENSEMBL:ENSMUSG00000085677
</td>
<td style="text-align:left;">
MGI:1923197
</td>
<td style="text-align:left;">
lncRNA gene
</td>
<td style="text-align:left;">
RIKEN cDNA 4930594O21 gene
</td>
</tr>
<tr>
<td style="text-align:left;">
3
</td>
<td style="text-align:left;">
gene
</td>
<td style="text-align:right;">
37.63858
</td>
<td style="text-align:right;">
37.64066
</td>
<td style="text-align:left;">
negative
</td>
<td style="text-align:left;">
MGI_C57BL6J_5663059
</td>
<td style="text-align:left;">
Gm42922
</td>
<td style="text-align:left;">
ENSEMBL:ENSMUSG00000104605
</td>
<td style="text-align:left;">
MGI:5663059
</td>
<td style="text-align:left;">
unclassified gene
</td>
<td style="text-align:left;">
predicted gene 42922
</td>
</tr>
<tr>
<td style="text-align:left;">
3
</td>
<td style="text-align:left;">
gene
</td>
<td style="text-align:right;">
37.63995
</td>
<td style="text-align:right;">
37.64460
</td>
<td style="text-align:left;">
positive
</td>
<td style="text-align:left;">
MGI_C57BL6J_1345139
</td>
<td style="text-align:left;">
Spry1
</td>
<td style="text-align:left;">
NCBI_Gene:24063,ENSEMBL:ENSMUSG00000037211
</td>
<td style="text-align:left;">
MGI:1345139
</td>
<td style="text-align:left;">
protein coding gene
</td>
<td style="text-align:left;">
sprouty RTK signaling antagonist 1
</td>
</tr>
<tr>
<td style="text-align:left;">
3
</td>
<td style="text-align:left;">
gene
</td>
<td style="text-align:right;">
37.64461
</td>
<td style="text-align:right;">
37.64795
</td>
<td style="text-align:left;">
positive
</td>
<td style="text-align:left;">
MGI_C57BL6J_5663957
</td>
<td style="text-align:left;">
Gm43820
</td>
<td style="text-align:left;">
ENSEMBL:ENSMUSG00000106062
</td>
<td style="text-align:left;">
MGI:5663957
</td>
<td style="text-align:left;">
unclassified gene
</td>
<td style="text-align:left;">
predicted gene 43820
</td>
</tr>
<tr>
<td style="text-align:left;">
3
</td>
<td style="text-align:left;">
gene
</td>
<td style="text-align:right;">
37.65268
</td>
<td style="text-align:right;">
37.65652
</td>
<td style="text-align:left;">
negative
</td>
<td style="text-align:left;">
MGI_C57BL6J_5826436
</td>
<td style="text-align:left;">
Gm46799
</td>
<td style="text-align:left;">
NCBI_Gene:108168864
</td>
<td style="text-align:left;">
MGI:5826436
</td>
<td style="text-align:left;">
lncRNA gene
</td>
<td style="text-align:left;">
predicted gene, 46799
</td>
</tr>
<tr>
<td style="text-align:left;">
3
</td>
<td style="text-align:left;">
gene
</td>
<td style="text-align:right;">
37.69046
</td>
<td style="text-align:right;">
37.69486
</td>
<td style="text-align:left;">
negative
</td>
<td style="text-align:left;">
MGI_C57BL6J_5625096
</td>
<td style="text-align:left;">
Gm42211
</td>
<td style="text-align:left;">
NCBI_Gene:105247029
</td>
<td style="text-align:left;">
MGI:5625096
</td>
<td style="text-align:left;">
lncRNA gene
</td>
<td style="text-align:left;">
predicted gene, 42211
</td>
</tr>
<tr>
<td style="text-align:left;">
3
</td>
<td style="text-align:left;">
gene
</td>
<td style="text-align:right;">
37.70802
</td>
<td style="text-align:right;">
37.71121
</td>
<td style="text-align:left;">
positive
</td>
<td style="text-align:left;">
MGI_C57BL6J_5663551
</td>
<td style="text-align:left;">
Gm43414
</td>
<td style="text-align:left;">
ENSEMBL:ENSMUSG00000104935
</td>
<td style="text-align:left;">
MGI:5663551
</td>
<td style="text-align:left;">
unclassified gene
</td>
<td style="text-align:left;">
predicted gene 43414
</td>
</tr>
<tr>
<td style="text-align:left;">
3
</td>
<td style="text-align:left;">
gene
</td>
<td style="text-align:right;">
37.71419
</td>
<td style="text-align:right;">
37.72449
</td>
<td style="text-align:left;">
negative
</td>
<td style="text-align:left;">
MGI_C57BL6J_3646006
</td>
<td style="text-align:left;">
Gm5148
</td>
<td style="text-align:left;">
NCBI_Gene:381438,ENSEMBL:ENSMUSG00000058174
</td>
<td style="text-align:left;">
MGI:3646006
</td>
<td style="text-align:left;">
protein coding gene
</td>
<td style="text-align:left;">
predicted gene 5148
</td>
</tr>
<tr>
<td style="text-align:left;">
3
</td>
<td style="text-align:left;">
pseudogene
</td>
<td style="text-align:right;">
37.71471
</td>
<td style="text-align:right;">
37.71522
</td>
<td style="text-align:left;">
positive
</td>
<td style="text-align:left;">
MGI_C57BL6J_3705429
</td>
<td style="text-align:left;">
Rps23-ps1
</td>
<td style="text-align:left;">
NCBI_Gene:100038875,ENSEMBL:ENSMUSG00000100755
</td>
<td style="text-align:left;">
MGI:3705429
</td>
<td style="text-align:left;">
pseudogene
</td>
<td style="text-align:left;">
ribosomal protein S23, pseudogene 1
</td>
</tr>
<tr>
<td style="text-align:left;">
3
</td>
<td style="text-align:left;">
gene
</td>
<td style="text-align:right;">
37.72423
</td>
<td style="text-align:right;">
37.74132
</td>
<td style="text-align:left;">
positive
</td>
<td style="text-align:left;">
MGI_C57BL6J_5595779
</td>
<td style="text-align:left;">
Gm36620
</td>
<td style="text-align:left;">
NCBI_Gene:102640592
</td>
<td style="text-align:left;">
MGI:5595779
</td>
<td style="text-align:left;">
lncRNA gene
</td>
<td style="text-align:left;">
predicted gene, 36620
</td>
</tr>
<tr>
<td style="text-align:left;">
3
</td>
<td style="text-align:left;">
pseudogene
</td>
<td style="text-align:right;">
37.73468
</td>
<td style="text-align:right;">
37.73514
</td>
<td style="text-align:left;">
positive
</td>
<td style="text-align:left;">
MGI_C57BL6J_3643738
</td>
<td style="text-align:left;">
Rpl31-ps15
</td>
<td style="text-align:left;">
NCBI_Gene:626367,ENSEMBL:ENSMUSG00000097304
</td>
<td style="text-align:left;">
MGI:3643738
</td>
<td style="text-align:left;">
pseudogene
</td>
<td style="text-align:left;">
ribosomal protein L31, pseudogene 15
</td>
</tr>
<tr>
<td style="text-align:left;">
3
</td>
<td style="text-align:left;">
gene
</td>
<td style="text-align:right;">
37.75320
</td>
<td style="text-align:right;">
37.77440
</td>
<td style="text-align:left;">
positive
</td>
<td style="text-align:left;">
MGI_C57BL6J_5625095
</td>
<td style="text-align:left;">
Gm42210
</td>
<td style="text-align:left;">
NCBI_Gene:105247028
</td>
<td style="text-align:left;">
MGI:5625095
</td>
<td style="text-align:left;">
lncRNA gene
</td>
<td style="text-align:left;">
predicted gene, 42210
</td>
</tr>
<tr>
<td style="text-align:left;">
3
</td>
<td style="text-align:left;">
gene
</td>
<td style="text-align:right;">
37.76104
</td>
<td style="text-align:right;">
37.76730
</td>
<td style="text-align:left;">
positive
</td>
<td style="text-align:left;">
MGI_C57BL6J_5663058
</td>
<td style="text-align:left;">
Gm42921
</td>
<td style="text-align:left;">
ENSEMBL:ENSMUSG00000106441
</td>
<td style="text-align:left;">
MGI:5663058
</td>
<td style="text-align:left;">
lncRNA gene
</td>
<td style="text-align:left;">
predicted gene 42921
</td>
</tr>
<tr>
<td style="text-align:left;">
3
</td>
<td style="text-align:left;">
gene
</td>
<td style="text-align:right;">
37.80254
</td>
<td style="text-align:right;">
37.80265
</td>
<td style="text-align:left;">
negative
</td>
<td style="text-align:left;">
MGI_C57BL6J_5456181
</td>
<td style="text-align:left;">
Gm26404
</td>
<td style="text-align:left;">
ENSEMBL:ENSMUSG00000089123
</td>
<td style="text-align:left;">
MGI:5456181
</td>
<td style="text-align:left;">
snoRNA gene
</td>
<td style="text-align:left;">
predicted gene, 26404
</td>
</tr>
<tr>
<td style="text-align:left;">
3
</td>
<td style="text-align:left;">
gene
</td>
<td style="text-align:right;">
37.82822
</td>
<td style="text-align:right;">
37.84829
</td>
<td style="text-align:left;">
positive
</td>
<td style="text-align:left;">
MGI_C57BL6J_1925294
</td>
<td style="text-align:left;">
4930556A17Rik
</td>
<td style="text-align:left;">
NCBI_Gene:102640520
</td>
<td style="text-align:left;">
MGI:1925294
</td>
<td style="text-align:left;">
lncRNA gene
</td>
<td style="text-align:left;">
RIKEN cDNA 4930556A17 gene
</td>
</tr>
<tr>
<td style="text-align:left;">
3
</td>
<td style="text-align:left;">
gene
</td>
<td style="text-align:right;">
37.83847
</td>
<td style="text-align:right;">
37.83957
</td>
<td style="text-align:left;">
positive
</td>
<td style="text-align:left;">
MGI_C57BL6J_5662889
</td>
<td style="text-align:left;">
Gm42752
</td>
<td style="text-align:left;">
ENSEMBL:ENSMUSG00000106621
</td>
<td style="text-align:left;">
MGI:5662889
</td>
<td style="text-align:left;">
unclassified gene
</td>
<td style="text-align:left;">
predicted gene 42752
</td>
</tr>
<tr>
<td style="text-align:left;">
3
</td>
<td style="text-align:left;">
gene
</td>
<td style="text-align:right;">
37.84180
</td>
<td style="text-align:right;">
37.84307
</td>
<td style="text-align:left;">
positive
</td>
<td style="text-align:left;">
MGI_C57BL6J_5662890
</td>
<td style="text-align:left;">
Gm42753
</td>
<td style="text-align:left;">
ENSEMBL:ENSMUSG00000105252
</td>
<td style="text-align:left;">
MGI:5662890
</td>
<td style="text-align:left;">
unclassified gene
</td>
<td style="text-align:left;">
predicted gene 42753
</td>
</tr>
<tr>
<td style="text-align:left;">
3
</td>
<td style="text-align:left;">
gene
</td>
<td style="text-align:right;">
37.89881
</td>
<td style="text-align:right;">
37.99273
</td>
<td style="text-align:left;">
positive
</td>
<td style="text-align:left;">
MGI_C57BL6J_5434111
</td>
<td style="text-align:left;">
Gm20755
</td>
<td style="text-align:left;">
NCBI_Gene:626410,ENSEMBL:ENSMUSG00000106461
</td>
<td style="text-align:left;">
MGI:5434111
</td>
<td style="text-align:left;">
lncRNA gene
</td>
<td style="text-align:left;">
predicted gene, 20755
</td>
</tr>
<tr>
<td style="text-align:left;">
3
</td>
<td style="text-align:left;">
pseudogene
</td>
<td style="text-align:right;">
37.95961
</td>
<td style="text-align:right;">
37.96009
</td>
<td style="text-align:left;">
negative
</td>
<td style="text-align:left;">
MGI_C57BL6J_3643378
</td>
<td style="text-align:left;">
Gm7799
</td>
<td style="text-align:left;">
ENSEMBL:ENSMUSG00000087457
</td>
<td style="text-align:left;">
MGI:3643378
</td>
<td style="text-align:left;">
pseudogene
</td>
<td style="text-align:left;">
predicted gene 7799
</td>
</tr>
<tr>
<td style="text-align:left;">
3
</td>
<td style="text-align:left;">
gene
</td>
<td style="text-align:right;">
37.98279
</td>
<td style="text-align:right;">
37.98335
</td>
<td style="text-align:left;">
positive
</td>
<td style="text-align:left;">
MGI_C57BL6J_5662888
</td>
<td style="text-align:left;">
Gm42751
</td>
<td style="text-align:left;">
ENSEMBL:ENSMUSG00000105099
</td>
<td style="text-align:left;">
MGI:5662888
</td>
<td style="text-align:left;">
unclassified gene
</td>
<td style="text-align:left;">
predicted gene 42751
</td>
</tr>
<tr>
<td style="text-align:left;">
3
</td>
<td style="text-align:left;">
gene
</td>
<td style="text-align:right;">
37.99511
</td>
<td style="text-align:right;">
37.99524
</td>
<td style="text-align:left;">
negative
</td>
<td style="text-align:left;">
MGI_C57BL6J_5452676
</td>
<td style="text-align:left;">
Gm22899
</td>
<td style="text-align:left;">
ENSEMBL:ENSMUSG00000077728
</td>
<td style="text-align:left;">
MGI:5452676
</td>
<td style="text-align:left;">
snoRNA gene
</td>
<td style="text-align:left;">
predicted gene, 22899
</td>
</tr>
<tr>
<td style="text-align:left;">
3
</td>
<td style="text-align:left;">
gene
</td>
<td style="text-align:right;">
38.09463
</td>
<td style="text-align:right;">
38.13019
</td>
<td style="text-align:left;">
negative
</td>
<td style="text-align:left;">
MGI_C57BL6J_5663958
</td>
<td style="text-align:left;">
Gm43821
</td>
<td style="text-align:left;">
ENSEMBL:ENSMUSG00000105219
</td>
<td style="text-align:left;">
MGI:5663958
</td>
<td style="text-align:left;">
lncRNA gene
</td>
<td style="text-align:left;">
predicted gene 43821
</td>
</tr>
<tr>
<td style="text-align:left;">
3
</td>
<td style="text-align:left;">
pseudogene
</td>
<td style="text-align:right;">
38.10123
</td>
<td style="text-align:right;">
38.10798
</td>
<td style="text-align:left;">
negative
</td>
<td style="text-align:left;">
MGI_C57BL6J_3643358
</td>
<td style="text-align:left;">
Rpl21-ps10
</td>
<td style="text-align:left;">
NCBI_Gene:665821,ENSEMBL:ENSMUSG00000105359
</td>
<td style="text-align:left;">
MGI:3643358
</td>
<td style="text-align:left;">
pseudogene
</td>
<td style="text-align:left;">
ribosomal protein L21, pseudogene 10
</td>
</tr>
<tr>
<td style="text-align:left;">
3
</td>
<td style="text-align:left;">
gene
</td>
<td style="text-align:right;">
38.15191
</td>
<td style="text-align:right;">
38.15915
</td>
<td style="text-align:left;">
positive
</td>
<td style="text-align:left;">
MGI_C57BL6J_5595839
</td>
<td style="text-align:left;">
Gm36680
</td>
<td style="text-align:left;">
NCBI_Gene:102640666
</td>
<td style="text-align:left;">
MGI:5595839
</td>
<td style="text-align:left;">
lncRNA gene
</td>
<td style="text-align:left;">
predicted gene, 36680
</td>
</tr>
<tr>
<td style="text-align:left;">
3
</td>
<td style="text-align:left;">
gene
</td>
<td style="text-align:right;">
38.20315
</td>
<td style="text-align:right;">
38.20812
</td>
<td style="text-align:left;">
negative
</td>
<td style="text-align:left;">
MGI_C57BL6J_1918592
</td>
<td style="text-align:left;">
5430434I15Rik
</td>
<td style="text-align:left;">
NCBI_Gene:71342,ENSEMBL:ENSMUSG00000106002
</td>
<td style="text-align:left;">
MGI:1918592
</td>
<td style="text-align:left;">
lncRNA gene
</td>
<td style="text-align:left;">
RIKEN cDNA 5430434I15 gene
</td>
</tr>
<tr>
<td style="text-align:left;">
3
</td>
<td style="text-align:left;">
pseudogene
</td>
<td style="text-align:right;">
38.22345
</td>
<td style="text-align:right;">
38.22382
</td>
<td style="text-align:left;">
positive
</td>
<td style="text-align:left;">
MGI_C57BL6J_3781143
</td>
<td style="text-align:left;">
Gm2965
</td>
<td style="text-align:left;">
NCBI_Gene:100040785,ENSEMBL:ENSMUSG00000105943
</td>
<td style="text-align:left;">
MGI:3781143
</td>
<td style="text-align:left;">
pseudogene
</td>
<td style="text-align:left;">
predicted gene 2965
</td>
</tr>
<tr>
<td style="text-align:left;">
3
</td>
<td style="text-align:left;">
pseudogene
</td>
<td style="text-align:right;">
38.25402
</td>
<td style="text-align:right;">
38.25480
</td>
<td style="text-align:left;">
positive
</td>
<td style="text-align:left;">
MGI_C57BL6J_5010140
</td>
<td style="text-align:left;">
Gm17955
</td>
<td style="text-align:left;">
NCBI_Gene:100416169
</td>
<td style="text-align:left;">
MGI:5010140
</td>
<td style="text-align:left;">
pseudogene
</td>
<td style="text-align:left;">
predicted gene, 17955
</td>
</tr>
<tr>
<td style="text-align:left;">
3
</td>
<td style="text-align:left;">
gene
</td>
<td style="text-align:right;">
38.26784
</td>
<td style="text-align:right;">
38.26808
</td>
<td style="text-align:left;">
negative
</td>
<td style="text-align:left;">
MGI_C57BL6J_5455351
</td>
<td style="text-align:left;">
Gm25574
</td>
<td style="text-align:left;">
ENSEMBL:ENSMUSG00000092986
</td>
<td style="text-align:left;">
MGI:5455351
</td>
<td style="text-align:left;">
unclassified non-coding RNA gene
</td>
<td style="text-align:left;">
predicted gene, 25574
</td>
</tr>
<tr>
<td style="text-align:left;">
3
</td>
<td style="text-align:left;">
gene
</td>
<td style="text-align:right;">
38.36615
</td>
<td style="text-align:right;">
38.36888
</td>
<td style="text-align:left;">
negative
</td>
<td style="text-align:left;">
MGI_C57BL6J_5625098
</td>
<td style="text-align:left;">
Gm42213
</td>
<td style="text-align:left;">
NCBI_Gene:105247031
</td>
<td style="text-align:left;">
MGI:5625098
</td>
<td style="text-align:left;">
lncRNA gene
</td>
<td style="text-align:left;">
predicted gene, 42213
</td>
</tr>
<tr>
<td style="text-align:left;">
3
</td>
<td style="text-align:left;">
gene
</td>
<td style="text-align:right;">
38.36940
</td>
<td style="text-align:right;">
38.37364
</td>
<td style="text-align:left;">
positive
</td>
<td style="text-align:left;">
MGI_C57BL6J_5663057
</td>
<td style="text-align:left;">
Gm42920
</td>
<td style="text-align:left;">
ENSEMBL:ENSMUSG00000106008
</td>
<td style="text-align:left;">
MGI:5663057
</td>
<td style="text-align:left;">
lncRNA gene
</td>
<td style="text-align:left;">
predicted gene 42920
</td>
</tr>
<tr>
<td style="text-align:left;">
3
</td>
<td style="text-align:left;">
gene
</td>
<td style="text-align:right;">
38.41897
</td>
<td style="text-align:right;">
38.42015
</td>
<td style="text-align:left;">
negative
</td>
<td style="text-align:left;">
MGI_C57BL6J_1917958
</td>
<td style="text-align:left;">
3830422I06Rik
</td>
<td style="text-align:left;">
ENSEMBL:ENSMUSG00000106485
</td>
<td style="text-align:left;">
MGI:1917958
</td>
<td style="text-align:left;">
unclassified gene
</td>
<td style="text-align:left;">
RIKEN cDNA 3830422I06 gene
</td>
</tr>
<tr>
<td style="text-align:left;">
3
</td>
<td style="text-align:left;">
gene
</td>
<td style="text-align:right;">
38.42655
</td>
<td style="text-align:right;">
38.44387
</td>
<td style="text-align:left;">
negative
</td>
<td style="text-align:left;">
MGI_C57BL6J_5595982
</td>
<td style="text-align:left;">
Gm36823
</td>
<td style="text-align:left;">
NCBI_Gene:102640855,ENSEMBL:ENSMUSG00000105516
</td>
<td style="text-align:left;">
MGI:5595982
</td>
<td style="text-align:left;">
lncRNA gene
</td>
<td style="text-align:left;">
predicted gene, 36823
</td>
</tr>
<tr>
<td style="text-align:left;">
3
</td>
<td style="text-align:left;">
pseudogene
</td>
<td style="text-align:right;">
38.43895
</td>
<td style="text-align:right;">
38.43984
</td>
<td style="text-align:left;">
negative
</td>
<td style="text-align:left;">
MGI_C57BL6J_3646285
</td>
<td style="text-align:left;">
Gm7815
</td>
<td style="text-align:left;">
NCBI_Gene:665838,ENSEMBL:ENSMUSG00000106201
</td>
<td style="text-align:left;">
MGI:3646285
</td>
<td style="text-align:left;">
pseudogene
</td>
<td style="text-align:left;">
predicted gene 7815
</td>
</tr>
<tr>
<td style="text-align:left;">
3
</td>
<td style="text-align:left;">
gene
</td>
<td style="text-align:right;">
38.44926
</td>
<td style="text-align:right;">
38.48486
</td>
<td style="text-align:left;">
negative
</td>
<td style="text-align:left;">
MGI_C57BL6J_2139777
</td>
<td style="text-align:left;">
Ankrd50
</td>
<td style="text-align:left;">
NCBI_Gene:99696,ENSEMBL:ENSMUSG00000044864
</td>
<td style="text-align:left;">
MGI:2139777
</td>
<td style="text-align:left;">
protein coding gene
</td>
<td style="text-align:left;">
ankyrin repeat domain 50
</td>
</tr>
<tr>
<td style="text-align:left;">
3
</td>
<td style="text-align:left;">
pseudogene
</td>
<td style="text-align:right;">
38.55797
</td>
<td style="text-align:right;">
38.55886
</td>
<td style="text-align:left;">
negative
</td>
<td style="text-align:left;">
MGI_C57BL6J_3779766
</td>
<td style="text-align:left;">
Gm7824
</td>
<td style="text-align:left;">
NCBI_Gene:665855,ENSEMBL:ENSMUSG00000105360
</td>
<td style="text-align:left;">
MGI:3779766
</td>
<td style="text-align:left;">
pseudogene
</td>
<td style="text-align:left;">
predicted gene 7824
</td>
</tr>
<tr>
<td style="text-align:left;">
3
</td>
<td style="text-align:left;">
gene
</td>
<td style="text-align:right;">
38.55811
</td>
<td style="text-align:right;">
38.55821
</td>
<td style="text-align:left;">
negative
</td>
<td style="text-align:left;">
MGI_C57BL6J_5454239
</td>
<td style="text-align:left;">
Gm24462
</td>
<td style="text-align:left;">
ENSEMBL:ENSMUSG00000076385
</td>
<td style="text-align:left;">
MGI:5454239
</td>
<td style="text-align:left;">
miRNA gene
</td>
<td style="text-align:left;">
predicted gene, 24462
</td>
</tr>
<tr>
<td style="text-align:left;">
3
</td>
<td style="text-align:left;">
gene
</td>
<td style="text-align:right;">
38.61425
</td>
<td style="text-align:right;">
38.61556
</td>
<td style="text-align:left;">
positive
</td>
<td style="text-align:left;">
MGI_C57BL6J_5663675
</td>
<td style="text-align:left;">
Gm43538
</td>
<td style="text-align:left;">
ENSEMBL:ENSMUSG00000106168
</td>
<td style="text-align:left;">
MGI:5663675
</td>
<td style="text-align:left;">
lncRNA gene
</td>
<td style="text-align:left;">
predicted gene 43538
</td>
</tr>
<tr>
<td style="text-align:left;">
3
</td>
<td style="text-align:left;">
gene
</td>
<td style="text-align:right;">
38.66361
</td>
<td style="text-align:right;">
38.70922
</td>
<td style="text-align:left;">
positive
</td>
<td style="text-align:left;">
MGI_C57BL6J_5589038
</td>
<td style="text-align:left;">
Gm29879
</td>
<td style="text-align:left;">
NCBI_Gene:102631576
</td>
<td style="text-align:left;">
MGI:5589038
</td>
<td style="text-align:left;">
lncRNA gene
</td>
<td style="text-align:left;">
predicted gene, 29879
</td>
</tr>
<tr>
<td style="text-align:left;">
3
</td>
<td style="text-align:left;">
gene
</td>
<td style="text-align:right;">
38.76048
</td>
<td style="text-align:right;">
38.76195
</td>
<td style="text-align:left;">
negative
</td>
<td style="text-align:left;">
MGI_C57BL6J_5589118
</td>
<td style="text-align:left;">
Gm29959
</td>
<td style="text-align:left;">
NCBI_Gene:102631683
</td>
<td style="text-align:left;">
MGI:5589118
</td>
<td style="text-align:left;">
lncRNA gene
</td>
<td style="text-align:left;">
predicted gene, 29959
</td>
</tr>
<tr>
<td style="text-align:left;">
3
</td>
<td style="text-align:left;">
gene
</td>
<td style="text-align:right;">
38.87457
</td>
<td style="text-align:right;">
38.88826
</td>
<td style="text-align:left;">
negative
</td>
<td style="text-align:left;">
MGI_C57BL6J_2441892
</td>
<td style="text-align:left;">
C230034O21Rik
</td>
<td style="text-align:left;">
NCBI_Gene:102632538,ENSEMBL:ENSMUSG00000086181
</td>
<td style="text-align:left;">
MGI:2441892
</td>
<td style="text-align:left;">
lncRNA gene
</td>
<td style="text-align:left;">
RIKEN cDNA C230034O21 gene
</td>
</tr>
<tr>
<td style="text-align:left;">
3
</td>
<td style="text-align:left;">
gene
</td>
<td style="text-align:right;">
38.88563
</td>
<td style="text-align:right;">
39.01199
</td>
<td style="text-align:left;">
positive
</td>
<td style="text-align:left;">
MGI_C57BL6J_3045256
</td>
<td style="text-align:left;">
Fat4
</td>
<td style="text-align:left;">
NCBI_Gene:329628,ENSEMBL:ENSMUSG00000046743
</td>
<td style="text-align:left;">
MGI:3045256
</td>
<td style="text-align:left;">
protein coding gene
</td>
<td style="text-align:left;">
FAT atypical cadherin 4
</td>
</tr>
<tr>
<td style="text-align:left;">
3
</td>
<td style="text-align:left;">
gene
</td>
<td style="text-align:right;">
39.00860
</td>
<td style="text-align:right;">
39.00968
</td>
<td style="text-align:left;">
negative
</td>
<td style="text-align:left;">
MGI_C57BL6J_5663676
</td>
<td style="text-align:left;">
Gm43539
</td>
<td style="text-align:left;">
ENSEMBL:ENSMUSG00000105910
</td>
<td style="text-align:left;">
MGI:5663676
</td>
<td style="text-align:left;">
lncRNA gene
</td>
<td style="text-align:left;">
predicted gene 43539
</td>
</tr>
<tr>
<td style="text-align:left;">
3
</td>
<td style="text-align:left;">
pseudogene
</td>
<td style="text-align:right;">
39.12853
</td>
<td style="text-align:right;">
39.12891
</td>
<td style="text-align:left;">
negative
</td>
<td style="text-align:left;">
MGI_C57BL6J_5663445
</td>
<td style="text-align:left;">
Gm43308
</td>
<td style="text-align:left;">
ENSEMBL:ENSMUSG00000104792
</td>
<td style="text-align:left;">
MGI:5663445
</td>
<td style="text-align:left;">
pseudogene
</td>
<td style="text-align:left;">
predicted gene 43308
</td>
</tr>
<tr>
<td style="text-align:left;">
3
</td>
<td style="text-align:left;">
gene
</td>
<td style="text-align:right;">
39.15083
</td>
<td style="text-align:right;">
39.15967
</td>
<td style="text-align:left;">
negative
</td>
<td style="text-align:left;">
MGI_C57BL6J_5589170
</td>
<td style="text-align:left;">
Gm30011
</td>
<td style="text-align:left;">
NCBI_Gene:102631750
</td>
<td style="text-align:left;">
MGI:5589170
</td>
<td style="text-align:left;">
lncRNA gene
</td>
<td style="text-align:left;">
predicted gene, 30011
</td>
</tr>
<tr>
<td style="text-align:left;">
3
</td>
<td style="text-align:left;">
pseudogene
</td>
<td style="text-align:right;">
39.21756
</td>
<td style="text-align:right;">
39.21785
</td>
<td style="text-align:left;">
negative
</td>
<td style="text-align:left;">
MGI_C57BL6J_5663145
</td>
<td style="text-align:left;">
Gm43008
</td>
<td style="text-align:left;">
ENSEMBL:ENSMUSG00000107333
</td>
<td style="text-align:left;">
MGI:5663145
</td>
<td style="text-align:left;">
pseudogene
</td>
<td style="text-align:left;">
predicted gene 43008
</td>
</tr>
<tr>
<td style="text-align:left;">
3
</td>
<td style="text-align:left;">
gene
</td>
<td style="text-align:right;">
39.27713
</td>
<td style="text-align:right;">
39.30593
</td>
<td style="text-align:left;">
positive
</td>
<td style="text-align:left;">
MGI_C57BL6J_5625099
</td>
<td style="text-align:left;">
Gm42214
</td>
<td style="text-align:left;">
NCBI_Gene:105247032
</td>
<td style="text-align:left;">
MGI:5625099
</td>
<td style="text-align:left;">
lncRNA gene
</td>
<td style="text-align:left;">
predicted gene, 42214
</td>
</tr>
<tr>
<td style="text-align:left;">
3
</td>
<td style="text-align:left;">
pseudogene
</td>
<td style="text-align:right;">
39.35737
</td>
<td style="text-align:right;">
39.35913
</td>
<td style="text-align:left;">
negative
</td>
<td style="text-align:left;">
MGI_C57BL6J_3704215
</td>
<td style="text-align:left;">
Gm9845
</td>
<td style="text-align:left;">
NCBI_Gene:100040853,ENSEMBL:ENSMUSG00000050533
</td>
<td style="text-align:left;">
MGI:3704215
</td>
<td style="text-align:left;">
pseudogene
</td>
<td style="text-align:left;">
predicted pseudogene 9845
</td>
</tr>
<tr>
<td style="text-align:left;">
3
</td>
<td style="text-align:left;">
pseudogene
</td>
<td style="text-align:right;">
39.44389
</td>
<td style="text-align:right;">
39.44435
</td>
<td style="text-align:left;">
positive
</td>
<td style="text-align:left;">
MGI_C57BL6J_5295673
</td>
<td style="text-align:left;">
Gm20566
</td>
<td style="text-align:left;">
NCBI_Gene:100047050
</td>
<td style="text-align:left;">
MGI:5295673
</td>
<td style="text-align:left;">
pseudogene
</td>
<td style="text-align:left;">
predicted gene, 20566
</td>
</tr>
<tr>
<td style="text-align:left;">
3
</td>
<td style="text-align:left;">
gene
</td>
<td style="text-align:right;">
39.46078
</td>
<td style="text-align:right;">
39.59865
</td>
<td style="text-align:left;">
positive
</td>
<td style="text-align:left;">
MGI_C57BL6J_5589222
</td>
<td style="text-align:left;">
Gm30063
</td>
<td style="text-align:left;">
NCBI_Gene:102631823
</td>
<td style="text-align:left;">
MGI:5589222
</td>
<td style="text-align:left;">
lncRNA gene
</td>
<td style="text-align:left;">
predicted gene, 30063
</td>
</tr>
<tr>
<td style="text-align:left;">
3
</td>
<td style="text-align:left;">
gene
</td>
<td style="text-align:right;">
39.52552
</td>
<td style="text-align:right;">
39.53415
</td>
<td style="text-align:left;">
negative
</td>
<td style="text-align:left;">
MGI_C57BL6J_5625100
</td>
<td style="text-align:left;">
Gm42215
</td>
<td style="text-align:left;">
NCBI_Gene:105247033
</td>
<td style="text-align:left;">
MGI:5625100
</td>
<td style="text-align:left;">
lncRNA gene
</td>
<td style="text-align:left;">
predicted gene, 42215
</td>
</tr>
<tr>
<td style="text-align:left;">
3
</td>
<td style="text-align:left;">
gene
</td>
<td style="text-align:right;">
39.63596
</td>
<td style="text-align:right;">
39.63608
</td>
<td style="text-align:left;">
negative
</td>
<td style="text-align:left;">
MGI_C57BL6J_4422060
</td>
<td style="text-align:left;">
n-R5s195
</td>
<td style="text-align:left;">
ENSEMBL:ENSMUSG00000093952
</td>
<td style="text-align:left;">
MGI:4422060
</td>
<td style="text-align:left;">
rRNA gene
</td>
<td style="text-align:left;">
nuclear encoded rRNA 5S 195
</td>
</tr>
<tr>
<td style="text-align:left;">
3
</td>
<td style="text-align:left;">
gene
</td>
<td style="text-align:right;">
39.63606
</td>
<td style="text-align:right;">
39.63878
</td>
<td style="text-align:left;">
positive
</td>
<td style="text-align:left;">
MGI_C57BL6J_5662918
</td>
<td style="text-align:left;">
Gm42781
</td>
<td style="text-align:left;">
ENSEMBL:ENSMUSG00000106786
</td>
<td style="text-align:left;">
MGI:5662918
</td>
<td style="text-align:left;">
unclassified gene
</td>
<td style="text-align:left;">
predicted gene 42781
</td>
</tr>
<tr>
<td style="text-align:left;">
3
</td>
<td style="text-align:left;">
gene
</td>
<td style="text-align:right;">
39.66629
</td>
<td style="text-align:right;">
39.66645
</td>
<td style="text-align:left;">
positive
</td>
<td style="text-align:left;">
MGI_C57BL6J_5453573
</td>
<td style="text-align:left;">
Gm23796
</td>
<td style="text-align:left;">
ENSEMBL:ENSMUSG00000065945
</td>
<td style="text-align:left;">
MGI:5453573
</td>
<td style="text-align:left;">
snRNA gene
</td>
<td style="text-align:left;">
predicted gene, 23796
</td>
</tr>
<tr>
<td style="text-align:left;">
3
</td>
<td style="text-align:left;">
gene
</td>
<td style="text-align:right;">
39.71315
</td>
<td style="text-align:right;">
39.72565
</td>
<td style="text-align:left;">
positive
</td>
<td style="text-align:left;">
MGI_C57BL6J_5826471
</td>
<td style="text-align:left;">
Gm46834
</td>
<td style="text-align:left;">
NCBI_Gene:108168914
</td>
<td style="text-align:left;">
MGI:5826471
</td>
<td style="text-align:left;">
lncRNA gene
</td>
<td style="text-align:left;">
predicted gene, 46834
</td>
</tr>
<tr>
<td style="text-align:left;">
3
</td>
<td style="text-align:left;">
gene
</td>
<td style="text-align:right;">
39.79963
</td>
<td style="text-align:right;">
39.79972
</td>
<td style="text-align:left;">
negative
</td>
<td style="text-align:left;">
MGI_C57BL6J_5453608
</td>
<td style="text-align:left;">
Gm23831
</td>
<td style="text-align:left;">
ENSEMBL:ENSMUSG00000089520
</td>
<td style="text-align:left;">
MGI:5453608
</td>
<td style="text-align:left;">
snoRNA gene
</td>
<td style="text-align:left;">
predicted gene, 23831
</td>
</tr>
<tr>
<td style="text-align:left;">
3
</td>
<td style="text-align:left;">
pseudogene
</td>
<td style="text-align:right;">
39.82048
</td>
<td style="text-align:right;">
39.82076
</td>
<td style="text-align:left;">
negative
</td>
<td style="text-align:left;">
MGI_C57BL6J_5662923
</td>
<td style="text-align:left;">
Gm42786
</td>
<td style="text-align:left;">
ENSEMBL:ENSMUSG00000106846
</td>
<td style="text-align:left;">
MGI:5662923
</td>
<td style="text-align:left;">
pseudogene
</td>
<td style="text-align:left;">
predicted gene 42786
</td>
</tr>
<tr>
<td style="text-align:left;">
3
</td>
<td style="text-align:left;">
pseudogene
</td>
<td style="text-align:right;">
39.82358
</td>
<td style="text-align:right;">
39.82741
</td>
<td style="text-align:left;">
positive
</td>
<td style="text-align:left;">
MGI_C57BL6J_5662922
</td>
<td style="text-align:left;">
Gm42785
</td>
<td style="text-align:left;">
ENSEMBL:ENSMUSG00000106948
</td>
<td style="text-align:left;">
MGI:5662922
</td>
<td style="text-align:left;">
pseudogene
</td>
<td style="text-align:left;">
predicted gene 42785
</td>
</tr>
<tr>
<td style="text-align:left;">
3
</td>
<td style="text-align:left;">
gene
</td>
<td style="text-align:right;">
40.42571
</td>
<td style="text-align:right;">
40.45380
</td>
<td style="text-align:left;">
negative
</td>
<td style="text-align:left;">
MGI_C57BL6J_5589343
</td>
<td style="text-align:left;">
Gm30184
</td>
<td style="text-align:left;">
NCBI_Gene:102632001
</td>
<td style="text-align:left;">
MGI:5589343
</td>
<td style="text-align:left;">
lncRNA gene
</td>
<td style="text-align:left;">
predicted gene, 30184
</td>
</tr>
<tr>
<td style="text-align:left;">
3
</td>
<td style="text-align:left;">
gene
</td>
<td style="text-align:right;">
40.45843
</td>
<td style="text-align:right;">
40.46115
</td>
<td style="text-align:left;">
positive
</td>
<td style="text-align:left;">
MGI_C57BL6J_5625101
</td>
<td style="text-align:left;">
Gm42216
</td>
<td style="text-align:left;">
NCBI_Gene:105247034
</td>
<td style="text-align:left;">
MGI:5625101
</td>
<td style="text-align:left;">
lncRNA gene
</td>
<td style="text-align:left;">
predicted gene, 42216
</td>
</tr>
<tr>
<td style="text-align:left;">
3
</td>
<td style="text-align:left;">
gene
</td>
<td style="text-align:right;">
40.48090
</td>
<td style="text-align:right;">
40.48573
</td>
<td style="text-align:left;">
positive
</td>
<td style="text-align:left;">
MGI_C57BL6J_5625102
</td>
<td style="text-align:left;">
Gm42217
</td>
<td style="text-align:left;">
NCBI_Gene:105247035
</td>
<td style="text-align:left;">
MGI:5625102
</td>
<td style="text-align:left;">
lncRNA gene
</td>
<td style="text-align:left;">
predicted gene, 42217
</td>
</tr>
<tr>
<td style="text-align:left;">
3
</td>
<td style="text-align:left;">
gene
</td>
<td style="text-align:right;">
40.48702
</td>
<td style="text-align:right;">
40.48993
</td>
<td style="text-align:left;">
positive
</td>
<td style="text-align:left;">
MGI_C57BL6J_5826438
</td>
<td style="text-align:left;">
Gm46801
</td>
<td style="text-align:left;">
NCBI_Gene:108168866
</td>
<td style="text-align:left;">
MGI:5826438
</td>
<td style="text-align:left;">
lncRNA gene
</td>
<td style="text-align:left;">
predicted gene, 46801
</td>
</tr>
<tr>
<td style="text-align:left;">
3
</td>
<td style="text-align:left;">
gene
</td>
<td style="text-align:right;">
40.50487
</td>
<td style="text-align:right;">
40.52291
</td>
<td style="text-align:left;">
negative
</td>
<td style="text-align:left;">
MGI_C57BL6J_1919477
</td>
<td style="text-align:left;">
1700017G19Rik
</td>
<td style="text-align:left;">
NCBI_Gene:100040882,ENSEMBL:ENSMUSG00000037884
</td>
<td style="text-align:left;">
MGI:1919477
</td>
<td style="text-align:left;">
lncRNA gene
</td>
<td style="text-align:left;">
RIKEN cDNA 1700017G19 gene
</td>
</tr>
<tr>
<td style="text-align:left;">
3
</td>
<td style="text-align:left;">
gene
</td>
<td style="text-align:right;">
40.50998
</td>
<td style="text-align:right;">
40.51183
</td>
<td style="text-align:left;">
positive
</td>
<td style="text-align:left;">
MGI_C57BL6J_5826437
</td>
<td style="text-align:left;">
Gm46800
</td>
<td style="text-align:left;">
NCBI_Gene:108168865
</td>
<td style="text-align:left;">
MGI:5826437
</td>
<td style="text-align:left;">
lncRNA gene
</td>
<td style="text-align:left;">
predicted gene, 46800
</td>
</tr>
<tr>
<td style="text-align:left;">
3
</td>
<td style="text-align:left;">
gene
</td>
<td style="text-align:right;">
40.51882
</td>
<td style="text-align:right;">
40.51889
</td>
<td style="text-align:left;">
positive
</td>
<td style="text-align:left;">
MGI_C57BL6J_5453587
</td>
<td style="text-align:left;">
Gm23810
</td>
<td style="text-align:left;">
ENSEMBL:ENSMUSG00000092891
</td>
<td style="text-align:left;">
MGI:5453587
</td>
<td style="text-align:left;">
miRNA gene
</td>
<td style="text-align:left;">
predicted gene, 23810
</td>
</tr>
<tr>
<td style="text-align:left;">
3
</td>
<td style="text-align:left;">
gene
</td>
<td style="text-align:right;">
40.53129
</td>
<td style="text-align:right;">
40.70478
</td>
<td style="text-align:left;">
positive
</td>
<td style="text-align:left;">
MGI_C57BL6J_2443752
</td>
<td style="text-align:left;">
Intu
</td>
<td style="text-align:left;">
NCBI_Gene:380614,ENSEMBL:ENSMUSG00000060798
</td>
<td style="text-align:left;">
MGI:2443752
</td>
<td style="text-align:left;">
protein coding gene
</td>
<td style="text-align:left;">
inturned planar cell polarity protein
</td>
</tr>
<tr>
<td style="text-align:left;">
3
</td>
<td style="text-align:left;">
pseudogene
</td>
<td style="text-align:right;">
40.65887
</td>
<td style="text-align:right;">
40.65917
</td>
<td style="text-align:left;">
negative
</td>
<td style="text-align:left;">
MGI_C57BL6J_3801939
</td>
<td style="text-align:left;">
Gm16114
</td>
<td style="text-align:left;">
ENSEMBL:ENSMUSG00000089932
</td>
<td style="text-align:left;">
MGI:3801939
</td>
<td style="text-align:left;">
pseudogene
</td>
<td style="text-align:left;">
predicted gene 16114
</td>
</tr>
<tr>
<td style="text-align:left;">
3
</td>
<td style="text-align:left;">
gene
</td>
<td style="text-align:right;">
40.70885
</td>
<td style="text-align:right;">
40.72610
</td>
<td style="text-align:left;">
positive
</td>
<td style="text-align:left;">
MGI_C57BL6J_1920583
</td>
<td style="text-align:left;">
Slc25a31
</td>
<td style="text-align:left;">
NCBI_Gene:73333,ENSEMBL:ENSMUSG00000069041
</td>
<td style="text-align:left;">
MGI:1920583
</td>
<td style="text-align:left;">
protein coding gene
</td>
<td style="text-align:left;">
solute carrier family 25 (mitochondrial carrier, adenine nucleotide translocator), member 31
</td>
</tr>
<tr>
<td style="text-align:left;">
3
</td>
<td style="text-align:left;">
gene
</td>
<td style="text-align:right;">
40.74440
</td>
<td style="text-align:right;">
40.79610
</td>
<td style="text-align:left;">
positive
</td>
<td style="text-align:left;">
MGI_C57BL6J_107422
</td>
<td style="text-align:left;">
Hspa4l
</td>
<td style="text-align:left;">
NCBI_Gene:18415,ENSEMBL:ENSMUSG00000025757
</td>
<td style="text-align:left;">
MGI:107422
</td>
<td style="text-align:left;">
protein coding gene
</td>
<td style="text-align:left;">
heat shock protein 4 like
</td>
</tr>
<tr>
<td style="text-align:left;">
3
</td>
<td style="text-align:left;">
gene
</td>
<td style="text-align:right;">
40.79364
</td>
<td style="text-align:right;">
40.79846
</td>
<td style="text-align:left;">
positive
</td>
<td style="text-align:left;">
MGI_C57BL6J_1918107
</td>
<td style="text-align:left;">
4921509J17Rik
</td>
<td style="text-align:left;">
NA
</td>
<td style="text-align:left;">
NA
</td>
<td style="text-align:left;">
unclassified gene
</td>
<td style="text-align:left;">
RIKEN cDNA 4921509J17 gene
</td>
</tr>
<tr>
<td style="text-align:left;">
3
</td>
<td style="text-align:left;">
gene
</td>
<td style="text-align:right;">
40.79995
</td>
<td style="text-align:right;">
40.81688
</td>
<td style="text-align:left;">
positive
</td>
<td style="text-align:left;">
MGI_C57BL6J_101783
</td>
<td style="text-align:left;">
Plk4
</td>
<td style="text-align:left;">
NCBI_Gene:20873,ENSEMBL:ENSMUSG00000025758
</td>
<td style="text-align:left;">
MGI:101783
</td>
<td style="text-align:left;">
protein coding gene
</td>
<td style="text-align:left;">
polo like kinase 4
</td>
</tr>
<tr>
<td style="text-align:left;">
3
</td>
<td style="text-align:left;">
gene
</td>
<td style="text-align:right;">
40.81810
</td>
<td style="text-align:right;">
40.84689
</td>
<td style="text-align:left;">
negative
</td>
<td style="text-align:left;">
MGI_C57BL6J_1919425
</td>
<td style="text-align:left;">
Mfsd8
</td>
<td style="text-align:left;">
NCBI_Gene:72175,ENSEMBL:ENSMUSG00000025759
</td>
<td style="text-align:left;">
MGI:1919425
</td>
<td style="text-align:left;">
protein coding gene
</td>
<td style="text-align:left;">
major facilitator superfamily domain containing 8
</td>
</tr>
<tr>
<td style="text-align:left;">
3
</td>
<td style="text-align:left;">
pseudogene
</td>
<td style="text-align:right;">
40.84682
</td>
<td style="text-align:right;">
40.87561
</td>
<td style="text-align:left;">
positive
</td>
<td style="text-align:left;">
MGI_C57BL6J_3780180
</td>
<td style="text-align:left;">
Gm2011
</td>
<td style="text-align:left;">
NCBI_Gene:100039027,ENSEMBL:ENSMUSG00000107741
</td>
<td style="text-align:left;">
MGI:3780180
</td>
<td style="text-align:left;">
pseudogene
</td>
<td style="text-align:left;">
predicted gene 2011
</td>
</tr>
<tr>
<td style="text-align:left;">
3
</td>
<td style="text-align:left;">
gene
</td>
<td style="text-align:right;">
40.84697
</td>
<td style="text-align:right;">
40.93814
</td>
<td style="text-align:left;">
positive
</td>
<td style="text-align:left;">
MGI_C57BL6J_1915468
</td>
<td style="text-align:left;">
Abhd18
</td>
<td style="text-align:left;">
NCBI_Gene:269423,ENSEMBL:ENSMUSG00000037818
</td>
<td style="text-align:left;">
MGI:1915468
</td>
<td style="text-align:left;">
protein coding gene
</td>
<td style="text-align:left;">
abhydrolase domain containing 18
</td>
</tr>
<tr>
<td style="text-align:left;">
3
</td>
<td style="text-align:left;">
pseudogene
</td>
<td style="text-align:right;">
40.85348
</td>
<td style="text-align:right;">
40.85529
</td>
<td style="text-align:left;">
negative
</td>
<td style="text-align:left;">
MGI_C57BL6J_3795712
</td>
<td style="text-align:left;">
Gm10731
</td>
<td style="text-align:left;">
NCBI_Gene:100039043,ENSEMBL:ENSMUSG00000074620
</td>
<td style="text-align:left;">
MGI:3795712
</td>
<td style="text-align:left;">
pseudogene
</td>
<td style="text-align:left;">
predicted gene 10731
</td>
</tr>
<tr>
<td style="text-align:left;">
3
</td>
<td style="text-align:left;">
gene
</td>
<td style="text-align:right;">
40.86395
</td>
<td style="text-align:right;">
40.86505
</td>
<td style="text-align:left;">
negative
</td>
<td style="text-align:left;">
MGI_C57BL6J_5690528
</td>
<td style="text-align:left;">
Gm44136
</td>
<td style="text-align:left;">
ENSEMBL:ENSMUSG00000107524
</td>
<td style="text-align:left;">
MGI:5690528
</td>
<td style="text-align:left;">
unclassified gene
</td>
<td style="text-align:left;">
predicted gene, 44136
</td>
</tr>
<tr>
<td style="text-align:left;">
3
</td>
<td style="text-align:left;">
gene
</td>
<td style="text-align:right;">
40.87665
</td>
<td style="text-align:right;">
40.89455
</td>
<td style="text-align:left;">
negative
</td>
<td style="text-align:left;">
MGI_C57BL6J_3648839
</td>
<td style="text-align:left;">
6430590A07Rik
</td>
<td style="text-align:left;">
NCBI_Gene:619719,ENSEMBL:ENSMUSG00000097583
</td>
<td style="text-align:left;">
MGI:3648839
</td>
<td style="text-align:left;">
lncRNA gene
</td>
<td style="text-align:left;">
RIKEN cDNA 6430590A07 gene
</td>
</tr>
<tr>
<td style="text-align:left;">
3
</td>
<td style="text-align:left;">
pseudogene
</td>
<td style="text-align:right;">
40.87698
</td>
<td style="text-align:right;">
40.87725
</td>
<td style="text-align:left;">
negative
</td>
<td style="text-align:left;">
MGI_C57BL6J_5753100
</td>
<td style="text-align:left;">
Gm44524
</td>
<td style="text-align:left;">
ENSEMBL:ENSMUSG00000107892
</td>
<td style="text-align:left;">
MGI:5753100
</td>
<td style="text-align:left;">
pseudogene
</td>
<td style="text-align:left;">
predicted gene 44524
</td>
</tr>
<tr>
<td style="text-align:left;">
3
</td>
<td style="text-align:left;">
gene
</td>
<td style="text-align:right;">
40.90020
</td>
<td style="text-align:right;">
40.90254
</td>
<td style="text-align:left;">
negative
</td>
<td style="text-align:left;">
MGI_C57BL6J_1920547
</td>
<td style="text-align:left;">
1700034I23Rik
</td>
<td style="text-align:left;">
NCBI_Gene:73297,ENSEMBL:ENSMUSG00000074619
</td>
<td style="text-align:left;">
MGI:1920547
</td>
<td style="text-align:left;">
protein coding gene
</td>
<td style="text-align:left;">
RIKEN cDNA 1700034I23 gene
</td>
</tr>
<tr>
<td style="text-align:left;">
3
</td>
<td style="text-align:left;">
gene
</td>
<td style="text-align:right;">
40.94869
</td>
<td style="text-align:right;">
40.94986
</td>
<td style="text-align:left;">
negative
</td>
<td style="text-align:left;">
MGI_C57BL6J_5826439
</td>
<td style="text-align:left;">
Gm46802
</td>
<td style="text-align:left;">
NCBI_Gene:108168867
</td>
<td style="text-align:left;">
MGI:5826439
</td>
<td style="text-align:left;">
lncRNA gene
</td>
<td style="text-align:left;">
predicted gene, 46802
</td>
</tr>
<tr>
<td style="text-align:left;">
3
</td>
<td style="text-align:left;">
gene
</td>
<td style="text-align:right;">
40.94972
</td>
<td style="text-align:right;">
41.04023
</td>
<td style="text-align:left;">
positive
</td>
<td style="text-align:left;">
MGI_C57BL6J_1914604
</td>
<td style="text-align:left;">
Larp1b
</td>
<td style="text-align:left;">
NCBI_Gene:214048,ENSEMBL:ENSMUSG00000025762
</td>
<td style="text-align:left;">
MGI:1914604
</td>
<td style="text-align:left;">
protein coding gene
</td>
<td style="text-align:left;">
La ribonucleoprotein domain family, member 1B
</td>
</tr>
<tr>
<td style="text-align:left;">
3
</td>
<td style="text-align:left;">
gene
</td>
<td style="text-align:right;">
40.97850
</td>
<td style="text-align:right;">
41.00609
</td>
<td style="text-align:left;">
positive
</td>
<td style="text-align:left;">
MGI_C57BL6J_5589448
</td>
<td style="text-align:left;">
Gm30289
</td>
<td style="text-align:left;">
NA
</td>
<td style="text-align:left;">
NA
</td>
<td style="text-align:left;">
protein coding gene
</td>
<td style="text-align:left;">
predicted gene%2c 30289
</td>
</tr>
<tr>
<td style="text-align:left;">
3
</td>
<td style="text-align:left;">
gene
</td>
<td style="text-align:right;">
40.99269
</td>
<td style="text-align:right;">
40.99285
</td>
<td style="text-align:left;">
positive
</td>
<td style="text-align:left;">
MGI_C57BL6J_5451820
</td>
<td style="text-align:left;">
Gm22043
</td>
<td style="text-align:left;">
ENSEMBL:ENSMUSG00000064927
</td>
<td style="text-align:left;">
MGI:5451820
</td>
<td style="text-align:left;">
snRNA gene
</td>
<td style="text-align:left;">
predicted gene, 22043
</td>
</tr>
<tr>
<td style="text-align:left;">
3
</td>
<td style="text-align:left;">
gene
</td>
<td style="text-align:right;">
41.00243
</td>
<td style="text-align:right;">
41.00609
</td>
<td style="text-align:left;">
positive
</td>
<td style="text-align:left;">
MGI_C57BL6J_5662573
</td>
<td style="text-align:left;">
Gm42436
</td>
<td style="text-align:left;">
ENSEMBL:ENSMUSG00000105517
</td>
<td style="text-align:left;">
MGI:5662573
</td>
<td style="text-align:left;">
unclassified gene
</td>
<td style="text-align:left;">
predicted gene 42436
</td>
</tr>
<tr>
<td style="text-align:left;">
3
</td>
<td style="text-align:left;">
gene
</td>
<td style="text-align:right;">
41.00552
</td>
<td style="text-align:right;">
41.00644
</td>
<td style="text-align:left;">
negative
</td>
<td style="text-align:left;">
MGI_C57BL6J_5662574
</td>
<td style="text-align:left;">
Gm42437
</td>
<td style="text-align:left;">
ENSEMBL:ENSMUSG00000105674
</td>
<td style="text-align:left;">
MGI:5662574
</td>
<td style="text-align:left;">
unclassified gene
</td>
<td style="text-align:left;">
predicted gene 42437
</td>
</tr>
<tr>
<td style="text-align:left;">
3
</td>
<td style="text-align:left;">
gene
</td>
<td style="text-align:right;">
41.06633
</td>
<td style="text-align:right;">
41.08305
</td>
<td style="text-align:left;">
negative
</td>
<td style="text-align:left;">
MGI_C57BL6J_1918054
</td>
<td style="text-align:left;">
Pgrmc2
</td>
<td style="text-align:left;">
NCBI_Gene:70804,ENSEMBL:ENSMUSG00000049940
</td>
<td style="text-align:left;">
MGI:1918054
</td>
<td style="text-align:left;">
protein coding gene
</td>
<td style="text-align:left;">
progesterone receptor membrane component 2
</td>
</tr>
<tr>
<td style="text-align:left;">
3
</td>
<td style="text-align:left;">
gene
</td>
<td style="text-align:right;">
41.15803
</td>
<td style="text-align:right;">
41.17346
</td>
<td style="text-align:left;">
negative
</td>
<td style="text-align:left;">
MGI_C57BL6J_5622923
</td>
<td style="text-align:left;">
Gm40038
</td>
<td style="text-align:left;">
NCBI_Gene:105244429,ENSEMBL:ENSMUSG00000105258
</td>
<td style="text-align:left;">
MGI:5622923
</td>
<td style="text-align:left;">
lncRNA gene
</td>
<td style="text-align:left;">
predicted gene, 40038
</td>
</tr>
<tr>
<td style="text-align:left;">
3
</td>
<td style="text-align:left;">
gene
</td>
<td style="text-align:right;">
41.28606
</td>
<td style="text-align:right;">
41.29059
</td>
<td style="text-align:left;">
negative
</td>
<td style="text-align:left;">
MGI_C57BL6J_3642810
</td>
<td style="text-align:left;">
Gm16508
</td>
<td style="text-align:left;">
ENSEMBL:ENSMUSG00000106000
</td>
<td style="text-align:left;">
MGI:3642810
</td>
<td style="text-align:left;">
lncRNA gene
</td>
<td style="text-align:left;">
predicted gene 16508
</td>
</tr>
<tr>
<td style="text-align:left;">
3
</td>
<td style="text-align:left;">
gene
</td>
<td style="text-align:right;">
41.35415
</td>
<td style="text-align:right;">
41.40446
</td>
<td style="text-align:left;">
positive
</td>
<td style="text-align:left;">
MGI_C57BL6J_1920712
</td>
<td style="text-align:left;">
1700052H01Rik
</td>
<td style="text-align:left;">
NCBI_Gene:73462,ENSEMBL:ENSMUSG00000105297
</td>
<td style="text-align:left;">
MGI:1920712
</td>
<td style="text-align:left;">
lncRNA gene
</td>
<td style="text-align:left;">
RIKEN cDNA 1700052H01 gene
</td>
</tr>
<tr>
<td style="text-align:left;">
3
</td>
<td style="text-align:left;">
gene
</td>
<td style="text-align:right;">
41.44134
</td>
<td style="text-align:right;">
41.44147
</td>
<td style="text-align:left;">
negative
</td>
<td style="text-align:left;">
MGI_C57BL6J_5455264
</td>
<td style="text-align:left;">
Gm25487
</td>
<td style="text-align:left;">
ENSEMBL:ENSMUSG00000089350
</td>
<td style="text-align:left;">
MGI:5455264
</td>
<td style="text-align:left;">
snoRNA gene
</td>
<td style="text-align:left;">
predicted gene, 25487
</td>
</tr>
<tr>
<td style="text-align:left;">
3
</td>
<td style="text-align:left;">
gene
</td>
<td style="text-align:right;">
41.47491
</td>
<td style="text-align:right;">
41.48400
</td>
<td style="text-align:left;">
negative
</td>
<td style="text-align:left;">
MGI_C57BL6J_5622925
</td>
<td style="text-align:left;">
Gm40040
</td>
<td style="text-align:left;">
NCBI_Gene:105244431,ENSEMBL:ENSMUSG00000105260
</td>
<td style="text-align:left;">
MGI:5622925
</td>
<td style="text-align:left;">
lncRNA gene
</td>
<td style="text-align:left;">
predicted gene, 40040
</td>
</tr>
<tr>
<td style="text-align:left;">
3
</td>
<td style="text-align:left;">
gene
</td>
<td style="text-align:right;">
41.48402
</td>
<td style="text-align:right;">
41.49320
</td>
<td style="text-align:left;">
negative
</td>
<td style="text-align:left;">
MGI_C57BL6J_1925956
</td>
<td style="text-align:left;">
Platr4
</td>
<td style="text-align:left;">
NCBI_Gene:78706,ENSEMBL:ENSMUSG00000097639
</td>
<td style="text-align:left;">
MGI:1925956
</td>
<td style="text-align:left;">
lncRNA gene
</td>
<td style="text-align:left;">
pluripotency associated transcript 4
</td>
</tr>
<tr>
<td style="text-align:left;">
3
</td>
<td style="text-align:left;">
pseudogene
</td>
<td style="text-align:right;">
41.48569
</td>
<td style="text-align:right;">
41.48641
</td>
<td style="text-align:left;">
positive
</td>
<td style="text-align:left;">
MGI_C57BL6J_3648422
</td>
<td style="text-align:left;">
Gm6069
</td>
<td style="text-align:left;">
NCBI_Gene:606542,ENSEMBL:ENSMUSG00000104616
</td>
<td style="text-align:left;">
MGI:3648422
</td>
<td style="text-align:left;">
pseudogene
</td>
<td style="text-align:left;">
predicted gene 6069
</td>
</tr>
<tr>
<td style="text-align:left;">
3
</td>
<td style="text-align:left;">
gene
</td>
<td style="text-align:right;">
41.50093
</td>
<td style="text-align:right;">
41.52847
</td>
<td style="text-align:left;">
negative
</td>
<td style="text-align:left;">
MGI_C57BL6J_1925898
</td>
<td style="text-align:left;">
2400006E01Rik
</td>
<td style="text-align:left;">
NCBI_Gene:78648,ENSEMBL:ENSMUSG00000106223
</td>
<td style="text-align:left;">
MGI:1925898
</td>
<td style="text-align:left;">
lncRNA gene
</td>
<td style="text-align:left;">
RIKEN cDNA 2400006E01 gene
</td>
</tr>
<tr>
<td style="text-align:left;">
3
</td>
<td style="text-align:left;">
gene
</td>
<td style="text-align:right;">
41.55573
</td>
<td style="text-align:right;">
41.61686
</td>
<td style="text-align:left;">
positive
</td>
<td style="text-align:left;">
MGI_C57BL6J_1925835
</td>
<td style="text-align:left;">
Jade1
</td>
<td style="text-align:left;">
NCBI_Gene:269424,ENSEMBL:ENSMUSG00000025764
</td>
<td style="text-align:left;">
MGI:1925835
</td>
<td style="text-align:left;">
protein coding gene
</td>
<td style="text-align:left;">
jade family PHD finger 1
</td>
</tr>
<tr>
<td style="text-align:left;">
3
</td>
<td style="text-align:left;">
gene
</td>
<td style="text-align:right;">
41.56050
</td>
<td style="text-align:right;">
41.56370
</td>
<td style="text-align:left;">
negative
</td>
<td style="text-align:left;">
MGI_C57BL6J_5590425
</td>
<td style="text-align:left;">
Gm31266
</td>
<td style="text-align:left;">
NCBI_Gene:105244433,ENSEMBL:ENSMUSG00000103843
</td>
<td style="text-align:left;">
MGI:5590425
</td>
<td style="text-align:left;">
lncRNA gene
</td>
<td style="text-align:left;">
predicted gene, 31266
</td>
</tr>
<tr>
<td style="text-align:left;">
3
</td>
<td style="text-align:left;">
gene
</td>
<td style="text-align:right;">
41.61755
</td>
<td style="text-align:right;">
41.62371
</td>
<td style="text-align:left;">
negative
</td>
<td style="text-align:left;">
MGI_C57BL6J_5622926
</td>
<td style="text-align:left;">
Gm40041
</td>
<td style="text-align:left;">
NCBI_Gene:105244432
</td>
<td style="text-align:left;">
MGI:5622926
</td>
<td style="text-align:left;">
lncRNA gene
</td>
<td style="text-align:left;">
predicted gene, 40041
</td>
</tr>
<tr>
<td style="text-align:left;">
3
</td>
<td style="text-align:left;">
gene
</td>
<td style="text-align:right;">
41.62671
</td>
<td style="text-align:right;">
41.74253
</td>
<td style="text-align:left;">
negative
</td>
<td style="text-align:left;">
MGI_C57BL6J_1914411
</td>
<td style="text-align:left;">
Sclt1
</td>
<td style="text-align:left;">
NCBI_Gene:67161,ENSEMBL:ENSMUSG00000059834
</td>
<td style="text-align:left;">
MGI:1914411
</td>
<td style="text-align:left;">
protein coding gene
</td>
<td style="text-align:left;">
sodium channel and clathrin linker 1
</td>
</tr>
<tr>
<td style="text-align:left;">
3
</td>
<td style="text-align:left;">
gene
</td>
<td style="text-align:right;">
41.64812
</td>
<td style="text-align:right;">
41.64925
</td>
<td style="text-align:left;">
negative
</td>
<td style="text-align:left;">
MGI_C57BL6J_5611275
</td>
<td style="text-align:left;">
Gm38047
</td>
<td style="text-align:left;">
ENSEMBL:ENSMUSG00000102436
</td>
<td style="text-align:left;">
MGI:5611275
</td>
<td style="text-align:left;">
unclassified gene
</td>
<td style="text-align:left;">
predicted gene, 38047
</td>
</tr>
<tr>
<td style="text-align:left;">
3
</td>
<td style="text-align:left;">
gene
</td>
<td style="text-align:right;">
41.74261
</td>
<td style="text-align:right;">
41.80332
</td>
<td style="text-align:left;">
positive
</td>
<td style="text-align:left;">
MGI_C57BL6J_1289213
</td>
<td style="text-align:left;">
D3Ertd751e
</td>
<td style="text-align:left;">
NCBI_Gene:73852,ENSEMBL:ENSMUSG00000025766
</td>
<td style="text-align:left;">
MGI:1289213
</td>
<td style="text-align:left;">
protein coding gene
</td>
<td style="text-align:left;">
DNA segment, Chr 3, ERATO Doi 751, expressed
</td>
</tr>
<tr>
<td style="text-align:left;">
3
</td>
<td style="text-align:left;">
gene
</td>
<td style="text-align:right;">
41.77318
</td>
<td style="text-align:right;">
41.77579
</td>
<td style="text-align:left;">
positive
</td>
<td style="text-align:left;">
MGI_C57BL6J_5610496
</td>
<td style="text-align:left;">
Gm37268
</td>
<td style="text-align:left;">
ENSEMBL:ENSMUSG00000103952
</td>
<td style="text-align:left;">
MGI:5610496
</td>
<td style="text-align:left;">
unclassified gene
</td>
<td style="text-align:left;">
predicted gene, 37268
</td>
</tr>
<tr>
<td style="text-align:left;">
3
</td>
<td style="text-align:left;">
pseudogene
</td>
<td style="text-align:right;">
41.97607
</td>
<td style="text-align:right;">
41.97812
</td>
<td style="text-align:left;">
negative
</td>
<td style="text-align:left;">
MGI_C57BL6J_3645044
</td>
<td style="text-align:left;">
Gm7899
</td>
<td style="text-align:left;">
NCBI_Gene:666031,ENSEMBL:ENSMUSG00000103823
</td>
<td style="text-align:left;">
MGI:3645044
</td>
<td style="text-align:left;">
pseudogene
</td>
<td style="text-align:left;">
predicted gene 7899
</td>
</tr>
<tr>
<td style="text-align:left;">
3
</td>
<td style="text-align:left;">
pseudogene
</td>
<td style="text-align:right;">
41.97633
</td>
<td style="text-align:right;">
41.97812
</td>
<td style="text-align:left;">
positive
</td>
<td style="text-align:left;">
MGI_C57BL6J_5610576
</td>
<td style="text-align:left;">
Gm37348
</td>
<td style="text-align:left;">
ENSEMBL:ENSMUSG00000102972
</td>
<td style="text-align:left;">
MGI:5610576
</td>
<td style="text-align:left;">
pseudogene
</td>
<td style="text-align:left;">
predicted gene, 37348
</td>
</tr>
<tr>
<td style="text-align:left;">
3
</td>
<td style="text-align:left;">
gene
</td>
<td style="text-align:right;">
42.16387
</td>
<td style="text-align:right;">
42.16764
</td>
<td style="text-align:left;">
positive
</td>
<td style="text-align:left;">
MGI_C57BL6J_5611074
</td>
<td style="text-align:left;">
Gm37846
</td>
<td style="text-align:left;">
ENSEMBL:ENSMUSG00000102610
</td>
<td style="text-align:left;">
MGI:5611074
</td>
<td style="text-align:left;">
unclassified gene
</td>
<td style="text-align:left;">
predicted gene, 37846
</td>
</tr>
<tr>
<td style="text-align:left;">
3
</td>
<td style="text-align:left;">
pseudogene
</td>
<td style="text-align:right;">
42.36263
</td>
<td style="text-align:right;">
42.36278
</td>
<td style="text-align:left;">
negative
</td>
<td style="text-align:left;">
MGI_C57BL6J_5611502
</td>
<td style="text-align:left;">
Gm38274
</td>
<td style="text-align:left;">
ENSEMBL:ENSMUSG00000104089
</td>
<td style="text-align:left;">
MGI:5611502
</td>
<td style="text-align:left;">
pseudogene
</td>
<td style="text-align:left;">
predicted gene, 38274
</td>
</tr>
<tr>
<td style="text-align:left;">
3
</td>
<td style="text-align:left;">
gene
</td>
<td style="text-align:right;">
42.57246
</td>
<td style="text-align:right;">
42.57300
</td>
<td style="text-align:left;">
positive
</td>
<td style="text-align:left;">
MGI_C57BL6J_5610761
</td>
<td style="text-align:left;">
Gm37533
</td>
<td style="text-align:left;">
ENSEMBL:ENSMUSG00000103426
</td>
<td style="text-align:left;">
MGI:5610761
</td>
<td style="text-align:left;">
unclassified gene
</td>
<td style="text-align:left;">
predicted gene, 37533
</td>
</tr>
<tr>
<td style="text-align:left;">
3
</td>
<td style="text-align:left;">
pseudogene
</td>
<td style="text-align:right;">
42.58511
</td>
<td style="text-align:right;">
42.58560
</td>
<td style="text-align:left;">
negative
</td>
<td style="text-align:left;">
MGI_C57BL6J_5611272
</td>
<td style="text-align:left;">
Gm38044
</td>
<td style="text-align:left;">
ENSEMBL:ENSMUSG00000104294
</td>
<td style="text-align:left;">
MGI:5611272
</td>
<td style="text-align:left;">
pseudogene
</td>
<td style="text-align:left;">
predicted gene, 38044
</td>
</tr>
<tr>
<td style="text-align:left;">
3
</td>
<td style="text-align:left;">
pseudogene
</td>
<td style="text-align:right;">
43.25039
</td>
<td style="text-align:right;">
43.25093
</td>
<td style="text-align:left;">
positive
</td>
<td style="text-align:left;">
MGI_C57BL6J_3780131
</td>
<td style="text-align:left;">
Gm9723
</td>
<td style="text-align:left;">
NCBI_Gene:677788,ENSEMBL:ENSMUSG00000102149
</td>
<td style="text-align:left;">
MGI:3780131
</td>
<td style="text-align:left;">
pseudogene
</td>
<td style="text-align:left;">
predicted gene 9723
</td>
</tr>
<tr>
<td style="text-align:left;">
3
</td>
<td style="text-align:left;">
pseudogene
</td>
<td style="text-align:right;">
43.39077
</td>
<td style="text-align:right;">
43.39194
</td>
<td style="text-align:left;">
negative
</td>
<td style="text-align:left;">
MGI_C57BL6J_5011050
</td>
<td style="text-align:left;">
Gm18865
</td>
<td style="text-align:left;">
NCBI_Gene:100417854,ENSEMBL:ENSMUSG00000102334
</td>
<td style="text-align:left;">
MGI:5011050
</td>
<td style="text-align:left;">
pseudogene
</td>
<td style="text-align:left;">
predicted gene, 18865
</td>
</tr>
<tr>
<td style="text-align:left;">
3
</td>
<td style="text-align:left;">
gene
</td>
<td style="text-align:right;">
43.54717
</td>
<td style="text-align:right;">
43.54828
</td>
<td style="text-align:left;">
positive
</td>
<td style="text-align:left;">
MGI_C57BL6J_5610204
</td>
<td style="text-align:left;">
Gm36976
</td>
<td style="text-align:left;">
ENSEMBL:ENSMUSG00000104303
</td>
<td style="text-align:left;">
MGI:5610204
</td>
<td style="text-align:left;">
unclassified gene
</td>
<td style="text-align:left;">
predicted gene, 36976
</td>
</tr>
<tr>
<td style="text-align:left;">
3
</td>
<td style="text-align:left;">
pseudogene
</td>
<td style="text-align:right;">
43.76405
</td>
<td style="text-align:right;">
43.76476
</td>
<td style="text-align:left;">
negative
</td>
<td style="text-align:left;">
MGI_C57BL6J_5588898
</td>
<td style="text-align:left;">
Gm29739
</td>
<td style="text-align:left;">
NCBI_Gene:101055891,ENSEMBL:ENSMUSG00000104083
</td>
<td style="text-align:left;">
MGI:5588898
</td>
<td style="text-align:left;">
pseudogene
</td>
<td style="text-align:left;">
predicted gene, 29739
</td>
</tr>
<tr>
<td style="text-align:left;">
3
</td>
<td style="text-align:left;">
gene
</td>
<td style="text-align:right;">
43.92557
</td>
<td style="text-align:right;">
43.92586
</td>
<td style="text-align:left;">
positive
</td>
<td style="text-align:left;">
MGI_C57BL6J_5455491
</td>
<td style="text-align:left;">
Gm25714
</td>
<td style="text-align:left;">
ENSEMBL:ENSMUSG00000089251
</td>
<td style="text-align:left;">
MGI:5455491
</td>
<td style="text-align:left;">
unclassified non-coding RNA gene
</td>
<td style="text-align:left;">
predicted gene, 25714
</td>
</tr>
<tr>
<td style="text-align:left;">
3
</td>
<td style="text-align:left;">
pseudogene
</td>
<td style="text-align:right;">
43.92760
</td>
<td style="text-align:right;">
43.92795
</td>
<td style="text-align:left;">
negative
</td>
<td style="text-align:left;">
MGI_C57BL6J_5591334
</td>
<td style="text-align:left;">
Gm32175
</td>
<td style="text-align:left;">
ENSEMBL:ENSMUSG00000102792
</td>
<td style="text-align:left;">
MGI:5591334
</td>
<td style="text-align:left;">
pseudogene
</td>
<td style="text-align:left;">
predicted gene, 32175
</td>
</tr>
<tr>
<td style="text-align:left;">
3
</td>
<td style="text-align:left;">
gene
</td>
<td style="text-align:right;">
43.95572
</td>
<td style="text-align:right;">
43.95592
</td>
<td style="text-align:left;">
positive
</td>
<td style="text-align:left;">
MGI_C57BL6J_5610974
</td>
<td style="text-align:left;">
Gm37746
</td>
<td style="text-align:left;">
ENSEMBL:ENSMUSG00000103148
</td>
<td style="text-align:left;">
MGI:5610974
</td>
<td style="text-align:left;">
unclassified gene
</td>
<td style="text-align:left;">
predicted gene, 37746
</td>
</tr>
<tr>
<td style="text-align:left;">
3
</td>
<td style="text-align:left;">
pseudogene
</td>
<td style="text-align:right;">
44.49350
</td>
<td style="text-align:right;">
44.49390
</td>
<td style="text-align:left;">
negative
</td>
<td style="text-align:left;">
MGI_C57BL6J_5610593
</td>
<td style="text-align:left;">
Gm37365
</td>
<td style="text-align:left;">
ENSEMBL:ENSMUSG00000104013
</td>
<td style="text-align:left;">
MGI:5610593
</td>
<td style="text-align:left;">
pseudogene
</td>
<td style="text-align:left;">
predicted gene, 37365
</td>
</tr>
<tr>
<td style="text-align:left;">
3
</td>
<td style="text-align:left;">
gene
</td>
<td style="text-align:right;">
44.65990
</td>
<td style="text-align:right;">
44.66315
</td>
<td style="text-align:left;">
positive
</td>
<td style="text-align:left;">
MGI_C57BL6J_5611356
</td>
<td style="text-align:left;">
Gm38128
</td>
<td style="text-align:left;">
ENSEMBL:ENSMUSG00000104051
</td>
<td style="text-align:left;">
MGI:5611356
</td>
<td style="text-align:left;">
unclassified gene
</td>
<td style="text-align:left;">
predicted gene, 38128
</td>
</tr>
<tr>
<td style="text-align:left;">
3
</td>
<td style="text-align:left;">
pseudogene
</td>
<td style="text-align:right;">
44.71813
</td>
<td style="text-align:right;">
44.71851
</td>
<td style="text-align:left;">
negative
</td>
<td style="text-align:left;">
MGI_C57BL6J_5610825
</td>
<td style="text-align:left;">
Gm37597
</td>
<td style="text-align:left;">
ENSEMBL:ENSMUSG00000103353
</td>
<td style="text-align:left;">
MGI:5610825
</td>
<td style="text-align:left;">
pseudogene
</td>
<td style="text-align:left;">
predicted gene, 37597
</td>
</tr>
<tr>
<td style="text-align:left;">
3
</td>
<td style="text-align:left;">
pseudogene
</td>
<td style="text-align:right;">
44.72516
</td>
<td style="text-align:right;">
44.72574
</td>
<td style="text-align:left;">
positive
</td>
<td style="text-align:left;">
MGI_C57BL6J_5011051
</td>
<td style="text-align:left;">
Gm18866
</td>
<td style="text-align:left;">
NCBI_Gene:100417856
</td>
<td style="text-align:left;">
MGI:5011051
</td>
<td style="text-align:left;">
pseudogene
</td>
<td style="text-align:left;">
predicted gene, 18866
</td>
</tr>
<tr>
<td style="text-align:left;">
3
</td>
<td style="text-align:left;">
gene
</td>
<td style="text-align:right;">
44.98320
</td>
<td style="text-align:right;">
44.98330
</td>
<td style="text-align:left;">
positive
</td>
<td style="text-align:left;">
MGI_C57BL6J_5454774
</td>
<td style="text-align:left;">
Gm24997
</td>
<td style="text-align:left;">
ENSEMBL:ENSMUSG00000088535
</td>
<td style="text-align:left;">
MGI:5454774
</td>
<td style="text-align:left;">
snRNA gene
</td>
<td style="text-align:left;">
predicted gene, 24997
</td>
</tr>
<tr>
<td style="text-align:left;">
3
</td>
<td style="text-align:left;">
gene
</td>
<td style="text-align:right;">
45.16069
</td>
<td style="text-align:right;">
45.16080
</td>
<td style="text-align:left;">
positive
</td>
<td style="text-align:left;">
MGI_C57BL6J_5531371
</td>
<td style="text-align:left;">
Gm27989
</td>
<td style="text-align:left;">
ENSEMBL:ENSMUSG00000099285
</td>
<td style="text-align:left;">
MGI:5531371
</td>
<td style="text-align:left;">
miRNA gene
</td>
<td style="text-align:left;">
predicted gene, 27989
</td>
</tr>
<tr>
<td style="text-align:left;">
3
</td>
<td style="text-align:left;">
pseudogene
</td>
<td style="text-align:right;">
45.17179
</td>
<td style="text-align:right;">
45.17324
</td>
<td style="text-align:left;">
negative
</td>
<td style="text-align:left;">
MGI_C57BL6J_5434370
</td>
<td style="text-align:left;">
Gm21015
</td>
<td style="text-align:left;">
NCBI_Gene:100534291
</td>
<td style="text-align:left;">
MGI:5434370
</td>
<td style="text-align:left;">
pseudogene
</td>
<td style="text-align:left;">
predicted gene, 21015
</td>
</tr>
<tr>
<td style="text-align:left;">
3
</td>
<td style="text-align:left;">
gene
</td>
<td style="text-align:right;">
45.26999
</td>
<td style="text-align:right;">
45.27257
</td>
<td style="text-align:left;">
negative
</td>
<td style="text-align:left;">
MGI_C57BL6J_5611352
</td>
<td style="text-align:left;">
Gm38124
</td>
<td style="text-align:left;">
ENSEMBL:ENSMUSG00000103082
</td>
<td style="text-align:left;">
MGI:5611352
</td>
<td style="text-align:left;">
unclassified gene
</td>
<td style="text-align:left;">
predicted gene, 38124
</td>
</tr>
<tr>
<td style="text-align:left;">
3
</td>
<td style="text-align:left;">
gene
</td>
<td style="text-align:right;">
45.27342
</td>
<td style="text-align:right;">
45.27497
</td>
<td style="text-align:left;">
negative
</td>
<td style="text-align:left;">
MGI_C57BL6J_1924316
</td>
<td style="text-align:left;">
5930409G06Rik
</td>
<td style="text-align:left;">
ENSEMBL:ENSMUSG00000103477
</td>
<td style="text-align:left;">
MGI:1924316
</td>
<td style="text-align:left;">
unclassified gene
</td>
<td style="text-align:left;">
RIKEN cDNA 5930409G06 gene
</td>
</tr>
<tr>
<td style="text-align:left;">
3
</td>
<td style="text-align:left;">
gene
</td>
<td style="text-align:right;">
45.28044
</td>
<td style="text-align:right;">
45.37826
</td>
<td style="text-align:left;">
negative
</td>
<td style="text-align:left;">
MGI_C57BL6J_1919761
</td>
<td style="text-align:left;">
2610316D01Rik
</td>
<td style="text-align:left;">
NCBI_Gene:72511,ENSEMBL:ENSMUSG00000097040
</td>
<td style="text-align:left;">
MGI:1919761
</td>
<td style="text-align:left;">
lncRNA gene
</td>
<td style="text-align:left;">
RIKEN cDNA 2610316D01 gene
</td>
</tr>
<tr>
<td style="text-align:left;">
3
</td>
<td style="text-align:left;">
pseudogene
</td>
<td style="text-align:right;">
45.33216
</td>
<td style="text-align:right;">
45.33241
</td>
<td style="text-align:left;">
negative
</td>
<td style="text-align:left;">
MGI_C57BL6J_5610224
</td>
<td style="text-align:left;">
Gm36996
</td>
<td style="text-align:left;">
ENSEMBL:ENSMUSG00000103430
</td>
<td style="text-align:left;">
MGI:5610224
</td>
<td style="text-align:left;">
pseudogene
</td>
<td style="text-align:left;">
predicted gene, 36996
</td>
</tr>
<tr>
<td style="text-align:left;">
3
</td>
<td style="text-align:left;">
gene
</td>
<td style="text-align:right;">
45.37840
</td>
<td style="text-align:right;">
45.43562
</td>
<td style="text-align:left;">
positive
</td>
<td style="text-align:left;">
MGI_C57BL6J_1338042
</td>
<td style="text-align:left;">
Pcdh10
</td>
<td style="text-align:left;">
NCBI_Gene:18526,ENSEMBL:ENSMUSG00000049100
</td>
<td style="text-align:left;">
MGI:1338042
</td>
<td style="text-align:left;">
protein coding gene
</td>
<td style="text-align:left;">
protocadherin 10
</td>
</tr>
<tr>
<td style="text-align:left;">
3
</td>
<td style="text-align:left;">
gene
</td>
<td style="text-align:right;">
45.38548
</td>
<td style="text-align:right;">
45.38960
</td>
<td style="text-align:left;">
positive
</td>
<td style="text-align:left;">
MGI_C57BL6J_2442672
</td>
<td style="text-align:left;">
6330566A10Rik
</td>
<td style="text-align:left;">
NA
</td>
<td style="text-align:left;">
NA
</td>
<td style="text-align:left;">
unclassified gene
</td>
<td style="text-align:left;">
RIKEN cDNA 6330566A10 gene
</td>
</tr>
<tr>
<td style="text-align:left;">
3
</td>
<td style="text-align:left;">
gene
</td>
<td style="text-align:right;">
45.39986
</td>
<td style="text-align:right;">
45.40200
</td>
<td style="text-align:left;">
positive
</td>
<td style="text-align:left;">
MGI_C57BL6J_2444936
</td>
<td style="text-align:left;">
D930002L09Rik
</td>
<td style="text-align:left;">
NA
</td>
<td style="text-align:left;">
NA
</td>
<td style="text-align:left;">
unclassified gene
</td>
<td style="text-align:left;">
RIKEN cDNA D930002L09 gene
</td>
</tr>
<tr>
<td style="text-align:left;">
3
</td>
<td style="text-align:left;">
gene
</td>
<td style="text-align:right;">
45.41658
</td>
<td style="text-align:right;">
45.43931
</td>
<td style="text-align:left;">
negative
</td>
<td style="text-align:left;">
MGI_C57BL6J_1919524
</td>
<td style="text-align:left;">
1700027H10Rik
</td>
<td style="text-align:left;">
NCBI_Gene:72274,ENSEMBL:ENSMUSG00000100799
</td>
<td style="text-align:left;">
MGI:1919524
</td>
<td style="text-align:left;">
lncRNA gene
</td>
<td style="text-align:left;">
RIKEN cDNA 1700027H10 gene
</td>
</tr>
<tr>
<td style="text-align:left;">
3
</td>
<td style="text-align:left;">
gene
</td>
<td style="text-align:right;">
45.43838
</td>
<td style="text-align:right;">
45.44096
</td>
<td style="text-align:left;">
positive
</td>
<td style="text-align:left;">
MGI_C57BL6J_3780305
</td>
<td style="text-align:left;">
Gm2136
</td>
<td style="text-align:left;">
ENSEMBL:ENSMUSG00000103713
</td>
<td style="text-align:left;">
MGI:3780305
</td>
<td style="text-align:left;">
protein coding gene
</td>
<td style="text-align:left;">
predicted gene 2136
</td>
</tr>
<tr>
<td style="text-align:left;">
3
</td>
<td style="text-align:left;">
gene
</td>
<td style="text-align:right;">
45.44567
</td>
<td style="text-align:right;">
45.44881
</td>
<td style="text-align:left;">
positive
</td>
<td style="text-align:left;">
MGI_C57BL6J_5610573
</td>
<td style="text-align:left;">
Gm37345
</td>
<td style="text-align:left;">
ENSEMBL:ENSMUSG00000103947
</td>
<td style="text-align:left;">
MGI:5610573
</td>
<td style="text-align:left;">
unclassified gene
</td>
<td style="text-align:left;">
predicted gene, 37345
</td>
</tr>
<tr>
<td style="text-align:left;">
3
</td>
<td style="text-align:left;">
pseudogene
</td>
<td style="text-align:right;">
45.69054
</td>
<td style="text-align:right;">
45.69090
</td>
<td style="text-align:left;">
negative
</td>
<td style="text-align:left;">
MGI_C57BL6J_5611063
</td>
<td style="text-align:left;">
Gm37835
</td>
<td style="text-align:left;">
ENSEMBL:ENSMUSG00000103534
</td>
<td style="text-align:left;">
MGI:5611063
</td>
<td style="text-align:left;">
pseudogene
</td>
<td style="text-align:left;">
predicted gene, 37835
</td>
</tr>
<tr>
<td style="text-align:left;">
3
</td>
<td style="text-align:left;">
gene
</td>
<td style="text-align:right;">
45.77542
</td>
<td style="text-align:right;">
45.77589
</td>
<td style="text-align:left;">
negative
</td>
<td style="text-align:left;">
MGI_C57BL6J_5611394
</td>
<td style="text-align:left;">
Gm38166
</td>
<td style="text-align:left;">
ENSEMBL:ENSMUSG00000102190
</td>
<td style="text-align:left;">
MGI:5611394
</td>
<td style="text-align:left;">
unclassified gene
</td>
<td style="text-align:left;">
predicted gene, 38166
</td>
</tr>
<tr>
<td style="text-align:left;">
3
</td>
<td style="text-align:left;">
pseudogene
</td>
<td style="text-align:right;">
45.77587
</td>
<td style="text-align:right;">
45.86058
</td>
<td style="text-align:left;">
negative
</td>
<td style="text-align:left;">
MGI_C57BL6J_5591499
</td>
<td style="text-align:left;">
Gm32340
</td>
<td style="text-align:left;">
NCBI_Gene:102634856,ENSEMBL:ENSMUSG00000104491
</td>
<td style="text-align:left;">
MGI:5591499
</td>
<td style="text-align:left;">
pseudogene
</td>
<td style="text-align:left;">
predicted gene, 32340
</td>
</tr>
<tr>
<td style="text-align:left;">
3
</td>
<td style="text-align:left;">
gene
</td>
<td style="text-align:right;">
45.81774
</td>
<td style="text-align:right;">
45.81790
</td>
<td style="text-align:left;">
positive
</td>
<td style="text-align:left;">
MGI_C57BL6J_5455573
</td>
<td style="text-align:left;">
Gm25796
</td>
<td style="text-align:left;">
ENSEMBL:ENSMUSG00000064579
</td>
<td style="text-align:left;">
MGI:5455573
</td>
<td style="text-align:left;">
snRNA gene
</td>
<td style="text-align:left;">
predicted gene, 25796
</td>
</tr>
<tr>
<td style="text-align:left;">
3
</td>
<td style="text-align:left;">
pseudogene
</td>
<td style="text-align:right;">
45.86121
</td>
<td style="text-align:right;">
45.86995
</td>
<td style="text-align:left;">
negative
</td>
<td style="text-align:left;">
MGI_C57BL6J_5591575
</td>
<td style="text-align:left;">
Gm32416
</td>
<td style="text-align:left;">
NCBI_Gene:102634958
</td>
<td style="text-align:left;">
MGI:5591575
</td>
<td style="text-align:left;">
pseudogene
</td>
<td style="text-align:left;">
predicted gene, 32416
</td>
</tr>
<tr>
<td style="text-align:left;">
3
</td>
<td style="text-align:left;">
gene
</td>
<td style="text-align:right;">
45.97858
</td>
<td style="text-align:right;">
45.97870
</td>
<td style="text-align:left;">
positive
</td>
<td style="text-align:left;">
MGI_C57BL6J_5453114
</td>
<td style="text-align:left;">
Gm23337
</td>
<td style="text-align:left;">
ENSEMBL:ENSMUSG00000089584
</td>
<td style="text-align:left;">
MGI:5453114
</td>
<td style="text-align:left;">
snRNA gene
</td>
<td style="text-align:left;">
predicted gene, 23337
</td>
</tr>
<tr>
<td style="text-align:left;">
3
</td>
<td style="text-align:left;">
gene
</td>
<td style="text-align:right;">
46.28983
</td>
<td style="text-align:right;">
46.28996
</td>
<td style="text-align:left;">
positive
</td>
<td style="text-align:left;">
MGI_C57BL6J_5455858
</td>
<td style="text-align:left;">
Gm26081
</td>
<td style="text-align:left;">
ENSEMBL:ENSMUSG00000077256
</td>
<td style="text-align:left;">
MGI:5455858
</td>
<td style="text-align:left;">
snoRNA gene
</td>
<td style="text-align:left;">
predicted gene, 26081
</td>
</tr>
<tr>
<td style="text-align:left;">
3
</td>
<td style="text-align:left;">
pseudogene
</td>
<td style="text-align:right;">
46.41340
</td>
<td style="text-align:right;">
46.41412
</td>
<td style="text-align:left;">
positive
</td>
<td style="text-align:left;">
MGI_C57BL6J_5011028
</td>
<td style="text-align:left;">
Gm18843
</td>
<td style="text-align:left;">
NCBI_Gene:100417811,ENSEMBL:ENSMUSG00000104511
</td>
<td style="text-align:left;">
MGI:5011028
</td>
<td style="text-align:left;">
pseudogene
</td>
<td style="text-align:left;">
predicted gene, 18843
</td>
</tr>
<tr>
<td style="text-align:left;">
3
</td>
<td style="text-align:left;">
gene
</td>
<td style="text-align:right;">
46.44085
</td>
<td style="text-align:right;">
46.44822
</td>
<td style="text-align:left;">
negative
</td>
<td style="text-align:left;">
MGI_C57BL6J_3643087
</td>
<td style="text-align:left;">
Pabpc4l
</td>
<td style="text-align:left;">
NCBI_Gene:241989,ENSEMBL:ENSMUSG00000090919
</td>
<td style="text-align:left;">
MGI:3643087
</td>
<td style="text-align:left;">
protein coding gene
</td>
<td style="text-align:left;">
poly(A) binding protein, cytoplasmic 4-like
</td>
</tr>
<tr>
<td style="text-align:left;">
3
</td>
<td style="text-align:left;">
pseudogene
</td>
<td style="text-align:right;">
46.62112
</td>
<td style="text-align:right;">
46.62136
</td>
<td style="text-align:left;">
positive
</td>
<td style="text-align:left;">
MGI_C57BL6J_5610673
</td>
<td style="text-align:left;">
Gm37445
</td>
<td style="text-align:left;">
ENSEMBL:ENSMUSG00000102589
</td>
<td style="text-align:left;">
MGI:5610673
</td>
<td style="text-align:left;">
pseudogene
</td>
<td style="text-align:left;">
predicted gene, 37445
</td>
</tr>
<tr>
<td style="text-align:left;">
3
</td>
<td style="text-align:left;">
gene
</td>
<td style="text-align:right;">
46.90067
</td>
<td style="text-align:right;">
46.90078
</td>
<td style="text-align:left;">
negative
</td>
<td style="text-align:left;">
MGI_C57BL6J_5531112
</td>
<td style="text-align:left;">
Mir6379
</td>
<td style="text-align:left;">
miRBase:MI0021910,NCBI_Gene:102465199,ENSEMBL:ENSMUSG00000098417
</td>
<td style="text-align:left;">
MGI:5531112
</td>
<td style="text-align:left;">
miRNA gene
</td>
<td style="text-align:left;">
microRNA 6379
</td>
</tr>
<tr>
<td style="text-align:left;">
3
</td>
<td style="text-align:left;">
pseudogene
</td>
<td style="text-align:right;">
47.04911
</td>
<td style="text-align:right;">
47.05178
</td>
<td style="text-align:left;">
negative
</td>
<td style="text-align:left;">
MGI_C57BL6J_3641761
</td>
<td style="text-align:left;">
Gm10356
</td>
<td style="text-align:left;">
NCBI_Gene:545517,ENSEMBL:ENSMUSG00000072387
</td>
<td style="text-align:left;">
MGI:3641761
</td>
<td style="text-align:left;">
pseudogene
</td>
<td style="text-align:left;">
predicted gene 10356
</td>
</tr>
<tr>
<td style="text-align:left;">
3
</td>
<td style="text-align:left;">
pseudogene
</td>
<td style="text-align:right;">
47.08453
</td>
<td style="text-align:right;">
47.08575
</td>
<td style="text-align:left;">
negative
</td>
<td style="text-align:left;">
MGI_C57BL6J_3646161
</td>
<td style="text-align:left;">
Gm5274
</td>
<td style="text-align:left;">
NCBI_Gene:383849,ENSEMBL:ENSMUSG00000097985
</td>
<td style="text-align:left;">
MGI:3646161
</td>
<td style="text-align:left;">
pseudogene
</td>
<td style="text-align:left;">
predicted gene 5274
</td>
</tr>
<tr>
<td style="text-align:left;">
3
</td>
<td style="text-align:left;">
gene
</td>
<td style="text-align:right;">
47.20063
</td>
<td style="text-align:right;">
47.20076
</td>
<td style="text-align:left;">
positive
</td>
<td style="text-align:left;">
MGI_C57BL6J_5453668
</td>
<td style="text-align:left;">
Gm23891
</td>
<td style="text-align:left;">
ENSEMBL:ENSMUSG00000088739
</td>
<td style="text-align:left;">
MGI:5453668
</td>
<td style="text-align:left;">
snoRNA gene
</td>
<td style="text-align:left;">
predicted gene, 23891
</td>
</tr>
<tr>
<td style="text-align:left;">
3
</td>
<td style="text-align:left;">
pseudogene
</td>
<td style="text-align:right;">
47.23050
</td>
<td style="text-align:right;">
47.23100
</td>
<td style="text-align:left;">
negative
</td>
<td style="text-align:left;">
MGI_C57BL6J_3643228
</td>
<td style="text-align:left;">
Rpl23a-ps5
</td>
<td style="text-align:left;">
NCBI_Gene:383850,ENSEMBL:ENSMUSG00000102220
</td>
<td style="text-align:left;">
MGI:3643228
</td>
<td style="text-align:left;">
pseudogene
</td>
<td style="text-align:left;">
ribosomal protein L23A, pseudogene 5
</td>
</tr>
<tr>
<td style="text-align:left;">
3
</td>
<td style="text-align:left;">
gene
</td>
<td style="text-align:right;">
47.26308
</td>
<td style="text-align:right;">
47.26580
</td>
<td style="text-align:left;">
negative
</td>
<td style="text-align:left;">
MGI_C57BL6J_3809922
</td>
<td style="text-align:left;">
Gm10730
</td>
<td style="text-align:left;">
ENSEMBL:ENSMUSG00000103491
</td>
<td style="text-align:left;">
MGI:3809922
</td>
<td style="text-align:left;">
unclassified gene
</td>
<td style="text-align:left;">
predicted gene 10730
</td>
</tr>
<tr>
<td style="text-align:left;">
3
</td>
<td style="text-align:left;">
pseudogene
</td>
<td style="text-align:right;">
47.45961
</td>
<td style="text-align:right;">
47.46034
</td>
<td style="text-align:left;">
negative
</td>
<td style="text-align:left;">
MGI_C57BL6J_3780399
</td>
<td style="text-align:left;">
Gm2229
</td>
<td style="text-align:left;">
NCBI_Gene:100039426,ENSEMBL:ENSMUSG00000103798
</td>
<td style="text-align:left;">
MGI:3780399
</td>
<td style="text-align:left;">
pseudogene
</td>
<td style="text-align:left;">
predicted gene 2229
</td>
</tr>
<tr>
<td style="text-align:left;">
3
</td>
<td style="text-align:left;">
pseudogene
</td>
<td style="text-align:right;">
48.02601
</td>
<td style="text-align:right;">
48.02676
</td>
<td style="text-align:left;">
negative
</td>
<td style="text-align:left;">
MGI_C57BL6J_3648898
</td>
<td style="text-align:left;">
Gm7977
</td>
<td style="text-align:left;">
NCBI_Gene:666202,ENSEMBL:ENSMUSG00000103257
</td>
<td style="text-align:left;">
MGI:3648898
</td>
<td style="text-align:left;">
pseudogene
</td>
<td style="text-align:left;">
predicted gene 7977
</td>
</tr>
<tr>
<td style="text-align:left;">
3
</td>
<td style="text-align:left;">
gene
</td>
<td style="text-align:right;">
48.07834
</td>
<td style="text-align:right;">
48.07845
</td>
<td style="text-align:left;">
positive
</td>
<td style="text-align:left;">
MGI_C57BL6J_5610523
</td>
<td style="text-align:left;">
Gm37295
</td>
<td style="text-align:left;">
ENSEMBL:ENSMUSG00000104110
</td>
<td style="text-align:left;">
MGI:5610523
</td>
<td style="text-align:left;">
unclassified gene
</td>
<td style="text-align:left;">
predicted gene, 37295
</td>
</tr>
<tr>
<td style="text-align:left;">
3
</td>
<td style="text-align:left;">
gene
</td>
<td style="text-align:right;">
48.54642
</td>
<td style="text-align:right;">
48.57947
</td>
<td style="text-align:left;">
positive
</td>
<td style="text-align:left;">
MGI_C57BL6J_5826472
</td>
<td style="text-align:left;">
Gm46835
</td>
<td style="text-align:left;">
NCBI_Gene:108168915
</td>
<td style="text-align:left;">
MGI:5826472
</td>
<td style="text-align:left;">
lncRNA gene
</td>
<td style="text-align:left;">
predicted gene, 46835
</td>
</tr>
<tr>
<td style="text-align:left;">
3
</td>
<td style="text-align:left;">
gene
</td>
<td style="text-align:right;">
48.60573
</td>
<td style="text-align:right;">
48.60910
</td>
<td style="text-align:left;">
negative
</td>
<td style="text-align:left;">
MGI_C57BL6J_1913582
</td>
<td style="text-align:left;">
1700018B24Rik
</td>
<td style="text-align:left;">
NCBI_Gene:66332,ENSEMBL:ENSMUSG00000037910
</td>
<td style="text-align:left;">
MGI:1913582
</td>
<td style="text-align:left;">
lncRNA gene
</td>
<td style="text-align:left;">
RIKEN cDNA 1700018B24 gene
</td>
</tr>
<tr>
<td style="text-align:left;">
3
</td>
<td style="text-align:left;">
gene
</td>
<td style="text-align:right;">
48.74851
</td>
<td style="text-align:right;">
48.76095
</td>
<td style="text-align:left;">
negative
</td>
<td style="text-align:left;">
MGI_C57BL6J_5610418
</td>
<td style="text-align:left;">
Gm37190
</td>
<td style="text-align:left;">
ENSEMBL:ENSMUSG00000102181
</td>
<td style="text-align:left;">
MGI:5610418
</td>
<td style="text-align:left;">
lncRNA gene
</td>
<td style="text-align:left;">
predicted gene, 37190
</td>
</tr>
<tr>
<td style="text-align:left;">
3
</td>
<td style="text-align:left;">
gene
</td>
<td style="text-align:right;">
48.95472
</td>
<td style="text-align:right;">
48.95488
</td>
<td style="text-align:left;">
positive
</td>
<td style="text-align:left;">
MGI_C57BL6J_5453288
</td>
<td style="text-align:left;">
Gm23511
</td>
<td style="text-align:left;">
ENSEMBL:ENSMUSG00000065018
</td>
<td style="text-align:left;">
MGI:5453288
</td>
<td style="text-align:left;">
snRNA gene
</td>
<td style="text-align:left;">
predicted gene, 23511
</td>
</tr>
<tr>
<td style="text-align:left;">
3
</td>
<td style="text-align:left;">
gene
</td>
<td style="text-align:right;">
49.06237
</td>
<td style="text-align:right;">
49.09636
</td>
<td style="text-align:left;">
positive
</td>
<td style="text-align:left;">
MGI_C57BL6J_5590574
</td>
<td style="text-align:left;">
Gm31415
</td>
<td style="text-align:left;">
NCBI_Gene:102633632,ENSEMBL:ENSMUSG00000102263
</td>
<td style="text-align:left;">
MGI:5590574
</td>
<td style="text-align:left;">
lncRNA gene
</td>
<td style="text-align:left;">
predicted gene, 31415
</td>
</tr>
<tr>
<td style="text-align:left;">
3
</td>
<td style="text-align:left;">
gene
</td>
<td style="text-align:right;">
49.13746
</td>
<td style="text-align:right;">
49.13853
</td>
<td style="text-align:left;">
positive
</td>
<td style="text-align:left;">
MGI_C57BL6J_5611047
</td>
<td style="text-align:left;">
Gm37819
</td>
<td style="text-align:left;">
ENSEMBL:ENSMUSG00000104186
</td>
<td style="text-align:left;">
MGI:5611047
</td>
<td style="text-align:left;">
unclassified gene
</td>
<td style="text-align:left;">
predicted gene, 37819
</td>
</tr>
<tr>
<td style="text-align:left;">
3
</td>
<td style="text-align:left;">
pseudogene
</td>
<td style="text-align:right;">
49.20771
</td>
<td style="text-align:right;">
49.20807
</td>
<td style="text-align:left;">
negative
</td>
<td style="text-align:left;">
MGI_C57BL6J_5610912
</td>
<td style="text-align:left;">
Gm37684
</td>
<td style="text-align:left;">
ENSEMBL:ENSMUSG00000103027
</td>
<td style="text-align:left;">
MGI:5610912
</td>
<td style="text-align:left;">
pseudogene
</td>
<td style="text-align:left;">
predicted gene, 37684
</td>
</tr>
<tr>
<td style="text-align:left;">
3
</td>
<td style="text-align:left;">
gene
</td>
<td style="text-align:right;">
49.32568
</td>
<td style="text-align:right;">
49.36736
</td>
<td style="text-align:left;">
positive
</td>
<td style="text-align:left;">
MGI_C57BL6J_5295664
</td>
<td style="text-align:left;">
Gm20557
</td>
<td style="text-align:left;">
NCBI_Gene:329633,ENSEMBL:ENSMUSG00000102679
</td>
<td style="text-align:left;">
MGI:5295664
</td>
<td style="text-align:left;">
lncRNA gene
</td>
<td style="text-align:left;">
predicted gene, 20557
</td>
</tr>
<tr>
<td style="text-align:left;">
3
</td>
<td style="text-align:left;">
pseudogene
</td>
<td style="text-align:right;">
49.38115
</td>
<td style="text-align:right;">
49.38285
</td>
<td style="text-align:left;">
negative
</td>
<td style="text-align:left;">
MGI_C57BL6J_3643624
</td>
<td style="text-align:left;">
Gm5846
</td>
<td style="text-align:left;">
NCBI_Gene:545519,ENSEMBL:ENSMUSG00000104309
</td>
<td style="text-align:left;">
MGI:3643624
</td>
<td style="text-align:left;">
pseudogene
</td>
<td style="text-align:left;">
predicted gene 5846
</td>
</tr>
<tr>
<td style="text-align:left;">
3
</td>
<td style="text-align:left;">
pseudogene
</td>
<td style="text-align:right;">
49.40082
</td>
<td style="text-align:right;">
49.46744
</td>
<td style="text-align:left;">
negative
</td>
<td style="text-align:left;">
MGI_C57BL6J_3780091
</td>
<td style="text-align:left;">
Gm9683
</td>
<td style="text-align:left;">
NCBI_Gene:676568
</td>
<td style="text-align:left;">
MGI:3780091
</td>
<td style="text-align:left;">
pseudogene
</td>
<td style="text-align:left;">
predicted gene 9683
</td>
</tr>
<tr>
<td style="text-align:left;">
3
</td>
<td style="text-align:left;">
gene
</td>
<td style="text-align:right;">
49.49135
</td>
<td style="text-align:right;">
49.49148
</td>
<td style="text-align:left;">
positive
</td>
<td style="text-align:left;">
MGI_C57BL6J_5452257
</td>
<td style="text-align:left;">
Gm22480
</td>
<td style="text-align:left;">
ENSEMBL:ENSMUSG00000084517
</td>
<td style="text-align:left;">
MGI:5452257
</td>
<td style="text-align:left;">
snoRNA gene
</td>
<td style="text-align:left;">
predicted gene, 22480
</td>
</tr>
<tr>
<td style="text-align:left;">
3
</td>
<td style="text-align:left;">
pseudogene
</td>
<td style="text-align:right;">
49.53075
</td>
<td style="text-align:right;">
49.53131
</td>
<td style="text-align:left;">
positive
</td>
<td style="text-align:left;">
MGI_C57BL6J_5610652
</td>
<td style="text-align:left;">
Gm37424
</td>
<td style="text-align:left;">
ENSEMBL:ENSMUSG00000104437
</td>
<td style="text-align:left;">
MGI:5610652
</td>
<td style="text-align:left;">
pseudogene
</td>
<td style="text-align:left;">
predicted gene, 37424
</td>
</tr>
<tr>
<td style="text-align:left;">
3
</td>
<td style="text-align:left;">
pseudogene
</td>
<td style="text-align:right;">
49.60356
</td>
<td style="text-align:right;">
49.60388
</td>
<td style="text-align:left;">
positive
</td>
<td style="text-align:left;">
MGI_C57BL6J_5611489
</td>
<td style="text-align:left;">
Gm38261
</td>
<td style="text-align:left;">
ENSEMBL:ENSMUSG00000102731
</td>
<td style="text-align:left;">
MGI:5611489
</td>
<td style="text-align:left;">
pseudogene
</td>
<td style="text-align:left;">
predicted gene, 38261
</td>
</tr>
<tr>
<td style="text-align:left;">
3
</td>
<td style="text-align:left;">
pseudogene
</td>
<td style="text-align:right;">
49.60423
</td>
<td style="text-align:right;">
49.60447
</td>
<td style="text-align:left;">
positive
</td>
<td style="text-align:left;">
MGI_C57BL6J_5611316
</td>
<td style="text-align:left;">
Gm38088
</td>
<td style="text-align:left;">
ENSEMBL:ENSMUSG00000103249
</td>
<td style="text-align:left;">
MGI:5611316
</td>
<td style="text-align:left;">
pseudogene
</td>
<td style="text-align:left;">
predicted gene, 38088
</td>
</tr>
<tr>
<td style="text-align:left;">
3
</td>
<td style="text-align:left;">
pseudogene
</td>
<td style="text-align:right;">
49.63244
</td>
<td style="text-align:right;">
49.63297
</td>
<td style="text-align:left;">
negative
</td>
<td style="text-align:left;">
MGI_C57BL6J_3780498
</td>
<td style="text-align:left;">
Gm2328
</td>
<td style="text-align:left;">
NCBI_Gene:100039587,ENSEMBL:ENSMUSG00000104134
</td>
<td style="text-align:left;">
MGI:3780498
</td>
<td style="text-align:left;">
pseudogene
</td>
<td style="text-align:left;">
predicted gene 2328
</td>
</tr>
<tr>
<td style="text-align:left;">
3
</td>
<td style="text-align:left;">
pseudogene
</td>
<td style="text-align:right;">
49.64348
</td>
<td style="text-align:right;">
49.64413
</td>
<td style="text-align:left;">
positive
</td>
<td style="text-align:left;">
MGI_C57BL6J_5588900
</td>
<td style="text-align:left;">
Gm29741
</td>
<td style="text-align:left;">
NCBI_Gene:101055903,ENSEMBL:ENSMUSG00000103568
</td>
<td style="text-align:left;">
MGI:5588900
</td>
<td style="text-align:left;">
pseudogene
</td>
<td style="text-align:left;">
predicted gene, 29741
</td>
</tr>
<tr>
<td style="text-align:left;">
3
</td>
<td style="text-align:left;">
gene
</td>
<td style="text-align:right;">
49.65876
</td>
<td style="text-align:right;">
49.65887
</td>
<td style="text-align:left;">
positive
</td>
<td style="text-align:left;">
MGI_C57BL6J_5452873
</td>
<td style="text-align:left;">
Gm23096
</td>
<td style="text-align:left;">
ENSEMBL:ENSMUSG00000065381
</td>
<td style="text-align:left;">
MGI:5452873
</td>
<td style="text-align:left;">
snRNA gene
</td>
<td style="text-align:left;">
predicted gene, 23096
</td>
</tr>
<tr>
<td style="text-align:left;">
3
</td>
<td style="text-align:left;">
gene
</td>
<td style="text-align:right;">
49.74329
</td>
<td style="text-align:right;">
49.75738
</td>
<td style="text-align:left;">
negative
</td>
<td style="text-align:left;">
MGI_C57BL6J_1920423
</td>
<td style="text-align:left;">
Pcdh18
</td>
<td style="text-align:left;">
NCBI_Gene:73173,ENSEMBL:ENSMUSG00000037892
</td>
<td style="text-align:left;">
MGI:1920423
</td>
<td style="text-align:left;">
protein coding gene
</td>
<td style="text-align:left;">
protocadherin 18
</td>
</tr>
<tr>
<td style="text-align:left;">
3
</td>
<td style="text-align:left;">
pseudogene
</td>
<td style="text-align:right;">
49.78043
</td>
<td style="text-align:right;">
49.78089
</td>
<td style="text-align:left;">
negative
</td>
<td style="text-align:left;">
MGI_C57BL6J_5610778
</td>
<td style="text-align:left;">
Gm37550
</td>
<td style="text-align:left;">
ENSEMBL:ENSMUSG00000102943
</td>
<td style="text-align:left;">
MGI:5610778
</td>
<td style="text-align:left;">
pseudogene
</td>
<td style="text-align:left;">
predicted gene, 37550
</td>
</tr>
<tr>
<td style="text-align:left;">
3
</td>
<td style="text-align:left;">
gene
</td>
<td style="text-align:right;">
49.89253
</td>
<td style="text-align:right;">
50.49909
</td>
<td style="text-align:left;">
negative
</td>
<td style="text-align:left;">
MGI_C57BL6J_1347355
</td>
<td style="text-align:left;">
Slc7a11
</td>
<td style="text-align:left;">
NCBI_Gene:26570,ENSEMBL:ENSMUSG00000027737
</td>
<td style="text-align:left;">
MGI:1347355
</td>
<td style="text-align:left;">
protein coding gene
</td>
<td style="text-align:left;">
solute carrier family 7 (cationic amino acid transporter, y+ system), member 11
</td>
</tr>
<tr>
<td style="text-align:left;">
3
</td>
<td style="text-align:left;">
gene
</td>
<td style="text-align:right;">
50.00197
</td>
<td style="text-align:right;">
50.01078
</td>
<td style="text-align:left;">
positive
</td>
<td style="text-align:left;">
MGI_C57BL6J_5611082
</td>
<td style="text-align:left;">
Gm37854
</td>
<td style="text-align:left;">
ENSEMBL:ENSMUSG00000102892
</td>
<td style="text-align:left;">
MGI:5611082
</td>
<td style="text-align:left;">
lncRNA gene
</td>
<td style="text-align:left;">
predicted gene, 37854
</td>
</tr>
<tr>
<td style="text-align:left;">
3
</td>
<td style="text-align:left;">
gene
</td>
<td style="text-align:right;">
50.01177
</td>
<td style="text-align:right;">
50.02624
</td>
<td style="text-align:left;">
negative
</td>
<td style="text-align:left;">
MGI_C57BL6J_5622927
</td>
<td style="text-align:left;">
Gm40042
</td>
<td style="text-align:left;">
NCBI_Gene:105244434
</td>
<td style="text-align:left;">
MGI:5622927
</td>
<td style="text-align:left;">
lncRNA gene
</td>
<td style="text-align:left;">
predicted gene, 40042
</td>
</tr>
<tr>
<td style="text-align:left;">
3
</td>
<td style="text-align:left;">
pseudogene
</td>
<td style="text-align:right;">
50.02967
</td>
<td style="text-align:right;">
50.02999
</td>
<td style="text-align:left;">
positive
</td>
<td style="text-align:left;">
MGI_C57BL6J_5611054
</td>
<td style="text-align:left;">
Gm37826
</td>
<td style="text-align:left;">
ENSEMBL:ENSMUSG00000104457
</td>
<td style="text-align:left;">
MGI:5611054
</td>
<td style="text-align:left;">
pseudogene
</td>
<td style="text-align:left;">
predicted gene, 37826
</td>
</tr>
<tr>
<td style="text-align:left;">
3
</td>
<td style="text-align:left;">
pseudogene
</td>
<td style="text-align:right;">
50.16666
</td>
<td style="text-align:right;">
50.16689
</td>
<td style="text-align:left;">
negative
</td>
<td style="text-align:left;">
MGI_C57BL6J_5610920
</td>
<td style="text-align:left;">
Gm37692
</td>
<td style="text-align:left;">
ENSEMBL:ENSMUSG00000104349
</td>
<td style="text-align:left;">
MGI:5610920
</td>
<td style="text-align:left;">
pseudogene
</td>
<td style="text-align:left;">
predicted gene, 37692
</td>
</tr>
<tr>
<td style="text-align:left;">
3
</td>
<td style="text-align:left;">
pseudogene
</td>
<td style="text-align:right;">
50.25485
</td>
<td style="text-align:right;">
50.25544
</td>
<td style="text-align:left;">
positive
</td>
<td style="text-align:left;">
MGI_C57BL6J_3780515
</td>
<td style="text-align:left;">
Gm2345
</td>
<td style="text-align:left;">
NCBI_Gene:100039628,ENSEMBL:ENSMUSG00000102167
</td>
<td style="text-align:left;">
MGI:3780515
</td>
<td style="text-align:left;">
pseudogene
</td>
<td style="text-align:left;">
predicted gene 2345
</td>
</tr>
<tr>
<td style="text-align:left;">
3
</td>
<td style="text-align:left;">
gene
</td>
<td style="text-align:right;">
50.40028
</td>
<td style="text-align:right;">
50.40428
</td>
<td style="text-align:left;">
negative
</td>
<td style="text-align:left;">
MGI_C57BL6J_5610726
</td>
<td style="text-align:left;">
Gm37498
</td>
<td style="text-align:left;">
ENSEMBL:ENSMUSG00000103907
</td>
<td style="text-align:left;">
MGI:5610726
</td>
<td style="text-align:left;">
unclassified gene
</td>
<td style="text-align:left;">
predicted gene, 37498
</td>
</tr>
<tr>
<td style="text-align:left;">
3
</td>
<td style="text-align:left;">
gene
</td>
<td style="text-align:right;">
50.51640
</td>
<td style="text-align:right;">
50.60731
</td>
<td style="text-align:left;">
negative
</td>
<td style="text-align:left;">
MGI_C57BL6J_3643374
</td>
<td style="text-align:left;">
Gm6209
</td>
<td style="text-align:left;">
NCBI_Gene:621304,ENSEMBL:ENSMUSG00000102715
</td>
<td style="text-align:left;">
MGI:3643374
</td>
<td style="text-align:left;">
lncRNA gene
</td>
<td style="text-align:left;">
predicted gene 6209
</td>
</tr>
<tr>
<td style="text-align:left;">
3
</td>
<td style="text-align:left;">
gene
</td>
<td style="text-align:right;">
50.57519
</td>
<td style="text-align:right;">
50.57652
</td>
<td style="text-align:left;">
negative
</td>
<td style="text-align:left;">
MGI_C57BL6J_1923259
</td>
<td style="text-align:left;">
5830415G21Rik
</td>
<td style="text-align:left;">
ENSEMBL:ENSMUSG00000102895
</td>
<td style="text-align:left;">
MGI:1923259
</td>
<td style="text-align:left;">
unclassified gene
</td>
<td style="text-align:left;">
RIKEN cDNA 5830415G21 gene
</td>
</tr>
<tr>
<td style="text-align:left;">
3
</td>
<td style="text-align:left;">
pseudogene
</td>
<td style="text-align:right;">
50.59799
</td>
<td style="text-align:right;">
50.59850
</td>
<td style="text-align:left;">
negative
</td>
<td style="text-align:left;">
MGI_C57BL6J_5611465
</td>
<td style="text-align:left;">
Gm38237
</td>
<td style="text-align:left;">
NCBI_Gene:108168851,ENSEMBL:ENSMUSG00000103745
</td>
<td style="text-align:left;">
MGI:5611465
</td>
<td style="text-align:left;">
pseudogene
</td>
<td style="text-align:left;">
predicted gene, 38237
</td>
</tr>
<tr>
<td style="text-align:left;">
3
</td>
<td style="text-align:left;">
gene
</td>
<td style="text-align:right;">
50.62435
</td>
<td style="text-align:right;">
50.62841
</td>
<td style="text-align:left;">
negative
</td>
<td style="text-align:left;">
MGI_C57BL6J_5610427
</td>
<td style="text-align:left;">
Gm37199
</td>
<td style="text-align:left;">
ENSEMBL:ENSMUSG00000104068
</td>
<td style="text-align:left;">
MGI:5610427
</td>
<td style="text-align:left;">
unclassified gene
</td>
<td style="text-align:left;">
predicted gene, 37199
</td>
</tr>
<tr>
<td style="text-align:left;">
3
</td>
<td style="text-align:left;">
gene
</td>
<td style="text-align:right;">
50.67970
</td>
<td style="text-align:right;">
50.68340
</td>
<td style="text-align:left;">
negative
</td>
<td style="text-align:left;">
MGI_C57BL6J_5610689
</td>
<td style="text-align:left;">
Gm37461
</td>
<td style="text-align:left;">
ENSEMBL:ENSMUSG00000104281
</td>
<td style="text-align:left;">
MGI:5610689
</td>
<td style="text-align:left;">
unclassified gene
</td>
<td style="text-align:left;">
predicted gene, 37461
</td>
</tr>
<tr>
<td style="text-align:left;">
3
</td>
<td style="text-align:left;">
gene
</td>
<td style="text-align:right;">
50.70690
</td>
<td style="text-align:right;">
50.71189
</td>
<td style="text-align:left;">
negative
</td>
<td style="text-align:left;">
MGI_C57BL6J_5622928
</td>
<td style="text-align:left;">
Gm40043
</td>
<td style="text-align:left;">
NCBI_Gene:105244435
</td>
<td style="text-align:left;">
MGI:5622928
</td>
<td style="text-align:left;">
lncRNA gene
</td>
<td style="text-align:left;">
predicted gene, 40043
</td>
</tr>
<tr>
<td style="text-align:left;">
3
</td>
<td style="text-align:left;">
gene
</td>
<td style="text-align:right;">
50.83858
</td>
<td style="text-align:right;">
50.83872
</td>
<td style="text-align:left;">
negative
</td>
<td style="text-align:left;">
MGI_C57BL6J_5454280
</td>
<td style="text-align:left;">
Gm24503
</td>
<td style="text-align:left;">
ENSEMBL:ENSMUSG00000089079
</td>
<td style="text-align:left;">
MGI:5454280
</td>
<td style="text-align:left;">
snoRNA gene
</td>
<td style="text-align:left;">
predicted gene, 24503
</td>
</tr>
<tr>
<td style="text-align:left;">
3
</td>
<td style="text-align:left;">
gene
</td>
<td style="text-align:right;">
50.88324
</td>
<td style="text-align:right;">
50.88889
</td>
<td style="text-align:left;">
negative
</td>
<td style="text-align:left;">
MGI_C57BL6J_5610776
</td>
<td style="text-align:left;">
Gm37548
</td>
<td style="text-align:left;">
NCBI_Gene:105244436,ENSEMBL:ENSMUSG00000103012
</td>
<td style="text-align:left;">
MGI:5610776
</td>
<td style="text-align:left;">
lncRNA gene
</td>
<td style="text-align:left;">
predicted gene, 37548
</td>
</tr>
<tr>
<td style="text-align:left;">
3
</td>
<td style="text-align:left;">
gene
</td>
<td style="text-align:right;">
50.91889
</td>
<td style="text-align:right;">
50.92153
</td>
<td style="text-align:left;">
negative
</td>
<td style="text-align:left;">
MGI_C57BL6J_5611071
</td>
<td style="text-align:left;">
Gm37843
</td>
<td style="text-align:left;">
ENSEMBL:ENSMUSG00000102614
</td>
<td style="text-align:left;">
MGI:5611071
</td>
<td style="text-align:left;">
unclassified gene
</td>
<td style="text-align:left;">
predicted gene, 37843
</td>
</tr>
<tr>
<td style="text-align:left;">
3
</td>
<td style="text-align:left;">
gene
</td>
<td style="text-align:right;">
50.96656
</td>
<td style="text-align:right;">
50.96677
</td>
<td style="text-align:left;">
positive
</td>
<td style="text-align:left;">
MGI_C57BL6J_2139603
</td>
<td style="text-align:left;">
AA672641
</td>
<td style="text-align:left;">
NA
</td>
<td style="text-align:left;">
NA
</td>
<td style="text-align:left;">
unclassified gene
</td>
<td style="text-align:left;">
expressed sequence AA672641
</td>
</tr>
<tr>
<td style="text-align:left;">
3
</td>
<td style="text-align:left;">
gene
</td>
<td style="text-align:right;">
50.97754
</td>
<td style="text-align:right;">
50.97881
</td>
<td style="text-align:left;">
negative
</td>
<td style="text-align:left;">
MGI_C57BL6J_5610437
</td>
<td style="text-align:left;">
Gm37209
</td>
<td style="text-align:left;">
ENSEMBL:ENSMUSG00000103652
</td>
<td style="text-align:left;">
MGI:5610437
</td>
<td style="text-align:left;">
unclassified gene
</td>
<td style="text-align:left;">
predicted gene, 37209
</td>
</tr>
<tr>
<td style="text-align:left;">
3
</td>
<td style="text-align:left;">
gene
</td>
<td style="text-align:right;">
51.03335
</td>
<td style="text-align:right;">
51.04284
</td>
<td style="text-align:left;">
negative
</td>
<td style="text-align:left;">
MGI_C57BL6J_5594915
</td>
<td style="text-align:left;">
Gm35756
</td>
<td style="text-align:left;">
NCBI_Gene:102639442
</td>
<td style="text-align:left;">
MGI:5594915
</td>
<td style="text-align:left;">
lncRNA gene
</td>
<td style="text-align:left;">
predicted gene, 35756
</td>
</tr>
<tr>
<td style="text-align:left;">
3
</td>
<td style="text-align:left;">
gene
</td>
<td style="text-align:right;">
51.05891
</td>
<td style="text-align:right;">
51.08934
</td>
<td style="text-align:left;">
negative
</td>
<td style="text-align:left;">
MGI_C57BL6J_5579936
</td>
<td style="text-align:left;">
Gm29230
</td>
<td style="text-align:left;">
NCBI_Gene:102639521,ENSEMBL:ENSMUSG00000102067
</td>
<td style="text-align:left;">
MGI:5579936
</td>
<td style="text-align:left;">
lncRNA gene
</td>
<td style="text-align:left;">
predicted gene 29230
</td>
</tr>
<tr>
<td style="text-align:left;">
3
</td>
<td style="text-align:left;">
gene
</td>
<td style="text-align:right;">
51.10366
</td>
<td style="text-align:right;">
51.10517
</td>
<td style="text-align:left;">
positive
</td>
<td style="text-align:left;">
MGI_C57BL6J_5611474
</td>
<td style="text-align:left;">
Gm38246
</td>
<td style="text-align:left;">
ENSEMBL:ENSMUSG00000103464
</td>
<td style="text-align:left;">
MGI:5611474
</td>
<td style="text-align:left;">
lncRNA gene
</td>
<td style="text-align:left;">
predicted gene, 38246
</td>
</tr>
<tr>
<td style="text-align:left;">
3
</td>
<td style="text-align:left;">
gene
</td>
<td style="text-align:right;">
51.12669
</td>
<td style="text-align:right;">
51.12894
</td>
<td style="text-align:left;">
negative
</td>
<td style="text-align:left;">
MGI_C57BL6J_5595050
</td>
<td style="text-align:left;">
Gm35891
</td>
<td style="text-align:left;">
NCBI_Gene:102639626
</td>
<td style="text-align:left;">
MGI:5595050
</td>
<td style="text-align:left;">
lncRNA gene
</td>
<td style="text-align:left;">
predicted gene, 35891
</td>
</tr>
<tr>
<td style="text-align:left;">
3
</td>
<td style="text-align:left;">
gene
</td>
<td style="text-align:right;">
51.21847
</td>
<td style="text-align:right;">
51.22033
</td>
<td style="text-align:left;">
negative
</td>
<td style="text-align:left;">
MGI_C57BL6J_5622929
</td>
<td style="text-align:left;">
Gm40044
</td>
<td style="text-align:left;">
NCBI_Gene:105244437
</td>
<td style="text-align:left;">
MGI:5622929
</td>
<td style="text-align:left;">
lncRNA gene
</td>
<td style="text-align:left;">
predicted gene, 40044
</td>
</tr>
<tr>
<td style="text-align:left;">
3
</td>
<td style="text-align:left;">
gene
</td>
<td style="text-align:right;">
51.22445
</td>
<td style="text-align:right;">
51.25165
</td>
<td style="text-align:left;">
positive
</td>
<td style="text-align:left;">
MGI_C57BL6J_109382
</td>
<td style="text-align:left;">
Noct
</td>
<td style="text-align:left;">
NCBI_Gene:12457,ENSEMBL:ENSMUSG00000023087
</td>
<td style="text-align:left;">
MGI:109382
</td>
<td style="text-align:left;">
protein coding gene
</td>
<td style="text-align:left;">
nocturnin
</td>
</tr>
<tr>
<td style="text-align:left;">
3
</td>
<td style="text-align:left;">
gene
</td>
<td style="text-align:right;">
51.23192
</td>
<td style="text-align:right;">
51.23438
</td>
<td style="text-align:left;">
positive
</td>
<td style="text-align:left;">
MGI_C57BL6J_5611585
</td>
<td style="text-align:left;">
Gm38357
</td>
<td style="text-align:left;">
ENSEMBL:ENSMUSG00000103313
</td>
<td style="text-align:left;">
MGI:5611585
</td>
<td style="text-align:left;">
unclassified gene
</td>
<td style="text-align:left;">
predicted gene, 38357
</td>
</tr>
<tr>
<td style="text-align:left;">
3
</td>
<td style="text-align:left;">
gene
</td>
<td style="text-align:right;">
51.25272
</td>
<td style="text-align:right;">
51.34092
</td>
<td style="text-align:left;">
negative
</td>
<td style="text-align:left;">
MGI_C57BL6J_1916507
</td>
<td style="text-align:left;">
Elf2
</td>
<td style="text-align:left;">
NCBI_Gene:69257,ENSEMBL:ENSMUSG00000037174
</td>
<td style="text-align:left;">
MGI:1916507
</td>
<td style="text-align:left;">
protein coding gene
</td>
<td style="text-align:left;">
E74-like factor 2
</td>
</tr>
<tr>
<td style="text-align:left;">
3
</td>
<td style="text-align:left;">
gene
</td>
<td style="text-align:right;">
51.27676
</td>
<td style="text-align:right;">
51.27841
</td>
<td style="text-align:left;">
positive
</td>
<td style="text-align:left;">
MGI_C57BL6J_1914996
</td>
<td style="text-align:left;">
4930577N17Rik
</td>
<td style="text-align:left;">
NCBI_Gene:67746,ENSEMBL:ENSMUSG00000087440
</td>
<td style="text-align:left;">
MGI:1914996
</td>
<td style="text-align:left;">
lncRNA gene
</td>
<td style="text-align:left;">
RIKEN cDNA 4930577N17 gene
</td>
</tr>
<tr>
<td style="text-align:left;">
3
</td>
<td style="text-align:left;">
gene
</td>
<td style="text-align:right;">
51.29718
</td>
<td style="text-align:right;">
51.29724
</td>
<td style="text-align:left;">
positive
</td>
<td style="text-align:left;">
MGI_C57BL6J_5454059
</td>
<td style="text-align:left;">
Gm24282
</td>
<td style="text-align:left;">
ENSEMBL:ENSMUSG00000077420
</td>
<td style="text-align:left;">
MGI:5454059
</td>
<td style="text-align:left;">
miRNA gene
</td>
<td style="text-align:left;">
predicted gene, 24282
</td>
</tr>
<tr>
<td style="text-align:left;">
3
</td>
<td style="text-align:left;">
gene
</td>
<td style="text-align:right;">
51.30249
</td>
<td style="text-align:right;">
51.30290
</td>
<td style="text-align:left;">
negative
</td>
<td style="text-align:left;">
MGI_C57BL6J_5610627
</td>
<td style="text-align:left;">
Gm37399
</td>
<td style="text-align:left;">
ENSEMBL:ENSMUSG00000104200
</td>
<td style="text-align:left;">
MGI:5610627
</td>
<td style="text-align:left;">
unclassified gene
</td>
<td style="text-align:left;">
predicted gene, 37399
</td>
</tr>
<tr>
<td style="text-align:left;">
3
</td>
<td style="text-align:left;">
gene
</td>
<td style="text-align:right;">
51.35835
</td>
<td style="text-align:right;">
51.35842
</td>
<td style="text-align:left;">
positive
</td>
<td style="text-align:left;">
MGI_C57BL6J_4414019
</td>
<td style="text-align:left;">
n-TPtgg7
</td>
<td style="text-align:left;">
NCBI_Gene:102467437
</td>
<td style="text-align:left;">
MGI:4414019
</td>
<td style="text-align:left;">
tRNA gene
</td>
<td style="text-align:left;">
nuclear encoded tRNA proline 7 (anticodon TGG)
</td>
</tr>
<tr>
<td style="text-align:left;">
3
</td>
<td style="text-align:left;">
gene
</td>
<td style="text-align:right;">
51.35941
</td>
<td style="text-align:right;">
51.35948
</td>
<td style="text-align:left;">
positive
</td>
<td style="text-align:left;">
MGI_C57BL6J_4414007
</td>
<td style="text-align:left;">
n-TPagg5
</td>
<td style="text-align:left;">
NCBI_Gene:102467366
</td>
<td style="text-align:left;">
MGI:4414007
</td>
<td style="text-align:left;">
tRNA gene
</td>
<td style="text-align:left;">
nuclear encoded tRNA proline 5 (anticodon AGG)
</td>
</tr>
<tr>
<td style="text-align:left;">
3
</td>
<td style="text-align:left;">
pseudogene
</td>
<td style="text-align:right;">
51.37022
</td>
<td style="text-align:right;">
51.37061
</td>
<td style="text-align:left;">
positive
</td>
<td style="text-align:left;">
MGI_C57BL6J_5610431
</td>
<td style="text-align:left;">
Gm37203
</td>
<td style="text-align:left;">
ENSEMBL:ENSMUSG00000103654
</td>
<td style="text-align:left;">
MGI:5610431
</td>
<td style="text-align:left;">
pseudogene
</td>
<td style="text-align:left;">
predicted gene, 37203
</td>
</tr>
<tr>
<td style="text-align:left;">
3
</td>
<td style="text-align:left;">
gene
</td>
<td style="text-align:right;">
51.37802
</td>
<td style="text-align:right;">
51.38632
</td>
<td style="text-align:left;">
positive
</td>
<td style="text-align:left;">
MGI_C57BL6J_3779459
</td>
<td style="text-align:left;">
Gm5103
</td>
<td style="text-align:left;">
NCBI_Gene:329636,ENSEMBL:ENSMUSG00000102544
</td>
<td style="text-align:left;">
MGI:3779459
</td>
<td style="text-align:left;">
lncRNA gene
</td>
<td style="text-align:left;">
predicted gene 5103
</td>
</tr>
<tr>
<td style="text-align:left;">
3
</td>
<td style="text-align:left;">
gene
</td>
<td style="text-align:right;">
51.38841
</td>
<td style="text-align:right;">
51.39674
</td>
<td style="text-align:left;">
negative
</td>
<td style="text-align:left;">
MGI_C57BL6J_1914999
</td>
<td style="text-align:left;">
Mgarp
</td>
<td style="text-align:left;">
NCBI_Gene:67749,ENSEMBL:ENSMUSG00000037161
</td>
<td style="text-align:left;">
MGI:1914999
</td>
<td style="text-align:left;">
protein coding gene
</td>
<td style="text-align:left;">
mitochondria localized glutamic acid rich protein
</td>
</tr>
<tr>
<td style="text-align:left;">
3
</td>
<td style="text-align:left;">
pseudogene
</td>
<td style="text-align:right;">
51.40246
</td>
<td style="text-align:right;">
51.40320
</td>
<td style="text-align:left;">
negative
</td>
<td style="text-align:left;">
MGI_C57BL6J_3779852
</td>
<td style="text-align:left;">
Gm9442
</td>
<td style="text-align:left;">
NCBI_Gene:669130,ENSEMBL:ENSMUSG00000104241
</td>
<td style="text-align:left;">
MGI:3779852
</td>
<td style="text-align:left;">
pseudogene
</td>
<td style="text-align:left;">
predicted gene 9442
</td>
</tr>
<tr>
<td style="text-align:left;">
3
</td>
<td style="text-align:left;">
gene
</td>
<td style="text-align:right;">
51.40468
</td>
<td style="text-align:right;">
51.40899
</td>
<td style="text-align:left;">
negative
</td>
<td style="text-align:left;">
MGI_C57BL6J_1913627
</td>
<td style="text-align:left;">
Ndufc1
</td>
<td style="text-align:left;">
NCBI_Gene:66377,ENSEMBL:ENSMUSG00000037152
</td>
<td style="text-align:left;">
MGI:1913627
</td>
<td style="text-align:left;">
protein coding gene
</td>
<td style="text-align:left;">
NADH:ubiquinone oxidoreductase subunit C1
</td>
</tr>
<tr>
<td style="text-align:left;">
3
</td>
<td style="text-align:left;">
gene
</td>
<td style="text-align:right;">
51.41515
</td>
<td style="text-align:right;">
51.47651
</td>
<td style="text-align:left;">
positive
</td>
<td style="text-align:left;">
MGI_C57BL6J_1922088
</td>
<td style="text-align:left;">
Naa15
</td>
<td style="text-align:left;">
NCBI_Gene:74838,ENSEMBL:ENSMUSG00000063273
</td>
<td style="text-align:left;">
MGI:1922088
</td>
<td style="text-align:left;">
protein coding gene
</td>
<td style="text-align:left;">
N(alpha)-acetyltransferase 15, NatA auxiliary subunit
</td>
</tr>
<tr>
<td style="text-align:left;">
3
</td>
<td style="text-align:left;">
gene
</td>
<td style="text-align:right;">
51.42361
</td>
<td style="text-align:right;">
51.42372
</td>
<td style="text-align:left;">
positive
</td>
<td style="text-align:left;">
MGI_C57BL6J_5610874
</td>
<td style="text-align:left;">
Gm37646
</td>
<td style="text-align:left;">
ENSEMBL:ENSMUSG00000103191
</td>
<td style="text-align:left;">
MGI:5610874
</td>
<td style="text-align:left;">
unclassified gene
</td>
<td style="text-align:left;">
predicted gene, 37646
</td>
</tr>
<tr>
<td style="text-align:left;">
3
</td>
<td style="text-align:left;">
gene
</td>
<td style="text-align:right;">
51.48233
</td>
<td style="text-align:right;">
51.48483
</td>
<td style="text-align:left;">
negative
</td>
<td style="text-align:left;">
MGI_C57BL6J_5611388
</td>
<td style="text-align:left;">
Gm38160
</td>
<td style="text-align:left;">
ENSEMBL:ENSMUSG00000102775
</td>
<td style="text-align:left;">
MGI:5611388
</td>
<td style="text-align:left;">
unclassified gene
</td>
<td style="text-align:left;">
predicted gene, 38160
</td>
</tr>
<tr>
<td style="text-align:left;">
3
</td>
<td style="text-align:left;">
gene
</td>
<td style="text-align:right;">
51.48392
</td>
<td style="text-align:right;">
51.49623
</td>
<td style="text-align:left;">
positive
</td>
<td style="text-align:left;">
MGI_C57BL6J_1330805
</td>
<td style="text-align:left;">
Rab33b
</td>
<td style="text-align:left;">
NCBI_Gene:19338,ENSEMBL:ENSMUSG00000027739
</td>
<td style="text-align:left;">
MGI:1330805
</td>
<td style="text-align:left;">
protein coding gene
</td>
<td style="text-align:left;">
RAB33B, member RAS oncogene family
</td>
</tr>
<tr>
<td style="text-align:left;">
3
</td>
<td style="text-align:left;">
gene
</td>
<td style="text-align:right;">
51.48962
</td>
<td style="text-align:right;">
51.49282
</td>
<td style="text-align:left;">
positive
</td>
<td style="text-align:left;">
MGI_C57BL6J_5611196
</td>
<td style="text-align:left;">
Gm37968
</td>
<td style="text-align:left;">
ENSEMBL:ENSMUSG00000102570
</td>
<td style="text-align:left;">
MGI:5611196
</td>
<td style="text-align:left;">
unclassified gene
</td>
<td style="text-align:left;">
predicted gene, 37968
</td>
</tr>
<tr>
<td style="text-align:left;">
3
</td>
<td style="text-align:left;">
gene
</td>
<td style="text-align:right;">
51.51099
</td>
<td style="text-align:right;">
51.51430
</td>
<td style="text-align:left;">
positive
</td>
<td style="text-align:left;">
MGI_C57BL6J_5595167
</td>
<td style="text-align:left;">
Gm36008
</td>
<td style="text-align:left;">
NCBI_Gene:102639777
</td>
<td style="text-align:left;">
MGI:5595167
</td>
<td style="text-align:left;">
lncRNA gene
</td>
<td style="text-align:left;">
predicted gene, 36008
</td>
</tr>
<tr>
<td style="text-align:left;">
3
</td>
<td style="text-align:left;">
gene
</td>
<td style="text-align:right;">
51.51532
</td>
<td style="text-align:right;">
51.56088
</td>
<td style="text-align:left;">
negative
</td>
<td style="text-align:left;">
MGI_C57BL6J_1920501
</td>
<td style="text-align:left;">
Setd7
</td>
<td style="text-align:left;">
NCBI_Gene:73251,ENSEMBL:ENSMUSG00000037111
</td>
<td style="text-align:left;">
MGI:1920501
</td>
<td style="text-align:left;">
protein coding gene
</td>
<td style="text-align:left;">
SET domain containing (lysine methyltransferase) 7
</td>
</tr>
<tr>
<td style="text-align:left;">
3
</td>
<td style="text-align:left;">
gene
</td>
<td style="text-align:right;">
51.55754
</td>
<td style="text-align:right;">
51.55965
</td>
<td style="text-align:left;">
negative
</td>
<td style="text-align:left;">
MGI_C57BL6J_5611428
</td>
<td style="text-align:left;">
Gm38200
</td>
<td style="text-align:left;">
ENSEMBL:ENSMUSG00000102466
</td>
<td style="text-align:left;">
MGI:5611428
</td>
<td style="text-align:left;">
unclassified gene
</td>
<td style="text-align:left;">
predicted gene, 38200
</td>
</tr>
<tr>
<td style="text-align:left;">
3
</td>
<td style="text-align:left;">
gene
</td>
<td style="text-align:right;">
51.55976
</td>
<td style="text-align:right;">
51.56712
</td>
<td style="text-align:left;">
positive
</td>
<td style="text-align:left;">
MGI_C57BL6J_1923230
</td>
<td style="text-align:left;">
5031434O11Rik
</td>
<td style="text-align:left;">
NCBI_Gene:100039684,ENSEMBL:ENSMUSG00000097885
</td>
<td style="text-align:left;">
MGI:1923230
</td>
<td style="text-align:left;">
lncRNA gene
</td>
<td style="text-align:left;">
RIKEN cDNA 5031434O11 gene
</td>
</tr>
<tr>
<td style="text-align:left;">
3
</td>
<td style="text-align:left;">
pseudogene
</td>
<td style="text-align:right;">
51.57111
</td>
<td style="text-align:right;">
51.57142
</td>
<td style="text-align:left;">
negative
</td>
<td style="text-align:left;">
MGI_C57BL6J_5611562
</td>
<td style="text-align:left;">
Gm38334
</td>
<td style="text-align:left;">
ENSEMBL:ENSMUSG00000104001
</td>
<td style="text-align:left;">
MGI:5611562
</td>
<td style="text-align:left;">
pseudogene
</td>
<td style="text-align:left;">
predicted gene, 38334
</td>
</tr>
<tr>
<td style="text-align:left;">
3
</td>
<td style="text-align:left;">
gene
</td>
<td style="text-align:right;">
51.58231
</td>
<td style="text-align:right;">
51.58285
</td>
<td style="text-align:left;">
positive
</td>
<td style="text-align:left;">
MGI_C57BL6J_1918620
</td>
<td style="text-align:left;">
5430433H01Rik
</td>
<td style="text-align:left;">
ENSEMBL:ENSMUSG00000104075
</td>
<td style="text-align:left;">
MGI:1918620
</td>
<td style="text-align:left;">
unclassified gene
</td>
<td style="text-align:left;">
RIKEN cDNA 5430433H01 gene
</td>
</tr>
<tr>
<td style="text-align:left;">
3
</td>
<td style="text-align:left;">
pseudogene
</td>
<td style="text-align:right;">
51.63649
</td>
<td style="text-align:right;">
51.63705
</td>
<td style="text-align:left;">
negative
</td>
<td style="text-align:left;">
MGI_C57BL6J_5611475
</td>
<td style="text-align:left;">
Gm38247
</td>
<td style="text-align:left;">
NCBI_Gene:108168852,ENSEMBL:ENSMUSG00000104166
</td>
<td style="text-align:left;">
MGI:5611475
</td>
<td style="text-align:left;">
pseudogene
</td>
<td style="text-align:left;">
predicted gene, 38247
</td>
</tr>
<tr>
<td style="text-align:left;">
3
</td>
<td style="text-align:left;">
gene
</td>
<td style="text-align:right;">
51.66036
</td>
<td style="text-align:right;">
51.68268
</td>
<td style="text-align:left;">
positive
</td>
<td style="text-align:left;">
MGI_C57BL6J_2448481
</td>
<td style="text-align:left;">
Mgst2
</td>
<td style="text-align:left;">
NCBI_Gene:211666,ENSEMBL:ENSMUSG00000074604
</td>
<td style="text-align:left;">
MGI:2448481
</td>
<td style="text-align:left;">
protein coding gene
</td>
<td style="text-align:left;">
microsomal glutathione S-transferase 2
</td>
</tr>
<tr>
<td style="text-align:left;">
3
</td>
<td style="text-align:left;">
gene
</td>
<td style="text-align:right;">
51.67280
</td>
<td style="text-align:right;">
51.67390
</td>
<td style="text-align:left;">
positive
</td>
<td style="text-align:left;">
MGI_C57BL6J_5610489
</td>
<td style="text-align:left;">
Gm37261
</td>
<td style="text-align:left;">
ENSEMBL:ENSMUSG00000103527
</td>
<td style="text-align:left;">
MGI:5610489
</td>
<td style="text-align:left;">
unclassified gene
</td>
<td style="text-align:left;">
predicted gene, 37261
</td>
</tr>
<tr>
<td style="text-align:left;">
3
</td>
<td style="text-align:left;">
gene
</td>
<td style="text-align:right;">
51.68591
</td>
<td style="text-align:right;">
52.10509
</td>
<td style="text-align:left;">
negative
</td>
<td style="text-align:left;">
MGI_C57BL6J_2389461
</td>
<td style="text-align:left;">
Maml3
</td>
<td style="text-align:left;">
NCBI_Gene:433586,ENSEMBL:ENSMUSG00000061143
</td>
<td style="text-align:left;">
MGI:2389461
</td>
<td style="text-align:left;">
protein coding gene
</td>
<td style="text-align:left;">
mastermind like transcriptional coactivator 3
</td>
</tr>
<tr>
<td style="text-align:left;">
3
</td>
<td style="text-align:left;">
gene
</td>
<td style="text-align:right;">
51.69372
</td>
<td style="text-align:right;">
51.69509
</td>
<td style="text-align:left;">
positive
</td>
<td style="text-align:left;">
MGI_C57BL6J_3642905
</td>
<td style="text-align:left;">
Gm10729
</td>
<td style="text-align:left;">
ENSEMBL:ENSMUSG00000074603
</td>
<td style="text-align:left;">
MGI:3642905
</td>
<td style="text-align:left;">
protein coding gene
</td>
<td style="text-align:left;">
predicted gene 10729
</td>
</tr>
<tr>
<td style="text-align:left;">
3
</td>
<td style="text-align:left;">
gene
</td>
<td style="text-align:right;">
51.70772
</td>
<td style="text-align:right;">
51.71225
</td>
<td style="text-align:left;">
positive
</td>
<td style="text-align:left;">
MGI_C57BL6J_5826440
</td>
<td style="text-align:left;">
Gm46803
</td>
<td style="text-align:left;">
NCBI_Gene:108168869
</td>
<td style="text-align:left;">
MGI:5826440
</td>
<td style="text-align:left;">
lncRNA gene
</td>
<td style="text-align:left;">
predicted gene, 46803
</td>
</tr>
<tr>
<td style="text-align:left;">
3
</td>
<td style="text-align:left;">
gene
</td>
<td style="text-align:right;">
51.75712
</td>
<td style="text-align:right;">
51.76209
</td>
<td style="text-align:left;">
negative
</td>
<td style="text-align:left;">
MGI_C57BL6J_5610570
</td>
<td style="text-align:left;">
Gm37342
</td>
<td style="text-align:left;">
ENSEMBL:ENSMUSG00000102732
</td>
<td style="text-align:left;">
MGI:5610570
</td>
<td style="text-align:left;">
unclassified gene
</td>
<td style="text-align:left;">
predicted gene, 37342
</td>
</tr>
<tr>
<td style="text-align:left;">
3
</td>
<td style="text-align:left;">
gene
</td>
<td style="text-align:right;">
51.81295
</td>
<td style="text-align:right;">
51.81931
</td>
<td style="text-align:left;">
positive
</td>
<td style="text-align:left;">
MGI_C57BL6J_5610457
</td>
<td style="text-align:left;">
Gm37229
</td>
<td style="text-align:left;">
ENSEMBL:ENSMUSG00000102539
</td>
<td style="text-align:left;">
MGI:5610457
</td>
<td style="text-align:left;">
lncRNA gene
</td>
<td style="text-align:left;">
predicted gene, 37229
</td>
</tr>
<tr>
<td style="text-align:left;">
3
</td>
<td style="text-align:left;">
gene
</td>
<td style="text-align:right;">
51.82891
</td>
<td style="text-align:right;">
51.82903
</td>
<td style="text-align:left;">
positive
</td>
<td style="text-align:left;">
MGI_C57BL6J_4422061
</td>
<td style="text-align:left;">
n-R5s196
</td>
<td style="text-align:left;">
ENSEMBL:ENSMUSG00000065070
</td>
<td style="text-align:left;">
MGI:4422061
</td>
<td style="text-align:left;">
rRNA gene
</td>
<td style="text-align:left;">
nuclear encoded rRNA 5S 196
</td>
</tr>
<tr>
<td style="text-align:left;">
3
</td>
<td style="text-align:left;">
gene
</td>
<td style="text-align:right;">
51.82939
</td>
<td style="text-align:right;">
51.83040
</td>
<td style="text-align:left;">
negative
</td>
<td style="text-align:left;">
MGI_C57BL6J_1920491
</td>
<td style="text-align:left;">
3110080O07Rik
</td>
<td style="text-align:left;">
ENSEMBL:ENSMUSG00000103284
</td>
<td style="text-align:left;">
MGI:1920491
</td>
<td style="text-align:left;">
unclassified gene
</td>
<td style="text-align:left;">
RIKEN cDNA 3110080O07 gene
</td>
</tr>
<tr>
<td style="text-align:left;">
3
</td>
<td style="text-align:left;">
gene
</td>
<td style="text-align:right;">
51.86735
</td>
<td style="text-align:right;">
51.86856
</td>
<td style="text-align:left;">
negative
</td>
<td style="text-align:left;">
MGI_C57BL6J_5611228
</td>
<td style="text-align:left;">
Gm38000
</td>
<td style="text-align:left;">
ENSEMBL:ENSMUSG00000103484
</td>
<td style="text-align:left;">
MGI:5611228
</td>
<td style="text-align:left;">
unclassified gene
</td>
<td style="text-align:left;">
predicted gene, 38000
</td>
</tr>
<tr>
<td style="text-align:left;">
3
</td>
<td style="text-align:left;">
gene
</td>
<td style="text-align:right;">
51.87299
</td>
<td style="text-align:right;">
51.87584
</td>
<td style="text-align:left;">
negative
</td>
<td style="text-align:left;">
MGI_C57BL6J_5610903
</td>
<td style="text-align:left;">
Gm37675
</td>
<td style="text-align:left;">
ENSEMBL:ENSMUSG00000102620
</td>
<td style="text-align:left;">
MGI:5610903
</td>
<td style="text-align:left;">
unclassified gene
</td>
<td style="text-align:left;">
predicted gene, 37675
</td>
</tr>
<tr>
<td style="text-align:left;">
3
</td>
<td style="text-align:left;">
gene
</td>
<td style="text-align:right;">
51.88428
</td>
<td style="text-align:right;">
51.88689
</td>
<td style="text-align:left;">
negative
</td>
<td style="text-align:left;">
MGI_C57BL6J_5611480
</td>
<td style="text-align:left;">
Gm38252
</td>
<td style="text-align:left;">
ENSEMBL:ENSMUSG00000102786
</td>
<td style="text-align:left;">
MGI:5611480
</td>
<td style="text-align:left;">
unclassified gene
</td>
<td style="text-align:left;">
predicted gene, 38252
</td>
</tr>
<tr>
<td style="text-align:left;">
3
</td>
<td style="text-align:left;">
pseudogene
</td>
<td style="text-align:right;">
51.90699
</td>
<td style="text-align:right;">
51.90882
</td>
<td style="text-align:left;">
negative
</td>
<td style="text-align:left;">
MGI_C57BL6J_3708672
</td>
<td style="text-align:left;">
Gm10728
</td>
<td style="text-align:left;">
ENSEMBL:ENSMUSG00000103701
</td>
<td style="text-align:left;">
MGI:3708672
</td>
<td style="text-align:left;">
pseudogene
</td>
<td style="text-align:left;">
predicted gene 10728
</td>
</tr>
<tr>
<td style="text-align:left;">
3
</td>
<td style="text-align:left;">
gene
</td>
<td style="text-align:right;">
51.98094
</td>
<td style="text-align:right;">
51.99007
</td>
<td style="text-align:left;">
positive
</td>
<td style="text-align:left;">
MGI_C57BL6J_5012274
</td>
<td style="text-align:left;">
Gm20089
</td>
<td style="text-align:left;">
ENSEMBL:ENSMUSG00000104161
</td>
<td style="text-align:left;">
MGI:5012274
</td>
<td style="text-align:left;">
lncRNA gene
</td>
<td style="text-align:left;">
predicted gene, 20089
</td>
</tr>
<tr>
<td style="text-align:left;">
3
</td>
<td style="text-align:left;">
gene
</td>
<td style="text-align:right;">
51.98763
</td>
<td style="text-align:right;">
51.98985
</td>
<td style="text-align:left;">
negative
</td>
<td style="text-align:left;">
MGI_C57BL6J_5611283
</td>
<td style="text-align:left;">
Gm38055
</td>
<td style="text-align:left;">
ENSEMBL:ENSMUSG00000102147
</td>
<td style="text-align:left;">
MGI:5611283
</td>
<td style="text-align:left;">
unclassified gene
</td>
<td style="text-align:left;">
predicted gene, 38055
</td>
</tr>
<tr>
<td style="text-align:left;">
3
</td>
<td style="text-align:left;">
gene
</td>
<td style="text-align:right;">
51.99134
</td>
<td style="text-align:right;">
51.99196
</td>
<td style="text-align:left;">
negative
</td>
<td style="text-align:left;">
MGI_C57BL6J_5610754
</td>
<td style="text-align:left;">
Gm37526
</td>
<td style="text-align:left;">
ENSEMBL:ENSMUSG00000103784
</td>
<td style="text-align:left;">
MGI:5610754
</td>
<td style="text-align:left;">
unclassified gene
</td>
<td style="text-align:left;">
predicted gene, 37526
</td>
</tr>
<tr>
<td style="text-align:left;">
3
</td>
<td style="text-align:left;">
gene
</td>
<td style="text-align:right;">
52.00074
</td>
<td style="text-align:right;">
52.00402
</td>
<td style="text-align:left;">
negative
</td>
<td style="text-align:left;">
MGI_C57BL6J_5610693
</td>
<td style="text-align:left;">
Gm37465
</td>
<td style="text-align:left;">
ENSEMBL:ENSMUSG00000102423
</td>
<td style="text-align:left;">
MGI:5610693
</td>
<td style="text-align:left;">
unclassified gene
</td>
<td style="text-align:left;">
predicted gene, 37465
</td>
</tr>
<tr>
<td style="text-align:left;">
3
</td>
<td style="text-align:left;">
gene
</td>
<td style="text-align:right;">
52.05729
</td>
<td style="text-align:right;">
52.06424
</td>
<td style="text-align:left;">
negative
</td>
<td style="text-align:left;">
MGI_C57BL6J_2444420
</td>
<td style="text-align:left;">
C130089K02Rik
</td>
<td style="text-align:left;">
ENSEMBL:ENSMUSG00000104339
</td>
<td style="text-align:left;">
MGI:2444420
</td>
<td style="text-align:left;">
unclassified gene
</td>
<td style="text-align:left;">
RIKEN cDNA C130089K02 gene
</td>
</tr>
<tr>
<td style="text-align:left;">
3
</td>
<td style="text-align:left;">
gene
</td>
<td style="text-align:right;">
52.07159
</td>
<td style="text-align:right;">
52.07339
</td>
<td style="text-align:left;">
negative
</td>
<td style="text-align:left;">
MGI_C57BL6J_1921741
</td>
<td style="text-align:left;">
5430420F09Rik
</td>
<td style="text-align:left;">
ENSEMBL:ENSMUSG00000102808
</td>
<td style="text-align:left;">
MGI:1921741
</td>
<td style="text-align:left;">
unclassified gene
</td>
<td style="text-align:left;">
RIKEN cDNA 5430420F09 gene
</td>
</tr>
<tr>
<td style="text-align:left;">
3
</td>
<td style="text-align:left;">
gene
</td>
<td style="text-align:right;">
52.13405
</td>
<td style="text-align:right;">
52.22200
</td>
<td style="text-align:left;">
negative
</td>
<td style="text-align:left;">
MGI_C57BL6J_5610430
</td>
<td style="text-align:left;">
Gm37202
</td>
<td style="text-align:left;">
ENSEMBL:ENSMUSG00000103655
</td>
<td style="text-align:left;">
MGI:5610430
</td>
<td style="text-align:left;">
lncRNA gene
</td>
<td style="text-align:left;">
predicted gene, 37202
</td>
</tr>
<tr>
<td style="text-align:left;">
3
</td>
<td style="text-align:left;">
gene
</td>
<td style="text-align:right;">
52.17108
</td>
<td style="text-align:right;">
52.18013
</td>
<td style="text-align:left;">
negative
</td>
<td style="text-align:left;">
MGI_C57BL6J_5589233
</td>
<td style="text-align:left;">
Gm30074
</td>
<td style="text-align:left;">
NCBI_Gene:102631843,ENSEMBL:ENSMUSG00000103726
</td>
<td style="text-align:left;">
MGI:5589233
</td>
<td style="text-align:left;">
lncRNA gene
</td>
<td style="text-align:left;">
predicted gene, 30074
</td>
</tr>
<tr>
<td style="text-align:left;">
3
</td>
<td style="text-align:left;">
gene
</td>
<td style="text-align:right;">
52.18657
</td>
<td style="text-align:right;">
52.18796
</td>
<td style="text-align:left;">
positive
</td>
<td style="text-align:left;">
MGI_C57BL6J_5611360
</td>
<td style="text-align:left;">
Gm38132
</td>
<td style="text-align:left;">
ENSEMBL:ENSMUSG00000104055
</td>
<td style="text-align:left;">
MGI:5611360
</td>
<td style="text-align:left;">
lncRNA gene
</td>
<td style="text-align:left;">
predicted gene, 38132
</td>
</tr>
<tr>
<td style="text-align:left;">
3
</td>
<td style="text-align:left;">
gene
</td>
<td style="text-align:right;">
52.19825
</td>
<td style="text-align:right;">
52.20049
</td>
<td style="text-align:left;">
positive
</td>
<td style="text-align:left;">
MGI_C57BL6J_5610398
</td>
<td style="text-align:left;">
Gm37170
</td>
<td style="text-align:left;">
ENSEMBL:ENSMUSG00000103170
</td>
<td style="text-align:left;">
MGI:5610398
</td>
<td style="text-align:left;">
lncRNA gene
</td>
<td style="text-align:left;">
predicted gene, 37170
</td>
</tr>
<tr>
<td style="text-align:left;">
3
</td>
<td style="text-align:left;">
pseudogene
</td>
<td style="text-align:right;">
52.23684
</td>
<td style="text-align:right;">
52.24694
</td>
<td style="text-align:left;">
negative
</td>
<td style="text-align:left;">
MGI_C57BL6J_3031232
</td>
<td style="text-align:left;">
Olfr1398-ps1
</td>
<td style="text-align:left;">
NCBI_Gene:404507,ENSEMBL:ENSMUSG00000102510
</td>
<td style="text-align:left;">
MGI:3031232
</td>
<td style="text-align:left;">
pseudogene
</td>
<td style="text-align:left;">
olfactory receptor 1398, pseudogene 1
</td>
</tr>
<tr>
<td style="text-align:left;">
3
</td>
<td style="text-align:left;">
gene
</td>
<td style="text-align:right;">
52.23745
</td>
<td style="text-align:right;">
52.25650
</td>
<td style="text-align:left;">
positive
</td>
<td style="text-align:left;">
MGI_C57BL6J_5589284
</td>
<td style="text-align:left;">
Gm30125
</td>
<td style="text-align:left;">
NCBI_Gene:102631910
</td>
<td style="text-align:left;">
MGI:5589284
</td>
<td style="text-align:left;">
lncRNA gene
</td>
<td style="text-align:left;">
predicted gene, 30125
</td>
</tr>
<tr>
<td style="text-align:left;">
3
</td>
<td style="text-align:left;">
gene
</td>
<td style="text-align:right;">
52.26834
</td>
<td style="text-align:right;">
52.35322
</td>
<td style="text-align:left;">
positive
</td>
<td style="text-align:left;">
MGI_C57BL6J_1890077
</td>
<td style="text-align:left;">
Foxo1
</td>
<td style="text-align:left;">
NCBI_Gene:56458,ENSEMBL:ENSMUSG00000044167
</td>
<td style="text-align:left;">
MGI:1890077
</td>
<td style="text-align:left;">
protein coding gene
</td>
<td style="text-align:left;">
forkhead box O1
</td>
</tr>
<tr>
<td style="text-align:left;">
3
</td>
<td style="text-align:left;">
gene
</td>
<td style="text-align:right;">
52.26843
</td>
<td style="text-align:right;">
52.26944
</td>
<td style="text-align:left;">
negative
</td>
<td style="text-align:left;">
MGI_C57BL6J_5141867
</td>
<td style="text-align:left;">
Gm20402
</td>
<td style="text-align:left;">
ENSEMBL:ENSMUSG00000092405
</td>
<td style="text-align:left;">
MGI:5141867
</td>
<td style="text-align:left;">
lncRNA gene
</td>
<td style="text-align:left;">
predicted gene 20402
</td>
</tr>
<tr>
<td style="text-align:left;">
3
</td>
<td style="text-align:left;">
gene
</td>
<td style="text-align:right;">
52.28224
</td>
<td style="text-align:right;">
52.28431
</td>
<td style="text-align:left;">
positive
</td>
<td style="text-align:left;">
MGI_C57BL6J_5611262
</td>
<td style="text-align:left;">
Gm38034
</td>
<td style="text-align:left;">
ENSEMBL:ENSMUSG00000102844
</td>
<td style="text-align:left;">
MGI:5611262
</td>
<td style="text-align:left;">
unclassified gene
</td>
<td style="text-align:left;">
predicted gene, 38034
</td>
</tr>
<tr>
<td style="text-align:left;">
3
</td>
<td style="text-align:left;">
pseudogene
</td>
<td style="text-align:right;">
52.42535
</td>
<td style="text-align:right;">
52.42573
</td>
<td style="text-align:left;">
positive
</td>
<td style="text-align:left;">
MGI_C57BL6J_5593462
</td>
<td style="text-align:left;">
Gm34303
</td>
<td style="text-align:left;">
NCBI_Gene:102637513
</td>
<td style="text-align:left;">
MGI:5593462
</td>
<td style="text-align:left;">
pseudogene
</td>
<td style="text-align:left;">
predicted gene, 34303
</td>
</tr>
<tr>
<td style="text-align:left;">
3
</td>
<td style="text-align:left;">
pseudogene
</td>
<td style="text-align:right;">
52.42541
</td>
<td style="text-align:right;">
52.42565
</td>
<td style="text-align:left;">
positive
</td>
<td style="text-align:left;">
MGI_C57BL6J_5611326
</td>
<td style="text-align:left;">
Gm38098
</td>
<td style="text-align:left;">
ENSEMBL:ENSMUSG00000103454
</td>
<td style="text-align:left;">
MGI:5611326
</td>
<td style="text-align:left;">
pseudogene
</td>
<td style="text-align:left;">
predicted gene, 38098
</td>
</tr>
<tr>
<td style="text-align:left;">
3
</td>
<td style="text-align:left;">
gene
</td>
<td style="text-align:right;">
52.46717
</td>
<td style="text-align:right;">
52.49327
</td>
<td style="text-align:left;">
positive
</td>
<td style="text-align:left;">
MGI_C57BL6J_5621422
</td>
<td style="text-align:left;">
Gm38537
</td>
<td style="text-align:left;">
NCBI_Gene:102641152
</td>
<td style="text-align:left;">
MGI:5621422
</td>
<td style="text-align:left;">
lncRNA gene
</td>
<td style="text-align:left;">
predicted gene, 38537
</td>
</tr>
<tr>
<td style="text-align:left;">
3
</td>
<td style="text-align:left;">
gene
</td>
<td style="text-align:right;">
52.50277
</td>
<td style="text-align:right;">
52.55022
</td>
<td style="text-align:left;">
positive
</td>
<td style="text-align:left;">
MGI_C57BL6J_5589332
</td>
<td style="text-align:left;">
Gm30173
</td>
<td style="text-align:left;">
NCBI_Gene:102631981,ENSEMBL:ENSMUSG00000103965
</td>
<td style="text-align:left;">
MGI:5589332
</td>
<td style="text-align:left;">
lncRNA gene
</td>
<td style="text-align:left;">
predicted gene, 30173
</td>
</tr>
<tr>
<td style="text-align:left;">
3
</td>
<td style="text-align:left;">
gene
</td>
<td style="text-align:right;">
52.58013
</td>
<td style="text-align:right;">
52.58907
</td>
<td style="text-align:left;">
positive
</td>
<td style="text-align:left;">
MGI_C57BL6J_5622932
</td>
<td style="text-align:left;">
Gm40047
</td>
<td style="text-align:left;">
NCBI_Gene:105244440
</td>
<td style="text-align:left;">
MGI:5622932
</td>
<td style="text-align:left;">
lncRNA gene
</td>
<td style="text-align:left;">
predicted gene, 40047
</td>
</tr>
<tr>
<td style="text-align:left;">
3
</td>
<td style="text-align:left;">
pseudogene
</td>
<td style="text-align:right;">
52.61282
</td>
<td style="text-align:right;">
52.61402
</td>
<td style="text-align:left;">
positive
</td>
<td style="text-align:left;">
MGI_C57BL6J_3704216
</td>
<td style="text-align:left;">
Gm10293
</td>
<td style="text-align:left;">
NCBI_Gene:100039762,ENSEMBL:ENSMUSG00000070490
</td>
<td style="text-align:left;">
MGI:3704216
</td>
<td style="text-align:left;">
pseudogene
</td>
<td style="text-align:left;">
predicted pseudogene 10293
</td>
</tr>
<tr>
<td style="text-align:left;">
3
</td>
<td style="text-align:left;">
gene
</td>
<td style="text-align:right;">
52.64989
</td>
<td style="text-align:right;">
52.70971
</td>
<td style="text-align:left;">
positive
</td>
<td style="text-align:left;">
MGI_C57BL6J_5589394
</td>
<td style="text-align:left;">
Gm30235
</td>
<td style="text-align:left;">
NCBI_Gene:102632062
</td>
<td style="text-align:left;">
MGI:5589394
</td>
<td style="text-align:left;">
lncRNA gene
</td>
<td style="text-align:left;">
predicted gene, 30235
</td>
</tr>
<tr>
<td style="text-align:left;">
3
</td>
<td style="text-align:left;">
gene
</td>
<td style="text-align:right;">
52.72265
</td>
<td style="text-align:right;">
52.72275
</td>
<td style="text-align:left;">
negative
</td>
<td style="text-align:left;">
MGI_C57BL6J_5452575
</td>
<td style="text-align:left;">
Gm22798
</td>
<td style="text-align:left;">
ENSEMBL:ENSMUSG00000065926
</td>
<td style="text-align:left;">
MGI:5452575
</td>
<td style="text-align:left;">
snRNA gene
</td>
<td style="text-align:left;">
predicted gene, 22798
</td>
</tr>
<tr>
<td style="text-align:left;">
3
</td>
<td style="text-align:left;">
gene
</td>
<td style="text-align:right;">
52.73972
</td>
<td style="text-align:right;">
52.77666
</td>
<td style="text-align:left;">
positive
</td>
<td style="text-align:left;">
MGI_C57BL6J_3780614
</td>
<td style="text-align:left;">
Gm2447
</td>
<td style="text-align:left;">
NCBI_Gene:329639,ENSEMBL:ENSMUSG00000102785
</td>
<td style="text-align:left;">
MGI:3780614
</td>
<td style="text-align:left;">
lncRNA gene
</td>
<td style="text-align:left;">
predicted gene 2447
</td>
</tr>
<tr>
<td style="text-align:left;">
3
</td>
<td style="text-align:left;">
gene
</td>
<td style="text-align:right;">
52.78900
</td>
<td style="text-align:right;">
52.79781
</td>
<td style="text-align:left;">
negative
</td>
<td style="text-align:left;">
MGI_C57BL6J_5589451
</td>
<td style="text-align:left;">
Gm30292
</td>
<td style="text-align:left;">
NCBI_Gene:102632139,ENSEMBL:ENSMUSG00000104109
</td>
<td style="text-align:left;">
MGI:5589451
</td>
<td style="text-align:left;">
lncRNA gene
</td>
<td style="text-align:left;">
predicted gene, 30292
</td>
</tr>
<tr>
<td style="text-align:left;">
3
</td>
<td style="text-align:left;">
gene
</td>
<td style="text-align:right;">
52.90676
</td>
<td style="text-align:right;">
52.92767
</td>
<td style="text-align:left;">
negative
</td>
<td style="text-align:left;">
MGI_C57BL6J_5434106
</td>
<td style="text-align:left;">
Gm20750
</td>
<td style="text-align:left;">
NCBI_Gene:622124,ENSEMBL:ENSMUSG00000104401
</td>
<td style="text-align:left;">
MGI:5434106
</td>
<td style="text-align:left;">
lncRNA gene
</td>
<td style="text-align:left;">
predicted gene, 20750
</td>
</tr>
<tr>
<td style="text-align:left;">
3
</td>
<td style="text-align:left;">
gene
</td>
<td style="text-align:right;">
52.93325
</td>
<td style="text-align:right;">
52.95628
</td>
<td style="text-align:left;">
positive
</td>
<td style="text-align:left;">
MGI_C57BL6J_5589510
</td>
<td style="text-align:left;">
Gm30351
</td>
<td style="text-align:left;">
NCBI_Gene:102632211
</td>
<td style="text-align:left;">
MGI:5589510
</td>
<td style="text-align:left;">
lncRNA gene
</td>
<td style="text-align:left;">
predicted gene, 30351
</td>
</tr>
<tr>
<td style="text-align:left;">
3
</td>
<td style="text-align:left;">
gene
</td>
<td style="text-align:right;">
52.95618
</td>
<td style="text-align:right;">
52.95789
</td>
<td style="text-align:left;">
negative
</td>
<td style="text-align:left;">
MGI_C57BL6J_5589561
</td>
<td style="text-align:left;">
Gm30402
</td>
<td style="text-align:left;">
NCBI_Gene:102632287
</td>
<td style="text-align:left;">
MGI:5589561
</td>
<td style="text-align:left;">
lncRNA gene
</td>
<td style="text-align:left;">
predicted gene, 30402
</td>
</tr>
<tr>
<td style="text-align:left;">
3
</td>
<td style="text-align:left;">
pseudogene
</td>
<td style="text-align:right;">
52.96005
</td>
<td style="text-align:right;">
52.96037
</td>
<td style="text-align:left;">
negative
</td>
<td style="text-align:left;">
MGI_C57BL6J_5610235
</td>
<td style="text-align:left;">
Gm37007
</td>
<td style="text-align:left;">
ENSEMBL:ENSMUSG00000103886
</td>
<td style="text-align:left;">
MGI:5610235
</td>
<td style="text-align:left;">
pseudogene
</td>
<td style="text-align:left;">
predicted gene, 37007
</td>
</tr>
<tr>
<td style="text-align:left;">
3
</td>
<td style="text-align:left;">
gene
</td>
<td style="text-align:right;">
52.98188
</td>
<td style="text-align:right;">
53.01724
</td>
<td style="text-align:left;">
negative
</td>
<td style="text-align:left;">
MGI_C57BL6J_1914792
</td>
<td style="text-align:left;">
Cog6
</td>
<td style="text-align:left;">
NCBI_Gene:67542,ENSEMBL:ENSMUSG00000027742
</td>
<td style="text-align:left;">
MGI:1914792
</td>
<td style="text-align:left;">
protein coding gene
</td>
<td style="text-align:left;">
component of oligomeric golgi complex 6
</td>
</tr>
<tr>
<td style="text-align:left;">
3
</td>
<td style="text-align:left;">
gene
</td>
<td style="text-align:right;">
53.03858
</td>
<td style="text-align:right;">
53.04176
</td>
<td style="text-align:left;">
negative
</td>
<td style="text-align:left;">
MGI_C57BL6J_5663038
</td>
<td style="text-align:left;">
Gm42901
</td>
<td style="text-align:left;">
ENSEMBL:ENSMUSG00000104850
</td>
<td style="text-align:left;">
MGI:5663038
</td>
<td style="text-align:left;">
lncRNA gene
</td>
<td style="text-align:left;">
predicted gene 42901
</td>
</tr>
<tr>
<td style="text-align:left;">
3
</td>
<td style="text-align:left;">
gene
</td>
<td style="text-align:right;">
53.04153
</td>
<td style="text-align:right;">
53.26168
</td>
<td style="text-align:left;">
positive
</td>
<td style="text-align:left;">
MGI_C57BL6J_1920048
</td>
<td style="text-align:left;">
Lhfp
</td>
<td style="text-align:left;">
NCBI_Gene:108927,ENSEMBL:ENSMUSG00000048332
</td>
<td style="text-align:left;">
MGI:1920048
</td>
<td style="text-align:left;">
protein coding gene
</td>
<td style="text-align:left;">
lipoma HMGIC fusion partner
</td>
</tr>
<tr>
<td style="text-align:left;">
3
</td>
<td style="text-align:left;">
gene
</td>
<td style="text-align:right;">
53.07216
</td>
<td style="text-align:right;">
53.07524
</td>
<td style="text-align:left;">
negative
</td>
<td style="text-align:left;">
MGI_C57BL6J_5663035
</td>
<td style="text-align:left;">
Gm42898
</td>
<td style="text-align:left;">
ENSEMBL:ENSMUSG00000104828
</td>
<td style="text-align:left;">
MGI:5663035
</td>
<td style="text-align:left;">
unclassified gene
</td>
<td style="text-align:left;">
predicted gene 42898
</td>
</tr>
<tr>
<td style="text-align:left;">
3
</td>
<td style="text-align:left;">
gene
</td>
<td style="text-align:right;">
53.08322
</td>
<td style="text-align:right;">
53.08588
</td>
<td style="text-align:left;">
positive
</td>
<td style="text-align:left;">
MGI_C57BL6J_5663036
</td>
<td style="text-align:left;">
Gm42899
</td>
<td style="text-align:left;">
ENSEMBL:ENSMUSG00000104687
</td>
<td style="text-align:left;">
MGI:5663036
</td>
<td style="text-align:left;">
unclassified gene
</td>
<td style="text-align:left;">
predicted gene 42899
</td>
</tr>
<tr>
<td style="text-align:left;">
3
</td>
<td style="text-align:left;">
gene
</td>
<td style="text-align:right;">
53.09900
</td>
<td style="text-align:right;">
53.10318
</td>
<td style="text-align:left;">
positive
</td>
<td style="text-align:left;">
MGI_C57BL6J_2443940
</td>
<td style="text-align:left;">
C130075A20Rik
</td>
<td style="text-align:left;">
ENSEMBL:ENSMUSG00000104735
</td>
<td style="text-align:left;">
MGI:2443940
</td>
<td style="text-align:left;">
unclassified gene
</td>
<td style="text-align:left;">
RIKEN cDNA C130075A20 gene
</td>
</tr>
<tr>
<td style="text-align:left;">
3
</td>
<td style="text-align:left;">
gene
</td>
<td style="text-align:right;">
53.10967
</td>
<td style="text-align:right;">
53.11012
</td>
<td style="text-align:left;">
positive
</td>
<td style="text-align:left;">
MGI_C57BL6J_5663941
</td>
<td style="text-align:left;">
Gm43804
</td>
<td style="text-align:left;">
ENSEMBL:ENSMUSG00000104615
</td>
<td style="text-align:left;">
MGI:5663941
</td>
<td style="text-align:left;">
unclassified gene
</td>
<td style="text-align:left;">
predicted gene 43804
</td>
</tr>
<tr>
<td style="text-align:left;">
3
</td>
<td style="text-align:left;">
gene
</td>
<td style="text-align:right;">
53.11040
</td>
<td style="text-align:right;">
53.11341
</td>
<td style="text-align:left;">
negative
</td>
<td style="text-align:left;">
MGI_C57BL6J_5663940
</td>
<td style="text-align:left;">
Gm43803
</td>
<td style="text-align:left;">
ENSEMBL:ENSMUSG00000104968
</td>
<td style="text-align:left;">
MGI:5663940
</td>
<td style="text-align:left;">
lncRNA gene
</td>
<td style="text-align:left;">
predicted gene 43803
</td>
</tr>
<tr>
<td style="text-align:left;">
3
</td>
<td style="text-align:left;">
gene
</td>
<td style="text-align:right;">
53.11217
</td>
<td style="text-align:right;">
53.11299
</td>
<td style="text-align:left;">
positive
</td>
<td style="text-align:left;">
MGI_C57BL6J_5663942
</td>
<td style="text-align:left;">
Gm43805
</td>
<td style="text-align:left;">
ENSEMBL:ENSMUSG00000105971
</td>
<td style="text-align:left;">
MGI:5663942
</td>
<td style="text-align:left;">
unclassified gene
</td>
<td style="text-align:left;">
predicted gene 43805
</td>
</tr>
<tr>
<td style="text-align:left;">
3
</td>
<td style="text-align:left;">
pseudogene
</td>
<td style="text-align:right;">
53.34649
</td>
<td style="text-align:right;">
53.34723
</td>
<td style="text-align:left;">
negative
</td>
<td style="text-align:left;">
MGI_C57BL6J_5663609
</td>
<td style="text-align:left;">
Gm43472
</td>
<td style="text-align:left;">
ENSEMBL:ENSMUSG00000106660
</td>
<td style="text-align:left;">
MGI:5663609
</td>
<td style="text-align:left;">
pseudogene
</td>
<td style="text-align:left;">
predicted gene 43472
</td>
</tr>
<tr>
<td style="text-align:left;">
3
</td>
<td style="text-align:left;">
gene
</td>
<td style="text-align:right;">
53.39575
</td>
<td style="text-align:right;">
53.39850
</td>
<td style="text-align:left;">
positive
</td>
<td style="text-align:left;">
MGI_C57BL6J_5622933
</td>
<td style="text-align:left;">
Gm40048
</td>
<td style="text-align:left;">
NCBI_Gene:105244441
</td>
<td style="text-align:left;">
MGI:5622933
</td>
<td style="text-align:left;">
lncRNA gene
</td>
<td style="text-align:left;">
predicted gene, 40048
</td>
</tr>
<tr>
<td style="text-align:left;">
3
</td>
<td style="text-align:left;">
pseudogene
</td>
<td style="text-align:right;">
53.40161
</td>
<td style="text-align:right;">
53.42457
</td>
<td style="text-align:left;">
negative
</td>
<td style="text-align:left;">
MGI_C57BL6J_5663608
</td>
<td style="text-align:left;">
Gm43471
</td>
<td style="text-align:left;">
NCBI_Gene:108168853,ENSEMBL:ENSMUSG00000105397
</td>
<td style="text-align:left;">
MGI:5663608
</td>
<td style="text-align:left;">
pseudogene
</td>
<td style="text-align:left;">
predicted gene 43471
</td>
</tr>
<tr>
<td style="text-align:left;">
3
</td>
<td style="text-align:left;">
gene
</td>
<td style="text-align:right;">
53.44858
</td>
<td style="text-align:right;">
53.46333
</td>
<td style="text-align:left;">
negative
</td>
<td style="text-align:left;">
MGI_C57BL6J_2444520
</td>
<td style="text-align:left;">
Nhlrc3
</td>
<td style="text-align:left;">
NCBI_Gene:212114,ENSEMBL:ENSMUSG00000042997
</td>
<td style="text-align:left;">
MGI:2444520
</td>
<td style="text-align:left;">
protein coding gene
</td>
<td style="text-align:left;">
NHL repeat containing 3
</td>
</tr>
<tr>
<td style="text-align:left;">
3
</td>
<td style="text-align:left;">
gene
</td>
<td style="text-align:right;">
53.45262
</td>
<td style="text-align:right;">
53.45805
</td>
<td style="text-align:left;">
positive
</td>
<td style="text-align:left;">
MGI_C57BL6J_3801870
</td>
<td style="text-align:left;">
Gm16206
</td>
<td style="text-align:left;">
NCBI_Gene:102632669,ENSEMBL:ENSMUSG00000085174
</td>
<td style="text-align:left;">
MGI:3801870
</td>
<td style="text-align:left;">
lncRNA gene
</td>
<td style="text-align:left;">
predicted gene 16206
</td>
</tr>
<tr>
<td style="text-align:left;">
3
</td>
<td style="text-align:left;">
gene
</td>
<td style="text-align:right;">
53.46367
</td>
<td style="text-align:right;">
53.48175
</td>
<td style="text-align:left;">
positive
</td>
<td style="text-align:left;">
MGI_C57BL6J_1919933
</td>
<td style="text-align:left;">
Proser1
</td>
<td style="text-align:left;">
NCBI_Gene:212127,ENSEMBL:ENSMUSG00000049504
</td>
<td style="text-align:left;">
MGI:1919933
</td>
<td style="text-align:left;">
protein coding gene
</td>
<td style="text-align:left;">
proline and serine rich 1
</td>
</tr>
<tr>
<td style="text-align:left;">
3
</td>
<td style="text-align:left;">
gene
</td>
<td style="text-align:right;">
53.48865
</td>
<td style="text-align:right;">
53.50850
</td>
<td style="text-align:left;">
positive
</td>
<td style="text-align:left;">
MGI_C57BL6J_2388072
</td>
<td style="text-align:left;">
Stoml3
</td>
<td style="text-align:left;">
NCBI_Gene:229277,ENSEMBL:ENSMUSG00000027744
</td>
<td style="text-align:left;">
MGI:2388072
</td>
<td style="text-align:left;">
protein coding gene
</td>
<td style="text-align:left;">
stomatin (Epb7.2)-like 3
</td>
</tr>
<tr>
<td style="text-align:left;">
3
</td>
<td style="text-align:left;">
gene
</td>
<td style="text-align:right;">
53.50893
</td>
<td style="text-align:right;">
53.51010
</td>
<td style="text-align:left;">
positive
</td>
<td style="text-align:left;">
MGI_C57BL6J_1924132
</td>
<td style="text-align:left;">
6430500D05Rik
</td>
<td style="text-align:left;">
ENSEMBL:ENSMUSG00000105481
</td>
<td style="text-align:left;">
MGI:1924132
</td>
<td style="text-align:left;">
unclassified gene
</td>
<td style="text-align:left;">
RIKEN cDNA 6430500D05 gene
</td>
</tr>
<tr>
<td style="text-align:left;">
3
</td>
<td style="text-align:left;">
gene
</td>
<td style="text-align:right;">
53.51200
</td>
<td style="text-align:right;">
53.51213
</td>
<td style="text-align:left;">
positive
</td>
<td style="text-align:left;">
MGI_C57BL6J_5453452
</td>
<td style="text-align:left;">
Gm23675
</td>
<td style="text-align:left;">
ENSEMBL:ENSMUSG00000087936
</td>
<td style="text-align:left;">
MGI:5453452
</td>
<td style="text-align:left;">
snoRNA gene
</td>
<td style="text-align:left;">
predicted gene, 23675
</td>
</tr>
<tr>
<td style="text-align:left;">
3
</td>
<td style="text-align:left;">
gene
</td>
<td style="text-align:right;">
53.51394
</td>
<td style="text-align:right;">
53.65736
</td>
<td style="text-align:left;">
negative
</td>
<td style="text-align:left;">
MGI_C57BL6J_2444465
</td>
<td style="text-align:left;">
Frem2
</td>
<td style="text-align:left;">
NCBI_Gene:242022,ENSEMBL:ENSMUSG00000037016
</td>
<td style="text-align:left;">
MGI:2444465
</td>
<td style="text-align:left;">
protein coding gene
</td>
<td style="text-align:left;">
Fras1 related extracellular matrix protein 2
</td>
</tr>
<tr>
<td style="text-align:left;">
3
</td>
<td style="text-align:left;">
pseudogene
</td>
<td style="text-align:right;">
53.66666
</td>
<td style="text-align:right;">
53.66739
</td>
<td style="text-align:left;">
negative
</td>
<td style="text-align:left;">
MGI_C57BL6J_3648425
</td>
<td style="text-align:left;">
Gm6073
</td>
<td style="text-align:left;">
NCBI_Gene:606547,ENSEMBL:ENSMUSG00000105840
</td>
<td style="text-align:left;">
MGI:3648425
</td>
<td style="text-align:left;">
pseudogene
</td>
<td style="text-align:left;">
predicted gene 6073
</td>
</tr>
<tr>
<td style="text-align:left;">
3
</td>
<td style="text-align:left;">
pseudogene
</td>
<td style="text-align:right;">
53.69237
</td>
<td style="text-align:right;">
53.69292
</td>
<td style="text-align:left;">
negative
</td>
<td style="text-align:left;">
MGI_C57BL6J_3649038
</td>
<td style="text-align:left;">
Gm6204
</td>
<td style="text-align:left;">
NCBI_Gene:621155,ENSEMBL:ENSMUSG00000105879
</td>
<td style="text-align:left;">
MGI:3649038
</td>
<td style="text-align:left;">
pseudogene
</td>
<td style="text-align:left;">
predicted gene 6204
</td>
</tr>
<tr>
<td style="text-align:left;">
3
</td>
<td style="text-align:left;">
gene
</td>
<td style="text-align:right;">
53.70304
</td>
<td style="text-align:right;">
53.73576
</td>
<td style="text-align:left;">
positive
</td>
<td style="text-align:left;">
MGI_C57BL6J_5589894
</td>
<td style="text-align:left;">
Gm30735
</td>
<td style="text-align:left;">
NCBI_Gene:102632743,ENSEMBL:ENSMUSG00000105168
</td>
<td style="text-align:left;">
MGI:5589894
</td>
<td style="text-align:left;">
lncRNA gene
</td>
<td style="text-align:left;">
predicted gene, 30735
</td>
</tr>
<tr>
<td style="text-align:left;">
3
</td>
<td style="text-align:left;">
gene
</td>
<td style="text-align:right;">
53.73622
</td>
<td style="text-align:right;">
53.73783
</td>
<td style="text-align:left;">
positive
</td>
<td style="text-align:left;">
MGI_C57BL6J_3642187
</td>
<td style="text-align:left;">
Gm10727
</td>
<td style="text-align:left;">
ENSEMBL:ENSMUSG00000105444
</td>
<td style="text-align:left;">
MGI:3642187
</td>
<td style="text-align:left;">
unclassified gene
</td>
<td style="text-align:left;">
predicted gene 10727
</td>
</tr>
<tr>
<td style="text-align:left;">
3
</td>
<td style="text-align:left;">
gene
</td>
<td style="text-align:right;">
53.75673
</td>
<td style="text-align:right;">
53.77295
</td>
<td style="text-align:left;">
positive
</td>
<td style="text-align:left;">
MGI_C57BL6J_2443105
</td>
<td style="text-align:left;">
C820005J03Rik
</td>
<td style="text-align:left;">
ENSEMBL:ENSMUSG00000106270
</td>
<td style="text-align:left;">
MGI:2443105
</td>
<td style="text-align:left;">
lncRNA gene
</td>
<td style="text-align:left;">
RIKEN cDNA C820005J03 gene
</td>
</tr>
<tr>
<td style="text-align:left;">
3
</td>
<td style="text-align:left;">
gene
</td>
<td style="text-align:right;">
53.84341
</td>
<td style="text-align:right;">
53.84351
</td>
<td style="text-align:left;">
positive
</td>
<td style="text-align:left;">
MGI_C57BL6J_5454628
</td>
<td style="text-align:left;">
Gm24851
</td>
<td style="text-align:left;">
ENSEMBL:ENSMUSG00000088653
</td>
<td style="text-align:left;">
MGI:5454628
</td>
<td style="text-align:left;">
snRNA gene
</td>
<td style="text-align:left;">
predicted gene, 24851
</td>
</tr>
<tr>
<td style="text-align:left;">
3
</td>
<td style="text-align:left;">
gene
</td>
<td style="text-align:right;">
53.84509
</td>
<td style="text-align:right;">
53.84528
</td>
<td style="text-align:left;">
positive
</td>
<td style="text-align:left;">
MGI_C57BL6J_3779200
</td>
<td style="text-align:left;">
Gm10985
</td>
<td style="text-align:left;">
ENSEMBL:ENSMUSG00000078742
</td>
<td style="text-align:left;">
MGI:3779200
</td>
<td style="text-align:left;">
protein coding gene
</td>
<td style="text-align:left;">
predicted gene 10985
</td>
</tr>
<tr>
<td style="text-align:left;">
3
</td>
<td style="text-align:left;">
gene
</td>
<td style="text-align:right;">
53.85338
</td>
<td style="text-align:right;">
53.86383
</td>
<td style="text-align:left;">
negative
</td>
<td style="text-align:left;">
MGI_C57BL6J_1915140
</td>
<td style="text-align:left;">
Ufm1
</td>
<td style="text-align:left;">
NCBI_Gene:67890,ENSEMBL:ENSMUSG00000027746
</td>
<td style="text-align:left;">
MGI:1915140
</td>
<td style="text-align:left;">
protein coding gene
</td>
<td style="text-align:left;">
ubiquitin-fold modifier 1
</td>
</tr>
<tr>
<td style="text-align:left;">
3
</td>
<td style="text-align:left;">
gene
</td>
<td style="text-align:right;">
53.92484
</td>
<td style="text-align:right;">
53.94353
</td>
<td style="text-align:left;">
positive
</td>
<td style="text-align:left;">
MGI_C57BL6J_5590185
</td>
<td style="text-align:left;">
Gm31026
</td>
<td style="text-align:left;">
NCBI_Gene:102633119,ENSEMBL:ENSMUSG00000106577
</td>
<td style="text-align:left;">
MGI:5590185
</td>
<td style="text-align:left;">
lncRNA gene
</td>
<td style="text-align:left;">
predicted gene, 31026
</td>
</tr>
<tr>
<td style="text-align:left;">
3
</td>
<td style="text-align:left;">
pseudogene
</td>
<td style="text-align:right;">
54.02115
</td>
<td style="text-align:right;">
54.02195
</td>
<td style="text-align:left;">
positive
</td>
<td style="text-align:left;">
MGI_C57BL6J_3645533
</td>
<td style="text-align:left;">
Gm8109
</td>
<td style="text-align:left;">
NCBI_Gene:666444,ENSEMBL:ENSMUSG00000103155
</td>
<td style="text-align:left;">
MGI:3645533
</td>
<td style="text-align:left;">
pseudogene
</td>
<td style="text-align:left;">
predicted gene 8109
</td>
</tr>
<tr>
<td style="text-align:left;">
3
</td>
<td style="text-align:left;">
pseudogene
</td>
<td style="text-align:right;">
54.02988
</td>
<td style="text-align:right;">
54.03041
</td>
<td style="text-align:left;">
positive
</td>
<td style="text-align:left;">
MGI_C57BL6J_5010203
</td>
<td style="text-align:left;">
Gm18018
</td>
<td style="text-align:left;">
NCBI_Gene:100416282,ENSEMBL:ENSMUSG00000105067
</td>
<td style="text-align:left;">
MGI:5010203
</td>
<td style="text-align:left;">
pseudogene
</td>
<td style="text-align:left;">
predicted gene, 18018
</td>
</tr>
<tr>
<td style="text-align:left;">
3
</td>
<td style="text-align:left;">
gene
</td>
<td style="text-align:right;">
54.15604
</td>
<td style="text-align:right;">
54.31847
</td>
<td style="text-align:left;">
positive
</td>
<td style="text-align:left;">
MGI_C57BL6J_109525
</td>
<td style="text-align:left;">
Trpc4
</td>
<td style="text-align:left;">
NCBI_Gene:22066,ENSEMBL:ENSMUSG00000027748
</td>
<td style="text-align:left;">
MGI:109525
</td>
<td style="text-align:left;">
protein coding gene
</td>
<td style="text-align:left;">
transient receptor potential cation channel, subfamily C, member 4
</td>
</tr>
<tr>
<td style="text-align:left;">
3
</td>
<td style="text-align:left;">
gene
</td>
<td style="text-align:right;">
54.35927
</td>
<td style="text-align:right;">
54.39104
</td>
<td style="text-align:left;">
positive
</td>
<td style="text-align:left;">
MGI_C57BL6J_1926321
</td>
<td style="text-align:left;">
Postn
</td>
<td style="text-align:left;">
NCBI_Gene:50706,ENSEMBL:ENSMUSG00000027750
</td>
<td style="text-align:left;">
MGI:1926321
</td>
<td style="text-align:left;">
protein coding gene
</td>
<td style="text-align:left;">
periostin, osteoblast specific factor
</td>
</tr>
<tr>
<td style="text-align:left;">
3
</td>
<td style="text-align:left;">
gene
</td>
<td style="text-align:right;">
54.35966
</td>
<td style="text-align:right;">
54.36101
</td>
<td style="text-align:left;">
negative
</td>
<td style="text-align:left;">
MGI_C57BL6J_5590254
</td>
<td style="text-align:left;">
Gm31095
</td>
<td style="text-align:left;">
NCBI_Gene:102633208
</td>
<td style="text-align:left;">
MGI:5590254
</td>
<td style="text-align:left;">
lncRNA gene
</td>
<td style="text-align:left;">
predicted gene, 31095
</td>
</tr>
<tr>
<td style="text-align:left;">
3
</td>
<td style="text-align:left;">
pseudogene
</td>
<td style="text-align:right;">
54.48141
</td>
<td style="text-align:right;">
54.48293
</td>
<td style="text-align:left;">
positive
</td>
<td style="text-align:left;">
MGI_C57BL6J_3645731
</td>
<td style="text-align:left;">
Gm5641
</td>
<td style="text-align:left;">
NCBI_Gene:434807,ENSEMBL:ENSMUSG00000069014
</td>
<td style="text-align:left;">
MGI:3645731
</td>
<td style="text-align:left;">
pseudogene
</td>
<td style="text-align:left;">
predicted gene 5641
</td>
</tr>
<tr>
<td style="text-align:left;">
3
</td>
<td style="text-align:left;">
gene
</td>
<td style="text-align:right;">
54.48856
</td>
<td style="text-align:right;">
54.53489
</td>
<td style="text-align:left;">
negative
</td>
<td style="text-align:left;">
MGI_C57BL6J_5590313
</td>
<td style="text-align:left;">
Gm31154
</td>
<td style="text-align:left;">
NCBI_Gene:102633293
</td>
<td style="text-align:left;">
MGI:5590313
</td>
<td style="text-align:left;">
lncRNA gene
</td>
<td style="text-align:left;">
predicted gene, 31154
</td>
</tr>
<tr>
<td style="text-align:left;">
3
</td>
<td style="text-align:left;">
gene
</td>
<td style="text-align:right;">
54.59055
</td>
<td style="text-align:right;">
54.60139
</td>
<td style="text-align:left;">
negative
</td>
<td style="text-align:left;">
MGI_C57BL6J_5622935
</td>
<td style="text-align:left;">
Gm40050
</td>
<td style="text-align:left;">
NCBI_Gene:105244443
</td>
<td style="text-align:left;">
MGI:5622935
</td>
<td style="text-align:left;">
lncRNA gene
</td>
<td style="text-align:left;">
predicted gene, 40050
</td>
</tr>
<tr>
<td style="text-align:left;">
3
</td>
<td style="text-align:left;">
gene
</td>
<td style="text-align:right;">
54.63144
</td>
<td style="text-align:right;">
54.63463
</td>
<td style="text-align:left;">
positive
</td>
<td style="text-align:left;">
MGI_C57BL6J_5477172
</td>
<td style="text-align:left;">
Gm26678
</td>
<td style="text-align:left;">
ENSEMBL:ENSMUSG00000097599
</td>
<td style="text-align:left;">
MGI:5477172
</td>
<td style="text-align:left;">
lncRNA gene
</td>
<td style="text-align:left;">
predicted gene, 26678
</td>
</tr>
<tr>
<td style="text-align:left;">
3
</td>
<td style="text-align:left;">
pseudogene
</td>
<td style="text-align:right;">
54.63288
</td>
<td style="text-align:right;">
54.63457
</td>
<td style="text-align:left;">
negative
</td>
<td style="text-align:left;">
MGI_C57BL6J_3588285
</td>
<td style="text-align:left;">
B020017C02Rik
</td>
<td style="text-align:left;">
NCBI_Gene:100503708,ENSEMBL:ENSMUSG00000104831
</td>
<td style="text-align:left;">
MGI:3588285
</td>
<td style="text-align:left;">
pseudogene
</td>
<td style="text-align:left;">
RIKEN cDNA B020017C02 gene
</td>
</tr>
<tr>
<td style="text-align:left;">
3
</td>
<td style="text-align:left;">
gene
</td>
<td style="text-align:right;">
54.69276
</td>
<td style="text-align:right;">
54.72877
</td>
<td style="text-align:left;">
positive
</td>
<td style="text-align:left;">
MGI_C57BL6J_1929651
</td>
<td style="text-align:left;">
Supt20
</td>
<td style="text-align:left;">
NCBI_Gene:56790,ENSEMBL:ENSMUSG00000027751
</td>
<td style="text-align:left;">
MGI:1929651
</td>
<td style="text-align:left;">
protein coding gene
</td>
<td style="text-align:left;">
SPT20 SAGA complex component
</td>
</tr>
<tr>
<td style="text-align:left;">
3
</td>
<td style="text-align:left;">
gene
</td>
<td style="text-align:right;">
54.71318
</td>
<td style="text-align:right;">
54.71435
</td>
<td style="text-align:left;">
negative
</td>
<td style="text-align:left;">
MGI_C57BL6J_5439427
</td>
<td style="text-align:left;">
Gm21958
</td>
<td style="text-align:left;">
ENSEMBL:ENSMUSG00000094487
</td>
<td style="text-align:left;">
MGI:5439427
</td>
<td style="text-align:left;">
protein coding gene
</td>
<td style="text-align:left;">
predicted gene, 21958
</td>
</tr>
<tr>
<td style="text-align:left;">
3
</td>
<td style="text-align:left;">
gene
</td>
<td style="text-align:right;">
54.72868
</td>
<td style="text-align:right;">
54.73539
</td>
<td style="text-align:left;">
negative
</td>
<td style="text-align:left;">
MGI_C57BL6J_1916889
</td>
<td style="text-align:left;">
Exosc8
</td>
<td style="text-align:left;">
NCBI_Gene:69639,ENSEMBL:ENSMUSG00000027752
</td>
<td style="text-align:left;">
MGI:1916889
</td>
<td style="text-align:left;">
protein coding gene
</td>
<td style="text-align:left;">
exosome component 8
</td>
</tr>
<tr>
<td style="text-align:left;">
3
</td>
<td style="text-align:left;">
gene
</td>
<td style="text-align:right;">
54.73554
</td>
<td style="text-align:right;">
54.75132
</td>
<td style="text-align:left;">
positive
</td>
<td style="text-align:left;">
MGI_C57BL6J_1913498
</td>
<td style="text-align:left;">
Alg5
</td>
<td style="text-align:left;">
NCBI_Gene:66248,ENSEMBL:ENSMUSG00000036632
</td>
<td style="text-align:left;">
MGI:1913498
</td>
<td style="text-align:left;">
protein coding gene
</td>
<td style="text-align:left;">
asparagine-linked glycosylation 5 (dolichyl-phosphate beta-glucosyltransferase)
</td>
</tr>
<tr>
<td style="text-align:left;">
3
</td>
<td style="text-align:left;">
gene
</td>
<td style="text-align:right;">
54.75545
</td>
<td style="text-align:right;">
54.80166
</td>
<td style="text-align:left;">
positive
</td>
<td style="text-align:left;">
MGI_C57BL6J_1859993
</td>
<td style="text-align:left;">
Smad9
</td>
<td style="text-align:left;">
NCBI_Gene:55994,ENSEMBL:ENSMUSG00000027796
</td>
<td style="text-align:left;">
MGI:1859993
</td>
<td style="text-align:left;">
protein coding gene
</td>
<td style="text-align:left;">
SMAD family member 9
</td>
</tr>
<tr>
<td style="text-align:left;">
3
</td>
<td style="text-align:left;">
gene
</td>
<td style="text-align:right;">
54.80311
</td>
<td style="text-align:right;">
54.80779
</td>
<td style="text-align:left;">
negative
</td>
<td style="text-align:left;">
MGI_C57BL6J_2180854
</td>
<td style="text-align:left;">
Rfxap
</td>
<td style="text-align:left;">
NCBI_Gene:170767,ENSEMBL:ENSMUSG00000036615
</td>
<td style="text-align:left;">
MGI:2180854
</td>
<td style="text-align:left;">
protein coding gene
</td>
<td style="text-align:left;">
regulatory factor X-associated protein
</td>
</tr>
<tr>
<td style="text-align:left;">
3
</td>
<td style="text-align:left;">
pseudogene
</td>
<td style="text-align:right;">
54.80997
</td>
<td style="text-align:right;">
54.81140
</td>
<td style="text-align:left;">
negative
</td>
<td style="text-align:left;">
MGI_C57BL6J_3708673
</td>
<td style="text-align:left;">
Gm10254
</td>
<td style="text-align:left;">
NCBI_Gene:545524,ENSEMBL:ENSMUSG00000069011
</td>
<td style="text-align:left;">
MGI:3708673
</td>
<td style="text-align:left;">
pseudogene
</td>
<td style="text-align:left;">
predicted gene 10254
</td>
</tr>
<tr>
<td style="text-align:left;">
3
</td>
<td style="text-align:left;">
gene
</td>
<td style="text-align:right;">
54.89707
</td>
<td style="text-align:right;">
54.91589
</td>
<td style="text-align:left;">
negative
</td>
<td style="text-align:left;">
MGI_C57BL6J_3607715
</td>
<td style="text-align:left;">
Sertm1
</td>
<td style="text-align:left;">
NCBI_Gene:329641,ENSEMBL:ENSMUSG00000056306
</td>
<td style="text-align:left;">
MGI:3607715
</td>
<td style="text-align:left;">
protein coding gene
</td>
<td style="text-align:left;">
serine rich and transmembrane domain containing 1
</td>
</tr>
</tbody>
</table>
<img src="figure/4.1.2_qtl.analysis_cont_age_ici.vs.eoi_snpsqc_pheno.corrected.cleaned_5.batches_0.Rmd/QTL%20effects-4.png" width="912" style="display: block; margin: auto;" /><img src="figure/4.1.2_qtl.analysis_cont_age_ici.vs.eoi_snpsqc_pheno.corrected.cleaned_5.batches_0.Rmd/QTL%20effects-5.png" width="912" style="display: block; margin: auto;" /><img src="figure/4.1.2_qtl.analysis_cont_age_ici.vs.eoi_snpsqc_pheno.corrected.cleaned_5.batches_0.Rmd/QTL%20effects-6.png" width="912" style="display: block; margin: auto;" />
<table class="table table-striped" style="margin-left: auto; margin-right: auto;">
<thead>
<tr>
<th style="text-align:left;">
chr
</th>
<th style="text-align:left;">
type
</th>
<th style="text-align:right;">
start
</th>
<th style="text-align:right;">
stop
</th>
<th style="text-align:left;">
strand
</th>
<th style="text-align:left;">
ID
</th>
<th style="text-align:left;">
Name
</th>
<th style="text-align:left;">
Dbxref
</th>
<th style="text-align:left;">
gene_id
</th>
<th style="text-align:left;">
mgi_type
</th>
<th style="text-align:left;">
description
</th>
</tr>
</thead>
<tbody>
<tr>
<td style="text-align:left;">
10
</td>
<td style="text-align:left;">
gene
</td>
<td style="text-align:right;">
70.80724
</td>
<td style="text-align:right;">
70.81382
</td>
<td style="text-align:left;">
positive
</td>
<td style="text-align:left;">
MGI_C57BL6J_6097330
</td>
<td style="text-align:left;">
Gm48021
</td>
<td style="text-align:left;">
ENSEMBL:ENSMUSG00000112752
</td>
<td style="text-align:left;">
MGI:6097330
</td>
<td style="text-align:left;">
lncRNA gene
</td>
<td style="text-align:left;">
predicted gene, 48021
</td>
</tr>
<tr>
<td style="text-align:left;">
10
</td>
<td style="text-align:left;">
gene
</td>
<td style="text-align:right;">
70.81385
</td>
<td style="text-align:right;">
70.81605
</td>
<td style="text-align:left;">
positive
</td>
<td style="text-align:left;">
MGI_C57BL6J_6097332
</td>
<td style="text-align:left;">
Gm48022
</td>
<td style="text-align:left;">
ENSEMBL:ENSMUSG00000112320
</td>
<td style="text-align:left;">
MGI:6097332
</td>
<td style="text-align:left;">
lncRNA gene
</td>
<td style="text-align:left;">
predicted gene, 48022
</td>
</tr>
<tr>
<td style="text-align:left;">
10
</td>
<td style="text-align:left;">
gene
</td>
<td style="text-align:right;">
70.83075
</td>
<td style="text-align:right;">
70.84294
</td>
<td style="text-align:left;">
positive
</td>
<td style="text-align:left;">
MGI_C57BL6J_5623574
</td>
<td style="text-align:left;">
Gm40689
</td>
<td style="text-align:left;">
NCBI_Gene:105245203,ENSEMBL:ENSMUSG00000112870
</td>
<td style="text-align:left;">
MGI:5623574
</td>
<td style="text-align:left;">
lncRNA gene
</td>
<td style="text-align:left;">
predicted gene, 40689
</td>
</tr>
<tr>
<td style="text-align:left;">
10
</td>
<td style="text-align:left;">
gene
</td>
<td style="text-align:right;">
70.84702
</td>
<td style="text-align:right;">
70.85373
</td>
<td style="text-align:left;">
positive
</td>
<td style="text-align:left;">
MGI_C57BL6J_6097334
</td>
<td style="text-align:left;">
Gm48023
</td>
<td style="text-align:left;">
ENSEMBL:ENSMUSG00000112883
</td>
<td style="text-align:left;">
MGI:6097334
</td>
<td style="text-align:left;">
lncRNA gene
</td>
<td style="text-align:left;">
predicted gene, 48023
</td>
</tr>
<tr>
<td style="text-align:left;">
10
</td>
<td style="text-align:left;">
gene
</td>
<td style="text-align:right;">
70.86340
</td>
<td style="text-align:right;">
70.86845
</td>
<td style="text-align:left;">
negative
</td>
<td style="text-align:left;">
MGI_C57BL6J_6097336
</td>
<td style="text-align:left;">
Gm48024
</td>
<td style="text-align:left;">
ENSEMBL:ENSMUSG00000112125
</td>
<td style="text-align:left;">
MGI:6097336
</td>
<td style="text-align:left;">
lncRNA gene
</td>
<td style="text-align:left;">
predicted gene, 48024
</td>
</tr>
<tr>
<td style="text-align:left;">
10
</td>
<td style="text-align:left;">
gene
</td>
<td style="text-align:right;">
70.86862
</td>
<td style="text-align:right;">
70.93432
</td>
<td style="text-align:left;">
positive
</td>
<td style="text-align:left;">
MGI_C57BL6J_1922439
</td>
<td style="text-align:left;">
4930533K18Rik
</td>
<td style="text-align:left;">
NCBI_Gene:75189,ENSEMBL:ENSMUSG00000047692
</td>
<td style="text-align:left;">
MGI:1922439
</td>
<td style="text-align:left;">
lncRNA gene
</td>
<td style="text-align:left;">
RIKEN cDNA 4930533K18 gene
</td>
</tr>
<tr>
<td style="text-align:left;">
10
</td>
<td style="text-align:left;">
gene
</td>
<td style="text-align:right;">
70.87000
</td>
<td style="text-align:right;">
70.87242
</td>
<td style="text-align:left;">
negative
</td>
<td style="text-align:left;">
MGI_C57BL6J_6097351
</td>
<td style="text-align:left;">
Gm48036
</td>
<td style="text-align:left;">
ENSEMBL:ENSMUSG00000112032
</td>
<td style="text-align:left;">
MGI:6097351
</td>
<td style="text-align:left;">
lncRNA gene
</td>
<td style="text-align:left;">
predicted gene, 48036
</td>
</tr>
<tr>
<td style="text-align:left;">
10
</td>
<td style="text-align:left;">
gene
</td>
<td style="text-align:right;">
70.88210
</td>
<td style="text-align:right;">
70.88221
</td>
<td style="text-align:left;">
positive
</td>
<td style="text-align:left;">
MGI_C57BL6J_5454263
</td>
<td style="text-align:left;">
Gm24486
</td>
<td style="text-align:left;">
ENSEMBL:ENSMUSG00000094703
</td>
<td style="text-align:left;">
MGI:5454263
</td>
<td style="text-align:left;">
snRNA gene
</td>
<td style="text-align:left;">
predicted gene, 24486
</td>
</tr>
<tr>
<td style="text-align:left;">
10
</td>
<td style="text-align:left;">
gene
</td>
<td style="text-align:right;">
70.92283
</td>
<td style="text-align:right;">
71.15970
</td>
<td style="text-align:left;">
negative
</td>
<td style="text-align:left;">
MGI_C57BL6J_1933388
</td>
<td style="text-align:left;">
Bicc1
</td>
<td style="text-align:left;">
NCBI_Gene:83675,ENSEMBL:ENSMUSG00000014329
</td>
<td style="text-align:left;">
MGI:1933388
</td>
<td style="text-align:left;">
protein coding gene
</td>
<td style="text-align:left;">
BicC family RNA binding protein 1
</td>
</tr>
<tr>
<td style="text-align:left;">
10
</td>
<td style="text-align:left;">
gene
</td>
<td style="text-align:right;">
70.97551
</td>
<td style="text-align:right;">
70.99304
</td>
<td style="text-align:left;">
negative
</td>
<td style="text-align:left;">
MGI_C57BL6J_6097338
</td>
<td style="text-align:left;">
Gm48025
</td>
<td style="text-align:left;">
ENSEMBL:ENSMUSG00000112745
</td>
<td style="text-align:left;">
MGI:6097338
</td>
<td style="text-align:left;">
lncRNA gene
</td>
<td style="text-align:left;">
predicted gene, 48025
</td>
</tr>
<tr>
<td style="text-align:left;">
10
</td>
<td style="text-align:left;">
gene
</td>
<td style="text-align:right;">
71.08288
</td>
<td style="text-align:right;">
71.08365
</td>
<td style="text-align:left;">
positive
</td>
<td style="text-align:left;">
MGI_C57BL6J_3783091
</td>
<td style="text-align:left;">
Gm15647
</td>
<td style="text-align:left;">
ENSEMBL:ENSMUSG00000090212
</td>
<td style="text-align:left;">
MGI:3783091
</td>
<td style="text-align:left;">
lncRNA gene
</td>
<td style="text-align:left;">
predicted gene 15647
</td>
</tr>
<tr>
<td style="text-align:left;">
10
</td>
<td style="text-align:left;">
gene
</td>
<td style="text-align:right;">
71.12566
</td>
<td style="text-align:right;">
71.14157
</td>
<td style="text-align:left;">
positive
</td>
<td style="text-align:left;">
MGI_C57BL6J_5593296
</td>
<td style="text-align:left;">
Gm34137
</td>
<td style="text-align:left;">
NCBI_Gene:102637280
</td>
<td style="text-align:left;">
MGI:5593296
</td>
<td style="text-align:left;">
lncRNA gene
</td>
<td style="text-align:left;">
predicted gene, 34137
</td>
</tr>
<tr>
<td style="text-align:left;">
10
</td>
<td style="text-align:left;">
pseudogene
</td>
<td style="text-align:right;">
71.14816
</td>
<td style="text-align:right;">
71.14981
</td>
<td style="text-align:left;">
positive
</td>
<td style="text-align:left;">
MGI_C57BL6J_5591184
</td>
<td style="text-align:left;">
Gm32025
</td>
<td style="text-align:left;">
NCBI_Gene:102634444,ENSEMBL:ENSMUSG00000112566
</td>
<td style="text-align:left;">
MGI:5591184
</td>
<td style="text-align:left;">
pseudogene
</td>
<td style="text-align:left;">
predicted gene, 32025
</td>
</tr>
<tr>
<td style="text-align:left;">
10
</td>
<td style="text-align:left;">
pseudogene
</td>
<td style="text-align:right;">
71.16291
</td>
<td style="text-align:right;">
71.16338
</td>
<td style="text-align:left;">
positive
</td>
<td style="text-align:left;">
MGI_C57BL6J_3643973
</td>
<td style="text-align:left;">
Gm6331
</td>
<td style="text-align:left;">
NCBI_Gene:622551,ENSEMBL:ENSMUSG00000112748
</td>
<td style="text-align:left;">
MGI:3643973
</td>
<td style="text-align:left;">
pseudogene
</td>
<td style="text-align:left;">
predicted gene 6331
</td>
</tr>
<tr>
<td style="text-align:left;">
10
</td>
<td style="text-align:left;">
gene
</td>
<td style="text-align:right;">
71.16874
</td>
<td style="text-align:right;">
71.18205
</td>
<td style="text-align:left;">
negative
</td>
<td style="text-align:left;">
MGI_C57BL6J_2442303
</td>
<td style="text-align:left;">
C730027H18Rik
</td>
<td style="text-align:left;">
NCBI_Gene:319572,ENSEMBL:ENSMUSG00000112189,ENSEMBL:ENSMUSG00000112366
</td>
<td style="text-align:left;">
MGI:2442303
</td>
<td style="text-align:left;">
lncRNA gene
</td>
<td style="text-align:left;">
RIKEN cDNA C730027H18 gene
</td>
</tr>
<tr>
<td style="text-align:left;">
10
</td>
<td style="text-align:left;">
gene
</td>
<td style="text-align:right;">
71.18104
</td>
<td style="text-align:right;">
71.20449
</td>
<td style="text-align:left;">
positive
</td>
<td style="text-align:left;">
MGI_C57BL6J_1925888
</td>
<td style="text-align:left;">
1700113B09Rik
</td>
<td style="text-align:left;">
NCBI_Gene:78638,ENSEMBL:ENSMUSG00000112028
</td>
<td style="text-align:left;">
MGI:1925888
</td>
<td style="text-align:left;">
lncRNA gene
</td>
<td style="text-align:left;">
RIKEN cDNA 1700113B09 gene
</td>
</tr>
<tr>
<td style="text-align:left;">
10
</td>
<td style="text-align:left;">
gene
</td>
<td style="text-align:right;">
71.18117
</td>
<td style="text-align:right;">
71.20114
</td>
<td style="text-align:left;">
positive
</td>
<td style="text-align:left;">
MGI_C57BL6J_5593487
</td>
<td style="text-align:left;">
Gm34328
</td>
<td style="text-align:left;">
NCBI_Gene:102637547
</td>
<td style="text-align:left;">
MGI:5593487
</td>
<td style="text-align:left;">
lncRNA gene
</td>
<td style="text-align:left;">
predicted gene, 34328
</td>
</tr>
<tr>
<td style="text-align:left;">
10
</td>
<td style="text-align:left;">
gene
</td>
<td style="text-align:right;">
71.22546
</td>
<td style="text-align:right;">
71.23833
</td>
<td style="text-align:left;">
negative
</td>
<td style="text-align:left;">
MGI_C57BL6J_107810
</td>
<td style="text-align:left;">
Tfam
</td>
<td style="text-align:left;">
NCBI_Gene:21780,ENSEMBL:ENSMUSG00000003923
</td>
<td style="text-align:left;">
MGI:107810
</td>
<td style="text-align:left;">
protein coding gene
</td>
<td style="text-align:left;">
transcription factor A, mitochondrial
</td>
</tr>
<tr>
<td style="text-align:left;">
10
</td>
<td style="text-align:left;">
gene
</td>
<td style="text-align:right;">
71.25498
</td>
<td style="text-align:right;">
71.28526
</td>
<td style="text-align:left;">
negative
</td>
<td style="text-align:left;">
MGI_C57BL6J_2384911
</td>
<td style="text-align:left;">
Ube2d1
</td>
<td style="text-align:left;">
NCBI_Gene:216080,ENSEMBL:ENSMUSG00000019927
</td>
<td style="text-align:left;">
MGI:2384911
</td>
<td style="text-align:left;">
protein coding gene
</td>
<td style="text-align:left;">
ubiquitin-conjugating enzyme E2D 1
</td>
</tr>
<tr>
<td style="text-align:left;">
10
</td>
<td style="text-align:left;">
gene
</td>
<td style="text-align:right;">
71.28773
</td>
<td style="text-align:right;">
71.29248
</td>
<td style="text-align:left;">
positive
</td>
<td style="text-align:left;">
MGI_C57BL6J_5593535
</td>
<td style="text-align:left;">
Gm34376
</td>
<td style="text-align:left;">
NCBI_Gene:102637612,ENSEMBL:ENSMUSG00000112813
</td>
<td style="text-align:left;">
MGI:5593535
</td>
<td style="text-align:left;">
lncRNA gene
</td>
<td style="text-align:left;">
predicted gene, 34376
</td>
</tr>
<tr>
<td style="text-align:left;">
10
</td>
<td style="text-align:left;">
gene
</td>
<td style="text-align:right;">
71.31074
</td>
<td style="text-align:right;">
71.31398
</td>
<td style="text-align:left;">
positive
</td>
<td style="text-align:left;">
MGI_C57BL6J_5313056
</td>
<td style="text-align:left;">
Gm20609
</td>
<td style="text-align:left;">
NCBI_Gene:102637684,ENSEMBL:ENSMUSG00000093732
</td>
<td style="text-align:left;">
MGI:5313056
</td>
<td style="text-align:left;">
lncRNA gene
</td>
<td style="text-align:left;">
predicted gene 20609
</td>
</tr>
<tr>
<td style="text-align:left;">
10
</td>
<td style="text-align:left;">
pseudogene
</td>
<td style="text-align:right;">
71.31873
</td>
<td style="text-align:right;">
71.31943
</td>
<td style="text-align:left;">
positive
</td>
<td style="text-align:left;">
MGI_C57BL6J_3643734
</td>
<td style="text-align:left;">
Rpl19-ps2
</td>
<td style="text-align:left;">
NCBI_Gene:665472,ENSEMBL:ENSMUSG00000112582
</td>
<td style="text-align:left;">
MGI:3643734
</td>
<td style="text-align:left;">
pseudogene
</td>
<td style="text-align:left;">
ribosomal protein L19, pseudogene 2
</td>
</tr>
<tr>
<td style="text-align:left;">
10
</td>
<td style="text-align:left;">
gene
</td>
<td style="text-align:right;">
71.33048
</td>
<td style="text-align:right;">
71.34495
</td>
<td style="text-align:left;">
negative
</td>
<td style="text-align:left;">
MGI_C57BL6J_1261855
</td>
<td style="text-align:left;">
Cisd1
</td>
<td style="text-align:left;">
NCBI_Gene:52637,ENSEMBL:ENSMUSG00000037710
</td>
<td style="text-align:left;">
MGI:1261855
</td>
<td style="text-align:left;">
protein coding gene
</td>
<td style="text-align:left;">
CDGSH iron sulfur domain 1
</td>
</tr>
<tr>
<td style="text-align:left;">
10
</td>
<td style="text-align:left;">
gene
</td>
<td style="text-align:right;">
71.33670
</td>
<td style="text-align:right;">
71.33962
</td>
<td style="text-align:left;">
negative
</td>
<td style="text-align:left;">
MGI_C57BL6J_2143791
</td>
<td style="text-align:left;">
4932439E07Rik
</td>
<td style="text-align:left;">
NA
</td>
<td style="text-align:left;">
NA
</td>
<td style="text-align:left;">
unclassified gene
</td>
<td style="text-align:left;">
RIKEN cDNA 4932439E07 gene
</td>
</tr>
<tr>
<td style="text-align:left;">
10
</td>
<td style="text-align:left;">
gene
</td>
<td style="text-align:right;">
71.34776
</td>
<td style="text-align:right;">
71.43333
</td>
<td style="text-align:left;">
positive
</td>
<td style="text-align:left;">
MGI_C57BL6J_1916968
</td>
<td style="text-align:left;">
Ipmk
</td>
<td style="text-align:left;">
NCBI_Gene:69718,ENSEMBL:ENSMUSG00000060733
</td>
<td style="text-align:left;">
MGI:1916968
</td>
<td style="text-align:left;">
protein coding gene
</td>
<td style="text-align:left;">
inositol polyphosphate multikinase
</td>
</tr>
<tr>
<td style="text-align:left;">
10
</td>
<td style="text-align:left;">
gene
</td>
<td style="text-align:right;">
71.47640
</td>
<td style="text-align:right;">
71.48552
</td>
<td style="text-align:left;">
negative
</td>
<td style="text-align:left;">
MGI_C57BL6J_1922563
</td>
<td style="text-align:left;">
4930551I15Rik
</td>
<td style="text-align:left;">
ENSEMBL:ENSMUSG00000112108
</td>
<td style="text-align:left;">
MGI:1922563
</td>
<td style="text-align:left;">
lncRNA gene
</td>
<td style="text-align:left;">
RIKEN cDNA 4930551I15 gene
</td>
</tr>
<tr>
<td style="text-align:left;">
10
</td>
<td style="text-align:left;">
pseudogene
</td>
<td style="text-align:right;">
71.48250
</td>
<td style="text-align:right;">
71.48364
</td>
<td style="text-align:left;">
positive
</td>
<td style="text-align:left;">
MGI_C57BL6J_5825870
</td>
<td style="text-align:left;">
Gm46233
</td>
<td style="text-align:left;">
NCBI_Gene:108167820,ENSEMBL:ENSMUSG00000112608
</td>
<td style="text-align:left;">
MGI:5825870
</td>
<td style="text-align:left;">
pseudogene
</td>
<td style="text-align:left;">
predicted gene, 46233
</td>
</tr>
<tr>
<td style="text-align:left;">
10
</td>
<td style="text-align:left;">
gene
</td>
<td style="text-align:right;">
71.52044
</td>
<td style="text-align:right;">
71.52230
</td>
<td style="text-align:left;">
negative
</td>
<td style="text-align:left;">
MGI_C57BL6J_6095721
</td>
<td style="text-align:left;">
Gm47028
</td>
<td style="text-align:left;">
ENSEMBL:ENSMUSG00000112416
</td>
<td style="text-align:left;">
MGI:6095721
</td>
<td style="text-align:left;">
lncRNA gene
</td>
<td style="text-align:left;">
predicted gene, 47028
</td>
</tr>
<tr>
<td style="text-align:left;">
10
</td>
<td style="text-align:left;">
gene
</td>
<td style="text-align:right;">
71.52727
</td>
<td style="text-align:right;">
71.56537
</td>
<td style="text-align:left;">
positive
</td>
<td style="text-align:left;">
MGI_C57BL6J_5593645
</td>
<td style="text-align:left;">
Gm34486
</td>
<td style="text-align:left;">
NCBI_Gene:102637756
</td>
<td style="text-align:left;">
MGI:5593645
</td>
<td style="text-align:left;">
lncRNA gene
</td>
<td style="text-align:left;">
predicted gene, 34486
</td>
</tr>
<tr>
<td style="text-align:left;">
10
</td>
<td style="text-align:left;">
gene
</td>
<td style="text-align:right;">
71.52728
</td>
<td style="text-align:right;">
71.67884
</td>
<td style="text-align:left;">
positive
</td>
<td style="text-align:left;">
MGI_C57BL6J_5593703
</td>
<td style="text-align:left;">
Gm34544
</td>
<td style="text-align:left;">
NCBI_Gene:102637834,ENSEMBL:ENSMUSG00000112126
</td>
<td style="text-align:left;">
MGI:5593703
</td>
<td style="text-align:left;">
lncRNA gene
</td>
<td style="text-align:left;">
predicted gene, 34544
</td>
</tr>
<tr>
<td style="text-align:left;">
10
</td>
<td style="text-align:left;">
pseudogene
</td>
<td style="text-align:right;">
71.59585
</td>
<td style="text-align:right;">
71.59713
</td>
<td style="text-align:left;">
negative
</td>
<td style="text-align:left;">
MGI_C57BL6J_3644095
</td>
<td style="text-align:left;">
Gm4797
</td>
<td style="text-align:left;">
NCBI_Gene:216081,ENSEMBL:ENSMUSG00000111965
</td>
<td style="text-align:left;">
MGI:3644095
</td>
<td style="text-align:left;">
pseudogene
</td>
<td style="text-align:left;">
predicted gene 4797
</td>
</tr>
<tr>
<td style="text-align:left;">
10
</td>
<td style="text-align:left;">
pseudogene
</td>
<td style="text-align:right;">
71.61165
</td>
<td style="text-align:right;">
71.61307
</td>
<td style="text-align:left;">
negative
</td>
<td style="text-align:left;">
MGI_C57BL6J_5011259
</td>
<td style="text-align:left;">
Gm19074
</td>
<td style="text-align:left;">
NCBI_Gene:100418206,ENSEMBL:ENSMUSG00000112624
</td>
<td style="text-align:left;">
MGI:5011259
</td>
<td style="text-align:left;">
pseudogene
</td>
<td style="text-align:left;">
predicted gene, 19074
</td>
</tr>
<tr>
<td style="text-align:left;">
10
</td>
<td style="text-align:left;">
pseudogene
</td>
<td style="text-align:right;">
71.70372
</td>
<td style="text-align:right;">
71.70417
</td>
<td style="text-align:left;">
positive
</td>
<td style="text-align:left;">
MGI_C57BL6J_6095749
</td>
<td style="text-align:left;">
Gm47045
</td>
<td style="text-align:left;">
ENSEMBL:ENSMUSG00000112152
</td>
<td style="text-align:left;">
MGI:6095749
</td>
<td style="text-align:left;">
pseudogene
</td>
<td style="text-align:left;">
predicted gene, 47045
</td>
</tr>
<tr>
<td style="text-align:left;">
10
</td>
<td style="text-align:left;">
pseudogene
</td>
<td style="text-align:right;">
71.70429
</td>
<td style="text-align:right;">
71.70786
</td>
<td style="text-align:left;">
positive
</td>
<td style="text-align:left;">
MGI_C57BL6J_3644096
</td>
<td style="text-align:left;">
Gm4798
</td>
<td style="text-align:left;">
NCBI_Gene:216082,ENSEMBL:ENSMUSG00000112714
</td>
<td style="text-align:left;">
MGI:3644096
</td>
<td style="text-align:left;">
pseudogene
</td>
<td style="text-align:left;">
predicted pseudogene 4798
</td>
</tr>
<tr>
<td style="text-align:left;">
10
</td>
<td style="text-align:left;">
pseudogene
</td>
<td style="text-align:right;">
71.71678
</td>
<td style="text-align:right;">
71.71802
</td>
<td style="text-align:left;">
negative
</td>
<td style="text-align:left;">
MGI_C57BL6J_3644102
</td>
<td style="text-align:left;">
Gm6362
</td>
<td style="text-align:left;">
NCBI_Gene:622812,ENSEMBL:ENSMUSG00000112717
</td>
<td style="text-align:left;">
MGI:3644102
</td>
<td style="text-align:left;">
pseudogene
</td>
<td style="text-align:left;">
predicted gene 6362
</td>
</tr>
<tr>
<td style="text-align:left;">
10
</td>
<td style="text-align:left;">
pseudogene
</td>
<td style="text-align:right;">
71.79895
</td>
<td style="text-align:right;">
71.79991
</td>
<td style="text-align:left;">
negative
</td>
<td style="text-align:left;">
MGI_C57BL6J_3648383
</td>
<td style="text-align:left;">
Gm5080
</td>
<td style="text-align:left;">
NCBI_Gene:327767,ENSEMBL:ENSMUSG00000111999
</td>
<td style="text-align:left;">
MGI:3648383
</td>
<td style="text-align:left;">
pseudogene
</td>
<td style="text-align:left;">
predicted gene 5080
</td>
</tr>
<tr>
<td style="text-align:left;">
10
</td>
<td style="text-align:left;">
gene
</td>
<td style="text-align:right;">
71.89804
</td>
<td style="text-align:right;">
71.89817
</td>
<td style="text-align:left;">
positive
</td>
<td style="text-align:left;">
MGI_C57BL6J_5452714
</td>
<td style="text-align:left;">
Gm22937
</td>
<td style="text-align:left;">
ENSEMBL:ENSMUSG00000077142
</td>
<td style="text-align:left;">
MGI:5452714
</td>
<td style="text-align:left;">
snoRNA gene
</td>
<td style="text-align:left;">
predicted gene, 22937
</td>
</tr>
<tr>
<td style="text-align:left;">
10
</td>
<td style="text-align:left;">
gene
</td>
<td style="text-align:right;">
71.96261
</td>
<td style="text-align:right;">
72.00535
</td>
<td style="text-align:left;">
positive
</td>
<td style="text-align:left;">
MGI_C57BL6J_5623575
</td>
<td style="text-align:left;">
Gm40690
</td>
<td style="text-align:left;">
NCBI_Gene:105245205
</td>
<td style="text-align:left;">
MGI:5623575
</td>
<td style="text-align:left;">
lncRNA gene
</td>
<td style="text-align:left;">
predicted gene, 40690
</td>
</tr>
<tr>
<td style="text-align:left;">
10
</td>
<td style="text-align:left;">
pseudogene
</td>
<td style="text-align:right;">
71.97988
</td>
<td style="text-align:right;">
71.98069
</td>
<td style="text-align:left;">
negative
</td>
<td style="text-align:left;">
MGI_C57BL6J_1920633
</td>
<td style="text-align:left;">
1700049L16Rik
</td>
<td style="text-align:left;">
NCBI_Gene:108950,ENSEMBL:ENSMUSG00000043859
</td>
<td style="text-align:left;">
MGI:1920633
</td>
<td style="text-align:left;">
pseudogene
</td>
<td style="text-align:left;">
RIKEN cDNA 1700049L16 gene
</td>
</tr>
<tr>
<td style="text-align:left;">
10
</td>
<td style="text-align:left;">
pseudogene
</td>
<td style="text-align:right;">
72.01921
</td>
<td style="text-align:right;">
72.01952
</td>
<td style="text-align:left;">
positive
</td>
<td style="text-align:left;">
MGI_C57BL6J_6095770
</td>
<td style="text-align:left;">
Gm47060
</td>
<td style="text-align:left;">
ENSEMBL:ENSMUSG00000112185
</td>
<td style="text-align:left;">
MGI:6095770
</td>
<td style="text-align:left;">
pseudogene
</td>
<td style="text-align:left;">
predicted gene, 47060
</td>
</tr>
<tr>
<td style="text-align:left;">
10
</td>
<td style="text-align:left;">
gene
</td>
<td style="text-align:right;">
72.02698
</td>
<td style="text-align:right;">
72.11244
</td>
<td style="text-align:left;">
positive
</td>
<td style="text-align:left;">
MGI_C57BL6J_5593768
</td>
<td style="text-align:left;">
Gm34609
</td>
<td style="text-align:left;">
NCBI_Gene:102637916,ENSEMBL:ENSMUSG00000112414
</td>
<td style="text-align:left;">
MGI:5593768
</td>
<td style="text-align:left;">
lncRNA gene
</td>
<td style="text-align:left;">
predicted gene, 34609
</td>
</tr>
<tr>
<td style="text-align:left;">
10
</td>
<td style="text-align:left;">
pseudogene
</td>
<td style="text-align:right;">
72.30920
</td>
<td style="text-align:right;">
72.31184
</td>
<td style="text-align:left;">
positive
</td>
<td style="text-align:left;">
MGI_C57BL6J_3704365
</td>
<td style="text-align:left;">
Gm9923
</td>
<td style="text-align:left;">
NCBI_Gene:100040505,ENSEMBL:ENSMUSG00000053830
</td>
<td style="text-align:left;">
MGI:3704365
</td>
<td style="text-align:left;">
pseudogene
</td>
<td style="text-align:left;">
predicted pseudogene 9923
</td>
</tr>
<tr>
<td style="text-align:left;">
10
</td>
<td style="text-align:left;">
gene
</td>
<td style="text-align:right;">
72.52755
</td>
<td style="text-align:right;">
72.52768
</td>
<td style="text-align:left;">
positive
</td>
<td style="text-align:left;">
MGI_C57BL6J_5454761
</td>
<td style="text-align:left;">
Gm24984
</td>
<td style="text-align:left;">
ENSEMBL:ENSMUSG00000077642
</td>
<td style="text-align:left;">
MGI:5454761
</td>
<td style="text-align:left;">
snoRNA gene
</td>
<td style="text-align:left;">
predicted gene, 24984
</td>
</tr>
<tr>
<td style="text-align:left;">
10
</td>
<td style="text-align:left;">
gene
</td>
<td style="text-align:right;">
72.62307
</td>
<td style="text-align:right;">
72.64304
</td>
<td style="text-align:left;">
negative
</td>
<td style="text-align:left;">
MGI_C57BL6J_5593818
</td>
<td style="text-align:left;">
Gm34659
</td>
<td style="text-align:left;">
NCBI_Gene:102637980
</td>
<td style="text-align:left;">
MGI:5593818
</td>
<td style="text-align:left;">
lncRNA gene
</td>
<td style="text-align:left;">
predicted gene, 34659
</td>
</tr>
<tr>
<td style="text-align:left;">
10
</td>
<td style="text-align:left;">
gene
</td>
<td style="text-align:right;">
72.65484
</td>
<td style="text-align:right;">
72.67496
</td>
<td style="text-align:left;">
positive
</td>
<td style="text-align:left;">
MGI_C57BL6J_1289227
</td>
<td style="text-align:left;">
Zwint
</td>
<td style="text-align:left;">
NCBI_Gene:52696,ENSEMBL:ENSMUSG00000019923
</td>
<td style="text-align:left;">
MGI:1289227
</td>
<td style="text-align:left;">
protein coding gene
</td>
<td style="text-align:left;">
ZW10 interactor
</td>
</tr>
<tr>
<td style="text-align:left;">
10
</td>
<td style="text-align:left;">
gene
</td>
<td style="text-align:right;">
72.65790
</td>
<td style="text-align:right;">
72.66177
</td>
<td style="text-align:left;">
positive
</td>
<td style="text-align:left;">
MGI_C57BL6J_1916486
</td>
<td style="text-align:left;">
2610034E01Rik
</td>
<td style="text-align:left;">
NA
</td>
<td style="text-align:left;">
NA
</td>
<td style="text-align:left;">
unclassified gene
</td>
<td style="text-align:left;">
RIKEN cDNA 2610034E01 gene
</td>
</tr>
<tr>
<td style="text-align:left;">
10
</td>
<td style="text-align:left;">
pseudogene
</td>
<td style="text-align:right;">
72.71793
</td>
<td style="text-align:right;">
72.71849
</td>
<td style="text-align:left;">
positive
</td>
<td style="text-align:left;">
MGI_C57BL6J_6095826
</td>
<td style="text-align:left;">
Gm47094
</td>
<td style="text-align:left;">
ENSEMBL:ENSMUSG00000112862
</td>
<td style="text-align:left;">
MGI:6095826
</td>
<td style="text-align:left;">
pseudogene
</td>
<td style="text-align:left;">
predicted gene, 47094
</td>
</tr>
<tr>
<td style="text-align:left;">
10
</td>
<td style="text-align:left;">
gene
</td>
<td style="text-align:right;">
72.93994
</td>
<td style="text-align:right;">
72.97316
</td>
<td style="text-align:left;">
negative
</td>
<td style="text-align:left;">
MGI_C57BL6J_5593935
</td>
<td style="text-align:left;">
Gm34776
</td>
<td style="text-align:left;">
NCBI_Gene:102638141,ENSEMBL:ENSMUSG00000112168
</td>
<td style="text-align:left;">
MGI:5593935
</td>
<td style="text-align:left;">
lncRNA gene
</td>
<td style="text-align:left;">
predicted gene, 34776
</td>
</tr>
<tr>
<td style="text-align:left;">
10
</td>
<td style="text-align:left;">
gene
</td>
<td style="text-align:right;">
72.97330
</td>
<td style="text-align:right;">
73.08670
</td>
<td style="text-align:left;">
negative
</td>
<td style="text-align:left;">
MGI_C57BL6J_3041175
</td>
<td style="text-align:left;">
A330049N07Rik
</td>
<td style="text-align:left;">
NCBI_Gene:327768,ENSEMBL:ENSMUSG00000111994
</td>
<td style="text-align:left;">
MGI:3041175
</td>
<td style="text-align:left;">
lncRNA gene
</td>
<td style="text-align:left;">
RIKEN cDNA A330049N07 gene
</td>
</tr>
<tr>
<td style="text-align:left;">
10
</td>
<td style="text-align:left;">
gene
</td>
<td style="text-align:right;">
73.09627
</td>
<td style="text-align:right;">
74.64983
</td>
<td style="text-align:left;">
positive
</td>
<td style="text-align:left;">
MGI_C57BL6J_1891428
</td>
<td style="text-align:left;">
Pcdh15
</td>
<td style="text-align:left;">
NCBI_Gene:11994,ENSEMBL:ENSMUSG00000052613
</td>
<td style="text-align:left;">
MGI:1891428
</td>
<td style="text-align:left;">
protein coding gene
</td>
<td style="text-align:left;">
protocadherin 15
</td>
</tr>
<tr>
<td style="text-align:left;">
10
</td>
<td style="text-align:left;">
gene
</td>
<td style="text-align:right;">
73.18293
</td>
<td style="text-align:right;">
73.18332
</td>
<td style="text-align:left;">
negative
</td>
<td style="text-align:left;">
MGI_C57BL6J_5610666
</td>
<td style="text-align:left;">
Gm37438
</td>
<td style="text-align:left;">
ENSEMBL:ENSMUSG00000102584
</td>
<td style="text-align:left;">
MGI:5610666
</td>
<td style="text-align:left;">
unclassified gene
</td>
<td style="text-align:left;">
predicted gene, 37438
</td>
</tr>
<tr>
<td style="text-align:left;">
10
</td>
<td style="text-align:left;">
gene
</td>
<td style="text-align:right;">
73.20140
</td>
<td style="text-align:right;">
73.32717
</td>
<td style="text-align:left;">
negative
</td>
<td style="text-align:left;">
MGI_C57BL6J_3705254
</td>
<td style="text-align:left;">
Gm15398
</td>
<td style="text-align:left;">
ENSEMBL:ENSMUSG00000085456
</td>
<td style="text-align:left;">
MGI:3705254
</td>
<td style="text-align:left;">
lncRNA gene
</td>
<td style="text-align:left;">
predicted gene 15398
</td>
</tr>
<tr>
<td style="text-align:left;">
10
</td>
<td style="text-align:left;">
pseudogene
</td>
<td style="text-align:right;">
73.20153
</td>
<td style="text-align:right;">
73.20277
</td>
<td style="text-align:left;">
positive
</td>
<td style="text-align:left;">
MGI_C57BL6J_3646275
</td>
<td style="text-align:left;">
Gm5778
</td>
<td style="text-align:left;">
NCBI_Gene:544705,ENSEMBL:ENSMUSG00000091825
</td>
<td style="text-align:left;">
MGI:3646275
</td>
<td style="text-align:left;">
pseudogene
</td>
<td style="text-align:left;">
predicted gene 5778
</td>
</tr>
<tr>
<td style="text-align:left;">
10
</td>
<td style="text-align:left;">
pseudogene
</td>
<td style="text-align:right;">
73.27574
</td>
<td style="text-align:right;">
73.27609
</td>
<td style="text-align:left;">
positive
</td>
<td style="text-align:left;">
MGI_C57BL6J_6095833
</td>
<td style="text-align:left;">
Gm47098
</td>
<td style="text-align:left;">
ENSEMBL:ENSMUSG00000112838
</td>
<td style="text-align:left;">
MGI:6095833
</td>
<td style="text-align:left;">
pseudogene
</td>
<td style="text-align:left;">
predicted gene, 47098
</td>
</tr>
<tr>
<td style="text-align:left;">
10
</td>
<td style="text-align:left;">
pseudogene
</td>
<td style="text-align:right;">
73.37233
</td>
<td style="text-align:right;">
73.37603
</td>
<td style="text-align:left;">
negative
</td>
<td style="text-align:left;">
MGI_C57BL6J_5011353
</td>
<td style="text-align:left;">
Gm19168
</td>
<td style="text-align:left;">
NCBI_Gene:100418377,ENSEMBL:ENSMUSG00000112427
</td>
<td style="text-align:left;">
MGI:5011353
</td>
<td style="text-align:left;">
pseudogene
</td>
<td style="text-align:left;">
predicted gene, 19168
</td>
</tr>
<tr>
<td style="text-align:left;">
10
</td>
<td style="text-align:left;">
gene
</td>
<td style="text-align:right;">
73.48997
</td>
<td style="text-align:right;">
73.51839
</td>
<td style="text-align:left;">
positive
</td>
<td style="text-align:left;">
MGI_C57BL6J_3705298
</td>
<td style="text-align:left;">
Gm15397
</td>
<td style="text-align:left;">
ENSEMBL:ENSMUSG00000085176
</td>
<td style="text-align:left;">
MGI:3705298
</td>
<td style="text-align:left;">
lncRNA gene
</td>
<td style="text-align:left;">
predicted gene 15397
</td>
</tr>
<tr>
<td style="text-align:left;">
10
</td>
<td style="text-align:left;">
pseudogene
</td>
<td style="text-align:right;">
73.69010
</td>
<td style="text-align:right;">
73.69101
</td>
<td style="text-align:left;">
negative
</td>
<td style="text-align:left;">
MGI_C57BL6J_3642965
</td>
<td style="text-align:left;">
Gm6407
</td>
<td style="text-align:left;">
NCBI_Gene:623191,ENSEMBL:ENSMUSG00000093548
</td>
<td style="text-align:left;">
MGI:3642965
</td>
<td style="text-align:left;">
pseudogene
</td>
<td style="text-align:left;">
predicted gene 6407
</td>
</tr>
<tr>
<td style="text-align:left;">
10
</td>
<td style="text-align:left;">
gene
</td>
<td style="text-align:right;">
73.88745
</td>
<td style="text-align:right;">
73.88812
</td>
<td style="text-align:left;">
positive
</td>
<td style="text-align:left;">
MGI_C57BL6J_1925054
</td>
<td style="text-align:left;">
A930019H14Rik
</td>
<td style="text-align:left;">
NA
</td>
<td style="text-align:left;">
NA
</td>
<td style="text-align:left;">
unclassified gene
</td>
<td style="text-align:left;">
RIKEN cDNA A930019H14 gene
</td>
</tr>
<tr>
<td style="text-align:left;">
10
</td>
<td style="text-align:left;">
gene
</td>
<td style="text-align:right;">
74.12754
</td>
<td style="text-align:right;">
74.13688
</td>
<td style="text-align:left;">
negative
</td>
<td style="text-align:left;">
MGI_C57BL6J_5623576
</td>
<td style="text-align:left;">
Gm40691
</td>
<td style="text-align:left;">
NCBI_Gene:105245206
</td>
<td style="text-align:left;">
MGI:5623576
</td>
<td style="text-align:left;">
lncRNA gene
</td>
<td style="text-align:left;">
predicted gene, 40691
</td>
</tr>
<tr>
<td style="text-align:left;">
10
</td>
<td style="text-align:left;">
pseudogene
</td>
<td style="text-align:right;">
74.82063
</td>
<td style="text-align:right;">
74.82149
</td>
<td style="text-align:left;">
positive
</td>
<td style="text-align:left;">
MGI_C57BL6J_3644650
</td>
<td style="text-align:left;">
Gm7034
</td>
<td style="text-align:left;">
NCBI_Gene:630453,ENSEMBL:ENSMUSG00000112686
</td>
<td style="text-align:left;">
MGI:3644650
</td>
<td style="text-align:left;">
pseudogene
</td>
<td style="text-align:left;">
predicted gene 7034
</td>
</tr>
<tr>
<td style="text-align:left;">
10
</td>
<td style="text-align:left;">
pseudogene
</td>
<td style="text-align:right;">
74.86286
</td>
<td style="text-align:right;">
74.86351
</td>
<td style="text-align:left;">
negative
</td>
<td style="text-align:left;">
MGI_C57BL6J_3647564
</td>
<td style="text-align:left;">
Gm6419
</td>
<td style="text-align:left;">
NCBI_Gene:623346,ENSEMBL:ENSMUSG00000072407
</td>
<td style="text-align:left;">
MGI:3647564
</td>
<td style="text-align:left;">
pseudogene
</td>
<td style="text-align:left;">
predicted gene 6419
</td>
</tr>
<tr>
<td style="text-align:left;">
10
</td>
<td style="text-align:left;">
gene
</td>
<td style="text-align:right;">
74.95748
</td>
<td style="text-align:right;">
75.03435
</td>
<td style="text-align:left;">
negative
</td>
<td style="text-align:left;">
MGI_C57BL6J_1918486
</td>
<td style="text-align:left;">
Rsph14
</td>
<td style="text-align:left;">
NCBI_Gene:71236,ENSEMBL:ENSMUSG00000009070
</td>
<td style="text-align:left;">
MGI:1918486
</td>
<td style="text-align:left;">
protein coding gene
</td>
<td style="text-align:left;">
radial spoke head homolog 14 (Chlamydomonas)
</td>
</tr>
<tr>
<td style="text-align:left;">
10
</td>
<td style="text-align:left;">
gene
</td>
<td style="text-align:right;">
74.96709
</td>
<td style="text-align:right;">
75.01691
</td>
<td style="text-align:left;">
positive
</td>
<td style="text-align:left;">
MGI_C57BL6J_95780
</td>
<td style="text-align:left;">
Gnaz
</td>
<td style="text-align:left;">
NCBI_Gene:14687,ENSEMBL:ENSMUSG00000040009
</td>
<td style="text-align:left;">
MGI:95780
</td>
<td style="text-align:left;">
protein coding gene
</td>
<td style="text-align:left;">
guanine nucleotide binding protein, alpha z subunit
</td>
</tr>
<tr>
<td style="text-align:left;">
10
</td>
<td style="text-align:left;">
gene
</td>
<td style="text-align:right;">
75.02537
</td>
<td style="text-align:right;">
75.03503
</td>
<td style="text-align:left;">
positive
</td>
<td style="text-align:left;">
MGI_C57BL6J_5623577
</td>
<td style="text-align:left;">
Gm40692
</td>
<td style="text-align:left;">
NCBI_Gene:105245207,ENSEMBL:ENSMUSG00000112136
</td>
<td style="text-align:left;">
MGI:5623577
</td>
<td style="text-align:left;">
lncRNA gene
</td>
<td style="text-align:left;">
predicted gene, 40692
</td>
</tr>
<tr>
<td style="text-align:left;">
10
</td>
<td style="text-align:left;">
gene
</td>
<td style="text-align:right;">
75.03690
</td>
<td style="text-align:right;">
75.05475
</td>
<td style="text-align:left;">
positive
</td>
<td style="text-align:left;">
MGI_C57BL6J_1924127
</td>
<td style="text-align:left;">
Rab36
</td>
<td style="text-align:left;">
NCBI_Gene:76877,ENSEMBL:ENSMUSG00000020175
</td>
<td style="text-align:left;">
MGI:1924127
</td>
<td style="text-align:left;">
protein coding gene
</td>
<td style="text-align:left;">
RAB36, member RAS oncogene family
</td>
</tr>
<tr>
<td style="text-align:left;">
10
</td>
<td style="text-align:left;">
gene
</td>
<td style="text-align:right;">
75.06059
</td>
<td style="text-align:right;">
75.18492
</td>
<td style="text-align:left;">
positive
</td>
<td style="text-align:left;">
MGI_C57BL6J_88141
</td>
<td style="text-align:left;">
Bcr
</td>
<td style="text-align:left;">
NCBI_Gene:110279,ENSEMBL:ENSMUSG00000009681
</td>
<td style="text-align:left;">
MGI:88141
</td>
<td style="text-align:left;">
protein coding gene
</td>
<td style="text-align:left;">
breakpoint cluster region
</td>
</tr>
<tr>
<td style="text-align:left;">
10
</td>
<td style="text-align:left;">
gene
</td>
<td style="text-align:right;">
75.21204
</td>
<td style="text-align:right;">
75.31274
</td>
<td style="text-align:left;">
positive
</td>
<td style="text-align:left;">
MGI_C57BL6J_1921642
</td>
<td style="text-align:left;">
Specc1l
</td>
<td style="text-align:left;">
NCBI_Gene:74392,ENSEMBL:ENSMUSG00000033444
</td>
<td style="text-align:left;">
MGI:1921642
</td>
<td style="text-align:left;">
protein coding gene
</td>
<td style="text-align:left;">
sperm antigen with calponin homology and coiled-coil domains 1-like
</td>
</tr>
<tr>
<td style="text-align:left;">
10
</td>
<td style="text-align:left;">
pseudogene
</td>
<td style="text-align:right;">
75.28135
</td>
<td style="text-align:right;">
75.28322
</td>
<td style="text-align:left;">
negative
</td>
<td style="text-align:left;">
MGI_C57BL6J_5825871
</td>
<td style="text-align:left;">
Gm46234
</td>
<td style="text-align:left;">
NCBI_Gene:108167821,ENSEMBL:ENSMUSG00000112116
</td>
<td style="text-align:left;">
MGI:5825871
</td>
<td style="text-align:left;">
pseudogene
</td>
<td style="text-align:left;">
predicted gene, 46234
</td>
</tr>
<tr>
<td style="text-align:left;">
10
</td>
<td style="text-align:left;">
gene
</td>
<td style="text-align:right;">
75.31686
</td>
<td style="text-align:right;">
75.33479
</td>
<td style="text-align:left;">
positive
</td>
<td style="text-align:left;">
MGI_C57BL6J_99402
</td>
<td style="text-align:left;">
Adora2a
</td>
<td style="text-align:left;">
NCBI_Gene:11540,ENSEMBL:ENSMUSG00000020178
</td>
<td style="text-align:left;">
MGI:99402
</td>
<td style="text-align:left;">
protein coding gene
</td>
<td style="text-align:left;">
adenosine A2a receptor
</td>
</tr>
<tr>
<td style="text-align:left;">
10
</td>
<td style="text-align:left;">
pseudogene
</td>
<td style="text-align:right;">
75.34447
</td>
<td style="text-align:right;">
75.35303
</td>
<td style="text-align:left;">
positive
</td>
<td style="text-align:left;">
MGI_C57BL6J_3645934
</td>
<td style="text-align:left;">
Gm5779
</td>
<td style="text-align:left;">
NCBI_Gene:544707,ENSEMBL:ENSMUSG00000112471
</td>
<td style="text-align:left;">
MGI:3645934
</td>
<td style="text-align:left;">
pseudogene
</td>
<td style="text-align:left;">
predicted gene 5779
</td>
</tr>
<tr>
<td style="text-align:left;">
10
</td>
<td style="text-align:left;">
gene
</td>
<td style="text-align:right;">
75.35111
</td>
<td style="text-align:right;">
75.51797
</td>
<td style="text-align:left;">
negative
</td>
<td style="text-align:left;">
MGI_C57BL6J_1916028
</td>
<td style="text-align:left;">
Gucd1
</td>
<td style="text-align:left;">
NCBI_Gene:68778,ENSEMBL:ENSMUSG00000033416
</td>
<td style="text-align:left;">
MGI:1916028
</td>
<td style="text-align:left;">
protein coding gene
</td>
<td style="text-align:left;">
guanylyl cyclase domain containing 1
</td>
</tr>
<tr>
<td style="text-align:left;">
10
</td>
<td style="text-align:left;">
gene
</td>
<td style="text-align:right;">
75.40075
</td>
<td style="text-align:right;">
75.40225
</td>
<td style="text-align:left;">
positive
</td>
<td style="text-align:left;">
MGI_C57BL6J_5623579
</td>
<td style="text-align:left;">
Gm40694
</td>
<td style="text-align:left;">
NCBI_Gene:105245209
</td>
<td style="text-align:left;">
MGI:5623579
</td>
<td style="text-align:left;">
lncRNA gene
</td>
<td style="text-align:left;">
predicted gene, 40694
</td>
</tr>
<tr>
<td style="text-align:left;">
10
</td>
<td style="text-align:left;">
gene
</td>
<td style="text-align:right;">
75.40112
</td>
<td style="text-align:right;">
75.44168
</td>
<td style="text-align:left;">
positive
</td>
<td style="text-align:left;">
MGI_C57BL6J_2143535
</td>
<td style="text-align:left;">
Upb1
</td>
<td style="text-align:left;">
NCBI_Gene:103149,ENSEMBL:ENSMUSG00000033427
</td>
<td style="text-align:left;">
MGI:2143535
</td>
<td style="text-align:left;">
protein coding gene
</td>
<td style="text-align:left;">
ureidopropionase, beta
</td>
</tr>
<tr>
<td style="text-align:left;">
10
</td>
<td style="text-align:left;">
gene
</td>
<td style="text-align:right;">
75.44262
</td>
<td style="text-align:right;">
75.44294
</td>
<td style="text-align:left;">
positive
</td>
<td style="text-align:left;">
MGI_C57BL6J_5455322
</td>
<td style="text-align:left;">
Gm25545
</td>
<td style="text-align:left;">
ENSEMBL:ENSMUSG00000084622
</td>
<td style="text-align:left;">
MGI:5455322
</td>
<td style="text-align:left;">
unclassified non-coding RNA gene
</td>
<td style="text-align:left;">
predicted gene, 25545
</td>
</tr>
<tr>
<td style="text-align:left;">
10
</td>
<td style="text-align:left;">
gene
</td>
<td style="text-align:right;">
75.44951
</td>
<td style="text-align:right;">
75.46310
</td>
<td style="text-align:left;">
positive
</td>
<td style="text-align:left;">
MGI_C57BL6J_1918375
</td>
<td style="text-align:left;">
4933407G14Rik
</td>
<td style="text-align:left;">
NCBI_Gene:71125
</td>
<td style="text-align:left;">
MGI:1918375
</td>
<td style="text-align:left;">
lncRNA gene
</td>
<td style="text-align:left;">
RIKEN cDNA 4933407G14 gene
</td>
</tr>
<tr>
<td style="text-align:left;">
10
</td>
<td style="text-align:left;">
pseudogene
</td>
<td style="text-align:right;">
75.48309
</td>
<td style="text-align:right;">
75.48339
</td>
<td style="text-align:left;">
negative
</td>
<td style="text-align:left;">
MGI_C57BL6J_6097527
</td>
<td style="text-align:left;">
Gm48158
</td>
<td style="text-align:left;">
ENSEMBL:ENSMUSG00000112764
</td>
<td style="text-align:left;">
MGI:6097527
</td>
<td style="text-align:left;">
pseudogene
</td>
<td style="text-align:left;">
predicted gene, 48158
</td>
</tr>
<tr>
<td style="text-align:left;">
10
</td>
<td style="text-align:left;">
pseudogene
</td>
<td style="text-align:right;">
75.49606
</td>
<td style="text-align:right;">
75.49616
</td>
<td style="text-align:left;">
positive
</td>
<td style="text-align:left;">
MGI_C57BL6J_6097516
</td>
<td style="text-align:left;">
Gm48148
</td>
<td style="text-align:left;">
ENSEMBL:ENSMUSG00000111955
</td>
<td style="text-align:left;">
MGI:6097516
</td>
<td style="text-align:left;">
pseudogene
</td>
<td style="text-align:left;">
predicted gene, 48148
</td>
</tr>
<tr>
<td style="text-align:left;">
10
</td>
<td style="text-align:left;">
gene
</td>
<td style="text-align:right;">
75.51755
</td>
<td style="text-align:right;">
75.53738
</td>
<td style="text-align:left;">
positive
</td>
<td style="text-align:left;">
MGI_C57BL6J_1914582
</td>
<td style="text-align:left;">
Snrpd3
</td>
<td style="text-align:left;">
NCBI_Gene:67332,ENSEMBL:ENSMUSG00000020180
</td>
<td style="text-align:left;">
MGI:1914582
</td>
<td style="text-align:left;">
protein coding gene
</td>
<td style="text-align:left;">
small nuclear ribonucleoprotein D3
</td>
</tr>
<tr>
<td style="text-align:left;">
10
</td>
<td style="text-align:left;">
pseudogene
</td>
<td style="text-align:right;">
75.52629
</td>
<td style="text-align:right;">
75.52694
</td>
<td style="text-align:left;">
negative
</td>
<td style="text-align:left;">
MGI_C57BL6J_5594220
</td>
<td style="text-align:left;">
Gm35061
</td>
<td style="text-align:left;">
NCBI_Gene:102638517
</td>
<td style="text-align:left;">
MGI:5594220
</td>
<td style="text-align:left;">
pseudogene
</td>
<td style="text-align:left;">
predicted gene, 35061
</td>
</tr>
<tr>
<td style="text-align:left;">
10
</td>
<td style="text-align:left;">
gene
</td>
<td style="text-align:right;">
75.55012
</td>
<td style="text-align:right;">
75.56092
</td>
<td style="text-align:left;">
negative
</td>
<td style="text-align:left;">
MGI_C57BL6J_2143657
</td>
<td style="text-align:left;">
Lrrc75b
</td>
<td style="text-align:left;">
NCBI_Gene:192734,ENSEMBL:ENSMUSG00000046807
</td>
<td style="text-align:left;">
MGI:2143657
</td>
<td style="text-align:left;">
protein coding gene
</td>
<td style="text-align:left;">
leucine rich repeat containing 75B
</td>
</tr>
<tr>
<td style="text-align:left;">
10
</td>
<td style="text-align:left;">
gene
</td>
<td style="text-align:right;">
75.56160
</td>
<td style="text-align:right;">
75.58620
</td>
<td style="text-align:left;">
positive
</td>
<td style="text-align:left;">
MGI_C57BL6J_95706
</td>
<td style="text-align:left;">
Ggt1
</td>
<td style="text-align:left;">
NCBI_Gene:14598,ENSEMBL:ENSMUSG00000006345
</td>
<td style="text-align:left;">
MGI:95706
</td>
<td style="text-align:left;">
protein coding gene
</td>
<td style="text-align:left;">
gamma-glutamyltransferase 1
</td>
</tr>
<tr>
<td style="text-align:left;">
10
</td>
<td style="text-align:left;">
gene
</td>
<td style="text-align:right;">
75.58932
</td>
<td style="text-align:right;">
75.61720
</td>
<td style="text-align:left;">
positive
</td>
<td style="text-align:left;">
MGI_C57BL6J_1346063
</td>
<td style="text-align:left;">
Ggt5
</td>
<td style="text-align:left;">
NCBI_Gene:23887,ENSEMBL:ENSMUSG00000006344
</td>
<td style="text-align:left;">
MGI:1346063
</td>
<td style="text-align:left;">
protein coding gene
</td>
<td style="text-align:left;">
gamma-glutamyltransferase 5
</td>
</tr>
<tr>
<td style="text-align:left;">
10
</td>
<td style="text-align:left;">
pseudogene
</td>
<td style="text-align:right;">
75.59014
</td>
<td style="text-align:right;">
75.59065
</td>
<td style="text-align:left;">
negative
</td>
<td style="text-align:left;">
MGI_C57BL6J_3708539
</td>
<td style="text-align:left;">
Gm9985
</td>
<td style="text-align:left;">
ENSEMBL:ENSMUSG00000055872
</td>
<td style="text-align:left;">
MGI:3708539
</td>
<td style="text-align:left;">
pseudogene
</td>
<td style="text-align:left;">
predicted gene 9985
</td>
</tr>
<tr>
<td style="text-align:left;">
10
</td>
<td style="text-align:left;">
gene
</td>
<td style="text-align:right;">
75.61316
</td>
<td style="text-align:right;">
75.61327
</td>
<td style="text-align:left;">
negative
</td>
<td style="text-align:left;">
MGI_C57BL6J_5456179
</td>
<td style="text-align:left;">
Gm26402
</td>
<td style="text-align:left;">
ENSEMBL:ENSMUSG00000089124
</td>
<td style="text-align:left;">
MGI:5456179
</td>
<td style="text-align:left;">
snRNA gene
</td>
<td style="text-align:left;">
predicted gene, 26402
</td>
</tr>
<tr>
<td style="text-align:left;">
10
</td>
<td style="text-align:left;">
pseudogene
</td>
<td style="text-align:right;">
75.61903
</td>
<td style="text-align:right;">
75.61977
</td>
<td style="text-align:left;">
negative
</td>
<td style="text-align:left;">
MGI_C57BL6J_5011873
</td>
<td style="text-align:left;">
Gm19688
</td>
<td style="text-align:left;">
NCBI_Gene:100503432,ENSEMBL:ENSMUSG00000112643
</td>
<td style="text-align:left;">
MGI:5011873
</td>
<td style="text-align:left;">
pseudogene
</td>
<td style="text-align:left;">
predicted gene, 19688
</td>
</tr>
<tr>
<td style="text-align:left;">
10
</td>
<td style="text-align:left;">
gene
</td>
<td style="text-align:right;">
75.63235
</td>
<td style="text-align:right;">
75.63640
</td>
<td style="text-align:left;">
negative
</td>
<td style="text-align:left;">
MGI_C57BL6J_5623580
</td>
<td style="text-align:left;">
Gm40695
</td>
<td style="text-align:left;">
NCBI_Gene:105245210
</td>
<td style="text-align:left;">
MGI:5623580
</td>
<td style="text-align:left;">
lncRNA gene
</td>
<td style="text-align:left;">
predicted gene, 40695
</td>
</tr>
<tr>
<td style="text-align:left;">
10
</td>
<td style="text-align:left;">
gene
</td>
<td style="text-align:right;">
75.63662
</td>
<td style="text-align:right;">
75.64401
</td>
<td style="text-align:left;">
negative
</td>
<td style="text-align:left;">
MGI_C57BL6J_1918983
</td>
<td style="text-align:left;">
Susd2
</td>
<td style="text-align:left;">
NCBI_Gene:71733,ENSEMBL:ENSMUSG00000006342
</td>
<td style="text-align:left;">
MGI:1918983
</td>
<td style="text-align:left;">
protein coding gene
</td>
<td style="text-align:left;">
sushi domain containing 2
</td>
</tr>
<tr>
<td style="text-align:left;">
10
</td>
<td style="text-align:left;">
gene
</td>
<td style="text-align:right;">
75.64611
</td>
<td style="text-align:right;">
75.76436
</td>
<td style="text-align:left;">
negative
</td>
<td style="text-align:left;">
MGI_C57BL6J_1298375
</td>
<td style="text-align:left;">
Cabin1
</td>
<td style="text-align:left;">
NCBI_Gene:104248,ENSEMBL:ENSMUSG00000020196
</td>
<td style="text-align:left;">
MGI:1298375
</td>
<td style="text-align:left;">
protein coding gene
</td>
<td style="text-align:left;">
calcineurin binding protein 1
</td>
</tr>
<tr>
<td style="text-align:left;">
10
</td>
<td style="text-align:left;">
gene
</td>
<td style="text-align:right;">
75.69505
</td>
<td style="text-align:right;">
75.70926
</td>
<td style="text-align:left;">
positive
</td>
<td style="text-align:left;">
MGI_C57BL6J_5623581
</td>
<td style="text-align:left;">
Gm40696
</td>
<td style="text-align:left;">
NCBI_Gene:105245211
</td>
<td style="text-align:left;">
MGI:5623581
</td>
<td style="text-align:left;">
lncRNA gene
</td>
<td style="text-align:left;">
predicted gene, 40696
</td>
</tr>
<tr>
<td style="text-align:left;">
10
</td>
<td style="text-align:left;">
pseudogene
</td>
<td style="text-align:right;">
75.76626
</td>
<td style="text-align:right;">
75.76791
</td>
<td style="text-align:left;">
positive
</td>
<td style="text-align:left;">
MGI_C57BL6J_5011511
</td>
<td style="text-align:left;">
Gm19326
</td>
<td style="text-align:left;">
NCBI_Gene:105245212,ENSEMBL:ENSMUSG00000112789
</td>
<td style="text-align:left;">
MGI:5011511
</td>
<td style="text-align:left;">
pseudogene
</td>
<td style="text-align:left;">
predicted gene, 19326
</td>
</tr>
<tr>
<td style="text-align:left;">
10
</td>
<td style="text-align:left;">
gene
</td>
<td style="text-align:right;">
75.77123
</td>
<td style="text-align:right;">
75.77341
</td>
<td style="text-align:left;">
negative
</td>
<td style="text-align:left;">
MGI_C57BL6J_1298381
</td>
<td style="text-align:left;">
Ddt
</td>
<td style="text-align:left;">
NCBI_Gene:13202,ENSEMBL:ENSMUSG00000001666
</td>
<td style="text-align:left;">
MGI:1298381
</td>
<td style="text-align:left;">
protein coding gene
</td>
<td style="text-align:left;">
D-dopachrome tautomerase
</td>
</tr>
<tr>
<td style="text-align:left;">
10
</td>
<td style="text-align:left;">
gene
</td>
<td style="text-align:right;">
75.77145
</td>
<td style="text-align:right;">
75.78095
</td>
<td style="text-align:left;">
negative
</td>
<td style="text-align:left;">
MGI_C57BL6J_5141906
</td>
<td style="text-align:left;">
Gm20441
</td>
<td style="text-align:left;">
ENSEMBL:ENSMUSG00000092360
</td>
<td style="text-align:left;">
MGI:5141906
</td>
<td style="text-align:left;">
protein coding gene
</td>
<td style="text-align:left;">
predicted gene 20441
</td>
</tr>
<tr>
<td style="text-align:left;">
10
</td>
<td style="text-align:left;">
gene
</td>
<td style="text-align:right;">
75.77411
</td>
<td style="text-align:right;">
75.78141
</td>
<td style="text-align:left;">
negative
</td>
<td style="text-align:left;">
MGI_C57BL6J_2143526
</td>
<td style="text-align:left;">
Gstt3
</td>
<td style="text-align:left;">
NCBI_Gene:103140,ENSEMBL:ENSMUSG00000001665
</td>
<td style="text-align:left;">
MGI:2143526
</td>
<td style="text-align:left;">
protein coding gene
</td>
<td style="text-align:left;">
glutathione S-transferase, theta 3
</td>
</tr>
<tr>
<td style="text-align:left;">
10
</td>
<td style="text-align:left;">
gene
</td>
<td style="text-align:right;">
75.78381
</td>
<td style="text-align:right;">
75.79858
</td>
<td style="text-align:left;">
negative
</td>
<td style="text-align:left;">
MGI_C57BL6J_107379
</td>
<td style="text-align:left;">
Gstt1
</td>
<td style="text-align:left;">
NCBI_Gene:14871,ENSEMBL:ENSMUSG00000001663
</td>
<td style="text-align:left;">
MGI:107379
</td>
<td style="text-align:left;">
protein coding gene
</td>
<td style="text-align:left;">
glutathione S-transferase, theta 1
</td>
</tr>
<tr>
<td style="text-align:left;">
10
</td>
<td style="text-align:left;">
gene
</td>
<td style="text-align:right;">
75.81494
</td>
<td style="text-align:right;">
75.82256
</td>
<td style="text-align:left;">
negative
</td>
<td style="text-align:left;">
MGI_C57BL6J_1923136
</td>
<td style="text-align:left;">
Gstt4
</td>
<td style="text-align:left;">
NCBI_Gene:75886,ENSEMBL:ENSMUSG00000009093
</td>
<td style="text-align:left;">
MGI:1923136
</td>
<td style="text-align:left;">
protein coding gene
</td>
<td style="text-align:left;">
glutathione S-transferase, theta 4
</td>
</tr>
<tr>
<td style="text-align:left;">
10
</td>
<td style="text-align:left;">
gene
</td>
<td style="text-align:right;">
75.83111
</td>
<td style="text-align:right;">
75.83742
</td>
<td style="text-align:left;">
negative
</td>
<td style="text-align:left;">
MGI_C57BL6J_106188
</td>
<td style="text-align:left;">
Gstt2
</td>
<td style="text-align:left;">
NCBI_Gene:14872,ENSEMBL:ENSMUSG00000033318
</td>
<td style="text-align:left;">
MGI:106188
</td>
<td style="text-align:left;">
protein coding gene
</td>
<td style="text-align:left;">
glutathione S-transferase, theta 2
</td>
</tr>
<tr>
<td style="text-align:left;">
10
</td>
<td style="text-align:left;">
gene
</td>
<td style="text-align:right;">
75.83598
</td>
<td style="text-align:right;">
75.83742
</td>
<td style="text-align:left;">
negative
</td>
<td style="text-align:left;">
MGI_C57BL6J_1918377
</td>
<td style="text-align:left;">
4933425E08Rik
</td>
<td style="text-align:left;">
NA
</td>
<td style="text-align:left;">
NA
</td>
<td style="text-align:left;">
unclassified gene
</td>
<td style="text-align:left;">
RIKEN cDNA 4933425E08 gene
</td>
</tr>
<tr>
<td style="text-align:left;">
10
</td>
<td style="text-align:left;">
gene
</td>
<td style="text-align:right;">
75.85935
</td>
<td style="text-align:right;">
75.86025
</td>
<td style="text-align:left;">
negative
</td>
<td style="text-align:left;">
MGI_C57BL6J_96982
</td>
<td style="text-align:left;">
Mif
</td>
<td style="text-align:left;">
NCBI_Gene:17319,ENSEMBL:ENSMUSG00000033307
</td>
<td style="text-align:left;">
MGI:96982
</td>
<td style="text-align:left;">
protein coding gene
</td>
<td style="text-align:left;">
macrophage migration inhibitory factor (glycosylation-inhibiting factor)
</td>
</tr>
<tr>
<td style="text-align:left;">
10
</td>
<td style="text-align:left;">
gene
</td>
<td style="text-align:right;">
75.86040
</td>
<td style="text-align:right;">
75.86170
</td>
<td style="text-align:left;">
positive
</td>
<td style="text-align:left;">
MGI_C57BL6J_5623582
</td>
<td style="text-align:left;">
Gm40697
</td>
<td style="text-align:left;">
NCBI_Gene:105245213
</td>
<td style="text-align:left;">
MGI:5623582
</td>
<td style="text-align:left;">
lncRNA gene
</td>
<td style="text-align:left;">
predicted gene, 40697
</td>
</tr>
<tr>
<td style="text-align:left;">
10
</td>
<td style="text-align:left;">
pseudogene
</td>
<td style="text-align:right;">
75.87543
</td>
<td style="text-align:right;">
75.87580
</td>
<td style="text-align:left;">
positive
</td>
<td style="text-align:left;">
MGI_C57BL6J_6096888
</td>
<td style="text-align:left;">
Gm47744
</td>
<td style="text-align:left;">
ENSEMBL:ENSMUSG00000112306
</td>
<td style="text-align:left;">
MGI:6096888
</td>
<td style="text-align:left;">
pseudogene
</td>
<td style="text-align:left;">
predicted gene, 47744
</td>
</tr>
<tr>
<td style="text-align:left;">
10
</td>
<td style="text-align:left;">
gene
</td>
<td style="text-align:right;">
75.89307
</td>
<td style="text-align:right;">
75.89594
</td>
<td style="text-align:left;">
positive
</td>
<td style="text-align:left;">
MGI_C57BL6J_1917627
</td>
<td style="text-align:left;">
Derl3
</td>
<td style="text-align:left;">
NCBI_Gene:70377,ENSEMBL:ENSMUSG00000009092
</td>
<td style="text-align:left;">
MGI:1917627
</td>
<td style="text-align:left;">
protein coding gene
</td>
<td style="text-align:left;">
Der1-like domain family, member 3
</td>
</tr>
<tr>
<td style="text-align:left;">
10
</td>
<td style="text-align:left;">
gene
</td>
<td style="text-align:right;">
75.89677
</td>
<td style="text-align:right;">
75.92162
</td>
<td style="text-align:left;">
negative
</td>
<td style="text-align:left;">
MGI_C57BL6J_1328366
</td>
<td style="text-align:left;">
Smarcb1
</td>
<td style="text-align:left;">
NCBI_Gene:20587,ENSEMBL:ENSMUSG00000000902
</td>
<td style="text-align:left;">
MGI:1328366
</td>
<td style="text-align:left;">
protein coding gene
</td>
<td style="text-align:left;">
SWI/SNF related, matrix associated, actin dependent regulator of chromatin, subfamily b, member 1
</td>
</tr>
<tr>
<td style="text-align:left;">
10
</td>
<td style="text-align:left;">
gene
</td>
<td style="text-align:right;">
75.92322
</td>
<td style="text-align:right;">
75.93650
</td>
<td style="text-align:left;">
negative
</td>
<td style="text-align:left;">
MGI_C57BL6J_97008
</td>
<td style="text-align:left;">
Mmp11
</td>
<td style="text-align:left;">
NCBI_Gene:17385,ENSEMBL:ENSMUSG00000000901
</td>
<td style="text-align:left;">
MGI:97008
</td>
<td style="text-align:left;">
protein coding gene
</td>
<td style="text-align:left;">
matrix metallopeptidase 11
</td>
</tr>
<tr>
<td style="text-align:left;">
10
</td>
<td style="text-align:left;">
gene
</td>
<td style="text-align:right;">
75.93297
</td>
<td style="text-align:right;">
75.93775
</td>
<td style="text-align:left;">
positive
</td>
<td style="text-align:left;">
MGI_C57BL6J_2143558
</td>
<td style="text-align:left;">
Chchd10
</td>
<td style="text-align:left;">
NCBI_Gene:103172,ENSEMBL:ENSMUSG00000049422
</td>
<td style="text-align:left;">
MGI:2143558
</td>
<td style="text-align:left;">
protein coding gene
</td>
<td style="text-align:left;">
coiled-coil-helix-coiled-coil-helix domain containing 10
</td>
</tr>
<tr>
<td style="text-align:left;">
10
</td>
<td style="text-align:left;">
gene
</td>
<td style="text-align:right;">
75.93772
</td>
<td style="text-align:right;">
75.94069
</td>
<td style="text-align:left;">
negative
</td>
<td style="text-align:left;">
MGI_C57BL6J_2685713
</td>
<td style="text-align:left;">
Gm867
</td>
<td style="text-align:left;">
NCBI_Gene:333670,ENSEMBL:ENSMUSG00000050157
</td>
<td style="text-align:left;">
MGI:2685713
</td>
<td style="text-align:left;">
protein coding gene
</td>
<td style="text-align:left;">
predicted gene 867
</td>
</tr>
<tr>
<td style="text-align:left;">
10
</td>
<td style="text-align:left;">
gene
</td>
<td style="text-align:right;">
75.94306
</td>
<td style="text-align:right;">
75.94966
</td>
<td style="text-align:left;">
positive
</td>
<td style="text-align:left;">
MGI_C57BL6J_98938
</td>
<td style="text-align:left;">
Vpreb3
</td>
<td style="text-align:left;">
NCBI_Gene:22364,ENSEMBL:ENSMUSG00000000903
</td>
<td style="text-align:left;">
MGI:98938
</td>
<td style="text-align:left;">
protein coding gene
</td>
<td style="text-align:left;">
pre-B lymphocyte gene 3
</td>
</tr>
<tr>
<td style="text-align:left;">
10
</td>
<td style="text-align:left;">
pseudogene
</td>
<td style="text-align:right;">
75.94530
</td>
<td style="text-align:right;">
75.94652
</td>
<td style="text-align:left;">
negative
</td>
<td style="text-align:left;">
MGI_C57BL6J_3802147
</td>
<td style="text-align:left;">
Gm16221
</td>
<td style="text-align:left;">
NCBI_Gene:102638935,ENSEMBL:ENSMUSG00000082234
</td>
<td style="text-align:left;">
MGI:3802147
</td>
<td style="text-align:left;">
pseudogene
</td>
<td style="text-align:left;">
predicted gene 16221
</td>
</tr>
<tr>
<td style="text-align:left;">
10
</td>
<td style="text-align:left;">
gene
</td>
<td style="text-align:right;">
75.95385
</td>
<td style="text-align:right;">
75.95428
</td>
<td style="text-align:left;">
negative
</td>
<td style="text-align:left;">
MGI_C57BL6J_5623583
</td>
<td style="text-align:left;">
Gm40698
</td>
<td style="text-align:left;">
NCBI_Gene:105245214
</td>
<td style="text-align:left;">
MGI:5623583
</td>
<td style="text-align:left;">
lncRNA gene
</td>
<td style="text-align:left;">
predicted gene, 40698
</td>
</tr>
<tr>
<td style="text-align:left;">
10
</td>
<td style="text-align:left;">
gene
</td>
<td style="text-align:right;">
75.95438
</td>
<td style="text-align:right;">
76.00959
</td>
<td style="text-align:left;">
positive
</td>
<td style="text-align:left;">
MGI_C57BL6J_3646667
</td>
<td style="text-align:left;">
Gm5134
</td>
<td style="text-align:left;">
NCBI_Gene:333669,ENSEMBL:ENSMUSG00000033255
</td>
<td style="text-align:left;">
MGI:3646667
</td>
<td style="text-align:left;">
protein coding gene
</td>
<td style="text-align:left;">
predicted gene 5134
</td>
</tr>
<tr>
<td style="text-align:left;">
10
</td>
<td style="text-align:left;">
pseudogene
</td>
<td style="text-align:right;">
75.96990
</td>
<td style="text-align:right;">
75.97017
</td>
<td style="text-align:left;">
positive
</td>
<td style="text-align:left;">
MGI_C57BL6J_3802165
</td>
<td style="text-align:left;">
Gm16219
</td>
<td style="text-align:left;">
ENSEMBL:ENSMUSG00000082619
</td>
<td style="text-align:left;">
MGI:3802165
</td>
<td style="text-align:left;">
pseudogene
</td>
<td style="text-align:left;">
predicted gene 16219
</td>
</tr>
<tr>
<td style="text-align:left;">
10
</td>
<td style="text-align:left;">
gene
</td>
<td style="text-align:right;">
75.99269
</td>
<td style="text-align:right;">
75.99399
</td>
<td style="text-align:left;">
negative
</td>
<td style="text-align:left;">
MGI_C57BL6J_3802166
</td>
<td style="text-align:left;">
Gm16220
</td>
<td style="text-align:left;">
ENSEMBL:ENSMUSG00000085662
</td>
<td style="text-align:left;">
MGI:3802166
</td>
<td style="text-align:left;">
lncRNA gene
</td>
<td style="text-align:left;">
predicted gene 16220
</td>
</tr>
<tr>
<td style="text-align:left;">
10
</td>
<td style="text-align:left;">
gene
</td>
<td style="text-align:right;">
76.00893
</td>
<td style="text-align:right;">
76.00902
</td>
<td style="text-align:left;">
positive
</td>
<td style="text-align:left;">
MGI_C57BL6J_5531169
</td>
<td style="text-align:left;">
Mir6409
</td>
<td style="text-align:left;">
miRBase:MI0021946,NCBI_Gene:102466646,ENSEMBL:ENSMUSG00000098817
</td>
<td style="text-align:left;">
MGI:5531169
</td>
<td style="text-align:left;">
miRNA gene
</td>
<td style="text-align:left;">
microRNA 6409
</td>
</tr>
<tr>
<td style="text-align:left;">
10
</td>
<td style="text-align:left;">
gene
</td>
<td style="text-align:right;">
76.03240
</td>
<td style="text-align:right;">
76.04323
</td>
<td style="text-align:left;">
positive
</td>
<td style="text-align:left;">
MGI_C57BL6J_1927865
</td>
<td style="text-align:left;">
Zfp280b
</td>
<td style="text-align:left;">
NCBI_Gene:64453,ENSEMBL:ENSMUSG00000049764
</td>
<td style="text-align:left;">
MGI:1927865
</td>
<td style="text-align:left;">
protein coding gene
</td>
<td style="text-align:left;">
zinc finger protein 280B
</td>
</tr>
<tr>
<td style="text-align:left;">
10
</td>
<td style="text-align:left;">
gene
</td>
<td style="text-align:right;">
76.05749
</td>
<td style="text-align:right;">
76.11102
</td>
<td style="text-align:left;">
negative
</td>
<td style="text-align:left;">
MGI_C57BL6J_1890478
</td>
<td style="text-align:left;">
Slc5a4b
</td>
<td style="text-align:left;">
NCBI_Gene:64454,ENSEMBL:ENSMUSG00000020226
</td>
<td style="text-align:left;">
MGI:1890478
</td>
<td style="text-align:left;">
protein coding gene
</td>
<td style="text-align:left;">
solute carrier family 5 (neutral amino acid transporters, system A), member 4b
</td>
</tr>
<tr>
<td style="text-align:left;">
10
</td>
<td style="text-align:left;">
gene
</td>
<td style="text-align:right;">
76.07862
</td>
<td style="text-align:right;">
76.08050
</td>
<td style="text-align:left;">
positive
</td>
<td style="text-align:left;">
MGI_C57BL6J_3801901
</td>
<td style="text-align:left;">
Gm16222
</td>
<td style="text-align:left;">
NCBI_Gene:102639053,ENSEMBL:ENSMUSG00000086798
</td>
<td style="text-align:left;">
MGI:3801901
</td>
<td style="text-align:left;">
lncRNA gene
</td>
<td style="text-align:left;">
predicted gene 16222
</td>
</tr>
<tr>
<td style="text-align:left;">
10
</td>
<td style="text-align:left;">
pseudogene
</td>
<td style="text-align:right;">
76.13343
</td>
<td style="text-align:right;">
76.14030
</td>
<td style="text-align:left;">
positive
</td>
<td style="text-align:left;">
MGI_C57BL6J_5011402
</td>
<td style="text-align:left;">
Gm19217
</td>
<td style="text-align:left;">
NCBI_Gene:100418445,ENSEMBL:ENSMUSG00000112128
</td>
<td style="text-align:left;">
MGI:5011402
</td>
<td style="text-align:left;">
pseudogene
</td>
<td style="text-align:left;">
predicted gene, 19217
</td>
</tr>
<tr>
<td style="text-align:left;">
10
</td>
<td style="text-align:left;">
gene
</td>
<td style="text-align:right;">
76.14745
</td>
<td style="text-align:right;">
76.18927
</td>
<td style="text-align:left;">
positive
</td>
<td style="text-align:left;">
MGI_C57BL6J_1927848
</td>
<td style="text-align:left;">
Slc5a4a
</td>
<td style="text-align:left;">
NCBI_Gene:64452,ENSEMBL:ENSMUSG00000020229
</td>
<td style="text-align:left;">
MGI:1927848
</td>
<td style="text-align:left;">
protein coding gene
</td>
<td style="text-align:left;">
solute carrier family 5, member 4a
</td>
</tr>
<tr>
<td style="text-align:left;">
10
</td>
<td style="text-align:left;">
gene
</td>
<td style="text-align:right;">
76.19780
</td>
<td style="text-align:right;">
76.19786
</td>
<td style="text-align:left;">
negative
</td>
<td style="text-align:left;">
MGI_C57BL6J_3028030
</td>
<td style="text-align:left;">
C530045E16Rik
</td>
<td style="text-align:left;">
NA
</td>
<td style="text-align:left;">
NA
</td>
<td style="text-align:left;">
unclassified gene
</td>
<td style="text-align:left;">
RIKEN cDNA C530045E16 gene
</td>
</tr>
<tr>
<td style="text-align:left;">
10
</td>
<td style="text-align:left;">
pseudogene
</td>
<td style="text-align:right;">
76.19819
</td>
<td style="text-align:right;">
76.19898
</td>
<td style="text-align:left;">
negative
</td>
<td style="text-align:left;">
MGI_C57BL6J_3780796
</td>
<td style="text-align:left;">
Gm2628
</td>
<td style="text-align:left;">
NCBI_Gene:100040150,ENSEMBL:ENSMUSG00000112625
</td>
<td style="text-align:left;">
MGI:3780796
</td>
<td style="text-align:left;">
pseudogene
</td>
<td style="text-align:left;">
predicted gene 2628
</td>
</tr>
<tr>
<td style="text-align:left;">
10
</td>
<td style="text-align:left;">
gene
</td>
<td style="text-align:right;">
76.20722
</td>
<td style="text-align:right;">
76.23786
</td>
<td style="text-align:left;">
negative
</td>
<td style="text-align:left;">
MGI_C57BL6J_1316652
</td>
<td style="text-align:left;">
Prmt2
</td>
<td style="text-align:left;">
NCBI_Gene:15468,ENSEMBL:ENSMUSG00000020230
</td>
<td style="text-align:left;">
MGI:1316652
</td>
<td style="text-align:left;">
protein coding gene
</td>
<td style="text-align:left;">
protein arginine N-methyltransferase 2
</td>
</tr>
<tr>
<td style="text-align:left;">
10
</td>
<td style="text-align:left;">
gene
</td>
<td style="text-align:right;">
76.20733
</td>
<td style="text-align:right;">
76.20741
</td>
<td style="text-align:left;">
negative
</td>
<td style="text-align:left;">
MGI_C57BL6J_3629884
</td>
<td style="text-align:left;">
Mir678
</td>
<td style="text-align:left;">
miRBase:MI0004635,NCBI_Gene:751554,ENSEMBL:ENSMUSG00000076068
</td>
<td style="text-align:left;">
MGI:3629884
</td>
<td style="text-align:left;">
miRNA gene
</td>
<td style="text-align:left;">
microRNA 678
</td>
</tr>
<tr>
<td style="text-align:left;">
10
</td>
<td style="text-align:left;">
gene
</td>
<td style="text-align:right;">
76.25384
</td>
<td style="text-align:right;">
76.26132
</td>
<td style="text-align:left;">
positive
</td>
<td style="text-align:left;">
MGI_C57BL6J_98217
</td>
<td style="text-align:left;">
S100b
</td>
<td style="text-align:left;">
NCBI_Gene:20203,ENSEMBL:ENSMUSG00000033208
</td>
<td style="text-align:left;">
MGI:98217
</td>
<td style="text-align:left;">
protein coding gene
</td>
<td style="text-align:left;">
S100 protein, beta polypeptide, neural
</td>
</tr>
<tr>
<td style="text-align:left;">
10
</td>
<td style="text-align:left;">
gene
</td>
<td style="text-align:right;">
76.25943
</td>
<td style="text-align:right;">
76.34529
</td>
<td style="text-align:left;">
negative
</td>
<td style="text-align:left;">
MGI_C57BL6J_2385920
</td>
<td style="text-align:left;">
Dip2a
</td>
<td style="text-align:left;">
NCBI_Gene:64451,ENSEMBL:ENSMUSG00000020231
</td>
<td style="text-align:left;">
MGI:2385920
</td>
<td style="text-align:left;">
protein coding gene
</td>
<td style="text-align:left;">
disco interacting protein 2 homolog A
</td>
</tr>
<tr>
<td style="text-align:left;">
10
</td>
<td style="text-align:left;">
gene
</td>
<td style="text-align:right;">
76.29317
</td>
<td style="text-align:right;">
76.29486
</td>
<td style="text-align:left;">
positive
</td>
<td style="text-align:left;">
MGI_C57BL6J_3801994
</td>
<td style="text-align:left;">
Gm16240
</td>
<td style="text-align:left;">
ENSEMBL:ENSMUSG00000089711
</td>
<td style="text-align:left;">
MGI:3801994
</td>
<td style="text-align:left;">
lncRNA gene
</td>
<td style="text-align:left;">
predicted gene 16240
</td>
</tr>
<tr>
<td style="text-align:left;">
10
</td>
<td style="text-align:left;">
gene
</td>
<td style="text-align:right;">
76.34538
</td>
<td style="text-align:right;">
76.34726
</td>
<td style="text-align:left;">
positive
</td>
<td style="text-align:left;">
MGI_C57BL6J_5623584
</td>
<td style="text-align:left;">
Gm40699
</td>
<td style="text-align:left;">
NCBI_Gene:105245215
</td>
<td style="text-align:left;">
MGI:5623584
</td>
<td style="text-align:left;">
lncRNA gene
</td>
<td style="text-align:left;">
predicted gene, 40699
</td>
</tr>
<tr>
<td style="text-align:left;">
10
</td>
<td style="text-align:left;">
gene
</td>
<td style="text-align:right;">
76.35125
</td>
<td style="text-align:right;">
76.44291
</td>
<td style="text-align:left;">
negative
</td>
<td style="text-align:left;">
MGI_C57BL6J_102722
</td>
<td style="text-align:left;">
Pcnt
</td>
<td style="text-align:left;">
NCBI_Gene:18541,ENSEMBL:ENSMUSG00000001151
</td>
<td style="text-align:left;">
MGI:102722
</td>
<td style="text-align:left;">
protein coding gene
</td>
<td style="text-align:left;">
pericentrin (kendrin)
</td>
</tr>
<tr>
<td style="text-align:left;">
10
</td>
<td style="text-align:left;">
gene
</td>
<td style="text-align:right;">
76.38902
</td>
<td style="text-align:right;">
76.40151
</td>
<td style="text-align:left;">
positive
</td>
<td style="text-align:left;">
MGI_C57BL6J_1914773
</td>
<td style="text-align:left;">
1700094J05Rik
</td>
<td style="text-align:left;">
NCBI_Gene:67523,ENSEMBL:ENSMUSG00000101695
</td>
<td style="text-align:left;">
MGI:1914773
</td>
<td style="text-align:left;">
lncRNA gene
</td>
<td style="text-align:left;">
RIKEN cDNA 1700094J05 gene
</td>
</tr>
<tr>
<td style="text-align:left;">
10
</td>
<td style="text-align:left;">
gene
</td>
<td style="text-align:right;">
76.44237
</td>
<td style="text-align:right;">
76.44387
</td>
<td style="text-align:left;">
positive
</td>
<td style="text-align:left;">
MGI_C57BL6J_3642478
</td>
<td style="text-align:left;">
Gm9744
</td>
<td style="text-align:left;">
NA
</td>
<td style="text-align:left;">
NA
</td>
<td style="text-align:left;">
protein coding gene
</td>
<td style="text-align:left;">
predicted gene 9744
</td>
</tr>
<tr>
<td style="text-align:left;">
10
</td>
<td style="text-align:left;">
gene
</td>
<td style="text-align:right;">
76.44902
</td>
<td style="text-align:right;">
76.46122
</td>
<td style="text-align:left;">
positive
</td>
<td style="text-align:left;">
MGI_C57BL6J_1924214
</td>
<td style="text-align:left;">
2610028H24Rik
</td>
<td style="text-align:left;">
NCBI_Gene:76964,ENSEMBL:ENSMUSG00000009114
</td>
<td style="text-align:left;">
MGI:1924214
</td>
<td style="text-align:left;">
protein coding gene
</td>
<td style="text-align:left;">
RIKEN cDNA 2610028H24 gene
</td>
</tr>
<tr>
<td style="text-align:left;">
10
</td>
<td style="text-align:left;">
gene
</td>
<td style="text-align:right;">
76.45813
</td>
<td style="text-align:right;">
76.46911
</td>
<td style="text-align:left;">
negative
</td>
<td style="text-align:left;">
MGI_C57BL6J_2656825
</td>
<td style="text-align:left;">
Ybey
</td>
<td style="text-align:left;">
NCBI_Gene:216119,ENSEMBL:ENSMUSG00000033126
</td>
<td style="text-align:left;">
MGI:2656825
</td>
<td style="text-align:left;">
protein coding gene
</td>
<td style="text-align:left;">
ybeY metallopeptidase
</td>
</tr>
<tr>
<td style="text-align:left;">
10
</td>
<td style="text-align:left;">
gene
</td>
<td style="text-align:right;">
76.46893
</td>
<td style="text-align:right;">
76.51586
</td>
<td style="text-align:left;">
positive
</td>
<td style="text-align:left;">
MGI_C57BL6J_1930089
</td>
<td style="text-align:left;">
Mcm3ap
</td>
<td style="text-align:left;">
NCBI_Gene:54387,ENSEMBL:ENSMUSG00000001150
</td>
<td style="text-align:left;">
MGI:1930089
</td>
<td style="text-align:left;">
protein coding gene
</td>
<td style="text-align:left;">
minichromosome maintenance complex component 3 associated protein
</td>
</tr>
<tr>
<td style="text-align:left;">
10
</td>
<td style="text-align:left;">
gene
</td>
<td style="text-align:right;">
76.53157
</td>
<td style="text-align:right;">
76.55714
</td>
<td style="text-align:left;">
positive
</td>
<td style="text-align:left;">
MGI_C57BL6J_1336155
</td>
<td style="text-align:left;">
Lss
</td>
<td style="text-align:left;">
NCBI_Gene:16987,ENSEMBL:ENSMUSG00000033105
</td>
<td style="text-align:left;">
MGI:1336155
</td>
<td style="text-align:left;">
protein coding gene
</td>
<td style="text-align:left;">
lanosterol synthase
</td>
</tr>
<tr>
<td style="text-align:left;">
10
</td>
<td style="text-align:left;">
gene
</td>
<td style="text-align:right;">
76.54587
</td>
<td style="text-align:right;">
76.56223
</td>
<td style="text-align:left;">
negative
</td>
<td style="text-align:left;">
MGI_C57BL6J_1922203
</td>
<td style="text-align:left;">
4930483K19Rik
</td>
<td style="text-align:left;">
NCBI_Gene:74953,ENSEMBL:ENSMUSG00000086241
</td>
<td style="text-align:left;">
MGI:1922203
</td>
<td style="text-align:left;">
lncRNA gene
</td>
<td style="text-align:left;">
RIKEN cDNA 4930483K19 gene
</td>
</tr>
<tr>
<td style="text-align:left;">
10
</td>
<td style="text-align:left;">
gene
</td>
<td style="text-align:right;">
76.56216
</td>
<td style="text-align:right;">
76.57020
</td>
<td style="text-align:left;">
positive
</td>
<td style="text-align:left;">
MGI_C57BL6J_1923823
</td>
<td style="text-align:left;">
Spatc1l
</td>
<td style="text-align:left;">
NCBI_Gene:76573,ENSEMBL:ENSMUSG00000009115
</td>
<td style="text-align:left;">
MGI:1923823
</td>
<td style="text-align:left;">
protein coding gene
</td>
<td style="text-align:left;">
spermatogenesis and centriole associated 1 like
</td>
</tr>
<tr>
<td style="text-align:left;">
10
</td>
<td style="text-align:left;">
gene
</td>
<td style="text-align:right;">
76.56457
</td>
<td style="text-align:right;">
76.57053
</td>
<td style="text-align:left;">
negative
</td>
<td style="text-align:left;">
MGI_C57BL6J_3705126
</td>
<td style="text-align:left;">
Gm15343
</td>
<td style="text-align:left;">
NCBI_Gene:102639115,ENSEMBL:ENSMUSG00000085468
</td>
<td style="text-align:left;">
MGI:3705126
</td>
<td style="text-align:left;">
lncRNA gene
</td>
<td style="text-align:left;">
predicted gene 15343
</td>
</tr>
<tr>
<td style="text-align:left;">
10
</td>
<td style="text-align:left;">
gene
</td>
<td style="text-align:right;">
76.57565
</td>
<td style="text-align:right;">
76.59034
</td>
<td style="text-align:left;">
positive
</td>
<td style="text-align:left;">
MGI_C57BL6J_1339962
</td>
<td style="text-align:left;">
Ftcd
</td>
<td style="text-align:left;">
NCBI_Gene:14317,ENSEMBL:ENSMUSG00000001155
</td>
<td style="text-align:left;">
MGI:1339962
</td>
<td style="text-align:left;">
protein coding gene
</td>
<td style="text-align:left;">
formiminotransferase cyclodeaminase
</td>
</tr>
<tr>
<td style="text-align:left;">
10
</td>
<td style="text-align:left;">
gene
</td>
<td style="text-align:right;">
76.59576
</td>
<td style="text-align:right;">
76.62363
</td>
<td style="text-align:left;">
negative
</td>
<td style="text-align:left;">
MGI_C57BL6J_88460
</td>
<td style="text-align:left;">
Col6a2
</td>
<td style="text-align:left;">
NCBI_Gene:12834,ENSEMBL:ENSMUSG00000020241
</td>
<td style="text-align:left;">
MGI:88460
</td>
<td style="text-align:left;">
protein coding gene
</td>
<td style="text-align:left;">
collagen, type VI, alpha 2
</td>
</tr>
<tr>
<td style="text-align:left;">
10
</td>
<td style="text-align:left;">
gene
</td>
<td style="text-align:right;">
76.67772
</td>
<td style="text-align:right;">
76.68680
</td>
<td style="text-align:left;">
positive
</td>
<td style="text-align:left;">
MGI_C57BL6J_5594767
</td>
<td style="text-align:left;">
Gm35608
</td>
<td style="text-align:left;">
NCBI_Gene:102639255,ENSEMBL:ENSMUSG00000112721
</td>
<td style="text-align:left;">
MGI:5594767
</td>
<td style="text-align:left;">
lncRNA gene
</td>
<td style="text-align:left;">
predicted gene, 35608
</td>
</tr>
<tr>
<td style="text-align:left;">
10
</td>
<td style="text-align:left;">
gene
</td>
<td style="text-align:right;">
76.70879
</td>
<td style="text-align:right;">
76.72617
</td>
<td style="text-align:left;">
negative
</td>
<td style="text-align:left;">
MGI_C57BL6J_88459
</td>
<td style="text-align:left;">
Col6a1
</td>
<td style="text-align:left;">
NCBI_Gene:12833,ENSEMBL:ENSMUSG00000001119
</td>
<td style="text-align:left;">
MGI:88459
</td>
<td style="text-align:left;">
protein coding gene
</td>
<td style="text-align:left;">
collagen, type VI, alpha 1
</td>
</tr>
<tr>
<td style="text-align:left;">
10
</td>
<td style="text-align:left;">
gene
</td>
<td style="text-align:right;">
76.72658
</td>
<td style="text-align:right;">
76.73027
</td>
<td style="text-align:left;">
negative
</td>
<td style="text-align:left;">
MGI_C57BL6J_6097704
</td>
<td style="text-align:left;">
Gm48276
</td>
<td style="text-align:left;">
ENSEMBL:ENSMUSG00000112291
</td>
<td style="text-align:left;">
MGI:6097704
</td>
<td style="text-align:left;">
lncRNA gene
</td>
<td style="text-align:left;">
predicted gene, 48276
</td>
</tr>
<tr>
<td style="text-align:left;">
10
</td>
<td style="text-align:left;">
gene
</td>
<td style="text-align:right;">
76.76185
</td>
<td style="text-align:right;">
76.96196
</td>
<td style="text-align:left;">
negative
</td>
<td style="text-align:left;">
MGI_C57BL6J_1890470
</td>
<td style="text-align:left;">
Pcbp3
</td>
<td style="text-align:left;">
NCBI_Gene:59093,ENSEMBL:ENSMUSG00000001120
</td>
<td style="text-align:left;">
MGI:1890470
</td>
<td style="text-align:left;">
protein coding gene
</td>
<td style="text-align:left;">
poly(rC) binding protein 3
</td>
</tr>
<tr>
<td style="text-align:left;">
10
</td>
<td style="text-align:left;">
gene
</td>
<td style="text-align:right;">
76.96214
</td>
<td style="text-align:right;">
76.96260
</td>
<td style="text-align:left;">
positive
</td>
<td style="text-align:left;">
MGI_C57BL6J_5594823
</td>
<td style="text-align:left;">
Gm35664
</td>
<td style="text-align:left;">
NCBI_Gene:102639326
</td>
<td style="text-align:left;">
MGI:5594823
</td>
<td style="text-align:left;">
lncRNA gene
</td>
<td style="text-align:left;">
predicted gene, 35664
</td>
</tr>
<tr>
<td style="text-align:left;">
10
</td>
<td style="text-align:left;">
gene
</td>
<td style="text-align:right;">
76.96394
</td>
<td style="text-align:right;">
77.00857
</td>
<td style="text-align:left;">
negative
</td>
<td style="text-align:left;">
MGI_C57BL6J_5594880
</td>
<td style="text-align:left;">
Gm35721
</td>
<td style="text-align:left;">
NCBI_Gene:102639395,ENSEMBL:ENSMUSG00000112143
</td>
<td style="text-align:left;">
MGI:5594880
</td>
<td style="text-align:left;">
lncRNA gene
</td>
<td style="text-align:left;">
predicted gene, 35721
</td>
</tr>
<tr>
<td style="text-align:left;">
10
</td>
<td style="text-align:left;">
gene
</td>
<td style="text-align:right;">
76.98322
</td>
<td style="text-align:right;">
76.98390
</td>
<td style="text-align:left;">
negative
</td>
<td style="text-align:left;">
MGI_C57BL6J_5611010
</td>
<td style="text-align:left;">
Gm37782
</td>
<td style="text-align:left;">
ENSEMBL:ENSMUSG00000102522
</td>
<td style="text-align:left;">
MGI:5611010
</td>
<td style="text-align:left;">
unclassified gene
</td>
<td style="text-align:left;">
predicted gene, 37782
</td>
</tr>
<tr>
<td style="text-align:left;">
10
</td>
<td style="text-align:left;">
gene
</td>
<td style="text-align:right;">
77.00477
</td>
<td style="text-align:right;">
77.02957
</td>
<td style="text-align:left;">
negative
</td>
<td style="text-align:left;">
MGI_C57BL6J_3781316
</td>
<td style="text-align:left;">
Gm3137
</td>
<td style="text-align:left;">
NCBI_Gene:100041099,ENSEMBL:ENSMUSG00000112773
</td>
<td style="text-align:left;">
MGI:3781316
</td>
<td style="text-align:left;">
lncRNA gene
</td>
<td style="text-align:left;">
predicted gene, 3137
</td>
</tr>
<tr>
<td style="text-align:left;">
10
</td>
<td style="text-align:left;">
gene
</td>
<td style="text-align:right;">
77.01199
</td>
<td style="text-align:right;">
77.02221
</td>
<td style="text-align:left;">
positive
</td>
<td style="text-align:left;">
MGI_C57BL6J_3642150
</td>
<td style="text-align:left;">
Gm10787
</td>
<td style="text-align:left;">
NCBI_Gene:100038479,ENSEMBL:ENSMUSG00000074919
</td>
<td style="text-align:left;">
MGI:3642150
</td>
<td style="text-align:left;">
lncRNA gene
</td>
<td style="text-align:left;">
predicted gene 10787
</td>
</tr>
<tr>
<td style="text-align:left;">
10
</td>
<td style="text-align:left;">
gene
</td>
<td style="text-align:right;">
77.03224
</td>
<td style="text-align:right;">
77.06100
</td>
<td style="text-align:left;">
positive
</td>
<td style="text-align:left;">
MGI_C57BL6J_103182
</td>
<td style="text-align:left;">
Slc19a1
</td>
<td style="text-align:left;">
NCBI_Gene:20509,ENSEMBL:ENSMUSG00000001436
</td>
<td style="text-align:left;">
MGI:103182
</td>
<td style="text-align:left;">
protein coding gene
</td>
<td style="text-align:left;">
solute carrier family 19 (folate transporter), member 1
</td>
</tr>
<tr>
<td style="text-align:left;">
10
</td>
<td style="text-align:left;">
gene
</td>
<td style="text-align:right;">
77.05218
</td>
<td style="text-align:right;">
77.16655
</td>
<td style="text-align:left;">
negative
</td>
<td style="text-align:left;">
MGI_C57BL6J_88451
</td>
<td style="text-align:left;">
Col18a1
</td>
<td style="text-align:left;">
NCBI_Gene:12822,ENSEMBL:ENSMUSG00000001435
</td>
<td style="text-align:left;">
MGI:88451
</td>
<td style="text-align:left;">
protein coding gene
</td>
<td style="text-align:left;">
collagen, type XVIII, alpha 1
</td>
</tr>
<tr>
<td style="text-align:left;">
10
</td>
<td style="text-align:left;">
pseudogene
</td>
<td style="text-align:right;">
77.13706
</td>
<td style="text-align:right;">
77.13824
</td>
<td style="text-align:left;">
positive
</td>
<td style="text-align:left;">
MGI_C57BL6J_3646054
</td>
<td style="text-align:left;">
Gm7775
</td>
<td style="text-align:left;">
NCBI_Gene:665762,ENSEMBL:ENSMUSG00000112612
</td>
<td style="text-align:left;">
MGI:3646054
</td>
<td style="text-align:left;">
pseudogene
</td>
<td style="text-align:left;">
predicted gene 7775
</td>
</tr>
<tr>
<td style="text-align:left;">
10
</td>
<td style="text-align:left;">
gene
</td>
<td style="text-align:right;">
77.20639
</td>
<td style="text-align:right;">
77.24304
</td>
<td style="text-align:left;">
positive
</td>
<td style="text-align:left;">
MGI_C57BL6J_5594996
</td>
<td style="text-align:left;">
Gm35837
</td>
<td style="text-align:left;">
NCBI_Gene:102639550
</td>
<td style="text-align:left;">
MGI:5594996
</td>
<td style="text-align:left;">
lncRNA gene
</td>
<td style="text-align:left;">
predicted gene, 35837
</td>
</tr>
<tr>
<td style="text-align:left;">
10
</td>
<td style="text-align:left;">
gene
</td>
<td style="text-align:right;">
77.21102
</td>
<td style="text-align:right;">
77.21193
</td>
<td style="text-align:left;">
positive
</td>
<td style="text-align:left;">
MGI_C57BL6J_5623589
</td>
<td style="text-align:left;">
Gm40704
</td>
<td style="text-align:left;">
NCBI_Gene:105245220
</td>
<td style="text-align:left;">
MGI:5623589
</td>
<td style="text-align:left;">
lncRNA gene
</td>
<td style="text-align:left;">
predicted gene, 40704
</td>
</tr>
<tr>
<td style="text-align:left;">
10
</td>
<td style="text-align:left;">
gene
</td>
<td style="text-align:right;">
77.25086
</td>
<td style="text-align:right;">
77.25209
</td>
<td style="text-align:left;">
negative
</td>
<td style="text-align:left;">
MGI_C57BL6J_5623588
</td>
<td style="text-align:left;">
Gm40703
</td>
<td style="text-align:left;">
NCBI_Gene:105245219
</td>
<td style="text-align:left;">
MGI:5623588
</td>
<td style="text-align:left;">
lncRNA gene
</td>
<td style="text-align:left;">
predicted gene, 40703
</td>
</tr>
<tr>
<td style="text-align:left;">
10
</td>
<td style="text-align:left;">
gene
</td>
<td style="text-align:right;">
77.25777
</td>
<td style="text-align:right;">
77.25922
</td>
<td style="text-align:left;">
negative
</td>
<td style="text-align:left;">
MGI_C57BL6J_3779151
</td>
<td style="text-align:left;">
Gm10941
</td>
<td style="text-align:left;">
NCBI_Gene:100169878,ENSEMBL:ENSMUSG00000078444
</td>
<td style="text-align:left;">
MGI:3779151
</td>
<td style="text-align:left;">
lncRNA gene
</td>
<td style="text-align:left;">
predicted gene 10941
</td>
</tr>
<tr>
<td style="text-align:left;">
10
</td>
<td style="text-align:left;">
gene
</td>
<td style="text-align:right;">
77.25922
</td>
<td style="text-align:right;">
77.26959
</td>
<td style="text-align:left;">
positive
</td>
<td style="text-align:left;">
MGI_C57BL6J_1916863
</td>
<td style="text-align:left;">
Pofut2
</td>
<td style="text-align:left;">
NCBI_Gene:80294,ENSEMBL:ENSMUSG00000020260
</td>
<td style="text-align:left;">
MGI:1916863
</td>
<td style="text-align:left;">
protein coding gene
</td>
<td style="text-align:left;">
protein O-fucosyltransferase 2
</td>
</tr>
<tr>
<td style="text-align:left;">
10
</td>
<td style="text-align:left;">
gene
</td>
<td style="text-align:right;">
77.29073
</td>
<td style="text-align:right;">
77.41827
</td>
<td style="text-align:left;">
negative
</td>
<td style="text-align:left;">
MGI_C57BL6J_891999
</td>
<td style="text-align:left;">
Adarb1
</td>
<td style="text-align:left;">
NCBI_Gene:110532,ENSEMBL:ENSMUSG00000020262
</td>
<td style="text-align:left;">
MGI:891999
</td>
<td style="text-align:left;">
protein coding gene
</td>
<td style="text-align:left;">
adenosine deaminase, RNA-specific, B1
</td>
</tr>
<tr>
<td style="text-align:left;">
10
</td>
<td style="text-align:left;">
gene
</td>
<td style="text-align:right;">
77.41838
</td>
<td style="text-align:right;">
77.42094
</td>
<td style="text-align:left;">
positive
</td>
<td style="text-align:left;">
MGI_C57BL6J_5009933
</td>
<td style="text-align:left;">
Gm17769
</td>
<td style="text-align:left;">
NCBI_Gene:100045653
</td>
<td style="text-align:left;">
MGI:5009933
</td>
<td style="text-align:left;">
lncRNA gene
</td>
<td style="text-align:left;">
predicted gene, 17769
</td>
</tr>
<tr>
<td style="text-align:left;">
10
</td>
<td style="text-align:left;">
gene
</td>
<td style="text-align:right;">
77.45792
</td>
<td style="text-align:right;">
77.47163
</td>
<td style="text-align:left;">
positive
</td>
<td style="text-align:left;">
MGI_C57BL6J_5595079
</td>
<td style="text-align:left;">
Gm35920
</td>
<td style="text-align:left;">
NCBI_Gene:102639662,ENSEMBL:ENSMUSG00000112818
</td>
<td style="text-align:left;">
MGI:5595079
</td>
<td style="text-align:left;">
lncRNA gene
</td>
<td style="text-align:left;">
predicted gene, 35920
</td>
</tr>
<tr>
<td style="text-align:left;">
10
</td>
<td style="text-align:left;">
gene
</td>
<td style="text-align:right;">
77.47060
</td>
<td style="text-align:right;">
77.47246
</td>
<td style="text-align:left;">
positive
</td>
<td style="text-align:left;">
MGI_C57BL6J_5595142
</td>
<td style="text-align:left;">
Gm35983
</td>
<td style="text-align:left;">
NCBI_Gene:102639743
</td>
<td style="text-align:left;">
MGI:5595142
</td>
<td style="text-align:left;">
lncRNA gene
</td>
<td style="text-align:left;">
predicted gene, 35983
</td>
</tr>
<tr>
<td style="text-align:left;">
10
</td>
<td style="text-align:left;">
gene
</td>
<td style="text-align:right;">
77.48665
</td>
<td style="text-align:right;">
77.51692
</td>
<td style="text-align:left;">
negative
</td>
<td style="text-align:left;">
MGI_C57BL6J_1916334
</td>
<td style="text-align:left;">
Fam207a
</td>
<td style="text-align:left;">
NCBI_Gene:108707,ENSEMBL:ENSMUSG00000032977
</td>
<td style="text-align:left;">
MGI:1916334
</td>
<td style="text-align:left;">
protein coding gene
</td>
<td style="text-align:left;">
family with sequence similarity 207, member A
</td>
</tr>
<tr>
<td style="text-align:left;">
10
</td>
<td style="text-align:left;">
gene
</td>
<td style="text-align:right;">
77.48666
</td>
<td style="text-align:right;">
77.51643
</td>
<td style="text-align:left;">
positive
</td>
<td style="text-align:left;">
MGI_C57BL6J_1917222
</td>
<td style="text-align:left;">
2810425M01Rik
</td>
<td style="text-align:left;">
ENSEMBL:ENSMUSG00000097811
</td>
<td style="text-align:left;">
MGI:1917222
</td>
<td style="text-align:left;">
lncRNA gene
</td>
<td style="text-align:left;">
RIKEN cDNA 2810425M01 gene
</td>
</tr>
<tr>
<td style="text-align:left;">
10
</td>
<td style="text-align:left;">
gene
</td>
<td style="text-align:right;">
77.52139
</td>
<td style="text-align:right;">
77.52254
</td>
<td style="text-align:left;">
positive
</td>
<td style="text-align:left;">
MGI_C57BL6J_5595199
</td>
<td style="text-align:left;">
Gm36040
</td>
<td style="text-align:left;">
NCBI_Gene:102639818
</td>
<td style="text-align:left;">
MGI:5595199
</td>
<td style="text-align:left;">
lncRNA gene
</td>
<td style="text-align:left;">
predicted gene, 36040
</td>
</tr>
<tr>
<td style="text-align:left;">
10
</td>
<td style="text-align:left;">
gene
</td>
<td style="text-align:right;">
77.53025
</td>
<td style="text-align:right;">
77.56571
</td>
<td style="text-align:left;">
positive
</td>
<td style="text-align:left;">
MGI_C57BL6J_96611
</td>
<td style="text-align:left;">
Itgb2
</td>
<td style="text-align:left;">
NCBI_Gene:16414,ENSEMBL:ENSMUSG00000000290
</td>
<td style="text-align:left;">
MGI:96611
</td>
<td style="text-align:left;">
protein coding gene
</td>
<td style="text-align:left;">
integrin beta 2
</td>
</tr>
<tr>
<td style="text-align:left;">
10
</td>
<td style="text-align:left;">
gene
</td>
<td style="text-align:right;">
77.58172
</td>
<td style="text-align:right;">
77.59873
</td>
<td style="text-align:left;">
positive
</td>
<td style="text-align:left;">
MGI_C57BL6J_2652132
</td>
<td style="text-align:left;">
Pttg1ip
</td>
<td style="text-align:left;">
NCBI_Gene:108705,ENSEMBL:ENSMUSG00000009291
</td>
<td style="text-align:left;">
MGI:2652132
</td>
<td style="text-align:left;">
protein coding gene
</td>
<td style="text-align:left;">
pituitary tumor-transforming 1 interacting protein
</td>
</tr>
<tr>
<td style="text-align:left;">
10
</td>
<td style="text-align:left;">
gene
</td>
<td style="text-align:right;">
77.60610
</td>
<td style="text-align:right;">
77.61833
</td>
<td style="text-align:left;">
positive
</td>
<td style="text-align:left;">
MGI_C57BL6J_1336201
</td>
<td style="text-align:left;">
Sumo3
</td>
<td style="text-align:left;">
NCBI_Gene:20610,ENSEMBL:ENSMUSG00000020265
</td>
<td style="text-align:left;">
MGI:1336201
</td>
<td style="text-align:left;">
protein coding gene
</td>
<td style="text-align:left;">
small ubiquitin-like modifier 3
</td>
</tr>
<tr>
<td style="text-align:left;">
10
</td>
<td style="text-align:left;">
gene
</td>
<td style="text-align:right;">
77.60620
</td>
<td style="text-align:right;">
77.64154
</td>
<td style="text-align:left;">
positive
</td>
<td style="text-align:left;">
MGI_C57BL6J_6121509
</td>
<td style="text-align:left;">
Gm49325
</td>
<td style="text-align:left;">
ENSEMBL:ENSMUSG00000112241
</td>
<td style="text-align:left;">
MGI:6121509
</td>
<td style="text-align:left;">
protein coding gene
</td>
<td style="text-align:left;">
predicted gene, 49325
</td>
</tr>
<tr>
<td style="text-align:left;">
10
</td>
<td style="text-align:left;">
gene
</td>
<td style="text-align:right;">
77.62228
</td>
<td style="text-align:right;">
77.64599
</td>
<td style="text-align:left;">
positive
</td>
<td style="text-align:left;">
MGI_C57BL6J_1343188
</td>
<td style="text-align:left;">
Ube2g2
</td>
<td style="text-align:left;">
NCBI_Gene:22213,ENSEMBL:ENSMUSG00000009293
</td>
<td style="text-align:left;">
MGI:1343188
</td>
<td style="text-align:left;">
protein coding gene
</td>
<td style="text-align:left;">
ubiquitin-conjugating enzyme E2G 2
</td>
</tr>
<tr>
<td style="text-align:left;">
10
</td>
<td style="text-align:left;">
pseudogene
</td>
<td style="text-align:right;">
77.63851
</td>
<td style="text-align:right;">
77.64106
</td>
<td style="text-align:left;">
negative
</td>
<td style="text-align:left;">
MGI_C57BL6J_5825873
</td>
<td style="text-align:left;">
Gm46236
</td>
<td style="text-align:left;">
NCBI_Gene:108167823
</td>
<td style="text-align:left;">
MGI:5825873
</td>
<td style="text-align:left;">
pseudogene
</td>
<td style="text-align:left;">
predicted gene, 46236
</td>
</tr>
<tr>
<td style="text-align:left;">
10
</td>
<td style="text-align:left;">
gene
</td>
<td style="text-align:right;">
77.64122
</td>
<td style="text-align:right;">
77.64131
</td>
<td style="text-align:left;">
positive
</td>
<td style="text-align:left;">
MGI_C57BL6J_3836965
</td>
<td style="text-align:left;">
Mir1930
</td>
<td style="text-align:left;">
miRBase:MI0009919,NCBI_Gene:100316811,ENSEMBL:ENSMUSG00000089424
</td>
<td style="text-align:left;">
MGI:3836965
</td>
<td style="text-align:left;">
miRNA gene
</td>
<td style="text-align:left;">
microRNA 1930
</td>
</tr>
<tr>
<td style="text-align:left;">
10
</td>
<td style="text-align:left;">
pseudogene
</td>
<td style="text-align:right;">
77.65274
</td>
<td style="text-align:right;">
77.65336
</td>
<td style="text-align:left;">
negative
</td>
<td style="text-align:left;">
MGI_C57BL6J_5010304
</td>
<td style="text-align:left;">
Gm18119
</td>
<td style="text-align:left;">
NCBI_Gene:100416456,ENSEMBL:ENSMUSG00000112007
</td>
<td style="text-align:left;">
MGI:5010304
</td>
<td style="text-align:left;">
pseudogene
</td>
<td style="text-align:left;">
predicted gene, 18119
</td>
</tr>
<tr>
<td style="text-align:left;">
10
</td>
<td style="text-align:left;">
pseudogene
</td>
<td style="text-align:right;">
77.68060
</td>
<td style="text-align:right;">
77.68109
</td>
<td style="text-align:left;">
negative
</td>
<td style="text-align:left;">
MGI_C57BL6J_6097530
</td>
<td style="text-align:left;">
Gm48160
</td>
<td style="text-align:left;">
ENSEMBL:ENSMUSG00000112912
</td>
<td style="text-align:left;">
MGI:6097530
</td>
<td style="text-align:left;">
pseudogene
</td>
<td style="text-align:left;">
predicted gene, 48160
</td>
</tr>
<tr>
<td style="text-align:left;">
10
</td>
<td style="text-align:left;">
gene
</td>
<td style="text-align:right;">
77.68603
</td>
<td style="text-align:right;">
77.88702
</td>
<td style="text-align:left;">
positive
</td>
<td style="text-align:left;">
MGI_C57BL6J_2671932
</td>
<td style="text-align:left;">
Tspear
</td>
<td style="text-align:left;">
NCBI_Gene:252974,ENSEMBL:ENSMUSG00000069581
</td>
<td style="text-align:left;">
MGI:2671932
</td>
<td style="text-align:left;">
protein coding gene
</td>
<td style="text-align:left;">
thrombospondin type laminin G domain and EAR repeats
</td>
</tr>
<tr>
<td style="text-align:left;">
10
</td>
<td style="text-align:left;">
gene
</td>
<td style="text-align:right;">
77.68966
</td>
<td style="text-align:right;">
77.69129
</td>
<td style="text-align:left;">
negative
</td>
<td style="text-align:left;">
MGI_C57BL6J_5011587
</td>
<td style="text-align:left;">
Gm19402
</td>
<td style="text-align:left;">
NCBI_Gene:100502831,ENSEMBL:ENSMUSG00000095970
</td>
<td style="text-align:left;">
MGI:5011587
</td>
<td style="text-align:left;">
protein coding gene
</td>
<td style="text-align:left;">
predicted gene, 19402
</td>
</tr>
<tr>
<td style="text-align:left;">
10
</td>
<td style="text-align:left;">
gene
</td>
<td style="text-align:right;">
77.69627
</td>
<td style="text-align:right;">
77.69723
</td>
<td style="text-align:left;">
negative
</td>
<td style="text-align:left;">
MGI_C57BL6J_3779918
</td>
<td style="text-align:left;">
Gm9508
</td>
<td style="text-align:left;">
NCBI_Gene:670895,ENSEMBL:ENSMUSG00000112170
</td>
<td style="text-align:left;">
MGI:3779918
</td>
<td style="text-align:left;">
protein coding gene
</td>
<td style="text-align:left;">
predicted gene 9508
</td>
</tr>
<tr>
<td style="text-align:left;">
10
</td>
<td style="text-align:left;">
gene
</td>
<td style="text-align:right;">
77.70659
</td>
<td style="text-align:right;">
77.70699
</td>
<td style="text-align:left;">
positive
</td>
<td style="text-align:left;">
MGI_C57BL6J_3642183
</td>
<td style="text-align:left;">
Gm10272
</td>
<td style="text-align:left;">
NCBI_Gene:16697,ENSEMBL:ENSMUSG00000069584
</td>
<td style="text-align:left;">
MGI:3642183
</td>
<td style="text-align:left;">
protein coding gene
</td>
<td style="text-align:left;">
predicted gene 10272
</td>
</tr>
<tr>
<td style="text-align:left;">
10
</td>
<td style="text-align:left;">
gene
</td>
<td style="text-align:right;">
77.71145
</td>
<td style="text-align:right;">
77.71201
</td>
<td style="text-align:left;">
positive
</td>
<td style="text-align:left;">
MGI_C57BL6J_3641784
</td>
<td style="text-align:left;">
Gm10024
</td>
<td style="text-align:left;">
NCBI_Gene:100009614,ENSEMBL:ENSMUSG00000094012
</td>
<td style="text-align:left;">
MGI:3641784
</td>
<td style="text-align:left;">
protein coding gene
</td>
<td style="text-align:left;">
predicted gene 10024
</td>
</tr>
<tr>
<td style="text-align:left;">
10
</td>
<td style="text-align:left;">
gene
</td>
<td style="text-align:right;">
77.71581
</td>
<td style="text-align:right;">
77.71617
</td>
<td style="text-align:left;">
positive
</td>
<td style="text-align:left;">
MGI_C57BL6J_3641725
</td>
<td style="text-align:left;">
Gm10142
</td>
<td style="text-align:left;">
NCBI_Gene:100502921,ENSEMBL:ENSMUSG00000094146
</td>
<td style="text-align:left;">
MGI:3641725
</td>
<td style="text-align:left;">
protein coding gene
</td>
<td style="text-align:left;">
predicted gene 10142
</td>
</tr>
<tr>
<td style="text-align:left;">
10
</td>
<td style="text-align:left;">
pseudogene
</td>
<td style="text-align:right;">
77.71790
</td>
<td style="text-align:right;">
77.71842
</td>
<td style="text-align:left;">
negative
</td>
<td style="text-align:left;">
MGI_C57BL6J_6097555
</td>
<td style="text-align:left;">
Gm48179
</td>
<td style="text-align:left;">
ENSEMBL:ENSMUSG00000112037
</td>
<td style="text-align:left;">
MGI:6097555
</td>
<td style="text-align:left;">
pseudogene
</td>
<td style="text-align:left;">
predicted gene, 48179
</td>
</tr>
<tr>
<td style="text-align:left;">
10
</td>
<td style="text-align:left;">
gene
</td>
<td style="text-align:right;">
77.72059
</td>
<td style="text-align:right;">
77.72126
</td>
<td style="text-align:left;">
positive
</td>
<td style="text-align:left;">
MGI_C57BL6J_1328315
</td>
<td style="text-align:left;">
Krtap12-1
</td>
<td style="text-align:left;">
NCBI_Gene:16694,ENSEMBL:ENSMUSG00000069583
</td>
<td style="text-align:left;">
MGI:1328315
</td>
<td style="text-align:left;">
protein coding gene
</td>
<td style="text-align:left;">
keratin associated protein 12-1
</td>
</tr>
<tr>
<td style="text-align:left;">
10
</td>
<td style="text-align:left;">
gene
</td>
<td style="text-align:right;">
77.72649
</td>
<td style="text-align:right;">
77.72685
</td>
<td style="text-align:left;">
positive
</td>
<td style="text-align:left;">
MGI_C57BL6J_3642388
</td>
<td style="text-align:left;">
Gm10100
</td>
<td style="text-align:left;">
NCBI_Gene:100502953,ENSEMBL:ENSMUSG00000096421
</td>
<td style="text-align:left;">
MGI:3642388
</td>
<td style="text-align:left;">
protein coding gene
</td>
<td style="text-align:left;">
predicted gene 10100
</td>
</tr>
<tr>
<td style="text-align:left;">
10
</td>
<td style="text-align:left;">
pseudogene
</td>
<td style="text-align:right;">
77.73484
</td>
<td style="text-align:right;">
77.73516
</td>
<td style="text-align:left;">
negative
</td>
<td style="text-align:left;">
MGI_C57BL6J_6097556
</td>
<td style="text-align:left;">
Gm48180
</td>
<td style="text-align:left;">
ENSEMBL:ENSMUSG00000112236
</td>
<td style="text-align:left;">
MGI:6097556
</td>
<td style="text-align:left;">
pseudogene
</td>
<td style="text-align:left;">
predicted gene, 48180
</td>
</tr>
<tr>
<td style="text-align:left;">
10
</td>
<td style="text-align:left;">
gene
</td>
<td style="text-align:right;">
77.74180
</td>
<td style="text-align:right;">
77.74264
</td>
<td style="text-align:left;">
negative
</td>
<td style="text-align:left;">
MGI_C57BL6J_5010781
</td>
<td style="text-align:left;">
Gm18596
</td>
<td style="text-align:left;">
NCBI_Gene:100417409,ENSEMBL:ENSMUSG00000112864
</td>
<td style="text-align:left;">
MGI:5010781
</td>
<td style="text-align:left;">
protein coding gene
</td>
<td style="text-align:left;">
predicted gene, 18596
</td>
</tr>
<tr>
<td style="text-align:left;">
10
</td>
<td style="text-align:left;">
pseudogene
</td>
<td style="text-align:right;">
77.74949
</td>
<td style="text-align:right;">
77.74958
</td>
<td style="text-align:left;">
negative
</td>
<td style="text-align:left;">
MGI_C57BL6J_6097554
</td>
<td style="text-align:left;">
Gm48178
</td>
<td style="text-align:left;">
ENSEMBL:ENSMUSG00000111976
</td>
<td style="text-align:left;">
MGI:6097554
</td>
<td style="text-align:left;">
pseudogene
</td>
<td style="text-align:left;">
predicted gene, 48178
</td>
</tr>
<tr>
<td style="text-align:left;">
10
</td>
<td style="text-align:left;">
gene
</td>
<td style="text-align:right;">
77.75051
</td>
<td style="text-align:right;">
77.75142
</td>
<td style="text-align:left;">
negative
</td>
<td style="text-align:left;">
MGI_C57BL6J_3842085
</td>
<td style="text-align:left;">
Gm9736
</td>
<td style="text-align:left;">
NCBI_Gene:77763,ENSEMBL:ENSMUSG00000112380
</td>
<td style="text-align:left;">
MGI:3842085
</td>
<td style="text-align:left;">
protein coding gene
</td>
<td style="text-align:left;">
predicted gene 9736
</td>
</tr>
<tr>
<td style="text-align:left;">
10
</td>
<td style="text-align:left;">
gene
</td>
<td style="text-align:right;">
77.75897
</td>
<td style="text-align:right;">
77.75984
</td>
<td style="text-align:left;">
negative
</td>
<td style="text-align:left;">
MGI_C57BL6J_3781411
</td>
<td style="text-align:left;">
Gm3233
</td>
<td style="text-align:left;">
NCBI_Gene:100041250,ENSEMBL:ENSMUSG00000094120
</td>
<td style="text-align:left;">
MGI:3781411
</td>
<td style="text-align:left;">
protein coding gene
</td>
<td style="text-align:left;">
predicted gene 3233
</td>
</tr>
<tr>
<td style="text-align:left;">
10
</td>
<td style="text-align:left;">
gene
</td>
<td style="text-align:right;">
77.77063
</td>
<td style="text-align:right;">
77.77133
</td>
<td style="text-align:left;">
negative
</td>
<td style="text-align:left;">
MGI_C57BL6J_3781416
</td>
<td style="text-align:left;">
Gm3238
</td>
<td style="text-align:left;">
NCBI_Gene:100041261,ENSEMBL:ENSMUSG00000095817
</td>
<td style="text-align:left;">
MGI:3781416
</td>
<td style="text-align:left;">
protein coding gene
</td>
<td style="text-align:left;">
predicted gene 3238
</td>
</tr>
<tr>
<td style="text-align:left;">
10
</td>
<td style="text-align:left;">
gene
</td>
<td style="text-align:right;">
77.77622
</td>
<td style="text-align:right;">
77.77694
</td>
<td style="text-align:left;">
negative
</td>
<td style="text-align:left;">
MGI_C57BL6J_3779678
</td>
<td style="text-align:left;">
Gm7138
</td>
<td style="text-align:left;">
NCBI_Gene:102640500,ENSEMBL:ENSMUSG00000095593
</td>
<td style="text-align:left;">
MGI:3779678
</td>
<td style="text-align:left;">
protein coding gene
</td>
<td style="text-align:left;">
predicted gene 7138
</td>
</tr>
<tr>
<td style="text-align:left;">
10
</td>
<td style="text-align:left;">
gene
</td>
<td style="text-align:right;">
77.78173
</td>
<td style="text-align:right;">
77.78254
</td>
<td style="text-align:left;">
negative
</td>
<td style="text-align:left;">
MGI_C57BL6J_3781428
</td>
<td style="text-align:left;">
Gm3250
</td>
<td style="text-align:left;">
NCBI_Gene:100041281,ENSEMBL:ENSMUSG00000096481
</td>
<td style="text-align:left;">
MGI:3781428
</td>
<td style="text-align:left;">
protein coding gene
</td>
<td style="text-align:left;">
predicted gene 3250
</td>
</tr>
<tr>
<td style="text-align:left;">
10
</td>
<td style="text-align:left;">
gene
</td>
<td style="text-align:right;">
77.78742
</td>
<td style="text-align:right;">
77.78820
</td>
<td style="text-align:left;">
negative
</td>
<td style="text-align:left;">
MGI_C57BL6J_3779677
</td>
<td style="text-align:left;">
Gm7137
</td>
<td style="text-align:left;">
NCBI_Gene:634504,ENSEMBL:ENSMUSG00000095721
</td>
<td style="text-align:left;">
MGI:3779677
</td>
<td style="text-align:left;">
protein coding gene
</td>
<td style="text-align:left;">
predicted gene 7137
</td>
</tr>
<tr>
<td style="text-align:left;">
10
</td>
<td style="text-align:left;">
gene
</td>
<td style="text-align:right;">
77.79440
</td>
<td style="text-align:right;">
77.79508
</td>
<td style="text-align:left;">
negative
</td>
<td style="text-align:left;">
MGI_C57BL6J_3780046
</td>
<td style="text-align:left;">
Gm9639
</td>
<td style="text-align:left;">
NCBI_Gene:675294,ENSEMBL:ENSMUSG00000096131
</td>
<td style="text-align:left;">
MGI:3780046
</td>
<td style="text-align:left;">
protein coding gene
</td>
<td style="text-align:left;">
predicted gene 9639
</td>
</tr>
<tr>
<td style="text-align:left;">
10
</td>
<td style="text-align:left;">
gene
</td>
<td style="text-align:right;">
77.79839
</td>
<td style="text-align:right;">
77.79913
</td>
<td style="text-align:left;">
negative
</td>
<td style="text-align:left;">
MGI_C57BL6J_5011853
</td>
<td style="text-align:left;">
Gm19668
</td>
<td style="text-align:left;">
NCBI_Gene:100503388,ENSEMBL:ENSMUSG00000096380
</td>
<td style="text-align:left;">
MGI:5011853
</td>
<td style="text-align:left;">
protein coding gene
</td>
<td style="text-align:left;">
predicted gene, 19668
</td>
</tr>
<tr>
<td style="text-align:left;">
10
</td>
<td style="text-align:left;">
gene
</td>
<td style="text-align:right;">
77.81097
</td>
<td style="text-align:right;">
77.81210
</td>
<td style="text-align:left;">
negative
</td>
<td style="text-align:left;">
MGI_C57BL6J_3779917
</td>
<td style="text-align:left;">
Gm9507
</td>
<td style="text-align:left;">
NCBI_Gene:670880,ENSEMBL:ENSMUSG00000094913
</td>
<td style="text-align:left;">
MGI:3779917
</td>
<td style="text-align:left;">
protein coding gene
</td>
<td style="text-align:left;">
predicted gene 9507
</td>
</tr>
<tr>
<td style="text-align:left;">
10
</td>
<td style="text-align:left;">
gene
</td>
<td style="text-align:right;">
77.81468
</td>
<td style="text-align:right;">
77.81540
</td>
<td style="text-align:left;">
positive
</td>
<td style="text-align:left;">
MGI_C57BL6J_3780866
</td>
<td style="text-align:left;">
Gm2696
</td>
<td style="text-align:left;">
NCBI_Gene:100040299
</td>
<td style="text-align:left;">
MGI:3780866
</td>
<td style="text-align:left;">
protein coding gene
</td>
<td style="text-align:left;">
predicted gene 2696
</td>
</tr>
<tr>
<td style="text-align:left;">
10
</td>
<td style="text-align:left;">
gene
</td>
<td style="text-align:right;">
77.81468
</td>
<td style="text-align:right;">
77.83665
</td>
<td style="text-align:left;">
positive
</td>
<td style="text-align:left;">
MGI_C57BL6J_6270621
</td>
<td style="text-align:left;">
Gm49918
</td>
<td style="text-align:left;">
ENSEMBL:ENSMUSG00000112600
</td>
<td style="text-align:left;">
MGI:6270621
</td>
<td style="text-align:left;">
protein coding gene
</td>
<td style="text-align:left;">
predicted gene, 49918
</td>
</tr>
<tr>
<td style="text-align:left;">
10
</td>
<td style="text-align:left;">
gene
</td>
<td style="text-align:right;">
77.82612
</td>
<td style="text-align:right;">
77.82709
</td>
<td style="text-align:left;">
negative
</td>
<td style="text-align:left;">
MGI_C57BL6J_1925013
</td>
<td style="text-align:left;">
Krtap10-4
</td>
<td style="text-align:left;">
NCBI_Gene:100191037,ENSEMBL:ENSMUSG00000069582
</td>
<td style="text-align:left;">
MGI:1925013
</td>
<td style="text-align:left;">
protein coding gene
</td>
<td style="text-align:left;">
keratin associated protein 10-4
</td>
</tr>
<tr>
<td style="text-align:left;">
10
</td>
<td style="text-align:left;">
gene
</td>
<td style="text-align:right;">
77.83595
</td>
<td style="text-align:right;">
77.83725
</td>
<td style="text-align:left;">
positive
</td>
<td style="text-align:left;">
MGI_C57BL6J_3645300
</td>
<td style="text-align:left;">
Krtap10-10
</td>
<td style="text-align:left;">
NCBI_Gene:544710,ENSEMBL:ENSMUSG00000112223
</td>
<td style="text-align:left;">
MGI:3645300
</td>
<td style="text-align:left;">
protein coding gene
</td>
<td style="text-align:left;">
keratin associated protein 10-10
</td>
</tr>
<tr>
<td style="text-align:left;">
10
</td>
<td style="text-align:left;">
gene
</td>
<td style="text-align:right;">
77.84680
</td>
<td style="text-align:right;">
77.84738
</td>
<td style="text-align:left;">
positive
</td>
<td style="text-align:left;">
MGI_C57BL6J_5595335
</td>
<td style="text-align:left;">
Gm36176
</td>
<td style="text-align:left;">
NCBI_Gene:102639995,ENSEMBL:ENSMUSG00000112653
</td>
<td style="text-align:left;">
MGI:5595335
</td>
<td style="text-align:left;">
protein coding gene
</td>
<td style="text-align:left;">
predicted gene, 36176
</td>
</tr>
<tr>
<td style="text-align:left;">
10
</td>
<td style="text-align:left;">
gene
</td>
<td style="text-align:right;">
77.85286
</td>
<td style="text-align:right;">
77.85379
</td>
<td style="text-align:left;">
positive
</td>
<td style="text-align:left;">
MGI_C57BL6J_3704118
</td>
<td style="text-align:left;">
Gm10318
</td>
<td style="text-align:left;">
NCBI_Gene:622629,ENSEMBL:ENSMUSG00000071195
</td>
<td style="text-align:left;">
MGI:3704118
</td>
<td style="text-align:left;">
protein coding gene
</td>
<td style="text-align:left;">
predicted gene 10318
</td>
</tr>
<tr>
<td style="text-align:left;">
10
</td>
<td style="text-align:left;">
gene
</td>
<td style="text-align:right;">
77.86198
</td>
<td style="text-align:right;">
77.86288
</td>
<td style="text-align:left;">
positive
</td>
<td style="text-align:left;">
MGI_C57BL6J_3781463
</td>
<td style="text-align:left;">
Gm3285
</td>
<td style="text-align:left;">
NCBI_Gene:100041351,ENSEMBL:ENSMUSG00000111915
</td>
<td style="text-align:left;">
MGI:3781463
</td>
<td style="text-align:left;">
protein coding gene
</td>
<td style="text-align:left;">
predicted gene 3285
</td>
</tr>
<tr>
<td style="text-align:left;">
10
</td>
<td style="text-align:left;">
gene
</td>
<td style="text-align:right;">
77.89342
</td>
<td style="text-align:right;">
77.89611
</td>
<td style="text-align:left;">
positive
</td>
<td style="text-align:left;">
MGI_C57BL6J_1914938
</td>
<td style="text-align:left;">
1700009J07Rik
</td>
<td style="text-align:left;">
NCBI_Gene:75188,ENSEMBL:ENSMUSG00000096573
</td>
<td style="text-align:left;">
MGI:1914938
</td>
<td style="text-align:left;">
lncRNA gene
</td>
<td style="text-align:left;">
RIKEN cDNA 1700009J07 gene
</td>
</tr>
<tr>
<td style="text-align:left;">
10
</td>
<td style="text-align:left;">
gene
</td>
<td style="text-align:right;">
77.89758
</td>
<td style="text-align:right;">
77.90254
</td>
<td style="text-align:left;">
negative
</td>
<td style="text-align:left;">
MGI_C57BL6J_2447899
</td>
<td style="text-align:left;">
Lrrc3
</td>
<td style="text-align:left;">
NCBI_Gene:237387,ENSEMBL:ENSMUSG00000051652
</td>
<td style="text-align:left;">
MGI:2447899
</td>
<td style="text-align:left;">
protein coding gene
</td>
<td style="text-align:left;">
leucine rich repeat containing 3
</td>
</tr>
<tr>
<td style="text-align:left;">
10
</td>
<td style="text-align:left;">
gene
</td>
<td style="text-align:right;">
77.90772
</td>
<td style="text-align:right;">
77.97056
</td>
<td style="text-align:left;">
negative
</td>
<td style="text-align:left;">
MGI_C57BL6J_1351901
</td>
<td style="text-align:left;">
Trpm2
</td>
<td style="text-align:left;">
NCBI_Gene:28240,ENSEMBL:ENSMUSG00000009292
</td>
<td style="text-align:left;">
MGI:1351901
</td>
<td style="text-align:left;">
protein coding gene
</td>
<td style="text-align:left;">
transient receptor potential cation channel, subfamily M, member 2
</td>
</tr>
<tr>
<td style="text-align:left;">
10
</td>
<td style="text-align:left;">
gene
</td>
<td style="text-align:right;">
77.91264
</td>
<td style="text-align:right;">
77.91271
</td>
<td style="text-align:left;">
negative
</td>
<td style="text-align:left;">
MGI_C57BL6J_5452171
</td>
<td style="text-align:left;">
Gm22394
</td>
<td style="text-align:left;">
ENSEMBL:ENSMUSG00000093784
</td>
<td style="text-align:left;">
MGI:5452171
</td>
<td style="text-align:left;">
snoRNA gene
</td>
<td style="text-align:left;">
predicted gene, 22394
</td>
</tr>
<tr>
<td style="text-align:left;">
10
</td>
<td style="text-align:left;">
gene
</td>
<td style="text-align:right;">
77.91380
</td>
<td style="text-align:right;">
77.91385
</td>
<td style="text-align:left;">
negative
</td>
<td style="text-align:left;">
MGI_C57BL6J_5452648
</td>
<td style="text-align:left;">
Gm22871
</td>
<td style="text-align:left;">
ENSEMBL:ENSMUSG00000093443
</td>
<td style="text-align:left;">
MGI:5452648
</td>
<td style="text-align:left;">
snoRNA gene
</td>
<td style="text-align:left;">
predicted gene, 22871
</td>
</tr>
<tr>
<td style="text-align:left;">
10
</td>
<td style="text-align:left;">
gene
</td>
<td style="text-align:right;">
77.93906
</td>
<td style="text-align:right;">
77.94026
</td>
<td style="text-align:left;">
negative
</td>
<td style="text-align:left;">
MGI_C57BL6J_5623590
</td>
<td style="text-align:left;">
Gm40705
</td>
<td style="text-align:left;">
NCBI_Gene:105245221
</td>
<td style="text-align:left;">
MGI:5623590
</td>
<td style="text-align:left;">
lncRNA gene
</td>
<td style="text-align:left;">
predicted gene, 40705
</td>
</tr>
<tr>
<td style="text-align:left;">
10
</td>
<td style="text-align:left;">
gene
</td>
<td style="text-align:right;">
77.97852
</td>
<td style="text-align:right;">
77.98690
</td>
<td style="text-align:left;">
positive
</td>
<td style="text-align:left;">
MGI_C57BL6J_1915134
</td>
<td style="text-align:left;">
Cfap410
</td>
<td style="text-align:left;">
NCBI_Gene:67884,ENSEMBL:ENSMUSG00000020284
</td>
<td style="text-align:left;">
MGI:1915134
</td>
<td style="text-align:left;">
protein coding gene
</td>
<td style="text-align:left;">
cilia and flagella associated protein 410
</td>
</tr>
<tr>
<td style="text-align:left;">
10
</td>
<td style="text-align:left;">
gene
</td>
<td style="text-align:right;">
77.98695
</td>
<td style="text-align:right;">
78.01008
</td>
<td style="text-align:left;">
negative
</td>
<td style="text-align:left;">
MGI_C57BL6J_97547
</td>
<td style="text-align:left;">
Pfkl
</td>
<td style="text-align:left;">
NCBI_Gene:18641,ENSEMBL:ENSMUSG00000020277
</td>
<td style="text-align:left;">
MGI:97547
</td>
<td style="text-align:left;">
protein coding gene
</td>
<td style="text-align:left;">
phosphofructokinase, liver, B-type
</td>
</tr>
<tr>
<td style="text-align:left;">
10
</td>
<td style="text-align:left;">
gene
</td>
<td style="text-align:right;">
78.03002
</td>
<td style="text-align:right;">
78.04361
</td>
<td style="text-align:left;">
negative
</td>
<td style="text-align:left;">
MGI_C57BL6J_1338803
</td>
<td style="text-align:left;">
Aire
</td>
<td style="text-align:left;">
NCBI_Gene:11634,ENSEMBL:ENSMUSG00000000731
</td>
<td style="text-align:left;">
MGI:1338803
</td>
<td style="text-align:left;">
protein coding gene
</td>
<td style="text-align:left;">
autoimmune regulator (autoimmune polyendocrinopathy candidiasis ectodermal dystrophy)
</td>
</tr>
<tr>
<td style="text-align:left;">
10
</td>
<td style="text-align:left;">
gene
</td>
<td style="text-align:right;">
78.04195
</td>
<td style="text-align:right;">
78.06362
</td>
<td style="text-align:left;">
positive
</td>
<td style="text-align:left;">
MGI_C57BL6J_1859287
</td>
<td style="text-align:left;">
Dnmt3l
</td>
<td style="text-align:left;">
NCBI_Gene:54427,ENSEMBL:ENSMUSG00000000730
</td>
<td style="text-align:left;">
MGI:1859287
</td>
<td style="text-align:left;">
protein coding gene
</td>
<td style="text-align:left;">
DNA (cytosine-5-)-methyltransferase 3-like
</td>
</tr>
<tr>
<td style="text-align:left;">
10
</td>
<td style="text-align:left;">
gene
</td>
<td style="text-align:right;">
78.06930
</td>
<td style="text-align:right;">
78.08391
</td>
<td style="text-align:left;">
positive
</td>
<td style="text-align:left;">
MGI_C57BL6J_1354701
</td>
<td style="text-align:left;">
Icosl
</td>
<td style="text-align:left;">
NCBI_Gene:50723,ENSEMBL:ENSMUSG00000000732
</td>
<td style="text-align:left;">
MGI:1354701
</td>
<td style="text-align:left;">
protein coding gene
</td>
<td style="text-align:left;">
icos ligand
</td>
</tr>
<tr>
<td style="text-align:left;">
10
</td>
<td style="text-align:left;">
gene
</td>
<td style="text-align:right;">
78.09134
</td>
<td style="text-align:right;">
78.09296
</td>
<td style="text-align:left;">
negative
</td>
<td style="text-align:left;">
MGI_C57BL6J_5589198
</td>
<td style="text-align:left;">
Gm30039
</td>
<td style="text-align:left;">
NCBI_Gene:102631793
</td>
<td style="text-align:left;">
MGI:5589198
</td>
<td style="text-align:left;">
lncRNA gene
</td>
<td style="text-align:left;">
predicted gene, 30039
</td>
</tr>
<tr>
<td style="text-align:left;">
10
</td>
<td style="text-align:left;">
gene
</td>
<td style="text-align:right;">
78.12143
</td>
<td style="text-align:right;">
78.13810
</td>
<td style="text-align:left;">
negative
</td>
<td style="text-align:left;">
MGI_C57BL6J_6097178
</td>
<td style="text-align:left;">
Gm47922
</td>
<td style="text-align:left;">
ENSEMBL:ENSMUSG00000112662
</td>
<td style="text-align:left;">
MGI:6097178
</td>
<td style="text-align:left;">
lncRNA gene
</td>
<td style="text-align:left;">
predicted gene, 47922
</td>
</tr>
<tr>
<td style="text-align:left;">
10
</td>
<td style="text-align:left;">
gene
</td>
<td style="text-align:right;">
78.16207
</td>
<td style="text-align:right;">
78.16978
</td>
<td style="text-align:left;">
negative
</td>
<td style="text-align:left;">
MGI_C57BL6J_1351861
</td>
<td style="text-align:left;">
Gatd3a
</td>
<td style="text-align:left;">
NCBI_Gene:28295,ENSEMBL:ENSMUSG00000053329
</td>
<td style="text-align:left;">
MGI:1351861
</td>
<td style="text-align:left;">
protein coding gene
</td>
<td style="text-align:left;">
glutamine amidotransferase like class 1 domain containing 3A
</td>
</tr>
<tr>
<td style="text-align:left;">
10
</td>
<td style="text-align:left;">
gene
</td>
<td style="text-align:right;">
78.17091
</td>
<td style="text-align:right;">
78.18517
</td>
<td style="text-align:left;">
negative
</td>
<td style="text-align:left;">
MGI_C57BL6J_1341200
</td>
<td style="text-align:left;">
Pwp2
</td>
<td style="text-align:left;">
NCBI_Gene:110816,ENSEMBL:ENSMUSG00000032834
</td>
<td style="text-align:left;">
MGI:1341200
</td>
<td style="text-align:left;">
protein coding gene
</td>
<td style="text-align:left;">
PWP2 periodic tryptophan protein homolog (yeast)
</td>
</tr>
<tr>
<td style="text-align:left;">
10
</td>
<td style="text-align:left;">
gene
</td>
<td style="text-align:right;">
78.18672
</td>
<td style="text-align:right;">
78.24464
</td>
<td style="text-align:left;">
negative
</td>
<td style="text-align:left;">
MGI_C57BL6J_1336209
</td>
<td style="text-align:left;">
Trappc10
</td>
<td style="text-align:left;">
NCBI_Gene:216131,ENSEMBL:ENSMUSG00000000374
</td>
<td style="text-align:left;">
MGI:1336209
</td>
<td style="text-align:left;">
protein coding gene
</td>
<td style="text-align:left;">
trafficking protein particle complex 10
</td>
</tr>
<tr>
<td style="text-align:left;">
10
</td>
<td style="text-align:left;">
gene
</td>
<td style="text-align:right;">
78.24662
</td>
<td style="text-align:right;">
78.25543
</td>
<td style="text-align:left;">
positive
</td>
<td style="text-align:left;">
MGI_C57BL6J_5623593
</td>
<td style="text-align:left;">
Gm40708
</td>
<td style="text-align:left;">
NCBI_Gene:105245224
</td>
<td style="text-align:left;">
MGI:5623593
</td>
<td style="text-align:left;">
lncRNA gene
</td>
<td style="text-align:left;">
predicted gene, 40708
</td>
</tr>
<tr>
<td style="text-align:left;">
10
</td>
<td style="text-align:left;">
gene
</td>
<td style="text-align:right;">
78.24888
</td>
<td style="text-align:right;">
78.26414
</td>
<td style="text-align:left;">
negative
</td>
<td style="text-align:left;">
MGI_C57BL6J_5589281
</td>
<td style="text-align:left;">
Gm30122
</td>
<td style="text-align:left;">
NCBI_Gene:102631905,ENSEMBL:ENSMUSG00000112433
</td>
<td style="text-align:left;">
MGI:5589281
</td>
<td style="text-align:left;">
lncRNA gene
</td>
<td style="text-align:left;">
predicted gene, 30122
</td>
</tr>
<tr>
<td style="text-align:left;">
10
</td>
<td style="text-align:left;">
gene
</td>
<td style="text-align:right;">
78.26453
</td>
<td style="text-align:right;">
78.26608
</td>
<td style="text-align:left;">
negative
</td>
<td style="text-align:left;">
MGI_C57BL6J_6097260
</td>
<td style="text-align:left;">
Gm47976
</td>
<td style="text-align:left;">
ENSEMBL:ENSMUSG00000112300
</td>
<td style="text-align:left;">
MGI:6097260
</td>
<td style="text-align:left;">
unclassified gene
</td>
<td style="text-align:left;">
predicted gene, 47976
</td>
</tr>
<tr>
<td style="text-align:left;">
10
</td>
<td style="text-align:left;">
gene
</td>
<td style="text-align:right;">
78.26917
</td>
<td style="text-align:right;">
78.35249
</td>
<td style="text-align:left;">
negative
</td>
<td style="text-align:left;">
MGI_C57BL6J_1336186
</td>
<td style="text-align:left;">
Agpat3
</td>
<td style="text-align:left;">
NCBI_Gene:28169,ENSEMBL:ENSMUSG00000001211
</td>
<td style="text-align:left;">
MGI:1336186
</td>
<td style="text-align:left;">
protein coding gene
</td>
<td style="text-align:left;">
1-acylglycerol-3-phosphate O-acyltransferase 3
</td>
</tr>
<tr>
<td style="text-align:left;">
10
</td>
<td style="text-align:left;">
gene
</td>
<td style="text-align:right;">
78.29832
</td>
<td style="text-align:right;">
78.30030
</td>
<td style="text-align:left;">
positive
</td>
<td style="text-align:left;">
MGI_C57BL6J_6097287
</td>
<td style="text-align:left;">
Gm47992
</td>
<td style="text-align:left;">
ENSEMBL:ENSMUSG00000112442
</td>
<td style="text-align:left;">
MGI:6097287
</td>
<td style="text-align:left;">
unclassified gene
</td>
<td style="text-align:left;">
predicted gene, 47992
</td>
</tr>
<tr>
<td style="text-align:left;">
10
</td>
<td style="text-align:left;">
gene
</td>
<td style="text-align:right;">
78.31417
</td>
<td style="text-align:right;">
78.31428
</td>
<td style="text-align:left;">
positive
</td>
<td style="text-align:left;">
MGI_C57BL6J_4421925
</td>
<td style="text-align:left;">
n-R5s77
</td>
<td style="text-align:left;">
ENSEMBL:ENSMUSG00000065691
</td>
<td style="text-align:left;">
MGI:4421925
</td>
<td style="text-align:left;">
rRNA gene
</td>
<td style="text-align:left;">
nuclear encoded rRNA 5S 77
</td>
</tr>
<tr>
<td style="text-align:left;">
10
</td>
<td style="text-align:left;">
gene
</td>
<td style="text-align:right;">
78.34015
</td>
<td style="text-align:right;">
78.34021
</td>
<td style="text-align:left;">
negative
</td>
<td style="text-align:left;">
MGI_C57BL6J_5530815
</td>
<td style="text-align:left;">
Mir7659
</td>
<td style="text-align:left;">
miRBase:MI0024999,NCBI_Gene:102465765,ENSEMBL:ENSMUSG00000098885
</td>
<td style="text-align:left;">
MGI:5530815
</td>
<td style="text-align:left;">
miRNA gene
</td>
<td style="text-align:left;">
microRNA 7659
</td>
</tr>
<tr>
<td style="text-align:left;">
10
</td>
<td style="text-align:left;">
gene
</td>
<td style="text-align:right;">
78.35247
</td>
<td style="text-align:right;">
78.35571
</td>
<td style="text-align:left;">
positive
</td>
<td style="text-align:left;">
MGI_C57BL6J_5623594
</td>
<td style="text-align:left;">
Gm40709
</td>
<td style="text-align:left;">
NCBI_Gene:105245225,ENSEMBL:ENSMUSG00000112641
</td>
<td style="text-align:left;">
MGI:5623594
</td>
<td style="text-align:left;">
lncRNA gene
</td>
<td style="text-align:left;">
predicted gene, 40709
</td>
</tr>
<tr>
<td style="text-align:left;">
10
</td>
<td style="text-align:left;">
gene
</td>
<td style="text-align:right;">
78.37001
</td>
<td style="text-align:right;">
78.37032
</td>
<td style="text-align:left;">
negative
</td>
<td style="text-align:left;">
MGI_C57BL6J_5452291
</td>
<td style="text-align:left;">
Gm22514
</td>
<td style="text-align:left;">
ENSEMBL:ENSMUSG00000084496
</td>
<td style="text-align:left;">
MGI:5452291
</td>
<td style="text-align:left;">
unclassified non-coding RNA gene
</td>
<td style="text-align:left;">
predicted gene, 22514
</td>
</tr>
<tr>
<td style="text-align:left;">
10
</td>
<td style="text-align:left;">
pseudogene
</td>
<td style="text-align:right;">
78.39330
</td>
<td style="text-align:right;">
78.39376
</td>
<td style="text-align:left;">
positive
</td>
<td style="text-align:left;">
MGI_C57BL6J_3704367
</td>
<td style="text-align:left;">
Gm10146
</td>
<td style="text-align:left;">
NCBI_Gene:100041500,ENSEMBL:ENSMUSG00000064317
</td>
<td style="text-align:left;">
MGI:3704367
</td>
<td style="text-align:left;">
pseudogene
</td>
<td style="text-align:left;">
predicted gene 10146
</td>
</tr>
<tr>
<td style="text-align:left;">
10
</td>
<td style="text-align:left;">
gene
</td>
<td style="text-align:right;">
78.40036
</td>
<td style="text-align:right;">
78.41304
</td>
<td style="text-align:left;">
negative
</td>
<td style="text-align:left;">
MGI_C57BL6J_1203500
</td>
<td style="text-align:left;">
Rrp1
</td>
<td style="text-align:left;">
NCBI_Gene:18114,ENSEMBL:ENSMUSG00000061032
</td>
<td style="text-align:left;">
MGI:1203500
</td>
<td style="text-align:left;">
protein coding gene
</td>
<td style="text-align:left;">
ribosomal RNA processing 1 homolog (S. cerevisiae)
</td>
</tr>
<tr>
<td style="text-align:left;">
10
</td>
<td style="text-align:left;">
gene
</td>
<td style="text-align:right;">
78.40847
</td>
<td style="text-align:right;">
78.40855
</td>
<td style="text-align:left;">
negative
</td>
<td style="text-align:left;">
MGI_C57BL6J_5530733
</td>
<td style="text-align:left;">
Mir6907
</td>
<td style="text-align:left;">
miRBase:MI0022754,NCBI_Gene:102465547,ENSEMBL:ENSMUSG00000098376
</td>
<td style="text-align:left;">
MGI:5530733
</td>
<td style="text-align:left;">
miRNA gene
</td>
<td style="text-align:left;">
microRNA 6907
</td>
</tr>
<tr>
<td style="text-align:left;">
10
</td>
<td style="text-align:left;">
gene
</td>
<td style="text-align:right;">
78.42567
</td>
<td style="text-align:right;">
78.42762
</td>
<td style="text-align:left;">
positive
</td>
<td style="text-align:left;">
MGI_C57BL6J_109514
</td>
<td style="text-align:left;">
Cstb
</td>
<td style="text-align:left;">
NCBI_Gene:13014,ENSEMBL:ENSMUSG00000005054
</td>
<td style="text-align:left;">
MGI:109514
</td>
<td style="text-align:left;">
protein coding gene
</td>
<td style="text-align:left;">
cystatin B
</td>
</tr>
<tr>
<td style="text-align:left;">
10
</td>
<td style="text-align:left;">
gene
</td>
<td style="text-align:right;">
78.42827
</td>
<td style="text-align:right;">
78.43885
</td>
<td style="text-align:left;">
positive
</td>
<td style="text-align:left;">
MGI_C57BL6J_3696861
</td>
<td style="text-align:left;">
4921516A02Rik
</td>
<td style="text-align:left;">
ENSEMBL:ENSMUSG00000112652
</td>
<td style="text-align:left;">
MGI:3696861
</td>
<td style="text-align:left;">
lncRNA gene
</td>
<td style="text-align:left;">
RIKEN cDNA 4921516A02 gene
</td>
</tr>
<tr>
<td style="text-align:left;">
10
</td>
<td style="text-align:left;">
gene
</td>
<td style="text-align:right;">
78.43480
</td>
<td style="text-align:right;">
78.43490
</td>
<td style="text-align:left;">
positive
</td>
<td style="text-align:left;">
MGI_C57BL6J_5531298
</td>
<td style="text-align:left;">
Mir6410
</td>
<td style="text-align:left;">
miRBase:MI0021947,NCBI_Gene:102465221,ENSEMBL:ENSMUSG00000098932
</td>
<td style="text-align:left;">
MGI:5531298
</td>
<td style="text-align:left;">
miRNA gene
</td>
<td style="text-align:left;">
microRNA 6410
</td>
</tr>
<tr>
<td style="text-align:left;">
10
</td>
<td style="text-align:left;">
gene
</td>
<td style="text-align:right;">
78.43674
</td>
<td style="text-align:right;">
78.46497
</td>
<td style="text-align:left;">
negative
</td>
<td style="text-align:left;">
MGI_C57BL6J_1351869
</td>
<td style="text-align:left;">
Pdxk
</td>
<td style="text-align:left;">
NCBI_Gene:216134,ENSEMBL:ENSMUSG00000032788
</td>
<td style="text-align:left;">
MGI:1351869
</td>
<td style="text-align:left;">
protein coding gene
</td>
<td style="text-align:left;">
pyridoxal (pyridoxine, vitamin B6) kinase
</td>
</tr>
<tr>
<td style="text-align:left;">
10
</td>
<td style="text-align:left;">
gene
</td>
<td style="text-align:right;">
78.44509
</td>
<td style="text-align:right;">
78.44516
</td>
<td style="text-align:left;">
negative
</td>
<td style="text-align:left;">
MGI_C57BL6J_5531209
</td>
<td style="text-align:left;">
Mir6908
</td>
<td style="text-align:left;">
miRBase:MI0022755,NCBI_Gene:102466762,ENSEMBL:ENSMUSG00000098279
</td>
<td style="text-align:left;">
MGI:5531209
</td>
<td style="text-align:left;">
miRNA gene
</td>
<td style="text-align:left;">
microRNA 6908
</td>
</tr>
<tr>
<td style="text-align:left;">
10
</td>
<td style="text-align:left;">
gene
</td>
<td style="text-align:right;">
78.46981
</td>
<td style="text-align:right;">
78.47126
</td>
<td style="text-align:left;">
negative
</td>
<td style="text-align:left;">
MGI_C57BL6J_1918516
</td>
<td style="text-align:left;">
4933436P19Rik
</td>
<td style="text-align:left;">
ENSEMBL:ENSMUSG00000112623
</td>
<td style="text-align:left;">
MGI:1918516
</td>
<td style="text-align:left;">
unclassified gene
</td>
<td style="text-align:left;">
RIKEN cDNA 4933436P19 gene
</td>
</tr>
<tr>
<td style="text-align:left;">
10
</td>
<td style="text-align:left;">
gene
</td>
<td style="text-align:right;">
78.48457
</td>
<td style="text-align:right;">
78.48784
</td>
<td style="text-align:left;">
negative
</td>
<td style="text-align:left;">
MGI_C57BL6J_3641806
</td>
<td style="text-align:left;">
Gm9978
</td>
<td style="text-align:left;">
ENSEMBL:ENSMUSG00000055704
</td>
<td style="text-align:left;">
MGI:3641806
</td>
<td style="text-align:left;">
protein coding gene
</td>
<td style="text-align:left;">
predicted gene 9978
</td>
</tr>
<tr>
<td style="text-align:left;">
10
</td>
<td style="text-align:left;">
gene
</td>
<td style="text-align:right;">
78.48706
</td>
<td style="text-align:right;">
78.49835
</td>
<td style="text-align:left;">
positive
</td>
<td style="text-align:left;">
MGI_C57BL6J_5589391
</td>
<td style="text-align:left;">
Gm30232
</td>
<td style="text-align:left;">
NCBI_Gene:102632059,ENSEMBL:ENSMUSG00000112842
</td>
<td style="text-align:left;">
MGI:5589391
</td>
<td style="text-align:left;">
lncRNA gene
</td>
<td style="text-align:left;">
predicted gene, 30232
</td>
</tr>
<tr>
<td style="text-align:left;">
10
</td>
<td style="text-align:left;">
gene
</td>
<td style="text-align:right;">
78.49805
</td>
<td style="text-align:right;">
78.52057
</td>
<td style="text-align:left;">
positive
</td>
<td style="text-align:left;">
MGI_C57BL6J_3031192
</td>
<td style="text-align:left;">
Olfr1358
</td>
<td style="text-align:left;">
NCBI_Gene:258224,ENSEMBL:ENSMUSG00000107822
</td>
<td style="text-align:left;">
MGI:3031192
</td>
<td style="text-align:left;">
protein coding gene
</td>
<td style="text-align:left;">
olfactory receptor 1358
</td>
</tr>
<tr>
<td style="text-align:left;">
10
</td>
<td style="text-align:left;">
gene
</td>
<td style="text-align:right;">
78.50192
</td>
<td style="text-align:right;">
78.52022
</td>
<td style="text-align:left;">
negative
</td>
<td style="text-align:left;">
MGI_C57BL6J_6097210
</td>
<td style="text-align:left;">
Gm47944
</td>
<td style="text-align:left;">
ENSEMBL:ENSMUSG00000112224
</td>
<td style="text-align:left;">
MGI:6097210
</td>
<td style="text-align:left;">
lncRNA gene
</td>
<td style="text-align:left;">
predicted gene, 47944
</td>
</tr>
<tr>
<td style="text-align:left;">
10
</td>
<td style="text-align:left;">
gene
</td>
<td style="text-align:right;">
78.51862
</td>
<td style="text-align:right;">
78.51874
</td>
<td style="text-align:left;">
positive
</td>
<td style="text-align:left;">
MGI_C57BL6J_5454941
</td>
<td style="text-align:left;">
Gm25164
</td>
<td style="text-align:left;">
ENSEMBL:ENSMUSG00000089533
</td>
<td style="text-align:left;">
MGI:5454941
</td>
<td style="text-align:left;">
rRNA gene
</td>
<td style="text-align:left;">
predicted gene, 25164
</td>
</tr>
<tr>
<td style="text-align:left;">
10
</td>
<td style="text-align:left;">
gene
</td>
<td style="text-align:right;">
78.52599
</td>
<td style="text-align:right;">
78.52611
</td>
<td style="text-align:left;">
positive
</td>
<td style="text-align:left;">
MGI_C57BL6J_5530923
</td>
<td style="text-align:left;">
Gm27541
</td>
<td style="text-align:left;">
ENSEMBL:ENSMUSG00000099247
</td>
<td style="text-align:left;">
MGI:5530923
</td>
<td style="text-align:left;">
miRNA gene
</td>
<td style="text-align:left;">
predicted gene, 27541
</td>
</tr>
<tr>
<td style="text-align:left;">
10
</td>
<td style="text-align:left;">
pseudogene
</td>
<td style="text-align:right;">
78.54327
</td>
<td style="text-align:right;">
78.54378
</td>
<td style="text-align:left;">
positive
</td>
<td style="text-align:left;">
MGI_C57BL6J_3647256
</td>
<td style="text-align:left;">
Gm5951
</td>
<td style="text-align:left;">
NCBI_Gene:546482,ENSEMBL:ENSMUSG00000112313
</td>
<td style="text-align:left;">
MGI:3647256
</td>
<td style="text-align:left;">
pseudogene
</td>
<td style="text-align:left;">
predicted gene 5951
</td>
</tr>
<tr>
<td style="text-align:left;">
10
</td>
<td style="text-align:left;">
pseudogene
</td>
<td style="text-align:right;">
78.55402
</td>
<td style="text-align:right;">
78.55442
</td>
<td style="text-align:left;">
positive
</td>
<td style="text-align:left;">
MGI_C57BL6J_3761345
</td>
<td style="text-align:left;">
Vmn2r-ps92
</td>
<td style="text-align:left;">
NCBI_Gene:100124575,ENSEMBL:ENSMUSG00000111881
</td>
<td style="text-align:left;">
MGI:3761345
</td>
<td style="text-align:left;">
pseudogene
</td>
<td style="text-align:left;">
vomeronasal 2, receptor, pseudogene 92
</td>
</tr>
<tr>
<td style="text-align:left;">
10
</td>
<td style="text-align:left;">
gene
</td>
<td style="text-align:right;">
78.57434
</td>
<td style="text-align:right;">
78.58450
</td>
<td style="text-align:left;">
positive
</td>
<td style="text-align:left;">
MGI_C57BL6J_1351911
</td>
<td style="text-align:left;">
Ilvbl
</td>
<td style="text-align:left;">
NCBI_Gene:216136,ENSEMBL:ENSMUSG00000032763
</td>
<td style="text-align:left;">
MGI:1351911
</td>
<td style="text-align:left;">
protein coding gene
</td>
<td style="text-align:left;">
ilvB (bacterial acetolactate synthase)-like
</td>
</tr>
<tr>
<td style="text-align:left;">
10
</td>
<td style="text-align:left;">
gene
</td>
<td style="text-align:right;">
78.58450
</td>
<td style="text-align:right;">
78.59196
</td>
<td style="text-align:left;">
negative
</td>
<td style="text-align:left;">
MGI_C57BL6J_1918959
</td>
<td style="text-align:left;">
Syde1
</td>
<td style="text-align:left;">
NCBI_Gene:71709,ENSEMBL:ENSMUSG00000032714
</td>
<td style="text-align:left;">
MGI:1918959
</td>
<td style="text-align:left;">
protein coding gene
</td>
<td style="text-align:left;">
synapse defective 1, Rho GTPase, homolog 1 (C. elegans)
</td>
</tr>
<tr>
<td style="text-align:left;">
10
</td>
<td style="text-align:left;">
gene
</td>
<td style="text-align:right;">
78.59171
</td>
<td style="text-align:right;">
78.59728
</td>
<td style="text-align:left;">
positive
</td>
<td style="text-align:left;">
MGI_C57BL6J_5589505
</td>
<td style="text-align:left;">
Gm30346
</td>
<td style="text-align:left;">
NCBI_Gene:102632206,ENSEMBL:ENSMUSG00000112021
</td>
<td style="text-align:left;">
MGI:5589505
</td>
<td style="text-align:left;">
lncRNA gene
</td>
<td style="text-align:left;">
predicted gene, 30346
</td>
</tr>
<tr>
<td style="text-align:left;">
10
</td>
<td style="text-align:left;">
gene
</td>
<td style="text-align:right;">
78.59912
</td>
<td style="text-align:right;">
78.60187
</td>
<td style="text-align:left;">
negative
</td>
<td style="text-align:left;">
MGI_C57BL6J_5610742
</td>
<td style="text-align:left;">
Gm37514
</td>
<td style="text-align:left;">
ENSEMBL:ENSMUSG00000104370
</td>
<td style="text-align:left;">
MGI:5610742
</td>
<td style="text-align:left;">
unclassified gene
</td>
<td style="text-align:left;">
predicted gene, 37514
</td>
</tr>
<tr>
<td style="text-align:left;">
10
</td>
<td style="text-align:left;">
gene
</td>
<td style="text-align:right;">
78.61100
</td>
<td style="text-align:right;">
78.61807
</td>
<td style="text-align:left;">
negative
</td>
<td style="text-align:left;">
MGI_C57BL6J_3031191
</td>
<td style="text-align:left;">
Olfr1357
</td>
<td style="text-align:left;">
NCBI_Gene:257883,ENSEMBL:ENSMUSG00000071185
</td>
<td style="text-align:left;">
MGI:3031191
</td>
<td style="text-align:left;">
protein coding gene
</td>
<td style="text-align:left;">
olfactory receptor 1357
</td>
</tr>
<tr>
<td style="text-align:left;">
10
</td>
<td style="text-align:left;">
pseudogene
</td>
<td style="text-align:right;">
78.62892
</td>
<td style="text-align:right;">
78.63030
</td>
<td style="text-align:left;">
negative
</td>
<td style="text-align:left;">
MGI_C57BL6J_3646991
</td>
<td style="text-align:left;">
Gm7842
</td>
<td style="text-align:left;">
NCBI_Gene:665890,ENSEMBL:ENSMUSG00000112017
</td>
<td style="text-align:left;">
MGI:3646991
</td>
<td style="text-align:left;">
pseudogene
</td>
<td style="text-align:left;">
predicted gene 7842
</td>
</tr>
<tr>
<td style="text-align:left;">
10
</td>
<td style="text-align:left;">
gene
</td>
<td style="text-align:right;">
78.62955
</td>
<td style="text-align:right;">
78.63262
</td>
<td style="text-align:left;">
positive
</td>
<td style="text-align:left;">
MGI_C57BL6J_6096421
</td>
<td style="text-align:left;">
Gm47459
</td>
<td style="text-align:left;">
ENSEMBL:ENSMUSG00000112674
</td>
<td style="text-align:left;">
MGI:6096421
</td>
<td style="text-align:left;">
lncRNA gene
</td>
<td style="text-align:left;">
predicted gene, 47459
</td>
</tr>
<tr>
<td style="text-align:left;">
10
</td>
<td style="text-align:left;">
pseudogene
</td>
<td style="text-align:right;">
78.69497
</td>
<td style="text-align:right;">
78.69881
</td>
<td style="text-align:left;">
negative
</td>
<td style="text-align:left;">
MGI_C57BL6J_6096448
</td>
<td style="text-align:left;">
Gm47476
</td>
<td style="text-align:left;">
ENSEMBL:ENSMUSG00000112689
</td>
<td style="text-align:left;">
MGI:6096448
</td>
<td style="text-align:left;">
pseudogene
</td>
<td style="text-align:left;">
predicted gene, 47476
</td>
</tr>
<tr>
<td style="text-align:left;">
10
</td>
<td style="text-align:left;">
pseudogene
</td>
<td style="text-align:right;">
78.69961
</td>
<td style="text-align:right;">
78.69987
</td>
<td style="text-align:left;">
positive
</td>
<td style="text-align:left;">
MGI_C57BL6J_6096478
</td>
<td style="text-align:left;">
Gm47495
</td>
<td style="text-align:left;">
ENSEMBL:ENSMUSG00000111946
</td>
<td style="text-align:left;">
MGI:6096478
</td>
<td style="text-align:left;">
pseudogene
</td>
<td style="text-align:left;">
predicted gene, 47495
</td>
</tr>
<tr>
<td style="text-align:left;">
10
</td>
<td style="text-align:left;">
gene
</td>
<td style="text-align:right;">
78.71199
</td>
<td style="text-align:right;">
78.71835
</td>
<td style="text-align:left;">
negative
</td>
<td style="text-align:left;">
MGI_C57BL6J_1335092
</td>
<td style="text-align:left;">
Casp14
</td>
<td style="text-align:left;">
NCBI_Gene:12365,ENSEMBL:ENSMUSG00000005355
</td>
<td style="text-align:left;">
MGI:1335092
</td>
<td style="text-align:left;">
protein coding gene
</td>
<td style="text-align:left;">
caspase 14
</td>
</tr>
<tr>
<td style="text-align:left;">
10
</td>
<td style="text-align:left;">
gene
</td>
<td style="text-align:right;">
78.74692
</td>
<td style="text-align:right;">
78.75307
</td>
<td style="text-align:left;">
negative
</td>
<td style="text-align:left;">
MGI_C57BL6J_1918226
</td>
<td style="text-align:left;">
Ccdc105
</td>
<td style="text-align:left;">
NCBI_Gene:70976,ENSEMBL:ENSMUSG00000078442
</td>
<td style="text-align:left;">
MGI:1918226
</td>
<td style="text-align:left;">
protein coding gene
</td>
<td style="text-align:left;">
coiled-coil domain containing 105
</td>
</tr>
<tr>
<td style="text-align:left;">
10
</td>
<td style="text-align:left;">
gene
</td>
<td style="text-align:right;">
78.74713
</td>
<td style="text-align:right;">
78.77236
</td>
<td style="text-align:left;">
positive
</td>
<td style="text-align:left;">
MGI_C57BL6J_5589559
</td>
<td style="text-align:left;">
Gm30400
</td>
<td style="text-align:left;">
NCBI_Gene:102632284,ENSEMBL:ENSMUSG00000109776
</td>
<td style="text-align:left;">
MGI:5589559
</td>
<td style="text-align:left;">
lncRNA gene
</td>
<td style="text-align:left;">
predicted gene, 30400
</td>
</tr>
<tr>
<td style="text-align:left;">
10
</td>
<td style="text-align:left;">
gene
</td>
<td style="text-align:right;">
78.77950
</td>
<td style="text-align:right;">
78.78782
</td>
<td style="text-align:left;">
negative
</td>
<td style="text-align:left;">
MGI_C57BL6J_5623595
</td>
<td style="text-align:left;">
Gm40710
</td>
<td style="text-align:left;">
NCBI_Gene:105245226
</td>
<td style="text-align:left;">
MGI:5623595
</td>
<td style="text-align:left;">
lncRNA gene
</td>
<td style="text-align:left;">
predicted gene, 40710
</td>
</tr>
<tr>
<td style="text-align:left;">
10
</td>
<td style="text-align:left;">
gene
</td>
<td style="text-align:right;">
78.78050
</td>
<td style="text-align:right;">
78.81482
</td>
<td style="text-align:left;">
positive
</td>
<td style="text-align:left;">
MGI_C57BL6J_1096331
</td>
<td style="text-align:left;">
Slc1a6
</td>
<td style="text-align:left;">
NCBI_Gene:20513,ENSEMBL:ENSMUSG00000005357
</td>
<td style="text-align:left;">
MGI:1096331
</td>
<td style="text-align:left;">
protein coding gene
</td>
<td style="text-align:left;">
solute carrier family 1 (high affinity aspartate/glutamate transporter), member 6
</td>
</tr>
<tr>
<td style="text-align:left;">
10
</td>
<td style="text-align:left;">
gene
</td>
<td style="text-align:right;">
78.84505
</td>
<td style="text-align:right;">
78.85217
</td>
<td style="text-align:left;">
negative
</td>
<td style="text-align:left;">
MGI_C57BL6J_3031190
</td>
<td style="text-align:left;">
Olfr1356
</td>
<td style="text-align:left;">
NCBI_Gene:258305,ENSEMBL:ENSMUSG00000051190
</td>
<td style="text-align:left;">
MGI:3031190
</td>
<td style="text-align:left;">
protein coding gene
</td>
<td style="text-align:left;">
olfactory receptor 1356
</td>
</tr>
<tr>
<td style="text-align:left;">
10
</td>
<td style="text-align:left;">
gene
</td>
<td style="text-align:right;">
78.87554
</td>
<td style="text-align:right;">
78.88227
</td>
<td style="text-align:left;">
positive
</td>
<td style="text-align:left;">
MGI_C57BL6J_3031189
</td>
<td style="text-align:left;">
Olfr1355
</td>
<td style="text-align:left;">
NCBI_Gene:257734,ENSEMBL:ENSMUSG00000062873
</td>
<td style="text-align:left;">
MGI:3031189
</td>
<td style="text-align:left;">
protein coding gene
</td>
<td style="text-align:left;">
olfactory receptor 1355
</td>
</tr>
<tr>
<td style="text-align:left;">
10
</td>
<td style="text-align:left;">
gene
</td>
<td style="text-align:right;">
78.91317
</td>
<td style="text-align:right;">
78.92040
</td>
<td style="text-align:left;">
positive
</td>
<td style="text-align:left;">
MGI_C57BL6J_3031188
</td>
<td style="text-align:left;">
Olfr1354
</td>
<td style="text-align:left;">
NCBI_Gene:259163,ENSEMBL:ENSMUSG00000094673
</td>
<td style="text-align:left;">
MGI:3031188
</td>
<td style="text-align:left;">
protein coding gene
</td>
<td style="text-align:left;">
olfactory receptor 1354
</td>
</tr>
<tr>
<td style="text-align:left;">
10
</td>
<td style="text-align:left;">
gene
</td>
<td style="text-align:right;">
78.95064
</td>
<td style="text-align:right;">
78.95838
</td>
<td style="text-align:left;">
positive
</td>
<td style="text-align:left;">
MGI_C57BL6J_107174
</td>
<td style="text-align:left;">
Olfr8
</td>
<td style="text-align:left;">
NCBI_Gene:18372,ENSEMBL:ENSMUSG00000094080
</td>
<td style="text-align:left;">
MGI:107174
</td>
<td style="text-align:left;">
protein coding gene
</td>
<td style="text-align:left;">
olfactory receptor 8
</td>
</tr>
<tr>
<td style="text-align:left;">
10
</td>
<td style="text-align:left;">
gene
</td>
<td style="text-align:right;">
78.96331
</td>
<td style="text-align:right;">
78.97134
</td>
<td style="text-align:left;">
positive
</td>
<td style="text-align:left;">
MGI_C57BL6J_3031187
</td>
<td style="text-align:left;">
Olfr1353
</td>
<td style="text-align:left;">
NCBI_Gene:259044,ENSEMBL:ENSMUSG00000042774
</td>
<td style="text-align:left;">
MGI:3031187
</td>
<td style="text-align:left;">
protein coding gene
</td>
<td style="text-align:left;">
olfactory receptor 1353
</td>
</tr>
<tr>
<td style="text-align:left;">
10
</td>
<td style="text-align:left;">
gene
</td>
<td style="text-align:right;">
78.97210
</td>
<td style="text-align:right;">
79.01289
</td>
<td style="text-align:left;">
negative
</td>
<td style="text-align:left;">
MGI_C57BL6J_6096238
</td>
<td style="text-align:left;">
Gm47343
</td>
<td style="text-align:left;">
ENSEMBL:ENSMUSG00000112732
</td>
<td style="text-align:left;">
MGI:6096238
</td>
<td style="text-align:left;">
lncRNA gene
</td>
<td style="text-align:left;">
predicted gene, 47343
</td>
</tr>
<tr>
<td style="text-align:left;">
10
</td>
<td style="text-align:left;">
gene
</td>
<td style="text-align:right;">
78.98105
</td>
<td style="text-align:right;">
78.98790
</td>
<td style="text-align:left;">
positive
</td>
<td style="text-align:left;">
MGI_C57BL6J_3031186
</td>
<td style="text-align:left;">
Olfr1352
</td>
<td style="text-align:left;">
NCBI_Gene:259074,ENSEMBL:ENSMUSG00000046493
</td>
<td style="text-align:left;">
MGI:3031186
</td>
<td style="text-align:left;">
protein coding gene
</td>
<td style="text-align:left;">
olfactory receptor 1352
</td>
</tr>
<tr>
<td style="text-align:left;">
10
</td>
<td style="text-align:left;">
gene
</td>
<td style="text-align:right;">
79.01247
</td>
<td style="text-align:right;">
79.01964
</td>
<td style="text-align:left;">
positive
</td>
<td style="text-align:left;">
MGI_C57BL6J_3031185
</td>
<td style="text-align:left;">
Olfr1351
</td>
<td style="text-align:left;">
NCBI_Gene:259042,ENSEMBL:ENSMUSG00000063216
</td>
<td style="text-align:left;">
MGI:3031185
</td>
<td style="text-align:left;">
protein coding gene
</td>
<td style="text-align:left;">
olfactory receptor 1351
</td>
</tr>
<tr>
<td style="text-align:left;">
10
</td>
<td style="text-align:left;">
gene
</td>
<td style="text-align:right;">
79.02781
</td>
<td style="text-align:right;">
79.03603
</td>
<td style="text-align:left;">
positive
</td>
<td style="text-align:left;">
MGI_C57BL6J_5623597
</td>
<td style="text-align:left;">
Gm40712
</td>
<td style="text-align:left;">
NCBI_Gene:105245228
</td>
<td style="text-align:left;">
MGI:5623597
</td>
<td style="text-align:left;">
lncRNA gene
</td>
<td style="text-align:left;">
predicted gene, 40712
</td>
</tr>
<tr>
<td style="text-align:left;">
10
</td>
<td style="text-align:left;">
gene
</td>
<td style="text-align:right;">
79.02874
</td>
<td style="text-align:right;">
79.03627
</td>
<td style="text-align:left;">
positive
</td>
<td style="text-align:left;">
MGI_C57BL6J_1333808
</td>
<td style="text-align:left;">
Olfr57
</td>
<td style="text-align:left;">
NCBI_Gene:18357,ENSEMBL:ENSMUSG00000060205
</td>
<td style="text-align:left;">
MGI:1333808
</td>
<td style="text-align:left;">
protein coding gene
</td>
<td style="text-align:left;">
olfactory receptor 57
</td>
</tr>
<tr>
<td style="text-align:left;">
10
</td>
<td style="text-align:left;">
gene
</td>
<td style="text-align:right;">
79.06437
</td>
<td style="text-align:right;">
79.09762
</td>
<td style="text-align:left;">
negative
</td>
<td style="text-align:left;">
MGI_C57BL6J_1919378
</td>
<td style="text-align:left;">
2610008E11Rik
</td>
<td style="text-align:left;">
NCBI_Gene:72128,ENSEMBL:ENSMUSG00000060301
</td>
<td style="text-align:left;">
MGI:1919378
</td>
<td style="text-align:left;">
protein coding gene
</td>
<td style="text-align:left;">
RIKEN cDNA 2610008E11 gene
</td>
</tr>
<tr>
<td style="text-align:left;">
10
</td>
<td style="text-align:left;">
gene
</td>
<td style="text-align:right;">
79.14877
</td>
<td style="text-align:right;">
79.19501
</td>
<td style="text-align:left;">
positive
</td>
<td style="text-align:left;">
MGI_C57BL6J_3646321
</td>
<td style="text-align:left;">
Vmn2r80
</td>
<td style="text-align:left;">
NCBI_Gene:624765,ENSEMBL:ENSMUSG00000091888
</td>
<td style="text-align:left;">
MGI:3646321
</td>
<td style="text-align:left;">
protein coding gene
</td>
<td style="text-align:left;">
vomeronasal 2, receptor 80
</td>
</tr>
<tr>
<td style="text-align:left;">
10
</td>
<td style="text-align:left;">
gene
</td>
<td style="text-align:right;">
79.24769
</td>
<td style="text-align:right;">
79.29453
</td>
<td style="text-align:left;">
positive
</td>
<td style="text-align:left;">
MGI_C57BL6J_2684056
</td>
<td style="text-align:left;">
Vmn2r81
</td>
<td style="text-align:left;">
NCBI_Gene:216144,ENSEMBL:ENSMUSG00000055515
</td>
<td style="text-align:left;">
MGI:2684056
</td>
<td style="text-align:left;">
protein coding gene
</td>
<td style="text-align:left;">
vomeronasal 2, receptor 81
</td>
</tr>
<tr>
<td style="text-align:left;">
10
</td>
<td style="text-align:left;">
gene
</td>
<td style="text-align:right;">
79.35652
</td>
<td style="text-align:right;">
79.39785
</td>
<td style="text-align:left;">
positive
</td>
<td style="text-align:left;">
MGI_C57BL6J_3646522
</td>
<td style="text-align:left;">
Vmn2r82
</td>
<td style="text-align:left;">
NCBI_Gene:624845,ENSEMBL:ENSMUSG00000091468
</td>
<td style="text-align:left;">
MGI:3646522
</td>
<td style="text-align:left;">
protein coding gene
</td>
<td style="text-align:left;">
vomeronasal 2, receptor 82
</td>
</tr>
<tr>
<td style="text-align:left;">
10
</td>
<td style="text-align:left;">
gene
</td>
<td style="text-align:right;">
79.41296
</td>
<td style="text-align:right;">
79.41791
</td>
<td style="text-align:left;">
negative
</td>
<td style="text-align:left;">
MGI_C57BL6J_6095697
</td>
<td style="text-align:left;">
Gm47015
</td>
<td style="text-align:left;">
ENSEMBL:ENSMUSG00000112907
</td>
<td style="text-align:left;">
MGI:6095697
</td>
<td style="text-align:left;">
lncRNA gene
</td>
<td style="text-align:left;">
predicted gene, 47015
</td>
</tr>
<tr>
<td style="text-align:left;">
10
</td>
<td style="text-align:left;">
gene
</td>
<td style="text-align:right;">
79.46873
</td>
<td style="text-align:right;">
79.49358
</td>
<td style="text-align:left;">
positive
</td>
<td style="text-align:left;">
MGI_C57BL6J_3644559
</td>
<td style="text-align:left;">
Vmn2r83
</td>
<td style="text-align:left;">
NCBI_Gene:625029,ENSEMBL:ENSMUSG00000091381
</td>
<td style="text-align:left;">
MGI:3644559
</td>
<td style="text-align:left;">
protein coding gene
</td>
<td style="text-align:left;">
vomeronasal 2, receptor 83
</td>
</tr>
<tr>
<td style="text-align:left;">
10
</td>
<td style="text-align:left;">
gene
</td>
<td style="text-align:right;">
79.52642
</td>
<td style="text-align:right;">
79.53380
</td>
<td style="text-align:left;">
negative
</td>
<td style="text-align:left;">
MGI_C57BL6J_1354945
</td>
<td style="text-align:left;">
Plpp2
</td>
<td style="text-align:left;">
NCBI_Gene:50784,ENSEMBL:ENSMUSG00000052151
</td>
<td style="text-align:left;">
MGI:1354945
</td>
<td style="text-align:left;">
protein coding gene
</td>
<td style="text-align:left;">
phospholipid phosphatase 2
</td>
</tr>
<tr>
<td style="text-align:left;">
10
</td>
<td style="text-align:left;">
gene
</td>
<td style="text-align:right;">
79.53103
</td>
<td style="text-align:right;">
79.53109
</td>
<td style="text-align:left;">
negative
</td>
<td style="text-align:left;">
MGI_C57BL6J_5531109
</td>
<td style="text-align:left;">
Mir6909
</td>
<td style="text-align:left;">
miRBase:MI0022756,NCBI_Gene:102465548,ENSEMBL:ENSMUSG00000098414
</td>
<td style="text-align:left;">
MGI:5531109
</td>
<td style="text-align:left;">
miRNA gene
</td>
<td style="text-align:left;">
microRNA 6909
</td>
</tr>
<tr>
<td style="text-align:left;">
10
</td>
<td style="text-align:left;">
gene
</td>
<td style="text-align:right;">
79.54024
</td>
<td style="text-align:right;">
79.55890
</td>
<td style="text-align:left;">
negative
</td>
<td style="text-align:left;">
MGI_C57BL6J_1917677
</td>
<td style="text-align:left;">
Mier2
</td>
<td style="text-align:left;">
NCBI_Gene:70427,ENSEMBL:ENSMUSG00000042570
</td>
<td style="text-align:left;">
MGI:1917677
</td>
<td style="text-align:left;">
protein coding gene
</td>
<td style="text-align:left;">
MIER family member 2
</td>
</tr>
<tr>
<td style="text-align:left;">
10
</td>
<td style="text-align:left;">
gene
</td>
<td style="text-align:right;">
79.57638
</td>
<td style="text-align:right;">
79.58733
</td>
<td style="text-align:left;">
negative
</td>
<td style="text-align:left;">
MGI_C57BL6J_1338756
</td>
<td style="text-align:left;">
Theg
</td>
<td style="text-align:left;">
NCBI_Gene:21830,ENSEMBL:ENSMUSG00000020317
</td>
<td style="text-align:left;">
MGI:1338756
</td>
<td style="text-align:left;">
protein coding gene
</td>
<td style="text-align:left;">
testicular haploid expressed gene
</td>
</tr>
<tr>
<td style="text-align:left;">
10
</td>
<td style="text-align:left;">
gene
</td>
<td style="text-align:right;">
79.60685
</td>
<td style="text-align:right;">
79.61402
</td>
<td style="text-align:left;">
negative
</td>
<td style="text-align:left;">
MGI_C57BL6J_2685084
</td>
<td style="text-align:left;">
C2cd4c
</td>
<td style="text-align:left;">
NCBI_Gene:237397,ENSEMBL:ENSMUSG00000045912
</td>
<td style="text-align:left;">
MGI:2685084
</td>
<td style="text-align:left;">
protein coding gene
</td>
<td style="text-align:left;">
C2 calcium-dependent domain containing 4C
</td>
</tr>
<tr>
<td style="text-align:left;">
10
</td>
<td style="text-align:left;">
gene
</td>
<td style="text-align:right;">
79.61793
</td>
<td style="text-align:right;">
79.63792
</td>
<td style="text-align:left;">
negative
</td>
<td style="text-align:left;">
MGI_C57BL6J_106180
</td>
<td style="text-align:left;">
Shc2
</td>
<td style="text-align:left;">
NCBI_Gene:216148,ENSEMBL:ENSMUSG00000020312
</td>
<td style="text-align:left;">
MGI:106180
</td>
<td style="text-align:left;">
protein coding gene
</td>
<td style="text-align:left;">
SHC (Src homology 2 domain containing) transforming protein 2
</td>
</tr>
<tr>
<td style="text-align:left;">
10
</td>
<td style="text-align:left;">
gene
</td>
<td style="text-align:right;">
79.63786
</td>
<td style="text-align:right;">
79.63952
</td>
<td style="text-align:left;">
positive
</td>
<td style="text-align:left;">
MGI_C57BL6J_6095939
</td>
<td style="text-align:left;">
Gm47163
</td>
<td style="text-align:left;">
ENSEMBL:ENSMUSG00000111977
</td>
<td style="text-align:left;">
MGI:6095939
</td>
<td style="text-align:left;">
lncRNA gene
</td>
<td style="text-align:left;">
predicted gene, 47163
</td>
</tr>
<tr>
<td style="text-align:left;">
10
</td>
<td style="text-align:left;">
gene
</td>
<td style="text-align:right;">
79.63953
</td>
<td style="text-align:right;">
79.64574
</td>
<td style="text-align:left;">
negative
</td>
<td style="text-align:left;">
MGI_C57BL6J_2686003
</td>
<td style="text-align:left;">
Odf3l2
</td>
<td style="text-align:left;">
NCBI_Gene:382384,ENSEMBL:ENSMUSG00000035963
</td>
<td style="text-align:left;">
MGI:2686003
</td>
<td style="text-align:left;">
protein coding gene
</td>
<td style="text-align:left;">
outer dense fiber of sperm tails 3-like 2
</td>
</tr>
<tr>
<td style="text-align:left;">
10
</td>
<td style="text-align:left;">
gene
</td>
<td style="text-align:right;">
79.66439
</td>
<td style="text-align:right;">
79.66854
</td>
<td style="text-align:left;">
positive
</td>
<td style="text-align:left;">
MGI_C57BL6J_103579
</td>
<td style="text-align:left;">
Madcam1
</td>
<td style="text-align:left;">
NCBI_Gene:17123,ENSEMBL:ENSMUSG00000020310
</td>
<td style="text-align:left;">
MGI:103579
</td>
<td style="text-align:left;">
protein coding gene
</td>
<td style="text-align:left;">
mucosal vascular addressin cell adhesion molecule 1
</td>
</tr>
<tr>
<td style="text-align:left;">
10
</td>
<td style="text-align:left;">
gene
</td>
<td style="text-align:right;">
79.66937
</td>
<td style="text-align:right;">
79.67613
</td>
<td style="text-align:left;">
positive
</td>
<td style="text-align:left;">
MGI_C57BL6J_106618
</td>
<td style="text-align:left;">
Tpgs1
</td>
<td style="text-align:left;">
NCBI_Gene:110012,ENSEMBL:ENSMUSG00000020308
</td>
<td style="text-align:left;">
MGI:106618
</td>
<td style="text-align:left;">
protein coding gene
</td>
<td style="text-align:left;">
tubulin polyglutamylase complex subunit 1
</td>
</tr>
<tr>
<td style="text-align:left;">
10
</td>
<td style="text-align:left;">
gene
</td>
<td style="text-align:right;">
79.68219
</td>
<td style="text-align:right;">
79.68840
</td>
<td style="text-align:left;">
positive
</td>
<td style="text-align:left;">
MGI_C57BL6J_102657
</td>
<td style="text-align:left;">
Cdc34
</td>
<td style="text-align:left;">
NCBI_Gene:216150,ENSEMBL:ENSMUSG00000020307
</td>
<td style="text-align:left;">
MGI:102657
</td>
<td style="text-align:left;">
protein coding gene
</td>
<td style="text-align:left;">
cell division cycle 34
</td>
</tr>
<tr>
<td style="text-align:left;">
10
</td>
<td style="text-align:left;">
gene
</td>
<td style="text-align:right;">
79.68902
</td>
<td style="text-align:right;">
79.69526
</td>
<td style="text-align:left;">
positive
</td>
<td style="text-align:left;">
MGI_C57BL6J_99549
</td>
<td style="text-align:left;">
Gzmm
</td>
<td style="text-align:left;">
NCBI_Gene:16904,ENSEMBL:ENSMUSG00000054206
</td>
<td style="text-align:left;">
MGI:99549
</td>
<td style="text-align:left;">
protein coding gene
</td>
<td style="text-align:left;">
granzyme M (lymphocyte met-ase 1)
</td>
</tr>
<tr>
<td style="text-align:left;">
10
</td>
<td style="text-align:left;">
gene
</td>
<td style="text-align:right;">
79.70348
</td>
<td style="text-align:right;">
79.70359
</td>
<td style="text-align:left;">
negative
</td>
<td style="text-align:left;">
MGI_C57BL6J_5455571
</td>
<td style="text-align:left;">
Gm25794
</td>
<td style="text-align:left;">
ENSEMBL:ENSMUSG00000065220
</td>
<td style="text-align:left;">
MGI:5455571
</td>
<td style="text-align:left;">
snRNA gene
</td>
<td style="text-align:left;">
predicted gene, 25794
</td>
</tr>
<tr>
<td style="text-align:left;">
10
</td>
<td style="text-align:left;">
gene
</td>
<td style="text-align:right;">
79.70436
</td>
<td style="text-align:right;">
79.71198
</td>
<td style="text-align:left;">
positive
</td>
<td style="text-align:left;">
MGI_C57BL6J_88208
</td>
<td style="text-align:left;">
Bsg
</td>
<td style="text-align:left;">
NCBI_Gene:12215,ENSEMBL:ENSMUSG00000023175
</td>
<td style="text-align:left;">
MGI:88208
</td>
<td style="text-align:left;">
protein coding gene
</td>
<td style="text-align:left;">
basigin
</td>
</tr>
<tr>
<td style="text-align:left;">
10
</td>
<td style="text-align:left;">
gene
</td>
<td style="text-align:right;">
79.71619
</td>
<td style="text-align:right;">
79.71691
</td>
<td style="text-align:left;">
negative
</td>
<td style="text-align:left;">
MGI_C57BL6J_5589639
</td>
<td style="text-align:left;">
Gm30480
</td>
<td style="text-align:left;">
NCBI_Gene:102632396
</td>
<td style="text-align:left;">
MGI:5589639
</td>
<td style="text-align:left;">
lncRNA gene
</td>
<td style="text-align:left;">
predicted gene, 30480
</td>
</tr>
<tr>
<td style="text-align:left;">
10
</td>
<td style="text-align:left;">
gene
</td>
<td style="text-align:right;">
79.71663
</td>
<td style="text-align:right;">
79.73611
</td>
<td style="text-align:left;">
positive
</td>
<td style="text-align:left;">
MGI_C57BL6J_1298210
</td>
<td style="text-align:left;">
Hcn2
</td>
<td style="text-align:left;">
NCBI_Gene:15166,ENSEMBL:ENSMUSG00000020331
</td>
<td style="text-align:left;">
MGI:1298210
</td>
<td style="text-align:left;">
protein coding gene
</td>
<td style="text-align:left;">
hyperpolarization-activated, cyclic nucleotide-gated K+ 2
</td>
</tr>
<tr>
<td style="text-align:left;">
10
</td>
<td style="text-align:left;">
gene
</td>
<td style="text-align:right;">
79.73612
</td>
<td style="text-align:right;">
79.74659
</td>
<td style="text-align:left;">
negative
</td>
<td style="text-align:left;">
MGI_C57BL6J_1915843
</td>
<td style="text-align:left;">
Polrmt
</td>
<td style="text-align:left;">
NCBI_Gene:216151,ENSEMBL:ENSMUSG00000020329
</td>
<td style="text-align:left;">
MGI:1915843
</td>
<td style="text-align:left;">
protein coding gene
</td>
<td style="text-align:left;">
polymerase (RNA) mitochondrial (DNA directed)
</td>
</tr>
<tr>
<td style="text-align:left;">
10
</td>
<td style="text-align:left;">
gene
</td>
<td style="text-align:right;">
79.75458
</td>
<td style="text-align:right;">
79.75860
</td>
<td style="text-align:left;">
positive
</td>
<td style="text-align:left;">
MGI_C57BL6J_1914362
</td>
<td style="text-align:left;">
Fgf22
</td>
<td style="text-align:left;">
NCBI_Gene:67112,ENSEMBL:ENSMUSG00000020327
</td>
<td style="text-align:left;">
MGI:1914362
</td>
<td style="text-align:left;">
protein coding gene
</td>
<td style="text-align:left;">
fibroblast growth factor 22
</td>
</tr>
<tr>
<td style="text-align:left;">
10
</td>
<td style="text-align:left;">
gene
</td>
<td style="text-align:right;">
79.75852
</td>
<td style="text-align:right;">
79.76701
</td>
<td style="text-align:left;">
negative
</td>
<td style="text-align:left;">
MGI_C57BL6J_1917544
</td>
<td style="text-align:left;">
Rnf126
</td>
<td style="text-align:left;">
NCBI_Gene:70294,ENSEMBL:ENSMUSG00000035890
</td>
<td style="text-align:left;">
MGI:1917544
</td>
<td style="text-align:left;">
protein coding gene
</td>
<td style="text-align:left;">
ring finger protein 126
</td>
</tr>
<tr>
<td style="text-align:left;">
10
</td>
<td style="text-align:left;">
gene
</td>
<td style="text-align:right;">
79.77727
</td>
<td style="text-align:right;">
79.78263
</td>
<td style="text-align:left;">
positive
</td>
<td style="text-align:left;">
MGI_C57BL6J_1890391
</td>
<td style="text-align:left;">
Fstl3
</td>
<td style="text-align:left;">
NCBI_Gene:83554,ENSEMBL:ENSMUSG00000020325
</td>
<td style="text-align:left;">
MGI:1890391
</td>
<td style="text-align:left;">
protein coding gene
</td>
<td style="text-align:left;">
follistatin-like 3
</td>
</tr>
<tr>
<td style="text-align:left;">
10
</td>
<td style="text-align:left;">
gene
</td>
<td style="text-align:right;">
79.78147
</td>
<td style="text-align:right;">
79.79474
</td>
<td style="text-align:left;">
negative
</td>
<td style="text-align:left;">
MGI_C57BL6J_1920356
</td>
<td style="text-align:left;">
Prss57
</td>
<td style="text-align:left;">
NCBI_Gene:73106,ENSEMBL:ENSMUSG00000020323
</td>
<td style="text-align:left;">
MGI:1920356
</td>
<td style="text-align:left;">
protein coding gene
</td>
<td style="text-align:left;">
protease, serine 57
</td>
</tr>
<tr>
<td style="text-align:left;">
10
</td>
<td style="text-align:left;">
gene
</td>
<td style="text-align:right;">
79.79357
</td>
<td style="text-align:right;">
79.82091
</td>
<td style="text-align:left;">
positive
</td>
<td style="text-align:left;">
MGI_C57BL6J_1261814
</td>
<td style="text-align:left;">
Palm
</td>
<td style="text-align:left;">
NCBI_Gene:18483,ENSEMBL:ENSMUSG00000035863
</td>
<td style="text-align:left;">
MGI:1261814
</td>
<td style="text-align:left;">
protein coding gene
</td>
<td style="text-align:left;">
paralemmin
</td>
</tr>
<tr>
<td style="text-align:left;">
10
</td>
<td style="text-align:left;">
gene
</td>
<td style="text-align:right;">
79.79378
</td>
<td style="text-align:right;">
79.79466
</td>
<td style="text-align:left;">
negative
</td>
<td style="text-align:left;">
MGI_C57BL6J_6098020
</td>
<td style="text-align:left;">
Gm48495
</td>
<td style="text-align:left;">
ENSEMBL:ENSMUSG00000112828
</td>
<td style="text-align:left;">
MGI:6098020
</td>
<td style="text-align:left;">
lncRNA gene
</td>
<td style="text-align:left;">
predicted gene, 48495
</td>
</tr>
<tr>
<td style="text-align:left;">
10
</td>
<td style="text-align:left;">
gene
</td>
<td style="text-align:right;">
79.80068
</td>
<td style="text-align:right;">
79.80702
</td>
<td style="text-align:left;">
negative
</td>
<td style="text-align:left;">
MGI_C57BL6J_5623598
</td>
<td style="text-align:left;">
Gm40713
</td>
<td style="text-align:left;">
NCBI_Gene:105245229
</td>
<td style="text-align:left;">
MGI:5623598
</td>
<td style="text-align:left;">
lncRNA gene
</td>
<td style="text-align:left;">
predicted gene, 40713
</td>
</tr>
<tr>
<td style="text-align:left;">
10
</td>
<td style="text-align:left;">
gene
</td>
<td style="text-align:right;">
79.82099
</td>
<td style="text-align:right;">
79.83049
</td>
<td style="text-align:left;">
positive
</td>
<td style="text-align:left;">
MGI_C57BL6J_1926156
</td>
<td style="text-align:left;">
Misp
</td>
<td style="text-align:left;">
NCBI_Gene:78906,ENSEMBL:ENSMUSG00000035852
</td>
<td style="text-align:left;">
MGI:1926156
</td>
<td style="text-align:left;">
protein coding gene
</td>
<td style="text-align:left;">
mitotic spindle positioning
</td>
</tr>
<tr>
<td style="text-align:left;">
10
</td>
<td style="text-align:left;">
gene
</td>
<td style="text-align:right;">
79.85138
</td>
<td style="text-align:right;">
79.85497
</td>
<td style="text-align:left;">
negative
</td>
<td style="text-align:left;">
MGI_C57BL6J_3041202
</td>
<td style="text-align:left;">
E130317F20Rik
</td>
<td style="text-align:left;">
NCBI_Gene:414101,ENSEMBL:ENSMUSG00000091994
</td>
<td style="text-align:left;">
MGI:3041202
</td>
<td style="text-align:left;">
lncRNA gene
</td>
<td style="text-align:left;">
RIKEN cDNA E130317F20 gene
</td>
</tr>
<tr>
<td style="text-align:left;">
10
</td>
<td style="text-align:left;">
gene
</td>
<td style="text-align:right;">
79.85443
</td>
<td style="text-align:right;">
79.86477
</td>
<td style="text-align:left;">
positive
</td>
<td style="text-align:left;">
MGI_C57BL6J_97791
</td>
<td style="text-align:left;">
Ptbp1
</td>
<td style="text-align:left;">
NCBI_Gene:19205,ENSEMBL:ENSMUSG00000006498
</td>
<td style="text-align:left;">
MGI:97791
</td>
<td style="text-align:left;">
protein coding gene
</td>
<td style="text-align:left;">
polypyrimidine tract binding protein 1
</td>
</tr>
<tr>
<td style="text-align:left;">
10
</td>
<td style="text-align:left;">
gene
</td>
<td style="text-align:right;">
79.86047
</td>
<td style="text-align:right;">
79.87463
</td>
<td style="text-align:left;">
negative
</td>
<td style="text-align:left;">
MGI_C57BL6J_2388640
</td>
<td style="text-align:left;">
Plppr3
</td>
<td style="text-align:left;">
NCBI_Gene:216152,ENSEMBL:ENSMUSG00000035835
</td>
<td style="text-align:left;">
MGI:2388640
</td>
<td style="text-align:left;">
protein coding gene
</td>
<td style="text-align:left;">
phospholipid phosphatase related 3
</td>
</tr>
<tr>
<td style="text-align:left;">
10
</td>
<td style="text-align:left;">
gene
</td>
<td style="text-align:right;">
79.86259
</td>
<td style="text-align:right;">
79.86265
</td>
<td style="text-align:left;">
positive
</td>
<td style="text-align:left;">
MGI_C57BL6J_5531145
</td>
<td style="text-align:left;">
Mir6910
</td>
<td style="text-align:left;">
miRBase:MI0022757,NCBI_Gene:102465549,ENSEMBL:ENSMUSG00000098562
</td>
<td style="text-align:left;">
MGI:5531145
</td>
<td style="text-align:left;">
miRNA gene
</td>
<td style="text-align:left;">
microRNA 6910
</td>
</tr>
<tr>
<td style="text-align:left;">
10
</td>
<td style="text-align:left;">
gene
</td>
<td style="text-align:right;">
79.86394
</td>
<td style="text-align:right;">
79.86443
</td>
<td style="text-align:left;">
negative
</td>
<td style="text-align:left;">
MGI_C57BL6J_4937961
</td>
<td style="text-align:left;">
Gm17134
</td>
<td style="text-align:left;">
ENSEMBL:ENSMUSG00000090860
</td>
<td style="text-align:left;">
MGI:4937961
</td>
<td style="text-align:left;">
lncRNA gene
</td>
<td style="text-align:left;">
predicted gene 17134
</td>
</tr>
<tr>
<td style="text-align:left;">
10
</td>
<td style="text-align:left;">
gene
</td>
<td style="text-align:right;">
79.87448
</td>
<td style="text-align:right;">
79.88317
</td>
<td style="text-align:left;">
positive
</td>
<td style="text-align:left;">
MGI_C57BL6J_893580
</td>
<td style="text-align:left;">
Prtn3
</td>
<td style="text-align:left;">
NCBI_Gene:19152,ENSEMBL:ENSMUSG00000057729
</td>
<td style="text-align:left;">
MGI:893580
</td>
<td style="text-align:left;">
protein coding gene
</td>
<td style="text-align:left;">
proteinase 3
</td>
</tr>
<tr>
<td style="text-align:left;">
10
</td>
<td style="text-align:left;">
gene
</td>
<td style="text-align:right;">
79.88625
</td>
<td style="text-align:right;">
79.88822
</td>
<td style="text-align:left;">
positive
</td>
<td style="text-align:left;">
MGI_C57BL6J_2679229
</td>
<td style="text-align:left;">
Elane
</td>
<td style="text-align:left;">
NCBI_Gene:50701,ENSEMBL:ENSMUSG00000020125
</td>
<td style="text-align:left;">
MGI:2679229
</td>
<td style="text-align:left;">
protein coding gene
</td>
<td style="text-align:left;">
elastase, neutrophil expressed
</td>
</tr>
<tr>
<td style="text-align:left;">
10
</td>
<td style="text-align:left;">
pseudogene
</td>
<td style="text-align:right;">
79.88866
</td>
<td style="text-align:right;">
79.88901
</td>
<td style="text-align:left;">
positive
</td>
<td style="text-align:left;">
MGI_C57BL6J_5011995
</td>
<td style="text-align:left;">
Gm19810
</td>
<td style="text-align:left;">
NCBI_Gene:102632476,ENSEMBL:ENSMUSG00000111897
</td>
<td style="text-align:left;">
MGI:5011995
</td>
<td style="text-align:left;">
pseudogene
</td>
<td style="text-align:left;">
predicted gene, 19810
</td>
</tr>
<tr>
<td style="text-align:left;">
10
</td>
<td style="text-align:left;">
gene
</td>
<td style="text-align:right;">
79.89085
</td>
<td style="text-align:right;">
79.89266
</td>
<td style="text-align:left;">
positive
</td>
<td style="text-align:left;">
MGI_C57BL6J_87931
</td>
<td style="text-align:left;">
Cfd
</td>
<td style="text-align:left;">
NCBI_Gene:11537,ENSEMBL:ENSMUSG00000061780
</td>
<td style="text-align:left;">
MGI:87931
</td>
<td style="text-align:left;">
protein coding gene
</td>
<td style="text-align:left;">
complement factor D (adipsin)
</td>
</tr>
<tr>
<td style="text-align:left;">
10
</td>
<td style="text-align:left;">
gene
</td>
<td style="text-align:right;">
79.89471
</td>
<td style="text-align:right;">
79.90894
</td>
<td style="text-align:left;">
negative
</td>
<td style="text-align:left;">
MGI_C57BL6J_2158394
</td>
<td style="text-align:left;">
Med16
</td>
<td style="text-align:left;">
NCBI_Gene:216154,ENSEMBL:ENSMUSG00000013833
</td>
<td style="text-align:left;">
MGI:2158394
</td>
<td style="text-align:left;">
protein coding gene
</td>
<td style="text-align:left;">
mediator complex subunit 16
</td>
</tr>
<tr>
<td style="text-align:left;">
10
</td>
<td style="text-align:left;">
gene
</td>
<td style="text-align:right;">
79.90919
</td>
<td style="text-align:right;">
79.90930
</td>
<td style="text-align:left;">
positive
</td>
<td style="text-align:left;">
MGI_C57BL6J_5453796
</td>
<td style="text-align:left;">
Gm24019
</td>
<td style="text-align:left;">
ENSEMBL:ENSMUSG00000095023
</td>
<td style="text-align:left;">
MGI:5453796
</td>
<td style="text-align:left;">
snRNA gene
</td>
<td style="text-align:left;">
predicted gene, 24019
</td>
</tr>
<tr>
<td style="text-align:left;">
10
</td>
<td style="text-align:left;">
gene
</td>
<td style="text-align:right;">
79.91005
</td>
<td style="text-align:right;">
79.92121
</td>
<td style="text-align:left;">
negative
</td>
<td style="text-align:left;">
MGI_C57BL6J_1924814
</td>
<td style="text-align:left;">
R3hdm4
</td>
<td style="text-align:left;">
NCBI_Gene:109284,ENSEMBL:ENSMUSG00000035781
</td>
<td style="text-align:left;">
MGI:1924814
</td>
<td style="text-align:left;">
protein coding gene
</td>
<td style="text-align:left;">
R3H domain containing 4
</td>
</tr>
<tr>
<td style="text-align:left;">
10
</td>
<td style="text-align:left;">
gene
</td>
<td style="text-align:right;">
79.91086
</td>
<td style="text-align:right;">
79.91114
</td>
<td style="text-align:left;">
positive
</td>
<td style="text-align:left;">
MGI_C57BL6J_5477096
</td>
<td style="text-align:left;">
Gm26602
</td>
<td style="text-align:left;">
ENSEMBL:ENSMUSG00000097854
</td>
<td style="text-align:left;">
MGI:5477096
</td>
<td style="text-align:left;">
protein coding gene
</td>
<td style="text-align:left;">
predicted gene, 26602
</td>
</tr>
<tr>
<td style="text-align:left;">
10
</td>
<td style="text-align:left;">
gene
</td>
<td style="text-align:right;">
79.91696
</td>
<td style="text-align:right;">
79.92409
</td>
<td style="text-align:left;">
positive
</td>
<td style="text-align:left;">
MGI_C57BL6J_2148793
</td>
<td style="text-align:left;">
Kiss1r
</td>
<td style="text-align:left;">
NCBI_Gene:114229,ENSEMBL:ENSMUSG00000035773
</td>
<td style="text-align:left;">
MGI:2148793
</td>
<td style="text-align:left;">
protein coding gene
</td>
<td style="text-align:left;">
KISS1 receptor
</td>
</tr>
<tr>
<td style="text-align:left;">
10
</td>
<td style="text-align:left;">
gene
</td>
<td style="text-align:right;">
79.92469
</td>
<td style="text-align:right;">
79.92693
</td>
<td style="text-align:left;">
negative
</td>
<td style="text-align:left;">
MGI_C57BL6J_5623600
</td>
<td style="text-align:left;">
Gm40715
</td>
<td style="text-align:left;">
NCBI_Gene:105245231
</td>
<td style="text-align:left;">
MGI:5623600
</td>
<td style="text-align:left;">
lncRNA gene
</td>
<td style="text-align:left;">
predicted gene, 40715
</td>
</tr>
<tr>
<td style="text-align:left;">
10
</td>
<td style="text-align:left;">
gene
</td>
<td style="text-align:right;">
79.92704
</td>
<td style="text-align:right;">
79.95502
</td>
<td style="text-align:left;">
positive
</td>
<td style="text-align:left;">
MGI_C57BL6J_1328360
</td>
<td style="text-align:left;">
Arid3a
</td>
<td style="text-align:left;">
NCBI_Gene:13496,ENSEMBL:ENSMUSG00000019564
</td>
<td style="text-align:left;">
MGI:1328360
</td>
<td style="text-align:left;">
protein coding gene
</td>
<td style="text-align:left;">
AT rich interactive domain 3A (BRIGHT-like)
</td>
</tr>
<tr>
<td style="text-align:left;">
10
</td>
<td style="text-align:left;">
gene
</td>
<td style="text-align:right;">
79.95999
</td>
<td style="text-align:right;">
79.97020
</td>
<td style="text-align:left;">
positive
</td>
<td style="text-align:left;">
MGI_C57BL6J_2158400
</td>
<td style="text-align:left;">
Wdr18
</td>
<td style="text-align:left;">
NCBI_Gene:216156,ENSEMBL:ENSMUSG00000035754
</td>
<td style="text-align:left;">
MGI:2158400
</td>
<td style="text-align:left;">
protein coding gene
</td>
<td style="text-align:left;">
WD repeat domain 18
</td>
</tr>
<tr>
<td style="text-align:left;">
10
</td>
<td style="text-align:left;">
gene
</td>
<td style="text-align:right;">
79.97040
</td>
<td style="text-align:right;">
79.97719
</td>
<td style="text-align:left;">
positive
</td>
<td style="text-align:left;">
MGI_C57BL6J_2150393
</td>
<td style="text-align:left;">
Grin3b
</td>
<td style="text-align:left;">
NCBI_Gene:170483,ENSEMBL:ENSMUSG00000035745
</td>
<td style="text-align:left;">
MGI:2150393
</td>
<td style="text-align:left;">
protein coding gene
</td>
<td style="text-align:left;">
glutamate receptor, ionotropic, NMDA3B
</td>
</tr>
<tr>
<td style="text-align:left;">
10
</td>
<td style="text-align:left;">
gene
</td>
<td style="text-align:right;">
79.97495
</td>
<td style="text-align:right;">
79.98433
</td>
<td style="text-align:left;">
negative
</td>
<td style="text-align:left;">
MGI_C57BL6J_2177957
</td>
<td style="text-align:left;">
Tmem259
</td>
<td style="text-align:left;">
NCBI_Gene:216157,ENSEMBL:ENSMUSG00000013858
</td>
<td style="text-align:left;">
MGI:2177957
</td>
<td style="text-align:left;">
protein coding gene
</td>
<td style="text-align:left;">
transmembrane protein 259
</td>
</tr>
<tr>
<td style="text-align:left;">
10
</td>
<td style="text-align:left;">
gene
</td>
<td style="text-align:right;">
79.98551
</td>
<td style="text-align:right;">
79.98561
</td>
<td style="text-align:left;">
positive
</td>
<td style="text-align:left;">
MGI_C57BL6J_5456020
</td>
<td style="text-align:left;">
Gm26243
</td>
<td style="text-align:left;">
ENSEMBL:ENSMUSG00000095615
</td>
<td style="text-align:left;">
MGI:5456020
</td>
<td style="text-align:left;">
snRNA gene
</td>
<td style="text-align:left;">
predicted gene, 26243
</td>
</tr>
<tr>
<td style="text-align:left;">
10
</td>
<td style="text-align:left;">
gene
</td>
<td style="text-align:right;">
79.98858
</td>
<td style="text-align:right;">
79.99606
</td>
<td style="text-align:left;">
positive
</td>
<td style="text-align:left;">
MGI_C57BL6J_105093
</td>
<td style="text-align:left;">
Cnn2
</td>
<td style="text-align:left;">
NCBI_Gene:12798,ENSEMBL:ENSMUSG00000004665
</td>
<td style="text-align:left;">
MGI:105093
</td>
<td style="text-align:left;">
protein coding gene
</td>
<td style="text-align:left;">
calponin 2
</td>
</tr>
<tr>
<td style="text-align:left;">
10
</td>
<td style="text-align:left;">
gene
</td>
<td style="text-align:right;">
79.99645
</td>
<td style="text-align:right;">
80.01557
</td>
<td style="text-align:left;">
positive
</td>
<td style="text-align:left;">
MGI_C57BL6J_1351646
</td>
<td style="text-align:left;">
Abca7
</td>
<td style="text-align:left;">
NCBI_Gene:27403,ENSEMBL:ENSMUSG00000035722
</td>
<td style="text-align:left;">
MGI:1351646
</td>
<td style="text-align:left;">
protein coding gene
</td>
<td style="text-align:left;">
ATP-binding cassette, sub-family A (ABC1), member 7
</td>
</tr>
<tr>
<td style="text-align:left;">
10
</td>
<td style="text-align:left;">
gene
</td>
<td style="text-align:right;">
80.01665
</td>
<td style="text-align:right;">
80.03147
</td>
<td style="text-align:left;">
positive
</td>
<td style="text-align:left;">
MGI_C57BL6J_1917969
</td>
<td style="text-align:left;">
Arhgap45
</td>
<td style="text-align:left;">
NCBI_Gene:70719,ENSEMBL:ENSMUSG00000035697
</td>
<td style="text-align:left;">
MGI:1917969
</td>
<td style="text-align:left;">
protein coding gene
</td>
<td style="text-align:left;">
Rho GTPase activating protein 45
</td>
</tr>
<tr>
<td style="text-align:left;">
10
</td>
<td style="text-align:left;">
gene
</td>
<td style="text-align:right;">
80.03595
</td>
<td style="text-align:right;">
80.03979
</td>
<td style="text-align:left;">
negative
</td>
<td style="text-align:left;">
MGI_C57BL6J_1913670
</td>
<td style="text-align:left;">
Polr2e
</td>
<td style="text-align:left;">
NCBI_Gene:66420,ENSEMBL:ENSMUSG00000004667
</td>
<td style="text-align:left;">
MGI:1913670
</td>
<td style="text-align:left;">
protein coding gene
</td>
<td style="text-align:left;">
polymerase (RNA) II (DNA directed) polypeptide E
</td>
</tr>
<tr>
<td style="text-align:left;">
10
</td>
<td style="text-align:left;">
gene
</td>
<td style="text-align:right;">
80.04717
</td>
<td style="text-align:right;">
80.05644
</td>
<td style="text-align:left;">
positive
</td>
<td style="text-align:left;">
MGI_C57BL6J_104767
</td>
<td style="text-align:left;">
Gpx4
</td>
<td style="text-align:left;">
NCBI_Gene:625249,ENSEMBL:ENSMUSG00000075706
</td>
<td style="text-align:left;">
MGI:104767
</td>
<td style="text-align:left;">
protein coding gene
</td>
<td style="text-align:left;">
glutathione peroxidase 4
</td>
</tr>
<tr>
<td style="text-align:left;">
10
</td>
<td style="text-align:left;">
gene
</td>
<td style="text-align:right;">
80.05699
</td>
<td style="text-align:right;">
80.10611
</td>
<td style="text-align:left;">
negative
</td>
<td style="text-align:left;">
MGI_C57BL6J_2448490
</td>
<td style="text-align:left;">
Sbno2
</td>
<td style="text-align:left;">
NCBI_Gene:216161,ENSEMBL:ENSMUSG00000035673
</td>
<td style="text-align:left;">
MGI:2448490
</td>
<td style="text-align:left;">
protein coding gene
</td>
<td style="text-align:left;">
strawberry notch 2
</td>
</tr>
<tr>
<td style="text-align:left;">
10
</td>
<td style="text-align:left;">
gene
</td>
<td style="text-align:right;">
80.07389
</td>
<td style="text-align:right;">
80.08172
</td>
<td style="text-align:left;">
positive
</td>
<td style="text-align:left;">
MGI_C57BL6J_5825874
</td>
<td style="text-align:left;">
Gm46237
</td>
<td style="text-align:left;">
NCBI_Gene:108167824
</td>
<td style="text-align:left;">
MGI:5825874
</td>
<td style="text-align:left;">
lncRNA gene
</td>
<td style="text-align:left;">
predicted gene, 46237
</td>
</tr>
<tr>
<td style="text-align:left;">
10
</td>
<td style="text-align:left;">
gene
</td>
<td style="text-align:right;">
80.11569
</td>
<td style="text-align:right;">
80.13068
</td>
<td style="text-align:left;">
positive
</td>
<td style="text-align:left;">
MGI_C57BL6J_1341870
</td>
<td style="text-align:left;">
Stk11
</td>
<td style="text-align:left;">
NCBI_Gene:20869,ENSEMBL:ENSMUSG00000003068
</td>
<td style="text-align:left;">
MGI:1341870
</td>
<td style="text-align:left;">
protein coding gene
</td>
<td style="text-align:left;">
serine/threonine kinase 11
</td>
</tr>
<tr>
<td style="text-align:left;">
10
</td>
<td style="text-align:left;">
gene
</td>
<td style="text-align:right;">
80.13043
</td>
<td style="text-align:right;">
80.14083
</td>
<td style="text-align:left;">
negative
</td>
<td style="text-align:left;">
MGI_C57BL6J_1354170
</td>
<td style="text-align:left;">
Cbarp
</td>
<td style="text-align:left;">
NCBI_Gene:100503659,ENSEMBL:ENSMUSG00000035640
</td>
<td style="text-align:left;">
MGI:1354170
</td>
<td style="text-align:left;">
protein coding gene
</td>
<td style="text-align:left;">
calcium channel, voltage-dependent, beta subunit associated regulatory protein
</td>
</tr>
<tr>
<td style="text-align:left;">
10
</td>
<td style="text-align:left;">
gene
</td>
<td style="text-align:right;">
80.13863
</td>
<td style="text-align:right;">
80.14582
</td>
<td style="text-align:left;">
positive
</td>
<td style="text-align:left;">
MGI_C57BL6J_1913293
</td>
<td style="text-align:left;">
Atp5d
</td>
<td style="text-align:left;">
NCBI_Gene:66043,ENSEMBL:ENSMUSG00000003072
</td>
<td style="text-align:left;">
MGI:1913293
</td>
<td style="text-align:left;">
protein coding gene
</td>
<td style="text-align:left;">
ATP synthase, H+ transporting, mitochondrial F1 complex, delta subunit
</td>
</tr>
<tr>
<td style="text-align:left;">
10
</td>
<td style="text-align:left;">
gene
</td>
<td style="text-align:right;">
80.14824
</td>
<td style="text-align:right;">
80.15837
</td>
<td style="text-align:left;">
positive
</td>
<td style="text-align:left;">
MGI_C57BL6J_1890222
</td>
<td style="text-align:left;">
Midn
</td>
<td style="text-align:left;">
NCBI_Gene:59090,ENSEMBL:ENSMUSG00000035621
</td>
<td style="text-align:left;">
MGI:1890222
</td>
<td style="text-align:left;">
protein coding gene
</td>
<td style="text-align:left;">
midnolin
</td>
</tr>
<tr>
<td style="text-align:left;">
10
</td>
<td style="text-align:left;">
gene
</td>
<td style="text-align:right;">
80.16599
</td>
<td style="text-align:right;">
80.17279
</td>
<td style="text-align:left;">
positive
</td>
<td style="text-align:left;">
MGI_C57BL6J_893588
</td>
<td style="text-align:left;">
Cirbp
</td>
<td style="text-align:left;">
NCBI_Gene:12696,ENSEMBL:ENSMUSG00000045193
</td>
<td style="text-align:left;">
MGI:893588
</td>
<td style="text-align:left;">
protein coding gene
</td>
<td style="text-align:left;">
cold inducible RNA binding protein
</td>
</tr>
<tr>
<td style="text-align:left;">
10
</td>
<td style="text-align:left;">
gene
</td>
<td style="text-align:right;">
80.17294
</td>
<td style="text-align:right;">
80.17515
</td>
<td style="text-align:left;">
positive
</td>
<td style="text-align:left;">
MGI_C57BL6J_1917020
</td>
<td style="text-align:left;">
1600002K03Rik
</td>
<td style="text-align:left;">
NCBI_Gene:69770,ENSEMBL:ENSMUSG00000035595
</td>
<td style="text-align:left;">
MGI:1917020
</td>
<td style="text-align:left;">
protein coding gene
</td>
<td style="text-align:left;">
RIKEN cDNA 1600002K03 gene
</td>
</tr>
<tr>
<td style="text-align:left;">
10
</td>
<td style="text-align:left;">
gene
</td>
<td style="text-align:right;">
80.17948
</td>
<td style="text-align:right;">
80.19001
</td>
<td style="text-align:left;">
positive
</td>
<td style="text-align:left;">
MGI_C57BL6J_102707
</td>
<td style="text-align:left;">
Efna2
</td>
<td style="text-align:left;">
NCBI_Gene:13637,ENSEMBL:ENSMUSG00000003070
</td>
<td style="text-align:left;">
MGI:102707
</td>
<td style="text-align:left;">
protein coding gene
</td>
<td style="text-align:left;">
ephrin A2
</td>
</tr>
<tr>
<td style="text-align:left;">
10
</td>
<td style="text-align:left;">
gene
</td>
<td style="text-align:right;">
80.22643
</td>
<td style="text-align:right;">
80.24514
</td>
<td style="text-align:left;">
positive
</td>
<td style="text-align:left;">
MGI_C57BL6J_1915364
</td>
<td style="text-align:left;">
Pwwp3a
</td>
<td style="text-align:left;">
NCBI_Gene:68114,ENSEMBL:ENSMUSG00000020156
</td>
<td style="text-align:left;">
MGI:1915364
</td>
<td style="text-align:left;">
protein coding gene
</td>
<td style="text-align:left;">
PWWP domain containing 3A, DNA repair factor
</td>
</tr>
<tr>
<td style="text-align:left;">
10
</td>
<td style="text-align:left;">
gene
</td>
<td style="text-align:right;">
80.24896
</td>
<td style="text-align:right;">
80.24904
</td>
<td style="text-align:left;">
negative
</td>
<td style="text-align:left;">
MGI_C57BL6J_4413998
</td>
<td style="text-align:left;">
n-TFgaa3
</td>
<td style="text-align:left;">
NCBI_Gene:102467582
</td>
<td style="text-align:left;">
MGI:4413998
</td>
<td style="text-align:left;">
tRNA gene
</td>
<td style="text-align:left;">
nuclear encoded tRNA pheylalanine 3 (anticodon GAA)
</td>
</tr>
<tr>
<td style="text-align:left;">
10
</td>
<td style="text-align:left;">
gene
</td>
<td style="text-align:right;">
80.24912
</td>
<td style="text-align:right;">
80.25679
</td>
<td style="text-align:left;">
positive
</td>
<td style="text-align:left;">
MGI_C57BL6J_1922656
</td>
<td style="text-align:left;">
Ndufs7
</td>
<td style="text-align:left;">
NCBI_Gene:75406,ENSEMBL:ENSMUSG00000020153
</td>
<td style="text-align:left;">
MGI:1922656
</td>
<td style="text-align:left;">
protein coding gene
</td>
<td style="text-align:left;">
NADH:ubiquinone oxidoreductase core subunit S7
</td>
</tr>
<tr>
<td style="text-align:left;">
10
</td>
<td style="text-align:left;">
gene
</td>
<td style="text-align:right;">
80.24916
</td>
<td style="text-align:right;">
80.24923
</td>
<td style="text-align:left;">
positive
</td>
<td style="text-align:left;">
MGI_C57BL6J_4413744
</td>
<td style="text-align:left;">
n-TNgtt7
</td>
<td style="text-align:left;">
NCBI_Gene:102467422
</td>
<td style="text-align:left;">
MGI:4413744
</td>
<td style="text-align:left;">
tRNA gene
</td>
<td style="text-align:left;">
nuclear encoded tRNA asparagine 7 (anticodon GTT)
</td>
</tr>
<tr>
<td style="text-align:left;">
10
</td>
<td style="text-align:left;">
gene
</td>
<td style="text-align:right;">
80.25815
</td>
<td style="text-align:right;">
80.26103
</td>
<td style="text-align:left;">
negative
</td>
<td style="text-align:left;">
MGI_C57BL6J_1098221
</td>
<td style="text-align:left;">
Gamt
</td>
<td style="text-align:left;">
NCBI_Gene:14431,ENSEMBL:ENSMUSG00000020150
</td>
<td style="text-align:left;">
MGI:1098221
</td>
<td style="text-align:left;">
protein coding gene
</td>
<td style="text-align:left;">
guanidinoacetate methyltransferase
</td>
</tr>
<tr>
<td style="text-align:left;">
10
</td>
<td style="text-align:left;">
gene
</td>
<td style="text-align:right;">
80.26147
</td>
<td style="text-align:right;">
80.28841
</td>
<td style="text-align:left;">
positive
</td>
<td style="text-align:left;">
MGI_C57BL6J_1917498
</td>
<td style="text-align:left;">
Dazap1
</td>
<td style="text-align:left;">
NCBI_Gene:70248,ENSEMBL:ENSMUSG00000069565
</td>
<td style="text-align:left;">
MGI:1917498
</td>
<td style="text-align:left;">
protein coding gene
</td>
<td style="text-align:left;">
DAZ associated protein 1
</td>
</tr>
<tr>
<td style="text-align:left;">
10
</td>
<td style="text-align:left;">
gene
</td>
<td style="text-align:right;">
80.27816
</td>
<td style="text-align:right;">
80.27823
</td>
<td style="text-align:left;">
positive
</td>
<td style="text-align:left;">
MGI_C57BL6J_5531204
</td>
<td style="text-align:left;">
Mir6911
</td>
<td style="text-align:left;">
miRBase:MI0022758,NCBI_Gene:102466207,ENSEMBL:ENSMUSG00000098672
</td>
<td style="text-align:left;">
MGI:5531204
</td>
<td style="text-align:left;">
miRNA gene
</td>
<td style="text-align:left;">
microRNA 6911
</td>
</tr>
<tr>
<td style="text-align:left;">
10
</td>
<td style="text-align:left;">
gene
</td>
<td style="text-align:right;">
80.28244
</td>
<td style="text-align:right;">
80.28301
</td>
<td style="text-align:left;">
negative
</td>
<td style="text-align:left;">
MGI_C57BL6J_3705268
</td>
<td style="text-align:left;">
Gm15122
</td>
<td style="text-align:left;">
ENSEMBL:ENSMUSG00000087672
</td>
<td style="text-align:left;">
MGI:3705268
</td>
<td style="text-align:left;">
lncRNA gene
</td>
<td style="text-align:left;">
predicted gene 15122
</td>
</tr>
<tr>
<td style="text-align:left;">
10
</td>
<td style="text-align:left;">
gene
</td>
<td style="text-align:right;">
80.29243
</td>
<td style="text-align:right;">
80.29411
</td>
<td style="text-align:left;">
positive
</td>
<td style="text-align:left;">
MGI_C57BL6J_98117
</td>
<td style="text-align:left;">
Rps15
</td>
<td style="text-align:left;">
NCBI_Gene:20054,ENSEMBL:ENSMUSG00000063457
</td>
<td style="text-align:left;">
MGI:98117
</td>
<td style="text-align:left;">
protein coding gene
</td>
<td style="text-align:left;">
ribosomal protein S15
</td>
</tr>
<tr>
<td style="text-align:left;">
10
</td>
<td style="text-align:left;">
gene
</td>
<td style="text-align:right;">
80.29420
</td>
<td style="text-align:right;">
80.29640
</td>
<td style="text-align:left;">
negative
</td>
<td style="text-align:left;">
MGI_C57BL6J_5589841
</td>
<td style="text-align:left;">
Gm30682
</td>
<td style="text-align:left;">
NCBI_Gene:102632666
</td>
<td style="text-align:left;">
MGI:5589841
</td>
<td style="text-align:left;">
lncRNA gene
</td>
<td style="text-align:left;">
predicted gene, 30682
</td>
</tr>
<tr>
<td style="text-align:left;">
10
</td>
<td style="text-align:left;">
gene
</td>
<td style="text-align:right;">
80.29596
</td>
<td style="text-align:right;">
80.31826
</td>
<td style="text-align:left;">
positive
</td>
<td style="text-align:left;">
MGI_C57BL6J_1346052
</td>
<td style="text-align:left;">
Apc2
</td>
<td style="text-align:left;">
NCBI_Gene:23805,ENSEMBL:ENSMUSG00000020135
</td>
<td style="text-align:left;">
MGI:1346052
</td>
<td style="text-align:left;">
protein coding gene
</td>
<td style="text-align:left;">
adenomatosis polyposis coli 2
</td>
</tr>
<tr>
<td style="text-align:left;">
10
</td>
<td style="text-align:left;">
gene
</td>
<td style="text-align:right;">
80.31825
</td>
<td style="text-align:right;">
80.32055
</td>
<td style="text-align:left;">
negative
</td>
<td style="text-align:left;">
MGI_C57BL6J_1913624
</td>
<td style="text-align:left;">
2310011J03Rik
</td>
<td style="text-align:left;">
NCBI_Gene:66374,ENSEMBL:ENSMUSG00000020133
</td>
<td style="text-align:left;">
MGI:1913624
</td>
<td style="text-align:left;">
protein coding gene
</td>
<td style="text-align:left;">
RIKEN cDNA 2310011J03 gene
</td>
</tr>
<tr>
<td style="text-align:left;">
10
</td>
<td style="text-align:left;">
gene
</td>
<td style="text-align:right;">
80.32128
</td>
<td style="text-align:right;">
80.32991
</td>
<td style="text-align:left;">
negative
</td>
<td style="text-align:left;">
MGI_C57BL6J_97514
</td>
<td style="text-align:left;">
Pcsk4
</td>
<td style="text-align:left;">
NCBI_Gene:18551,ENSEMBL:ENSMUSG00000020131
</td>
<td style="text-align:left;">
MGI:97514
</td>
<td style="text-align:left;">
protein coding gene
</td>
<td style="text-align:left;">
proprotein convertase subtilisin/kexin type 4
</td>
</tr>
<tr>
<td style="text-align:left;">
10
</td>
<td style="text-align:left;">
gene
</td>
<td style="text-align:right;">
80.32995
</td>
<td style="text-align:right;">
80.33644
</td>
<td style="text-align:left;">
positive
</td>
<td style="text-align:left;">
MGI_C57BL6J_1917585
</td>
<td style="text-align:left;">
Reep6
</td>
<td style="text-align:left;">
NCBI_Gene:70335,ENSEMBL:ENSMUSG00000035504
</td>
<td style="text-align:left;">
MGI:1917585
</td>
<td style="text-align:left;">
protein coding gene
</td>
<td style="text-align:left;">
receptor accessory protein 6
</td>
</tr>
<tr>
<td style="text-align:left;">
10
</td>
<td style="text-align:left;">
gene
</td>
<td style="text-align:right;">
80.33911
</td>
<td style="text-align:right;">
80.34904
</td>
<td style="text-align:left;">
negative
</td>
<td style="text-align:left;">
MGI_C57BL6J_1913798
</td>
<td style="text-align:left;">
Adamtsl5
</td>
<td style="text-align:left;">
NCBI_Gene:66548,ENSEMBL:ENSMUSG00000043822
</td>
<td style="text-align:left;">
MGI:1913798
</td>
<td style="text-align:left;">
protein coding gene
</td>
<td style="text-align:left;">
ADAMTS-like 5
</td>
</tr>
<tr>
<td style="text-align:left;">
10
</td>
<td style="text-align:left;">
gene
</td>
<td style="text-align:right;">
80.35646
</td>
<td style="text-align:right;">
80.36549
</td>
<td style="text-align:left;">
positive
</td>
<td style="text-align:left;">
MGI_C57BL6J_3026984
</td>
<td style="text-align:left;">
Plk5
</td>
<td style="text-align:left;">
NCBI_Gene:216166,ENSEMBL:ENSMUSG00000035486
</td>
<td style="text-align:left;">
MGI:3026984
</td>
<td style="text-align:left;">
protein coding gene
</td>
<td style="text-align:left;">
polo like kinase 5
</td>
</tr>
<tr>
<td style="text-align:left;">
10
</td>
<td style="text-align:left;">
gene
</td>
<td style="text-align:right;">
80.37190
</td>
<td style="text-align:right;">
80.37243
</td>
<td style="text-align:left;">
positive
</td>
<td style="text-align:left;">
MGI_C57BL6J_3705256
</td>
<td style="text-align:left;">
Gm15123
</td>
<td style="text-align:left;">
ENSEMBL:ENSMUSG00000087432
</td>
<td style="text-align:left;">
MGI:3705256
</td>
<td style="text-align:left;">
lncRNA gene
</td>
<td style="text-align:left;">
predicted gene 15123
</td>
</tr>
<tr>
<td style="text-align:left;">
10
</td>
<td style="text-align:left;">
gene
</td>
<td style="text-align:right;">
80.37195
</td>
<td style="text-align:right;">
80.37322
</td>
<td style="text-align:left;">
positive
</td>
<td style="text-align:left;">
MGI_C57BL6J_3705261
</td>
<td style="text-align:left;">
Gm15124
</td>
<td style="text-align:left;">
ENSEMBL:ENSMUSG00000085712
</td>
<td style="text-align:left;">
MGI:3705261
</td>
<td style="text-align:left;">
lncRNA gene
</td>
<td style="text-align:left;">
predicted gene 15124
</td>
</tr>
<tr>
<td style="text-align:left;">
10
</td>
<td style="text-align:left;">
pseudogene
</td>
<td style="text-align:right;">
80.37717
</td>
<td style="text-align:right;">
80.37755
</td>
<td style="text-align:left;">
negative
</td>
<td style="text-align:left;">
MGI_C57BL6J_6096965
</td>
<td style="text-align:left;">
Gm47793
</td>
<td style="text-align:left;">
ENSEMBL:ENSMUSG00000112648
</td>
<td style="text-align:left;">
MGI:6096965
</td>
<td style="text-align:left;">
pseudogene
</td>
<td style="text-align:left;">
predicted gene, 47793
</td>
</tr>
<tr>
<td style="text-align:left;">
10
</td>
<td style="text-align:left;">
gene
</td>
<td style="text-align:right;">
80.38035
</td>
<td style="text-align:right;">
80.38766
</td>
<td style="text-align:left;">
negative
</td>
<td style="text-align:left;">
MGI_C57BL6J_2681847
</td>
<td style="text-align:left;">
Mex3d
</td>
<td style="text-align:left;">
NCBI_Gene:237400,ENSEMBL:ENSMUSG00000048696
</td>
<td style="text-align:left;">
MGI:2681847
</td>
<td style="text-align:left;">
protein coding gene
</td>
<td style="text-align:left;">
mex3 RNA binding family member D
</td>
</tr>
<tr>
<td style="text-align:left;">
10
</td>
<td style="text-align:left;">
gene
</td>
<td style="text-align:right;">
80.38230
</td>
<td style="text-align:right;">
80.38238
</td>
<td style="text-align:left;">
positive
</td>
<td style="text-align:left;">
MGI_C57BL6J_5452498
</td>
<td style="text-align:left;">
Gm22721
</td>
<td style="text-align:left;">
ENSEMBL:ENSMUSG00000084658
</td>
<td style="text-align:left;">
MGI:5452498
</td>
<td style="text-align:left;">
miRNA gene
</td>
<td style="text-align:left;">
predicted gene, 22721
</td>
</tr>
<tr>
<td style="text-align:left;">
10
</td>
<td style="text-align:left;">
gene
</td>
<td style="text-align:right;">
80.38776
</td>
<td style="text-align:right;">
80.39507
</td>
<td style="text-align:left;">
positive
</td>
<td style="text-align:left;">
MGI_C57BL6J_5589982
</td>
<td style="text-align:left;">
Gm30823
</td>
<td style="text-align:left;">
NCBI_Gene:102632860
</td>
<td style="text-align:left;">
MGI:5589982
</td>
<td style="text-align:left;">
lncRNA gene
</td>
<td style="text-align:left;">
predicted gene, 30823
</td>
</tr>
<tr>
<td style="text-align:left;">
10
</td>
<td style="text-align:left;">
gene
</td>
<td style="text-align:right;">
80.39254
</td>
<td style="text-align:right;">
80.39955
</td>
<td style="text-align:left;">
negative
</td>
<td style="text-align:left;">
MGI_C57BL6J_1333812
</td>
<td style="text-align:left;">
Mbd3
</td>
<td style="text-align:left;">
NCBI_Gene:17192,ENSEMBL:ENSMUSG00000035478
</td>
<td style="text-align:left;">
MGI:1333812
</td>
<td style="text-align:left;">
protein coding gene
</td>
<td style="text-align:left;">
methyl-CpG binding domain protein 3
</td>
</tr>
<tr>
<td style="text-align:left;">
10
</td>
<td style="text-align:left;">
gene
</td>
<td style="text-align:right;">
80.40131
</td>
<td style="text-align:right;">
80.40167
</td>
<td style="text-align:left;">
negative
</td>
<td style="text-align:left;">
MGI_C57BL6J_1917496
</td>
<td style="text-align:left;">
3300002P09Rik
</td>
<td style="text-align:left;">
NA
</td>
<td style="text-align:left;">
NA
</td>
<td style="text-align:left;">
unclassified gene
</td>
<td style="text-align:left;">
RIKEN cDNA 3300002P09 gene
</td>
</tr>
<tr>
<td style="text-align:left;">
10
</td>
<td style="text-align:left;">
gene
</td>
<td style="text-align:right;">
80.40300
</td>
<td style="text-align:right;">
80.40683
</td>
<td style="text-align:left;">
negative
</td>
<td style="text-align:left;">
MGI_C57BL6J_1913844
</td>
<td style="text-align:left;">
Uqcr11
</td>
<td style="text-align:left;">
NCBI_Gene:66594,ENSEMBL:ENSMUSG00000020163
</td>
<td style="text-align:left;">
MGI:1913844
</td>
<td style="text-align:left;">
protein coding gene
</td>
<td style="text-align:left;">
ubiquinol-cytochrome c reductase, complex III subunit XI
</td>
</tr>
<tr>
<td style="text-align:left;">
10
</td>
<td style="text-align:left;">
gene
</td>
<td style="text-align:right;">
80.40917
</td>
<td style="text-align:right;">
80.43396
</td>
<td style="text-align:left;">
negative
</td>
<td style="text-align:left;">
MGI_C57BL6J_98510
</td>
<td style="text-align:left;">
Tcf3
</td>
<td style="text-align:left;">
NCBI_Gene:21423,ENSEMBL:ENSMUSG00000020167
</td>
<td style="text-align:left;">
MGI:98510
</td>
<td style="text-align:left;">
protein coding gene
</td>
<td style="text-align:left;">
transcription factor 3
</td>
</tr>
<tr>
<td style="text-align:left;">
10
</td>
<td style="text-align:left;">
gene
</td>
<td style="text-align:right;">
80.47141
</td>
<td style="text-align:right;">
80.47538
</td>
<td style="text-align:left;">
negative
</td>
<td style="text-align:left;">
MGI_C57BL6J_5623601
</td>
<td style="text-align:left;">
Gm40716
</td>
<td style="text-align:left;">
NCBI_Gene:105245232
</td>
<td style="text-align:left;">
MGI:5623601
</td>
<td style="text-align:left;">
lncRNA gene
</td>
<td style="text-align:left;">
predicted gene, 40716
</td>
</tr>
<tr>
<td style="text-align:left;">
10
</td>
<td style="text-align:left;">
gene
</td>
<td style="text-align:right;">
80.47635
</td>
<td style="text-align:right;">
80.47646
</td>
<td style="text-align:left;">
positive
</td>
<td style="text-align:left;">
MGI_C57BL6J_5454821
</td>
<td style="text-align:left;">
Gm25044
</td>
<td style="text-align:left;">
ENSEMBL:ENSMUSG00000089228
</td>
<td style="text-align:left;">
MGI:5454821
</td>
<td style="text-align:left;">
snRNA gene
</td>
<td style="text-align:left;">
predicted gene, 25044
</td>
</tr>
<tr>
<td style="text-align:left;">
10
</td>
<td style="text-align:left;">
gene
</td>
<td style="text-align:right;">
80.49483
</td>
<td style="text-align:right;">
80.51728
</td>
<td style="text-align:left;">
positive
</td>
<td style="text-align:left;">
MGI_C57BL6J_1891409
</td>
<td style="text-align:left;">
Onecut3
</td>
<td style="text-align:left;">
NCBI_Gene:246086,ENSEMBL:ENSMUSG00000045518
</td>
<td style="text-align:left;">
MGI:1891409
</td>
<td style="text-align:left;">
protein coding gene
</td>
<td style="text-align:left;">
one cut domain, family member 3
</td>
</tr>
<tr>
<td style="text-align:left;">
10
</td>
<td style="text-align:left;">
gene
</td>
<td style="text-align:right;">
80.50877
</td>
<td style="text-align:right;">
80.50962
</td>
<td style="text-align:left;">
positive
</td>
<td style="text-align:left;">
MGI_C57BL6J_1923612
</td>
<td style="text-align:left;">
2610034O05Rik
</td>
<td style="text-align:left;">
ENSEMBL:ENSMUSG00000104429
</td>
<td style="text-align:left;">
MGI:1923612
</td>
<td style="text-align:left;">
unclassified gene
</td>
<td style="text-align:left;">
RIKEN cDNA 2610034O05 gene
</td>
</tr>
<tr>
<td style="text-align:left;">
10
</td>
<td style="text-align:left;">
gene
</td>
<td style="text-align:right;">
80.51958
</td>
<td style="text-align:right;">
80.53915
</td>
<td style="text-align:left;">
negative
</td>
<td style="text-align:left;">
MGI_C57BL6J_1914581
</td>
<td style="text-align:left;">
Atp8b3
</td>
<td style="text-align:left;">
NCBI_Gene:67331,ENSEMBL:ENSMUSG00000003341
</td>
<td style="text-align:left;">
MGI:1914581
</td>
<td style="text-align:left;">
protein coding gene
</td>
<td style="text-align:left;">
ATPase, class I, type 8B, member 3
</td>
</tr>
<tr>
<td style="text-align:left;">
10
</td>
<td style="text-align:left;">
gene
</td>
<td style="text-align:right;">
80.54092
</td>
<td style="text-align:right;">
80.56227
</td>
<td style="text-align:left;">
negative
</td>
<td style="text-align:left;">
MGI_C57BL6J_1914182
</td>
<td style="text-align:left;">
Rexo1
</td>
<td style="text-align:left;">
NCBI_Gene:66932,ENSEMBL:ENSMUSG00000047417
</td>
<td style="text-align:left;">
MGI:1914182
</td>
<td style="text-align:left;">
protein coding gene
</td>
<td style="text-align:left;">
REX1, RNA exonuclease 1
</td>
</tr>
<tr>
<td style="text-align:left;">
10
</td>
<td style="text-align:left;">
gene
</td>
<td style="text-align:right;">
80.54749
</td>
<td style="text-align:right;">
80.55530
</td>
<td style="text-align:left;">
positive
</td>
<td style="text-align:left;">
MGI_C57BL6J_5590216
</td>
<td style="text-align:left;">
Gm31057
</td>
<td style="text-align:left;">
NCBI_Gene:102633159
</td>
<td style="text-align:left;">
MGI:5590216
</td>
<td style="text-align:left;">
lncRNA gene
</td>
<td style="text-align:left;">
predicted gene, 31057
</td>
</tr>
<tr>
<td style="text-align:left;">
10
</td>
<td style="text-align:left;">
gene
</td>
<td style="text-align:right;">
80.56712
</td>
<td style="text-align:right;">
80.57732
</td>
<td style="text-align:left;">
negative
</td>
<td style="text-align:left;">
MGI_C57BL6J_2153049
</td>
<td style="text-align:left;">
Klf16
</td>
<td style="text-align:left;">
NCBI_Gene:118445,ENSEMBL:ENSMUSG00000035397
</td>
<td style="text-align:left;">
MGI:2153049
</td>
<td style="text-align:left;">
protein coding gene
</td>
<td style="text-align:left;">
Kruppel-like factor 16
</td>
</tr>
<tr>
<td style="text-align:left;">
10
</td>
<td style="text-align:left;">
gene
</td>
<td style="text-align:right;">
80.56992
</td>
<td style="text-align:right;">
80.58068
</td>
<td style="text-align:left;">
positive
</td>
<td style="text-align:left;">
MGI_C57BL6J_5477204
</td>
<td style="text-align:left;">
Gm26710
</td>
<td style="text-align:left;">
NCBI_Gene:102633244,ENSEMBL:ENSMUSG00000097013
</td>
<td style="text-align:left;">
MGI:5477204
</td>
<td style="text-align:left;">
lncRNA gene
</td>
<td style="text-align:left;">
predicted gene, 26710
</td>
</tr>
<tr>
<td style="text-align:left;">
10
</td>
<td style="text-align:left;">
gene
</td>
<td style="text-align:right;">
80.58365
</td>
<td style="text-align:right;">
80.59034
</td>
<td style="text-align:left;">
negative
</td>
<td style="text-align:left;">
MGI_C57BL6J_106388
</td>
<td style="text-align:left;">
Abhd17a
</td>
<td style="text-align:left;">
NCBI_Gene:216169,ENSEMBL:ENSMUSG00000003346
</td>
<td style="text-align:left;">
MGI:106388
</td>
<td style="text-align:left;">
protein coding gene
</td>
<td style="text-align:left;">
abhydrolase domain containing 17A
</td>
</tr>
<tr>
<td style="text-align:left;">
10
</td>
<td style="text-align:left;">
gene
</td>
<td style="text-align:right;">
80.58796
</td>
<td style="text-align:right;">
80.58979
</td>
<td style="text-align:left;">
negative
</td>
<td style="text-align:left;">
MGI_C57BL6J_5579799
</td>
<td style="text-align:left;">
Gm29093
</td>
<td style="text-align:left;">
ENSEMBL:ENSMUSG00000100201
</td>
<td style="text-align:left;">
MGI:5579799
</td>
<td style="text-align:left;">
lncRNA gene
</td>
<td style="text-align:left;">
predicted gene 29093
</td>
</tr>
<tr>
<td style="text-align:left;">
10
</td>
<td style="text-align:left;">
gene
</td>
<td style="text-align:right;">
80.60287
</td>
<td style="text-align:right;">
80.61578
</td>
<td style="text-align:left;">
positive
</td>
<td style="text-align:left;">
MGI_C57BL6J_1928947
</td>
<td style="text-align:left;">
Scamp4
</td>
<td style="text-align:left;">
NCBI_Gene:56214,ENSEMBL:ENSMUSG00000113949
</td>
<td style="text-align:left;">
MGI:1928947
</td>
<td style="text-align:left;">
protein coding gene
</td>
<td style="text-align:left;">
secretory carrier membrane protein 4
</td>
</tr>
<tr>
<td style="text-align:left;">
10
</td>
<td style="text-align:left;">
gene
</td>
<td style="text-align:right;">
80.60288
</td>
<td style="text-align:right;">
80.60765
</td>
<td style="text-align:left;">
positive
</td>
<td style="text-align:left;">
MGI_C57BL6J_1924344
</td>
<td style="text-align:left;">
Adat3
</td>
<td style="text-align:left;">
NCBI_Gene:100113398,ENSEMBL:ENSMUSG00000113640
</td>
<td style="text-align:left;">
MGI:1924344
</td>
<td style="text-align:left;">
protein coding gene
</td>
<td style="text-align:left;">
adenosine deaminase, tRNA-specific 3
</td>
</tr>
<tr>
<td style="text-align:left;">
10
</td>
<td style="text-align:left;">
gene
</td>
<td style="text-align:right;">
80.60602
</td>
<td style="text-align:right;">
80.61578
</td>
<td style="text-align:left;">
positive
</td>
<td style="text-align:left;">
MGI_C57BL6J_6121503
</td>
<td style="text-align:left;">
Gm49322
</td>
<td style="text-align:left;">
ENSEMBL:ENSMUSG00000035370
</td>
<td style="text-align:left;">
MGI:6121503
</td>
<td style="text-align:left;">
protein coding gene
</td>
<td style="text-align:left;">
predicted gene, 49322
</td>
</tr>
<tr>
<td style="text-align:left;">
10
</td>
<td style="text-align:left;">
gene
</td>
<td style="text-align:right;">
80.60926
</td>
<td style="text-align:right;">
80.60933
</td>
<td style="text-align:left;">
positive
</td>
<td style="text-align:left;">
MGI_C57BL6J_5531372
</td>
<td style="text-align:left;">
Mir6912
</td>
<td style="text-align:left;">
miRBase:MI0022759,NCBI_Gene:102465550,ENSEMBL:ENSMUSG00000099283
</td>
<td style="text-align:left;">
MGI:5531372
</td>
<td style="text-align:left;">
miRNA gene
</td>
<td style="text-align:left;">
microRNA 6912
</td>
</tr>
<tr>
<td style="text-align:left;">
10
</td>
<td style="text-align:left;">
gene
</td>
<td style="text-align:right;">
80.61296
</td>
<td style="text-align:right;">
80.62261
</td>
<td style="text-align:left;">
negative
</td>
<td style="text-align:left;">
MGI_C57BL6J_5590457
</td>
<td style="text-align:left;">
Gm31298
</td>
<td style="text-align:left;">
NCBI_Gene:102633484
</td>
<td style="text-align:left;">
MGI:5590457
</td>
<td style="text-align:left;">
lncRNA gene
</td>
<td style="text-align:left;">
predicted gene, 31298
</td>
</tr>
<tr>
<td style="text-align:left;">
10
</td>
<td style="text-align:left;">
gene
</td>
<td style="text-align:right;">
80.62270
</td>
<td style="text-align:right;">
80.64077
</td>
<td style="text-align:left;">
positive
</td>
<td style="text-align:left;">
MGI_C57BL6J_1920014
</td>
<td style="text-align:left;">
Csnk1g2
</td>
<td style="text-align:left;">
NCBI_Gene:103236,ENSEMBL:ENSMUSG00000003345
</td>
<td style="text-align:left;">
MGI:1920014
</td>
<td style="text-align:left;">
protein coding gene
</td>
<td style="text-align:left;">
casein kinase 1, gamma 2
</td>
</tr>
<tr>
<td style="text-align:left;">
10
</td>
<td style="text-align:left;">
gene
</td>
<td style="text-align:right;">
80.64262
</td>
<td style="text-align:right;">
80.65712
</td>
<td style="text-align:left;">
negative
</td>
<td style="text-align:left;">
MGI_C57BL6J_1933831
</td>
<td style="text-align:left;">
Btbd2
</td>
<td style="text-align:left;">
NCBI_Gene:208198,ENSEMBL:ENSMUSG00000003344
</td>
<td style="text-align:left;">
MGI:1933831
</td>
<td style="text-align:left;">
protein coding gene
</td>
<td style="text-align:left;">
BTB (POZ) domain containing 2
</td>
</tr>
<tr>
<td style="text-align:left;">
10
</td>
<td style="text-align:left;">
gene
</td>
<td style="text-align:right;">
80.66532
</td>
<td style="text-align:right;">
80.67811
</td>
<td style="text-align:left;">
negative
</td>
<td style="text-align:left;">
MGI_C57BL6J_894279
</td>
<td style="text-align:left;">
Mknk2
</td>
<td style="text-align:left;">
NCBI_Gene:17347,ENSEMBL:ENSMUSG00000020190
</td>
<td style="text-align:left;">
MGI:894279
</td>
<td style="text-align:left;">
protein coding gene
</td>
<td style="text-align:left;">
MAP kinase-interacting serine/threonine kinase 2
</td>
</tr>
<tr>
<td style="text-align:left;">
10
</td>
<td style="text-align:left;">
gene
</td>
<td style="text-align:right;">
80.68184
</td>
<td style="text-align:right;">
80.68718
</td>
<td style="text-align:left;">
positive
</td>
<td style="text-align:left;">
MGI_C57BL6J_5477035
</td>
<td style="text-align:left;">
Gm26541
</td>
<td style="text-align:left;">
ENSEMBL:ENSMUSG00000097889
</td>
<td style="text-align:left;">
MGI:5477035
</td>
<td style="text-align:left;">
lncRNA gene
</td>
<td style="text-align:left;">
predicted gene, 26541
</td>
</tr>
<tr>
<td style="text-align:left;">
10
</td>
<td style="text-align:left;">
gene
</td>
<td style="text-align:right;">
80.68525
</td>
<td style="text-align:right;">
80.70198
</td>
<td style="text-align:left;">
negative
</td>
<td style="text-align:left;">
MGI_C57BL6J_3050117
</td>
<td style="text-align:left;">
Mob3a
</td>
<td style="text-align:left;">
NCBI_Gene:208228,ENSEMBL:ENSMUSG00000003348
</td>
<td style="text-align:left;">
MGI:3050117
</td>
<td style="text-align:left;">
protein coding gene
</td>
<td style="text-align:left;">
MOB kinase activator 3A
</td>
</tr>
<tr>
<td style="text-align:left;">
10
</td>
<td style="text-align:left;">
gene
</td>
<td style="text-align:right;">
80.70201
</td>
<td style="text-align:right;">
80.70538
</td>
<td style="text-align:left;">
positive
</td>
<td style="text-align:left;">
MGI_C57BL6J_1918814
</td>
<td style="text-align:left;">
Izumo4
</td>
<td style="text-align:left;">
NCBI_Gene:71564,ENSEMBL:ENSMUSG00000055862
</td>
<td style="text-align:left;">
MGI:1918814
</td>
<td style="text-align:left;">
protein coding gene
</td>
<td style="text-align:left;">
IZUMO family member 4
</td>
</tr>
<tr>
<td style="text-align:left;">
10
</td>
<td style="text-align:left;">
gene
</td>
<td style="text-align:right;">
80.70696
</td>
<td style="text-align:right;">
80.74228
</td>
<td style="text-align:left;">
negative
</td>
<td style="text-align:left;">
MGI_C57BL6J_107734
</td>
<td style="text-align:left;">
Ap3d1
</td>
<td style="text-align:left;">
NCBI_Gene:11776,ENSEMBL:ENSMUSG00000020198
</td>
<td style="text-align:left;">
MGI:107734
</td>
<td style="text-align:left;">
protein coding gene
</td>
<td style="text-align:left;">
adaptor-related protein complex 3, delta 1 subunit
</td>
</tr>
<tr>
<td style="text-align:left;">
10
</td>
<td style="text-align:left;">
gene
</td>
<td style="text-align:right;">
80.75513
</td>
<td style="text-align:right;">
80.79546
</td>
<td style="text-align:left;">
positive
</td>
<td style="text-align:left;">
MGI_C57BL6J_2143886
</td>
<td style="text-align:left;">
Dot1l
</td>
<td style="text-align:left;">
NCBI_Gene:208266,ENSEMBL:ENSMUSG00000061589
</td>
<td style="text-align:left;">
MGI:2143886
</td>
<td style="text-align:left;">
protein coding gene
</td>
<td style="text-align:left;">
DOT1-like, histone H3 methyltransferase (S. cerevisiae)
</td>
</tr>
<tr>
<td style="text-align:left;">
10
</td>
<td style="text-align:left;">
gene
</td>
<td style="text-align:right;">
80.76686
</td>
<td style="text-align:right;">
80.76899
</td>
<td style="text-align:left;">
negative
</td>
<td style="text-align:left;">
MGI_C57BL6J_4937978
</td>
<td style="text-align:left;">
Gm17151
</td>
<td style="text-align:left;">
ENSEMBL:ENSMUSG00000075558
</td>
<td style="text-align:left;">
MGI:4937978
</td>
<td style="text-align:left;">
lncRNA gene
</td>
<td style="text-align:left;">
predicted gene 17151
</td>
</tr>
<tr>
<td style="text-align:left;">
10
</td>
<td style="text-align:left;">
gene
</td>
<td style="text-align:right;">
80.79610
</td>
<td style="text-align:right;">
80.79863
</td>
<td style="text-align:left;">
negative
</td>
<td style="text-align:left;">
MGI_C57BL6J_1925920
</td>
<td style="text-align:left;">
Plekhj1
</td>
<td style="text-align:left;">
NCBI_Gene:78670,ENSEMBL:ENSMUSG00000035278
</td>
<td style="text-align:left;">
MGI:1925920
</td>
<td style="text-align:left;">
protein coding gene
</td>
<td style="text-align:left;">
pleckstrin homology domain containing, family J member 1
</td>
</tr>
<tr>
<td style="text-align:left;">
10
</td>
<td style="text-align:left;">
gene
</td>
<td style="text-align:right;">
80.79820
</td>
<td style="text-align:right;">
80.80492
</td>
<td style="text-align:left;">
positive
</td>
<td style="text-align:left;">
MGI_C57BL6J_104912
</td>
<td style="text-align:left;">
Sf3a2
</td>
<td style="text-align:left;">
NCBI_Gene:20222,ENSEMBL:ENSMUSG00000020211
</td>
<td style="text-align:left;">
MGI:104912
</td>
<td style="text-align:left;">
protein coding gene
</td>
<td style="text-align:left;">
splicing factor 3a, subunit 2
</td>
</tr>
<tr>
<td style="text-align:left;">
10
</td>
<td style="text-align:left;">
gene
</td>
<td style="text-align:right;">
80.80503
</td>
<td style="text-align:right;">
80.80768
</td>
<td style="text-align:left;">
positive
</td>
<td style="text-align:left;">
MGI_C57BL6J_88006
</td>
<td style="text-align:left;">
Amh
</td>
<td style="text-align:left;">
NCBI_Gene:11705,ENSEMBL:ENSMUSG00000035262
</td>
<td style="text-align:left;">
MGI:88006
</td>
<td style="text-align:left;">
protein coding gene
</td>
<td style="text-align:left;">
anti-Mullerian hormone
</td>
</tr>
<tr>
<td style="text-align:left;">
10
</td>
<td style="text-align:left;">
gene
</td>
<td style="text-align:right;">
80.80849
</td>
<td style="text-align:right;">
80.81373
</td>
<td style="text-align:left;">
negative
</td>
<td style="text-align:left;">
MGI_C57BL6J_1916700
</td>
<td style="text-align:left;">
Jsrp1
</td>
<td style="text-align:left;">
NCBI_Gene:71912,ENSEMBL:ENSMUSG00000020216
</td>
<td style="text-align:left;">
MGI:1916700
</td>
<td style="text-align:left;">
protein coding gene
</td>
<td style="text-align:left;">
junctional sarcoplasmic reticulum protein 1
</td>
</tr>
<tr>
<td style="text-align:left;">
10
</td>
<td style="text-align:left;">
gene
</td>
<td style="text-align:right;">
80.82666
</td>
<td style="text-align:right;">
80.82929
</td>
<td style="text-align:left;">
positive
</td>
<td style="text-align:left;">
MGI_C57BL6J_109433
</td>
<td style="text-align:left;">
Oaz1
</td>
<td style="text-align:left;">
NCBI_Gene:18245,ENSEMBL:ENSMUSG00000035242
</td>
<td style="text-align:left;">
MGI:109433
</td>
<td style="text-align:left;">
protein coding gene
</td>
<td style="text-align:left;">
ornithine decarboxylase antizyme 1
</td>
</tr>
<tr>
<td style="text-align:left;">
10
</td>
<td style="text-align:left;">
gene
</td>
<td style="text-align:right;">
80.82880
</td>
<td style="text-align:right;">
80.82887
</td>
<td style="text-align:left;">
positive
</td>
<td style="text-align:left;">
MGI_C57BL6J_3837227
</td>
<td style="text-align:left;">
Mir1982
</td>
<td style="text-align:left;">
miRBase:MI0009993,NCBI_Gene:100316778,ENSEMBL:ENSMUSG00000087993
</td>
<td style="text-align:left;">
MGI:3837227
</td>
<td style="text-align:left;">
miRNA gene
</td>
<td style="text-align:left;">
microRNA 1982
</td>
</tr>
<tr>
<td style="text-align:left;">
10
</td>
<td style="text-align:left;">
gene
</td>
<td style="text-align:right;">
80.83280
</td>
<td style="text-align:right;">
80.84404
</td>
<td style="text-align:left;">
negative
</td>
<td style="text-align:left;">
MGI_C57BL6J_3609246
</td>
<td style="text-align:left;">
Lingo3
</td>
<td style="text-align:left;">
NCBI_Gene:237403,ENSEMBL:ENSMUSG00000051067
</td>
<td style="text-align:left;">
MGI:3609246
</td>
<td style="text-align:left;">
protein coding gene
</td>
<td style="text-align:left;">
leucine rich repeat and Ig domain containing 3
</td>
</tr>
<tr>
<td style="text-align:left;">
10
</td>
<td style="text-align:left;">
gene
</td>
<td style="text-align:right;">
80.85282
</td>
<td style="text-align:right;">
80.85525
</td>
<td style="text-align:left;">
negative
</td>
<td style="text-align:left;">
MGI_C57BL6J_1913344
</td>
<td style="text-align:left;">
Lsm7
</td>
<td style="text-align:left;">
NCBI_Gene:66094,ENSEMBL:ENSMUSG00000035215
</td>
<td style="text-align:left;">
MGI:1913344
</td>
<td style="text-align:left;">
protein coding gene
</td>
<td style="text-align:left;">
LSM7 homolog, U6 small nuclear RNA and mRNA degradation associated
</td>
</tr>
<tr>
<td style="text-align:left;">
10
</td>
<td style="text-align:left;">
gene
</td>
<td style="text-align:right;">
80.85525
</td>
<td style="text-align:right;">
80.86871
</td>
<td style="text-align:left;">
positive
</td>
<td style="text-align:left;">
MGI_C57BL6J_1920468
</td>
<td style="text-align:left;">
Sppl2b
</td>
<td style="text-align:left;">
NCBI_Gene:73218,ENSEMBL:ENSMUSG00000035206
</td>
<td style="text-align:left;">
MGI:1920468
</td>
<td style="text-align:left;">
protein coding gene
</td>
<td style="text-align:left;">
signal peptide peptidase like 2B
</td>
</tr>
<tr>
<td style="text-align:left;">
10
</td>
<td style="text-align:left;">
gene
</td>
<td style="text-align:right;">
80.87112
</td>
<td style="text-align:right;">
80.89950
</td>
<td style="text-align:left;">
positive
</td>
<td style="text-align:left;">
MGI_C57BL6J_3612246
</td>
<td style="text-align:left;">
Tmprss9
</td>
<td style="text-align:left;">
NCBI_Gene:432478,ENSEMBL:ENSMUSG00000059406
</td>
<td style="text-align:left;">
MGI:3612246
</td>
<td style="text-align:left;">
protein coding gene
</td>
<td style="text-align:left;">
transmembrane protease, serine 9
</td>
</tr>
<tr>
<td style="text-align:left;">
10
</td>
<td style="text-align:left;">
gene
</td>
<td style="text-align:right;">
80.89945
</td>
<td style="text-align:right;">
80.90097
</td>
<td style="text-align:left;">
negative
</td>
<td style="text-align:left;">
MGI_C57BL6J_1353432
</td>
<td style="text-align:left;">
Timm13
</td>
<td style="text-align:left;">
NCBI_Gene:30055,ENSEMBL:ENSMUSG00000020219
</td>
<td style="text-align:left;">
MGI:1353432
</td>
<td style="text-align:left;">
protein coding gene
</td>
<td style="text-align:left;">
translocase of inner mitochondrial membrane 13
</td>
</tr>
<tr>
<td style="text-align:left;">
10
</td>
<td style="text-align:left;">
gene
</td>
<td style="text-align:right;">
80.90120
</td>
<td style="text-align:right;">
80.91824
</td>
<td style="text-align:left;">
negative
</td>
<td style="text-align:left;">
MGI_C57BL6J_96796
</td>
<td style="text-align:left;">
Lmnb2
</td>
<td style="text-align:left;">
NCBI_Gene:16907,ENSEMBL:ENSMUSG00000062075
</td>
<td style="text-align:left;">
MGI:96796
</td>
<td style="text-align:left;">
protein coding gene
</td>
<td style="text-align:left;">
lamin B2
</td>
</tr>
<tr>
<td style="text-align:left;">
10
</td>
<td style="text-align:left;">
gene
</td>
<td style="text-align:right;">
80.93007
</td>
<td style="text-align:right;">
80.93220
</td>
<td style="text-align:left;">
positive
</td>
<td style="text-align:left;">
MGI_C57BL6J_107776
</td>
<td style="text-align:left;">
Gadd45b
</td>
<td style="text-align:left;">
NCBI_Gene:17873,ENSEMBL:ENSMUSG00000015312
</td>
<td style="text-align:left;">
MGI:107776
</td>
<td style="text-align:left;">
protein coding gene
</td>
<td style="text-align:left;">
growth arrest and DNA-damage-inducible 45 beta
</td>
</tr>
<tr>
<td style="text-align:left;">
10
</td>
<td style="text-align:left;">
gene
</td>
<td style="text-align:right;">
80.94862
</td>
<td style="text-align:right;">
81.01494
</td>
<td style="text-align:left;">
negative
</td>
<td style="text-align:left;">
MGI_C57BL6J_95787
</td>
<td style="text-align:left;">
Gng7
</td>
<td style="text-align:left;">
NCBI_Gene:14708,ENSEMBL:ENSMUSG00000048240
</td>
<td style="text-align:left;">
MGI:95787
</td>
<td style="text-align:left;">
protein coding gene
</td>
<td style="text-align:left;">
guanine nucleotide binding protein (G protein), gamma 7
</td>
</tr>
<tr>
<td style="text-align:left;">
10
</td>
<td style="text-align:left;">
pseudogene
</td>
<td style="text-align:right;">
80.95425
</td>
<td style="text-align:right;">
80.95561
</td>
<td style="text-align:left;">
negative
</td>
<td style="text-align:left;">
MGI_C57BL6J_3782000
</td>
<td style="text-align:left;">
Gm3828
</td>
<td style="text-align:left;">
NCBI_Gene:102633671,ENSEMBL:ENSMUSG00000083808
</td>
<td style="text-align:left;">
MGI:3782000
</td>
<td style="text-align:left;">
pseudogene
</td>
<td style="text-align:left;">
predicted gene 3828
</td>
</tr>
<tr>
<td style="text-align:left;">
10
</td>
<td style="text-align:left;">
gene
</td>
<td style="text-align:right;">
81.01959
</td>
<td style="text-align:right;">
81.02582
</td>
<td style="text-align:left;">
negative
</td>
<td style="text-align:left;">
MGI_C57BL6J_2183442
</td>
<td style="text-align:left;">
Diras1
</td>
<td style="text-align:left;">
NCBI_Gene:208666,ENSEMBL:ENSMUSG00000043670
</td>
<td style="text-align:left;">
MGI:2183442
</td>
<td style="text-align:left;">
protein coding gene
</td>
<td style="text-align:left;">
DIRAS family, GTP-binding RAS-like 1
</td>
</tr>
<tr>
<td style="text-align:left;">
10
</td>
<td style="text-align:left;">
gene
</td>
<td style="text-align:right;">
81.02481
</td>
<td style="text-align:right;">
81.02657
</td>
<td style="text-align:left;">
positive
</td>
<td style="text-align:left;">
MGI_C57BL6J_3802061
</td>
<td style="text-align:left;">
Gm16099
</td>
<td style="text-align:left;">
ENSEMBL:ENSMUSG00000087114
</td>
<td style="text-align:left;">
MGI:3802061
</td>
<td style="text-align:left;">
lncRNA gene
</td>
<td style="text-align:left;">
predicted gene 16099
</td>
</tr>
<tr>
<td style="text-align:left;">
10
</td>
<td style="text-align:left;">
gene
</td>
<td style="text-align:right;">
81.02854
</td>
<td style="text-align:right;">
81.03743
</td>
<td style="text-align:left;">
negative
</td>
<td style="text-align:left;">
MGI_C57BL6J_2147269
</td>
<td style="text-align:left;">
Slc39a3
</td>
<td style="text-align:left;">
NCBI_Gene:106947,ENSEMBL:ENSMUSG00000046822
</td>
<td style="text-align:left;">
MGI:2147269
</td>
<td style="text-align:left;">
protein coding gene
</td>
<td style="text-align:left;">
solute carrier family 39 (zinc transporter), member 3
</td>
</tr>
<tr>
<td style="text-align:left;">
10
</td>
<td style="text-align:left;">
gene
</td>
<td style="text-align:right;">
81.04407
</td>
<td style="text-align:right;">
81.06023
</td>
<td style="text-align:left;">
negative
</td>
<td style="text-align:left;">
MGI_C57BL6J_1098703
</td>
<td style="text-align:left;">
Sgta
</td>
<td style="text-align:left;">
NCBI_Gene:52551,ENSEMBL:ENSMUSG00000004937
</td>
<td style="text-align:left;">
MGI:1098703
</td>
<td style="text-align:left;">
protein coding gene
</td>
<td style="text-align:left;">
small glutamine-rich tetratricopeptide repeat (TPR)-containing, alpha
</td>
</tr>
<tr>
<td style="text-align:left;">
10
</td>
<td style="text-align:left;">
gene
</td>
<td style="text-align:right;">
81.07004
</td>
<td style="text-align:right;">
81.08256
</td>
<td style="text-align:left;">
positive
</td>
<td style="text-align:left;">
MGI_C57BL6J_1354165
</td>
<td style="text-align:left;">
Thop1
</td>
<td style="text-align:left;">
NCBI_Gene:50492,ENSEMBL:ENSMUSG00000004929
</td>
<td style="text-align:left;">
MGI:1354165
</td>
<td style="text-align:left;">
protein coding gene
</td>
<td style="text-align:left;">
thimet oligopeptidase 1
</td>
</tr>
<tr>
<td style="text-align:left;">
10
</td>
<td style="text-align:left;">
gene
</td>
<td style="text-align:right;">
81.08432
</td>
<td style="text-align:right;">
81.09887
</td>
<td style="text-align:left;">
negative
</td>
<td style="text-align:left;">
MGI_C57BL6J_2384786
</td>
<td style="text-align:left;">
Creb3l3
</td>
<td style="text-align:left;">
NCBI_Gene:208677,ENSEMBL:ENSMUSG00000035041
</td>
<td style="text-align:left;">
MGI:2384786
</td>
<td style="text-align:left;">
protein coding gene
</td>
<td style="text-align:left;">
cAMP responsive element binding protein 3-like 3
</td>
</tr>
<tr>
<td style="text-align:left;">
10
</td>
<td style="text-align:left;">
gene
</td>
<td style="text-align:right;">
81.10461
</td>
<td style="text-align:right;">
81.10467
</td>
<td style="text-align:left;">
negative
</td>
<td style="text-align:left;">
MGI_C57BL6J_5562727
</td>
<td style="text-align:left;">
Mir5615-1
</td>
<td style="text-align:left;">
miRBase:MI0019181,NCBI_Gene:100885825,ENSEMBL:ENSMUSG00000106347
</td>
<td style="text-align:left;">
MGI:5562727
</td>
<td style="text-align:left;">
miRNA gene
</td>
<td style="text-align:left;">
microRNA 5615-1
</td>
</tr>
<tr>
<td style="text-align:left;">
10
</td>
<td style="text-align:left;">
gene
</td>
<td style="text-align:right;">
81.10462
</td>
<td style="text-align:right;">
81.10468
</td>
<td style="text-align:left;">
positive
</td>
<td style="text-align:left;">
MGI_C57BL6J_5453144
</td>
<td style="text-align:left;">
Mir5615-2
</td>
<td style="text-align:left;">
miRBase:MI0019182,NCBI_Gene:100885826,ENSEMBL:ENSMUSG00000093498
</td>
<td style="text-align:left;">
MGI:5453144
</td>
<td style="text-align:left;">
miRNA gene
</td>
<td style="text-align:left;">
microRNA 5615-2
</td>
</tr>
<tr>
<td style="text-align:left;">
10
</td>
<td style="text-align:left;">
gene
</td>
<td style="text-align:right;">
81.10591
</td>
<td style="text-align:right;">
81.13398
</td>
<td style="text-align:left;">
positive
</td>
<td style="text-align:left;">
MGI_C57BL6J_1346867
</td>
<td style="text-align:left;">
Map2k2
</td>
<td style="text-align:left;">
NCBI_Gene:26396,ENSEMBL:ENSMUSG00000035027
</td>
<td style="text-align:left;">
MGI:1346867
</td>
<td style="text-align:left;">
protein coding gene
</td>
<td style="text-align:left;">
mitogen-activated protein kinase kinase 2
</td>
</tr>
<tr>
<td style="text-align:left;">
10
</td>
<td style="text-align:left;">
gene
</td>
<td style="text-align:right;">
81.12898
</td>
<td style="text-align:right;">
81.13069
</td>
<td style="text-align:left;">
positive
</td>
<td style="text-align:left;">
MGI_C57BL6J_5590696
</td>
<td style="text-align:left;">
Gm31537
</td>
<td style="text-align:left;">
NA
</td>
<td style="text-align:left;">
NA
</td>
<td style="text-align:left;">
unclassified non-coding RNA gene
</td>
<td style="text-align:left;">
predicted gene%2c 31537
</td>
</tr>
<tr>
<td style="text-align:left;">
10
</td>
<td style="text-align:left;">
gene
</td>
<td style="text-align:right;">
81.13522
</td>
<td style="text-align:right;">
81.15300
</td>
<td style="text-align:left;">
positive
</td>
<td style="text-align:left;">
MGI_C57BL6J_1335091
</td>
<td style="text-align:left;">
Zbtb7a
</td>
<td style="text-align:left;">
NCBI_Gene:16969,ENSEMBL:ENSMUSG00000035011
</td>
<td style="text-align:left;">
MGI:1335091
</td>
<td style="text-align:left;">
protein coding gene
</td>
<td style="text-align:left;">
zinc finger and BTB domain containing 7a
</td>
</tr>
<tr>
<td style="text-align:left;">
10
</td>
<td style="text-align:left;">
gene
</td>
<td style="text-align:right;">
81.15327
</td>
<td style="text-align:right;">
81.16792
</td>
<td style="text-align:left;">
negative
</td>
<td style="text-align:left;">
MGI_C57BL6J_2136940
</td>
<td style="text-align:left;">
Pias4
</td>
<td style="text-align:left;">
NCBI_Gene:59004,ENSEMBL:ENSMUSG00000004934
</td>
<td style="text-align:left;">
MGI:2136940
</td>
<td style="text-align:left;">
protein coding gene
</td>
<td style="text-align:left;">
protein inhibitor of activated STAT 4
</td>
</tr>
<tr>
<td style="text-align:left;">
10
</td>
<td style="text-align:left;">
gene
</td>
<td style="text-align:right;">
81.17663
</td>
<td style="text-align:right;">
81.18251
</td>
<td style="text-align:left;">
positive
</td>
<td style="text-align:left;">
MGI_C57BL6J_95288
</td>
<td style="text-align:left;">
Eef2
</td>
<td style="text-align:left;">
NCBI_Gene:13629,ENSEMBL:ENSMUSG00000034994
</td>
<td style="text-align:left;">
MGI:95288
</td>
<td style="text-align:left;">
protein coding gene
</td>
<td style="text-align:left;">
eukaryotic translation elongation factor 2
</td>
</tr>
<tr>
<td style="text-align:left;">
10
</td>
<td style="text-align:left;">
gene
</td>
<td style="text-align:right;">
81.17895
</td>
<td style="text-align:right;">
81.17901
</td>
<td style="text-align:left;">
positive
</td>
<td style="text-align:left;">
MGI_C57BL6J_3819530
</td>
<td style="text-align:left;">
Snord37
</td>
<td style="text-align:left;">
NCBI_Gene:100217454,ENSEMBL:ENSMUSG00000064441
</td>
<td style="text-align:left;">
MGI:3819530
</td>
<td style="text-align:left;">
snoRNA gene
</td>
<td style="text-align:left;">
small nucleolar RNA, C/D box 37
</td>
</tr>
<tr>
<td style="text-align:left;">
10
</td>
<td style="text-align:left;">
gene
</td>
<td style="text-align:right;">
81.18260
</td>
<td style="text-align:right;">
81.18385
</td>
<td style="text-align:left;">
negative
</td>
<td style="text-align:left;">
MGI_C57BL6J_1922144
</td>
<td style="text-align:left;">
4930442H23Rik
</td>
<td style="text-align:left;">
ENSEMBL:ENSMUSG00000053603
</td>
<td style="text-align:left;">
MGI:1922144
</td>
<td style="text-align:left;">
lncRNA gene
</td>
<td style="text-align:left;">
RIKEN cDNA 4930442H23 gene
</td>
</tr>
<tr>
<td style="text-align:left;">
10
</td>
<td style="text-align:left;">
gene
</td>
<td style="text-align:right;">
81.18299
</td>
<td style="text-align:right;">
81.19320
</td>
<td style="text-align:left;">
positive
</td>
<td style="text-align:left;">
MGI_C57BL6J_1203520
</td>
<td style="text-align:left;">
Dapk3
</td>
<td style="text-align:left;">
NCBI_Gene:13144,ENSEMBL:ENSMUSG00000034974
</td>
<td style="text-align:left;">
MGI:1203520
</td>
<td style="text-align:left;">
protein coding gene
</td>
<td style="text-align:left;">
death-associated protein kinase 3
</td>
</tr>
<tr>
<td style="text-align:left;">
10
</td>
<td style="text-align:left;">
gene
</td>
<td style="text-align:right;">
81.19461
</td>
<td style="text-align:right;">
81.21088
</td>
<td style="text-align:left;">
positive
</td>
<td style="text-align:left;">
MGI_C57BL6J_1916928
</td>
<td style="text-align:left;">
Atcayos
</td>
<td style="text-align:left;">
NCBI_Gene:69678,ENSEMBL:ENSMUSG00000085779
</td>
<td style="text-align:left;">
MGI:1916928
</td>
<td style="text-align:left;">
antisense lncRNA gene
</td>
<td style="text-align:left;">
ataxia, cerebellar, Cayman type, opposite strand
</td>
</tr>
<tr>
<td style="text-align:left;">
10
</td>
<td style="text-align:left;">
gene
</td>
<td style="text-align:right;">
81.19817
</td>
<td style="text-align:right;">
81.20221
</td>
<td style="text-align:left;">
negative
</td>
<td style="text-align:left;">
MGI_C57BL6J_1916814
</td>
<td style="text-align:left;">
Nmrk2
</td>
<td style="text-align:left;">
NCBI_Gene:69564,ENSEMBL:ENSMUSG00000004939
</td>
<td style="text-align:left;">
MGI:1916814
</td>
<td style="text-align:left;">
protein coding gene
</td>
<td style="text-align:left;">
nicotinamide riboside kinase 2
</td>
</tr>
<tr>
<td style="text-align:left;">
10
</td>
<td style="text-align:left;">
gene
</td>
<td style="text-align:right;">
81.20451
</td>
<td style="text-align:right;">
81.23083
</td>
<td style="text-align:left;">
negative
</td>
<td style="text-align:left;">
MGI_C57BL6J_2448730
</td>
<td style="text-align:left;">
Atcay
</td>
<td style="text-align:left;">
NCBI_Gene:16467,ENSEMBL:ENSMUSG00000034958
</td>
<td style="text-align:left;">
MGI:2448730
</td>
<td style="text-align:left;">
protein coding gene
</td>
<td style="text-align:left;">
ataxia, cerebellar, Cayman type
</td>
</tr>
<tr>
<td style="text-align:left;">
10
</td>
<td style="text-align:left;">
gene
</td>
<td style="text-align:right;">
81.23314
</td>
<td style="text-align:right;">
81.25212
</td>
<td style="text-align:left;">
positive
</td>
<td style="text-align:left;">
MGI_C57BL6J_2143792
</td>
<td style="text-align:left;">
Zfr2
</td>
<td style="text-align:left;">
NCBI_Gene:103406,ENSEMBL:ENSMUSG00000034949
</td>
<td style="text-align:left;">
MGI:2143792
</td>
<td style="text-align:left;">
protein coding gene
</td>
<td style="text-align:left;">
zinc finger RNA binding protein 2
</td>
</tr>
<tr>
<td style="text-align:left;">
10
</td>
<td style="text-align:left;">
gene
</td>
<td style="text-align:right;">
81.25280
</td>
<td style="text-align:right;">
81.26336
</td>
<td style="text-align:left;">
positive
</td>
<td style="text-align:left;">
MGI_C57BL6J_99259
</td>
<td style="text-align:left;">
Matk
</td>
<td style="text-align:left;">
NCBI_Gene:17179,ENSEMBL:ENSMUSG00000004933
</td>
<td style="text-align:left;">
MGI:99259
</td>
<td style="text-align:left;">
protein coding gene
</td>
<td style="text-align:left;">
megakaryocyte-associated tyrosine kinase
</td>
</tr>
<tr>
<td style="text-align:left;">
10
</td>
<td style="text-align:left;">
gene
</td>
<td style="text-align:right;">
81.26471
</td>
<td style="text-align:right;">
81.26693
</td>
<td style="text-align:left;">
negative
</td>
<td style="text-align:left;">
MGI_C57BL6J_1913297
</td>
<td style="text-align:left;">
Mrpl54
</td>
<td style="text-align:left;">
NCBI_Gene:66047,ENSEMBL:ENSMUSG00000034932
</td>
<td style="text-align:left;">
MGI:1913297
</td>
<td style="text-align:left;">
protein coding gene
</td>
<td style="text-align:left;">
mitochondrial ribosomal protein L54
</td>
</tr>
<tr>
<td style="text-align:left;">
10
</td>
<td style="text-align:left;">
gene
</td>
<td style="text-align:right;">
81.26696
</td>
<td style="text-align:right;">
81.27325
</td>
<td style="text-align:left;">
positive
</td>
<td style="text-align:left;">
MGI_C57BL6J_1888527
</td>
<td style="text-align:left;">
Apba3
</td>
<td style="text-align:left;">
NCBI_Gene:57267,ENSEMBL:ENSMUSG00000004931
</td>
<td style="text-align:left;">
MGI:1888527
</td>
<td style="text-align:left;">
protein coding gene
</td>
<td style="text-align:left;">
amyloid beta (A4) precursor protein-binding, family A, member 3
</td>
</tr>
<tr>
<td style="text-align:left;">
10
</td>
<td style="text-align:left;">
gene
</td>
<td style="text-align:right;">
81.27160
</td>
<td style="text-align:right;">
81.27169
</td>
<td style="text-align:left;">
positive
</td>
<td style="text-align:left;">
MGI_C57BL6J_4834230
</td>
<td style="text-align:left;">
Mir3057
</td>
<td style="text-align:left;">
miRBase:MI0014020,NCBI_Gene:100526465,ENSEMBL:ENSMUSG00000092781
</td>
<td style="text-align:left;">
MGI:4834230
</td>
<td style="text-align:left;">
miRNA gene
</td>
<td style="text-align:left;">
microRNA 3057
</td>
</tr>
<tr>
<td style="text-align:left;">
10
</td>
<td style="text-align:left;">
gene
</td>
<td style="text-align:right;">
81.27319
</td>
<td style="text-align:right;">
81.29158
</td>
<td style="text-align:left;">
negative
</td>
<td style="text-align:left;">
MGI_C57BL6J_1351650
</td>
<td style="text-align:left;">
Tjp3
</td>
<td style="text-align:left;">
NCBI_Gene:27375,ENSEMBL:ENSMUSG00000034917
</td>
<td style="text-align:left;">
MGI:1351650
</td>
<td style="text-align:left;">
protein coding gene
</td>
<td style="text-align:left;">
tight junction protein 3
</td>
</tr>
<tr>
<td style="text-align:left;">
10
</td>
<td style="text-align:left;">
gene
</td>
<td style="text-align:right;">
81.29296
</td>
<td style="text-align:right;">
81.31997
</td>
<td style="text-align:left;">
positive
</td>
<td style="text-align:left;">
MGI_C57BL6J_1298224
</td>
<td style="text-align:left;">
Pip5k1c
</td>
<td style="text-align:left;">
NCBI_Gene:18717,ENSEMBL:ENSMUSG00000034902
</td>
<td style="text-align:left;">
MGI:1298224
</td>
<td style="text-align:left;">
protein coding gene
</td>
<td style="text-align:left;">
phosphatidylinositol-4-phosphate 5-kinase, type 1 gamma
</td>
</tr>
<tr>
<td style="text-align:left;">
10
</td>
<td style="text-align:left;">
gene
</td>
<td style="text-align:right;">
81.30170
</td>
<td style="text-align:right;">
81.30294
</td>
<td style="text-align:left;">
negative
</td>
<td style="text-align:left;">
MGI_C57BL6J_3826564
</td>
<td style="text-align:left;">
Gm16315
</td>
<td style="text-align:left;">
NCBI_Gene:108167825,ENSEMBL:ENSMUSG00000090230
</td>
<td style="text-align:left;">
MGI:3826564
</td>
<td style="text-align:left;">
lncRNA gene
</td>
<td style="text-align:left;">
predicted gene 16315
</td>
</tr>
<tr>
<td style="text-align:left;">
10
</td>
<td style="text-align:left;">
gene
</td>
<td style="text-align:right;">
81.32110
</td>
<td style="text-align:right;">
81.32625
</td>
<td style="text-align:left;">
positive
</td>
<td style="text-align:left;">
MGI_C57BL6J_1917562
</td>
<td style="text-align:left;">
Cactin
</td>
<td style="text-align:left;">
NCBI_Gene:70312,ENSEMBL:ENSMUSG00000034889
</td>
<td style="text-align:left;">
MGI:1917562
</td>
<td style="text-align:left;">
protein coding gene
</td>
<td style="text-align:left;">
cactin, spliceosome C complex subunit
</td>
</tr>
<tr>
<td style="text-align:left;">
10
</td>
<td style="text-align:left;">
gene
</td>
<td style="text-align:right;">
81.32826
</td>
<td style="text-align:right;">
81.33518
</td>
<td style="text-align:left;">
positive
</td>
<td style="text-align:left;">
MGI_C57BL6J_98496
</td>
<td style="text-align:left;">
Tbxa2r
</td>
<td style="text-align:left;">
NCBI_Gene:21390,ENSEMBL:ENSMUSG00000034881
</td>
<td style="text-align:left;">
MGI:98496
</td>
<td style="text-align:left;">
protein coding gene
</td>
<td style="text-align:left;">
thromboxane A2 receptor
</td>
</tr>
<tr>
<td style="text-align:left;">
10
</td>
<td style="text-align:left;">
gene
</td>
<td style="text-align:right;">
81.33527
</td>
<td style="text-align:right;">
81.34374
</td>
<td style="text-align:left;">
negative
</td>
<td style="text-align:left;">
MGI_C57BL6J_2387006
</td>
<td style="text-align:left;">
Gipc3
</td>
<td style="text-align:left;">
NCBI_Gene:209047,ENSEMBL:ENSMUSG00000034872
</td>
<td style="text-align:left;">
MGI:2387006
</td>
<td style="text-align:left;">
protein coding gene
</td>
<td style="text-align:left;">
GIPC PDZ domain containing family, member 3
</td>
</tr>
<tr>
<td style="text-align:left;">
10
</td>
<td style="text-align:left;">
gene
</td>
<td style="text-align:right;">
81.34605
</td>
<td style="text-align:right;">
81.35051
</td>
<td style="text-align:left;">
negative
</td>
<td style="text-align:left;">
MGI_C57BL6J_1341190
</td>
<td style="text-align:left;">
Hmg20b
</td>
<td style="text-align:left;">
NCBI_Gene:15353,ENSEMBL:ENSMUSG00000020232
</td>
<td style="text-align:left;">
MGI:1341190
</td>
<td style="text-align:left;">
protein coding gene
</td>
<td style="text-align:left;">
high mobility group 20B
</td>
</tr>
<tr>
<td style="text-align:left;">
10
</td>
<td style="text-align:left;">
gene
</td>
<td style="text-align:right;">
81.35747
</td>
<td style="text-align:right;">
81.36623
</td>
<td style="text-align:left;">
positive
</td>
<td style="text-align:left;">
MGI_C57BL6J_3604804
</td>
<td style="text-align:left;">
Mfsd12
</td>
<td style="text-align:left;">
NCBI_Gene:73822,ENSEMBL:ENSMUSG00000034854
</td>
<td style="text-align:left;">
MGI:3604804
</td>
<td style="text-align:left;">
protein coding gene
</td>
<td style="text-align:left;">
major facilitator superfamily domain containing 12
</td>
</tr>
<tr>
<td style="text-align:left;">
10
</td>
<td style="text-align:left;">
gene
</td>
<td style="text-align:right;">
81.36307
</td>
<td style="text-align:right;">
81.36582
</td>
<td style="text-align:left;">
negative
</td>
<td style="text-align:left;">
MGI_C57BL6J_1921072
</td>
<td style="text-align:left;">
4930404N11Rik
</td>
<td style="text-align:left;">
NCBI_Gene:432479,ENSEMBL:ENSMUSG00000020234
</td>
<td style="text-align:left;">
MGI:1921072
</td>
<td style="text-align:left;">
protein coding gene
</td>
<td style="text-align:left;">
RIKEN cDNA 4930404N11 gene
</td>
</tr>
<tr>
<td style="text-align:left;">
10
</td>
<td style="text-align:left;">
gene
</td>
<td style="text-align:right;">
81.36622
</td>
<td style="text-align:right;">
81.37852
</td>
<td style="text-align:left;">
negative
</td>
<td style="text-align:left;">
MGI_C57BL6J_1926790
</td>
<td style="text-align:left;">
Fzr1
</td>
<td style="text-align:left;">
NCBI_Gene:56371,ENSEMBL:ENSMUSG00000020235
</td>
<td style="text-align:left;">
MGI:1926790
</td>
<td style="text-align:left;">
protein coding gene
</td>
<td style="text-align:left;">
fizzy and cell division cycle 20 related 1
</td>
</tr>
<tr>
<td style="text-align:left;">
10
</td>
<td style="text-align:left;">
gene
</td>
<td style="text-align:right;">
81.38443
</td>
<td style="text-align:right;">
81.38835
</td>
<td style="text-align:left;">
positive
</td>
<td style="text-align:left;">
MGI_C57BL6J_1915964
</td>
<td style="text-align:left;">
Dohh
</td>
<td style="text-align:left;">
NCBI_Gene:102115,ENSEMBL:ENSMUSG00000078440
</td>
<td style="text-align:left;">
MGI:1915964
</td>
<td style="text-align:left;">
protein coding gene
</td>
<td style="text-align:left;">
deoxyhypusine hydroxylase/monooxygenase
</td>
</tr>
<tr>
<td style="text-align:left;">
10
</td>
<td style="text-align:left;">
gene
</td>
<td style="text-align:right;">
81.38447
</td>
<td style="text-align:right;">
81.39156
</td>
<td style="text-align:left;">
positive
</td>
<td style="text-align:left;">
MGI_C57BL6J_6098103
</td>
<td style="text-align:left;">
Gm48551
</td>
<td style="text-align:left;">
ENSEMBL:ENSMUSG00000113262
</td>
<td style="text-align:left;">
MGI:6098103
</td>
<td style="text-align:left;">
protein coding gene
</td>
<td style="text-align:left;">
predicted gene, 48551
</td>
</tr>
<tr>
<td style="text-align:left;">
10
</td>
<td style="text-align:left;">
gene
</td>
<td style="text-align:right;">
81.38635
</td>
<td style="text-align:right;">
81.38641
</td>
<td style="text-align:left;">
positive
</td>
<td style="text-align:left;">
MGI_C57BL6J_5531170
</td>
<td style="text-align:left;">
Mir6913
</td>
<td style="text-align:left;">
miRBase:MI0022760,NCBI_Gene:102466763,ENSEMBL:ENSMUSG00000098811
</td>
<td style="text-align:left;">
MGI:5531170
</td>
<td style="text-align:left;">
miRNA gene
</td>
<td style="text-align:left;">
microRNA 6913
</td>
</tr>
<tr>
<td style="text-align:left;">
10
</td>
<td style="text-align:left;">
gene
</td>
<td style="text-align:right;">
81.38900
</td>
<td style="text-align:right;">
81.39127
</td>
<td style="text-align:left;">
positive
</td>
<td style="text-align:left;">
MGI_C57BL6J_6098105
</td>
<td style="text-align:left;">
Gm48552
</td>
<td style="text-align:left;">
ENSEMBL:ENSMUSG00000114004
</td>
<td style="text-align:left;">
MGI:6098105
</td>
<td style="text-align:left;">
protein coding gene
</td>
<td style="text-align:left;">
predicted gene, 48552
</td>
</tr>
<tr>
<td style="text-align:left;">
10
</td>
<td style="text-align:left;">
gene
</td>
<td style="text-align:right;">
81.39305
</td>
<td style="text-align:right;">
81.39539
</td>
<td style="text-align:left;">
positive
</td>
<td style="text-align:left;">
MGI_C57BL6J_1919523
</td>
<td style="text-align:left;">
Smim24
</td>
<td style="text-align:left;">
NCBI_Gene:72273,ENSEMBL:ENSMUSG00000078439
</td>
<td style="text-align:left;">
MGI:1919523
</td>
<td style="text-align:left;">
protein coding gene
</td>
<td style="text-align:left;">
small integral membrane protein 24
</td>
</tr>
<tr>
<td style="text-align:left;">
10
</td>
<td style="text-align:left;">
gene
</td>
<td style="text-align:right;">
81.39619
</td>
<td style="text-align:right;">
81.45564
</td>
<td style="text-align:left;">
negative
</td>
<td style="text-align:left;">
MGI_C57BL6J_109591
</td>
<td style="text-align:left;">
Nfic
</td>
<td style="text-align:left;">
NCBI_Gene:18029,ENSEMBL:ENSMUSG00000055053
</td>
<td style="text-align:left;">
MGI:109591
</td>
<td style="text-align:left;">
protein coding gene
</td>
<td style="text-align:left;">
nuclear factor I/C
</td>
</tr>
<tr>
<td style="text-align:left;">
10
</td>
<td style="text-align:left;">
gene
</td>
<td style="text-align:right;">
81.40958
</td>
<td style="text-align:right;">
81.41160
</td>
<td style="text-align:left;">
positive
</td>
<td style="text-align:left;">
MGI_C57BL6J_3801749
</td>
<td style="text-align:left;">
Gm16104
</td>
<td style="text-align:left;">
NCBI_Gene:102634174,ENSEMBL:ENSMUSG00000085327
</td>
<td style="text-align:left;">
MGI:3801749
</td>
<td style="text-align:left;">
lncRNA gene
</td>
<td style="text-align:left;">
predicted gene 16104
</td>
</tr>
<tr>
<td style="text-align:left;">
10
</td>
<td style="text-align:left;">
gene
</td>
<td style="text-align:right;">
81.41624
</td>
<td style="text-align:right;">
81.41630
</td>
<td style="text-align:left;">
negative
</td>
<td style="text-align:left;">
MGI_C57BL6J_5562728
</td>
<td style="text-align:left;">
Mir1191b
</td>
<td style="text-align:left;">
miRBase:MI0025021,NCBI_Gene:102465778,ENSEMBL:ENSMUSG00000105379
</td>
<td style="text-align:left;">
MGI:5562728
</td>
<td style="text-align:left;">
miRNA gene
</td>
<td style="text-align:left;">
microRNA 1191b
</td>
</tr>
<tr>
<td style="text-align:left;">
10
</td>
<td style="text-align:left;">
gene
</td>
<td style="text-align:right;">
81.43147
</td>
<td style="text-align:right;">
81.43192
</td>
<td style="text-align:left;">
positive
</td>
<td style="text-align:left;">
MGI_C57BL6J_5590923
</td>
<td style="text-align:left;">
Gm31764
</td>
<td style="text-align:left;">
NCBI_Gene:102634102
</td>
<td style="text-align:left;">
MGI:5590923
</td>
<td style="text-align:left;">
lncRNA gene
</td>
<td style="text-align:left;">
predicted gene, 31764
</td>
</tr>
<tr>
<td style="text-align:left;">
10
</td>
<td style="text-align:left;">
gene
</td>
<td style="text-align:right;">
81.45923
</td>
<td style="text-align:right;">
81.48272
</td>
<td style="text-align:left;">
negative
</td>
<td style="text-align:left;">
MGI_C57BL6J_2442333
</td>
<td style="text-align:left;">
Celf5
</td>
<td style="text-align:left;">
NCBI_Gene:319586,ENSEMBL:ENSMUSG00000034818
</td>
<td style="text-align:left;">
MGI:2442333
</td>
<td style="text-align:left;">
protein coding gene
</td>
<td style="text-align:left;">
CUGBP, Elav-like family member 5
</td>
</tr>
<tr>
<td style="text-align:left;">
10
</td>
<td style="text-align:left;">
gene
</td>
<td style="text-align:right;">
81.46116
</td>
<td style="text-align:right;">
81.46297
</td>
<td style="text-align:left;">
positive
</td>
<td style="text-align:left;">
MGI_C57BL6J_3801737
</td>
<td style="text-align:left;">
Gm16105
</td>
<td style="text-align:left;">
ENSEMBL:ENSMUSG00000084849
</td>
<td style="text-align:left;">
MGI:3801737
</td>
<td style="text-align:left;">
lncRNA gene
</td>
<td style="text-align:left;">
predicted gene 16105
</td>
</tr>
<tr>
<td style="text-align:left;">
10
</td>
<td style="text-align:left;">
gene
</td>
<td style="text-align:right;">
81.48625
</td>
<td style="text-align:right;">
81.49669
</td>
<td style="text-align:left;">
negative
</td>
<td style="text-align:left;">
MGI_C57BL6J_1926081
</td>
<td style="text-align:left;">
Ncln
</td>
<td style="text-align:left;">
NCBI_Gene:103425,ENSEMBL:ENSMUSG00000020238
</td>
<td style="text-align:left;">
MGI:1926081
</td>
<td style="text-align:left;">
protein coding gene
</td>
<td style="text-align:left;">
nicalin
</td>
</tr>
<tr>
<td style="text-align:left;">
10
</td>
<td style="text-align:left;">
gene
</td>
<td style="text-align:right;">
81.49774
</td>
<td style="text-align:right;">
81.50014
</td>
<td style="text-align:left;">
negative
</td>
<td style="text-align:left;">
MGI_C57BL6J_1333809
</td>
<td style="text-align:left;">
S1pr4
</td>
<td style="text-align:left;">
NCBI_Gene:13611,ENSEMBL:ENSMUSG00000044199
</td>
<td style="text-align:left;">
MGI:1333809
</td>
<td style="text-align:left;">
protein coding gene
</td>
<td style="text-align:left;">
sphingosine-1-phosphate receptor 4
</td>
</tr>
<tr>
<td style="text-align:left;">
10
</td>
<td style="text-align:left;">
gene
</td>
<td style="text-align:right;">
81.50231
</td>
<td style="text-align:right;">
81.52448
</td>
<td style="text-align:left;">
negative
</td>
<td style="text-align:left;">
MGI_C57BL6J_95770
</td>
<td style="text-align:left;">
Gna15
</td>
<td style="text-align:left;">
NCBI_Gene:14676,ENSEMBL:ENSMUSG00000034792
</td>
<td style="text-align:left;">
MGI:95770
</td>
<td style="text-align:left;">
protein coding gene
</td>
<td style="text-align:left;">
guanine nucleotide binding protein, alpha 15
</td>
</tr>
<tr>
<td style="text-align:left;">
10
</td>
<td style="text-align:left;">
pseudogene
</td>
<td style="text-align:right;">
81.50447
</td>
<td style="text-align:right;">
81.50477
</td>
<td style="text-align:left;">
positive
</td>
<td style="text-align:left;">
MGI_C57BL6J_3801781
</td>
<td style="text-align:left;">
Gm16106
</td>
<td style="text-align:left;">
ENSEMBL:ENSMUSG00000082828
</td>
<td style="text-align:left;">
MGI:3801781
</td>
<td style="text-align:left;">
pseudogene
</td>
<td style="text-align:left;">
predicted gene 16106
</td>
</tr>
<tr>
<td style="text-align:left;">
10
</td>
<td style="text-align:left;">
gene
</td>
<td style="text-align:right;">
81.50462
</td>
<td style="text-align:right;">
81.50472
</td>
<td style="text-align:left;">
positive
</td>
<td style="text-align:left;">
MGI_C57BL6J_5455372
</td>
<td style="text-align:left;">
Gm25595
</td>
<td style="text-align:left;">
ENSEMBL:ENSMUSG00000076228
</td>
<td style="text-align:left;">
MGI:5455372
</td>
<td style="text-align:left;">
miRNA gene
</td>
<td style="text-align:left;">
predicted gene, 25595
</td>
</tr>
<tr>
<td style="text-align:left;">
10
</td>
<td style="text-align:left;">
gene
</td>
<td style="text-align:right;">
81.52541
</td>
<td style="text-align:right;">
81.52699
</td>
<td style="text-align:left;">
positive
</td>
<td style="text-align:left;">
MGI_C57BL6J_6096808
</td>
<td style="text-align:left;">
Gm47697
</td>
<td style="text-align:left;">
ENSEMBL:ENSMUSG00000112350
</td>
<td style="text-align:left;">
MGI:6096808
</td>
<td style="text-align:left;">
lncRNA gene
</td>
<td style="text-align:left;">
predicted gene, 47697
</td>
</tr>
<tr>
<td style="text-align:left;">
10
</td>
<td style="text-align:left;">
gene
</td>
<td style="text-align:right;">
81.52872
</td>
<td style="text-align:right;">
81.54519
</td>
<td style="text-align:left;">
negative
</td>
<td style="text-align:left;">
MGI_C57BL6J_95766
</td>
<td style="text-align:left;">
Gna11
</td>
<td style="text-align:left;">
NCBI_Gene:14672,ENSEMBL:ENSMUSG00000034781
</td>
<td style="text-align:left;">
MGI:95766
</td>
<td style="text-align:left;">
protein coding gene
</td>
<td style="text-align:left;">
guanine nucleotide binding protein, alpha 11
</td>
</tr>
<tr>
<td style="text-align:left;">
10
</td>
<td style="text-align:left;">
gene
</td>
<td style="text-align:right;">
81.55944
</td>
<td style="text-align:right;">
81.56637
</td>
<td style="text-align:left;">
positive
</td>
<td style="text-align:left;">
MGI_C57BL6J_95806
</td>
<td style="text-align:left;">
Tle5
</td>
<td style="text-align:left;">
NCBI_Gene:14797,ENSEMBL:ENSMUSG00000054452
</td>
<td style="text-align:left;">
MGI:95806
</td>
<td style="text-align:left;">
protein coding gene
</td>
<td style="text-align:left;">
TLE family member 5, transcriptional modulator
</td>
</tr>
<tr>
<td style="text-align:left;">
10
</td>
<td style="text-align:left;">
gene
</td>
<td style="text-align:right;">
81.56905
</td>
<td style="text-align:right;">
81.57179
</td>
<td style="text-align:left;">
positive
</td>
<td style="text-align:left;">
MGI_C57BL6J_5591095
</td>
<td style="text-align:left;">
Gm31936
</td>
<td style="text-align:left;">
NCBI_Gene:102634328
</td>
<td style="text-align:left;">
MGI:5591095
</td>
<td style="text-align:left;">
lncRNA gene
</td>
<td style="text-align:left;">
predicted gene, 31936
</td>
</tr>
<tr>
<td style="text-align:left;">
10
</td>
<td style="text-align:left;">
gene
</td>
<td style="text-align:right;">
81.57261
</td>
<td style="text-align:right;">
81.57594
</td>
<td style="text-align:left;">
negative
</td>
<td style="text-align:left;">
MGI_C57BL6J_3801972
</td>
<td style="text-align:left;">
Gm15917
</td>
<td style="text-align:left;">
ENSEMBL:ENSMUSG00000090034
</td>
<td style="text-align:left;">
MGI:3801972
</td>
<td style="text-align:left;">
lncRNA gene
</td>
<td style="text-align:left;">
predicted gene 15917
</td>
</tr>
<tr>
<td style="text-align:left;">
10
</td>
<td style="text-align:left;">
gene
</td>
<td style="text-align:right;">
81.57456
</td>
<td style="text-align:right;">
81.59258
</td>
<td style="text-align:left;">
positive
</td>
<td style="text-align:left;">
MGI_C57BL6J_104635
</td>
<td style="text-align:left;">
Tle2
</td>
<td style="text-align:left;">
NCBI_Gene:21886,ENSEMBL:ENSMUSG00000034771
</td>
<td style="text-align:left;">
MGI:104635
</td>
<td style="text-align:left;">
protein coding gene
</td>
<td style="text-align:left;">
transducin-like enhancer of split 2
</td>
</tr>
<tr>
<td style="text-align:left;">
10
</td>
<td style="text-align:left;">
gene
</td>
<td style="text-align:right;">
81.59090
</td>
<td style="text-align:right;">
81.60119
</td>
<td style="text-align:left;">
negative
</td>
<td style="text-align:left;">
MGI_C57BL6J_2149593
</td>
<td style="text-align:left;">
Tle6
</td>
<td style="text-align:left;">
NCBI_Gene:114606,ENSEMBL:ENSMUSG00000034758
</td>
<td style="text-align:left;">
MGI:2149593
</td>
<td style="text-align:left;">
protein coding gene
</td>
<td style="text-align:left;">
transducin-like enhancer of split 6
</td>
</tr>
<tr>
<td style="text-align:left;">
10
</td>
<td style="text-align:left;">
gene
</td>
<td style="text-align:right;">
81.59451
</td>
<td style="text-align:right;">
81.59910
</td>
<td style="text-align:left;">
positive
</td>
<td style="text-align:left;">
MGI_C57BL6J_5591183
</td>
<td style="text-align:left;">
Gm32024
</td>
<td style="text-align:left;">
NCBI_Gene:102634443
</td>
<td style="text-align:left;">
MGI:5591183
</td>
<td style="text-align:left;">
lncRNA gene
</td>
<td style="text-align:left;">
predicted gene, 32024
</td>
</tr>
<tr>
<td style="text-align:left;">
10
</td>
<td style="text-align:left;">
gene
</td>
<td style="text-align:right;">
81.60627
</td>
<td style="text-align:right;">
81.62070
</td>
<td style="text-align:left;">
positive
</td>
<td style="text-align:left;">
MGI_C57BL6J_2670983
</td>
<td style="text-align:left;">
BC025920
</td>
<td style="text-align:left;">
NCBI_Gene:268319,ENSEMBL:ENSMUSG00000074862
</td>
<td style="text-align:left;">
MGI:2670983
</td>
<td style="text-align:left;">
protein coding gene
</td>
<td style="text-align:left;">
cDNA sequence BC025920
</td>
</tr>
<tr>
<td style="text-align:left;">
10
</td>
<td style="text-align:left;">
gene
</td>
<td style="text-align:right;">
81.60637
</td>
<td style="text-align:right;">
81.60979
</td>
<td style="text-align:left;">
negative
</td>
<td style="text-align:left;">
MGI_C57BL6J_5477390
</td>
<td style="text-align:left;">
Gm26896
</td>
<td style="text-align:left;">
ENSEMBL:ENSMUSG00000097718
</td>
<td style="text-align:left;">
MGI:5477390
</td>
<td style="text-align:left;">
lncRNA gene
</td>
<td style="text-align:left;">
predicted gene, 26896
</td>
</tr>
<tr>
<td style="text-align:left;">
10
</td>
<td style="text-align:left;">
gene
</td>
<td style="text-align:right;">
81.62179
</td>
<td style="text-align:right;">
81.62780
</td>
<td style="text-align:left;">
negative
</td>
<td style="text-align:left;">
MGI_C57BL6J_1354161
</td>
<td style="text-align:left;">
Sirt6
</td>
<td style="text-align:left;">
NCBI_Gene:50721,ENSEMBL:ENSMUSG00000034748
</td>
<td style="text-align:left;">
MGI:1354161
</td>
<td style="text-align:left;">
protein coding gene
</td>
<td style="text-align:left;">
sirtuin 6
</td>
</tr>
<tr>
<td style="text-align:left;">
10
</td>
<td style="text-align:left;">
gene
</td>
<td style="text-align:right;">
81.62769
</td>
<td style="text-align:right;">
81.64761
</td>
<td style="text-align:left;">
positive
</td>
<td style="text-align:left;">
MGI_C57BL6J_1890394
</td>
<td style="text-align:left;">
Ankrd24
</td>
<td style="text-align:left;">
NCBI_Gene:70615,ENSEMBL:ENSMUSG00000054708
</td>
<td style="text-align:left;">
MGI:1890394
</td>
<td style="text-align:left;">
protein coding gene
</td>
<td style="text-align:left;">
ankyrin repeat domain 24
</td>
</tr>
<tr>
<td style="text-align:left;">
10
</td>
<td style="text-align:left;">
gene
</td>
<td style="text-align:right;">
81.64906
</td>
<td style="text-align:right;">
81.66736
</td>
<td style="text-align:left;">
positive
</td>
<td style="text-align:left;">
MGI_C57BL6J_3809656
</td>
<td style="text-align:left;">
Gm10778
</td>
<td style="text-align:left;">
NCBI_Gene:100233208,ENSEMBL:ENSMUSG00000096856
</td>
<td style="text-align:left;">
MGI:3809656
</td>
<td style="text-align:left;">
protein coding gene
</td>
<td style="text-align:left;">
predicted gene 10778
</td>
</tr>
<tr>
<td style="text-align:left;">
10
</td>
<td style="text-align:left;">
pseudogene
</td>
<td style="text-align:right;">
81.67912
</td>
<td style="text-align:right;">
81.68051
</td>
<td style="text-align:left;">
positive
</td>
<td style="text-align:left;">
MGI_C57BL6J_6096529
</td>
<td style="text-align:left;">
Gm47527
</td>
<td style="text-align:left;">
ENSEMBL:ENSMUSG00000112585
</td>
<td style="text-align:left;">
MGI:6096529
</td>
<td style="text-align:left;">
pseudogene
</td>
<td style="text-align:left;">
predicted gene, 47527
</td>
</tr>
<tr>
<td style="text-align:left;">
10
</td>
<td style="text-align:left;">
pseudogene
</td>
<td style="text-align:right;">
81.69368
</td>
<td style="text-align:right;">
81.69524
</td>
<td style="text-align:left;">
negative
</td>
<td style="text-align:left;">
MGI_C57BL6J_3645531
</td>
<td style="text-align:left;">
Gm8112
</td>
<td style="text-align:left;">
NCBI_Gene:108167775,ENSEMBL:ENSMUSG00000112001
</td>
<td style="text-align:left;">
MGI:3645531
</td>
<td style="text-align:left;">
pseudogene
</td>
<td style="text-align:left;">
predicted gene 8112
</td>
</tr>
<tr>
<td style="text-align:left;">
10
</td>
<td style="text-align:left;">
gene
</td>
<td style="text-align:right;">
81.70419
</td>
<td style="text-align:right;">
81.72669
</td>
<td style="text-align:left;">
positive
</td>
<td style="text-align:left;">
MGI_C57BL6J_1920860
</td>
<td style="text-align:left;">
Zfp433
</td>
<td style="text-align:left;">
NCBI_Gene:73610,ENSEMBL:ENSMUSG00000096795
</td>
<td style="text-align:left;">
MGI:1920860
</td>
<td style="text-align:left;">
protein coding gene
</td>
<td style="text-align:left;">
zinc finger protein 433
</td>
</tr>
<tr>
<td style="text-align:left;">
10
</td>
<td style="text-align:left;">
pseudogene
</td>
<td style="text-align:right;">
81.71460
</td>
<td style="text-align:right;">
81.71650
</td>
<td style="text-align:left;">
positive
</td>
<td style="text-align:left;">
MGI_C57BL6J_3647617
</td>
<td style="text-align:left;">
Gm8249
</td>
<td style="text-align:left;">
NCBI_Gene:105245234,ENSEMBL:ENSMUSG00000112061
</td>
<td style="text-align:left;">
MGI:3647617
</td>
<td style="text-align:left;">
pseudogene
</td>
<td style="text-align:left;">
predicted gene 8249
</td>
</tr>
<tr>
<td style="text-align:left;">
10
</td>
<td style="text-align:left;">
pseudogene
</td>
<td style="text-align:right;">
81.71704
</td>
<td style="text-align:right;">
81.71756
</td>
<td style="text-align:left;">
negative
</td>
<td style="text-align:left;">
MGI_C57BL6J_6096532
</td>
<td style="text-align:left;">
Gm47529
</td>
<td style="text-align:left;">
ENSEMBL:ENSMUSG00000112339
</td>
<td style="text-align:left;">
MGI:6096532
</td>
<td style="text-align:left;">
pseudogene
</td>
<td style="text-align:left;">
predicted gene, 47529
</td>
</tr>
<tr>
<td style="text-align:left;">
10
</td>
<td style="text-align:left;">
pseudogene
</td>
<td style="text-align:right;">
81.72790
</td>
<td style="text-align:right;">
81.72909
</td>
<td style="text-align:left;">
positive
</td>
<td style="text-align:left;">
MGI_C57BL6J_3642939
</td>
<td style="text-align:left;">
Gm5173
</td>
<td style="text-align:left;">
NCBI_Gene:382388,ENSEMBL:ENSMUSG00000112621
</td>
<td style="text-align:left;">
MGI:3642939
</td>
<td style="text-align:left;">
pseudogene
</td>
<td style="text-align:left;">
predicted gene 5173
</td>
</tr>
<tr>
<td style="text-align:left;">
10
</td>
<td style="text-align:left;">
gene
</td>
<td style="text-align:right;">
81.74282
</td>
<td style="text-align:right;">
81.77116
</td>
<td style="text-align:left;">
negative
</td>
<td style="text-align:left;">
MGI_C57BL6J_3809197
</td>
<td style="text-align:left;">
Gm3055
</td>
<td style="text-align:left;">
NCBI_Gene:100040944,ENSEMBL:ENSMUSG00000094622
</td>
<td style="text-align:left;">
MGI:3809197
</td>
<td style="text-align:left;">
protein coding gene
</td>
<td style="text-align:left;">
predicted gene 3055
</td>
</tr>
<tr>
<td style="text-align:left;">
10
</td>
<td style="text-align:left;">
pseudogene
</td>
<td style="text-align:right;">
81.80443
</td>
<td style="text-align:right;">
81.80570
</td>
<td style="text-align:left;">
positive
</td>
<td style="text-align:left;">
MGI_C57BL6J_6096545
</td>
<td style="text-align:left;">
Gm47537
</td>
<td style="text-align:left;">
ENSEMBL:ENSMUSG00000112877
</td>
<td style="text-align:left;">
MGI:6096545
</td>
<td style="text-align:left;">
pseudogene
</td>
<td style="text-align:left;">
predicted gene, 47537
</td>
</tr>
<tr>
<td style="text-align:left;">
10
</td>
<td style="text-align:left;">
gene
</td>
<td style="text-align:right;">
81.81194
</td>
<td style="text-align:right;">
81.82649
</td>
<td style="text-align:left;">
positive
</td>
<td style="text-align:left;">
MGI_C57BL6J_3644409
</td>
<td style="text-align:left;">
Gm4767
</td>
<td style="text-align:left;">
NCBI_Gene:210583,ENSEMBL:ENSMUSG00000094076
</td>
<td style="text-align:left;">
MGI:3644409
</td>
<td style="text-align:left;">
protein coding gene
</td>
<td style="text-align:left;">
predicted gene 4767
</td>
</tr>
<tr>
<td style="text-align:left;">
10
</td>
<td style="text-align:left;">
pseudogene
</td>
<td style="text-align:right;">
81.81843
</td>
<td style="text-align:right;">
81.81898
</td>
<td style="text-align:left;">
negative
</td>
<td style="text-align:left;">
MGI_C57BL6J_3647363
</td>
<td style="text-align:left;">
Gm8264
</td>
<td style="text-align:left;">
NCBI_Gene:666738,ENSEMBL:ENSMUSG00000112740
</td>
<td style="text-align:left;">
MGI:3647363
</td>
<td style="text-align:left;">
pseudogene
</td>
<td style="text-align:left;">
predicted gene 8264
</td>
</tr>
<tr>
<td style="text-align:left;">
10
</td>
<td style="text-align:left;">
pseudogene
</td>
<td style="text-align:right;">
81.85277
</td>
<td style="text-align:right;">
81.85332
</td>
<td style="text-align:left;">
negative
</td>
<td style="text-align:left;">
MGI_C57BL6J_3647467
</td>
<td style="text-align:left;">
Gm8137
</td>
<td style="text-align:left;">
NCBI_Gene:666501,ENSEMBL:ENSMUSG00000112697
</td>
<td style="text-align:left;">
MGI:3647467
</td>
<td style="text-align:left;">
pseudogene
</td>
<td style="text-align:left;">
predicted gene 8137
</td>
</tr>
<tr>
<td style="text-align:left;">
10
</td>
<td style="text-align:left;">
gene
</td>
<td style="text-align:right;">
81.86344
</td>
<td style="text-align:right;">
81.88291
</td>
<td style="text-align:left;">
positive
</td>
<td style="text-align:left;">
MGI_C57BL6J_5591846
</td>
<td style="text-align:left;">
Gm32687
</td>
<td style="text-align:left;">
NCBI_Gene:102635315,ENSEMBL:ENSMUSG00000112640
</td>
<td style="text-align:left;">
MGI:5591846
</td>
<td style="text-align:left;">
protein coding gene
</td>
<td style="text-align:left;">
predicted gene, 32687
</td>
</tr>
<tr>
<td style="text-align:left;">
10
</td>
<td style="text-align:left;">
pseudogene
</td>
<td style="text-align:right;">
81.87371
</td>
<td style="text-align:right;">
81.87557
</td>
<td style="text-align:left;">
positive
</td>
<td style="text-align:left;">
MGI_C57BL6J_3648016
</td>
<td style="text-align:left;">
Gm8274
</td>
<td style="text-align:left;">
NCBI_Gene:105245237,ENSEMBL:ENSMUSG00000112196
</td>
<td style="text-align:left;">
MGI:3648016
</td>
<td style="text-align:left;">
pseudogene
</td>
<td style="text-align:left;">
predicted gene 8274
</td>
</tr>
<tr>
<td style="text-align:left;">
10
</td>
<td style="text-align:left;">
pseudogene
</td>
<td style="text-align:right;">
81.87615
</td>
<td style="text-align:right;">
81.87667
</td>
<td style="text-align:left;">
negative
</td>
<td style="text-align:left;">
MGI_C57BL6J_6096547
</td>
<td style="text-align:left;">
Gm47539
</td>
<td style="text-align:left;">
ENSEMBL:ENSMUSG00000112760
</td>
<td style="text-align:left;">
MGI:6096547
</td>
<td style="text-align:left;">
pseudogene
</td>
<td style="text-align:left;">
predicted gene, 47539
</td>
</tr>
<tr>
<td style="text-align:left;">
10
</td>
<td style="text-align:left;">
pseudogene
</td>
<td style="text-align:right;">
81.88699
</td>
<td style="text-align:right;">
81.88818
</td>
<td style="text-align:left;">
positive
</td>
<td style="text-align:left;">
MGI_C57BL6J_3644502
</td>
<td style="text-align:left;">
Gm8153
</td>
<td style="text-align:left;">
NCBI_Gene:666537,ENSEMBL:ENSMUSG00000112311
</td>
<td style="text-align:left;">
MGI:3644502
</td>
<td style="text-align:left;">
pseudogene
</td>
<td style="text-align:left;">
predicted gene 8153
</td>
</tr>
<tr>
<td style="text-align:left;">
10
</td>
<td style="text-align:left;">
gene
</td>
<td style="text-align:right;">
81.90191
</td>
<td style="text-align:right;">
81.93127
</td>
<td style="text-align:left;">
negative
</td>
<td style="text-align:left;">
MGI_C57BL6J_3696710
</td>
<td style="text-align:left;">
Zfp781
</td>
<td style="text-align:left;">
NCBI_Gene:331188,ENSEMBL:ENSMUSG00000096718
</td>
<td style="text-align:left;">
MGI:3696710
</td>
<td style="text-align:left;">
protein coding gene
</td>
<td style="text-align:left;">
zinc finger protein 781
</td>
</tr>
<tr>
<td style="text-align:left;">
10
</td>
<td style="text-align:left;">
pseudogene
</td>
<td style="text-align:right;">
81.96380
</td>
<td style="text-align:right;">
81.96509
</td>
<td style="text-align:left;">
positive
</td>
<td style="text-align:left;">
MGI_C57BL6J_6096560
</td>
<td style="text-align:left;">
Gm47546
</td>
<td style="text-align:left;">
ENSEMBL:ENSMUSG00000112205
</td>
<td style="text-align:left;">
MGI:6096560
</td>
<td style="text-align:left;">
pseudogene
</td>
<td style="text-align:left;">
predicted gene, 47546
</td>
</tr>
<tr>
<td style="text-align:left;">
10
</td>
<td style="text-align:left;">
gene
</td>
<td style="text-align:right;">
81.97809
</td>
<td style="text-align:right;">
81.99536
</td>
<td style="text-align:left;">
positive
</td>
<td style="text-align:left;">
MGI_C57BL6J_5011987
</td>
<td style="text-align:left;">
Gm19802
</td>
<td style="text-align:left;">
NCBI_Gene:105245236
</td>
<td style="text-align:left;">
MGI:5011987
</td>
<td style="text-align:left;">
protein coding gene
</td>
<td style="text-align:left;">
predicted gene, 19802
</td>
</tr>
<tr>
<td style="text-align:left;">
10
</td>
<td style="text-align:left;">
pseudogene
</td>
<td style="text-align:right;">
81.98590
</td>
<td style="text-align:right;">
81.98645
</td>
<td style="text-align:left;">
negative
</td>
<td style="text-align:left;">
MGI_C57BL6J_3647883
</td>
<td style="text-align:left;">
Gm8158
</td>
<td style="text-align:left;">
NCBI_Gene:666548,ENSEMBL:ENSMUSG00000112051
</td>
<td style="text-align:left;">
MGI:3647883
</td>
<td style="text-align:left;">
pseudogene
</td>
<td style="text-align:left;">
predicted gene 8158
</td>
</tr>
<tr>
<td style="text-align:left;">
10
</td>
<td style="text-align:left;">
gene
</td>
<td style="text-align:right;">
81.99597
</td>
<td style="text-align:right;">
82.01164
</td>
<td style="text-align:left;">
positive
</td>
<td style="text-align:left;">
MGI_C57BL6J_3642231
</td>
<td style="text-align:left;">
Gm10777
</td>
<td style="text-align:left;">
NCBI_Gene:102635133
</td>
<td style="text-align:left;">
MGI:3642231
</td>
<td style="text-align:left;">
lncRNA gene
</td>
<td style="text-align:left;">
predicted gene 10777
</td>
</tr>
<tr>
<td style="text-align:left;">
10
</td>
<td style="text-align:left;">
pseudogene
</td>
<td style="text-align:right;">
82.00829
</td>
<td style="text-align:right;">
82.00981
</td>
<td style="text-align:left;">
positive
</td>
<td style="text-align:left;">
MGI_C57BL6J_3648461
</td>
<td style="text-align:left;">
Gm8290
</td>
<td style="text-align:left;">
NCBI_Gene:666790,ENSEMBL:ENSMUSG00000111985
</td>
<td style="text-align:left;">
MGI:3648461
</td>
<td style="text-align:left;">
pseudogene
</td>
<td style="text-align:left;">
predicted gene 8290
</td>
</tr>
<tr>
<td style="text-align:left;">
10
</td>
<td style="text-align:left;">
pseudogene
</td>
<td style="text-align:right;">
82.00948
</td>
<td style="text-align:right;">
82.00980
</td>
<td style="text-align:left;">
negative
</td>
<td style="text-align:left;">
MGI_C57BL6J_5825831
</td>
<td style="text-align:left;">
Gm46194
</td>
<td style="text-align:left;">
NCBI_Gene:108167776,ENSEMBL:ENSMUSG00000112502
</td>
<td style="text-align:left;">
MGI:5825831
</td>
<td style="text-align:left;">
pseudogene
</td>
<td style="text-align:left;">
predicted gene, 46194
</td>
</tr>
<tr>
<td style="text-align:left;">
10
</td>
<td style="text-align:left;">
gene
</td>
<td style="text-align:right;">
82.01170
</td>
<td style="text-align:right;">
82.01723
</td>
<td style="text-align:left;">
positive
</td>
<td style="text-align:left;">
MGI_C57BL6J_5592537
</td>
<td style="text-align:left;">
Gm33378
</td>
<td style="text-align:left;">
NCBI_Gene:102636264,ENSEMBL:ENSMUSG00000112790
</td>
<td style="text-align:left;">
MGI:5592537
</td>
<td style="text-align:left;">
protein coding gene
</td>
<td style="text-align:left;">
predicted gene, 33378
</td>
</tr>
<tr>
<td style="text-align:left;">
10
</td>
<td style="text-align:left;">
pseudogene
</td>
<td style="text-align:right;">
82.02053
</td>
<td style="text-align:right;">
82.02171
</td>
<td style="text-align:left;">
positive
</td>
<td style="text-align:left;">
MGI_C57BL6J_3644928
</td>
<td style="text-align:left;">
Gm8170
</td>
<td style="text-align:left;">
NCBI_Gene:666571,ENSEMBL:ENSMUSG00000112381
</td>
<td style="text-align:left;">
MGI:3644928
</td>
<td style="text-align:left;">
pseudogene
</td>
<td style="text-align:left;">
predicted gene 8170
</td>
</tr>
<tr>
<td style="text-align:left;">
10
</td>
<td style="text-align:left;">
gene
</td>
<td style="text-align:right;">
82.04464
</td>
<td style="text-align:right;">
82.04804
</td>
<td style="text-align:left;">
negative
</td>
<td style="text-align:left;">
MGI_C57BL6J_3034336
</td>
<td style="text-align:left;">
AL117821
</td>
<td style="text-align:left;">
NA
</td>
<td style="text-align:left;">
NA
</td>
<td style="text-align:left;">
unclassified gene
</td>
<td style="text-align:left;">
expressed sequence AL117821
</td>
</tr>
<tr>
<td style="text-align:left;">
10
</td>
<td style="text-align:left;">
gene
</td>
<td style="text-align:right;">
82.04645
</td>
<td style="text-align:right;">
82.04805
</td>
<td style="text-align:left;">
negative
</td>
<td style="text-align:left;">
MGI_C57BL6J_5825832
</td>
<td style="text-align:left;">
Gm46195
</td>
<td style="text-align:left;">
NCBI_Gene:108167777
</td>
<td style="text-align:left;">
MGI:5825832
</td>
<td style="text-align:left;">
lncRNA gene
</td>
<td style="text-align:left;">
predicted gene, 46195
</td>
</tr>
<tr>
<td style="text-align:left;">
10
</td>
<td style="text-align:left;">
gene
</td>
<td style="text-align:right;">
82.04809
</td>
<td style="text-align:right;">
82.07773
</td>
<td style="text-align:left;">
positive
</td>
<td style="text-align:left;">
MGI_C57BL6J_3040689
</td>
<td style="text-align:left;">
Zfp873
</td>
<td style="text-align:left;">
NCBI_Gene:408062,ENSEMBL:ENSMUSG00000061371
</td>
<td style="text-align:left;">
MGI:3040689
</td>
<td style="text-align:left;">
protein coding gene
</td>
<td style="text-align:left;">
zinc finger protein 873
</td>
</tr>
<tr>
<td style="text-align:left;">
10
</td>
<td style="text-align:left;">
pseudogene
</td>
<td style="text-align:right;">
82.05611
</td>
<td style="text-align:right;">
82.05724
</td>
<td style="text-align:left;">
negative
</td>
<td style="text-align:left;">
MGI_C57BL6J_5010741
</td>
<td style="text-align:left;">
Gm18556
</td>
<td style="text-align:left;">
NCBI_Gene:100417356,ENSEMBL:ENSMUSG00000112373
</td>
<td style="text-align:left;">
MGI:5010741
</td>
<td style="text-align:left;">
pseudogene
</td>
<td style="text-align:left;">
predicted gene, 18556
</td>
</tr>
<tr>
<td style="text-align:left;">
10
</td>
<td style="text-align:left;">
gene
</td>
<td style="text-align:right;">
82.08301
</td>
<td style="text-align:right;">
82.08362
</td>
<td style="text-align:left;">
negative
</td>
<td style="text-align:left;">
MGI_C57BL6J_3041223
</td>
<td style="text-align:left;">
D430050E20Rik
</td>
<td style="text-align:left;">
NA
</td>
<td style="text-align:left;">
NA
</td>
<td style="text-align:left;">
unclassified non-coding RNA gene
</td>
<td style="text-align:left;">
RIKEN cDNA D430050E20 gene
</td>
</tr>
<tr>
<td style="text-align:left;">
10
</td>
<td style="text-align:left;">
gene
</td>
<td style="text-align:right;">
82.08332
</td>
<td style="text-align:right;">
82.11122
</td>
<td style="text-align:left;">
positive
</td>
<td style="text-align:left;">
MGI_C57BL6J_2670978
</td>
<td style="text-align:left;">
BC024063
</td>
<td style="text-align:left;">
NCBI_Gene:666584,ENSEMBL:ENSMUSG00000112160
</td>
<td style="text-align:left;">
MGI:2670978
</td>
<td style="text-align:left;">
protein coding gene
</td>
<td style="text-align:left;">
cDNA sequence BC024063
</td>
</tr>
<tr>
<td style="text-align:left;">
10
</td>
<td style="text-align:left;">
gene
</td>
<td style="text-align:right;">
82.08341
</td>
<td style="text-align:right;">
82.08567
</td>
<td style="text-align:left;">
positive
</td>
<td style="text-align:left;">
MGI_C57BL6J_3041160
</td>
<td style="text-align:left;">
4933439G19Rik
</td>
<td style="text-align:left;">
NA
</td>
<td style="text-align:left;">
NA
</td>
<td style="text-align:left;">
unclassified non-coding RNA gene
</td>
<td style="text-align:left;">
RIKEN cDNA 4933439G19 gene
</td>
</tr>
<tr>
<td style="text-align:left;">
10
</td>
<td style="text-align:left;">
gene
</td>
<td style="text-align:right;">
82.12668
</td>
<td style="text-align:right;">
82.12829
</td>
<td style="text-align:left;">
negative
</td>
<td style="text-align:left;">
MGI_C57BL6J_1918064
</td>
<td style="text-align:left;">
4921506L19Rik
</td>
<td style="text-align:left;">
NA
</td>
<td style="text-align:left;">
NA
</td>
<td style="text-align:left;">
unclassified gene
</td>
<td style="text-align:left;">
RIKEN cDNA 4921506L19 gene
</td>
</tr>
<tr>
<td style="text-align:left;">
10
</td>
<td style="text-align:left;">
gene
</td>
<td style="text-align:right;">
82.12801
</td>
<td style="text-align:right;">
82.15306
</td>
<td style="text-align:left;">
positive
</td>
<td style="text-align:left;">
MGI_C57BL6J_2143755
</td>
<td style="text-align:left;">
AU041133
</td>
<td style="text-align:left;">
NCBI_Gene:216177,ENSEMBL:ENSMUSG00000078435
</td>
<td style="text-align:left;">
MGI:2143755
</td>
<td style="text-align:left;">
protein coding gene
</td>
<td style="text-align:left;">
expressed sequence AU041133
</td>
</tr>
<tr>
<td style="text-align:left;">
10
</td>
<td style="text-align:left;">
gene
</td>
<td style="text-align:right;">
82.15710
</td>
<td style="text-align:right;">
82.16046
</td>
<td style="text-align:left;">
positive
</td>
<td style="text-align:left;">
MGI_C57BL6J_6096602
</td>
<td style="text-align:left;">
Gm47572
</td>
<td style="text-align:left;">
ENSEMBL:ENSMUSG00000112922
</td>
<td style="text-align:left;">
MGI:6096602
</td>
<td style="text-align:left;">
unclassified gene
</td>
<td style="text-align:left;">
predicted gene, 47572
</td>
</tr>
<tr>
<td style="text-align:left;">
10
</td>
<td style="text-align:left;">
gene
</td>
<td style="text-align:right;">
82.16425
</td>
<td style="text-align:right;">
82.16678
</td>
<td style="text-align:left;">
positive
</td>
<td style="text-align:left;">
MGI_C57BL6J_6096604
</td>
<td style="text-align:left;">
Gm47573
</td>
<td style="text-align:left;">
ENSEMBL:ENSMUSG00000112847
</td>
<td style="text-align:left;">
MGI:6096604
</td>
<td style="text-align:left;">
unclassified gene
</td>
<td style="text-align:left;">
predicted gene, 47573
</td>
</tr>
<tr>
<td style="text-align:left;">
10
</td>
<td style="text-align:left;">
gene
</td>
<td style="text-align:right;">
82.17763
</td>
<td style="text-align:right;">
82.19129
</td>
<td style="text-align:left;">
positive
</td>
<td style="text-align:left;">
MGI_C57BL6J_5623605
</td>
<td style="text-align:left;">
Gm40720
</td>
<td style="text-align:left;">
NCBI_Gene:105245239
</td>
<td style="text-align:left;">
MGI:5623605
</td>
<td style="text-align:left;">
lncRNA gene
</td>
<td style="text-align:left;">
predicted gene, 40720
</td>
</tr>
<tr>
<td style="text-align:left;">
10
</td>
<td style="text-align:left;">
gene
</td>
<td style="text-align:right;">
82.17824
</td>
<td style="text-align:right;">
82.20149
</td>
<td style="text-align:left;">
positive
</td>
<td style="text-align:left;">
MGI_C57BL6J_3801804
</td>
<td style="text-align:left;">
Gm15608
</td>
<td style="text-align:left;">
NCBI_Gene:105245241,ENSEMBL:ENSMUSG00000112110
</td>
<td style="text-align:left;">
MGI:3801804
</td>
<td style="text-align:left;">
lncRNA gene
</td>
<td style="text-align:left;">
predicted gene 15608
</td>
</tr>
<tr>
<td style="text-align:left;">
10
</td>
<td style="text-align:left;">
gene
</td>
<td style="text-align:right;">
82.17842
</td>
<td style="text-align:right;">
82.18109
</td>
<td style="text-align:left;">
positive
</td>
<td style="text-align:left;">
MGI_C57BL6J_3642233
</td>
<td style="text-align:left;">
Gm10776
</td>
<td style="text-align:left;">
NA
</td>
<td style="text-align:left;">
NA
</td>
<td style="text-align:left;">
unclassified gene
</td>
<td style="text-align:left;">
predicted gene 10776
</td>
</tr>
<tr>
<td style="text-align:left;">
10
</td>
<td style="text-align:left;">
pseudogene
</td>
<td style="text-align:right;">
82.21905
</td>
<td style="text-align:right;">
82.22446
</td>
<td style="text-align:left;">
positive
</td>
<td style="text-align:left;">
MGI_C57BL6J_3782155
</td>
<td style="text-align:left;">
Gm3982
</td>
<td style="text-align:left;">
NCBI_Gene:105245240
</td>
<td style="text-align:left;">
MGI:3782155
</td>
<td style="text-align:left;">
pseudogene
</td>
<td style="text-align:left;">
predicted gene 3982
</td>
</tr>
<tr>
<td style="text-align:left;">
10
</td>
<td style="text-align:left;">
gene
</td>
<td style="text-align:right;">
82.22457
</td>
<td style="text-align:right;">
82.24128
</td>
<td style="text-align:left;">
negative
</td>
<td style="text-align:left;">
MGI_C57BL6J_3621440
</td>
<td style="text-align:left;">
Zfp938
</td>
<td style="text-align:left;">
NCBI_Gene:237411,ENSEMBL:ENSMUSG00000062931
</td>
<td style="text-align:left;">
MGI:3621440
</td>
<td style="text-align:left;">
protein coding gene
</td>
<td style="text-align:left;">
zinc finger protein 938
</td>
</tr>
<tr>
<td style="text-align:left;">
10
</td>
<td style="text-align:left;">
pseudogene
</td>
<td style="text-align:right;">
82.26645
</td>
<td style="text-align:right;">
82.26687
</td>
<td style="text-align:left;">
negative
</td>
<td style="text-align:left;">
MGI_C57BL6J_6096302
</td>
<td style="text-align:left;">
Gm47382
</td>
<td style="text-align:left;">
ENSEMBL:ENSMUSG00000111930
</td>
<td style="text-align:left;">
MGI:6096302
</td>
<td style="text-align:left;">
pseudogene
</td>
<td style="text-align:left;">
predicted gene, 47382
</td>
</tr>
<tr>
<td style="text-align:left;">
10
</td>
<td style="text-align:left;">
gene
</td>
<td style="text-align:right;">
82.28211
</td>
<td style="text-align:right;">
82.31658
</td>
<td style="text-align:left;">
negative
</td>
<td style="text-align:left;">
MGI_C57BL6J_3045298
</td>
<td style="text-align:left;">
4932415D10Rik
</td>
<td style="text-align:left;">
NCBI_Gene:102635990,ENSEMBL:ENSMUSG00000044581
</td>
<td style="text-align:left;">
MGI:3045298
</td>
<td style="text-align:left;">
protein coding gene
</td>
<td style="text-align:left;">
RIKEN cDNA 4932415D10 gene
</td>
</tr>
<tr>
<td style="text-align:left;">
10
</td>
<td style="text-align:left;">
gene
</td>
<td style="text-align:right;">
82.31538
</td>
<td style="text-align:right;">
82.31656
</td>
<td style="text-align:left;">
negative
</td>
<td style="text-align:left;">
MGI_C57BL6J_1918439
</td>
<td style="text-align:left;">
4933426I03Rik
</td>
<td style="text-align:left;">
NA
</td>
<td style="text-align:left;">
NA
</td>
<td style="text-align:left;">
unclassified gene
</td>
<td style="text-align:left;">
RIKEN cDNA 4933426I03 gene
</td>
</tr>
<tr>
<td style="text-align:left;">
10
</td>
<td style="text-align:left;">
gene
</td>
<td style="text-align:right;">
82.35422
</td>
<td style="text-align:right;">
82.42288
</td>
<td style="text-align:left;">
positive
</td>
<td style="text-align:left;">
MGI_C57BL6J_3643133
</td>
<td style="text-align:left;">
Gm4924
</td>
<td style="text-align:left;">
NCBI_Gene:237412,ENSEMBL:ENSMUSG00000073427
</td>
<td style="text-align:left;">
MGI:3643133
</td>
<td style="text-align:left;">
protein coding gene
</td>
<td style="text-align:left;">
predicted gene 4924
</td>
</tr>
<tr>
<td style="text-align:left;">
10
</td>
<td style="text-align:left;">
pseudogene
</td>
<td style="text-align:right;">
82.43070
</td>
<td style="text-align:right;">
82.43239
</td>
<td style="text-align:left;">
positive
</td>
<td style="text-align:left;">
MGI_C57BL6J_3646272
</td>
<td style="text-align:left;">
Gm8188
</td>
<td style="text-align:left;">
NCBI_Gene:666611,ENSEMBL:ENSMUSG00000112163
</td>
<td style="text-align:left;">
MGI:3646272
</td>
<td style="text-align:left;">
pseudogene
</td>
<td style="text-align:left;">
predicted gene 8188
</td>
</tr>
<tr>
<td style="text-align:left;">
10
</td>
<td style="text-align:left;">
pseudogene
</td>
<td style="text-align:right;">
82.44262
</td>
<td style="text-align:right;">
82.44294
</td>
<td style="text-align:left;">
positive
</td>
<td style="text-align:left;">
MGI_C57BL6J_6096993
</td>
<td style="text-align:left;">
Gm47809
</td>
<td style="text-align:left;">
ENSEMBL:ENSMUSG00000112070
</td>
<td style="text-align:left;">
MGI:6096993
</td>
<td style="text-align:left;">
pseudogene
</td>
<td style="text-align:left;">
predicted gene, 47809
</td>
</tr>
<tr>
<td style="text-align:left;">
10
</td>
<td style="text-align:left;">
gene
</td>
<td style="text-align:right;">
82.48653
</td>
<td style="text-align:right;">
82.49262
</td>
<td style="text-align:left;">
negative
</td>
<td style="text-align:left;">
MGI_C57BL6J_2686399
</td>
<td style="text-align:left;">
Gm1553
</td>
<td style="text-align:left;">
NCBI_Gene:432480,ENSEMBL:ENSMUSG00000094186
</td>
<td style="text-align:left;">
MGI:2686399
</td>
<td style="text-align:left;">
protein coding gene
</td>
<td style="text-align:left;">
predicted gene 1553
</td>
</tr>
<tr>
<td style="text-align:left;">
10
</td>
<td style="text-align:left;">
pseudogene
</td>
<td style="text-align:right;">
82.56677
</td>
<td style="text-align:right;">
82.56751
</td>
<td style="text-align:left;">
positive
</td>
<td style="text-align:left;">
MGI_C57BL6J_5011371
</td>
<td style="text-align:left;">
Gm19186
</td>
<td style="text-align:left;">
NCBI_Gene:100418403,ENSEMBL:ENSMUSG00000112560
</td>
<td style="text-align:left;">
MGI:5011371
</td>
<td style="text-align:left;">
pseudogene
</td>
<td style="text-align:left;">
predicted gene, 19186
</td>
</tr>
<tr>
<td style="text-align:left;">
10
</td>
<td style="text-align:left;">
pseudogene
</td>
<td style="text-align:right;">
82.58538
</td>
<td style="text-align:right;">
82.58754
</td>
<td style="text-align:left;">
positive
</td>
<td style="text-align:left;">
MGI_C57BL6J_3645307
</td>
<td style="text-align:left;">
Ap3m1-ps
</td>
<td style="text-align:left;">
NCBI_Gene:544716,ENSEMBL:ENSMUSG00000094683
</td>
<td style="text-align:left;">
MGI:3645307
</td>
<td style="text-align:left;">
pseudogene
</td>
<td style="text-align:left;">
adaptor-related protein complex 3, mu 1 subunit, pseudogene
</td>
</tr>
<tr>
<td style="text-align:left;">
10
</td>
<td style="text-align:left;">
gene
</td>
<td style="text-align:right;">
82.61326
</td>
<td style="text-align:right;">
82.61895
</td>
<td style="text-align:left;">
negative
</td>
<td style="text-align:left;">
MGI_C57BL6J_5623606
</td>
<td style="text-align:left;">
Gm40721
</td>
<td style="text-align:left;">
NCBI_Gene:105245242
</td>
<td style="text-align:left;">
MGI:5623606
</td>
<td style="text-align:left;">
lncRNA gene
</td>
<td style="text-align:left;">
predicted gene, 40721
</td>
</tr>
<tr>
<td style="text-align:left;">
10
</td>
<td style="text-align:left;">
gene
</td>
<td style="text-align:right;">
82.61985
</td>
<td style="text-align:right;">
82.62323
</td>
<td style="text-align:left;">
negative
</td>
<td style="text-align:left;">
MGI_C57BL6J_3698433
</td>
<td style="text-align:left;">
1190007I07Rik
</td>
<td style="text-align:left;">
NCBI_Gene:544717,ENSEMBL:ENSMUSG00000063320
</td>
<td style="text-align:left;">
MGI:3698433
</td>
<td style="text-align:left;">
protein coding gene
</td>
<td style="text-align:left;">
RIKEN cDNA 1190007I07 gene
</td>
</tr>
<tr>
<td style="text-align:left;">
10
</td>
<td style="text-align:left;">
gene
</td>
<td style="text-align:right;">
82.62982
</td>
<td style="text-align:right;">
82.65080
</td>
<td style="text-align:left;">
positive
</td>
<td style="text-align:left;">
MGI_C57BL6J_108247
</td>
<td style="text-align:left;">
Tdg
</td>
<td style="text-align:left;">
NCBI_Gene:21665,ENSEMBL:ENSMUSG00000034674
</td>
<td style="text-align:left;">
MGI:108247
</td>
<td style="text-align:left;">
protein coding gene
</td>
<td style="text-align:left;">
thymine DNA glycosylase
</td>
</tr>
<tr>
<td style="text-align:left;">
10
</td>
<td style="text-align:left;">
gene
</td>
<td style="text-align:right;">
82.65043
</td>
<td style="text-align:right;">
82.69412
</td>
<td style="text-align:left;">
negative
</td>
<td style="text-align:left;">
MGI_C57BL6J_1922032
</td>
<td style="text-align:left;">
Glt8d2
</td>
<td style="text-align:left;">
NCBI_Gene:74782,ENSEMBL:ENSMUSG00000020251
</td>
<td style="text-align:left;">
MGI:1922032
</td>
<td style="text-align:left;">
protein coding gene
</td>
<td style="text-align:left;">
glycosyltransferase 8 domain containing 2
</td>
</tr>
<tr>
<td style="text-align:left;">
10
</td>
<td style="text-align:left;">
gene
</td>
<td style="text-align:right;">
82.69616
</td>
<td style="text-align:right;">
82.79587
</td>
<td style="text-align:left;">
positive
</td>
<td style="text-align:left;">
MGI_C57BL6J_1915183
</td>
<td style="text-align:left;">
Hcfc2
</td>
<td style="text-align:left;">
NCBI_Gene:67933,ENSEMBL:ENSMUSG00000020246
</td>
<td style="text-align:left;">
MGI:1915183
</td>
<td style="text-align:left;">
protein coding gene
</td>
<td style="text-align:left;">
host cell factor C2
</td>
</tr>
<tr>
<td style="text-align:left;">
10
</td>
<td style="text-align:left;">
gene
</td>
<td style="text-align:right;">
82.74870
</td>
<td style="text-align:right;">
82.76414
</td>
<td style="text-align:left;">
negative
</td>
<td style="text-align:left;">
MGI_C57BL6J_97317
</td>
<td style="text-align:left;">
Nfyb
</td>
<td style="text-align:left;">
NCBI_Gene:18045,ENSEMBL:ENSMUSG00000020248
</td>
<td style="text-align:left;">
MGI:97317
</td>
<td style="text-align:left;">
protein coding gene
</td>
<td style="text-align:left;">
nuclear transcription factor-Y beta
</td>
</tr>
<tr>
<td style="text-align:left;">
10
</td>
<td style="text-align:left;">
pseudogene
</td>
<td style="text-align:right;">
82.76974
</td>
<td style="text-align:right;">
82.76990
</td>
<td style="text-align:left;">
negative
</td>
<td style="text-align:left;">
MGI_C57BL6J_6096583
</td>
<td style="text-align:left;">
Gm47560
</td>
<td style="text-align:left;">
ENSEMBL:ENSMUSG00000112107
</td>
<td style="text-align:left;">
MGI:6096583
</td>
<td style="text-align:left;">
pseudogene
</td>
<td style="text-align:left;">
predicted gene, 47560
</td>
</tr>
<tr>
<td style="text-align:left;">
10
</td>
<td style="text-align:left;">
gene
</td>
<td style="text-align:right;">
82.77132
</td>
<td style="text-align:right;">
82.77778
</td>
<td style="text-align:left;">
positive
</td>
<td style="text-align:left;">
MGI_C57BL6J_6096584
</td>
<td style="text-align:left;">
Gm47561
</td>
<td style="text-align:left;">
ENSEMBL:ENSMUSG00000112286
</td>
<td style="text-align:left;">
MGI:6096584
</td>
<td style="text-align:left;">
lncRNA gene
</td>
<td style="text-align:left;">
predicted gene, 47561
</td>
</tr>
<tr>
<td style="text-align:left;">
10
</td>
<td style="text-align:left;">
gene
</td>
<td style="text-align:right;">
82.81212
</td>
<td style="text-align:right;">
82.82424
</td>
<td style="text-align:left;">
positive
</td>
<td style="text-align:left;">
MGI_C57BL6J_1917253
</td>
<td style="text-align:left;">
1700028I16Rik
</td>
<td style="text-align:left;">
NCBI_Gene:70003,ENSEMBL:ENSMUSG00000056821
</td>
<td style="text-align:left;">
MGI:1917253
</td>
<td style="text-align:left;">
lncRNA gene
</td>
<td style="text-align:left;">
RIKEN cDNA 1700028I16 gene
</td>
</tr>
<tr>
<td style="text-align:left;">
10
</td>
<td style="text-align:left;">
gene
</td>
<td style="text-align:right;">
82.83395
</td>
<td style="text-align:right;">
82.89772
</td>
<td style="text-align:left;">
positive
</td>
<td style="text-align:left;">
MGI_C57BL6J_1354175
</td>
<td style="text-align:left;">
Txnrd1
</td>
<td style="text-align:left;">
NCBI_Gene:50493,ENSEMBL:ENSMUSG00000020250
</td>
<td style="text-align:left;">
MGI:1354175
</td>
<td style="text-align:left;">
protein coding gene
</td>
<td style="text-align:left;">
thioredoxin reductase 1
</td>
</tr>
<tr>
<td style="text-align:left;">
10
</td>
<td style="text-align:left;">
gene
</td>
<td style="text-align:right;">
82.84534
</td>
<td style="text-align:right;">
82.85921
</td>
<td style="text-align:left;">
negative
</td>
<td style="text-align:left;">
MGI_C57BL6J_5621445
</td>
<td style="text-align:left;">
Gm38560
</td>
<td style="text-align:left;">
NCBI_Gene:102641476,ENSEMBL:ENSMUSG00000112375
</td>
<td style="text-align:left;">
MGI:5621445
</td>
<td style="text-align:left;">
lncRNA gene
</td>
<td style="text-align:left;">
predicted gene, 38560
</td>
</tr>
<tr>
<td style="text-align:left;">
10
</td>
<td style="text-align:left;">
gene
</td>
<td style="text-align:right;">
82.85125
</td>
<td style="text-align:right;">
82.85265
</td>
<td style="text-align:left;">
negative
</td>
<td style="text-align:left;">
MGI_C57BL6J_6095792
</td>
<td style="text-align:left;">
Gm47073
</td>
<td style="text-align:left;">
ENSEMBL:ENSMUSG00000112889
</td>
<td style="text-align:left;">
MGI:6095792
</td>
<td style="text-align:left;">
unclassified gene
</td>
<td style="text-align:left;">
predicted gene, 47073
</td>
</tr>
<tr>
<td style="text-align:left;">
10
</td>
<td style="text-align:left;">
gene
</td>
<td style="text-align:right;">
82.86663
</td>
<td style="text-align:right;">
82.86793
</td>
<td style="text-align:left;">
positive
</td>
<td style="text-align:left;">
MGI_C57BL6J_1913591
</td>
<td style="text-align:left;">
Eid3
</td>
<td style="text-align:left;">
NCBI_Gene:66341,ENSEMBL:ENSMUSG00000109864
</td>
<td style="text-align:left;">
MGI:1913591
</td>
<td style="text-align:left;">
protein coding gene
</td>
<td style="text-align:left;">
EP300 interacting inhibitor of differentiation 3
</td>
</tr>
<tr>
<td style="text-align:left;">
10
</td>
<td style="text-align:left;">
gene
</td>
<td style="text-align:right;">
82.89138
</td>
<td style="text-align:right;">
82.89502
</td>
<td style="text-align:left;">
positive
</td>
<td style="text-align:left;">
MGI_C57BL6J_2442548
</td>
<td style="text-align:left;">
9130221J17Rik
</td>
<td style="text-align:left;">
NA
</td>
<td style="text-align:left;">
NA
</td>
<td style="text-align:left;">
unclassified gene
</td>
<td style="text-align:left;">
RIKEN cDNA 9130221J17 gene
</td>
</tr>
<tr>
<td style="text-align:left;">
10
</td>
<td style="text-align:left;">
gene
</td>
<td style="text-align:right;">
82.92700
</td>
<td style="text-align:right;">
82.93464
</td>
<td style="text-align:left;">
negative
</td>
<td style="text-align:left;">
MGI_C57BL6J_5592940
</td>
<td style="text-align:left;">
Gm33781
</td>
<td style="text-align:left;">
NCBI_Gene:102636812
</td>
<td style="text-align:left;">
MGI:5592940
</td>
<td style="text-align:left;">
lncRNA gene
</td>
<td style="text-align:left;">
predicted gene, 33781
</td>
</tr>
<tr>
<td style="text-align:left;">
10
</td>
<td style="text-align:left;">
pseudogene
</td>
<td style="text-align:right;">
82.95433
</td>
<td style="text-align:right;">
82.95508
</td>
<td style="text-align:left;">
positive
</td>
<td style="text-align:left;">
MGI_C57BL6J_3648850
</td>
<td style="text-align:left;">
Gm4799
</td>
<td style="text-align:left;">
NCBI_Gene:216185,ENSEMBL:ENSMUSG00000071151
</td>
<td style="text-align:left;">
MGI:3648850
</td>
<td style="text-align:left;">
pseudogene
</td>
<td style="text-align:left;">
predicted gene 4799
</td>
</tr>
<tr>
<td style="text-align:left;">
10
</td>
<td style="text-align:left;">
gene
</td>
<td style="text-align:right;">
82.98538
</td>
<td style="text-align:right;">
83.19590
</td>
<td style="text-align:left;">
positive
</td>
<td style="text-align:left;">
MGI_C57BL6J_1927166
</td>
<td style="text-align:left;">
Chst11
</td>
<td style="text-align:left;">
NCBI_Gene:58250,ENSEMBL:ENSMUSG00000034612
</td>
<td style="text-align:left;">
MGI:1927166
</td>
<td style="text-align:left;">
protein coding gene
</td>
<td style="text-align:left;">
carbohydrate sulfotransferase 11
</td>
</tr>
<tr>
<td style="text-align:left;">
10
</td>
<td style="text-align:left;">
gene
</td>
<td style="text-align:right;">
83.02776
</td>
<td style="text-align:right;">
83.03169
</td>
<td style="text-align:left;">
negative
</td>
<td style="text-align:left;">
MGI_C57BL6J_3641797
</td>
<td style="text-align:left;">
Gm10773
</td>
<td style="text-align:left;">
ENSEMBL:ENSMUSG00000112084
</td>
<td style="text-align:left;">
MGI:3641797
</td>
<td style="text-align:left;">
protein coding gene
</td>
<td style="text-align:left;">
predicted gene 10773
</td>
</tr>
<tr>
<td style="text-align:left;">
10
</td>
<td style="text-align:left;">
gene
</td>
<td style="text-align:right;">
83.13915
</td>
<td style="text-align:right;">
83.14204
</td>
<td style="text-align:left;">
negative
</td>
<td style="text-align:left;">
MGI_C57BL6J_1916665
</td>
<td style="text-align:left;">
1700025N21Rik
</td>
<td style="text-align:left;">
ENSEMBL:ENSMUSG00000090200
</td>
<td style="text-align:left;">
MGI:1916665
</td>
<td style="text-align:left;">
lncRNA gene
</td>
<td style="text-align:left;">
RIKEN cDNA 1700025N21 gene
</td>
</tr>
<tr>
<td style="text-align:left;">
10
</td>
<td style="text-align:left;">
gene
</td>
<td style="text-align:right;">
83.15802
</td>
<td style="text-align:right;">
83.15813
</td>
<td style="text-align:left;">
negative
</td>
<td style="text-align:left;">
MGI_C57BL6J_5452899
</td>
<td style="text-align:left;">
Gm23122
</td>
<td style="text-align:left;">
ENSEMBL:ENSMUSG00000088270
</td>
<td style="text-align:left;">
MGI:5452899
</td>
<td style="text-align:left;">
snRNA gene
</td>
<td style="text-align:left;">
predicted gene, 23122
</td>
</tr>
<tr>
<td style="text-align:left;">
10
</td>
<td style="text-align:left;">
gene
</td>
<td style="text-align:right;">
83.23085
</td>
<td style="text-align:right;">
83.33833
</td>
<td style="text-align:left;">
negative
</td>
<td style="text-align:left;">
MGI_C57BL6J_2442940
</td>
<td style="text-align:left;">
Slc41a2
</td>
<td style="text-align:left;">
NCBI_Gene:338365,ENSEMBL:ENSMUSG00000034591
</td>
<td style="text-align:left;">
MGI:2442940
</td>
<td style="text-align:left;">
protein coding gene
</td>
<td style="text-align:left;">
solute carrier family 41, member 2
</td>
</tr>
<tr>
<td style="text-align:left;">
10
</td>
<td style="text-align:left;">
pseudogene
</td>
<td style="text-align:right;">
83.33272
</td>
<td style="text-align:right;">
83.33336
</td>
<td style="text-align:left;">
positive
</td>
<td style="text-align:left;">
MGI_C57BL6J_6095950
</td>
<td style="text-align:left;">
Gm47170
</td>
<td style="text-align:left;">
ENSEMBL:ENSMUSG00000112577
</td>
<td style="text-align:left;">
MGI:6095950
</td>
<td style="text-align:left;">
pseudogene
</td>
<td style="text-align:left;">
predicted gene, 47170
</td>
</tr>
<tr>
<td style="text-align:left;">
10
</td>
<td style="text-align:left;">
gene
</td>
<td style="text-align:right;">
83.33870
</td>
<td style="text-align:right;">
83.33963
</td>
<td style="text-align:left;">
positive
</td>
<td style="text-align:left;">
MGI_C57BL6J_5623607
</td>
<td style="text-align:left;">
Gm40722
</td>
<td style="text-align:left;">
NCBI_Gene:105245243
</td>
<td style="text-align:left;">
MGI:5623607
</td>
<td style="text-align:left;">
lncRNA gene
</td>
<td style="text-align:left;">
predicted gene, 40722
</td>
</tr>
<tr>
<td style="text-align:left;">
10
</td>
<td style="text-align:left;">
gene
</td>
<td style="text-align:right;">
83.34395
</td>
<td style="text-align:right;">
83.35139
</td>
<td style="text-align:left;">
positive
</td>
<td style="text-align:left;">
MGI_C57BL6J_5623608
</td>
<td style="text-align:left;">
Gm40723
</td>
<td style="text-align:left;">
NCBI_Gene:105245244,ENSEMBL:ENSMUSG00000112134
</td>
<td style="text-align:left;">
MGI:5623608
</td>
<td style="text-align:left;">
lncRNA gene
</td>
<td style="text-align:left;">
predicted gene, 40723
</td>
</tr>
<tr>
<td style="text-align:left;">
10
</td>
<td style="text-align:left;">
gene
</td>
<td style="text-align:right;">
83.36022
</td>
<td style="text-align:right;">
83.48851
</td>
<td style="text-align:left;">
positive
</td>
<td style="text-align:left;">
MGI_C57BL6J_106381
</td>
<td style="text-align:left;">
D10Wsu102e
</td>
<td style="text-align:left;">
NCBI_Gene:28109,ENSEMBL:ENSMUSG00000020255
</td>
<td style="text-align:left;">
MGI:106381
</td>
<td style="text-align:left;">
protein coding gene
</td>
<td style="text-align:left;">
DNA segment, Chr 10, Wayne State University 102, expressed
</td>
</tr>
<tr>
<td style="text-align:left;">
10
</td>
<td style="text-align:left;">
pseudogene
</td>
<td style="text-align:right;">
83.47800
</td>
<td style="text-align:right;">
83.48125
</td>
<td style="text-align:left;">
negative
</td>
<td style="text-align:left;">
MGI_C57BL6J_3648990
</td>
<td style="text-align:left;">
Gm8270
</td>
<td style="text-align:left;">
NCBI_Gene:102636894
</td>
<td style="text-align:left;">
MGI:3648990
</td>
<td style="text-align:left;">
pseudogene
</td>
<td style="text-align:left;">
predicted gene 8270
</td>
</tr>
<tr>
<td style="text-align:left;">
10
</td>
<td style="text-align:left;">
gene
</td>
<td style="text-align:right;">
83.48745
</td>
<td style="text-align:right;">
83.53415
</td>
<td style="text-align:left;">
negative
</td>
<td style="text-align:left;">
MGI_C57BL6J_2444680
</td>
<td style="text-align:left;">
Aldh1l2
</td>
<td style="text-align:left;">
NCBI_Gene:216188,ENSEMBL:ENSMUSG00000020256
</td>
<td style="text-align:left;">
MGI:2444680
</td>
<td style="text-align:left;">
protein coding gene
</td>
<td style="text-align:left;">
aldehyde dehydrogenase 1 family, member L2
</td>
</tr>
<tr>
<td style="text-align:left;">
10
</td>
<td style="text-align:left;">
gene
</td>
<td style="text-align:right;">
83.54375
</td>
<td style="text-align:right;">
83.59647
</td>
<td style="text-align:left;">
positive
</td>
<td style="text-align:left;">
MGI_C57BL6J_2441787
</td>
<td style="text-align:left;">
Washc4
</td>
<td style="text-align:left;">
NCBI_Gene:319277,ENSEMBL:ENSMUSG00000034560
</td>
<td style="text-align:left;">
MGI:2441787
</td>
<td style="text-align:left;">
protein coding gene
</td>
<td style="text-align:left;">
WASH complex subunit 4
</td>
</tr>
<tr>
<td style="text-align:left;">
10
</td>
<td style="text-align:left;">
gene
</td>
<td style="text-align:right;">
83.60003
</td>
<td style="text-align:right;">
83.64886
</td>
<td style="text-align:left;">
negative
</td>
<td style="text-align:left;">
MGI_C57BL6J_2384914
</td>
<td style="text-align:left;">
Appl2
</td>
<td style="text-align:left;">
NCBI_Gene:216190,ENSEMBL:ENSMUSG00000020263
</td>
<td style="text-align:left;">
MGI:2384914
</td>
<td style="text-align:left;">
protein coding gene
</td>
<td style="text-align:left;">
adaptor protein, phosphotyrosine interaction, PH domain and leucine zipper containing 2
</td>
</tr>
<tr>
<td style="text-align:left;">
10
</td>
<td style="text-align:left;">
pseudogene
</td>
<td style="text-align:right;">
83.65927
</td>
<td style="text-align:right;">
83.66062
</td>
<td style="text-align:left;">
positive
</td>
<td style="text-align:left;">
MGI_C57BL6J_6096068
</td>
<td style="text-align:left;">
Gm47244
</td>
<td style="text-align:left;">
ENSEMBL:ENSMUSG00000112072
</td>
<td style="text-align:left;">
MGI:6096068
</td>
<td style="text-align:left;">
pseudogene
</td>
<td style="text-align:left;">
predicted gene, 47244
</td>
</tr>
<tr>
<td style="text-align:left;">
10
</td>
<td style="text-align:left;">
gene
</td>
<td style="text-align:right;">
83.72266
</td>
<td style="text-align:right;">
83.76276
</td>
<td style="text-align:left;">
positive
</td>
<td style="text-align:left;">
MGI_C57BL6J_1917034
</td>
<td style="text-align:left;">
1500009L16Rik
</td>
<td style="text-align:left;">
NCBI_Gene:69784,ENSEMBL:ENSMUSG00000087651
</td>
<td style="text-align:left;">
MGI:1917034
</td>
<td style="text-align:left;">
protein coding gene
</td>
<td style="text-align:left;">
RIKEN cDNA 1500009L16 gene
</td>
</tr>
<tr>
<td style="text-align:left;">
10
</td>
<td style="text-align:left;">
pseudogene
</td>
<td style="text-align:right;">
83.81016
</td>
<td style="text-align:right;">
83.81157
</td>
<td style="text-align:left;">
negative
</td>
<td style="text-align:left;">
MGI_C57BL6J_6096069
</td>
<td style="text-align:left;">
Gm47245
</td>
<td style="text-align:left;">
ENSEMBL:ENSMUSG00000112077
</td>
<td style="text-align:left;">
MGI:6096069
</td>
<td style="text-align:left;">
pseudogene
</td>
<td style="text-align:left;">
predicted gene, 47245
</td>
</tr>
<tr>
<td style="text-align:left;">
10
</td>
<td style="text-align:left;">
pseudogene
</td>
<td style="text-align:right;">
83.90185
</td>
<td style="text-align:right;">
83.90213
</td>
<td style="text-align:left;">
positive
</td>
<td style="text-align:left;">
MGI_C57BL6J_6096072
</td>
<td style="text-align:left;">
Gm47247
</td>
<td style="text-align:left;">
ENSEMBL:ENSMUSG00000112133
</td>
<td style="text-align:left;">
MGI:6096072
</td>
<td style="text-align:left;">
pseudogene
</td>
<td style="text-align:left;">
predicted gene, 47247
</td>
</tr>
<tr>
<td style="text-align:left;">
10
</td>
<td style="text-align:left;">
gene
</td>
<td style="text-align:right;">
83.92682
</td>
<td style="text-align:right;">
83.93072
</td>
<td style="text-align:left;">
positive
</td>
<td style="text-align:left;">
MGI_C57BL6J_5593078
</td>
<td style="text-align:left;">
Gm33919
</td>
<td style="text-align:left;">
NCBI_Gene:102637003
</td>
<td style="text-align:left;">
MGI:5593078
</td>
<td style="text-align:left;">
lncRNA gene
</td>
<td style="text-align:left;">
predicted gene, 33919
</td>
</tr>
<tr>
<td style="text-align:left;">
10
</td>
<td style="text-align:left;">
gene
</td>
<td style="text-align:right;">
83.93077
</td>
<td style="text-align:right;">
83.93353
</td>
<td style="text-align:left;">
negative
</td>
<td style="text-align:left;">
MGI_C57BL6J_5593139
</td>
<td style="text-align:left;">
Gm33980
</td>
<td style="text-align:left;">
NCBI_Gene:102637078
</td>
<td style="text-align:left;">
MGI:5593139
</td>
<td style="text-align:left;">
lncRNA gene
</td>
<td style="text-align:left;">
predicted gene, 33980
</td>
</tr>
<tr>
<td style="text-align:left;">
10
</td>
<td style="text-align:left;">
gene
</td>
<td style="text-align:right;">
83.93662
</td>
<td style="text-align:right;">
83.96372
</td>
<td style="text-align:left;">
negative
</td>
<td style="text-align:left;">
MGI_C57BL6J_5593202
</td>
<td style="text-align:left;">
Gm34043
</td>
<td style="text-align:left;">
NCBI_Gene:102637159
</td>
<td style="text-align:left;">
MGI:5593202
</td>
<td style="text-align:left;">
lncRNA gene
</td>
<td style="text-align:left;">
predicted gene, 34043
</td>
</tr>
<tr>
<td style="text-align:left;">
10
</td>
<td style="text-align:left;">
gene
</td>
<td style="text-align:right;">
83.97161
</td>
<td style="text-align:right;">
83.98720
</td>
<td style="text-align:left;">
positive
</td>
<td style="text-align:left;">
MGI_C57BL6J_5623610
</td>
<td style="text-align:left;">
Gm40725
</td>
<td style="text-align:left;">
NCBI_Gene:105245246
</td>
<td style="text-align:left;">
MGI:5623610
</td>
<td style="text-align:left;">
lncRNA gene
</td>
<td style="text-align:left;">
predicted gene, 40725
</td>
</tr>
<tr>
<td style="text-align:left;">
10
</td>
<td style="text-align:left;">
gene
</td>
<td style="text-align:right;">
83.99273
</td>
<td style="text-align:right;">
84.00486
</td>
<td style="text-align:left;">
negative
</td>
<td style="text-align:left;">
MGI_C57BL6J_2443595
</td>
<td style="text-align:left;">
C230099D08Rik
</td>
<td style="text-align:left;">
ENSEMBL:ENSMUSG00000111699
</td>
<td style="text-align:left;">
MGI:2443595
</td>
<td style="text-align:left;">
lncRNA gene
</td>
<td style="text-align:left;">
RIKEN cDNA C230099D08 gene
</td>
</tr>
<tr>
<td style="text-align:left;">
10
</td>
<td style="text-align:left;">
gene
</td>
<td style="text-align:right;">
84.02846
</td>
<td style="text-align:right;">
84.03116
</td>
<td style="text-align:left;">
positive
</td>
<td style="text-align:left;">
MGI_C57BL6J_5623611
</td>
<td style="text-align:left;">
Gm40726
</td>
<td style="text-align:left;">
NCBI_Gene:105245247
</td>
<td style="text-align:left;">
MGI:5623611
</td>
<td style="text-align:left;">
lncRNA gene
</td>
<td style="text-align:left;">
predicted gene, 40726
</td>
</tr>
<tr>
<td style="text-align:left;">
10
</td>
<td style="text-align:left;">
gene
</td>
<td style="text-align:right;">
84.03656
</td>
<td style="text-align:right;">
84.10088
</td>
<td style="text-align:left;">
negative
</td>
<td style="text-align:left;">
MGI_C57BL6J_5593343
</td>
<td style="text-align:left;">
Gm34184
</td>
<td style="text-align:left;">
NCBI_Gene:102637349,ENSEMBL:ENSMUSG00000111049
</td>
<td style="text-align:left;">
MGI:5593343
</td>
<td style="text-align:left;">
lncRNA gene
</td>
<td style="text-align:left;">
predicted gene, 34184
</td>
</tr>
<tr>
<td style="text-align:left;">
10
</td>
<td style="text-align:left;">
gene
</td>
<td style="text-align:right;">
84.06176
</td>
<td style="text-align:right;">
84.06223
</td>
<td style="text-align:left;">
positive
</td>
<td style="text-align:left;">
MGI_C57BL6J_5611136
</td>
<td style="text-align:left;">
Gm37908
</td>
<td style="text-align:left;">
ENSEMBL:ENSMUSG00000104317
</td>
<td style="text-align:left;">
MGI:5611136
</td>
<td style="text-align:left;">
unclassified gene
</td>
<td style="text-align:left;">
predicted gene, 37908
</td>
</tr>
<tr>
<td style="text-align:left;">
10
</td>
<td style="text-align:left;">
gene
</td>
<td style="text-align:right;">
84.08463
</td>
<td style="text-align:right;">
84.09625
</td>
<td style="text-align:left;">
positive
</td>
<td style="text-align:left;">
MGI_C57BL6J_5593272
</td>
<td style="text-align:left;">
Gm34113
</td>
<td style="text-align:left;">
NCBI_Gene:102637247,ENSEMBL:ENSMUSG00000111524
</td>
<td style="text-align:left;">
MGI:5593272
</td>
<td style="text-align:left;">
lncRNA gene
</td>
<td style="text-align:left;">
predicted gene, 34113
</td>
</tr>
<tr>
<td style="text-align:left;">
10
</td>
<td style="text-align:left;">
gene
</td>
<td style="text-align:right;">
84.10438
</td>
<td style="text-align:right;">
84.11042
</td>
<td style="text-align:left;">
negative
</td>
<td style="text-align:left;">
MGI_C57BL6J_5623612
</td>
<td style="text-align:left;">
Gm40727
</td>
<td style="text-align:left;">
NCBI_Gene:105245248
</td>
<td style="text-align:left;">
MGI:5623612
</td>
<td style="text-align:left;">
lncRNA gene
</td>
<td style="text-align:left;">
predicted gene, 40727
</td>
</tr>
<tr>
<td style="text-align:left;">
10
</td>
<td style="text-align:left;">
gene
</td>
<td style="text-align:right;">
84.19182
</td>
<td style="text-align:right;">
84.22206
</td>
<td style="text-align:left;">
negative
</td>
<td style="text-align:left;">
MGI_C57BL6J_6096082
</td>
<td style="text-align:left;">
Gm47253
</td>
<td style="text-align:left;">
ENSEMBL:ENSMUSG00000111062
</td>
<td style="text-align:left;">
MGI:6096082
</td>
<td style="text-align:left;">
lncRNA gene
</td>
<td style="text-align:left;">
predicted gene, 47253
</td>
</tr>
<tr>
<td style="text-align:left;">
10
</td>
<td style="text-align:left;">
pseudogene
</td>
<td style="text-align:right;">
84.30887
</td>
<td style="text-align:right;">
84.30936
</td>
<td style="text-align:left;">
positive
</td>
<td style="text-align:left;">
MGI_C57BL6J_3643901
</td>
<td style="text-align:left;">
Gm5425
</td>
<td style="text-align:left;">
NCBI_Gene:432482,ENSEMBL:ENSMUSG00000111875
</td>
<td style="text-align:left;">
MGI:3643901
</td>
<td style="text-align:left;">
pseudogene
</td>
<td style="text-align:left;">
predicted gene 5425
</td>
</tr>
<tr>
<td style="text-align:left;">
10
</td>
<td style="text-align:left;">
gene
</td>
<td style="text-align:right;">
84.31732
</td>
<td style="text-align:right;">
84.33575
</td>
<td style="text-align:left;">
negative
</td>
<td style="text-align:left;">
MGI_C57BL6J_5825838
</td>
<td style="text-align:left;">
Gm46201
</td>
<td style="text-align:left;">
NCBI_Gene:108167783
</td>
<td style="text-align:left;">
MGI:5825838
</td>
<td style="text-align:left;">
lncRNA gene
</td>
<td style="text-align:left;">
predicted gene, 46201
</td>
</tr>
<tr>
<td style="text-align:left;">
10
</td>
<td style="text-align:left;">
gene
</td>
<td style="text-align:right;">
84.37090
</td>
<td style="text-align:right;">
84.44060
</td>
<td style="text-align:left;">
negative
</td>
<td style="text-align:left;">
MGI_C57BL6J_1925226
</td>
<td style="text-align:left;">
Nuak1
</td>
<td style="text-align:left;">
NCBI_Gene:77976,ENSEMBL:ENSMUSG00000020032
</td>
<td style="text-align:left;">
MGI:1925226
</td>
<td style="text-align:left;">
protein coding gene
</td>
<td style="text-align:left;">
NUAK family, SNF1-like kinase, 1
</td>
</tr>
<tr>
<td style="text-align:left;">
10
</td>
<td style="text-align:left;">
gene
</td>
<td style="text-align:right;">
84.44083
</td>
<td style="text-align:right;">
84.46701
</td>
<td style="text-align:left;">
positive
</td>
<td style="text-align:left;">
MGI_C57BL6J_6097239
</td>
<td style="text-align:left;">
Gm47962
</td>
<td style="text-align:left;">
ENSEMBL:ENSMUSG00000111583
</td>
<td style="text-align:left;">
MGI:6097239
</td>
<td style="text-align:left;">
lncRNA gene
</td>
<td style="text-align:left;">
predicted gene, 47962
</td>
</tr>
<tr>
<td style="text-align:left;">
10
</td>
<td style="text-align:left;">
gene
</td>
<td style="text-align:right;">
84.48829
</td>
<td style="text-align:right;">
84.49768
</td>
<td style="text-align:left;">
positive
</td>
<td style="text-align:left;">
MGI_C57BL6J_1923054
</td>
<td style="text-align:left;">
4930463O16Rik
</td>
<td style="text-align:left;">
NCBI_Gene:75804,ENSEMBL:ENSMUSG00000020033
</td>
<td style="text-align:left;">
MGI:1923054
</td>
<td style="text-align:left;">
lncRNA gene
</td>
<td style="text-align:left;">
RIKEN cDNA 4930463O16 gene
</td>
</tr>
<tr>
<td style="text-align:left;">
10
</td>
<td style="text-align:left;">
gene
</td>
<td style="text-align:right;">
84.49887
</td>
<td style="text-align:right;">
84.49963
</td>
<td style="text-align:left;">
positive
</td>
<td style="text-align:left;">
MGI_C57BL6J_1921541
</td>
<td style="text-align:left;">
1700092E16Rik
</td>
<td style="text-align:left;">
ENSEMBL:ENSMUSG00000111487
</td>
<td style="text-align:left;">
MGI:1921541
</td>
<td style="text-align:left;">
unclassified non-coding RNA gene
</td>
<td style="text-align:left;">
RIKEN cDNA 1700092E16 gene
</td>
</tr>
<tr>
<td style="text-align:left;">
10
</td>
<td style="text-align:left;">
gene
</td>
<td style="text-align:right;">
84.50004
</td>
<td style="text-align:right;">
84.51710
</td>
<td style="text-align:left;">
negative
</td>
<td style="text-align:left;">
MGI_C57BL6J_5593564
</td>
<td style="text-align:left;">
Gm34405
</td>
<td style="text-align:left;">
NCBI_Gene:102637645
</td>
<td style="text-align:left;">
MGI:5593564
</td>
<td style="text-align:left;">
lncRNA gene
</td>
<td style="text-align:left;">
predicted gene, 34405
</td>
</tr>
<tr>
<td style="text-align:left;">
10
</td>
<td style="text-align:left;">
pseudogene
</td>
<td style="text-align:right;">
84.50203
</td>
<td style="text-align:right;">
84.50256
</td>
<td style="text-align:left;">
negative
</td>
<td style="text-align:left;">
MGI_C57BL6J_5010436
</td>
<td style="text-align:left;">
Gm18251
</td>
<td style="text-align:left;">
NCBI_Gene:100416788,ENSEMBL:ENSMUSG00000111478
</td>
<td style="text-align:left;">
MGI:5010436
</td>
<td style="text-align:left;">
pseudogene
</td>
<td style="text-align:left;">
predicted gene, 18251
</td>
</tr>
<tr>
<td style="text-align:left;">
10
</td>
<td style="text-align:left;">
gene
</td>
<td style="text-align:right;">
84.51805
</td>
<td style="text-align:right;">
84.52327
</td>
<td style="text-align:left;">
positive
</td>
<td style="text-align:left;">
MGI_C57BL6J_3045388
</td>
<td style="text-align:left;">
E230014E18Rik
</td>
<td style="text-align:left;">
ENSEMBL:ENSMUSG00000111514
</td>
<td style="text-align:left;">
MGI:3045388
</td>
<td style="text-align:left;">
lncRNA gene
</td>
<td style="text-align:left;">
RIKEN cDNA E230014E18 gene
</td>
</tr>
<tr>
<td style="text-align:left;">
10
</td>
<td style="text-align:left;">
gene
</td>
<td style="text-align:right;">
84.52630
</td>
<td style="text-align:right;">
84.53406
</td>
<td style="text-align:left;">
negative
</td>
<td style="text-align:left;">
MGI_C57BL6J_2444926
</td>
<td style="text-align:left;">
Ckap4
</td>
<td style="text-align:left;">
NCBI_Gene:216197,ENSEMBL:ENSMUSG00000046841
</td>
<td style="text-align:left;">
MGI:2444926
</td>
<td style="text-align:left;">
protein coding gene
</td>
<td style="text-align:left;">
cytoskeleton-associated protein 4
</td>
</tr>
<tr>
<td style="text-align:left;">
10
</td>
<td style="text-align:left;">
gene
</td>
<td style="text-align:right;">
84.54742
</td>
<td style="text-align:right;">
84.55334
</td>
<td style="text-align:left;">
negative
</td>
<td style="text-align:left;">
MGI_C57BL6J_3045387
</td>
<td style="text-align:left;">
Platr7
</td>
<td style="text-align:left;">
NCBI_Gene:442847,ENSEMBL:ENSMUSG00000097076
</td>
<td style="text-align:left;">
MGI:3045387
</td>
<td style="text-align:left;">
lncRNA gene
</td>
<td style="text-align:left;">
pluripotency associated transcript 7
</td>
</tr>
<tr>
<td style="text-align:left;">
10
</td>
<td style="text-align:left;">
gene
</td>
<td style="text-align:right;">
84.57232
</td>
<td style="text-align:right;">
84.57507
</td>
<td style="text-align:left;">
positive
</td>
<td style="text-align:left;">
MGI_C57BL6J_5593674
</td>
<td style="text-align:left;">
Gm34515
</td>
<td style="text-align:left;">
NCBI_Gene:102637793
</td>
<td style="text-align:left;">
MGI:5593674
</td>
<td style="text-align:left;">
lncRNA gene
</td>
<td style="text-align:left;">
predicted gene, 34515
</td>
</tr>
<tr>
<td style="text-align:left;">
10
</td>
<td style="text-align:left;">
gene
</td>
<td style="text-align:right;">
84.57663
</td>
<td style="text-align:right;">
84.61436
</td>
<td style="text-align:left;">
positive
</td>
<td style="text-align:left;">
MGI_C57BL6J_2444679
</td>
<td style="text-align:left;">
Tcp11l2
</td>
<td style="text-align:left;">
NCBI_Gene:216198,ENSEMBL:ENSMUSG00000020034
</td>
<td style="text-align:left;">
MGI:2444679
</td>
<td style="text-align:left;">
protein coding gene
</td>
<td style="text-align:left;">
t-complex 11 (mouse) like 2
</td>
</tr>
<tr>
<td style="text-align:left;">
10
</td>
<td style="text-align:left;">
gene
</td>
<td style="text-align:right;">
84.61472
</td>
<td style="text-align:right;">
84.61481
</td>
<td style="text-align:left;">
positive
</td>
<td style="text-align:left;">
MGI_C57BL6J_5455434
</td>
<td style="text-align:left;">
Gm25657
</td>
<td style="text-align:left;">
ENSEMBL:ENSMUSG00000089096
</td>
<td style="text-align:left;">
MGI:5455434
</td>
<td style="text-align:left;">
snRNA gene
</td>
<td style="text-align:left;">
predicted gene, 25657
</td>
</tr>
<tr>
<td style="text-align:left;">
10
</td>
<td style="text-align:left;">
gene
</td>
<td style="text-align:right;">
84.61834
</td>
<td style="text-align:right;">
84.62753
</td>
<td style="text-align:left;">
negative
</td>
<td style="text-align:left;">
MGI_C57BL6J_4936883
</td>
<td style="text-align:left;">
Gm17249
</td>
<td style="text-align:left;">
NCBI_Gene:102637873,ENSEMBL:ENSMUSG00000097309
</td>
<td style="text-align:left;">
MGI:4936883
</td>
<td style="text-align:left;">
lncRNA gene
</td>
<td style="text-align:left;">
predicted gene, 17249
</td>
</tr>
<tr>
<td style="text-align:left;">
10
</td>
<td style="text-align:left;">
gene
</td>
<td style="text-align:right;">
84.62229
</td>
<td style="text-align:right;">
84.72718
</td>
<td style="text-align:left;">
positive
</td>
<td style="text-align:left;">
MGI_C57BL6J_1917678
</td>
<td style="text-align:left;">
Polr3b
</td>
<td style="text-align:left;">
NCBI_Gene:70428,ENSEMBL:ENSMUSG00000034453
</td>
<td style="text-align:left;">
MGI:1917678
</td>
<td style="text-align:left;">
protein coding gene
</td>
<td style="text-align:left;">
polymerase (RNA) III (DNA directed) polypeptide B
</td>
</tr>
<tr>
<td style="text-align:left;">
10
</td>
<td style="text-align:left;">
gene
</td>
<td style="text-align:right;">
84.75605
</td>
<td style="text-align:right;">
84.90654
</td>
<td style="text-align:left;">
positive
</td>
<td style="text-align:left;">
MGI_C57BL6J_1918387
</td>
<td style="text-align:left;">
Rfx4
</td>
<td style="text-align:left;">
NCBI_Gene:71137,ENSEMBL:ENSMUSG00000020037
</td>
<td style="text-align:left;">
MGI:1918387
</td>
<td style="text-align:left;">
protein coding gene
</td>
<td style="text-align:left;">
regulatory factor X, 4 (influences HLA class II expression)
</td>
</tr>
<tr>
<td style="text-align:left;">
10
</td>
<td style="text-align:left;">
gene
</td>
<td style="text-align:right;">
84.79795
</td>
<td style="text-align:right;">
84.86329
</td>
<td style="text-align:left;">
negative
</td>
<td style="text-align:left;">
MGI_C57BL6J_5623613
</td>
<td style="text-align:left;">
Gm40728
</td>
<td style="text-align:left;">
NCBI_Gene:105245249
</td>
<td style="text-align:left;">
MGI:5623613
</td>
<td style="text-align:left;">
lncRNA gene
</td>
<td style="text-align:left;">
predicted gene, 40728
</td>
</tr>
<tr>
<td style="text-align:left;">
10
</td>
<td style="text-align:left;">
gene
</td>
<td style="text-align:right;">
84.81109
</td>
<td style="text-align:right;">
84.81118
</td>
<td style="text-align:left;">
negative
</td>
<td style="text-align:left;">
MGI_C57BL6J_5454003
</td>
<td style="text-align:left;">
Gm24226
</td>
<td style="text-align:left;">
ENSEMBL:ENSMUSG00000089451
</td>
<td style="text-align:left;">
MGI:5454003
</td>
<td style="text-align:left;">
snRNA gene
</td>
<td style="text-align:left;">
predicted gene, 24226
</td>
</tr>
<tr>
<td style="text-align:left;">
10
</td>
<td style="text-align:left;">
gene
</td>
<td style="text-align:right;">
84.91725
</td>
<td style="text-align:right;">
85.01834
</td>
<td style="text-align:left;">
positive
</td>
<td style="text-align:left;">
MGI_C57BL6J_2682307
</td>
<td style="text-align:left;">
Ric8b
</td>
<td style="text-align:left;">
NCBI_Gene:237422,ENSEMBL:ENSMUSG00000035620
</td>
<td style="text-align:left;">
MGI:2682307
</td>
<td style="text-align:left;">
protein coding gene
</td>
<td style="text-align:left;">
RIC8 guanine nucleotide exchange factor B
</td>
</tr>
<tr>
<td style="text-align:left;">
10
</td>
<td style="text-align:left;">
gene
</td>
<td style="text-align:right;">
84.98771
</td>
<td style="text-align:right;">
85.00365
</td>
<td style="text-align:left;">
negative
</td>
<td style="text-align:left;">
MGI_C57BL6J_5593792
</td>
<td style="text-align:left;">
Gm34633
</td>
<td style="text-align:left;">
NCBI_Gene:102637948
</td>
<td style="text-align:left;">
MGI:5593792
</td>
<td style="text-align:left;">
lncRNA gene
</td>
<td style="text-align:left;">
predicted gene, 34633
</td>
</tr>
<tr>
<td style="text-align:left;">
10
</td>
<td style="text-align:left;">
gene
</td>
<td style="text-align:right;">
85.09629
</td>
<td style="text-align:right;">
85.10250
</td>
<td style="text-align:left;">
negative
</td>
<td style="text-align:left;">
MGI_C57BL6J_1338765
</td>
<td style="text-align:left;">
Fhl4
</td>
<td style="text-align:left;">
NCBI_Gene:14202,ENSEMBL:ENSMUSG00000050035
</td>
<td style="text-align:left;">
MGI:1338765
</td>
<td style="text-align:left;">
protein coding gene
</td>
<td style="text-align:left;">
four and a half LIM domains 4
</td>
</tr>
<tr>
<td style="text-align:left;">
10
</td>
<td style="text-align:left;">
gene
</td>
<td style="text-align:right;">
85.10263
</td>
<td style="text-align:right;">
85.12062
</td>
<td style="text-align:left;">
positive
</td>
<td style="text-align:left;">
MGI_C57BL6J_2143652
</td>
<td style="text-align:left;">
Tmem263
</td>
<td style="text-align:left;">
NCBI_Gene:103266,ENSEMBL:ENSMUSG00000060935
</td>
<td style="text-align:left;">
MGI:2143652
</td>
<td style="text-align:left;">
protein coding gene
</td>
<td style="text-align:left;">
transmembrane protein 263
</td>
</tr>
<tr>
<td style="text-align:left;">
10
</td>
<td style="text-align:left;">
gene
</td>
<td style="text-align:right;">
85.11943
</td>
<td style="text-align:right;">
85.12807
</td>
<td style="text-align:left;">
negative
</td>
<td style="text-align:left;">
MGI_C57BL6J_1921488
</td>
<td style="text-align:left;">
Mterf2
</td>
<td style="text-align:left;">
NCBI_Gene:74238,ENSEMBL:ENSMUSG00000049038
</td>
<td style="text-align:left;">
MGI:1921488
</td>
<td style="text-align:left;">
protein coding gene
</td>
<td style="text-align:left;">
mitochondrial transcription termination factor 2
</td>
</tr>
<tr>
<td style="text-align:left;">
10
</td>
<td style="text-align:left;">
gene
</td>
<td style="text-align:right;">
85.13170
</td>
<td style="text-align:right;">
85.18506
</td>
<td style="text-align:left;">
negative
</td>
<td style="text-align:left;">
MGI_C57BL6J_1270841
</td>
<td style="text-align:left;">
Cry1
</td>
<td style="text-align:left;">
NCBI_Gene:12952,ENSEMBL:ENSMUSG00000020038
</td>
<td style="text-align:left;">
MGI:1270841
</td>
<td style="text-align:left;">
protein coding gene
</td>
<td style="text-align:left;">
cryptochrome 1 (photolyase-like)
</td>
</tr>
<tr>
<td style="text-align:left;">
10
</td>
<td style="text-align:left;">
gene
</td>
<td style="text-align:right;">
85.22094
</td>
<td style="text-align:right;">
85.26639
</td>
<td style="text-align:left;">
negative
</td>
<td style="text-align:left;">
MGI_C57BL6J_5621460
</td>
<td style="text-align:left;">
Gm38575
</td>
<td style="text-align:left;">
ENSEMBL:ENSMUSG00000111752
</td>
<td style="text-align:left;">
MGI:5621460
</td>
<td style="text-align:left;">
lncRNA gene
</td>
<td style="text-align:left;">
predicted gene, 38575
</td>
</tr>
<tr>
<td style="text-align:left;">
10
</td>
<td style="text-align:left;">
gene
</td>
<td style="text-align:right;">
85.28423
</td>
<td style="text-align:right;">
85.28433
</td>
<td style="text-align:left;">
negative
</td>
<td style="text-align:left;">
MGI_C57BL6J_5530774
</td>
<td style="text-align:left;">
Gm27392
</td>
<td style="text-align:left;">
ENSEMBL:ENSMUSG00000098852
</td>
<td style="text-align:left;">
MGI:5530774
</td>
<td style="text-align:left;">
miRNA gene
</td>
<td style="text-align:left;">
predicted gene, 27392
</td>
</tr>
<tr>
<td style="text-align:left;">
10
</td>
<td style="text-align:left;">
pseudogene
</td>
<td style="text-align:right;">
85.31347
</td>
<td style="text-align:right;">
85.31444
</td>
<td style="text-align:left;">
positive
</td>
<td style="text-align:left;">
MGI_C57BL6J_3647964
</td>
<td style="text-align:left;">
Gm8394
</td>
<td style="text-align:left;">
NCBI_Gene:666974,ENSEMBL:ENSMUSG00000050490
</td>
<td style="text-align:left;">
MGI:3647964
</td>
<td style="text-align:left;">
pseudogene
</td>
<td style="text-align:left;">
predicted gene 8394
</td>
</tr>
<tr>
<td style="text-align:left;">
10
</td>
<td style="text-align:left;">
gene
</td>
<td style="text-align:right;">
85.33269
</td>
<td style="text-align:right;">
85.34757
</td>
<td style="text-align:left;">
negative
</td>
<td style="text-align:left;">
MGI_C57BL6J_5593871
</td>
<td style="text-align:left;">
Gm34712
</td>
<td style="text-align:left;">
NCBI_Gene:102638057
</td>
<td style="text-align:left;">
MGI:5593871
</td>
<td style="text-align:left;">
lncRNA gene
</td>
<td style="text-align:left;">
predicted gene, 34712
</td>
</tr>
<tr>
<td style="text-align:left;">
10
</td>
<td style="text-align:left;">
gene
</td>
<td style="text-align:right;">
85.33390
</td>
<td style="text-align:right;">
85.33400
</td>
<td style="text-align:left;">
positive
</td>
<td style="text-align:left;">
MGI_C57BL6J_5455058
</td>
<td style="text-align:left;">
Gm25281
</td>
<td style="text-align:left;">
ENSEMBL:ENSMUSG00000087895
</td>
<td style="text-align:left;">
MGI:5455058
</td>
<td style="text-align:left;">
miRNA gene
</td>
<td style="text-align:left;">
predicted gene, 25281
</td>
</tr>
<tr>
<td style="text-align:left;">
10
</td>
<td style="text-align:left;">
gene
</td>
<td style="text-align:right;">
85.33901
</td>
<td style="text-align:right;">
85.33995
</td>
<td style="text-align:left;">
positive
</td>
<td style="text-align:left;">
MGI_C57BL6J_5623615
</td>
<td style="text-align:left;">
Gm40730
</td>
<td style="text-align:left;">
NCBI_Gene:105245251
</td>
<td style="text-align:left;">
MGI:5623615
</td>
<td style="text-align:left;">
lncRNA gene
</td>
<td style="text-align:left;">
predicted gene, 40730
</td>
</tr>
<tr>
<td style="text-align:left;">
10
</td>
<td style="text-align:left;">
gene
</td>
<td style="text-align:right;">
85.38677
</td>
<td style="text-align:right;">
85.66029
</td>
<td style="text-align:left;">
positive
</td>
<td style="text-align:left;">
MGI_C57BL6J_1921257
</td>
<td style="text-align:left;">
Btbd11
</td>
<td style="text-align:left;">
NCBI_Gene:74007,ENSEMBL:ENSMUSG00000020042
</td>
<td style="text-align:left;">
MGI:1921257
</td>
<td style="text-align:left;">
protein coding gene
</td>
<td style="text-align:left;">
BTB (POZ) domain containing 11
</td>
</tr>
<tr>
<td style="text-align:left;">
10
</td>
<td style="text-align:left;">
gene
</td>
<td style="text-align:right;">
85.57888
</td>
<td style="text-align:right;">
85.58399
</td>
<td style="text-align:left;">
negative
</td>
<td style="text-align:left;">
MGI_C57BL6J_5623616
</td>
<td style="text-align:left;">
Gm40731
</td>
<td style="text-align:left;">
NCBI_Gene:105245252
</td>
<td style="text-align:left;">
MGI:5623616
</td>
<td style="text-align:left;">
lncRNA gene
</td>
<td style="text-align:left;">
predicted gene, 40731
</td>
</tr>
<tr>
<td style="text-align:left;">
10
</td>
<td style="text-align:left;">
gene
</td>
<td style="text-align:right;">
85.69438
</td>
<td style="text-align:right;">
85.71125
</td>
<td style="text-align:left;">
negative
</td>
<td style="text-align:left;">
MGI_C57BL6J_3645890
</td>
<td style="text-align:left;">
Gm6713
</td>
<td style="text-align:left;">
NCBI_Gene:626858,ENSEMBL:ENSMUSG00000111609
</td>
<td style="text-align:left;">
MGI:3645890
</td>
<td style="text-align:left;">
protein coding gene
</td>
<td style="text-align:left;">
predicted gene 6713
</td>
</tr>
<tr>
<td style="text-align:left;">
10
</td>
<td style="text-align:left;">
gene
</td>
<td style="text-align:right;">
85.70768
</td>
<td style="text-align:right;">
85.71237
</td>
<td style="text-align:left;">
positive
</td>
<td style="text-align:left;">
MGI_C57BL6J_6096264
</td>
<td style="text-align:left;">
Gm47358
</td>
<td style="text-align:left;">
ENSEMBL:ENSMUSG00000111435
</td>
<td style="text-align:left;">
MGI:6096264
</td>
<td style="text-align:left;">
lncRNA gene
</td>
<td style="text-align:left;">
predicted gene, 47358
</td>
</tr>
<tr>
<td style="text-align:left;">
10
</td>
<td style="text-align:left;">
pseudogene
</td>
<td style="text-align:right;">
85.72661
</td>
<td style="text-align:right;">
85.73307
</td>
<td style="text-align:left;">
negative
</td>
<td style="text-align:left;">
MGI_C57BL6J_3645675
</td>
<td style="text-align:left;">
Gm6714
</td>
<td style="text-align:left;">
NCBI_Gene:626878,ENSEMBL:ENSMUSG00000112451
</td>
<td style="text-align:left;">
MGI:3645675
</td>
<td style="text-align:left;">
pseudogene
</td>
<td style="text-align:left;">
predicted gene 6714
</td>
</tr>
<tr>
<td style="text-align:left;">
10
</td>
<td style="text-align:left;">
pseudogene
</td>
<td style="text-align:right;">
85.73075
</td>
<td style="text-align:right;">
85.73210
</td>
<td style="text-align:left;">
negative
</td>
<td style="text-align:left;">
MGI_C57BL6J_5434381
</td>
<td style="text-align:left;">
Gm21026
</td>
<td style="text-align:left;">
NCBI_Gene:100534313,ENSEMBL:ENSMUSG00000112479
</td>
<td style="text-align:left;">
MGI:5434381
</td>
<td style="text-align:left;">
pseudogene
</td>
<td style="text-align:left;">
predicted gene, 21026
</td>
</tr>
<tr>
<td style="text-align:left;">
10
</td>
<td style="text-align:left;">
gene
</td>
<td style="text-align:right;">
85.73970
</td>
<td style="text-align:right;">
85.74034
</td>
<td style="text-align:left;">
negative
</td>
<td style="text-align:left;">
MGI_C57BL6J_5623617
</td>
<td style="text-align:left;">
Gm40732
</td>
<td style="text-align:left;">
NCBI_Gene:105245253
</td>
<td style="text-align:left;">
MGI:5623617
</td>
<td style="text-align:left;">
lncRNA gene
</td>
<td style="text-align:left;">
predicted gene, 40732
</td>
</tr>
<tr>
<td style="text-align:left;">
10
</td>
<td style="text-align:left;">
pseudogene
</td>
<td style="text-align:right;">
85.74417
</td>
<td style="text-align:right;">
85.74507
</td>
<td style="text-align:left;">
negative
</td>
<td style="text-align:left;">
MGI_C57BL6J_6096271
</td>
<td style="text-align:left;">
Gm47362
</td>
<td style="text-align:left;">
ENSEMBL:ENSMUSG00000112119
</td>
<td style="text-align:left;">
MGI:6096271
</td>
<td style="text-align:left;">
pseudogene
</td>
<td style="text-align:left;">
predicted gene, 47362
</td>
</tr>
<tr>
<td style="text-align:left;">
10
</td>
<td style="text-align:left;">
pseudogene
</td>
<td style="text-align:right;">
85.74540
</td>
<td style="text-align:right;">
85.74595
</td>
<td style="text-align:left;">
negative
</td>
<td style="text-align:left;">
MGI_C57BL6J_6096272
</td>
<td style="text-align:left;">
Gm47363
</td>
<td style="text-align:left;">
ENSEMBL:ENSMUSG00000112568
</td>
<td style="text-align:left;">
MGI:6096272
</td>
<td style="text-align:left;">
pseudogene
</td>
<td style="text-align:left;">
predicted gene, 47363
</td>
</tr>
<tr>
<td style="text-align:left;">
10
</td>
<td style="text-align:left;">
pseudogene
</td>
<td style="text-align:right;">
85.76426
</td>
<td style="text-align:right;">
85.76576
</td>
<td style="text-align:left;">
negative
</td>
<td style="text-align:left;">
MGI_C57BL6J_6096275
</td>
<td style="text-align:left;">
Gm47366
</td>
<td style="text-align:left;">
ENSEMBL:ENSMUSG00000111891
</td>
<td style="text-align:left;">
MGI:6096275
</td>
<td style="text-align:left;">
pseudogene
</td>
<td style="text-align:left;">
predicted gene, 47366
</td>
</tr>
<tr>
<td style="text-align:left;">
10
</td>
<td style="text-align:left;">
pseudogene
</td>
<td style="text-align:right;">
85.76459
</td>
<td style="text-align:right;">
85.76576
</td>
<td style="text-align:left;">
negative
</td>
<td style="text-align:left;">
MGI_C57BL6J_6096274
</td>
<td style="text-align:left;">
Gm47365
</td>
<td style="text-align:left;">
ENSEMBL:ENSMUSG00000111996
</td>
<td style="text-align:left;">
MGI:6096274
</td>
<td style="text-align:left;">
pseudogene
</td>
<td style="text-align:left;">
predicted gene, 47365
</td>
</tr>
<tr>
<td style="text-align:left;">
10
</td>
<td style="text-align:left;">
pseudogene
</td>
<td style="text-align:right;">
85.76590
</td>
<td style="text-align:right;">
85.76758
</td>
<td style="text-align:left;">
negative
</td>
<td style="text-align:left;">
MGI_C57BL6J_6096276
</td>
<td style="text-align:left;">
Gm47367
</td>
<td style="text-align:left;">
ENSEMBL:ENSMUSG00000112504
</td>
<td style="text-align:left;">
MGI:6096276
</td>
<td style="text-align:left;">
pseudogene
</td>
<td style="text-align:left;">
predicted gene, 47367
</td>
</tr>
<tr>
<td style="text-align:left;">
10
</td>
<td style="text-align:left;">
pseudogene
</td>
<td style="text-align:right;">
85.78530
</td>
<td style="text-align:right;">
85.79208
</td>
<td style="text-align:left;">
positive
</td>
<td style="text-align:left;">
MGI_C57BL6J_3779876
</td>
<td style="text-align:left;">
Gm9466
</td>
<td style="text-align:left;">
NCBI_Gene:669723,ENSEMBL:ENSMUSG00000112418
</td>
<td style="text-align:left;">
MGI:3779876
</td>
<td style="text-align:left;">
pseudogene
</td>
<td style="text-align:left;">
predicted gene 9466
</td>
</tr>
<tr>
<td style="text-align:left;">
10
</td>
<td style="text-align:left;">
pseudogene
</td>
<td style="text-align:right;">
85.78621
</td>
<td style="text-align:right;">
85.78744
</td>
<td style="text-align:left;">
positive
</td>
<td style="text-align:left;">
MGI_C57BL6J_5011797
</td>
<td style="text-align:left;">
Gm19612
</td>
<td style="text-align:left;">
NCBI_Gene:100503252,ENSEMBL:ENSMUSG00000111884
</td>
<td style="text-align:left;">
MGI:5011797
</td>
<td style="text-align:left;">
pseudogene
</td>
<td style="text-align:left;">
predicted gene, 19612
</td>
</tr>
<tr>
<td style="text-align:left;">
10
</td>
<td style="text-align:left;">
gene
</td>
<td style="text-align:right;">
85.80621
</td>
<td style="text-align:right;">
85.81089
</td>
<td style="text-align:left;">
negative
</td>
<td style="text-align:left;">
MGI_C57BL6J_2686405
</td>
<td style="text-align:left;">
Gm1559
</td>
<td style="text-align:left;">
ENSEMBL:ENSMUSG00000112826
</td>
<td style="text-align:left;">
MGI:2686405
</td>
<td style="text-align:left;">
lncRNA gene
</td>
<td style="text-align:left;">
predicted gene 1559
</td>
</tr>
<tr>
<td style="text-align:left;">
10
</td>
<td style="text-align:left;">
pseudogene
</td>
<td style="text-align:right;">
85.80767
</td>
<td style="text-align:right;">
85.82274
</td>
<td style="text-align:left;">
positive
</td>
<td style="text-align:left;">
MGI_C57BL6J_3645077
</td>
<td style="text-align:left;">
Gm6721
</td>
<td style="text-align:left;">
NCBI_Gene:626940,ENSEMBL:ENSMUSG00000112318
</td>
<td style="text-align:left;">
MGI:3645077
</td>
<td style="text-align:left;">
pseudogene
</td>
<td style="text-align:left;">
predicted gene 6721
</td>
</tr>
<tr>
<td style="text-align:left;">
10
</td>
<td style="text-align:left;">
gene
</td>
<td style="text-align:right;">
85.82949
</td>
<td style="text-align:right;">
85.88910
</td>
<td style="text-align:left;">
positive
</td>
<td style="text-align:left;">
MGI_C57BL6J_1914735
</td>
<td style="text-align:left;">
Pwp1
</td>
<td style="text-align:left;">
NCBI_Gene:103136,ENSEMBL:ENSMUSG00000001785
</td>
<td style="text-align:left;">
MGI:1914735
</td>
<td style="text-align:left;">
protein coding gene
</td>
<td style="text-align:left;">
PWP1 homolog, endonuclein
</td>
</tr>
<tr>
<td style="text-align:left;">
10
</td>
<td style="text-align:left;">
pseudogene
</td>
<td style="text-align:right;">
85.82959
</td>
<td style="text-align:right;">
85.83340
</td>
<td style="text-align:left;">
positive
</td>
<td style="text-align:left;">
MGI_C57BL6J_5011809
</td>
<td style="text-align:left;">
Gm19624
</td>
<td style="text-align:left;">
NCBI_Gene:102639184,ENSEMBL:ENSMUSG00000112326
</td>
<td style="text-align:left;">
MGI:5011809
</td>
<td style="text-align:left;">
pseudogene
</td>
<td style="text-align:left;">
predicted gene, 19624
</td>
</tr>
<tr>
<td style="text-align:left;">
10
</td>
<td style="text-align:left;">
pseudogene
</td>
<td style="text-align:right;">
85.84914
</td>
<td style="text-align:right;">
85.85087
</td>
<td style="text-align:left;">
positive
</td>
<td style="text-align:left;">
MGI_C57BL6J_6095684
</td>
<td style="text-align:left;">
Gm47006
</td>
<td style="text-align:left;">
ENSEMBL:ENSMUSG00000112836
</td>
<td style="text-align:left;">
MGI:6095684
</td>
<td style="text-align:left;">
pseudogene
</td>
<td style="text-align:left;">
predicted gene, 47006
</td>
</tr>
<tr>
<td style="text-align:left;">
10
</td>
<td style="text-align:left;">
gene
</td>
<td style="text-align:right;">
85.89196
</td>
<td style="text-align:right;">
85.91715
</td>
<td style="text-align:left;">
negative
</td>
<td style="text-align:left;">
MGI_C57BL6J_1920093
</td>
<td style="text-align:left;">
Prdm4
</td>
<td style="text-align:left;">
NCBI_Gene:72843,ENSEMBL:ENSMUSG00000035529
</td>
<td style="text-align:left;">
MGI:1920093
</td>
<td style="text-align:left;">
protein coding gene
</td>
<td style="text-align:left;">
PR domain containing 4
</td>
</tr>
<tr>
<td style="text-align:left;">
10
</td>
<td style="text-align:left;">
gene
</td>
<td style="text-align:right;">
85.92768
</td>
<td style="text-align:right;">
85.94621
</td>
<td style="text-align:left;">
positive
</td>
<td style="text-align:left;">
MGI_C57BL6J_1914591
</td>
<td style="text-align:left;">
Ascl4
</td>
<td style="text-align:left;">
NCBI_Gene:67341,ENSEMBL:ENSMUSG00000085111
</td>
<td style="text-align:left;">
MGI:1914591
</td>
<td style="text-align:left;">
protein coding gene
</td>
<td style="text-align:left;">
achaete-scute family bHLH transcription factor 4
</td>
</tr>
<tr>
<td style="text-align:left;">
10
</td>
<td style="text-align:left;">
gene
</td>
<td style="text-align:right;">
85.93864
</td>
<td style="text-align:right;">
85.95782
</td>
<td style="text-align:left;">
negative
</td>
<td style="text-align:left;">
MGI_C57BL6J_106379
</td>
<td style="text-align:left;">
Rtcb
</td>
<td style="text-align:left;">
NCBI_Gene:28088,ENSEMBL:ENSMUSG00000001783
</td>
<td style="text-align:left;">
MGI:106379
</td>
<td style="text-align:left;">
protein coding gene
</td>
<td style="text-align:left;">
RNA 2’,3’-cyclic phosphate and 5’-OH ligase
</td>
</tr>
<tr>
<td style="text-align:left;">
10
</td>
<td style="text-align:left;">
pseudogene
</td>
<td style="text-align:right;">
85.95496
</td>
<td style="text-align:right;">
85.95514
</td>
<td style="text-align:left;">
positive
</td>
<td style="text-align:left;">
MGI_C57BL6J_6097551
</td>
<td style="text-align:left;">
Gm48175
</td>
<td style="text-align:left;">
ENSEMBL:ENSMUSG00000112613
</td>
<td style="text-align:left;">
MGI:6097551
</td>
<td style="text-align:left;">
pseudogene
</td>
<td style="text-align:left;">
predicted gene, 48175
</td>
</tr>
<tr>
<td style="text-align:left;">
10
</td>
<td style="text-align:left;">
gene
</td>
<td style="text-align:right;">
85.95934
</td>
<td style="text-align:right;">
86.01189
</td>
<td style="text-align:left;">
negative
</td>
<td style="text-align:left;">
MGI_C57BL6J_3026884
</td>
<td style="text-align:left;">
Bpifc
</td>
<td style="text-align:left;">
NCBI_Gene:270757,ENSEMBL:ENSMUSG00000050108
</td>
<td style="text-align:left;">
MGI:3026884
</td>
<td style="text-align:left;">
protein coding gene
</td>
<td style="text-align:left;">
BPI fold containing family C
</td>
</tr>
<tr>
<td style="text-align:left;">
10
</td>
<td style="text-align:left;">
gene
</td>
<td style="text-align:right;">
86.02044
</td>
<td style="text-align:right;">
86.02233
</td>
<td style="text-align:left;">
negative
</td>
<td style="text-align:left;">
MGI_C57BL6J_3642156
</td>
<td style="text-align:left;">
A230060F14Rik
</td>
<td style="text-align:left;">
ENSEMBL:ENSMUSG00000097692
</td>
<td style="text-align:left;">
MGI:3642156
</td>
<td style="text-align:left;">
lncRNA gene
</td>
<td style="text-align:left;">
RIKEN cDNA A230060F14 gene
</td>
</tr>
<tr>
<td style="text-align:left;">
10
</td>
<td style="text-align:left;">
gene
</td>
<td style="text-align:right;">
86.02187
</td>
<td style="text-align:right;">
86.05187
</td>
<td style="text-align:left;">
positive
</td>
<td style="text-align:left;">
MGI_C57BL6J_1917004
</td>
<td style="text-align:left;">
Fbxo7
</td>
<td style="text-align:left;">
NCBI_Gene:69754,ENSEMBL:ENSMUSG00000001786
</td>
<td style="text-align:left;">
MGI:1917004
</td>
<td style="text-align:left;">
protein coding gene
</td>
<td style="text-align:left;">
F-box protein 7
</td>
</tr>
<tr>
<td style="text-align:left;">
10
</td>
<td style="text-align:left;">
gene
</td>
<td style="text-align:right;">
86.03869
</td>
<td style="text-align:right;">
86.03879
</td>
<td style="text-align:left;">
negative
</td>
<td style="text-align:left;">
MGI_C57BL6J_5454207
</td>
<td style="text-align:left;">
Gm24430
</td>
<td style="text-align:left;">
ENSEMBL:ENSMUSG00000077265
</td>
<td style="text-align:left;">
MGI:5454207
</td>
<td style="text-align:left;">
snRNA gene
</td>
<td style="text-align:left;">
predicted gene, 24430
</td>
</tr>
<tr>
<td style="text-align:left;">
10
</td>
<td style="text-align:left;">
gene
</td>
<td style="text-align:right;">
86.04875
</td>
<td style="text-align:right;">
86.49903
</td>
<td style="text-align:left;">
negative
</td>
<td style="text-align:left;">
MGI_C57BL6J_1351334
</td>
<td style="text-align:left;">
Syn3
</td>
<td style="text-align:left;">
NCBI_Gene:27204,ENSEMBL:ENSMUSG00000059602
</td>
<td style="text-align:left;">
MGI:1351334
</td>
<td style="text-align:left;">
protein coding gene
</td>
<td style="text-align:left;">
synapsin III
</td>
</tr>
<tr>
<td style="text-align:left;">
10
</td>
<td style="text-align:left;">
gene
</td>
<td style="text-align:right;">
86.09585
</td>
<td style="text-align:right;">
86.11359
</td>
<td style="text-align:left;">
positive
</td>
<td style="text-align:left;">
MGI_C57BL6J_1925420
</td>
<td style="text-align:left;">
4930486F22Rik
</td>
<td style="text-align:left;">
NCBI_Gene:78170,ENSEMBL:ENSMUSG00000112493
</td>
<td style="text-align:left;">
MGI:1925420
</td>
<td style="text-align:left;">
lncRNA gene
</td>
<td style="text-align:left;">
RIKEN cDNA 4930486F22 gene
</td>
</tr>
<tr>
<td style="text-align:left;">
10
</td>
<td style="text-align:left;">
gene
</td>
<td style="text-align:right;">
86.12583
</td>
<td style="text-align:right;">
86.12985
</td>
<td style="text-align:left;">
positive
</td>
<td style="text-align:left;">
MGI_C57BL6J_3801993
</td>
<td style="text-align:left;">
Gm15990
</td>
<td style="text-align:left;">
ENSEMBL:ENSMUSG00000087256
</td>
<td style="text-align:left;">
MGI:3801993
</td>
<td style="text-align:left;">
lncRNA gene
</td>
<td style="text-align:left;">
predicted gene 15990
</td>
</tr>
<tr>
<td style="text-align:left;">
10
</td>
<td style="text-align:left;">
gene
</td>
<td style="text-align:right;">
86.15613
</td>
<td style="text-align:right;">
86.18954
</td>
<td style="text-align:left;">
positive
</td>
<td style="text-align:left;">
MGI_C57BL6J_5594795
</td>
<td style="text-align:left;">
Gm35636
</td>
<td style="text-align:left;">
NCBI_Gene:102639289
</td>
<td style="text-align:left;">
MGI:5594795
</td>
<td style="text-align:left;">
lncRNA gene
</td>
<td style="text-align:left;">
predicted gene, 35636
</td>
</tr>
<tr>
<td style="text-align:left;">
10
</td>
<td style="text-align:left;">
gene
</td>
<td style="text-align:right;">
86.30037
</td>
<td style="text-align:right;">
86.34951
</td>
<td style="text-align:left;">
positive
</td>
<td style="text-align:left;">
MGI_C57BL6J_98754
</td>
<td style="text-align:left;">
Timp3
</td>
<td style="text-align:left;">
NCBI_Gene:21859,ENSEMBL:ENSMUSG00000020044
</td>
<td style="text-align:left;">
MGI:98754
</td>
<td style="text-align:left;">
protein coding gene
</td>
<td style="text-align:left;">
tissue inhibitor of metalloproteinase 3
</td>
</tr>
<tr>
<td style="text-align:left;">
10
</td>
<td style="text-align:left;">
gene
</td>
<td style="text-align:right;">
86.32703
</td>
<td style="text-align:right;">
86.32930
</td>
<td style="text-align:left;">
positive
</td>
<td style="text-align:left;">
MGI_C57BL6J_2443425
</td>
<td style="text-align:left;">
B930095M22Rik
</td>
<td style="text-align:left;">
NA
</td>
<td style="text-align:left;">
NA
</td>
<td style="text-align:left;">
unclassified gene
</td>
<td style="text-align:left;">
RIKEN cDNA B930095M22 gene
</td>
</tr>
<tr>
<td style="text-align:left;">
10
</td>
<td style="text-align:left;">
gene
</td>
<td style="text-align:right;">
86.52630
</td>
<td style="text-align:right;">
86.54187
</td>
<td style="text-align:left;">
negative
</td>
<td style="text-align:left;">
MGI_C57BL6J_3646339
</td>
<td style="text-align:left;">
Gm6729
</td>
<td style="text-align:left;">
NCBI_Gene:627035,ENSEMBL:ENSMUSG00000090816,ENSEMBL:ENSMUSG00000112586
</td>
<td style="text-align:left;">
MGI:3646339
</td>
<td style="text-align:left;">
protein coding gene
</td>
<td style="text-align:left;">
predicted gene 6729
</td>
</tr>
<tr>
<td style="text-align:left;">
10
</td>
<td style="text-align:left;">
pseudogene
</td>
<td style="text-align:right;">
86.55081
</td>
<td style="text-align:right;">
86.55266
</td>
<td style="text-align:left;">
negative
</td>
<td style="text-align:left;">
MGI_C57BL6J_3779991
</td>
<td style="text-align:left;">
Gm9583
</td>
<td style="text-align:left;">
NCBI_Gene:672902,ENSEMBL:ENSMUSG00000112893
</td>
<td style="text-align:left;">
MGI:3779991
</td>
<td style="text-align:left;">
pseudogene
</td>
<td style="text-align:left;">
predicted gene 9583
</td>
</tr>
<tr>
<td style="text-align:left;">
10
</td>
<td style="text-align:left;">
pseudogene
</td>
<td style="text-align:right;">
86.55495
</td>
<td style="text-align:right;">
86.55618
</td>
<td style="text-align:left;">
negative
</td>
<td style="text-align:left;">
MGI_C57BL6J_5434382
</td>
<td style="text-align:left;">
Gm21027
</td>
<td style="text-align:left;">
NCBI_Gene:100534314,ENSEMBL:ENSMUSG00000112115
</td>
<td style="text-align:left;">
MGI:5434382
</td>
<td style="text-align:left;">
pseudogene
</td>
<td style="text-align:left;">
predicted gene, 21027
</td>
</tr>
<tr>
<td style="text-align:left;">
10
</td>
<td style="text-align:left;">
pseudogene
</td>
<td style="text-align:right;">
86.56993
</td>
<td style="text-align:right;">
86.57168
</td>
<td style="text-align:left;">
positive
</td>
<td style="text-align:left;">
MGI_C57BL6J_6097960
</td>
<td style="text-align:left;">
Gm48448
</td>
<td style="text-align:left;">
ENSEMBL:ENSMUSG00000112207
</td>
<td style="text-align:left;">
MGI:6097960
</td>
<td style="text-align:left;">
pseudogene
</td>
<td style="text-align:left;">
predicted gene, 48448
</td>
</tr>
<tr>
<td style="text-align:left;">
10
</td>
<td style="text-align:left;">
pseudogene
</td>
<td style="text-align:right;">
86.57176
</td>
<td style="text-align:right;">
86.57297
</td>
<td style="text-align:left;">
positive
</td>
<td style="text-align:left;">
MGI_C57BL6J_6097963
</td>
<td style="text-align:left;">
Gm48450
</td>
<td style="text-align:left;">
ENSEMBL:ENSMUSG00000112579
</td>
<td style="text-align:left;">
MGI:6097963
</td>
<td style="text-align:left;">
pseudogene
</td>
<td style="text-align:left;">
predicted gene, 48450
</td>
</tr>
<tr>
<td style="text-align:left;">
10
</td>
<td style="text-align:left;">
pseudogene
</td>
<td style="text-align:right;">
86.57304
</td>
<td style="text-align:right;">
86.57425
</td>
<td style="text-align:left;">
positive
</td>
<td style="text-align:left;">
MGI_C57BL6J_6097964
</td>
<td style="text-align:left;">
Gm48451
</td>
<td style="text-align:left;">
ENSEMBL:ENSMUSG00000112562
</td>
<td style="text-align:left;">
MGI:6097964
</td>
<td style="text-align:left;">
pseudogene
</td>
<td style="text-align:left;">
predicted gene, 48451
</td>
</tr>
<tr>
<td style="text-align:left;">
10
</td>
<td style="text-align:left;">
pseudogene
</td>
<td style="text-align:right;">
86.57432
</td>
<td style="text-align:right;">
86.57553
</td>
<td style="text-align:left;">
positive
</td>
<td style="text-align:left;">
MGI_C57BL6J_6097968
</td>
<td style="text-align:left;">
Gm48454
</td>
<td style="text-align:left;">
ENSEMBL:ENSMUSG00000112325
</td>
<td style="text-align:left;">
MGI:6097968
</td>
<td style="text-align:left;">
pseudogene
</td>
<td style="text-align:left;">
predicted gene, 48454
</td>
</tr>
<tr>
<td style="text-align:left;">
10
</td>
<td style="text-align:left;">
pseudogene
</td>
<td style="text-align:right;">
86.57562
</td>
<td style="text-align:right;">
86.57585
</td>
<td style="text-align:left;">
positive
</td>
<td style="text-align:left;">
MGI_C57BL6J_6097974
</td>
<td style="text-align:left;">
Gm48460
</td>
<td style="text-align:left;">
ENSEMBL:ENSMUSG00000112268
</td>
<td style="text-align:left;">
MGI:6097974
</td>
<td style="text-align:left;">
pseudogene
</td>
<td style="text-align:left;">
predicted gene, 48460
</td>
</tr>
<tr>
<td style="text-align:left;">
10
</td>
<td style="text-align:left;">
pseudogene
</td>
<td style="text-align:right;">
86.59235
</td>
<td style="text-align:right;">
86.59366
</td>
<td style="text-align:left;">
positive
</td>
<td style="text-align:left;">
MGI_C57BL6J_5825828
</td>
<td style="text-align:left;">
Gm46191
</td>
<td style="text-align:left;">
NCBI_Gene:108167772,ENSEMBL:ENSMUSG00000112735
</td>
<td style="text-align:left;">
MGI:5825828
</td>
<td style="text-align:left;">
pseudogene
</td>
<td style="text-align:left;">
predicted gene, 46191
</td>
</tr>
<tr>
<td style="text-align:left;">
10
</td>
<td style="text-align:left;">
pseudogene
</td>
<td style="text-align:right;">
86.60092
</td>
<td style="text-align:right;">
86.60144
</td>
<td style="text-align:left;">
positive
</td>
<td style="text-align:left;">
MGI_C57BL6J_6097975
</td>
<td style="text-align:left;">
Gm48461
</td>
<td style="text-align:left;">
ENSEMBL:ENSMUSG00000112010
</td>
<td style="text-align:left;">
MGI:6097975
</td>
<td style="text-align:left;">
pseudogene
</td>
<td style="text-align:left;">
predicted gene, 48461
</td>
</tr>
<tr>
<td style="text-align:left;">
10
</td>
<td style="text-align:left;">
pseudogene
</td>
<td style="text-align:right;">
86.60157
</td>
<td style="text-align:right;">
86.60174
</td>
<td style="text-align:left;">
positive
</td>
<td style="text-align:left;">
MGI_C57BL6J_6097978
</td>
<td style="text-align:left;">
Gm48463
</td>
<td style="text-align:left;">
ENSEMBL:ENSMUSG00000112638
</td>
<td style="text-align:left;">
MGI:6097978
</td>
<td style="text-align:left;">
pseudogene
</td>
<td style="text-align:left;">
predicted gene, 48463
</td>
</tr>
<tr>
<td style="text-align:left;">
10
</td>
<td style="text-align:left;">
pseudogene
</td>
<td style="text-align:right;">
86.60178
</td>
<td style="text-align:right;">
86.60293
</td>
<td style="text-align:left;">
positive
</td>
<td style="text-align:left;">
MGI_C57BL6J_6097979
</td>
<td style="text-align:left;">
Gm48464
</td>
<td style="text-align:left;">
ENSEMBL:ENSMUSG00000112834
</td>
<td style="text-align:left;">
MGI:6097979
</td>
<td style="text-align:left;">
pseudogene
</td>
<td style="text-align:left;">
predicted gene, 48464
</td>
</tr>
<tr>
<td style="text-align:left;">
10
</td>
<td style="text-align:left;">
pseudogene
</td>
<td style="text-align:right;">
86.60301
</td>
<td style="text-align:right;">
86.60420
</td>
<td style="text-align:left;">
positive
</td>
<td style="text-align:left;">
MGI_C57BL6J_6097980
</td>
<td style="text-align:left;">
Gm48465
</td>
<td style="text-align:left;">
ENSEMBL:ENSMUSG00000111998
</td>
<td style="text-align:left;">
MGI:6097980
</td>
<td style="text-align:left;">
pseudogene
</td>
<td style="text-align:left;">
predicted gene, 48465
</td>
</tr>
<tr>
<td style="text-align:left;">
10
</td>
<td style="text-align:left;">
pseudogene
</td>
<td style="text-align:right;">
86.60429
</td>
<td style="text-align:right;">
86.60476
</td>
<td style="text-align:left;">
positive
</td>
<td style="text-align:left;">
MGI_C57BL6J_6097981
</td>
<td style="text-align:left;">
Gm48466
</td>
<td style="text-align:left;">
ENSEMBL:ENSMUSG00000112078
</td>
<td style="text-align:left;">
MGI:6097981
</td>
<td style="text-align:left;">
pseudogene
</td>
<td style="text-align:left;">
predicted gene, 48466
</td>
</tr>
<tr>
<td style="text-align:left;">
10
</td>
<td style="text-align:left;">
pseudogene
</td>
<td style="text-align:right;">
86.62065
</td>
<td style="text-align:right;">
86.62093
</td>
<td style="text-align:left;">
negative
</td>
<td style="text-align:left;">
MGI_C57BL6J_6097988
</td>
<td style="text-align:left;">
Gm48473
</td>
<td style="text-align:left;">
ENSEMBL:ENSMUSG00000112565
</td>
<td style="text-align:left;">
MGI:6097988
</td>
<td style="text-align:left;">
pseudogene
</td>
<td style="text-align:left;">
predicted gene, 48473
</td>
</tr>
<tr>
<td style="text-align:left;">
10
</td>
<td style="text-align:left;">
pseudogene
</td>
<td style="text-align:right;">
86.62098
</td>
<td style="text-align:right;">
86.62213
</td>
<td style="text-align:left;">
negative
</td>
<td style="text-align:left;">
MGI_C57BL6J_6097989
</td>
<td style="text-align:left;">
Gm48474
</td>
<td style="text-align:left;">
ENSEMBL:ENSMUSG00000112699
</td>
<td style="text-align:left;">
MGI:6097989
</td>
<td style="text-align:left;">
pseudogene
</td>
<td style="text-align:left;">
predicted gene, 48474
</td>
</tr>
<tr>
<td style="text-align:left;">
10
</td>
<td style="text-align:left;">
pseudogene
</td>
<td style="text-align:right;">
86.62226
</td>
<td style="text-align:right;">
86.62400
</td>
<td style="text-align:left;">
negative
</td>
<td style="text-align:left;">
MGI_C57BL6J_6097990
</td>
<td style="text-align:left;">
Gm48475
</td>
<td style="text-align:left;">
ENSEMBL:ENSMUSG00000112359
</td>
<td style="text-align:left;">
MGI:6097990
</td>
<td style="text-align:left;">
pseudogene
</td>
<td style="text-align:left;">
predicted gene, 48475
</td>
</tr>
<tr>
<td style="text-align:left;">
10
</td>
<td style="text-align:left;">
pseudogene
</td>
<td style="text-align:right;">
86.63714
</td>
<td style="text-align:right;">
86.63839
</td>
<td style="text-align:left;">
positive
</td>
<td style="text-align:left;">
MGI_C57BL6J_5012174
</td>
<td style="text-align:left;">
Gm19989
</td>
<td style="text-align:left;">
NCBI_Gene:100534315,ENSEMBL:ENSMUSG00000112282
</td>
<td style="text-align:left;">
MGI:5012174
</td>
<td style="text-align:left;">
pseudogene
</td>
<td style="text-align:left;">
predicted gene, 19989
</td>
</tr>
<tr>
<td style="text-align:left;">
10
</td>
<td style="text-align:left;">
pseudogene
</td>
<td style="text-align:right;">
86.64068
</td>
<td style="text-align:right;">
86.64253
</td>
<td style="text-align:left;">
positive
</td>
<td style="text-align:left;">
MGI_C57BL6J_5011061
</td>
<td style="text-align:left;">
Gm18876
</td>
<td style="text-align:left;">
NCBI_Gene:100417881,ENSEMBL:ENSMUSG00000112098
</td>
<td style="text-align:left;">
MGI:5011061
</td>
<td style="text-align:left;">
pseudogene
</td>
<td style="text-align:left;">
predicted gene, 18876
</td>
</tr>
<tr>
<td style="text-align:left;">
10
</td>
<td style="text-align:left;">
gene
</td>
<td style="text-align:right;">
86.65543
</td>
<td style="text-align:right;">
86.67102
</td>
<td style="text-align:left;">
positive
</td>
<td style="text-align:left;">
MGI_C57BL6J_3646116
</td>
<td style="text-align:left;">
Gm5174
</td>
<td style="text-align:left;">
NCBI_Gene:382395,ENSEMBL:ENSMUSG00000090308
</td>
<td style="text-align:left;">
MGI:3646116
</td>
<td style="text-align:left;">
protein coding gene
</td>
<td style="text-align:left;">
predicted gene 5174
</td>
</tr>
<tr>
<td style="text-align:left;">
10
</td>
<td style="text-align:left;">
gene
</td>
<td style="text-align:right;">
86.65740
</td>
<td style="text-align:right;">
86.67102
</td>
<td style="text-align:left;">
positive
</td>
<td style="text-align:left;">
MGI_C57BL6J_6098005
</td>
<td style="text-align:left;">
Gm48485
</td>
<td style="text-align:left;">
ENSEMBL:ENSMUSG00000112666
</td>
<td style="text-align:left;">
MGI:6098005
</td>
<td style="text-align:left;">
lncRNA gene
</td>
<td style="text-align:left;">
predicted gene, 48485
</td>
</tr>
<tr>
<td style="text-align:left;">
10
</td>
<td style="text-align:left;">
pseudogene
</td>
<td style="text-align:right;">
86.68552
</td>
<td style="text-align:right;">
86.69481
</td>
<td style="text-align:left;">
positive
</td>
<td style="text-align:left;">
MGI_C57BL6J_1913513
</td>
<td style="text-align:left;">
1810014B01Rik
</td>
<td style="text-align:left;">
NCBI_Gene:66263,ENSEMBL:ENSMUSG00000097412
</td>
<td style="text-align:left;">
MGI:1913513
</td>
<td style="text-align:left;">
pseudogene
</td>
<td style="text-align:left;">
RIKEN cDNA 1810014B01 gene
</td>
</tr>
<tr>
<td style="text-align:left;">
10
</td>
<td style="text-align:left;">
gene
</td>
<td style="text-align:right;">
86.69021
</td>
<td style="text-align:right;">
86.70551
</td>
<td style="text-align:left;">
negative
</td>
<td style="text-align:left;">
MGI_C57BL6J_98817
</td>
<td style="text-align:left;">
Hsp90b1
</td>
<td style="text-align:left;">
NCBI_Gene:22027,ENSEMBL:ENSMUSG00000020048
</td>
<td style="text-align:left;">
MGI:98817
</td>
<td style="text-align:left;">
protein coding gene
</td>
<td style="text-align:left;">
heat shock protein 90, beta (Grp94), member 1
</td>
</tr>
<tr>
<td style="text-align:left;">
10
</td>
<td style="text-align:left;">
gene
</td>
<td style="text-align:right;">
86.69212
</td>
<td style="text-align:right;">
86.69309
</td>
<td style="text-align:left;">
negative
</td>
<td style="text-align:left;">
MGI_C57BL6J_3708098
</td>
<td style="text-align:left;">
Gm15344
</td>
<td style="text-align:left;">
ENSEMBL:ENSMUSG00000084899
</td>
<td style="text-align:left;">
MGI:3708098
</td>
<td style="text-align:left;">
lncRNA gene
</td>
<td style="text-align:left;">
predicted gene 15344
</td>
</tr>
<tr>
<td style="text-align:left;">
10
</td>
<td style="text-align:left;">
gene
</td>
<td style="text-align:right;">
86.70532
</td>
<td style="text-align:right;">
86.70545
</td>
<td style="text-align:left;">
negative
</td>
<td style="text-align:left;">
MGI_C57BL6J_5455522
</td>
<td style="text-align:left;">
Gm25745
</td>
<td style="text-align:left;">
ENSEMBL:ENSMUSG00000092686
</td>
<td style="text-align:left;">
MGI:5455522
</td>
<td style="text-align:left;">
miRNA gene
</td>
<td style="text-align:left;">
predicted gene, 25745
</td>
</tr>
<tr>
<td style="text-align:left;">
10
</td>
<td style="text-align:left;">
gene
</td>
<td style="text-align:right;">
86.70562
</td>
<td style="text-align:right;">
86.77684
</td>
<td style="text-align:left;">
positive
</td>
<td style="text-align:left;">
MGI_C57BL6J_2387653
</td>
<td style="text-align:left;">
Ttc41
</td>
<td style="text-align:left;">
NCBI_Gene:103220,ENSEMBL:ENSMUSG00000044937
</td>
<td style="text-align:left;">
MGI:2387653
</td>
<td style="text-align:left;">
protein coding gene
</td>
<td style="text-align:left;">
tetratricopeptide repeat domain 41
</td>
</tr>
<tr>
<td style="text-align:left;">
10
</td>
<td style="text-align:left;">
pseudogene
</td>
<td style="text-align:right;">
86.73175
</td>
<td style="text-align:right;">
86.73239
</td>
<td style="text-align:left;">
negative
</td>
<td style="text-align:left;">
MGI_C57BL6J_101929
</td>
<td style="text-align:left;">
Fabp3-ps1
</td>
<td style="text-align:left;">
NCBI_Gene:14074,ENSEMBL:ENSMUSG00000056366
</td>
<td style="text-align:left;">
MGI:101929
</td>
<td style="text-align:left;">
pseudogene
</td>
<td style="text-align:left;">
fatty acid binding protein 3, muscle and heart, pseudogene 1
</td>
</tr>
<tr>
<td style="text-align:left;">
10
</td>
<td style="text-align:left;">
gene
</td>
<td style="text-align:right;">
86.76397
</td>
<td style="text-align:right;">
86.83397
</td>
<td style="text-align:left;">
positive
</td>
<td style="text-align:left;">
MGI_C57BL6J_6121566
</td>
<td style="text-align:left;">
Gm49358
</td>
<td style="text-align:left;">
ENSEMBL:ENSMUSG00000112743
</td>
<td style="text-align:left;">
MGI:6121566
</td>
<td style="text-align:left;">
protein coding gene
</td>
<td style="text-align:left;">
predicted gene, 49358
</td>
</tr>
<tr>
<td style="text-align:left;">
10
</td>
<td style="text-align:left;">
gene
</td>
<td style="text-align:right;">
86.77899
</td>
<td style="text-align:right;">
86.83839
</td>
<td style="text-align:left;">
positive
</td>
<td style="text-align:left;">
MGI_C57BL6J_3513266
</td>
<td style="text-align:left;">
Nt5dc3
</td>
<td style="text-align:left;">
NCBI_Gene:103466,ENSEMBL:ENSMUSG00000054027
</td>
<td style="text-align:left;">
MGI:3513266
</td>
<td style="text-align:left;">
protein coding gene
</td>
<td style="text-align:left;">
5’-nucleotidase domain containing 3
</td>
</tr>
<tr>
<td style="text-align:left;">
10
</td>
<td style="text-align:left;">
gene
</td>
<td style="text-align:right;">
86.83909
</td>
<td style="text-align:right;">
86.87495
</td>
<td style="text-align:left;">
positive
</td>
<td style="text-align:left;">
MGI_C57BL6J_3826557
</td>
<td style="text-align:left;">
Gm16268
</td>
<td style="text-align:left;">
ENSEMBL:ENSMUSG00000089959
</td>
<td style="text-align:left;">
MGI:3826557
</td>
<td style="text-align:left;">
lncRNA gene
</td>
<td style="text-align:left;">
predicted gene 16268
</td>
</tr>
<tr>
<td style="text-align:left;">
10
</td>
<td style="text-align:left;">
gene
</td>
<td style="text-align:right;">
86.84120
</td>
<td style="text-align:right;">
87.00803
</td>
<td style="text-align:left;">
negative
</td>
<td style="text-align:left;">
MGI_C57BL6J_2178743
</td>
<td style="text-align:left;">
Stab2
</td>
<td style="text-align:left;">
NCBI_Gene:192188,ENSEMBL:ENSMUSG00000035459
</td>
<td style="text-align:left;">
MGI:2178743
</td>
<td style="text-align:left;">
protein coding gene
</td>
<td style="text-align:left;">
stabilin 2
</td>
</tr>
<tr>
<td style="text-align:left;">
10
</td>
<td style="text-align:left;">
gene
</td>
<td style="text-align:right;">
86.90876
</td>
<td style="text-align:right;">
86.91001
</td>
<td style="text-align:left;">
positive
</td>
<td style="text-align:left;">
MGI_C57BL6J_3809105
</td>
<td style="text-align:left;">
Gm16280
</td>
<td style="text-align:left;">
ENSEMBL:ENSMUSG00000089881
</td>
<td style="text-align:left;">
MGI:3809105
</td>
<td style="text-align:left;">
lncRNA gene
</td>
<td style="text-align:left;">
predicted gene 16280
</td>
</tr>
<tr>
<td style="text-align:left;">
10
</td>
<td style="text-align:left;">
pseudogene
</td>
<td style="text-align:right;">
86.90948
</td>
<td style="text-align:right;">
86.91272
</td>
<td style="text-align:left;">
positive
</td>
<td style="text-align:left;">
MGI_C57BL6J_3841260
</td>
<td style="text-align:left;">
Gm16270
</td>
<td style="text-align:left;">
NCBI_Gene:100416105,ENSEMBL:ENSMUSG00000089775
</td>
<td style="text-align:left;">
MGI:3841260
</td>
<td style="text-align:left;">
pseudogene
</td>
<td style="text-align:left;">
predicted gene 16270
</td>
</tr>
<tr>
<td style="text-align:left;">
10
</td>
<td style="text-align:left;">
gene
</td>
<td style="text-align:right;">
86.91350
</td>
<td style="text-align:right;">
86.91468
</td>
<td style="text-align:left;">
positive
</td>
<td style="text-align:left;">
MGI_C57BL6J_1890547
</td>
<td style="text-align:left;">
Tex18
</td>
<td style="text-align:left;">
NA
</td>
<td style="text-align:left;">
NA
</td>
<td style="text-align:left;">
unclassified non-coding RNA gene
</td>
<td style="text-align:left;">
testis expressed gene 18
</td>
</tr>
<tr>
<td style="text-align:left;">
10
</td>
<td style="text-align:left;">
gene
</td>
<td style="text-align:right;">
86.97222
</td>
<td style="text-align:right;">
87.02801
</td>
<td style="text-align:left;">
positive
</td>
<td style="text-align:left;">
MGI_C57BL6J_3826610
</td>
<td style="text-align:left;">
Gm16271
</td>
<td style="text-align:left;">
ENSEMBL:ENSMUSG00000089656
</td>
<td style="text-align:left;">
MGI:3826610
</td>
<td style="text-align:left;">
lncRNA gene
</td>
<td style="text-align:left;">
predicted gene 16271
</td>
</tr>
<tr>
<td style="text-align:left;">
10
</td>
<td style="text-align:left;">
pseudogene
</td>
<td style="text-align:right;">
86.99996
</td>
<td style="text-align:right;">
87.00073
</td>
<td style="text-align:left;">
positive
</td>
<td style="text-align:left;">
MGI_C57BL6J_3826558
</td>
<td style="text-align:left;">
Gm16269
</td>
<td style="text-align:left;">
NCBI_Gene:108167826,ENSEMBL:ENSMUSG00000089666
</td>
<td style="text-align:left;">
MGI:3826558
</td>
<td style="text-align:left;">
pseudogene
</td>
<td style="text-align:left;">
predicted gene 16269
</td>
</tr>
<tr>
<td style="text-align:left;">
10
</td>
<td style="text-align:left;">
gene
</td>
<td style="text-align:right;">
87.01893
</td>
<td style="text-align:right;">
87.05883
</td>
<td style="text-align:left;">
negative
</td>
<td style="text-align:left;">
MGI_C57BL6J_5595966
</td>
<td style="text-align:left;">
Gm36807
</td>
<td style="text-align:left;">
NCBI_Gene:102640833
</td>
<td style="text-align:left;">
MGI:5595966
</td>
<td style="text-align:left;">
protein coding gene
</td>
<td style="text-align:left;">
predicted gene, 36807
</td>
</tr>
<tr>
<td style="text-align:left;">
10
</td>
<td style="text-align:left;">
gene
</td>
<td style="text-align:right;">
87.05805
</td>
<td style="text-align:right;">
87.23060
</td>
<td style="text-align:left;">
positive
</td>
<td style="text-align:left;">
MGI_C57BL6J_1923890
</td>
<td style="text-align:left;">
1700113H08Rik
</td>
<td style="text-align:left;">
NCBI_Gene:76640,ENSEMBL:ENSMUSG00000047129
</td>
<td style="text-align:left;">
MGI:1923890
</td>
<td style="text-align:left;">
protein coding gene
</td>
<td style="text-align:left;">
RIKEN cDNA 1700113H08 gene
</td>
</tr>
<tr>
<td style="text-align:left;">
10
</td>
<td style="text-align:left;">
gene
</td>
<td style="text-align:right;">
87.23076
</td>
<td style="text-align:right;">
87.29756
</td>
<td style="text-align:left;">
positive
</td>
<td style="text-align:left;">
MGI_C57BL6J_1922532
</td>
<td style="text-align:left;">
4930555G07Rik
</td>
<td style="text-align:left;">
ENSEMBL:ENSMUSG00000112213
</td>
<td style="text-align:left;">
MGI:1922532
</td>
<td style="text-align:left;">
lncRNA gene
</td>
<td style="text-align:left;">
RIKEN cDNA 4930555G07 gene
</td>
</tr>
<tr>
<td style="text-align:left;">
10
</td>
<td style="text-align:left;">
gene
</td>
<td style="text-align:right;">
87.28949
</td>
<td style="text-align:right;">
87.29183
</td>
<td style="text-align:left;">
positive
</td>
<td style="text-align:left;">
MGI_C57BL6J_3641850
</td>
<td style="text-align:left;">
Gm10764
</td>
<td style="text-align:left;">
ENSEMBL:ENSMUSG00000103862
</td>
<td style="text-align:left;">
MGI:3641850
</td>
<td style="text-align:left;">
unclassified gene
</td>
<td style="text-align:left;">
predicted gene 10764
</td>
</tr>
<tr>
<td style="text-align:left;">
10
</td>
<td style="text-align:left;">
gene
</td>
<td style="text-align:right;">
87.36404
</td>
<td style="text-align:right;">
87.36415
</td>
<td style="text-align:left;">
negative
</td>
<td style="text-align:left;">
MGI_C57BL6J_5452968
</td>
<td style="text-align:left;">
Gm23191
</td>
<td style="text-align:left;">
ENSEMBL:ENSMUSG00000096692
</td>
<td style="text-align:left;">
MGI:5452968
</td>
<td style="text-align:left;">
snRNA gene
</td>
<td style="text-align:left;">
predicted gene, 23191
</td>
</tr>
<tr>
<td style="text-align:left;">
10
</td>
<td style="text-align:left;">
gene
</td>
<td style="text-align:right;">
87.49082
</td>
<td style="text-align:right;">
87.49366
</td>
<td style="text-align:left;">
negative
</td>
<td style="text-align:left;">
MGI_C57BL6J_96919
</td>
<td style="text-align:left;">
Ascl1
</td>
<td style="text-align:left;">
NCBI_Gene:17172,ENSEMBL:ENSMUSG00000020052
</td>
<td style="text-align:left;">
MGI:96919
</td>
<td style="text-align:left;">
protein coding gene
</td>
<td style="text-align:left;">
achaete-scute family bHLH transcription factor 1
</td>
</tr>
<tr>
<td style="text-align:left;">
10
</td>
<td style="text-align:left;">
gene
</td>
<td style="text-align:right;">
87.50283
</td>
<td style="text-align:right;">
87.50862
</td>
<td style="text-align:left;">
negative
</td>
<td style="text-align:left;">
MGI_C57BL6J_5589074
</td>
<td style="text-align:left;">
Gm29915
</td>
<td style="text-align:left;">
NCBI_Gene:102631620
</td>
<td style="text-align:left;">
MGI:5589074
</td>
<td style="text-align:left;">
lncRNA gene
</td>
<td style="text-align:left;">
predicted gene, 29915
</td>
</tr>
<tr>
<td style="text-align:left;">
10
</td>
<td style="text-align:left;">
pseudogene
</td>
<td style="text-align:right;">
87.50786
</td>
<td style="text-align:right;">
87.50848
</td>
<td style="text-align:left;">
positive
</td>
<td style="text-align:left;">
MGI_C57BL6J_6097476
</td>
<td style="text-align:left;">
Gm48120
</td>
<td style="text-align:left;">
ENSEMBL:ENSMUSG00000112460
</td>
<td style="text-align:left;">
MGI:6097476
</td>
<td style="text-align:left;">
pseudogene
</td>
<td style="text-align:left;">
predicted gene, 48120
</td>
</tr>
<tr>
<td style="text-align:left;">
10
</td>
<td style="text-align:left;">
gene
</td>
<td style="text-align:right;">
87.52179
</td>
<td style="text-align:right;">
87.58414
</td>
<td style="text-align:left;">
positive
</td>
<td style="text-align:left;">
MGI_C57BL6J_97473
</td>
<td style="text-align:left;">
Pah
</td>
<td style="text-align:left;">
NCBI_Gene:18478,ENSEMBL:ENSMUSG00000020051
</td>
<td style="text-align:left;">
MGI:97473
</td>
<td style="text-align:left;">
protein coding gene
</td>
<td style="text-align:left;">
phenylalanine hydroxylase
</td>
</tr>
<tr>
<td style="text-align:left;">
10
</td>
<td style="text-align:left;">
gene
</td>
<td style="text-align:right;">
87.57144
</td>
<td style="text-align:right;">
87.57175
</td>
<td style="text-align:left;">
positive
</td>
<td style="text-align:left;">
MGI_C57BL6J_1925634
</td>
<td style="text-align:left;">
2210420L05Rik
</td>
<td style="text-align:left;">
NA
</td>
<td style="text-align:left;">
NA
</td>
<td style="text-align:left;">
unclassified gene
</td>
<td style="text-align:left;">
RIKEN cDNA 2210420L05 gene
</td>
</tr>
<tr>
<td style="text-align:left;">
10
</td>
<td style="text-align:left;">
pseudogene
</td>
<td style="text-align:right;">
87.68858
</td>
<td style="text-align:right;">
87.68903
</td>
<td style="text-align:left;">
positive
</td>
<td style="text-align:left;">
MGI_C57BL6J_6097580
</td>
<td style="text-align:left;">
Gm48195
</td>
<td style="text-align:left;">
ENSEMBL:ENSMUSG00000112536
</td>
<td style="text-align:left;">
MGI:6097580
</td>
<td style="text-align:left;">
pseudogene
</td>
<td style="text-align:left;">
predicted gene, 48195
</td>
</tr>
<tr>
<td style="text-align:left;">
10
</td>
<td style="text-align:left;">
pseudogene
</td>
<td style="text-align:right;">
87.75914
</td>
<td style="text-align:right;">
87.75957
</td>
<td style="text-align:left;">
positive
</td>
<td style="text-align:left;">
MGI_C57BL6J_5012191
</td>
<td style="text-align:left;">
Gm20006
</td>
<td style="text-align:left;">
NCBI_Gene:100504000
</td>
<td style="text-align:left;">
MGI:5012191
</td>
<td style="text-align:left;">
pseudogene
</td>
<td style="text-align:left;">
predicted gene, 20006
</td>
</tr>
<tr>
<td style="text-align:left;">
10
</td>
<td style="text-align:left;">
gene
</td>
<td style="text-align:right;">
87.77724
</td>
<td style="text-align:right;">
87.79837
</td>
<td style="text-align:left;">
negative
</td>
<td style="text-align:left;">
MGI_C57BL6J_5589232
</td>
<td style="text-align:left;">
Gm30073
</td>
<td style="text-align:left;">
NCBI_Gene:102631839
</td>
<td style="text-align:left;">
MGI:5589232
</td>
<td style="text-align:left;">
lncRNA gene
</td>
<td style="text-align:left;">
predicted gene, 30073
</td>
</tr>
<tr>
<td style="text-align:left;">
10
</td>
<td style="text-align:left;">
gene
</td>
<td style="text-align:right;">
87.82638
</td>
<td style="text-align:right;">
87.86472
</td>
<td style="text-align:left;">
negative
</td>
<td style="text-align:left;">
MGI_C57BL6J_3841250
</td>
<td style="text-align:left;">
Igf1os
</td>
<td style="text-align:left;">
NCBI_Gene:627270,ENSEMBL:ENSMUSG00000058934
</td>
<td style="text-align:left;">
MGI:3841250
</td>
<td style="text-align:left;">
antisense lncRNA gene
</td>
<td style="text-align:left;">
insulin-like growth factor 1, opposite strand
</td>
</tr>
<tr>
<td style="text-align:left;">
10
</td>
<td style="text-align:left;">
gene
</td>
<td style="text-align:right;">
87.82989
</td>
<td style="text-align:right;">
87.82996
</td>
<td style="text-align:left;">
negative
</td>
<td style="text-align:left;">
MGI_C57BL6J_4413968
</td>
<td style="text-align:left;">
n-TKctt21
</td>
<td style="text-align:left;">
NCBI_Gene:102467349
</td>
<td style="text-align:left;">
MGI:4413968
</td>
<td style="text-align:left;">
tRNA gene
</td>
<td style="text-align:left;">
nuclear encoded tRNA lysine 21 (anticodon TTT)
</td>
</tr>
<tr>
<td style="text-align:left;">
10
</td>
<td style="text-align:left;">
gene
</td>
<td style="text-align:right;">
87.85827
</td>
<td style="text-align:right;">
87.93705
</td>
<td style="text-align:left;">
positive
</td>
<td style="text-align:left;">
MGI_C57BL6J_96432
</td>
<td style="text-align:left;">
Igf1
</td>
<td style="text-align:left;">
NCBI_Gene:16000,ENSEMBL:ENSMUSG00000020053
</td>
<td style="text-align:left;">
MGI:96432
</td>
<td style="text-align:left;">
protein coding gene
</td>
<td style="text-align:left;">
insulin-like growth factor 1
</td>
</tr>
<tr>
<td style="text-align:left;">
10
</td>
<td style="text-align:left;">
pseudogene
</td>
<td style="text-align:right;">
87.96667
</td>
<td style="text-align:right;">
87.96802
</td>
<td style="text-align:left;">
positive
</td>
<td style="text-align:left;">
MGI_C57BL6J_98879
</td>
<td style="text-align:left;">
Tyms-ps
</td>
<td style="text-align:left;">
NCBI_Gene:22172,ENSEMBL:ENSMUSG00000112222
</td>
<td style="text-align:left;">
MGI:98879
</td>
<td style="text-align:left;">
pseudogene
</td>
<td style="text-align:left;">
thymidylate synthase, pseudogene
</td>
</tr>
<tr>
<td style="text-align:left;">
10
</td>
<td style="text-align:left;">
gene
</td>
<td style="text-align:right;">
87.98046
</td>
<td style="text-align:right;">
87.98422
</td>
<td style="text-align:left;">
positive
</td>
<td style="text-align:left;">
MGI_C57BL6J_6097752
</td>
<td style="text-align:left;">
Gm48306
</td>
<td style="text-align:left;">
ENSEMBL:ENSMUSG00000112183
</td>
<td style="text-align:left;">
MGI:6097752
</td>
<td style="text-align:left;">
lncRNA gene
</td>
<td style="text-align:left;">
predicted gene, 48306
</td>
</tr>
<tr>
<td style="text-align:left;">
10
</td>
<td style="text-align:left;">
gene
</td>
<td style="text-align:right;">
88.03822
</td>
<td style="text-align:right;">
88.03968
</td>
<td style="text-align:left;">
positive
</td>
<td style="text-align:left;">
MGI_C57BL6J_6097754
</td>
<td style="text-align:left;">
Gm48307
</td>
<td style="text-align:left;">
ENSEMBL:ENSMUSG00000112235
</td>
<td style="text-align:left;">
MGI:6097754
</td>
<td style="text-align:left;">
unclassified gene
</td>
<td style="text-align:left;">
predicted gene, 48307
</td>
</tr>
<tr>
<td style="text-align:left;">
10
</td>
<td style="text-align:left;">
gene
</td>
<td style="text-align:right;">
88.03822
</td>
<td style="text-align:right;">
88.03968
</td>
<td style="text-align:left;">
negative
</td>
<td style="text-align:left;">
MGI_C57BL6J_3641951
</td>
<td style="text-align:left;">
Gm10763
</td>
<td style="text-align:left;">
NA
</td>
<td style="text-align:left;">
NA
</td>
<td style="text-align:left;">
unclassified gene
</td>
<td style="text-align:left;">
predicted gene 10763
</td>
</tr>
<tr>
<td style="text-align:left;">
10
</td>
<td style="text-align:left;">
gene
</td>
<td style="text-align:right;">
88.06807
</td>
<td style="text-align:right;">
88.09035
</td>
<td style="text-align:left;">
negative
</td>
<td style="text-align:left;">
MGI_C57BL6J_5589392
</td>
<td style="text-align:left;">
Gm30233
</td>
<td style="text-align:left;">
NCBI_Gene:102632060
</td>
<td style="text-align:left;">
MGI:5589392
</td>
<td style="text-align:left;">
lncRNA gene
</td>
<td style="text-align:left;">
predicted gene, 30233
</td>
</tr>
<tr>
<td style="text-align:left;">
10
</td>
<td style="text-align:left;">
gene
</td>
<td style="text-align:right;">
88.08820
</td>
<td style="text-align:right;">
88.08982
</td>
<td style="text-align:left;">
positive
</td>
<td style="text-align:left;">
MGI_C57BL6J_6097756
</td>
<td style="text-align:left;">
Gm48308
</td>
<td style="text-align:left;">
ENSEMBL:ENSMUSG00000112932
</td>
<td style="text-align:left;">
MGI:6097756
</td>
<td style="text-align:left;">
unclassified gene
</td>
<td style="text-align:left;">
predicted gene, 48308
</td>
</tr>
<tr>
<td style="text-align:left;">
10
</td>
<td style="text-align:left;">
gene
</td>
<td style="text-align:right;">
88.09107
</td>
<td style="text-align:right;">
88.09238
</td>
<td style="text-align:left;">
positive
</td>
<td style="text-align:left;">
MGI_C57BL6J_97629
</td>
<td style="text-align:left;">
Pmch
</td>
<td style="text-align:left;">
NCBI_Gene:110312,ENSEMBL:ENSMUSG00000035383
</td>
<td style="text-align:left;">
MGI:97629
</td>
<td style="text-align:left;">
protein coding gene
</td>
<td style="text-align:left;">
pro-melanin-concentrating hormone
</td>
</tr>
<tr>
<td style="text-align:left;">
10
</td>
<td style="text-align:left;">
gene
</td>
<td style="text-align:right;">
88.09140
</td>
<td style="text-align:right;">
88.14695
</td>
<td style="text-align:left;">
negative
</td>
<td style="text-align:left;">
MGI_C57BL6J_1922567
</td>
<td style="text-align:left;">
Parpbp
</td>
<td style="text-align:left;">
NCBI_Gene:75317,ENSEMBL:ENSMUSG00000035365
</td>
<td style="text-align:left;">
MGI:1922567
</td>
<td style="text-align:left;">
protein coding gene
</td>
<td style="text-align:left;">
PARP1 binding protein
</td>
</tr>
<tr>
<td style="text-align:left;">
10
</td>
<td style="text-align:left;">
gene
</td>
<td style="text-align:right;">
88.13438
</td>
<td style="text-align:right;">
88.13758
</td>
<td style="text-align:left;">
negative
</td>
<td style="text-align:left;">
MGI_C57BL6J_3647484
</td>
<td style="text-align:left;">
Gm6653
</td>
<td style="text-align:left;">
NCBI_Gene:626115,ENSEMBL:ENSMUSG00000112026
</td>
<td style="text-align:left;">
MGI:3647484
</td>
<td style="text-align:left;">
unclassified gene
</td>
<td style="text-align:left;">
predicted gene 6653
</td>
</tr>
<tr>
<td style="text-align:left;">
10
</td>
<td style="text-align:left;">
gene
</td>
<td style="text-align:right;">
88.13592
</td>
<td style="text-align:right;">
88.13753
</td>
<td style="text-align:left;">
positive
</td>
<td style="text-align:left;">
MGI_C57BL6J_1918118
</td>
<td style="text-align:left;">
4921515L22Rik
</td>
<td style="text-align:left;">
ENSEMBL:ENSMUSG00000112406
</td>
<td style="text-align:left;">
MGI:1918118
</td>
<td style="text-align:left;">
lncRNA gene
</td>
<td style="text-align:left;">
RIKEN cDNA 4921515L22 gene
</td>
</tr>
<tr>
<td style="text-align:left;">
10
</td>
<td style="text-align:left;">
gene
</td>
<td style="text-align:right;">
88.14697
</td>
<td style="text-align:right;">
88.17839
</td>
<td style="text-align:left;">
positive
</td>
<td style="text-align:left;">
MGI_C57BL6J_1919964
</td>
<td style="text-align:left;">
Nup37
</td>
<td style="text-align:left;">
NCBI_Gene:69736,ENSEMBL:ENSMUSG00000035351
</td>
<td style="text-align:left;">
MGI:1919964
</td>
<td style="text-align:left;">
protein coding gene
</td>
<td style="text-align:left;">
nucleoporin 37
</td>
</tr>
<tr>
<td style="text-align:left;">
10
</td>
<td style="text-align:left;">
gene
</td>
<td style="text-align:right;">
88.16496
</td>
<td style="text-align:right;">
88.16507
</td>
<td style="text-align:left;">
negative
</td>
<td style="text-align:left;">
MGI_C57BL6J_5456134
</td>
<td style="text-align:left;">
Gm26357
</td>
<td style="text-align:left;">
ENSEMBL:ENSMUSG00000088843
</td>
<td style="text-align:left;">
MGI:5456134
</td>
<td style="text-align:left;">
snRNA gene
</td>
<td style="text-align:left;">
predicted gene, 26357
</td>
</tr>
<tr>
<td style="text-align:left;">
10
</td>
<td style="text-align:left;">
gene
</td>
<td style="text-align:right;">
88.20109
</td>
<td style="text-align:right;">
88.26082
</td>
<td style="text-align:left;">
positive
</td>
<td style="text-align:left;">
MGI_C57BL6J_1914532
</td>
<td style="text-align:left;">
Washc3
</td>
<td style="text-align:left;">
NCBI_Gene:67282,ENSEMBL:ENSMUSG00000020056
</td>
<td style="text-align:left;">
MGI:1914532
</td>
<td style="text-align:left;">
protein coding gene
</td>
<td style="text-align:left;">
WASH complex subunit 3
</td>
</tr>
<tr>
<td style="text-align:left;">
10
</td>
<td style="text-align:left;">
pseudogene
</td>
<td style="text-align:right;">
88.24742
</td>
<td style="text-align:right;">
88.24863
</td>
<td style="text-align:left;">
positive
</td>
<td style="text-align:left;">
MGI_C57BL6J_3648865
</td>
<td style="text-align:left;">
Gm8415
</td>
<td style="text-align:left;">
NCBI_Gene:667000
</td>
<td style="text-align:left;">
MGI:3648865
</td>
<td style="text-align:left;">
pseudogene
</td>
<td style="text-align:left;">
predicted gene 8415
</td>
</tr>
<tr>
<td style="text-align:left;">
10
</td>
<td style="text-align:left;">
pseudogene
</td>
<td style="text-align:right;">
88.27079
</td>
<td style="text-align:right;">
88.27325
</td>
<td style="text-align:left;">
negative
</td>
<td style="text-align:left;">
MGI_C57BL6J_5010165
</td>
<td style="text-align:left;">
Gm17980
</td>
<td style="text-align:left;">
NCBI_Gene:100416224,ENSEMBL:ENSMUSG00000112259
</td>
<td style="text-align:left;">
MGI:5010165
</td>
<td style="text-align:left;">
pseudogene
</td>
<td style="text-align:left;">
predicted gene, 17980
</td>
</tr>
<tr>
<td style="text-align:left;">
10
</td>
<td style="text-align:left;">
pseudogene
</td>
<td style="text-align:right;">
88.30981
</td>
<td style="text-align:right;">
88.31013
</td>
<td style="text-align:left;">
negative
</td>
<td style="text-align:left;">
MGI_C57BL6J_6097991
</td>
<td style="text-align:left;">
Gm48476
</td>
<td style="text-align:left;">
ENSEMBL:ENSMUSG00000112715
</td>
<td style="text-align:left;">
MGI:6097991
</td>
<td style="text-align:left;">
pseudogene
</td>
<td style="text-align:left;">
predicted gene, 48476
</td>
</tr>
<tr>
<td style="text-align:left;">
10
</td>
<td style="text-align:left;">
gene
</td>
<td style="text-align:right;">
88.32280
</td>
<td style="text-align:right;">
88.37908
</td>
<td style="text-align:left;">
negative
</td>
<td style="text-align:left;">
MGI_C57BL6J_1918962
</td>
<td style="text-align:left;">
Dram1
</td>
<td style="text-align:left;">
NCBI_Gene:71712,ENSEMBL:ENSMUSG00000020057
</td>
<td style="text-align:left;">
MGI:1918962
</td>
<td style="text-align:left;">
protein coding gene
</td>
<td style="text-align:left;">
DNA-damage regulated autophagy modulator 1
</td>
</tr>
<tr>
<td style="text-align:left;">
10
</td>
<td style="text-align:left;">
gene
</td>
<td style="text-align:right;">
88.35732
</td>
<td style="text-align:right;">
88.37164
</td>
<td style="text-align:left;">
positive
</td>
<td style="text-align:left;">
MGI_C57BL6J_5623619
</td>
<td style="text-align:left;">
Gm40734
</td>
<td style="text-align:left;">
NCBI_Gene:105245256
</td>
<td style="text-align:left;">
MGI:5623619
</td>
<td style="text-align:left;">
lncRNA gene
</td>
<td style="text-align:left;">
predicted gene, 40734
</td>
</tr>
<tr>
<td style="text-align:left;">
10
</td>
<td style="text-align:left;">
gene
</td>
<td style="text-align:right;">
88.36805
</td>
<td style="text-align:right;">
88.37913
</td>
<td style="text-align:left;">
negative
</td>
<td style="text-align:left;">
MGI_C57BL6J_5623618
</td>
<td style="text-align:left;">
Gm40733
</td>
<td style="text-align:left;">
NCBI_Gene:105245255
</td>
<td style="text-align:left;">
MGI:5623618
</td>
<td style="text-align:left;">
lncRNA gene
</td>
<td style="text-align:left;">
predicted gene, 40733
</td>
</tr>
<tr>
<td style="text-align:left;">
10
</td>
<td style="text-align:left;">
gene
</td>
<td style="text-align:right;">
88.37913
</td>
<td style="text-align:right;">
88.44733
</td>
<td style="text-align:left;">
positive
</td>
<td style="text-align:left;">
MGI_C57BL6J_3643902
</td>
<td style="text-align:left;">
Gnptab
</td>
<td style="text-align:left;">
NCBI_Gene:432486,ENSEMBL:ENSMUSG00000035311
</td>
<td style="text-align:left;">
MGI:3643902
</td>
<td style="text-align:left;">
protein coding gene
</td>
<td style="text-align:left;">
N-acetylglucosamine-1-phosphate transferase, alpha and beta subunits
</td>
</tr>
<tr>
<td style="text-align:left;">
10
</td>
<td style="text-align:left;">
gene
</td>
<td style="text-align:right;">
88.37924
</td>
<td style="text-align:right;">
88.38371
</td>
<td style="text-align:left;">
negative
</td>
<td style="text-align:left;">
MGI_C57BL6J_5477040
</td>
<td style="text-align:left;">
Gm26546
</td>
<td style="text-align:left;">
ENSEMBL:ENSMUSG00000096945
</td>
<td style="text-align:left;">
MGI:5477040
</td>
<td style="text-align:left;">
lncRNA gene
</td>
<td style="text-align:left;">
predicted gene, 26546
</td>
</tr>
<tr>
<td style="text-align:left;">
10
</td>
<td style="text-align:left;">
gene
</td>
<td style="text-align:right;">
88.41550
</td>
<td style="text-align:right;">
88.41589
</td>
<td style="text-align:left;">
positive
</td>
<td style="text-align:left;">
MGI_C57BL6J_1925854
</td>
<td style="text-align:left;">
B430006N09Rik
</td>
<td style="text-align:left;">
NA
</td>
<td style="text-align:left;">
NA
</td>
<td style="text-align:left;">
unclassified non-coding RNA gene
</td>
<td style="text-align:left;">
RIKEN cDNA B430006N09 gene
</td>
</tr>
<tr>
<td style="text-align:left;">
10
</td>
<td style="text-align:left;">
gene
</td>
<td style="text-align:right;">
88.44424
</td>
<td style="text-align:right;">
88.44989
</td>
<td style="text-align:left;">
negative
</td>
<td style="text-align:left;">
MGI_C57BL6J_3801878
</td>
<td style="text-align:left;">
Gm16235
</td>
<td style="text-align:left;">
ENSEMBL:ENSMUSG00000087429
</td>
<td style="text-align:left;">
MGI:3801878
</td>
<td style="text-align:left;">
lncRNA gene
</td>
<td style="text-align:left;">
predicted gene 16235
</td>
</tr>
<tr>
<td style="text-align:left;">
10
</td>
<td style="text-align:left;">
gene
</td>
<td style="text-align:right;">
88.45274
</td>
<td style="text-align:right;">
88.50407
</td>
<td style="text-align:left;">
negative
</td>
<td style="text-align:left;">
MGI_C57BL6J_2384841
</td>
<td style="text-align:left;">
Chpt1
</td>
<td style="text-align:left;">
NCBI_Gene:212862,ENSEMBL:ENSMUSG00000060002
</td>
<td style="text-align:left;">
MGI:2384841
</td>
<td style="text-align:left;">
protein coding gene
</td>
<td style="text-align:left;">
choline phosphotransferase 1
</td>
</tr>
<tr>
<td style="text-align:left;">
10
</td>
<td style="text-align:left;">
gene
</td>
<td style="text-align:right;">
88.45957
</td>
<td style="text-align:right;">
88.47324
</td>
<td style="text-align:left;">
positive
</td>
<td style="text-align:left;">
MGI_C57BL6J_109542
</td>
<td style="text-align:left;">
Sycp3
</td>
<td style="text-align:left;">
NCBI_Gene:20962,ENSEMBL:ENSMUSG00000020059
</td>
<td style="text-align:left;">
MGI:109542
</td>
<td style="text-align:left;">
protein coding gene
</td>
<td style="text-align:left;">
synaptonemal complex protein 3
</td>
</tr>
<tr>
<td style="text-align:left;">
10
</td>
<td style="text-align:left;">
gene
</td>
<td style="text-align:right;">
88.51828
</td>
<td style="text-align:right;">
88.60524
</td>
<td style="text-align:left;">
negative
</td>
<td style="text-align:left;">
MGI_C57BL6J_1336213
</td>
<td style="text-align:left;">
Mybpc1
</td>
<td style="text-align:left;">
NCBI_Gene:109272,ENSEMBL:ENSMUSG00000020061
</td>
<td style="text-align:left;">
MGI:1336213
</td>
<td style="text-align:left;">
protein coding gene
</td>
<td style="text-align:left;">
myosin binding protein C, slow-type
</td>
</tr>
<tr>
<td style="text-align:left;">
10
</td>
<td style="text-align:left;">
gene
</td>
<td style="text-align:right;">
88.60247
</td>
<td style="text-align:right;">
88.60534
</td>
<td style="text-align:left;">
positive
</td>
<td style="text-align:left;">
MGI_C57BL6J_6098429
</td>
<td style="text-align:left;">
Gm48752
</td>
<td style="text-align:left;">
ENSEMBL:ENSMUSG00000112841
</td>
<td style="text-align:left;">
MGI:6098429
</td>
<td style="text-align:left;">
unclassified gene
</td>
<td style="text-align:left;">
predicted gene, 48752
</td>
</tr>
<tr>
<td style="text-align:left;">
10
</td>
<td style="text-align:left;">
pseudogene
</td>
<td style="text-align:right;">
88.64136
</td>
<td style="text-align:right;">
88.64221
</td>
<td style="text-align:left;">
positive
</td>
<td style="text-align:left;">
MGI_C57BL6J_3643053
</td>
<td style="text-align:left;">
Gm8435
</td>
<td style="text-align:left;">
NCBI_Gene:667046,ENSEMBL:ENSMUSG00000112231
</td>
<td style="text-align:left;">
MGI:3643053
</td>
<td style="text-align:left;">
pseudogene
</td>
<td style="text-align:left;">
predicted gene 8435
</td>
</tr>
<tr>
<td style="text-align:left;">
10
</td>
<td style="text-align:left;">
pseudogene
</td>
<td style="text-align:right;">
88.65011
</td>
<td style="text-align:right;">
88.65823
</td>
<td style="text-align:left;">
negative
</td>
<td style="text-align:left;">
MGI_C57BL6J_6097458
</td>
<td style="text-align:left;">
Gm48106
</td>
<td style="text-align:left;">
ENSEMBL:ENSMUSG00000112428
</td>
<td style="text-align:left;">
MGI:6097458
</td>
<td style="text-align:left;">
pseudogene
</td>
<td style="text-align:left;">
predicted gene, 48106
</td>
</tr>
<tr>
<td style="text-align:left;">
10
</td>
<td style="text-align:left;">
pseudogene
</td>
<td style="text-align:right;">
88.67050
</td>
<td style="text-align:right;">
88.67127
</td>
<td style="text-align:left;">
negative
</td>
<td style="text-align:left;">
MGI_C57BL6J_6097461
</td>
<td style="text-align:left;">
Gm48108
</td>
<td style="text-align:left;">
ENSEMBL:ENSMUSG00000112869
</td>
<td style="text-align:left;">
MGI:6097461
</td>
<td style="text-align:left;">
pseudogene
</td>
<td style="text-align:left;">
predicted gene, 48108
</td>
</tr>
<tr>
<td style="text-align:left;">
10
</td>
<td style="text-align:left;">
pseudogene
</td>
<td style="text-align:right;">
88.67302
</td>
<td style="text-align:right;">
88.67339
</td>
<td style="text-align:left;">
positive
</td>
<td style="text-align:left;">
MGI_C57BL6J_5588966
</td>
<td style="text-align:left;">
Gm29807
</td>
<td style="text-align:left;">
NCBI_Gene:101056238,ENSEMBL:ENSMUSG00000112885
</td>
<td style="text-align:left;">
MGI:5588966
</td>
<td style="text-align:left;">
pseudogene
</td>
<td style="text-align:left;">
predicted gene, 29807
</td>
</tr>
<tr>
<td style="text-align:left;">
10
</td>
<td style="text-align:left;">
gene
</td>
<td style="text-align:right;">
88.67477
</td>
<td style="text-align:right;">
88.68502
</td>
<td style="text-align:left;">
negative
</td>
<td style="text-align:left;">
MGI_C57BL6J_1341168
</td>
<td style="text-align:left;">
Spic
</td>
<td style="text-align:left;">
NCBI_Gene:20728,ENSEMBL:ENSMUSG00000004359
</td>
<td style="text-align:left;">
MGI:1341168
</td>
<td style="text-align:left;">
protein coding gene
</td>
<td style="text-align:left;">
Spi-C transcription factor (Spi-1/PU.1 related)
</td>
</tr>
<tr>
<td style="text-align:left;">
10
</td>
<td style="text-align:left;">
pseudogene
</td>
<td style="text-align:right;">
88.69101
</td>
<td style="text-align:right;">
88.70511
</td>
<td style="text-align:left;">
positive
</td>
<td style="text-align:left;">
MGI_C57BL6J_5825829
</td>
<td style="text-align:left;">
Gm46192
</td>
<td style="text-align:left;">
NCBI_Gene:108167773,ENSEMBL:ENSMUSG00000111919
</td>
<td style="text-align:left;">
MGI:5825829
</td>
<td style="text-align:left;">
pseudogene
</td>
<td style="text-align:left;">
predicted gene, 46192
</td>
</tr>
<tr>
<td style="text-align:left;">
10
</td>
<td style="text-align:left;">
gene
</td>
<td style="text-align:right;">
88.71001
</td>
<td style="text-align:right;">
88.71013
</td>
<td style="text-align:left;">
negative
</td>
<td style="text-align:left;">
MGI_C57BL6J_4421927
</td>
<td style="text-align:left;">
n-R5s79
</td>
<td style="text-align:left;">
ENSEMBL:ENSMUSG00000065005
</td>
<td style="text-align:left;">
MGI:4421927
</td>
<td style="text-align:left;">
rRNA gene
</td>
<td style="text-align:left;">
nuclear encoded rRNA 5S 79
</td>
</tr>
<tr>
<td style="text-align:left;">
10
</td>
<td style="text-align:left;">
pseudogene
</td>
<td style="text-align:right;">
88.71565
</td>
<td style="text-align:right;">
88.71609
</td>
<td style="text-align:left;">
negative
</td>
<td style="text-align:left;">
MGI_C57BL6J_6097511
</td>
<td style="text-align:left;">
Gm48144
</td>
<td style="text-align:left;">
ENSEMBL:ENSMUSG00000112467
</td>
<td style="text-align:left;">
MGI:6097511
</td>
<td style="text-align:left;">
pseudogene
</td>
<td style="text-align:left;">
predicted gene, 48144
</td>
</tr>
<tr>
<td style="text-align:left;">
10
</td>
<td style="text-align:left;">
pseudogene
</td>
<td style="text-align:right;">
88.72954
</td>
<td style="text-align:right;">
88.73110
</td>
<td style="text-align:left;">
negative
</td>
<td style="text-align:left;">
MGI_C57BL6J_3643371
</td>
<td style="text-align:left;">
Gm4925
</td>
<td style="text-align:left;">
NCBI_Gene:237433,ENSEMBL:ENSMUSG00000069540
</td>
<td style="text-align:left;">
MGI:3643371
</td>
<td style="text-align:left;">
pseudogene
</td>
<td style="text-align:left;">
predicted gene 4925
</td>
</tr>
<tr>
<td style="text-align:left;">
10
</td>
<td style="text-align:left;">
gene
</td>
<td style="text-align:right;">
88.73086
</td>
<td style="text-align:right;">
88.74409
</td>
<td style="text-align:left;">
positive
</td>
<td style="text-align:left;">
MGI_C57BL6J_99436
</td>
<td style="text-align:left;">
Arl1
</td>
<td style="text-align:left;">
NCBI_Gene:104303,ENSEMBL:ENSMUSG00000060904
</td>
<td style="text-align:left;">
MGI:99436
</td>
<td style="text-align:left;">
protein coding gene
</td>
<td style="text-align:left;">
ADP-ribosylation factor-like 1
</td>
</tr>
<tr>
<td style="text-align:left;">
10
</td>
<td style="text-align:left;">
gene
</td>
<td style="text-align:right;">
88.74661
</td>
<td style="text-align:right;">
88.82681
</td>
<td style="text-align:left;">
negative
</td>
<td style="text-align:left;">
MGI_C57BL6J_1917933
</td>
<td style="text-align:left;">
Utp20
</td>
<td style="text-align:left;">
NCBI_Gene:70683,ENSEMBL:ENSMUSG00000004356
</td>
<td style="text-align:left;">
MGI:1917933
</td>
<td style="text-align:left;">
protein coding gene
</td>
<td style="text-align:left;">
UTP20 small subunit processome component
</td>
</tr>
<tr>
<td style="text-align:left;">
10
</td>
<td style="text-align:left;">
gene
</td>
<td style="text-align:right;">
88.87460
</td>
<td style="text-align:right;">
88.87580
</td>
<td style="text-align:left;">
negative
</td>
<td style="text-align:left;">
MGI_C57BL6J_1924528
</td>
<td style="text-align:left;">
9430024C03Rik
</td>
<td style="text-align:left;">
NA
</td>
<td style="text-align:left;">
NA
</td>
<td style="text-align:left;">
unclassified gene
</td>
<td style="text-align:left;">
RIKEN cDNA 9430024C03 gene
</td>
</tr>
<tr>
<td style="text-align:left;">
10
</td>
<td style="text-align:left;">
gene
</td>
<td style="text-align:right;">
88.88599
</td>
<td style="text-align:right;">
88.92951
</td>
<td style="text-align:left;">
positive
</td>
<td style="text-align:left;">
MGI_C57BL6J_2384916
</td>
<td style="text-align:left;">
Slc5a8
</td>
<td style="text-align:left;">
NCBI_Gene:216225,ENSEMBL:ENSMUSG00000020062
</td>
<td style="text-align:left;">
MGI:2384916
</td>
<td style="text-align:left;">
protein coding gene
</td>
<td style="text-align:left;">
solute carrier family 5 (iodide transporter), member 8
</td>
</tr>
<tr>
<td style="text-align:left;">
10
</td>
<td style="text-align:left;">
gene
</td>
<td style="text-align:right;">
88.94893
</td>
<td style="text-align:right;">
89.34508
</td>
<td style="text-align:left;">
negative
</td>
<td style="text-align:left;">
MGI_C57BL6J_2443344
</td>
<td style="text-align:left;">
Ano4
</td>
<td style="text-align:left;">
NCBI_Gene:320091,ENSEMBL:ENSMUSG00000035189
</td>
<td style="text-align:left;">
MGI:2443344
</td>
<td style="text-align:left;">
protein coding gene
</td>
<td style="text-align:left;">
anoctamin 4
</td>
</tr>
<tr>
<td style="text-align:left;">
10
</td>
<td style="text-align:left;">
gene
</td>
<td style="text-align:right;">
89.25713
</td>
<td style="text-align:right;">
89.25914
</td>
<td style="text-align:left;">
positive
</td>
<td style="text-align:left;">
MGI_C57BL6J_5589928
</td>
<td style="text-align:left;">
Gm30769
</td>
<td style="text-align:left;">
NCBI_Gene:102632785
</td>
<td style="text-align:left;">
MGI:5589928
</td>
<td style="text-align:left;">
lncRNA gene
</td>
<td style="text-align:left;">
predicted gene, 30769
</td>
</tr>
<tr>
<td style="text-align:left;">
10
</td>
<td style="text-align:left;">
gene
</td>
<td style="text-align:right;">
89.40882
</td>
<td style="text-align:right;">
89.44437
</td>
<td style="text-align:left;">
negative
</td>
<td style="text-align:left;">
MGI_C57BL6J_1918780
</td>
<td style="text-align:left;">
Gas2l3
</td>
<td style="text-align:left;">
NCBI_Gene:237436,ENSEMBL:ENSMUSG00000074802
</td>
<td style="text-align:left;">
MGI:1918780
</td>
<td style="text-align:left;">
protein coding gene
</td>
<td style="text-align:left;">
growth arrest-specific 2 like 3
</td>
</tr>
<tr>
<td style="text-align:left;">
10
</td>
<td style="text-align:left;">
gene
</td>
<td style="text-align:right;">
89.45423
</td>
<td style="text-align:right;">
89.53365
</td>
<td style="text-align:left;">
negative
</td>
<td style="text-align:left;">
MGI_C57BL6J_1352464
</td>
<td style="text-align:left;">
Nr1h4
</td>
<td style="text-align:left;">
NCBI_Gene:20186,ENSEMBL:ENSMUSG00000047638
</td>
<td style="text-align:left;">
MGI:1352464
</td>
<td style="text-align:left;">
protein coding gene
</td>
<td style="text-align:left;">
nuclear receptor subfamily 1, group H, member 4
</td>
</tr>
<tr>
<td style="text-align:left;">
10
</td>
<td style="text-align:left;">
pseudogene
</td>
<td style="text-align:right;">
89.55508
</td>
<td style="text-align:right;">
89.55572
</td>
<td style="text-align:left;">
negative
</td>
<td style="text-align:left;">
MGI_C57BL6J_6097428
</td>
<td style="text-align:left;">
Gm48087
</td>
<td style="text-align:left;">
ENSEMBL:ENSMUSG00000112272
</td>
<td style="text-align:left;">
MGI:6097428
</td>
<td style="text-align:left;">
pseudogene
</td>
<td style="text-align:left;">
predicted gene, 48087
</td>
</tr>
<tr>
<td style="text-align:left;">
10
</td>
<td style="text-align:left;">
gene
</td>
<td style="text-align:right;">
89.57402
</td>
<td style="text-align:right;">
89.62125
</td>
<td style="text-align:left;">
negative
</td>
<td style="text-align:left;">
MGI_C57BL6J_3039629
</td>
<td style="text-align:left;">
Slc17a8
</td>
<td style="text-align:left;">
NCBI_Gene:216227,ENSEMBL:ENSMUSG00000019935
</td>
<td style="text-align:left;">
MGI:3039629
</td>
<td style="text-align:left;">
protein coding gene
</td>
<td style="text-align:left;">
solute carrier family 17 (sodium-dependent inorganic phosphate cotransporter), member 8
</td>
</tr>
<tr>
<td style="text-align:left;">
10
</td>
<td style="text-align:left;">
gene
</td>
<td style="text-align:right;">
89.63872
</td>
<td style="text-align:right;">
89.68629
</td>
<td style="text-align:left;">
negative
</td>
<td style="text-align:left;">
MGI_C57BL6J_1289172
</td>
<td style="text-align:left;">
Scyl2
</td>
<td style="text-align:left;">
NCBI_Gene:213326,ENSEMBL:ENSMUSG00000069539
</td>
<td style="text-align:left;">
MGI:1289172
</td>
<td style="text-align:left;">
protein coding gene
</td>
<td style="text-align:left;">
SCY1-like 2 (S. cerevisiae)
</td>
</tr>
<tr>
<td style="text-align:left;">
10
</td>
<td style="text-align:left;">
gene
</td>
<td style="text-align:right;">
89.68636
</td>
<td style="text-align:right;">
89.70087
</td>
<td style="text-align:left;">
positive
</td>
<td style="text-align:left;">
MGI_C57BL6J_1916252
</td>
<td style="text-align:left;">
1500026H17Rik
</td>
<td style="text-align:left;">
NCBI_Gene:69002,ENSEMBL:ENSMUSG00000097383
</td>
<td style="text-align:left;">
MGI:1916252
</td>
<td style="text-align:left;">
lncRNA gene
</td>
<td style="text-align:left;">
RIKEN cDNA 1500026H17 gene
</td>
</tr>
<tr>
<td style="text-align:left;">
10
</td>
<td style="text-align:left;">
gene
</td>
<td style="text-align:right;">
89.71197
</td>
<td style="text-align:right;">
89.73231
</td>
<td style="text-align:left;">
negative
</td>
<td style="text-align:left;">
MGI_C57BL6J_1914269
</td>
<td style="text-align:left;">
Actr6
</td>
<td style="text-align:left;">
NCBI_Gene:67019,ENSEMBL:ENSMUSG00000019948
</td>
<td style="text-align:left;">
MGI:1914269
</td>
<td style="text-align:left;">
protein coding gene
</td>
<td style="text-align:left;">
ARP6 actin-related protein 6
</td>
</tr>
<tr>
<td style="text-align:left;">
10
</td>
<td style="text-align:left;">
gene
</td>
<td style="text-align:right;">
89.73678
</td>
<td style="text-align:right;">
89.74141
</td>
<td style="text-align:left;">
negative
</td>
<td style="text-align:left;">
MGI_C57BL6J_5590122
</td>
<td style="text-align:left;">
Gm30963
</td>
<td style="text-align:left;">
NCBI_Gene:102633042
</td>
<td style="text-align:left;">
MGI:5590122
</td>
<td style="text-align:left;">
lncRNA gene
</td>
<td style="text-align:left;">
predicted gene, 30963
</td>
</tr>
<tr>
<td style="text-align:left;">
10
</td>
<td style="text-align:left;">
gene
</td>
<td style="text-align:right;">
89.74362
</td>
<td style="text-align:right;">
89.74582
</td>
<td style="text-align:left;">
negative
</td>
<td style="text-align:left;">
MGI_C57BL6J_3642333
</td>
<td style="text-align:left;">
E330037I15Rik
</td>
<td style="text-align:left;">
NA
</td>
<td style="text-align:left;">
NA
</td>
<td style="text-align:left;">
unclassified gene
</td>
<td style="text-align:left;">
RIKEN cDNA E330037I15 gene
</td>
</tr>
<tr>
<td style="text-align:left;">
10
</td>
<td style="text-align:left;">
gene
</td>
<td style="text-align:right;">
89.74499
</td>
<td style="text-align:right;">
89.81989
</td>
<td style="text-align:left;">
positive
</td>
<td style="text-align:left;">
MGI_C57BL6J_2442888
</td>
<td style="text-align:left;">
Uhrf1bp1l
</td>
<td style="text-align:left;">
NCBI_Gene:75089,ENSEMBL:ENSMUSG00000019951
</td>
<td style="text-align:left;">
MGI:2442888
</td>
<td style="text-align:left;">
protein coding gene
</td>
<td style="text-align:left;">
UHRF1 (ICBP90) binding protein 1-like
</td>
</tr>
<tr>
<td style="text-align:left;">
10
</td>
<td style="text-align:left;">
gene
</td>
<td style="text-align:right;">
89.81216
</td>
<td style="text-align:right;">
89.81389
</td>
<td style="text-align:left;">
positive
</td>
<td style="text-align:left;">
MGI_C57BL6J_5590249
</td>
<td style="text-align:left;">
Gm31090
</td>
<td style="text-align:left;">
NCBI_Gene:102633203
</td>
<td style="text-align:left;">
MGI:5590249
</td>
<td style="text-align:left;">
lncRNA gene
</td>
<td style="text-align:left;">
predicted gene, 31090
</td>
</tr>
<tr>
<td style="text-align:left;">
10
</td>
<td style="text-align:left;">
gene
</td>
<td style="text-align:right;">
89.82134
</td>
<td style="text-align:right;">
89.82145
</td>
<td style="text-align:left;">
negative
</td>
<td style="text-align:left;">
MGI_C57BL6J_5455957
</td>
<td style="text-align:left;">
Gm26180
</td>
<td style="text-align:left;">
ENSEMBL:ENSMUSG00000093967
</td>
<td style="text-align:left;">
MGI:5455957
</td>
<td style="text-align:left;">
snRNA gene
</td>
<td style="text-align:left;">
predicted gene, 26180
</td>
</tr>
<tr>
<td style="text-align:left;">
10
</td>
<td style="text-align:left;">
gene
</td>
<td style="text-align:right;">
89.82288
</td>
<td style="text-align:right;">
89.82654
</td>
<td style="text-align:left;">
positive
</td>
<td style="text-align:left;">
MGI_C57BL6J_5825875
</td>
<td style="text-align:left;">
Gm46238
</td>
<td style="text-align:left;">
NCBI_Gene:108167827
</td>
<td style="text-align:left;">
MGI:5825875
</td>
<td style="text-align:left;">
protein coding gene
</td>
<td style="text-align:left;">
predicted gene, 46238
</td>
</tr>
<tr>
<td style="text-align:left;">
10
</td>
<td style="text-align:left;">
gene
</td>
<td style="text-align:right;">
89.87302
</td>
<td style="text-align:right;">
90.97330
</td>
<td style="text-align:left;">
positive
</td>
<td style="text-align:left;">
MGI_C57BL6J_1924781
</td>
<td style="text-align:left;">
Anks1b
</td>
<td style="text-align:left;">
NCBI_Gene:77531,ENSEMBL:ENSMUSG00000058589
</td>
<td style="text-align:left;">
MGI:1924781
</td>
<td style="text-align:left;">
protein coding gene
</td>
<td style="text-align:left;">
ankyrin repeat and sterile alpha motif domain containing 1B
</td>
</tr>
<tr>
<td style="text-align:left;">
10
</td>
<td style="text-align:left;">
gene
</td>
<td style="text-align:right;">
89.87827
</td>
<td style="text-align:right;">
89.87860
</td>
<td style="text-align:left;">
positive
</td>
<td style="text-align:left;">
MGI_C57BL6J_3782360
</td>
<td style="text-align:left;">
Gm4184
</td>
<td style="text-align:left;">
NCBI_Gene:100043039
</td>
<td style="text-align:left;">
MGI:3782360
</td>
<td style="text-align:left;">
protein coding gene
</td>
<td style="text-align:left;">
predicted gene 4184
</td>
</tr>
<tr>
<td style="text-align:left;">
10
</td>
<td style="text-align:left;">
pseudogene
</td>
<td style="text-align:right;">
90.13298
</td>
<td style="text-align:right;">
90.13482
</td>
<td style="text-align:left;">
negative
</td>
<td style="text-align:left;">
MGI_C57BL6J_3645288
</td>
<td style="text-align:left;">
Gm5780
</td>
<td style="text-align:left;">
NCBI_Gene:544719,ENSEMBL:ENSMUSG00000112140
</td>
<td style="text-align:left;">
MGI:3645288
</td>
<td style="text-align:left;">
pseudogene
</td>
<td style="text-align:left;">
predicted gene 5780
</td>
</tr>
<tr>
<td style="text-align:left;">
10
</td>
<td style="text-align:left;">
gene
</td>
<td style="text-align:right;">
90.42659
</td>
<td style="text-align:right;">
90.42806
</td>
<td style="text-align:left;">
positive
</td>
<td style="text-align:left;">
MGI_C57BL6J_2441893
</td>
<td style="text-align:left;">
C230047L17Rik
</td>
<td style="text-align:left;">
NA
</td>
<td style="text-align:left;">
NA
</td>
<td style="text-align:left;">
unclassified gene
</td>
<td style="text-align:left;">
RIKEN cDNA C230047L17 gene
</td>
</tr>
<tr>
<td style="text-align:left;">
10
</td>
<td style="text-align:left;">
gene
</td>
<td style="text-align:right;">
90.45091
</td>
<td style="text-align:right;">
90.45100
</td>
<td style="text-align:left;">
positive
</td>
<td style="text-align:left;">
MGI_C57BL6J_5452379
</td>
<td style="text-align:left;">
Gm22602
</td>
<td style="text-align:left;">
ENSEMBL:ENSMUSG00000088221
</td>
<td style="text-align:left;">
MGI:5452379
</td>
<td style="text-align:left;">
snRNA gene
</td>
<td style="text-align:left;">
predicted gene, 22602
</td>
</tr>
<tr>
<td style="text-align:left;">
10
</td>
<td style="text-align:left;">
gene
</td>
<td style="text-align:right;">
90.59763
</td>
<td style="text-align:right;">
90.59916
</td>
<td style="text-align:left;">
positive
</td>
<td style="text-align:left;">
MGI_C57BL6J_1920193
</td>
<td style="text-align:left;">
2900018E21Rik
</td>
<td style="text-align:left;">
NA
</td>
<td style="text-align:left;">
NA
</td>
<td style="text-align:left;">
unclassified gene
</td>
<td style="text-align:left;">
RIKEN cDNA 2900018E21 gene
</td>
</tr>
<tr>
<td style="text-align:left;">
10
</td>
<td style="text-align:left;">
gene
</td>
<td style="text-align:right;">
90.72896
</td>
<td style="text-align:right;">
90.73633
</td>
<td style="text-align:left;">
positive
</td>
<td style="text-align:left;">
MGI_C57BL6J_6096636
</td>
<td style="text-align:left;">
Gm47591
</td>
<td style="text-align:left;">
ENSEMBL:ENSMUSG00000112153
</td>
<td style="text-align:left;">
MGI:6096636
</td>
<td style="text-align:left;">
lncRNA gene
</td>
<td style="text-align:left;">
predicted gene, 47591
</td>
</tr>
<tr>
<td style="text-align:left;">
10
</td>
<td style="text-align:left;">
pseudogene
</td>
<td style="text-align:right;">
90.74999
</td>
<td style="text-align:right;">
90.75069
</td>
<td style="text-align:left;">
negative
</td>
<td style="text-align:left;">
MGI_C57BL6J_5010855
</td>
<td style="text-align:left;">
Gm18670
</td>
<td style="text-align:left;">
NCBI_Gene:100417532,ENSEMBL:ENSMUSG00000111909
</td>
<td style="text-align:left;">
MGI:5010855
</td>
<td style="text-align:left;">
pseudogene
</td>
<td style="text-align:left;">
predicted gene, 18670
</td>
</tr>
<tr>
<td style="text-align:left;">
10
</td>
<td style="text-align:left;">
pseudogene
</td>
<td style="text-align:right;">
90.75087
</td>
<td style="text-align:right;">
90.75107
</td>
<td style="text-align:left;">
negative
</td>
<td style="text-align:left;">
MGI_C57BL6J_6096633
</td>
<td style="text-align:left;">
Gm47589
</td>
<td style="text-align:left;">
ENSEMBL:ENSMUSG00000112507
</td>
<td style="text-align:left;">
MGI:6096633
</td>
<td style="text-align:left;">
pseudogene
</td>
<td style="text-align:left;">
predicted gene, 47589
</td>
</tr>
<tr>
<td style="text-align:left;">
10
</td>
<td style="text-align:left;">
gene
</td>
<td style="text-align:right;">
90.81308
</td>
<td style="text-align:right;">
90.81439
</td>
<td style="text-align:left;">
positive
</td>
<td style="text-align:left;">
MGI_C57BL6J_6096638
</td>
<td style="text-align:left;">
Gm47592
</td>
<td style="text-align:left;">
ENSEMBL:ENSMUSG00000112015
</td>
<td style="text-align:left;">
MGI:6096638
</td>
<td style="text-align:left;">
unclassified gene
</td>
<td style="text-align:left;">
predicted gene, 47592
</td>
</tr>
<tr>
<td style="text-align:left;">
10
</td>
<td style="text-align:left;">
gene
</td>
<td style="text-align:right;">
90.85110
</td>
<td style="text-align:right;">
90.85407
</td>
<td style="text-align:left;">
positive
</td>
<td style="text-align:left;">
MGI_C57BL6J_6096634
</td>
<td style="text-align:left;">
Gm47590
</td>
<td style="text-align:left;">
ENSEMBL:ENSMUSG00000112601
</td>
<td style="text-align:left;">
MGI:6096634
</td>
<td style="text-align:left;">
unclassified gene
</td>
<td style="text-align:left;">
predicted gene, 47590
</td>
</tr>
<tr>
<td style="text-align:left;">
10
</td>
<td style="text-align:left;">
gene
</td>
<td style="text-align:right;">
90.98931
</td>
<td style="text-align:right;">
91.08277
</td>
<td style="text-align:left;">
negative
</td>
<td style="text-align:left;">
MGI_C57BL6J_1306796
</td>
<td style="text-align:left;">
Apaf1
</td>
<td style="text-align:left;">
NCBI_Gene:11783,ENSEMBL:ENSMUSG00000019979
</td>
<td style="text-align:left;">
MGI:1306796
</td>
<td style="text-align:left;">
protein coding gene
</td>
<td style="text-align:left;">
apoptotic peptidase activating factor 1
</td>
</tr>
<tr>
<td style="text-align:left;">
10
</td>
<td style="text-align:left;">
gene
</td>
<td style="text-align:right;">
91.00574
</td>
<td style="text-align:right;">
91.00584
</td>
<td style="text-align:left;">
positive
</td>
<td style="text-align:left;">
MGI_C57BL6J_5455167
</td>
<td style="text-align:left;">
Gm25390
</td>
<td style="text-align:left;">
ENSEMBL:ENSMUSG00000087966
</td>
<td style="text-align:left;">
MGI:5455167
</td>
<td style="text-align:left;">
snRNA gene
</td>
<td style="text-align:left;">
predicted gene, 25390
</td>
</tr>
<tr>
<td style="text-align:left;">
10
</td>
<td style="text-align:left;">
gene
</td>
<td style="text-align:right;">
91.08269
</td>
<td style="text-align:right;">
91.10261
</td>
<td style="text-align:left;">
positive
</td>
<td style="text-align:left;">
MGI_C57BL6J_1914704
</td>
<td style="text-align:left;">
Ikbip
</td>
<td style="text-align:left;">
NCBI_Gene:67454,ENSEMBL:ENSMUSG00000019975
</td>
<td style="text-align:left;">
MGI:1914704
</td>
<td style="text-align:left;">
protein coding gene
</td>
<td style="text-align:left;">
IKBKB interacting protein
</td>
</tr>
<tr>
<td style="text-align:left;">
10
</td>
<td style="text-align:left;">
gene
</td>
<td style="text-align:right;">
91.11378
</td>
<td style="text-align:right;">
91.11425
</td>
<td style="text-align:left;">
negative
</td>
<td style="text-align:left;">
MGI_C57BL6J_1925088
</td>
<td style="text-align:left;">
A930036M01Rik
</td>
<td style="text-align:left;">
NA
</td>
<td style="text-align:left;">
NA
</td>
<td style="text-align:left;">
unclassified non-coding RNA gene
</td>
<td style="text-align:left;">
RIKEN cDNA A930036M01 gene
</td>
</tr>
<tr>
<td style="text-align:left;">
10
</td>
<td style="text-align:left;">
gene
</td>
<td style="text-align:right;">
91.11657
</td>
<td style="text-align:right;">
91.12406
</td>
<td style="text-align:left;">
negative
</td>
<td style="text-align:left;">
MGI_C57BL6J_1353498
</td>
<td style="text-align:left;">
Slc25a3
</td>
<td style="text-align:left;">
NCBI_Gene:18674,ENSEMBL:ENSMUSG00000061904
</td>
<td style="text-align:left;">
MGI:1353498
</td>
<td style="text-align:left;">
protein coding gene
</td>
<td style="text-align:left;">
solute carrier family 25 (mitochondrial carrier, phosphate carrier), member 3
</td>
</tr>
<tr>
<td style="text-align:left;">
10
</td>
<td style="text-align:left;">
gene
</td>
<td style="text-align:right;">
91.11829
</td>
<td style="text-align:right;">
91.11854
</td>
<td style="text-align:left;">
negative
</td>
<td style="text-align:left;">
MGI_C57BL6J_5453896
</td>
<td style="text-align:left;">
Gm24119
</td>
<td style="text-align:left;">
ENSEMBL:ENSMUSG00000077167
</td>
<td style="text-align:left;">
MGI:5453896
</td>
<td style="text-align:left;">
snoRNA gene
</td>
<td style="text-align:left;">
predicted gene, 24119
</td>
</tr>
<tr>
<td style="text-align:left;">
10
</td>
<td style="text-align:left;">
gene
</td>
<td style="text-align:right;">
91.14757
</td>
<td style="text-align:right;">
91.18172
</td>
<td style="text-align:left;">
negative
</td>
<td style="text-align:left;">
MGI_C57BL6J_106920
</td>
<td style="text-align:left;">
Tmpo
</td>
<td style="text-align:left;">
NCBI_Gene:21917,ENSEMBL:ENSMUSG00000019961
</td>
<td style="text-align:left;">
MGI:106920
</td>
<td style="text-align:left;">
protein coding gene
</td>
<td style="text-align:left;">
thymopoietin
</td>
</tr>
<tr>
<td style="text-align:left;">
10
</td>
<td style="text-align:left;">
gene
</td>
<td style="text-align:right;">
91.14951
</td>
<td style="text-align:right;">
91.15059
</td>
<td style="text-align:left;">
positive
</td>
<td style="text-align:left;">
MGI_C57BL6J_5477259
</td>
<td style="text-align:left;">
Gm26765
</td>
<td style="text-align:left;">
ENSEMBL:ENSMUSG00000097664
</td>
<td style="text-align:left;">
MGI:5477259
</td>
<td style="text-align:left;">
lncRNA gene
</td>
<td style="text-align:left;">
predicted gene, 26765
</td>
</tr>
<tr>
<td style="text-align:left;">
10
</td>
<td style="text-align:left;">
gene
</td>
<td style="text-align:right;">
91.15008
</td>
<td style="text-align:right;">
91.15164
</td>
<td style="text-align:left;">
negative
</td>
<td style="text-align:left;">
MGI_C57BL6J_1923371
</td>
<td style="text-align:left;">
5830485P09Rik
</td>
<td style="text-align:left;">
NA
</td>
<td style="text-align:left;">
NA
</td>
<td style="text-align:left;">
unclassified gene
</td>
<td style="text-align:left;">
RIKEN cDNA 5830485P09 gene
</td>
</tr>
<tr>
<td style="text-align:left;">
10
</td>
<td style="text-align:left;">
gene
</td>
<td style="text-align:right;">
91.18141
</td>
<td style="text-align:right;">
91.18148
</td>
<td style="text-align:left;">
negative
</td>
<td style="text-align:left;">
MGI_C57BL6J_4414067
</td>
<td style="text-align:left;">
n-TWcca2
</td>
<td style="text-align:left;">
NCBI_Gene:102467609
</td>
<td style="text-align:left;">
MGI:4414067
</td>
<td style="text-align:left;">
tRNA gene
</td>
<td style="text-align:left;">
nuclear encoded tRNA trytophan 2 (anticodon CCA)
</td>
</tr>
<tr>
<td style="text-align:left;">
10
</td>
<td style="text-align:left;">
gene
</td>
<td style="text-align:right;">
91.18210
</td>
<td style="text-align:right;">
91.18217
</td>
<td style="text-align:left;">
negative
</td>
<td style="text-align:left;">
MGI_C57BL6J_4413767
</td>
<td style="text-align:left;">
n-TDgtc16
</td>
<td style="text-align:left;">
NCBI_Gene:102467266
</td>
<td style="text-align:left;">
MGI:4413767
</td>
<td style="text-align:left;">
tRNA gene
</td>
<td style="text-align:left;">
nuclear encoded tRNA aspartic acid 16 (anticodon GTC)
</td>
</tr>
<tr>
<td style="text-align:left;">
10
</td>
<td style="text-align:left;">
gene
</td>
<td style="text-align:right;">
91.20826
</td>
<td style="text-align:right;">
91.22064
</td>
<td style="text-align:left;">
negative
</td>
<td style="text-align:left;">
MGI_C57BL6J_1922441
</td>
<td style="text-align:left;">
4930534H03Rik
</td>
<td style="text-align:left;">
NCBI_Gene:102633753,ENSEMBL:ENSMUSG00000111544
</td>
<td style="text-align:left;">
MGI:1922441
</td>
<td style="text-align:left;">
lncRNA gene
</td>
<td style="text-align:left;">
RIKEN cDNA 4930534H03 gene
</td>
</tr>
<tr>
<td style="text-align:left;">
10
</td>
<td style="text-align:left;">
pseudogene
</td>
<td style="text-align:right;">
91.22819
</td>
<td style="text-align:right;">
91.22850
</td>
<td style="text-align:left;">
positive
</td>
<td style="text-align:left;">
MGI_C57BL6J_6095807
</td>
<td style="text-align:left;">
Gm47081
</td>
<td style="text-align:left;">
ENSEMBL:ENSMUSG00000110870
</td>
<td style="text-align:left;">
MGI:6095807
</td>
<td style="text-align:left;">
pseudogene
</td>
<td style="text-align:left;">
predicted gene, 47081
</td>
</tr>
<tr>
<td style="text-align:left;">
10
</td>
<td style="text-align:left;">
pseudogene
</td>
<td style="text-align:right;">
91.25214
</td>
<td style="text-align:right;">
91.25950
</td>
<td style="text-align:left;">
negative
</td>
<td style="text-align:left;">
MGI_C57BL6J_5010890
</td>
<td style="text-align:left;">
Gm18705
</td>
<td style="text-align:left;">
NCBI_Gene:100417593,ENSEMBL:ENSMUSG00000110847
</td>
<td style="text-align:left;">
MGI:5010890
</td>
<td style="text-align:left;">
pseudogene
</td>
<td style="text-align:left;">
predicted gene, 18705
</td>
</tr>
<tr>
<td style="text-align:left;">
10
</td>
<td style="text-align:left;">
gene
</td>
<td style="text-align:right;">
91.31034
</td>
<td style="text-align:right;">
91.31245
</td>
<td style="text-align:left;">
positive
</td>
<td style="text-align:left;">
MGI_C57BL6J_6095808
</td>
<td style="text-align:left;">
Gm47082
</td>
<td style="text-align:left;">
ENSEMBL:ENSMUSG00000111525
</td>
<td style="text-align:left;">
MGI:6095808
</td>
<td style="text-align:left;">
lncRNA gene
</td>
<td style="text-align:left;">
predicted gene, 47082
</td>
</tr>
<tr>
<td style="text-align:left;">
10
</td>
<td style="text-align:left;">
gene
</td>
<td style="text-align:right;">
91.41060
</td>
<td style="text-align:right;">
91.43962
</td>
<td style="text-align:left;">
positive
</td>
<td style="text-align:left;">
MGI_C57BL6J_5623621
</td>
<td style="text-align:left;">
Gm40736
</td>
<td style="text-align:left;">
NCBI_Gene:105245258
</td>
<td style="text-align:left;">
MGI:5623621
</td>
<td style="text-align:left;">
lncRNA gene
</td>
<td style="text-align:left;">
predicted gene, 40736
</td>
</tr>
<tr>
<td style="text-align:left;">
10
</td>
<td style="text-align:left;">
pseudogene
</td>
<td style="text-align:right;">
91.42335
</td>
<td style="text-align:right;">
91.42373
</td>
<td style="text-align:left;">
positive
</td>
<td style="text-align:left;">
MGI_C57BL6J_6095810
</td>
<td style="text-align:left;">
Gm47083
</td>
<td style="text-align:left;">
ENSEMBL:ENSMUSG00000111591
</td>
<td style="text-align:left;">
MGI:6095810
</td>
<td style="text-align:left;">
pseudogene
</td>
<td style="text-align:left;">
predicted gene, 47083
</td>
</tr>
<tr>
<td style="text-align:left;">
10
</td>
<td style="text-align:left;">
pseudogene
</td>
<td style="text-align:right;">
91.44822
</td>
<td style="text-align:right;">
91.44838
</td>
<td style="text-align:left;">
negative
</td>
<td style="text-align:left;">
MGI_C57BL6J_6095811
</td>
<td style="text-align:left;">
Gm47084
</td>
<td style="text-align:left;">
ENSEMBL:ENSMUSG00000111051
</td>
<td style="text-align:left;">
MGI:6095811
</td>
<td style="text-align:left;">
pseudogene
</td>
<td style="text-align:left;">
predicted gene, 47084
</td>
</tr>
<tr>
<td style="text-align:left;">
10
</td>
<td style="text-align:left;">
pseudogene
</td>
<td style="text-align:right;">
91.64070
</td>
<td style="text-align:right;">
91.64089
</td>
<td style="text-align:left;">
negative
</td>
<td style="text-align:left;">
MGI_C57BL6J_6095821
</td>
<td style="text-align:left;">
Gm47091
</td>
<td style="text-align:left;">
ENSEMBL:ENSMUSG00000111782
</td>
<td style="text-align:left;">
MGI:6095821
</td>
<td style="text-align:left;">
pseudogene
</td>
<td style="text-align:left;">
predicted gene, 47091
</td>
</tr>
<tr>
<td style="text-align:left;">
10
</td>
<td style="text-align:left;">
gene
</td>
<td style="text-align:right;">
91.88859
</td>
<td style="text-align:right;">
91.95362
</td>
<td style="text-align:left;">
positive
</td>
<td style="text-align:left;">
MGI_C57BL6J_5590751
</td>
<td style="text-align:left;">
Gm31592
</td>
<td style="text-align:left;">
NCBI_Gene:102633867,ENSEMBL:ENSMUSG00000111657
</td>
<td style="text-align:left;">
MGI:5590751
</td>
<td style="text-align:left;">
lncRNA gene
</td>
<td style="text-align:left;">
predicted gene, 31592
</td>
</tr>
</tbody>
</table>
</div>
</div>
<div id="additive-covariates" class="section level2">
<h2>Additive covariates</h2>
<div id="significance-thresholds-1" class="section level3">
<h3>Significance thresholds</h3>
<pre class="r"><code>operm &lt;- scan1perm(pr.qc, covars[&quot;age.of.onset&quot;], model=&quot;normal&quot;, addcovar=addcovar, n_perm=10, perm_Xsp=TRUE, chr_lengths=chr_lengths(gm$gmap))

summary_table&lt;-data.frame(unclass(summary(operm, alpha=c(0.01,  0.05, 0.1))))
names(summary_table) &lt;- c(&quot;autosomes&quot;,&quot;X&quot;)
summary_table$significance.level &lt;- rownames(summary_table)

rownames(summary_table) &lt;- NULL

summary_table[c(3,1:2)] %&gt;%
  kable(escape = F,align = c(&quot;ccc&quot;)) %&gt;%
  kable_styling(&quot;striped&quot;, full_width = T) %&gt;%
  column_spec(1, bold=TRUE)</code></pre>
<table class="table table-striped" style="margin-left: auto; margin-right: auto;">
<thead>
<tr>
<th style="text-align:center;">
significance.level
</th>
<th style="text-align:center;">
autosomes
</th>
<th style="text-align:center;">
X
</th>
</tr>
</thead>
<tbody>
<tr>
<td style="text-align:center;font-weight: bold;">
0.01
</td>
<td style="text-align:center;">
3.153089
</td>
<td style="text-align:center;">
2.504958
</td>
</tr>
<tr>
<td style="text-align:center;font-weight: bold;">
0.05
</td>
<td style="text-align:center;">
2.777666
</td>
<td style="text-align:center;">
2.341595
</td>
</tr>
<tr>
<td style="text-align:center;font-weight: bold;">
0.1
</td>
<td style="text-align:center;">
2.307215
</td>
<td style="text-align:center;">
2.127970
</td>
</tr>
</tbody>
</table>
</div>
<div id="genome-wide-scan-1" class="section level3">
<h3>Genome-wide scan</h3>
<pre class="r"><code>plot_lod&lt;-function(out,map){
  for (i in 1:dim(out)[2]){
#par(mar=c(5.1, 6.1, 1.1, 1.1))
    ymx &lt;- maxlod(out) # overall maximum LOD score
    plot(out, map, lodcolumn=i, col=&quot;slateblue&quot;, ylim=c(0, ymx+0.5))
    #legend(&quot;topright&quot;, lwd=2, colnames(out)[i], bg=&quot;gray90&quot;)
    title(main = paste0(colnames(out)[i], &quot; - ICI vs. EOI (ith additive covariates) [positions in cM]&quot;))
    add_threshold(map,  summary(operm,alpha=0.1), col = &#39;purple&#39;)
    add_threshold(map,  summary(operm, alpha=0.05), col = &#39;red&#39;)
    add_threshold(map,  summary(operm, alpha=0.01), col = &#39;blue&#39;)

  }
}


print(&quot;with no kinship&quot;)</code></pre>
<p>[1] “with no kinship”</p>
<pre class="r"><code>outfa &lt;- scan1(pr.qc, covars[&quot;age.of.onset&quot;], Xcovar=Xcovar, model=&quot;normal&quot;, addcovar=addcovar)

plot_lod(outfa,gm$gmap)</code></pre>
<p><img src="figure/4.1.2_qtl.analysis_cont_age_ici.vs.eoi_snpsqc_pheno.corrected.cleaned_5.batches_0.Rmd/scanonea-1.png" width="912" style="display: block; margin: auto;" /></p>
<pre class="r"><code>print(&quot;with normal kinship&quot;)</code></pre>
<p>[1] “with normal kinship”</p>
<pre class="r"><code>outka &lt;- scan1(pr.qc, covars[&quot;age.of.onset&quot;], Xcovar=Xcovar, model=&quot;normal&quot;, addcovar=addcovar, kinship=kinship)

plot_lod(outka,gm$gmap)</code></pre>
<p><img src="figure/4.1.2_qtl.analysis_cont_age_ici.vs.eoi_snpsqc_pheno.corrected.cleaned_5.batches_0.Rmd/scanonea-2.png" width="912" style="display: block; margin: auto;" /></p>
<pre class="r"><code>print(&quot;with loco kinship&quot;)</code></pre>
<p>[1] “with loco kinship”</p>
<pre class="r"><code>outa &lt;- scan1(pr.qc, covars[&quot;age.of.onset&quot;], Xcovar=Xcovar, model=&quot;normal&quot;, addcovar=addcovar, kinship=K)

plot_lod(outa,gm$gmap)</code></pre>
<p><img src="figure/4.1.2_qtl.analysis_cont_age_ici.vs.eoi_snpsqc_pheno.corrected.cleaned_5.batches_0.Rmd/scanonea-3.png" width="912" style="display: block; margin: auto;" /></p>
<pre class="r"><code>out = outa

print(paste0(&quot;number of samples in analysis = &quot;, as.data.frame(attributes(out)$sample_size)[1,]))</code></pre>
<p>[1] “number of samples in analysis = 266”</p>
</div>
<div id="lod-peaks-1" class="section level3">
<h3>LOD peaks</h3>
<div id="centimorgan-cm-1" class="section level4">
<h4>Centimorgan (cM)</h4>
<pre class="r"><code>peaks&lt;-find_peaks(out, gm$gmap, threshold=summary(operm,alpha=0.05)$A, thresholdX = summary(operm,alpha=0.05)$X, peakdrop=3, drop=1.5)
if(nrow(peaks) &gt; 0){
peaks$marker &lt;- find_marker(gm$gmap, chr=peaks$chr,pos=peaks$pos)
names(peaks)[2] &lt;- c(&quot;phenotype&quot;)
peaks &lt;- peaks[-1]

rownames(peaks) &lt;- NULL
print(kable(peaks, escape = F, align = c(&quot;cccccccc&quot;), &quot;html&quot;) 
  %&gt;% kable_styling(&quot;striped&quot;, full_width = T)%&gt;%
  column_spec(1, bold=TRUE)
  )

#plot only peak chromosomes

plot_lod_chr&lt;-function(out,map,chrom){
  for (i in 1:dim(out)[2]){
    #png(filename=paste0(&quot;/Users/chenm/Documents/qtl/Jai/&quot;,colnames(out)[i],  &quot;_lod.png&quot;))
    
    #par(mar=c(5.1, 6.1, 1.1, 1.1))
    ymx &lt;- maxlod(out) # overall maximum LOD score
    plot(out, map, chr = chrom, lodcolumn=i, col=&quot;slateblue&quot;, ylim=c(0, ymx+0.5))
    #legend(&quot;topright&quot;, lwd=2, colnames(out)[i], bg=&quot;gray90&quot;)
    title(main = paste0(colnames(out)[i], &quot; - chr&quot;, chrom, &quot; - ICI vs. EOI (with additive covariates) [positions in cM]&quot;))
    add_threshold(map,  summary(operm,alpha=0.1), col = &#39;purple&#39;)
    add_threshold(map,  summary(operm, alpha=0.05), col = &#39;red&#39;)
    add_threshold(map,  summary(operm, alpha=0.01), col = &#39;blue&#39;)
    #for (j in 1: dim(summary_table)[1]){
    #  abline(h=summary_table[j, i],col=&quot;red&quot;)
    #  text(x=400, y =summary_table[j, i]+0.12, labels = paste(&quot;p=&quot;, row.names(summary_table)[j]))
    #}
    #dev.off()

  }
}


} else {
  print(paste0(&quot;There are no peaks that have a LOD that reaches suggestive (p&lt;0.05) level of &quot;,summary(operm,alpha=0.05)$A, &quot; [autosomes]/&quot;,summary(operm,alpha=0.05)$X, &quot; [x-chromosome]&quot;))
}</code></pre>
<p>[1] “There are no peaks that have a LOD that reaches suggestive (p&lt;0.05) level of 2.77766557899907 [autosomes]/2.34159545294357 [x-chromosome]”</p>
<pre class="r"><code>if(nrow(peaks) &lt; 50 &amp; nrow(peaks) &gt; 0){

for(i in unique(peaks$chr)){
#for (i in 1:nrow(peaks)){
  #plot_lod_chr(out,gm$gmap, peaks$chr[i])
  plot_lod_chr(out,gm$gmap, i)
}

} else {
  print(paste0(&quot;There are too many peaks (&quot;,nrow(peaks),&quot; peaks) that have a LOD that reaches suggestive (p&lt;0.05) level of &quot;,summary(operm,alpha=0.05)$A, &quot; [autosomes]/&quot;,summary(operm,alpha=0.05)$X, &quot; [x-chromosome]&quot;))
}</code></pre>
<p>[1] “There are too many peaks (0 peaks) that have a LOD that reaches suggestive (p&lt;0.05) level of 2.77766557899907 [autosomes]/2.34159545294357 [x-chromosome]”</p>
</div>
<div id="megabase-mb-1" class="section level4">
<h4>Megabase (MB)</h4>
<pre class="r"><code>peaks_mba&lt;-find_peaks(out, gm$pmap, threshold=summary(operm,alpha=0.05)$A, thresholdX = summary(operm,alpha=0.05)$X, peakdrop=3, drop=1.5)
if(nrow(peaks) &gt; 0){
peaks_mba$marker &lt;- find_marker(gm$pmap, chr=peaks_mba$chr,pos=peaks_mba$pos)
names(peaks_mba)[2] &lt;- c(&quot;phenotype&quot;)
peaks_mba &lt;- peaks_mba[-1]

rownames(peaks_mba) &lt;- NULL
print(kable(peaks_mba, escape = F, align = c(&quot;cccccccc&quot;), &quot;html&quot;) 
  %&gt;% kable_styling(&quot;striped&quot;, full_width = T)%&gt;%
  column_spec(1, bold=TRUE)
  )

plot_lod_chr_mb&lt;-function(out,map,chrom){
  for (i in 1:dim(out)[2]){
    #png(filename=paste0(&quot;/Users/chenm/Documents/qtl/Jai/&quot;,colnames(out)[i],  &quot;_lod.png&quot;))
    
    #par(mar=c(5.1, 6.1, 1.1, 1.1))
    ymx &lt;- maxlod(out) # overall maximum LOD score
    plot(out, map, chr = chrom, lodcolumn=i, col=&quot;slateblue&quot;, ylim=c(0, ymx+0.5))
    #legend(&quot;topright&quot;, lwd=2, colnames(out)[i], bg=&quot;gray90&quot;)
    title(main = paste0(colnames(out)[i], &quot; - chr&quot;, chrom, &quot; - ICI vs. EOI (with additive covariates) [positions in cM]&quot;))
    add_threshold(map,  summary(operm,alpha=0.1), col = &#39;purple&#39;)
    add_threshold(map,  summary(operm, alpha=0.05), col = &#39;red&#39;)
    add_threshold(map,  summary(operm, alpha=0.01), col = &#39;blue&#39;)
    #for (j in 1: dim(summary_table)[1]){
    #  abline(h=summary_table[j, i],col=&quot;red&quot;)
    #  text(x=400, y =summary_table[j, i]+0.12, labels = paste(&quot;p=&quot;, row.names(summary_table)[j]))
    #}
    #dev.off()
  }
}

} else {
  print(paste0(&quot;There are no peaks that have a LOD that reaches suggestive (p&lt;0.05) level of &quot;,summary(operm,alpha=0.05)$A, &quot; [autosomes]/&quot;,summary(operm,alpha=0.05)$X, &quot; [x-chromosome]&quot;))
}</code></pre>
<p>[1] “There are no peaks that have a LOD that reaches suggestive (p&lt;0.05) level of 2.77766557899907 [autosomes]/2.34159545294357 [x-chromosome]”</p>
<pre class="r"><code>if(nrow(peaks_mba) &lt; 50 &amp; nrow(peaks_mba) &gt; 0){

for(i in unique(peaks_mba$chr)){
#for (i in 1:nrow(peaks_mba)){
  #plot_lod_chr_mb(out,gm$pmap, peaks_mba$chr[i])
  plot_lod_chr_mb(out,gm$pmap,i)
}

} else {
  print(paste0(&quot;There are too many peaks (&quot;,nrow(peaks_mba),&quot; peaks) that have a LOD that reaches suggestive (p&lt;0.05) level of &quot;,summary(operm,alpha=0.05)$A, &quot; [autosomes]/&quot;,summary(operm,alpha=0.05)$X, &quot; [x-chromosome]&quot;))
}</code></pre>
<p>[1] “There are too many peaks (0 peaks) that have a LOD that reaches suggestive (p&lt;0.05) level of 2.77766557899907 [autosomes]/2.34159545294357 [x-chromosome]”</p>
</div>
</div>
<div id="qtl-effects-1" class="section level3">
<h3>QTL effects</h3>
<p>For each peak LOD location we give a list of gene</p>
<pre class="r"><code>if(nrow(peaks) &lt; 50 &amp; nrow(peaks) &gt; 0){
for (i in 1:nrow(peaks)){
#for (i in 1:1){
  #Plot 1
  g &lt;- maxmarg(pr.qc, gm$gmap, chr=peaks$chr[i], pos=peaks$pos[i], return_char=TRUE)
  #png(filename=paste0(&quot;/Users/chenm/Documents/qtl/Jai/&quot;,&quot;qtl_effect_&quot;, i, &quot;.png&quot;))
  #par(mar=c(4.1, 4.1, 1.5, 0.6))
  plot_pxg(g, covars[,peaks$phenotype[i]], ylab=peaks$phenotype[i], sort=FALSE)
  title(main = paste0(&quot;chr: &quot;, chr=peaks$chr[i],&quot;; pos: &quot;, peaks$pos[i], &quot;cM /&quot;,peaks_mba$pos[i],&quot;MB\n(&quot;,peaks$phenotype[i],&quot; )&quot;), line=0.2)
  ##dev.off()

  chr = peaks$chr[i]

# Plot 2
  pr_sub &lt;- pull_genoprobint(pr.qc, gm$gmap, chr, c(peaks$ci_lo[i], peaks$ci_hi[i]))
  #coeff &lt;- scan1coef(pr[,chr], cross$pheno[,peaks$lodcolumn[i]], addcovar = addcovar)
  #coeff &lt;- scan1coef(pr[,chr], cross$pheno[,peaks$lodcolumn[i]], Xcovar=Xcovar)
  #coeff &lt;- scan1coef(pr.qc[,chr], gm$covar[peaks$lodcolumn[i]], model=&quot;binary&quot;)
  #coeff_sub &lt;- scan1coef(pr_sub[,chr], gm$covar[peaks$lodcolumn[i]], model=&quot;binary&quot;)
  blup &lt;- scan1blup(pr.qc[,chr], covars[peaks$phenotype[i]])
  blup_sub &lt;- scan1blup(pr_sub[,chr], covars[peaks$phenotype[i]])

  write.csv(as.data.frame(blup_sub), paste0(&quot;data/ici.vs.eoi_&quot;,peaks$phenotype[i],&quot;-additive.covariates_blup_sub_chr-&quot;,chr,&quot;_peak.marker-&quot;,peaks$marker[i],&quot;_lod.drop-1.5_snpsqc_5.batches_0.csv&quot;), quote=F)

  #plot_coef(coeff, 
  #     gm$gmap, columns=1:2,
  #     bgcolor=&quot;gray95&quot;, legend=&quot;bottomleft&quot;, 
  #     main = paste(&quot;chr&quot;, chr=peaks$chr[i],&quot;; pos: &quot;, peaks$pos[i], &quot;cM /&quot;,peaks_mba$pos[i],&quot;MB\n(&quot;,peaks$lodcolumn[i],&quot; [scan1coeff; positions in cM] )&quot;)
  #     )

  #plot_coef(coeff_sub, 
  #     gm$gmap, columns=1:2,
  #     bgcolor=&quot;gray95&quot;, legend=&quot;bottomleft&quot;, 
  #     main = paste(&quot;chr&quot;, chr=peaks$chr[i],&quot;; pos: &quot;, peaks$pos[i], &quot;cM /&quot;,peaks_mba$pos[i],&quot;MB\n(&quot;,peaks$lodcolumn[i],&quot;; 1.5 LOD drop interval [scan1coeff; positions in cM] ) &quot;)
  #     )


  plot_coef(blup, 
       gm$gmap, columns=1:2,
       bgcolor=&quot;gray95&quot;, legend=&quot;bottomleft&quot;, 
       main = paste0(&quot;chr: &quot;, chr=peaks$chr[i],&quot;; pos: &quot;, peaks$pos[i], &quot;cM /&quot;,peaks_mba$pos[i],&quot;MB\n(&quot;,peaks$phenotype[i],&quot; [scan1blup; positions in cM] )&quot;)
       )

  plot_coef(blup_sub, 
       gm$gmap, columns=1:2,
       bgcolor=&quot;gray95&quot;, legend=&quot;bottomleft&quot;, 
       main = paste0(&quot;chr: &quot;, chr=peaks$chr[i],&quot;; pos: &quot;, peaks$pos[i], &quot;cM /&quot;,peaks_mba$pos[i],&quot;MB\n(&quot;,peaks$phenotype[i],&quot;; 1.5 LOD drop interval [scan1blup; positions in cM] )&quot;)
       )


 # Plot 3
  #c2effB &lt;- scan1coef(pr.qc[,chr], gm$covar[peaks$lodcolumn[i]], model=&quot;binary&quot;, contrasts=cbind(a=c(-1, 0), d=c(0, -1)))
  #c2effBb &lt;- scan1blup(pr.qc[,chr], gm$covar[peaks$lodcolumn[i]], contrasts=cbind(a=c(-1, 0), d=c(0, -1)))
  ##c2effB &lt;- scan1coef(pr[,chr], cross$pheno[,peaks$lodcolumn[i]], addcovar = addcovar, contrasts=cbind(mu=c(1,1,1), a=c(-1, 0, 1), d=c(0, 1, 0)))
  ##c2effB &lt;- scan1coef(pr[,chr], cross$pheno[,peaks$lodcolumn[i]],Xcovar=Xcovar, contrasts=cbind(mu=c(1,1,1), a=c(-1, 0, 1), d=c(0, 1, 0)))
  #plot(c2effB, gm$gmap[chr], columns=1:2,
  #     bgcolor=&quot;gray95&quot;, legend=&quot;bottomleft&quot;, 
  #     main = paste(&quot;chr&quot;, chr=peaks$chr[i], &quot;pos&quot;, peaks$pos[i], &quot;(&quot;,peaks$lodcolumn[i],&quot;)&quot;)
  #     )
  #plot(c2effBb, gm$gmap[chr], columns=1:2,
  #     bgcolor=&quot;gray95&quot;, legend=&quot;bottomleft&quot;, 
  #     main = paste(&quot;chr&quot;, chr=peaks$chr[i], &quot;pos&quot;, peaks$pos[i], &quot;(&quot;,peaks$lodcolumn[i],&quot;)&quot;)
  #     )
  ##last_coef &lt;- unclass(c2effB)[nrow(c2effB),2:3] # last two coefficients
  ##for(t in seq(along=last_coef))
  ##  axis(side=4, at=last_coef[t], names(last_coef)[t], tick=FALSE)


  #Table 1
  chr = peaks_mba$chr[i]
  start=as.numeric(peaks_mba$ci_lo[i])
  end=as.numeric(peaks_mba$ci_hi[i])

  genesgss = query_genes(chr, start, end)

  write.csv(genesgss, file=paste0(&quot;data/ici.vs.eoi_&quot;,peaks$phenotype[i],&quot;-additive.covariates_genes_chr-&quot;,chr,&quot;_peak.marker-&quot;,peaks$marker[i],&quot;_lod.drop-1.5_snpsqc_5.batches_0.csv&quot;), quote=F)

  rownames(genesgss) &lt;- NULL
  genesgss$strand_old = genesgss$strand
  genesgss$strand[genesgss$strand==&quot;+&quot;] &lt;- &quot;positive&quot;
  genesgss$strand[genesgss$strand==&quot;-&quot;] &lt;- &quot;negative&quot;

  #genesgss &lt;- 
  #table &lt;- 
  #genesgss[,c(&quot;chr&quot;,&quot;type&quot;,&quot;start&quot;,&quot;stop&quot;,&quot;strand&quot;,&quot;ID&quot;,&quot;Name&quot;,&quot;Dbxref&quot;,&quot;gene_id&quot;,&quot;mgi_type&quot;,&quot;description&quot;)] %&gt;%
  #kable(escape = F,align = c(&quot;ccccccccccc&quot;)) %&gt;%
  #kable_styling(&quot;striped&quot;, full_width = T) #%&gt;% 
  #cat #%&gt;%
  #column_spec(1, bold=TRUE)
#
  #print(kable(genesgss[,c(&quot;chr&quot;,&quot;type&quot;,&quot;start&quot;,&quot;stop&quot;,&quot;strand&quot;,&quot;ID&quot;,&quot;Name&quot;,&quot;Dbxref&quot;,&quot;gene_id&quot;,&quot;mgi_type&quot;,&quot;description&quot;)], escape = F,align = c(&quot;ccccccccccc&quot;)))

  print(kable(genesgss[,c(&quot;chr&quot;,&quot;type&quot;,&quot;start&quot;,&quot;stop&quot;,&quot;strand&quot;,&quot;ID&quot;,&quot;Name&quot;,&quot;Dbxref&quot;,&quot;gene_id&quot;,&quot;mgi_type&quot;,&quot;description&quot;)], &quot;html&quot;) %&gt;% kable_styling(&quot;striped&quot;, full_width = T))

  #table
  

}
} else {
  print(paste0(&quot;There are too many peaks (&quot;,nrow(peaks_mba),&quot; peaks) that have a LOD that reaches suggestive (p&lt;0.05) level of &quot;,summary(operm,alpha=0.05)$A, &quot; [autosomes]/&quot;,summary(operm,alpha=0.05)$X, &quot; [x-chromosome]&quot;))
}</code></pre>
<p>[1] “There are too many peaks (0 peaks) that have a LOD that reaches suggestive (p&lt;0.05) level of 2.77766557899907 [autosomes]/2.34159545294357 [x-chromosome]”</p>
</div>
</div>
<div id="interactive-term" class="section level2">
<h2>Interactive term</h2>
<div id="significance-thresholds-2" class="section level3">
<h3>Significance thresholds</h3>
<pre class="r"><code>operm &lt;- scan1perm(pr.qc, covars[&quot;age.of.onset&quot;], model=&quot;normal&quot;, addcovar=addcovar, intcovar=addcovar, n_perm=10, perm_Xsp=TRUE, chr_lengths=chr_lengths(gm$gmap))

summary_table&lt;-data.frame(unclass(summary(operm, alpha=c(0.01,  0.05, 0.1))))
names(summary_table) &lt;- c(&quot;autosomes&quot;,&quot;X&quot;)
summary_table$significance.level &lt;- rownames(summary_table)

rownames(summary_table) &lt;- NULL

summary_table[c(3,1:2)] %&gt;%
  kable(escape = F,align = c(&quot;ccc&quot;)) %&gt;%
  kable_styling(&quot;striped&quot;, full_width = T) %&gt;%
  column_spec(1, bold=TRUE)</code></pre>
<table class="table table-striped" style="margin-left: auto; margin-right: auto;">
<thead>
<tr>
<th style="text-align:center;">
significance.level
</th>
<th style="text-align:center;">
autosomes
</th>
<th style="text-align:center;">
X
</th>
</tr>
</thead>
<tbody>
<tr>
<td style="text-align:center;font-weight: bold;">
0.01
</td>
<td style="text-align:center;">
16.374005
</td>
<td style="text-align:center;">
6.085862
</td>
</tr>
<tr>
<td style="text-align:center;font-weight: bold;">
0.05
</td>
<td style="text-align:center;">
12.752893
</td>
<td style="text-align:center;">
5.739027
</td>
</tr>
<tr>
<td style="text-align:center;font-weight: bold;">
0.1
</td>
<td style="text-align:center;">
8.215202
</td>
<td style="text-align:center;">
5.285479
</td>
</tr>
</tbody>
</table>
</div>
<div id="genome-wide-scan-2" class="section level3">
<h3>Genome-wide scan</h3>
<pre class="r"><code>plot_lod&lt;-function(out,map){
  for (i in 1:dim(out)[2]){
#par(mar=c(5.1, 6.1, 1.1, 1.1))
    ymx &lt;- maxlod(out) # overall maximum LOD score
    plot(out, map, lodcolumn=i, col=&quot;slateblue&quot;, ylim=c(0, ymx+0.5))
    #legend(&quot;topright&quot;, lwd=2, colnames(out)[i], bg=&quot;gray90&quot;)
    title(main = paste0(colnames(out)[i], &quot; - ICI vs. EOI (with interactive covariate) [positions in cM]&quot;))
    add_threshold(map,  summary(operm,alpha=0.1), col = &#39;purple&#39;)
    add_threshold(map,  summary(operm, alpha=0.05), col = &#39;red&#39;)
    add_threshold(map,  summary(operm, alpha=0.01), col = &#39;blue&#39;)

  }
}


print(&quot;with no kinship&quot;)</code></pre>
<p>[1] “with no kinship”</p>
<pre class="r"><code>outfi &lt;- scan1(pr.qc, covars[&quot;age.of.onset&quot;], Xcovar=Xcovar, model=&quot;normal&quot;, addcovar=addcovar, intcovar=addcovar)

plot_lod(outfi,gm$gmap)</code></pre>
<p><img src="figure/4.1.2_qtl.analysis_cont_age_ici.vs.eoi_snpsqc_pheno.corrected.cleaned_5.batches_0.Rmd/scanonei-1.png" width="912" style="display: block; margin: auto;" /></p>
<pre class="r"><code>outf.new &lt;- outfi-outfa

plot_lod(outf.new,gm$gmap)</code></pre>
<p><img src="figure/4.1.2_qtl.analysis_cont_age_ici.vs.eoi_snpsqc_pheno.corrected.cleaned_5.batches_0.Rmd/scanonei-2.png" width="912" style="display: block; margin: auto;" /></p>
<pre class="r"><code>print(&quot;with normal kinship&quot;)</code></pre>
<p>[1] “with normal kinship”</p>
<pre class="r"><code>outki &lt;- scan1(pr.qc, covars[&quot;age.of.onset&quot;], Xcovar=Xcovar, model=&quot;normal&quot;, addcovar=addcovar, intcovar=addcovar, kinship=kinship)

plot_lod(outki,gm$gmap)</code></pre>
<p><img src="figure/4.1.2_qtl.analysis_cont_age_ici.vs.eoi_snpsqc_pheno.corrected.cleaned_5.batches_0.Rmd/scanonei-3.png" width="912" style="display: block; margin: auto;" /></p>
<pre class="r"><code>outk.new &lt;- outki-outka

plot_lod(outk.new,gm$gmap)</code></pre>
<p><img src="figure/4.1.2_qtl.analysis_cont_age_ici.vs.eoi_snpsqc_pheno.corrected.cleaned_5.batches_0.Rmd/scanonei-4.png" width="912" style="display: block; margin: auto;" /></p>
<pre class="r"><code>print(&quot;with loco kinship&quot;)</code></pre>
<p>[1] “with loco kinship”</p>
<pre class="r"><code>outi &lt;- scan1(pr.qc, covars[&quot;age.of.onset&quot;], Xcovar=Xcovar, model=&quot;normal&quot;, addcovar=addcovar, intcovar=addcovar, kinship=K)

plot_lod(outi,gm$gmap)</code></pre>
<p><img src="figure/4.1.2_qtl.analysis_cont_age_ici.vs.eoi_snpsqc_pheno.corrected.cleaned_5.batches_0.Rmd/scanonei-5.png" width="912" style="display: block; margin: auto;" /></p>
<pre class="r"><code>outi.new &lt;- outi-outa

plot_lod(outi.new,gm$gmap)</code></pre>
<p><img src="figure/4.1.2_qtl.analysis_cont_age_ici.vs.eoi_snpsqc_pheno.corrected.cleaned_5.batches_0.Rmd/scanonei-6.png" width="912" style="display: block; margin: auto;" /></p>
<pre class="r"><code>out = outi.new


print(paste0(&quot;number of samples in analysis = &quot;, as.data.frame(attributes(out)$sample_size)[1,]))</code></pre>
<p>[1] “number of samples in analysis = 266”</p>
</div>
<div id="lod-peaks-2" class="section level3">
<h3>LOD peaks</h3>
<div id="centimorgan-cm-2" class="section level4">
<h4>Centimorgan (cM)</h4>
<pre class="r"><code>peaks&lt;-find_peaks(out, gm$gmap, threshold=summary(operm,alpha=0.05)$A, thresholdX = summary(operm,alpha=0.05)$X, peakdrop=3, drop=1.5)
if(nrow(peaks) &gt; 0){
peaks$marker &lt;- find_marker(gm$gmap, chr=peaks$chr,pos=peaks$pos)
names(peaks)[2] &lt;- c(&quot;phenotype&quot;)
peaks &lt;- peaks[-1]

rownames(peaks) &lt;- NULL
print(kable(peaks, escape = F, align = c(&quot;cccccccc&quot;), &quot;html&quot;) 
  %&gt;% kable_styling(&quot;striped&quot;, full_width = T)%&gt;%
  column_spec(1, bold=TRUE)
  )

#plot only peak chromosomes

plot_lod_chr&lt;-function(out,map,chrom){
  for (i in 1:dim(out)[2]){
    #png(filename=paste0(&quot;/Users/chenm/Documents/qtl/Jai/&quot;,colnames(out)[i],  &quot;_lod.png&quot;))
    
    #par(mar=c(5.1, 6.1, 1.1, 1.1))
    ymx &lt;- maxlod(out) # overall maximum LOD score
    plot(out, map, chr = chrom, lodcolumn=i, col=&quot;slateblue&quot;, ylim=c(0, ymx+0.5))
    #legend(&quot;topright&quot;, lwd=2, colnames(out)[i], bg=&quot;gray90&quot;)
    title(main = paste0(colnames(out)[i], &quot; - chr&quot;, chrom, &quot; - ICI vs. EOI (with interactive covariate) [positions in cM]&quot;))
    add_threshold(map,  summary(operm,alpha=0.1), col = &#39;purple&#39;)
    add_threshold(map,  summary(operm, alpha=0.05), col = &#39;red&#39;)
    add_threshold(map,  summary(operm, alpha=0.01), col = &#39;blue&#39;)
    #for (j in 1: dim(summary_table)[1]){
    #  abline(h=summary_table[j, i],col=&quot;red&quot;)
    #  text(x=400, y =summary_table[j, i]+0.12, labels = paste(&quot;p=&quot;, row.names(summary_table)[j]))
    #}
    #dev.off()

  }
}


} else {
  print(paste0(&quot;There are no peaks that have a LOD that reaches suggestive (p&lt;0.05) level of &quot;,summary(operm,alpha=0.05)$A, &quot; [autosomes]/&quot;,summary(operm,alpha=0.05)$X, &quot; [x-chromosome]&quot;))
}</code></pre>
<p>[1] “There are no peaks that have a LOD that reaches suggestive (p&lt;0.05) level of 12.7528926653117 [autosomes]/5.73902655624868 [x-chromosome]”</p>
<pre class="r"><code>if(nrow(peaks) &lt; 50 &amp; nrow(peaks) &gt; 0){

for(i in unique(peaks$chr)){
#for (i in 1:nrow(peaks)){
  #plot_lod_chr(out,gm$gmap, peaks$chr[i])
  plot_lod_chr(out,gm$gmap, i)
}

} else {
  print(paste0(&quot;There are too many peaks (&quot;,nrow(peaks),&quot; peaks) that have a LOD that reaches suggestive (p&lt;0.05) level of &quot;,summary(operm,alpha=0.05)$A, &quot; [autosomes]/&quot;,summary(operm,alpha=0.05)$X, &quot; [x-chromosome]&quot;))
}</code></pre>
<p>[1] “There are too many peaks (0 peaks) that have a LOD that reaches suggestive (p&lt;0.05) level of 12.7528926653117 [autosomes]/5.73902655624868 [x-chromosome]”</p>
</div>
<div id="megabase-mb-2" class="section level4">
<h4>Megabase (MB)</h4>
<pre class="r"><code>peaks_mba&lt;-find_peaks(out, gm$pmap, threshold=summary(operm,alpha=0.05)$A, thresholdX = summary(operm,alpha=0.05)$X, peakdrop=3, drop=1.5)
if(nrow(peaks) &gt; 0){
peaks_mba$marker &lt;- find_marker(gm$pmap, chr=peaks_mba$chr,pos=peaks_mba$pos)
names(peaks_mba)[2] &lt;- c(&quot;phenotype&quot;)
peaks_mba &lt;- peaks_mba[-1]

rownames(peaks_mba) &lt;- NULL
print(kable(peaks_mba, escape = F, align = c(&quot;cccccccc&quot;), &quot;html&quot;) 
  %&gt;% kable_styling(&quot;striped&quot;, full_width = T)%&gt;%
  column_spec(1, bold=TRUE)
  )

plot_lod_chr_mb&lt;-function(out,map,chrom){
  for (i in 1:dim(out)[2]){
    #png(filename=paste0(&quot;/Users/chenm/Documents/qtl/Jai/&quot;,colnames(out)[i],  &quot;_lod.png&quot;))
    
    #par(mar=c(5.1, 6.1, 1.1, 1.1))
    ymx &lt;- maxlod(out) # overall maximum LOD score
    plot(out, map, chr = chrom, lodcolumn=i, col=&quot;slateblue&quot;, ylim=c(0, ymx+0.5))
    #legend(&quot;topright&quot;, lwd=2, colnames(out)[i], bg=&quot;gray90&quot;)
    title(main = paste0(colnames(out)[i], &quot; - chr&quot;, chrom, &quot; - ICI vs. EOI (with interactive covariate) [positions in cM]&quot;))
    add_threshold(map,  summary(operm,alpha=0.1), col = &#39;purple&#39;)
    add_threshold(map,  summary(operm, alpha=0.05), col = &#39;red&#39;)
    add_threshold(map,  summary(operm, alpha=0.01), col = &#39;blue&#39;)
    #for (j in 1: dim(summary_table)[1]){
    #  abline(h=summary_table[j, i],col=&quot;red&quot;)
    #  text(x=400, y =summary_table[j, i]+0.12, labels = paste(&quot;p=&quot;, row.names(summary_table)[j]))
    #}
    #dev.off()
  }
}

} else {
  print(paste0(&quot;There are no peaks that have a LOD that reaches suggestive (p&lt;0.05) level of &quot;,summary(operm,alpha=0.05)$A, &quot; [autosomes]/&quot;,summary(operm,alpha=0.05)$X, &quot; [x-chromosome]&quot;))
}</code></pre>
<p>[1] “There are no peaks that have a LOD that reaches suggestive (p&lt;0.05) level of 12.7528926653117 [autosomes]/5.73902655624868 [x-chromosome]”</p>
<pre class="r"><code>if(nrow(peaks_mba) &lt; 50 &amp; nrow(peaks_mba) &gt; 0){

for(i in unique(peaks_mba$chr)){
#for (i in 1:nrow(peaks_mba)){
  #plot_lod_chr_mb(out,gm$pmap, peaks_mba$chr[i])
  plot_lod_chr_mb(out,gm$pmap,i)
}

} else {
  print(paste0(&quot;There are too many peaks (&quot;,nrow(peaks_mba),&quot; peaks) that have a LOD that reaches suggestive (p&lt;0.05) level of &quot;,summary(operm,alpha=0.05)$A, &quot; [autosomes]/&quot;,summary(operm,alpha=0.05)$X, &quot; [x-chromosome]&quot;))
}</code></pre>
<p>[1] “There are too many peaks (0 peaks) that have a LOD that reaches suggestive (p&lt;0.05) level of 12.7528926653117 [autosomes]/5.73902655624868 [x-chromosome]”</p>
</div>
</div>
<div id="qtl-effects-2" class="section level3">
<h3>QTL effects</h3>
<p>For each peak LOD location we give a list of gene</p>
<pre class="r"><code>if(nrow(peaks) &lt; 50 &amp; nrow(peaks) &gt; 0){
for (i in 1:nrow(peaks)){
#for (i in 1:1){
  #Plot 1
  g &lt;- maxmarg(pr.qc, gm$gmap, chr=peaks$chr[i], pos=peaks$pos[i], return_char=TRUE)
  #png(filename=paste0(&quot;/Users/chenm/Documents/qtl/Jai/&quot;,&quot;qtl_effect_&quot;, i, &quot;.png&quot;))
  #par(mar=c(4.1, 4.1, 1.5, 0.6))
  plot_pxg(g, covars[,peaks$phenotype[i]], ylab=peaks$phenotype[i], sort=FALSE)
  title(main = paste0(&quot;chr: &quot;, chr=peaks$chr[i],&quot;; pos: &quot;, peaks$pos[i], &quot;cM /&quot;,peaks_mba$pos[i],&quot;MB\n(&quot;,peaks$phenotype[i],&quot; )&quot;), line=0.2)
  ##dev.off()

  chr = peaks$chr[i]

# Plot 2
  pr_sub &lt;- pull_genoprobint(pr.qc, gm$gmap, chr, c(peaks$ci_lo[i], peaks$ci_hi[i]))
  #coeff &lt;- scan1coef(pr[,chr], cross$pheno[,peaks$lodcolumn[i]], addcovar = addcovar)
  #coeff &lt;- scan1coef(pr[,chr], cross$pheno[,peaks$lodcolumn[i]], Xcovar=Xcovar)
  #coeff &lt;- scan1coef(pr.qc[,chr], gm$covar[peaks$lodcolumn[i]], model=&quot;binary&quot;)
  #coeff_sub &lt;- scan1coef(pr_sub[,chr], gm$covar[peaks$lodcolumn[i]], model=&quot;binary&quot;)
  blup &lt;- scan1blup(pr.qc[,chr], covars[peaks$phenotype[i]])
  blup_sub &lt;- scan1blup(pr_sub[,chr], covars[peaks$phenotype[i]])

  write.csv(as.data.frame(blup_sub), paste0(&quot;data/ici.vs.eoi_&quot;,peaks$phenotype[i],&quot;-interactive.covariate_blup_sub_chr-&quot;,chr,&quot;_peak.marker-&quot;,peaks$marker[i],&quot;_lod.drop-1.5_snpsqc_5.batches_0.csv&quot;), quote=F)

  #plot_coef(coeff, 
  #     gm$gmap, columns=1:2,
  #     bgcolor=&quot;gray95&quot;, legend=&quot;bottomleft&quot;, 
  #     main = paste(&quot;chr&quot;, chr=peaks$chr[i],&quot;; pos: &quot;, peaks$pos[i], &quot;cM /&quot;,peaks_mba$pos[i],&quot;MB\n(&quot;,peaks$lodcolumn[i],&quot; [scan1coeff; positions in cM] )&quot;)
  #     )

  #plot_coef(coeff_sub, 
  #     gm$gmap, columns=1:2,
  #     bgcolor=&quot;gray95&quot;, legend=&quot;bottomleft&quot;, 
  #     main = paste(&quot;chr&quot;, chr=peaks$chr[i],&quot;; pos: &quot;, peaks$pos[i], &quot;cM /&quot;,peaks_mba$pos[i],&quot;MB\n(&quot;,peaks$lodcolumn[i],&quot;; 1.5 LOD drop interval [scan1coeff; positions in cM] ) &quot;)
  #     )


  plot_coef(blup, 
       gm$gmap, columns=1:2,
       bgcolor=&quot;gray95&quot;, legend=&quot;bottomleft&quot;, 
       main = paste0(&quot;chr: &quot;, chr=peaks$chr[i],&quot;; pos: &quot;, peaks$pos[i], &quot;cM /&quot;,peaks_mba$pos[i],&quot;MB\n(&quot;,peaks$phenotype[i],&quot; [scan1blup; positions in cM] )&quot;)
       )

  plot_coef(blup_sub, 
       gm$gmap, columns=1:2,
       bgcolor=&quot;gray95&quot;, legend=&quot;bottomleft&quot;, 
       main = paste0(&quot;chr: &quot;, chr=peaks$chr[i],&quot;; pos: &quot;, peaks$pos[i], &quot;cM /&quot;,peaks_mba$pos[i],&quot;MB\n(&quot;,peaks$phenotype[i],&quot;; 1.5 LOD drop interval [scan1blup; positions in cM] )&quot;)
       )


 # Plot 3
  #c2effB &lt;- scan1coef(pr.qc[,chr], gm$covar[peaks$lodcolumn[i]], model=&quot;binary&quot;, contrasts=cbind(a=c(-1, 0), d=c(0, -1)))
  #c2effBb &lt;- scan1blup(pr.qc[,chr], gm$covar[peaks$lodcolumn[i]], contrasts=cbind(a=c(-1, 0), d=c(0, -1)))
  ##c2effB &lt;- scan1coef(pr[,chr], cross$pheno[,peaks$lodcolumn[i]], addcovar = addcovar, contrasts=cbind(mu=c(1,1,1), a=c(-1, 0, 1), d=c(0, 1, 0)))
  ##c2effB &lt;- scan1coef(pr[,chr], cross$pheno[,peaks$lodcolumn[i]],Xcovar=Xcovar, contrasts=cbind(mu=c(1,1,1), a=c(-1, 0, 1), d=c(0, 1, 0)))
  #plot(c2effB, gm$gmap[chr], columns=1:2,
  #     bgcolor=&quot;gray95&quot;, legend=&quot;bottomleft&quot;, 
  #     main = paste(&quot;chr&quot;, chr=peaks$chr[i], &quot;pos&quot;, peaks$pos[i], &quot;(&quot;,peaks$lodcolumn[i],&quot;)&quot;)
  #     )
  #plot(c2effBb, gm$gmap[chr], columns=1:2,
  #     bgcolor=&quot;gray95&quot;, legend=&quot;bottomleft&quot;, 
  #     main = paste(&quot;chr&quot;, chr=peaks$chr[i], &quot;pos&quot;, peaks$pos[i], &quot;(&quot;,peaks$lodcolumn[i],&quot;)&quot;)
  #     )
  ##last_coef &lt;- unclass(c2effB)[nrow(c2effB),2:3] # last two coefficients
  ##for(t in seq(along=last_coef))
  ##  axis(side=4, at=last_coef[t], names(last_coef)[t], tick=FALSE)


  #Table 1
  chr = peaks_mba$chr[i]
  start=as.numeric(peaks_mba$ci_lo[i])
  end=as.numeric(peaks_mba$ci_hi[i])

  genesgss = query_genes(chr, start, end)

  write.csv(genesgss, file=paste0(&quot;data/ici.vs.eoi_&quot;,peaks$phenotype[i],&quot;-interactive.covariate_genes_chr-&quot;,chr,&quot;_peak.marker-&quot;,peaks$marker[i],&quot;_lod.drop-1.5_snpsqc_5.batches_0.csv&quot;), quote=F)

  rownames(genesgss) &lt;- NULL
  genesgss$strand_old = genesgss$strand
  genesgss$strand[genesgss$strand==&quot;+&quot;] &lt;- &quot;positive&quot;
  genesgss$strand[genesgss$strand==&quot;-&quot;] &lt;- &quot;negative&quot;

  #genesgss &lt;- 
  #table &lt;- 
  #genesgss[,c(&quot;chr&quot;,&quot;type&quot;,&quot;start&quot;,&quot;stop&quot;,&quot;strand&quot;,&quot;ID&quot;,&quot;Name&quot;,&quot;Dbxref&quot;,&quot;gene_id&quot;,&quot;mgi_type&quot;,&quot;description&quot;)] %&gt;%
  #kable(escape = F,align = c(&quot;ccccccccccc&quot;)) %&gt;%
  #kable_styling(&quot;striped&quot;, full_width = T) #%&gt;% 
  #cat #%&gt;%
  #column_spec(1, bold=TRUE)
#
  #print(kable(genesgss[,c(&quot;chr&quot;,&quot;type&quot;,&quot;start&quot;,&quot;stop&quot;,&quot;strand&quot;,&quot;ID&quot;,&quot;Name&quot;,&quot;Dbxref&quot;,&quot;gene_id&quot;,&quot;mgi_type&quot;,&quot;description&quot;)], escape = F,align = c(&quot;ccccccccccc&quot;)))

  print(kable(genesgss[,c(&quot;chr&quot;,&quot;type&quot;,&quot;start&quot;,&quot;stop&quot;,&quot;strand&quot;,&quot;ID&quot;,&quot;Name&quot;,&quot;Dbxref&quot;,&quot;gene_id&quot;,&quot;mgi_type&quot;,&quot;description&quot;)], &quot;html&quot;) %&gt;% kable_styling(&quot;striped&quot;, full_width = T))

  #table
  

}
} else {
  print(paste0(&quot;There are too many peaks (&quot;,nrow(peaks_mba),&quot; peaks) that have a LOD that reaches suggestive (p&lt;0.05) level of &quot;,summary(operm,alpha=0.05)$A, &quot; [autosomes]/&quot;,summary(operm,alpha=0.05)$X, &quot; [x-chromosome]&quot;))
}</code></pre>
<p>[1] “There are too many peaks (0 peaks) that have a LOD that reaches suggestive (p&lt;0.05) level of 12.7528926653117 [autosomes]/5.73902655624868 [x-chromosome]”</p>
<br>
<p>
<button type="button" class="btn btn-default btn-workflowr btn-workflowr-sessioninfo" data-toggle="collapse" data-target="#workflowr-sessioninfo" style="display: block;">
<span class="glyphicon glyphicon-wrench" aria-hidden="true"></span> Session information
</button>
</p>
<div id="workflowr-sessioninfo" class="collapse">
<pre><code>R version 3.6.2 (2019-12-12)
Platform: x86_64-apple-darwin15.6.0 (64-bit)
Running under: macOS Catalina 10.15.7

Matrix products: default
BLAS:   /Library/Frameworks/R.framework/Versions/3.6/Resources/lib/libRblas.0.dylib
LAPACK: /Library/Frameworks/R.framework/Versions/3.6/Resources/lib/libRlapack.dylib

locale:
[1] en_AU.UTF-8/en_AU.UTF-8/en_AU.UTF-8/C/en_AU.UTF-8/en_AU.UTF-8

attached base packages:
[1] stats     graphics  grDevices utils     datasets  methods   base     

other attached packages:
 [1] abind_1.4-5       qtl2_0.22         reshape2_1.4.4    ggplot2_3.3.5    
 [5] tibble_3.1.2      psych_2.0.7       readxl_1.3.1      cluster_2.1.0    
 [9] dplyr_1.0.8       optparse_1.6.6    rhdf5_2.28.1      mclust_5.4.6     
[13] tidyr_1.0.2       data.table_1.14.0 knitr_1.33        kableExtra_1.1.0 
[17] workflowr_1.6.2  

loaded via a namespace (and not attached):
 [1] httr_1.4.1        bit64_4.0.5       viridisLite_0.4.0 assertthat_0.2.1 
 [5] highr_0.9         blob_1.2.1        cellranger_1.1.0  yaml_2.2.1       
 [9] pillar_1.6.1      RSQLite_2.2.7     backports_1.2.1   lattice_0.20-38  
[13] glue_1.4.2        digest_0.6.27     promises_1.1.0    rvest_0.3.5      
[17] colorspace_2.0-2  htmltools_0.5.1.1 httpuv_1.5.2      plyr_1.8.6       
[21] pkgconfig_2.0.3   purrr_0.3.4       scales_1.1.1      webshot_0.5.2    
[25] getopt_1.20.3     later_1.0.0       git2r_0.26.1      generics_0.0.2   
[29] ellipsis_0.3.2    cachem_1.0.5      withr_2.4.2       cli_3.0.0        
[33] mnormt_1.5-7      magrittr_2.0.1    crayon_1.4.1      memoise_2.0.0    
[37] evaluate_0.14     fs_1.4.1          fansi_0.5.0       nlme_3.1-142     
[41] xml2_1.3.1        tools_3.6.2       hms_0.5.3         lifecycle_1.0.1  
[45] stringr_1.4.0     Rhdf5lib_1.6.3    munsell_0.5.0     compiler_3.6.2   
[49] rlang_1.0.2       grid_3.6.2        rstudioapi_0.13   rmarkdown_2.1    
[53] gtable_0.3.0      DBI_1.1.1         R6_2.5.0          fastmap_1.1.0    
[57] bit_4.0.4         utf8_1.2.1        rprojroot_1.3-2   readr_1.3.1      
[61] stringi_1.7.2     parallel_3.6.2    Rcpp_1.0.7        vctrs_0.3.8      
[65] tidyselect_1.1.2  xfun_0.24        </code></pre>
</div>
</div>
</div>
</div>


<!-- Adjust MathJax settings so that all math formulae are shown using
TeX fonts only; see
http://docs.mathjax.org/en/latest/configuration.html.  This will make
the presentation more consistent at the cost of the webpage sometimes
taking slightly longer to load. Note that this only works because the
footer is added to webpages before the MathJax javascript. -->
<script type="text/x-mathjax-config">
  MathJax.Hub.Config({
    "HTML-CSS": { availableFonts: ["TeX"] }
  });
</script>





</div>

<script>

// add bootstrap table styles to pandoc tables
function bootstrapStylePandocTables() {
  $('tr.header').parent('thead').parent('table').addClass('table table-condensed');
}
$(document).ready(function () {
  bootstrapStylePandocTables();
});


</script>

<!-- tabsets -->

<script>
$(document).ready(function () {
  window.buildTabsets("TOC");
});

$(document).ready(function () {
  $('.tabset-dropdown > .nav-tabs > li').click(function () {
    $(this).parent().toggleClass('nav-tabs-open')
  });
});
</script>

<!-- code folding -->


<!-- dynamically load mathjax for compatibility with self-contained -->
<script>
  (function () {
    var script = document.createElement("script");
    script.type = "text/javascript";
    script.src  = "https://mathjax.rstudio.com/latest/MathJax.js?config=TeX-AMS-MML_HTMLorMML";
    document.getElementsByTagName("head")[0].appendChild(script);
  })();
</script>

</body>
</html>
