<!DOCTYPE html>

<html>

<head>

<meta charset="utf-8" />
<meta name="generator" content="pandoc" />
<meta http-equiv="X-UA-Compatible" content="IE=EDGE" />


<meta name="author" content="Belinda Cornes" />

<meta name="date" content="2023-04-20" />

<title>QTL Analysis - binary [ICI vs PBS]</title>

<script src="site_libs/header-attrs-2.6/header-attrs.js"></script>
<script src="site_libs/jquery-1.11.3/jquery.min.js"></script>
<meta name="viewport" content="width=device-width, initial-scale=1" />
<link href="site_libs/bootstrap-3.3.5/css/spacelab.min.css" rel="stylesheet" />
<script src="site_libs/bootstrap-3.3.5/js/bootstrap.min.js"></script>
<script src="site_libs/bootstrap-3.3.5/shim/html5shiv.min.js"></script>
<script src="site_libs/bootstrap-3.3.5/shim/respond.min.js"></script>
<script src="site_libs/navigation-1.1/tabsets.js"></script>
<link href="site_libs/highlightjs-9.12.0/textmate.css" rel="stylesheet" />
<script src="site_libs/highlightjs-9.12.0/highlight.js"></script>
<script src="site_libs/kePrint-0.0.1/kePrint.js"></script>
<link href="site_libs/lightable-0.0.1/lightable.css" rel="stylesheet" />
<link href="site_libs/font-awesome-5.1.0/css/all.css" rel="stylesheet" />
<link href="site_libs/font-awesome-5.1.0/css/v4-shims.css" rel="stylesheet" />

<link rel="icon" href="https://github.com/workflowr/workflowr-assets/raw/master/img/reproducible.png">
<!-- Add a small amount of space between sections. -->
<style type="text/css">
div.section {
  padding-top: 12px;
}
</style>



<style type="text/css">
  code{white-space: pre-wrap;}
  span.smallcaps{font-variant: small-caps;}
  span.underline{text-decoration: underline;}
  div.column{display: inline-block; vertical-align: top; width: 50%;}
  div.hanging-indent{margin-left: 1.5em; text-indent: -1.5em;}
  ul.task-list{list-style: none;}
    </style>

<style type="text/css">code{white-space: pre;}</style>
<style type="text/css">
  pre:not([class]) {
    background-color: white;
  }
</style>
<script type="text/javascript">
if (window.hljs) {
  hljs.configure({languages: []});
  hljs.initHighlightingOnLoad();
  if (document.readyState && document.readyState === "complete") {
    window.setTimeout(function() { hljs.initHighlighting(); }, 0);
  }
}
</script>



<style type="text/css">
h1 {
  font-size: 34px;
}
h1.title {
  font-size: 38px;
}
h2 {
  font-size: 30px;
}
h3 {
  font-size: 24px;
}
h4 {
  font-size: 18px;
}
h5 {
  font-size: 16px;
}
h6 {
  font-size: 12px;
}
.table th:not([align]) {
  text-align: left;
}
</style>





<style type = "text/css">
.main-container {
  max-width: 940px;
  margin-left: auto;
  margin-right: auto;
}
code {
  color: inherit;
  background-color: rgba(0, 0, 0, 0.04);
}
img {
  max-width:100%;
}
.tabbed-pane {
  padding-top: 12px;
}
.html-widget {
  margin-bottom: 20px;
}
button.code-folding-btn:focus {
  outline: none;
}
summary {
  display: list-item;
}
</style>


<style type="text/css">
/* padding for bootstrap navbar */
body {
  padding-top: 51px;
  padding-bottom: 40px;
}
/* offset scroll position for anchor links (for fixed navbar)  */
.section h1 {
  padding-top: 56px;
  margin-top: -56px;
}
.section h2 {
  padding-top: 56px;
  margin-top: -56px;
}
.section h3 {
  padding-top: 56px;
  margin-top: -56px;
}
.section h4 {
  padding-top: 56px;
  margin-top: -56px;
}
.section h5 {
  padding-top: 56px;
  margin-top: -56px;
}
.section h6 {
  padding-top: 56px;
  margin-top: -56px;
}
.dropdown-submenu {
  position: relative;
}
.dropdown-submenu>.dropdown-menu {
  top: 0;
  left: 100%;
  margin-top: -6px;
  margin-left: -1px;
  border-radius: 0 6px 6px 6px;
}
.dropdown-submenu:hover>.dropdown-menu {
  display: block;
}
.dropdown-submenu>a:after {
  display: block;
  content: " ";
  float: right;
  width: 0;
  height: 0;
  border-color: transparent;
  border-style: solid;
  border-width: 5px 0 5px 5px;
  border-left-color: #cccccc;
  margin-top: 5px;
  margin-right: -10px;
}
.dropdown-submenu:hover>a:after {
  border-left-color: #ffffff;
}
.dropdown-submenu.pull-left {
  float: none;
}
.dropdown-submenu.pull-left>.dropdown-menu {
  left: -100%;
  margin-left: 10px;
  border-radius: 6px 0 6px 6px;
}
</style>

<script>
// manage active state of menu based on current page
$(document).ready(function () {
  // active menu anchor
  href = window.location.pathname
  href = href.substr(href.lastIndexOf('/') + 1)
  if (href === "")
    href = "index.html";
  var menuAnchor = $('a[href="' + href + '"]');

  // mark it active
  menuAnchor.parent().addClass('active');

  // if it's got a parent navbar menu mark it active as well
  menuAnchor.closest('li.dropdown').addClass('active');
});
</script>

<!-- tabsets -->

<style type="text/css">
.tabset-dropdown > .nav-tabs {
  display: inline-table;
  max-height: 500px;
  min-height: 44px;
  overflow-y: auto;
  background: white;
  border: 1px solid #ddd;
  border-radius: 4px;
}

.tabset-dropdown > .nav-tabs > li.active:before {
  content: "";
  font-family: 'Glyphicons Halflings';
  display: inline-block;
  padding: 10px;
  border-right: 1px solid #ddd;
}

.tabset-dropdown > .nav-tabs.nav-tabs-open > li.active:before {
  content: "&#xe258;";
  border: none;
}

.tabset-dropdown > .nav-tabs.nav-tabs-open:before {
  content: "";
  font-family: 'Glyphicons Halflings';
  display: inline-block;
  padding: 10px;
  border-right: 1px solid #ddd;
}

.tabset-dropdown > .nav-tabs > li.active {
  display: block;
}

.tabset-dropdown > .nav-tabs > li > a,
.tabset-dropdown > .nav-tabs > li > a:focus,
.tabset-dropdown > .nav-tabs > li > a:hover {
  border: none;
  display: inline-block;
  border-radius: 4px;
  background-color: transparent;
}

.tabset-dropdown > .nav-tabs.nav-tabs-open > li {
  display: block;
  float: none;
}

.tabset-dropdown > .nav-tabs > li {
  display: none;
}
</style>

<!-- code folding -->




</head>

<body>


<div class="container-fluid main-container">




<div class="navbar navbar-default  navbar-fixed-top" role="navigation">
  <div class="container">
    <div class="navbar-header">
      <button type="button" class="navbar-toggle collapsed" data-toggle="collapse" data-target="#navbar">
        <span class="icon-bar"></span>
        <span class="icon-bar"></span>
        <span class="icon-bar"></span>
      </button>
      <a class="navbar-brand" href="index.html">Serreze Autoimmune Workflow</a>
    </div>
    <div id="navbar" class="navbar-collapse collapse">
      <ul class="nav navbar-nav">
        <li>
  <a href="index.html">Home</a>
</li>
<li>
  <a href="about.html">About</a>
</li>
<li>
  <a href="license.html">License</a>
</li>
      </ul>
      <ul class="nav navbar-nav navbar-right">
        <li>
  <a href="https://github.com/TheJacksonLaboratory/Serreze-T1D_Workflow">
    <span class="fas fa-github"></span>
     
    Source code
  </a>
</li>
      </ul>
    </div><!--/.nav-collapse -->
  </div><!--/.container -->
</div><!--/.navbar -->

<div class="fluid-row" id="header">



<h1 class="title toc-ignore">QTL Analysis - binary [ICI vs PBS]</h1>
<h4 class="author">Belinda Cornes</h4>
<h4 class="date">2023-04-20</h4>

</div>

<div id="TOC">
<ul>
<li><a href="#data-information">Data Information</a></li>
<li><a href="#loading-data">Loading Data</a></li>
<li><a href="#genome-wide-scan">Genome-wide scan</a></li>
<li><a href="#lod-peaks">LOD peaks</a>
<ul>
<li><a href="#centimorgan-cm">Centimorgan (cM)</a></li>
<li><a href="#megabase-mb">Megabase (MB)</a></li>
</ul></li>
<li><a href="#qtl-effects">QTL effects</a></li>
<li><a href="#rqtl">R/qtl</a>
<ul>
<li><a href="#scanone">scanone</a></li>
</ul></li>
</ul>
</div>

<p>
<button type="button" class="btn btn-default btn-workflowr btn-workflowr-report" data-toggle="collapse" data-target="#workflowr-report">
<span class="glyphicon glyphicon-list" aria-hidden="true"></span> workflowr <span class="glyphicon glyphicon-exclamation-sign text-danger" aria-hidden="true"></span>
</button>
</p>
<div id="workflowr-report" class="collapse">
<ul class="nav nav-tabs">
<li class="active">
<a data-toggle="tab" href="#summary">Summary</a>
</li>
<li>
<a data-toggle="tab" href="#checks"> Checks <span class="glyphicon glyphicon-exclamation-sign text-danger" aria-hidden="true"></span> </a>
</li>
<li>
<a data-toggle="tab" href="#versions">Past versions</a>
</li>
</ul>
<div class="tab-content">
<div id="summary" class="tab-pane fade in active">
<p>
<strong>Last updated:</strong> 2023-04-20
</p>
<p>
<strong>Checks:</strong> <span class="glyphicon glyphicon-ok text-success" aria-hidden="true"></span> 5 <span class="glyphicon glyphicon-exclamation-sign text-danger" aria-hidden="true"></span> 2
</p>
<p>
<strong>Knit directory:</strong> <code>Serreze-T1D_Workflow/</code> <span class="glyphicon glyphicon-question-sign" aria-hidden="true" title="This is the local directory in which the code in this file was executed."> </span>
</p>
<p>
This reproducible <a href="http://rmarkdown.rstudio.com">R Markdown</a> analysis was created with <a
  href="https://github.com/jdblischak/workflowr">workflowr</a> (version 1.6.2). The <em>Checks</em> tab describes the reproducibility checks that were applied when the results were created. The <em>Past versions</em> tab lists the development history.
</p>
<hr>
</div>
<div id="checks" class="tab-pane fade">
<div id="workflowr-checks" class="panel-group">
<div class="panel panel-default">
<div class="panel-heading">
<p class="panel-title">
<a data-toggle="collapse" data-parent="#workflowr-checks" href="#strongRMarkdownfilestronguncommittedchanges"> <span class="glyphicon glyphicon-exclamation-sign text-danger" aria-hidden="true"></span> <strong>R Markdown file:</strong> uncommitted changes </a>
</p>
</div>
<div id="strongRMarkdownfilestronguncommittedchanges" class="panel-collapse collapse">
<div class="panel-body">
<p>The R Markdown is untracked by Git. To know which version of the R Markdown file created these results, you’ll want to first commit it to the Git repo. If you’re still working on the analysis, you can ignore this warning. When you’re finished, you can run <code>wflow_publish</code> to commit the R Markdown file and build the HTML.</p>
</div>
</div>
</div>
<div class="panel panel-default">
<div class="panel-heading">
<p class="panel-title">
<a data-toggle="collapse" data-parent="#workflowr-checks" href="#strongEnvironmentstrongempty"> <span class="glyphicon glyphicon-ok text-success" aria-hidden="true"></span> <strong>Environment:</strong> empty </a>
</p>
</div>
<div id="strongEnvironmentstrongempty" class="panel-collapse collapse">
<div class="panel-body">
<p>Great job! The global environment was empty. Objects defined in the global environment can affect the analysis in your R Markdown file in unknown ways. For reproduciblity it’s best to always run the code in an empty environment.</p>
</div>
</div>
</div>
<div class="panel panel-default">
<div class="panel-heading">
<p class="panel-title">
<a data-toggle="collapse" data-parent="#workflowr-checks" href="#strongSeedstrongcodesetseed20230413code"> <span class="glyphicon glyphicon-ok text-success" aria-hidden="true"></span> <strong>Seed:</strong> <code>set.seed(20230413)</code> </a>
</p>
</div>
<div id="strongSeedstrongcodesetseed20230413code" class="panel-collapse collapse">
<div class="panel-body">
<p>The command <code>set.seed(20230413)</code> was run prior to running the code in the R Markdown file. Setting a seed ensures that any results that rely on randomness, e.g. subsampling or permutations, are reproducible.</p>
</div>
</div>
</div>
<div class="panel panel-default">
<div class="panel-heading">
<p class="panel-title">
<a data-toggle="collapse" data-parent="#workflowr-checks" href="#strongSessioninformationstrongrecorded"> <span class="glyphicon glyphicon-ok text-success" aria-hidden="true"></span> <strong>Session information:</strong> recorded </a>
</p>
</div>
<div id="strongSessioninformationstrongrecorded" class="panel-collapse collapse">
<div class="panel-body">
<p>Great job! Recording the operating system, R version, and package versions is critical for reproducibility.</p>
</div>
</div>
</div>
<div class="panel panel-default">
<div class="panel-heading">
<p class="panel-title">
<a data-toggle="collapse" data-parent="#workflowr-checks" href="#strongCachestrongnone"> <span class="glyphicon glyphicon-ok text-success" aria-hidden="true"></span> <strong>Cache:</strong> none </a>
</p>
</div>
<div id="strongCachestrongnone" class="panel-collapse collapse">
<div class="panel-body">
<p>Nice! There were no cached chunks for this analysis, so you can be confident that you successfully produced the results during this run.</p>
</div>
</div>
</div>
<div class="panel panel-default">
<div class="panel-heading">
<p class="panel-title">
<a data-toggle="collapse" data-parent="#workflowr-checks" href="#strongFilepathsstrongabsolute"> <span class="glyphicon glyphicon-exclamation-sign text-danger" aria-hidden="true"></span> <strong>File paths:</strong> absolute </a>
</p>
</div>
<div id="strongFilepathsstrongabsolute" class="panel-collapse collapse">
<div class="panel-body">
<p>
Using absolute paths to the files within your workflowr project makes it difficult for you and others to run your code on a different machine. Change the absolute path(s) below to the suggested relative path(s) to make your code more reproducible.
</p>
<table class="table table-condensed table-hover">
<thead>
<tr>
<th style="text-align:left;">
absolute
</th>
<th style="text-align:left;">
relative
</th>
</tr>
</thead>
<tbody>
<tr>
<td style="text-align:left;">
/projects/serreze-lab/USERS/corneb/qtl2/workflowr/2023.04.13/Serreze-T1D_Workflow
</td>
<td style="text-align:left;">
.
</td>
</tr>
</tbody>
</table>
</div>
</div>
</div>
<div class="panel panel-default">
<div class="panel-heading">
<p class="panel-title">
<a data-toggle="collapse" data-parent="#workflowr-checks" href="#strongRepositoryversionstrong03e83ae"> <span class="glyphicon glyphicon-ok text-success" aria-hidden="true"></span> <strong>Repository version:</strong> 03e83ae </a>
</p>
</div>
<div id="strongRepositoryversionstrong03e83ae" class="panel-collapse collapse">
<div class="panel-body">
<p>
Great! You are using Git for version control. Tracking code development and connecting the code version to the results is critical for reproducibility.
</p>
<p>
The results in this page were generated with repository version 03e83ae. See the <em>Past versions</em> tab to see a history of the changes made to the R Markdown and HTML files.
</p>
<p>
Note that you need to be careful to ensure that all relevant files for the analysis have been committed to Git prior to generating the results (you can use <code>wflow_publish</code> or <code>wflow_git_commit</code>). workflowr only checks the R Markdown file, but you know if there are other scripts or data files that it depends on. Below is the status of the Git repository when the results were generated:
</p>
<pre><code>
Untracked files:
    Untracked:  analysis/0.1.1_preparing.data_bqc_7.batches_myo.Rmd
    Untracked:  analysis/0.1_samples_batch_20230124.Rmd
    Untracked:  analysis/0.1_samples_batch_20230225.Rmd
    Untracked:  analysis/0.1_samples_batch_20230313.Rmd
    Untracked:  analysis/0.2_haplotype_comparison_bqc_7.batches_myo_minprob.Rmd
    Untracked:  analysis/2.1_sample_bqc_7.batches_myo.Rmd
    Untracked:  analysis/2.2.1_snp_qc_7.batches_myo.Rmd
    Untracked:  analysis/2.2.1_snp_qc_7.batches_myo_mis.Rmd
    Untracked:  analysis/2.4_preparing.data_aqc_7.batches_myo.Rmd
    Untracked:  analysis/2.4_preparing.data_aqc_7.batches_myo_mis.Rmd
    Untracked:  analysis/3.1_phenotype.qc_corrected_7.batches_myo.Rmd
    Untracked:  analysis/3.1_phenotype.qc_corrected_7.batches_myo_mis.Rmd
    Untracked:  analysis/4.1.1_qtl.analysis_binary_het-ici-myo-yes.vs.het-ici-myo-no_snpsqc_7.batches_myo.Rmd
    Untracked:  analysis/4.1.1_qtl.analysis_binary_het-ici-myo-yes.vs.het-ici-myo-no_snpsqc_7.batches_myo_mis.Rmd
    Untracked:  analysis/4.1.1_qtl.analysis_binary_het-ici-myo-yes.vs.het-ici-myo-no_snpsqc_dis_no-x_updated_7.batches_myo.Rmd
    Untracked:  analysis/4.1.1_qtl.analysis_binary_het-ici-myo-yes.vs.het-ici-myo-no_snpsqc_dis_no-x_updated_7.batches_myo_mis.Rmd
    Untracked:  analysis/4.1.1_qtl.analysis_binary_het-ici-myo-yes.vs.het-ici-myo-no_snpsqc_dis_no-x_updated_7.batches_myo_vo.Rmd
    Untracked:  analysis/4.1.1_qtl.analysis_binary_het-ici-myo-yes.vs.het-ici-myo-no_snpsqc_dis_no-x_updated_no-sex_7.batches_myo.Rmd
    Untracked:  analysis/4.1.1_qtl.analysis_binary_het-ici-myo-yes.vs.het-ici-myo-no_snpsqc_dis_no-x_updated_no-sex_7.batches_myo_mis.Rmd
    Untracked:  analysis/4.1.1_qtl.analysis_binary_het-ici-myo-yes.vs.het-ici-myo-no_snpsqc_dis_no-x_updated_no-sex_7.batches_myo_vo.Rmd
    Untracked:  analysis/4.1.1_qtl.analysis_binary_het-ici-myo-yes.vs.het-ici-myo-no_snpsqc_no-sex_7.batches_myo.Rmd
    Untracked:  analysis/4.1.1_qtl.analysis_binary_het-ici-myo-yes.vs.het-ici-myo-no_snpsqc_no-sex_7.batches_myo_mis.Rmd
    Untracked:  analysis/4.1.1_qtl.analysis_binary_het-ici.vs.het-pbs_snpsqc_7.batches_myo.Rmd
    Untracked:  analysis/4.1.1_qtl.analysis_binary_het-ici.vs.het-pbs_snpsqc_7.batches_myo_mis.Rmd
    Untracked:  analysis/4.1.1_qtl.analysis_binary_het-ici.vs.het-pbs_snpsqc_dis_no-x_updated_7.batches_myo.Rmd
    Untracked:  analysis/4.1.1_qtl.analysis_binary_het-ici.vs.het-pbs_snpsqc_dis_no-x_updated_7.batches_myo_mis.Rmd
    Untracked:  analysis/4.1.1_qtl.analysis_binary_het-ici.vs.het-pbs_snpsqc_dis_no-x_updated_no-sex_7.batches_myo.Rmd
    Untracked:  analysis/4.1.1_qtl.analysis_binary_het-ici.vs.het-pbs_snpsqc_dis_no-x_updated_no-sex_7.batches_myo_mis.Rmd
    Untracked:  analysis/4.1.1_qtl.analysis_binary_het-ici.vs.het-pbs_snpsqc_no-sex_7.batches_myo.Rmd
    Untracked:  analysis/4.1.1_qtl.analysis_binary_het-ici.vs.het-pbs_snpsqc_no-sex_7.batches_myo_mis.Rmd
    Untracked:  analysis/4.1.1_qtl.analysis_binary_ici-myo-yes.vs.ici-myo-no_snpsqc_7.batches_myo.Rmd
    Untracked:  analysis/4.1.1_qtl.analysis_binary_ici-myo-yes.vs.ici-myo-no_snpsqc_7.batches_myo_mis.Rmd
    Untracked:  analysis/4.1.1_qtl.analysis_binary_ici-myo-yes.vs.ici-myo-no_snpsqc_dis_no-x_updated_7.batches_myo.Rmd
    Untracked:  analysis/4.1.1_qtl.analysis_binary_ici-myo-yes.vs.ici-myo-no_snpsqc_dis_no-x_updated_7.batches_myo_mis.Rmd
    Untracked:  analysis/4.1.1_qtl.analysis_binary_ici-myo-yes.vs.ici-myo-no_snpsqc_dis_no-x_updated_no-sex_7.batches_myo.Rmd
    Untracked:  analysis/4.1.1_qtl.analysis_binary_ici-myo-yes.vs.ici-myo-no_snpsqc_dis_no-x_updated_no-sex_7.batches_myo_mis.Rmd
    Untracked:  analysis/4.1.1_qtl.analysis_binary_ici-myo-yes.vs.ici-myo-no_snpsqc_no-sex_7.batches_myo.Rmd
    Untracked:  analysis/4.1.1_qtl.analysis_binary_ici-myo-yes.vs.ici-myo-no_snpsqc_no-sex_7.batches_myo_mis.Rmd
    Untracked:  analysis/4.1.1_qtl.analysis_binary_ici-sick.vs.ici-eoi_snpsqc_7.batches_myo.Rmd
    Untracked:  analysis/4.1.1_qtl.analysis_binary_ici-sick.vs.ici-eoi_snpsqc_7.batches_myo_mis.Rmd
    Untracked:  analysis/4.1.1_qtl.analysis_binary_ici-sick.vs.ici-eoi_snpsqc_dis_no-x_updated_7.batches_myo.Rmd
    Untracked:  analysis/4.1.1_qtl.analysis_binary_ici-sick.vs.ici-eoi_snpsqc_dis_no-x_updated_7.batches_myo_mis.Rmd
    Untracked:  analysis/4.1.1_qtl.analysis_binary_ici-sick.vs.ici-eoi_snpsqc_dis_no-x_updated_no-sex_7.batches_myo.Rmd
    Untracked:  analysis/4.1.1_qtl.analysis_binary_ici-sick.vs.ici-eoi_snpsqc_dis_no-x_updated_no-sex_7.batches_myo_mis.Rmd
    Untracked:  analysis/4.1.1_qtl.analysis_binary_ici-sick.vs.ici-eoi_snpsqc_no-sex_7.batches_myo.Rmd
    Untracked:  analysis/4.1.1_qtl.analysis_binary_ici-sick.vs.ici-eoi_snpsqc_no-sex_7.batches_myo_mis.Rmd
    Untracked:  analysis/4.1.1_qtl.analysis_binary_ici.vs.pbs_snpsqc_7.batches_myo.Rmd
    Untracked:  analysis/4.1.1_qtl.analysis_binary_ici.vs.pbs_snpsqc_7.batches_myo_mis.Rmd
    Untracked:  analysis/4.1.1_qtl.analysis_binary_ici.vs.pbs_snpsqc_dis_no-x_updated_7.batches_myo.Rmd
    Untracked:  analysis/4.1.1_qtl.analysis_binary_ici.vs.pbs_snpsqc_dis_no-x_updated_7.batches_myo_mis.Rmd
    Untracked:  analysis/4.1.1_qtl.analysis_binary_ici.vs.pbs_snpsqc_dis_no-x_updated_no-sex_7.batches_myo.Rmd
    Untracked:  analysis/4.1.1_qtl.analysis_binary_ici.vs.pbs_snpsqc_dis_no-x_updated_no-sex_7.batches_myo_mis.Rmd
    Untracked:  analysis/4.1.1_qtl.analysis_binary_ici.vs.pbs_snpsqc_no-sex_7.batches_myo.Rmd
    Untracked:  analysis/4.1.1_qtl.analysis_binary_ici.vs.pbs_snpsqc_no-sex_7.batches_myo_mis.Rmd
    Untracked:  analysis/4.1.1_qtl.analysis_binary_myo-yes.vs.myo-no_snpsqc_7.batches_myo.Rmd
    Untracked:  analysis/4.1.1_qtl.analysis_binary_myo-yes.vs.myo-no_snpsqc_7.batches_myo_mis.Rmd
    Untracked:  analysis/4.1.1_qtl.analysis_binary_myo-yes.vs.myo-no_snpsqc_dis_no-x_updated_7.batches_myo.Rmd
    Untracked:  analysis/4.1.1_qtl.analysis_binary_myo-yes.vs.myo-no_snpsqc_dis_no-x_updated_7.batches_myo_mis.Rmd
    Untracked:  analysis/4.1.1_qtl.analysis_binary_myo-yes.vs.myo-no_snpsqc_dis_no-x_updated_no-sex_7.batches_myo.Rmd
    Untracked:  analysis/4.1.1_qtl.analysis_binary_myo-yes.vs.myo-no_snpsqc_dis_no-x_updated_no-sex_7.batches_myo_mis.Rmd
    Untracked:  analysis/4.1.1_qtl.analysis_binary_myo-yes.vs.myo-no_snpsqc_no-sex_7.batches_myo.Rmd
    Untracked:  analysis/4.1.1_qtl.analysis_binary_myo-yes.vs.myo-no_snpsqc_no-sex_7.batches_myo_mis.Rmd
    Untracked:  analysis/4.1.1_qtl.analysis_binary_pbs-myo-yes.vs.pbs-myo-no_snpsqc_7.batches_myo.Rmd
    Untracked:  analysis/4.1.1_qtl.analysis_binary_pbs-myo-yes.vs.pbs-myo-no_snpsqc_7.batches_myo_mis.Rmd
    Untracked:  analysis/4.1.1_qtl.analysis_binary_pbs-myo-yes.vs.pbs-myo-no_snpsqc_dis_no-x_updated_7.batches_myo.Rmd
    Untracked:  analysis/4.1.1_qtl.analysis_binary_pbs-myo-yes.vs.pbs-myo-no_snpsqc_dis_no-x_updated_7.batches_myo_mis.Rmd
    Untracked:  analysis/4.1.1_qtl.analysis_binary_pbs-myo-yes.vs.pbs-myo-no_snpsqc_dis_no-x_updated_no-sex_7.batches_myo.Rmd
    Untracked:  analysis/4.1.1_qtl.analysis_binary_pbs-myo-yes.vs.pbs-myo-no_snpsqc_dis_no-x_updated_no-sex_7.batches_myo_mis.Rmd
    Untracked:  analysis/4.1.1_qtl.analysis_binary_pbs-myo-yes.vs.pbs-myo-no_snpsqc_no-sex_7.batches_myo.Rmd
    Untracked:  analysis/4.1.1_qtl.analysis_binary_pbs-myo-yes.vs.pbs-myo-no_snpsqc_no-sex_7.batches_myo_mis.Rmd
    Untracked:  analysis/genotype.frequencies_het-ici-myo-yes.vs.het-ici-myo-no_7.batches_myo.Rmd
    Untracked:  analysis/genotype.frequencies_het-ici-myo-yes.vs.het-ici-myo-no_7.batches_myo_mis.Rmd
    Untracked:  analysis/genotype.frequencies_het-ici.vs.het-pbs_7.batches_myo.Rmd
    Untracked:  analysis/genotype.frequencies_het-ici.vs.het-pbs_7.batches_myo_mis.Rmd
    Untracked:  analysis/genotype.frequencies_ici-myo-yes.vs.ici-myo-no_7.batches_myo.Rmd
    Untracked:  analysis/genotype.frequencies_ici-myo-yes.vs.ici-myo-no_7.batches_myo_mis.Rmd
    Untracked:  analysis/genotype.frequencies_ici-sick.vs.ici-eoi_7.batches_myo.Rmd
    Untracked:  analysis/genotype.frequencies_ici-sick.vs.ici-eoi_7.batches_myo_mis.Rmd
    Untracked:  analysis/genotype.frequencies_ici.vs.pbs_7.batches_myo.Rmd
    Untracked:  analysis/genotype.frequencies_ici.vs.pbs_7.batches_myo_mis.Rmd
    Untracked:  analysis/genotype.frequencies_myo-yes.vs.myo-no_7.batches_myo.Rmd
    Untracked:  analysis/genotype.frequencies_myo-yes.vs.myo-no_7.batches_myo_mis.Rmd
    Untracked:  analysis/genotype.frequencies_pbs-myo-yes.vs.pbs-myo-no_7.batches_myo.Rmd
    Untracked:  analysis/genotype.frequencies_pbs-myo-yes.vs.pbs-myo-no_7.batches_myo_mis.Rmd
    Untracked:  analysis/index_7.batches_myo.Rmd
    Untracked:  analysis/power.analysis_het-ici-myo-yes.vs.het-ici-myo-no_7.batches_myo.Rmd
    Untracked:  analysis/power.analysis_het-ici-myo-yes.vs.het-ici-myo-no_7.batches_myo_mis.Rmd
    Untracked:  analysis/power.analysis_het-ici.vs.het-pbs_7.batches_myo.Rmd
    Untracked:  analysis/power.analysis_het-ici.vs.het-pbs_7.batches_myo_mis.Rmd
    Untracked:  analysis/power.analysis_ici-myo-yes.vs.ici-myo-no_7.batches_myo.Rmd
    Untracked:  analysis/power.analysis_ici-myo-yes.vs.ici-myo-no_7.batches_myo_mis.Rmd
    Untracked:  analysis/power.analysis_ici-sick.vs.ici-eoi_7.batches_myo.Rmd
    Untracked:  analysis/power.analysis_ici-sick.vs.ici-eoi_7.batches_myo_mis.Rmd
    Untracked:  analysis/power.analysis_ici.vs.pbs_7.batches_myo.Rmd
    Untracked:  analysis/power.analysis_ici.vs.pbs_7.batches_myo_mis.Rmd
    Untracked:  analysis/power.analysis_myo-yes.vs.myo-no_7.batches_myo.Rmd
    Untracked:  analysis/power.analysis_myo-yes.vs.myo-no_7.batches_myo_mis.Rmd
    Untracked:  analysis/power.analysis_pbs-myo-yes.vs.pbs-myo-no_7.batches_myo.Rmd
    Untracked:  analysis/power.analysis_pbs-myo-yes.vs.pbs-myo-no_7.batches_myo_mis.Rmd
    Untracked:  code/cc_variants.sqlite
    Untracked:  code/mouse_genes.sqlite
    Untracked:  code/mouse_genes_mgi.sqlite
    Untracked:  data/GM_covar_7.batches_myo.csv
    Untracked:  data/bad_markers_all_7.batches_myo.RData
    Untracked:  data/covar_corrected.cleaned_het-ici-myo-yes.vs.het-ici-myo-no_7.batches_myo.csv
    Untracked:  data/covar_corrected.cleaned_het-ici-myo-yes.vs.het-ici-myo-no_7.batches_myo_mis.csv
    Untracked:  data/covar_corrected.cleaned_het-ici.vs.het-pbs_7.batches_myo.csv
    Untracked:  data/covar_corrected.cleaned_het-ici.vs.het-pbs_7.batches_myo_mis.csv
    Untracked:  data/covar_corrected.cleaned_ici-myo-yes.vs.ici-myo-no_7.batches_myo.csv
    Untracked:  data/covar_corrected.cleaned_ici-myo-yes.vs.ici-myo-no_7.batches_myo_mis.csv
    Untracked:  data/covar_corrected.cleaned_ici-sick.vs.ici-eoi_7.batches_myo.csv
    Untracked:  data/covar_corrected.cleaned_ici-sick.vs.ici-eoi_7.batches_myo_mis.csv
    Untracked:  data/covar_corrected.cleaned_ici.vs.pbs_7.batches_myo.csv
    Untracked:  data/covar_corrected.cleaned_ici.vs.pbs_7.batches_myo_mis.csv
    Untracked:  data/covar_corrected.cleaned_myo-yes.vs.myo-no_7.batches_myo.csv
    Untracked:  data/covar_corrected.cleaned_myo-yes.vs.myo-no_7.batches_myo_mis.csv
    Untracked:  data/covar_corrected.cleaned_pbs-myo-yes.vs.pbs-myo-no_7.batches_myo.csv
    Untracked:  data/covar_corrected.cleaned_pbs-myo-yes.vs.pbs-myo-no_7.batches_myo_mis.csv
    Untracked:  data/covar_corrected_het-ici-myo-yes.vs.het-ici-myo-no_7.batches_myo.csv
    Untracked:  data/covar_corrected_het-ici-myo-yes.vs.het-ici-myo-no_7.batches_myo_mis.csv
    Untracked:  data/covar_corrected_het-ici.vs.het-pbs_7.batches_myo.csv
    Untracked:  data/covar_corrected_het-ici.vs.het-pbs_7.batches_myo_mis.csv
    Untracked:  data/covar_corrected_ici-myo-yes.vs.ici-myo-no_7.batches_myo.csv
    Untracked:  data/covar_corrected_ici-myo-yes.vs.ici-myo-no_7.batches_myo_mis.csv
    Untracked:  data/covar_corrected_ici-sick.vs.ici-eoi_7.batches_myo.csv
    Untracked:  data/covar_corrected_ici-sick.vs.ici-eoi_7.batches_myo_mis.csv
    Untracked:  data/covar_corrected_ici.vs.pbs_7.batches_myo.csv
    Untracked:  data/covar_corrected_ici.vs.pbs_7.batches_myo_mis.csv
    Untracked:  data/covar_corrected_myo-yes.vs.myo-no_7.batches_myo.csv
    Untracked:  data/covar_corrected_myo-yes.vs.myo-no_7.batches_myo_mis.csv
    Untracked:  data/covar_corrected_pbs-myo-yes.vs.pbs-myo-no_7.batches_myo.csv
    Untracked:  data/covar_corrected_pbs-myo-yes.vs.pbs-myo-no_7.batches_myo_mis.csv
    Untracked:  data/e_7.batches_myo.RData
    Untracked:  data/e_snpg_samqc_7.batches_myo.RData
    Untracked:  data/errors_ind_7.batches_myo.RData
    Untracked:  data/genetic_map_7.batches_myo.csv
    Untracked:  data/genotype_errors_marker_7.batches_myo.RData
    Untracked:  data/genotype_freq_marker_7.batches_myo.RData
    Untracked:  data/gm_allqc_7.batches_myo.RData
    Untracked:  data/gm_allqc_7.batches_myo_mis.RData
    Untracked:  data/gm_samqc_7.batches_myo.RData
    Untracked:  data/gm_serreze_7.batches_myo.RData
    Untracked:  data/gm_serreze_bc_7.batches_myo.RData
    Untracked:  data/het-ici-myo-yes.vs.het-ici-myo-no_gm_qtl_snpsqc_7.batches_myo_mis.csv
    Untracked:  data/het-ici-myo-yes.vs.het-ici-myo-no_gm_qtl_snpsqc_dis_no-x_updated_7.batches_myo_mis.csv
    Untracked:  data/het-ici-myo-yes.vs.het-ici-myo-no_gm_qtl_snpsqc_dis_no-x_updated_no-sex_7.batches_myo_mis.csv
    Untracked:  data/het-ici-myo-yes.vs.het-ici-myo-no_marker.freq_low.geno.freq.removed_geno.ratio_7.batches_myo.csv
    Untracked:  data/het-ici-myo-yes.vs.het-ici-myo-no_marker.freq_low.geno.freq.removed_geno.ratio_7.batches_myo_mis.csv
    Untracked:  data/het-ici-myo-yes.vs.het-ici-myo-no_marker.freq_low.geno.freq.removed_sample.outliers.removed_geno.ratio_7.batches_myo.csv
    Untracked:  data/het-ici-myo-yes.vs.het-ici-myo-no_marker.freq_low.geno.freq.removed_sample.outliers.removed_geno.ratio_7.batches_myo_mis.csv
    Untracked:  data/het-ici-myo-yes.vs.het-ici-myo-no_marker.freq_low.probs.freq.removed_geno.ratio_7.batches_myo.csv
    Untracked:  data/het-ici-myo-yes.vs.het-ici-myo-no_marker.freq_low.probs.freq.removed_geno.ratio_7.batches_myo_mis.csv
    Untracked:  data/het-ici-myo-yes.vs.het-ici-myo-no_marker.freq_low.probs.freq.removed_sample.outliers.removed_geno.ratio_7.batches_myo.csv
    Untracked:  data/het-ici-myo-yes.vs.het-ici-myo-no_marker.freq_low.probs.freq.removed_sample.outliers.removed_geno.ratio_7.batches_myo_mis.csv
    Untracked:  data/het-ici-myo-yes.vs.het-ici-myo-no_sample.genos_marker.freq_low.geno.freq.removed_7.batches_myo.csv
    Untracked:  data/het-ici-myo-yes.vs.het-ici-myo-no_sample.genos_marker.freq_low.geno.freq.removed_7.batches_myo_mis.csv
    Untracked:  data/het-ici-myo-yes.vs.het-ici-myo-no_sample.genos_marker.freq_low.geno.freq.removed_sample.outliers.removed_7.batches_myo.csv
    Untracked:  data/het-ici-myo-yes.vs.het-ici-myo-no_sample.genos_marker.freq_low.geno.freq.removed_sample.outliers.removed_7.batches_myo_mis.csv
    Untracked:  data/het-ici-myo-yes.vs.het-ici-myo-no_sample.genos_marker.freq_low.probs.freq.removed_7.batches_myo.csv
    Untracked:  data/het-ici-myo-yes.vs.het-ici-myo-no_sample.genos_marker.freq_low.probs.freq.removed_7.batches_myo_mis.csv
    Untracked:  data/het-ici-myo-yes.vs.het-ici-myo-no_sample.genos_marker.freq_low.probs.freq.removed_sample.outliers.removed_7.batches_myo.csv
    Untracked:  data/het-ici-myo-yes.vs.het-ici-myo-no_sample.genos_marker.freq_low.probs.freq.removed_sample.outliers.removed_7.batches_myo_mis.csv
    Untracked:  data/het-ici.vs.het-pbs_gm_qtl_snpsqc_7.batches_myo.csv
    Untracked:  data/het-ici.vs.het-pbs_gm_qtl_snpsqc_7.batches_myo_mis.csv
    Untracked:  data/het-ici.vs.het-pbs_gm_qtl_snpsqc_dis_no-x_updated_7.batches_myo.csv
    Untracked:  data/het-ici.vs.het-pbs_gm_qtl_snpsqc_dis_no-x_updated_7.batches_myo_mis.csv
    Untracked:  data/het-ici.vs.het-pbs_gm_qtl_snpsqc_dis_no-x_updated_no-sex_7.batches_myo.csv
    Untracked:  data/het-ici.vs.het-pbs_gm_qtl_snpsqc_dis_no-x_updated_no-sex_7.batches_myo_mis.csv
    Untracked:  data/het-ici.vs.het-pbs_marker.freq_low.geno.freq.removed_geno.ratio_7.batches_myo.csv
    Untracked:  data/het-ici.vs.het-pbs_marker.freq_low.geno.freq.removed_geno.ratio_7.batches_myo_mis.csv
    Untracked:  data/het-ici.vs.het-pbs_marker.freq_low.geno.freq.removed_sample.outliers.removed_geno.ratio_7.batches_myo.csv
    Untracked:  data/het-ici.vs.het-pbs_marker.freq_low.geno.freq.removed_sample.outliers.removed_geno.ratio_7.batches_myo_mis.csv
    Untracked:  data/het-ici.vs.het-pbs_marker.freq_low.probs.freq.removed_geno.ratio_7.batches_myo.csv
    Untracked:  data/het-ici.vs.het-pbs_marker.freq_low.probs.freq.removed_geno.ratio_7.batches_myo_mis.csv
    Untracked:  data/het-ici.vs.het-pbs_marker.freq_low.probs.freq.removed_sample.outliers.removed_geno.ratio_7.batches_myo.csv
    Untracked:  data/het-ici.vs.het-pbs_marker.freq_low.probs.freq.removed_sample.outliers.removed_geno.ratio_7.batches_myo_mis.csv
    Untracked:  data/het-ici.vs.het-pbs_sample.genos_marker.freq_low.geno.freq.removed_7.batches_myo.csv
    Untracked:  data/het-ici.vs.het-pbs_sample.genos_marker.freq_low.geno.freq.removed_7.batches_myo_mis.csv
    Untracked:  data/het-ici.vs.het-pbs_sample.genos_marker.freq_low.geno.freq.removed_sample.outliers.removed_7.batches_myo.csv
    Untracked:  data/het-ici.vs.het-pbs_sample.genos_marker.freq_low.geno.freq.removed_sample.outliers.removed_7.batches_myo_mis.csv
    Untracked:  data/het-ici.vs.het-pbs_sample.genos_marker.freq_low.probs.freq.removed_7.batches_myo.csv
    Untracked:  data/het-ici.vs.het-pbs_sample.genos_marker.freq_low.probs.freq.removed_7.batches_myo_mis.csv
    Untracked:  data/het-ici.vs.het-pbs_sample.genos_marker.freq_low.probs.freq.removed_sample.outliers.removed_7.batches_myo.csv
    Untracked:  data/het-ici.vs.het-pbs_sample.genos_marker.freq_low.probs.freq.removed_sample.outliers.removed_7.batches_myo_mis.csv
    Untracked:  data/ici-myo-yes.vs.ici-myo-no_gm_qtl_snpsqc_7.batches_myo.csv
    Untracked:  data/ici-myo-yes.vs.ici-myo-no_gm_qtl_snpsqc_7.batches_myo_mis.csv
    Untracked:  data/ici-myo-yes.vs.ici-myo-no_gm_qtl_snpsqc_dis_no-x_updated_7.batches_myo.csv
    Untracked:  data/ici-myo-yes.vs.ici-myo-no_gm_qtl_snpsqc_dis_no-x_updated_7.batches_myo_mis.csv
    Untracked:  data/ici-myo-yes.vs.ici-myo-no_gm_qtl_snpsqc_dis_no-x_updated_no-sex_7.batches_myo.csv
    Untracked:  data/ici-myo-yes.vs.ici-myo-no_gm_qtl_snpsqc_dis_no-x_updated_no-sex_7.batches_myo_mis.csv
    Untracked:  data/ici-myo-yes.vs.ici-myo-no_marker.freq_low.geno.freq.removed_geno.ratio_7.batches_myo.csv
    Untracked:  data/ici-myo-yes.vs.ici-myo-no_marker.freq_low.geno.freq.removed_geno.ratio_7.batches_myo_mis.csv
    Untracked:  data/ici-myo-yes.vs.ici-myo-no_marker.freq_low.geno.freq.removed_sample.outliers.removed_geno.ratio_7.batches_myo.csv
    Untracked:  data/ici-myo-yes.vs.ici-myo-no_marker.freq_low.geno.freq.removed_sample.outliers.removed_geno.ratio_7.batches_myo_mis.csv
    Untracked:  data/ici-myo-yes.vs.ici-myo-no_marker.freq_low.probs.freq.removed_geno.ratio_7.batches_myo.csv
    Untracked:  data/ici-myo-yes.vs.ici-myo-no_marker.freq_low.probs.freq.removed_geno.ratio_7.batches_myo_mis.csv
    Untracked:  data/ici-myo-yes.vs.ici-myo-no_marker.freq_low.probs.freq.removed_sample.outliers.removed_geno.ratio_7.batches_myo.csv
    Untracked:  data/ici-myo-yes.vs.ici-myo-no_marker.freq_low.probs.freq.removed_sample.outliers.removed_geno.ratio_7.batches_myo_mis.csv
    Untracked:  data/ici-myo-yes.vs.ici-myo-no_sample.genos_marker.freq_low.geno.freq.removed_7.batches_myo.csv
    Untracked:  data/ici-myo-yes.vs.ici-myo-no_sample.genos_marker.freq_low.geno.freq.removed_7.batches_myo_mis.csv
    Untracked:  data/ici-myo-yes.vs.ici-myo-no_sample.genos_marker.freq_low.geno.freq.removed_sample.outliers.removed_7.batches_myo.csv
    Untracked:  data/ici-myo-yes.vs.ici-myo-no_sample.genos_marker.freq_low.geno.freq.removed_sample.outliers.removed_7.batches_myo_mis.csv
    Untracked:  data/ici-myo-yes.vs.ici-myo-no_sample.genos_marker.freq_low.probs.freq.removed_7.batches_myo.csv
    Untracked:  data/ici-myo-yes.vs.ici-myo-no_sample.genos_marker.freq_low.probs.freq.removed_7.batches_myo_mis.csv
    Untracked:  data/ici-myo-yes.vs.ici-myo-no_sample.genos_marker.freq_low.probs.freq.removed_sample.outliers.removed_7.batches_myo.csv
    Untracked:  data/ici-myo-yes.vs.ici-myo-no_sample.genos_marker.freq_low.probs.freq.removed_sample.outliers.removed_7.batches_myo_mis.csv
    Untracked:  data/ici-sick.vs.ici-eoi_blup_sub_chr-10_peak.marker-UNC18343990_lod.drop-1.5_snpsqc_dis_no-x_updated_7.batches_myo.csv
    Untracked:  data/ici-sick.vs.ici-eoi_blup_sub_chr-10_peak.marker-UNC18343990_lod.drop-1.5_snpsqc_dis_no-x_updated_7.batches_myo_mis.csv
    Untracked:  data/ici-sick.vs.ici-eoi_blup_sub_chr-10_peak.marker-UNC18343990_lod.drop-1.5_snpsqc_dis_no-x_updated_no-sex_7.batches_myo.csv
    Untracked:  data/ici-sick.vs.ici-eoi_blup_sub_chr-10_peak.marker-UNC18343990_lod.drop-1.5_snpsqc_dis_no-x_updated_no-sex_7.batches_myo_mis.csv
    Untracked:  data/ici-sick.vs.ici-eoi_blup_sub_chr-11_peak.marker-UNC20070077_lod.drop-1.5_snpsqc_dis_no-x_updated_7.batches_myo.csv
    Untracked:  data/ici-sick.vs.ici-eoi_blup_sub_chr-11_peak.marker-UNC20070077_lod.drop-1.5_snpsqc_dis_no-x_updated_7.batches_myo_mis.csv
    Untracked:  data/ici-sick.vs.ici-eoi_blup_sub_chr-11_peak.marker-UNC20070077_lod.drop-1.5_snpsqc_dis_no-x_updated_no-sex_7.batches_myo.csv
    Untracked:  data/ici-sick.vs.ici-eoi_blup_sub_chr-11_peak.marker-UNC20070077_lod.drop-1.5_snpsqc_dis_no-x_updated_no-sex_7.batches_myo_mis.csv
    Untracked:  data/ici-sick.vs.ici-eoi_blup_sub_chr-12_peak.marker-UNC21652584_lod.drop-1.5_snpsqc_dis_no-x_updated_7.batches_myo.csv
    Untracked:  data/ici-sick.vs.ici-eoi_blup_sub_chr-12_peak.marker-UNC21652584_lod.drop-1.5_snpsqc_dis_no-x_updated_7.batches_myo_mis.csv
    Untracked:  data/ici-sick.vs.ici-eoi_blup_sub_chr-12_peak.marker-UNC21652584_lod.drop-1.5_snpsqc_dis_no-x_updated_no-sex_7.batches_myo.csv
    Untracked:  data/ici-sick.vs.ici-eoi_blup_sub_chr-12_peak.marker-UNC21652584_lod.drop-1.5_snpsqc_dis_no-x_updated_no-sex_7.batches_myo_mis.csv
    Untracked:  data/ici-sick.vs.ici-eoi_blup_sub_chr-13_peak.marker-UNCHS036579_lod.drop-1.5_snpsqc_dis_no-x_updated_7.batches_myo.csv
    Untracked:  data/ici-sick.vs.ici-eoi_blup_sub_chr-13_peak.marker-UNCHS036579_lod.drop-1.5_snpsqc_dis_no-x_updated_7.batches_myo_mis.csv
    Untracked:  data/ici-sick.vs.ici-eoi_blup_sub_chr-13_peak.marker-UNCHS036579_lod.drop-1.5_snpsqc_dis_no-x_updated_no-sex_7.batches_myo.csv
    Untracked:  data/ici-sick.vs.ici-eoi_blup_sub_chr-13_peak.marker-UNCHS036579_lod.drop-1.5_snpsqc_dis_no-x_updated_no-sex_7.batches_myo_mis.csv
    Untracked:  data/ici-sick.vs.ici-eoi_blup_sub_chr-14_peak.marker-UNCHS037782_lod.drop-1.5_snpsqc_dis_no-x_updated_7.batches_myo.csv
    Untracked:  data/ici-sick.vs.ici-eoi_blup_sub_chr-14_peak.marker-UNCHS037782_lod.drop-1.5_snpsqc_dis_no-x_updated_7.batches_myo_mis.csv
    Untracked:  data/ici-sick.vs.ici-eoi_blup_sub_chr-14_peak.marker-UNCHS037782_lod.drop-1.5_snpsqc_dis_no-x_updated_no-sex_7.batches_myo.csv
    Untracked:  data/ici-sick.vs.ici-eoi_blup_sub_chr-14_peak.marker-UNCHS037782_lod.drop-1.5_snpsqc_dis_no-x_updated_no-sex_7.batches_myo_mis.csv
    Untracked:  data/ici-sick.vs.ici-eoi_blup_sub_chr-15_peak.marker-UNC26069905_lod.drop-1.5_snpsqc_dis_no-x_updated_7.batches_myo.csv
    Untracked:  data/ici-sick.vs.ici-eoi_blup_sub_chr-15_peak.marker-UNC26069905_lod.drop-1.5_snpsqc_dis_no-x_updated_no-sex_7.batches_myo.csv
    Untracked:  data/ici-sick.vs.ici-eoi_blup_sub_chr-15_peak.marker-UNCHS041223_lod.drop-1.5_snpsqc_dis_no-x_updated_7.batches_myo_mis.csv
    Untracked:  data/ici-sick.vs.ici-eoi_blup_sub_chr-15_peak.marker-UNCHS041223_lod.drop-1.5_snpsqc_dis_no-x_updated_no-sex_7.batches_myo_mis.csv
    Untracked:  data/ici-sick.vs.ici-eoi_blup_sub_chr-16_peak.marker-JAX00070117_lod.drop-1.5_snpsqc_dis_no-x_updated_7.batches_myo.csv
    Untracked:  data/ici-sick.vs.ici-eoi_blup_sub_chr-16_peak.marker-JAX00070117_lod.drop-1.5_snpsqc_dis_no-x_updated_7.batches_myo_mis.csv
    Untracked:  data/ici-sick.vs.ici-eoi_blup_sub_chr-16_peak.marker-JAX00070117_lod.drop-1.5_snpsqc_dis_no-x_updated_no-sex_7.batches_myo.csv
    Untracked:  data/ici-sick.vs.ici-eoi_blup_sub_chr-16_peak.marker-JAX00070117_lod.drop-1.5_snpsqc_dis_no-x_updated_no-sex_7.batches_myo_mis.csv
    Untracked:  data/ici-sick.vs.ici-eoi_blup_sub_chr-17_peak.marker-UNC28542319_lod.drop-1.5_snpsqc_dis_no-x_updated_7.batches_myo_mis.csv
    Untracked:  data/ici-sick.vs.ici-eoi_blup_sub_chr-17_peak.marker-UNC28542319_lod.drop-1.5_snpsqc_dis_no-x_updated_no-sex_7.batches_myo_mis.csv
    Untracked:  data/ici-sick.vs.ici-eoi_blup_sub_chr-17_peak.marker-UNCJPD006670_lod.drop-1.5_snpsqc_dis_no-x_updated_7.batches_myo.csv
    Untracked:  data/ici-sick.vs.ici-eoi_blup_sub_chr-17_peak.marker-UNCJPD006670_lod.drop-1.5_snpsqc_dis_no-x_updated_no-sex_7.batches_myo.csv
    Untracked:  data/ici-sick.vs.ici-eoi_blup_sub_chr-18_peak.marker-UNC28776739_lod.drop-1.5_snpsqc_dis_no-x_updated_7.batches_myo.csv
    Untracked:  data/ici-sick.vs.ici-eoi_blup_sub_chr-18_peak.marker-UNC28776739_lod.drop-1.5_snpsqc_dis_no-x_updated_no-sex_7.batches_myo.csv
    Untracked:  data/ici-sick.vs.ici-eoi_blup_sub_chr-18_peak.marker-UNCHS045594_lod.drop-1.5_snpsqc_dis_no-x_updated_7.batches_myo_mis.csv
    Untracked:  data/ici-sick.vs.ici-eoi_blup_sub_chr-18_peak.marker-UNCHS045594_lod.drop-1.5_snpsqc_dis_no-x_updated_no-sex_7.batches_myo_mis.csv
    Untracked:  data/ici-sick.vs.ici-eoi_blup_sub_chr-19_peak.marker-UNC30414168_lod.drop-1.5_snpsqc_dis_no-x_updated_7.batches_myo.csv
    Untracked:  data/ici-sick.vs.ici-eoi_blup_sub_chr-19_peak.marker-UNC30414168_lod.drop-1.5_snpsqc_dis_no-x_updated_no-sex_7.batches_myo.csv
    Untracked:  data/ici-sick.vs.ici-eoi_blup_sub_chr-19_peak.marker-UNC30426276_lod.drop-1.5_snpsqc_dis_no-x_updated_7.batches_myo_mis.csv
    Untracked:  data/ici-sick.vs.ici-eoi_blup_sub_chr-19_peak.marker-UNC30426276_lod.drop-1.5_snpsqc_dis_no-x_updated_no-sex_7.batches_myo_mis.csv
    Untracked:  data/ici-sick.vs.ici-eoi_blup_sub_chr-1_peak.marker-UNC2031646_lod.drop-1.5_snpsqc_dis_no-x_updated_7.batches_myo.csv
    Untracked:  data/ici-sick.vs.ici-eoi_blup_sub_chr-1_peak.marker-UNC2031646_lod.drop-1.5_snpsqc_dis_no-x_updated_no-sex_7.batches_myo.csv
    Untracked:  data/ici-sick.vs.ici-eoi_blup_sub_chr-1_peak.marker-UNCHS003700_lod.drop-1.5_snpsqc_dis_no-x_updated_7.batches_myo_mis.csv
    Untracked:  data/ici-sick.vs.ici-eoi_blup_sub_chr-1_peak.marker-UNCHS003700_lod.drop-1.5_snpsqc_dis_no-x_updated_no-sex_7.batches_myo_mis.csv
    Untracked:  data/ici-sick.vs.ici-eoi_blup_sub_chr-2_peak.marker-ICR5131_lod.drop-1.5_snpsqc_dis_no-x_updated_7.batches_myo.csv
    Untracked:  data/ici-sick.vs.ici-eoi_blup_sub_chr-2_peak.marker-ICR5131_lod.drop-1.5_snpsqc_dis_no-x_updated_no-sex_7.batches_myo.csv
    Untracked:  data/ici-sick.vs.ici-eoi_blup_sub_chr-2_peak.marker-UNCHS006420_lod.drop-1.5_snpsqc_dis_no-x_updated_7.batches_myo_mis.csv
    Untracked:  data/ici-sick.vs.ici-eoi_blup_sub_chr-2_peak.marker-UNCHS006420_lod.drop-1.5_snpsqc_dis_no-x_updated_no-sex_7.batches_myo_mis.csv
    Untracked:  data/ici-sick.vs.ici-eoi_blup_sub_chr-3_peak.marker-UNC5667757_lod.drop-1.5_snpsqc_dis_no-x_updated_7.batches_myo.csv
    Untracked:  data/ici-sick.vs.ici-eoi_blup_sub_chr-3_peak.marker-UNC5667757_lod.drop-1.5_snpsqc_dis_no-x_updated_7.batches_myo_mis.csv
    Untracked:  data/ici-sick.vs.ici-eoi_blup_sub_chr-3_peak.marker-UNC5667757_lod.drop-1.5_snpsqc_dis_no-x_updated_no-sex_7.batches_myo.csv
    Untracked:  data/ici-sick.vs.ici-eoi_blup_sub_chr-3_peak.marker-UNC5667757_lod.drop-1.5_snpsqc_dis_no-x_updated_no-sex_7.batches_myo_mis.csv
    Untracked:  data/ici-sick.vs.ici-eoi_blup_sub_chr-4_peak.marker-UNC6759992_lod.drop-1.5_snpsqc_dis_no-x_updated_7.batches_myo_mis.csv
    Untracked:  data/ici-sick.vs.ici-eoi_blup_sub_chr-4_peak.marker-UNC6759992_lod.drop-1.5_snpsqc_dis_no-x_updated_no-sex_7.batches_myo_mis.csv
    Untracked:  data/ici-sick.vs.ici-eoi_blup_sub_chr-4_peak.marker-UNC6765178_lod.drop-1.5_snpsqc_dis_no-x_updated_7.batches_myo.csv
    Untracked:  data/ici-sick.vs.ici-eoi_blup_sub_chr-4_peak.marker-UNC6765178_lod.drop-1.5_snpsqc_dis_no-x_updated_no-sex_7.batches_myo.csv
    Untracked:  data/ici-sick.vs.ici-eoi_blup_sub_chr-5_peak.marker-UNC9889957_lod.drop-1.5_snpsqc_dis_no-x_updated_7.batches_myo_mis.csv
    Untracked:  data/ici-sick.vs.ici-eoi_blup_sub_chr-5_peak.marker-UNC9889957_lod.drop-1.5_snpsqc_dis_no-x_updated_no-sex_7.batches_myo_mis.csv
    Untracked:  data/ici-sick.vs.ici-eoi_blup_sub_chr-5_peak.marker-UNC9900273_lod.drop-1.5_snpsqc_dis_no-x_updated_7.batches_myo.csv
    Untracked:  data/ici-sick.vs.ici-eoi_blup_sub_chr-5_peak.marker-UNC9900273_lod.drop-1.5_snpsqc_dis_no-x_updated_no-sex_7.batches_myo.csv
    Untracked:  data/ici-sick.vs.ici-eoi_blup_sub_chr-6_peak.marker-UNC10800126_lod.drop-1.5_snpsqc_dis_no-x_updated_7.batches_myo_mis.csv
    Untracked:  data/ici-sick.vs.ici-eoi_blup_sub_chr-6_peak.marker-UNC10800126_lod.drop-1.5_snpsqc_dis_no-x_updated_no-sex_7.batches_myo_mis.csv
    Untracked:  data/ici-sick.vs.ici-eoi_blup_sub_chr-6_peak.marker-UNC10832076_lod.drop-1.5_snpsqc_dis_no-x_updated_7.batches_myo.csv
    Untracked:  data/ici-sick.vs.ici-eoi_blup_sub_chr-6_peak.marker-UNC10832076_lod.drop-1.5_snpsqc_dis_no-x_updated_no-sex_7.batches_myo.csv
    Untracked:  data/ici-sick.vs.ici-eoi_blup_sub_chr-7_peak.marker-UNCHS020903_lod.drop-1.5_snpsqc_dis_no-x_updated_7.batches_myo_mis.csv
    Untracked:  data/ici-sick.vs.ici-eoi_blup_sub_chr-7_peak.marker-UNCHS020903_lod.drop-1.5_snpsqc_dis_no-x_updated_no-sex_7.batches_myo_mis.csv
    Untracked:  data/ici-sick.vs.ici-eoi_blup_sub_chr-7_peak.marker-UNCHS021163_lod.drop-1.5_snpsqc_dis_no-x_updated_7.batches_myo.csv
    Untracked:  data/ici-sick.vs.ici-eoi_blup_sub_chr-7_peak.marker-UNCHS021163_lod.drop-1.5_snpsqc_dis_no-x_updated_no-sex_7.batches_myo.csv
    Untracked:  data/ici-sick.vs.ici-eoi_blup_sub_chr-8_peak.marker-UNC15471847_lod.drop-1.5_snpsqc_dis_no-x_updated_7.batches_myo.csv
    Untracked:  data/ici-sick.vs.ici-eoi_blup_sub_chr-8_peak.marker-UNC15471847_lod.drop-1.5_snpsqc_dis_no-x_updated_no-sex_7.batches_myo.csv
    Untracked:  data/ici-sick.vs.ici-eoi_blup_sub_chr-8_peak.marker-UNC15548888_lod.drop-1.5_snpsqc_dis_no-x_updated_7.batches_myo_mis.csv
    Untracked:  data/ici-sick.vs.ici-eoi_blup_sub_chr-8_peak.marker-UNC15548888_lod.drop-1.5_snpsqc_dis_no-x_updated_no-sex_7.batches_myo_mis.csv
    Untracked:  data/ici-sick.vs.ici-eoi_blup_sub_chr-9_peak.marker-UNC16231874_lod.drop-1.5_snpsqc_dis_no-x_updated_7.batches_myo.csv
    Untracked:  data/ici-sick.vs.ici-eoi_blup_sub_chr-9_peak.marker-UNC16231874_lod.drop-1.5_snpsqc_dis_no-x_updated_7.batches_myo_mis.csv
    Untracked:  data/ici-sick.vs.ici-eoi_blup_sub_chr-9_peak.marker-UNC16231874_lod.drop-1.5_snpsqc_dis_no-x_updated_no-sex_7.batches_myo.csv
    Untracked:  data/ici-sick.vs.ici-eoi_blup_sub_chr-9_peak.marker-UNC16231874_lod.drop-1.5_snpsqc_dis_no-x_updated_no-sex_7.batches_myo_mis.csv
    Untracked:  data/ici-sick.vs.ici-eoi_blup_sub_chr-X_peak.marker-UNCHS049472_lod.drop-1.5_snpsqc_dis_no-x_updated_7.batches_myo.csv
    Untracked:  data/ici-sick.vs.ici-eoi_blup_sub_chr-X_peak.marker-UNCHS049472_lod.drop-1.5_snpsqc_dis_no-x_updated_7.batches_myo_mis.csv
    Untracked:  data/ici-sick.vs.ici-eoi_blup_sub_chr-X_peak.marker-UNCHS049472_lod.drop-1.5_snpsqc_dis_no-x_updated_no-sex_7.batches_myo.csv
    Untracked:  data/ici-sick.vs.ici-eoi_blup_sub_chr-X_peak.marker-UNCHS049472_lod.drop-1.5_snpsqc_dis_no-x_updated_no-sex_7.batches_myo_mis.csv
    Untracked:  data/ici-sick.vs.ici-eoi_genes_chr-10_peak.marker-UNC18343990_lod.drop-1.5_snpsqc_dis_no-x_updated_7.batches_myo.csv
    Untracked:  data/ici-sick.vs.ici-eoi_genes_chr-10_peak.marker-UNC18343990_lod.drop-1.5_snpsqc_dis_no-x_updated_7.batches_myo_mis.csv
    Untracked:  data/ici-sick.vs.ici-eoi_genes_chr-10_peak.marker-UNC18343990_lod.drop-1.5_snpsqc_dis_no-x_updated_no-sex_7.batches_myo.csv
    Untracked:  data/ici-sick.vs.ici-eoi_genes_chr-10_peak.marker-UNC18343990_lod.drop-1.5_snpsqc_dis_no-x_updated_no-sex_7.batches_myo_mis.csv
    Untracked:  data/ici-sick.vs.ici-eoi_genes_chr-11_peak.marker-UNC20070077_lod.drop-1.5_snpsqc_dis_no-x_updated_7.batches_myo.csv
    Untracked:  data/ici-sick.vs.ici-eoi_genes_chr-11_peak.marker-UNC20070077_lod.drop-1.5_snpsqc_dis_no-x_updated_7.batches_myo_mis.csv
    Untracked:  data/ici-sick.vs.ici-eoi_genes_chr-11_peak.marker-UNC20070077_lod.drop-1.5_snpsqc_dis_no-x_updated_no-sex_7.batches_myo.csv
    Untracked:  data/ici-sick.vs.ici-eoi_genes_chr-11_peak.marker-UNC20070077_lod.drop-1.5_snpsqc_dis_no-x_updated_no-sex_7.batches_myo_mis.csv
    Untracked:  data/ici-sick.vs.ici-eoi_genes_chr-12_peak.marker-UNC21652584_lod.drop-1.5_snpsqc_dis_no-x_updated_7.batches_myo.csv
    Untracked:  data/ici-sick.vs.ici-eoi_genes_chr-12_peak.marker-UNC21652584_lod.drop-1.5_snpsqc_dis_no-x_updated_7.batches_myo_mis.csv
    Untracked:  data/ici-sick.vs.ici-eoi_genes_chr-12_peak.marker-UNC21652584_lod.drop-1.5_snpsqc_dis_no-x_updated_no-sex_7.batches_myo.csv
    Untracked:  data/ici-sick.vs.ici-eoi_genes_chr-12_peak.marker-UNC21652584_lod.drop-1.5_snpsqc_dis_no-x_updated_no-sex_7.batches_myo_mis.csv
    Untracked:  data/ici-sick.vs.ici-eoi_genes_chr-13_peak.marker-UNCHS036579_lod.drop-1.5_snpsqc_dis_no-x_updated_7.batches_myo.csv
    Untracked:  data/ici-sick.vs.ici-eoi_genes_chr-13_peak.marker-UNCHS036579_lod.drop-1.5_snpsqc_dis_no-x_updated_7.batches_myo_mis.csv
    Untracked:  data/ici-sick.vs.ici-eoi_genes_chr-13_peak.marker-UNCHS036579_lod.drop-1.5_snpsqc_dis_no-x_updated_no-sex_7.batches_myo.csv
    Untracked:  data/ici-sick.vs.ici-eoi_genes_chr-13_peak.marker-UNCHS036579_lod.drop-1.5_snpsqc_dis_no-x_updated_no-sex_7.batches_myo_mis.csv
    Untracked:  data/ici-sick.vs.ici-eoi_genes_chr-14_peak.marker-UNCHS037782_lod.drop-1.5_snpsqc_dis_no-x_updated_7.batches_myo.csv
    Untracked:  data/ici-sick.vs.ici-eoi_genes_chr-14_peak.marker-UNCHS037782_lod.drop-1.5_snpsqc_dis_no-x_updated_7.batches_myo_mis.csv
    Untracked:  data/ici-sick.vs.ici-eoi_genes_chr-14_peak.marker-UNCHS037782_lod.drop-1.5_snpsqc_dis_no-x_updated_no-sex_7.batches_myo.csv
    Untracked:  data/ici-sick.vs.ici-eoi_genes_chr-14_peak.marker-UNCHS037782_lod.drop-1.5_snpsqc_dis_no-x_updated_no-sex_7.batches_myo_mis.csv
    Untracked:  data/ici-sick.vs.ici-eoi_genes_chr-15_peak.marker-UNC26069905_lod.drop-1.5_snpsqc_dis_no-x_updated_7.batches_myo.csv
    Untracked:  data/ici-sick.vs.ici-eoi_genes_chr-15_peak.marker-UNC26069905_lod.drop-1.5_snpsqc_dis_no-x_updated_no-sex_7.batches_myo.csv
    Untracked:  data/ici-sick.vs.ici-eoi_genes_chr-15_peak.marker-UNCHS041223_lod.drop-1.5_snpsqc_dis_no-x_updated_7.batches_myo_mis.csv
    Untracked:  data/ici-sick.vs.ici-eoi_genes_chr-15_peak.marker-UNCHS041223_lod.drop-1.5_snpsqc_dis_no-x_updated_no-sex_7.batches_myo_mis.csv
    Untracked:  data/ici-sick.vs.ici-eoi_genes_chr-16_peak.marker-JAX00070117_lod.drop-1.5_snpsqc_dis_no-x_updated_7.batches_myo.csv
    Untracked:  data/ici-sick.vs.ici-eoi_genes_chr-16_peak.marker-JAX00070117_lod.drop-1.5_snpsqc_dis_no-x_updated_7.batches_myo_mis.csv
    Untracked:  data/ici-sick.vs.ici-eoi_genes_chr-16_peak.marker-JAX00070117_lod.drop-1.5_snpsqc_dis_no-x_updated_no-sex_7.batches_myo.csv
    Untracked:  data/ici-sick.vs.ici-eoi_genes_chr-16_peak.marker-JAX00070117_lod.drop-1.5_snpsqc_dis_no-x_updated_no-sex_7.batches_myo_mis.csv
    Untracked:  data/ici-sick.vs.ici-eoi_genes_chr-17_peak.marker-UNC28542319_lod.drop-1.5_snpsqc_dis_no-x_updated_7.batches_myo_mis.csv
    Untracked:  data/ici-sick.vs.ici-eoi_genes_chr-17_peak.marker-UNC28542319_lod.drop-1.5_snpsqc_dis_no-x_updated_no-sex_7.batches_myo_mis.csv
    Untracked:  data/ici-sick.vs.ici-eoi_genes_chr-17_peak.marker-UNCJPD006670_lod.drop-1.5_snpsqc_dis_no-x_updated_7.batches_myo.csv
    Untracked:  data/ici-sick.vs.ici-eoi_genes_chr-17_peak.marker-UNCJPD006670_lod.drop-1.5_snpsqc_dis_no-x_updated_no-sex_7.batches_myo.csv
    Untracked:  data/ici-sick.vs.ici-eoi_genes_chr-18_peak.marker-UNC28776739_lod.drop-1.5_snpsqc_dis_no-x_updated_7.batches_myo.csv
    Untracked:  data/ici-sick.vs.ici-eoi_genes_chr-18_peak.marker-UNC28776739_lod.drop-1.5_snpsqc_dis_no-x_updated_no-sex_7.batches_myo.csv
    Untracked:  data/ici-sick.vs.ici-eoi_genes_chr-18_peak.marker-UNCHS045594_lod.drop-1.5_snpsqc_dis_no-x_updated_7.batches_myo_mis.csv
    Untracked:  data/ici-sick.vs.ici-eoi_genes_chr-18_peak.marker-UNCHS045594_lod.drop-1.5_snpsqc_dis_no-x_updated_no-sex_7.batches_myo_mis.csv
    Untracked:  data/ici-sick.vs.ici-eoi_genes_chr-19_peak.marker-UNC30414168_lod.drop-1.5_snpsqc_dis_no-x_updated_7.batches_myo.csv
    Untracked:  data/ici-sick.vs.ici-eoi_genes_chr-19_peak.marker-UNC30414168_lod.drop-1.5_snpsqc_dis_no-x_updated_no-sex_7.batches_myo.csv
    Untracked:  data/ici-sick.vs.ici-eoi_genes_chr-19_peak.marker-UNC30426276_lod.drop-1.5_snpsqc_dis_no-x_updated_7.batches_myo_mis.csv
    Untracked:  data/ici-sick.vs.ici-eoi_genes_chr-19_peak.marker-UNC30426276_lod.drop-1.5_snpsqc_dis_no-x_updated_no-sex_7.batches_myo_mis.csv
    Untracked:  data/ici-sick.vs.ici-eoi_genes_chr-1_peak.marker-UNC2031646_lod.drop-1.5_snpsqc_dis_no-x_updated_7.batches_myo.csv
    Untracked:  data/ici-sick.vs.ici-eoi_genes_chr-1_peak.marker-UNC2031646_lod.drop-1.5_snpsqc_dis_no-x_updated_no-sex_7.batches_myo.csv
    Untracked:  data/ici-sick.vs.ici-eoi_genes_chr-1_peak.marker-UNCHS003700_lod.drop-1.5_snpsqc_dis_no-x_updated_7.batches_myo_mis.csv
    Untracked:  data/ici-sick.vs.ici-eoi_genes_chr-1_peak.marker-UNCHS003700_lod.drop-1.5_snpsqc_dis_no-x_updated_no-sex_7.batches_myo_mis.csv
    Untracked:  data/ici-sick.vs.ici-eoi_genes_chr-2_peak.marker-ICR5131_lod.drop-1.5_snpsqc_dis_no-x_updated_7.batches_myo.csv
    Untracked:  data/ici-sick.vs.ici-eoi_genes_chr-2_peak.marker-ICR5131_lod.drop-1.5_snpsqc_dis_no-x_updated_no-sex_7.batches_myo.csv
    Untracked:  data/ici-sick.vs.ici-eoi_genes_chr-2_peak.marker-UNCHS006420_lod.drop-1.5_snpsqc_dis_no-x_updated_7.batches_myo_mis.csv
    Untracked:  data/ici-sick.vs.ici-eoi_genes_chr-2_peak.marker-UNCHS006420_lod.drop-1.5_snpsqc_dis_no-x_updated_no-sex_7.batches_myo_mis.csv
    Untracked:  data/ici-sick.vs.ici-eoi_genes_chr-3_peak.marker-UNC5667757_lod.drop-1.5_snpsqc_dis_no-x_updated_7.batches_myo.csv
    Untracked:  data/ici-sick.vs.ici-eoi_genes_chr-3_peak.marker-UNC5667757_lod.drop-1.5_snpsqc_dis_no-x_updated_7.batches_myo_mis.csv
    Untracked:  data/ici-sick.vs.ici-eoi_genes_chr-3_peak.marker-UNC5667757_lod.drop-1.5_snpsqc_dis_no-x_updated_no-sex_7.batches_myo.csv
    Untracked:  data/ici-sick.vs.ici-eoi_genes_chr-3_peak.marker-UNC5667757_lod.drop-1.5_snpsqc_dis_no-x_updated_no-sex_7.batches_myo_mis.csv
    Untracked:  data/ici-sick.vs.ici-eoi_genes_chr-4_peak.marker-UNC6759992_lod.drop-1.5_snpsqc_dis_no-x_updated_7.batches_myo_mis.csv
    Untracked:  data/ici-sick.vs.ici-eoi_genes_chr-4_peak.marker-UNC6759992_lod.drop-1.5_snpsqc_dis_no-x_updated_no-sex_7.batches_myo_mis.csv
    Untracked:  data/ici-sick.vs.ici-eoi_genes_chr-4_peak.marker-UNC6765178_lod.drop-1.5_snpsqc_dis_no-x_updated_7.batches_myo.csv
    Untracked:  data/ici-sick.vs.ici-eoi_genes_chr-4_peak.marker-UNC6765178_lod.drop-1.5_snpsqc_dis_no-x_updated_no-sex_7.batches_myo.csv
    Untracked:  data/ici-sick.vs.ici-eoi_genes_chr-5_peak.marker-UNC9889957_lod.drop-1.5_snpsqc_dis_no-x_updated_7.batches_myo_mis.csv
    Untracked:  data/ici-sick.vs.ici-eoi_genes_chr-5_peak.marker-UNC9889957_lod.drop-1.5_snpsqc_dis_no-x_updated_no-sex_7.batches_myo_mis.csv
    Untracked:  data/ici-sick.vs.ici-eoi_genes_chr-5_peak.marker-UNC9900273_lod.drop-1.5_snpsqc_dis_no-x_updated_7.batches_myo.csv
    Untracked:  data/ici-sick.vs.ici-eoi_genes_chr-5_peak.marker-UNC9900273_lod.drop-1.5_snpsqc_dis_no-x_updated_no-sex_7.batches_myo.csv
    Untracked:  data/ici-sick.vs.ici-eoi_genes_chr-6_peak.marker-UNC10800126_lod.drop-1.5_snpsqc_dis_no-x_updated_7.batches_myo_mis.csv
    Untracked:  data/ici-sick.vs.ici-eoi_genes_chr-6_peak.marker-UNC10800126_lod.drop-1.5_snpsqc_dis_no-x_updated_no-sex_7.batches_myo_mis.csv
    Untracked:  data/ici-sick.vs.ici-eoi_genes_chr-6_peak.marker-UNC10832076_lod.drop-1.5_snpsqc_dis_no-x_updated_7.batches_myo.csv
    Untracked:  data/ici-sick.vs.ici-eoi_genes_chr-6_peak.marker-UNC10832076_lod.drop-1.5_snpsqc_dis_no-x_updated_no-sex_7.batches_myo.csv
    Untracked:  data/ici-sick.vs.ici-eoi_genes_chr-7_peak.marker-UNCHS020903_lod.drop-1.5_snpsqc_dis_no-x_updated_7.batches_myo_mis.csv
    Untracked:  data/ici-sick.vs.ici-eoi_genes_chr-7_peak.marker-UNCHS020903_lod.drop-1.5_snpsqc_dis_no-x_updated_no-sex_7.batches_myo_mis.csv
    Untracked:  data/ici-sick.vs.ici-eoi_genes_chr-7_peak.marker-UNCHS021163_lod.drop-1.5_snpsqc_dis_no-x_updated_7.batches_myo.csv
    Untracked:  data/ici-sick.vs.ici-eoi_genes_chr-7_peak.marker-UNCHS021163_lod.drop-1.5_snpsqc_dis_no-x_updated_no-sex_7.batches_myo.csv
    Untracked:  data/ici-sick.vs.ici-eoi_genes_chr-8_peak.marker-UNC15471847_lod.drop-1.5_snpsqc_dis_no-x_updated_7.batches_myo.csv
    Untracked:  data/ici-sick.vs.ici-eoi_genes_chr-8_peak.marker-UNC15471847_lod.drop-1.5_snpsqc_dis_no-x_updated_no-sex_7.batches_myo.csv
    Untracked:  data/ici-sick.vs.ici-eoi_genes_chr-8_peak.marker-UNC15548888_lod.drop-1.5_snpsqc_dis_no-x_updated_7.batches_myo_mis.csv
    Untracked:  data/ici-sick.vs.ici-eoi_genes_chr-8_peak.marker-UNC15548888_lod.drop-1.5_snpsqc_dis_no-x_updated_no-sex_7.batches_myo_mis.csv
    Untracked:  data/ici-sick.vs.ici-eoi_genes_chr-9_peak.marker-UNC16231874_lod.drop-1.5_snpsqc_dis_no-x_updated_7.batches_myo.csv
    Untracked:  data/ici-sick.vs.ici-eoi_genes_chr-9_peak.marker-UNC16231874_lod.drop-1.5_snpsqc_dis_no-x_updated_7.batches_myo_mis.csv
    Untracked:  data/ici-sick.vs.ici-eoi_genes_chr-9_peak.marker-UNC16231874_lod.drop-1.5_snpsqc_dis_no-x_updated_no-sex_7.batches_myo.csv
    Untracked:  data/ici-sick.vs.ici-eoi_genes_chr-9_peak.marker-UNC16231874_lod.drop-1.5_snpsqc_dis_no-x_updated_no-sex_7.batches_myo_mis.csv
    Untracked:  data/ici-sick.vs.ici-eoi_genes_chr-X_peak.marker-UNCHS049472_lod.drop-1.5_snpsqc_dis_no-x_updated_7.batches_myo.csv
    Untracked:  data/ici-sick.vs.ici-eoi_genes_chr-X_peak.marker-UNCHS049472_lod.drop-1.5_snpsqc_dis_no-x_updated_7.batches_myo_mis.csv
    Untracked:  data/ici-sick.vs.ici-eoi_genes_chr-X_peak.marker-UNCHS049472_lod.drop-1.5_snpsqc_dis_no-x_updated_no-sex_7.batches_myo.csv
    Untracked:  data/ici-sick.vs.ici-eoi_genes_chr-X_peak.marker-UNCHS049472_lod.drop-1.5_snpsqc_dis_no-x_updated_no-sex_7.batches_myo_mis.csv
    Untracked:  data/ici-sick.vs.ici-eoi_gm_qtl_snpsqc_dis_no-x_updated_7.batches_myo.csv
    Untracked:  data/ici-sick.vs.ici-eoi_gm_qtl_snpsqc_dis_no-x_updated_7.batches_myo_mis.csv
    Untracked:  data/ici-sick.vs.ici-eoi_gm_qtl_snpsqc_dis_no-x_updated_no-sex_7.batches_myo.csv
    Untracked:  data/ici-sick.vs.ici-eoi_gm_qtl_snpsqc_dis_no-x_updated_no-sex_7.batches_myo_mis.csv
    Untracked:  data/ici-sick.vs.ici-eoi_marker.freq_low.geno.freq.removed_geno.ratio_7.batches_myo.csv
    Untracked:  data/ici-sick.vs.ici-eoi_marker.freq_low.geno.freq.removed_geno.ratio_7.batches_myo_mis.csv
    Untracked:  data/ici-sick.vs.ici-eoi_marker.freq_low.geno.freq.removed_sample.outliers.removed_geno.ratio_7.batches_myo.csv
    Untracked:  data/ici-sick.vs.ici-eoi_marker.freq_low.geno.freq.removed_sample.outliers.removed_geno.ratio_7.batches_myo_mis.csv
    Untracked:  data/ici-sick.vs.ici-eoi_marker.freq_low.probs.freq.removed_geno.ratio_7.batches_myo.csv
    Untracked:  data/ici-sick.vs.ici-eoi_marker.freq_low.probs.freq.removed_geno.ratio_7.batches_myo_mis.csv
    Untracked:  data/ici-sick.vs.ici-eoi_marker.freq_low.probs.freq.removed_sample.outliers.removed_geno.ratio_7.batches_myo.csv
    Untracked:  data/ici-sick.vs.ici-eoi_marker.freq_low.probs.freq.removed_sample.outliers.removed_geno.ratio_7.batches_myo_mis.csv
    Untracked:  data/ici-sick.vs.ici-eoi_sample.genos_marker.freq_low.geno.freq.removed_7.batches_myo.csv
    Untracked:  data/ici-sick.vs.ici-eoi_sample.genos_marker.freq_low.geno.freq.removed_7.batches_myo_mis.csv
    Untracked:  data/ici-sick.vs.ici-eoi_sample.genos_marker.freq_low.geno.freq.removed_sample.outliers.removed_7.batches_myo.csv
    Untracked:  data/ici-sick.vs.ici-eoi_sample.genos_marker.freq_low.geno.freq.removed_sample.outliers.removed_7.batches_myo_mis.csv
    Untracked:  data/ici-sick.vs.ici-eoi_sample.genos_marker.freq_low.probs.freq.removed_7.batches_myo.csv
    Untracked:  data/ici-sick.vs.ici-eoi_sample.genos_marker.freq_low.probs.freq.removed_7.batches_myo_mis.csv
    Untracked:  data/ici-sick.vs.ici-eoi_sample.genos_marker.freq_low.probs.freq.removed_sample.outliers.removed_7.batches_myo.csv
    Untracked:  data/ici-sick.vs.ici-eoi_sample.genos_marker.freq_low.probs.freq.removed_sample.outliers.removed_7.batches_myo_mis.csv
    Untracked:  data/ici.vs.pbs_blup_sub_chr-7_peak.marker-UNCHS020711_lod.drop-1.5_snpsqc_dis_no-x_updated_7.batches_myo.csv
    Untracked:  data/ici.vs.pbs_blup_sub_chr-7_peak.marker-UNCHS020711_lod.drop-1.5_snpsqc_dis_no-x_updated_7.batches_myo_mis.csv
    Untracked:  data/ici.vs.pbs_blup_sub_chr-7_peak.marker-UNCHS020711_lod.drop-1.5_snpsqc_dis_no-x_updated_no-sex_7.batches_myo.csv
    Untracked:  data/ici.vs.pbs_blup_sub_chr-7_peak.marker-UNCHS020711_lod.drop-1.5_snpsqc_dis_no-x_updated_no-sex_7.batches_myo_mis.csv
    Untracked:  data/ici.vs.pbs_genes_chr-7_peak.marker-UNCHS020711_lod.drop-1.5_snpsqc_dis_no-x_updated_7.batches_myo.csv
    Untracked:  data/ici.vs.pbs_genes_chr-7_peak.marker-UNCHS020711_lod.drop-1.5_snpsqc_dis_no-x_updated_7.batches_myo_mis.csv
    Untracked:  data/ici.vs.pbs_genes_chr-7_peak.marker-UNCHS020711_lod.drop-1.5_snpsqc_dis_no-x_updated_no-sex_7.batches_myo.csv
    Untracked:  data/ici.vs.pbs_genes_chr-7_peak.marker-UNCHS020711_lod.drop-1.5_snpsqc_dis_no-x_updated_no-sex_7.batches_myo_mis.csv
    Untracked:  data/ici.vs.pbs_gm_qtl_snpsqc_dis_no-x_updated_7.batches_myo.csv
    Untracked:  data/ici.vs.pbs_gm_qtl_snpsqc_dis_no-x_updated_7.batches_myo_mis.csv
    Untracked:  data/ici.vs.pbs_gm_qtl_snpsqc_dis_no-x_updated_no-sex_7.batches_myo.csv
    Untracked:  data/ici.vs.pbs_gm_qtl_snpsqc_dis_no-x_updated_no-sex_7.batches_myo_mis.csv
    Untracked:  data/ici.vs.pbs_marker.freq_low.geno.freq.removed_geno.ratio_7.batches_myo.csv
    Untracked:  data/ici.vs.pbs_marker.freq_low.geno.freq.removed_geno.ratio_7.batches_myo_mis.csv
    Untracked:  data/ici.vs.pbs_marker.freq_low.geno.freq.removed_sample.outliers.removed_geno.ratio_7.batches_myo.csv
    Untracked:  data/ici.vs.pbs_marker.freq_low.geno.freq.removed_sample.outliers.removed_geno.ratio_7.batches_myo_mis.csv
    Untracked:  data/ici.vs.pbs_marker.freq_low.probs.freq.removed_geno.ratio_7.batches_myo.csv
    Untracked:  data/ici.vs.pbs_marker.freq_low.probs.freq.removed_geno.ratio_7.batches_myo_mis.csv
    Untracked:  data/ici.vs.pbs_marker.freq_low.probs.freq.removed_sample.outliers.removed_geno.ratio_7.batches_myo.csv
    Untracked:  data/ici.vs.pbs_marker.freq_low.probs.freq.removed_sample.outliers.removed_geno.ratio_7.batches_myo_mis.csv
    Untracked:  data/ici.vs.pbs_sample.genos_marker.freq_low.geno.freq.removed_7.batches_myo.csv
    Untracked:  data/ici.vs.pbs_sample.genos_marker.freq_low.geno.freq.removed_7.batches_myo_mis.csv
    Untracked:  data/ici.vs.pbs_sample.genos_marker.freq_low.geno.freq.removed_sample.outliers.removed_7.batches_myo.csv
    Untracked:  data/ici.vs.pbs_sample.genos_marker.freq_low.geno.freq.removed_sample.outliers.removed_7.batches_myo_mis.csv
    Untracked:  data/ici.vs.pbs_sample.genos_marker.freq_low.probs.freq.removed_7.batches_myo.csv
    Untracked:  data/ici.vs.pbs_sample.genos_marker.freq_low.probs.freq.removed_7.batches_myo_mis.csv
    Untracked:  data/ici.vs.pbs_sample.genos_marker.freq_low.probs.freq.removed_sample.outliers.removed_7.batches_myo.csv
    Untracked:  data/ici.vs.pbs_sample.genos_marker.freq_low.probs.freq.removed_sample.outliers.removed_7.batches_myo_mis.csv
    Untracked:  data/mean.differences_age.of.onset_het-ici-myo-yes.vs.het-ici-myo-no_sample.genos_marker.freq_low.geno.freq.removed_7.batches_myo.csv
    Untracked:  data/mean.differences_age.of.onset_het-ici-myo-yes.vs.het-ici-myo-no_sample.genos_marker.freq_low.geno.freq.removed_7.batches_myo_mis.csv
    Untracked:  data/mean.differences_age.of.onset_het-ici-myo-yes.vs.het-ici-myo-no_sample.genos_marker.freq_low.geno.freq.removed_sample.outliers.removed_7.batches_myo.csv
    Untracked:  data/mean.differences_age.of.onset_het-ici-myo-yes.vs.het-ici-myo-no_sample.genos_marker.freq_low.geno.freq.removed_sample.outliers.removed_7.batches_myo_mis.csv
    Untracked:  data/mean.differences_age.of.onset_het-ici.vs.het-pbs_sample.genos_marker.freq_low.geno.freq.removed_7.batches_myo.csv
    Untracked:  data/mean.differences_age.of.onset_het-ici.vs.het-pbs_sample.genos_marker.freq_low.geno.freq.removed_7.batches_myo_mis.csv
    Untracked:  data/mean.differences_age.of.onset_het-ici.vs.het-pbs_sample.genos_marker.freq_low.geno.freq.removed_sample.outliers.removed_7.batches_myo.csv
    Untracked:  data/mean.differences_age.of.onset_het-ici.vs.het-pbs_sample.genos_marker.freq_low.geno.freq.removed_sample.outliers.removed_7.batches_myo_mis.csv
    Untracked:  data/mean.differences_age.of.onset_ici-myo-yes.vs.ici-myo-no_sample.genos_marker.freq_low.geno.freq.removed_7.batches_myo.csv
    Untracked:  data/mean.differences_age.of.onset_ici-myo-yes.vs.ici-myo-no_sample.genos_marker.freq_low.geno.freq.removed_7.batches_myo_mis.csv
    Untracked:  data/mean.differences_age.of.onset_ici-myo-yes.vs.ici-myo-no_sample.genos_marker.freq_low.geno.freq.removed_sample.outliers.removed_7.batches_myo.csv
    Untracked:  data/mean.differences_age.of.onset_ici-myo-yes.vs.ici-myo-no_sample.genos_marker.freq_low.geno.freq.removed_sample.outliers.removed_7.batches_myo_mis.csv
    Untracked:  data/mean.differences_age.of.onset_ici-sick.vs.ici-eoi_sample.genos_marker.freq_low.geno.freq.removed_7.batches_myo.csv
    Untracked:  data/mean.differences_age.of.onset_ici-sick.vs.ici-eoi_sample.genos_marker.freq_low.geno.freq.removed_7.batches_myo_mis.csv
    Untracked:  data/mean.differences_age.of.onset_ici-sick.vs.ici-eoi_sample.genos_marker.freq_low.geno.freq.removed_sample.outliers.removed_7.batches_myo.csv
    Untracked:  data/mean.differences_age.of.onset_ici-sick.vs.ici-eoi_sample.genos_marker.freq_low.geno.freq.removed_sample.outliers.removed_7.batches_myo_mis.csv
    Untracked:  data/mean.differences_age.of.onset_ici.vs.pbs_sample.genos_marker.freq_low.geno.freq.removed_7.batches_myo.csv
    Untracked:  data/mean.differences_age.of.onset_ici.vs.pbs_sample.genos_marker.freq_low.geno.freq.removed_7.batches_myo_mis.csv
    Untracked:  data/mean.differences_age.of.onset_ici.vs.pbs_sample.genos_marker.freq_low.geno.freq.removed_sample.outliers.removed_7.batches_myo.csv
    Untracked:  data/mean.differences_age.of.onset_ici.vs.pbs_sample.genos_marker.freq_low.geno.freq.removed_sample.outliers.removed_7.batches_myo_mis.csv
    Untracked:  data/mean.differences_age.of.onset_myo-yes.vs.myo-no_sample.genos_marker.freq_low.geno.freq.removed_7.batches_myo.csv
    Untracked:  data/mean.differences_age.of.onset_myo-yes.vs.myo-no_sample.genos_marker.freq_low.geno.freq.removed_7.batches_myo_mis.csv
    Untracked:  data/mean.differences_age.of.onset_myo-yes.vs.myo-no_sample.genos_marker.freq_low.geno.freq.removed_sample.outliers.removed_7.batches_myo.csv
    Untracked:  data/mean.differences_age.of.onset_myo-yes.vs.myo-no_sample.genos_marker.freq_low.geno.freq.removed_sample.outliers.removed_7.batches_myo_mis.csv
    Untracked:  data/mean.differences_age.of.onset_pbs-myo-yes.vs.pbs-myo-no_sample.genos_marker.freq_low.geno.freq.removed_7.batches_myo.csv
    Untracked:  data/mean.differences_age.of.onset_pbs-myo-yes.vs.pbs-myo-no_sample.genos_marker.freq_low.geno.freq.removed_7.batches_myo_mis.csv
    Untracked:  data/mean.differences_age.of.onset_pbs-myo-yes.vs.pbs-myo-no_sample.genos_marker.freq_low.geno.freq.removed_sample.outliers.removed_7.batches_myo.csv
    Untracked:  data/mean.differences_age.of.onset_pbs-myo-yes.vs.pbs-myo-no_sample.genos_marker.freq_low.geno.freq.removed_sample.outliers.removed_7.batches_myo_mis.csv
    Untracked:  data/mean.differences_group_het-ici-myo-yes.vs.het-ici-myo-no_sample.genos_marker.freq_low.geno.freq.removed_7.batches_myo.csv
    Untracked:  data/mean.differences_group_het-ici-myo-yes.vs.het-ici-myo-no_sample.genos_marker.freq_low.geno.freq.removed_7.batches_myo_mis.csv
    Untracked:  data/mean.differences_group_het-ici-myo-yes.vs.het-ici-myo-no_sample.genos_marker.freq_low.geno.freq.removed_sample.outliers.removed_7.batches_myo.csv
    Untracked:  data/mean.differences_group_het-ici-myo-yes.vs.het-ici-myo-no_sample.genos_marker.freq_low.geno.freq.removed_sample.outliers.removed_7.batches_myo_mis.csv
    Untracked:  data/mean.differences_group_het-ici.vs.het-pbs_sample.genos_marker.freq_low.geno.freq.removed_7.batches_myo.csv
    Untracked:  data/mean.differences_group_het-ici.vs.het-pbs_sample.genos_marker.freq_low.geno.freq.removed_7.batches_myo_mis.csv
    Untracked:  data/mean.differences_group_het-ici.vs.het-pbs_sample.genos_marker.freq_low.geno.freq.removed_sample.outliers.removed_7.batches_myo.csv
    Untracked:  data/mean.differences_group_het-ici.vs.het-pbs_sample.genos_marker.freq_low.geno.freq.removed_sample.outliers.removed_7.batches_myo_mis.csv
    Untracked:  data/mean.differences_group_ici-myo-yes.vs.ici-myo-no_sample.genos_marker.freq_low.geno.freq.removed_7.batches_myo.csv
    Untracked:  data/mean.differences_group_ici-myo-yes.vs.ici-myo-no_sample.genos_marker.freq_low.geno.freq.removed_7.batches_myo_mis.csv
    Untracked:  data/mean.differences_group_ici-myo-yes.vs.ici-myo-no_sample.genos_marker.freq_low.geno.freq.removed_sample.outliers.removed_7.batches_myo.csv
    Untracked:  data/mean.differences_group_ici-myo-yes.vs.ici-myo-no_sample.genos_marker.freq_low.geno.freq.removed_sample.outliers.removed_7.batches_myo_mis.csv
    Untracked:  data/mean.differences_group_ici-sick.vs.ici-eoi_sample.genos_marker.freq_low.geno.freq.removed_7.batches_myo.csv
    Untracked:  data/mean.differences_group_ici-sick.vs.ici-eoi_sample.genos_marker.freq_low.geno.freq.removed_7.batches_myo_mis.csv
    Untracked:  data/mean.differences_group_ici-sick.vs.ici-eoi_sample.genos_marker.freq_low.geno.freq.removed_sample.outliers.removed_7.batches_myo.csv
    Untracked:  data/mean.differences_group_ici-sick.vs.ici-eoi_sample.genos_marker.freq_low.geno.freq.removed_sample.outliers.removed_7.batches_myo_mis.csv
    Untracked:  data/mean.differences_group_ici.vs.pbs_sample.genos_marker.freq_low.geno.freq.removed_7.batches_myo.csv
    Untracked:  data/mean.differences_group_ici.vs.pbs_sample.genos_marker.freq_low.geno.freq.removed_7.batches_myo_mis.csv
    Untracked:  data/mean.differences_group_ici.vs.pbs_sample.genos_marker.freq_low.geno.freq.removed_sample.outliers.removed_7.batches_myo.csv
    Untracked:  data/mean.differences_group_ici.vs.pbs_sample.genos_marker.freq_low.geno.freq.removed_sample.outliers.removed_7.batches_myo_mis.csv
    Untracked:  data/mean.differences_group_myo-yes.vs.myo-no_sample.genos_marker.freq_low.geno.freq.removed_7.batches_myo.csv
    Untracked:  data/mean.differences_group_myo-yes.vs.myo-no_sample.genos_marker.freq_low.geno.freq.removed_7.batches_myo_mis.csv
    Untracked:  data/mean.differences_group_myo-yes.vs.myo-no_sample.genos_marker.freq_low.geno.freq.removed_sample.outliers.removed_7.batches_myo.csv
    Untracked:  data/mean.differences_group_myo-yes.vs.myo-no_sample.genos_marker.freq_low.geno.freq.removed_sample.outliers.removed_7.batches_myo_mis.csv
    Untracked:  data/mean.differences_group_pbs-myo-yes.vs.pbs-myo-no_sample.genos_marker.freq_low.geno.freq.removed_7.batches_myo.csv
    Untracked:  data/mean.differences_group_pbs-myo-yes.vs.pbs-myo-no_sample.genos_marker.freq_low.geno.freq.removed_7.batches_myo_mis.csv
    Untracked:  data/mean.differences_group_pbs-myo-yes.vs.pbs-myo-no_sample.genos_marker.freq_low.geno.freq.removed_sample.outliers.removed_7.batches_myo.csv
    Untracked:  data/mean.differences_group_pbs-myo-yes.vs.pbs-myo-no_sample.genos_marker.freq_low.geno.freq.removed_sample.outliers.removed_7.batches_myo_mis.csv
    Untracked:  data/myo-yes.vs.myo-no_gm_qtl_snpsqc_7.batches_myo.csv
    Untracked:  data/myo-yes.vs.myo-no_gm_qtl_snpsqc_7.batches_myo_mis.csv
    Untracked:  data/myo-yes.vs.myo-no_gm_qtl_snpsqc_dis_no-x_updated_7.batches_myo.csv
    Untracked:  data/myo-yes.vs.myo-no_gm_qtl_snpsqc_dis_no-x_updated_7.batches_myo_mis.csv
    Untracked:  data/myo-yes.vs.myo-no_gm_qtl_snpsqc_dis_no-x_updated_no-sex_7.batches_myo.csv
    Untracked:  data/myo-yes.vs.myo-no_gm_qtl_snpsqc_dis_no-x_updated_no-sex_7.batches_myo_mis.csv
    Untracked:  data/myo-yes.vs.myo-no_marker.freq_low.geno.freq.removed_geno.ratio_7.batches_myo.csv
    Untracked:  data/myo-yes.vs.myo-no_marker.freq_low.geno.freq.removed_geno.ratio_7.batches_myo_mis.csv
    Untracked:  data/myo-yes.vs.myo-no_marker.freq_low.geno.freq.removed_sample.outliers.removed_geno.ratio_7.batches_myo.csv
    Untracked:  data/myo-yes.vs.myo-no_marker.freq_low.geno.freq.removed_sample.outliers.removed_geno.ratio_7.batches_myo_mis.csv
    Untracked:  data/myo-yes.vs.myo-no_marker.freq_low.probs.freq.removed_geno.ratio_7.batches_myo.csv
    Untracked:  data/myo-yes.vs.myo-no_marker.freq_low.probs.freq.removed_geno.ratio_7.batches_myo_mis.csv
    Untracked:  data/myo-yes.vs.myo-no_marker.freq_low.probs.freq.removed_sample.outliers.removed_geno.ratio_7.batches_myo.csv
    Untracked:  data/myo-yes.vs.myo-no_marker.freq_low.probs.freq.removed_sample.outliers.removed_geno.ratio_7.batches_myo_mis.csv
    Untracked:  data/myo-yes.vs.myo-no_sample.genos_marker.freq_low.geno.freq.removed_7.batches_myo.csv
    Untracked:  data/myo-yes.vs.myo-no_sample.genos_marker.freq_low.geno.freq.removed_7.batches_myo_mis.csv
    Untracked:  data/myo-yes.vs.myo-no_sample.genos_marker.freq_low.geno.freq.removed_sample.outliers.removed_7.batches_myo.csv
    Untracked:  data/myo-yes.vs.myo-no_sample.genos_marker.freq_low.geno.freq.removed_sample.outliers.removed_7.batches_myo_mis.csv
    Untracked:  data/myo-yes.vs.myo-no_sample.genos_marker.freq_low.probs.freq.removed_7.batches_myo.csv
    Untracked:  data/myo-yes.vs.myo-no_sample.genos_marker.freq_low.probs.freq.removed_7.batches_myo_mis.csv
    Untracked:  data/myo-yes.vs.myo-no_sample.genos_marker.freq_low.probs.freq.removed_sample.outliers.removed_7.batches_myo.csv
    Untracked:  data/myo-yes.vs.myo-no_sample.genos_marker.freq_low.probs.freq.removed_sample.outliers.removed_7.batches_myo_mis.csv
    Untracked:  data/pbs-myo-yes.vs.pbs-myo-no_gm_qtl_snpsqc_7.batches_myo.csv
    Untracked:  data/pbs-myo-yes.vs.pbs-myo-no_gm_qtl_snpsqc_7.batches_myo_mis.csv
    Untracked:  data/pbs-myo-yes.vs.pbs-myo-no_gm_qtl_snpsqc_dis_no-x_updated_7.batches_myo.csv
    Untracked:  data/pbs-myo-yes.vs.pbs-myo-no_gm_qtl_snpsqc_dis_no-x_updated_7.batches_myo_mis.csv
    Untracked:  data/pbs-myo-yes.vs.pbs-myo-no_gm_qtl_snpsqc_dis_no-x_updated_no-sex_7.batches_myo.csv
    Untracked:  data/pbs-myo-yes.vs.pbs-myo-no_gm_qtl_snpsqc_dis_no-x_updated_no-sex_7.batches_myo_mis.csv
    Untracked:  data/pbs-myo-yes.vs.pbs-myo-no_marker.freq_low.geno.freq.removed_geno.ratio_7.batches_myo.csv
    Untracked:  data/pbs-myo-yes.vs.pbs-myo-no_marker.freq_low.geno.freq.removed_geno.ratio_7.batches_myo_mis.csv
    Untracked:  data/pbs-myo-yes.vs.pbs-myo-no_marker.freq_low.geno.freq.removed_sample.outliers.removed_geno.ratio_7.batches_myo.csv
    Untracked:  data/pbs-myo-yes.vs.pbs-myo-no_marker.freq_low.geno.freq.removed_sample.outliers.removed_geno.ratio_7.batches_myo_mis.csv
    Untracked:  data/pbs-myo-yes.vs.pbs-myo-no_marker.freq_low.probs.freq.removed_geno.ratio_7.batches_myo.csv
    Untracked:  data/pbs-myo-yes.vs.pbs-myo-no_marker.freq_low.probs.freq.removed_geno.ratio_7.batches_myo_mis.csv
    Untracked:  data/pbs-myo-yes.vs.pbs-myo-no_marker.freq_low.probs.freq.removed_sample.outliers.removed_geno.ratio_7.batches_myo.csv
    Untracked:  data/pbs-myo-yes.vs.pbs-myo-no_marker.freq_low.probs.freq.removed_sample.outliers.removed_geno.ratio_7.batches_myo_mis.csv
    Untracked:  data/pbs-myo-yes.vs.pbs-myo-no_sample.genos_marker.freq_low.geno.freq.removed_7.batches_myo.csv
    Untracked:  data/pbs-myo-yes.vs.pbs-myo-no_sample.genos_marker.freq_low.geno.freq.removed_7.batches_myo_mis.csv
    Untracked:  data/pbs-myo-yes.vs.pbs-myo-no_sample.genos_marker.freq_low.geno.freq.removed_sample.outliers.removed_7.batches_myo.csv
    Untracked:  data/pbs-myo-yes.vs.pbs-myo-no_sample.genos_marker.freq_low.geno.freq.removed_sample.outliers.removed_7.batches_myo_mis.csv
    Untracked:  data/pbs-myo-yes.vs.pbs-myo-no_sample.genos_marker.freq_low.probs.freq.removed_7.batches_myo.csv
    Untracked:  data/pbs-myo-yes.vs.pbs-myo-no_sample.genos_marker.freq_low.probs.freq.removed_7.batches_myo_mis.csv
    Untracked:  data/pbs-myo-yes.vs.pbs-myo-no_sample.genos_marker.freq_low.probs.freq.removed_sample.outliers.removed_7.batches_myo.csv
    Untracked:  data/pbs-myo-yes.vs.pbs-myo-no_sample.genos_marker.freq_low.probs.freq.removed_sample.outliers.removed_7.batches_myo_mis.csv
    Untracked:  data/percent_missing_id_7.batches_myo.RData
    Untracked:  data/percent_missing_marker_7.batches_myo.RData
    Untracked:  data/pheno_7.batches_myo.csv
    Untracked:  data/physical_map_7.batches_myo.csv
    Untracked:  data/probs_36state_bc_7.batches_myo.rds
    Untracked:  data/probs_8state_bc_7.batches_myo.rds
    Untracked:  data/qc_info_bad_sample_7.batches_myo.RData
    Untracked:  data/sample_geno_AHB_7.batches_myo.csv
    Untracked:  data/sample_geno_AHB_7.batches_myo_orig.id.csv
    Untracked:  data/sample_geno_bc_7.batches_myo.csv
    Untracked:  data/sample_geno_bc_7.batches_myo_orig.id.csv
    Untracked:  data/serreze_probs_7.batches_myo.rds
    Untracked:  data/serreze_probs_allqc_7.batches_myo.rds
    Untracked:  data/serreze_probs_allqc_7.batches_myo_mis.rds
    Untracked:  data/summary.cg_7.batches_myo.RData
    Untracked:  data/summary.counts_EOI_ici.sick_vs_ici.eoi
    Untracked:  data/summary.counts_SICK_ici.sick_vs_ici.eoi
    Untracked:  data/summary.frequency_EOI_ici.sick_vs_ici.eoi
    Untracked:  data/summary.frequency_SICK_ici.sick_vs_ici.eoi

Unstaged changes:
    Modified:   analysis/_site.yml
    Modified:   analysis/index.Rmd

</code></pre>
<p>
Note that any generated files, e.g. HTML, png, CSS, etc., are not included in this status report because it is ok for generated content to have uncommitted changes.
</p>
</div>
</div>
</div>
</div>
<hr>
</div>
<div id="versions" class="tab-pane fade">
<p>
There are no past versions. Publish this analysis with <code>wflow_publish()</code> to start tracking its development.
</p>
<hr>
</div>
</div>
</div>
<div id="data-information" class="section level2">
<h2>Data Information</h2>
<ul>
<li>Myocarditis Status: Yes</li>
<li>Murine MHC KO Status: Homo</li>
<li>Drug Treatment(s): ICI, PBS</li>
<li>Clinical Phenotype(s): Sick, EOI</li>
<li>Covariates (Myocarditis Score, Age of Clinical Phenotype – if Null, use 40 age at which surviving mice were end of incidence without clinical symptoms, sex)</li>
</ul>
</div>
<div id="loading-data" class="section level2">
<h2>Loading Data</h2>
<p>We will load the data and subset indivials out that are in the groups of interest.</p>
<pre class="r"><code>load(&quot;data/gm_allqc_7.batches_myo_mis.RData&quot;)

#gm_allqc
gm=gm_allqc
gm</code></pre>
<pre><code>Object of class cross2 (crosstype &quot;bc&quot;)

Total individuals               357
No. genotyped individuals       357
No. phenotyped individuals      357
No. with both geno &amp; pheno      357

No. phenotypes                    1
No. covariates                   11
No. phenotype covariates          0

No. chromosomes                  20
Total markers                130890

No. markers by chr:
   1    2    3    4    5    6    7    8    9   10   11   12   13   14   15   16 
9927 9959 7817 7558 7564 7722 7374 6434 6686 6356 7128 6091 6053 5944 5322 5003 
  17   18   19    X 
5053 4590 3547 4762 </code></pre>
<pre class="r"><code>#pr &lt;- readRDS(&quot;data/serreze_probs_allqc.rds&quot;)
#pr &lt;- readRDS(&quot;data/serreze_probs.rds&quot;)

##extracting animals with ici and pbs group status
#miceinfo &lt;- covars[gm$covar$group == &quot;PBS&quot; | gm$covar$group == &quot;ICI&quot;,]
#table(miceinfo$group)
#mice.ids &lt;- rownames(miceinfo)

#gm &lt;- gm[mice.ids]
#gm
#table(gm$covar$group)

#gm$covar$ici_vs_pbs &lt;- ifelse(gm$covar$group == &quot;PBS&quot;, 0, 1)
#gm.full &lt;- gm

covars &lt;- read_csv(&quot;data/covar_corrected_ici.vs.pbs_7.batches_myo_mis.csv&quot;)
#removing any missing info
#covars &lt;- subset(covars, covars$ici_vs_pbs!=&#39;&#39;)
nrow(covars)</code></pre>
<pre><code>[1] 212</code></pre>
<pre class="r"><code>table(covars$&quot;Myocarditis Status&quot;)</code></pre>
<pre><code>
YES 
212 </code></pre>
<pre class="r"><code>table(covars$&quot;Murine MHC KO Status&quot;)</code></pre>
<pre><code>
HOM 
212 </code></pre>
<pre class="r"><code>table(covars$&quot;Drug Treatment&quot;)</code></pre>
<pre><code>
ICI PBS 
133  79 </code></pre>
<pre class="r"><code>table(covars$&quot;clinical pheno&quot;)</code></pre>
<pre><code>
 EOI SICK 
  74  138 </code></pre>
<pre class="r"><code>#keeping only informative mice
gm &lt;- gm[covars$Mouse.ID]
gm</code></pre>
<pre><code>Object of class cross2 (crosstype &quot;bc&quot;)

Total individuals               212
No. genotyped individuals       212
No. phenotyped individuals      212
No. with both geno &amp; pheno      212

No. phenotypes                    1
No. covariates                   11
No. phenotype covariates          0

No. chromosomes                  20
Total markers                130890

No. markers by chr:
   1    2    3    4    5    6    7    8    9   10   11   12   13   14   15   16 
9927 9959 7817 7558 7564 7722 7374 6434 6686 6356 7128 6091 6053 5944 5322 5003 
  17   18   19    X 
5053 4590 3547 4762 </code></pre>
<pre class="r"><code>table(gm$covar$&quot;Myocarditis Status&quot;)</code></pre>
<pre><code>
YES 
212 </code></pre>
<pre class="r"><code>table(gm$covar$&quot;Murine MHC KO Status&quot;)</code></pre>
<pre><code>
HOM 
212 </code></pre>
<pre class="r"><code>table(gm$covar$&quot;Drug Treatment&quot;)</code></pre>
<pre><code>
ICI PBS 
133  79 </code></pre>
<pre class="r"><code>table(gm$covar$&quot;clinical pheno&quot;)</code></pre>
<pre><code>
 EOI SICK 
  74  138 </code></pre>
<pre class="r"><code>#pr.qc.ids &lt;- pr
#for (i in 1:20){pr.qc.ids[[i]] = pr.qc.ids[[i]][covars$Mouse.ID,,]}

##removing problmetic marker

#gm &lt;- drop_markers(gm, &quot;UNCHS013106&quot;)

##dropping monomorphic markers within the dataset

g &lt;- do.call(&quot;cbind&quot;, gm$geno)

gf_mar &lt;- t(apply(g, 2, function(a) table(factor(a, 1:2))/sum(a != 0)))
#gn_mar &lt;- t(apply(g, 2, function(a) table(factor(a, 1:2))))

gf_mar &lt;- gf_mar[gf_mar[,2] != &quot;NaN&quot;,]

count &lt;- rowSums(gf_mar &lt;=0.05)
low_freq_df &lt;- merge(as.data.frame(gf_mar),as.data.frame(count), by=&quot;row.names&quot;,all=T)
low_freq_df[is.na(low_freq_df)] &lt;- &#39;&#39;
low_freq_df &lt;- low_freq_df[low_freq_df$count == 1,]
rownames(low_freq_df) &lt;- low_freq_df$Row.names

low_freq &lt;- find_markerpos(gm, rownames(low_freq_df))
low_freq$id &lt;- rownames(low_freq)

nrow(low_freq)</code></pre>
<pre><code>[1] 97826</code></pre>
<pre class="r"><code>low_freq_bad &lt;- merge(low_freq,low_freq_df, by=&quot;row.names&quot;,all=T)
names(low_freq_bad)[1] &lt;- c(&quot;marker&quot;)

gf_mar &lt;- gf_mar[gf_mar[,2] != &quot;NaN&quot;,]
MAF &lt;- apply(gf_mar, 1, function(x) min(x))
MAF &lt;- as.data.frame(MAF)
MAF$index &lt;- 1:nrow(gf_mar)
gf_mar_maf &lt;- merge(gf_mar,as.data.frame(MAF), by=&quot;row.names&quot;)
gf_mar_maf &lt;- gf_mar_maf[order(gf_mar_maf$index),]

gfmar &lt;- NULL
gfmar$gfmar_mar_0 &lt;- sum(gf_mar_maf$MAF==0)
gfmar$gfmar_mar_1 &lt;- sum(gf_mar_maf$MAF&lt; 0.01)
gfmar$gfmar_mar_5 &lt;- sum(gf_mar_maf$MAF&lt; 0.05)
gfmar$gfmar_mar_10 &lt;- sum(gf_mar_maf$MAF&lt; 0.10)
gfmar$gfmar_mar_15 &lt;- sum(gf_mar_maf$MAF&lt; 0.15)
gfmar$gfmar_mar_25 &lt;- sum(gf_mar_maf$MAF&lt; 0.25)
gfmar$gfmar_mar_50 &lt;- sum(gf_mar_maf$MAF&lt; 0.50)
gfmar$total_snps &lt;- nrow(as.data.frame(gf_mar_maf))

gfmar &lt;- t(as.data.frame(gfmar))
gfmar &lt;- as.data.frame(gfmar)
gfmar$count &lt;- gfmar$V1

gfmar[c(2)] %&gt;%
  kable(escape = F,align = c(&quot;ccccccccc&quot;),linesep =&quot;\\hline&quot;) %&gt;%
  kable_styling(full_width = F) %&gt;%
  kable_styling(&quot;striped&quot;, full_width = F)  %&gt;%
  row_spec(8 ,bold=T,color= &quot;white&quot;,background = &quot;black&quot;)</code></pre>
<table class="table table table-striped" style="width: auto !important; margin-left: auto; margin-right: auto; width: auto !important; margin-left: auto; margin-right: auto;">
<thead>
<tr>
<th style="text-align:left;">
</th>
<th style="text-align:center;">
count
</th>
</tr>
</thead>
<tbody>
<tr>
<td style="text-align:left;">
gfmar_mar_0
</td>
<td style="text-align:center;">
92816
</td>
</tr>
<tr>
<td style="text-align:left;">
gfmar_mar_1
</td>
<td style="text-align:center;">
95853
</td>
</tr>
<tr>
<td style="text-align:left;">
gfmar_mar_5
</td>
<td style="text-align:center;">
97824
</td>
</tr>
<tr>
<td style="text-align:left;">
gfmar_mar_10
</td>
<td style="text-align:center;">
98351
</td>
</tr>
<tr>
<td style="text-align:left;">
gfmar_mar_15
</td>
<td style="text-align:center;">
98491
</td>
</tr>
<tr>
<td style="text-align:left;">
gfmar_mar_25
</td>
<td style="text-align:center;">
98650
</td>
</tr>
<tr>
<td style="text-align:left;">
gfmar_mar_50
</td>
<td style="text-align:center;">
129807
</td>
</tr>
<tr>
<td style="text-align:left;font-weight: bold;color: white !important;background-color: black !important;">
total_snps
</td>
<td style="text-align:center;font-weight: bold;color: white !important;background-color: black !important;">
130890
</td>
</tr>
</tbody>
</table>
<pre class="r"><code>gm_qc &lt;- drop_markers(gm, low_freq_bad$marker)
gm_qc &lt;- drop_nullmarkers(gm_qc)

gm = gm_qc
gm</code></pre>
<pre><code>Object of class cross2 (crosstype &quot;bc&quot;)

Total individuals              212
No. genotyped individuals      212
No. phenotyped individuals     212
No. with both geno &amp; pheno     212

No. phenotypes                   1
No. covariates                  11
No. phenotype covariates         0

No. chromosomes                 20
Total markers                33064

No. markers by chr:
   1    2    3    4    5    6    7    8    9   10   11   12   13   14   15   16 
2943 2788 2056 2053 1929 2057 1862 1692 1998 1239 2078 1409 1630 1707 1276  953 
  17   18   19    X 
 691 1083 1053  567 </code></pre>
<pre class="r"><code>## dropping disproportionate markers
#dismark &lt;- read.csv(&quot;data/ici.vs.pbs_marker.freq_low.geno.freq.removed_geno.ratio_7.batches_myo_mis.csv&quot;)
#nrow(dismark)
#names(dismark)[1] &lt;- c(&quot;marker&quot;)
#dismark &lt;- dismark[!dismark$Include,]
#nrow(dismark)

#gm_qc_dis &lt;- drop_markers(gm_qc, dismark$marker)
#gm_qc_dis &lt;- drop_nullmarkers(gm_qc_dis)

#gm = gm_qc_dis
#gm

markers &lt;- marker_names(gm)
gmapdf &lt;- read.csv(&quot;data/genetic_map_7.batches_myo.csv&quot;)
pmapdf &lt;- read.csv(&quot;data/physical_map_7.batches_myo.csv&quot;)
#mapdf &lt;- merge(gmapdf,pmapdf, by=c(&quot;marker&quot;,&quot;chr&quot;), all=T)
#rownames(mapdf) &lt;- mapdf$marker
#mapdf &lt;- mapdf[markers,]
#names(mapdf) &lt;- c(&#39;marker&#39;,&#39;chr&#39;,&#39;gmapdf&#39;,&#39;pmapdf&#39;)
#mapdfnd &lt;- mapdf[!duplicated(mapdf[c(2:3)]),]

pr.qc &lt;- calc_genoprob(gm)

colnames(covars) &lt;- gsub(&quot; &quot;, &quot;.&quot;, colnames(covars))</code></pre>
</div>
<div id="genome-wide-scan" class="section level2">
<h2>Genome-wide scan</h2>
<p>For each of the phenotype analyzed, permutations were used for each model to obtain genome-wide LOD significance threshold for p &lt; 0.01, p &lt; 0.05, p &lt; 0.10, respectively, separately for X and automsomes (A).</p>
<p>The table shows the estimated significance thresholds from permutation test.</p>
<p>We also looked at the kinship to see how correlated each sample is. Kinship values between pairs of samples range between 0 (no relationship) and 1.0 (completely identical). The darker the colour the more indentical the pairs are.</p>
<pre class="r"><code>#Xcovar &lt;- get_x_covar(gm)
addcovar = model.matrix(~sex+age.of.onset+Histology.Score, data = covars)[,-1]
covars$ici_vs_pbs= as.numeric(covars$ici_vs_pbs)

kinship &lt;- calc_kinship(pr.qc)
heatmap(kinship)</code></pre>
<p><img src="figure/4.1.1_qtl.analysis_binary_ici.vs.pbs_snpsqc_7.batches_myo_mis.Rmd/permutation-1.png" width="912" style="display: block; margin: auto;" /></p>
<pre class="r"><code>operm &lt;- scan1perm(pr.qc, covars[&quot;ici_vs_pbs&quot;], model=&quot;binary&quot;, addcovar=addcovar, n_perm=1000, perm_Xsp=TRUE, chr_lengths=chr_lengths(gm$gmap))

summary_table&lt;-data.frame(unclass(summary(operm, alpha=c(0.01,  0.05, 0.1))))
names(summary_table) &lt;- c(&quot;autosomes&quot;,&quot;X&quot;)
summary_table$significance.level &lt;- rownames(summary_table)

rownames(summary_table) &lt;- NULL

summary_table[c(3,1:2)] %&gt;%
  kable(escape = F,align = c(&quot;ccc&quot;)) %&gt;%
  kable_styling(&quot;striped&quot;, full_width = T) %&gt;%
  column_spec(1, bold=TRUE)</code></pre>
<table class="table table-striped" style="margin-left: auto; margin-right: auto;">
<thead>
<tr>
<th style="text-align:center;">
significance.level
</th>
<th style="text-align:center;">
autosomes
</th>
<th style="text-align:center;">
X
</th>
</tr>
</thead>
<tbody>
<tr>
<td style="text-align:center;font-weight: bold;">
0.01
</td>
<td style="text-align:center;">
5.041473
</td>
<td style="text-align:center;">
5.128848
</td>
</tr>
<tr>
<td style="text-align:center;font-weight: bold;">
0.05
</td>
<td style="text-align:center;">
3.855353
</td>
<td style="text-align:center;">
4.463416
</td>
</tr>
<tr>
<td style="text-align:center;font-weight: bold;">
0.1
</td>
<td style="text-align:center;">
3.520573
</td>
<td style="text-align:center;">
4.077599
</td>
</tr>
</tbody>
</table>
<p>The figures below show QTL maps for each phenotype</p>
<pre class="r"><code>#out &lt;- scan1(pr.qc, covars[&quot;ici_vs_pbs&quot;], Xcovar=Xcovar, model=&quot;binary&quot;)
out &lt;- scan1(pr.qc, covars[&quot;ici_vs_pbs&quot;], model=&quot;binary&quot;,addcovar=addcovar)

summary_table&lt;-data.frame(unclass(summary(operm, alpha=c(0.01,  0.05, 0.1))))


plot_lod&lt;-function(out,map){
  for (i in 1:dim(out)[2]){
    #png(filename=paste0(&quot;/Users/chenm/Documents/qtl/Jai/&quot;,colnames(out)[i],  &quot;_lod.png&quot;))
    
    ymx &lt;- maxlod(out) # overall maximum LOD score
    plot(out, map, lodcolumn=i, col=&quot;slateblue&quot;, ylim=c(0, ymx+0.5))
    #legend(&quot;topright&quot;, lwd=2, colnames(out)[i], bg=&quot;gray90&quot;)
    title(main = paste0(colnames(out)[i], &quot; [positions in cM]&quot;))
    add_threshold(map,  summary(operm,alpha=0.1), col = &#39;purple&#39;)
    add_threshold(map,  summary(operm, alpha=0.05), col = &#39;red&#39;)
    add_threshold(map,  summary(operm, alpha=0.01), col = &#39;blue&#39;)

    ##par(mar=c(5.1, 6.1, 1.1, 1.1))
    #ymx &lt;- 11 # overall maximum LOD score
    #plot(out, map, lodcolumn=i, col=&quot;slateblue&quot;, ylim=c(0, ymx+0.5))
    ##legend(&quot;topright&quot;, lwd=2, colnames(out)[i], bg=&quot;gray90&quot;)
    #title(main = paste0(colnames(out)[i], &quot; [positions in cM] \n(using same scale as eoi vs ici for easier comparison)&quot;))
    #add_threshold(map,  summary(operm, alpha=0.1), col = &#39;purple&#39;)
    #add_threshold(map,  summary(operm, alpha=0.05), col = &#39;red&#39;)
    #add_threshold(map,  summary(operm, alpha=0.01), col = &#39;blue&#39;)
    ##for (j in 1: dim(summary_table)[1]){
    ##  abline(h=summary_table[j, i],col=&quot;red&quot;)
    ##  text(x=400, y =summary_table[j, i]+0.12, labels = paste(&quot;p=&quot;, row.names(summary_table)[j]))
    ##}
    ##dev.off()
  }
}

plot_lod(out,gm$gmap)</code></pre>
<p><img src="figure/4.1.1_qtl.analysis_binary_ici.vs.pbs_snpsqc_7.batches_myo_mis.Rmd/Genome-wide%20scan-1.png" width="912" style="display: block; margin: auto;" /></p>
</div>
<div id="lod-peaks" class="section level2">
<h2>LOD peaks</h2>
<p>The table below shows QTL peaks associated with the phenotype. We use the 95% threshold from the permutations to find peaks.</p>
<div id="centimorgan-cm" class="section level3">
<h3>Centimorgan (cM)</h3>
<pre class="r"><code>peaks &lt;- find_peaks(out, gm$gmap, threshold=summary(operm,alpha=0.05)$A, thresholdX = summary(operm,alpha=0.05)$X, peakdrop=3, drop=1.5)

if(nrow(peaks) &gt;0){
peaks$marker &lt;- find_marker(gm$gmap, chr=peaks$chr,pos=peaks$pos)
names(peaks)[2] &lt;- c(&quot;phenotype&quot;)
peaks &lt;- peaks[-1]

rownames(peaks) &lt;- NULL
print(kable(peaks, escape = F, align = c(&quot;cccccccc&quot;), &quot;html&quot;) 
  %&gt;% kable_styling(&quot;striped&quot;, full_width = T)%&gt;%
  column_spec(1, bold=TRUE)
  )

#plot only peak chromosomes

plot_lod_chr&lt;-function(out,map,chrom){
  for (i in 1:dim(out)[2]){
    #png(filename=paste0(&quot;/Users/chenm/Documents/qtl/Jai/&quot;,colnames(out)[i],  &quot;_lod.png&quot;))
    
    #par(mar=c(5.1, 6.1, 1.1, 1.1))
    ymx &lt;- maxlod(out) # overall maximum LOD score
    plot(out, map, chr = chrom, lodcolumn=i, col=&quot;slateblue&quot;, ylim=c(0, ymx+0.5))
    #legend(&quot;topright&quot;, lwd=2, colnames(out)[i], bg=&quot;gray90&quot;)
    title(main = paste0(colnames(out)[i], &quot; - chr&quot;, chrom, &quot; [positions in cM]&quot;))
    add_threshold(map,  summary(operm,alpha=0.1), col = &#39;purple&#39;)
    add_threshold(map,  summary(operm, alpha=0.05), col = &#39;red&#39;)
    add_threshold(map,  summary(operm, alpha=0.01), col = &#39;blue&#39;)
    #for (j in 1: dim(summary_table)[1]){
    #  abline(h=summary_table[j, i],col=&quot;red&quot;)
    #  text(x=400, y =summary_table[j, i]+0.12, labels = paste(&quot;p=&quot;, row.names(summary_table)[j]))
    #}
    #dev.off()

    
    #ymx &lt;- 11
    #plot(out, map, chr = chrom, lodcolumn=i, col=&quot;slateblue&quot;, ylim=c(0, ymx+0.5))
    ##legend(&quot;topright&quot;, lwd=2, colnames(out)[i], bg=&quot;gray90&quot;)
    #title(main = paste0(colnames(out)[i], &quot; - chr&quot;, chrom, &quot; [positions in cM]\n(using same scale as eoi vs. ici for easier comparison)&quot;))
    #add_threshold(map,  summary(operm,alpha=0.1), col = &#39;purple&#39;)
    #add_threshold(map,  summary(operm, alpha=0.05), col = &#39;red&#39;)
    #add_threshold(map,  summary(operm, alpha=0.01), col = &#39;blue&#39;)

  }
}


for(i in unique(peaks$chr)){
#for (i in 1:nrow(peaks)){
  #plot_lod_chr(out,gm$gmap, peaks$chr[i])
  plot_lod_chr(out,gm$gmap, i)
}

} else {
  print(paste0(&quot;There are no peaks that have a LOD that reaches suggestive (p&lt;0.05) level of &quot;,summary(operm,alpha=0.05)$A, &quot; [autosomes]/&quot;,summary(operm,alpha=0.05)$X, &quot; [x-chromosome]&quot;))
}</code></pre>
<table class="table table-striped" style="margin-left: auto; margin-right: auto;">
<thead>
<tr>
<th style="text-align:center;">
phenotype
</th>
<th style="text-align:center;">
chr
</th>
<th style="text-align:center;">
pos
</th>
<th style="text-align:center;">
lod
</th>
<th style="text-align:center;">
ci_lo
</th>
<th style="text-align:center;">
ci_hi
</th>
<th style="text-align:center;">
marker
</th>
</tr>
</thead>
<tbody>
<tr>
<td style="text-align:center;font-weight: bold;">
ici_vs_pbs
</td>
<td style="text-align:center;">
7
</td>
<td style="text-align:center;">
51.658
</td>
<td style="text-align:center;">
4.766357
</td>
<td style="text-align:center;">
49.015
</td>
<td style="text-align:center;">
55.93
</td>
<td style="text-align:center;">
UNC13381931
</td>
</tr>
</tbody>
</table>
<p><img src="figure/4.1.1_qtl.analysis_binary_ici.vs.pbs_snpsqc_7.batches_myo_mis.Rmd/LOD%20peaks-1.png" width="912" style="display: block; margin: auto;" /></p>
</div>
<div id="megabase-mb" class="section level3">
<h3>Megabase (MB)</h3>
<pre class="r"><code>print(&quot;peaks in MB positions&quot;)</code></pre>
<p>[1] “peaks in MB positions”</p>
<pre class="r"><code>peaks_mba &lt;- find_peaks(out, gm$pmap, threshold=summary(operm,alpha=0.05)$A, thresholdX = summary(operm,alpha=0.05)$X, peakdrop=3, drop=1.5)

if(nrow(peaks) &gt;0){
peaks_mba$marker &lt;- find_marker(gm$pmap, chr=peaks_mba$chr,pos=peaks_mba$pos)
names(peaks_mba)[2] &lt;- c(&quot;phenotype&quot;)
peaks_mba &lt;- peaks_mba[-1]


rownames(peaks_mba) &lt;- NULL
print(kable(peaks_mba, escape = F, align = c(&quot;cccccccc&quot;), &quot;html&quot;) 
  %&gt;% kable_styling(&quot;striped&quot;, full_width = T)%&gt;%
  column_spec(1, bold=TRUE)
  )

plot_lod_chr_mb&lt;-function(out,map,chrom){
  for (i in 1:dim(out)[2]){
    #png(filename=paste0(&quot;/Users/chenm/Documents/qtl/Jai/&quot;,colnames(out)[i],  &quot;_lod.png&quot;))
    
    #par(mar=c(5.1, 6.1, 1.1, 1.1))
    ymx &lt;- maxlod(out) # overall maximum LOD score
    plot(out, map, chr = chrom, lodcolumn=i, col=&quot;slateblue&quot;, ylim=c(0, ymx+0.5))
    #legend(&quot;topright&quot;, lwd=2, colnames(out)[i], bg=&quot;gray90&quot;)
    title(main = paste0(colnames(out)[i], &quot; - chr&quot;, chrom, &quot; [positions in MB]&quot;))
    add_threshold(map,  summary(operm,alpha=0.1), col = &#39;purple&#39;)
    add_threshold(map,  summary(operm, alpha=0.05), col = &#39;red&#39;)
    add_threshold(map,  summary(operm, alpha=0.01), col = &#39;blue&#39;)
    #for (j in 1: dim(summary_table)[1]){
    #  abline(h=summary_table[j, i],col=&quot;red&quot;)
    #  text(x=400, y =summary_table[j, i]+0.12, labels = paste(&quot;p=&quot;, row.names(summary_table)[j]))
    #}
    #dev.off()

    #ymx &lt;- 11
    #plot(out, map, chr = chrom, lodcolumn=i, col=&quot;slateblue&quot;, ylim=c(0, ymx+0.5))
    ##legend(&quot;topright&quot;, lwd=2, colnames(out)[i], bg=&quot;gray90&quot;)
    #title(main = paste0(colnames(out)[i], &quot; - chr&quot;, chrom, &quot; [positions in MB]\n(using same scale as eoi vs. ici for easier comparison)&quot;))
    #add_threshold(map,  summary(operm,alpha=0.1), col = &#39;purple&#39;)
    #add_threshold(map,  summary(operm, alpha=0.05), col = &#39;red&#39;)
    #add_threshold(map,  summary(operm, alpha=0.01), col = &#39;blue&#39;)


  }
}

for(i in unique(peaks_mba$chr)){
#for (i in 1:nrow(peaks_mba)){
  #plot_lod_chr_mb(out,gm$pmap, peaks_mba$chr[i])
  plot_lod_chr_mb(out,gm$pmap,i)
}

} else {
  print(paste0(&quot;There are no peaks that have a LOD that reaches suggestive (p&lt;0.05) level of &quot;,summary(operm,alpha=0.05)$A, &quot; [autosomes]/&quot;,summary(operm,alpha=0.05)$X, &quot; [x-chromosome]&quot;))
}</code></pre>
<table class="table table-striped" style="margin-left: auto; margin-right: auto;">
<thead>
<tr>
<th style="text-align:center;">
phenotype
</th>
<th style="text-align:center;">
chr
</th>
<th style="text-align:center;">
pos
</th>
<th style="text-align:center;">
lod
</th>
<th style="text-align:center;">
ci_lo
</th>
<th style="text-align:center;">
ci_hi
</th>
<th style="text-align:center;">
marker
</th>
</tr>
</thead>
<tbody>
<tr>
<td style="text-align:center;font-weight: bold;">
ici_vs_pbs
</td>
<td style="text-align:center;">
7
</td>
<td style="text-align:center;">
93.10503
</td>
<td style="text-align:center;">
4.766357
</td>
<td style="text-align:center;">
87.72927
</td>
<td style="text-align:center;">
105.688
</td>
<td style="text-align:center;">
UNC13381931
</td>
</tr>
</tbody>
</table>
<p><img src="figure/4.1.1_qtl.analysis_binary_ici.vs.pbs_snpsqc_7.batches_myo_mis.Rmd/LOD%20peaks_mba-1.png" width="912" style="display: block; margin: auto;" /></p>
</div>
</div>
<div id="qtl-effects" class="section level2">
<h2>QTL effects</h2>
<p>For each peak LOD location we give a list of gene</p>
<pre class="r"><code>query_variants &lt;- create_variant_query_func(&quot;code/cc_variants.sqlite&quot;)
query_genes &lt;- create_gene_query_func(&quot;code/mouse_genes_mgi.sqlite&quot;)

if(nrow(peaks) &gt;0){
for (i in 1:nrow(peaks)){


  g &lt;- maxmarg(pr.qc, gm$gmap, chr=peaks$chr[i], pos=peaks$pos[i], return_char=TRUE)
  #png(filename=paste0(&quot;/Users/chenm/Documents/qtl/Jai/&quot;,&quot;qtl_effect_&quot;, i, &quot;.png&quot;))
  #par(mar=c(4.1, 4.1, 1.5, 0.6))
  plot_pxg(g, covars[,peaks$phenotype[i]], ylab=peaks$phenotype[i], sort=FALSE)
  title(main = paste0(&quot;chr: &quot;, chr=peaks$chr[i],&quot;; pos: &quot;, peaks$pos[i], &quot;cM /&quot;,peaks_mba$pos[i],&quot;MB\n(&quot;,peaks$phenotype[i],&quot; )&quot;), line=0.2)
  ##dev.off()

  chr = peaks$chr[i]

# Plot 2
  pr_sub &lt;- pull_genoprobint(pr.qc, gm$gmap, chr, c(peaks$ci_lo[i], peaks$ci_hi[i]))
  blup &lt;- scan1blup(pr.qc[,chr], covars[peaks$phenotype[i]],addcovar = addcovar)
  blup_sub &lt;- scan1blup(pr_sub[,chr], covars[peaks$phenotype[i]], addcovar = addcovar)

  write.csv(as.data.frame(blup_sub), paste0(&quot;data/ici.vs.pbs_blup_sub_chr-&quot;,chr,&quot;_peak.marker-&quot;,peaks$marker[i],&quot;_lod.drop-1.5_snpsqc_7.batches_myo_mis.csv&quot;), quote=F)

  plot_coef(blup, 
       gm$gmap, columns=1:2,
       bgcolor=&quot;gray95&quot;, legend=&quot;bottomleft&quot;, 
       main = paste0(&quot;chr: &quot;, chr=peaks$chr[i],&quot;; pos: &quot;, peaks$pos[i], &quot;cM /&quot;,peaks_mba$pos[i],&quot;MB\n(&quot;,peaks$phenotype[i],&quot; [scan1blup; positions in cM])&quot;)
       )

  plot_coef(blup_sub, 
       gm$gmap, columns=1:2,
       bgcolor=&quot;gray95&quot;, legend=&quot;bottomleft&quot;, 
       main = paste0(&quot;chr: &quot;, chr=peaks$chr[i],&quot;; pos: &quot;, peaks$pos[i], &quot;cM /&quot;,peaks_mba$pos[i],&quot;MB\n(&quot;,peaks$phenotype[i],&quot;; 1.5 LOD drop interval [scan1blup; positions in cM])&quot;)
       )

  #Table 1
  chr = peaks_mba$chr[i]
  start=as.numeric(peaks_mba$ci_lo[i])
  end=as.numeric(peaks_mba$ci_hi[i])

  genesgss = query_genes(chr, start, end)

  write.csv(genesgss, file=paste0(&quot;data/ici.vs.pbs_genes_chr-&quot;,chr,&quot;_peak.marker-&quot;,peaks$marker[i],&quot;_lod.drop-1.5_snpsqc_7.batches_myo_mis.csv&quot;), quote=F)

  rownames(genesgss) &lt;- NULL
  genesgss$strand_old = genesgss$strand
  genesgss$strand[genesgss$strand==&quot;+&quot;] &lt;- &quot;positive&quot;
  genesgss$strand[genesgss$strand==&quot;-&quot;] &lt;- &quot;negative&quot;

  print(kable(genesgss[,c(&quot;chr&quot;,&quot;type&quot;,&quot;start&quot;,&quot;stop&quot;,&quot;strand&quot;,&quot;ID&quot;,&quot;Name&quot;,&quot;Dbxref&quot;,&quot;gene_id&quot;,&quot;mgi_type&quot;,&quot;description&quot;)], &quot;html&quot;) %&gt;% kable_styling(&quot;striped&quot;, full_width = T))


}

} else {
  print(paste0(&quot;There are no peaks that have a LOD that reaches suggestive (p&lt;0.05) level of &quot;,summary(operm,alpha=0.05)$A, &quot; [autosomes]/&quot;,summary(operm,alpha=0.05)$X, &quot; [x-chromosome]&quot;))
}</code></pre>
<img src="figure/4.1.1_qtl.analysis_binary_ici.vs.pbs_snpsqc_7.batches_myo_mis.Rmd/QTL%20effects-1.png" width="912" style="display: block; margin: auto;" /><img src="figure/4.1.1_qtl.analysis_binary_ici.vs.pbs_snpsqc_7.batches_myo_mis.Rmd/QTL%20effects-2.png" width="912" style="display: block; margin: auto;" /><img src="figure/4.1.1_qtl.analysis_binary_ici.vs.pbs_snpsqc_7.batches_myo_mis.Rmd/QTL%20effects-3.png" width="912" style="display: block; margin: auto;" />
<table class="table table-striped" style="margin-left: auto; margin-right: auto;">
<thead>
<tr>
<th style="text-align:left;">
chr
</th>
<th style="text-align:left;">
type
</th>
<th style="text-align:right;">
start
</th>
<th style="text-align:right;">
stop
</th>
<th style="text-align:left;">
strand
</th>
<th style="text-align:left;">
ID
</th>
<th style="text-align:left;">
Name
</th>
<th style="text-align:left;">
Dbxref
</th>
<th style="text-align:left;">
gene_id
</th>
<th style="text-align:left;">
mgi_type
</th>
<th style="text-align:left;">
description
</th>
</tr>
</thead>
<tbody>
<tr>
<td style="text-align:left;">
7
</td>
<td style="text-align:left;">
gene
</td>
<td style="text-align:right;">
87.58398
</td>
<td style="text-align:right;">
88.13506
</td>
<td style="text-align:left;">
positive
</td>
<td style="text-align:left;">
MGI_C57BL6J_1351342
</td>
<td style="text-align:left;">
Grm5
</td>
<td style="text-align:left;">
NCBI_Gene:108071,ENSEMBL:ENSMUSG00000049583
</td>
<td style="text-align:left;">
MGI:1351342
</td>
<td style="text-align:left;">
protein coding gene
</td>
<td style="text-align:left;">
glutamate receptor, metabotropic 5
</td>
</tr>
<tr>
<td style="text-align:left;">
7
</td>
<td style="text-align:left;">
gene
</td>
<td style="text-align:right;">
87.77257
</td>
<td style="text-align:right;">
87.77439
</td>
<td style="text-align:left;">
positive
</td>
<td style="text-align:left;">
MGI_C57BL6J_5753502
</td>
<td style="text-align:left;">
Gm44926
</td>
<td style="text-align:left;">
ENSEMBL:ENSMUSG00000108983
</td>
<td style="text-align:left;">
MGI:5753502
</td>
<td style="text-align:left;">
unclassified gene
</td>
<td style="text-align:left;">
predicted gene 44926
</td>
</tr>
<tr>
<td style="text-align:left;">
7
</td>
<td style="text-align:left;">
gene
</td>
<td style="text-align:right;">
87.89075
</td>
<td style="text-align:right;">
87.89146
</td>
<td style="text-align:left;">
positive
</td>
<td style="text-align:left;">
MGI_C57BL6J_5753503
</td>
<td style="text-align:left;">
Gm44927
</td>
<td style="text-align:left;">
ENSEMBL:ENSMUSG00000108893
</td>
<td style="text-align:left;">
MGI:5753503
</td>
<td style="text-align:left;">
unclassified gene
</td>
<td style="text-align:left;">
predicted gene 44927
</td>
</tr>
<tr>
<td style="text-align:left;">
7
</td>
<td style="text-align:left;">
gene
</td>
<td style="text-align:right;">
88.06306
</td>
<td style="text-align:right;">
88.06629
</td>
<td style="text-align:left;">
positive
</td>
<td style="text-align:left;">
MGI_C57BL6J_5753424
</td>
<td style="text-align:left;">
Gm44848
</td>
<td style="text-align:left;">
ENSEMBL:ENSMUSG00000109461
</td>
<td style="text-align:left;">
MGI:5753424
</td>
<td style="text-align:left;">
unclassified gene
</td>
<td style="text-align:left;">
predicted gene 44848
</td>
</tr>
<tr>
<td style="text-align:left;">
7
</td>
<td style="text-align:left;">
gene
</td>
<td style="text-align:right;">
88.08281
</td>
<td style="text-align:right;">
88.08480
</td>
<td style="text-align:left;">
positive
</td>
<td style="text-align:left;">
MGI_C57BL6J_5753423
</td>
<td style="text-align:left;">
Gm44847
</td>
<td style="text-align:left;">
ENSEMBL:ENSMUSG00000108890
</td>
<td style="text-align:left;">
MGI:5753423
</td>
<td style="text-align:left;">
unclassified gene
</td>
<td style="text-align:left;">
predicted gene 44847
</td>
</tr>
<tr>
<td style="text-align:left;">
7
</td>
<td style="text-align:left;">
gene
</td>
<td style="text-align:right;">
88.08719
</td>
<td style="text-align:right;">
88.08725
</td>
<td style="text-align:left;">
negative
</td>
<td style="text-align:left;">
MGI_C57BL6J_5454026
</td>
<td style="text-align:left;">
Gm24249
</td>
<td style="text-align:left;">
ENSEMBL:ENSMUSG00000087947
</td>
<td style="text-align:left;">
MGI:5454026
</td>
<td style="text-align:left;">
snRNA gene
</td>
<td style="text-align:left;">
predicted gene, 24249
</td>
</tr>
<tr>
<td style="text-align:left;">
7
</td>
<td style="text-align:left;">
gene
</td>
<td style="text-align:right;">
88.11916
</td>
<td style="text-align:right;">
88.12240
</td>
<td style="text-align:left;">
positive
</td>
<td style="text-align:left;">
MGI_C57BL6J_5753422
</td>
<td style="text-align:left;">
Gm44846
</td>
<td style="text-align:left;">
ENSEMBL:ENSMUSG00000108993
</td>
<td style="text-align:left;">
MGI:5753422
</td>
<td style="text-align:left;">
unclassified gene
</td>
<td style="text-align:left;">
predicted gene 44846
</td>
</tr>
<tr>
<td style="text-align:left;">
7
</td>
<td style="text-align:left;">
gene
</td>
<td style="text-align:right;">
88.27802
</td>
<td style="text-align:right;">
88.31586
</td>
<td style="text-align:left;">
positive
</td>
<td style="text-align:left;">
MGI_C57BL6J_109553
</td>
<td style="text-align:left;">
Ctsc
</td>
<td style="text-align:left;">
NCBI_Gene:13032,ENSEMBL:ENSMUSG00000030560
</td>
<td style="text-align:left;">
MGI:109553
</td>
<td style="text-align:left;">
protein coding gene
</td>
<td style="text-align:left;">
cathepsin C
</td>
</tr>
<tr>
<td style="text-align:left;">
7
</td>
<td style="text-align:left;">
gene
</td>
<td style="text-align:right;">
88.27879
</td>
<td style="text-align:right;">
88.28353
</td>
<td style="text-align:left;">
negative
</td>
<td style="text-align:left;">
MGI_C57BL6J_5477016
</td>
<td style="text-align:left;">
Gm26522
</td>
<td style="text-align:left;">
ENSEMBL:ENSMUSG00000097423
</td>
<td style="text-align:left;">
MGI:5477016
</td>
<td style="text-align:left;">
lncRNA gene
</td>
<td style="text-align:left;">
predicted gene, 26522
</td>
</tr>
<tr>
<td style="text-align:left;">
7
</td>
<td style="text-align:left;">
gene
</td>
<td style="text-align:right;">
88.31148
</td>
<td style="text-align:right;">
88.31586
</td>
<td style="text-align:left;">
positive
</td>
<td style="text-align:left;">
MGI_C57BL6J_5753327
</td>
<td style="text-align:left;">
Gm44751
</td>
<td style="text-align:left;">
ENSEMBL:ENSMUSG00000109244
</td>
<td style="text-align:left;">
MGI:5753327
</td>
<td style="text-align:left;">
lncRNA gene
</td>
<td style="text-align:left;">
predicted gene 44751
</td>
</tr>
<tr>
<td style="text-align:left;">
7
</td>
<td style="text-align:left;">
gene
</td>
<td style="text-align:right;">
88.34397
</td>
<td style="text-align:right;">
88.34410
</td>
<td style="text-align:left;">
positive
</td>
<td style="text-align:left;">
MGI_C57BL6J_5455122
</td>
<td style="text-align:left;">
Gm25345
</td>
<td style="text-align:left;">
ENSEMBL:ENSMUSG00000077693
</td>
<td style="text-align:left;">
MGI:5455122
</td>
<td style="text-align:left;">
snoRNA gene
</td>
<td style="text-align:left;">
predicted gene, 25345
</td>
</tr>
<tr>
<td style="text-align:left;">
7
</td>
<td style="text-align:left;">
gene
</td>
<td style="text-align:right;">
88.34815
</td>
<td style="text-align:right;">
88.34906
</td>
<td style="text-align:left;">
negative
</td>
<td style="text-align:left;">
MGI_C57BL6J_1924892
</td>
<td style="text-align:left;">
C330001P17Rik
</td>
<td style="text-align:left;">
NA
</td>
<td style="text-align:left;">
NA
</td>
<td style="text-align:left;">
unclassified gene
</td>
<td style="text-align:left;">
RIKEN cDNA C330001P17 gene
</td>
</tr>
<tr>
<td style="text-align:left;">
7
</td>
<td style="text-align:left;">
pseudogene
</td>
<td style="text-align:right;">
88.34931
</td>
<td style="text-align:right;">
88.35659
</td>
<td style="text-align:left;">
negative
</td>
<td style="text-align:left;">
MGI_C57BL6J_3780575
</td>
<td style="text-align:left;">
Gm2407
</td>
<td style="text-align:left;">
NCBI_Gene:100039756
</td>
<td style="text-align:left;">
MGI:3780575
</td>
<td style="text-align:left;">
pseudogene
</td>
<td style="text-align:left;">
predicted gene 2407
</td>
</tr>
<tr>
<td style="text-align:left;">
7
</td>
<td style="text-align:left;">
gene
</td>
<td style="text-align:right;">
88.43027
</td>
<td style="text-align:right;">
88.49157
</td>
<td style="text-align:left;">
positive
</td>
<td style="text-align:left;">
MGI_C57BL6J_1919683
</td>
<td style="text-align:left;">
Rab38
</td>
<td style="text-align:left;">
NCBI_Gene:72433,ENSEMBL:ENSMUSG00000030559
</td>
<td style="text-align:left;">
MGI:1919683
</td>
<td style="text-align:left;">
protein coding gene
</td>
<td style="text-align:left;">
RAB38, member RAS oncogene family
</td>
</tr>
<tr>
<td style="text-align:left;">
7
</td>
<td style="text-align:left;">
pseudogene
</td>
<td style="text-align:right;">
88.44867
</td>
<td style="text-align:right;">
88.44923
</td>
<td style="text-align:left;">
negative
</td>
<td style="text-align:left;">
MGI_C57BL6J_3783103
</td>
<td style="text-align:left;">
Gm15661
</td>
<td style="text-align:left;">
ENSEMBL:ENSMUSG00000080748
</td>
<td style="text-align:left;">
MGI:3783103
</td>
<td style="text-align:left;">
pseudogene
</td>
<td style="text-align:left;">
predicted gene 15661
</td>
</tr>
<tr>
<td style="text-align:left;">
7
</td>
<td style="text-align:left;">
pseudogene
</td>
<td style="text-align:right;">
88.53040
</td>
<td style="text-align:right;">
88.53092
</td>
<td style="text-align:left;">
negative
</td>
<td style="text-align:left;">
MGI_C57BL6J_3704295
</td>
<td style="text-align:left;">
Rps13-ps2
</td>
<td style="text-align:left;">
NCBI_Gene:100039924,ENSEMBL:ENSMUSG00000069972
</td>
<td style="text-align:left;">
MGI:3704295
</td>
<td style="text-align:left;">
pseudogene
</td>
<td style="text-align:left;">
ribosomal protein S13, pseudogene 2
</td>
</tr>
<tr>
<td style="text-align:left;">
7
</td>
<td style="text-align:left;">
pseudogene
</td>
<td style="text-align:right;">
88.57167
</td>
<td style="text-align:right;">
88.57286
</td>
<td style="text-align:left;">
negative
</td>
<td style="text-align:left;">
MGI_C57BL6J_3643325
</td>
<td style="text-align:left;">
Gm8183
</td>
<td style="text-align:left;">
NCBI_Gene:666598,ENSEMBL:ENSMUSG00000098174
</td>
<td style="text-align:left;">
MGI:3643325
</td>
<td style="text-align:left;">
pseudogene
</td>
<td style="text-align:left;">
predicted gene 8183
</td>
</tr>
<tr>
<td style="text-align:left;">
7
</td>
<td style="text-align:left;">
pseudogene
</td>
<td style="text-align:right;">
88.78192
</td>
<td style="text-align:right;">
88.78240
</td>
<td style="text-align:left;">
negative
</td>
<td style="text-align:left;">
MGI_C57BL6J_3648708
</td>
<td style="text-align:left;">
Gm6240
</td>
<td style="text-align:left;">
NCBI_Gene:621562,ENSEMBL:ENSMUSG00000109135
</td>
<td style="text-align:left;">
MGI:3648708
</td>
<td style="text-align:left;">
pseudogene
</td>
<td style="text-align:left;">
predicted gene 6240
</td>
</tr>
<tr>
<td style="text-align:left;">
7
</td>
<td style="text-align:left;">
gene
</td>
<td style="text-align:right;">
88.85304
</td>
<td style="text-align:right;">
88.86489
</td>
<td style="text-align:left;">
negative
</td>
<td style="text-align:left;">
MGI_C57BL6J_5753720
</td>
<td style="text-align:left;">
Gm45144
</td>
<td style="text-align:left;">
ENSEMBL:ENSMUSG00000109278
</td>
<td style="text-align:left;">
MGI:5753720
</td>
<td style="text-align:left;">
lncRNA gene
</td>
<td style="text-align:left;">
predicted gene 45144
</td>
</tr>
<tr>
<td style="text-align:left;">
7
</td>
<td style="text-align:left;">
gene
</td>
<td style="text-align:right;">
89.13971
</td>
<td style="text-align:right;">
89.40427
</td>
<td style="text-align:left;">
negative
</td>
<td style="text-align:left;">
MGI_C57BL6J_1920009
</td>
<td style="text-align:left;">
Tmem135
</td>
<td style="text-align:left;">
NCBI_Gene:72759,ENSEMBL:ENSMUSG00000039428
</td>
<td style="text-align:left;">
MGI:1920009
</td>
<td style="text-align:left;">
protein coding gene
</td>
<td style="text-align:left;">
transmembrane protein 135
</td>
</tr>
<tr>
<td style="text-align:left;">
7
</td>
<td style="text-align:left;">
gene
</td>
<td style="text-align:right;">
89.21511
</td>
<td style="text-align:right;">
89.21710
</td>
<td style="text-align:left;">
negative
</td>
<td style="text-align:left;">
MGI_C57BL6J_5753280
</td>
<td style="text-align:left;">
Gm44704
</td>
<td style="text-align:left;">
ENSEMBL:ENSMUSG00000109576
</td>
<td style="text-align:left;">
MGI:5753280
</td>
<td style="text-align:left;">
unclassified gene
</td>
<td style="text-align:left;">
predicted gene 44704
</td>
</tr>
<tr>
<td style="text-align:left;">
7
</td>
<td style="text-align:left;">
gene
</td>
<td style="text-align:right;">
89.29119
</td>
<td style="text-align:right;">
89.29130
</td>
<td style="text-align:left;">
negative
</td>
<td style="text-align:left;">
MGI_C57BL6J_5454436
</td>
<td style="text-align:left;">
Gm24659
</td>
<td style="text-align:left;">
ENSEMBL:ENSMUSG00000094780
</td>
<td style="text-align:left;">
MGI:5454436
</td>
<td style="text-align:left;">
miRNA gene
</td>
<td style="text-align:left;">
predicted gene, 24659
</td>
</tr>
<tr>
<td style="text-align:left;">
7
</td>
<td style="text-align:left;">
gene
</td>
<td style="text-align:right;">
89.29575
</td>
<td style="text-align:right;">
89.29586
</td>
<td style="text-align:left;">
negative
</td>
<td style="text-align:left;">
MGI_C57BL6J_5451948
</td>
<td style="text-align:left;">
Gm22171
</td>
<td style="text-align:left;">
ENSEMBL:ENSMUSG00000096631
</td>
<td style="text-align:left;">
MGI:5451948
</td>
<td style="text-align:left;">
miRNA gene
</td>
<td style="text-align:left;">
predicted gene, 22171
</td>
</tr>
<tr>
<td style="text-align:left;">
7
</td>
<td style="text-align:left;">
gene
</td>
<td style="text-align:right;">
89.33837
</td>
<td style="text-align:right;">
89.34629
</td>
<td style="text-align:left;">
positive
</td>
<td style="text-align:left;">
MGI_C57BL6J_5753571
</td>
<td style="text-align:left;">
Gm44995
</td>
<td style="text-align:left;">
ENSEMBL:ENSMUSG00000109327
</td>
<td style="text-align:left;">
MGI:5753571
</td>
<td style="text-align:left;">
lncRNA gene
</td>
<td style="text-align:left;">
predicted gene 44995
</td>
</tr>
<tr>
<td style="text-align:left;">
7
</td>
<td style="text-align:left;">
gene
</td>
<td style="text-align:right;">
89.38834
</td>
<td style="text-align:right;">
89.39132
</td>
<td style="text-align:left;">
negative
</td>
<td style="text-align:left;">
MGI_C57BL6J_5753572
</td>
<td style="text-align:left;">
Gm44996
</td>
<td style="text-align:left;">
ENSEMBL:ENSMUSG00000109199
</td>
<td style="text-align:left;">
MGI:5753572
</td>
<td style="text-align:left;">
unclassified gene
</td>
<td style="text-align:left;">
predicted gene 44996
</td>
</tr>
<tr>
<td style="text-align:left;">
7
</td>
<td style="text-align:left;">
gene
</td>
<td style="text-align:right;">
89.40435
</td>
<td style="text-align:right;">
89.41313
</td>
<td style="text-align:left;">
positive
</td>
<td style="text-align:left;">
MGI_C57BL6J_108520
</td>
<td style="text-align:left;">
Fzd4
</td>
<td style="text-align:left;">
NCBI_Gene:14366,ENSEMBL:ENSMUSG00000049791
</td>
<td style="text-align:left;">
MGI:108520
</td>
<td style="text-align:left;">
protein coding gene
</td>
<td style="text-align:left;">
frizzled class receptor 4
</td>
</tr>
<tr>
<td style="text-align:left;">
7
</td>
<td style="text-align:left;">
gene
</td>
<td style="text-align:right;">
89.42631
</td>
<td style="text-align:right;">
89.44193
</td>
<td style="text-align:left;">
positive
</td>
<td style="text-align:left;">
MGI_C57BL6J_2141898
</td>
<td style="text-align:left;">
AI314278
</td>
<td style="text-align:left;">
NCBI_Gene:101521,ENSEMBL:ENSMUSG00000109311
</td>
<td style="text-align:left;">
MGI:2141898
</td>
<td style="text-align:left;">
lncRNA gene
</td>
<td style="text-align:left;">
expressed sequence AI314278
</td>
</tr>
<tr>
<td style="text-align:left;">
7
</td>
<td style="text-align:left;">
gene
</td>
<td style="text-align:right;">
89.50778
</td>
<td style="text-align:right;">
89.52724
</td>
<td style="text-align:left;">
negative
</td>
<td style="text-align:left;">
MGI_C57BL6J_1923703
</td>
<td style="text-align:left;">
Prss23
</td>
<td style="text-align:left;">
NCBI_Gene:76453,ENSEMBL:ENSMUSG00000039405
</td>
<td style="text-align:left;">
MGI:1923703
</td>
<td style="text-align:left;">
protein coding gene
</td>
<td style="text-align:left;">
protease, serine 23
</td>
</tr>
<tr>
<td style="text-align:left;">
7
</td>
<td style="text-align:left;">
gene
</td>
<td style="text-align:right;">
89.51001
</td>
<td style="text-align:right;">
89.51187
</td>
<td style="text-align:left;">
positive
</td>
<td style="text-align:left;">
MGI_C57BL6J_1916654
</td>
<td style="text-align:left;">
Prss23os
</td>
<td style="text-align:left;">
NCBI_Gene:108167445,ENSEMBL:ENSMUSG00000030623
</td>
<td style="text-align:left;">
MGI:1916654
</td>
<td style="text-align:left;">
antisense lncRNA gene
</td>
<td style="text-align:left;">
protease, serine 23, opposite strand
</td>
</tr>
<tr>
<td style="text-align:left;">
7
</td>
<td style="text-align:left;">
gene
</td>
<td style="text-align:right;">
89.55237
</td>
<td style="text-align:right;">
89.56808
</td>
<td style="text-align:left;">
positive
</td>
<td style="text-align:left;">
MGI_C57BL6J_5753640
</td>
<td style="text-align:left;">
Gm45064
</td>
<td style="text-align:left;">
ENSEMBL:ENSMUSG00000108875
</td>
<td style="text-align:left;">
MGI:5753640
</td>
<td style="text-align:left;">
lncRNA gene
</td>
<td style="text-align:left;">
predicted gene 45064
</td>
</tr>
<tr>
<td style="text-align:left;">
7
</td>
<td style="text-align:left;">
pseudogene
</td>
<td style="text-align:right;">
89.60769
</td>
<td style="text-align:right;">
89.60889
</td>
<td style="text-align:left;">
positive
</td>
<td style="text-align:left;">
MGI_C57BL6J_3780714
</td>
<td style="text-align:left;">
Gm2546
</td>
<td style="text-align:left;">
NCBI_Gene:100040001,ENSEMBL:ENSMUSG00000098019
</td>
<td style="text-align:left;">
MGI:3780714
</td>
<td style="text-align:left;">
pseudogene
</td>
<td style="text-align:left;">
predicted gene 2546
</td>
</tr>
<tr>
<td style="text-align:left;">
7
</td>
<td style="text-align:left;">
gene
</td>
<td style="text-align:right;">
89.63210
</td>
<td style="text-align:right;">
89.63312
</td>
<td style="text-align:left;">
negative
</td>
<td style="text-align:left;">
MGI_C57BL6J_3642744
</td>
<td style="text-align:left;">
A230065N10Rik
</td>
<td style="text-align:left;">
ENSEMBL:ENSMUSG00000092071
</td>
<td style="text-align:left;">
MGI:3642744
</td>
<td style="text-align:left;">
lncRNA gene
</td>
<td style="text-align:left;">
RIKEN cDNA A230065N10 gene
</td>
</tr>
<tr>
<td style="text-align:left;">
7
</td>
<td style="text-align:left;">
gene
</td>
<td style="text-align:right;">
89.63220
</td>
<td style="text-align:right;">
89.85436
</td>
<td style="text-align:left;">
positive
</td>
<td style="text-align:left;">
MGI_C57BL6J_1916679
</td>
<td style="text-align:left;">
Me3
</td>
<td style="text-align:left;">
NCBI_Gene:109264,ENSEMBL:ENSMUSG00000030621
</td>
<td style="text-align:left;">
MGI:1916679
</td>
<td style="text-align:left;">
protein coding gene
</td>
<td style="text-align:left;">
malic enzyme 3, NADP(+)-dependent, mitochondrial
</td>
</tr>
<tr>
<td style="text-align:left;">
7
</td>
<td style="text-align:left;">
pseudogene
</td>
<td style="text-align:right;">
89.76141
</td>
<td style="text-align:right;">
89.76217
</td>
<td style="text-align:left;">
negative
</td>
<td style="text-align:left;">
MGI_C57BL6J_3783186
</td>
<td style="text-align:left;">
Gm15744
</td>
<td style="text-align:left;">
NCBI_Gene:100502804,ENSEMBL:ENSMUSG00000089938
</td>
<td style="text-align:left;">
MGI:3783186
</td>
<td style="text-align:left;">
pseudogene
</td>
<td style="text-align:left;">
predicted gene 15744
</td>
</tr>
<tr>
<td style="text-align:left;">
7
</td>
<td style="text-align:left;">
gene
</td>
<td style="text-align:right;">
89.82722
</td>
<td style="text-align:right;">
89.82856
</td>
<td style="text-align:left;">
positive
</td>
<td style="text-align:left;">
MGI_C57BL6J_5804908
</td>
<td style="text-align:left;">
Gm45793
</td>
<td style="text-align:left;">
ENSEMBL:ENSMUSG00000109150
</td>
<td style="text-align:left;">
MGI:5804908
</td>
<td style="text-align:left;">
unclassified gene
</td>
<td style="text-align:left;">
predicted gene 45793
</td>
</tr>
<tr>
<td style="text-align:left;">
7
</td>
<td style="text-align:left;">
gene
</td>
<td style="text-align:right;">
89.86615
</td>
<td style="text-align:right;">
89.90363
</td>
<td style="text-align:left;">
negative
</td>
<td style="text-align:left;">
MGI_C57BL6J_1918134
</td>
<td style="text-align:left;">
Ccdc81
</td>
<td style="text-align:left;">
NCBI_Gene:70884,ENSEMBL:ENSMUSG00000039391
</td>
<td style="text-align:left;">
MGI:1918134
</td>
<td style="text-align:left;">
protein coding gene
</td>
<td style="text-align:left;">
coiled-coil domain containing 81
</td>
</tr>
<tr>
<td style="text-align:left;">
7
</td>
<td style="text-align:left;">
gene
</td>
<td style="text-align:right;">
89.91753
</td>
<td style="text-align:right;">
89.94122
</td>
<td style="text-align:left;">
negative
</td>
<td style="text-align:left;">
MGI_C57BL6J_96738
</td>
<td style="text-align:left;">
Hikeshi
</td>
<td style="text-align:left;">
NCBI_Gene:67669,ENSEMBL:ENSMUSG00000062797
</td>
<td style="text-align:left;">
MGI:96738
</td>
<td style="text-align:left;">
protein coding gene
</td>
<td style="text-align:left;">
heat shock protein nuclear import factor
</td>
</tr>
<tr>
<td style="text-align:left;">
7
</td>
<td style="text-align:left;">
gene
</td>
<td style="text-align:right;">
89.95465
</td>
<td style="text-align:right;">
89.98142
</td>
<td style="text-align:left;">
negative
</td>
<td style="text-align:left;">
MGI_C57BL6J_95286
</td>
<td style="text-align:left;">
Eed
</td>
<td style="text-align:left;">
NCBI_Gene:13626,ENSEMBL:ENSMUSG00000030619
</td>
<td style="text-align:left;">
MGI:95286
</td>
<td style="text-align:left;">
protein coding gene
</td>
<td style="text-align:left;">
embryonic ectoderm development
</td>
</tr>
<tr>
<td style="text-align:left;">
7
</td>
<td style="text-align:left;">
gene
</td>
<td style="text-align:right;">
89.96499
</td>
<td style="text-align:right;">
89.96539
</td>
<td style="text-align:left;">
negative
</td>
<td style="text-align:left;">
MGI_C57BL6J_5753696
</td>
<td style="text-align:left;">
Gm45120
</td>
<td style="text-align:left;">
ENSEMBL:ENSMUSG00000109038
</td>
<td style="text-align:left;">
MGI:5753696
</td>
<td style="text-align:left;">
unclassified gene
</td>
<td style="text-align:left;">
predicted gene 45120
</td>
</tr>
<tr>
<td style="text-align:left;">
7
</td>
<td style="text-align:left;">
gene
</td>
<td style="text-align:right;">
89.98072
</td>
<td style="text-align:right;">
90.04907
</td>
<td style="text-align:left;">
positive
</td>
<td style="text-align:left;">
MGI_C57BL6J_2442580
</td>
<td style="text-align:left;">
E230029C05Rik
</td>
<td style="text-align:left;">
NCBI_Gene:319711,ENSEMBL:ENSMUSG00000097585
</td>
<td style="text-align:left;">
MGI:2442580
</td>
<td style="text-align:left;">
lncRNA gene
</td>
<td style="text-align:left;">
RIKEN cDNA E230029C05 gene
</td>
</tr>
<tr>
<td style="text-align:left;">
7
</td>
<td style="text-align:left;">
gene
</td>
<td style="text-align:right;">
90.03057
</td>
<td style="text-align:right;">
90.03292
</td>
<td style="text-align:left;">
positive
</td>
<td style="text-align:left;">
MGI_C57BL6J_5753436
</td>
<td style="text-align:left;">
Gm44860
</td>
<td style="text-align:left;">
ENSEMBL:ENSMUSG00000109245
</td>
<td style="text-align:left;">
MGI:5753436
</td>
<td style="text-align:left;">
unclassified gene
</td>
<td style="text-align:left;">
predicted gene 44860
</td>
</tr>
<tr>
<td style="text-align:left;">
7
</td>
<td style="text-align:left;">
gene
</td>
<td style="text-align:right;">
90.04270
</td>
<td style="text-align:right;">
90.04392
</td>
<td style="text-align:left;">
positive
</td>
<td style="text-align:left;">
MGI_C57BL6J_5753437
</td>
<td style="text-align:left;">
Gm44861
</td>
<td style="text-align:left;">
ENSEMBL:ENSMUSG00000108897
</td>
<td style="text-align:left;">
MGI:5753437
</td>
<td style="text-align:left;">
unclassified gene
</td>
<td style="text-align:left;">
predicted gene 44861
</td>
</tr>
<tr>
<td style="text-align:left;">
7
</td>
<td style="text-align:left;">
gene
</td>
<td style="text-align:right;">
90.04343
</td>
<td style="text-align:right;">
90.04752
</td>
<td style="text-align:left;">
negative
</td>
<td style="text-align:left;">
MGI_C57BL6J_5825630
</td>
<td style="text-align:left;">
Gm45993
</td>
<td style="text-align:left;">
NCBI_Gene:108167446
</td>
<td style="text-align:left;">
MGI:5825630
</td>
<td style="text-align:left;">
lncRNA gene
</td>
<td style="text-align:left;">
predicted gene, 45993
</td>
</tr>
<tr>
<td style="text-align:left;">
7
</td>
<td style="text-align:left;">
gene
</td>
<td style="text-align:right;">
90.05537
</td>
<td style="text-align:right;">
90.06351
</td>
<td style="text-align:left;">
negative
</td>
<td style="text-align:left;">
MGI_C57BL6J_5591323
</td>
<td style="text-align:left;">
Gm32164
</td>
<td style="text-align:left;">
NCBI_Gene:102634625
</td>
<td style="text-align:left;">
MGI:5591323
</td>
<td style="text-align:left;">
lncRNA gene
</td>
<td style="text-align:left;">
predicted gene, 32164
</td>
</tr>
<tr>
<td style="text-align:left;">
7
</td>
<td style="text-align:left;">
pseudogene
</td>
<td style="text-align:right;">
90.10722
</td>
<td style="text-align:right;">
90.10809
</td>
<td style="text-align:left;">
negative
</td>
<td style="text-align:left;">
MGI_C57BL6J_3644965
</td>
<td style="text-align:left;">
Gm5341
</td>
<td style="text-align:left;">
NCBI_Gene:384719,ENSEMBL:ENSMUSG00000062235
</td>
<td style="text-align:left;">
MGI:3644965
</td>
<td style="text-align:left;">
pseudogene
</td>
<td style="text-align:left;">
predicted pseudogene 5341
</td>
</tr>
<tr>
<td style="text-align:left;">
7
</td>
<td style="text-align:left;">
gene
</td>
<td style="text-align:right;">
90.12482
</td>
<td style="text-align:right;">
90.12998
</td>
<td style="text-align:left;">
negative
</td>
<td style="text-align:left;">
MGI_C57BL6J_1925579
</td>
<td style="text-align:left;">
2310010J17Rik
</td>
<td style="text-align:left;">
NCBI_Gene:78329,ENSEMBL:ENSMUSG00000097162
</td>
<td style="text-align:left;">
MGI:1925579
</td>
<td style="text-align:left;">
lncRNA gene
</td>
<td style="text-align:left;">
RIKEN cDNA 2310010J17 gene
</td>
</tr>
<tr>
<td style="text-align:left;">
7
</td>
<td style="text-align:left;">
gene
</td>
<td style="text-align:right;">
90.13021
</td>
<td style="text-align:right;">
90.21346
</td>
<td style="text-align:left;">
positive
</td>
<td style="text-align:left;">
MGI_C57BL6J_2385902
</td>
<td style="text-align:left;">
Picalm
</td>
<td style="text-align:left;">
NCBI_Gene:233489,ENSEMBL:ENSMUSG00000039361
</td>
<td style="text-align:left;">
MGI:2385902
</td>
<td style="text-align:left;">
protein coding gene
</td>
<td style="text-align:left;">
phosphatidylinositol binding clathrin assembly protein
</td>
</tr>
<tr>
<td style="text-align:left;">
7
</td>
<td style="text-align:left;">
gene
</td>
<td style="text-align:right;">
90.13249
</td>
<td style="text-align:right;">
90.13763
</td>
<td style="text-align:left;">
positive
</td>
<td style="text-align:left;">
MGI_C57BL6J_5753799
</td>
<td style="text-align:left;">
Gm45223
</td>
<td style="text-align:left;">
ENSEMBL:ENSMUSG00000109429
</td>
<td style="text-align:left;">
MGI:5753799
</td>
<td style="text-align:left;">
unclassified gene
</td>
<td style="text-align:left;">
predicted gene 45223
</td>
</tr>
<tr>
<td style="text-align:left;">
7
</td>
<td style="text-align:left;">
gene
</td>
<td style="text-align:right;">
90.14595
</td>
<td style="text-align:right;">
90.15025
</td>
<td style="text-align:left;">
positive
</td>
<td style="text-align:left;">
MGI_C57BL6J_5753798
</td>
<td style="text-align:left;">
Gm45222
</td>
<td style="text-align:left;">
ENSEMBL:ENSMUSG00000109498
</td>
<td style="text-align:left;">
MGI:5753798
</td>
<td style="text-align:left;">
unclassified gene
</td>
<td style="text-align:left;">
predicted gene 45222
</td>
</tr>
<tr>
<td style="text-align:left;">
7
</td>
<td style="text-align:left;">
gene
</td>
<td style="text-align:right;">
90.18593
</td>
<td style="text-align:right;">
90.18820
</td>
<td style="text-align:left;">
positive
</td>
<td style="text-align:left;">
MGI_C57BL6J_5753797
</td>
<td style="text-align:left;">
Gm45221
</td>
<td style="text-align:left;">
ENSEMBL:ENSMUSG00000109005
</td>
<td style="text-align:left;">
MGI:5753797
</td>
<td style="text-align:left;">
unclassified gene
</td>
<td style="text-align:left;">
predicted gene 45221
</td>
</tr>
<tr>
<td style="text-align:left;">
7
</td>
<td style="text-align:left;">
gene
</td>
<td style="text-align:right;">
90.20128
</td>
<td style="text-align:right;">
90.20379
</td>
<td style="text-align:left;">
positive
</td>
<td style="text-align:left;">
MGI_C57BL6J_5753796
</td>
<td style="text-align:left;">
Gm45220
</td>
<td style="text-align:left;">
ENSEMBL:ENSMUSG00000109279
</td>
<td style="text-align:left;">
MGI:5753796
</td>
<td style="text-align:left;">
unclassified gene
</td>
<td style="text-align:left;">
predicted gene 45220
</td>
</tr>
<tr>
<td style="text-align:left;">
7
</td>
<td style="text-align:left;">
gene
</td>
<td style="text-align:right;">
90.22350
</td>
<td style="text-align:right;">
90.26578
</td>
<td style="text-align:left;">
negative
</td>
<td style="text-align:left;">
MGI_C57BL6J_1918255
</td>
<td style="text-align:left;">
Ccdc83
</td>
<td style="text-align:left;">
NCBI_Gene:75338,ENSEMBL:ENSMUSG00000030617
</td>
<td style="text-align:left;">
MGI:1918255
</td>
<td style="text-align:left;">
protein coding gene
</td>
<td style="text-align:left;">
coiled-coil domain containing 83
</td>
</tr>
<tr>
<td style="text-align:left;">
7
</td>
<td style="text-align:left;">
gene
</td>
<td style="text-align:right;">
90.29373
</td>
<td style="text-align:right;">
90.29385
</td>
<td style="text-align:left;">
positive
</td>
<td style="text-align:left;">
MGI_C57BL6J_4422018
</td>
<td style="text-align:left;">
n-R5s155
</td>
<td style="text-align:left;">
ENSEMBL:ENSMUSG00000084480
</td>
<td style="text-align:left;">
MGI:4422018
</td>
<td style="text-align:left;">
rRNA gene
</td>
<td style="text-align:left;">
nuclear encoded rRNA 5S 155
</td>
</tr>
<tr>
<td style="text-align:left;">
7
</td>
<td style="text-align:left;">
gene
</td>
<td style="text-align:right;">
90.30221
</td>
<td style="text-align:right;">
90.41072
</td>
<td style="text-align:left;">
positive
</td>
<td style="text-align:left;">
MGI_C57BL6J_1933366
</td>
<td style="text-align:left;">
Sytl2
</td>
<td style="text-align:left;">
NCBI_Gene:83671,ENSEMBL:ENSMUSG00000030616
</td>
<td style="text-align:left;">
MGI:1933366
</td>
<td style="text-align:left;">
protein coding gene
</td>
<td style="text-align:left;">
synaptotagmin-like 2
</td>
</tr>
<tr>
<td style="text-align:left;">
7
</td>
<td style="text-align:left;">
gene
</td>
<td style="text-align:right;">
90.42631
</td>
<td style="text-align:right;">
90.42867
</td>
<td style="text-align:left;">
positive
</td>
<td style="text-align:left;">
MGI_C57BL6J_1917304
</td>
<td style="text-align:left;">
Ccdc89
</td>
<td style="text-align:left;">
NCBI_Gene:70054,ENSEMBL:ENSMUSG00000044362
</td>
<td style="text-align:left;">
MGI:1917304
</td>
<td style="text-align:left;">
protein coding gene
</td>
<td style="text-align:left;">
coiled-coil domain containing 89
</td>
</tr>
<tr>
<td style="text-align:left;">
7
</td>
<td style="text-align:left;">
gene
</td>
<td style="text-align:right;">
90.44273
</td>
<td style="text-align:right;">
90.44838
</td>
<td style="text-align:left;">
positive
</td>
<td style="text-align:left;">
MGI_C57BL6J_2675296
</td>
<td style="text-align:left;">
Crebzf
</td>
<td style="text-align:left;">
NCBI_Gene:233490,ENSEMBL:ENSMUSG00000051451
</td>
<td style="text-align:left;">
MGI:2675296
</td>
<td style="text-align:left;">
protein coding gene
</td>
<td style="text-align:left;">
CREB/ATF bZIP transcription factor
</td>
</tr>
<tr>
<td style="text-align:left;">
7
</td>
<td style="text-align:left;">
gene
</td>
<td style="text-align:right;">
90.45070
</td>
<td style="text-align:right;">
90.45723
</td>
<td style="text-align:left;">
negative
</td>
<td style="text-align:left;">
MGI_C57BL6J_1913521
</td>
<td style="text-align:left;">
Tmem126a
</td>
<td style="text-align:left;">
NCBI_Gene:66271,ENSEMBL:ENSMUSG00000030615
</td>
<td style="text-align:left;">
MGI:1913521
</td>
<td style="text-align:left;">
protein coding gene
</td>
<td style="text-align:left;">
transmembrane protein 126A
</td>
</tr>
<tr>
<td style="text-align:left;">
7
</td>
<td style="text-align:left;">
gene
</td>
<td style="text-align:right;">
90.45725
</td>
<td style="text-align:right;">
90.45888
</td>
<td style="text-align:left;">
positive
</td>
<td style="text-align:left;">
MGI_C57BL6J_5753483
</td>
<td style="text-align:left;">
Gm44907
</td>
<td style="text-align:left;">
ENSEMBL:ENSMUSG00000109216
</td>
<td style="text-align:left;">
MGI:5753483
</td>
<td style="text-align:left;">
unclassified gene
</td>
<td style="text-align:left;">
predicted gene 44907
</td>
</tr>
<tr>
<td style="text-align:left;">
7
</td>
<td style="text-align:left;">
gene
</td>
<td style="text-align:right;">
90.46744
</td>
<td style="text-align:right;">
90.47600
</td>
<td style="text-align:left;">
negative
</td>
<td style="text-align:left;">
MGI_C57BL6J_1915722
</td>
<td style="text-align:left;">
Tmem126b
</td>
<td style="text-align:left;">
NCBI_Gene:68472,ENSEMBL:ENSMUSG00000030614
</td>
<td style="text-align:left;">
MGI:1915722
</td>
<td style="text-align:left;">
protein coding gene
</td>
<td style="text-align:left;">
transmembrane protein 126B
</td>
</tr>
<tr>
<td style="text-align:left;">
7
</td>
<td style="text-align:left;">
gene
</td>
<td style="text-align:right;">
90.47618
</td>
<td style="text-align:right;">
92.44925
</td>
<td style="text-align:left;">
positive
</td>
<td style="text-align:left;">
MGI_C57BL6J_1344351
</td>
<td style="text-align:left;">
Dlg2
</td>
<td style="text-align:left;">
NCBI_Gene:23859,ENSEMBL:ENSMUSG00000052572
</td>
<td style="text-align:left;">
MGI:1344351
</td>
<td style="text-align:left;">
protein coding gene
</td>
<td style="text-align:left;">
discs large MAGUK scaffold protein 2
</td>
</tr>
<tr>
<td style="text-align:left;">
7
</td>
<td style="text-align:left;">
pseudogene
</td>
<td style="text-align:right;">
90.81154
</td>
<td style="text-align:right;">
90.81216
</td>
<td style="text-align:left;">
negative
</td>
<td style="text-align:left;">
MGI_C57BL6J_5753738
</td>
<td style="text-align:left;">
Gm45162
</td>
<td style="text-align:left;">
ENSEMBL:ENSMUSG00000109074
</td>
<td style="text-align:left;">
MGI:5753738
</td>
<td style="text-align:left;">
pseudogene
</td>
<td style="text-align:left;">
predicted gene 45162
</td>
</tr>
<tr>
<td style="text-align:left;">
7
</td>
<td style="text-align:left;">
gene
</td>
<td style="text-align:right;">
90.87160
</td>
<td style="text-align:right;">
90.87278
</td>
<td style="text-align:left;">
positive
</td>
<td style="text-align:left;">
MGI_C57BL6J_1924466
</td>
<td style="text-align:left;">
A930002H02Rik
</td>
<td style="text-align:left;">
ENSEMBL:ENSMUSG00000109242
</td>
<td style="text-align:left;">
MGI:1924466
</td>
<td style="text-align:left;">
unclassified gene
</td>
<td style="text-align:left;">
RIKEN cDNA A930002H02 gene
</td>
</tr>
<tr>
<td style="text-align:left;">
7
</td>
<td style="text-align:left;">
gene
</td>
<td style="text-align:right;">
90.88544
</td>
<td style="text-align:right;">
90.88771
</td>
<td style="text-align:left;">
negative
</td>
<td style="text-align:left;">
MGI_C57BL6J_5753737
</td>
<td style="text-align:left;">
Gm45161
</td>
<td style="text-align:left;">
ENSEMBL:ENSMUSG00000109431
</td>
<td style="text-align:left;">
MGI:5753737
</td>
<td style="text-align:left;">
unclassified gene
</td>
<td style="text-align:left;">
predicted gene 45161
</td>
</tr>
<tr>
<td style="text-align:left;">
7
</td>
<td style="text-align:left;">
gene
</td>
<td style="text-align:right;">
90.88707
</td>
<td style="text-align:right;">
90.94005
</td>
<td style="text-align:left;">
positive
</td>
<td style="text-align:left;">
MGI_C57BL6J_5753735
</td>
<td style="text-align:left;">
Gm45159
</td>
<td style="text-align:left;">
ENSEMBL:ENSMUSG00000109125
</td>
<td style="text-align:left;">
MGI:5753735
</td>
<td style="text-align:left;">
lncRNA gene
</td>
<td style="text-align:left;">
predicted gene 45159
</td>
</tr>
<tr>
<td style="text-align:left;">
7
</td>
<td style="text-align:left;">
gene
</td>
<td style="text-align:right;">
90.99531
</td>
<td style="text-align:right;">
90.99942
</td>
<td style="text-align:left;">
positive
</td>
<td style="text-align:left;">
MGI_C57BL6J_5753754
</td>
<td style="text-align:left;">
Gm45178
</td>
<td style="text-align:left;">
ENSEMBL:ENSMUSG00000109562
</td>
<td style="text-align:left;">
MGI:5753754
</td>
<td style="text-align:left;">
unclassified gene
</td>
<td style="text-align:left;">
predicted gene 45178
</td>
</tr>
<tr>
<td style="text-align:left;">
7
</td>
<td style="text-align:left;">
gene
</td>
<td style="text-align:right;">
91.01574
</td>
<td style="text-align:right;">
91.01630
</td>
<td style="text-align:left;">
positive
</td>
<td style="text-align:left;">
MGI_C57BL6J_5621933
</td>
<td style="text-align:left;">
Gm39048
</td>
<td style="text-align:left;">
NCBI_Gene:105242994
</td>
<td style="text-align:left;">
MGI:5621933
</td>
<td style="text-align:left;">
lncRNA gene
</td>
<td style="text-align:left;">
predicted gene, 39048
</td>
</tr>
<tr>
<td style="text-align:left;">
7
</td>
<td style="text-align:left;">
gene
</td>
<td style="text-align:right;">
91.04900
</td>
<td style="text-align:right;">
91.05334
</td>
<td style="text-align:left;">
positive
</td>
<td style="text-align:left;">
MGI_C57BL6J_5753755
</td>
<td style="text-align:left;">
Gm45179
</td>
<td style="text-align:left;">
ENSEMBL:ENSMUSG00000108956
</td>
<td style="text-align:left;">
MGI:5753755
</td>
<td style="text-align:left;">
unclassified gene
</td>
<td style="text-align:left;">
predicted gene 45179
</td>
</tr>
<tr>
<td style="text-align:left;">
7
</td>
<td style="text-align:left;">
gene
</td>
<td style="text-align:right;">
91.12300
</td>
<td style="text-align:right;">
91.12321
</td>
<td style="text-align:left;">
negative
</td>
<td style="text-align:left;">
MGI_C57BL6J_5530852
</td>
<td style="text-align:left;">
Gm27470
</td>
<td style="text-align:left;">
ENSEMBL:ENSMUSG00000098805
</td>
<td style="text-align:left;">
MGI:5530852
</td>
<td style="text-align:left;">
unclassified non-coding RNA gene
</td>
<td style="text-align:left;">
predicted gene, 27470
</td>
</tr>
<tr>
<td style="text-align:left;">
7
</td>
<td style="text-align:left;">
gene
</td>
<td style="text-align:right;">
91.12635
</td>
<td style="text-align:right;">
91.12655
</td>
<td style="text-align:left;">
negative
</td>
<td style="text-align:left;">
MGI_C57BL6J_5531042
</td>
<td style="text-align:left;">
Gm27660
</td>
<td style="text-align:left;">
ENSEMBL:ENSMUSG00000099001
</td>
<td style="text-align:left;">
MGI:5531042
</td>
<td style="text-align:left;">
unclassified non-coding RNA gene
</td>
<td style="text-align:left;">
predicted gene, 27660
</td>
</tr>
<tr>
<td style="text-align:left;">
7
</td>
<td style="text-align:left;">
gene
</td>
<td style="text-align:right;">
91.13184
</td>
<td style="text-align:right;">
91.13594
</td>
<td style="text-align:left;">
positive
</td>
<td style="text-align:left;">
MGI_C57BL6J_5753752
</td>
<td style="text-align:left;">
Gm45176
</td>
<td style="text-align:left;">
ENSEMBL:ENSMUSG00000109239
</td>
<td style="text-align:left;">
MGI:5753752
</td>
<td style="text-align:left;">
unclassified gene
</td>
<td style="text-align:left;">
predicted gene 45176
</td>
</tr>
<tr>
<td style="text-align:left;">
7
</td>
<td style="text-align:left;">
gene
</td>
<td style="text-align:right;">
91.13996
</td>
<td style="text-align:right;">
91.14227
</td>
<td style="text-align:left;">
positive
</td>
<td style="text-align:left;">
MGI_C57BL6J_5753753
</td>
<td style="text-align:left;">
Gm45177
</td>
<td style="text-align:left;">
ENSEMBL:ENSMUSG00000108819
</td>
<td style="text-align:left;">
MGI:5753753
</td>
<td style="text-align:left;">
unclassified gene
</td>
<td style="text-align:left;">
predicted gene 45177
</td>
</tr>
<tr>
<td style="text-align:left;">
7
</td>
<td style="text-align:left;">
gene
</td>
<td style="text-align:right;">
91.15860
</td>
<td style="text-align:right;">
91.16100
</td>
<td style="text-align:left;">
positive
</td>
<td style="text-align:left;">
MGI_C57BL6J_5753758
</td>
<td style="text-align:left;">
Gm45182
</td>
<td style="text-align:left;">
ENSEMBL:ENSMUSG00000109387
</td>
<td style="text-align:left;">
MGI:5753758
</td>
<td style="text-align:left;">
unclassified gene
</td>
<td style="text-align:left;">
predicted gene 45182
</td>
</tr>
<tr>
<td style="text-align:left;">
7
</td>
<td style="text-align:left;">
gene
</td>
<td style="text-align:right;">
91.24299
</td>
<td style="text-align:right;">
91.24310
</td>
<td style="text-align:left;">
positive
</td>
<td style="text-align:left;">
MGI_C57BL6J_5454329
</td>
<td style="text-align:left;">
Gm24552
</td>
<td style="text-align:left;">
ENSEMBL:ENSMUSG00000089271
</td>
<td style="text-align:left;">
MGI:5454329
</td>
<td style="text-align:left;">
snRNA gene
</td>
<td style="text-align:left;">
predicted gene, 24552
</td>
</tr>
<tr>
<td style="text-align:left;">
7
</td>
<td style="text-align:left;">
gene
</td>
<td style="text-align:right;">
91.30469
</td>
<td style="text-align:right;">
91.30619
</td>
<td style="text-align:left;">
positive
</td>
<td style="text-align:left;">
MGI_C57BL6J_5753759
</td>
<td style="text-align:left;">
Gm45183
</td>
<td style="text-align:left;">
ENSEMBL:ENSMUSG00000109353
</td>
<td style="text-align:left;">
MGI:5753759
</td>
<td style="text-align:left;">
unclassified gene
</td>
<td style="text-align:left;">
predicted gene 45183
</td>
</tr>
<tr>
<td style="text-align:left;">
7
</td>
<td style="text-align:left;">
gene
</td>
<td style="text-align:right;">
91.34002
</td>
<td style="text-align:right;">
91.34279
</td>
<td style="text-align:left;">
positive
</td>
<td style="text-align:left;">
MGI_C57BL6J_5753254
</td>
<td style="text-align:left;">
Gm44678
</td>
<td style="text-align:left;">
ENSEMBL:ENSMUSG00000108920
</td>
<td style="text-align:left;">
MGI:5753254
</td>
<td style="text-align:left;">
unclassified gene
</td>
<td style="text-align:left;">
predicted gene 44678
</td>
</tr>
<tr>
<td style="text-align:left;">
7
</td>
<td style="text-align:left;">
gene
</td>
<td style="text-align:right;">
91.39258
</td>
<td style="text-align:right;">
91.39269
</td>
<td style="text-align:left;">
negative
</td>
<td style="text-align:left;">
MGI_C57BL6J_5451857
</td>
<td style="text-align:left;">
Gm22080
</td>
<td style="text-align:left;">
ENSEMBL:ENSMUSG00000088216
</td>
<td style="text-align:left;">
MGI:5451857
</td>
<td style="text-align:left;">
snRNA gene
</td>
<td style="text-align:left;">
predicted gene, 22080
</td>
</tr>
<tr>
<td style="text-align:left;">
7
</td>
<td style="text-align:left;">
gene
</td>
<td style="text-align:right;">
91.51911
</td>
<td style="text-align:right;">
91.52202
</td>
<td style="text-align:left;">
positive
</td>
<td style="text-align:left;">
MGI_C57BL6J_5753255
</td>
<td style="text-align:left;">
Gm44679
</td>
<td style="text-align:left;">
ENSEMBL:ENSMUSG00000109320
</td>
<td style="text-align:left;">
MGI:5753255
</td>
<td style="text-align:left;">
unclassified gene
</td>
<td style="text-align:left;">
predicted gene 44679
</td>
</tr>
<tr>
<td style="text-align:left;">
7
</td>
<td style="text-align:left;">
gene
</td>
<td style="text-align:right;">
91.56437
</td>
<td style="text-align:right;">
91.56729
</td>
<td style="text-align:left;">
positive
</td>
<td style="text-align:left;">
MGI_C57BL6J_5753251
</td>
<td style="text-align:left;">
Gm44675
</td>
<td style="text-align:left;">
ENSEMBL:ENSMUSG00000109370
</td>
<td style="text-align:left;">
MGI:5753251
</td>
<td style="text-align:left;">
unclassified gene
</td>
<td style="text-align:left;">
predicted gene 44675
</td>
</tr>
<tr>
<td style="text-align:left;">
7
</td>
<td style="text-align:left;">
gene
</td>
<td style="text-align:right;">
91.60896
</td>
<td style="text-align:right;">
91.61201
</td>
<td style="text-align:left;">
positive
</td>
<td style="text-align:left;">
MGI_C57BL6J_5753252
</td>
<td style="text-align:left;">
Gm44676
</td>
<td style="text-align:left;">
ENSEMBL:ENSMUSG00000109217
</td>
<td style="text-align:left;">
MGI:5753252
</td>
<td style="text-align:left;">
unclassified gene
</td>
<td style="text-align:left;">
predicted gene 44676
</td>
</tr>
<tr>
<td style="text-align:left;">
7
</td>
<td style="text-align:left;">
gene
</td>
<td style="text-align:right;">
91.62727
</td>
<td style="text-align:right;">
91.63101
</td>
<td style="text-align:left;">
positive
</td>
<td style="text-align:left;">
MGI_C57BL6J_5753253
</td>
<td style="text-align:left;">
Gm44677
</td>
<td style="text-align:left;">
ENSEMBL:ENSMUSG00000109505
</td>
<td style="text-align:left;">
MGI:5753253
</td>
<td style="text-align:left;">
unclassified gene
</td>
<td style="text-align:left;">
predicted gene 44677
</td>
</tr>
<tr>
<td style="text-align:left;">
7
</td>
<td style="text-align:left;">
gene
</td>
<td style="text-align:right;">
91.63524
</td>
<td style="text-align:right;">
91.63815
</td>
<td style="text-align:left;">
positive
</td>
<td style="text-align:left;">
MGI_C57BL6J_5753256
</td>
<td style="text-align:left;">
Gm44680
</td>
<td style="text-align:left;">
ENSEMBL:ENSMUSG00000109044
</td>
<td style="text-align:left;">
MGI:5753256
</td>
<td style="text-align:left;">
unclassified gene
</td>
<td style="text-align:left;">
predicted gene 44680
</td>
</tr>
<tr>
<td style="text-align:left;">
7
</td>
<td style="text-align:left;">
gene
</td>
<td style="text-align:right;">
91.64771
</td>
<td style="text-align:right;">
91.64855
</td>
<td style="text-align:left;">
positive
</td>
<td style="text-align:left;">
MGI_C57BL6J_5753257
</td>
<td style="text-align:left;">
Gm44681
</td>
<td style="text-align:left;">
ENSEMBL:ENSMUSG00000109256
</td>
<td style="text-align:left;">
MGI:5753257
</td>
<td style="text-align:left;">
unclassified gene
</td>
<td style="text-align:left;">
predicted gene 44681
</td>
</tr>
<tr>
<td style="text-align:left;">
7
</td>
<td style="text-align:left;">
gene
</td>
<td style="text-align:right;">
91.66715
</td>
<td style="text-align:right;">
91.66729
</td>
<td style="text-align:left;">
negative
</td>
<td style="text-align:left;">
MGI_C57BL6J_5453705
</td>
<td style="text-align:left;">
Gm23928
</td>
<td style="text-align:left;">
ENSEMBL:ENSMUSG00000064403
</td>
<td style="text-align:left;">
MGI:5453705
</td>
<td style="text-align:left;">
snRNA gene
</td>
<td style="text-align:left;">
predicted gene, 23928
</td>
</tr>
<tr>
<td style="text-align:left;">
7
</td>
<td style="text-align:left;">
gene
</td>
<td style="text-align:right;">
91.68947
</td>
<td style="text-align:right;">
91.69021
</td>
<td style="text-align:left;">
positive
</td>
<td style="text-align:left;">
MGI_C57BL6J_1924862
</td>
<td style="text-align:left;">
C030038I04Rik
</td>
<td style="text-align:left;">
ENSEMBL:ENSMUSG00000109037
</td>
<td style="text-align:left;">
MGI:1924862
</td>
<td style="text-align:left;">
unclassified gene
</td>
<td style="text-align:left;">
RIKEN cDNA C030038I04 gene
</td>
</tr>
<tr>
<td style="text-align:left;">
7
</td>
<td style="text-align:left;">
gene
</td>
<td style="text-align:right;">
91.81617
</td>
<td style="text-align:right;">
91.81955
</td>
<td style="text-align:left;">
positive
</td>
<td style="text-align:left;">
MGI_C57BL6J_5753706
</td>
<td style="text-align:left;">
Gm45130
</td>
<td style="text-align:left;">
ENSEMBL:ENSMUSG00000109481
</td>
<td style="text-align:left;">
MGI:5753706
</td>
<td style="text-align:left;">
unclassified gene
</td>
<td style="text-align:left;">
predicted gene 45130
</td>
</tr>
<tr>
<td style="text-align:left;">
7
</td>
<td style="text-align:left;">
gene
</td>
<td style="text-align:right;">
91.90806
</td>
<td style="text-align:right;">
91.91029
</td>
<td style="text-align:left;">
positive
</td>
<td style="text-align:left;">
MGI_C57BL6J_5753705
</td>
<td style="text-align:left;">
Gm45129
</td>
<td style="text-align:left;">
ENSEMBL:ENSMUSG00000108953
</td>
<td style="text-align:left;">
MGI:5753705
</td>
<td style="text-align:left;">
unclassified gene
</td>
<td style="text-align:left;">
predicted gene 45129
</td>
</tr>
<tr>
<td style="text-align:left;">
7
</td>
<td style="text-align:left;">
gene
</td>
<td style="text-align:right;">
91.93277
</td>
<td style="text-align:right;">
91.96076
</td>
<td style="text-align:left;">
negative
</td>
<td style="text-align:left;">
MGI_C57BL6J_5591512
</td>
<td style="text-align:left;">
Gm32353
</td>
<td style="text-align:left;">
NCBI_Gene:102634874
</td>
<td style="text-align:left;">
MGI:5591512
</td>
<td style="text-align:left;">
lncRNA gene
</td>
<td style="text-align:left;">
predicted gene, 32353
</td>
</tr>
<tr>
<td style="text-align:left;">
7
</td>
<td style="text-align:left;">
gene
</td>
<td style="text-align:right;">
92.06459
</td>
<td style="text-align:right;">
92.06717
</td>
<td style="text-align:left;">
positive
</td>
<td style="text-align:left;">
MGI_C57BL6J_5753707
</td>
<td style="text-align:left;">
Gm45131
</td>
<td style="text-align:left;">
ENSEMBL:ENSMUSG00000109191
</td>
<td style="text-align:left;">
MGI:5753707
</td>
<td style="text-align:left;">
unclassified gene
</td>
<td style="text-align:left;">
predicted gene 45131
</td>
</tr>
<tr>
<td style="text-align:left;">
7
</td>
<td style="text-align:left;">
gene
</td>
<td style="text-align:right;">
92.08109
</td>
<td style="text-align:right;">
92.08254
</td>
<td style="text-align:left;">
negative
</td>
<td style="text-align:left;">
MGI_C57BL6J_1918245
</td>
<td style="text-align:left;">
4931412I15Rik
</td>
<td style="text-align:left;">
ENSEMBL:ENSMUSG00000108826
</td>
<td style="text-align:left;">
MGI:1918245
</td>
<td style="text-align:left;">
unclassified gene
</td>
<td style="text-align:left;">
RIKEN cDNA 4931412I15 gene
</td>
</tr>
<tr>
<td style="text-align:left;">
7
</td>
<td style="text-align:left;">
gene
</td>
<td style="text-align:right;">
92.09221
</td>
<td style="text-align:right;">
92.11951
</td>
<td style="text-align:left;">
negative
</td>
<td style="text-align:left;">
MGI_C57BL6J_1923095
</td>
<td style="text-align:left;">
4930567K12Rik
</td>
<td style="text-align:left;">
NCBI_Gene:75845,ENSEMBL:ENSMUSG00000109382
</td>
<td style="text-align:left;">
MGI:1923095
</td>
<td style="text-align:left;">
lncRNA gene
</td>
<td style="text-align:left;">
RIKEN cDNA 4930567K12 gene
</td>
</tr>
<tr>
<td style="text-align:left;">
7
</td>
<td style="text-align:left;">
gene
</td>
<td style="text-align:right;">
92.40818
</td>
<td style="text-align:right;">
92.41137
</td>
<td style="text-align:left;">
positive
</td>
<td style="text-align:left;">
MGI_C57BL6J_1926094
</td>
<td style="text-align:left;">
B230206I08Rik
</td>
<td style="text-align:left;">
ENSEMBL:ENSMUSG00000109574
</td>
<td style="text-align:left;">
MGI:1926094
</td>
<td style="text-align:left;">
unclassified gene
</td>
<td style="text-align:left;">
RIKEN cDNA B230206I08 gene
</td>
</tr>
<tr>
<td style="text-align:left;">
7
</td>
<td style="text-align:left;">
gene
</td>
<td style="text-align:right;">
92.43117
</td>
<td style="text-align:right;">
92.43385
</td>
<td style="text-align:left;">
positive
</td>
<td style="text-align:left;">
MGI_C57BL6J_5753777
</td>
<td style="text-align:left;">
Gm45201
</td>
<td style="text-align:left;">
ENSEMBL:ENSMUSG00000109031
</td>
<td style="text-align:left;">
MGI:5753777
</td>
<td style="text-align:left;">
unclassified gene
</td>
<td style="text-align:left;">
predicted gene 45201
</td>
</tr>
<tr>
<td style="text-align:left;">
7
</td>
<td style="text-align:left;">
pseudogene
</td>
<td style="text-align:right;">
92.46923
</td>
<td style="text-align:right;">
92.47823
</td>
<td style="text-align:left;">
positive
</td>
<td style="text-align:left;">
MGI_C57BL6J_5590650
</td>
<td style="text-align:left;">
Gm31491
</td>
<td style="text-align:left;">
NCBI_Gene:102633738
</td>
<td style="text-align:left;">
MGI:5590650
</td>
<td style="text-align:left;">
pseudogene
</td>
<td style="text-align:left;">
predicted gene, 31491
</td>
</tr>
<tr>
<td style="text-align:left;">
7
</td>
<td style="text-align:left;">
gene
</td>
<td style="text-align:right;">
92.56115
</td>
<td style="text-align:right;">
92.58229
</td>
<td style="text-align:left;">
positive
</td>
<td style="text-align:left;">
MGI_C57BL6J_1913615
</td>
<td style="text-align:left;">
Ccdc90b
</td>
<td style="text-align:left;">
NCBI_Gene:66365,ENSEMBL:ENSMUSG00000030613
</td>
<td style="text-align:left;">
MGI:1913615
</td>
<td style="text-align:left;">
protein coding gene
</td>
<td style="text-align:left;">
coiled-coil domain containing 90B
</td>
</tr>
<tr>
<td style="text-align:left;">
7
</td>
<td style="text-align:left;">
gene
</td>
<td style="text-align:right;">
92.58172
</td>
<td style="text-align:right;">
92.63714
</td>
<td style="text-align:left;">
negative
</td>
<td style="text-align:left;">
MGI_C57BL6J_1921095
</td>
<td style="text-align:left;">
Ankrd42
</td>
<td style="text-align:left;">
NCBI_Gene:73845,ENSEMBL:ENSMUSG00000041343
</td>
<td style="text-align:left;">
MGI:1921095
</td>
<td style="text-align:left;">
protein coding gene
</td>
<td style="text-align:left;">
ankyrin repeat domain 42
</td>
</tr>
<tr>
<td style="text-align:left;">
7
</td>
<td style="text-align:left;">
gene
</td>
<td style="text-align:right;">
92.63719
</td>
<td style="text-align:right;">
92.64830
</td>
<td style="text-align:left;">
positive
</td>
<td style="text-align:left;">
MGI_C57BL6J_5504059
</td>
<td style="text-align:left;">
Gm26944
</td>
<td style="text-align:left;">
NCBI_Gene:102635071,ENSEMBL:ENSMUSG00000098066
</td>
<td style="text-align:left;">
MGI:5504059
</td>
<td style="text-align:left;">
lncRNA gene
</td>
<td style="text-align:left;">
predicted gene, 26944
</td>
</tr>
<tr>
<td style="text-align:left;">
7
</td>
<td style="text-align:left;">
gene
</td>
<td style="text-align:right;">
92.63721
</td>
<td style="text-align:right;">
92.63753
</td>
<td style="text-align:left;">
positive
</td>
<td style="text-align:left;">
MGI_C57BL6J_5610236
</td>
<td style="text-align:left;">
Gm37008
</td>
<td style="text-align:left;">
ENSEMBL:ENSMUSG00000103887
</td>
<td style="text-align:left;">
MGI:5610236
</td>
<td style="text-align:left;">
unclassified gene
</td>
<td style="text-align:left;">
predicted gene, 37008
</td>
</tr>
<tr>
<td style="text-align:left;">
7
</td>
<td style="text-align:left;">
gene
</td>
<td style="text-align:right;">
92.63995
</td>
<td style="text-align:right;">
92.64378
</td>
<td style="text-align:left;">
positive
</td>
<td style="text-align:left;">
MGI_C57BL6J_2443259
</td>
<td style="text-align:left;">
6430511E19Rik
</td>
<td style="text-align:left;">
ENSEMBL:ENSMUSG00000102555
</td>
<td style="text-align:left;">
MGI:2443259
</td>
<td style="text-align:left;">
unclassified gene
</td>
<td style="text-align:left;">
RIKEN cDNA 6430511E19 gene
</td>
</tr>
<tr>
<td style="text-align:left;">
7
</td>
<td style="text-align:left;">
gene
</td>
<td style="text-align:right;">
92.64354
</td>
<td style="text-align:right;">
92.67005
</td>
<td style="text-align:left;">
negative
</td>
<td style="text-align:left;">
MGI_C57BL6J_1919579
</td>
<td style="text-align:left;">
Pcf11
</td>
<td style="text-align:left;">
NCBI_Gene:74737,ENSEMBL:ENSMUSG00000041328
</td>
<td style="text-align:left;">
MGI:1919579
</td>
<td style="text-align:left;">
protein coding gene
</td>
<td style="text-align:left;">
PCF11 cleavage and polyadenylation factor subunit
</td>
</tr>
<tr>
<td style="text-align:left;">
7
</td>
<td style="text-align:left;">
pseudogene
</td>
<td style="text-align:right;">
92.69368
</td>
<td style="text-align:right;">
92.69791
</td>
<td style="text-align:left;">
negative
</td>
<td style="text-align:left;">
MGI_C57BL6J_3780719
</td>
<td style="text-align:left;">
Gm2551
</td>
<td style="text-align:left;">
NCBI_Gene:100040007
</td>
<td style="text-align:left;">
MGI:3780719
</td>
<td style="text-align:left;">
pseudogene
</td>
<td style="text-align:left;">
predicted gene 2551
</td>
</tr>
<tr>
<td style="text-align:left;">
7
</td>
<td style="text-align:left;">
gene
</td>
<td style="text-align:right;">
92.70677
</td>
<td style="text-align:right;">
92.70828
</td>
<td style="text-align:left;">
positive
</td>
<td style="text-align:left;">
MGI_C57BL6J_5753775
</td>
<td style="text-align:left;">
Gm45199
</td>
<td style="text-align:left;">
ENSEMBL:ENSMUSG00000109395
</td>
<td style="text-align:left;">
MGI:5753775
</td>
<td style="text-align:left;">
unclassified gene
</td>
<td style="text-align:left;">
predicted gene 45199
</td>
</tr>
<tr>
<td style="text-align:left;">
7
</td>
<td style="text-align:left;">
gene
</td>
<td style="text-align:right;">
92.72984
</td>
<td style="text-align:right;">
92.73375
</td>
<td style="text-align:left;">
positive
</td>
<td style="text-align:left;">
MGI_C57BL6J_5504096
</td>
<td style="text-align:left;">
Gm26981
</td>
<td style="text-align:left;">
ENSEMBL:ENSMUSG00000098041
</td>
<td style="text-align:left;">
MGI:5504096
</td>
<td style="text-align:left;">
lncRNA gene
</td>
<td style="text-align:left;">
predicted gene, 26981
</td>
</tr>
<tr>
<td style="text-align:left;">
7
</td>
<td style="text-align:left;">
gene
</td>
<td style="text-align:right;">
92.73417
</td>
<td style="text-align:right;">
92.74147
</td>
<td style="text-align:left;">
negative
</td>
<td style="text-align:left;">
MGI_C57BL6J_1915436
</td>
<td style="text-align:left;">
4632427E13Rik
</td>
<td style="text-align:left;">
NCBI_Gene:666737,ENSEMBL:ENSMUSG00000074024
</td>
<td style="text-align:left;">
MGI:1915436
</td>
<td style="text-align:left;">
lncRNA gene
</td>
<td style="text-align:left;">
RIKEN cDNA 4632427E13 gene
</td>
</tr>
<tr>
<td style="text-align:left;">
7
</td>
<td style="text-align:left;">
gene
</td>
<td style="text-align:right;">
92.74160
</td>
<td style="text-align:right;">
92.84453
</td>
<td style="text-align:left;">
positive
</td>
<td style="text-align:left;">
MGI_C57BL6J_1923235
</td>
<td style="text-align:left;">
Rab30
</td>
<td style="text-align:left;">
NCBI_Gene:75985,ENSEMBL:ENSMUSG00000030643
</td>
<td style="text-align:left;">
MGI:1923235
</td>
<td style="text-align:left;">
protein coding gene
</td>
<td style="text-align:left;">
RAB30, member RAS oncogene family
</td>
</tr>
<tr>
<td style="text-align:left;">
7
</td>
<td style="text-align:left;">
gene
</td>
<td style="text-align:right;">
92.77850
</td>
<td style="text-align:right;">
92.77942
</td>
<td style="text-align:left;">
positive
</td>
<td style="text-align:left;">
MGI_C57BL6J_5753776
</td>
<td style="text-align:left;">
Gm45200
</td>
<td style="text-align:left;">
ENSEMBL:ENSMUSG00000109257
</td>
<td style="text-align:left;">
MGI:5753776
</td>
<td style="text-align:left;">
unclassified gene
</td>
<td style="text-align:left;">
predicted gene 45200
</td>
</tr>
<tr>
<td style="text-align:left;">
7
</td>
<td style="text-align:left;">
gene
</td>
<td style="text-align:right;">
92.82056
</td>
<td style="text-align:right;">
92.82077
</td>
<td style="text-align:left;">
positive
</td>
<td style="text-align:left;">
MGI_C57BL6J_1917506
</td>
<td style="text-align:left;">
2010107C10Rik
</td>
<td style="text-align:left;">
NA
</td>
<td style="text-align:left;">
NA
</td>
<td style="text-align:left;">
unclassified gene
</td>
<td style="text-align:left;">
RIKEN cDNA 2010107C10 gene
</td>
</tr>
<tr>
<td style="text-align:left;">
7
</td>
<td style="text-align:left;">
gene
</td>
<td style="text-align:right;">
92.85752
</td>
<td style="text-align:right;">
92.87429
</td>
<td style="text-align:left;">
negative
</td>
<td style="text-align:left;">
MGI_C57BL6J_1921291
</td>
<td style="text-align:left;">
Ddias
</td>
<td style="text-align:left;">
NCBI_Gene:74041,ENSEMBL:ENSMUSG00000030641
</td>
<td style="text-align:left;">
MGI:1921291
</td>
<td style="text-align:left;">
protein coding gene
</td>
<td style="text-align:left;">
DNA damage-induced apoptosis suppressor
</td>
</tr>
<tr>
<td style="text-align:left;">
7
</td>
<td style="text-align:left;">
gene
</td>
<td style="text-align:right;">
92.87447
</td>
<td style="text-align:right;">
92.93458
</td>
<td style="text-align:left;">
positive
</td>
<td style="text-align:left;">
MGI_C57BL6J_1919711
</td>
<td style="text-align:left;">
Prcp
</td>
<td style="text-align:left;">
NCBI_Gene:72461,ENSEMBL:ENSMUSG00000061119
</td>
<td style="text-align:left;">
MGI:1919711
</td>
<td style="text-align:left;">
protein coding gene
</td>
<td style="text-align:left;">
prolylcarboxypeptidase (angiotensinase C)
</td>
</tr>
<tr>
<td style="text-align:left;">
7
</td>
<td style="text-align:left;">
gene
</td>
<td style="text-align:right;">
92.87743
</td>
<td style="text-align:right;">
92.88999
</td>
<td style="text-align:left;">
positive
</td>
<td style="text-align:left;">
MGI_C57BL6J_5623333
</td>
<td style="text-align:left;">
Gm40448
</td>
<td style="text-align:left;">
NCBI_Gene:105244926
</td>
<td style="text-align:left;">
MGI:5623333
</td>
<td style="text-align:left;">
lncRNA gene
</td>
<td style="text-align:left;">
predicted gene, 40448
</td>
</tr>
<tr>
<td style="text-align:left;">
7
</td>
<td style="text-align:left;">
gene
</td>
<td style="text-align:right;">
92.90218
</td>
<td style="text-align:right;">
92.90287
</td>
<td style="text-align:left;">
positive
</td>
<td style="text-align:left;">
MGI_C57BL6J_1925814
</td>
<td style="text-align:left;">
9530078K11Rik
</td>
<td style="text-align:left;">
ENSEMBL:ENSMUSG00000109268
</td>
<td style="text-align:left;">
MGI:1925814
</td>
<td style="text-align:left;">
unclassified gene
</td>
<td style="text-align:left;">
RIKEN cDNA 9530078K11 gene
</td>
</tr>
<tr>
<td style="text-align:left;">
7
</td>
<td style="text-align:left;">
pseudogene
</td>
<td style="text-align:right;">
92.97925
</td>
<td style="text-align:right;">
93.00654
</td>
<td style="text-align:left;">
positive
</td>
<td style="text-align:left;">
MGI_C57BL6J_5590822
</td>
<td style="text-align:left;">
Gm31663
</td>
<td style="text-align:left;">
NCBI_Gene:102633966,ENSEMBL:ENSMUSG00000108994
</td>
<td style="text-align:left;">
MGI:5590822
</td>
<td style="text-align:left;">
pseudogene
</td>
<td style="text-align:left;">
predicted gene, 31663
</td>
</tr>
<tr>
<td style="text-align:left;">
7
</td>
<td style="text-align:left;">
gene
</td>
<td style="text-align:right;">
93.00698
</td>
<td style="text-align:right;">
93.10479
</td>
<td style="text-align:left;">
positive
</td>
<td style="text-align:left;">
MGI_C57BL6J_5477356
</td>
<td style="text-align:left;">
Gm26862
</td>
<td style="text-align:left;">
ENSEMBL:ENSMUSG00000097644
</td>
<td style="text-align:left;">
MGI:5477356
</td>
<td style="text-align:left;">
lncRNA gene
</td>
<td style="text-align:left;">
predicted gene, 26862
</td>
</tr>
<tr>
<td style="text-align:left;">
7
</td>
<td style="text-align:left;">
gene
</td>
<td style="text-align:right;">
93.05208
</td>
<td style="text-align:right;">
93.08103
</td>
<td style="text-align:left;">
negative
</td>
<td style="text-align:left;">
MGI_C57BL6J_3641747
</td>
<td style="text-align:left;">
Gm9934
</td>
<td style="text-align:left;">
ENSEMBL:ENSMUSG00000054061
</td>
<td style="text-align:left;">
MGI:3641747
</td>
<td style="text-align:left;">
lncRNA gene
</td>
<td style="text-align:left;">
predicted gene 9934
</td>
</tr>
<tr>
<td style="text-align:left;">
7
</td>
<td style="text-align:left;">
gene
</td>
<td style="text-align:right;">
93.07986
</td>
<td style="text-align:right;">
93.08187
</td>
<td style="text-align:left;">
positive
</td>
<td style="text-align:left;">
MGI_C57BL6J_1930951
</td>
<td style="text-align:left;">
Fam181b
</td>
<td style="text-align:left;">
NCBI_Gene:58238,ENSEMBL:ENSMUSG00000051515
</td>
<td style="text-align:left;">
MGI:1930951
</td>
<td style="text-align:left;">
protein coding gene
</td>
<td style="text-align:left;">
family with sequence similarity 181, member B
</td>
</tr>
<tr>
<td style="text-align:left;">
7
</td>
<td style="text-align:left;">
gene
</td>
<td style="text-align:right;">
93.14785
</td>
<td style="text-align:right;">
93.14797
</td>
<td style="text-align:left;">
negative
</td>
<td style="text-align:left;">
MGI_C57BL6J_5455637
</td>
<td style="text-align:left;">
Gm25860
</td>
<td style="text-align:left;">
ENSEMBL:ENSMUSG00000088877
</td>
<td style="text-align:left;">
MGI:5455637
</td>
<td style="text-align:left;">
snoRNA gene
</td>
<td style="text-align:left;">
predicted gene, 25860
</td>
</tr>
<tr>
<td style="text-align:left;">
7
</td>
<td style="text-align:left;">
pseudogene
</td>
<td style="text-align:right;">
93.16883
</td>
<td style="text-align:right;">
93.17267
</td>
<td style="text-align:left;">
positive
</td>
<td style="text-align:left;">
MGI_C57BL6J_5753291
</td>
<td style="text-align:left;">
Gm44715
</td>
<td style="text-align:left;">
ENSEMBL:ENSMUSG00000109196
</td>
<td style="text-align:left;">
MGI:5753291
</td>
<td style="text-align:left;">
pseudogene
</td>
<td style="text-align:left;">
predicted gene 44715
</td>
</tr>
<tr>
<td style="text-align:left;">
7
</td>
<td style="text-align:left;">
pseudogene
</td>
<td style="text-align:right;">
93.17898
</td>
<td style="text-align:right;">
93.18418
</td>
<td style="text-align:left;">
negative
</td>
<td style="text-align:left;">
MGI_C57BL6J_3704296
</td>
<td style="text-align:left;">
Gm15501
</td>
<td style="text-align:left;">
NCBI_Gene:100040298,ENSEMBL:ENSMUSG00000087412
</td>
<td style="text-align:left;">
MGI:3704296
</td>
<td style="text-align:left;">
pseudogene
</td>
<td style="text-align:left;">
predicted pseudogene 15501
</td>
</tr>
<tr>
<td style="text-align:left;">
7
</td>
<td style="text-align:left;">
gene
</td>
<td style="text-align:right;">
93.29070
</td>
<td style="text-align:right;">
93.32395
</td>
<td style="text-align:left;">
negative
</td>
<td style="text-align:left;">
MGI_C57BL6J_5753523
</td>
<td style="text-align:left;">
Gm44947
</td>
<td style="text-align:left;">
ENSEMBL:ENSMUSG00000109175
</td>
<td style="text-align:left;">
MGI:5753523
</td>
<td style="text-align:left;">
lncRNA gene
</td>
<td style="text-align:left;">
predicted gene 44947
</td>
</tr>
<tr>
<td style="text-align:left;">
7
</td>
<td style="text-align:left;">
gene
</td>
<td style="text-align:right;">
93.36168
</td>
<td style="text-align:right;">
93.38086
</td>
<td style="text-align:left;">
positive
</td>
<td style="text-align:left;">
MGI_C57BL6J_5753613
</td>
<td style="text-align:left;">
Gm45037
</td>
<td style="text-align:left;">
ENSEMBL:ENSMUSG00000109315
</td>
<td style="text-align:left;">
MGI:5753613
</td>
<td style="text-align:left;">
lncRNA gene
</td>
<td style="text-align:left;">
predicted gene 45037
</td>
</tr>
<tr>
<td style="text-align:left;">
7
</td>
<td style="text-align:left;">
pseudogene
</td>
<td style="text-align:right;">
93.52306
</td>
<td style="text-align:right;">
93.52690
</td>
<td style="text-align:left;">
positive
</td>
<td style="text-align:left;">
MGI_C57BL6J_5753614
</td>
<td style="text-align:left;">
Gm45038
</td>
<td style="text-align:left;">
ENSEMBL:ENSMUSG00000109195
</td>
<td style="text-align:left;">
MGI:5753614
</td>
<td style="text-align:left;">
pseudogene
</td>
<td style="text-align:left;">
predicted gene 45038
</td>
</tr>
<tr>
<td style="text-align:left;">
7
</td>
<td style="text-align:left;">
pseudogene
</td>
<td style="text-align:right;">
93.56494
</td>
<td style="text-align:right;">
93.56570
</td>
<td style="text-align:left;">
negative
</td>
<td style="text-align:left;">
MGI_C57BL6J_3645456
</td>
<td style="text-align:left;">
Gm8285
</td>
<td style="text-align:left;">
NCBI_Gene:666783,ENSEMBL:ENSMUSG00000109077
</td>
<td style="text-align:left;">
MGI:3645456
</td>
<td style="text-align:left;">
pseudogene
</td>
<td style="text-align:left;">
predicted gene 8285
</td>
</tr>
<tr>
<td style="text-align:left;">
7
</td>
<td style="text-align:left;">
gene
</td>
<td style="text-align:right;">
93.60484
</td>
<td style="text-align:right;">
93.60832
</td>
<td style="text-align:left;">
negative
</td>
<td style="text-align:left;">
MGI_C57BL6J_5753615
</td>
<td style="text-align:left;">
Gm45039
</td>
<td style="text-align:left;">
ENSEMBL:ENSMUSG00000109226
</td>
<td style="text-align:left;">
MGI:5753615
</td>
<td style="text-align:left;">
unclassified gene
</td>
<td style="text-align:left;">
predicted gene 45039
</td>
</tr>
<tr>
<td style="text-align:left;">
7
</td>
<td style="text-align:left;">
pseudogene
</td>
<td style="text-align:right;">
93.66024
</td>
<td style="text-align:right;">
93.66056
</td>
<td style="text-align:left;">
negative
</td>
<td style="text-align:left;">
MGI_C57BL6J_5052079
</td>
<td style="text-align:left;">
Bc1-ps1
</td>
<td style="text-align:left;">
NCBI_Gene:12031,ENSEMBL:ENSMUSG00000109440
</td>
<td style="text-align:left;">
MGI:5052079
</td>
<td style="text-align:left;">
pseudogene
</td>
<td style="text-align:left;">
brain cytoplasmic RNA 1, pseudogene 1
</td>
</tr>
<tr>
<td style="text-align:left;">
7
</td>
<td style="text-align:left;">
pseudogene
</td>
<td style="text-align:right;">
93.84690
</td>
<td style="text-align:right;">
93.84783
</td>
<td style="text-align:left;">
negative
</td>
<td style="text-align:left;">
MGI_C57BL6J_5010382
</td>
<td style="text-align:left;">
Gm18197
</td>
<td style="text-align:left;">
NCBI_Gene:100416690
</td>
<td style="text-align:left;">
MGI:5010382
</td>
<td style="text-align:left;">
pseudogene
</td>
<td style="text-align:left;">
predicted gene, 18197
</td>
</tr>
<tr>
<td style="text-align:left;">
7
</td>
<td style="text-align:left;">
gene
</td>
<td style="text-align:right;">
94.04250
</td>
<td style="text-align:right;">
95.32643
</td>
<td style="text-align:left;">
positive
</td>
<td style="text-align:left;">
MGI_C57BL6J_5591806
</td>
<td style="text-align:left;">
Gm32647
</td>
<td style="text-align:left;">
NCBI_Gene:102635262,ENSEMBL:ENSMUSG00000108532
</td>
<td style="text-align:left;">
MGI:5591806
</td>
<td style="text-align:left;">
lncRNA gene
</td>
<td style="text-align:left;">
predicted gene, 32647
</td>
</tr>
<tr>
<td style="text-align:left;">
7
</td>
<td style="text-align:left;">
pseudogene
</td>
<td style="text-align:right;">
94.19642
</td>
<td style="text-align:right;">
94.19756
</td>
<td style="text-align:left;">
positive
</td>
<td style="text-align:left;">
MGI_C57BL6J_3643908
</td>
<td style="text-align:left;">
Gm5899
</td>
<td style="text-align:left;">
NCBI_Gene:545982,ENSEMBL:ENSMUSG00000096033
</td>
<td style="text-align:left;">
MGI:3643908
</td>
<td style="text-align:left;">
pseudogene
</td>
<td style="text-align:left;">
predicted pseudogene 5899
</td>
</tr>
<tr>
<td style="text-align:left;">
7
</td>
<td style="text-align:left;">
gene
</td>
<td style="text-align:right;">
94.36898
</td>
<td style="text-align:right;">
94.37257
</td>
<td style="text-align:left;">
negative
</td>
<td style="text-align:left;">
MGI_C57BL6J_5753177
</td>
<td style="text-align:left;">
Gm44601
</td>
<td style="text-align:left;">
ENSEMBL:ENSMUSG00000108315
</td>
<td style="text-align:left;">
MGI:5753177
</td>
<td style="text-align:left;">
lncRNA gene
</td>
<td style="text-align:left;">
predicted gene 44601
</td>
</tr>
<tr>
<td style="text-align:left;">
7
</td>
<td style="text-align:left;">
pseudogene
</td>
<td style="text-align:right;">
94.47558
</td>
<td style="text-align:right;">
94.47571
</td>
<td style="text-align:left;">
positive
</td>
<td style="text-align:left;">
MGI_C57BL6J_5753178
</td>
<td style="text-align:left;">
Gm44602
</td>
<td style="text-align:left;">
ENSEMBL:ENSMUSG00000108516
</td>
<td style="text-align:left;">
MGI:5753178
</td>
<td style="text-align:left;">
pseudogene
</td>
<td style="text-align:left;">
predicted gene 44602
</td>
</tr>
<tr>
<td style="text-align:left;">
7
</td>
<td style="text-align:left;">
pseudogene
</td>
<td style="text-align:right;">
94.60182
</td>
<td style="text-align:right;">
94.60263
</td>
<td style="text-align:left;">
negative
</td>
<td style="text-align:left;">
MGI_C57BL6J_5011946
</td>
<td style="text-align:left;">
Gm19761
</td>
<td style="text-align:left;">
NCBI_Gene:100503542,ENSEMBL:ENSMUSG00000108433
</td>
<td style="text-align:left;">
MGI:5011946
</td>
<td style="text-align:left;">
pseudogene
</td>
<td style="text-align:left;">
predicted gene, 19761
</td>
</tr>
<tr>
<td style="text-align:left;">
7
</td>
<td style="text-align:left;">
gene
</td>
<td style="text-align:right;">
95.17186
</td>
<td style="text-align:right;">
95.17216
</td>
<td style="text-align:left;">
positive
</td>
<td style="text-align:left;">
MGI_C57BL6J_5453661
</td>
<td style="text-align:left;">
Gm23884
</td>
<td style="text-align:left;">
ENSEMBL:ENSMUSG00000084636
</td>
<td style="text-align:left;">
MGI:5453661
</td>
<td style="text-align:left;">
unclassified non-coding RNA gene
</td>
<td style="text-align:left;">
predicted gene, 23884
</td>
</tr>
<tr>
<td style="text-align:left;">
7
</td>
<td style="text-align:left;">
gene
</td>
<td style="text-align:right;">
95.44235
</td>
<td style="text-align:right;">
95.44246
</td>
<td style="text-align:left;">
negative
</td>
<td style="text-align:left;">
MGI_C57BL6J_5455461
</td>
<td style="text-align:left;">
Gm25684
</td>
<td style="text-align:left;">
ENSEMBL:ENSMUSG00000065875
</td>
<td style="text-align:left;">
MGI:5455461
</td>
<td style="text-align:left;">
snRNA gene
</td>
<td style="text-align:left;">
predicted gene, 25684
</td>
</tr>
<tr>
<td style="text-align:left;">
7
</td>
<td style="text-align:left;">
pseudogene
</td>
<td style="text-align:right;">
95.46191
</td>
<td style="text-align:right;">
95.46356
</td>
<td style="text-align:left;">
positive
</td>
<td style="text-align:left;">
MGI_C57BL6J_3643545
</td>
<td style="text-align:left;">
Gm8309
</td>
<td style="text-align:left;">
NCBI_Gene:666823,ENSEMBL:ENSMUSG00000108786
</td>
<td style="text-align:left;">
MGI:3643545
</td>
<td style="text-align:left;">
pseudogene
</td>
<td style="text-align:left;">
predicted gene 8309
</td>
</tr>
<tr>
<td style="text-align:left;">
7
</td>
<td style="text-align:left;">
pseudogene
</td>
<td style="text-align:right;">
95.47495
</td>
<td style="text-align:right;">
95.47531
</td>
<td style="text-align:left;">
positive
</td>
<td style="text-align:left;">
MGI_C57BL6J_5753179
</td>
<td style="text-align:left;">
Gm44603
</td>
<td style="text-align:left;">
ENSEMBL:ENSMUSG00000108429
</td>
<td style="text-align:left;">
MGI:5753179
</td>
<td style="text-align:left;">
pseudogene
</td>
<td style="text-align:left;">
predicted gene 44603
</td>
</tr>
<tr>
<td style="text-align:left;">
7
</td>
<td style="text-align:left;">
gene
</td>
<td style="text-align:right;">
95.77545
</td>
<td style="text-align:right;">
95.78117
</td>
<td style="text-align:left;">
positive
</td>
<td style="text-align:left;">
MGI_C57BL6J_5621934
</td>
<td style="text-align:left;">
Gm39049
</td>
<td style="text-align:left;">
NCBI_Gene:105242997
</td>
<td style="text-align:left;">
MGI:5621934
</td>
<td style="text-align:left;">
lncRNA gene
</td>
<td style="text-align:left;">
predicted gene, 39049
</td>
</tr>
<tr>
<td style="text-align:left;">
7
</td>
<td style="text-align:left;">
pseudogene
</td>
<td style="text-align:right;">
95.91275
</td>
<td style="text-align:right;">
95.91339
</td>
<td style="text-align:left;">
negative
</td>
<td style="text-align:left;">
MGI_C57BL6J_3648478
</td>
<td style="text-align:left;">
Gm5037
</td>
<td style="text-align:left;">
NCBI_Gene:260347,ENSEMBL:ENSMUSG00000108628
</td>
<td style="text-align:left;">
MGI:3648478
</td>
<td style="text-align:left;">
pseudogene
</td>
<td style="text-align:left;">
predicted gene 5037
</td>
</tr>
<tr>
<td style="text-align:left;">
7
</td>
<td style="text-align:left;">
gene
</td>
<td style="text-align:right;">
95.95821
</td>
<td style="text-align:right;">
95.96161
</td>
<td style="text-align:left;">
positive
</td>
<td style="text-align:left;">
MGI_C57BL6J_3641871
</td>
<td style="text-align:left;">
Gm9966
</td>
<td style="text-align:left;">
NA
</td>
<td style="text-align:left;">
NA
</td>
<td style="text-align:left;">
protein coding gene
</td>
<td style="text-align:left;">
predicted gene 9966
</td>
</tr>
<tr>
<td style="text-align:left;">
7
</td>
<td style="text-align:left;">
gene
</td>
<td style="text-align:right;">
96.17124
</td>
<td style="text-align:right;">
96.91109
</td>
<td style="text-align:left;">
positive
</td>
<td style="text-align:left;">
MGI_C57BL6J_2447063
</td>
<td style="text-align:left;">
Tenm4
</td>
<td style="text-align:left;">
NCBI_Gene:23966,ENSEMBL:ENSMUSG00000048078
</td>
<td style="text-align:left;">
MGI:2447063
</td>
<td style="text-align:left;">
protein coding gene
</td>
<td style="text-align:left;">
teneurin transmembrane protein 4
</td>
</tr>
<tr>
<td style="text-align:left;">
7
</td>
<td style="text-align:left;">
pseudogene
</td>
<td style="text-align:right;">
96.20239
</td>
<td style="text-align:right;">
96.20364
</td>
<td style="text-align:left;">
negative
</td>
<td style="text-align:left;">
MGI_C57BL6J_3646693
</td>
<td style="text-align:left;">
Gm8319
</td>
<td style="text-align:left;">
NCBI_Gene:666836,ENSEMBL:ENSMUSG00000108528
</td>
<td style="text-align:left;">
MGI:3646693
</td>
<td style="text-align:left;">
pseudogene
</td>
<td style="text-align:left;">
predicted gene 8319
</td>
</tr>
<tr>
<td style="text-align:left;">
7
</td>
<td style="text-align:left;">
gene
</td>
<td style="text-align:right;">
96.24942
</td>
<td style="text-align:right;">
96.24953
</td>
<td style="text-align:left;">
positive
</td>
<td style="text-align:left;">
MGI_C57BL6J_3629686
</td>
<td style="text-align:left;">
Mir708
</td>
<td style="text-align:left;">
miRBase:MI0004692,NCBI_Gene:735284,ENSEMBL:ENSMUSG00000076143
</td>
<td style="text-align:left;">
MGI:3629686
</td>
<td style="text-align:left;">
miRNA gene
</td>
<td style="text-align:left;">
microRNA 708
</td>
</tr>
<tr>
<td style="text-align:left;">
7
</td>
<td style="text-align:left;">
gene
</td>
<td style="text-align:right;">
96.30567
</td>
<td style="text-align:right;">
96.30576
</td>
<td style="text-align:left;">
positive
</td>
<td style="text-align:left;">
MGI_C57BL6J_5531311
</td>
<td style="text-align:left;">
Mir6394
</td>
<td style="text-align:left;">
miRBase:MI0021928,NCBI_Gene:102465210,ENSEMBL:ENSMUSG00000099030
</td>
<td style="text-align:left;">
MGI:5531311
</td>
<td style="text-align:left;">
miRNA gene
</td>
<td style="text-align:left;">
microRNA 6394
</td>
</tr>
<tr>
<td style="text-align:left;">
7
</td>
<td style="text-align:left;">
gene
</td>
<td style="text-align:right;">
96.33948
</td>
<td style="text-align:right;">
96.34196
</td>
<td style="text-align:left;">
negative
</td>
<td style="text-align:left;">
MGI_C57BL6J_3705100
</td>
<td style="text-align:left;">
Gm15412
</td>
<td style="text-align:left;">
NCBI_Gene:670727,ENSEMBL:ENSMUSG00000085751
</td>
<td style="text-align:left;">
MGI:3705100
</td>
<td style="text-align:left;">
lncRNA gene
</td>
<td style="text-align:left;">
predicted gene 15412
</td>
</tr>
<tr>
<td style="text-align:left;">
7
</td>
<td style="text-align:left;">
pseudogene
</td>
<td style="text-align:right;">
96.43482
</td>
<td style="text-align:right;">
96.43675
</td>
<td style="text-align:left;">
negative
</td>
<td style="text-align:left;">
MGI_C57BL6J_3647437
</td>
<td style="text-align:left;">
Rps11-ps5
</td>
<td style="text-align:left;">
NCBI_Gene:102635492,ENSEMBL:ENSMUSG00000082780
</td>
<td style="text-align:left;">
MGI:3647437
</td>
<td style="text-align:left;">
pseudogene
</td>
<td style="text-align:left;">
ribosomal protein S11, pseudogene 5
</td>
</tr>
<tr>
<td style="text-align:left;">
7
</td>
<td style="text-align:left;">
gene
</td>
<td style="text-align:right;">
96.58581
</td>
<td style="text-align:right;">
96.58691
</td>
<td style="text-align:left;">
positive
</td>
<td style="text-align:left;">
MGI_C57BL6J_1924457
</td>
<td style="text-align:left;">
8030425K09Rik
</td>
<td style="text-align:left;">
NA
</td>
<td style="text-align:left;">
NA
</td>
<td style="text-align:left;">
unclassified gene
</td>
<td style="text-align:left;">
RIKEN cDNA 8030425K09 gene
</td>
</tr>
<tr>
<td style="text-align:left;">
7
</td>
<td style="text-align:left;">
gene
</td>
<td style="text-align:right;">
96.59665
</td>
<td style="text-align:right;">
96.62907
</td>
<td style="text-align:left;">
negative
</td>
<td style="text-align:left;">
MGI_C57BL6J_3705307
</td>
<td style="text-align:left;">
Gm15414
</td>
<td style="text-align:left;">
ENSEMBL:ENSMUSG00000085792
</td>
<td style="text-align:left;">
MGI:3705307
</td>
<td style="text-align:left;">
lncRNA gene
</td>
<td style="text-align:left;">
predicted gene 15414
</td>
</tr>
<tr>
<td style="text-align:left;">
7
</td>
<td style="text-align:left;">
gene
</td>
<td style="text-align:right;">
96.79143
</td>
<td style="text-align:right;">
96.80155
</td>
<td style="text-align:left;">
negative
</td>
<td style="text-align:left;">
MGI_C57BL6J_3642665
</td>
<td style="text-align:left;">
Gm15413
</td>
<td style="text-align:left;">
NCBI_Gene:791381,ENSEMBL:ENSMUSG00000053049
</td>
<td style="text-align:left;">
MGI:3642665
</td>
<td style="text-align:left;">
lncRNA gene
</td>
<td style="text-align:left;">
predicted gene 15413
</td>
</tr>
<tr>
<td style="text-align:left;">
7
</td>
<td style="text-align:left;">
gene
</td>
<td style="text-align:right;">
96.81243
</td>
<td style="text-align:right;">
96.81749
</td>
<td style="text-align:left;">
negative
</td>
<td style="text-align:left;">
MGI_C57BL6J_3705308
</td>
<td style="text-align:left;">
Gm15416
</td>
<td style="text-align:left;">
ENSEMBL:ENSMUSG00000085443
</td>
<td style="text-align:left;">
MGI:3705308
</td>
<td style="text-align:left;">
lncRNA gene
</td>
<td style="text-align:left;">
predicted gene 15416
</td>
</tr>
<tr>
<td style="text-align:left;">
7
</td>
<td style="text-align:left;">
gene
</td>
<td style="text-align:right;">
96.81387
</td>
<td style="text-align:right;">
96.81831
</td>
<td style="text-align:left;">
negative
</td>
<td style="text-align:left;">
MGI_C57BL6J_5753209
</td>
<td style="text-align:left;">
Gm44633
</td>
<td style="text-align:left;">
ENSEMBL:ENSMUSG00000108635
</td>
<td style="text-align:left;">
MGI:5753209
</td>
<td style="text-align:left;">
lncRNA gene
</td>
<td style="text-align:left;">
predicted gene 44633
</td>
</tr>
<tr>
<td style="text-align:left;">
7
</td>
<td style="text-align:left;">
gene
</td>
<td style="text-align:right;">
96.86329
</td>
<td style="text-align:right;">
96.86338
</td>
<td style="text-align:left;">
negative
</td>
<td style="text-align:left;">
MGI_C57BL6J_5455489
</td>
<td style="text-align:left;">
Gm25712
</td>
<td style="text-align:left;">
ENSEMBL:ENSMUSG00000089259
</td>
<td style="text-align:left;">
MGI:5455489
</td>
<td style="text-align:left;">
rRNA gene
</td>
<td style="text-align:left;">
predicted gene, 25712
</td>
</tr>
<tr>
<td style="text-align:left;">
7
</td>
<td style="text-align:left;">
gene
</td>
<td style="text-align:right;">
96.90683
</td>
<td style="text-align:right;">
96.94703
</td>
<td style="text-align:left;">
negative
</td>
<td style="text-align:left;">
MGI_C57BL6J_3713275
</td>
<td style="text-align:left;">
Gm15415
</td>
<td style="text-align:left;">
NA
</td>
<td style="text-align:left;">
NA
</td>
<td style="text-align:left;">
antisense lncRNA gene
</td>
<td style="text-align:left;">
predicted gene 15415
</td>
</tr>
<tr>
<td style="text-align:left;">
7
</td>
<td style="text-align:left;">
gene
</td>
<td style="text-align:right;">
96.90683
</td>
<td style="text-align:right;">
96.95189
</td>
<td style="text-align:left;">
negative
</td>
<td style="text-align:left;">
MGI_C57BL6J_3697433
</td>
<td style="text-align:left;">
C230038L03Rik
</td>
<td style="text-align:left;">
NCBI_Gene:108167512,ENSEMBL:ENSMUSG00000085560
</td>
<td style="text-align:left;">
MGI:3697433
</td>
<td style="text-align:left;">
lncRNA gene
</td>
<td style="text-align:left;">
RIKEN cDNA C230038L03 gene
</td>
</tr>
<tr>
<td style="text-align:left;">
7
</td>
<td style="text-align:left;">
gene
</td>
<td style="text-align:right;">
96.91999
</td>
<td style="text-align:right;">
96.92010
</td>
<td style="text-align:left;">
negative
</td>
<td style="text-align:left;">
MGI_C57BL6J_5452003
</td>
<td style="text-align:left;">
Gm22226
</td>
<td style="text-align:left;">
ENSEMBL:ENSMUSG00000087980
</td>
<td style="text-align:left;">
MGI:5452003
</td>
<td style="text-align:left;">
snRNA gene
</td>
<td style="text-align:left;">
predicted gene, 22226
</td>
</tr>
<tr>
<td style="text-align:left;">
7
</td>
<td style="text-align:left;">
gene
</td>
<td style="text-align:right;">
96.95150
</td>
<td style="text-align:right;">
97.06476
</td>
<td style="text-align:left;">
positive
</td>
<td style="text-align:left;">
MGI_C57BL6J_2142075
</td>
<td style="text-align:left;">
Nars2
</td>
<td style="text-align:left;">
NCBI_Gene:244141,ENSEMBL:ENSMUSG00000018995
</td>
<td style="text-align:left;">
MGI:2142075
</td>
<td style="text-align:left;">
protein coding gene
</td>
<td style="text-align:left;">
asparaginyl-tRNA synthetase 2 (mitochondrial)(putative)
</td>
</tr>
<tr>
<td style="text-align:left;">
7
</td>
<td style="text-align:left;">
gene
</td>
<td style="text-align:right;">
97.08159
</td>
<td style="text-align:right;">
97.30895
</td>
<td style="text-align:left;">
positive
</td>
<td style="text-align:left;">
MGI_C57BL6J_1333854
</td>
<td style="text-align:left;">
Gab2
</td>
<td style="text-align:left;">
NCBI_Gene:14389,ENSEMBL:ENSMUSG00000004508
</td>
<td style="text-align:left;">
MGI:1333854
</td>
<td style="text-align:left;">
protein coding gene
</td>
<td style="text-align:left;">
growth factor receptor bound protein 2-associated protein 2
</td>
</tr>
<tr>
<td style="text-align:left;">
7
</td>
<td style="text-align:left;">
gene
</td>
<td style="text-align:right;">
97.30938
</td>
<td style="text-align:right;">
97.33229
</td>
<td style="text-align:left;">
negative
</td>
<td style="text-align:left;">
MGI_C57BL6J_2685339
</td>
<td style="text-align:left;">
Usp35
</td>
<td style="text-align:left;">
NCBI_Gene:244144,ENSEMBL:ENSMUSG00000035713
</td>
<td style="text-align:left;">
MGI:2685339
</td>
<td style="text-align:left;">
protein coding gene
</td>
<td style="text-align:left;">
ubiquitin specific peptidase 35
</td>
</tr>
<tr>
<td style="text-align:left;">
7
</td>
<td style="text-align:left;">
gene
</td>
<td style="text-align:right;">
97.33232
</td>
<td style="text-align:right;">
97.35022
</td>
<td style="text-align:left;">
positive
</td>
<td style="text-align:left;">
MGI_C57BL6J_3643121
</td>
<td style="text-align:left;">
Kctd21
</td>
<td style="text-align:left;">
NCBI_Gene:622320,ENSEMBL:ENSMUSG00000044952
</td>
<td style="text-align:left;">
MGI:3643121
</td>
<td style="text-align:left;">
protein coding gene
</td>
<td style="text-align:left;">
potassium channel tetramerisation domain containing 21
</td>
</tr>
<tr>
<td style="text-align:left;">
7
</td>
<td style="text-align:left;">
gene
</td>
<td style="text-align:right;">
97.37159
</td>
<td style="text-align:right;">
97.39218
</td>
<td style="text-align:left;">
positive
</td>
<td style="text-align:left;">
MGI_C57BL6J_2141959
</td>
<td style="text-align:left;">
Alg8
</td>
<td style="text-align:left;">
NCBI_Gene:381903,ENSEMBL:ENSMUSG00000035704
</td>
<td style="text-align:left;">
MGI:2141959
</td>
<td style="text-align:left;">
protein coding gene
</td>
<td style="text-align:left;">
asparagine-linked glycosylation 8 (alpha-1,3-glucosyltransferase)
</td>
</tr>
<tr>
<td style="text-align:left;">
7
</td>
<td style="text-align:left;">
gene
</td>
<td style="text-align:right;">
97.40000
</td>
<td style="text-align:right;">
97.40780
</td>
<td style="text-align:left;">
positive
</td>
<td style="text-align:left;">
MGI_C57BL6J_1344370
</td>
<td style="text-align:left;">
Ndufc2
</td>
<td style="text-align:left;">
NCBI_Gene:68197,ENSEMBL:ENSMUSG00000030647
</td>
<td style="text-align:left;">
MGI:1344370
</td>
<td style="text-align:left;">
protein coding gene
</td>
<td style="text-align:left;">
NADH:ubiquinone oxidoreductase subunit C2
</td>
</tr>
<tr>
<td style="text-align:left;">
7
</td>
<td style="text-align:left;">
pseudogene
</td>
<td style="text-align:right;">
97.40614
</td>
<td style="text-align:right;">
97.40633
</td>
<td style="text-align:left;">
positive
</td>
<td style="text-align:left;">
MGI_C57BL6J_3802033
</td>
<td style="text-align:left;">
Gm16053
</td>
<td style="text-align:left;">
ENSEMBL:ENSMUSG00000083974
</td>
<td style="text-align:left;">
MGI:3802033
</td>
<td style="text-align:left;">
pseudogene
</td>
<td style="text-align:left;">
predicted gene 16053
</td>
</tr>
<tr>
<td style="text-align:left;">
7
</td>
<td style="text-align:left;">
gene
</td>
<td style="text-align:right;">
97.41293
</td>
<td style="text-align:right;">
97.41773
</td>
<td style="text-align:left;">
negative
</td>
<td style="text-align:left;">
MGI_C57BL6J_109126
</td>
<td style="text-align:left;">
Thrsp
</td>
<td style="text-align:left;">
NCBI_Gene:21835,ENSEMBL:ENSMUSG00000035686
</td>
<td style="text-align:left;">
MGI:109126
</td>
<td style="text-align:left;">
protein coding gene
</td>
<td style="text-align:left;">
thyroid hormone responsive
</td>
</tr>
<tr>
<td style="text-align:left;">
7
</td>
<td style="text-align:left;">
gene
</td>
<td style="text-align:right;">
97.41865
</td>
<td style="text-align:right;">
97.42014
</td>
<td style="text-align:left;">
positive
</td>
<td style="text-align:left;">
MGI_C57BL6J_5592104
</td>
<td style="text-align:left;">
Gm32945
</td>
<td style="text-align:left;">
NCBI_Gene:102635665
</td>
<td style="text-align:left;">
MGI:5592104
</td>
<td style="text-align:left;">
lncRNA gene
</td>
<td style="text-align:left;">
predicted gene, 32945
</td>
</tr>
<tr>
<td style="text-align:left;">
7
</td>
<td style="text-align:left;">
gene
</td>
<td style="text-align:right;">
97.43638
</td>
<td style="text-align:right;">
97.45956
</td>
<td style="text-align:left;">
positive
</td>
<td style="text-align:left;">
MGI_C57BL6J_1289222
</td>
<td style="text-align:left;">
Kctd14
</td>
<td style="text-align:left;">
NCBI_Gene:233529,ENSEMBL:ENSMUSG00000051727
</td>
<td style="text-align:left;">
MGI:1289222
</td>
<td style="text-align:left;">
protein coding gene
</td>
<td style="text-align:left;">
potassium channel tetramerisation domain containing 14
</td>
</tr>
<tr>
<td style="text-align:left;">
7
</td>
<td style="text-align:left;">
gene
</td>
<td style="text-align:right;">
97.45527
</td>
<td style="text-align:right;">
97.45671
</td>
<td style="text-align:left;">
positive
</td>
<td style="text-align:left;">
MGI_C57BL6J_3028073
</td>
<td style="text-align:left;">
7030407A21Rik
</td>
<td style="text-align:left;">
NA
</td>
<td style="text-align:left;">
NA
</td>
<td style="text-align:left;">
unclassified gene
</td>
<td style="text-align:left;">
RIKEN cDNA 7030407A21 gene
</td>
</tr>
<tr>
<td style="text-align:left;">
7
</td>
<td style="text-align:left;">
gene
</td>
<td style="text-align:right;">
97.48095
</td>
<td style="text-align:right;">
97.54140
</td>
<td style="text-align:left;">
positive
</td>
<td style="text-align:left;">
MGI_C57BL6J_1917164
</td>
<td style="text-align:left;">
Ints4
</td>
<td style="text-align:left;">
NCBI_Gene:101861,ENSEMBL:ENSMUSG00000025133
</td>
<td style="text-align:left;">
MGI:1917164
</td>
<td style="text-align:left;">
protein coding gene
</td>
<td style="text-align:left;">
integrator complex subunit 4
</td>
</tr>
<tr>
<td style="text-align:left;">
7
</td>
<td style="text-align:left;">
gene
</td>
<td style="text-align:right;">
97.52181
</td>
<td style="text-align:right;">
97.52192
</td>
<td style="text-align:left;">
positive
</td>
<td style="text-align:left;">
MGI_C57BL6J_5454189
</td>
<td style="text-align:left;">
Gm24412
</td>
<td style="text-align:left;">
ENSEMBL:ENSMUSG00000065360
</td>
<td style="text-align:left;">
MGI:5454189
</td>
<td style="text-align:left;">
snoRNA gene
</td>
<td style="text-align:left;">
predicted gene, 24412
</td>
</tr>
<tr>
<td style="text-align:left;">
7
</td>
<td style="text-align:left;">
gene
</td>
<td style="text-align:right;">
97.55033
</td>
<td style="text-align:right;">
97.57951
</td>
<td style="text-align:left;">
negative
</td>
<td style="text-align:left;">
MGI_C57BL6J_1913523
</td>
<td style="text-align:left;">
Aamdc
</td>
<td style="text-align:left;">
NCBI_Gene:66273,ENSEMBL:ENSMUSG00000035642
</td>
<td style="text-align:left;">
MGI:1913523
</td>
<td style="text-align:left;">
protein coding gene
</td>
<td style="text-align:left;">
adipogenesis associated Mth938 domain containing
</td>
</tr>
<tr>
<td style="text-align:left;">
7
</td>
<td style="text-align:left;">
gene
</td>
<td style="text-align:right;">
97.57977
</td>
<td style="text-align:right;">
97.69278
</td>
<td style="text-align:left;">
positive
</td>
<td style="text-align:left;">
MGI_C57BL6J_2682305
</td>
<td style="text-align:left;">
Rsf1
</td>
<td style="text-align:left;">
NCBI_Gene:233532,ENSEMBL:ENSMUSG00000035623
</td>
<td style="text-align:left;">
MGI:2682305
</td>
<td style="text-align:left;">
protein coding gene
</td>
<td style="text-align:left;">
remodeling and spacing factor 1
</td>
</tr>
<tr>
<td style="text-align:left;">
7
</td>
<td style="text-align:left;">
pseudogene
</td>
<td style="text-align:right;">
97.59149
</td>
<td style="text-align:right;">
97.59245
</td>
<td style="text-align:left;">
negative
</td>
<td style="text-align:left;">
MGI_C57BL6J_5010024
</td>
<td style="text-align:left;">
Ywhaq-ps1
</td>
<td style="text-align:left;">
NCBI_Gene:100415958,ENSEMBL:ENSMUSG00000108405
</td>
<td style="text-align:left;">
MGI:5010024
</td>
<td style="text-align:left;">
pseudogene
</td>
<td style="text-align:left;">
tyrosine 3-monooxygenase/tryptophan 5-monooxygenase activation protein theta, pseudogene 1
</td>
</tr>
<tr>
<td style="text-align:left;">
7
</td>
<td style="text-align:left;">
pseudogene
</td>
<td style="text-align:right;">
97.59484
</td>
<td style="text-align:right;">
97.59544
</td>
<td style="text-align:left;">
negative
</td>
<td style="text-align:left;">
MGI_C57BL6J_5010232
</td>
<td style="text-align:left;">
Rpl15-ps5
</td>
<td style="text-align:left;">
NCBI_Gene:100416334,ENSEMBL:ENSMUSG00000108442
</td>
<td style="text-align:left;">
MGI:5010232
</td>
<td style="text-align:left;">
pseudogene
</td>
<td style="text-align:left;">
ribosomal protein L15, pseudogene 5
</td>
</tr>
<tr>
<td style="text-align:left;">
7
</td>
<td style="text-align:left;">
gene
</td>
<td style="text-align:right;">
97.59835
</td>
<td style="text-align:right;">
97.64511
</td>
<td style="text-align:left;">
negative
</td>
<td style="text-align:left;">
MGI_C57BL6J_1922698
</td>
<td style="text-align:left;">
Rsf1os1
</td>
<td style="text-align:left;">
NCBI_Gene:75448,ENSEMBL:ENSMUSG00000086433
</td>
<td style="text-align:left;">
MGI:1922698
</td>
<td style="text-align:left;">
antisense lncRNA gene
</td>
<td style="text-align:left;">
remodeling and spacing factor 1, opposite strand 1
</td>
</tr>
<tr>
<td style="text-align:left;">
7
</td>
<td style="text-align:left;">
gene
</td>
<td style="text-align:right;">
97.64900
</td>
<td style="text-align:right;">
97.65232
</td>
<td style="text-align:left;">
positive
</td>
<td style="text-align:left;">
MGI_C57BL6J_2441926
</td>
<td style="text-align:left;">
C920008N22Rik
</td>
<td style="text-align:left;">
NA
</td>
<td style="text-align:left;">
NA
</td>
<td style="text-align:left;">
unclassified gene
</td>
<td style="text-align:left;">
RIKEN cDNA C920008N22 gene
</td>
</tr>
<tr>
<td style="text-align:left;">
7
</td>
<td style="text-align:left;">
gene
</td>
<td style="text-align:right;">
97.65001
</td>
<td style="text-align:right;">
97.69683
</td>
<td style="text-align:left;">
negative
</td>
<td style="text-align:left;">
MGI_C57BL6J_3642179
</td>
<td style="text-align:left;">
Rsf1os2
</td>
<td style="text-align:left;">
NCBI_Gene:102641859,ENSEMBL:ENSMUSG00000086993
</td>
<td style="text-align:left;">
MGI:3642179
</td>
<td style="text-align:left;">
antisense lncRNA gene
</td>
<td style="text-align:left;">
remodeling and spacing factor 1, opposite strand 2
</td>
</tr>
<tr>
<td style="text-align:left;">
7
</td>
<td style="text-align:left;">
pseudogene
</td>
<td style="text-align:right;">
97.65428
</td>
<td style="text-align:right;">
97.65475
</td>
<td style="text-align:left;">
negative
</td>
<td style="text-align:left;">
MGI_C57BL6J_5753762
</td>
<td style="text-align:left;">
Gm45186
</td>
<td style="text-align:left;">
ENSEMBL:ENSMUSG00000108547
</td>
<td style="text-align:left;">
MGI:5753762
</td>
<td style="text-align:left;">
pseudogene
</td>
<td style="text-align:left;">
predicted gene 45186
</td>
</tr>
<tr>
<td style="text-align:left;">
7
</td>
<td style="text-align:left;">
gene
</td>
<td style="text-align:right;">
97.69663
</td>
<td style="text-align:right;">
97.72080
</td>
<td style="text-align:left;">
positive
</td>
<td style="text-align:left;">
MGI_C57BL6J_109638
</td>
<td style="text-align:left;">
Clns1a
</td>
<td style="text-align:left;">
NCBI_Gene:12729,ENSEMBL:ENSMUSG00000025439
</td>
<td style="text-align:left;">
MGI:109638
</td>
<td style="text-align:left;">
protein coding gene
</td>
<td style="text-align:left;">
chloride channel, nucleotide-sensitive, 1A
</td>
</tr>
<tr>
<td style="text-align:left;">
7
</td>
<td style="text-align:left;">
gene
</td>
<td style="text-align:right;">
97.72084
</td>
<td style="text-align:right;">
97.72292
</td>
<td style="text-align:left;">
positive
</td>
<td style="text-align:left;">
MGI_C57BL6J_3642272
</td>
<td style="text-align:left;">
Gm9990
</td>
<td style="text-align:left;">
NA
</td>
<td style="text-align:left;">
NA
</td>
<td style="text-align:left;">
protein coding gene
</td>
<td style="text-align:left;">
predicted gene 9990
</td>
</tr>
<tr>
<td style="text-align:left;">
7
</td>
<td style="text-align:left;">
gene
</td>
<td style="text-align:right;">
97.72401
</td>
<td style="text-align:right;">
97.73829
</td>
<td style="text-align:left;">
negative
</td>
<td style="text-align:left;">
MGI_C57BL6J_1913583
</td>
<td style="text-align:left;">
Aqp11
</td>
<td style="text-align:left;">
NCBI_Gene:66333,ENSEMBL:ENSMUSG00000042797
</td>
<td style="text-align:left;">
MGI:1913583
</td>
<td style="text-align:left;">
protein coding gene
</td>
<td style="text-align:left;">
aquaporin 11
</td>
</tr>
<tr>
<td style="text-align:left;">
7
</td>
<td style="text-align:left;">
gene
</td>
<td style="text-align:right;">
97.75932
</td>
<td style="text-align:right;">
97.80116
</td>
<td style="text-align:left;">
negative
</td>
<td style="text-align:left;">
MGI_C57BL6J_5621935
</td>
<td style="text-align:left;">
Gm39050
</td>
<td style="text-align:left;">
NCBI_Gene:105243000
</td>
<td style="text-align:left;">
MGI:5621935
</td>
<td style="text-align:left;">
lncRNA gene
</td>
<td style="text-align:left;">
predicted gene, 39050
</td>
</tr>
<tr>
<td style="text-align:left;">
7
</td>
<td style="text-align:left;">
gene
</td>
<td style="text-align:right;">
97.78854
</td>
<td style="text-align:right;">
97.91238
</td>
<td style="text-align:left;">
positive
</td>
<td style="text-align:left;">
MGI_C57BL6J_1339975
</td>
<td style="text-align:left;">
Pak1
</td>
<td style="text-align:left;">
NCBI_Gene:18479,ENSEMBL:ENSMUSG00000030774
</td>
<td style="text-align:left;">
MGI:1339975
</td>
<td style="text-align:left;">
protein coding gene
</td>
<td style="text-align:left;">
p21 (RAC1) activated kinase 1
</td>
</tr>
<tr>
<td style="text-align:left;">
7
</td>
<td style="text-align:left;">
pseudogene
</td>
<td style="text-align:right;">
97.91824
</td>
<td style="text-align:right;">
97.91863
</td>
<td style="text-align:left;">
positive
</td>
<td style="text-align:left;">
MGI_C57BL6J_5753166
</td>
<td style="text-align:left;">
Gm44590
</td>
<td style="text-align:left;">
ENSEMBL:ENSMUSG00000108424
</td>
<td style="text-align:left;">
MGI:5753166
</td>
<td style="text-align:left;">
pseudogene
</td>
<td style="text-align:left;">
predicted gene 44590
</td>
</tr>
<tr>
<td style="text-align:left;">
7
</td>
<td style="text-align:left;">
gene
</td>
<td style="text-align:right;">
97.91994
</td>
<td style="text-align:right;">
98.04966
</td>
<td style="text-align:left;">
positive
</td>
<td style="text-align:left;">
MGI_C57BL6J_3606573
</td>
<td style="text-align:left;">
Gdpd4
</td>
<td style="text-align:left;">
NCBI_Gene:233537,ENSEMBL:ENSMUSG00000035582
</td>
<td style="text-align:left;">
MGI:3606573
</td>
<td style="text-align:left;">
protein coding gene
</td>
<td style="text-align:left;">
glycerophosphodiester phosphodiesterase domain containing 4
</td>
</tr>
<tr>
<td style="text-align:left;">
7
</td>
<td style="text-align:left;">
pseudogene
</td>
<td style="text-align:right;">
97.98456
</td>
<td style="text-align:right;">
97.98539
</td>
<td style="text-align:left;">
negative
</td>
<td style="text-align:left;">
MGI_C57BL6J_5011389
</td>
<td style="text-align:left;">
Gm19204
</td>
<td style="text-align:left;">
NCBI_Gene:100418424
</td>
<td style="text-align:left;">
MGI:5011389
</td>
<td style="text-align:left;">
pseudogene
</td>
<td style="text-align:left;">
predicted gene, 19204
</td>
</tr>
<tr>
<td style="text-align:left;">
7
</td>
<td style="text-align:left;">
gene
</td>
<td style="text-align:right;">
98.05105
</td>
<td style="text-align:right;">
98.11952
</td>
<td style="text-align:left;">
negative
</td>
<td style="text-align:left;">
MGI_C57BL6J_104510
</td>
<td style="text-align:left;">
Myo7a
</td>
<td style="text-align:left;">
NCBI_Gene:17921,ENSEMBL:ENSMUSG00000030761
</td>
<td style="text-align:left;">
MGI:104510
</td>
<td style="text-align:left;">
protein coding gene
</td>
<td style="text-align:left;">
myosin VIIA
</td>
</tr>
<tr>
<td style="text-align:left;">
7
</td>
<td style="text-align:left;">
gene
</td>
<td style="text-align:right;">
98.12156
</td>
<td style="text-align:right;">
98.17827
</td>
<td style="text-align:left;">
negative
</td>
<td style="text-align:left;">
MGI_C57BL6J_1100859
</td>
<td style="text-align:left;">
Capn5
</td>
<td style="text-align:left;">
NCBI_Gene:12337,ENSEMBL:ENSMUSG00000035547
</td>
<td style="text-align:left;">
MGI:1100859
</td>
<td style="text-align:left;">
protein coding gene
</td>
<td style="text-align:left;">
calpain 5
</td>
</tr>
<tr>
<td style="text-align:left;">
7
</td>
<td style="text-align:left;">
gene
</td>
<td style="text-align:right;">
98.14336
</td>
<td style="text-align:right;">
98.14550
</td>
<td style="text-align:left;">
negative
</td>
<td style="text-align:left;">
MGI_C57BL6J_97436
</td>
<td style="text-align:left;">
Omp
</td>
<td style="text-align:left;">
NCBI_Gene:18378,ENSEMBL:ENSMUSG00000074006
</td>
<td style="text-align:left;">
MGI:97436
</td>
<td style="text-align:left;">
protein coding gene
</td>
<td style="text-align:left;">
olfactory marker protein
</td>
</tr>
<tr>
<td style="text-align:left;">
7
</td>
<td style="text-align:left;">
gene
</td>
<td style="text-align:right;">
98.17719
</td>
<td style="text-align:right;">
98.18480
</td>
<td style="text-align:left;">
positive
</td>
<td style="text-align:left;">
MGI_C57BL6J_4439862
</td>
<td style="text-align:left;">
Gm16938
</td>
<td style="text-align:left;">
NCBI_Gene:330599,ENSEMBL:ENSMUSG00000097749
</td>
<td style="text-align:left;">
MGI:4439862
</td>
<td style="text-align:left;">
lncRNA gene
</td>
<td style="text-align:left;">
predicted gene, 16938
</td>
</tr>
<tr>
<td style="text-align:left;">
7
</td>
<td style="text-align:left;">
gene
</td>
<td style="text-align:right;">
98.19241
</td>
<td style="text-align:right;">
98.19948
</td>
<td style="text-align:left;">
negative
</td>
<td style="text-align:left;">
MGI_C57BL6J_3039603
</td>
<td style="text-align:left;">
B3gnt6
</td>
<td style="text-align:left;">
NCBI_Gene:272411,ENSEMBL:ENSMUSG00000074004
</td>
<td style="text-align:left;">
MGI:3039603
</td>
<td style="text-align:left;">
protein coding gene
</td>
<td style="text-align:left;">
UDP-GlcNAc:betaGal beta-1,3-N-acetylglucosaminyltransferase 6 (core 3 synthase)
</td>
</tr>
<tr>
<td style="text-align:left;">
7
</td>
<td style="text-align:left;">
gene
</td>
<td style="text-align:right;">
98.20639
</td>
<td style="text-align:right;">
98.21038
</td>
<td style="text-align:left;">
negative
</td>
<td style="text-align:left;">
MGI_C57BL6J_2142310
</td>
<td style="text-align:left;">
E230006M18Rik
</td>
<td style="text-align:left;">
NA
</td>
<td style="text-align:left;">
NA
</td>
<td style="text-align:left;">
unclassified gene
</td>
<td style="text-align:left;">
RIKEN cDNA E230006M18 gene
</td>
</tr>
<tr>
<td style="text-align:left;">
7
</td>
<td style="text-align:left;">
gene
</td>
<td style="text-align:right;">
98.20639
</td>
<td style="text-align:right;">
98.32121
</td>
<td style="text-align:left;">
negative
</td>
<td style="text-align:left;">
MGI_C57BL6J_1913440
</td>
<td style="text-align:left;">
Acer3
</td>
<td style="text-align:left;">
NCBI_Gene:66190,ENSEMBL:ENSMUSG00000030760
</td>
<td style="text-align:left;">
MGI:1913440
</td>
<td style="text-align:left;">
protein coding gene
</td>
<td style="text-align:left;">
alkaline ceramidase 3
</td>
</tr>
<tr>
<td style="text-align:left;">
7
</td>
<td style="text-align:left;">
gene
</td>
<td style="text-align:right;">
98.35067
</td>
<td style="text-align:right;">
98.36139
</td>
<td style="text-align:left;">
negative
</td>
<td style="text-align:left;">
MGI_C57BL6J_2443855
</td>
<td style="text-align:left;">
Tsku
</td>
<td style="text-align:left;">
NCBI_Gene:244152,ENSEMBL:ENSMUSG00000049580
</td>
<td style="text-align:left;">
MGI:2443855
</td>
<td style="text-align:left;">
protein coding gene
</td>
<td style="text-align:left;">
tsukushi, small leucine rich proteoglycan
</td>
</tr>
<tr>
<td style="text-align:left;">
7
</td>
<td style="text-align:left;">
gene
</td>
<td style="text-align:right;">
98.36699
</td>
<td style="text-align:right;">
98.41980
</td>
<td style="text-align:left;">
negative
</td>
<td style="text-align:left;">
MGI_C57BL6J_5753083
</td>
<td style="text-align:left;">
Gm44507
</td>
<td style="text-align:left;">
NCBI_Gene:108167447,ENSEMBL:ENSMUSG00000108449
</td>
<td style="text-align:left;">
MGI:5753083
</td>
<td style="text-align:left;">
lncRNA gene
</td>
<td style="text-align:left;">
predicted gene 44507
</td>
</tr>
<tr>
<td style="text-align:left;">
7
</td>
<td style="text-align:left;">
gene
</td>
<td style="text-align:right;">
98.43348
</td>
<td style="text-align:right;">
98.43795
</td>
<td style="text-align:left;">
positive
</td>
<td style="text-align:left;">
MGI_C57BL6J_5621938
</td>
<td style="text-align:left;">
Gm39053
</td>
<td style="text-align:left;">
NCBI_Gene:105243003
</td>
<td style="text-align:left;">
MGI:5621938
</td>
<td style="text-align:left;">
lncRNA gene
</td>
<td style="text-align:left;">
predicted gene, 39053
</td>
</tr>
<tr>
<td style="text-align:left;">
7
</td>
<td style="text-align:left;">
pseudogene
</td>
<td style="text-align:right;">
98.43721
</td>
<td style="text-align:right;">
98.43752
</td>
<td style="text-align:left;">
negative
</td>
<td style="text-align:left;">
MGI_C57BL6J_5753556
</td>
<td style="text-align:left;">
Gm44980
</td>
<td style="text-align:left;">
ENSEMBL:ENSMUSG00000108346
</td>
<td style="text-align:left;">
MGI:5753556
</td>
<td style="text-align:left;">
pseudogene
</td>
<td style="text-align:left;">
predicted gene 44980
</td>
</tr>
<tr>
<td style="text-align:left;">
7
</td>
<td style="text-align:left;">
gene
</td>
<td style="text-align:right;">
98.44042
</td>
<td style="text-align:right;">
98.47748
</td>
<td style="text-align:left;">
positive
</td>
<td style="text-align:left;">
MGI_C57BL6J_106030
</td>
<td style="text-align:left;">
Gucy2d
</td>
<td style="text-align:left;">
NCBI_Gene:14918,ENSEMBL:ENSMUSG00000074003
</td>
<td style="text-align:left;">
MGI:106030
</td>
<td style="text-align:left;">
protein coding gene
</td>
<td style="text-align:left;">
guanylate cyclase 2d
</td>
</tr>
<tr>
<td style="text-align:left;">
7
</td>
<td style="text-align:left;">
pseudogene
</td>
<td style="text-align:right;">
98.45509
</td>
<td style="text-align:right;">
98.45568
</td>
<td style="text-align:left;">
positive
</td>
<td style="text-align:left;">
MGI_C57BL6J_3781104
</td>
<td style="text-align:left;">
Gm2926
</td>
<td style="text-align:left;">
NCBI_Gene:100040721,ENSEMBL:ENSMUSG00000108764
</td>
<td style="text-align:left;">
MGI:3781104
</td>
<td style="text-align:left;">
pseudogene
</td>
<td style="text-align:left;">
predicted gene 2926
</td>
</tr>
<tr>
<td style="text-align:left;">
7
</td>
<td style="text-align:left;">
pseudogene
</td>
<td style="text-align:right;">
98.46513
</td>
<td style="text-align:right;">
98.46648
</td>
<td style="text-align:left;">
negative
</td>
<td style="text-align:left;">
MGI_C57BL6J_5753554
</td>
<td style="text-align:left;">
Gm44978
</td>
<td style="text-align:left;">
NCBI_Gene:108167471,ENSEMBL:ENSMUSG00000108508
</td>
<td style="text-align:left;">
MGI:5753554
</td>
<td style="text-align:left;">
pseudogene
</td>
<td style="text-align:left;">
predicted gene 44978
</td>
</tr>
<tr>
<td style="text-align:left;">
7
</td>
<td style="text-align:left;">
gene
</td>
<td style="text-align:right;">
98.48126
</td>
<td style="text-align:right;">
98.48622
</td>
<td style="text-align:left;">
positive
</td>
<td style="text-align:left;">
MGI_C57BL6J_5621473
</td>
<td style="text-align:left;">
Gm38588
</td>
<td style="text-align:left;">
NCBI_Gene:102641966
</td>
<td style="text-align:left;">
MGI:5621473
</td>
<td style="text-align:left;">
lncRNA gene
</td>
<td style="text-align:left;">
predicted gene, 38588
</td>
</tr>
<tr>
<td style="text-align:left;">
7
</td>
<td style="text-align:left;">
gene
</td>
<td style="text-align:right;">
98.48928
</td>
<td style="text-align:right;">
98.50218
</td>
<td style="text-align:left;">
positive
</td>
<td style="text-align:left;">
MGI_C57BL6J_93882
</td>
<td style="text-align:left;">
Lrrc32
</td>
<td style="text-align:left;">
NCBI_Gene:434215,ENSEMBL:ENSMUSG00000090958
</td>
<td style="text-align:left;">
MGI:93882
</td>
<td style="text-align:left;">
protein coding gene
</td>
<td style="text-align:left;">
leucine rich repeat containing 32
</td>
</tr>
<tr>
<td style="text-align:left;">
7
</td>
<td style="text-align:left;">
gene
</td>
<td style="text-align:right;">
98.50259
</td>
<td style="text-align:right;">
98.50665
</td>
<td style="text-align:left;">
negative
</td>
<td style="text-align:left;">
MGI_C57BL6J_5592553
</td>
<td style="text-align:left;">
Gm33394
</td>
<td style="text-align:left;">
NCBI_Gene:102636286
</td>
<td style="text-align:left;">
MGI:5592553
</td>
<td style="text-align:left;">
lncRNA gene
</td>
<td style="text-align:left;">
predicted gene, 33394
</td>
</tr>
<tr>
<td style="text-align:left;">
7
</td>
<td style="text-align:left;">
gene
</td>
<td style="text-align:right;">
98.50695
</td>
<td style="text-align:right;">
98.50712
</td>
<td style="text-align:left;">
positive
</td>
<td style="text-align:left;">
MGI_C57BL6J_5452817
</td>
<td style="text-align:left;">
Gm23040
</td>
<td style="text-align:left;">
ENSEMBL:ENSMUSG00000065803
</td>
<td style="text-align:left;">
MGI:5452817
</td>
<td style="text-align:left;">
snRNA gene
</td>
<td style="text-align:left;">
predicted gene, 23040
</td>
</tr>
<tr>
<td style="text-align:left;">
7
</td>
<td style="text-align:left;">
gene
</td>
<td style="text-align:right;">
98.54332
</td>
<td style="text-align:right;">
98.56299
</td>
<td style="text-align:left;">
negative
</td>
<td style="text-align:left;">
MGI_C57BL6J_2442837
</td>
<td style="text-align:left;">
A630091E08Rik
</td>
<td style="text-align:left;">
ENSEMBL:ENSMUSG00000055446
</td>
<td style="text-align:left;">
MGI:2442837
</td>
<td style="text-align:left;">
lncRNA gene
</td>
<td style="text-align:left;">
RIKEN cDNA A630091E08 gene
</td>
</tr>
<tr>
<td style="text-align:left;">
7
</td>
<td style="text-align:left;">
gene
</td>
<td style="text-align:right;">
98.58713
</td>
<td style="text-align:right;">
98.65956
</td>
<td style="text-align:left;">
negative
</td>
<td style="text-align:left;">
MGI_C57BL6J_1924203
</td>
<td style="text-align:left;">
Emsy
</td>
<td style="text-align:left;">
NCBI_Gene:233545,ENSEMBL:ENSMUSG00000035401
</td>
<td style="text-align:left;">
MGI:1924203
</td>
<td style="text-align:left;">
protein coding gene
</td>
<td style="text-align:left;">
EMSY, BRCA2-interacting transcriptional repressor
</td>
</tr>
<tr>
<td style="text-align:left;">
7
</td>
<td style="text-align:left;">
gene
</td>
<td style="text-align:right;">
98.65620
</td>
<td style="text-align:right;">
98.65783
</td>
<td style="text-align:left;">
positive
</td>
<td style="text-align:left;">
MGI_C57BL6J_1923090
</td>
<td style="text-align:left;">
4930558N01Rik
</td>
<td style="text-align:left;">
NA
</td>
<td style="text-align:left;">
NA
</td>
<td style="text-align:left;">
unclassified gene
</td>
<td style="text-align:left;">
RIKEN cDNA 4930558N01 gene
</td>
</tr>
<tr>
<td style="text-align:left;">
7
</td>
<td style="text-align:left;">
gene
</td>
<td style="text-align:right;">
98.67811
</td>
<td style="text-align:right;">
98.68065
</td>
<td style="text-align:left;">
negative
</td>
<td style="text-align:left;">
MGI_C57BL6J_5753708
</td>
<td style="text-align:left;">
Gm45132
</td>
<td style="text-align:left;">
ENSEMBL:ENSMUSG00000109101
</td>
<td style="text-align:left;">
MGI:5753708
</td>
<td style="text-align:left;">
unclassified gene
</td>
<td style="text-align:left;">
predicted gene 45132
</td>
</tr>
<tr>
<td style="text-align:left;">
7
</td>
<td style="text-align:left;">
gene
</td>
<td style="text-align:right;">
98.68233
</td>
<td style="text-align:right;">
98.70293
</td>
<td style="text-align:left;">
negative
</td>
<td style="text-align:left;">
MGI_C57BL6J_3782954
</td>
<td style="text-align:left;">
Gm15506
</td>
<td style="text-align:left;">
NCBI_Gene:100040769,ENSEMBL:ENSMUSG00000086477
</td>
<td style="text-align:left;">
MGI:3782954
</td>
<td style="text-align:left;">
lncRNA gene
</td>
<td style="text-align:left;">
predicted gene 15506
</td>
</tr>
<tr>
<td style="text-align:left;">
7
</td>
<td style="text-align:left;">
gene
</td>
<td style="text-align:right;">
98.70310
</td>
<td style="text-align:right;">
98.71806
</td>
<td style="text-align:left;">
positive
</td>
<td style="text-align:left;">
MGI_C57BL6J_1920231
</td>
<td style="text-align:left;">
Thap12
</td>
<td style="text-align:left;">
NCBI_Gene:72981,ENSEMBL:ENSMUSG00000030753
</td>
<td style="text-align:left;">
MGI:1920231
</td>
<td style="text-align:left;">
protein coding gene
</td>
<td style="text-align:left;">
THAP domain containing 12
</td>
</tr>
<tr>
<td style="text-align:left;">
7
</td>
<td style="text-align:left;">
gene
</td>
<td style="text-align:right;">
98.71950
</td>
<td style="text-align:right;">
98.72876
</td>
<td style="text-align:left;">
negative
</td>
<td style="text-align:left;">
MGI_C57BL6J_5011841
</td>
<td style="text-align:left;">
Gm19656
</td>
<td style="text-align:left;">
NCBI_Gene:105243004,ENSEMBL:ENSMUSG00000109149
</td>
<td style="text-align:left;">
MGI:5011841
</td>
<td style="text-align:left;">
lncRNA gene
</td>
<td style="text-align:left;">
predicted gene, 19656
</td>
</tr>
<tr>
<td style="text-align:left;">
7
</td>
<td style="text-align:left;">
gene
</td>
<td style="text-align:right;">
98.73475
</td>
<td style="text-align:right;">
98.73508
</td>
<td style="text-align:left;">
negative
</td>
<td style="text-align:left;">
MGI_C57BL6J_5753510
</td>
<td style="text-align:left;">
Gm44934
</td>
<td style="text-align:left;">
ENSEMBL:ENSMUSG00000108988
</td>
<td style="text-align:left;">
MGI:5753510
</td>
<td style="text-align:left;">
lncRNA gene
</td>
<td style="text-align:left;">
predicted gene 44934
</td>
</tr>
<tr>
<td style="text-align:left;">
7
</td>
<td style="text-align:left;">
pseudogene
</td>
<td style="text-align:right;">
98.76148
</td>
<td style="text-align:right;">
98.76194
</td>
<td style="text-align:left;">
positive
</td>
<td style="text-align:left;">
MGI_C57BL6J_5011435
</td>
<td style="text-align:left;">
Gm19250
</td>
<td style="text-align:left;">
NCBI_Gene:100462934
</td>
<td style="text-align:left;">
MGI:5011435
</td>
<td style="text-align:left;">
pseudogene
</td>
<td style="text-align:left;">
predicted gene, 19250
</td>
</tr>
<tr>
<td style="text-align:left;">
7
</td>
<td style="text-align:left;">
gene
</td>
<td style="text-align:right;">
98.76302
</td>
<td style="text-align:right;">
98.76984
</td>
<td style="text-align:left;">
positive
</td>
<td style="text-align:left;">
MGI_C57BL6J_5592879
</td>
<td style="text-align:left;">
Gm33720
</td>
<td style="text-align:left;">
NCBI_Gene:102636728
</td>
<td style="text-align:left;">
MGI:5592879
</td>
<td style="text-align:left;">
lncRNA gene
</td>
<td style="text-align:left;">
predicted gene, 33720
</td>
</tr>
<tr>
<td style="text-align:left;">
7
</td>
<td style="text-align:left;">
gene
</td>
<td style="text-align:right;">
98.81542
</td>
<td style="text-align:right;">
98.81549
</td>
<td style="text-align:left;">
positive
</td>
<td style="text-align:left;">
MGI_C57BL6J_4414020
</td>
<td style="text-align:left;">
n-TPtgg8
</td>
<td style="text-align:left;">
NCBI_Gene:102467371
</td>
<td style="text-align:left;">
MGI:4414020
</td>
<td style="text-align:left;">
tRNA gene
</td>
<td style="text-align:left;">
nuclear encoded tRNA proline 8 (anticodon TGG)
</td>
</tr>
<tr>
<td style="text-align:left;">
7
</td>
<td style="text-align:left;">
gene
</td>
<td style="text-align:right;">
98.81568
</td>
<td style="text-align:right;">
98.81575
</td>
<td style="text-align:left;">
negative
</td>
<td style="text-align:left;">
MGI_C57BL6J_4414009
</td>
<td style="text-align:left;">
n-TPagg7
</td>
<td style="text-align:left;">
NCBI_Gene:102467367
</td>
<td style="text-align:left;">
MGI:4414009
</td>
<td style="text-align:left;">
tRNA gene
</td>
<td style="text-align:left;">
nuclear encoded tRNA proline 7 (anticodon AGG)
</td>
</tr>
<tr>
<td style="text-align:left;">
7
</td>
<td style="text-align:left;">
gene
</td>
<td style="text-align:right;">
98.81613
</td>
<td style="text-align:right;">
98.82193
</td>
<td style="text-align:left;">
positive
</td>
<td style="text-align:left;">
MGI_C57BL6J_5621939
</td>
<td style="text-align:left;">
Gm39054
</td>
<td style="text-align:left;">
NCBI_Gene:105243005
</td>
<td style="text-align:left;">
MGI:5621939
</td>
<td style="text-align:left;">
lncRNA gene
</td>
<td style="text-align:left;">
predicted gene, 39054
</td>
</tr>
<tr>
<td style="text-align:left;">
7
</td>
<td style="text-align:left;">
gene
</td>
<td style="text-align:right;">
98.83066
</td>
<td style="text-align:right;">
98.83606
</td>
<td style="text-align:left;">
negative
</td>
<td style="text-align:left;">
MGI_C57BL6J_5592929
</td>
<td style="text-align:left;">
Gm33770
</td>
<td style="text-align:left;">
NCBI_Gene:102636798
</td>
<td style="text-align:left;">
MGI:5592929
</td>
<td style="text-align:left;">
lncRNA gene
</td>
<td style="text-align:left;">
predicted gene, 33770
</td>
</tr>
<tr>
<td style="text-align:left;">
7
</td>
<td style="text-align:left;">
gene
</td>
<td style="text-align:right;">
98.83492
</td>
<td style="text-align:right;">
98.85519
</td>
<td style="text-align:left;">
positive
</td>
<td style="text-align:left;">
MGI_C57BL6J_101948
</td>
<td style="text-align:left;">
Wnt11
</td>
<td style="text-align:left;">
NCBI_Gene:22411,ENSEMBL:ENSMUSG00000015957
</td>
<td style="text-align:left;">
MGI:101948
</td>
<td style="text-align:left;">
protein coding gene
</td>
<td style="text-align:left;">
wingless-type MMTV integration site family, member 11
</td>
</tr>
<tr>
<td style="text-align:left;">
7
</td>
<td style="text-align:left;">
pseudogene
</td>
<td style="text-align:right;">
98.85473
</td>
<td style="text-align:right;">
98.85510
</td>
<td style="text-align:left;">
negative
</td>
<td style="text-align:left;">
MGI_C57BL6J_3648175
</td>
<td style="text-align:left;">
Gm8398
</td>
<td style="text-align:left;">
ENSEMBL:ENSMUSG00000091496
</td>
<td style="text-align:left;">
MGI:3648175
</td>
<td style="text-align:left;">
pseudogene
</td>
<td style="text-align:left;">
predicted gene 8398
</td>
</tr>
<tr>
<td style="text-align:left;">
7
</td>
<td style="text-align:left;">
gene
</td>
<td style="text-align:right;">
98.86926
</td>
<td style="text-align:right;">
98.87663
</td>
<td style="text-align:left;">
negative
</td>
<td style="text-align:left;">
MGI_C57BL6J_5592986
</td>
<td style="text-align:left;">
Gm33827
</td>
<td style="text-align:left;">
NCBI_Gene:102636877
</td>
<td style="text-align:left;">
MGI:5592986
</td>
<td style="text-align:left;">
lncRNA gene
</td>
<td style="text-align:left;">
predicted gene, 33827
</td>
</tr>
<tr>
<td style="text-align:left;">
7
</td>
<td style="text-align:left;">
gene
</td>
<td style="text-align:right;">
98.88502
</td>
<td style="text-align:right;">
99.14117
</td>
<td style="text-align:left;">
negative
</td>
<td style="text-align:left;">
MGI_C57BL6J_1925860
</td>
<td style="text-align:left;">
Uvrag
</td>
<td style="text-align:left;">
NCBI_Gene:78610,ENSEMBL:ENSMUSG00000035354
</td>
<td style="text-align:left;">
MGI:1925860
</td>
<td style="text-align:left;">
protein coding gene
</td>
<td style="text-align:left;">
UV radiation resistance associated gene
</td>
</tr>
<tr>
<td style="text-align:left;">
7
</td>
<td style="text-align:left;">
gene
</td>
<td style="text-align:right;">
98.89522
</td>
<td style="text-align:right;">
98.90649
</td>
<td style="text-align:left;">
positive
</td>
<td style="text-align:left;">
MGI_C57BL6J_3647698
</td>
<td style="text-align:left;">
Gm8149
</td>
<td style="text-align:left;">
NCBI_Gene:666529
</td>
<td style="text-align:left;">
MGI:3647698
</td>
<td style="text-align:left;">
unclassified gene
</td>
<td style="text-align:left;">
predicted gene 8149
</td>
</tr>
<tr>
<td style="text-align:left;">
7
</td>
<td style="text-align:left;">
gene
</td>
<td style="text-align:right;">
98.89593
</td>
<td style="text-align:right;">
98.89955
</td>
<td style="text-align:left;">
positive
</td>
<td style="text-align:left;">
MGI_C57BL6J_5753764
</td>
<td style="text-align:left;">
Gm45188
</td>
<td style="text-align:left;">
ENSEMBL:ENSMUSG00000109282
</td>
<td style="text-align:left;">
MGI:5753764
</td>
<td style="text-align:left;">
lncRNA gene
</td>
<td style="text-align:left;">
predicted gene 45188
</td>
</tr>
<tr>
<td style="text-align:left;">
7
</td>
<td style="text-align:left;">
gene
</td>
<td style="text-align:right;">
98.90427
</td>
<td style="text-align:right;">
98.90806
</td>
<td style="text-align:left;">
positive
</td>
<td style="text-align:left;">
MGI_C57BL6J_5753763
</td>
<td style="text-align:left;">
Gm45187
</td>
<td style="text-align:left;">
ENSEMBL:ENSMUSG00000109130
</td>
<td style="text-align:left;">
MGI:5753763
</td>
<td style="text-align:left;">
lncRNA gene
</td>
<td style="text-align:left;">
predicted gene 45187
</td>
</tr>
<tr>
<td style="text-align:left;">
7
</td>
<td style="text-align:left;">
gene
</td>
<td style="text-align:right;">
98.90659
</td>
<td style="text-align:right;">
98.91100
</td>
<td style="text-align:left;">
positive
</td>
<td style="text-align:left;">
MGI_C57BL6J_5621940
</td>
<td style="text-align:left;">
Gm39055
</td>
<td style="text-align:left;">
NCBI_Gene:105243006
</td>
<td style="text-align:left;">
MGI:5621940
</td>
<td style="text-align:left;">
lncRNA gene
</td>
<td style="text-align:left;">
predicted gene, 39055
</td>
</tr>
<tr>
<td style="text-align:left;">
7
</td>
<td style="text-align:left;">
gene
</td>
<td style="text-align:right;">
98.93589
</td>
<td style="text-align:right;">
98.93601
</td>
<td style="text-align:left;">
positive
</td>
<td style="text-align:left;">
MGI_C57BL6J_5453256
</td>
<td style="text-align:left;">
Gm23479
</td>
<td style="text-align:left;">
ENSEMBL:ENSMUSG00000089501
</td>
<td style="text-align:left;">
MGI:5453256
</td>
<td style="text-align:left;">
snoRNA gene
</td>
<td style="text-align:left;">
predicted gene, 23479
</td>
</tr>
<tr>
<td style="text-align:left;">
7
</td>
<td style="text-align:left;">
pseudogene
</td>
<td style="text-align:right;">
99.08946
</td>
<td style="text-align:right;">
99.09088
</td>
<td style="text-align:left;">
positive
</td>
<td style="text-align:left;">
MGI_C57BL6J_5593041
</td>
<td style="text-align:left;">
Gm33882
</td>
<td style="text-align:left;">
NCBI_Gene:102636953,ENSEMBL:ENSMUSG00000109492
</td>
<td style="text-align:left;">
MGI:5593041
</td>
<td style="text-align:left;">
pseudogene
</td>
<td style="text-align:left;">
predicted gene, 33882
</td>
</tr>
<tr>
<td style="text-align:left;">
7
</td>
<td style="text-align:left;">
gene
</td>
<td style="text-align:right;">
99.12207
</td>
<td style="text-align:right;">
99.12346
</td>
<td style="text-align:left;">
negative
</td>
<td style="text-align:left;">
MGI_C57BL6J_5753551
</td>
<td style="text-align:left;">
Gm44975
</td>
<td style="text-align:left;">
ENSEMBL:ENSMUSG00000109073
</td>
<td style="text-align:left;">
MGI:5753551
</td>
<td style="text-align:left;">
unclassified gene
</td>
<td style="text-align:left;">
predicted gene 44975
</td>
</tr>
<tr>
<td style="text-align:left;">
7
</td>
<td style="text-align:left;">
gene
</td>
<td style="text-align:right;">
99.14159
</td>
<td style="text-align:right;">
99.14200
</td>
<td style="text-align:left;">
positive
</td>
<td style="text-align:left;">
MGI_C57BL6J_5753761
</td>
<td style="text-align:left;">
Gm45185
</td>
<td style="text-align:left;">
ENSEMBL:ENSMUSG00000109468
</td>
<td style="text-align:left;">
MGI:5753761
</td>
<td style="text-align:left;">
unclassified gene
</td>
<td style="text-align:left;">
predicted gene 45185
</td>
</tr>
<tr>
<td style="text-align:left;">
7
</td>
<td style="text-align:left;">
gene
</td>
<td style="text-align:right;">
99.15366
</td>
<td style="text-align:right;">
99.18272
</td>
<td style="text-align:left;">
negative
</td>
<td style="text-align:left;">
MGI_C57BL6J_1915050
</td>
<td style="text-align:left;">
Dgat2
</td>
<td style="text-align:left;">
NCBI_Gene:67800,ENSEMBL:ENSMUSG00000030747
</td>
<td style="text-align:left;">
MGI:1915050
</td>
<td style="text-align:left;">
protein coding gene
</td>
<td style="text-align:left;">
diacylglycerol O-acyltransferase 2
</td>
</tr>
<tr>
<td style="text-align:left;">
7
</td>
<td style="text-align:left;">
gene
</td>
<td style="text-align:right;">
99.20237
</td>
<td style="text-align:right;">
99.22708
</td>
<td style="text-align:left;">
positive
</td>
<td style="text-align:left;">
MGI_C57BL6J_5753588
</td>
<td style="text-align:left;">
Gm45012
</td>
<td style="text-align:left;">
ENSEMBL:ENSMUSG00000109052
</td>
<td style="text-align:left;">
MGI:5753588
</td>
<td style="text-align:left;">
lncRNA gene
</td>
<td style="text-align:left;">
predicted gene 45012
</td>
</tr>
<tr>
<td style="text-align:left;">
7
</td>
<td style="text-align:left;">
gene
</td>
<td style="text-align:right;">
99.21908
</td>
<td style="text-align:right;">
99.23862
</td>
<td style="text-align:left;">
negative
</td>
<td style="text-align:left;">
MGI_C57BL6J_2663253
</td>
<td style="text-align:left;">
Mogat2
</td>
<td style="text-align:left;">
NCBI_Gene:233549,ENSEMBL:ENSMUSG00000052396
</td>
<td style="text-align:left;">
MGI:2663253
</td>
<td style="text-align:left;">
protein coding gene
</td>
<td style="text-align:left;">
monoacylglycerol O-acyltransferase 2
</td>
</tr>
<tr>
<td style="text-align:left;">
7
</td>
<td style="text-align:left;">
pseudogene
</td>
<td style="text-align:right;">
99.24757
</td>
<td style="text-align:right;">
99.24897
</td>
<td style="text-align:left;">
negative
</td>
<td style="text-align:left;">
MGI_C57BL6J_5011128
</td>
<td style="text-align:left;">
Gm18943
</td>
<td style="text-align:left;">
NCBI_Gene:100418007,ENSEMBL:ENSMUSG00000109472
</td>
<td style="text-align:left;">
MGI:5011128
</td>
<td style="text-align:left;">
pseudogene
</td>
<td style="text-align:left;">
predicted gene, 18943
</td>
</tr>
<tr>
<td style="text-align:left;">
7
</td>
<td style="text-align:left;">
gene
</td>
<td style="text-align:right;">
99.25151
</td>
<td style="text-align:right;">
99.25519
</td>
<td style="text-align:left;">
positive
</td>
<td style="text-align:left;">
MGI_C57BL6J_5593221
</td>
<td style="text-align:left;">
Gm34062
</td>
<td style="text-align:left;">
NCBI_Gene:102637185
</td>
<td style="text-align:left;">
MGI:5593221
</td>
<td style="text-align:left;">
lncRNA gene
</td>
<td style="text-align:left;">
predicted gene, 34062
</td>
</tr>
<tr>
<td style="text-align:left;">
7
</td>
<td style="text-align:left;">
gene
</td>
<td style="text-align:right;">
99.26048
</td>
<td style="text-align:right;">
99.26722
</td>
<td style="text-align:left;">
negative
</td>
<td style="text-align:left;">
MGI_C57BL6J_5477199
</td>
<td style="text-align:left;">
Gm26705
</td>
<td style="text-align:left;">
NCBI_Gene:381967,ENSEMBL:ENSMUSG00000097015
</td>
<td style="text-align:left;">
MGI:5477199
</td>
<td style="text-align:left;">
lncRNA gene
</td>
<td style="text-align:left;">
predicted gene, 26705
</td>
</tr>
<tr>
<td style="text-align:left;">
7
</td>
<td style="text-align:left;">
gene
</td>
<td style="text-align:right;">
99.26718
</td>
<td style="text-align:right;">
99.33714
</td>
<td style="text-align:left;">
positive
</td>
<td style="text-align:left;">
MGI_C57BL6J_1201690
</td>
<td style="text-align:left;">
Map6
</td>
<td style="text-align:left;">
NCBI_Gene:17760,ENSEMBL:ENSMUSG00000055407
</td>
<td style="text-align:left;">
MGI:1201690
</td>
<td style="text-align:left;">
protein coding gene
</td>
<td style="text-align:left;">
microtubule-associated protein 6
</td>
</tr>
<tr>
<td style="text-align:left;">
7
</td>
<td style="text-align:left;">
gene
</td>
<td style="text-align:right;">
99.30027
</td>
<td style="text-align:right;">
99.30449
</td>
<td style="text-align:left;">
negative
</td>
<td style="text-align:left;">
MGI_C57BL6J_5623334
</td>
<td style="text-align:left;">
Gm40449
</td>
<td style="text-align:left;">
NCBI_Gene:105244927
</td>
<td style="text-align:left;">
MGI:5623334
</td>
<td style="text-align:left;">
lncRNA gene
</td>
<td style="text-align:left;">
predicted gene, 40449
</td>
</tr>
<tr>
<td style="text-align:left;">
7
</td>
<td style="text-align:left;">
gene
</td>
<td style="text-align:right;">
99.32120
</td>
<td style="text-align:right;">
99.32246
</td>
<td style="text-align:left;">
positive
</td>
<td style="text-align:left;">
MGI_C57BL6J_5791427
</td>
<td style="text-align:left;">
Gm45591
</td>
<td style="text-align:left;">
ENSEMBL:ENSMUSG00000109460
</td>
<td style="text-align:left;">
MGI:5791427
</td>
<td style="text-align:left;">
lncRNA gene
</td>
<td style="text-align:left;">
predicted gene 45591
</td>
</tr>
<tr>
<td style="text-align:left;">
7
</td>
<td style="text-align:left;">
gene
</td>
<td style="text-align:right;">
99.34538
</td>
<td style="text-align:right;">
99.35324
</td>
<td style="text-align:left;">
negative
</td>
<td style="text-align:left;">
MGI_C57BL6J_88283
</td>
<td style="text-align:left;">
Serpinh1
</td>
<td style="text-align:left;">
NCBI_Gene:12406,ENSEMBL:ENSMUSG00000070436
</td>
<td style="text-align:left;">
MGI:88283
</td>
<td style="text-align:left;">
protein coding gene
</td>
<td style="text-align:left;">
serine (or cysteine) peptidase inhibitor, clade H, member 1
</td>
</tr>
<tr>
<td style="text-align:left;">
7
</td>
<td style="text-align:left;">
gene
</td>
<td style="text-align:right;">
99.38141
</td>
<td style="text-align:right;">
99.46188
</td>
<td style="text-align:left;">
positive
</td>
<td style="text-align:left;">
MGI_C57BL6J_2686926
</td>
<td style="text-align:left;">
Gdpd5
</td>
<td style="text-align:left;">
NCBI_Gene:233552,ENSEMBL:ENSMUSG00000035314
</td>
<td style="text-align:left;">
MGI:2686926
</td>
<td style="text-align:left;">
protein coding gene
</td>
<td style="text-align:left;">
glycerophosphodiester phosphodiesterase domain containing 5
</td>
</tr>
<tr>
<td style="text-align:left;">
7
</td>
<td style="text-align:left;">
gene
</td>
<td style="text-align:right;">
99.46600
</td>
<td style="text-align:right;">
99.47402
</td>
<td style="text-align:left;">
positive
</td>
<td style="text-align:left;">
MGI_C57BL6J_1919434
</td>
<td style="text-align:left;">
Klhl35
</td>
<td style="text-align:left;">
NCBI_Gene:72184,ENSEMBL:ENSMUSG00000035298
</td>
<td style="text-align:left;">
MGI:1919434
</td>
<td style="text-align:left;">
protein coding gene
</td>
<td style="text-align:left;">
kelch-like 35
</td>
</tr>
<tr>
<td style="text-align:left;">
7
</td>
<td style="text-align:left;">
gene
</td>
<td style="text-align:right;">
99.47790
</td>
<td style="text-align:right;">
99.48374
</td>
<td style="text-align:left;">
negative
</td>
<td style="text-align:left;">
MGI_C57BL6J_1350917
</td>
<td style="text-align:left;">
Rps3
</td>
<td style="text-align:left;">
NCBI_Gene:27050,ENSEMBL:ENSMUSG00000030744
</td>
<td style="text-align:left;">
MGI:1350917
</td>
<td style="text-align:left;">
protein coding gene
</td>
<td style="text-align:left;">
ribosomal protein S3
</td>
</tr>
<tr>
<td style="text-align:left;">
7
</td>
<td style="text-align:left;">
gene
</td>
<td style="text-align:right;">
99.47956
</td>
<td style="text-align:right;">
99.47971
</td>
<td style="text-align:left;">
negative
</td>
<td style="text-align:left;">
MGI_C57BL6J_3779515
</td>
<td style="text-align:left;">
Snord15b
</td>
<td style="text-align:left;">
NCBI_Gene:449631,ENSEMBL:ENSMUSG00000064966
</td>
<td style="text-align:left;">
MGI:3779515
</td>
<td style="text-align:left;">
snoRNA gene
</td>
<td style="text-align:left;">
small nucleolar RNA, C/D box 14B
</td>
</tr>
<tr>
<td style="text-align:left;">
7
</td>
<td style="text-align:left;">
gene
</td>
<td style="text-align:right;">
99.48279
</td>
<td style="text-align:right;">
99.48293
</td>
<td style="text-align:left;">
negative
</td>
<td style="text-align:left;">
MGI_C57BL6J_3645887
</td>
<td style="text-align:left;">
Snord15a
</td>
<td style="text-align:left;">
NCBI_Gene:449630,ENSEMBL:ENSMUSG00000065822
</td>
<td style="text-align:left;">
MGI:3645887
</td>
<td style="text-align:left;">
snoRNA gene
</td>
<td style="text-align:left;">
small nucleolar RNA, C/D box 15A
</td>
</tr>
<tr>
<td style="text-align:left;">
7
</td>
<td style="text-align:left;">
gene
</td>
<td style="text-align:right;">
99.49772
</td>
<td style="text-align:right;">
99.49782
</td>
<td style="text-align:left;">
positive
</td>
<td style="text-align:left;">
MGI_C57BL6J_4422019
</td>
<td style="text-align:left;">
n-R5s156
</td>
<td style="text-align:left;">
ENSEMBL:ENSMUSG00000065311
</td>
<td style="text-align:left;">
MGI:4422019
</td>
<td style="text-align:left;">
rRNA gene
</td>
<td style="text-align:left;">
nuclear encoded rRNA 5S 156
</td>
</tr>
<tr>
<td style="text-align:left;">
7
</td>
<td style="text-align:left;">
gene
</td>
<td style="text-align:right;">
99.50046
</td>
<td style="text-align:right;">
99.50594
</td>
<td style="text-align:left;">
positive
</td>
<td style="text-align:left;">
MGI_C57BL6J_5621941
</td>
<td style="text-align:left;">
Gm39056
</td>
<td style="text-align:left;">
NCBI_Gene:105243007
</td>
<td style="text-align:left;">
MGI:5621941
</td>
<td style="text-align:left;">
lncRNA gene
</td>
<td style="text-align:left;">
predicted gene, 39056
</td>
</tr>
<tr>
<td style="text-align:left;">
7
</td>
<td style="text-align:left;">
gene
</td>
<td style="text-align:right;">
99.51165
</td>
<td style="text-align:right;">
99.51715
</td>
<td style="text-align:left;">
positive
</td>
<td style="text-align:left;">
MGI_C57BL6J_5623335
</td>
<td style="text-align:left;">
Gm40450
</td>
<td style="text-align:left;">
NCBI_Gene:105244928
</td>
<td style="text-align:left;">
MGI:5623335
</td>
<td style="text-align:left;">
lncRNA gene
</td>
<td style="text-align:left;">
predicted gene, 40450
</td>
</tr>
<tr>
<td style="text-align:left;">
7
</td>
<td style="text-align:left;">
gene
</td>
<td style="text-align:right;">
99.51664
</td>
<td style="text-align:right;">
99.52964
</td>
<td style="text-align:left;">
negative
</td>
<td style="text-align:left;">
MGI_C57BL6J_5593331
</td>
<td style="text-align:left;">
Gm34172
</td>
<td style="text-align:left;">
NCBI_Gene:102637333
</td>
<td style="text-align:left;">
MGI:5593331
</td>
<td style="text-align:left;">
lncRNA gene
</td>
<td style="text-align:left;">
predicted gene, 34172
</td>
</tr>
<tr>
<td style="text-align:left;">
7
</td>
<td style="text-align:left;">
gene
</td>
<td style="text-align:right;">
99.53547
</td>
<td style="text-align:right;">
99.60677
</td>
<td style="text-align:left;">
positive
</td>
<td style="text-align:left;">
MGI_C57BL6J_99473
</td>
<td style="text-align:left;">
Arrb1
</td>
<td style="text-align:left;">
NCBI_Gene:109689,ENSEMBL:ENSMUSG00000018909
</td>
<td style="text-align:left;">
MGI:99473
</td>
<td style="text-align:left;">
protein coding gene
</td>
<td style="text-align:left;">
arrestin, beta 1
</td>
</tr>
<tr>
<td style="text-align:left;">
7
</td>
<td style="text-align:left;">
gene
</td>
<td style="text-align:right;">
99.54390
</td>
<td style="text-align:right;">
99.54397
</td>
<td style="text-align:left;">
negative
</td>
<td style="text-align:left;">
MGI_C57BL6J_4413843
</td>
<td style="text-align:left;">
n-TEctc3
</td>
<td style="text-align:left;">
NCBI_Gene:102467644
</td>
<td style="text-align:left;">
MGI:4413843
</td>
<td style="text-align:left;">
tRNA gene
</td>
<td style="text-align:left;">
nuclear encoded tRNA glutamic acid 3 (anticodon CTC)
</td>
</tr>
<tr>
<td style="text-align:left;">
7
</td>
<td style="text-align:left;">
gene
</td>
<td style="text-align:right;">
99.55227
</td>
<td style="text-align:right;">
99.55236
</td>
<td style="text-align:left;">
positive
</td>
<td style="text-align:left;">
MGI_C57BL6J_3619338
</td>
<td style="text-align:left;">
Mir326
</td>
<td style="text-align:left;">
miRBase:MI0000598,NCBI_Gene:723840,ENSEMBL:ENSMUSG00000065571
</td>
<td style="text-align:left;">
MGI:3619338
</td>
<td style="text-align:left;">
miRNA gene
</td>
<td style="text-align:left;">
microRNA 326
</td>
</tr>
<tr>
<td style="text-align:left;">
7
</td>
<td style="text-align:left;">
gene
</td>
<td style="text-align:right;">
99.56812
</td>
<td style="text-align:right;">
99.57084
</td>
<td style="text-align:left;">
negative
</td>
<td style="text-align:left;">
MGI_C57BL6J_5753490
</td>
<td style="text-align:left;">
Gm44914
</td>
<td style="text-align:left;">
ENSEMBL:ENSMUSG00000108850
</td>
<td style="text-align:left;">
MGI:5753490
</td>
<td style="text-align:left;">
unclassified gene
</td>
<td style="text-align:left;">
predicted gene 44914
</td>
</tr>
<tr>
<td style="text-align:left;">
7
</td>
<td style="text-align:left;">
gene
</td>
<td style="text-align:right;">
99.62407
</td>
<td style="text-align:right;">
99.62710
</td>
<td style="text-align:left;">
negative
</td>
<td style="text-align:left;">
MGI_C57BL6J_3646425
</td>
<td style="text-align:left;">
Tpbgl
</td>
<td style="text-align:left;">
NCBI_Gene:100503386,ENSEMBL:ENSMUSG00000096606
</td>
<td style="text-align:left;">
MGI:3646425
</td>
<td style="text-align:left;">
protein coding gene
</td>
<td style="text-align:left;">
trophoblast glycoprotein-like
</td>
</tr>
<tr>
<td style="text-align:left;">
7
</td>
<td style="text-align:left;">
gene
</td>
<td style="text-align:right;">
99.62594
</td>
<td style="text-align:right;">
99.63171
</td>
<td style="text-align:left;">
positive
</td>
<td style="text-align:left;">
MGI_C57BL6J_3642596
</td>
<td style="text-align:left;">
Gm10605
</td>
<td style="text-align:left;">
ENSEMBL:ENSMUSG00000097974
</td>
<td style="text-align:left;">
MGI:3642596
</td>
<td style="text-align:left;">
lncRNA gene
</td>
<td style="text-align:left;">
predicted gene 10605
</td>
</tr>
<tr>
<td style="text-align:left;">
7
</td>
<td style="text-align:left;">
gene
</td>
<td style="text-align:right;">
99.65369
</td>
<td style="text-align:right;">
99.65382
</td>
<td style="text-align:left;">
negative
</td>
<td style="text-align:left;">
MGI_C57BL6J_5455186
</td>
<td style="text-align:left;">
Gm25409
</td>
<td style="text-align:left;">
ENSEMBL:ENSMUSG00000065698
</td>
<td style="text-align:left;">
MGI:5455186
</td>
<td style="text-align:left;">
snoRNA gene
</td>
<td style="text-align:left;">
predicted gene, 25409
</td>
</tr>
<tr>
<td style="text-align:left;">
7
</td>
<td style="text-align:left;">
gene
</td>
<td style="text-align:right;">
99.65780
</td>
<td style="text-align:right;">
99.71134
</td>
<td style="text-align:left;">
negative
</td>
<td style="text-align:left;">
MGI_C57BL6J_1351872
</td>
<td style="text-align:left;">
Slco2b1
</td>
<td style="text-align:left;">
NCBI_Gene:101488,ENSEMBL:ENSMUSG00000030737
</td>
<td style="text-align:left;">
MGI:1351872
</td>
<td style="text-align:left;">
protein coding gene
</td>
<td style="text-align:left;">
solute carrier organic anion transporter family, member 2b1
</td>
</tr>
<tr>
<td style="text-align:left;">
7
</td>
<td style="text-align:left;">
gene
</td>
<td style="text-align:right;">
99.66013
</td>
<td style="text-align:right;">
99.66750
</td>
<td style="text-align:left;">
positive
</td>
<td style="text-align:left;">
MGI_C57BL6J_3783079
</td>
<td style="text-align:left;">
Gm15635
</td>
<td style="text-align:left;">
ENSEMBL:ENSMUSG00000085095
</td>
<td style="text-align:left;">
MGI:3783079
</td>
<td style="text-align:left;">
lncRNA gene
</td>
<td style="text-align:left;">
predicted gene 15635
</td>
</tr>
<tr>
<td style="text-align:left;">
7
</td>
<td style="text-align:left;">
gene
</td>
<td style="text-align:right;">
99.69303
</td>
<td style="text-align:right;">
99.71423
</td>
<td style="text-align:left;">
positive
</td>
<td style="text-align:left;">
MGI_C57BL6J_5593439
</td>
<td style="text-align:left;">
Gm34280
</td>
<td style="text-align:left;">
NCBI_Gene:102637485,ENSEMBL:ENSMUSG00000109559
</td>
<td style="text-align:left;">
MGI:5593439
</td>
<td style="text-align:left;">
lncRNA gene
</td>
<td style="text-align:left;">
predicted gene, 34280
</td>
</tr>
<tr>
<td style="text-align:left;">
7
</td>
<td style="text-align:left;">
pseudogene
</td>
<td style="text-align:right;">
99.72521
</td>
<td style="text-align:right;">
99.72538
</td>
<td style="text-align:left;">
negative
</td>
<td style="text-align:left;">
MGI_C57BL6J_5753127
</td>
<td style="text-align:left;">
Gm44551
</td>
<td style="text-align:left;">
ENSEMBL:ENSMUSG00000108965
</td>
<td style="text-align:left;">
MGI:5753127
</td>
<td style="text-align:left;">
pseudogene
</td>
<td style="text-align:left;">
predicted gene 44551
</td>
</tr>
<tr>
<td style="text-align:left;">
7
</td>
<td style="text-align:left;">
gene
</td>
<td style="text-align:right;">
99.73027
</td>
<td style="text-align:right;">
99.73640
</td>
<td style="text-align:left;">
positive
</td>
<td style="text-align:left;">
MGI_C57BL6J_3030354
</td>
<td style="text-align:left;">
Olfr520
</td>
<td style="text-align:left;">
NCBI_Gene:259066,ENSEMBL:ENSMUSG00000073998
</td>
<td style="text-align:left;">
MGI:3030354
</td>
<td style="text-align:left;">
protein coding gene
</td>
<td style="text-align:left;">
olfactory receptor 520
</td>
</tr>
<tr>
<td style="text-align:left;">
7
</td>
<td style="text-align:left;">
pseudogene
</td>
<td style="text-align:right;">
99.74021
</td>
<td style="text-align:right;">
99.74079
</td>
<td style="text-align:left;">
negative
</td>
<td style="text-align:left;">
MGI_C57BL6J_5011367
</td>
<td style="text-align:left;">
Gm19182
</td>
<td style="text-align:left;">
NCBI_Gene:100418394,ENSEMBL:ENSMUSG00000109275
</td>
<td style="text-align:left;">
MGI:5011367
</td>
<td style="text-align:left;">
pseudogene
</td>
<td style="text-align:left;">
predicted gene, 19182
</td>
</tr>
<tr>
<td style="text-align:left;">
7
</td>
<td style="text-align:left;">
gene
</td>
<td style="text-align:right;">
99.76351
</td>
<td style="text-align:right;">
99.76932
</td>
<td style="text-align:left;">
positive
</td>
<td style="text-align:left;">
MGI_C57BL6J_3030355
</td>
<td style="text-align:left;">
Olfr521
</td>
<td style="text-align:left;">
NCBI_Gene:258353,ENSEMBL:ENSMUSG00000073997
</td>
<td style="text-align:left;">
MGI:3030355
</td>
<td style="text-align:left;">
protein coding gene
</td>
<td style="text-align:left;">
olfactory receptor 521
</td>
</tr>
<tr>
<td style="text-align:left;">
7
</td>
<td style="text-align:left;">
gene
</td>
<td style="text-align:right;">
99.78000
</td>
<td style="text-align:right;">
99.78154
</td>
<td style="text-align:left;">
negative
</td>
<td style="text-align:left;">
MGI_C57BL6J_3045385
</td>
<td style="text-align:left;">
F730035P03Rik
</td>
<td style="text-align:left;">
ENSEMBL:ENSMUSG00000053360
</td>
<td style="text-align:left;">
MGI:3045385
</td>
<td style="text-align:left;">
lncRNA gene
</td>
<td style="text-align:left;">
RIKEN cDNA F730035P03 gene
</td>
</tr>
<tr>
<td style="text-align:left;">
7
</td>
<td style="text-align:left;">
gene
</td>
<td style="text-align:right;">
99.81144
</td>
<td style="text-align:right;">
99.82842
</td>
<td style="text-align:left;">
negative
</td>
<td style="text-align:left;">
MGI_C57BL6J_1355305
</td>
<td style="text-align:left;">
Neu3
</td>
<td style="text-align:left;">
NCBI_Gene:50877,ENSEMBL:ENSMUSG00000035239
</td>
<td style="text-align:left;">
MGI:1355305
</td>
<td style="text-align:left;">
protein coding gene
</td>
<td style="text-align:left;">
neuraminidase 3
</td>
</tr>
<tr>
<td style="text-align:left;">
7
</td>
<td style="text-align:left;">
gene
</td>
<td style="text-align:right;">
99.83757
</td>
<td style="text-align:right;">
99.87034
</td>
<td style="text-align:left;">
negative
</td>
<td style="text-align:left;">
MGI_C57BL6J_1913874
</td>
<td style="text-align:left;">
Spcs2
</td>
<td style="text-align:left;">
NCBI_Gene:66624,ENSEMBL:ENSMUSG00000035227
</td>
<td style="text-align:left;">
MGI:1913874
</td>
<td style="text-align:left;">
protein coding gene
</td>
<td style="text-align:left;">
signal peptidase complex subunit 2 homolog (S. cerevisiae)
</td>
</tr>
<tr>
<td style="text-align:left;">
7
</td>
<td style="text-align:left;">
gene
</td>
<td style="text-align:right;">
99.85912
</td>
<td style="text-align:right;">
99.91782
</td>
<td style="text-align:left;">
positive
</td>
<td style="text-align:left;">
MGI_C57BL6J_2181647
</td>
<td style="text-align:left;">
Xrra1
</td>
<td style="text-align:left;">
NCBI_Gene:446101,ENSEMBL:ENSMUSG00000035211
</td>
<td style="text-align:left;">
MGI:2181647
</td>
<td style="text-align:left;">
protein coding gene
</td>
<td style="text-align:left;">
X-ray radiation resistance associated 1
</td>
</tr>
<tr>
<td style="text-align:left;">
7
</td>
<td style="text-align:left;">
gene
</td>
<td style="text-align:right;">
99.87067
</td>
<td style="text-align:right;">
99.91738
</td>
<td style="text-align:left;">
negative
</td>
<td style="text-align:left;">
MGI_C57BL6J_5621290
</td>
<td style="text-align:left;">
Gm38405
</td>
<td style="text-align:left;">
NCBI_Gene:233561,ENSEMBL:ENSMUSG00000109002
</td>
<td style="text-align:left;">
MGI:5621290
</td>
<td style="text-align:left;">
lncRNA gene
</td>
<td style="text-align:left;">
predicted gene, 38405
</td>
</tr>
<tr>
<td style="text-align:left;">
7
</td>
<td style="text-align:left;">
gene
</td>
<td style="text-align:right;">
99.90019
</td>
<td style="text-align:right;">
99.90416
</td>
<td style="text-align:left;">
negative
</td>
<td style="text-align:left;">
MGI_C57BL6J_1924443
</td>
<td style="text-align:left;">
A930030B08Rik
</td>
<td style="text-align:left;">
ENSEMBL:ENSMUSG00000108985
</td>
<td style="text-align:left;">
MGI:1924443
</td>
<td style="text-align:left;">
unclassified gene
</td>
<td style="text-align:left;">
RIKEN cDNA A930030B08 gene
</td>
</tr>
<tr>
<td style="text-align:left;">
7
</td>
<td style="text-align:left;">
gene
</td>
<td style="text-align:right;">
99.92025
</td>
<td style="text-align:right;">
99.98069
</td>
<td style="text-align:left;">
negative
</td>
<td style="text-align:left;">
MGI_C57BL6J_1920257
</td>
<td style="text-align:left;">
Rnf169
</td>
<td style="text-align:left;">
NCBI_Gene:108937,ENSEMBL:ENSMUSG00000058761
</td>
<td style="text-align:left;">
MGI:1920257
</td>
<td style="text-align:left;">
protein coding gene
</td>
<td style="text-align:left;">
ring finger protein 169
</td>
</tr>
<tr>
<td style="text-align:left;">
7
</td>
<td style="text-align:left;">
gene
</td>
<td style="text-align:right;">
99.92965
</td>
<td style="text-align:right;">
99.93229
</td>
<td style="text-align:left;">
negative
</td>
<td style="text-align:left;">
MGI_C57BL6J_3643844
</td>
<td style="text-align:left;">
Gm5115
</td>
<td style="text-align:left;">
NCBI_Gene:330602
</td>
<td style="text-align:left;">
MGI:3643844
</td>
<td style="text-align:left;">
unclassified gene
</td>
<td style="text-align:left;">
predicted gene 5115
</td>
</tr>
<tr>
<td style="text-align:left;">
7
</td>
<td style="text-align:left;">
pseudogene
</td>
<td style="text-align:right;">
99.94506
</td>
<td style="text-align:right;">
99.94572
</td>
<td style="text-align:left;">
positive
</td>
<td style="text-align:left;">
MGI_C57BL6J_5011144
</td>
<td style="text-align:left;">
Gm18959
</td>
<td style="text-align:left;">
NCBI_Gene:100418037,ENSEMBL:ENSMUSG00000108823
</td>
<td style="text-align:left;">
MGI:5011144
</td>
<td style="text-align:left;">
pseudogene
</td>
<td style="text-align:left;">
predicted gene, 18959
</td>
</tr>
<tr>
<td style="text-align:left;">
7
</td>
<td style="text-align:left;">
pseudogene
</td>
<td style="text-align:right;">
99.99677
</td>
<td style="text-align:right;">
99.99741
</td>
<td style="text-align:left;">
positive
</td>
<td style="text-align:left;">
MGI_C57BL6J_5010116
</td>
<td style="text-align:left;">
Gm17931
</td>
<td style="text-align:left;">
NCBI_Gene:100416129,ENSEMBL:ENSMUSG00000109384
</td>
<td style="text-align:left;">
MGI:5010116
</td>
<td style="text-align:left;">
pseudogene
</td>
<td style="text-align:left;">
predicted gene, 17931
</td>
</tr>
<tr>
<td style="text-align:left;">
7
</td>
<td style="text-align:left;">
gene
</td>
<td style="text-align:right;">
100.00617
</td>
<td style="text-align:right;">
100.03473
</td>
<td style="text-align:left;">
positive
</td>
<td style="text-align:left;">
MGI_C57BL6J_1916371
</td>
<td style="text-align:left;">
Chrdl2
</td>
<td style="text-align:left;">
NCBI_Gene:69121,ENSEMBL:ENSMUSG00000030732
</td>
<td style="text-align:left;">
MGI:1916371
</td>
<td style="text-align:left;">
protein coding gene
</td>
<td style="text-align:left;">
chordin-like 2
</td>
</tr>
<tr>
<td style="text-align:left;">
7
</td>
<td style="text-align:left;">
gene
</td>
<td style="text-align:right;">
100.08211
</td>
<td style="text-align:right;">
100.12157
</td>
<td style="text-align:left;">
negative
</td>
<td style="text-align:left;">
MGI_C57BL6J_1915217
</td>
<td style="text-align:left;">
Pold3
</td>
<td style="text-align:left;">
NCBI_Gene:67967,ENSEMBL:ENSMUSG00000030726
</td>
<td style="text-align:left;">
MGI:1915217
</td>
<td style="text-align:left;">
protein coding gene
</td>
<td style="text-align:left;">
polymerase (DNA-directed), delta 3, accessory subunit
</td>
</tr>
<tr>
<td style="text-align:left;">
7
</td>
<td style="text-align:left;">
gene
</td>
<td style="text-align:right;">
100.13188
</td>
<td style="text-align:right;">
100.13749
</td>
<td style="text-align:left;">
positive
</td>
<td style="text-align:left;">
MGI_C57BL6J_5593919
</td>
<td style="text-align:left;">
Gm34760
</td>
<td style="text-align:left;">
NCBI_Gene:102638122
</td>
<td style="text-align:left;">
MGI:5593919
</td>
<td style="text-align:left;">
lncRNA gene
</td>
<td style="text-align:left;">
predicted gene, 34760
</td>
</tr>
<tr>
<td style="text-align:left;">
7
</td>
<td style="text-align:left;">
gene
</td>
<td style="text-align:right;">
100.15928
</td>
<td style="text-align:right;">
100.16137
</td>
<td style="text-align:left;">
positive
</td>
<td style="text-align:left;">
MGI_C57BL6J_1914414
</td>
<td style="text-align:left;">
Lipt2
</td>
<td style="text-align:left;">
NCBI_Gene:67164,ENSEMBL:ENSMUSG00000030725
</td>
<td style="text-align:left;">
MGI:1914414
</td>
<td style="text-align:left;">
protein coding gene
</td>
<td style="text-align:left;">
lipoyl(octanoyl) transferase 2 (putative)
</td>
</tr>
<tr>
<td style="text-align:left;">
7
</td>
<td style="text-align:left;">
gene
</td>
<td style="text-align:right;">
100.16332
</td>
<td style="text-align:right;">
100.17116
</td>
<td style="text-align:left;">
negative
</td>
<td style="text-align:left;">
MGI_C57BL6J_5621942
</td>
<td style="text-align:left;">
Gm39057
</td>
<td style="text-align:left;">
NCBI_Gene:105243008
</td>
<td style="text-align:left;">
MGI:5621942
</td>
<td style="text-align:left;">
lncRNA gene
</td>
<td style="text-align:left;">
predicted gene, 39057
</td>
</tr>
<tr>
<td style="text-align:left;">
7
</td>
<td style="text-align:left;">
gene
</td>
<td style="text-align:right;">
100.17142
</td>
<td style="text-align:right;">
100.22721
</td>
<td style="text-align:left;">
negative
</td>
<td style="text-align:left;">
MGI_C57BL6J_5593980
</td>
<td style="text-align:left;">
Gm34821
</td>
<td style="text-align:left;">
NCBI_Gene:102638204,ENSEMBL:ENSMUSG00000109186
</td>
<td style="text-align:left;">
MGI:5593980
</td>
<td style="text-align:left;">
lncRNA gene
</td>
<td style="text-align:left;">
predicted gene, 34821
</td>
</tr>
<tr>
<td style="text-align:left;">
7
</td>
<td style="text-align:left;">
gene
</td>
<td style="text-align:right;">
100.17650
</td>
<td style="text-align:right;">
100.18487
</td>
<td style="text-align:left;">
positive
</td>
<td style="text-align:left;">
MGI_C57BL6J_1891124
</td>
<td style="text-align:left;">
Kcne3
</td>
<td style="text-align:left;">
NCBI_Gene:57442,ENSEMBL:ENSMUSG00000035165
</td>
<td style="text-align:left;">
MGI:1891124
</td>
<td style="text-align:left;">
protein coding gene
</td>
<td style="text-align:left;">
potassium voltage-gated channel, Isk-related subfamily, gene 3
</td>
</tr>
<tr>
<td style="text-align:left;">
7
</td>
<td style="text-align:left;">
gene
</td>
<td style="text-align:right;">
100.18926
</td>
<td style="text-align:right;">
100.19267
</td>
<td style="text-align:left;">
negative
</td>
<td style="text-align:left;">
MGI_C57BL6J_5825631
</td>
<td style="text-align:left;">
Gm45994
</td>
<td style="text-align:left;">
NCBI_Gene:108167448
</td>
<td style="text-align:left;">
MGI:5825631
</td>
<td style="text-align:left;">
lncRNA gene
</td>
<td style="text-align:left;">
predicted gene, 45994
</td>
</tr>
<tr>
<td style="text-align:left;">
7
</td>
<td style="text-align:left;">
pseudogene
</td>
<td style="text-align:right;">
100.19710
</td>
<td style="text-align:right;">
100.19835
</td>
<td style="text-align:left;">
negative
</td>
<td style="text-align:left;">
MGI_C57BL6J_3779664
</td>
<td style="text-align:left;">
Gm7067
</td>
<td style="text-align:left;">
NCBI_Gene:631577,ENSEMBL:ENSMUSG00000109435
</td>
<td style="text-align:left;">
MGI:3779664
</td>
<td style="text-align:left;">
pseudogene
</td>
<td style="text-align:left;">
predicted gene 7067
</td>
</tr>
<tr>
<td style="text-align:left;">
7
</td>
<td style="text-align:left;">
gene
</td>
<td style="text-align:right;">
100.20078
</td>
<td style="text-align:right;">
100.22721
</td>
<td style="text-align:left;">
negative
</td>
<td style="text-align:left;">
MGI_C57BL6J_5594036
</td>
<td style="text-align:left;">
Gm34877
</td>
<td style="text-align:left;">
NCBI_Gene:102638275
</td>
<td style="text-align:left;">
MGI:5594036
</td>
<td style="text-align:left;">
lncRNA gene
</td>
<td style="text-align:left;">
predicted gene, 34877
</td>
</tr>
<tr>
<td style="text-align:left;">
7
</td>
<td style="text-align:left;">
gene
</td>
<td style="text-align:right;">
100.22728
</td>
<td style="text-align:right;">
100.27887
</td>
<td style="text-align:left;">
positive
</td>
<td style="text-align:left;">
MGI_C57BL6J_1918224
</td>
<td style="text-align:left;">
Pgm2l1
</td>
<td style="text-align:left;">
NCBI_Gene:70974,ENSEMBL:ENSMUSG00000030729
</td>
<td style="text-align:left;">
MGI:1918224
</td>
<td style="text-align:left;">
protein coding gene
</td>
<td style="text-align:left;">
phosphoglucomutase 2-like 1
</td>
</tr>
<tr>
<td style="text-align:left;">
7
</td>
<td style="text-align:left;">
pseudogene
</td>
<td style="text-align:right;">
100.26454
</td>
<td style="text-align:right;">
100.26556
</td>
<td style="text-align:left;">
positive
</td>
<td style="text-align:left;">
MGI_C57BL6J_106627
</td>
<td style="text-align:left;">
Gpx2-ps1
</td>
<td style="text-align:left;">
NCBI_Gene:14777,ENSEMBL:ENSMUSG00000089987
</td>
<td style="text-align:left;">
MGI:106627
</td>
<td style="text-align:left;">
pseudogene
</td>
<td style="text-align:left;">
glutathione peroxidase 2, pseudogene 1
</td>
</tr>
<tr>
<td style="text-align:left;">
7
</td>
<td style="text-align:left;">
gene
</td>
<td style="text-align:right;">
100.28549
</td>
<td style="text-align:right;">
100.31970
</td>
<td style="text-align:left;">
positive
</td>
<td style="text-align:left;">
MGI_C57BL6J_2444049
</td>
<td style="text-align:left;">
P4ha3
</td>
<td style="text-align:left;">
NCBI_Gene:320452,ENSEMBL:ENSMUSG00000051048
</td>
<td style="text-align:left;">
MGI:2444049
</td>
<td style="text-align:left;">
protein coding gene
</td>
<td style="text-align:left;">
procollagen-proline, 2-oxoglutarate 4-dioxygenase (proline 4-hydroxylase), alpha polypeptide III
</td>
</tr>
<tr>
<td style="text-align:left;">
7
</td>
<td style="text-align:left;">
gene
</td>
<td style="text-align:right;">
100.32674
</td>
<td style="text-align:right;">
100.37231
</td>
<td style="text-align:left;">
negative
</td>
<td style="text-align:left;">
MGI_C57BL6J_1919840
</td>
<td style="text-align:left;">
Ppme1
</td>
<td style="text-align:left;">
NCBI_Gene:72590,ENSEMBL:ENSMUSG00000030718
</td>
<td style="text-align:left;">
MGI:1919840
</td>
<td style="text-align:left;">
protein coding gene
</td>
<td style="text-align:left;">
protein phosphatase methylesterase 1
</td>
</tr>
<tr>
<td style="text-align:left;">
7
</td>
<td style="text-align:left;">
gene
</td>
<td style="text-align:right;">
100.37223
</td>
<td style="text-align:right;">
100.47016
</td>
<td style="text-align:left;">
positive
</td>
<td style="text-align:left;">
MGI_C57BL6J_2142166
</td>
<td style="text-align:left;">
C2cd3
</td>
<td style="text-align:left;">
NCBI_Gene:277939,ENSEMBL:ENSMUSG00000047248
</td>
<td style="text-align:left;">
MGI:2142166
</td>
<td style="text-align:left;">
protein coding gene
</td>
<td style="text-align:left;">
C2 calcium-dependent domain containing 3
</td>
</tr>
<tr>
<td style="text-align:left;">
7
</td>
<td style="text-align:left;">
pseudogene
</td>
<td style="text-align:right;">
100.38159
</td>
<td style="text-align:right;">
100.38231
</td>
<td style="text-align:left;">
positive
</td>
<td style="text-align:left;">
MGI_C57BL6J_5753147
</td>
<td style="text-align:left;">
Gm44571
</td>
<td style="text-align:left;">
ENSEMBL:ENSMUSG00000109046
</td>
<td style="text-align:left;">
MGI:5753147
</td>
<td style="text-align:left;">
pseudogene
</td>
<td style="text-align:left;">
predicted gene 44571
</td>
</tr>
<tr>
<td style="text-align:left;">
7
</td>
<td style="text-align:left;">
gene
</td>
<td style="text-align:right;">
100.44674
</td>
<td style="text-align:right;">
100.47485
</td>
<td style="text-align:left;">
negative
</td>
<td style="text-align:left;">
MGI_C57BL6J_3642592
</td>
<td style="text-align:left;">
Gm10603
</td>
<td style="text-align:left;">
NCBI_Gene:100038497,ENSEMBL:ENSMUSG00000099137
</td>
<td style="text-align:left;">
MGI:3642592
</td>
<td style="text-align:left;">
lncRNA gene
</td>
<td style="text-align:left;">
predicted gene 10603
</td>
</tr>
<tr>
<td style="text-align:left;">
7
</td>
<td style="text-align:left;">
gene
</td>
<td style="text-align:right;">
100.45846
</td>
<td style="text-align:right;">
100.45941
</td>
<td style="text-align:left;">
positive
</td>
<td style="text-align:left;">
MGI_C57BL6J_5610944
</td>
<td style="text-align:left;">
Gm37716
</td>
<td style="text-align:left;">
ENSEMBL:ENSMUSG00000102241
</td>
<td style="text-align:left;">
MGI:5610944
</td>
<td style="text-align:left;">
unclassified gene
</td>
<td style="text-align:left;">
predicted gene, 37716
</td>
</tr>
<tr>
<td style="text-align:left;">
7
</td>
<td style="text-align:left;">
gene
</td>
<td style="text-align:right;">
100.47299
</td>
<td style="text-align:right;">
100.48643
</td>
<td style="text-align:left;">
positive
</td>
<td style="text-align:left;">
MGI_C57BL6J_1099787
</td>
<td style="text-align:left;">
Ucp3
</td>
<td style="text-align:left;">
NCBI_Gene:22229,ENSEMBL:ENSMUSG00000032942
</td>
<td style="text-align:left;">
MGI:1099787
</td>
<td style="text-align:left;">
protein coding gene
</td>
<td style="text-align:left;">
uncoupling protein 3 (mitochondrial, proton carrier)
</td>
</tr>
<tr>
<td style="text-align:left;">
7
</td>
<td style="text-align:left;">
gene
</td>
<td style="text-align:right;">
100.49334
</td>
<td style="text-align:right;">
100.50202
</td>
<td style="text-align:left;">
positive
</td>
<td style="text-align:left;">
MGI_C57BL6J_109354
</td>
<td style="text-align:left;">
Ucp2
</td>
<td style="text-align:left;">
NCBI_Gene:22228,ENSEMBL:ENSMUSG00000033685
</td>
<td style="text-align:left;">
MGI:109354
</td>
<td style="text-align:left;">
protein coding gene
</td>
<td style="text-align:left;">
uncoupling protein 2 (mitochondrial, proton carrier)
</td>
</tr>
<tr>
<td style="text-align:left;">
7
</td>
<td style="text-align:left;">
gene
</td>
<td style="text-align:right;">
100.50172
</td>
<td style="text-align:right;">
100.51496
</td>
<td style="text-align:left;">
negative
</td>
<td style="text-align:left;">
MGI_C57BL6J_1916637
</td>
<td style="text-align:left;">
Dnajb13
</td>
<td style="text-align:left;">
NCBI_Gene:69387,ENSEMBL:ENSMUSG00000030708
</td>
<td style="text-align:left;">
MGI:1916637
</td>
<td style="text-align:left;">
protein coding gene
</td>
<td style="text-align:left;">
DnaJ heat shock protein family (Hsp40) member B13
</td>
</tr>
<tr>
<td style="text-align:left;">
7
</td>
<td style="text-align:left;">
pseudogene
</td>
<td style="text-align:right;">
100.52398
</td>
<td style="text-align:right;">
100.52467
</td>
<td style="text-align:left;">
positive
</td>
<td style="text-align:left;">
MGI_C57BL6J_3644490
</td>
<td style="text-align:left;">
Gm8463
</td>
<td style="text-align:left;">
NCBI_Gene:667112,ENSEMBL:ENSMUSG00000109560
</td>
<td style="text-align:left;">
MGI:3644490
</td>
<td style="text-align:left;">
pseudogene
</td>
<td style="text-align:left;">
predicted gene 8463
</td>
</tr>
<tr>
<td style="text-align:left;">
7
</td>
<td style="text-align:left;">
gene
</td>
<td style="text-align:right;">
100.53559
</td>
<td style="text-align:right;">
100.53698
</td>
<td style="text-align:left;">
negative
</td>
<td style="text-align:left;">
MGI_C57BL6J_5621943
</td>
<td style="text-align:left;">
Gm39058
</td>
<td style="text-align:left;">
NCBI_Gene:105243010
</td>
<td style="text-align:left;">
MGI:5621943
</td>
<td style="text-align:left;">
lncRNA gene
</td>
<td style="text-align:left;">
predicted gene, 39058
</td>
</tr>
<tr>
<td style="text-align:left;">
7
</td>
<td style="text-align:left;">
gene
</td>
<td style="text-align:right;">
100.53707
</td>
<td style="text-align:right;">
100.54037
</td>
<td style="text-align:left;">
positive
</td>
<td style="text-align:left;">
MGI_C57BL6J_1915435
</td>
<td style="text-align:left;">
Coa4
</td>
<td style="text-align:left;">
NCBI_Gene:68185,ENSEMBL:ENSMUSG00000044881
</td>
<td style="text-align:left;">
MGI:1915435
</td>
<td style="text-align:left;">
protein coding gene
</td>
<td style="text-align:left;">
cytochrome c oxidase assembly factor 4
</td>
</tr>
<tr>
<td style="text-align:left;">
7
</td>
<td style="text-align:left;">
gene
</td>
<td style="text-align:right;">
100.54135
</td>
<td style="text-align:right;">
100.54412
</td>
<td style="text-align:left;">
negative
</td>
<td style="text-align:left;">
MGI_C57BL6J_5621944
</td>
<td style="text-align:left;">
Gm39059
</td>
<td style="text-align:left;">
NCBI_Gene:105243011,ENSEMBL:ENSMUSG00000109123
</td>
<td style="text-align:left;">
MGI:5621944
</td>
<td style="text-align:left;">
lncRNA gene
</td>
<td style="text-align:left;">
predicted gene, 39059
</td>
</tr>
<tr>
<td style="text-align:left;">
7
</td>
<td style="text-align:left;">
gene
</td>
<td style="text-align:right;">
100.54574
</td>
<td style="text-align:right;">
100.54779
</td>
<td style="text-align:left;">
negative
</td>
<td style="text-align:left;">
MGI_C57BL6J_3028054
</td>
<td style="text-align:left;">
D630004N19Rik
</td>
<td style="text-align:left;">
NA
</td>
<td style="text-align:left;">
NA
</td>
<td style="text-align:left;">
unclassified non-coding RNA gene
</td>
<td style="text-align:left;">
RIKEN cDNA D630004N19 gene
</td>
</tr>
<tr>
<td style="text-align:left;">
7
</td>
<td style="text-align:left;">
gene
</td>
<td style="text-align:right;">
100.54575
</td>
<td style="text-align:right;">
100.60830
</td>
<td style="text-align:left;">
negative
</td>
<td style="text-align:left;">
MGI_C57BL6J_1289321
</td>
<td style="text-align:left;">
Mrpl48
</td>
<td style="text-align:left;">
NCBI_Gene:52443,ENSEMBL:ENSMUSG00000030706
</td>
<td style="text-align:left;">
MGI:1289321
</td>
<td style="text-align:left;">
protein coding gene
</td>
<td style="text-align:left;">
mitochondrial ribosomal protein L48
</td>
</tr>
<tr>
<td style="text-align:left;">
7
</td>
<td style="text-align:left;">
pseudogene
</td>
<td style="text-align:right;">
100.57192
</td>
<td style="text-align:right;">
100.57225
</td>
<td style="text-align:left;">
positive
</td>
<td style="text-align:left;">
MGI_C57BL6J_5753224
</td>
<td style="text-align:left;">
Gm44648
</td>
<td style="text-align:left;">
ENSEMBL:ENSMUSG00000108990
</td>
<td style="text-align:left;">
MGI:5753224
</td>
<td style="text-align:left;">
pseudogene
</td>
<td style="text-align:left;">
predicted gene 44648
</td>
</tr>
<tr>
<td style="text-align:left;">
7
</td>
<td style="text-align:left;">
pseudogene
</td>
<td style="text-align:right;">
100.57841
</td>
<td style="text-align:right;">
100.58064
</td>
<td style="text-align:left;">
positive
</td>
<td style="text-align:left;">
MGI_C57BL6J_3783125
</td>
<td style="text-align:left;">
Cox20b
</td>
<td style="text-align:left;">
NCBI_Gene:108167449,ENSEMBL:ENSMUSG00000081359
</td>
<td style="text-align:left;">
MGI:3783125
</td>
<td style="text-align:left;">
pseudogene
</td>
<td style="text-align:left;">
cytochrome c oxidase assembly protein 20B
</td>
</tr>
<tr>
<td style="text-align:left;">
7
</td>
<td style="text-align:left;">
gene
</td>
<td style="text-align:right;">
100.60741
</td>
<td style="text-align:right;">
100.64127
</td>
<td style="text-align:left;">
positive
</td>
<td style="text-align:left;">
MGI_C57BL6J_894313
</td>
<td style="text-align:left;">
Rab6a
</td>
<td style="text-align:left;">
NCBI_Gene:19346,ENSEMBL:ENSMUSG00000030704
</td>
<td style="text-align:left;">
MGI:894313
</td>
<td style="text-align:left;">
protein coding gene
</td>
<td style="text-align:left;">
RAB6A, member RAS oncogene family
</td>
</tr>
<tr>
<td style="text-align:left;">
7
</td>
<td style="text-align:left;">
gene
</td>
<td style="text-align:right;">
100.61877
</td>
<td style="text-align:right;">
100.62126
</td>
<td style="text-align:left;">
positive
</td>
<td style="text-align:left;">
MGI_C57BL6J_2442194
</td>
<td style="text-align:left;">
D930046H04Rik
</td>
<td style="text-align:left;">
NA
</td>
<td style="text-align:left;">
NA
</td>
<td style="text-align:left;">
unclassified non-coding RNA gene
</td>
<td style="text-align:left;">
RIKEN cDNA D930046H04 gene
</td>
</tr>
<tr>
<td style="text-align:left;">
7
</td>
<td style="text-align:left;">
pseudogene
</td>
<td style="text-align:right;">
100.62452
</td>
<td style="text-align:right;">
100.62495
</td>
<td style="text-align:left;">
positive
</td>
<td style="text-align:left;">
MGI_C57BL6J_3650372
</td>
<td style="text-align:left;">
Gm14382
</td>
<td style="text-align:left;">
ENSEMBL:ENSMUSG00000082210
</td>
<td style="text-align:left;">
MGI:3650372
</td>
<td style="text-align:left;">
pseudogene
</td>
<td style="text-align:left;">
predicted gene 14382
</td>
</tr>
<tr>
<td style="text-align:left;">
7
</td>
<td style="text-align:left;">
gene
</td>
<td style="text-align:right;">
100.64289
</td>
<td style="text-align:right;">
100.66241
</td>
<td style="text-align:left;">
negative
</td>
<td style="text-align:left;">
MGI_C57BL6J_1351469
</td>
<td style="text-align:left;">
Plekhb1
</td>
<td style="text-align:left;">
NCBI_Gene:27276,ENSEMBL:ENSMUSG00000030701
</td>
<td style="text-align:left;">
MGI:1351469
</td>
<td style="text-align:left;">
protein coding gene
</td>
<td style="text-align:left;">
pleckstrin homology domain containing, family B (evectins) member 1
</td>
</tr>
<tr>
<td style="text-align:left;">
7
</td>
<td style="text-align:left;">
gene
</td>
<td style="text-align:right;">
100.67499
</td>
<td style="text-align:right;">
100.67814
</td>
<td style="text-align:left;">
positive
</td>
<td style="text-align:left;">
MGI_C57BL6J_5594241
</td>
<td style="text-align:left;">
Gm35082
</td>
<td style="text-align:left;">
NCBI_Gene:102638545,ENSEMBL:ENSMUSG00000108866
</td>
<td style="text-align:left;">
MGI:5594241
</td>
<td style="text-align:left;">
lncRNA gene
</td>
<td style="text-align:left;">
predicted gene, 35082
</td>
</tr>
<tr>
<td style="text-align:left;">
7
</td>
<td style="text-align:left;">
gene
</td>
<td style="text-align:right;">
100.70664
</td>
<td style="text-align:right;">
100.84166
</td>
<td style="text-align:left;">
positive
</td>
<td style="text-align:left;">
MGI_C57BL6J_2442372
</td>
<td style="text-align:left;">
Fam168a
</td>
<td style="text-align:left;">
NCBI_Gene:319604,ENSEMBL:ENSMUSG00000029461
</td>
<td style="text-align:left;">
MGI:2442372
</td>
<td style="text-align:left;">
protein coding gene
</td>
<td style="text-align:left;">
family with sequence similarity 168, member A
</td>
</tr>
<tr>
<td style="text-align:left;">
7
</td>
<td style="text-align:left;">
pseudogene
</td>
<td style="text-align:right;">
100.73805
</td>
<td style="text-align:right;">
100.73863
</td>
<td style="text-align:left;">
negative
</td>
<td style="text-align:left;">
MGI_C57BL6J_5753785
</td>
<td style="text-align:left;">
Gm45209
</td>
<td style="text-align:left;">
ENSEMBL:ENSMUSG00000109035
</td>
<td style="text-align:left;">
MGI:5753785
</td>
<td style="text-align:left;">
pseudogene
</td>
<td style="text-align:left;">
predicted gene 45209
</td>
</tr>
<tr>
<td style="text-align:left;">
7
</td>
<td style="text-align:left;">
pseudogene
</td>
<td style="text-align:right;">
100.76062
</td>
<td style="text-align:right;">
100.76182
</td>
<td style="text-align:left;">
negative
</td>
<td style="text-align:left;">
MGI_C57BL6J_3781379
</td>
<td style="text-align:left;">
Gm3200
</td>
<td style="text-align:left;">
NCBI_Gene:100041204,ENSEMBL:ENSMUSG00000097388
</td>
<td style="text-align:left;">
MGI:3781379
</td>
<td style="text-align:left;">
pseudogene
</td>
<td style="text-align:left;">
predicted pseudogene 3200
</td>
</tr>
<tr>
<td style="text-align:left;">
7
</td>
<td style="text-align:left;">
gene
</td>
<td style="text-align:right;">
100.81135
</td>
<td style="text-align:right;">
100.81264
</td>
<td style="text-align:left;">
positive
</td>
<td style="text-align:left;">
MGI_C57BL6J_5753529
</td>
<td style="text-align:left;">
Gm44953
</td>
<td style="text-align:left;">
ENSEMBL:ENSMUSG00000109109
</td>
<td style="text-align:left;">
MGI:5753529
</td>
<td style="text-align:left;">
unclassified gene
</td>
<td style="text-align:left;">
predicted gene 44953
</td>
</tr>
<tr>
<td style="text-align:left;">
7
</td>
<td style="text-align:left;">
gene
</td>
<td style="text-align:right;">
100.84585
</td>
<td style="text-align:right;">
100.86348
</td>
<td style="text-align:left;">
negative
</td>
<td style="text-align:left;">
MGI_C57BL6J_2443373
</td>
<td style="text-align:left;">
Relt
</td>
<td style="text-align:left;">
NCBI_Gene:320100,ENSEMBL:ENSMUSG00000008318
</td>
<td style="text-align:left;">
MGI:2443373
</td>
<td style="text-align:left;">
protein coding gene
</td>
<td style="text-align:left;">
RELT tumor necrosis factor receptor
</td>
</tr>
<tr>
<td style="text-align:left;">
7
</td>
<td style="text-align:left;">
gene
</td>
<td style="text-align:right;">
100.86975
</td>
<td style="text-align:right;">
100.93216
</td>
<td style="text-align:left;">
negative
</td>
<td style="text-align:left;">
MGI_C57BL6J_2673002
</td>
<td style="text-align:left;">
Arhgef17
</td>
<td style="text-align:left;">
NCBI_Gene:207212,ENSEMBL:ENSMUSG00000032875
</td>
<td style="text-align:left;">
MGI:2673002
</td>
<td style="text-align:left;">
protein coding gene
</td>
<td style="text-align:left;">
Rho guanine nucleotide exchange factor (GEF) 17
</td>
</tr>
<tr>
<td style="text-align:left;">
7
</td>
<td style="text-align:left;">
gene
</td>
<td style="text-align:right;">
100.88231
</td>
<td style="text-align:right;">
100.88241
</td>
<td style="text-align:left;">
negative
</td>
<td style="text-align:left;">
MGI_C57BL6J_4834321
</td>
<td style="text-align:left;">
Mir3102
</td>
<td style="text-align:left;">
miRBase:MI0014099,NCBI_Gene:100526508,ENSEMBL:ENSMUSG00000093296
</td>
<td style="text-align:left;">
MGI:4834321
</td>
<td style="text-align:left;">
miRNA gene
</td>
<td style="text-align:left;">
microRNA 3102
</td>
</tr>
<tr>
<td style="text-align:left;">
7
</td>
<td style="text-align:left;">
gene
</td>
<td style="text-align:right;">
100.93763
</td>
<td style="text-align:right;">
100.97550
</td>
<td style="text-align:left;">
negative
</td>
<td style="text-align:left;">
MGI_C57BL6J_2673874
</td>
<td style="text-align:left;">
P2ry6
</td>
<td style="text-align:left;">
NCBI_Gene:233571,ENSEMBL:ENSMUSG00000048779
</td>
<td style="text-align:left;">
MGI:2673874
</td>
<td style="text-align:left;">
protein coding gene
</td>
<td style="text-align:left;">
pyrimidinergic receptor P2Y, G-protein coupled, 6
</td>
</tr>
<tr>
<td style="text-align:left;">
7
</td>
<td style="text-align:left;">
gene
</td>
<td style="text-align:right;">
100.95082
</td>
<td style="text-align:right;">
100.95156
</td>
<td style="text-align:left;">
positive
</td>
<td style="text-align:left;">
MGI_C57BL6J_5753121
</td>
<td style="text-align:left;">
Gm44545
</td>
<td style="text-align:left;">
ENSEMBL:ENSMUSG00000109132
</td>
<td style="text-align:left;">
MGI:5753121
</td>
<td style="text-align:left;">
unclassified gene
</td>
<td style="text-align:left;">
predicted gene 44545
</td>
</tr>
<tr>
<td style="text-align:left;">
7
</td>
<td style="text-align:left;">
gene
</td>
<td style="text-align:right;">
100.99657
</td>
<td style="text-align:right;">
101.01287
</td>
<td style="text-align:left;">
negative
</td>
<td style="text-align:left;">
MGI_C57BL6J_105107
</td>
<td style="text-align:left;">
P2ry2
</td>
<td style="text-align:left;">
NCBI_Gene:18442,ENSEMBL:ENSMUSG00000032860
</td>
<td style="text-align:left;">
MGI:105107
</td>
<td style="text-align:left;">
protein coding gene
</td>
<td style="text-align:left;">
purinergic receptor P2Y, G-protein coupled 2
</td>
</tr>
<tr>
<td style="text-align:left;">
7
</td>
<td style="text-align:left;">
pseudogene
</td>
<td style="text-align:right;">
101.01862
</td>
<td style="text-align:right;">
101.01887
</td>
<td style="text-align:left;">
negative
</td>
<td style="text-align:left;">
MGI_C57BL6J_5753646
</td>
<td style="text-align:left;">
Gm45070
</td>
<td style="text-align:left;">
ENSEMBL:ENSMUSG00000108895
</td>
<td style="text-align:left;">
MGI:5753646
</td>
<td style="text-align:left;">
pseudogene
</td>
<td style="text-align:left;">
predicted gene 45070
</td>
</tr>
<tr>
<td style="text-align:left;">
7
</td>
<td style="text-align:left;">
pseudogene
</td>
<td style="text-align:right;">
101.06122
</td>
<td style="text-align:right;">
101.06148
</td>
<td style="text-align:left;">
negative
</td>
<td style="text-align:left;">
MGI_C57BL6J_3645951
</td>
<td style="text-align:left;">
Gm5735
</td>
<td style="text-align:left;">
NCBI_Gene:435996,ENSEMBL:ENSMUSG00000109171
</td>
<td style="text-align:left;">
MGI:3645951
</td>
<td style="text-align:left;">
pseudogene
</td>
<td style="text-align:left;">
predicted gene 5735
</td>
</tr>
<tr>
<td style="text-align:left;">
7
</td>
<td style="text-align:left;">
gene
</td>
<td style="text-align:right;">
101.09286
</td>
<td style="text-align:right;">
101.28441
</td>
<td style="text-align:left;">
positive
</td>
<td style="text-align:left;">
MGI_C57BL6J_2448475
</td>
<td style="text-align:left;">
Fchsd2
</td>
<td style="text-align:left;">
NCBI_Gene:207278,ENSEMBL:ENSMUSG00000030691
</td>
<td style="text-align:left;">
MGI:2448475
</td>
<td style="text-align:left;">
protein coding gene
</td>
<td style="text-align:left;">
FCH and double SH3 domains 2
</td>
</tr>
<tr>
<td style="text-align:left;">
7
</td>
<td style="text-align:left;">
gene
</td>
<td style="text-align:right;">
101.11355
</td>
<td style="text-align:right;">
101.11780
</td>
<td style="text-align:left;">
positive
</td>
<td style="text-align:left;">
MGI_C57BL6J_2442785
</td>
<td style="text-align:left;">
6030496E16Rik
</td>
<td style="text-align:left;">
NA
</td>
<td style="text-align:left;">
NA
</td>
<td style="text-align:left;">
unclassified gene
</td>
<td style="text-align:left;">
RIKEN cDNA 6030496E16 gene
</td>
</tr>
<tr>
<td style="text-align:left;">
7
</td>
<td style="text-align:left;">
pseudogene
</td>
<td style="text-align:right;">
101.15460
</td>
<td style="text-align:right;">
101.15514
</td>
<td style="text-align:left;">
negative
</td>
<td style="text-align:left;">
MGI_C57BL6J_3643600
</td>
<td style="text-align:left;">
Gm6341
</td>
<td style="text-align:left;">
NCBI_Gene:622673,ENSEMBL:ENSMUSG00000083121
</td>
<td style="text-align:left;">
MGI:3643600
</td>
<td style="text-align:left;">
pseudogene
</td>
<td style="text-align:left;">
predicted pseudogene 6341
</td>
</tr>
<tr>
<td style="text-align:left;">
7
</td>
<td style="text-align:left;">
gene
</td>
<td style="text-align:right;">
101.20767
</td>
<td style="text-align:right;">
101.21103
</td>
<td style="text-align:left;">
positive
</td>
<td style="text-align:left;">
MGI_C57BL6J_3026972
</td>
<td style="text-align:left;">
6430502G17Rik
</td>
<td style="text-align:left;">
NA
</td>
<td style="text-align:left;">
NA
</td>
<td style="text-align:left;">
unclassified gene
</td>
<td style="text-align:left;">
RIKEN cDNA 6430502G17 gene
</td>
</tr>
<tr>
<td style="text-align:left;">
7
</td>
<td style="text-align:left;">
pseudogene
</td>
<td style="text-align:right;">
101.23822
</td>
<td style="text-align:right;">
101.23928
</td>
<td style="text-align:left;">
positive
</td>
<td style="text-align:left;">
MGI_C57BL6J_6096209
</td>
<td style="text-align:left;">
Gm47324
</td>
<td style="text-align:left;">
ENSEMBL:ENSMUSG00000111425
</td>
<td style="text-align:left;">
MGI:6096209
</td>
<td style="text-align:left;">
pseudogene
</td>
<td style="text-align:left;">
predicted gene, 47324
</td>
</tr>
<tr>
<td style="text-align:left;">
7
</td>
<td style="text-align:left;">
gene
</td>
<td style="text-align:right;">
101.26303
</td>
<td style="text-align:right;">
101.30227
</td>
<td style="text-align:left;">
negative
</td>
<td style="text-align:left;">
MGI_C57BL6J_1920933
</td>
<td style="text-align:left;">
Atg16l2
</td>
<td style="text-align:left;">
NCBI_Gene:73683,ENSEMBL:ENSMUSG00000047767
</td>
<td style="text-align:left;">
MGI:1920933
</td>
<td style="text-align:left;">
protein coding gene
</td>
<td style="text-align:left;">
autophagy related 16-like 2 (S. cerevisiae)
</td>
</tr>
<tr>
<td style="text-align:left;">
7
</td>
<td style="text-align:left;">
gene
</td>
<td style="text-align:right;">
101.31709
</td>
<td style="text-align:right;">
101.34663
</td>
<td style="text-align:left;">
positive
</td>
<td style="text-align:left;">
MGI_C57BL6J_1860093
</td>
<td style="text-align:left;">
Stard10
</td>
<td style="text-align:left;">
NCBI_Gene:56018,ENSEMBL:ENSMUSG00000030688
</td>
<td style="text-align:left;">
MGI:1860093
</td>
<td style="text-align:left;">
protein coding gene
</td>
<td style="text-align:left;">
START domain containing 10
</td>
</tr>
<tr>
<td style="text-align:left;">
7
</td>
<td style="text-align:left;">
gene
</td>
<td style="text-align:right;">
101.31772
</td>
<td style="text-align:right;">
101.32623
</td>
<td style="text-align:left;">
negative
</td>
<td style="text-align:left;">
MGI_C57BL6J_5141941
</td>
<td style="text-align:left;">
Gm20476
</td>
<td style="text-align:left;">
ENSEMBL:ENSMUSG00000092364
</td>
<td style="text-align:left;">
MGI:5141941
</td>
<td style="text-align:left;">
lncRNA gene
</td>
<td style="text-align:left;">
predicted gene 20476
</td>
</tr>
<tr>
<td style="text-align:left;">
7
</td>
<td style="text-align:left;">
pseudogene
</td>
<td style="text-align:right;">
101.33813
</td>
<td style="text-align:right;">
101.33847
</td>
<td style="text-align:left;">
positive
</td>
<td style="text-align:left;">
MGI_C57BL6J_5791180
</td>
<td style="text-align:left;">
Gm45344
</td>
<td style="text-align:left;">
ENSEMBL:ENSMUSG00000110099
</td>
<td style="text-align:left;">
MGI:5791180
</td>
<td style="text-align:left;">
pseudogene
</td>
<td style="text-align:left;">
predicted gene 45344
</td>
</tr>
<tr>
<td style="text-align:left;">
7
</td>
<td style="text-align:left;">
gene
</td>
<td style="text-align:right;">
101.34691
</td>
<td style="text-align:right;">
101.34798
</td>
<td style="text-align:left;">
negative
</td>
<td style="text-align:left;">
MGI_C57BL6J_5791456
</td>
<td style="text-align:left;">
Gm45620
</td>
<td style="text-align:left;">
NCBI_Gene:108167329,ENSEMBL:ENSMUSG00000110239
</td>
<td style="text-align:left;">
MGI:5791456
</td>
<td style="text-align:left;">
lncRNA gene
</td>
<td style="text-align:left;">
predicted gene 45620
</td>
</tr>
<tr>
<td style="text-align:left;">
7
</td>
<td style="text-align:left;">
gene
</td>
<td style="text-align:right;">
101.34765
</td>
<td style="text-align:right;">
101.41259
</td>
<td style="text-align:left;">
positive
</td>
<td style="text-align:left;">
MGI_C57BL6J_1916960
</td>
<td style="text-align:left;">
Arap1
</td>
<td style="text-align:left;">
NCBI_Gene:69710,ENSEMBL:ENSMUSG00000032812
</td>
<td style="text-align:left;">
MGI:1916960
</td>
<td style="text-align:left;">
protein coding gene
</td>
<td style="text-align:left;">
ArfGAP with RhoGAP domain, ankyrin repeat and PH domain 1
</td>
</tr>
<tr>
<td style="text-align:left;">
7
</td>
<td style="text-align:left;">
gene
</td>
<td style="text-align:right;">
101.34821
</td>
<td style="text-align:right;">
101.35542
</td>
<td style="text-align:left;">
negative
</td>
<td style="text-align:left;">
MGI_C57BL6J_5594378
</td>
<td style="text-align:left;">
Gm35219
</td>
<td style="text-align:left;">
NCBI_Gene:102638725
</td>
<td style="text-align:left;">
MGI:5594378
</td>
<td style="text-align:left;">
lncRNA gene
</td>
<td style="text-align:left;">
predicted gene, 35219
</td>
</tr>
<tr>
<td style="text-align:left;">
7
</td>
<td style="text-align:left;">
gene
</td>
<td style="text-align:right;">
101.35127
</td>
<td style="text-align:right;">
101.35444
</td>
<td style="text-align:left;">
negative
</td>
<td style="text-align:left;">
MGI_C57BL6J_5791384
</td>
<td style="text-align:left;">
Gm45548
</td>
<td style="text-align:left;">
ENSEMBL:ENSMUSG00000109656
</td>
<td style="text-align:left;">
MGI:5791384
</td>
<td style="text-align:left;">
unclassified gene
</td>
<td style="text-align:left;">
predicted gene 45548
</td>
</tr>
<tr>
<td style="text-align:left;">
7
</td>
<td style="text-align:left;">
gene
</td>
<td style="text-align:right;">
101.41089
</td>
<td style="text-align:right;">
101.51282
</td>
<td style="text-align:left;">
positive
</td>
<td style="text-align:left;">
MGI_C57BL6J_5804952
</td>
<td style="text-align:left;">
Gm45837
</td>
<td style="text-align:left;">
ENSEMBL:ENSMUSG00000030653
</td>
<td style="text-align:left;">
MGI:5804952
</td>
<td style="text-align:left;">
protein coding gene
</td>
<td style="text-align:left;">
predicted gene 45837
</td>
</tr>
<tr>
<td style="text-align:left;">
7
</td>
<td style="text-align:left;">
gene
</td>
<td style="text-align:right;">
101.42169
</td>
<td style="text-align:right;">
101.51283
</td>
<td style="text-align:left;">
positive
</td>
<td style="text-align:left;">
MGI_C57BL6J_2446107
</td>
<td style="text-align:left;">
Pde2a
</td>
<td style="text-align:left;">
NCBI_Gene:207728,ENSEMBL:ENSMUSG00000110195
</td>
<td style="text-align:left;">
MGI:2446107
</td>
<td style="text-align:left;">
protein coding gene
</td>
<td style="text-align:left;">
phosphodiesterase 2A, cGMP-stimulated
</td>
</tr>
<tr>
<td style="text-align:left;">
7
</td>
<td style="text-align:left;">
gene
</td>
<td style="text-align:right;">
101.47538
</td>
<td style="text-align:right;">
101.47544
</td>
<td style="text-align:left;">
positive
</td>
<td style="text-align:left;">
MGI_C57BL6J_2676824
</td>
<td style="text-align:left;">
Mir139
</td>
<td style="text-align:left;">
miRBase:MI0000693,NCBI_Gene:387157,ENSEMBL:ENSMUSG00000065446
</td>
<td style="text-align:left;">
MGI:2676824
</td>
<td style="text-align:left;">
miRNA gene
</td>
<td style="text-align:left;">
microRNA 139
</td>
</tr>
<tr>
<td style="text-align:left;">
7
</td>
<td style="text-align:left;">
gene
</td>
<td style="text-align:right;">
101.51819
</td>
<td style="text-align:right;">
101.53866
</td>
<td style="text-align:left;">
negative
</td>
<td style="text-align:left;">
MGI_C57BL6J_5594522
</td>
<td style="text-align:left;">
Gm35363
</td>
<td style="text-align:left;">
NCBI_Gene:102638917,ENSEMBL:ENSMUSG00000110301
</td>
<td style="text-align:left;">
MGI:5594522
</td>
<td style="text-align:left;">
lncRNA gene
</td>
<td style="text-align:left;">
predicted gene, 35363
</td>
</tr>
<tr>
<td style="text-align:left;">
7
</td>
<td style="text-align:left;">
gene
</td>
<td style="text-align:right;">
101.54787
</td>
<td style="text-align:right;">
101.55746
</td>
<td style="text-align:left;">
positive
</td>
<td style="text-align:left;">
MGI_C57BL6J_5011950
</td>
<td style="text-align:left;">
Gm19765
</td>
<td style="text-align:left;">
NCBI_Gene:102638993
</td>
<td style="text-align:left;">
MGI:5011950
</td>
<td style="text-align:left;">
lncRNA gene
</td>
<td style="text-align:left;">
predicted gene, 19765
</td>
</tr>
<tr>
<td style="text-align:left;">
7
</td>
<td style="text-align:left;">
pseudogene
</td>
<td style="text-align:right;">
101.55245
</td>
<td style="text-align:right;">
101.56087
</td>
<td style="text-align:left;">
negative
</td>
<td style="text-align:left;">
MGI_C57BL6J_107546
</td>
<td style="text-align:left;">
Art2a
</td>
<td style="text-align:left;">
NCBI_Gene:11871,ENSEMBL:ENSMUSG00000092517
</td>
<td style="text-align:left;">
MGI:107546
</td>
<td style="text-align:left;">
polymorphic pseudogene
</td>
<td style="text-align:left;">
ADP-ribosyltransferase 2a
</td>
</tr>
<tr>
<td style="text-align:left;">
7
</td>
<td style="text-align:left;">
pseudogene
</td>
<td style="text-align:right;">
101.57342
</td>
<td style="text-align:right;">
101.57436
</td>
<td style="text-align:left;">
negative
</td>
<td style="text-align:left;">
MGI_C57BL6J_3644555
</td>
<td style="text-align:left;">
Gm7027
</td>
<td style="text-align:left;">
NCBI_Gene:630138,ENSEMBL:ENSMUSG00000067121
</td>
<td style="text-align:left;">
MGI:3644555
</td>
<td style="text-align:left;">
pseudogene
</td>
<td style="text-align:left;">
predicted gene 7027
</td>
</tr>
<tr>
<td style="text-align:left;">
7
</td>
<td style="text-align:left;">
gene
</td>
<td style="text-align:right;">
101.57573
</td>
<td style="text-align:right;">
101.58560
</td>
<td style="text-align:left;">
negative
</td>
<td style="text-align:left;">
MGI_C57BL6J_107545
</td>
<td style="text-align:left;">
Art2b
</td>
<td style="text-align:left;">
NCBI_Gene:11872,ENSEMBL:ENSMUSG00000030651
</td>
<td style="text-align:left;">
MGI:107545
</td>
<td style="text-align:left;">
protein coding gene
</td>
<td style="text-align:left;">
ADP-ribosyltransferase 2b
</td>
</tr>
<tr>
<td style="text-align:left;">
7
</td>
<td style="text-align:left;">
gene
</td>
<td style="text-align:right;">
101.58159
</td>
<td style="text-align:right;">
101.58283
</td>
<td style="text-align:left;">
positive
</td>
<td style="text-align:left;">
MGI_C57BL6J_5825632
</td>
<td style="text-align:left;">
Gm45995
</td>
<td style="text-align:left;">
NCBI_Gene:108167451
</td>
<td style="text-align:left;">
MGI:5825632
</td>
<td style="text-align:left;">
lncRNA gene
</td>
<td style="text-align:left;">
predicted gene, 45995
</td>
</tr>
<tr>
<td style="text-align:left;">
7
</td>
<td style="text-align:left;">
gene
</td>
<td style="text-align:right;">
101.66359
</td>
<td style="text-align:right;">
101.79551
</td>
<td style="text-align:left;">
positive
</td>
<td style="text-align:left;">
MGI_C57BL6J_1100517
</td>
<td style="text-align:left;">
Clpb
</td>
<td style="text-align:left;">
NCBI_Gene:20480,ENSEMBL:ENSMUSG00000001829
</td>
<td style="text-align:left;">
MGI:1100517
</td>
<td style="text-align:left;">
protein coding gene
</td>
<td style="text-align:left;">
ClpB caseinolytic peptidase B
</td>
</tr>
<tr>
<td style="text-align:left;">
7
</td>
<td style="text-align:left;">
pseudogene
</td>
<td style="text-align:right;">
101.79054
</td>
<td style="text-align:right;">
101.79085
</td>
<td style="text-align:left;">
positive
</td>
<td style="text-align:left;">
MGI_C57BL6J_5791216
</td>
<td style="text-align:left;">
Gm45380
</td>
<td style="text-align:left;">
ENSEMBL:ENSMUSG00000110360
</td>
<td style="text-align:left;">
MGI:5791216
</td>
<td style="text-align:left;">
pseudogene
</td>
<td style="text-align:left;">
predicted gene 45380
</td>
</tr>
<tr>
<td style="text-align:left;">
7
</td>
<td style="text-align:left;">
gene
</td>
<td style="text-align:right;">
101.81831
</td>
<td style="text-align:right;">
101.82273
</td>
<td style="text-align:left;">
positive
</td>
<td style="text-align:left;">
MGI_C57BL6J_106633
</td>
<td style="text-align:left;">
Phox2a
</td>
<td style="text-align:left;">
NCBI_Gene:11859,ENSEMBL:ENSMUSG00000007946
</td>
<td style="text-align:left;">
MGI:106633
</td>
<td style="text-align:left;">
protein coding gene
</td>
<td style="text-align:left;">
paired-like homeobox 2a
</td>
</tr>
<tr>
<td style="text-align:left;">
7
</td>
<td style="text-align:left;">
gene
</td>
<td style="text-align:right;">
101.82263
</td>
<td style="text-align:right;">
101.83823
</td>
<td style="text-align:left;">
negative
</td>
<td style="text-align:left;">
MGI_C57BL6J_1333787
</td>
<td style="text-align:left;">
Inppl1
</td>
<td style="text-align:left;">
NCBI_Gene:16332,ENSEMBL:ENSMUSG00000032737
</td>
<td style="text-align:left;">
MGI:1333787
</td>
<td style="text-align:left;">
protein coding gene
</td>
<td style="text-align:left;">
inositol polyphosphate phosphatase-like 1
</td>
</tr>
<tr>
<td style="text-align:left;">
7
</td>
<td style="text-align:left;">
gene
</td>
<td style="text-align:right;">
101.83650
</td>
<td style="text-align:right;">
101.84453
</td>
<td style="text-align:left;">
positive
</td>
<td style="text-align:left;">
MGI_C57BL6J_3708759
</td>
<td style="text-align:left;">
Gm10602
</td>
<td style="text-align:left;">
NCBI_Gene:102639173,ENSEMBL:ENSMUSG00000073985
</td>
<td style="text-align:left;">
MGI:3708759
</td>
<td style="text-align:left;">
lncRNA gene
</td>
<td style="text-align:left;">
predicted gene 10602
</td>
</tr>
<tr>
<td style="text-align:left;">
7
</td>
<td style="text-align:left;">
gene
</td>
<td style="text-align:right;">
101.83999
</td>
<td style="text-align:right;">
101.85719
</td>
<td style="text-align:left;">
negative
</td>
<td style="text-align:left;">
MGI_C57BL6J_95569
</td>
<td style="text-align:left;">
Folr2
</td>
<td style="text-align:left;">
NCBI_Gene:14276,ENSEMBL:ENSMUSG00000032725
</td>
<td style="text-align:left;">
MGI:95569
</td>
<td style="text-align:left;">
protein coding gene
</td>
<td style="text-align:left;">
folate receptor 2 (fetal)
</td>
</tr>
<tr>
<td style="text-align:left;">
7
</td>
<td style="text-align:left;">
pseudogene
</td>
<td style="text-align:right;">
101.85296
</td>
<td style="text-align:right;">
101.85328
</td>
<td style="text-align:left;">
positive
</td>
<td style="text-align:left;">
MGI_C57BL6J_5791155
</td>
<td style="text-align:left;">
Gm45319
</td>
<td style="text-align:left;">
ENSEMBL:ENSMUSG00000110295
</td>
<td style="text-align:left;">
MGI:5791155
</td>
<td style="text-align:left;">
pseudogene
</td>
<td style="text-align:left;">
predicted gene 45319
</td>
</tr>
<tr>
<td style="text-align:left;">
7
</td>
<td style="text-align:left;">
gene
</td>
<td style="text-align:right;">
101.85833
</td>
<td style="text-align:right;">
101.87079
</td>
<td style="text-align:left;">
negative
</td>
<td style="text-align:left;">
MGI_C57BL6J_95568
</td>
<td style="text-align:left;">
Folr1
</td>
<td style="text-align:left;">
NCBI_Gene:14275,ENSEMBL:ENSMUSG00000001827
</td>
<td style="text-align:left;">
MGI:95568
</td>
<td style="text-align:left;">
protein coding gene
</td>
<td style="text-align:left;">
folate receptor 1 (adult)
</td>
</tr>
<tr>
<td style="text-align:left;">
7
</td>
<td style="text-align:left;">
gene
</td>
<td style="text-align:right;">
101.86371
</td>
<td style="text-align:right;">
101.90185
</td>
<td style="text-align:left;">
positive
</td>
<td style="text-align:left;">
MGI_C57BL6J_1922680
</td>
<td style="text-align:left;">
Anapc15
</td>
<td style="text-align:left;">
NCBI_Gene:75430,ENSEMBL:ENSMUSG00000030649
</td>
<td style="text-align:left;">
MGI:1922680
</td>
<td style="text-align:left;">
protein coding gene
</td>
<td style="text-align:left;">
anaphase prompoting complex C subunit 15
</td>
</tr>
<tr>
<td style="text-align:left;">
7
</td>
<td style="text-align:left;">
gene
</td>
<td style="text-align:right;">
101.87086
</td>
<td style="text-align:right;">
101.87958
</td>
<td style="text-align:left;">
positive
</td>
<td style="text-align:left;">
MGI_C57BL6J_5594756
</td>
<td style="text-align:left;">
Gm35597
</td>
<td style="text-align:left;">
NCBI_Gene:102639241
</td>
<td style="text-align:left;">
MGI:5594756
</td>
<td style="text-align:left;">
lncRNA gene
</td>
<td style="text-align:left;">
predicted gene, 35597
</td>
</tr>
<tr>
<td style="text-align:left;">
7
</td>
<td style="text-align:left;">
gene
</td>
<td style="text-align:right;">
101.88118
</td>
<td style="text-align:right;">
101.88128
</td>
<td style="text-align:left;">
positive
</td>
<td style="text-align:left;">
MGI_C57BL6J_5452432
</td>
<td style="text-align:left;">
Gm22655
</td>
<td style="text-align:left;">
ENSEMBL:ENSMUSG00000093388
</td>
<td style="text-align:left;">
MGI:5452432
</td>
<td style="text-align:left;">
miRNA gene
</td>
<td style="text-align:left;">
predicted gene, 22655
</td>
</tr>
<tr>
<td style="text-align:left;">
7
</td>
<td style="text-align:left;">
gene
</td>
<td style="text-align:right;">
101.89837
</td>
<td style="text-align:right;">
101.90641
</td>
<td style="text-align:left;">
negative
</td>
<td style="text-align:left;">
MGI_C57BL6J_3769724
</td>
<td style="text-align:left;">
Tomt
</td>
<td style="text-align:left;">
NCBI_Gene:791260,ENSEMBL:ENSMUSG00000078630
</td>
<td style="text-align:left;">
MGI:3769724
</td>
<td style="text-align:left;">
protein coding gene
</td>
<td style="text-align:left;">
transmembrane O-methyltransferase
</td>
</tr>
<tr>
<td style="text-align:left;">
7
</td>
<td style="text-align:left;">
gene
</td>
<td style="text-align:right;">
101.90584
</td>
<td style="text-align:right;">
101.92668
</td>
<td style="text-align:left;">
positive
</td>
<td style="text-align:left;">
MGI_C57BL6J_1913758
</td>
<td style="text-align:left;">
Lamtor1
</td>
<td style="text-align:left;">
NCBI_Gene:66508,ENSEMBL:ENSMUSG00000030842
</td>
<td style="text-align:left;">
MGI:1913758
</td>
<td style="text-align:left;">
protein coding gene
</td>
<td style="text-align:left;">
late endosomal/lysosomal adaptor, MAPK and MTOR activator 1
</td>
</tr>
<tr>
<td style="text-align:left;">
7
</td>
<td style="text-align:left;">
gene
</td>
<td style="text-align:right;">
101.91298
</td>
<td style="text-align:right;">
101.93392
</td>
<td style="text-align:left;">
negative
</td>
<td style="text-align:left;">
MGI_C57BL6J_1916608
</td>
<td style="text-align:left;">
Lrrc51
</td>
<td style="text-align:left;">
NCBI_Gene:69358,ENSEMBL:ENSMUSG00000064307
</td>
<td style="text-align:left;">
MGI:1916608
</td>
<td style="text-align:left;">
protein coding gene
</td>
<td style="text-align:left;">
leucine rich repeat containing 51
</td>
</tr>
<tr>
<td style="text-align:left;">
7
</td>
<td style="text-align:left;">
gene
</td>
<td style="text-align:right;">
101.93400
</td>
<td style="text-align:right;">
102.01496
</td>
<td style="text-align:left;">
positive
</td>
<td style="text-align:left;">
MGI_C57BL6J_2443665
</td>
<td style="text-align:left;">
Numa1
</td>
<td style="text-align:left;">
NCBI_Gene:101706,ENSEMBL:ENSMUSG00000066306
</td>
<td style="text-align:left;">
MGI:2443665
</td>
<td style="text-align:left;">
protein coding gene
</td>
<td style="text-align:left;">
nuclear mitotic apparatus protein 1
</td>
</tr>
<tr>
<td style="text-align:left;">
7
</td>
<td style="text-align:left;">
pseudogene
</td>
<td style="text-align:right;">
101.94596
</td>
<td style="text-align:right;">
101.94651
</td>
<td style="text-align:left;">
negative
</td>
<td style="text-align:left;">
MGI_C57BL6J_3779801
</td>
<td style="text-align:left;">
Gm8523
</td>
<td style="text-align:left;">
NCBI_Gene:100417414,ENSEMBL:ENSMUSG00000110150
</td>
<td style="text-align:left;">
MGI:3779801
</td>
<td style="text-align:left;">
pseudogene
</td>
<td style="text-align:left;">
predicted gene 8523
</td>
</tr>
<tr>
<td style="text-align:left;">
7
</td>
<td style="text-align:left;">
pseudogene
</td>
<td style="text-align:right;">
101.95697
</td>
<td style="text-align:right;">
101.95728
</td>
<td style="text-align:left;">
negative
</td>
<td style="text-align:left;">
MGI_C57BL6J_5791148
</td>
<td style="text-align:left;">
Gm45312
</td>
<td style="text-align:left;">
ENSEMBL:ENSMUSG00000109872
</td>
<td style="text-align:left;">
MGI:5791148
</td>
<td style="text-align:left;">
pseudogene
</td>
<td style="text-align:left;">
predicted gene 45312
</td>
</tr>
<tr>
<td style="text-align:left;">
7
</td>
<td style="text-align:left;">
gene
</td>
<td style="text-align:right;">
101.97966
</td>
<td style="text-align:right;">
101.98229
</td>
<td style="text-align:left;">
positive
</td>
<td style="text-align:left;">
MGI_C57BL6J_2444705
</td>
<td style="text-align:left;">
C030040A22Rik
</td>
<td style="text-align:left;">
NA
</td>
<td style="text-align:left;">
NA
</td>
<td style="text-align:left;">
unclassified gene
</td>
<td style="text-align:left;">
RIKEN cDNA C030040A22 gene
</td>
</tr>
<tr>
<td style="text-align:left;">
7
</td>
<td style="text-align:left;">
pseudogene
</td>
<td style="text-align:right;">
101.98008
</td>
<td style="text-align:right;">
101.98026
</td>
<td style="text-align:left;">
negative
</td>
<td style="text-align:left;">
MGI_C57BL6J_5791149
</td>
<td style="text-align:left;">
Gm45313
</td>
<td style="text-align:left;">
ENSEMBL:ENSMUSG00000110287
</td>
<td style="text-align:left;">
MGI:5791149
</td>
<td style="text-align:left;">
pseudogene
</td>
<td style="text-align:left;">
predicted gene 45313
</td>
</tr>
<tr>
<td style="text-align:left;">
7
</td>
<td style="text-align:left;">
gene
</td>
<td style="text-align:right;">
101.98867
</td>
<td style="text-align:right;">
101.99095
</td>
<td style="text-align:left;">
negative
</td>
<td style="text-align:left;">
MGI_C57BL6J_5621946
</td>
<td style="text-align:left;">
Gm39061
</td>
<td style="text-align:left;">
NCBI_Gene:105243016
</td>
<td style="text-align:left;">
MGI:5621946
</td>
<td style="text-align:left;">
lncRNA gene
</td>
<td style="text-align:left;">
predicted gene, 39061
</td>
</tr>
<tr>
<td style="text-align:left;">
7
</td>
<td style="text-align:left;">
gene
</td>
<td style="text-align:right;">
102.01489
</td>
<td style="text-align:right;">
102.01869
</td>
<td style="text-align:left;">
negative
</td>
<td style="text-align:left;">
MGI_C57BL6J_1333800
</td>
<td style="text-align:left;">
Il18bp
</td>
<td style="text-align:left;">
NCBI_Gene:16068,ENSEMBL:ENSMUSG00000070427
</td>
<td style="text-align:left;">
MGI:1333800
</td>
<td style="text-align:left;">
protein coding gene
</td>
<td style="text-align:left;">
interleukin 18 binding protein
</td>
</tr>
<tr>
<td style="text-align:left;">
7
</td>
<td style="text-align:left;">
gene
</td>
<td style="text-align:right;">
102.01914
</td>
<td style="text-align:right;">
102.07244
</td>
<td style="text-align:left;">
negative
</td>
<td style="text-align:left;">
MGI_C57BL6J_1922462
</td>
<td style="text-align:left;">
Rnf121
</td>
<td style="text-align:left;">
NCBI_Gene:75212,ENSEMBL:ENSMUSG00000070426
</td>
<td style="text-align:left;">
MGI:1922462
</td>
<td style="text-align:left;">
protein coding gene
</td>
<td style="text-align:left;">
ring finger protein 121
</td>
</tr>
<tr>
<td style="text-align:left;">
7
</td>
<td style="text-align:left;">
gene
</td>
<td style="text-align:right;">
102.06549
</td>
<td style="text-align:right;">
102.08376
</td>
<td style="text-align:left;">
positive
</td>
<td style="text-align:left;">
MGI_C57BL6J_5546359
</td>
<td style="text-align:left;">
Xndc1
</td>
<td style="text-align:left;">
NCBI_Gene:102443350,ENSEMBL:ENSMUSG00000099481
</td>
<td style="text-align:left;">
MGI:5546359
</td>
<td style="text-align:left;">
protein coding gene
</td>
<td style="text-align:left;">
Xrcc1 N-terminal domain containing 1
</td>
</tr>
<tr>
<td style="text-align:left;">
7
</td>
<td style="text-align:left;">
gene
</td>
<td style="text-align:right;">
102.06549
</td>
<td style="text-align:right;">
102.09686
</td>
<td style="text-align:left;">
positive
</td>
<td style="text-align:left;">
MGI_C57BL6J_5546370
</td>
<td style="text-align:left;">
Xntrpc
</td>
<td style="text-align:left;">
NCBI_Gene:102443351,ENSEMBL:ENSMUSG00000070425
</td>
<td style="text-align:left;">
MGI:5546370
</td>
<td style="text-align:left;">
protein coding gene
</td>
<td style="text-align:left;">
Xndc1-transient receptor potential cation channel, subfamily C, member 2 readthrough
</td>
</tr>
<tr>
<td style="text-align:left;">
7
</td>
<td style="text-align:left;">
gene
</td>
<td style="text-align:right;">
102.08312
</td>
<td style="text-align:right;">
102.09686
</td>
<td style="text-align:left;">
positive
</td>
<td style="text-align:left;">
MGI_C57BL6J_109527
</td>
<td style="text-align:left;">
Trpc2
</td>
<td style="text-align:left;">
NCBI_Gene:22064,ENSEMBL:ENSMUSG00000100254
</td>
<td style="text-align:left;">
MGI:109527
</td>
<td style="text-align:left;">
protein coding gene
</td>
<td style="text-align:left;">
transient receptor potential cation channel, subfamily C, member 2
</td>
</tr>
<tr>
<td style="text-align:left;">
7
</td>
<td style="text-align:left;">
gene
</td>
<td style="text-align:right;">
102.09688
</td>
<td style="text-align:right;">
102.10949
</td>
<td style="text-align:left;">
negative
</td>
<td style="text-align:left;">
MGI_C57BL6J_107948
</td>
<td style="text-align:left;">
Art5
</td>
<td style="text-align:left;">
NCBI_Gene:11875,ENSEMBL:ENSMUSG00000070424
</td>
<td style="text-align:left;">
MGI:107948
</td>
<td style="text-align:left;">
protein coding gene
</td>
<td style="text-align:left;">
ADP-ribosyltransferase 5
</td>
</tr>
<tr>
<td style="text-align:left;">
7
</td>
<td style="text-align:left;">
gene
</td>
<td style="text-align:right;">
102.10171
</td>
<td style="text-align:right;">
102.11393
</td>
<td style="text-align:left;">
positive
</td>
<td style="text-align:left;">
MGI_C57BL6J_107511
</td>
<td style="text-align:left;">
Art1
</td>
<td style="text-align:left;">
NCBI_Gene:11870,ENSEMBL:ENSMUSG00000030996
</td>
<td style="text-align:left;">
MGI:107511
</td>
<td style="text-align:left;">
protein coding gene
</td>
<td style="text-align:left;">
ADP-ribosyltransferase 1
</td>
</tr>
<tr>
<td style="text-align:left;">
7
</td>
<td style="text-align:left;">
gene
</td>
<td style="text-align:right;">
102.11126
</td>
<td style="text-align:right;">
102.11698
</td>
<td style="text-align:left;">
negative
</td>
<td style="text-align:left;">
MGI_C57BL6J_3609260
</td>
<td style="text-align:left;">
Chrna10
</td>
<td style="text-align:left;">
NCBI_Gene:504186,ENSEMBL:ENSMUSG00000066279
</td>
<td style="text-align:left;">
MGI:3609260
</td>
<td style="text-align:left;">
protein coding gene
</td>
<td style="text-align:left;">
cholinergic receptor, nicotinic, alpha polypeptide 10
</td>
</tr>
<tr>
<td style="text-align:left;">
7
</td>
<td style="text-align:left;">
gene
</td>
<td style="text-align:right;">
102.11339
</td>
<td style="text-align:right;">
102.12206
</td>
<td style="text-align:left;">
positive
</td>
<td style="text-align:left;">
MGI_C57BL6J_5621947
</td>
<td style="text-align:left;">
Gm39062
</td>
<td style="text-align:left;">
NCBI_Gene:105243018
</td>
<td style="text-align:left;">
MGI:5621947
</td>
<td style="text-align:left;">
lncRNA gene
</td>
<td style="text-align:left;">
predicted gene, 39062
</td>
</tr>
<tr>
<td style="text-align:left;">
7
</td>
<td style="text-align:left;">
gene
</td>
<td style="text-align:right;">
102.11940
</td>
<td style="text-align:right;">
102.21019
</td>
<td style="text-align:left;">
negative
</td>
<td style="text-align:left;">
MGI_C57BL6J_109404
</td>
<td style="text-align:left;">
Nup98
</td>
<td style="text-align:left;">
NCBI_Gene:269966,ENSEMBL:ENSMUSG00000063550
</td>
<td style="text-align:left;">
MGI:109404
</td>
<td style="text-align:left;">
protein coding gene
</td>
<td style="text-align:left;">
nucleoporin 98
</td>
</tr>
<tr>
<td style="text-align:left;">
7
</td>
<td style="text-align:left;">
gene
</td>
<td style="text-align:right;">
102.21021
</td>
<td style="text-align:right;">
102.23857
</td>
<td style="text-align:left;">
positive
</td>
<td style="text-align:left;">
MGI_C57BL6J_2385286
</td>
<td style="text-align:left;">
Pgap2
</td>
<td style="text-align:left;">
NCBI_Gene:233575,ENSEMBL:ENSMUSG00000030990
</td>
<td style="text-align:left;">
MGI:2385286
</td>
<td style="text-align:left;">
protein coding gene
</td>
<td style="text-align:left;">
post-GPI attachment to proteins 2
</td>
</tr>
<tr>
<td style="text-align:left;">
7
</td>
<td style="text-align:left;">
gene
</td>
<td style="text-align:right;">
102.23912
</td>
<td style="text-align:right;">
102.25605
</td>
<td style="text-align:left;">
negative
</td>
<td style="text-align:left;">
MGI_C57BL6J_1928370
</td>
<td style="text-align:left;">
Rhog
</td>
<td style="text-align:left;">
NCBI_Gene:56212,ENSEMBL:ENSMUSG00000073982
</td>
<td style="text-align:left;">
MGI:1928370
</td>
<td style="text-align:left;">
protein coding gene
</td>
<td style="text-align:left;">
ras homolog family member G
</td>
</tr>
<tr>
<td style="text-align:left;">
7
</td>
<td style="text-align:left;">
gene
</td>
<td style="text-align:right;">
102.26151
</td>
<td style="text-align:right;">
102.43732
</td>
<td style="text-align:left;">
positive
</td>
<td style="text-align:left;">
MGI_C57BL6J_107476
</td>
<td style="text-align:left;">
Stim1
</td>
<td style="text-align:left;">
NCBI_Gene:20866,ENSEMBL:ENSMUSG00000030987
</td>
<td style="text-align:left;">
MGI:107476
</td>
<td style="text-align:left;">
protein coding gene
</td>
<td style="text-align:left;">
stromal interaction molecule 1
</td>
</tr>
<tr>
<td style="text-align:left;">
7
</td>
<td style="text-align:left;">
gene
</td>
<td style="text-align:right;">
102.36097
</td>
<td style="text-align:right;">
102.36348
</td>
<td style="text-align:left;">
positive
</td>
<td style="text-align:left;">
MGI_C57BL6J_2443525
</td>
<td style="text-align:left;">
A630057J21Rik
</td>
<td style="text-align:left;">
NA
</td>
<td style="text-align:left;">
NA
</td>
<td style="text-align:left;">
unclassified non-coding RNA gene
</td>
<td style="text-align:left;">
RIKEN cDNA A630057J21 gene
</td>
</tr>
<tr>
<td style="text-align:left;">
7
</td>
<td style="text-align:left;">
pseudogene
</td>
<td style="text-align:right;">
102.37750
</td>
<td style="text-align:right;">
102.37834
</td>
<td style="text-align:left;">
negative
</td>
<td style="text-align:left;">
MGI_C57BL6J_5010440
</td>
<td style="text-align:left;">
Gm18255
</td>
<td style="text-align:left;">
NCBI_Gene:100416793,ENSEMBL:ENSMUSG00000110175
</td>
<td style="text-align:left;">
MGI:5010440
</td>
<td style="text-align:left;">
pseudogene
</td>
<td style="text-align:left;">
predicted gene, 18255
</td>
</tr>
<tr>
<td style="text-align:left;">
7
</td>
<td style="text-align:left;">
gene
</td>
<td style="text-align:right;">
102.44169
</td>
<td style="text-align:right;">
102.46977
</td>
<td style="text-align:left;">
positive
</td>
<td style="text-align:left;">
MGI_C57BL6J_98180
</td>
<td style="text-align:left;">
Rrm1
</td>
<td style="text-align:left;">
NCBI_Gene:20133,ENSEMBL:ENSMUSG00000030978
</td>
<td style="text-align:left;">
MGI:98180
</td>
<td style="text-align:left;">
protein coding gene
</td>
<td style="text-align:left;">
ribonucleotide reductase M1
</td>
</tr>
<tr>
<td style="text-align:left;">
7
</td>
<td style="text-align:left;">
gene
</td>
<td style="text-align:right;">
102.45825
</td>
<td style="text-align:right;">
102.45926
</td>
<td style="text-align:left;">
positive
</td>
<td style="text-align:left;">
MGI_C57BL6J_5791235
</td>
<td style="text-align:left;">
Gm45399
</td>
<td style="text-align:left;">
ENSEMBL:ENSMUSG00000109940
</td>
<td style="text-align:left;">
MGI:5791235
</td>
<td style="text-align:left;">
unclassified gene
</td>
<td style="text-align:left;">
predicted gene 45399
</td>
</tr>
<tr>
<td style="text-align:left;">
7
</td>
<td style="text-align:left;">
gene
</td>
<td style="text-align:right;">
102.47046
</td>
<td style="text-align:right;">
102.47668
</td>
<td style="text-align:left;">
positive
</td>
<td style="text-align:left;">
MGI_C57BL6J_5825633
</td>
<td style="text-align:left;">
Gm45996
</td>
<td style="text-align:left;">
NCBI_Gene:108167452
</td>
<td style="text-align:left;">
MGI:5825633
</td>
<td style="text-align:left;">
lncRNA gene
</td>
<td style="text-align:left;">
predicted gene, 45996
</td>
</tr>
<tr>
<td style="text-align:left;">
7
</td>
<td style="text-align:left;">
gene
</td>
<td style="text-align:right;">
102.47658
</td>
<td style="text-align:right;">
102.48168
</td>
<td style="text-align:left;">
negative
</td>
<td style="text-align:left;">
MGI_C57BL6J_3030377
</td>
<td style="text-align:left;">
Olfr543
</td>
<td style="text-align:left;">
NCBI_Gene:257947,ENSEMBL:ENSMUSG00000044814
</td>
<td style="text-align:left;">
MGI:3030377
</td>
<td style="text-align:left;">
protein coding gene
</td>
<td style="text-align:left;">
olfactory receptor 543
</td>
</tr>
<tr>
<td style="text-align:left;">
7
</td>
<td style="text-align:left;">
gene
</td>
<td style="text-align:right;">
102.48203
</td>
<td style="text-align:right;">
102.48831
</td>
<td style="text-align:left;">
negative
</td>
<td style="text-align:left;">
MGI_C57BL6J_3030378
</td>
<td style="text-align:left;">
Olfr544
</td>
<td style="text-align:left;">
NCBI_Gene:257926,ENSEMBL:ENSMUSG00000043925
</td>
<td style="text-align:left;">
MGI:3030378
</td>
<td style="text-align:left;">
protein coding gene
</td>
<td style="text-align:left;">
olfactory receptor 544
</td>
</tr>
<tr>
<td style="text-align:left;">
7
</td>
<td style="text-align:left;">
gene
</td>
<td style="text-align:right;">
102.49147
</td>
<td style="text-align:right;">
102.49957
</td>
<td style="text-align:left;">
negative
</td>
<td style="text-align:left;">
MGI_C57BL6J_3030379
</td>
<td style="text-align:left;">
Olfr545
</td>
<td style="text-align:left;">
NCBI_Gene:258837,ENSEMBL:ENSMUSG00000044824
</td>
<td style="text-align:left;">
MGI:3030379
</td>
<td style="text-align:left;">
protein coding gene
</td>
<td style="text-align:left;">
olfactory receptor 545
</td>
</tr>
<tr>
<td style="text-align:left;">
7
</td>
<td style="text-align:left;">
gene
</td>
<td style="text-align:right;">
102.49227
</td>
<td style="text-align:right;">
102.49372
</td>
<td style="text-align:left;">
positive
</td>
<td style="text-align:left;">
MGI_C57BL6J_3039577
</td>
<td style="text-align:left;">
BC049285
</td>
<td style="text-align:left;">
NA
</td>
<td style="text-align:left;">
NA
</td>
<td style="text-align:left;">
unclassified gene
</td>
<td style="text-align:left;">
cDNA sequence BC049285
</td>
</tr>
<tr>
<td style="text-align:left;">
7
</td>
<td style="text-align:left;">
gene
</td>
<td style="text-align:right;">
102.50548
</td>
<td style="text-align:right;">
102.50642
</td>
<td style="text-align:left;">
positive
</td>
<td style="text-align:left;">
MGI_C57BL6J_5621512
</td>
<td style="text-align:left;">
Gm38627
</td>
<td style="text-align:left;">
NCBI_Gene:102642720
</td>
<td style="text-align:left;">
MGI:5621512
</td>
<td style="text-align:left;">
lncRNA gene
</td>
<td style="text-align:left;">
predicted gene, 38627
</td>
</tr>
<tr>
<td style="text-align:left;">
7
</td>
<td style="text-align:left;">
pseudogene
</td>
<td style="text-align:right;">
102.52035
</td>
<td style="text-align:right;">
102.52106
</td>
<td style="text-align:left;">
negative
</td>
<td style="text-align:left;">
MGI_C57BL6J_3649082
</td>
<td style="text-align:left;">
Gm8556
</td>
<td style="text-align:left;">
NCBI_Gene:667284,ENSEMBL:ENSMUSG00000109822
</td>
<td style="text-align:left;">
MGI:3649082
</td>
<td style="text-align:left;">
pseudogene
</td>
<td style="text-align:left;">
predicted gene 8556
</td>
</tr>
<tr>
<td style="text-align:left;">
7
</td>
<td style="text-align:left;">
pseudogene
</td>
<td style="text-align:right;">
102.52069
</td>
<td style="text-align:right;">
102.52166
</td>
<td style="text-align:left;">
positive
</td>
<td style="text-align:left;">
MGI_C57BL6J_3030380
</td>
<td style="text-align:left;">
Olfr546-ps1
</td>
<td style="text-align:left;">
NCBI_Gene:404405,ENSEMBL:ENSMUSG00000109906
</td>
<td style="text-align:left;">
MGI:3030380
</td>
<td style="text-align:left;">
pseudogene
</td>
<td style="text-align:left;">
olfactory receptor 546, pseudogene 1
</td>
</tr>
<tr>
<td style="text-align:left;">
7
</td>
<td style="text-align:left;">
gene
</td>
<td style="text-align:right;">
102.53475
</td>
<td style="text-align:right;">
102.53569
</td>
<td style="text-align:left;">
positive
</td>
<td style="text-align:left;">
MGI_C57BL6J_3030381
</td>
<td style="text-align:left;">
Olfr547
</td>
<td style="text-align:left;">
NCBI_Gene:259083,ENSEMBL:ENSMUSG00000073979
</td>
<td style="text-align:left;">
MGI:3030381
</td>
<td style="text-align:left;">
protein coding gene
</td>
<td style="text-align:left;">
olfactory receptor 547
</td>
</tr>
<tr>
<td style="text-align:left;">
7
</td>
<td style="text-align:left;">
pseudogene
</td>
<td style="text-align:right;">
102.54090
</td>
<td style="text-align:right;">
102.54589
</td>
<td style="text-align:left;">
positive
</td>
<td style="text-align:left;">
MGI_C57BL6J_3030382
</td>
<td style="text-align:left;">
Olfr548-ps1
</td>
<td style="text-align:left;">
NCBI_Gene:258229,ENSEMBL:ENSMUSG00000073978
</td>
<td style="text-align:left;">
MGI:3030382
</td>
<td style="text-align:left;">
pseudogene
</td>
<td style="text-align:left;">
olfactory receptor 548, pseudogene 1
</td>
</tr>
<tr>
<td style="text-align:left;">
7
</td>
<td style="text-align:left;">
gene
</td>
<td style="text-align:right;">
102.54987
</td>
<td style="text-align:right;">
102.55584
</td>
<td style="text-align:left;">
positive
</td>
<td style="text-align:left;">
MGI_C57BL6J_3030383
</td>
<td style="text-align:left;">
Olfr549
</td>
<td style="text-align:left;">
NCBI_Gene:259105,ENSEMBL:ENSMUSG00000073977
</td>
<td style="text-align:left;">
MGI:3030383
</td>
<td style="text-align:left;">
protein coding gene
</td>
<td style="text-align:left;">
olfactory receptor 549
</td>
</tr>
<tr>
<td style="text-align:left;">
7
</td>
<td style="text-align:left;">
gene
</td>
<td style="text-align:right;">
102.55792
</td>
<td style="text-align:right;">
102.56549
</td>
<td style="text-align:left;">
negative
</td>
<td style="text-align:left;">
MGI_C57BL6J_106657
</td>
<td style="text-align:left;">
Trim21
</td>
<td style="text-align:left;">
NCBI_Gene:20821,ENSEMBL:ENSMUSG00000030966
</td>
<td style="text-align:left;">
MGI:106657
</td>
<td style="text-align:left;">
protein coding gene
</td>
<td style="text-align:left;">
tripartite motif-containing 21
</td>
</tr>
<tr>
<td style="text-align:left;">
7
</td>
<td style="text-align:left;">
pseudogene
</td>
<td style="text-align:right;">
102.56848
</td>
<td style="text-align:right;">
102.56973
</td>
<td style="text-align:left;">
negative
</td>
<td style="text-align:left;">
MGI_C57BL6J_3645143
</td>
<td style="text-align:left;">
Gm5339
</td>
<td style="text-align:left;">
NCBI_Gene:384699,ENSEMBL:ENSMUSG00000109756
</td>
<td style="text-align:left;">
MGI:3645143
</td>
<td style="text-align:left;">
pseudogene
</td>
<td style="text-align:left;">
predicted gene 5339
</td>
</tr>
<tr>
<td style="text-align:left;">
7
</td>
<td style="text-align:left;">
gene
</td>
<td style="text-align:right;">
102.57143
</td>
<td style="text-align:right;">
102.58301
</td>
<td style="text-align:left;">
positive
</td>
<td style="text-align:left;">
MGI_C57BL6J_3030384
</td>
<td style="text-align:left;">
Olfr550
</td>
<td style="text-align:left;">
NCBI_Gene:259108,ENSEMBL:ENSMUSG00000073975
</td>
<td style="text-align:left;">
MGI:3030384
</td>
<td style="text-align:left;">
protein coding gene
</td>
<td style="text-align:left;">
olfactory receptor 550
</td>
</tr>
<tr>
<td style="text-align:left;">
7
</td>
<td style="text-align:left;">
gene
</td>
<td style="text-align:right;">
102.58351
</td>
<td style="text-align:right;">
102.59189
</td>
<td style="text-align:left;">
negative
</td>
<td style="text-align:left;">
MGI_C57BL6J_3030385
</td>
<td style="text-align:left;">
Olfr551
</td>
<td style="text-align:left;">
NCBI_Gene:258750,ENSEMBL:ENSMUSG00000073974
</td>
<td style="text-align:left;">
MGI:3030385
</td>
<td style="text-align:left;">
protein coding gene
</td>
<td style="text-align:left;">
olfactory receptor 551
</td>
</tr>
<tr>
<td style="text-align:left;">
7
</td>
<td style="text-align:left;">
gene
</td>
<td style="text-align:right;">
102.59777
</td>
<td style="text-align:right;">
102.60813
</td>
<td style="text-align:left;">
positive
</td>
<td style="text-align:left;">
MGI_C57BL6J_3030386
</td>
<td style="text-align:left;">
Olfr552
</td>
<td style="text-align:left;">
NCBI_Gene:259106,ENSEMBL:ENSMUSG00000073973
</td>
<td style="text-align:left;">
MGI:3030386
</td>
<td style="text-align:left;">
protein coding gene
</td>
<td style="text-align:left;">
olfactory receptor 552
</td>
</tr>
<tr>
<td style="text-align:left;">
7
</td>
<td style="text-align:left;">
gene
</td>
<td style="text-align:right;">
102.61128
</td>
<td style="text-align:right;">
102.61822
</td>
<td style="text-align:left;">
negative
</td>
<td style="text-align:left;">
MGI_C57BL6J_3030387
</td>
<td style="text-align:left;">
Olfr553
</td>
<td style="text-align:left;">
NCBI_Gene:233578,ENSEMBL:ENSMUSG00000073972
</td>
<td style="text-align:left;">
MGI:3030387
</td>
<td style="text-align:left;">
protein coding gene
</td>
<td style="text-align:left;">
olfactory receptor 553
</td>
</tr>
<tr>
<td style="text-align:left;">
7
</td>
<td style="text-align:left;">
gene
</td>
<td style="text-align:right;">
102.63843
</td>
<td style="text-align:right;">
102.64256
</td>
<td style="text-align:left;">
positive
</td>
<td style="text-align:left;">
MGI_C57BL6J_3030388
</td>
<td style="text-align:left;">
Olfr554
</td>
<td style="text-align:left;">
NCBI_Gene:258322,ENSEMBL:ENSMUSG00000073971
</td>
<td style="text-align:left;">
MGI:3030388
</td>
<td style="text-align:left;">
protein coding gene
</td>
<td style="text-align:left;">
olfactory receptor 554
</td>
</tr>
<tr>
<td style="text-align:left;">
7
</td>
<td style="text-align:left;">
pseudogene
</td>
<td style="text-align:right;">
102.64763
</td>
<td style="text-align:right;">
102.64822
</td>
<td style="text-align:left;">
negative
</td>
<td style="text-align:left;">
MGI_C57BL6J_3781174
</td>
<td style="text-align:left;">
Gm2996
</td>
<td style="text-align:left;">
NCBI_Gene:100040840,ENSEMBL:ENSMUSG00000110251
</td>
<td style="text-align:left;">
MGI:3781174
</td>
<td style="text-align:left;">
pseudogene
</td>
<td style="text-align:left;">
predicted gene 2996
</td>
</tr>
<tr>
<td style="text-align:left;">
7
</td>
<td style="text-align:left;">
gene
</td>
<td style="text-align:right;">
102.65882
</td>
<td style="text-align:right;">
102.65977
</td>
<td style="text-align:left;">
positive
</td>
<td style="text-align:left;">
MGI_C57BL6J_3030389
</td>
<td style="text-align:left;">
Olfr555
</td>
<td style="text-align:left;">
NCBI_Gene:259107,ENSEMBL:ENSMUSG00000073970
</td>
<td style="text-align:left;">
MGI:3030389
</td>
<td style="text-align:left;">
protein coding gene
</td>
<td style="text-align:left;">
olfactory receptor 555
</td>
</tr>
<tr>
<td style="text-align:left;">
7
</td>
<td style="text-align:left;">
gene
</td>
<td style="text-align:right;">
102.66490
</td>
<td style="text-align:right;">
102.67327
</td>
<td style="text-align:left;">
positive
</td>
<td style="text-align:left;">
MGI_C57BL6J_3030390
</td>
<td style="text-align:left;">
Olfr556
</td>
<td style="text-align:left;">
NCBI_Gene:258749,ENSEMBL:ENSMUSG00000073969
</td>
<td style="text-align:left;">
MGI:3030390
</td>
<td style="text-align:left;">
protein coding gene
</td>
<td style="text-align:left;">
olfactory receptor 556
</td>
</tr>
<tr>
<td style="text-align:left;">
7
</td>
<td style="text-align:left;">
pseudogene
</td>
<td style="text-align:right;">
102.67654
</td>
<td style="text-align:right;">
102.67669
</td>
<td style="text-align:left;">
positive
</td>
<td style="text-align:left;">
MGI_C57BL6J_5791523
</td>
<td style="text-align:left;">
Gm45687
</td>
<td style="text-align:left;">
ENSEMBL:ENSMUSG00000110352
</td>
<td style="text-align:left;">
MGI:5791523
</td>
<td style="text-align:left;">
pseudogene
</td>
<td style="text-align:left;">
predicted gene 45687
</td>
</tr>
<tr>
<td style="text-align:left;">
7
</td>
<td style="text-align:left;">
gene
</td>
<td style="text-align:right;">
102.67758
</td>
<td style="text-align:right;">
102.68733
</td>
<td style="text-align:left;">
negative
</td>
<td style="text-align:left;">
MGI_C57BL6J_2142077
</td>
<td style="text-align:left;">
Trim68
</td>
<td style="text-align:left;">
NCBI_Gene:101700,ENSEMBL:ENSMUSG00000073968
</td>
<td style="text-align:left;">
MGI:2142077
</td>
<td style="text-align:left;">
protein coding gene
</td>
<td style="text-align:left;">
tripartite motif-containing 68
</td>
</tr>
<tr>
<td style="text-align:left;">
7
</td>
<td style="text-align:left;">
gene
</td>
<td style="text-align:right;">
102.68545
</td>
<td style="text-align:right;">
102.68622
</td>
<td style="text-align:left;">
negative
</td>
<td style="text-align:left;">
MGI_C57BL6J_5791363
</td>
<td style="text-align:left;">
Gm45527
</td>
<td style="text-align:left;">
ENSEMBL:ENSMUSG00000109618
</td>
<td style="text-align:left;">
MGI:5791363
</td>
<td style="text-align:left;">
unclassified gene
</td>
<td style="text-align:left;">
predicted gene 45527
</td>
</tr>
<tr>
<td style="text-align:left;">
7
</td>
<td style="text-align:left;">
gene
</td>
<td style="text-align:right;">
102.69591
</td>
<td style="text-align:right;">
102.70071
</td>
<td style="text-align:left;">
positive
</td>
<td style="text-align:left;">
MGI_C57BL6J_3030391
</td>
<td style="text-align:left;">
Olfr557
</td>
<td style="text-align:left;">
NCBI_Gene:258358,ENSEMBL:ENSMUSG00000073967
</td>
<td style="text-align:left;">
MGI:3030391
</td>
<td style="text-align:left;">
protein coding gene
</td>
<td style="text-align:left;">
olfactory receptor 557
</td>
</tr>
<tr>
<td style="text-align:left;">
7
</td>
<td style="text-align:left;">
gene
</td>
<td style="text-align:right;">
102.70221
</td>
<td style="text-align:right;">
102.71206
</td>
<td style="text-align:left;">
positive
</td>
<td style="text-align:left;">
MGI_C57BL6J_3030392
</td>
<td style="text-align:left;">
Olfr558
</td>
<td style="text-align:left;">
NCBI_Gene:259097,ENSEMBL:ENSMUSG00000070423
</td>
<td style="text-align:left;">
MGI:3030392
</td>
<td style="text-align:left;">
protein coding gene
</td>
<td style="text-align:left;">
olfactory receptor 558
</td>
</tr>
<tr>
<td style="text-align:left;">
7
</td>
<td style="text-align:left;">
pseudogene
</td>
<td style="text-align:right;">
102.71138
</td>
<td style="text-align:right;">
102.71148
</td>
<td style="text-align:left;">
negative
</td>
<td style="text-align:left;">
MGI_C57BL6J_5791524
</td>
<td style="text-align:left;">
Gm45688
</td>
<td style="text-align:left;">
ENSEMBL:ENSMUSG00000110376
</td>
<td style="text-align:left;">
MGI:5791524
</td>
<td style="text-align:left;">
pseudogene
</td>
<td style="text-align:left;">
predicted gene 45688
</td>
</tr>
<tr>
<td style="text-align:left;">
7
</td>
<td style="text-align:left;">
gene
</td>
<td style="text-align:right;">
102.71215
</td>
<td style="text-align:right;">
102.71907
</td>
<td style="text-align:left;">
negative
</td>
<td style="text-align:left;">
MGI_C57BL6J_109302
</td>
<td style="text-align:left;">
Olfr33
</td>
<td style="text-align:left;">
NCBI_Gene:18332,ENSEMBL:ENSMUSG00000066273
</td>
<td style="text-align:left;">
MGI:109302
</td>
<td style="text-align:left;">
protein coding gene
</td>
<td style="text-align:left;">
olfactory receptor 33
</td>
</tr>
<tr>
<td style="text-align:left;">
7
</td>
<td style="text-align:left;">
gene
</td>
<td style="text-align:right;">
102.72321
</td>
<td style="text-align:right;">
102.72749
</td>
<td style="text-align:left;">
negative
</td>
<td style="text-align:left;">
MGI_C57BL6J_3030393
</td>
<td style="text-align:left;">
Olfr559
</td>
<td style="text-align:left;">
NCBI_Gene:259116,ENSEMBL:ENSMUSG00000066272
</td>
<td style="text-align:left;">
MGI:3030393
</td>
<td style="text-align:left;">
protein coding gene
</td>
<td style="text-align:left;">
olfactory receptor 559
</td>
</tr>
<tr>
<td style="text-align:left;">
7
</td>
<td style="text-align:left;">
gene
</td>
<td style="text-align:right;">
102.73458
</td>
<td style="text-align:right;">
102.73471
</td>
<td style="text-align:left;">
negative
</td>
<td style="text-align:left;">
MGI_C57BL6J_5453987
</td>
<td style="text-align:left;">
Gm24210
</td>
<td style="text-align:left;">
ENSEMBL:ENSMUSG00000088888
</td>
<td style="text-align:left;">
MGI:5453987
</td>
<td style="text-align:left;">
snoRNA gene
</td>
<td style="text-align:left;">
predicted gene, 24210
</td>
</tr>
<tr>
<td style="text-align:left;">
7
</td>
<td style="text-align:left;">
gene
</td>
<td style="text-align:right;">
102.73851
</td>
<td style="text-align:right;">
102.75947
</td>
<td style="text-align:left;">
negative
</td>
<td style="text-align:left;">
MGI_C57BL6J_2157548
</td>
<td style="text-align:left;">
Olfr78
</td>
<td style="text-align:left;">
NCBI_Gene:170639,ENSEMBL:ENSMUSG00000043366
</td>
<td style="text-align:left;">
MGI:2157548
</td>
<td style="text-align:left;">
protein coding gene
</td>
<td style="text-align:left;">
olfactory receptor 78
</td>
</tr>
<tr>
<td style="text-align:left;">
7
</td>
<td style="text-align:left;">
gene
</td>
<td style="text-align:right;">
102.75182
</td>
<td style="text-align:right;">
102.75937
</td>
<td style="text-align:left;">
negative
</td>
<td style="text-align:left;">
MGI_C57BL6J_3030394
</td>
<td style="text-align:left;">
Olfr560
</td>
<td style="text-align:left;">
NCBI_Gene:259117,ENSEMBL:ENSMUSG00000110008
</td>
<td style="text-align:left;">
MGI:3030394
</td>
<td style="text-align:left;">
protein coding gene
</td>
<td style="text-align:left;">
olfactory receptor 560
</td>
</tr>
<tr>
<td style="text-align:left;">
7
</td>
<td style="text-align:left;">
gene
</td>
<td style="text-align:right;">
102.77122
</td>
<td style="text-align:right;">
102.77686
</td>
<td style="text-align:left;">
positive
</td>
<td style="text-align:left;">
MGI_C57BL6J_3030395
</td>
<td style="text-align:left;">
Olfr561
</td>
<td style="text-align:left;">
NCBI_Gene:259096,ENSEMBL:ENSMUSG00000073966
</td>
<td style="text-align:left;">
MGI:3030395
</td>
<td style="text-align:left;">
protein coding gene
</td>
<td style="text-align:left;">
olfactory receptor 561
</td>
</tr>
<tr>
<td style="text-align:left;">
7
</td>
<td style="text-align:left;">
pseudogene
</td>
<td style="text-align:right;">
102.78148
</td>
<td style="text-align:right;">
102.78243
</td>
<td style="text-align:left;">
positive
</td>
<td style="text-align:left;">
MGI_C57BL6J_3030396
</td>
<td style="text-align:left;">
Olfr562-ps1
</td>
<td style="text-align:left;">
NCBI_Gene:259090,ENSEMBL:ENSMUSG00000110122
</td>
<td style="text-align:left;">
MGI:3030396
</td>
<td style="text-align:left;">
polymorphic pseudogene
</td>
<td style="text-align:left;">
olfactory receptor 562, pseudogene 1
</td>
</tr>
<tr>
<td style="text-align:left;">
7
</td>
<td style="text-align:left;">
pseudogene
</td>
<td style="text-align:right;">
102.79125
</td>
<td style="text-align:right;">
102.79234
</td>
<td style="text-align:left;">
positive
</td>
<td style="text-align:left;">
MGI_C57BL6J_3030397
</td>
<td style="text-align:left;">
Olfr563-ps1
</td>
<td style="text-align:left;">
NCBI_Gene:259157,ENSEMBL:ENSMUSG00000110074
</td>
<td style="text-align:left;">
MGI:3030397
</td>
<td style="text-align:left;">
pseudogene
</td>
<td style="text-align:left;">
olfactory receptor 563, pseudogene 1
</td>
</tr>
<tr>
<td style="text-align:left;">
7
</td>
<td style="text-align:left;">
gene
</td>
<td style="text-align:right;">
102.80348
</td>
<td style="text-align:right;">
102.80443
</td>
<td style="text-align:left;">
positive
</td>
<td style="text-align:left;">
MGI_C57BL6J_3030398
</td>
<td style="text-align:left;">
Olfr564
</td>
<td style="text-align:left;">
NCBI_Gene:258356,ENSEMBL:ENSMUSG00000048469
</td>
<td style="text-align:left;">
MGI:3030398
</td>
<td style="text-align:left;">
protein coding gene
</td>
<td style="text-align:left;">
olfactory receptor 564
</td>
</tr>
<tr>
<td style="text-align:left;">
7
</td>
<td style="text-align:left;">
gene
</td>
<td style="text-align:right;">
102.81331
</td>
<td style="text-align:right;">
102.90171
</td>
<td style="text-align:left;">
positive
</td>
<td style="text-align:left;">
MGI_C57BL6J_3030404
</td>
<td style="text-align:left;">
Olfr570
</td>
<td style="text-align:left;">
NCBI_Gene:259114,ENSEMBL:ENSMUSG00000073964
</td>
<td style="text-align:left;">
MGI:3030404
</td>
<td style="text-align:left;">
protein coding gene
</td>
<td style="text-align:left;">
olfactory receptor 570
</td>
</tr>
<tr>
<td style="text-align:left;">
7
</td>
<td style="text-align:left;">
pseudogene
</td>
<td style="text-align:right;">
102.81755
</td>
<td style="text-align:right;">
102.81849
</td>
<td style="text-align:left;">
negative
</td>
<td style="text-align:left;">
MGI_C57BL6J_3030399
</td>
<td style="text-align:left;">
Olfr565-ps1
</td>
<td style="text-align:left;">
NCBI_Gene:258142,ENSEMBL:ENSMUSG00000109596
</td>
<td style="text-align:left;">
MGI:3030399
</td>
<td style="text-align:left;">
pseudogene
</td>
<td style="text-align:left;">
olfactory receptor 565, pseudogene 1
</td>
</tr>
<tr>
<td style="text-align:left;">
7
</td>
<td style="text-align:left;">
gene
</td>
<td style="text-align:right;">
102.85534
</td>
<td style="text-align:right;">
102.85877
</td>
<td style="text-align:left;">
negative
</td>
<td style="text-align:left;">
MGI_C57BL6J_3030400
</td>
<td style="text-align:left;">
Olfr566
</td>
<td style="text-align:left;">
NCBI_Gene:258168,ENSEMBL:ENSMUSG00000060888
</td>
<td style="text-align:left;">
MGI:3030400
</td>
<td style="text-align:left;">
protein coding gene
</td>
<td style="text-align:left;">
olfactory receptor 566
</td>
</tr>
<tr>
<td style="text-align:left;">
7
</td>
<td style="text-align:left;">
pseudogene
</td>
<td style="text-align:right;">
102.86116
</td>
<td style="text-align:right;">
102.86129
</td>
<td style="text-align:left;">
negative
</td>
<td style="text-align:left;">
MGI_C57BL6J_5791525
</td>
<td style="text-align:left;">
Gm45689
</td>
<td style="text-align:left;">
ENSEMBL:ENSMUSG00000110152
</td>
<td style="text-align:left;">
MGI:5791525
</td>
<td style="text-align:left;">
pseudogene
</td>
<td style="text-align:left;">
predicted gene 45689
</td>
</tr>
<tr>
<td style="text-align:left;">
7
</td>
<td style="text-align:left;">
pseudogene
</td>
<td style="text-align:right;">
102.86392
</td>
<td style="text-align:right;">
102.86478
</td>
<td style="text-align:left;">
negative
</td>
<td style="text-align:left;">
MGI_C57BL6J_3030401
</td>
<td style="text-align:left;">
Olfr567-ps1
</td>
<td style="text-align:left;">
NCBI_Gene:258140,ENSEMBL:ENSMUSG00000109878
</td>
<td style="text-align:left;">
MGI:3030401
</td>
<td style="text-align:left;">
pseudogene
</td>
<td style="text-align:left;">
olfactory receptor 567, pseudogene 1
</td>
</tr>
<tr>
<td style="text-align:left;">
7
</td>
<td style="text-align:left;">
gene
</td>
<td style="text-align:right;">
102.87712
</td>
<td style="text-align:right;">
102.87806
</td>
<td style="text-align:left;">
positive
</td>
<td style="text-align:left;">
MGI_C57BL6J_3030402
</td>
<td style="text-align:left;">
Olfr568
</td>
<td style="text-align:left;">
NCBI_Gene:259095,ENSEMBL:ENSMUSG00000073965
</td>
<td style="text-align:left;">
MGI:3030402
</td>
<td style="text-align:left;">
protein coding gene
</td>
<td style="text-align:left;">
olfactory receptor 568
</td>
</tr>
<tr>
<td style="text-align:left;">
7
</td>
<td style="text-align:left;">
gene
</td>
<td style="text-align:right;">
102.88371
</td>
<td style="text-align:right;">
102.89567
</td>
<td style="text-align:left;">
negative
</td>
<td style="text-align:left;">
MGI_C57BL6J_3030403
</td>
<td style="text-align:left;">
Olfr569
</td>
<td style="text-align:left;">
NCBI_Gene:259092,ENSEMBL:ENSMUSG00000062142
</td>
<td style="text-align:left;">
MGI:3030403
</td>
<td style="text-align:left;">
protein coding gene
</td>
<td style="text-align:left;">
olfactory receptor 569
</td>
</tr>
<tr>
<td style="text-align:left;">
7
</td>
<td style="text-align:left;">
gene
</td>
<td style="text-align:right;">
102.90509
</td>
<td style="text-align:right;">
102.91754
</td>
<td style="text-align:left;">
negative
</td>
<td style="text-align:left;">
MGI_C57BL6J_3030405
</td>
<td style="text-align:left;">
Olfr571
</td>
<td style="text-align:left;">
NCBI_Gene:259089,ENSEMBL:ENSMUSG00000043310
</td>
<td style="text-align:left;">
MGI:3030405
</td>
<td style="text-align:left;">
protein coding gene
</td>
<td style="text-align:left;">
olfactory receptor 571
</td>
</tr>
<tr>
<td style="text-align:left;">
7
</td>
<td style="text-align:left;">
gene
</td>
<td style="text-align:right;">
102.92441
</td>
<td style="text-align:right;">
102.92861
</td>
<td style="text-align:left;">
positive
</td>
<td style="text-align:left;">
MGI_C57BL6J_3030406
</td>
<td style="text-align:left;">
Olfr572
</td>
<td style="text-align:left;">
NCBI_Gene:259093,ENSEMBL:ENSMUSG00000073963
</td>
<td style="text-align:left;">
MGI:3030406
</td>
<td style="text-align:left;">
protein coding gene
</td>
<td style="text-align:left;">
olfactory receptor 572
</td>
</tr>
<tr>
<td style="text-align:left;">
7
</td>
<td style="text-align:left;">
pseudogene
</td>
<td style="text-align:right;">
102.94164
</td>
<td style="text-align:right;">
102.94258
</td>
<td style="text-align:left;">
negative
</td>
<td style="text-align:left;">
MGI_C57BL6J_3030407
</td>
<td style="text-align:left;">
Olfr573-ps1
</td>
<td style="text-align:left;">
NCBI_Gene:258230,ENSEMBL:ENSMUSG00000052785
</td>
<td style="text-align:left;">
MGI:3030407
</td>
<td style="text-align:left;">
pseudogene
</td>
<td style="text-align:left;">
olfactory receptor 573, pseudogene 1
</td>
</tr>
<tr>
<td style="text-align:left;">
7
</td>
<td style="text-align:left;">
gene
</td>
<td style="text-align:right;">
102.94847
</td>
<td style="text-align:right;">
102.94951
</td>
<td style="text-align:left;">
positive
</td>
<td style="text-align:left;">
MGI_C57BL6J_3030408
</td>
<td style="text-align:left;">
Olfr574
</td>
<td style="text-align:left;">
NCBI_Gene:258357,ENSEMBL:ENSMUSG00000045824
</td>
<td style="text-align:left;">
MGI:3030408
</td>
<td style="text-align:left;">
protein coding gene
</td>
<td style="text-align:left;">
olfactory receptor 574
</td>
</tr>
<tr>
<td style="text-align:left;">
7
</td>
<td style="text-align:left;">
gene
</td>
<td style="text-align:right;">
102.95227
</td>
<td style="text-align:right;">
102.95878
</td>
<td style="text-align:left;">
negative
</td>
<td style="text-align:left;">
MGI_C57BL6J_3030409
</td>
<td style="text-align:left;">
Olfr575
</td>
<td style="text-align:left;">
NCBI_Gene:259118,ENSEMBL:ENSMUSG00000066269
</td>
<td style="text-align:left;">
MGI:3030409
</td>
<td style="text-align:left;">
protein coding gene
</td>
<td style="text-align:left;">
olfactory receptor 575
</td>
</tr>
<tr>
<td style="text-align:left;">
7
</td>
<td style="text-align:left;">
gene
</td>
<td style="text-align:right;">
102.96266
</td>
<td style="text-align:right;">
102.97516
</td>
<td style="text-align:left;">
positive
</td>
<td style="text-align:left;">
MGI_C57BL6J_3030410
</td>
<td style="text-align:left;">
Olfr576
</td>
<td style="text-align:left;">
NCBI_Gene:258248,ENSEMBL:ENSMUSG00000073962
</td>
<td style="text-align:left;">
MGI:3030410
</td>
<td style="text-align:left;">
protein coding gene
</td>
<td style="text-align:left;">
olfactory receptor 576
</td>
</tr>
<tr>
<td style="text-align:left;">
7
</td>
<td style="text-align:left;">
gene
</td>
<td style="text-align:right;">
102.97118
</td>
<td style="text-align:right;">
102.97594
</td>
<td style="text-align:left;">
negative
</td>
<td style="text-align:left;">
MGI_C57BL6J_3030411
</td>
<td style="text-align:left;">
Olfr577
</td>
<td style="text-align:left;">
NCBI_Gene:259113,ENSEMBL:ENSMUSG00000043354
</td>
<td style="text-align:left;">
MGI:3030411
</td>
<td style="text-align:left;">
protein coding gene
</td>
<td style="text-align:left;">
olfactory receptor 577
</td>
</tr>
<tr>
<td style="text-align:left;">
7
</td>
<td style="text-align:left;">
gene
</td>
<td style="text-align:right;">
102.98161
</td>
<td style="text-align:right;">
102.98932
</td>
<td style="text-align:left;">
negative
</td>
<td style="text-align:left;">
MGI_C57BL6J_3030412
</td>
<td style="text-align:left;">
Olfr578
</td>
<td style="text-align:left;">
NCBI_Gene:259119,ENSEMBL:ENSMUSG00000045792
</td>
<td style="text-align:left;">
MGI:3030412
</td>
<td style="text-align:left;">
protein coding gene
</td>
<td style="text-align:left;">
olfactory receptor 578
</td>
</tr>
<tr>
<td style="text-align:left;">
7
</td>
<td style="text-align:left;">
pseudogene
</td>
<td style="text-align:right;">
102.99893
</td>
<td style="text-align:right;">
102.99975
</td>
<td style="text-align:left;">
negative
</td>
<td style="text-align:left;">
MGI_C57BL6J_3031361
</td>
<td style="text-align:left;">
Olfr1527-ps1
</td>
<td style="text-align:left;">
NCBI_Gene:259158
</td>
<td style="text-align:left;">
MGI:3031361
</td>
<td style="text-align:left;">
pseudogene
</td>
<td style="text-align:left;">
olfactory receptor 1527, pseudogene 1
</td>
</tr>
<tr>
<td style="text-align:left;">
7
</td>
<td style="text-align:left;">
pseudogene
</td>
<td style="text-align:right;">
103.00461
</td>
<td style="text-align:right;">
103.00520
</td>
<td style="text-align:left;">
negative
</td>
<td style="text-align:left;">
MGI_C57BL6J_3030413
</td>
<td style="text-align:left;">
Olfr579-ps1
</td>
<td style="text-align:left;">
NCBI_Gene:258141
</td>
<td style="text-align:left;">
MGI:3030413
</td>
<td style="text-align:left;">
pseudogene
</td>
<td style="text-align:left;">
olfactory receptor 579, pseudogene 1
</td>
</tr>
<tr>
<td style="text-align:left;">
7
</td>
<td style="text-align:left;">
pseudogene
</td>
<td style="text-align:right;">
103.01314
</td>
<td style="text-align:right;">
103.01449
</td>
<td style="text-align:left;">
negative
</td>
<td style="text-align:left;">
MGI_C57BL6J_3030414
</td>
<td style="text-align:left;">
Olfr580-ps1
</td>
<td style="text-align:left;">
NCBI_Gene:258213,ENSEMBL:ENSMUSG00000110143
</td>
<td style="text-align:left;">
MGI:3030414
</td>
<td style="text-align:left;">
pseudogene
</td>
<td style="text-align:left;">
olfactory receptor 580, pseudogene 1
</td>
</tr>
<tr>
<td style="text-align:left;">
7
</td>
<td style="text-align:left;">
pseudogene
</td>
<td style="text-align:right;">
103.02909
</td>
<td style="text-align:right;">
103.02965
</td>
<td style="text-align:left;">
positive
</td>
<td style="text-align:left;">
MGI_C57BL6J_3030415
</td>
<td style="text-align:left;">
Olfr581-ps1
</td>
<td style="text-align:left;">
NCBI_Gene:404406,ENSEMBL:ENSMUSG00000109845
</td>
<td style="text-align:left;">
MGI:3030415
</td>
<td style="text-align:left;">
pseudogene
</td>
<td style="text-align:left;">
olfactory receptor 581, pseudogene 1
</td>
</tr>
<tr>
<td style="text-align:left;">
7
</td>
<td style="text-align:left;">
gene
</td>
<td style="text-align:right;">
103.03802
</td>
<td style="text-align:right;">
103.04404
</td>
<td style="text-align:left;">
positive
</td>
<td style="text-align:left;">
MGI_C57BL6J_3030416
</td>
<td style="text-align:left;">
Olfr582
</td>
<td style="text-align:left;">
NCBI_Gene:259055,ENSEMBL:ENSMUSG00000073961
</td>
<td style="text-align:left;">
MGI:3030416
</td>
<td style="text-align:left;">
protein coding gene
</td>
<td style="text-align:left;">
olfactory receptor 582
</td>
</tr>
<tr>
<td style="text-align:left;">
7
</td>
<td style="text-align:left;">
gene
</td>
<td style="text-align:right;">
103.05130
</td>
<td style="text-align:right;">
103.05226
</td>
<td style="text-align:left;">
positive
</td>
<td style="text-align:left;">
MGI_C57BL6J_3030417
</td>
<td style="text-align:left;">
Olfr583
</td>
<td style="text-align:left;">
NCBI_Gene:258752,ENSEMBL:ENSMUSG00000073960
</td>
<td style="text-align:left;">
MGI:3030417
</td>
<td style="text-align:left;">
protein coding gene
</td>
<td style="text-align:left;">
olfactory receptor 583
</td>
</tr>
<tr>
<td style="text-align:left;">
7
</td>
<td style="text-align:left;">
gene
</td>
<td style="text-align:right;">
103.08188
</td>
<td style="text-align:right;">
103.08831
</td>
<td style="text-align:left;">
positive
</td>
<td style="text-align:left;">
MGI_C57BL6J_3030418
</td>
<td style="text-align:left;">
Olfr584
</td>
<td style="text-align:left;">
NCBI_Gene:259056,ENSEMBL:ENSMUSG00000073959
</td>
<td style="text-align:left;">
MGI:3030418
</td>
<td style="text-align:left;">
protein coding gene
</td>
<td style="text-align:left;">
olfactory receptor 584
</td>
</tr>
<tr>
<td style="text-align:left;">
7
</td>
<td style="text-align:left;">
gene
</td>
<td style="text-align:right;">
103.09772
</td>
<td style="text-align:right;">
103.09875
</td>
<td style="text-align:left;">
positive
</td>
<td style="text-align:left;">
MGI_C57BL6J_3030419
</td>
<td style="text-align:left;">
Olfr585
</td>
<td style="text-align:left;">
NCBI_Gene:259091,ENSEMBL:ENSMUSG00000078080
</td>
<td style="text-align:left;">
MGI:3030419
</td>
<td style="text-align:left;">
protein coding gene
</td>
<td style="text-align:left;">
olfactory receptor 585
</td>
</tr>
<tr>
<td style="text-align:left;">
7
</td>
<td style="text-align:left;">
gene
</td>
<td style="text-align:right;">
103.11840
</td>
<td style="text-align:right;">
103.12564
</td>
<td style="text-align:left;">
negative
</td>
<td style="text-align:left;">
MGI_C57BL6J_3030420
</td>
<td style="text-align:left;">
Olfr586
</td>
<td style="text-align:left;">
NCBI_Gene:259115,ENSEMBL:ENSMUSG00000066268
</td>
<td style="text-align:left;">
MGI:3030420
</td>
<td style="text-align:left;">
protein coding gene
</td>
<td style="text-align:left;">
olfactory receptor 586
</td>
</tr>
<tr>
<td style="text-align:left;">
7
</td>
<td style="text-align:left;">
pseudogene
</td>
<td style="text-align:right;">
103.13438
</td>
<td style="text-align:right;">
103.13532
</td>
<td style="text-align:left;">
positive
</td>
<td style="text-align:left;">
MGI_C57BL6J_3030421
</td>
<td style="text-align:left;">
Olfr587-ps1
</td>
<td style="text-align:left;">
NCBI_Gene:404407,ENSEMBL:ENSMUSG00000080891
</td>
<td style="text-align:left;">
MGI:3030421
</td>
<td style="text-align:left;">
polymorphic pseudogene
</td>
<td style="text-align:left;">
olfactory receptor 587, pseudogene 1
</td>
</tr>
<tr>
<td style="text-align:left;">
7
</td>
<td style="text-align:left;">
pseudogene
</td>
<td style="text-align:right;">
103.13574
</td>
<td style="text-align:right;">
103.13604
</td>
<td style="text-align:left;">
positive
</td>
<td style="text-align:left;">
MGI_C57BL6J_3705492
</td>
<td style="text-align:left;">
Gm15120
</td>
<td style="text-align:left;">
ENSEMBL:ENSMUSG00000082714
</td>
<td style="text-align:left;">
MGI:3705492
</td>
<td style="text-align:left;">
pseudogene
</td>
<td style="text-align:left;">
predicted gene 15120
</td>
</tr>
<tr>
<td style="text-align:left;">
7
</td>
<td style="text-align:left;">
pseudogene
</td>
<td style="text-align:right;">
103.14088
</td>
<td style="text-align:right;">
103.14786
</td>
<td style="text-align:left;">
positive
</td>
<td style="text-align:left;">
MGI_C57BL6J_3030422
</td>
<td style="text-align:left;">
Olfr588-ps1
</td>
<td style="text-align:left;">
NCBI_Gene:259094,ENSEMBL:ENSMUSG00000084262
</td>
<td style="text-align:left;">
MGI:3030422
</td>
<td style="text-align:left;">
pseudogene
</td>
<td style="text-align:left;">
olfactory receptor 588, pseudogene 1
</td>
</tr>
<tr>
<td style="text-align:left;">
7
</td>
<td style="text-align:left;">
pseudogene
</td>
<td style="text-align:right;">
103.14799
</td>
<td style="text-align:right;">
103.14867
</td>
<td style="text-align:left;">
negative
</td>
<td style="text-align:left;">
MGI_C57BL6J_3705687
</td>
<td style="text-align:left;">
Gm15116
</td>
<td style="text-align:left;">
NCBI_Gene:100417214,ENSEMBL:ENSMUSG00000083254
</td>
<td style="text-align:left;">
MGI:3705687
</td>
<td style="text-align:left;">
pseudogene
</td>
<td style="text-align:left;">
predicted gene 15116
</td>
</tr>
<tr>
<td style="text-align:left;">
7
</td>
<td style="text-align:left;">
gene
</td>
<td style="text-align:right;">
103.15479
</td>
<td style="text-align:right;">
103.15636
</td>
<td style="text-align:left;">
negative
</td>
<td style="text-align:left;">
MGI_C57BL6J_3030423
</td>
<td style="text-align:left;">
Olfr589
</td>
<td style="text-align:left;">
NCBI_Gene:259054,ENSEMBL:ENSMUSG00000051362
</td>
<td style="text-align:left;">
MGI:3030423
</td>
<td style="text-align:left;">
protein coding gene
</td>
<td style="text-align:left;">
olfactory receptor 589
</td>
</tr>
<tr>
<td style="text-align:left;">
7
</td>
<td style="text-align:left;">
pseudogene
</td>
<td style="text-align:right;">
103.16585
</td>
<td style="text-align:right;">
103.16675
</td>
<td style="text-align:left;">
positive
</td>
<td style="text-align:left;">
MGI_C57BL6J_3030424
</td>
<td style="text-align:left;">
Olfr590-ps1
</td>
<td style="text-align:left;">
NCBI_Gene:258170,ENSEMBL:ENSMUSG00000084356
</td>
<td style="text-align:left;">
MGI:3030424
</td>
<td style="text-align:left;">
pseudogene
</td>
<td style="text-align:left;">
olfactory receptor 590, pseudogene 1
</td>
</tr>
<tr>
<td style="text-align:left;">
7
</td>
<td style="text-align:left;">
gene
</td>
<td style="text-align:right;">
103.16707
</td>
<td style="text-align:right;">
103.17834
</td>
<td style="text-align:left;">
negative
</td>
<td style="text-align:left;">
MGI_C57BL6J_3030425
</td>
<td style="text-align:left;">
Olfr591
</td>
<td style="text-align:left;">
NCBI_Gene:258139,ENSEMBL:ENSMUSG00000057461
</td>
<td style="text-align:left;">
MGI:3030425
</td>
<td style="text-align:left;">
protein coding gene
</td>
<td style="text-align:left;">
olfactory receptor 591
</td>
</tr>
<tr>
<td style="text-align:left;">
7
</td>
<td style="text-align:left;">
gene
</td>
<td style="text-align:right;">
103.18480
</td>
<td style="text-align:right;">
103.19297
</td>
<td style="text-align:left;">
positive
</td>
<td style="text-align:left;">
MGI_C57BL6J_3030426
</td>
<td style="text-align:left;">
Olfr592
</td>
<td style="text-align:left;">
NCBI_Gene:404317,ENSEMBL:ENSMUSG00000073956
</td>
<td style="text-align:left;">
MGI:3030426
</td>
<td style="text-align:left;">
protein coding gene
</td>
<td style="text-align:left;">
olfactory receptor 592
</td>
</tr>
<tr>
<td style="text-align:left;">
7
</td>
<td style="text-align:left;">
gene
</td>
<td style="text-align:right;">
103.21186
</td>
<td style="text-align:right;">
103.21285
</td>
<td style="text-align:left;">
positive
</td>
<td style="text-align:left;">
MGI_C57BL6J_3030427
</td>
<td style="text-align:left;">
Olfr593
</td>
<td style="text-align:left;">
NCBI_Gene:258378,ENSEMBL:ENSMUSG00000073955
</td>
<td style="text-align:left;">
MGI:3030427
</td>
<td style="text-align:left;">
protein coding gene
</td>
<td style="text-align:left;">
olfactory receptor 593
</td>
</tr>
<tr>
<td style="text-align:left;">
7
</td>
<td style="text-align:left;">
gene
</td>
<td style="text-align:right;">
103.21700
</td>
<td style="text-align:right;">
103.22231
</td>
<td style="text-align:left;">
positive
</td>
<td style="text-align:left;">
MGI_C57BL6J_3030428
</td>
<td style="text-align:left;">
Olfr594
</td>
<td style="text-align:left;">
NCBI_Gene:258246,ENSEMBL:ENSMUSG00000073954
</td>
<td style="text-align:left;">
MGI:3030428
</td>
<td style="text-align:left;">
protein coding gene
</td>
<td style="text-align:left;">
olfactory receptor 594
</td>
</tr>
<tr>
<td style="text-align:left;">
7
</td>
<td style="text-align:left;">
pseudogene
</td>
<td style="text-align:right;">
103.23563
</td>
<td style="text-align:right;">
103.23659
</td>
<td style="text-align:left;">
positive
</td>
<td style="text-align:left;">
MGI_C57BL6J_3030429
</td>
<td style="text-align:left;">
Olfr595-ps1
</td>
<td style="text-align:left;">
NCBI_Gene:258175,ENSEMBL:ENSMUSG00000083383
</td>
<td style="text-align:left;">
MGI:3030429
</td>
<td style="text-align:left;">
pseudogene
</td>
<td style="text-align:left;">
olfactory receptor 595, pseudogene 1
</td>
</tr>
<tr>
<td style="text-align:left;">
7
</td>
<td style="text-align:left;">
gene
</td>
<td style="text-align:right;">
103.24974
</td>
<td style="text-align:right;">
103.25250
</td>
<td style="text-align:left;">
negative
</td>
<td style="text-align:left;">
MGI_C57BL6J_3051372
</td>
<td style="text-align:left;">
Usp17ld
</td>
<td style="text-align:left;">
NCBI_Gene:384701,ENSEMBL:ENSMUSG00000057321
</td>
<td style="text-align:left;">
MGI:3051372
</td>
<td style="text-align:left;">
protein coding gene
</td>
<td style="text-align:left;">
ubiquitin specific peptidase 17-like D
</td>
</tr>
<tr>
<td style="text-align:left;">
7
</td>
<td style="text-align:left;">
gene
</td>
<td style="text-align:right;">
103.26088
</td>
<td style="text-align:right;">
103.26100
</td>
<td style="text-align:left;">
negative
</td>
<td style="text-align:left;">
MGI_C57BL6J_5454948
</td>
<td style="text-align:left;">
Gm25171
</td>
<td style="text-align:left;">
ENSEMBL:ENSMUSG00000095000
</td>
<td style="text-align:left;">
MGI:5454948
</td>
<td style="text-align:left;">
miRNA gene
</td>
<td style="text-align:left;">
predicted gene, 25171
</td>
</tr>
<tr>
<td style="text-align:left;">
7
</td>
<td style="text-align:left;">
gene
</td>
<td style="text-align:right;">
103.26712
</td>
<td style="text-align:right;">
103.26723
</td>
<td style="text-align:left;">
negative
</td>
<td style="text-align:left;">
MGI_C57BL6J_5453327
</td>
<td style="text-align:left;">
Gm23550
</td>
<td style="text-align:left;">
ENSEMBL:ENSMUSG00000094572
</td>
<td style="text-align:left;">
MGI:5453327
</td>
<td style="text-align:left;">
miRNA gene
</td>
<td style="text-align:left;">
predicted gene, 23550
</td>
</tr>
<tr>
<td style="text-align:left;">
7
</td>
<td style="text-align:left;">
gene
</td>
<td style="text-align:right;">
103.27155
</td>
<td style="text-align:right;">
103.27187
</td>
<td style="text-align:left;">
positive
</td>
<td style="text-align:left;">
MGI_C57BL6J_5453750
</td>
<td style="text-align:left;">
Gm23973
</td>
<td style="text-align:left;">
ENSEMBL:ENSMUSG00000065254
</td>
<td style="text-align:left;">
MGI:5453750
</td>
<td style="text-align:left;">
unclassified non-coding RNA gene
</td>
<td style="text-align:left;">
predicted gene, 23973
</td>
</tr>
<tr>
<td style="text-align:left;">
7
</td>
<td style="text-align:left;">
pseudogene
</td>
<td style="text-align:right;">
103.29700
</td>
<td style="text-align:right;">
103.30042
</td>
<td style="text-align:left;">
positive
</td>
<td style="text-align:left;">
MGI_C57BL6J_5825651
</td>
<td style="text-align:left;">
Gm46014
</td>
<td style="text-align:left;">
NCBI_Gene:108167486
</td>
<td style="text-align:left;">
MGI:5825651
</td>
<td style="text-align:left;">
pseudogene
</td>
<td style="text-align:left;">
predicted gene, 46014
</td>
</tr>
<tr>
<td style="text-align:left;">
7
</td>
<td style="text-align:left;">
gene
</td>
<td style="text-align:right;">
103.30565
</td>
<td style="text-align:right;">
103.31278
</td>
<td style="text-align:left;">
positive
</td>
<td style="text-align:left;">
MGI_C57BL6J_3030430
</td>
<td style="text-align:left;">
Olfr596
</td>
<td style="text-align:left;">
NCBI_Gene:100041187,ENSEMBL:ENSMUSG00000073953
</td>
<td style="text-align:left;">
MGI:3030430
</td>
<td style="text-align:left;">
protein coding gene
</td>
<td style="text-align:left;">
olfactory receptor 596
</td>
</tr>
<tr>
<td style="text-align:left;">
7
</td>
<td style="text-align:left;">
gene
</td>
<td style="text-align:right;">
103.31632
</td>
<td style="text-align:right;">
103.32280
</td>
<td style="text-align:left;">
positive
</td>
<td style="text-align:left;">
MGI_C57BL6J_3030431
</td>
<td style="text-align:left;">
Olfr597
</td>
<td style="text-align:left;">
NCBI_Gene:258135,ENSEMBL:ENSMUSG00000073952
</td>
<td style="text-align:left;">
MGI:3030431
</td>
<td style="text-align:left;">
protein coding gene
</td>
<td style="text-align:left;">
olfactory receptor 597
</td>
</tr>
<tr>
<td style="text-align:left;">
7
</td>
<td style="text-align:left;">
gene
</td>
<td style="text-align:right;">
103.32357
</td>
<td style="text-align:right;">
103.32989
</td>
<td style="text-align:left;">
positive
</td>
<td style="text-align:left;">
MGI_C57BL6J_3030432
</td>
<td style="text-align:left;">
Olfr598
</td>
<td style="text-align:left;">
NCBI_Gene:257975,ENSEMBL:ENSMUSG00000073951
</td>
<td style="text-align:left;">
MGI:3030432
</td>
<td style="text-align:left;">
protein coding gene
</td>
<td style="text-align:left;">
olfactory receptor 598
</td>
</tr>
<tr>
<td style="text-align:left;">
7
</td>
<td style="text-align:left;">
gene
</td>
<td style="text-align:right;">
103.33592
</td>
<td style="text-align:right;">
103.33956
</td>
<td style="text-align:left;">
positive
</td>
<td style="text-align:left;">
MGI_C57BL6J_3030433
</td>
<td style="text-align:left;">
Olfr599
</td>
<td style="text-align:left;">
NCBI_Gene:258726,ENSEMBL:ENSMUSG00000073950
</td>
<td style="text-align:left;">
MGI:3030433
</td>
<td style="text-align:left;">
protein coding gene
</td>
<td style="text-align:left;">
olfactory receptor 599
</td>
</tr>
<tr>
<td style="text-align:left;">
7
</td>
<td style="text-align:left;">
gene
</td>
<td style="text-align:right;">
103.34348
</td>
<td style="text-align:right;">
103.34967
</td>
<td style="text-align:left;">
negative
</td>
<td style="text-align:left;">
MGI_C57BL6J_3030434
</td>
<td style="text-align:left;">
Olfr600
</td>
<td style="text-align:left;">
NCBI_Gene:259048,ENSEMBL:ENSMUSG00000045540
</td>
<td style="text-align:left;">
MGI:3030434
</td>
<td style="text-align:left;">
protein coding gene
</td>
<td style="text-align:left;">
olfactory receptor 600
</td>
</tr>
<tr>
<td style="text-align:left;">
7
</td>
<td style="text-align:left;">
gene
</td>
<td style="text-align:right;">
103.35709
</td>
<td style="text-align:right;">
103.36266
</td>
<td style="text-align:left;">
negative
</td>
<td style="text-align:left;">
MGI_C57BL6J_3030435
</td>
<td style="text-align:left;">
Olfr601
</td>
<td style="text-align:left;">
NCBI_Gene:258311,ENSEMBL:ENSMUSG00000109951
</td>
<td style="text-align:left;">
MGI:3030435
</td>
<td style="text-align:left;">
protein coding gene
</td>
<td style="text-align:left;">
olfactory receptor 601
</td>
</tr>
<tr>
<td style="text-align:left;">
7
</td>
<td style="text-align:left;">
pseudogene
</td>
<td style="text-align:right;">
103.36752
</td>
<td style="text-align:right;">
103.37244
</td>
<td style="text-align:left;">
negative
</td>
<td style="text-align:left;">
MGI_C57BL6J_3030436
</td>
<td style="text-align:left;">
Olfr602-ps1
</td>
<td style="text-align:left;">
NCBI_Gene:257976,ENSEMBL:ENSMUSG00000082739
</td>
<td style="text-align:left;">
MGI:3030436
</td>
<td style="text-align:left;">
pseudogene
</td>
<td style="text-align:left;">
olfactory receptor 602, pseudogene 1
</td>
</tr>
<tr>
<td style="text-align:left;">
7
</td>
<td style="text-align:left;">
gene
</td>
<td style="text-align:right;">
103.38306
</td>
<td style="text-align:right;">
103.38618
</td>
<td style="text-align:left;">
negative
</td>
<td style="text-align:left;">
MGI_C57BL6J_3030437
</td>
<td style="text-align:left;">
Olfr603
</td>
<td style="text-align:left;">
NCBI_Gene:259073,ENSEMBL:ENSMUSG00000059874
</td>
<td style="text-align:left;">
MGI:3030437
</td>
<td style="text-align:left;">
protein coding gene
</td>
<td style="text-align:left;">
olfactory receptor 603
</td>
</tr>
<tr>
<td style="text-align:left;">
7
</td>
<td style="text-align:left;">
gene
</td>
<td style="text-align:right;">
103.41507
</td>
<td style="text-align:right;">
103.41988
</td>
<td style="text-align:left;">
positive
</td>
<td style="text-align:left;">
MGI_C57BL6J_107698
</td>
<td style="text-align:left;">
Usp17lc
</td>
<td style="text-align:left;">
NCBI_Gene:13532,ENSEMBL:ENSMUSG00000058976
</td>
<td style="text-align:left;">
MGI:107698
</td>
<td style="text-align:left;">
protein coding gene
</td>
<td style="text-align:left;">
ubiquitin specific peptidase 17-like C
</td>
</tr>
<tr>
<td style="text-align:left;">
7
</td>
<td style="text-align:left;">
gene
</td>
<td style="text-align:right;">
103.41581
</td>
<td style="text-align:right;">
103.41593
</td>
<td style="text-align:left;">
positive
</td>
<td style="text-align:left;">
MGI_C57BL6J_5455551
</td>
<td style="text-align:left;">
Gm25774
</td>
<td style="text-align:left;">
ENSEMBL:ENSMUSG00000080367
</td>
<td style="text-align:left;">
MGI:5455551
</td>
<td style="text-align:left;">
miRNA gene
</td>
<td style="text-align:left;">
predicted gene, 25774
</td>
</tr>
<tr>
<td style="text-align:left;">
7
</td>
<td style="text-align:left;">
gene
</td>
<td style="text-align:right;">
103.44041
</td>
<td style="text-align:right;">
103.44544
</td>
<td style="text-align:left;">
negative
</td>
<td style="text-align:left;">
MGI_C57BL6J_3030439
</td>
<td style="text-align:left;">
Olfr605
</td>
<td style="text-align:left;">
NCBI_Gene:258156,ENSEMBL:ENSMUSG00000109659
</td>
<td style="text-align:left;">
MGI:3030439
</td>
<td style="text-align:left;">
protein coding gene
</td>
<td style="text-align:left;">
olfactory receptor 605
</td>
</tr>
<tr>
<td style="text-align:left;">
7
</td>
<td style="text-align:left;">
gene
</td>
<td style="text-align:right;">
103.44905
</td>
<td style="text-align:right;">
103.45353
</td>
<td style="text-align:left;">
positive
</td>
<td style="text-align:left;">
MGI_C57BL6J_3030440
</td>
<td style="text-align:left;">
Olfr606
</td>
<td style="text-align:left;">
NCBI_Gene:259098,ENSEMBL:ENSMUSG00000073949
</td>
<td style="text-align:left;">
MGI:3030440
</td>
<td style="text-align:left;">
protein coding gene
</td>
<td style="text-align:left;">
olfactory receptor 606
</td>
</tr>
<tr>
<td style="text-align:left;">
7
</td>
<td style="text-align:left;">
gene
</td>
<td style="text-align:right;">
103.45553
</td>
<td style="text-align:right;">
103.46415
</td>
<td style="text-align:left;">
negative
</td>
<td style="text-align:left;">
MGI_C57BL6J_3030441
</td>
<td style="text-align:left;">
Olfr607
</td>
<td style="text-align:left;">
NCBI_Gene:546989,ENSEMBL:ENSMUSG00000081945
</td>
<td style="text-align:left;">
MGI:3030441
</td>
<td style="text-align:left;">
protein coding gene
</td>
<td style="text-align:left;">
olfactory receptor 607
</td>
</tr>
<tr>
<td style="text-align:left;">
7
</td>
<td style="text-align:left;">
gene
</td>
<td style="text-align:right;">
103.46988
</td>
<td style="text-align:right;">
103.47136
</td>
<td style="text-align:left;">
positive
</td>
<td style="text-align:left;">
MGI_C57BL6J_3030442
</td>
<td style="text-align:left;">
Olfr608
</td>
<td style="text-align:left;">
NCBI_Gene:258751,ENSEMBL:ENSMUSG00000073948
</td>
<td style="text-align:left;">
MGI:3030442
</td>
<td style="text-align:left;">
protein coding gene
</td>
<td style="text-align:left;">
olfactory receptor 608
</td>
</tr>
<tr>
<td style="text-align:left;">
7
</td>
<td style="text-align:left;">
gene
</td>
<td style="text-align:right;">
103.49003
</td>
<td style="text-align:right;">
103.49806
</td>
<td style="text-align:left;">
negative
</td>
<td style="text-align:left;">
MGI_C57BL6J_3030443
</td>
<td style="text-align:left;">
Olfr609
</td>
<td style="text-align:left;">
NCBI_Gene:259086,ENSEMBL:ENSMUSG00000046396
</td>
<td style="text-align:left;">
MGI:3030443
</td>
<td style="text-align:left;">
protein coding gene
</td>
<td style="text-align:left;">
olfactory receptor 609
</td>
</tr>
<tr>
<td style="text-align:left;">
7
</td>
<td style="text-align:left;">
gene
</td>
<td style="text-align:right;">
103.50538
</td>
<td style="text-align:right;">
103.51212
</td>
<td style="text-align:left;">
negative
</td>
<td style="text-align:left;">
MGI_C57BL6J_3030444
</td>
<td style="text-align:left;">
Olfr610
</td>
<td style="text-align:left;">
NCBI_Gene:259085,ENSEMBL:ENSMUSG00000045584
</td>
<td style="text-align:left;">
MGI:3030444
</td>
<td style="text-align:left;">
protein coding gene
</td>
<td style="text-align:left;">
olfactory receptor 610
</td>
</tr>
<tr>
<td style="text-align:left;">
7
</td>
<td style="text-align:left;">
gene
</td>
<td style="text-align:right;">
103.51646
</td>
<td style="text-align:right;">
103.52085
</td>
<td style="text-align:left;">
negative
</td>
<td style="text-align:left;">
MGI_C57BL6J_3030445
</td>
<td style="text-align:left;">
Olfr611
</td>
<td style="text-align:left;">
NCBI_Gene:258722,ENSEMBL:ENSMUSG00000096584
</td>
<td style="text-align:left;">
MGI:3030445
</td>
<td style="text-align:left;">
protein coding gene
</td>
<td style="text-align:left;">
olfactory receptor 611
</td>
</tr>
<tr>
<td style="text-align:left;">
7
</td>
<td style="text-align:left;">
gene
</td>
<td style="text-align:right;">
103.53722
</td>
<td style="text-align:right;">
103.54272
</td>
<td style="text-align:left;">
negative
</td>
<td style="text-align:left;">
MGI_C57BL6J_3030446
</td>
<td style="text-align:left;">
Olfr612
</td>
<td style="text-align:left;">
NCBI_Gene:545985,ENSEMBL:ENSMUSG00000094119
</td>
<td style="text-align:left;">
MGI:3030446
</td>
<td style="text-align:left;">
protein coding gene
</td>
<td style="text-align:left;">
olfactory receptor 612
</td>
</tr>
<tr>
<td style="text-align:left;">
7
</td>
<td style="text-align:left;">
gene
</td>
<td style="text-align:right;">
103.54807
</td>
<td style="text-align:right;">
103.55551
</td>
<td style="text-align:left;">
positive
</td>
<td style="text-align:left;">
MGI_C57BL6J_3030447
</td>
<td style="text-align:left;">
Olfr613
</td>
<td style="text-align:left;">
NCBI_Gene:259104,ENSEMBL:ENSMUSG00000078624
</td>
<td style="text-align:left;">
MGI:3030447
</td>
<td style="text-align:left;">
protein coding gene
</td>
<td style="text-align:left;">
olfactory receptor 613
</td>
</tr>
<tr>
<td style="text-align:left;">
7
</td>
<td style="text-align:left;">
gene
</td>
<td style="text-align:right;">
103.55608
</td>
<td style="text-align:right;">
103.56380
</td>
<td style="text-align:left;">
positive
</td>
<td style="text-align:left;">
MGI_C57BL6J_3030449
</td>
<td style="text-align:left;">
Olfr615
</td>
<td style="text-align:left;">
NCBI_Gene:259084,ENSEMBL:ENSMUSG00000073947
</td>
<td style="text-align:left;">
MGI:3030449
</td>
<td style="text-align:left;">
protein coding gene
</td>
<td style="text-align:left;">
olfactory receptor 615
</td>
</tr>
<tr>
<td style="text-align:left;">
7
</td>
<td style="text-align:left;">
gene
</td>
<td style="text-align:right;">
103.56375
</td>
<td style="text-align:right;">
103.56914
</td>
<td style="text-align:left;">
negative
</td>
<td style="text-align:left;">
MGI_C57BL6J_3030450
</td>
<td style="text-align:left;">
Olfr616
</td>
<td style="text-align:left;">
NCBI_Gene:259103,ENSEMBL:ENSMUSG00000047544
</td>
<td style="text-align:left;">
MGI:3030450
</td>
<td style="text-align:left;">
protein coding gene
</td>
<td style="text-align:left;">
olfactory receptor 616
</td>
</tr>
<tr>
<td style="text-align:left;">
7
</td>
<td style="text-align:left;">
gene
</td>
<td style="text-align:right;">
103.57306
</td>
<td style="text-align:right;">
103.58679
</td>
<td style="text-align:left;">
positive
</td>
<td style="text-align:left;">
MGI_C57BL6J_3030451
</td>
<td style="text-align:left;">
Olfr617
</td>
<td style="text-align:left;">
NCBI_Gene:258838,ENSEMBL:ENSMUSG00000073946
</td>
<td style="text-align:left;">
MGI:3030451
</td>
<td style="text-align:left;">
protein coding gene
</td>
<td style="text-align:left;">
olfactory receptor 617
</td>
</tr>
<tr>
<td style="text-align:left;">
7
</td>
<td style="text-align:left;">
gene
</td>
<td style="text-align:right;">
103.58127
</td>
<td style="text-align:right;">
103.58138
</td>
<td style="text-align:left;">
positive
</td>
<td style="text-align:left;">
MGI_C57BL6J_5456283
</td>
<td style="text-align:left;">
Gm26506
</td>
<td style="text-align:left;">
ENSEMBL:ENSMUSG00000096906
</td>
<td style="text-align:left;">
MGI:5456283
</td>
<td style="text-align:left;">
miRNA gene
</td>
<td style="text-align:left;">
predicted gene, 26506
</td>
</tr>
<tr>
<td style="text-align:left;">
7
</td>
<td style="text-align:left;">
gene
</td>
<td style="text-align:right;">
103.59010
</td>
<td style="text-align:right;">
103.60045
</td>
<td style="text-align:left;">
positive
</td>
<td style="text-align:left;">
MGI_C57BL6J_3030452
</td>
<td style="text-align:left;">
Olfr618
</td>
<td style="text-align:left;">
NCBI_Gene:259049,ENSEMBL:ENSMUSG00000073945
</td>
<td style="text-align:left;">
MGI:3030452
</td>
<td style="text-align:left;">
protein coding gene
</td>
<td style="text-align:left;">
olfactory receptor 618
</td>
</tr>
<tr>
<td style="text-align:left;">
7
</td>
<td style="text-align:left;">
gene
</td>
<td style="text-align:right;">
103.60058
</td>
<td style="text-align:right;">
103.60486
</td>
<td style="text-align:left;">
positive
</td>
<td style="text-align:left;">
MGI_C57BL6J_3030453
</td>
<td style="text-align:left;">
Olfr619
</td>
<td style="text-align:left;">
NCBI_Gene:259080,ENSEMBL:ENSMUSG00000073944
</td>
<td style="text-align:left;">
MGI:3030453
</td>
<td style="text-align:left;">
protein coding gene
</td>
<td style="text-align:left;">
olfactory receptor 619
</td>
</tr>
<tr>
<td style="text-align:left;">
7
</td>
<td style="text-align:left;">
gene
</td>
<td style="text-align:right;">
103.60489
</td>
<td style="text-align:right;">
103.61253
</td>
<td style="text-align:left;">
negative
</td>
<td style="text-align:left;">
MGI_C57BL6J_3030454
</td>
<td style="text-align:left;">
Olfr620
</td>
<td style="text-align:left;">
NCBI_Gene:258808,ENSEMBL:ENSMUSG00000045132
</td>
<td style="text-align:left;">
MGI:3030454
</td>
<td style="text-align:left;">
protein coding gene
</td>
<td style="text-align:left;">
olfactory receptor 620
</td>
</tr>
<tr>
<td style="text-align:left;">
7
</td>
<td style="text-align:left;">
pseudogene
</td>
<td style="text-align:right;">
103.62902
</td>
<td style="text-align:right;">
103.62996
</td>
<td style="text-align:left;">
negative
</td>
<td style="text-align:left;">
MGI_C57BL6J_3030455
</td>
<td style="text-align:left;">
Olfr621-ps1
</td>
<td style="text-align:left;">
NCBI_Gene:667639,ENSEMBL:ENSMUSG00000081315
</td>
<td style="text-align:left;">
MGI:3030455
</td>
<td style="text-align:left;">
pseudogene
</td>
<td style="text-align:left;">
olfactory receptor 621, pseudogene 1
</td>
</tr>
<tr>
<td style="text-align:left;">
7
</td>
<td style="text-align:left;">
gene
</td>
<td style="text-align:right;">
103.63845
</td>
<td style="text-align:right;">
103.64270
</td>
<td style="text-align:left;">
negative
</td>
<td style="text-align:left;">
MGI_C57BL6J_3030456
</td>
<td style="text-align:left;">
Olfr622
</td>
<td style="text-align:left;">
NCBI_Gene:259087,ENSEMBL:ENSMUSG00000050085
</td>
<td style="text-align:left;">
MGI:3030456
</td>
<td style="text-align:left;">
protein coding gene
</td>
<td style="text-align:left;">
olfactory receptor 622
</td>
</tr>
<tr>
<td style="text-align:left;">
7
</td>
<td style="text-align:left;">
gene
</td>
<td style="text-align:right;">
103.65770
</td>
<td style="text-align:right;">
103.66370
</td>
<td style="text-align:left;">
negative
</td>
<td style="text-align:left;">
MGI_C57BL6J_3030457
</td>
<td style="text-align:left;">
Olfr623
</td>
<td style="text-align:left;">
NCBI_Gene:259126,ENSEMBL:ENSMUSG00000099687
</td>
<td style="text-align:left;">
MGI:3030457
</td>
<td style="text-align:left;">
protein coding gene
</td>
<td style="text-align:left;">
olfactory receptor 623
</td>
</tr>
<tr>
<td style="text-align:left;">
7
</td>
<td style="text-align:left;">
gene
</td>
<td style="text-align:right;">
103.66546
</td>
<td style="text-align:right;">
103.67119
</td>
<td style="text-align:left;">
negative
</td>
<td style="text-align:left;">
MGI_C57BL6J_3030458
</td>
<td style="text-align:left;">
Olfr624
</td>
<td style="text-align:left;">
NCBI_Gene:258189,ENSEMBL:ENSMUSG00000045780
</td>
<td style="text-align:left;">
MGI:3030458
</td>
<td style="text-align:left;">
protein coding gene
</td>
<td style="text-align:left;">
olfactory receptor 624
</td>
</tr>
<tr>
<td style="text-align:left;">
7
</td>
<td style="text-align:left;">
pseudogene
</td>
<td style="text-align:right;">
103.68272
</td>
<td style="text-align:right;">
103.68368
</td>
<td style="text-align:left;">
positive
</td>
<td style="text-align:left;">
MGI_C57BL6J_3030459
</td>
<td style="text-align:left;">
Olfr625-ps1
</td>
<td style="text-align:left;">
NCBI_Gene:258169,ENSEMBL:ENSMUSG00000073943
</td>
<td style="text-align:left;">
MGI:3030459
</td>
<td style="text-align:left;">
pseudogene
</td>
<td style="text-align:left;">
olfactory receptor 625, pseudogene 1
</td>
</tr>
<tr>
<td style="text-align:left;">
7
</td>
<td style="text-align:left;">
pseudogene
</td>
<td style="text-align:right;">
103.69157
</td>
<td style="text-align:right;">
103.69212
</td>
<td style="text-align:left;">
negative
</td>
<td style="text-align:left;">
MGI_C57BL6J_3030460
</td>
<td style="text-align:left;">
Olfr626-ps1
</td>
<td style="text-align:left;">
NCBI_Gene:404366,ENSEMBL:ENSMUSG00000081038
</td>
<td style="text-align:left;">
MGI:3030460
</td>
<td style="text-align:left;">
pseudogene
</td>
<td style="text-align:left;">
olfactory receptor 626, pseudogene 1
</td>
</tr>
<tr>
<td style="text-align:left;">
7
</td>
<td style="text-align:left;">
pseudogene
</td>
<td style="text-align:right;">
103.69958
</td>
<td style="text-align:right;">
103.70075
</td>
<td style="text-align:left;">
negative
</td>
<td style="text-align:left;">
MGI_C57BL6J_3030094
</td>
<td style="text-align:left;">
Olfr260-ps1
</td>
<td style="text-align:left;">
NCBI_Gene:404365,ENSEMBL:ENSMUSG00000082077
</td>
<td style="text-align:left;">
MGI:3030094
</td>
<td style="text-align:left;">
pseudogene
</td>
<td style="text-align:left;">
olfactory receptor 260, pseudogene 1
</td>
</tr>
<tr>
<td style="text-align:left;">
7
</td>
<td style="text-align:left;">
gene
</td>
<td style="text-align:right;">
103.71467
</td>
<td style="text-align:right;">
103.72117
</td>
<td style="text-align:left;">
positive
</td>
<td style="text-align:left;">
MGI_C57BL6J_3030077
</td>
<td style="text-align:left;">
Olfr243
</td>
<td style="text-align:left;">
NCBI_Gene:436002,ENSEMBL:ENSMUSG00000094822
</td>
<td style="text-align:left;">
MGI:3030077
</td>
<td style="text-align:left;">
protein coding gene
</td>
<td style="text-align:left;">
olfactory receptor 243
</td>
</tr>
<tr>
<td style="text-align:left;">
7
</td>
<td style="text-align:left;">
gene
</td>
<td style="text-align:right;">
103.73037
</td>
<td style="text-align:right;">
103.73332
</td>
<td style="text-align:left;">
positive
</td>
<td style="text-align:left;">
MGI_C57BL6J_3030462
</td>
<td style="text-align:left;">
Olfr628
</td>
<td style="text-align:left;">
NCBI_Gene:259159,ENSEMBL:ENSMUSG00000096516
</td>
<td style="text-align:left;">
MGI:3030462
</td>
<td style="text-align:left;">
protein coding gene
</td>
<td style="text-align:left;">
olfactory receptor 628
</td>
</tr>
<tr>
<td style="text-align:left;">
7
</td>
<td style="text-align:left;">
gene
</td>
<td style="text-align:right;">
103.73977
</td>
<td style="text-align:right;">
103.74494
</td>
<td style="text-align:left;">
negative
</td>
<td style="text-align:left;">
MGI_C57BL6J_3030463
</td>
<td style="text-align:left;">
Olfr629
</td>
<td style="text-align:left;">
NCBI_Gene:258818,ENSEMBL:ENSMUSG00000047545
</td>
<td style="text-align:left;">
MGI:3030463
</td>
<td style="text-align:left;">
protein coding gene
</td>
<td style="text-align:left;">
olfactory receptor 629
</td>
</tr>
<tr>
<td style="text-align:left;">
7
</td>
<td style="text-align:left;">
gene
</td>
<td style="text-align:right;">
103.75348
</td>
<td style="text-align:right;">
103.75735
</td>
<td style="text-align:left;">
negative
</td>
<td style="text-align:left;">
MGI_C57BL6J_3030464
</td>
<td style="text-align:left;">
Olfr630
</td>
<td style="text-align:left;">
NCBI_Gene:259102,ENSEMBL:ENSMUSG00000050281
</td>
<td style="text-align:left;">
MGI:3030464
</td>
<td style="text-align:left;">
protein coding gene
</td>
<td style="text-align:left;">
olfactory receptor 630
</td>
</tr>
<tr>
<td style="text-align:left;">
7
</td>
<td style="text-align:left;">
gene
</td>
<td style="text-align:right;">
103.76581
</td>
<td style="text-align:right;">
103.77159
</td>
<td style="text-align:left;">
negative
</td>
<td style="text-align:left;">
MGI_C57BL6J_1341789
</td>
<td style="text-align:left;">
Olfr69
</td>
<td style="text-align:left;">
NCBI_Gene:18370,ENSEMBL:ENSMUSG00000058662
</td>
<td style="text-align:left;">
MGI:1341789
</td>
<td style="text-align:left;">
protein coding gene
</td>
<td style="text-align:left;">
olfactory receptor 69
</td>
</tr>
<tr>
<td style="text-align:left;">
7
</td>
<td style="text-align:left;">
gene
</td>
<td style="text-align:right;">
103.77436
</td>
<td style="text-align:right;">
103.77970
</td>
<td style="text-align:left;">
negative
</td>
<td style="text-align:left;">
MGI_C57BL6J_1341790
</td>
<td style="text-align:left;">
Olfr68
</td>
<td style="text-align:left;">
NCBI_Gene:18369,ENSEMBL:ENSMUSG00000061626
</td>
<td style="text-align:left;">
MGI:1341790
</td>
<td style="text-align:left;">
protein coding gene
</td>
<td style="text-align:left;">
olfactory receptor 68
</td>
</tr>
<tr>
<td style="text-align:left;">
7
</td>
<td style="text-align:left;">
gene
</td>
<td style="text-align:right;">
103.78459
</td>
<td style="text-align:right;">
103.79183
</td>
<td style="text-align:left;">
negative
</td>
<td style="text-align:left;">
MGI_C57BL6J_1341911
</td>
<td style="text-align:left;">
Olfr67
</td>
<td style="text-align:left;">
NCBI_Gene:18368,ENSEMBL:ENSMUSG00000047535
</td>
<td style="text-align:left;">
MGI:1341911
</td>
<td style="text-align:left;">
protein coding gene
</td>
<td style="text-align:left;">
olfactory receptor 67
</td>
</tr>
<tr>
<td style="text-align:left;">
7
</td>
<td style="text-align:left;">
gene
</td>
<td style="text-align:right;">
103.81252
</td>
<td style="text-align:right;">
103.81400
</td>
<td style="text-align:left;">
negative
</td>
<td style="text-align:left;">
MGI_C57BL6J_5474850
</td>
<td style="text-align:left;">
Hbb-bt
</td>
<td style="text-align:left;">
NCBI_Gene:101488143,ENSEMBL:ENSMUSG00000073940
</td>
<td style="text-align:left;">
MGI:5474850
</td>
<td style="text-align:left;">
protein coding gene
</td>
<td style="text-align:left;">
hemoglobin, beta adult t chain
</td>
</tr>
<tr>
<td style="text-align:left;">
7
</td>
<td style="text-align:left;">
gene
</td>
<td style="text-align:right;">
103.82652
</td>
<td style="text-align:right;">
103.82810
</td>
<td style="text-align:left;">
negative
</td>
<td style="text-align:left;">
MGI_C57BL6J_5474852
</td>
<td style="text-align:left;">
Hbb-bs
</td>
<td style="text-align:left;">
NCBI_Gene:100503605,ENSEMBL:ENSMUSG00000052305
</td>
<td style="text-align:left;">
MGI:5474852
</td>
<td style="text-align:left;">
protein coding gene
</td>
<td style="text-align:left;">
hemoglobin, beta adult s chain
</td>
</tr>
<tr>
<td style="text-align:left;">
7
</td>
<td style="text-align:left;">
pseudogene
</td>
<td style="text-align:right;">
103.82959
</td>
<td style="text-align:right;">
103.82996
</td>
<td style="text-align:left;">
positive
</td>
<td style="text-align:left;">
MGI_C57BL6J_3642219
</td>
<td style="text-align:left;">
Gm15115
</td>
<td style="text-align:left;">
NCBI_Gene:108167487,ENSEMBL:ENSMUSG00000083967
</td>
<td style="text-align:left;">
MGI:3642219
</td>
<td style="text-align:left;">
pseudogene
</td>
<td style="text-align:left;">
predicted gene 15115
</td>
</tr>
<tr>
<td style="text-align:left;">
7
</td>
<td style="text-align:left;">
pseudogene
</td>
<td style="text-align:right;">
103.83263
</td>
<td style="text-align:right;">
103.83373
</td>
<td style="text-align:left;">
negative
</td>
<td style="text-align:left;">
MGI_C57BL6J_96026
</td>
<td style="text-align:left;">
Hbb-bh3
</td>
<td style="text-align:left;">
NCBI_Gene:15134,ENSEMBL:ENSMUSG00000083216
</td>
<td style="text-align:left;">
MGI:96026
</td>
<td style="text-align:left;">
pseudogene
</td>
<td style="text-align:left;">
hemoglobin beta, pseudogene bh3
</td>
</tr>
<tr>
<td style="text-align:left;">
7
</td>
<td style="text-align:left;">
gene
</td>
<td style="text-align:right;">
103.83880
</td>
<td style="text-align:right;">
103.84073
</td>
<td style="text-align:left;">
negative
</td>
<td style="text-align:left;">
MGI_C57BL6J_96025
</td>
<td style="text-align:left;">
Hbb-bh2
</td>
<td style="text-align:left;">
NCBI_Gene:436003,ENSEMBL:ENSMUSG00000078621
</td>
<td style="text-align:left;">
MGI:96025
</td>
<td style="text-align:left;">
protein coding gene
</td>
<td style="text-align:left;">
hemoglobin beta, bh2
</td>
</tr>
<tr>
<td style="text-align:left;">
7
</td>
<td style="text-align:left;">
gene
</td>
<td style="text-align:right;">
103.84164
</td>
<td style="text-align:right;">
103.84316
</td>
<td style="text-align:left;">
negative
</td>
<td style="text-align:left;">
MGI_C57BL6J_96024
</td>
<td style="text-align:left;">
Hbb-bh1
</td>
<td style="text-align:left;">
NCBI_Gene:15132,ENSEMBL:ENSMUSG00000052217
</td>
<td style="text-align:left;">
MGI:96024
</td>
<td style="text-align:left;">
protein coding gene
</td>
<td style="text-align:left;">
hemoglobin Z, beta-like embryonic chain
</td>
</tr>
<tr>
<td style="text-align:left;">
7
</td>
<td style="text-align:left;">
pseudogene
</td>
<td style="text-align:right;">
103.85002
</td>
<td style="text-align:right;">
103.85015
</td>
<td style="text-align:left;">
negative
</td>
<td style="text-align:left;">
MGI_C57BL6J_96023
</td>
<td style="text-align:left;">
Hbb-bh0
</td>
<td style="text-align:left;">
ENSEMBL:ENSMUSG00000085700
</td>
<td style="text-align:left;">
MGI:96023
</td>
<td style="text-align:left;">
pseudogene
</td>
<td style="text-align:left;">
hemoglobin, beta, pseudogene bh0
</td>
</tr>
<tr>
<td style="text-align:left;">
7
</td>
<td style="text-align:left;">
gene
</td>
<td style="text-align:right;">
103.85175
</td>
<td style="text-align:right;">
103.85322
</td>
<td style="text-align:left;">
negative
</td>
<td style="text-align:left;">
MGI_C57BL6J_96027
</td>
<td style="text-align:left;">
Hbb-y
</td>
<td style="text-align:left;">
NCBI_Gene:15135,ENSEMBL:ENSMUSG00000052187
</td>
<td style="text-align:left;">
MGI:96027
</td>
<td style="text-align:left;">
protein coding gene
</td>
<td style="text-align:left;">
hemoglobin Y, beta-like embryonic chain
</td>
</tr>
<tr>
<td style="text-align:left;">
7
</td>
<td style="text-align:left;">
gene
</td>
<td style="text-align:right;">
103.87944
</td>
<td style="text-align:right;">
103.88625
</td>
<td style="text-align:left;">
negative
</td>
<td style="text-align:left;">
MGI_C57BL6J_1341906
</td>
<td style="text-align:left;">
Olfr66
</td>
<td style="text-align:left;">
NCBI_Gene:18367,ENSEMBL:ENSMUSG00000058200
</td>
<td style="text-align:left;">
MGI:1341906
</td>
<td style="text-align:left;">
protein coding gene
</td>
<td style="text-align:left;">
olfactory receptor 66
</td>
</tr>
<tr>
<td style="text-align:left;">
7
</td>
<td style="text-align:left;">
pseudogene
</td>
<td style="text-align:right;">
103.88257
</td>
<td style="text-align:right;">
103.88310
</td>
<td style="text-align:left;">
negative
</td>
<td style="text-align:left;">
MGI_C57BL6J_3709029
</td>
<td style="text-align:left;">
Dnajc19-ps
</td>
<td style="text-align:left;">
NCBI_Gene:100503724,ENSEMBL:ENSMUSG00000083854
</td>
<td style="text-align:left;">
MGI:3709029
</td>
<td style="text-align:left;">
pseudogene
</td>
<td style="text-align:left;">
DnaJ heat shock protein family (Hsp40) member C19, pseudogene
</td>
</tr>
<tr>
<td style="text-align:left;">
7
</td>
<td style="text-align:left;">
gene
</td>
<td style="text-align:right;">
103.88948
</td>
<td style="text-align:right;">
103.89449
</td>
<td style="text-align:left;">
negative
</td>
<td style="text-align:left;">
MGI_C57BL6J_1341900
</td>
<td style="text-align:left;">
Olfr64
</td>
<td style="text-align:left;">
NCBI_Gene:18366,ENSEMBL:ENSMUSG00000063615
</td>
<td style="text-align:left;">
MGI:1341900
</td>
<td style="text-align:left;">
protein coding gene
</td>
<td style="text-align:left;">
olfactory receptor 64
</td>
</tr>
<tr>
<td style="text-align:left;">
7
</td>
<td style="text-align:left;">
gene
</td>
<td style="text-align:right;">
103.90349
</td>
<td style="text-align:right;">
103.90865
</td>
<td style="text-align:left;">
positive
</td>
<td style="text-align:left;">
MGI_C57BL6J_1341910
</td>
<td style="text-align:left;">
Olfr65
</td>
<td style="text-align:left;">
NCBI_Gene:18365,ENSEMBL:ENSMUSG00000110259
</td>
<td style="text-align:left;">
MGI:1341910
</td>
<td style="text-align:left;">
protein coding gene
</td>
<td style="text-align:left;">
olfactory receptor 65
</td>
</tr>
<tr>
<td style="text-align:left;">
7
</td>
<td style="text-align:left;">
gene
</td>
<td style="text-align:right;">
103.91506
</td>
<td style="text-align:right;">
103.93067
</td>
<td style="text-align:left;">
positive
</td>
<td style="text-align:left;">
MGI_C57BL6J_3030465
</td>
<td style="text-align:left;">
Olfr631
</td>
<td style="text-align:left;">
NCBI_Gene:258961,ENSEMBL:ENSMUSG00000042219
</td>
<td style="text-align:left;">
MGI:3030465
</td>
<td style="text-align:left;">
protein coding gene
</td>
<td style="text-align:left;">
olfactory receptor 631
</td>
</tr>
<tr>
<td style="text-align:left;">
7
</td>
<td style="text-align:left;">
gene
</td>
<td style="text-align:right;">
103.93435
</td>
<td style="text-align:right;">
103.94058
</td>
<td style="text-align:left;">
positive
</td>
<td style="text-align:left;">
MGI_C57BL6J_3030466
</td>
<td style="text-align:left;">
Olfr632
</td>
<td style="text-align:left;">
NCBI_Gene:259123,ENSEMBL:ENSMUSG00000073938
</td>
<td style="text-align:left;">
MGI:3030466
</td>
<td style="text-align:left;">
protein coding gene
</td>
<td style="text-align:left;">
olfactory receptor 632
</td>
</tr>
<tr>
<td style="text-align:left;">
7
</td>
<td style="text-align:left;">
pseudogene
</td>
<td style="text-align:right;">
103.93975
</td>
<td style="text-align:right;">
103.93991
</td>
<td style="text-align:left;">
positive
</td>
<td style="text-align:left;">
MGI_C57BL6J_5791491
</td>
<td style="text-align:left;">
Gm45655
</td>
<td style="text-align:left;">
ENSEMBL:ENSMUSG00000109837
</td>
<td style="text-align:left;">
MGI:5791491
</td>
<td style="text-align:left;">
pseudogene
</td>
<td style="text-align:left;">
predicted gene 45655
</td>
</tr>
<tr>
<td style="text-align:left;">
7
</td>
<td style="text-align:left;">
gene
</td>
<td style="text-align:right;">
103.94444
</td>
<td style="text-align:right;">
103.94896
</td>
<td style="text-align:left;">
positive
</td>
<td style="text-align:left;">
MGI_C57BL6J_3030467
</td>
<td style="text-align:left;">
Olfr633
</td>
<td style="text-align:left;">
NCBI_Gene:258351,ENSEMBL:ENSMUSG00000073937
</td>
<td style="text-align:left;">
MGI:3030467
</td>
<td style="text-align:left;">
protein coding gene
</td>
<td style="text-align:left;">
olfactory receptor 633
</td>
</tr>
<tr>
<td style="text-align:left;">
7
</td>
<td style="text-align:left;">
pseudogene
</td>
<td style="text-align:right;">
103.95641
</td>
<td style="text-align:right;">
103.95737
</td>
<td style="text-align:left;">
positive
</td>
<td style="text-align:left;">
MGI_C57BL6J_3030468
</td>
<td style="text-align:left;">
Olfr634-ps1
</td>
<td style="text-align:left;">
NCBI_Gene:257949,ENSEMBL:ENSMUSG00000080751
</td>
<td style="text-align:left;">
MGI:3030468
</td>
<td style="text-align:left;">
polymorphic pseudogene
</td>
<td style="text-align:left;">
olfactory receptor 634, pseudogene 1
</td>
</tr>
<tr>
<td style="text-align:left;">
7
</td>
<td style="text-align:left;">
gene
</td>
<td style="text-align:right;">
103.97115
</td>
<td style="text-align:right;">
103.98754
</td>
<td style="text-align:left;">
positive
</td>
<td style="text-align:left;">
MGI_C57BL6J_3030469
</td>
<td style="text-align:left;">
Olfr635
</td>
<td style="text-align:left;">
NCBI_Gene:259122,ENSEMBL:ENSMUSG00000094520
</td>
<td style="text-align:left;">
MGI:3030469
</td>
<td style="text-align:left;">
protein coding gene
</td>
<td style="text-align:left;">
olfactory receptor 635
</td>
</tr>
<tr>
<td style="text-align:left;">
7
</td>
<td style="text-align:left;">
pseudogene
</td>
<td style="text-align:right;">
103.97336
</td>
<td style="text-align:right;">
103.97711
</td>
<td style="text-align:left;">
negative
</td>
<td style="text-align:left;">
MGI_C57BL6J_3645929
</td>
<td style="text-align:left;">
Gm4886
</td>
<td style="text-align:left;">
NCBI_Gene:233614,ENSEMBL:ENSMUSG00000082281
</td>
<td style="text-align:left;">
MGI:3645929
</td>
<td style="text-align:left;">
pseudogene
</td>
<td style="text-align:left;">
predicted gene 4886
</td>
</tr>
<tr>
<td style="text-align:left;">
7
</td>
<td style="text-align:left;">
pseudogene
</td>
<td style="text-align:right;">
103.98910
</td>
<td style="text-align:right;">
103.99004
</td>
<td style="text-align:left;">
negative
</td>
<td style="text-align:left;">
MGI_C57BL6J_3030470
</td>
<td style="text-align:left;">
Olfr636-ps1
</td>
<td style="text-align:left;">
NCBI_Gene:258144,ENSEMBL:ENSMUSG00000055621
</td>
<td style="text-align:left;">
MGI:3030470
</td>
<td style="text-align:left;">
pseudogene
</td>
<td style="text-align:left;">
olfactory receptor 636, pseudogene 1
</td>
</tr>
<tr>
<td style="text-align:left;">
7
</td>
<td style="text-align:left;">
pseudogene
</td>
<td style="text-align:right;">
103.99598
</td>
<td style="text-align:right;">
103.99664
</td>
<td style="text-align:left;">
positive
</td>
<td style="text-align:left;">
MGI_C57BL6J_3030471
</td>
<td style="text-align:left;">
Olfr637-ps1
</td>
<td style="text-align:left;">
NCBI_Gene:258143,ENSEMBL:ENSMUSG00000084236
</td>
<td style="text-align:left;">
MGI:3030471
</td>
<td style="text-align:left;">
pseudogene
</td>
<td style="text-align:left;">
olfactory receptor 637, pseudogene 1
</td>
</tr>
<tr>
<td style="text-align:left;">
7
</td>
<td style="text-align:left;">
gene
</td>
<td style="text-align:right;">
103.99875
</td>
<td style="text-align:right;">
104.00436
</td>
<td style="text-align:left;">
positive
</td>
<td style="text-align:left;">
MGI_C57BL6J_3030472
</td>
<td style="text-align:left;">
Olfr638
</td>
<td style="text-align:left;">
NCBI_Gene:259124,ENSEMBL:ENSMUSG00000094063
</td>
<td style="text-align:left;">
MGI:3030472
</td>
<td style="text-align:left;">
protein coding gene
</td>
<td style="text-align:left;">
olfactory receptor 638
</td>
</tr>
<tr>
<td style="text-align:left;">
7
</td>
<td style="text-align:left;">
gene
</td>
<td style="text-align:right;">
103.99880
</td>
<td style="text-align:right;">
104.13741
</td>
<td style="text-align:left;">
negative
</td>
<td style="text-align:left;">
MGI_C57BL6J_3030477
</td>
<td style="text-align:left;">
Olfr643
</td>
<td style="text-align:left;">
NCBI_Gene:259081,ENSEMBL:ENSMUSG00000109824
</td>
<td style="text-align:left;">
MGI:3030477
</td>
<td style="text-align:left;">
protein coding gene
</td>
<td style="text-align:left;">
olfactory receptor 643
</td>
</tr>
<tr>
<td style="text-align:left;">
7
</td>
<td style="text-align:left;">
gene
</td>
<td style="text-align:right;">
104.00623
</td>
<td style="text-align:right;">
104.01730
</td>
<td style="text-align:left;">
negative
</td>
<td style="text-align:left;">
MGI_C57BL6J_3030473
</td>
<td style="text-align:left;">
Olfr639
</td>
<td style="text-align:left;">
NCBI_Gene:259088,ENSEMBL:ENSMUSG00000066263
</td>
<td style="text-align:left;">
MGI:3030473
</td>
<td style="text-align:left;">
protein coding gene
</td>
<td style="text-align:left;">
olfactory receptor 639
</td>
</tr>
<tr>
<td style="text-align:left;">
7
</td>
<td style="text-align:left;">
gene
</td>
<td style="text-align:right;">
104.01869
</td>
<td style="text-align:right;">
104.02557
</td>
<td style="text-align:left;">
negative
</td>
<td style="text-align:left;">
MGI_C57BL6J_3030474
</td>
<td style="text-align:left;">
Olfr640
</td>
<td style="text-align:left;">
NCBI_Gene:258819,ENSEMBL:ENSMUSG00000066262
</td>
<td style="text-align:left;">
MGI:3030474
</td>
<td style="text-align:left;">
protein coding gene
</td>
<td style="text-align:left;">
olfactory receptor 640
</td>
</tr>
<tr>
<td style="text-align:left;">
7
</td>
<td style="text-align:left;">
gene
</td>
<td style="text-align:right;">
104.03544
</td>
<td style="text-align:right;">
104.04207
</td>
<td style="text-align:left;">
positive
</td>
<td style="text-align:left;">
MGI_C57BL6J_3030475
</td>
<td style="text-align:left;">
Olfr641
</td>
<td style="text-align:left;">
NCBI_Gene:259075,ENSEMBL:ENSMUSG00000073932
</td>
<td style="text-align:left;">
MGI:3030475
</td>
<td style="text-align:left;">
protein coding gene
</td>
<td style="text-align:left;">
olfactory receptor 641
</td>
</tr>
<tr>
<td style="text-align:left;">
7
</td>
<td style="text-align:left;">
gene
</td>
<td style="text-align:right;">
104.04653
</td>
<td style="text-align:right;">
104.05267
</td>
<td style="text-align:left;">
negative
</td>
<td style="text-align:left;">
MGI_C57BL6J_3030476
</td>
<td style="text-align:left;">
Olfr642
</td>
<td style="text-align:left;">
NCBI_Gene:258326,ENSEMBL:ENSMUSG00000049797
</td>
<td style="text-align:left;">
MGI:3030476
</td>
<td style="text-align:left;">
protein coding gene
</td>
<td style="text-align:left;">
olfactory receptor 642
</td>
</tr>
<tr>
<td style="text-align:left;">
7
</td>
<td style="text-align:left;">
gene
</td>
<td style="text-align:right;">
104.05360
</td>
<td style="text-align:right;">
104.06145
</td>
<td style="text-align:left;">
negative
</td>
<td style="text-align:left;">
MGI_C57BL6J_1924125
</td>
<td style="text-align:left;">
4930516K23Rik
</td>
<td style="text-align:left;">
ENSEMBL:ENSMUSG00000090219
</td>
<td style="text-align:left;">
MGI:1924125
</td>
<td style="text-align:left;">
protein coding gene
</td>
<td style="text-align:left;">
RIKEN cDNA 4930516K23 gene
</td>
</tr>
<tr>
<td style="text-align:left;">
7
</td>
<td style="text-align:left;">
gene
</td>
<td style="text-align:right;">
104.06629
</td>
<td style="text-align:right;">
104.07187
</td>
<td style="text-align:left;">
negative
</td>
<td style="text-align:left;">
MGI_C57BL6J_3030478
</td>
<td style="text-align:left;">
Olfr644
</td>
<td style="text-align:left;">
NCBI_Gene:259125,ENSEMBL:ENSMUSG00000110012
</td>
<td style="text-align:left;">
MGI:3030478
</td>
<td style="text-align:left;">
protein coding gene
</td>
<td style="text-align:left;">
olfactory receptor 644
</td>
</tr>
<tr>
<td style="text-align:left;">
7
</td>
<td style="text-align:left;">
gene
</td>
<td style="text-align:right;">
104.08179
</td>
<td style="text-align:right;">
104.08546
</td>
<td style="text-align:left;">
negative
</td>
<td style="text-align:left;">
MGI_C57BL6J_3030479
</td>
<td style="text-align:left;">
Olfr645
</td>
<td style="text-align:left;">
NCBI_Gene:258247,ENSEMBL:ENSMUSG00000051340
</td>
<td style="text-align:left;">
MGI:3030479
</td>
<td style="text-align:left;">
protein coding gene
</td>
<td style="text-align:left;">
olfactory receptor 645
</td>
</tr>
<tr>
<td style="text-align:left;">
7
</td>
<td style="text-align:left;">
gene
</td>
<td style="text-align:right;">
104.09732
</td>
<td style="text-align:right;">
104.10977
</td>
<td style="text-align:left;">
positive
</td>
<td style="text-align:left;">
MGI_C57BL6J_3030480
</td>
<td style="text-align:left;">
Olfr646
</td>
<td style="text-align:left;">
NCBI_Gene:259058,ENSEMBL:ENSMUSG00000073931
</td>
<td style="text-align:left;">
MGI:3030480
</td>
<td style="text-align:left;">
protein coding gene
</td>
<td style="text-align:left;">
olfactory receptor 646
</td>
</tr>
<tr>
<td style="text-align:left;">
7
</td>
<td style="text-align:left;">
gene
</td>
<td style="text-align:right;">
104.12791
</td>
<td style="text-align:right;">
104.12983
</td>
<td style="text-align:left;">
negative
</td>
<td style="text-align:left;">
MGI_C57BL6J_1918230
</td>
<td style="text-align:left;">
Ubqln5
</td>
<td style="text-align:left;">
NCBI_Gene:70980,ENSEMBL:ENSMUSG00000055643
</td>
<td style="text-align:left;">
MGI:1918230
</td>
<td style="text-align:left;">
protein coding gene
</td>
<td style="text-align:left;">
ubiquilin 5
</td>
</tr>
<tr>
<td style="text-align:left;">
7
</td>
<td style="text-align:left;">
gene
</td>
<td style="text-align:right;">
104.14062
</td>
<td style="text-align:right;">
104.14332
</td>
<td style="text-align:left;">
negative
</td>
<td style="text-align:left;">
MGI_C57BL6J_3045291
</td>
<td style="text-align:left;">
Ubqln3
</td>
<td style="text-align:left;">
NCBI_Gene:244178,ENSEMBL:ENSMUSG00000051618
</td>
<td style="text-align:left;">
MGI:3045291
</td>
<td style="text-align:left;">
protein coding gene
</td>
<td style="text-align:left;">
ubiquilin 3
</td>
</tr>
<tr>
<td style="text-align:left;">
7
</td>
<td style="text-align:left;">
gene
</td>
<td style="text-align:right;">
104.14826
</td>
<td style="text-align:right;">
104.15056
</td>
<td style="text-align:left;">
negative
</td>
<td style="text-align:left;">
MGI_C57BL6J_2685336
</td>
<td style="text-align:left;">
Ubqlnl
</td>
<td style="text-align:left;">
NCBI_Gene:244179,ENSEMBL:ENSMUSG00000051437
</td>
<td style="text-align:left;">
MGI:2685336
</td>
<td style="text-align:left;">
protein coding gene
</td>
<td style="text-align:left;">
ubiquilin-like
</td>
</tr>
<tr>
<td style="text-align:left;">
7
</td>
<td style="text-align:left;">
gene
</td>
<td style="text-align:right;">
104.15301
</td>
<td style="text-align:right;">
104.16483
</td>
<td style="text-align:left;">
negative
</td>
<td style="text-align:left;">
MGI_C57BL6J_2443346
</td>
<td style="text-align:left;">
Olfm5
</td>
<td style="text-align:left;">
NCBI_Gene:244180,ENSEMBL:ENSMUSG00000044265
</td>
<td style="text-align:left;">
MGI:2443346
</td>
<td style="text-align:left;">
protein coding gene
</td>
<td style="text-align:left;">
olfactomedin 5
</td>
</tr>
<tr>
<td style="text-align:left;">
7
</td>
<td style="text-align:left;">
pseudogene
</td>
<td style="text-align:right;">
104.16738
</td>
<td style="text-align:right;">
104.16834
</td>
<td style="text-align:left;">
negative
</td>
<td style="text-align:left;">
MGI_C57BL6J_3030481
</td>
<td style="text-align:left;">
Olfr647-ps1
</td>
<td style="text-align:left;">
NCBI_Gene:258137,ENSEMBL:ENSMUSG00000083513
</td>
<td style="text-align:left;">
MGI:3030481
</td>
<td style="text-align:left;">
pseudogene
</td>
<td style="text-align:left;">
olfactory receptor 647, pseudogene 1
</td>
</tr>
<tr>
<td style="text-align:left;">
7
</td>
<td style="text-align:left;">
gene
</td>
<td style="text-align:right;">
104.17760
</td>
<td style="text-align:right;">
104.18339
</td>
<td style="text-align:left;">
negative
</td>
<td style="text-align:left;">
MGI_C57BL6J_3030482
</td>
<td style="text-align:left;">
Olfr648
</td>
<td style="text-align:left;">
NCBI_Gene:258746,ENSEMBL:ENSMUSG00000042909
</td>
<td style="text-align:left;">
MGI:3030482
</td>
<td style="text-align:left;">
protein coding gene
</td>
<td style="text-align:left;">
olfactory receptor 648
</td>
</tr>
<tr>
<td style="text-align:left;">
7
</td>
<td style="text-align:left;">
gene
</td>
<td style="text-align:right;">
104.18899
</td>
<td style="text-align:right;">
104.19598
</td>
<td style="text-align:left;">
negative
</td>
<td style="text-align:left;">
MGI_C57BL6J_3030483
</td>
<td style="text-align:left;">
Olfr649
</td>
<td style="text-align:left;">
NCBI_Gene:259057,ENSEMBL:ENSMUSG00000044899
</td>
<td style="text-align:left;">
MGI:3030483
</td>
<td style="text-align:left;">
protein coding gene
</td>
<td style="text-align:left;">
olfactory receptor 649
</td>
</tr>
<tr>
<td style="text-align:left;">
7
</td>
<td style="text-align:left;">
pseudogene
</td>
<td style="text-align:right;">
104.20344
</td>
<td style="text-align:right;">
104.20786
</td>
<td style="text-align:left;">
positive
</td>
<td style="text-align:left;">
MGI_C57BL6J_3030484
</td>
<td style="text-align:left;">
Olfr650-ps1
</td>
<td style="text-align:left;">
NCBI_Gene:258212,ENSEMBL:ENSMUSG00000081859
</td>
<td style="text-align:left;">
MGI:3030484
</td>
<td style="text-align:left;">
polymorphic pseudogene
</td>
<td style="text-align:left;">
olfactory receptor 650, pseudogene 1
</td>
</tr>
<tr>
<td style="text-align:left;">
7
</td>
<td style="text-align:left;">
gene
</td>
<td style="text-align:right;">
104.21878
</td>
<td style="text-align:right;">
104.23515
</td>
<td style="text-align:left;">
positive
</td>
<td style="text-align:left;">
MGI_C57BL6J_2137352
</td>
<td style="text-align:left;">
Trim6
</td>
<td style="text-align:left;">
NCBI_Gene:94088,ENSEMBL:ENSMUSG00000072244
</td>
<td style="text-align:left;">
MGI:2137352
</td>
<td style="text-align:left;">
protein coding gene
</td>
<td style="text-align:left;">
tripartite motif-containing 6
</td>
</tr>
<tr>
<td style="text-align:left;">
7
</td>
<td style="text-align:left;">
pseudogene
</td>
<td style="text-align:right;">
104.22159
</td>
<td style="text-align:right;">
104.22298
</td>
<td style="text-align:left;">
negative
</td>
<td style="text-align:left;">
MGI_C57BL6J_3648748
</td>
<td style="text-align:left;">
Gm8667
</td>
<td style="text-align:left;">
NCBI_Gene:667497,ENSEMBL:ENSMUSG00000109712
</td>
<td style="text-align:left;">
MGI:3648748
</td>
<td style="text-align:left;">
pseudogene
</td>
<td style="text-align:left;">
predicted gene 8667
</td>
</tr>
<tr>
<td style="text-align:left;">
7
</td>
<td style="text-align:left;">
gene
</td>
<td style="text-align:right;">
104.24444
</td>
<td style="text-align:right;">
104.26224
</td>
<td style="text-align:left;">
positive
</td>
<td style="text-align:left;">
MGI_C57BL6J_2137359
</td>
<td style="text-align:left;">
Trim34a
</td>
<td style="text-align:left;">
NCBI_Gene:94094,ENSEMBL:ENSMUSG00000056144
</td>
<td style="text-align:left;">
MGI:2137359
</td>
<td style="text-align:left;">
protein coding gene
</td>
<td style="text-align:left;">
tripartite motif-containing 34A
</td>
</tr>
<tr>
<td style="text-align:left;">
7
</td>
<td style="text-align:left;">
gene
</td>
<td style="text-align:right;">
104.26147
</td>
<td style="text-align:right;">
104.26217
</td>
<td style="text-align:left;">
negative
</td>
<td style="text-align:left;">
MGI_C57BL6J_6096073
</td>
<td style="text-align:left;">
Gm47248
</td>
<td style="text-align:left;">
ENSEMBL:ENSMUSG00000111737
</td>
<td style="text-align:left;">
MGI:6096073
</td>
<td style="text-align:left;">
lncRNA gene
</td>
<td style="text-align:left;">
predicted gene, 47248
</td>
</tr>
<tr>
<td style="text-align:left;">
7
</td>
<td style="text-align:left;">
gene
</td>
<td style="text-align:right;">
104.26339
</td>
<td style="text-align:right;">
104.28842
</td>
<td style="text-align:left;">
negative
</td>
<td style="text-align:left;">
MGI_C57BL6J_3646853
</td>
<td style="text-align:left;">
Trim5
</td>
<td style="text-align:left;">
NCBI_Gene:667823,ENSEMBL:ENSMUSG00000060441
</td>
<td style="text-align:left;">
MGI:3646853
</td>
<td style="text-align:left;">
protein coding gene
</td>
<td style="text-align:left;">
tripartite motif-containing 5
</td>
</tr>
<tr>
<td style="text-align:left;">
7
</td>
<td style="text-align:left;">
gene
</td>
<td style="text-align:right;">
104.26468
</td>
<td style="text-align:right;">
104.26479
</td>
<td style="text-align:left;">
negative
</td>
<td style="text-align:left;">
MGI_C57BL6J_5454623
</td>
<td style="text-align:left;">
Gm24846
</td>
<td style="text-align:left;">
ENSEMBL:ENSMUSG00000088658
</td>
<td style="text-align:left;">
MGI:5454623
</td>
<td style="text-align:left;">
snRNA gene
</td>
<td style="text-align:left;">
predicted gene, 24846
</td>
</tr>
<tr>
<td style="text-align:left;">
7
</td>
<td style="text-align:left;">
gene
</td>
<td style="text-align:right;">
104.26522
</td>
<td style="text-align:right;">
104.26590
</td>
<td style="text-align:left;">
negative
</td>
<td style="text-align:left;">
MGI_C57BL6J_4821182
</td>
<td style="text-align:left;">
Trim12b
</td>
<td style="text-align:left;">
NA
</td>
<td style="text-align:left;">
NA
</td>
<td style="text-align:left;">
protein coding gene
</td>
<td style="text-align:left;">
tripartite motif-containing 12B
</td>
</tr>
<tr>
<td style="text-align:left;">
7
</td>
<td style="text-align:left;">
gene
</td>
<td style="text-align:right;">
104.29989
</td>
<td style="text-align:right;">
104.31585
</td>
<td style="text-align:left;">
negative
</td>
<td style="text-align:left;">
MGI_C57BL6J_1923931
</td>
<td style="text-align:left;">
Trim12a
</td>
<td style="text-align:left;">
NCBI_Gene:76681,ENSEMBL:ENSMUSG00000066258
</td>
<td style="text-align:left;">
MGI:1923931
</td>
<td style="text-align:left;">
protein coding gene
</td>
<td style="text-align:left;">
tripartite motif-containing 12A
</td>
</tr>
<tr>
<td style="text-align:left;">
7
</td>
<td style="text-align:left;">
gene
</td>
<td style="text-align:right;">
104.31051
</td>
<td style="text-align:right;">
104.31106
</td>
<td style="text-align:left;">
negative
</td>
<td style="text-align:left;">
MGI_C57BL6J_1925028
</td>
<td style="text-align:left;">
A430102O06Rik
</td>
<td style="text-align:left;">
NA
</td>
<td style="text-align:left;">
NA
</td>
<td style="text-align:left;">
unclassified gene
</td>
<td style="text-align:left;">
RIKEN cDNA A430102O06 gene
</td>
</tr>
<tr>
<td style="text-align:left;">
7
</td>
<td style="text-align:left;">
gene
</td>
<td style="text-align:right;">
104.31502
</td>
<td style="text-align:right;">
104.32484
</td>
<td style="text-align:left;">
positive
</td>
<td style="text-align:left;">
MGI_C57BL6J_3705170
</td>
<td style="text-align:left;">
Gm15133
</td>
<td style="text-align:left;">
NCBI_Gene:622070,ENSEMBL:ENSMUSG00000087175
</td>
<td style="text-align:left;">
MGI:3705170
</td>
<td style="text-align:left;">
lncRNA gene
</td>
<td style="text-align:left;">
predicted gene 15133
</td>
</tr>
<tr>
<td style="text-align:left;">
7
</td>
<td style="text-align:left;">
pseudogene
</td>
<td style="text-align:right;">
104.31907
</td>
<td style="text-align:right;">
104.32018
</td>
<td style="text-align:left;">
negative
</td>
<td style="text-align:left;">
MGI_C57BL6J_5009952
</td>
<td style="text-align:left;">
Gm17788
</td>
<td style="text-align:left;">
NCBI_Gene:100048312
</td>
<td style="text-align:left;">
MGI:5009952
</td>
<td style="text-align:left;">
pseudogene
</td>
<td style="text-align:left;">
predicted gene, 17788
</td>
</tr>
<tr>
<td style="text-align:left;">
7
</td>
<td style="text-align:left;">
gene
</td>
<td style="text-align:right;">
104.32947
</td>
<td style="text-align:right;">
104.33691
</td>
<td style="text-align:left;">
positive
</td>
<td style="text-align:left;">
MGI_C57BL6J_4821264
</td>
<td style="text-align:left;">
Trim34b
</td>
<td style="text-align:left;">
NCBI_Gene:434218,ENSEMBL:ENSMUSG00000090215
</td>
<td style="text-align:left;">
MGI:4821264
</td>
<td style="text-align:left;">
protein coding gene
</td>
<td style="text-align:left;">
tripartite motif-containing 34B
</td>
</tr>
<tr>
<td style="text-align:left;">
7
</td>
<td style="text-align:left;">
gene
</td>
<td style="text-align:right;">
104.33875
</td>
<td style="text-align:right;">
104.35336
</td>
<td style="text-align:left;">
negative
</td>
<td style="text-align:left;">
MGI_C57BL6J_4821183
</td>
<td style="text-align:left;">
Trim12c
</td>
<td style="text-align:left;">
NCBI_Gene:319236,ENSEMBL:ENSMUSG00000057143
</td>
<td style="text-align:left;">
MGI:4821183
</td>
<td style="text-align:left;">
protein coding gene
</td>
<td style="text-align:left;">
tripartite motif-containing 12C
</td>
</tr>
<tr>
<td style="text-align:left;">
7
</td>
<td style="text-align:left;">
gene
</td>
<td style="text-align:right;">
104.34009
</td>
<td style="text-align:right;">
104.34019
</td>
<td style="text-align:left;">
negative
</td>
<td style="text-align:left;">
MGI_C57BL6J_5531377
</td>
<td style="text-align:left;">
Gm27995
</td>
<td style="text-align:left;">
ENSEMBL:ENSMUSG00000098399
</td>
<td style="text-align:left;">
MGI:5531377
</td>
<td style="text-align:left;">
snRNA gene
</td>
<td style="text-align:left;">
predicted gene, 27995
</td>
</tr>
<tr>
<td style="text-align:left;">
7
</td>
<td style="text-align:left;">
gene
</td>
<td style="text-align:right;">
104.35182
</td>
<td style="text-align:right;">
104.35575
</td>
<td style="text-align:left;">
positive
</td>
<td style="text-align:left;">
MGI_C57BL6J_5595589
</td>
<td style="text-align:left;">
Gm36430
</td>
<td style="text-align:left;">
NCBI_Gene:102640341
</td>
<td style="text-align:left;">
MGI:5595589
</td>
<td style="text-align:left;">
lncRNA gene
</td>
<td style="text-align:left;">
predicted gene, 36430
</td>
</tr>
<tr>
<td style="text-align:left;">
7
</td>
<td style="text-align:left;">
gene
</td>
<td style="text-align:right;">
104.35538
</td>
<td style="text-align:right;">
104.36988
</td>
<td style="text-align:left;">
negative
</td>
<td style="text-align:left;">
MGI_C57BL6J_4821256
</td>
<td style="text-align:left;">
Trim30b
</td>
<td style="text-align:left;">
NCBI_Gene:244183,ENSEMBL:ENSMUSG00000052749
</td>
<td style="text-align:left;">
MGI:4821256
</td>
<td style="text-align:left;">
protein coding gene
</td>
<td style="text-align:left;">
tripartite motif-containing 30B
</td>
</tr>
<tr>
<td style="text-align:left;">
7
</td>
<td style="text-align:left;">
gene
</td>
<td style="text-align:right;">
104.35540
</td>
<td style="text-align:right;">
104.36988
</td>
<td style="text-align:left;">
negative
</td>
<td style="text-align:left;">
MGI_C57BL6J_5621410
</td>
<td style="text-align:left;">
Gm38525
</td>
<td style="text-align:left;">
NCBI_Gene:102640171
</td>
<td style="text-align:left;">
MGI:5621410
</td>
<td style="text-align:left;">
protein coding gene
</td>
<td style="text-align:left;">
predicted gene, 38525
</td>
</tr>
<tr>
<td style="text-align:left;">
7
</td>
<td style="text-align:left;">
gene
</td>
<td style="text-align:right;">
104.35616
</td>
<td style="text-align:right;">
104.36269
</td>
<td style="text-align:left;">
positive
</td>
<td style="text-align:left;">
MGI_C57BL6J_5825634
</td>
<td style="text-align:left;">
Gm45997
</td>
<td style="text-align:left;">
NCBI_Gene:108167453
</td>
<td style="text-align:left;">
MGI:5825634
</td>
<td style="text-align:left;">
lncRNA gene
</td>
<td style="text-align:left;">
predicted gene, 45997
</td>
</tr>
<tr>
<td style="text-align:left;">
7
</td>
<td style="text-align:left;">
gene
</td>
<td style="text-align:right;">
104.37912
</td>
<td style="text-align:right;">
104.40085
</td>
<td style="text-align:left;">
negative
</td>
<td style="text-align:left;">
MGI_C57BL6J_4821257
</td>
<td style="text-align:left;">
Trim30c
</td>
<td style="text-align:left;">
NCBI_Gene:434219,ENSEMBL:ENSMUSG00000078616
</td>
<td style="text-align:left;">
MGI:4821257
</td>
<td style="text-align:left;">
protein coding gene
</td>
<td style="text-align:left;">
tripartite motif-containing 30C
</td>
</tr>
<tr>
<td style="text-align:left;">
7
</td>
<td style="text-align:left;">
pseudogene
</td>
<td style="text-align:right;">
104.40540
</td>
<td style="text-align:right;">
104.40653
</td>
<td style="text-align:left;">
negative
</td>
<td style="text-align:left;">
MGI_C57BL6J_5791307
</td>
<td style="text-align:left;">
Gm45471
</td>
<td style="text-align:left;">
ENSEMBL:ENSMUSG00000110114
</td>
<td style="text-align:left;">
MGI:5791307
</td>
<td style="text-align:left;">
pseudogene
</td>
<td style="text-align:left;">
predicted gene 45471
</td>
</tr>
<tr>
<td style="text-align:left;">
7
</td>
<td style="text-align:left;">
gene
</td>
<td style="text-align:right;">
104.40902
</td>
<td style="text-align:right;">
104.46519
</td>
<td style="text-align:left;">
negative
</td>
<td style="text-align:left;">
MGI_C57BL6J_98178
</td>
<td style="text-align:left;">
Trim30a
</td>
<td style="text-align:left;">
NCBI_Gene:20128,ENSEMBL:ENSMUSG00000030921
</td>
<td style="text-align:left;">
MGI:98178
</td>
<td style="text-align:left;">
protein coding gene
</td>
<td style="text-align:left;">
tripartite motif-containing 30A
</td>
</tr>
<tr>
<td style="text-align:left;">
7
</td>
<td style="text-align:left;">
gene
</td>
<td style="text-align:right;">
104.42327
</td>
<td style="text-align:right;">
104.42338
</td>
<td style="text-align:left;">
positive
</td>
<td style="text-align:left;">
MGI_C57BL6J_5455182
</td>
<td style="text-align:left;">
Gm25405
</td>
<td style="text-align:left;">
ENSEMBL:ENSMUSG00000064907
</td>
<td style="text-align:left;">
MGI:5455182
</td>
<td style="text-align:left;">
snRNA gene
</td>
<td style="text-align:left;">
predicted gene, 25405
</td>
</tr>
<tr>
<td style="text-align:left;">
7
</td>
<td style="text-align:left;">
gene
</td>
<td style="text-align:right;">
104.47001
</td>
<td style="text-align:right;">
104.50785
</td>
<td style="text-align:left;">
negative
</td>
<td style="text-align:left;">
MGI_C57BL6J_3035181
</td>
<td style="text-align:left;">
Trim30d
</td>
<td style="text-align:left;">
NCBI_Gene:209387,ENSEMBL:ENSMUSG00000057596
</td>
<td style="text-align:left;">
MGI:3035181
</td>
<td style="text-align:left;">
protein coding gene
</td>
<td style="text-align:left;">
tripartite motif-containing 30D
</td>
</tr>
<tr>
<td style="text-align:left;">
7
</td>
<td style="text-align:left;">
pseudogene
</td>
<td style="text-align:right;">
104.49906
</td>
<td style="text-align:right;">
104.50215
</td>
<td style="text-align:left;">
negative
</td>
<td style="text-align:left;">
MGI_C57BL6J_3643716
</td>
<td style="text-align:left;">
Gm16464
</td>
<td style="text-align:left;">
NCBI_Gene:672495,ENSEMBL:ENSMUSG00000082154
</td>
<td style="text-align:left;">
MGI:3643716
</td>
<td style="text-align:left;">
pseudogene
</td>
<td style="text-align:left;">
predicted gene 16464
</td>
</tr>
<tr>
<td style="text-align:left;">
7
</td>
<td style="text-align:left;">
pseudogene
</td>
<td style="text-align:right;">
104.52861
</td>
<td style="text-align:right;">
104.52909
</td>
<td style="text-align:left;">
negative
</td>
<td style="text-align:left;">
MGI_C57BL6J_3643499
</td>
<td style="text-align:left;">
Gm6574
</td>
<td style="text-align:left;">
NCBI_Gene:625305,ENSEMBL:ENSMUSG00000083248
</td>
<td style="text-align:left;">
MGI:3643499
</td>
<td style="text-align:left;">
pseudogene
</td>
<td style="text-align:left;">
predicted gene 6574
</td>
</tr>
<tr>
<td style="text-align:left;">
7
</td>
<td style="text-align:left;">
pseudogene
</td>
<td style="text-align:right;">
104.53296
</td>
<td style="text-align:right;">
104.53536
</td>
<td style="text-align:left;">
negative
</td>
<td style="text-align:left;">
MGI_C57BL6J_4821258
</td>
<td style="text-align:left;">
Trim30e-ps1
</td>
<td style="text-align:left;">
NCBI_Gene:625321,ENSEMBL:ENSMUSG00000073929
</td>
<td style="text-align:left;">
MGI:4821258
</td>
<td style="text-align:left;">
pseudogene
</td>
<td style="text-align:left;">
tripartite motif-containing 30E, pseudogene 1
</td>
</tr>
<tr>
<td style="text-align:left;">
7
</td>
<td style="text-align:left;">
gene
</td>
<td style="text-align:right;">
104.55013
</td>
<td style="text-align:right;">
104.55442
</td>
<td style="text-align:left;">
positive
</td>
<td style="text-align:left;">
MGI_C57BL6J_3030485
</td>
<td style="text-align:left;">
Olfr651
</td>
<td style="text-align:left;">
NCBI_Gene:258809,ENSEMBL:ENSMUSG00000073928
</td>
<td style="text-align:left;">
MGI:3030485
</td>
<td style="text-align:left;">
protein coding gene
</td>
<td style="text-align:left;">
olfactory receptor 651
</td>
</tr>
<tr>
<td style="text-align:left;">
7
</td>
<td style="text-align:left;">
gene
</td>
<td style="text-align:right;">
104.56090
</td>
<td style="text-align:right;">
104.56863
</td>
<td style="text-align:left;">
positive
</td>
<td style="text-align:left;">
MGI_C57BL6J_3030486
</td>
<td style="text-align:left;">
Olfr652
</td>
<td style="text-align:left;">
NCBI_Gene:259050,ENSEMBL:ENSMUSG00000073927
</td>
<td style="text-align:left;">
MGI:3030486
</td>
<td style="text-align:left;">
protein coding gene
</td>
<td style="text-align:left;">
olfactory receptor 652
</td>
</tr>
<tr>
<td style="text-align:left;">
7
</td>
<td style="text-align:left;">
gene
</td>
<td style="text-align:right;">
104.57731
</td>
<td style="text-align:right;">
104.58412
</td>
<td style="text-align:left;">
positive
</td>
<td style="text-align:left;">
MGI_C57BL6J_3030487
</td>
<td style="text-align:left;">
Olfr653
</td>
<td style="text-align:left;">
NCBI_Gene:57250,ENSEMBL:ENSMUSG00000073926
</td>
<td style="text-align:left;">
MGI:3030487
</td>
<td style="text-align:left;">
protein coding gene
</td>
<td style="text-align:left;">
olfactory receptor 653
</td>
</tr>
<tr>
<td style="text-align:left;">
7
</td>
<td style="text-align:left;">
gene
</td>
<td style="text-align:right;">
104.58702
</td>
<td style="text-align:right;">
104.59072
</td>
<td style="text-align:left;">
positive
</td>
<td style="text-align:left;">
MGI_C57BL6J_3030488
</td>
<td style="text-align:left;">
Olfr654
</td>
<td style="text-align:left;">
NCBI_Gene:258377,ENSEMBL:ENSMUSG00000073925
</td>
<td style="text-align:left;">
MGI:3030488
</td>
<td style="text-align:left;">
protein coding gene
</td>
<td style="text-align:left;">
olfactory receptor 654
</td>
</tr>
<tr>
<td style="text-align:left;">
7
</td>
<td style="text-align:left;">
gene
</td>
<td style="text-align:right;">
104.59530
</td>
<td style="text-align:right;">
104.60178
</td>
<td style="text-align:left;">
negative
</td>
<td style="text-align:left;">
MGI_C57BL6J_3030489
</td>
<td style="text-align:left;">
Olfr655
</td>
<td style="text-align:left;">
NCBI_Gene:258817,ENSEMBL:ENSMUSG00000051182
</td>
<td style="text-align:left;">
MGI:3030489
</td>
<td style="text-align:left;">
protein coding gene
</td>
<td style="text-align:left;">
olfactory receptor 655
</td>
</tr>
<tr>
<td style="text-align:left;">
7
</td>
<td style="text-align:left;">
gene
</td>
<td style="text-align:right;">
104.61396
</td>
<td style="text-align:right;">
104.62160
</td>
<td style="text-align:left;">
positive
</td>
<td style="text-align:left;">
MGI_C57BL6J_3030490
</td>
<td style="text-align:left;">
Olfr656
</td>
<td style="text-align:left;">
NCBI_Gene:259078,ENSEMBL:ENSMUSG00000073924
</td>
<td style="text-align:left;">
MGI:3030490
</td>
<td style="text-align:left;">
protein coding gene
</td>
<td style="text-align:left;">
olfactory receptor 656
</td>
</tr>
<tr>
<td style="text-align:left;">
7
</td>
<td style="text-align:left;">
gene
</td>
<td style="text-align:right;">
104.62479
</td>
<td style="text-align:right;">
104.63748
</td>
<td style="text-align:left;">
positive
</td>
<td style="text-align:left;">
MGI_C57BL6J_3030491
</td>
<td style="text-align:left;">
Olfr657
</td>
<td style="text-align:left;">
NCBI_Gene:258309,ENSEMBL:ENSMUSG00000073923
</td>
<td style="text-align:left;">
MGI:3030491
</td>
<td style="text-align:left;">
protein coding gene
</td>
<td style="text-align:left;">
olfactory receptor 657
</td>
</tr>
<tr>
<td style="text-align:left;">
7
</td>
<td style="text-align:left;">
pseudogene
</td>
<td style="text-align:right;">
104.63411
</td>
<td style="text-align:right;">
104.63462
</td>
<td style="text-align:left;">
negative
</td>
<td style="text-align:left;">
MGI_C57BL6J_5588865
</td>
<td style="text-align:left;">
Gm29706
</td>
<td style="text-align:left;">
NCBI_Gene:101055662
</td>
<td style="text-align:left;">
MGI:5588865
</td>
<td style="text-align:left;">
pseudogene
</td>
<td style="text-align:left;">
predicted gene, 29706
</td>
</tr>
<tr>
<td style="text-align:left;">
7
</td>
<td style="text-align:left;">
pseudogene
</td>
<td style="text-align:right;">
104.63464
</td>
<td style="text-align:right;">
104.63522
</td>
<td style="text-align:left;">
positive
</td>
<td style="text-align:left;">
MGI_C57BL6J_5010596
</td>
<td style="text-align:left;">
Gm18411
</td>
<td style="text-align:left;">
NCBI_Gene:100417128
</td>
<td style="text-align:left;">
MGI:5010596
</td>
<td style="text-align:left;">
pseudogene
</td>
<td style="text-align:left;">
predicted gene, 18411
</td>
</tr>
<tr>
<td style="text-align:left;">
7
</td>
<td style="text-align:left;">
gene
</td>
<td style="text-align:right;">
104.64288
</td>
<td style="text-align:right;">
104.64731
</td>
<td style="text-align:left;">
negative
</td>
<td style="text-align:left;">
MGI_C57BL6J_3030492
</td>
<td style="text-align:left;">
Olfr658
</td>
<td style="text-align:left;">
NCBI_Gene:259051,ENSEMBL:ENSMUSG00000070421
</td>
<td style="text-align:left;">
MGI:3030492
</td>
<td style="text-align:left;">
protein coding gene
</td>
<td style="text-align:left;">
olfactory receptor 658
</td>
</tr>
<tr>
<td style="text-align:left;">
7
</td>
<td style="text-align:left;">
pseudogene
</td>
<td style="text-align:right;">
104.64725
</td>
<td style="text-align:right;">
104.65321
</td>
<td style="text-align:left;">
positive
</td>
<td style="text-align:left;">
MGI_C57BL6J_3582591
</td>
<td style="text-align:left;">
Usp17le-ps
</td>
<td style="text-align:left;">
NCBI_Gene:667882,ENSEMBL:ENSMUSG00000082084
</td>
<td style="text-align:left;">
MGI:3582591
</td>
<td style="text-align:left;">
pseudogene
</td>
<td style="text-align:left;">
ubiquitin specific peptidase 17-like E, pseudogene
</td>
</tr>
<tr>
<td style="text-align:left;">
7
</td>
<td style="text-align:left;">
gene
</td>
<td style="text-align:right;">
104.66675
</td>
<td style="text-align:right;">
104.67495
</td>
<td style="text-align:left;">
positive
</td>
<td style="text-align:left;">
MGI_C57BL6J_3030493
</td>
<td style="text-align:left;">
Olfr659
</td>
<td style="text-align:left;">
NCBI_Gene:259052,ENSEMBL:ENSMUSG00000073922
</td>
<td style="text-align:left;">
MGI:3030493
</td>
<td style="text-align:left;">
protein coding gene
</td>
<td style="text-align:left;">
olfactory receptor 659
</td>
</tr>
<tr>
<td style="text-align:left;">
7
</td>
<td style="text-align:left;">
pseudogene
</td>
<td style="text-align:right;">
104.67872
</td>
<td style="text-align:right;">
104.67909
</td>
<td style="text-align:left;">
positive
</td>
<td style="text-align:left;">
MGI_C57BL6J_3030494
</td>
<td style="text-align:left;">
Olfr660-ps1
</td>
<td style="text-align:left;">
NCBI_Gene:257836,ENSEMBL:ENSMUSG00000081694
</td>
<td style="text-align:left;">
MGI:3030494
</td>
<td style="text-align:left;">
pseudogene
</td>
<td style="text-align:left;">
olfactory receptor 660, pseudogene 1
</td>
</tr>
<tr>
<td style="text-align:left;">
7
</td>
<td style="text-align:left;">
gene
</td>
<td style="text-align:right;">
104.68650
</td>
<td style="text-align:right;">
104.69135
</td>
<td style="text-align:left;">
positive
</td>
<td style="text-align:left;">
MGI_C57BL6J_3030495
</td>
<td style="text-align:left;">
Olfr661
</td>
<td style="text-align:left;">
NCBI_Gene:258743,ENSEMBL:ENSMUSG00000073920
</td>
<td style="text-align:left;">
MGI:3030495
</td>
<td style="text-align:left;">
protein coding gene
</td>
<td style="text-align:left;">
olfactory receptor 661
</td>
</tr>
<tr>
<td style="text-align:left;">
7
</td>
<td style="text-align:left;">
pseudogene
</td>
<td style="text-align:right;">
104.69230
</td>
<td style="text-align:right;">
104.69343
</td>
<td style="text-align:left;">
negative
</td>
<td style="text-align:left;">
MGI_C57BL6J_3644209
</td>
<td style="text-align:left;">
Gm6593
</td>
<td style="text-align:left;">
NCBI_Gene:625495,ENSEMBL:ENSMUSG00000109821
</td>
<td style="text-align:left;">
MGI:3644209
</td>
<td style="text-align:left;">
pseudogene
</td>
<td style="text-align:left;">
predicted gene 6593
</td>
</tr>
<tr>
<td style="text-align:left;">
7
</td>
<td style="text-align:left;">
pseudogene
</td>
<td style="text-align:right;">
104.70021
</td>
<td style="text-align:right;">
104.70065
</td>
<td style="text-align:left;">
positive
</td>
<td style="text-align:left;">
MGI_C57BL6J_3030496
</td>
<td style="text-align:left;">
Olfr662-ps1
</td>
<td style="text-align:left;">
NCBI_Gene:404413,ENSEMBL:ENSMUSG00000109832
</td>
<td style="text-align:left;">
MGI:3030496
</td>
<td style="text-align:left;">
pseudogene
</td>
<td style="text-align:left;">
olfactory receptor 662, pseudogene 1
</td>
</tr>
<tr>
<td style="text-align:left;">
7
</td>
<td style="text-align:left;">
pseudogene
</td>
<td style="text-align:right;">
104.70357
</td>
<td style="text-align:right;">
104.70460
</td>
<td style="text-align:left;">
positive
</td>
<td style="text-align:left;">
MGI_C57BL6J_3030497
</td>
<td style="text-align:left;">
Olfr663
</td>
<td style="text-align:left;">
NCBI_Gene:257914,ENSEMBL:ENSMUSG00000109806
</td>
<td style="text-align:left;">
MGI:3030497
</td>
<td style="text-align:left;">
polymorphic pseudogene
</td>
<td style="text-align:left;">
olfactory receptor 663
</td>
</tr>
<tr>
<td style="text-align:left;">
7
</td>
<td style="text-align:left;">
pseudogene
</td>
<td style="text-align:right;">
104.73338
</td>
<td style="text-align:right;">
104.73436
</td>
<td style="text-align:left;">
negative
</td>
<td style="text-align:left;">
MGI_C57BL6J_3030498
</td>
<td style="text-align:left;">
Olfr664
</td>
<td style="text-align:left;">
NCBI_Gene:667918,ENSEMBL:ENSMUSG00000051885
</td>
<td style="text-align:left;">
MGI:3030498
</td>
<td style="text-align:left;">
polymorphic pseudogene
</td>
<td style="text-align:left;">
olfactory receptor 664
</td>
</tr>
<tr>
<td style="text-align:left;">
7
</td>
<td style="text-align:left;">
gene
</td>
<td style="text-align:right;">
104.76796
</td>
<td style="text-align:right;">
104.77747
</td>
<td style="text-align:left;">
negative
</td>
<td style="text-align:left;">
MGI_C57BL6J_107697
</td>
<td style="text-align:left;">
Usp17le
</td>
<td style="text-align:left;">
NCBI_Gene:625530,ENSEMBL:ENSMUSG00000043073
</td>
<td style="text-align:left;">
MGI:107697
</td>
<td style="text-align:left;">
protein coding gene
</td>
<td style="text-align:left;">
ubiquitin specific peptidase 17-like E
</td>
</tr>
<tr>
<td style="text-align:left;">
7
</td>
<td style="text-align:left;">
pseudogene
</td>
<td style="text-align:right;">
104.82115
</td>
<td style="text-align:right;">
104.82228
</td>
<td style="text-align:left;">
positive
</td>
<td style="text-align:left;">
MGI_C57BL6J_3645730
</td>
<td style="text-align:left;">
Gm4887
</td>
<td style="text-align:left;">
NCBI_Gene:233637,ENSEMBL:ENSMUSG00000086229
</td>
<td style="text-align:left;">
MGI:3645730
</td>
<td style="text-align:left;">
pseudogene
</td>
<td style="text-align:left;">
predicted gene 4887
</td>
</tr>
<tr>
<td style="text-align:left;">
7
</td>
<td style="text-align:left;">
gene
</td>
<td style="text-align:right;">
104.84026
</td>
<td style="text-align:right;">
104.84260
</td>
<td style="text-align:left;">
negative
</td>
<td style="text-align:left;">
MGI_C57BL6J_3051498
</td>
<td style="text-align:left;">
Usp17lb
</td>
<td style="text-align:left;">
NCBI_Gene:381944,ENSEMBL:ENSMUSG00000062369
</td>
<td style="text-align:left;">
MGI:3051498
</td>
<td style="text-align:left;">
protein coding gene
</td>
<td style="text-align:left;">
ubiquitin specific peptidase 17-like B
</td>
</tr>
<tr>
<td style="text-align:left;">
7
</td>
<td style="text-align:left;">
gene
</td>
<td style="text-align:right;">
104.85701
</td>
<td style="text-align:right;">
104.86267
</td>
<td style="text-align:left;">
positive
</td>
<td style="text-align:left;">
MGI_C57BL6J_107699
</td>
<td style="text-align:left;">
Usp17la
</td>
<td style="text-align:left;">
NCBI_Gene:13531,ENSEMBL:ENSMUSG00000054568
</td>
<td style="text-align:left;">
MGI:107699
</td>
<td style="text-align:left;">
protein coding gene
</td>
<td style="text-align:left;">
ubiquitin specific peptidase 17-like A
</td>
</tr>
<tr>
<td style="text-align:left;">
7
</td>
<td style="text-align:left;">
gene
</td>
<td style="text-align:right;">
104.86799
</td>
<td style="text-align:right;">
104.87177
</td>
<td style="text-align:left;">
positive
</td>
<td style="text-align:left;">
MGI_C57BL6J_5791392
</td>
<td style="text-align:left;">
Gm45556
</td>
<td style="text-align:left;">
ENSEMBL:ENSMUSG00000109641
</td>
<td style="text-align:left;">
MGI:5791392
</td>
<td style="text-align:left;">
lncRNA gene
</td>
<td style="text-align:left;">
predicted gene 45556
</td>
</tr>
<tr>
<td style="text-align:left;">
7
</td>
<td style="text-align:left;">
gene
</td>
<td style="text-align:right;">
104.87803
</td>
<td style="text-align:right;">
104.88291
</td>
<td style="text-align:left;">
positive
</td>
<td style="text-align:left;">
MGI_C57BL6J_3030499
</td>
<td style="text-align:left;">
Olfr665
</td>
<td style="text-align:left;">
NCBI_Gene:258810,ENSEMBL:ENSMUSG00000073917
</td>
<td style="text-align:left;">
MGI:3030499
</td>
<td style="text-align:left;">
protein coding gene
</td>
<td style="text-align:left;">
olfactory receptor 665
</td>
</tr>
<tr>
<td style="text-align:left;">
7
</td>
<td style="text-align:left;">
gene
</td>
<td style="text-align:right;">
104.89182
</td>
<td style="text-align:right;">
104.89595
</td>
<td style="text-align:left;">
negative
</td>
<td style="text-align:left;">
MGI_C57BL6J_3030500
</td>
<td style="text-align:left;">
Olfr666
</td>
<td style="text-align:left;">
NCBI_Gene:259100,ENSEMBL:ENSMUSG00000063582
</td>
<td style="text-align:left;">
MGI:3030500
</td>
<td style="text-align:left;">
protein coding gene
</td>
<td style="text-align:left;">
olfactory receptor 666
</td>
</tr>
<tr>
<td style="text-align:left;">
7
</td>
<td style="text-align:left;">
gene
</td>
<td style="text-align:right;">
104.90875
</td>
<td style="text-align:right;">
104.91334
</td>
<td style="text-align:left;">
positive
</td>
<td style="text-align:left;">
MGI_C57BL6J_5791108
</td>
<td style="text-align:left;">
Gm45272
</td>
<td style="text-align:left;">
ENSEMBL:ENSMUSG00000109734
</td>
<td style="text-align:left;">
MGI:5791108
</td>
<td style="text-align:left;">
lncRNA gene
</td>
<td style="text-align:left;">
predicted gene 45272
</td>
</tr>
<tr>
<td style="text-align:left;">
7
</td>
<td style="text-align:left;">
pseudogene
</td>
<td style="text-align:right;">
104.91101
</td>
<td style="text-align:right;">
104.91138
</td>
<td style="text-align:left;">
positive
</td>
<td style="text-align:left;">
MGI_C57BL6J_5791111
</td>
<td style="text-align:left;">
Gm45275
</td>
<td style="text-align:left;">
ENSEMBL:ENSMUSG00000110330
</td>
<td style="text-align:left;">
MGI:5791111
</td>
<td style="text-align:left;">
pseudogene
</td>
<td style="text-align:left;">
predicted gene 45275
</td>
</tr>
<tr>
<td style="text-align:left;">
7
</td>
<td style="text-align:left;">
gene
</td>
<td style="text-align:right;">
104.91474
</td>
<td style="text-align:right;">
104.92148
</td>
<td style="text-align:left;">
negative
</td>
<td style="text-align:left;">
MGI_C57BL6J_3030501
</td>
<td style="text-align:left;">
Olfr667
</td>
<td style="text-align:left;">
NCBI_Gene:259062,ENSEMBL:ENSMUSG00000056782
</td>
<td style="text-align:left;">
MGI:3030501
</td>
<td style="text-align:left;">
protein coding gene
</td>
<td style="text-align:left;">
olfactory receptor 667
</td>
</tr>
<tr>
<td style="text-align:left;">
7
</td>
<td style="text-align:left;">
gene
</td>
<td style="text-align:right;">
104.92229
</td>
<td style="text-align:right;">
104.92806
</td>
<td style="text-align:left;">
negative
</td>
<td style="text-align:left;">
MGI_C57BL6J_3030502
</td>
<td style="text-align:left;">
Olfr668
</td>
<td style="text-align:left;">
NCBI_Gene:259061,ENSEMBL:ENSMUSG00000057770
</td>
<td style="text-align:left;">
MGI:3030502
</td>
<td style="text-align:left;">
protein coding gene
</td>
<td style="text-align:left;">
olfactory receptor 668
</td>
</tr>
<tr>
<td style="text-align:left;">
7
</td>
<td style="text-align:left;">
gene
</td>
<td style="text-align:right;">
104.93424
</td>
<td style="text-align:right;">
104.94048
</td>
<td style="text-align:left;">
positive
</td>
<td style="text-align:left;">
MGI_C57BL6J_3030503
</td>
<td style="text-align:left;">
Olfr669
</td>
<td style="text-align:left;">
NCBI_Gene:259045,ENSEMBL:ENSMUSG00000073916
</td>
<td style="text-align:left;">
MGI:3030503
</td>
<td style="text-align:left;">
protein coding gene
</td>
<td style="text-align:left;">
olfactory receptor 669
</td>
</tr>
<tr>
<td style="text-align:left;">
7
</td>
<td style="text-align:left;">
pseudogene
</td>
<td style="text-align:right;">
104.94297
</td>
<td style="text-align:right;">
104.94325
</td>
<td style="text-align:left;">
positive
</td>
<td style="text-align:left;">
MGI_C57BL6J_5791109
</td>
<td style="text-align:left;">
Gm45273
</td>
<td style="text-align:left;">
ENSEMBL:ENSMUSG00000110284
</td>
<td style="text-align:left;">
MGI:5791109
</td>
<td style="text-align:left;">
pseudogene
</td>
<td style="text-align:left;">
predicted gene 45273
</td>
</tr>
<tr>
<td style="text-align:left;">
7
</td>
<td style="text-align:left;">
pseudogene
</td>
<td style="text-align:right;">
104.94918
</td>
<td style="text-align:right;">
104.95043
</td>
<td style="text-align:left;">
negative
</td>
<td style="text-align:left;">
MGI_C57BL6J_3643907
</td>
<td style="text-align:left;">
Gm5900
</td>
<td style="text-align:left;">
NCBI_Gene:545987,ENSEMBL:ENSMUSG00000094685
</td>
<td style="text-align:left;">
MGI:3643907
</td>
<td style="text-align:left;">
pseudogene
</td>
<td style="text-align:left;">
predicted pseudogene 5900
</td>
</tr>
<tr>
<td style="text-align:left;">
7
</td>
<td style="text-align:left;">
gene
</td>
<td style="text-align:right;">
104.95711
</td>
<td style="text-align:right;">
104.96262
</td>
<td style="text-align:left;">
negative
</td>
<td style="text-align:left;">
MGI_C57BL6J_3030504
</td>
<td style="text-align:left;">
Olfr670
</td>
<td style="text-align:left;">
NCBI_Gene:384703,ENSEMBL:ENSMUSG00000044705
</td>
<td style="text-align:left;">
MGI:3030504
</td>
<td style="text-align:left;">
protein coding gene
</td>
<td style="text-align:left;">
olfactory receptor 670
</td>
</tr>
<tr>
<td style="text-align:left;">
7
</td>
<td style="text-align:left;">
gene
</td>
<td style="text-align:right;">
104.97248
</td>
<td style="text-align:right;">
104.97913
</td>
<td style="text-align:left;">
negative
</td>
<td style="text-align:left;">
MGI_C57BL6J_3030505
</td>
<td style="text-align:left;">
Olfr671
</td>
<td style="text-align:left;">
NCBI_Gene:257910,ENSEMBL:ENSMUSG00000094531
</td>
<td style="text-align:left;">
MGI:3030505
</td>
<td style="text-align:left;">
protein coding gene
</td>
<td style="text-align:left;">
olfactory receptor 671
</td>
</tr>
<tr>
<td style="text-align:left;">
7
</td>
<td style="text-align:left;">
gene
</td>
<td style="text-align:right;">
104.99576
</td>
<td style="text-align:right;">
105.00203
</td>
<td style="text-align:left;">
negative
</td>
<td style="text-align:left;">
MGI_C57BL6J_3030506
</td>
<td style="text-align:left;">
Olfr672
</td>
<td style="text-align:left;">
NCBI_Gene:258755,ENSEMBL:ENSMUSG00000051172
</td>
<td style="text-align:left;">
MGI:3030506
</td>
<td style="text-align:left;">
protein coding gene
</td>
<td style="text-align:left;">
olfactory receptor 672
</td>
</tr>
<tr>
<td style="text-align:left;">
7
</td>
<td style="text-align:left;">
pseudogene
</td>
<td style="text-align:right;">
105.00422
</td>
<td style="text-align:right;">
105.00517
</td>
<td style="text-align:left;">
negative
</td>
<td style="text-align:left;">
MGI_C57BL6J_3030507
</td>
<td style="text-align:left;">
Olfr673-ps1
</td>
<td style="text-align:left;">
NCBI_Gene:257774,ENSEMBL:ENSMUSG00000110041
</td>
<td style="text-align:left;">
MGI:3030507
</td>
<td style="text-align:left;">
pseudogene
</td>
<td style="text-align:left;">
olfactory receptor 673, pseudogene 1
</td>
</tr>
<tr>
<td style="text-align:left;">
7
</td>
<td style="text-align:left;">
pseudogene
</td>
<td style="text-align:right;">
105.00631
</td>
<td style="text-align:right;">
105.00726
</td>
<td style="text-align:left;">
negative
</td>
<td style="text-align:left;">
MGI_C57BL6J_3030508
</td>
<td style="text-align:left;">
Olfr674-ps1
</td>
<td style="text-align:left;">
NCBI_Gene:258172,ENSEMBL:ENSMUSG00000109671
</td>
<td style="text-align:left;">
MGI:3030508
</td>
<td style="text-align:left;">
pseudogene
</td>
<td style="text-align:left;">
olfactory receptor 674, pseudogene 1
</td>
</tr>
<tr>
<td style="text-align:left;">
7
</td>
<td style="text-align:left;">
gene
</td>
<td style="text-align:right;">
105.02137
</td>
<td style="text-align:right;">
105.02846
</td>
<td style="text-align:left;">
negative
</td>
<td style="text-align:left;">
MGI_C57BL6J_3030509
</td>
<td style="text-align:left;">
Olfr675
</td>
<td style="text-align:left;">
NCBI_Gene:258147,ENSEMBL:ENSMUSG00000096773
</td>
<td style="text-align:left;">
MGI:3030509
</td>
<td style="text-align:left;">
protein coding gene
</td>
<td style="text-align:left;">
olfactory receptor 675
</td>
</tr>
<tr>
<td style="text-align:left;">
7
</td>
<td style="text-align:left;">
gene
</td>
<td style="text-align:right;">
105.03157
</td>
<td style="text-align:right;">
105.03775
</td>
<td style="text-align:left;">
positive
</td>
<td style="text-align:left;">
MGI_C57BL6J_3030510
</td>
<td style="text-align:left;">
Olfr676
</td>
<td style="text-align:left;">
NCBI_Gene:259099,ENSEMBL:ENSMUSG00000073915
</td>
<td style="text-align:left;">
MGI:3030510
</td>
<td style="text-align:left;">
protein coding gene
</td>
<td style="text-align:left;">
olfactory receptor 676
</td>
</tr>
<tr>
<td style="text-align:left;">
7
</td>
<td style="text-align:left;">
gene
</td>
<td style="text-align:right;">
105.05137
</td>
<td style="text-align:right;">
105.05901
</td>
<td style="text-align:left;">
positive
</td>
<td style="text-align:left;">
MGI_C57BL6J_3030511
</td>
<td style="text-align:left;">
Olfr677
</td>
<td style="text-align:left;">
NCBI_Gene:258355,ENSEMBL:ENSMUSG00000073914
</td>
<td style="text-align:left;">
MGI:3030511
</td>
<td style="text-align:left;">
protein coding gene
</td>
<td style="text-align:left;">
olfactory receptor 677
</td>
</tr>
<tr>
<td style="text-align:left;">
7
</td>
<td style="text-align:left;">
pseudogene
</td>
<td style="text-align:right;">
105.06023
</td>
<td style="text-align:right;">
105.06087
</td>
<td style="text-align:left;">
positive
</td>
<td style="text-align:left;">
MGI_C57BL6J_3646463
</td>
<td style="text-align:left;">
Gm8913
</td>
<td style="text-align:left;">
NCBI_Gene:667987,ENSEMBL:ENSMUSG00000109676
</td>
<td style="text-align:left;">
MGI:3646463
</td>
<td style="text-align:left;">
pseudogene
</td>
<td style="text-align:left;">
predicted gene 8913
</td>
</tr>
<tr>
<td style="text-align:left;">
7
</td>
<td style="text-align:left;">
gene
</td>
<td style="text-align:right;">
105.06448
</td>
<td style="text-align:right;">
105.07164
</td>
<td style="text-align:left;">
positive
</td>
<td style="text-align:left;">
MGI_C57BL6J_3030512
</td>
<td style="text-align:left;">
Olfr678
</td>
<td style="text-align:left;">
NCBI_Gene:258753,ENSEMBL:ENSMUSG00000073913
</td>
<td style="text-align:left;">
MGI:3030512
</td>
<td style="text-align:left;">
protein coding gene
</td>
<td style="text-align:left;">
olfactory receptor 678
</td>
</tr>
<tr>
<td style="text-align:left;">
7
</td>
<td style="text-align:left;">
pseudogene
</td>
<td style="text-align:right;">
105.07692
</td>
<td style="text-align:right;">
105.07818
</td>
<td style="text-align:left;">
positive
</td>
<td style="text-align:left;">
MGI_C57BL6J_5010174
</td>
<td style="text-align:left;">
Gm17989
</td>
<td style="text-align:left;">
NCBI_Gene:100416234,ENSEMBL:ENSMUSG00000109972
</td>
<td style="text-align:left;">
MGI:5010174
</td>
<td style="text-align:left;">
pseudogene
</td>
<td style="text-align:left;">
predicted gene, 17989
</td>
</tr>
<tr>
<td style="text-align:left;">
7
</td>
<td style="text-align:left;">
gene
</td>
<td style="text-align:right;">
105.07843
</td>
<td style="text-align:right;">
105.08008
</td>
<td style="text-align:left;">
negative
</td>
<td style="text-align:left;">
MGI_C57BL6J_5791393
</td>
<td style="text-align:left;">
Gm45557
</td>
<td style="text-align:left;">
ENSEMBL:ENSMUSG00000110293
</td>
<td style="text-align:left;">
MGI:5791393
</td>
<td style="text-align:left;">
lncRNA gene
</td>
<td style="text-align:left;">
predicted gene 45557
</td>
</tr>
<tr>
<td style="text-align:left;">
7
</td>
<td style="text-align:left;">
gene
</td>
<td style="text-align:right;">
105.08120
</td>
<td style="text-align:right;">
105.09064
</td>
<td style="text-align:left;">
positive
</td>
<td style="text-align:left;">
MGI_C57BL6J_3030513
</td>
<td style="text-align:left;">
Olfr679
</td>
<td style="text-align:left;">
NCBI_Gene:259046,ENSEMBL:ENSMUSG00000096029
</td>
<td style="text-align:left;">
MGI:3030513
</td>
<td style="text-align:left;">
protein coding gene
</td>
<td style="text-align:left;">
olfactory receptor 679
</td>
</tr>
<tr>
<td style="text-align:left;">
7
</td>
<td style="text-align:left;">
pseudogene
</td>
<td style="text-align:right;">
105.09070
</td>
<td style="text-align:right;">
105.09282
</td>
<td style="text-align:left;">
negative
</td>
<td style="text-align:left;">
MGI_C57BL6J_3030514
</td>
<td style="text-align:left;">
Olfr680-ps1
</td>
<td style="text-align:left;">
NCBI_Gene:257981,ENSEMBL:ENSMUSG00000041885
</td>
<td style="text-align:left;">
MGI:3030514
</td>
<td style="text-align:left;">
pseudogene
</td>
<td style="text-align:left;">
olfactory receptor 680, pseudogene 1
</td>
</tr>
<tr>
<td style="text-align:left;">
7
</td>
<td style="text-align:left;">
pseudogene
</td>
<td style="text-align:right;">
105.10603
</td>
<td style="text-align:right;">
105.10782
</td>
<td style="text-align:left;">
positive
</td>
<td style="text-align:left;">
MGI_C57BL6J_5010175
</td>
<td style="text-align:left;">
Gm17990
</td>
<td style="text-align:left;">
NCBI_Gene:100416235,ENSEMBL:ENSMUSG00000110117
</td>
<td style="text-align:left;">
MGI:5010175
</td>
<td style="text-align:left;">
pseudogene
</td>
<td style="text-align:left;">
predicted gene, 17990
</td>
</tr>
<tr>
<td style="text-align:left;">
7
</td>
<td style="text-align:left;">
gene
</td>
<td style="text-align:right;">
105.10831
</td>
<td style="text-align:right;">
105.11020
</td>
<td style="text-align:left;">
negative
</td>
<td style="text-align:left;">
MGI_C57BL6J_5791394
</td>
<td style="text-align:left;">
Gm45558
</td>
<td style="text-align:left;">
ENSEMBL:ENSMUSG00000109742
</td>
<td style="text-align:left;">
MGI:5791394
</td>
<td style="text-align:left;">
lncRNA gene
</td>
<td style="text-align:left;">
predicted gene 45558
</td>
</tr>
<tr>
<td style="text-align:left;">
7
</td>
<td style="text-align:left;">
gene
</td>
<td style="text-align:right;">
105.11684
</td>
<td style="text-align:right;">
105.12552
</td>
<td style="text-align:left;">
positive
</td>
<td style="text-align:left;">
MGI_C57BL6J_3030515
</td>
<td style="text-align:left;">
Olfr681
</td>
<td style="text-align:left;">
NCBI_Gene:404318,ENSEMBL:ENSMUSG00000095248
</td>
<td style="text-align:left;">
MGI:3030515
</td>
<td style="text-align:left;">
protein coding gene
</td>
<td style="text-align:left;">
olfactory receptor 681
</td>
</tr>
<tr>
<td style="text-align:left;">
7
</td>
<td style="text-align:left;">
pseudogene
</td>
<td style="text-align:right;">
105.12637
</td>
<td style="text-align:right;">
105.12843
</td>
<td style="text-align:left;">
negative
</td>
<td style="text-align:left;">
MGI_C57BL6J_3030516
</td>
<td style="text-align:left;">
Olfr682-ps1
</td>
<td style="text-align:left;">
NCBI_Gene:258187,ENSEMBL:ENSMUSG00000059768
</td>
<td style="text-align:left;">
MGI:3030516
</td>
<td style="text-align:left;">
polymorphic pseudogene
</td>
<td style="text-align:left;">
olfactory receptor 682, pseudogene 1
</td>
</tr>
<tr>
<td style="text-align:left;">
7
</td>
<td style="text-align:left;">
gene
</td>
<td style="text-align:right;">
105.14084
</td>
<td style="text-align:right;">
105.14701
</td>
<td style="text-align:left;">
negative
</td>
<td style="text-align:left;">
MGI_C57BL6J_3030517
</td>
<td style="text-align:left;">
Olfr683
</td>
<td style="text-align:left;">
NCBI_Gene:259047,ENSEMBL:ENSMUSG00000044120
</td>
<td style="text-align:left;">
MGI:3030517
</td>
<td style="text-align:left;">
protein coding gene
</td>
<td style="text-align:left;">
olfactory receptor 683
</td>
</tr>
<tr>
<td style="text-align:left;">
7
</td>
<td style="text-align:left;">
pseudogene
</td>
<td style="text-align:right;">
105.14541
</td>
<td style="text-align:right;">
105.14555
</td>
<td style="text-align:left;">
negative
</td>
<td style="text-align:left;">
MGI_C57BL6J_5791210
</td>
<td style="text-align:left;">
Gm45374
</td>
<td style="text-align:left;">
ENSEMBL:ENSMUSG00000109918
</td>
<td style="text-align:left;">
MGI:5791210
</td>
<td style="text-align:left;">
pseudogene
</td>
<td style="text-align:left;">
predicted gene 45374
</td>
</tr>
<tr>
<td style="text-align:left;">
7
</td>
<td style="text-align:left;">
gene
</td>
<td style="text-align:right;">
105.15469
</td>
<td style="text-align:right;">
105.16365
</td>
<td style="text-align:left;">
negative
</td>
<td style="text-align:left;">
MGI_C57BL6J_3030518
</td>
<td style="text-align:left;">
Olfr684
</td>
<td style="text-align:left;">
NCBI_Gene:244187,ENSEMBL:ENSMUSG00000047225
</td>
<td style="text-align:left;">
MGI:3030518
</td>
<td style="text-align:left;">
protein coding gene
</td>
<td style="text-align:left;">
olfactory receptor 684
</td>
</tr>
<tr>
<td style="text-align:left;">
7
</td>
<td style="text-align:left;">
gene
</td>
<td style="text-align:right;">
105.17842
</td>
<td style="text-align:right;">
105.18601
</td>
<td style="text-align:left;">
negative
</td>
<td style="text-align:left;">
MGI_C57BL6J_3030519
</td>
<td style="text-align:left;">
Olfr685
</td>
<td style="text-align:left;">
NCBI_Gene:258160,ENSEMBL:ENSMUSG00000047794
</td>
<td style="text-align:left;">
MGI:3030519
</td>
<td style="text-align:left;">
protein coding gene
</td>
<td style="text-align:left;">
olfactory receptor 685
</td>
</tr>
<tr>
<td style="text-align:left;">
7
</td>
<td style="text-align:left;">
gene
</td>
<td style="text-align:right;">
105.20186
</td>
<td style="text-align:right;">
105.20749
</td>
<td style="text-align:left;">
negative
</td>
<td style="text-align:left;">
MGI_C57BL6J_3030520
</td>
<td style="text-align:left;">
Olfr686
</td>
<td style="text-align:left;">
NCBI_Gene:259072,ENSEMBL:ENSMUSG00000048425
</td>
<td style="text-align:left;">
MGI:3030520
</td>
<td style="text-align:left;">
protein coding gene
</td>
<td style="text-align:left;">
olfactory receptor 686
</td>
</tr>
<tr>
<td style="text-align:left;">
7
</td>
<td style="text-align:left;">
gene
</td>
<td style="text-align:right;">
105.22772
</td>
<td style="text-align:right;">
105.27654
</td>
<td style="text-align:left;">
positive
</td>
<td style="text-align:left;">
MGI_C57BL6J_3030521
</td>
<td style="text-align:left;">
Olfr687
</td>
<td style="text-align:left;">
NCBI_Gene:668035,ENSEMBL:ENSMUSG00000073908
</td>
<td style="text-align:left;">
MGI:3030521
</td>
<td style="text-align:left;">
protein coding gene
</td>
<td style="text-align:left;">
olfactory receptor 687
</td>
</tr>
<tr>
<td style="text-align:left;">
7
</td>
<td style="text-align:left;">
gene
</td>
<td style="text-align:right;">
105.28809
</td>
<td style="text-align:right;">
105.28906
</td>
<td style="text-align:left;">
positive
</td>
<td style="text-align:left;">
MGI_C57BL6J_3030522
</td>
<td style="text-align:left;">
Olfr688
</td>
<td style="text-align:left;">
NCBI_Gene:259161,ENSEMBL:ENSMUSG00000073909
</td>
<td style="text-align:left;">
MGI:3030522
</td>
<td style="text-align:left;">
protein coding gene
</td>
<td style="text-align:left;">
olfactory receptor 688
</td>
</tr>
<tr>
<td style="text-align:left;">
7
</td>
<td style="text-align:left;">
gene
</td>
<td style="text-align:right;">
105.31117
</td>
<td style="text-align:right;">
105.31966
</td>
<td style="text-align:left;">
positive
</td>
<td style="text-align:left;">
MGI_C57BL6J_3030523
</td>
<td style="text-align:left;">
Olfr689
</td>
<td style="text-align:left;">
NCBI_Gene:258745,ENSEMBL:ENSMUSG00000073907
</td>
<td style="text-align:left;">
MGI:3030523
</td>
<td style="text-align:left;">
protein coding gene
</td>
<td style="text-align:left;">
olfactory receptor 689
</td>
</tr>
<tr>
<td style="text-align:left;">
7
</td>
<td style="text-align:left;">
pseudogene
</td>
<td style="text-align:right;">
105.32666
</td>
<td style="text-align:right;">
105.32721
</td>
<td style="text-align:left;">
positive
</td>
<td style="text-align:left;">
MGI_C57BL6J_5791167
</td>
<td style="text-align:left;">
Gm45331
</td>
<td style="text-align:left;">
ENSEMBL:ENSMUSG00000110265
</td>
<td style="text-align:left;">
MGI:5791167
</td>
<td style="text-align:left;">
pseudogene
</td>
<td style="text-align:left;">
predicted gene 45331
</td>
</tr>
<tr>
<td style="text-align:left;">
7
</td>
<td style="text-align:left;">
gene
</td>
<td style="text-align:right;">
105.32823
</td>
<td style="text-align:right;">
105.33410
</td>
<td style="text-align:left;">
negative
</td>
<td style="text-align:left;">
MGI_C57BL6J_3030524
</td>
<td style="text-align:left;">
Olfr690
</td>
<td style="text-align:left;">
NCBI_Gene:56860,ENSEMBL:ENSMUSG00000050266
</td>
<td style="text-align:left;">
MGI:3030524
</td>
<td style="text-align:left;">
protein coding gene
</td>
<td style="text-align:left;">
olfactory receptor 690
</td>
</tr>
<tr>
<td style="text-align:left;">
7
</td>
<td style="text-align:left;">
gene
</td>
<td style="text-align:right;">
105.33551
</td>
<td style="text-align:right;">
105.34227
</td>
<td style="text-align:left;">
negative
</td>
<td style="text-align:left;">
MGI_C57BL6J_3030525
</td>
<td style="text-align:left;">
Olfr691
</td>
<td style="text-align:left;">
NCBI_Gene:259063,ENSEMBL:ENSMUSG00000043948
</td>
<td style="text-align:left;">
MGI:3030525
</td>
<td style="text-align:left;">
protein coding gene
</td>
<td style="text-align:left;">
olfactory receptor 691
</td>
</tr>
<tr>
<td style="text-align:left;">
7
</td>
<td style="text-align:left;">
gene
</td>
<td style="text-align:right;">
105.36823
</td>
<td style="text-align:right;">
105.36935
</td>
<td style="text-align:left;">
positive
</td>
<td style="text-align:left;">
MGI_C57BL6J_3030526
</td>
<td style="text-align:left;">
Olfr692
</td>
<td style="text-align:left;">
NCBI_Gene:258352,ENSEMBL:ENSMUSG00000073906
</td>
<td style="text-align:left;">
MGI:3030526
</td>
<td style="text-align:left;">
protein coding gene
</td>
<td style="text-align:left;">
olfactory receptor 692
</td>
</tr>
<tr>
<td style="text-align:left;">
7
</td>
<td style="text-align:left;">
gene
</td>
<td style="text-align:right;">
105.37121
</td>
<td style="text-align:right;">
105.40005
</td>
<td style="text-align:left;">
negative
</td>
<td style="text-align:left;">
MGI_C57BL6J_1921599
</td>
<td style="text-align:left;">
Fam160a2
</td>
<td style="text-align:left;">
NCBI_Gene:74349,ENSEMBL:ENSMUSG00000044465
</td>
<td style="text-align:left;">
MGI:1921599
</td>
<td style="text-align:left;">
protein coding gene
</td>
<td style="text-align:left;">
family with sequence similarity 160, member A2
</td>
</tr>
<tr>
<td style="text-align:left;">
7
</td>
<td style="text-align:left;">
gene
</td>
<td style="text-align:right;">
105.37504
</td>
<td style="text-align:right;">
105.37838
</td>
<td style="text-align:left;">
positive
</td>
<td style="text-align:left;">
MGI_C57BL6J_3643909
</td>
<td style="text-align:left;">
Gm5901
</td>
<td style="text-align:left;">
NCBI_Gene:100503879,ENSEMBL:ENSMUSG00000078611
</td>
<td style="text-align:left;">
MGI:3643909
</td>
<td style="text-align:left;">
protein coding gene
</td>
<td style="text-align:left;">
predicted gene 5901
</td>
</tr>
<tr>
<td style="text-align:left;">
7
</td>
<td style="text-align:left;">
gene
</td>
<td style="text-align:right;">
105.39745
</td>
<td style="text-align:right;">
105.40910
</td>
<td style="text-align:left;">
positive
</td>
<td style="text-align:left;">
MGI_C57BL6J_2664099
</td>
<td style="text-align:left;">
Cnga4
</td>
<td style="text-align:left;">
NCBI_Gene:233649,ENSEMBL:ENSMUSG00000030897
</td>
<td style="text-align:left;">
MGI:2664099
</td>
<td style="text-align:left;">
protein coding gene
</td>
<td style="text-align:left;">
cyclic nucleotide gated channel alpha 4
</td>
</tr>
<tr>
<td style="text-align:left;">
7
</td>
<td style="text-align:left;">
gene
</td>
<td style="text-align:right;">
105.40031
</td>
<td style="text-align:right;">
105.40189
</td>
<td style="text-align:left;">
positive
</td>
<td style="text-align:left;">
MGI_C57BL6J_5805000
</td>
<td style="text-align:left;">
Gm45885
</td>
<td style="text-align:left;">
ENSEMBL:ENSMUSG00000110366
</td>
<td style="text-align:left;">
MGI:5805000
</td>
<td style="text-align:left;">
lncRNA gene
</td>
<td style="text-align:left;">
predicted gene 45885
</td>
</tr>
<tr>
<td style="text-align:left;">
7
</td>
<td style="text-align:left;">
pseudogene
</td>
<td style="text-align:right;">
105.41270
</td>
<td style="text-align:right;">
105.41544
</td>
<td style="text-align:left;">
negative
</td>
<td style="text-align:left;">
MGI_C57BL6J_3643689
</td>
<td style="text-align:left;">
Gm4972
</td>
<td style="text-align:left;">
NCBI_Gene:244189,ENSEMBL:ENSMUSG00000109609
</td>
<td style="text-align:left;">
MGI:3643689
</td>
<td style="text-align:left;">
pseudogene
</td>
<td style="text-align:left;">
predicted gene 4972
</td>
</tr>
<tr>
<td style="text-align:left;">
7
</td>
<td style="text-align:left;">
gene
</td>
<td style="text-align:right;">
105.42556
</td>
<td style="text-align:right;">
105.47090
</td>
<td style="text-align:left;">
positive
</td>
<td style="text-align:left;">
MGI_C57BL6J_99479
</td>
<td style="text-align:left;">
Cckbr
</td>
<td style="text-align:left;">
NCBI_Gene:12426,ENSEMBL:ENSMUSG00000030898
</td>
<td style="text-align:left;">
MGI:99479
</td>
<td style="text-align:left;">
protein coding gene
</td>
<td style="text-align:left;">
cholecystokinin B receptor
</td>
</tr>
<tr>
<td style="text-align:left;">
7
</td>
<td style="text-align:left;">
gene
</td>
<td style="text-align:right;">
105.48008
</td>
<td style="text-align:right;">
105.48230
</td>
<td style="text-align:left;">
negative
</td>
<td style="text-align:left;">
MGI_C57BL6J_1923422
</td>
<td style="text-align:left;">
Cavin3
</td>
<td style="text-align:left;">
NCBI_Gene:109042,ENSEMBL:ENSMUSG00000037060
</td>
<td style="text-align:left;">
MGI:1923422
</td>
<td style="text-align:left;">
protein coding gene
</td>
<td style="text-align:left;">
caveolae associated 3
</td>
</tr>
<tr>
<td style="text-align:left;">
7
</td>
<td style="text-align:left;">
gene
</td>
<td style="text-align:right;">
105.48193
</td>
<td style="text-align:right;">
105.48324
</td>
<td style="text-align:left;">
positive
</td>
<td style="text-align:left;">
MGI_C57BL6J_5791503
</td>
<td style="text-align:left;">
Gm45667
</td>
<td style="text-align:left;">
ENSEMBL:ENSMUSG00000109735
</td>
<td style="text-align:left;">
MGI:5791503
</td>
<td style="text-align:left;">
unclassified gene
</td>
<td style="text-align:left;">
predicted gene 45667
</td>
</tr>
<tr>
<td style="text-align:left;">
7
</td>
<td style="text-align:left;">
gene
</td>
<td style="text-align:right;">
105.48264
</td>
<td style="text-align:right;">
105.52267
</td>
<td style="text-align:left;">
positive
</td>
<td style="text-align:left;">
MGI_C57BL6J_5596006
</td>
<td style="text-align:left;">
Gm36847
</td>
<td style="text-align:left;">
NCBI_Gene:102640889
</td>
<td style="text-align:left;">
MGI:5596006
</td>
<td style="text-align:left;">
lncRNA gene
</td>
<td style="text-align:left;">
predicted gene, 36847
</td>
</tr>
<tr>
<td style="text-align:left;">
7
</td>
<td style="text-align:left;">
gene
</td>
<td style="text-align:right;">
105.54490
</td>
<td style="text-align:right;">
105.55428
</td>
<td style="text-align:left;">
negative
</td>
<td style="text-align:left;">
MGI_C57BL6J_5621948
</td>
<td style="text-align:left;">
Gm39063
</td>
<td style="text-align:left;">
NCBI_Gene:105243022
</td>
<td style="text-align:left;">
MGI:5621948
</td>
<td style="text-align:left;">
lncRNA gene
</td>
<td style="text-align:left;">
predicted gene, 39063
</td>
</tr>
<tr>
<td style="text-align:left;">
7
</td>
<td style="text-align:left;">
gene
</td>
<td style="text-align:right;">
105.55436
</td>
<td style="text-align:right;">
105.55839
</td>
<td style="text-align:left;">
positive
</td>
<td style="text-align:left;">
MGI_C57BL6J_98325
</td>
<td style="text-align:left;">
Smpd1
</td>
<td style="text-align:left;">
NCBI_Gene:20597,ENSEMBL:ENSMUSG00000037049
</td>
<td style="text-align:left;">
MGI:98325
</td>
<td style="text-align:left;">
protein coding gene
</td>
<td style="text-align:left;">
sphingomyelin phosphodiesterase 1, acid lysosomal
</td>
</tr>
<tr>
<td style="text-align:left;">
7
</td>
<td style="text-align:left;">
gene
</td>
<td style="text-align:right;">
105.55846
</td>
<td style="text-align:right;">
105.58192
</td>
<td style="text-align:left;">
negative
</td>
<td style="text-align:left;">
MGI_C57BL6J_107765
</td>
<td style="text-align:left;">
Apbb1
</td>
<td style="text-align:left;">
NCBI_Gene:11785,ENSEMBL:ENSMUSG00000037032
</td>
<td style="text-align:left;">
MGI:107765
</td>
<td style="text-align:left;">
protein coding gene
</td>
<td style="text-align:left;">
amyloid beta (A4) precursor protein-binding, family B, member 1
</td>
</tr>
<tr>
<td style="text-align:left;">
7
</td>
<td style="text-align:left;">
gene
</td>
<td style="text-align:right;">
105.59161
</td>
<td style="text-align:right;">
105.60014
</td>
<td style="text-align:left;">
negative
</td>
<td style="text-align:left;">
MGI_C57BL6J_105112
</td>
<td style="text-align:left;">
Hpx
</td>
<td style="text-align:left;">
NCBI_Gene:15458,ENSEMBL:ENSMUSG00000030895
</td>
<td style="text-align:left;">
MGI:105112
</td>
<td style="text-align:left;">
protein coding gene
</td>
<td style="text-align:left;">
hemopexin
</td>
</tr>
<tr>
<td style="text-align:left;">
7
</td>
<td style="text-align:left;">
gene
</td>
<td style="text-align:right;">
105.60446
</td>
<td style="text-align:right;">
105.63357
</td>
<td style="text-align:left;">
negative
</td>
<td style="text-align:left;">
MGI_C57BL6J_1860040
</td>
<td style="text-align:left;">
Trim3
</td>
<td style="text-align:left;">
NCBI_Gene:55992,ENSEMBL:ENSMUSG00000036989
</td>
<td style="text-align:left;">
MGI:1860040
</td>
<td style="text-align:left;">
protein coding gene
</td>
<td style="text-align:left;">
tripartite motif-containing 3
</td>
</tr>
<tr>
<td style="text-align:left;">
7
</td>
<td style="text-align:left;">
gene
</td>
<td style="text-align:right;">
105.62282
</td>
<td style="text-align:right;">
105.62470
</td>
<td style="text-align:left;">
negative
</td>
<td style="text-align:left;">
MGI_C57BL6J_5791468
</td>
<td style="text-align:left;">
Gm45632
</td>
<td style="text-align:left;">
ENSEMBL:ENSMUSG00000110077
</td>
<td style="text-align:left;">
MGI:5791468
</td>
<td style="text-align:left;">
unclassified gene
</td>
<td style="text-align:left;">
predicted gene 45632
</td>
</tr>
<tr>
<td style="text-align:left;">
7
</td>
<td style="text-align:left;">
gene
</td>
<td style="text-align:right;">
105.63420
</td>
<td style="text-align:right;">
105.64042
</td>
<td style="text-align:left;">
negative
</td>
<td style="text-align:left;">
MGI_C57BL6J_1924182
</td>
<td style="text-align:left;">
Arfip2
</td>
<td style="text-align:left;">
NCBI_Gene:76932,ENSEMBL:ENSMUSG00000030881
</td>
<td style="text-align:left;">
MGI:1924182
</td>
<td style="text-align:left;">
protein coding gene
</td>
<td style="text-align:left;">
ADP-ribosylation factor interacting protein 2
</td>
</tr>
<tr>
<td style="text-align:left;">
7
</td>
<td style="text-align:left;">
gene
</td>
<td style="text-align:right;">
105.64006
</td>
<td style="text-align:right;">
105.65699
</td>
<td style="text-align:left;">
positive
</td>
<td style="text-align:left;">
MGI_C57BL6J_1315196
</td>
<td style="text-align:left;">
Timm10b
</td>
<td style="text-align:left;">
NCBI_Gene:14356,ENSEMBL:ENSMUSG00000089847
</td>
<td style="text-align:left;">
MGI:1315196
</td>
<td style="text-align:left;">
protein coding gene
</td>
<td style="text-align:left;">
translocase of inner mitochondrial membrane 10B
</td>
</tr>
<tr>
<td style="text-align:left;">
7
</td>
<td style="text-align:left;">
gene
</td>
<td style="text-align:right;">
105.64055
</td>
<td style="text-align:right;">
105.68444
</td>
<td style="text-align:left;">
positive
</td>
<td style="text-align:left;">
MGI_C57BL6J_5804914
</td>
<td style="text-align:left;">
Gm45799
</td>
<td style="text-align:left;">
ENSEMBL:ENSMUSG00000110234
</td>
<td style="text-align:left;">
MGI:5804914
</td>
<td style="text-align:left;">
protein coding gene
</td>
<td style="text-align:left;">
predicted gene 45799
</td>
</tr>
<tr>
<td style="text-align:left;">
7
</td>
<td style="text-align:left;">
gene
</td>
<td style="text-align:right;">
105.64234
</td>
<td style="text-align:right;">
105.64248
</td>
<td style="text-align:left;">
negative
</td>
<td style="text-align:left;">
MGI_C57BL6J_5452281
</td>
<td style="text-align:left;">
Gm22504
</td>
<td style="text-align:left;">
ENSEMBL:ENSMUSG00000064897
</td>
<td style="text-align:left;">
MGI:5452281
</td>
<td style="text-align:left;">
snoRNA gene
</td>
<td style="text-align:left;">
predicted gene, 22504
</td>
</tr>
<tr>
<td style="text-align:left;">
7
</td>
<td style="text-align:left;">
gene
</td>
<td style="text-align:right;">
105.65081
</td>
<td style="text-align:right;">
105.72180
</td>
<td style="text-align:left;">
positive
</td>
<td style="text-align:left;">
MGI_C57BL6J_1924755
</td>
<td style="text-align:left;">
Dnhd1
</td>
<td style="text-align:left;">
NCBI_Gene:77505,ENSEMBL:ENSMUSG00000030882
</td>
<td style="text-align:left;">
MGI:1924755
</td>
<td style="text-align:left;">
protein coding gene
</td>
<td style="text-align:left;">
dynein heavy chain domain 1
</td>
</tr>
</tbody>
</table>
</div>
<div id="rqtl" class="section level2">
<h2>R/qtl</h2>
<div id="scanone" class="section level3">
<h3>scanone</h3>
<br>
<p>
<button type="button" class="btn btn-default btn-workflowr btn-workflowr-sessioninfo" data-toggle="collapse" data-target="#workflowr-sessioninfo" style="display: block;">
<span class="glyphicon glyphicon-wrench" aria-hidden="true"></span> Session information
</button>
</p>
<div id="workflowr-sessioninfo" class="collapse">
<pre><code>R version 4.0.3 (2020-10-10)
Platform: x86_64-pc-linux-gnu (64-bit)
Running under: Ubuntu 20.04.5 LTS

Matrix products: default
BLAS:   /usr/lib/x86_64-linux-gnu/openblas-pthread/libblas.so.3
LAPACK: /usr/lib/x86_64-linux-gnu/openblas-pthread/liblapack.so.3

locale:
 [1] LC_CTYPE=en_US.UTF-8       LC_NUMERIC=C              
 [3] LC_TIME=en_US.UTF-8        LC_COLLATE=en_US.UTF-8    
 [5] LC_MONETARY=en_US.UTF-8    LC_MESSAGES=en_US.UTF-8   
 [7] LC_PAPER=en_US.UTF-8       LC_NAME=C                 
 [9] LC_ADDRESS=C               LC_TELEPHONE=C            
[11] LC_MEASUREMENT=en_US.UTF-8 LC_IDENTIFICATION=C       

attached base packages:
[1] stats     graphics  grDevices utils     datasets  methods   base     

other attached packages:
 [1] abind_1.4-5       qtl2_0.24         reshape2_1.4.4    ggplot2_3.3.3    
 [5] tibble_3.0.6      psych_2.0.12      readxl_1.3.1      cluster_2.1.0    
 [9] dplyr_1.0.4       optparse_1.6.6    rhdf5_2.34.0      mclust_5.4.7     
[13] tidyr_1.1.2       data.table_1.13.6 knitr_1.31        kableExtra_1.3.1 
[17] workflowr_1.6.2  

loaded via a namespace (and not attached):
 [1] Rcpp_1.0.10        lattice_0.20-41    assertthat_0.2.1   rprojroot_2.0.2   
 [5] digest_0.6.27      R6_2.5.0           cellranger_1.1.0   plyr_1.8.6        
 [9] RSQLite_2.2.3      evaluate_0.14      httr_1.4.2         highr_0.8         
[13] pillar_1.4.7       rlang_0.4.10       rstudioapi_0.13    blob_1.2.1        
[17] rmarkdown_2.6      webshot_0.5.2      stringr_1.4.0      bit_4.0.4         
[21] munsell_0.5.0      compiler_4.0.3     httpuv_1.5.5       xfun_0.21         
[25] pkgconfig_2.0.3    mnormt_2.0.2       tmvnsim_1.0-2      htmltools_0.5.1.1 
[29] tidyselect_1.1.0   viridisLite_0.3.0  crayon_1.4.1       withr_2.4.1       
[33] later_1.1.0.1      rhdf5filters_1.2.1 grid_4.0.3         nlme_3.1-152      
[37] gtable_0.3.0       lifecycle_0.2.0    DBI_1.1.1          git2r_0.28.0      
[41] magrittr_2.0.1     scales_1.1.1       cachem_1.0.3       stringi_1.5.3     
[45] fs_1.5.0           promises_1.1.1     getopt_1.20.3      xml2_1.3.2        
[49] ellipsis_0.3.1     generics_0.1.0     vctrs_0.3.6        Rhdf5lib_1.12.1   
[53] tools_4.0.3        bit64_4.0.5        glue_1.4.2         purrr_0.3.4       
[57] fastmap_1.1.0      parallel_4.0.3     yaml_2.2.1         colorspace_2.0-0  
[61] rvest_0.3.6        memoise_2.0.0     </code></pre>
</div>
</div>
</div>


<!-- Adjust MathJax settings so that all math formulae are shown using
TeX fonts only; see
http://docs.mathjax.org/en/latest/configuration.html.  This will make
the presentation more consistent at the cost of the webpage sometimes
taking slightly longer to load. Note that this only works because the
footer is added to webpages before the MathJax javascript. -->
<script type="text/x-mathjax-config">
  MathJax.Hub.Config({
    "HTML-CSS": { availableFonts: ["TeX"] }
  });
</script>





</div>

<script>

// add bootstrap table styles to pandoc tables
function bootstrapStylePandocTables() {
  $('tr.odd').parent('tbody').parent('table').addClass('table table-condensed');
}
$(document).ready(function () {
  bootstrapStylePandocTables();
});


</script>

<!-- tabsets -->

<script>
$(document).ready(function () {
  window.buildTabsets("TOC");
});

$(document).ready(function () {
  $('.tabset-dropdown > .nav-tabs > li').click(function () {
    $(this).parent().toggleClass('nav-tabs-open')
  });
});
</script>

<!-- code folding -->


<!-- dynamically load mathjax for compatibility with self-contained -->
<script>
  (function () {
    var script = document.createElement("script");
    script.type = "text/javascript";
    script.src  = "https://mathjax.rstudio.com/latest/MathJax.js?config=TeX-AMS-MML_HTMLorMML";
    document.getElementsByTagName("head")[0].appendChild(script);
  })();
</script>

</body>
</html>
