<!DOCTYPE html>

<html>

<head>

<meta charset="utf-8" />
<meta name="generator" content="pandoc" />
<meta http-equiv="X-UA-Compatible" content="IE=EDGE" />


<meta name="author" content="Belinda Cornes" />

<meta name="date" content="2022-04-14" />

<title>QTL Analysis - rz transformed age of onset (rz.age) [ICI vs EOI] (corrected phenotype with outliers removed if any)</title>

<script src="site_libs/header-attrs-2.1/header-attrs.js"></script>
<script src="site_libs/jquery-1.11.3/jquery.min.js"></script>
<meta name="viewport" content="width=device-width, initial-scale=1" />
<link href="site_libs/bootstrap-3.3.5/css/spacelab.min.css" rel="stylesheet" />
<script src="site_libs/bootstrap-3.3.5/js/bootstrap.min.js"></script>
<script src="site_libs/bootstrap-3.3.5/shim/html5shiv.min.js"></script>
<script src="site_libs/bootstrap-3.3.5/shim/respond.min.js"></script>
<script src="site_libs/navigation-1.1/tabsets.js"></script>
<link href="site_libs/highlightjs-9.12.0/textmate.css" rel="stylesheet" />
<script src="site_libs/highlightjs-9.12.0/highlight.js"></script>
<script src="site_libs/kePrint-0.0.1/kePrint.js"></script>
<link href="site_libs/font-awesome-5.1.0/css/all.css" rel="stylesheet" />
<link href="site_libs/font-awesome-5.1.0/css/v4-shims.css" rel="stylesheet" />

<link rel="icon" href="https://github.com/workflowr/workflowr-assets/raw/master/img/reproducible.png">
<!-- Add a small amount of space between sections. -->
<style type="text/css">
div.section {
  padding-top: 12px;
}
</style>



<style type="text/css">code{white-space: pre;}</style>
<style type="text/css">
  pre:not([class]) {
    background-color: white;
  }
</style>
<script type="text/javascript">
if (window.hljs) {
  hljs.configure({languages: []});
  hljs.initHighlightingOnLoad();
  if (document.readyState && document.readyState === "complete") {
    window.setTimeout(function() { hljs.initHighlighting(); }, 0);
  }
}
</script>



<style type="text/css">
h1 {
  font-size: 34px;
}
h1.title {
  font-size: 38px;
}
h2 {
  font-size: 30px;
}
h3 {
  font-size: 24px;
}
h4 {
  font-size: 18px;
}
h5 {
  font-size: 16px;
}
h6 {
  font-size: 12px;
}
.table th:not([align]) {
  text-align: left;
}
</style>




<style type = "text/css">
.main-container {
  max-width: 940px;
  margin-left: auto;
  margin-right: auto;
}
code {
  color: inherit;
  background-color: rgba(0, 0, 0, 0.04);
}
img {
  max-width:100%;
}
.tabbed-pane {
  padding-top: 12px;
}
.html-widget {
  margin-bottom: 20px;
}
button.code-folding-btn:focus {
  outline: none;
}
summary {
  display: list-item;
}
</style>


<style type="text/css">
/* padding for bootstrap navbar */
body {
  padding-top: 51px;
  padding-bottom: 40px;
}
/* offset scroll position for anchor links (for fixed navbar)  */
.section h1 {
  padding-top: 56px;
  margin-top: -56px;
}
.section h2 {
  padding-top: 56px;
  margin-top: -56px;
}
.section h3 {
  padding-top: 56px;
  margin-top: -56px;
}
.section h4 {
  padding-top: 56px;
  margin-top: -56px;
}
.section h5 {
  padding-top: 56px;
  margin-top: -56px;
}
.section h6 {
  padding-top: 56px;
  margin-top: -56px;
}
.dropdown-submenu {
  position: relative;
}
.dropdown-submenu>.dropdown-menu {
  top: 0;
  left: 100%;
  margin-top: -6px;
  margin-left: -1px;
  border-radius: 0 6px 6px 6px;
}
.dropdown-submenu:hover>.dropdown-menu {
  display: block;
}
.dropdown-submenu>a:after {
  display: block;
  content: " ";
  float: right;
  width: 0;
  height: 0;
  border-color: transparent;
  border-style: solid;
  border-width: 5px 0 5px 5px;
  border-left-color: #cccccc;
  margin-top: 5px;
  margin-right: -10px;
}
.dropdown-submenu:hover>a:after {
  border-left-color: #ffffff;
}
.dropdown-submenu.pull-left {
  float: none;
}
.dropdown-submenu.pull-left>.dropdown-menu {
  left: -100%;
  margin-left: 10px;
  border-radius: 6px 0 6px 6px;
}
</style>

<script>
// manage active state of menu based on current page
$(document).ready(function () {
  // active menu anchor
  href = window.location.pathname
  href = href.substr(href.lastIndexOf('/') + 1)
  if (href === "")
    href = "index.html";
  var menuAnchor = $('a[href="' + href + '"]');

  // mark it active
  menuAnchor.parent().addClass('active');

  // if it's got a parent navbar menu mark it active as well
  menuAnchor.closest('li.dropdown').addClass('active');
});
</script>

<!-- tabsets -->

<style type="text/css">
.tabset-dropdown > .nav-tabs {
  display: inline-table;
  max-height: 500px;
  min-height: 44px;
  overflow-y: auto;
  background: white;
  border: 1px solid #ddd;
  border-radius: 4px;
}

.tabset-dropdown > .nav-tabs > li.active:before {
  content: "";
  font-family: 'Glyphicons Halflings';
  display: inline-block;
  padding: 10px;
  border-right: 1px solid #ddd;
}

.tabset-dropdown > .nav-tabs.nav-tabs-open > li.active:before {
  content: "&#xe258;";
  border: none;
}

.tabset-dropdown > .nav-tabs.nav-tabs-open:before {
  content: "";
  font-family: 'Glyphicons Halflings';
  display: inline-block;
  padding: 10px;
  border-right: 1px solid #ddd;
}

.tabset-dropdown > .nav-tabs > li.active {
  display: block;
}

.tabset-dropdown > .nav-tabs > li > a,
.tabset-dropdown > .nav-tabs > li > a:focus,
.tabset-dropdown > .nav-tabs > li > a:hover {
  border: none;
  display: inline-block;
  border-radius: 4px;
  background-color: transparent;
}

.tabset-dropdown > .nav-tabs.nav-tabs-open > li {
  display: block;
  float: none;
}

.tabset-dropdown > .nav-tabs > li {
  display: none;
}
</style>

<!-- code folding -->




</head>

<body>


<div class="container-fluid main-container">




<div class="navbar navbar-default  navbar-fixed-top" role="navigation">
  <div class="container">
    <div class="navbar-header">
      <button type="button" class="navbar-toggle collapsed" data-toggle="collapse" data-target="#navbar">
        <span class="icon-bar"></span>
        <span class="icon-bar"></span>
        <span class="icon-bar"></span>
      </button>
      <a class="navbar-brand" href="index.html">Serreze T1D Workflow</a>
    </div>
    <div id="navbar" class="navbar-collapse collapse">
      <ul class="nav navbar-nav">
        <li>
  <a href="index.html">Home</a>
</li>
<li>
  <a href="about.html">About</a>
</li>
<li>
  <a href="license.html">License</a>
</li>
      </ul>
      <ul class="nav navbar-nav navbar-right">
        <li>
  <a href="https://github.com/TheJacksonLaboratory/Serreze-T1D_Workflow">
    <span class="fa fa-github"></span>
     
    Source code
  </a>
</li>
      </ul>
    </div><!--/.nav-collapse -->
  </div><!--/.container -->
</div><!--/.navbar -->

<div class="fluid-row" id="header">



<h1 class="title toc-ignore">QTL Analysis - rz transformed age of onset (rz.age) [ICI vs EOI] (corrected phenotype with outliers removed if any)</h1>
<h4 class="author">Belinda Cornes</h4>
<h4 class="date">2022-04-14</h4>

</div>

<div id="TOC">
<ul>
<li><a href="#loading-data">Loading Data</a></li>
<li><a href="#preparing-necessary-info">Preparing necessary info</a>
<ul>
<li><a href="#no-additive-covariates">No additive covariates</a>
<ul>
<li><a href="#significance-thresholds">Significance thresholds</a></li>
<li><a href="#genome-wide-scan">Genome-wide scan</a></li>
<li><a href="#lod-peaks">LOD peaks</a></li>
<li><a href="#qtl-effects">QTL effects</a></li>
</ul></li>
<li><a href="#additive-covariates">Additive covariates</a>
<ul>
<li><a href="#significance-thresholds-1">Significance thresholds</a></li>
<li><a href="#genome-wide-scan-1">Genome-wide scan</a></li>
<li><a href="#lod-peaks-1">LOD peaks</a></li>
<li><a href="#qtl-effects-1">QTL effects</a></li>
</ul></li>
<li><a href="#interactive-term">Interactive term</a>
<ul>
<li><a href="#significance-thresholds-2">Significance thresholds</a></li>
<li><a href="#genome-wide-scan-2">Genome-wide scan</a></li>
<li><a href="#lod-peaks-2">LOD peaks</a></li>
<li><a href="#qtl-effects-2">QTL effects</a></li>
</ul></li>
</ul></li>
</ul>
</div>

<p>
<button type="button" class="btn btn-default btn-workflowr btn-workflowr-report" data-toggle="collapse" data-target="#workflowr-report">
<span class="glyphicon glyphicon-list" aria-hidden="true"></span> workflowr <span class="glyphicon glyphicon-exclamation-sign text-danger" aria-hidden="true"></span>
</button>
</p>
<div id="workflowr-report" class="collapse">
<ul class="nav nav-tabs">
<li class="active">
<a data-toggle="tab" href="#summary">Summary</a>
</li>
<li>
<a data-toggle="tab" href="#checks"> Checks <span class="glyphicon glyphicon-exclamation-sign text-danger" aria-hidden="true"></span> </a>
</li>
<li>
<a data-toggle="tab" href="#versions">Past versions</a>
</li>
</ul>
<div class="tab-content">
<div id="summary" class="tab-pane fade in active">
<p>
<strong>Last updated:</strong> 2022-04-14
</p>
<p>
<strong>Checks:</strong> <span class="glyphicon glyphicon-ok text-success" aria-hidden="true"></span> 5 <span class="glyphicon glyphicon-exclamation-sign text-danger" aria-hidden="true"></span> 2
</p>
<p>
<strong>Knit directory:</strong> <code>Serreze-T1D_Workflow/</code> <span class="glyphicon glyphicon-question-sign" aria-hidden="true" title="This is the local directory in which the code in this file was executed."> </span>
</p>
<p>
This reproducible <a href="http://rmarkdown.rstudio.com">R Markdown</a> analysis was created with <a
  href="https://github.com/jdblischak/workflowr">workflowr</a> (version 1.6.2). The <em>Checks</em> tab describes the reproducibility checks that were applied when the results were created. The <em>Past versions</em> tab lists the development history.
</p>
<hr>
</div>
<div id="checks" class="tab-pane fade">
<div id="workflowr-checks" class="panel-group">
<div class="panel panel-default">
<div class="panel-heading">
<p class="panel-title">
<a data-toggle="collapse" data-parent="#workflowr-checks" href="#strongRMarkdownfilestronguncommittedchanges"> <span class="glyphicon glyphicon-exclamation-sign text-danger" aria-hidden="true"></span> <strong>R Markdown file:</strong> uncommitted changes </a>
</p>
</div>
<div id="strongRMarkdownfilestronguncommittedchanges" class="panel-collapse collapse">
<div class="panel-body">
<p>The R Markdown is untracked by Git. To know which version of the R Markdown file created these results, you’ll want to first commit it to the Git repo. If you’re still working on the analysis, you can ignore this warning. When you’re finished, you can run <code>wflow_publish</code> to commit the R Markdown file and build the HTML.</p>
</div>
</div>
</div>
<div class="panel panel-default">
<div class="panel-heading">
<p class="panel-title">
<a data-toggle="collapse" data-parent="#workflowr-checks" href="#strongEnvironmentstrongempty"> <span class="glyphicon glyphicon-ok text-success" aria-hidden="true"></span> <strong>Environment:</strong> empty </a>
</p>
</div>
<div id="strongEnvironmentstrongempty" class="panel-collapse collapse">
<div class="panel-body">
<p>Great job! The global environment was empty. Objects defined in the global environment can affect the analysis in your R Markdown file in unknown ways. For reproduciblity it’s best to always run the code in an empty environment.</p>
</div>
</div>
</div>
<div class="panel panel-default">
<div class="panel-heading">
<p class="panel-title">
<a data-toggle="collapse" data-parent="#workflowr-checks" href="#strongSeedstrongcodesetseed20220210code"> <span class="glyphicon glyphicon-ok text-success" aria-hidden="true"></span> <strong>Seed:</strong> <code>set.seed(20220210)</code> </a>
</p>
</div>
<div id="strongSeedstrongcodesetseed20220210code" class="panel-collapse collapse">
<div class="panel-body">
<p>The command <code>set.seed(20220210)</code> was run prior to running the code in the R Markdown file. Setting a seed ensures that any results that rely on randomness, e.g. subsampling or permutations, are reproducible.</p>
</div>
</div>
</div>
<div class="panel panel-default">
<div class="panel-heading">
<p class="panel-title">
<a data-toggle="collapse" data-parent="#workflowr-checks" href="#strongSessioninformationstrongrecorded"> <span class="glyphicon glyphicon-ok text-success" aria-hidden="true"></span> <strong>Session information:</strong> recorded </a>
</p>
</div>
<div id="strongSessioninformationstrongrecorded" class="panel-collapse collapse">
<div class="panel-body">
<p>Great job! Recording the operating system, R version, and package versions is critical for reproducibility.</p>
</div>
</div>
</div>
<div class="panel panel-default">
<div class="panel-heading">
<p class="panel-title">
<a data-toggle="collapse" data-parent="#workflowr-checks" href="#strongCachestrongnone"> <span class="glyphicon glyphicon-ok text-success" aria-hidden="true"></span> <strong>Cache:</strong> none </a>
</p>
</div>
<div id="strongCachestrongnone" class="panel-collapse collapse">
<div class="panel-body">
<p>Nice! There were no cached chunks for this analysis, so you can be confident that you successfully produced the results during this run.</p>
</div>
</div>
</div>
<div class="panel panel-default">
<div class="panel-heading">
<p class="panel-title">
<a data-toggle="collapse" data-parent="#workflowr-checks" href="#strongFilepathsstrongabsolute"> <span class="glyphicon glyphicon-exclamation-sign text-danger" aria-hidden="true"></span> <strong>File paths:</strong> absolute </a>
</p>
</div>
<div id="strongFilepathsstrongabsolute" class="panel-collapse collapse">
<div class="panel-body">
<p>
Using absolute paths to the files within your workflowr project makes it difficult for you and others to run your code on a different machine. Change the absolute path(s) below to the suggested relative path(s) to make your code more reproducible.
</p>
<table class="table table-condensed table-hover">
<thead>
<tr>
<th style="text-align:left;">
absolute
</th>
<th style="text-align:left;">
relative
</th>
</tr>
</thead>
<tbody>
<tr>
<td style="text-align:left;">
/Users/corneb/Documents/MyJax/CS/Projects/Serreze/qc/workflowr/Serreze-T1D_Workflow
</td>
<td style="text-align:left;">
.
</td>
</tr>
</tbody>
</table>
</div>
</div>
</div>
<div class="panel panel-default">
<div class="panel-heading">
<p class="panel-title">
<a data-toggle="collapse" data-parent="#workflowr-checks" href="#strongRepositoryversionstrongahrefhttpsgithubcomTheJacksonLaboratorySerrezeT1DWorkflowtreea8c2d1a11f223361b54952ce8455e8c17017c914targetblanka8c2d1aa"> <span class="glyphicon glyphicon-ok text-success" aria-hidden="true"></span> <strong>Repository version:</strong> <a href="https://github.com/TheJacksonLaboratory/Serreze-T1D_Workflow/tree/a8c2d1a11f223361b54952ce8455e8c17017c914" target="_blank">a8c2d1a</a> </a>
</p>
</div>
<div id="strongRepositoryversionstrongahrefhttpsgithubcomTheJacksonLaboratorySerrezeT1DWorkflowtreea8c2d1a11f223361b54952ce8455e8c17017c914targetblanka8c2d1aa" class="panel-collapse collapse">
<div class="panel-body">
<p>
Great! You are using Git for version control. Tracking code development and connecting the code version to the results is critical for reproducibility.
</p>
<p>
The results in this page were generated with repository version <a href="https://github.com/TheJacksonLaboratory/Serreze-T1D_Workflow/tree/a8c2d1a11f223361b54952ce8455e8c17017c914" target="_blank">a8c2d1a</a>. See the <em>Past versions</em> tab to see a history of the changes made to the R Markdown and HTML files.
</p>
<p>
Note that you need to be careful to ensure that all relevant files for the analysis have been committed to Git prior to generating the results (you can use <code>wflow_publish</code> or <code>wflow_git_commit</code>). workflowr only checks the R Markdown file, but you know if there are other scripts or data files that it depends on. Below is the status of the Git repository when the results were generated:
</p>
<pre><code>
Ignored files:
    Ignored:    .DS_Store
    Ignored:    analysis/.DS_Store
    Ignored:    data/.DS_Store

Untracked files:
    Untracked:  analysis/3.1_phenotype.qc_corrected_5.batches_0.Rmd
    Untracked:  analysis/3.1_phenotype.qc_corrected_5.batches_52.Rmd
    Untracked:  analysis/3.1_phenotype.qc_corrected_5.batches_mis_0.Rmd
    Untracked:  analysis/3.1_phenotype.qc_corrected_5.batches_mis_52.Rmd
    Untracked:  analysis/3.1_phenotype.qc_corrected_5.batches_mis_vo.Rmd
    Untracked:  analysis/3.1_phenotype.qc_corrected_5.batches_vo.Rmd
    Untracked:  analysis/4.1.1_qtl.analysis_binary_ici.vs.eoi_snpsqc_dis_no-x_updated_5.batches_11.Rmd
    Untracked:  analysis/4.1.1_qtl.analysis_binary_ici.vs.eoi_snpsqc_dis_no-x_updated_5.batches_mis_11.Rmd
    Untracked:  analysis/4.1.1_qtl.analysis_binary_ici.vs.pbs_snpsqc_5.batches_1.Rmd
    Untracked:  analysis/4.1.1_qtl.analysis_binary_ici.vs.pbs_snpsqc_5.batches_mis_1.Rmd
    Untracked:  analysis/4.1.1_qtl.analysis_binary_ici.vs.pbs_snpsqc_dis_no-x_updated_5.batches_1.Rmd
    Untracked:  analysis/4.1.1_qtl.analysis_binary_ici.vs.pbs_snpsqc_dis_no-x_updated_5.batches_11.Rmd
    Untracked:  analysis/4.1.1_qtl.analysis_binary_ici.vs.pbs_snpsqc_dis_no-x_updated_5.batches_mis_11.Rmd
    Untracked:  analysis/4.1.2_qtl.analysis_cont_age_ici.vs.eoi_pheno.corrected.cleaned_5.batches_0.Rmd
    Untracked:  analysis/4.1.2_qtl.analysis_cont_age_ici.vs.eoi_pheno.corrected.cleaned_5.batches_52.Rmd
    Untracked:  analysis/4.1.2_qtl.analysis_cont_age_ici.vs.eoi_pheno.corrected.cleaned_5.batches_changed.Rmd
    Untracked:  analysis/4.1.2_qtl.analysis_cont_age_ici.vs.eoi_pheno.corrected.cleaned_5.batches_mis_0.Rmd
    Untracked:  analysis/4.1.2_qtl.analysis_cont_age_ici.vs.eoi_pheno.corrected.cleaned_5.batches_mis_52.Rmd
    Untracked:  analysis/4.1.2_qtl.analysis_cont_age_ici.vs.eoi_pheno.corrected.cleaned_5.batches_mis_changed.Rmd
    Untracked:  analysis/4.1.2_qtl.analysis_cont_age_ici.vs.eoi_pheno.corrected.cleaned_scanone_5.batches.Rmd
    Untracked:  analysis/4.1.2_qtl.analysis_cont_age_ici.vs.eoi_pheno.corrected.cleaned_scanone_5.batches_mis.Rmd
    Untracked:  analysis/4.1.2_qtl.analysis_cont_age_ici.vs.eoi_snpsqc_pheno.corrected.cleaned_5.batches_0.Rmd
    Untracked:  analysis/4.1.2_qtl.analysis_cont_age_ici.vs.eoi_snpsqc_pheno.corrected.cleaned_5.batches_52.Rmd
    Untracked:  analysis/4.1.2_qtl.analysis_cont_age_ici.vs.eoi_snpsqc_pheno.corrected.cleaned_5.batches_mis_0.Rmd
    Untracked:  analysis/4.1.2_qtl.analysis_cont_age_ici.vs.eoi_snpsqc_pheno.corrected.cleaned_5.batches_mis_52.Rmd
    Untracked:  analysis/4.1.2_qtl.analysis_cont_age_ici.vs.eoi_snpsqc_pheno.corrected.cleaned_5.batches_oops.Rmd
    Untracked:  analysis/4.1.2_qtl.analysis_cont_age_ici.vs.eoi_snpsqc_pheno.corrected.cleaned_dis.Rmd.R
    Untracked:  analysis/4.1.2_qtl.analysis_cont_age_ici.vs.eoi_snpsqc_pheno.corrected.cleaned_dis_no-xk_5.batches_0.Rmd
    Untracked:  analysis/4.1.2_qtl.analysis_cont_age_ici.vs.eoi_snpsqc_pheno.corrected.cleaned_dis_no-xk_5.batches_52.Rmd
    Untracked:  analysis/4.1.2_qtl.analysis_cont_age_ici.vs.eoi_snpsqc_pheno.corrected.cleaned_dis_no-xk_5.batches_mis_0.Rmd
    Untracked:  analysis/4.1.2_qtl.analysis_cont_age_ici.vs.eoi_snpsqc_pheno.corrected.cleaned_dis_no-xk_5.batches_mis_52.Rmd
    Untracked:  analysis/4.1.2_qtl.analysis_cont_age_ici.vs.eoi_snpsqc_pheno.corrected.cleaned_test.with.4.Rmd
    Untracked:  analysis/4.1.2_qtl.analysis_cont_age_ici.vs.eoi_snpsqc_pheno.corrected.cleaned_test.with.4.Rmd.R
    Untracked:  analysis/4.1.2_qtl.analysis_cont_age_ici.vs.eoi_snpsqc_pheno.corrected.cleaned_test.with.4_miss.Rmd
    Untracked:  analysis/4.1.2_qtl.analysis_cont_age_ici.vs.eoi_snpsqc_pheno.corrected.cleaned_test.with.4_miss.Rmd.R
    Untracked:  analysis/4.1.2_qtl.analysis_cont_age_ici.vs.eoi_snpsqc_pheno.corrected.cleaned_test.with.5.Rmd
    Untracked:  analysis/4.1.2_qtl.analysis_cont_age_ici.vs.eoi_snpsqc_pheno.corrected.cleaned_test.with.5.Rmd.R
    Untracked:  analysis/4.1.2_qtl.analysis_cont_age_ici.vs.eoi_snpsqc_pheno.corrected.cleaned_test.with.5_0.Rmd
    Untracked:  analysis/4.1.2_qtl.analysis_cont_age_ici.vs.eoi_snpsqc_pheno.corrected.cleaned_test.with.5_0.Rmd.R
    Untracked:  analysis/4.1.2_qtl.analysis_cont_rz.age_ici.vs.eoi_pheno.corrected.cleaned_5.batches_0.Rmd
    Untracked:  analysis/4.1.2_qtl.analysis_cont_rz.age_ici.vs.eoi_pheno.corrected.cleaned_5.batches_52.Rmd
    Untracked:  analysis/4.1.2_qtl.analysis_cont_rz.age_ici.vs.eoi_pheno.corrected.cleaned_5.batches_mis_0.Rmd
    Untracked:  analysis/4.1.2_qtl.analysis_cont_rz.age_ici.vs.eoi_pheno.corrected.cleaned_5.batches_mis_52.Rmd
    Untracked:  analysis/4.1.2_qtl.analysis_cont_rz.age_ici.vs.eoi_pheno.corrected.cleaned_5.batches_mis_vo.Rmd
    Untracked:  analysis/4.1.2_qtl.analysis_cont_rz.age_ici.vs.eoi_pheno.corrected.cleaned_5.batches_mis_vo1.Rmd
    Untracked:  analysis/4.1.2_qtl.analysis_cont_rz.age_ici.vs.eoi_pheno.corrected.cleaned_5.batches_vo.Rmd
    Untracked:  analysis/4.1.2_qtl.analysis_cont_rz.age_ici.vs.eoi_pheno.corrected.cleaned_5.batches_vo1.Rmd
    Untracked:  analysis/4.1.2_qtl.analysis_cont_rz.age_ici.vs.eoi_snpsqc_pheno.corrected.cleaned_5.batches_0.Rmd
    Untracked:  analysis/4.1.2_qtl.analysis_cont_rz.age_ici.vs.eoi_snpsqc_pheno.corrected.cleaned_5.batches_52.Rmd
    Untracked:  analysis/4.1.2_qtl.analysis_cont_rz.age_ici.vs.eoi_snpsqc_pheno.corrected.cleaned_5.batches_mis_0.Rmd
    Untracked:  analysis/4.1.2_qtl.analysis_cont_rz.age_ici.vs.eoi_snpsqc_pheno.corrected.cleaned_5.batches_mis_52.Rmd
    Untracked:  analysis/4.1.2_qtl.analysis_cont_rz.age_ici.vs.eoi_snpsqc_pheno.corrected.cleaned_dis_no-xk_5.batches_0.Rmd
    Untracked:  analysis/4.1.2_qtl.analysis_cont_rz.age_ici.vs.eoi_snpsqc_pheno.corrected.cleaned_dis_no-xk_5.batches_52.Rmd
    Untracked:  analysis/4.1.2_qtl.analysis_cont_rz.age_ici.vs.eoi_snpsqc_pheno.corrected.cleaned_dis_no-xk_5.batches_mis_0.Rmd
    Untracked:  analysis/4.1.2_qtl.analysis_cont_rz.age_ici.vs.eoi_snpsqc_pheno.corrected.cleaned_dis_no-xk_5.batches_mis_52.Rmd
    Untracked:  analysis/genotype.frequencies_ici.vs.eoi_5.batches_0.Rmd
    Untracked:  analysis/genotype.frequencies_ici.vs.eoi_5.batches_52.Rmd
    Untracked:  analysis/genotype.frequencies_ici.vs.eoi_5.batches_mis_0.Rmd
    Untracked:  analysis/genotype.frequencies_ici.vs.eoi_5.batches_mis_52.Rmd
    Untracked:  analysis/index_5.batches_additional_vo.Rmd
    Untracked:  data/GM_covar.csv
    Untracked:  data/GM_covar_BC312.csv
    Untracked:  data/bad_markers_all_4.batches.RData
    Untracked:  data/bad_markers_all_5.batches.RData
    Untracked:  data/blup_sub_chr10_lod.drop-1.5_5.batches.csv
    Untracked:  data/blup_sub_chr3_lod.drop-1.5.csv
    Untracked:  data/blup_sub_chr3_lod.drop-1.5_5.batches.csv
    Untracked:  data/blup_sub_chr4_lod.drop-1.5.csv
    Untracked:  data/blup_sub_chr4_lod.drop-1.5_5.batches.csv
    Untracked:  data/covar_cleaned_ici.vs.eoi.csv
    Untracked:  data/covar_cleaned_ici.vs.pbs.csv
    Untracked:  data/covar_corrected.cleaned_ici-early.vs.pbs_5.batches.csv
    Untracked:  data/covar_corrected.cleaned_ici-early.vs.pbs_5.batches_0.csv
    Untracked:  data/covar_corrected.cleaned_ici-early.vs.pbs_5.batches_52.csv
    Untracked:  data/covar_corrected.cleaned_ici-early.vs.pbs_5.batches_mis.csv
    Untracked:  data/covar_corrected.cleaned_ici-early.vs.pbs_5.batches_mis_0.csv
    Untracked:  data/covar_corrected.cleaned_ici-early.vs.pbs_5.batches_mis_52.csv
    Untracked:  data/covar_corrected.cleaned_ici.vs.eoi.csv
    Untracked:  data/covar_corrected.cleaned_ici.vs.eoi1.csv
    Untracked:  data/covar_corrected.cleaned_ici.vs.eoi_5.batches.csv
    Untracked:  data/covar_corrected.cleaned_ici.vs.eoi_5.batches_0.csv
    Untracked:  data/covar_corrected.cleaned_ici.vs.eoi_5.batches_52.csv
    Untracked:  data/covar_corrected.cleaned_ici.vs.eoi_5.batches_mis.csv
    Untracked:  data/covar_corrected.cleaned_ici.vs.eoi_5.batches_mis_0.csv
    Untracked:  data/covar_corrected.cleaned_ici.vs.eoi_5.batches_mis_52.csv
    Untracked:  data/covar_corrected.cleaned_ici.vs.pbs.csv
    Untracked:  data/covar_corrected.cleaned_ici.vs.pbs1.csv
    Untracked:  data/covar_corrected.cleaned_ici.vs.pbs_5.batches.csv
    Untracked:  data/covar_corrected.cleaned_ici.vs.pbs_5.batches_0.csv
    Untracked:  data/covar_corrected.cleaned_ici.vs.pbs_5.batches_52.csv
    Untracked:  data/covar_corrected.cleaned_ici.vs.pbs_5.batches_mis.csv
    Untracked:  data/covar_corrected.cleaned_ici.vs.pbs_5.batches_mis_0.csv
    Untracked:  data/covar_corrected.cleaned_ici.vs.pbs_5.batches_mis_52.csv
    Untracked:  data/covar_corrected_ici-early.vs.pbs_5.batches.csv
    Untracked:  data/covar_corrected_ici-early.vs.pbs_5.batches_0.csv
    Untracked:  data/covar_corrected_ici-early.vs.pbs_5.batches_52.csv
    Untracked:  data/covar_corrected_ici-early.vs.pbs_5.batches_mis.csv
    Untracked:  data/covar_corrected_ici-early.vs.pbs_5.batches_mis_0.csv
    Untracked:  data/covar_corrected_ici-early.vs.pbs_5.batches_mis_52.csv
    Untracked:  data/covar_corrected_ici.vs.eoi.csv
    Untracked:  data/covar_corrected_ici.vs.eoi1.csv
    Untracked:  data/covar_corrected_ici.vs.eoi_5.batches.csv
    Untracked:  data/covar_corrected_ici.vs.eoi_5.batches_0.csv
    Untracked:  data/covar_corrected_ici.vs.eoi_5.batches_52.csv
    Untracked:  data/covar_corrected_ici.vs.eoi_5.batches_mis.csv
    Untracked:  data/covar_corrected_ici.vs.eoi_5.batches_mis_0.csv
    Untracked:  data/covar_corrected_ici.vs.eoi_5.batches_mis_52.csv
    Untracked:  data/covar_corrected_ici.vs.pbs.csv
    Untracked:  data/covar_corrected_ici.vs.pbs1.csv
    Untracked:  data/covar_corrected_ici.vs.pbs_5.batches.csv
    Untracked:  data/covar_corrected_ici.vs.pbs_5.batches_0.csv
    Untracked:  data/covar_corrected_ici.vs.pbs_5.batches_52.csv
    Untracked:  data/covar_corrected_ici.vs.pbs_5.batches_mis.csv
    Untracked:  data/covar_corrected_ici.vs.pbs_5.batches_mis_0.csv
    Untracked:  data/covar_corrected_ici.vs.pbs_5.batches_mis_52.csv
    Untracked:  data/e.RData
    Untracked:  data/e_BC312.RData
    Untracked:  data/e_snpg_samqc_4.batches.RData
    Untracked:  data/e_snpg_samqc_4.batches_bc.RData
    Untracked:  data/e_snpg_samqc_5.batches.RData
    Untracked:  data/errors_ind_4.batches.RData
    Untracked:  data/errors_ind_4.batches_bc.RData
    Untracked:  data/errors_ind_5.batches.RData
    Untracked:  data/files.to.sync.txt
    Untracked:  data/fitqtl_chr3.peak_chr4.peak_additive.txt
    Untracked:  data/fitqtl_chr3.peak_chr4.peak_interacting.txt
    Untracked:  data/fitqtl_chr3.peak_chr4.peak_sex_additive.txt
    Untracked:  data/fitqtl_chr3.peak_chr4.peak_sex_interacting.txt
    Untracked:  data/g2blup_effects.csv
    Untracked:  data/g2blup_effects.xlsx
    Untracked:  data/genes_chr10_lod.drop-1.5_5.batches.csv
    Untracked:  data/genes_chr3_lod.drop-1.5.csv
    Untracked:  data/genes_chr3_lod.drop-1.5_5.batches.csv
    Untracked:  data/genes_chr4_lod.drop-1.5.csv
    Untracked:  data/genetic_map.csv
    Untracked:  data/genetic_map_BC312.csv
    Untracked:  data/genotype_errors_marker_4.batches.RData
    Untracked:  data/genotype_errors_marker_5.batches.RData
    Untracked:  data/genotype_freq_marker_4.batches.RData
    Untracked:  data/genotype_freq_marker_5.batches.RData
    Untracked:  data/gm_allqc_4.batches.RData
    Untracked:  data/gm_allqc_5.batches.RData
    Untracked:  data/gm_allqc_5.batches_mis.RData
    Untracked:  data/gm_samqc_3.batches.RData
    Untracked:  data/gm_samqc_4.batches.RData
    Untracked:  data/gm_samqc_4.batches_bc.RData
    Untracked:  data/gm_samqc_5.batches.RData
    Untracked:  data/gm_serreze.192.RData
    Untracked:  data/gm_serreze.BC312.RData
    Untracked:  data/ici-early.vs.pbs_age.of.onset-additive.covariates_blup_sub_chr-7_peak.marker-UNC13388811_lod.drop-1.5_snpsqc_5.batches.csv
    Untracked:  data/ici-early.vs.pbs_age.of.onset-additive.covariates_genes_chr-7_peak.marker-UNC13388811_lod.drop-1.5_snpsqc_5.batches.csv
    Untracked:  data/ici-early.vs.pbs_age.of.onset-no.covariates_blup_sub_chr-14_peak.marker-UNC24056202_lod.drop-1.5_snpsqc_5.batches_mis.csv
    Untracked:  data/ici-early.vs.pbs_age.of.onset-no.covariates_blup_sub_chr-17_peak.marker-UNCJPD006614_lod.drop-1.5_snpsqc_5.batches_mis.csv
    Untracked:  data/ici-early.vs.pbs_age.of.onset-no.covariates_blup_sub_chr-18_peak.marker-UNCHS045343_lod.drop-1.5_snpsqc_5.batches_mis.csv
    Untracked:  data/ici-early.vs.pbs_age.of.onset-no.covariates_blup_sub_chr-8_peak.marker-UNC15524531_lod.drop-1.5_snpsqc_5.batches_mis.csv
    Untracked:  data/ici-early.vs.pbs_age.of.onset-no.covariates_blup_sub_chr-X_peak.marker-UNCHS048314_lod.drop-1.5_snpsqc_5.batches_mis.csv
    Untracked:  data/ici-early.vs.pbs_age.of.onset-no.covariates_genes_chr-14_peak.marker-UNC24056202_lod.drop-1.5_snpsqc_5.batches_mis.csv
    Untracked:  data/ici-early.vs.pbs_age.of.onset-no.covariates_genes_chr-17_peak.marker-UNCJPD006614_lod.drop-1.5_snpsqc_5.batches_mis.csv
    Untracked:  data/ici-early.vs.pbs_age.of.onset-no.covariates_genes_chr-18_peak.marker-UNCHS045343_lod.drop-1.5_snpsqc_5.batches_mis.csv
    Untracked:  data/ici-early.vs.pbs_age.of.onset-no.covariates_genes_chr-8_peak.marker-UNC15524531_lod.drop-1.5_snpsqc_5.batches_mis.csv
    Untracked:  data/ici-early.vs.pbs_age.of.onset-no.covariates_genes_chr-X_peak.marker-UNCHS048314_lod.drop-1.5_snpsqc_5.batches_mis.csv
    Untracked:  data/ici-early.vs.pbs_blup.full_chr-10_5.batches.csv
    Untracked:  data/ici-early.vs.pbs_blup.full_chr-10_5.batches_mis.csv
    Untracked:  data/ici-early.vs.pbs_blup.full_chr-10_snpsqc_5.batches.csv
    Untracked:  data/ici-early.vs.pbs_blup.full_chr-10_snpsqc_5.batches_mis.csv
    Untracked:  data/ici-early.vs.pbs_blup.full_chr-10_snpsqc_dis_no-x_updated_5.batches.csv
    Untracked:  data/ici-early.vs.pbs_blup.full_chr-10_snpsqc_dis_no-x_updated_5.batches_mis.csv
    Untracked:  data/ici-early.vs.pbs_blup.full_chr-11_5.batches.csv
    Untracked:  data/ici-early.vs.pbs_blup.full_chr-11_5.batches_mis.csv
    Untracked:  data/ici-early.vs.pbs_blup.full_chr-11_snpsqc_5.batches.csv
    Untracked:  data/ici-early.vs.pbs_blup.full_chr-11_snpsqc_5.batches_mis.csv
    Untracked:  data/ici-early.vs.pbs_blup.full_chr-11_snpsqc_dis_no-x_updated_5.batches.csv
    Untracked:  data/ici-early.vs.pbs_blup.full_chr-11_snpsqc_dis_no-x_updated_5.batches_mis.csv
    Untracked:  data/ici-early.vs.pbs_blup.full_chr-12_5.batches.csv
    Untracked:  data/ici-early.vs.pbs_blup.full_chr-12_5.batches_mis.csv
    Untracked:  data/ici-early.vs.pbs_blup.full_chr-12_snpsqc_5.batches.csv
    Untracked:  data/ici-early.vs.pbs_blup.full_chr-12_snpsqc_5.batches_mis.csv
    Untracked:  data/ici-early.vs.pbs_blup.full_chr-12_snpsqc_dis_no-x_updated_5.batches.csv
    Untracked:  data/ici-early.vs.pbs_blup.full_chr-12_snpsqc_dis_no-x_updated_5.batches_mis.csv
    Untracked:  data/ici-early.vs.pbs_blup.full_chr-13_5.batches.csv
    Untracked:  data/ici-early.vs.pbs_blup.full_chr-13_5.batches_mis.csv
    Untracked:  data/ici-early.vs.pbs_blup.full_chr-13_snpsqc_5.batches.csv
    Untracked:  data/ici-early.vs.pbs_blup.full_chr-13_snpsqc_5.batches_mis.csv
    Untracked:  data/ici-early.vs.pbs_blup.full_chr-13_snpsqc_dis_no-x_updated_5.batches.csv
    Untracked:  data/ici-early.vs.pbs_blup.full_chr-13_snpsqc_dis_no-x_updated_5.batches_mis.csv
    Untracked:  data/ici-early.vs.pbs_blup.full_chr-14_5.batches.csv
    Untracked:  data/ici-early.vs.pbs_blup.full_chr-14_5.batches_mis.csv
    Untracked:  data/ici-early.vs.pbs_blup.full_chr-14_snpsqc_5.batches.csv
    Untracked:  data/ici-early.vs.pbs_blup.full_chr-14_snpsqc_5.batches_mis.csv
    Untracked:  data/ici-early.vs.pbs_blup.full_chr-14_snpsqc_dis_no-x_updated_5.batches.csv
    Untracked:  data/ici-early.vs.pbs_blup.full_chr-14_snpsqc_dis_no-x_updated_5.batches_mis.csv
    Untracked:  data/ici-early.vs.pbs_blup.full_chr-15_5.batches.csv
    Untracked:  data/ici-early.vs.pbs_blup.full_chr-15_5.batches_mis.csv
    Untracked:  data/ici-early.vs.pbs_blup.full_chr-15_snpsqc_5.batches.csv
    Untracked:  data/ici-early.vs.pbs_blup.full_chr-15_snpsqc_5.batches_mis.csv
    Untracked:  data/ici-early.vs.pbs_blup.full_chr-15_snpsqc_dis_no-x_updated_5.batches.csv
    Untracked:  data/ici-early.vs.pbs_blup.full_chr-15_snpsqc_dis_no-x_updated_5.batches_mis.csv
    Untracked:  data/ici-early.vs.pbs_blup.full_chr-16_5.batches.csv
    Untracked:  data/ici-early.vs.pbs_blup.full_chr-16_5.batches_mis.csv
    Untracked:  data/ici-early.vs.pbs_blup.full_chr-16_snpsqc_5.batches.csv
    Untracked:  data/ici-early.vs.pbs_blup.full_chr-16_snpsqc_5.batches_mis.csv
    Untracked:  data/ici-early.vs.pbs_blup.full_chr-16_snpsqc_dis_no-x_updated_5.batches.csv
    Untracked:  data/ici-early.vs.pbs_blup.full_chr-16_snpsqc_dis_no-x_updated_5.batches_mis.csv
    Untracked:  data/ici-early.vs.pbs_blup.full_chr-17_5.batches.csv
    Untracked:  data/ici-early.vs.pbs_blup.full_chr-17_5.batches_mis.csv
    Untracked:  data/ici-early.vs.pbs_blup.full_chr-17_snpsqc_5.batches.csv
    Untracked:  data/ici-early.vs.pbs_blup.full_chr-17_snpsqc_5.batches_mis.csv
    Untracked:  data/ici-early.vs.pbs_blup.full_chr-17_snpsqc_dis_no-x_updated_5.batches.csv
    Untracked:  data/ici-early.vs.pbs_blup.full_chr-17_snpsqc_dis_no-x_updated_5.batches_mis.csv
    Untracked:  data/ici-early.vs.pbs_blup.full_chr-18_5.batches.csv
    Untracked:  data/ici-early.vs.pbs_blup.full_chr-18_5.batches_mis.csv
    Untracked:  data/ici-early.vs.pbs_blup.full_chr-18_snpsqc_5.batches.csv
    Untracked:  data/ici-early.vs.pbs_blup.full_chr-18_snpsqc_5.batches_mis.csv
    Untracked:  data/ici-early.vs.pbs_blup.full_chr-18_snpsqc_dis_no-x_updated_5.batches.csv
    Untracked:  data/ici-early.vs.pbs_blup.full_chr-18_snpsqc_dis_no-x_updated_5.batches_mis.csv
    Untracked:  data/ici-early.vs.pbs_blup.full_chr-19_5.batches.csv
    Untracked:  data/ici-early.vs.pbs_blup.full_chr-19_5.batches_mis.csv
    Untracked:  data/ici-early.vs.pbs_blup.full_chr-19_snpsqc_5.batches.csv
    Untracked:  data/ici-early.vs.pbs_blup.full_chr-19_snpsqc_5.batches_mis.csv
    Untracked:  data/ici-early.vs.pbs_blup.full_chr-19_snpsqc_dis_no-x_updated_5.batches.csv
    Untracked:  data/ici-early.vs.pbs_blup.full_chr-19_snpsqc_dis_no-x_updated_5.batches_mis.csv
    Untracked:  data/ici-early.vs.pbs_blup.full_chr-1_5.batches.csv
    Untracked:  data/ici-early.vs.pbs_blup.full_chr-1_5.batches_mis.csv
    Untracked:  data/ici-early.vs.pbs_blup.full_chr-1_snpsqc_5.batches.csv
    Untracked:  data/ici-early.vs.pbs_blup.full_chr-1_snpsqc_5.batches_mis.csv
    Untracked:  data/ici-early.vs.pbs_blup.full_chr-1_snpsqc_dis_no-x_updated_5.batches.csv
    Untracked:  data/ici-early.vs.pbs_blup.full_chr-1_snpsqc_dis_no-x_updated_5.batches_mis.csv
    Untracked:  data/ici-early.vs.pbs_blup.full_chr-2_5.batches.csv
    Untracked:  data/ici-early.vs.pbs_blup.full_chr-2_5.batches_mis.csv
    Untracked:  data/ici-early.vs.pbs_blup.full_chr-2_snpsqc_5.batches.csv
    Untracked:  data/ici-early.vs.pbs_blup.full_chr-2_snpsqc_5.batches_mis.csv
    Untracked:  data/ici-early.vs.pbs_blup.full_chr-2_snpsqc_dis_no-x_updated_5.batches.csv
    Untracked:  data/ici-early.vs.pbs_blup.full_chr-2_snpsqc_dis_no-x_updated_5.batches_mis.csv
    Untracked:  data/ici-early.vs.pbs_blup.full_chr-3_5.batches.csv
    Untracked:  data/ici-early.vs.pbs_blup.full_chr-3_5.batches_mis.csv
    Untracked:  data/ici-early.vs.pbs_blup.full_chr-3_snpsqc_5.batches.csv
    Untracked:  data/ici-early.vs.pbs_blup.full_chr-3_snpsqc_5.batches_mis.csv
    Untracked:  data/ici-early.vs.pbs_blup.full_chr-3_snpsqc_dis_no-x_updated_5.batches.csv
    Untracked:  data/ici-early.vs.pbs_blup.full_chr-3_snpsqc_dis_no-x_updated_5.batches_mis.csv
    Untracked:  data/ici-early.vs.pbs_blup.full_chr-4_5.batches.csv
    Untracked:  data/ici-early.vs.pbs_blup.full_chr-4_5.batches_mis.csv
    Untracked:  data/ici-early.vs.pbs_blup.full_chr-4_snpsqc_5.batches.csv
    Untracked:  data/ici-early.vs.pbs_blup.full_chr-4_snpsqc_5.batches_mis.csv
    Untracked:  data/ici-early.vs.pbs_blup.full_chr-4_snpsqc_dis_no-x_updated_5.batches.csv
    Untracked:  data/ici-early.vs.pbs_blup.full_chr-4_snpsqc_dis_no-x_updated_5.batches_mis.csv
    Untracked:  data/ici-early.vs.pbs_blup.full_chr-5_5.batches.csv
    Untracked:  data/ici-early.vs.pbs_blup.full_chr-5_5.batches_mis.csv
    Untracked:  data/ici-early.vs.pbs_blup.full_chr-5_snpsqc_5.batches.csv
    Untracked:  data/ici-early.vs.pbs_blup.full_chr-5_snpsqc_5.batches_mis.csv
    Untracked:  data/ici-early.vs.pbs_blup.full_chr-5_snpsqc_dis_no-x_updated_5.batches.csv
    Untracked:  data/ici-early.vs.pbs_blup.full_chr-5_snpsqc_dis_no-x_updated_5.batches_mis.csv
    Untracked:  data/ici-early.vs.pbs_blup.full_chr-6_5.batches.csv
    Untracked:  data/ici-early.vs.pbs_blup.full_chr-6_5.batches_mis.csv
    Untracked:  data/ici-early.vs.pbs_blup.full_chr-6_snpsqc_5.batches.csv
    Untracked:  data/ici-early.vs.pbs_blup.full_chr-6_snpsqc_5.batches_mis.csv
    Untracked:  data/ici-early.vs.pbs_blup.full_chr-6_snpsqc_dis_no-x_updated_5.batches.csv
    Untracked:  data/ici-early.vs.pbs_blup.full_chr-6_snpsqc_dis_no-x_updated_5.batches_mis.csv
    Untracked:  data/ici-early.vs.pbs_blup.full_chr-7_5.batches.csv
    Untracked:  data/ici-early.vs.pbs_blup.full_chr-7_5.batches_mis.csv
    Untracked:  data/ici-early.vs.pbs_blup.full_chr-7_snpsqc_5.batches.csv
    Untracked:  data/ici-early.vs.pbs_blup.full_chr-7_snpsqc_5.batches_mis.csv
    Untracked:  data/ici-early.vs.pbs_blup.full_chr-7_snpsqc_dis_no-x_updated_5.batches.csv
    Untracked:  data/ici-early.vs.pbs_blup.full_chr-7_snpsqc_dis_no-x_updated_5.batches_mis.csv
    Untracked:  data/ici-early.vs.pbs_blup.full_chr-8_5.batches.csv
    Untracked:  data/ici-early.vs.pbs_blup.full_chr-8_5.batches_mis.csv
    Untracked:  data/ici-early.vs.pbs_blup.full_chr-8_snpsqc_5.batches.csv
    Untracked:  data/ici-early.vs.pbs_blup.full_chr-8_snpsqc_5.batches_mis.csv
    Untracked:  data/ici-early.vs.pbs_blup.full_chr-8_snpsqc_dis_no-x_updated_5.batches.csv
    Untracked:  data/ici-early.vs.pbs_blup.full_chr-8_snpsqc_dis_no-x_updated_5.batches_mis.csv
    Untracked:  data/ici-early.vs.pbs_blup.full_chr-9_5.batches.csv
    Untracked:  data/ici-early.vs.pbs_blup.full_chr-9_5.batches_mis.csv
    Untracked:  data/ici-early.vs.pbs_blup.full_chr-9_snpsqc_5.batches.csv
    Untracked:  data/ici-early.vs.pbs_blup.full_chr-9_snpsqc_5.batches_mis.csv
    Untracked:  data/ici-early.vs.pbs_blup.full_chr-9_snpsqc_dis_no-x_updated_5.batches.csv
    Untracked:  data/ici-early.vs.pbs_blup.full_chr-9_snpsqc_dis_no-x_updated_5.batches_mis.csv
    Untracked:  data/ici-early.vs.pbs_blup.full_chr-X_5.batches.csv
    Untracked:  data/ici-early.vs.pbs_blup.full_chr-X_5.batches_mis.csv
    Untracked:  data/ici-early.vs.pbs_blup.full_chr-X_snpsqc_5.batches.csv
    Untracked:  data/ici-early.vs.pbs_blup.full_chr-X_snpsqc_5.batches_mis.csv
    Untracked:  data/ici-early.vs.pbs_blup.full_chr-X_snpsqc_dis_no-x_updated_5.batches.csv
    Untracked:  data/ici-early.vs.pbs_blup.full_chr-X_snpsqc_dis_no-x_updated_5.batches_mis.csv
    Untracked:  data/ici-early.vs.pbs_blup.qc_chr-10_5.batches.csv
    Untracked:  data/ici-early.vs.pbs_blup.qc_chr-10_5.batches_mis.csv
    Untracked:  data/ici-early.vs.pbs_blup.qc_chr-10_snpsqc_5.batches.csv
    Untracked:  data/ici-early.vs.pbs_blup.qc_chr-10_snpsqc_5.batches_mis.csv
    Untracked:  data/ici-early.vs.pbs_blup.qc_chr-10_snpsqc_dis_no-x_updated_5.batches.csv
    Untracked:  data/ici-early.vs.pbs_blup.qc_chr-10_snpsqc_dis_no-x_updated_5.batches_mis.csv
    Untracked:  data/ici-early.vs.pbs_blup.qc_chr-11_5.batches.csv
    Untracked:  data/ici-early.vs.pbs_blup.qc_chr-11_5.batches_mis.csv
    Untracked:  data/ici-early.vs.pbs_blup.qc_chr-11_snpsqc_5.batches.csv
    Untracked:  data/ici-early.vs.pbs_blup.qc_chr-11_snpsqc_5.batches_mis.csv
    Untracked:  data/ici-early.vs.pbs_blup.qc_chr-11_snpsqc_dis_no-x_updated_5.batches.csv
    Untracked:  data/ici-early.vs.pbs_blup.qc_chr-11_snpsqc_dis_no-x_updated_5.batches_mis.csv
    Untracked:  data/ici-early.vs.pbs_blup.qc_chr-12_5.batches.csv
    Untracked:  data/ici-early.vs.pbs_blup.qc_chr-12_5.batches_mis.csv
    Untracked:  data/ici-early.vs.pbs_blup.qc_chr-12_snpsqc_5.batches.csv
    Untracked:  data/ici-early.vs.pbs_blup.qc_chr-12_snpsqc_5.batches_mis.csv
    Untracked:  data/ici-early.vs.pbs_blup.qc_chr-12_snpsqc_dis_no-x_updated_5.batches.csv
    Untracked:  data/ici-early.vs.pbs_blup.qc_chr-12_snpsqc_dis_no-x_updated_5.batches_mis.csv
    Untracked:  data/ici-early.vs.pbs_blup.qc_chr-13_5.batches.csv
    Untracked:  data/ici-early.vs.pbs_blup.qc_chr-13_5.batches_mis.csv
    Untracked:  data/ici-early.vs.pbs_blup.qc_chr-13_snpsqc_5.batches.csv
    Untracked:  data/ici-early.vs.pbs_blup.qc_chr-13_snpsqc_5.batches_mis.csv
    Untracked:  data/ici-early.vs.pbs_blup.qc_chr-13_snpsqc_dis_no-x_updated_5.batches.csv
    Untracked:  data/ici-early.vs.pbs_blup.qc_chr-13_snpsqc_dis_no-x_updated_5.batches_mis.csv
    Untracked:  data/ici-early.vs.pbs_blup.qc_chr-14_5.batches.csv
    Untracked:  data/ici-early.vs.pbs_blup.qc_chr-14_5.batches_mis.csv
    Untracked:  data/ici-early.vs.pbs_blup.qc_chr-14_snpsqc_5.batches.csv
    Untracked:  data/ici-early.vs.pbs_blup.qc_chr-14_snpsqc_5.batches_mis.csv
    Untracked:  data/ici-early.vs.pbs_blup.qc_chr-14_snpsqc_dis_no-x_updated_5.batches.csv
    Untracked:  data/ici-early.vs.pbs_blup.qc_chr-14_snpsqc_dis_no-x_updated_5.batches_mis.csv
    Untracked:  data/ici-early.vs.pbs_blup.qc_chr-15_5.batches.csv
    Untracked:  data/ici-early.vs.pbs_blup.qc_chr-15_5.batches_mis.csv
    Untracked:  data/ici-early.vs.pbs_blup.qc_chr-15_snpsqc_5.batches.csv
    Untracked:  data/ici-early.vs.pbs_blup.qc_chr-15_snpsqc_5.batches_mis.csv
    Untracked:  data/ici-early.vs.pbs_blup.qc_chr-16_5.batches.csv
    Untracked:  data/ici-early.vs.pbs_blup.qc_chr-16_5.batches_mis.csv
    Untracked:  data/ici-early.vs.pbs_blup.qc_chr-16_snpsqc_5.batches.csv
    Untracked:  data/ici-early.vs.pbs_blup.qc_chr-16_snpsqc_5.batches_mis.csv
    Untracked:  data/ici-early.vs.pbs_blup.qc_chr-16_snpsqc_dis_no-x_updated_5.batches.csv
    Untracked:  data/ici-early.vs.pbs_blup.qc_chr-16_snpsqc_dis_no-x_updated_5.batches_mis.csv
    Untracked:  data/ici-early.vs.pbs_blup.qc_chr-17_5.batches.csv
    Untracked:  data/ici-early.vs.pbs_blup.qc_chr-17_5.batches_mis.csv
    Untracked:  data/ici-early.vs.pbs_blup.qc_chr-17_snpsqc_5.batches.csv
    Untracked:  data/ici-early.vs.pbs_blup.qc_chr-17_snpsqc_5.batches_mis.csv
    Untracked:  data/ici-early.vs.pbs_blup.qc_chr-18_5.batches.csv
    Untracked:  data/ici-early.vs.pbs_blup.qc_chr-18_5.batches_mis.csv
    Untracked:  data/ici-early.vs.pbs_blup.qc_chr-18_snpsqc_5.batches.csv
    Untracked:  data/ici-early.vs.pbs_blup.qc_chr-18_snpsqc_5.batches_mis.csv
    Untracked:  data/ici-early.vs.pbs_blup.qc_chr-18_snpsqc_dis_no-x_updated_5.batches.csv
    Untracked:  data/ici-early.vs.pbs_blup.qc_chr-18_snpsqc_dis_no-x_updated_5.batches_mis.csv
    Untracked:  data/ici-early.vs.pbs_blup.qc_chr-19_5.batches.csv
    Untracked:  data/ici-early.vs.pbs_blup.qc_chr-19_5.batches_mis.csv
    Untracked:  data/ici-early.vs.pbs_blup.qc_chr-19_snpsqc_5.batches.csv
    Untracked:  data/ici-early.vs.pbs_blup.qc_chr-19_snpsqc_5.batches_mis.csv
    Untracked:  data/ici-early.vs.pbs_blup.qc_chr-19_snpsqc_dis_no-x_updated_5.batches.csv
    Untracked:  data/ici-early.vs.pbs_blup.qc_chr-19_snpsqc_dis_no-x_updated_5.batches_mis.csv
    Untracked:  data/ici-early.vs.pbs_blup.qc_chr-1_5.batches.csv
    Untracked:  data/ici-early.vs.pbs_blup.qc_chr-1_5.batches_mis.csv
    Untracked:  data/ici-early.vs.pbs_blup.qc_chr-1_snpsqc_5.batches.csv
    Untracked:  data/ici-early.vs.pbs_blup.qc_chr-1_snpsqc_5.batches_mis.csv
    Untracked:  data/ici-early.vs.pbs_blup.qc_chr-1_snpsqc_dis_no-x_updated_5.batches.csv
    Untracked:  data/ici-early.vs.pbs_blup.qc_chr-1_snpsqc_dis_no-x_updated_5.batches_mis.csv
    Untracked:  data/ici-early.vs.pbs_blup.qc_chr-2_5.batches.csv
    Untracked:  data/ici-early.vs.pbs_blup.qc_chr-2_5.batches_mis.csv
    Untracked:  data/ici-early.vs.pbs_blup.qc_chr-2_snpsqc_5.batches.csv
    Untracked:  data/ici-early.vs.pbs_blup.qc_chr-2_snpsqc_5.batches_mis.csv
    Untracked:  data/ici-early.vs.pbs_blup.qc_chr-2_snpsqc_dis_no-x_updated_5.batches.csv
    Untracked:  data/ici-early.vs.pbs_blup.qc_chr-2_snpsqc_dis_no-x_updated_5.batches_mis.csv
    Untracked:  data/ici-early.vs.pbs_blup.qc_chr-3_5.batches.csv
    Untracked:  data/ici-early.vs.pbs_blup.qc_chr-3_5.batches_mis.csv
    Untracked:  data/ici-early.vs.pbs_blup.qc_chr-3_snpsqc_5.batches.csv
    Untracked:  data/ici-early.vs.pbs_blup.qc_chr-3_snpsqc_5.batches_mis.csv
    Untracked:  data/ici-early.vs.pbs_blup.qc_chr-3_snpsqc_dis_no-x_updated_5.batches.csv
    Untracked:  data/ici-early.vs.pbs_blup.qc_chr-3_snpsqc_dis_no-x_updated_5.batches_mis.csv
    Untracked:  data/ici-early.vs.pbs_blup.qc_chr-4_5.batches.csv
    Untracked:  data/ici-early.vs.pbs_blup.qc_chr-4_5.batches_mis.csv
    Untracked:  data/ici-early.vs.pbs_blup.qc_chr-4_snpsqc_5.batches.csv
    Untracked:  data/ici-early.vs.pbs_blup.qc_chr-4_snpsqc_5.batches_mis.csv
    Untracked:  data/ici-early.vs.pbs_blup.qc_chr-4_snpsqc_dis_no-x_updated_5.batches.csv
    Untracked:  data/ici-early.vs.pbs_blup.qc_chr-4_snpsqc_dis_no-x_updated_5.batches_mis.csv
    Untracked:  data/ici-early.vs.pbs_blup.qc_chr-5_5.batches.csv
    Untracked:  data/ici-early.vs.pbs_blup.qc_chr-5_5.batches_mis.csv
    Untracked:  data/ici-early.vs.pbs_blup.qc_chr-5_snpsqc_5.batches.csv
    Untracked:  data/ici-early.vs.pbs_blup.qc_chr-5_snpsqc_5.batches_mis.csv
    Untracked:  data/ici-early.vs.pbs_blup.qc_chr-5_snpsqc_dis_no-x_updated_5.batches.csv
    Untracked:  data/ici-early.vs.pbs_blup.qc_chr-5_snpsqc_dis_no-x_updated_5.batches_mis.csv
    Untracked:  data/ici-early.vs.pbs_blup.qc_chr-6_5.batches.csv
    Untracked:  data/ici-early.vs.pbs_blup.qc_chr-6_5.batches_mis.csv
    Untracked:  data/ici-early.vs.pbs_blup.qc_chr-6_snpsqc_5.batches.csv
    Untracked:  data/ici-early.vs.pbs_blup.qc_chr-6_snpsqc_5.batches_mis.csv
    Untracked:  data/ici-early.vs.pbs_blup.qc_chr-6_snpsqc_dis_no-x_updated_5.batches.csv
    Untracked:  data/ici-early.vs.pbs_blup.qc_chr-6_snpsqc_dis_no-x_updated_5.batches_mis.csv
    Untracked:  data/ici-early.vs.pbs_blup.qc_chr-7_5.batches.csv
    Untracked:  data/ici-early.vs.pbs_blup.qc_chr-7_5.batches_mis.csv
    Untracked:  data/ici-early.vs.pbs_blup.qc_chr-7_snpsqc_5.batches.csv
    Untracked:  data/ici-early.vs.pbs_blup.qc_chr-7_snpsqc_5.batches_mis.csv
    Untracked:  data/ici-early.vs.pbs_blup.qc_chr-7_snpsqc_dis_no-x_updated_5.batches.csv
    Untracked:  data/ici-early.vs.pbs_blup.qc_chr-7_snpsqc_dis_no-x_updated_5.batches_mis.csv
    Untracked:  data/ici-early.vs.pbs_blup.qc_chr-8_5.batches.csv
    Untracked:  data/ici-early.vs.pbs_blup.qc_chr-8_5.batches_mis.csv
    Untracked:  data/ici-early.vs.pbs_blup.qc_chr-8_snpsqc_5.batches.csv
    Untracked:  data/ici-early.vs.pbs_blup.qc_chr-8_snpsqc_5.batches_mis.csv
    Untracked:  data/ici-early.vs.pbs_blup.qc_chr-8_snpsqc_dis_no-x_updated_5.batches.csv
    Untracked:  data/ici-early.vs.pbs_blup.qc_chr-8_snpsqc_dis_no-x_updated_5.batches_mis.csv
    Untracked:  data/ici-early.vs.pbs_blup.qc_chr-9_5.batches.csv
    Untracked:  data/ici-early.vs.pbs_blup.qc_chr-9_5.batches_mis.csv
    Untracked:  data/ici-early.vs.pbs_blup.qc_chr-9_snpsqc_5.batches.csv
    Untracked:  data/ici-early.vs.pbs_blup.qc_chr-9_snpsqc_5.batches_mis.csv
    Untracked:  data/ici-early.vs.pbs_blup.qc_chr-9_snpsqc_dis_no-x_updated_5.batches.csv
    Untracked:  data/ici-early.vs.pbs_blup.qc_chr-9_snpsqc_dis_no-x_updated_5.batches_mis.csv
    Untracked:  data/ici-early.vs.pbs_blup.qc_chr-X_5.batches.csv
    Untracked:  data/ici-early.vs.pbs_blup.qc_chr-X_5.batches_mis.csv
    Untracked:  data/ici-early.vs.pbs_blup.qc_chr-X_snpsqc_5.batches.csv
    Untracked:  data/ici-early.vs.pbs_blup.qc_chr-X_snpsqc_5.batches_mis.csv
    Untracked:  data/ici-early.vs.pbs_blup.qc_chr-X_snpsqc_dis_no-x_updated_5.batches.csv
    Untracked:  data/ici-early.vs.pbs_blup.qc_chr-X_snpsqc_dis_no-x_updated_5.batches_mis.csv
    Untracked:  data/ici-early.vs.pbs_blup_sub_chr-18_peak.marker-UNCHS045343_lod.drop-1.5_snpsqc_5.batches_mis.csv
    Untracked:  data/ici-early.vs.pbs_genes_chr-18_peak.marker-UNCHS045343_lod.drop-1.5_snpsqc_5.batches_mis.csv
    Untracked:  data/ici-early.vs.pbs_gm_qtl_5.batches.csv
    Untracked:  data/ici-early.vs.pbs_gm_qtl_5.batches_mis.csv
    Untracked:  data/ici-early.vs.pbs_gm_qtl_snpsqc_5.batches.csv
    Untracked:  data/ici-early.vs.pbs_gm_qtl_snpsqc_5.batches_mis.csv
    Untracked:  data/ici-early.vs.pbs_gm_qtl_snpsqc_dis_no-x_updated_5.batches.csv
    Untracked:  data/ici-early.vs.pbs_gm_qtl_snpsqc_dis_no-x_updated_5.batches_mis.csv
    Untracked:  data/ici-early.vs.pbs_marker.freq_low.geno.freq.removed_geno.ratio_5.batches.csv
    Untracked:  data/ici-early.vs.pbs_marker.freq_low.geno.freq.removed_geno.ratio_5.batches_mis.csv
    Untracked:  data/ici-early.vs.pbs_marker.freq_low.geno.freq.removed_sample.outliers.removed_geno.ratio_5.batches.csv
    Untracked:  data/ici-early.vs.pbs_marker.freq_low.geno.freq.removed_sample.outliers.removed_geno.ratio_5.batches_mis.csv
    Untracked:  data/ici-early.vs.pbs_marker.freq_low.probs.freq.removed_geno.ratio_5.batches.csv
    Untracked:  data/ici-early.vs.pbs_marker.freq_low.probs.freq.removed_geno.ratio_5.batches_mis.csv
    Untracked:  data/ici-early.vs.pbs_marker.freq_low.probs.freq.removed_sample.outliers.removed_geno.ratio_5.batches.csv
    Untracked:  data/ici-early.vs.pbs_marker.freq_low.probs.freq.removed_sample.outliers.removed_geno.ratio_5.batches_mis.csv
    Untracked:  data/ici-early.vs.pbs_rz.age-additive.covariates_blup_sub_chr-7_peak.marker-UNC13239362_lod.drop-1.5_snpsqc_5.batches.csv
    Untracked:  data/ici-early.vs.pbs_rz.age-additive.covariates_blup_sub_chr-7_peak.marker-UNC13239362_lod.drop-1.5_snpsqc_5.batches_mis.csv
    Untracked:  data/ici-early.vs.pbs_rz.age-additive.covariates_blup_sub_chr-7_peak.marker-UNC13239362_lod.drop-1.5_snpsqc_dis_no-x_updated_5.batches.csv
    Untracked:  data/ici-early.vs.pbs_rz.age-additive.covariates_blup_sub_chr-7_peak.marker-UNC13239362_lod.drop-1.5_snpsqc_dis_no-x_updated_5.batches_mis.csv
    Untracked:  data/ici-early.vs.pbs_rz.age-additive.covariates_genes_chr-7_peak.marker-UNC13239362_lod.drop-1.5_snpsqc_5.batches.csv
    Untracked:  data/ici-early.vs.pbs_rz.age-additive.covariates_genes_chr-7_peak.marker-UNC13239362_lod.drop-1.5_snpsqc_5.batches_mis.csv
    Untracked:  data/ici-early.vs.pbs_rz.age-additive.covariates_genes_chr-7_peak.marker-UNC13239362_lod.drop-1.5_snpsqc_dis_no-x_updated_5.batches.csv
    Untracked:  data/ici-early.vs.pbs_rz.age-additive.covariates_genes_chr-7_peak.marker-UNC13239362_lod.drop-1.5_snpsqc_dis_no-x_updated_5.batches_mis.csv
    Untracked:  data/ici-early.vs.pbs_scanone_5.batches.Rdata
    Untracked:  data/ici-early.vs.pbs_scanone_5.batches_mis.Rdata
    Untracked:  data/ici-early.vs.pbs_scanone_snpsqc_5.batches.Rdata
    Untracked:  data/ici-early.vs.pbs_scanone_snpsqc_5.batches_mis.Rdata
    Untracked:  data/ici-early.vs.pbs_scanone_snpsqc_dis_no-x_updated_5.batches.Rdata
    Untracked:  data/ici-early.vs.pbs_scanone_snpsqc_dis_no-x_updated_5.batches_mis.Rdata
    Untracked:  data/ici.vs.eoi_age.of.onset-no.covariates_blup_sub_chr-10_peak.marker-JAX00020646_lod.drop-1.5_5.batches_0.csv
    Untracked:  data/ici.vs.eoi_age.of.onset-no.covariates_blup_sub_chr-10_peak.marker-JAX00292499_lod.drop-1.5_5.batches_52.csv
    Untracked:  data/ici.vs.eoi_age.of.onset-no.covariates_blup_sub_chr-10_peak.marker-JAX00292927_lod.drop-1.5_5.batches_0.csv
    Untracked:  data/ici.vs.eoi_age.of.onset-no.covariates_blup_sub_chr-10_peak.marker-JAX00292927_lod.drop-1.5_5.batches_52.csv
    Untracked:  data/ici.vs.eoi_age.of.onset-no.covariates_blup_sub_chr-10_peak.marker-JAX00294019_lod.drop-1.5_snpsqc_dis_no-x_updated_5.batches_0.csv
    Untracked:  data/ici.vs.eoi_age.of.onset-no.covariates_blup_sub_chr-10_peak.marker-UNC18216614_lod.drop-1.5_5.batches_52.csv
    Untracked:  data/ici.vs.eoi_age.of.onset-no.covariates_blup_sub_chr-10_peak.marker-UNC18240977_lod.drop-1.5_snpsqc_dis_no-x_updated_5.batches_52.csv
    Untracked:  data/ici.vs.eoi_age.of.onset-no.covariates_blup_sub_chr-10_peak.marker-UNC18311938_lod.drop-1.5_5.batches_0.csv
    Untracked:  data/ici.vs.eoi_age.of.onset-no.covariates_blup_sub_chr-10_peak.marker-UNC18311938_lod.drop-1.5_5.batches_52.csv
    Untracked:  data/ici.vs.eoi_age.of.onset-no.covariates_blup_sub_chr-10_peak.marker-UNC18311938_lod.drop-1.5_snpsqc_5.batches_52.csv
    Untracked:  data/ici.vs.eoi_age.of.onset-no.covariates_blup_sub_chr-10_peak.marker-UNC18311938_lod.drop-1.5_snpsqc_5.batches_mis_52.csv
    Untracked:  data/ici.vs.eoi_age.of.onset-no.covariates_blup_sub_chr-10_peak.marker-UNC18343181_lod.drop-1.5_5.batches_0.csv
    Untracked:  data/ici.vs.eoi_age.of.onset-no.covariates_blup_sub_chr-10_peak.marker-UNC18363544_lod.drop-1.5_5.batches_0.csv
    Untracked:  data/ici.vs.eoi_age.of.onset-no.covariates_blup_sub_chr-10_peak.marker-UNC18363544_lod.drop-1.5_5.batches_52.csv
    Untracked:  data/ici.vs.eoi_age.of.onset-no.covariates_blup_sub_chr-10_peak.marker-UNC18363544_lod.drop-1.5_snpsqc_5.batches_mis_0.csv
    Untracked:  data/ici.vs.eoi_age.of.onset-no.covariates_blup_sub_chr-10_peak.marker-UNC18376338_lod.drop-1.5_snpsqc_5.batches_0.csv
    Untracked:  data/ici.vs.eoi_age.of.onset-no.covariates_blup_sub_chr-10_peak.marker-UNCHS028236_lod.drop-1.5_snpsqc_5.batches_mis_0.csv
    Untracked:  data/ici.vs.eoi_age.of.onset-no.covariates_blup_sub_chr-10_peak.marker-UNCHS028236_lod.drop-1.5_snpsqc_5.batches_mis_52.csv
    Untracked:  data/ici.vs.eoi_age.of.onset-no.covariates_blup_sub_chr-10_peak.marker-UNCHS028536_lod.drop-1.5_snpsqc_dis_no-x_updated_5.batches_mis_0.csv
    Untracked:  data/ici.vs.eoi_age.of.onset-no.covariates_blup_sub_chr-10_peak.marker-UNCHS028536_lod.drop-1.5_snpsqc_dis_no-x_updated_5.batches_mis_52.csv
    Untracked:  data/ici.vs.eoi_age.of.onset-no.covariates_blup_sub_chr-11_peak.marker-UNC19970181_lod.drop-1.5_snpsqc_5.batches_mis_0.csv
    Untracked:  data/ici.vs.eoi_age.of.onset-no.covariates_blup_sub_chr-11_peak.marker-UNC19970181_lod.drop-1.5_snpsqc_5.batches_mis_52.csv
    Untracked:  data/ici.vs.eoi_age.of.onset-no.covariates_blup_sub_chr-11_peak.marker-UNC20090524_lod.drop-1.5_snpsqc_5.batches_52.csv
    Untracked:  data/ici.vs.eoi_age.of.onset-no.covariates_blup_sub_chr-12_peak.marker-ICR499_lod.drop-1.5_snpsqc_5.batches_mis_0.csv
    Untracked:  data/ici.vs.eoi_age.of.onset-no.covariates_blup_sub_chr-12_peak.marker-ICR499_lod.drop-1.5_snpsqc_5.batches_mis_52.csv
    Untracked:  data/ici.vs.eoi_age.of.onset-no.covariates_blup_sub_chr-12_peak.marker-ICR499_lod.drop-1.5_snpsqc_dis_no-x_updated_5.batches_mis_0.csv
    Untracked:  data/ici.vs.eoi_age.of.onset-no.covariates_blup_sub_chr-13_peak.marker-UNCHS036773_lod.drop-1.5_snpsqc_5.batches_mis_0.csv
    Untracked:  data/ici.vs.eoi_age.of.onset-no.covariates_blup_sub_chr-13_peak.marker-UNCHS036773_lod.drop-1.5_snpsqc_5.batches_mis_52.csv
    Untracked:  data/ici.vs.eoi_age.of.onset-no.covariates_blup_sub_chr-14_peak.marker-UNC24056202_lod.drop-1.5_snpsqc_5.batches_mis_0.csv
    Untracked:  data/ici.vs.eoi_age.of.onset-no.covariates_blup_sub_chr-14_peak.marker-UNC24056202_lod.drop-1.5_snpsqc_5.batches_mis_52.csv
    Untracked:  data/ici.vs.eoi_age.of.onset-no.covariates_blup_sub_chr-15_peak.marker-UNC26070435_lod.drop-1.5_snpsqc_dis_no-x_updated_5.batches_mis_0.csv
    Untracked:  data/ici.vs.eoi_age.of.onset-no.covariates_blup_sub_chr-15_peak.marker-UNC26070435_lod.drop-1.5_snpsqc_dis_no-x_updated_5.batches_mis_52.csv
    Untracked:  data/ici.vs.eoi_age.of.onset-no.covariates_blup_sub_chr-17_peak.marker-UNCHS044241_lod.drop-1.5_snpsqc_5.batches_mis_0.csv
    Untracked:  data/ici.vs.eoi_age.of.onset-no.covariates_blup_sub_chr-17_peak.marker-UNCHS044241_lod.drop-1.5_snpsqc_5.batches_mis_52.csv
    Untracked:  data/ici.vs.eoi_age.of.onset-no.covariates_blup_sub_chr-17_peak.marker-UNCJPD006614_lod.drop-1.5_snpsqc_dis_no-x_updated_5.batches_mis_0.csv
    Untracked:  data/ici.vs.eoi_age.of.onset-no.covariates_blup_sub_chr-17_peak.marker-UNCrs47191360_lod.drop-1.5_snpsqc_dis_no-x_updated_5.batches_mis_52.csv
    Untracked:  data/ici.vs.eoi_age.of.onset-no.covariates_blup_sub_chr-18_peak.marker-UNCHS045343_lod.drop-1.5_snpsqc_5.batches_mis_0.csv
    Untracked:  data/ici.vs.eoi_age.of.onset-no.covariates_blup_sub_chr-18_peak.marker-UNCHS045343_lod.drop-1.5_snpsqc_5.batches_mis_52.csv
    Untracked:  data/ici.vs.eoi_age.of.onset-no.covariates_blup_sub_chr-2_peak.marker-UNC4609527_lod.drop-1.5_snpsqc_dis_no-x_updated_5.batches_mis_0.csv
    Untracked:  data/ici.vs.eoi_age.of.onset-no.covariates_blup_sub_chr-2_peak.marker-UNC4609527_lod.drop-1.5_snpsqc_dis_no-x_updated_5.batches_mis_52.csv
    Untracked:  data/ici.vs.eoi_age.of.onset-no.covariates_blup_sub_chr-2_peak.marker-UNCHS008007_lod.drop-1.5_5.batches_mis.csv
    Untracked:  data/ici.vs.eoi_age.of.onset-no.covariates_blup_sub_chr-2_peak.marker-UNCHS008007_lod.drop-1.5_5.batches_mis_0.csv
    Untracked:  data/ici.vs.eoi_age.of.onset-no.covariates_blup_sub_chr-3_peak.marker-ICR1338_lod.drop-1.5_5.batches_0.csv
    Untracked:  data/ici.vs.eoi_age.of.onset-no.covariates_blup_sub_chr-3_peak.marker-ICR1338_lod.drop-1.5_5.batches_52.csv
    Untracked:  data/ici.vs.eoi_age.of.onset-no.covariates_blup_sub_chr-3_peak.marker-ICR1338_lod.drop-1.5_5.batches_mis_0.csv
    Untracked:  data/ici.vs.eoi_age.of.onset-no.covariates_blup_sub_chr-3_peak.marker-ICR5263_lod.drop-1.5_5.batches_mis.csv
    Untracked:  data/ici.vs.eoi_age.of.onset-no.covariates_blup_sub_chr-3_peak.marker-ICR5263_lod.drop-1.5_5.batches_mis_0.csv
    Untracked:  data/ici.vs.eoi_age.of.onset-no.covariates_blup_sub_chr-3_peak.marker-JAX00105649_lod.drop-1.5_5.batches_mis.csv
    Untracked:  data/ici.vs.eoi_age.of.onset-no.covariates_blup_sub_chr-3_peak.marker-JAX00105649_lod.drop-1.5_5.batches_mis_0.csv
    Untracked:  data/ici.vs.eoi_age.of.onset-no.covariates_blup_sub_chr-3_peak.marker-JAX00106210_lod.drop-1.5_5.batches_mis_0.csv
    Untracked:  data/ici.vs.eoi_age.of.onset-no.covariates_blup_sub_chr-3_peak.marker-JAX00106957_lod.drop-1.5_5.batches_0.csv
    Untracked:  data/ici.vs.eoi_age.of.onset-no.covariates_blup_sub_chr-3_peak.marker-JAX00106957_lod.drop-1.5_5.batches_52.csv
    Untracked:  data/ici.vs.eoi_age.of.onset-no.covariates_blup_sub_chr-3_peak.marker-JAX00106957_lod.drop-1.5_5.batches_mis_0.csv
    Untracked:  data/ici.vs.eoi_age.of.onset-no.covariates_blup_sub_chr-3_peak.marker-JAX00107193_lod.drop-1.5_5.batches_mis_0.csv
    Untracked:  data/ici.vs.eoi_age.of.onset-no.covariates_blup_sub_chr-3_peak.marker-JAX00220631_lod.drop-1.5_5.batches_0.csv
    Untracked:  data/ici.vs.eoi_age.of.onset-no.covariates_blup_sub_chr-3_peak.marker-JAX00220631_lod.drop-1.5_5.batches_mis_0.csv
    Untracked:  data/ici.vs.eoi_age.of.onset-no.covariates_blup_sub_chr-3_peak.marker-JAX00514662_lod.drop-1.5_5.batches_0.csv
    Untracked:  data/ici.vs.eoi_age.of.onset-no.covariates_blup_sub_chr-3_peak.marker-JAX00514662_lod.drop-1.5_5.batches_52.csv
    Untracked:  data/ici.vs.eoi_age.of.onset-no.covariates_blup_sub_chr-3_peak.marker-JAX00516812_lod.drop-1.5_5.batches_mis.csv
    Untracked:  data/ici.vs.eoi_age.of.onset-no.covariates_blup_sub_chr-3_peak.marker-JAX00516812_lod.drop-1.5_5.batches_mis_0.csv
    Untracked:  data/ici.vs.eoi_age.of.onset-no.covariates_blup_sub_chr-3_peak.marker-JAX00517674_lod.drop-1.5_5.batches_mis.csv
    Untracked:  data/ici.vs.eoi_age.of.onset-no.covariates_blup_sub_chr-3_peak.marker-JAX00517674_lod.drop-1.5_5.batches_mis_0.csv
    Untracked:  data/ici.vs.eoi_age.of.onset-no.covariates_blup_sub_chr-3_peak.marker-JAX00518911r_lod.drop-1.5_5.batches_mis.csv
    Untracked:  data/ici.vs.eoi_age.of.onset-no.covariates_blup_sub_chr-3_peak.marker-JAX00518911r_lod.drop-1.5_5.batches_mis_0.csv
    Untracked:  data/ici.vs.eoi_age.of.onset-no.covariates_blup_sub_chr-3_peak.marker-JAX00522922_lod.drop-1.5_5.batches_mis_0.csv
    Untracked:  data/ici.vs.eoi_age.of.onset-no.covariates_blup_sub_chr-3_peak.marker-JAX00524205_lod.drop-1.5_5.batches_mis_0.csv
    Untracked:  data/ici.vs.eoi_age.of.onset-no.covariates_blup_sub_chr-3_peak.marker-JAX00529707r_lod.drop-1.5_5.batches_0.csv
    Untracked:  data/ici.vs.eoi_age.of.onset-no.covariates_blup_sub_chr-3_peak.marker-JAX00529707r_lod.drop-1.5_5.batches_52.csv
    Untracked:  data/ici.vs.eoi_age.of.onset-no.covariates_blup_sub_chr-3_peak.marker-JAX00535764_lod.drop-1.5_5.batches_52.csv
    Untracked:  data/ici.vs.eoi_age.of.onset-no.covariates_blup_sub_chr-3_peak.marker-UNC4809586_lod.drop-1.5_5.batches_mis.csv
    Untracked:  data/ici.vs.eoi_age.of.onset-no.covariates_blup_sub_chr-3_peak.marker-UNC4809586_lod.drop-1.5_5.batches_mis_0.csv
    Untracked:  data/ici.vs.eoi_age.of.onset-no.covariates_blup_sub_chr-3_peak.marker-UNC4813186_lod.drop-1.5_5.batches_0.csv
    Untracked:  data/ici.vs.eoi_age.of.onset-no.covariates_blup_sub_chr-3_peak.marker-UNC4813186_lod.drop-1.5_5.batches_52.csv
    Untracked:  data/ici.vs.eoi_age.of.onset-no.covariates_blup_sub_chr-3_peak.marker-UNC4813186_lod.drop-1.5_5.batches_mis.csv
    Untracked:  data/ici.vs.eoi_age.of.onset-no.covariates_blup_sub_chr-3_peak.marker-UNC4813186_lod.drop-1.5_5.batches_mis_0.csv
    Untracked:  data/ici.vs.eoi_age.of.onset-no.covariates_blup_sub_chr-3_peak.marker-UNC4820379_lod.drop-1.5_5.batches_mis.csv
    Untracked:  data/ici.vs.eoi_age.of.onset-no.covariates_blup_sub_chr-3_peak.marker-UNC4820379_lod.drop-1.5_5.batches_mis_0.csv
    Untracked:  data/ici.vs.eoi_age.of.onset-no.covariates_blup_sub_chr-3_peak.marker-UNC4836550_lod.drop-1.5_5.batches_0.csv
    Untracked:  data/ici.vs.eoi_age.of.onset-no.covariates_blup_sub_chr-3_peak.marker-UNC4836550_lod.drop-1.5_5.batches_52.csv
    Untracked:  data/ici.vs.eoi_age.of.onset-no.covariates_blup_sub_chr-3_peak.marker-UNC4836550_lod.drop-1.5_5.batches_mis_0.csv
    Untracked:  data/ici.vs.eoi_age.of.onset-no.covariates_blup_sub_chr-3_peak.marker-UNC4875653_lod.drop-1.5_5.batches_mis.csv
    Untracked:  data/ici.vs.eoi_age.of.onset-no.covariates_blup_sub_chr-3_peak.marker-UNC4875653_lod.drop-1.5_5.batches_mis_0.csv
    Untracked:  data/ici.vs.eoi_age.of.onset-no.covariates_blup_sub_chr-3_peak.marker-UNC4901083_lod.drop-1.5_5.batches_mis.csv
    Untracked:  data/ici.vs.eoi_age.of.onset-no.covariates_blup_sub_chr-3_peak.marker-UNC4901083_lod.drop-1.5_5.batches_mis_0.csv
    Untracked:  data/ici.vs.eoi_age.of.onset-no.covariates_blup_sub_chr-3_peak.marker-UNC4909519_lod.drop-1.5_5.batches_mis.csv
    Untracked:  data/ici.vs.eoi_age.of.onset-no.covariates_blup_sub_chr-3_peak.marker-UNC4909519_lod.drop-1.5_5.batches_mis_0.csv
    Untracked:  data/ici.vs.eoi_age.of.onset-no.covariates_blup_sub_chr-3_peak.marker-UNC4912039_lod.drop-1.5_5.batches_0.csv
    Untracked:  data/ici.vs.eoi_age.of.onset-no.covariates_blup_sub_chr-3_peak.marker-UNC4912039_lod.drop-1.5_5.batches_mis_0.csv
    Untracked:  data/ici.vs.eoi_age.of.onset-no.covariates_blup_sub_chr-3_peak.marker-UNC4912288_lod.drop-1.5_5.batches_mis.csv
    Untracked:  data/ici.vs.eoi_age.of.onset-no.covariates_blup_sub_chr-3_peak.marker-UNC4912288_lod.drop-1.5_5.batches_mis_0.csv
    Untracked:  data/ici.vs.eoi_age.of.onset-no.covariates_blup_sub_chr-3_peak.marker-UNC4915446_lod.drop-1.5_5.batches_mis.csv
    Untracked:  data/ici.vs.eoi_age.of.onset-no.covariates_blup_sub_chr-3_peak.marker-UNC4915446_lod.drop-1.5_5.batches_mis_0.csv
    Untracked:  data/ici.vs.eoi_age.of.onset-no.covariates_blup_sub_chr-3_peak.marker-UNC4923398_lod.drop-1.5_5.batches_mis.csv
    Untracked:  data/ici.vs.eoi_age.of.onset-no.covariates_blup_sub_chr-3_peak.marker-UNC4923398_lod.drop-1.5_5.batches_mis_0.csv
    Untracked:  data/ici.vs.eoi_age.of.onset-no.covariates_blup_sub_chr-3_peak.marker-UNC4924377_lod.drop-1.5_5.batches_0.csv
    Untracked:  data/ici.vs.eoi_age.of.onset-no.covariates_blup_sub_chr-3_peak.marker-UNC4924377_lod.drop-1.5_5.batches_52.csv
    Untracked:  data/ici.vs.eoi_age.of.onset-no.covariates_blup_sub_chr-3_peak.marker-UNC4924377_lod.drop-1.5_5.batches_mis_0.csv
    Untracked:  data/ici.vs.eoi_age.of.onset-no.covariates_blup_sub_chr-3_peak.marker-UNC4938565_lod.drop-1.5_5.batches_mis.csv
    Untracked:  data/ici.vs.eoi_age.of.onset-no.covariates_blup_sub_chr-3_peak.marker-UNC4938565_lod.drop-1.5_5.batches_mis_0.csv
    Untracked:  data/ici.vs.eoi_age.of.onset-no.covariates_blup_sub_chr-3_peak.marker-UNC4945186_lod.drop-1.5_5.batches_mis.csv
    Untracked:  data/ici.vs.eoi_age.of.onset-no.covariates_blup_sub_chr-3_peak.marker-UNC4945186_lod.drop-1.5_5.batches_mis_0.csv
    Untracked:  data/ici.vs.eoi_age.of.onset-no.covariates_blup_sub_chr-3_peak.marker-UNC4947492_lod.drop-1.5_5.batches_mis.csv
    Untracked:  data/ici.vs.eoi_age.of.onset-no.covariates_blup_sub_chr-3_peak.marker-UNC4947492_lod.drop-1.5_5.batches_mis_0.csv
    Untracked:  data/ici.vs.eoi_age.of.onset-no.covariates_blup_sub_chr-3_peak.marker-UNC4956618_lod.drop-1.5_5.batches_0.csv
    Untracked:  data/ici.vs.eoi_age.of.onset-no.covariates_blup_sub_chr-3_peak.marker-UNC4956618_lod.drop-1.5_5.batches_mis_0.csv
    Untracked:  data/ici.vs.eoi_age.of.onset-no.covariates_blup_sub_chr-3_peak.marker-UNC4957168_lod.drop-1.5_5.batches_52.csv
    Untracked:  data/ici.vs.eoi_age.of.onset-no.covariates_blup_sub_chr-3_peak.marker-UNC4957859_lod.drop-1.5_5.batches_mis.csv
    Untracked:  data/ici.vs.eoi_age.of.onset-no.covariates_blup_sub_chr-3_peak.marker-UNC4957859_lod.drop-1.5_5.batches_mis_0.csv
    Untracked:  data/ici.vs.eoi_age.of.onset-no.covariates_blup_sub_chr-3_peak.marker-UNC4959266_lod.drop-1.5_5.batches_mis.csv
    Untracked:  data/ici.vs.eoi_age.of.onset-no.covariates_blup_sub_chr-3_peak.marker-UNC4959266_lod.drop-1.5_5.batches_mis_0.csv
    Untracked:  data/ici.vs.eoi_age.of.onset-no.covariates_blup_sub_chr-3_peak.marker-UNC4960612_lod.drop-1.5_5.batches_mis.csv
    Untracked:  data/ici.vs.eoi_age.of.onset-no.covariates_blup_sub_chr-3_peak.marker-UNC4960612_lod.drop-1.5_5.batches_mis_0.csv
    Untracked:  data/ici.vs.eoi_age.of.onset-no.covariates_blup_sub_chr-3_peak.marker-UNC4961542_lod.drop-1.5_5.batches_mis.csv
    Untracked:  data/ici.vs.eoi_age.of.onset-no.covariates_blup_sub_chr-3_peak.marker-UNC4961542_lod.drop-1.5_5.batches_mis_0.csv
    Untracked:  data/ici.vs.eoi_age.of.onset-no.covariates_blup_sub_chr-3_peak.marker-UNC4979914_lod.drop-1.5_5.batches_mis.csv
    Untracked:  data/ici.vs.eoi_age.of.onset-no.covariates_blup_sub_chr-3_peak.marker-UNC4979914_lod.drop-1.5_5.batches_mis_0.csv
    Untracked:  data/ici.vs.eoi_age.of.onset-no.covariates_blup_sub_chr-3_peak.marker-UNC4990111_lod.drop-1.5_5.batches_mis.csv
    Untracked:  data/ici.vs.eoi_age.of.onset-no.covariates_blup_sub_chr-3_peak.marker-UNC4990111_lod.drop-1.5_5.batches_mis_0.csv
    Untracked:  data/ici.vs.eoi_age.of.onset-no.covariates_blup_sub_chr-3_peak.marker-UNC5008656_lod.drop-1.5_5.batches_0.csv
    Untracked:  data/ici.vs.eoi_age.of.onset-no.covariates_blup_sub_chr-3_peak.marker-UNC5008656_lod.drop-1.5_5.batches_52.csv
    Untracked:  data/ici.vs.eoi_age.of.onset-no.covariates_blup_sub_chr-3_peak.marker-UNC5008656_lod.drop-1.5_5.batches_mis.csv
    Untracked:  data/ici.vs.eoi_age.of.onset-no.covariates_blup_sub_chr-3_peak.marker-UNC5008656_lod.drop-1.5_5.batches_mis_0.csv
    Untracked:  data/ici.vs.eoi_age.of.onset-no.covariates_blup_sub_chr-3_peak.marker-UNC5046545_lod.drop-1.5_5.batches_mis_0.csv
    Untracked:  data/ici.vs.eoi_age.of.onset-no.covariates_blup_sub_chr-3_peak.marker-UNC5055259_lod.drop-1.5_5.batches_mis_0.csv
    Untracked:  data/ici.vs.eoi_age.of.onset-no.covariates_blup_sub_chr-3_peak.marker-UNC5062524_lod.drop-1.5_5.batches_mis_0.csv
    Untracked:  data/ici.vs.eoi_age.of.onset-no.covariates_blup_sub_chr-3_peak.marker-UNC5068215_lod.drop-1.5_5.batches_mis_0.csv
    Untracked:  data/ici.vs.eoi_age.of.onset-no.covariates_blup_sub_chr-3_peak.marker-UNC5076935_lod.drop-1.5_5.batches_mis_0.csv
    Untracked:  data/ici.vs.eoi_age.of.onset-no.covariates_blup_sub_chr-3_peak.marker-UNC5082757_lod.drop-1.5_5.batches_mis_0.csv
    Untracked:  data/ici.vs.eoi_age.of.onset-no.covariates_blup_sub_chr-3_peak.marker-UNC5083168_lod.drop-1.5_5.batches_0.csv
    Untracked:  data/ici.vs.eoi_age.of.onset-no.covariates_blup_sub_chr-3_peak.marker-UNC5083168_lod.drop-1.5_5.batches_52.csv
    Untracked:  data/ici.vs.eoi_age.of.onset-no.covariates_blup_sub_chr-3_peak.marker-UNC5083168_lod.drop-1.5_5.batches_mis_0.csv
    Untracked:  data/ici.vs.eoi_age.of.onset-no.covariates_blup_sub_chr-3_peak.marker-UNC5173293_lod.drop-1.5_5.batches_mis_0.csv
    Untracked:  data/ici.vs.eoi_age.of.onset-no.covariates_blup_sub_chr-3_peak.marker-UNC5192582_lod.drop-1.5_5.batches_0.csv
    Untracked:  data/ici.vs.eoi_age.of.onset-no.covariates_blup_sub_chr-3_peak.marker-UNC5192582_lod.drop-1.5_5.batches_52.csv
    Untracked:  data/ici.vs.eoi_age.of.onset-no.covariates_blup_sub_chr-3_peak.marker-UNC5192582_lod.drop-1.5_5.batches_mis_0.csv
    Untracked:  data/ici.vs.eoi_age.of.onset-no.covariates_blup_sub_chr-3_peak.marker-UNC5193970_lod.drop-1.5_5.batches_mis_0.csv
    Untracked:  data/ici.vs.eoi_age.of.onset-no.covariates_blup_sub_chr-3_peak.marker-UNC5213888_lod.drop-1.5_snpsqc_5.batches_mis_0.csv
    Untracked:  data/ici.vs.eoi_age.of.onset-no.covariates_blup_sub_chr-3_peak.marker-UNC5213888_lod.drop-1.5_snpsqc_5.batches_mis_52.csv
    Untracked:  data/ici.vs.eoi_age.of.onset-no.covariates_blup_sub_chr-3_peak.marker-UNC5257268_lod.drop-1.5_5.batches_mis_0.csv
    Untracked:  data/ici.vs.eoi_age.of.onset-no.covariates_blup_sub_chr-3_peak.marker-UNC5262472_lod.drop-1.5_5.batches_0.csv
    Untracked:  data/ici.vs.eoi_age.of.onset-no.covariates_blup_sub_chr-3_peak.marker-UNC5262472_lod.drop-1.5_5.batches_52.csv
    Untracked:  data/ici.vs.eoi_age.of.onset-no.covariates_blup_sub_chr-3_peak.marker-UNC5262472_lod.drop-1.5_5.batches_mis_0.csv
    Untracked:  data/ici.vs.eoi_age.of.onset-no.covariates_blup_sub_chr-3_peak.marker-UNC5264696_lod.drop-1.5_5.batches_mis_0.csv
    Untracked:  data/ici.vs.eoi_age.of.onset-no.covariates_blup_sub_chr-3_peak.marker-UNC5267813_lod.drop-1.5_5.batches_mis_0.csv
    Untracked:  data/ici.vs.eoi_age.of.onset-no.covariates_blup_sub_chr-3_peak.marker-UNC5270225_lod.drop-1.5_5.batches_mis_0.csv
    Untracked:  data/ici.vs.eoi_age.of.onset-no.covariates_blup_sub_chr-3_peak.marker-UNC5277061_lod.drop-1.5_5.batches_mis_0.csv
    Untracked:  data/ici.vs.eoi_age.of.onset-no.covariates_blup_sub_chr-3_peak.marker-UNC5278229_lod.drop-1.5_5.batches_0.csv
    Untracked:  data/ici.vs.eoi_age.of.onset-no.covariates_blup_sub_chr-3_peak.marker-UNC5278229_lod.drop-1.5_5.batches_mis_0.csv
    Untracked:  data/ici.vs.eoi_age.of.onset-no.covariates_blup_sub_chr-3_peak.marker-UNC5282259_lod.drop-1.5_5.batches_mis_0.csv
    Untracked:  data/ici.vs.eoi_age.of.onset-no.covariates_blup_sub_chr-3_peak.marker-UNC5287977_lod.drop-1.5_5.batches_mis_0.csv
    Untracked:  data/ici.vs.eoi_age.of.onset-no.covariates_blup_sub_chr-3_peak.marker-UNC5301460_lod.drop-1.5_5.batches_mis_0.csv
    Untracked:  data/ici.vs.eoi_age.of.onset-no.covariates_blup_sub_chr-3_peak.marker-UNC5309136_lod.drop-1.5_5.batches_mis_0.csv
    Untracked:  data/ici.vs.eoi_age.of.onset-no.covariates_blup_sub_chr-3_peak.marker-UNC5321559_lod.drop-1.5_5.batches_mis_0.csv
    Untracked:  data/ici.vs.eoi_age.of.onset-no.covariates_blup_sub_chr-3_peak.marker-UNC5322455_lod.drop-1.5_5.batches_mis_0.csv
    Untracked:  data/ici.vs.eoi_age.of.onset-no.covariates_blup_sub_chr-3_peak.marker-UNC5329171_lod.drop-1.5_5.batches_mis_0.csv
    Untracked:  data/ici.vs.eoi_age.of.onset-no.covariates_blup_sub_chr-3_peak.marker-UNC5360055_lod.drop-1.5_5.batches_52.csv
    Untracked:  data/ici.vs.eoi_age.of.onset-no.covariates_blup_sub_chr-3_peak.marker-UNC5454839_lod.drop-1.5_5.batches_0.csv
    Untracked:  data/ici.vs.eoi_age.of.onset-no.covariates_blup_sub_chr-3_peak.marker-UNC5454839_lod.drop-1.5_5.batches_52.csv
    Untracked:  data/ici.vs.eoi_age.of.onset-no.covariates_blup_sub_chr-3_peak.marker-UNC5454839_lod.drop-1.5_5.batches_mis_0.csv
    Untracked:  data/ici.vs.eoi_age.of.onset-no.covariates_blup_sub_chr-3_peak.marker-UNC5645844_lod.drop-1.5_5.batches_0.csv
    Untracked:  data/ici.vs.eoi_age.of.onset-no.covariates_blup_sub_chr-3_peak.marker-UNC5645844_lod.drop-1.5_5.batches_52.csv
    Untracked:  data/ici.vs.eoi_age.of.onset-no.covariates_blup_sub_chr-3_peak.marker-UNC5645844_lod.drop-1.5_5.batches_mis_0.csv
    Untracked:  data/ici.vs.eoi_age.of.onset-no.covariates_blup_sub_chr-3_peak.marker-UNC5778977_lod.drop-1.5_5.batches_0.csv
    Untracked:  data/ici.vs.eoi_age.of.onset-no.covariates_blup_sub_chr-3_peak.marker-UNC5808246_lod.drop-1.5_5.batches_52.csv
    Untracked:  data/ici.vs.eoi_age.of.onset-no.covariates_blup_sub_chr-3_peak.marker-UNC5916997_lod.drop-1.5_5.batches_52.csv
    Untracked:  data/ici.vs.eoi_age.of.onset-no.covariates_blup_sub_chr-3_peak.marker-UNCHS008182_lod.drop-1.5_5.batches_0.csv
    Untracked:  data/ici.vs.eoi_age.of.onset-no.covariates_blup_sub_chr-3_peak.marker-UNCHS008182_lod.drop-1.5_5.batches_52.csv
    Untracked:  data/ici.vs.eoi_age.of.onset-no.covariates_blup_sub_chr-3_peak.marker-UNCHS008182_lod.drop-1.5_5.batches_mis_0.csv
    Untracked:  data/ici.vs.eoi_age.of.onset-no.covariates_blup_sub_chr-3_peak.marker-UNCHS008193_lod.drop-1.5_5.batches_mis.csv
    Untracked:  data/ici.vs.eoi_age.of.onset-no.covariates_blup_sub_chr-3_peak.marker-UNCHS008193_lod.drop-1.5_5.batches_mis_0.csv
    Untracked:  data/ici.vs.eoi_age.of.onset-no.covariates_blup_sub_chr-3_peak.marker-UNCHS008209_lod.drop-1.5_5.batches_mis.csv
    Untracked:  data/ici.vs.eoi_age.of.onset-no.covariates_blup_sub_chr-3_peak.marker-UNCHS008209_lod.drop-1.5_5.batches_mis_0.csv
    Untracked:  data/ici.vs.eoi_age.of.onset-no.covariates_blup_sub_chr-3_peak.marker-UNCHS008215_lod.drop-1.5_5.batches_mis.csv
    Untracked:  data/ici.vs.eoi_age.of.onset-no.covariates_blup_sub_chr-3_peak.marker-UNCHS008215_lod.drop-1.5_5.batches_mis_0.csv
    Untracked:  data/ici.vs.eoi_age.of.onset-no.covariates_blup_sub_chr-3_peak.marker-UNCHS008219_lod.drop-1.5_5.batches_mis.csv
    Untracked:  data/ici.vs.eoi_age.of.onset-no.covariates_blup_sub_chr-3_peak.marker-UNCHS008219_lod.drop-1.5_5.batches_mis_0.csv
    Untracked:  data/ici.vs.eoi_age.of.onset-no.covariates_blup_sub_chr-3_peak.marker-UNCHS008229_lod.drop-1.5_5.batches_mis.csv
    Untracked:  data/ici.vs.eoi_age.of.onset-no.covariates_blup_sub_chr-3_peak.marker-UNCHS008229_lod.drop-1.5_5.batches_mis_0.csv
    Untracked:  data/ici.vs.eoi_age.of.onset-no.covariates_blup_sub_chr-3_peak.marker-UNCHS008234_lod.drop-1.5_5.batches_mis.csv
    Untracked:  data/ici.vs.eoi_age.of.onset-no.covariates_blup_sub_chr-3_peak.marker-UNCHS008234_lod.drop-1.5_5.batches_mis_0.csv
    Untracked:  data/ici.vs.eoi_age.of.onset-no.covariates_blup_sub_chr-3_peak.marker-UNCHS008242_lod.drop-1.5_5.batches_52.csv
    Untracked:  data/ici.vs.eoi_age.of.onset-no.covariates_blup_sub_chr-3_peak.marker-UNCHS008260_lod.drop-1.5_5.batches_mis.csv
    Untracked:  data/ici.vs.eoi_age.of.onset-no.covariates_blup_sub_chr-3_peak.marker-UNCHS008260_lod.drop-1.5_5.batches_mis_0.csv
    Untracked:  data/ici.vs.eoi_age.of.onset-no.covariates_blup_sub_chr-3_peak.marker-UNCHS008271_lod.drop-1.5_5.batches_mis.csv
    Untracked:  data/ici.vs.eoi_age.of.onset-no.covariates_blup_sub_chr-3_peak.marker-UNCHS008271_lod.drop-1.5_5.batches_mis_0.csv
    Untracked:  data/ici.vs.eoi_age.of.onset-no.covariates_blup_sub_chr-3_peak.marker-UNCHS008281_lod.drop-1.5_5.batches_mis.csv
    Untracked:  data/ici.vs.eoi_age.of.onset-no.covariates_blup_sub_chr-3_peak.marker-UNCHS008281_lod.drop-1.5_5.batches_mis_0.csv
    Untracked:  data/ici.vs.eoi_age.of.onset-no.covariates_blup_sub_chr-3_peak.marker-UNCHS008286_lod.drop-1.5_5.batches_0.csv
    Untracked:  data/ici.vs.eoi_age.of.onset-no.covariates_blup_sub_chr-3_peak.marker-UNCHS008286_lod.drop-1.5_5.batches_52.csv
    Untracked:  data/ici.vs.eoi_age.of.onset-no.covariates_blup_sub_chr-3_peak.marker-UNCHS008286_lod.drop-1.5_5.batches_mis_0.csv
    Untracked:  data/ici.vs.eoi_age.of.onset-no.covariates_blup_sub_chr-3_peak.marker-UNCHS008338_lod.drop-1.5_5.batches_mis.csv
    Untracked:  data/ici.vs.eoi_age.of.onset-no.covariates_blup_sub_chr-3_peak.marker-UNCHS008338_lod.drop-1.5_5.batches_mis_0.csv
    Untracked:  data/ici.vs.eoi_age.of.onset-no.covariates_blup_sub_chr-3_peak.marker-UNCHS008363_lod.drop-1.5_5.batches_mis.csv
    Untracked:  data/ici.vs.eoi_age.of.onset-no.covariates_blup_sub_chr-3_peak.marker-UNCHS008363_lod.drop-1.5_5.batches_mis_0.csv
    Untracked:  data/ici.vs.eoi_age.of.onset-no.covariates_blup_sub_chr-3_peak.marker-UNCHS008379_lod.drop-1.5_5.batches_mis.csv
    Untracked:  data/ici.vs.eoi_age.of.onset-no.covariates_blup_sub_chr-3_peak.marker-UNCHS008379_lod.drop-1.5_5.batches_mis_0.csv
    Untracked:  data/ici.vs.eoi_age.of.onset-no.covariates_blup_sub_chr-3_peak.marker-UNCHS008409_lod.drop-1.5_5.batches_mis_0.csv
    Untracked:  data/ici.vs.eoi_age.of.onset-no.covariates_blup_sub_chr-3_peak.marker-UNCHS008432_lod.drop-1.5_5.batches_mis_0.csv
    Untracked:  data/ici.vs.eoi_age.of.onset-no.covariates_blup_sub_chr-3_peak.marker-UNCHS008487_lod.drop-1.5_snpsqc_5.batches_0.csv
    Untracked:  data/ici.vs.eoi_age.of.onset-no.covariates_blup_sub_chr-3_peak.marker-UNCHS008487_lod.drop-1.5_snpsqc_5.batches_52.csv
    Untracked:  data/ici.vs.eoi_age.of.onset-no.covariates_blup_sub_chr-3_peak.marker-UNCHS008487_lod.drop-1.5_snpsqc_dis_no-x_updated_5.batches_0.csv
    Untracked:  data/ici.vs.eoi_age.of.onset-no.covariates_blup_sub_chr-3_peak.marker-UNCHS008487_lod.drop-1.5_snpsqc_dis_no-x_updated_5.batches_52.csv
    Untracked:  data/ici.vs.eoi_age.of.onset-no.covariates_blup_sub_chr-3_peak.marker-UNCHS008487_lod.drop-1.5_snpsqc_dis_no-x_updated_5.batches_mis_0.csv
    Untracked:  data/ici.vs.eoi_age.of.onset-no.covariates_blup_sub_chr-3_peak.marker-UNCHS008487_lod.drop-1.5_snpsqc_dis_no-x_updated_5.batches_mis_52.csv
    Untracked:  data/ici.vs.eoi_age.of.onset-no.covariates_blup_sub_chr-3_peak.marker-UNCHS008511_lod.drop-1.5_snpsqc_5.batches_mis_0.csv
    Untracked:  data/ici.vs.eoi_age.of.onset-no.covariates_blup_sub_chr-3_peak.marker-UNCHS008511_lod.drop-1.5_snpsqc_5.batches_mis_52.csv
    Untracked:  data/ici.vs.eoi_age.of.onset-no.covariates_blup_sub_chr-3_peak.marker-UNCHS008609_lod.drop-1.5_5.batches_mis_0.csv
    Untracked:  data/ici.vs.eoi_age.of.onset-no.covariates_blup_sub_chr-3_peak.marker-UNCHS008613_lod.drop-1.5_5.batches_0.csv
    Untracked:  data/ici.vs.eoi_age.of.onset-no.covariates_blup_sub_chr-3_peak.marker-UNCHS008613_lod.drop-1.5_5.batches_52.csv
    Untracked:  data/ici.vs.eoi_age.of.onset-no.covariates_blup_sub_chr-3_peak.marker-UNCHS008614_lod.drop-1.5_5.batches_mis_0.csv
    Untracked:  data/ici.vs.eoi_age.of.onset-no.covariates_blup_sub_chr-3_peak.marker-UNCHS008627_lod.drop-1.5_5.batches_mis_0.csv
    Untracked:  data/ici.vs.eoi_age.of.onset-no.covariates_blup_sub_chr-3_peak.marker-UNCHS008659_lod.drop-1.5_5.batches_52.csv
    Untracked:  data/ici.vs.eoi_age.of.onset-no.covariates_blup_sub_chr-3_peak.marker-UNCHS008725_lod.drop-1.5_5.batches_mis_0.csv
    Untracked:  data/ici.vs.eoi_age.of.onset-no.covariates_blup_sub_chr-3_peak.marker-UNCHS008815_lod.drop-1.5_5.batches_0.csv
    Untracked:  data/ici.vs.eoi_age.of.onset-no.covariates_blup_sub_chr-3_peak.marker-UNCHS008815_lod.drop-1.5_5.batches_52.csv
    Untracked:  data/ici.vs.eoi_age.of.onset-no.covariates_blup_sub_chr-3_peak.marker-UNCHS008815_lod.drop-1.5_5.batches_mis_0.csv
    Untracked:  data/ici.vs.eoi_age.of.onset-no.covariates_blup_sub_chr-3_peak.marker-UNCHS009066_lod.drop-1.5_5.batches_52.csv
    Untracked:  data/ici.vs.eoi_age.of.onset-no.covariates_blup_sub_chr-3_peak.marker-UNCHS009067_lod.drop-1.5_5.batches_0.csv
    Untracked:  data/ici.vs.eoi_age.of.onset-no.covariates_blup_sub_chr-3_peak.marker-UNCHS009068_lod.drop-1.5_5.batches_mis_0.csv
    Untracked:  data/ici.vs.eoi_age.of.onset-no.covariates_blup_sub_chr-3_peak.marker-UNCHS009112_lod.drop-1.5_5.batches_52.csv
    Untracked:  data/ici.vs.eoi_age.of.onset-no.covariates_blup_sub_chr-3_peak.marker-UNCHS009224_lod.drop-1.5_5.batches_0.csv
    Untracked:  data/ici.vs.eoi_age.of.onset-no.covariates_blup_sub_chr-3_peak.marker-UNCHS009224_lod.drop-1.5_5.batches_52.csv
    Untracked:  data/ici.vs.eoi_age.of.onset-no.covariates_blup_sub_chr-3_peak.marker-UNCHS009383_lod.drop-1.5_5.batches_0.csv
    Untracked:  data/ici.vs.eoi_age.of.onset-no.covariates_blup_sub_chr-3_peak.marker-UNCHS009383_lod.drop-1.5_5.batches_52.csv
    Untracked:  data/ici.vs.eoi_age.of.onset-no.covariates_blup_sub_chr-3_peak.marker-UNCHS009693_lod.drop-1.5_5.batches_52.csv
    Untracked:  data/ici.vs.eoi_age.of.onset-no.covariates_blup_sub_chr-3_peak.marker-UNCHS009821_lod.drop-1.5_5.batches_52.csv
    Untracked:  data/ici.vs.eoi_age.of.onset-no.covariates_blup_sub_chr-3_peak.marker-UNCJPD001170_lod.drop-1.5_5.batches_mis.csv
    Untracked:  data/ici.vs.eoi_age.of.onset-no.covariates_blup_sub_chr-3_peak.marker-UNCJPD001170_lod.drop-1.5_5.batches_mis_0.csv
    Untracked:  data/ici.vs.eoi_age.of.onset-no.covariates_blup_sub_chr-3_peak.marker-UNCJPD001198_lod.drop-1.5_5.batches_mis.csv
    Untracked:  data/ici.vs.eoi_age.of.onset-no.covariates_blup_sub_chr-3_peak.marker-UNCJPD001198_lod.drop-1.5_5.batches_mis_0.csv
    Untracked:  data/ici.vs.eoi_age.of.onset-no.covariates_blup_sub_chr-3_peak.marker-UNCJPD001276_lod.drop-1.5_5.batches_mis_0.csv
    Untracked:  data/ici.vs.eoi_age.of.onset-no.covariates_blup_sub_chr-3_peak.marker-sanger2496q_lod.drop-1.5_5.batches_mis.csv
    Untracked:  data/ici.vs.eoi_age.of.onset-no.covariates_blup_sub_chr-3_peak.marker-sanger2496q_lod.drop-1.5_5.batches_mis_0.csv
    Untracked:  data/ici.vs.eoi_age.of.onset-no.covariates_blup_sub_chr-4_peak.marker-UNC8250659_lod.drop-1.5_snpsqc_dis_no-x_updated_5.batches_52.csv
    Untracked:  data/ici.vs.eoi_age.of.onset-no.covariates_blup_sub_chr-4_peak.marker-UNC8439633_lod.drop-1.5_snpsqc_5.batches_52.csv
    Untracked:  data/ici.vs.eoi_age.of.onset-no.covariates_blup_sub_chr-4_peak.marker-UNC8439633_lod.drop-1.5_snpsqc_5.batches_mis_52.csv
    Untracked:  data/ici.vs.eoi_age.of.onset-no.covariates_blup_sub_chr-4_peak.marker-UNCHS012955_lod.drop-1.5_snpsqc_dis_no-x_updated_5.batches_mis_52.csv
    Untracked:  data/ici.vs.eoi_age.of.onset-no.covariates_blup_sub_chr-6_peak.marker-UNC11108920_lod.drop-1.5_snpsqc_5.batches.csv
    Untracked:  data/ici.vs.eoi_age.of.onset-no.covariates_blup_sub_chr-6_peak.marker-UNC11108920_lod.drop-1.5_snpsqc_5.batches_mis.csv
    Untracked:  data/ici.vs.eoi_age.of.onset-no.covariates_blup_sub_chr-7_peak.marker-UNCHS020066_lod.drop-1.5_snpsqc_5.batches_mis_0.csv
    Untracked:  data/ici.vs.eoi_age.of.onset-no.covariates_blup_sub_chr-7_peak.marker-UNCHS020066_lod.drop-1.5_snpsqc_5.batches_mis_52.csv
    Untracked:  data/ici.vs.eoi_age.of.onset-no.covariates_blup_sub_chr-7_peak.marker-UNCHS020066_lod.drop-1.5_snpsqc_dis_no-x_updated_5.batches_mis_0.csv
    Untracked:  data/ici.vs.eoi_age.of.onset-no.covariates_blup_sub_chr-7_peak.marker-UNCHS020066_lod.drop-1.5_snpsqc_dis_no-x_updated_5.batches_mis_52.csv
    Untracked:  data/ici.vs.eoi_age.of.onset-no.covariates_blup_sub_chr-8_peak.marker-UNC15524531_lod.drop-1.5_snpsqc_5.batches_mis_0.csv
    Untracked:  data/ici.vs.eoi_age.of.onset-no.covariates_blup_sub_chr-8_peak.marker-UNC15524531_lod.drop-1.5_snpsqc_5.batches_mis_52.csv
    Untracked:  data/ici.vs.eoi_age.of.onset-no.covariates_blup_sub_chr-8_peak.marker-UNC15524531_lod.drop-1.5_snpsqc_dis_no-x_updated_5.batches_mis_0.csv
    Untracked:  data/ici.vs.eoi_age.of.onset-no.covariates_blup_sub_chr-8_peak.marker-UNC15524531_lod.drop-1.5_snpsqc_dis_no-x_updated_5.batches_mis_52.csv
    Untracked:  data/ici.vs.eoi_age.of.onset-no.covariates_blup_sub_chr-9_peak.marker-UNC17203597_lod.drop-1.5_snpsqc_5.batches_mis_0.csv
    Untracked:  data/ici.vs.eoi_age.of.onset-no.covariates_blup_sub_chr-9_peak.marker-UNC17203597_lod.drop-1.5_snpsqc_5.batches_mis_52.csv
    Untracked:  data/ici.vs.eoi_age.of.onset-no.covariates_blup_sub_chr-X_peak.marker-UNCHS048314_lod.drop-1.5_snpsqc_5.batches_mis_0.csv
    Untracked:  data/ici.vs.eoi_age.of.onset-no.covariates_blup_sub_chr-X_peak.marker-UNCHS048314_lod.drop-1.5_snpsqc_5.batches_mis_52.csv
    Untracked:  data/ici.vs.eoi_age.of.onset-no.covariates_blup_sub_chr-X_peak.marker-UNCHS048314_lod.drop-1.5_snpsqc_dis_no-x_updated_5.batches_mis_0.csv
    Untracked:  data/ici.vs.eoi_age.of.onset-no.covariates_blup_sub_chr-X_peak.marker-UNCHS048314_lod.drop-1.5_snpsqc_dis_no-x_updated_5.batches_mis_52.csv
    Untracked:  data/ici.vs.eoi_age.of.onset-no.covariates_genes_chr-10_peak.marker-JAX00020646_lod.drop-1.5_5.batches_0.csv
    Untracked:  data/ici.vs.eoi_age.of.onset-no.covariates_genes_chr-10_peak.marker-JAX00292499_lod.drop-1.5_5.batches_52.csv
    Untracked:  data/ici.vs.eoi_age.of.onset-no.covariates_genes_chr-10_peak.marker-JAX00292927_lod.drop-1.5_5.batches_0.csv
    Untracked:  data/ici.vs.eoi_age.of.onset-no.covariates_genes_chr-10_peak.marker-JAX00292927_lod.drop-1.5_5.batches_52.csv
    Untracked:  data/ici.vs.eoi_age.of.onset-no.covariates_genes_chr-10_peak.marker-JAX00294019_lod.drop-1.5_snpsqc_dis_no-x_updated_5.batches_0.csv
    Untracked:  data/ici.vs.eoi_age.of.onset-no.covariates_genes_chr-10_peak.marker-UNC18216614_lod.drop-1.5_5.batches_52.csv
    Untracked:  data/ici.vs.eoi_age.of.onset-no.covariates_genes_chr-10_peak.marker-UNC18240977_lod.drop-1.5_snpsqc_dis_no-x_updated_5.batches_52.csv
    Untracked:  data/ici.vs.eoi_age.of.onset-no.covariates_genes_chr-10_peak.marker-UNC18311938_lod.drop-1.5_5.batches_0.csv
    Untracked:  data/ici.vs.eoi_age.of.onset-no.covariates_genes_chr-10_peak.marker-UNC18311938_lod.drop-1.5_5.batches_52.csv
    Untracked:  data/ici.vs.eoi_age.of.onset-no.covariates_genes_chr-10_peak.marker-UNC18311938_lod.drop-1.5_snpsqc_5.batches_52.csv
    Untracked:  data/ici.vs.eoi_age.of.onset-no.covariates_genes_chr-10_peak.marker-UNC18311938_lod.drop-1.5_snpsqc_5.batches_mis_52.csv
    Untracked:  data/ici.vs.eoi_age.of.onset-no.covariates_genes_chr-10_peak.marker-UNC18343181_lod.drop-1.5_5.batches_0.csv
    Untracked:  data/ici.vs.eoi_age.of.onset-no.covariates_genes_chr-10_peak.marker-UNC18363544_lod.drop-1.5_5.batches_0.csv
    Untracked:  data/ici.vs.eoi_age.of.onset-no.covariates_genes_chr-10_peak.marker-UNC18363544_lod.drop-1.5_5.batches_52.csv
    Untracked:  data/ici.vs.eoi_age.of.onset-no.covariates_genes_chr-10_peak.marker-UNC18363544_lod.drop-1.5_snpsqc_5.batches_mis_0.csv
    Untracked:  data/ici.vs.eoi_age.of.onset-no.covariates_genes_chr-10_peak.marker-UNC18376338_lod.drop-1.5_snpsqc_5.batches_0.csv
    Untracked:  data/ici.vs.eoi_age.of.onset-no.covariates_genes_chr-10_peak.marker-UNCHS028236_lod.drop-1.5_snpsqc_5.batches_mis_0.csv
    Untracked:  data/ici.vs.eoi_age.of.onset-no.covariates_genes_chr-10_peak.marker-UNCHS028236_lod.drop-1.5_snpsqc_5.batches_mis_52.csv
    Untracked:  data/ici.vs.eoi_age.of.onset-no.covariates_genes_chr-10_peak.marker-UNCHS028536_lod.drop-1.5_snpsqc_dis_no-x_updated_5.batches_mis_0.csv
    Untracked:  data/ici.vs.eoi_age.of.onset-no.covariates_genes_chr-10_peak.marker-UNCHS028536_lod.drop-1.5_snpsqc_dis_no-x_updated_5.batches_mis_52.csv
    Untracked:  data/ici.vs.eoi_age.of.onset-no.covariates_genes_chr-11_peak.marker-UNC19970181_lod.drop-1.5_snpsqc_5.batches_mis_0.csv
    Untracked:  data/ici.vs.eoi_age.of.onset-no.covariates_genes_chr-11_peak.marker-UNC19970181_lod.drop-1.5_snpsqc_5.batches_mis_52.csv
    Untracked:  data/ici.vs.eoi_age.of.onset-no.covariates_genes_chr-11_peak.marker-UNC20090524_lod.drop-1.5_snpsqc_5.batches_52.csv
    Untracked:  data/ici.vs.eoi_age.of.onset-no.covariates_genes_chr-12_peak.marker-ICR499_lod.drop-1.5_snpsqc_5.batches_mis_0.csv
    Untracked:  data/ici.vs.eoi_age.of.onset-no.covariates_genes_chr-12_peak.marker-ICR499_lod.drop-1.5_snpsqc_5.batches_mis_52.csv
    Untracked:  data/ici.vs.eoi_age.of.onset-no.covariates_genes_chr-12_peak.marker-ICR499_lod.drop-1.5_snpsqc_dis_no-x_updated_5.batches_mis_0.csv
    Untracked:  data/ici.vs.eoi_age.of.onset-no.covariates_genes_chr-13_peak.marker-UNCHS036773_lod.drop-1.5_snpsqc_5.batches_mis_0.csv
    Untracked:  data/ici.vs.eoi_age.of.onset-no.covariates_genes_chr-13_peak.marker-UNCHS036773_lod.drop-1.5_snpsqc_5.batches_mis_52.csv
    Untracked:  data/ici.vs.eoi_age.of.onset-no.covariates_genes_chr-14_peak.marker-UNC24056202_lod.drop-1.5_snpsqc_5.batches_mis_0.csv
    Untracked:  data/ici.vs.eoi_age.of.onset-no.covariates_genes_chr-14_peak.marker-UNC24056202_lod.drop-1.5_snpsqc_5.batches_mis_52.csv
    Untracked:  data/ici.vs.eoi_age.of.onset-no.covariates_genes_chr-15_peak.marker-UNC26070435_lod.drop-1.5_snpsqc_dis_no-x_updated_5.batches_mis_0.csv
    Untracked:  data/ici.vs.eoi_age.of.onset-no.covariates_genes_chr-15_peak.marker-UNC26070435_lod.drop-1.5_snpsqc_dis_no-x_updated_5.batches_mis_52.csv
    Untracked:  data/ici.vs.eoi_age.of.onset-no.covariates_genes_chr-17_peak.marker-UNCHS044241_lod.drop-1.5_snpsqc_5.batches_mis_0.csv
    Untracked:  data/ici.vs.eoi_age.of.onset-no.covariates_genes_chr-17_peak.marker-UNCHS044241_lod.drop-1.5_snpsqc_5.batches_mis_52.csv
    Untracked:  data/ici.vs.eoi_age.of.onset-no.covariates_genes_chr-17_peak.marker-UNCJPD006614_lod.drop-1.5_snpsqc_dis_no-x_updated_5.batches_mis_0.csv
    Untracked:  data/ici.vs.eoi_age.of.onset-no.covariates_genes_chr-17_peak.marker-UNCrs47191360_lod.drop-1.5_snpsqc_dis_no-x_updated_5.batches_mis_52.csv
    Untracked:  data/ici.vs.eoi_age.of.onset-no.covariates_genes_chr-18_peak.marker-UNCHS045343_lod.drop-1.5_snpsqc_5.batches_mis_0.csv
    Untracked:  data/ici.vs.eoi_age.of.onset-no.covariates_genes_chr-18_peak.marker-UNCHS045343_lod.drop-1.5_snpsqc_5.batches_mis_52.csv
    Untracked:  data/ici.vs.eoi_age.of.onset-no.covariates_genes_chr-2_peak.marker-UNC4609527_lod.drop-1.5_snpsqc_dis_no-x_updated_5.batches_mis_0.csv
    Untracked:  data/ici.vs.eoi_age.of.onset-no.covariates_genes_chr-2_peak.marker-UNC4609527_lod.drop-1.5_snpsqc_dis_no-x_updated_5.batches_mis_52.csv
    Untracked:  data/ici.vs.eoi_age.of.onset-no.covariates_genes_chr-2_peak.marker-UNCHS008007_lod.drop-1.5_5.batches_mis.csv
    Untracked:  data/ici.vs.eoi_age.of.onset-no.covariates_genes_chr-2_peak.marker-UNCHS008007_lod.drop-1.5_5.batches_mis_0.csv
    Untracked:  data/ici.vs.eoi_age.of.onset-no.covariates_genes_chr-3_peak.marker-ICR1338_lod.drop-1.5_5.batches_0.csv
    Untracked:  data/ici.vs.eoi_age.of.onset-no.covariates_genes_chr-3_peak.marker-ICR1338_lod.drop-1.5_5.batches_52.csv
    Untracked:  data/ici.vs.eoi_age.of.onset-no.covariates_genes_chr-3_peak.marker-ICR1338_lod.drop-1.5_5.batches_mis_0.csv
    Untracked:  data/ici.vs.eoi_age.of.onset-no.covariates_genes_chr-3_peak.marker-ICR5263_lod.drop-1.5_5.batches_mis.csv
    Untracked:  data/ici.vs.eoi_age.of.onset-no.covariates_genes_chr-3_peak.marker-ICR5263_lod.drop-1.5_5.batches_mis_0.csv
    Untracked:  data/ici.vs.eoi_age.of.onset-no.covariates_genes_chr-3_peak.marker-JAX00105649_lod.drop-1.5_5.batches_mis.csv
    Untracked:  data/ici.vs.eoi_age.of.onset-no.covariates_genes_chr-3_peak.marker-JAX00105649_lod.drop-1.5_5.batches_mis_0.csv
    Untracked:  data/ici.vs.eoi_age.of.onset-no.covariates_genes_chr-3_peak.marker-JAX00106210_lod.drop-1.5_5.batches_mis_0.csv
    Untracked:  data/ici.vs.eoi_age.of.onset-no.covariates_genes_chr-3_peak.marker-JAX00106957_lod.drop-1.5_5.batches_0.csv
    Untracked:  data/ici.vs.eoi_age.of.onset-no.covariates_genes_chr-3_peak.marker-JAX00106957_lod.drop-1.5_5.batches_52.csv
    Untracked:  data/ici.vs.eoi_age.of.onset-no.covariates_genes_chr-3_peak.marker-JAX00106957_lod.drop-1.5_5.batches_mis_0.csv
    Untracked:  data/ici.vs.eoi_age.of.onset-no.covariates_genes_chr-3_peak.marker-JAX00107193_lod.drop-1.5_5.batches_mis_0.csv
    Untracked:  data/ici.vs.eoi_age.of.onset-no.covariates_genes_chr-3_peak.marker-JAX00220631_lod.drop-1.5_5.batches_0.csv
    Untracked:  data/ici.vs.eoi_age.of.onset-no.covariates_genes_chr-3_peak.marker-JAX00220631_lod.drop-1.5_5.batches_mis_0.csv
    Untracked:  data/ici.vs.eoi_age.of.onset-no.covariates_genes_chr-3_peak.marker-JAX00514662_lod.drop-1.5_5.batches_0.csv
    Untracked:  data/ici.vs.eoi_age.of.onset-no.covariates_genes_chr-3_peak.marker-JAX00514662_lod.drop-1.5_5.batches_52.csv
    Untracked:  data/ici.vs.eoi_age.of.onset-no.covariates_genes_chr-3_peak.marker-JAX00516812_lod.drop-1.5_5.batches_mis.csv
    Untracked:  data/ici.vs.eoi_age.of.onset-no.covariates_genes_chr-3_peak.marker-JAX00516812_lod.drop-1.5_5.batches_mis_0.csv
    Untracked:  data/ici.vs.eoi_age.of.onset-no.covariates_genes_chr-3_peak.marker-JAX00517674_lod.drop-1.5_5.batches_mis.csv
    Untracked:  data/ici.vs.eoi_age.of.onset-no.covariates_genes_chr-3_peak.marker-JAX00517674_lod.drop-1.5_5.batches_mis_0.csv
    Untracked:  data/ici.vs.eoi_age.of.onset-no.covariates_genes_chr-3_peak.marker-JAX00518911r_lod.drop-1.5_5.batches_mis.csv
    Untracked:  data/ici.vs.eoi_age.of.onset-no.covariates_genes_chr-3_peak.marker-JAX00518911r_lod.drop-1.5_5.batches_mis_0.csv
    Untracked:  data/ici.vs.eoi_age.of.onset-no.covariates_genes_chr-3_peak.marker-JAX00522922_lod.drop-1.5_5.batches_mis_0.csv
    Untracked:  data/ici.vs.eoi_age.of.onset-no.covariates_genes_chr-3_peak.marker-JAX00524205_lod.drop-1.5_5.batches_mis_0.csv
    Untracked:  data/ici.vs.eoi_age.of.onset-no.covariates_genes_chr-3_peak.marker-JAX00529707r_lod.drop-1.5_5.batches_0.csv
    Untracked:  data/ici.vs.eoi_age.of.onset-no.covariates_genes_chr-3_peak.marker-JAX00529707r_lod.drop-1.5_5.batches_52.csv
    Untracked:  data/ici.vs.eoi_age.of.onset-no.covariates_genes_chr-3_peak.marker-JAX00535764_lod.drop-1.5_5.batches_52.csv
    Untracked:  data/ici.vs.eoi_age.of.onset-no.covariates_genes_chr-3_peak.marker-UNC4809586_lod.drop-1.5_5.batches_mis.csv
    Untracked:  data/ici.vs.eoi_age.of.onset-no.covariates_genes_chr-3_peak.marker-UNC4809586_lod.drop-1.5_5.batches_mis_0.csv
    Untracked:  data/ici.vs.eoi_age.of.onset-no.covariates_genes_chr-3_peak.marker-UNC4813186_lod.drop-1.5_5.batches_0.csv
    Untracked:  data/ici.vs.eoi_age.of.onset-no.covariates_genes_chr-3_peak.marker-UNC4813186_lod.drop-1.5_5.batches_52.csv
    Untracked:  data/ici.vs.eoi_age.of.onset-no.covariates_genes_chr-3_peak.marker-UNC4813186_lod.drop-1.5_5.batches_mis.csv
    Untracked:  data/ici.vs.eoi_age.of.onset-no.covariates_genes_chr-3_peak.marker-UNC4813186_lod.drop-1.5_5.batches_mis_0.csv
    Untracked:  data/ici.vs.eoi_age.of.onset-no.covariates_genes_chr-3_peak.marker-UNC4820379_lod.drop-1.5_5.batches_mis.csv
    Untracked:  data/ici.vs.eoi_age.of.onset-no.covariates_genes_chr-3_peak.marker-UNC4820379_lod.drop-1.5_5.batches_mis_0.csv
    Untracked:  data/ici.vs.eoi_age.of.onset-no.covariates_genes_chr-3_peak.marker-UNC4836550_lod.drop-1.5_5.batches_0.csv
    Untracked:  data/ici.vs.eoi_age.of.onset-no.covariates_genes_chr-3_peak.marker-UNC4836550_lod.drop-1.5_5.batches_52.csv
    Untracked:  data/ici.vs.eoi_age.of.onset-no.covariates_genes_chr-3_peak.marker-UNC4836550_lod.drop-1.5_5.batches_mis_0.csv
    Untracked:  data/ici.vs.eoi_age.of.onset-no.covariates_genes_chr-3_peak.marker-UNC4875653_lod.drop-1.5_5.batches_mis.csv
    Untracked:  data/ici.vs.eoi_age.of.onset-no.covariates_genes_chr-3_peak.marker-UNC4875653_lod.drop-1.5_5.batches_mis_0.csv
    Untracked:  data/ici.vs.eoi_age.of.onset-no.covariates_genes_chr-3_peak.marker-UNC4901083_lod.drop-1.5_5.batches_mis.csv
    Untracked:  data/ici.vs.eoi_age.of.onset-no.covariates_genes_chr-3_peak.marker-UNC4901083_lod.drop-1.5_5.batches_mis_0.csv
    Untracked:  data/ici.vs.eoi_age.of.onset-no.covariates_genes_chr-3_peak.marker-UNC4909519_lod.drop-1.5_5.batches_mis.csv
    Untracked:  data/ici.vs.eoi_age.of.onset-no.covariates_genes_chr-3_peak.marker-UNC4909519_lod.drop-1.5_5.batches_mis_0.csv
    Untracked:  data/ici.vs.eoi_age.of.onset-no.covariates_genes_chr-3_peak.marker-UNC4912039_lod.drop-1.5_5.batches_0.csv
    Untracked:  data/ici.vs.eoi_age.of.onset-no.covariates_genes_chr-3_peak.marker-UNC4912039_lod.drop-1.5_5.batches_mis_0.csv
    Untracked:  data/ici.vs.eoi_age.of.onset-no.covariates_genes_chr-3_peak.marker-UNC4912288_lod.drop-1.5_5.batches_mis.csv
    Untracked:  data/ici.vs.eoi_age.of.onset-no.covariates_genes_chr-3_peak.marker-UNC4912288_lod.drop-1.5_5.batches_mis_0.csv
    Untracked:  data/ici.vs.eoi_age.of.onset-no.covariates_genes_chr-3_peak.marker-UNC4915446_lod.drop-1.5_5.batches_mis.csv
    Untracked:  data/ici.vs.eoi_age.of.onset-no.covariates_genes_chr-3_peak.marker-UNC4915446_lod.drop-1.5_5.batches_mis_0.csv
    Untracked:  data/ici.vs.eoi_age.of.onset-no.covariates_genes_chr-3_peak.marker-UNC4923398_lod.drop-1.5_5.batches_mis.csv
    Untracked:  data/ici.vs.eoi_age.of.onset-no.covariates_genes_chr-3_peak.marker-UNC4923398_lod.drop-1.5_5.batches_mis_0.csv
    Untracked:  data/ici.vs.eoi_age.of.onset-no.covariates_genes_chr-3_peak.marker-UNC4924377_lod.drop-1.5_5.batches_0.csv
    Untracked:  data/ici.vs.eoi_age.of.onset-no.covariates_genes_chr-3_peak.marker-UNC4924377_lod.drop-1.5_5.batches_52.csv
    Untracked:  data/ici.vs.eoi_age.of.onset-no.covariates_genes_chr-3_peak.marker-UNC4924377_lod.drop-1.5_5.batches_mis_0.csv
    Untracked:  data/ici.vs.eoi_age.of.onset-no.covariates_genes_chr-3_peak.marker-UNC4938565_lod.drop-1.5_5.batches_mis.csv
    Untracked:  data/ici.vs.eoi_age.of.onset-no.covariates_genes_chr-3_peak.marker-UNC4938565_lod.drop-1.5_5.batches_mis_0.csv
    Untracked:  data/ici.vs.eoi_age.of.onset-no.covariates_genes_chr-3_peak.marker-UNC4945186_lod.drop-1.5_5.batches_mis.csv
    Untracked:  data/ici.vs.eoi_age.of.onset-no.covariates_genes_chr-3_peak.marker-UNC4945186_lod.drop-1.5_5.batches_mis_0.csv
    Untracked:  data/ici.vs.eoi_age.of.onset-no.covariates_genes_chr-3_peak.marker-UNC4947492_lod.drop-1.5_5.batches_mis.csv
    Untracked:  data/ici.vs.eoi_age.of.onset-no.covariates_genes_chr-3_peak.marker-UNC4947492_lod.drop-1.5_5.batches_mis_0.csv
    Untracked:  data/ici.vs.eoi_age.of.onset-no.covariates_genes_chr-3_peak.marker-UNC4956618_lod.drop-1.5_5.batches_0.csv
    Untracked:  data/ici.vs.eoi_age.of.onset-no.covariates_genes_chr-3_peak.marker-UNC4956618_lod.drop-1.5_5.batches_mis_0.csv
    Untracked:  data/ici.vs.eoi_age.of.onset-no.covariates_genes_chr-3_peak.marker-UNC4957168_lod.drop-1.5_5.batches_52.csv
    Untracked:  data/ici.vs.eoi_age.of.onset-no.covariates_genes_chr-3_peak.marker-UNC4957859_lod.drop-1.5_5.batches_mis.csv
    Untracked:  data/ici.vs.eoi_age.of.onset-no.covariates_genes_chr-3_peak.marker-UNC4957859_lod.drop-1.5_5.batches_mis_0.csv
    Untracked:  data/ici.vs.eoi_age.of.onset-no.covariates_genes_chr-3_peak.marker-UNC4959266_lod.drop-1.5_5.batches_mis.csv
    Untracked:  data/ici.vs.eoi_age.of.onset-no.covariates_genes_chr-3_peak.marker-UNC4959266_lod.drop-1.5_5.batches_mis_0.csv
    Untracked:  data/ici.vs.eoi_age.of.onset-no.covariates_genes_chr-3_peak.marker-UNC4960612_lod.drop-1.5_5.batches_mis.csv
    Untracked:  data/ici.vs.eoi_age.of.onset-no.covariates_genes_chr-3_peak.marker-UNC4960612_lod.drop-1.5_5.batches_mis_0.csv
    Untracked:  data/ici.vs.eoi_age.of.onset-no.covariates_genes_chr-3_peak.marker-UNC4961542_lod.drop-1.5_5.batches_mis.csv
    Untracked:  data/ici.vs.eoi_age.of.onset-no.covariates_genes_chr-3_peak.marker-UNC4961542_lod.drop-1.5_5.batches_mis_0.csv
    Untracked:  data/ici.vs.eoi_age.of.onset-no.covariates_genes_chr-3_peak.marker-UNC4979914_lod.drop-1.5_5.batches_mis.csv
    Untracked:  data/ici.vs.eoi_age.of.onset-no.covariates_genes_chr-3_peak.marker-UNC4979914_lod.drop-1.5_5.batches_mis_0.csv
    Untracked:  data/ici.vs.eoi_age.of.onset-no.covariates_genes_chr-3_peak.marker-UNC4990111_lod.drop-1.5_5.batches_mis.csv
    Untracked:  data/ici.vs.eoi_age.of.onset-no.covariates_genes_chr-3_peak.marker-UNC4990111_lod.drop-1.5_5.batches_mis_0.csv
    Untracked:  data/ici.vs.eoi_age.of.onset-no.covariates_genes_chr-3_peak.marker-UNC5008656_lod.drop-1.5_5.batches_0.csv
    Untracked:  data/ici.vs.eoi_age.of.onset-no.covariates_genes_chr-3_peak.marker-UNC5008656_lod.drop-1.5_5.batches_52.csv
    Untracked:  data/ici.vs.eoi_age.of.onset-no.covariates_genes_chr-3_peak.marker-UNC5008656_lod.drop-1.5_5.batches_mis.csv
    Untracked:  data/ici.vs.eoi_age.of.onset-no.covariates_genes_chr-3_peak.marker-UNC5008656_lod.drop-1.5_5.batches_mis_0.csv
    Untracked:  data/ici.vs.eoi_age.of.onset-no.covariates_genes_chr-3_peak.marker-UNC5046545_lod.drop-1.5_5.batches_mis_0.csv
    Untracked:  data/ici.vs.eoi_age.of.onset-no.covariates_genes_chr-3_peak.marker-UNC5055259_lod.drop-1.5_5.batches_mis_0.csv
    Untracked:  data/ici.vs.eoi_age.of.onset-no.covariates_genes_chr-3_peak.marker-UNC5062524_lod.drop-1.5_5.batches_mis_0.csv
    Untracked:  data/ici.vs.eoi_age.of.onset-no.covariates_genes_chr-3_peak.marker-UNC5068215_lod.drop-1.5_5.batches_mis_0.csv
    Untracked:  data/ici.vs.eoi_age.of.onset-no.covariates_genes_chr-3_peak.marker-UNC5076935_lod.drop-1.5_5.batches_mis_0.csv
    Untracked:  data/ici.vs.eoi_age.of.onset-no.covariates_genes_chr-3_peak.marker-UNC5082757_lod.drop-1.5_5.batches_mis_0.csv
    Untracked:  data/ici.vs.eoi_age.of.onset-no.covariates_genes_chr-3_peak.marker-UNC5083168_lod.drop-1.5_5.batches_0.csv
    Untracked:  data/ici.vs.eoi_age.of.onset-no.covariates_genes_chr-3_peak.marker-UNC5083168_lod.drop-1.5_5.batches_52.csv
    Untracked:  data/ici.vs.eoi_age.of.onset-no.covariates_genes_chr-3_peak.marker-UNC5083168_lod.drop-1.5_5.batches_mis_0.csv
    Untracked:  data/ici.vs.eoi_age.of.onset-no.covariates_genes_chr-3_peak.marker-UNC5173293_lod.drop-1.5_5.batches_mis_0.csv
    Untracked:  data/ici.vs.eoi_age.of.onset-no.covariates_genes_chr-3_peak.marker-UNC5192582_lod.drop-1.5_5.batches_0.csv
    Untracked:  data/ici.vs.eoi_age.of.onset-no.covariates_genes_chr-3_peak.marker-UNC5192582_lod.drop-1.5_5.batches_52.csv
    Untracked:  data/ici.vs.eoi_age.of.onset-no.covariates_genes_chr-3_peak.marker-UNC5192582_lod.drop-1.5_5.batches_mis_0.csv
    Untracked:  data/ici.vs.eoi_age.of.onset-no.covariates_genes_chr-3_peak.marker-UNC5193970_lod.drop-1.5_5.batches_mis_0.csv
    Untracked:  data/ici.vs.eoi_age.of.onset-no.covariates_genes_chr-3_peak.marker-UNC5213888_lod.drop-1.5_snpsqc_5.batches_mis_0.csv
    Untracked:  data/ici.vs.eoi_age.of.onset-no.covariates_genes_chr-3_peak.marker-UNC5213888_lod.drop-1.5_snpsqc_5.batches_mis_52.csv
    Untracked:  data/ici.vs.eoi_age.of.onset-no.covariates_genes_chr-3_peak.marker-UNC5257268_lod.drop-1.5_5.batches_mis_0.csv
    Untracked:  data/ici.vs.eoi_age.of.onset-no.covariates_genes_chr-3_peak.marker-UNC5262472_lod.drop-1.5_5.batches_0.csv
    Untracked:  data/ici.vs.eoi_age.of.onset-no.covariates_genes_chr-3_peak.marker-UNC5262472_lod.drop-1.5_5.batches_52.csv
    Untracked:  data/ici.vs.eoi_age.of.onset-no.covariates_genes_chr-3_peak.marker-UNC5262472_lod.drop-1.5_5.batches_mis_0.csv
    Untracked:  data/ici.vs.eoi_age.of.onset-no.covariates_genes_chr-3_peak.marker-UNC5264696_lod.drop-1.5_5.batches_mis_0.csv
    Untracked:  data/ici.vs.eoi_age.of.onset-no.covariates_genes_chr-3_peak.marker-UNC5267813_lod.drop-1.5_5.batches_mis_0.csv
    Untracked:  data/ici.vs.eoi_age.of.onset-no.covariates_genes_chr-3_peak.marker-UNC5270225_lod.drop-1.5_5.batches_mis_0.csv
    Untracked:  data/ici.vs.eoi_age.of.onset-no.covariates_genes_chr-3_peak.marker-UNC5277061_lod.drop-1.5_5.batches_mis_0.csv
    Untracked:  data/ici.vs.eoi_age.of.onset-no.covariates_genes_chr-3_peak.marker-UNC5278229_lod.drop-1.5_5.batches_0.csv
    Untracked:  data/ici.vs.eoi_age.of.onset-no.covariates_genes_chr-3_peak.marker-UNC5278229_lod.drop-1.5_5.batches_mis_0.csv
    Untracked:  data/ici.vs.eoi_age.of.onset-no.covariates_genes_chr-3_peak.marker-UNC5282259_lod.drop-1.5_5.batches_mis_0.csv
    Untracked:  data/ici.vs.eoi_age.of.onset-no.covariates_genes_chr-3_peak.marker-UNC5287977_lod.drop-1.5_5.batches_mis_0.csv
    Untracked:  data/ici.vs.eoi_age.of.onset-no.covariates_genes_chr-3_peak.marker-UNC5301460_lod.drop-1.5_5.batches_mis_0.csv
    Untracked:  data/ici.vs.eoi_age.of.onset-no.covariates_genes_chr-3_peak.marker-UNC5309136_lod.drop-1.5_5.batches_mis_0.csv
    Untracked:  data/ici.vs.eoi_age.of.onset-no.covariates_genes_chr-3_peak.marker-UNC5321559_lod.drop-1.5_5.batches_mis_0.csv
    Untracked:  data/ici.vs.eoi_age.of.onset-no.covariates_genes_chr-3_peak.marker-UNC5322455_lod.drop-1.5_5.batches_mis_0.csv
    Untracked:  data/ici.vs.eoi_age.of.onset-no.covariates_genes_chr-3_peak.marker-UNC5329171_lod.drop-1.5_5.batches_mis_0.csv
    Untracked:  data/ici.vs.eoi_age.of.onset-no.covariates_genes_chr-3_peak.marker-UNC5360055_lod.drop-1.5_5.batches_52.csv
    Untracked:  data/ici.vs.eoi_age.of.onset-no.covariates_genes_chr-3_peak.marker-UNC5454839_lod.drop-1.5_5.batches_0.csv
    Untracked:  data/ici.vs.eoi_age.of.onset-no.covariates_genes_chr-3_peak.marker-UNC5454839_lod.drop-1.5_5.batches_52.csv
    Untracked:  data/ici.vs.eoi_age.of.onset-no.covariates_genes_chr-3_peak.marker-UNC5454839_lod.drop-1.5_5.batches_mis_0.csv
    Untracked:  data/ici.vs.eoi_age.of.onset-no.covariates_genes_chr-3_peak.marker-UNC5645844_lod.drop-1.5_5.batches_0.csv
    Untracked:  data/ici.vs.eoi_age.of.onset-no.covariates_genes_chr-3_peak.marker-UNC5645844_lod.drop-1.5_5.batches_52.csv
    Untracked:  data/ici.vs.eoi_age.of.onset-no.covariates_genes_chr-3_peak.marker-UNC5645844_lod.drop-1.5_5.batches_mis_0.csv
    Untracked:  data/ici.vs.eoi_age.of.onset-no.covariates_genes_chr-3_peak.marker-UNC5778977_lod.drop-1.5_5.batches_0.csv
    Untracked:  data/ici.vs.eoi_age.of.onset-no.covariates_genes_chr-3_peak.marker-UNC5808246_lod.drop-1.5_5.batches_52.csv
    Untracked:  data/ici.vs.eoi_age.of.onset-no.covariates_genes_chr-3_peak.marker-UNC5916997_lod.drop-1.5_5.batches_52.csv
    Untracked:  data/ici.vs.eoi_age.of.onset-no.covariates_genes_chr-3_peak.marker-UNCHS008182_lod.drop-1.5_5.batches_0.csv
    Untracked:  data/ici.vs.eoi_age.of.onset-no.covariates_genes_chr-3_peak.marker-UNCHS008182_lod.drop-1.5_5.batches_52.csv
    Untracked:  data/ici.vs.eoi_age.of.onset-no.covariates_genes_chr-3_peak.marker-UNCHS008182_lod.drop-1.5_5.batches_mis_0.csv
    Untracked:  data/ici.vs.eoi_age.of.onset-no.covariates_genes_chr-3_peak.marker-UNCHS008193_lod.drop-1.5_5.batches_mis.csv
    Untracked:  data/ici.vs.eoi_age.of.onset-no.covariates_genes_chr-3_peak.marker-UNCHS008193_lod.drop-1.5_5.batches_mis_0.csv
    Untracked:  data/ici.vs.eoi_age.of.onset-no.covariates_genes_chr-3_peak.marker-UNCHS008209_lod.drop-1.5_5.batches_mis.csv
    Untracked:  data/ici.vs.eoi_age.of.onset-no.covariates_genes_chr-3_peak.marker-UNCHS008209_lod.drop-1.5_5.batches_mis_0.csv
    Untracked:  data/ici.vs.eoi_age.of.onset-no.covariates_genes_chr-3_peak.marker-UNCHS008215_lod.drop-1.5_5.batches_mis.csv
    Untracked:  data/ici.vs.eoi_age.of.onset-no.covariates_genes_chr-3_peak.marker-UNCHS008215_lod.drop-1.5_5.batches_mis_0.csv
    Untracked:  data/ici.vs.eoi_age.of.onset-no.covariates_genes_chr-3_peak.marker-UNCHS008219_lod.drop-1.5_5.batches_mis.csv
    Untracked:  data/ici.vs.eoi_age.of.onset-no.covariates_genes_chr-3_peak.marker-UNCHS008219_lod.drop-1.5_5.batches_mis_0.csv
    Untracked:  data/ici.vs.eoi_age.of.onset-no.covariates_genes_chr-3_peak.marker-UNCHS008229_lod.drop-1.5_5.batches_mis.csv
    Untracked:  data/ici.vs.eoi_age.of.onset-no.covariates_genes_chr-3_peak.marker-UNCHS008229_lod.drop-1.5_5.batches_mis_0.csv
    Untracked:  data/ici.vs.eoi_age.of.onset-no.covariates_genes_chr-3_peak.marker-UNCHS008234_lod.drop-1.5_5.batches_mis.csv
    Untracked:  data/ici.vs.eoi_age.of.onset-no.covariates_genes_chr-3_peak.marker-UNCHS008234_lod.drop-1.5_5.batches_mis_0.csv
    Untracked:  data/ici.vs.eoi_age.of.onset-no.covariates_genes_chr-3_peak.marker-UNCHS008242_lod.drop-1.5_5.batches_52.csv
    Untracked:  data/ici.vs.eoi_age.of.onset-no.covariates_genes_chr-3_peak.marker-UNCHS008260_lod.drop-1.5_5.batches_mis.csv
    Untracked:  data/ici.vs.eoi_age.of.onset-no.covariates_genes_chr-3_peak.marker-UNCHS008260_lod.drop-1.5_5.batches_mis_0.csv
    Untracked:  data/ici.vs.eoi_age.of.onset-no.covariates_genes_chr-3_peak.marker-UNCHS008271_lod.drop-1.5_5.batches_mis.csv
    Untracked:  data/ici.vs.eoi_age.of.onset-no.covariates_genes_chr-3_peak.marker-UNCHS008271_lod.drop-1.5_5.batches_mis_0.csv
    Untracked:  data/ici.vs.eoi_age.of.onset-no.covariates_genes_chr-3_peak.marker-UNCHS008281_lod.drop-1.5_5.batches_mis.csv
    Untracked:  data/ici.vs.eoi_age.of.onset-no.covariates_genes_chr-3_peak.marker-UNCHS008281_lod.drop-1.5_5.batches_mis_0.csv
    Untracked:  data/ici.vs.eoi_age.of.onset-no.covariates_genes_chr-3_peak.marker-UNCHS008286_lod.drop-1.5_5.batches_0.csv
    Untracked:  data/ici.vs.eoi_age.of.onset-no.covariates_genes_chr-3_peak.marker-UNCHS008286_lod.drop-1.5_5.batches_52.csv
    Untracked:  data/ici.vs.eoi_age.of.onset-no.covariates_genes_chr-3_peak.marker-UNCHS008286_lod.drop-1.5_5.batches_mis_0.csv
    Untracked:  data/ici.vs.eoi_age.of.onset-no.covariates_genes_chr-3_peak.marker-UNCHS008338_lod.drop-1.5_5.batches_mis.csv
    Untracked:  data/ici.vs.eoi_age.of.onset-no.covariates_genes_chr-3_peak.marker-UNCHS008338_lod.drop-1.5_5.batches_mis_0.csv
    Untracked:  data/ici.vs.eoi_age.of.onset-no.covariates_genes_chr-3_peak.marker-UNCHS008363_lod.drop-1.5_5.batches_mis.csv
    Untracked:  data/ici.vs.eoi_age.of.onset-no.covariates_genes_chr-3_peak.marker-UNCHS008363_lod.drop-1.5_5.batches_mis_0.csv
    Untracked:  data/ici.vs.eoi_age.of.onset-no.covariates_genes_chr-3_peak.marker-UNCHS008379_lod.drop-1.5_5.batches_mis.csv
    Untracked:  data/ici.vs.eoi_age.of.onset-no.covariates_genes_chr-3_peak.marker-UNCHS008379_lod.drop-1.5_5.batches_mis_0.csv
    Untracked:  data/ici.vs.eoi_age.of.onset-no.covariates_genes_chr-3_peak.marker-UNCHS008409_lod.drop-1.5_5.batches_mis_0.csv
    Untracked:  data/ici.vs.eoi_age.of.onset-no.covariates_genes_chr-3_peak.marker-UNCHS008432_lod.drop-1.5_5.batches_mis_0.csv
    Untracked:  data/ici.vs.eoi_age.of.onset-no.covariates_genes_chr-3_peak.marker-UNCHS008487_lod.drop-1.5_snpsqc_5.batches_0.csv
    Untracked:  data/ici.vs.eoi_age.of.onset-no.covariates_genes_chr-3_peak.marker-UNCHS008487_lod.drop-1.5_snpsqc_5.batches_52.csv
    Untracked:  data/ici.vs.eoi_age.of.onset-no.covariates_genes_chr-3_peak.marker-UNCHS008487_lod.drop-1.5_snpsqc_dis_no-x_updated_5.batches_0.csv
    Untracked:  data/ici.vs.eoi_age.of.onset-no.covariates_genes_chr-3_peak.marker-UNCHS008487_lod.drop-1.5_snpsqc_dis_no-x_updated_5.batches_52.csv
    Untracked:  data/ici.vs.eoi_age.of.onset-no.covariates_genes_chr-3_peak.marker-UNCHS008487_lod.drop-1.5_snpsqc_dis_no-x_updated_5.batches_mis_0.csv
    Untracked:  data/ici.vs.eoi_age.of.onset-no.covariates_genes_chr-3_peak.marker-UNCHS008487_lod.drop-1.5_snpsqc_dis_no-x_updated_5.batches_mis_52.csv
    Untracked:  data/ici.vs.eoi_age.of.onset-no.covariates_genes_chr-3_peak.marker-UNCHS008511_lod.drop-1.5_snpsqc_5.batches_mis_0.csv
    Untracked:  data/ici.vs.eoi_age.of.onset-no.covariates_genes_chr-3_peak.marker-UNCHS008511_lod.drop-1.5_snpsqc_5.batches_mis_52.csv
    Untracked:  data/ici.vs.eoi_age.of.onset-no.covariates_genes_chr-3_peak.marker-UNCHS008609_lod.drop-1.5_5.batches_mis_0.csv
    Untracked:  data/ici.vs.eoi_age.of.onset-no.covariates_genes_chr-3_peak.marker-UNCHS008613_lod.drop-1.5_5.batches_0.csv
    Untracked:  data/ici.vs.eoi_age.of.onset-no.covariates_genes_chr-3_peak.marker-UNCHS008613_lod.drop-1.5_5.batches_52.csv
    Untracked:  data/ici.vs.eoi_age.of.onset-no.covariates_genes_chr-3_peak.marker-UNCHS008614_lod.drop-1.5_5.batches_mis_0.csv
    Untracked:  data/ici.vs.eoi_age.of.onset-no.covariates_genes_chr-3_peak.marker-UNCHS008627_lod.drop-1.5_5.batches_mis_0.csv
    Untracked:  data/ici.vs.eoi_age.of.onset-no.covariates_genes_chr-3_peak.marker-UNCHS008659_lod.drop-1.5_5.batches_52.csv
    Untracked:  data/ici.vs.eoi_age.of.onset-no.covariates_genes_chr-3_peak.marker-UNCHS008725_lod.drop-1.5_5.batches_mis_0.csv
    Untracked:  data/ici.vs.eoi_age.of.onset-no.covariates_genes_chr-3_peak.marker-UNCHS008815_lod.drop-1.5_5.batches_0.csv
    Untracked:  data/ici.vs.eoi_age.of.onset-no.covariates_genes_chr-3_peak.marker-UNCHS008815_lod.drop-1.5_5.batches_52.csv
    Untracked:  data/ici.vs.eoi_age.of.onset-no.covariates_genes_chr-3_peak.marker-UNCHS008815_lod.drop-1.5_5.batches_mis_0.csv
    Untracked:  data/ici.vs.eoi_age.of.onset-no.covariates_genes_chr-3_peak.marker-UNCHS009066_lod.drop-1.5_5.batches_52.csv
    Untracked:  data/ici.vs.eoi_age.of.onset-no.covariates_genes_chr-3_peak.marker-UNCHS009067_lod.drop-1.5_5.batches_0.csv
    Untracked:  data/ici.vs.eoi_age.of.onset-no.covariates_genes_chr-3_peak.marker-UNCHS009068_lod.drop-1.5_5.batches_mis_0.csv
    Untracked:  data/ici.vs.eoi_age.of.onset-no.covariates_genes_chr-3_peak.marker-UNCHS009112_lod.drop-1.5_5.batches_52.csv
    Untracked:  data/ici.vs.eoi_age.of.onset-no.covariates_genes_chr-3_peak.marker-UNCHS009224_lod.drop-1.5_5.batches_0.csv
    Untracked:  data/ici.vs.eoi_age.of.onset-no.covariates_genes_chr-3_peak.marker-UNCHS009224_lod.drop-1.5_5.batches_52.csv
    Untracked:  data/ici.vs.eoi_age.of.onset-no.covariates_genes_chr-3_peak.marker-UNCHS009383_lod.drop-1.5_5.batches_0.csv
    Untracked:  data/ici.vs.eoi_age.of.onset-no.covariates_genes_chr-3_peak.marker-UNCHS009383_lod.drop-1.5_5.batches_52.csv
    Untracked:  data/ici.vs.eoi_age.of.onset-no.covariates_genes_chr-3_peak.marker-UNCHS009693_lod.drop-1.5_5.batches_52.csv
    Untracked:  data/ici.vs.eoi_age.of.onset-no.covariates_genes_chr-3_peak.marker-UNCHS009821_lod.drop-1.5_5.batches_52.csv
    Untracked:  data/ici.vs.eoi_age.of.onset-no.covariates_genes_chr-3_peak.marker-UNCJPD001170_lod.drop-1.5_5.batches_mis.csv
    Untracked:  data/ici.vs.eoi_age.of.onset-no.covariates_genes_chr-3_peak.marker-UNCJPD001170_lod.drop-1.5_5.batches_mis_0.csv
    Untracked:  data/ici.vs.eoi_age.of.onset-no.covariates_genes_chr-3_peak.marker-UNCJPD001198_lod.drop-1.5_5.batches_mis.csv
    Untracked:  data/ici.vs.eoi_age.of.onset-no.covariates_genes_chr-3_peak.marker-UNCJPD001198_lod.drop-1.5_5.batches_mis_0.csv
    Untracked:  data/ici.vs.eoi_age.of.onset-no.covariates_genes_chr-3_peak.marker-UNCJPD001276_lod.drop-1.5_5.batches_mis_0.csv
    Untracked:  data/ici.vs.eoi_age.of.onset-no.covariates_genes_chr-3_peak.marker-sanger2496q_lod.drop-1.5_5.batches_mis.csv
    Untracked:  data/ici.vs.eoi_age.of.onset-no.covariates_genes_chr-3_peak.marker-sanger2496q_lod.drop-1.5_5.batches_mis_0.csv
    Untracked:  data/ici.vs.eoi_age.of.onset-no.covariates_genes_chr-4_peak.marker-UNC8250659_lod.drop-1.5_snpsqc_dis_no-x_updated_5.batches_52.csv
    Untracked:  data/ici.vs.eoi_age.of.onset-no.covariates_genes_chr-4_peak.marker-UNC8439633_lod.drop-1.5_snpsqc_5.batches_52.csv
    Untracked:  data/ici.vs.eoi_age.of.onset-no.covariates_genes_chr-4_peak.marker-UNC8439633_lod.drop-1.5_snpsqc_5.batches_mis_52.csv
    Untracked:  data/ici.vs.eoi_age.of.onset-no.covariates_genes_chr-4_peak.marker-UNCHS012955_lod.drop-1.5_snpsqc_dis_no-x_updated_5.batches_mis_52.csv
    Untracked:  data/ici.vs.eoi_age.of.onset-no.covariates_genes_chr-6_peak.marker-UNC11108920_lod.drop-1.5_snpsqc_5.batches.csv
    Untracked:  data/ici.vs.eoi_age.of.onset-no.covariates_genes_chr-6_peak.marker-UNC11108920_lod.drop-1.5_snpsqc_5.batches_mis.csv
    Untracked:  data/ici.vs.eoi_age.of.onset-no.covariates_genes_chr-7_peak.marker-UNCHS020066_lod.drop-1.5_snpsqc_5.batches_mis_0.csv
    Untracked:  data/ici.vs.eoi_age.of.onset-no.covariates_genes_chr-7_peak.marker-UNCHS020066_lod.drop-1.5_snpsqc_5.batches_mis_52.csv
    Untracked:  data/ici.vs.eoi_age.of.onset-no.covariates_genes_chr-7_peak.marker-UNCHS020066_lod.drop-1.5_snpsqc_dis_no-x_updated_5.batches_mis_0.csv
    Untracked:  data/ici.vs.eoi_age.of.onset-no.covariates_genes_chr-7_peak.marker-UNCHS020066_lod.drop-1.5_snpsqc_dis_no-x_updated_5.batches_mis_52.csv
    Untracked:  data/ici.vs.eoi_age.of.onset-no.covariates_genes_chr-8_peak.marker-UNC15524531_lod.drop-1.5_snpsqc_5.batches_mis_0.csv
    Untracked:  data/ici.vs.eoi_age.of.onset-no.covariates_genes_chr-8_peak.marker-UNC15524531_lod.drop-1.5_snpsqc_5.batches_mis_52.csv
    Untracked:  data/ici.vs.eoi_age.of.onset-no.covariates_genes_chr-8_peak.marker-UNC15524531_lod.drop-1.5_snpsqc_dis_no-x_updated_5.batches_mis_0.csv
    Untracked:  data/ici.vs.eoi_age.of.onset-no.covariates_genes_chr-8_peak.marker-UNC15524531_lod.drop-1.5_snpsqc_dis_no-x_updated_5.batches_mis_52.csv
    Untracked:  data/ici.vs.eoi_age.of.onset-no.covariates_genes_chr-9_peak.marker-UNC17203597_lod.drop-1.5_snpsqc_5.batches_mis_0.csv
    Untracked:  data/ici.vs.eoi_age.of.onset-no.covariates_genes_chr-9_peak.marker-UNC17203597_lod.drop-1.5_snpsqc_5.batches_mis_52.csv
    Untracked:  data/ici.vs.eoi_age.of.onset-no.covariates_genes_chr-X_peak.marker-UNCHS048314_lod.drop-1.5_snpsqc_5.batches_mis_0.csv
    Untracked:  data/ici.vs.eoi_age.of.onset-no.covariates_genes_chr-X_peak.marker-UNCHS048314_lod.drop-1.5_snpsqc_5.batches_mis_52.csv
    Untracked:  data/ici.vs.eoi_age.of.onset-no.covariates_genes_chr-X_peak.marker-UNCHS048314_lod.drop-1.5_snpsqc_dis_no-x_updated_5.batches_mis_0.csv
    Untracked:  data/ici.vs.eoi_age.of.onset-no.covariates_genes_chr-X_peak.marker-UNCHS048314_lod.drop-1.5_snpsqc_dis_no-x_updated_5.batches_mis_52.csv
    Untracked:  data/ici.vs.eoi_blup.full_chr-10_snpsqc_5.batches.csv
    Untracked:  data/ici.vs.eoi_blup.full_chr-10_snpsqc_5.batches_mis.csv
    Untracked:  data/ici.vs.eoi_blup.full_chr-11_snpsqc_5.batches.csv
    Untracked:  data/ici.vs.eoi_blup.full_chr-11_snpsqc_5.batches_mis.csv
    Untracked:  data/ici.vs.eoi_blup.full_chr-12_snpsqc_5.batches.csv
    Untracked:  data/ici.vs.eoi_blup.full_chr-12_snpsqc_5.batches_mis.csv
    Untracked:  data/ici.vs.eoi_blup.full_chr-13_snpsqc_5.batches.csv
    Untracked:  data/ici.vs.eoi_blup.full_chr-13_snpsqc_5.batches_mis.csv
    Untracked:  data/ici.vs.eoi_blup.full_chr-14_snpsqc_5.batches.csv
    Untracked:  data/ici.vs.eoi_blup.full_chr-14_snpsqc_5.batches_mis.csv
    Untracked:  data/ici.vs.eoi_blup.full_chr-15_snpsqc_5.batches.csv
    Untracked:  data/ici.vs.eoi_blup.full_chr-15_snpsqc_5.batches_mis.csv
    Untracked:  data/ici.vs.eoi_blup.full_chr-16_snpsqc_5.batches.csv
    Untracked:  data/ici.vs.eoi_blup.full_chr-16_snpsqc_5.batches_mis.csv
    Untracked:  data/ici.vs.eoi_blup.full_chr-17_snpsqc_5.batches.csv
    Untracked:  data/ici.vs.eoi_blup.full_chr-17_snpsqc_5.batches_mis.csv
    Untracked:  data/ici.vs.eoi_blup.full_chr-18_snpsqc_5.batches.csv
    Untracked:  data/ici.vs.eoi_blup.full_chr-18_snpsqc_5.batches_mis.csv
    Untracked:  data/ici.vs.eoi_blup.full_chr-19_snpsqc_5.batches.csv
    Untracked:  data/ici.vs.eoi_blup.full_chr-19_snpsqc_5.batches_mis.csv
    Untracked:  data/ici.vs.eoi_blup.full_chr-1_snpsqc_5.batches.csv
    Untracked:  data/ici.vs.eoi_blup.full_chr-1_snpsqc_5.batches_mis.csv
    Untracked:  data/ici.vs.eoi_blup.full_chr-2_snpsqc_5.batches.csv
    Untracked:  data/ici.vs.eoi_blup.full_chr-2_snpsqc_5.batches_mis.csv
    Untracked:  data/ici.vs.eoi_blup.full_chr-3_snpsqc_5.batches.csv
    Untracked:  data/ici.vs.eoi_blup.full_chr-3_snpsqc_5.batches_mis.csv
    Untracked:  data/ici.vs.eoi_blup.full_chr-4_snpsqc_5.batches.csv
    Untracked:  data/ici.vs.eoi_blup.full_chr-4_snpsqc_5.batches_mis.csv
    Untracked:  data/ici.vs.eoi_blup.full_chr-5_snpsqc_5.batches.csv
    Untracked:  data/ici.vs.eoi_blup.full_chr-5_snpsqc_5.batches_mis.csv
    Untracked:  data/ici.vs.eoi_blup.full_chr-6_snpsqc_5.batches.csv
    Untracked:  data/ici.vs.eoi_blup.full_chr-6_snpsqc_5.batches_mis.csv
    Untracked:  data/ici.vs.eoi_blup.full_chr-7_snpsqc_5.batches.csv
    Untracked:  data/ici.vs.eoi_blup.full_chr-7_snpsqc_5.batches_mis.csv
    Untracked:  data/ici.vs.eoi_blup.full_chr-8_snpsqc_5.batches.csv
    Untracked:  data/ici.vs.eoi_blup.full_chr-8_snpsqc_5.batches_mis.csv
    Untracked:  data/ici.vs.eoi_blup.full_chr-9_snpsqc_5.batches.csv
    Untracked:  data/ici.vs.eoi_blup.full_chr-9_snpsqc_5.batches_mis.csv
    Untracked:  data/ici.vs.eoi_blup.full_chr-X_snpsqc_5.batches.csv
    Untracked:  data/ici.vs.eoi_blup.full_chr-X_snpsqc_5.batches_mis.csv
    Untracked:  data/ici.vs.eoi_blup.qc_chr-10_5.batches.csv
    Untracked:  data/ici.vs.eoi_blup.qc_chr-10_5.batches_mis.csv
    Untracked:  data/ici.vs.eoi_blup.qc_chr-10_lod.drop-1.5_snpsqc_5.batches.csv
    Untracked:  data/ici.vs.eoi_blup.qc_chr-10_snpsqc_5.batches.csv
    Untracked:  data/ici.vs.eoi_blup.qc_chr-10_snpsqc_5.batches_mis.csv
    Untracked:  data/ici.vs.eoi_blup.qc_chr-10_snpsqc_dis_no-x_updated_5.batches.csv
    Untracked:  data/ici.vs.eoi_blup.qc_chr-10_snpsqc_dis_no-x_updated_5.batches_mis.csv
    Untracked:  data/ici.vs.eoi_blup.qc_chr-11_5.batches.csv
    Untracked:  data/ici.vs.eoi_blup.qc_chr-11_5.batches_mis.csv
    Untracked:  data/ici.vs.eoi_blup.qc_chr-11_lod.drop-1.5_snpsqc_5.batches.csv
    Untracked:  data/ici.vs.eoi_blup.qc_chr-11_snpsqc_5.batches.csv
    Untracked:  data/ici.vs.eoi_blup.qc_chr-11_snpsqc_5.batches_mis.csv
    Untracked:  data/ici.vs.eoi_blup.qc_chr-11_snpsqc_dis_no-x_updated_5.batches.csv
    Untracked:  data/ici.vs.eoi_blup.qc_chr-11_snpsqc_dis_no-x_updated_5.batches_mis.csv
    Untracked:  data/ici.vs.eoi_blup.qc_chr-12_5.batches.csv
    Untracked:  data/ici.vs.eoi_blup.qc_chr-12_5.batches_mis.csv
    Untracked:  data/ici.vs.eoi_blup.qc_chr-12_lod.drop-1.5_snpsqc_5.batches.csv
    Untracked:  data/ici.vs.eoi_blup.qc_chr-12_snpsqc_5.batches.csv
    Untracked:  data/ici.vs.eoi_blup.qc_chr-12_snpsqc_5.batches_mis.csv
    Untracked:  data/ici.vs.eoi_blup.qc_chr-12_snpsqc_dis_no-x_updated_5.batches.csv
    Untracked:  data/ici.vs.eoi_blup.qc_chr-12_snpsqc_dis_no-x_updated_5.batches_mis.csv
    Untracked:  data/ici.vs.eoi_blup.qc_chr-13_5.batches.csv
    Untracked:  data/ici.vs.eoi_blup.qc_chr-13_5.batches_mis.csv
    Untracked:  data/ici.vs.eoi_blup.qc_chr-13_lod.drop-1.5_snpsqc_5.batches.csv
    Untracked:  data/ici.vs.eoi_blup.qc_chr-13_snpsqc_5.batches.csv
    Untracked:  data/ici.vs.eoi_blup.qc_chr-13_snpsqc_5.batches_mis.csv
    Untracked:  data/ici.vs.eoi_blup.qc_chr-13_snpsqc_dis_no-x_updated_5.batches.csv
    Untracked:  data/ici.vs.eoi_blup.qc_chr-13_snpsqc_dis_no-x_updated_5.batches_mis.csv
    Untracked:  data/ici.vs.eoi_blup.qc_chr-14_5.batches.csv
    Untracked:  data/ici.vs.eoi_blup.qc_chr-14_5.batches_mis.csv
    Untracked:  data/ici.vs.eoi_blup.qc_chr-14_snpsqc_5.batches.csv
    Untracked:  data/ici.vs.eoi_blup.qc_chr-14_snpsqc_5.batches_mis.csv
    Untracked:  data/ici.vs.eoi_blup.qc_chr-14_snpsqc_dis_no-x_updated_5.batches.csv
    Untracked:  data/ici.vs.eoi_blup.qc_chr-14_snpsqc_dis_no-x_updated_5.batches_mis.csv
    Untracked:  data/ici.vs.eoi_blup.qc_chr-15_5.batches.csv
    Untracked:  data/ici.vs.eoi_blup.qc_chr-15_5.batches_mis.csv
    Untracked:  data/ici.vs.eoi_blup.qc_chr-15_snpsqc_5.batches.csv
    Untracked:  data/ici.vs.eoi_blup.qc_chr-15_snpsqc_5.batches_mis.csv
    Untracked:  data/ici.vs.eoi_blup.qc_chr-15_snpsqc_dis_no-x_updated_5.batches_mis.csv
    Untracked:  data/ici.vs.eoi_blup.qc_chr-16_5.batches.csv
    Untracked:  data/ici.vs.eoi_blup.qc_chr-16_5.batches_mis.csv
    Untracked:  data/ici.vs.eoi_blup.qc_chr-16_snpsqc_5.batches.csv
    Untracked:  data/ici.vs.eoi_blup.qc_chr-16_snpsqc_5.batches_mis.csv
    Untracked:  data/ici.vs.eoi_blup.qc_chr-16_snpsqc_dis_no-x_updated_5.batches.csv
    Untracked:  data/ici.vs.eoi_blup.qc_chr-16_snpsqc_dis_no-x_updated_5.batches_mis.csv
    Untracked:  data/ici.vs.eoi_blup.qc_chr-17_5.batches.csv
    Untracked:  data/ici.vs.eoi_blup.qc_chr-17_5.batches_mis.csv
    Untracked:  data/ici.vs.eoi_blup.qc_chr-17_snpsqc_5.batches.csv
    Untracked:  data/ici.vs.eoi_blup.qc_chr-17_snpsqc_5.batches_mis.csv
    Untracked:  data/ici.vs.eoi_blup.qc_chr-17_snpsqc_dis_no-x_updated_5.batches.csv
    Untracked:  data/ici.vs.eoi_blup.qc_chr-17_snpsqc_dis_no-x_updated_5.batches_mis.csv
    Untracked:  data/ici.vs.eoi_blup.qc_chr-18_5.batches.csv
    Untracked:  data/ici.vs.eoi_blup.qc_chr-18_5.batches_mis.csv
    Untracked:  data/ici.vs.eoi_blup.qc_chr-18_snpsqc_5.batches.csv
    Untracked:  data/ici.vs.eoi_blup.qc_chr-18_snpsqc_5.batches_mis.csv
    Untracked:  data/ici.vs.eoi_blup.qc_chr-18_snpsqc_dis_no-x_updated_5.batches.csv
    Untracked:  data/ici.vs.eoi_blup.qc_chr-18_snpsqc_dis_no-x_updated_5.batches_mis.csv
    Untracked:  data/ici.vs.eoi_blup.qc_chr-19_5.batches.csv
    Untracked:  data/ici.vs.eoi_blup.qc_chr-19_5.batches_mis.csv
    Untracked:  data/ici.vs.eoi_blup.qc_chr-19_snpsqc_5.batches.csv
    Untracked:  data/ici.vs.eoi_blup.qc_chr-19_snpsqc_5.batches_mis.csv
    Untracked:  data/ici.vs.eoi_blup.qc_chr-19_snpsqc_dis_no-x_updated_5.batches.csv
    Untracked:  data/ici.vs.eoi_blup.qc_chr-19_snpsqc_dis_no-x_updated_5.batches_mis.csv
    Untracked:  data/ici.vs.eoi_blup.qc_chr-1_5.batches.csv
    Untracked:  data/ici.vs.eoi_blup.qc_chr-1_5.batches_mis.csv
    Untracked:  data/ici.vs.eoi_blup.qc_chr-1_lod.drop-1.5_snpsqc_5.batches.csv
    Untracked:  data/ici.vs.eoi_blup.qc_chr-1_snpsqc_5.batches.csv
    Untracked:  data/ici.vs.eoi_blup.qc_chr-1_snpsqc_5.batches_mis.csv
    Untracked:  data/ici.vs.eoi_blup.qc_chr-1_snpsqc_dis_no-x_updated_5.batches.csv
    Untracked:  data/ici.vs.eoi_blup.qc_chr-1_snpsqc_dis_no-x_updated_5.batches_mis.csv
    Untracked:  data/ici.vs.eoi_blup.qc_chr-2_5.batches.csv
    Untracked:  data/ici.vs.eoi_blup.qc_chr-2_5.batches_mis.csv
    Untracked:  data/ici.vs.eoi_blup.qc_chr-2_lod.drop-1.5_snpsqc_5.batches.csv
    Untracked:  data/ici.vs.eoi_blup.qc_chr-2_snpsqc_5.batches.csv
    Untracked:  data/ici.vs.eoi_blup.qc_chr-2_snpsqc_5.batches_mis.csv
    Untracked:  data/ici.vs.eoi_blup.qc_chr-2_snpsqc_dis_no-x_updated_5.batches.csv
    Untracked:  data/ici.vs.eoi_blup.qc_chr-2_snpsqc_dis_no-x_updated_5.batches_mis.csv
    Untracked:  data/ici.vs.eoi_blup.qc_chr-3_5.batches.csv
    Untracked:  data/ici.vs.eoi_blup.qc_chr-3_5.batches_mis.csv
    Untracked:  data/ici.vs.eoi_blup.qc_chr-3_lod.drop-1.5_snpsqc_5.batches.csv
    Untracked:  data/ici.vs.eoi_blup.qc_chr-3_snpsqc_5.batches.csv
    Untracked:  data/ici.vs.eoi_blup.qc_chr-3_snpsqc_5.batches_mis.csv
    Untracked:  data/ici.vs.eoi_blup.qc_chr-3_snpsqc_dis_no-x_updated_5.batches.csv
    Untracked:  data/ici.vs.eoi_blup.qc_chr-3_snpsqc_dis_no-x_updated_5.batches_mis.csv
    Untracked:  data/ici.vs.eoi_blup.qc_chr-4_5.batches.csv
    Untracked:  data/ici.vs.eoi_blup.qc_chr-4_5.batches_mis.csv
    Untracked:  data/ici.vs.eoi_blup.qc_chr-4_lod.drop-1.5_snpsqc_5.batches.csv
    Untracked:  data/ici.vs.eoi_blup.qc_chr-4_snpsqc_5.batches.csv
    Untracked:  data/ici.vs.eoi_blup.qc_chr-4_snpsqc_5.batches_mis.csv
    Untracked:  data/ici.vs.eoi_blup.qc_chr-4_snpsqc_dis_no-x_updated_5.batches.csv
    Untracked:  data/ici.vs.eoi_blup.qc_chr-4_snpsqc_dis_no-x_updated_5.batches_mis.csv
    Untracked:  data/ici.vs.eoi_blup.qc_chr-5_5.batches.csv
    Untracked:  data/ici.vs.eoi_blup.qc_chr-5_5.batches_mis.csv
    Untracked:  data/ici.vs.eoi_blup.qc_chr-5_lod.drop-1.5_snpsqc_5.batches.csv
    Untracked:  data/ici.vs.eoi_blup.qc_chr-5_snpsqc_5.batches.csv
    Untracked:  data/ici.vs.eoi_blup.qc_chr-5_snpsqc_5.batches_mis.csv
    Untracked:  data/ici.vs.eoi_blup.qc_chr-5_snpsqc_dis_no-x_updated_5.batches.csv
    Untracked:  data/ici.vs.eoi_blup.qc_chr-5_snpsqc_dis_no-x_updated_5.batches_mis.csv
    Untracked:  data/ici.vs.eoi_blup.qc_chr-6_5.batches.csv
    Untracked:  data/ici.vs.eoi_blup.qc_chr-6_5.batches_mis.csv
    Untracked:  data/ici.vs.eoi_blup.qc_chr-6_lod.drop-1.5_snpsqc_5.batches.csv
    Untracked:  data/ici.vs.eoi_blup.qc_chr-6_snpsqc_5.batches.csv
    Untracked:  data/ici.vs.eoi_blup.qc_chr-6_snpsqc_5.batches_mis.csv
    Untracked:  data/ici.vs.eoi_blup.qc_chr-6_snpsqc_dis_no-x_updated_5.batches.csv
    Untracked:  data/ici.vs.eoi_blup.qc_chr-6_snpsqc_dis_no-x_updated_5.batches_mis.csv
    Untracked:  data/ici.vs.eoi_blup.qc_chr-7_5.batches.csv
    Untracked:  data/ici.vs.eoi_blup.qc_chr-7_5.batches_mis.csv
    Untracked:  data/ici.vs.eoi_blup.qc_chr-7_lod.drop-1.5_snpsqc_5.batches.csv
    Untracked:  data/ici.vs.eoi_blup.qc_chr-7_snpsqc_5.batches.csv
    Untracked:  data/ici.vs.eoi_blup.qc_chr-7_snpsqc_5.batches_mis.csv
    Untracked:  data/ici.vs.eoi_blup.qc_chr-7_snpsqc_dis_no-x_updated_5.batches.csv
    Untracked:  data/ici.vs.eoi_blup.qc_chr-7_snpsqc_dis_no-x_updated_5.batches_mis.csv
    Untracked:  data/ici.vs.eoi_blup.qc_chr-8_5.batches.csv
    Untracked:  data/ici.vs.eoi_blup.qc_chr-8_5.batches_mis.csv
    Untracked:  data/ici.vs.eoi_blup.qc_chr-8_lod.drop-1.5_snpsqc_5.batches.csv
    Untracked:  data/ici.vs.eoi_blup.qc_chr-8_snpsqc_5.batches.csv
    Untracked:  data/ici.vs.eoi_blup.qc_chr-8_snpsqc_5.batches_mis.csv
    Untracked:  data/ici.vs.eoi_blup.qc_chr-8_snpsqc_dis_no-x_updated_5.batches.csv
    Untracked:  data/ici.vs.eoi_blup.qc_chr-8_snpsqc_dis_no-x_updated_5.batches_mis.csv
    Untracked:  data/ici.vs.eoi_blup.qc_chr-9_5.batches.csv
    Untracked:  data/ici.vs.eoi_blup.qc_chr-9_5.batches_mis.csv
    Untracked:  data/ici.vs.eoi_blup.qc_chr-9_lod.drop-1.5_snpsqc_5.batches.csv
    Untracked:  data/ici.vs.eoi_blup.qc_chr-9_snpsqc_5.batches.csv
    Untracked:  data/ici.vs.eoi_blup.qc_chr-9_snpsqc_5.batches_mis.csv
    Untracked:  data/ici.vs.eoi_blup.qc_chr-9_snpsqc_dis_no-x_updated_5.batches.csv
    Untracked:  data/ici.vs.eoi_blup.qc_chr-9_snpsqc_dis_no-x_updated_5.batches_mis.csv
    Untracked:  data/ici.vs.eoi_blup.qc_chr-X_5.batches.csv
    Untracked:  data/ici.vs.eoi_blup.qc_chr-X_5.batches_mis.csv
    Untracked:  data/ici.vs.eoi_blup.qc_chr-X_snpsqc_5.batches.csv
    Untracked:  data/ici.vs.eoi_blup.qc_chr-X_snpsqc_5.batches_mis.csv
    Untracked:  data/ici.vs.eoi_blup.qc_chr-X_snpsqc_dis_no-x_updated_5.batches.csv
    Untracked:  data/ici.vs.eoi_blup.qc_chr-X_snpsqc_dis_no-x_updated_5.batches_mis.csv
    Untracked:  data/ici.vs.eoi_blup_sub_chr-10_peak.marker-ICR4295_lod.drop-1.5_5.batches_mis.csv
    Untracked:  data/ici.vs.eoi_blup_sub_chr-10_peak.marker-JAX00292499_lod.drop-1.5_5.batches.csv
    Untracked:  data/ici.vs.eoi_blup_sub_chr-10_peak.marker-JAX00292927_lod.drop-1.5_5.batches.csv
    Untracked:  data/ici.vs.eoi_blup_sub_chr-10_peak.marker-JAX00292927_lod.drop-1.5_snpsqc_5.batches.csv
    Untracked:  data/ici.vs.eoi_blup_sub_chr-10_peak.marker-JAX00292927_lod.drop-1.5_snpsqc_5.batches_mis.csv
    Untracked:  data/ici.vs.eoi_blup_sub_chr-10_peak.marker-UNC18154330_lod.drop-1.5_5.batches.csv
    Untracked:  data/ici.vs.eoi_blup_sub_chr-10_peak.marker-UNC18195765_lod.drop-1.5_5.batches.csv
    Untracked:  data/ici.vs.eoi_blup_sub_chr-10_peak.marker-UNC18216614_lod.drop-1.5_5.batches.csv
    Untracked:  data/ici.vs.eoi_blup_sub_chr-10_peak.marker-UNC18240977_lod.drop-1.5_snpsqc_dis_no-x_updated_5.batches.csv
    Untracked:  data/ici.vs.eoi_blup_sub_chr-10_peak.marker-UNC18311938_lod.drop-1.5_5.batches.csv
    Untracked:  data/ici.vs.eoi_blup_sub_chr-10_peak.marker-UNC18856953_lod.drop-1.5_5.batches_mis.csv
    Untracked:  data/ici.vs.eoi_blup_sub_chr-10_peak.marker-UNCHS028236_lod.drop-1.5_snpsqc_5.batches_mis.csv
    Untracked:  data/ici.vs.eoi_blup_sub_chr-10_peak.marker-UNCHS028536_lod.drop-1.5_snpsqc_dis_no-x_updated_5.batches_mis.csv
    Untracked:  data/ici.vs.eoi_blup_sub_chr-10_peak.marker-UNCJPD004316_lod.drop-1.5_5.batches_mis.csv
    Untracked:  data/ici.vs.eoi_blup_sub_chr-10_peak.marker-UNCJPD009532_lod.drop-1.5_5.batches_mis.csv
    Untracked:  data/ici.vs.eoi_blup_sub_chr-11_peak.marker-UNC19155926_lod.drop-1.5_5.batches_mis.csv
    Untracked:  data/ici.vs.eoi_blup_sub_chr-11_peak.marker-UNC19481366_lod.drop-1.5_snpsqc_5.batches_mis.csv
    Untracked:  data/ici.vs.eoi_blup_sub_chr-11_peak.marker-UNC19970181_lod.drop-1.5_5.batches_mis.csv
    Untracked:  data/ici.vs.eoi_blup_sub_chr-11_peak.marker-UNC19970181_lod.drop-1.5_snpsqc_5.batches_mis.csv
    Untracked:  data/ici.vs.eoi_blup_sub_chr-11_peak.marker-UNCJPD004792_lod.drop-1.5_5.batches_mis.csv
    Untracked:  data/ici.vs.eoi_blup_sub_chr-12_peak.marker-ICR4497_lod.drop-1.5_5.batches_mis.csv
    Untracked:  data/ici.vs.eoi_blup_sub_chr-12_peak.marker-ICR499_lod.drop-1.5_snpsqc_5.batches_mis.csv
    Untracked:  data/ici.vs.eoi_blup_sub_chr-13_peak.marker-UNCHS036773_lod.drop-1.5_snpsqc_5.batches_mis.csv
    Untracked:  data/ici.vs.eoi_blup_sub_chr-13_peak.marker-UNCHS036964_lod.drop-1.5_5.batches_mis.csv
    Untracked:  data/ici.vs.eoi_blup_sub_chr-14_peak.marker-UNC24056202_lod.drop-1.5_5.batches_mis.csv
    Untracked:  data/ici.vs.eoi_blup_sub_chr-14_peak.marker-UNC24056202_lod.drop-1.5_snpsqc_5.batches_mis.csv
    Untracked:  data/ici.vs.eoi_blup_sub_chr-15_peak.marker-UNC26070435_lod.drop-1.5_snpsqc_dis_no-x_updated_5.batches_mis.csv
    Untracked:  data/ici.vs.eoi_blup_sub_chr-17_peak.marker-UNCHS044241_lod.drop-1.5_5.batches_mis.csv
    Untracked:  data/ici.vs.eoi_blup_sub_chr-17_peak.marker-UNCHS044241_lod.drop-1.5_snpsqc_5.batches_mis.csv
    Untracked:  data/ici.vs.eoi_blup_sub_chr-17_peak.marker-UNCrs47191360_lod.drop-1.5_5.batches_mis.csv
    Untracked:  data/ici.vs.eoi_blup_sub_chr-17_peak.marker-UNCrs47191360_lod.drop-1.5_snpsqc_dis_no-x_updated_5.batches_mis.csv
    Untracked:  data/ici.vs.eoi_blup_sub_chr-18_peak.marker-UNCHS045343_lod.drop-1.5_snpsqc_5.batches_mis.csv
    Untracked:  data/ici.vs.eoi_blup_sub_chr-18_peak.marker-UNCHS045344_lod.drop-1.5_5.batches_mis.csv
    Untracked:  data/ici.vs.eoi_blup_sub_chr-19_peak.marker-UNCJPD007087_lod.drop-1.5_5.batches_mis.csv
    Untracked:  data/ici.vs.eoi_blup_sub_chr-1_peak.marker-ICR010_lod.drop-1.5_5.batches.csv
    Untracked:  data/ici.vs.eoi_blup_sub_chr-1_peak.marker-ICR010_lod.drop-1.5_5.batches_mis.csv
    Untracked:  data/ici.vs.eoi_blup_sub_chr-2_peak.marker-UNC4609527_lod.drop-1.5_snpsqc_dis_no-x_updated_5.batches_mis.csv
    Untracked:  data/ici.vs.eoi_blup_sub_chr-2_peak.marker-UNC4609660_lod.drop-1.5_5.batches_mis.csv
    Untracked:  data/ici.vs.eoi_blup_sub_chr-3_peak.marker-B6_rs31740628_lod.drop-1.5_5.batches_mis.csv
    Untracked:  data/ici.vs.eoi_blup_sub_chr-3_peak.marker-ICR1269_lod.drop-1.5_5.batches_mis.csv
    Untracked:  data/ici.vs.eoi_blup_sub_chr-3_peak.marker-ICR1338_lod.drop-1.5_5.batches.csv
    Untracked:  data/ici.vs.eoi_blup_sub_chr-3_peak.marker-ICR5325_lod.drop-1.5_5.batches_mis.csv
    Untracked:  data/ici.vs.eoi_blup_sub_chr-3_peak.marker-ICR5347_lod.drop-1.5_5.batches.csv
    Untracked:  data/ici.vs.eoi_blup_sub_chr-3_peak.marker-ICR5348_lod.drop-1.5_5.batches_mis.csv
    Untracked:  data/ici.vs.eoi_blup_sub_chr-3_peak.marker-JAX00104971_lod.drop-1.5_5.batches_mis.csv
    Untracked:  data/ici.vs.eoi_blup_sub_chr-3_peak.marker-JAX00105059_lod.drop-1.5_5.batches.csv
    Untracked:  data/ici.vs.eoi_blup_sub_chr-3_peak.marker-JAX00105203_lod.drop-1.5_5.batches_mis.csv
    Untracked:  data/ici.vs.eoi_blup_sub_chr-3_peak.marker-JAX00106957_lod.drop-1.5_5.batches.csv
    Untracked:  data/ici.vs.eoi_blup_sub_chr-3_peak.marker-JAX00106957_lod.drop-1.5_5.batches_mis.csv
    Untracked:  data/ici.vs.eoi_blup_sub_chr-3_peak.marker-JAX00107930_lod.drop-1.5_5.batches_mis.csv
    Untracked:  data/ici.vs.eoi_blup_sub_chr-3_peak.marker-JAX00107980_lod.drop-1.5_5.batches_mis.csv
    Untracked:  data/ici.vs.eoi_blup_sub_chr-3_peak.marker-JAX00108068_lod.drop-1.5_5.batches_mis.csv
    Untracked:  data/ici.vs.eoi_blup_sub_chr-3_peak.marker-JAX00108114_lod.drop-1.5_5.batches_mis.csv
    Untracked:  data/ici.vs.eoi_blup_sub_chr-3_peak.marker-JAX00109750_lod.drop-1.5_5.batches_mis.csv
    Untracked:  data/ici.vs.eoi_blup_sub_chr-3_peak.marker-JAX00189245_lod.drop-1.5_5.batches_mis.csv
    Untracked:  data/ici.vs.eoi_blup_sub_chr-3_peak.marker-JAX00514662_lod.drop-1.5_5.batches.csv
    Untracked:  data/ici.vs.eoi_blup_sub_chr-3_peak.marker-JAX00517218_lod.drop-1.5_5.batches_mis.csv
    Untracked:  data/ici.vs.eoi_blup_sub_chr-3_peak.marker-JAX00517674_lod.drop-1.5_5.batches_mis.csv
    Untracked:  data/ici.vs.eoi_blup_sub_chr-3_peak.marker-JAX00517906_lod.drop-1.5_5.batches_mis.csv
    Untracked:  data/ici.vs.eoi_blup_sub_chr-3_peak.marker-JAX00518418_lod.drop-1.5_5.batches_mis.csv
    Untracked:  data/ici.vs.eoi_blup_sub_chr-3_peak.marker-JAX00518684_lod.drop-1.5_5.batches_mis.csv
    Untracked:  data/ici.vs.eoi_blup_sub_chr-3_peak.marker-JAX00518911r_lod.drop-1.5_5.batches_mis.csv
    Untracked:  data/ici.vs.eoi_blup_sub_chr-3_peak.marker-JAX00520122_lod.drop-1.5_5.batches_mis.csv
    Untracked:  data/ici.vs.eoi_blup_sub_chr-3_peak.marker-JAX00524717_lod.drop-1.5_5.batches_mis.csv
    Untracked:  data/ici.vs.eoi_blup_sub_chr-3_peak.marker-JAX00524828_lod.drop-1.5_5.batches_mis.csv
    Untracked:  data/ici.vs.eoi_blup_sub_chr-3_peak.marker-JAX00525380_lod.drop-1.5_5.batches_mis.csv
    Untracked:  data/ici.vs.eoi_blup_sub_chr-3_peak.marker-JAX00525709_lod.drop-1.5_5.batches_mis.csv
    Untracked:  data/ici.vs.eoi_blup_sub_chr-3_peak.marker-JAX00527291_lod.drop-1.5_5.batches_mis.csv
    Untracked:  data/ici.vs.eoi_blup_sub_chr-3_peak.marker-JAX00527587_lod.drop-1.5_5.batches_mis.csv
    Untracked:  data/ici.vs.eoi_blup_sub_chr-3_peak.marker-JAX00529707_lod.drop-1.5_5.batches_mis.csv
    Untracked:  data/ici.vs.eoi_blup_sub_chr-3_peak.marker-JAX00529707r_lod.drop-1.5_5.batches.csv
    Untracked:  data/ici.vs.eoi_blup_sub_chr-3_peak.marker-JAX00531601_lod.drop-1.5_5.batches_mis.csv
    Untracked:  data/ici.vs.eoi_blup_sub_chr-3_peak.marker-JAX00532122_lod.drop-1.5_5.batches_mis.csv
    Untracked:  data/ici.vs.eoi_blup_sub_chr-3_peak.marker-UNC4673471_lod.drop-1.5_5.batches_mis.csv
    Untracked:  data/ici.vs.eoi_blup_sub_chr-3_peak.marker-UNC4681411_lod.drop-1.5_5.batches_mis.csv
    Untracked:  data/ici.vs.eoi_blup_sub_chr-3_peak.marker-UNC4793677_lod.drop-1.5_5.batches_mis.csv
    Untracked:  data/ici.vs.eoi_blup_sub_chr-3_peak.marker-UNC4799475_lod.drop-1.5_5.batches_mis.csv
    Untracked:  data/ici.vs.eoi_blup_sub_chr-3_peak.marker-UNC4809586_lod.drop-1.5_5.batches_mis.csv
    Untracked:  data/ici.vs.eoi_blup_sub_chr-3_peak.marker-UNC4813186_lod.drop-1.5_5.batches.csv
    Untracked:  data/ici.vs.eoi_blup_sub_chr-3_peak.marker-UNC4813186_lod.drop-1.5_5.batches_mis.csv
    Untracked:  data/ici.vs.eoi_blup_sub_chr-3_peak.marker-UNC4820379_lod.drop-1.5_5.batches_mis.csv
    Untracked:  data/ici.vs.eoi_blup_sub_chr-3_peak.marker-UNC4875653_lod.drop-1.5_5.batches_mis.csv
    Untracked:  data/ici.vs.eoi_blup_sub_chr-3_peak.marker-UNC4909830_lod.drop-1.5_5.batches_mis.csv
    Untracked:  data/ici.vs.eoi_blup_sub_chr-3_peak.marker-UNC4912288_lod.drop-1.5_5.batches_mis.csv
    Untracked:  data/ici.vs.eoi_blup_sub_chr-3_peak.marker-UNC4914130_lod.drop-1.5_5.batches_mis.csv
    Untracked:  data/ici.vs.eoi_blup_sub_chr-3_peak.marker-UNC4923398_lod.drop-1.5_5.batches_mis.csv
    Untracked:  data/ici.vs.eoi_blup_sub_chr-3_peak.marker-UNC4924377_lod.drop-1.5_5.batches.csv
    Untracked:  data/ici.vs.eoi_blup_sub_chr-3_peak.marker-UNC4945186_lod.drop-1.5_5.batches_mis.csv
    Untracked:  data/ici.vs.eoi_blup_sub_chr-3_peak.marker-UNC4947492_lod.drop-1.5_5.batches_mis.csv
    Untracked:  data/ici.vs.eoi_blup_sub_chr-3_peak.marker-UNC4957168_lod.drop-1.5_5.batches.csv
    Untracked:  data/ici.vs.eoi_blup_sub_chr-3_peak.marker-UNC4957859_lod.drop-1.5_5.batches_mis.csv
    Untracked:  data/ici.vs.eoi_blup_sub_chr-3_peak.marker-UNC4964062_lod.drop-1.5_5.batches_mis.csv
    Untracked:  data/ici.vs.eoi_blup_sub_chr-3_peak.marker-UNC4979914_lod.drop-1.5_5.batches_mis.csv
    Untracked:  data/ici.vs.eoi_blup_sub_chr-3_peak.marker-UNC4985064_lod.drop-1.5_5.batches_mis.csv
    Untracked:  data/ici.vs.eoi_blup_sub_chr-3_peak.marker-UNC4990111_lod.drop-1.5_5.batches_mis.csv
    Untracked:  data/ici.vs.eoi_blup_sub_chr-3_peak.marker-UNC4991565_lod.drop-1.5_5.batches_mis.csv
    Untracked:  data/ici.vs.eoi_blup_sub_chr-3_peak.marker-UNC5001723_lod.drop-1.5_5.batches_mis.csv
    Untracked:  data/ici.vs.eoi_blup_sub_chr-3_peak.marker-UNC5008656_lod.drop-1.5_5.batches.csv
    Untracked:  data/ici.vs.eoi_blup_sub_chr-3_peak.marker-UNC5008656_lod.drop-1.5_5.batches_mis.csv
    Untracked:  data/ici.vs.eoi_blup_sub_chr-3_peak.marker-UNC5042361_lod.drop-1.5_5.batches_mis.csv
    Untracked:  data/ici.vs.eoi_blup_sub_chr-3_peak.marker-UNC5055259_lod.drop-1.5_5.batches_mis.csv
    Untracked:  data/ici.vs.eoi_blup_sub_chr-3_peak.marker-UNC5076935_lod.drop-1.5_5.batches_mis.csv
    Untracked:  data/ici.vs.eoi_blup_sub_chr-3_peak.marker-UNC5082757_lod.drop-1.5_5.batches_mis.csv
    Untracked:  data/ici.vs.eoi_blup_sub_chr-3_peak.marker-UNC5083168_lod.drop-1.5_5.batches.csv
    Untracked:  data/ici.vs.eoi_blup_sub_chr-3_peak.marker-UNC5173293_lod.drop-1.5_5.batches_mis.csv
    Untracked:  data/ici.vs.eoi_blup_sub_chr-3_peak.marker-UNC5182782_lod.drop-1.5_5.batches_mis.csv
    Untracked:  data/ici.vs.eoi_blup_sub_chr-3_peak.marker-UNC5192582_lod.drop-1.5_5.batches.csv
    Untracked:  data/ici.vs.eoi_blup_sub_chr-3_peak.marker-UNC5193970_lod.drop-1.5_5.batches_mis.csv
    Untracked:  data/ici.vs.eoi_blup_sub_chr-3_peak.marker-UNC5213888_lod.drop-1.5_snpsqc_5.batches_mis.csv
    Untracked:  data/ici.vs.eoi_blup_sub_chr-3_peak.marker-UNC5240484_lod.drop-1.5_5.batches_mis.csv
    Untracked:  data/ici.vs.eoi_blup_sub_chr-3_peak.marker-UNC5256922_lod.drop-1.5_5.batches_mis.csv
    Untracked:  data/ici.vs.eoi_blup_sub_chr-3_peak.marker-UNC5266664_lod.drop-1.5_5.batches_mis.csv
    Untracked:  data/ici.vs.eoi_blup_sub_chr-3_peak.marker-UNC5266711_lod.drop-1.5_5.batches.csv
    Untracked:  data/ici.vs.eoi_blup_sub_chr-3_peak.marker-UNC5267813_lod.drop-1.5_5.batches_mis.csv
    Untracked:  data/ici.vs.eoi_blup_sub_chr-3_peak.marker-UNC5274554_lod.drop-1.5_5.batches_mis.csv
    Untracked:  data/ici.vs.eoi_blup_sub_chr-3_peak.marker-UNC5277061_lod.drop-1.5_5.batches_mis.csv
    Untracked:  data/ici.vs.eoi_blup_sub_chr-3_peak.marker-UNC5282259_lod.drop-1.5_5.batches_mis.csv
    Untracked:  data/ici.vs.eoi_blup_sub_chr-3_peak.marker-UNC5287977_lod.drop-1.5_5.batches_mis.csv
    Untracked:  data/ici.vs.eoi_blup_sub_chr-3_peak.marker-UNC5301460_lod.drop-1.5_5.batches_mis.csv
    Untracked:  data/ici.vs.eoi_blup_sub_chr-3_peak.marker-UNC5321559_lod.drop-1.5_5.batches_mis.csv
    Untracked:  data/ici.vs.eoi_blup_sub_chr-3_peak.marker-UNC5322455_lod.drop-1.5_5.batches_mis.csv
    Untracked:  data/ici.vs.eoi_blup_sub_chr-3_peak.marker-UNC5328596_lod.drop-1.5_5.batches_mis.csv
    Untracked:  data/ici.vs.eoi_blup_sub_chr-3_peak.marker-UNC5350384_lod.drop-1.5_5.batches_mis.csv
    Untracked:  data/ici.vs.eoi_blup_sub_chr-3_peak.marker-UNC5354724_lod.drop-1.5_5.batches_mis.csv
    Untracked:  data/ici.vs.eoi_blup_sub_chr-3_peak.marker-UNC5375208_lod.drop-1.5_5.batches_mis.csv
    Untracked:  data/ici.vs.eoi_blup_sub_chr-3_peak.marker-UNC5384072_lod.drop-1.5_5.batches_mis.csv
    Untracked:  data/ici.vs.eoi_blup_sub_chr-3_peak.marker-UNC5400314_lod.drop-1.5_5.batches_mis.csv
    Untracked:  data/ici.vs.eoi_blup_sub_chr-3_peak.marker-UNC5433945_lod.drop-1.5_5.batches_mis.csv
    Untracked:  data/ici.vs.eoi_blup_sub_chr-3_peak.marker-UNC5435874_lod.drop-1.5_5.batches_mis.csv
    Untracked:  data/ici.vs.eoi_blup_sub_chr-3_peak.marker-UNC5454839_lod.drop-1.5_5.batches.csv
    Untracked:  data/ici.vs.eoi_blup_sub_chr-3_peak.marker-UNC5454839_lod.drop-1.5_5.batches_mis.csv
    Untracked:  data/ici.vs.eoi_blup_sub_chr-3_peak.marker-UNC5466669_lod.drop-1.5_5.batches_mis.csv
    Untracked:  data/ici.vs.eoi_blup_sub_chr-3_peak.marker-UNC5517972_lod.drop-1.5_5.batches_mis.csv
    Untracked:  data/ici.vs.eoi_blup_sub_chr-3_peak.marker-UNC5522257_lod.drop-1.5_5.batches_mis.csv
    Untracked:  data/ici.vs.eoi_blup_sub_chr-3_peak.marker-UNC5523265_lod.drop-1.5_5.batches_mis.csv
    Untracked:  data/ici.vs.eoi_blup_sub_chr-3_peak.marker-UNC5532424_lod.drop-1.5_5.batches_mis.csv
    Untracked:  data/ici.vs.eoi_blup_sub_chr-3_peak.marker-UNC5553882_lod.drop-1.5_5.batches_mis.csv
    Untracked:  data/ici.vs.eoi_blup_sub_chr-3_peak.marker-UNC5572051_lod.drop-1.5_5.batches_mis.csv
    Untracked:  data/ici.vs.eoi_blup_sub_chr-3_peak.marker-UNC5582753_lod.drop-1.5_5.batches.csv
    Untracked:  data/ici.vs.eoi_blup_sub_chr-3_peak.marker-UNC5589260_lod.drop-1.5_5.batches_mis.csv
    Untracked:  data/ici.vs.eoi_blup_sub_chr-3_peak.marker-UNC5595555_lod.drop-1.5_5.batches_mis.csv
    Untracked:  data/ici.vs.eoi_blup_sub_chr-3_peak.marker-UNC5623239_lod.drop-1.5_5.batches_mis.csv
    Untracked:  data/ici.vs.eoi_blup_sub_chr-3_peak.marker-UNC5639830_lod.drop-1.5_5.batches_mis.csv
    Untracked:  data/ici.vs.eoi_blup_sub_chr-3_peak.marker-UNC5645844_lod.drop-1.5_5.batches.csv
    Untracked:  data/ici.vs.eoi_blup_sub_chr-3_peak.marker-UNC5645844_lod.drop-1.5_5.batches_mis.csv
    Untracked:  data/ici.vs.eoi_blup_sub_chr-3_peak.marker-UNC5680234_lod.drop-1.5_5.batches_mis.csv
    Untracked:  data/ici.vs.eoi_blup_sub_chr-3_peak.marker-UNC5682314_lod.drop-1.5_5.batches_mis.csv
    Untracked:  data/ici.vs.eoi_blup_sub_chr-3_peak.marker-UNC5725356_lod.drop-1.5_5.batches_mis.csv
    Untracked:  data/ici.vs.eoi_blup_sub_chr-3_peak.marker-UNC5727351_lod.drop-1.5_5.batches_mis.csv
    Untracked:  data/ici.vs.eoi_blup_sub_chr-3_peak.marker-UNC5734420_lod.drop-1.5_5.batches_mis.csv
    Untracked:  data/ici.vs.eoi_blup_sub_chr-3_peak.marker-UNC5743257_lod.drop-1.5_5.batches_mis.csv
    Untracked:  data/ici.vs.eoi_blup_sub_chr-3_peak.marker-UNC5751567_lod.drop-1.5_5.batches_mis.csv
    Untracked:  data/ici.vs.eoi_blup_sub_chr-3_peak.marker-UNC5752623_lod.drop-1.5_5.batches_mis.csv
    Untracked:  data/ici.vs.eoi_blup_sub_chr-3_peak.marker-UNC5776974_lod.drop-1.5_5.batches_mis.csv
    Untracked:  data/ici.vs.eoi_blup_sub_chr-3_peak.marker-UNC5778977_lod.drop-1.5_5.batches_mis.csv
    Untracked:  data/ici.vs.eoi_blup_sub_chr-3_peak.marker-UNC5788507_lod.drop-1.5_5.batches_mis.csv
    Untracked:  data/ici.vs.eoi_blup_sub_chr-3_peak.marker-UNC5792468_lod.drop-1.5_5.batches_mis.csv
    Untracked:  data/ici.vs.eoi_blup_sub_chr-3_peak.marker-UNC5794598_lod.drop-1.5_5.batches_mis.csv
    Untracked:  data/ici.vs.eoi_blup_sub_chr-3_peak.marker-UNC5795561_lod.drop-1.5_5.batches_mis.csv
    Untracked:  data/ici.vs.eoi_blup_sub_chr-3_peak.marker-UNC5802298_lod.drop-1.5_5.batches_mis.csv
    Untracked:  data/ici.vs.eoi_blup_sub_chr-3_peak.marker-UNC5806628_lod.drop-1.5_5.batches_mis.csv
    Untracked:  data/ici.vs.eoi_blup_sub_chr-3_peak.marker-UNC5808246_lod.drop-1.5_5.batches.csv
    Untracked:  data/ici.vs.eoi_blup_sub_chr-3_peak.marker-UNC5817478_lod.drop-1.5_5.batches_mis.csv
    Untracked:  data/ici.vs.eoi_blup_sub_chr-3_peak.marker-UNC5836999_lod.drop-1.5_5.batches_mis.csv
    Untracked:  data/ici.vs.eoi_blup_sub_chr-3_peak.marker-UNC5841501_lod.drop-1.5_5.batches_mis.csv
    Untracked:  data/ici.vs.eoi_blup_sub_chr-3_peak.marker-UNC5847927_lod.drop-1.5_5.batches_mis.csv
    Untracked:  data/ici.vs.eoi_blup_sub_chr-3_peak.marker-UNC5966417_lod.drop-1.5_5.batches_mis.csv
    Untracked:  data/ici.vs.eoi_blup_sub_chr-3_peak.marker-UNCHS008129_lod.drop-1.5_5.batches_mis.csv
    Untracked:  data/ici.vs.eoi_blup_sub_chr-3_peak.marker-UNCHS008173_lod.drop-1.5_5.batches_mis.csv
    Untracked:  data/ici.vs.eoi_blup_sub_chr-3_peak.marker-UNCHS008182_lod.drop-1.5_5.batches.csv
    Untracked:  data/ici.vs.eoi_blup_sub_chr-3_peak.marker-UNCHS008193_lod.drop-1.5_5.batches_mis.csv
    Untracked:  data/ici.vs.eoi_blup_sub_chr-3_peak.marker-UNCHS008209_lod.drop-1.5_5.batches_mis.csv
    Untracked:  data/ici.vs.eoi_blup_sub_chr-3_peak.marker-UNCHS008215_lod.drop-1.5_5.batches_mis.csv
    Untracked:  data/ici.vs.eoi_blup_sub_chr-3_peak.marker-UNCHS008219_lod.drop-1.5_5.batches_mis.csv
    Untracked:  data/ici.vs.eoi_blup_sub_chr-3_peak.marker-UNCHS008231_lod.drop-1.5_5.batches_mis.csv
    Untracked:  data/ici.vs.eoi_blup_sub_chr-3_peak.marker-UNCHS008235_lod.drop-1.5_5.batches_mis.csv
    Untracked:  data/ici.vs.eoi_blup_sub_chr-3_peak.marker-UNCHS008242_lod.drop-1.5_5.batches.csv
    Untracked:  data/ici.vs.eoi_blup_sub_chr-3_peak.marker-UNCHS008253_lod.drop-1.5_5.batches_mis.csv
    Untracked:  data/ici.vs.eoi_blup_sub_chr-3_peak.marker-UNCHS008256_lod.drop-1.5_5.batches_mis.csv
    Untracked:  data/ici.vs.eoi_blup_sub_chr-3_peak.marker-UNCHS008259_lod.drop-1.5_5.batches_mis.csv
    Untracked:  data/ici.vs.eoi_blup_sub_chr-3_peak.marker-UNCHS008271_lod.drop-1.5_5.batches_mis.csv
    Untracked:  data/ici.vs.eoi_blup_sub_chr-3_peak.marker-UNCHS008286_lod.drop-1.5_5.batches.csv
    Untracked:  data/ici.vs.eoi_blup_sub_chr-3_peak.marker-UNCHS008286_lod.drop-1.5_5.batches_mis.csv
    Untracked:  data/ici.vs.eoi_blup_sub_chr-3_peak.marker-UNCHS008311_lod.drop-1.5_5.batches_mis.csv
    Untracked:  data/ici.vs.eoi_blup_sub_chr-3_peak.marker-UNCHS008317_lod.drop-1.5_5.batches_mis.csv
    Untracked:  data/ici.vs.eoi_blup_sub_chr-3_peak.marker-UNCHS008372_lod.drop-1.5_5.batches_mis.csv
    Untracked:  data/ici.vs.eoi_blup_sub_chr-3_peak.marker-UNCHS008379_lod.drop-1.5_5.batches_mis.csv
    Untracked:  data/ici.vs.eoi_blup_sub_chr-3_peak.marker-UNCHS008410_lod.drop-1.5_5.batches_mis.csv
    Untracked:  data/ici.vs.eoi_blup_sub_chr-3_peak.marker-UNCHS008415_lod.drop-1.5_5.batches_mis.csv
    Untracked:  data/ici.vs.eoi_blup_sub_chr-3_peak.marker-UNCHS008431_lod.drop-1.5_5.batches_mis.csv
    Untracked:  data/ici.vs.eoi_blup_sub_chr-3_peak.marker-UNCHS008487_lod.drop-1.5_snpsqc_5.batches.csv
    Untracked:  data/ici.vs.eoi_blup_sub_chr-3_peak.marker-UNCHS008487_lod.drop-1.5_snpsqc_dis_no-x_updated_5.batches.csv
    Untracked:  data/ici.vs.eoi_blup_sub_chr-3_peak.marker-UNCHS008487_lod.drop-1.5_snpsqc_dis_no-x_updated_5.batches_mis.csv
    Untracked:  data/ici.vs.eoi_blup_sub_chr-3_peak.marker-UNCHS008511_lod.drop-1.5_snpsqc_5.batches_mis.csv
    Untracked:  data/ici.vs.eoi_blup_sub_chr-3_peak.marker-UNCHS008609_lod.drop-1.5_5.batches_mis.csv
    Untracked:  data/ici.vs.eoi_blup_sub_chr-3_peak.marker-UNCHS008614_lod.drop-1.5_5.batches.csv
    Untracked:  data/ici.vs.eoi_blup_sub_chr-3_peak.marker-UNCHS008649_lod.drop-1.5_5.batches_mis.csv
    Untracked:  data/ici.vs.eoi_blup_sub_chr-3_peak.marker-UNCHS008651_lod.drop-1.5_5.batches.csv
    Untracked:  data/ici.vs.eoi_blup_sub_chr-3_peak.marker-UNCHS008656_lod.drop-1.5_5.batches_mis.csv
    Untracked:  data/ici.vs.eoi_blup_sub_chr-3_peak.marker-UNCHS008723_lod.drop-1.5_5.batches_mis.csv
    Untracked:  data/ici.vs.eoi_blup_sub_chr-3_peak.marker-UNCHS008802_lod.drop-1.5_5.batches_mis.csv
    Untracked:  data/ici.vs.eoi_blup_sub_chr-3_peak.marker-UNCHS008803_lod.drop-1.5_5.batches_mis.csv
    Untracked:  data/ici.vs.eoi_blup_sub_chr-3_peak.marker-UNCHS008808_lod.drop-1.5_5.batches_mis.csv
    Untracked:  data/ici.vs.eoi_blup_sub_chr-3_peak.marker-UNCHS008815_lod.drop-1.5_5.batches.csv
    Untracked:  data/ici.vs.eoi_blup_sub_chr-3_peak.marker-UNCHS008815_lod.drop-1.5_5.batches_mis.csv
    Untracked:  data/ici.vs.eoi_blup_sub_chr-3_peak.marker-UNCHS008834_lod.drop-1.5_5.batches_mis.csv
    Untracked:  data/ici.vs.eoi_blup_sub_chr-3_peak.marker-UNCHS008838_lod.drop-1.5_5.batches_mis.csv
    Untracked:  data/ici.vs.eoi_blup_sub_chr-3_peak.marker-UNCHS008914_lod.drop-1.5_5.batches_mis.csv
    Untracked:  data/ici.vs.eoi_blup_sub_chr-3_peak.marker-UNCHS008928_lod.drop-1.5_5.batches_mis.csv
    Untracked:  data/ici.vs.eoi_blup_sub_chr-3_peak.marker-UNCHS008948_lod.drop-1.5_5.batches_mis.csv
    Untracked:  data/ici.vs.eoi_blup_sub_chr-3_peak.marker-UNCHS008959_lod.drop-1.5_5.batches_mis.csv
    Untracked:  data/ici.vs.eoi_blup_sub_chr-3_peak.marker-UNCHS008973_lod.drop-1.5_5.batches_mis.csv
    Untracked:  data/ici.vs.eoi_blup_sub_chr-3_peak.marker-UNCHS008987_lod.drop-1.5_5.batches_mis.csv
    Untracked:  data/ici.vs.eoi_blup_sub_chr-3_peak.marker-UNCHS009001_lod.drop-1.5_5.batches_mis.csv
    Untracked:  data/ici.vs.eoi_blup_sub_chr-3_peak.marker-UNCHS009025_lod.drop-1.5_5.batches_mis.csv
    Untracked:  data/ici.vs.eoi_blup_sub_chr-3_peak.marker-UNCHS009034_lod.drop-1.5_5.batches_mis.csv
    Untracked:  data/ici.vs.eoi_blup_sub_chr-3_peak.marker-UNCHS009087_lod.drop-1.5_5.batches_mis.csv
    Untracked:  data/ici.vs.eoi_blup_sub_chr-3_peak.marker-UNCHS009109_lod.drop-1.5_5.batches_mis.csv
    Untracked:  data/ici.vs.eoi_blup_sub_chr-3_peak.marker-UNCHS009142_lod.drop-1.5_5.batches_mis.csv
    Untracked:  data/ici.vs.eoi_blup_sub_chr-3_peak.marker-UNCHS009207_lod.drop-1.5_5.batches_mis.csv
    Untracked:  data/ici.vs.eoi_blup_sub_chr-3_peak.marker-UNCHS009216_lod.drop-1.5_5.batches_mis.csv
    Untracked:  data/ici.vs.eoi_blup_sub_chr-3_peak.marker-UNCHS009224_lod.drop-1.5_5.batches.csv
    Untracked:  data/ici.vs.eoi_blup_sub_chr-3_peak.marker-UNCHS009306_lod.drop-1.5_5.batches_mis.csv
    Untracked:  data/ici.vs.eoi_blup_sub_chr-3_peak.marker-UNCHS009316_lod.drop-1.5_5.batches_mis.csv
    Untracked:  data/ici.vs.eoi_blup_sub_chr-3_peak.marker-UNCHS009364_lod.drop-1.5_5.batches_mis.csv
    Untracked:  data/ici.vs.eoi_blup_sub_chr-3_peak.marker-UNCHS009377_lod.drop-1.5_5.batches_mis.csv
    Untracked:  data/ici.vs.eoi_blup_sub_chr-3_peak.marker-UNCHS009382_lod.drop-1.5_5.batches_mis.csv
    Untracked:  data/ici.vs.eoi_blup_sub_chr-3_peak.marker-UNCHS009383_lod.drop-1.5_5.batches.csv
    Untracked:  data/ici.vs.eoi_blup_sub_chr-3_peak.marker-UNCHS009387_lod.drop-1.5_5.batches_mis.csv
    Untracked:  data/ici.vs.eoi_blup_sub_chr-3_peak.marker-UNCHS009395_lod.drop-1.5_5.batches_mis.csv
    Untracked:  data/ici.vs.eoi_blup_sub_chr-3_peak.marker-UNCHS009413_lod.drop-1.5_5.batches_mis.csv
    Untracked:  data/ici.vs.eoi_blup_sub_chr-3_peak.marker-UNCHS009428_lod.drop-1.5_5.batches_mis.csv
    Untracked:  data/ici.vs.eoi_blup_sub_chr-3_peak.marker-UNCHS009463_lod.drop-1.5_5.batches_mis.csv
    Untracked:  data/ici.vs.eoi_blup_sub_chr-3_peak.marker-UNCHS009503_lod.drop-1.5_5.batches_mis.csv
    Untracked:  data/ici.vs.eoi_blup_sub_chr-3_peak.marker-UNCHS009532_lod.drop-1.5_5.batches_mis.csv
    Untracked:  data/ici.vs.eoi_blup_sub_chr-3_peak.marker-UNCHS009541_lod.drop-1.5_5.batches_mis.csv
    Untracked:  data/ici.vs.eoi_blup_sub_chr-3_peak.marker-UNCHS009560_lod.drop-1.5_5.batches_mis.csv
    Untracked:  data/ici.vs.eoi_blup_sub_chr-3_peak.marker-UNCHS009574_lod.drop-1.5_5.batches_mis.csv
    Untracked:  data/ici.vs.eoi_blup_sub_chr-3_peak.marker-UNCJPD001170_lod.drop-1.5_5.batches_mis.csv
    Untracked:  data/ici.vs.eoi_blup_sub_chr-3_peak.marker-UNCJPD001198_lod.drop-1.5_5.batches_mis.csv
    Untracked:  data/ici.vs.eoi_blup_sub_chr-3_peak.marker-UNCJPD001276_lod.drop-1.5_5.batches_mis.csv
    Untracked:  data/ici.vs.eoi_blup_sub_chr-3_peak.marker-UNCJPD001315_lod.drop-1.5_5.batches_mis.csv
    Untracked:  data/ici.vs.eoi_blup_sub_chr-3_peak.marker-UNCJPD001355_lod.drop-1.5_5.batches_mis.csv
    Untracked:  data/ici.vs.eoi_blup_sub_chr-3_peak.marker-UNCJPD001373_lod.drop-1.5_5.batches_mis.csv
    Untracked:  data/ici.vs.eoi_blup_sub_chr-3_peak.marker-UNCJPD001382_lod.drop-1.5_5.batches_mis.csv
    Untracked:  data/ici.vs.eoi_blup_sub_chr-4_peak.marker-UNC6851205_lod.drop-1.5_5.batches_mis.csv
    Untracked:  data/ici.vs.eoi_blup_sub_chr-4_peak.marker-UNC8250659_lod.drop-1.5_snpsqc_dis_no-x_updated_5.batches.csv
    Untracked:  data/ici.vs.eoi_blup_sub_chr-4_peak.marker-UNC8299912_lod.drop-1.5_5.batches_mis.csv
    Untracked:  data/ici.vs.eoi_blup_sub_chr-4_peak.marker-UNC8300499_lod.drop-1.5_5.batches.csv
    Untracked:  data/ici.vs.eoi_blup_sub_chr-4_peak.marker-UNC8381981_lod.drop-1.5_5.batches_mis.csv
    Untracked:  data/ici.vs.eoi_blup_sub_chr-4_peak.marker-UNC8436434_lod.drop-1.5_5.batches_mis.csv
    Untracked:  data/ici.vs.eoi_blup_sub_chr-4_peak.marker-UNC8439633_lod.drop-1.5_5.batches_mis.csv
    Untracked:  data/ici.vs.eoi_blup_sub_chr-4_peak.marker-UNC8530763_lod.drop-1.5_5.batches_mis.csv
    Untracked:  data/ici.vs.eoi_blup_sub_chr-4_peak.marker-UNCHS012948_lod.drop-1.5_snpsqc_dis_no-x_updated_5.batches_mis.csv
    Untracked:  data/ici.vs.eoi_blup_sub_chr-4_peak.marker-UNCHS012976_lod.drop-1.5_5.batches_mis.csv
    Untracked:  data/ici.vs.eoi_blup_sub_chr-4_peak.marker-UNCHS013118_lod.drop-1.5_5.batches.csv
    Untracked:  data/ici.vs.eoi_blup_sub_chr-4_peak.marker-UNCHS013118_lod.drop-1.5_snpsqc_5.batches.csv
    Untracked:  data/ici.vs.eoi_blup_sub_chr-4_peak.marker-UNCHS013118_lod.drop-1.5_snpsqc_5.batches_mis.csv
    Untracked:  data/ici.vs.eoi_blup_sub_chr-4_peak.marker-UNCHS013130_lod.drop-1.5_5.batches_mis.csv
    Untracked:  data/ici.vs.eoi_blup_sub_chr-4_peak.marker-UNCHS013139_lod.drop-1.5_5.batches_mis.csv
    Untracked:  data/ici.vs.eoi_blup_sub_chr-4_peak.marker-UNCHS013304_lod.drop-1.5_5.batches.csv
    Untracked:  data/ici.vs.eoi_blup_sub_chr-5_peak.marker-JAX00590770_lod.drop-1.5_5.batches_mis.csv
    Untracked:  data/ici.vs.eoi_blup_sub_chr-5_peak.marker-UNC10044126_lod.drop-1.5_5.batches_mis.csv
    Untracked:  data/ici.vs.eoi_blup_sub_chr-7_peak.marker-ICR1830_lod.drop-1.5_5.batches_mis.csv
    Untracked:  data/ici.vs.eoi_blup_sub_chr-7_peak.marker-UNC13170794_lod.drop-1.5_5.batches_mis.csv
    Untracked:  data/ici.vs.eoi_blup_sub_chr-7_peak.marker-UNCHS019480_lod.drop-1.5_5.batches.csv
    Untracked:  data/ici.vs.eoi_blup_sub_chr-7_peak.marker-UNCHS020066_lod.drop-1.5_snpsqc_5.batches_mis.csv
    Untracked:  data/ici.vs.eoi_blup_sub_chr-7_peak.marker-UNCHS020066_lod.drop-1.5_snpsqc_dis_no-x_updated_5.batches_mis.csv
    Untracked:  data/ici.vs.eoi_blup_sub_chr-7_peak.marker-cr31snv87_lod.drop-1.5_5.batches_mis.csv
    Untracked:  data/ici.vs.eoi_blup_sub_chr-8_peak.marker-JAX00667121_lod.drop-1.5_5.batches_mis.csv
    Untracked:  data/ici.vs.eoi_blup_sub_chr-8_peak.marker-UNC15430711_lod.drop-1.5_5.batches_mis.csv
    Untracked:  data/ici.vs.eoi_blup_sub_chr-8_peak.marker-UNC15524531_lod.drop-1.5_5.batches_mis.csv
    Untracked:  data/ici.vs.eoi_blup_sub_chr-8_peak.marker-UNC15524531_lod.drop-1.5_snpsqc_5.batches_mis.csv
    Untracked:  data/ici.vs.eoi_blup_sub_chr-8_peak.marker-UNC15524531_lod.drop-1.5_snpsqc_dis_no-x_updated_5.batches_mis.csv
    Untracked:  data/ici.vs.eoi_blup_sub_chr-9_peak.marker-UNC17203329_lod.drop-1.5_5.batches_mis.csv
    Untracked:  data/ici.vs.eoi_blup_sub_chr-9_peak.marker-UNC17203597_lod.drop-1.5_snpsqc_5.batches_mis.csv
    Untracked:  data/ici.vs.eoi_blup_sub_chr-9_peak.marker-UNCHS024593_lod.drop-1.5_5.batches_mis.csv
    Untracked:  data/ici.vs.eoi_blup_sub_chr-X_peak.marker-UNC30627130_lod.drop-1.5_5.batches_mis.csv
    Untracked:  data/ici.vs.eoi_blup_sub_chr-X_peak.marker-UNCHS048313_lod.drop-1.5_5.batches_mis.csv
    Untracked:  data/ici.vs.eoi_blup_sub_chr-X_peak.marker-UNCHS048314_lod.drop-1.5_snpsqc_5.batches_mis.csv
    Untracked:  data/ici.vs.eoi_blup_sub_chr-X_peak.marker-UNCHS048314_lod.drop-1.5_snpsqc_dis_no-x_updated_5.batches_mis.csv
    Untracked:  data/ici.vs.eoi_fitqtl_peaks_additive_5.batches.txt
    Untracked:  data/ici.vs.eoi_fitqtl_peaks_additive_5.batches_mis.txt
    Untracked:  data/ici.vs.eoi_fitqtl_peaks_additive_snpsqc_5.batches.txt
    Untracked:  data/ici.vs.eoi_fitqtl_peaks_additive_snpsqc_dis_no-x_updated_5.batches.txt
    Untracked:  data/ici.vs.eoi_fitqtl_peaks_additive_snpsqc_dis_no-x_updated_5.batches_mis.txt
    Untracked:  data/ici.vs.eoi_fitqtl_peaks_interacting_5.batches.txt
    Untracked:  data/ici.vs.eoi_fitqtl_peaks_interacting_5.batches_mis.txt
    Untracked:  data/ici.vs.eoi_fitqtl_peaks_interacting_snpsqc_5.batches.txt
    Untracked:  data/ici.vs.eoi_fitqtl_peaks_interacting_snpsqc_dis_no-x_updated_5.batches.txt
    Untracked:  data/ici.vs.eoi_fitqtl_peaks_interacting_snpsqc_dis_no-x_updated_5.batches_mis.txt
    Untracked:  data/ici.vs.eoi_fitqtl_peaks_sex_additive_5.batches.txt
    Untracked:  data/ici.vs.eoi_fitqtl_peaks_sex_additive_5.batches_mis.txt
    Untracked:  data/ici.vs.eoi_fitqtl_peaks_sex_additive_snpsqc_5.batches.txt
    Untracked:  data/ici.vs.eoi_fitqtl_peaks_sex_additive_snpsqc_dis_no-x_updated_5.batches.txt
    Untracked:  data/ici.vs.eoi_fitqtl_peaks_sex_additive_snpsqc_dis_no-x_updated_5.batches_mis.txt
    Untracked:  data/ici.vs.eoi_fitqtl_peaks_sex_interacting_5.batches.txt
    Untracked:  data/ici.vs.eoi_fitqtl_peaks_sex_interacting_5.batches_mis.txt
    Untracked:  data/ici.vs.eoi_fitqtl_peaks_sex_interacting_snpsqc_5.batches.txt
    Untracked:  data/ici.vs.eoi_fitqtl_peaks_sex_interacting_snpsqc_dis_no-x_updated_5.batches.txt
    Untracked:  data/ici.vs.eoi_fitqtl_peaks_sex_interacting_snpsqc_dis_no-x_updated_5.batches_mis.mis.txt
    Untracked:  data/ici.vs.eoi_genes_chr-10_peak.marker-ICR4295_lod.drop-1.5_5.batches_mis.csv
    Untracked:  data/ici.vs.eoi_genes_chr-10_peak.marker-JAX00292499_lod.drop-1.5_5.batches.csv
    Untracked:  data/ici.vs.eoi_genes_chr-10_peak.marker-JAX00292927_lod.drop-1.5_5.batches.csv
    Untracked:  data/ici.vs.eoi_genes_chr-10_peak.marker-JAX00292927_lod.drop-1.5_snpsqc_5.batches.csv
    Untracked:  data/ici.vs.eoi_genes_chr-10_peak.marker-JAX00292927_lod.drop-1.5_snpsqc_5.batches_mis.csv
    Untracked:  data/ici.vs.eoi_genes_chr-10_peak.marker-UNC18154330_lod.drop-1.5_5.batches.csv
    Untracked:  data/ici.vs.eoi_genes_chr-10_peak.marker-UNC18195765_lod.drop-1.5_5.batches.csv
    Untracked:  data/ici.vs.eoi_genes_chr-10_peak.marker-UNC18216614_lod.drop-1.5_5.batches.csv
    Untracked:  data/ici.vs.eoi_genes_chr-10_peak.marker-UNC18240977_lod.drop-1.5_snpsqc_dis_no-x_updated_5.batches.csv
    Untracked:  data/ici.vs.eoi_genes_chr-10_peak.marker-UNC18311938_lod.drop-1.5_5.batches.csv
    Untracked:  data/ici.vs.eoi_genes_chr-10_peak.marker-UNC18856953_lod.drop-1.5_5.batches_mis.csv
    Untracked:  data/ici.vs.eoi_genes_chr-10_peak.marker-UNCHS028236_lod.drop-1.5_snpsqc_5.batches_mis.csv
    Untracked:  data/ici.vs.eoi_genes_chr-10_peak.marker-UNCHS028536_lod.drop-1.5_snpsqc_dis_no-x_updated_5.batches_mis.csv
    Untracked:  data/ici.vs.eoi_genes_chr-10_peak.marker-UNCJPD004316_lod.drop-1.5_5.batches_mis.csv
    Untracked:  data/ici.vs.eoi_genes_chr-10_peak.marker-UNCJPD009532_lod.drop-1.5_5.batches_mis.csv
    Untracked:  data/ici.vs.eoi_genes_chr-11_peak.marker-UNC19155926_lod.drop-1.5_5.batches_mis.csv
    Untracked:  data/ici.vs.eoi_genes_chr-11_peak.marker-UNC19481366_lod.drop-1.5_snpsqc_5.batches_mis.csv
    Untracked:  data/ici.vs.eoi_genes_chr-11_peak.marker-UNC19970181_lod.drop-1.5_5.batches_mis.csv
    Untracked:  data/ici.vs.eoi_genes_chr-11_peak.marker-UNC19970181_lod.drop-1.5_snpsqc_5.batches_mis.csv
    Untracked:  data/ici.vs.eoi_genes_chr-11_peak.marker-UNCJPD004792_lod.drop-1.5_5.batches_mis.csv
    Untracked:  data/ici.vs.eoi_genes_chr-12_peak.marker-ICR4497_lod.drop-1.5_5.batches_mis.csv
    Untracked:  data/ici.vs.eoi_genes_chr-12_peak.marker-ICR499_lod.drop-1.5_snpsqc_5.batches_mis.csv
    Untracked:  data/ici.vs.eoi_genes_chr-13_peak.marker-UNCHS036773_lod.drop-1.5_snpsqc_5.batches_mis.csv
    Untracked:  data/ici.vs.eoi_genes_chr-13_peak.marker-UNCHS036964_lod.drop-1.5_5.batches_mis.csv
    Untracked:  data/ici.vs.eoi_genes_chr-14_peak.marker-UNC24056202_lod.drop-1.5_5.batches_mis.csv
    Untracked:  data/ici.vs.eoi_genes_chr-14_peak.marker-UNC24056202_lod.drop-1.5_snpsqc_5.batches_mis.csv
    Untracked:  data/ici.vs.eoi_genes_chr-15_peak.marker-UNC26070435_lod.drop-1.5_snpsqc_dis_no-x_updated_5.batches_mis.csv
    Untracked:  data/ici.vs.eoi_genes_chr-17_peak.marker-UNCHS044241_lod.drop-1.5_5.batches_mis.csv
    Untracked:  data/ici.vs.eoi_genes_chr-17_peak.marker-UNCHS044241_lod.drop-1.5_snpsqc_5.batches_mis.csv
    Untracked:  data/ici.vs.eoi_genes_chr-17_peak.marker-UNCrs47191360_lod.drop-1.5_5.batches_mis.csv
    Untracked:  data/ici.vs.eoi_genes_chr-17_peak.marker-UNCrs47191360_lod.drop-1.5_snpsqc_dis_no-x_updated_5.batches_mis.csv
    Untracked:  data/ici.vs.eoi_genes_chr-18_peak.marker-UNCHS045343_lod.drop-1.5_snpsqc_5.batches_mis.csv
    Untracked:  data/ici.vs.eoi_genes_chr-18_peak.marker-UNCHS045344_lod.drop-1.5_5.batches_mis.csv
    Untracked:  data/ici.vs.eoi_genes_chr-19_peak.marker-UNCJPD007087_lod.drop-1.5_5.batches_mis.csv
    Untracked:  data/ici.vs.eoi_genes_chr-1_peak.marker-ICR010_lod.drop-1.5_5.batches.csv
    Untracked:  data/ici.vs.eoi_genes_chr-1_peak.marker-ICR010_lod.drop-1.5_5.batches_mis.csv
    Untracked:  data/ici.vs.eoi_genes_chr-2_peak.marker-UNC4609527_lod.drop-1.5_snpsqc_dis_no-x_updated_5.batches_mis.csv
    Untracked:  data/ici.vs.eoi_genes_chr-2_peak.marker-UNC4609660_lod.drop-1.5_5.batches_mis.csv
    Untracked:  data/ici.vs.eoi_genes_chr-3_peak.marker-B6_rs31740628_lod.drop-1.5_5.batches_mis.csv
    Untracked:  data/ici.vs.eoi_genes_chr-3_peak.marker-ICR1269_lod.drop-1.5_5.batches_mis.csv
    Untracked:  data/ici.vs.eoi_genes_chr-3_peak.marker-ICR1338_lod.drop-1.5_5.batches.csv
    Untracked:  data/ici.vs.eoi_genes_chr-3_peak.marker-ICR5325_lod.drop-1.5_5.batches_mis.csv
    Untracked:  data/ici.vs.eoi_genes_chr-3_peak.marker-ICR5347_lod.drop-1.5_5.batches.csv
    Untracked:  data/ici.vs.eoi_genes_chr-3_peak.marker-ICR5348_lod.drop-1.5_5.batches_mis.csv
    Untracked:  data/ici.vs.eoi_genes_chr-3_peak.marker-JAX00104971_lod.drop-1.5_5.batches_mis.csv
    Untracked:  data/ici.vs.eoi_genes_chr-3_peak.marker-JAX00105059_lod.drop-1.5_5.batches.csv
    Untracked:  data/ici.vs.eoi_genes_chr-3_peak.marker-JAX00105203_lod.drop-1.5_5.batches_mis.csv
    Untracked:  data/ici.vs.eoi_genes_chr-3_peak.marker-JAX00106957_lod.drop-1.5_5.batches.csv
    Untracked:  data/ici.vs.eoi_genes_chr-3_peak.marker-JAX00106957_lod.drop-1.5_5.batches_mis.csv
    Untracked:  data/ici.vs.eoi_genes_chr-3_peak.marker-JAX00107930_lod.drop-1.5_5.batches_mis.csv
    Untracked:  data/ici.vs.eoi_genes_chr-3_peak.marker-JAX00107980_lod.drop-1.5_5.batches_mis.csv
    Untracked:  data/ici.vs.eoi_genes_chr-3_peak.marker-JAX00108068_lod.drop-1.5_5.batches_mis.csv
    Untracked:  data/ici.vs.eoi_genes_chr-3_peak.marker-JAX00108114_lod.drop-1.5_5.batches_mis.csv
    Untracked:  data/ici.vs.eoi_genes_chr-3_peak.marker-JAX00109750_lod.drop-1.5_5.batches_mis.csv
    Untracked:  data/ici.vs.eoi_genes_chr-3_peak.marker-JAX00189245_lod.drop-1.5_5.batches_mis.csv
    Untracked:  data/ici.vs.eoi_genes_chr-3_peak.marker-JAX00514662_lod.drop-1.5_5.batches.csv
    Untracked:  data/ici.vs.eoi_genes_chr-3_peak.marker-JAX00517218_lod.drop-1.5_5.batches_mis.csv
    Untracked:  data/ici.vs.eoi_genes_chr-3_peak.marker-JAX00517674_lod.drop-1.5_5.batches_mis.csv
    Untracked:  data/ici.vs.eoi_genes_chr-3_peak.marker-JAX00517906_lod.drop-1.5_5.batches_mis.csv
    Untracked:  data/ici.vs.eoi_genes_chr-3_peak.marker-JAX00518418_lod.drop-1.5_5.batches_mis.csv
    Untracked:  data/ici.vs.eoi_genes_chr-3_peak.marker-JAX00518684_lod.drop-1.5_5.batches_mis.csv
    Untracked:  data/ici.vs.eoi_genes_chr-3_peak.marker-JAX00518911r_lod.drop-1.5_5.batches_mis.csv
    Untracked:  data/ici.vs.eoi_genes_chr-3_peak.marker-JAX00520122_lod.drop-1.5_5.batches_mis.csv
    Untracked:  data/ici.vs.eoi_genes_chr-3_peak.marker-JAX00524717_lod.drop-1.5_5.batches_mis.csv
    Untracked:  data/ici.vs.eoi_genes_chr-3_peak.marker-JAX00524828_lod.drop-1.5_5.batches_mis.csv
    Untracked:  data/ici.vs.eoi_genes_chr-3_peak.marker-JAX00525380_lod.drop-1.5_5.batches_mis.csv
    Untracked:  data/ici.vs.eoi_genes_chr-3_peak.marker-JAX00525709_lod.drop-1.5_5.batches_mis.csv
    Untracked:  data/ici.vs.eoi_genes_chr-3_peak.marker-JAX00527291_lod.drop-1.5_5.batches_mis.csv
    Untracked:  data/ici.vs.eoi_genes_chr-3_peak.marker-JAX00527587_lod.drop-1.5_5.batches_mis.csv
    Untracked:  data/ici.vs.eoi_genes_chr-3_peak.marker-JAX00529707_lod.drop-1.5_5.batches_mis.csv
    Untracked:  data/ici.vs.eoi_genes_chr-3_peak.marker-JAX00529707r_lod.drop-1.5_5.batches.csv
    Untracked:  data/ici.vs.eoi_genes_chr-3_peak.marker-JAX00531601_lod.drop-1.5_5.batches_mis.csv
    Untracked:  data/ici.vs.eoi_genes_chr-3_peak.marker-JAX00532122_lod.drop-1.5_5.batches_mis.csv
    Untracked:  data/ici.vs.eoi_genes_chr-3_peak.marker-UNC4673471_lod.drop-1.5_5.batches_mis.csv
    Untracked:  data/ici.vs.eoi_genes_chr-3_peak.marker-UNC4681411_lod.drop-1.5_5.batches_mis.csv
    Untracked:  data/ici.vs.eoi_genes_chr-3_peak.marker-UNC4793677_lod.drop-1.5_5.batches_mis.csv
    Untracked:  data/ici.vs.eoi_genes_chr-3_peak.marker-UNC4799475_lod.drop-1.5_5.batches_mis.csv
    Untracked:  data/ici.vs.eoi_genes_chr-3_peak.marker-UNC4809586_lod.drop-1.5_5.batches_mis.csv
    Untracked:  data/ici.vs.eoi_genes_chr-3_peak.marker-UNC4813186_lod.drop-1.5_5.batches.csv
    Untracked:  data/ici.vs.eoi_genes_chr-3_peak.marker-UNC4813186_lod.drop-1.5_5.batches_mis.csv
    Untracked:  data/ici.vs.eoi_genes_chr-3_peak.marker-UNC4820379_lod.drop-1.5_5.batches_mis.csv
    Untracked:  data/ici.vs.eoi_genes_chr-3_peak.marker-UNC4875653_lod.drop-1.5_5.batches_mis.csv
    Untracked:  data/ici.vs.eoi_genes_chr-3_peak.marker-UNC4909830_lod.drop-1.5_5.batches_mis.csv
    Untracked:  data/ici.vs.eoi_genes_chr-3_peak.marker-UNC4912288_lod.drop-1.5_5.batches_mis.csv
    Untracked:  data/ici.vs.eoi_genes_chr-3_peak.marker-UNC4914130_lod.drop-1.5_5.batches_mis.csv
    Untracked:  data/ici.vs.eoi_genes_chr-3_peak.marker-UNC4923398_lod.drop-1.5_5.batches_mis.csv
    Untracked:  data/ici.vs.eoi_genes_chr-3_peak.marker-UNC4924377_lod.drop-1.5_5.batches.csv
    Untracked:  data/ici.vs.eoi_genes_chr-3_peak.marker-UNC4945186_lod.drop-1.5_5.batches_mis.csv
    Untracked:  data/ici.vs.eoi_genes_chr-3_peak.marker-UNC4947492_lod.drop-1.5_5.batches_mis.csv
    Untracked:  data/ici.vs.eoi_genes_chr-3_peak.marker-UNC4957168_lod.drop-1.5_5.batches.csv
    Untracked:  data/ici.vs.eoi_genes_chr-3_peak.marker-UNC4957859_lod.drop-1.5_5.batches_mis.csv
    Untracked:  data/ici.vs.eoi_genes_chr-3_peak.marker-UNC4964062_lod.drop-1.5_5.batches_mis.csv
    Untracked:  data/ici.vs.eoi_genes_chr-3_peak.marker-UNC4979914_lod.drop-1.5_5.batches_mis.csv
    Untracked:  data/ici.vs.eoi_genes_chr-3_peak.marker-UNC4985064_lod.drop-1.5_5.batches_mis.csv
    Untracked:  data/ici.vs.eoi_genes_chr-3_peak.marker-UNC4990111_lod.drop-1.5_5.batches_mis.csv
    Untracked:  data/ici.vs.eoi_genes_chr-3_peak.marker-UNC4991565_lod.drop-1.5_5.batches_mis.csv
    Untracked:  data/ici.vs.eoi_genes_chr-3_peak.marker-UNC5001723_lod.drop-1.5_5.batches_mis.csv
    Untracked:  data/ici.vs.eoi_genes_chr-3_peak.marker-UNC5008656_lod.drop-1.5_5.batches.csv
    Untracked:  data/ici.vs.eoi_genes_chr-3_peak.marker-UNC5008656_lod.drop-1.5_5.batches_mis.csv
    Untracked:  data/ici.vs.eoi_genes_chr-3_peak.marker-UNC5042361_lod.drop-1.5_5.batches_mis.csv
    Untracked:  data/ici.vs.eoi_genes_chr-3_peak.marker-UNC5055259_lod.drop-1.5_5.batches_mis.csv
    Untracked:  data/ici.vs.eoi_genes_chr-3_peak.marker-UNC5076935_lod.drop-1.5_5.batches_mis.csv
    Untracked:  data/ici.vs.eoi_genes_chr-3_peak.marker-UNC5082757_lod.drop-1.5_5.batches_mis.csv
    Untracked:  data/ici.vs.eoi_genes_chr-3_peak.marker-UNC5083168_lod.drop-1.5_5.batches.csv
    Untracked:  data/ici.vs.eoi_genes_chr-3_peak.marker-UNC5173293_lod.drop-1.5_5.batches_mis.csv
    Untracked:  data/ici.vs.eoi_genes_chr-3_peak.marker-UNC5182782_lod.drop-1.5_5.batches_mis.csv
    Untracked:  data/ici.vs.eoi_genes_chr-3_peak.marker-UNC5192582_lod.drop-1.5_5.batches.csv
    Untracked:  data/ici.vs.eoi_genes_chr-3_peak.marker-UNC5193970_lod.drop-1.5_5.batches_mis.csv
    Untracked:  data/ici.vs.eoi_genes_chr-3_peak.marker-UNC5213888_lod.drop-1.5_snpsqc_5.batches_mis.csv
    Untracked:  data/ici.vs.eoi_genes_chr-3_peak.marker-UNC5240484_lod.drop-1.5_5.batches_mis.csv
    Untracked:  data/ici.vs.eoi_genes_chr-3_peak.marker-UNC5256922_lod.drop-1.5_5.batches_mis.csv
    Untracked:  data/ici.vs.eoi_genes_chr-3_peak.marker-UNC5266664_lod.drop-1.5_5.batches_mis.csv
    Untracked:  data/ici.vs.eoi_genes_chr-3_peak.marker-UNC5266711_lod.drop-1.5_5.batches.csv
    Untracked:  data/ici.vs.eoi_genes_chr-3_peak.marker-UNC5267813_lod.drop-1.5_5.batches_mis.csv
    Untracked:  data/ici.vs.eoi_genes_chr-3_peak.marker-UNC5274554_lod.drop-1.5_5.batches_mis.csv
    Untracked:  data/ici.vs.eoi_genes_chr-3_peak.marker-UNC5277061_lod.drop-1.5_5.batches_mis.csv
    Untracked:  data/ici.vs.eoi_genes_chr-3_peak.marker-UNC5282259_lod.drop-1.5_5.batches_mis.csv
    Untracked:  data/ici.vs.eoi_genes_chr-3_peak.marker-UNC5287977_lod.drop-1.5_5.batches_mis.csv
    Untracked:  data/ici.vs.eoi_genes_chr-3_peak.marker-UNC5301460_lod.drop-1.5_5.batches_mis.csv
    Untracked:  data/ici.vs.eoi_genes_chr-3_peak.marker-UNC5321559_lod.drop-1.5_5.batches_mis.csv
    Untracked:  data/ici.vs.eoi_genes_chr-3_peak.marker-UNC5322455_lod.drop-1.5_5.batches_mis.csv
    Untracked:  data/ici.vs.eoi_genes_chr-3_peak.marker-UNC5328596_lod.drop-1.5_5.batches_mis.csv
    Untracked:  data/ici.vs.eoi_genes_chr-3_peak.marker-UNC5350384_lod.drop-1.5_5.batches_mis.csv
    Untracked:  data/ici.vs.eoi_genes_chr-3_peak.marker-UNC5354724_lod.drop-1.5_5.batches_mis.csv
    Untracked:  data/ici.vs.eoi_genes_chr-3_peak.marker-UNC5375208_lod.drop-1.5_5.batches_mis.csv
    Untracked:  data/ici.vs.eoi_genes_chr-3_peak.marker-UNC5384072_lod.drop-1.5_5.batches_mis.csv
    Untracked:  data/ici.vs.eoi_genes_chr-3_peak.marker-UNC5400314_lod.drop-1.5_5.batches_mis.csv
    Untracked:  data/ici.vs.eoi_genes_chr-3_peak.marker-UNC5433945_lod.drop-1.5_5.batches_mis.csv
    Untracked:  data/ici.vs.eoi_genes_chr-3_peak.marker-UNC5435874_lod.drop-1.5_5.batches_mis.csv
    Untracked:  data/ici.vs.eoi_genes_chr-3_peak.marker-UNC5454839_lod.drop-1.5_5.batches.csv
    Untracked:  data/ici.vs.eoi_genes_chr-3_peak.marker-UNC5454839_lod.drop-1.5_5.batches_mis.csv
    Untracked:  data/ici.vs.eoi_genes_chr-3_peak.marker-UNC5466669_lod.drop-1.5_5.batches_mis.csv
    Untracked:  data/ici.vs.eoi_genes_chr-3_peak.marker-UNC5517972_lod.drop-1.5_5.batches_mis.csv
    Untracked:  data/ici.vs.eoi_genes_chr-3_peak.marker-UNC5522257_lod.drop-1.5_5.batches_mis.csv
    Untracked:  data/ici.vs.eoi_genes_chr-3_peak.marker-UNC5523265_lod.drop-1.5_5.batches_mis.csv
    Untracked:  data/ici.vs.eoi_genes_chr-3_peak.marker-UNC5532424_lod.drop-1.5_5.batches_mis.csv
    Untracked:  data/ici.vs.eoi_genes_chr-3_peak.marker-UNC5553882_lod.drop-1.5_5.batches_mis.csv
    Untracked:  data/ici.vs.eoi_genes_chr-3_peak.marker-UNC5572051_lod.drop-1.5_5.batches_mis.csv
    Untracked:  data/ici.vs.eoi_genes_chr-3_peak.marker-UNC5582753_lod.drop-1.5_5.batches.csv
    Untracked:  data/ici.vs.eoi_genes_chr-3_peak.marker-UNC5589260_lod.drop-1.5_5.batches_mis.csv
    Untracked:  data/ici.vs.eoi_genes_chr-3_peak.marker-UNC5595555_lod.drop-1.5_5.batches_mis.csv
    Untracked:  data/ici.vs.eoi_genes_chr-3_peak.marker-UNC5623239_lod.drop-1.5_5.batches_mis.csv
    Untracked:  data/ici.vs.eoi_genes_chr-3_peak.marker-UNC5639830_lod.drop-1.5_5.batches_mis.csv
    Untracked:  data/ici.vs.eoi_genes_chr-3_peak.marker-UNC5645844_lod.drop-1.5_5.batches.csv
    Untracked:  data/ici.vs.eoi_genes_chr-3_peak.marker-UNC5645844_lod.drop-1.5_5.batches_mis.csv
    Untracked:  data/ici.vs.eoi_genes_chr-3_peak.marker-UNC5680234_lod.drop-1.5_5.batches_mis.csv
    Untracked:  data/ici.vs.eoi_genes_chr-3_peak.marker-UNC5682314_lod.drop-1.5_5.batches_mis.csv
    Untracked:  data/ici.vs.eoi_genes_chr-3_peak.marker-UNC5725356_lod.drop-1.5_5.batches_mis.csv
    Untracked:  data/ici.vs.eoi_genes_chr-3_peak.marker-UNC5727351_lod.drop-1.5_5.batches_mis.csv
    Untracked:  data/ici.vs.eoi_genes_chr-3_peak.marker-UNC5734420_lod.drop-1.5_5.batches_mis.csv
    Untracked:  data/ici.vs.eoi_genes_chr-3_peak.marker-UNC5743257_lod.drop-1.5_5.batches_mis.csv
    Untracked:  data/ici.vs.eoi_genes_chr-3_peak.marker-UNC5751567_lod.drop-1.5_5.batches_mis.csv
    Untracked:  data/ici.vs.eoi_genes_chr-3_peak.marker-UNC5752623_lod.drop-1.5_5.batches_mis.csv
    Untracked:  data/ici.vs.eoi_genes_chr-3_peak.marker-UNC5776974_lod.drop-1.5_5.batches_mis.csv
    Untracked:  data/ici.vs.eoi_genes_chr-3_peak.marker-UNC5778977_lod.drop-1.5_5.batches_mis.csv
    Untracked:  data/ici.vs.eoi_genes_chr-3_peak.marker-UNC5788507_lod.drop-1.5_5.batches_mis.csv
    Untracked:  data/ici.vs.eoi_genes_chr-3_peak.marker-UNC5792468_lod.drop-1.5_5.batches_mis.csv
    Untracked:  data/ici.vs.eoi_genes_chr-3_peak.marker-UNC5794598_lod.drop-1.5_5.batches_mis.csv
    Untracked:  data/ici.vs.eoi_genes_chr-3_peak.marker-UNC5795561_lod.drop-1.5_5.batches_mis.csv
    Untracked:  data/ici.vs.eoi_genes_chr-3_peak.marker-UNC5802298_lod.drop-1.5_5.batches_mis.csv
    Untracked:  data/ici.vs.eoi_genes_chr-3_peak.marker-UNC5806628_lod.drop-1.5_5.batches_mis.csv
    Untracked:  data/ici.vs.eoi_genes_chr-3_peak.marker-UNC5808246_lod.drop-1.5_5.batches.csv
    Untracked:  data/ici.vs.eoi_genes_chr-3_peak.marker-UNC5817478_lod.drop-1.5_5.batches_mis.csv
    Untracked:  data/ici.vs.eoi_genes_chr-3_peak.marker-UNC5836999_lod.drop-1.5_5.batches_mis.csv
    Untracked:  data/ici.vs.eoi_genes_chr-3_peak.marker-UNC5841501_lod.drop-1.5_5.batches_mis.csv
    Untracked:  data/ici.vs.eoi_genes_chr-3_peak.marker-UNC5847927_lod.drop-1.5_5.batches_mis.csv
    Untracked:  data/ici.vs.eoi_genes_chr-3_peak.marker-UNC5966417_lod.drop-1.5_5.batches_mis.csv
    Untracked:  data/ici.vs.eoi_genes_chr-3_peak.marker-UNCHS008129_lod.drop-1.5_5.batches_mis.csv
    Untracked:  data/ici.vs.eoi_genes_chr-3_peak.marker-UNCHS008173_lod.drop-1.5_5.batches_mis.csv
    Untracked:  data/ici.vs.eoi_genes_chr-3_peak.marker-UNCHS008182_lod.drop-1.5_5.batches.csv
    Untracked:  data/ici.vs.eoi_genes_chr-3_peak.marker-UNCHS008193_lod.drop-1.5_5.batches_mis.csv
    Untracked:  data/ici.vs.eoi_genes_chr-3_peak.marker-UNCHS008209_lod.drop-1.5_5.batches_mis.csv
    Untracked:  data/ici.vs.eoi_genes_chr-3_peak.marker-UNCHS008215_lod.drop-1.5_5.batches_mis.csv
    Untracked:  data/ici.vs.eoi_genes_chr-3_peak.marker-UNCHS008219_lod.drop-1.5_5.batches_mis.csv
    Untracked:  data/ici.vs.eoi_genes_chr-3_peak.marker-UNCHS008231_lod.drop-1.5_5.batches_mis.csv
    Untracked:  data/ici.vs.eoi_genes_chr-3_peak.marker-UNCHS008235_lod.drop-1.5_5.batches_mis.csv
    Untracked:  data/ici.vs.eoi_genes_chr-3_peak.marker-UNCHS008242_lod.drop-1.5_5.batches.csv
    Untracked:  data/ici.vs.eoi_genes_chr-3_peak.marker-UNCHS008253_lod.drop-1.5_5.batches_mis.csv
    Untracked:  data/ici.vs.eoi_genes_chr-3_peak.marker-UNCHS008256_lod.drop-1.5_5.batches_mis.csv
    Untracked:  data/ici.vs.eoi_genes_chr-3_peak.marker-UNCHS008259_lod.drop-1.5_5.batches_mis.csv
    Untracked:  data/ici.vs.eoi_genes_chr-3_peak.marker-UNCHS008271_lod.drop-1.5_5.batches_mis.csv
    Untracked:  data/ici.vs.eoi_genes_chr-3_peak.marker-UNCHS008286_lod.drop-1.5_5.batches.csv
    Untracked:  data/ici.vs.eoi_genes_chr-3_peak.marker-UNCHS008286_lod.drop-1.5_5.batches_mis.csv
    Untracked:  data/ici.vs.eoi_genes_chr-3_peak.marker-UNCHS008311_lod.drop-1.5_5.batches_mis.csv
    Untracked:  data/ici.vs.eoi_genes_chr-3_peak.marker-UNCHS008317_lod.drop-1.5_5.batches_mis.csv
    Untracked:  data/ici.vs.eoi_genes_chr-3_peak.marker-UNCHS008372_lod.drop-1.5_5.batches_mis.csv
    Untracked:  data/ici.vs.eoi_genes_chr-3_peak.marker-UNCHS008379_lod.drop-1.5_5.batches_mis.csv
    Untracked:  data/ici.vs.eoi_genes_chr-3_peak.marker-UNCHS008410_lod.drop-1.5_5.batches_mis.csv
    Untracked:  data/ici.vs.eoi_genes_chr-3_peak.marker-UNCHS008415_lod.drop-1.5_5.batches_mis.csv
    Untracked:  data/ici.vs.eoi_genes_chr-3_peak.marker-UNCHS008431_lod.drop-1.5_5.batches_mis.csv
    Untracked:  data/ici.vs.eoi_genes_chr-3_peak.marker-UNCHS008487_lod.drop-1.5_snpsqc_5.batches.csv
    Untracked:  data/ici.vs.eoi_genes_chr-3_peak.marker-UNCHS008487_lod.drop-1.5_snpsqc_dis_no-x_updated_5.batches.csv
    Untracked:  data/ici.vs.eoi_genes_chr-3_peak.marker-UNCHS008487_lod.drop-1.5_snpsqc_dis_no-x_updated_5.batches_mis.csv
    Untracked:  data/ici.vs.eoi_genes_chr-3_peak.marker-UNCHS008511_lod.drop-1.5_snpsqc_5.batches_mis.csv
    Untracked:  data/ici.vs.eoi_genes_chr-3_peak.marker-UNCHS008609_lod.drop-1.5_5.batches_mis.csv
    Untracked:  data/ici.vs.eoi_genes_chr-3_peak.marker-UNCHS008614_lod.drop-1.5_5.batches.csv
    Untracked:  data/ici.vs.eoi_genes_chr-3_peak.marker-UNCHS008649_lod.drop-1.5_5.batches_mis.csv
    Untracked:  data/ici.vs.eoi_genes_chr-3_peak.marker-UNCHS008651_lod.drop-1.5_5.batches.csv
    Untracked:  data/ici.vs.eoi_genes_chr-3_peak.marker-UNCHS008656_lod.drop-1.5_5.batches_mis.csv
    Untracked:  data/ici.vs.eoi_genes_chr-3_peak.marker-UNCHS008723_lod.drop-1.5_5.batches_mis.csv
    Untracked:  data/ici.vs.eoi_genes_chr-3_peak.marker-UNCHS008802_lod.drop-1.5_5.batches_mis.csv
    Untracked:  data/ici.vs.eoi_genes_chr-3_peak.marker-UNCHS008803_lod.drop-1.5_5.batches_mis.csv
    Untracked:  data/ici.vs.eoi_genes_chr-3_peak.marker-UNCHS008808_lod.drop-1.5_5.batches_mis.csv
    Untracked:  data/ici.vs.eoi_genes_chr-3_peak.marker-UNCHS008815_lod.drop-1.5_5.batches.csv
    Untracked:  data/ici.vs.eoi_genes_chr-3_peak.marker-UNCHS008815_lod.drop-1.5_5.batches_mis.csv
    Untracked:  data/ici.vs.eoi_genes_chr-3_peak.marker-UNCHS008834_lod.drop-1.5_5.batches_mis.csv
    Untracked:  data/ici.vs.eoi_genes_chr-3_peak.marker-UNCHS008838_lod.drop-1.5_5.batches_mis.csv
    Untracked:  data/ici.vs.eoi_genes_chr-3_peak.marker-UNCHS008914_lod.drop-1.5_5.batches_mis.csv
    Untracked:  data/ici.vs.eoi_genes_chr-3_peak.marker-UNCHS008928_lod.drop-1.5_5.batches_mis.csv
    Untracked:  data/ici.vs.eoi_genes_chr-3_peak.marker-UNCHS008948_lod.drop-1.5_5.batches_mis.csv
    Untracked:  data/ici.vs.eoi_genes_chr-3_peak.marker-UNCHS008959_lod.drop-1.5_5.batches_mis.csv
    Untracked:  data/ici.vs.eoi_genes_chr-3_peak.marker-UNCHS008973_lod.drop-1.5_5.batches_mis.csv
    Untracked:  data/ici.vs.eoi_genes_chr-3_peak.marker-UNCHS008987_lod.drop-1.5_5.batches_mis.csv
    Untracked:  data/ici.vs.eoi_genes_chr-3_peak.marker-UNCHS009001_lod.drop-1.5_5.batches_mis.csv
    Untracked:  data/ici.vs.eoi_genes_chr-3_peak.marker-UNCHS009025_lod.drop-1.5_5.batches_mis.csv
    Untracked:  data/ici.vs.eoi_genes_chr-3_peak.marker-UNCHS009034_lod.drop-1.5_5.batches_mis.csv
    Untracked:  data/ici.vs.eoi_genes_chr-3_peak.marker-UNCHS009087_lod.drop-1.5_5.batches_mis.csv
    Untracked:  data/ici.vs.eoi_genes_chr-3_peak.marker-UNCHS009109_lod.drop-1.5_5.batches_mis.csv
    Untracked:  data/ici.vs.eoi_genes_chr-3_peak.marker-UNCHS009142_lod.drop-1.5_5.batches_mis.csv
    Untracked:  data/ici.vs.eoi_genes_chr-3_peak.marker-UNCHS009207_lod.drop-1.5_5.batches_mis.csv
    Untracked:  data/ici.vs.eoi_genes_chr-3_peak.marker-UNCHS009216_lod.drop-1.5_5.batches_mis.csv
    Untracked:  data/ici.vs.eoi_genes_chr-3_peak.marker-UNCHS009224_lod.drop-1.5_5.batches.csv
    Untracked:  data/ici.vs.eoi_genes_chr-3_peak.marker-UNCHS009306_lod.drop-1.5_5.batches_mis.csv
    Untracked:  data/ici.vs.eoi_genes_chr-3_peak.marker-UNCHS009316_lod.drop-1.5_5.batches_mis.csv
    Untracked:  data/ici.vs.eoi_genes_chr-3_peak.marker-UNCHS009364_lod.drop-1.5_5.batches_mis.csv
    Untracked:  data/ici.vs.eoi_genes_chr-3_peak.marker-UNCHS009377_lod.drop-1.5_5.batches_mis.csv
    Untracked:  data/ici.vs.eoi_genes_chr-3_peak.marker-UNCHS009382_lod.drop-1.5_5.batches_mis.csv
    Untracked:  data/ici.vs.eoi_genes_chr-3_peak.marker-UNCHS009383_lod.drop-1.5_5.batches.csv
    Untracked:  data/ici.vs.eoi_genes_chr-3_peak.marker-UNCHS009387_lod.drop-1.5_5.batches_mis.csv
    Untracked:  data/ici.vs.eoi_genes_chr-3_peak.marker-UNCHS009395_lod.drop-1.5_5.batches_mis.csv
    Untracked:  data/ici.vs.eoi_genes_chr-3_peak.marker-UNCHS009413_lod.drop-1.5_5.batches_mis.csv
    Untracked:  data/ici.vs.eoi_genes_chr-3_peak.marker-UNCHS009428_lod.drop-1.5_5.batches_mis.csv
    Untracked:  data/ici.vs.eoi_genes_chr-3_peak.marker-UNCHS009463_lod.drop-1.5_5.batches_mis.csv
    Untracked:  data/ici.vs.eoi_genes_chr-3_peak.marker-UNCHS009503_lod.drop-1.5_5.batches_mis.csv
    Untracked:  data/ici.vs.eoi_genes_chr-3_peak.marker-UNCHS009532_lod.drop-1.5_5.batches_mis.csv
    Untracked:  data/ici.vs.eoi_genes_chr-3_peak.marker-UNCHS009541_lod.drop-1.5_5.batches_mis.csv
    Untracked:  data/ici.vs.eoi_genes_chr-3_peak.marker-UNCHS009560_lod.drop-1.5_5.batches_mis.csv
    Untracked:  data/ici.vs.eoi_genes_chr-3_peak.marker-UNCHS009574_lod.drop-1.5_5.batches_mis.csv
    Untracked:  data/ici.vs.eoi_genes_chr-3_peak.marker-UNCJPD001170_lod.drop-1.5_5.batches_mis.csv
    Untracked:  data/ici.vs.eoi_genes_chr-3_peak.marker-UNCJPD001198_lod.drop-1.5_5.batches_mis.csv
    Untracked:  data/ici.vs.eoi_genes_chr-3_peak.marker-UNCJPD001276_lod.drop-1.5_5.batches_mis.csv
    Untracked:  data/ici.vs.eoi_genes_chr-3_peak.marker-UNCJPD001315_lod.drop-1.5_5.batches_mis.csv
    Untracked:  data/ici.vs.eoi_genes_chr-3_peak.marker-UNCJPD001355_lod.drop-1.5_5.batches_mis.csv
    Untracked:  data/ici.vs.eoi_genes_chr-3_peak.marker-UNCJPD001373_lod.drop-1.5_5.batches_mis.csv
    Untracked:  data/ici.vs.eoi_genes_chr-3_peak.marker-UNCJPD001382_lod.drop-1.5_5.batches_mis.csv
    Untracked:  data/ici.vs.eoi_genes_chr-4_peak.marker-UNC6851205_lod.drop-1.5_5.batches_mis.csv
    Untracked:  data/ici.vs.eoi_genes_chr-4_peak.marker-UNC8250659_lod.drop-1.5_snpsqc_dis_no-x_updated_5.batches.csv
    Untracked:  data/ici.vs.eoi_genes_chr-4_peak.marker-UNC8299912_lod.drop-1.5_5.batches_mis.csv
    Untracked:  data/ici.vs.eoi_genes_chr-4_peak.marker-UNC8300499_lod.drop-1.5_5.batches.csv
    Untracked:  data/ici.vs.eoi_genes_chr-4_peak.marker-UNC8381981_lod.drop-1.5_5.batches_mis.csv
    Untracked:  data/ici.vs.eoi_genes_chr-4_peak.marker-UNC8436434_lod.drop-1.5_5.batches_mis.csv
    Untracked:  data/ici.vs.eoi_genes_chr-4_peak.marker-UNC8439633_lod.drop-1.5_5.batches_mis.csv
    Untracked:  data/ici.vs.eoi_genes_chr-4_peak.marker-UNC8530763_lod.drop-1.5_5.batches_mis.csv
    Untracked:  data/ici.vs.eoi_genes_chr-4_peak.marker-UNCHS012948_lod.drop-1.5_snpsqc_dis_no-x_updated_5.batches_mis.csv
    Untracked:  data/ici.vs.eoi_genes_chr-4_peak.marker-UNCHS012976_lod.drop-1.5_5.batches_mis.csv
    Untracked:  data/ici.vs.eoi_genes_chr-4_peak.marker-UNCHS013118_lod.drop-1.5_5.batches.csv
    Untracked:  data/ici.vs.eoi_genes_chr-4_peak.marker-UNCHS013118_lod.drop-1.5_snpsqc_5.batches.csv
    Untracked:  data/ici.vs.eoi_genes_chr-4_peak.marker-UNCHS013118_lod.drop-1.5_snpsqc_5.batches_mis.csv
    Untracked:  data/ici.vs.eoi_genes_chr-4_peak.marker-UNCHS013130_lod.drop-1.5_5.batches_mis.csv
    Untracked:  data/ici.vs.eoi_genes_chr-4_peak.marker-UNCHS013139_lod.drop-1.5_5.batches_mis.csv
    Untracked:  data/ici.vs.eoi_genes_chr-4_peak.marker-UNCHS013304_lod.drop-1.5_5.batches.csv
    Untracked:  data/ici.vs.eoi_genes_chr-5_peak.marker-JAX00590770_lod.drop-1.5_5.batches_mis.csv
    Untracked:  data/ici.vs.eoi_genes_chr-5_peak.marker-UNC10044126_lod.drop-1.5_5.batches_mis.csv
    Untracked:  data/ici.vs.eoi_genes_chr-7_peak.marker-ICR1830_lod.drop-1.5_5.batches_mis.csv
    Untracked:  data/ici.vs.eoi_genes_chr-7_peak.marker-UNC13170794_lod.drop-1.5_5.batches_mis.csv
    Untracked:  data/ici.vs.eoi_genes_chr-7_peak.marker-UNCHS019480_lod.drop-1.5_5.batches.csv
    Untracked:  data/ici.vs.eoi_genes_chr-7_peak.marker-UNCHS020066_lod.drop-1.5_snpsqc_5.batches_mis.csv
    Untracked:  data/ici.vs.eoi_genes_chr-7_peak.marker-UNCHS020066_lod.drop-1.5_snpsqc_dis_no-x_updated_5.batches_mis.csv
    Untracked:  data/ici.vs.eoi_genes_chr-7_peak.marker-cr31snv87_lod.drop-1.5_5.batches_mis.csv
    Untracked:  data/ici.vs.eoi_genes_chr-8_peak.marker-JAX00667121_lod.drop-1.5_5.batches_mis.csv
    Untracked:  data/ici.vs.eoi_genes_chr-8_peak.marker-UNC15430711_lod.drop-1.5_5.batches_mis.csv
    Untracked:  data/ici.vs.eoi_genes_chr-8_peak.marker-UNC15524531_lod.drop-1.5_5.batches_mis.csv
    Untracked:  data/ici.vs.eoi_genes_chr-8_peak.marker-UNC15524531_lod.drop-1.5_snpsqc_5.batches_mis.csv
    Untracked:  data/ici.vs.eoi_genes_chr-8_peak.marker-UNC15524531_lod.drop-1.5_snpsqc_dis_no-x_updated_5.batches_mis.csv
    Untracked:  data/ici.vs.eoi_genes_chr-9_peak.marker-UNC17203329_lod.drop-1.5_5.batches_mis.csv
    Untracked:  data/ici.vs.eoi_genes_chr-9_peak.marker-UNC17203597_lod.drop-1.5_snpsqc_5.batches_mis.csv
    Untracked:  data/ici.vs.eoi_genes_chr-9_peak.marker-UNCHS024593_lod.drop-1.5_5.batches_mis.csv
    Untracked:  data/ici.vs.eoi_genes_chr-X_peak.marker-UNC30627130_lod.drop-1.5_5.batches_mis.csv
    Untracked:  data/ici.vs.eoi_genes_chr-X_peak.marker-UNCHS048313_lod.drop-1.5_5.batches_mis.csv
    Untracked:  data/ici.vs.eoi_genes_chr-X_peak.marker-UNCHS048314_lod.drop-1.5_snpsqc_5.batches_mis.csv
    Untracked:  data/ici.vs.eoi_genes_chr-X_peak.marker-UNCHS048314_lod.drop-1.5_snpsqc_dis_no-x_updated_5.batches_mis.csv
    Untracked:  data/ici.vs.eoi_gm_qtl_5.batches.csv
    Untracked:  data/ici.vs.eoi_gm_qtl_5.batches_mis.csv
    Untracked:  data/ici.vs.eoi_gm_qtl_snpsqc_5.batches.csv
    Untracked:  data/ici.vs.eoi_gm_qtl_snpsqc_5.batches_mis.csv
    Untracked:  data/ici.vs.eoi_gm_qtl_snpsqc_dis_no-x_updated_5.batches.csv
    Untracked:  data/ici.vs.eoi_gm_qtl_snpsqc_dis_no-x_updated_5.batches_mis.csv
    Untracked:  data/ici.vs.eoi_marker.freq_low.geno.freq.removed_geno.ratio.csv
    Untracked:  data/ici.vs.eoi_marker.freq_low.geno.freq.removed_geno.ratio_5.batches.csv
    Untracked:  data/ici.vs.eoi_marker.freq_low.geno.freq.removed_geno.ratio_5.batches_0.csv
    Untracked:  data/ici.vs.eoi_marker.freq_low.geno.freq.removed_geno.ratio_5.batches_52.csv
    Untracked:  data/ici.vs.eoi_marker.freq_low.geno.freq.removed_geno.ratio_5.batches_mis.csv
    Untracked:  data/ici.vs.eoi_marker.freq_low.geno.freq.removed_geno.ratio_5.batches_mis_0.csv
    Untracked:  data/ici.vs.eoi_marker.freq_low.geno.freq.removed_geno.ratio_5.batches_mis_52.csv
    Untracked:  data/ici.vs.eoi_marker.freq_low.geno.freq.removed_sample.outliers.removed_geno.ratio.csv
    Untracked:  data/ici.vs.eoi_marker.freq_low.geno.freq.removed_sample.outliers.removed_geno.ratio_5.batches.csv
    Untracked:  data/ici.vs.eoi_marker.freq_low.geno.freq.removed_sample.outliers.removed_geno.ratio_5.batches_0.csv
    Untracked:  data/ici.vs.eoi_marker.freq_low.geno.freq.removed_sample.outliers.removed_geno.ratio_5.batches_52.csv
    Untracked:  data/ici.vs.eoi_marker.freq_low.geno.freq.removed_sample.outliers.removed_geno.ratio_5.batches_mis.csv
    Untracked:  data/ici.vs.eoi_marker.freq_low.geno.freq.removed_sample.outliers.removed_geno.ratio_5.batches_mis_0.csv
    Untracked:  data/ici.vs.eoi_marker.freq_low.geno.freq.removed_sample.outliers.removed_geno.ratio_5.batches_mis_52.csv
    Untracked:  data/ici.vs.eoi_marker.freq_low.probs.freq.removed_geno.ratio.csv
    Untracked:  data/ici.vs.eoi_marker.freq_low.probs.freq.removed_geno.ratio_5.batches.csv
    Untracked:  data/ici.vs.eoi_marker.freq_low.probs.freq.removed_geno.ratio_5.batches_0.csv
    Untracked:  data/ici.vs.eoi_marker.freq_low.probs.freq.removed_geno.ratio_5.batches_52.csv
    Untracked:  data/ici.vs.eoi_marker.freq_low.probs.freq.removed_geno.ratio_5.batches_mis.csv
    Untracked:  data/ici.vs.eoi_marker.freq_low.probs.freq.removed_geno.ratio_5.batches_mis_0.csv
    Untracked:  data/ici.vs.eoi_marker.freq_low.probs.freq.removed_geno.ratio_5.batches_mis_52.csv
    Untracked:  data/ici.vs.eoi_marker.freq_low.probs.freq.removed_sample.outliers.removed_geno.ratio.csv
    Untracked:  data/ici.vs.eoi_marker.freq_low.probs.freq.removed_sample.outliers.removed_geno.ratio_5.batches.csv
    Untracked:  data/ici.vs.eoi_marker.freq_low.probs.freq.removed_sample.outliers.removed_geno.ratio_5.batches_0.csv
    Untracked:  data/ici.vs.eoi_marker.freq_low.probs.freq.removed_sample.outliers.removed_geno.ratio_5.batches_52.csv
    Untracked:  data/ici.vs.eoi_marker.freq_low.probs.freq.removed_sample.outliers.removed_geno.ratio_5.batches_mis.csv
    Untracked:  data/ici.vs.eoi_marker.freq_low.probs.freq.removed_sample.outliers.removed_geno.ratio_5.batches_mis_0.csv
    Untracked:  data/ici.vs.eoi_marker.freq_low.probs.freq.removed_sample.outliers.removed_geno.ratio_5.batches_mis_52.csv
    Untracked:  data/ici.vs.eoi_rz.age-additive.covariates_blup_sub_chr-7_peak.marker-UNC13239362_lod.drop-1.5_snpsqc_5.batches.csv
    Untracked:  data/ici.vs.eoi_rz.age-additive.covariates_genes_chr-7_peak.marker-UNC13239362_lod.drop-1.5_snpsqc_5.batches.csv
    Untracked:  data/ici.vs.eoi_rz.age-no.covariates_blup_sub_chr-10_peak.marker-JAX00292927_lod.drop-1.5_5.batches_0.csv
    Untracked:  data/ici.vs.eoi_rz.age-no.covariates_blup_sub_chr-10_peak.marker-UNC18311938_lod.drop-1.5_5.batches_0.csv
    Untracked:  data/ici.vs.eoi_rz.age-no.covariates_blup_sub_chr-10_peak.marker-UNC18343181_lod.drop-1.5_5.batches_0.csv
    Untracked:  data/ici.vs.eoi_rz.age-no.covariates_blup_sub_chr-10_peak.marker-UNC18376338_lod.drop-1.5_5.batches_0.csv
    Untracked:  data/ici.vs.eoi_rz.age-no.covariates_blup_sub_chr-10_peak.marker-UNC18376338_lod.drop-1.5_snpsqc_5.batches_0.csv
    Untracked:  data/ici.vs.eoi_rz.age-no.covariates_blup_sub_chr-10_peak.marker-UNC18376338_lod.drop-1.5_snpsqc_5.batches_mis_0.csv
    Untracked:  data/ici.vs.eoi_rz.age-no.covariates_blup_sub_chr-10_peak.marker-UNC18382395_lod.drop-1.5_snpsqc_dis_no-x_updated_5.batches_0.csv
    Untracked:  data/ici.vs.eoi_rz.age-no.covariates_blup_sub_chr-10_peak.marker-UNCHS028236_lod.drop-1.5_snpsqc_5.batches_mis_0.csv
    Untracked:  data/ici.vs.eoi_rz.age-no.covariates_blup_sub_chr-10_peak.marker-UNCHS028236_lod.drop-1.5_snpsqc_5.batches_mis_52.csv
    Untracked:  data/ici.vs.eoi_rz.age-no.covariates_blup_sub_chr-10_peak.marker-UNCHS028536_lod.drop-1.5_snpsqc_dis_no-x_updated_5.batches_mis_0.csv
    Untracked:  data/ici.vs.eoi_rz.age-no.covariates_blup_sub_chr-10_peak.marker-UNCHS029024_lod.drop-1.5_5.batches_0.csv
    Untracked:  data/ici.vs.eoi_rz.age-no.covariates_blup_sub_chr-11_peak.marker-UNC19970181_lod.drop-1.5_snpsqc_5.batches_mis_0.csv
    Untracked:  data/ici.vs.eoi_rz.age-no.covariates_blup_sub_chr-11_peak.marker-UNC19970181_lod.drop-1.5_snpsqc_5.batches_mis_52.csv
    Untracked:  data/ici.vs.eoi_rz.age-no.covariates_blup_sub_chr-12_peak.marker-ICR499_lod.drop-1.5_snpsqc_5.batches_mis_0.csv
    Untracked:  data/ici.vs.eoi_rz.age-no.covariates_blup_sub_chr-12_peak.marker-ICR499_lod.drop-1.5_snpsqc_5.batches_mis_52.csv
    Untracked:  data/ici.vs.eoi_rz.age-no.covariates_blup_sub_chr-13_peak.marker-UNCHS036773_lod.drop-1.5_snpsqc_5.batches_mis_0.csv
    Untracked:  data/ici.vs.eoi_rz.age-no.covariates_blup_sub_chr-13_peak.marker-UNCHS036773_lod.drop-1.5_snpsqc_5.batches_mis_52.csv
    Untracked:  data/ici.vs.eoi_rz.age-no.covariates_blup_sub_chr-14_peak.marker-UNC24056202_lod.drop-1.5_snpsqc_5.batches_mis_0.csv
    Untracked:  data/ici.vs.eoi_rz.age-no.covariates_blup_sub_chr-14_peak.marker-UNC24056202_lod.drop-1.5_snpsqc_5.batches_mis_52.csv
    Untracked:  data/ici.vs.eoi_rz.age-no.covariates_blup_sub_chr-15_peak.marker-UNC26070435_lod.drop-1.5_snpsqc_dis_no-x_updated_5.batches_mis_0.csv
    Untracked:  data/ici.vs.eoi_rz.age-no.covariates_blup_sub_chr-15_peak.marker-UNC26070435_lod.drop-1.5_snpsqc_dis_no-x_updated_5.batches_mis_52.csv
    Untracked:  data/ici.vs.eoi_rz.age-no.covariates_blup_sub_chr-17_peak.marker-UNCHS044241_lod.drop-1.5_snpsqc_5.batches_mis_0.csv
    Untracked:  data/ici.vs.eoi_rz.age-no.covariates_blup_sub_chr-17_peak.marker-UNCHS044241_lod.drop-1.5_snpsqc_5.batches_mis_52.csv
    Untracked:  data/ici.vs.eoi_rz.age-no.covariates_blup_sub_chr-17_peak.marker-UNCHS044241_lod.drop-1.5_snpsqc_dis_no-x_updated_5.batches_mis_0.csv
    Untracked:  data/ici.vs.eoi_rz.age-no.covariates_blup_sub_chr-17_peak.marker-UNCrs47191360_lod.drop-1.5_snpsqc_dis_no-x_updated_5.batches_mis_52.csv
    Untracked:  data/ici.vs.eoi_rz.age-no.covariates_blup_sub_chr-18_peak.marker-UNCHS045343_lod.drop-1.5_snpsqc_5.batches_mis_0.csv
    Untracked:  data/ici.vs.eoi_rz.age-no.covariates_blup_sub_chr-18_peak.marker-UNCHS045343_lod.drop-1.5_snpsqc_5.batches_mis_52.csv
    Untracked:  data/ici.vs.eoi_rz.age-no.covariates_blup_sub_chr-1_peak.marker-UNC228500_lod.drop-1.5_snpsqc_dis_no-x_updated_5.batches_52.csv
    Untracked:  data/ici.vs.eoi_rz.age-no.covariates_blup_sub_chr-1_peak.marker-UNC228500_lod.drop-1.5_snpsqc_dis_no-x_updated_5.batches_mis_52.csv
    Untracked:  data/ici.vs.eoi_rz.age-no.covariates_blup_sub_chr-2_peak.marker-UNC4609527_lod.drop-1.5_snpsqc_dis_no-x_updated_5.batches_mis_0.csv
    Untracked:  data/ici.vs.eoi_rz.age-no.covariates_blup_sub_chr-2_peak.marker-UNC4609527_lod.drop-1.5_snpsqc_dis_no-x_updated_5.batches_mis_52.csv
    Untracked:  data/ici.vs.eoi_rz.age-no.covariates_blup_sub_chr-3_peak.marker-ICR1338_lod.drop-1.5_5.batches_0.csv
    Untracked:  data/ici.vs.eoi_rz.age-no.covariates_blup_sub_chr-3_peak.marker-ICR1338_lod.drop-1.5_5.batches_52.csv
    Untracked:  data/ici.vs.eoi_rz.age-no.covariates_blup_sub_chr-3_peak.marker-ICR2047_lod.drop-1.5_5.batches_52.csv
    Untracked:  data/ici.vs.eoi_rz.age-no.covariates_blup_sub_chr-3_peak.marker-JAX00106957_lod.drop-1.5_5.batches_0.csv
    Untracked:  data/ici.vs.eoi_rz.age-no.covariates_blup_sub_chr-3_peak.marker-JAX00112499_lod.drop-1.5_5.batches_52.csv
    Untracked:  data/ici.vs.eoi_rz.age-no.covariates_blup_sub_chr-3_peak.marker-JAX00219554_lod.drop-1.5_5.batches_0.csv
    Untracked:  data/ici.vs.eoi_rz.age-no.covariates_blup_sub_chr-3_peak.marker-JAX00220631_lod.drop-1.5_5.batches_0.csv
    Untracked:  data/ici.vs.eoi_rz.age-no.covariates_blup_sub_chr-3_peak.marker-JAX00514662_lod.drop-1.5_5.batches_52.csv
    Untracked:  data/ici.vs.eoi_rz.age-no.covariates_blup_sub_chr-3_peak.marker-JAX00529707_lod.drop-1.5_5.batches_52.csv
    Untracked:  data/ici.vs.eoi_rz.age-no.covariates_blup_sub_chr-3_peak.marker-UNC4813186_lod.drop-1.5_5.batches_0.csv
    Untracked:  data/ici.vs.eoi_rz.age-no.covariates_blup_sub_chr-3_peak.marker-UNC4813186_lod.drop-1.5_5.batches_52.csv
    Untracked:  data/ici.vs.eoi_rz.age-no.covariates_blup_sub_chr-3_peak.marker-UNC4836550_lod.drop-1.5_5.batches_52.csv
    Untracked:  data/ici.vs.eoi_rz.age-no.covariates_blup_sub_chr-3_peak.marker-UNC4837069_lod.drop-1.5_5.batches_0.csv
    Untracked:  data/ici.vs.eoi_rz.age-no.covariates_blup_sub_chr-3_peak.marker-UNC4912039_lod.drop-1.5_5.batches_0.csv
    Untracked:  data/ici.vs.eoi_rz.age-no.covariates_blup_sub_chr-3_peak.marker-UNC4912039_lod.drop-1.5_5.batches_52.csv
    Untracked:  data/ici.vs.eoi_rz.age-no.covariates_blup_sub_chr-3_peak.marker-UNC4924377_lod.drop-1.5_5.batches_0.csv
    Untracked:  data/ici.vs.eoi_rz.age-no.covariates_blup_sub_chr-3_peak.marker-UNC4924377_lod.drop-1.5_5.batches_52.csv
    Untracked:  data/ici.vs.eoi_rz.age-no.covariates_blup_sub_chr-3_peak.marker-UNC4956618_lod.drop-1.5_5.batches_52.csv
    Untracked:  data/ici.vs.eoi_rz.age-no.covariates_blup_sub_chr-3_peak.marker-UNC4957168_lod.drop-1.5_5.batches_0.csv
    Untracked:  data/ici.vs.eoi_rz.age-no.covariates_blup_sub_chr-3_peak.marker-UNC5006948_lod.drop-1.5_5.batches_52.csv
    Untracked:  data/ici.vs.eoi_rz.age-no.covariates_blup_sub_chr-3_peak.marker-UNC5008656_lod.drop-1.5_5.batches_0.csv
    Untracked:  data/ici.vs.eoi_rz.age-no.covariates_blup_sub_chr-3_peak.marker-UNC5083168_lod.drop-1.5_5.batches_0.csv
    Untracked:  data/ici.vs.eoi_rz.age-no.covariates_blup_sub_chr-3_peak.marker-UNC5083168_lod.drop-1.5_5.batches_52.csv
    Untracked:  data/ici.vs.eoi_rz.age-no.covariates_blup_sub_chr-3_peak.marker-UNC5192582_lod.drop-1.5_5.batches_0.csv
    Untracked:  data/ici.vs.eoi_rz.age-no.covariates_blup_sub_chr-3_peak.marker-UNC5213888_lod.drop-1.5_snpsqc_5.batches_mis_0.csv
    Untracked:  data/ici.vs.eoi_rz.age-no.covariates_blup_sub_chr-3_peak.marker-UNC5213888_lod.drop-1.5_snpsqc_5.batches_mis_52.csv
    Untracked:  data/ici.vs.eoi_rz.age-no.covariates_blup_sub_chr-3_peak.marker-UNC5262472_lod.drop-1.5_5.batches_0.csv
    Untracked:  data/ici.vs.eoi_rz.age-no.covariates_blup_sub_chr-3_peak.marker-UNC5262472_lod.drop-1.5_5.batches_52.csv
    Untracked:  data/ici.vs.eoi_rz.age-no.covariates_blup_sub_chr-3_peak.marker-UNC5266664_lod.drop-1.5_5.batches_0.csv
    Untracked:  data/ici.vs.eoi_rz.age-no.covariates_blup_sub_chr-3_peak.marker-UNC5300878_lod.drop-1.5_5.batches_52.csv
    Untracked:  data/ici.vs.eoi_rz.age-no.covariates_blup_sub_chr-3_peak.marker-UNC5360055_lod.drop-1.5_5.batches_52.csv
    Untracked:  data/ici.vs.eoi_rz.age-no.covariates_blup_sub_chr-3_peak.marker-UNC5454839_lod.drop-1.5_5.batches_0.csv
    Untracked:  data/ici.vs.eoi_rz.age-no.covariates_blup_sub_chr-3_peak.marker-UNC5454839_lod.drop-1.5_5.batches_52.csv
    Untracked:  data/ici.vs.eoi_rz.age-no.covariates_blup_sub_chr-3_peak.marker-UNC5645844_lod.drop-1.5_5.batches_0.csv
    Untracked:  data/ici.vs.eoi_rz.age-no.covariates_blup_sub_chr-3_peak.marker-UNC5645844_lod.drop-1.5_5.batches_52.csv
    Untracked:  data/ici.vs.eoi_rz.age-no.covariates_blup_sub_chr-3_peak.marker-UNC5679155_lod.drop-1.5_5.batches_52.csv
    Untracked:  data/ici.vs.eoi_rz.age-no.covariates_blup_sub_chr-3_peak.marker-UNC5808246_lod.drop-1.5_5.batches_52.csv
    Untracked:  data/ici.vs.eoi_rz.age-no.covariates_blup_sub_chr-3_peak.marker-UNC5945286_lod.drop-1.5_5.batches_52.csv
    Untracked:  data/ici.vs.eoi_rz.age-no.covariates_blup_sub_chr-3_peak.marker-UNC6227106_lod.drop-1.5_5.batches_52.csv
    Untracked:  data/ici.vs.eoi_rz.age-no.covariates_blup_sub_chr-3_peak.marker-UNCHS008182_lod.drop-1.5_5.batches_52.csv
    Untracked:  data/ici.vs.eoi_rz.age-no.covariates_blup_sub_chr-3_peak.marker-UNCHS008286_lod.drop-1.5_5.batches_0.csv
    Untracked:  data/ici.vs.eoi_rz.age-no.covariates_blup_sub_chr-3_peak.marker-UNCHS008286_lod.drop-1.5_5.batches_52.csv
    Untracked:  data/ici.vs.eoi_rz.age-no.covariates_blup_sub_chr-3_peak.marker-UNCHS008487_lod.drop-1.5_snpsqc_5.batches_0.csv
    Untracked:  data/ici.vs.eoi_rz.age-no.covariates_blup_sub_chr-3_peak.marker-UNCHS008487_lod.drop-1.5_snpsqc_5.batches_52.csv
    Untracked:  data/ici.vs.eoi_rz.age-no.covariates_blup_sub_chr-3_peak.marker-UNCHS008487_lod.drop-1.5_snpsqc_dis_no-x_updated_5.batches_0.csv
    Untracked:  data/ici.vs.eoi_rz.age-no.covariates_blup_sub_chr-3_peak.marker-UNCHS008487_lod.drop-1.5_snpsqc_dis_no-x_updated_5.batches_52.csv
    Untracked:  data/ici.vs.eoi_rz.age-no.covariates_blup_sub_chr-3_peak.marker-UNCHS008487_lod.drop-1.5_snpsqc_dis_no-x_updated_5.batches_mis_0.csv
    Untracked:  data/ici.vs.eoi_rz.age-no.covariates_blup_sub_chr-3_peak.marker-UNCHS008487_lod.drop-1.5_snpsqc_dis_no-x_updated_5.batches_mis_52.csv
    Untracked:  data/ici.vs.eoi_rz.age-no.covariates_blup_sub_chr-3_peak.marker-UNCHS008511_lod.drop-1.5_snpsqc_5.batches_mis_0.csv
    Untracked:  data/ici.vs.eoi_rz.age-no.covariates_blup_sub_chr-3_peak.marker-UNCHS008511_lod.drop-1.5_snpsqc_5.batches_mis_52.csv
    Untracked:  data/ici.vs.eoi_rz.age-no.covariates_blup_sub_chr-3_peak.marker-UNCHS008595_lod.drop-1.5_5.batches_52.csv
    Untracked:  data/ici.vs.eoi_rz.age-no.covariates_blup_sub_chr-3_peak.marker-UNCHS008613_lod.drop-1.5_5.batches_0.csv
    Untracked:  data/ici.vs.eoi_rz.age-no.covariates_blup_sub_chr-3_peak.marker-UNCHS008613_lod.drop-1.5_5.batches_52.csv
    Untracked:  data/ici.vs.eoi_rz.age-no.covariates_blup_sub_chr-3_peak.marker-UNCHS008815_lod.drop-1.5_5.batches_0.csv
    Untracked:  data/ici.vs.eoi_rz.age-no.covariates_blup_sub_chr-3_peak.marker-UNCHS008815_lod.drop-1.5_5.batches_52.csv
    Untracked:  data/ici.vs.eoi_rz.age-no.covariates_blup_sub_chr-3_peak.marker-UNCHS009067_lod.drop-1.5_5.batches_0.csv
    Untracked:  data/ici.vs.eoi_rz.age-no.covariates_blup_sub_chr-3_peak.marker-UNCHS009383_lod.drop-1.5_5.batches_52.csv
    Untracked:  data/ici.vs.eoi_rz.age-no.covariates_blup_sub_chr-3_peak.marker-UNCHS009693_lod.drop-1.5_5.batches_52.csv
    Untracked:  data/ici.vs.eoi_rz.age-no.covariates_blup_sub_chr-3_peak.marker-UNCJPD001246_lod.drop-1.5_5.batches_52.csv
    Untracked:  data/ici.vs.eoi_rz.age-no.covariates_blup_sub_chr-3_peak.marker-UNCJPD001341_lod.drop-1.5_5.batches_52.csv
    Untracked:  data/ici.vs.eoi_rz.age-no.covariates_blup_sub_chr-3_peak.marker-sanger2623_lod.drop-1.5_5.batches_52.csv
    Untracked:  data/ici.vs.eoi_rz.age-no.covariates_blup_sub_chr-4_peak.marker-UNCHS013125_lod.drop-1.5_snpsqc_dis_no-x_updated_5.batches_52.csv
    Untracked:  data/ici.vs.eoi_rz.age-no.covariates_blup_sub_chr-4_peak.marker-UNCHS013125_lod.drop-1.5_snpsqc_dis_no-x_updated_5.batches_mis_52.csv
    Untracked:  data/ici.vs.eoi_rz.age-no.covariates_blup_sub_chr-6_peak.marker-UNCHS018170_lod.drop-1.5_snpsqc_5.batches_0.csv
    Untracked:  data/ici.vs.eoi_rz.age-no.covariates_blup_sub_chr-6_peak.marker-UNCHS018231_lod.drop-1.5_snpsqc_5.batches_mis_0.csv
    Untracked:  data/ici.vs.eoi_rz.age-no.covariates_blup_sub_chr-7_peak.marker-UNCHS020066_lod.drop-1.5_snpsqc_5.batches_mis_0.csv
    Untracked:  data/ici.vs.eoi_rz.age-no.covariates_blup_sub_chr-7_peak.marker-UNCHS020066_lod.drop-1.5_snpsqc_5.batches_mis_52.csv
    Untracked:  data/ici.vs.eoi_rz.age-no.covariates_blup_sub_chr-7_peak.marker-UNCHS020066_lod.drop-1.5_snpsqc_dis_no-x_updated_5.batches_mis_0.csv
    Untracked:  data/ici.vs.eoi_rz.age-no.covariates_blup_sub_chr-7_peak.marker-UNCHS020066_lod.drop-1.5_snpsqc_dis_no-x_updated_5.batches_mis_52.csv
    Untracked:  data/ici.vs.eoi_rz.age-no.covariates_blup_sub_chr-8_peak.marker-UNC15524531_lod.drop-1.5_snpsqc_5.batches_mis_0.csv
    Untracked:  data/ici.vs.eoi_rz.age-no.covariates_blup_sub_chr-8_peak.marker-UNC15524531_lod.drop-1.5_snpsqc_5.batches_mis_52.csv
    Untracked:  data/ici.vs.eoi_rz.age-no.covariates_blup_sub_chr-8_peak.marker-UNC15524531_lod.drop-1.5_snpsqc_dis_no-x_updated_5.batches_mis_0.csv
    Untracked:  data/ici.vs.eoi_rz.age-no.covariates_blup_sub_chr-8_peak.marker-UNC15524531_lod.drop-1.5_snpsqc_dis_no-x_updated_5.batches_mis_52.csv
    Untracked:  data/ici.vs.eoi_rz.age-no.covariates_blup_sub_chr-9_peak.marker-UNC17203597_lod.drop-1.5_snpsqc_5.batches_mis_0.csv
    Untracked:  data/ici.vs.eoi_rz.age-no.covariates_blup_sub_chr-9_peak.marker-UNC17203597_lod.drop-1.5_snpsqc_5.batches_mis_52.csv
    Untracked:  data/ici.vs.eoi_rz.age-no.covariates_blup_sub_chr-X_peak.marker-UNCHS048314_lod.drop-1.5_snpsqc_5.batches_mis_0.csv
    Untracked:  data/ici.vs.eoi_rz.age-no.covariates_blup_sub_chr-X_peak.marker-UNCHS048314_lod.drop-1.5_snpsqc_5.batches_mis_52.csv
    Untracked:  data/ici.vs.eoi_rz.age-no.covariates_blup_sub_chr-X_peak.marker-UNCHS048314_lod.drop-1.5_snpsqc_dis_no-x_updated_5.batches_mis_0.csv
    Untracked:  data/ici.vs.eoi_rz.age-no.covariates_blup_sub_chr-X_peak.marker-UNCHS048314_lod.drop-1.5_snpsqc_dis_no-x_updated_5.batches_mis_52.csv
    Untracked:  data/ici.vs.eoi_rz.age-no.covariates_genes_chr-10_peak.marker-JAX00292927_lod.drop-1.5_5.batches_0.csv
    Untracked:  data/ici.vs.eoi_rz.age-no.covariates_genes_chr-10_peak.marker-UNC18311938_lod.drop-1.5_5.batches_0.csv
    Untracked:  data/ici.vs.eoi_rz.age-no.covariates_genes_chr-10_peak.marker-UNC18343181_lod.drop-1.5_5.batches_0.csv
    Untracked:  data/ici.vs.eoi_rz.age-no.covariates_genes_chr-10_peak.marker-UNC18376338_lod.drop-1.5_5.batches_0.csv
    Untracked:  data/ici.vs.eoi_rz.age-no.covariates_genes_chr-10_peak.marker-UNC18376338_lod.drop-1.5_snpsqc_5.batches_0.csv
    Untracked:  data/ici.vs.eoi_rz.age-no.covariates_genes_chr-10_peak.marker-UNC18376338_lod.drop-1.5_snpsqc_5.batches_mis_0.csv
    Untracked:  data/ici.vs.eoi_rz.age-no.covariates_genes_chr-10_peak.marker-UNC18382395_lod.drop-1.5_snpsqc_dis_no-x_updated_5.batches_0.csv
    Untracked:  data/ici.vs.eoi_rz.age-no.covariates_genes_chr-10_peak.marker-UNCHS028236_lod.drop-1.5_snpsqc_5.batches_mis_0.csv
    Untracked:  data/ici.vs.eoi_rz.age-no.covariates_genes_chr-10_peak.marker-UNCHS028236_lod.drop-1.5_snpsqc_5.batches_mis_52.csv
    Untracked:  data/ici.vs.eoi_rz.age-no.covariates_genes_chr-10_peak.marker-UNCHS028536_lod.drop-1.5_snpsqc_dis_no-x_updated_5.batches_mis_0.csv
    Untracked:  data/ici.vs.eoi_rz.age-no.covariates_genes_chr-10_peak.marker-UNCHS029024_lod.drop-1.5_5.batches_0.csv
    Untracked:  data/ici.vs.eoi_rz.age-no.covariates_genes_chr-11_peak.marker-UNC19970181_lod.drop-1.5_snpsqc_5.batches_mis_0.csv
    Untracked:  data/ici.vs.eoi_rz.age-no.covariates_genes_chr-11_peak.marker-UNC19970181_lod.drop-1.5_snpsqc_5.batches_mis_52.csv
    Untracked:  data/ici.vs.eoi_rz.age-no.covariates_genes_chr-12_peak.marker-ICR499_lod.drop-1.5_snpsqc_5.batches_mis_0.csv
    Untracked:  data/ici.vs.eoi_rz.age-no.covariates_genes_chr-12_peak.marker-ICR499_lod.drop-1.5_snpsqc_5.batches_mis_52.csv
    Untracked:  data/ici.vs.eoi_rz.age-no.covariates_genes_chr-13_peak.marker-UNCHS036773_lod.drop-1.5_snpsqc_5.batches_mis_0.csv
    Untracked:  data/ici.vs.eoi_rz.age-no.covariates_genes_chr-13_peak.marker-UNCHS036773_lod.drop-1.5_snpsqc_5.batches_mis_52.csv
    Untracked:  data/ici.vs.eoi_rz.age-no.covariates_genes_chr-14_peak.marker-UNC24056202_lod.drop-1.5_snpsqc_5.batches_mis_0.csv
    Untracked:  data/ici.vs.eoi_rz.age-no.covariates_genes_chr-14_peak.marker-UNC24056202_lod.drop-1.5_snpsqc_5.batches_mis_52.csv
    Untracked:  data/ici.vs.eoi_rz.age-no.covariates_genes_chr-15_peak.marker-UNC26070435_lod.drop-1.5_snpsqc_dis_no-x_updated_5.batches_mis_0.csv
    Untracked:  data/ici.vs.eoi_rz.age-no.covariates_genes_chr-15_peak.marker-UNC26070435_lod.drop-1.5_snpsqc_dis_no-x_updated_5.batches_mis_52.csv
    Untracked:  data/ici.vs.eoi_rz.age-no.covariates_genes_chr-17_peak.marker-UNCHS044241_lod.drop-1.5_snpsqc_5.batches_mis_0.csv
    Untracked:  data/ici.vs.eoi_rz.age-no.covariates_genes_chr-17_peak.marker-UNCHS044241_lod.drop-1.5_snpsqc_5.batches_mis_52.csv
    Untracked:  data/ici.vs.eoi_rz.age-no.covariates_genes_chr-17_peak.marker-UNCHS044241_lod.drop-1.5_snpsqc_dis_no-x_updated_5.batches_mis_0.csv
    Untracked:  data/ici.vs.eoi_rz.age-no.covariates_genes_chr-17_peak.marker-UNCrs47191360_lod.drop-1.5_snpsqc_dis_no-x_updated_5.batches_mis_52.csv
    Untracked:  data/ici.vs.eoi_rz.age-no.covariates_genes_chr-18_peak.marker-UNCHS045343_lod.drop-1.5_snpsqc_5.batches_mis_0.csv
    Untracked:  data/ici.vs.eoi_rz.age-no.covariates_genes_chr-18_peak.marker-UNCHS045343_lod.drop-1.5_snpsqc_5.batches_mis_52.csv
    Untracked:  data/ici.vs.eoi_rz.age-no.covariates_genes_chr-1_peak.marker-UNC228500_lod.drop-1.5_snpsqc_dis_no-x_updated_5.batches_52.csv
    Untracked:  data/ici.vs.eoi_rz.age-no.covariates_genes_chr-1_peak.marker-UNC228500_lod.drop-1.5_snpsqc_dis_no-x_updated_5.batches_mis_52.csv
    Untracked:  data/ici.vs.eoi_rz.age-no.covariates_genes_chr-2_peak.marker-UNC4609527_lod.drop-1.5_snpsqc_dis_no-x_updated_5.batches_mis_0.csv
    Untracked:  data/ici.vs.eoi_rz.age-no.covariates_genes_chr-2_peak.marker-UNC4609527_lod.drop-1.5_snpsqc_dis_no-x_updated_5.batches_mis_52.csv
    Untracked:  data/ici.vs.eoi_rz.age-no.covariates_genes_chr-3_peak.marker-ICR1338_lod.drop-1.5_5.batches_0.csv
    Untracked:  data/ici.vs.eoi_rz.age-no.covariates_genes_chr-3_peak.marker-ICR1338_lod.drop-1.5_5.batches_52.csv
    Untracked:  data/ici.vs.eoi_rz.age-no.covariates_genes_chr-3_peak.marker-ICR2047_lod.drop-1.5_5.batches_52.csv
    Untracked:  data/ici.vs.eoi_rz.age-no.covariates_genes_chr-3_peak.marker-JAX00106957_lod.drop-1.5_5.batches_0.csv
    Untracked:  data/ici.vs.eoi_rz.age-no.covariates_genes_chr-3_peak.marker-JAX00112499_lod.drop-1.5_5.batches_52.csv
    Untracked:  data/ici.vs.eoi_rz.age-no.covariates_genes_chr-3_peak.marker-JAX00219554_lod.drop-1.5_5.batches_0.csv
    Untracked:  data/ici.vs.eoi_rz.age-no.covariates_genes_chr-3_peak.marker-JAX00220631_lod.drop-1.5_5.batches_0.csv
    Untracked:  data/ici.vs.eoi_rz.age-no.covariates_genes_chr-3_peak.marker-JAX00514662_lod.drop-1.5_5.batches_52.csv
    Untracked:  data/ici.vs.eoi_rz.age-no.covariates_genes_chr-3_peak.marker-JAX00529707_lod.drop-1.5_5.batches_52.csv
    Untracked:  data/ici.vs.eoi_rz.age-no.covariates_genes_chr-3_peak.marker-UNC4813186_lod.drop-1.5_5.batches_0.csv
    Untracked:  data/ici.vs.eoi_rz.age-no.covariates_genes_chr-3_peak.marker-UNC4813186_lod.drop-1.5_5.batches_52.csv
    Untracked:  data/ici.vs.eoi_rz.age-no.covariates_genes_chr-3_peak.marker-UNC4836550_lod.drop-1.5_5.batches_52.csv
    Untracked:  data/ici.vs.eoi_rz.age-no.covariates_genes_chr-3_peak.marker-UNC4837069_lod.drop-1.5_5.batches_0.csv
    Untracked:  data/ici.vs.eoi_rz.age-no.covariates_genes_chr-3_peak.marker-UNC4912039_lod.drop-1.5_5.batches_0.csv
    Untracked:  data/ici.vs.eoi_rz.age-no.covariates_genes_chr-3_peak.marker-UNC4912039_lod.drop-1.5_5.batches_52.csv
    Untracked:  data/ici.vs.eoi_rz.age-no.covariates_genes_chr-3_peak.marker-UNC4924377_lod.drop-1.5_5.batches_0.csv
    Untracked:  data/ici.vs.eoi_rz.age-no.covariates_genes_chr-3_peak.marker-UNC4924377_lod.drop-1.5_5.batches_52.csv
    Untracked:  data/ici.vs.eoi_rz.age-no.covariates_genes_chr-3_peak.marker-UNC4956618_lod.drop-1.5_5.batches_52.csv
    Untracked:  data/ici.vs.eoi_rz.age-no.covariates_genes_chr-3_peak.marker-UNC4957168_lod.drop-1.5_5.batches_0.csv
    Untracked:  data/ici.vs.eoi_rz.age-no.covariates_genes_chr-3_peak.marker-UNC5006948_lod.drop-1.5_5.batches_52.csv
    Untracked:  data/ici.vs.eoi_rz.age-no.covariates_genes_chr-3_peak.marker-UNC5008656_lod.drop-1.5_5.batches_0.csv
    Untracked:  data/ici.vs.eoi_rz.age-no.covariates_genes_chr-3_peak.marker-UNC5083168_lod.drop-1.5_5.batches_0.csv
    Untracked:  data/ici.vs.eoi_rz.age-no.covariates_genes_chr-3_peak.marker-UNC5083168_lod.drop-1.5_5.batches_52.csv
    Untracked:  data/ici.vs.eoi_rz.age-no.covariates_genes_chr-3_peak.marker-UNC5192582_lod.drop-1.5_5.batches_0.csv
    Untracked:  data/ici.vs.eoi_rz.age-no.covariates_genes_chr-3_peak.marker-UNC5213888_lod.drop-1.5_snpsqc_5.batches_mis_0.csv
    Untracked:  data/ici.vs.eoi_rz.age-no.covariates_genes_chr-3_peak.marker-UNC5213888_lod.drop-1.5_snpsqc_5.batches_mis_52.csv
    Untracked:  data/ici.vs.eoi_rz.age-no.covariates_genes_chr-3_peak.marker-UNC5262472_lod.drop-1.5_5.batches_0.csv
    Untracked:  data/ici.vs.eoi_rz.age-no.covariates_genes_chr-3_peak.marker-UNC5262472_lod.drop-1.5_5.batches_52.csv
    Untracked:  data/ici.vs.eoi_rz.age-no.covariates_genes_chr-3_peak.marker-UNC5266664_lod.drop-1.5_5.batches_0.csv
    Untracked:  data/ici.vs.eoi_rz.age-no.covariates_genes_chr-3_peak.marker-UNC5300878_lod.drop-1.5_5.batches_52.csv
    Untracked:  data/ici.vs.eoi_rz.age-no.covariates_genes_chr-3_peak.marker-UNC5360055_lod.drop-1.5_5.batches_52.csv
    Untracked:  data/ici.vs.eoi_rz.age-no.covariates_genes_chr-3_peak.marker-UNC5454839_lod.drop-1.5_5.batches_0.csv
    Untracked:  data/ici.vs.eoi_rz.age-no.covariates_genes_chr-3_peak.marker-UNC5454839_lod.drop-1.5_5.batches_52.csv
    Untracked:  data/ici.vs.eoi_rz.age-no.covariates_genes_chr-3_peak.marker-UNC5645844_lod.drop-1.5_5.batches_0.csv
    Untracked:  data/ici.vs.eoi_rz.age-no.covariates_genes_chr-3_peak.marker-UNC5645844_lod.drop-1.5_5.batches_52.csv
    Untracked:  data/ici.vs.eoi_rz.age-no.covariates_genes_chr-3_peak.marker-UNC5679155_lod.drop-1.5_5.batches_52.csv
    Untracked:  data/ici.vs.eoi_rz.age-no.covariates_genes_chr-3_peak.marker-UNC5808246_lod.drop-1.5_5.batches_52.csv
    Untracked:  data/ici.vs.eoi_rz.age-no.covariates_genes_chr-3_peak.marker-UNC5945286_lod.drop-1.5_5.batches_52.csv
    Untracked:  data/ici.vs.eoi_rz.age-no.covariates_genes_chr-3_peak.marker-UNC6227106_lod.drop-1.5_5.batches_52.csv
    Untracked:  data/ici.vs.eoi_rz.age-no.covariates_genes_chr-3_peak.marker-UNCHS008182_lod.drop-1.5_5.batches_52.csv
    Untracked:  data/ici.vs.eoi_rz.age-no.covariates_genes_chr-3_peak.marker-UNCHS008286_lod.drop-1.5_5.batches_0.csv
    Untracked:  data/ici.vs.eoi_rz.age-no.covariates_genes_chr-3_peak.marker-UNCHS008286_lod.drop-1.5_5.batches_52.csv
    Untracked:  data/ici.vs.eoi_rz.age-no.covariates_genes_chr-3_peak.marker-UNCHS008487_lod.drop-1.5_snpsqc_5.batches_0.csv
    Untracked:  data/ici.vs.eoi_rz.age-no.covariates_genes_chr-3_peak.marker-UNCHS008487_lod.drop-1.5_snpsqc_5.batches_52.csv
    Untracked:  data/ici.vs.eoi_rz.age-no.covariates_genes_chr-3_peak.marker-UNCHS008487_lod.drop-1.5_snpsqc_dis_no-x_updated_5.batches_0.csv
    Untracked:  data/ici.vs.eoi_rz.age-no.covariates_genes_chr-3_peak.marker-UNCHS008487_lod.drop-1.5_snpsqc_dis_no-x_updated_5.batches_52.csv
    Untracked:  data/ici.vs.eoi_rz.age-no.covariates_genes_chr-3_peak.marker-UNCHS008487_lod.drop-1.5_snpsqc_dis_no-x_updated_5.batches_mis_0.csv
    Untracked:  data/ici.vs.eoi_rz.age-no.covariates_genes_chr-3_peak.marker-UNCHS008487_lod.drop-1.5_snpsqc_dis_no-x_updated_5.batches_mis_52.csv
    Untracked:  data/ici.vs.eoi_rz.age-no.covariates_genes_chr-3_peak.marker-UNCHS008511_lod.drop-1.5_snpsqc_5.batches_mis_0.csv
    Untracked:  data/ici.vs.eoi_rz.age-no.covariates_genes_chr-3_peak.marker-UNCHS008511_lod.drop-1.5_snpsqc_5.batches_mis_52.csv
    Untracked:  data/ici.vs.eoi_rz.age-no.covariates_genes_chr-3_peak.marker-UNCHS008595_lod.drop-1.5_5.batches_52.csv
    Untracked:  data/ici.vs.eoi_rz.age-no.covariates_genes_chr-3_peak.marker-UNCHS008613_lod.drop-1.5_5.batches_0.csv
    Untracked:  data/ici.vs.eoi_rz.age-no.covariates_genes_chr-3_peak.marker-UNCHS008613_lod.drop-1.5_5.batches_52.csv
    Untracked:  data/ici.vs.eoi_rz.age-no.covariates_genes_chr-3_peak.marker-UNCHS008815_lod.drop-1.5_5.batches_0.csv
    Untracked:  data/ici.vs.eoi_rz.age-no.covariates_genes_chr-3_peak.marker-UNCHS008815_lod.drop-1.5_5.batches_52.csv
    Untracked:  data/ici.vs.eoi_rz.age-no.covariates_genes_chr-3_peak.marker-UNCHS009067_lod.drop-1.5_5.batches_0.csv
    Untracked:  data/ici.vs.eoi_rz.age-no.covariates_genes_chr-3_peak.marker-UNCHS009383_lod.drop-1.5_5.batches_52.csv
    Untracked:  data/ici.vs.eoi_rz.age-no.covariates_genes_chr-3_peak.marker-UNCHS009693_lod.drop-1.5_5.batches_52.csv
    Untracked:  data/ici.vs.eoi_rz.age-no.covariates_genes_chr-3_peak.marker-UNCJPD001246_lod.drop-1.5_5.batches_52.csv
    Untracked:  data/ici.vs.eoi_rz.age-no.covariates_genes_chr-3_peak.marker-UNCJPD001341_lod.drop-1.5_5.batches_52.csv
    Untracked:  data/ici.vs.eoi_rz.age-no.covariates_genes_chr-3_peak.marker-sanger2623_lod.drop-1.5_5.batches_52.csv
    Untracked:  data/ici.vs.eoi_rz.age-no.covariates_genes_chr-4_peak.marker-UNCHS013125_lod.drop-1.5_snpsqc_dis_no-x_updated_5.batches_52.csv
    Untracked:  data/ici.vs.eoi_rz.age-no.covariates_genes_chr-4_peak.marker-UNCHS013125_lod.drop-1.5_snpsqc_dis_no-x_updated_5.batches_mis_52.csv
    Untracked:  data/ici.vs.eoi_rz.age-no.covariates_genes_chr-6_peak.marker-UNCHS018170_lod.drop-1.5_snpsqc_5.batches_0.csv
    Untracked:  data/ici.vs.eoi_rz.age-no.covariates_genes_chr-6_peak.marker-UNCHS018231_lod.drop-1.5_snpsqc_5.batches_mis_0.csv
    Untracked:  data/ici.vs.eoi_rz.age-no.covariates_genes_chr-7_peak.marker-UNCHS020066_lod.drop-1.5_snpsqc_5.batches_mis_0.csv
    Untracked:  data/ici.vs.eoi_rz.age-no.covariates_genes_chr-7_peak.marker-UNCHS020066_lod.drop-1.5_snpsqc_5.batches_mis_52.csv
    Untracked:  data/ici.vs.eoi_rz.age-no.covariates_genes_chr-7_peak.marker-UNCHS020066_lod.drop-1.5_snpsqc_dis_no-x_updated_5.batches_mis_0.csv
    Untracked:  data/ici.vs.eoi_rz.age-no.covariates_genes_chr-7_peak.marker-UNCHS020066_lod.drop-1.5_snpsqc_dis_no-x_updated_5.batches_mis_52.csv
    Untracked:  data/ici.vs.eoi_rz.age-no.covariates_genes_chr-8_peak.marker-UNC15524531_lod.drop-1.5_snpsqc_5.batches_mis_0.csv
    Untracked:  data/ici.vs.eoi_rz.age-no.covariates_genes_chr-8_peak.marker-UNC15524531_lod.drop-1.5_snpsqc_5.batches_mis_52.csv
    Untracked:  data/ici.vs.eoi_rz.age-no.covariates_genes_chr-8_peak.marker-UNC15524531_lod.drop-1.5_snpsqc_dis_no-x_updated_5.batches_mis_0.csv
    Untracked:  data/ici.vs.eoi_rz.age-no.covariates_genes_chr-8_peak.marker-UNC15524531_lod.drop-1.5_snpsqc_dis_no-x_updated_5.batches_mis_52.csv
    Untracked:  data/ici.vs.eoi_rz.age-no.covariates_genes_chr-9_peak.marker-UNC17203597_lod.drop-1.5_snpsqc_5.batches_mis_0.csv
    Untracked:  data/ici.vs.eoi_rz.age-no.covariates_genes_chr-9_peak.marker-UNC17203597_lod.drop-1.5_snpsqc_5.batches_mis_52.csv
    Untracked:  data/ici.vs.eoi_rz.age-no.covariates_genes_chr-X_peak.marker-UNCHS048314_lod.drop-1.5_snpsqc_5.batches_mis_0.csv
    Untracked:  data/ici.vs.eoi_rz.age-no.covariates_genes_chr-X_peak.marker-UNCHS048314_lod.drop-1.5_snpsqc_5.batches_mis_52.csv
    Untracked:  data/ici.vs.eoi_rz.age-no.covariates_genes_chr-X_peak.marker-UNCHS048314_lod.drop-1.5_snpsqc_dis_no-x_updated_5.batches_mis_0.csv
    Untracked:  data/ici.vs.eoi_rz.age-no.covariates_genes_chr-X_peak.marker-UNCHS048314_lod.drop-1.5_snpsqc_dis_no-x_updated_5.batches_mis_52.csv
    Untracked:  data/ici.vs.eoi_sample.genos_marker.freq_low.geno.freq.removed.csv
    Untracked:  data/ici.vs.eoi_sample.genos_marker.freq_low.geno.freq.removed_sample.outliers.removed.csv
    Untracked:  data/ici.vs.eoi_sample.genos_marker.freq_low.probs.freq.removed.csv
    Untracked:  data/ici.vs.eoi_sample.genos_marker.freq_low.probs.freq.removed_sample.outliers.removed.csv
    Untracked:  data/ici.vs.eoi_scanone_5.batches.Rdata
    Untracked:  data/ici.vs.eoi_scanone_5.batches_mis.Rdata
    Untracked:  data/ici.vs.eoi_scanone_snpsqc_5.batches.Rdata
    Untracked:  data/ici.vs.eoi_scanone_snpsqc_5.batches_mis.Rdata
    Untracked:  data/ici.vs.eoi_scanone_snpsqc_dis_no-x_updated_5.batches.Rdata
    Untracked:  data/ici.vs.pbs_age.of.onset-no.covariates_blup_sub_chr-14_peak.marker-UNC24056202_lod.drop-1.5_5.batches_mis.csv
    Untracked:  data/ici.vs.pbs_age.of.onset-no.covariates_blup_sub_chr-14_peak.marker-UNC24056202_lod.drop-1.5_snpsqc_5.batches_mis.csv
    Untracked:  data/ici.vs.pbs_age.of.onset-no.covariates_blup_sub_chr-17_peak.marker-UNCJPD006614_lod.drop-1.5_snpsqc_5.batches_mis.csv
    Untracked:  data/ici.vs.pbs_age.of.onset-no.covariates_blup_sub_chr-18_peak.marker-UNC28655293_lod.drop-1.5_5.batches_mis.csv
    Untracked:  data/ici.vs.pbs_age.of.onset-no.covariates_blup_sub_chr-18_peak.marker-UNCHS045343_lod.drop-1.5_snpsqc_5.batches_mis.csv
    Untracked:  data/ici.vs.pbs_age.of.onset-no.covariates_blup_sub_chr-5_peak.marker-UNC10044126_lod.drop-1.5_5.batches_mis.csv
    Untracked:  data/ici.vs.pbs_age.of.onset-no.covariates_blup_sub_chr-5_peak.marker-UNC8675939_lod.drop-1.5_snpsqc_5.batches_mis.csv
    Untracked:  data/ici.vs.pbs_age.of.onset-no.covariates_blup_sub_chr-8_peak.marker-JAX00667121_lod.drop-1.5_5.batches_mis.csv
    Untracked:  data/ici.vs.pbs_age.of.onset-no.covariates_blup_sub_chr-8_peak.marker-UNC15524531_lod.drop-1.5_snpsqc_5.batches_mis.csv
    Untracked:  data/ici.vs.pbs_age.of.onset-no.covariates_blup_sub_chr-X_peak.marker-UNCHS048314_lod.drop-1.5_snpsqc_5.batches_mis.csv
    Untracked:  data/ici.vs.pbs_age.of.onset-no.covariates_genes_chr-14_peak.marker-UNC24056202_lod.drop-1.5_5.batches_mis.csv
    Untracked:  data/ici.vs.pbs_age.of.onset-no.covariates_genes_chr-14_peak.marker-UNC24056202_lod.drop-1.5_snpsqc_5.batches_mis.csv
    Untracked:  data/ici.vs.pbs_age.of.onset-no.covariates_genes_chr-17_peak.marker-UNCJPD006614_lod.drop-1.5_snpsqc_5.batches_mis.csv
    Untracked:  data/ici.vs.pbs_age.of.onset-no.covariates_genes_chr-18_peak.marker-UNC28655293_lod.drop-1.5_5.batches_mis.csv
    Untracked:  data/ici.vs.pbs_age.of.onset-no.covariates_genes_chr-18_peak.marker-UNCHS045343_lod.drop-1.5_snpsqc_5.batches_mis.csv
    Untracked:  data/ici.vs.pbs_age.of.onset-no.covariates_genes_chr-5_peak.marker-UNC10044126_lod.drop-1.5_5.batches_mis.csv
    Untracked:  data/ici.vs.pbs_age.of.onset-no.covariates_genes_chr-5_peak.marker-UNC8675939_lod.drop-1.5_snpsqc_5.batches_mis.csv
    Untracked:  data/ici.vs.pbs_age.of.onset-no.covariates_genes_chr-8_peak.marker-JAX00667121_lod.drop-1.5_5.batches_mis.csv
    Untracked:  data/ici.vs.pbs_age.of.onset-no.covariates_genes_chr-8_peak.marker-UNC15524531_lod.drop-1.5_snpsqc_5.batches_mis.csv
    Untracked:  data/ici.vs.pbs_age.of.onset-no.covariates_genes_chr-X_peak.marker-UNCHS048314_lod.drop-1.5_snpsqc_5.batches_mis.csv
    Untracked:  data/ici.vs.pbs_bad.markers_snpsqc_5.batches_mis.csv
    Untracked:  data/ici.vs.pbs_blup.full_chr-10_lod.drop-1.5_5.batches.csv
    Untracked:  data/ici.vs.pbs_blup.full_chr-10_snpsqc_5.batches.csv
    Untracked:  data/ici.vs.pbs_blup.full_chr-10_snpsqc_5.batches_mis.csv
    Untracked:  data/ici.vs.pbs_blup.full_chr-10_snpsqc_dis_no-x_updated_5.batches.csv
    Untracked:  data/ici.vs.pbs_blup.full_chr-10_snpsqc_dis_no-x_updated_5.batches_mis.csv
    Untracked:  data/ici.vs.pbs_blup.full_chr-11_lod.drop-1.5_5.batches.csv
    Untracked:  data/ici.vs.pbs_blup.full_chr-11_snpsqc_5.batches.csv
    Untracked:  data/ici.vs.pbs_blup.full_chr-11_snpsqc_5.batches_mis.csv
    Untracked:  data/ici.vs.pbs_blup.full_chr-11_snpsqc_dis_no-x_updated_5.batches.csv
    Untracked:  data/ici.vs.pbs_blup.full_chr-11_snpsqc_dis_no-x_updated_5.batches_mis.csv
    Untracked:  data/ici.vs.pbs_blup.full_chr-12_lod.drop-1.5_5.batches.csv
    Untracked:  data/ici.vs.pbs_blup.full_chr-12_snpsqc_5.batches.csv
    Untracked:  data/ici.vs.pbs_blup.full_chr-12_snpsqc_5.batches_mis.csv
    Untracked:  data/ici.vs.pbs_blup.full_chr-12_snpsqc_dis_no-x_updated_5.batches.csv
    Untracked:  data/ici.vs.pbs_blup.full_chr-12_snpsqc_dis_no-x_updated_5.batches_mis.csv
    Untracked:  data/ici.vs.pbs_blup.full_chr-13_lod.drop-1.5_5.batches.csv
    Untracked:  data/ici.vs.pbs_blup.full_chr-13_snpsqc_5.batches.csv
    Untracked:  data/ici.vs.pbs_blup.full_chr-13_snpsqc_5.batches_mis.csv
    Untracked:  data/ici.vs.pbs_blup.full_chr-13_snpsqc_dis_no-x_updated_5.batches.csv
    Untracked:  data/ici.vs.pbs_blup.full_chr-13_snpsqc_dis_no-x_updated_5.batches_mis.csv
    Untracked:  data/ici.vs.pbs_blup.full_chr-14_lod.drop-1.5_5.batches.csv
    Untracked:  data/ici.vs.pbs_blup.full_chr-14_snpsqc_5.batches.csv
    Untracked:  data/ici.vs.pbs_blup.full_chr-14_snpsqc_5.batches_mis.csv
    Untracked:  data/ici.vs.pbs_blup.full_chr-14_snpsqc_dis_no-x_updated_5.batches.csv
    Untracked:  data/ici.vs.pbs_blup.full_chr-14_snpsqc_dis_no-x_updated_5.batches_mis.csv
    Untracked:  data/ici.vs.pbs_blup.full_chr-15_lod.drop-1.5_5.batches.csv
    Untracked:  data/ici.vs.pbs_blup.full_chr-15_snpsqc_5.batches.csv
    Untracked:  data/ici.vs.pbs_blup.full_chr-15_snpsqc_5.batches_mis.csv
    Untracked:  data/ici.vs.pbs_blup.full_chr-15_snpsqc_dis_no-x_updated_5.batches.csv
    Untracked:  data/ici.vs.pbs_blup.full_chr-15_snpsqc_dis_no-x_updated_5.batches_mis.csv
    Untracked:  data/ici.vs.pbs_blup.full_chr-16_lod.drop-1.5_5.batches.csv
    Untracked:  data/ici.vs.pbs_blup.full_chr-16_snpsqc_5.batches.csv
    Untracked:  data/ici.vs.pbs_blup.full_chr-16_snpsqc_5.batches_mis.csv
    Untracked:  data/ici.vs.pbs_blup.full_chr-16_snpsqc_dis_no-x_updated_5.batches.csv
    Untracked:  data/ici.vs.pbs_blup.full_chr-16_snpsqc_dis_no-x_updated_5.batches_mis.csv
    Untracked:  data/ici.vs.pbs_blup.full_chr-17_lod.drop-1.5_5.batches.csv
    Untracked:  data/ici.vs.pbs_blup.full_chr-17_snpsqc_5.batches.csv
    Untracked:  data/ici.vs.pbs_blup.full_chr-17_snpsqc_5.batches_mis.csv
    Untracked:  data/ici.vs.pbs_blup.full_chr-17_snpsqc_dis_no-x_updated_5.batches.csv
    Untracked:  data/ici.vs.pbs_blup.full_chr-17_snpsqc_dis_no-x_updated_5.batches_mis.csv
    Untracked:  data/ici.vs.pbs_blup.full_chr-18_lod.drop-1.5_5.batches.csv
    Untracked:  data/ici.vs.pbs_blup.full_chr-18_snpsqc_5.batches.csv
    Untracked:  data/ici.vs.pbs_blup.full_chr-18_snpsqc_5.batches_mis.csv
    Untracked:  data/ici.vs.pbs_blup.full_chr-18_snpsqc_dis_no-x_updated_5.batches.csv
    Untracked:  data/ici.vs.pbs_blup.full_chr-18_snpsqc_dis_no-x_updated_5.batches_mis.csv
    Untracked:  data/ici.vs.pbs_blup.full_chr-19_lod.drop-1.5_5.batches.csv
    Untracked:  data/ici.vs.pbs_blup.full_chr-19_snpsqc_5.batches.csv
    Untracked:  data/ici.vs.pbs_blup.full_chr-19_snpsqc_5.batches_mis.csv
    Untracked:  data/ici.vs.pbs_blup.full_chr-19_snpsqc_dis_no-x_updated_5.batches.csv
    Untracked:  data/ici.vs.pbs_blup.full_chr-19_snpsqc_dis_no-x_updated_5.batches_mis.csv
    Untracked:  data/ici.vs.pbs_blup.full_chr-1_lod.drop-1.5_5.batches.csv
    Untracked:  data/ici.vs.pbs_blup.full_chr-1_snpsqc_5.batches.csv
    Untracked:  data/ici.vs.pbs_blup.full_chr-1_snpsqc_5.batches_mis.csv
    Untracked:  data/ici.vs.pbs_blup.full_chr-1_snpsqc_dis_no-x_updated_5.batches.csv
    Untracked:  data/ici.vs.pbs_blup.full_chr-1_snpsqc_dis_no-x_updated_5.batches_mis.csv
    Untracked:  data/ici.vs.pbs_blup.full_chr-2_lod.drop-1.5_5.batches.csv
    Untracked:  data/ici.vs.pbs_blup.full_chr-2_snpsqc_5.batches.csv
    Untracked:  data/ici.vs.pbs_blup.full_chr-2_snpsqc_5.batches_mis.csv
    Untracked:  data/ici.vs.pbs_blup.full_chr-2_snpsqc_dis_no-x_updated_5.batches.csv
    Untracked:  data/ici.vs.pbs_blup.full_chr-2_snpsqc_dis_no-x_updated_5.batches_mis.csv
    Untracked:  data/ici.vs.pbs_blup.full_chr-3_lod.drop-1.5_5.batches.csv
    Untracked:  data/ici.vs.pbs_blup.full_chr-3_snpsqc_5.batches.csv
    Untracked:  data/ici.vs.pbs_blup.full_chr-3_snpsqc_5.batches_mis.csv
    Untracked:  data/ici.vs.pbs_blup.full_chr-3_snpsqc_dis_no-x_updated_5.batches.csv
    Untracked:  data/ici.vs.pbs_blup.full_chr-3_snpsqc_dis_no-x_updated_5.batches_mis.csv
    Untracked:  data/ici.vs.pbs_blup.full_chr-4_lod.drop-1.5_5.batches.csv
    Untracked:  data/ici.vs.pbs_blup.full_chr-4_snpsqc_5.batches.csv
    Untracked:  data/ici.vs.pbs_blup.full_chr-4_snpsqc_5.batches_mis.csv
    Untracked:  data/ici.vs.pbs_blup.full_chr-4_snpsqc_dis_no-x_updated_5.batches.csv
    Untracked:  data/ici.vs.pbs_blup.full_chr-4_snpsqc_dis_no-x_updated_5.batches_mis.csv
    Untracked:  data/ici.vs.pbs_blup.full_chr-5_lod.drop-1.5_5.batches.csv
    Untracked:  data/ici.vs.pbs_blup.full_chr-5_snpsqc_5.batches.csv
    Untracked:  data/ici.vs.pbs_blup.full_chr-5_snpsqc_5.batches_mis.csv
    Untracked:  data/ici.vs.pbs_blup.full_chr-5_snpsqc_dis_no-x_updated_5.batches.csv
    Untracked:  data/ici.vs.pbs_blup.full_chr-5_snpsqc_dis_no-x_updated_5.batches_mis.csv
    Untracked:  data/ici.vs.pbs_blup.full_chr-6_lod.drop-1.5_5.batches.csv
    Untracked:  data/ici.vs.pbs_blup.full_chr-6_snpsqc_5.batches.csv
    Untracked:  data/ici.vs.pbs_blup.full_chr-6_snpsqc_5.batches_mis.csv
    Untracked:  data/ici.vs.pbs_blup.full_chr-6_snpsqc_dis_no-x_updated_5.batches.csv
    Untracked:  data/ici.vs.pbs_blup.full_chr-6_snpsqc_dis_no-x_updated_5.batches_mis.csv
    Untracked:  data/ici.vs.pbs_blup.full_chr-7_lod.drop-1.5_5.batches.csv
    Untracked:  data/ici.vs.pbs_blup.full_chr-7_snpsqc_5.batches.csv
    Untracked:  data/ici.vs.pbs_blup.full_chr-7_snpsqc_5.batches_mis.csv
    Untracked:  data/ici.vs.pbs_blup.full_chr-7_snpsqc_dis_no-x_updated_5.batches.csv
    Untracked:  data/ici.vs.pbs_blup.full_chr-7_snpsqc_dis_no-x_updated_5.batches_mis.csv
    Untracked:  data/ici.vs.pbs_blup.full_chr-8_lod.drop-1.5_5.batches.csv
    Untracked:  data/ici.vs.pbs_blup.full_chr-8_snpsqc_5.batches.csv
    Untracked:  data/ici.vs.pbs_blup.full_chr-8_snpsqc_5.batches_mis.csv
    Untracked:  data/ici.vs.pbs_blup.full_chr-8_snpsqc_dis_no-x_updated_5.batches.csv
    Untracked:  data/ici.vs.pbs_blup.full_chr-8_snpsqc_dis_no-x_updated_5.batches_mis.csv
    Untracked:  data/ici.vs.pbs_blup.full_chr-9_lod.drop-1.5_5.batches.csv
    Untracked:  data/ici.vs.pbs_blup.full_chr-9_snpsqc_5.batches.csv
    Untracked:  data/ici.vs.pbs_blup.full_chr-9_snpsqc_5.batches_mis.csv
    Untracked:  data/ici.vs.pbs_blup.full_chr-9_snpsqc_dis_no-x_updated_5.batches.csv
    Untracked:  data/ici.vs.pbs_blup.full_chr-9_snpsqc_dis_no-x_updated_5.batches_mis.csv
    Untracked:  data/ici.vs.pbs_blup.full_chr-X_lod.drop-1.5_5.batches.csv
    Untracked:  data/ici.vs.pbs_blup.full_chr-X_snpsqc_5.batches.csv
    Untracked:  data/ici.vs.pbs_blup.full_chr-X_snpsqc_5.batches_mis.csv
    Untracked:  data/ici.vs.pbs_blup.full_chr-X_snpsqc_dis_no-x_updated_5.batches.csv
    Untracked:  data/ici.vs.pbs_blup.full_chr-X_snpsqc_dis_no-x_updated_5.batches_mis.csv
    Untracked:  data/ici.vs.pbs_blup.qc_chr-10_5.batches.csv
    Untracked:  data/ici.vs.pbs_blup.qc_chr-10_5.batches_mis.csv
    Untracked:  data/ici.vs.pbs_blup.qc_chr-10_lod.drop-1.5_5.batches.csv
    Untracked:  data/ici.vs.pbs_blup.qc_chr-10_lod.drop-1.5_5.batches_mis.csv
    Untracked:  data/ici.vs.pbs_blup.qc_chr-10_snpsqc_5.batches.csv
    Untracked:  data/ici.vs.pbs_blup.qc_chr-10_snpsqc_5.batches_mis.csv
    Untracked:  data/ici.vs.pbs_blup.qc_chr-10_snpsqc_dis_no-x_updated_5.batches.csv
    Untracked:  data/ici.vs.pbs_blup.qc_chr-10_snpsqc_dis_no-x_updated_5.batches_mis.csv
    Untracked:  data/ici.vs.pbs_blup.qc_chr-11_5.batches.csv
    Untracked:  data/ici.vs.pbs_blup.qc_chr-11_5.batches_mis.csv
    Untracked:  data/ici.vs.pbs_blup.qc_chr-11_lod.drop-1.5_5.batches.csv
    Untracked:  data/ici.vs.pbs_blup.qc_chr-11_lod.drop-1.5_5.batches_mis.csv
    Untracked:  data/ici.vs.pbs_blup.qc_chr-11_snpsqc_5.batches.csv
    Untracked:  data/ici.vs.pbs_blup.qc_chr-11_snpsqc_5.batches_mis.csv
    Untracked:  data/ici.vs.pbs_blup.qc_chr-11_snpsqc_dis_no-x_updated_5.batches.csv
    Untracked:  data/ici.vs.pbs_blup.qc_chr-11_snpsqc_dis_no-x_updated_5.batches_mis.csv
    Untracked:  data/ici.vs.pbs_blup.qc_chr-12_5.batches.csv
    Untracked:  data/ici.vs.pbs_blup.qc_chr-12_5.batches_mis.csv
    Untracked:  data/ici.vs.pbs_blup.qc_chr-12_lod.drop-1.5_5.batches.csv
    Untracked:  data/ici.vs.pbs_blup.qc_chr-12_lod.drop-1.5_5.batches_mis.csv
    Untracked:  data/ici.vs.pbs_blup.qc_chr-12_snpsqc_5.batches.csv
    Untracked:  data/ici.vs.pbs_blup.qc_chr-12_snpsqc_5.batches_mis.csv
    Untracked:  data/ici.vs.pbs_blup.qc_chr-12_snpsqc_dis_no-x_updated_5.batches.csv
    Untracked:  data/ici.vs.pbs_blup.qc_chr-12_snpsqc_dis_no-x_updated_5.batches_mis.csv
    Untracked:  data/ici.vs.pbs_blup.qc_chr-13_5.batches.csv
    Untracked:  data/ici.vs.pbs_blup.qc_chr-13_5.batches_mis.csv
    Untracked:  data/ici.vs.pbs_blup.qc_chr-13_lod.drop-1.5_5.batches.csv
    Untracked:  data/ici.vs.pbs_blup.qc_chr-13_lod.drop-1.5_5.batches_mis.csv
    Untracked:  data/ici.vs.pbs_blup.qc_chr-13_snpsqc_5.batches.csv
    Untracked:  data/ici.vs.pbs_blup.qc_chr-13_snpsqc_5.batches_mis.csv
    Untracked:  data/ici.vs.pbs_blup.qc_chr-13_snpsqc_dis_no-x_updated_5.batches.csv
    Untracked:  data/ici.vs.pbs_blup.qc_chr-13_snpsqc_dis_no-x_updated_5.batches_mis.csv
    Untracked:  data/ici.vs.pbs_blup.qc_chr-14_5.batches.csv
    Untracked:  data/ici.vs.pbs_blup.qc_chr-14_5.batches_mis.csv
    Untracked:  data/ici.vs.pbs_blup.qc_chr-14_lod.drop-1.5_5.batches.csv
    Untracked:  data/ici.vs.pbs_blup.qc_chr-14_snpsqc_5.batches.csv
    Untracked:  data/ici.vs.pbs_blup.qc_chr-14_snpsqc_5.batches_mis.csv
    Untracked:  data/ici.vs.pbs_blup.qc_chr-14_snpsqc_dis_no-x_updated_5.batches.csv
    Untracked:  data/ici.vs.pbs_blup.qc_chr-14_snpsqc_dis_no-x_updated_5.batches_mis.csv
    Untracked:  data/ici.vs.pbs_blup.qc_chr-15_5.batches.csv
    Untracked:  data/ici.vs.pbs_blup.qc_chr-15_5.batches_mis.csv
    Untracked:  data/ici.vs.pbs_blup.qc_chr-15_lod.drop-1.5_5.batches.csv
    Untracked:  data/ici.vs.pbs_blup.qc_chr-15_snpsqc_5.batches.csv
    Untracked:  data/ici.vs.pbs_blup.qc_chr-15_snpsqc_5.batches_mis.csv
    Untracked:  data/ici.vs.pbs_blup.qc_chr-16_5.batches.csv
    Untracked:  data/ici.vs.pbs_blup.qc_chr-16_5.batches_mis.csv
    Untracked:  data/ici.vs.pbs_blup.qc_chr-16_lod.drop-1.5_5.batches.csv
    Untracked:  data/ici.vs.pbs_blup.qc_chr-16_snpsqc_5.batches.csv
    Untracked:  data/ici.vs.pbs_blup.qc_chr-16_snpsqc_5.batches_mis.csv
    Untracked:  data/ici.vs.pbs_blup.qc_chr-16_snpsqc_dis_no-x_updated_5.batches.csv
    Untracked:  data/ici.vs.pbs_blup.qc_chr-16_snpsqc_dis_no-x_updated_5.batches_mis.csv
    Untracked:  data/ici.vs.pbs_blup.qc_chr-17_5.batches.csv
    Untracked:  data/ici.vs.pbs_blup.qc_chr-17_5.batches_mis.csv
    Untracked:  data/ici.vs.pbs_blup.qc_chr-17_lod.drop-1.5_5.batches.csv
    Untracked:  data/ici.vs.pbs_blup.qc_chr-17_snpsqc_5.batches.csv
    Untracked:  data/ici.vs.pbs_blup.qc_chr-17_snpsqc_5.batches_mis.csv
    Untracked:  data/ici.vs.pbs_blup.qc_chr-18_5.batches.csv
    Untracked:  data/ici.vs.pbs_blup.qc_chr-18_5.batches_mis.csv
    Untracked:  data/ici.vs.pbs_blup.qc_chr-18_lod.drop-1.5_5.batches.csv
    Untracked:  data/ici.vs.pbs_blup.qc_chr-18_snpsqc_5.batches.csv
    Untracked:  data/ici.vs.pbs_blup.qc_chr-18_snpsqc_5.batches_mis.csv
    Untracked:  data/ici.vs.pbs_blup.qc_chr-18_snpsqc_dis_no-x_updated_5.batches.csv
    Untracked:  data/ici.vs.pbs_blup.qc_chr-18_snpsqc_dis_no-x_updated_5.batches_mis.csv
    Untracked:  data/ici.vs.pbs_blup.qc_chr-19_5.batches.csv
    Untracked:  data/ici.vs.pbs_blup.qc_chr-19_5.batches_mis.csv
    Untracked:  data/ici.vs.pbs_blup.qc_chr-19_lod.drop-1.5_5.batches.csv
    Untracked:  data/ici.vs.pbs_blup.qc_chr-19_snpsqc_5.batches.csv
    Untracked:  data/ici.vs.pbs_blup.qc_chr-19_snpsqc_5.batches_mis.csv
    Untracked:  data/ici.vs.pbs_blup.qc_chr-19_snpsqc_dis_no-x_updated_5.batches.csv
    Untracked:  data/ici.vs.pbs_blup.qc_chr-19_snpsqc_dis_no-x_updated_5.batches_mis.csv
    Untracked:  data/ici.vs.pbs_blup.qc_chr-1_5.batches.csv
    Untracked:  data/ici.vs.pbs_blup.qc_chr-1_5.batches_mis.csv
    Untracked:  data/ici.vs.pbs_blup.qc_chr-1_lod.drop-1.5_5.batches.csv
    Untracked:  data/ici.vs.pbs_blup.qc_chr-1_lod.drop-1.5_5.batches_mis.csv
    Untracked:  data/ici.vs.pbs_blup.qc_chr-1_snpsqc_5.batches.csv
    Untracked:  data/ici.vs.pbs_blup.qc_chr-1_snpsqc_5.batches_mis.csv
    Untracked:  data/ici.vs.pbs_blup.qc_chr-1_snpsqc_dis_no-x_updated_5.batches.csv
    Untracked:  data/ici.vs.pbs_blup.qc_chr-1_snpsqc_dis_no-x_updated_5.batches_mis.csv
    Untracked:  data/ici.vs.pbs_blup.qc_chr-2_5.batches.csv
    Untracked:  data/ici.vs.pbs_blup.qc_chr-2_5.batches_mis.csv
    Untracked:  data/ici.vs.pbs_blup.qc_chr-2_lod.drop-1.5_5.batches.csv
    Untracked:  data/ici.vs.pbs_blup.qc_chr-2_lod.drop-1.5_5.batches_mis.csv
    Untracked:  data/ici.vs.pbs_blup.qc_chr-2_snpsqc_5.batches.csv
    Untracked:  data/ici.vs.pbs_blup.qc_chr-2_snpsqc_5.batches_mis.csv
    Untracked:  data/ici.vs.pbs_blup.qc_chr-2_snpsqc_dis_no-x_updated_5.batches.csv
    Untracked:  data/ici.vs.pbs_blup.qc_chr-2_snpsqc_dis_no-x_updated_5.batches_mis.csv
    Untracked:  data/ici.vs.pbs_blup.qc_chr-3_5.batches.csv
    Untracked:  data/ici.vs.pbs_blup.qc_chr-3_5.batches_mis.csv
    Untracked:  data/ici.vs.pbs_blup.qc_chr-3_lod.drop-1.5_5.batches.csv
    Untracked:  data/ici.vs.pbs_blup.qc_chr-3_lod.drop-1.5_5.batches_mis.csv
    Untracked:  data/ici.vs.pbs_blup.qc_chr-3_snpsqc_5.batches.csv
    Untracked:  data/ici.vs.pbs_blup.qc_chr-3_snpsqc_5.batches_mis.csv
    Untracked:  data/ici.vs.pbs_blup.qc_chr-3_snpsqc_dis_no-x_updated_5.batches.csv
    Untracked:  data/ici.vs.pbs_blup.qc_chr-3_snpsqc_dis_no-x_updated_5.batches_mis.csv
    Untracked:  data/ici.vs.pbs_blup.qc_chr-4_5.batches.csv
    Untracked:  data/ici.vs.pbs_blup.qc_chr-4_5.batches_mis.csv
    Untracked:  data/ici.vs.pbs_blup.qc_chr-4_lod.drop-1.5_5.batches.csv
    Untracked:  data/ici.vs.pbs_blup.qc_chr-4_lod.drop-1.5_5.batches_mis.csv
    Untracked:  data/ici.vs.pbs_blup.qc_chr-4_snpsqc_5.batches.csv
    Untracked:  data/ici.vs.pbs_blup.qc_chr-4_snpsqc_5.batches_mis.csv
    Untracked:  data/ici.vs.pbs_blup.qc_chr-4_snpsqc_dis_no-x_updated_5.batches.csv
    Untracked:  data/ici.vs.pbs_blup.qc_chr-4_snpsqc_dis_no-x_updated_5.batches_mis.csv
    Untracked:  data/ici.vs.pbs_blup.qc_chr-5_5.batches.csv
    Untracked:  data/ici.vs.pbs_blup.qc_chr-5_5.batches_mis.csv
    Untracked:  data/ici.vs.pbs_blup.qc_chr-5_lod.drop-1.5_5.batches.csv
    Untracked:  data/ici.vs.pbs_blup.qc_chr-5_lod.drop-1.5_5.batches_mis.csv
    Untracked:  data/ici.vs.pbs_blup.qc_chr-5_snpsqc_5.batches.csv
    Untracked:  data/ici.vs.pbs_blup.qc_chr-5_snpsqc_5.batches_mis.csv
    Untracked:  data/ici.vs.pbs_blup.qc_chr-5_snpsqc_dis_no-x_updated_5.batches.csv
    Untracked:  data/ici.vs.pbs_blup.qc_chr-5_snpsqc_dis_no-x_updated_5.batches_mis.csv
    Untracked:  data/ici.vs.pbs_blup.qc_chr-6_5.batches.csv
    Untracked:  data/ici.vs.pbs_blup.qc_chr-6_5.batches_mis.csv
    Untracked:  data/ici.vs.pbs_blup.qc_chr-6_lod.drop-1.5_5.batches.csv
    Untracked:  data/ici.vs.pbs_blup.qc_chr-6_lod.drop-1.5_5.batches_mis.csv
    Untracked:  data/ici.vs.pbs_blup.qc_chr-6_snpsqc_5.batches.csv
    Untracked:  data/ici.vs.pbs_blup.qc_chr-6_snpsqc_5.batches_mis.csv
    Untracked:  data/ici.vs.pbs_blup.qc_chr-6_snpsqc_dis_no-x_updated_5.batches.csv
    Untracked:  data/ici.vs.pbs_blup.qc_chr-6_snpsqc_dis_no-x_updated_5.batches_mis.csv
    Untracked:  data/ici.vs.pbs_blup.qc_chr-7_5.batches.csv
    Untracked:  data/ici.vs.pbs_blup.qc_chr-7_5.batches_mis.csv
    Untracked:  data/ici.vs.pbs_blup.qc_chr-7_lod.drop-1.5_5.batches.csv
    Untracked:  data/ici.vs.pbs_blup.qc_chr-7_lod.drop-1.5_5.batches_mis.csv
    Untracked:  data/ici.vs.pbs_blup.qc_chr-7_snpsqc_5.batches.csv
    Untracked:  data/ici.vs.pbs_blup.qc_chr-7_snpsqc_5.batches_mis.csv
    Untracked:  data/ici.vs.pbs_blup.qc_chr-7_snpsqc_dis_no-x_updated_5.batches.csv
    Untracked:  data/ici.vs.pbs_blup.qc_chr-7_snpsqc_dis_no-x_updated_5.batches_mis.csv
    Untracked:  data/ici.vs.pbs_blup.qc_chr-8_5.batches.csv
    Untracked:  data/ici.vs.pbs_blup.qc_chr-8_5.batches_mis.csv
    Untracked:  data/ici.vs.pbs_blup.qc_chr-8_lod.drop-1.5_5.batches.csv
    Untracked:  data/ici.vs.pbs_blup.qc_chr-8_lod.drop-1.5_5.batches_mis.csv
    Untracked:  data/ici.vs.pbs_blup.qc_chr-8_snpsqc_5.batches.csv
    Untracked:  data/ici.vs.pbs_blup.qc_chr-8_snpsqc_5.batches_mis.csv
    Untracked:  data/ici.vs.pbs_blup.qc_chr-8_snpsqc_dis_no-x_updated_5.batches.csv
    Untracked:  data/ici.vs.pbs_blup.qc_chr-8_snpsqc_dis_no-x_updated_5.batches_mis.csv
    Untracked:  data/ici.vs.pbs_blup.qc_chr-9_5.batches.csv
    Untracked:  data/ici.vs.pbs_blup.qc_chr-9_5.batches_mis.csv
    Untracked:  data/ici.vs.pbs_blup.qc_chr-9_lod.drop-1.5_5.batches.csv
    Untracked:  data/ici.vs.pbs_blup.qc_chr-9_lod.drop-1.5_5.batches_mis.csv
    Untracked:  data/ici.vs.pbs_blup.qc_chr-9_snpsqc_5.batches.csv
    Untracked:  data/ici.vs.pbs_blup.qc_chr-9_snpsqc_5.batches_mis.csv
    Untracked:  data/ici.vs.pbs_blup.qc_chr-9_snpsqc_dis_no-x_updated_5.batches.csv
    Untracked:  data/ici.vs.pbs_blup.qc_chr-9_snpsqc_dis_no-x_updated_5.batches_mis.csv
    Untracked:  data/ici.vs.pbs_blup.qc_chr-X_5.batches.csv
    Untracked:  data/ici.vs.pbs_blup.qc_chr-X_5.batches_mis.csv
    Untracked:  data/ici.vs.pbs_blup.qc_chr-X_lod.drop-1.5_5.batches.csv
    Untracked:  data/ici.vs.pbs_blup.qc_chr-X_snpsqc_5.batches.csv
    Untracked:  data/ici.vs.pbs_blup.qc_chr-X_snpsqc_5.batches_mis.csv
    Untracked:  data/ici.vs.pbs_blup.qc_chr-X_snpsqc_dis_no-x_updated_5.batches.csv
    Untracked:  data/ici.vs.pbs_blup.qc_chr-X_snpsqc_dis_no-x_updated_5.batches_mis.csv
    Untracked:  data/ici.vs.pbs_blup_chr-10_lod.drop-1.5_5.batches.csv
    Untracked:  data/ici.vs.pbs_blup_chr-10_lod.drop-1.5_snpsqc_5.batches.csv
    Untracked:  data/ici.vs.pbs_blup_chr-10_lod.drop-1.5_snpsqc_5.batches_mis.csv
    Untracked:  data/ici.vs.pbs_blup_chr-10_lod.drop-1.5_snpsqc_dis_no-x_updated_5.batches.csv
    Untracked:  data/ici.vs.pbs_blup_chr-10_lod.drop-1.5_snpsqc_dis_no-x_updated_5.batches_mis.csv
    Untracked:  data/ici.vs.pbs_blup_chr-11_lod.drop-1.5_5.batches.csv
    Untracked:  data/ici.vs.pbs_blup_chr-11_lod.drop-1.5_snpsqc_5.batches.csv
    Untracked:  data/ici.vs.pbs_blup_chr-11_lod.drop-1.5_snpsqc_5.batches_mis.csv
    Untracked:  data/ici.vs.pbs_blup_chr-11_lod.drop-1.5_snpsqc_dis_no-x_updated_5.batches.csv
    Untracked:  data/ici.vs.pbs_blup_chr-11_lod.drop-1.5_snpsqc_dis_no-x_updated_5.batches_mis.csv
    Untracked:  data/ici.vs.pbs_blup_chr-12_lod.drop-1.5_5.batches.csv
    Untracked:  data/ici.vs.pbs_blup_chr-12_lod.drop-1.5_snpsqc_5.batches.csv
    Untracked:  data/ici.vs.pbs_blup_chr-12_lod.drop-1.5_snpsqc_5.batches_mis.csv
    Untracked:  data/ici.vs.pbs_blup_chr-12_lod.drop-1.5_snpsqc_dis_no-x_updated_5.batches.csv
    Untracked:  data/ici.vs.pbs_blup_chr-12_lod.drop-1.5_snpsqc_dis_no-x_updated_5.batches_mis.csv
    Untracked:  data/ici.vs.pbs_blup_chr-13_lod.drop-1.5_5.batches.csv
    Untracked:  data/ici.vs.pbs_blup_chr-13_lod.drop-1.5_snpsqc_5.batches.csv
    Untracked:  data/ici.vs.pbs_blup_chr-13_lod.drop-1.5_snpsqc_5.batches_mis.csv
    Untracked:  data/ici.vs.pbs_blup_chr-13_lod.drop-1.5_snpsqc_dis_no-x_updated_5.batches.csv
    Untracked:  data/ici.vs.pbs_blup_chr-13_lod.drop-1.5_snpsqc_dis_no-x_updated_5.batches_mis.csv
    Untracked:  data/ici.vs.pbs_blup_chr-14_lod.drop-1.5_5.batches.csv
    Untracked:  data/ici.vs.pbs_blup_chr-14_lod.drop-1.5_snpsqc_5.batches.csv
    Untracked:  data/ici.vs.pbs_blup_chr-14_lod.drop-1.5_snpsqc_5.batches_mis.csv
    Untracked:  data/ici.vs.pbs_blup_chr-14_lod.drop-1.5_snpsqc_dis_no-x_updated_5.batches.csv
    Untracked:  data/ici.vs.pbs_blup_chr-14_lod.drop-1.5_snpsqc_dis_no-x_updated_5.batches_mis.csv
    Untracked:  data/ici.vs.pbs_blup_chr-15_lod.drop-1.5_5.batches.csv
    Untracked:  data/ici.vs.pbs_blup_chr-15_lod.drop-1.5_snpsqc_5.batches.csv
    Untracked:  data/ici.vs.pbs_blup_chr-15_lod.drop-1.5_snpsqc_5.batches_mis.csv
    Untracked:  data/ici.vs.pbs_blup_chr-16_lod.drop-1.5_5.batches.csv
    Untracked:  data/ici.vs.pbs_blup_chr-16_lod.drop-1.5_snpsqc_5.batches.csv
    Untracked:  data/ici.vs.pbs_blup_chr-16_lod.drop-1.5_snpsqc_5.batches_mis.csv
    Untracked:  data/ici.vs.pbs_blup_chr-16_lod.drop-1.5_snpsqc_dis_no-x_updated_5.batches.csv
    Untracked:  data/ici.vs.pbs_blup_chr-16_lod.drop-1.5_snpsqc_dis_no-x_updated_5.batches_mis.csv
    Untracked:  data/ici.vs.pbs_blup_chr-17_lod.drop-1.5_5.batches.csv
    Untracked:  data/ici.vs.pbs_blup_chr-17_lod.drop-1.5_snpsqc_5.batches.csv
    Untracked:  data/ici.vs.pbs_blup_chr-17_lod.drop-1.5_snpsqc_5.batches_mis.csv
    Untracked:  data/ici.vs.pbs_blup_chr-18_lod.drop-1.5_5.batches.csv
    Untracked:  data/ici.vs.pbs_blup_chr-18_lod.drop-1.5_snpsqc_5.batches.csv
    Untracked:  data/ici.vs.pbs_blup_chr-18_lod.drop-1.5_snpsqc_5.batches_mis.csv
    Untracked:  data/ici.vs.pbs_blup_chr-18_lod.drop-1.5_snpsqc_dis_no-x_updated_5.batches.csv
    Untracked:  data/ici.vs.pbs_blup_chr-18_lod.drop-1.5_snpsqc_dis_no-x_updated_5.batches_mis.csv
    Untracked:  data/ici.vs.pbs_blup_chr-19_lod.drop-1.5_5.batches.csv
    Untracked:  data/ici.vs.pbs_blup_chr-19_lod.drop-1.5_snpsqc_5.batches.csv
    Untracked:  data/ici.vs.pbs_blup_chr-19_lod.drop-1.5_snpsqc_5.batches_mis.csv
    Untracked:  data/ici.vs.pbs_blup_chr-19_lod.drop-1.5_snpsqc_dis_no-x_updated_5.batches.csv
    Untracked:  data/ici.vs.pbs_blup_chr-19_lod.drop-1.5_snpsqc_dis_no-x_updated_5.batches_mis.csv
    Untracked:  data/ici.vs.pbs_blup_chr-1_lod.drop-1.5_5.batches.csv
    Untracked:  data/ici.vs.pbs_blup_chr-1_lod.drop-1.5_snpsqc_5.batches.csv
    Untracked:  data/ici.vs.pbs_blup_chr-1_lod.drop-1.5_snpsqc_5.batches_mis.csv
    Untracked:  data/ici.vs.pbs_blup_chr-1_lod.drop-1.5_snpsqc_dis_no-x_updated_5.batches.csv
    Untracked:  data/ici.vs.pbs_blup_chr-1_lod.drop-1.5_snpsqc_dis_no-x_updated_5.batches_mis.csv
    Untracked:  data/ici.vs.pbs_blup_chr-2_lod.drop-1.5_5.batches.csv
    Untracked:  data/ici.vs.pbs_blup_chr-2_lod.drop-1.5_snpsqc_5.batches.csv
    Untracked:  data/ici.vs.pbs_blup_chr-2_lod.drop-1.5_snpsqc_5.batches_mis.csv
    Untracked:  data/ici.vs.pbs_blup_chr-2_lod.drop-1.5_snpsqc_dis_no-x_updated_5.batches.csv
    Untracked:  data/ici.vs.pbs_blup_chr-2_lod.drop-1.5_snpsqc_dis_no-x_updated_5.batches_mis.csv
    Untracked:  data/ici.vs.pbs_blup_chr-3_lod.drop-1.5_5.batches.csv
    Untracked:  data/ici.vs.pbs_blup_chr-3_lod.drop-1.5_snpsqc_5.batches.csv
    Untracked:  data/ici.vs.pbs_blup_chr-3_lod.drop-1.5_snpsqc_5.batches_mis.csv
    Untracked:  data/ici.vs.pbs_blup_chr-3_lod.drop-1.5_snpsqc_dis_no-x_updated_5.batches.csv
    Untracked:  data/ici.vs.pbs_blup_chr-3_lod.drop-1.5_snpsqc_dis_no-x_updated_5.batches_mis.csv
    Untracked:  data/ici.vs.pbs_blup_chr-4_lod.drop-1.5_5.batches.csv
    Untracked:  data/ici.vs.pbs_blup_chr-4_lod.drop-1.5_snpsqc_5.batches.csv
    Untracked:  data/ici.vs.pbs_blup_chr-4_lod.drop-1.5_snpsqc_5.batches_mis.csv
    Untracked:  data/ici.vs.pbs_blup_chr-4_lod.drop-1.5_snpsqc_dis_no-x_updated_5.batches.csv
    Untracked:  data/ici.vs.pbs_blup_chr-4_lod.drop-1.5_snpsqc_dis_no-x_updated_5.batches_mis.csv
    Untracked:  data/ici.vs.pbs_blup_chr-5_lod.drop-1.5_5.batches.csv
    Untracked:  data/ici.vs.pbs_blup_chr-5_lod.drop-1.5_snpsqc_5.batches.csv
    Untracked:  data/ici.vs.pbs_blup_chr-5_lod.drop-1.5_snpsqc_5.batches_mis.csv
    Untracked:  data/ici.vs.pbs_blup_chr-5_lod.drop-1.5_snpsqc_dis_no-x_updated_5.batches.csv
    Untracked:  data/ici.vs.pbs_blup_chr-5_lod.drop-1.5_snpsqc_dis_no-x_updated_5.batches_mis.csv
    Untracked:  data/ici.vs.pbs_blup_chr-6_lod.drop-1.5_5.batches.csv
    Untracked:  data/ici.vs.pbs_blup_chr-6_lod.drop-1.5_snpsqc_5.batches.csv
    Untracked:  data/ici.vs.pbs_blup_chr-6_lod.drop-1.5_snpsqc_5.batches_mis.csv
    Untracked:  data/ici.vs.pbs_blup_chr-6_lod.drop-1.5_snpsqc_dis_no-x_updated_5.batches.csv
    Untracked:  data/ici.vs.pbs_blup_chr-6_lod.drop-1.5_snpsqc_dis_no-x_updated_5.batches_mis.csv
    Untracked:  data/ici.vs.pbs_blup_chr-7_lod.drop-1.5_5.batches.csv
    Untracked:  data/ici.vs.pbs_blup_chr-7_lod.drop-1.5_snpsqc_5.batches.csv
    Untracked:  data/ici.vs.pbs_blup_chr-7_lod.drop-1.5_snpsqc_5.batches_mis.csv
    Untracked:  data/ici.vs.pbs_blup_chr-7_lod.drop-1.5_snpsqc_dis_no-x_updated_5.batches.csv
    Untracked:  data/ici.vs.pbs_blup_chr-7_lod.drop-1.5_snpsqc_dis_no-x_updated_5.batches_mis.csv
    Untracked:  data/ici.vs.pbs_blup_chr-8_lod.drop-1.5_5.batches.csv
    Untracked:  data/ici.vs.pbs_blup_chr-8_lod.drop-1.5_snpsqc_5.batches.csv
    Untracked:  data/ici.vs.pbs_blup_chr-8_lod.drop-1.5_snpsqc_5.batches_mis.csv
    Untracked:  data/ici.vs.pbs_blup_chr-8_lod.drop-1.5_snpsqc_dis_no-x_updated_5.batches.csv
    Untracked:  data/ici.vs.pbs_blup_chr-8_lod.drop-1.5_snpsqc_dis_no-x_updated_5.batches_mis.csv
    Untracked:  data/ici.vs.pbs_blup_chr-9_lod.drop-1.5_5.batches.csv
    Untracked:  data/ici.vs.pbs_blup_chr-9_lod.drop-1.5_snpsqc_5.batches.csv
    Untracked:  data/ici.vs.pbs_blup_chr-9_lod.drop-1.5_snpsqc_5.batches_mis.csv
    Untracked:  data/ici.vs.pbs_blup_chr-9_lod.drop-1.5_snpsqc_dis_no-x_updated_5.batches.csv
    Untracked:  data/ici.vs.pbs_blup_chr-9_lod.drop-1.5_snpsqc_dis_no-x_updated_5.batches_mis.csv
    Untracked:  data/ici.vs.pbs_blup_chr-X_lod.drop-1.5_5.batches.csv
    Untracked:  data/ici.vs.pbs_blup_chr-X_lod.drop-1.5_snpsqc_5.batches.csv
    Untracked:  data/ici.vs.pbs_blup_chr-X_lod.drop-1.5_snpsqc_5.batches_mis.csv
    Untracked:  data/ici.vs.pbs_blup_chr-X_lod.drop-1.5_snpsqc_dis_no-x_updated_5.batches.csv
    Untracked:  data/ici.vs.pbs_blup_chr-X_lod.drop-1.5_snpsqc_dis_no-x_updated_5.batches_mis.csv
    Untracked:  data/ici.vs.pbs_blup_sub_chr-13_peak.marker-UNC22384241_lod.drop-1.5_snpsqc_snpsqc_5.batches_peaks.csv
    Untracked:  data/ici.vs.pbs_blup_sub_chr-18_peak.marker-UNCHS045343_lod.drop-1.5_snpsqc_5.batches_mis.csv
    Untracked:  data/ici.vs.pbs_blup_sub_chr-19_peak.marker-UNC29893228_lod.drop-1.5_snpsqc_snpsqc_5.batches_peaks.csv
    Untracked:  data/ici.vs.pbs_blup_sub_chr-19_peak.marker-UNC29990033_lod.drop-1.5_snpsqc_dis_no-x_updated_5.batches_peaks.csv
    Untracked:  data/ici.vs.pbs_blup_sub_chr-19_peak.marker-UNCHS047192_lod.drop-1.5_snpsqc_dis_no-x_updated_5.batches_mis_peaks.csv
    Untracked:  data/ici.vs.pbs_blup_sub_chr-8_peak.marker-UNCHS023353_lod.drop-1.5_snpsqc_dis_no-x_updated_5.batches_mis_peaks.csv
    Untracked:  data/ici.vs.pbs_blup_sub_chr-8_peak.marker-UNCHS023353_lod.drop-1.5_snpsqc_dis_no-x_updated_5.batches_peaks.csv
    Untracked:  data/ici.vs.pbs_blup_sub_chr-8_peak.marker-UNCHS023364_lod.drop-1.5_snpsqc_snpsqc_5.batches_peaks.csv
    Untracked:  data/ici.vs.pbs_fitqtl_peaks_additive_snpsqc_dis_no-x_updated_5.batches_mis_peaks.txt
    Untracked:  data/ici.vs.pbs_fitqtl_peaks_additive_snpsqc_dis_no-x_updated_5.batches_peaks.txt
    Untracked:  data/ici.vs.pbs_fitqtl_peaks_additive_snpsqc_snpsqc_5.batches_peaks.txt
    Untracked:  data/ici.vs.pbs_fitqtl_peaks_interacting_snpsqc_dis_no-x_updated_5.batches_mis_peaks.txt
    Untracked:  data/ici.vs.pbs_fitqtl_peaks_interacting_snpsqc_dis_no-x_updated_5.batches_peaks.txt
    Untracked:  data/ici.vs.pbs_fitqtl_peaks_interacting_snpsqc_snpsqc_5.batches_peaks.txt
    Untracked:  data/ici.vs.pbs_fitqtl_peaks_sex_additive_snpsqc_dis_no-x_updated_5.batches_mis_peaks.txt
    Untracked:  data/ici.vs.pbs_fitqtl_peaks_sex_additive_snpsqc_dis_no-x_updated_5.batches_peaks.txt
    Untracked:  data/ici.vs.pbs_fitqtl_peaks_sex_additive_snpsqc_snpsqc_5.batches_peaks.txt
    Untracked:  data/ici.vs.pbs_fitqtl_peaks_sex_interacting_snpsqc_dis_no-x_updated_5.batches_mis_peaks.txt
    Untracked:  data/ici.vs.pbs_fitqtl_peaks_sex_interacting_snpsqc_dis_no-x_updated_5.batches_peaks.txt
    Untracked:  data/ici.vs.pbs_fitqtl_peaks_sex_interacting_snpsqc_snpsqc_5.batches_peaks.txt
    Untracked:  data/ici.vs.pbs_genes_chr-13_peak.marker-UNC22384241_lod.drop-1.5_snpsqc_snpsqc_5.batches_peaks.csv
    Untracked:  data/ici.vs.pbs_genes_chr-18_peak.marker-UNCHS045343_lod.drop-1.5_snpsqc_5.batches_mis.csv
    Untracked:  data/ici.vs.pbs_genes_chr-19_peak.marker-UNC29893228_lod.drop-1.5_snpsqc_snpsqc_5.batches_peaks.csv
    Untracked:  data/ici.vs.pbs_genes_chr-19_peak.marker-UNC29990033_lod.drop-1.5_snpsqc_dis_no-x_updated_5.batches_peaks.csv
    Untracked:  data/ici.vs.pbs_genes_chr-19_peak.marker-UNCHS047192_lod.drop-1.5_snpsqc_dis_no-x_updated_5.batches_mis_peaks.csv
    Untracked:  data/ici.vs.pbs_genes_chr-8_peak.marker-UNCHS023353_lod.drop-1.5_snpsqc_dis_no-x_updated_5.batches_mis_peaks.csv
    Untracked:  data/ici.vs.pbs_genes_chr-8_peak.marker-UNCHS023353_lod.drop-1.5_snpsqc_dis_no-x_updated_5.batches_peaks.csv
    Untracked:  data/ici.vs.pbs_genes_chr-8_peak.marker-UNCHS023364_lod.drop-1.5_snpsqc_snpsqc_5.batches_peaks.csv
    Untracked:  data/ici.vs.pbs_gm_qtl_5.batches.csv
    Untracked:  data/ici.vs.pbs_gm_qtl_5.batches_mis.csv
    Untracked:  data/ici.vs.pbs_gm_qtl_snpsqc_5.batches.csv
    Untracked:  data/ici.vs.pbs_gm_qtl_snpsqc_5.batches_mis.csv
    Untracked:  data/ici.vs.pbs_gm_qtl_snpsqc_5.batches_peaks.csv
    Untracked:  data/ici.vs.pbs_gm_qtl_snpsqc_dis_no-x_updated_5.batches.csv
    Untracked:  data/ici.vs.pbs_gm_qtl_snpsqc_dis_no-x_updated_5.batches_mis.csv
    Untracked:  data/ici.vs.pbs_gm_qtl_snpsqc_dis_no-x_updated_5.batches_mis_peaks.csv
    Untracked:  data/ici.vs.pbs_gm_qtl_snpsqc_dis_no-x_updated_5.batches_peaks.csv
    Untracked:  data/ici.vs.pbs_marker.freq_low.geno.freq.removed_geno.ratio.csv
    Untracked:  data/ici.vs.pbs_marker.freq_low.geno.freq.removed_geno.ratio_5.batches.csv
    Untracked:  data/ici.vs.pbs_marker.freq_low.geno.freq.removed_geno.ratio_5.batches_mis.csv
    Untracked:  data/ici.vs.pbs_marker.freq_low.geno.freq.removed_sample.outliers.removed_geno.ratio.csv
    Untracked:  data/ici.vs.pbs_marker.freq_low.geno.freq.removed_sample.outliers.removed_geno.ratio_5.batches.csv
    Untracked:  data/ici.vs.pbs_marker.freq_low.geno.freq.removed_sample.outliers.removed_geno.ratio_5.batches_mis.csv
    Untracked:  data/ici.vs.pbs_marker.freq_low.probs.freq.removed_geno.ratio.csv
    Untracked:  data/ici.vs.pbs_marker.freq_low.probs.freq.removed_geno.ratio_5.batches.csv
    Untracked:  data/ici.vs.pbs_marker.freq_low.probs.freq.removed_geno.ratio_5.batches_mis.csv
    Untracked:  data/ici.vs.pbs_marker.freq_low.probs.freq.removed_sample.outliers.removed_geno.ratio.csv
    Untracked:  data/ici.vs.pbs_marker.freq_low.probs.freq.removed_sample.outliers.removed_geno.ratio_5.batches.csv
    Untracked:  data/ici.vs.pbs_marker.freq_low.probs.freq.removed_sample.outliers.removed_geno.ratio_5.batches_mis.csv
    Untracked:  data/ici.vs.pbs_rz.age-additive.covariates_blup_sub_chr-7_peak.marker-JAX00153527_lod.drop-1.5_5.batches.csv
    Untracked:  data/ici.vs.pbs_rz.age-additive.covariates_blup_sub_chr-7_peak.marker-JAX00153527_lod.drop-1.5_snpsqc_5.batches.csv
    Untracked:  data/ici.vs.pbs_rz.age-additive.covariates_blup_sub_chr-7_peak.marker-UNCHS020443_lod.drop-1.5_snpsqc_dis_no-x_updated_5.batches.csv
    Untracked:  data/ici.vs.pbs_rz.age-additive.covariates_blup_sub_chr-7_peak.marker-UNCHS020443_lod.drop-1.5_snpsqc_dis_no-x_updated_5.batches_mis.csv
    Untracked:  data/ici.vs.pbs_rz.age-additive.covariates_blup_sub_chr-7_peak.marker-UNCHS020486_lod.drop-1.5_snpsqc_5.batches_mis.csv
    Untracked:  data/ici.vs.pbs_rz.age-additive.covariates_genes_chr-7_peak.marker-JAX00153527_lod.drop-1.5_5.batches.csv
    Untracked:  data/ici.vs.pbs_rz.age-additive.covariates_genes_chr-7_peak.marker-JAX00153527_lod.drop-1.5_snpsqc_5.batches.csv
    Untracked:  data/ici.vs.pbs_rz.age-additive.covariates_genes_chr-7_peak.marker-UNCHS020443_lod.drop-1.5_snpsqc_dis_no-x_updated_5.batches.csv
    Untracked:  data/ici.vs.pbs_rz.age-additive.covariates_genes_chr-7_peak.marker-UNCHS020443_lod.drop-1.5_snpsqc_dis_no-x_updated_5.batches_mis.csv
    Untracked:  data/ici.vs.pbs_rz.age-additive.covariates_genes_chr-7_peak.marker-UNCHS020486_lod.drop-1.5_snpsqc_5.batches_mis.csv
    Untracked:  data/ici.vs.pbs_sample.genos_marker.freq_low.geno.freq.removed.csv
    Untracked:  data/ici.vs.pbs_sample.genos_marker.freq_low.geno.freq.removed_sample.outliers.removed.csv
    Untracked:  data/ici.vs.pbs_sample.genos_marker.freq_low.probs.freq.removed.csv
    Untracked:  data/ici.vs.pbs_sample.genos_marker.freq_low.probs.freq.removed_sample.outliers.removed.csv
    Untracked:  data/ici.vs.pbs_scanone_5.batches.Rdata
    Untracked:  data/ici.vs.pbs_scanone_5.batches_mis.Rdata
    Untracked:  data/ici.vs.pbs_scanone_snpsqc_5.batches.Rdata
    Untracked:  data/ici.vs.pbs_scanone_snpsqc_5.batches_mis.Rdata
    Untracked:  data/ici.vs.pbs_scanone_snpsqc_dis_no-x_updated_5.batches.Rdata
    Untracked:  data/ici.vs.pbs_scanone_snpsqc_dis_no-x_updated_5.batches_mis.Rdata
    Untracked:  data/percent_missing_id_3.batches.RData
    Untracked:  data/percent_missing_id_4.batches.RData
    Untracked:  data/percent_missing_id_4.batches_bc.RData
    Untracked:  data/percent_missing_id_5.batches.RData
    Untracked:  data/percent_missing_marker_4.batches.RData
    Untracked:  data/percent_missing_marker_5.batches.RData
    Untracked:  data/pheno.csv
    Untracked:  data/pheno_BC312.csv
    Untracked:  data/physical_map.csv
    Untracked:  data/physical_map_BC312.csv
    Untracked:  data/qc_info_bad_sample_3.batches.RData
    Untracked:  data/qc_info_bad_sample_4.batches.RData
    Untracked:  data/qc_info_bad_sample_4.batches_bc.RData
    Untracked:  data/qc_info_bad_sample_5.batches.RData
    Untracked:  data/remaining.markers_geno.freq.xlsx
    Untracked:  data/sample_geno.csv
    Untracked:  data/sample_geno_AHB_BC312.csv
    Untracked:  data/sample_geno_bc.csv
    Untracked:  data/sample_geno_bc_BC312.csv
    Untracked:  data/serreze_probs.rds
    Untracked:  data/serreze_probs_BC312.rds
    Untracked:  data/serreze_probs_allqc.rds
    Untracked:  data/serreze_probs_allqc_5.batches.rds
    Untracked:  data/serreze_probs_allqc_5.batches_mis.rds
    Untracked:  data/summary.cg_3.batches.RData
    Untracked:  data/summary.cg_4.batches.RData
    Untracked:  data/summary.cg_4.batches_bc.RData
    Untracked:  data/summary.cg_5.batches.RData
    Untracked:  output/Percent_missing_genotype_data_5.batches.pdf
    Untracked:  output/Percent_missing_genotype_data_per_marker_5.batches.pdf
    Untracked:  output/Proportion_matching_genotypes_before_removal_of_bad_samples_5.batches.pdf
    Untracked:  output/genotype_error_marker_5.batches.pdf
    Untracked:  output/genotype_frequency_marker_5.batches.pdf

Unstaged changes:
    Modified:   analysis/4.1.1_qtl.analysis_binary_ici.vs.eoi_snpsqc_dis_no-x_updated.Rmd
    Modified:   analysis/4.1.1_qtl.analysis_binary_ici.vs.pbs_snpsqc_dis_no-x_updated.Rmd
    Modified:   analysis/4.1.2_qtl.analysis_cont_age_ici-early.vs.pbs_pheno.corrected.cleaned_5.batches.Rmd
    Modified:   analysis/4.1.2_qtl.analysis_cont_age_ici-early.vs.pbs_pheno.corrected.cleaned_5.batches_mis.Rmd
    Modified:   analysis/4.1.2_qtl.analysis_cont_age_ici-early.vs.pbs_snpsqc_pheno.corrected.cleaned_5.batches.Rmd
    Modified:   analysis/4.1.2_qtl.analysis_cont_age_ici-early.vs.pbs_snpsqc_pheno.corrected.cleaned_5.batches_mis.Rmd
    Modified:   analysis/4.1.2_qtl.analysis_cont_age_ici-early.vs.pbs_snpsqc_pheno.corrected.cleaned_dis_no-xk_5.batches.Rmd
    Modified:   analysis/4.1.2_qtl.analysis_cont_age_ici-early.vs.pbs_snpsqc_pheno.corrected.cleaned_dis_no-xk_5.batches_mis.Rmd
    Modified:   analysis/4.1.2_qtl.analysis_cont_age_ici.vs.eoi_pheno.corrected.cleaned_5.batches.Rmd
    Modified:   analysis/4.1.2_qtl.analysis_cont_age_ici.vs.eoi_pheno.corrected.cleaned_5.batches_mis.Rmd
    Modified:   analysis/4.1.2_qtl.analysis_cont_age_ici.vs.eoi_snpsqc_pheno.corrected.cleaned_5.batches.Rmd
    Modified:   analysis/4.1.2_qtl.analysis_cont_age_ici.vs.eoi_snpsqc_pheno.corrected.cleaned_5.batches_mis.Rmd
    Modified:   analysis/4.1.2_qtl.analysis_cont_age_ici.vs.eoi_snpsqc_pheno.corrected.cleaned_dis_no-xk_5.batches.Rmd
    Modified:   analysis/4.1.2_qtl.analysis_cont_age_ici.vs.eoi_snpsqc_pheno.corrected.cleaned_dis_no-xk_5.batches_mis.Rmd
    Modified:   analysis/4.1.2_qtl.analysis_cont_age_ici.vs.pbs_pheno.corrected.cleaned_5.batches.Rmd
    Modified:   analysis/4.1.2_qtl.analysis_cont_age_ici.vs.pbs_pheno.corrected.cleaned_5.batches_mis.Rmd
    Modified:   analysis/4.1.2_qtl.analysis_cont_age_ici.vs.pbs_snpsqc_pheno.corrected.cleaned_5.batches.Rmd
    Modified:   analysis/4.1.2_qtl.analysis_cont_age_ici.vs.pbs_snpsqc_pheno.corrected.cleaned_5.batches_mis.Rmd
    Modified:   analysis/4.1.2_qtl.analysis_cont_age_ici.vs.pbs_snpsqc_pheno.corrected.cleaned_dis_no-xk_5.batches.Rmd
    Modified:   analysis/4.1.2_qtl.analysis_cont_age_ici.vs.pbs_snpsqc_pheno.corrected.cleaned_dis_no-xk_5.batches_mis.Rmd
    Modified:   analysis/4.1.2_qtl.analysis_cont_rz.age_ici-early.vs.pbs_pheno.corrected.cleaned_5.batches.Rmd
    Modified:   analysis/4.1.2_qtl.analysis_cont_rz.age_ici-early.vs.pbs_pheno.corrected.cleaned_5.batches_mis.Rmd
    Modified:   analysis/4.1.2_qtl.analysis_cont_rz.age_ici-early.vs.pbs_snpsqc_pheno.corrected.cleaned_5.batches.Rmd
    Modified:   analysis/4.1.2_qtl.analysis_cont_rz.age_ici-early.vs.pbs_snpsqc_pheno.corrected.cleaned_5.batches_mis.Rmd
    Modified:   analysis/4.1.2_qtl.analysis_cont_rz.age_ici-early.vs.pbs_snpsqc_pheno.corrected.cleaned_dis_no-xk_5.batches.Rmd
    Modified:   analysis/4.1.2_qtl.analysis_cont_rz.age_ici-early.vs.pbs_snpsqc_pheno.corrected.cleaned_dis_no-xk_5.batches_mis.Rmd
    Modified:   analysis/4.1.2_qtl.analysis_cont_rz.age_ici.vs.eoi_pheno.corrected.cleaned_5.batches.Rmd
    Modified:   analysis/4.1.2_qtl.analysis_cont_rz.age_ici.vs.eoi_pheno.corrected.cleaned_5.batches_mis.Rmd
    Modified:   analysis/4.1.2_qtl.analysis_cont_rz.age_ici.vs.eoi_snpsqc_pheno.corrected.cleaned_5.batches.Rmd
    Modified:   analysis/4.1.2_qtl.analysis_cont_rz.age_ici.vs.eoi_snpsqc_pheno.corrected.cleaned_5.batches_mis.Rmd
    Modified:   analysis/4.1.2_qtl.analysis_cont_rz.age_ici.vs.eoi_snpsqc_pheno.corrected.cleaned_dis_no-xk_5.batches.Rmd
    Modified:   analysis/4.1.2_qtl.analysis_cont_rz.age_ici.vs.eoi_snpsqc_pheno.corrected.cleaned_dis_no-xk_5.batches_mis.Rmd
    Modified:   analysis/4.1.2_qtl.analysis_cont_rz.age_ici.vs.pbs_pheno.corrected.cleaned_5.batches.Rmd
    Modified:   analysis/4.1.2_qtl.analysis_cont_rz.age_ici.vs.pbs_pheno.corrected.cleaned_5.batches_mis.Rmd
    Modified:   analysis/4.1.2_qtl.analysis_cont_rz.age_ici.vs.pbs_snpsqc_pheno.corrected.cleaned_5.batches.Rmd
    Modified:   analysis/4.1.2_qtl.analysis_cont_rz.age_ici.vs.pbs_snpsqc_pheno.corrected.cleaned_5.batches_mis.Rmd
    Modified:   analysis/4.1.2_qtl.analysis_cont_rz.age_ici.vs.pbs_snpsqc_pheno.corrected.cleaned_dis_no-xk_5.batches.Rmd
    Modified:   analysis/4.1.2_qtl.analysis_cont_rz.age_ici.vs.pbs_snpsqc_pheno.corrected.cleaned_dis_no-xk_5.batches_mis.Rmd
    Modified:   analysis/index_5.batches.Rmd
    Modified:   analysis/index_5.batches_additional.Rmd

</code></pre>
<p>
Note that any generated files, e.g. HTML, png, CSS, etc., are not included in this status report because it is ok for generated content to have uncommitted changes.
</p>
</div>
</div>
</div>
</div>
<hr>
</div>
<div id="versions" class="tab-pane fade">
<p>
There are no past versions. Publish this analysis with <code>wflow_publish()</code> to start tracking its development.
</p>
<hr>
</div>
</div>
</div>
<div id="loading-data" class="section level1">
<h1>Loading Data</h1>
<pre class="r"><code>load(&quot;data/gm_allqc_5.batches.RData&quot;)

#gm_allqc
gm=gm_allqc
gm</code></pre>
<pre><code>Object of class cross2 (crosstype &quot;bc&quot;)

Total individuals              308
No. genotyped individuals      308
No. phenotyped individuals     308
No. with both geno &amp; pheno     308

No. phenotypes                   1
No. covariates                   6
No. phenotype covariates         0

No. chromosomes                 20
Total markers                34537

No. markers by chr:
   1    2    3    4    5    6    7    8    9   10   11   12   13   14   15   16 
2643 2629 1857 1890 1774 1941 1672 1627 1878 1176 1871 1300 1549 1578 1257  935 
  17   18   19    X 
 501  913 1014 4532 </code></pre>
<pre class="r"><code>#pr &lt;- readRDS(&quot;data/serreze_probs_allqc.rds&quot;)
#pr &lt;- readRDS(&quot;data/serreze_probs.rds&quot;)

##extracting animals with ici and eoi group status
miceinfo &lt;- gm$covar[gm$covar$group == &quot;EOI&quot; | gm$covar$group == &quot;ICI&quot;,]
table(miceinfo$group)</code></pre>
<pre><code>
EOI ICI 
164 104 </code></pre>
<pre class="r"><code>mice.ids &lt;- rownames(miceinfo)

gm &lt;- gm[mice.ids]
gm</code></pre>
<pre><code>Object of class cross2 (crosstype &quot;bc&quot;)

Total individuals              268
No. genotyped individuals      268
No. phenotyped individuals     268
No. with both geno &amp; pheno     268

No. phenotypes                   1
No. covariates                   6
No. phenotype covariates         0

No. chromosomes                 20
Total markers                34537

No. markers by chr:
   1    2    3    4    5    6    7    8    9   10   11   12   13   14   15   16 
2643 2629 1857 1890 1774 1941 1672 1627 1878 1176 1871 1300 1549 1578 1257  935 
  17   18   19    X 
 501  913 1014 4532 </code></pre>
<pre class="r"><code>table(gm$covar$group)</code></pre>
<pre><code>
EOI ICI 
164 104 </code></pre>
<pre class="r"><code>gm.full &lt;- gm

covars &lt;- read_csv(&quot;data/covar_corrected.cleaned_ici.vs.eoi_5.batches_52.csv&quot;)
#removing any missing info
missing_ids &lt;- covars[which(covars$&quot;rz.age&quot;==&quot;&quot;),]$Mouse.ID
length(missing_ids)</code></pre>
<pre><code>[1] 1</code></pre>
<pre class="r"><code>missing_ids</code></pre>
<pre><code>[1] &quot;NG01228-ICI&quot;</code></pre>
<pre class="r"><code>if(length(missing_ids) != 0){
gm &lt;- gm[paste0(&quot;-&quot;,as.character(missing_ids)),]
gm
table(gm$covar$group)

#keeping only mouseids in covars
covars &lt;- covars[gm$covar$id,]
table(covars$group)
}</code></pre>
<pre><code>
EOI ICI 
164 103 </code></pre>
<pre class="r"><code>#covars[covars$age.of.onset==0,]$age.of.onset &lt;- &#39;&#39;


##removing problmetic marker
gm &lt;- drop_markers(gm, &quot;UNCHS013106&quot;)

query_variants &lt;- create_variant_query_func(&quot;/Users/corneb/Documents/MyJax/CS/Projects/support.files/qtl2/cc_variants.sqlite&quot;)
query_genes &lt;- create_gene_query_func(&quot;/Users/corneb/Documents/MyJax/CS/Projects/support.files/qtl2/mouse_genes_mgi.sqlite&quot;)


##dropping monomorphic markers within the dataset

g &lt;- do.call(&quot;cbind&quot;, gm$geno)
#g[1,1:ncol(g)]

gf_mar &lt;- t(apply(g, 2, function(a) table(factor(a, 1:2))/sum(a != 0)))
#gn_mar &lt;- t(apply(g, 2, function(a) table(factor(a, 1:2))))

gf_mar &lt;- gf_mar[gf_mar[,2] != &quot;NaN&quot;,]

count &lt;- rowSums(gf_mar &lt;=0.05)
low_freq_df &lt;- merge(as.data.frame(gf_mar),as.data.frame(count), by=&quot;row.names&quot;,all=T)
low_freq_df[is.na(low_freq_df)] &lt;- &#39;&#39;
low_freq_df &lt;- low_freq_df[low_freq_df$count == 1,]
rownames(low_freq_df) &lt;- low_freq_df$Row.names

low_freq &lt;- find_markerpos(gm, rownames(low_freq_df))
low_freq$id &lt;- rownames(low_freq)

nrow(low_freq)</code></pre>
<pre><code>[1] 8398</code></pre>
<pre class="r"><code>low_freq_bad &lt;- merge(low_freq,low_freq_df, by=&quot;row.names&quot;,all=T)
names(low_freq_bad)[1] &lt;- c(&quot;marker&quot;)

gf_mar &lt;- gf_mar[gf_mar[,2] != &quot;NaN&quot;,]
MAF &lt;- apply(gf_mar, 1, function(x) min(x))
MAF &lt;- as.data.frame(MAF)
MAF$index &lt;- 1:nrow(gf_mar)
gf_mar_maf &lt;- merge(gf_mar,as.data.frame(MAF), by=&quot;row.names&quot;)
gf_mar_maf &lt;- gf_mar_maf[order(gf_mar_maf$index),]

gfmar &lt;- NULL
gfmar$gfmar_mar_0 &lt;- sum(gf_mar_maf$MAF==0)
gfmar$gfmar_mar_1 &lt;- sum(gf_mar_maf$MAF&lt; 0.01)
gfmar$gfmar_mar_5 &lt;- sum(gf_mar_maf$MAF&lt; 0.05)
gfmar$gfmar_mar_10 &lt;- sum(gf_mar_maf$MAF&lt; 0.10)
gfmar$gfmar_mar_15 &lt;- sum(gf_mar_maf$MAF&lt; 0.15)
gfmar$gfmar_mar_25 &lt;- sum(gf_mar_maf$MAF&lt; 0.25)
gfmar$gfmar_mar_50 &lt;- sum(gf_mar_maf$MAF&lt; 0.50)
gfmar$total_snps &lt;- nrow(as.data.frame(gf_mar_maf))

gfmar &lt;- t(as.data.frame(gfmar))
gfmar &lt;- as.data.frame(gfmar)
gfmar$count &lt;- gfmar$V1

gfmar[c(2)] %&gt;%
  kable(escape = F,align = c(&quot;ccccccccc&quot;),linesep =&quot;\\hline&quot;) %&gt;%
  kable_styling(full_width = F) %&gt;%
  kable_styling(&quot;striped&quot;, full_width = F)  %&gt;%
  row_spec(8 ,bold=T,color= &quot;white&quot;,background = &quot;black&quot;)</code></pre>
<table class="table table table-striped" style="width: auto !important; margin-left: auto; margin-right: auto; width: auto !important; margin-left: auto; margin-right: auto;">
<thead>
<tr>
<th style="text-align:left;">
</th>
<th style="text-align:center;">
count
</th>
</tr>
</thead>
<tbody>
<tr>
<td style="text-align:left;">
gfmar_mar_0
</td>
<td style="text-align:center;">
3816
</td>
</tr>
<tr>
<td style="text-align:left;">
gfmar_mar_1
</td>
<td style="text-align:center;">
4123
</td>
</tr>
<tr>
<td style="text-align:left;">
gfmar_mar_5
</td>
<td style="text-align:center;">
8398
</td>
</tr>
<tr>
<td style="text-align:left;">
gfmar_mar_10
</td>
<td style="text-align:center;">
8545
</td>
</tr>
<tr>
<td style="text-align:left;">
gfmar_mar_15
</td>
<td style="text-align:center;">
8618
</td>
</tr>
<tr>
<td style="text-align:left;">
gfmar_mar_25
</td>
<td style="text-align:center;">
9026
</td>
</tr>
<tr>
<td style="text-align:left;">
gfmar_mar_50
</td>
<td style="text-align:center;">
34313
</td>
</tr>
<tr>
<td style="text-align:left;font-weight: bold;color: white !important;background-color: black !important;">
total_snps
</td>
<td style="text-align:center;font-weight: bold;color: white !important;background-color: black !important;">
34537
</td>
</tr>
</tbody>
</table>
<pre class="r"><code>gm_qc &lt;- drop_markers(gm, low_freq_bad$marker)
gm_qc &lt;- drop_nullmarkers(gm_qc)
 
gm = gm_qc
gm</code></pre>
<pre><code>Object of class cross2 (crosstype &quot;bc&quot;)

Total individuals              267
No. genotyped individuals      267
No. phenotyped individuals     267
No. with both geno &amp; pheno     267

No. phenotypes                   1
No. covariates                   6
No. phenotype covariates         0

No. chromosomes                 20
Total markers                26139

No. markers by chr:
   1    2    3    4    5    6    7    8    9   10   11   12   13   14   15   16 
2336 2342 1589 1661 1535 1681 1450 1438 1654  947 1651 1107 1359 1396  945  728 
  17   18   19    X 
 303  747  903  367 </code></pre>
<pre class="r"><code>markers &lt;- marker_names(gm)
gmapdf &lt;- read.csv(&quot;/Users/corneb/Documents/MyJax/CS/Projects/Serreze/haplotype.reconstruction/output_5.batches/genetic_map.csv&quot;)
pmapdf &lt;- read.csv(&quot;/Users/corneb/Documents/MyJax/CS/Projects/Serreze/haplotype.reconstruction/output_5.batches/physical_map.csv&quot;)
#mapdf &lt;- merge(gmapdf,pmapdf, by=c(&quot;marker&quot;,&quot;chr&quot;), all=T)
#rownames(mapdf) &lt;- mapdf$marker
#mapdf &lt;- mapdf[markers,]
#names(mapdf) &lt;- c(&#39;marker&#39;,&#39;chr&#39;,&#39;gmapdf&#39;,&#39;pmapdf&#39;)
#mapdfnd &lt;- mapdf[!duplicated(mapdf[c(2:3)]),]

pr.qc &lt;- calc_genoprob(gm)</code></pre>
</div>
<div id="preparing-necessary-info" class="section level1">
<h1>Preparing necessary info</h1>
<pre class="r"><code>covars$rz.age &lt;- as.numeric(covars$rz.age)
Xcovar &lt;- get_x_covar(gm)
addcovar = model.matrix(~ICI.vs.EOI, data = covars)[,-1]
#addcovar.i = model.matrix(~ICI.vs.EOI, data = covars)[,-1]

K &lt;- calc_kinship(pr.qc, type = &quot;loco&quot;)
#heatmap(K[[1]])
K.overall &lt;- calc_kinship(pr.qc, type = &quot;overall&quot;)
#heatmap(K.overall)
kinship &lt;- calc_kinship(pr.qc)
heatmap(kinship)</code></pre>
<p><img src="figure/4.1.2_qtl.analysis_cont_rz.age_ici.vs.eoi_snpsqc_pheno.corrected.cleaned_5.batches_52.Rmd/permutation%20prelim-1.png" width="912" style="display: block; margin: auto;" /></p>
<div id="no-additive-covariates" class="section level2">
<h2>No additive covariates</h2>
<div id="significance-thresholds" class="section level3">
<h3>Significance thresholds</h3>
<pre class="r"><code>operm &lt;- scan1perm(pr.qc, covars[&quot;rz.age&quot;], model=&quot;normal&quot;, n_perm=10, perm_Xsp=TRUE, chr_lengths=chr_lengths(gm$gmap))

summary_table&lt;-data.frame(unclass(summary(operm, alpha=c(0.01,  0.05, 0.1))))
names(summary_table) &lt;- c(&quot;autosomes&quot;,&quot;X&quot;)
summary_table$significance.level &lt;- rownames(summary_table)

rownames(summary_table) &lt;- NULL

summary_table[c(3,1:2)] %&gt;%
  kable(escape = F,align = c(&quot;ccc&quot;)) %&gt;%
  kable_styling(&quot;striped&quot;, full_width = T) %&gt;%
  column_spec(1, bold=TRUE)</code></pre>
<table class="table table-striped" style="margin-left: auto; margin-right: auto;">
<thead>
<tr>
<th style="text-align:center;">
significance.level
</th>
<th style="text-align:center;">
autosomes
</th>
<th style="text-align:center;">
X
</th>
</tr>
</thead>
<tbody>
<tr>
<td style="text-align:center;font-weight: bold;">
0.01
</td>
<td style="text-align:center;">
3.455405
</td>
<td style="text-align:center;">
2.326645
</td>
</tr>
<tr>
<td style="text-align:center;font-weight: bold;">
0.05
</td>
<td style="text-align:center;">
3.387383
</td>
<td style="text-align:center;">
2.213590
</td>
</tr>
<tr>
<td style="text-align:center;font-weight: bold;">
0.1
</td>
<td style="text-align:center;">
3.302144
</td>
<td style="text-align:center;">
2.065751
</td>
</tr>
</tbody>
</table>
</div>
<div id="genome-wide-scan" class="section level3">
<h3>Genome-wide scan</h3>
<pre class="r"><code>plot_lod&lt;-function(out,map){
  for (i in 1:dim(out)[2]){
#par(mar=c(5.1, 6.1, 1.1, 1.1))
    ymx &lt;- maxlod(out) # overall maximum LOD score
    plot(out, map, lodcolumn=i, col=&quot;slateblue&quot;, ylim=c(0, ymx+0.5))
    #legend(&quot;topright&quot;, lwd=2, colnames(out)[i], bg=&quot;gray90&quot;)
    title(main = paste0(colnames(out)[i], &quot; - ICI vs. EOI (no covariates) [positions in cM]&quot;))
    add_threshold(map,  summary(operm,alpha=0.1), col = &#39;purple&#39;)
    add_threshold(map,  summary(operm, alpha=0.05), col = &#39;red&#39;)
    add_threshold(map,  summary(operm, alpha=0.01), col = &#39;blue&#39;)

  }
}


print(&quot;with no kinship&quot;)</code></pre>
<p>[1] “with no kinship”</p>
<pre class="r"><code>outf &lt;- scan1(pr.qc, covars[&quot;rz.age&quot;], Xcovar=Xcovar, model=&quot;normal&quot;)

plot_lod(outf,gm$gmap)</code></pre>
<p><img src="figure/4.1.2_qtl.analysis_cont_rz.age_ici.vs.eoi_snpsqc_pheno.corrected.cleaned_5.batches_52.Rmd/scanone-1.png" width="912" style="display: block; margin: auto;" /></p>
<pre class="r"><code>print(&quot;with normal kinship&quot;)</code></pre>
<p>[1] “with normal kinship”</p>
<pre class="r"><code>outk &lt;- scan1(pr.qc, covars[&quot;rz.age&quot;], Xcovar=Xcovar, model=&quot;normal&quot;, kinship=kinship)

plot_lod(outk,gm$gmap)</code></pre>
<p><img src="figure/4.1.2_qtl.analysis_cont_rz.age_ici.vs.eoi_snpsqc_pheno.corrected.cleaned_5.batches_52.Rmd/scanone-2.png" width="912" style="display: block; margin: auto;" /></p>
<pre class="r"><code>print(&quot;with loco kinship&quot;)</code></pre>
<p>[1] “with loco kinship”</p>
<pre class="r"><code>out &lt;- scan1(pr.qc, covars[&quot;rz.age&quot;], Xcovar=Xcovar, model=&quot;normal&quot;, kinship=K)

plot_lod(out,gm$gmap)</code></pre>
<p><img src="figure/4.1.2_qtl.analysis_cont_rz.age_ici.vs.eoi_snpsqc_pheno.corrected.cleaned_5.batches_52.Rmd/scanone-3.png" width="912" style="display: block; margin: auto;" /></p>
<pre class="r"><code>print(paste0(&quot;number of samples in analysis = &quot;, as.data.frame(attributes(out)$sample_size)[1,]))</code></pre>
<p>[1] “number of samples in analysis = 267”</p>
</div>
<div id="lod-peaks" class="section level3">
<h3>LOD peaks</h3>
<div id="centimorgan-cm" class="section level4">
<h4>Centimorgan (cM)</h4>
<pre class="r"><code>peaks&lt;-find_peaks(out, gm$gmap, threshold=summary(operm,alpha=0.05)$A, thresholdX = summary(operm,alpha=0.05)$X, peakdrop=3, drop=1.5)

if(nrow(peaks) &gt; 0){
peaks$marker &lt;- find_marker(gm$gmap, chr=peaks$chr,pos=peaks$pos)
names(peaks)[2] &lt;- c(&quot;phenotype&quot;)
peaks &lt;- peaks[-1]

rownames(peaks) &lt;- NULL
print(kable(peaks, escape = F, align = c(&quot;cccccccc&quot;), &quot;html&quot;) 
  %&gt;% kable_styling(&quot;striped&quot;, full_width = T)%&gt;%
  column_spec(1, bold=TRUE)
  )

#plot only peak chromosomes

plot_lod_chr&lt;-function(out,map,chrom){
  for (i in 1:dim(out)[2]){
    #png(filename=paste0(&quot;/Users/chenm/Documents/qtl/Jai/&quot;,colnames(out)[i],  &quot;_lod.png&quot;))
    
    #par(mar=c(5.1, 6.1, 1.1, 1.1))
    ymx &lt;- maxlod(out) # overall maximum LOD score
    plot(out, map, chr = chrom, lodcolumn=i, col=&quot;slateblue&quot;, ylim=c(0, ymx+0.5))
    #legend(&quot;topright&quot;, lwd=2, colnames(out)[i], bg=&quot;gray90&quot;)
    title(main = paste0(colnames(out)[i], &quot; - chr&quot;, chrom, &quot; - ICI vs. EOI (no covariates) [positions in cM]&quot;))
    add_threshold(map,  summary(operm,alpha=0.1), col = &#39;purple&#39;)
    add_threshold(map,  summary(operm, alpha=0.05), col = &#39;red&#39;)
    add_threshold(map,  summary(operm, alpha=0.01), col = &#39;blue&#39;)
    #for (j in 1: dim(summary_table)[1]){
    #  abline(h=summary_table[j, i],col=&quot;red&quot;)
    #  text(x=400, y =summary_table[j, i]+0.12, labels = paste(&quot;p=&quot;, row.names(summary_table)[j]))
    #}
    #dev.off()

  }
}


} else {
  print(paste0(&quot;There are no peaks that have a LOD that reaches suggestive (p&lt;0.05) level of &quot;,summary(operm,alpha=0.05)$A, &quot; [autosomes]/&quot;,summary(operm,alpha=0.05)$X, &quot; [x-chromosome]&quot;))
}</code></pre>
<table class="table table-striped" style="margin-left: auto; margin-right: auto;">
<thead>
<tr>
<th style="text-align:center;">
phenotype
</th>
<th style="text-align:center;">
chr
</th>
<th style="text-align:center;">
pos
</th>
<th style="text-align:center;">
lod
</th>
<th style="text-align:center;">
ci_lo
</th>
<th style="text-align:center;">
ci_hi
</th>
<th style="text-align:center;">
marker
</th>
</tr>
</thead>
<tbody>
<tr>
<td style="text-align:center;font-weight: bold;">
rz.age
</td>
<td style="text-align:center;">
3
</td>
<td style="text-align:center;">
18.749
</td>
<td style="text-align:center;">
11.14063
</td>
<td style="text-align:center;">
16.581
</td>
<td style="text-align:center;">
21.701
</td>
<td style="text-align:center;">
UNCHS008487
</td>
</tr>
</tbody>
</table>
<pre class="r"><code>if(nrow(peaks) &lt; 50 &amp; nrow(peaks) &gt; 0){

for(i in unique(peaks$chr)){
#for (i in 1:nrow(peaks)){
  #plot_lod_chr(out,gm$gmap, peaks$chr[i])
  plot_lod_chr(out,gm$gmap, i)
}

} else {
  print(paste0(&quot;There are too many peaks (&quot;,nrow(peaks),&quot; peaks) that have a LOD that reaches suggestive (p&lt;0.05) level of &quot;,summary(operm,alpha=0.05)$A, &quot; [autosomes]/&quot;,summary(operm,alpha=0.05)$X, &quot; [x-chromosome]&quot;))
}</code></pre>
<p><img src="figure/4.1.2_qtl.analysis_cont_rz.age_ici.vs.eoi_snpsqc_pheno.corrected.cleaned_5.batches_52.Rmd/LOD%20peaks-1.png" width="912" style="display: block; margin: auto;" /></p>
</div>
<div id="megabase-mb" class="section level4">
<h4>Megabase (MB)</h4>
<pre class="r"><code>peaks_mba&lt;-find_peaks(out, gm$pmap, threshold=summary(operm,alpha=0.05)$A, thresholdX = summary(operm,alpha=0.05)$X, peakdrop=3, drop=1.5)
if(nrow(peaks) &gt; 0){
peaks_mba$marker &lt;- find_marker(gm$pmap, chr=peaks_mba$chr,pos=peaks_mba$pos)
names(peaks_mba)[2] &lt;- c(&quot;phenotype&quot;)
peaks_mba &lt;- peaks_mba[-1]

rownames(peaks_mba) &lt;- NULL
print(kable(peaks_mba, escape = F, align = c(&quot;cccccccc&quot;), &quot;html&quot;) 
  %&gt;% kable_styling(&quot;striped&quot;, full_width = T)%&gt;%
  column_spec(1, bold=TRUE)
  )

  plot_lod_chr_mb&lt;-function(out,map,chrom){
  for (i in 1:dim(out)[2]){
    #png(filename=paste0(&quot;/Users/chenm/Documents/qtl/Jai/&quot;,colnames(out)[i],  &quot;_lod.png&quot;))
    
    #par(mar=c(5.1, 6.1, 1.1, 1.1))
    ymx &lt;- maxlod(out) # overall maximum LOD score
    plot(out, map, chr = chrom, lodcolumn=i, col=&quot;slateblue&quot;, ylim=c(0, ymx+0.5))
    #legend(&quot;topright&quot;, lwd=2, colnames(out)[i], bg=&quot;gray90&quot;)
    title(main = paste0(colnames(out)[i], &quot; - chr&quot;, chrom, &quot; - ICI vs. EOI (no covariates) [positions in cM]&quot;))
    add_threshold(map,  summary(operm,alpha=0.1), col = &#39;purple&#39;)
    add_threshold(map,  summary(operm, alpha=0.05), col = &#39;red&#39;)
    add_threshold(map,  summary(operm, alpha=0.01), col = &#39;blue&#39;)
    #for (j in 1: dim(summary_table)[1]){
    #  abline(h=summary_table[j, i],col=&quot;red&quot;)
    #  text(x=400, y =summary_table[j, i]+0.12, labels = paste(&quot;p=&quot;, row.names(summary_table)[j]))
    #}
    #dev.off()
  }
}

} else {
  print(paste0(&quot;There are no peaks that have a LOD that reaches suggestive (p&lt;0.05) level of &quot;,summary(operm,alpha=0.05)$A, &quot; [autosomes]/&quot;,summary(operm,alpha=0.05)$X, &quot; [x-chromosome]&quot;))
}</code></pre>
<table class="table table-striped" style="margin-left: auto; margin-right: auto;">
<thead>
<tr>
<th style="text-align:center;">
phenotype
</th>
<th style="text-align:center;">
chr
</th>
<th style="text-align:center;">
pos
</th>
<th style="text-align:center;">
lod
</th>
<th style="text-align:center;">
ci_lo
</th>
<th style="text-align:center;">
ci_hi
</th>
<th style="text-align:center;">
marker
</th>
</tr>
</thead>
<tbody>
<tr>
<td style="text-align:center;font-weight: bold;">
rz.age
</td>
<td style="text-align:center;">
3
</td>
<td style="text-align:center;">
38.95539
</td>
<td style="text-align:center;">
11.14063
</td>
<td style="text-align:center;">
34.11194
</td>
<td style="text-align:center;">
49.9433
</td>
<td style="text-align:center;">
UNCHS008487
</td>
</tr>
</tbody>
</table>
<pre class="r"><code>if(nrow(peaks_mba) &lt; 50 &amp; nrow(peaks_mba) &gt; 0){

for(i in unique(peaks_mba$chr)){
#for (i in 1:nrow(peaks_mba)){
  #plot_lod_chr_mb(out,gm$pmap, peaks_mba$chr[i])
  plot_lod_chr_mb(out,gm$pmap,i)
}

} else {
  print(paste0(&quot;There are too many peaks (&quot;,nrow(peaks_mba),&quot; peaks) that have a LOD that reaches suggestive (p&lt;0.05) level of &quot;,summary(operm,alpha=0.05)$A, &quot; [autosomes]/&quot;,summary(operm,alpha=0.05)$X, &quot; [x-chromosome]&quot;))
}</code></pre>
<p><img src="figure/4.1.2_qtl.analysis_cont_rz.age_ici.vs.eoi_snpsqc_pheno.corrected.cleaned_5.batches_52.Rmd/LOD%20peaks_mba-1.png" width="912" style="display: block; margin: auto;" /></p>
</div>
</div>
<div id="qtl-effects" class="section level3">
<h3>QTL effects</h3>
<p>For each peak LOD location we give a list of gene</p>
<pre class="r"><code>if(nrow(peaks) &lt; 50 &amp; nrow(peaks) &gt; 0){
for (i in 1:nrow(peaks)){
#for (i in 1:1){
  #Plot 1
  g &lt;- maxmarg(pr.qc, gm$gmap, chr=peaks$chr[i], pos=peaks$pos[i], return_char=TRUE)
  #png(filename=paste0(&quot;/Users/chenm/Documents/qtl/Jai/&quot;,&quot;qtl_effect_&quot;, i, &quot;.png&quot;))
  #par(mar=c(4.1, 4.1, 1.5, 0.6))
  plot_pxg(g, covars[,peaks$phenotype[i]], ylab=peaks$phenotype[i], sort=FALSE)
  title(main = paste0(&quot;chr: &quot;, chr=peaks$chr[i],&quot;; pos: &quot;, peaks$pos[i], &quot;cM /&quot;,peaks_mba$pos[i],&quot;MB\n(&quot;,peaks$phenotype[i],&quot; )&quot;), line=0.2)
  ##dev.off()

  chr = peaks$chr[i]

# Plot 2
  pr_sub &lt;- pull_genoprobint(pr.qc, gm$gmap, chr, c(peaks$ci_lo[i], peaks$ci_hi[i]))
  #coeff &lt;- scan1coef(pr[,chr], cross$pheno[,peaks$lodcolumn[i]], addcovar = addcovar)
  #coeff &lt;- scan1coef(pr[,chr], cross$pheno[,peaks$lodcolumn[i]], Xcovar=Xcovar)
  #coeff &lt;- scan1coef(pr.qc[,chr], gm$covar[peaks$lodcolumn[i]], model=&quot;binary&quot;)
  #coeff_sub &lt;- scan1coef(pr_sub[,chr], gm$covar[peaks$lodcolumn[i]], model=&quot;binary&quot;)
  blup &lt;- scan1blup(pr.qc[,chr], covars[peaks$phenotype[i]])
  blup_sub &lt;- scan1blup(pr_sub[,chr], covars[peaks$phenotype[i]])

  write.csv(as.data.frame(blup_sub), paste0(&quot;data/ici.vs.eoi_&quot;,peaks$phenotype[i],&quot;-no.covariates_blup_sub_chr-&quot;,chr,&quot;_peak.marker-&quot;,peaks$marker[i],&quot;_lod.drop-1.5_snpsqc_5.batches_52.csv&quot;), quote=F)

  #plot_coef(coeff, 
  #     gm$gmap, columns=1:2,
  #     bgcolor=&quot;gray95&quot;, legend=&quot;bottomleft&quot;, 
  #     main = paste(&quot;chr&quot;, chr=peaks$chr[i],&quot;; pos: &quot;, peaks$pos[i], &quot;cM /&quot;,peaks_mba$pos[i],&quot;MB\n(&quot;,peaks$lodcolumn[i],&quot; [scan1coeff; positions in cM] )&quot;)
  #     )

  #plot_coef(coeff_sub, 
  #     gm$gmap, columns=1:2,
  #     bgcolor=&quot;gray95&quot;, legend=&quot;bottomleft&quot;, 
  #     main = paste(&quot;chr&quot;, chr=peaks$chr[i],&quot;; pos: &quot;, peaks$pos[i], &quot;cM /&quot;,peaks_mba$pos[i],&quot;MB\n(&quot;,peaks$lodcolumn[i],&quot;; 1.5 LOD drop interval [scan1coeff; positions in cM] ) &quot;)
  #     )


  plot_coef(blup, 
       gm$gmap, columns=1:2,
       bgcolor=&quot;gray95&quot;, legend=&quot;bottomleft&quot;, 
       main = paste0(&quot;chr: &quot;, chr=peaks$chr[i],&quot;; pos: &quot;, peaks$pos[i], &quot;cM /&quot;,peaks_mba$pos[i],&quot;MB\n(&quot;,peaks$phenotype[i],&quot; [scan1blup; positions in cM] )&quot;)
       )

  plot_coef(blup_sub, 
       gm$gmap, columns=1:2,
       bgcolor=&quot;gray95&quot;, legend=&quot;bottomleft&quot;, 
       main = paste0(&quot;chr: &quot;, chr=peaks$chr[i],&quot;; pos: &quot;, peaks$pos[i], &quot;cM /&quot;,peaks_mba$pos[i],&quot;MB\n(&quot;,peaks$phenotype[i],&quot;; 1.5 LOD drop interval [scan1blup; positions in cM] )&quot;)
       )


 # Plot 3
  #c2effB &lt;- scan1coef(pr.qc[,chr], gm$covar[peaks$lodcolumn[i]], model=&quot;binary&quot;, contrasts=cbind(a=c(-1, 0), d=c(0, -1)))
  #c2effBb &lt;- scan1blup(pr.qc[,chr], gm$covar[peaks$lodcolumn[i]], contrasts=cbind(a=c(-1, 0), d=c(0, -1)))
  ##c2effB &lt;- scan1coef(pr[,chr], cross$pheno[,peaks$lodcolumn[i]], addcovar = addcovar, contrasts=cbind(mu=c(1,1,1), a=c(-1, 0, 1), d=c(0, 1, 0)))
  ##c2effB &lt;- scan1coef(pr[,chr], cross$pheno[,peaks$lodcolumn[i]],Xcovar=Xcovar, contrasts=cbind(mu=c(1,1,1), a=c(-1, 0, 1), d=c(0, 1, 0)))
  #plot(c2effB, gm$gmap[chr], columns=1:2,
  #     bgcolor=&quot;gray95&quot;, legend=&quot;bottomleft&quot;, 
  #     main = paste(&quot;chr&quot;, chr=peaks$chr[i], &quot;pos&quot;, peaks$pos[i], &quot;(&quot;,peaks$lodcolumn[i],&quot;)&quot;)
  #     )
  #plot(c2effBb, gm$gmap[chr], columns=1:2,
  #     bgcolor=&quot;gray95&quot;, legend=&quot;bottomleft&quot;, 
  #     main = paste(&quot;chr&quot;, chr=peaks$chr[i], &quot;pos&quot;, peaks$pos[i], &quot;(&quot;,peaks$lodcolumn[i],&quot;)&quot;)
  #     )
  ##last_coef &lt;- unclass(c2effB)[nrow(c2effB),2:3] # last two coefficients
  ##for(t in seq(along=last_coef))
  ##  axis(side=4, at=last_coef[t], names(last_coef)[t], tick=FALSE)


  #Table 1
  chr = peaks_mba$chr[i]
  start=as.numeric(peaks_mba$ci_lo[i])
  end=as.numeric(peaks_mba$ci_hi[i])

  genesgss = query_genes(chr, start, end)

  write.csv(genesgss, file=paste0(&quot;data/ici.vs.eoi_&quot;,peaks$phenotype[i],&quot;-no.covariates_genes_chr-&quot;,chr,&quot;_peak.marker-&quot;,peaks$marker[i],&quot;_lod.drop-1.5_snpsqc_5.batches_52.csv&quot;), quote=F)

  rownames(genesgss) &lt;- NULL
  genesgss$strand_old = genesgss$strand
  genesgss$strand[genesgss$strand==&quot;+&quot;] &lt;- &quot;positive&quot;
  genesgss$strand[genesgss$strand==&quot;-&quot;] &lt;- &quot;negative&quot;

  #genesgss &lt;- 
  #table &lt;- 
  #genesgss[,c(&quot;chr&quot;,&quot;type&quot;,&quot;start&quot;,&quot;stop&quot;,&quot;strand&quot;,&quot;ID&quot;,&quot;Name&quot;,&quot;Dbxref&quot;,&quot;gene_id&quot;,&quot;mgi_type&quot;,&quot;description&quot;)] %&gt;%
  #kable(escape = F,align = c(&quot;ccccccccccc&quot;)) %&gt;%
  #kable_styling(&quot;striped&quot;, full_width = T) #%&gt;% 
  #cat #%&gt;%
  #column_spec(1, bold=TRUE)
#
  #print(kable(genesgss[,c(&quot;chr&quot;,&quot;type&quot;,&quot;start&quot;,&quot;stop&quot;,&quot;strand&quot;,&quot;ID&quot;,&quot;Name&quot;,&quot;Dbxref&quot;,&quot;gene_id&quot;,&quot;mgi_type&quot;,&quot;description&quot;)], escape = F,align = c(&quot;ccccccccccc&quot;)))

  print(kable(genesgss[,c(&quot;chr&quot;,&quot;type&quot;,&quot;start&quot;,&quot;stop&quot;,&quot;strand&quot;,&quot;ID&quot;,&quot;Name&quot;,&quot;Dbxref&quot;,&quot;gene_id&quot;,&quot;mgi_type&quot;,&quot;description&quot;)], &quot;html&quot;) %&gt;% kable_styling(&quot;striped&quot;, full_width = T))

  #table
  

}

} else {
  print(paste0(&quot;There are too many peaks (&quot;,nrow(peaks_mba),&quot; peaks) that have a LOD that reaches suggestive (p&lt;0.05) level of &quot;,summary(operm,alpha=0.05)$A, &quot; [autosomes]/&quot;,summary(operm,alpha=0.05)$X, &quot; [x-chromosome]&quot;))
}</code></pre>
<img src="figure/4.1.2_qtl.analysis_cont_rz.age_ici.vs.eoi_snpsqc_pheno.corrected.cleaned_5.batches_52.Rmd/QTL%20effects-1.png" width="912" style="display: block; margin: auto;" /><img src="figure/4.1.2_qtl.analysis_cont_rz.age_ici.vs.eoi_snpsqc_pheno.corrected.cleaned_5.batches_52.Rmd/QTL%20effects-2.png" width="912" style="display: block; margin: auto;" /><img src="figure/4.1.2_qtl.analysis_cont_rz.age_ici.vs.eoi_snpsqc_pheno.corrected.cleaned_5.batches_52.Rmd/QTL%20effects-3.png" width="912" style="display: block; margin: auto;" />
<table class="table table-striped" style="margin-left: auto; margin-right: auto;">
<thead>
<tr>
<th style="text-align:left;">
chr
</th>
<th style="text-align:left;">
type
</th>
<th style="text-align:right;">
start
</th>
<th style="text-align:right;">
stop
</th>
<th style="text-align:left;">
strand
</th>
<th style="text-align:left;">
ID
</th>
<th style="text-align:left;">
Name
</th>
<th style="text-align:left;">
Dbxref
</th>
<th style="text-align:left;">
gene_id
</th>
<th style="text-align:left;">
mgi_type
</th>
<th style="text-align:left;">
description
</th>
</tr>
</thead>
<tbody>
<tr>
<td style="text-align:left;">
3
</td>
<td style="text-align:left;">
gene
</td>
<td style="text-align:right;">
34.10427
</td>
<td style="text-align:right;">
34.68262
</td>
<td style="text-align:left;">
positive
</td>
<td style="text-align:left;">
MGI_C57BL6J_2444112
</td>
<td style="text-align:left;">
Sox2ot
</td>
<td style="text-align:left;">
NCBI_Gene:320478,ENSEMBL:ENSMUSG00000105265
</td>
<td style="text-align:left;">
MGI:2444112
</td>
<td style="text-align:left;">
lncRNA gene
</td>
<td style="text-align:left;">
SOX2 overlapping transcript (non-protein coding)
</td>
</tr>
<tr>
<td style="text-align:left;">
3
</td>
<td style="text-align:left;">
pseudogene
</td>
<td style="text-align:right;">
34.18331
</td>
<td style="text-align:right;">
34.18419
</td>
<td style="text-align:left;">
positive
</td>
<td style="text-align:left;">
MGI_C57BL6J_3647478
</td>
<td style="text-align:left;">
Gm5708
</td>
<td style="text-align:left;">
NCBI_Gene:435727,ENSEMBL:ENSMUSG00000105739
</td>
<td style="text-align:left;">
MGI:3647478
</td>
<td style="text-align:left;">
pseudogene
</td>
<td style="text-align:left;">
predicted gene 5708
</td>
</tr>
<tr>
<td style="text-align:left;">
3
</td>
<td style="text-align:left;">
gene
</td>
<td style="text-align:right;">
34.28133
</td>
<td style="text-align:right;">
34.35199
</td>
<td style="text-align:left;">
negative
</td>
<td style="text-align:left;">
MGI_C57BL6J_5621390
</td>
<td style="text-align:left;">
Gm38505
</td>
<td style="text-align:left;">
NCBI_Gene:102638008,ENSEMBL:ENSMUSG00000092196
</td>
<td style="text-align:left;">
MGI:5621390
</td>
<td style="text-align:left;">
lncRNA gene
</td>
<td style="text-align:left;">
predicted gene, 38505
</td>
</tr>
<tr>
<td style="text-align:left;">
3
</td>
<td style="text-align:left;">
gene
</td>
<td style="text-align:right;">
34.41544
</td>
<td style="text-align:right;">
34.41816
</td>
<td style="text-align:left;">
positive
</td>
<td style="text-align:left;">
MGI_C57BL6J_5593758
</td>
<td style="text-align:left;">
Gm34599
</td>
<td style="text-align:left;">
NCBI_Gene:102637903,ENSEMBL:ENSMUSG00000105577
</td>
<td style="text-align:left;">
MGI:5593758
</td>
<td style="text-align:left;">
lncRNA gene
</td>
<td style="text-align:left;">
predicted gene, 34599
</td>
</tr>
<tr>
<td style="text-align:left;">
3
</td>
<td style="text-align:left;">
gene
</td>
<td style="text-align:right;">
34.42737
</td>
<td style="text-align:right;">
34.43510
</td>
<td style="text-align:left;">
negative
</td>
<td style="text-align:left;">
MGI_C57BL6J_5141980
</td>
<td style="text-align:left;">
Gm20515
</td>
<td style="text-align:left;">
ENSEMBL:ENSMUSG00000092242
</td>
<td style="text-align:left;">
MGI:5141980
</td>
<td style="text-align:left;">
lncRNA gene
</td>
<td style="text-align:left;">
predicted gene 20515
</td>
</tr>
<tr>
<td style="text-align:left;">
3
</td>
<td style="text-align:left;">
gene
</td>
<td style="text-align:right;">
34.48143
</td>
<td style="text-align:right;">
34.48221
</td>
<td style="text-align:left;">
negative
</td>
<td style="text-align:left;">
MGI_C57BL6J_5579841
</td>
<td style="text-align:left;">
Gm29135
</td>
<td style="text-align:left;">
ENSEMBL:ENSMUSG00000099437
</td>
<td style="text-align:left;">
MGI:5579841
</td>
<td style="text-align:left;">
lncRNA gene
</td>
<td style="text-align:left;">
predicted gene 29135
</td>
</tr>
<tr>
<td style="text-align:left;">
3
</td>
<td style="text-align:left;">
gene
</td>
<td style="text-align:right;">
34.56034
</td>
<td style="text-align:right;">
34.56055
</td>
<td style="text-align:left;">
positive
</td>
<td style="text-align:left;">
MGI_C57BL6J_5530785
</td>
<td style="text-align:left;">
Gm27403
</td>
<td style="text-align:left;">
ENSEMBL:ENSMUSG00000098795
</td>
<td style="text-align:left;">
MGI:5530785
</td>
<td style="text-align:left;">
unclassified non-coding RNA gene
</td>
<td style="text-align:left;">
predicted gene, 27403
</td>
</tr>
<tr>
<td style="text-align:left;">
3
</td>
<td style="text-align:left;">
gene
</td>
<td style="text-align:right;">
34.56526
</td>
<td style="text-align:right;">
34.57220
</td>
<td style="text-align:left;">
negative
</td>
<td style="text-align:left;">
MGI_C57BL6J_5662832
</td>
<td style="text-align:left;">
Gm42695
</td>
<td style="text-align:left;">
ENSEMBL:ENSMUSG00000106525
</td>
<td style="text-align:left;">
MGI:5662832
</td>
<td style="text-align:left;">
lncRNA gene
</td>
<td style="text-align:left;">
predicted gene 42695
</td>
</tr>
<tr>
<td style="text-align:left;">
3
</td>
<td style="text-align:left;">
gene
</td>
<td style="text-align:right;">
34.59782
</td>
<td style="text-align:right;">
34.60488
</td>
<td style="text-align:left;">
negative
</td>
<td style="text-align:left;">
MGI_C57BL6J_5625090
</td>
<td style="text-align:left;">
Gm42205
</td>
<td style="text-align:left;">
NCBI_Gene:105247023,ENSEMBL:ENSMUSG00000105222
</td>
<td style="text-align:left;">
MGI:5625090
</td>
<td style="text-align:left;">
lncRNA gene
</td>
<td style="text-align:left;">
predicted gene, 42205
</td>
</tr>
<tr>
<td style="text-align:left;">
3
</td>
<td style="text-align:left;">
gene
</td>
<td style="text-align:right;">
34.63846
</td>
<td style="text-align:right;">
34.63854
</td>
<td style="text-align:left;">
positive
</td>
<td style="text-align:left;">
MGI_C57BL6J_3811407
</td>
<td style="text-align:left;">
Mir1897
</td>
<td style="text-align:left;">
miRBase:MI0008314,NCBI_Gene:100316679,ENSEMBL:ENSMUSG00000104797
</td>
<td style="text-align:left;">
MGI:3811407
</td>
<td style="text-align:left;">
miRNA gene
</td>
<td style="text-align:left;">
microRNA 1897
</td>
</tr>
<tr>
<td style="text-align:left;">
3
</td>
<td style="text-align:left;">
gene
</td>
<td style="text-align:right;">
34.64326
</td>
<td style="text-align:right;">
34.64610
</td>
<td style="text-align:left;">
positive
</td>
<td style="text-align:left;">
MGI_C57BL6J_5662829
</td>
<td style="text-align:left;">
Gm42692
</td>
<td style="text-align:left;">
ENSEMBL:ENSMUSG00000104901
</td>
<td style="text-align:left;">
MGI:5662829
</td>
<td style="text-align:left;">
unclassified gene
</td>
<td style="text-align:left;">
predicted gene 42692
</td>
</tr>
<tr>
<td style="text-align:left;">
3
</td>
<td style="text-align:left;">
gene
</td>
<td style="text-align:right;">
34.64999
</td>
<td style="text-align:right;">
34.65246
</td>
<td style="text-align:left;">
positive
</td>
<td style="text-align:left;">
MGI_C57BL6J_98364
</td>
<td style="text-align:left;">
Sox2
</td>
<td style="text-align:left;">
NCBI_Gene:20674,ENSEMBL:ENSMUSG00000074637
</td>
<td style="text-align:left;">
MGI:98364
</td>
<td style="text-align:left;">
protein coding gene
</td>
<td style="text-align:left;">
SRY (sex determining region Y)-box 2
</td>
</tr>
<tr>
<td style="text-align:left;">
3
</td>
<td style="text-align:left;">
gene
</td>
<td style="text-align:right;">
34.65318
</td>
<td style="text-align:right;">
34.65328
</td>
<td style="text-align:left;">
positive
</td>
<td style="text-align:left;">
MGI_C57BL6J_5530993
</td>
<td style="text-align:left;">
Gm27611
</td>
<td style="text-align:left;">
ENSEMBL:ENSMUSG00000098254
</td>
<td style="text-align:left;">
MGI:5530993
</td>
<td style="text-align:left;">
unclassified non-coding RNA gene
</td>
<td style="text-align:left;">
predicted gene, 27611
</td>
</tr>
<tr>
<td style="text-align:left;">
3
</td>
<td style="text-align:left;">
gene
</td>
<td style="text-align:right;">
34.66366
</td>
<td style="text-align:right;">
34.66429
</td>
<td style="text-align:left;">
negative
</td>
<td style="text-align:left;">
MGI_C57BL6J_5662830
</td>
<td style="text-align:left;">
Gm42693
</td>
<td style="text-align:left;">
ENSEMBL:ENSMUSG00000105284
</td>
<td style="text-align:left;">
MGI:5662830
</td>
<td style="text-align:left;">
unclassified gene
</td>
<td style="text-align:left;">
predicted gene 42693
</td>
</tr>
<tr>
<td style="text-align:left;">
3
</td>
<td style="text-align:left;">
gene
</td>
<td style="text-align:right;">
34.68263
</td>
<td style="text-align:right;">
34.68275
</td>
<td style="text-align:left;">
positive
</td>
<td style="text-align:left;">
MGI_C57BL6J_5663343
</td>
<td style="text-align:left;">
Gm43206
</td>
<td style="text-align:left;">
ENSEMBL:ENSMUSG00000105101
</td>
<td style="text-align:left;">
MGI:5663343
</td>
<td style="text-align:left;">
unclassified gene
</td>
<td style="text-align:left;">
predicted gene 43206
</td>
</tr>
<tr>
<td style="text-align:left;">
3
</td>
<td style="text-align:left;">
gene
</td>
<td style="text-align:right;">
34.68938
</td>
<td style="text-align:right;">
34.69292
</td>
<td style="text-align:left;">
positive
</td>
<td style="text-align:left;">
MGI_C57BL6J_5594100
</td>
<td style="text-align:left;">
Gm34941
</td>
<td style="text-align:left;">
NCBI_Gene:102638356
</td>
<td style="text-align:left;">
MGI:5594100
</td>
<td style="text-align:left;">
lncRNA gene
</td>
<td style="text-align:left;">
predicted gene, 34941
</td>
</tr>
<tr>
<td style="text-align:left;">
3
</td>
<td style="text-align:left;">
gene
</td>
<td style="text-align:right;">
34.69964
</td>
<td style="text-align:right;">
34.71666
</td>
<td style="text-align:left;">
negative
</td>
<td style="text-align:left;">
MGI_C57BL6J_3781322
</td>
<td style="text-align:left;">
Gm3143
</td>
<td style="text-align:left;">
NCBI_Gene:100041108,ENSEMBL:ENSMUSG00000105153
</td>
<td style="text-align:left;">
MGI:3781322
</td>
<td style="text-align:left;">
lncRNA gene
</td>
<td style="text-align:left;">
predicted gene 3143
</td>
</tr>
<tr>
<td style="text-align:left;">
3
</td>
<td style="text-align:left;">
gene
</td>
<td style="text-align:right;">
34.77208
</td>
<td style="text-align:right;">
34.78235
</td>
<td style="text-align:left;">
positive
</td>
<td style="text-align:left;">
MGI_C57BL6J_5621394
</td>
<td style="text-align:left;">
Gm38509
</td>
<td style="text-align:left;">
NCBI_Gene:102638436,ENSEMBL:ENSMUSG00000106519
</td>
<td style="text-align:left;">
MGI:5621394
</td>
<td style="text-align:left;">
lncRNA gene
</td>
<td style="text-align:left;">
predicted gene, 38509
</td>
</tr>
<tr>
<td style="text-align:left;">
3
</td>
<td style="text-align:left;">
gene
</td>
<td style="text-align:right;">
34.82656
</td>
<td style="text-align:right;">
34.87928
</td>
<td style="text-align:left;">
positive
</td>
<td style="text-align:left;">
MGI_C57BL6J_5434743
</td>
<td style="text-align:left;">
Gm21388
</td>
<td style="text-align:left;">
NCBI_Gene:100861998,ENSEMBL:ENSMUSG00000114832
</td>
<td style="text-align:left;">
MGI:5434743
</td>
<td style="text-align:left;">
lncRNA gene
</td>
<td style="text-align:left;">
predicted gene, 21388
</td>
</tr>
<tr>
<td style="text-align:left;">
3
</td>
<td style="text-align:left;">
gene
</td>
<td style="text-align:right;">
34.92254
</td>
<td style="text-align:right;">
34.92265
</td>
<td style="text-align:left;">
negative
</td>
<td style="text-align:left;">
MGI_C57BL6J_5531088
</td>
<td style="text-align:left;">
Mir6378
</td>
<td style="text-align:left;">
miRBase:MI0021909,NCBI_Gene:102466155,ENSEMBL:ENSMUSG00000098862
</td>
<td style="text-align:left;">
MGI:5531088
</td>
<td style="text-align:left;">
miRNA gene
</td>
<td style="text-align:left;">
microRNA 6378
</td>
</tr>
<tr>
<td style="text-align:left;">
3
</td>
<td style="text-align:left;">
gene
</td>
<td style="text-align:right;">
34.95023
</td>
<td style="text-align:right;">
34.95953
</td>
<td style="text-align:left;">
positive
</td>
<td style="text-align:left;">
MGI_C57BL6J_5594414
</td>
<td style="text-align:left;">
Gm35255
</td>
<td style="text-align:left;">
NCBI_Gene:102638771
</td>
<td style="text-align:left;">
MGI:5594414
</td>
<td style="text-align:left;">
lncRNA gene
</td>
<td style="text-align:left;">
predicted gene, 35255
</td>
</tr>
<tr>
<td style="text-align:left;">
3
</td>
<td style="text-align:left;">
gene
</td>
<td style="text-align:right;">
35.25646
</td>
<td style="text-align:right;">
35.27613
</td>
<td style="text-align:left;">
negative
</td>
<td style="text-align:left;">
MGI_C57BL6J_1923648
</td>
<td style="text-align:left;">
1700017M07Rik
</td>
<td style="text-align:left;">
NCBI_Gene:76398,ENSEMBL:ENSMUSG00000105651
</td>
<td style="text-align:left;">
MGI:1923648
</td>
<td style="text-align:left;">
lncRNA gene
</td>
<td style="text-align:left;">
RIKEN cDNA 1700017M07 gene
</td>
</tr>
<tr>
<td style="text-align:left;">
3
</td>
<td style="text-align:left;">
gene
</td>
<td style="text-align:right;">
35.31844
</td>
<td style="text-align:right;">
35.33011
</td>
<td style="text-align:left;">
positive
</td>
<td style="text-align:left;">
MGI_C57BL6J_5594584
</td>
<td style="text-align:left;">
Gm35425
</td>
<td style="text-align:left;">
NCBI_Gene:102639001
</td>
<td style="text-align:left;">
MGI:5594584
</td>
<td style="text-align:left;">
lncRNA gene
</td>
<td style="text-align:left;">
predicted gene, 35425
</td>
</tr>
<tr>
<td style="text-align:left;">
3
</td>
<td style="text-align:left;">
gene
</td>
<td style="text-align:right;">
35.32561
</td>
<td style="text-align:right;">
35.32570
</td>
<td style="text-align:left;">
positive
</td>
<td style="text-align:left;">
MGI_C57BL6J_5455219
</td>
<td style="text-align:left;">
Gm25442
</td>
<td style="text-align:left;">
ENSEMBL:ENSMUSG00000093499
</td>
<td style="text-align:left;">
MGI:5455219
</td>
<td style="text-align:left;">
miRNA gene
</td>
<td style="text-align:left;">
predicted gene, 25442
</td>
</tr>
<tr>
<td style="text-align:left;">
3
</td>
<td style="text-align:left;">
gene
</td>
<td style="text-align:right;">
35.33770
</td>
<td style="text-align:right;">
35.40595
</td>
<td style="text-align:left;">
negative
</td>
<td style="text-align:left;">
MGI_C57BL6J_5621397
</td>
<td style="text-align:left;">
Gm38512
</td>
<td style="text-align:left;">
NCBI_Gene:102638848
</td>
<td style="text-align:left;">
MGI:5621397
</td>
<td style="text-align:left;">
lncRNA gene
</td>
<td style="text-align:left;">
predicted gene, 38512
</td>
</tr>
<tr>
<td style="text-align:left;">
3
</td>
<td style="text-align:left;">
gene
</td>
<td style="text-align:right;">
35.41383
</td>
<td style="text-align:right;">
35.41400
</td>
<td style="text-align:left;">
positive
</td>
<td style="text-align:left;">
MGI_C57BL6J_5663215
</td>
<td style="text-align:left;">
Gm43078
</td>
<td style="text-align:left;">
ENSEMBL:ENSMUSG00000106512
</td>
<td style="text-align:left;">
MGI:5663215
</td>
<td style="text-align:left;">
unclassified gene
</td>
<td style="text-align:left;">
predicted gene 43078
</td>
</tr>
<tr>
<td style="text-align:left;">
3
</td>
<td style="text-align:left;">
pseudogene
</td>
<td style="text-align:right;">
35.53688
</td>
<td style="text-align:right;">
35.53780
</td>
<td style="text-align:left;">
negative
</td>
<td style="text-align:left;">
MGI_C57BL6J_3643628
</td>
<td style="text-align:left;">
Gm7733
</td>
<td style="text-align:left;">
NCBI_Gene:665655,ENSEMBL:ENSMUSG00000106496
</td>
<td style="text-align:left;">
MGI:3643628
</td>
<td style="text-align:left;">
pseudogene
</td>
<td style="text-align:left;">
predicted gene 7733
</td>
</tr>
<tr>
<td style="text-align:left;">
3
</td>
<td style="text-align:left;">
pseudogene
</td>
<td style="text-align:right;">
35.54589
</td>
<td style="text-align:right;">
35.54633
</td>
<td style="text-align:left;">
positive
</td>
<td style="text-align:left;">
MGI_C57BL6J_1347022
</td>
<td style="text-align:left;">
Ube2l3-ps1
</td>
<td style="text-align:left;">
ENSEMBL:ENSMUSG00000105040
</td>
<td style="text-align:left;">
MGI:1347022
</td>
<td style="text-align:left;">
pseudogene
</td>
<td style="text-align:left;">
ubiquitin-conjugating enzyme E2L 3, pseudogene 1
</td>
</tr>
<tr>
<td style="text-align:left;">
3
</td>
<td style="text-align:left;">
gene
</td>
<td style="text-align:right;">
35.59715
</td>
<td style="text-align:right;">
35.66616
</td>
<td style="text-align:left;">
negative
</td>
<td style="text-align:left;">
MGI_C57BL6J_3779620
</td>
<td style="text-align:left;">
Gm6639
</td>
<td style="text-align:left;">
NCBI_Gene:625963,ENSEMBL:ENSMUSG00000105662
</td>
<td style="text-align:left;">
MGI:3779620
</td>
<td style="text-align:left;">
lncRNA gene
</td>
<td style="text-align:left;">
predicted gene 6639
</td>
</tr>
<tr>
<td style="text-align:left;">
3
</td>
<td style="text-align:left;">
gene
</td>
<td style="text-align:right;">
35.73537
</td>
<td style="text-align:right;">
35.73548
</td>
<td style="text-align:left;">
positive
</td>
<td style="text-align:left;">
MGI_C57BL6J_5455473
</td>
<td style="text-align:left;">
Gm25696
</td>
<td style="text-align:left;">
ENSEMBL:ENSMUSG00000064834
</td>
<td style="text-align:left;">
MGI:5455473
</td>
<td style="text-align:left;">
snRNA gene
</td>
<td style="text-align:left;">
predicted gene, 25696
</td>
</tr>
<tr>
<td style="text-align:left;">
3
</td>
<td style="text-align:left;">
gene
</td>
<td style="text-align:right;">
35.75406
</td>
<td style="text-align:right;">
35.85628
</td>
<td style="text-align:left;">
positive
</td>
<td style="text-align:left;">
MGI_C57BL6J_1923545
</td>
<td style="text-align:left;">
Atp11b
</td>
<td style="text-align:left;">
NCBI_Gene:76295,ENSEMBL:ENSMUSG00000037400
</td>
<td style="text-align:left;">
MGI:1923545
</td>
<td style="text-align:left;">
protein coding gene
</td>
<td style="text-align:left;">
ATPase, class VI, type 11B
</td>
</tr>
<tr>
<td style="text-align:left;">
3
</td>
<td style="text-align:left;">
gene
</td>
<td style="text-align:right;">
35.84534
</td>
<td style="text-align:right;">
35.84590
</td>
<td style="text-align:left;">
positive
</td>
<td style="text-align:left;">
MGI_C57BL6J_5663340
</td>
<td style="text-align:left;">
Gm43203
</td>
<td style="text-align:left;">
ENSEMBL:ENSMUSG00000104897
</td>
<td style="text-align:left;">
MGI:5663340
</td>
<td style="text-align:left;">
unclassified gene
</td>
<td style="text-align:left;">
predicted gene 43203
</td>
</tr>
<tr>
<td style="text-align:left;">
3
</td>
<td style="text-align:left;">
gene
</td>
<td style="text-align:right;">
35.86162
</td>
<td style="text-align:right;">
35.86212
</td>
<td style="text-align:left;">
positive
</td>
<td style="text-align:left;">
MGI_C57BL6J_1915850
</td>
<td style="text-align:left;">
1110021P09Rik
</td>
<td style="text-align:left;">
NA
</td>
<td style="text-align:left;">
NA
</td>
<td style="text-align:left;">
unclassified gene
</td>
<td style="text-align:left;">
RIKEN cDNA 1110021P09 gene
</td>
</tr>
<tr>
<td style="text-align:left;">
3
</td>
<td style="text-align:left;">
pseudogene
</td>
<td style="text-align:right;">
35.86386
</td>
<td style="text-align:right;">
35.86507
</td>
<td style="text-align:left;">
positive
</td>
<td style="text-align:left;">
MGI_C57BL6J_3647088
</td>
<td style="text-align:left;">
Gm7741
</td>
<td style="text-align:left;">
NCBI_Gene:665667,ENSEMBL:ENSMUSG00000097713
</td>
<td style="text-align:left;">
MGI:3647088
</td>
<td style="text-align:left;">
pseudogene
</td>
<td style="text-align:left;">
predicted gene 7741
</td>
</tr>
<tr>
<td style="text-align:left;">
3
</td>
<td style="text-align:left;">
pseudogene
</td>
<td style="text-align:right;">
35.87565
</td>
<td style="text-align:right;">
35.88335
</td>
<td style="text-align:left;">
negative
</td>
<td style="text-align:left;">
MGI_C57BL6J_5295701
</td>
<td style="text-align:left;">
Gm20595
</td>
<td style="text-align:left;">
NCBI_Gene:100463514
</td>
<td style="text-align:left;">
MGI:5295701
</td>
<td style="text-align:left;">
pseudogene
</td>
<td style="text-align:left;">
predicted gene, 20595
</td>
</tr>
<tr>
<td style="text-align:left;">
3
</td>
<td style="text-align:left;">
gene
</td>
<td style="text-align:right;">
35.89211
</td>
<td style="text-align:right;">
35.93744
</td>
<td style="text-align:left;">
negative
</td>
<td style="text-align:left;">
MGI_C57BL6J_2150386
</td>
<td style="text-align:left;">
Dcun1d1
</td>
<td style="text-align:left;">
NCBI_Gene:114893,ENSEMBL:ENSMUSG00000027708
</td>
<td style="text-align:left;">
MGI:2150386
</td>
<td style="text-align:left;">
protein coding gene
</td>
<td style="text-align:left;">
DCN1, defective in cullin neddylation 1, domain containing 1 (S. cerevisiae)
</td>
</tr>
<tr>
<td style="text-align:left;">
3
</td>
<td style="text-align:left;">
gene
</td>
<td style="text-align:right;">
35.93304
</td>
<td style="text-align:right;">
35.99084
</td>
<td style="text-align:left;">
positive
</td>
<td style="text-align:left;">
MGI_C57BL6J_3802149
</td>
<td style="text-align:left;">
Gm15952
</td>
<td style="text-align:left;">
NCBI_Gene:102639351,ENSEMBL:ENSMUSG00000085655
</td>
<td style="text-align:left;">
MGI:3802149
</td>
<td style="text-align:left;">
lncRNA gene
</td>
<td style="text-align:left;">
predicted gene 15952
</td>
</tr>
<tr>
<td style="text-align:left;">
3
</td>
<td style="text-align:left;">
gene
</td>
<td style="text-align:right;">
35.95544
</td>
<td style="text-align:right;">
35.95626
</td>
<td style="text-align:left;">
negative
</td>
<td style="text-align:left;">
MGI_C57BL6J_1925753
</td>
<td style="text-align:left;">
8430422M14Rik
</td>
<td style="text-align:left;">
ENSEMBL:ENSMUSG00000105095
</td>
<td style="text-align:left;">
MGI:1925753
</td>
<td style="text-align:left;">
unclassified gene
</td>
<td style="text-align:left;">
RIKEN cDNA 8430422M14 gene
</td>
</tr>
<tr>
<td style="text-align:left;">
3
</td>
<td style="text-align:left;">
gene
</td>
<td style="text-align:right;">
35.95629
</td>
<td style="text-align:right;">
36.00068
</td>
<td style="text-align:left;">
negative
</td>
<td style="text-align:left;">
MGI_C57BL6J_1919289
</td>
<td style="text-align:left;">
Mccc1
</td>
<td style="text-align:left;">
NCBI_Gene:72039,ENSEMBL:ENSMUSG00000027709
</td>
<td style="text-align:left;">
MGI:1919289
</td>
<td style="text-align:left;">
protein coding gene
</td>
<td style="text-align:left;">
methylcrotonoyl-Coenzyme A carboxylase 1 (alpha)
</td>
</tr>
<tr>
<td style="text-align:left;">
3
</td>
<td style="text-align:left;">
gene
</td>
<td style="text-align:right;">
35.96938
</td>
<td style="text-align:right;">
35.97203
</td>
<td style="text-align:left;">
negative
</td>
<td style="text-align:left;">
MGI_C57BL6J_5663217
</td>
<td style="text-align:left;">
Gm43080
</td>
<td style="text-align:left;">
ENSEMBL:ENSMUSG00000105556
</td>
<td style="text-align:left;">
MGI:5663217
</td>
<td style="text-align:left;">
unclassified gene
</td>
<td style="text-align:left;">
predicted gene 43080
</td>
</tr>
<tr>
<td style="text-align:left;">
3
</td>
<td style="text-align:left;">
pseudogene
</td>
<td style="text-align:right;">
35.97377
</td>
<td style="text-align:right;">
35.97409
</td>
<td style="text-align:left;">
positive
</td>
<td style="text-align:left;">
MGI_C57BL6J_3704213
</td>
<td style="text-align:left;">
B230207O21Rik
</td>
<td style="text-align:left;">
ENSEMBL:ENSMUSG00000106096
</td>
<td style="text-align:left;">
MGI:3704213
</td>
<td style="text-align:left;">
pseudogene
</td>
<td style="text-align:left;">
RIKEN cDNA B230207O21 gene
</td>
</tr>
<tr>
<td style="text-align:left;">
3
</td>
<td style="text-align:left;">
gene
</td>
<td style="text-align:right;">
35.99983
</td>
<td style="text-align:right;">
36.02511
</td>
<td style="text-align:left;">
positive
</td>
<td style="text-align:left;">
MGI_C57BL6J_3590669
</td>
<td style="text-align:left;">
Mccc1os
</td>
<td style="text-align:left;">
NCBI_Gene:654424,ENSEMBL:ENSMUSG00000086392
</td>
<td style="text-align:left;">
MGI:3590669
</td>
<td style="text-align:left;">
antisense lncRNA gene
</td>
<td style="text-align:left;">
methylcrotonoyl-Coenzyme A carboxylase 1 (alpha), opposite strand
</td>
</tr>
<tr>
<td style="text-align:left;">
3
</td>
<td style="text-align:left;">
gene
</td>
<td style="text-align:right;">
36.00724
</td>
<td style="text-align:right;">
36.05355
</td>
<td style="text-align:left;">
negative
</td>
<td style="text-align:left;">
MGI_C57BL6J_2681852
</td>
<td style="text-align:left;">
Ccdc144b
</td>
<td style="text-align:left;">
NCBI_Gene:241943,ENSEMBL:ENSMUSG00000047696
</td>
<td style="text-align:left;">
MGI:2681852
</td>
<td style="text-align:left;">
protein coding gene
</td>
<td style="text-align:left;">
coiled-coil domain containing 144B
</td>
</tr>
<tr>
<td style="text-align:left;">
3
</td>
<td style="text-align:left;">
gene
</td>
<td style="text-align:right;">
36.06598
</td>
<td style="text-align:right;">
36.09286
</td>
<td style="text-align:left;">
positive
</td>
<td style="text-align:left;">
MGI_C57BL6J_1914272
</td>
<td style="text-align:left;">
Acad9
</td>
<td style="text-align:left;">
NCBI_Gene:229211,ENSEMBL:ENSMUSG00000027710
</td>
<td style="text-align:left;">
MGI:1914272
</td>
<td style="text-align:left;">
protein coding gene
</td>
<td style="text-align:left;">
acyl-Coenzyme A dehydrogenase family, member 9
</td>
</tr>
<tr>
<td style="text-align:left;">
3
</td>
<td style="text-align:left;">
gene
</td>
<td style="text-align:right;">
36.08886
</td>
<td style="text-align:right;">
36.09145
</td>
<td style="text-align:left;">
positive
</td>
<td style="text-align:left;">
MGI_C57BL6J_5663216
</td>
<td style="text-align:left;">
Gm43079
</td>
<td style="text-align:left;">
ENSEMBL:ENSMUSG00000105055
</td>
<td style="text-align:left;">
MGI:5663216
</td>
<td style="text-align:left;">
lncRNA gene
</td>
<td style="text-align:left;">
predicted gene 43079
</td>
</tr>
<tr>
<td style="text-align:left;">
3
</td>
<td style="text-align:left;">
gene
</td>
<td style="text-align:right;">
36.09718
</td>
<td style="text-align:right;">
36.09914
</td>
<td style="text-align:left;">
positive
</td>
<td style="text-align:left;">
MGI_C57BL6J_5663342
</td>
<td style="text-align:left;">
Gm43205
</td>
<td style="text-align:left;">
ENSEMBL:ENSMUSG00000105419
</td>
<td style="text-align:left;">
MGI:5663342
</td>
<td style="text-align:left;">
unclassified gene
</td>
<td style="text-align:left;">
predicted gene 43205
</td>
</tr>
<tr>
<td style="text-align:left;">
3
</td>
<td style="text-align:left;">
gene
</td>
<td style="text-align:right;">
36.15102
</td>
<td style="text-align:right;">
36.17034
</td>
<td style="text-align:left;">
positive
</td>
<td style="text-align:left;">
MGI_C57BL6J_1098769
</td>
<td style="text-align:left;">
Zfp267
</td>
<td style="text-align:left;">
NCBI_Gene:241944,ENSEMBL:ENSMUSG00000033883
</td>
<td style="text-align:left;">
MGI:1098769
</td>
<td style="text-align:left;">
protein coding gene
</td>
<td style="text-align:left;">
zinc finger protein 267
</td>
</tr>
<tr>
<td style="text-align:left;">
3
</td>
<td style="text-align:left;">
gene
</td>
<td style="text-align:right;">
36.17942
</td>
<td style="text-align:right;">
36.22231
</td>
<td style="text-align:left;">
negative
</td>
<td style="text-align:left;">
MGI_C57BL6J_2677633
</td>
<td style="text-align:left;">
Qrfpr
</td>
<td style="text-align:left;">
NCBI_Gene:229214,ENSEMBL:ENSMUSG00000058400
</td>
<td style="text-align:left;">
MGI:2677633
</td>
<td style="text-align:left;">
protein coding gene
</td>
<td style="text-align:left;">
pyroglutamylated RFamide peptide receptor
</td>
</tr>
<tr>
<td style="text-align:left;">
3
</td>
<td style="text-align:left;">
gene
</td>
<td style="text-align:right;">
36.20329
</td>
<td style="text-align:right;">
36.20562
</td>
<td style="text-align:left;">
positive
</td>
<td style="text-align:left;">
MGI_C57BL6J_5663392
</td>
<td style="text-align:left;">
Gm43255
</td>
<td style="text-align:left;">
ENSEMBL:ENSMUSG00000104568
</td>
<td style="text-align:left;">
MGI:5663392
</td>
<td style="text-align:left;">
lncRNA gene
</td>
<td style="text-align:left;">
predicted gene 43255
</td>
</tr>
<tr>
<td style="text-align:left;">
3
</td>
<td style="text-align:left;">
pseudogene
</td>
<td style="text-align:right;">
36.21381
</td>
<td style="text-align:right;">
36.21476
</td>
<td style="text-align:left;">
negative
</td>
<td style="text-align:left;">
MGI_C57BL6J_5011078
</td>
<td style="text-align:left;">
Gm18893
</td>
<td style="text-align:left;">
NCBI_Gene:100417907,ENSEMBL:ENSMUSG00000106180
</td>
<td style="text-align:left;">
MGI:5011078
</td>
<td style="text-align:left;">
pseudogene
</td>
<td style="text-align:left;">
predicted gene, 18893
</td>
</tr>
<tr>
<td style="text-align:left;">
3
</td>
<td style="text-align:left;">
gene
</td>
<td style="text-align:right;">
36.27233
</td>
<td style="text-align:right;">
36.27896
</td>
<td style="text-align:left;">
negative
</td>
<td style="text-align:left;">
MGI_C57BL6J_5625093
</td>
<td style="text-align:left;">
Gm42208
</td>
<td style="text-align:left;">
NCBI_Gene:105247026,ENSEMBL:ENSMUSG00000106140
</td>
<td style="text-align:left;">
MGI:5625093
</td>
<td style="text-align:left;">
lncRNA gene
</td>
<td style="text-align:left;">
predicted gene, 42208
</td>
</tr>
<tr>
<td style="text-align:left;">
3
</td>
<td style="text-align:left;">
gene
</td>
<td style="text-align:right;">
36.35422
</td>
<td style="text-align:right;">
36.40651
</td>
<td style="text-align:left;">
negative
</td>
<td style="text-align:left;">
MGI_C57BL6J_5625094
</td>
<td style="text-align:left;">
Gm42209
</td>
<td style="text-align:left;">
NCBI_Gene:105247027
</td>
<td style="text-align:left;">
MGI:5625094
</td>
<td style="text-align:left;">
lncRNA gene
</td>
<td style="text-align:left;">
predicted gene, 42209
</td>
</tr>
<tr>
<td style="text-align:left;">
3
</td>
<td style="text-align:left;">
pseudogene
</td>
<td style="text-align:right;">
36.39470
</td>
<td style="text-align:right;">
36.39520
</td>
<td style="text-align:left;">
negative
</td>
<td style="text-align:left;">
MGI_C57BL6J_5010020
</td>
<td style="text-align:left;">
Gm17835
</td>
<td style="text-align:left;">
NCBI_Gene:100415952,ENSEMBL:ENSMUSG00000105281
</td>
<td style="text-align:left;">
MGI:5010020
</td>
<td style="text-align:left;">
pseudogene
</td>
<td style="text-align:left;">
predicted gene, 17835
</td>
</tr>
<tr>
<td style="text-align:left;">
3
</td>
<td style="text-align:left;">
pseudogene
</td>
<td style="text-align:right;">
36.42158
</td>
<td style="text-align:right;">
36.42212
</td>
<td style="text-align:left;">
positive
</td>
<td style="text-align:left;">
MGI_C57BL6J_5010072
</td>
<td style="text-align:left;">
Gm17887
</td>
<td style="text-align:left;">
NCBI_Gene:100416041,ENSEMBL:ENSMUSG00000105792
</td>
<td style="text-align:left;">
MGI:5010072
</td>
<td style="text-align:left;">
pseudogene
</td>
<td style="text-align:left;">
predicted gene, 17887
</td>
</tr>
<tr>
<td style="text-align:left;">
3
</td>
<td style="text-align:left;">
gene
</td>
<td style="text-align:right;">
36.44892
</td>
<td style="text-align:right;">
36.47589
</td>
<td style="text-align:left;">
negative
</td>
<td style="text-align:left;">
MGI_C57BL6J_106008
</td>
<td style="text-align:left;">
Anxa5
</td>
<td style="text-align:left;">
NCBI_Gene:11747,ENSEMBL:ENSMUSG00000027712
</td>
<td style="text-align:left;">
MGI:106008
</td>
<td style="text-align:left;">
protein coding gene
</td>
<td style="text-align:left;">
annexin A5
</td>
</tr>
<tr>
<td style="text-align:left;">
3
</td>
<td style="text-align:left;">
gene
</td>
<td style="text-align:right;">
36.47546
</td>
<td style="text-align:right;">
36.47552
</td>
<td style="text-align:left;">
negative
</td>
<td style="text-align:left;">
MGI_C57BL6J_5530854
</td>
<td style="text-align:left;">
Mir7009
</td>
<td style="text-align:left;">
miRBase:MI0022858,NCBI_Gene:102466786,ENSEMBL:ENSMUSG00000098808
</td>
<td style="text-align:left;">
MGI:5530854
</td>
<td style="text-align:left;">
miRNA gene
</td>
<td style="text-align:left;">
microRNA 7009
</td>
</tr>
<tr>
<td style="text-align:left;">
3
</td>
<td style="text-align:left;">
gene
</td>
<td style="text-align:right;">
36.47594
</td>
<td style="text-align:right;">
36.48230
</td>
<td style="text-align:left;">
positive
</td>
<td style="text-align:left;">
MGI_C57BL6J_1919532
</td>
<td style="text-align:left;">
1810062G17Rik
</td>
<td style="text-align:left;">
NCBI_Gene:72282,ENSEMBL:ENSMUSG00000027713
</td>
<td style="text-align:left;">
MGI:1919532
</td>
<td style="text-align:left;">
protein coding gene
</td>
<td style="text-align:left;">
RIKEN cDNA 1810062G17 gene
</td>
</tr>
<tr>
<td style="text-align:left;">
3
</td>
<td style="text-align:left;">
gene
</td>
<td style="text-align:right;">
36.49980
</td>
<td style="text-align:right;">
36.50681
</td>
<td style="text-align:left;">
negative
</td>
<td style="text-align:left;">
MGI_C57BL6J_3650340
</td>
<td style="text-align:left;">
Gm11548
</td>
<td style="text-align:left;">
NCBI_Gene:329623,ENSEMBL:ENSMUSG00000085320
</td>
<td style="text-align:left;">
MGI:3650340
</td>
<td style="text-align:left;">
lncRNA gene
</td>
<td style="text-align:left;">
predicted gene 11548
</td>
</tr>
<tr>
<td style="text-align:left;">
3
</td>
<td style="text-align:left;">
gene
</td>
<td style="text-align:right;">
36.51506
</td>
<td style="text-align:right;">
36.52181
</td>
<td style="text-align:left;">
negative
</td>
<td style="text-align:left;">
MGI_C57BL6J_3650339
</td>
<td style="text-align:left;">
Gm11549
</td>
<td style="text-align:left;">
NCBI_Gene:100503068,ENSEMBL:ENSMUSG00000085007
</td>
<td style="text-align:left;">
MGI:3650339
</td>
<td style="text-align:left;">
lincRNA gene
</td>
<td style="text-align:left;">
predicted gene 11549
</td>
</tr>
<tr>
<td style="text-align:left;">
3
</td>
<td style="text-align:left;">
gene
</td>
<td style="text-align:right;">
36.52922
</td>
<td style="text-align:right;">
36.53233
</td>
<td style="text-align:left;">
negative
</td>
<td style="text-align:left;">
MGI_C57BL6J_5663674
</td>
<td style="text-align:left;">
Gm43537
</td>
<td style="text-align:left;">
ENSEMBL:ENSMUSG00000105022
</td>
<td style="text-align:left;">
MGI:5663674
</td>
<td style="text-align:left;">
unclassified gene
</td>
<td style="text-align:left;">
predicted gene 43537
</td>
</tr>
<tr>
<td style="text-align:left;">
3
</td>
<td style="text-align:left;">
pseudogene
</td>
<td style="text-align:right;">
36.53917
</td>
<td style="text-align:right;">
36.54010
</td>
<td style="text-align:left;">
negative
</td>
<td style="text-align:left;">
MGI_C57BL6J_3650891
</td>
<td style="text-align:left;">
Gm11550
</td>
<td style="text-align:left;">
NCBI_Gene:665709,ENSEMBL:ENSMUSG00000083388
</td>
<td style="text-align:left;">
MGI:3650891
</td>
<td style="text-align:left;">
pseudogene
</td>
<td style="text-align:left;">
predicted gene 11550
</td>
</tr>
<tr>
<td style="text-align:left;">
3
</td>
<td style="text-align:left;">
gene
</td>
<td style="text-align:right;">
36.55261
</td>
<td style="text-align:right;">
36.56573
</td>
<td style="text-align:left;">
positive
</td>
<td style="text-align:left;">
MGI_C57BL6J_1355319
</td>
<td style="text-align:left;">
Exosc9
</td>
<td style="text-align:left;">
NCBI_Gene:50911,ENSEMBL:ENSMUSG00000027714
</td>
<td style="text-align:left;">
MGI:1355319
</td>
<td style="text-align:left;">
protein coding gene
</td>
<td style="text-align:left;">
exosome component 9
</td>
</tr>
<tr>
<td style="text-align:left;">
3
</td>
<td style="text-align:left;">
gene
</td>
<td style="text-align:right;">
36.56486
</td>
<td style="text-align:right;">
36.57303
</td>
<td style="text-align:left;">
negative
</td>
<td style="text-align:left;">
MGI_C57BL6J_108069
</td>
<td style="text-align:left;">
Ccna2
</td>
<td style="text-align:left;">
NCBI_Gene:12428,ENSEMBL:ENSMUSG00000027715
</td>
<td style="text-align:left;">
MGI:108069
</td>
<td style="text-align:left;">
protein coding gene
</td>
<td style="text-align:left;">
cyclin A2
</td>
</tr>
<tr>
<td style="text-align:left;">
3
</td>
<td style="text-align:left;">
gene
</td>
<td style="text-align:right;">
36.57314
</td>
<td style="text-align:right;">
36.61349
</td>
<td style="text-align:left;">
negative
</td>
<td style="text-align:left;">
MGI_C57BL6J_1918742
</td>
<td style="text-align:left;">
Bbs7
</td>
<td style="text-align:left;">
NCBI_Gene:71492,ENSEMBL:ENSMUSG00000037325
</td>
<td style="text-align:left;">
MGI:1918742
</td>
<td style="text-align:left;">
protein coding gene
</td>
<td style="text-align:left;">
Bardet-Biedl syndrome 7 (human)
</td>
</tr>
<tr>
<td style="text-align:left;">
3
</td>
<td style="text-align:left;">
gene
</td>
<td style="text-align:right;">
36.62048
</td>
<td style="text-align:right;">
36.69022
</td>
<td style="text-align:left;">
negative
</td>
<td style="text-align:left;">
MGI_C57BL6J_109526
</td>
<td style="text-align:left;">
Trpc3
</td>
<td style="text-align:left;">
NCBI_Gene:22065,ENSEMBL:ENSMUSG00000027716
</td>
<td style="text-align:left;">
MGI:109526
</td>
<td style="text-align:left;">
protein coding gene
</td>
<td style="text-align:left;">
transient receptor potential cation channel, subfamily C, member 3
</td>
</tr>
<tr>
<td style="text-align:left;">
3
</td>
<td style="text-align:left;">
gene
</td>
<td style="text-align:right;">
36.65407
</td>
<td style="text-align:right;">
36.65501
</td>
<td style="text-align:left;">
negative
</td>
<td style="text-align:left;">
MGI_C57BL6J_5595015
</td>
<td style="text-align:left;">
Gm35856
</td>
<td style="text-align:left;">
NCBI_Gene:102639576
</td>
<td style="text-align:left;">
MGI:5595015
</td>
<td style="text-align:left;">
lncRNA gene
</td>
<td style="text-align:left;">
predicted gene, 35856
</td>
</tr>
<tr>
<td style="text-align:left;">
3
</td>
<td style="text-align:left;">
pseudogene
</td>
<td style="text-align:right;">
36.69480
</td>
<td style="text-align:right;">
36.69505
</td>
<td style="text-align:left;">
negative
</td>
<td style="text-align:left;">
MGI_C57BL6J_3651607
</td>
<td style="text-align:left;">
Gm12377
</td>
<td style="text-align:left;">
ENSEMBL:ENSMUSG00000082548
</td>
<td style="text-align:left;">
MGI:3651607
</td>
<td style="text-align:left;">
pseudogene
</td>
<td style="text-align:left;">
predicted gene 12377
</td>
</tr>
<tr>
<td style="text-align:left;">
3
</td>
<td style="text-align:left;">
pseudogene
</td>
<td style="text-align:right;">
36.82435
</td>
<td style="text-align:right;">
36.82506
</td>
<td style="text-align:left;">
negative
</td>
<td style="text-align:left;">
MGI_C57BL6J_3651608
</td>
<td style="text-align:left;">
Gm12376
</td>
<td style="text-align:left;">
NCBI_Gene:100417988,ENSEMBL:ENSMUSG00000084027
</td>
<td style="text-align:left;">
MGI:3651608
</td>
<td style="text-align:left;">
pseudogene
</td>
<td style="text-align:left;">
predicted gene 12376
</td>
</tr>
<tr>
<td style="text-align:left;">
3
</td>
<td style="text-align:left;">
gene
</td>
<td style="text-align:right;">
36.86306
</td>
<td style="text-align:right;">
37.05303
</td>
<td style="text-align:left;">
positive
</td>
<td style="text-align:left;">
MGI_C57BL6J_2444631
</td>
<td style="text-align:left;">
4932438A13Rik
</td>
<td style="text-align:left;">
NCBI_Gene:229227,ENSEMBL:ENSMUSG00000037270
</td>
<td style="text-align:left;">
MGI:2444631
</td>
<td style="text-align:left;">
protein coding gene
</td>
<td style="text-align:left;">
RIKEN cDNA 4932438A13 gene
</td>
</tr>
<tr>
<td style="text-align:left;">
3
</td>
<td style="text-align:left;">
gene
</td>
<td style="text-align:right;">
37.05221
</td>
<td style="text-align:right;">
37.06003
</td>
<td style="text-align:left;">
negative
</td>
<td style="text-align:left;">
MGI_C57BL6J_3650531
</td>
<td style="text-align:left;">
Gm12531
</td>
<td style="text-align:left;">
ENSEMBL:ENSMUSG00000086090
</td>
<td style="text-align:left;">
MGI:3650531
</td>
<td style="text-align:left;">
lncRNA gene
</td>
<td style="text-align:left;">
predicted gene 12531
</td>
</tr>
<tr>
<td style="text-align:left;">
3
</td>
<td style="text-align:left;">
gene
</td>
<td style="text-align:right;">
37.06309
</td>
<td style="text-align:right;">
37.12193
</td>
<td style="text-align:left;">
positive
</td>
<td style="text-align:left;">
MGI_C57BL6J_103258
</td>
<td style="text-align:left;">
Adad1
</td>
<td style="text-align:left;">
NCBI_Gene:21744,ENSEMBL:ENSMUSG00000027719
</td>
<td style="text-align:left;">
MGI:103258
</td>
<td style="text-align:left;">
protein coding gene
</td>
<td style="text-align:left;">
adenosine deaminase domain containing 1 (testis specific)
</td>
</tr>
<tr>
<td style="text-align:left;">
3
</td>
<td style="text-align:left;">
pseudogene
</td>
<td style="text-align:right;">
37.07756
</td>
<td style="text-align:right;">
37.07891
</td>
<td style="text-align:left;">
positive
</td>
<td style="text-align:left;">
MGI_C57BL6J_3651489
</td>
<td style="text-align:left;">
Gm12582
</td>
<td style="text-align:left;">
NCBI_Gene:652987,ENSEMBL:ENSMUSG00000081705
</td>
<td style="text-align:left;">
MGI:3651489
</td>
<td style="text-align:left;">
pseudogene
</td>
<td style="text-align:left;">
predicted gene 12582
</td>
</tr>
<tr>
<td style="text-align:left;">
3
</td>
<td style="text-align:left;">
pseudogene
</td>
<td style="text-align:right;">
37.10808
</td>
<td style="text-align:right;">
37.10933
</td>
<td style="text-align:left;">
positive
</td>
<td style="text-align:left;">
MGI_C57BL6J_3651485
</td>
<td style="text-align:left;">
Gm12583
</td>
<td style="text-align:left;">
NCBI_Gene:100416060,ENSEMBL:ENSMUSG00000082878
</td>
<td style="text-align:left;">
MGI:3651485
</td>
<td style="text-align:left;">
pseudogene
</td>
<td style="text-align:left;">
predicted gene 12583
</td>
</tr>
<tr>
<td style="text-align:left;">
3
</td>
<td style="text-align:left;">
pseudogene
</td>
<td style="text-align:right;">
37.11673
</td>
<td style="text-align:right;">
37.11784
</td>
<td style="text-align:left;">
positive
</td>
<td style="text-align:left;">
MGI_C57BL6J_3651486
</td>
<td style="text-align:left;">
Gm12584
</td>
<td style="text-align:left;">
ENSEMBL:ENSMUSG00000083857
</td>
<td style="text-align:left;">
MGI:3651486
</td>
<td style="text-align:left;">
pseudogene
</td>
<td style="text-align:left;">
predicted gene 12584
</td>
</tr>
<tr>
<td style="text-align:left;">
3
</td>
<td style="text-align:left;">
gene
</td>
<td style="text-align:right;">
37.12052
</td>
<td style="text-align:right;">
37.12596
</td>
<td style="text-align:left;">
negative
</td>
<td style="text-align:left;">
MGI_C57BL6J_96548
</td>
<td style="text-align:left;">
Il2
</td>
<td style="text-align:left;">
NCBI_Gene:16183,ENSEMBL:ENSMUSG00000027720
</td>
<td style="text-align:left;">
MGI:96548
</td>
<td style="text-align:left;">
protein coding gene
</td>
<td style="text-align:left;">
interleukin 2
</td>
</tr>
<tr>
<td style="text-align:left;">
3
</td>
<td style="text-align:left;">
gene
</td>
<td style="text-align:right;">
37.16972
</td>
<td style="text-align:right;">
37.17665
</td>
<td style="text-align:left;">
negative
</td>
<td style="text-align:left;">
MGI_C57BL6J_5595159
</td>
<td style="text-align:left;">
Gm36000
</td>
<td style="text-align:left;">
NCBI_Gene:102639768
</td>
<td style="text-align:left;">
MGI:5595159
</td>
<td style="text-align:left;">
lncRNA gene
</td>
<td style="text-align:left;">
predicted gene, 36000
</td>
</tr>
<tr>
<td style="text-align:left;">
3
</td>
<td style="text-align:left;">
gene
</td>
<td style="text-align:right;">
37.18218
</td>
<td style="text-align:right;">
37.18860
</td>
<td style="text-align:left;">
positive
</td>
<td style="text-align:left;">
MGI_C57BL6J_3650618
</td>
<td style="text-align:left;">
Gm12532
</td>
<td style="text-align:left;">
NCBI_Gene:102639848,ENSEMBL:ENSMUSG00000084754
</td>
<td style="text-align:left;">
MGI:3650618
</td>
<td style="text-align:left;">
lncRNA gene
</td>
<td style="text-align:left;">
predicted gene 12532
</td>
</tr>
<tr>
<td style="text-align:left;">
3
</td>
<td style="text-align:left;">
gene
</td>
<td style="text-align:right;">
37.22276
</td>
<td style="text-align:right;">
37.23264
</td>
<td style="text-align:left;">
negative
</td>
<td style="text-align:left;">
MGI_C57BL6J_1890474
</td>
<td style="text-align:left;">
Il21
</td>
<td style="text-align:left;">
NCBI_Gene:60505,ENSEMBL:ENSMUSG00000027718
</td>
<td style="text-align:left;">
MGI:1890474
</td>
<td style="text-align:left;">
protein coding gene
</td>
<td style="text-align:left;">
interleukin 21
</td>
</tr>
<tr>
<td style="text-align:left;">
3
</td>
<td style="text-align:left;">
gene
</td>
<td style="text-align:right;">
37.22538
</td>
<td style="text-align:right;">
37.22711
</td>
<td style="text-align:left;">
positive
</td>
<td style="text-align:left;">
MGI_C57BL6J_5595327
</td>
<td style="text-align:left;">
Gm36168
</td>
<td style="text-align:left;">
NCBI_Gene:102639985
</td>
<td style="text-align:left;">
MGI:5595327
</td>
<td style="text-align:left;">
lncRNA gene
</td>
<td style="text-align:left;">
predicted gene, 36168
</td>
</tr>
<tr>
<td style="text-align:left;">
3
</td>
<td style="text-align:left;">
gene
</td>
<td style="text-align:right;">
37.23133
</td>
<td style="text-align:right;">
37.23247
</td>
<td style="text-align:left;">
positive
</td>
<td style="text-align:left;">
MGI_C57BL6J_3650303
</td>
<td style="text-align:left;">
Gm12534
</td>
<td style="text-align:left;">
ENSEMBL:ENSMUSG00000086517
</td>
<td style="text-align:left;">
MGI:3650303
</td>
<td style="text-align:left;">
lncRNA gene
</td>
<td style="text-align:left;">
predicted gene 12534
</td>
</tr>
<tr>
<td style="text-align:left;">
3
</td>
<td style="text-align:left;">
gene
</td>
<td style="text-align:right;">
37.23287
</td>
<td style="text-align:right;">
37.23334
</td>
<td style="text-align:left;">
positive
</td>
<td style="text-align:left;">
MGI_C57BL6J_5663959
</td>
<td style="text-align:left;">
Gm43822
</td>
<td style="text-align:left;">
ENSEMBL:ENSMUSG00000104998
</td>
<td style="text-align:left;">
MGI:5663959
</td>
<td style="text-align:left;">
unclassified gene
</td>
<td style="text-align:left;">
predicted gene 43822
</td>
</tr>
<tr>
<td style="text-align:left;">
3
</td>
<td style="text-align:left;">
gene
</td>
<td style="text-align:right;">
37.25678
</td>
<td style="text-align:right;">
37.25689
</td>
<td style="text-align:left;">
negative
</td>
<td style="text-align:left;">
MGI_C57BL6J_5453946
</td>
<td style="text-align:left;">
Gm24169
</td>
<td style="text-align:left;">
ENSEMBL:ENSMUSG00000094007
</td>
<td style="text-align:left;">
MGI:5453946
</td>
<td style="text-align:left;">
miRNA gene
</td>
<td style="text-align:left;">
predicted gene, 24169
</td>
</tr>
<tr>
<td style="text-align:left;">
3
</td>
<td style="text-align:left;">
gene
</td>
<td style="text-align:right;">
37.30775
</td>
<td style="text-align:right;">
37.31271
</td>
<td style="text-align:left;">
negative
</td>
<td style="text-align:left;">
MGI_C57BL6J_2677454
</td>
<td style="text-align:left;">
Cetn4
</td>
<td style="text-align:left;">
NCBI_Gene:207175,ENSEMBL:ENSMUSG00000045031
</td>
<td style="text-align:left;">
MGI:2677454
</td>
<td style="text-align:left;">
protein coding gene
</td>
<td style="text-align:left;">
centrin 4
</td>
</tr>
<tr>
<td style="text-align:left;">
3
</td>
<td style="text-align:left;">
gene
</td>
<td style="text-align:right;">
37.31249
</td>
<td style="text-align:right;">
37.32145
</td>
<td style="text-align:left;">
positive
</td>
<td style="text-align:left;">
MGI_C57BL6J_2686651
</td>
<td style="text-align:left;">
Bbs12
</td>
<td style="text-align:left;">
NCBI_Gene:241950,ENSEMBL:ENSMUSG00000051444
</td>
<td style="text-align:left;">
MGI:2686651
</td>
<td style="text-align:left;">
protein coding gene
</td>
<td style="text-align:left;">
Bardet-Biedl syndrome 12 (human)
</td>
</tr>
<tr>
<td style="text-align:left;">
3
</td>
<td style="text-align:left;">
gene
</td>
<td style="text-align:right;">
37.31252
</td>
<td style="text-align:right;">
37.40487
</td>
<td style="text-align:left;">
positive
</td>
<td style="text-align:left;">
MGI_C57BL6J_5663576
</td>
<td style="text-align:left;">
Gm43439
</td>
<td style="text-align:left;">
ENSEMBL:ENSMUSG00000106407
</td>
<td style="text-align:left;">
MGI:5663576
</td>
<td style="text-align:left;">
lncRNA gene
</td>
<td style="text-align:left;">
predicted gene 43439
</td>
</tr>
<tr>
<td style="text-align:left;">
3
</td>
<td style="text-align:left;">
pseudogene
</td>
<td style="text-align:right;">
37.31253
</td>
<td style="text-align:right;">
37.32919
</td>
<td style="text-align:left;">
positive
</td>
<td style="text-align:left;">
MGI_C57BL6J_3651849
</td>
<td style="text-align:left;">
Gm12540
</td>
<td style="text-align:left;">
NCBI_Gene:102640067,ENSEMBL:ENSMUSG00000083353
</td>
<td style="text-align:left;">
MGI:3651849
</td>
<td style="text-align:left;">
pseudogene
</td>
<td style="text-align:left;">
predicted gene 12540
</td>
</tr>
<tr>
<td style="text-align:left;">
3
</td>
<td style="text-align:left;">
gene
</td>
<td style="text-align:right;">
37.33329
</td>
<td style="text-align:right;">
37.35103
</td>
<td style="text-align:left;">
negative
</td>
<td style="text-align:left;">
MGI_C57BL6J_3649376
</td>
<td style="text-align:left;">
Fgf2os
</td>
<td style="text-align:left;">
NCBI_Gene:329626,ENSEMBL:ENSMUSG00000054779
</td>
<td style="text-align:left;">
MGI:3649376
</td>
<td style="text-align:left;">
antisense lncRNA gene
</td>
<td style="text-align:left;">
fibroblast growth factor 2, opposite strand
</td>
</tr>
<tr>
<td style="text-align:left;">
3
</td>
<td style="text-align:left;">
gene
</td>
<td style="text-align:right;">
37.33902
</td>
<td style="text-align:right;">
37.34152
</td>
<td style="text-align:left;">
positive
</td>
<td style="text-align:left;">
MGI_C57BL6J_5663625
</td>
<td style="text-align:left;">
Gm43488
</td>
<td style="text-align:left;">
ENSEMBL:ENSMUSG00000105063
</td>
<td style="text-align:left;">
MGI:5663625
</td>
<td style="text-align:left;">
unclassified gene
</td>
<td style="text-align:left;">
predicted gene 43488
</td>
</tr>
<tr>
<td style="text-align:left;">
3
</td>
<td style="text-align:left;">
gene
</td>
<td style="text-align:right;">
37.34835
</td>
<td style="text-align:right;">
37.41011
</td>
<td style="text-align:left;">
positive
</td>
<td style="text-align:left;">
MGI_C57BL6J_95516
</td>
<td style="text-align:left;">
Fgf2
</td>
<td style="text-align:left;">
NCBI_Gene:14173,ENSEMBL:ENSMUSG00000037225
</td>
<td style="text-align:left;">
MGI:95516
</td>
<td style="text-align:left;">
protein coding gene
</td>
<td style="text-align:left;">
fibroblast growth factor 2
</td>
</tr>
<tr>
<td style="text-align:left;">
3
</td>
<td style="text-align:left;">
gene
</td>
<td style="text-align:right;">
37.39221
</td>
<td style="text-align:right;">
37.39510
</td>
<td style="text-align:left;">
negative
</td>
<td style="text-align:left;">
MGI_C57BL6J_5663060
</td>
<td style="text-align:left;">
Gm42923
</td>
<td style="text-align:left;">
ENSEMBL:ENSMUSG00000105134
</td>
<td style="text-align:left;">
MGI:5663060
</td>
<td style="text-align:left;">
unclassified gene
</td>
<td style="text-align:left;">
predicted gene 42923
</td>
</tr>
<tr>
<td style="text-align:left;">
3
</td>
<td style="text-align:left;">
gene
</td>
<td style="text-align:right;">
37.40491
</td>
<td style="text-align:right;">
37.42021
</td>
<td style="text-align:left;">
negative
</td>
<td style="text-align:left;">
MGI_C57BL6J_2387618
</td>
<td style="text-align:left;">
Nudt6
</td>
<td style="text-align:left;">
NCBI_Gene:229228,ENSEMBL:ENSMUSG00000050174
</td>
<td style="text-align:left;">
MGI:2387618
</td>
<td style="text-align:left;">
protein coding gene
</td>
<td style="text-align:left;">
nudix (nucleoside diphosphate linked moiety X)-type motif 6
</td>
</tr>
<tr>
<td style="text-align:left;">
3
</td>
<td style="text-align:left;">
gene
</td>
<td style="text-align:right;">
37.41990
</td>
<td style="text-align:right;">
37.57910
</td>
<td style="text-align:left;">
positive
</td>
<td style="text-align:left;">
MGI_C57BL6J_1927170
</td>
<td style="text-align:left;">
Spata5
</td>
<td style="text-align:left;">
NCBI_Gene:57815,ENSEMBL:ENSMUSG00000027722
</td>
<td style="text-align:left;">
MGI:1927170
</td>
<td style="text-align:left;">
protein coding gene
</td>
<td style="text-align:left;">
spermatogenesis associated 5
</td>
</tr>
<tr>
<td style="text-align:left;">
3
</td>
<td style="text-align:left;">
gene
</td>
<td style="text-align:right;">
37.48883
</td>
<td style="text-align:right;">
37.49479
</td>
<td style="text-align:left;">
positive
</td>
<td style="text-align:left;">
MGI_C57BL6J_5663621
</td>
<td style="text-align:left;">
Gm43484
</td>
<td style="text-align:left;">
ENSEMBL:ENSMUSG00000106166
</td>
<td style="text-align:left;">
MGI:5663621
</td>
<td style="text-align:left;">
unclassified gene
</td>
<td style="text-align:left;">
predicted gene 43484
</td>
</tr>
<tr>
<td style="text-align:left;">
3
</td>
<td style="text-align:left;">
gene
</td>
<td style="text-align:right;">
37.54188
</td>
<td style="text-align:right;">
37.60239
</td>
<td style="text-align:left;">
negative
</td>
<td style="text-align:left;">
MGI_C57BL6J_5595571
</td>
<td style="text-align:left;">
Gm36412
</td>
<td style="text-align:left;">
NCBI_Gene:102640314
</td>
<td style="text-align:left;">
MGI:5595571
</td>
<td style="text-align:left;">
lncRNA gene
</td>
<td style="text-align:left;">
predicted gene, 36412
</td>
</tr>
<tr>
<td style="text-align:left;">
3
</td>
<td style="text-align:left;">
pseudogene
</td>
<td style="text-align:right;">
37.55618
</td>
<td style="text-align:right;">
37.55650
</td>
<td style="text-align:left;">
negative
</td>
<td style="text-align:left;">
MGI_C57BL6J_3652207
</td>
<td style="text-align:left;">
Gm12564
</td>
<td style="text-align:left;">
ENSEMBL:ENSMUSG00000080723
</td>
<td style="text-align:left;">
MGI:3652207
</td>
<td style="text-align:left;">
pseudogene
</td>
<td style="text-align:left;">
predicted gene 12564
</td>
</tr>
<tr>
<td style="text-align:left;">
3
</td>
<td style="text-align:left;">
pseudogene
</td>
<td style="text-align:right;">
37.59942
</td>
<td style="text-align:right;">
37.60007
</td>
<td style="text-align:left;">
negative
</td>
<td style="text-align:left;">
MGI_C57BL6J_3652206
</td>
<td style="text-align:left;">
Gm12565
</td>
<td style="text-align:left;">
NCBI_Gene:665778,ENSEMBL:ENSMUSG00000081093
</td>
<td style="text-align:left;">
MGI:3652206
</td>
<td style="text-align:left;">
pseudogene
</td>
<td style="text-align:left;">
predicted gene 12565
</td>
</tr>
<tr>
<td style="text-align:left;">
3
</td>
<td style="text-align:left;">
gene
</td>
<td style="text-align:right;">
37.60989
</td>
<td style="text-align:right;">
37.61528
</td>
<td style="text-align:left;">
positive
</td>
<td style="text-align:left;">
MGI_C57BL6J_1923197
</td>
<td style="text-align:left;">
4930594O21Rik
</td>
<td style="text-align:left;">
NCBI_Gene:75947,ENSEMBL:ENSMUSG00000085677
</td>
<td style="text-align:left;">
MGI:1923197
</td>
<td style="text-align:left;">
lncRNA gene
</td>
<td style="text-align:left;">
RIKEN cDNA 4930594O21 gene
</td>
</tr>
<tr>
<td style="text-align:left;">
3
</td>
<td style="text-align:left;">
gene
</td>
<td style="text-align:right;">
37.63858
</td>
<td style="text-align:right;">
37.64066
</td>
<td style="text-align:left;">
negative
</td>
<td style="text-align:left;">
MGI_C57BL6J_5663059
</td>
<td style="text-align:left;">
Gm42922
</td>
<td style="text-align:left;">
ENSEMBL:ENSMUSG00000104605
</td>
<td style="text-align:left;">
MGI:5663059
</td>
<td style="text-align:left;">
unclassified gene
</td>
<td style="text-align:left;">
predicted gene 42922
</td>
</tr>
<tr>
<td style="text-align:left;">
3
</td>
<td style="text-align:left;">
gene
</td>
<td style="text-align:right;">
37.63995
</td>
<td style="text-align:right;">
37.64460
</td>
<td style="text-align:left;">
positive
</td>
<td style="text-align:left;">
MGI_C57BL6J_1345139
</td>
<td style="text-align:left;">
Spry1
</td>
<td style="text-align:left;">
NCBI_Gene:24063,ENSEMBL:ENSMUSG00000037211
</td>
<td style="text-align:left;">
MGI:1345139
</td>
<td style="text-align:left;">
protein coding gene
</td>
<td style="text-align:left;">
sprouty RTK signaling antagonist 1
</td>
</tr>
<tr>
<td style="text-align:left;">
3
</td>
<td style="text-align:left;">
gene
</td>
<td style="text-align:right;">
37.64461
</td>
<td style="text-align:right;">
37.64795
</td>
<td style="text-align:left;">
positive
</td>
<td style="text-align:left;">
MGI_C57BL6J_5663957
</td>
<td style="text-align:left;">
Gm43820
</td>
<td style="text-align:left;">
ENSEMBL:ENSMUSG00000106062
</td>
<td style="text-align:left;">
MGI:5663957
</td>
<td style="text-align:left;">
unclassified gene
</td>
<td style="text-align:left;">
predicted gene 43820
</td>
</tr>
<tr>
<td style="text-align:left;">
3
</td>
<td style="text-align:left;">
gene
</td>
<td style="text-align:right;">
37.65268
</td>
<td style="text-align:right;">
37.65652
</td>
<td style="text-align:left;">
negative
</td>
<td style="text-align:left;">
MGI_C57BL6J_5826436
</td>
<td style="text-align:left;">
Gm46799
</td>
<td style="text-align:left;">
NCBI_Gene:108168864
</td>
<td style="text-align:left;">
MGI:5826436
</td>
<td style="text-align:left;">
lncRNA gene
</td>
<td style="text-align:left;">
predicted gene, 46799
</td>
</tr>
<tr>
<td style="text-align:left;">
3
</td>
<td style="text-align:left;">
gene
</td>
<td style="text-align:right;">
37.69046
</td>
<td style="text-align:right;">
37.69486
</td>
<td style="text-align:left;">
negative
</td>
<td style="text-align:left;">
MGI_C57BL6J_5625096
</td>
<td style="text-align:left;">
Gm42211
</td>
<td style="text-align:left;">
NCBI_Gene:105247029
</td>
<td style="text-align:left;">
MGI:5625096
</td>
<td style="text-align:left;">
lncRNA gene
</td>
<td style="text-align:left;">
predicted gene, 42211
</td>
</tr>
<tr>
<td style="text-align:left;">
3
</td>
<td style="text-align:left;">
gene
</td>
<td style="text-align:right;">
37.70802
</td>
<td style="text-align:right;">
37.71121
</td>
<td style="text-align:left;">
positive
</td>
<td style="text-align:left;">
MGI_C57BL6J_5663551
</td>
<td style="text-align:left;">
Gm43414
</td>
<td style="text-align:left;">
ENSEMBL:ENSMUSG00000104935
</td>
<td style="text-align:left;">
MGI:5663551
</td>
<td style="text-align:left;">
unclassified gene
</td>
<td style="text-align:left;">
predicted gene 43414
</td>
</tr>
<tr>
<td style="text-align:left;">
3
</td>
<td style="text-align:left;">
gene
</td>
<td style="text-align:right;">
37.71419
</td>
<td style="text-align:right;">
37.72449
</td>
<td style="text-align:left;">
negative
</td>
<td style="text-align:left;">
MGI_C57BL6J_3646006
</td>
<td style="text-align:left;">
Gm5148
</td>
<td style="text-align:left;">
NCBI_Gene:381438,ENSEMBL:ENSMUSG00000058174
</td>
<td style="text-align:left;">
MGI:3646006
</td>
<td style="text-align:left;">
protein coding gene
</td>
<td style="text-align:left;">
predicted gene 5148
</td>
</tr>
<tr>
<td style="text-align:left;">
3
</td>
<td style="text-align:left;">
pseudogene
</td>
<td style="text-align:right;">
37.71471
</td>
<td style="text-align:right;">
37.71522
</td>
<td style="text-align:left;">
positive
</td>
<td style="text-align:left;">
MGI_C57BL6J_3705429
</td>
<td style="text-align:left;">
Rps23-ps1
</td>
<td style="text-align:left;">
NCBI_Gene:100038875,ENSEMBL:ENSMUSG00000100755
</td>
<td style="text-align:left;">
MGI:3705429
</td>
<td style="text-align:left;">
pseudogene
</td>
<td style="text-align:left;">
ribosomal protein S23, pseudogene 1
</td>
</tr>
<tr>
<td style="text-align:left;">
3
</td>
<td style="text-align:left;">
gene
</td>
<td style="text-align:right;">
37.72423
</td>
<td style="text-align:right;">
37.74132
</td>
<td style="text-align:left;">
positive
</td>
<td style="text-align:left;">
MGI_C57BL6J_5595779
</td>
<td style="text-align:left;">
Gm36620
</td>
<td style="text-align:left;">
NCBI_Gene:102640592
</td>
<td style="text-align:left;">
MGI:5595779
</td>
<td style="text-align:left;">
lncRNA gene
</td>
<td style="text-align:left;">
predicted gene, 36620
</td>
</tr>
<tr>
<td style="text-align:left;">
3
</td>
<td style="text-align:left;">
pseudogene
</td>
<td style="text-align:right;">
37.73468
</td>
<td style="text-align:right;">
37.73514
</td>
<td style="text-align:left;">
positive
</td>
<td style="text-align:left;">
MGI_C57BL6J_3643738
</td>
<td style="text-align:left;">
Rpl31-ps15
</td>
<td style="text-align:left;">
NCBI_Gene:626367,ENSEMBL:ENSMUSG00000097304
</td>
<td style="text-align:left;">
MGI:3643738
</td>
<td style="text-align:left;">
pseudogene
</td>
<td style="text-align:left;">
ribosomal protein L31, pseudogene 15
</td>
</tr>
<tr>
<td style="text-align:left;">
3
</td>
<td style="text-align:left;">
gene
</td>
<td style="text-align:right;">
37.75320
</td>
<td style="text-align:right;">
37.77440
</td>
<td style="text-align:left;">
positive
</td>
<td style="text-align:left;">
MGI_C57BL6J_5625095
</td>
<td style="text-align:left;">
Gm42210
</td>
<td style="text-align:left;">
NCBI_Gene:105247028
</td>
<td style="text-align:left;">
MGI:5625095
</td>
<td style="text-align:left;">
lncRNA gene
</td>
<td style="text-align:left;">
predicted gene, 42210
</td>
</tr>
<tr>
<td style="text-align:left;">
3
</td>
<td style="text-align:left;">
gene
</td>
<td style="text-align:right;">
37.76104
</td>
<td style="text-align:right;">
37.76730
</td>
<td style="text-align:left;">
positive
</td>
<td style="text-align:left;">
MGI_C57BL6J_5663058
</td>
<td style="text-align:left;">
Gm42921
</td>
<td style="text-align:left;">
ENSEMBL:ENSMUSG00000106441
</td>
<td style="text-align:left;">
MGI:5663058
</td>
<td style="text-align:left;">
lncRNA gene
</td>
<td style="text-align:left;">
predicted gene 42921
</td>
</tr>
<tr>
<td style="text-align:left;">
3
</td>
<td style="text-align:left;">
gene
</td>
<td style="text-align:right;">
37.80254
</td>
<td style="text-align:right;">
37.80265
</td>
<td style="text-align:left;">
negative
</td>
<td style="text-align:left;">
MGI_C57BL6J_5456181
</td>
<td style="text-align:left;">
Gm26404
</td>
<td style="text-align:left;">
ENSEMBL:ENSMUSG00000089123
</td>
<td style="text-align:left;">
MGI:5456181
</td>
<td style="text-align:left;">
snoRNA gene
</td>
<td style="text-align:left;">
predicted gene, 26404
</td>
</tr>
<tr>
<td style="text-align:left;">
3
</td>
<td style="text-align:left;">
gene
</td>
<td style="text-align:right;">
37.82822
</td>
<td style="text-align:right;">
37.84829
</td>
<td style="text-align:left;">
positive
</td>
<td style="text-align:left;">
MGI_C57BL6J_1925294
</td>
<td style="text-align:left;">
4930556A17Rik
</td>
<td style="text-align:left;">
NCBI_Gene:102640520
</td>
<td style="text-align:left;">
MGI:1925294
</td>
<td style="text-align:left;">
lncRNA gene
</td>
<td style="text-align:left;">
RIKEN cDNA 4930556A17 gene
</td>
</tr>
<tr>
<td style="text-align:left;">
3
</td>
<td style="text-align:left;">
gene
</td>
<td style="text-align:right;">
37.83847
</td>
<td style="text-align:right;">
37.83957
</td>
<td style="text-align:left;">
positive
</td>
<td style="text-align:left;">
MGI_C57BL6J_5662889
</td>
<td style="text-align:left;">
Gm42752
</td>
<td style="text-align:left;">
ENSEMBL:ENSMUSG00000106621
</td>
<td style="text-align:left;">
MGI:5662889
</td>
<td style="text-align:left;">
unclassified gene
</td>
<td style="text-align:left;">
predicted gene 42752
</td>
</tr>
<tr>
<td style="text-align:left;">
3
</td>
<td style="text-align:left;">
gene
</td>
<td style="text-align:right;">
37.84180
</td>
<td style="text-align:right;">
37.84307
</td>
<td style="text-align:left;">
positive
</td>
<td style="text-align:left;">
MGI_C57BL6J_5662890
</td>
<td style="text-align:left;">
Gm42753
</td>
<td style="text-align:left;">
ENSEMBL:ENSMUSG00000105252
</td>
<td style="text-align:left;">
MGI:5662890
</td>
<td style="text-align:left;">
unclassified gene
</td>
<td style="text-align:left;">
predicted gene 42753
</td>
</tr>
<tr>
<td style="text-align:left;">
3
</td>
<td style="text-align:left;">
gene
</td>
<td style="text-align:right;">
37.89881
</td>
<td style="text-align:right;">
37.99273
</td>
<td style="text-align:left;">
positive
</td>
<td style="text-align:left;">
MGI_C57BL6J_5434111
</td>
<td style="text-align:left;">
Gm20755
</td>
<td style="text-align:left;">
NCBI_Gene:626410,ENSEMBL:ENSMUSG00000106461
</td>
<td style="text-align:left;">
MGI:5434111
</td>
<td style="text-align:left;">
lncRNA gene
</td>
<td style="text-align:left;">
predicted gene, 20755
</td>
</tr>
<tr>
<td style="text-align:left;">
3
</td>
<td style="text-align:left;">
pseudogene
</td>
<td style="text-align:right;">
37.95961
</td>
<td style="text-align:right;">
37.96009
</td>
<td style="text-align:left;">
negative
</td>
<td style="text-align:left;">
MGI_C57BL6J_3643378
</td>
<td style="text-align:left;">
Gm7799
</td>
<td style="text-align:left;">
ENSEMBL:ENSMUSG00000087457
</td>
<td style="text-align:left;">
MGI:3643378
</td>
<td style="text-align:left;">
pseudogene
</td>
<td style="text-align:left;">
predicted gene 7799
</td>
</tr>
<tr>
<td style="text-align:left;">
3
</td>
<td style="text-align:left;">
gene
</td>
<td style="text-align:right;">
37.98279
</td>
<td style="text-align:right;">
37.98335
</td>
<td style="text-align:left;">
positive
</td>
<td style="text-align:left;">
MGI_C57BL6J_5662888
</td>
<td style="text-align:left;">
Gm42751
</td>
<td style="text-align:left;">
ENSEMBL:ENSMUSG00000105099
</td>
<td style="text-align:left;">
MGI:5662888
</td>
<td style="text-align:left;">
unclassified gene
</td>
<td style="text-align:left;">
predicted gene 42751
</td>
</tr>
<tr>
<td style="text-align:left;">
3
</td>
<td style="text-align:left;">
gene
</td>
<td style="text-align:right;">
37.99511
</td>
<td style="text-align:right;">
37.99524
</td>
<td style="text-align:left;">
negative
</td>
<td style="text-align:left;">
MGI_C57BL6J_5452676
</td>
<td style="text-align:left;">
Gm22899
</td>
<td style="text-align:left;">
ENSEMBL:ENSMUSG00000077728
</td>
<td style="text-align:left;">
MGI:5452676
</td>
<td style="text-align:left;">
snoRNA gene
</td>
<td style="text-align:left;">
predicted gene, 22899
</td>
</tr>
<tr>
<td style="text-align:left;">
3
</td>
<td style="text-align:left;">
gene
</td>
<td style="text-align:right;">
38.09463
</td>
<td style="text-align:right;">
38.13019
</td>
<td style="text-align:left;">
negative
</td>
<td style="text-align:left;">
MGI_C57BL6J_5663958
</td>
<td style="text-align:left;">
Gm43821
</td>
<td style="text-align:left;">
ENSEMBL:ENSMUSG00000105219
</td>
<td style="text-align:left;">
MGI:5663958
</td>
<td style="text-align:left;">
lncRNA gene
</td>
<td style="text-align:left;">
predicted gene 43821
</td>
</tr>
<tr>
<td style="text-align:left;">
3
</td>
<td style="text-align:left;">
pseudogene
</td>
<td style="text-align:right;">
38.10123
</td>
<td style="text-align:right;">
38.10798
</td>
<td style="text-align:left;">
negative
</td>
<td style="text-align:left;">
MGI_C57BL6J_3643358
</td>
<td style="text-align:left;">
Rpl21-ps10
</td>
<td style="text-align:left;">
NCBI_Gene:665821,ENSEMBL:ENSMUSG00000105359
</td>
<td style="text-align:left;">
MGI:3643358
</td>
<td style="text-align:left;">
pseudogene
</td>
<td style="text-align:left;">
ribosomal protein L21, pseudogene 10
</td>
</tr>
<tr>
<td style="text-align:left;">
3
</td>
<td style="text-align:left;">
gene
</td>
<td style="text-align:right;">
38.15191
</td>
<td style="text-align:right;">
38.15915
</td>
<td style="text-align:left;">
positive
</td>
<td style="text-align:left;">
MGI_C57BL6J_5595839
</td>
<td style="text-align:left;">
Gm36680
</td>
<td style="text-align:left;">
NCBI_Gene:102640666
</td>
<td style="text-align:left;">
MGI:5595839
</td>
<td style="text-align:left;">
lncRNA gene
</td>
<td style="text-align:left;">
predicted gene, 36680
</td>
</tr>
<tr>
<td style="text-align:left;">
3
</td>
<td style="text-align:left;">
gene
</td>
<td style="text-align:right;">
38.20315
</td>
<td style="text-align:right;">
38.20812
</td>
<td style="text-align:left;">
negative
</td>
<td style="text-align:left;">
MGI_C57BL6J_1918592
</td>
<td style="text-align:left;">
5430434I15Rik
</td>
<td style="text-align:left;">
NCBI_Gene:71342,ENSEMBL:ENSMUSG00000106002
</td>
<td style="text-align:left;">
MGI:1918592
</td>
<td style="text-align:left;">
lncRNA gene
</td>
<td style="text-align:left;">
RIKEN cDNA 5430434I15 gene
</td>
</tr>
<tr>
<td style="text-align:left;">
3
</td>
<td style="text-align:left;">
pseudogene
</td>
<td style="text-align:right;">
38.22345
</td>
<td style="text-align:right;">
38.22382
</td>
<td style="text-align:left;">
positive
</td>
<td style="text-align:left;">
MGI_C57BL6J_3781143
</td>
<td style="text-align:left;">
Gm2965
</td>
<td style="text-align:left;">
NCBI_Gene:100040785,ENSEMBL:ENSMUSG00000105943
</td>
<td style="text-align:left;">
MGI:3781143
</td>
<td style="text-align:left;">
pseudogene
</td>
<td style="text-align:left;">
predicted gene 2965
</td>
</tr>
<tr>
<td style="text-align:left;">
3
</td>
<td style="text-align:left;">
pseudogene
</td>
<td style="text-align:right;">
38.25402
</td>
<td style="text-align:right;">
38.25480
</td>
<td style="text-align:left;">
positive
</td>
<td style="text-align:left;">
MGI_C57BL6J_5010140
</td>
<td style="text-align:left;">
Gm17955
</td>
<td style="text-align:left;">
NCBI_Gene:100416169
</td>
<td style="text-align:left;">
MGI:5010140
</td>
<td style="text-align:left;">
pseudogene
</td>
<td style="text-align:left;">
predicted gene, 17955
</td>
</tr>
<tr>
<td style="text-align:left;">
3
</td>
<td style="text-align:left;">
gene
</td>
<td style="text-align:right;">
38.26784
</td>
<td style="text-align:right;">
38.26808
</td>
<td style="text-align:left;">
negative
</td>
<td style="text-align:left;">
MGI_C57BL6J_5455351
</td>
<td style="text-align:left;">
Gm25574
</td>
<td style="text-align:left;">
ENSEMBL:ENSMUSG00000092986
</td>
<td style="text-align:left;">
MGI:5455351
</td>
<td style="text-align:left;">
unclassified non-coding RNA gene
</td>
<td style="text-align:left;">
predicted gene, 25574
</td>
</tr>
<tr>
<td style="text-align:left;">
3
</td>
<td style="text-align:left;">
gene
</td>
<td style="text-align:right;">
38.36615
</td>
<td style="text-align:right;">
38.36888
</td>
<td style="text-align:left;">
negative
</td>
<td style="text-align:left;">
MGI_C57BL6J_5625098
</td>
<td style="text-align:left;">
Gm42213
</td>
<td style="text-align:left;">
NCBI_Gene:105247031
</td>
<td style="text-align:left;">
MGI:5625098
</td>
<td style="text-align:left;">
lncRNA gene
</td>
<td style="text-align:left;">
predicted gene, 42213
</td>
</tr>
<tr>
<td style="text-align:left;">
3
</td>
<td style="text-align:left;">
gene
</td>
<td style="text-align:right;">
38.36940
</td>
<td style="text-align:right;">
38.37364
</td>
<td style="text-align:left;">
positive
</td>
<td style="text-align:left;">
MGI_C57BL6J_5663057
</td>
<td style="text-align:left;">
Gm42920
</td>
<td style="text-align:left;">
ENSEMBL:ENSMUSG00000106008
</td>
<td style="text-align:left;">
MGI:5663057
</td>
<td style="text-align:left;">
lncRNA gene
</td>
<td style="text-align:left;">
predicted gene 42920
</td>
</tr>
<tr>
<td style="text-align:left;">
3
</td>
<td style="text-align:left;">
gene
</td>
<td style="text-align:right;">
38.41897
</td>
<td style="text-align:right;">
38.42015
</td>
<td style="text-align:left;">
negative
</td>
<td style="text-align:left;">
MGI_C57BL6J_1917958
</td>
<td style="text-align:left;">
3830422I06Rik
</td>
<td style="text-align:left;">
ENSEMBL:ENSMUSG00000106485
</td>
<td style="text-align:left;">
MGI:1917958
</td>
<td style="text-align:left;">
unclassified gene
</td>
<td style="text-align:left;">
RIKEN cDNA 3830422I06 gene
</td>
</tr>
<tr>
<td style="text-align:left;">
3
</td>
<td style="text-align:left;">
gene
</td>
<td style="text-align:right;">
38.42655
</td>
<td style="text-align:right;">
38.44387
</td>
<td style="text-align:left;">
negative
</td>
<td style="text-align:left;">
MGI_C57BL6J_5595982
</td>
<td style="text-align:left;">
Gm36823
</td>
<td style="text-align:left;">
NCBI_Gene:102640855,ENSEMBL:ENSMUSG00000105516
</td>
<td style="text-align:left;">
MGI:5595982
</td>
<td style="text-align:left;">
lncRNA gene
</td>
<td style="text-align:left;">
predicted gene, 36823
</td>
</tr>
<tr>
<td style="text-align:left;">
3
</td>
<td style="text-align:left;">
pseudogene
</td>
<td style="text-align:right;">
38.43895
</td>
<td style="text-align:right;">
38.43984
</td>
<td style="text-align:left;">
negative
</td>
<td style="text-align:left;">
MGI_C57BL6J_3646285
</td>
<td style="text-align:left;">
Gm7815
</td>
<td style="text-align:left;">
NCBI_Gene:665838,ENSEMBL:ENSMUSG00000106201
</td>
<td style="text-align:left;">
MGI:3646285
</td>
<td style="text-align:left;">
pseudogene
</td>
<td style="text-align:left;">
predicted gene 7815
</td>
</tr>
<tr>
<td style="text-align:left;">
3
</td>
<td style="text-align:left;">
gene
</td>
<td style="text-align:right;">
38.44926
</td>
<td style="text-align:right;">
38.48486
</td>
<td style="text-align:left;">
negative
</td>
<td style="text-align:left;">
MGI_C57BL6J_2139777
</td>
<td style="text-align:left;">
Ankrd50
</td>
<td style="text-align:left;">
NCBI_Gene:99696,ENSEMBL:ENSMUSG00000044864
</td>
<td style="text-align:left;">
MGI:2139777
</td>
<td style="text-align:left;">
protein coding gene
</td>
<td style="text-align:left;">
ankyrin repeat domain 50
</td>
</tr>
<tr>
<td style="text-align:left;">
3
</td>
<td style="text-align:left;">
pseudogene
</td>
<td style="text-align:right;">
38.55797
</td>
<td style="text-align:right;">
38.55886
</td>
<td style="text-align:left;">
negative
</td>
<td style="text-align:left;">
MGI_C57BL6J_3779766
</td>
<td style="text-align:left;">
Gm7824
</td>
<td style="text-align:left;">
NCBI_Gene:665855,ENSEMBL:ENSMUSG00000105360
</td>
<td style="text-align:left;">
MGI:3779766
</td>
<td style="text-align:left;">
pseudogene
</td>
<td style="text-align:left;">
predicted gene 7824
</td>
</tr>
<tr>
<td style="text-align:left;">
3
</td>
<td style="text-align:left;">
gene
</td>
<td style="text-align:right;">
38.55811
</td>
<td style="text-align:right;">
38.55821
</td>
<td style="text-align:left;">
negative
</td>
<td style="text-align:left;">
MGI_C57BL6J_5454239
</td>
<td style="text-align:left;">
Gm24462
</td>
<td style="text-align:left;">
ENSEMBL:ENSMUSG00000076385
</td>
<td style="text-align:left;">
MGI:5454239
</td>
<td style="text-align:left;">
miRNA gene
</td>
<td style="text-align:left;">
predicted gene, 24462
</td>
</tr>
<tr>
<td style="text-align:left;">
3
</td>
<td style="text-align:left;">
gene
</td>
<td style="text-align:right;">
38.61425
</td>
<td style="text-align:right;">
38.61556
</td>
<td style="text-align:left;">
positive
</td>
<td style="text-align:left;">
MGI_C57BL6J_5663675
</td>
<td style="text-align:left;">
Gm43538
</td>
<td style="text-align:left;">
ENSEMBL:ENSMUSG00000106168
</td>
<td style="text-align:left;">
MGI:5663675
</td>
<td style="text-align:left;">
lncRNA gene
</td>
<td style="text-align:left;">
predicted gene 43538
</td>
</tr>
<tr>
<td style="text-align:left;">
3
</td>
<td style="text-align:left;">
gene
</td>
<td style="text-align:right;">
38.66361
</td>
<td style="text-align:right;">
38.70922
</td>
<td style="text-align:left;">
positive
</td>
<td style="text-align:left;">
MGI_C57BL6J_5589038
</td>
<td style="text-align:left;">
Gm29879
</td>
<td style="text-align:left;">
NCBI_Gene:102631576
</td>
<td style="text-align:left;">
MGI:5589038
</td>
<td style="text-align:left;">
lncRNA gene
</td>
<td style="text-align:left;">
predicted gene, 29879
</td>
</tr>
<tr>
<td style="text-align:left;">
3
</td>
<td style="text-align:left;">
gene
</td>
<td style="text-align:right;">
38.76048
</td>
<td style="text-align:right;">
38.76195
</td>
<td style="text-align:left;">
negative
</td>
<td style="text-align:left;">
MGI_C57BL6J_5589118
</td>
<td style="text-align:left;">
Gm29959
</td>
<td style="text-align:left;">
NCBI_Gene:102631683
</td>
<td style="text-align:left;">
MGI:5589118
</td>
<td style="text-align:left;">
lncRNA gene
</td>
<td style="text-align:left;">
predicted gene, 29959
</td>
</tr>
<tr>
<td style="text-align:left;">
3
</td>
<td style="text-align:left;">
gene
</td>
<td style="text-align:right;">
38.87457
</td>
<td style="text-align:right;">
38.88826
</td>
<td style="text-align:left;">
negative
</td>
<td style="text-align:left;">
MGI_C57BL6J_2441892
</td>
<td style="text-align:left;">
C230034O21Rik
</td>
<td style="text-align:left;">
NCBI_Gene:102632538,ENSEMBL:ENSMUSG00000086181
</td>
<td style="text-align:left;">
MGI:2441892
</td>
<td style="text-align:left;">
lncRNA gene
</td>
<td style="text-align:left;">
RIKEN cDNA C230034O21 gene
</td>
</tr>
<tr>
<td style="text-align:left;">
3
</td>
<td style="text-align:left;">
gene
</td>
<td style="text-align:right;">
38.88563
</td>
<td style="text-align:right;">
39.01199
</td>
<td style="text-align:left;">
positive
</td>
<td style="text-align:left;">
MGI_C57BL6J_3045256
</td>
<td style="text-align:left;">
Fat4
</td>
<td style="text-align:left;">
NCBI_Gene:329628,ENSEMBL:ENSMUSG00000046743
</td>
<td style="text-align:left;">
MGI:3045256
</td>
<td style="text-align:left;">
protein coding gene
</td>
<td style="text-align:left;">
FAT atypical cadherin 4
</td>
</tr>
<tr>
<td style="text-align:left;">
3
</td>
<td style="text-align:left;">
gene
</td>
<td style="text-align:right;">
39.00860
</td>
<td style="text-align:right;">
39.00968
</td>
<td style="text-align:left;">
negative
</td>
<td style="text-align:left;">
MGI_C57BL6J_5663676
</td>
<td style="text-align:left;">
Gm43539
</td>
<td style="text-align:left;">
ENSEMBL:ENSMUSG00000105910
</td>
<td style="text-align:left;">
MGI:5663676
</td>
<td style="text-align:left;">
lncRNA gene
</td>
<td style="text-align:left;">
predicted gene 43539
</td>
</tr>
<tr>
<td style="text-align:left;">
3
</td>
<td style="text-align:left;">
pseudogene
</td>
<td style="text-align:right;">
39.12853
</td>
<td style="text-align:right;">
39.12891
</td>
<td style="text-align:left;">
negative
</td>
<td style="text-align:left;">
MGI_C57BL6J_5663445
</td>
<td style="text-align:left;">
Gm43308
</td>
<td style="text-align:left;">
ENSEMBL:ENSMUSG00000104792
</td>
<td style="text-align:left;">
MGI:5663445
</td>
<td style="text-align:left;">
pseudogene
</td>
<td style="text-align:left;">
predicted gene 43308
</td>
</tr>
<tr>
<td style="text-align:left;">
3
</td>
<td style="text-align:left;">
gene
</td>
<td style="text-align:right;">
39.15083
</td>
<td style="text-align:right;">
39.15967
</td>
<td style="text-align:left;">
negative
</td>
<td style="text-align:left;">
MGI_C57BL6J_5589170
</td>
<td style="text-align:left;">
Gm30011
</td>
<td style="text-align:left;">
NCBI_Gene:102631750
</td>
<td style="text-align:left;">
MGI:5589170
</td>
<td style="text-align:left;">
lncRNA gene
</td>
<td style="text-align:left;">
predicted gene, 30011
</td>
</tr>
<tr>
<td style="text-align:left;">
3
</td>
<td style="text-align:left;">
pseudogene
</td>
<td style="text-align:right;">
39.21756
</td>
<td style="text-align:right;">
39.21785
</td>
<td style="text-align:left;">
negative
</td>
<td style="text-align:left;">
MGI_C57BL6J_5663145
</td>
<td style="text-align:left;">
Gm43008
</td>
<td style="text-align:left;">
ENSEMBL:ENSMUSG00000107333
</td>
<td style="text-align:left;">
MGI:5663145
</td>
<td style="text-align:left;">
pseudogene
</td>
<td style="text-align:left;">
predicted gene 43008
</td>
</tr>
<tr>
<td style="text-align:left;">
3
</td>
<td style="text-align:left;">
gene
</td>
<td style="text-align:right;">
39.27713
</td>
<td style="text-align:right;">
39.30593
</td>
<td style="text-align:left;">
positive
</td>
<td style="text-align:left;">
MGI_C57BL6J_5625099
</td>
<td style="text-align:left;">
Gm42214
</td>
<td style="text-align:left;">
NCBI_Gene:105247032
</td>
<td style="text-align:left;">
MGI:5625099
</td>
<td style="text-align:left;">
lncRNA gene
</td>
<td style="text-align:left;">
predicted gene, 42214
</td>
</tr>
<tr>
<td style="text-align:left;">
3
</td>
<td style="text-align:left;">
pseudogene
</td>
<td style="text-align:right;">
39.35737
</td>
<td style="text-align:right;">
39.35913
</td>
<td style="text-align:left;">
negative
</td>
<td style="text-align:left;">
MGI_C57BL6J_3704215
</td>
<td style="text-align:left;">
Gm9845
</td>
<td style="text-align:left;">
NCBI_Gene:100040853,ENSEMBL:ENSMUSG00000050533
</td>
<td style="text-align:left;">
MGI:3704215
</td>
<td style="text-align:left;">
pseudogene
</td>
<td style="text-align:left;">
predicted pseudogene 9845
</td>
</tr>
<tr>
<td style="text-align:left;">
3
</td>
<td style="text-align:left;">
pseudogene
</td>
<td style="text-align:right;">
39.44389
</td>
<td style="text-align:right;">
39.44435
</td>
<td style="text-align:left;">
positive
</td>
<td style="text-align:left;">
MGI_C57BL6J_5295673
</td>
<td style="text-align:left;">
Gm20566
</td>
<td style="text-align:left;">
NCBI_Gene:100047050
</td>
<td style="text-align:left;">
MGI:5295673
</td>
<td style="text-align:left;">
pseudogene
</td>
<td style="text-align:left;">
predicted gene, 20566
</td>
</tr>
<tr>
<td style="text-align:left;">
3
</td>
<td style="text-align:left;">
gene
</td>
<td style="text-align:right;">
39.46078
</td>
<td style="text-align:right;">
39.59865
</td>
<td style="text-align:left;">
positive
</td>
<td style="text-align:left;">
MGI_C57BL6J_5589222
</td>
<td style="text-align:left;">
Gm30063
</td>
<td style="text-align:left;">
NCBI_Gene:102631823
</td>
<td style="text-align:left;">
MGI:5589222
</td>
<td style="text-align:left;">
lncRNA gene
</td>
<td style="text-align:left;">
predicted gene, 30063
</td>
</tr>
<tr>
<td style="text-align:left;">
3
</td>
<td style="text-align:left;">
gene
</td>
<td style="text-align:right;">
39.52552
</td>
<td style="text-align:right;">
39.53415
</td>
<td style="text-align:left;">
negative
</td>
<td style="text-align:left;">
MGI_C57BL6J_5625100
</td>
<td style="text-align:left;">
Gm42215
</td>
<td style="text-align:left;">
NCBI_Gene:105247033
</td>
<td style="text-align:left;">
MGI:5625100
</td>
<td style="text-align:left;">
lncRNA gene
</td>
<td style="text-align:left;">
predicted gene, 42215
</td>
</tr>
<tr>
<td style="text-align:left;">
3
</td>
<td style="text-align:left;">
gene
</td>
<td style="text-align:right;">
39.63596
</td>
<td style="text-align:right;">
39.63608
</td>
<td style="text-align:left;">
negative
</td>
<td style="text-align:left;">
MGI_C57BL6J_4422060
</td>
<td style="text-align:left;">
n-R5s195
</td>
<td style="text-align:left;">
ENSEMBL:ENSMUSG00000093952
</td>
<td style="text-align:left;">
MGI:4422060
</td>
<td style="text-align:left;">
rRNA gene
</td>
<td style="text-align:left;">
nuclear encoded rRNA 5S 195
</td>
</tr>
<tr>
<td style="text-align:left;">
3
</td>
<td style="text-align:left;">
gene
</td>
<td style="text-align:right;">
39.63606
</td>
<td style="text-align:right;">
39.63878
</td>
<td style="text-align:left;">
positive
</td>
<td style="text-align:left;">
MGI_C57BL6J_5662918
</td>
<td style="text-align:left;">
Gm42781
</td>
<td style="text-align:left;">
ENSEMBL:ENSMUSG00000106786
</td>
<td style="text-align:left;">
MGI:5662918
</td>
<td style="text-align:left;">
unclassified gene
</td>
<td style="text-align:left;">
predicted gene 42781
</td>
</tr>
<tr>
<td style="text-align:left;">
3
</td>
<td style="text-align:left;">
gene
</td>
<td style="text-align:right;">
39.66629
</td>
<td style="text-align:right;">
39.66645
</td>
<td style="text-align:left;">
positive
</td>
<td style="text-align:left;">
MGI_C57BL6J_5453573
</td>
<td style="text-align:left;">
Gm23796
</td>
<td style="text-align:left;">
ENSEMBL:ENSMUSG00000065945
</td>
<td style="text-align:left;">
MGI:5453573
</td>
<td style="text-align:left;">
snRNA gene
</td>
<td style="text-align:left;">
predicted gene, 23796
</td>
</tr>
<tr>
<td style="text-align:left;">
3
</td>
<td style="text-align:left;">
gene
</td>
<td style="text-align:right;">
39.71315
</td>
<td style="text-align:right;">
39.72565
</td>
<td style="text-align:left;">
positive
</td>
<td style="text-align:left;">
MGI_C57BL6J_5826471
</td>
<td style="text-align:left;">
Gm46834
</td>
<td style="text-align:left;">
NCBI_Gene:108168914
</td>
<td style="text-align:left;">
MGI:5826471
</td>
<td style="text-align:left;">
lncRNA gene
</td>
<td style="text-align:left;">
predicted gene, 46834
</td>
</tr>
<tr>
<td style="text-align:left;">
3
</td>
<td style="text-align:left;">
gene
</td>
<td style="text-align:right;">
39.79963
</td>
<td style="text-align:right;">
39.79972
</td>
<td style="text-align:left;">
negative
</td>
<td style="text-align:left;">
MGI_C57BL6J_5453608
</td>
<td style="text-align:left;">
Gm23831
</td>
<td style="text-align:left;">
ENSEMBL:ENSMUSG00000089520
</td>
<td style="text-align:left;">
MGI:5453608
</td>
<td style="text-align:left;">
snoRNA gene
</td>
<td style="text-align:left;">
predicted gene, 23831
</td>
</tr>
<tr>
<td style="text-align:left;">
3
</td>
<td style="text-align:left;">
pseudogene
</td>
<td style="text-align:right;">
39.82048
</td>
<td style="text-align:right;">
39.82076
</td>
<td style="text-align:left;">
negative
</td>
<td style="text-align:left;">
MGI_C57BL6J_5662923
</td>
<td style="text-align:left;">
Gm42786
</td>
<td style="text-align:left;">
ENSEMBL:ENSMUSG00000106846
</td>
<td style="text-align:left;">
MGI:5662923
</td>
<td style="text-align:left;">
pseudogene
</td>
<td style="text-align:left;">
predicted gene 42786
</td>
</tr>
<tr>
<td style="text-align:left;">
3
</td>
<td style="text-align:left;">
pseudogene
</td>
<td style="text-align:right;">
39.82358
</td>
<td style="text-align:right;">
39.82741
</td>
<td style="text-align:left;">
positive
</td>
<td style="text-align:left;">
MGI_C57BL6J_5662922
</td>
<td style="text-align:left;">
Gm42785
</td>
<td style="text-align:left;">
ENSEMBL:ENSMUSG00000106948
</td>
<td style="text-align:left;">
MGI:5662922
</td>
<td style="text-align:left;">
pseudogene
</td>
<td style="text-align:left;">
predicted gene 42785
</td>
</tr>
<tr>
<td style="text-align:left;">
3
</td>
<td style="text-align:left;">
gene
</td>
<td style="text-align:right;">
40.42571
</td>
<td style="text-align:right;">
40.45380
</td>
<td style="text-align:left;">
negative
</td>
<td style="text-align:left;">
MGI_C57BL6J_5589343
</td>
<td style="text-align:left;">
Gm30184
</td>
<td style="text-align:left;">
NCBI_Gene:102632001
</td>
<td style="text-align:left;">
MGI:5589343
</td>
<td style="text-align:left;">
lncRNA gene
</td>
<td style="text-align:left;">
predicted gene, 30184
</td>
</tr>
<tr>
<td style="text-align:left;">
3
</td>
<td style="text-align:left;">
gene
</td>
<td style="text-align:right;">
40.45843
</td>
<td style="text-align:right;">
40.46115
</td>
<td style="text-align:left;">
positive
</td>
<td style="text-align:left;">
MGI_C57BL6J_5625101
</td>
<td style="text-align:left;">
Gm42216
</td>
<td style="text-align:left;">
NCBI_Gene:105247034
</td>
<td style="text-align:left;">
MGI:5625101
</td>
<td style="text-align:left;">
lncRNA gene
</td>
<td style="text-align:left;">
predicted gene, 42216
</td>
</tr>
<tr>
<td style="text-align:left;">
3
</td>
<td style="text-align:left;">
gene
</td>
<td style="text-align:right;">
40.48090
</td>
<td style="text-align:right;">
40.48573
</td>
<td style="text-align:left;">
positive
</td>
<td style="text-align:left;">
MGI_C57BL6J_5625102
</td>
<td style="text-align:left;">
Gm42217
</td>
<td style="text-align:left;">
NCBI_Gene:105247035
</td>
<td style="text-align:left;">
MGI:5625102
</td>
<td style="text-align:left;">
lncRNA gene
</td>
<td style="text-align:left;">
predicted gene, 42217
</td>
</tr>
<tr>
<td style="text-align:left;">
3
</td>
<td style="text-align:left;">
gene
</td>
<td style="text-align:right;">
40.48702
</td>
<td style="text-align:right;">
40.48993
</td>
<td style="text-align:left;">
positive
</td>
<td style="text-align:left;">
MGI_C57BL6J_5826438
</td>
<td style="text-align:left;">
Gm46801
</td>
<td style="text-align:left;">
NCBI_Gene:108168866
</td>
<td style="text-align:left;">
MGI:5826438
</td>
<td style="text-align:left;">
lncRNA gene
</td>
<td style="text-align:left;">
predicted gene, 46801
</td>
</tr>
<tr>
<td style="text-align:left;">
3
</td>
<td style="text-align:left;">
gene
</td>
<td style="text-align:right;">
40.50487
</td>
<td style="text-align:right;">
40.52291
</td>
<td style="text-align:left;">
negative
</td>
<td style="text-align:left;">
MGI_C57BL6J_1919477
</td>
<td style="text-align:left;">
1700017G19Rik
</td>
<td style="text-align:left;">
NCBI_Gene:100040882,ENSEMBL:ENSMUSG00000037884
</td>
<td style="text-align:left;">
MGI:1919477
</td>
<td style="text-align:left;">
lncRNA gene
</td>
<td style="text-align:left;">
RIKEN cDNA 1700017G19 gene
</td>
</tr>
<tr>
<td style="text-align:left;">
3
</td>
<td style="text-align:left;">
gene
</td>
<td style="text-align:right;">
40.50998
</td>
<td style="text-align:right;">
40.51183
</td>
<td style="text-align:left;">
positive
</td>
<td style="text-align:left;">
MGI_C57BL6J_5826437
</td>
<td style="text-align:left;">
Gm46800
</td>
<td style="text-align:left;">
NCBI_Gene:108168865
</td>
<td style="text-align:left;">
MGI:5826437
</td>
<td style="text-align:left;">
lncRNA gene
</td>
<td style="text-align:left;">
predicted gene, 46800
</td>
</tr>
<tr>
<td style="text-align:left;">
3
</td>
<td style="text-align:left;">
gene
</td>
<td style="text-align:right;">
40.51882
</td>
<td style="text-align:right;">
40.51889
</td>
<td style="text-align:left;">
positive
</td>
<td style="text-align:left;">
MGI_C57BL6J_5453587
</td>
<td style="text-align:left;">
Gm23810
</td>
<td style="text-align:left;">
ENSEMBL:ENSMUSG00000092891
</td>
<td style="text-align:left;">
MGI:5453587
</td>
<td style="text-align:left;">
miRNA gene
</td>
<td style="text-align:left;">
predicted gene, 23810
</td>
</tr>
<tr>
<td style="text-align:left;">
3
</td>
<td style="text-align:left;">
gene
</td>
<td style="text-align:right;">
40.53129
</td>
<td style="text-align:right;">
40.70478
</td>
<td style="text-align:left;">
positive
</td>
<td style="text-align:left;">
MGI_C57BL6J_2443752
</td>
<td style="text-align:left;">
Intu
</td>
<td style="text-align:left;">
NCBI_Gene:380614,ENSEMBL:ENSMUSG00000060798
</td>
<td style="text-align:left;">
MGI:2443752
</td>
<td style="text-align:left;">
protein coding gene
</td>
<td style="text-align:left;">
inturned planar cell polarity protein
</td>
</tr>
<tr>
<td style="text-align:left;">
3
</td>
<td style="text-align:left;">
pseudogene
</td>
<td style="text-align:right;">
40.65887
</td>
<td style="text-align:right;">
40.65917
</td>
<td style="text-align:left;">
negative
</td>
<td style="text-align:left;">
MGI_C57BL6J_3801939
</td>
<td style="text-align:left;">
Gm16114
</td>
<td style="text-align:left;">
ENSEMBL:ENSMUSG00000089932
</td>
<td style="text-align:left;">
MGI:3801939
</td>
<td style="text-align:left;">
pseudogene
</td>
<td style="text-align:left;">
predicted gene 16114
</td>
</tr>
<tr>
<td style="text-align:left;">
3
</td>
<td style="text-align:left;">
gene
</td>
<td style="text-align:right;">
40.70885
</td>
<td style="text-align:right;">
40.72610
</td>
<td style="text-align:left;">
positive
</td>
<td style="text-align:left;">
MGI_C57BL6J_1920583
</td>
<td style="text-align:left;">
Slc25a31
</td>
<td style="text-align:left;">
NCBI_Gene:73333,ENSEMBL:ENSMUSG00000069041
</td>
<td style="text-align:left;">
MGI:1920583
</td>
<td style="text-align:left;">
protein coding gene
</td>
<td style="text-align:left;">
solute carrier family 25 (mitochondrial carrier, adenine nucleotide translocator), member 31
</td>
</tr>
<tr>
<td style="text-align:left;">
3
</td>
<td style="text-align:left;">
gene
</td>
<td style="text-align:right;">
40.74440
</td>
<td style="text-align:right;">
40.79610
</td>
<td style="text-align:left;">
positive
</td>
<td style="text-align:left;">
MGI_C57BL6J_107422
</td>
<td style="text-align:left;">
Hspa4l
</td>
<td style="text-align:left;">
NCBI_Gene:18415,ENSEMBL:ENSMUSG00000025757
</td>
<td style="text-align:left;">
MGI:107422
</td>
<td style="text-align:left;">
protein coding gene
</td>
<td style="text-align:left;">
heat shock protein 4 like
</td>
</tr>
<tr>
<td style="text-align:left;">
3
</td>
<td style="text-align:left;">
gene
</td>
<td style="text-align:right;">
40.79364
</td>
<td style="text-align:right;">
40.79846
</td>
<td style="text-align:left;">
positive
</td>
<td style="text-align:left;">
MGI_C57BL6J_1918107
</td>
<td style="text-align:left;">
4921509J17Rik
</td>
<td style="text-align:left;">
NA
</td>
<td style="text-align:left;">
NA
</td>
<td style="text-align:left;">
unclassified gene
</td>
<td style="text-align:left;">
RIKEN cDNA 4921509J17 gene
</td>
</tr>
<tr>
<td style="text-align:left;">
3
</td>
<td style="text-align:left;">
gene
</td>
<td style="text-align:right;">
40.79995
</td>
<td style="text-align:right;">
40.81688
</td>
<td style="text-align:left;">
positive
</td>
<td style="text-align:left;">
MGI_C57BL6J_101783
</td>
<td style="text-align:left;">
Plk4
</td>
<td style="text-align:left;">
NCBI_Gene:20873,ENSEMBL:ENSMUSG00000025758
</td>
<td style="text-align:left;">
MGI:101783
</td>
<td style="text-align:left;">
protein coding gene
</td>
<td style="text-align:left;">
polo like kinase 4
</td>
</tr>
<tr>
<td style="text-align:left;">
3
</td>
<td style="text-align:left;">
gene
</td>
<td style="text-align:right;">
40.81810
</td>
<td style="text-align:right;">
40.84689
</td>
<td style="text-align:left;">
negative
</td>
<td style="text-align:left;">
MGI_C57BL6J_1919425
</td>
<td style="text-align:left;">
Mfsd8
</td>
<td style="text-align:left;">
NCBI_Gene:72175,ENSEMBL:ENSMUSG00000025759
</td>
<td style="text-align:left;">
MGI:1919425
</td>
<td style="text-align:left;">
protein coding gene
</td>
<td style="text-align:left;">
major facilitator superfamily domain containing 8
</td>
</tr>
<tr>
<td style="text-align:left;">
3
</td>
<td style="text-align:left;">
pseudogene
</td>
<td style="text-align:right;">
40.84682
</td>
<td style="text-align:right;">
40.87561
</td>
<td style="text-align:left;">
positive
</td>
<td style="text-align:left;">
MGI_C57BL6J_3780180
</td>
<td style="text-align:left;">
Gm2011
</td>
<td style="text-align:left;">
NCBI_Gene:100039027,ENSEMBL:ENSMUSG00000107741
</td>
<td style="text-align:left;">
MGI:3780180
</td>
<td style="text-align:left;">
pseudogene
</td>
<td style="text-align:left;">
predicted gene 2011
</td>
</tr>
<tr>
<td style="text-align:left;">
3
</td>
<td style="text-align:left;">
gene
</td>
<td style="text-align:right;">
40.84697
</td>
<td style="text-align:right;">
40.93814
</td>
<td style="text-align:left;">
positive
</td>
<td style="text-align:left;">
MGI_C57BL6J_1915468
</td>
<td style="text-align:left;">
Abhd18
</td>
<td style="text-align:left;">
NCBI_Gene:269423,ENSEMBL:ENSMUSG00000037818
</td>
<td style="text-align:left;">
MGI:1915468
</td>
<td style="text-align:left;">
protein coding gene
</td>
<td style="text-align:left;">
abhydrolase domain containing 18
</td>
</tr>
<tr>
<td style="text-align:left;">
3
</td>
<td style="text-align:left;">
pseudogene
</td>
<td style="text-align:right;">
40.85348
</td>
<td style="text-align:right;">
40.85529
</td>
<td style="text-align:left;">
negative
</td>
<td style="text-align:left;">
MGI_C57BL6J_3795712
</td>
<td style="text-align:left;">
Gm10731
</td>
<td style="text-align:left;">
NCBI_Gene:100039043,ENSEMBL:ENSMUSG00000074620
</td>
<td style="text-align:left;">
MGI:3795712
</td>
<td style="text-align:left;">
pseudogene
</td>
<td style="text-align:left;">
predicted gene 10731
</td>
</tr>
<tr>
<td style="text-align:left;">
3
</td>
<td style="text-align:left;">
gene
</td>
<td style="text-align:right;">
40.86395
</td>
<td style="text-align:right;">
40.86505
</td>
<td style="text-align:left;">
negative
</td>
<td style="text-align:left;">
MGI_C57BL6J_5690528
</td>
<td style="text-align:left;">
Gm44136
</td>
<td style="text-align:left;">
ENSEMBL:ENSMUSG00000107524
</td>
<td style="text-align:left;">
MGI:5690528
</td>
<td style="text-align:left;">
unclassified gene
</td>
<td style="text-align:left;">
predicted gene, 44136
</td>
</tr>
<tr>
<td style="text-align:left;">
3
</td>
<td style="text-align:left;">
gene
</td>
<td style="text-align:right;">
40.87665
</td>
<td style="text-align:right;">
40.89455
</td>
<td style="text-align:left;">
negative
</td>
<td style="text-align:left;">
MGI_C57BL6J_3648839
</td>
<td style="text-align:left;">
6430590A07Rik
</td>
<td style="text-align:left;">
NCBI_Gene:619719,ENSEMBL:ENSMUSG00000097583
</td>
<td style="text-align:left;">
MGI:3648839
</td>
<td style="text-align:left;">
lncRNA gene
</td>
<td style="text-align:left;">
RIKEN cDNA 6430590A07 gene
</td>
</tr>
<tr>
<td style="text-align:left;">
3
</td>
<td style="text-align:left;">
pseudogene
</td>
<td style="text-align:right;">
40.87698
</td>
<td style="text-align:right;">
40.87725
</td>
<td style="text-align:left;">
negative
</td>
<td style="text-align:left;">
MGI_C57BL6J_5753100
</td>
<td style="text-align:left;">
Gm44524
</td>
<td style="text-align:left;">
ENSEMBL:ENSMUSG00000107892
</td>
<td style="text-align:left;">
MGI:5753100
</td>
<td style="text-align:left;">
pseudogene
</td>
<td style="text-align:left;">
predicted gene 44524
</td>
</tr>
<tr>
<td style="text-align:left;">
3
</td>
<td style="text-align:left;">
gene
</td>
<td style="text-align:right;">
40.90020
</td>
<td style="text-align:right;">
40.90254
</td>
<td style="text-align:left;">
negative
</td>
<td style="text-align:left;">
MGI_C57BL6J_1920547
</td>
<td style="text-align:left;">
1700034I23Rik
</td>
<td style="text-align:left;">
NCBI_Gene:73297,ENSEMBL:ENSMUSG00000074619
</td>
<td style="text-align:left;">
MGI:1920547
</td>
<td style="text-align:left;">
protein coding gene
</td>
<td style="text-align:left;">
RIKEN cDNA 1700034I23 gene
</td>
</tr>
<tr>
<td style="text-align:left;">
3
</td>
<td style="text-align:left;">
gene
</td>
<td style="text-align:right;">
40.94869
</td>
<td style="text-align:right;">
40.94986
</td>
<td style="text-align:left;">
negative
</td>
<td style="text-align:left;">
MGI_C57BL6J_5826439
</td>
<td style="text-align:left;">
Gm46802
</td>
<td style="text-align:left;">
NCBI_Gene:108168867
</td>
<td style="text-align:left;">
MGI:5826439
</td>
<td style="text-align:left;">
lncRNA gene
</td>
<td style="text-align:left;">
predicted gene, 46802
</td>
</tr>
<tr>
<td style="text-align:left;">
3
</td>
<td style="text-align:left;">
gene
</td>
<td style="text-align:right;">
40.94972
</td>
<td style="text-align:right;">
41.04023
</td>
<td style="text-align:left;">
positive
</td>
<td style="text-align:left;">
MGI_C57BL6J_1914604
</td>
<td style="text-align:left;">
Larp1b
</td>
<td style="text-align:left;">
NCBI_Gene:214048,ENSEMBL:ENSMUSG00000025762
</td>
<td style="text-align:left;">
MGI:1914604
</td>
<td style="text-align:left;">
protein coding gene
</td>
<td style="text-align:left;">
La ribonucleoprotein domain family, member 1B
</td>
</tr>
<tr>
<td style="text-align:left;">
3
</td>
<td style="text-align:left;">
gene
</td>
<td style="text-align:right;">
40.97850
</td>
<td style="text-align:right;">
41.00609
</td>
<td style="text-align:left;">
positive
</td>
<td style="text-align:left;">
MGI_C57BL6J_5589448
</td>
<td style="text-align:left;">
Gm30289
</td>
<td style="text-align:left;">
NA
</td>
<td style="text-align:left;">
NA
</td>
<td style="text-align:left;">
protein coding gene
</td>
<td style="text-align:left;">
predicted gene%2c 30289
</td>
</tr>
<tr>
<td style="text-align:left;">
3
</td>
<td style="text-align:left;">
gene
</td>
<td style="text-align:right;">
40.99269
</td>
<td style="text-align:right;">
40.99285
</td>
<td style="text-align:left;">
positive
</td>
<td style="text-align:left;">
MGI_C57BL6J_5451820
</td>
<td style="text-align:left;">
Gm22043
</td>
<td style="text-align:left;">
ENSEMBL:ENSMUSG00000064927
</td>
<td style="text-align:left;">
MGI:5451820
</td>
<td style="text-align:left;">
snRNA gene
</td>
<td style="text-align:left;">
predicted gene, 22043
</td>
</tr>
<tr>
<td style="text-align:left;">
3
</td>
<td style="text-align:left;">
gene
</td>
<td style="text-align:right;">
41.00243
</td>
<td style="text-align:right;">
41.00609
</td>
<td style="text-align:left;">
positive
</td>
<td style="text-align:left;">
MGI_C57BL6J_5662573
</td>
<td style="text-align:left;">
Gm42436
</td>
<td style="text-align:left;">
ENSEMBL:ENSMUSG00000105517
</td>
<td style="text-align:left;">
MGI:5662573
</td>
<td style="text-align:left;">
unclassified gene
</td>
<td style="text-align:left;">
predicted gene 42436
</td>
</tr>
<tr>
<td style="text-align:left;">
3
</td>
<td style="text-align:left;">
gene
</td>
<td style="text-align:right;">
41.00552
</td>
<td style="text-align:right;">
41.00644
</td>
<td style="text-align:left;">
negative
</td>
<td style="text-align:left;">
MGI_C57BL6J_5662574
</td>
<td style="text-align:left;">
Gm42437
</td>
<td style="text-align:left;">
ENSEMBL:ENSMUSG00000105674
</td>
<td style="text-align:left;">
MGI:5662574
</td>
<td style="text-align:left;">
unclassified gene
</td>
<td style="text-align:left;">
predicted gene 42437
</td>
</tr>
<tr>
<td style="text-align:left;">
3
</td>
<td style="text-align:left;">
gene
</td>
<td style="text-align:right;">
41.06633
</td>
<td style="text-align:right;">
41.08305
</td>
<td style="text-align:left;">
negative
</td>
<td style="text-align:left;">
MGI_C57BL6J_1918054
</td>
<td style="text-align:left;">
Pgrmc2
</td>
<td style="text-align:left;">
NCBI_Gene:70804,ENSEMBL:ENSMUSG00000049940
</td>
<td style="text-align:left;">
MGI:1918054
</td>
<td style="text-align:left;">
protein coding gene
</td>
<td style="text-align:left;">
progesterone receptor membrane component 2
</td>
</tr>
<tr>
<td style="text-align:left;">
3
</td>
<td style="text-align:left;">
gene
</td>
<td style="text-align:right;">
41.15803
</td>
<td style="text-align:right;">
41.17346
</td>
<td style="text-align:left;">
negative
</td>
<td style="text-align:left;">
MGI_C57BL6J_5622923
</td>
<td style="text-align:left;">
Gm40038
</td>
<td style="text-align:left;">
NCBI_Gene:105244429,ENSEMBL:ENSMUSG00000105258
</td>
<td style="text-align:left;">
MGI:5622923
</td>
<td style="text-align:left;">
lncRNA gene
</td>
<td style="text-align:left;">
predicted gene, 40038
</td>
</tr>
<tr>
<td style="text-align:left;">
3
</td>
<td style="text-align:left;">
gene
</td>
<td style="text-align:right;">
41.28606
</td>
<td style="text-align:right;">
41.29059
</td>
<td style="text-align:left;">
negative
</td>
<td style="text-align:left;">
MGI_C57BL6J_3642810
</td>
<td style="text-align:left;">
Gm16508
</td>
<td style="text-align:left;">
ENSEMBL:ENSMUSG00000106000
</td>
<td style="text-align:left;">
MGI:3642810
</td>
<td style="text-align:left;">
lncRNA gene
</td>
<td style="text-align:left;">
predicted gene 16508
</td>
</tr>
<tr>
<td style="text-align:left;">
3
</td>
<td style="text-align:left;">
gene
</td>
<td style="text-align:right;">
41.35415
</td>
<td style="text-align:right;">
41.40446
</td>
<td style="text-align:left;">
positive
</td>
<td style="text-align:left;">
MGI_C57BL6J_1920712
</td>
<td style="text-align:left;">
1700052H01Rik
</td>
<td style="text-align:left;">
NCBI_Gene:73462,ENSEMBL:ENSMUSG00000105297
</td>
<td style="text-align:left;">
MGI:1920712
</td>
<td style="text-align:left;">
lncRNA gene
</td>
<td style="text-align:left;">
RIKEN cDNA 1700052H01 gene
</td>
</tr>
<tr>
<td style="text-align:left;">
3
</td>
<td style="text-align:left;">
gene
</td>
<td style="text-align:right;">
41.44134
</td>
<td style="text-align:right;">
41.44147
</td>
<td style="text-align:left;">
negative
</td>
<td style="text-align:left;">
MGI_C57BL6J_5455264
</td>
<td style="text-align:left;">
Gm25487
</td>
<td style="text-align:left;">
ENSEMBL:ENSMUSG00000089350
</td>
<td style="text-align:left;">
MGI:5455264
</td>
<td style="text-align:left;">
snoRNA gene
</td>
<td style="text-align:left;">
predicted gene, 25487
</td>
</tr>
<tr>
<td style="text-align:left;">
3
</td>
<td style="text-align:left;">
gene
</td>
<td style="text-align:right;">
41.47491
</td>
<td style="text-align:right;">
41.48400
</td>
<td style="text-align:left;">
negative
</td>
<td style="text-align:left;">
MGI_C57BL6J_5622925
</td>
<td style="text-align:left;">
Gm40040
</td>
<td style="text-align:left;">
NCBI_Gene:105244431,ENSEMBL:ENSMUSG00000105260
</td>
<td style="text-align:left;">
MGI:5622925
</td>
<td style="text-align:left;">
lncRNA gene
</td>
<td style="text-align:left;">
predicted gene, 40040
</td>
</tr>
<tr>
<td style="text-align:left;">
3
</td>
<td style="text-align:left;">
gene
</td>
<td style="text-align:right;">
41.48402
</td>
<td style="text-align:right;">
41.49320
</td>
<td style="text-align:left;">
negative
</td>
<td style="text-align:left;">
MGI_C57BL6J_1925956
</td>
<td style="text-align:left;">
Platr4
</td>
<td style="text-align:left;">
NCBI_Gene:78706,ENSEMBL:ENSMUSG00000097639
</td>
<td style="text-align:left;">
MGI:1925956
</td>
<td style="text-align:left;">
lncRNA gene
</td>
<td style="text-align:left;">
pluripotency associated transcript 4
</td>
</tr>
<tr>
<td style="text-align:left;">
3
</td>
<td style="text-align:left;">
pseudogene
</td>
<td style="text-align:right;">
41.48569
</td>
<td style="text-align:right;">
41.48641
</td>
<td style="text-align:left;">
positive
</td>
<td style="text-align:left;">
MGI_C57BL6J_3648422
</td>
<td style="text-align:left;">
Gm6069
</td>
<td style="text-align:left;">
NCBI_Gene:606542,ENSEMBL:ENSMUSG00000104616
</td>
<td style="text-align:left;">
MGI:3648422
</td>
<td style="text-align:left;">
pseudogene
</td>
<td style="text-align:left;">
predicted gene 6069
</td>
</tr>
<tr>
<td style="text-align:left;">
3
</td>
<td style="text-align:left;">
gene
</td>
<td style="text-align:right;">
41.50093
</td>
<td style="text-align:right;">
41.52847
</td>
<td style="text-align:left;">
negative
</td>
<td style="text-align:left;">
MGI_C57BL6J_1925898
</td>
<td style="text-align:left;">
2400006E01Rik
</td>
<td style="text-align:left;">
NCBI_Gene:78648,ENSEMBL:ENSMUSG00000106223
</td>
<td style="text-align:left;">
MGI:1925898
</td>
<td style="text-align:left;">
lncRNA gene
</td>
<td style="text-align:left;">
RIKEN cDNA 2400006E01 gene
</td>
</tr>
<tr>
<td style="text-align:left;">
3
</td>
<td style="text-align:left;">
gene
</td>
<td style="text-align:right;">
41.55573
</td>
<td style="text-align:right;">
41.61686
</td>
<td style="text-align:left;">
positive
</td>
<td style="text-align:left;">
MGI_C57BL6J_1925835
</td>
<td style="text-align:left;">
Jade1
</td>
<td style="text-align:left;">
NCBI_Gene:269424,ENSEMBL:ENSMUSG00000025764
</td>
<td style="text-align:left;">
MGI:1925835
</td>
<td style="text-align:left;">
protein coding gene
</td>
<td style="text-align:left;">
jade family PHD finger 1
</td>
</tr>
<tr>
<td style="text-align:left;">
3
</td>
<td style="text-align:left;">
gene
</td>
<td style="text-align:right;">
41.56050
</td>
<td style="text-align:right;">
41.56370
</td>
<td style="text-align:left;">
negative
</td>
<td style="text-align:left;">
MGI_C57BL6J_5590425
</td>
<td style="text-align:left;">
Gm31266
</td>
<td style="text-align:left;">
NCBI_Gene:105244433,ENSEMBL:ENSMUSG00000103843
</td>
<td style="text-align:left;">
MGI:5590425
</td>
<td style="text-align:left;">
lncRNA gene
</td>
<td style="text-align:left;">
predicted gene, 31266
</td>
</tr>
<tr>
<td style="text-align:left;">
3
</td>
<td style="text-align:left;">
gene
</td>
<td style="text-align:right;">
41.61755
</td>
<td style="text-align:right;">
41.62371
</td>
<td style="text-align:left;">
negative
</td>
<td style="text-align:left;">
MGI_C57BL6J_5622926
</td>
<td style="text-align:left;">
Gm40041
</td>
<td style="text-align:left;">
NCBI_Gene:105244432
</td>
<td style="text-align:left;">
MGI:5622926
</td>
<td style="text-align:left;">
lncRNA gene
</td>
<td style="text-align:left;">
predicted gene, 40041
</td>
</tr>
<tr>
<td style="text-align:left;">
3
</td>
<td style="text-align:left;">
gene
</td>
<td style="text-align:right;">
41.62671
</td>
<td style="text-align:right;">
41.74253
</td>
<td style="text-align:left;">
negative
</td>
<td style="text-align:left;">
MGI_C57BL6J_1914411
</td>
<td style="text-align:left;">
Sclt1
</td>
<td style="text-align:left;">
NCBI_Gene:67161,ENSEMBL:ENSMUSG00000059834
</td>
<td style="text-align:left;">
MGI:1914411
</td>
<td style="text-align:left;">
protein coding gene
</td>
<td style="text-align:left;">
sodium channel and clathrin linker 1
</td>
</tr>
<tr>
<td style="text-align:left;">
3
</td>
<td style="text-align:left;">
gene
</td>
<td style="text-align:right;">
41.64812
</td>
<td style="text-align:right;">
41.64925
</td>
<td style="text-align:left;">
negative
</td>
<td style="text-align:left;">
MGI_C57BL6J_5611275
</td>
<td style="text-align:left;">
Gm38047
</td>
<td style="text-align:left;">
ENSEMBL:ENSMUSG00000102436
</td>
<td style="text-align:left;">
MGI:5611275
</td>
<td style="text-align:left;">
unclassified gene
</td>
<td style="text-align:left;">
predicted gene, 38047
</td>
</tr>
<tr>
<td style="text-align:left;">
3
</td>
<td style="text-align:left;">
gene
</td>
<td style="text-align:right;">
41.74261
</td>
<td style="text-align:right;">
41.80332
</td>
<td style="text-align:left;">
positive
</td>
<td style="text-align:left;">
MGI_C57BL6J_1289213
</td>
<td style="text-align:left;">
D3Ertd751e
</td>
<td style="text-align:left;">
NCBI_Gene:73852,ENSEMBL:ENSMUSG00000025766
</td>
<td style="text-align:left;">
MGI:1289213
</td>
<td style="text-align:left;">
protein coding gene
</td>
<td style="text-align:left;">
DNA segment, Chr 3, ERATO Doi 751, expressed
</td>
</tr>
<tr>
<td style="text-align:left;">
3
</td>
<td style="text-align:left;">
gene
</td>
<td style="text-align:right;">
41.77318
</td>
<td style="text-align:right;">
41.77579
</td>
<td style="text-align:left;">
positive
</td>
<td style="text-align:left;">
MGI_C57BL6J_5610496
</td>
<td style="text-align:left;">
Gm37268
</td>
<td style="text-align:left;">
ENSEMBL:ENSMUSG00000103952
</td>
<td style="text-align:left;">
MGI:5610496
</td>
<td style="text-align:left;">
unclassified gene
</td>
<td style="text-align:left;">
predicted gene, 37268
</td>
</tr>
<tr>
<td style="text-align:left;">
3
</td>
<td style="text-align:left;">
pseudogene
</td>
<td style="text-align:right;">
41.97607
</td>
<td style="text-align:right;">
41.97812
</td>
<td style="text-align:left;">
negative
</td>
<td style="text-align:left;">
MGI_C57BL6J_3645044
</td>
<td style="text-align:left;">
Gm7899
</td>
<td style="text-align:left;">
NCBI_Gene:666031,ENSEMBL:ENSMUSG00000103823
</td>
<td style="text-align:left;">
MGI:3645044
</td>
<td style="text-align:left;">
pseudogene
</td>
<td style="text-align:left;">
predicted gene 7899
</td>
</tr>
<tr>
<td style="text-align:left;">
3
</td>
<td style="text-align:left;">
pseudogene
</td>
<td style="text-align:right;">
41.97633
</td>
<td style="text-align:right;">
41.97812
</td>
<td style="text-align:left;">
positive
</td>
<td style="text-align:left;">
MGI_C57BL6J_5610576
</td>
<td style="text-align:left;">
Gm37348
</td>
<td style="text-align:left;">
ENSEMBL:ENSMUSG00000102972
</td>
<td style="text-align:left;">
MGI:5610576
</td>
<td style="text-align:left;">
pseudogene
</td>
<td style="text-align:left;">
predicted gene, 37348
</td>
</tr>
<tr>
<td style="text-align:left;">
3
</td>
<td style="text-align:left;">
gene
</td>
<td style="text-align:right;">
42.16387
</td>
<td style="text-align:right;">
42.16764
</td>
<td style="text-align:left;">
positive
</td>
<td style="text-align:left;">
MGI_C57BL6J_5611074
</td>
<td style="text-align:left;">
Gm37846
</td>
<td style="text-align:left;">
ENSEMBL:ENSMUSG00000102610
</td>
<td style="text-align:left;">
MGI:5611074
</td>
<td style="text-align:left;">
unclassified gene
</td>
<td style="text-align:left;">
predicted gene, 37846
</td>
</tr>
<tr>
<td style="text-align:left;">
3
</td>
<td style="text-align:left;">
pseudogene
</td>
<td style="text-align:right;">
42.36263
</td>
<td style="text-align:right;">
42.36278
</td>
<td style="text-align:left;">
negative
</td>
<td style="text-align:left;">
MGI_C57BL6J_5611502
</td>
<td style="text-align:left;">
Gm38274
</td>
<td style="text-align:left;">
ENSEMBL:ENSMUSG00000104089
</td>
<td style="text-align:left;">
MGI:5611502
</td>
<td style="text-align:left;">
pseudogene
</td>
<td style="text-align:left;">
predicted gene, 38274
</td>
</tr>
<tr>
<td style="text-align:left;">
3
</td>
<td style="text-align:left;">
gene
</td>
<td style="text-align:right;">
42.57246
</td>
<td style="text-align:right;">
42.57300
</td>
<td style="text-align:left;">
positive
</td>
<td style="text-align:left;">
MGI_C57BL6J_5610761
</td>
<td style="text-align:left;">
Gm37533
</td>
<td style="text-align:left;">
ENSEMBL:ENSMUSG00000103426
</td>
<td style="text-align:left;">
MGI:5610761
</td>
<td style="text-align:left;">
unclassified gene
</td>
<td style="text-align:left;">
predicted gene, 37533
</td>
</tr>
<tr>
<td style="text-align:left;">
3
</td>
<td style="text-align:left;">
pseudogene
</td>
<td style="text-align:right;">
42.58511
</td>
<td style="text-align:right;">
42.58560
</td>
<td style="text-align:left;">
negative
</td>
<td style="text-align:left;">
MGI_C57BL6J_5611272
</td>
<td style="text-align:left;">
Gm38044
</td>
<td style="text-align:left;">
ENSEMBL:ENSMUSG00000104294
</td>
<td style="text-align:left;">
MGI:5611272
</td>
<td style="text-align:left;">
pseudogene
</td>
<td style="text-align:left;">
predicted gene, 38044
</td>
</tr>
<tr>
<td style="text-align:left;">
3
</td>
<td style="text-align:left;">
pseudogene
</td>
<td style="text-align:right;">
43.25039
</td>
<td style="text-align:right;">
43.25093
</td>
<td style="text-align:left;">
positive
</td>
<td style="text-align:left;">
MGI_C57BL6J_3780131
</td>
<td style="text-align:left;">
Gm9723
</td>
<td style="text-align:left;">
NCBI_Gene:677788,ENSEMBL:ENSMUSG00000102149
</td>
<td style="text-align:left;">
MGI:3780131
</td>
<td style="text-align:left;">
pseudogene
</td>
<td style="text-align:left;">
predicted gene 9723
</td>
</tr>
<tr>
<td style="text-align:left;">
3
</td>
<td style="text-align:left;">
pseudogene
</td>
<td style="text-align:right;">
43.39077
</td>
<td style="text-align:right;">
43.39194
</td>
<td style="text-align:left;">
negative
</td>
<td style="text-align:left;">
MGI_C57BL6J_5011050
</td>
<td style="text-align:left;">
Gm18865
</td>
<td style="text-align:left;">
NCBI_Gene:100417854,ENSEMBL:ENSMUSG00000102334
</td>
<td style="text-align:left;">
MGI:5011050
</td>
<td style="text-align:left;">
pseudogene
</td>
<td style="text-align:left;">
predicted gene, 18865
</td>
</tr>
<tr>
<td style="text-align:left;">
3
</td>
<td style="text-align:left;">
gene
</td>
<td style="text-align:right;">
43.54717
</td>
<td style="text-align:right;">
43.54828
</td>
<td style="text-align:left;">
positive
</td>
<td style="text-align:left;">
MGI_C57BL6J_5610204
</td>
<td style="text-align:left;">
Gm36976
</td>
<td style="text-align:left;">
ENSEMBL:ENSMUSG00000104303
</td>
<td style="text-align:left;">
MGI:5610204
</td>
<td style="text-align:left;">
unclassified gene
</td>
<td style="text-align:left;">
predicted gene, 36976
</td>
</tr>
<tr>
<td style="text-align:left;">
3
</td>
<td style="text-align:left;">
pseudogene
</td>
<td style="text-align:right;">
43.76405
</td>
<td style="text-align:right;">
43.76476
</td>
<td style="text-align:left;">
negative
</td>
<td style="text-align:left;">
MGI_C57BL6J_5588898
</td>
<td style="text-align:left;">
Gm29739
</td>
<td style="text-align:left;">
NCBI_Gene:101055891,ENSEMBL:ENSMUSG00000104083
</td>
<td style="text-align:left;">
MGI:5588898
</td>
<td style="text-align:left;">
pseudogene
</td>
<td style="text-align:left;">
predicted gene, 29739
</td>
</tr>
<tr>
<td style="text-align:left;">
3
</td>
<td style="text-align:left;">
gene
</td>
<td style="text-align:right;">
43.92557
</td>
<td style="text-align:right;">
43.92586
</td>
<td style="text-align:left;">
positive
</td>
<td style="text-align:left;">
MGI_C57BL6J_5455491
</td>
<td style="text-align:left;">
Gm25714
</td>
<td style="text-align:left;">
ENSEMBL:ENSMUSG00000089251
</td>
<td style="text-align:left;">
MGI:5455491
</td>
<td style="text-align:left;">
unclassified non-coding RNA gene
</td>
<td style="text-align:left;">
predicted gene, 25714
</td>
</tr>
<tr>
<td style="text-align:left;">
3
</td>
<td style="text-align:left;">
pseudogene
</td>
<td style="text-align:right;">
43.92760
</td>
<td style="text-align:right;">
43.92795
</td>
<td style="text-align:left;">
negative
</td>
<td style="text-align:left;">
MGI_C57BL6J_5591334
</td>
<td style="text-align:left;">
Gm32175
</td>
<td style="text-align:left;">
ENSEMBL:ENSMUSG00000102792
</td>
<td style="text-align:left;">
MGI:5591334
</td>
<td style="text-align:left;">
pseudogene
</td>
<td style="text-align:left;">
predicted gene, 32175
</td>
</tr>
<tr>
<td style="text-align:left;">
3
</td>
<td style="text-align:left;">
gene
</td>
<td style="text-align:right;">
43.95572
</td>
<td style="text-align:right;">
43.95592
</td>
<td style="text-align:left;">
positive
</td>
<td style="text-align:left;">
MGI_C57BL6J_5610974
</td>
<td style="text-align:left;">
Gm37746
</td>
<td style="text-align:left;">
ENSEMBL:ENSMUSG00000103148
</td>
<td style="text-align:left;">
MGI:5610974
</td>
<td style="text-align:left;">
unclassified gene
</td>
<td style="text-align:left;">
predicted gene, 37746
</td>
</tr>
<tr>
<td style="text-align:left;">
3
</td>
<td style="text-align:left;">
pseudogene
</td>
<td style="text-align:right;">
44.49350
</td>
<td style="text-align:right;">
44.49390
</td>
<td style="text-align:left;">
negative
</td>
<td style="text-align:left;">
MGI_C57BL6J_5610593
</td>
<td style="text-align:left;">
Gm37365
</td>
<td style="text-align:left;">
ENSEMBL:ENSMUSG00000104013
</td>
<td style="text-align:left;">
MGI:5610593
</td>
<td style="text-align:left;">
pseudogene
</td>
<td style="text-align:left;">
predicted gene, 37365
</td>
</tr>
<tr>
<td style="text-align:left;">
3
</td>
<td style="text-align:left;">
gene
</td>
<td style="text-align:right;">
44.65990
</td>
<td style="text-align:right;">
44.66315
</td>
<td style="text-align:left;">
positive
</td>
<td style="text-align:left;">
MGI_C57BL6J_5611356
</td>
<td style="text-align:left;">
Gm38128
</td>
<td style="text-align:left;">
ENSEMBL:ENSMUSG00000104051
</td>
<td style="text-align:left;">
MGI:5611356
</td>
<td style="text-align:left;">
unclassified gene
</td>
<td style="text-align:left;">
predicted gene, 38128
</td>
</tr>
<tr>
<td style="text-align:left;">
3
</td>
<td style="text-align:left;">
pseudogene
</td>
<td style="text-align:right;">
44.71813
</td>
<td style="text-align:right;">
44.71851
</td>
<td style="text-align:left;">
negative
</td>
<td style="text-align:left;">
MGI_C57BL6J_5610825
</td>
<td style="text-align:left;">
Gm37597
</td>
<td style="text-align:left;">
ENSEMBL:ENSMUSG00000103353
</td>
<td style="text-align:left;">
MGI:5610825
</td>
<td style="text-align:left;">
pseudogene
</td>
<td style="text-align:left;">
predicted gene, 37597
</td>
</tr>
<tr>
<td style="text-align:left;">
3
</td>
<td style="text-align:left;">
pseudogene
</td>
<td style="text-align:right;">
44.72516
</td>
<td style="text-align:right;">
44.72574
</td>
<td style="text-align:left;">
positive
</td>
<td style="text-align:left;">
MGI_C57BL6J_5011051
</td>
<td style="text-align:left;">
Gm18866
</td>
<td style="text-align:left;">
NCBI_Gene:100417856
</td>
<td style="text-align:left;">
MGI:5011051
</td>
<td style="text-align:left;">
pseudogene
</td>
<td style="text-align:left;">
predicted gene, 18866
</td>
</tr>
<tr>
<td style="text-align:left;">
3
</td>
<td style="text-align:left;">
gene
</td>
<td style="text-align:right;">
44.98320
</td>
<td style="text-align:right;">
44.98330
</td>
<td style="text-align:left;">
positive
</td>
<td style="text-align:left;">
MGI_C57BL6J_5454774
</td>
<td style="text-align:left;">
Gm24997
</td>
<td style="text-align:left;">
ENSEMBL:ENSMUSG00000088535
</td>
<td style="text-align:left;">
MGI:5454774
</td>
<td style="text-align:left;">
snRNA gene
</td>
<td style="text-align:left;">
predicted gene, 24997
</td>
</tr>
<tr>
<td style="text-align:left;">
3
</td>
<td style="text-align:left;">
gene
</td>
<td style="text-align:right;">
45.16069
</td>
<td style="text-align:right;">
45.16080
</td>
<td style="text-align:left;">
positive
</td>
<td style="text-align:left;">
MGI_C57BL6J_5531371
</td>
<td style="text-align:left;">
Gm27989
</td>
<td style="text-align:left;">
ENSEMBL:ENSMUSG00000099285
</td>
<td style="text-align:left;">
MGI:5531371
</td>
<td style="text-align:left;">
miRNA gene
</td>
<td style="text-align:left;">
predicted gene, 27989
</td>
</tr>
<tr>
<td style="text-align:left;">
3
</td>
<td style="text-align:left;">
pseudogene
</td>
<td style="text-align:right;">
45.17179
</td>
<td style="text-align:right;">
45.17324
</td>
<td style="text-align:left;">
negative
</td>
<td style="text-align:left;">
MGI_C57BL6J_5434370
</td>
<td style="text-align:left;">
Gm21015
</td>
<td style="text-align:left;">
NCBI_Gene:100534291
</td>
<td style="text-align:left;">
MGI:5434370
</td>
<td style="text-align:left;">
pseudogene
</td>
<td style="text-align:left;">
predicted gene, 21015
</td>
</tr>
<tr>
<td style="text-align:left;">
3
</td>
<td style="text-align:left;">
gene
</td>
<td style="text-align:right;">
45.26999
</td>
<td style="text-align:right;">
45.27257
</td>
<td style="text-align:left;">
negative
</td>
<td style="text-align:left;">
MGI_C57BL6J_5611352
</td>
<td style="text-align:left;">
Gm38124
</td>
<td style="text-align:left;">
ENSEMBL:ENSMUSG00000103082
</td>
<td style="text-align:left;">
MGI:5611352
</td>
<td style="text-align:left;">
unclassified gene
</td>
<td style="text-align:left;">
predicted gene, 38124
</td>
</tr>
<tr>
<td style="text-align:left;">
3
</td>
<td style="text-align:left;">
gene
</td>
<td style="text-align:right;">
45.27342
</td>
<td style="text-align:right;">
45.27497
</td>
<td style="text-align:left;">
negative
</td>
<td style="text-align:left;">
MGI_C57BL6J_1924316
</td>
<td style="text-align:left;">
5930409G06Rik
</td>
<td style="text-align:left;">
ENSEMBL:ENSMUSG00000103477
</td>
<td style="text-align:left;">
MGI:1924316
</td>
<td style="text-align:left;">
unclassified gene
</td>
<td style="text-align:left;">
RIKEN cDNA 5930409G06 gene
</td>
</tr>
<tr>
<td style="text-align:left;">
3
</td>
<td style="text-align:left;">
gene
</td>
<td style="text-align:right;">
45.28044
</td>
<td style="text-align:right;">
45.37826
</td>
<td style="text-align:left;">
negative
</td>
<td style="text-align:left;">
MGI_C57BL6J_1919761
</td>
<td style="text-align:left;">
2610316D01Rik
</td>
<td style="text-align:left;">
NCBI_Gene:72511,ENSEMBL:ENSMUSG00000097040
</td>
<td style="text-align:left;">
MGI:1919761
</td>
<td style="text-align:left;">
lncRNA gene
</td>
<td style="text-align:left;">
RIKEN cDNA 2610316D01 gene
</td>
</tr>
<tr>
<td style="text-align:left;">
3
</td>
<td style="text-align:left;">
pseudogene
</td>
<td style="text-align:right;">
45.33216
</td>
<td style="text-align:right;">
45.33241
</td>
<td style="text-align:left;">
negative
</td>
<td style="text-align:left;">
MGI_C57BL6J_5610224
</td>
<td style="text-align:left;">
Gm36996
</td>
<td style="text-align:left;">
ENSEMBL:ENSMUSG00000103430
</td>
<td style="text-align:left;">
MGI:5610224
</td>
<td style="text-align:left;">
pseudogene
</td>
<td style="text-align:left;">
predicted gene, 36996
</td>
</tr>
<tr>
<td style="text-align:left;">
3
</td>
<td style="text-align:left;">
gene
</td>
<td style="text-align:right;">
45.37840
</td>
<td style="text-align:right;">
45.43562
</td>
<td style="text-align:left;">
positive
</td>
<td style="text-align:left;">
MGI_C57BL6J_1338042
</td>
<td style="text-align:left;">
Pcdh10
</td>
<td style="text-align:left;">
NCBI_Gene:18526,ENSEMBL:ENSMUSG00000049100
</td>
<td style="text-align:left;">
MGI:1338042
</td>
<td style="text-align:left;">
protein coding gene
</td>
<td style="text-align:left;">
protocadherin 10
</td>
</tr>
<tr>
<td style="text-align:left;">
3
</td>
<td style="text-align:left;">
gene
</td>
<td style="text-align:right;">
45.38548
</td>
<td style="text-align:right;">
45.38960
</td>
<td style="text-align:left;">
positive
</td>
<td style="text-align:left;">
MGI_C57BL6J_2442672
</td>
<td style="text-align:left;">
6330566A10Rik
</td>
<td style="text-align:left;">
NA
</td>
<td style="text-align:left;">
NA
</td>
<td style="text-align:left;">
unclassified gene
</td>
<td style="text-align:left;">
RIKEN cDNA 6330566A10 gene
</td>
</tr>
<tr>
<td style="text-align:left;">
3
</td>
<td style="text-align:left;">
gene
</td>
<td style="text-align:right;">
45.39986
</td>
<td style="text-align:right;">
45.40200
</td>
<td style="text-align:left;">
positive
</td>
<td style="text-align:left;">
MGI_C57BL6J_2444936
</td>
<td style="text-align:left;">
D930002L09Rik
</td>
<td style="text-align:left;">
NA
</td>
<td style="text-align:left;">
NA
</td>
<td style="text-align:left;">
unclassified gene
</td>
<td style="text-align:left;">
RIKEN cDNA D930002L09 gene
</td>
</tr>
<tr>
<td style="text-align:left;">
3
</td>
<td style="text-align:left;">
gene
</td>
<td style="text-align:right;">
45.41658
</td>
<td style="text-align:right;">
45.43931
</td>
<td style="text-align:left;">
negative
</td>
<td style="text-align:left;">
MGI_C57BL6J_1919524
</td>
<td style="text-align:left;">
1700027H10Rik
</td>
<td style="text-align:left;">
NCBI_Gene:72274,ENSEMBL:ENSMUSG00000100799
</td>
<td style="text-align:left;">
MGI:1919524
</td>
<td style="text-align:left;">
lncRNA gene
</td>
<td style="text-align:left;">
RIKEN cDNA 1700027H10 gene
</td>
</tr>
<tr>
<td style="text-align:left;">
3
</td>
<td style="text-align:left;">
gene
</td>
<td style="text-align:right;">
45.43838
</td>
<td style="text-align:right;">
45.44096
</td>
<td style="text-align:left;">
positive
</td>
<td style="text-align:left;">
MGI_C57BL6J_3780305
</td>
<td style="text-align:left;">
Gm2136
</td>
<td style="text-align:left;">
ENSEMBL:ENSMUSG00000103713
</td>
<td style="text-align:left;">
MGI:3780305
</td>
<td style="text-align:left;">
protein coding gene
</td>
<td style="text-align:left;">
predicted gene 2136
</td>
</tr>
<tr>
<td style="text-align:left;">
3
</td>
<td style="text-align:left;">
gene
</td>
<td style="text-align:right;">
45.44567
</td>
<td style="text-align:right;">
45.44881
</td>
<td style="text-align:left;">
positive
</td>
<td style="text-align:left;">
MGI_C57BL6J_5610573
</td>
<td style="text-align:left;">
Gm37345
</td>
<td style="text-align:left;">
ENSEMBL:ENSMUSG00000103947
</td>
<td style="text-align:left;">
MGI:5610573
</td>
<td style="text-align:left;">
unclassified gene
</td>
<td style="text-align:left;">
predicted gene, 37345
</td>
</tr>
<tr>
<td style="text-align:left;">
3
</td>
<td style="text-align:left;">
pseudogene
</td>
<td style="text-align:right;">
45.69054
</td>
<td style="text-align:right;">
45.69090
</td>
<td style="text-align:left;">
negative
</td>
<td style="text-align:left;">
MGI_C57BL6J_5611063
</td>
<td style="text-align:left;">
Gm37835
</td>
<td style="text-align:left;">
ENSEMBL:ENSMUSG00000103534
</td>
<td style="text-align:left;">
MGI:5611063
</td>
<td style="text-align:left;">
pseudogene
</td>
<td style="text-align:left;">
predicted gene, 37835
</td>
</tr>
<tr>
<td style="text-align:left;">
3
</td>
<td style="text-align:left;">
gene
</td>
<td style="text-align:right;">
45.77542
</td>
<td style="text-align:right;">
45.77589
</td>
<td style="text-align:left;">
negative
</td>
<td style="text-align:left;">
MGI_C57BL6J_5611394
</td>
<td style="text-align:left;">
Gm38166
</td>
<td style="text-align:left;">
ENSEMBL:ENSMUSG00000102190
</td>
<td style="text-align:left;">
MGI:5611394
</td>
<td style="text-align:left;">
unclassified gene
</td>
<td style="text-align:left;">
predicted gene, 38166
</td>
</tr>
<tr>
<td style="text-align:left;">
3
</td>
<td style="text-align:left;">
pseudogene
</td>
<td style="text-align:right;">
45.77587
</td>
<td style="text-align:right;">
45.86058
</td>
<td style="text-align:left;">
negative
</td>
<td style="text-align:left;">
MGI_C57BL6J_5591499
</td>
<td style="text-align:left;">
Gm32340
</td>
<td style="text-align:left;">
NCBI_Gene:102634856,ENSEMBL:ENSMUSG00000104491
</td>
<td style="text-align:left;">
MGI:5591499
</td>
<td style="text-align:left;">
pseudogene
</td>
<td style="text-align:left;">
predicted gene, 32340
</td>
</tr>
<tr>
<td style="text-align:left;">
3
</td>
<td style="text-align:left;">
gene
</td>
<td style="text-align:right;">
45.81774
</td>
<td style="text-align:right;">
45.81790
</td>
<td style="text-align:left;">
positive
</td>
<td style="text-align:left;">
MGI_C57BL6J_5455573
</td>
<td style="text-align:left;">
Gm25796
</td>
<td style="text-align:left;">
ENSEMBL:ENSMUSG00000064579
</td>
<td style="text-align:left;">
MGI:5455573
</td>
<td style="text-align:left;">
snRNA gene
</td>
<td style="text-align:left;">
predicted gene, 25796
</td>
</tr>
<tr>
<td style="text-align:left;">
3
</td>
<td style="text-align:left;">
pseudogene
</td>
<td style="text-align:right;">
45.86121
</td>
<td style="text-align:right;">
45.86995
</td>
<td style="text-align:left;">
negative
</td>
<td style="text-align:left;">
MGI_C57BL6J_5591575
</td>
<td style="text-align:left;">
Gm32416
</td>
<td style="text-align:left;">
NCBI_Gene:102634958
</td>
<td style="text-align:left;">
MGI:5591575
</td>
<td style="text-align:left;">
pseudogene
</td>
<td style="text-align:left;">
predicted gene, 32416
</td>
</tr>
<tr>
<td style="text-align:left;">
3
</td>
<td style="text-align:left;">
gene
</td>
<td style="text-align:right;">
45.97858
</td>
<td style="text-align:right;">
45.97870
</td>
<td style="text-align:left;">
positive
</td>
<td style="text-align:left;">
MGI_C57BL6J_5453114
</td>
<td style="text-align:left;">
Gm23337
</td>
<td style="text-align:left;">
ENSEMBL:ENSMUSG00000089584
</td>
<td style="text-align:left;">
MGI:5453114
</td>
<td style="text-align:left;">
snRNA gene
</td>
<td style="text-align:left;">
predicted gene, 23337
</td>
</tr>
<tr>
<td style="text-align:left;">
3
</td>
<td style="text-align:left;">
gene
</td>
<td style="text-align:right;">
46.28983
</td>
<td style="text-align:right;">
46.28996
</td>
<td style="text-align:left;">
positive
</td>
<td style="text-align:left;">
MGI_C57BL6J_5455858
</td>
<td style="text-align:left;">
Gm26081
</td>
<td style="text-align:left;">
ENSEMBL:ENSMUSG00000077256
</td>
<td style="text-align:left;">
MGI:5455858
</td>
<td style="text-align:left;">
snoRNA gene
</td>
<td style="text-align:left;">
predicted gene, 26081
</td>
</tr>
<tr>
<td style="text-align:left;">
3
</td>
<td style="text-align:left;">
pseudogene
</td>
<td style="text-align:right;">
46.41340
</td>
<td style="text-align:right;">
46.41412
</td>
<td style="text-align:left;">
positive
</td>
<td style="text-align:left;">
MGI_C57BL6J_5011028
</td>
<td style="text-align:left;">
Gm18843
</td>
<td style="text-align:left;">
NCBI_Gene:100417811,ENSEMBL:ENSMUSG00000104511
</td>
<td style="text-align:left;">
MGI:5011028
</td>
<td style="text-align:left;">
pseudogene
</td>
<td style="text-align:left;">
predicted gene, 18843
</td>
</tr>
<tr>
<td style="text-align:left;">
3
</td>
<td style="text-align:left;">
gene
</td>
<td style="text-align:right;">
46.44085
</td>
<td style="text-align:right;">
46.44822
</td>
<td style="text-align:left;">
negative
</td>
<td style="text-align:left;">
MGI_C57BL6J_3643087
</td>
<td style="text-align:left;">
Pabpc4l
</td>
<td style="text-align:left;">
NCBI_Gene:241989,ENSEMBL:ENSMUSG00000090919
</td>
<td style="text-align:left;">
MGI:3643087
</td>
<td style="text-align:left;">
protein coding gene
</td>
<td style="text-align:left;">
poly(A) binding protein, cytoplasmic 4-like
</td>
</tr>
<tr>
<td style="text-align:left;">
3
</td>
<td style="text-align:left;">
pseudogene
</td>
<td style="text-align:right;">
46.62112
</td>
<td style="text-align:right;">
46.62136
</td>
<td style="text-align:left;">
positive
</td>
<td style="text-align:left;">
MGI_C57BL6J_5610673
</td>
<td style="text-align:left;">
Gm37445
</td>
<td style="text-align:left;">
ENSEMBL:ENSMUSG00000102589
</td>
<td style="text-align:left;">
MGI:5610673
</td>
<td style="text-align:left;">
pseudogene
</td>
<td style="text-align:left;">
predicted gene, 37445
</td>
</tr>
<tr>
<td style="text-align:left;">
3
</td>
<td style="text-align:left;">
gene
</td>
<td style="text-align:right;">
46.90067
</td>
<td style="text-align:right;">
46.90078
</td>
<td style="text-align:left;">
negative
</td>
<td style="text-align:left;">
MGI_C57BL6J_5531112
</td>
<td style="text-align:left;">
Mir6379
</td>
<td style="text-align:left;">
miRBase:MI0021910,NCBI_Gene:102465199,ENSEMBL:ENSMUSG00000098417
</td>
<td style="text-align:left;">
MGI:5531112
</td>
<td style="text-align:left;">
miRNA gene
</td>
<td style="text-align:left;">
microRNA 6379
</td>
</tr>
<tr>
<td style="text-align:left;">
3
</td>
<td style="text-align:left;">
pseudogene
</td>
<td style="text-align:right;">
47.04911
</td>
<td style="text-align:right;">
47.05178
</td>
<td style="text-align:left;">
negative
</td>
<td style="text-align:left;">
MGI_C57BL6J_3641761
</td>
<td style="text-align:left;">
Gm10356
</td>
<td style="text-align:left;">
NCBI_Gene:545517,ENSEMBL:ENSMUSG00000072387
</td>
<td style="text-align:left;">
MGI:3641761
</td>
<td style="text-align:left;">
pseudogene
</td>
<td style="text-align:left;">
predicted gene 10356
</td>
</tr>
<tr>
<td style="text-align:left;">
3
</td>
<td style="text-align:left;">
pseudogene
</td>
<td style="text-align:right;">
47.08453
</td>
<td style="text-align:right;">
47.08575
</td>
<td style="text-align:left;">
negative
</td>
<td style="text-align:left;">
MGI_C57BL6J_3646161
</td>
<td style="text-align:left;">
Gm5274
</td>
<td style="text-align:left;">
NCBI_Gene:383849,ENSEMBL:ENSMUSG00000097985
</td>
<td style="text-align:left;">
MGI:3646161
</td>
<td style="text-align:left;">
pseudogene
</td>
<td style="text-align:left;">
predicted gene 5274
</td>
</tr>
<tr>
<td style="text-align:left;">
3
</td>
<td style="text-align:left;">
gene
</td>
<td style="text-align:right;">
47.20063
</td>
<td style="text-align:right;">
47.20076
</td>
<td style="text-align:left;">
positive
</td>
<td style="text-align:left;">
MGI_C57BL6J_5453668
</td>
<td style="text-align:left;">
Gm23891
</td>
<td style="text-align:left;">
ENSEMBL:ENSMUSG00000088739
</td>
<td style="text-align:left;">
MGI:5453668
</td>
<td style="text-align:left;">
snoRNA gene
</td>
<td style="text-align:left;">
predicted gene, 23891
</td>
</tr>
<tr>
<td style="text-align:left;">
3
</td>
<td style="text-align:left;">
pseudogene
</td>
<td style="text-align:right;">
47.23050
</td>
<td style="text-align:right;">
47.23100
</td>
<td style="text-align:left;">
negative
</td>
<td style="text-align:left;">
MGI_C57BL6J_3643228
</td>
<td style="text-align:left;">
Rpl23a-ps5
</td>
<td style="text-align:left;">
NCBI_Gene:383850,ENSEMBL:ENSMUSG00000102220
</td>
<td style="text-align:left;">
MGI:3643228
</td>
<td style="text-align:left;">
pseudogene
</td>
<td style="text-align:left;">
ribosomal protein L23A, pseudogene 5
</td>
</tr>
<tr>
<td style="text-align:left;">
3
</td>
<td style="text-align:left;">
gene
</td>
<td style="text-align:right;">
47.26308
</td>
<td style="text-align:right;">
47.26580
</td>
<td style="text-align:left;">
negative
</td>
<td style="text-align:left;">
MGI_C57BL6J_3809922
</td>
<td style="text-align:left;">
Gm10730
</td>
<td style="text-align:left;">
ENSEMBL:ENSMUSG00000103491
</td>
<td style="text-align:left;">
MGI:3809922
</td>
<td style="text-align:left;">
unclassified gene
</td>
<td style="text-align:left;">
predicted gene 10730
</td>
</tr>
<tr>
<td style="text-align:left;">
3
</td>
<td style="text-align:left;">
pseudogene
</td>
<td style="text-align:right;">
47.45961
</td>
<td style="text-align:right;">
47.46034
</td>
<td style="text-align:left;">
negative
</td>
<td style="text-align:left;">
MGI_C57BL6J_3780399
</td>
<td style="text-align:left;">
Gm2229
</td>
<td style="text-align:left;">
NCBI_Gene:100039426,ENSEMBL:ENSMUSG00000103798
</td>
<td style="text-align:left;">
MGI:3780399
</td>
<td style="text-align:left;">
pseudogene
</td>
<td style="text-align:left;">
predicted gene 2229
</td>
</tr>
<tr>
<td style="text-align:left;">
3
</td>
<td style="text-align:left;">
pseudogene
</td>
<td style="text-align:right;">
48.02601
</td>
<td style="text-align:right;">
48.02676
</td>
<td style="text-align:left;">
negative
</td>
<td style="text-align:left;">
MGI_C57BL6J_3648898
</td>
<td style="text-align:left;">
Gm7977
</td>
<td style="text-align:left;">
NCBI_Gene:666202,ENSEMBL:ENSMUSG00000103257
</td>
<td style="text-align:left;">
MGI:3648898
</td>
<td style="text-align:left;">
pseudogene
</td>
<td style="text-align:left;">
predicted gene 7977
</td>
</tr>
<tr>
<td style="text-align:left;">
3
</td>
<td style="text-align:left;">
gene
</td>
<td style="text-align:right;">
48.07834
</td>
<td style="text-align:right;">
48.07845
</td>
<td style="text-align:left;">
positive
</td>
<td style="text-align:left;">
MGI_C57BL6J_5610523
</td>
<td style="text-align:left;">
Gm37295
</td>
<td style="text-align:left;">
ENSEMBL:ENSMUSG00000104110
</td>
<td style="text-align:left;">
MGI:5610523
</td>
<td style="text-align:left;">
unclassified gene
</td>
<td style="text-align:left;">
predicted gene, 37295
</td>
</tr>
<tr>
<td style="text-align:left;">
3
</td>
<td style="text-align:left;">
gene
</td>
<td style="text-align:right;">
48.54642
</td>
<td style="text-align:right;">
48.57947
</td>
<td style="text-align:left;">
positive
</td>
<td style="text-align:left;">
MGI_C57BL6J_5826472
</td>
<td style="text-align:left;">
Gm46835
</td>
<td style="text-align:left;">
NCBI_Gene:108168915
</td>
<td style="text-align:left;">
MGI:5826472
</td>
<td style="text-align:left;">
lncRNA gene
</td>
<td style="text-align:left;">
predicted gene, 46835
</td>
</tr>
<tr>
<td style="text-align:left;">
3
</td>
<td style="text-align:left;">
gene
</td>
<td style="text-align:right;">
48.60573
</td>
<td style="text-align:right;">
48.60910
</td>
<td style="text-align:left;">
negative
</td>
<td style="text-align:left;">
MGI_C57BL6J_1913582
</td>
<td style="text-align:left;">
1700018B24Rik
</td>
<td style="text-align:left;">
NCBI_Gene:66332,ENSEMBL:ENSMUSG00000037910
</td>
<td style="text-align:left;">
MGI:1913582
</td>
<td style="text-align:left;">
lncRNA gene
</td>
<td style="text-align:left;">
RIKEN cDNA 1700018B24 gene
</td>
</tr>
<tr>
<td style="text-align:left;">
3
</td>
<td style="text-align:left;">
gene
</td>
<td style="text-align:right;">
48.74851
</td>
<td style="text-align:right;">
48.76095
</td>
<td style="text-align:left;">
negative
</td>
<td style="text-align:left;">
MGI_C57BL6J_5610418
</td>
<td style="text-align:left;">
Gm37190
</td>
<td style="text-align:left;">
ENSEMBL:ENSMUSG00000102181
</td>
<td style="text-align:left;">
MGI:5610418
</td>
<td style="text-align:left;">
lncRNA gene
</td>
<td style="text-align:left;">
predicted gene, 37190
</td>
</tr>
<tr>
<td style="text-align:left;">
3
</td>
<td style="text-align:left;">
gene
</td>
<td style="text-align:right;">
48.95472
</td>
<td style="text-align:right;">
48.95488
</td>
<td style="text-align:left;">
positive
</td>
<td style="text-align:left;">
MGI_C57BL6J_5453288
</td>
<td style="text-align:left;">
Gm23511
</td>
<td style="text-align:left;">
ENSEMBL:ENSMUSG00000065018
</td>
<td style="text-align:left;">
MGI:5453288
</td>
<td style="text-align:left;">
snRNA gene
</td>
<td style="text-align:left;">
predicted gene, 23511
</td>
</tr>
<tr>
<td style="text-align:left;">
3
</td>
<td style="text-align:left;">
gene
</td>
<td style="text-align:right;">
49.06237
</td>
<td style="text-align:right;">
49.09636
</td>
<td style="text-align:left;">
positive
</td>
<td style="text-align:left;">
MGI_C57BL6J_5590574
</td>
<td style="text-align:left;">
Gm31415
</td>
<td style="text-align:left;">
NCBI_Gene:102633632,ENSEMBL:ENSMUSG00000102263
</td>
<td style="text-align:left;">
MGI:5590574
</td>
<td style="text-align:left;">
lncRNA gene
</td>
<td style="text-align:left;">
predicted gene, 31415
</td>
</tr>
<tr>
<td style="text-align:left;">
3
</td>
<td style="text-align:left;">
gene
</td>
<td style="text-align:right;">
49.13746
</td>
<td style="text-align:right;">
49.13853
</td>
<td style="text-align:left;">
positive
</td>
<td style="text-align:left;">
MGI_C57BL6J_5611047
</td>
<td style="text-align:left;">
Gm37819
</td>
<td style="text-align:left;">
ENSEMBL:ENSMUSG00000104186
</td>
<td style="text-align:left;">
MGI:5611047
</td>
<td style="text-align:left;">
unclassified gene
</td>
<td style="text-align:left;">
predicted gene, 37819
</td>
</tr>
<tr>
<td style="text-align:left;">
3
</td>
<td style="text-align:left;">
pseudogene
</td>
<td style="text-align:right;">
49.20771
</td>
<td style="text-align:right;">
49.20807
</td>
<td style="text-align:left;">
negative
</td>
<td style="text-align:left;">
MGI_C57BL6J_5610912
</td>
<td style="text-align:left;">
Gm37684
</td>
<td style="text-align:left;">
ENSEMBL:ENSMUSG00000103027
</td>
<td style="text-align:left;">
MGI:5610912
</td>
<td style="text-align:left;">
pseudogene
</td>
<td style="text-align:left;">
predicted gene, 37684
</td>
</tr>
<tr>
<td style="text-align:left;">
3
</td>
<td style="text-align:left;">
gene
</td>
<td style="text-align:right;">
49.32568
</td>
<td style="text-align:right;">
49.36736
</td>
<td style="text-align:left;">
positive
</td>
<td style="text-align:left;">
MGI_C57BL6J_5295664
</td>
<td style="text-align:left;">
Gm20557
</td>
<td style="text-align:left;">
NCBI_Gene:329633,ENSEMBL:ENSMUSG00000102679
</td>
<td style="text-align:left;">
MGI:5295664
</td>
<td style="text-align:left;">
lncRNA gene
</td>
<td style="text-align:left;">
predicted gene, 20557
</td>
</tr>
<tr>
<td style="text-align:left;">
3
</td>
<td style="text-align:left;">
pseudogene
</td>
<td style="text-align:right;">
49.38115
</td>
<td style="text-align:right;">
49.38285
</td>
<td style="text-align:left;">
negative
</td>
<td style="text-align:left;">
MGI_C57BL6J_3643624
</td>
<td style="text-align:left;">
Gm5846
</td>
<td style="text-align:left;">
NCBI_Gene:545519,ENSEMBL:ENSMUSG00000104309
</td>
<td style="text-align:left;">
MGI:3643624
</td>
<td style="text-align:left;">
pseudogene
</td>
<td style="text-align:left;">
predicted gene 5846
</td>
</tr>
<tr>
<td style="text-align:left;">
3
</td>
<td style="text-align:left;">
pseudogene
</td>
<td style="text-align:right;">
49.40082
</td>
<td style="text-align:right;">
49.46744
</td>
<td style="text-align:left;">
negative
</td>
<td style="text-align:left;">
MGI_C57BL6J_3780091
</td>
<td style="text-align:left;">
Gm9683
</td>
<td style="text-align:left;">
NCBI_Gene:676568
</td>
<td style="text-align:left;">
MGI:3780091
</td>
<td style="text-align:left;">
pseudogene
</td>
<td style="text-align:left;">
predicted gene 9683
</td>
</tr>
<tr>
<td style="text-align:left;">
3
</td>
<td style="text-align:left;">
gene
</td>
<td style="text-align:right;">
49.49135
</td>
<td style="text-align:right;">
49.49148
</td>
<td style="text-align:left;">
positive
</td>
<td style="text-align:left;">
MGI_C57BL6J_5452257
</td>
<td style="text-align:left;">
Gm22480
</td>
<td style="text-align:left;">
ENSEMBL:ENSMUSG00000084517
</td>
<td style="text-align:left;">
MGI:5452257
</td>
<td style="text-align:left;">
snoRNA gene
</td>
<td style="text-align:left;">
predicted gene, 22480
</td>
</tr>
<tr>
<td style="text-align:left;">
3
</td>
<td style="text-align:left;">
pseudogene
</td>
<td style="text-align:right;">
49.53075
</td>
<td style="text-align:right;">
49.53131
</td>
<td style="text-align:left;">
positive
</td>
<td style="text-align:left;">
MGI_C57BL6J_5610652
</td>
<td style="text-align:left;">
Gm37424
</td>
<td style="text-align:left;">
ENSEMBL:ENSMUSG00000104437
</td>
<td style="text-align:left;">
MGI:5610652
</td>
<td style="text-align:left;">
pseudogene
</td>
<td style="text-align:left;">
predicted gene, 37424
</td>
</tr>
<tr>
<td style="text-align:left;">
3
</td>
<td style="text-align:left;">
pseudogene
</td>
<td style="text-align:right;">
49.60356
</td>
<td style="text-align:right;">
49.60388
</td>
<td style="text-align:left;">
positive
</td>
<td style="text-align:left;">
MGI_C57BL6J_5611489
</td>
<td style="text-align:left;">
Gm38261
</td>
<td style="text-align:left;">
ENSEMBL:ENSMUSG00000102731
</td>
<td style="text-align:left;">
MGI:5611489
</td>
<td style="text-align:left;">
pseudogene
</td>
<td style="text-align:left;">
predicted gene, 38261
</td>
</tr>
<tr>
<td style="text-align:left;">
3
</td>
<td style="text-align:left;">
pseudogene
</td>
<td style="text-align:right;">
49.60423
</td>
<td style="text-align:right;">
49.60447
</td>
<td style="text-align:left;">
positive
</td>
<td style="text-align:left;">
MGI_C57BL6J_5611316
</td>
<td style="text-align:left;">
Gm38088
</td>
<td style="text-align:left;">
ENSEMBL:ENSMUSG00000103249
</td>
<td style="text-align:left;">
MGI:5611316
</td>
<td style="text-align:left;">
pseudogene
</td>
<td style="text-align:left;">
predicted gene, 38088
</td>
</tr>
<tr>
<td style="text-align:left;">
3
</td>
<td style="text-align:left;">
pseudogene
</td>
<td style="text-align:right;">
49.63244
</td>
<td style="text-align:right;">
49.63297
</td>
<td style="text-align:left;">
negative
</td>
<td style="text-align:left;">
MGI_C57BL6J_3780498
</td>
<td style="text-align:left;">
Gm2328
</td>
<td style="text-align:left;">
NCBI_Gene:100039587,ENSEMBL:ENSMUSG00000104134
</td>
<td style="text-align:left;">
MGI:3780498
</td>
<td style="text-align:left;">
pseudogene
</td>
<td style="text-align:left;">
predicted gene 2328
</td>
</tr>
<tr>
<td style="text-align:left;">
3
</td>
<td style="text-align:left;">
pseudogene
</td>
<td style="text-align:right;">
49.64348
</td>
<td style="text-align:right;">
49.64413
</td>
<td style="text-align:left;">
positive
</td>
<td style="text-align:left;">
MGI_C57BL6J_5588900
</td>
<td style="text-align:left;">
Gm29741
</td>
<td style="text-align:left;">
NCBI_Gene:101055903,ENSEMBL:ENSMUSG00000103568
</td>
<td style="text-align:left;">
MGI:5588900
</td>
<td style="text-align:left;">
pseudogene
</td>
<td style="text-align:left;">
predicted gene, 29741
</td>
</tr>
<tr>
<td style="text-align:left;">
3
</td>
<td style="text-align:left;">
gene
</td>
<td style="text-align:right;">
49.65876
</td>
<td style="text-align:right;">
49.65887
</td>
<td style="text-align:left;">
positive
</td>
<td style="text-align:left;">
MGI_C57BL6J_5452873
</td>
<td style="text-align:left;">
Gm23096
</td>
<td style="text-align:left;">
ENSEMBL:ENSMUSG00000065381
</td>
<td style="text-align:left;">
MGI:5452873
</td>
<td style="text-align:left;">
snRNA gene
</td>
<td style="text-align:left;">
predicted gene, 23096
</td>
</tr>
<tr>
<td style="text-align:left;">
3
</td>
<td style="text-align:left;">
gene
</td>
<td style="text-align:right;">
49.74329
</td>
<td style="text-align:right;">
49.75738
</td>
<td style="text-align:left;">
negative
</td>
<td style="text-align:left;">
MGI_C57BL6J_1920423
</td>
<td style="text-align:left;">
Pcdh18
</td>
<td style="text-align:left;">
NCBI_Gene:73173,ENSEMBL:ENSMUSG00000037892
</td>
<td style="text-align:left;">
MGI:1920423
</td>
<td style="text-align:left;">
protein coding gene
</td>
<td style="text-align:left;">
protocadherin 18
</td>
</tr>
<tr>
<td style="text-align:left;">
3
</td>
<td style="text-align:left;">
pseudogene
</td>
<td style="text-align:right;">
49.78043
</td>
<td style="text-align:right;">
49.78089
</td>
<td style="text-align:left;">
negative
</td>
<td style="text-align:left;">
MGI_C57BL6J_5610778
</td>
<td style="text-align:left;">
Gm37550
</td>
<td style="text-align:left;">
ENSEMBL:ENSMUSG00000102943
</td>
<td style="text-align:left;">
MGI:5610778
</td>
<td style="text-align:left;">
pseudogene
</td>
<td style="text-align:left;">
predicted gene, 37550
</td>
</tr>
<tr>
<td style="text-align:left;">
3
</td>
<td style="text-align:left;">
gene
</td>
<td style="text-align:right;">
49.89253
</td>
<td style="text-align:right;">
50.49909
</td>
<td style="text-align:left;">
negative
</td>
<td style="text-align:left;">
MGI_C57BL6J_1347355
</td>
<td style="text-align:left;">
Slc7a11
</td>
<td style="text-align:left;">
NCBI_Gene:26570,ENSEMBL:ENSMUSG00000027737
</td>
<td style="text-align:left;">
MGI:1347355
</td>
<td style="text-align:left;">
protein coding gene
</td>
<td style="text-align:left;">
solute carrier family 7 (cationic amino acid transporter, y+ system), member 11
</td>
</tr>
</tbody>
</table>
</div>
</div>
<div id="additive-covariates" class="section level2">
<h2>Additive covariates</h2>
<div id="significance-thresholds-1" class="section level3">
<h3>Significance thresholds</h3>
<pre class="r"><code>operm &lt;- scan1perm(pr.qc, covars[&quot;rz.age&quot;], model=&quot;normal&quot;, addcovar=addcovar, n_perm=10, perm_Xsp=TRUE, chr_lengths=chr_lengths(gm$gmap))

summary_table&lt;-data.frame(unclass(summary(operm, alpha=c(0.01,  0.05, 0.1))))
names(summary_table) &lt;- c(&quot;autosomes&quot;,&quot;X&quot;)
summary_table$significance.level &lt;- rownames(summary_table)

rownames(summary_table) &lt;- NULL

summary_table[c(3,1:2)] %&gt;%
  kable(escape = F,align = c(&quot;ccc&quot;)) %&gt;%
  kable_styling(&quot;striped&quot;, full_width = T) %&gt;%
  column_spec(1, bold=TRUE)</code></pre>
<table class="table table-striped" style="margin-left: auto; margin-right: auto;">
<thead>
<tr>
<th style="text-align:center;">
significance.level
</th>
<th style="text-align:center;">
autosomes
</th>
<th style="text-align:center;">
X
</th>
</tr>
</thead>
<tbody>
<tr>
<td style="text-align:center;font-weight: bold;">
0.01
</td>
<td style="text-align:center;">
3.286066
</td>
<td style="text-align:center;">
4.064109
</td>
</tr>
<tr>
<td style="text-align:center;font-weight: bold;">
0.05
</td>
<td style="text-align:center;">
3.190259
</td>
<td style="text-align:center;">
3.397087
</td>
</tr>
<tr>
<td style="text-align:center;font-weight: bold;">
0.1
</td>
<td style="text-align:center;">
3.070201
</td>
<td style="text-align:center;">
2.524840
</td>
</tr>
</tbody>
</table>
</div>
<div id="genome-wide-scan-1" class="section level3">
<h3>Genome-wide scan</h3>
<pre class="r"><code>plot_lod&lt;-function(out,map){
  for (i in 1:dim(out)[2]){
#par(mar=c(5.1, 6.1, 1.1, 1.1))
    ymx &lt;- maxlod(out) # overall maximum LOD score
    plot(out, map, lodcolumn=i, col=&quot;slateblue&quot;, ylim=c(0, ymx+0.5))
    #legend(&quot;topright&quot;, lwd=2, colnames(out)[i], bg=&quot;gray90&quot;)
    title(main = paste0(colnames(out)[i], &quot; - ICI vs. EOI (with additive covariates) [positions in cM]&quot;))
    add_threshold(map,  summary(operm,alpha=0.1), col = &#39;purple&#39;)
    add_threshold(map,  summary(operm, alpha=0.05), col = &#39;red&#39;)
    add_threshold(map,  summary(operm, alpha=0.01), col = &#39;blue&#39;)

  }
}


print(&quot;with no kinship&quot;)</code></pre>
<p>[1] “with no kinship”</p>
<pre class="r"><code>outfa &lt;- scan1(pr.qc, covars[&quot;rz.age&quot;], Xcovar=Xcovar, model=&quot;normal&quot;, addcovar=addcovar)

plot_lod(outfa,gm$gmap)</code></pre>
<p><img src="figure/4.1.2_qtl.analysis_cont_rz.age_ici.vs.eoi_snpsqc_pheno.corrected.cleaned_5.batches_52.Rmd/scanonea-1.png" width="912" style="display: block; margin: auto;" /></p>
<pre class="r"><code>print(&quot;with normal kinship&quot;)</code></pre>
<p>[1] “with normal kinship”</p>
<pre class="r"><code>outka &lt;- scan1(pr.qc, covars[&quot;rz.age&quot;], Xcovar=Xcovar, model=&quot;normal&quot;, addcovar=addcovar, kinship=kinship)

plot_lod(outka,gm$gmap)</code></pre>
<p><img src="figure/4.1.2_qtl.analysis_cont_rz.age_ici.vs.eoi_snpsqc_pheno.corrected.cleaned_5.batches_52.Rmd/scanonea-2.png" width="912" style="display: block; margin: auto;" /></p>
<pre class="r"><code>print(&quot;with loco kinship&quot;)</code></pre>
<p>[1] “with loco kinship”</p>
<pre class="r"><code>outa &lt;- scan1(pr.qc, covars[&quot;rz.age&quot;], Xcovar=Xcovar, model=&quot;normal&quot;, addcovar=addcovar, kinship=K)

plot_lod(outa,gm$gmap)</code></pre>
<p><img src="figure/4.1.2_qtl.analysis_cont_rz.age_ici.vs.eoi_snpsqc_pheno.corrected.cleaned_5.batches_52.Rmd/scanonea-3.png" width="912" style="display: block; margin: auto;" /></p>
<pre class="r"><code>out = outa

print(paste0(&quot;number of samples in analysis = &quot;, as.data.frame(attributes(out)$sample_size)[1,]))</code></pre>
<p>[1] “number of samples in analysis = 267”</p>
</div>
<div id="lod-peaks-1" class="section level3">
<h3>LOD peaks</h3>
<div id="centimorgan-cm-1" class="section level4">
<h4>Centimorgan (cM)</h4>
<pre class="r"><code>peaks&lt;-find_peaks(out, gm$gmap, threshold=summary(operm,alpha=0.05)$A, thresholdX = summary(operm,alpha=0.05)$X, peakdrop=3, drop=1.5)
if(nrow(peaks) &gt; 0){
peaks$marker &lt;- find_marker(gm$gmap, chr=peaks$chr,pos=peaks$pos)
names(peaks)[2] &lt;- c(&quot;phenotype&quot;)
peaks &lt;- peaks[-1]

rownames(peaks) &lt;- NULL
print(kable(peaks, escape = F, align = c(&quot;cccccccc&quot;), &quot;html&quot;) 
  %&gt;% kable_styling(&quot;striped&quot;, full_width = T)%&gt;%
  column_spec(1, bold=TRUE)
  )

#plot only peak chromosomes

plot_lod_chr&lt;-function(out,map,chrom){
  for (i in 1:dim(out)[2]){
    #png(filename=paste0(&quot;/Users/chenm/Documents/qtl/Jai/&quot;,colnames(out)[i],  &quot;_lod.png&quot;))
    
    #par(mar=c(5.1, 6.1, 1.1, 1.1))
    ymx &lt;- maxlod(out) # overall maximum LOD score
    plot(out, map, chr = chrom, lodcolumn=i, col=&quot;slateblue&quot;, ylim=c(0, ymx+0.5))
    #legend(&quot;topright&quot;, lwd=2, colnames(out)[i], bg=&quot;gray90&quot;)
    title(main = paste0(colnames(out)[i], &quot; - chr&quot;, chrom, &quot; - ICI vs. EOI (with additive covariates) [positions in cM]&quot;))
    add_threshold(map,  summary(operm,alpha=0.1), col = &#39;purple&#39;)
    add_threshold(map,  summary(operm, alpha=0.05), col = &#39;red&#39;)
    add_threshold(map,  summary(operm, alpha=0.01), col = &#39;blue&#39;)
    #for (j in 1: dim(summary_table)[1]){
    #  abline(h=summary_table[j, i],col=&quot;red&quot;)
    #  text(x=400, y =summary_table[j, i]+0.12, labels = paste(&quot;p=&quot;, row.names(summary_table)[j]))
    #}
    #dev.off()

  }
}


} else {
  print(paste0(&quot;There are no peaks that have a LOD that reaches suggestive (p&lt;0.05) level of &quot;,summary(operm,alpha=0.05)$A, &quot; [autosomes]/&quot;,summary(operm,alpha=0.05)$X, &quot; [x-chromosome]&quot;))
}</code></pre>
<p>[1] “There are no peaks that have a LOD that reaches suggestive (p&lt;0.05) level of 3.1902587318752 [autosomes]/3.39708697838892 [x-chromosome]”</p>
<pre class="r"><code>if(nrow(peaks) &lt; 50 &amp; nrow(peaks) &gt; 0){

for(i in unique(peaks$chr)){
#for (i in 1:nrow(peaks)){
  #plot_lod_chr(out,gm$gmap, peaks$chr[i])
  plot_lod_chr(out,gm$gmap, i)
}

} else {
  print(paste0(&quot;There are too many peaks (&quot;,nrow(peaks),&quot; peaks) that have a LOD that reaches suggestive (p&lt;0.05) level of &quot;,summary(operm,alpha=0.05)$A, &quot; [autosomes]/&quot;,summary(operm,alpha=0.05)$X, &quot; [x-chromosome]&quot;))
}</code></pre>
<p>[1] “There are too many peaks (0 peaks) that have a LOD that reaches suggestive (p&lt;0.05) level of 3.1902587318752 [autosomes]/3.39708697838892 [x-chromosome]”</p>
</div>
<div id="megabase-mb-1" class="section level4">
<h4>Megabase (MB)</h4>
<pre class="r"><code>peaks_mba&lt;-find_peaks(out, gm$pmap, threshold=summary(operm,alpha=0.05)$A, thresholdX = summary(operm,alpha=0.05)$X, peakdrop=3, drop=1.5)
if(nrow(peaks) &gt; 0){
peaks_mba$marker &lt;- find_marker(gm$pmap, chr=peaks_mba$chr,pos=peaks_mba$pos)
names(peaks_mba)[2] &lt;- c(&quot;phenotype&quot;)
peaks_mba &lt;- peaks_mba[-1]

rownames(peaks_mba) &lt;- NULL
print(kable(peaks_mba, escape = F, align = c(&quot;cccccccc&quot;), &quot;html&quot;) 
  %&gt;% kable_styling(&quot;striped&quot;, full_width = T)%&gt;%
  column_spec(1, bold=TRUE)
  )

plot_lod_chr_mb&lt;-function(out,map,chrom){
  for (i in 1:dim(out)[2]){
    #png(filename=paste0(&quot;/Users/chenm/Documents/qtl/Jai/&quot;,colnames(out)[i],  &quot;_lod.png&quot;))
    
    #par(mar=c(5.1, 6.1, 1.1, 1.1))
    ymx &lt;- maxlod(out) # overall maximum LOD score
    plot(out, map, chr = chrom, lodcolumn=i, col=&quot;slateblue&quot;, ylim=c(0, ymx+0.5))
    #legend(&quot;topright&quot;, lwd=2, colnames(out)[i], bg=&quot;gray90&quot;)
    title(main = paste0(colnames(out)[i], &quot; - chr&quot;, chrom, &quot; - ICI vs. EOI (with additive covariates) [positions in cM]&quot;))
    add_threshold(map,  summary(operm,alpha=0.1), col = &#39;purple&#39;)
    add_threshold(map,  summary(operm, alpha=0.05), col = &#39;red&#39;)
    add_threshold(map,  summary(operm, alpha=0.01), col = &#39;blue&#39;)
    #for (j in 1: dim(summary_table)[1]){
    #  abline(h=summary_table[j, i],col=&quot;red&quot;)
    #  text(x=400, y =summary_table[j, i]+0.12, labels = paste(&quot;p=&quot;, row.names(summary_table)[j]))
    #}
    #dev.off()
  }
}

} else {
  print(paste0(&quot;There are no peaks that have a LOD that reaches suggestive (p&lt;0.05) level of &quot;,summary(operm,alpha=0.05)$A, &quot; [autosomes]/&quot;,summary(operm,alpha=0.05)$X, &quot; [x-chromosome]&quot;))
}</code></pre>
<p>[1] “There are no peaks that have a LOD that reaches suggestive (p&lt;0.05) level of 3.1902587318752 [autosomes]/3.39708697838892 [x-chromosome]”</p>
<pre class="r"><code>if(nrow(peaks_mba) &lt; 50 &amp; nrow(peaks_mba) &gt; 0){

for(i in unique(peaks_mba$chr)){
#for (i in 1:nrow(peaks_mba)){
  #plot_lod_chr_mb(out,gm$pmap, peaks_mba$chr[i])
  plot_lod_chr_mb(out,gm$pmap,i)
}

} else {
  print(paste0(&quot;There are too many peaks (&quot;,nrow(peaks_mba),&quot; peaks) that have a LOD that reaches suggestive (p&lt;0.05) level of &quot;,summary(operm,alpha=0.05)$A, &quot; [autosomes]/&quot;,summary(operm,alpha=0.05)$X, &quot; [x-chromosome]&quot;))
}</code></pre>
<p>[1] “There are too many peaks (0 peaks) that have a LOD that reaches suggestive (p&lt;0.05) level of 3.1902587318752 [autosomes]/3.39708697838892 [x-chromosome]”</p>
</div>
</div>
<div id="qtl-effects-1" class="section level3">
<h3>QTL effects</h3>
<p>For each peak LOD location we give a list of gene</p>
<pre class="r"><code>if(nrow(peaks) &lt; 50 &amp; nrow(peaks) &gt; 0){
for (i in 1:nrow(peaks)){
#for (i in 1:1){
  #Plot 1
  g &lt;- maxmarg(pr.qc, gm$gmap, chr=peaks$chr[i], pos=peaks$pos[i], return_char=TRUE)
  #png(filename=paste0(&quot;/Users/chenm/Documents/qtl/Jai/&quot;,&quot;qtl_effect_&quot;, i, &quot;.png&quot;))
  #par(mar=c(4.1, 4.1, 1.5, 0.6))
  plot_pxg(g, covars[,peaks$phenotype[i]], ylab=peaks$phenotype[i], sort=FALSE)
  title(main = paste0(&quot;chr: &quot;, chr=peaks$chr[i],&quot;; pos: &quot;, peaks$pos[i], &quot;cM /&quot;,peaks_mba$pos[i],&quot;MB\n(&quot;,peaks$phenotype[i],&quot; )&quot;), line=0.2)
  ##dev.off()

  chr = peaks$chr[i]

# Plot 2
  pr_sub &lt;- pull_genoprobint(pr.qc, gm$gmap, chr, c(peaks$ci_lo[i], peaks$ci_hi[i]))
  #coeff &lt;- scan1coef(pr[,chr], cross$pheno[,peaks$lodcolumn[i]], addcovar = addcovar)
  #coeff &lt;- scan1coef(pr[,chr], cross$pheno[,peaks$lodcolumn[i]], Xcovar=Xcovar)
  #coeff &lt;- scan1coef(pr.qc[,chr], gm$covar[peaks$lodcolumn[i]], model=&quot;binary&quot;)
  #coeff_sub &lt;- scan1coef(pr_sub[,chr], gm$covar[peaks$lodcolumn[i]], model=&quot;binary&quot;)
  blup &lt;- scan1blup(pr.qc[,chr], covars[peaks$phenotype[i]])
  blup_sub &lt;- scan1blup(pr_sub[,chr], covars[peaks$phenotype[i]])

  write.csv(as.data.frame(blup_sub), paste0(&quot;data/ici.vs.eoi_&quot;,peaks$phenotype[i],&quot;-additive.covariates_blup_sub_chr-&quot;,chr,&quot;_peak.marker-&quot;,peaks$marker[i],&quot;_lod.drop-1.5_snpsqc_5.batches_52.csv&quot;), quote=F)

  #plot_coef(coeff, 
  #     gm$gmap, columns=1:2,
  #     bgcolor=&quot;gray95&quot;, legend=&quot;bottomleft&quot;, 
  #     main = paste(&quot;chr&quot;, chr=peaks$chr[i],&quot;; pos: &quot;, peaks$pos[i], &quot;cM /&quot;,peaks_mba$pos[i],&quot;MB\n(&quot;,peaks$lodcolumn[i],&quot; [scan1coeff; positions in cM] )&quot;)
  #     )

  #plot_coef(coeff_sub, 
  #     gm$gmap, columns=1:2,
  #     bgcolor=&quot;gray95&quot;, legend=&quot;bottomleft&quot;, 
  #     main = paste(&quot;chr&quot;, chr=peaks$chr[i],&quot;; pos: &quot;, peaks$pos[i], &quot;cM /&quot;,peaks_mba$pos[i],&quot;MB\n(&quot;,peaks$lodcolumn[i],&quot;; 1.5 LOD drop interval [scan1coeff; positions in cM] ) &quot;)
  #     )


  plot_coef(blup, 
       gm$gmap, columns=1:2,
       bgcolor=&quot;gray95&quot;, legend=&quot;bottomleft&quot;, 
       main = paste0(&quot;chr: &quot;, chr=peaks$chr[i],&quot;; pos: &quot;, peaks$pos[i], &quot;cM /&quot;,peaks_mba$pos[i],&quot;MB\n(&quot;,peaks$phenotype[i],&quot; [scan1blup; positions in cM] )&quot;)
       )

  plot_coef(blup_sub, 
       gm$gmap, columns=1:2,
       bgcolor=&quot;gray95&quot;, legend=&quot;bottomleft&quot;, 
       main = paste0(&quot;chr: &quot;, chr=peaks$chr[i],&quot;; pos: &quot;, peaks$pos[i], &quot;cM /&quot;,peaks_mba$pos[i],&quot;MB\n(&quot;,peaks$phenotype[i],&quot;; 1.5 LOD drop interval [scan1blup; positions in cM] )&quot;)
       )


 # Plot 3
  #c2effB &lt;- scan1coef(pr.qc[,chr], gm$covar[peaks$lodcolumn[i]], model=&quot;binary&quot;, contrasts=cbind(a=c(-1, 0), d=c(0, -1)))
  #c2effBb &lt;- scan1blup(pr.qc[,chr], gm$covar[peaks$lodcolumn[i]], contrasts=cbind(a=c(-1, 0), d=c(0, -1)))
  ##c2effB &lt;- scan1coef(pr[,chr], cross$pheno[,peaks$lodcolumn[i]], addcovar = addcovar, contrasts=cbind(mu=c(1,1,1), a=c(-1, 0, 1), d=c(0, 1, 0)))
  ##c2effB &lt;- scan1coef(pr[,chr], cross$pheno[,peaks$lodcolumn[i]],Xcovar=Xcovar, contrasts=cbind(mu=c(1,1,1), a=c(-1, 0, 1), d=c(0, 1, 0)))
  #plot(c2effB, gm$gmap[chr], columns=1:2,
  #     bgcolor=&quot;gray95&quot;, legend=&quot;bottomleft&quot;, 
  #     main = paste(&quot;chr&quot;, chr=peaks$chr[i], &quot;pos&quot;, peaks$pos[i], &quot;(&quot;,peaks$lodcolumn[i],&quot;)&quot;)
  #     )
  #plot(c2effBb, gm$gmap[chr], columns=1:2,
  #     bgcolor=&quot;gray95&quot;, legend=&quot;bottomleft&quot;, 
  #     main = paste(&quot;chr&quot;, chr=peaks$chr[i], &quot;pos&quot;, peaks$pos[i], &quot;(&quot;,peaks$lodcolumn[i],&quot;)&quot;)
  #     )
  ##last_coef &lt;- unclass(c2effB)[nrow(c2effB),2:3] # last two coefficients
  ##for(t in seq(along=last_coef))
  ##  axis(side=4, at=last_coef[t], names(last_coef)[t], tick=FALSE)


  #Table 1
  chr = peaks_mba$chr[i]
  start=as.numeric(peaks_mba$ci_lo[i])
  end=as.numeric(peaks_mba$ci_hi[i])

  genesgss = query_genes(chr, start, end)

  write.csv(genesgss, file=paste0(&quot;data/ici.vs.eoi_&quot;,peaks$phenotype[i],&quot;-additive.covariates_genes_chr-&quot;,chr,&quot;_peak.marker-&quot;,peaks$marker[i],&quot;_lod.drop-1.5_snpsqc_5.batches_52.csv&quot;), quote=F)

  rownames(genesgss) &lt;- NULL
  genesgss$strand_old = genesgss$strand
  genesgss$strand[genesgss$strand==&quot;+&quot;] &lt;- &quot;positive&quot;
  genesgss$strand[genesgss$strand==&quot;-&quot;] &lt;- &quot;negative&quot;

  #genesgss &lt;- 
  #table &lt;- 
  #genesgss[,c(&quot;chr&quot;,&quot;type&quot;,&quot;start&quot;,&quot;stop&quot;,&quot;strand&quot;,&quot;ID&quot;,&quot;Name&quot;,&quot;Dbxref&quot;,&quot;gene_id&quot;,&quot;mgi_type&quot;,&quot;description&quot;)] %&gt;%
  #kable(escape = F,align = c(&quot;ccccccccccc&quot;)) %&gt;%
  #kable_styling(&quot;striped&quot;, full_width = T) #%&gt;% 
  #cat #%&gt;%
  #column_spec(1, bold=TRUE)
#
  #print(kable(genesgss[,c(&quot;chr&quot;,&quot;type&quot;,&quot;start&quot;,&quot;stop&quot;,&quot;strand&quot;,&quot;ID&quot;,&quot;Name&quot;,&quot;Dbxref&quot;,&quot;gene_id&quot;,&quot;mgi_type&quot;,&quot;description&quot;)], escape = F,align = c(&quot;ccccccccccc&quot;)))

  print(kable(genesgss[,c(&quot;chr&quot;,&quot;type&quot;,&quot;start&quot;,&quot;stop&quot;,&quot;strand&quot;,&quot;ID&quot;,&quot;Name&quot;,&quot;Dbxref&quot;,&quot;gene_id&quot;,&quot;mgi_type&quot;,&quot;description&quot;)], &quot;html&quot;) %&gt;% kable_styling(&quot;striped&quot;, full_width = T))

  #table
  

}
} else {
  print(paste0(&quot;There are too many peaks (&quot;,nrow(peaks_mba),&quot; peaks) that have a LOD that reaches suggestive (p&lt;0.05) level of &quot;,summary(operm,alpha=0.05)$A, &quot; [autosomes]/&quot;,summary(operm,alpha=0.05)$X, &quot; [x-chromosome]&quot;))
}</code></pre>
<p>[1] “There are too many peaks (0 peaks) that have a LOD that reaches suggestive (p&lt;0.05) level of 3.1902587318752 [autosomes]/3.39708697838892 [x-chromosome]”</p>
</div>
</div>
<div id="interactive-term" class="section level2">
<h2>Interactive term</h2>
<div id="significance-thresholds-2" class="section level3">
<h3>Significance thresholds</h3>
<pre class="r"><code>operm &lt;- scan1perm(pr.qc, covars[&quot;rz.age&quot;], model=&quot;normal&quot;, addcovar=addcovar, intcovar=addcovar, n_perm=10, perm_Xsp=TRUE, chr_lengths=chr_lengths(gm$gmap))

summary_table&lt;-data.frame(unclass(summary(operm, alpha=c(0.01,  0.05, 0.1))))
names(summary_table) &lt;- c(&quot;autosomes&quot;,&quot;X&quot;)
summary_table$significance.level &lt;- rownames(summary_table)

rownames(summary_table) &lt;- NULL

summary_table[c(3,1:2)] %&gt;%
  kable(escape = F,align = c(&quot;ccc&quot;)) %&gt;%
  kable_styling(&quot;striped&quot;, full_width = T) %&gt;%
  column_spec(1, bold=TRUE)</code></pre>
<table class="table table-striped" style="margin-left: auto; margin-right: auto;">
<thead>
<tr>
<th style="text-align:center;">
significance.level
</th>
<th style="text-align:center;">
autosomes
</th>
<th style="text-align:center;">
X
</th>
</tr>
</thead>
<tbody>
<tr>
<td style="text-align:center;font-weight: bold;">
0.01
</td>
<td style="text-align:center;">
6.215976
</td>
<td style="text-align:center;">
6.246922
</td>
</tr>
<tr>
<td style="text-align:center;font-weight: bold;">
0.05
</td>
<td style="text-align:center;">
6.116958
</td>
<td style="text-align:center;">
6.240626
</td>
</tr>
<tr>
<td style="text-align:center;font-weight: bold;">
0.1
</td>
<td style="text-align:center;">
5.992876
</td>
<td style="text-align:center;">
6.232392
</td>
</tr>
</tbody>
</table>
</div>
<div id="genome-wide-scan-2" class="section level3">
<h3>Genome-wide scan</h3>
<pre class="r"><code>plot_lod&lt;-function(out,map){
  for (i in 1:dim(out)[2]){
#par(mar=c(5.1, 6.1, 1.1, 1.1))
    ymx &lt;- maxlod(out) # overall maximum LOD score
    plot(out, map, lodcolumn=i, col=&quot;slateblue&quot;, ylim=c(0, ymx+0.5))
    #legend(&quot;topright&quot;, lwd=2, colnames(out)[i], bg=&quot;gray90&quot;)
    title(main = paste0(colnames(out)[i], &quot; - ICI vs. EOI (with interactive covariate) [positions in cM]&quot;))
    add_threshold(map,  summary(operm,alpha=0.1), col = &#39;purple&#39;)
    add_threshold(map,  summary(operm, alpha=0.05), col = &#39;red&#39;)
    add_threshold(map,  summary(operm, alpha=0.01), col = &#39;blue&#39;)

  }
}


print(&quot;with no kinship&quot;)</code></pre>
<p>[1] “with no kinship”</p>
<pre class="r"><code>outfi &lt;- scan1(pr.qc, covars[&quot;rz.age&quot;], Xcovar=Xcovar, model=&quot;normal&quot;, addcovar=addcovar, intcovar=addcovar)

plot_lod(outfi,gm$gmap)</code></pre>
<p><img src="figure/4.1.2_qtl.analysis_cont_rz.age_ici.vs.eoi_snpsqc_pheno.corrected.cleaned_5.batches_52.Rmd/scanonei-1.png" width="912" style="display: block; margin: auto;" /></p>
<pre class="r"><code>outf.new &lt;- outfi-outfa

plot_lod(outf.new,gm$gmap)</code></pre>
<p><img src="figure/4.1.2_qtl.analysis_cont_rz.age_ici.vs.eoi_snpsqc_pheno.corrected.cleaned_5.batches_52.Rmd/scanonei-2.png" width="912" style="display: block; margin: auto;" /></p>
<pre class="r"><code>print(&quot;with normal kinship&quot;)</code></pre>
<p>[1] “with normal kinship”</p>
<pre class="r"><code>outki &lt;- scan1(pr.qc, covars[&quot;rz.age&quot;], Xcovar=Xcovar, model=&quot;normal&quot;, addcovar=addcovar, intcovar=addcovar, kinship=kinship)

plot_lod(outki,gm$gmap)</code></pre>
<p><img src="figure/4.1.2_qtl.analysis_cont_rz.age_ici.vs.eoi_snpsqc_pheno.corrected.cleaned_5.batches_52.Rmd/scanonei-3.png" width="912" style="display: block; margin: auto;" /></p>
<pre class="r"><code>outk.new &lt;- outki-outka

plot_lod(outk.new,gm$gmap)</code></pre>
<p><img src="figure/4.1.2_qtl.analysis_cont_rz.age_ici.vs.eoi_snpsqc_pheno.corrected.cleaned_5.batches_52.Rmd/scanonei-4.png" width="912" style="display: block; margin: auto;" /></p>
<pre class="r"><code>print(&quot;with loco kinship&quot;)</code></pre>
<p>[1] “with loco kinship”</p>
<pre class="r"><code>outi &lt;- scan1(pr.qc, covars[&quot;rz.age&quot;], Xcovar=Xcovar, model=&quot;normal&quot;, addcovar=addcovar, intcovar=addcovar, kinship=K)

plot_lod(outi,gm$gmap)</code></pre>
<p><img src="figure/4.1.2_qtl.analysis_cont_rz.age_ici.vs.eoi_snpsqc_pheno.corrected.cleaned_5.batches_52.Rmd/scanonei-5.png" width="912" style="display: block; margin: auto;" /></p>
<pre class="r"><code>outi.new &lt;- outi-outa

plot_lod(outi.new,gm$gmap)</code></pre>
<p><img src="figure/4.1.2_qtl.analysis_cont_rz.age_ici.vs.eoi_snpsqc_pheno.corrected.cleaned_5.batches_52.Rmd/scanonei-6.png" width="912" style="display: block; margin: auto;" /></p>
<pre class="r"><code>out = outi.new


print(paste0(&quot;number of samples in analysis = &quot;, as.data.frame(attributes(out)$sample_size)[1,]))</code></pre>
<p>[1] “number of samples in analysis = 267”</p>
</div>
<div id="lod-peaks-2" class="section level3">
<h3>LOD peaks</h3>
<div id="centimorgan-cm-2" class="section level4">
<h4>Centimorgan (cM)</h4>
<pre class="r"><code>peaks&lt;-find_peaks(out, gm$gmap, threshold=summary(operm,alpha=0.05)$A, thresholdX = summary(operm,alpha=0.05)$X, peakdrop=3, drop=1.5)
if(nrow(peaks) &gt; 0){
peaks$marker &lt;- find_marker(gm$gmap, chr=peaks$chr,pos=peaks$pos)
names(peaks)[2] &lt;- c(&quot;phenotype&quot;)
peaks &lt;- peaks[-1]

rownames(peaks) &lt;- NULL
print(kable(peaks, escape = F, align = c(&quot;cccccccc&quot;), &quot;html&quot;) 
  %&gt;% kable_styling(&quot;striped&quot;, full_width = T)%&gt;%
  column_spec(1, bold=TRUE)
  )

#plot only peak chromosomes

plot_lod_chr&lt;-function(out,map,chrom){
  for (i in 1:dim(out)[2]){
    #png(filename=paste0(&quot;/Users/chenm/Documents/qtl/Jai/&quot;,colnames(out)[i],  &quot;_lod.png&quot;))
    
    #par(mar=c(5.1, 6.1, 1.1, 1.1))
    ymx &lt;- maxlod(out) # overall maximum LOD score
    plot(out, map, chr = chrom, lodcolumn=i, col=&quot;slateblue&quot;, ylim=c(0, ymx+0.5))
    #legend(&quot;topright&quot;, lwd=2, colnames(out)[i], bg=&quot;gray90&quot;)
    title(main = paste0(colnames(out)[i], &quot; - chr&quot;, chrom, &quot; - ICI vs. EOI (with interactive covariate) [positions in cM]&quot;))
    add_threshold(map,  summary(operm,alpha=0.1), col = &#39;purple&#39;)
    add_threshold(map,  summary(operm, alpha=0.05), col = &#39;red&#39;)
    add_threshold(map,  summary(operm, alpha=0.01), col = &#39;blue&#39;)
    #for (j in 1: dim(summary_table)[1]){
    #  abline(h=summary_table[j, i],col=&quot;red&quot;)
    #  text(x=400, y =summary_table[j, i]+0.12, labels = paste(&quot;p=&quot;, row.names(summary_table)[j]))
    #}
    #dev.off()

  }
}


} else {
  print(paste0(&quot;There are no peaks that have a LOD that reaches suggestive (p&lt;0.05) level of &quot;,summary(operm,alpha=0.05)$A, &quot; [autosomes]/&quot;,summary(operm,alpha=0.05)$X, &quot; [x-chromosome]&quot;))
}</code></pre>
<p>[1] “There are no peaks that have a LOD that reaches suggestive (p&lt;0.05) level of 6.11695769275088 [autosomes]/6.24062574447046 [x-chromosome]”</p>
<pre class="r"><code>if(nrow(peaks) &lt; 50 &amp; nrow(peaks) &gt; 0){

for(i in unique(peaks$chr)){
#for (i in 1:nrow(peaks)){
  #plot_lod_chr(out,gm$gmap, peaks$chr[i])
  plot_lod_chr(out,gm$gmap, i)
}

} else {
  print(paste0(&quot;There are too many peaks (&quot;,nrow(peaks),&quot; peaks) that have a LOD that reaches suggestive (p&lt;0.05) level of &quot;,summary(operm,alpha=0.05)$A, &quot; [autosomes]/&quot;,summary(operm,alpha=0.05)$X, &quot; [x-chromosome]&quot;))
}</code></pre>
<p>[1] “There are too many peaks (0 peaks) that have a LOD that reaches suggestive (p&lt;0.05) level of 6.11695769275088 [autosomes]/6.24062574447046 [x-chromosome]”</p>
</div>
<div id="megabase-mb-2" class="section level4">
<h4>Megabase (MB)</h4>
<pre class="r"><code>peaks_mba&lt;-find_peaks(out, gm$pmap, threshold=summary(operm,alpha=0.05)$A, thresholdX = summary(operm,alpha=0.05)$X, peakdrop=3, drop=1.5)
if(nrow(peaks) &gt; 0){
peaks_mba$marker &lt;- find_marker(gm$pmap, chr=peaks_mba$chr,pos=peaks_mba$pos)
names(peaks_mba)[2] &lt;- c(&quot;phenotype&quot;)
peaks_mba &lt;- peaks_mba[-1]

rownames(peaks_mba) &lt;- NULL
print(kable(peaks_mba, escape = F, align = c(&quot;cccccccc&quot;), &quot;html&quot;) 
  %&gt;% kable_styling(&quot;striped&quot;, full_width = T)%&gt;%
  column_spec(1, bold=TRUE)
  )

plot_lod_chr_mb&lt;-function(out,map,chrom){
  for (i in 1:dim(out)[2]){
    #png(filename=paste0(&quot;/Users/chenm/Documents/qtl/Jai/&quot;,colnames(out)[i],  &quot;_lod.png&quot;))
    
    #par(mar=c(5.1, 6.1, 1.1, 1.1))
    ymx &lt;- maxlod(out) # overall maximum LOD score
    plot(out, map, chr = chrom, lodcolumn=i, col=&quot;slateblue&quot;, ylim=c(0, ymx+0.5))
    #legend(&quot;topright&quot;, lwd=2, colnames(out)[i], bg=&quot;gray90&quot;)
    title(main = paste0(colnames(out)[i], &quot; - chr&quot;, chrom, &quot; - ICI vs. EOI (with interactive covariate) [positions in cM]&quot;))
    add_threshold(map,  summary(operm,alpha=0.1), col = &#39;purple&#39;)
    add_threshold(map,  summary(operm, alpha=0.05), col = &#39;red&#39;)
    add_threshold(map,  summary(operm, alpha=0.01), col = &#39;blue&#39;)
    #for (j in 1: dim(summary_table)[1]){
    #  abline(h=summary_table[j, i],col=&quot;red&quot;)
    #  text(x=400, y =summary_table[j, i]+0.12, labels = paste(&quot;p=&quot;, row.names(summary_table)[j]))
    #}
    #dev.off()
  }
}

} else {
  print(paste0(&quot;There are no peaks that have a LOD that reaches suggestive (p&lt;0.05) level of &quot;,summary(operm,alpha=0.05)$A, &quot; [autosomes]/&quot;,summary(operm,alpha=0.05)$X, &quot; [x-chromosome]&quot;))
}</code></pre>
<p>[1] “There are no peaks that have a LOD that reaches suggestive (p&lt;0.05) level of 6.11695769275088 [autosomes]/6.24062574447046 [x-chromosome]”</p>
<pre class="r"><code>if(nrow(peaks_mba) &lt; 50 &amp; nrow(peaks_mba) &gt; 0){

for(i in unique(peaks_mba$chr)){
#for (i in 1:nrow(peaks_mba)){
  #plot_lod_chr_mb(out,gm$pmap, peaks_mba$chr[i])
  plot_lod_chr_mb(out,gm$pmap,i)
}

} else {
  print(paste0(&quot;There are too many peaks (&quot;,nrow(peaks_mba),&quot; peaks) that have a LOD that reaches suggestive (p&lt;0.05) level of &quot;,summary(operm,alpha=0.05)$A, &quot; [autosomes]/&quot;,summary(operm,alpha=0.05)$X, &quot; [x-chromosome]&quot;))
}</code></pre>
<p>[1] “There are too many peaks (0 peaks) that have a LOD that reaches suggestive (p&lt;0.05) level of 6.11695769275088 [autosomes]/6.24062574447046 [x-chromosome]”</p>
</div>
</div>
<div id="qtl-effects-2" class="section level3">
<h3>QTL effects</h3>
<p>For each peak LOD location we give a list of gene</p>
<pre class="r"><code>if(nrow(peaks) &lt; 50 &amp; nrow(peaks) &gt; 0){
for (i in 1:nrow(peaks)){
#for (i in 1:1){
  #Plot 1
  g &lt;- maxmarg(pr.qc, gm$gmap, chr=peaks$chr[i], pos=peaks$pos[i], return_char=TRUE)
  #png(filename=paste0(&quot;/Users/chenm/Documents/qtl/Jai/&quot;,&quot;qtl_effect_&quot;, i, &quot;.png&quot;))
  #par(mar=c(4.1, 4.1, 1.5, 0.6))
  plot_pxg(g, covars[,peaks$phenotype[i]], ylab=peaks$phenotype[i], sort=FALSE)
  title(main = paste0(&quot;chr: &quot;, chr=peaks$chr[i],&quot;; pos: &quot;, peaks$pos[i], &quot;cM /&quot;,peaks_mba$pos[i],&quot;MB\n(&quot;,peaks$phenotype[i],&quot; )&quot;), line=0.2)
  ##dev.off()

  chr = peaks$chr[i]

# Plot 2
  pr_sub &lt;- pull_genoprobint(pr.qc, gm$gmap, chr, c(peaks$ci_lo[i], peaks$ci_hi[i]))
  #coeff &lt;- scan1coef(pr[,chr], cross$pheno[,peaks$lodcolumn[i]], addcovar = addcovar)
  #coeff &lt;- scan1coef(pr[,chr], cross$pheno[,peaks$lodcolumn[i]], Xcovar=Xcovar)
  #coeff &lt;- scan1coef(pr.qc[,chr], gm$covar[peaks$lodcolumn[i]], model=&quot;binary&quot;)
  #coeff_sub &lt;- scan1coef(pr_sub[,chr], gm$covar[peaks$lodcolumn[i]], model=&quot;binary&quot;)
  blup &lt;- scan1blup(pr.qc[,chr], covars[peaks$phenotype[i]])
  blup_sub &lt;- scan1blup(pr_sub[,chr], covars[peaks$phenotype[i]])

  write.csv(as.data.frame(blup_sub), paste0(&quot;data/ici.vs.eoi_&quot;,peaks$phenotype[i],&quot;-interactive.covariate_blup_sub_chr-&quot;,chr,&quot;_peak.marker-&quot;,peaks$marker[i],&quot;_lod.drop-1.5_snpsqc_5.batches_52.csv&quot;), quote=F)

  #plot_coef(coeff, 
  #     gm$gmap, columns=1:2,
  #     bgcolor=&quot;gray95&quot;, legend=&quot;bottomleft&quot;, 
  #     main = paste(&quot;chr&quot;, chr=peaks$chr[i],&quot;; pos: &quot;, peaks$pos[i], &quot;cM /&quot;,peaks_mba$pos[i],&quot;MB\n(&quot;,peaks$lodcolumn[i],&quot; [scan1coeff; positions in cM] )&quot;)
  #     )

  #plot_coef(coeff_sub, 
  #     gm$gmap, columns=1:2,
  #     bgcolor=&quot;gray95&quot;, legend=&quot;bottomleft&quot;, 
  #     main = paste(&quot;chr&quot;, chr=peaks$chr[i],&quot;; pos: &quot;, peaks$pos[i], &quot;cM /&quot;,peaks_mba$pos[i],&quot;MB\n(&quot;,peaks$lodcolumn[i],&quot;; 1.5 LOD drop interval [scan1coeff; positions in cM] ) &quot;)
  #     )


  plot_coef(blup, 
       gm$gmap, columns=1:2,
       bgcolor=&quot;gray95&quot;, legend=&quot;bottomleft&quot;, 
       main = paste0(&quot;chr: &quot;, chr=peaks$chr[i],&quot;; pos: &quot;, peaks$pos[i], &quot;cM /&quot;,peaks_mba$pos[i],&quot;MB\n(&quot;,peaks$phenotype[i],&quot; [scan1blup; positions in cM] )&quot;)
       )

  plot_coef(blup_sub, 
       gm$gmap, columns=1:2,
       bgcolor=&quot;gray95&quot;, legend=&quot;bottomleft&quot;, 
       main = paste0(&quot;chr: &quot;, chr=peaks$chr[i],&quot;; pos: &quot;, peaks$pos[i], &quot;cM /&quot;,peaks_mba$pos[i],&quot;MB\n(&quot;,peaks$phenotype[i],&quot;; 1.5 LOD drop interval [scan1blup; positions in cM] )&quot;)
       )


 # Plot 3
  #c2effB &lt;- scan1coef(pr.qc[,chr], gm$covar[peaks$lodcolumn[i]], model=&quot;binary&quot;, contrasts=cbind(a=c(-1, 0), d=c(0, -1)))
  #c2effBb &lt;- scan1blup(pr.qc[,chr], gm$covar[peaks$lodcolumn[i]], contrasts=cbind(a=c(-1, 0), d=c(0, -1)))
  ##c2effB &lt;- scan1coef(pr[,chr], cross$pheno[,peaks$lodcolumn[i]], addcovar = addcovar, contrasts=cbind(mu=c(1,1,1), a=c(-1, 0, 1), d=c(0, 1, 0)))
  ##c2effB &lt;- scan1coef(pr[,chr], cross$pheno[,peaks$lodcolumn[i]],Xcovar=Xcovar, contrasts=cbind(mu=c(1,1,1), a=c(-1, 0, 1), d=c(0, 1, 0)))
  #plot(c2effB, gm$gmap[chr], columns=1:2,
  #     bgcolor=&quot;gray95&quot;, legend=&quot;bottomleft&quot;, 
  #     main = paste(&quot;chr&quot;, chr=peaks$chr[i], &quot;pos&quot;, peaks$pos[i], &quot;(&quot;,peaks$lodcolumn[i],&quot;)&quot;)
  #     )
  #plot(c2effBb, gm$gmap[chr], columns=1:2,
  #     bgcolor=&quot;gray95&quot;, legend=&quot;bottomleft&quot;, 
  #     main = paste(&quot;chr&quot;, chr=peaks$chr[i], &quot;pos&quot;, peaks$pos[i], &quot;(&quot;,peaks$lodcolumn[i],&quot;)&quot;)
  #     )
  ##last_coef &lt;- unclass(c2effB)[nrow(c2effB),2:3] # last two coefficients
  ##for(t in seq(along=last_coef))
  ##  axis(side=4, at=last_coef[t], names(last_coef)[t], tick=FALSE)


  #Table 1
  chr = peaks_mba$chr[i]
  start=as.numeric(peaks_mba$ci_lo[i])
  end=as.numeric(peaks_mba$ci_hi[i])

  genesgss = query_genes(chr, start, end)

  write.csv(genesgss, file=paste0(&quot;data/ici.vs.eoi_&quot;,peaks$phenotype[i],&quot;-interactive.covariate_genes_chr-&quot;,chr,&quot;_peak.marker-&quot;,peaks$marker[i],&quot;_lod.drop-1.5_snpsqc_5.batches_52.csv&quot;), quote=F)

  rownames(genesgss) &lt;- NULL
  genesgss$strand_old = genesgss$strand
  genesgss$strand[genesgss$strand==&quot;+&quot;] &lt;- &quot;positive&quot;
  genesgss$strand[genesgss$strand==&quot;-&quot;] &lt;- &quot;negative&quot;

  #genesgss &lt;- 
  #table &lt;- 
  #genesgss[,c(&quot;chr&quot;,&quot;type&quot;,&quot;start&quot;,&quot;stop&quot;,&quot;strand&quot;,&quot;ID&quot;,&quot;Name&quot;,&quot;Dbxref&quot;,&quot;gene_id&quot;,&quot;mgi_type&quot;,&quot;description&quot;)] %&gt;%
  #kable(escape = F,align = c(&quot;ccccccccccc&quot;)) %&gt;%
  #kable_styling(&quot;striped&quot;, full_width = T) #%&gt;% 
  #cat #%&gt;%
  #column_spec(1, bold=TRUE)
#
  #print(kable(genesgss[,c(&quot;chr&quot;,&quot;type&quot;,&quot;start&quot;,&quot;stop&quot;,&quot;strand&quot;,&quot;ID&quot;,&quot;Name&quot;,&quot;Dbxref&quot;,&quot;gene_id&quot;,&quot;mgi_type&quot;,&quot;description&quot;)], escape = F,align = c(&quot;ccccccccccc&quot;)))

  print(kable(genesgss[,c(&quot;chr&quot;,&quot;type&quot;,&quot;start&quot;,&quot;stop&quot;,&quot;strand&quot;,&quot;ID&quot;,&quot;Name&quot;,&quot;Dbxref&quot;,&quot;gene_id&quot;,&quot;mgi_type&quot;,&quot;description&quot;)], &quot;html&quot;) %&gt;% kable_styling(&quot;striped&quot;, full_width = T))

  #table
  

}
} else {
  print(paste0(&quot;There are too many peaks (&quot;,nrow(peaks_mba),&quot; peaks) that have a LOD that reaches suggestive (p&lt;0.05) level of &quot;,summary(operm,alpha=0.05)$A, &quot; [autosomes]/&quot;,summary(operm,alpha=0.05)$X, &quot; [x-chromosome]&quot;))
}</code></pre>
<p>[1] “There are too many peaks (0 peaks) that have a LOD that reaches suggestive (p&lt;0.05) level of 6.11695769275088 [autosomes]/6.24062574447046 [x-chromosome]”</p>
<br>
<p>
<button type="button" class="btn btn-default btn-workflowr btn-workflowr-sessioninfo" data-toggle="collapse" data-target="#workflowr-sessioninfo" style="display: block;">
<span class="glyphicon glyphicon-wrench" aria-hidden="true"></span> Session information
</button>
</p>
<div id="workflowr-sessioninfo" class="collapse">
<pre><code>R version 3.6.2 (2019-12-12)
Platform: x86_64-apple-darwin15.6.0 (64-bit)
Running under: macOS Catalina 10.15.7

Matrix products: default
BLAS:   /Library/Frameworks/R.framework/Versions/3.6/Resources/lib/libRblas.0.dylib
LAPACK: /Library/Frameworks/R.framework/Versions/3.6/Resources/lib/libRlapack.dylib

locale:
[1] en_AU.UTF-8/en_AU.UTF-8/en_AU.UTF-8/C/en_AU.UTF-8/en_AU.UTF-8

attached base packages:
[1] stats     graphics  grDevices utils     datasets  methods   base     

other attached packages:
 [1] abind_1.4-5       qtl2_0.22         reshape2_1.4.4    ggplot2_3.3.5    
 [5] tibble_3.1.2      psych_2.0.7       readxl_1.3.1      cluster_2.1.0    
 [9] dplyr_1.0.8       optparse_1.6.6    rhdf5_2.28.1      mclust_5.4.6     
[13] tidyr_1.0.2       data.table_1.14.0 knitr_1.33        kableExtra_1.1.0 
[17] workflowr_1.6.2  

loaded via a namespace (and not attached):
 [1] httr_1.4.1        bit64_4.0.5       viridisLite_0.4.0 assertthat_0.2.1 
 [5] highr_0.9         blob_1.2.1        cellranger_1.1.0  yaml_2.2.1       
 [9] pillar_1.6.1      RSQLite_2.2.7     backports_1.2.1   lattice_0.20-38  
[13] glue_1.4.2        digest_0.6.27     promises_1.1.0    rvest_0.3.5      
[17] colorspace_2.0-2  htmltools_0.5.1.1 httpuv_1.5.2      plyr_1.8.6       
[21] pkgconfig_2.0.3   purrr_0.3.4       scales_1.1.1      webshot_0.5.2    
[25] getopt_1.20.3     later_1.0.0       git2r_0.26.1      generics_0.0.2   
[29] ellipsis_0.3.2    cachem_1.0.5      withr_2.4.2       cli_3.0.0        
[33] mnormt_1.5-7      magrittr_2.0.1    crayon_1.4.1      memoise_2.0.0    
[37] evaluate_0.14     fs_1.4.1          fansi_0.5.0       nlme_3.1-142     
[41] xml2_1.3.1        tools_3.6.2       hms_0.5.3         lifecycle_1.0.1  
[45] stringr_1.4.0     Rhdf5lib_1.6.3    munsell_0.5.0     compiler_3.6.2   
[49] rlang_1.0.2       grid_3.6.2        rstudioapi_0.13   rmarkdown_2.1    
[53] gtable_0.3.0      DBI_1.1.1         R6_2.5.0          fastmap_1.1.0    
[57] bit_4.0.4         utf8_1.2.1        rprojroot_1.3-2   readr_1.3.1      
[61] stringi_1.7.2     parallel_3.6.2    Rcpp_1.0.7        vctrs_0.3.8      
[65] tidyselect_1.1.2  xfun_0.24        </code></pre>
</div>
</div>
</div>
</div>


<!-- Adjust MathJax settings so that all math formulae are shown using
TeX fonts only; see
http://docs.mathjax.org/en/latest/configuration.html.  This will make
the presentation more consistent at the cost of the webpage sometimes
taking slightly longer to load. Note that this only works because the
footer is added to webpages before the MathJax javascript. -->
<script type="text/x-mathjax-config">
  MathJax.Hub.Config({
    "HTML-CSS": { availableFonts: ["TeX"] }
  });
</script>





</div>

<script>

// add bootstrap table styles to pandoc tables
function bootstrapStylePandocTables() {
  $('tr.header').parent('thead').parent('table').addClass('table table-condensed');
}
$(document).ready(function () {
  bootstrapStylePandocTables();
});


</script>

<!-- tabsets -->

<script>
$(document).ready(function () {
  window.buildTabsets("TOC");
});

$(document).ready(function () {
  $('.tabset-dropdown > .nav-tabs > li').click(function () {
    $(this).parent().toggleClass('nav-tabs-open')
  });
});
</script>

<!-- code folding -->


<!-- dynamically load mathjax for compatibility with self-contained -->
<script>
  (function () {
    var script = document.createElement("script");
    script.type = "text/javascript";
    script.src  = "https://mathjax.rstudio.com/latest/MathJax.js?config=TeX-AMS-MML_HTMLorMML";
    document.getElementsByTagName("head")[0].appendChild(script);
  })();
</script>

</body>
</html>
