<!DOCTYPE html>

<html>

<head>

<meta charset="utf-8" />
<meta name="generator" content="pandoc" />
<meta http-equiv="X-UA-Compatible" content="IE=EDGE" />


<meta name="author" content="Belinda Cornes" />

<meta name="date" content="2022-04-15" />

<title>QTL Analysis - age of onset [ICI-Early vs PBS &amp; ICI-Late] (corrected phenotype with outliers removed if any)</title>

<script src="site_libs/header-attrs-2.1/header-attrs.js"></script>
<script src="site_libs/jquery-1.11.3/jquery.min.js"></script>
<meta name="viewport" content="width=device-width, initial-scale=1" />
<link href="site_libs/bootstrap-3.3.5/css/spacelab.min.css" rel="stylesheet" />
<script src="site_libs/bootstrap-3.3.5/js/bootstrap.min.js"></script>
<script src="site_libs/bootstrap-3.3.5/shim/html5shiv.min.js"></script>
<script src="site_libs/bootstrap-3.3.5/shim/respond.min.js"></script>
<script src="site_libs/navigation-1.1/tabsets.js"></script>
<link href="site_libs/highlightjs-9.12.0/textmate.css" rel="stylesheet" />
<script src="site_libs/highlightjs-9.12.0/highlight.js"></script>
<script src="site_libs/kePrint-0.0.1/kePrint.js"></script>
<link href="site_libs/font-awesome-5.1.0/css/all.css" rel="stylesheet" />
<link href="site_libs/font-awesome-5.1.0/css/v4-shims.css" rel="stylesheet" />

<link rel="icon" href="https://github.com/workflowr/workflowr-assets/raw/master/img/reproducible.png">
<!-- Add a small amount of space between sections. -->
<style type="text/css">
div.section {
  padding-top: 12px;
}
</style>



<style type="text/css">code{white-space: pre;}</style>
<style type="text/css">
  pre:not([class]) {
    background-color: white;
  }
</style>
<script type="text/javascript">
if (window.hljs) {
  hljs.configure({languages: []});
  hljs.initHighlightingOnLoad();
  if (document.readyState && document.readyState === "complete") {
    window.setTimeout(function() { hljs.initHighlighting(); }, 0);
  }
}
</script>



<style type="text/css">
h1 {
  font-size: 34px;
}
h1.title {
  font-size: 38px;
}
h2 {
  font-size: 30px;
}
h3 {
  font-size: 24px;
}
h4 {
  font-size: 18px;
}
h5 {
  font-size: 16px;
}
h6 {
  font-size: 12px;
}
.table th:not([align]) {
  text-align: left;
}
</style>




<style type = "text/css">
.main-container {
  max-width: 940px;
  margin-left: auto;
  margin-right: auto;
}
code {
  color: inherit;
  background-color: rgba(0, 0, 0, 0.04);
}
img {
  max-width:100%;
}
.tabbed-pane {
  padding-top: 12px;
}
.html-widget {
  margin-bottom: 20px;
}
button.code-folding-btn:focus {
  outline: none;
}
summary {
  display: list-item;
}
</style>


<style type="text/css">
/* padding for bootstrap navbar */
body {
  padding-top: 51px;
  padding-bottom: 40px;
}
/* offset scroll position for anchor links (for fixed navbar)  */
.section h1 {
  padding-top: 56px;
  margin-top: -56px;
}
.section h2 {
  padding-top: 56px;
  margin-top: -56px;
}
.section h3 {
  padding-top: 56px;
  margin-top: -56px;
}
.section h4 {
  padding-top: 56px;
  margin-top: -56px;
}
.section h5 {
  padding-top: 56px;
  margin-top: -56px;
}
.section h6 {
  padding-top: 56px;
  margin-top: -56px;
}
.dropdown-submenu {
  position: relative;
}
.dropdown-submenu>.dropdown-menu {
  top: 0;
  left: 100%;
  margin-top: -6px;
  margin-left: -1px;
  border-radius: 0 6px 6px 6px;
}
.dropdown-submenu:hover>.dropdown-menu {
  display: block;
}
.dropdown-submenu>a:after {
  display: block;
  content: " ";
  float: right;
  width: 0;
  height: 0;
  border-color: transparent;
  border-style: solid;
  border-width: 5px 0 5px 5px;
  border-left-color: #cccccc;
  margin-top: 5px;
  margin-right: -10px;
}
.dropdown-submenu:hover>a:after {
  border-left-color: #ffffff;
}
.dropdown-submenu.pull-left {
  float: none;
}
.dropdown-submenu.pull-left>.dropdown-menu {
  left: -100%;
  margin-left: 10px;
  border-radius: 6px 0 6px 6px;
}
</style>

<script>
// manage active state of menu based on current page
$(document).ready(function () {
  // active menu anchor
  href = window.location.pathname
  href = href.substr(href.lastIndexOf('/') + 1)
  if (href === "")
    href = "index.html";
  var menuAnchor = $('a[href="' + href + '"]');

  // mark it active
  menuAnchor.parent().addClass('active');

  // if it's got a parent navbar menu mark it active as well
  menuAnchor.closest('li.dropdown').addClass('active');
});
</script>

<!-- tabsets -->

<style type="text/css">
.tabset-dropdown > .nav-tabs {
  display: inline-table;
  max-height: 500px;
  min-height: 44px;
  overflow-y: auto;
  background: white;
  border: 1px solid #ddd;
  border-radius: 4px;
}

.tabset-dropdown > .nav-tabs > li.active:before {
  content: "";
  font-family: 'Glyphicons Halflings';
  display: inline-block;
  padding: 10px;
  border-right: 1px solid #ddd;
}

.tabset-dropdown > .nav-tabs.nav-tabs-open > li.active:before {
  content: "&#xe258;";
  border: none;
}

.tabset-dropdown > .nav-tabs.nav-tabs-open:before {
  content: "";
  font-family: 'Glyphicons Halflings';
  display: inline-block;
  padding: 10px;
  border-right: 1px solid #ddd;
}

.tabset-dropdown > .nav-tabs > li.active {
  display: block;
}

.tabset-dropdown > .nav-tabs > li > a,
.tabset-dropdown > .nav-tabs > li > a:focus,
.tabset-dropdown > .nav-tabs > li > a:hover {
  border: none;
  display: inline-block;
  border-radius: 4px;
  background-color: transparent;
}

.tabset-dropdown > .nav-tabs.nav-tabs-open > li {
  display: block;
  float: none;
}

.tabset-dropdown > .nav-tabs > li {
  display: none;
}
</style>

<!-- code folding -->




</head>

<body>


<div class="container-fluid main-container">




<div class="navbar navbar-default  navbar-fixed-top" role="navigation">
  <div class="container">
    <div class="navbar-header">
      <button type="button" class="navbar-toggle collapsed" data-toggle="collapse" data-target="#navbar">
        <span class="icon-bar"></span>
        <span class="icon-bar"></span>
        <span class="icon-bar"></span>
      </button>
      <a class="navbar-brand" href="index.html">Serreze T1D Workflow</a>
    </div>
    <div id="navbar" class="navbar-collapse collapse">
      <ul class="nav navbar-nav">
        <li>
  <a href="index.html">Home</a>
</li>
<li>
  <a href="about.html">About</a>
</li>
<li>
  <a href="license.html">License</a>
</li>
      </ul>
      <ul class="nav navbar-nav navbar-right">
        <li>
  <a href="https://github.com/TheJacksonLaboratory/Serreze-T1D_Workflow">
    <span class="fa fa-github"></span>
     
    Source code
  </a>
</li>
      </ul>
    </div><!--/.nav-collapse -->
  </div><!--/.container -->
</div><!--/.navbar -->

<div class="fluid-row" id="header">



<h1 class="title toc-ignore">QTL Analysis - age of onset [ICI-Early vs PBS &amp; ICI-Late] (corrected phenotype with outliers removed if any)</h1>
<h4 class="author">Belinda Cornes</h4>
<h4 class="date">2022-04-15</h4>

</div>

<div id="TOC">
<ul>
<li><a href="#loading-data">Loading Data</a></li>
<li><a href="#preparing-necessary-info">Preparing necessary info</a>
<ul>
<li><a href="#no-additive-covariates">No additive covariates</a>
<ul>
<li><a href="#significance-thresholds">Significance thresholds</a></li>
<li><a href="#genome-wide-scan">Genome-wide scan</a></li>
<li><a href="#lod-peaks">LOD peaks</a></li>
<li><a href="#qtl-effects">QTL effects</a></li>
</ul></li>
<li><a href="#additive-covariates">Additive covariates</a>
<ul>
<li><a href="#significance-thresholds-1">Significance thresholds</a></li>
<li><a href="#genome-wide-scan-1">Genome-wide scan</a></li>
<li><a href="#lod-peaks-1">LOD peaks</a></li>
<li><a href="#qtl-effects-1">QTL effects</a></li>
</ul></li>
<li><a href="#interactive-term">Interactive term</a>
<ul>
<li><a href="#significance-thresholds-2">Significance thresholds</a></li>
<li><a href="#genome-wide-scan-2">Genome-wide scan</a></li>
<li><a href="#lod-peaks-2">LOD peaks</a></li>
<li><a href="#qtl-effects-2">QTL effects</a></li>
</ul></li>
</ul></li>
</ul>
</div>

<p>
<button type="button" class="btn btn-default btn-workflowr btn-workflowr-report" data-toggle="collapse" data-target="#workflowr-report">
<span class="glyphicon glyphicon-list" aria-hidden="true"></span> workflowr <span class="glyphicon glyphicon-exclamation-sign text-danger" aria-hidden="true"></span>
</button>
</p>
<div id="workflowr-report" class="collapse">
<ul class="nav nav-tabs">
<li class="active">
<a data-toggle="tab" href="#summary">Summary</a>
</li>
<li>
<a data-toggle="tab" href="#checks"> Checks <span class="glyphicon glyphicon-exclamation-sign text-danger" aria-hidden="true"></span> </a>
</li>
<li>
<a data-toggle="tab" href="#versions">Past versions</a>
</li>
</ul>
<div class="tab-content">
<div id="summary" class="tab-pane fade in active">
<p>
<strong>Last updated:</strong> 2022-04-15
</p>
<p>
<strong>Checks:</strong> <span class="glyphicon glyphicon-ok text-success" aria-hidden="true"></span> 5 <span class="glyphicon glyphicon-exclamation-sign text-danger" aria-hidden="true"></span> 2
</p>
<p>
<strong>Knit directory:</strong> <code>Serreze-T1D_Workflow/</code> <span class="glyphicon glyphicon-question-sign" aria-hidden="true" title="This is the local directory in which the code in this file was executed."> </span>
</p>
<p>
This reproducible <a href="http://rmarkdown.rstudio.com">R Markdown</a> analysis was created with <a
  href="https://github.com/jdblischak/workflowr">workflowr</a> (version 1.6.2). The <em>Checks</em> tab describes the reproducibility checks that were applied when the results were created. The <em>Past versions</em> tab lists the development history.
</p>
<hr>
</div>
<div id="checks" class="tab-pane fade">
<div id="workflowr-checks" class="panel-group">
<div class="panel panel-default">
<div class="panel-heading">
<p class="panel-title">
<a data-toggle="collapse" data-parent="#workflowr-checks" href="#strongRMarkdownfilestronguncommittedchanges"> <span class="glyphicon glyphicon-exclamation-sign text-danger" aria-hidden="true"></span> <strong>R Markdown file:</strong> uncommitted changes </a>
</p>
</div>
<div id="strongRMarkdownfilestronguncommittedchanges" class="panel-collapse collapse">
<div class="panel-body">
<p>The R Markdown is untracked by Git. To know which version of the R Markdown file created these results, you’ll want to first commit it to the Git repo. If you’re still working on the analysis, you can ignore this warning. When you’re finished, you can run <code>wflow_publish</code> to commit the R Markdown file and build the HTML.</p>
</div>
</div>
</div>
<div class="panel panel-default">
<div class="panel-heading">
<p class="panel-title">
<a data-toggle="collapse" data-parent="#workflowr-checks" href="#strongEnvironmentstrongempty"> <span class="glyphicon glyphicon-ok text-success" aria-hidden="true"></span> <strong>Environment:</strong> empty </a>
</p>
</div>
<div id="strongEnvironmentstrongempty" class="panel-collapse collapse">
<div class="panel-body">
<p>Great job! The global environment was empty. Objects defined in the global environment can affect the analysis in your R Markdown file in unknown ways. For reproduciblity it’s best to always run the code in an empty environment.</p>
</div>
</div>
</div>
<div class="panel panel-default">
<div class="panel-heading">
<p class="panel-title">
<a data-toggle="collapse" data-parent="#workflowr-checks" href="#strongSeedstrongcodesetseed20220210code"> <span class="glyphicon glyphicon-ok text-success" aria-hidden="true"></span> <strong>Seed:</strong> <code>set.seed(20220210)</code> </a>
</p>
</div>
<div id="strongSeedstrongcodesetseed20220210code" class="panel-collapse collapse">
<div class="panel-body">
<p>The command <code>set.seed(20220210)</code> was run prior to running the code in the R Markdown file. Setting a seed ensures that any results that rely on randomness, e.g. subsampling or permutations, are reproducible.</p>
</div>
</div>
</div>
<div class="panel panel-default">
<div class="panel-heading">
<p class="panel-title">
<a data-toggle="collapse" data-parent="#workflowr-checks" href="#strongSessioninformationstrongrecorded"> <span class="glyphicon glyphicon-ok text-success" aria-hidden="true"></span> <strong>Session information:</strong> recorded </a>
</p>
</div>
<div id="strongSessioninformationstrongrecorded" class="panel-collapse collapse">
<div class="panel-body">
<p>Great job! Recording the operating system, R version, and package versions is critical for reproducibility.</p>
</div>
</div>
</div>
<div class="panel panel-default">
<div class="panel-heading">
<p class="panel-title">
<a data-toggle="collapse" data-parent="#workflowr-checks" href="#strongCachestrongnone"> <span class="glyphicon glyphicon-ok text-success" aria-hidden="true"></span> <strong>Cache:</strong> none </a>
</p>
</div>
<div id="strongCachestrongnone" class="panel-collapse collapse">
<div class="panel-body">
<p>Nice! There were no cached chunks for this analysis, so you can be confident that you successfully produced the results during this run.</p>
</div>
</div>
</div>
<div class="panel panel-default">
<div class="panel-heading">
<p class="panel-title">
<a data-toggle="collapse" data-parent="#workflowr-checks" href="#strongFilepathsstrongabsolute"> <span class="glyphicon glyphicon-exclamation-sign text-danger" aria-hidden="true"></span> <strong>File paths:</strong> absolute </a>
</p>
</div>
<div id="strongFilepathsstrongabsolute" class="panel-collapse collapse">
<div class="panel-body">
<p>
Using absolute paths to the files within your workflowr project makes it difficult for you and others to run your code on a different machine. Change the absolute path(s) below to the suggested relative path(s) to make your code more reproducible.
</p>
<table class="table table-condensed table-hover">
<thead>
<tr>
<th style="text-align:left;">
absolute
</th>
<th style="text-align:left;">
relative
</th>
</tr>
</thead>
<tbody>
<tr>
<td style="text-align:left;">
/Users/corneb/Documents/MyJax/CS/Projects/Serreze/qc/workflowr/Serreze-T1D_Workflow
</td>
<td style="text-align:left;">
.
</td>
</tr>
</tbody>
</table>
</div>
</div>
</div>
<div class="panel panel-default">
<div class="panel-heading">
<p class="panel-title">
<a data-toggle="collapse" data-parent="#workflowr-checks" href="#strongRepositoryversionstrongahrefhttpsgithubcomTheJacksonLaboratorySerrezeT1DWorkflowtreea8c2d1a11f223361b54952ce8455e8c17017c914targetblanka8c2d1aa"> <span class="glyphicon glyphicon-ok text-success" aria-hidden="true"></span> <strong>Repository version:</strong> <a href="https://github.com/TheJacksonLaboratory/Serreze-T1D_Workflow/tree/a8c2d1a11f223361b54952ce8455e8c17017c914" target="_blank">a8c2d1a</a> </a>
</p>
</div>
<div id="strongRepositoryversionstrongahrefhttpsgithubcomTheJacksonLaboratorySerrezeT1DWorkflowtreea8c2d1a11f223361b54952ce8455e8c17017c914targetblanka8c2d1aa" class="panel-collapse collapse">
<div class="panel-body">
<p>
Great! You are using Git for version control. Tracking code development and connecting the code version to the results is critical for reproducibility.
</p>
<p>
The results in this page were generated with repository version <a href="https://github.com/TheJacksonLaboratory/Serreze-T1D_Workflow/tree/a8c2d1a11f223361b54952ce8455e8c17017c914" target="_blank">a8c2d1a</a>. See the <em>Past versions</em> tab to see a history of the changes made to the R Markdown and HTML files.
</p>
<p>
Note that you need to be careful to ensure that all relevant files for the analysis have been committed to Git prior to generating the results (you can use <code>wflow_publish</code> or <code>wflow_git_commit</code>). workflowr only checks the R Markdown file, but you know if there are other scripts or data files that it depends on. Below is the status of the Git repository when the results were generated:
</p>
<pre><code>
Ignored files:
    Ignored:    .DS_Store
    Ignored:    analysis/.DS_Store
    Ignored:    data/.DS_Store

Untracked files:
    Untracked:  analysis/3.1_phenotype.qc_corrected_5.batches.Rmd.R
    Untracked:  analysis/3.1_phenotype.qc_corrected_5.batches_0.Rmd
    Untracked:  analysis/3.1_phenotype.qc_corrected_5.batches_52.Rmd
    Untracked:  analysis/3.1_phenotype.qc_corrected_5.batches_ici-early.vs.pbs.ici-late_mis.Rmd.R
    Untracked:  analysis/3.1_phenotype.qc_corrected_5.batches_mis.Rmd.R
    Untracked:  analysis/3.1_phenotype.qc_corrected_5.batches_mis_0.Rmd
    Untracked:  analysis/3.1_phenotype.qc_corrected_5.batches_mis_52.Rmd
    Untracked:  analysis/3.1_phenotype.qc_corrected_5.batches_mis_vo.Rmd
    Untracked:  analysis/3.1_phenotype.qc_corrected_5.batches_vo.Rmd
    Untracked:  analysis/3.1_phenotype.qc_corrected_ici-early.vs.pbs.ici-late_5.batches.Rmd
    Untracked:  analysis/3.1_phenotype.qc_corrected_ici-early.vs.pbs.ici-late_5.batches.Rmd.R
    Untracked:  analysis/3.1_phenotype.qc_corrected_ici-early.vs.pbs.ici-late_5.batches_mis.Rmd
    Untracked:  analysis/4.1.1_qtl.analysis_binary_ici-early.vs.pbs.ici-late_5.batches.Rmd
    Untracked:  analysis/4.1.1_qtl.analysis_binary_ici-early.vs.pbs.ici-late_5.batches.Rmd.R
    Untracked:  analysis/4.1.1_qtl.analysis_binary_ici-early.vs.pbs.ici-late_5.batches_mis.Rmd
    Untracked:  analysis/4.1.1_qtl.analysis_binary_ici-early.vs.pbs.ici-late_5.batches_mis.Rmd.R
    Untracked:  analysis/4.1.1_qtl.analysis_binary_ici-early.vs.pbs.ici-late_snpsqc_5.batches.Rmd
    Untracked:  analysis/4.1.1_qtl.analysis_binary_ici-early.vs.pbs.ici-late_snpsqc_5.batches.Rmd.R
    Untracked:  analysis/4.1.1_qtl.analysis_binary_ici-early.vs.pbs.ici-late_snpsqc_5.batches_mis.Rmd
    Untracked:  analysis/4.1.1_qtl.analysis_binary_ici-early.vs.pbs.ici-late_snpsqc_5.batches_mis.Rmd.R
    Untracked:  analysis/4.1.1_qtl.analysis_binary_ici-early.vs.pbs.ici-late_snpsqc_dis_no-x_updated_5.batches.Rmd
    Untracked:  analysis/4.1.1_qtl.analysis_binary_ici-early.vs.pbs.ici-late_snpsqc_dis_no-x_updated_5.batches.Rmd.R
    Untracked:  analysis/4.1.1_qtl.analysis_binary_ici-early.vs.pbs.ici-late_snpsqc_dis_no-x_updated_5.batches_mis.Rmd
    Untracked:  analysis/4.1.1_qtl.analysis_binary_ici-early.vs.pbs.ici-late_snpsqc_dis_no-x_updated_5.batches_mis.Rmd.R
    Untracked:  analysis/4.1.1_qtl.analysis_binary_ici.vs.eoi_snpsqc_dis_no-x_updated_5.batches_11.Rmd
    Untracked:  analysis/4.1.1_qtl.analysis_binary_ici.vs.eoi_snpsqc_dis_no-x_updated_5.batches_mis_11.Rmd
    Untracked:  analysis/4.1.1_qtl.analysis_binary_ici.vs.pbs_snpsqc_5.batches_1.Rmd
    Untracked:  analysis/4.1.1_qtl.analysis_binary_ici.vs.pbs_snpsqc_5.batches_mis_1.Rmd
    Untracked:  analysis/4.1.1_qtl.analysis_binary_ici.vs.pbs_snpsqc_dis_no-x_updated_5.batches_1.Rmd
    Untracked:  analysis/4.1.1_qtl.analysis_binary_ici.vs.pbs_snpsqc_dis_no-x_updated_5.batches_11.Rmd
    Untracked:  analysis/4.1.1_qtl.analysis_binary_ici.vs.pbs_snpsqc_dis_no-x_updated_5.batches_mis_11.Rmd
    Untracked:  analysis/4.1.2_qtl.analysis_cont_age_ici-early.vs.pbs.ici-late_pheno.corrected.cleaned_5.batches.Rmd
    Untracked:  analysis/4.1.2_qtl.analysis_cont_age_ici-early.vs.pbs.ici-late_pheno.corrected.cleaned_5.batches.Rmd.R
    Untracked:  analysis/4.1.2_qtl.analysis_cont_age_ici-early.vs.pbs.ici-late_pheno.corrected.cleaned_5.batches_mis.Rmd
    Untracked:  analysis/4.1.2_qtl.analysis_cont_age_ici-early.vs.pbs.ici-late_pheno.corrected.cleaned_5.batches_mis.Rmd.R
    Untracked:  analysis/4.1.2_qtl.analysis_cont_age_ici-early.vs.pbs.ici-late_snpsqc_pheno.corrected.cleaned_5.batches.Rmd
    Untracked:  analysis/4.1.2_qtl.analysis_cont_age_ici-early.vs.pbs.ici-late_snpsqc_pheno.corrected.cleaned_5.batches.Rmd.R
    Untracked:  analysis/4.1.2_qtl.analysis_cont_age_ici-early.vs.pbs.ici-late_snpsqc_pheno.corrected.cleaned_5.batches_mis.Rmd
    Untracked:  analysis/4.1.2_qtl.analysis_cont_age_ici-early.vs.pbs.ici-late_snpsqc_pheno.corrected.cleaned_5.batches_mis.Rmd.R
    Untracked:  analysis/4.1.2_qtl.analysis_cont_age_ici-early.vs.pbs.ici-late_snpsqc_pheno.corrected.cleaned_dis_no-xk_5.batches.Rmd
    Untracked:  analysis/4.1.2_qtl.analysis_cont_age_ici-early.vs.pbs.ici-late_snpsqc_pheno.corrected.cleaned_dis_no-xk_5.batches.Rmd.R
    Untracked:  analysis/4.1.2_qtl.analysis_cont_age_ici-early.vs.pbs.ici-late_snpsqc_pheno.corrected.cleaned_dis_no-xk_5.batches_mis.Rmd
    Untracked:  analysis/4.1.2_qtl.analysis_cont_age_ici-early.vs.pbs.ici-late_snpsqc_pheno.corrected.cleaned_dis_no-xk_5.batches_mis.Rmd.R
    Untracked:  analysis/4.1.2_qtl.analysis_cont_age_ici.vs.eoi_pheno.corrected.cleaned_5.batches_0.Rmd
    Untracked:  analysis/4.1.2_qtl.analysis_cont_age_ici.vs.eoi_pheno.corrected.cleaned_5.batches_52.Rmd
    Untracked:  analysis/4.1.2_qtl.analysis_cont_age_ici.vs.eoi_pheno.corrected.cleaned_5.batches_changed.Rmd
    Untracked:  analysis/4.1.2_qtl.analysis_cont_age_ici.vs.eoi_pheno.corrected.cleaned_5.batches_mis_0.Rmd
    Untracked:  analysis/4.1.2_qtl.analysis_cont_age_ici.vs.eoi_pheno.corrected.cleaned_5.batches_mis_52.Rmd
    Untracked:  analysis/4.1.2_qtl.analysis_cont_age_ici.vs.eoi_pheno.corrected.cleaned_5.batches_mis_changed.Rmd
    Untracked:  analysis/4.1.2_qtl.analysis_cont_age_ici.vs.eoi_pheno.corrected.cleaned_scanone_5.batches.Rmd
    Untracked:  analysis/4.1.2_qtl.analysis_cont_age_ici.vs.eoi_pheno.corrected.cleaned_scanone_5.batches_mis.Rmd
    Untracked:  analysis/4.1.2_qtl.analysis_cont_age_ici.vs.eoi_snpsqc_pheno.corrected.cleaned_5.batches_0.Rmd
    Untracked:  analysis/4.1.2_qtl.analysis_cont_age_ici.vs.eoi_snpsqc_pheno.corrected.cleaned_5.batches_52.Rmd
    Untracked:  analysis/4.1.2_qtl.analysis_cont_age_ici.vs.eoi_snpsqc_pheno.corrected.cleaned_5.batches_mis_0.Rmd
    Untracked:  analysis/4.1.2_qtl.analysis_cont_age_ici.vs.eoi_snpsqc_pheno.corrected.cleaned_5.batches_mis_52.Rmd
    Untracked:  analysis/4.1.2_qtl.analysis_cont_age_ici.vs.eoi_snpsqc_pheno.corrected.cleaned_5.batches_oops.Rmd
    Untracked:  analysis/4.1.2_qtl.analysis_cont_age_ici.vs.eoi_snpsqc_pheno.corrected.cleaned_dis.Rmd.R
    Untracked:  analysis/4.1.2_qtl.analysis_cont_age_ici.vs.eoi_snpsqc_pheno.corrected.cleaned_dis_no-xk_5.batches_0.Rmd
    Untracked:  analysis/4.1.2_qtl.analysis_cont_age_ici.vs.eoi_snpsqc_pheno.corrected.cleaned_dis_no-xk_5.batches_52.Rmd
    Untracked:  analysis/4.1.2_qtl.analysis_cont_age_ici.vs.eoi_snpsqc_pheno.corrected.cleaned_dis_no-xk_5.batches_mis_0.Rmd
    Untracked:  analysis/4.1.2_qtl.analysis_cont_age_ici.vs.eoi_snpsqc_pheno.corrected.cleaned_dis_no-xk_5.batches_mis_52.Rmd
    Untracked:  analysis/4.1.2_qtl.analysis_cont_age_ici.vs.eoi_snpsqc_pheno.corrected.cleaned_test.with.4.Rmd
    Untracked:  analysis/4.1.2_qtl.analysis_cont_age_ici.vs.eoi_snpsqc_pheno.corrected.cleaned_test.with.4.Rmd.R
    Untracked:  analysis/4.1.2_qtl.analysis_cont_age_ici.vs.eoi_snpsqc_pheno.corrected.cleaned_test.with.4_miss.Rmd
    Untracked:  analysis/4.1.2_qtl.analysis_cont_age_ici.vs.eoi_snpsqc_pheno.corrected.cleaned_test.with.4_miss.Rmd.R
    Untracked:  analysis/4.1.2_qtl.analysis_cont_age_ici.vs.eoi_snpsqc_pheno.corrected.cleaned_test.with.5.Rmd
    Untracked:  analysis/4.1.2_qtl.analysis_cont_age_ici.vs.eoi_snpsqc_pheno.corrected.cleaned_test.with.5.Rmd.R
    Untracked:  analysis/4.1.2_qtl.analysis_cont_age_ici.vs.eoi_snpsqc_pheno.corrected.cleaned_test.with.5_0.Rmd
    Untracked:  analysis/4.1.2_qtl.analysis_cont_age_ici.vs.eoi_snpsqc_pheno.corrected.cleaned_test.with.5_0.Rmd.R
    Untracked:  analysis/4.1.2_qtl.analysis_cont_rz.age_ici-early.vs.pbs.ici-late_pheno.corrected.cleaned_5.batches.Rmd
    Untracked:  analysis/4.1.2_qtl.analysis_cont_rz.age_ici-early.vs.pbs.ici-late_pheno.corrected.cleaned_5.batches.Rmd.R
    Untracked:  analysis/4.1.2_qtl.analysis_cont_rz.age_ici-early.vs.pbs.ici-late_pheno.corrected.cleaned_5.batches_mis.Rmd
    Untracked:  analysis/4.1.2_qtl.analysis_cont_rz.age_ici-early.vs.pbs.ici-late_pheno.corrected.cleaned_5.batches_mis.Rmd.R
    Untracked:  analysis/4.1.2_qtl.analysis_cont_rz.age_ici-early.vs.pbs.ici-late_snpsqc_pheno.corrected.cleaned_5.batches.Rmd
    Untracked:  analysis/4.1.2_qtl.analysis_cont_rz.age_ici-early.vs.pbs.ici-late_snpsqc_pheno.corrected.cleaned_5.batches.Rmd.R
    Untracked:  analysis/4.1.2_qtl.analysis_cont_rz.age_ici-early.vs.pbs.ici-late_snpsqc_pheno.corrected.cleaned_5.batches_mis.Rmd
    Untracked:  analysis/4.1.2_qtl.analysis_cont_rz.age_ici-early.vs.pbs.ici-late_snpsqc_pheno.corrected.cleaned_5.batches_mis.Rmd.R
    Untracked:  analysis/4.1.2_qtl.analysis_cont_rz.age_ici-early.vs.pbs.ici-late_snpsqc_pheno.corrected.cleaned_dis_no-xk_5.batches.Rmd
    Untracked:  analysis/4.1.2_qtl.analysis_cont_rz.age_ici-early.vs.pbs.ici-late_snpsqc_pheno.corrected.cleaned_dis_no-xk_5.batches.Rmd.R
    Untracked:  analysis/4.1.2_qtl.analysis_cont_rz.age_ici-early.vs.pbs.ici-late_snpsqc_pheno.corrected.cleaned_dis_no-xk_5.batches_mis.Rmd
    Untracked:  analysis/4.1.2_qtl.analysis_cont_rz.age_ici-early.vs.pbs.ici-late_snpsqc_pheno.corrected.cleaned_dis_no-xk_5.batches_mis.Rmd.R
    Untracked:  analysis/4.1.2_qtl.analysis_cont_rz.age_ici.vs.eoi_pheno.corrected.cleaned_5.batches_0.Rmd
    Untracked:  analysis/4.1.2_qtl.analysis_cont_rz.age_ici.vs.eoi_pheno.corrected.cleaned_5.batches_52.Rmd
    Untracked:  analysis/4.1.2_qtl.analysis_cont_rz.age_ici.vs.eoi_pheno.corrected.cleaned_5.batches_mis_0.Rmd
    Untracked:  analysis/4.1.2_qtl.analysis_cont_rz.age_ici.vs.eoi_pheno.corrected.cleaned_5.batches_mis_52.Rmd
    Untracked:  analysis/4.1.2_qtl.analysis_cont_rz.age_ici.vs.eoi_pheno.corrected.cleaned_5.batches_mis_vo.Rmd
    Untracked:  analysis/4.1.2_qtl.analysis_cont_rz.age_ici.vs.eoi_pheno.corrected.cleaned_5.batches_mis_vo1.Rmd
    Untracked:  analysis/4.1.2_qtl.analysis_cont_rz.age_ici.vs.eoi_pheno.corrected.cleaned_5.batches_vo.Rmd
    Untracked:  analysis/4.1.2_qtl.analysis_cont_rz.age_ici.vs.eoi_pheno.corrected.cleaned_5.batches_vo1.Rmd
    Untracked:  analysis/4.1.2_qtl.analysis_cont_rz.age_ici.vs.eoi_snpsqc_pheno.corrected.cleaned_5.batches_0.Rmd
    Untracked:  analysis/4.1.2_qtl.analysis_cont_rz.age_ici.vs.eoi_snpsqc_pheno.corrected.cleaned_5.batches_52.Rmd
    Untracked:  analysis/4.1.2_qtl.analysis_cont_rz.age_ici.vs.eoi_snpsqc_pheno.corrected.cleaned_5.batches_mis_0.Rmd
    Untracked:  analysis/4.1.2_qtl.analysis_cont_rz.age_ici.vs.eoi_snpsqc_pheno.corrected.cleaned_5.batches_mis_52.Rmd
    Untracked:  analysis/4.1.2_qtl.analysis_cont_rz.age_ici.vs.eoi_snpsqc_pheno.corrected.cleaned_dis_no-xk_5.batches_0.Rmd
    Untracked:  analysis/4.1.2_qtl.analysis_cont_rz.age_ici.vs.eoi_snpsqc_pheno.corrected.cleaned_dis_no-xk_5.batches_52.Rmd
    Untracked:  analysis/4.1.2_qtl.analysis_cont_rz.age_ici.vs.eoi_snpsqc_pheno.corrected.cleaned_dis_no-xk_5.batches_mis_0.Rmd
    Untracked:  analysis/4.1.2_qtl.analysis_cont_rz.age_ici.vs.eoi_snpsqc_pheno.corrected.cleaned_dis_no-xk_5.batches_mis_52.Rmd
    Untracked:  analysis/genotype.frequencies_ici-early.vs.pbs.ici-late_5.batches.Rmd
    Untracked:  analysis/genotype.frequencies_ici-early.vs.pbs.ici-late_5.batches.Rmd.R
    Untracked:  analysis/genotype.frequencies_ici-early.vs.pbs.ici-late_5.batches_mis.Rmd
    Untracked:  analysis/genotype.frequencies_ici-early.vs.pbs.ici-late_5.batches_mis.Rmd.R
    Untracked:  analysis/genotype.frequencies_ici.vs.eoi_5.batches_0.Rmd
    Untracked:  analysis/genotype.frequencies_ici.vs.eoi_5.batches_52.Rmd
    Untracked:  analysis/genotype.frequencies_ici.vs.eoi_5.batches_mis_0.Rmd
    Untracked:  analysis/genotype.frequencies_ici.vs.eoi_5.batches_mis_52.Rmd
    Untracked:  analysis/index_5.batches_additional_vo.Rmd
    Untracked:  data/GM_covar.csv
    Untracked:  data/GM_covar_BC312.csv
    Untracked:  data/bad_markers_all_4.batches.RData
    Untracked:  data/bad_markers_all_5.batches.RData
    Untracked:  data/blup_sub_chr10_lod.drop-1.5_5.batches.csv
    Untracked:  data/blup_sub_chr3_lod.drop-1.5.csv
    Untracked:  data/blup_sub_chr3_lod.drop-1.5_5.batches.csv
    Untracked:  data/blup_sub_chr4_lod.drop-1.5.csv
    Untracked:  data/blup_sub_chr4_lod.drop-1.5_5.batches.csv
    Untracked:  data/covar_cleaned_ici.vs.eoi.csv
    Untracked:  data/covar_cleaned_ici.vs.pbs.csv
    Untracked:  data/covar_corrected.cleaned_ici-early.vs.pbs.ici-late_5.batches.csv
    Untracked:  data/covar_corrected.cleaned_ici-early.vs.pbs.ici-late_5.batches_mis.csv
    Untracked:  data/covar_corrected.cleaned_ici-early.vs.pbs_5.batches.csv
    Untracked:  data/covar_corrected.cleaned_ici-early.vs.pbs_5.batches_0.csv
    Untracked:  data/covar_corrected.cleaned_ici-early.vs.pbs_5.batches_52.csv
    Untracked:  data/covar_corrected.cleaned_ici-early.vs.pbs_5.batches_mis.csv
    Untracked:  data/covar_corrected.cleaned_ici-early.vs.pbs_5.batches_mis_0.csv
    Untracked:  data/covar_corrected.cleaned_ici-early.vs.pbs_5.batches_mis_52.csv
    Untracked:  data/covar_corrected.cleaned_ici.vs.eoi.csv
    Untracked:  data/covar_corrected.cleaned_ici.vs.eoi1.csv
    Untracked:  data/covar_corrected.cleaned_ici.vs.eoi_5.batches.csv
    Untracked:  data/covar_corrected.cleaned_ici.vs.eoi_5.batches_0.csv
    Untracked:  data/covar_corrected.cleaned_ici.vs.eoi_5.batches_52.csv
    Untracked:  data/covar_corrected.cleaned_ici.vs.eoi_5.batches_mis.csv
    Untracked:  data/covar_corrected.cleaned_ici.vs.eoi_5.batches_mis_0.csv
    Untracked:  data/covar_corrected.cleaned_ici.vs.eoi_5.batches_mis_52.csv
    Untracked:  data/covar_corrected.cleaned_ici.vs.pbs.csv
    Untracked:  data/covar_corrected.cleaned_ici.vs.pbs1.csv
    Untracked:  data/covar_corrected.cleaned_ici.vs.pbs_5.batches.csv
    Untracked:  data/covar_corrected.cleaned_ici.vs.pbs_5.batches_0.csv
    Untracked:  data/covar_corrected.cleaned_ici.vs.pbs_5.batches_52.csv
    Untracked:  data/covar_corrected.cleaned_ici.vs.pbs_5.batches_mis.csv
    Untracked:  data/covar_corrected.cleaned_ici.vs.pbs_5.batches_mis_0.csv
    Untracked:  data/covar_corrected.cleaned_ici.vs.pbs_5.batches_mis_52.csv
    Untracked:  data/covar_corrected_ici-early.vs.pbs.ici-late_5.batches.csv
    Untracked:  data/covar_corrected_ici-early.vs.pbs.ici-late_5.batches_mis.csv
    Untracked:  data/covar_corrected_ici-early.vs.pbs_5.batches.csv
    Untracked:  data/covar_corrected_ici-early.vs.pbs_5.batches_0.csv
    Untracked:  data/covar_corrected_ici-early.vs.pbs_5.batches_52.csv
    Untracked:  data/covar_corrected_ici-early.vs.pbs_5.batches_mis.csv
    Untracked:  data/covar_corrected_ici-early.vs.pbs_5.batches_mis_0.csv
    Untracked:  data/covar_corrected_ici-early.vs.pbs_5.batches_mis_52.csv
    Untracked:  data/covar_corrected_ici.vs.eoi.csv
    Untracked:  data/covar_corrected_ici.vs.eoi1.csv
    Untracked:  data/covar_corrected_ici.vs.eoi_5.batches.csv
    Untracked:  data/covar_corrected_ici.vs.eoi_5.batches_0.csv
    Untracked:  data/covar_corrected_ici.vs.eoi_5.batches_52.csv
    Untracked:  data/covar_corrected_ici.vs.eoi_5.batches_mis.csv
    Untracked:  data/covar_corrected_ici.vs.eoi_5.batches_mis_0.csv
    Untracked:  data/covar_corrected_ici.vs.eoi_5.batches_mis_52.csv
    Untracked:  data/covar_corrected_ici.vs.pbs.csv
    Untracked:  data/covar_corrected_ici.vs.pbs1.csv
    Untracked:  data/covar_corrected_ici.vs.pbs_5.batches.csv
    Untracked:  data/covar_corrected_ici.vs.pbs_5.batches_0.csv
    Untracked:  data/covar_corrected_ici.vs.pbs_5.batches_52.csv
    Untracked:  data/covar_corrected_ici.vs.pbs_5.batches_mis.csv
    Untracked:  data/covar_corrected_ici.vs.pbs_5.batches_mis_0.csv
    Untracked:  data/covar_corrected_ici.vs.pbs_5.batches_mis_52.csv
    Untracked:  data/e.RData
    Untracked:  data/e_BC312.RData
    Untracked:  data/e_snpg_samqc_4.batches.RData
    Untracked:  data/e_snpg_samqc_4.batches_bc.RData
    Untracked:  data/e_snpg_samqc_5.batches.RData
    Untracked:  data/errors_ind_4.batches.RData
    Untracked:  data/errors_ind_4.batches_bc.RData
    Untracked:  data/errors_ind_5.batches.RData
    Untracked:  data/files.to.sync.txt
    Untracked:  data/fitqtl_chr3.peak_chr4.peak_additive.txt
    Untracked:  data/fitqtl_chr3.peak_chr4.peak_interacting.txt
    Untracked:  data/fitqtl_chr3.peak_chr4.peak_sex_additive.txt
    Untracked:  data/fitqtl_chr3.peak_chr4.peak_sex_interacting.txt
    Untracked:  data/g2blup_effects.csv
    Untracked:  data/g2blup_effects.xlsx
    Untracked:  data/genes_chr10_lod.drop-1.5_5.batches.csv
    Untracked:  data/genes_chr3_lod.drop-1.5.csv
    Untracked:  data/genes_chr3_lod.drop-1.5_5.batches.csv
    Untracked:  data/genes_chr4_lod.drop-1.5.csv
    Untracked:  data/genetic_map.csv
    Untracked:  data/genetic_map_BC312.csv
    Untracked:  data/genotype_errors_marker_4.batches.RData
    Untracked:  data/genotype_errors_marker_5.batches.RData
    Untracked:  data/genotype_freq_marker_4.batches.RData
    Untracked:  data/genotype_freq_marker_5.batches.RData
    Untracked:  data/gm_allqc_4.batches.RData
    Untracked:  data/gm_allqc_5.batches.RData
    Untracked:  data/gm_allqc_5.batches_mis.RData
    Untracked:  data/gm_samqc_3.batches.RData
    Untracked:  data/gm_samqc_4.batches.RData
    Untracked:  data/gm_samqc_4.batches_bc.RData
    Untracked:  data/gm_samqc_5.batches.RData
    Untracked:  data/gm_serreze.192.RData
    Untracked:  data/gm_serreze.BC312.RData
    Untracked:  data/ici-early.vs.pbs.ici-late_age.of.onset-additive.covariates_blup_sub_chr-6_peak.marker-cr27snv203_lod.drop-1.5_5.batches_mis.csv
    Untracked:  data/ici-early.vs.pbs.ici-late_age.of.onset-additive.covariates_genes_chr-6_peak.marker-cr27snv203_lod.drop-1.5_5.batches_mis.csv
    Untracked:  data/ici-early.vs.pbs.ici-late_age.of.onset-interactive.covariate_blup_sub_chr-1_peak.marker-ICR134_lod.drop-1.5_5.batches_mis.csv
    Untracked:  data/ici-early.vs.pbs.ici-late_age.of.onset-interactive.covariate_blup_sub_chr-1_peak.marker-ICR4119_lod.drop-1.5_5.batches_mis.csv
    Untracked:  data/ici-early.vs.pbs.ici-late_age.of.onset-interactive.covariate_blup_sub_chr-1_peak.marker-JAX00262446_lod.drop-1.5_5.batches_mis.csv
    Untracked:  data/ici-early.vs.pbs.ici-late_age.of.onset-interactive.covariate_blup_sub_chr-1_peak.marker-UNCHS001938_lod.drop-1.5_5.batches_mis.csv
    Untracked:  data/ici-early.vs.pbs.ici-late_age.of.onset-interactive.covariate_blup_sub_chr-6_peak.marker-UNC11008761_lod.drop-1.5_5.batches_mis.csv
    Untracked:  data/ici-early.vs.pbs.ici-late_age.of.onset-interactive.covariate_blup_sub_chr-6_peak.marker-UNCHS016990_lod.drop-1.5_5.batches_mis.csv
    Untracked:  data/ici-early.vs.pbs.ici-late_age.of.onset-interactive.covariate_genes_chr-1_peak.marker-ICR134_lod.drop-1.5_5.batches_mis.csv
    Untracked:  data/ici-early.vs.pbs.ici-late_age.of.onset-interactive.covariate_genes_chr-1_peak.marker-ICR4119_lod.drop-1.5_5.batches_mis.csv
    Untracked:  data/ici-early.vs.pbs.ici-late_age.of.onset-interactive.covariate_genes_chr-1_peak.marker-JAX00262446_lod.drop-1.5_5.batches_mis.csv
    Untracked:  data/ici-early.vs.pbs.ici-late_age.of.onset-interactive.covariate_genes_chr-1_peak.marker-UNCHS001938_lod.drop-1.5_5.batches_mis.csv
    Untracked:  data/ici-early.vs.pbs.ici-late_age.of.onset-interactive.covariate_genes_chr-6_peak.marker-UNC11008761_lod.drop-1.5_5.batches_mis.csv
    Untracked:  data/ici-early.vs.pbs.ici-late_age.of.onset-interactive.covariate_genes_chr-6_peak.marker-UNCHS016990_lod.drop-1.5_5.batches_mis.csv
    Untracked:  data/ici-early.vs.pbs.ici-late_age.of.onset-no.covariates_blup_sub_chr-14_peak.marker-UNC24056202_lod.drop-1.5_5.batches_mis.csv
    Untracked:  data/ici-early.vs.pbs.ici-late_age.of.onset-no.covariates_blup_sub_chr-14_peak.marker-UNC24056202_lod.drop-1.5_snpsqc_5.batches_mis.csv
    Untracked:  data/ici-early.vs.pbs.ici-late_age.of.onset-no.covariates_blup_sub_chr-17_peak.marker-UNCJPD006614_lod.drop-1.5_snpsqc_5.batches_mis.csv
    Untracked:  data/ici-early.vs.pbs.ici-late_age.of.onset-no.covariates_blup_sub_chr-18_peak.marker-UNC28655293_lod.drop-1.5_5.batches_mis.csv
    Untracked:  data/ici-early.vs.pbs.ici-late_age.of.onset-no.covariates_blup_sub_chr-18_peak.marker-UNCHS045343_lod.drop-1.5_snpsqc_5.batches_mis.csv
    Untracked:  data/ici-early.vs.pbs.ici-late_age.of.onset-no.covariates_blup_sub_chr-5_peak.marker-UNC10044126_lod.drop-1.5_5.batches_mis.csv
    Untracked:  data/ici-early.vs.pbs.ici-late_age.of.onset-no.covariates_blup_sub_chr-5_peak.marker-UNC8675939_lod.drop-1.5_snpsqc_5.batches_mis.csv
    Untracked:  data/ici-early.vs.pbs.ici-late_age.of.onset-no.covariates_blup_sub_chr-8_peak.marker-JAX00667121_lod.drop-1.5_5.batches_mis.csv
    Untracked:  data/ici-early.vs.pbs.ici-late_age.of.onset-no.covariates_blup_sub_chr-8_peak.marker-UNC15524531_lod.drop-1.5_snpsqc_5.batches_mis.csv
    Untracked:  data/ici-early.vs.pbs.ici-late_age.of.onset-no.covariates_blup_sub_chr-X_peak.marker-UNCHS048314_lod.drop-1.5_snpsqc_5.batches_mis.csv
    Untracked:  data/ici-early.vs.pbs.ici-late_age.of.onset-no.covariates_genes_chr-14_peak.marker-UNC24056202_lod.drop-1.5_5.batches_mis.csv
    Untracked:  data/ici-early.vs.pbs.ici-late_age.of.onset-no.covariates_genes_chr-14_peak.marker-UNC24056202_lod.drop-1.5_snpsqc_5.batches_mis.csv
    Untracked:  data/ici-early.vs.pbs.ici-late_age.of.onset-no.covariates_genes_chr-17_peak.marker-UNCJPD006614_lod.drop-1.5_snpsqc_5.batches_mis.csv
    Untracked:  data/ici-early.vs.pbs.ici-late_age.of.onset-no.covariates_genes_chr-18_peak.marker-UNC28655293_lod.drop-1.5_5.batches_mis.csv
    Untracked:  data/ici-early.vs.pbs.ici-late_age.of.onset-no.covariates_genes_chr-18_peak.marker-UNCHS045343_lod.drop-1.5_snpsqc_5.batches_mis.csv
    Untracked:  data/ici-early.vs.pbs.ici-late_age.of.onset-no.covariates_genes_chr-5_peak.marker-UNC10044126_lod.drop-1.5_5.batches_mis.csv
    Untracked:  data/ici-early.vs.pbs.ici-late_age.of.onset-no.covariates_genes_chr-5_peak.marker-UNC8675939_lod.drop-1.5_snpsqc_5.batches_mis.csv
    Untracked:  data/ici-early.vs.pbs.ici-late_age.of.onset-no.covariates_genes_chr-8_peak.marker-JAX00667121_lod.drop-1.5_5.batches_mis.csv
    Untracked:  data/ici-early.vs.pbs.ici-late_age.of.onset-no.covariates_genes_chr-8_peak.marker-UNC15524531_lod.drop-1.5_snpsqc_5.batches_mis.csv
    Untracked:  data/ici-early.vs.pbs.ici-late_age.of.onset-no.covariates_genes_chr-X_peak.marker-UNCHS048314_lod.drop-1.5_snpsqc_5.batches_mis.csv
    Untracked:  data/ici-early.vs.pbs.ici-late_blup.full_chr-10_5.batches.csv
    Untracked:  data/ici-early.vs.pbs.ici-late_blup.full_chr-10_5.batches_mis.csv
    Untracked:  data/ici-early.vs.pbs.ici-late_blup.full_chr-10_snpsqc_5.batches.csv
    Untracked:  data/ici-early.vs.pbs.ici-late_blup.full_chr-10_snpsqc_5.batches_mis.csv
    Untracked:  data/ici-early.vs.pbs.ici-late_blup.full_chr-10_snpsqc_dis_no-x_updated_5.batches.csv
    Untracked:  data/ici-early.vs.pbs.ici-late_blup.full_chr-10_snpsqc_dis_no-x_updated_5.batches_mis.csv
    Untracked:  data/ici-early.vs.pbs.ici-late_blup.full_chr-11_5.batches.csv
    Untracked:  data/ici-early.vs.pbs.ici-late_blup.full_chr-11_5.batches_mis.csv
    Untracked:  data/ici-early.vs.pbs.ici-late_blup.full_chr-11_snpsqc_5.batches.csv
    Untracked:  data/ici-early.vs.pbs.ici-late_blup.full_chr-11_snpsqc_5.batches_mis.csv
    Untracked:  data/ici-early.vs.pbs.ici-late_blup.full_chr-11_snpsqc_dis_no-x_updated_5.batches.csv
    Untracked:  data/ici-early.vs.pbs.ici-late_blup.full_chr-11_snpsqc_dis_no-x_updated_5.batches_mis.csv
    Untracked:  data/ici-early.vs.pbs.ici-late_blup.full_chr-12_5.batches.csv
    Untracked:  data/ici-early.vs.pbs.ici-late_blup.full_chr-12_5.batches_mis.csv
    Untracked:  data/ici-early.vs.pbs.ici-late_blup.full_chr-12_snpsqc_5.batches.csv
    Untracked:  data/ici-early.vs.pbs.ici-late_blup.full_chr-12_snpsqc_5.batches_mis.csv
    Untracked:  data/ici-early.vs.pbs.ici-late_blup.full_chr-12_snpsqc_dis_no-x_updated_5.batches.csv
    Untracked:  data/ici-early.vs.pbs.ici-late_blup.full_chr-12_snpsqc_dis_no-x_updated_5.batches_mis.csv
    Untracked:  data/ici-early.vs.pbs.ici-late_blup.full_chr-13_5.batches.csv
    Untracked:  data/ici-early.vs.pbs.ici-late_blup.full_chr-13_5.batches_mis.csv
    Untracked:  data/ici-early.vs.pbs.ici-late_blup.full_chr-13_snpsqc_5.batches.csv
    Untracked:  data/ici-early.vs.pbs.ici-late_blup.full_chr-13_snpsqc_5.batches_mis.csv
    Untracked:  data/ici-early.vs.pbs.ici-late_blup.full_chr-13_snpsqc_dis_no-x_updated_5.batches.csv
    Untracked:  data/ici-early.vs.pbs.ici-late_blup.full_chr-13_snpsqc_dis_no-x_updated_5.batches_mis.csv
    Untracked:  data/ici-early.vs.pbs.ici-late_blup.full_chr-14_5.batches.csv
    Untracked:  data/ici-early.vs.pbs.ici-late_blup.full_chr-14_5.batches_mis.csv
    Untracked:  data/ici-early.vs.pbs.ici-late_blup.full_chr-14_snpsqc_5.batches.csv
    Untracked:  data/ici-early.vs.pbs.ici-late_blup.full_chr-14_snpsqc_5.batches_mis.csv
    Untracked:  data/ici-early.vs.pbs.ici-late_blup.full_chr-14_snpsqc_dis_no-x_updated_5.batches.csv
    Untracked:  data/ici-early.vs.pbs.ici-late_blup.full_chr-14_snpsqc_dis_no-x_updated_5.batches_mis.csv
    Untracked:  data/ici-early.vs.pbs.ici-late_blup.full_chr-15_5.batches.csv
    Untracked:  data/ici-early.vs.pbs.ici-late_blup.full_chr-15_5.batches_mis.csv
    Untracked:  data/ici-early.vs.pbs.ici-late_blup.full_chr-15_snpsqc_5.batches.csv
    Untracked:  data/ici-early.vs.pbs.ici-late_blup.full_chr-15_snpsqc_5.batches_mis.csv
    Untracked:  data/ici-early.vs.pbs.ici-late_blup.full_chr-15_snpsqc_dis_no-x_updated_5.batches.csv
    Untracked:  data/ici-early.vs.pbs.ici-late_blup.full_chr-15_snpsqc_dis_no-x_updated_5.batches_mis.csv
    Untracked:  data/ici-early.vs.pbs.ici-late_blup.full_chr-16_5.batches.csv
    Untracked:  data/ici-early.vs.pbs.ici-late_blup.full_chr-16_5.batches_mis.csv
    Untracked:  data/ici-early.vs.pbs.ici-late_blup.full_chr-16_snpsqc_5.batches.csv
    Untracked:  data/ici-early.vs.pbs.ici-late_blup.full_chr-16_snpsqc_5.batches_mis.csv
    Untracked:  data/ici-early.vs.pbs.ici-late_blup.full_chr-16_snpsqc_dis_no-x_updated_5.batches.csv
    Untracked:  data/ici-early.vs.pbs.ici-late_blup.full_chr-16_snpsqc_dis_no-x_updated_5.batches_mis.csv
    Untracked:  data/ici-early.vs.pbs.ici-late_blup.full_chr-17_5.batches.csv
    Untracked:  data/ici-early.vs.pbs.ici-late_blup.full_chr-17_5.batches_mis.csv
    Untracked:  data/ici-early.vs.pbs.ici-late_blup.full_chr-17_snpsqc_5.batches.csv
    Untracked:  data/ici-early.vs.pbs.ici-late_blup.full_chr-17_snpsqc_5.batches_mis.csv
    Untracked:  data/ici-early.vs.pbs.ici-late_blup.full_chr-17_snpsqc_dis_no-x_updated_5.batches.csv
    Untracked:  data/ici-early.vs.pbs.ici-late_blup.full_chr-17_snpsqc_dis_no-x_updated_5.batches_mis.csv
    Untracked:  data/ici-early.vs.pbs.ici-late_blup.full_chr-18_5.batches.csv
    Untracked:  data/ici-early.vs.pbs.ici-late_blup.full_chr-18_5.batches_mis.csv
    Untracked:  data/ici-early.vs.pbs.ici-late_blup.full_chr-18_snpsqc_5.batches.csv
    Untracked:  data/ici-early.vs.pbs.ici-late_blup.full_chr-18_snpsqc_5.batches_mis.csv
    Untracked:  data/ici-early.vs.pbs.ici-late_blup.full_chr-18_snpsqc_dis_no-x_updated_5.batches.csv
    Untracked:  data/ici-early.vs.pbs.ici-late_blup.full_chr-18_snpsqc_dis_no-x_updated_5.batches_mis.csv
    Untracked:  data/ici-early.vs.pbs.ici-late_blup.full_chr-19_5.batches.csv
    Untracked:  data/ici-early.vs.pbs.ici-late_blup.full_chr-19_5.batches_mis.csv
    Untracked:  data/ici-early.vs.pbs.ici-late_blup.full_chr-19_snpsqc_5.batches.csv
    Untracked:  data/ici-early.vs.pbs.ici-late_blup.full_chr-19_snpsqc_5.batches_mis.csv
    Untracked:  data/ici-early.vs.pbs.ici-late_blup.full_chr-19_snpsqc_dis_no-x_updated_5.batches.csv
    Untracked:  data/ici-early.vs.pbs.ici-late_blup.full_chr-19_snpsqc_dis_no-x_updated_5.batches_mis.csv
    Untracked:  data/ici-early.vs.pbs.ici-late_blup.full_chr-1_5.batches.csv
    Untracked:  data/ici-early.vs.pbs.ici-late_blup.full_chr-1_5.batches_mis.csv
    Untracked:  data/ici-early.vs.pbs.ici-late_blup.full_chr-1_snpsqc_5.batches.csv
    Untracked:  data/ici-early.vs.pbs.ici-late_blup.full_chr-1_snpsqc_5.batches_mis.csv
    Untracked:  data/ici-early.vs.pbs.ici-late_blup.full_chr-1_snpsqc_dis_no-x_updated_5.batches.csv
    Untracked:  data/ici-early.vs.pbs.ici-late_blup.full_chr-1_snpsqc_dis_no-x_updated_5.batches_mis.csv
    Untracked:  data/ici-early.vs.pbs.ici-late_blup.full_chr-2_5.batches.csv
    Untracked:  data/ici-early.vs.pbs.ici-late_blup.full_chr-2_5.batches_mis.csv
    Untracked:  data/ici-early.vs.pbs.ici-late_blup.full_chr-2_snpsqc_5.batches.csv
    Untracked:  data/ici-early.vs.pbs.ici-late_blup.full_chr-2_snpsqc_5.batches_mis.csv
    Untracked:  data/ici-early.vs.pbs.ici-late_blup.full_chr-2_snpsqc_dis_no-x_updated_5.batches.csv
    Untracked:  data/ici-early.vs.pbs.ici-late_blup.full_chr-2_snpsqc_dis_no-x_updated_5.batches_mis.csv
    Untracked:  data/ici-early.vs.pbs.ici-late_blup.full_chr-3_5.batches.csv
    Untracked:  data/ici-early.vs.pbs.ici-late_blup.full_chr-3_5.batches_mis.csv
    Untracked:  data/ici-early.vs.pbs.ici-late_blup.full_chr-3_snpsqc_5.batches.csv
    Untracked:  data/ici-early.vs.pbs.ici-late_blup.full_chr-3_snpsqc_5.batches_mis.csv
    Untracked:  data/ici-early.vs.pbs.ici-late_blup.full_chr-3_snpsqc_dis_no-x_updated_5.batches.csv
    Untracked:  data/ici-early.vs.pbs.ici-late_blup.full_chr-3_snpsqc_dis_no-x_updated_5.batches_mis.csv
    Untracked:  data/ici-early.vs.pbs.ici-late_blup.full_chr-4_5.batches.csv
    Untracked:  data/ici-early.vs.pbs.ici-late_blup.full_chr-4_5.batches_mis.csv
    Untracked:  data/ici-early.vs.pbs.ici-late_blup.full_chr-4_snpsqc_5.batches.csv
    Untracked:  data/ici-early.vs.pbs.ici-late_blup.full_chr-4_snpsqc_5.batches_mis.csv
    Untracked:  data/ici-early.vs.pbs.ici-late_blup.full_chr-4_snpsqc_dis_no-x_updated_5.batches.csv
    Untracked:  data/ici-early.vs.pbs.ici-late_blup.full_chr-4_snpsqc_dis_no-x_updated_5.batches_mis.csv
    Untracked:  data/ici-early.vs.pbs.ici-late_blup.full_chr-5_5.batches.csv
    Untracked:  data/ici-early.vs.pbs.ici-late_blup.full_chr-5_5.batches_mis.csv
    Untracked:  data/ici-early.vs.pbs.ici-late_blup.full_chr-5_snpsqc_5.batches.csv
    Untracked:  data/ici-early.vs.pbs.ici-late_blup.full_chr-5_snpsqc_5.batches_mis.csv
    Untracked:  data/ici-early.vs.pbs.ici-late_blup.full_chr-5_snpsqc_dis_no-x_updated_5.batches.csv
    Untracked:  data/ici-early.vs.pbs.ici-late_blup.full_chr-5_snpsqc_dis_no-x_updated_5.batches_mis.csv
    Untracked:  data/ici-early.vs.pbs.ici-late_blup.full_chr-6_5.batches.csv
    Untracked:  data/ici-early.vs.pbs.ici-late_blup.full_chr-6_5.batches_mis.csv
    Untracked:  data/ici-early.vs.pbs.ici-late_blup.full_chr-6_snpsqc_5.batches.csv
    Untracked:  data/ici-early.vs.pbs.ici-late_blup.full_chr-6_snpsqc_5.batches_mis.csv
    Untracked:  data/ici-early.vs.pbs.ici-late_blup.full_chr-6_snpsqc_dis_no-x_updated_5.batches.csv
    Untracked:  data/ici-early.vs.pbs.ici-late_blup.full_chr-6_snpsqc_dis_no-x_updated_5.batches_mis.csv
    Untracked:  data/ici-early.vs.pbs.ici-late_blup.full_chr-7_5.batches.csv
    Untracked:  data/ici-early.vs.pbs.ici-late_blup.full_chr-7_5.batches_mis.csv
    Untracked:  data/ici-early.vs.pbs.ici-late_blup.full_chr-7_snpsqc_5.batches.csv
    Untracked:  data/ici-early.vs.pbs.ici-late_blup.full_chr-7_snpsqc_5.batches_mis.csv
    Untracked:  data/ici-early.vs.pbs.ici-late_blup.full_chr-7_snpsqc_dis_no-x_updated_5.batches.csv
    Untracked:  data/ici-early.vs.pbs.ici-late_blup.full_chr-7_snpsqc_dis_no-x_updated_5.batches_mis.csv
    Untracked:  data/ici-early.vs.pbs.ici-late_blup.full_chr-8_5.batches.csv
    Untracked:  data/ici-early.vs.pbs.ici-late_blup.full_chr-8_5.batches_mis.csv
    Untracked:  data/ici-early.vs.pbs.ici-late_blup.full_chr-8_snpsqc_5.batches.csv
    Untracked:  data/ici-early.vs.pbs.ici-late_blup.full_chr-8_snpsqc_5.batches_mis.csv
    Untracked:  data/ici-early.vs.pbs.ici-late_blup.full_chr-8_snpsqc_dis_no-x_updated_5.batches.csv
    Untracked:  data/ici-early.vs.pbs.ici-late_blup.full_chr-8_snpsqc_dis_no-x_updated_5.batches_mis.csv
    Untracked:  data/ici-early.vs.pbs.ici-late_blup.full_chr-9_5.batches.csv
    Untracked:  data/ici-early.vs.pbs.ici-late_blup.full_chr-9_5.batches_mis.csv
    Untracked:  data/ici-early.vs.pbs.ici-late_blup.full_chr-9_snpsqc_5.batches.csv
    Untracked:  data/ici-early.vs.pbs.ici-late_blup.full_chr-9_snpsqc_5.batches_mis.csv
    Untracked:  data/ici-early.vs.pbs.ici-late_blup.full_chr-9_snpsqc_dis_no-x_updated_5.batches.csv
    Untracked:  data/ici-early.vs.pbs.ici-late_blup.full_chr-9_snpsqc_dis_no-x_updated_5.batches_mis.csv
    Untracked:  data/ici-early.vs.pbs.ici-late_blup.full_chr-X_5.batches.csv
    Untracked:  data/ici-early.vs.pbs.ici-late_blup.full_chr-X_5.batches_mis.csv
    Untracked:  data/ici-early.vs.pbs.ici-late_blup.full_chr-X_snpsqc_5.batches.csv
    Untracked:  data/ici-early.vs.pbs.ici-late_blup.full_chr-X_snpsqc_5.batches_mis.csv
    Untracked:  data/ici-early.vs.pbs.ici-late_blup.full_chr-X_snpsqc_dis_no-x_updated_5.batches.csv
    Untracked:  data/ici-early.vs.pbs.ici-late_blup.full_chr-X_snpsqc_dis_no-x_updated_5.batches_mis.csv
    Untracked:  data/ici-early.vs.pbs.ici-late_blup.qc_chr-10_5.batches.csv
    Untracked:  data/ici-early.vs.pbs.ici-late_blup.qc_chr-10_5.batches_mis.csv
    Untracked:  data/ici-early.vs.pbs.ici-late_blup.qc_chr-10_snpsqc_5.batches.csv
    Untracked:  data/ici-early.vs.pbs.ici-late_blup.qc_chr-10_snpsqc_5.batches_mis.csv
    Untracked:  data/ici-early.vs.pbs.ici-late_blup.qc_chr-10_snpsqc_dis_no-x_updated_5.batches.csv
    Untracked:  data/ici-early.vs.pbs.ici-late_blup.qc_chr-10_snpsqc_dis_no-x_updated_5.batches_mis.csv
    Untracked:  data/ici-early.vs.pbs.ici-late_blup.qc_chr-11_5.batches.csv
    Untracked:  data/ici-early.vs.pbs.ici-late_blup.qc_chr-11_5.batches_mis.csv
    Untracked:  data/ici-early.vs.pbs.ici-late_blup.qc_chr-11_snpsqc_5.batches.csv
    Untracked:  data/ici-early.vs.pbs.ici-late_blup.qc_chr-11_snpsqc_5.batches_mis.csv
    Untracked:  data/ici-early.vs.pbs.ici-late_blup.qc_chr-11_snpsqc_dis_no-x_updated_5.batches.csv
    Untracked:  data/ici-early.vs.pbs.ici-late_blup.qc_chr-11_snpsqc_dis_no-x_updated_5.batches_mis.csv
    Untracked:  data/ici-early.vs.pbs.ici-late_blup.qc_chr-12_5.batches.csv
    Untracked:  data/ici-early.vs.pbs.ici-late_blup.qc_chr-12_5.batches_mis.csv
    Untracked:  data/ici-early.vs.pbs.ici-late_blup.qc_chr-12_snpsqc_5.batches.csv
    Untracked:  data/ici-early.vs.pbs.ici-late_blup.qc_chr-12_snpsqc_5.batches_mis.csv
    Untracked:  data/ici-early.vs.pbs.ici-late_blup.qc_chr-12_snpsqc_dis_no-x_updated_5.batches.csv
    Untracked:  data/ici-early.vs.pbs.ici-late_blup.qc_chr-12_snpsqc_dis_no-x_updated_5.batches_mis.csv
    Untracked:  data/ici-early.vs.pbs.ici-late_blup.qc_chr-13_5.batches.csv
    Untracked:  data/ici-early.vs.pbs.ici-late_blup.qc_chr-13_5.batches_mis.csv
    Untracked:  data/ici-early.vs.pbs.ici-late_blup.qc_chr-13_snpsqc_5.batches.csv
    Untracked:  data/ici-early.vs.pbs.ici-late_blup.qc_chr-13_snpsqc_5.batches_mis.csv
    Untracked:  data/ici-early.vs.pbs.ici-late_blup.qc_chr-13_snpsqc_dis_no-x_updated_5.batches.csv
    Untracked:  data/ici-early.vs.pbs.ici-late_blup.qc_chr-13_snpsqc_dis_no-x_updated_5.batches_mis.csv
    Untracked:  data/ici-early.vs.pbs.ici-late_blup.qc_chr-14_5.batches.csv
    Untracked:  data/ici-early.vs.pbs.ici-late_blup.qc_chr-14_5.batches_mis.csv
    Untracked:  data/ici-early.vs.pbs.ici-late_blup.qc_chr-14_snpsqc_5.batches.csv
    Untracked:  data/ici-early.vs.pbs.ici-late_blup.qc_chr-14_snpsqc_5.batches_mis.csv
    Untracked:  data/ici-early.vs.pbs.ici-late_blup.qc_chr-14_snpsqc_dis_no-x_updated_5.batches.csv
    Untracked:  data/ici-early.vs.pbs.ici-late_blup.qc_chr-14_snpsqc_dis_no-x_updated_5.batches_mis.csv
    Untracked:  data/ici-early.vs.pbs.ici-late_blup.qc_chr-15_5.batches.csv
    Untracked:  data/ici-early.vs.pbs.ici-late_blup.qc_chr-15_5.batches_mis.csv
    Untracked:  data/ici-early.vs.pbs.ici-late_blup.qc_chr-15_snpsqc_5.batches.csv
    Untracked:  data/ici-early.vs.pbs.ici-late_blup.qc_chr-15_snpsqc_5.batches_mis.csv
    Untracked:  data/ici-early.vs.pbs.ici-late_blup.qc_chr-16_5.batches.csv
    Untracked:  data/ici-early.vs.pbs.ici-late_blup.qc_chr-16_5.batches_mis.csv
    Untracked:  data/ici-early.vs.pbs.ici-late_blup.qc_chr-16_snpsqc_5.batches.csv
    Untracked:  data/ici-early.vs.pbs.ici-late_blup.qc_chr-16_snpsqc_5.batches_mis.csv
    Untracked:  data/ici-early.vs.pbs.ici-late_blup.qc_chr-16_snpsqc_dis_no-x_updated_5.batches.csv
    Untracked:  data/ici-early.vs.pbs.ici-late_blup.qc_chr-16_snpsqc_dis_no-x_updated_5.batches_mis.csv
    Untracked:  data/ici-early.vs.pbs.ici-late_blup.qc_chr-17_5.batches.csv
    Untracked:  data/ici-early.vs.pbs.ici-late_blup.qc_chr-17_5.batches_mis.csv
    Untracked:  data/ici-early.vs.pbs.ici-late_blup.qc_chr-17_snpsqc_5.batches.csv
    Untracked:  data/ici-early.vs.pbs.ici-late_blup.qc_chr-17_snpsqc_5.batches_mis.csv
    Untracked:  data/ici-early.vs.pbs.ici-late_blup.qc_chr-18_5.batches.csv
    Untracked:  data/ici-early.vs.pbs.ici-late_blup.qc_chr-18_5.batches_mis.csv
    Untracked:  data/ici-early.vs.pbs.ici-late_blup.qc_chr-18_snpsqc_5.batches.csv
    Untracked:  data/ici-early.vs.pbs.ici-late_blup.qc_chr-18_snpsqc_5.batches_mis.csv
    Untracked:  data/ici-early.vs.pbs.ici-late_blup.qc_chr-18_snpsqc_dis_no-x_updated_5.batches.csv
    Untracked:  data/ici-early.vs.pbs.ici-late_blup.qc_chr-18_snpsqc_dis_no-x_updated_5.batches_mis.csv
    Untracked:  data/ici-early.vs.pbs.ici-late_blup.qc_chr-19_5.batches.csv
    Untracked:  data/ici-early.vs.pbs.ici-late_blup.qc_chr-19_5.batches_mis.csv
    Untracked:  data/ici-early.vs.pbs.ici-late_blup.qc_chr-19_snpsqc_5.batches.csv
    Untracked:  data/ici-early.vs.pbs.ici-late_blup.qc_chr-19_snpsqc_5.batches_mis.csv
    Untracked:  data/ici-early.vs.pbs.ici-late_blup.qc_chr-19_snpsqc_dis_no-x_updated_5.batches.csv
    Untracked:  data/ici-early.vs.pbs.ici-late_blup.qc_chr-19_snpsqc_dis_no-x_updated_5.batches_mis.csv
    Untracked:  data/ici-early.vs.pbs.ici-late_blup.qc_chr-1_5.batches.csv
    Untracked:  data/ici-early.vs.pbs.ici-late_blup.qc_chr-1_5.batches_mis.csv
    Untracked:  data/ici-early.vs.pbs.ici-late_blup.qc_chr-1_snpsqc_5.batches.csv
    Untracked:  data/ici-early.vs.pbs.ici-late_blup.qc_chr-1_snpsqc_5.batches_mis.csv
    Untracked:  data/ici-early.vs.pbs.ici-late_blup.qc_chr-1_snpsqc_dis_no-x_updated_5.batches.csv
    Untracked:  data/ici-early.vs.pbs.ici-late_blup.qc_chr-1_snpsqc_dis_no-x_updated_5.batches_mis.csv
    Untracked:  data/ici-early.vs.pbs.ici-late_blup.qc_chr-2_5.batches.csv
    Untracked:  data/ici-early.vs.pbs.ici-late_blup.qc_chr-2_5.batches_mis.csv
    Untracked:  data/ici-early.vs.pbs.ici-late_blup.qc_chr-2_snpsqc_5.batches.csv
    Untracked:  data/ici-early.vs.pbs.ici-late_blup.qc_chr-2_snpsqc_5.batches_mis.csv
    Untracked:  data/ici-early.vs.pbs.ici-late_blup.qc_chr-2_snpsqc_dis_no-x_updated_5.batches.csv
    Untracked:  data/ici-early.vs.pbs.ici-late_blup.qc_chr-2_snpsqc_dis_no-x_updated_5.batches_mis.csv
    Untracked:  data/ici-early.vs.pbs.ici-late_blup.qc_chr-3_5.batches.csv
    Untracked:  data/ici-early.vs.pbs.ici-late_blup.qc_chr-3_5.batches_mis.csv
    Untracked:  data/ici-early.vs.pbs.ici-late_blup.qc_chr-3_snpsqc_5.batches.csv
    Untracked:  data/ici-early.vs.pbs.ici-late_blup.qc_chr-3_snpsqc_5.batches_mis.csv
    Untracked:  data/ici-early.vs.pbs.ici-late_blup.qc_chr-3_snpsqc_dis_no-x_updated_5.batches.csv
    Untracked:  data/ici-early.vs.pbs.ici-late_blup.qc_chr-3_snpsqc_dis_no-x_updated_5.batches_mis.csv
    Untracked:  data/ici-early.vs.pbs.ici-late_blup.qc_chr-4_5.batches.csv
    Untracked:  data/ici-early.vs.pbs.ici-late_blup.qc_chr-4_5.batches_mis.csv
    Untracked:  data/ici-early.vs.pbs.ici-late_blup.qc_chr-4_snpsqc_5.batches.csv
    Untracked:  data/ici-early.vs.pbs.ici-late_blup.qc_chr-4_snpsqc_5.batches_mis.csv
    Untracked:  data/ici-early.vs.pbs.ici-late_blup.qc_chr-4_snpsqc_dis_no-x_updated_5.batches.csv
    Untracked:  data/ici-early.vs.pbs.ici-late_blup.qc_chr-4_snpsqc_dis_no-x_updated_5.batches_mis.csv
    Untracked:  data/ici-early.vs.pbs.ici-late_blup.qc_chr-5_5.batches.csv
    Untracked:  data/ici-early.vs.pbs.ici-late_blup.qc_chr-5_5.batches_mis.csv
    Untracked:  data/ici-early.vs.pbs.ici-late_blup.qc_chr-5_snpsqc_5.batches.csv
    Untracked:  data/ici-early.vs.pbs.ici-late_blup.qc_chr-5_snpsqc_5.batches_mis.csv
    Untracked:  data/ici-early.vs.pbs.ici-late_blup.qc_chr-5_snpsqc_dis_no-x_updated_5.batches.csv
    Untracked:  data/ici-early.vs.pbs.ici-late_blup.qc_chr-5_snpsqc_dis_no-x_updated_5.batches_mis.csv
    Untracked:  data/ici-early.vs.pbs.ici-late_blup.qc_chr-6_5.batches.csv
    Untracked:  data/ici-early.vs.pbs.ici-late_blup.qc_chr-6_5.batches_mis.csv
    Untracked:  data/ici-early.vs.pbs.ici-late_blup.qc_chr-6_snpsqc_5.batches.csv
    Untracked:  data/ici-early.vs.pbs.ici-late_blup.qc_chr-6_snpsqc_5.batches_mis.csv
    Untracked:  data/ici-early.vs.pbs.ici-late_blup.qc_chr-6_snpsqc_dis_no-x_updated_5.batches.csv
    Untracked:  data/ici-early.vs.pbs.ici-late_blup.qc_chr-6_snpsqc_dis_no-x_updated_5.batches_mis.csv
    Untracked:  data/ici-early.vs.pbs.ici-late_blup.qc_chr-7_5.batches.csv
    Untracked:  data/ici-early.vs.pbs.ici-late_blup.qc_chr-7_5.batches_mis.csv
    Untracked:  data/ici-early.vs.pbs.ici-late_blup.qc_chr-7_snpsqc_5.batches.csv
    Untracked:  data/ici-early.vs.pbs.ici-late_blup.qc_chr-7_snpsqc_5.batches_mis.csv
    Untracked:  data/ici-early.vs.pbs.ici-late_blup.qc_chr-7_snpsqc_dis_no-x_updated_5.batches.csv
    Untracked:  data/ici-early.vs.pbs.ici-late_blup.qc_chr-7_snpsqc_dis_no-x_updated_5.batches_mis.csv
    Untracked:  data/ici-early.vs.pbs.ici-late_blup.qc_chr-8_5.batches.csv
    Untracked:  data/ici-early.vs.pbs.ici-late_blup.qc_chr-8_5.batches_mis.csv
    Untracked:  data/ici-early.vs.pbs.ici-late_blup.qc_chr-8_snpsqc_5.batches.csv
    Untracked:  data/ici-early.vs.pbs.ici-late_blup.qc_chr-8_snpsqc_5.batches_mis.csv
    Untracked:  data/ici-early.vs.pbs.ici-late_blup.qc_chr-8_snpsqc_dis_no-x_updated_5.batches.csv
    Untracked:  data/ici-early.vs.pbs.ici-late_blup.qc_chr-8_snpsqc_dis_no-x_updated_5.batches_mis.csv
    Untracked:  data/ici-early.vs.pbs.ici-late_blup.qc_chr-9_5.batches.csv
    Untracked:  data/ici-early.vs.pbs.ici-late_blup.qc_chr-9_5.batches_mis.csv
    Untracked:  data/ici-early.vs.pbs.ici-late_blup.qc_chr-9_snpsqc_5.batches.csv
    Untracked:  data/ici-early.vs.pbs.ici-late_blup.qc_chr-9_snpsqc_5.batches_mis.csv
    Untracked:  data/ici-early.vs.pbs.ici-late_blup.qc_chr-9_snpsqc_dis_no-x_updated_5.batches.csv
    Untracked:  data/ici-early.vs.pbs.ici-late_blup.qc_chr-9_snpsqc_dis_no-x_updated_5.batches_mis.csv
    Untracked:  data/ici-early.vs.pbs.ici-late_blup.qc_chr-X_5.batches.csv
    Untracked:  data/ici-early.vs.pbs.ici-late_blup.qc_chr-X_5.batches_mis.csv
    Untracked:  data/ici-early.vs.pbs.ici-late_blup.qc_chr-X_snpsqc_5.batches.csv
    Untracked:  data/ici-early.vs.pbs.ici-late_blup.qc_chr-X_snpsqc_5.batches_mis.csv
    Untracked:  data/ici-early.vs.pbs.ici-late_blup.qc_chr-X_snpsqc_dis_no-x_updated_5.batches.csv
    Untracked:  data/ici-early.vs.pbs.ici-late_blup.qc_chr-X_snpsqc_dis_no-x_updated_5.batches_mis.csv
    Untracked:  data/ici-early.vs.pbs.ici-late_blup_sub_chr-18_peak.marker-UNCHS045343_lod.drop-1.5_snpsqc_5.batches_mis.csv
    Untracked:  data/ici-early.vs.pbs.ici-late_genes_chr-18_peak.marker-UNCHS045343_lod.drop-1.5_snpsqc_5.batches_mis.csv
    Untracked:  data/ici-early.vs.pbs.ici-late_gm_qtl_5.batches.csv
    Untracked:  data/ici-early.vs.pbs.ici-late_gm_qtl_5.batches_mis.csv
    Untracked:  data/ici-early.vs.pbs.ici-late_gm_qtl_snpsqc_5.batches.csv
    Untracked:  data/ici-early.vs.pbs.ici-late_gm_qtl_snpsqc_5.batches_mis.csv
    Untracked:  data/ici-early.vs.pbs.ici-late_gm_qtl_snpsqc_dis_no-x_updated_5.batches.csv
    Untracked:  data/ici-early.vs.pbs.ici-late_gm_qtl_snpsqc_dis_no-x_updated_5.batches_mis.csv
    Untracked:  data/ici-early.vs.pbs.ici-late_marker.freq_low.geno.freq.removed_geno.ratio_5.batches.csv
    Untracked:  data/ici-early.vs.pbs.ici-late_marker.freq_low.geno.freq.removed_geno.ratio_5.batches_mis.csv
    Untracked:  data/ici-early.vs.pbs.ici-late_marker.freq_low.geno.freq.removed_sample.outliers.removed_geno.ratio_5.batches.csv
    Untracked:  data/ici-early.vs.pbs.ici-late_marker.freq_low.geno.freq.removed_sample.outliers.removed_geno.ratio_5.batches_mis.csv
    Untracked:  data/ici-early.vs.pbs.ici-late_marker.freq_low.probs.freq.removed_geno.ratio_5.batches.csv
    Untracked:  data/ici-early.vs.pbs.ici-late_marker.freq_low.probs.freq.removed_geno.ratio_5.batches_mis.csv
    Untracked:  data/ici-early.vs.pbs.ici-late_marker.freq_low.probs.freq.removed_sample.outliers.removed_geno.ratio_5.batches.csv
    Untracked:  data/ici-early.vs.pbs.ici-late_marker.freq_low.probs.freq.removed_sample.outliers.removed_geno.ratio_5.batches_mis.csv
    Untracked:  data/ici-early.vs.pbs.ici-late_scanone_5.batches.Rdata
    Untracked:  data/ici-early.vs.pbs.ici-late_scanone_5.batches_mis.Rdata
    Untracked:  data/ici-early.vs.pbs.ici-late_scanone_snpsqc_5.batches.Rdata
    Untracked:  data/ici-early.vs.pbs.ici-late_scanone_snpsqc_5.batches_mis.Rdata
    Untracked:  data/ici-early.vs.pbs.ici-late_scanone_snpsqc_dis_no-x_updated_5.batches.Rdata
    Untracked:  data/ici-early.vs.pbs.ici-late_scanone_snpsqc_dis_no-x_updated_5.batches_mis.Rdata
    Untracked:  data/ici-early.vs.pbs_age.of.onset-additive.covariates_blup_sub_chr-7_peak.marker-UNC13388811_lod.drop-1.5_snpsqc_5.batches.csv
    Untracked:  data/ici-early.vs.pbs_age.of.onset-additive.covariates_genes_chr-7_peak.marker-UNC13388811_lod.drop-1.5_snpsqc_5.batches.csv
    Untracked:  data/ici-early.vs.pbs_age.of.onset-no.covariates_blup_sub_chr-14_peak.marker-UNC24056202_lod.drop-1.5_snpsqc_5.batches_mis.csv
    Untracked:  data/ici-early.vs.pbs_age.of.onset-no.covariates_blup_sub_chr-17_peak.marker-UNCJPD006614_lod.drop-1.5_snpsqc_5.batches_mis.csv
    Untracked:  data/ici-early.vs.pbs_age.of.onset-no.covariates_blup_sub_chr-18_peak.marker-UNCHS045343_lod.drop-1.5_snpsqc_5.batches_mis.csv
    Untracked:  data/ici-early.vs.pbs_age.of.onset-no.covariates_blup_sub_chr-8_peak.marker-UNC15524531_lod.drop-1.5_snpsqc_5.batches_mis.csv
    Untracked:  data/ici-early.vs.pbs_age.of.onset-no.covariates_blup_sub_chr-X_peak.marker-UNCHS048314_lod.drop-1.5_snpsqc_5.batches_mis.csv
    Untracked:  data/ici-early.vs.pbs_age.of.onset-no.covariates_genes_chr-14_peak.marker-UNC24056202_lod.drop-1.5_snpsqc_5.batches_mis.csv
    Untracked:  data/ici-early.vs.pbs_age.of.onset-no.covariates_genes_chr-17_peak.marker-UNCJPD006614_lod.drop-1.5_snpsqc_5.batches_mis.csv
    Untracked:  data/ici-early.vs.pbs_age.of.onset-no.covariates_genes_chr-18_peak.marker-UNCHS045343_lod.drop-1.5_snpsqc_5.batches_mis.csv
    Untracked:  data/ici-early.vs.pbs_age.of.onset-no.covariates_genes_chr-8_peak.marker-UNC15524531_lod.drop-1.5_snpsqc_5.batches_mis.csv
    Untracked:  data/ici-early.vs.pbs_age.of.onset-no.covariates_genes_chr-X_peak.marker-UNCHS048314_lod.drop-1.5_snpsqc_5.batches_mis.csv
    Untracked:  data/ici-early.vs.pbs_blup.full_chr-10_5.batches.csv
    Untracked:  data/ici-early.vs.pbs_blup.full_chr-10_5.batches_mis.csv
    Untracked:  data/ici-early.vs.pbs_blup.full_chr-10_snpsqc_5.batches.csv
    Untracked:  data/ici-early.vs.pbs_blup.full_chr-10_snpsqc_5.batches_mis.csv
    Untracked:  data/ici-early.vs.pbs_blup.full_chr-10_snpsqc_dis_no-x_updated_5.batches.csv
    Untracked:  data/ici-early.vs.pbs_blup.full_chr-10_snpsqc_dis_no-x_updated_5.batches_mis.csv
    Untracked:  data/ici-early.vs.pbs_blup.full_chr-11_5.batches.csv
    Untracked:  data/ici-early.vs.pbs_blup.full_chr-11_5.batches_mis.csv
    Untracked:  data/ici-early.vs.pbs_blup.full_chr-11_snpsqc_5.batches.csv
    Untracked:  data/ici-early.vs.pbs_blup.full_chr-11_snpsqc_5.batches_mis.csv
    Untracked:  data/ici-early.vs.pbs_blup.full_chr-11_snpsqc_dis_no-x_updated_5.batches.csv
    Untracked:  data/ici-early.vs.pbs_blup.full_chr-11_snpsqc_dis_no-x_updated_5.batches_mis.csv
    Untracked:  data/ici-early.vs.pbs_blup.full_chr-12_5.batches.csv
    Untracked:  data/ici-early.vs.pbs_blup.full_chr-12_5.batches_mis.csv
    Untracked:  data/ici-early.vs.pbs_blup.full_chr-12_snpsqc_5.batches.csv
    Untracked:  data/ici-early.vs.pbs_blup.full_chr-12_snpsqc_5.batches_mis.csv
    Untracked:  data/ici-early.vs.pbs_blup.full_chr-12_snpsqc_dis_no-x_updated_5.batches.csv
    Untracked:  data/ici-early.vs.pbs_blup.full_chr-12_snpsqc_dis_no-x_updated_5.batches_mis.csv
    Untracked:  data/ici-early.vs.pbs_blup.full_chr-13_5.batches.csv
    Untracked:  data/ici-early.vs.pbs_blup.full_chr-13_5.batches_mis.csv
    Untracked:  data/ici-early.vs.pbs_blup.full_chr-13_snpsqc_5.batches.csv
    Untracked:  data/ici-early.vs.pbs_blup.full_chr-13_snpsqc_5.batches_mis.csv
    Untracked:  data/ici-early.vs.pbs_blup.full_chr-13_snpsqc_dis_no-x_updated_5.batches.csv
    Untracked:  data/ici-early.vs.pbs_blup.full_chr-13_snpsqc_dis_no-x_updated_5.batches_mis.csv
    Untracked:  data/ici-early.vs.pbs_blup.full_chr-14_5.batches.csv
    Untracked:  data/ici-early.vs.pbs_blup.full_chr-14_5.batches_mis.csv
    Untracked:  data/ici-early.vs.pbs_blup.full_chr-14_snpsqc_5.batches.csv
    Untracked:  data/ici-early.vs.pbs_blup.full_chr-14_snpsqc_5.batches_mis.csv
    Untracked:  data/ici-early.vs.pbs_blup.full_chr-14_snpsqc_dis_no-x_updated_5.batches.csv
    Untracked:  data/ici-early.vs.pbs_blup.full_chr-14_snpsqc_dis_no-x_updated_5.batches_mis.csv
    Untracked:  data/ici-early.vs.pbs_blup.full_chr-15_5.batches.csv
    Untracked:  data/ici-early.vs.pbs_blup.full_chr-15_5.batches_mis.csv
    Untracked:  data/ici-early.vs.pbs_blup.full_chr-15_snpsqc_5.batches.csv
    Untracked:  data/ici-early.vs.pbs_blup.full_chr-15_snpsqc_5.batches_mis.csv
    Untracked:  data/ici-early.vs.pbs_blup.full_chr-15_snpsqc_dis_no-x_updated_5.batches.csv
    Untracked:  data/ici-early.vs.pbs_blup.full_chr-15_snpsqc_dis_no-x_updated_5.batches_mis.csv
    Untracked:  data/ici-early.vs.pbs_blup.full_chr-16_5.batches.csv
    Untracked:  data/ici-early.vs.pbs_blup.full_chr-16_5.batches_mis.csv
    Untracked:  data/ici-early.vs.pbs_blup.full_chr-16_snpsqc_5.batches.csv
    Untracked:  data/ici-early.vs.pbs_blup.full_chr-16_snpsqc_5.batches_mis.csv
    Untracked:  data/ici-early.vs.pbs_blup.full_chr-16_snpsqc_dis_no-x_updated_5.batches.csv
    Untracked:  data/ici-early.vs.pbs_blup.full_chr-16_snpsqc_dis_no-x_updated_5.batches_mis.csv
    Untracked:  data/ici-early.vs.pbs_blup.full_chr-17_5.batches.csv
    Untracked:  data/ici-early.vs.pbs_blup.full_chr-17_5.batches_mis.csv
    Untracked:  data/ici-early.vs.pbs_blup.full_chr-17_snpsqc_5.batches.csv
    Untracked:  data/ici-early.vs.pbs_blup.full_chr-17_snpsqc_5.batches_mis.csv
    Untracked:  data/ici-early.vs.pbs_blup.full_chr-17_snpsqc_dis_no-x_updated_5.batches.csv
    Untracked:  data/ici-early.vs.pbs_blup.full_chr-17_snpsqc_dis_no-x_updated_5.batches_mis.csv
    Untracked:  data/ici-early.vs.pbs_blup.full_chr-18_5.batches.csv
    Untracked:  data/ici-early.vs.pbs_blup.full_chr-18_5.batches_mis.csv
    Untracked:  data/ici-early.vs.pbs_blup.full_chr-18_snpsqc_5.batches.csv
    Untracked:  data/ici-early.vs.pbs_blup.full_chr-18_snpsqc_5.batches_mis.csv
    Untracked:  data/ici-early.vs.pbs_blup.full_chr-18_snpsqc_dis_no-x_updated_5.batches.csv
    Untracked:  data/ici-early.vs.pbs_blup.full_chr-18_snpsqc_dis_no-x_updated_5.batches_mis.csv
    Untracked:  data/ici-early.vs.pbs_blup.full_chr-19_5.batches.csv
    Untracked:  data/ici-early.vs.pbs_blup.full_chr-19_5.batches_mis.csv
    Untracked:  data/ici-early.vs.pbs_blup.full_chr-19_snpsqc_5.batches.csv
    Untracked:  data/ici-early.vs.pbs_blup.full_chr-19_snpsqc_5.batches_mis.csv
    Untracked:  data/ici-early.vs.pbs_blup.full_chr-19_snpsqc_dis_no-x_updated_5.batches.csv
    Untracked:  data/ici-early.vs.pbs_blup.full_chr-19_snpsqc_dis_no-x_updated_5.batches_mis.csv
    Untracked:  data/ici-early.vs.pbs_blup.full_chr-1_5.batches.csv
    Untracked:  data/ici-early.vs.pbs_blup.full_chr-1_5.batches_mis.csv
    Untracked:  data/ici-early.vs.pbs_blup.full_chr-1_snpsqc_5.batches.csv
    Untracked:  data/ici-early.vs.pbs_blup.full_chr-1_snpsqc_5.batches_mis.csv
    Untracked:  data/ici-early.vs.pbs_blup.full_chr-1_snpsqc_dis_no-x_updated_5.batches.csv
    Untracked:  data/ici-early.vs.pbs_blup.full_chr-1_snpsqc_dis_no-x_updated_5.batches_mis.csv
    Untracked:  data/ici-early.vs.pbs_blup.full_chr-2_5.batches.csv
    Untracked:  data/ici-early.vs.pbs_blup.full_chr-2_5.batches_mis.csv
    Untracked:  data/ici-early.vs.pbs_blup.full_chr-2_snpsqc_5.batches.csv
    Untracked:  data/ici-early.vs.pbs_blup.full_chr-2_snpsqc_5.batches_mis.csv
    Untracked:  data/ici-early.vs.pbs_blup.full_chr-2_snpsqc_dis_no-x_updated_5.batches.csv
    Untracked:  data/ici-early.vs.pbs_blup.full_chr-2_snpsqc_dis_no-x_updated_5.batches_mis.csv
    Untracked:  data/ici-early.vs.pbs_blup.full_chr-3_5.batches.csv
    Untracked:  data/ici-early.vs.pbs_blup.full_chr-3_5.batches_mis.csv
    Untracked:  data/ici-early.vs.pbs_blup.full_chr-3_snpsqc_5.batches.csv
    Untracked:  data/ici-early.vs.pbs_blup.full_chr-3_snpsqc_5.batches_mis.csv
    Untracked:  data/ici-early.vs.pbs_blup.full_chr-3_snpsqc_dis_no-x_updated_5.batches.csv
    Untracked:  data/ici-early.vs.pbs_blup.full_chr-3_snpsqc_dis_no-x_updated_5.batches_mis.csv
    Untracked:  data/ici-early.vs.pbs_blup.full_chr-4_5.batches.csv
    Untracked:  data/ici-early.vs.pbs_blup.full_chr-4_5.batches_mis.csv
    Untracked:  data/ici-early.vs.pbs_blup.full_chr-4_snpsqc_5.batches.csv
    Untracked:  data/ici-early.vs.pbs_blup.full_chr-4_snpsqc_5.batches_mis.csv
    Untracked:  data/ici-early.vs.pbs_blup.full_chr-4_snpsqc_dis_no-x_updated_5.batches.csv
    Untracked:  data/ici-early.vs.pbs_blup.full_chr-4_snpsqc_dis_no-x_updated_5.batches_mis.csv
    Untracked:  data/ici-early.vs.pbs_blup.full_chr-5_5.batches.csv
    Untracked:  data/ici-early.vs.pbs_blup.full_chr-5_5.batches_mis.csv
    Untracked:  data/ici-early.vs.pbs_blup.full_chr-5_snpsqc_5.batches.csv
    Untracked:  data/ici-early.vs.pbs_blup.full_chr-5_snpsqc_5.batches_mis.csv
    Untracked:  data/ici-early.vs.pbs_blup.full_chr-5_snpsqc_dis_no-x_updated_5.batches.csv
    Untracked:  data/ici-early.vs.pbs_blup.full_chr-5_snpsqc_dis_no-x_updated_5.batches_mis.csv
    Untracked:  data/ici-early.vs.pbs_blup.full_chr-6_5.batches.csv
    Untracked:  data/ici-early.vs.pbs_blup.full_chr-6_5.batches_mis.csv
    Untracked:  data/ici-early.vs.pbs_blup.full_chr-6_snpsqc_5.batches.csv
    Untracked:  data/ici-early.vs.pbs_blup.full_chr-6_snpsqc_5.batches_mis.csv
    Untracked:  data/ici-early.vs.pbs_blup.full_chr-6_snpsqc_dis_no-x_updated_5.batches.csv
    Untracked:  data/ici-early.vs.pbs_blup.full_chr-6_snpsqc_dis_no-x_updated_5.batches_mis.csv
    Untracked:  data/ici-early.vs.pbs_blup.full_chr-7_5.batches.csv
    Untracked:  data/ici-early.vs.pbs_blup.full_chr-7_5.batches_mis.csv
    Untracked:  data/ici-early.vs.pbs_blup.full_chr-7_snpsqc_5.batches.csv
    Untracked:  data/ici-early.vs.pbs_blup.full_chr-7_snpsqc_5.batches_mis.csv
    Untracked:  data/ici-early.vs.pbs_blup.full_chr-7_snpsqc_dis_no-x_updated_5.batches.csv
    Untracked:  data/ici-early.vs.pbs_blup.full_chr-7_snpsqc_dis_no-x_updated_5.batches_mis.csv
    Untracked:  data/ici-early.vs.pbs_blup.full_chr-8_5.batches.csv
    Untracked:  data/ici-early.vs.pbs_blup.full_chr-8_5.batches_mis.csv
    Untracked:  data/ici-early.vs.pbs_blup.full_chr-8_snpsqc_5.batches.csv
    Untracked:  data/ici-early.vs.pbs_blup.full_chr-8_snpsqc_5.batches_mis.csv
    Untracked:  data/ici-early.vs.pbs_blup.full_chr-8_snpsqc_dis_no-x_updated_5.batches.csv
    Untracked:  data/ici-early.vs.pbs_blup.full_chr-8_snpsqc_dis_no-x_updated_5.batches_mis.csv
    Untracked:  data/ici-early.vs.pbs_blup.full_chr-9_5.batches.csv
    Untracked:  data/ici-early.vs.pbs_blup.full_chr-9_5.batches_mis.csv
    Untracked:  data/ici-early.vs.pbs_blup.full_chr-9_snpsqc_5.batches.csv
    Untracked:  data/ici-early.vs.pbs_blup.full_chr-9_snpsqc_5.batches_mis.csv
    Untracked:  data/ici-early.vs.pbs_blup.full_chr-9_snpsqc_dis_no-x_updated_5.batches.csv
    Untracked:  data/ici-early.vs.pbs_blup.full_chr-9_snpsqc_dis_no-x_updated_5.batches_mis.csv
    Untracked:  data/ici-early.vs.pbs_blup.full_chr-X_5.batches.csv
    Untracked:  data/ici-early.vs.pbs_blup.full_chr-X_5.batches_mis.csv
    Untracked:  data/ici-early.vs.pbs_blup.full_chr-X_snpsqc_5.batches.csv
    Untracked:  data/ici-early.vs.pbs_blup.full_chr-X_snpsqc_5.batches_mis.csv
    Untracked:  data/ici-early.vs.pbs_blup.full_chr-X_snpsqc_dis_no-x_updated_5.batches.csv
    Untracked:  data/ici-early.vs.pbs_blup.full_chr-X_snpsqc_dis_no-x_updated_5.batches_mis.csv
    Untracked:  data/ici-early.vs.pbs_blup.qc_chr-10_5.batches.csv
    Untracked:  data/ici-early.vs.pbs_blup.qc_chr-10_5.batches_mis.csv
    Untracked:  data/ici-early.vs.pbs_blup.qc_chr-10_snpsqc_5.batches.csv
    Untracked:  data/ici-early.vs.pbs_blup.qc_chr-10_snpsqc_5.batches_mis.csv
    Untracked:  data/ici-early.vs.pbs_blup.qc_chr-10_snpsqc_dis_no-x_updated_5.batches.csv
    Untracked:  data/ici-early.vs.pbs_blup.qc_chr-10_snpsqc_dis_no-x_updated_5.batches_mis.csv
    Untracked:  data/ici-early.vs.pbs_blup.qc_chr-11_5.batches.csv
    Untracked:  data/ici-early.vs.pbs_blup.qc_chr-11_5.batches_mis.csv
    Untracked:  data/ici-early.vs.pbs_blup.qc_chr-11_snpsqc_5.batches.csv
    Untracked:  data/ici-early.vs.pbs_blup.qc_chr-11_snpsqc_5.batches_mis.csv
    Untracked:  data/ici-early.vs.pbs_blup.qc_chr-11_snpsqc_dis_no-x_updated_5.batches.csv
    Untracked:  data/ici-early.vs.pbs_blup.qc_chr-11_snpsqc_dis_no-x_updated_5.batches_mis.csv
    Untracked:  data/ici-early.vs.pbs_blup.qc_chr-12_5.batches.csv
    Untracked:  data/ici-early.vs.pbs_blup.qc_chr-12_5.batches_mis.csv
    Untracked:  data/ici-early.vs.pbs_blup.qc_chr-12_snpsqc_5.batches.csv
    Untracked:  data/ici-early.vs.pbs_blup.qc_chr-12_snpsqc_5.batches_mis.csv
    Untracked:  data/ici-early.vs.pbs_blup.qc_chr-12_snpsqc_dis_no-x_updated_5.batches.csv
    Untracked:  data/ici-early.vs.pbs_blup.qc_chr-12_snpsqc_dis_no-x_updated_5.batches_mis.csv
    Untracked:  data/ici-early.vs.pbs_blup.qc_chr-13_5.batches.csv
    Untracked:  data/ici-early.vs.pbs_blup.qc_chr-13_5.batches_mis.csv
    Untracked:  data/ici-early.vs.pbs_blup.qc_chr-13_snpsqc_5.batches.csv
    Untracked:  data/ici-early.vs.pbs_blup.qc_chr-13_snpsqc_5.batches_mis.csv
    Untracked:  data/ici-early.vs.pbs_blup.qc_chr-13_snpsqc_dis_no-x_updated_5.batches.csv
    Untracked:  data/ici-early.vs.pbs_blup.qc_chr-13_snpsqc_dis_no-x_updated_5.batches_mis.csv
    Untracked:  data/ici-early.vs.pbs_blup.qc_chr-14_5.batches.csv
    Untracked:  data/ici-early.vs.pbs_blup.qc_chr-14_5.batches_mis.csv
    Untracked:  data/ici-early.vs.pbs_blup.qc_chr-14_snpsqc_5.batches.csv
    Untracked:  data/ici-early.vs.pbs_blup.qc_chr-14_snpsqc_5.batches_mis.csv
    Untracked:  data/ici-early.vs.pbs_blup.qc_chr-14_snpsqc_dis_no-x_updated_5.batches.csv
    Untracked:  data/ici-early.vs.pbs_blup.qc_chr-14_snpsqc_dis_no-x_updated_5.batches_mis.csv
    Untracked:  data/ici-early.vs.pbs_blup.qc_chr-15_5.batches.csv
    Untracked:  data/ici-early.vs.pbs_blup.qc_chr-15_5.batches_mis.csv
    Untracked:  data/ici-early.vs.pbs_blup.qc_chr-15_snpsqc_5.batches.csv
    Untracked:  data/ici-early.vs.pbs_blup.qc_chr-15_snpsqc_5.batches_mis.csv
    Untracked:  data/ici-early.vs.pbs_blup.qc_chr-16_5.batches.csv
    Untracked:  data/ici-early.vs.pbs_blup.qc_chr-16_5.batches_mis.csv
    Untracked:  data/ici-early.vs.pbs_blup.qc_chr-16_snpsqc_5.batches.csv
    Untracked:  data/ici-early.vs.pbs_blup.qc_chr-16_snpsqc_5.batches_mis.csv
    Untracked:  data/ici-early.vs.pbs_blup.qc_chr-16_snpsqc_dis_no-x_updated_5.batches.csv
    Untracked:  data/ici-early.vs.pbs_blup.qc_chr-16_snpsqc_dis_no-x_updated_5.batches_mis.csv
    Untracked:  data/ici-early.vs.pbs_blup.qc_chr-17_5.batches.csv
    Untracked:  data/ici-early.vs.pbs_blup.qc_chr-17_5.batches_mis.csv
    Untracked:  data/ici-early.vs.pbs_blup.qc_chr-17_snpsqc_5.batches.csv
    Untracked:  data/ici-early.vs.pbs_blup.qc_chr-17_snpsqc_5.batches_mis.csv
    Untracked:  data/ici-early.vs.pbs_blup.qc_chr-18_5.batches.csv
    Untracked:  data/ici-early.vs.pbs_blup.qc_chr-18_5.batches_mis.csv
    Untracked:  data/ici-early.vs.pbs_blup.qc_chr-18_snpsqc_5.batches.csv
    Untracked:  data/ici-early.vs.pbs_blup.qc_chr-18_snpsqc_5.batches_mis.csv
    Untracked:  data/ici-early.vs.pbs_blup.qc_chr-18_snpsqc_dis_no-x_updated_5.batches.csv
    Untracked:  data/ici-early.vs.pbs_blup.qc_chr-18_snpsqc_dis_no-x_updated_5.batches_mis.csv
    Untracked:  data/ici-early.vs.pbs_blup.qc_chr-19_5.batches.csv
    Untracked:  data/ici-early.vs.pbs_blup.qc_chr-19_5.batches_mis.csv
    Untracked:  data/ici-early.vs.pbs_blup.qc_chr-19_snpsqc_5.batches.csv
    Untracked:  data/ici-early.vs.pbs_blup.qc_chr-19_snpsqc_5.batches_mis.csv
    Untracked:  data/ici-early.vs.pbs_blup.qc_chr-19_snpsqc_dis_no-x_updated_5.batches.csv
    Untracked:  data/ici-early.vs.pbs_blup.qc_chr-19_snpsqc_dis_no-x_updated_5.batches_mis.csv
    Untracked:  data/ici-early.vs.pbs_blup.qc_chr-1_5.batches.csv
    Untracked:  data/ici-early.vs.pbs_blup.qc_chr-1_5.batches_mis.csv
    Untracked:  data/ici-early.vs.pbs_blup.qc_chr-1_snpsqc_5.batches.csv
    Untracked:  data/ici-early.vs.pbs_blup.qc_chr-1_snpsqc_5.batches_mis.csv
    Untracked:  data/ici-early.vs.pbs_blup.qc_chr-1_snpsqc_dis_no-x_updated_5.batches.csv
    Untracked:  data/ici-early.vs.pbs_blup.qc_chr-1_snpsqc_dis_no-x_updated_5.batches_mis.csv
    Untracked:  data/ici-early.vs.pbs_blup.qc_chr-2_5.batches.csv
    Untracked:  data/ici-early.vs.pbs_blup.qc_chr-2_5.batches_mis.csv
    Untracked:  data/ici-early.vs.pbs_blup.qc_chr-2_snpsqc_5.batches.csv
    Untracked:  data/ici-early.vs.pbs_blup.qc_chr-2_snpsqc_5.batches_mis.csv
    Untracked:  data/ici-early.vs.pbs_blup.qc_chr-2_snpsqc_dis_no-x_updated_5.batches.csv
    Untracked:  data/ici-early.vs.pbs_blup.qc_chr-2_snpsqc_dis_no-x_updated_5.batches_mis.csv
    Untracked:  data/ici-early.vs.pbs_blup.qc_chr-3_5.batches.csv
    Untracked:  data/ici-early.vs.pbs_blup.qc_chr-3_5.batches_mis.csv
    Untracked:  data/ici-early.vs.pbs_blup.qc_chr-3_snpsqc_5.batches.csv
    Untracked:  data/ici-early.vs.pbs_blup.qc_chr-3_snpsqc_5.batches_mis.csv
    Untracked:  data/ici-early.vs.pbs_blup.qc_chr-3_snpsqc_dis_no-x_updated_5.batches.csv
    Untracked:  data/ici-early.vs.pbs_blup.qc_chr-3_snpsqc_dis_no-x_updated_5.batches_mis.csv
    Untracked:  data/ici-early.vs.pbs_blup.qc_chr-4_5.batches.csv
    Untracked:  data/ici-early.vs.pbs_blup.qc_chr-4_5.batches_mis.csv
    Untracked:  data/ici-early.vs.pbs_blup.qc_chr-4_snpsqc_5.batches.csv
    Untracked:  data/ici-early.vs.pbs_blup.qc_chr-4_snpsqc_5.batches_mis.csv
    Untracked:  data/ici-early.vs.pbs_blup.qc_chr-4_snpsqc_dis_no-x_updated_5.batches.csv
    Untracked:  data/ici-early.vs.pbs_blup.qc_chr-4_snpsqc_dis_no-x_updated_5.batches_mis.csv
    Untracked:  data/ici-early.vs.pbs_blup.qc_chr-5_5.batches.csv
    Untracked:  data/ici-early.vs.pbs_blup.qc_chr-5_5.batches_mis.csv
    Untracked:  data/ici-early.vs.pbs_blup.qc_chr-5_snpsqc_5.batches.csv
    Untracked:  data/ici-early.vs.pbs_blup.qc_chr-5_snpsqc_5.batches_mis.csv
    Untracked:  data/ici-early.vs.pbs_blup.qc_chr-5_snpsqc_dis_no-x_updated_5.batches.csv
    Untracked:  data/ici-early.vs.pbs_blup.qc_chr-5_snpsqc_dis_no-x_updated_5.batches_mis.csv
    Untracked:  data/ici-early.vs.pbs_blup.qc_chr-6_5.batches.csv
    Untracked:  data/ici-early.vs.pbs_blup.qc_chr-6_5.batches_mis.csv
    Untracked:  data/ici-early.vs.pbs_blup.qc_chr-6_snpsqc_5.batches.csv
    Untracked:  data/ici-early.vs.pbs_blup.qc_chr-6_snpsqc_5.batches_mis.csv
    Untracked:  data/ici-early.vs.pbs_blup.qc_chr-6_snpsqc_dis_no-x_updated_5.batches.csv
    Untracked:  data/ici-early.vs.pbs_blup.qc_chr-6_snpsqc_dis_no-x_updated_5.batches_mis.csv
    Untracked:  data/ici-early.vs.pbs_blup.qc_chr-7_5.batches.csv
    Untracked:  data/ici-early.vs.pbs_blup.qc_chr-7_5.batches_mis.csv
    Untracked:  data/ici-early.vs.pbs_blup.qc_chr-7_snpsqc_5.batches.csv
    Untracked:  data/ici-early.vs.pbs_blup.qc_chr-7_snpsqc_5.batches_mis.csv
    Untracked:  data/ici-early.vs.pbs_blup.qc_chr-7_snpsqc_dis_no-x_updated_5.batches.csv
    Untracked:  data/ici-early.vs.pbs_blup.qc_chr-7_snpsqc_dis_no-x_updated_5.batches_mis.csv
    Untracked:  data/ici-early.vs.pbs_blup.qc_chr-8_5.batches.csv
    Untracked:  data/ici-early.vs.pbs_blup.qc_chr-8_5.batches_mis.csv
    Untracked:  data/ici-early.vs.pbs_blup.qc_chr-8_snpsqc_5.batches.csv
    Untracked:  data/ici-early.vs.pbs_blup.qc_chr-8_snpsqc_5.batches_mis.csv
    Untracked:  data/ici-early.vs.pbs_blup.qc_chr-8_snpsqc_dis_no-x_updated_5.batches.csv
    Untracked:  data/ici-early.vs.pbs_blup.qc_chr-8_snpsqc_dis_no-x_updated_5.batches_mis.csv
    Untracked:  data/ici-early.vs.pbs_blup.qc_chr-9_5.batches.csv
    Untracked:  data/ici-early.vs.pbs_blup.qc_chr-9_5.batches_mis.csv
    Untracked:  data/ici-early.vs.pbs_blup.qc_chr-9_snpsqc_5.batches.csv
    Untracked:  data/ici-early.vs.pbs_blup.qc_chr-9_snpsqc_5.batches_mis.csv
    Untracked:  data/ici-early.vs.pbs_blup.qc_chr-9_snpsqc_dis_no-x_updated_5.batches.csv
    Untracked:  data/ici-early.vs.pbs_blup.qc_chr-9_snpsqc_dis_no-x_updated_5.batches_mis.csv
    Untracked:  data/ici-early.vs.pbs_blup.qc_chr-X_5.batches.csv
    Untracked:  data/ici-early.vs.pbs_blup.qc_chr-X_5.batches_mis.csv
    Untracked:  data/ici-early.vs.pbs_blup.qc_chr-X_snpsqc_5.batches.csv
    Untracked:  data/ici-early.vs.pbs_blup.qc_chr-X_snpsqc_5.batches_mis.csv
    Untracked:  data/ici-early.vs.pbs_blup.qc_chr-X_snpsqc_dis_no-x_updated_5.batches.csv
    Untracked:  data/ici-early.vs.pbs_blup.qc_chr-X_snpsqc_dis_no-x_updated_5.batches_mis.csv
    Untracked:  data/ici-early.vs.pbs_blup_sub_chr-18_peak.marker-UNCHS045343_lod.drop-1.5_snpsqc_5.batches_mis.csv
    Untracked:  data/ici-early.vs.pbs_genes_chr-18_peak.marker-UNCHS045343_lod.drop-1.5_snpsqc_5.batches_mis.csv
    Untracked:  data/ici-early.vs.pbs_gm_qtl_5.batches.csv
    Untracked:  data/ici-early.vs.pbs_gm_qtl_5.batches_mis.csv
    Untracked:  data/ici-early.vs.pbs_gm_qtl_snpsqc_5.batches.csv
    Untracked:  data/ici-early.vs.pbs_gm_qtl_snpsqc_5.batches_mis.csv
    Untracked:  data/ici-early.vs.pbs_gm_qtl_snpsqc_dis_no-x_updated_5.batches.csv
    Untracked:  data/ici-early.vs.pbs_gm_qtl_snpsqc_dis_no-x_updated_5.batches_mis.csv
    Untracked:  data/ici-early.vs.pbs_marker.freq_low.geno.freq.removed_geno.ratio_5.batches.csv
    Untracked:  data/ici-early.vs.pbs_marker.freq_low.geno.freq.removed_geno.ratio_5.batches_mis.csv
    Untracked:  data/ici-early.vs.pbs_marker.freq_low.geno.freq.removed_sample.outliers.removed_geno.ratio_5.batches.csv
    Untracked:  data/ici-early.vs.pbs_marker.freq_low.geno.freq.removed_sample.outliers.removed_geno.ratio_5.batches_mis.csv
    Untracked:  data/ici-early.vs.pbs_marker.freq_low.probs.freq.removed_geno.ratio_5.batches.csv
    Untracked:  data/ici-early.vs.pbs_marker.freq_low.probs.freq.removed_geno.ratio_5.batches_mis.csv
    Untracked:  data/ici-early.vs.pbs_marker.freq_low.probs.freq.removed_sample.outliers.removed_geno.ratio_5.batches.csv
    Untracked:  data/ici-early.vs.pbs_marker.freq_low.probs.freq.removed_sample.outliers.removed_geno.ratio_5.batches_mis.csv
    Untracked:  data/ici-early.vs.pbs_rz.age-additive.covariates_blup_sub_chr-7_peak.marker-UNC13239362_lod.drop-1.5_snpsqc_5.batches.csv
    Untracked:  data/ici-early.vs.pbs_rz.age-additive.covariates_blup_sub_chr-7_peak.marker-UNC13239362_lod.drop-1.5_snpsqc_5.batches_mis.csv
    Untracked:  data/ici-early.vs.pbs_rz.age-additive.covariates_blup_sub_chr-7_peak.marker-UNC13239362_lod.drop-1.5_snpsqc_dis_no-x_updated_5.batches.csv
    Untracked:  data/ici-early.vs.pbs_rz.age-additive.covariates_blup_sub_chr-7_peak.marker-UNC13239362_lod.drop-1.5_snpsqc_dis_no-x_updated_5.batches_mis.csv
    Untracked:  data/ici-early.vs.pbs_rz.age-additive.covariates_genes_chr-7_peak.marker-UNC13239362_lod.drop-1.5_snpsqc_5.batches.csv
    Untracked:  data/ici-early.vs.pbs_rz.age-additive.covariates_genes_chr-7_peak.marker-UNC13239362_lod.drop-1.5_snpsqc_5.batches_mis.csv
    Untracked:  data/ici-early.vs.pbs_rz.age-additive.covariates_genes_chr-7_peak.marker-UNC13239362_lod.drop-1.5_snpsqc_dis_no-x_updated_5.batches.csv
    Untracked:  data/ici-early.vs.pbs_rz.age-additive.covariates_genes_chr-7_peak.marker-UNC13239362_lod.drop-1.5_snpsqc_dis_no-x_updated_5.batches_mis.csv
    Untracked:  data/ici-early.vs.pbs_scanone_5.batches.Rdata
    Untracked:  data/ici-early.vs.pbs_scanone_5.batches_mis.Rdata
    Untracked:  data/ici-early.vs.pbs_scanone_snpsqc_5.batches.Rdata
    Untracked:  data/ici-early.vs.pbs_scanone_snpsqc_5.batches_mis.Rdata
    Untracked:  data/ici-early.vs.pbs_scanone_snpsqc_dis_no-x_updated_5.batches.Rdata
    Untracked:  data/ici-early.vs.pbs_scanone_snpsqc_dis_no-x_updated_5.batches_mis.Rdata
    Untracked:  data/ici.vs.eoi_age.of.onset-no.covariates_blup_sub_chr-10_peak.marker-JAX00020646_lod.drop-1.5_5.batches_0.csv
    Untracked:  data/ici.vs.eoi_age.of.onset-no.covariates_blup_sub_chr-10_peak.marker-JAX00292499_lod.drop-1.5_5.batches_52.csv
    Untracked:  data/ici.vs.eoi_age.of.onset-no.covariates_blup_sub_chr-10_peak.marker-JAX00292927_lod.drop-1.5_5.batches_0.csv
    Untracked:  data/ici.vs.eoi_age.of.onset-no.covariates_blup_sub_chr-10_peak.marker-JAX00292927_lod.drop-1.5_5.batches_52.csv
    Untracked:  data/ici.vs.eoi_age.of.onset-no.covariates_blup_sub_chr-10_peak.marker-JAX00294019_lod.drop-1.5_snpsqc_dis_no-x_updated_5.batches_0.csv
    Untracked:  data/ici.vs.eoi_age.of.onset-no.covariates_blup_sub_chr-10_peak.marker-UNC18216614_lod.drop-1.5_5.batches_52.csv
    Untracked:  data/ici.vs.eoi_age.of.onset-no.covariates_blup_sub_chr-10_peak.marker-UNC18240977_lod.drop-1.5_snpsqc_dis_no-x_updated_5.batches_52.csv
    Untracked:  data/ici.vs.eoi_age.of.onset-no.covariates_blup_sub_chr-10_peak.marker-UNC18311938_lod.drop-1.5_5.batches_0.csv
    Untracked:  data/ici.vs.eoi_age.of.onset-no.covariates_blup_sub_chr-10_peak.marker-UNC18311938_lod.drop-1.5_5.batches_52.csv
    Untracked:  data/ici.vs.eoi_age.of.onset-no.covariates_blup_sub_chr-10_peak.marker-UNC18311938_lod.drop-1.5_snpsqc_5.batches_52.csv
    Untracked:  data/ici.vs.eoi_age.of.onset-no.covariates_blup_sub_chr-10_peak.marker-UNC18311938_lod.drop-1.5_snpsqc_5.batches_mis_52.csv
    Untracked:  data/ici.vs.eoi_age.of.onset-no.covariates_blup_sub_chr-10_peak.marker-UNC18343181_lod.drop-1.5_5.batches_0.csv
    Untracked:  data/ici.vs.eoi_age.of.onset-no.covariates_blup_sub_chr-10_peak.marker-UNC18363544_lod.drop-1.5_5.batches_0.csv
    Untracked:  data/ici.vs.eoi_age.of.onset-no.covariates_blup_sub_chr-10_peak.marker-UNC18363544_lod.drop-1.5_5.batches_52.csv
    Untracked:  data/ici.vs.eoi_age.of.onset-no.covariates_blup_sub_chr-10_peak.marker-UNC18363544_lod.drop-1.5_snpsqc_5.batches_mis_0.csv
    Untracked:  data/ici.vs.eoi_age.of.onset-no.covariates_blup_sub_chr-10_peak.marker-UNC18376338_lod.drop-1.5_snpsqc_5.batches_0.csv
    Untracked:  data/ici.vs.eoi_age.of.onset-no.covariates_blup_sub_chr-10_peak.marker-UNCHS028236_lod.drop-1.5_snpsqc_5.batches_mis_0.csv
    Untracked:  data/ici.vs.eoi_age.of.onset-no.covariates_blup_sub_chr-10_peak.marker-UNCHS028236_lod.drop-1.5_snpsqc_5.batches_mis_52.csv
    Untracked:  data/ici.vs.eoi_age.of.onset-no.covariates_blup_sub_chr-10_peak.marker-UNCHS028536_lod.drop-1.5_snpsqc_dis_no-x_updated_5.batches_mis_0.csv
    Untracked:  data/ici.vs.eoi_age.of.onset-no.covariates_blup_sub_chr-10_peak.marker-UNCHS028536_lod.drop-1.5_snpsqc_dis_no-x_updated_5.batches_mis_52.csv
    Untracked:  data/ici.vs.eoi_age.of.onset-no.covariates_blup_sub_chr-11_peak.marker-UNC19970181_lod.drop-1.5_snpsqc_5.batches_mis_0.csv
    Untracked:  data/ici.vs.eoi_age.of.onset-no.covariates_blup_sub_chr-11_peak.marker-UNC19970181_lod.drop-1.5_snpsqc_5.batches_mis_52.csv
    Untracked:  data/ici.vs.eoi_age.of.onset-no.covariates_blup_sub_chr-11_peak.marker-UNC20090524_lod.drop-1.5_snpsqc_5.batches_52.csv
    Untracked:  data/ici.vs.eoi_age.of.onset-no.covariates_blup_sub_chr-12_peak.marker-ICR499_lod.drop-1.5_snpsqc_5.batches_mis_0.csv
    Untracked:  data/ici.vs.eoi_age.of.onset-no.covariates_blup_sub_chr-12_peak.marker-ICR499_lod.drop-1.5_snpsqc_5.batches_mis_52.csv
    Untracked:  data/ici.vs.eoi_age.of.onset-no.covariates_blup_sub_chr-12_peak.marker-ICR499_lod.drop-1.5_snpsqc_dis_no-x_updated_5.batches_mis_0.csv
    Untracked:  data/ici.vs.eoi_age.of.onset-no.covariates_blup_sub_chr-13_peak.marker-UNCHS036773_lod.drop-1.5_snpsqc_5.batches_mis_0.csv
    Untracked:  data/ici.vs.eoi_age.of.onset-no.covariates_blup_sub_chr-13_peak.marker-UNCHS036773_lod.drop-1.5_snpsqc_5.batches_mis_52.csv
    Untracked:  data/ici.vs.eoi_age.of.onset-no.covariates_blup_sub_chr-14_peak.marker-UNC24056202_lod.drop-1.5_snpsqc_5.batches_mis_0.csv
    Untracked:  data/ici.vs.eoi_age.of.onset-no.covariates_blup_sub_chr-14_peak.marker-UNC24056202_lod.drop-1.5_snpsqc_5.batches_mis_52.csv
    Untracked:  data/ici.vs.eoi_age.of.onset-no.covariates_blup_sub_chr-15_peak.marker-UNC26070435_lod.drop-1.5_snpsqc_dis_no-x_updated_5.batches_mis_0.csv
    Untracked:  data/ici.vs.eoi_age.of.onset-no.covariates_blup_sub_chr-15_peak.marker-UNC26070435_lod.drop-1.5_snpsqc_dis_no-x_updated_5.batches_mis_52.csv
    Untracked:  data/ici.vs.eoi_age.of.onset-no.covariates_blup_sub_chr-17_peak.marker-UNCHS044241_lod.drop-1.5_snpsqc_5.batches_mis_0.csv
    Untracked:  data/ici.vs.eoi_age.of.onset-no.covariates_blup_sub_chr-17_peak.marker-UNCHS044241_lod.drop-1.5_snpsqc_5.batches_mis_52.csv
    Untracked:  data/ici.vs.eoi_age.of.onset-no.covariates_blup_sub_chr-17_peak.marker-UNCJPD006614_lod.drop-1.5_snpsqc_dis_no-x_updated_5.batches_mis_0.csv
    Untracked:  data/ici.vs.eoi_age.of.onset-no.covariates_blup_sub_chr-17_peak.marker-UNCrs47191360_lod.drop-1.5_snpsqc_dis_no-x_updated_5.batches_mis_52.csv
    Untracked:  data/ici.vs.eoi_age.of.onset-no.covariates_blup_sub_chr-18_peak.marker-UNCHS045343_lod.drop-1.5_snpsqc_5.batches_mis_0.csv
    Untracked:  data/ici.vs.eoi_age.of.onset-no.covariates_blup_sub_chr-18_peak.marker-UNCHS045343_lod.drop-1.5_snpsqc_5.batches_mis_52.csv
    Untracked:  data/ici.vs.eoi_age.of.onset-no.covariates_blup_sub_chr-2_peak.marker-UNC4609527_lod.drop-1.5_snpsqc_dis_no-x_updated_5.batches_mis_0.csv
    Untracked:  data/ici.vs.eoi_age.of.onset-no.covariates_blup_sub_chr-2_peak.marker-UNC4609527_lod.drop-1.5_snpsqc_dis_no-x_updated_5.batches_mis_52.csv
    Untracked:  data/ici.vs.eoi_age.of.onset-no.covariates_blup_sub_chr-2_peak.marker-UNCHS008007_lod.drop-1.5_5.batches_mis.csv
    Untracked:  data/ici.vs.eoi_age.of.onset-no.covariates_blup_sub_chr-2_peak.marker-UNCHS008007_lod.drop-1.5_5.batches_mis_0.csv
    Untracked:  data/ici.vs.eoi_age.of.onset-no.covariates_blup_sub_chr-3_peak.marker-ICR1338_lod.drop-1.5_5.batches_0.csv
    Untracked:  data/ici.vs.eoi_age.of.onset-no.covariates_blup_sub_chr-3_peak.marker-ICR1338_lod.drop-1.5_5.batches_52.csv
    Untracked:  data/ici.vs.eoi_age.of.onset-no.covariates_blup_sub_chr-3_peak.marker-ICR1338_lod.drop-1.5_5.batches_mis_0.csv
    Untracked:  data/ici.vs.eoi_age.of.onset-no.covariates_blup_sub_chr-3_peak.marker-ICR5263_lod.drop-1.5_5.batches_mis.csv
    Untracked:  data/ici.vs.eoi_age.of.onset-no.covariates_blup_sub_chr-3_peak.marker-ICR5263_lod.drop-1.5_5.batches_mis_0.csv
    Untracked:  data/ici.vs.eoi_age.of.onset-no.covariates_blup_sub_chr-3_peak.marker-JAX00105649_lod.drop-1.5_5.batches_mis.csv
    Untracked:  data/ici.vs.eoi_age.of.onset-no.covariates_blup_sub_chr-3_peak.marker-JAX00105649_lod.drop-1.5_5.batches_mis_0.csv
    Untracked:  data/ici.vs.eoi_age.of.onset-no.covariates_blup_sub_chr-3_peak.marker-JAX00106210_lod.drop-1.5_5.batches_mis_0.csv
    Untracked:  data/ici.vs.eoi_age.of.onset-no.covariates_blup_sub_chr-3_peak.marker-JAX00106957_lod.drop-1.5_5.batches_0.csv
    Untracked:  data/ici.vs.eoi_age.of.onset-no.covariates_blup_sub_chr-3_peak.marker-JAX00106957_lod.drop-1.5_5.batches_52.csv
    Untracked:  data/ici.vs.eoi_age.of.onset-no.covariates_blup_sub_chr-3_peak.marker-JAX00106957_lod.drop-1.5_5.batches_mis_0.csv
    Untracked:  data/ici.vs.eoi_age.of.onset-no.covariates_blup_sub_chr-3_peak.marker-JAX00107193_lod.drop-1.5_5.batches_mis_0.csv
    Untracked:  data/ici.vs.eoi_age.of.onset-no.covariates_blup_sub_chr-3_peak.marker-JAX00220631_lod.drop-1.5_5.batches_0.csv
    Untracked:  data/ici.vs.eoi_age.of.onset-no.covariates_blup_sub_chr-3_peak.marker-JAX00220631_lod.drop-1.5_5.batches_mis_0.csv
    Untracked:  data/ici.vs.eoi_age.of.onset-no.covariates_blup_sub_chr-3_peak.marker-JAX00514662_lod.drop-1.5_5.batches_0.csv
    Untracked:  data/ici.vs.eoi_age.of.onset-no.covariates_blup_sub_chr-3_peak.marker-JAX00514662_lod.drop-1.5_5.batches_52.csv
    Untracked:  data/ici.vs.eoi_age.of.onset-no.covariates_blup_sub_chr-3_peak.marker-JAX00516812_lod.drop-1.5_5.batches_mis.csv
    Untracked:  data/ici.vs.eoi_age.of.onset-no.covariates_blup_sub_chr-3_peak.marker-JAX00516812_lod.drop-1.5_5.batches_mis_0.csv
    Untracked:  data/ici.vs.eoi_age.of.onset-no.covariates_blup_sub_chr-3_peak.marker-JAX00517674_lod.drop-1.5_5.batches_mis.csv
    Untracked:  data/ici.vs.eoi_age.of.onset-no.covariates_blup_sub_chr-3_peak.marker-JAX00517674_lod.drop-1.5_5.batches_mis_0.csv
    Untracked:  data/ici.vs.eoi_age.of.onset-no.covariates_blup_sub_chr-3_peak.marker-JAX00518911r_lod.drop-1.5_5.batches_mis.csv
    Untracked:  data/ici.vs.eoi_age.of.onset-no.covariates_blup_sub_chr-3_peak.marker-JAX00518911r_lod.drop-1.5_5.batches_mis_0.csv
    Untracked:  data/ici.vs.eoi_age.of.onset-no.covariates_blup_sub_chr-3_peak.marker-JAX00522922_lod.drop-1.5_5.batches_mis_0.csv
    Untracked:  data/ici.vs.eoi_age.of.onset-no.covariates_blup_sub_chr-3_peak.marker-JAX00524205_lod.drop-1.5_5.batches_mis_0.csv
    Untracked:  data/ici.vs.eoi_age.of.onset-no.covariates_blup_sub_chr-3_peak.marker-JAX00529707r_lod.drop-1.5_5.batches_0.csv
    Untracked:  data/ici.vs.eoi_age.of.onset-no.covariates_blup_sub_chr-3_peak.marker-JAX00529707r_lod.drop-1.5_5.batches_52.csv
    Untracked:  data/ici.vs.eoi_age.of.onset-no.covariates_blup_sub_chr-3_peak.marker-JAX00535764_lod.drop-1.5_5.batches_52.csv
    Untracked:  data/ici.vs.eoi_age.of.onset-no.covariates_blup_sub_chr-3_peak.marker-UNC4809586_lod.drop-1.5_5.batches_mis.csv
    Untracked:  data/ici.vs.eoi_age.of.onset-no.covariates_blup_sub_chr-3_peak.marker-UNC4809586_lod.drop-1.5_5.batches_mis_0.csv
    Untracked:  data/ici.vs.eoi_age.of.onset-no.covariates_blup_sub_chr-3_peak.marker-UNC4813186_lod.drop-1.5_5.batches_0.csv
    Untracked:  data/ici.vs.eoi_age.of.onset-no.covariates_blup_sub_chr-3_peak.marker-UNC4813186_lod.drop-1.5_5.batches_52.csv
    Untracked:  data/ici.vs.eoi_age.of.onset-no.covariates_blup_sub_chr-3_peak.marker-UNC4813186_lod.drop-1.5_5.batches_mis.csv
    Untracked:  data/ici.vs.eoi_age.of.onset-no.covariates_blup_sub_chr-3_peak.marker-UNC4813186_lod.drop-1.5_5.batches_mis_0.csv
    Untracked:  data/ici.vs.eoi_age.of.onset-no.covariates_blup_sub_chr-3_peak.marker-UNC4820379_lod.drop-1.5_5.batches_mis.csv
    Untracked:  data/ici.vs.eoi_age.of.onset-no.covariates_blup_sub_chr-3_peak.marker-UNC4820379_lod.drop-1.5_5.batches_mis_0.csv
    Untracked:  data/ici.vs.eoi_age.of.onset-no.covariates_blup_sub_chr-3_peak.marker-UNC4836550_lod.drop-1.5_5.batches_0.csv
    Untracked:  data/ici.vs.eoi_age.of.onset-no.covariates_blup_sub_chr-3_peak.marker-UNC4836550_lod.drop-1.5_5.batches_52.csv
    Untracked:  data/ici.vs.eoi_age.of.onset-no.covariates_blup_sub_chr-3_peak.marker-UNC4836550_lod.drop-1.5_5.batches_mis_0.csv
    Untracked:  data/ici.vs.eoi_age.of.onset-no.covariates_blup_sub_chr-3_peak.marker-UNC4875653_lod.drop-1.5_5.batches_mis.csv
    Untracked:  data/ici.vs.eoi_age.of.onset-no.covariates_blup_sub_chr-3_peak.marker-UNC4875653_lod.drop-1.5_5.batches_mis_0.csv
    Untracked:  data/ici.vs.eoi_age.of.onset-no.covariates_blup_sub_chr-3_peak.marker-UNC4901083_lod.drop-1.5_5.batches_mis.csv
    Untracked:  data/ici.vs.eoi_age.of.onset-no.covariates_blup_sub_chr-3_peak.marker-UNC4901083_lod.drop-1.5_5.batches_mis_0.csv
    Untracked:  data/ici.vs.eoi_age.of.onset-no.covariates_blup_sub_chr-3_peak.marker-UNC4909519_lod.drop-1.5_5.batches_mis.csv
    Untracked:  data/ici.vs.eoi_age.of.onset-no.covariates_blup_sub_chr-3_peak.marker-UNC4909519_lod.drop-1.5_5.batches_mis_0.csv
    Untracked:  data/ici.vs.eoi_age.of.onset-no.covariates_blup_sub_chr-3_peak.marker-UNC4912039_lod.drop-1.5_5.batches_0.csv
    Untracked:  data/ici.vs.eoi_age.of.onset-no.covariates_blup_sub_chr-3_peak.marker-UNC4912039_lod.drop-1.5_5.batches_mis_0.csv
    Untracked:  data/ici.vs.eoi_age.of.onset-no.covariates_blup_sub_chr-3_peak.marker-UNC4912288_lod.drop-1.5_5.batches_mis.csv
    Untracked:  data/ici.vs.eoi_age.of.onset-no.covariates_blup_sub_chr-3_peak.marker-UNC4912288_lod.drop-1.5_5.batches_mis_0.csv
    Untracked:  data/ici.vs.eoi_age.of.onset-no.covariates_blup_sub_chr-3_peak.marker-UNC4915446_lod.drop-1.5_5.batches_mis.csv
    Untracked:  data/ici.vs.eoi_age.of.onset-no.covariates_blup_sub_chr-3_peak.marker-UNC4915446_lod.drop-1.5_5.batches_mis_0.csv
    Untracked:  data/ici.vs.eoi_age.of.onset-no.covariates_blup_sub_chr-3_peak.marker-UNC4923398_lod.drop-1.5_5.batches_mis.csv
    Untracked:  data/ici.vs.eoi_age.of.onset-no.covariates_blup_sub_chr-3_peak.marker-UNC4923398_lod.drop-1.5_5.batches_mis_0.csv
    Untracked:  data/ici.vs.eoi_age.of.onset-no.covariates_blup_sub_chr-3_peak.marker-UNC4924377_lod.drop-1.5_5.batches_0.csv
    Untracked:  data/ici.vs.eoi_age.of.onset-no.covariates_blup_sub_chr-3_peak.marker-UNC4924377_lod.drop-1.5_5.batches_52.csv
    Untracked:  data/ici.vs.eoi_age.of.onset-no.covariates_blup_sub_chr-3_peak.marker-UNC4924377_lod.drop-1.5_5.batches_mis_0.csv
    Untracked:  data/ici.vs.eoi_age.of.onset-no.covariates_blup_sub_chr-3_peak.marker-UNC4938565_lod.drop-1.5_5.batches_mis.csv
    Untracked:  data/ici.vs.eoi_age.of.onset-no.covariates_blup_sub_chr-3_peak.marker-UNC4938565_lod.drop-1.5_5.batches_mis_0.csv
    Untracked:  data/ici.vs.eoi_age.of.onset-no.covariates_blup_sub_chr-3_peak.marker-UNC4945186_lod.drop-1.5_5.batches_mis.csv
    Untracked:  data/ici.vs.eoi_age.of.onset-no.covariates_blup_sub_chr-3_peak.marker-UNC4945186_lod.drop-1.5_5.batches_mis_0.csv
    Untracked:  data/ici.vs.eoi_age.of.onset-no.covariates_blup_sub_chr-3_peak.marker-UNC4947492_lod.drop-1.5_5.batches_mis.csv
    Untracked:  data/ici.vs.eoi_age.of.onset-no.covariates_blup_sub_chr-3_peak.marker-UNC4947492_lod.drop-1.5_5.batches_mis_0.csv
    Untracked:  data/ici.vs.eoi_age.of.onset-no.covariates_blup_sub_chr-3_peak.marker-UNC4956618_lod.drop-1.5_5.batches_0.csv
    Untracked:  data/ici.vs.eoi_age.of.onset-no.covariates_blup_sub_chr-3_peak.marker-UNC4956618_lod.drop-1.5_5.batches_mis_0.csv
    Untracked:  data/ici.vs.eoi_age.of.onset-no.covariates_blup_sub_chr-3_peak.marker-UNC4957168_lod.drop-1.5_5.batches_52.csv
    Untracked:  data/ici.vs.eoi_age.of.onset-no.covariates_blup_sub_chr-3_peak.marker-UNC4957859_lod.drop-1.5_5.batches_mis.csv
    Untracked:  data/ici.vs.eoi_age.of.onset-no.covariates_blup_sub_chr-3_peak.marker-UNC4957859_lod.drop-1.5_5.batches_mis_0.csv
    Untracked:  data/ici.vs.eoi_age.of.onset-no.covariates_blup_sub_chr-3_peak.marker-UNC4959266_lod.drop-1.5_5.batches_mis.csv
    Untracked:  data/ici.vs.eoi_age.of.onset-no.covariates_blup_sub_chr-3_peak.marker-UNC4959266_lod.drop-1.5_5.batches_mis_0.csv
    Untracked:  data/ici.vs.eoi_age.of.onset-no.covariates_blup_sub_chr-3_peak.marker-UNC4960612_lod.drop-1.5_5.batches_mis.csv
    Untracked:  data/ici.vs.eoi_age.of.onset-no.covariates_blup_sub_chr-3_peak.marker-UNC4960612_lod.drop-1.5_5.batches_mis_0.csv
    Untracked:  data/ici.vs.eoi_age.of.onset-no.covariates_blup_sub_chr-3_peak.marker-UNC4961542_lod.drop-1.5_5.batches_mis.csv
    Untracked:  data/ici.vs.eoi_age.of.onset-no.covariates_blup_sub_chr-3_peak.marker-UNC4961542_lod.drop-1.5_5.batches_mis_0.csv
    Untracked:  data/ici.vs.eoi_age.of.onset-no.covariates_blup_sub_chr-3_peak.marker-UNC4979914_lod.drop-1.5_5.batches_mis.csv
    Untracked:  data/ici.vs.eoi_age.of.onset-no.covariates_blup_sub_chr-3_peak.marker-UNC4979914_lod.drop-1.5_5.batches_mis_0.csv
    Untracked:  data/ici.vs.eoi_age.of.onset-no.covariates_blup_sub_chr-3_peak.marker-UNC4990111_lod.drop-1.5_5.batches_mis.csv
    Untracked:  data/ici.vs.eoi_age.of.onset-no.covariates_blup_sub_chr-3_peak.marker-UNC4990111_lod.drop-1.5_5.batches_mis_0.csv
    Untracked:  data/ici.vs.eoi_age.of.onset-no.covariates_blup_sub_chr-3_peak.marker-UNC5008656_lod.drop-1.5_5.batches_0.csv
    Untracked:  data/ici.vs.eoi_age.of.onset-no.covariates_blup_sub_chr-3_peak.marker-UNC5008656_lod.drop-1.5_5.batches_52.csv
    Untracked:  data/ici.vs.eoi_age.of.onset-no.covariates_blup_sub_chr-3_peak.marker-UNC5008656_lod.drop-1.5_5.batches_mis.csv
    Untracked:  data/ici.vs.eoi_age.of.onset-no.covariates_blup_sub_chr-3_peak.marker-UNC5008656_lod.drop-1.5_5.batches_mis_0.csv
    Untracked:  data/ici.vs.eoi_age.of.onset-no.covariates_blup_sub_chr-3_peak.marker-UNC5046545_lod.drop-1.5_5.batches_mis_0.csv
    Untracked:  data/ici.vs.eoi_age.of.onset-no.covariates_blup_sub_chr-3_peak.marker-UNC5055259_lod.drop-1.5_5.batches_mis_0.csv
    Untracked:  data/ici.vs.eoi_age.of.onset-no.covariates_blup_sub_chr-3_peak.marker-UNC5062524_lod.drop-1.5_5.batches_mis_0.csv
    Untracked:  data/ici.vs.eoi_age.of.onset-no.covariates_blup_sub_chr-3_peak.marker-UNC5068215_lod.drop-1.5_5.batches_mis_0.csv
    Untracked:  data/ici.vs.eoi_age.of.onset-no.covariates_blup_sub_chr-3_peak.marker-UNC5076935_lod.drop-1.5_5.batches_mis_0.csv
    Untracked:  data/ici.vs.eoi_age.of.onset-no.covariates_blup_sub_chr-3_peak.marker-UNC5082757_lod.drop-1.5_5.batches_mis_0.csv
    Untracked:  data/ici.vs.eoi_age.of.onset-no.covariates_blup_sub_chr-3_peak.marker-UNC5083168_lod.drop-1.5_5.batches_0.csv
    Untracked:  data/ici.vs.eoi_age.of.onset-no.covariates_blup_sub_chr-3_peak.marker-UNC5083168_lod.drop-1.5_5.batches_52.csv
    Untracked:  data/ici.vs.eoi_age.of.onset-no.covariates_blup_sub_chr-3_peak.marker-UNC5083168_lod.drop-1.5_5.batches_mis_0.csv
    Untracked:  data/ici.vs.eoi_age.of.onset-no.covariates_blup_sub_chr-3_peak.marker-UNC5173293_lod.drop-1.5_5.batches_mis_0.csv
    Untracked:  data/ici.vs.eoi_age.of.onset-no.covariates_blup_sub_chr-3_peak.marker-UNC5192582_lod.drop-1.5_5.batches_0.csv
    Untracked:  data/ici.vs.eoi_age.of.onset-no.covariates_blup_sub_chr-3_peak.marker-UNC5192582_lod.drop-1.5_5.batches_52.csv
    Untracked:  data/ici.vs.eoi_age.of.onset-no.covariates_blup_sub_chr-3_peak.marker-UNC5192582_lod.drop-1.5_5.batches_mis_0.csv
    Untracked:  data/ici.vs.eoi_age.of.onset-no.covariates_blup_sub_chr-3_peak.marker-UNC5193970_lod.drop-1.5_5.batches_mis_0.csv
    Untracked:  data/ici.vs.eoi_age.of.onset-no.covariates_blup_sub_chr-3_peak.marker-UNC5213888_lod.drop-1.5_snpsqc_5.batches_mis_0.csv
    Untracked:  data/ici.vs.eoi_age.of.onset-no.covariates_blup_sub_chr-3_peak.marker-UNC5213888_lod.drop-1.5_snpsqc_5.batches_mis_52.csv
    Untracked:  data/ici.vs.eoi_age.of.onset-no.covariates_blup_sub_chr-3_peak.marker-UNC5257268_lod.drop-1.5_5.batches_mis_0.csv
    Untracked:  data/ici.vs.eoi_age.of.onset-no.covariates_blup_sub_chr-3_peak.marker-UNC5262472_lod.drop-1.5_5.batches_0.csv
    Untracked:  data/ici.vs.eoi_age.of.onset-no.covariates_blup_sub_chr-3_peak.marker-UNC5262472_lod.drop-1.5_5.batches_52.csv
    Untracked:  data/ici.vs.eoi_age.of.onset-no.covariates_blup_sub_chr-3_peak.marker-UNC5262472_lod.drop-1.5_5.batches_mis_0.csv
    Untracked:  data/ici.vs.eoi_age.of.onset-no.covariates_blup_sub_chr-3_peak.marker-UNC5264696_lod.drop-1.5_5.batches_mis_0.csv
    Untracked:  data/ici.vs.eoi_age.of.onset-no.covariates_blup_sub_chr-3_peak.marker-UNC5267813_lod.drop-1.5_5.batches_mis_0.csv
    Untracked:  data/ici.vs.eoi_age.of.onset-no.covariates_blup_sub_chr-3_peak.marker-UNC5270225_lod.drop-1.5_5.batches_mis_0.csv
    Untracked:  data/ici.vs.eoi_age.of.onset-no.covariates_blup_sub_chr-3_peak.marker-UNC5277061_lod.drop-1.5_5.batches_mis_0.csv
    Untracked:  data/ici.vs.eoi_age.of.onset-no.covariates_blup_sub_chr-3_peak.marker-UNC5278229_lod.drop-1.5_5.batches_0.csv
    Untracked:  data/ici.vs.eoi_age.of.onset-no.covariates_blup_sub_chr-3_peak.marker-UNC5278229_lod.drop-1.5_5.batches_mis_0.csv
    Untracked:  data/ici.vs.eoi_age.of.onset-no.covariates_blup_sub_chr-3_peak.marker-UNC5282259_lod.drop-1.5_5.batches_mis_0.csv
    Untracked:  data/ici.vs.eoi_age.of.onset-no.covariates_blup_sub_chr-3_peak.marker-UNC5287977_lod.drop-1.5_5.batches_mis_0.csv
    Untracked:  data/ici.vs.eoi_age.of.onset-no.covariates_blup_sub_chr-3_peak.marker-UNC5301460_lod.drop-1.5_5.batches_mis_0.csv
    Untracked:  data/ici.vs.eoi_age.of.onset-no.covariates_blup_sub_chr-3_peak.marker-UNC5309136_lod.drop-1.5_5.batches_mis_0.csv
    Untracked:  data/ici.vs.eoi_age.of.onset-no.covariates_blup_sub_chr-3_peak.marker-UNC5321559_lod.drop-1.5_5.batches_mis_0.csv
    Untracked:  data/ici.vs.eoi_age.of.onset-no.covariates_blup_sub_chr-3_peak.marker-UNC5322455_lod.drop-1.5_5.batches_mis_0.csv
    Untracked:  data/ici.vs.eoi_age.of.onset-no.covariates_blup_sub_chr-3_peak.marker-UNC5329171_lod.drop-1.5_5.batches_mis_0.csv
    Untracked:  data/ici.vs.eoi_age.of.onset-no.covariates_blup_sub_chr-3_peak.marker-UNC5360055_lod.drop-1.5_5.batches_52.csv
    Untracked:  data/ici.vs.eoi_age.of.onset-no.covariates_blup_sub_chr-3_peak.marker-UNC5454839_lod.drop-1.5_5.batches_0.csv
    Untracked:  data/ici.vs.eoi_age.of.onset-no.covariates_blup_sub_chr-3_peak.marker-UNC5454839_lod.drop-1.5_5.batches_52.csv
    Untracked:  data/ici.vs.eoi_age.of.onset-no.covariates_blup_sub_chr-3_peak.marker-UNC5454839_lod.drop-1.5_5.batches_mis_0.csv
    Untracked:  data/ici.vs.eoi_age.of.onset-no.covariates_blup_sub_chr-3_peak.marker-UNC5645844_lod.drop-1.5_5.batches_0.csv
    Untracked:  data/ici.vs.eoi_age.of.onset-no.covariates_blup_sub_chr-3_peak.marker-UNC5645844_lod.drop-1.5_5.batches_52.csv
    Untracked:  data/ici.vs.eoi_age.of.onset-no.covariates_blup_sub_chr-3_peak.marker-UNC5645844_lod.drop-1.5_5.batches_mis_0.csv
    Untracked:  data/ici.vs.eoi_age.of.onset-no.covariates_blup_sub_chr-3_peak.marker-UNC5778977_lod.drop-1.5_5.batches_0.csv
    Untracked:  data/ici.vs.eoi_age.of.onset-no.covariates_blup_sub_chr-3_peak.marker-UNC5808246_lod.drop-1.5_5.batches_52.csv
    Untracked:  data/ici.vs.eoi_age.of.onset-no.covariates_blup_sub_chr-3_peak.marker-UNC5916997_lod.drop-1.5_5.batches_52.csv
    Untracked:  data/ici.vs.eoi_age.of.onset-no.covariates_blup_sub_chr-3_peak.marker-UNCHS008182_lod.drop-1.5_5.batches_0.csv
    Untracked:  data/ici.vs.eoi_age.of.onset-no.covariates_blup_sub_chr-3_peak.marker-UNCHS008182_lod.drop-1.5_5.batches_52.csv
    Untracked:  data/ici.vs.eoi_age.of.onset-no.covariates_blup_sub_chr-3_peak.marker-UNCHS008182_lod.drop-1.5_5.batches_mis_0.csv
    Untracked:  data/ici.vs.eoi_age.of.onset-no.covariates_blup_sub_chr-3_peak.marker-UNCHS008193_lod.drop-1.5_5.batches_mis.csv
    Untracked:  data/ici.vs.eoi_age.of.onset-no.covariates_blup_sub_chr-3_peak.marker-UNCHS008193_lod.drop-1.5_5.batches_mis_0.csv
    Untracked:  data/ici.vs.eoi_age.of.onset-no.covariates_blup_sub_chr-3_peak.marker-UNCHS008209_lod.drop-1.5_5.batches_mis.csv
    Untracked:  data/ici.vs.eoi_age.of.onset-no.covariates_blup_sub_chr-3_peak.marker-UNCHS008209_lod.drop-1.5_5.batches_mis_0.csv
    Untracked:  data/ici.vs.eoi_age.of.onset-no.covariates_blup_sub_chr-3_peak.marker-UNCHS008215_lod.drop-1.5_5.batches_mis.csv
    Untracked:  data/ici.vs.eoi_age.of.onset-no.covariates_blup_sub_chr-3_peak.marker-UNCHS008215_lod.drop-1.5_5.batches_mis_0.csv
    Untracked:  data/ici.vs.eoi_age.of.onset-no.covariates_blup_sub_chr-3_peak.marker-UNCHS008219_lod.drop-1.5_5.batches_mis.csv
    Untracked:  data/ici.vs.eoi_age.of.onset-no.covariates_blup_sub_chr-3_peak.marker-UNCHS008219_lod.drop-1.5_5.batches_mis_0.csv
    Untracked:  data/ici.vs.eoi_age.of.onset-no.covariates_blup_sub_chr-3_peak.marker-UNCHS008229_lod.drop-1.5_5.batches_mis.csv
    Untracked:  data/ici.vs.eoi_age.of.onset-no.covariates_blup_sub_chr-3_peak.marker-UNCHS008229_lod.drop-1.5_5.batches_mis_0.csv
    Untracked:  data/ici.vs.eoi_age.of.onset-no.covariates_blup_sub_chr-3_peak.marker-UNCHS008234_lod.drop-1.5_5.batches_mis.csv
    Untracked:  data/ici.vs.eoi_age.of.onset-no.covariates_blup_sub_chr-3_peak.marker-UNCHS008234_lod.drop-1.5_5.batches_mis_0.csv
    Untracked:  data/ici.vs.eoi_age.of.onset-no.covariates_blup_sub_chr-3_peak.marker-UNCHS008242_lod.drop-1.5_5.batches_52.csv
    Untracked:  data/ici.vs.eoi_age.of.onset-no.covariates_blup_sub_chr-3_peak.marker-UNCHS008260_lod.drop-1.5_5.batches_mis.csv
    Untracked:  data/ici.vs.eoi_age.of.onset-no.covariates_blup_sub_chr-3_peak.marker-UNCHS008260_lod.drop-1.5_5.batches_mis_0.csv
    Untracked:  data/ici.vs.eoi_age.of.onset-no.covariates_blup_sub_chr-3_peak.marker-UNCHS008271_lod.drop-1.5_5.batches_mis.csv
    Untracked:  data/ici.vs.eoi_age.of.onset-no.covariates_blup_sub_chr-3_peak.marker-UNCHS008271_lod.drop-1.5_5.batches_mis_0.csv
    Untracked:  data/ici.vs.eoi_age.of.onset-no.covariates_blup_sub_chr-3_peak.marker-UNCHS008281_lod.drop-1.5_5.batches_mis.csv
    Untracked:  data/ici.vs.eoi_age.of.onset-no.covariates_blup_sub_chr-3_peak.marker-UNCHS008281_lod.drop-1.5_5.batches_mis_0.csv
    Untracked:  data/ici.vs.eoi_age.of.onset-no.covariates_blup_sub_chr-3_peak.marker-UNCHS008286_lod.drop-1.5_5.batches_0.csv
    Untracked:  data/ici.vs.eoi_age.of.onset-no.covariates_blup_sub_chr-3_peak.marker-UNCHS008286_lod.drop-1.5_5.batches_52.csv
    Untracked:  data/ici.vs.eoi_age.of.onset-no.covariates_blup_sub_chr-3_peak.marker-UNCHS008286_lod.drop-1.5_5.batches_mis_0.csv
    Untracked:  data/ici.vs.eoi_age.of.onset-no.covariates_blup_sub_chr-3_peak.marker-UNCHS008338_lod.drop-1.5_5.batches_mis.csv
    Untracked:  data/ici.vs.eoi_age.of.onset-no.covariates_blup_sub_chr-3_peak.marker-UNCHS008338_lod.drop-1.5_5.batches_mis_0.csv
    Untracked:  data/ici.vs.eoi_age.of.onset-no.covariates_blup_sub_chr-3_peak.marker-UNCHS008363_lod.drop-1.5_5.batches_mis.csv
    Untracked:  data/ici.vs.eoi_age.of.onset-no.covariates_blup_sub_chr-3_peak.marker-UNCHS008363_lod.drop-1.5_5.batches_mis_0.csv
    Untracked:  data/ici.vs.eoi_age.of.onset-no.covariates_blup_sub_chr-3_peak.marker-UNCHS008379_lod.drop-1.5_5.batches_mis.csv
    Untracked:  data/ici.vs.eoi_age.of.onset-no.covariates_blup_sub_chr-3_peak.marker-UNCHS008379_lod.drop-1.5_5.batches_mis_0.csv
    Untracked:  data/ici.vs.eoi_age.of.onset-no.covariates_blup_sub_chr-3_peak.marker-UNCHS008409_lod.drop-1.5_5.batches_mis_0.csv
    Untracked:  data/ici.vs.eoi_age.of.onset-no.covariates_blup_sub_chr-3_peak.marker-UNCHS008432_lod.drop-1.5_5.batches_mis_0.csv
    Untracked:  data/ici.vs.eoi_age.of.onset-no.covariates_blup_sub_chr-3_peak.marker-UNCHS008487_lod.drop-1.5_snpsqc_5.batches_0.csv
    Untracked:  data/ici.vs.eoi_age.of.onset-no.covariates_blup_sub_chr-3_peak.marker-UNCHS008487_lod.drop-1.5_snpsqc_5.batches_52.csv
    Untracked:  data/ici.vs.eoi_age.of.onset-no.covariates_blup_sub_chr-3_peak.marker-UNCHS008487_lod.drop-1.5_snpsqc_dis_no-x_updated_5.batches_0.csv
    Untracked:  data/ici.vs.eoi_age.of.onset-no.covariates_blup_sub_chr-3_peak.marker-UNCHS008487_lod.drop-1.5_snpsqc_dis_no-x_updated_5.batches_52.csv
    Untracked:  data/ici.vs.eoi_age.of.onset-no.covariates_blup_sub_chr-3_peak.marker-UNCHS008487_lod.drop-1.5_snpsqc_dis_no-x_updated_5.batches_mis_0.csv
    Untracked:  data/ici.vs.eoi_age.of.onset-no.covariates_blup_sub_chr-3_peak.marker-UNCHS008487_lod.drop-1.5_snpsqc_dis_no-x_updated_5.batches_mis_52.csv
    Untracked:  data/ici.vs.eoi_age.of.onset-no.covariates_blup_sub_chr-3_peak.marker-UNCHS008511_lod.drop-1.5_snpsqc_5.batches_mis_0.csv
    Untracked:  data/ici.vs.eoi_age.of.onset-no.covariates_blup_sub_chr-3_peak.marker-UNCHS008511_lod.drop-1.5_snpsqc_5.batches_mis_52.csv
    Untracked:  data/ici.vs.eoi_age.of.onset-no.covariates_blup_sub_chr-3_peak.marker-UNCHS008609_lod.drop-1.5_5.batches_mis_0.csv
    Untracked:  data/ici.vs.eoi_age.of.onset-no.covariates_blup_sub_chr-3_peak.marker-UNCHS008613_lod.drop-1.5_5.batches_0.csv
    Untracked:  data/ici.vs.eoi_age.of.onset-no.covariates_blup_sub_chr-3_peak.marker-UNCHS008613_lod.drop-1.5_5.batches_52.csv
    Untracked:  data/ici.vs.eoi_age.of.onset-no.covariates_blup_sub_chr-3_peak.marker-UNCHS008614_lod.drop-1.5_5.batches_mis_0.csv
    Untracked:  data/ici.vs.eoi_age.of.onset-no.covariates_blup_sub_chr-3_peak.marker-UNCHS008627_lod.drop-1.5_5.batches_mis_0.csv
    Untracked:  data/ici.vs.eoi_age.of.onset-no.covariates_blup_sub_chr-3_peak.marker-UNCHS008659_lod.drop-1.5_5.batches_52.csv
    Untracked:  data/ici.vs.eoi_age.of.onset-no.covariates_blup_sub_chr-3_peak.marker-UNCHS008725_lod.drop-1.5_5.batches_mis_0.csv
    Untracked:  data/ici.vs.eoi_age.of.onset-no.covariates_blup_sub_chr-3_peak.marker-UNCHS008815_lod.drop-1.5_5.batches_0.csv
    Untracked:  data/ici.vs.eoi_age.of.onset-no.covariates_blup_sub_chr-3_peak.marker-UNCHS008815_lod.drop-1.5_5.batches_52.csv
    Untracked:  data/ici.vs.eoi_age.of.onset-no.covariates_blup_sub_chr-3_peak.marker-UNCHS008815_lod.drop-1.5_5.batches_mis_0.csv
    Untracked:  data/ici.vs.eoi_age.of.onset-no.covariates_blup_sub_chr-3_peak.marker-UNCHS009066_lod.drop-1.5_5.batches_52.csv
    Untracked:  data/ici.vs.eoi_age.of.onset-no.covariates_blup_sub_chr-3_peak.marker-UNCHS009067_lod.drop-1.5_5.batches_0.csv
    Untracked:  data/ici.vs.eoi_age.of.onset-no.covariates_blup_sub_chr-3_peak.marker-UNCHS009068_lod.drop-1.5_5.batches_mis_0.csv
    Untracked:  data/ici.vs.eoi_age.of.onset-no.covariates_blup_sub_chr-3_peak.marker-UNCHS009112_lod.drop-1.5_5.batches_52.csv
    Untracked:  data/ici.vs.eoi_age.of.onset-no.covariates_blup_sub_chr-3_peak.marker-UNCHS009224_lod.drop-1.5_5.batches_0.csv
    Untracked:  data/ici.vs.eoi_age.of.onset-no.covariates_blup_sub_chr-3_peak.marker-UNCHS009224_lod.drop-1.5_5.batches_52.csv
    Untracked:  data/ici.vs.eoi_age.of.onset-no.covariates_blup_sub_chr-3_peak.marker-UNCHS009383_lod.drop-1.5_5.batches_0.csv
    Untracked:  data/ici.vs.eoi_age.of.onset-no.covariates_blup_sub_chr-3_peak.marker-UNCHS009383_lod.drop-1.5_5.batches_52.csv
    Untracked:  data/ici.vs.eoi_age.of.onset-no.covariates_blup_sub_chr-3_peak.marker-UNCHS009693_lod.drop-1.5_5.batches_52.csv
    Untracked:  data/ici.vs.eoi_age.of.onset-no.covariates_blup_sub_chr-3_peak.marker-UNCHS009821_lod.drop-1.5_5.batches_52.csv
    Untracked:  data/ici.vs.eoi_age.of.onset-no.covariates_blup_sub_chr-3_peak.marker-UNCJPD001170_lod.drop-1.5_5.batches_mis.csv
    Untracked:  data/ici.vs.eoi_age.of.onset-no.covariates_blup_sub_chr-3_peak.marker-UNCJPD001170_lod.drop-1.5_5.batches_mis_0.csv
    Untracked:  data/ici.vs.eoi_age.of.onset-no.covariates_blup_sub_chr-3_peak.marker-UNCJPD001198_lod.drop-1.5_5.batches_mis.csv
    Untracked:  data/ici.vs.eoi_age.of.onset-no.covariates_blup_sub_chr-3_peak.marker-UNCJPD001198_lod.drop-1.5_5.batches_mis_0.csv
    Untracked:  data/ici.vs.eoi_age.of.onset-no.covariates_blup_sub_chr-3_peak.marker-UNCJPD001276_lod.drop-1.5_5.batches_mis_0.csv
    Untracked:  data/ici.vs.eoi_age.of.onset-no.covariates_blup_sub_chr-3_peak.marker-sanger2496q_lod.drop-1.5_5.batches_mis.csv
    Untracked:  data/ici.vs.eoi_age.of.onset-no.covariates_blup_sub_chr-3_peak.marker-sanger2496q_lod.drop-1.5_5.batches_mis_0.csv
    Untracked:  data/ici.vs.eoi_age.of.onset-no.covariates_blup_sub_chr-4_peak.marker-UNC8250659_lod.drop-1.5_snpsqc_dis_no-x_updated_5.batches_52.csv
    Untracked:  data/ici.vs.eoi_age.of.onset-no.covariates_blup_sub_chr-4_peak.marker-UNC8439633_lod.drop-1.5_snpsqc_5.batches_52.csv
    Untracked:  data/ici.vs.eoi_age.of.onset-no.covariates_blup_sub_chr-4_peak.marker-UNC8439633_lod.drop-1.5_snpsqc_5.batches_mis_52.csv
    Untracked:  data/ici.vs.eoi_age.of.onset-no.covariates_blup_sub_chr-4_peak.marker-UNCHS012955_lod.drop-1.5_snpsqc_dis_no-x_updated_5.batches_mis_52.csv
    Untracked:  data/ici.vs.eoi_age.of.onset-no.covariates_blup_sub_chr-6_peak.marker-UNC11108920_lod.drop-1.5_snpsqc_5.batches.csv
    Untracked:  data/ici.vs.eoi_age.of.onset-no.covariates_blup_sub_chr-6_peak.marker-UNC11108920_lod.drop-1.5_snpsqc_5.batches_mis.csv
    Untracked:  data/ici.vs.eoi_age.of.onset-no.covariates_blup_sub_chr-7_peak.marker-UNCHS020066_lod.drop-1.5_snpsqc_5.batches_mis_0.csv
    Untracked:  data/ici.vs.eoi_age.of.onset-no.covariates_blup_sub_chr-7_peak.marker-UNCHS020066_lod.drop-1.5_snpsqc_5.batches_mis_52.csv
    Untracked:  data/ici.vs.eoi_age.of.onset-no.covariates_blup_sub_chr-7_peak.marker-UNCHS020066_lod.drop-1.5_snpsqc_dis_no-x_updated_5.batches_mis_0.csv
    Untracked:  data/ici.vs.eoi_age.of.onset-no.covariates_blup_sub_chr-7_peak.marker-UNCHS020066_lod.drop-1.5_snpsqc_dis_no-x_updated_5.batches_mis_52.csv
    Untracked:  data/ici.vs.eoi_age.of.onset-no.covariates_blup_sub_chr-8_peak.marker-UNC15524531_lod.drop-1.5_snpsqc_5.batches_mis_0.csv
    Untracked:  data/ici.vs.eoi_age.of.onset-no.covariates_blup_sub_chr-8_peak.marker-UNC15524531_lod.drop-1.5_snpsqc_5.batches_mis_52.csv
    Untracked:  data/ici.vs.eoi_age.of.onset-no.covariates_blup_sub_chr-8_peak.marker-UNC15524531_lod.drop-1.5_snpsqc_dis_no-x_updated_5.batches_mis_0.csv
    Untracked:  data/ici.vs.eoi_age.of.onset-no.covariates_blup_sub_chr-8_peak.marker-UNC15524531_lod.drop-1.5_snpsqc_dis_no-x_updated_5.batches_mis_52.csv
    Untracked:  data/ici.vs.eoi_age.of.onset-no.covariates_blup_sub_chr-9_peak.marker-UNC17203597_lod.drop-1.5_snpsqc_5.batches_mis_0.csv
    Untracked:  data/ici.vs.eoi_age.of.onset-no.covariates_blup_sub_chr-9_peak.marker-UNC17203597_lod.drop-1.5_snpsqc_5.batches_mis_52.csv
    Untracked:  data/ici.vs.eoi_age.of.onset-no.covariates_blup_sub_chr-X_peak.marker-UNCHS048314_lod.drop-1.5_snpsqc_5.batches_mis_0.csv
    Untracked:  data/ici.vs.eoi_age.of.onset-no.covariates_blup_sub_chr-X_peak.marker-UNCHS048314_lod.drop-1.5_snpsqc_5.batches_mis_52.csv
    Untracked:  data/ici.vs.eoi_age.of.onset-no.covariates_blup_sub_chr-X_peak.marker-UNCHS048314_lod.drop-1.5_snpsqc_dis_no-x_updated_5.batches_mis_0.csv
    Untracked:  data/ici.vs.eoi_age.of.onset-no.covariates_blup_sub_chr-X_peak.marker-UNCHS048314_lod.drop-1.5_snpsqc_dis_no-x_updated_5.batches_mis_52.csv
    Untracked:  data/ici.vs.eoi_age.of.onset-no.covariates_genes_chr-10_peak.marker-JAX00020646_lod.drop-1.5_5.batches_0.csv
    Untracked:  data/ici.vs.eoi_age.of.onset-no.covariates_genes_chr-10_peak.marker-JAX00292499_lod.drop-1.5_5.batches_52.csv
    Untracked:  data/ici.vs.eoi_age.of.onset-no.covariates_genes_chr-10_peak.marker-JAX00292927_lod.drop-1.5_5.batches_0.csv
    Untracked:  data/ici.vs.eoi_age.of.onset-no.covariates_genes_chr-10_peak.marker-JAX00292927_lod.drop-1.5_5.batches_52.csv
    Untracked:  data/ici.vs.eoi_age.of.onset-no.covariates_genes_chr-10_peak.marker-JAX00294019_lod.drop-1.5_snpsqc_dis_no-x_updated_5.batches_0.csv
    Untracked:  data/ici.vs.eoi_age.of.onset-no.covariates_genes_chr-10_peak.marker-UNC18216614_lod.drop-1.5_5.batches_52.csv
    Untracked:  data/ici.vs.eoi_age.of.onset-no.covariates_genes_chr-10_peak.marker-UNC18240977_lod.drop-1.5_snpsqc_dis_no-x_updated_5.batches_52.csv
    Untracked:  data/ici.vs.eoi_age.of.onset-no.covariates_genes_chr-10_peak.marker-UNC18311938_lod.drop-1.5_5.batches_0.csv
    Untracked:  data/ici.vs.eoi_age.of.onset-no.covariates_genes_chr-10_peak.marker-UNC18311938_lod.drop-1.5_5.batches_52.csv
    Untracked:  data/ici.vs.eoi_age.of.onset-no.covariates_genes_chr-10_peak.marker-UNC18311938_lod.drop-1.5_snpsqc_5.batches_52.csv
    Untracked:  data/ici.vs.eoi_age.of.onset-no.covariates_genes_chr-10_peak.marker-UNC18311938_lod.drop-1.5_snpsqc_5.batches_mis_52.csv
    Untracked:  data/ici.vs.eoi_age.of.onset-no.covariates_genes_chr-10_peak.marker-UNC18343181_lod.drop-1.5_5.batches_0.csv
    Untracked:  data/ici.vs.eoi_age.of.onset-no.covariates_genes_chr-10_peak.marker-UNC18363544_lod.drop-1.5_5.batches_0.csv
    Untracked:  data/ici.vs.eoi_age.of.onset-no.covariates_genes_chr-10_peak.marker-UNC18363544_lod.drop-1.5_5.batches_52.csv
    Untracked:  data/ici.vs.eoi_age.of.onset-no.covariates_genes_chr-10_peak.marker-UNC18363544_lod.drop-1.5_snpsqc_5.batches_mis_0.csv
    Untracked:  data/ici.vs.eoi_age.of.onset-no.covariates_genes_chr-10_peak.marker-UNC18376338_lod.drop-1.5_snpsqc_5.batches_0.csv
    Untracked:  data/ici.vs.eoi_age.of.onset-no.covariates_genes_chr-10_peak.marker-UNCHS028236_lod.drop-1.5_snpsqc_5.batches_mis_0.csv
    Untracked:  data/ici.vs.eoi_age.of.onset-no.covariates_genes_chr-10_peak.marker-UNCHS028236_lod.drop-1.5_snpsqc_5.batches_mis_52.csv
    Untracked:  data/ici.vs.eoi_age.of.onset-no.covariates_genes_chr-10_peak.marker-UNCHS028536_lod.drop-1.5_snpsqc_dis_no-x_updated_5.batches_mis_0.csv
    Untracked:  data/ici.vs.eoi_age.of.onset-no.covariates_genes_chr-10_peak.marker-UNCHS028536_lod.drop-1.5_snpsqc_dis_no-x_updated_5.batches_mis_52.csv
    Untracked:  data/ici.vs.eoi_age.of.onset-no.covariates_genes_chr-11_peak.marker-UNC19970181_lod.drop-1.5_snpsqc_5.batches_mis_0.csv
    Untracked:  data/ici.vs.eoi_age.of.onset-no.covariates_genes_chr-11_peak.marker-UNC19970181_lod.drop-1.5_snpsqc_5.batches_mis_52.csv
    Untracked:  data/ici.vs.eoi_age.of.onset-no.covariates_genes_chr-11_peak.marker-UNC20090524_lod.drop-1.5_snpsqc_5.batches_52.csv
    Untracked:  data/ici.vs.eoi_age.of.onset-no.covariates_genes_chr-12_peak.marker-ICR499_lod.drop-1.5_snpsqc_5.batches_mis_0.csv
    Untracked:  data/ici.vs.eoi_age.of.onset-no.covariates_genes_chr-12_peak.marker-ICR499_lod.drop-1.5_snpsqc_5.batches_mis_52.csv
    Untracked:  data/ici.vs.eoi_age.of.onset-no.covariates_genes_chr-12_peak.marker-ICR499_lod.drop-1.5_snpsqc_dis_no-x_updated_5.batches_mis_0.csv
    Untracked:  data/ici.vs.eoi_age.of.onset-no.covariates_genes_chr-13_peak.marker-UNCHS036773_lod.drop-1.5_snpsqc_5.batches_mis_0.csv
    Untracked:  data/ici.vs.eoi_age.of.onset-no.covariates_genes_chr-13_peak.marker-UNCHS036773_lod.drop-1.5_snpsqc_5.batches_mis_52.csv
    Untracked:  data/ici.vs.eoi_age.of.onset-no.covariates_genes_chr-14_peak.marker-UNC24056202_lod.drop-1.5_snpsqc_5.batches_mis_0.csv
    Untracked:  data/ici.vs.eoi_age.of.onset-no.covariates_genes_chr-14_peak.marker-UNC24056202_lod.drop-1.5_snpsqc_5.batches_mis_52.csv
    Untracked:  data/ici.vs.eoi_age.of.onset-no.covariates_genes_chr-15_peak.marker-UNC26070435_lod.drop-1.5_snpsqc_dis_no-x_updated_5.batches_mis_0.csv
    Untracked:  data/ici.vs.eoi_age.of.onset-no.covariates_genes_chr-15_peak.marker-UNC26070435_lod.drop-1.5_snpsqc_dis_no-x_updated_5.batches_mis_52.csv
    Untracked:  data/ici.vs.eoi_age.of.onset-no.covariates_genes_chr-17_peak.marker-UNCHS044241_lod.drop-1.5_snpsqc_5.batches_mis_0.csv
    Untracked:  data/ici.vs.eoi_age.of.onset-no.covariates_genes_chr-17_peak.marker-UNCHS044241_lod.drop-1.5_snpsqc_5.batches_mis_52.csv
    Untracked:  data/ici.vs.eoi_age.of.onset-no.covariates_genes_chr-17_peak.marker-UNCJPD006614_lod.drop-1.5_snpsqc_dis_no-x_updated_5.batches_mis_0.csv
    Untracked:  data/ici.vs.eoi_age.of.onset-no.covariates_genes_chr-17_peak.marker-UNCrs47191360_lod.drop-1.5_snpsqc_dis_no-x_updated_5.batches_mis_52.csv
    Untracked:  data/ici.vs.eoi_age.of.onset-no.covariates_genes_chr-18_peak.marker-UNCHS045343_lod.drop-1.5_snpsqc_5.batches_mis_0.csv
    Untracked:  data/ici.vs.eoi_age.of.onset-no.covariates_genes_chr-18_peak.marker-UNCHS045343_lod.drop-1.5_snpsqc_5.batches_mis_52.csv
    Untracked:  data/ici.vs.eoi_age.of.onset-no.covariates_genes_chr-2_peak.marker-UNC4609527_lod.drop-1.5_snpsqc_dis_no-x_updated_5.batches_mis_0.csv
    Untracked:  data/ici.vs.eoi_age.of.onset-no.covariates_genes_chr-2_peak.marker-UNC4609527_lod.drop-1.5_snpsqc_dis_no-x_updated_5.batches_mis_52.csv
    Untracked:  data/ici.vs.eoi_age.of.onset-no.covariates_genes_chr-2_peak.marker-UNCHS008007_lod.drop-1.5_5.batches_mis.csv
    Untracked:  data/ici.vs.eoi_age.of.onset-no.covariates_genes_chr-2_peak.marker-UNCHS008007_lod.drop-1.5_5.batches_mis_0.csv
    Untracked:  data/ici.vs.eoi_age.of.onset-no.covariates_genes_chr-3_peak.marker-ICR1338_lod.drop-1.5_5.batches_0.csv
    Untracked:  data/ici.vs.eoi_age.of.onset-no.covariates_genes_chr-3_peak.marker-ICR1338_lod.drop-1.5_5.batches_52.csv
    Untracked:  data/ici.vs.eoi_age.of.onset-no.covariates_genes_chr-3_peak.marker-ICR1338_lod.drop-1.5_5.batches_mis_0.csv
    Untracked:  data/ici.vs.eoi_age.of.onset-no.covariates_genes_chr-3_peak.marker-ICR5263_lod.drop-1.5_5.batches_mis.csv
    Untracked:  data/ici.vs.eoi_age.of.onset-no.covariates_genes_chr-3_peak.marker-ICR5263_lod.drop-1.5_5.batches_mis_0.csv
    Untracked:  data/ici.vs.eoi_age.of.onset-no.covariates_genes_chr-3_peak.marker-JAX00105649_lod.drop-1.5_5.batches_mis.csv
    Untracked:  data/ici.vs.eoi_age.of.onset-no.covariates_genes_chr-3_peak.marker-JAX00105649_lod.drop-1.5_5.batches_mis_0.csv
    Untracked:  data/ici.vs.eoi_age.of.onset-no.covariates_genes_chr-3_peak.marker-JAX00106210_lod.drop-1.5_5.batches_mis_0.csv
    Untracked:  data/ici.vs.eoi_age.of.onset-no.covariates_genes_chr-3_peak.marker-JAX00106957_lod.drop-1.5_5.batches_0.csv
    Untracked:  data/ici.vs.eoi_age.of.onset-no.covariates_genes_chr-3_peak.marker-JAX00106957_lod.drop-1.5_5.batches_52.csv
    Untracked:  data/ici.vs.eoi_age.of.onset-no.covariates_genes_chr-3_peak.marker-JAX00106957_lod.drop-1.5_5.batches_mis_0.csv
    Untracked:  data/ici.vs.eoi_age.of.onset-no.covariates_genes_chr-3_peak.marker-JAX00107193_lod.drop-1.5_5.batches_mis_0.csv
    Untracked:  data/ici.vs.eoi_age.of.onset-no.covariates_genes_chr-3_peak.marker-JAX00220631_lod.drop-1.5_5.batches_0.csv
    Untracked:  data/ici.vs.eoi_age.of.onset-no.covariates_genes_chr-3_peak.marker-JAX00220631_lod.drop-1.5_5.batches_mis_0.csv
    Untracked:  data/ici.vs.eoi_age.of.onset-no.covariates_genes_chr-3_peak.marker-JAX00514662_lod.drop-1.5_5.batches_0.csv
    Untracked:  data/ici.vs.eoi_age.of.onset-no.covariates_genes_chr-3_peak.marker-JAX00514662_lod.drop-1.5_5.batches_52.csv
    Untracked:  data/ici.vs.eoi_age.of.onset-no.covariates_genes_chr-3_peak.marker-JAX00516812_lod.drop-1.5_5.batches_mis.csv
    Untracked:  data/ici.vs.eoi_age.of.onset-no.covariates_genes_chr-3_peak.marker-JAX00516812_lod.drop-1.5_5.batches_mis_0.csv
    Untracked:  data/ici.vs.eoi_age.of.onset-no.covariates_genes_chr-3_peak.marker-JAX00517674_lod.drop-1.5_5.batches_mis.csv
    Untracked:  data/ici.vs.eoi_age.of.onset-no.covariates_genes_chr-3_peak.marker-JAX00517674_lod.drop-1.5_5.batches_mis_0.csv
    Untracked:  data/ici.vs.eoi_age.of.onset-no.covariates_genes_chr-3_peak.marker-JAX00518911r_lod.drop-1.5_5.batches_mis.csv
    Untracked:  data/ici.vs.eoi_age.of.onset-no.covariates_genes_chr-3_peak.marker-JAX00518911r_lod.drop-1.5_5.batches_mis_0.csv
    Untracked:  data/ici.vs.eoi_age.of.onset-no.covariates_genes_chr-3_peak.marker-JAX00522922_lod.drop-1.5_5.batches_mis_0.csv
    Untracked:  data/ici.vs.eoi_age.of.onset-no.covariates_genes_chr-3_peak.marker-JAX00524205_lod.drop-1.5_5.batches_mis_0.csv
    Untracked:  data/ici.vs.eoi_age.of.onset-no.covariates_genes_chr-3_peak.marker-JAX00529707r_lod.drop-1.5_5.batches_0.csv
    Untracked:  data/ici.vs.eoi_age.of.onset-no.covariates_genes_chr-3_peak.marker-JAX00529707r_lod.drop-1.5_5.batches_52.csv
    Untracked:  data/ici.vs.eoi_age.of.onset-no.covariates_genes_chr-3_peak.marker-JAX00535764_lod.drop-1.5_5.batches_52.csv
    Untracked:  data/ici.vs.eoi_age.of.onset-no.covariates_genes_chr-3_peak.marker-UNC4809586_lod.drop-1.5_5.batches_mis.csv
    Untracked:  data/ici.vs.eoi_age.of.onset-no.covariates_genes_chr-3_peak.marker-UNC4809586_lod.drop-1.5_5.batches_mis_0.csv
    Untracked:  data/ici.vs.eoi_age.of.onset-no.covariates_genes_chr-3_peak.marker-UNC4813186_lod.drop-1.5_5.batches_0.csv
    Untracked:  data/ici.vs.eoi_age.of.onset-no.covariates_genes_chr-3_peak.marker-UNC4813186_lod.drop-1.5_5.batches_52.csv
    Untracked:  data/ici.vs.eoi_age.of.onset-no.covariates_genes_chr-3_peak.marker-UNC4813186_lod.drop-1.5_5.batches_mis.csv
    Untracked:  data/ici.vs.eoi_age.of.onset-no.covariates_genes_chr-3_peak.marker-UNC4813186_lod.drop-1.5_5.batches_mis_0.csv
    Untracked:  data/ici.vs.eoi_age.of.onset-no.covariates_genes_chr-3_peak.marker-UNC4820379_lod.drop-1.5_5.batches_mis.csv
    Untracked:  data/ici.vs.eoi_age.of.onset-no.covariates_genes_chr-3_peak.marker-UNC4820379_lod.drop-1.5_5.batches_mis_0.csv
    Untracked:  data/ici.vs.eoi_age.of.onset-no.covariates_genes_chr-3_peak.marker-UNC4836550_lod.drop-1.5_5.batches_0.csv
    Untracked:  data/ici.vs.eoi_age.of.onset-no.covariates_genes_chr-3_peak.marker-UNC4836550_lod.drop-1.5_5.batches_52.csv
    Untracked:  data/ici.vs.eoi_age.of.onset-no.covariates_genes_chr-3_peak.marker-UNC4836550_lod.drop-1.5_5.batches_mis_0.csv
    Untracked:  data/ici.vs.eoi_age.of.onset-no.covariates_genes_chr-3_peak.marker-UNC4875653_lod.drop-1.5_5.batches_mis.csv
    Untracked:  data/ici.vs.eoi_age.of.onset-no.covariates_genes_chr-3_peak.marker-UNC4875653_lod.drop-1.5_5.batches_mis_0.csv
    Untracked:  data/ici.vs.eoi_age.of.onset-no.covariates_genes_chr-3_peak.marker-UNC4901083_lod.drop-1.5_5.batches_mis.csv
    Untracked:  data/ici.vs.eoi_age.of.onset-no.covariates_genes_chr-3_peak.marker-UNC4901083_lod.drop-1.5_5.batches_mis_0.csv
    Untracked:  data/ici.vs.eoi_age.of.onset-no.covariates_genes_chr-3_peak.marker-UNC4909519_lod.drop-1.5_5.batches_mis.csv
    Untracked:  data/ici.vs.eoi_age.of.onset-no.covariates_genes_chr-3_peak.marker-UNC4909519_lod.drop-1.5_5.batches_mis_0.csv
    Untracked:  data/ici.vs.eoi_age.of.onset-no.covariates_genes_chr-3_peak.marker-UNC4912039_lod.drop-1.5_5.batches_0.csv
    Untracked:  data/ici.vs.eoi_age.of.onset-no.covariates_genes_chr-3_peak.marker-UNC4912039_lod.drop-1.5_5.batches_mis_0.csv
    Untracked:  data/ici.vs.eoi_age.of.onset-no.covariates_genes_chr-3_peak.marker-UNC4912288_lod.drop-1.5_5.batches_mis.csv
    Untracked:  data/ici.vs.eoi_age.of.onset-no.covariates_genes_chr-3_peak.marker-UNC4912288_lod.drop-1.5_5.batches_mis_0.csv
    Untracked:  data/ici.vs.eoi_age.of.onset-no.covariates_genes_chr-3_peak.marker-UNC4915446_lod.drop-1.5_5.batches_mis.csv
    Untracked:  data/ici.vs.eoi_age.of.onset-no.covariates_genes_chr-3_peak.marker-UNC4915446_lod.drop-1.5_5.batches_mis_0.csv
    Untracked:  data/ici.vs.eoi_age.of.onset-no.covariates_genes_chr-3_peak.marker-UNC4923398_lod.drop-1.5_5.batches_mis.csv
    Untracked:  data/ici.vs.eoi_age.of.onset-no.covariates_genes_chr-3_peak.marker-UNC4923398_lod.drop-1.5_5.batches_mis_0.csv
    Untracked:  data/ici.vs.eoi_age.of.onset-no.covariates_genes_chr-3_peak.marker-UNC4924377_lod.drop-1.5_5.batches_0.csv
    Untracked:  data/ici.vs.eoi_age.of.onset-no.covariates_genes_chr-3_peak.marker-UNC4924377_lod.drop-1.5_5.batches_52.csv
    Untracked:  data/ici.vs.eoi_age.of.onset-no.covariates_genes_chr-3_peak.marker-UNC4924377_lod.drop-1.5_5.batches_mis_0.csv
    Untracked:  data/ici.vs.eoi_age.of.onset-no.covariates_genes_chr-3_peak.marker-UNC4938565_lod.drop-1.5_5.batches_mis.csv
    Untracked:  data/ici.vs.eoi_age.of.onset-no.covariates_genes_chr-3_peak.marker-UNC4938565_lod.drop-1.5_5.batches_mis_0.csv
    Untracked:  data/ici.vs.eoi_age.of.onset-no.covariates_genes_chr-3_peak.marker-UNC4945186_lod.drop-1.5_5.batches_mis.csv
    Untracked:  data/ici.vs.eoi_age.of.onset-no.covariates_genes_chr-3_peak.marker-UNC4945186_lod.drop-1.5_5.batches_mis_0.csv
    Untracked:  data/ici.vs.eoi_age.of.onset-no.covariates_genes_chr-3_peak.marker-UNC4947492_lod.drop-1.5_5.batches_mis.csv
    Untracked:  data/ici.vs.eoi_age.of.onset-no.covariates_genes_chr-3_peak.marker-UNC4947492_lod.drop-1.5_5.batches_mis_0.csv
    Untracked:  data/ici.vs.eoi_age.of.onset-no.covariates_genes_chr-3_peak.marker-UNC4956618_lod.drop-1.5_5.batches_0.csv
    Untracked:  data/ici.vs.eoi_age.of.onset-no.covariates_genes_chr-3_peak.marker-UNC4956618_lod.drop-1.5_5.batches_mis_0.csv
    Untracked:  data/ici.vs.eoi_age.of.onset-no.covariates_genes_chr-3_peak.marker-UNC4957168_lod.drop-1.5_5.batches_52.csv
    Untracked:  data/ici.vs.eoi_age.of.onset-no.covariates_genes_chr-3_peak.marker-UNC4957859_lod.drop-1.5_5.batches_mis.csv
    Untracked:  data/ici.vs.eoi_age.of.onset-no.covariates_genes_chr-3_peak.marker-UNC4957859_lod.drop-1.5_5.batches_mis_0.csv
    Untracked:  data/ici.vs.eoi_age.of.onset-no.covariates_genes_chr-3_peak.marker-UNC4959266_lod.drop-1.5_5.batches_mis.csv
    Untracked:  data/ici.vs.eoi_age.of.onset-no.covariates_genes_chr-3_peak.marker-UNC4959266_lod.drop-1.5_5.batches_mis_0.csv
    Untracked:  data/ici.vs.eoi_age.of.onset-no.covariates_genes_chr-3_peak.marker-UNC4960612_lod.drop-1.5_5.batches_mis.csv
    Untracked:  data/ici.vs.eoi_age.of.onset-no.covariates_genes_chr-3_peak.marker-UNC4960612_lod.drop-1.5_5.batches_mis_0.csv
    Untracked:  data/ici.vs.eoi_age.of.onset-no.covariates_genes_chr-3_peak.marker-UNC4961542_lod.drop-1.5_5.batches_mis.csv
    Untracked:  data/ici.vs.eoi_age.of.onset-no.covariates_genes_chr-3_peak.marker-UNC4961542_lod.drop-1.5_5.batches_mis_0.csv
    Untracked:  data/ici.vs.eoi_age.of.onset-no.covariates_genes_chr-3_peak.marker-UNC4979914_lod.drop-1.5_5.batches_mis.csv
    Untracked:  data/ici.vs.eoi_age.of.onset-no.covariates_genes_chr-3_peak.marker-UNC4979914_lod.drop-1.5_5.batches_mis_0.csv
    Untracked:  data/ici.vs.eoi_age.of.onset-no.covariates_genes_chr-3_peak.marker-UNC4990111_lod.drop-1.5_5.batches_mis.csv
    Untracked:  data/ici.vs.eoi_age.of.onset-no.covariates_genes_chr-3_peak.marker-UNC4990111_lod.drop-1.5_5.batches_mis_0.csv
    Untracked:  data/ici.vs.eoi_age.of.onset-no.covariates_genes_chr-3_peak.marker-UNC5008656_lod.drop-1.5_5.batches_0.csv
    Untracked:  data/ici.vs.eoi_age.of.onset-no.covariates_genes_chr-3_peak.marker-UNC5008656_lod.drop-1.5_5.batches_52.csv
    Untracked:  data/ici.vs.eoi_age.of.onset-no.covariates_genes_chr-3_peak.marker-UNC5008656_lod.drop-1.5_5.batches_mis.csv
    Untracked:  data/ici.vs.eoi_age.of.onset-no.covariates_genes_chr-3_peak.marker-UNC5008656_lod.drop-1.5_5.batches_mis_0.csv
    Untracked:  data/ici.vs.eoi_age.of.onset-no.covariates_genes_chr-3_peak.marker-UNC5046545_lod.drop-1.5_5.batches_mis_0.csv
    Untracked:  data/ici.vs.eoi_age.of.onset-no.covariates_genes_chr-3_peak.marker-UNC5055259_lod.drop-1.5_5.batches_mis_0.csv
    Untracked:  data/ici.vs.eoi_age.of.onset-no.covariates_genes_chr-3_peak.marker-UNC5062524_lod.drop-1.5_5.batches_mis_0.csv
    Untracked:  data/ici.vs.eoi_age.of.onset-no.covariates_genes_chr-3_peak.marker-UNC5068215_lod.drop-1.5_5.batches_mis_0.csv
    Untracked:  data/ici.vs.eoi_age.of.onset-no.covariates_genes_chr-3_peak.marker-UNC5076935_lod.drop-1.5_5.batches_mis_0.csv
    Untracked:  data/ici.vs.eoi_age.of.onset-no.covariates_genes_chr-3_peak.marker-UNC5082757_lod.drop-1.5_5.batches_mis_0.csv
    Untracked:  data/ici.vs.eoi_age.of.onset-no.covariates_genes_chr-3_peak.marker-UNC5083168_lod.drop-1.5_5.batches_0.csv
    Untracked:  data/ici.vs.eoi_age.of.onset-no.covariates_genes_chr-3_peak.marker-UNC5083168_lod.drop-1.5_5.batches_52.csv
    Untracked:  data/ici.vs.eoi_age.of.onset-no.covariates_genes_chr-3_peak.marker-UNC5083168_lod.drop-1.5_5.batches_mis_0.csv
    Untracked:  data/ici.vs.eoi_age.of.onset-no.covariates_genes_chr-3_peak.marker-UNC5173293_lod.drop-1.5_5.batches_mis_0.csv
    Untracked:  data/ici.vs.eoi_age.of.onset-no.covariates_genes_chr-3_peak.marker-UNC5192582_lod.drop-1.5_5.batches_0.csv
    Untracked:  data/ici.vs.eoi_age.of.onset-no.covariates_genes_chr-3_peak.marker-UNC5192582_lod.drop-1.5_5.batches_52.csv
    Untracked:  data/ici.vs.eoi_age.of.onset-no.covariates_genes_chr-3_peak.marker-UNC5192582_lod.drop-1.5_5.batches_mis_0.csv
    Untracked:  data/ici.vs.eoi_age.of.onset-no.covariates_genes_chr-3_peak.marker-UNC5193970_lod.drop-1.5_5.batches_mis_0.csv
    Untracked:  data/ici.vs.eoi_age.of.onset-no.covariates_genes_chr-3_peak.marker-UNC5213888_lod.drop-1.5_snpsqc_5.batches_mis_0.csv
    Untracked:  data/ici.vs.eoi_age.of.onset-no.covariates_genes_chr-3_peak.marker-UNC5213888_lod.drop-1.5_snpsqc_5.batches_mis_52.csv
    Untracked:  data/ici.vs.eoi_age.of.onset-no.covariates_genes_chr-3_peak.marker-UNC5257268_lod.drop-1.5_5.batches_mis_0.csv
    Untracked:  data/ici.vs.eoi_age.of.onset-no.covariates_genes_chr-3_peak.marker-UNC5262472_lod.drop-1.5_5.batches_0.csv
    Untracked:  data/ici.vs.eoi_age.of.onset-no.covariates_genes_chr-3_peak.marker-UNC5262472_lod.drop-1.5_5.batches_52.csv
    Untracked:  data/ici.vs.eoi_age.of.onset-no.covariates_genes_chr-3_peak.marker-UNC5262472_lod.drop-1.5_5.batches_mis_0.csv
    Untracked:  data/ici.vs.eoi_age.of.onset-no.covariates_genes_chr-3_peak.marker-UNC5264696_lod.drop-1.5_5.batches_mis_0.csv
    Untracked:  data/ici.vs.eoi_age.of.onset-no.covariates_genes_chr-3_peak.marker-UNC5267813_lod.drop-1.5_5.batches_mis_0.csv
    Untracked:  data/ici.vs.eoi_age.of.onset-no.covariates_genes_chr-3_peak.marker-UNC5270225_lod.drop-1.5_5.batches_mis_0.csv
    Untracked:  data/ici.vs.eoi_age.of.onset-no.covariates_genes_chr-3_peak.marker-UNC5277061_lod.drop-1.5_5.batches_mis_0.csv
    Untracked:  data/ici.vs.eoi_age.of.onset-no.covariates_genes_chr-3_peak.marker-UNC5278229_lod.drop-1.5_5.batches_0.csv
    Untracked:  data/ici.vs.eoi_age.of.onset-no.covariates_genes_chr-3_peak.marker-UNC5278229_lod.drop-1.5_5.batches_mis_0.csv
    Untracked:  data/ici.vs.eoi_age.of.onset-no.covariates_genes_chr-3_peak.marker-UNC5282259_lod.drop-1.5_5.batches_mis_0.csv
    Untracked:  data/ici.vs.eoi_age.of.onset-no.covariates_genes_chr-3_peak.marker-UNC5287977_lod.drop-1.5_5.batches_mis_0.csv
    Untracked:  data/ici.vs.eoi_age.of.onset-no.covariates_genes_chr-3_peak.marker-UNC5301460_lod.drop-1.5_5.batches_mis_0.csv
    Untracked:  data/ici.vs.eoi_age.of.onset-no.covariates_genes_chr-3_peak.marker-UNC5309136_lod.drop-1.5_5.batches_mis_0.csv
    Untracked:  data/ici.vs.eoi_age.of.onset-no.covariates_genes_chr-3_peak.marker-UNC5321559_lod.drop-1.5_5.batches_mis_0.csv
    Untracked:  data/ici.vs.eoi_age.of.onset-no.covariates_genes_chr-3_peak.marker-UNC5322455_lod.drop-1.5_5.batches_mis_0.csv
    Untracked:  data/ici.vs.eoi_age.of.onset-no.covariates_genes_chr-3_peak.marker-UNC5329171_lod.drop-1.5_5.batches_mis_0.csv
    Untracked:  data/ici.vs.eoi_age.of.onset-no.covariates_genes_chr-3_peak.marker-UNC5360055_lod.drop-1.5_5.batches_52.csv
    Untracked:  data/ici.vs.eoi_age.of.onset-no.covariates_genes_chr-3_peak.marker-UNC5454839_lod.drop-1.5_5.batches_0.csv
    Untracked:  data/ici.vs.eoi_age.of.onset-no.covariates_genes_chr-3_peak.marker-UNC5454839_lod.drop-1.5_5.batches_52.csv
    Untracked:  data/ici.vs.eoi_age.of.onset-no.covariates_genes_chr-3_peak.marker-UNC5454839_lod.drop-1.5_5.batches_mis_0.csv
    Untracked:  data/ici.vs.eoi_age.of.onset-no.covariates_genes_chr-3_peak.marker-UNC5645844_lod.drop-1.5_5.batches_0.csv
    Untracked:  data/ici.vs.eoi_age.of.onset-no.covariates_genes_chr-3_peak.marker-UNC5645844_lod.drop-1.5_5.batches_52.csv
    Untracked:  data/ici.vs.eoi_age.of.onset-no.covariates_genes_chr-3_peak.marker-UNC5645844_lod.drop-1.5_5.batches_mis_0.csv
    Untracked:  data/ici.vs.eoi_age.of.onset-no.covariates_genes_chr-3_peak.marker-UNC5778977_lod.drop-1.5_5.batches_0.csv
    Untracked:  data/ici.vs.eoi_age.of.onset-no.covariates_genes_chr-3_peak.marker-UNC5808246_lod.drop-1.5_5.batches_52.csv
    Untracked:  data/ici.vs.eoi_age.of.onset-no.covariates_genes_chr-3_peak.marker-UNC5916997_lod.drop-1.5_5.batches_52.csv
    Untracked:  data/ici.vs.eoi_age.of.onset-no.covariates_genes_chr-3_peak.marker-UNCHS008182_lod.drop-1.5_5.batches_0.csv
    Untracked:  data/ici.vs.eoi_age.of.onset-no.covariates_genes_chr-3_peak.marker-UNCHS008182_lod.drop-1.5_5.batches_52.csv
    Untracked:  data/ici.vs.eoi_age.of.onset-no.covariates_genes_chr-3_peak.marker-UNCHS008182_lod.drop-1.5_5.batches_mis_0.csv
    Untracked:  data/ici.vs.eoi_age.of.onset-no.covariates_genes_chr-3_peak.marker-UNCHS008193_lod.drop-1.5_5.batches_mis.csv
    Untracked:  data/ici.vs.eoi_age.of.onset-no.covariates_genes_chr-3_peak.marker-UNCHS008193_lod.drop-1.5_5.batches_mis_0.csv
    Untracked:  data/ici.vs.eoi_age.of.onset-no.covariates_genes_chr-3_peak.marker-UNCHS008209_lod.drop-1.5_5.batches_mis.csv
    Untracked:  data/ici.vs.eoi_age.of.onset-no.covariates_genes_chr-3_peak.marker-UNCHS008209_lod.drop-1.5_5.batches_mis_0.csv
    Untracked:  data/ici.vs.eoi_age.of.onset-no.covariates_genes_chr-3_peak.marker-UNCHS008215_lod.drop-1.5_5.batches_mis.csv
    Untracked:  data/ici.vs.eoi_age.of.onset-no.covariates_genes_chr-3_peak.marker-UNCHS008215_lod.drop-1.5_5.batches_mis_0.csv
    Untracked:  data/ici.vs.eoi_age.of.onset-no.covariates_genes_chr-3_peak.marker-UNCHS008219_lod.drop-1.5_5.batches_mis.csv
    Untracked:  data/ici.vs.eoi_age.of.onset-no.covariates_genes_chr-3_peak.marker-UNCHS008219_lod.drop-1.5_5.batches_mis_0.csv
    Untracked:  data/ici.vs.eoi_age.of.onset-no.covariates_genes_chr-3_peak.marker-UNCHS008229_lod.drop-1.5_5.batches_mis.csv
    Untracked:  data/ici.vs.eoi_age.of.onset-no.covariates_genes_chr-3_peak.marker-UNCHS008229_lod.drop-1.5_5.batches_mis_0.csv
    Untracked:  data/ici.vs.eoi_age.of.onset-no.covariates_genes_chr-3_peak.marker-UNCHS008234_lod.drop-1.5_5.batches_mis.csv
    Untracked:  data/ici.vs.eoi_age.of.onset-no.covariates_genes_chr-3_peak.marker-UNCHS008234_lod.drop-1.5_5.batches_mis_0.csv
    Untracked:  data/ici.vs.eoi_age.of.onset-no.covariates_genes_chr-3_peak.marker-UNCHS008242_lod.drop-1.5_5.batches_52.csv
    Untracked:  data/ici.vs.eoi_age.of.onset-no.covariates_genes_chr-3_peak.marker-UNCHS008260_lod.drop-1.5_5.batches_mis.csv
    Untracked:  data/ici.vs.eoi_age.of.onset-no.covariates_genes_chr-3_peak.marker-UNCHS008260_lod.drop-1.5_5.batches_mis_0.csv
    Untracked:  data/ici.vs.eoi_age.of.onset-no.covariates_genes_chr-3_peak.marker-UNCHS008271_lod.drop-1.5_5.batches_mis.csv
    Untracked:  data/ici.vs.eoi_age.of.onset-no.covariates_genes_chr-3_peak.marker-UNCHS008271_lod.drop-1.5_5.batches_mis_0.csv
    Untracked:  data/ici.vs.eoi_age.of.onset-no.covariates_genes_chr-3_peak.marker-UNCHS008281_lod.drop-1.5_5.batches_mis.csv
    Untracked:  data/ici.vs.eoi_age.of.onset-no.covariates_genes_chr-3_peak.marker-UNCHS008281_lod.drop-1.5_5.batches_mis_0.csv
    Untracked:  data/ici.vs.eoi_age.of.onset-no.covariates_genes_chr-3_peak.marker-UNCHS008286_lod.drop-1.5_5.batches_0.csv
    Untracked:  data/ici.vs.eoi_age.of.onset-no.covariates_genes_chr-3_peak.marker-UNCHS008286_lod.drop-1.5_5.batches_52.csv
    Untracked:  data/ici.vs.eoi_age.of.onset-no.covariates_genes_chr-3_peak.marker-UNCHS008286_lod.drop-1.5_5.batches_mis_0.csv
    Untracked:  data/ici.vs.eoi_age.of.onset-no.covariates_genes_chr-3_peak.marker-UNCHS008338_lod.drop-1.5_5.batches_mis.csv
    Untracked:  data/ici.vs.eoi_age.of.onset-no.covariates_genes_chr-3_peak.marker-UNCHS008338_lod.drop-1.5_5.batches_mis_0.csv
    Untracked:  data/ici.vs.eoi_age.of.onset-no.covariates_genes_chr-3_peak.marker-UNCHS008363_lod.drop-1.5_5.batches_mis.csv
    Untracked:  data/ici.vs.eoi_age.of.onset-no.covariates_genes_chr-3_peak.marker-UNCHS008363_lod.drop-1.5_5.batches_mis_0.csv
    Untracked:  data/ici.vs.eoi_age.of.onset-no.covariates_genes_chr-3_peak.marker-UNCHS008379_lod.drop-1.5_5.batches_mis.csv
    Untracked:  data/ici.vs.eoi_age.of.onset-no.covariates_genes_chr-3_peak.marker-UNCHS008379_lod.drop-1.5_5.batches_mis_0.csv
    Untracked:  data/ici.vs.eoi_age.of.onset-no.covariates_genes_chr-3_peak.marker-UNCHS008409_lod.drop-1.5_5.batches_mis_0.csv
    Untracked:  data/ici.vs.eoi_age.of.onset-no.covariates_genes_chr-3_peak.marker-UNCHS008432_lod.drop-1.5_5.batches_mis_0.csv
    Untracked:  data/ici.vs.eoi_age.of.onset-no.covariates_genes_chr-3_peak.marker-UNCHS008487_lod.drop-1.5_snpsqc_5.batches_0.csv
    Untracked:  data/ici.vs.eoi_age.of.onset-no.covariates_genes_chr-3_peak.marker-UNCHS008487_lod.drop-1.5_snpsqc_5.batches_52.csv
    Untracked:  data/ici.vs.eoi_age.of.onset-no.covariates_genes_chr-3_peak.marker-UNCHS008487_lod.drop-1.5_snpsqc_dis_no-x_updated_5.batches_0.csv
    Untracked:  data/ici.vs.eoi_age.of.onset-no.covariates_genes_chr-3_peak.marker-UNCHS008487_lod.drop-1.5_snpsqc_dis_no-x_updated_5.batches_52.csv
    Untracked:  data/ici.vs.eoi_age.of.onset-no.covariates_genes_chr-3_peak.marker-UNCHS008487_lod.drop-1.5_snpsqc_dis_no-x_updated_5.batches_mis_0.csv
    Untracked:  data/ici.vs.eoi_age.of.onset-no.covariates_genes_chr-3_peak.marker-UNCHS008487_lod.drop-1.5_snpsqc_dis_no-x_updated_5.batches_mis_52.csv
    Untracked:  data/ici.vs.eoi_age.of.onset-no.covariates_genes_chr-3_peak.marker-UNCHS008511_lod.drop-1.5_snpsqc_5.batches_mis_0.csv
    Untracked:  data/ici.vs.eoi_age.of.onset-no.covariates_genes_chr-3_peak.marker-UNCHS008511_lod.drop-1.5_snpsqc_5.batches_mis_52.csv
    Untracked:  data/ici.vs.eoi_age.of.onset-no.covariates_genes_chr-3_peak.marker-UNCHS008609_lod.drop-1.5_5.batches_mis_0.csv
    Untracked:  data/ici.vs.eoi_age.of.onset-no.covariates_genes_chr-3_peak.marker-UNCHS008613_lod.drop-1.5_5.batches_0.csv
    Untracked:  data/ici.vs.eoi_age.of.onset-no.covariates_genes_chr-3_peak.marker-UNCHS008613_lod.drop-1.5_5.batches_52.csv
    Untracked:  data/ici.vs.eoi_age.of.onset-no.covariates_genes_chr-3_peak.marker-UNCHS008614_lod.drop-1.5_5.batches_mis_0.csv
    Untracked:  data/ici.vs.eoi_age.of.onset-no.covariates_genes_chr-3_peak.marker-UNCHS008627_lod.drop-1.5_5.batches_mis_0.csv
    Untracked:  data/ici.vs.eoi_age.of.onset-no.covariates_genes_chr-3_peak.marker-UNCHS008659_lod.drop-1.5_5.batches_52.csv
    Untracked:  data/ici.vs.eoi_age.of.onset-no.covariates_genes_chr-3_peak.marker-UNCHS008725_lod.drop-1.5_5.batches_mis_0.csv
    Untracked:  data/ici.vs.eoi_age.of.onset-no.covariates_genes_chr-3_peak.marker-UNCHS008815_lod.drop-1.5_5.batches_0.csv
    Untracked:  data/ici.vs.eoi_age.of.onset-no.covariates_genes_chr-3_peak.marker-UNCHS008815_lod.drop-1.5_5.batches_52.csv
    Untracked:  data/ici.vs.eoi_age.of.onset-no.covariates_genes_chr-3_peak.marker-UNCHS008815_lod.drop-1.5_5.batches_mis_0.csv
    Untracked:  data/ici.vs.eoi_age.of.onset-no.covariates_genes_chr-3_peak.marker-UNCHS009066_lod.drop-1.5_5.batches_52.csv
    Untracked:  data/ici.vs.eoi_age.of.onset-no.covariates_genes_chr-3_peak.marker-UNCHS009067_lod.drop-1.5_5.batches_0.csv
    Untracked:  data/ici.vs.eoi_age.of.onset-no.covariates_genes_chr-3_peak.marker-UNCHS009068_lod.drop-1.5_5.batches_mis_0.csv
    Untracked:  data/ici.vs.eoi_age.of.onset-no.covariates_genes_chr-3_peak.marker-UNCHS009112_lod.drop-1.5_5.batches_52.csv
    Untracked:  data/ici.vs.eoi_age.of.onset-no.covariates_genes_chr-3_peak.marker-UNCHS009224_lod.drop-1.5_5.batches_0.csv
    Untracked:  data/ici.vs.eoi_age.of.onset-no.covariates_genes_chr-3_peak.marker-UNCHS009224_lod.drop-1.5_5.batches_52.csv
    Untracked:  data/ici.vs.eoi_age.of.onset-no.covariates_genes_chr-3_peak.marker-UNCHS009383_lod.drop-1.5_5.batches_0.csv
    Untracked:  data/ici.vs.eoi_age.of.onset-no.covariates_genes_chr-3_peak.marker-UNCHS009383_lod.drop-1.5_5.batches_52.csv
    Untracked:  data/ici.vs.eoi_age.of.onset-no.covariates_genes_chr-3_peak.marker-UNCHS009693_lod.drop-1.5_5.batches_52.csv
    Untracked:  data/ici.vs.eoi_age.of.onset-no.covariates_genes_chr-3_peak.marker-UNCHS009821_lod.drop-1.5_5.batches_52.csv
    Untracked:  data/ici.vs.eoi_age.of.onset-no.covariates_genes_chr-3_peak.marker-UNCJPD001170_lod.drop-1.5_5.batches_mis.csv
    Untracked:  data/ici.vs.eoi_age.of.onset-no.covariates_genes_chr-3_peak.marker-UNCJPD001170_lod.drop-1.5_5.batches_mis_0.csv
    Untracked:  data/ici.vs.eoi_age.of.onset-no.covariates_genes_chr-3_peak.marker-UNCJPD001198_lod.drop-1.5_5.batches_mis.csv
    Untracked:  data/ici.vs.eoi_age.of.onset-no.covariates_genes_chr-3_peak.marker-UNCJPD001198_lod.drop-1.5_5.batches_mis_0.csv
    Untracked:  data/ici.vs.eoi_age.of.onset-no.covariates_genes_chr-3_peak.marker-UNCJPD001276_lod.drop-1.5_5.batches_mis_0.csv
    Untracked:  data/ici.vs.eoi_age.of.onset-no.covariates_genes_chr-3_peak.marker-sanger2496q_lod.drop-1.5_5.batches_mis.csv
    Untracked:  data/ici.vs.eoi_age.of.onset-no.covariates_genes_chr-3_peak.marker-sanger2496q_lod.drop-1.5_5.batches_mis_0.csv
    Untracked:  data/ici.vs.eoi_age.of.onset-no.covariates_genes_chr-4_peak.marker-UNC8250659_lod.drop-1.5_snpsqc_dis_no-x_updated_5.batches_52.csv
    Untracked:  data/ici.vs.eoi_age.of.onset-no.covariates_genes_chr-4_peak.marker-UNC8439633_lod.drop-1.5_snpsqc_5.batches_52.csv
    Untracked:  data/ici.vs.eoi_age.of.onset-no.covariates_genes_chr-4_peak.marker-UNC8439633_lod.drop-1.5_snpsqc_5.batches_mis_52.csv
    Untracked:  data/ici.vs.eoi_age.of.onset-no.covariates_genes_chr-4_peak.marker-UNCHS012955_lod.drop-1.5_snpsqc_dis_no-x_updated_5.batches_mis_52.csv
    Untracked:  data/ici.vs.eoi_age.of.onset-no.covariates_genes_chr-6_peak.marker-UNC11108920_lod.drop-1.5_snpsqc_5.batches.csv
    Untracked:  data/ici.vs.eoi_age.of.onset-no.covariates_genes_chr-6_peak.marker-UNC11108920_lod.drop-1.5_snpsqc_5.batches_mis.csv
    Untracked:  data/ici.vs.eoi_age.of.onset-no.covariates_genes_chr-7_peak.marker-UNCHS020066_lod.drop-1.5_snpsqc_5.batches_mis_0.csv
    Untracked:  data/ici.vs.eoi_age.of.onset-no.covariates_genes_chr-7_peak.marker-UNCHS020066_lod.drop-1.5_snpsqc_5.batches_mis_52.csv
    Untracked:  data/ici.vs.eoi_age.of.onset-no.covariates_genes_chr-7_peak.marker-UNCHS020066_lod.drop-1.5_snpsqc_dis_no-x_updated_5.batches_mis_0.csv
    Untracked:  data/ici.vs.eoi_age.of.onset-no.covariates_genes_chr-7_peak.marker-UNCHS020066_lod.drop-1.5_snpsqc_dis_no-x_updated_5.batches_mis_52.csv
    Untracked:  data/ici.vs.eoi_age.of.onset-no.covariates_genes_chr-8_peak.marker-UNC15524531_lod.drop-1.5_snpsqc_5.batches_mis_0.csv
    Untracked:  data/ici.vs.eoi_age.of.onset-no.covariates_genes_chr-8_peak.marker-UNC15524531_lod.drop-1.5_snpsqc_5.batches_mis_52.csv
    Untracked:  data/ici.vs.eoi_age.of.onset-no.covariates_genes_chr-8_peak.marker-UNC15524531_lod.drop-1.5_snpsqc_dis_no-x_updated_5.batches_mis_0.csv
    Untracked:  data/ici.vs.eoi_age.of.onset-no.covariates_genes_chr-8_peak.marker-UNC15524531_lod.drop-1.5_snpsqc_dis_no-x_updated_5.batches_mis_52.csv
    Untracked:  data/ici.vs.eoi_age.of.onset-no.covariates_genes_chr-9_peak.marker-UNC17203597_lod.drop-1.5_snpsqc_5.batches_mis_0.csv
    Untracked:  data/ici.vs.eoi_age.of.onset-no.covariates_genes_chr-9_peak.marker-UNC17203597_lod.drop-1.5_snpsqc_5.batches_mis_52.csv
    Untracked:  data/ici.vs.eoi_age.of.onset-no.covariates_genes_chr-X_peak.marker-UNCHS048314_lod.drop-1.5_snpsqc_5.batches_mis_0.csv
    Untracked:  data/ici.vs.eoi_age.of.onset-no.covariates_genes_chr-X_peak.marker-UNCHS048314_lod.drop-1.5_snpsqc_5.batches_mis_52.csv
    Untracked:  data/ici.vs.eoi_age.of.onset-no.covariates_genes_chr-X_peak.marker-UNCHS048314_lod.drop-1.5_snpsqc_dis_no-x_updated_5.batches_mis_0.csv
    Untracked:  data/ici.vs.eoi_age.of.onset-no.covariates_genes_chr-X_peak.marker-UNCHS048314_lod.drop-1.5_snpsqc_dis_no-x_updated_5.batches_mis_52.csv
    Untracked:  data/ici.vs.eoi_blup.full_chr-10_snpsqc_5.batches.csv
    Untracked:  data/ici.vs.eoi_blup.full_chr-10_snpsqc_5.batches_mis.csv
    Untracked:  data/ici.vs.eoi_blup.full_chr-11_snpsqc_5.batches.csv
    Untracked:  data/ici.vs.eoi_blup.full_chr-11_snpsqc_5.batches_mis.csv
    Untracked:  data/ici.vs.eoi_blup.full_chr-12_snpsqc_5.batches.csv
    Untracked:  data/ici.vs.eoi_blup.full_chr-12_snpsqc_5.batches_mis.csv
    Untracked:  data/ici.vs.eoi_blup.full_chr-13_snpsqc_5.batches.csv
    Untracked:  data/ici.vs.eoi_blup.full_chr-13_snpsqc_5.batches_mis.csv
    Untracked:  data/ici.vs.eoi_blup.full_chr-14_snpsqc_5.batches.csv
    Untracked:  data/ici.vs.eoi_blup.full_chr-14_snpsqc_5.batches_mis.csv
    Untracked:  data/ici.vs.eoi_blup.full_chr-15_snpsqc_5.batches.csv
    Untracked:  data/ici.vs.eoi_blup.full_chr-15_snpsqc_5.batches_mis.csv
    Untracked:  data/ici.vs.eoi_blup.full_chr-16_snpsqc_5.batches.csv
    Untracked:  data/ici.vs.eoi_blup.full_chr-16_snpsqc_5.batches_mis.csv
    Untracked:  data/ici.vs.eoi_blup.full_chr-17_snpsqc_5.batches.csv
    Untracked:  data/ici.vs.eoi_blup.full_chr-17_snpsqc_5.batches_mis.csv
    Untracked:  data/ici.vs.eoi_blup.full_chr-18_snpsqc_5.batches.csv
    Untracked:  data/ici.vs.eoi_blup.full_chr-18_snpsqc_5.batches_mis.csv
    Untracked:  data/ici.vs.eoi_blup.full_chr-19_snpsqc_5.batches.csv
    Untracked:  data/ici.vs.eoi_blup.full_chr-19_snpsqc_5.batches_mis.csv
    Untracked:  data/ici.vs.eoi_blup.full_chr-1_snpsqc_5.batches.csv
    Untracked:  data/ici.vs.eoi_blup.full_chr-1_snpsqc_5.batches_mis.csv
    Untracked:  data/ici.vs.eoi_blup.full_chr-2_snpsqc_5.batches.csv
    Untracked:  data/ici.vs.eoi_blup.full_chr-2_snpsqc_5.batches_mis.csv
    Untracked:  data/ici.vs.eoi_blup.full_chr-3_snpsqc_5.batches.csv
    Untracked:  data/ici.vs.eoi_blup.full_chr-3_snpsqc_5.batches_mis.csv
    Untracked:  data/ici.vs.eoi_blup.full_chr-4_snpsqc_5.batches.csv
    Untracked:  data/ici.vs.eoi_blup.full_chr-4_snpsqc_5.batches_mis.csv
    Untracked:  data/ici.vs.eoi_blup.full_chr-5_snpsqc_5.batches.csv
    Untracked:  data/ici.vs.eoi_blup.full_chr-5_snpsqc_5.batches_mis.csv
    Untracked:  data/ici.vs.eoi_blup.full_chr-6_snpsqc_5.batches.csv
    Untracked:  data/ici.vs.eoi_blup.full_chr-6_snpsqc_5.batches_mis.csv
    Untracked:  data/ici.vs.eoi_blup.full_chr-7_snpsqc_5.batches.csv
    Untracked:  data/ici.vs.eoi_blup.full_chr-7_snpsqc_5.batches_mis.csv
    Untracked:  data/ici.vs.eoi_blup.full_chr-8_snpsqc_5.batches.csv
    Untracked:  data/ici.vs.eoi_blup.full_chr-8_snpsqc_5.batches_mis.csv
    Untracked:  data/ici.vs.eoi_blup.full_chr-9_snpsqc_5.batches.csv
    Untracked:  data/ici.vs.eoi_blup.full_chr-9_snpsqc_5.batches_mis.csv
    Untracked:  data/ici.vs.eoi_blup.full_chr-X_snpsqc_5.batches.csv
    Untracked:  data/ici.vs.eoi_blup.full_chr-X_snpsqc_5.batches_mis.csv
    Untracked:  data/ici.vs.eoi_blup.qc_chr-10_5.batches.csv
    Untracked:  data/ici.vs.eoi_blup.qc_chr-10_5.batches_mis.csv
    Untracked:  data/ici.vs.eoi_blup.qc_chr-10_lod.drop-1.5_snpsqc_5.batches.csv
    Untracked:  data/ici.vs.eoi_blup.qc_chr-10_snpsqc_5.batches.csv
    Untracked:  data/ici.vs.eoi_blup.qc_chr-10_snpsqc_5.batches_mis.csv
    Untracked:  data/ici.vs.eoi_blup.qc_chr-10_snpsqc_dis_no-x_updated_5.batches.csv
    Untracked:  data/ici.vs.eoi_blup.qc_chr-10_snpsqc_dis_no-x_updated_5.batches_mis.csv
    Untracked:  data/ici.vs.eoi_blup.qc_chr-11_5.batches.csv
    Untracked:  data/ici.vs.eoi_blup.qc_chr-11_5.batches_mis.csv
    Untracked:  data/ici.vs.eoi_blup.qc_chr-11_lod.drop-1.5_snpsqc_5.batches.csv
    Untracked:  data/ici.vs.eoi_blup.qc_chr-11_snpsqc_5.batches.csv
    Untracked:  data/ici.vs.eoi_blup.qc_chr-11_snpsqc_5.batches_mis.csv
    Untracked:  data/ici.vs.eoi_blup.qc_chr-11_snpsqc_dis_no-x_updated_5.batches.csv
    Untracked:  data/ici.vs.eoi_blup.qc_chr-11_snpsqc_dis_no-x_updated_5.batches_mis.csv
    Untracked:  data/ici.vs.eoi_blup.qc_chr-12_5.batches.csv
    Untracked:  data/ici.vs.eoi_blup.qc_chr-12_5.batches_mis.csv
    Untracked:  data/ici.vs.eoi_blup.qc_chr-12_lod.drop-1.5_snpsqc_5.batches.csv
    Untracked:  data/ici.vs.eoi_blup.qc_chr-12_snpsqc_5.batches.csv
    Untracked:  data/ici.vs.eoi_blup.qc_chr-12_snpsqc_5.batches_mis.csv
    Untracked:  data/ici.vs.eoi_blup.qc_chr-12_snpsqc_dis_no-x_updated_5.batches.csv
    Untracked:  data/ici.vs.eoi_blup.qc_chr-12_snpsqc_dis_no-x_updated_5.batches_mis.csv
    Untracked:  data/ici.vs.eoi_blup.qc_chr-13_5.batches.csv
    Untracked:  data/ici.vs.eoi_blup.qc_chr-13_5.batches_mis.csv
    Untracked:  data/ici.vs.eoi_blup.qc_chr-13_lod.drop-1.5_snpsqc_5.batches.csv
    Untracked:  data/ici.vs.eoi_blup.qc_chr-13_snpsqc_5.batches.csv
    Untracked:  data/ici.vs.eoi_blup.qc_chr-13_snpsqc_5.batches_mis.csv
    Untracked:  data/ici.vs.eoi_blup.qc_chr-13_snpsqc_dis_no-x_updated_5.batches.csv
    Untracked:  data/ici.vs.eoi_blup.qc_chr-13_snpsqc_dis_no-x_updated_5.batches_mis.csv
    Untracked:  data/ici.vs.eoi_blup.qc_chr-14_5.batches.csv
    Untracked:  data/ici.vs.eoi_blup.qc_chr-14_5.batches_mis.csv
    Untracked:  data/ici.vs.eoi_blup.qc_chr-14_snpsqc_5.batches.csv
    Untracked:  data/ici.vs.eoi_blup.qc_chr-14_snpsqc_5.batches_mis.csv
    Untracked:  data/ici.vs.eoi_blup.qc_chr-14_snpsqc_dis_no-x_updated_5.batches.csv
    Untracked:  data/ici.vs.eoi_blup.qc_chr-14_snpsqc_dis_no-x_updated_5.batches_mis.csv
    Untracked:  data/ici.vs.eoi_blup.qc_chr-15_5.batches.csv
    Untracked:  data/ici.vs.eoi_blup.qc_chr-15_5.batches_mis.csv
    Untracked:  data/ici.vs.eoi_blup.qc_chr-15_snpsqc_5.batches.csv
    Untracked:  data/ici.vs.eoi_blup.qc_chr-15_snpsqc_5.batches_mis.csv
    Untracked:  data/ici.vs.eoi_blup.qc_chr-15_snpsqc_dis_no-x_updated_5.batches_mis.csv
    Untracked:  data/ici.vs.eoi_blup.qc_chr-16_5.batches.csv
    Untracked:  data/ici.vs.eoi_blup.qc_chr-16_5.batches_mis.csv
    Untracked:  data/ici.vs.eoi_blup.qc_chr-16_snpsqc_5.batches.csv
    Untracked:  data/ici.vs.eoi_blup.qc_chr-16_snpsqc_5.batches_mis.csv
    Untracked:  data/ici.vs.eoi_blup.qc_chr-16_snpsqc_dis_no-x_updated_5.batches.csv
    Untracked:  data/ici.vs.eoi_blup.qc_chr-16_snpsqc_dis_no-x_updated_5.batches_mis.csv
    Untracked:  data/ici.vs.eoi_blup.qc_chr-17_5.batches.csv
    Untracked:  data/ici.vs.eoi_blup.qc_chr-17_5.batches_mis.csv
    Untracked:  data/ici.vs.eoi_blup.qc_chr-17_snpsqc_5.batches.csv
    Untracked:  data/ici.vs.eoi_blup.qc_chr-17_snpsqc_5.batches_mis.csv
    Untracked:  data/ici.vs.eoi_blup.qc_chr-17_snpsqc_dis_no-x_updated_5.batches.csv
    Untracked:  data/ici.vs.eoi_blup.qc_chr-17_snpsqc_dis_no-x_updated_5.batches_mis.csv
    Untracked:  data/ici.vs.eoi_blup.qc_chr-18_5.batches.csv
    Untracked:  data/ici.vs.eoi_blup.qc_chr-18_5.batches_mis.csv
    Untracked:  data/ici.vs.eoi_blup.qc_chr-18_snpsqc_5.batches.csv
    Untracked:  data/ici.vs.eoi_blup.qc_chr-18_snpsqc_5.batches_mis.csv
    Untracked:  data/ici.vs.eoi_blup.qc_chr-18_snpsqc_dis_no-x_updated_5.batches.csv
    Untracked:  data/ici.vs.eoi_blup.qc_chr-18_snpsqc_dis_no-x_updated_5.batches_mis.csv
    Untracked:  data/ici.vs.eoi_blup.qc_chr-19_5.batches.csv
    Untracked:  data/ici.vs.eoi_blup.qc_chr-19_5.batches_mis.csv
    Untracked:  data/ici.vs.eoi_blup.qc_chr-19_snpsqc_5.batches.csv
    Untracked:  data/ici.vs.eoi_blup.qc_chr-19_snpsqc_5.batches_mis.csv
    Untracked:  data/ici.vs.eoi_blup.qc_chr-19_snpsqc_dis_no-x_updated_5.batches.csv
    Untracked:  data/ici.vs.eoi_blup.qc_chr-19_snpsqc_dis_no-x_updated_5.batches_mis.csv
    Untracked:  data/ici.vs.eoi_blup.qc_chr-1_5.batches.csv
    Untracked:  data/ici.vs.eoi_blup.qc_chr-1_5.batches_mis.csv
    Untracked:  data/ici.vs.eoi_blup.qc_chr-1_lod.drop-1.5_snpsqc_5.batches.csv
    Untracked:  data/ici.vs.eoi_blup.qc_chr-1_snpsqc_5.batches.csv
    Untracked:  data/ici.vs.eoi_blup.qc_chr-1_snpsqc_5.batches_mis.csv
    Untracked:  data/ici.vs.eoi_blup.qc_chr-1_snpsqc_dis_no-x_updated_5.batches.csv
    Untracked:  data/ici.vs.eoi_blup.qc_chr-1_snpsqc_dis_no-x_updated_5.batches_mis.csv
    Untracked:  data/ici.vs.eoi_blup.qc_chr-2_5.batches.csv
    Untracked:  data/ici.vs.eoi_blup.qc_chr-2_5.batches_mis.csv
    Untracked:  data/ici.vs.eoi_blup.qc_chr-2_lod.drop-1.5_snpsqc_5.batches.csv
    Untracked:  data/ici.vs.eoi_blup.qc_chr-2_snpsqc_5.batches.csv
    Untracked:  data/ici.vs.eoi_blup.qc_chr-2_snpsqc_5.batches_mis.csv
    Untracked:  data/ici.vs.eoi_blup.qc_chr-2_snpsqc_dis_no-x_updated_5.batches.csv
    Untracked:  data/ici.vs.eoi_blup.qc_chr-2_snpsqc_dis_no-x_updated_5.batches_mis.csv
    Untracked:  data/ici.vs.eoi_blup.qc_chr-3_5.batches.csv
    Untracked:  data/ici.vs.eoi_blup.qc_chr-3_5.batches_mis.csv
    Untracked:  data/ici.vs.eoi_blup.qc_chr-3_lod.drop-1.5_snpsqc_5.batches.csv
    Untracked:  data/ici.vs.eoi_blup.qc_chr-3_snpsqc_5.batches.csv
    Untracked:  data/ici.vs.eoi_blup.qc_chr-3_snpsqc_5.batches_mis.csv
    Untracked:  data/ici.vs.eoi_blup.qc_chr-3_snpsqc_dis_no-x_updated_5.batches.csv
    Untracked:  data/ici.vs.eoi_blup.qc_chr-3_snpsqc_dis_no-x_updated_5.batches_mis.csv
    Untracked:  data/ici.vs.eoi_blup.qc_chr-4_5.batches.csv
    Untracked:  data/ici.vs.eoi_blup.qc_chr-4_5.batches_mis.csv
    Untracked:  data/ici.vs.eoi_blup.qc_chr-4_lod.drop-1.5_snpsqc_5.batches.csv
    Untracked:  data/ici.vs.eoi_blup.qc_chr-4_snpsqc_5.batches.csv
    Untracked:  data/ici.vs.eoi_blup.qc_chr-4_snpsqc_5.batches_mis.csv
    Untracked:  data/ici.vs.eoi_blup.qc_chr-4_snpsqc_dis_no-x_updated_5.batches.csv
    Untracked:  data/ici.vs.eoi_blup.qc_chr-4_snpsqc_dis_no-x_updated_5.batches_mis.csv
    Untracked:  data/ici.vs.eoi_blup.qc_chr-5_5.batches.csv
    Untracked:  data/ici.vs.eoi_blup.qc_chr-5_5.batches_mis.csv
    Untracked:  data/ici.vs.eoi_blup.qc_chr-5_lod.drop-1.5_snpsqc_5.batches.csv
    Untracked:  data/ici.vs.eoi_blup.qc_chr-5_snpsqc_5.batches.csv
    Untracked:  data/ici.vs.eoi_blup.qc_chr-5_snpsqc_5.batches_mis.csv
    Untracked:  data/ici.vs.eoi_blup.qc_chr-5_snpsqc_dis_no-x_updated_5.batches.csv
    Untracked:  data/ici.vs.eoi_blup.qc_chr-5_snpsqc_dis_no-x_updated_5.batches_mis.csv
    Untracked:  data/ici.vs.eoi_blup.qc_chr-6_5.batches.csv
    Untracked:  data/ici.vs.eoi_blup.qc_chr-6_5.batches_mis.csv
    Untracked:  data/ici.vs.eoi_blup.qc_chr-6_lod.drop-1.5_snpsqc_5.batches.csv
    Untracked:  data/ici.vs.eoi_blup.qc_chr-6_snpsqc_5.batches.csv
    Untracked:  data/ici.vs.eoi_blup.qc_chr-6_snpsqc_5.batches_mis.csv
    Untracked:  data/ici.vs.eoi_blup.qc_chr-6_snpsqc_dis_no-x_updated_5.batches.csv
    Untracked:  data/ici.vs.eoi_blup.qc_chr-6_snpsqc_dis_no-x_updated_5.batches_mis.csv
    Untracked:  data/ici.vs.eoi_blup.qc_chr-7_5.batches.csv
    Untracked:  data/ici.vs.eoi_blup.qc_chr-7_5.batches_mis.csv
    Untracked:  data/ici.vs.eoi_blup.qc_chr-7_lod.drop-1.5_snpsqc_5.batches.csv
    Untracked:  data/ici.vs.eoi_blup.qc_chr-7_snpsqc_5.batches.csv
    Untracked:  data/ici.vs.eoi_blup.qc_chr-7_snpsqc_5.batches_mis.csv
    Untracked:  data/ici.vs.eoi_blup.qc_chr-7_snpsqc_dis_no-x_updated_5.batches.csv
    Untracked:  data/ici.vs.eoi_blup.qc_chr-7_snpsqc_dis_no-x_updated_5.batches_mis.csv
    Untracked:  data/ici.vs.eoi_blup.qc_chr-8_5.batches.csv
    Untracked:  data/ici.vs.eoi_blup.qc_chr-8_5.batches_mis.csv
    Untracked:  data/ici.vs.eoi_blup.qc_chr-8_lod.drop-1.5_snpsqc_5.batches.csv
    Untracked:  data/ici.vs.eoi_blup.qc_chr-8_snpsqc_5.batches.csv
    Untracked:  data/ici.vs.eoi_blup.qc_chr-8_snpsqc_5.batches_mis.csv
    Untracked:  data/ici.vs.eoi_blup.qc_chr-8_snpsqc_dis_no-x_updated_5.batches.csv
    Untracked:  data/ici.vs.eoi_blup.qc_chr-8_snpsqc_dis_no-x_updated_5.batches_mis.csv
    Untracked:  data/ici.vs.eoi_blup.qc_chr-9_5.batches.csv
    Untracked:  data/ici.vs.eoi_blup.qc_chr-9_5.batches_mis.csv
    Untracked:  data/ici.vs.eoi_blup.qc_chr-9_lod.drop-1.5_snpsqc_5.batches.csv
    Untracked:  data/ici.vs.eoi_blup.qc_chr-9_snpsqc_5.batches.csv
    Untracked:  data/ici.vs.eoi_blup.qc_chr-9_snpsqc_5.batches_mis.csv
    Untracked:  data/ici.vs.eoi_blup.qc_chr-9_snpsqc_dis_no-x_updated_5.batches.csv
    Untracked:  data/ici.vs.eoi_blup.qc_chr-9_snpsqc_dis_no-x_updated_5.batches_mis.csv
    Untracked:  data/ici.vs.eoi_blup.qc_chr-X_5.batches.csv
    Untracked:  data/ici.vs.eoi_blup.qc_chr-X_5.batches_mis.csv
    Untracked:  data/ici.vs.eoi_blup.qc_chr-X_snpsqc_5.batches.csv
    Untracked:  data/ici.vs.eoi_blup.qc_chr-X_snpsqc_5.batches_mis.csv
    Untracked:  data/ici.vs.eoi_blup.qc_chr-X_snpsqc_dis_no-x_updated_5.batches.csv
    Untracked:  data/ici.vs.eoi_blup.qc_chr-X_snpsqc_dis_no-x_updated_5.batches_mis.csv
    Untracked:  data/ici.vs.eoi_blup_sub_chr-10_peak.marker-ICR4295_lod.drop-1.5_5.batches_mis.csv
    Untracked:  data/ici.vs.eoi_blup_sub_chr-10_peak.marker-JAX00292499_lod.drop-1.5_5.batches.csv
    Untracked:  data/ici.vs.eoi_blup_sub_chr-10_peak.marker-JAX00292927_lod.drop-1.5_5.batches.csv
    Untracked:  data/ici.vs.eoi_blup_sub_chr-10_peak.marker-JAX00292927_lod.drop-1.5_snpsqc_5.batches.csv
    Untracked:  data/ici.vs.eoi_blup_sub_chr-10_peak.marker-JAX00292927_lod.drop-1.5_snpsqc_5.batches_mis.csv
    Untracked:  data/ici.vs.eoi_blup_sub_chr-10_peak.marker-UNC18154330_lod.drop-1.5_5.batches.csv
    Untracked:  data/ici.vs.eoi_blup_sub_chr-10_peak.marker-UNC18195765_lod.drop-1.5_5.batches.csv
    Untracked:  data/ici.vs.eoi_blup_sub_chr-10_peak.marker-UNC18216614_lod.drop-1.5_5.batches.csv
    Untracked:  data/ici.vs.eoi_blup_sub_chr-10_peak.marker-UNC18240977_lod.drop-1.5_snpsqc_dis_no-x_updated_5.batches.csv
    Untracked:  data/ici.vs.eoi_blup_sub_chr-10_peak.marker-UNC18311938_lod.drop-1.5_5.batches.csv
    Untracked:  data/ici.vs.eoi_blup_sub_chr-10_peak.marker-UNC18856953_lod.drop-1.5_5.batches_mis.csv
    Untracked:  data/ici.vs.eoi_blup_sub_chr-10_peak.marker-UNCHS028236_lod.drop-1.5_snpsqc_5.batches_mis.csv
    Untracked:  data/ici.vs.eoi_blup_sub_chr-10_peak.marker-UNCHS028536_lod.drop-1.5_snpsqc_dis_no-x_updated_5.batches_mis.csv
    Untracked:  data/ici.vs.eoi_blup_sub_chr-10_peak.marker-UNCJPD004316_lod.drop-1.5_5.batches_mis.csv
    Untracked:  data/ici.vs.eoi_blup_sub_chr-10_peak.marker-UNCJPD009532_lod.drop-1.5_5.batches_mis.csv
    Untracked:  data/ici.vs.eoi_blup_sub_chr-11_peak.marker-UNC19155926_lod.drop-1.5_5.batches_mis.csv
    Untracked:  data/ici.vs.eoi_blup_sub_chr-11_peak.marker-UNC19481366_lod.drop-1.5_snpsqc_5.batches_mis.csv
    Untracked:  data/ici.vs.eoi_blup_sub_chr-11_peak.marker-UNC19970181_lod.drop-1.5_5.batches_mis.csv
    Untracked:  data/ici.vs.eoi_blup_sub_chr-11_peak.marker-UNC19970181_lod.drop-1.5_snpsqc_5.batches_mis.csv
    Untracked:  data/ici.vs.eoi_blup_sub_chr-11_peak.marker-UNCJPD004792_lod.drop-1.5_5.batches_mis.csv
    Untracked:  data/ici.vs.eoi_blup_sub_chr-12_peak.marker-ICR4497_lod.drop-1.5_5.batches_mis.csv
    Untracked:  data/ici.vs.eoi_blup_sub_chr-12_peak.marker-ICR499_lod.drop-1.5_snpsqc_5.batches_mis.csv
    Untracked:  data/ici.vs.eoi_blup_sub_chr-13_peak.marker-UNCHS036773_lod.drop-1.5_snpsqc_5.batches_mis.csv
    Untracked:  data/ici.vs.eoi_blup_sub_chr-13_peak.marker-UNCHS036964_lod.drop-1.5_5.batches_mis.csv
    Untracked:  data/ici.vs.eoi_blup_sub_chr-14_peak.marker-UNC24056202_lod.drop-1.5_5.batches_mis.csv
    Untracked:  data/ici.vs.eoi_blup_sub_chr-14_peak.marker-UNC24056202_lod.drop-1.5_snpsqc_5.batches_mis.csv
    Untracked:  data/ici.vs.eoi_blup_sub_chr-15_peak.marker-UNC26070435_lod.drop-1.5_snpsqc_dis_no-x_updated_5.batches_mis.csv
    Untracked:  data/ici.vs.eoi_blup_sub_chr-17_peak.marker-UNCHS044241_lod.drop-1.5_5.batches_mis.csv
    Untracked:  data/ici.vs.eoi_blup_sub_chr-17_peak.marker-UNCHS044241_lod.drop-1.5_snpsqc_5.batches_mis.csv
    Untracked:  data/ici.vs.eoi_blup_sub_chr-17_peak.marker-UNCrs47191360_lod.drop-1.5_5.batches_mis.csv
    Untracked:  data/ici.vs.eoi_blup_sub_chr-17_peak.marker-UNCrs47191360_lod.drop-1.5_snpsqc_dis_no-x_updated_5.batches_mis.csv
    Untracked:  data/ici.vs.eoi_blup_sub_chr-18_peak.marker-UNCHS045343_lod.drop-1.5_snpsqc_5.batches_mis.csv
    Untracked:  data/ici.vs.eoi_blup_sub_chr-18_peak.marker-UNCHS045344_lod.drop-1.5_5.batches_mis.csv
    Untracked:  data/ici.vs.eoi_blup_sub_chr-19_peak.marker-UNCJPD007087_lod.drop-1.5_5.batches_mis.csv
    Untracked:  data/ici.vs.eoi_blup_sub_chr-1_peak.marker-ICR010_lod.drop-1.5_5.batches.csv
    Untracked:  data/ici.vs.eoi_blup_sub_chr-1_peak.marker-ICR010_lod.drop-1.5_5.batches_mis.csv
    Untracked:  data/ici.vs.eoi_blup_sub_chr-2_peak.marker-UNC4609527_lod.drop-1.5_snpsqc_dis_no-x_updated_5.batches_mis.csv
    Untracked:  data/ici.vs.eoi_blup_sub_chr-2_peak.marker-UNC4609660_lod.drop-1.5_5.batches_mis.csv
    Untracked:  data/ici.vs.eoi_blup_sub_chr-3_peak.marker-B6_rs31740628_lod.drop-1.5_5.batches_mis.csv
    Untracked:  data/ici.vs.eoi_blup_sub_chr-3_peak.marker-ICR1269_lod.drop-1.5_5.batches_mis.csv
    Untracked:  data/ici.vs.eoi_blup_sub_chr-3_peak.marker-ICR1338_lod.drop-1.5_5.batches.csv
    Untracked:  data/ici.vs.eoi_blup_sub_chr-3_peak.marker-ICR5325_lod.drop-1.5_5.batches_mis.csv
    Untracked:  data/ici.vs.eoi_blup_sub_chr-3_peak.marker-ICR5347_lod.drop-1.5_5.batches.csv
    Untracked:  data/ici.vs.eoi_blup_sub_chr-3_peak.marker-ICR5348_lod.drop-1.5_5.batches_mis.csv
    Untracked:  data/ici.vs.eoi_blup_sub_chr-3_peak.marker-JAX00104971_lod.drop-1.5_5.batches_mis.csv
    Untracked:  data/ici.vs.eoi_blup_sub_chr-3_peak.marker-JAX00105059_lod.drop-1.5_5.batches.csv
    Untracked:  data/ici.vs.eoi_blup_sub_chr-3_peak.marker-JAX00105203_lod.drop-1.5_5.batches_mis.csv
    Untracked:  data/ici.vs.eoi_blup_sub_chr-3_peak.marker-JAX00106957_lod.drop-1.5_5.batches.csv
    Untracked:  data/ici.vs.eoi_blup_sub_chr-3_peak.marker-JAX00106957_lod.drop-1.5_5.batches_mis.csv
    Untracked:  data/ici.vs.eoi_blup_sub_chr-3_peak.marker-JAX00107930_lod.drop-1.5_5.batches_mis.csv
    Untracked:  data/ici.vs.eoi_blup_sub_chr-3_peak.marker-JAX00107980_lod.drop-1.5_5.batches_mis.csv
    Untracked:  data/ici.vs.eoi_blup_sub_chr-3_peak.marker-JAX00108068_lod.drop-1.5_5.batches_mis.csv
    Untracked:  data/ici.vs.eoi_blup_sub_chr-3_peak.marker-JAX00108114_lod.drop-1.5_5.batches_mis.csv
    Untracked:  data/ici.vs.eoi_blup_sub_chr-3_peak.marker-JAX00109750_lod.drop-1.5_5.batches_mis.csv
    Untracked:  data/ici.vs.eoi_blup_sub_chr-3_peak.marker-JAX00189245_lod.drop-1.5_5.batches_mis.csv
    Untracked:  data/ici.vs.eoi_blup_sub_chr-3_peak.marker-JAX00514662_lod.drop-1.5_5.batches.csv
    Untracked:  data/ici.vs.eoi_blup_sub_chr-3_peak.marker-JAX00517218_lod.drop-1.5_5.batches_mis.csv
    Untracked:  data/ici.vs.eoi_blup_sub_chr-3_peak.marker-JAX00517674_lod.drop-1.5_5.batches_mis.csv
    Untracked:  data/ici.vs.eoi_blup_sub_chr-3_peak.marker-JAX00517906_lod.drop-1.5_5.batches_mis.csv
    Untracked:  data/ici.vs.eoi_blup_sub_chr-3_peak.marker-JAX00518418_lod.drop-1.5_5.batches_mis.csv
    Untracked:  data/ici.vs.eoi_blup_sub_chr-3_peak.marker-JAX00518684_lod.drop-1.5_5.batches_mis.csv
    Untracked:  data/ici.vs.eoi_blup_sub_chr-3_peak.marker-JAX00518911r_lod.drop-1.5_5.batches_mis.csv
    Untracked:  data/ici.vs.eoi_blup_sub_chr-3_peak.marker-JAX00520122_lod.drop-1.5_5.batches_mis.csv
    Untracked:  data/ici.vs.eoi_blup_sub_chr-3_peak.marker-JAX00524717_lod.drop-1.5_5.batches_mis.csv
    Untracked:  data/ici.vs.eoi_blup_sub_chr-3_peak.marker-JAX00524828_lod.drop-1.5_5.batches_mis.csv
    Untracked:  data/ici.vs.eoi_blup_sub_chr-3_peak.marker-JAX00525380_lod.drop-1.5_5.batches_mis.csv
    Untracked:  data/ici.vs.eoi_blup_sub_chr-3_peak.marker-JAX00525709_lod.drop-1.5_5.batches_mis.csv
    Untracked:  data/ici.vs.eoi_blup_sub_chr-3_peak.marker-JAX00527291_lod.drop-1.5_5.batches_mis.csv
    Untracked:  data/ici.vs.eoi_blup_sub_chr-3_peak.marker-JAX00527587_lod.drop-1.5_5.batches_mis.csv
    Untracked:  data/ici.vs.eoi_blup_sub_chr-3_peak.marker-JAX00529707_lod.drop-1.5_5.batches_mis.csv
    Untracked:  data/ici.vs.eoi_blup_sub_chr-3_peak.marker-JAX00529707r_lod.drop-1.5_5.batches.csv
    Untracked:  data/ici.vs.eoi_blup_sub_chr-3_peak.marker-JAX00531601_lod.drop-1.5_5.batches_mis.csv
    Untracked:  data/ici.vs.eoi_blup_sub_chr-3_peak.marker-JAX00532122_lod.drop-1.5_5.batches_mis.csv
    Untracked:  data/ici.vs.eoi_blup_sub_chr-3_peak.marker-UNC4673471_lod.drop-1.5_5.batches_mis.csv
    Untracked:  data/ici.vs.eoi_blup_sub_chr-3_peak.marker-UNC4681411_lod.drop-1.5_5.batches_mis.csv
    Untracked:  data/ici.vs.eoi_blup_sub_chr-3_peak.marker-UNC4793677_lod.drop-1.5_5.batches_mis.csv
    Untracked:  data/ici.vs.eoi_blup_sub_chr-3_peak.marker-UNC4799475_lod.drop-1.5_5.batches_mis.csv
    Untracked:  data/ici.vs.eoi_blup_sub_chr-3_peak.marker-UNC4809586_lod.drop-1.5_5.batches_mis.csv
    Untracked:  data/ici.vs.eoi_blup_sub_chr-3_peak.marker-UNC4813186_lod.drop-1.5_5.batches.csv
    Untracked:  data/ici.vs.eoi_blup_sub_chr-3_peak.marker-UNC4813186_lod.drop-1.5_5.batches_mis.csv
    Untracked:  data/ici.vs.eoi_blup_sub_chr-3_peak.marker-UNC4820379_lod.drop-1.5_5.batches_mis.csv
    Untracked:  data/ici.vs.eoi_blup_sub_chr-3_peak.marker-UNC4875653_lod.drop-1.5_5.batches_mis.csv
    Untracked:  data/ici.vs.eoi_blup_sub_chr-3_peak.marker-UNC4909830_lod.drop-1.5_5.batches_mis.csv
    Untracked:  data/ici.vs.eoi_blup_sub_chr-3_peak.marker-UNC4912288_lod.drop-1.5_5.batches_mis.csv
    Untracked:  data/ici.vs.eoi_blup_sub_chr-3_peak.marker-UNC4914130_lod.drop-1.5_5.batches_mis.csv
    Untracked:  data/ici.vs.eoi_blup_sub_chr-3_peak.marker-UNC4923398_lod.drop-1.5_5.batches_mis.csv
    Untracked:  data/ici.vs.eoi_blup_sub_chr-3_peak.marker-UNC4924377_lod.drop-1.5_5.batches.csv
    Untracked:  data/ici.vs.eoi_blup_sub_chr-3_peak.marker-UNC4945186_lod.drop-1.5_5.batches_mis.csv
    Untracked:  data/ici.vs.eoi_blup_sub_chr-3_peak.marker-UNC4947492_lod.drop-1.5_5.batches_mis.csv
    Untracked:  data/ici.vs.eoi_blup_sub_chr-3_peak.marker-UNC4957168_lod.drop-1.5_5.batches.csv
    Untracked:  data/ici.vs.eoi_blup_sub_chr-3_peak.marker-UNC4957859_lod.drop-1.5_5.batches_mis.csv
    Untracked:  data/ici.vs.eoi_blup_sub_chr-3_peak.marker-UNC4964062_lod.drop-1.5_5.batches_mis.csv
    Untracked:  data/ici.vs.eoi_blup_sub_chr-3_peak.marker-UNC4979914_lod.drop-1.5_5.batches_mis.csv
    Untracked:  data/ici.vs.eoi_blup_sub_chr-3_peak.marker-UNC4985064_lod.drop-1.5_5.batches_mis.csv
    Untracked:  data/ici.vs.eoi_blup_sub_chr-3_peak.marker-UNC4990111_lod.drop-1.5_5.batches_mis.csv
    Untracked:  data/ici.vs.eoi_blup_sub_chr-3_peak.marker-UNC4991565_lod.drop-1.5_5.batches_mis.csv
    Untracked:  data/ici.vs.eoi_blup_sub_chr-3_peak.marker-UNC5001723_lod.drop-1.5_5.batches_mis.csv
    Untracked:  data/ici.vs.eoi_blup_sub_chr-3_peak.marker-UNC5008656_lod.drop-1.5_5.batches.csv
    Untracked:  data/ici.vs.eoi_blup_sub_chr-3_peak.marker-UNC5008656_lod.drop-1.5_5.batches_mis.csv
    Untracked:  data/ici.vs.eoi_blup_sub_chr-3_peak.marker-UNC5042361_lod.drop-1.5_5.batches_mis.csv
    Untracked:  data/ici.vs.eoi_blup_sub_chr-3_peak.marker-UNC5055259_lod.drop-1.5_5.batches_mis.csv
    Untracked:  data/ici.vs.eoi_blup_sub_chr-3_peak.marker-UNC5076935_lod.drop-1.5_5.batches_mis.csv
    Untracked:  data/ici.vs.eoi_blup_sub_chr-3_peak.marker-UNC5082757_lod.drop-1.5_5.batches_mis.csv
    Untracked:  data/ici.vs.eoi_blup_sub_chr-3_peak.marker-UNC5083168_lod.drop-1.5_5.batches.csv
    Untracked:  data/ici.vs.eoi_blup_sub_chr-3_peak.marker-UNC5173293_lod.drop-1.5_5.batches_mis.csv
    Untracked:  data/ici.vs.eoi_blup_sub_chr-3_peak.marker-UNC5182782_lod.drop-1.5_5.batches_mis.csv
    Untracked:  data/ici.vs.eoi_blup_sub_chr-3_peak.marker-UNC5192582_lod.drop-1.5_5.batches.csv
    Untracked:  data/ici.vs.eoi_blup_sub_chr-3_peak.marker-UNC5193970_lod.drop-1.5_5.batches_mis.csv
    Untracked:  data/ici.vs.eoi_blup_sub_chr-3_peak.marker-UNC5213888_lod.drop-1.5_snpsqc_5.batches_mis.csv
    Untracked:  data/ici.vs.eoi_blup_sub_chr-3_peak.marker-UNC5240484_lod.drop-1.5_5.batches_mis.csv
    Untracked:  data/ici.vs.eoi_blup_sub_chr-3_peak.marker-UNC5256922_lod.drop-1.5_5.batches_mis.csv
    Untracked:  data/ici.vs.eoi_blup_sub_chr-3_peak.marker-UNC5266664_lod.drop-1.5_5.batches_mis.csv
    Untracked:  data/ici.vs.eoi_blup_sub_chr-3_peak.marker-UNC5266711_lod.drop-1.5_5.batches.csv
    Untracked:  data/ici.vs.eoi_blup_sub_chr-3_peak.marker-UNC5267813_lod.drop-1.5_5.batches_mis.csv
    Untracked:  data/ici.vs.eoi_blup_sub_chr-3_peak.marker-UNC5274554_lod.drop-1.5_5.batches_mis.csv
    Untracked:  data/ici.vs.eoi_blup_sub_chr-3_peak.marker-UNC5277061_lod.drop-1.5_5.batches_mis.csv
    Untracked:  data/ici.vs.eoi_blup_sub_chr-3_peak.marker-UNC5282259_lod.drop-1.5_5.batches_mis.csv
    Untracked:  data/ici.vs.eoi_blup_sub_chr-3_peak.marker-UNC5287977_lod.drop-1.5_5.batches_mis.csv
    Untracked:  data/ici.vs.eoi_blup_sub_chr-3_peak.marker-UNC5301460_lod.drop-1.5_5.batches_mis.csv
    Untracked:  data/ici.vs.eoi_blup_sub_chr-3_peak.marker-UNC5321559_lod.drop-1.5_5.batches_mis.csv
    Untracked:  data/ici.vs.eoi_blup_sub_chr-3_peak.marker-UNC5322455_lod.drop-1.5_5.batches_mis.csv
    Untracked:  data/ici.vs.eoi_blup_sub_chr-3_peak.marker-UNC5328596_lod.drop-1.5_5.batches_mis.csv
    Untracked:  data/ici.vs.eoi_blup_sub_chr-3_peak.marker-UNC5350384_lod.drop-1.5_5.batches_mis.csv
    Untracked:  data/ici.vs.eoi_blup_sub_chr-3_peak.marker-UNC5354724_lod.drop-1.5_5.batches_mis.csv
    Untracked:  data/ici.vs.eoi_blup_sub_chr-3_peak.marker-UNC5375208_lod.drop-1.5_5.batches_mis.csv
    Untracked:  data/ici.vs.eoi_blup_sub_chr-3_peak.marker-UNC5384072_lod.drop-1.5_5.batches_mis.csv
    Untracked:  data/ici.vs.eoi_blup_sub_chr-3_peak.marker-UNC5400314_lod.drop-1.5_5.batches_mis.csv
    Untracked:  data/ici.vs.eoi_blup_sub_chr-3_peak.marker-UNC5433945_lod.drop-1.5_5.batches_mis.csv
    Untracked:  data/ici.vs.eoi_blup_sub_chr-3_peak.marker-UNC5435874_lod.drop-1.5_5.batches_mis.csv
    Untracked:  data/ici.vs.eoi_blup_sub_chr-3_peak.marker-UNC5454839_lod.drop-1.5_5.batches.csv
    Untracked:  data/ici.vs.eoi_blup_sub_chr-3_peak.marker-UNC5454839_lod.drop-1.5_5.batches_mis.csv
    Untracked:  data/ici.vs.eoi_blup_sub_chr-3_peak.marker-UNC5466669_lod.drop-1.5_5.batches_mis.csv
    Untracked:  data/ici.vs.eoi_blup_sub_chr-3_peak.marker-UNC5517972_lod.drop-1.5_5.batches_mis.csv
    Untracked:  data/ici.vs.eoi_blup_sub_chr-3_peak.marker-UNC5522257_lod.drop-1.5_5.batches_mis.csv
    Untracked:  data/ici.vs.eoi_blup_sub_chr-3_peak.marker-UNC5523265_lod.drop-1.5_5.batches_mis.csv
    Untracked:  data/ici.vs.eoi_blup_sub_chr-3_peak.marker-UNC5532424_lod.drop-1.5_5.batches_mis.csv
    Untracked:  data/ici.vs.eoi_blup_sub_chr-3_peak.marker-UNC5553882_lod.drop-1.5_5.batches_mis.csv
    Untracked:  data/ici.vs.eoi_blup_sub_chr-3_peak.marker-UNC5572051_lod.drop-1.5_5.batches_mis.csv
    Untracked:  data/ici.vs.eoi_blup_sub_chr-3_peak.marker-UNC5582753_lod.drop-1.5_5.batches.csv
    Untracked:  data/ici.vs.eoi_blup_sub_chr-3_peak.marker-UNC5589260_lod.drop-1.5_5.batches_mis.csv
    Untracked:  data/ici.vs.eoi_blup_sub_chr-3_peak.marker-UNC5595555_lod.drop-1.5_5.batches_mis.csv
    Untracked:  data/ici.vs.eoi_blup_sub_chr-3_peak.marker-UNC5623239_lod.drop-1.5_5.batches_mis.csv
    Untracked:  data/ici.vs.eoi_blup_sub_chr-3_peak.marker-UNC5639830_lod.drop-1.5_5.batches_mis.csv
    Untracked:  data/ici.vs.eoi_blup_sub_chr-3_peak.marker-UNC5645844_lod.drop-1.5_5.batches.csv
    Untracked:  data/ici.vs.eoi_blup_sub_chr-3_peak.marker-UNC5645844_lod.drop-1.5_5.batches_mis.csv
    Untracked:  data/ici.vs.eoi_blup_sub_chr-3_peak.marker-UNC5680234_lod.drop-1.5_5.batches_mis.csv
    Untracked:  data/ici.vs.eoi_blup_sub_chr-3_peak.marker-UNC5682314_lod.drop-1.5_5.batches_mis.csv
    Untracked:  data/ici.vs.eoi_blup_sub_chr-3_peak.marker-UNC5725356_lod.drop-1.5_5.batches_mis.csv
    Untracked:  data/ici.vs.eoi_blup_sub_chr-3_peak.marker-UNC5727351_lod.drop-1.5_5.batches_mis.csv
    Untracked:  data/ici.vs.eoi_blup_sub_chr-3_peak.marker-UNC5734420_lod.drop-1.5_5.batches_mis.csv
    Untracked:  data/ici.vs.eoi_blup_sub_chr-3_peak.marker-UNC5743257_lod.drop-1.5_5.batches_mis.csv
    Untracked:  data/ici.vs.eoi_blup_sub_chr-3_peak.marker-UNC5751567_lod.drop-1.5_5.batches_mis.csv
    Untracked:  data/ici.vs.eoi_blup_sub_chr-3_peak.marker-UNC5752623_lod.drop-1.5_5.batches_mis.csv
    Untracked:  data/ici.vs.eoi_blup_sub_chr-3_peak.marker-UNC5776974_lod.drop-1.5_5.batches_mis.csv
    Untracked:  data/ici.vs.eoi_blup_sub_chr-3_peak.marker-UNC5778977_lod.drop-1.5_5.batches_mis.csv
    Untracked:  data/ici.vs.eoi_blup_sub_chr-3_peak.marker-UNC5788507_lod.drop-1.5_5.batches_mis.csv
    Untracked:  data/ici.vs.eoi_blup_sub_chr-3_peak.marker-UNC5792468_lod.drop-1.5_5.batches_mis.csv
    Untracked:  data/ici.vs.eoi_blup_sub_chr-3_peak.marker-UNC5794598_lod.drop-1.5_5.batches_mis.csv
    Untracked:  data/ici.vs.eoi_blup_sub_chr-3_peak.marker-UNC5795561_lod.drop-1.5_5.batches_mis.csv
    Untracked:  data/ici.vs.eoi_blup_sub_chr-3_peak.marker-UNC5802298_lod.drop-1.5_5.batches_mis.csv
    Untracked:  data/ici.vs.eoi_blup_sub_chr-3_peak.marker-UNC5806628_lod.drop-1.5_5.batches_mis.csv
    Untracked:  data/ici.vs.eoi_blup_sub_chr-3_peak.marker-UNC5808246_lod.drop-1.5_5.batches.csv
    Untracked:  data/ici.vs.eoi_blup_sub_chr-3_peak.marker-UNC5817478_lod.drop-1.5_5.batches_mis.csv
    Untracked:  data/ici.vs.eoi_blup_sub_chr-3_peak.marker-UNC5836999_lod.drop-1.5_5.batches_mis.csv
    Untracked:  data/ici.vs.eoi_blup_sub_chr-3_peak.marker-UNC5841501_lod.drop-1.5_5.batches_mis.csv
    Untracked:  data/ici.vs.eoi_blup_sub_chr-3_peak.marker-UNC5847927_lod.drop-1.5_5.batches_mis.csv
    Untracked:  data/ici.vs.eoi_blup_sub_chr-3_peak.marker-UNC5966417_lod.drop-1.5_5.batches_mis.csv
    Untracked:  data/ici.vs.eoi_blup_sub_chr-3_peak.marker-UNCHS008129_lod.drop-1.5_5.batches_mis.csv
    Untracked:  data/ici.vs.eoi_blup_sub_chr-3_peak.marker-UNCHS008173_lod.drop-1.5_5.batches_mis.csv
    Untracked:  data/ici.vs.eoi_blup_sub_chr-3_peak.marker-UNCHS008182_lod.drop-1.5_5.batches.csv
    Untracked:  data/ici.vs.eoi_blup_sub_chr-3_peak.marker-UNCHS008193_lod.drop-1.5_5.batches_mis.csv
    Untracked:  data/ici.vs.eoi_blup_sub_chr-3_peak.marker-UNCHS008209_lod.drop-1.5_5.batches_mis.csv
    Untracked:  data/ici.vs.eoi_blup_sub_chr-3_peak.marker-UNCHS008215_lod.drop-1.5_5.batches_mis.csv
    Untracked:  data/ici.vs.eoi_blup_sub_chr-3_peak.marker-UNCHS008219_lod.drop-1.5_5.batches_mis.csv
    Untracked:  data/ici.vs.eoi_blup_sub_chr-3_peak.marker-UNCHS008231_lod.drop-1.5_5.batches_mis.csv
    Untracked:  data/ici.vs.eoi_blup_sub_chr-3_peak.marker-UNCHS008235_lod.drop-1.5_5.batches_mis.csv
    Untracked:  data/ici.vs.eoi_blup_sub_chr-3_peak.marker-UNCHS008242_lod.drop-1.5_5.batches.csv
    Untracked:  data/ici.vs.eoi_blup_sub_chr-3_peak.marker-UNCHS008253_lod.drop-1.5_5.batches_mis.csv
    Untracked:  data/ici.vs.eoi_blup_sub_chr-3_peak.marker-UNCHS008256_lod.drop-1.5_5.batches_mis.csv
    Untracked:  data/ici.vs.eoi_blup_sub_chr-3_peak.marker-UNCHS008259_lod.drop-1.5_5.batches_mis.csv
    Untracked:  data/ici.vs.eoi_blup_sub_chr-3_peak.marker-UNCHS008271_lod.drop-1.5_5.batches_mis.csv
    Untracked:  data/ici.vs.eoi_blup_sub_chr-3_peak.marker-UNCHS008286_lod.drop-1.5_5.batches.csv
    Untracked:  data/ici.vs.eoi_blup_sub_chr-3_peak.marker-UNCHS008286_lod.drop-1.5_5.batches_mis.csv
    Untracked:  data/ici.vs.eoi_blup_sub_chr-3_peak.marker-UNCHS008311_lod.drop-1.5_5.batches_mis.csv
    Untracked:  data/ici.vs.eoi_blup_sub_chr-3_peak.marker-UNCHS008317_lod.drop-1.5_5.batches_mis.csv
    Untracked:  data/ici.vs.eoi_blup_sub_chr-3_peak.marker-UNCHS008372_lod.drop-1.5_5.batches_mis.csv
    Untracked:  data/ici.vs.eoi_blup_sub_chr-3_peak.marker-UNCHS008379_lod.drop-1.5_5.batches_mis.csv
    Untracked:  data/ici.vs.eoi_blup_sub_chr-3_peak.marker-UNCHS008410_lod.drop-1.5_5.batches_mis.csv
    Untracked:  data/ici.vs.eoi_blup_sub_chr-3_peak.marker-UNCHS008415_lod.drop-1.5_5.batches_mis.csv
    Untracked:  data/ici.vs.eoi_blup_sub_chr-3_peak.marker-UNCHS008431_lod.drop-1.5_5.batches_mis.csv
    Untracked:  data/ici.vs.eoi_blup_sub_chr-3_peak.marker-UNCHS008487_lod.drop-1.5_snpsqc_5.batches.csv
    Untracked:  data/ici.vs.eoi_blup_sub_chr-3_peak.marker-UNCHS008487_lod.drop-1.5_snpsqc_dis_no-x_updated_5.batches.csv
    Untracked:  data/ici.vs.eoi_blup_sub_chr-3_peak.marker-UNCHS008487_lod.drop-1.5_snpsqc_dis_no-x_updated_5.batches_mis.csv
    Untracked:  data/ici.vs.eoi_blup_sub_chr-3_peak.marker-UNCHS008511_lod.drop-1.5_snpsqc_5.batches_mis.csv
    Untracked:  data/ici.vs.eoi_blup_sub_chr-3_peak.marker-UNCHS008609_lod.drop-1.5_5.batches_mis.csv
    Untracked:  data/ici.vs.eoi_blup_sub_chr-3_peak.marker-UNCHS008614_lod.drop-1.5_5.batches.csv
    Untracked:  data/ici.vs.eoi_blup_sub_chr-3_peak.marker-UNCHS008649_lod.drop-1.5_5.batches_mis.csv
    Untracked:  data/ici.vs.eoi_blup_sub_chr-3_peak.marker-UNCHS008651_lod.drop-1.5_5.batches.csv
    Untracked:  data/ici.vs.eoi_blup_sub_chr-3_peak.marker-UNCHS008656_lod.drop-1.5_5.batches_mis.csv
    Untracked:  data/ici.vs.eoi_blup_sub_chr-3_peak.marker-UNCHS008723_lod.drop-1.5_5.batches_mis.csv
    Untracked:  data/ici.vs.eoi_blup_sub_chr-3_peak.marker-UNCHS008802_lod.drop-1.5_5.batches_mis.csv
    Untracked:  data/ici.vs.eoi_blup_sub_chr-3_peak.marker-UNCHS008803_lod.drop-1.5_5.batches_mis.csv
    Untracked:  data/ici.vs.eoi_blup_sub_chr-3_peak.marker-UNCHS008808_lod.drop-1.5_5.batches_mis.csv
    Untracked:  data/ici.vs.eoi_blup_sub_chr-3_peak.marker-UNCHS008815_lod.drop-1.5_5.batches.csv
    Untracked:  data/ici.vs.eoi_blup_sub_chr-3_peak.marker-UNCHS008815_lod.drop-1.5_5.batches_mis.csv
    Untracked:  data/ici.vs.eoi_blup_sub_chr-3_peak.marker-UNCHS008834_lod.drop-1.5_5.batches_mis.csv
    Untracked:  data/ici.vs.eoi_blup_sub_chr-3_peak.marker-UNCHS008838_lod.drop-1.5_5.batches_mis.csv
    Untracked:  data/ici.vs.eoi_blup_sub_chr-3_peak.marker-UNCHS008914_lod.drop-1.5_5.batches_mis.csv
    Untracked:  data/ici.vs.eoi_blup_sub_chr-3_peak.marker-UNCHS008928_lod.drop-1.5_5.batches_mis.csv
    Untracked:  data/ici.vs.eoi_blup_sub_chr-3_peak.marker-UNCHS008948_lod.drop-1.5_5.batches_mis.csv
    Untracked:  data/ici.vs.eoi_blup_sub_chr-3_peak.marker-UNCHS008959_lod.drop-1.5_5.batches_mis.csv
    Untracked:  data/ici.vs.eoi_blup_sub_chr-3_peak.marker-UNCHS008973_lod.drop-1.5_5.batches_mis.csv
    Untracked:  data/ici.vs.eoi_blup_sub_chr-3_peak.marker-UNCHS008987_lod.drop-1.5_5.batches_mis.csv
    Untracked:  data/ici.vs.eoi_blup_sub_chr-3_peak.marker-UNCHS009001_lod.drop-1.5_5.batches_mis.csv
    Untracked:  data/ici.vs.eoi_blup_sub_chr-3_peak.marker-UNCHS009025_lod.drop-1.5_5.batches_mis.csv
    Untracked:  data/ici.vs.eoi_blup_sub_chr-3_peak.marker-UNCHS009034_lod.drop-1.5_5.batches_mis.csv
    Untracked:  data/ici.vs.eoi_blup_sub_chr-3_peak.marker-UNCHS009087_lod.drop-1.5_5.batches_mis.csv
    Untracked:  data/ici.vs.eoi_blup_sub_chr-3_peak.marker-UNCHS009109_lod.drop-1.5_5.batches_mis.csv
    Untracked:  data/ici.vs.eoi_blup_sub_chr-3_peak.marker-UNCHS009142_lod.drop-1.5_5.batches_mis.csv
    Untracked:  data/ici.vs.eoi_blup_sub_chr-3_peak.marker-UNCHS009207_lod.drop-1.5_5.batches_mis.csv
    Untracked:  data/ici.vs.eoi_blup_sub_chr-3_peak.marker-UNCHS009216_lod.drop-1.5_5.batches_mis.csv
    Untracked:  data/ici.vs.eoi_blup_sub_chr-3_peak.marker-UNCHS009224_lod.drop-1.5_5.batches.csv
    Untracked:  data/ici.vs.eoi_blup_sub_chr-3_peak.marker-UNCHS009306_lod.drop-1.5_5.batches_mis.csv
    Untracked:  data/ici.vs.eoi_blup_sub_chr-3_peak.marker-UNCHS009316_lod.drop-1.5_5.batches_mis.csv
    Untracked:  data/ici.vs.eoi_blup_sub_chr-3_peak.marker-UNCHS009364_lod.drop-1.5_5.batches_mis.csv
    Untracked:  data/ici.vs.eoi_blup_sub_chr-3_peak.marker-UNCHS009377_lod.drop-1.5_5.batches_mis.csv
    Untracked:  data/ici.vs.eoi_blup_sub_chr-3_peak.marker-UNCHS009382_lod.drop-1.5_5.batches_mis.csv
    Untracked:  data/ici.vs.eoi_blup_sub_chr-3_peak.marker-UNCHS009383_lod.drop-1.5_5.batches.csv
    Untracked:  data/ici.vs.eoi_blup_sub_chr-3_peak.marker-UNCHS009387_lod.drop-1.5_5.batches_mis.csv
    Untracked:  data/ici.vs.eoi_blup_sub_chr-3_peak.marker-UNCHS009395_lod.drop-1.5_5.batches_mis.csv
    Untracked:  data/ici.vs.eoi_blup_sub_chr-3_peak.marker-UNCHS009413_lod.drop-1.5_5.batches_mis.csv
    Untracked:  data/ici.vs.eoi_blup_sub_chr-3_peak.marker-UNCHS009428_lod.drop-1.5_5.batches_mis.csv
    Untracked:  data/ici.vs.eoi_blup_sub_chr-3_peak.marker-UNCHS009463_lod.drop-1.5_5.batches_mis.csv
    Untracked:  data/ici.vs.eoi_blup_sub_chr-3_peak.marker-UNCHS009503_lod.drop-1.5_5.batches_mis.csv
    Untracked:  data/ici.vs.eoi_blup_sub_chr-3_peak.marker-UNCHS009532_lod.drop-1.5_5.batches_mis.csv
    Untracked:  data/ici.vs.eoi_blup_sub_chr-3_peak.marker-UNCHS009541_lod.drop-1.5_5.batches_mis.csv
    Untracked:  data/ici.vs.eoi_blup_sub_chr-3_peak.marker-UNCHS009560_lod.drop-1.5_5.batches_mis.csv
    Untracked:  data/ici.vs.eoi_blup_sub_chr-3_peak.marker-UNCHS009574_lod.drop-1.5_5.batches_mis.csv
    Untracked:  data/ici.vs.eoi_blup_sub_chr-3_peak.marker-UNCJPD001170_lod.drop-1.5_5.batches_mis.csv
    Untracked:  data/ici.vs.eoi_blup_sub_chr-3_peak.marker-UNCJPD001198_lod.drop-1.5_5.batches_mis.csv
    Untracked:  data/ici.vs.eoi_blup_sub_chr-3_peak.marker-UNCJPD001276_lod.drop-1.5_5.batches_mis.csv
    Untracked:  data/ici.vs.eoi_blup_sub_chr-3_peak.marker-UNCJPD001315_lod.drop-1.5_5.batches_mis.csv
    Untracked:  data/ici.vs.eoi_blup_sub_chr-3_peak.marker-UNCJPD001355_lod.drop-1.5_5.batches_mis.csv
    Untracked:  data/ici.vs.eoi_blup_sub_chr-3_peak.marker-UNCJPD001373_lod.drop-1.5_5.batches_mis.csv
    Untracked:  data/ici.vs.eoi_blup_sub_chr-3_peak.marker-UNCJPD001382_lod.drop-1.5_5.batches_mis.csv
    Untracked:  data/ici.vs.eoi_blup_sub_chr-4_peak.marker-UNC6851205_lod.drop-1.5_5.batches_mis.csv
    Untracked:  data/ici.vs.eoi_blup_sub_chr-4_peak.marker-UNC8250659_lod.drop-1.5_snpsqc_dis_no-x_updated_5.batches.csv
    Untracked:  data/ici.vs.eoi_blup_sub_chr-4_peak.marker-UNC8299912_lod.drop-1.5_5.batches_mis.csv
    Untracked:  data/ici.vs.eoi_blup_sub_chr-4_peak.marker-UNC8300499_lod.drop-1.5_5.batches.csv
    Untracked:  data/ici.vs.eoi_blup_sub_chr-4_peak.marker-UNC8381981_lod.drop-1.5_5.batches_mis.csv
    Untracked:  data/ici.vs.eoi_blup_sub_chr-4_peak.marker-UNC8436434_lod.drop-1.5_5.batches_mis.csv
    Untracked:  data/ici.vs.eoi_blup_sub_chr-4_peak.marker-UNC8439633_lod.drop-1.5_5.batches_mis.csv
    Untracked:  data/ici.vs.eoi_blup_sub_chr-4_peak.marker-UNC8530763_lod.drop-1.5_5.batches_mis.csv
    Untracked:  data/ici.vs.eoi_blup_sub_chr-4_peak.marker-UNCHS012948_lod.drop-1.5_snpsqc_dis_no-x_updated_5.batches_mis.csv
    Untracked:  data/ici.vs.eoi_blup_sub_chr-4_peak.marker-UNCHS012976_lod.drop-1.5_5.batches_mis.csv
    Untracked:  data/ici.vs.eoi_blup_sub_chr-4_peak.marker-UNCHS013118_lod.drop-1.5_5.batches.csv
    Untracked:  data/ici.vs.eoi_blup_sub_chr-4_peak.marker-UNCHS013118_lod.drop-1.5_snpsqc_5.batches.csv
    Untracked:  data/ici.vs.eoi_blup_sub_chr-4_peak.marker-UNCHS013118_lod.drop-1.5_snpsqc_5.batches_mis.csv
    Untracked:  data/ici.vs.eoi_blup_sub_chr-4_peak.marker-UNCHS013130_lod.drop-1.5_5.batches_mis.csv
    Untracked:  data/ici.vs.eoi_blup_sub_chr-4_peak.marker-UNCHS013139_lod.drop-1.5_5.batches_mis.csv
    Untracked:  data/ici.vs.eoi_blup_sub_chr-4_peak.marker-UNCHS013304_lod.drop-1.5_5.batches.csv
    Untracked:  data/ici.vs.eoi_blup_sub_chr-5_peak.marker-JAX00590770_lod.drop-1.5_5.batches_mis.csv
    Untracked:  data/ici.vs.eoi_blup_sub_chr-5_peak.marker-UNC10044126_lod.drop-1.5_5.batches_mis.csv
    Untracked:  data/ici.vs.eoi_blup_sub_chr-7_peak.marker-ICR1830_lod.drop-1.5_5.batches_mis.csv
    Untracked:  data/ici.vs.eoi_blup_sub_chr-7_peak.marker-UNC13170794_lod.drop-1.5_5.batches_mis.csv
    Untracked:  data/ici.vs.eoi_blup_sub_chr-7_peak.marker-UNCHS019480_lod.drop-1.5_5.batches.csv
    Untracked:  data/ici.vs.eoi_blup_sub_chr-7_peak.marker-UNCHS020066_lod.drop-1.5_snpsqc_5.batches_mis.csv
    Untracked:  data/ici.vs.eoi_blup_sub_chr-7_peak.marker-UNCHS020066_lod.drop-1.5_snpsqc_dis_no-x_updated_5.batches_mis.csv
    Untracked:  data/ici.vs.eoi_blup_sub_chr-7_peak.marker-cr31snv87_lod.drop-1.5_5.batches_mis.csv
    Untracked:  data/ici.vs.eoi_blup_sub_chr-8_peak.marker-JAX00667121_lod.drop-1.5_5.batches_mis.csv
    Untracked:  data/ici.vs.eoi_blup_sub_chr-8_peak.marker-UNC15430711_lod.drop-1.5_5.batches_mis.csv
    Untracked:  data/ici.vs.eoi_blup_sub_chr-8_peak.marker-UNC15524531_lod.drop-1.5_5.batches_mis.csv
    Untracked:  data/ici.vs.eoi_blup_sub_chr-8_peak.marker-UNC15524531_lod.drop-1.5_snpsqc_5.batches_mis.csv
    Untracked:  data/ici.vs.eoi_blup_sub_chr-8_peak.marker-UNC15524531_lod.drop-1.5_snpsqc_dis_no-x_updated_5.batches_mis.csv
    Untracked:  data/ici.vs.eoi_blup_sub_chr-9_peak.marker-UNC17203329_lod.drop-1.5_5.batches_mis.csv
    Untracked:  data/ici.vs.eoi_blup_sub_chr-9_peak.marker-UNC17203597_lod.drop-1.5_snpsqc_5.batches_mis.csv
    Untracked:  data/ici.vs.eoi_blup_sub_chr-9_peak.marker-UNCHS024593_lod.drop-1.5_5.batches_mis.csv
    Untracked:  data/ici.vs.eoi_blup_sub_chr-X_peak.marker-UNC30627130_lod.drop-1.5_5.batches_mis.csv
    Untracked:  data/ici.vs.eoi_blup_sub_chr-X_peak.marker-UNCHS048313_lod.drop-1.5_5.batches_mis.csv
    Untracked:  data/ici.vs.eoi_blup_sub_chr-X_peak.marker-UNCHS048314_lod.drop-1.5_snpsqc_5.batches_mis.csv
    Untracked:  data/ici.vs.eoi_blup_sub_chr-X_peak.marker-UNCHS048314_lod.drop-1.5_snpsqc_dis_no-x_updated_5.batches_mis.csv
    Untracked:  data/ici.vs.eoi_fitqtl_peaks_additive_5.batches.txt
    Untracked:  data/ici.vs.eoi_fitqtl_peaks_additive_5.batches_mis.txt
    Untracked:  data/ici.vs.eoi_fitqtl_peaks_additive_snpsqc_5.batches.txt
    Untracked:  data/ici.vs.eoi_fitqtl_peaks_additive_snpsqc_dis_no-x_updated_5.batches.txt
    Untracked:  data/ici.vs.eoi_fitqtl_peaks_additive_snpsqc_dis_no-x_updated_5.batches_mis.txt
    Untracked:  data/ici.vs.eoi_fitqtl_peaks_interacting_5.batches.txt
    Untracked:  data/ici.vs.eoi_fitqtl_peaks_interacting_5.batches_mis.txt
    Untracked:  data/ici.vs.eoi_fitqtl_peaks_interacting_snpsqc_5.batches.txt
    Untracked:  data/ici.vs.eoi_fitqtl_peaks_interacting_snpsqc_dis_no-x_updated_5.batches.txt
    Untracked:  data/ici.vs.eoi_fitqtl_peaks_interacting_snpsqc_dis_no-x_updated_5.batches_mis.txt
    Untracked:  data/ici.vs.eoi_fitqtl_peaks_sex_additive_5.batches.txt
    Untracked:  data/ici.vs.eoi_fitqtl_peaks_sex_additive_5.batches_mis.txt
    Untracked:  data/ici.vs.eoi_fitqtl_peaks_sex_additive_snpsqc_5.batches.txt
    Untracked:  data/ici.vs.eoi_fitqtl_peaks_sex_additive_snpsqc_dis_no-x_updated_5.batches.txt
    Untracked:  data/ici.vs.eoi_fitqtl_peaks_sex_additive_snpsqc_dis_no-x_updated_5.batches_mis.txt
    Untracked:  data/ici.vs.eoi_fitqtl_peaks_sex_interacting_5.batches.txt
    Untracked:  data/ici.vs.eoi_fitqtl_peaks_sex_interacting_5.batches_mis.txt
    Untracked:  data/ici.vs.eoi_fitqtl_peaks_sex_interacting_snpsqc_5.batches.txt
    Untracked:  data/ici.vs.eoi_fitqtl_peaks_sex_interacting_snpsqc_dis_no-x_updated_5.batches.txt
    Untracked:  data/ici.vs.eoi_fitqtl_peaks_sex_interacting_snpsqc_dis_no-x_updated_5.batches_mis.mis.txt
    Untracked:  data/ici.vs.eoi_genes_chr-10_peak.marker-ICR4295_lod.drop-1.5_5.batches_mis.csv
    Untracked:  data/ici.vs.eoi_genes_chr-10_peak.marker-JAX00292499_lod.drop-1.5_5.batches.csv
    Untracked:  data/ici.vs.eoi_genes_chr-10_peak.marker-JAX00292927_lod.drop-1.5_5.batches.csv
    Untracked:  data/ici.vs.eoi_genes_chr-10_peak.marker-JAX00292927_lod.drop-1.5_snpsqc_5.batches.csv
    Untracked:  data/ici.vs.eoi_genes_chr-10_peak.marker-JAX00292927_lod.drop-1.5_snpsqc_5.batches_mis.csv
    Untracked:  data/ici.vs.eoi_genes_chr-10_peak.marker-UNC18154330_lod.drop-1.5_5.batches.csv
    Untracked:  data/ici.vs.eoi_genes_chr-10_peak.marker-UNC18195765_lod.drop-1.5_5.batches.csv
    Untracked:  data/ici.vs.eoi_genes_chr-10_peak.marker-UNC18216614_lod.drop-1.5_5.batches.csv
    Untracked:  data/ici.vs.eoi_genes_chr-10_peak.marker-UNC18240977_lod.drop-1.5_snpsqc_dis_no-x_updated_5.batches.csv
    Untracked:  data/ici.vs.eoi_genes_chr-10_peak.marker-UNC18311938_lod.drop-1.5_5.batches.csv
    Untracked:  data/ici.vs.eoi_genes_chr-10_peak.marker-UNC18856953_lod.drop-1.5_5.batches_mis.csv
    Untracked:  data/ici.vs.eoi_genes_chr-10_peak.marker-UNCHS028236_lod.drop-1.5_snpsqc_5.batches_mis.csv
    Untracked:  data/ici.vs.eoi_genes_chr-10_peak.marker-UNCHS028536_lod.drop-1.5_snpsqc_dis_no-x_updated_5.batches_mis.csv
    Untracked:  data/ici.vs.eoi_genes_chr-10_peak.marker-UNCJPD004316_lod.drop-1.5_5.batches_mis.csv
    Untracked:  data/ici.vs.eoi_genes_chr-10_peak.marker-UNCJPD009532_lod.drop-1.5_5.batches_mis.csv
    Untracked:  data/ici.vs.eoi_genes_chr-11_peak.marker-UNC19155926_lod.drop-1.5_5.batches_mis.csv
    Untracked:  data/ici.vs.eoi_genes_chr-11_peak.marker-UNC19481366_lod.drop-1.5_snpsqc_5.batches_mis.csv
    Untracked:  data/ici.vs.eoi_genes_chr-11_peak.marker-UNC19970181_lod.drop-1.5_5.batches_mis.csv
    Untracked:  data/ici.vs.eoi_genes_chr-11_peak.marker-UNC19970181_lod.drop-1.5_snpsqc_5.batches_mis.csv
    Untracked:  data/ici.vs.eoi_genes_chr-11_peak.marker-UNCJPD004792_lod.drop-1.5_5.batches_mis.csv
    Untracked:  data/ici.vs.eoi_genes_chr-12_peak.marker-ICR4497_lod.drop-1.5_5.batches_mis.csv
    Untracked:  data/ici.vs.eoi_genes_chr-12_peak.marker-ICR499_lod.drop-1.5_snpsqc_5.batches_mis.csv
    Untracked:  data/ici.vs.eoi_genes_chr-13_peak.marker-UNCHS036773_lod.drop-1.5_snpsqc_5.batches_mis.csv
    Untracked:  data/ici.vs.eoi_genes_chr-13_peak.marker-UNCHS036964_lod.drop-1.5_5.batches_mis.csv
    Untracked:  data/ici.vs.eoi_genes_chr-14_peak.marker-UNC24056202_lod.drop-1.5_5.batches_mis.csv
    Untracked:  data/ici.vs.eoi_genes_chr-14_peak.marker-UNC24056202_lod.drop-1.5_snpsqc_5.batches_mis.csv
    Untracked:  data/ici.vs.eoi_genes_chr-15_peak.marker-UNC26070435_lod.drop-1.5_snpsqc_dis_no-x_updated_5.batches_mis.csv
    Untracked:  data/ici.vs.eoi_genes_chr-17_peak.marker-UNCHS044241_lod.drop-1.5_5.batches_mis.csv
    Untracked:  data/ici.vs.eoi_genes_chr-17_peak.marker-UNCHS044241_lod.drop-1.5_snpsqc_5.batches_mis.csv
    Untracked:  data/ici.vs.eoi_genes_chr-17_peak.marker-UNCrs47191360_lod.drop-1.5_5.batches_mis.csv
    Untracked:  data/ici.vs.eoi_genes_chr-17_peak.marker-UNCrs47191360_lod.drop-1.5_snpsqc_dis_no-x_updated_5.batches_mis.csv
    Untracked:  data/ici.vs.eoi_genes_chr-18_peak.marker-UNCHS045343_lod.drop-1.5_snpsqc_5.batches_mis.csv
    Untracked:  data/ici.vs.eoi_genes_chr-18_peak.marker-UNCHS045344_lod.drop-1.5_5.batches_mis.csv
    Untracked:  data/ici.vs.eoi_genes_chr-19_peak.marker-UNCJPD007087_lod.drop-1.5_5.batches_mis.csv
    Untracked:  data/ici.vs.eoi_genes_chr-1_peak.marker-ICR010_lod.drop-1.5_5.batches.csv
    Untracked:  data/ici.vs.eoi_genes_chr-1_peak.marker-ICR010_lod.drop-1.5_5.batches_mis.csv
    Untracked:  data/ici.vs.eoi_genes_chr-2_peak.marker-UNC4609527_lod.drop-1.5_snpsqc_dis_no-x_updated_5.batches_mis.csv
    Untracked:  data/ici.vs.eoi_genes_chr-2_peak.marker-UNC4609660_lod.drop-1.5_5.batches_mis.csv
    Untracked:  data/ici.vs.eoi_genes_chr-3_peak.marker-B6_rs31740628_lod.drop-1.5_5.batches_mis.csv
    Untracked:  data/ici.vs.eoi_genes_chr-3_peak.marker-ICR1269_lod.drop-1.5_5.batches_mis.csv
    Untracked:  data/ici.vs.eoi_genes_chr-3_peak.marker-ICR1338_lod.drop-1.5_5.batches.csv
    Untracked:  data/ici.vs.eoi_genes_chr-3_peak.marker-ICR5325_lod.drop-1.5_5.batches_mis.csv
    Untracked:  data/ici.vs.eoi_genes_chr-3_peak.marker-ICR5347_lod.drop-1.5_5.batches.csv
    Untracked:  data/ici.vs.eoi_genes_chr-3_peak.marker-ICR5348_lod.drop-1.5_5.batches_mis.csv
    Untracked:  data/ici.vs.eoi_genes_chr-3_peak.marker-JAX00104971_lod.drop-1.5_5.batches_mis.csv
    Untracked:  data/ici.vs.eoi_genes_chr-3_peak.marker-JAX00105059_lod.drop-1.5_5.batches.csv
    Untracked:  data/ici.vs.eoi_genes_chr-3_peak.marker-JAX00105203_lod.drop-1.5_5.batches_mis.csv
    Untracked:  data/ici.vs.eoi_genes_chr-3_peak.marker-JAX00106957_lod.drop-1.5_5.batches.csv
    Untracked:  data/ici.vs.eoi_genes_chr-3_peak.marker-JAX00106957_lod.drop-1.5_5.batches_mis.csv
    Untracked:  data/ici.vs.eoi_genes_chr-3_peak.marker-JAX00107930_lod.drop-1.5_5.batches_mis.csv
    Untracked:  data/ici.vs.eoi_genes_chr-3_peak.marker-JAX00107980_lod.drop-1.5_5.batches_mis.csv
    Untracked:  data/ici.vs.eoi_genes_chr-3_peak.marker-JAX00108068_lod.drop-1.5_5.batches_mis.csv
    Untracked:  data/ici.vs.eoi_genes_chr-3_peak.marker-JAX00108114_lod.drop-1.5_5.batches_mis.csv
    Untracked:  data/ici.vs.eoi_genes_chr-3_peak.marker-JAX00109750_lod.drop-1.5_5.batches_mis.csv
    Untracked:  data/ici.vs.eoi_genes_chr-3_peak.marker-JAX00189245_lod.drop-1.5_5.batches_mis.csv
    Untracked:  data/ici.vs.eoi_genes_chr-3_peak.marker-JAX00514662_lod.drop-1.5_5.batches.csv
    Untracked:  data/ici.vs.eoi_genes_chr-3_peak.marker-JAX00517218_lod.drop-1.5_5.batches_mis.csv
    Untracked:  data/ici.vs.eoi_genes_chr-3_peak.marker-JAX00517674_lod.drop-1.5_5.batches_mis.csv
    Untracked:  data/ici.vs.eoi_genes_chr-3_peak.marker-JAX00517906_lod.drop-1.5_5.batches_mis.csv
    Untracked:  data/ici.vs.eoi_genes_chr-3_peak.marker-JAX00518418_lod.drop-1.5_5.batches_mis.csv
    Untracked:  data/ici.vs.eoi_genes_chr-3_peak.marker-JAX00518684_lod.drop-1.5_5.batches_mis.csv
    Untracked:  data/ici.vs.eoi_genes_chr-3_peak.marker-JAX00518911r_lod.drop-1.5_5.batches_mis.csv
    Untracked:  data/ici.vs.eoi_genes_chr-3_peak.marker-JAX00520122_lod.drop-1.5_5.batches_mis.csv
    Untracked:  data/ici.vs.eoi_genes_chr-3_peak.marker-JAX00524717_lod.drop-1.5_5.batches_mis.csv
    Untracked:  data/ici.vs.eoi_genes_chr-3_peak.marker-JAX00524828_lod.drop-1.5_5.batches_mis.csv
    Untracked:  data/ici.vs.eoi_genes_chr-3_peak.marker-JAX00525380_lod.drop-1.5_5.batches_mis.csv
    Untracked:  data/ici.vs.eoi_genes_chr-3_peak.marker-JAX00525709_lod.drop-1.5_5.batches_mis.csv
    Untracked:  data/ici.vs.eoi_genes_chr-3_peak.marker-JAX00527291_lod.drop-1.5_5.batches_mis.csv
    Untracked:  data/ici.vs.eoi_genes_chr-3_peak.marker-JAX00527587_lod.drop-1.5_5.batches_mis.csv
    Untracked:  data/ici.vs.eoi_genes_chr-3_peak.marker-JAX00529707_lod.drop-1.5_5.batches_mis.csv
    Untracked:  data/ici.vs.eoi_genes_chr-3_peak.marker-JAX00529707r_lod.drop-1.5_5.batches.csv
    Untracked:  data/ici.vs.eoi_genes_chr-3_peak.marker-JAX00531601_lod.drop-1.5_5.batches_mis.csv
    Untracked:  data/ici.vs.eoi_genes_chr-3_peak.marker-JAX00532122_lod.drop-1.5_5.batches_mis.csv
    Untracked:  data/ici.vs.eoi_genes_chr-3_peak.marker-UNC4673471_lod.drop-1.5_5.batches_mis.csv
    Untracked:  data/ici.vs.eoi_genes_chr-3_peak.marker-UNC4681411_lod.drop-1.5_5.batches_mis.csv
    Untracked:  data/ici.vs.eoi_genes_chr-3_peak.marker-UNC4793677_lod.drop-1.5_5.batches_mis.csv
    Untracked:  data/ici.vs.eoi_genes_chr-3_peak.marker-UNC4799475_lod.drop-1.5_5.batches_mis.csv
    Untracked:  data/ici.vs.eoi_genes_chr-3_peak.marker-UNC4809586_lod.drop-1.5_5.batches_mis.csv
    Untracked:  data/ici.vs.eoi_genes_chr-3_peak.marker-UNC4813186_lod.drop-1.5_5.batches.csv
    Untracked:  data/ici.vs.eoi_genes_chr-3_peak.marker-UNC4813186_lod.drop-1.5_5.batches_mis.csv
    Untracked:  data/ici.vs.eoi_genes_chr-3_peak.marker-UNC4820379_lod.drop-1.5_5.batches_mis.csv
    Untracked:  data/ici.vs.eoi_genes_chr-3_peak.marker-UNC4875653_lod.drop-1.5_5.batches_mis.csv
    Untracked:  data/ici.vs.eoi_genes_chr-3_peak.marker-UNC4909830_lod.drop-1.5_5.batches_mis.csv
    Untracked:  data/ici.vs.eoi_genes_chr-3_peak.marker-UNC4912288_lod.drop-1.5_5.batches_mis.csv
    Untracked:  data/ici.vs.eoi_genes_chr-3_peak.marker-UNC4914130_lod.drop-1.5_5.batches_mis.csv
    Untracked:  data/ici.vs.eoi_genes_chr-3_peak.marker-UNC4923398_lod.drop-1.5_5.batches_mis.csv
    Untracked:  data/ici.vs.eoi_genes_chr-3_peak.marker-UNC4924377_lod.drop-1.5_5.batches.csv
    Untracked:  data/ici.vs.eoi_genes_chr-3_peak.marker-UNC4945186_lod.drop-1.5_5.batches_mis.csv
    Untracked:  data/ici.vs.eoi_genes_chr-3_peak.marker-UNC4947492_lod.drop-1.5_5.batches_mis.csv
    Untracked:  data/ici.vs.eoi_genes_chr-3_peak.marker-UNC4957168_lod.drop-1.5_5.batches.csv
    Untracked:  data/ici.vs.eoi_genes_chr-3_peak.marker-UNC4957859_lod.drop-1.5_5.batches_mis.csv
    Untracked:  data/ici.vs.eoi_genes_chr-3_peak.marker-UNC4964062_lod.drop-1.5_5.batches_mis.csv
    Untracked:  data/ici.vs.eoi_genes_chr-3_peak.marker-UNC4979914_lod.drop-1.5_5.batches_mis.csv
    Untracked:  data/ici.vs.eoi_genes_chr-3_peak.marker-UNC4985064_lod.drop-1.5_5.batches_mis.csv
    Untracked:  data/ici.vs.eoi_genes_chr-3_peak.marker-UNC4990111_lod.drop-1.5_5.batches_mis.csv
    Untracked:  data/ici.vs.eoi_genes_chr-3_peak.marker-UNC4991565_lod.drop-1.5_5.batches_mis.csv
    Untracked:  data/ici.vs.eoi_genes_chr-3_peak.marker-UNC5001723_lod.drop-1.5_5.batches_mis.csv
    Untracked:  data/ici.vs.eoi_genes_chr-3_peak.marker-UNC5008656_lod.drop-1.5_5.batches.csv
    Untracked:  data/ici.vs.eoi_genes_chr-3_peak.marker-UNC5008656_lod.drop-1.5_5.batches_mis.csv
    Untracked:  data/ici.vs.eoi_genes_chr-3_peak.marker-UNC5042361_lod.drop-1.5_5.batches_mis.csv
    Untracked:  data/ici.vs.eoi_genes_chr-3_peak.marker-UNC5055259_lod.drop-1.5_5.batches_mis.csv
    Untracked:  data/ici.vs.eoi_genes_chr-3_peak.marker-UNC5076935_lod.drop-1.5_5.batches_mis.csv
    Untracked:  data/ici.vs.eoi_genes_chr-3_peak.marker-UNC5082757_lod.drop-1.5_5.batches_mis.csv
    Untracked:  data/ici.vs.eoi_genes_chr-3_peak.marker-UNC5083168_lod.drop-1.5_5.batches.csv
    Untracked:  data/ici.vs.eoi_genes_chr-3_peak.marker-UNC5173293_lod.drop-1.5_5.batches_mis.csv
    Untracked:  data/ici.vs.eoi_genes_chr-3_peak.marker-UNC5182782_lod.drop-1.5_5.batches_mis.csv
    Untracked:  data/ici.vs.eoi_genes_chr-3_peak.marker-UNC5192582_lod.drop-1.5_5.batches.csv
    Untracked:  data/ici.vs.eoi_genes_chr-3_peak.marker-UNC5193970_lod.drop-1.5_5.batches_mis.csv
    Untracked:  data/ici.vs.eoi_genes_chr-3_peak.marker-UNC5213888_lod.drop-1.5_snpsqc_5.batches_mis.csv
    Untracked:  data/ici.vs.eoi_genes_chr-3_peak.marker-UNC5240484_lod.drop-1.5_5.batches_mis.csv
    Untracked:  data/ici.vs.eoi_genes_chr-3_peak.marker-UNC5256922_lod.drop-1.5_5.batches_mis.csv
    Untracked:  data/ici.vs.eoi_genes_chr-3_peak.marker-UNC5266664_lod.drop-1.5_5.batches_mis.csv
    Untracked:  data/ici.vs.eoi_genes_chr-3_peak.marker-UNC5266711_lod.drop-1.5_5.batches.csv
    Untracked:  data/ici.vs.eoi_genes_chr-3_peak.marker-UNC5267813_lod.drop-1.5_5.batches_mis.csv
    Untracked:  data/ici.vs.eoi_genes_chr-3_peak.marker-UNC5274554_lod.drop-1.5_5.batches_mis.csv
    Untracked:  data/ici.vs.eoi_genes_chr-3_peak.marker-UNC5277061_lod.drop-1.5_5.batches_mis.csv
    Untracked:  data/ici.vs.eoi_genes_chr-3_peak.marker-UNC5282259_lod.drop-1.5_5.batches_mis.csv
    Untracked:  data/ici.vs.eoi_genes_chr-3_peak.marker-UNC5287977_lod.drop-1.5_5.batches_mis.csv
    Untracked:  data/ici.vs.eoi_genes_chr-3_peak.marker-UNC5301460_lod.drop-1.5_5.batches_mis.csv
    Untracked:  data/ici.vs.eoi_genes_chr-3_peak.marker-UNC5321559_lod.drop-1.5_5.batches_mis.csv
    Untracked:  data/ici.vs.eoi_genes_chr-3_peak.marker-UNC5322455_lod.drop-1.5_5.batches_mis.csv
    Untracked:  data/ici.vs.eoi_genes_chr-3_peak.marker-UNC5328596_lod.drop-1.5_5.batches_mis.csv
    Untracked:  data/ici.vs.eoi_genes_chr-3_peak.marker-UNC5350384_lod.drop-1.5_5.batches_mis.csv
    Untracked:  data/ici.vs.eoi_genes_chr-3_peak.marker-UNC5354724_lod.drop-1.5_5.batches_mis.csv
    Untracked:  data/ici.vs.eoi_genes_chr-3_peak.marker-UNC5375208_lod.drop-1.5_5.batches_mis.csv
    Untracked:  data/ici.vs.eoi_genes_chr-3_peak.marker-UNC5384072_lod.drop-1.5_5.batches_mis.csv
    Untracked:  data/ici.vs.eoi_genes_chr-3_peak.marker-UNC5400314_lod.drop-1.5_5.batches_mis.csv
    Untracked:  data/ici.vs.eoi_genes_chr-3_peak.marker-UNC5433945_lod.drop-1.5_5.batches_mis.csv
    Untracked:  data/ici.vs.eoi_genes_chr-3_peak.marker-UNC5435874_lod.drop-1.5_5.batches_mis.csv
    Untracked:  data/ici.vs.eoi_genes_chr-3_peak.marker-UNC5454839_lod.drop-1.5_5.batches.csv
    Untracked:  data/ici.vs.eoi_genes_chr-3_peak.marker-UNC5454839_lod.drop-1.5_5.batches_mis.csv
    Untracked:  data/ici.vs.eoi_genes_chr-3_peak.marker-UNC5466669_lod.drop-1.5_5.batches_mis.csv
    Untracked:  data/ici.vs.eoi_genes_chr-3_peak.marker-UNC5517972_lod.drop-1.5_5.batches_mis.csv
    Untracked:  data/ici.vs.eoi_genes_chr-3_peak.marker-UNC5522257_lod.drop-1.5_5.batches_mis.csv
    Untracked:  data/ici.vs.eoi_genes_chr-3_peak.marker-UNC5523265_lod.drop-1.5_5.batches_mis.csv
    Untracked:  data/ici.vs.eoi_genes_chr-3_peak.marker-UNC5532424_lod.drop-1.5_5.batches_mis.csv
    Untracked:  data/ici.vs.eoi_genes_chr-3_peak.marker-UNC5553882_lod.drop-1.5_5.batches_mis.csv
    Untracked:  data/ici.vs.eoi_genes_chr-3_peak.marker-UNC5572051_lod.drop-1.5_5.batches_mis.csv
    Untracked:  data/ici.vs.eoi_genes_chr-3_peak.marker-UNC5582753_lod.drop-1.5_5.batches.csv
    Untracked:  data/ici.vs.eoi_genes_chr-3_peak.marker-UNC5589260_lod.drop-1.5_5.batches_mis.csv
    Untracked:  data/ici.vs.eoi_genes_chr-3_peak.marker-UNC5595555_lod.drop-1.5_5.batches_mis.csv
    Untracked:  data/ici.vs.eoi_genes_chr-3_peak.marker-UNC5623239_lod.drop-1.5_5.batches_mis.csv
    Untracked:  data/ici.vs.eoi_genes_chr-3_peak.marker-UNC5639830_lod.drop-1.5_5.batches_mis.csv
    Untracked:  data/ici.vs.eoi_genes_chr-3_peak.marker-UNC5645844_lod.drop-1.5_5.batches.csv
    Untracked:  data/ici.vs.eoi_genes_chr-3_peak.marker-UNC5645844_lod.drop-1.5_5.batches_mis.csv
    Untracked:  data/ici.vs.eoi_genes_chr-3_peak.marker-UNC5680234_lod.drop-1.5_5.batches_mis.csv
    Untracked:  data/ici.vs.eoi_genes_chr-3_peak.marker-UNC5682314_lod.drop-1.5_5.batches_mis.csv
    Untracked:  data/ici.vs.eoi_genes_chr-3_peak.marker-UNC5725356_lod.drop-1.5_5.batches_mis.csv
    Untracked:  data/ici.vs.eoi_genes_chr-3_peak.marker-UNC5727351_lod.drop-1.5_5.batches_mis.csv
    Untracked:  data/ici.vs.eoi_genes_chr-3_peak.marker-UNC5734420_lod.drop-1.5_5.batches_mis.csv
    Untracked:  data/ici.vs.eoi_genes_chr-3_peak.marker-UNC5743257_lod.drop-1.5_5.batches_mis.csv
    Untracked:  data/ici.vs.eoi_genes_chr-3_peak.marker-UNC5751567_lod.drop-1.5_5.batches_mis.csv
    Untracked:  data/ici.vs.eoi_genes_chr-3_peak.marker-UNC5752623_lod.drop-1.5_5.batches_mis.csv
    Untracked:  data/ici.vs.eoi_genes_chr-3_peak.marker-UNC5776974_lod.drop-1.5_5.batches_mis.csv
    Untracked:  data/ici.vs.eoi_genes_chr-3_peak.marker-UNC5778977_lod.drop-1.5_5.batches_mis.csv
    Untracked:  data/ici.vs.eoi_genes_chr-3_peak.marker-UNC5788507_lod.drop-1.5_5.batches_mis.csv
    Untracked:  data/ici.vs.eoi_genes_chr-3_peak.marker-UNC5792468_lod.drop-1.5_5.batches_mis.csv
    Untracked:  data/ici.vs.eoi_genes_chr-3_peak.marker-UNC5794598_lod.drop-1.5_5.batches_mis.csv
    Untracked:  data/ici.vs.eoi_genes_chr-3_peak.marker-UNC5795561_lod.drop-1.5_5.batches_mis.csv
    Untracked:  data/ici.vs.eoi_genes_chr-3_peak.marker-UNC5802298_lod.drop-1.5_5.batches_mis.csv
    Untracked:  data/ici.vs.eoi_genes_chr-3_peak.marker-UNC5806628_lod.drop-1.5_5.batches_mis.csv
    Untracked:  data/ici.vs.eoi_genes_chr-3_peak.marker-UNC5808246_lod.drop-1.5_5.batches.csv
    Untracked:  data/ici.vs.eoi_genes_chr-3_peak.marker-UNC5817478_lod.drop-1.5_5.batches_mis.csv
    Untracked:  data/ici.vs.eoi_genes_chr-3_peak.marker-UNC5836999_lod.drop-1.5_5.batches_mis.csv
    Untracked:  data/ici.vs.eoi_genes_chr-3_peak.marker-UNC5841501_lod.drop-1.5_5.batches_mis.csv
    Untracked:  data/ici.vs.eoi_genes_chr-3_peak.marker-UNC5847927_lod.drop-1.5_5.batches_mis.csv
    Untracked:  data/ici.vs.eoi_genes_chr-3_peak.marker-UNC5966417_lod.drop-1.5_5.batches_mis.csv
    Untracked:  data/ici.vs.eoi_genes_chr-3_peak.marker-UNCHS008129_lod.drop-1.5_5.batches_mis.csv
    Untracked:  data/ici.vs.eoi_genes_chr-3_peak.marker-UNCHS008173_lod.drop-1.5_5.batches_mis.csv
    Untracked:  data/ici.vs.eoi_genes_chr-3_peak.marker-UNCHS008182_lod.drop-1.5_5.batches.csv
    Untracked:  data/ici.vs.eoi_genes_chr-3_peak.marker-UNCHS008193_lod.drop-1.5_5.batches_mis.csv
    Untracked:  data/ici.vs.eoi_genes_chr-3_peak.marker-UNCHS008209_lod.drop-1.5_5.batches_mis.csv
    Untracked:  data/ici.vs.eoi_genes_chr-3_peak.marker-UNCHS008215_lod.drop-1.5_5.batches_mis.csv
    Untracked:  data/ici.vs.eoi_genes_chr-3_peak.marker-UNCHS008219_lod.drop-1.5_5.batches_mis.csv
    Untracked:  data/ici.vs.eoi_genes_chr-3_peak.marker-UNCHS008231_lod.drop-1.5_5.batches_mis.csv
    Untracked:  data/ici.vs.eoi_genes_chr-3_peak.marker-UNCHS008235_lod.drop-1.5_5.batches_mis.csv
    Untracked:  data/ici.vs.eoi_genes_chr-3_peak.marker-UNCHS008242_lod.drop-1.5_5.batches.csv
    Untracked:  data/ici.vs.eoi_genes_chr-3_peak.marker-UNCHS008253_lod.drop-1.5_5.batches_mis.csv
    Untracked:  data/ici.vs.eoi_genes_chr-3_peak.marker-UNCHS008256_lod.drop-1.5_5.batches_mis.csv
    Untracked:  data/ici.vs.eoi_genes_chr-3_peak.marker-UNCHS008259_lod.drop-1.5_5.batches_mis.csv
    Untracked:  data/ici.vs.eoi_genes_chr-3_peak.marker-UNCHS008271_lod.drop-1.5_5.batches_mis.csv
    Untracked:  data/ici.vs.eoi_genes_chr-3_peak.marker-UNCHS008286_lod.drop-1.5_5.batches.csv
    Untracked:  data/ici.vs.eoi_genes_chr-3_peak.marker-UNCHS008286_lod.drop-1.5_5.batches_mis.csv
    Untracked:  data/ici.vs.eoi_genes_chr-3_peak.marker-UNCHS008311_lod.drop-1.5_5.batches_mis.csv
    Untracked:  data/ici.vs.eoi_genes_chr-3_peak.marker-UNCHS008317_lod.drop-1.5_5.batches_mis.csv
    Untracked:  data/ici.vs.eoi_genes_chr-3_peak.marker-UNCHS008372_lod.drop-1.5_5.batches_mis.csv
    Untracked:  data/ici.vs.eoi_genes_chr-3_peak.marker-UNCHS008379_lod.drop-1.5_5.batches_mis.csv
    Untracked:  data/ici.vs.eoi_genes_chr-3_peak.marker-UNCHS008410_lod.drop-1.5_5.batches_mis.csv
    Untracked:  data/ici.vs.eoi_genes_chr-3_peak.marker-UNCHS008415_lod.drop-1.5_5.batches_mis.csv
    Untracked:  data/ici.vs.eoi_genes_chr-3_peak.marker-UNCHS008431_lod.drop-1.5_5.batches_mis.csv
    Untracked:  data/ici.vs.eoi_genes_chr-3_peak.marker-UNCHS008487_lod.drop-1.5_snpsqc_5.batches.csv
    Untracked:  data/ici.vs.eoi_genes_chr-3_peak.marker-UNCHS008487_lod.drop-1.5_snpsqc_dis_no-x_updated_5.batches.csv
    Untracked:  data/ici.vs.eoi_genes_chr-3_peak.marker-UNCHS008487_lod.drop-1.5_snpsqc_dis_no-x_updated_5.batches_mis.csv
    Untracked:  data/ici.vs.eoi_genes_chr-3_peak.marker-UNCHS008511_lod.drop-1.5_snpsqc_5.batches_mis.csv
    Untracked:  data/ici.vs.eoi_genes_chr-3_peak.marker-UNCHS008609_lod.drop-1.5_5.batches_mis.csv
    Untracked:  data/ici.vs.eoi_genes_chr-3_peak.marker-UNCHS008614_lod.drop-1.5_5.batches.csv
    Untracked:  data/ici.vs.eoi_genes_chr-3_peak.marker-UNCHS008649_lod.drop-1.5_5.batches_mis.csv
    Untracked:  data/ici.vs.eoi_genes_chr-3_peak.marker-UNCHS008651_lod.drop-1.5_5.batches.csv
    Untracked:  data/ici.vs.eoi_genes_chr-3_peak.marker-UNCHS008656_lod.drop-1.5_5.batches_mis.csv
    Untracked:  data/ici.vs.eoi_genes_chr-3_peak.marker-UNCHS008723_lod.drop-1.5_5.batches_mis.csv
    Untracked:  data/ici.vs.eoi_genes_chr-3_peak.marker-UNCHS008802_lod.drop-1.5_5.batches_mis.csv
    Untracked:  data/ici.vs.eoi_genes_chr-3_peak.marker-UNCHS008803_lod.drop-1.5_5.batches_mis.csv
    Untracked:  data/ici.vs.eoi_genes_chr-3_peak.marker-UNCHS008808_lod.drop-1.5_5.batches_mis.csv
    Untracked:  data/ici.vs.eoi_genes_chr-3_peak.marker-UNCHS008815_lod.drop-1.5_5.batches.csv
    Untracked:  data/ici.vs.eoi_genes_chr-3_peak.marker-UNCHS008815_lod.drop-1.5_5.batches_mis.csv
    Untracked:  data/ici.vs.eoi_genes_chr-3_peak.marker-UNCHS008834_lod.drop-1.5_5.batches_mis.csv
    Untracked:  data/ici.vs.eoi_genes_chr-3_peak.marker-UNCHS008838_lod.drop-1.5_5.batches_mis.csv
    Untracked:  data/ici.vs.eoi_genes_chr-3_peak.marker-UNCHS008914_lod.drop-1.5_5.batches_mis.csv
    Untracked:  data/ici.vs.eoi_genes_chr-3_peak.marker-UNCHS008928_lod.drop-1.5_5.batches_mis.csv
    Untracked:  data/ici.vs.eoi_genes_chr-3_peak.marker-UNCHS008948_lod.drop-1.5_5.batches_mis.csv
    Untracked:  data/ici.vs.eoi_genes_chr-3_peak.marker-UNCHS008959_lod.drop-1.5_5.batches_mis.csv
    Untracked:  data/ici.vs.eoi_genes_chr-3_peak.marker-UNCHS008973_lod.drop-1.5_5.batches_mis.csv
    Untracked:  data/ici.vs.eoi_genes_chr-3_peak.marker-UNCHS008987_lod.drop-1.5_5.batches_mis.csv
    Untracked:  data/ici.vs.eoi_genes_chr-3_peak.marker-UNCHS009001_lod.drop-1.5_5.batches_mis.csv
    Untracked:  data/ici.vs.eoi_genes_chr-3_peak.marker-UNCHS009025_lod.drop-1.5_5.batches_mis.csv
    Untracked:  data/ici.vs.eoi_genes_chr-3_peak.marker-UNCHS009034_lod.drop-1.5_5.batches_mis.csv
    Untracked:  data/ici.vs.eoi_genes_chr-3_peak.marker-UNCHS009087_lod.drop-1.5_5.batches_mis.csv
    Untracked:  data/ici.vs.eoi_genes_chr-3_peak.marker-UNCHS009109_lod.drop-1.5_5.batches_mis.csv
    Untracked:  data/ici.vs.eoi_genes_chr-3_peak.marker-UNCHS009142_lod.drop-1.5_5.batches_mis.csv
    Untracked:  data/ici.vs.eoi_genes_chr-3_peak.marker-UNCHS009207_lod.drop-1.5_5.batches_mis.csv
    Untracked:  data/ici.vs.eoi_genes_chr-3_peak.marker-UNCHS009216_lod.drop-1.5_5.batches_mis.csv
    Untracked:  data/ici.vs.eoi_genes_chr-3_peak.marker-UNCHS009224_lod.drop-1.5_5.batches.csv
    Untracked:  data/ici.vs.eoi_genes_chr-3_peak.marker-UNCHS009306_lod.drop-1.5_5.batches_mis.csv
    Untracked:  data/ici.vs.eoi_genes_chr-3_peak.marker-UNCHS009316_lod.drop-1.5_5.batches_mis.csv
    Untracked:  data/ici.vs.eoi_genes_chr-3_peak.marker-UNCHS009364_lod.drop-1.5_5.batches_mis.csv
    Untracked:  data/ici.vs.eoi_genes_chr-3_peak.marker-UNCHS009377_lod.drop-1.5_5.batches_mis.csv
    Untracked:  data/ici.vs.eoi_genes_chr-3_peak.marker-UNCHS009382_lod.drop-1.5_5.batches_mis.csv
    Untracked:  data/ici.vs.eoi_genes_chr-3_peak.marker-UNCHS009383_lod.drop-1.5_5.batches.csv
    Untracked:  data/ici.vs.eoi_genes_chr-3_peak.marker-UNCHS009387_lod.drop-1.5_5.batches_mis.csv
    Untracked:  data/ici.vs.eoi_genes_chr-3_peak.marker-UNCHS009395_lod.drop-1.5_5.batches_mis.csv
    Untracked:  data/ici.vs.eoi_genes_chr-3_peak.marker-UNCHS009413_lod.drop-1.5_5.batches_mis.csv
    Untracked:  data/ici.vs.eoi_genes_chr-3_peak.marker-UNCHS009428_lod.drop-1.5_5.batches_mis.csv
    Untracked:  data/ici.vs.eoi_genes_chr-3_peak.marker-UNCHS009463_lod.drop-1.5_5.batches_mis.csv
    Untracked:  data/ici.vs.eoi_genes_chr-3_peak.marker-UNCHS009503_lod.drop-1.5_5.batches_mis.csv
    Untracked:  data/ici.vs.eoi_genes_chr-3_peak.marker-UNCHS009532_lod.drop-1.5_5.batches_mis.csv
    Untracked:  data/ici.vs.eoi_genes_chr-3_peak.marker-UNCHS009541_lod.drop-1.5_5.batches_mis.csv
    Untracked:  data/ici.vs.eoi_genes_chr-3_peak.marker-UNCHS009560_lod.drop-1.5_5.batches_mis.csv
    Untracked:  data/ici.vs.eoi_genes_chr-3_peak.marker-UNCHS009574_lod.drop-1.5_5.batches_mis.csv
    Untracked:  data/ici.vs.eoi_genes_chr-3_peak.marker-UNCJPD001170_lod.drop-1.5_5.batches_mis.csv
    Untracked:  data/ici.vs.eoi_genes_chr-3_peak.marker-UNCJPD001198_lod.drop-1.5_5.batches_mis.csv
    Untracked:  data/ici.vs.eoi_genes_chr-3_peak.marker-UNCJPD001276_lod.drop-1.5_5.batches_mis.csv
    Untracked:  data/ici.vs.eoi_genes_chr-3_peak.marker-UNCJPD001315_lod.drop-1.5_5.batches_mis.csv
    Untracked:  data/ici.vs.eoi_genes_chr-3_peak.marker-UNCJPD001355_lod.drop-1.5_5.batches_mis.csv
    Untracked:  data/ici.vs.eoi_genes_chr-3_peak.marker-UNCJPD001373_lod.drop-1.5_5.batches_mis.csv
    Untracked:  data/ici.vs.eoi_genes_chr-3_peak.marker-UNCJPD001382_lod.drop-1.5_5.batches_mis.csv
    Untracked:  data/ici.vs.eoi_genes_chr-4_peak.marker-UNC6851205_lod.drop-1.5_5.batches_mis.csv
    Untracked:  data/ici.vs.eoi_genes_chr-4_peak.marker-UNC8250659_lod.drop-1.5_snpsqc_dis_no-x_updated_5.batches.csv
    Untracked:  data/ici.vs.eoi_genes_chr-4_peak.marker-UNC8299912_lod.drop-1.5_5.batches_mis.csv
    Untracked:  data/ici.vs.eoi_genes_chr-4_peak.marker-UNC8300499_lod.drop-1.5_5.batches.csv
    Untracked:  data/ici.vs.eoi_genes_chr-4_peak.marker-UNC8381981_lod.drop-1.5_5.batches_mis.csv
    Untracked:  data/ici.vs.eoi_genes_chr-4_peak.marker-UNC8436434_lod.drop-1.5_5.batches_mis.csv
    Untracked:  data/ici.vs.eoi_genes_chr-4_peak.marker-UNC8439633_lod.drop-1.5_5.batches_mis.csv
    Untracked:  data/ici.vs.eoi_genes_chr-4_peak.marker-UNC8530763_lod.drop-1.5_5.batches_mis.csv
    Untracked:  data/ici.vs.eoi_genes_chr-4_peak.marker-UNCHS012948_lod.drop-1.5_snpsqc_dis_no-x_updated_5.batches_mis.csv
    Untracked:  data/ici.vs.eoi_genes_chr-4_peak.marker-UNCHS012976_lod.drop-1.5_5.batches_mis.csv
    Untracked:  data/ici.vs.eoi_genes_chr-4_peak.marker-UNCHS013118_lod.drop-1.5_5.batches.csv
    Untracked:  data/ici.vs.eoi_genes_chr-4_peak.marker-UNCHS013118_lod.drop-1.5_snpsqc_5.batches.csv
    Untracked:  data/ici.vs.eoi_genes_chr-4_peak.marker-UNCHS013118_lod.drop-1.5_snpsqc_5.batches_mis.csv
    Untracked:  data/ici.vs.eoi_genes_chr-4_peak.marker-UNCHS013130_lod.drop-1.5_5.batches_mis.csv
    Untracked:  data/ici.vs.eoi_genes_chr-4_peak.marker-UNCHS013139_lod.drop-1.5_5.batches_mis.csv
    Untracked:  data/ici.vs.eoi_genes_chr-4_peak.marker-UNCHS013304_lod.drop-1.5_5.batches.csv
    Untracked:  data/ici.vs.eoi_genes_chr-5_peak.marker-JAX00590770_lod.drop-1.5_5.batches_mis.csv
    Untracked:  data/ici.vs.eoi_genes_chr-5_peak.marker-UNC10044126_lod.drop-1.5_5.batches_mis.csv
    Untracked:  data/ici.vs.eoi_genes_chr-7_peak.marker-ICR1830_lod.drop-1.5_5.batches_mis.csv
    Untracked:  data/ici.vs.eoi_genes_chr-7_peak.marker-UNC13170794_lod.drop-1.5_5.batches_mis.csv
    Untracked:  data/ici.vs.eoi_genes_chr-7_peak.marker-UNCHS019480_lod.drop-1.5_5.batches.csv
    Untracked:  data/ici.vs.eoi_genes_chr-7_peak.marker-UNCHS020066_lod.drop-1.5_snpsqc_5.batches_mis.csv
    Untracked:  data/ici.vs.eoi_genes_chr-7_peak.marker-UNCHS020066_lod.drop-1.5_snpsqc_dis_no-x_updated_5.batches_mis.csv
    Untracked:  data/ici.vs.eoi_genes_chr-7_peak.marker-cr31snv87_lod.drop-1.5_5.batches_mis.csv
    Untracked:  data/ici.vs.eoi_genes_chr-8_peak.marker-JAX00667121_lod.drop-1.5_5.batches_mis.csv
    Untracked:  data/ici.vs.eoi_genes_chr-8_peak.marker-UNC15430711_lod.drop-1.5_5.batches_mis.csv
    Untracked:  data/ici.vs.eoi_genes_chr-8_peak.marker-UNC15524531_lod.drop-1.5_5.batches_mis.csv
    Untracked:  data/ici.vs.eoi_genes_chr-8_peak.marker-UNC15524531_lod.drop-1.5_snpsqc_5.batches_mis.csv
    Untracked:  data/ici.vs.eoi_genes_chr-8_peak.marker-UNC15524531_lod.drop-1.5_snpsqc_dis_no-x_updated_5.batches_mis.csv
    Untracked:  data/ici.vs.eoi_genes_chr-9_peak.marker-UNC17203329_lod.drop-1.5_5.batches_mis.csv
    Untracked:  data/ici.vs.eoi_genes_chr-9_peak.marker-UNC17203597_lod.drop-1.5_snpsqc_5.batches_mis.csv
    Untracked:  data/ici.vs.eoi_genes_chr-9_peak.marker-UNCHS024593_lod.drop-1.5_5.batches_mis.csv
    Untracked:  data/ici.vs.eoi_genes_chr-X_peak.marker-UNC30627130_lod.drop-1.5_5.batches_mis.csv
    Untracked:  data/ici.vs.eoi_genes_chr-X_peak.marker-UNCHS048313_lod.drop-1.5_5.batches_mis.csv
    Untracked:  data/ici.vs.eoi_genes_chr-X_peak.marker-UNCHS048314_lod.drop-1.5_snpsqc_5.batches_mis.csv
    Untracked:  data/ici.vs.eoi_genes_chr-X_peak.marker-UNCHS048314_lod.drop-1.5_snpsqc_dis_no-x_updated_5.batches_mis.csv
    Untracked:  data/ici.vs.eoi_gm_qtl_5.batches.csv
    Untracked:  data/ici.vs.eoi_gm_qtl_5.batches_mis.csv
    Untracked:  data/ici.vs.eoi_gm_qtl_snpsqc_5.batches.csv
    Untracked:  data/ici.vs.eoi_gm_qtl_snpsqc_5.batches_mis.csv
    Untracked:  data/ici.vs.eoi_gm_qtl_snpsqc_dis_no-x_updated_5.batches.csv
    Untracked:  data/ici.vs.eoi_gm_qtl_snpsqc_dis_no-x_updated_5.batches_mis.csv
    Untracked:  data/ici.vs.eoi_marker.freq_low.geno.freq.removed_geno.ratio.csv
    Untracked:  data/ici.vs.eoi_marker.freq_low.geno.freq.removed_geno.ratio_5.batches.csv
    Untracked:  data/ici.vs.eoi_marker.freq_low.geno.freq.removed_geno.ratio_5.batches_0.csv
    Untracked:  data/ici.vs.eoi_marker.freq_low.geno.freq.removed_geno.ratio_5.batches_52.csv
    Untracked:  data/ici.vs.eoi_marker.freq_low.geno.freq.removed_geno.ratio_5.batches_mis.csv
    Untracked:  data/ici.vs.eoi_marker.freq_low.geno.freq.removed_geno.ratio_5.batches_mis_0.csv
    Untracked:  data/ici.vs.eoi_marker.freq_low.geno.freq.removed_geno.ratio_5.batches_mis_52.csv
    Untracked:  data/ici.vs.eoi_marker.freq_low.geno.freq.removed_sample.outliers.removed_geno.ratio.csv
    Untracked:  data/ici.vs.eoi_marker.freq_low.geno.freq.removed_sample.outliers.removed_geno.ratio_5.batches.csv
    Untracked:  data/ici.vs.eoi_marker.freq_low.geno.freq.removed_sample.outliers.removed_geno.ratio_5.batches_0.csv
    Untracked:  data/ici.vs.eoi_marker.freq_low.geno.freq.removed_sample.outliers.removed_geno.ratio_5.batches_52.csv
    Untracked:  data/ici.vs.eoi_marker.freq_low.geno.freq.removed_sample.outliers.removed_geno.ratio_5.batches_mis.csv
    Untracked:  data/ici.vs.eoi_marker.freq_low.geno.freq.removed_sample.outliers.removed_geno.ratio_5.batches_mis_0.csv
    Untracked:  data/ici.vs.eoi_marker.freq_low.geno.freq.removed_sample.outliers.removed_geno.ratio_5.batches_mis_52.csv
    Untracked:  data/ici.vs.eoi_marker.freq_low.probs.freq.removed_geno.ratio.csv
    Untracked:  data/ici.vs.eoi_marker.freq_low.probs.freq.removed_geno.ratio_5.batches.csv
    Untracked:  data/ici.vs.eoi_marker.freq_low.probs.freq.removed_geno.ratio_5.batches_0.csv
    Untracked:  data/ici.vs.eoi_marker.freq_low.probs.freq.removed_geno.ratio_5.batches_52.csv
    Untracked:  data/ici.vs.eoi_marker.freq_low.probs.freq.removed_geno.ratio_5.batches_mis.csv
    Untracked:  data/ici.vs.eoi_marker.freq_low.probs.freq.removed_geno.ratio_5.batches_mis_0.csv
    Untracked:  data/ici.vs.eoi_marker.freq_low.probs.freq.removed_geno.ratio_5.batches_mis_52.csv
    Untracked:  data/ici.vs.eoi_marker.freq_low.probs.freq.removed_sample.outliers.removed_geno.ratio.csv
    Untracked:  data/ici.vs.eoi_marker.freq_low.probs.freq.removed_sample.outliers.removed_geno.ratio_5.batches.csv
    Untracked:  data/ici.vs.eoi_marker.freq_low.probs.freq.removed_sample.outliers.removed_geno.ratio_5.batches_0.csv
    Untracked:  data/ici.vs.eoi_marker.freq_low.probs.freq.removed_sample.outliers.removed_geno.ratio_5.batches_52.csv
    Untracked:  data/ici.vs.eoi_marker.freq_low.probs.freq.removed_sample.outliers.removed_geno.ratio_5.batches_mis.csv
    Untracked:  data/ici.vs.eoi_marker.freq_low.probs.freq.removed_sample.outliers.removed_geno.ratio_5.batches_mis_0.csv
    Untracked:  data/ici.vs.eoi_marker.freq_low.probs.freq.removed_sample.outliers.removed_geno.ratio_5.batches_mis_52.csv
    Untracked:  data/ici.vs.eoi_rz.age-additive.covariates_blup_sub_chr-7_peak.marker-UNC13239362_lod.drop-1.5_snpsqc_5.batches.csv
    Untracked:  data/ici.vs.eoi_rz.age-additive.covariates_genes_chr-7_peak.marker-UNC13239362_lod.drop-1.5_snpsqc_5.batches.csv
    Untracked:  data/ici.vs.eoi_rz.age-no.covariates_blup_sub_chr-10_peak.marker-JAX00292927_lod.drop-1.5_5.batches_0.csv
    Untracked:  data/ici.vs.eoi_rz.age-no.covariates_blup_sub_chr-10_peak.marker-UNC18311938_lod.drop-1.5_5.batches_0.csv
    Untracked:  data/ici.vs.eoi_rz.age-no.covariates_blup_sub_chr-10_peak.marker-UNC18343181_lod.drop-1.5_5.batches_0.csv
    Untracked:  data/ici.vs.eoi_rz.age-no.covariates_blup_sub_chr-10_peak.marker-UNC18376338_lod.drop-1.5_5.batches_0.csv
    Untracked:  data/ici.vs.eoi_rz.age-no.covariates_blup_sub_chr-10_peak.marker-UNC18376338_lod.drop-1.5_snpsqc_5.batches_0.csv
    Untracked:  data/ici.vs.eoi_rz.age-no.covariates_blup_sub_chr-10_peak.marker-UNC18376338_lod.drop-1.5_snpsqc_5.batches_mis_0.csv
    Untracked:  data/ici.vs.eoi_rz.age-no.covariates_blup_sub_chr-10_peak.marker-UNC18382395_lod.drop-1.5_snpsqc_dis_no-x_updated_5.batches_0.csv
    Untracked:  data/ici.vs.eoi_rz.age-no.covariates_blup_sub_chr-10_peak.marker-UNCHS028236_lod.drop-1.5_snpsqc_5.batches_mis_0.csv
    Untracked:  data/ici.vs.eoi_rz.age-no.covariates_blup_sub_chr-10_peak.marker-UNCHS028236_lod.drop-1.5_snpsqc_5.batches_mis_52.csv
    Untracked:  data/ici.vs.eoi_rz.age-no.covariates_blup_sub_chr-10_peak.marker-UNCHS028536_lod.drop-1.5_snpsqc_dis_no-x_updated_5.batches_mis_0.csv
    Untracked:  data/ici.vs.eoi_rz.age-no.covariates_blup_sub_chr-10_peak.marker-UNCHS029024_lod.drop-1.5_5.batches_0.csv
    Untracked:  data/ici.vs.eoi_rz.age-no.covariates_blup_sub_chr-11_peak.marker-UNC19970181_lod.drop-1.5_snpsqc_5.batches_mis_0.csv
    Untracked:  data/ici.vs.eoi_rz.age-no.covariates_blup_sub_chr-11_peak.marker-UNC19970181_lod.drop-1.5_snpsqc_5.batches_mis_52.csv
    Untracked:  data/ici.vs.eoi_rz.age-no.covariates_blup_sub_chr-12_peak.marker-ICR499_lod.drop-1.5_snpsqc_5.batches_mis_0.csv
    Untracked:  data/ici.vs.eoi_rz.age-no.covariates_blup_sub_chr-12_peak.marker-ICR499_lod.drop-1.5_snpsqc_5.batches_mis_52.csv
    Untracked:  data/ici.vs.eoi_rz.age-no.covariates_blup_sub_chr-13_peak.marker-UNCHS036773_lod.drop-1.5_snpsqc_5.batches_mis_0.csv
    Untracked:  data/ici.vs.eoi_rz.age-no.covariates_blup_sub_chr-13_peak.marker-UNCHS036773_lod.drop-1.5_snpsqc_5.batches_mis_52.csv
    Untracked:  data/ici.vs.eoi_rz.age-no.covariates_blup_sub_chr-14_peak.marker-UNC24056202_lod.drop-1.5_snpsqc_5.batches_mis_0.csv
    Untracked:  data/ici.vs.eoi_rz.age-no.covariates_blup_sub_chr-14_peak.marker-UNC24056202_lod.drop-1.5_snpsqc_5.batches_mis_52.csv
    Untracked:  data/ici.vs.eoi_rz.age-no.covariates_blup_sub_chr-15_peak.marker-UNC26070435_lod.drop-1.5_snpsqc_dis_no-x_updated_5.batches_mis_0.csv
    Untracked:  data/ici.vs.eoi_rz.age-no.covariates_blup_sub_chr-15_peak.marker-UNC26070435_lod.drop-1.5_snpsqc_dis_no-x_updated_5.batches_mis_52.csv
    Untracked:  data/ici.vs.eoi_rz.age-no.covariates_blup_sub_chr-17_peak.marker-UNCHS044241_lod.drop-1.5_snpsqc_5.batches_mis_0.csv
    Untracked:  data/ici.vs.eoi_rz.age-no.covariates_blup_sub_chr-17_peak.marker-UNCHS044241_lod.drop-1.5_snpsqc_5.batches_mis_52.csv
    Untracked:  data/ici.vs.eoi_rz.age-no.covariates_blup_sub_chr-17_peak.marker-UNCHS044241_lod.drop-1.5_snpsqc_dis_no-x_updated_5.batches_mis_0.csv
    Untracked:  data/ici.vs.eoi_rz.age-no.covariates_blup_sub_chr-17_peak.marker-UNCrs47191360_lod.drop-1.5_snpsqc_dis_no-x_updated_5.batches_mis_52.csv
    Untracked:  data/ici.vs.eoi_rz.age-no.covariates_blup_sub_chr-18_peak.marker-UNCHS045343_lod.drop-1.5_snpsqc_5.batches_mis_0.csv
    Untracked:  data/ici.vs.eoi_rz.age-no.covariates_blup_sub_chr-18_peak.marker-UNCHS045343_lod.drop-1.5_snpsqc_5.batches_mis_52.csv
    Untracked:  data/ici.vs.eoi_rz.age-no.covariates_blup_sub_chr-1_peak.marker-UNC228500_lod.drop-1.5_snpsqc_dis_no-x_updated_5.batches_52.csv
    Untracked:  data/ici.vs.eoi_rz.age-no.covariates_blup_sub_chr-1_peak.marker-UNC228500_lod.drop-1.5_snpsqc_dis_no-x_updated_5.batches_mis_52.csv
    Untracked:  data/ici.vs.eoi_rz.age-no.covariates_blup_sub_chr-2_peak.marker-UNC4609527_lod.drop-1.5_snpsqc_dis_no-x_updated_5.batches_mis_0.csv
    Untracked:  data/ici.vs.eoi_rz.age-no.covariates_blup_sub_chr-2_peak.marker-UNC4609527_lod.drop-1.5_snpsqc_dis_no-x_updated_5.batches_mis_52.csv
    Untracked:  data/ici.vs.eoi_rz.age-no.covariates_blup_sub_chr-3_peak.marker-ICR1338_lod.drop-1.5_5.batches_0.csv
    Untracked:  data/ici.vs.eoi_rz.age-no.covariates_blup_sub_chr-3_peak.marker-ICR1338_lod.drop-1.5_5.batches_52.csv
    Untracked:  data/ici.vs.eoi_rz.age-no.covariates_blup_sub_chr-3_peak.marker-ICR2047_lod.drop-1.5_5.batches_52.csv
    Untracked:  data/ici.vs.eoi_rz.age-no.covariates_blup_sub_chr-3_peak.marker-JAX00106957_lod.drop-1.5_5.batches_0.csv
    Untracked:  data/ici.vs.eoi_rz.age-no.covariates_blup_sub_chr-3_peak.marker-JAX00112499_lod.drop-1.5_5.batches_52.csv
    Untracked:  data/ici.vs.eoi_rz.age-no.covariates_blup_sub_chr-3_peak.marker-JAX00219554_lod.drop-1.5_5.batches_0.csv
    Untracked:  data/ici.vs.eoi_rz.age-no.covariates_blup_sub_chr-3_peak.marker-JAX00220631_lod.drop-1.5_5.batches_0.csv
    Untracked:  data/ici.vs.eoi_rz.age-no.covariates_blup_sub_chr-3_peak.marker-JAX00514662_lod.drop-1.5_5.batches_52.csv
    Untracked:  data/ici.vs.eoi_rz.age-no.covariates_blup_sub_chr-3_peak.marker-JAX00529707_lod.drop-1.5_5.batches_52.csv
    Untracked:  data/ici.vs.eoi_rz.age-no.covariates_blup_sub_chr-3_peak.marker-UNC4813186_lod.drop-1.5_5.batches_0.csv
    Untracked:  data/ici.vs.eoi_rz.age-no.covariates_blup_sub_chr-3_peak.marker-UNC4813186_lod.drop-1.5_5.batches_52.csv
    Untracked:  data/ici.vs.eoi_rz.age-no.covariates_blup_sub_chr-3_peak.marker-UNC4836550_lod.drop-1.5_5.batches_52.csv
    Untracked:  data/ici.vs.eoi_rz.age-no.covariates_blup_sub_chr-3_peak.marker-UNC4837069_lod.drop-1.5_5.batches_0.csv
    Untracked:  data/ici.vs.eoi_rz.age-no.covariates_blup_sub_chr-3_peak.marker-UNC4912039_lod.drop-1.5_5.batches_0.csv
    Untracked:  data/ici.vs.eoi_rz.age-no.covariates_blup_sub_chr-3_peak.marker-UNC4912039_lod.drop-1.5_5.batches_52.csv
    Untracked:  data/ici.vs.eoi_rz.age-no.covariates_blup_sub_chr-3_peak.marker-UNC4924377_lod.drop-1.5_5.batches_0.csv
    Untracked:  data/ici.vs.eoi_rz.age-no.covariates_blup_sub_chr-3_peak.marker-UNC4924377_lod.drop-1.5_5.batches_52.csv
    Untracked:  data/ici.vs.eoi_rz.age-no.covariates_blup_sub_chr-3_peak.marker-UNC4956618_lod.drop-1.5_5.batches_52.csv
    Untracked:  data/ici.vs.eoi_rz.age-no.covariates_blup_sub_chr-3_peak.marker-UNC4957168_lod.drop-1.5_5.batches_0.csv
    Untracked:  data/ici.vs.eoi_rz.age-no.covariates_blup_sub_chr-3_peak.marker-UNC5006948_lod.drop-1.5_5.batches_52.csv
    Untracked:  data/ici.vs.eoi_rz.age-no.covariates_blup_sub_chr-3_peak.marker-UNC5008656_lod.drop-1.5_5.batches_0.csv
    Untracked:  data/ici.vs.eoi_rz.age-no.covariates_blup_sub_chr-3_peak.marker-UNC5083168_lod.drop-1.5_5.batches_0.csv
    Untracked:  data/ici.vs.eoi_rz.age-no.covariates_blup_sub_chr-3_peak.marker-UNC5083168_lod.drop-1.5_5.batches_52.csv
    Untracked:  data/ici.vs.eoi_rz.age-no.covariates_blup_sub_chr-3_peak.marker-UNC5192582_lod.drop-1.5_5.batches_0.csv
    Untracked:  data/ici.vs.eoi_rz.age-no.covariates_blup_sub_chr-3_peak.marker-UNC5213888_lod.drop-1.5_snpsqc_5.batches_mis_0.csv
    Untracked:  data/ici.vs.eoi_rz.age-no.covariates_blup_sub_chr-3_peak.marker-UNC5213888_lod.drop-1.5_snpsqc_5.batches_mis_52.csv
    Untracked:  data/ici.vs.eoi_rz.age-no.covariates_blup_sub_chr-3_peak.marker-UNC5262472_lod.drop-1.5_5.batches_0.csv
    Untracked:  data/ici.vs.eoi_rz.age-no.covariates_blup_sub_chr-3_peak.marker-UNC5262472_lod.drop-1.5_5.batches_52.csv
    Untracked:  data/ici.vs.eoi_rz.age-no.covariates_blup_sub_chr-3_peak.marker-UNC5266664_lod.drop-1.5_5.batches_0.csv
    Untracked:  data/ici.vs.eoi_rz.age-no.covariates_blup_sub_chr-3_peak.marker-UNC5300878_lod.drop-1.5_5.batches_52.csv
    Untracked:  data/ici.vs.eoi_rz.age-no.covariates_blup_sub_chr-3_peak.marker-UNC5360055_lod.drop-1.5_5.batches_52.csv
    Untracked:  data/ici.vs.eoi_rz.age-no.covariates_blup_sub_chr-3_peak.marker-UNC5454839_lod.drop-1.5_5.batches_0.csv
    Untracked:  data/ici.vs.eoi_rz.age-no.covariates_blup_sub_chr-3_peak.marker-UNC5454839_lod.drop-1.5_5.batches_52.csv
    Untracked:  data/ici.vs.eoi_rz.age-no.covariates_blup_sub_chr-3_peak.marker-UNC5645844_lod.drop-1.5_5.batches_0.csv
    Untracked:  data/ici.vs.eoi_rz.age-no.covariates_blup_sub_chr-3_peak.marker-UNC5645844_lod.drop-1.5_5.batches_52.csv
    Untracked:  data/ici.vs.eoi_rz.age-no.covariates_blup_sub_chr-3_peak.marker-UNC5679155_lod.drop-1.5_5.batches_52.csv
    Untracked:  data/ici.vs.eoi_rz.age-no.covariates_blup_sub_chr-3_peak.marker-UNC5808246_lod.drop-1.5_5.batches_52.csv
    Untracked:  data/ici.vs.eoi_rz.age-no.covariates_blup_sub_chr-3_peak.marker-UNC5945286_lod.drop-1.5_5.batches_52.csv
    Untracked:  data/ici.vs.eoi_rz.age-no.covariates_blup_sub_chr-3_peak.marker-UNC6227106_lod.drop-1.5_5.batches_52.csv
    Untracked:  data/ici.vs.eoi_rz.age-no.covariates_blup_sub_chr-3_peak.marker-UNCHS008182_lod.drop-1.5_5.batches_52.csv
    Untracked:  data/ici.vs.eoi_rz.age-no.covariates_blup_sub_chr-3_peak.marker-UNCHS008286_lod.drop-1.5_5.batches_0.csv
    Untracked:  data/ici.vs.eoi_rz.age-no.covariates_blup_sub_chr-3_peak.marker-UNCHS008286_lod.drop-1.5_5.batches_52.csv
    Untracked:  data/ici.vs.eoi_rz.age-no.covariates_blup_sub_chr-3_peak.marker-UNCHS008487_lod.drop-1.5_snpsqc_5.batches_0.csv
    Untracked:  data/ici.vs.eoi_rz.age-no.covariates_blup_sub_chr-3_peak.marker-UNCHS008487_lod.drop-1.5_snpsqc_5.batches_52.csv
    Untracked:  data/ici.vs.eoi_rz.age-no.covariates_blup_sub_chr-3_peak.marker-UNCHS008487_lod.drop-1.5_snpsqc_dis_no-x_updated_5.batches_0.csv
    Untracked:  data/ici.vs.eoi_rz.age-no.covariates_blup_sub_chr-3_peak.marker-UNCHS008487_lod.drop-1.5_snpsqc_dis_no-x_updated_5.batches_52.csv
    Untracked:  data/ici.vs.eoi_rz.age-no.covariates_blup_sub_chr-3_peak.marker-UNCHS008487_lod.drop-1.5_snpsqc_dis_no-x_updated_5.batches_mis_0.csv
    Untracked:  data/ici.vs.eoi_rz.age-no.covariates_blup_sub_chr-3_peak.marker-UNCHS008487_lod.drop-1.5_snpsqc_dis_no-x_updated_5.batches_mis_52.csv
    Untracked:  data/ici.vs.eoi_rz.age-no.covariates_blup_sub_chr-3_peak.marker-UNCHS008511_lod.drop-1.5_snpsqc_5.batches_mis_0.csv
    Untracked:  data/ici.vs.eoi_rz.age-no.covariates_blup_sub_chr-3_peak.marker-UNCHS008511_lod.drop-1.5_snpsqc_5.batches_mis_52.csv
    Untracked:  data/ici.vs.eoi_rz.age-no.covariates_blup_sub_chr-3_peak.marker-UNCHS008595_lod.drop-1.5_5.batches_52.csv
    Untracked:  data/ici.vs.eoi_rz.age-no.covariates_blup_sub_chr-3_peak.marker-UNCHS008613_lod.drop-1.5_5.batches_0.csv
    Untracked:  data/ici.vs.eoi_rz.age-no.covariates_blup_sub_chr-3_peak.marker-UNCHS008613_lod.drop-1.5_5.batches_52.csv
    Untracked:  data/ici.vs.eoi_rz.age-no.covariates_blup_sub_chr-3_peak.marker-UNCHS008815_lod.drop-1.5_5.batches_0.csv
    Untracked:  data/ici.vs.eoi_rz.age-no.covariates_blup_sub_chr-3_peak.marker-UNCHS008815_lod.drop-1.5_5.batches_52.csv
    Untracked:  data/ici.vs.eoi_rz.age-no.covariates_blup_sub_chr-3_peak.marker-UNCHS009067_lod.drop-1.5_5.batches_0.csv
    Untracked:  data/ici.vs.eoi_rz.age-no.covariates_blup_sub_chr-3_peak.marker-UNCHS009383_lod.drop-1.5_5.batches_52.csv
    Untracked:  data/ici.vs.eoi_rz.age-no.covariates_blup_sub_chr-3_peak.marker-UNCHS009693_lod.drop-1.5_5.batches_52.csv
    Untracked:  data/ici.vs.eoi_rz.age-no.covariates_blup_sub_chr-3_peak.marker-UNCJPD001246_lod.drop-1.5_5.batches_52.csv
    Untracked:  data/ici.vs.eoi_rz.age-no.covariates_blup_sub_chr-3_peak.marker-UNCJPD001341_lod.drop-1.5_5.batches_52.csv
    Untracked:  data/ici.vs.eoi_rz.age-no.covariates_blup_sub_chr-3_peak.marker-sanger2623_lod.drop-1.5_5.batches_52.csv
    Untracked:  data/ici.vs.eoi_rz.age-no.covariates_blup_sub_chr-4_peak.marker-UNCHS013125_lod.drop-1.5_snpsqc_dis_no-x_updated_5.batches_52.csv
    Untracked:  data/ici.vs.eoi_rz.age-no.covariates_blup_sub_chr-4_peak.marker-UNCHS013125_lod.drop-1.5_snpsqc_dis_no-x_updated_5.batches_mis_52.csv
    Untracked:  data/ici.vs.eoi_rz.age-no.covariates_blup_sub_chr-6_peak.marker-UNCHS018170_lod.drop-1.5_snpsqc_5.batches_0.csv
    Untracked:  data/ici.vs.eoi_rz.age-no.covariates_blup_sub_chr-6_peak.marker-UNCHS018231_lod.drop-1.5_snpsqc_5.batches_mis_0.csv
    Untracked:  data/ici.vs.eoi_rz.age-no.covariates_blup_sub_chr-7_peak.marker-UNCHS020066_lod.drop-1.5_snpsqc_5.batches_mis_0.csv
    Untracked:  data/ici.vs.eoi_rz.age-no.covariates_blup_sub_chr-7_peak.marker-UNCHS020066_lod.drop-1.5_snpsqc_5.batches_mis_52.csv
    Untracked:  data/ici.vs.eoi_rz.age-no.covariates_blup_sub_chr-7_peak.marker-UNCHS020066_lod.drop-1.5_snpsqc_dis_no-x_updated_5.batches_mis_0.csv
    Untracked:  data/ici.vs.eoi_rz.age-no.covariates_blup_sub_chr-7_peak.marker-UNCHS020066_lod.drop-1.5_snpsqc_dis_no-x_updated_5.batches_mis_52.csv
    Untracked:  data/ici.vs.eoi_rz.age-no.covariates_blup_sub_chr-8_peak.marker-UNC15524531_lod.drop-1.5_snpsqc_5.batches_mis_0.csv
    Untracked:  data/ici.vs.eoi_rz.age-no.covariates_blup_sub_chr-8_peak.marker-UNC15524531_lod.drop-1.5_snpsqc_5.batches_mis_52.csv
    Untracked:  data/ici.vs.eoi_rz.age-no.covariates_blup_sub_chr-8_peak.marker-UNC15524531_lod.drop-1.5_snpsqc_dis_no-x_updated_5.batches_mis_0.csv
    Untracked:  data/ici.vs.eoi_rz.age-no.covariates_blup_sub_chr-8_peak.marker-UNC15524531_lod.drop-1.5_snpsqc_dis_no-x_updated_5.batches_mis_52.csv
    Untracked:  data/ici.vs.eoi_rz.age-no.covariates_blup_sub_chr-9_peak.marker-UNC17203597_lod.drop-1.5_snpsqc_5.batches_mis_0.csv
    Untracked:  data/ici.vs.eoi_rz.age-no.covariates_blup_sub_chr-9_peak.marker-UNC17203597_lod.drop-1.5_snpsqc_5.batches_mis_52.csv
    Untracked:  data/ici.vs.eoi_rz.age-no.covariates_blup_sub_chr-X_peak.marker-UNCHS048314_lod.drop-1.5_snpsqc_5.batches_mis_0.csv
    Untracked:  data/ici.vs.eoi_rz.age-no.covariates_blup_sub_chr-X_peak.marker-UNCHS048314_lod.drop-1.5_snpsqc_5.batches_mis_52.csv
    Untracked:  data/ici.vs.eoi_rz.age-no.covariates_blup_sub_chr-X_peak.marker-UNCHS048314_lod.drop-1.5_snpsqc_dis_no-x_updated_5.batches_mis_0.csv
    Untracked:  data/ici.vs.eoi_rz.age-no.covariates_blup_sub_chr-X_peak.marker-UNCHS048314_lod.drop-1.5_snpsqc_dis_no-x_updated_5.batches_mis_52.csv
    Untracked:  data/ici.vs.eoi_rz.age-no.covariates_genes_chr-10_peak.marker-JAX00292927_lod.drop-1.5_5.batches_0.csv
    Untracked:  data/ici.vs.eoi_rz.age-no.covariates_genes_chr-10_peak.marker-UNC18311938_lod.drop-1.5_5.batches_0.csv
    Untracked:  data/ici.vs.eoi_rz.age-no.covariates_genes_chr-10_peak.marker-UNC18343181_lod.drop-1.5_5.batches_0.csv
    Untracked:  data/ici.vs.eoi_rz.age-no.covariates_genes_chr-10_peak.marker-UNC18376338_lod.drop-1.5_5.batches_0.csv
    Untracked:  data/ici.vs.eoi_rz.age-no.covariates_genes_chr-10_peak.marker-UNC18376338_lod.drop-1.5_snpsqc_5.batches_0.csv
    Untracked:  data/ici.vs.eoi_rz.age-no.covariates_genes_chr-10_peak.marker-UNC18376338_lod.drop-1.5_snpsqc_5.batches_mis_0.csv
    Untracked:  data/ici.vs.eoi_rz.age-no.covariates_genes_chr-10_peak.marker-UNC18382395_lod.drop-1.5_snpsqc_dis_no-x_updated_5.batches_0.csv
    Untracked:  data/ici.vs.eoi_rz.age-no.covariates_genes_chr-10_peak.marker-UNCHS028236_lod.drop-1.5_snpsqc_5.batches_mis_0.csv
    Untracked:  data/ici.vs.eoi_rz.age-no.covariates_genes_chr-10_peak.marker-UNCHS028236_lod.drop-1.5_snpsqc_5.batches_mis_52.csv
    Untracked:  data/ici.vs.eoi_rz.age-no.covariates_genes_chr-10_peak.marker-UNCHS028536_lod.drop-1.5_snpsqc_dis_no-x_updated_5.batches_mis_0.csv
    Untracked:  data/ici.vs.eoi_rz.age-no.covariates_genes_chr-10_peak.marker-UNCHS029024_lod.drop-1.5_5.batches_0.csv
    Untracked:  data/ici.vs.eoi_rz.age-no.covariates_genes_chr-11_peak.marker-UNC19970181_lod.drop-1.5_snpsqc_5.batches_mis_0.csv
    Untracked:  data/ici.vs.eoi_rz.age-no.covariates_genes_chr-11_peak.marker-UNC19970181_lod.drop-1.5_snpsqc_5.batches_mis_52.csv
    Untracked:  data/ici.vs.eoi_rz.age-no.covariates_genes_chr-12_peak.marker-ICR499_lod.drop-1.5_snpsqc_5.batches_mis_0.csv
    Untracked:  data/ici.vs.eoi_rz.age-no.covariates_genes_chr-12_peak.marker-ICR499_lod.drop-1.5_snpsqc_5.batches_mis_52.csv
    Untracked:  data/ici.vs.eoi_rz.age-no.covariates_genes_chr-13_peak.marker-UNCHS036773_lod.drop-1.5_snpsqc_5.batches_mis_0.csv
    Untracked:  data/ici.vs.eoi_rz.age-no.covariates_genes_chr-13_peak.marker-UNCHS036773_lod.drop-1.5_snpsqc_5.batches_mis_52.csv
    Untracked:  data/ici.vs.eoi_rz.age-no.covariates_genes_chr-14_peak.marker-UNC24056202_lod.drop-1.5_snpsqc_5.batches_mis_0.csv
    Untracked:  data/ici.vs.eoi_rz.age-no.covariates_genes_chr-14_peak.marker-UNC24056202_lod.drop-1.5_snpsqc_5.batches_mis_52.csv
    Untracked:  data/ici.vs.eoi_rz.age-no.covariates_genes_chr-15_peak.marker-UNC26070435_lod.drop-1.5_snpsqc_dis_no-x_updated_5.batches_mis_0.csv
    Untracked:  data/ici.vs.eoi_rz.age-no.covariates_genes_chr-15_peak.marker-UNC26070435_lod.drop-1.5_snpsqc_dis_no-x_updated_5.batches_mis_52.csv
    Untracked:  data/ici.vs.eoi_rz.age-no.covariates_genes_chr-17_peak.marker-UNCHS044241_lod.drop-1.5_snpsqc_5.batches_mis_0.csv
    Untracked:  data/ici.vs.eoi_rz.age-no.covariates_genes_chr-17_peak.marker-UNCHS044241_lod.drop-1.5_snpsqc_5.batches_mis_52.csv
    Untracked:  data/ici.vs.eoi_rz.age-no.covariates_genes_chr-17_peak.marker-UNCHS044241_lod.drop-1.5_snpsqc_dis_no-x_updated_5.batches_mis_0.csv
    Untracked:  data/ici.vs.eoi_rz.age-no.covariates_genes_chr-17_peak.marker-UNCrs47191360_lod.drop-1.5_snpsqc_dis_no-x_updated_5.batches_mis_52.csv
    Untracked:  data/ici.vs.eoi_rz.age-no.covariates_genes_chr-18_peak.marker-UNCHS045343_lod.drop-1.5_snpsqc_5.batches_mis_0.csv
    Untracked:  data/ici.vs.eoi_rz.age-no.covariates_genes_chr-18_peak.marker-UNCHS045343_lod.drop-1.5_snpsqc_5.batches_mis_52.csv
    Untracked:  data/ici.vs.eoi_rz.age-no.covariates_genes_chr-1_peak.marker-UNC228500_lod.drop-1.5_snpsqc_dis_no-x_updated_5.batches_52.csv
    Untracked:  data/ici.vs.eoi_rz.age-no.covariates_genes_chr-1_peak.marker-UNC228500_lod.drop-1.5_snpsqc_dis_no-x_updated_5.batches_mis_52.csv
    Untracked:  data/ici.vs.eoi_rz.age-no.covariates_genes_chr-2_peak.marker-UNC4609527_lod.drop-1.5_snpsqc_dis_no-x_updated_5.batches_mis_0.csv
    Untracked:  data/ici.vs.eoi_rz.age-no.covariates_genes_chr-2_peak.marker-UNC4609527_lod.drop-1.5_snpsqc_dis_no-x_updated_5.batches_mis_52.csv
    Untracked:  data/ici.vs.eoi_rz.age-no.covariates_genes_chr-3_peak.marker-ICR1338_lod.drop-1.5_5.batches_0.csv
    Untracked:  data/ici.vs.eoi_rz.age-no.covariates_genes_chr-3_peak.marker-ICR1338_lod.drop-1.5_5.batches_52.csv
    Untracked:  data/ici.vs.eoi_rz.age-no.covariates_genes_chr-3_peak.marker-ICR2047_lod.drop-1.5_5.batches_52.csv
    Untracked:  data/ici.vs.eoi_rz.age-no.covariates_genes_chr-3_peak.marker-JAX00106957_lod.drop-1.5_5.batches_0.csv
    Untracked:  data/ici.vs.eoi_rz.age-no.covariates_genes_chr-3_peak.marker-JAX00112499_lod.drop-1.5_5.batches_52.csv
    Untracked:  data/ici.vs.eoi_rz.age-no.covariates_genes_chr-3_peak.marker-JAX00219554_lod.drop-1.5_5.batches_0.csv
    Untracked:  data/ici.vs.eoi_rz.age-no.covariates_genes_chr-3_peak.marker-JAX00220631_lod.drop-1.5_5.batches_0.csv
    Untracked:  data/ici.vs.eoi_rz.age-no.covariates_genes_chr-3_peak.marker-JAX00514662_lod.drop-1.5_5.batches_52.csv
    Untracked:  data/ici.vs.eoi_rz.age-no.covariates_genes_chr-3_peak.marker-JAX00529707_lod.drop-1.5_5.batches_52.csv
    Untracked:  data/ici.vs.eoi_rz.age-no.covariates_genes_chr-3_peak.marker-UNC4813186_lod.drop-1.5_5.batches_0.csv
    Untracked:  data/ici.vs.eoi_rz.age-no.covariates_genes_chr-3_peak.marker-UNC4813186_lod.drop-1.5_5.batches_52.csv
    Untracked:  data/ici.vs.eoi_rz.age-no.covariates_genes_chr-3_peak.marker-UNC4836550_lod.drop-1.5_5.batches_52.csv
    Untracked:  data/ici.vs.eoi_rz.age-no.covariates_genes_chr-3_peak.marker-UNC4837069_lod.drop-1.5_5.batches_0.csv
    Untracked:  data/ici.vs.eoi_rz.age-no.covariates_genes_chr-3_peak.marker-UNC4912039_lod.drop-1.5_5.batches_0.csv
    Untracked:  data/ici.vs.eoi_rz.age-no.covariates_genes_chr-3_peak.marker-UNC4912039_lod.drop-1.5_5.batches_52.csv
    Untracked:  data/ici.vs.eoi_rz.age-no.covariates_genes_chr-3_peak.marker-UNC4924377_lod.drop-1.5_5.batches_0.csv
    Untracked:  data/ici.vs.eoi_rz.age-no.covariates_genes_chr-3_peak.marker-UNC4924377_lod.drop-1.5_5.batches_52.csv
    Untracked:  data/ici.vs.eoi_rz.age-no.covariates_genes_chr-3_peak.marker-UNC4956618_lod.drop-1.5_5.batches_52.csv
    Untracked:  data/ici.vs.eoi_rz.age-no.covariates_genes_chr-3_peak.marker-UNC4957168_lod.drop-1.5_5.batches_0.csv
    Untracked:  data/ici.vs.eoi_rz.age-no.covariates_genes_chr-3_peak.marker-UNC5006948_lod.drop-1.5_5.batches_52.csv
    Untracked:  data/ici.vs.eoi_rz.age-no.covariates_genes_chr-3_peak.marker-UNC5008656_lod.drop-1.5_5.batches_0.csv
    Untracked:  data/ici.vs.eoi_rz.age-no.covariates_genes_chr-3_peak.marker-UNC5083168_lod.drop-1.5_5.batches_0.csv
    Untracked:  data/ici.vs.eoi_rz.age-no.covariates_genes_chr-3_peak.marker-UNC5083168_lod.drop-1.5_5.batches_52.csv
    Untracked:  data/ici.vs.eoi_rz.age-no.covariates_genes_chr-3_peak.marker-UNC5192582_lod.drop-1.5_5.batches_0.csv
    Untracked:  data/ici.vs.eoi_rz.age-no.covariates_genes_chr-3_peak.marker-UNC5213888_lod.drop-1.5_snpsqc_5.batches_mis_0.csv
    Untracked:  data/ici.vs.eoi_rz.age-no.covariates_genes_chr-3_peak.marker-UNC5213888_lod.drop-1.5_snpsqc_5.batches_mis_52.csv
    Untracked:  data/ici.vs.eoi_rz.age-no.covariates_genes_chr-3_peak.marker-UNC5262472_lod.drop-1.5_5.batches_0.csv
    Untracked:  data/ici.vs.eoi_rz.age-no.covariates_genes_chr-3_peak.marker-UNC5262472_lod.drop-1.5_5.batches_52.csv
    Untracked:  data/ici.vs.eoi_rz.age-no.covariates_genes_chr-3_peak.marker-UNC5266664_lod.drop-1.5_5.batches_0.csv
    Untracked:  data/ici.vs.eoi_rz.age-no.covariates_genes_chr-3_peak.marker-UNC5300878_lod.drop-1.5_5.batches_52.csv
    Untracked:  data/ici.vs.eoi_rz.age-no.covariates_genes_chr-3_peak.marker-UNC5360055_lod.drop-1.5_5.batches_52.csv
    Untracked:  data/ici.vs.eoi_rz.age-no.covariates_genes_chr-3_peak.marker-UNC5454839_lod.drop-1.5_5.batches_0.csv
    Untracked:  data/ici.vs.eoi_rz.age-no.covariates_genes_chr-3_peak.marker-UNC5454839_lod.drop-1.5_5.batches_52.csv
    Untracked:  data/ici.vs.eoi_rz.age-no.covariates_genes_chr-3_peak.marker-UNC5645844_lod.drop-1.5_5.batches_0.csv
    Untracked:  data/ici.vs.eoi_rz.age-no.covariates_genes_chr-3_peak.marker-UNC5645844_lod.drop-1.5_5.batches_52.csv
    Untracked:  data/ici.vs.eoi_rz.age-no.covariates_genes_chr-3_peak.marker-UNC5679155_lod.drop-1.5_5.batches_52.csv
    Untracked:  data/ici.vs.eoi_rz.age-no.covariates_genes_chr-3_peak.marker-UNC5808246_lod.drop-1.5_5.batches_52.csv
    Untracked:  data/ici.vs.eoi_rz.age-no.covariates_genes_chr-3_peak.marker-UNC5945286_lod.drop-1.5_5.batches_52.csv
    Untracked:  data/ici.vs.eoi_rz.age-no.covariates_genes_chr-3_peak.marker-UNC6227106_lod.drop-1.5_5.batches_52.csv
    Untracked:  data/ici.vs.eoi_rz.age-no.covariates_genes_chr-3_peak.marker-UNCHS008182_lod.drop-1.5_5.batches_52.csv
    Untracked:  data/ici.vs.eoi_rz.age-no.covariates_genes_chr-3_peak.marker-UNCHS008286_lod.drop-1.5_5.batches_0.csv
    Untracked:  data/ici.vs.eoi_rz.age-no.covariates_genes_chr-3_peak.marker-UNCHS008286_lod.drop-1.5_5.batches_52.csv
    Untracked:  data/ici.vs.eoi_rz.age-no.covariates_genes_chr-3_peak.marker-UNCHS008487_lod.drop-1.5_snpsqc_5.batches_0.csv
    Untracked:  data/ici.vs.eoi_rz.age-no.covariates_genes_chr-3_peak.marker-UNCHS008487_lod.drop-1.5_snpsqc_5.batches_52.csv
    Untracked:  data/ici.vs.eoi_rz.age-no.covariates_genes_chr-3_peak.marker-UNCHS008487_lod.drop-1.5_snpsqc_dis_no-x_updated_5.batches_0.csv
    Untracked:  data/ici.vs.eoi_rz.age-no.covariates_genes_chr-3_peak.marker-UNCHS008487_lod.drop-1.5_snpsqc_dis_no-x_updated_5.batches_52.csv
    Untracked:  data/ici.vs.eoi_rz.age-no.covariates_genes_chr-3_peak.marker-UNCHS008487_lod.drop-1.5_snpsqc_dis_no-x_updated_5.batches_mis_0.csv
    Untracked:  data/ici.vs.eoi_rz.age-no.covariates_genes_chr-3_peak.marker-UNCHS008487_lod.drop-1.5_snpsqc_dis_no-x_updated_5.batches_mis_52.csv
    Untracked:  data/ici.vs.eoi_rz.age-no.covariates_genes_chr-3_peak.marker-UNCHS008511_lod.drop-1.5_snpsqc_5.batches_mis_0.csv
    Untracked:  data/ici.vs.eoi_rz.age-no.covariates_genes_chr-3_peak.marker-UNCHS008511_lod.drop-1.5_snpsqc_5.batches_mis_52.csv
    Untracked:  data/ici.vs.eoi_rz.age-no.covariates_genes_chr-3_peak.marker-UNCHS008595_lod.drop-1.5_5.batches_52.csv
    Untracked:  data/ici.vs.eoi_rz.age-no.covariates_genes_chr-3_peak.marker-UNCHS008613_lod.drop-1.5_5.batches_0.csv
    Untracked:  data/ici.vs.eoi_rz.age-no.covariates_genes_chr-3_peak.marker-UNCHS008613_lod.drop-1.5_5.batches_52.csv
    Untracked:  data/ici.vs.eoi_rz.age-no.covariates_genes_chr-3_peak.marker-UNCHS008815_lod.drop-1.5_5.batches_0.csv
    Untracked:  data/ici.vs.eoi_rz.age-no.covariates_genes_chr-3_peak.marker-UNCHS008815_lod.drop-1.5_5.batches_52.csv
    Untracked:  data/ici.vs.eoi_rz.age-no.covariates_genes_chr-3_peak.marker-UNCHS009067_lod.drop-1.5_5.batches_0.csv
    Untracked:  data/ici.vs.eoi_rz.age-no.covariates_genes_chr-3_peak.marker-UNCHS009383_lod.drop-1.5_5.batches_52.csv
    Untracked:  data/ici.vs.eoi_rz.age-no.covariates_genes_chr-3_peak.marker-UNCHS009693_lod.drop-1.5_5.batches_52.csv
    Untracked:  data/ici.vs.eoi_rz.age-no.covariates_genes_chr-3_peak.marker-UNCJPD001246_lod.drop-1.5_5.batches_52.csv
    Untracked:  data/ici.vs.eoi_rz.age-no.covariates_genes_chr-3_peak.marker-UNCJPD001341_lod.drop-1.5_5.batches_52.csv
    Untracked:  data/ici.vs.eoi_rz.age-no.covariates_genes_chr-3_peak.marker-sanger2623_lod.drop-1.5_5.batches_52.csv
    Untracked:  data/ici.vs.eoi_rz.age-no.covariates_genes_chr-4_peak.marker-UNCHS013125_lod.drop-1.5_snpsqc_dis_no-x_updated_5.batches_52.csv
    Untracked:  data/ici.vs.eoi_rz.age-no.covariates_genes_chr-4_peak.marker-UNCHS013125_lod.drop-1.5_snpsqc_dis_no-x_updated_5.batches_mis_52.csv
    Untracked:  data/ici.vs.eoi_rz.age-no.covariates_genes_chr-6_peak.marker-UNCHS018170_lod.drop-1.5_snpsqc_5.batches_0.csv
    Untracked:  data/ici.vs.eoi_rz.age-no.covariates_genes_chr-6_peak.marker-UNCHS018231_lod.drop-1.5_snpsqc_5.batches_mis_0.csv
    Untracked:  data/ici.vs.eoi_rz.age-no.covariates_genes_chr-7_peak.marker-UNCHS020066_lod.drop-1.5_snpsqc_5.batches_mis_0.csv
    Untracked:  data/ici.vs.eoi_rz.age-no.covariates_genes_chr-7_peak.marker-UNCHS020066_lod.drop-1.5_snpsqc_5.batches_mis_52.csv
    Untracked:  data/ici.vs.eoi_rz.age-no.covariates_genes_chr-7_peak.marker-UNCHS020066_lod.drop-1.5_snpsqc_dis_no-x_updated_5.batches_mis_0.csv
    Untracked:  data/ici.vs.eoi_rz.age-no.covariates_genes_chr-7_peak.marker-UNCHS020066_lod.drop-1.5_snpsqc_dis_no-x_updated_5.batches_mis_52.csv
    Untracked:  data/ici.vs.eoi_rz.age-no.covariates_genes_chr-8_peak.marker-UNC15524531_lod.drop-1.5_snpsqc_5.batches_mis_0.csv
    Untracked:  data/ici.vs.eoi_rz.age-no.covariates_genes_chr-8_peak.marker-UNC15524531_lod.drop-1.5_snpsqc_5.batches_mis_52.csv
    Untracked:  data/ici.vs.eoi_rz.age-no.covariates_genes_chr-8_peak.marker-UNC15524531_lod.drop-1.5_snpsqc_dis_no-x_updated_5.batches_mis_0.csv
    Untracked:  data/ici.vs.eoi_rz.age-no.covariates_genes_chr-8_peak.marker-UNC15524531_lod.drop-1.5_snpsqc_dis_no-x_updated_5.batches_mis_52.csv
    Untracked:  data/ici.vs.eoi_rz.age-no.covariates_genes_chr-9_peak.marker-UNC17203597_lod.drop-1.5_snpsqc_5.batches_mis_0.csv
    Untracked:  data/ici.vs.eoi_rz.age-no.covariates_genes_chr-9_peak.marker-UNC17203597_lod.drop-1.5_snpsqc_5.batches_mis_52.csv
    Untracked:  data/ici.vs.eoi_rz.age-no.covariates_genes_chr-X_peak.marker-UNCHS048314_lod.drop-1.5_snpsqc_5.batches_mis_0.csv
    Untracked:  data/ici.vs.eoi_rz.age-no.covariates_genes_chr-X_peak.marker-UNCHS048314_lod.drop-1.5_snpsqc_5.batches_mis_52.csv
    Untracked:  data/ici.vs.eoi_rz.age-no.covariates_genes_chr-X_peak.marker-UNCHS048314_lod.drop-1.5_snpsqc_dis_no-x_updated_5.batches_mis_0.csv
    Untracked:  data/ici.vs.eoi_rz.age-no.covariates_genes_chr-X_peak.marker-UNCHS048314_lod.drop-1.5_snpsqc_dis_no-x_updated_5.batches_mis_52.csv
    Untracked:  data/ici.vs.eoi_sample.genos_marker.freq_low.geno.freq.removed.csv
    Untracked:  data/ici.vs.eoi_sample.genos_marker.freq_low.geno.freq.removed_sample.outliers.removed.csv
    Untracked:  data/ici.vs.eoi_sample.genos_marker.freq_low.probs.freq.removed.csv
    Untracked:  data/ici.vs.eoi_sample.genos_marker.freq_low.probs.freq.removed_sample.outliers.removed.csv
    Untracked:  data/ici.vs.eoi_scanone_5.batches.Rdata
    Untracked:  data/ici.vs.eoi_scanone_5.batches_mis.Rdata
    Untracked:  data/ici.vs.eoi_scanone_snpsqc_5.batches.Rdata
    Untracked:  data/ici.vs.eoi_scanone_snpsqc_5.batches_mis.Rdata
    Untracked:  data/ici.vs.eoi_scanone_snpsqc_dis_no-x_updated_5.batches.Rdata
    Untracked:  data/ici.vs.pbs_age.of.onset-no.covariates_blup_sub_chr-14_peak.marker-UNC24056202_lod.drop-1.5_5.batches_mis.csv
    Untracked:  data/ici.vs.pbs_age.of.onset-no.covariates_blup_sub_chr-14_peak.marker-UNC24056202_lod.drop-1.5_snpsqc_5.batches_mis.csv
    Untracked:  data/ici.vs.pbs_age.of.onset-no.covariates_blup_sub_chr-17_peak.marker-UNCJPD006614_lod.drop-1.5_snpsqc_5.batches_mis.csv
    Untracked:  data/ici.vs.pbs_age.of.onset-no.covariates_blup_sub_chr-18_peak.marker-UNC28655293_lod.drop-1.5_5.batches_mis.csv
    Untracked:  data/ici.vs.pbs_age.of.onset-no.covariates_blup_sub_chr-18_peak.marker-UNCHS045343_lod.drop-1.5_snpsqc_5.batches_mis.csv
    Untracked:  data/ici.vs.pbs_age.of.onset-no.covariates_blup_sub_chr-5_peak.marker-UNC10044126_lod.drop-1.5_5.batches_mis.csv
    Untracked:  data/ici.vs.pbs_age.of.onset-no.covariates_blup_sub_chr-5_peak.marker-UNC8675939_lod.drop-1.5_snpsqc_5.batches_mis.csv
    Untracked:  data/ici.vs.pbs_age.of.onset-no.covariates_blup_sub_chr-8_peak.marker-JAX00667121_lod.drop-1.5_5.batches_mis.csv
    Untracked:  data/ici.vs.pbs_age.of.onset-no.covariates_blup_sub_chr-8_peak.marker-UNC15524531_lod.drop-1.5_snpsqc_5.batches_mis.csv
    Untracked:  data/ici.vs.pbs_age.of.onset-no.covariates_blup_sub_chr-X_peak.marker-UNCHS048314_lod.drop-1.5_snpsqc_5.batches_mis.csv
    Untracked:  data/ici.vs.pbs_age.of.onset-no.covariates_genes_chr-14_peak.marker-UNC24056202_lod.drop-1.5_5.batches_mis.csv
    Untracked:  data/ici.vs.pbs_age.of.onset-no.covariates_genes_chr-14_peak.marker-UNC24056202_lod.drop-1.5_snpsqc_5.batches_mis.csv
    Untracked:  data/ici.vs.pbs_age.of.onset-no.covariates_genes_chr-17_peak.marker-UNCJPD006614_lod.drop-1.5_snpsqc_5.batches_mis.csv
    Untracked:  data/ici.vs.pbs_age.of.onset-no.covariates_genes_chr-18_peak.marker-UNC28655293_lod.drop-1.5_5.batches_mis.csv
    Untracked:  data/ici.vs.pbs_age.of.onset-no.covariates_genes_chr-18_peak.marker-UNCHS045343_lod.drop-1.5_snpsqc_5.batches_mis.csv
    Untracked:  data/ici.vs.pbs_age.of.onset-no.covariates_genes_chr-5_peak.marker-UNC10044126_lod.drop-1.5_5.batches_mis.csv
    Untracked:  data/ici.vs.pbs_age.of.onset-no.covariates_genes_chr-5_peak.marker-UNC8675939_lod.drop-1.5_snpsqc_5.batches_mis.csv
    Untracked:  data/ici.vs.pbs_age.of.onset-no.covariates_genes_chr-8_peak.marker-JAX00667121_lod.drop-1.5_5.batches_mis.csv
    Untracked:  data/ici.vs.pbs_age.of.onset-no.covariates_genes_chr-8_peak.marker-UNC15524531_lod.drop-1.5_snpsqc_5.batches_mis.csv
    Untracked:  data/ici.vs.pbs_age.of.onset-no.covariates_genes_chr-X_peak.marker-UNCHS048314_lod.drop-1.5_snpsqc_5.batches_mis.csv
    Untracked:  data/ici.vs.pbs_bad.markers_snpsqc_5.batches_mis.csv
    Untracked:  data/ici.vs.pbs_blup.full_chr-10_lod.drop-1.5_5.batches.csv
    Untracked:  data/ici.vs.pbs_blup.full_chr-10_snpsqc_5.batches.csv
    Untracked:  data/ici.vs.pbs_blup.full_chr-10_snpsqc_5.batches_mis.csv
    Untracked:  data/ici.vs.pbs_blup.full_chr-10_snpsqc_dis_no-x_updated_5.batches.csv
    Untracked:  data/ici.vs.pbs_blup.full_chr-10_snpsqc_dis_no-x_updated_5.batches_mis.csv
    Untracked:  data/ici.vs.pbs_blup.full_chr-11_lod.drop-1.5_5.batches.csv
    Untracked:  data/ici.vs.pbs_blup.full_chr-11_snpsqc_5.batches.csv
    Untracked:  data/ici.vs.pbs_blup.full_chr-11_snpsqc_5.batches_mis.csv
    Untracked:  data/ici.vs.pbs_blup.full_chr-11_snpsqc_dis_no-x_updated_5.batches.csv
    Untracked:  data/ici.vs.pbs_blup.full_chr-11_snpsqc_dis_no-x_updated_5.batches_mis.csv
    Untracked:  data/ici.vs.pbs_blup.full_chr-12_lod.drop-1.5_5.batches.csv
    Untracked:  data/ici.vs.pbs_blup.full_chr-12_snpsqc_5.batches.csv
    Untracked:  data/ici.vs.pbs_blup.full_chr-12_snpsqc_5.batches_mis.csv
    Untracked:  data/ici.vs.pbs_blup.full_chr-12_snpsqc_dis_no-x_updated_5.batches.csv
    Untracked:  data/ici.vs.pbs_blup.full_chr-12_snpsqc_dis_no-x_updated_5.batches_mis.csv
    Untracked:  data/ici.vs.pbs_blup.full_chr-13_lod.drop-1.5_5.batches.csv
    Untracked:  data/ici.vs.pbs_blup.full_chr-13_snpsqc_5.batches.csv
    Untracked:  data/ici.vs.pbs_blup.full_chr-13_snpsqc_5.batches_mis.csv
    Untracked:  data/ici.vs.pbs_blup.full_chr-13_snpsqc_dis_no-x_updated_5.batches.csv
    Untracked:  data/ici.vs.pbs_blup.full_chr-13_snpsqc_dis_no-x_updated_5.batches_mis.csv
    Untracked:  data/ici.vs.pbs_blup.full_chr-14_lod.drop-1.5_5.batches.csv
    Untracked:  data/ici.vs.pbs_blup.full_chr-14_snpsqc_5.batches.csv
    Untracked:  data/ici.vs.pbs_blup.full_chr-14_snpsqc_5.batches_mis.csv
    Untracked:  data/ici.vs.pbs_blup.full_chr-14_snpsqc_dis_no-x_updated_5.batches.csv
    Untracked:  data/ici.vs.pbs_blup.full_chr-14_snpsqc_dis_no-x_updated_5.batches_mis.csv
    Untracked:  data/ici.vs.pbs_blup.full_chr-15_lod.drop-1.5_5.batches.csv
    Untracked:  data/ici.vs.pbs_blup.full_chr-15_snpsqc_5.batches.csv
    Untracked:  data/ici.vs.pbs_blup.full_chr-15_snpsqc_5.batches_mis.csv
    Untracked:  data/ici.vs.pbs_blup.full_chr-15_snpsqc_dis_no-x_updated_5.batches.csv
    Untracked:  data/ici.vs.pbs_blup.full_chr-15_snpsqc_dis_no-x_updated_5.batches_mis.csv
    Untracked:  data/ici.vs.pbs_blup.full_chr-16_lod.drop-1.5_5.batches.csv
    Untracked:  data/ici.vs.pbs_blup.full_chr-16_snpsqc_5.batches.csv
    Untracked:  data/ici.vs.pbs_blup.full_chr-16_snpsqc_5.batches_mis.csv
    Untracked:  data/ici.vs.pbs_blup.full_chr-16_snpsqc_dis_no-x_updated_5.batches.csv
    Untracked:  data/ici.vs.pbs_blup.full_chr-16_snpsqc_dis_no-x_updated_5.batches_mis.csv
    Untracked:  data/ici.vs.pbs_blup.full_chr-17_lod.drop-1.5_5.batches.csv
    Untracked:  data/ici.vs.pbs_blup.full_chr-17_snpsqc_5.batches.csv
    Untracked:  data/ici.vs.pbs_blup.full_chr-17_snpsqc_5.batches_mis.csv
    Untracked:  data/ici.vs.pbs_blup.full_chr-17_snpsqc_dis_no-x_updated_5.batches.csv
    Untracked:  data/ici.vs.pbs_blup.full_chr-17_snpsqc_dis_no-x_updated_5.batches_mis.csv
    Untracked:  data/ici.vs.pbs_blup.full_chr-18_lod.drop-1.5_5.batches.csv
    Untracked:  data/ici.vs.pbs_blup.full_chr-18_snpsqc_5.batches.csv
    Untracked:  data/ici.vs.pbs_blup.full_chr-18_snpsqc_5.batches_mis.csv
    Untracked:  data/ici.vs.pbs_blup.full_chr-18_snpsqc_dis_no-x_updated_5.batches.csv
    Untracked:  data/ici.vs.pbs_blup.full_chr-18_snpsqc_dis_no-x_updated_5.batches_mis.csv
    Untracked:  data/ici.vs.pbs_blup.full_chr-19_lod.drop-1.5_5.batches.csv
    Untracked:  data/ici.vs.pbs_blup.full_chr-19_snpsqc_5.batches.csv
    Untracked:  data/ici.vs.pbs_blup.full_chr-19_snpsqc_5.batches_mis.csv
    Untracked:  data/ici.vs.pbs_blup.full_chr-19_snpsqc_dis_no-x_updated_5.batches.csv
    Untracked:  data/ici.vs.pbs_blup.full_chr-19_snpsqc_dis_no-x_updated_5.batches_mis.csv
    Untracked:  data/ici.vs.pbs_blup.full_chr-1_lod.drop-1.5_5.batches.csv
    Untracked:  data/ici.vs.pbs_blup.full_chr-1_snpsqc_5.batches.csv
    Untracked:  data/ici.vs.pbs_blup.full_chr-1_snpsqc_5.batches_mis.csv
    Untracked:  data/ici.vs.pbs_blup.full_chr-1_snpsqc_dis_no-x_updated_5.batches.csv
    Untracked:  data/ici.vs.pbs_blup.full_chr-1_snpsqc_dis_no-x_updated_5.batches_mis.csv
    Untracked:  data/ici.vs.pbs_blup.full_chr-2_lod.drop-1.5_5.batches.csv
    Untracked:  data/ici.vs.pbs_blup.full_chr-2_snpsqc_5.batches.csv
    Untracked:  data/ici.vs.pbs_blup.full_chr-2_snpsqc_5.batches_mis.csv
    Untracked:  data/ici.vs.pbs_blup.full_chr-2_snpsqc_dis_no-x_updated_5.batches.csv
    Untracked:  data/ici.vs.pbs_blup.full_chr-2_snpsqc_dis_no-x_updated_5.batches_mis.csv
    Untracked:  data/ici.vs.pbs_blup.full_chr-3_lod.drop-1.5_5.batches.csv
    Untracked:  data/ici.vs.pbs_blup.full_chr-3_snpsqc_5.batches.csv
    Untracked:  data/ici.vs.pbs_blup.full_chr-3_snpsqc_5.batches_mis.csv
    Untracked:  data/ici.vs.pbs_blup.full_chr-3_snpsqc_dis_no-x_updated_5.batches.csv
    Untracked:  data/ici.vs.pbs_blup.full_chr-3_snpsqc_dis_no-x_updated_5.batches_mis.csv
    Untracked:  data/ici.vs.pbs_blup.full_chr-4_lod.drop-1.5_5.batches.csv
    Untracked:  data/ici.vs.pbs_blup.full_chr-4_snpsqc_5.batches.csv
    Untracked:  data/ici.vs.pbs_blup.full_chr-4_snpsqc_5.batches_mis.csv
    Untracked:  data/ici.vs.pbs_blup.full_chr-4_snpsqc_dis_no-x_updated_5.batches.csv
    Untracked:  data/ici.vs.pbs_blup.full_chr-4_snpsqc_dis_no-x_updated_5.batches_mis.csv
    Untracked:  data/ici.vs.pbs_blup.full_chr-5_lod.drop-1.5_5.batches.csv
    Untracked:  data/ici.vs.pbs_blup.full_chr-5_snpsqc_5.batches.csv
    Untracked:  data/ici.vs.pbs_blup.full_chr-5_snpsqc_5.batches_mis.csv
    Untracked:  data/ici.vs.pbs_blup.full_chr-5_snpsqc_dis_no-x_updated_5.batches.csv
    Untracked:  data/ici.vs.pbs_blup.full_chr-5_snpsqc_dis_no-x_updated_5.batches_mis.csv
    Untracked:  data/ici.vs.pbs_blup.full_chr-6_lod.drop-1.5_5.batches.csv
    Untracked:  data/ici.vs.pbs_blup.full_chr-6_snpsqc_5.batches.csv
    Untracked:  data/ici.vs.pbs_blup.full_chr-6_snpsqc_5.batches_mis.csv
    Untracked:  data/ici.vs.pbs_blup.full_chr-6_snpsqc_dis_no-x_updated_5.batches.csv
    Untracked:  data/ici.vs.pbs_blup.full_chr-6_snpsqc_dis_no-x_updated_5.batches_mis.csv
    Untracked:  data/ici.vs.pbs_blup.full_chr-7_lod.drop-1.5_5.batches.csv
    Untracked:  data/ici.vs.pbs_blup.full_chr-7_snpsqc_5.batches.csv
    Untracked:  data/ici.vs.pbs_blup.full_chr-7_snpsqc_5.batches_mis.csv
    Untracked:  data/ici.vs.pbs_blup.full_chr-7_snpsqc_dis_no-x_updated_5.batches.csv
    Untracked:  data/ici.vs.pbs_blup.full_chr-7_snpsqc_dis_no-x_updated_5.batches_mis.csv
    Untracked:  data/ici.vs.pbs_blup.full_chr-8_lod.drop-1.5_5.batches.csv
    Untracked:  data/ici.vs.pbs_blup.full_chr-8_snpsqc_5.batches.csv
    Untracked:  data/ici.vs.pbs_blup.full_chr-8_snpsqc_5.batches_mis.csv
    Untracked:  data/ici.vs.pbs_blup.full_chr-8_snpsqc_dis_no-x_updated_5.batches.csv
    Untracked:  data/ici.vs.pbs_blup.full_chr-8_snpsqc_dis_no-x_updated_5.batches_mis.csv
    Untracked:  data/ici.vs.pbs_blup.full_chr-9_lod.drop-1.5_5.batches.csv
    Untracked:  data/ici.vs.pbs_blup.full_chr-9_snpsqc_5.batches.csv
    Untracked:  data/ici.vs.pbs_blup.full_chr-9_snpsqc_5.batches_mis.csv
    Untracked:  data/ici.vs.pbs_blup.full_chr-9_snpsqc_dis_no-x_updated_5.batches.csv
    Untracked:  data/ici.vs.pbs_blup.full_chr-9_snpsqc_dis_no-x_updated_5.batches_mis.csv
    Untracked:  data/ici.vs.pbs_blup.full_chr-X_lod.drop-1.5_5.batches.csv
    Untracked:  data/ici.vs.pbs_blup.full_chr-X_snpsqc_5.batches.csv
    Untracked:  data/ici.vs.pbs_blup.full_chr-X_snpsqc_5.batches_mis.csv
    Untracked:  data/ici.vs.pbs_blup.full_chr-X_snpsqc_dis_no-x_updated_5.batches.csv
    Untracked:  data/ici.vs.pbs_blup.full_chr-X_snpsqc_dis_no-x_updated_5.batches_mis.csv
    Untracked:  data/ici.vs.pbs_blup.qc_chr-10_5.batches.csv
    Untracked:  data/ici.vs.pbs_blup.qc_chr-10_5.batches_mis.csv
    Untracked:  data/ici.vs.pbs_blup.qc_chr-10_lod.drop-1.5_5.batches.csv
    Untracked:  data/ici.vs.pbs_blup.qc_chr-10_lod.drop-1.5_5.batches_mis.csv
    Untracked:  data/ici.vs.pbs_blup.qc_chr-10_snpsqc_5.batches.csv
    Untracked:  data/ici.vs.pbs_blup.qc_chr-10_snpsqc_5.batches_mis.csv
    Untracked:  data/ici.vs.pbs_blup.qc_chr-10_snpsqc_dis_no-x_updated_5.batches.csv
    Untracked:  data/ici.vs.pbs_blup.qc_chr-10_snpsqc_dis_no-x_updated_5.batches_mis.csv
    Untracked:  data/ici.vs.pbs_blup.qc_chr-11_5.batches.csv
    Untracked:  data/ici.vs.pbs_blup.qc_chr-11_5.batches_mis.csv
    Untracked:  data/ici.vs.pbs_blup.qc_chr-11_lod.drop-1.5_5.batches.csv
    Untracked:  data/ici.vs.pbs_blup.qc_chr-11_lod.drop-1.5_5.batches_mis.csv
    Untracked:  data/ici.vs.pbs_blup.qc_chr-11_snpsqc_5.batches.csv
    Untracked:  data/ici.vs.pbs_blup.qc_chr-11_snpsqc_5.batches_mis.csv
    Untracked:  data/ici.vs.pbs_blup.qc_chr-11_snpsqc_dis_no-x_updated_5.batches.csv
    Untracked:  data/ici.vs.pbs_blup.qc_chr-11_snpsqc_dis_no-x_updated_5.batches_mis.csv
    Untracked:  data/ici.vs.pbs_blup.qc_chr-12_5.batches.csv
    Untracked:  data/ici.vs.pbs_blup.qc_chr-12_5.batches_mis.csv
    Untracked:  data/ici.vs.pbs_blup.qc_chr-12_lod.drop-1.5_5.batches.csv
    Untracked:  data/ici.vs.pbs_blup.qc_chr-12_lod.drop-1.5_5.batches_mis.csv
    Untracked:  data/ici.vs.pbs_blup.qc_chr-12_snpsqc_5.batches.csv
    Untracked:  data/ici.vs.pbs_blup.qc_chr-12_snpsqc_5.batches_mis.csv
    Untracked:  data/ici.vs.pbs_blup.qc_chr-12_snpsqc_dis_no-x_updated_5.batches.csv
    Untracked:  data/ici.vs.pbs_blup.qc_chr-12_snpsqc_dis_no-x_updated_5.batches_mis.csv
    Untracked:  data/ici.vs.pbs_blup.qc_chr-13_5.batches.csv
    Untracked:  data/ici.vs.pbs_blup.qc_chr-13_5.batches_mis.csv
    Untracked:  data/ici.vs.pbs_blup.qc_chr-13_lod.drop-1.5_5.batches.csv
    Untracked:  data/ici.vs.pbs_blup.qc_chr-13_lod.drop-1.5_5.batches_mis.csv
    Untracked:  data/ici.vs.pbs_blup.qc_chr-13_snpsqc_5.batches.csv
    Untracked:  data/ici.vs.pbs_blup.qc_chr-13_snpsqc_5.batches_mis.csv
    Untracked:  data/ici.vs.pbs_blup.qc_chr-13_snpsqc_dis_no-x_updated_5.batches.csv
    Untracked:  data/ici.vs.pbs_blup.qc_chr-13_snpsqc_dis_no-x_updated_5.batches_mis.csv
    Untracked:  data/ici.vs.pbs_blup.qc_chr-14_5.batches.csv
    Untracked:  data/ici.vs.pbs_blup.qc_chr-14_5.batches_mis.csv
    Untracked:  data/ici.vs.pbs_blup.qc_chr-14_lod.drop-1.5_5.batches.csv
    Untracked:  data/ici.vs.pbs_blup.qc_chr-14_snpsqc_5.batches.csv
    Untracked:  data/ici.vs.pbs_blup.qc_chr-14_snpsqc_5.batches_mis.csv
    Untracked:  data/ici.vs.pbs_blup.qc_chr-14_snpsqc_dis_no-x_updated_5.batches.csv
    Untracked:  data/ici.vs.pbs_blup.qc_chr-14_snpsqc_dis_no-x_updated_5.batches_mis.csv
    Untracked:  data/ici.vs.pbs_blup.qc_chr-15_5.batches.csv
    Untracked:  data/ici.vs.pbs_blup.qc_chr-15_5.batches_mis.csv
    Untracked:  data/ici.vs.pbs_blup.qc_chr-15_lod.drop-1.5_5.batches.csv
    Untracked:  data/ici.vs.pbs_blup.qc_chr-15_snpsqc_5.batches.csv
    Untracked:  data/ici.vs.pbs_blup.qc_chr-15_snpsqc_5.batches_mis.csv
    Untracked:  data/ici.vs.pbs_blup.qc_chr-16_5.batches.csv
    Untracked:  data/ici.vs.pbs_blup.qc_chr-16_5.batches_mis.csv
    Untracked:  data/ici.vs.pbs_blup.qc_chr-16_lod.drop-1.5_5.batches.csv
    Untracked:  data/ici.vs.pbs_blup.qc_chr-16_snpsqc_5.batches.csv
    Untracked:  data/ici.vs.pbs_blup.qc_chr-16_snpsqc_5.batches_mis.csv
    Untracked:  data/ici.vs.pbs_blup.qc_chr-16_snpsqc_dis_no-x_updated_5.batches.csv
    Untracked:  data/ici.vs.pbs_blup.qc_chr-16_snpsqc_dis_no-x_updated_5.batches_mis.csv
    Untracked:  data/ici.vs.pbs_blup.qc_chr-17_5.batches.csv
    Untracked:  data/ici.vs.pbs_blup.qc_chr-17_5.batches_mis.csv
    Untracked:  data/ici.vs.pbs_blup.qc_chr-17_lod.drop-1.5_5.batches.csv
    Untracked:  data/ici.vs.pbs_blup.qc_chr-17_snpsqc_5.batches.csv
    Untracked:  data/ici.vs.pbs_blup.qc_chr-17_snpsqc_5.batches_mis.csv
    Untracked:  data/ici.vs.pbs_blup.qc_chr-18_5.batches.csv
    Untracked:  data/ici.vs.pbs_blup.qc_chr-18_5.batches_mis.csv
    Untracked:  data/ici.vs.pbs_blup.qc_chr-18_lod.drop-1.5_5.batches.csv
    Untracked:  data/ici.vs.pbs_blup.qc_chr-18_snpsqc_5.batches.csv
    Untracked:  data/ici.vs.pbs_blup.qc_chr-18_snpsqc_5.batches_mis.csv
    Untracked:  data/ici.vs.pbs_blup.qc_chr-18_snpsqc_dis_no-x_updated_5.batches.csv
    Untracked:  data/ici.vs.pbs_blup.qc_chr-18_snpsqc_dis_no-x_updated_5.batches_mis.csv
    Untracked:  data/ici.vs.pbs_blup.qc_chr-19_5.batches.csv
    Untracked:  data/ici.vs.pbs_blup.qc_chr-19_5.batches_mis.csv
    Untracked:  data/ici.vs.pbs_blup.qc_chr-19_lod.drop-1.5_5.batches.csv
    Untracked:  data/ici.vs.pbs_blup.qc_chr-19_snpsqc_5.batches.csv
    Untracked:  data/ici.vs.pbs_blup.qc_chr-19_snpsqc_5.batches_mis.csv
    Untracked:  data/ici.vs.pbs_blup.qc_chr-19_snpsqc_dis_no-x_updated_5.batches.csv
    Untracked:  data/ici.vs.pbs_blup.qc_chr-19_snpsqc_dis_no-x_updated_5.batches_mis.csv
    Untracked:  data/ici.vs.pbs_blup.qc_chr-1_5.batches.csv
    Untracked:  data/ici.vs.pbs_blup.qc_chr-1_5.batches_mis.csv
    Untracked:  data/ici.vs.pbs_blup.qc_chr-1_lod.drop-1.5_5.batches.csv
    Untracked:  data/ici.vs.pbs_blup.qc_chr-1_lod.drop-1.5_5.batches_mis.csv
    Untracked:  data/ici.vs.pbs_blup.qc_chr-1_snpsqc_5.batches.csv
    Untracked:  data/ici.vs.pbs_blup.qc_chr-1_snpsqc_5.batches_mis.csv
    Untracked:  data/ici.vs.pbs_blup.qc_chr-1_snpsqc_dis_no-x_updated_5.batches.csv
    Untracked:  data/ici.vs.pbs_blup.qc_chr-1_snpsqc_dis_no-x_updated_5.batches_mis.csv
    Untracked:  data/ici.vs.pbs_blup.qc_chr-2_5.batches.csv
    Untracked:  data/ici.vs.pbs_blup.qc_chr-2_5.batches_mis.csv
    Untracked:  data/ici.vs.pbs_blup.qc_chr-2_lod.drop-1.5_5.batches.csv
    Untracked:  data/ici.vs.pbs_blup.qc_chr-2_lod.drop-1.5_5.batches_mis.csv
    Untracked:  data/ici.vs.pbs_blup.qc_chr-2_snpsqc_5.batches.csv
    Untracked:  data/ici.vs.pbs_blup.qc_chr-2_snpsqc_5.batches_mis.csv
    Untracked:  data/ici.vs.pbs_blup.qc_chr-2_snpsqc_dis_no-x_updated_5.batches.csv
    Untracked:  data/ici.vs.pbs_blup.qc_chr-2_snpsqc_dis_no-x_updated_5.batches_mis.csv
    Untracked:  data/ici.vs.pbs_blup.qc_chr-3_5.batches.csv
    Untracked:  data/ici.vs.pbs_blup.qc_chr-3_5.batches_mis.csv
    Untracked:  data/ici.vs.pbs_blup.qc_chr-3_lod.drop-1.5_5.batches.csv
    Untracked:  data/ici.vs.pbs_blup.qc_chr-3_lod.drop-1.5_5.batches_mis.csv
    Untracked:  data/ici.vs.pbs_blup.qc_chr-3_snpsqc_5.batches.csv
    Untracked:  data/ici.vs.pbs_blup.qc_chr-3_snpsqc_5.batches_mis.csv
    Untracked:  data/ici.vs.pbs_blup.qc_chr-3_snpsqc_dis_no-x_updated_5.batches.csv
    Untracked:  data/ici.vs.pbs_blup.qc_chr-3_snpsqc_dis_no-x_updated_5.batches_mis.csv
    Untracked:  data/ici.vs.pbs_blup.qc_chr-4_5.batches.csv
    Untracked:  data/ici.vs.pbs_blup.qc_chr-4_5.batches_mis.csv
    Untracked:  data/ici.vs.pbs_blup.qc_chr-4_lod.drop-1.5_5.batches.csv
    Untracked:  data/ici.vs.pbs_blup.qc_chr-4_lod.drop-1.5_5.batches_mis.csv
    Untracked:  data/ici.vs.pbs_blup.qc_chr-4_snpsqc_5.batches.csv
    Untracked:  data/ici.vs.pbs_blup.qc_chr-4_snpsqc_5.batches_mis.csv
    Untracked:  data/ici.vs.pbs_blup.qc_chr-4_snpsqc_dis_no-x_updated_5.batches.csv
    Untracked:  data/ici.vs.pbs_blup.qc_chr-4_snpsqc_dis_no-x_updated_5.batches_mis.csv
    Untracked:  data/ici.vs.pbs_blup.qc_chr-5_5.batches.csv
    Untracked:  data/ici.vs.pbs_blup.qc_chr-5_5.batches_mis.csv
    Untracked:  data/ici.vs.pbs_blup.qc_chr-5_lod.drop-1.5_5.batches.csv
    Untracked:  data/ici.vs.pbs_blup.qc_chr-5_lod.drop-1.5_5.batches_mis.csv
    Untracked:  data/ici.vs.pbs_blup.qc_chr-5_snpsqc_5.batches.csv
    Untracked:  data/ici.vs.pbs_blup.qc_chr-5_snpsqc_5.batches_mis.csv
    Untracked:  data/ici.vs.pbs_blup.qc_chr-5_snpsqc_dis_no-x_updated_5.batches.csv
    Untracked:  data/ici.vs.pbs_blup.qc_chr-5_snpsqc_dis_no-x_updated_5.batches_mis.csv
    Untracked:  data/ici.vs.pbs_blup.qc_chr-6_5.batches.csv
    Untracked:  data/ici.vs.pbs_blup.qc_chr-6_5.batches_mis.csv
    Untracked:  data/ici.vs.pbs_blup.qc_chr-6_lod.drop-1.5_5.batches.csv
    Untracked:  data/ici.vs.pbs_blup.qc_chr-6_lod.drop-1.5_5.batches_mis.csv
    Untracked:  data/ici.vs.pbs_blup.qc_chr-6_snpsqc_5.batches.csv
    Untracked:  data/ici.vs.pbs_blup.qc_chr-6_snpsqc_5.batches_mis.csv
    Untracked:  data/ici.vs.pbs_blup.qc_chr-6_snpsqc_dis_no-x_updated_5.batches.csv
    Untracked:  data/ici.vs.pbs_blup.qc_chr-6_snpsqc_dis_no-x_updated_5.batches_mis.csv
    Untracked:  data/ici.vs.pbs_blup.qc_chr-7_5.batches.csv
    Untracked:  data/ici.vs.pbs_blup.qc_chr-7_5.batches_mis.csv
    Untracked:  data/ici.vs.pbs_blup.qc_chr-7_lod.drop-1.5_5.batches.csv
    Untracked:  data/ici.vs.pbs_blup.qc_chr-7_lod.drop-1.5_5.batches_mis.csv
    Untracked:  data/ici.vs.pbs_blup.qc_chr-7_snpsqc_5.batches.csv
    Untracked:  data/ici.vs.pbs_blup.qc_chr-7_snpsqc_5.batches_mis.csv
    Untracked:  data/ici.vs.pbs_blup.qc_chr-7_snpsqc_dis_no-x_updated_5.batches.csv
    Untracked:  data/ici.vs.pbs_blup.qc_chr-7_snpsqc_dis_no-x_updated_5.batches_mis.csv
    Untracked:  data/ici.vs.pbs_blup.qc_chr-8_5.batches.csv
    Untracked:  data/ici.vs.pbs_blup.qc_chr-8_5.batches_mis.csv
    Untracked:  data/ici.vs.pbs_blup.qc_chr-8_lod.drop-1.5_5.batches.csv
    Untracked:  data/ici.vs.pbs_blup.qc_chr-8_lod.drop-1.5_5.batches_mis.csv
    Untracked:  data/ici.vs.pbs_blup.qc_chr-8_snpsqc_5.batches.csv
    Untracked:  data/ici.vs.pbs_blup.qc_chr-8_snpsqc_5.batches_mis.csv
    Untracked:  data/ici.vs.pbs_blup.qc_chr-8_snpsqc_dis_no-x_updated_5.batches.csv
    Untracked:  data/ici.vs.pbs_blup.qc_chr-8_snpsqc_dis_no-x_updated_5.batches_mis.csv
    Untracked:  data/ici.vs.pbs_blup.qc_chr-9_5.batches.csv
    Untracked:  data/ici.vs.pbs_blup.qc_chr-9_5.batches_mis.csv
    Untracked:  data/ici.vs.pbs_blup.qc_chr-9_lod.drop-1.5_5.batches.csv
    Untracked:  data/ici.vs.pbs_blup.qc_chr-9_lod.drop-1.5_5.batches_mis.csv
    Untracked:  data/ici.vs.pbs_blup.qc_chr-9_snpsqc_5.batches.csv
    Untracked:  data/ici.vs.pbs_blup.qc_chr-9_snpsqc_5.batches_mis.csv
    Untracked:  data/ici.vs.pbs_blup.qc_chr-9_snpsqc_dis_no-x_updated_5.batches.csv
    Untracked:  data/ici.vs.pbs_blup.qc_chr-9_snpsqc_dis_no-x_updated_5.batches_mis.csv
    Untracked:  data/ici.vs.pbs_blup.qc_chr-X_5.batches.csv
    Untracked:  data/ici.vs.pbs_blup.qc_chr-X_5.batches_mis.csv
    Untracked:  data/ici.vs.pbs_blup.qc_chr-X_lod.drop-1.5_5.batches.csv
    Untracked:  data/ici.vs.pbs_blup.qc_chr-X_snpsqc_5.batches.csv
    Untracked:  data/ici.vs.pbs_blup.qc_chr-X_snpsqc_5.batches_mis.csv
    Untracked:  data/ici.vs.pbs_blup.qc_chr-X_snpsqc_dis_no-x_updated_5.batches.csv
    Untracked:  data/ici.vs.pbs_blup.qc_chr-X_snpsqc_dis_no-x_updated_5.batches_mis.csv
    Untracked:  data/ici.vs.pbs_blup_chr-10_lod.drop-1.5_5.batches.csv
    Untracked:  data/ici.vs.pbs_blup_chr-10_lod.drop-1.5_snpsqc_5.batches.csv
    Untracked:  data/ici.vs.pbs_blup_chr-10_lod.drop-1.5_snpsqc_5.batches_mis.csv
    Untracked:  data/ici.vs.pbs_blup_chr-10_lod.drop-1.5_snpsqc_dis_no-x_updated_5.batches.csv
    Untracked:  data/ici.vs.pbs_blup_chr-10_lod.drop-1.5_snpsqc_dis_no-x_updated_5.batches_mis.csv
    Untracked:  data/ici.vs.pbs_blup_chr-11_lod.drop-1.5_5.batches.csv
    Untracked:  data/ici.vs.pbs_blup_chr-11_lod.drop-1.5_snpsqc_5.batches.csv
    Untracked:  data/ici.vs.pbs_blup_chr-11_lod.drop-1.5_snpsqc_5.batches_mis.csv
    Untracked:  data/ici.vs.pbs_blup_chr-11_lod.drop-1.5_snpsqc_dis_no-x_updated_5.batches.csv
    Untracked:  data/ici.vs.pbs_blup_chr-11_lod.drop-1.5_snpsqc_dis_no-x_updated_5.batches_mis.csv
    Untracked:  data/ici.vs.pbs_blup_chr-12_lod.drop-1.5_5.batches.csv
    Untracked:  data/ici.vs.pbs_blup_chr-12_lod.drop-1.5_snpsqc_5.batches.csv
    Untracked:  data/ici.vs.pbs_blup_chr-12_lod.drop-1.5_snpsqc_5.batches_mis.csv
    Untracked:  data/ici.vs.pbs_blup_chr-12_lod.drop-1.5_snpsqc_dis_no-x_updated_5.batches.csv
    Untracked:  data/ici.vs.pbs_blup_chr-12_lod.drop-1.5_snpsqc_dis_no-x_updated_5.batches_mis.csv
    Untracked:  data/ici.vs.pbs_blup_chr-13_lod.drop-1.5_5.batches.csv
    Untracked:  data/ici.vs.pbs_blup_chr-13_lod.drop-1.5_snpsqc_5.batches.csv
    Untracked:  data/ici.vs.pbs_blup_chr-13_lod.drop-1.5_snpsqc_5.batches_mis.csv
    Untracked:  data/ici.vs.pbs_blup_chr-13_lod.drop-1.5_snpsqc_dis_no-x_updated_5.batches.csv
    Untracked:  data/ici.vs.pbs_blup_chr-13_lod.drop-1.5_snpsqc_dis_no-x_updated_5.batches_mis.csv
    Untracked:  data/ici.vs.pbs_blup_chr-14_lod.drop-1.5_5.batches.csv
    Untracked:  data/ici.vs.pbs_blup_chr-14_lod.drop-1.5_snpsqc_5.batches.csv
    Untracked:  data/ici.vs.pbs_blup_chr-14_lod.drop-1.5_snpsqc_5.batches_mis.csv
    Untracked:  data/ici.vs.pbs_blup_chr-14_lod.drop-1.5_snpsqc_dis_no-x_updated_5.batches.csv
    Untracked:  data/ici.vs.pbs_blup_chr-14_lod.drop-1.5_snpsqc_dis_no-x_updated_5.batches_mis.csv
    Untracked:  data/ici.vs.pbs_blup_chr-15_lod.drop-1.5_5.batches.csv
    Untracked:  data/ici.vs.pbs_blup_chr-15_lod.drop-1.5_snpsqc_5.batches.csv
    Untracked:  data/ici.vs.pbs_blup_chr-15_lod.drop-1.5_snpsqc_5.batches_mis.csv
    Untracked:  data/ici.vs.pbs_blup_chr-16_lod.drop-1.5_5.batches.csv
    Untracked:  data/ici.vs.pbs_blup_chr-16_lod.drop-1.5_snpsqc_5.batches.csv
    Untracked:  data/ici.vs.pbs_blup_chr-16_lod.drop-1.5_snpsqc_5.batches_mis.csv
    Untracked:  data/ici.vs.pbs_blup_chr-16_lod.drop-1.5_snpsqc_dis_no-x_updated_5.batches.csv
    Untracked:  data/ici.vs.pbs_blup_chr-16_lod.drop-1.5_snpsqc_dis_no-x_updated_5.batches_mis.csv
    Untracked:  data/ici.vs.pbs_blup_chr-17_lod.drop-1.5_5.batches.csv
    Untracked:  data/ici.vs.pbs_blup_chr-17_lod.drop-1.5_snpsqc_5.batches.csv
    Untracked:  data/ici.vs.pbs_blup_chr-17_lod.drop-1.5_snpsqc_5.batches_mis.csv
    Untracked:  data/ici.vs.pbs_blup_chr-18_lod.drop-1.5_5.batches.csv
    Untracked:  data/ici.vs.pbs_blup_chr-18_lod.drop-1.5_snpsqc_5.batches.csv
    Untracked:  data/ici.vs.pbs_blup_chr-18_lod.drop-1.5_snpsqc_5.batches_mis.csv
    Untracked:  data/ici.vs.pbs_blup_chr-18_lod.drop-1.5_snpsqc_dis_no-x_updated_5.batches.csv
    Untracked:  data/ici.vs.pbs_blup_chr-18_lod.drop-1.5_snpsqc_dis_no-x_updated_5.batches_mis.csv
    Untracked:  data/ici.vs.pbs_blup_chr-19_lod.drop-1.5_5.batches.csv
    Untracked:  data/ici.vs.pbs_blup_chr-19_lod.drop-1.5_snpsqc_5.batches.csv
    Untracked:  data/ici.vs.pbs_blup_chr-19_lod.drop-1.5_snpsqc_5.batches_mis.csv
    Untracked:  data/ici.vs.pbs_blup_chr-19_lod.drop-1.5_snpsqc_dis_no-x_updated_5.batches.csv
    Untracked:  data/ici.vs.pbs_blup_chr-19_lod.drop-1.5_snpsqc_dis_no-x_updated_5.batches_mis.csv
    Untracked:  data/ici.vs.pbs_blup_chr-1_lod.drop-1.5_5.batches.csv
    Untracked:  data/ici.vs.pbs_blup_chr-1_lod.drop-1.5_snpsqc_5.batches.csv
    Untracked:  data/ici.vs.pbs_blup_chr-1_lod.drop-1.5_snpsqc_5.batches_mis.csv
    Untracked:  data/ici.vs.pbs_blup_chr-1_lod.drop-1.5_snpsqc_dis_no-x_updated_5.batches.csv
    Untracked:  data/ici.vs.pbs_blup_chr-1_lod.drop-1.5_snpsqc_dis_no-x_updated_5.batches_mis.csv
    Untracked:  data/ici.vs.pbs_blup_chr-2_lod.drop-1.5_5.batches.csv
    Untracked:  data/ici.vs.pbs_blup_chr-2_lod.drop-1.5_snpsqc_5.batches.csv
    Untracked:  data/ici.vs.pbs_blup_chr-2_lod.drop-1.5_snpsqc_5.batches_mis.csv
    Untracked:  data/ici.vs.pbs_blup_chr-2_lod.drop-1.5_snpsqc_dis_no-x_updated_5.batches.csv
    Untracked:  data/ici.vs.pbs_blup_chr-2_lod.drop-1.5_snpsqc_dis_no-x_updated_5.batches_mis.csv
    Untracked:  data/ici.vs.pbs_blup_chr-3_lod.drop-1.5_5.batches.csv
    Untracked:  data/ici.vs.pbs_blup_chr-3_lod.drop-1.5_snpsqc_5.batches.csv
    Untracked:  data/ici.vs.pbs_blup_chr-3_lod.drop-1.5_snpsqc_5.batches_mis.csv
    Untracked:  data/ici.vs.pbs_blup_chr-3_lod.drop-1.5_snpsqc_dis_no-x_updated_5.batches.csv
    Untracked:  data/ici.vs.pbs_blup_chr-3_lod.drop-1.5_snpsqc_dis_no-x_updated_5.batches_mis.csv
    Untracked:  data/ici.vs.pbs_blup_chr-4_lod.drop-1.5_5.batches.csv
    Untracked:  data/ici.vs.pbs_blup_chr-4_lod.drop-1.5_snpsqc_5.batches.csv
    Untracked:  data/ici.vs.pbs_blup_chr-4_lod.drop-1.5_snpsqc_5.batches_mis.csv
    Untracked:  data/ici.vs.pbs_blup_chr-4_lod.drop-1.5_snpsqc_dis_no-x_updated_5.batches.csv
    Untracked:  data/ici.vs.pbs_blup_chr-4_lod.drop-1.5_snpsqc_dis_no-x_updated_5.batches_mis.csv
    Untracked:  data/ici.vs.pbs_blup_chr-5_lod.drop-1.5_5.batches.csv
    Untracked:  data/ici.vs.pbs_blup_chr-5_lod.drop-1.5_snpsqc_5.batches.csv
    Untracked:  data/ici.vs.pbs_blup_chr-5_lod.drop-1.5_snpsqc_5.batches_mis.csv
    Untracked:  data/ici.vs.pbs_blup_chr-5_lod.drop-1.5_snpsqc_dis_no-x_updated_5.batches.csv
    Untracked:  data/ici.vs.pbs_blup_chr-5_lod.drop-1.5_snpsqc_dis_no-x_updated_5.batches_mis.csv
    Untracked:  data/ici.vs.pbs_blup_chr-6_lod.drop-1.5_5.batches.csv
    Untracked:  data/ici.vs.pbs_blup_chr-6_lod.drop-1.5_snpsqc_5.batches.csv
    Untracked:  data/ici.vs.pbs_blup_chr-6_lod.drop-1.5_snpsqc_5.batches_mis.csv
    Untracked:  data/ici.vs.pbs_blup_chr-6_lod.drop-1.5_snpsqc_dis_no-x_updated_5.batches.csv
    Untracked:  data/ici.vs.pbs_blup_chr-6_lod.drop-1.5_snpsqc_dis_no-x_updated_5.batches_mis.csv
    Untracked:  data/ici.vs.pbs_blup_chr-7_lod.drop-1.5_5.batches.csv
    Untracked:  data/ici.vs.pbs_blup_chr-7_lod.drop-1.5_snpsqc_5.batches.csv
    Untracked:  data/ici.vs.pbs_blup_chr-7_lod.drop-1.5_snpsqc_5.batches_mis.csv
    Untracked:  data/ici.vs.pbs_blup_chr-7_lod.drop-1.5_snpsqc_dis_no-x_updated_5.batches.csv
    Untracked:  data/ici.vs.pbs_blup_chr-7_lod.drop-1.5_snpsqc_dis_no-x_updated_5.batches_mis.csv
    Untracked:  data/ici.vs.pbs_blup_chr-8_lod.drop-1.5_5.batches.csv
    Untracked:  data/ici.vs.pbs_blup_chr-8_lod.drop-1.5_snpsqc_5.batches.csv
    Untracked:  data/ici.vs.pbs_blup_chr-8_lod.drop-1.5_snpsqc_5.batches_mis.csv
    Untracked:  data/ici.vs.pbs_blup_chr-8_lod.drop-1.5_snpsqc_dis_no-x_updated_5.batches.csv
    Untracked:  data/ici.vs.pbs_blup_chr-8_lod.drop-1.5_snpsqc_dis_no-x_updated_5.batches_mis.csv
    Untracked:  data/ici.vs.pbs_blup_chr-9_lod.drop-1.5_5.batches.csv
    Untracked:  data/ici.vs.pbs_blup_chr-9_lod.drop-1.5_snpsqc_5.batches.csv
    Untracked:  data/ici.vs.pbs_blup_chr-9_lod.drop-1.5_snpsqc_5.batches_mis.csv
    Untracked:  data/ici.vs.pbs_blup_chr-9_lod.drop-1.5_snpsqc_dis_no-x_updated_5.batches.csv
    Untracked:  data/ici.vs.pbs_blup_chr-9_lod.drop-1.5_snpsqc_dis_no-x_updated_5.batches_mis.csv
    Untracked:  data/ici.vs.pbs_blup_chr-X_lod.drop-1.5_5.batches.csv
    Untracked:  data/ici.vs.pbs_blup_chr-X_lod.drop-1.5_snpsqc_5.batches.csv
    Untracked:  data/ici.vs.pbs_blup_chr-X_lod.drop-1.5_snpsqc_5.batches_mis.csv
    Untracked:  data/ici.vs.pbs_blup_chr-X_lod.drop-1.5_snpsqc_dis_no-x_updated_5.batches.csv
    Untracked:  data/ici.vs.pbs_blup_chr-X_lod.drop-1.5_snpsqc_dis_no-x_updated_5.batches_mis.csv
    Untracked:  data/ici.vs.pbs_blup_sub_chr-13_peak.marker-UNC22384241_lod.drop-1.5_snpsqc_snpsqc_5.batches_peaks.csv
    Untracked:  data/ici.vs.pbs_blup_sub_chr-18_peak.marker-UNCHS045343_lod.drop-1.5_snpsqc_5.batches_mis.csv
    Untracked:  data/ici.vs.pbs_blup_sub_chr-19_peak.marker-UNC29893228_lod.drop-1.5_snpsqc_snpsqc_5.batches_peaks.csv
    Untracked:  data/ici.vs.pbs_blup_sub_chr-19_peak.marker-UNC29990033_lod.drop-1.5_snpsqc_dis_no-x_updated_5.batches_peaks.csv
    Untracked:  data/ici.vs.pbs_blup_sub_chr-19_peak.marker-UNCHS047192_lod.drop-1.5_snpsqc_dis_no-x_updated_5.batches_mis_peaks.csv
    Untracked:  data/ici.vs.pbs_blup_sub_chr-8_peak.marker-UNCHS023353_lod.drop-1.5_snpsqc_dis_no-x_updated_5.batches_mis_peaks.csv
    Untracked:  data/ici.vs.pbs_blup_sub_chr-8_peak.marker-UNCHS023353_lod.drop-1.5_snpsqc_dis_no-x_updated_5.batches_peaks.csv
    Untracked:  data/ici.vs.pbs_blup_sub_chr-8_peak.marker-UNCHS023364_lod.drop-1.5_snpsqc_snpsqc_5.batches_peaks.csv
    Untracked:  data/ici.vs.pbs_fitqtl_peaks_additive_snpsqc_dis_no-x_updated_5.batches_mis_peaks.txt
    Untracked:  data/ici.vs.pbs_fitqtl_peaks_additive_snpsqc_dis_no-x_updated_5.batches_peaks.txt
    Untracked:  data/ici.vs.pbs_fitqtl_peaks_additive_snpsqc_snpsqc_5.batches_peaks.txt
    Untracked:  data/ici.vs.pbs_fitqtl_peaks_interacting_snpsqc_dis_no-x_updated_5.batches_mis_peaks.txt
    Untracked:  data/ici.vs.pbs_fitqtl_peaks_interacting_snpsqc_dis_no-x_updated_5.batches_peaks.txt
    Untracked:  data/ici.vs.pbs_fitqtl_peaks_interacting_snpsqc_snpsqc_5.batches_peaks.txt
    Untracked:  data/ici.vs.pbs_fitqtl_peaks_sex_additive_snpsqc_dis_no-x_updated_5.batches_mis_peaks.txt
    Untracked:  data/ici.vs.pbs_fitqtl_peaks_sex_additive_snpsqc_dis_no-x_updated_5.batches_peaks.txt
    Untracked:  data/ici.vs.pbs_fitqtl_peaks_sex_additive_snpsqc_snpsqc_5.batches_peaks.txt
    Untracked:  data/ici.vs.pbs_fitqtl_peaks_sex_interacting_snpsqc_dis_no-x_updated_5.batches_mis_peaks.txt
    Untracked:  data/ici.vs.pbs_fitqtl_peaks_sex_interacting_snpsqc_dis_no-x_updated_5.batches_peaks.txt
    Untracked:  data/ici.vs.pbs_fitqtl_peaks_sex_interacting_snpsqc_snpsqc_5.batches_peaks.txt
    Untracked:  data/ici.vs.pbs_genes_chr-13_peak.marker-UNC22384241_lod.drop-1.5_snpsqc_snpsqc_5.batches_peaks.csv
    Untracked:  data/ici.vs.pbs_genes_chr-18_peak.marker-UNCHS045343_lod.drop-1.5_snpsqc_5.batches_mis.csv
    Untracked:  data/ici.vs.pbs_genes_chr-19_peak.marker-UNC29893228_lod.drop-1.5_snpsqc_snpsqc_5.batches_peaks.csv
    Untracked:  data/ici.vs.pbs_genes_chr-19_peak.marker-UNC29990033_lod.drop-1.5_snpsqc_dis_no-x_updated_5.batches_peaks.csv
    Untracked:  data/ici.vs.pbs_genes_chr-19_peak.marker-UNCHS047192_lod.drop-1.5_snpsqc_dis_no-x_updated_5.batches_mis_peaks.csv
    Untracked:  data/ici.vs.pbs_genes_chr-8_peak.marker-UNCHS023353_lod.drop-1.5_snpsqc_dis_no-x_updated_5.batches_mis_peaks.csv
    Untracked:  data/ici.vs.pbs_genes_chr-8_peak.marker-UNCHS023353_lod.drop-1.5_snpsqc_dis_no-x_updated_5.batches_peaks.csv
    Untracked:  data/ici.vs.pbs_genes_chr-8_peak.marker-UNCHS023364_lod.drop-1.5_snpsqc_snpsqc_5.batches_peaks.csv
    Untracked:  data/ici.vs.pbs_gm_qtl_5.batches.csv
    Untracked:  data/ici.vs.pbs_gm_qtl_5.batches_mis.csv
    Untracked:  data/ici.vs.pbs_gm_qtl_snpsqc_5.batches.csv
    Untracked:  data/ici.vs.pbs_gm_qtl_snpsqc_5.batches_mis.csv
    Untracked:  data/ici.vs.pbs_gm_qtl_snpsqc_5.batches_peaks.csv
    Untracked:  data/ici.vs.pbs_gm_qtl_snpsqc_dis_no-x_updated_5.batches.csv
    Untracked:  data/ici.vs.pbs_gm_qtl_snpsqc_dis_no-x_updated_5.batches_mis.csv
    Untracked:  data/ici.vs.pbs_gm_qtl_snpsqc_dis_no-x_updated_5.batches_mis_peaks.csv
    Untracked:  data/ici.vs.pbs_gm_qtl_snpsqc_dis_no-x_updated_5.batches_peaks.csv
    Untracked:  data/ici.vs.pbs_marker.freq_low.geno.freq.removed_geno.ratio.csv
    Untracked:  data/ici.vs.pbs_marker.freq_low.geno.freq.removed_geno.ratio_5.batches.csv
    Untracked:  data/ici.vs.pbs_marker.freq_low.geno.freq.removed_geno.ratio_5.batches_mis.csv
    Untracked:  data/ici.vs.pbs_marker.freq_low.geno.freq.removed_sample.outliers.removed_geno.ratio.csv
    Untracked:  data/ici.vs.pbs_marker.freq_low.geno.freq.removed_sample.outliers.removed_geno.ratio_5.batches.csv
    Untracked:  data/ici.vs.pbs_marker.freq_low.geno.freq.removed_sample.outliers.removed_geno.ratio_5.batches_mis.csv
    Untracked:  data/ici.vs.pbs_marker.freq_low.probs.freq.removed_geno.ratio.csv
    Untracked:  data/ici.vs.pbs_marker.freq_low.probs.freq.removed_geno.ratio_5.batches.csv
    Untracked:  data/ici.vs.pbs_marker.freq_low.probs.freq.removed_geno.ratio_5.batches_mis.csv
    Untracked:  data/ici.vs.pbs_marker.freq_low.probs.freq.removed_sample.outliers.removed_geno.ratio.csv
    Untracked:  data/ici.vs.pbs_marker.freq_low.probs.freq.removed_sample.outliers.removed_geno.ratio_5.batches.csv
    Untracked:  data/ici.vs.pbs_marker.freq_low.probs.freq.removed_sample.outliers.removed_geno.ratio_5.batches_mis.csv
    Untracked:  data/ici.vs.pbs_rz.age-additive.covariates_blup_sub_chr-7_peak.marker-JAX00153527_lod.drop-1.5_5.batches.csv
    Untracked:  data/ici.vs.pbs_rz.age-additive.covariates_blup_sub_chr-7_peak.marker-JAX00153527_lod.drop-1.5_snpsqc_5.batches.csv
    Untracked:  data/ici.vs.pbs_rz.age-additive.covariates_blup_sub_chr-7_peak.marker-UNCHS020443_lod.drop-1.5_snpsqc_dis_no-x_updated_5.batches.csv
    Untracked:  data/ici.vs.pbs_rz.age-additive.covariates_blup_sub_chr-7_peak.marker-UNCHS020443_lod.drop-1.5_snpsqc_dis_no-x_updated_5.batches_mis.csv
    Untracked:  data/ici.vs.pbs_rz.age-additive.covariates_blup_sub_chr-7_peak.marker-UNCHS020486_lod.drop-1.5_snpsqc_5.batches_mis.csv
    Untracked:  data/ici.vs.pbs_rz.age-additive.covariates_genes_chr-7_peak.marker-JAX00153527_lod.drop-1.5_5.batches.csv
    Untracked:  data/ici.vs.pbs_rz.age-additive.covariates_genes_chr-7_peak.marker-JAX00153527_lod.drop-1.5_snpsqc_5.batches.csv
    Untracked:  data/ici.vs.pbs_rz.age-additive.covariates_genes_chr-7_peak.marker-UNCHS020443_lod.drop-1.5_snpsqc_dis_no-x_updated_5.batches.csv
    Untracked:  data/ici.vs.pbs_rz.age-additive.covariates_genes_chr-7_peak.marker-UNCHS020443_lod.drop-1.5_snpsqc_dis_no-x_updated_5.batches_mis.csv
    Untracked:  data/ici.vs.pbs_rz.age-additive.covariates_genes_chr-7_peak.marker-UNCHS020486_lod.drop-1.5_snpsqc_5.batches_mis.csv
    Untracked:  data/ici.vs.pbs_sample.genos_marker.freq_low.geno.freq.removed.csv
    Untracked:  data/ici.vs.pbs_sample.genos_marker.freq_low.geno.freq.removed_sample.outliers.removed.csv
    Untracked:  data/ici.vs.pbs_sample.genos_marker.freq_low.probs.freq.removed.csv
    Untracked:  data/ici.vs.pbs_sample.genos_marker.freq_low.probs.freq.removed_sample.outliers.removed.csv
    Untracked:  data/ici.vs.pbs_scanone_5.batches.Rdata
    Untracked:  data/ici.vs.pbs_scanone_5.batches_mis.Rdata
    Untracked:  data/ici.vs.pbs_scanone_snpsqc_5.batches.Rdata
    Untracked:  data/ici.vs.pbs_scanone_snpsqc_5.batches_mis.Rdata
    Untracked:  data/ici.vs.pbs_scanone_snpsqc_dis_no-x_updated_5.batches.Rdata
    Untracked:  data/ici.vs.pbs_scanone_snpsqc_dis_no-x_updated_5.batches_mis.Rdata
    Untracked:  data/percent_missing_id_3.batches.RData
    Untracked:  data/percent_missing_id_4.batches.RData
    Untracked:  data/percent_missing_id_4.batches_bc.RData
    Untracked:  data/percent_missing_id_5.batches.RData
    Untracked:  data/percent_missing_marker_4.batches.RData
    Untracked:  data/percent_missing_marker_5.batches.RData
    Untracked:  data/pheno.csv
    Untracked:  data/pheno_BC312.csv
    Untracked:  data/physical_map.csv
    Untracked:  data/physical_map_BC312.csv
    Untracked:  data/qc_info_bad_sample_3.batches.RData
    Untracked:  data/qc_info_bad_sample_4.batches.RData
    Untracked:  data/qc_info_bad_sample_4.batches_bc.RData
    Untracked:  data/qc_info_bad_sample_5.batches.RData
    Untracked:  data/remaining.markers_geno.freq.xlsx
    Untracked:  data/sample_geno.csv
    Untracked:  data/sample_geno_AHB_BC312.csv
    Untracked:  data/sample_geno_bc.csv
    Untracked:  data/sample_geno_bc_BC312.csv
    Untracked:  data/serreze_probs.rds
    Untracked:  data/serreze_probs_BC312.rds
    Untracked:  data/serreze_probs_allqc.rds
    Untracked:  data/serreze_probs_allqc_5.batches.rds
    Untracked:  data/serreze_probs_allqc_5.batches_mis.rds
    Untracked:  data/summary.cg_3.batches.RData
    Untracked:  data/summary.cg_4.batches.RData
    Untracked:  data/summary.cg_4.batches_bc.RData
    Untracked:  data/summary.cg_5.batches.RData
    Untracked:  output/Percent_missing_genotype_data_5.batches.pdf
    Untracked:  output/Percent_missing_genotype_data_per_marker_5.batches.pdf
    Untracked:  output/Proportion_matching_genotypes_before_removal_of_bad_samples_5.batches.pdf
    Untracked:  output/genotype_error_marker_5.batches.pdf
    Untracked:  output/genotype_frequency_marker_5.batches.pdf

Unstaged changes:
    Modified:   analysis/4.1.1_qtl.analysis_binary_ici-early.vs.pbs_5.batches.Rmd
    Modified:   analysis/4.1.1_qtl.analysis_binary_ici-early.vs.pbs_5.batches_mis.Rmd
    Modified:   analysis/4.1.1_qtl.analysis_binary_ici-early.vs.pbs_snpsqc_5.batches.Rmd
    Modified:   analysis/4.1.1_qtl.analysis_binary_ici-early.vs.pbs_snpsqc_5.batches_mis.Rmd
    Modified:   analysis/4.1.1_qtl.analysis_binary_ici-early.vs.pbs_snpsqc_dis_no-x_updated_5.batches.Rmd
    Modified:   analysis/4.1.1_qtl.analysis_binary_ici-early.vs.pbs_snpsqc_dis_no-x_updated_5.batches_mis.Rmd
    Modified:   analysis/4.1.1_qtl.analysis_binary_ici.vs.eoi_snpsqc_dis_no-x_updated.Rmd
    Modified:   analysis/4.1.1_qtl.analysis_binary_ici.vs.pbs_snpsqc_dis_no-x_updated.Rmd
    Modified:   analysis/4.1.2_qtl.analysis_cont_age_ici-early.vs.pbs_pheno.corrected.cleaned_5.batches.Rmd
    Modified:   analysis/4.1.2_qtl.analysis_cont_age_ici-early.vs.pbs_pheno.corrected.cleaned_5.batches_mis.Rmd
    Modified:   analysis/4.1.2_qtl.analysis_cont_age_ici-early.vs.pbs_snpsqc_pheno.corrected.cleaned_5.batches.Rmd
    Modified:   analysis/4.1.2_qtl.analysis_cont_age_ici-early.vs.pbs_snpsqc_pheno.corrected.cleaned_5.batches_mis.Rmd
    Modified:   analysis/4.1.2_qtl.analysis_cont_age_ici-early.vs.pbs_snpsqc_pheno.corrected.cleaned_dis_no-xk_5.batches.Rmd
    Modified:   analysis/4.1.2_qtl.analysis_cont_age_ici-early.vs.pbs_snpsqc_pheno.corrected.cleaned_dis_no-xk_5.batches_mis.Rmd
    Modified:   analysis/4.1.2_qtl.analysis_cont_age_ici.vs.eoi_pheno.corrected.cleaned_5.batches.Rmd
    Modified:   analysis/4.1.2_qtl.analysis_cont_age_ici.vs.eoi_pheno.corrected.cleaned_5.batches_mis.Rmd
    Modified:   analysis/4.1.2_qtl.analysis_cont_age_ici.vs.eoi_snpsqc_pheno.corrected.cleaned_5.batches.Rmd
    Modified:   analysis/4.1.2_qtl.analysis_cont_age_ici.vs.eoi_snpsqc_pheno.corrected.cleaned_5.batches_mis.Rmd
    Modified:   analysis/4.1.2_qtl.analysis_cont_age_ici.vs.eoi_snpsqc_pheno.corrected.cleaned_dis_no-xk_5.batches.Rmd
    Modified:   analysis/4.1.2_qtl.analysis_cont_age_ici.vs.eoi_snpsqc_pheno.corrected.cleaned_dis_no-xk_5.batches_mis.Rmd
    Modified:   analysis/4.1.2_qtl.analysis_cont_age_ici.vs.pbs_pheno.corrected.cleaned_5.batches.Rmd
    Modified:   analysis/4.1.2_qtl.analysis_cont_age_ici.vs.pbs_pheno.corrected.cleaned_5.batches_mis.Rmd
    Modified:   analysis/4.1.2_qtl.analysis_cont_age_ici.vs.pbs_snpsqc_pheno.corrected.cleaned_5.batches.Rmd
    Modified:   analysis/4.1.2_qtl.analysis_cont_age_ici.vs.pbs_snpsqc_pheno.corrected.cleaned_5.batches_mis.Rmd
    Modified:   analysis/4.1.2_qtl.analysis_cont_age_ici.vs.pbs_snpsqc_pheno.corrected.cleaned_dis_no-xk_5.batches.Rmd
    Modified:   analysis/4.1.2_qtl.analysis_cont_age_ici.vs.pbs_snpsqc_pheno.corrected.cleaned_dis_no-xk_5.batches_mis.Rmd
    Modified:   analysis/4.1.2_qtl.analysis_cont_rz.age_ici-early.vs.pbs_pheno.corrected.cleaned_5.batches.Rmd
    Modified:   analysis/4.1.2_qtl.analysis_cont_rz.age_ici-early.vs.pbs_pheno.corrected.cleaned_5.batches_mis.Rmd
    Modified:   analysis/4.1.2_qtl.analysis_cont_rz.age_ici-early.vs.pbs_snpsqc_pheno.corrected.cleaned_5.batches.Rmd
    Modified:   analysis/4.1.2_qtl.analysis_cont_rz.age_ici-early.vs.pbs_snpsqc_pheno.corrected.cleaned_5.batches_mis.Rmd
    Modified:   analysis/4.1.2_qtl.analysis_cont_rz.age_ici-early.vs.pbs_snpsqc_pheno.corrected.cleaned_dis_no-xk_5.batches.Rmd
    Modified:   analysis/4.1.2_qtl.analysis_cont_rz.age_ici-early.vs.pbs_snpsqc_pheno.corrected.cleaned_dis_no-xk_5.batches_mis.Rmd
    Modified:   analysis/4.1.2_qtl.analysis_cont_rz.age_ici.vs.eoi_pheno.corrected.cleaned_5.batches.Rmd
    Modified:   analysis/4.1.2_qtl.analysis_cont_rz.age_ici.vs.eoi_pheno.corrected.cleaned_5.batches_mis.Rmd
    Modified:   analysis/4.1.2_qtl.analysis_cont_rz.age_ici.vs.eoi_snpsqc_pheno.corrected.cleaned_5.batches.Rmd
    Modified:   analysis/4.1.2_qtl.analysis_cont_rz.age_ici.vs.eoi_snpsqc_pheno.corrected.cleaned_5.batches_mis.Rmd
    Modified:   analysis/4.1.2_qtl.analysis_cont_rz.age_ici.vs.eoi_snpsqc_pheno.corrected.cleaned_dis_no-xk_5.batches.Rmd
    Modified:   analysis/4.1.2_qtl.analysis_cont_rz.age_ici.vs.eoi_snpsqc_pheno.corrected.cleaned_dis_no-xk_5.batches_mis.Rmd
    Modified:   analysis/4.1.2_qtl.analysis_cont_rz.age_ici.vs.pbs_pheno.corrected.cleaned_5.batches.Rmd
    Modified:   analysis/4.1.2_qtl.analysis_cont_rz.age_ici.vs.pbs_pheno.corrected.cleaned_5.batches_mis.Rmd
    Modified:   analysis/4.1.2_qtl.analysis_cont_rz.age_ici.vs.pbs_snpsqc_pheno.corrected.cleaned_5.batches.Rmd
    Modified:   analysis/4.1.2_qtl.analysis_cont_rz.age_ici.vs.pbs_snpsqc_pheno.corrected.cleaned_5.batches_mis.Rmd
    Modified:   analysis/4.1.2_qtl.analysis_cont_rz.age_ici.vs.pbs_snpsqc_pheno.corrected.cleaned_dis_no-xk_5.batches.Rmd
    Modified:   analysis/4.1.2_qtl.analysis_cont_rz.age_ici.vs.pbs_snpsqc_pheno.corrected.cleaned_dis_no-xk_5.batches_mis.Rmd
    Modified:   analysis/index_5.batches.Rmd
    Modified:   analysis/index_5.batches_additional.Rmd

</code></pre>
<p>
Note that any generated files, e.g. HTML, png, CSS, etc., are not included in this status report because it is ok for generated content to have uncommitted changes.
</p>
</div>
</div>
</div>
</div>
<hr>
</div>
<div id="versions" class="tab-pane fade">
<p>
There are no past versions. Publish this analysis with <code>wflow_publish()</code> to start tracking its development.
</p>
<hr>
</div>
</div>
</div>
<div id="loading-data" class="section level1">
<h1>Loading Data</h1>
<pre class="r"><code>load(&quot;data/gm_allqc_5.batches_mis.RData&quot;)

#gm_allqc
gm=gm_allqc
gm</code></pre>
<pre><code>Object of class cross2 (crosstype &quot;bc&quot;)

Total individuals               308
No. genotyped individuals       308
No. phenotyped individuals      308
No. with both geno &amp; pheno      308

No. phenotypes                    1
No. covariates                    6
No. phenotype covariates          0

No. chromosomes                  20
Total markers                131356

No. markers by chr:
   1    2    3    4    5    6    7    8    9   10   11   12   13   14   15   16 
9956 9987 7848 7586 7609 7736 7399 6458 6713 6385 7143 6110 6082 5966 5346 5015 
  17   18   19    X 
5080 4605 3562 4770 </code></pre>
<pre class="r"><code>table(gm$covar$&quot;diabetic status&quot;)</code></pre>
<pre><code>
  Both Projects   Diabetes Proj ICI-No Diabetes     Myocarditis      Spont. T1D 
              1               3             164               2              34 
        T1D &lt;17          T1D&lt;17          T1D&gt;17 
             36              51              17 </code></pre>
<pre class="r"><code>table(gm$covar$group)</code></pre>
<pre><code>
 (A2 Parental) (DQ8 Parental) B6.g7 Parental            EOI             F1 
             2              2              1            164              1 
           ICI            PBS 
           104             34 </code></pre>
<pre class="r"><code>#ICI-Early vs (PBS-T1D + ICI &gt; 17 weeks of age)
##extracting animals with ici-early and pbs group status
miceinfo &lt;- gm$covar[gm$covar$group == &quot;PBS&quot; | gm$covar$group == &quot;ICI&quot;,]
table(miceinfo$group)</code></pre>
<pre><code>
ICI PBS 
104  34 </code></pre>
<pre class="r"><code>mice.ids &lt;- rownames(miceinfo)

gm &lt;- gm[mice.ids]
gm</code></pre>
<pre><code>Object of class cross2 (crosstype &quot;bc&quot;)

Total individuals               138
No. genotyped individuals       138
No. phenotyped individuals      138
No. with both geno &amp; pheno      138

No. phenotypes                    1
No. covariates                    6
No. phenotype covariates          0

No. chromosomes                  20
Total markers                131356

No. markers by chr:
   1    2    3    4    5    6    7    8    9   10   11   12   13   14   15   16 
9956 9987 7848 7586 7609 7736 7399 6458 6713 6385 7143 6110 6082 5966 5346 5015 
  17   18   19    X 
5080 4605 3562 4770 </code></pre>
<pre class="r"><code>table(gm$covar$&quot;diabetic status&quot;)</code></pre>
<pre><code>
Spont. T1D    T1D &lt;17     T1D&lt;17     T1D&gt;17 
        34         36         51         17 </code></pre>
<pre class="r"><code>table(gm$covar$group)</code></pre>
<pre><code>
ICI PBS 
104  34 </code></pre>
<pre class="r"><code>#Creating a new lable for pbs+ici late and ici-early
table(gm$covar$group)</code></pre>
<pre><code>
ICI PBS 
104  34 </code></pre>
<pre class="r"><code>gm$covar$group.new &lt;- ifelse(gm$covar$&quot;diabetic status&quot; == &quot;T1D&gt;17&quot; | gm$covar$group == &quot;PBS&quot;, &quot;PBS_ICI.Late&quot;, &quot;ICI.Early&quot;)
table(gm$covar$group.new)</code></pre>
<pre><code>
   ICI.Early PBS_ICI.Late 
          87           51 </code></pre>
<pre class="r"><code>#binary Yes/No T1D
gm$covar$ICI.Early.vs.PBS_ICI.Late &lt;- ifelse(gm$covar$group.new == &quot;PBS_ICI.Late&quot;, 0, 1)

table(gm$covar$&quot;diabetic status&quot;)</code></pre>
<pre><code>
Spont. T1D    T1D &lt;17     T1D&lt;17     T1D&gt;17 
        34         36         51         17 </code></pre>
<pre class="r"><code>table(gm$covar$group)</code></pre>
<pre><code>
ICI PBS 
104  34 </code></pre>
<pre class="r"><code>table(gm$covar$group.new)</code></pre>
<pre><code>
   ICI.Early PBS_ICI.Late 
          87           51 </code></pre>
<pre class="r"><code>table(gm$covar$ICI.Early.vs.PBS_ICI.Late)</code></pre>
<pre><code>
 0  1 
51 87 </code></pre>
<pre class="r"><code>gm.full &lt;- gm

covars &lt;- read_csv(&quot;data/covar_corrected.cleaned_ici-early.vs.pbs.ici-late_5.batches_mis.csv&quot;)
#removing any missing info
missing_ids &lt;- covars[which(covars$&quot;age.of.onset&quot;==&quot;&quot;),]$Mouse.ID
length(missing_ids)</code></pre>
<pre><code>[1] 0</code></pre>
<pre class="r"><code>missing_ids</code></pre>
<pre><code>character(0)</code></pre>
<pre class="r"><code>if(length(missing_ids) != 0){
gm &lt;- gm[paste0(&quot;-&quot;,as.character(missing_ids)),]
gm
table(gm$covar$group)
table(gm$covar$group.new)

#keeping only mouseids in covars
covars &lt;- covars[gm$covar$id,]
table(covars$group)
table(covars$group.new)
}

##removing problmetic marker

gm &lt;- drop_markers(gm, &quot;UNCHS013106&quot;)

query_variants &lt;- create_variant_query_func(&quot;/Users/corneb/Documents/MyJax/CS/Projects/support.files/qtl2/cc_variants.sqlite&quot;)
query_genes &lt;- create_gene_query_func(&quot;/Users/corneb/Documents/MyJax/CS/Projects/support.files/qtl2/mouse_genes_mgi.sqlite&quot;)


##dropping monomorphic markers within the dataset

g &lt;- do.call(&quot;cbind&quot;, gm$geno)

gf_mar &lt;- t(apply(g, 2, function(a) table(factor(a, 1:2))/sum(a != 0)))
#gn_mar &lt;- t(apply(g, 2, function(a) table(factor(a, 1:2))))

gf_mar &lt;- gf_mar[gf_mar[,2] != &quot;NaN&quot;,]

count &lt;- rowSums(gf_mar &lt;=0.05)
low_freq_df &lt;- merge(as.data.frame(gf_mar),as.data.frame(count), by=&quot;row.names&quot;,all=T)
low_freq_df[is.na(low_freq_df)] &lt;- &#39;&#39;
low_freq_df &lt;- low_freq_df[low_freq_df$count == 1,]
rownames(low_freq_df) &lt;- low_freq_df$Row.names

low_freq &lt;- find_markerpos(gm, rownames(low_freq_df))
low_freq$id &lt;- rownames(low_freq)

nrow(low_freq)</code></pre>
<pre><code>[1] 98665</code></pre>
<pre class="r"><code>low_freq_bad &lt;- merge(low_freq,low_freq_df, by=&quot;row.names&quot;,all=T)
names(low_freq_bad)[1] &lt;- c(&quot;marker&quot;)

gf_mar &lt;- gf_mar[gf_mar[,2] != &quot;NaN&quot;,]
MAF &lt;- apply(gf_mar, 1, function(x) min(x))
MAF &lt;- as.data.frame(MAF)
MAF$index &lt;- 1:nrow(gf_mar)
gf_mar_maf &lt;- merge(gf_mar,as.data.frame(MAF), by=&quot;row.names&quot;)
gf_mar_maf &lt;- gf_mar_maf[order(gf_mar_maf$index),]

gfmar &lt;- NULL
gfmar$gfmar_mar_0 &lt;- sum(gf_mar_maf$MAF==0)
gfmar$gfmar_mar_1 &lt;- sum(gf_mar_maf$MAF&lt; 0.01)
gfmar$gfmar_mar_5 &lt;- sum(gf_mar_maf$MAF&lt; 0.05)
gfmar$gfmar_mar_10 &lt;- sum(gf_mar_maf$MAF&lt; 0.10)
gfmar$gfmar_mar_15 &lt;- sum(gf_mar_maf$MAF&lt; 0.15)
gfmar$gfmar_mar_25 &lt;- sum(gf_mar_maf$MAF&lt; 0.25)
gfmar$gfmar_mar_50 &lt;- sum(gf_mar_maf$MAF&lt; 0.50)
gfmar$total_snps &lt;- nrow(as.data.frame(gf_mar_maf))

gfmar &lt;- t(as.data.frame(gfmar))
gfmar &lt;- as.data.frame(gfmar)
gfmar$count &lt;- gfmar$V1

gfmar[c(2)] %&gt;%
  kable(escape = F,align = c(&quot;ccccccccc&quot;),linesep =&quot;\\hline&quot;) %&gt;%
  kable_styling(full_width = F) %&gt;%
  kable_styling(&quot;striped&quot;, full_width = F)  %&gt;%
  row_spec(8 ,bold=T,color= &quot;white&quot;,background = &quot;black&quot;)</code></pre>
<table class="table table table-striped" style="width: auto !important; margin-left: auto; margin-right: auto; width: auto !important; margin-left: auto; margin-right: auto;">
<thead>
<tr>
<th style="text-align:left;">
</th>
<th style="text-align:center;">
count
</th>
</tr>
</thead>
<tbody>
<tr>
<td style="text-align:left;">
gfmar_mar_0
</td>
<td style="text-align:center;">
88924
</td>
</tr>
<tr>
<td style="text-align:left;">
gfmar_mar_1
</td>
<td style="text-align:center;">
92460
</td>
</tr>
<tr>
<td style="text-align:left;">
gfmar_mar_5
</td>
<td style="text-align:center;">
98665
</td>
</tr>
<tr>
<td style="text-align:left;">
gfmar_mar_10
</td>
<td style="text-align:center;">
99264
</td>
</tr>
<tr>
<td style="text-align:left;">
gfmar_mar_15
</td>
<td style="text-align:center;">
99368
</td>
</tr>
<tr>
<td style="text-align:left;">
gfmar_mar_25
</td>
<td style="text-align:center;">
100303
</td>
</tr>
<tr>
<td style="text-align:left;">
gfmar_mar_50
</td>
<td style="text-align:center;">
130381
</td>
</tr>
<tr>
<td style="text-align:left;font-weight: bold;color: white !important;background-color: black !important;">
total_snps
</td>
<td style="text-align:center;font-weight: bold;color: white !important;background-color: black !important;">
131355
</td>
</tr>
</tbody>
</table>
<pre class="r"><code>gm_qc &lt;- drop_markers(gm, low_freq_bad$marker)
gm_qc &lt;- drop_nullmarkers(gm_qc)

gm_qc</code></pre>
<pre><code>Object of class cross2 (crosstype &quot;bc&quot;)

Total individuals              138
No. genotyped individuals      138
No. phenotyped individuals     138
No. with both geno &amp; pheno     138

No. phenotypes                   1
No. covariates                   8
No. phenotype covariates         0

No. chromosomes                 20
Total markers                32690

No. markers by chr:
   1    2    3    4    5    6    7    8    9   10   11   12   13   14   15   16 
2952 2874 2058 2071 1955 2053 1862 1700 2009 1229 2069 1396 1628 1692 1064  940 
  17   18   19    X 
 401 1095 1067  575 </code></pre>
<pre class="r"><code>## dropping disproportionate markers
dismark &lt;- read.csv(&quot;data/ici-early.vs.pbs.ici-late_marker.freq_low.geno.freq.removed_sample.outliers.removed_geno.ratio_5.batches_mis.csv&quot;)
nrow(dismark)</code></pre>
<pre><code>[1] 32691</code></pre>
<pre class="r"><code>names(dismark)[1] &lt;- c(&quot;marker&quot;)
dismark &lt;- dismark[!dismark$Include,]
nrow(dismark)</code></pre>
<pre><code>[1] 24509</code></pre>
<pre class="r"><code>gm_qc_dis &lt;- drop_markers(gm_qc, dismark$marker)
gm_qc_dis &lt;- drop_nullmarkers(gm_qc_dis)

gm = gm_qc_dis
gm</code></pre>
<pre><code>Object of class cross2 (crosstype &quot;bc&quot;)

Total individuals             138
No. genotyped individuals     138
No. phenotyped individuals    138
No. with both geno &amp; pheno    138

No. phenotypes                  1
No. covariates                  8
No. phenotype covariates        0

No. chromosomes                18
Total markers                8182

No. markers by chr:
   1    2    3    4    5    6    7    8    9   10   11   12   13   14   16   18 
 192 1031    6  311  522  242  975 1475  681  159  508  560   74  333   40  706 
  19    X 
 200  167 </code></pre>
<pre class="r"><code>markers &lt;- marker_names(gm)
gmapdf &lt;- read.csv(&quot;/Users/corneb/Documents/MyJax/CS/Projects/Serreze/haplotype.reconstruction/output_5.batches/genetic_map.csv&quot;)
pmapdf &lt;- read.csv(&quot;/Users/corneb/Documents/MyJax/CS/Projects/Serreze/haplotype.reconstruction/output_5.batches/physical_map.csv&quot;)
#mapdf &lt;- merge(gmapdf,pmapdf, by=c(&quot;marker&quot;,&quot;chr&quot;), all=T)
#rownames(mapdf) &lt;- mapdf$marker
#mapdf &lt;- mapdf[markers,]
#names(mapdf) &lt;- c(&#39;marker&#39;,&#39;chr&#39;,&#39;gmapdf&#39;,&#39;pmapdf&#39;)
#mapdfnd &lt;- mapdf[!duplicated(mapdf[c(2:3)]),]

pr.qc &lt;- calc_genoprob(gm)</code></pre>
</div>
<div id="preparing-necessary-info" class="section level1">
<h1>Preparing necessary info</h1>
<pre class="r"><code>covars$age.of.onset &lt;- as.numeric(covars$age.of.onset)
Xcovar &lt;- get_x_covar(gm)
addcovar = model.matrix(~ICI.Early.vs.PBS_ICI.Late, data = covars)[,-1]
#addcovar.i = model.matrix(~ICI.Early.vs.PBS_ICI.Late, data = covars)[,-1]

K &lt;- calc_kinship(pr.qc, type = &quot;loco&quot;)
#heatmap(K[[1]])
K.overall &lt;- calc_kinship(pr.qc, type = &quot;overall&quot;)
#heatmap(K.overall)
kinship &lt;- calc_kinship(pr.qc)
heatmap(kinship)</code></pre>
<p><img src="figure/4.1.2_qtl.analysis_cont_age_ici-early.vs.pbs.ici-late_snpsqc_pheno.corrected.cleaned_dis_no-xk_5.batches_mis.Rmd/permutation%20prelim-1.png" width="912" style="display: block; margin: auto;" /></p>
<div id="no-additive-covariates" class="section level2">
<h2>No additive covariates</h2>
<div id="significance-thresholds" class="section level3">
<h3>Significance thresholds</h3>
<pre class="r"><code>operm &lt;- scan1perm(pr.qc, covars[&quot;age.of.onset&quot;], model=&quot;normal&quot;, n_perm=1000, perm_Xsp=TRUE, chr_lengths=chr_lengths(gm$gmap))

summary_table&lt;-data.frame(unclass(summary(operm, alpha=c(0.01,  0.05, 0.1))))
names(summary_table) &lt;- c(&quot;autosomes&quot;,&quot;X&quot;)
summary_table$significance.level &lt;- rownames(summary_table)

rownames(summary_table) &lt;- NULL

summary_table[c(3,1:2)] %&gt;%
  kable(escape = F,align = c(&quot;ccc&quot;)) %&gt;%
  kable_styling(&quot;striped&quot;, full_width = T) %&gt;%
  column_spec(1, bold=TRUE)</code></pre>
<table class="table table-striped" style="margin-left: auto; margin-right: auto;">
<thead>
<tr>
<th style="text-align:center;">
significance.level
</th>
<th style="text-align:center;">
autosomes
</th>
<th style="text-align:center;">
X
</th>
</tr>
</thead>
<tbody>
<tr>
<td style="text-align:center;font-weight: bold;">
0.01
</td>
<td style="text-align:center;">
3.500212
</td>
<td style="text-align:center;">
3.224159
</td>
</tr>
<tr>
<td style="text-align:center;font-weight: bold;">
0.05
</td>
<td style="text-align:center;">
2.825249
</td>
<td style="text-align:center;">
2.696672
</td>
</tr>
<tr>
<td style="text-align:center;font-weight: bold;">
0.1
</td>
<td style="text-align:center;">
2.566901
</td>
<td style="text-align:center;">
2.468319
</td>
</tr>
</tbody>
</table>
</div>
<div id="genome-wide-scan" class="section level3">
<h3>Genome-wide scan</h3>
<pre class="r"><code>plot_lod&lt;-function(out,map){
  for (i in 1:dim(out)[2]){
#par(mar=c(5.1, 6.1, 1.1, 1.1))
    ymx &lt;- maxlod(out) # overall maximum LOD score
    plot(out, map, lodcolumn=i, col=&quot;slateblue&quot;, ylim=c(0, ymx+0.5))
    #legend(&quot;topright&quot;, lwd=2, colnames(out)[i], bg=&quot;gray90&quot;)
    title(main = paste0(colnames(out)[i], &quot; - ICI-Early vs PBS &amp; ICI-Late (no covariates) [positions in cM]&quot;))
    add_threshold(map,  summary(operm,alpha=0.1), col = &#39;purple&#39;)
    add_threshold(map,  summary(operm, alpha=0.05), col = &#39;red&#39;)
    add_threshold(map,  summary(operm, alpha=0.01), col = &#39;blue&#39;)

  }
}


#print(&quot;with no kinship&quot;)
#outf &lt;- scan1(pr.qc, covars[&quot;age.of.onset&quot;], Xcovar=Xcovar, model=&quot;normal&quot;)

#plot_lod(outf,gm$gmap)

#print(&quot;with normal kinship&quot;)
#outk &lt;- scan1(pr.qc, covars[&quot;age.of.onset&quot;], Xcovar=Xcovar, model=&quot;normal&quot;, kinship=kinship)

#plot_lod(outk,gm$gmap)


#print(&quot;with loco kinship&quot;)
#outlk &lt;- scan1(pr.qc, covars[&quot;age.of.onset&quot;], Xcovar=Xcovar, model=&quot;normal&quot;, kinship=K)

#plot_lod(outlk,gm$gmap)

print(&quot;no kinship and no xcovar&quot;)</code></pre>
<p>[1] “no kinship and no xcovar”</p>
<pre class="r"><code>out &lt;- scan1(pr.qc, covars[&quot;age.of.onset&quot;], model=&quot;normal&quot;)

plot_lod(out,gm$gmap)</code></pre>
<p><img src="figure/4.1.2_qtl.analysis_cont_age_ici-early.vs.pbs.ici-late_snpsqc_pheno.corrected.cleaned_dis_no-xk_5.batches_mis.Rmd/scanone-1.png" width="912" style="display: block; margin: auto;" /></p>
<pre class="r"><code>print(paste0(&quot;number of samples in analysis = &quot;, as.data.frame(attributes(out)$sample_size)[1,]))</code></pre>
<p>[1] “number of samples in analysis = 138”</p>
</div>
<div id="lod-peaks" class="section level3">
<h3>LOD peaks</h3>
<div id="centimorgan-cm" class="section level4">
<h4>Centimorgan (cM)</h4>
<pre class="r"><code>peaks&lt;-find_peaks(out, gm$gmap, threshold=summary(operm,alpha=0.1)$A, thresholdX = summary(operm,alpha=0.1)$X, peakdrop=3, drop=1.5)


if(nrow(peaks) &gt;0){
peaks$marker &lt;- find_marker(gm$gmap, chr=peaks$chr,pos=peaks$pos)
names(peaks)[2] &lt;- c(&quot;phenotype&quot;)
peaks &lt;- peaks[-1]

rownames(peaks) &lt;- NULL
print(kable(peaks, escape = F, align = c(&quot;cccccccc&quot;), &quot;html&quot;) 
  %&gt;% kable_styling(&quot;striped&quot;, full_width = T)%&gt;%
  column_spec(1, bold=TRUE)
  )

#plot only peak chromosomes

plot_lod_chr&lt;-function(out,map,chrom){
  for (i in 1:dim(out)[2]){
    #png(filename=paste0(&quot;/Users/chenm/Documents/qtl/Jai/&quot;,colnames(out)[i],  &quot;_lod.png&quot;))
    
    #par(mar=c(5.1, 6.1, 1.1, 1.1))
    ymx &lt;- maxlod(out) # overall maximum LOD score
    plot(out, map, chr = chrom, lodcolumn=i, col=&quot;slateblue&quot;, ylim=c(0, ymx+0.5))
    #legend(&quot;topright&quot;, lwd=2, colnames(out)[i], bg=&quot;gray90&quot;)
    title(main = paste0(colnames(out)[i], &quot; - chr&quot;, chrom, &quot; - ICI-Early vs PBS &amp; ICI-Late (no covariates) [positions in cM]&quot;))
    add_threshold(map,  summary(operm,alpha=0.1), col = &#39;purple&#39;)
    add_threshold(map,  summary(operm, alpha=0.05), col = &#39;red&#39;)
    add_threshold(map,  summary(operm, alpha=0.01), col = &#39;blue&#39;)
    #for (j in 1: dim(summary_table)[1]){
    #  abline(h=summary_table[j, i],col=&quot;red&quot;)
    #  text(x=400, y =summary_table[j, i]+0.12, labels = paste(&quot;p=&quot;, row.names(summary_table)[j]))
    #}
    #dev.off()

  }
}


for(i in unique(peaks$chr)){
#for (i in 1:nrow(peaks)){
  #plot_lod_chr(out,gm$gmap, peaks$chr[i])
  plot_lod_chr(out,gm$gmap, i)
}

} else {
  print(paste0(&quot;There are no peaks that have a LOD that reaches suggestive (p&lt;0.1) level of &quot;,summary(operm,alpha=0.1)$A, &quot; [autosomes]/&quot;,summary(operm,alpha=0.1)$X, &quot; [x-chromosome]&quot;))
}</code></pre>
<p>[1] “There are no peaks that have a LOD that reaches suggestive (p&lt;0.1) level of 2.56690086175578 [autosomes]/2.46831877337068 [x-chromosome]”</p>
</div>
<div id="megabase-mb" class="section level4">
<h4>Megabase (MB)</h4>
<pre class="r"><code>peaks_mba&lt;-find_peaks(out, gm$pmap, threshold=summary(operm,alpha=0.1)$A, thresholdX = summary(operm,alpha=0.1)$X, peakdrop=3, drop=1.5)

if(nrow(peaks) &gt;0){
peaks_mba$marker &lt;- find_marker(gm$pmap, chr=peaks_mba$chr,pos=peaks_mba$pos)
names(peaks_mba)[2] &lt;- c(&quot;phenotype&quot;)
peaks_mba &lt;- peaks_mba[-1]

rownames(peaks_mba) &lt;- NULL
print(kable(peaks_mba, escape = F, align = c(&quot;cccccccc&quot;), &quot;html&quot;) 
  %&gt;% kable_styling(&quot;striped&quot;, full_width = T)%&gt;%
  column_spec(1, bold=TRUE)
  )

  plot_lod_chr_mb&lt;-function(out,map,chrom){
  for (i in 1:dim(out)[2]){
    #png(filename=paste0(&quot;/Users/chenm/Documents/qtl/Jai/&quot;,colnames(out)[i],  &quot;_lod.png&quot;))
    
    #par(mar=c(5.1, 6.1, 1.1, 1.1))
    ymx &lt;- maxlod(out) # overall maximum LOD score
    plot(out, map, chr = chrom, lodcolumn=i, col=&quot;slateblue&quot;, ylim=c(0, ymx+0.5))
    #legend(&quot;topright&quot;, lwd=2, colnames(out)[i], bg=&quot;gray90&quot;)
    title(main = paste0(colnames(out)[i], &quot; - chr&quot;, chrom, &quot; - ICI-Early vs PBS &amp; ICI-Late (no covariates) [positions in cM]&quot;))
    add_threshold(map,  summary(operm,alpha=0.1), col = &#39;purple&#39;)
    add_threshold(map,  summary(operm, alpha=0.05), col = &#39;red&#39;)
    add_threshold(map,  summary(operm, alpha=0.01), col = &#39;blue&#39;)
    #for (j in 1: dim(summary_table)[1]){
    #  abline(h=summary_table[j, i],col=&quot;red&quot;)
    #  text(x=400, y =summary_table[j, i]+0.12, labels = paste(&quot;p=&quot;, row.names(summary_table)[j]))
    #}
    #dev.off()
  }
}

for(i in unique(peaks_mba$chr)){
#for (i in 1:nrow(peaks_mba)){
  #plot_lod_chr_mb(out,gm$pmap, peaks_mba$chr[i])
  plot_lod_chr_mb(out,gm$pmap,i)
}

} else {
  print(paste0(&quot;There are no peaks that have a LOD that reaches suggestive (p&lt;0.1) level of &quot;,summary(operm,alpha=0.1)$A, &quot; [autosomes]/&quot;,summary(operm,alpha=0.1)$X, &quot; [x-chromosome]&quot;))
}</code></pre>
<p>[1] “There are no peaks that have a LOD that reaches suggestive (p&lt;0.1) level of 2.56690086175578 [autosomes]/2.46831877337068 [x-chromosome]”</p>
</div>
</div>
<div id="qtl-effects" class="section level3">
<h3>QTL effects</h3>
<p>For each peak LOD location we give a list of gene</p>
<pre class="r"><code>if(nrow(peaks) &gt;0){
for (i in 1:nrow(peaks)){
#for (i in 1:1){
  #Plot 1
  g &lt;- maxmarg(pr.qc, gm$gmap, chr=peaks$chr[i], pos=peaks$pos[i], return_char=TRUE)
  #png(filename=paste0(&quot;/Users/chenm/Documents/qtl/Jai/&quot;,&quot;qtl_effect_&quot;, i, &quot;.png&quot;))
  #par(mar=c(4.1, 4.1, 1.5, 0.6))
  plot_pxg(g, covars[,peaks$phenotype[i]], ylab=peaks$phenotype[i], sort=FALSE)
  title(main = paste0(&quot;chr: &quot;, chr=peaks$chr[i],&quot;; pos: &quot;, peaks$pos[i], &quot;cM /&quot;,peaks_mba$pos[i],&quot;MB\n(&quot;,peaks$phenotype[i],&quot; )&quot;), line=0.2)
  ##dev.off()

  chr = peaks$chr[i]

# Plot 2
  pr_sub &lt;- pull_genoprobint(pr.qc, gm$gmap, chr, c(peaks$ci_lo[i], peaks$ci_hi[i]))
  #coeff &lt;- scan1coef(pr[,chr], cross$pheno[,peaks$lodcolumn[i]], addcovar = addcovar)
  #coeff &lt;- scan1coef(pr[,chr], cross$pheno[,peaks$lodcolumn[i]], Xcovar=Xcovar)
  #coeff &lt;- scan1coef(pr.qc[,chr], gm$covar[peaks$lodcolumn[i]], model=&quot;binary&quot;)
  #coeff_sub &lt;- scan1coef(pr_sub[,chr], gm$covar[peaks$lodcolumn[i]], model=&quot;binary&quot;)
  blup &lt;- scan1blup(pr.qc[,chr], covars[peaks$phenotype[i]])
  blup_sub &lt;- scan1blup(pr_sub[,chr], covars[peaks$phenotype[i]])

  write.csv(as.data.frame(blup_sub), paste0(&quot;data/ici-early.vs.pbs.ici-late_&quot;,peaks$phenotype[i],&quot;-no.covariates_blup_sub_chr-&quot;,chr,&quot;_peak.marker-&quot;,peaks$marker[i],&quot;_lod.drop-1.5_snpsqc_dis_no-x_updated_5.batches_mis.csv&quot;), quote=F)

  #plot_coef(coeff, 
  #     gm$gmap, columns=1:2,
  #     bgcolor=&quot;gray95&quot;, legend=&quot;bottomleft&quot;, 
  #     main = paste(&quot;chr&quot;, chr=peaks$chr[i],&quot;; pos: &quot;, peaks$pos[i], &quot;cM /&quot;,peaks_mba$pos[i],&quot;MB\n(&quot;,peaks$lodcolumn[i],&quot; [scan1coeff; positions in cM] )&quot;)
  #     )

  #plot_coef(coeff_sub, 
  #     gm$gmap, columns=1:2,
  #     bgcolor=&quot;gray95&quot;, legend=&quot;bottomleft&quot;, 
  #     main = paste(&quot;chr&quot;, chr=peaks$chr[i],&quot;; pos: &quot;, peaks$pos[i], &quot;cM /&quot;,peaks_mba$pos[i],&quot;MB\n(&quot;,peaks$lodcolumn[i],&quot;; 1.5 LOD drop interval [scan1coeff; positions in cM] ) &quot;)
  #     )


  plot_coef(blup, 
       gm$gmap, columns=1:2,
       bgcolor=&quot;gray95&quot;, legend=&quot;bottomleft&quot;, 
       main = paste0(&quot;chr: &quot;, chr=peaks$chr[i],&quot;; pos: &quot;, peaks$pos[i], &quot;cM /&quot;,peaks_mba$pos[i],&quot;MB\n(&quot;,peaks$phenotype[i],&quot; [scan1blup; positions in cM] )&quot;)
       )

  plot_coef(blup_sub, 
       gm$gmap, columns=1:2,
       bgcolor=&quot;gray95&quot;, legend=&quot;bottomleft&quot;, 
       main = paste0(&quot;chr: &quot;, chr=peaks$chr[i],&quot;; pos: &quot;, peaks$pos[i], &quot;cM /&quot;,peaks_mba$pos[i],&quot;MB\n(&quot;,peaks$phenotype[i],&quot;; 1.5 LOD drop interval [scan1blup; positions in cM] )&quot;)
       )


 # Plot 3
  #c2effB &lt;- scan1coef(pr.qc[,chr], gm$covar[peaks$lodcolumn[i]], model=&quot;binary&quot;, contrasts=cbind(a=c(-1, 0), d=c(0, -1)))
  #c2effBb &lt;- scan1blup(pr.qc[,chr], gm$covar[peaks$lodcolumn[i]], contrasts=cbind(a=c(-1, 0), d=c(0, -1)))
  ##c2effB &lt;- scan1coef(pr[,chr], cross$pheno[,peaks$lodcolumn[i]], addcovar = addcovar, contrasts=cbind(mu=c(1,1,1), a=c(-1, 0, 1), d=c(0, 1, 0)))
  ##c2effB &lt;- scan1coef(pr[,chr], cross$pheno[,peaks$lodcolumn[i]],Xcovar=Xcovar, contrasts=cbind(mu=c(1,1,1), a=c(-1, 0, 1), d=c(0, 1, 0)))
  #plot(c2effB, gm$gmap[chr], columns=1:2,
  #     bgcolor=&quot;gray95&quot;, legend=&quot;bottomleft&quot;, 
  #     main = paste(&quot;chr&quot;, chr=peaks$chr[i], &quot;pos&quot;, peaks$pos[i], &quot;(&quot;,peaks$lodcolumn[i],&quot;)&quot;)
  #     )
  #plot(c2effBb, gm$gmap[chr], columns=1:2,
  #     bgcolor=&quot;gray95&quot;, legend=&quot;bottomleft&quot;, 
  #     main = paste(&quot;chr&quot;, chr=peaks$chr[i], &quot;pos&quot;, peaks$pos[i], &quot;(&quot;,peaks$lodcolumn[i],&quot;)&quot;)
  #     )
  ##last_coef &lt;- unclass(c2effB)[nrow(c2effB),2:3] # last two coefficients
  ##for(t in seq(along=last_coef))
  ##  axis(side=4, at=last_coef[t], names(last_coef)[t], tick=FALSE)


  #Table 1
  chr = peaks_mba$chr[i]
  start=as.numeric(peaks_mba$ci_lo[i])
  end=as.numeric(peaks_mba$ci_hi[i])

  genesgss = query_genes(chr, start, end)

  write.csv(genesgss, file=paste0(&quot;data/ici-early.vs.pbs.ici-late_&quot;,peaks$phenotype[i],&quot;-no.covariates_genes_chr-&quot;,chr,&quot;_peak.marker-&quot;,peaks$marker[i],&quot;_lod.drop-1.5_snpsqc_dis_no-x_updated_5.batches_mis.csv&quot;), quote=F)

  rownames(genesgss) &lt;- NULL
  genesgss$strand_old = genesgss$strand
  genesgss$strand[genesgss$strand==&quot;+&quot;] &lt;- &quot;positive&quot;
  genesgss$strand[genesgss$strand==&quot;-&quot;] &lt;- &quot;negative&quot;

  #genesgss &lt;- 
  #table &lt;- 
  #genesgss[,c(&quot;chr&quot;,&quot;type&quot;,&quot;start&quot;,&quot;stop&quot;,&quot;strand&quot;,&quot;ID&quot;,&quot;Name&quot;,&quot;Dbxref&quot;,&quot;gene_id&quot;,&quot;mgi_type&quot;,&quot;description&quot;)] %&gt;%
  #kable(escape = F,align = c(&quot;ccccccccccc&quot;)) %&gt;%
  #kable_styling(&quot;striped&quot;, full_width = T) #%&gt;% 
  #cat #%&gt;%
  #column_spec(1, bold=TRUE)
#
  #print(kable(genesgss[,c(&quot;chr&quot;,&quot;type&quot;,&quot;start&quot;,&quot;stop&quot;,&quot;strand&quot;,&quot;ID&quot;,&quot;Name&quot;,&quot;Dbxref&quot;,&quot;gene_id&quot;,&quot;mgi_type&quot;,&quot;description&quot;)], escape = F,align = c(&quot;ccccccccccc&quot;)))

  print(kable(genesgss[,c(&quot;chr&quot;,&quot;type&quot;,&quot;start&quot;,&quot;stop&quot;,&quot;strand&quot;,&quot;ID&quot;,&quot;Name&quot;,&quot;Dbxref&quot;,&quot;gene_id&quot;,&quot;mgi_type&quot;,&quot;description&quot;)], &quot;html&quot;) %&gt;% kable_styling(&quot;striped&quot;, full_width = T))

  #table
  

}

} else {
  print(paste0(&quot;There are no peaks that have a LOD that reaches suggestive (p&lt;0.1) level of &quot;,summary(operm,alpha=0.1)$A, &quot; [autosomes]/&quot;,summary(operm,alpha=0.1)$X, &quot; [x-chromosome]&quot;))
}</code></pre>
<p>[1] “There are no peaks that have a LOD that reaches suggestive (p&lt;0.1) level of 2.56690086175578 [autosomes]/2.46831877337068 [x-chromosome]”</p>
</div>
</div>
<div id="additive-covariates" class="section level2">
<h2>Additive covariates</h2>
<div id="significance-thresholds-1" class="section level3">
<h3>Significance thresholds</h3>
<pre class="r"><code>operm &lt;- scan1perm(pr.qc, covars[&quot;age.of.onset&quot;], model=&quot;normal&quot;, addcovar=addcovar, n_perm=1000, perm_Xsp=TRUE, chr_lengths=chr_lengths(gm$gmap))

summary_table&lt;-data.frame(unclass(summary(operm, alpha=c(0.01,  0.05, 0.1))))
names(summary_table) &lt;- c(&quot;autosomes&quot;,&quot;X&quot;)
summary_table$significance.level &lt;- rownames(summary_table)

rownames(summary_table) &lt;- NULL

summary_table[c(3,1:2)] %&gt;%
  kable(escape = F,align = c(&quot;ccc&quot;)) %&gt;%
  kable_styling(&quot;striped&quot;, full_width = T) %&gt;%
  column_spec(1, bold=TRUE)</code></pre>
<table class="table table-striped" style="margin-left: auto; margin-right: auto;">
<thead>
<tr>
<th style="text-align:center;">
significance.level
</th>
<th style="text-align:center;">
autosomes
</th>
<th style="text-align:center;">
X
</th>
</tr>
</thead>
<tbody>
<tr>
<td style="text-align:center;font-weight: bold;">
0.01
</td>
<td style="text-align:center;">
3.422057
</td>
<td style="text-align:center;">
3.264682
</td>
</tr>
<tr>
<td style="text-align:center;font-weight: bold;">
0.05
</td>
<td style="text-align:center;">
2.811477
</td>
<td style="text-align:center;">
2.745594
</td>
</tr>
<tr>
<td style="text-align:center;font-weight: bold;">
0.1
</td>
<td style="text-align:center;">
2.466958
</td>
<td style="text-align:center;">
2.454072
</td>
</tr>
</tbody>
</table>
</div>
<div id="genome-wide-scan-1" class="section level3">
<h3>Genome-wide scan</h3>
<pre class="r"><code>plot_lod&lt;-function(out,map){
  for (i in 1:dim(out)[2]){
#par(mar=c(5.1, 6.1, 1.1, 1.1))
    ymx &lt;- maxlod(out) # overall maximum LOD score
    plot(out, map, lodcolumn=i, col=&quot;slateblue&quot;, ylim=c(0, ymx+0.5))
    #legend(&quot;topright&quot;, lwd=2, colnames(out)[i], bg=&quot;gray90&quot;)
    title(main = paste0(colnames(out)[i], &quot; - ICI-Early vs PBS &amp; ICI-Late (with additive covariates) [positions in cM]&quot;))
    add_threshold(map,  summary(operm,alpha=0.1), col = &#39;purple&#39;)
    add_threshold(map,  summary(operm, alpha=0.05), col = &#39;red&#39;)
    add_threshold(map,  summary(operm, alpha=0.01), col = &#39;blue&#39;)

  }
}


#print(&quot;with no kinship&quot;)
#outfa &lt;- scan1(pr.qc, covars[&quot;age.of.onset&quot;], Xcovar=Xcovar, model=&quot;normal&quot;, addcovar=addcovar)

#plot_lod(outfa,gm$gmap)

#print(&quot;with normal kinship&quot;)
#outka &lt;- scan1(pr.qc, covars[&quot;age.of.onset&quot;], Xcovar=Xcovar, model=&quot;normal&quot;, addcovar=addcovar, kinship=kinship)

#plot_lod(outka,gm$gmap)

#print(&quot;with loco kinship&quot;)
#outlka &lt;- scan1(pr.qc, covars[&quot;age.of.onset&quot;], Xcovar=Xcovar, model=&quot;normal&quot;, addcovar=addcovar, kinship=K)

#plot_lod(outlka,gm$gmap)

print(&quot;no kinship and no xcovar&quot;)</code></pre>
<p>[1] “no kinship and no xcovar”</p>
<pre class="r"><code>outa &lt;- scan1(pr.qc, covars[&quot;age.of.onset&quot;], model=&quot;normal&quot;,addcovar=addcovar)

plot_lod(outa,gm$gmap)</code></pre>
<p><img src="figure/4.1.2_qtl.analysis_cont_age_ici-early.vs.pbs.ici-late_snpsqc_pheno.corrected.cleaned_dis_no-xk_5.batches_mis.Rmd/scanonea-1.png" width="912" style="display: block; margin: auto;" /></p>
<pre class="r"><code>out = outa

print(paste0(&quot;number of samples in analysis = &quot;, as.data.frame(attributes(out)$sample_size)[1,]))</code></pre>
<p>[1] “number of samples in analysis = 138”</p>
</div>
<div id="lod-peaks-1" class="section level3">
<h3>LOD peaks</h3>
<div id="centimorgan-cm-1" class="section level4">
<h4>Centimorgan (cM)</h4>
<pre class="r"><code>peaks&lt;-find_peaks(out, gm$gmap, threshold=summary(operm,alpha=0.1)$A, thresholdX = summary(operm,alpha=0.1)$X, peakdrop=3, drop=1.5)

if(nrow(peaks) &gt;0){
peaks$marker &lt;- find_marker(gm$gmap, chr=peaks$chr,pos=peaks$pos)
names(peaks)[2] &lt;- c(&quot;phenotype&quot;)
peaks &lt;- peaks[-1]

rownames(peaks) &lt;- NULL
print(kable(peaks, escape = F, align = c(&quot;cccccccc&quot;), &quot;html&quot;) 
  %&gt;% kable_styling(&quot;striped&quot;, full_width = T)%&gt;%
  column_spec(1, bold=TRUE)
  )

#plot only peak chromosomes

plot_lod_chr&lt;-function(out,map,chrom){
  for (i in 1:dim(out)[2]){
    #png(filename=paste0(&quot;/Users/chenm/Documents/qtl/Jai/&quot;,colnames(out)[i],  &quot;_lod.png&quot;))
    
    #par(mar=c(5.1, 6.1, 1.1, 1.1))
    ymx &lt;- maxlod(out) # overall maximum LOD score
    plot(out, map, chr = chrom, lodcolumn=i, col=&quot;slateblue&quot;, ylim=c(0, ymx+0.5))
    #legend(&quot;topright&quot;, lwd=2, colnames(out)[i], bg=&quot;gray90&quot;)
    title(main = paste0(colnames(out)[i], &quot; - chr&quot;, chrom, &quot; - ICI-Early vs PBS &amp; ICI-Late (with additive covariates) [positions in cM]&quot;))
    add_threshold(map,  summary(operm,alpha=0.1), col = &#39;purple&#39;)
    add_threshold(map,  summary(operm, alpha=0.05), col = &#39;red&#39;)
    add_threshold(map,  summary(operm, alpha=0.01), col = &#39;blue&#39;)
    #for (j in 1: dim(summary_table)[1]){
    #  abline(h=summary_table[j, i],col=&quot;red&quot;)
    #  text(x=400, y =summary_table[j, i]+0.12, labels = paste(&quot;p=&quot;, row.names(summary_table)[j]))
    #}
    #dev.off()

  }
}


for(i in unique(peaks$chr)){
#for (i in 1:nrow(peaks)){
  #plot_lod_chr(out,gm$gmap, peaks$chr[i])
  plot_lod_chr(out,gm$gmap, i)
}

} else {
  print(paste0(&quot;There are no peaks that have a LOD that reaches suggestive (p&lt;0.1) level of &quot;,summary(operm,alpha=0.1)$A, &quot; [autosomes]/&quot;,summary(operm,alpha=0.1)$X, &quot; [x-chromosome]&quot;))
}</code></pre>
<table class="table table-striped" style="margin-left: auto; margin-right: auto;">
<thead>
<tr>
<th style="text-align:center;">
phenotype
</th>
<th style="text-align:center;">
chr
</th>
<th style="text-align:center;">
pos
</th>
<th style="text-align:center;">
lod
</th>
<th style="text-align:center;">
ci_lo
</th>
<th style="text-align:center;">
ci_hi
</th>
<th style="text-align:center;">
marker
</th>
</tr>
</thead>
<tbody>
<tr>
<td style="text-align:center;font-weight: bold;">
age.of.onset
</td>
<td style="text-align:center;">
6
</td>
<td style="text-align:center;">
22.044
</td>
<td style="text-align:center;">
2.479518
</td>
<td style="text-align:center;">
13.995
</td>
<td style="text-align:center;">
29.389
</td>
<td style="text-align:center;">
UNC11006703
</td>
</tr>
</tbody>
</table>
<p><img src="figure/4.1.2_qtl.analysis_cont_age_ici-early.vs.pbs.ici-late_snpsqc_pheno.corrected.cleaned_dis_no-xk_5.batches_mis.Rmd/LOD%20peaksa-1.png" width="912" style="display: block; margin: auto;" /></p>
</div>
<div id="megabase-mb-1" class="section level4">
<h4>Megabase (MB)</h4>
<pre class="r"><code>peaks_mba&lt;-find_peaks(out, gm$pmap, threshold=summary(operm,alpha=0.1)$A, thresholdX = summary(operm,alpha=0.1)$X, peakdrop=3, drop=1.5)

if(nrow(peaks) &gt;0){
peaks_mba$marker &lt;- find_marker(gm$pmap, chr=peaks_mba$chr,pos=peaks_mba$pos)
names(peaks_mba)[2] &lt;- c(&quot;phenotype&quot;)
peaks_mba &lt;- peaks_mba[-1]

rownames(peaks_mba) &lt;- NULL
print(kable(peaks_mba, escape = F, align = c(&quot;cccccccc&quot;), &quot;html&quot;) 
  %&gt;% kable_styling(&quot;striped&quot;, full_width = T)%&gt;%
  column_spec(1, bold=TRUE)
  )

plot_lod_chr_mb&lt;-function(out,map,chrom){
  for (i in 1:dim(out)[2]){
    #png(filename=paste0(&quot;/Users/chenm/Documents/qtl/Jai/&quot;,colnames(out)[i],  &quot;_lod.png&quot;))
    
    #par(mar=c(5.1, 6.1, 1.1, 1.1))
    ymx &lt;- maxlod(out) # overall maximum LOD score
    plot(out, map, chr = chrom, lodcolumn=i, col=&quot;slateblue&quot;, ylim=c(0, ymx+0.5))
    #legend(&quot;topright&quot;, lwd=2, colnames(out)[i], bg=&quot;gray90&quot;)
    title(main = paste0(colnames(out)[i], &quot; - chr&quot;, chrom, &quot; - ICI-Early vs PBS &amp; ICI-Late (with additive covariates) [positions in cM]&quot;))
    add_threshold(map,  summary(operm,alpha=0.1), col = &#39;purple&#39;)
    add_threshold(map,  summary(operm, alpha=0.05), col = &#39;red&#39;)
    add_threshold(map,  summary(operm, alpha=0.01), col = &#39;blue&#39;)
    #for (j in 1: dim(summary_table)[1]){
    #  abline(h=summary_table[j, i],col=&quot;red&quot;)
    #  text(x=400, y =summary_table[j, i]+0.12, labels = paste(&quot;p=&quot;, row.names(summary_table)[j]))
    #}
    #dev.off()
  }
}

for(i in unique(peaks_mba$chr)){
#for (i in 1:nrow(peaks_mba)){
  #plot_lod_chr_mb(out,gm$pmap, peaks_mba$chr[i])
  plot_lod_chr_mb(out,gm$pmap,i)
}

} else {
  print(paste0(&quot;There are no peaks that have a LOD that reaches suggestive (p&lt;0.1) level of &quot;,summary(operm,alpha=0.1)$A, &quot; [autosomes]/&quot;,summary(operm,alpha=0.1)$X, &quot; [x-chromosome]&quot;))
}</code></pre>
<table class="table table-striped" style="margin-left: auto; margin-right: auto;">
<thead>
<tr>
<th style="text-align:center;">
phenotype
</th>
<th style="text-align:center;">
chr
</th>
<th style="text-align:center;">
pos
</th>
<th style="text-align:center;">
lod
</th>
<th style="text-align:center;">
ci_lo
</th>
<th style="text-align:center;">
ci_hi
</th>
<th style="text-align:center;">
marker
</th>
</tr>
</thead>
<tbody>
<tr>
<td style="text-align:center;font-weight: bold;">
age.of.onset
</td>
<td style="text-align:center;">
6
</td>
<td style="text-align:center;">
46.11426
</td>
<td style="text-align:center;">
2.479518
</td>
<td style="text-align:center;">
32.95827
</td>
<td style="text-align:center;">
62.29163
</td>
<td style="text-align:center;">
UNC11006703
</td>
</tr>
</tbody>
</table>
<p><img src="figure/4.1.2_qtl.analysis_cont_age_ici-early.vs.pbs.ici-late_snpsqc_pheno.corrected.cleaned_dis_no-xk_5.batches_mis.Rmd/LOD%20peaks_mbaa-1.png" width="912" style="display: block; margin: auto;" /></p>
</div>
</div>
<div id="qtl-effects-1" class="section level3">
<h3>QTL effects</h3>
<p>For each peak LOD location we give a list of gene</p>
<pre class="r"><code>if(nrow(peaks) &gt;0){
for (i in 1:nrow(peaks)){
#for (i in 1:1){
  #Plot 1
  g &lt;- maxmarg(pr.qc, gm$gmap, chr=peaks$chr[i], pos=peaks$pos[i], return_char=TRUE)
  #png(filename=paste0(&quot;/Users/chenm/Documents/qtl/Jai/&quot;,&quot;qtl_effect_&quot;, i, &quot;.png&quot;))
  #par(mar=c(4.1, 4.1, 1.5, 0.6))
  plot_pxg(g, covars[,peaks$phenotype[i]], ylab=peaks$phenotype[i], sort=FALSE)
  title(main = paste0(&quot;chr: &quot;, chr=peaks$chr[i],&quot;; pos: &quot;, peaks$pos[i], &quot;cM /&quot;,peaks_mba$pos[i],&quot;MB\n(&quot;,peaks$phenotype[i],&quot; )&quot;), line=0.2)
  ##dev.off()

  chr = peaks$chr[i]

# Plot 2
  pr_sub &lt;- pull_genoprobint(pr.qc, gm$gmap, chr, c(peaks$ci_lo[i], peaks$ci_hi[i]))
  #coeff &lt;- scan1coef(pr[,chr], cross$pheno[,peaks$lodcolumn[i]], addcovar = addcovar)
  #coeff &lt;- scan1coef(pr[,chr], cross$pheno[,peaks$lodcolumn[i]], Xcovar=Xcovar)
  #coeff &lt;- scan1coef(pr.qc[,chr], gm$covar[peaks$lodcolumn[i]], model=&quot;binary&quot;)
  #coeff_sub &lt;- scan1coef(pr_sub[,chr], gm$covar[peaks$lodcolumn[i]], model=&quot;binary&quot;)
  blup &lt;- scan1blup(pr.qc[,chr], covars[peaks$phenotype[i]])
  blup_sub &lt;- scan1blup(pr_sub[,chr], covars[peaks$phenotype[i]])

  write.csv(as.data.frame(blup_sub), paste0(&quot;data/ici-early.vs.pbs.ici-late_&quot;,peaks$phenotype[i],&quot;-additive.covariates_blup_sub_chr-&quot;,chr,&quot;_peak.marker-&quot;,peaks$marker[i],&quot;_lod.drop-1.5_snpsqc_dis_no-x_updated_5.batches_mis.csv&quot;), quote=F)

  #plot_coef(coeff, 
  #     gm$gmap, columns=1:2,
  #     bgcolor=&quot;gray95&quot;, legend=&quot;bottomleft&quot;, 
  #     main = paste(&quot;chr&quot;, chr=peaks$chr[i],&quot;; pos: &quot;, peaks$pos[i], &quot;cM /&quot;,peaks_mba$pos[i],&quot;MB\n(&quot;,peaks$lodcolumn[i],&quot; [scan1coeff; positions in cM] )&quot;)
  #     )

  #plot_coef(coeff_sub, 
  #     gm$gmap, columns=1:2,
  #     bgcolor=&quot;gray95&quot;, legend=&quot;bottomleft&quot;, 
  #     main = paste(&quot;chr&quot;, chr=peaks$chr[i],&quot;; pos: &quot;, peaks$pos[i], &quot;cM /&quot;,peaks_mba$pos[i],&quot;MB\n(&quot;,peaks$lodcolumn[i],&quot;; 1.5 LOD drop interval [scan1coeff; positions in cM] ) &quot;)
  #     )


  plot_coef(blup, 
       gm$gmap, columns=1:2,
       bgcolor=&quot;gray95&quot;, legend=&quot;bottomleft&quot;, 
       main = paste0(&quot;chr: &quot;, chr=peaks$chr[i],&quot;; pos: &quot;, peaks$pos[i], &quot;cM /&quot;,peaks_mba$pos[i],&quot;MB\n(&quot;,peaks$phenotype[i],&quot; [scan1blup; positions in cM] )&quot;)
       )

  plot_coef(blup_sub, 
       gm$gmap, columns=1:2,
       bgcolor=&quot;gray95&quot;, legend=&quot;bottomleft&quot;, 
       main = paste0(&quot;chr: &quot;, chr=peaks$chr[i],&quot;; pos: &quot;, peaks$pos[i], &quot;cM /&quot;,peaks_mba$pos[i],&quot;MB\n(&quot;,peaks$phenotype[i],&quot;; 1.5 LOD drop interval [scan1blup; positions in cM] )&quot;)
       )


 # Plot 3
  #c2effB &lt;- scan1coef(pr.qc[,chr], gm$covar[peaks$lodcolumn[i]], model=&quot;binary&quot;, contrasts=cbind(a=c(-1, 0), d=c(0, -1)))
  #c2effBb &lt;- scan1blup(pr.qc[,chr], gm$covar[peaks$lodcolumn[i]], contrasts=cbind(a=c(-1, 0), d=c(0, -1)))
  ##c2effB &lt;- scan1coef(pr[,chr], cross$pheno[,peaks$lodcolumn[i]], addcovar = addcovar, contrasts=cbind(mu=c(1,1,1), a=c(-1, 0, 1), d=c(0, 1, 0)))
  ##c2effB &lt;- scan1coef(pr[,chr], cross$pheno[,peaks$lodcolumn[i]],Xcovar=Xcovar, contrasts=cbind(mu=c(1,1,1), a=c(-1, 0, 1), d=c(0, 1, 0)))
  #plot(c2effB, gm$gmap[chr], columns=1:2,
  #     bgcolor=&quot;gray95&quot;, legend=&quot;bottomleft&quot;, 
  #     main = paste(&quot;chr&quot;, chr=peaks$chr[i], &quot;pos&quot;, peaks$pos[i], &quot;(&quot;,peaks$lodcolumn[i],&quot;)&quot;)
  #     )
  #plot(c2effBb, gm$gmap[chr], columns=1:2,
  #     bgcolor=&quot;gray95&quot;, legend=&quot;bottomleft&quot;, 
  #     main = paste(&quot;chr&quot;, chr=peaks$chr[i], &quot;pos&quot;, peaks$pos[i], &quot;(&quot;,peaks$lodcolumn[i],&quot;)&quot;)
  #     )
  ##last_coef &lt;- unclass(c2effB)[nrow(c2effB),2:3] # last two coefficients
  ##for(t in seq(along=last_coef))
  ##  axis(side=4, at=last_coef[t], names(last_coef)[t], tick=FALSE)


  #Table 1
  chr = peaks_mba$chr[i]
  start=as.numeric(peaks_mba$ci_lo[i])
  end=as.numeric(peaks_mba$ci_hi[i])

  genesgss = query_genes(chr, start, end)

  write.csv(genesgss, file=paste0(&quot;data/ici-early.vs.pbs.ici-late_&quot;,peaks$phenotype[i],&quot;-additive.covariates_genes_chr-&quot;,chr,&quot;_peak.marker-&quot;,peaks$marker[i],&quot;_lod.drop-1.5_snpsqc_dis_no-x_updated_5.batches_mis.csv&quot;), quote=F)

  rownames(genesgss) &lt;- NULL
  genesgss$strand_old = genesgss$strand
  genesgss$strand[genesgss$strand==&quot;+&quot;] &lt;- &quot;positive&quot;
  genesgss$strand[genesgss$strand==&quot;-&quot;] &lt;- &quot;negative&quot;

  #genesgss &lt;- 
  #table &lt;- 
  #genesgss[,c(&quot;chr&quot;,&quot;type&quot;,&quot;start&quot;,&quot;stop&quot;,&quot;strand&quot;,&quot;ID&quot;,&quot;Name&quot;,&quot;Dbxref&quot;,&quot;gene_id&quot;,&quot;mgi_type&quot;,&quot;description&quot;)] %&gt;%
  #kable(escape = F,align = c(&quot;ccccccccccc&quot;)) %&gt;%
  #kable_styling(&quot;striped&quot;, full_width = T) #%&gt;% 
  #cat #%&gt;%
  #column_spec(1, bold=TRUE)
#
  #print(kable(genesgss[,c(&quot;chr&quot;,&quot;type&quot;,&quot;start&quot;,&quot;stop&quot;,&quot;strand&quot;,&quot;ID&quot;,&quot;Name&quot;,&quot;Dbxref&quot;,&quot;gene_id&quot;,&quot;mgi_type&quot;,&quot;description&quot;)], escape = F,align = c(&quot;ccccccccccc&quot;)))

  print(kable(genesgss[,c(&quot;chr&quot;,&quot;type&quot;,&quot;start&quot;,&quot;stop&quot;,&quot;strand&quot;,&quot;ID&quot;,&quot;Name&quot;,&quot;Dbxref&quot;,&quot;gene_id&quot;,&quot;mgi_type&quot;,&quot;description&quot;)], &quot;html&quot;) %&gt;% kable_styling(&quot;striped&quot;, full_width = T))

  #table
  

}
} else {
  print(paste0(&quot;There are no peaks that have a LOD that reaches suggestive (p&lt;0.1) level of &quot;,summary(operm,alpha=0.1)$A, &quot; [autosomes]/&quot;,summary(operm,alpha=0.1)$X, &quot; [x-chromosome]&quot;))
}</code></pre>
<img src="figure/4.1.2_qtl.analysis_cont_age_ici-early.vs.pbs.ici-late_snpsqc_pheno.corrected.cleaned_dis_no-xk_5.batches_mis.Rmd/QTL%20effectsa-1.png" width="912" style="display: block; margin: auto;" /><img src="figure/4.1.2_qtl.analysis_cont_age_ici-early.vs.pbs.ici-late_snpsqc_pheno.corrected.cleaned_dis_no-xk_5.batches_mis.Rmd/QTL%20effectsa-2.png" width="912" style="display: block; margin: auto;" /><img src="figure/4.1.2_qtl.analysis_cont_age_ici-early.vs.pbs.ici-late_snpsqc_pheno.corrected.cleaned_dis_no-xk_5.batches_mis.Rmd/QTL%20effectsa-3.png" width="912" style="display: block; margin: auto;" />
<table class="table table-striped" style="margin-left: auto; margin-right: auto;">
<thead>
<tr>
<th style="text-align:left;">
chr
</th>
<th style="text-align:left;">
type
</th>
<th style="text-align:right;">
start
</th>
<th style="text-align:right;">
stop
</th>
<th style="text-align:left;">
strand
</th>
<th style="text-align:left;">
ID
</th>
<th style="text-align:left;">
Name
</th>
<th style="text-align:left;">
Dbxref
</th>
<th style="text-align:left;">
gene_id
</th>
<th style="text-align:left;">
mgi_type
</th>
<th style="text-align:left;">
description
</th>
</tr>
</thead>
<tbody>
<tr>
<td style="text-align:left;">
6
</td>
<td style="text-align:left;">
gene
</td>
<td style="text-align:right;">
32.79098
</td>
<td style="text-align:right;">
33.06028
</td>
<td style="text-align:left;">
negative
</td>
<td style="text-align:left;">
MGI_C57BL6J_1913325
</td>
<td style="text-align:left;">
Chchd3
</td>
<td style="text-align:left;">
NCBI_Gene:66075,ENSEMBL:ENSMUSG00000053768
</td>
<td style="text-align:left;">
MGI:1913325
</td>
<td style="text-align:left;">
protein coding gene
</td>
<td style="text-align:left;">
coiled-coil-helix-coiled-coil-helix domain containing 3
</td>
</tr>
<tr>
<td style="text-align:left;">
6
</td>
<td style="text-align:left;">
gene
</td>
<td style="text-align:right;">
33.03847
</td>
<td style="text-align:right;">
33.04322
</td>
<td style="text-align:left;">
positive
</td>
<td style="text-align:left;">
MGI_C57BL6J_3705140
</td>
<td style="text-align:left;">
Gm14540
</td>
<td style="text-align:left;">
ENSEMBL:ENSMUSG00000085847
</td>
<td style="text-align:left;">
MGI:3705140
</td>
<td style="text-align:left;">
lncRNA gene
</td>
<td style="text-align:left;">
predicted gene 14540
</td>
</tr>
<tr>
<td style="text-align:left;">
6
</td>
<td style="text-align:left;">
gene
</td>
<td style="text-align:right;">
33.03912
</td>
<td style="text-align:right;">
33.04212
</td>
<td style="text-align:left;">
negative
</td>
<td style="text-align:left;">
MGI_C57BL6J_5663032
</td>
<td style="text-align:left;">
Gm42895
</td>
<td style="text-align:left;">
ENSEMBL:ENSMUSG00000106676
</td>
<td style="text-align:left;">
MGI:5663032
</td>
<td style="text-align:left;">
unclassified gene
</td>
<td style="text-align:left;">
predicted gene 42895
</td>
</tr>
<tr>
<td style="text-align:left;">
6
</td>
<td style="text-align:left;">
gene
</td>
<td style="text-align:right;">
33.08865
</td>
<td style="text-align:right;">
33.08875
</td>
<td style="text-align:left;">
negative
</td>
<td style="text-align:left;">
MGI_C57BL6J_5452098
</td>
<td style="text-align:left;">
Gm22321
</td>
<td style="text-align:left;">
ENSEMBL:ENSMUSG00000087777
</td>
<td style="text-align:left;">
MGI:5452098
</td>
<td style="text-align:left;">
snRNA gene
</td>
<td style="text-align:left;">
predicted gene, 22321
</td>
</tr>
<tr>
<td style="text-align:left;">
6
</td>
<td style="text-align:left;">
pseudogene
</td>
<td style="text-align:right;">
33.24727
</td>
<td style="text-align:right;">
33.24811
</td>
<td style="text-align:left;">
negative
</td>
<td style="text-align:left;">
MGI_C57BL6J_2686247
</td>
<td style="text-align:left;">
Gm1401
</td>
<td style="text-align:left;">
NCBI_Gene:102634733,ENSEMBL:ENSMUSG00000082217
</td>
<td style="text-align:left;">
MGI:2686247
</td>
<td style="text-align:left;">
pseudogene
</td>
<td style="text-align:left;">
predicted gene 1401
</td>
</tr>
<tr>
<td style="text-align:left;">
6
</td>
<td style="text-align:left;">
gene
</td>
<td style="text-align:right;">
33.24909
</td>
<td style="text-align:right;">
33.97398
</td>
<td style="text-align:left;">
positive
</td>
<td style="text-align:left;">
MGI_C57BL6J_1096376
</td>
<td style="text-align:left;">
Exoc4
</td>
<td style="text-align:left;">
NCBI_Gene:20336,ENSEMBL:ENSMUSG00000029763
</td>
<td style="text-align:left;">
MGI:1096376
</td>
<td style="text-align:left;">
protein coding gene
</td>
<td style="text-align:left;">
exocyst complex component 4
</td>
</tr>
<tr>
<td style="text-align:left;">
6
</td>
<td style="text-align:left;">
pseudogene
</td>
<td style="text-align:right;">
33.37511
</td>
<td style="text-align:right;">
33.37745
</td>
<td style="text-align:left;">
negative
</td>
<td style="text-align:left;">
MGI_C57BL6J_3651023
</td>
<td style="text-align:left;">
Gm13854
</td>
<td style="text-align:left;">
NCBI_Gene:100416262,ENSEMBL:ENSMUSG00000083728
</td>
<td style="text-align:left;">
MGI:3651023
</td>
<td style="text-align:left;">
pseudogene
</td>
<td style="text-align:left;">
predicted gene 13854
</td>
</tr>
<tr>
<td style="text-align:left;">
6
</td>
<td style="text-align:left;">
gene
</td>
<td style="text-align:right;">
33.43301
</td>
<td style="text-align:right;">
33.43462
</td>
<td style="text-align:left;">
positive
</td>
<td style="text-align:left;">
MGI_C57BL6J_5663305
</td>
<td style="text-align:left;">
Gm43168
</td>
<td style="text-align:left;">
ENSEMBL:ENSMUSG00000107358
</td>
<td style="text-align:left;">
MGI:5663305
</td>
<td style="text-align:left;">
unclassified gene
</td>
<td style="text-align:left;">
predicted gene 43168
</td>
</tr>
<tr>
<td style="text-align:left;">
6
</td>
<td style="text-align:left;">
gene
</td>
<td style="text-align:right;">
33.46412
</td>
<td style="text-align:right;">
33.47484
</td>
<td style="text-align:left;">
positive
</td>
<td style="text-align:left;">
MGI_C57BL6J_5621670
</td>
<td style="text-align:left;">
Gm38785
</td>
<td style="text-align:left;">
NCBI_Gene:105242624
</td>
<td style="text-align:left;">
MGI:5621670
</td>
<td style="text-align:left;">
lncRNA gene
</td>
<td style="text-align:left;">
predicted gene, 38785
</td>
</tr>
<tr>
<td style="text-align:left;">
6
</td>
<td style="text-align:left;">
gene
</td>
<td style="text-align:right;">
33.69003
</td>
<td style="text-align:right;">
33.69215
</td>
<td style="text-align:left;">
negative
</td>
<td style="text-align:left;">
MGI_C57BL6J_5663301
</td>
<td style="text-align:left;">
Gm43164
</td>
<td style="text-align:left;">
ENSEMBL:ENSMUSG00000107103
</td>
<td style="text-align:left;">
MGI:5663301
</td>
<td style="text-align:left;">
unclassified gene
</td>
<td style="text-align:left;">
predicted gene 43164
</td>
</tr>
<tr>
<td style="text-align:left;">
6
</td>
<td style="text-align:left;">
gene
</td>
<td style="text-align:right;">
33.84311
</td>
<td style="text-align:right;">
33.84495
</td>
<td style="text-align:left;">
positive
</td>
<td style="text-align:left;">
MGI_C57BL6J_1921750
</td>
<td style="text-align:left;">
5430439C14Rik
</td>
<td style="text-align:left;">
NA
</td>
<td style="text-align:left;">
NA
</td>
<td style="text-align:left;">
unclassified gene
</td>
<td style="text-align:left;">
RIKEN cDNA 5430439C14 gene
</td>
</tr>
<tr>
<td style="text-align:left;">
6
</td>
<td style="text-align:left;">
gene
</td>
<td style="text-align:right;">
33.98181
</td>
<td style="text-align:right;">
33.99248
</td>
<td style="text-align:left;">
negative
</td>
<td style="text-align:left;">
MGI_C57BL6J_3649279
</td>
<td style="text-align:left;">
Gm13853
</td>
<td style="text-align:left;">
NCBI_Gene:102634837,ENSEMBL:ENSMUSG00000087441
</td>
<td style="text-align:left;">
MGI:3649279
</td>
<td style="text-align:left;">
lncRNA gene
</td>
<td style="text-align:left;">
predicted gene 13853
</td>
</tr>
<tr>
<td style="text-align:left;">
6
</td>
<td style="text-align:left;">
gene
</td>
<td style="text-align:right;">
34.02241
</td>
<td style="text-align:right;">
34.02972
</td>
<td style="text-align:left;">
negative
</td>
<td style="text-align:left;">
MGI_C57BL6J_5621671
</td>
<td style="text-align:left;">
Gm38786
</td>
<td style="text-align:left;">
NCBI_Gene:105242625
</td>
<td style="text-align:left;">
MGI:5621671
</td>
<td style="text-align:left;">
lncRNA gene
</td>
<td style="text-align:left;">
predicted gene, 38786
</td>
</tr>
<tr>
<td style="text-align:left;">
6
</td>
<td style="text-align:left;">
gene
</td>
<td style="text-align:right;">
34.02899
</td>
<td style="text-align:right;">
34.13403
</td>
<td style="text-align:left;">
positive
</td>
<td style="text-align:left;">
MGI_C57BL6J_1921604
</td>
<td style="text-align:left;">
Lrguk
</td>
<td style="text-align:left;">
NCBI_Gene:74354,ENSEMBL:ENSMUSG00000056215
</td>
<td style="text-align:left;">
MGI:1921604
</td>
<td style="text-align:left;">
protein coding gene
</td>
<td style="text-align:left;">
leucine-rich repeats and guanylate kinase domain containing
</td>
</tr>
<tr>
<td style="text-align:left;">
6
</td>
<td style="text-align:left;">
gene
</td>
<td style="text-align:right;">
34.15193
</td>
<td style="text-align:right;">
34.17424
</td>
<td style="text-align:left;">
positive
</td>
<td style="text-align:left;">
MGI_C57BL6J_3652199
</td>
<td style="text-align:left;">
Gm13856
</td>
<td style="text-align:left;">
ENSEMBL:ENSMUSG00000087356
</td>
<td style="text-align:left;">
MGI:3652199
</td>
<td style="text-align:left;">
lncRNA gene
</td>
<td style="text-align:left;">
predicted gene 13856
</td>
</tr>
<tr>
<td style="text-align:left;">
6
</td>
<td style="text-align:left;">
gene
</td>
<td style="text-align:right;">
34.15338
</td>
<td style="text-align:right;">
34.17711
</td>
<td style="text-align:left;">
negative
</td>
<td style="text-align:left;">
MGI_C57BL6J_1931249
</td>
<td style="text-align:left;">
Slc35b4
</td>
<td style="text-align:left;">
NCBI_Gene:58246,ENSEMBL:ENSMUSG00000018999
</td>
<td style="text-align:left;">
MGI:1931249
</td>
<td style="text-align:left;">
protein coding gene
</td>
<td style="text-align:left;">
solute carrier family 35, member B4
</td>
</tr>
<tr>
<td style="text-align:left;">
6
</td>
<td style="text-align:left;">
gene
</td>
<td style="text-align:right;">
34.17717
</td>
<td style="text-align:right;">
34.18639
</td>
<td style="text-align:left;">
positive
</td>
<td style="text-align:left;">
MGI_C57BL6J_3652198
</td>
<td style="text-align:left;">
Gm13855
</td>
<td style="text-align:left;">
NCBI_Gene:105242626,ENSEMBL:ENSMUSG00000087613
</td>
<td style="text-align:left;">
MGI:3652198
</td>
<td style="text-align:left;">
lncRNA gene
</td>
<td style="text-align:left;">
predicted gene 13855
</td>
</tr>
<tr>
<td style="text-align:left;">
6
</td>
<td style="text-align:left;">
gene
</td>
<td style="text-align:right;">
34.19717
</td>
<td style="text-align:right;">
34.22708
</td>
<td style="text-align:left;">
positive
</td>
<td style="text-align:left;">
MGI_C57BL6J_5591692
</td>
<td style="text-align:left;">
Gm32533
</td>
<td style="text-align:left;">
NCBI_Gene:102635113
</td>
<td style="text-align:left;">
MGI:5591692
</td>
<td style="text-align:left;">
lncRNA gene
</td>
<td style="text-align:left;">
predicted gene, 32533
</td>
</tr>
<tr>
<td style="text-align:left;">
6
</td>
<td style="text-align:left;">
gene
</td>
<td style="text-align:right;">
34.29026
</td>
<td style="text-align:right;">
34.32425
</td>
<td style="text-align:left;">
positive
</td>
<td style="text-align:left;">
MGI_C57BL6J_3705210
</td>
<td style="text-align:left;">
Gm14546
</td>
<td style="text-align:left;">
ENSEMBL:ENSMUSG00000086762
</td>
<td style="text-align:left;">
MGI:3705210
</td>
<td style="text-align:left;">
lncRNA gene
</td>
<td style="text-align:left;">
predicted gene 14546
</td>
</tr>
<tr>
<td style="text-align:left;">
6
</td>
<td style="text-align:left;">
gene
</td>
<td style="text-align:right;">
34.30243
</td>
<td style="text-align:right;">
34.31749
</td>
<td style="text-align:left;">
negative
</td>
<td style="text-align:left;">
MGI_C57BL6J_1353494
</td>
<td style="text-align:left;">
Akr1b3
</td>
<td style="text-align:left;">
NCBI_Gene:11677,ENSEMBL:ENSMUSG00000001642
</td>
<td style="text-align:left;">
MGI:1353494
</td>
<td style="text-align:left;">
protein coding gene
</td>
<td style="text-align:left;">
aldo-keto reductase family 1, member B3 (aldose reductase)
</td>
</tr>
<tr>
<td style="text-align:left;">
6
</td>
<td style="text-align:left;">
gene
</td>
<td style="text-align:right;">
34.35412
</td>
<td style="text-align:right;">
34.36846
</td>
<td style="text-align:left;">
positive
</td>
<td style="text-align:left;">
MGI_C57BL6J_107673
</td>
<td style="text-align:left;">
Akr1b8
</td>
<td style="text-align:left;">
NCBI_Gene:14187,ENSEMBL:ENSMUSG00000029762
</td>
<td style="text-align:left;">
MGI:107673
</td>
<td style="text-align:left;">
protein coding gene
</td>
<td style="text-align:left;">
aldo-keto reductase family 1, member B8
</td>
</tr>
<tr>
<td style="text-align:left;">
6
</td>
<td style="text-align:left;">
pseudogene
</td>
<td style="text-align:right;">
34.36245
</td>
<td style="text-align:right;">
34.38045
</td>
<td style="text-align:left;">
negative
</td>
<td style="text-align:left;">
MGI_C57BL6J_3649681
</td>
<td style="text-align:left;">
Gm13858
</td>
<td style="text-align:left;">
NCBI_Gene:664942,ENSEMBL:ENSMUSG00000081196
</td>
<td style="text-align:left;">
MGI:3649681
</td>
<td style="text-align:left;">
pseudogene
</td>
<td style="text-align:left;">
predicted gene 13858
</td>
</tr>
<tr>
<td style="text-align:left;">
6
</td>
<td style="text-align:left;">
pseudogene
</td>
<td style="text-align:right;">
34.37369
</td>
<td style="text-align:right;">
34.38079
</td>
<td style="text-align:left;">
negative
</td>
<td style="text-align:left;">
MGI_C57BL6J_3649465
</td>
<td style="text-align:left;">
Gm13859
</td>
<td style="text-align:left;">
NCBI_Gene:100416263,ENSEMBL:ENSMUSG00000081991
</td>
<td style="text-align:left;">
MGI:3649465
</td>
<td style="text-align:left;">
pseudogene
</td>
<td style="text-align:left;">
predicted gene 13859
</td>
</tr>
<tr>
<td style="text-align:left;">
6
</td>
<td style="text-align:left;">
gene
</td>
<td style="text-align:right;">
34.38421
</td>
<td style="text-align:right;">
34.39696
</td>
<td style="text-align:left;">
positive
</td>
<td style="text-align:left;">
MGI_C57BL6J_1915111
</td>
<td style="text-align:left;">
Akr1b10
</td>
<td style="text-align:left;">
NCBI_Gene:67861,ENSEMBL:ENSMUSG00000061758
</td>
<td style="text-align:left;">
MGI:1915111
</td>
<td style="text-align:left;">
protein coding gene
</td>
<td style="text-align:left;">
aldo-keto reductase family 1, member B10 (aldose reductase)
</td>
</tr>
<tr>
<td style="text-align:left;">
6
</td>
<td style="text-align:left;">
gene
</td>
<td style="text-align:right;">
34.41233
</td>
<td style="text-align:right;">
34.42314
</td>
<td style="text-align:left;">
positive
</td>
<td style="text-align:left;">
MGI_C57BL6J_101918
</td>
<td style="text-align:left;">
Akr1b7
</td>
<td style="text-align:left;">
NCBI_Gene:11997,ENSEMBL:ENSMUSG00000052131
</td>
<td style="text-align:left;">
MGI:101918
</td>
<td style="text-align:left;">
protein coding gene
</td>
<td style="text-align:left;">
aldo-keto reductase family 1, member B7
</td>
</tr>
<tr>
<td style="text-align:left;">
6
</td>
<td style="text-align:left;">
pseudogene
</td>
<td style="text-align:right;">
34.41940
</td>
<td style="text-align:right;">
34.44544
</td>
<td style="text-align:left;">
negative
</td>
<td style="text-align:left;">
MGI_C57BL6J_3649688
</td>
<td style="text-align:left;">
Gm13857
</td>
<td style="text-align:left;">
NCBI_Gene:102635489,ENSEMBL:ENSMUSG00000084965
</td>
<td style="text-align:left;">
MGI:3649688
</td>
<td style="text-align:left;">
pseudogene
</td>
<td style="text-align:left;">
predicted gene 13857
</td>
</tr>
<tr>
<td style="text-align:left;">
6
</td>
<td style="text-align:left;">
gene
</td>
<td style="text-align:right;">
34.47621
</td>
<td style="text-align:right;">
34.50561
</td>
<td style="text-align:left;">
positive
</td>
<td style="text-align:left;">
MGI_C57BL6J_1098242
</td>
<td style="text-align:left;">
Bpgm
</td>
<td style="text-align:left;">
NCBI_Gene:12183,ENSEMBL:ENSMUSG00000038871
</td>
<td style="text-align:left;">
MGI:1098242
</td>
<td style="text-align:left;">
protein coding gene
</td>
<td style="text-align:left;">
2,3-bisphosphoglycerate mutase
</td>
</tr>
<tr>
<td style="text-align:left;">
6
</td>
<td style="text-align:left;">
gene
</td>
<td style="text-align:right;">
34.53355
</td>
<td style="text-align:right;">
34.54139
</td>
<td style="text-align:left;">
positive
</td>
<td style="text-align:left;">
MGI_C57BL6J_3649464
</td>
<td style="text-align:left;">
Gm13860
</td>
<td style="text-align:left;">
ENSEMBL:ENSMUSG00000085187
</td>
<td style="text-align:left;">
MGI:3649464
</td>
<td style="text-align:left;">
lncRNA gene
</td>
<td style="text-align:left;">
predicted gene 13860
</td>
</tr>
<tr>
<td style="text-align:left;">
6
</td>
<td style="text-align:left;">
gene
</td>
<td style="text-align:right;">
34.53406
</td>
<td style="text-align:right;">
34.53691
</td>
<td style="text-align:left;">
negative
</td>
<td style="text-align:left;">
MGI_C57BL6J_5592101
</td>
<td style="text-align:left;">
Gm32942
</td>
<td style="text-align:left;">
NCBI_Gene:102635662
</td>
<td style="text-align:left;">
MGI:5592101
</td>
<td style="text-align:left;">
lncRNA gene
</td>
<td style="text-align:left;">
predicted gene, 32942
</td>
</tr>
<tr>
<td style="text-align:left;">
6
</td>
<td style="text-align:left;">
gene
</td>
<td style="text-align:right;">
34.59850
</td>
<td style="text-align:right;">
34.77547
</td>
<td style="text-align:left;">
positive
</td>
<td style="text-align:left;">
MGI_C57BL6J_88250
</td>
<td style="text-align:left;">
Cald1
</td>
<td style="text-align:left;">
NCBI_Gene:109624,ENSEMBL:ENSMUSG00000029761
</td>
<td style="text-align:left;">
MGI:88250
</td>
<td style="text-align:left;">
protein coding gene
</td>
<td style="text-align:left;">
caldesmon 1
</td>
</tr>
<tr>
<td style="text-align:left;">
6
</td>
<td style="text-align:left;">
gene
</td>
<td style="text-align:right;">
34.65135
</td>
<td style="text-align:right;">
34.65433
</td>
<td style="text-align:left;">
negative
</td>
<td style="text-align:left;">
MGI_C57BL6J_3649466
</td>
<td style="text-align:left;">
Gm13861
</td>
<td style="text-align:left;">
NCBI_Gene:100039184,ENSEMBL:ENSMUSG00000085382
</td>
<td style="text-align:left;">
MGI:3649466
</td>
<td style="text-align:left;">
lncRNA gene
</td>
<td style="text-align:left;">
predicted gene 13861
</td>
</tr>
<tr>
<td style="text-align:left;">
6
</td>
<td style="text-align:left;">
gene
</td>
<td style="text-align:right;">
34.71229
</td>
<td style="text-align:right;">
34.71316
</td>
<td style="text-align:left;">
negative
</td>
<td style="text-align:left;">
MGI_C57BL6J_5621672
</td>
<td style="text-align:left;">
Gm38787
</td>
<td style="text-align:left;">
NCBI_Gene:105242628
</td>
<td style="text-align:left;">
MGI:5621672
</td>
<td style="text-align:left;">
lncRNA gene
</td>
<td style="text-align:left;">
predicted gene, 38787
</td>
</tr>
<tr>
<td style="text-align:left;">
6
</td>
<td style="text-align:left;">
gene
</td>
<td style="text-align:right;">
34.71480
</td>
<td style="text-align:right;">
34.71666
</td>
<td style="text-align:left;">
positive
</td>
<td style="text-align:left;">
MGI_C57BL6J_5012151
</td>
<td style="text-align:left;">
Gm19966
</td>
<td style="text-align:left;">
NA
</td>
<td style="text-align:left;">
NA
</td>
<td style="text-align:left;">
unclassified gene
</td>
<td style="text-align:left;">
predicted gene%2c 19966
</td>
</tr>
<tr>
<td style="text-align:left;">
6
</td>
<td style="text-align:left;">
gene
</td>
<td style="text-align:right;">
34.72667
</td>
<td style="text-align:right;">
34.72691
</td>
<td style="text-align:left;">
negative
</td>
<td style="text-align:left;">
MGI_C57BL6J_104972
</td>
<td style="text-align:left;">
Npn2
</td>
<td style="text-align:left;">
ENSEMBL:ENSMUSG00000060411
</td>
<td style="text-align:left;">
MGI:104972
</td>
<td style="text-align:left;">
lncRNA gene
</td>
<td style="text-align:left;">
neoplastic progression 2
</td>
</tr>
<tr>
<td style="text-align:left;">
6
</td>
<td style="text-align:left;">
gene
</td>
<td style="text-align:right;">
34.72722
</td>
<td style="text-align:right;">
34.73539
</td>
<td style="text-align:left;">
negative
</td>
<td style="text-align:left;">
MGI_C57BL6J_3705289
</td>
<td style="text-align:left;">
Gm14547
</td>
<td style="text-align:left;">
ENSEMBL:ENSMUSG00000086834
</td>
<td style="text-align:left;">
MGI:3705289
</td>
<td style="text-align:left;">
lncRNA gene
</td>
<td style="text-align:left;">
predicted gene 14547
</td>
</tr>
<tr>
<td style="text-align:left;">
6
</td>
<td style="text-align:left;">
gene
</td>
<td style="text-align:right;">
34.78043
</td>
<td style="text-align:right;">
34.85946
</td>
<td style="text-align:left;">
positive
</td>
<td style="text-align:left;">
MGI_C57BL6J_1923473
</td>
<td style="text-align:left;">
Agbl3
</td>
<td style="text-align:left;">
NCBI_Gene:76223,ENSEMBL:ENSMUSG00000038836
</td>
<td style="text-align:left;">
MGI:1923473
</td>
<td style="text-align:left;">
protein coding gene
</td>
<td style="text-align:left;">
ATP/GTP binding protein-like 3
</td>
</tr>
<tr>
<td style="text-align:left;">
6
</td>
<td style="text-align:left;">
pseudogene
</td>
<td style="text-align:right;">
34.78666
</td>
<td style="text-align:right;">
34.78751
</td>
<td style="text-align:left;">
positive
</td>
<td style="text-align:left;">
MGI_C57BL6J_3651491
</td>
<td style="text-align:left;">
Gm13864
</td>
<td style="text-align:left;">
NCBI_Gene:100384877,ENSEMBL:ENSMUSG00000082920
</td>
<td style="text-align:left;">
MGI:3651491
</td>
<td style="text-align:left;">
pseudogene
</td>
<td style="text-align:left;">
predicted gene 13864
</td>
</tr>
<tr>
<td style="text-align:left;">
6
</td>
<td style="text-align:left;">
pseudogene
</td>
<td style="text-align:right;">
34.81063
</td>
<td style="text-align:right;">
34.81083
</td>
<td style="text-align:left;">
negative
</td>
<td style="text-align:left;">
MGI_C57BL6J_3651503
</td>
<td style="text-align:left;">
Gm13862
</td>
<td style="text-align:left;">
ENSEMBL:ENSMUSG00000082118
</td>
<td style="text-align:left;">
MGI:3651503
</td>
<td style="text-align:left;">
pseudogene
</td>
<td style="text-align:left;">
predicted gene 13862
</td>
</tr>
<tr>
<td style="text-align:left;">
6
</td>
<td style="text-align:left;">
pseudogene
</td>
<td style="text-align:right;">
34.82020
</td>
<td style="text-align:right;">
34.82090
</td>
<td style="text-align:left;">
negative
</td>
<td style="text-align:left;">
MGI_C57BL6J_3652088
</td>
<td style="text-align:left;">
Gm13863
</td>
<td style="text-align:left;">
ENSEMBL:ENSMUSG00000082446
</td>
<td style="text-align:left;">
MGI:3652088
</td>
<td style="text-align:left;">
pseudogene
</td>
<td style="text-align:left;">
predicted gene 13863
</td>
</tr>
<tr>
<td style="text-align:left;">
6
</td>
<td style="text-align:left;">
gene
</td>
<td style="text-align:right;">
34.84134
</td>
<td style="text-align:right;">
34.84954
</td>
<td style="text-align:left;">
negative
</td>
<td style="text-align:left;">
MGI_C57BL6J_5826594
</td>
<td style="text-align:left;">
Gm46957
</td>
<td style="text-align:left;">
NCBI_Gene:108169138
</td>
<td style="text-align:left;">
MGI:5826594
</td>
<td style="text-align:left;">
lncRNA gene
</td>
<td style="text-align:left;">
predicted gene, 46957
</td>
</tr>
<tr>
<td style="text-align:left;">
6
</td>
<td style="text-align:left;">
gene
</td>
<td style="text-align:right;">
34.85970
</td>
<td style="text-align:right;">
34.86094
</td>
<td style="text-align:left;">
positive
</td>
<td style="text-align:left;">
MGI_C57BL6J_5663885
</td>
<td style="text-align:left;">
Gm43748
</td>
<td style="text-align:left;">
ENSEMBL:ENSMUSG00000106707
</td>
<td style="text-align:left;">
MGI:5663885
</td>
<td style="text-align:left;">
unclassified gene
</td>
<td style="text-align:left;">
predicted gene 43748
</td>
</tr>
<tr>
<td style="text-align:left;">
6
</td>
<td style="text-align:left;">
gene
</td>
<td style="text-align:right;">
34.86311
</td>
<td style="text-align:right;">
34.87495
</td>
<td style="text-align:left;">
positive
</td>
<td style="text-align:left;">
MGI_C57BL6J_1915737
</td>
<td style="text-align:left;">
Tmem140
</td>
<td style="text-align:left;">
NCBI_Gene:68487,ENSEMBL:ENSMUSG00000057137
</td>
<td style="text-align:left;">
MGI:1915737
</td>
<td style="text-align:left;">
protein coding gene
</td>
<td style="text-align:left;">
transmembrane protein 140
</td>
</tr>
<tr>
<td style="text-align:left;">
6
</td>
<td style="text-align:left;">
gene
</td>
<td style="text-align:right;">
34.87177
</td>
<td style="text-align:right;">
34.87806
</td>
<td style="text-align:left;">
negative
</td>
<td style="text-align:left;">
MGI_C57BL6J_1925662
</td>
<td style="text-align:left;">
Cyren
</td>
<td style="text-align:left;">
NCBI_Gene:78412,ENSEMBL:ENSMUSG00000046806
</td>
<td style="text-align:left;">
MGI:1925662
</td>
<td style="text-align:left;">
protein coding gene
</td>
<td style="text-align:left;">
cell cycle regulator of NHEJ
</td>
</tr>
<tr>
<td style="text-align:left;">
6
</td>
<td style="text-align:left;">
gene
</td>
<td style="text-align:right;">
34.87759
</td>
<td style="text-align:right;">
34.91089
</td>
<td style="text-align:left;">
negative
</td>
<td style="text-align:left;">
MGI_C57BL6J_2141558
</td>
<td style="text-align:left;">
Wdr91
</td>
<td style="text-align:left;">
NCBI_Gene:101240,ENSEMBL:ENSMUSG00000058486
</td>
<td style="text-align:left;">
MGI:2141558
</td>
<td style="text-align:left;">
protein coding gene
</td>
<td style="text-align:left;">
WD repeat domain 91
</td>
</tr>
<tr>
<td style="text-align:left;">
6
</td>
<td style="text-align:left;">
gene
</td>
<td style="text-align:right;">
34.92016
</td>
<td style="text-align:right;">
34.93934
</td>
<td style="text-align:left;">
positive
</td>
<td style="text-align:left;">
MGI_C57BL6J_107917
</td>
<td style="text-align:left;">
Stra8
</td>
<td style="text-align:left;">
NCBI_Gene:20899,ENSEMBL:ENSMUSG00000029848
</td>
<td style="text-align:left;">
MGI:107917
</td>
<td style="text-align:left;">
protein coding gene
</td>
<td style="text-align:left;">
stimulated by retinoic acid gene 8
</td>
</tr>
<tr>
<td style="text-align:left;">
6
</td>
<td style="text-align:left;">
gene
</td>
<td style="text-align:right;">
34.94528
</td>
<td style="text-align:right;">
34.98850
</td>
<td style="text-align:left;">
negative
</td>
<td style="text-align:left;">
MGI_C57BL6J_1917272
</td>
<td style="text-align:left;">
Slc23a4
</td>
<td style="text-align:left;">
NCBI_Gene:243753,ENSEMBL:ENSMUSG00000029847
</td>
<td style="text-align:left;">
MGI:1917272
</td>
<td style="text-align:left;">
protein coding gene
</td>
<td style="text-align:left;">
solute carrier family 23 member 4
</td>
</tr>
<tr>
<td style="text-align:left;">
6
</td>
<td style="text-align:left;">
gene
</td>
<td style="text-align:right;">
35.02206
</td>
<td style="text-align:right;">
35.13374
</td>
<td style="text-align:left;">
negative
</td>
<td style="text-align:left;">
MGI_C57BL6J_1859026
</td>
<td style="text-align:left;">
Cnot4
</td>
<td style="text-align:left;">
NCBI_Gene:53621,ENSEMBL:ENSMUSG00000038784
</td>
<td style="text-align:left;">
MGI:1859026
</td>
<td style="text-align:left;">
protein coding gene
</td>
<td style="text-align:left;">
CCR4-NOT transcription complex, subunit 4
</td>
</tr>
<tr>
<td style="text-align:left;">
6
</td>
<td style="text-align:left;">
gene
</td>
<td style="text-align:right;">
35.12152
</td>
<td style="text-align:right;">
35.12393
</td>
<td style="text-align:left;">
negative
</td>
<td style="text-align:left;">
MGI_C57BL6J_2443917
</td>
<td style="text-align:left;">
A230071A22Rik
</td>
<td style="text-align:left;">
NA
</td>
<td style="text-align:left;">
NA
</td>
<td style="text-align:left;">
unclassified gene
</td>
<td style="text-align:left;">
RIKEN cDNA A230071A22 gene
</td>
</tr>
<tr>
<td style="text-align:left;">
6
</td>
<td style="text-align:left;">
gene
</td>
<td style="text-align:right;">
35.13361
</td>
<td style="text-align:right;">
35.15214
</td>
<td style="text-align:left;">
positive
</td>
<td style="text-align:left;">
MGI_C57BL6J_5592205
</td>
<td style="text-align:left;">
Gm33046
</td>
<td style="text-align:left;">
NCBI_Gene:102635798
</td>
<td style="text-align:left;">
MGI:5592205
</td>
<td style="text-align:left;">
lncRNA gene
</td>
<td style="text-align:left;">
predicted gene, 33046
</td>
</tr>
<tr>
<td style="text-align:left;">
6
</td>
<td style="text-align:left;">
gene
</td>
<td style="text-align:right;">
35.17742
</td>
<td style="text-align:right;">
35.24760
</td>
<td style="text-align:left;">
positive
</td>
<td style="text-align:left;">
MGI_C57BL6J_2141625
</td>
<td style="text-align:left;">
Nup205
</td>
<td style="text-align:left;">
NCBI_Gene:70699,ENSEMBL:ENSMUSG00000038759
</td>
<td style="text-align:left;">
MGI:2141625
</td>
<td style="text-align:left;">
protein coding gene
</td>
<td style="text-align:left;">
nucleoporin 205
</td>
</tr>
<tr>
<td style="text-align:left;">
6
</td>
<td style="text-align:left;">
gene
</td>
<td style="text-align:right;">
35.24359
</td>
<td style="text-align:right;">
35.24375
</td>
<td style="text-align:left;">
positive
</td>
<td style="text-align:left;">
MGI_C57BL6J_5453212
</td>
<td style="text-align:left;">
Gm23435
</td>
<td style="text-align:left;">
ENSEMBL:ENSMUSG00000087808
</td>
<td style="text-align:left;">
MGI:5453212
</td>
<td style="text-align:left;">
snRNA gene
</td>
<td style="text-align:left;">
predicted gene, 23435
</td>
</tr>
<tr>
<td style="text-align:left;">
6
</td>
<td style="text-align:left;">
gene
</td>
<td style="text-align:right;">
35.25265
</td>
<td style="text-align:right;">
35.26350
</td>
<td style="text-align:left;">
positive
</td>
<td style="text-align:left;">
MGI_C57BL6J_1914955
</td>
<td style="text-align:left;">
1810058I24Rik
</td>
<td style="text-align:left;">
NCBI_Gene:67705,ENSEMBL:ENSMUSG00000073155
</td>
<td style="text-align:left;">
MGI:1914955
</td>
<td style="text-align:left;">
lncRNA gene
</td>
<td style="text-align:left;">
RIKEN cDNA 1810058I24 gene
</td>
</tr>
<tr>
<td style="text-align:left;">
6
</td>
<td style="text-align:left;">
gene
</td>
<td style="text-align:right;">
35.26795
</td>
<td style="text-align:right;">
35.30813
</td>
<td style="text-align:left;">
negative
</td>
<td style="text-align:left;">
MGI_C57BL6J_2442367
</td>
<td style="text-align:left;">
Slc13a4
</td>
<td style="text-align:left;">
NCBI_Gene:243755,ENSEMBL:ENSMUSG00000029843
</td>
<td style="text-align:left;">
MGI:2442367
</td>
<td style="text-align:left;">
protein coding gene
</td>
<td style="text-align:left;">
solute carrier family 13 (sodium/sulfate symporters), member 4
</td>
</tr>
<tr>
<td style="text-align:left;">
6
</td>
<td style="text-align:left;">
gene
</td>
<td style="text-align:right;">
35.31267
</td>
<td style="text-align:right;">
35.32614
</td>
<td style="text-align:left;">
negative
</td>
<td style="text-align:left;">
MGI_C57BL6J_3039626
</td>
<td style="text-align:left;">
Fam180a
</td>
<td style="text-align:left;">
NCBI_Gene:208164,ENSEMBL:ENSMUSG00000047420
</td>
<td style="text-align:left;">
MGI:3039626
</td>
<td style="text-align:left;">
protein coding gene
</td>
<td style="text-align:left;">
family with sequence similarity 180, member A
</td>
</tr>
<tr>
<td style="text-align:left;">
6
</td>
<td style="text-align:left;">
gene
</td>
<td style="text-align:right;">
35.31315
</td>
<td style="text-align:right;">
35.32232
</td>
<td style="text-align:left;">
positive
</td>
<td style="text-align:left;">
MGI_C57BL6J_5621673
</td>
<td style="text-align:left;">
Gm38788
</td>
<td style="text-align:left;">
NCBI_Gene:105242629
</td>
<td style="text-align:left;">
MGI:5621673
</td>
<td style="text-align:left;">
lncRNA gene
</td>
<td style="text-align:left;">
predicted gene, 38788
</td>
</tr>
<tr>
<td style="text-align:left;">
6
</td>
<td style="text-align:left;">
gene
</td>
<td style="text-align:right;">
35.33085
</td>
<td style="text-align:right;">
35.33683
</td>
<td style="text-align:left;">
positive
</td>
<td style="text-align:left;">
MGI_C57BL6J_1920659
</td>
<td style="text-align:left;">
1700065J11Rik
</td>
<td style="text-align:left;">
NCBI_Gene:73409,ENSEMBL:ENSMUSG00000107341
</td>
<td style="text-align:left;">
MGI:1920659
</td>
<td style="text-align:left;">
lncRNA gene
</td>
<td style="text-align:left;">
RIKEN cDNA 1700065J11 gene
</td>
</tr>
<tr>
<td style="text-align:left;">
6
</td>
<td style="text-align:left;">
gene
</td>
<td style="text-align:right;">
35.41280
</td>
<td style="text-align:right;">
35.41901
</td>
<td style="text-align:left;">
positive
</td>
<td style="text-align:left;">
MGI_C57BL6J_5621674
</td>
<td style="text-align:left;">
Gm38789
</td>
<td style="text-align:left;">
NCBI_Gene:105242630
</td>
<td style="text-align:left;">
MGI:5621674
</td>
<td style="text-align:left;">
lncRNA gene
</td>
<td style="text-align:left;">
predicted gene, 38789
</td>
</tr>
<tr>
<td style="text-align:left;">
6
</td>
<td style="text-align:left;">
gene
</td>
<td style="text-align:right;">
35.50882
</td>
<td style="text-align:right;">
35.53989
</td>
<td style="text-align:left;">
negative
</td>
<td style="text-align:left;">
MGI_C57BL6J_99445
</td>
<td style="text-align:left;">
Mtpn
</td>
<td style="text-align:left;">
NCBI_Gene:14489,ENSEMBL:ENSMUSG00000029840
</td>
<td style="text-align:left;">
MGI:99445
</td>
<td style="text-align:left;">
protein coding gene
</td>
<td style="text-align:left;">
myotrophin
</td>
</tr>
<tr>
<td style="text-align:left;">
6
</td>
<td style="text-align:left;">
gene
</td>
<td style="text-align:right;">
35.52454
</td>
<td style="text-align:right;">
35.52464
</td>
<td style="text-align:left;">
negative
</td>
<td style="text-align:left;">
MGI_C57BL6J_5452830
</td>
<td style="text-align:left;">
Gm23053
</td>
<td style="text-align:left;">
ENSEMBL:ENSMUSG00000095986
</td>
<td style="text-align:left;">
MGI:5452830
</td>
<td style="text-align:left;">
miRNA gene
</td>
<td style="text-align:left;">
predicted gene, 23053
</td>
</tr>
<tr>
<td style="text-align:left;">
6
</td>
<td style="text-align:left;">
gene
</td>
<td style="text-align:right;">
35.61109
</td>
<td style="text-align:right;">
35.61567
</td>
<td style="text-align:left;">
negative
</td>
<td style="text-align:left;">
MGI_C57BL6J_5663068
</td>
<td style="text-align:left;">
Gm42931
</td>
<td style="text-align:left;">
ENSEMBL:ENSMUSG00000107110
</td>
<td style="text-align:left;">
MGI:5663068
</td>
<td style="text-align:left;">
lncRNA gene
</td>
<td style="text-align:left;">
predicted gene 42931
</td>
</tr>
<tr>
<td style="text-align:left;">
6
</td>
<td style="text-align:left;">
gene
</td>
<td style="text-align:right;">
35.88328
</td>
<td style="text-align:right;">
35.88544
</td>
<td style="text-align:left;">
negative
</td>
<td style="text-align:left;">
MGI_C57BL6J_1921788
</td>
<td style="text-align:left;">
9030409K20Rik
</td>
<td style="text-align:left;">
NA
</td>
<td style="text-align:left;">
NA
</td>
<td style="text-align:left;">
unclassified gene
</td>
<td style="text-align:left;">
RIKEN cDNA 9030409K20 gene
</td>
</tr>
<tr>
<td style="text-align:left;">
6
</td>
<td style="text-align:left;">
pseudogene
</td>
<td style="text-align:right;">
35.92671
</td>
<td style="text-align:right;">
35.92712
</td>
<td style="text-align:left;">
positive
</td>
<td style="text-align:left;">
MGI_C57BL6J_5663579
</td>
<td style="text-align:left;">
Gm43442
</td>
<td style="text-align:left;">
ENSEMBL:ENSMUSG00000107145
</td>
<td style="text-align:left;">
MGI:5663579
</td>
<td style="text-align:left;">
pseudogene
</td>
<td style="text-align:left;">
predicted gene 43442
</td>
</tr>
<tr>
<td style="text-align:left;">
6
</td>
<td style="text-align:left;">
gene
</td>
<td style="text-align:right;">
35.96588
</td>
<td style="text-align:right;">
35.99970
</td>
<td style="text-align:left;">
negative
</td>
<td style="text-align:left;">
MGI_C57BL6J_5621675
</td>
<td style="text-align:left;">
Gm38790
</td>
<td style="text-align:left;">
NCBI_Gene:105242631
</td>
<td style="text-align:left;">
MGI:5621675
</td>
<td style="text-align:left;">
lncRNA gene
</td>
<td style="text-align:left;">
predicted gene, 38790
</td>
</tr>
<tr>
<td style="text-align:left;">
6
</td>
<td style="text-align:left;">
gene
</td>
<td style="text-align:right;">
35.97533
</td>
<td style="text-align:right;">
35.97546
</td>
<td style="text-align:left;">
positive
</td>
<td style="text-align:left;">
MGI_C57BL6J_5453050
</td>
<td style="text-align:left;">
Gm23273
</td>
<td style="text-align:left;">
ENSEMBL:ENSMUSG00000077246
</td>
<td style="text-align:left;">
MGI:5453050
</td>
<td style="text-align:left;">
snoRNA gene
</td>
<td style="text-align:left;">
predicted gene, 23273
</td>
</tr>
<tr>
<td style="text-align:left;">
6
</td>
<td style="text-align:left;">
gene
</td>
<td style="text-align:right;">
36.18671
</td>
<td style="text-align:right;">
36.18910
</td>
<td style="text-align:left;">
negative
</td>
<td style="text-align:left;">
MGI_C57BL6J_5592279
</td>
<td style="text-align:left;">
Gm33120
</td>
<td style="text-align:left;">
NA
</td>
<td style="text-align:left;">
NA
</td>
<td style="text-align:left;">
unclassified non-coding RNA gene
</td>
<td style="text-align:left;">
predicted gene%2c 33120
</td>
</tr>
<tr>
<td style="text-align:left;">
6
</td>
<td style="text-align:left;">
gene
</td>
<td style="text-align:right;">
36.18671
</td>
<td style="text-align:right;">
36.18910
</td>
<td style="text-align:left;">
negative
</td>
<td style="text-align:left;">
MGI_C57BL6J_5663580
</td>
<td style="text-align:left;">
Gm43443
</td>
<td style="text-align:left;">
ENSEMBL:ENSMUSG00000107089
</td>
<td style="text-align:left;">
MGI:5663580
</td>
<td style="text-align:left;">
unclassified gene
</td>
<td style="text-align:left;">
predicted gene 43443
</td>
</tr>
<tr>
<td style="text-align:left;">
6
</td>
<td style="text-align:left;">
gene
</td>
<td style="text-align:right;">
36.20962
</td>
<td style="text-align:right;">
36.39019
</td>
<td style="text-align:left;">
negative
</td>
<td style="text-align:left;">
MGI_C57BL6J_2442094
</td>
<td style="text-align:left;">
9330158H04Rik
</td>
<td style="text-align:left;">
NCBI_Gene:319472,ENSEMBL:ENSMUSG00000073154
</td>
<td style="text-align:left;">
MGI:2442094
</td>
<td style="text-align:left;">
lncRNA gene
</td>
<td style="text-align:left;">
RIKEN cDNA 9330158H04 gene
</td>
</tr>
<tr>
<td style="text-align:left;">
6
</td>
<td style="text-align:left;">
gene
</td>
<td style="text-align:right;">
36.38808
</td>
<td style="text-align:right;">
36.52864
</td>
<td style="text-align:left;">
positive
</td>
<td style="text-align:left;">
MGI_C57BL6J_88397
</td>
<td style="text-align:left;">
Chrm2
</td>
<td style="text-align:left;">
NCBI_Gene:243764,ENSEMBL:ENSMUSG00000045613
</td>
<td style="text-align:left;">
MGI:88397
</td>
<td style="text-align:left;">
protein coding gene
</td>
<td style="text-align:left;">
cholinergic receptor, muscarinic 2, cardiac
</td>
</tr>
<tr>
<td style="text-align:left;">
6
</td>
<td style="text-align:left;">
gene
</td>
<td style="text-align:right;">
36.42174
</td>
<td style="text-align:right;">
36.42182
</td>
<td style="text-align:left;">
positive
</td>
<td style="text-align:left;">
MGI_C57BL6J_3629662
</td>
<td style="text-align:left;">
Mir490
</td>
<td style="text-align:left;">
miRBase:MI0005002,NCBI_Gene:735279,ENSEMBL:ENSMUSG00000070075
</td>
<td style="text-align:left;">
MGI:3629662
</td>
<td style="text-align:left;">
miRNA gene
</td>
<td style="text-align:left;">
microRNA 490
</td>
</tr>
<tr>
<td style="text-align:left;">
6
</td>
<td style="text-align:left;">
gene
</td>
<td style="text-align:right;">
36.52502
</td>
<td style="text-align:right;">
36.52864
</td>
<td style="text-align:left;">
positive
</td>
<td style="text-align:left;">
MGI_C57BL6J_2444158
</td>
<td style="text-align:left;">
9330162L04Rik
</td>
<td style="text-align:left;">
NA
</td>
<td style="text-align:left;">
NA
</td>
<td style="text-align:left;">
unclassified gene
</td>
<td style="text-align:left;">
RIKEN cDNA 9330162L04 gene
</td>
</tr>
<tr>
<td style="text-align:left;">
6
</td>
<td style="text-align:left;">
pseudogene
</td>
<td style="text-align:right;">
36.58778
</td>
<td style="text-align:right;">
36.58798
</td>
<td style="text-align:left;">
positive
</td>
<td style="text-align:left;">
MGI_C57BL6J_5663578
</td>
<td style="text-align:left;">
Gm43441
</td>
<td style="text-align:left;">
ENSEMBL:ENSMUSG00000106817
</td>
<td style="text-align:left;">
MGI:5663578
</td>
<td style="text-align:left;">
pseudogene
</td>
<td style="text-align:left;">
predicted gene 43441
</td>
</tr>
<tr>
<td style="text-align:left;">
6
</td>
<td style="text-align:left;">
pseudogene
</td>
<td style="text-align:right;">
36.63981
</td>
<td style="text-align:right;">
36.64027
</td>
<td style="text-align:left;">
negative
</td>
<td style="text-align:left;">
MGI_C57BL6J_3780359
</td>
<td style="text-align:left;">
Gm2189
</td>
<td style="text-align:left;">
NCBI_Gene:100039370,ENSEMBL:ENSMUSG00000106872
</td>
<td style="text-align:left;">
MGI:3780359
</td>
<td style="text-align:left;">
pseudogene
</td>
<td style="text-align:left;">
predicted gene 2189
</td>
</tr>
<tr>
<td style="text-align:left;">
6
</td>
<td style="text-align:left;">
gene
</td>
<td style="text-align:right;">
36.69940
</td>
<td style="text-align:right;">
36.69954
</td>
<td style="text-align:left;">
positive
</td>
<td style="text-align:left;">
MGI_C57BL6J_5454888
</td>
<td style="text-align:left;">
Gm25111
</td>
<td style="text-align:left;">
ENSEMBL:ENSMUSG00000080336
</td>
<td style="text-align:left;">
MGI:5454888
</td>
<td style="text-align:left;">
miRNA gene
</td>
<td style="text-align:left;">
predicted gene, 25111
</td>
</tr>
<tr>
<td style="text-align:left;">
6
</td>
<td style="text-align:left;">
gene
</td>
<td style="text-align:right;">
36.71492
</td>
<td style="text-align:right;">
36.81136
</td>
<td style="text-align:left;">
negative
</td>
<td style="text-align:left;">
MGI_C57BL6J_97804
</td>
<td style="text-align:left;">
Ptn
</td>
<td style="text-align:left;">
NCBI_Gene:19242,ENSEMBL:ENSMUSG00000029838
</td>
<td style="text-align:left;">
MGI:97804
</td>
<td style="text-align:left;">
protein coding gene
</td>
<td style="text-align:left;">
pleiotrophin
</td>
</tr>
<tr>
<td style="text-align:left;">
6
</td>
<td style="text-align:left;">
gene
</td>
<td style="text-align:right;">
36.83738
</td>
<td style="text-align:right;">
37.30018
</td>
<td style="text-align:left;">
negative
</td>
<td style="text-align:left;">
MGI_C57BL6J_2443430
</td>
<td style="text-align:left;">
Dgki
</td>
<td style="text-align:left;">
NCBI_Gene:320127,ENSEMBL:ENSMUSG00000038665
</td>
<td style="text-align:left;">
MGI:2443430
</td>
<td style="text-align:left;">
protein coding gene
</td>
<td style="text-align:left;">
diacylglycerol kinase, iota
</td>
</tr>
<tr>
<td style="text-align:left;">
6
</td>
<td style="text-align:left;">
gene
</td>
<td style="text-align:right;">
36.84021
</td>
<td style="text-align:right;">
36.84399
</td>
<td style="text-align:left;">
negative
</td>
<td style="text-align:left;">
MGI_C57BL6J_2443216
</td>
<td style="text-align:left;">
6430604M11Rik
</td>
<td style="text-align:left;">
NA
</td>
<td style="text-align:left;">
NA
</td>
<td style="text-align:left;">
unclassified non-coding RNA gene
</td>
<td style="text-align:left;">
RIKEN cDNA 6430604M11 gene
</td>
</tr>
<tr>
<td style="text-align:left;">
6
</td>
<td style="text-align:left;">
gene
</td>
<td style="text-align:right;">
36.93705
</td>
<td style="text-align:right;">
36.98578
</td>
<td style="text-align:left;">
positive
</td>
<td style="text-align:left;">
MGI_C57BL6J_1920835
</td>
<td style="text-align:left;">
1700111E14Rik
</td>
<td style="text-align:left;">
ENSEMBL:ENSMUSG00000029837
</td>
<td style="text-align:left;">
MGI:1920835
</td>
<td style="text-align:left;">
lncRNA gene
</td>
<td style="text-align:left;">
RIKEN cDNA 1700111E14 gene
</td>
</tr>
<tr>
<td style="text-align:left;">
6
</td>
<td style="text-align:left;">
pseudogene
</td>
<td style="text-align:right;">
36.95665
</td>
<td style="text-align:right;">
36.96043
</td>
<td style="text-align:left;">
positive
</td>
<td style="text-align:left;">
MGI_C57BL6J_3645151
</td>
<td style="text-align:left;">
Gm7452
</td>
<td style="text-align:left;">
NCBI_Gene:665024,ENSEMBL:ENSMUSG00000107310
</td>
<td style="text-align:left;">
MGI:3645151
</td>
<td style="text-align:left;">
pseudogene
</td>
<td style="text-align:left;">
predicted pseudogene 7452
</td>
</tr>
<tr>
<td style="text-align:left;">
6
</td>
<td style="text-align:left;">
gene
</td>
<td style="text-align:right;">
37.32726
</td>
<td style="text-align:right;">
37.44219
</td>
<td style="text-align:left;">
negative
</td>
<td style="text-align:left;">
MGI_C57BL6J_2442695
</td>
<td style="text-align:left;">
Creb3l2
</td>
<td style="text-align:left;">
NCBI_Gene:208647,ENSEMBL:ENSMUSG00000038648
</td>
<td style="text-align:left;">
MGI:2442695
</td>
<td style="text-align:left;">
protein coding gene
</td>
<td style="text-align:left;">
cAMP responsive element binding protein 3-like 2
</td>
</tr>
<tr>
<td style="text-align:left;">
6
</td>
<td style="text-align:left;">
gene
</td>
<td style="text-align:right;">
37.41819
</td>
<td style="text-align:right;">
37.41945
</td>
<td style="text-align:left;">
positive
</td>
<td style="text-align:left;">
MGI_C57BL6J_5592434
</td>
<td style="text-align:left;">
Gm33275
</td>
<td style="text-align:left;">
NCBI_Gene:102636112
</td>
<td style="text-align:left;">
MGI:5592434
</td>
<td style="text-align:left;">
lncRNA gene
</td>
<td style="text-align:left;">
predicted gene, 33275
</td>
</tr>
<tr>
<td style="text-align:left;">
6
</td>
<td style="text-align:left;">
gene
</td>
<td style="text-align:right;">
37.44250
</td>
<td style="text-align:right;">
37.44741
</td>
<td style="text-align:left;">
positive
</td>
<td style="text-align:left;">
MGI_C57BL6J_5592477
</td>
<td style="text-align:left;">
Gm33318
</td>
<td style="text-align:left;">
NCBI_Gene:102636178
</td>
<td style="text-align:left;">
MGI:5592477
</td>
<td style="text-align:left;">
lncRNA gene
</td>
<td style="text-align:left;">
predicted gene, 33318
</td>
</tr>
<tr>
<td style="text-align:left;">
6
</td>
<td style="text-align:left;">
gene
</td>
<td style="text-align:right;">
37.52643
</td>
<td style="text-align:right;">
37.53140
</td>
<td style="text-align:left;">
negative
</td>
<td style="text-align:left;">
MGI_C57BL6J_5592588
</td>
<td style="text-align:left;">
Gm33429
</td>
<td style="text-align:left;">
NCBI_Gene:102636331
</td>
<td style="text-align:left;">
MGI:5592588
</td>
<td style="text-align:left;">
lncRNA gene
</td>
<td style="text-align:left;">
predicted gene, 33429
</td>
</tr>
<tr>
<td style="text-align:left;">
6
</td>
<td style="text-align:left;">
gene
</td>
<td style="text-align:right;">
37.52998
</td>
<td style="text-align:right;">
37.57493
</td>
<td style="text-align:left;">
positive
</td>
<td style="text-align:left;">
MGI_C57BL6J_2384785
</td>
<td style="text-align:left;">
Akr1d1
</td>
<td style="text-align:left;">
NCBI_Gene:208665,ENSEMBL:ENSMUSG00000038641
</td>
<td style="text-align:left;">
MGI:2384785
</td>
<td style="text-align:left;">
protein coding gene
</td>
<td style="text-align:left;">
aldo-keto reductase family 1, member D1
</td>
</tr>
<tr>
<td style="text-align:left;">
6
</td>
<td style="text-align:left;">
pseudogene
</td>
<td style="text-align:right;">
37.54141
</td>
<td style="text-align:right;">
37.54199
</td>
<td style="text-align:left;">
positive
</td>
<td style="text-align:left;">
MGI_C57BL6J_3645336
</td>
<td style="text-align:left;">
Gm7463
</td>
<td style="text-align:left;">
NCBI_Gene:665046,ENSEMBL:ENSMUSG00000106889
</td>
<td style="text-align:left;">
MGI:3645336
</td>
<td style="text-align:left;">
pseudogene
</td>
<td style="text-align:left;">
predicted gene 7463
</td>
</tr>
<tr>
<td style="text-align:left;">
6
</td>
<td style="text-align:left;">
pseudogene
</td>
<td style="text-align:right;">
37.68564
</td>
<td style="text-align:right;">
37.68731
</td>
<td style="text-align:left;">
negative
</td>
<td style="text-align:left;">
MGI_C57BL6J_2137671
</td>
<td style="text-align:left;">
Ybx1-ps2
</td>
<td style="text-align:left;">
NCBI_Gene:94283,ENSEMBL:ENSMUSG00000107035
</td>
<td style="text-align:left;">
MGI:2137671
</td>
<td style="text-align:left;">
pseudogene
</td>
<td style="text-align:left;">
Y box protein 1, pseudogene 2
</td>
</tr>
<tr>
<td style="text-align:left;">
6
</td>
<td style="text-align:left;">
gene
</td>
<td style="text-align:right;">
37.71466
</td>
<td style="text-align:right;">
37.73438
</td>
<td style="text-align:left;">
positive
</td>
<td style="text-align:left;">
MGI_C57BL6J_5621676
</td>
<td style="text-align:left;">
Gm38791
</td>
<td style="text-align:left;">
NCBI_Gene:105242632
</td>
<td style="text-align:left;">
MGI:5621676
</td>
<td style="text-align:left;">
lncRNA gene
</td>
<td style="text-align:left;">
predicted gene, 38791
</td>
</tr>
<tr>
<td style="text-align:left;">
6
</td>
<td style="text-align:left;">
pseudogene
</td>
<td style="text-align:right;">
37.80597
</td>
<td style="text-align:right;">
37.80711
</td>
<td style="text-align:left;">
positive
</td>
<td style="text-align:left;">
MGI_C57BL6J_2151816
</td>
<td style="text-align:left;">
Atp6v0c-ps2
</td>
<td style="text-align:left;">
NCBI_Gene:100039636,ENSEMBL:ENSMUSG00000080242
</td>
<td style="text-align:left;">
MGI:2151816
</td>
<td style="text-align:left;">
pseudogene
</td>
<td style="text-align:left;">
ATPase, H+ transporting, lysosomal V0 subunit C, pseudogene 2
</td>
</tr>
<tr>
<td style="text-align:left;">
6
</td>
<td style="text-align:left;">
pseudogene
</td>
<td style="text-align:right;">
37.86584
</td>
<td style="text-align:right;">
37.86656
</td>
<td style="text-align:left;">
negative
</td>
<td style="text-align:left;">
MGI_C57BL6J_3648381
</td>
<td style="text-align:left;">
Gm7492
</td>
<td style="text-align:left;">
NCBI_Gene:665102,ENSEMBL:ENSMUSG00000106853
</td>
<td style="text-align:left;">
MGI:3648381
</td>
<td style="text-align:left;">
pseudogene
</td>
<td style="text-align:left;">
predicted gene 7492
</td>
</tr>
<tr>
<td style="text-align:left;">
6
</td>
<td style="text-align:left;">
gene
</td>
<td style="text-align:right;">
37.86988
</td>
<td style="text-align:right;">
37.96845
</td>
<td style="text-align:left;">
positive
</td>
<td style="text-align:left;">
MGI_C57BL6J_109275
</td>
<td style="text-align:left;">
Trim24
</td>
<td style="text-align:left;">
NCBI_Gene:21848,ENSEMBL:ENSMUSG00000029833
</td>
<td style="text-align:left;">
MGI:109275
</td>
<td style="text-align:left;">
protein coding gene
</td>
<td style="text-align:left;">
tripartite motif-containing 24
</td>
</tr>
<tr>
<td style="text-align:left;">
6
</td>
<td style="text-align:left;">
gene
</td>
<td style="text-align:right;">
37.94102
</td>
<td style="text-align:right;">
37.94241
</td>
<td style="text-align:left;">
positive
</td>
<td style="text-align:left;">
MGI_C57BL6J_1918761
</td>
<td style="text-align:left;">
9030601B04Rik
</td>
<td style="text-align:left;">
NA
</td>
<td style="text-align:left;">
NA
</td>
<td style="text-align:left;">
unclassified gene
</td>
<td style="text-align:left;">
RIKEN cDNA 9030601B04 gene
</td>
</tr>
<tr>
<td style="text-align:left;">
6
</td>
<td style="text-align:left;">
gene
</td>
<td style="text-align:right;">
37.98374
</td>
<td style="text-align:right;">
38.04701
</td>
<td style="text-align:left;">
negative
</td>
<td style="text-align:left;">
MGI_C57BL6J_2444335
</td>
<td style="text-align:left;">
Svopl
</td>
<td style="text-align:left;">
NCBI_Gene:320590,ENSEMBL:ENSMUSG00000029830
</td>
<td style="text-align:left;">
MGI:2444335
</td>
<td style="text-align:left;">
protein coding gene
</td>
<td style="text-align:left;">
SV2 related protein homolog (rat)-like
</td>
</tr>
<tr>
<td style="text-align:left;">
6
</td>
<td style="text-align:left;">
pseudogene
</td>
<td style="text-align:right;">
37.99466
</td>
<td style="text-align:right;">
37.99532
</td>
<td style="text-align:left;">
negative
</td>
<td style="text-align:left;">
MGI_C57BL6J_5010586
</td>
<td style="text-align:left;">
Gm18401
</td>
<td style="text-align:left;">
NCBI_Gene:100417109
</td>
<td style="text-align:left;">
MGI:5010586
</td>
<td style="text-align:left;">
pseudogene
</td>
<td style="text-align:left;">
predicted gene, 18401
</td>
</tr>
<tr>
<td style="text-align:left;">
6
</td>
<td style="text-align:left;">
gene
</td>
<td style="text-align:right;">
38.03266
</td>
<td style="text-align:right;">
38.03551
</td>
<td style="text-align:left;">
positive
</td>
<td style="text-align:left;">
MGI_C57BL6J_5592653
</td>
<td style="text-align:left;">
Gm33494
</td>
<td style="text-align:left;">
NCBI_Gene:102636429
</td>
<td style="text-align:left;">
MGI:5592653
</td>
<td style="text-align:left;">
lncRNA gene
</td>
<td style="text-align:left;">
predicted gene, 33494
</td>
</tr>
<tr>
<td style="text-align:left;">
6
</td>
<td style="text-align:left;">
gene
</td>
<td style="text-align:right;">
38.04848
</td>
<td style="text-align:right;">
38.12459
</td>
<td style="text-align:left;">
negative
</td>
<td style="text-align:left;">
MGI_C57BL6J_2153480
</td>
<td style="text-align:left;">
Atp6v0a4
</td>
<td style="text-align:left;">
NCBI_Gene:140494,ENSEMBL:ENSMUSG00000038600
</td>
<td style="text-align:left;">
MGI:2153480
</td>
<td style="text-align:left;">
protein coding gene
</td>
<td style="text-align:left;">
ATPase, H+ transporting, lysosomal V0 subunit A4
</td>
</tr>
<tr>
<td style="text-align:left;">
6
</td>
<td style="text-align:left;">
gene
</td>
<td style="text-align:right;">
38.10925
</td>
<td style="text-align:right;">
38.11581
</td>
<td style="text-align:left;">
positive
</td>
<td style="text-align:left;">
MGI_C57BL6J_1924772
</td>
<td style="text-align:left;">
Tmem213
</td>
<td style="text-align:left;">
NCBI_Gene:77522,ENSEMBL:ENSMUSG00000029829
</td>
<td style="text-align:left;">
MGI:1924772
</td>
<td style="text-align:left;">
protein coding gene
</td>
<td style="text-align:left;">
transmembrane protein 213
</td>
</tr>
<tr>
<td style="text-align:left;">
6
</td>
<td style="text-align:left;">
gene
</td>
<td style="text-align:right;">
38.11838
</td>
<td style="text-align:right;">
38.11849
</td>
<td style="text-align:left;">
positive
</td>
<td style="text-align:left;">
MGI_C57BL6J_5453008
</td>
<td style="text-align:left;">
Gm23231
</td>
<td style="text-align:left;">
ENSEMBL:ENSMUSG00000087924
</td>
<td style="text-align:left;">
MGI:5453008
</td>
<td style="text-align:left;">
snRNA gene
</td>
<td style="text-align:left;">
predicted gene, 23231
</td>
</tr>
<tr>
<td style="text-align:left;">
6
</td>
<td style="text-align:left;">
gene
</td>
<td style="text-align:right;">
38.12317
</td>
<td style="text-align:right;">
38.25401
</td>
<td style="text-align:left;">
negative
</td>
<td style="text-align:left;">
MGI_C57BL6J_2669829
</td>
<td style="text-align:left;">
D630045J12Rik
</td>
<td style="text-align:left;">
NCBI_Gene:330286,ENSEMBL:ENSMUSG00000063455
</td>
<td style="text-align:left;">
MGI:2669829
</td>
<td style="text-align:left;">
protein coding gene
</td>
<td style="text-align:left;">
RIKEN cDNA D630045J12 gene
</td>
</tr>
<tr>
<td style="text-align:left;">
6
</td>
<td style="text-align:left;">
pseudogene
</td>
<td style="text-align:right;">
38.28444
</td>
<td style="text-align:right;">
38.28534
</td>
<td style="text-align:left;">
positive
</td>
<td style="text-align:left;">
MGI_C57BL6J_3643241
</td>
<td style="text-align:left;">
Gm7504
</td>
<td style="text-align:left;">
NCBI_Gene:665125,ENSEMBL:ENSMUSG00000107026
</td>
<td style="text-align:left;">
MGI:3643241
</td>
<td style="text-align:left;">
pseudogene
</td>
<td style="text-align:left;">
predicted gene 7504
</td>
</tr>
<tr>
<td style="text-align:left;">
6
</td>
<td style="text-align:left;">
gene
</td>
<td style="text-align:right;">
38.28739
</td>
<td style="text-align:right;">
38.29926
</td>
<td style="text-align:left;">
negative
</td>
<td style="text-align:left;">
MGI_C57BL6J_2443387
</td>
<td style="text-align:left;">
Zc3hav1l
</td>
<td style="text-align:left;">
NCBI_Gene:209032,ENSEMBL:ENSMUSG00000047749
</td>
<td style="text-align:left;">
MGI:2443387
</td>
<td style="text-align:left;">
protein coding gene
</td>
<td style="text-align:left;">
zinc finger CCCH-type, antiviral 1-like
</td>
</tr>
<tr>
<td style="text-align:left;">
6
</td>
<td style="text-align:left;">
gene
</td>
<td style="text-align:right;">
38.30529
</td>
<td style="text-align:right;">
38.35557
</td>
<td style="text-align:left;">
negative
</td>
<td style="text-align:left;">
MGI_C57BL6J_1926031
</td>
<td style="text-align:left;">
Zc3hav1
</td>
<td style="text-align:left;">
NCBI_Gene:78781,ENSEMBL:ENSMUSG00000029826
</td>
<td style="text-align:left;">
MGI:1926031
</td>
<td style="text-align:left;">
protein coding gene
</td>
<td style="text-align:left;">
zinc finger CCCH type, antiviral 1
</td>
</tr>
<tr>
<td style="text-align:left;">
6
</td>
<td style="text-align:left;">
pseudogene
</td>
<td style="text-align:right;">
38.36523
</td>
<td style="text-align:right;">
38.36590
</td>
<td style="text-align:left;">
negative
</td>
<td style="text-align:left;">
MGI_C57BL6J_5663715
</td>
<td style="text-align:left;">
Gm43578
</td>
<td style="text-align:left;">
ENSEMBL:ENSMUSG00000106836
</td>
<td style="text-align:left;">
MGI:5663715
</td>
<td style="text-align:left;">
pseudogene
</td>
<td style="text-align:left;">
predicted gene 43578
</td>
</tr>
<tr>
<td style="text-align:left;">
6
</td>
<td style="text-align:left;">
pseudogene
</td>
<td style="text-align:right;">
38.37674
</td>
<td style="text-align:right;">
38.37719
</td>
<td style="text-align:left;">
positive
</td>
<td style="text-align:left;">
MGI_C57BL6J_1321399
</td>
<td style="text-align:left;">
Rpl30-ps1
</td>
<td style="text-align:left;">
NCBI_Gene:19947,ENSEMBL:ENSMUSG00000057696
</td>
<td style="text-align:left;">
MGI:1321399
</td>
<td style="text-align:left;">
pseudogene
</td>
<td style="text-align:left;">
ribosomal protein L30, pseudogene 1
</td>
</tr>
<tr>
<td style="text-align:left;">
6
</td>
<td style="text-align:left;">
gene
</td>
<td style="text-align:right;">
38.38147
</td>
<td style="text-align:right;">
38.42765
</td>
<td style="text-align:left;">
positive
</td>
<td style="text-align:left;">
MGI_C57BL6J_2444853
</td>
<td style="text-align:left;">
Ttc26
</td>
<td style="text-align:left;">
NCBI_Gene:264134,ENSEMBL:ENSMUSG00000056832
</td>
<td style="text-align:left;">
MGI:2444853
</td>
<td style="text-align:left;">
protein coding gene
</td>
<td style="text-align:left;">
tetratricopeptide repeat domain 26
</td>
</tr>
<tr>
<td style="text-align:left;">
6
</td>
<td style="text-align:left;">
pseudogene
</td>
<td style="text-align:right;">
38.39627
</td>
<td style="text-align:right;">
38.39669
</td>
<td style="text-align:left;">
positive
</td>
<td style="text-align:left;">
MGI_C57BL6J_4414983
</td>
<td style="text-align:left;">
Gm16563
</td>
<td style="text-align:left;">
ENSEMBL:ENSMUSG00000089905
</td>
<td style="text-align:left;">
MGI:4414983
</td>
<td style="text-align:left;">
pseudogene
</td>
<td style="text-align:left;">
predicted gene 16563
</td>
</tr>
<tr>
<td style="text-align:left;">
6
</td>
<td style="text-align:left;">
gene
</td>
<td style="text-align:right;">
38.43304
</td>
<td style="text-align:right;">
38.43378
</td>
<td style="text-align:left;">
negative
</td>
<td style="text-align:left;">
MGI_C57BL6J_1919478
</td>
<td style="text-align:left;">
1700021L22Rik
</td>
<td style="text-align:left;">
NA
</td>
<td style="text-align:left;">
NA
</td>
<td style="text-align:left;">
unclassified gene
</td>
<td style="text-align:left;">
RIKEN cDNA 1700021L22 gene
</td>
</tr>
<tr>
<td style="text-align:left;">
6
</td>
<td style="text-align:left;">
gene
</td>
<td style="text-align:right;">
38.43393
</td>
<td style="text-align:right;">
38.52482
</td>
<td style="text-align:left;">
positive
</td>
<td style="text-align:left;">
MGI_C57BL6J_2444236
</td>
<td style="text-align:left;">
Ubn2
</td>
<td style="text-align:left;">
NCBI_Gene:320538,ENSEMBL:ENSMUSG00000038538
</td>
<td style="text-align:left;">
MGI:2444236
</td>
<td style="text-align:left;">
protein coding gene
</td>
<td style="text-align:left;">
ubinuclein 2
</td>
</tr>
<tr>
<td style="text-align:left;">
6
</td>
<td style="text-align:left;">
gene
</td>
<td style="text-align:right;">
38.44536
</td>
<td style="text-align:right;">
38.44705
</td>
<td style="text-align:left;">
positive
</td>
<td style="text-align:left;">
MGI_C57BL6J_2443652
</td>
<td style="text-align:left;">
E430026H10Rik
</td>
<td style="text-align:left;">
NA
</td>
<td style="text-align:left;">
NA
</td>
<td style="text-align:left;">
unclassified gene
</td>
<td style="text-align:left;">
RIKEN cDNA E430026H10 gene
</td>
</tr>
<tr>
<td style="text-align:left;">
6
</td>
<td style="text-align:left;">
gene
</td>
<td style="text-align:right;">
38.50733
</td>
<td style="text-align:right;">
38.50739
</td>
<td style="text-align:left;">
positive
</td>
<td style="text-align:left;">
MGI_C57BL6J_5531089
</td>
<td style="text-align:left;">
Mir7670
</td>
<td style="text-align:left;">
miRBase:MI0025010,NCBI_Gene:102466843,ENSEMBL:ENSMUSG00000098285
</td>
<td style="text-align:left;">
MGI:5531089
</td>
<td style="text-align:left;">
miRNA gene
</td>
<td style="text-align:left;">
microRNA 7670
</td>
</tr>
<tr>
<td style="text-align:left;">
6
</td>
<td style="text-align:left;">
gene
</td>
<td style="text-align:right;">
38.53258
</td>
<td style="text-align:right;">
38.53388
</td>
<td style="text-align:left;">
positive
</td>
<td style="text-align:left;">
MGI_C57BL6J_1920489
</td>
<td style="text-align:left;">
3110054G05Rik
</td>
<td style="text-align:left;">
NA
</td>
<td style="text-align:left;">
NA
</td>
<td style="text-align:left;">
unclassified gene
</td>
<td style="text-align:left;">
RIKEN cDNA 3110054G05 gene
</td>
</tr>
<tr>
<td style="text-align:left;">
6
</td>
<td style="text-align:left;">
gene
</td>
<td style="text-align:right;">
38.53350
</td>
<td style="text-align:right;">
38.53945
</td>
<td style="text-align:left;">
positive
</td>
<td style="text-align:left;">
MGI_C57BL6J_1913367
</td>
<td style="text-align:left;">
Fmc1
</td>
<td style="text-align:left;">
NCBI_Gene:66117,ENSEMBL:ENSMUSG00000019689
</td>
<td style="text-align:left;">
MGI:1913367
</td>
<td style="text-align:left;">
protein coding gene
</td>
<td style="text-align:left;">
formation of mitochondrial complex V assembly factor 1
</td>
</tr>
<tr>
<td style="text-align:left;">
6
</td>
<td style="text-align:left;">
gene
</td>
<td style="text-align:right;">
38.53393
</td>
<td style="text-align:right;">
38.53400
</td>
<td style="text-align:left;">
positive
</td>
<td style="text-align:left;">
MGI_C57BL6J_4413723
</td>
<td style="text-align:left;">
n-TRcct1
</td>
<td style="text-align:left;">
NCBI_Gene:102467248
</td>
<td style="text-align:left;">
MGI:4413723
</td>
<td style="text-align:left;">
tRNA gene
</td>
<td style="text-align:left;">
nuclear encoded tRNA arginine 1 (anticodon CCT)
</td>
</tr>
<tr>
<td style="text-align:left;">
6
</td>
<td style="text-align:left;">
pseudogene
</td>
<td style="text-align:right;">
38.54230
</td>
<td style="text-align:right;">
38.54610
</td>
<td style="text-align:left;">
negative
</td>
<td style="text-align:left;">
MGI_C57BL6J_3783138
</td>
<td style="text-align:left;">
Gm15697
</td>
<td style="text-align:left;">
NCBI_Gene:621228,ENSEMBL:ENSMUSG00000081670
</td>
<td style="text-align:left;">
MGI:3783138
</td>
<td style="text-align:left;">
pseudogene
</td>
<td style="text-align:left;">
predicted gene 15697
</td>
</tr>
<tr>
<td style="text-align:left;">
6
</td>
<td style="text-align:left;">
gene
</td>
<td style="text-align:right;">
38.55042
</td>
<td style="text-align:right;">
38.55125
</td>
<td style="text-align:left;">
negative
</td>
<td style="text-align:left;">
MGI_C57BL6J_2443093
</td>
<td style="text-align:left;">
D630014H12Rik
</td>
<td style="text-align:left;">
NA
</td>
<td style="text-align:left;">
NA
</td>
<td style="text-align:left;">
unclassified gene
</td>
<td style="text-align:left;">
RIKEN cDNA D630014H12 gene
</td>
</tr>
<tr>
<td style="text-align:left;">
6
</td>
<td style="text-align:left;">
gene
</td>
<td style="text-align:right;">
38.55133
</td>
<td style="text-align:right;">
38.60995
</td>
<td style="text-align:left;">
positive
</td>
<td style="text-align:left;">
MGI_C57BL6J_2183260
</td>
<td style="text-align:left;">
Luc7l2
</td>
<td style="text-align:left;">
NCBI_Gene:192196,ENSEMBL:ENSMUSG00000029823
</td>
<td style="text-align:left;">
MGI:2183260
</td>
<td style="text-align:left;">
protein coding gene
</td>
<td style="text-align:left;">
LUC7-like 2 (S. cerevisiae)
</td>
</tr>
<tr>
<td style="text-align:left;">
6
</td>
<td style="text-align:left;">
gene
</td>
<td style="text-align:right;">
38.55153
</td>
<td style="text-align:right;">
38.55401
</td>
<td style="text-align:left;">
negative
</td>
<td style="text-align:left;">
MGI_C57BL6J_5477193
</td>
<td style="text-align:left;">
Gm26699
</td>
<td style="text-align:left;">
ENSEMBL:ENSMUSG00000097206
</td>
<td style="text-align:left;">
MGI:5477193
</td>
<td style="text-align:left;">
lncRNA gene
</td>
<td style="text-align:left;">
predicted gene, 26699
</td>
</tr>
<tr>
<td style="text-align:left;">
6
</td>
<td style="text-align:left;">
gene
</td>
<td style="text-align:right;">
38.58236
</td>
<td style="text-align:right;">
38.58449
</td>
<td style="text-align:left;">
negative
</td>
<td style="text-align:left;">
MGI_C57BL6J_5012207
</td>
<td style="text-align:left;">
Gm20022
</td>
<td style="text-align:left;">
NA
</td>
<td style="text-align:left;">
NA
</td>
<td style="text-align:left;">
unclassified gene
</td>
<td style="text-align:left;">
predicted gene%2c 20022
</td>
</tr>
<tr>
<td style="text-align:left;">
6
</td>
<td style="text-align:left;">
gene
</td>
<td style="text-align:right;">
38.61005
</td>
<td style="text-align:right;">
38.61096
</td>
<td style="text-align:left;">
positive
</td>
<td style="text-align:left;">
MGI_C57BL6J_1924759
</td>
<td style="text-align:left;">
9330153N18Rik
</td>
<td style="text-align:left;">
NA
</td>
<td style="text-align:left;">
NA
</td>
<td style="text-align:left;">
unclassified gene
</td>
<td style="text-align:left;">
RIKEN cDNA 9330153N18 gene
</td>
</tr>
<tr>
<td style="text-align:left;">
6
</td>
<td style="text-align:left;">
gene
</td>
<td style="text-align:right;">
38.62141
</td>
<td style="text-align:right;">
38.63771
</td>
<td style="text-align:left;">
negative
</td>
<td style="text-align:left;">
MGI_C57BL6J_1921503
</td>
<td style="text-align:left;">
Klrg2
</td>
<td style="text-align:left;">
NCBI_Gene:74253,ENSEMBL:ENSMUSG00000071537
</td>
<td style="text-align:left;">
MGI:1921503
</td>
<td style="text-align:left;">
protein coding gene
</td>
<td style="text-align:left;">
killer cell lectin-like receptor subfamily G, member 2
</td>
</tr>
<tr>
<td style="text-align:left;">
6
</td>
<td style="text-align:left;">
gene
</td>
<td style="text-align:right;">
38.66307
</td>
<td style="text-align:right;">
38.68086
</td>
<td style="text-align:left;">
positive
</td>
<td style="text-align:left;">
MGI_C57BL6J_2141402
</td>
<td style="text-align:left;">
Clec2l
</td>
<td style="text-align:left;">
NCBI_Gene:665180,ENSEMBL:ENSMUSG00000079598
</td>
<td style="text-align:left;">
MGI:2141402
</td>
<td style="text-align:left;">
protein coding gene
</td>
<td style="text-align:left;">
C-type lectin domain family 2, member L
</td>
</tr>
<tr>
<td style="text-align:left;">
6
</td>
<td style="text-align:left;">
gene
</td>
<td style="text-align:right;">
38.68751
</td>
<td style="text-align:right;">
38.87619
</td>
<td style="text-align:left;">
negative
</td>
<td style="text-align:left;">
MGI_C57BL6J_1314872
</td>
<td style="text-align:left;">
Hipk2
</td>
<td style="text-align:left;">
NCBI_Gene:15258,ENSEMBL:ENSMUSG00000061436
</td>
<td style="text-align:left;">
MGI:1314872
</td>
<td style="text-align:left;">
protein coding gene
</td>
<td style="text-align:left;">
homeodomain interacting protein kinase 2
</td>
</tr>
<tr>
<td style="text-align:left;">
6
</td>
<td style="text-align:left;">
gene
</td>
<td style="text-align:right;">
38.69974
</td>
<td style="text-align:right;">
38.74190
</td>
<td style="text-align:left;">
positive
</td>
<td style="text-align:left;">
MGI_C57BL6J_5621381
</td>
<td style="text-align:left;">
Gm38496
</td>
<td style="text-align:left;">
NCBI_Gene:102637181
</td>
<td style="text-align:left;">
MGI:5621381
</td>
<td style="text-align:left;">
lncRNA gene
</td>
<td style="text-align:left;">
predicted gene, 38496
</td>
</tr>
<tr>
<td style="text-align:left;">
6
</td>
<td style="text-align:left;">
gene
</td>
<td style="text-align:right;">
38.75224
</td>
<td style="text-align:right;">
38.78048
</td>
<td style="text-align:left;">
positive
</td>
<td style="text-align:left;">
MGI_C57BL6J_2444621
</td>
<td style="text-align:left;">
4930502C15Rik
</td>
<td style="text-align:left;">
NCBI_Gene:320733
</td>
<td style="text-align:left;">
MGI:2444621
</td>
<td style="text-align:left;">
lncRNA gene
</td>
<td style="text-align:left;">
RIKEN cDNA 4930502C15 gene
</td>
</tr>
<tr>
<td style="text-align:left;">
6
</td>
<td style="text-align:left;">
gene
</td>
<td style="text-align:right;">
38.87538
</td>
<td style="text-align:right;">
39.08459
</td>
<td style="text-align:left;">
positive
</td>
<td style="text-align:left;">
MGI_C57BL6J_98497
</td>
<td style="text-align:left;">
Tbxas1
</td>
<td style="text-align:left;">
NCBI_Gene:21391,ENSEMBL:ENSMUSG00000029925
</td>
<td style="text-align:left;">
MGI:98497
</td>
<td style="text-align:left;">
protein coding gene
</td>
<td style="text-align:left;">
thromboxane A synthase 1, platelet
</td>
</tr>
<tr>
<td style="text-align:left;">
6
</td>
<td style="text-align:left;">
gene
</td>
<td style="text-align:right;">
38.87611
</td>
<td style="text-align:right;">
38.88505
</td>
<td style="text-align:left;">
positive
</td>
<td style="text-align:left;">
MGI_C57BL6J_5826625
</td>
<td style="text-align:left;">
Gm46988
</td>
<td style="text-align:left;">
NCBI_Gene:108169201
</td>
<td style="text-align:left;">
MGI:5826625
</td>
<td style="text-align:left;">
protein coding gene
</td>
<td style="text-align:left;">
predicted gene, 46988
</td>
</tr>
<tr>
<td style="text-align:left;">
6
</td>
<td style="text-align:left;">
gene
</td>
<td style="text-align:right;">
38.91710
</td>
<td style="text-align:right;">
38.98432
</td>
<td style="text-align:left;">
negative
</td>
<td style="text-align:left;">
MGI_C57BL6J_5621678
</td>
<td style="text-align:left;">
Gm38793
</td>
<td style="text-align:left;">
NA
</td>
<td style="text-align:left;">
NA
</td>
<td style="text-align:left;">
lncRNA gene
</td>
<td style="text-align:left;">
predicted gene%2c 38793
</td>
</tr>
<tr>
<td style="text-align:left;">
6
</td>
<td style="text-align:left;">
gene
</td>
<td style="text-align:right;">
38.91710
</td>
<td style="text-align:right;">
38.98432
</td>
<td style="text-align:left;">
negative
</td>
<td style="text-align:left;">
MGI_C57BL6J_5663099
</td>
<td style="text-align:left;">
Gm42962
</td>
<td style="text-align:left;">
ENSEMBL:ENSMUSG00000106737
</td>
<td style="text-align:left;">
MGI:5663099
</td>
<td style="text-align:left;">
lncRNA gene
</td>
<td style="text-align:left;">
predicted gene 42962
</td>
</tr>
<tr>
<td style="text-align:left;">
6
</td>
<td style="text-align:left;">
gene
</td>
<td style="text-align:right;">
38.94437
</td>
<td style="text-align:right;">
38.95028
</td>
<td style="text-align:left;">
negative
</td>
<td style="text-align:left;">
MGI_C57BL6J_5592926
</td>
<td style="text-align:left;">
Gm33767
</td>
<td style="text-align:left;">
NCBI_Gene:102636794
</td>
<td style="text-align:left;">
MGI:5592926
</td>
<td style="text-align:left;">
lncRNA gene
</td>
<td style="text-align:left;">
predicted gene, 33767
</td>
</tr>
<tr>
<td style="text-align:left;">
6
</td>
<td style="text-align:left;">
gene
</td>
<td style="text-align:right;">
38.98884
</td>
<td style="text-align:right;">
39.00644
</td>
<td style="text-align:left;">
negative
</td>
<td style="text-align:left;">
MGI_C57BL6J_5621573
</td>
<td style="text-align:left;">
Gm38688
</td>
<td style="text-align:left;">
NCBI_Gene:105242415
</td>
<td style="text-align:left;">
MGI:5621573
</td>
<td style="text-align:left;">
lncRNA gene
</td>
<td style="text-align:left;">
predicted gene, 38688
</td>
</tr>
<tr>
<td style="text-align:left;">
6
</td>
<td style="text-align:left;">
gene
</td>
<td style="text-align:right;">
39.01367
</td>
<td style="text-align:right;">
39.01379
</td>
<td style="text-align:left;">
positive
</td>
<td style="text-align:left;">
MGI_C57BL6J_5452787
</td>
<td style="text-align:left;">
Gm23010
</td>
<td style="text-align:left;">
ENSEMBL:ENSMUSG00000077339
</td>
<td style="text-align:left;">
MGI:5452787
</td>
<td style="text-align:left;">
snoRNA gene
</td>
<td style="text-align:left;">
predicted gene, 23010
</td>
</tr>
<tr>
<td style="text-align:left;">
6
</td>
<td style="text-align:left;">
gene
</td>
<td style="text-align:right;">
39.01920
</td>
<td style="text-align:right;">
39.04383
</td>
<td style="text-align:left;">
negative
</td>
<td style="text-align:left;">
MGI_C57BL6J_5621374
</td>
<td style="text-align:left;">
Gm38489
</td>
<td style="text-align:left;">
NCBI_Gene:102636584
</td>
<td style="text-align:left;">
MGI:5621374
</td>
<td style="text-align:left;">
lncRNA gene
</td>
<td style="text-align:left;">
predicted gene, 38489
</td>
</tr>
<tr>
<td style="text-align:left;">
6
</td>
<td style="text-align:left;">
gene
</td>
<td style="text-align:right;">
39.04417
</td>
<td style="text-align:right;">
39.05652
</td>
<td style="text-align:left;">
negative
</td>
<td style="text-align:left;">
MGI_C57BL6J_5826626
</td>
<td style="text-align:left;">
Gm46989
</td>
<td style="text-align:left;">
NCBI_Gene:108169202
</td>
<td style="text-align:left;">
MGI:5826626
</td>
<td style="text-align:left;">
lncRNA gene
</td>
<td style="text-align:left;">
predicted gene, 46989
</td>
</tr>
<tr>
<td style="text-align:left;">
6
</td>
<td style="text-align:left;">
gene
</td>
<td style="text-align:right;">
39.06388
</td>
<td style="text-align:right;">
39.06760
</td>
<td style="text-align:left;">
negative
</td>
<td style="text-align:left;">
MGI_C57BL6J_1917236
</td>
<td style="text-align:left;">
1700025N23Rik
</td>
<td style="text-align:left;">
NCBI_Gene:69986,ENSEMBL:ENSMUSG00000100121
</td>
<td style="text-align:left;">
MGI:1917236
</td>
<td style="text-align:left;">
lncRNA gene
</td>
<td style="text-align:left;">
RIKEN cDNA 1700025N23 gene
</td>
</tr>
<tr>
<td style="text-align:left;">
6
</td>
<td style="text-align:left;">
gene
</td>
<td style="text-align:right;">
39.08641
</td>
<td style="text-align:right;">
39.11835
</td>
<td style="text-align:left;">
negative
</td>
<td style="text-align:left;">
MGI_C57BL6J_2143990
</td>
<td style="text-align:left;">
Parp12
</td>
<td style="text-align:left;">
NCBI_Gene:243771,ENSEMBL:ENSMUSG00000038507
</td>
<td style="text-align:left;">
MGI:2143990
</td>
<td style="text-align:left;">
protein coding gene
</td>
<td style="text-align:left;">
poly (ADP-ribose) polymerase family, member 12
</td>
</tr>
<tr>
<td style="text-align:left;">
6
</td>
<td style="text-align:left;">
gene
</td>
<td style="text-align:right;">
39.11847
</td>
<td style="text-align:right;">
39.14831
</td>
<td style="text-align:left;">
positive
</td>
<td style="text-align:left;">
MGI_C57BL6J_1922629
</td>
<td style="text-align:left;">
4930599N23Rik
</td>
<td style="text-align:left;">
NCBI_Gene:75379,ENSEMBL:ENSMUSG00000073144
</td>
<td style="text-align:left;">
MGI:1922629
</td>
<td style="text-align:left;">
lncRNA gene
</td>
<td style="text-align:left;">
RIKEN cDNA 4930599N23 gene
</td>
</tr>
<tr>
<td style="text-align:left;">
6
</td>
<td style="text-align:left;">
gene
</td>
<td style="text-align:right;">
39.12517
</td>
<td style="text-align:right;">
39.12638
</td>
<td style="text-align:left;">
negative
</td>
<td style="text-align:left;">
MGI_C57BL6J_5663100
</td>
<td style="text-align:left;">
Gm42963
</td>
<td style="text-align:left;">
ENSEMBL:ENSMUSG00000106976
</td>
<td style="text-align:left;">
MGI:5663100
</td>
<td style="text-align:left;">
unclassified gene
</td>
<td style="text-align:left;">
predicted gene 42963
</td>
</tr>
<tr>
<td style="text-align:left;">
6
</td>
<td style="text-align:left;">
gene
</td>
<td style="text-align:right;">
39.13662
</td>
<td style="text-align:right;">
39.20679
</td>
<td style="text-align:left;">
negative
</td>
<td style="text-align:left;">
MGI_C57BL6J_2443388
</td>
<td style="text-align:left;">
Kdm7a
</td>
<td style="text-align:left;">
NCBI_Gene:338523,ENSEMBL:ENSMUSG00000042599
</td>
<td style="text-align:left;">
MGI:2443388
</td>
<td style="text-align:left;">
protein coding gene
</td>
<td style="text-align:left;">
lysine (K)-specific demethylase 7A
</td>
</tr>
<tr>
<td style="text-align:left;">
6
</td>
<td style="text-align:left;">
gene
</td>
<td style="text-align:right;">
39.20707
</td>
<td style="text-align:right;">
39.20761
</td>
<td style="text-align:left;">
positive
</td>
<td style="text-align:left;">
MGI_C57BL6J_6118414
</td>
<td style="text-align:left;">
Gm49041
</td>
<td style="text-align:left;">
ENSEMBL:ENSMUSG00000115026
</td>
<td style="text-align:left;">
MGI:6118414
</td>
<td style="text-align:left;">
lncRNA gene
</td>
<td style="text-align:left;">
predicted gene, 49041
</td>
</tr>
<tr>
<td style="text-align:left;">
6
</td>
<td style="text-align:left;">
gene
</td>
<td style="text-align:right;">
39.22756
</td>
<td style="text-align:right;">
39.24535
</td>
<td style="text-align:left;">
positive
</td>
<td style="text-align:left;">
MGI_C57BL6J_5621679
</td>
<td style="text-align:left;">
Gm38794
</td>
<td style="text-align:left;">
NCBI_Gene:105242635
</td>
<td style="text-align:left;">
MGI:5621679
</td>
<td style="text-align:left;">
lncRNA gene
</td>
<td style="text-align:left;">
predicted gene, 38794
</td>
</tr>
<tr>
<td style="text-align:left;">
6
</td>
<td style="text-align:left;">
gene
</td>
<td style="text-align:right;">
39.24011
</td>
<td style="text-align:right;">
39.24254
</td>
<td style="text-align:left;">
positive
</td>
<td style="text-align:left;">
MGI_C57BL6J_5663616
</td>
<td style="text-align:left;">
Gm43479
</td>
<td style="text-align:left;">
ENSEMBL:ENSMUSG00000107365
</td>
<td style="text-align:left;">
MGI:5663616
</td>
<td style="text-align:left;">
lncRNA gene
</td>
<td style="text-align:left;">
predicted gene 43479
</td>
</tr>
<tr>
<td style="text-align:left;">
6
</td>
<td style="text-align:left;">
gene
</td>
<td style="text-align:right;">
39.24615
</td>
<td style="text-align:right;">
39.26561
</td>
<td style="text-align:left;">
negative
</td>
<td style="text-align:left;">
MGI_C57BL6J_5593306
</td>
<td style="text-align:left;">
Gm34147
</td>
<td style="text-align:left;">
NCBI_Gene:102637300
</td>
<td style="text-align:left;">
MGI:5593306
</td>
<td style="text-align:left;">
lncRNA gene
</td>
<td style="text-align:left;">
predicted gene, 34147
</td>
</tr>
<tr>
<td style="text-align:left;">
6
</td>
<td style="text-align:left;">
gene
</td>
<td style="text-align:right;">
39.26565
</td>
<td style="text-align:right;">
39.29957
</td>
<td style="text-align:left;">
negative
</td>
<td style="text-align:left;">
MGI_C57BL6J_5455785
</td>
<td style="text-align:left;">
Gm26008
</td>
<td style="text-align:left;">
NCBI_Gene:102637404,ENSEMBL:ENSMUSG00000089422
</td>
<td style="text-align:left;">
MGI:5455785
</td>
<td style="text-align:left;">
snRNA gene
</td>
<td style="text-align:left;">
predicted gene, 26008
</td>
</tr>
<tr>
<td style="text-align:left;">
6
</td>
<td style="text-align:left;">
gene
</td>
<td style="text-align:right;">
39.33477
</td>
<td style="text-align:right;">
39.37772
</td>
<td style="text-align:left;">
negative
</td>
<td style="text-align:left;">
MGI_C57BL6J_1919394
</td>
<td style="text-align:left;">
Slc37a3
</td>
<td style="text-align:left;">
NCBI_Gene:72144,ENSEMBL:ENSMUSG00000029924
</td>
<td style="text-align:left;">
MGI:1919394
</td>
<td style="text-align:left;">
protein coding gene
</td>
<td style="text-align:left;">
solute carrier family 37 (glycerol-3-phosphate transporter), member 3
</td>
</tr>
<tr>
<td style="text-align:left;">
6
</td>
<td style="text-align:left;">
gene
</td>
<td style="text-align:right;">
39.38117
</td>
<td style="text-align:right;">
39.39067
</td>
<td style="text-align:left;">
positive
</td>
<td style="text-align:left;">
MGI_C57BL6J_103292
</td>
<td style="text-align:left;">
Rab19
</td>
<td style="text-align:left;">
NCBI_Gene:19331,ENSEMBL:ENSMUSG00000029923
</td>
<td style="text-align:left;">
MGI:103292
</td>
<td style="text-align:left;">
protein coding gene
</td>
<td style="text-align:left;">
RAB19, member RAS oncogene family
</td>
</tr>
<tr>
<td style="text-align:left;">
6
</td>
<td style="text-align:left;">
gene
</td>
<td style="text-align:right;">
39.39780
</td>
<td style="text-align:right;">
39.42330
</td>
<td style="text-align:left;">
negative
</td>
<td style="text-align:left;">
MGI_C57BL6J_1859353
</td>
<td style="text-align:left;">
Mkrn1
</td>
<td style="text-align:left;">
NCBI_Gene:54484,ENSEMBL:ENSMUSG00000029922
</td>
<td style="text-align:left;">
MGI:1859353
</td>
<td style="text-align:left;">
protein coding gene
</td>
<td style="text-align:left;">
makorin, ring finger protein, 1
</td>
</tr>
<tr>
<td style="text-align:left;">
6
</td>
<td style="text-align:left;">
gene
</td>
<td style="text-align:right;">
39.41989
</td>
<td style="text-align:right;">
39.42129
</td>
<td style="text-align:left;">
positive
</td>
<td style="text-align:left;">
MGI_C57BL6J_3642504
</td>
<td style="text-align:left;">
Gm10244
</td>
<td style="text-align:left;">
NCBI_Gene:621427,ENSEMBL:ENSMUSG00000068601
</td>
<td style="text-align:left;">
MGI:3642504
</td>
<td style="text-align:left;">
lncRNA gene
</td>
<td style="text-align:left;">
predicted gene 10244
</td>
</tr>
<tr>
<td style="text-align:left;">
6
</td>
<td style="text-align:left;">
gene
</td>
<td style="text-align:right;">
39.42229
</td>
<td style="text-align:right;">
39.42242
</td>
<td style="text-align:left;">
negative
</td>
<td style="text-align:left;">
MGI_C57BL6J_5455179
</td>
<td style="text-align:left;">
Gm25402
</td>
<td style="text-align:left;">
ENSEMBL:ENSMUSG00000064900
</td>
<td style="text-align:left;">
MGI:5455179
</td>
<td style="text-align:left;">
snoRNA gene
</td>
<td style="text-align:left;">
predicted gene, 25402
</td>
</tr>
<tr>
<td style="text-align:left;">
6
</td>
<td style="text-align:left;">
gene
</td>
<td style="text-align:right;">
39.46237
</td>
<td style="text-align:right;">
39.55817
</td>
<td style="text-align:left;">
negative
</td>
<td style="text-align:left;">
MGI_C57BL6J_2444961
</td>
<td style="text-align:left;">
Dennd2a
</td>
<td style="text-align:left;">
NCBI_Gene:209773,ENSEMBL:ENSMUSG00000038456
</td>
<td style="text-align:left;">
MGI:2444961
</td>
<td style="text-align:left;">
protein coding gene
</td>
<td style="text-align:left;">
DENN/MADD domain containing 2A
</td>
</tr>
<tr>
<td style="text-align:left;">
6
</td>
<td style="text-align:left;">
gene
</td>
<td style="text-align:right;">
39.52751
</td>
<td style="text-align:right;">
39.54449
</td>
<td style="text-align:left;">
positive
</td>
<td style="text-align:left;">
MGI_C57BL6J_1924459
</td>
<td style="text-align:left;">
8030453O22Rik
</td>
<td style="text-align:left;">
NCBI_Gene:77209,ENSEMBL:ENSMUSG00000085058
</td>
<td style="text-align:left;">
MGI:1924459
</td>
<td style="text-align:left;">
lncRNA gene
</td>
<td style="text-align:left;">
RIKEN cDNA 8030453O22 gene
</td>
</tr>
<tr>
<td style="text-align:left;">
6
</td>
<td style="text-align:left;">
gene
</td>
<td style="text-align:right;">
39.57384
</td>
<td style="text-align:right;">
39.58877
</td>
<td style="text-align:left;">
positive
</td>
<td style="text-align:left;">
MGI_C57BL6J_1889336
</td>
<td style="text-align:left;">
Adck2
</td>
<td style="text-align:left;">
NCBI_Gene:57869,ENSEMBL:ENSMUSG00000046947
</td>
<td style="text-align:left;">
MGI:1889336
</td>
<td style="text-align:left;">
protein coding gene
</td>
<td style="text-align:left;">
aarF domain containing kinase 2
</td>
</tr>
<tr>
<td style="text-align:left;">
6
</td>
<td style="text-align:left;">
gene
</td>
<td style="text-align:right;">
39.58409
</td>
<td style="text-align:right;">
39.59278
</td>
<td style="text-align:left;">
negative
</td>
<td style="text-align:left;">
MGI_C57BL6J_3826603
</td>
<td style="text-align:left;">
Gm16272
</td>
<td style="text-align:left;">
ENSEMBL:ENSMUSG00000086582
</td>
<td style="text-align:left;">
MGI:3826603
</td>
<td style="text-align:left;">
lncRNA gene
</td>
<td style="text-align:left;">
predicted gene 16272
</td>
</tr>
<tr>
<td style="text-align:left;">
6
</td>
<td style="text-align:left;">
gene
</td>
<td style="text-align:right;">
39.58575
</td>
<td style="text-align:right;">
39.59660
</td>
<td style="text-align:left;">
positive
</td>
<td style="text-align:left;">
MGI_C57BL6J_5649067
</td>
<td style="text-align:left;">
Gm42420
</td>
<td style="text-align:left;">
ENSEMBL:ENSMUSG00000107071
</td>
<td style="text-align:left;">
MGI:5649067
</td>
<td style="text-align:left;">
protein coding gene
</td>
<td style="text-align:left;">
predicted gene, 42420
</td>
</tr>
<tr>
<td style="text-align:left;">
6
</td>
<td style="text-align:left;">
gene
</td>
<td style="text-align:right;">
39.59255
</td>
<td style="text-align:right;">
39.60338
</td>
<td style="text-align:left;">
positive
</td>
<td style="text-align:left;">
MGI_C57BL6J_1915448
</td>
<td style="text-align:left;">
Ndufb2
</td>
<td style="text-align:left;">
NCBI_Gene:68198,ENSEMBL:ENSMUSG00000002416
</td>
<td style="text-align:left;">
MGI:1915448
</td>
<td style="text-align:left;">
protein coding gene
</td>
<td style="text-align:left;">
NADH:ubiquinone oxidoreductase subunit B2
</td>
</tr>
<tr>
<td style="text-align:left;">
6
</td>
<td style="text-align:left;">
gene
</td>
<td style="text-align:right;">
39.60323
</td>
<td style="text-align:right;">
39.72566
</td>
<td style="text-align:left;">
negative
</td>
<td style="text-align:left;">
MGI_C57BL6J_88190
</td>
<td style="text-align:left;">
Braf
</td>
<td style="text-align:left;">
NCBI_Gene:109880,ENSEMBL:ENSMUSG00000002413
</td>
<td style="text-align:left;">
MGI:88190
</td>
<td style="text-align:left;">
protein coding gene
</td>
<td style="text-align:left;">
Braf transforming gene
</td>
</tr>
<tr>
<td style="text-align:left;">
6
</td>
<td style="text-align:left;">
gene
</td>
<td style="text-align:right;">
39.61650
</td>
<td style="text-align:right;">
39.62507
</td>
<td style="text-align:left;">
positive
</td>
<td style="text-align:left;">
MGI_C57BL6J_5593548
</td>
<td style="text-align:left;">
Gm34389
</td>
<td style="text-align:left;">
NCBI_Gene:102637627
</td>
<td style="text-align:left;">
MGI:5593548
</td>
<td style="text-align:left;">
lncRNA gene
</td>
<td style="text-align:left;">
predicted gene, 34389
</td>
</tr>
<tr>
<td style="text-align:left;">
6
</td>
<td style="text-align:left;">
pseudogene
</td>
<td style="text-align:right;">
39.65330
</td>
<td style="text-align:right;">
39.65396
</td>
<td style="text-align:left;">
negative
</td>
<td style="text-align:left;">
MGI_C57BL6J_104594
</td>
<td style="text-align:left;">
Fth-ps2
</td>
<td style="text-align:left;">
NCBI_Gene:14321,ENSEMBL:ENSMUSG00000091313
</td>
<td style="text-align:left;">
MGI:104594
</td>
<td style="text-align:left;">
pseudogene
</td>
<td style="text-align:left;">
ferritin heavy chain, pseudogene 2
</td>
</tr>
<tr>
<td style="text-align:left;">
6
</td>
<td style="text-align:left;">
gene
</td>
<td style="text-align:right;">
39.80180
</td>
<td style="text-align:right;">
39.81099
</td>
<td style="text-align:left;">
negative
</td>
<td style="text-align:left;">
MGI_C57BL6J_1338046
</td>
<td style="text-align:left;">
Mrps33
</td>
<td style="text-align:left;">
NCBI_Gene:14548,ENSEMBL:ENSMUSG00000029918
</td>
<td style="text-align:left;">
MGI:1338046
</td>
<td style="text-align:left;">
protein coding gene
</td>
<td style="text-align:left;">
mitochondrial ribosomal protein S33
</td>
</tr>
<tr>
<td style="text-align:left;">
6
</td>
<td style="text-align:left;">
gene
</td>
<td style="text-align:right;">
39.81018
</td>
<td style="text-align:right;">
39.81065
</td>
<td style="text-align:left;">
negative
</td>
<td style="text-align:left;">
MGI_C57BL6J_1925337
</td>
<td style="text-align:left;">
5730402E23Rik
</td>
<td style="text-align:left;">
NA
</td>
<td style="text-align:left;">
NA
</td>
<td style="text-align:left;">
unclassified gene
</td>
<td style="text-align:left;">
RIKEN cDNA 5730402E23 gene
</td>
</tr>
<tr>
<td style="text-align:left;">
6
</td>
<td style="text-align:left;">
pseudogene
</td>
<td style="text-align:right;">
39.82815
</td>
<td style="text-align:right;">
39.82881
</td>
<td style="text-align:left;">
negative
</td>
<td style="text-align:left;">
MGI_C57BL6J_3646122
</td>
<td style="text-align:left;">
Gm7554
</td>
<td style="text-align:left;">
NCBI_Gene:665247,ENSEMBL:ENSMUSG00000106936
</td>
<td style="text-align:left;">
MGI:3646122
</td>
<td style="text-align:left;">
pseudogene
</td>
<td style="text-align:left;">
predicted gene 7554
</td>
</tr>
<tr>
<td style="text-align:left;">
6
</td>
<td style="text-align:left;">
pseudogene
</td>
<td style="text-align:right;">
39.84302
</td>
<td style="text-align:right;">
39.84448
</td>
<td style="text-align:left;">
negative
</td>
<td style="text-align:left;">
MGI_C57BL6J_3645467
</td>
<td style="text-align:left;">
Gm7556
</td>
<td style="text-align:left;">
NCBI_Gene:665254,ENSEMBL:ENSMUSG00000106827
</td>
<td style="text-align:left;">
MGI:3645467
</td>
<td style="text-align:left;">
pseudogene
</td>
<td style="text-align:left;">
predicted gene 7556
</td>
</tr>
<tr>
<td style="text-align:left;">
6
</td>
<td style="text-align:left;">
gene
</td>
<td style="text-align:right;">
39.87068
</td>
<td style="text-align:right;">
40.25515
</td>
<td style="text-align:left;">
positive
</td>
<td style="text-align:left;">
MGI_C57BL6J_3647581
</td>
<td style="text-align:left;">
Tmem178b
</td>
<td style="text-align:left;">
NCBI_Gene:434008,ENSEMBL:ENSMUSG00000057716
</td>
<td style="text-align:left;">
MGI:3647581
</td>
<td style="text-align:left;">
protein coding gene
</td>
<td style="text-align:left;">
transmembrane protein 178B
</td>
</tr>
<tr>
<td style="text-align:left;">
6
</td>
<td style="text-align:left;">
gene
</td>
<td style="text-align:right;">
39.87153
</td>
<td style="text-align:right;">
39.87197
</td>
<td style="text-align:left;">
negative
</td>
<td style="text-align:left;">
MGI_C57BL6J_5477327
</td>
<td style="text-align:left;">
Gm26833
</td>
<td style="text-align:left;">
ENSEMBL:ENSMUSG00000097406
</td>
<td style="text-align:left;">
MGI:5477327
</td>
<td style="text-align:left;">
lncRNA gene
</td>
<td style="text-align:left;">
predicted gene, 26833
</td>
</tr>
<tr>
<td style="text-align:left;">
6
</td>
<td style="text-align:left;">
gene
</td>
<td style="text-align:right;">
40.02689
</td>
<td style="text-align:right;">
40.02861
</td>
<td style="text-align:left;">
positive
</td>
<td style="text-align:left;">
MGI_C57BL6J_5611223
</td>
<td style="text-align:left;">
Gm37995
</td>
<td style="text-align:left;">
ENSEMBL:ENSMUSG00000103331
</td>
<td style="text-align:left;">
MGI:5611223
</td>
<td style="text-align:left;">
unclassified gene
</td>
<td style="text-align:left;">
predicted gene, 37995
</td>
</tr>
<tr>
<td style="text-align:left;">
6
</td>
<td style="text-align:left;">
gene
</td>
<td style="text-align:right;">
40.09872
</td>
<td style="text-align:right;">
40.10117
</td>
<td style="text-align:left;">
positive
</td>
<td style="text-align:left;">
MGI_C57BL6J_5610901
</td>
<td style="text-align:left;">
Gm37673
</td>
<td style="text-align:left;">
ENSEMBL:ENSMUSG00000102623
</td>
<td style="text-align:left;">
MGI:5610901
</td>
<td style="text-align:left;">
unclassified gene
</td>
<td style="text-align:left;">
predicted gene, 37673
</td>
</tr>
<tr>
<td style="text-align:left;">
6
</td>
<td style="text-align:left;">
gene
</td>
<td style="text-align:right;">
40.11878
</td>
<td style="text-align:right;">
40.12948
</td>
<td style="text-align:left;">
positive
</td>
<td style="text-align:left;">
MGI_C57BL6J_5477132
</td>
<td style="text-align:left;">
Gm26638
</td>
<td style="text-align:left;">
ENSEMBL:ENSMUSG00000097563
</td>
<td style="text-align:left;">
MGI:5477132
</td>
<td style="text-align:left;">
lncRNA gene
</td>
<td style="text-align:left;">
predicted gene, 26638
</td>
</tr>
<tr>
<td style="text-align:left;">
6
</td>
<td style="text-align:left;">
gene
</td>
<td style="text-align:right;">
40.20330
</td>
<td style="text-align:right;">
40.20926
</td>
<td style="text-align:left;">
negative
</td>
<td style="text-align:left;">
MGI_C57BL6J_5477301
</td>
<td style="text-align:left;">
Gm26807
</td>
<td style="text-align:left;">
ENSEMBL:ENSMUSG00000097484
</td>
<td style="text-align:left;">
MGI:5477301
</td>
<td style="text-align:left;">
lncRNA gene
</td>
<td style="text-align:left;">
predicted gene, 26807
</td>
</tr>
<tr>
<td style="text-align:left;">
6
</td>
<td style="text-align:left;">
gene
</td>
<td style="text-align:right;">
40.22622
</td>
<td style="text-align:right;">
40.22904
</td>
<td style="text-align:left;">
positive
</td>
<td style="text-align:left;">
MGI_C57BL6J_5610231
</td>
<td style="text-align:left;">
Gm37003
</td>
<td style="text-align:left;">
ENSEMBL:ENSMUSG00000102287
</td>
<td style="text-align:left;">
MGI:5610231
</td>
<td style="text-align:left;">
unclassified gene
</td>
<td style="text-align:left;">
predicted gene, 37003
</td>
</tr>
<tr>
<td style="text-align:left;">
6
</td>
<td style="text-align:left;">
gene
</td>
<td style="text-align:right;">
40.32516
</td>
<td style="text-align:right;">
40.39756
</td>
<td style="text-align:left;">
positive
</td>
<td style="text-align:left;">
MGI_C57BL6J_1917173
</td>
<td style="text-align:left;">
Agk
</td>
<td style="text-align:left;">
NCBI_Gene:69923,ENSEMBL:ENSMUSG00000029916
</td>
<td style="text-align:left;">
MGI:1917173
</td>
<td style="text-align:left;">
protein coding gene
</td>
<td style="text-align:left;">
acylglycerol kinase
</td>
</tr>
<tr>
<td style="text-align:left;">
6
</td>
<td style="text-align:left;">
gene
</td>
<td style="text-align:right;">
40.40138
</td>
<td style="text-align:right;">
40.43649
</td>
<td style="text-align:left;">
negative
</td>
<td style="text-align:left;">
MGI_C57BL6J_2444256
</td>
<td style="text-align:left;">
E330009J07Rik
</td>
<td style="text-align:left;">
NCBI_Gene:243780,ENSEMBL:ENSMUSG00000037172
</td>
<td style="text-align:left;">
MGI:2444256
</td>
<td style="text-align:left;">
protein coding gene
</td>
<td style="text-align:left;">
RIKEN cDNA E330009J07 gene
</td>
</tr>
<tr>
<td style="text-align:left;">
6
</td>
<td style="text-align:left;">
gene
</td>
<td style="text-align:right;">
40.43540
</td>
<td style="text-align:right;">
40.46681
</td>
<td style="text-align:left;">
positive
</td>
<td style="text-align:left;">
MGI_C57BL6J_3027899
</td>
<td style="text-align:left;">
Wee2
</td>
<td style="text-align:left;">
NCBI_Gene:381759,ENSEMBL:ENSMUSG00000037159
</td>
<td style="text-align:left;">
MGI:3027899
</td>
<td style="text-align:left;">
protein coding gene
</td>
<td style="text-align:left;">
WEE1 homolog 2 (S. pombe)
</td>
</tr>
<tr>
<td style="text-align:left;">
6
</td>
<td style="text-align:left;">
gene
</td>
<td style="text-align:right;">
40.47054
</td>
<td style="text-align:right;">
40.47128
</td>
<td style="text-align:left;">
negative
</td>
<td style="text-align:left;">
MGI_C57BL6J_1925080
</td>
<td style="text-align:left;">
A930033M24Rik
</td>
<td style="text-align:left;">
NA
</td>
<td style="text-align:left;">
NA
</td>
<td style="text-align:left;">
unclassified gene
</td>
<td style="text-align:left;">
RIKEN cDNA A930033M24 gene
</td>
</tr>
<tr>
<td style="text-align:left;">
6
</td>
<td style="text-align:left;">
gene
</td>
<td style="text-align:right;">
40.47135
</td>
<td style="text-align:right;">
40.48470
</td>
<td style="text-align:left;">
positive
</td>
<td style="text-align:left;">
MGI_C57BL6J_1920040
</td>
<td style="text-align:left;">
Ssbp1
</td>
<td style="text-align:left;">
NCBI_Gene:381760,ENSEMBL:ENSMUSG00000029911
</td>
<td style="text-align:left;">
MGI:1920040
</td>
<td style="text-align:left;">
protein coding gene
</td>
<td style="text-align:left;">
single-stranded DNA binding protein 1
</td>
</tr>
<tr>
<td style="text-align:left;">
6
</td>
<td style="text-align:left;">
gene
</td>
<td style="text-align:right;">
40.49123
</td>
<td style="text-align:right;">
40.49231
</td>
<td style="text-align:left;">
positive
</td>
<td style="text-align:left;">
MGI_C57BL6J_3606604
</td>
<td style="text-align:left;">
Tas2r137
</td>
<td style="text-align:left;">
NCBI_Gene:574417,ENSEMBL:ENSMUSG00000052850
</td>
<td style="text-align:left;">
MGI:3606604
</td>
<td style="text-align:left;">
protein coding gene
</td>
<td style="text-align:left;">
taste receptor, type 2, member 137
</td>
</tr>
<tr>
<td style="text-align:left;">
6
</td>
<td style="text-align:left;">
gene
</td>
<td style="text-align:right;">
40.49353
</td>
<td style="text-align:right;">
40.49454
</td>
<td style="text-align:left;">
positive
</td>
<td style="text-align:left;">
MGI_C57BL6J_2681210
</td>
<td style="text-align:left;">
Tas2r108
</td>
<td style="text-align:left;">
NCBI_Gene:57253,ENSEMBL:ENSMUSG00000037140
</td>
<td style="text-align:left;">
MGI:2681210
</td>
<td style="text-align:left;">
protein coding gene
</td>
<td style="text-align:left;">
taste receptor, type 2, member 108
</td>
</tr>
<tr>
<td style="text-align:left;">
6
</td>
<td style="text-align:left;">
gene
</td>
<td style="text-align:right;">
40.51482
</td>
<td style="text-align:right;">
40.51955
</td>
<td style="text-align:left;">
negative
</td>
<td style="text-align:left;">
MGI_C57BL6J_1914940
</td>
<td style="text-align:left;">
Prss37
</td>
<td style="text-align:left;">
NCBI_Gene:67690,ENSEMBL:ENSMUSG00000029909
</td>
<td style="text-align:left;">
MGI:1914940
</td>
<td style="text-align:left;">
protein coding gene
</td>
<td style="text-align:left;">
protease, serine 37
</td>
</tr>
<tr>
<td style="text-align:left;">
6
</td>
<td style="text-align:left;">
pseudogene
</td>
<td style="text-align:right;">
40.52280
</td>
<td style="text-align:right;">
40.52322
</td>
<td style="text-align:left;">
negative
</td>
<td style="text-align:left;">
MGI_C57BL6J_5663432
</td>
<td style="text-align:left;">
Gm43295
</td>
<td style="text-align:left;">
ENSEMBL:ENSMUSG00000107359
</td>
<td style="text-align:left;">
MGI:5663432
</td>
<td style="text-align:left;">
pseudogene
</td>
<td style="text-align:left;">
predicted gene 43295
</td>
</tr>
<tr>
<td style="text-align:left;">
6
</td>
<td style="text-align:left;">
gene
</td>
<td style="text-align:right;">
40.54217
</td>
<td style="text-align:right;">
40.55505
</td>
<td style="text-align:left;">
negative
</td>
<td style="text-align:left;">
MGI_C57BL6J_3030295
</td>
<td style="text-align:left;">
Olfr461
</td>
<td style="text-align:left;">
NCBI_Gene:258380,ENSEMBL:ENSMUSG00000068259
</td>
<td style="text-align:left;">
MGI:3030295
</td>
<td style="text-align:left;">
protein coding gene
</td>
<td style="text-align:left;">
olfactory receptor 461
</td>
</tr>
<tr>
<td style="text-align:left;">
6
</td>
<td style="text-align:left;">
gene
</td>
<td style="text-align:right;">
40.56915
</td>
<td style="text-align:right;">
40.57434
</td>
<td style="text-align:left;">
positive
</td>
<td style="text-align:left;">
MGI_C57BL6J_3030294
</td>
<td style="text-align:left;">
Olfr460
</td>
<td style="text-align:left;">
NCBI_Gene:258381,ENSEMBL:ENSMUSG00000045514
</td>
<td style="text-align:left;">
MGI:3030294
</td>
<td style="text-align:left;">
protein coding gene
</td>
<td style="text-align:left;">
olfactory receptor 460
</td>
</tr>
<tr>
<td style="text-align:left;">
6
</td>
<td style="text-align:left;">
gene
</td>
<td style="text-align:right;">
40.57489
</td>
<td style="text-align:right;">
40.58582
</td>
<td style="text-align:left;">
negative
</td>
<td style="text-align:left;">
MGI_C57BL6J_1345151
</td>
<td style="text-align:left;">
Clec5a
</td>
<td style="text-align:left;">
NCBI_Gene:23845,ENSEMBL:ENSMUSG00000029915
</td>
<td style="text-align:left;">
MGI:1345151
</td>
<td style="text-align:left;">
protein coding gene
</td>
<td style="text-align:left;">
C-type lectin domain family 5, member a
</td>
</tr>
<tr>
<td style="text-align:left;">
6
</td>
<td style="text-align:left;">
gene
</td>
<td style="text-align:right;">
40.61232
</td>
<td style="text-align:right;">
40.61331
</td>
<td style="text-align:left;">
negative
</td>
<td style="text-align:left;">
MGI_C57BL6J_2681306
</td>
<td style="text-align:left;">
Tas2r138
</td>
<td style="text-align:left;">
NCBI_Gene:387513,ENSEMBL:ENSMUSG00000058250
</td>
<td style="text-align:left;">
MGI:2681306
</td>
<td style="text-align:left;">
protein coding gene
</td>
<td style="text-align:left;">
taste receptor, type 2, member 138
</td>
</tr>
<tr>
<td style="text-align:left;">
6
</td>
<td style="text-align:left;">
gene
</td>
<td style="text-align:right;">
40.62883
</td>
<td style="text-align:right;">
40.76912
</td>
<td style="text-align:left;">
positive
</td>
<td style="text-align:left;">
MGI_C57BL6J_1203495
</td>
<td style="text-align:left;">
Mgam
</td>
<td style="text-align:left;">
NCBI_Gene:232714,ENSEMBL:ENSMUSG00000068587
</td>
<td style="text-align:left;">
MGI:1203495
</td>
<td style="text-align:left;">
protein coding gene
</td>
<td style="text-align:left;">
maltase-glucoamylase
</td>
</tr>
<tr>
<td style="text-align:left;">
6
</td>
<td style="text-align:left;">
pseudogene
</td>
<td style="text-align:right;">
40.77871
</td>
<td style="text-align:right;">
40.86816
</td>
<td style="text-align:left;">
positive
</td>
<td style="text-align:left;">
MGI_C57BL6J_3645497
</td>
<td style="text-align:left;">
Mgam2-ps
</td>
<td style="text-align:left;">
NCBI_Gene:638904,ENSEMBL:ENSMUSG00000102802
</td>
<td style="text-align:left;">
MGI:3645497
</td>
<td style="text-align:left;">
pseudogene
</td>
<td style="text-align:left;">
maltase-glucoamylase 2, pseudogene
</td>
</tr>
<tr>
<td style="text-align:left;">
6
</td>
<td style="text-align:left;">
gene
</td>
<td style="text-align:right;">
40.87879
</td>
<td style="text-align:right;">
40.88749
</td>
<td style="text-align:left;">
negative
</td>
<td style="text-align:left;">
MGI_C57BL6J_2388042
</td>
<td style="text-align:left;">
Moxd2
</td>
<td style="text-align:left;">
NCBI_Gene:194357,ENSEMBL:ENSMUSG00000029885
</td>
<td style="text-align:left;">
MGI:2388042
</td>
<td style="text-align:left;">
protein coding gene
</td>
<td style="text-align:left;">
monooxygenase, DBH-like 2
</td>
</tr>
<tr>
<td style="text-align:left;">
6
</td>
<td style="text-align:left;">
gene
</td>
<td style="text-align:right;">
40.88953
</td>
<td style="text-align:right;">
40.89087
</td>
<td style="text-align:left;">
negative
</td>
<td style="text-align:left;">
MGI_C57BL6J_5611504
</td>
<td style="text-align:left;">
Gm38276
</td>
<td style="text-align:left;">
ENSEMBL:ENSMUSG00000104084
</td>
<td style="text-align:left;">
MGI:5611504
</td>
<td style="text-align:left;">
unclassified gene
</td>
<td style="text-align:left;">
predicted gene, 38276
</td>
</tr>
<tr>
<td style="text-align:left;">
6
</td>
<td style="text-align:left;">
gene
</td>
<td style="text-align:right;">
40.89123
</td>
<td style="text-align:right;">
40.89189
</td>
<td style="text-align:left;">
positive
</td>
<td style="text-align:left;">
MGI_C57BL6J_98594
</td>
<td style="text-align:left;">
Trbv1
</td>
<td style="text-align:left;">
NCBI_Gene:621968,ENSEMBL:ENSMUSG00000076461
</td>
<td style="text-align:left;">
MGI:98594
</td>
<td style="text-align:left;">
gene segment
</td>
<td style="text-align:left;">
T cell receptor beta, variable 1
</td>
</tr>
<tr>
<td style="text-align:left;">
6
</td>
<td style="text-align:left;">
gene
</td>
<td style="text-align:right;">
40.89526
</td>
<td style="text-align:right;">
40.90039
</td>
<td style="text-align:left;">
negative
</td>
<td style="text-align:left;">
MGI_C57BL6J_3608323
</td>
<td style="text-align:left;">
Prss58
</td>
<td style="text-align:left;">
NCBI_Gene:232717,ENSEMBL:ENSMUSG00000051936
</td>
<td style="text-align:left;">
MGI:3608323
</td>
<td style="text-align:left;">
protein coding gene
</td>
<td style="text-align:left;">
protease, serine 58
</td>
</tr>
<tr>
<td style="text-align:left;">
6
</td>
<td style="text-align:left;">
gene
</td>
<td style="text-align:right;">
40.92044
</td>
<td style="text-align:right;">
40.94097
</td>
<td style="text-align:left;">
negative
</td>
<td style="text-align:left;">
MGI_C57BL6J_1920731
</td>
<td style="text-align:left;">
1700074P13Rik
</td>
<td style="text-align:left;">
NCBI_Gene:73481,ENSEMBL:ENSMUSG00000029883
</td>
<td style="text-align:left;">
MGI:1920731
</td>
<td style="text-align:left;">
protein coding gene
</td>
<td style="text-align:left;">
RIKEN cDNA 1700074P13 gene
</td>
</tr>
<tr>
<td style="text-align:left;">
6
</td>
<td style="text-align:left;">
gene
</td>
<td style="text-align:right;">
40.94837
</td>
<td style="text-align:right;">
40.95435
</td>
<td style="text-align:left;">
negative
</td>
<td style="text-align:left;">
MGI_C57BL6J_3643181
</td>
<td style="text-align:left;">
Gm4744
</td>
<td style="text-align:left;">
NCBI_Gene:194360,ENSEMBL:ENSMUSG00000063252
</td>
<td style="text-align:left;">
MGI:3643181
</td>
<td style="text-align:left;">
protein coding gene
</td>
<td style="text-align:left;">
predicted gene 4744
</td>
</tr>
<tr>
<td style="text-align:left;">
6
</td>
<td style="text-align:left;">
gene
</td>
<td style="text-align:right;">
40.95022
</td>
<td style="text-align:right;">
40.95052
</td>
<td style="text-align:left;">
positive
</td>
<td style="text-align:left;">
MGI_C57BL6J_5580294
</td>
<td style="text-align:left;">
Gm29588
</td>
<td style="text-align:left;">
ENSEMBL:ENSMUSG00000101629
</td>
<td style="text-align:left;">
MGI:5580294
</td>
<td style="text-align:left;">
lncRNA gene
</td>
<td style="text-align:left;">
predicted gene 29588
</td>
</tr>
<tr>
<td style="text-align:left;">
6
</td>
<td style="text-align:left;">
gene
</td>
<td style="text-align:right;">
40.96475
</td>
<td style="text-align:right;">
40.96843
</td>
<td style="text-align:left;">
positive
</td>
<td style="text-align:left;">
MGI_C57BL6J_1920876
</td>
<td style="text-align:left;">
1810009J06Rik
</td>
<td style="text-align:left;">
NCBI_Gene:73626,ENSEMBL:ENSMUSG00000094808
</td>
<td style="text-align:left;">
MGI:1920876
</td>
<td style="text-align:left;">
protein coding gene
</td>
<td style="text-align:left;">
RIKEN cDNA 1810009J06 gene
</td>
</tr>
<tr>
<td style="text-align:left;">
6
</td>
<td style="text-align:left;">
gene
</td>
<td style="text-align:right;">
40.99582
</td>
<td style="text-align:right;">
40.99948
</td>
<td style="text-align:left;">
negative
</td>
<td style="text-align:left;">
MGI_C57BL6J_3780832
</td>
<td style="text-align:left;">
Gm2663
</td>
<td style="text-align:left;">
NCBI_Gene:100040208,ENSEMBL:ENSMUSG00000096525
</td>
<td style="text-align:left;">
MGI:3780832
</td>
<td style="text-align:left;">
protein coding gene
</td>
<td style="text-align:left;">
predicted gene 2663
</td>
</tr>
<tr>
<td style="text-align:left;">
6
</td>
<td style="text-align:left;">
pseudogene
</td>
<td style="text-align:right;">
41.01118
</td>
<td style="text-align:right;">
41.01137
</td>
<td style="text-align:left;">
negative
</td>
<td style="text-align:left;">
MGI_C57BL6J_5611243
</td>
<td style="text-align:left;">
Gm38015
</td>
<td style="text-align:left;">
ENSEMBL:ENSMUSG00000104408
</td>
<td style="text-align:left;">
MGI:5611243
</td>
<td style="text-align:left;">
pseudogene
</td>
<td style="text-align:left;">
predicted gene, 38015
</td>
</tr>
<tr>
<td style="text-align:left;">
6
</td>
<td style="text-align:left;">
pseudogene
</td>
<td style="text-align:right;">
41.02124
</td>
<td style="text-align:right;">
41.02188
</td>
<td style="text-align:left;">
negative
</td>
<td style="text-align:left;">
MGI_C57BL6J_5611106
</td>
<td style="text-align:left;">
Gm37878
</td>
<td style="text-align:left;">
ENSEMBL:ENSMUSG00000104105
</td>
<td style="text-align:left;">
MGI:5611106
</td>
<td style="text-align:left;">
pseudogene
</td>
<td style="text-align:left;">
predicted gene, 37878
</td>
</tr>
<tr>
<td style="text-align:left;">
6
</td>
<td style="text-align:left;">
pseudogene
</td>
<td style="text-align:right;">
41.02627
</td>
<td style="text-align:right;">
41.02665
</td>
<td style="text-align:left;">
positive
</td>
<td style="text-align:left;">
MGI_C57BL6J_5611522
</td>
<td style="text-align:left;">
Gm38294
</td>
<td style="text-align:left;">
ENSEMBL:ENSMUSG00000103157
</td>
<td style="text-align:left;">
MGI:5611522
</td>
<td style="text-align:left;">
pseudogene
</td>
<td style="text-align:left;">
predicted gene, 38294
</td>
</tr>
<tr>
<td style="text-align:left;">
6
</td>
<td style="text-align:left;">
gene
</td>
<td style="text-align:right;">
41.03027
</td>
<td style="text-align:right;">
41.03551
</td>
<td style="text-align:left;">
negative
</td>
<td style="text-align:left;">
MGI_C57BL6J_1914623
</td>
<td style="text-align:left;">
2210010C04Rik
</td>
<td style="text-align:left;">
NCBI_Gene:67373,ENSEMBL:ENSMUSG00000029882
</td>
<td style="text-align:left;">
MGI:1914623
</td>
<td style="text-align:left;">
protein coding gene
</td>
<td style="text-align:left;">
RIKEN cDNA 2210010C04 gene
</td>
</tr>
<tr>
<td style="text-align:left;">
6
</td>
<td style="text-align:left;">
gene
</td>
<td style="text-align:right;">
41.04734
</td>
<td style="text-align:right;">
41.04800
</td>
<td style="text-align:left;">
positive
</td>
<td style="text-align:left;">
MGI_C57BL6J_98599
</td>
<td style="text-align:left;">
Trbv2
</td>
<td style="text-align:left;">
NCBI_Gene:100124659,ENSEMBL:ENSMUSG00000076462
</td>
<td style="text-align:left;">
MGI:98599
</td>
<td style="text-align:left;">
gene segment
</td>
<td style="text-align:left;">
T cell receptor beta, variable 2
</td>
</tr>
<tr>
<td style="text-align:left;">
6
</td>
<td style="text-align:left;">
gene
</td>
<td style="text-align:right;">
41.04829
</td>
<td style="text-align:right;">
41.04883
</td>
<td style="text-align:left;">
positive
</td>
<td style="text-align:left;">
MGI_C57BL6J_98590
</td>
<td style="text-align:left;">
Trbv3
</td>
<td style="text-align:left;">
NCBI_Gene:100039790,ENSEMBL:ENSMUSG00000076463
</td>
<td style="text-align:left;">
MGI:98590
</td>
<td style="text-align:left;">
gene segment
</td>
<td style="text-align:left;">
T cell receptor beta, variable 3
</td>
</tr>
<tr>
<td style="text-align:left;">
6
</td>
<td style="text-align:left;">
gene
</td>
<td style="text-align:right;">
41.05588
</td>
<td style="text-align:right;">
41.06310
</td>
<td style="text-align:left;">
positive
</td>
<td style="text-align:left;">
MGI_C57BL6J_1923265
</td>
<td style="text-align:left;">
5830405F06Rik
</td>
<td style="text-align:left;">
ENSEMBL:ENSMUSG00000103586
</td>
<td style="text-align:left;">
MGI:1923265
</td>
<td style="text-align:left;">
lncRNA gene
</td>
<td style="text-align:left;">
RIKEN cDNA 5830405F06 gene
</td>
</tr>
<tr>
<td style="text-align:left;">
6
</td>
<td style="text-align:left;">
gene
</td>
<td style="text-align:right;">
41.05939
</td>
<td style="text-align:right;">
41.05989
</td>
<td style="text-align:left;">
positive
</td>
<td style="text-align:left;">
MGI_C57BL6J_98584
</td>
<td style="text-align:left;">
Trbv4
</td>
<td style="text-align:left;">
NCBI_Gene:100124660,ENSEMBL:ENSMUSG00000107486
</td>
<td style="text-align:left;">
MGI:98584
</td>
<td style="text-align:left;">
gene segment
</td>
<td style="text-align:left;">
T cell receptor beta, variable 10
</td>
</tr>
<tr>
<td style="text-align:left;">
6
</td>
<td style="text-align:left;">
gene
</td>
<td style="text-align:right;">
41.06233
</td>
<td style="text-align:right;">
41.06280
</td>
<td style="text-align:left;">
positive
</td>
<td style="text-align:left;">
MGI_C57BL6J_98583
</td>
<td style="text-align:left;">
Trbv5
</td>
<td style="text-align:left;">
NCBI_Gene:100124661,ENSEMBL:ENSMUSG00000076465
</td>
<td style="text-align:left;">
MGI:98583
</td>
<td style="text-align:left;">
gene segment
</td>
<td style="text-align:left;">
T cell receptor beta, variable 5
</td>
</tr>
<tr>
<td style="text-align:left;">
6
</td>
<td style="text-align:left;">
pseudogene
</td>
<td style="text-align:right;">
41.06687
</td>
<td style="text-align:right;">
41.06715
</td>
<td style="text-align:left;">
positive
</td>
<td style="text-align:left;">
MGI_C57BL6J_5009962
</td>
<td style="text-align:left;">
Trbv6
</td>
<td style="text-align:left;">
NCBI_Gene:100124662,ENSEMBL:ENSMUSG00000103807
</td>
<td style="text-align:left;">
MGI:5009962
</td>
<td style="text-align:left;">
pseudogenic gene segment
</td>
<td style="text-align:left;">
T cell receptor beta variable 6
</td>
</tr>
<tr>
<td style="text-align:left;">
6
</td>
<td style="text-align:left;">
pseudogene
</td>
<td style="text-align:right;">
41.08025
</td>
<td style="text-align:right;">
41.08065
</td>
<td style="text-align:left;">
positive
</td>
<td style="text-align:left;">
MGI_C57BL6J_5009963
</td>
<td style="text-align:left;">
Trbv7
</td>
<td style="text-align:left;">
NCBI_Gene:100124663,ENSEMBL:ENSMUSG00000102622
</td>
<td style="text-align:left;">
MGI:5009963
</td>
<td style="text-align:left;">
pseudogenic gene segment
</td>
<td style="text-align:left;">
T cell receptor beta variable 7
</td>
</tr>
<tr>
<td style="text-align:left;">
6
</td>
<td style="text-align:left;">
gene
</td>
<td style="text-align:right;">
41.08159
</td>
<td style="text-align:right;">
41.09198
</td>
<td style="text-align:left;">
positive
</td>
<td style="text-align:left;">
MGI_C57BL6J_6270568
</td>
<td style="text-align:left;">
Gm49885
</td>
<td style="text-align:left;">
ENSEMBL:ENSMUSG00000103833
</td>
<td style="text-align:left;">
MGI:6270568
</td>
<td style="text-align:left;">
lncRNA gene
</td>
<td style="text-align:left;">
predicted gene, 49885
</td>
</tr>
<tr>
<td style="text-align:left;">
6
</td>
<td style="text-align:left;">
pseudogene
</td>
<td style="text-align:right;">
41.08467
</td>
<td style="text-align:right;">
41.08511
</td>
<td style="text-align:left;">
positive
</td>
<td style="text-align:left;">
MGI_C57BL6J_5009964
</td>
<td style="text-align:left;">
Trbv8
</td>
<td style="text-align:left;">
NCBI_Gene:100124664,ENSEMBL:ENSMUSG00000102821
</td>
<td style="text-align:left;">
MGI:5009964
</td>
<td style="text-align:left;">
pseudogenic gene segment
</td>
<td style="text-align:left;">
T cell receptor beta variable 8
</td>
</tr>
<tr>
<td style="text-align:left;">
6
</td>
<td style="text-align:left;">
pseudogene
</td>
<td style="text-align:right;">
41.08799
</td>
<td style="text-align:right;">
41.08829
</td>
<td style="text-align:left;">
positive
</td>
<td style="text-align:left;">
MGI_C57BL6J_5009965
</td>
<td style="text-align:left;">
Trbv9
</td>
<td style="text-align:left;">
NCBI_Gene:100124665,ENSEMBL:ENSMUSG00000104198
</td>
<td style="text-align:left;">
MGI:5009965
</td>
<td style="text-align:left;">
pseudogenic gene segment
</td>
<td style="text-align:left;">
T cell receptor beta variable 9
</td>
</tr>
<tr>
<td style="text-align:left;">
6
</td>
<td style="text-align:left;">
pseudogene
</td>
<td style="text-align:right;">
41.09193
</td>
<td style="text-align:right;">
41.09223
</td>
<td style="text-align:left;">
positive
</td>
<td style="text-align:left;">
MGI_C57BL6J_4937160
</td>
<td style="text-align:left;">
Trbv10
</td>
<td style="text-align:left;">
NCBI_Gene:100124670,ENSEMBL:ENSMUSG00000091671
</td>
<td style="text-align:left;">
MGI:4937160
</td>
<td style="text-align:left;">
pseudogenic gene segment
</td>
<td style="text-align:left;">
T cell receptor beta variable 10
</td>
</tr>
<tr>
<td style="text-align:left;">
6
</td>
<td style="text-align:left;">
pseudogene
</td>
<td style="text-align:right;">
41.10681
</td>
<td style="text-align:right;">
41.10730
</td>
<td style="text-align:left;">
positive
</td>
<td style="text-align:left;">
MGI_C57BL6J_5009966
</td>
<td style="text-align:left;">
Trbv11
</td>
<td style="text-align:left;">
NCBI_Gene:100124671,ENSEMBL:ENSMUSG00000102903
</td>
<td style="text-align:left;">
MGI:5009966
</td>
<td style="text-align:left;">
pseudogenic gene segment
</td>
<td style="text-align:left;">
T cell receptor beta variable 11
</td>
</tr>
<tr>
<td style="text-align:left;">
6
</td>
<td style="text-align:left;">
gene
</td>
<td style="text-align:right;">
41.11355
</td>
<td style="text-align:right;">
41.11407
</td>
<td style="text-align:left;">
positive
</td>
<td style="text-align:left;">
MGI_C57BL6J_98602
</td>
<td style="text-align:left;">
Trbv12-1
</td>
<td style="text-align:left;">
NCBI_Gene:100124672,ENSEMBL:ENSMUSG00000095574
</td>
<td style="text-align:left;">
MGI:98602
</td>
<td style="text-align:left;">
gene segment
</td>
<td style="text-align:left;">
T cell receptor beta, variable 12-1
</td>
</tr>
<tr>
<td style="text-align:left;">
6
</td>
<td style="text-align:left;">
gene
</td>
<td style="text-align:right;">
41.11601
</td>
<td style="text-align:right;">
41.11647
</td>
<td style="text-align:left;">
positive
</td>
<td style="text-align:left;">
MGI_C57BL6J_98609
</td>
<td style="text-align:left;">
Trbv13-1
</td>
<td style="text-align:left;">
NCBI_Gene:100124673,ENSEMBL:ENSMUSG00000076467
</td>
<td style="text-align:left;">
MGI:98609
</td>
<td style="text-align:left;">
gene segment
</td>
<td style="text-align:left;">
T cell receptor beta, variable 13-1
</td>
</tr>
<tr>
<td style="text-align:left;">
6
</td>
<td style="text-align:left;">
gene
</td>
<td style="text-align:right;">
41.11885
</td>
<td style="text-align:right;">
41.11936
</td>
<td style="text-align:left;">
positive
</td>
<td style="text-align:left;">
MGI_C57BL6J_98601
</td>
<td style="text-align:left;">
Trbv12-2
</td>
<td style="text-align:left;">
NCBI_Gene:100124674,ENSEMBL:ENSMUSG00000094525
</td>
<td style="text-align:left;">
MGI:98601
</td>
<td style="text-align:left;">
gene segment
</td>
<td style="text-align:left;">
T cell receptor beta, variable 12-2
</td>
</tr>
<tr>
<td style="text-align:left;">
6
</td>
<td style="text-align:left;">
gene
</td>
<td style="text-align:right;">
41.12140
</td>
<td style="text-align:right;">
41.12183
</td>
<td style="text-align:left;">
positive
</td>
<td style="text-align:left;">
MGI_C57BL6J_98608
</td>
<td style="text-align:left;">
Trbv13-2
</td>
<td style="text-align:left;">
NCBI_Gene:100124675,ENSEMBL:ENSMUSG00000076469
</td>
<td style="text-align:left;">
MGI:98608
</td>
<td style="text-align:left;">
gene segment
</td>
<td style="text-align:left;">
T cell receptor beta, variable 13-2
</td>
</tr>
<tr>
<td style="text-align:left;">
6
</td>
<td style="text-align:left;">
pseudogene
</td>
<td style="text-align:right;">
41.12759
</td>
<td style="text-align:right;">
41.12808
</td>
<td style="text-align:left;">
positive
</td>
<td style="text-align:left;">
MGI_C57BL6J_98603
</td>
<td style="text-align:left;">
Trbv12-3
</td>
<td style="text-align:left;">
NCBI_Gene:100124676,ENSEMBL:ENSMUSG00000102327
</td>
<td style="text-align:left;">
MGI:98603
</td>
<td style="text-align:left;">
pseudogenic gene segment
</td>
<td style="text-align:left;">
T cell receptor beta, variable 12-3
</td>
</tr>
<tr>
<td style="text-align:left;">
6
</td>
<td style="text-align:left;">
gene
</td>
<td style="text-align:right;">
41.13015
</td>
<td style="text-align:right;">
41.13059
</td>
<td style="text-align:left;">
positive
</td>
<td style="text-align:left;">
MGI_C57BL6J_98607
</td>
<td style="text-align:left;">
Trbv13-3
</td>
<td style="text-align:left;">
NCBI_Gene:100124677,ENSEMBL:ENSMUSG00000076470
</td>
<td style="text-align:left;">
MGI:98607
</td>
<td style="text-align:left;">
gene segment
</td>
<td style="text-align:left;">
T cell receptor beta, variable 13-3
</td>
</tr>
<tr>
<td style="text-align:left;">
6
</td>
<td style="text-align:left;">
gene
</td>
<td style="text-align:right;">
41.13515
</td>
<td style="text-align:right;">
41.13562
</td>
<td style="text-align:left;">
positive
</td>
<td style="text-align:left;">
MGI_C57BL6J_98587
</td>
<td style="text-align:left;">
Trbv14
</td>
<td style="text-align:left;">
NCBI_Gene:100124678,ENSEMBL:ENSMUSG00000076471
</td>
<td style="text-align:left;">
MGI:98587
</td>
<td style="text-align:left;">
gene segment
</td>
<td style="text-align:left;">
T cell receptor beta, variable 14
</td>
</tr>
<tr>
<td style="text-align:left;">
6
</td>
<td style="text-align:left;">
gene
</td>
<td style="text-align:right;">
41.14119
</td>
<td style="text-align:right;">
41.14166
</td>
<td style="text-align:left;">
positive
</td>
<td style="text-align:left;">
MGI_C57BL6J_98586
</td>
<td style="text-align:left;">
Trbv15
</td>
<td style="text-align:left;">
NCBI_Gene:100124679,ENSEMBL:ENSMUSG00000076472
</td>
<td style="text-align:left;">
MGI:98586
</td>
<td style="text-align:left;">
gene segment
</td>
<td style="text-align:left;">
T cell receptor beta, variable 15
</td>
</tr>
<tr>
<td style="text-align:left;">
6
</td>
<td style="text-align:left;">
gene
</td>
<td style="text-align:right;">
41.15174
</td>
<td style="text-align:right;">
41.15223
</td>
<td style="text-align:left;">
positive
</td>
<td style="text-align:left;">
MGI_C57BL6J_98585
</td>
<td style="text-align:left;">
Trbv16
</td>
<td style="text-align:left;">
NCBI_Gene:100124680,ENSEMBL:ENSMUSG00000076473
</td>
<td style="text-align:left;">
MGI:98585
</td>
<td style="text-align:left;">
gene segment
</td>
<td style="text-align:left;">
T cell receptor beta, variable 16
</td>
</tr>
<tr>
<td style="text-align:left;">
6
</td>
<td style="text-align:left;">
gene
</td>
<td style="text-align:right;">
41.16306
</td>
<td style="text-align:right;">
41.16356
</td>
<td style="text-align:left;">
positive
</td>
<td style="text-align:left;">
MGI_C57BL6J_98610
</td>
<td style="text-align:left;">
Trbv17
</td>
<td style="text-align:left;">
NCBI_Gene:100124681,ENSEMBL:ENSMUSG00000076474
</td>
<td style="text-align:left;">
MGI:98610
</td>
<td style="text-align:left;">
gene segment
</td>
<td style="text-align:left;">
T cell receptor beta, variable 17
</td>
</tr>
<tr>
<td style="text-align:left;">
6
</td>
<td style="text-align:left;">
pseudogene
</td>
<td style="text-align:right;">
41.17514
</td>
<td style="text-align:right;">
41.17561
</td>
<td style="text-align:left;">
positive
</td>
<td style="text-align:left;">
MGI_C57BL6J_5009967
</td>
<td style="text-align:left;">
Trbv18
</td>
<td style="text-align:left;">
NCBI_Gene:100124682,ENSEMBL:ENSMUSG00000102409
</td>
<td style="text-align:left;">
MGI:5009967
</td>
<td style="text-align:left;">
pseudogenic gene segment
</td>
<td style="text-align:left;">
T cell receptor beta variable 18
</td>
</tr>
<tr>
<td style="text-align:left;">
6
</td>
<td style="text-align:left;">
gene
</td>
<td style="text-align:right;">
41.17838
</td>
<td style="text-align:right;">
41.17905
</td>
<td style="text-align:left;">
positive
</td>
<td style="text-align:left;">
MGI_C57BL6J_98604
</td>
<td style="text-align:left;">
Trbv19
</td>
<td style="text-align:left;">
NCBI_Gene:100124684,ENSEMBL:ENSMUSG00000076475
</td>
<td style="text-align:left;">
MGI:98604
</td>
<td style="text-align:left;">
gene segment
</td>
<td style="text-align:left;">
T cell receptor beta, variable 19
</td>
</tr>
<tr>
<td style="text-align:left;">
6
</td>
<td style="text-align:left;">
gene
</td>
<td style="text-align:right;">
41.18827
</td>
<td style="text-align:right;">
41.18898
</td>
<td style="text-align:left;">
positive
</td>
<td style="text-align:left;">
MGI_C57BL6J_98589
</td>
<td style="text-align:left;">
Trbv20
</td>
<td style="text-align:left;">
NCBI_Gene:100124685,ENSEMBL:ENSMUSG00000076476
</td>
<td style="text-align:left;">
MGI:98589
</td>
<td style="text-align:left;">
gene segment
</td>
<td style="text-align:left;">
T cell receptor beta, variable 20
</td>
</tr>
<tr>
<td style="text-align:left;">
6
</td>
<td style="text-align:left;">
gene
</td>
<td style="text-align:right;">
41.20255
</td>
<td style="text-align:right;">
41.20304
</td>
<td style="text-align:left;">
positive
</td>
<td style="text-align:left;">
MGI_C57BL6J_98593
</td>
<td style="text-align:left;">
Trbv21
</td>
<td style="text-align:left;">
NCBI_Gene:100124686,ENSEMBL:ENSMUSG00000076477
</td>
<td style="text-align:left;">
MGI:98593
</td>
<td style="text-align:left;">
gene segment
</td>
<td style="text-align:left;">
T cell receptor beta, variable 21
</td>
</tr>
<tr>
<td style="text-align:left;">
6
</td>
<td style="text-align:left;">
pseudogene
</td>
<td style="text-align:right;">
41.21207
</td>
<td style="text-align:right;">
41.21253
</td>
<td style="text-align:left;">
positive
</td>
<td style="text-align:left;">
MGI_C57BL6J_5009968
</td>
<td style="text-align:left;">
Trbv22
</td>
<td style="text-align:left;">
NCBI_Gene:100124687,ENSEMBL:ENSMUSG00000102988
</td>
<td style="text-align:left;">
MGI:5009968
</td>
<td style="text-align:left;">
pseudogenic gene segment
</td>
<td style="text-align:left;">
T cell receptor beta variable 22
</td>
</tr>
<tr>
<td style="text-align:left;">
6
</td>
<td style="text-align:left;">
gene
</td>
<td style="text-align:right;">
41.21607
</td>
<td style="text-align:right;">
41.21653
</td>
<td style="text-align:left;">
positive
</td>
<td style="text-align:left;">
MGI_C57BL6J_98595
</td>
<td style="text-align:left;">
Trbv23
</td>
<td style="text-align:left;">
NCBI_Gene:100124688,ENSEMBL:ENSMUSG00000076478
</td>
<td style="text-align:left;">
MGI:98595
</td>
<td style="text-align:left;">
gene segment
</td>
<td style="text-align:left;">
T cell receptor beta, variable V23
</td>
</tr>
<tr>
<td style="text-align:left;">
6
</td>
<td style="text-align:left;">
gene
</td>
<td style="text-align:right;">
41.21809
</td>
<td style="text-align:right;">
41.21856
</td>
<td style="text-align:left;">
positive
</td>
<td style="text-align:left;">
MGI_C57BL6J_98591
</td>
<td style="text-align:left;">
Trbv24
</td>
<td style="text-align:left;">
NCBI_Gene:100124689,ENSEMBL:ENSMUSG00000102955
</td>
<td style="text-align:left;">
MGI:98591
</td>
<td style="text-align:left;">
gene segment
</td>
<td style="text-align:left;">
T cell receptor beta, variable 24
</td>
</tr>
<tr>
<td style="text-align:left;">
6
</td>
<td style="text-align:left;">
pseudogene
</td>
<td style="text-align:right;">
41.22167
</td>
<td style="text-align:right;">
41.22221
</td>
<td style="text-align:left;">
positive
</td>
<td style="text-align:left;">
MGI_C57BL6J_5009969
</td>
<td style="text-align:left;">
Trbv25
</td>
<td style="text-align:left;">
NCBI_Gene:100124690,ENSEMBL:ENSMUSG00000104473
</td>
<td style="text-align:left;">
MGI:5009969
</td>
<td style="text-align:left;">
pseudogenic gene segment
</td>
<td style="text-align:left;">
T cell receptor beta variable 25
</td>
</tr>
<tr>
<td style="text-align:left;">
6
</td>
<td style="text-align:left;">
gene
</td>
<td style="text-align:right;">
41.22751
</td>
<td style="text-align:right;">
41.22799
</td>
<td style="text-align:left;">
positive
</td>
<td style="text-align:left;">
MGI_C57BL6J_98596
</td>
<td style="text-align:left;">
Trbv26
</td>
<td style="text-align:left;">
NCBI_Gene:110315,ENSEMBL:ENSMUSG00000076479
</td>
<td style="text-align:left;">
MGI:98596
</td>
<td style="text-align:left;">
gene segment
</td>
<td style="text-align:left;">
T cell receptor beta, variable 26
</td>
</tr>
<tr>
<td style="text-align:left;">
6
</td>
<td style="text-align:left;">
pseudogene
</td>
<td style="text-align:right;">
41.25323
</td>
<td style="text-align:right;">
41.25433
</td>
<td style="text-align:left;">
positive
</td>
<td style="text-align:left;">
MGI_C57BL6J_3780888
</td>
<td style="text-align:left;">
Gm2719
</td>
<td style="text-align:left;">
NCBI_Gene:100040333,ENSEMBL:ENSMUSG00000103042
</td>
<td style="text-align:left;">
MGI:3780888
</td>
<td style="text-align:left;">
pseudogene
</td>
<td style="text-align:left;">
predicted pseudogene 2719
</td>
</tr>
<tr>
<td style="text-align:left;">
6
</td>
<td style="text-align:left;">
pseudogene
</td>
<td style="text-align:right;">
41.25812
</td>
<td style="text-align:right;">
41.25858
</td>
<td style="text-align:left;">
positive
</td>
<td style="text-align:left;">
MGI_C57BL6J_5009970
</td>
<td style="text-align:left;">
Trbv27
</td>
<td style="text-align:left;">
NCBI_Gene:100124693,ENSEMBL:ENSMUSG00000104254
</td>
<td style="text-align:left;">
MGI:5009970
</td>
<td style="text-align:left;">
pseudogenic gene segment
</td>
<td style="text-align:left;">
T cell receptor beta variable 27
</td>
</tr>
<tr>
<td style="text-align:left;">
6
</td>
<td style="text-align:left;">
pseudogene
</td>
<td style="text-align:right;">
41.26670
</td>
<td style="text-align:right;">
41.26715
</td>
<td style="text-align:left;">
positive
</td>
<td style="text-align:left;">
MGI_C57BL6J_5009971
</td>
<td style="text-align:left;">
Trbv28
</td>
<td style="text-align:left;">
NCBI_Gene:100124694,ENSEMBL:ENSMUSG00000104326
</td>
<td style="text-align:left;">
MGI:5009971
</td>
<td style="text-align:left;">
pseudogenic gene segment
</td>
<td style="text-align:left;">
T cell receptor beta variable 28
</td>
</tr>
<tr>
<td style="text-align:left;">
6
</td>
<td style="text-align:left;">
pseudogene
</td>
<td style="text-align:right;">
41.27139
</td>
<td style="text-align:right;">
41.27188
</td>
<td style="text-align:left;">
positive
</td>
<td style="text-align:left;">
MGI_C57BL6J_98605
</td>
<td style="text-align:left;">
Trbv29
</td>
<td style="text-align:left;">
NCBI_Gene:100124698,ENSEMBL:ENSMUSG00000076480
</td>
<td style="text-align:left;">
MGI:98605
</td>
<td style="text-align:left;">
pseudogenic gene segment
</td>
<td style="text-align:left;">
T cell receptor beta, variable 29
</td>
</tr>
<tr>
<td style="text-align:left;">
6
</td>
<td style="text-align:left;">
gene
</td>
<td style="text-align:right;">
41.28138
</td>
<td style="text-align:right;">
41.28199
</td>
<td style="text-align:left;">
positive
</td>
<td style="text-align:left;">
MGI_C57BL6J_98592
</td>
<td style="text-align:left;">
Trbv30
</td>
<td style="text-align:left;">
NCBI_Gene:100124699,ENSEMBL:ENSMUSG00000076481
</td>
<td style="text-align:left;">
MGI:98592
</td>
<td style="text-align:left;">
gene segment
</td>
<td style="text-align:left;">
T cell receptor beta, variable 30
</td>
</tr>
<tr>
<td style="text-align:left;">
6
</td>
<td style="text-align:left;">
gene
</td>
<td style="text-align:right;">
41.30227
</td>
<td style="text-align:right;">
41.30553
</td>
<td style="text-align:left;">
positive
</td>
<td style="text-align:left;">
MGI_C57BL6J_102757
</td>
<td style="text-align:left;">
Try4
</td>
<td style="text-align:left;">
NCBI_Gene:22074,ENSEMBL:ENSMUSG00000054106
</td>
<td style="text-align:left;">
MGI:102757
</td>
<td style="text-align:left;">
protein coding gene
</td>
<td style="text-align:left;">
trypsin 4
</td>
</tr>
<tr>
<td style="text-align:left;">
6
</td>
<td style="text-align:left;">
pseudogene
</td>
<td style="text-align:right;">
41.30620
</td>
<td style="text-align:right;">
41.30676
</td>
<td style="text-align:left;">
negative
</td>
<td style="text-align:left;">
MGI_C57BL6J_5011149
</td>
<td style="text-align:left;">
Gm18964
</td>
<td style="text-align:left;">
NCBI_Gene:100418045,ENSEMBL:ENSMUSG00000103797
</td>
<td style="text-align:left;">
MGI:5011149
</td>
<td style="text-align:left;">
pseudogene
</td>
<td style="text-align:left;">
predicted gene, 18964
</td>
</tr>
<tr>
<td style="text-align:left;">
6
</td>
<td style="text-align:left;">
gene
</td>
<td style="text-align:right;">
41.31123
</td>
<td style="text-align:right;">
41.31684
</td>
<td style="text-align:left;">
negative
</td>
<td style="text-align:left;">
MGI_C57BL6J_102756
</td>
<td style="text-align:left;">
Try5
</td>
<td style="text-align:left;">
NCBI_Gene:103964,ENSEMBL:ENSMUSG00000036938
</td>
<td style="text-align:left;">
MGI:102756
</td>
<td style="text-align:left;">
protein coding gene
</td>
<td style="text-align:left;">
trypsin 5
</td>
</tr>
<tr>
<td style="text-align:left;">
6
</td>
<td style="text-align:left;">
gene
</td>
<td style="text-align:right;">
41.35410
</td>
<td style="text-align:right;">
41.35794
</td>
<td style="text-align:left;">
positive
</td>
<td style="text-align:left;">
MGI_C57BL6J_3687012
</td>
<td style="text-align:left;">
Try10
</td>
<td style="text-align:left;">
NCBI_Gene:436522,ENSEMBL:ENSMUSG00000071521
</td>
<td style="text-align:left;">
MGI:3687012
</td>
<td style="text-align:left;">
protein coding gene
</td>
<td style="text-align:left;">
trypsin 10
</td>
</tr>
<tr>
<td style="text-align:left;">
6
</td>
<td style="text-align:left;">
gene
</td>
<td style="text-align:right;">
41.37376
</td>
<td style="text-align:right;">
41.37768
</td>
<td style="text-align:left;">
negative
</td>
<td style="text-align:left;">
MGI_C57BL6J_102758
</td>
<td style="text-align:left;">
Prss3
</td>
<td style="text-align:left;">
NCBI_Gene:22073,ENSEMBL:ENSMUSG00000071519
</td>
<td style="text-align:left;">
MGI:102758
</td>
<td style="text-align:left;">
protein coding gene
</td>
<td style="text-align:left;">
protease, serine 3
</td>
</tr>
<tr>
<td style="text-align:left;">
6
</td>
<td style="text-align:left;">
gene
</td>
<td style="text-align:right;">
41.39236
</td>
<td style="text-align:right;">
41.39726
</td>
<td style="text-align:left;">
positive
</td>
<td style="text-align:left;">
MGI_C57BL6J_3646222
</td>
<td style="text-align:left;">
Gm5771
</td>
<td style="text-align:left;">
NCBI_Gene:436523,ENSEMBL:ENSMUSG00000058119
</td>
<td style="text-align:left;">
MGI:3646222
</td>
<td style="text-align:left;">
protein coding gene
</td>
<td style="text-align:left;">
predicted gene 5771
</td>
</tr>
<tr>
<td style="text-align:left;">
6
</td>
<td style="text-align:left;">
pseudogene
</td>
<td style="text-align:right;">
41.40508
</td>
<td style="text-align:right;">
41.41963
</td>
<td style="text-align:left;">
positive
</td>
<td style="text-align:left;">
MGI_C57BL6J_3643903
</td>
<td style="text-align:left;">
Gm5409
</td>
<td style="text-align:left;">
NCBI_Gene:386551,ENSEMBL:ENSMUSG00000059014
</td>
<td style="text-align:left;">
MGI:3643903
</td>
<td style="text-align:left;">
pseudogene
</td>
<td style="text-align:left;">
predicted pseudogene 5409
</td>
</tr>
<tr>
<td style="text-align:left;">
6
</td>
<td style="text-align:left;">
pseudogene
</td>
<td style="text-align:right;">
41.40691
</td>
<td style="text-align:right;">
41.40727
</td>
<td style="text-align:left;">
positive
</td>
<td style="text-align:left;">
MGI_C57BL6J_5611534
</td>
<td style="text-align:left;">
Gm38306
</td>
<td style="text-align:left;">
ENSEMBL:ENSMUSG00000103397
</td>
<td style="text-align:left;">
MGI:5611534
</td>
<td style="text-align:left;">
pseudogene
</td>
<td style="text-align:left;">
predicted gene, 38306
</td>
</tr>
<tr>
<td style="text-align:left;">
6
</td>
<td style="text-align:left;">
gene
</td>
<td style="text-align:right;">
41.44221
</td>
<td style="text-align:right;">
41.44615
</td>
<td style="text-align:left;">
negative
</td>
<td style="text-align:left;">
MGI_C57BL6J_3641889
</td>
<td style="text-align:left;">
Gm10334
</td>
<td style="text-align:left;">
NCBI_Gene:100040233,ENSEMBL:ENSMUSG00000071517
</td>
<td style="text-align:left;">
MGI:3641889
</td>
<td style="text-align:left;">
protein coding gene
</td>
<td style="text-align:left;">
predicted gene 10334
</td>
</tr>
<tr>
<td style="text-align:left;">
6
</td>
<td style="text-align:left;">
gene
</td>
<td style="text-align:right;">
41.45893
</td>
<td style="text-align:right;">
41.46379
</td>
<td style="text-align:left;">
positive
</td>
<td style="text-align:left;">
MGI_C57BL6J_98839
</td>
<td style="text-align:left;">
Prss1
</td>
<td style="text-align:left;">
NCBI_Gene:114228,ENSEMBL:ENSMUSG00000062751
</td>
<td style="text-align:left;">
MGI:98839
</td>
<td style="text-align:left;">
protein coding gene
</td>
<td style="text-align:left;">
protease, serine 1 (trypsin 1)
</td>
</tr>
<tr>
<td style="text-align:left;">
6
</td>
<td style="text-align:left;">
pseudogene
</td>
<td style="text-align:right;">
41.48051
</td>
<td style="text-align:right;">
41.48172
</td>
<td style="text-align:left;">
positive
</td>
<td style="text-align:left;">
MGI_C57BL6J_5610529
</td>
<td style="text-align:left;">
Gm37301
</td>
<td style="text-align:left;">
ENSEMBL:ENSMUSG00000102221
</td>
<td style="text-align:left;">
MGI:5610529
</td>
<td style="text-align:left;">
pseudogene
</td>
<td style="text-align:left;">
predicted gene, 37301
</td>
</tr>
<tr>
<td style="text-align:left;">
6
</td>
<td style="text-align:left;">
pseudogene
</td>
<td style="text-align:right;">
41.49338
</td>
<td style="text-align:right;">
41.49515
</td>
<td style="text-align:left;">
negative
</td>
<td style="text-align:left;">
MGI_C57BL6J_5588972
</td>
<td style="text-align:left;">
Gm29813
</td>
<td style="text-align:left;">
NCBI_Gene:101954205,ENSEMBL:ENSMUSG00000104513
</td>
<td style="text-align:left;">
MGI:5588972
</td>
<td style="text-align:left;">
pseudogene
</td>
<td style="text-align:left;">
predicted gene, 29813
</td>
</tr>
<tr>
<td style="text-align:left;">
6
</td>
<td style="text-align:left;">
pseudogene
</td>
<td style="text-align:right;">
41.49693
</td>
<td style="text-align:right;">
41.49730
</td>
<td style="text-align:left;">
positive
</td>
<td style="text-align:left;">
MGI_C57BL6J_5611069
</td>
<td style="text-align:left;">
Gm37841
</td>
<td style="text-align:left;">
ENSEMBL:ENSMUSG00000102616
</td>
<td style="text-align:left;">
MGI:5611069
</td>
<td style="text-align:left;">
pseudogene
</td>
<td style="text-align:left;">
predicted gene, 37841
</td>
</tr>
<tr>
<td style="text-align:left;">
6
</td>
<td style="text-align:left;">
pseudogene
</td>
<td style="text-align:right;">
41.50412
</td>
<td style="text-align:right;">
41.50565
</td>
<td style="text-align:left;">
positive
</td>
<td style="text-align:left;">
MGI_C57BL6J_5611246
</td>
<td style="text-align:left;">
Gm38018
</td>
<td style="text-align:left;">
ENSEMBL:ENSMUSG00000103699
</td>
<td style="text-align:left;">
MGI:5611246
</td>
<td style="text-align:left;">
pseudogene
</td>
<td style="text-align:left;">
predicted gene, 38018
</td>
</tr>
<tr>
<td style="text-align:left;">
6
</td>
<td style="text-align:left;">
gene
</td>
<td style="text-align:right;">
41.52178
</td>
<td style="text-align:right;">
41.52508
</td>
<td style="text-align:left;">
positive
</td>
<td style="text-align:left;">
MGI_C57BL6J_102759
</td>
<td style="text-align:left;">
Prss2
</td>
<td style="text-align:left;">
NCBI_Gene:22072,ENSEMBL:ENSMUSG00000057163
</td>
<td style="text-align:left;">
MGI:102759
</td>
<td style="text-align:left;">
protein coding gene
</td>
<td style="text-align:left;">
protease, serine 2
</td>
</tr>
<tr>
<td style="text-align:left;">
6
</td>
<td style="text-align:left;">
gene
</td>
<td style="text-align:right;">
41.53320
</td>
<td style="text-align:right;">
41.53321
</td>
<td style="text-align:left;">
positive
</td>
<td style="text-align:left;">
MGI_C57BL6J_4439571
</td>
<td style="text-align:left;">
Trbd1
</td>
<td style="text-align:left;">
NCBI_Gene:100125246,ENSEMBL:ENSMUSG00000095668
</td>
<td style="text-align:left;">
MGI:4439571
</td>
<td style="text-align:left;">
gene segment
</td>
<td style="text-align:left;">
T cell receptor beta, D region 1
</td>
</tr>
<tr>
<td style="text-align:left;">
6
</td>
<td style="text-align:left;">
gene
</td>
<td style="text-align:right;">
41.53386
</td>
<td style="text-align:right;">
41.53391
</td>
<td style="text-align:left;">
positive
</td>
<td style="text-align:left;">
MGI_C57BL6J_4439570
</td>
<td style="text-align:left;">
Trbj1-1
</td>
<td style="text-align:left;">
NCBI_Gene:100125248,ENSEMBL:ENSMUSG00000076483
</td>
<td style="text-align:left;">
MGI:4439570
</td>
<td style="text-align:left;">
gene segment
</td>
<td style="text-align:left;">
T cell receptor beta joining 1-1
</td>
</tr>
<tr>
<td style="text-align:left;">
6
</td>
<td style="text-align:left;">
gene
</td>
<td style="text-align:right;">
41.53400
</td>
<td style="text-align:right;">
41.53405
</td>
<td style="text-align:left;">
positive
</td>
<td style="text-align:left;">
MGI_C57BL6J_4439569
</td>
<td style="text-align:left;">
Trbj1-2
</td>
<td style="text-align:left;">
NCBI_Gene:100125249,ENSEMBL:ENSMUSG00000076484
</td>
<td style="text-align:left;">
MGI:4439569
</td>
<td style="text-align:left;">
gene segment
</td>
<td style="text-align:left;">
T cell receptor beta joining 1-2
</td>
</tr>
<tr>
<td style="text-align:left;">
6
</td>
<td style="text-align:left;">
gene
</td>
<td style="text-align:right;">
41.53432
</td>
<td style="text-align:right;">
41.53438
</td>
<td style="text-align:left;">
positive
</td>
<td style="text-align:left;">
MGI_C57BL6J_4439568
</td>
<td style="text-align:left;">
Trbj1-3
</td>
<td style="text-align:left;">
NCBI_Gene:100125250,ENSEMBL:ENSMUSG00000076485
</td>
<td style="text-align:left;">
MGI:4439568
</td>
<td style="text-align:left;">
gene segment
</td>
<td style="text-align:left;">
T cell receptor beta joining 1-3
</td>
</tr>
<tr>
<td style="text-align:left;">
6
</td>
<td style="text-align:left;">
gene
</td>
<td style="text-align:right;">
41.53481
</td>
<td style="text-align:right;">
41.53486
</td>
<td style="text-align:left;">
positive
</td>
<td style="text-align:left;">
MGI_C57BL6J_4439567
</td>
<td style="text-align:left;">
Trbj1-4
</td>
<td style="text-align:left;">
NCBI_Gene:100125251,ENSEMBL:ENSMUSG00000076486
</td>
<td style="text-align:left;">
MGI:4439567
</td>
<td style="text-align:left;">
gene segment
</td>
<td style="text-align:left;">
T cell receptor beta joining 1-4
</td>
</tr>
<tr>
<td style="text-align:left;">
6
</td>
<td style="text-align:left;">
gene
</td>
<td style="text-align:right;">
41.53508
</td>
<td style="text-align:right;">
41.53513
</td>
<td style="text-align:left;">
positive
</td>
<td style="text-align:left;">
MGI_C57BL6J_4439566
</td>
<td style="text-align:left;">
Trbj1-5
</td>
<td style="text-align:left;">
NCBI_Gene:100125252,ENSEMBL:ENSMUSG00000076487
</td>
<td style="text-align:left;">
MGI:4439566
</td>
<td style="text-align:left;">
gene segment
</td>
<td style="text-align:left;">
T cell receptor beta joining 1-5
</td>
</tr>
<tr>
<td style="text-align:left;">
6
</td>
<td style="text-align:left;">
gene
</td>
<td style="text-align:right;">
41.53555
</td>
<td style="text-align:right;">
41.53561
</td>
<td style="text-align:left;">
positive
</td>
<td style="text-align:left;">
MGI_C57BL6J_4439575
</td>
<td style="text-align:left;">
Trbj1-6
</td>
<td style="text-align:left;">
NCBI_Gene:100125253,ENSEMBL:ENSMUSG00000076488
</td>
<td style="text-align:left;">
MGI:4439575
</td>
<td style="text-align:left;">
gene segment
</td>
<td style="text-align:left;">
T cell receptor beta joining 1-6
</td>
</tr>
<tr>
<td style="text-align:left;">
6
</td>
<td style="text-align:left;">
gene
</td>
<td style="text-align:right;">
41.53564
</td>
<td style="text-align:right;">
41.53569
</td>
<td style="text-align:left;">
positive
</td>
<td style="text-align:left;">
MGI_C57BL6J_4439574
</td>
<td style="text-align:left;">
Trbj1-7
</td>
<td style="text-align:left;">
NCBI_Gene:100125254,ENSEMBL:ENSMUSG00000076489
</td>
<td style="text-align:left;">
MGI:4439574
</td>
<td style="text-align:left;">
gene segment
</td>
<td style="text-align:left;">
T cell receptor beta joining 1-7
</td>
</tr>
<tr>
<td style="text-align:left;">
6
</td>
<td style="text-align:left;">
gene
</td>
<td style="text-align:right;">
41.53822
</td>
<td style="text-align:right;">
41.53988
</td>
<td style="text-align:left;">
positive
</td>
<td style="text-align:left;">
MGI_C57BL6J_4439726
</td>
<td style="text-align:left;">
Trbc1
</td>
<td style="text-align:left;">
NCBI_Gene:100125262,ENSEMBL:ENSMUSG00000076490
</td>
<td style="text-align:left;">
MGI:4439726
</td>
<td style="text-align:left;">
gene segment
</td>
<td style="text-align:left;">
T cell receptor beta, constant region 1
</td>
</tr>
<tr>
<td style="text-align:left;">
6
</td>
<td style="text-align:left;">
gene
</td>
<td style="text-align:right;">
41.54216
</td>
<td style="text-align:right;">
41.54218
</td>
<td style="text-align:left;">
positive
</td>
<td style="text-align:left;">
MGI_C57BL6J_4439727
</td>
<td style="text-align:left;">
Trbd2
</td>
<td style="text-align:left;">
NCBI_Gene:100125247,ENSEMBL:ENSMUSG00000094569
</td>
<td style="text-align:left;">
MGI:4439727
</td>
<td style="text-align:left;">
gene segment
</td>
<td style="text-align:left;">
T cell receptor beta, D region 2
</td>
</tr>
<tr>
<td style="text-align:left;">
6
</td>
<td style="text-align:left;">
gene
</td>
<td style="text-align:right;">
41.54275
</td>
<td style="text-align:right;">
41.54280
</td>
<td style="text-align:left;">
positive
</td>
<td style="text-align:left;">
MGI_C57BL6J_4439728
</td>
<td style="text-align:left;">
Trbj2-1
</td>
<td style="text-align:left;">
NCBI_Gene:100125255,ENSEMBL:ENSMUSG00000076492
</td>
<td style="text-align:left;">
MGI:4439728
</td>
<td style="text-align:left;">
gene segment
</td>
<td style="text-align:left;">
T cell receptor beta joining 2-1
</td>
</tr>
<tr>
<td style="text-align:left;">
6
</td>
<td style="text-align:left;">
gene
</td>
<td style="text-align:right;">
41.54296
</td>
<td style="text-align:right;">
41.54301
</td>
<td style="text-align:left;">
positive
</td>
<td style="text-align:left;">
MGI_C57BL6J_4439729
</td>
<td style="text-align:left;">
Trbj2-2
</td>
<td style="text-align:left;">
NCBI_Gene:100125256,ENSEMBL:ENSMUSG00000076493
</td>
<td style="text-align:left;">
MGI:4439729
</td>
<td style="text-align:left;">
gene segment
</td>
<td style="text-align:left;">
T cell receptor beta joining 2-2
</td>
</tr>
<tr>
<td style="text-align:left;">
6
</td>
<td style="text-align:left;">
gene
</td>
<td style="text-align:right;">
41.54322
</td>
<td style="text-align:right;">
41.54327
</td>
<td style="text-align:left;">
positive
</td>
<td style="text-align:left;">
MGI_C57BL6J_4439898
</td>
<td style="text-align:left;">
Trbj2-3
</td>
<td style="text-align:left;">
NCBI_Gene:100125257,ENSEMBL:ENSMUSG00000076494
</td>
<td style="text-align:left;">
MGI:4439898
</td>
<td style="text-align:left;">
gene segment
</td>
<td style="text-align:left;">
T cell receptor beta joining 2-3
</td>
</tr>
<tr>
<td style="text-align:left;">
6
</td>
<td style="text-align:left;">
gene
</td>
<td style="text-align:right;">
41.54336
</td>
<td style="text-align:right;">
41.54341
</td>
<td style="text-align:left;">
positive
</td>
<td style="text-align:left;">
MGI_C57BL6J_4439730
</td>
<td style="text-align:left;">
Trbj2-4
</td>
<td style="text-align:left;">
NCBI_Gene:100125258,ENSEMBL:ENSMUSG00000076495
</td>
<td style="text-align:left;">
MGI:4439730
</td>
<td style="text-align:left;">
gene segment
</td>
<td style="text-align:left;">
T cell receptor beta joining 2-4
</td>
</tr>
<tr>
<td style="text-align:left;">
6
</td>
<td style="text-align:left;">
gene
</td>
<td style="text-align:right;">
41.54345
</td>
<td style="text-align:right;">
41.54350
</td>
<td style="text-align:left;">
positive
</td>
<td style="text-align:left;">
MGI_C57BL6J_4439899
</td>
<td style="text-align:left;">
Trbj2-5
</td>
<td style="text-align:left;">
NCBI_Gene:100125259,ENSEMBL:ENSMUSG00000076496
</td>
<td style="text-align:left;">
MGI:4439899
</td>
<td style="text-align:left;">
gene segment
</td>
<td style="text-align:left;">
T cell receptor beta joining 2-5
</td>
</tr>
<tr>
<td style="text-align:left;">
6
</td>
<td style="text-align:left;">
pseudogene
</td>
<td style="text-align:right;">
41.54360
</td>
<td style="text-align:right;">
41.54364
</td>
<td style="text-align:left;">
positive
</td>
<td style="text-align:left;">
MGI_C57BL6J_4937236
</td>
<td style="text-align:left;">
Trbj2-6
</td>
<td style="text-align:left;">
NCBI_Gene:100125260,ENSEMBL:ENSMUSG00000090818
</td>
<td style="text-align:left;">
MGI:4937236
</td>
<td style="text-align:left;">
pseudogenic gene segment
</td>
<td style="text-align:left;">
T cell receptor beta joining 2-6
</td>
</tr>
<tr>
<td style="text-align:left;">
6
</td>
<td style="text-align:left;">
gene
</td>
<td style="text-align:right;">
41.54381
</td>
<td style="text-align:right;">
41.54386
</td>
<td style="text-align:left;">
positive
</td>
<td style="text-align:left;">
MGI_C57BL6J_4439731
</td>
<td style="text-align:left;">
Trbj2-7
</td>
<td style="text-align:left;">
NCBI_Gene:100125261,ENSEMBL:ENSMUSG00000076497
</td>
<td style="text-align:left;">
MGI:4439731
</td>
<td style="text-align:left;">
gene segment
</td>
<td style="text-align:left;">
T cell receptor beta joining 2-7
</td>
</tr>
<tr>
<td style="text-align:left;">
6
</td>
<td style="text-align:left;">
gene
</td>
<td style="text-align:right;">
41.54673
</td>
<td style="text-align:right;">
41.54835
</td>
<td style="text-align:left;">
positive
</td>
<td style="text-align:left;">
MGI_C57BL6J_4835227
</td>
<td style="text-align:left;">
Trbc2
</td>
<td style="text-align:left;">
NCBI_Gene:100125263,ENSEMBL:ENSMUSG00000076498
</td>
<td style="text-align:left;">
MGI:4835227
</td>
<td style="text-align:left;">
gene segment
</td>
<td style="text-align:left;">
T cell receptor beta, constant 2
</td>
</tr>
<tr>
<td style="text-align:left;">
6
</td>
<td style="text-align:left;">
gene
</td>
<td style="text-align:right;">
41.55769
</td>
<td style="text-align:right;">
41.55845
</td>
<td style="text-align:left;">
negative
</td>
<td style="text-align:left;">
MGI_C57BL6J_98588
</td>
<td style="text-align:left;">
Trbv31
</td>
<td style="text-align:left;">
NCBI_Gene:269846,ENSEMBL:ENSMUSG00000076499
</td>
<td style="text-align:left;">
MGI:98588
</td>
<td style="text-align:left;">
gene segment
</td>
<td style="text-align:left;">
T cell receptor beta, variable 31
</td>
</tr>
<tr>
<td style="text-align:left;">
6
</td>
<td style="text-align:left;">
gene
</td>
<td style="text-align:right;">
41.55849
</td>
<td style="text-align:right;">
41.60345
</td>
<td style="text-align:left;">
positive
</td>
<td style="text-align:left;">
MGI_C57BL6J_5621467
</td>
<td style="text-align:left;">
Gm38582
</td>
<td style="text-align:left;">
NCBI_Gene:102641876
</td>
<td style="text-align:left;">
MGI:5621467
</td>
<td style="text-align:left;">
lncRNA gene
</td>
<td style="text-align:left;">
predicted gene, 38582
</td>
</tr>
<tr>
<td style="text-align:left;">
6
</td>
<td style="text-align:left;">
gene
</td>
<td style="text-align:right;">
41.56729
</td>
<td style="text-align:right;">
41.57071
</td>
<td style="text-align:left;">
positive
</td>
<td style="text-align:left;">
MGI_C57BL6J_2444452
</td>
<td style="text-align:left;">
B630006K09Rik
</td>
<td style="text-align:left;">
NA
</td>
<td style="text-align:left;">
NA
</td>
<td style="text-align:left;">
unclassified gene
</td>
<td style="text-align:left;">
RIKEN cDNA B630006K09 gene
</td>
</tr>
<tr>
<td style="text-align:left;">
6
</td>
<td style="text-align:left;">
gene
</td>
<td style="text-align:right;">
41.60224
</td>
<td style="text-align:right;">
41.60231
</td>
<td style="text-align:left;">
negative
</td>
<td style="text-align:left;">
MGI_C57BL6J_5530931
</td>
<td style="text-align:left;">
Gm27549
</td>
<td style="text-align:left;">
ENSEMBL:ENSMUSG00000098970
</td>
<td style="text-align:left;">
MGI:5530931
</td>
<td style="text-align:left;">
snRNA gene
</td>
<td style="text-align:left;">
predicted gene, 27549
</td>
</tr>
<tr>
<td style="text-align:left;">
6
</td>
<td style="text-align:left;">
gene
</td>
<td style="text-align:right;">
41.60504
</td>
<td style="text-align:right;">
41.62051
</td>
<td style="text-align:left;">
positive
</td>
<td style="text-align:left;">
MGI_C57BL6J_1096338
</td>
<td style="text-align:left;">
Ephb6
</td>
<td style="text-align:left;">
NCBI_Gene:13848,ENSEMBL:ENSMUSG00000029869
</td>
<td style="text-align:left;">
MGI:1096338
</td>
<td style="text-align:left;">
protein coding gene
</td>
<td style="text-align:left;">
Eph receptor B6
</td>
</tr>
<tr>
<td style="text-align:left;">
6
</td>
<td style="text-align:left;">
gene
</td>
<td style="text-align:right;">
41.62062
</td>
<td style="text-align:right;">
41.63641
</td>
<td style="text-align:left;">
negative
</td>
<td style="text-align:left;">
MGI_C57BL6J_1927259
</td>
<td style="text-align:left;">
Trpv6
</td>
<td style="text-align:left;">
NCBI_Gene:64177,ENSEMBL:ENSMUSG00000029868
</td>
<td style="text-align:left;">
MGI:1927259
</td>
<td style="text-align:left;">
protein coding gene
</td>
<td style="text-align:left;">
transient receptor potential cation channel, subfamily V, member 6
</td>
</tr>
<tr>
<td style="text-align:left;">
6
</td>
<td style="text-align:left;">
gene
</td>
<td style="text-align:right;">
41.62178
</td>
<td style="text-align:right;">
41.62406
</td>
<td style="text-align:left;">
positive
</td>
<td style="text-align:left;">
MGI_C57BL6J_5621680
</td>
<td style="text-align:left;">
Gm38795
</td>
<td style="text-align:left;">
NCBI_Gene:105242636
</td>
<td style="text-align:left;">
MGI:5621680
</td>
<td style="text-align:left;">
lncRNA gene
</td>
<td style="text-align:left;">
predicted gene, 38795
</td>
</tr>
<tr>
<td style="text-align:left;">
6
</td>
<td style="text-align:left;">
gene
</td>
<td style="text-align:right;">
41.64173
</td>
<td style="text-align:right;">
41.65880
</td>
<td style="text-align:left;">
positive
</td>
<td style="text-align:left;">
MGI_C57BL6J_5593801
</td>
<td style="text-align:left;">
Gm34642
</td>
<td style="text-align:left;">
NCBI_Gene:102637960
</td>
<td style="text-align:left;">
MGI:5593801
</td>
<td style="text-align:left;">
lncRNA gene
</td>
<td style="text-align:left;">
predicted gene, 34642
</td>
</tr>
<tr>
<td style="text-align:left;">
6
</td>
<td style="text-align:left;">
gene
</td>
<td style="text-align:right;">
41.65166
</td>
<td style="text-align:right;">
41.68145
</td>
<td style="text-align:left;">
negative
</td>
<td style="text-align:left;">
MGI_C57BL6J_2429764
</td>
<td style="text-align:left;">
Trpv5
</td>
<td style="text-align:left;">
NCBI_Gene:194352,ENSEMBL:ENSMUSG00000036899
</td>
<td style="text-align:left;">
MGI:2429764
</td>
<td style="text-align:left;">
protein coding gene
</td>
<td style="text-align:left;">
transient receptor potential cation channel, subfamily V, member 5
</td>
</tr>
<tr>
<td style="text-align:left;">
6
</td>
<td style="text-align:left;">
gene
</td>
<td style="text-align:right;">
41.68435
</td>
<td style="text-align:right;">
41.68572
</td>
<td style="text-align:left;">
positive
</td>
<td style="text-align:left;">
MGI_C57BL6J_1923856
</td>
<td style="text-align:left;">
Llcfc1
</td>
<td style="text-align:left;">
NCBI_Gene:76606,ENSEMBL:ENSMUSG00000029867
</td>
<td style="text-align:left;">
MGI:1923856
</td>
<td style="text-align:left;">
protein coding gene
</td>
<td style="text-align:left;">
LLLL and CFNLAS motif containing 1
</td>
</tr>
<tr>
<td style="text-align:left;">
6
</td>
<td style="text-align:left;">
gene
</td>
<td style="text-align:right;">
41.68633
</td>
<td style="text-align:right;">
41.70434
</td>
<td style="text-align:left;">
negative
</td>
<td style="text-align:left;">
MGI_C57BL6J_1346053
</td>
<td style="text-align:left;">
Kel
</td>
<td style="text-align:left;">
NCBI_Gene:23925,ENSEMBL:ENSMUSG00000029866
</td>
<td style="text-align:left;">
MGI:1346053
</td>
<td style="text-align:left;">
protein coding gene
</td>
<td style="text-align:left;">
Kell blood group
</td>
</tr>
<tr>
<td style="text-align:left;">
6
</td>
<td style="text-align:left;">
gene
</td>
<td style="text-align:right;">
41.69264
</td>
<td style="text-align:right;">
41.69619
</td>
<td style="text-align:left;">
negative
</td>
<td style="text-align:left;">
MGI_C57BL6J_5610755
</td>
<td style="text-align:left;">
Gm37527
</td>
<td style="text-align:left;">
ENSEMBL:ENSMUSG00000103788
</td>
<td style="text-align:left;">
MGI:5610755
</td>
<td style="text-align:left;">
unclassified gene
</td>
<td style="text-align:left;">
predicted gene, 37527
</td>
</tr>
<tr>
<td style="text-align:left;">
6
</td>
<td style="text-align:left;">
gene
</td>
<td style="text-align:right;">
41.70619
</td>
<td style="text-align:right;">
41.71014
</td>
<td style="text-align:left;">
positive
</td>
<td style="text-align:left;">
MGI_C57BL6J_5621681
</td>
<td style="text-align:left;">
Gm38796
</td>
<td style="text-align:left;">
NCBI_Gene:105242637
</td>
<td style="text-align:left;">
MGI:5621681
</td>
<td style="text-align:left;">
lncRNA gene
</td>
<td style="text-align:left;">
predicted gene, 38796
</td>
</tr>
<tr>
<td style="text-align:left;">
6
</td>
<td style="text-align:left;">
gene
</td>
<td style="text-align:right;">
41.76877
</td>
<td style="text-align:right;">
41.77518
</td>
<td style="text-align:left;">
negative
</td>
<td style="text-align:left;">
MGI_C57BL6J_3030293
</td>
<td style="text-align:left;">
Olfr459
</td>
<td style="text-align:left;">
NCBI_Gene:258569,ENSEMBL:ENSMUSG00000045479
</td>
<td style="text-align:left;">
MGI:3030293
</td>
<td style="text-align:left;">
protein coding gene
</td>
<td style="text-align:left;">
olfactory receptor 459
</td>
</tr>
<tr>
<td style="text-align:left;">
6
</td>
<td style="text-align:left;">
gene
</td>
<td style="text-align:right;">
41.84754
</td>
<td style="text-align:right;">
41.85207
</td>
<td style="text-align:left;">
positive
</td>
<td style="text-align:left;">
MGI_C57BL6J_102696
</td>
<td style="text-align:left;">
Pip
</td>
<td style="text-align:left;">
NCBI_Gene:18716,ENSEMBL:ENSMUSG00000058499
</td>
<td style="text-align:left;">
MGI:102696
</td>
<td style="text-align:left;">
protein coding gene
</td>
<td style="text-align:left;">
prolactin induced protein
</td>
</tr>
<tr>
<td style="text-align:left;">
6
</td>
<td style="text-align:left;">
gene
</td>
<td style="text-align:right;">
41.85299
</td>
<td style="text-align:right;">
41.86441
</td>
<td style="text-align:left;">
positive
</td>
<td style="text-align:left;">
MGI_C57BL6J_1934254
</td>
<td style="text-align:left;">
Sval2
</td>
<td style="text-align:left;">
NCBI_Gene:84543,ENSEMBL:ENSMUSG00000014104
</td>
<td style="text-align:left;">
MGI:1934254
</td>
<td style="text-align:left;">
protein coding gene
</td>
<td style="text-align:left;">
seminal vesicle antigen-like 2
</td>
</tr>
<tr>
<td style="text-align:left;">
6
</td>
<td style="text-align:left;">
gene
</td>
<td style="text-align:right;">
41.95163
</td>
<td style="text-align:right;">
41.95610
</td>
<td style="text-align:left;">
positive
</td>
<td style="text-align:left;">
MGI_C57BL6J_1918828
</td>
<td style="text-align:left;">
Sval1
</td>
<td style="text-align:left;">
NCBI_Gene:71578,ENSEMBL:ENSMUSG00000029865
</td>
<td style="text-align:left;">
MGI:1918828
</td>
<td style="text-align:left;">
protein coding gene
</td>
<td style="text-align:left;">
seminal vesicle antigen-like 1
</td>
</tr>
<tr>
<td style="text-align:left;">
6
</td>
<td style="text-align:left;">
gene
</td>
<td style="text-align:right;">
41.96814
</td>
<td style="text-align:right;">
41.97309
</td>
<td style="text-align:left;">
positive
</td>
<td style="text-align:left;">
MGI_C57BL6J_3047714
</td>
<td style="text-align:left;">
Sval3
</td>
<td style="text-align:left;">
NCBI_Gene:387564,ENSEMBL:ENSMUSG00000062833
</td>
<td style="text-align:left;">
MGI:3047714
</td>
<td style="text-align:left;">
protein coding gene
</td>
<td style="text-align:left;">
seminal vesicle antigen-like 3
</td>
</tr>
<tr>
<td style="text-align:left;">
6
</td>
<td style="text-align:left;">
gene
</td>
<td style="text-align:right;">
42.00990
</td>
<td style="text-align:right;">
42.03309
</td>
<td style="text-align:left;">
positive
</td>
<td style="text-align:left;">
MGI_C57BL6J_2141598
</td>
<td style="text-align:left;">
AU046084
</td>
<td style="text-align:left;">
NA
</td>
<td style="text-align:left;">
NA
</td>
<td style="text-align:left;">
unclassified gene
</td>
<td style="text-align:left;">
expressed sequence AU046084
</td>
</tr>
<tr>
<td style="text-align:left;">
6
</td>
<td style="text-align:left;">
gene
</td>
<td style="text-align:right;">
42.03839
</td>
<td style="text-align:right;">
42.04285
</td>
<td style="text-align:left;">
positive
</td>
<td style="text-align:left;">
MGI_C57BL6J_102785
</td>
<td style="text-align:left;">
Sva
</td>
<td style="text-align:left;">
NCBI_Gene:20939,ENSEMBL:ENSMUSG00000023289
</td>
<td style="text-align:left;">
MGI:102785
</td>
<td style="text-align:left;">
protein coding gene
</td>
<td style="text-align:left;">
seminal vesicle antigen
</td>
</tr>
<tr>
<td style="text-align:left;">
6
</td>
<td style="text-align:left;">
gene
</td>
<td style="text-align:right;">
42.10570
</td>
<td style="text-align:right;">
42.11452
</td>
<td style="text-align:left;">
negative
</td>
<td style="text-align:left;">
MGI_C57BL6J_5593860
</td>
<td style="text-align:left;">
Gm34701
</td>
<td style="text-align:left;">
NCBI_Gene:102638036,ENSEMBL:ENSMUSG00000107543
</td>
<td style="text-align:left;">
MGI:5593860
</td>
<td style="text-align:left;">
lncRNA gene
</td>
<td style="text-align:left;">
predicted gene, 34701
</td>
</tr>
<tr>
<td style="text-align:left;">
6
</td>
<td style="text-align:left;">
gene
</td>
<td style="text-align:right;">
42.14094
</td>
<td style="text-align:right;">
42.14189
</td>
<td style="text-align:left;">
positive
</td>
<td style="text-align:left;">
MGI_C57BL6J_2681308
</td>
<td style="text-align:left;">
Tas2r139
</td>
<td style="text-align:left;">
NCBI_Gene:353148,ENSEMBL:ENSMUSG00000047102
</td>
<td style="text-align:left;">
MGI:2681308
</td>
<td style="text-align:left;">
protein coding gene
</td>
<td style="text-align:left;">
taste receptor, type 2, member 139
</td>
</tr>
<tr>
<td style="text-align:left;">
6
</td>
<td style="text-align:left;">
gene
</td>
<td style="text-align:right;">
42.21533
</td>
<td style="text-align:right;">
42.21629
</td>
<td style="text-align:left;">
positive
</td>
<td style="text-align:left;">
MGI_C57BL6J_2681312
</td>
<td style="text-align:left;">
Tas2r144
</td>
<td style="text-align:left;">
NCBI_Gene:387515,ENSEMBL:ENSMUSG00000051917
</td>
<td style="text-align:left;">
MGI:2681312
</td>
<td style="text-align:left;">
protein coding gene
</td>
<td style="text-align:left;">
taste receptor, type 2, member 144
</td>
</tr>
<tr>
<td style="text-align:left;">
6
</td>
<td style="text-align:left;">
gene
</td>
<td style="text-align:right;">
42.24594
</td>
<td style="text-align:right;">
42.25045
</td>
<td style="text-align:left;">
positive
</td>
<td style="text-align:left;">
MGI_C57BL6J_1923513
</td>
<td style="text-align:left;">
Gstk1
</td>
<td style="text-align:left;">
NCBI_Gene:76263,ENSEMBL:ENSMUSG00000029864
</td>
<td style="text-align:left;">
MGI:1923513
</td>
<td style="text-align:left;">
protein coding gene
</td>
<td style="text-align:left;">
glutathione S-transferase kappa 1
</td>
</tr>
<tr>
<td style="text-align:left;">
6
</td>
<td style="text-align:left;">
gene
</td>
<td style="text-align:right;">
42.26192
</td>
<td style="text-align:right;">
42.26456
</td>
<td style="text-align:left;">
positive
</td>
<td style="text-align:left;">
MGI_C57BL6J_1924444
</td>
<td style="text-align:left;">
Tmem139
</td>
<td style="text-align:left;">
NCBI_Gene:109218,ENSEMBL:ENSMUSG00000071506
</td>
<td style="text-align:left;">
MGI:1924444
</td>
<td style="text-align:left;">
protein coding gene
</td>
<td style="text-align:left;">
transmembrane protein 139
</td>
</tr>
<tr>
<td style="text-align:left;">
6
</td>
<td style="text-align:left;">
gene
</td>
<td style="text-align:right;">
42.26490
</td>
<td style="text-align:right;">
42.28251
</td>
<td style="text-align:left;">
positive
</td>
<td style="text-align:left;">
MGI_C57BL6J_97295
</td>
<td style="text-align:left;">
Casp2
</td>
<td style="text-align:left;">
NCBI_Gene:12366,ENSEMBL:ENSMUSG00000029863
</td>
<td style="text-align:left;">
MGI:97295
</td>
<td style="text-align:left;">
protein coding gene
</td>
<td style="text-align:left;">
caspase 2
</td>
</tr>
<tr>
<td style="text-align:left;">
6
</td>
<td style="text-align:left;">
gene
</td>
<td style="text-align:right;">
42.27565
</td>
<td style="text-align:right;">
42.27573
</td>
<td style="text-align:left;">
positive
</td>
<td style="text-align:left;">
MGI_C57BL6J_5454885
</td>
<td style="text-align:left;">
Gm25108
</td>
<td style="text-align:left;">
ENSEMBL:ENSMUSG00000084422
</td>
<td style="text-align:left;">
MGI:5454885
</td>
<td style="text-align:left;">
snoRNA gene
</td>
<td style="text-align:left;">
predicted gene, 25108
</td>
</tr>
<tr>
<td style="text-align:left;">
6
</td>
<td style="text-align:left;">
gene
</td>
<td style="text-align:right;">
42.28260
</td>
<td style="text-align:right;">
42.31576
</td>
<td style="text-align:left;">
positive
</td>
<td style="text-align:left;">
MGI_C57BL6J_88417
</td>
<td style="text-align:left;">
Clcn1
</td>
<td style="text-align:left;">
NCBI_Gene:12723,ENSEMBL:ENSMUSG00000029862
</td>
<td style="text-align:left;">
MGI:88417
</td>
<td style="text-align:left;">
protein coding gene
</td>
<td style="text-align:left;">
chloride channel, voltage-sensitive 1
</td>
</tr>
<tr>
<td style="text-align:left;">
6
</td>
<td style="text-align:left;">
gene
</td>
<td style="text-align:right;">
42.31015
</td>
<td style="text-align:right;">
42.31303
</td>
<td style="text-align:left;">
negative
</td>
<td style="text-align:left;">
MGI_C57BL6J_5621682
</td>
<td style="text-align:left;">
Gm38797
</td>
<td style="text-align:left;">
NCBI_Gene:105242638
</td>
<td style="text-align:left;">
MGI:5621682
</td>
<td style="text-align:left;">
lncRNA gene
</td>
<td style="text-align:left;">
predicted gene, 38797
</td>
</tr>
<tr>
<td style="text-align:left;">
6
</td>
<td style="text-align:left;">
gene
</td>
<td style="text-align:right;">
42.31531
</td>
<td style="text-align:right;">
42.32464
</td>
<td style="text-align:left;">
negative
</td>
<td style="text-align:left;">
MGI_C57BL6J_1923406
</td>
<td style="text-align:left;">
Fam131b
</td>
<td style="text-align:left;">
NCBI_Gene:76156,ENSEMBL:ENSMUSG00000029861
</td>
<td style="text-align:left;">
MGI:1923406
</td>
<td style="text-align:left;">
protein coding gene
</td>
<td style="text-align:left;">
family with sequence similarity 131, member B
</td>
</tr>
<tr>
<td style="text-align:left;">
6
</td>
<td style="text-align:left;">
gene
</td>
<td style="text-align:right;">
42.32871
</td>
<td style="text-align:right;">
42.35086
</td>
<td style="text-align:left;">
negative
</td>
<td style="text-align:left;">
MGI_C57BL6J_5690676
</td>
<td style="text-align:left;">
Gm44284
</td>
<td style="text-align:left;">
ENSEMBL:ENSMUSG00000108033
</td>
<td style="text-align:left;">
MGI:5690676
</td>
<td style="text-align:left;">
lncRNA gene
</td>
<td style="text-align:left;">
predicted gene, 44284
</td>
</tr>
<tr>
<td style="text-align:left;">
6
</td>
<td style="text-align:left;">
gene
</td>
<td style="text-align:right;">
42.33768
</td>
<td style="text-align:right;">
42.33778
</td>
<td style="text-align:left;">
negative
</td>
<td style="text-align:left;">
MGI_C57BL6J_5454388
</td>
<td style="text-align:left;">
Gm24611
</td>
<td style="text-align:left;">
ENSEMBL:ENSMUSG00000065206
</td>
<td style="text-align:left;">
MGI:5454388
</td>
<td style="text-align:left;">
snRNA gene
</td>
<td style="text-align:left;">
predicted gene, 24611
</td>
</tr>
<tr>
<td style="text-align:left;">
6
</td>
<td style="text-align:left;">
gene
</td>
<td style="text-align:right;">
42.34962
</td>
<td style="text-align:right;">
42.36021
</td>
<td style="text-align:left;">
positive
</td>
<td style="text-align:left;">
MGI_C57BL6J_103072
</td>
<td style="text-align:left;">
Zyx
</td>
<td style="text-align:left;">
NCBI_Gene:22793,ENSEMBL:ENSMUSG00000029860
</td>
<td style="text-align:left;">
MGI:103072
</td>
<td style="text-align:left;">
protein coding gene
</td>
<td style="text-align:left;">
zyxin
</td>
</tr>
<tr>
<td style="text-align:left;">
6
</td>
<td style="text-align:left;">
gene
</td>
<td style="text-align:right;">
42.35849
</td>
<td style="text-align:right;">
42.37331
</td>
<td style="text-align:left;">
negative
</td>
<td style="text-align:left;">
MGI_C57BL6J_107381
</td>
<td style="text-align:left;">
Epha1
</td>
<td style="text-align:left;">
NCBI_Gene:13835,ENSEMBL:ENSMUSG00000029859
</td>
<td style="text-align:left;">
MGI:107381
</td>
<td style="text-align:left;">
protein coding gene
</td>
<td style="text-align:left;">
Eph receptor A1
</td>
</tr>
<tr>
<td style="text-align:left;">
6
</td>
<td style="text-align:left;">
gene
</td>
<td style="text-align:right;">
42.37067
</td>
<td style="text-align:right;">
42.38121
</td>
<td style="text-align:left;">
positive
</td>
<td style="text-align:left;">
MGI_C57BL6J_1919381
</td>
<td style="text-align:left;">
2010310C07Rik
</td>
<td style="text-align:left;">
NCBI_Gene:72131,ENSEMBL:ENSMUSG00000101268
</td>
<td style="text-align:left;">
MGI:1919381
</td>
<td style="text-align:left;">
lncRNA gene
</td>
<td style="text-align:left;">
RIKEN cDNA 2010310C07 gene
</td>
</tr>
<tr>
<td style="text-align:left;">
6
</td>
<td style="text-align:left;">
gene
</td>
<td style="text-align:right;">
42.40024
</td>
<td style="text-align:right;">
42.40112
</td>
<td style="text-align:left;">
positive
</td>
<td style="text-align:left;">
MGI_C57BL6J_2681310
</td>
<td style="text-align:left;">
Tas2r143
</td>
<td style="text-align:left;">
NCBI_Gene:387514,ENSEMBL:ENSMUSG00000046652
</td>
<td style="text-align:left;">
MGI:2681310
</td>
<td style="text-align:left;">
protein coding gene
</td>
<td style="text-align:left;">
taste receptor, type 2, member 143
</td>
</tr>
<tr>
<td style="text-align:left;">
6
</td>
<td style="text-align:left;">
gene
</td>
<td style="text-align:right;">
42.40543
</td>
<td style="text-align:right;">
42.40653
</td>
<td style="text-align:left;">
positive
</td>
<td style="text-align:left;">
MGI_C57BL6J_2681302
</td>
<td style="text-align:left;">
Tas2r135
</td>
<td style="text-align:left;">
NCBI_Gene:387512,ENSEMBL:ENSMUSG00000056203
</td>
<td style="text-align:left;">
MGI:2681302
</td>
<td style="text-align:left;">
protein coding gene
</td>
<td style="text-align:left;">
taste receptor, type 2, member 135
</td>
</tr>
<tr>
<td style="text-align:left;">
6
</td>
<td style="text-align:left;">
gene
</td>
<td style="text-align:right;">
42.43453
</td>
<td style="text-align:right;">
42.43546
</td>
<td style="text-align:left;">
positive
</td>
<td style="text-align:left;">
MGI_C57BL6J_2681273
</td>
<td style="text-align:left;">
Tas2r126
</td>
<td style="text-align:left;">
NCBI_Gene:387353,ENSEMBL:ENSMUSG00000048284
</td>
<td style="text-align:left;">
MGI:2681273
</td>
<td style="text-align:left;">
protein coding gene
</td>
<td style="text-align:left;">
taste receptor, type 2, member 126
</td>
</tr>
<tr>
<td style="text-align:left;">
6
</td>
<td style="text-align:left;">
pseudogene
</td>
<td style="text-align:right;">
42.44379
</td>
<td style="text-align:right;">
42.44393
</td>
<td style="text-align:left;">
negative
</td>
<td style="text-align:left;">
MGI_C57BL6J_5690862
</td>
<td style="text-align:left;">
Gm44470
</td>
<td style="text-align:left;">
ENSEMBL:ENSMUSG00000108150
</td>
<td style="text-align:left;">
MGI:5690862
</td>
<td style="text-align:left;">
pseudogene
</td>
<td style="text-align:left;">
predicted gene, 44470
</td>
</tr>
<tr>
<td style="text-align:left;">
6
</td>
<td style="text-align:left;">
pseudogene
</td>
<td style="text-align:right;">
42.44524
</td>
<td style="text-align:right;">
42.44591
</td>
<td style="text-align:left;">
negative
</td>
<td style="text-align:left;">
MGI_C57BL6J_5593977
</td>
<td style="text-align:left;">
Gm34818
</td>
<td style="text-align:left;">
NCBI_Gene:102638200
</td>
<td style="text-align:left;">
MGI:5593977
</td>
<td style="text-align:left;">
pseudogene
</td>
<td style="text-align:left;">
predicted gene, 34818
</td>
</tr>
<tr>
<td style="text-align:left;">
6
</td>
<td style="text-align:left;">
gene
</td>
<td style="text-align:right;">
42.45998
</td>
<td style="text-align:right;">
42.46520
</td>
<td style="text-align:left;">
negative
</td>
<td style="text-align:left;">
MGI_C57BL6J_3030292
</td>
<td style="text-align:left;">
Olfr458
</td>
<td style="text-align:left;">
NCBI_Gene:258436,ENSEMBL:ENSMUSG00000068574
</td>
<td style="text-align:left;">
MGI:3030292
</td>
<td style="text-align:left;">
protein coding gene
</td>
<td style="text-align:left;">
olfactory receptor 458
</td>
</tr>
<tr>
<td style="text-align:left;">
6
</td>
<td style="text-align:left;">
gene
</td>
<td style="text-align:right;">
42.46802
</td>
<td style="text-align:right;">
42.47633
</td>
<td style="text-align:left;">
negative
</td>
<td style="text-align:left;">
MGI_C57BL6J_3030291
</td>
<td style="text-align:left;">
Olfr457
</td>
<td style="text-align:left;">
NCBI_Gene:258989,ENSEMBL:ENSMUSG00000091983
</td>
<td style="text-align:left;">
MGI:3030291
</td>
<td style="text-align:left;">
protein coding gene
</td>
<td style="text-align:left;">
olfactory receptor 457
</td>
</tr>
<tr>
<td style="text-align:left;">
6
</td>
<td style="text-align:left;">
gene
</td>
<td style="text-align:right;">
42.48616
</td>
<td style="text-align:right;">
42.48721
</td>
<td style="text-align:left;">
negative
</td>
<td style="text-align:left;">
MGI_C57BL6J_3030290
</td>
<td style="text-align:left;">
Olfr456
</td>
<td style="text-align:left;">
NCBI_Gene:259144,ENSEMBL:ENSMUSG00000090631
</td>
<td style="text-align:left;">
MGI:3030290
</td>
<td style="text-align:left;">
protein coding gene
</td>
<td style="text-align:left;">
olfactory receptor 456
</td>
</tr>
<tr>
<td style="text-align:left;">
6
</td>
<td style="text-align:left;">
gene
</td>
<td style="text-align:right;">
42.50963
</td>
<td style="text-align:right;">
42.52147
</td>
<td style="text-align:left;">
positive
</td>
<td style="text-align:left;">
MGI_C57BL6J_5593911
</td>
<td style="text-align:left;">
Gm34752
</td>
<td style="text-align:left;">
NCBI_Gene:102638113
</td>
<td style="text-align:left;">
MGI:5593911
</td>
<td style="text-align:left;">
lncRNA gene
</td>
<td style="text-align:left;">
predicted gene, 34752
</td>
</tr>
<tr>
<td style="text-align:left;">
6
</td>
<td style="text-align:left;">
pseudogene
</td>
<td style="text-align:right;">
42.51947
</td>
<td style="text-align:right;">
42.52033
</td>
<td style="text-align:left;">
negative
</td>
<td style="text-align:left;">
MGI_C57BL6J_3779444
</td>
<td style="text-align:left;">
Gm4877
</td>
<td style="text-align:left;">
NCBI_Gene:232745,ENSEMBL:ENSMUSG00000078177
</td>
<td style="text-align:left;">
MGI:3779444
</td>
<td style="text-align:left;">
pseudogene
</td>
<td style="text-align:left;">
predicted gene 4877
</td>
</tr>
<tr>
<td style="text-align:left;">
6
</td>
<td style="text-align:left;">
gene
</td>
<td style="text-align:right;">
42.53566
</td>
<td style="text-align:right;">
42.54759
</td>
<td style="text-align:left;">
negative
</td>
<td style="text-align:left;">
MGI_C57BL6J_3030289
</td>
<td style="text-align:left;">
Olfr455
</td>
<td style="text-align:left;">
NCBI_Gene:546896,ENSEMBL:ENSMUSG00000071494
</td>
<td style="text-align:left;">
MGI:3030289
</td>
<td style="text-align:left;">
protein coding gene
</td>
<td style="text-align:left;">
olfactory receptor 455
</td>
</tr>
<tr>
<td style="text-align:left;">
6
</td>
<td style="text-align:left;">
gene
</td>
<td style="text-align:right;">
42.57823
</td>
<td style="text-align:right;">
42.58178
</td>
<td style="text-align:left;">
negative
</td>
<td style="text-align:left;">
MGI_C57BL6J_5621683
</td>
<td style="text-align:left;">
Gm38798
</td>
<td style="text-align:left;">
NCBI_Gene:105242639
</td>
<td style="text-align:left;">
MGI:5621683
</td>
<td style="text-align:left;">
lncRNA gene
</td>
<td style="text-align:left;">
predicted gene, 38798
</td>
</tr>
<tr>
<td style="text-align:left;">
6
</td>
<td style="text-align:left;">
gene
</td>
<td style="text-align:right;">
42.58487
</td>
<td style="text-align:right;">
42.59769
</td>
<td style="text-align:left;">
negative
</td>
<td style="text-align:left;">
MGI_C57BL6J_3042585
</td>
<td style="text-align:left;">
Tcaf3
</td>
<td style="text-align:left;">
NCBI_Gene:403088,ENSEMBL:ENSMUSG00000018656
</td>
<td style="text-align:left;">
MGI:3042585
</td>
<td style="text-align:left;">
protein coding gene
</td>
<td style="text-align:left;">
TRPM8 channel-associated factor 3
</td>
</tr>
<tr>
<td style="text-align:left;">
6
</td>
<td style="text-align:left;">
gene
</td>
<td style="text-align:right;">
42.62302
</td>
<td style="text-align:right;">
42.64525
</td>
<td style="text-align:left;">
negative
</td>
<td style="text-align:left;">
MGI_C57BL6J_2385258
</td>
<td style="text-align:left;">
Tcaf2
</td>
<td style="text-align:left;">
NCBI_Gene:232748,ENSEMBL:ENSMUSG00000029851
</td>
<td style="text-align:left;">
MGI:2385258
</td>
<td style="text-align:left;">
protein coding gene
</td>
<td style="text-align:left;">
TRPM8 channel-associated factor 2
</td>
</tr>
<tr>
<td style="text-align:left;">
6
</td>
<td style="text-align:left;">
gene
</td>
<td style="text-align:right;">
42.66800
</td>
<td style="text-align:right;">
42.71011
</td>
<td style="text-align:left;">
negative
</td>
<td style="text-align:left;">
MGI_C57BL6J_1914665
</td>
<td style="text-align:left;">
Tcaf1
</td>
<td style="text-align:left;">
NCBI_Gene:77574,ENSEMBL:ENSMUSG00000036667
</td>
<td style="text-align:left;">
MGI:1914665
</td>
<td style="text-align:left;">
protein coding gene
</td>
<td style="text-align:left;">
TRPM8 channel-associated factor 1
</td>
</tr>
<tr>
<td style="text-align:left;">
6
</td>
<td style="text-align:left;">
gene
</td>
<td style="text-align:right;">
42.69392
</td>
<td style="text-align:right;">
42.69576
</td>
<td style="text-align:left;">
negative
</td>
<td style="text-align:left;">
MGI_C57BL6J_5608918
</td>
<td style="text-align:left;">
B930083L10Rik
</td>
<td style="text-align:left;">
NA
</td>
<td style="text-align:left;">
NA
</td>
<td style="text-align:left;">
unclassified gene
</td>
<td style="text-align:left;">
Riken cDNA B930083L10 gene
</td>
</tr>
<tr>
<td style="text-align:left;">
6
</td>
<td style="text-align:left;">
pseudogene
</td>
<td style="text-align:right;">
42.72134
</td>
<td style="text-align:right;">
42.72226
</td>
<td style="text-align:left;">
positive
</td>
<td style="text-align:left;">
MGI_C57BL6J_3030288
</td>
<td style="text-align:left;">
Olfr454-ps1
</td>
<td style="text-align:left;">
NCBI_Gene:257777,ENSEMBL:ENSMUSG00000107986
</td>
<td style="text-align:left;">
MGI:3030288
</td>
<td style="text-align:left;">
pseudogene
</td>
<td style="text-align:left;">
olfactory receptor 454, pseudogene 1
</td>
</tr>
<tr>
<td style="text-align:left;">
6
</td>
<td style="text-align:left;">
pseudogene
</td>
<td style="text-align:right;">
42.72559
</td>
<td style="text-align:right;">
42.72602
</td>
<td style="text-align:left;">
positive
</td>
<td style="text-align:left;">
MGI_C57BL6J_5690422
</td>
<td style="text-align:left;">
Gm44030
</td>
<td style="text-align:left;">
ENSEMBL:ENSMUSG00000107416
</td>
<td style="text-align:left;">
MGI:5690422
</td>
<td style="text-align:left;">
pseudogene
</td>
<td style="text-align:left;">
predicted gene, 44030
</td>
</tr>
<tr>
<td style="text-align:left;">
6
</td>
<td style="text-align:left;">
pseudogene
</td>
<td style="text-align:right;">
42.73393
</td>
<td style="text-align:right;">
42.73410
</td>
<td style="text-align:left;">
negative
</td>
<td style="text-align:left;">
MGI_C57BL6J_5690424
</td>
<td style="text-align:left;">
Gm44032
</td>
<td style="text-align:left;">
ENSEMBL:ENSMUSG00000107650
</td>
<td style="text-align:left;">
MGI:5690424
</td>
<td style="text-align:left;">
pseudogene
</td>
<td style="text-align:left;">
predicted gene, 44032
</td>
</tr>
<tr>
<td style="text-align:left;">
6
</td>
<td style="text-align:left;">
gene
</td>
<td style="text-align:right;">
42.73937
</td>
<td style="text-align:right;">
42.74696
</td>
<td style="text-align:left;">
positive
</td>
<td style="text-align:left;">
MGI_C57BL6J_3030287
</td>
<td style="text-align:left;">
Olfr453
</td>
<td style="text-align:left;">
NCBI_Gene:258016,ENSEMBL:ENSMUSG00000095831
</td>
<td style="text-align:left;">
MGI:3030287
</td>
<td style="text-align:left;">
protein coding gene
</td>
<td style="text-align:left;">
olfactory receptor 453
</td>
</tr>
<tr>
<td style="text-align:left;">
6
</td>
<td style="text-align:left;">
pseudogene
</td>
<td style="text-align:right;">
42.74519
</td>
<td style="text-align:right;">
42.74525
</td>
<td style="text-align:left;">
positive
</td>
<td style="text-align:left;">
MGI_C57BL6J_5690426
</td>
<td style="text-align:left;">
Gm44034
</td>
<td style="text-align:left;">
ENSEMBL:ENSMUSG00000107818
</td>
<td style="text-align:left;">
MGI:5690426
</td>
<td style="text-align:left;">
pseudogene
</td>
<td style="text-align:left;">
predicted gene, 44034
</td>
</tr>
<tr>
<td style="text-align:left;">
6
</td>
<td style="text-align:left;">
gene
</td>
<td style="text-align:right;">
42.75640
</td>
<td style="text-align:right;">
42.76323
</td>
<td style="text-align:left;">
positive
</td>
<td style="text-align:left;">
MGI_C57BL6J_1313140
</td>
<td style="text-align:left;">
Olfr38
</td>
<td style="text-align:left;">
NCBI_Gene:258988,ENSEMBL:ENSMUSG00000095236
</td>
<td style="text-align:left;">
MGI:1313140
</td>
<td style="text-align:left;">
protein coding gene
</td>
<td style="text-align:left;">
olfactory receptor 38
</td>
</tr>
<tr>
<td style="text-align:left;">
6
</td>
<td style="text-align:left;">
gene
</td>
<td style="text-align:right;">
42.77557
</td>
<td style="text-align:right;">
42.77568
</td>
<td style="text-align:left;">
negative
</td>
<td style="text-align:left;">
MGI_C57BL6J_5452140
</td>
<td style="text-align:left;">
Gm22363
</td>
<td style="text-align:left;">
ENSEMBL:ENSMUSG00000065034
</td>
<td style="text-align:left;">
MGI:5452140
</td>
<td style="text-align:left;">
snRNA gene
</td>
<td style="text-align:left;">
predicted gene, 22363
</td>
</tr>
<tr>
<td style="text-align:left;">
6
</td>
<td style="text-align:left;">
gene
</td>
<td style="text-align:right;">
42.78509
</td>
<td style="text-align:right;">
42.79162
</td>
<td style="text-align:left;">
positive
</td>
<td style="text-align:left;">
MGI_C57BL6J_3030286
</td>
<td style="text-align:left;">
Olfr452
</td>
<td style="text-align:left;">
NCBI_Gene:258207,ENSEMBL:ENSMUSG00000094192
</td>
<td style="text-align:left;">
MGI:3030286
</td>
<td style="text-align:left;">
protein coding gene
</td>
<td style="text-align:left;">
olfactory receptor 452
</td>
</tr>
<tr>
<td style="text-align:left;">
6
</td>
<td style="text-align:left;">
pseudogene
</td>
<td style="text-align:right;">
42.80074
</td>
<td style="text-align:right;">
42.80168
</td>
<td style="text-align:left;">
positive
</td>
<td style="text-align:left;">
MGI_C57BL6J_3030285
</td>
<td style="text-align:left;">
Olfr451-ps1
</td>
<td style="text-align:left;">
NCBI_Gene:258017,ENSEMBL:ENSMUSG00000108058
</td>
<td style="text-align:left;">
MGI:3030285
</td>
<td style="text-align:left;">
polymorphic pseudogene
</td>
<td style="text-align:left;">
olfactory receptor 451, pseudogene 1
</td>
</tr>
<tr>
<td style="text-align:left;">
6
</td>
<td style="text-align:left;">
gene
</td>
<td style="text-align:right;">
42.81194
</td>
<td style="text-align:right;">
42.81908
</td>
<td style="text-align:left;">
positive
</td>
<td style="text-align:left;">
MGI_C57BL6J_3030284
</td>
<td style="text-align:left;">
Olfr450
</td>
<td style="text-align:left;">
NCBI_Gene:258437,ENSEMBL:ENSMUSG00000054431
</td>
<td style="text-align:left;">
MGI:3030284
</td>
<td style="text-align:left;">
protein coding gene
</td>
<td style="text-align:left;">
olfactory receptor 450
</td>
</tr>
<tr>
<td style="text-align:left;">
6
</td>
<td style="text-align:left;">
gene
</td>
<td style="text-align:right;">
42.83278
</td>
<td style="text-align:right;">
42.83288
</td>
<td style="text-align:left;">
positive
</td>
<td style="text-align:left;">
MGI_C57BL6J_5453759
</td>
<td style="text-align:left;">
Gm23982
</td>
<td style="text-align:left;">
ENSEMBL:ENSMUSG00000087766
</td>
<td style="text-align:left;">
MGI:5453759
</td>
<td style="text-align:left;">
snRNA gene
</td>
<td style="text-align:left;">
predicted gene, 23982
</td>
</tr>
<tr>
<td style="text-align:left;">
6
</td>
<td style="text-align:left;">
gene
</td>
<td style="text-align:right;">
42.83438
</td>
<td style="text-align:right;">
42.83952
</td>
<td style="text-align:left;">
positive
</td>
<td style="text-align:left;">
MGI_C57BL6J_3030283
</td>
<td style="text-align:left;">
Olfr449
</td>
<td style="text-align:left;">
NCBI_Gene:259067,ENSEMBL:ENSMUSG00000049168
</td>
<td style="text-align:left;">
MGI:3030283
</td>
<td style="text-align:left;">
protein coding gene
</td>
<td style="text-align:left;">
olfactory receptor 449
</td>
</tr>
<tr>
<td style="text-align:left;">
6
</td>
<td style="text-align:left;">
gene
</td>
<td style="text-align:right;">
42.89044
</td>
<td style="text-align:right;">
42.89908
</td>
<td style="text-align:left;">
positive
</td>
<td style="text-align:left;">
MGI_C57BL6J_3030282
</td>
<td style="text-align:left;">
Olfr448
</td>
<td style="text-align:left;">
NCBI_Gene:258270,ENSEMBL:ENSMUSG00000043119
</td>
<td style="text-align:left;">
MGI:3030282
</td>
<td style="text-align:left;">
protein coding gene
</td>
<td style="text-align:left;">
olfactory receptor 448
</td>
</tr>
<tr>
<td style="text-align:left;">
6
</td>
<td style="text-align:left;">
gene
</td>
<td style="text-align:right;">
42.90888
</td>
<td style="text-align:right;">
42.91427
</td>
<td style="text-align:left;">
positive
</td>
<td style="text-align:left;">
MGI_C57BL6J_3030281
</td>
<td style="text-align:left;">
Olfr447
</td>
<td style="text-align:left;">
NCBI_Gene:258990,ENSEMBL:ENSMUSG00000045708
</td>
<td style="text-align:left;">
MGI:3030281
</td>
<td style="text-align:left;">
protein coding gene
</td>
<td style="text-align:left;">
olfactory receptor 447
</td>
</tr>
<tr>
<td style="text-align:left;">
6
</td>
<td style="text-align:left;">
gene
</td>
<td style="text-align:right;">
42.92138
</td>
<td style="text-align:right;">
42.93114
</td>
<td style="text-align:left;">
positive
</td>
<td style="text-align:left;">
MGI_C57BL6J_3030280
</td>
<td style="text-align:left;">
Olfr446
</td>
<td style="text-align:left;">
NCBI_Gene:258292,ENSEMBL:ENSMUSG00000073111
</td>
<td style="text-align:left;">
MGI:3030280
</td>
<td style="text-align:left;">
protein coding gene
</td>
<td style="text-align:left;">
olfactory receptor 446
</td>
</tr>
<tr>
<td style="text-align:left;">
6
</td>
<td style="text-align:left;">
pseudogene
</td>
<td style="text-align:right;">
42.93407
</td>
<td style="text-align:right;">
42.93494
</td>
<td style="text-align:left;">
positive
</td>
<td style="text-align:left;">
MGI_C57BL6J_3030279
</td>
<td style="text-align:left;">
Olrf445-ps1
</td>
<td style="text-align:left;">
NCBI_Gene:257781,ENSEMBL:ENSMUSG00000108157
</td>
<td style="text-align:left;">
MGI:3030279
</td>
<td style="text-align:left;">
pseudogene
</td>
<td style="text-align:left;">
olfactory receptor 445, pseudogene 1
</td>
</tr>
<tr>
<td style="text-align:left;">
6
</td>
<td style="text-align:left;">
gene
</td>
<td style="text-align:right;">
42.94907
</td>
<td style="text-align:right;">
42.95676
</td>
<td style="text-align:left;">
positive
</td>
<td style="text-align:left;">
MGI_C57BL6J_3030278
</td>
<td style="text-align:left;">
Olfr444
</td>
<td style="text-align:left;">
NCBI_Gene:258650,ENSEMBL:ENSMUSG00000073110
</td>
<td style="text-align:left;">
MGI:3030278
</td>
<td style="text-align:left;">
protein coding gene
</td>
<td style="text-align:left;">
olfactory receptor 444
</td>
</tr>
<tr>
<td style="text-align:left;">
6
</td>
<td style="text-align:left;">
gene
</td>
<td style="text-align:right;">
42.98069
</td>
<td style="text-align:right;">
42.98115
</td>
<td style="text-align:left;">
positive
</td>
<td style="text-align:left;">
MGI_C57BL6J_4937106
</td>
<td style="text-align:left;">
Gm17472
</td>
<td style="text-align:left;">
ENSEMBL:ENSMUSG00000091008
</td>
<td style="text-align:left;">
MGI:4937106
</td>
<td style="text-align:left;">
gene segment
</td>
<td style="text-align:left;">
predicted gene, 17472
</td>
</tr>
<tr>
<td style="text-align:left;">
6
</td>
<td style="text-align:left;">
gene
</td>
<td style="text-align:right;">
43.01450
</td>
<td style="text-align:right;">
43.06690
</td>
<td style="text-align:left;">
negative
</td>
<td style="text-align:left;">
MGI_C57BL6J_5594064
</td>
<td style="text-align:left;">
Gm34905
</td>
<td style="text-align:left;">
NCBI_Gene:102638311
</td>
<td style="text-align:left;">
MGI:5594064
</td>
<td style="text-align:left;">
lncRNA gene
</td>
<td style="text-align:left;">
predicted gene, 34905
</td>
</tr>
<tr>
<td style="text-align:left;">
6
</td>
<td style="text-align:left;">
pseudogene
</td>
<td style="text-align:right;">
43.06731
</td>
<td style="text-align:right;">
43.09558
</td>
<td style="text-align:left;">
positive
</td>
<td style="text-align:left;">
MGI_C57BL6J_3030277
</td>
<td style="text-align:left;">
Olfr443-ps1
</td>
<td style="text-align:left;">
NCBI_Gene:258106,ENSEMBL:ENSMUSG00000091386
</td>
<td style="text-align:left;">
MGI:3030277
</td>
<td style="text-align:left;">
pseudogene
</td>
<td style="text-align:left;">
olfactory receptor 443, pseudogene 1
</td>
</tr>
<tr>
<td style="text-align:left;">
6
</td>
<td style="text-align:left;">
gene
</td>
<td style="text-align:right;">
43.06746
</td>
<td style="text-align:right;">
43.09046
</td>
<td style="text-align:left;">
positive
</td>
<td style="text-align:left;">
MGI_C57BL6J_5621684
</td>
<td style="text-align:left;">
Gm38799
</td>
<td style="text-align:left;">
NCBI_Gene:105242640
</td>
<td style="text-align:left;">
MGI:5621684
</td>
<td style="text-align:left;">
lncRNA gene
</td>
<td style="text-align:left;">
predicted gene, 38799
</td>
</tr>
<tr>
<td style="text-align:left;">
6
</td>
<td style="text-align:left;">
gene
</td>
<td style="text-align:right;">
43.08152
</td>
<td style="text-align:right;">
43.08210
</td>
<td style="text-align:left;">
negative
</td>
<td style="text-align:left;">
MGI_C57BL6J_5313039
</td>
<td style="text-align:left;">
Gm20730
</td>
<td style="text-align:left;">
ENSEMBL:ENSMUSG00000076500
</td>
<td style="text-align:left;">
MGI:5313039
</td>
<td style="text-align:left;">
gene segment
</td>
<td style="text-align:left;">
predicted gene, 20730
</td>
</tr>
<tr>
<td style="text-align:left;">
6
</td>
<td style="text-align:left;">
pseudogene
</td>
<td style="text-align:right;">
43.09911
</td>
<td style="text-align:right;">
43.09972
</td>
<td style="text-align:left;">
positive
</td>
<td style="text-align:left;">
MGI_C57BL6J_3030276
</td>
<td style="text-align:left;">
Olfr442-ps1
</td>
<td style="text-align:left;">
NCBI_Gene:257780,ENSEMBL:ENSMUSG00000108193
</td>
<td style="text-align:left;">
MGI:3030276
</td>
<td style="text-align:left;">
pseudogene
</td>
<td style="text-align:left;">
olfactory receptor 442, pseudogene 1
</td>
</tr>
<tr>
<td style="text-align:left;">
6
</td>
<td style="text-align:left;">
gene
</td>
<td style="text-align:right;">
43.10637
</td>
<td style="text-align:right;">
43.11780
</td>
<td style="text-align:left;">
positive
</td>
<td style="text-align:left;">
MGI_C57BL6J_3030275
</td>
<td style="text-align:left;">
Olfr441
</td>
<td style="text-align:left;">
NCBI_Gene:258649,ENSEMBL:ENSMUSG00000094669
</td>
<td style="text-align:left;">
MGI:3030275
</td>
<td style="text-align:left;">
protein coding gene
</td>
<td style="text-align:left;">
olfactory receptor 441
</td>
</tr>
<tr>
<td style="text-align:left;">
6
</td>
<td style="text-align:left;">
pseudogene
</td>
<td style="text-align:right;">
43.12235
</td>
<td style="text-align:right;">
43.12327
</td>
<td style="text-align:left;">
positive
</td>
<td style="text-align:left;">
MGI_C57BL6J_3030274
</td>
<td style="text-align:left;">
Olfr440-ps1
</td>
<td style="text-align:left;">
NCBI_Gene:258107,ENSEMBL:ENSMUSG00000107889
</td>
<td style="text-align:left;">
MGI:3030274
</td>
<td style="text-align:left;">
pseudogene
</td>
<td style="text-align:left;">
olfactory receptor 440, pseudogene 1
</td>
</tr>
<tr>
<td style="text-align:left;">
6
</td>
<td style="text-align:left;">
pseudogene
</td>
<td style="text-align:right;">
43.13783
</td>
<td style="text-align:right;">
43.13844
</td>
<td style="text-align:left;">
positive
</td>
<td style="text-align:left;">
MGI_C57BL6J_3030273
</td>
<td style="text-align:left;">
Olfr439-ps1
</td>
<td style="text-align:left;">
NCBI_Gene:257779,ENSEMBL:ENSMUSG00000108149
</td>
<td style="text-align:left;">
MGI:3030273
</td>
<td style="text-align:left;">
pseudogene
</td>
<td style="text-align:left;">
olfactory receptor 439, pseudogene 1
</td>
</tr>
<tr>
<td style="text-align:left;">
6
</td>
<td style="text-align:left;">
gene
</td>
<td style="text-align:right;">
43.14657
</td>
<td style="text-align:right;">
43.15717
</td>
<td style="text-align:left;">
positive
</td>
<td style="text-align:left;">
MGI_C57BL6J_3030071
</td>
<td style="text-align:left;">
Olfr237-ps1
</td>
<td style="text-align:left;">
NCBI_Gene:258648,ENSEMBL:ENSMUSG00000094200
</td>
<td style="text-align:left;">
MGI:3030071
</td>
<td style="text-align:left;">
protein coding gene
</td>
<td style="text-align:left;">
olfactory receptor 237, pseudogene 1
</td>
</tr>
<tr>
<td style="text-align:left;">
6
</td>
<td style="text-align:left;">
gene
</td>
<td style="text-align:right;">
43.16245
</td>
<td style="text-align:right;">
43.17486
</td>
<td style="text-align:left;">
negative
</td>
<td style="text-align:left;">
MGI_C57BL6J_1922770
</td>
<td style="text-align:left;">
1700024N05Rik
</td>
<td style="text-align:left;">
NCBI_Gene:75520
</td>
<td style="text-align:left;">
MGI:1922770
</td>
<td style="text-align:left;">
lncRNA gene
</td>
<td style="text-align:left;">
RIKEN cDNA 1700024N05 gene
</td>
</tr>
<tr>
<td style="text-align:left;">
6
</td>
<td style="text-align:left;">
gene
</td>
<td style="text-align:right;">
43.16328
</td>
<td style="text-align:right;">
43.16986
</td>
<td style="text-align:left;">
positive
</td>
<td style="text-align:left;">
MGI_C57BL6J_3030271
</td>
<td style="text-align:left;">
Olfr437
</td>
<td style="text-align:left;">
NCBI_Gene:258293,ENSEMBL:ENSMUSG00000071481
</td>
<td style="text-align:left;">
MGI:3030271
</td>
<td style="text-align:left;">
protein coding gene
</td>
<td style="text-align:left;">
olfactory receptor 437
</td>
</tr>
<tr>
<td style="text-align:left;">
6
</td>
<td style="text-align:left;">
gene
</td>
<td style="text-align:right;">
43.17214
</td>
<td style="text-align:right;">
43.17746
</td>
<td style="text-align:left;">
positive
</td>
<td style="text-align:left;">
MGI_C57BL6J_104812
</td>
<td style="text-align:left;">
Olfr13
</td>
<td style="text-align:left;">
NCBI_Gene:18310,ENSEMBL:ENSMUSG00000043605
</td>
<td style="text-align:left;">
MGI:104812
</td>
<td style="text-align:left;">
protein coding gene
</td>
<td style="text-align:left;">
olfactory receptor 13
</td>
</tr>
<tr>
<td style="text-align:left;">
6
</td>
<td style="text-align:left;">
pseudogene
</td>
<td style="text-align:right;">
43.18665
</td>
<td style="text-align:right;">
43.18770
</td>
<td style="text-align:left;">
positive
</td>
<td style="text-align:left;">
MGI_C57BL6J_3030270
</td>
<td style="text-align:left;">
Olfr436-ps1
</td>
<td style="text-align:left;">
NCBI_Gene:257778,ENSEMBL:ENSMUSG00000107469
</td>
<td style="text-align:left;">
MGI:3030270
</td>
<td style="text-align:left;">
pseudogene
</td>
<td style="text-align:left;">
olfactory receptor 436, pseudogene 1
</td>
</tr>
<tr>
<td style="text-align:left;">
6
</td>
<td style="text-align:left;">
gene
</td>
<td style="text-align:right;">
43.19715
</td>
<td style="text-align:right;">
43.20363
</td>
<td style="text-align:left;">
positive
</td>
<td style="text-align:left;">
MGI_C57BL6J_3030269
</td>
<td style="text-align:left;">
Olfr435
</td>
<td style="text-align:left;">
NCBI_Gene:258647,ENSEMBL:ENSMUSG00000048693
</td>
<td style="text-align:left;">
MGI:3030269
</td>
<td style="text-align:left;">
protein coding gene
</td>
<td style="text-align:left;">
olfactory receptor 435
</td>
</tr>
<tr>
<td style="text-align:left;">
6
</td>
<td style="text-align:left;">
pseudogene
</td>
<td style="text-align:right;">
43.21033
</td>
<td style="text-align:right;">
43.21075
</td>
<td style="text-align:left;">
positive
</td>
<td style="text-align:left;">
MGI_C57BL6J_5433991
</td>
<td style="text-align:left;">
Gm21827
</td>
<td style="text-align:left;">
ENSEMBL:ENSMUSG00000096158
</td>
<td style="text-align:left;">
MGI:5433991
</td>
<td style="text-align:left;">
pseudogene
</td>
<td style="text-align:left;">
predicted gene, 21827
</td>
</tr>
<tr>
<td style="text-align:left;">
6
</td>
<td style="text-align:left;">
gene
</td>
<td style="text-align:right;">
43.21302
</td>
<td style="text-align:right;">
43.21811
</td>
<td style="text-align:left;">
positive
</td>
<td style="text-align:left;">
MGI_C57BL6J_3030268
</td>
<td style="text-align:left;">
Olfr434
</td>
<td style="text-align:left;">
NCBI_Gene:258366,ENSEMBL:ENSMUSG00000059411
</td>
<td style="text-align:left;">
MGI:3030268
</td>
<td style="text-align:left;">
protein coding gene
</td>
<td style="text-align:left;">
olfactory receptor 434
</td>
</tr>
<tr>
<td style="text-align:left;">
6
</td>
<td style="text-align:left;">
gene
</td>
<td style="text-align:right;">
43.23153
</td>
<td style="text-align:right;">
43.23696
</td>
<td style="text-align:left;">
positive
</td>
<td style="text-align:left;">
MGI_C57BL6J_1333821
</td>
<td style="text-align:left;">
Olfr47
</td>
<td style="text-align:left;">
NCBI_Gene:18346,ENSEMBL:ENSMUSG00000061210
</td>
<td style="text-align:left;">
MGI:1333821
</td>
<td style="text-align:left;">
protein coding gene
</td>
<td style="text-align:left;">
olfactory receptor 47
</td>
</tr>
<tr>
<td style="text-align:left;">
6
</td>
<td style="text-align:left;">
gene
</td>
<td style="text-align:right;">
43.26454
</td>
<td style="text-align:right;">
43.26586
</td>
<td style="text-align:left;">
negative
</td>
<td style="text-align:left;">
MGI_C57BL6J_5753307
</td>
<td style="text-align:left;">
Gm44731
</td>
<td style="text-align:left;">
ENSEMBL:ENSMUSG00000108190
</td>
<td style="text-align:left;">
MGI:5753307
</td>
<td style="text-align:left;">
lncRNA gene
</td>
<td style="text-align:left;">
predicted gene 44731
</td>
</tr>
<tr>
<td style="text-align:left;">
6
</td>
<td style="text-align:left;">
gene
</td>
<td style="text-align:right;">
43.26558
</td>
<td style="text-align:right;">
43.28932
</td>
<td style="text-align:left;">
positive
</td>
<td style="text-align:left;">
MGI_C57BL6J_1858952
</td>
<td style="text-align:left;">
Arhgef5
</td>
<td style="text-align:left;">
NCBI_Gene:54324,ENSEMBL:ENSMUSG00000033542
</td>
<td style="text-align:left;">
MGI:1858952
</td>
<td style="text-align:left;">
protein coding gene
</td>
<td style="text-align:left;">
Rho guanine nucleotide exchange factor (GEF) 5
</td>
</tr>
<tr>
<td style="text-align:left;">
6
</td>
<td style="text-align:left;">
gene
</td>
<td style="text-align:right;">
43.27775
</td>
<td style="text-align:right;">
43.27838
</td>
<td style="text-align:left;">
positive
</td>
<td style="text-align:left;">
MGI_C57BL6J_5690645
</td>
<td style="text-align:left;">
Gm44253
</td>
<td style="text-align:left;">
ENSEMBL:ENSMUSG00000107861
</td>
<td style="text-align:left;">
MGI:5690645
</td>
<td style="text-align:left;">
unclassified gene
</td>
<td style="text-align:left;">
predicted gene, 44253
</td>
</tr>
<tr>
<td style="text-align:left;">
6
</td>
<td style="text-align:left;">
gene
</td>
<td style="text-align:right;">
43.30367
</td>
<td style="text-align:right;">
43.31007
</td>
<td style="text-align:left;">
negative
</td>
<td style="text-align:left;">
MGI_C57BL6J_108011
</td>
<td style="text-align:left;">
Nobox
</td>
<td style="text-align:left;">
NCBI_Gene:18291,ENSEMBL:ENSMUSG00000029736
</td>
<td style="text-align:left;">
MGI:108011
</td>
<td style="text-align:left;">
protein coding gene
</td>
<td style="text-align:left;">
NOBOX oogenesis homeobox
</td>
</tr>
<tr>
<td style="text-align:left;">
6
</td>
<td style="text-align:left;">
gene
</td>
<td style="text-align:right;">
43.34500
</td>
<td style="text-align:right;">
43.66649
</td>
<td style="text-align:left;">
negative
</td>
<td style="text-align:left;">
MGI_C57BL6J_1352500
</td>
<td style="text-align:left;">
Tpk1
</td>
<td style="text-align:left;">
NCBI_Gene:29807,ENSEMBL:ENSMUSG00000029735
</td>
<td style="text-align:left;">
MGI:1352500
</td>
<td style="text-align:left;">
protein coding gene
</td>
<td style="text-align:left;">
thiamine pyrophosphokinase
</td>
</tr>
<tr>
<td style="text-align:left;">
6
</td>
<td style="text-align:left;">
pseudogene
</td>
<td style="text-align:right;">
43.37380
</td>
<td style="text-align:right;">
43.37555
</td>
<td style="text-align:left;">
positive
</td>
<td style="text-align:left;">
MGI_C57BL6J_3782999
</td>
<td style="text-align:left;">
Gm15550
</td>
<td style="text-align:left;">
NCBI_Gene:665747,ENSEMBL:ENSMUSG00000084325
</td>
<td style="text-align:left;">
MGI:3782999
</td>
<td style="text-align:left;">
pseudogene
</td>
<td style="text-align:left;">
predicted gene 15550
</td>
</tr>
<tr>
<td style="text-align:left;">
6
</td>
<td style="text-align:left;">
gene
</td>
<td style="text-align:right;">
43.37609
</td>
<td style="text-align:right;">
43.39253
</td>
<td style="text-align:left;">
positive
</td>
<td style="text-align:left;">
MGI_C57BL6J_5594272
</td>
<td style="text-align:left;">
Gm35113
</td>
<td style="text-align:left;">
NCBI_Gene:102638584
</td>
<td style="text-align:left;">
MGI:5594272
</td>
<td style="text-align:left;">
lncRNA gene
</td>
<td style="text-align:left;">
predicted gene, 35113
</td>
</tr>
<tr>
<td style="text-align:left;">
6
</td>
<td style="text-align:left;">
gene
</td>
<td style="text-align:right;">
43.41594
</td>
<td style="text-align:right;">
43.41867
</td>
<td style="text-align:left;">
positive
</td>
<td style="text-align:left;">
MGI_C57BL6J_5621685
</td>
<td style="text-align:left;">
Gm38800
</td>
<td style="text-align:left;">
NCBI_Gene:105242642
</td>
<td style="text-align:left;">
MGI:5621685
</td>
<td style="text-align:left;">
lncRNA gene
</td>
<td style="text-align:left;">
predicted gene, 38800
</td>
</tr>
<tr>
<td style="text-align:left;">
6
</td>
<td style="text-align:left;">
gene
</td>
<td style="text-align:right;">
43.44243
</td>
<td style="text-align:right;">
43.47432
</td>
<td style="text-align:left;">
positive
</td>
<td style="text-align:left;">
MGI_C57BL6J_2442776
</td>
<td style="text-align:left;">
9430018G01Rik
</td>
<td style="text-align:left;">
NCBI_Gene:319812,ENSEMBL:ENSMUSG00000107709
</td>
<td style="text-align:left;">
MGI:2442776
</td>
<td style="text-align:left;">
lncRNA gene
</td>
<td style="text-align:left;">
RIKEN cDNA 9430018G01 gene
</td>
</tr>
<tr>
<td style="text-align:left;">
6
</td>
<td style="text-align:left;">
gene
</td>
<td style="text-align:right;">
43.67881
</td>
<td style="text-align:right;">
43.67972
</td>
<td style="text-align:left;">
negative
</td>
<td style="text-align:left;">
MGI_C57BL6J_5826621
</td>
<td style="text-align:left;">
Gm46984
</td>
<td style="text-align:left;">
NCBI_Gene:108169189
</td>
<td style="text-align:left;">
MGI:5826621
</td>
<td style="text-align:left;">
lncRNA gene
</td>
<td style="text-align:left;">
predicted gene, 46984
</td>
</tr>
<tr>
<td style="text-align:left;">
6
</td>
<td style="text-align:left;">
pseudogene
</td>
<td style="text-align:right;">
43.76656
</td>
<td style="text-align:right;">
43.76732
</td>
<td style="text-align:left;">
negative
</td>
<td style="text-align:left;">
MGI_C57BL6J_3648301
</td>
<td style="text-align:left;">
Gm5990
</td>
<td style="text-align:left;">
NCBI_Gene:546897,ENSEMBL:ENSMUSG00000107465
</td>
<td style="text-align:left;">
MGI:3648301
</td>
<td style="text-align:left;">
pseudogene
</td>
<td style="text-align:left;">
predicted gene 5990
</td>
</tr>
<tr>
<td style="text-align:left;">
6
</td>
<td style="text-align:left;">
gene
</td>
<td style="text-align:right;">
43.83926
</td>
<td style="text-align:right;">
43.85238
</td>
<td style="text-align:left;">
positive
</td>
<td style="text-align:left;">
MGI_C57BL6J_5594322
</td>
<td style="text-align:left;">
Gm35163
</td>
<td style="text-align:left;">
NCBI_Gene:102638650
</td>
<td style="text-align:left;">
MGI:5594322
</td>
<td style="text-align:left;">
lncRNA gene
</td>
<td style="text-align:left;">
predicted gene, 35163
</td>
</tr>
<tr>
<td style="text-align:left;">
6
</td>
<td style="text-align:left;">
pseudogene
</td>
<td style="text-align:right;">
43.85608
</td>
<td style="text-align:right;">
43.85692
</td>
<td style="text-align:left;">
negative
</td>
<td style="text-align:left;">
MGI_C57BL6J_3643099
</td>
<td style="text-align:left;">
Gm7783
</td>
<td style="text-align:left;">
NCBI_Gene:665779,ENSEMBL:ENSMUSG00000107920
</td>
<td style="text-align:left;">
MGI:3643099
</td>
<td style="text-align:left;">
pseudogene
</td>
<td style="text-align:left;">
predicted gene 7783
</td>
</tr>
<tr>
<td style="text-align:left;">
6
</td>
<td style="text-align:left;">
gene
</td>
<td style="text-align:right;">
43.88719
</td>
<td style="text-align:right;">
43.88732
</td>
<td style="text-align:left;">
positive
</td>
<td style="text-align:left;">
MGI_C57BL6J_5452716
</td>
<td style="text-align:left;">
Gm22939
</td>
<td style="text-align:left;">
ENSEMBL:ENSMUSG00000077146
</td>
<td style="text-align:left;">
MGI:5452716
</td>
<td style="text-align:left;">
snoRNA gene
</td>
<td style="text-align:left;">
predicted gene, 22939
</td>
</tr>
<tr>
<td style="text-align:left;">
6
</td>
<td style="text-align:left;">
gene
</td>
<td style="text-align:right;">
44.08049
</td>
<td style="text-align:right;">
44.08338
</td>
<td style="text-align:left;">
negative
</td>
<td style="text-align:left;">
MGI_C57BL6J_5594375
</td>
<td style="text-align:left;">
Gm35216
</td>
<td style="text-align:left;">
NCBI_Gene:102638722,ENSEMBL:ENSMUSG00000107598
</td>
<td style="text-align:left;">
MGI:5594375
</td>
<td style="text-align:left;">
lncRNA gene
</td>
<td style="text-align:left;">
predicted gene, 35216
</td>
</tr>
<tr>
<td style="text-align:left;">
6
</td>
<td style="text-align:left;">
pseudogene
</td>
<td style="text-align:right;">
44.66729
</td>
<td style="text-align:right;">
44.66743
</td>
<td style="text-align:left;">
negative
</td>
<td style="text-align:left;">
MGI_C57BL6J_5690603
</td>
<td style="text-align:left;">
Gm44211
</td>
<td style="text-align:left;">
ENSEMBL:ENSMUSG00000107953
</td>
<td style="text-align:left;">
MGI:5690603
</td>
<td style="text-align:left;">
pseudogene
</td>
<td style="text-align:left;">
predicted gene, 44211
</td>
</tr>
<tr>
<td style="text-align:left;">
6
</td>
<td style="text-align:left;">
pseudogene
</td>
<td style="text-align:right;">
44.82157
</td>
<td style="text-align:right;">
44.82247
</td>
<td style="text-align:left;">
positive
</td>
<td style="text-align:left;">
MGI_C57BL6J_5690604
</td>
<td style="text-align:left;">
Gm44212
</td>
<td style="text-align:left;">
ENSEMBL:ENSMUSG00000107845
</td>
<td style="text-align:left;">
MGI:5690604
</td>
<td style="text-align:left;">
pseudogene
</td>
<td style="text-align:left;">
predicted gene, 44212
</td>
</tr>
<tr>
<td style="text-align:left;">
6
</td>
<td style="text-align:left;">
pseudogene
</td>
<td style="text-align:right;">
44.96592
</td>
<td style="text-align:right;">
44.96663
</td>
<td style="text-align:left;">
positive
</td>
<td style="text-align:left;">
MGI_C57BL6J_5690605
</td>
<td style="text-align:left;">
Gm44213
</td>
<td style="text-align:left;">
ENSEMBL:ENSMUSG00000107904
</td>
<td style="text-align:left;">
MGI:5690605
</td>
<td style="text-align:left;">
pseudogene
</td>
<td style="text-align:left;">
predicted gene, 44213
</td>
</tr>
<tr>
<td style="text-align:left;">
6
</td>
<td style="text-align:left;">
gene
</td>
<td style="text-align:right;">
45.05936
</td>
<td style="text-align:right;">
47.30421
</td>
<td style="text-align:left;">
positive
</td>
<td style="text-align:left;">
MGI_C57BL6J_1914047
</td>
<td style="text-align:left;">
Cntnap2
</td>
<td style="text-align:left;">
NCBI_Gene:66797,ENSEMBL:ENSMUSG00000039419
</td>
<td style="text-align:left;">
MGI:1914047
</td>
<td style="text-align:left;">
protein coding gene
</td>
<td style="text-align:left;">
contactin associated protein-like 2
</td>
</tr>
<tr>
<td style="text-align:left;">
6
</td>
<td style="text-align:left;">
gene
</td>
<td style="text-align:right;">
45.07357
</td>
<td style="text-align:right;">
45.07479
</td>
<td style="text-align:left;">
positive
</td>
<td style="text-align:left;">
MGI_C57BL6J_1925234
</td>
<td style="text-align:left;">
B230112I24Rik
</td>
<td style="text-align:left;">
ENSEMBL:ENSMUSG00000107729
</td>
<td style="text-align:left;">
MGI:1925234
</td>
<td style="text-align:left;">
unclassified gene
</td>
<td style="text-align:left;">
RIKEN cDNA B230112I24 gene
</td>
</tr>
<tr>
<td style="text-align:left;">
6
</td>
<td style="text-align:left;">
gene
</td>
<td style="text-align:right;">
45.08075
</td>
<td style="text-align:right;">
45.08451
</td>
<td style="text-align:left;">
positive
</td>
<td style="text-align:left;">
MGI_C57BL6J_5690271
</td>
<td style="text-align:left;">
Gm43879
</td>
<td style="text-align:left;">
ENSEMBL:ENSMUSG00000107858
</td>
<td style="text-align:left;">
MGI:5690271
</td>
<td style="text-align:left;">
unclassified gene
</td>
<td style="text-align:left;">
predicted gene, 43879
</td>
</tr>
<tr>
<td style="text-align:left;">
6
</td>
<td style="text-align:left;">
gene
</td>
<td style="text-align:right;">
45.13527
</td>
<td style="text-align:right;">
45.13748
</td>
<td style="text-align:left;">
positive
</td>
<td style="text-align:left;">
MGI_C57BL6J_5690264
</td>
<td style="text-align:left;">
Gm43872
</td>
<td style="text-align:left;">
ENSEMBL:ENSMUSG00000108070
</td>
<td style="text-align:left;">
MGI:5690264
</td>
<td style="text-align:left;">
unclassified gene
</td>
<td style="text-align:left;">
predicted gene, 43872
</td>
</tr>
<tr>
<td style="text-align:left;">
6
</td>
<td style="text-align:left;">
gene
</td>
<td style="text-align:right;">
45.19488
</td>
<td style="text-align:right;">
45.19742
</td>
<td style="text-align:left;">
positive
</td>
<td style="text-align:left;">
MGI_C57BL6J_5690277
</td>
<td style="text-align:left;">
Gm43885
</td>
<td style="text-align:left;">
ENSEMBL:ENSMUSG00000107643
</td>
<td style="text-align:left;">
MGI:5690277
</td>
<td style="text-align:left;">
unclassified gene
</td>
<td style="text-align:left;">
predicted gene, 43885
</td>
</tr>
<tr>
<td style="text-align:left;">
6
</td>
<td style="text-align:left;">
gene
</td>
<td style="text-align:right;">
45.20381
</td>
<td style="text-align:right;">
45.20749
</td>
<td style="text-align:left;">
positive
</td>
<td style="text-align:left;">
MGI_C57BL6J_5690281
</td>
<td style="text-align:left;">
Gm43889
</td>
<td style="text-align:left;">
ENSEMBL:ENSMUSG00000108146
</td>
<td style="text-align:left;">
MGI:5690281
</td>
<td style="text-align:left;">
unclassified gene
</td>
<td style="text-align:left;">
predicted gene, 43889
</td>
</tr>
<tr>
<td style="text-align:left;">
6
</td>
<td style="text-align:left;">
gene
</td>
<td style="text-align:right;">
45.22979
</td>
<td style="text-align:right;">
45.23377
</td>
<td style="text-align:left;">
positive
</td>
<td style="text-align:left;">
MGI_C57BL6J_2441759
</td>
<td style="text-align:left;">
9630039A02Rik
</td>
<td style="text-align:left;">
NA
</td>
<td style="text-align:left;">
NA
</td>
<td style="text-align:left;">
unclassified gene
</td>
<td style="text-align:left;">
RIKEN cDNA 9630039A02 gene
</td>
</tr>
<tr>
<td style="text-align:left;">
6
</td>
<td style="text-align:left;">
gene
</td>
<td style="text-align:right;">
45.24564
</td>
<td style="text-align:right;">
45.24843
</td>
<td style="text-align:left;">
positive
</td>
<td style="text-align:left;">
MGI_C57BL6J_5690266
</td>
<td style="text-align:left;">
Gm43874
</td>
<td style="text-align:left;">
ENSEMBL:ENSMUSG00000107779
</td>
<td style="text-align:left;">
MGI:5690266
</td>
<td style="text-align:left;">
unclassified gene
</td>
<td style="text-align:left;">
predicted gene, 43874
</td>
</tr>
<tr>
<td style="text-align:left;">
6
</td>
<td style="text-align:left;">
gene
</td>
<td style="text-align:right;">
45.30110
</td>
<td style="text-align:right;">
45.30251
</td>
<td style="text-align:left;">
positive
</td>
<td style="text-align:left;">
MGI_C57BL6J_5690261
</td>
<td style="text-align:left;">
Gm43869
</td>
<td style="text-align:left;">
ENSEMBL:ENSMUSG00000108008
</td>
<td style="text-align:left;">
MGI:5690261
</td>
<td style="text-align:left;">
unclassified gene
</td>
<td style="text-align:left;">
predicted gene, 43869
</td>
</tr>
<tr>
<td style="text-align:left;">
6
</td>
<td style="text-align:left;">
gene
</td>
<td style="text-align:right;">
45.30612
</td>
<td style="text-align:right;">
45.30739
</td>
<td style="text-align:left;">
positive
</td>
<td style="text-align:left;">
MGI_C57BL6J_1918013
</td>
<td style="text-align:left;">
6330419E04Rik
</td>
<td style="text-align:left;">
ENSEMBL:ENSMUSG00000107414
</td>
<td style="text-align:left;">
MGI:1918013
</td>
<td style="text-align:left;">
unclassified gene
</td>
<td style="text-align:left;">
RIKEN cDNA 6330419E04 gene
</td>
</tr>
<tr>
<td style="text-align:left;">
6
</td>
<td style="text-align:left;">
gene
</td>
<td style="text-align:right;">
45.32640
</td>
<td style="text-align:right;">
45.32867
</td>
<td style="text-align:left;">
positive
</td>
<td style="text-align:left;">
MGI_C57BL6J_5690274
</td>
<td style="text-align:left;">
Gm43882
</td>
<td style="text-align:left;">
ENSEMBL:ENSMUSG00000107433
</td>
<td style="text-align:left;">
MGI:5690274
</td>
<td style="text-align:left;">
unclassified gene
</td>
<td style="text-align:left;">
predicted gene, 43882
</td>
</tr>
<tr>
<td style="text-align:left;">
6
</td>
<td style="text-align:left;">
gene
</td>
<td style="text-align:right;">
45.32924
</td>
<td style="text-align:right;">
45.32980
</td>
<td style="text-align:left;">
positive
</td>
<td style="text-align:left;">
MGI_C57BL6J_5690276
</td>
<td style="text-align:left;">
Gm43884
</td>
<td style="text-align:left;">
ENSEMBL:ENSMUSG00000108123
</td>
<td style="text-align:left;">
MGI:5690276
</td>
<td style="text-align:left;">
unclassified gene
</td>
<td style="text-align:left;">
predicted gene, 43884
</td>
</tr>
<tr>
<td style="text-align:left;">
6
</td>
<td style="text-align:left;">
gene
</td>
<td style="text-align:right;">
45.41324
</td>
<td style="text-align:right;">
45.41540
</td>
<td style="text-align:left;">
positive
</td>
<td style="text-align:left;">
MGI_C57BL6J_5690272
</td>
<td style="text-align:left;">
Gm43880
</td>
<td style="text-align:left;">
ENSEMBL:ENSMUSG00000107886
</td>
<td style="text-align:left;">
MGI:5690272
</td>
<td style="text-align:left;">
unclassified gene
</td>
<td style="text-align:left;">
predicted gene, 43880
</td>
</tr>
<tr>
<td style="text-align:left;">
6
</td>
<td style="text-align:left;">
gene
</td>
<td style="text-align:right;">
45.52908
</td>
<td style="text-align:right;">
45.53231
</td>
<td style="text-align:left;">
positive
</td>
<td style="text-align:left;">
MGI_C57BL6J_5690268
</td>
<td style="text-align:left;">
Gm43876
</td>
<td style="text-align:left;">
ENSEMBL:ENSMUSG00000108055
</td>
<td style="text-align:left;">
MGI:5690268
</td>
<td style="text-align:left;">
unclassified gene
</td>
<td style="text-align:left;">
predicted gene, 43876
</td>
</tr>
<tr>
<td style="text-align:left;">
6
</td>
<td style="text-align:left;">
pseudogene
</td>
<td style="text-align:right;">
45.89447
</td>
<td style="text-align:right;">
45.89503
</td>
<td style="text-align:left;">
negative
</td>
<td style="text-align:left;">
MGI_C57BL6J_5010668
</td>
<td style="text-align:left;">
Gm18483
</td>
<td style="text-align:left;">
NCBI_Gene:100417253
</td>
<td style="text-align:left;">
MGI:5010668
</td>
<td style="text-align:left;">
pseudogene
</td>
<td style="text-align:left;">
predicted gene, 18483
</td>
</tr>
<tr>
<td style="text-align:left;">
6
</td>
<td style="text-align:left;">
gene
</td>
<td style="text-align:right;">
45.90194
</td>
<td style="text-align:right;">
45.90432
</td>
<td style="text-align:left;">
positive
</td>
<td style="text-align:left;">
MGI_C57BL6J_5690278
</td>
<td style="text-align:left;">
Gm43886
</td>
<td style="text-align:left;">
ENSEMBL:ENSMUSG00000107515
</td>
<td style="text-align:left;">
MGI:5690278
</td>
<td style="text-align:left;">
unclassified gene
</td>
<td style="text-align:left;">
predicted gene, 43886
</td>
</tr>
<tr>
<td style="text-align:left;">
6
</td>
<td style="text-align:left;">
gene
</td>
<td style="text-align:right;">
46.03778
</td>
<td style="text-align:right;">
46.03808
</td>
<td style="text-align:left;">
positive
</td>
<td style="text-align:left;">
MGI_C57BL6J_5452616
</td>
<td style="text-align:left;">
Gm22839
</td>
<td style="text-align:left;">
ENSEMBL:ENSMUSG00000089061
</td>
<td style="text-align:left;">
MGI:5452616
</td>
<td style="text-align:left;">
unclassified non-coding RNA gene
</td>
<td style="text-align:left;">
predicted gene, 22839
</td>
</tr>
<tr>
<td style="text-align:left;">
6
</td>
<td style="text-align:left;">
gene
</td>
<td style="text-align:right;">
46.04195
</td>
<td style="text-align:right;">
46.04391
</td>
<td style="text-align:left;">
positive
</td>
<td style="text-align:left;">
MGI_C57BL6J_5690263
</td>
<td style="text-align:left;">
Gm43871
</td>
<td style="text-align:left;">
ENSEMBL:ENSMUSG00000107913
</td>
<td style="text-align:left;">
MGI:5690263
</td>
<td style="text-align:left;">
unclassified gene
</td>
<td style="text-align:left;">
predicted gene, 43871
</td>
</tr>
<tr>
<td style="text-align:left;">
6
</td>
<td style="text-align:left;">
gene
</td>
<td style="text-align:right;">
46.04514
</td>
<td style="text-align:right;">
46.04908
</td>
<td style="text-align:left;">
positive
</td>
<td style="text-align:left;">
MGI_C57BL6J_5690265
</td>
<td style="text-align:left;">
Gm43873
</td>
<td style="text-align:left;">
ENSEMBL:ENSMUSG00000107606
</td>
<td style="text-align:left;">
MGI:5690265
</td>
<td style="text-align:left;">
unclassified gene
</td>
<td style="text-align:left;">
predicted gene, 43873
</td>
</tr>
<tr>
<td style="text-align:left;">
6
</td>
<td style="text-align:left;">
gene
</td>
<td style="text-align:right;">
46.04916
</td>
<td style="text-align:right;">
46.05139
</td>
<td style="text-align:left;">
positive
</td>
<td style="text-align:left;">
MGI_C57BL6J_5690269
</td>
<td style="text-align:left;">
Gm43877
</td>
<td style="text-align:left;">
ENSEMBL:ENSMUSG00000107571
</td>
<td style="text-align:left;">
MGI:5690269
</td>
<td style="text-align:left;">
unclassified gene
</td>
<td style="text-align:left;">
predicted gene, 43877
</td>
</tr>
<tr>
<td style="text-align:left;">
6
</td>
<td style="text-align:left;">
gene
</td>
<td style="text-align:right;">
46.07241
</td>
<td style="text-align:right;">
46.07500
</td>
<td style="text-align:left;">
positive
</td>
<td style="text-align:left;">
MGI_C57BL6J_5690280
</td>
<td style="text-align:left;">
Gm43888
</td>
<td style="text-align:left;">
ENSEMBL:ENSMUSG00000107782
</td>
<td style="text-align:left;">
MGI:5690280
</td>
<td style="text-align:left;">
unclassified gene
</td>
<td style="text-align:left;">
predicted gene, 43888
</td>
</tr>
<tr>
<td style="text-align:left;">
6
</td>
<td style="text-align:left;">
gene
</td>
<td style="text-align:right;">
46.07618
</td>
<td style="text-align:right;">
46.07936
</td>
<td style="text-align:left;">
positive
</td>
<td style="text-align:left;">
MGI_C57BL6J_5690279
</td>
<td style="text-align:left;">
Gm43887
</td>
<td style="text-align:left;">
ENSEMBL:ENSMUSG00000108189
</td>
<td style="text-align:left;">
MGI:5690279
</td>
<td style="text-align:left;">
unclassified gene
</td>
<td style="text-align:left;">
predicted gene, 43887
</td>
</tr>
<tr>
<td style="text-align:left;">
6
</td>
<td style="text-align:left;">
gene
</td>
<td style="text-align:right;">
46.13410
</td>
<td style="text-align:right;">
46.13847
</td>
<td style="text-align:left;">
positive
</td>
<td style="text-align:left;">
MGI_C57BL6J_5690262
</td>
<td style="text-align:left;">
Gm43870
</td>
<td style="text-align:left;">
ENSEMBL:ENSMUSG00000107712
</td>
<td style="text-align:left;">
MGI:5690262
</td>
<td style="text-align:left;">
unclassified gene
</td>
<td style="text-align:left;">
predicted gene, 43870
</td>
</tr>
<tr>
<td style="text-align:left;">
6
</td>
<td style="text-align:left;">
gene
</td>
<td style="text-align:right;">
46.41376
</td>
<td style="text-align:right;">
46.41629
</td>
<td style="text-align:left;">
positive
</td>
<td style="text-align:left;">
MGI_C57BL6J_5690273
</td>
<td style="text-align:left;">
Gm43881
</td>
<td style="text-align:left;">
ENSEMBL:ENSMUSG00000107442
</td>
<td style="text-align:left;">
MGI:5690273
</td>
<td style="text-align:left;">
unclassified gene
</td>
<td style="text-align:left;">
predicted gene, 43881
</td>
</tr>
<tr>
<td style="text-align:left;">
6
</td>
<td style="text-align:left;">
gene
</td>
<td style="text-align:right;">
46.48171
</td>
<td style="text-align:right;">
46.48184
</td>
<td style="text-align:left;">
negative
</td>
<td style="text-align:left;">
MGI_C57BL6J_5454938
</td>
<td style="text-align:left;">
Gm25161
</td>
<td style="text-align:left;">
ENSEMBL:ENSMUSG00000089538
</td>
<td style="text-align:left;">
MGI:5454938
</td>
<td style="text-align:left;">
snoRNA gene
</td>
<td style="text-align:left;">
predicted gene, 25161
</td>
</tr>
<tr>
<td style="text-align:left;">
6
</td>
<td style="text-align:left;">
gene
</td>
<td style="text-align:right;">
46.48664
</td>
<td style="text-align:right;">
46.48912
</td>
<td style="text-align:left;">
positive
</td>
<td style="text-align:left;">
MGI_C57BL6J_5690267
</td>
<td style="text-align:left;">
Gm43875
</td>
<td style="text-align:left;">
ENSEMBL:ENSMUSG00000107454
</td>
<td style="text-align:left;">
MGI:5690267
</td>
<td style="text-align:left;">
unclassified gene
</td>
<td style="text-align:left;">
predicted gene, 43875
</td>
</tr>
<tr>
<td style="text-align:left;">
6
</td>
<td style="text-align:left;">
pseudogene
</td>
<td style="text-align:right;">
46.52935
</td>
<td style="text-align:right;">
46.53079
</td>
<td style="text-align:left;">
negative
</td>
<td style="text-align:left;">
MGI_C57BL6J_96525
</td>
<td style="text-align:left;">
Rbpj-ps3
</td>
<td style="text-align:left;">
NCBI_Gene:19667,ENSEMBL:ENSMUSG00000079575
</td>
<td style="text-align:left;">
MGI:96525
</td>
<td style="text-align:left;">
pseudogene
</td>
<td style="text-align:left;">
recombination signal binding protein for immunoglobulin kappa J region, pseudogene 3
</td>
</tr>
<tr>
<td style="text-align:left;">
6
</td>
<td style="text-align:left;">
pseudogene
</td>
<td style="text-align:right;">
46.63725
</td>
<td style="text-align:right;">
46.63830
</td>
<td style="text-align:left;">
negative
</td>
<td style="text-align:left;">
MGI_C57BL6J_5010669
</td>
<td style="text-align:left;">
Gm18484
</td>
<td style="text-align:left;">
NCBI_Gene:100417254,ENSEMBL:ENSMUSG00000107725
</td>
<td style="text-align:left;">
MGI:5010669
</td>
<td style="text-align:left;">
pseudogene
</td>
<td style="text-align:left;">
predicted gene, 18484
</td>
</tr>
<tr>
<td style="text-align:left;">
6
</td>
<td style="text-align:left;">
pseudogene
</td>
<td style="text-align:right;">
46.77073
</td>
<td style="text-align:right;">
46.77965
</td>
<td style="text-align:left;">
negative
</td>
<td style="text-align:left;">
MGI_C57BL6J_5595090
</td>
<td style="text-align:left;">
Gm35931
</td>
<td style="text-align:left;">
NCBI_Gene:105242644,ENSEMBL:ENSMUSG00000107532
</td>
<td style="text-align:left;">
MGI:5595090
</td>
<td style="text-align:left;">
pseudogene
</td>
<td style="text-align:left;">
predicted gene, 35931
</td>
</tr>
<tr>
<td style="text-align:left;">
6
</td>
<td style="text-align:left;">
gene
</td>
<td style="text-align:right;">
46.78595
</td>
<td style="text-align:right;">
46.78970
</td>
<td style="text-align:left;">
positive
</td>
<td style="text-align:left;">
MGI_C57BL6J_5690275
</td>
<td style="text-align:left;">
Gm43883
</td>
<td style="text-align:left;">
ENSEMBL:ENSMUSG00000108217
</td>
<td style="text-align:left;">
MGI:5690275
</td>
<td style="text-align:left;">
unclassified gene
</td>
<td style="text-align:left;">
predicted gene, 43883
</td>
</tr>
<tr>
<td style="text-align:left;">
6
</td>
<td style="text-align:left;">
gene
</td>
<td style="text-align:right;">
46.95238
</td>
<td style="text-align:right;">
46.95823
</td>
<td style="text-align:left;">
negative
</td>
<td style="text-align:left;">
MGI_C57BL6J_5621686
</td>
<td style="text-align:left;">
Gm38801
</td>
<td style="text-align:left;">
NCBI_Gene:105242643
</td>
<td style="text-align:left;">
MGI:5621686
</td>
<td style="text-align:left;">
lncRNA gene
</td>
<td style="text-align:left;">
predicted gene, 38801
</td>
</tr>
<tr>
<td style="text-align:left;">
6
</td>
<td style="text-align:left;">
gene
</td>
<td style="text-align:right;">
47.00655
</td>
<td style="text-align:right;">
47.04080
</td>
<td style="text-align:left;">
negative
</td>
<td style="text-align:left;">
MGI_C57BL6J_5595034
</td>
<td style="text-align:left;">
Gm35875
</td>
<td style="text-align:left;">
NCBI_Gene:102639602
</td>
<td style="text-align:left;">
MGI:5595034
</td>
<td style="text-align:left;">
lncRNA gene
</td>
<td style="text-align:left;">
predicted gene, 35875
</td>
</tr>
<tr>
<td style="text-align:left;">
6
</td>
<td style="text-align:left;">
pseudogene
</td>
<td style="text-align:right;">
47.04516
</td>
<td style="text-align:right;">
47.04629
</td>
<td style="text-align:left;">
positive
</td>
<td style="text-align:left;">
MGI_C57BL6J_3647816
</td>
<td style="text-align:left;">
Gm6352
</td>
<td style="text-align:left;">
NCBI_Gene:622745,ENSEMBL:ENSMUSG00000107952
</td>
<td style="text-align:left;">
MGI:3647816
</td>
<td style="text-align:left;">
pseudogene
</td>
<td style="text-align:left;">
predicted gene 6352
</td>
</tr>
<tr>
<td style="text-align:left;">
6
</td>
<td style="text-align:left;">
gene
</td>
<td style="text-align:right;">
47.12001
</td>
<td style="text-align:right;">
47.12496
</td>
<td style="text-align:left;">
negative
</td>
<td style="text-align:left;">
MGI_C57BL6J_5594957
</td>
<td style="text-align:left;">
Gm35798
</td>
<td style="text-align:left;">
NCBI_Gene:102639497
</td>
<td style="text-align:left;">
MGI:5594957
</td>
<td style="text-align:left;">
lncRNA gene
</td>
<td style="text-align:left;">
predicted gene, 35798
</td>
</tr>
<tr>
<td style="text-align:left;">
6
</td>
<td style="text-align:left;">
gene
</td>
<td style="text-align:right;">
47.13921
</td>
<td style="text-align:right;">
47.16423
</td>
<td style="text-align:left;">
negative
</td>
<td style="text-align:left;">
MGI_C57BL6J_1915519
</td>
<td style="text-align:left;">
4930432J01Rik
</td>
<td style="text-align:left;">
NA
</td>
<td style="text-align:left;">
NA
</td>
<td style="text-align:left;">
unclassified non-coding RNA gene
</td>
<td style="text-align:left;">
RIKEN cDNA 4930432J01 gene
</td>
</tr>
<tr>
<td style="text-align:left;">
6
</td>
<td style="text-align:left;">
gene
</td>
<td style="text-align:right;">
47.19138
</td>
<td style="text-align:right;">
47.25916
</td>
<td style="text-align:left;">
negative
</td>
<td style="text-align:left;">
MGI_C57BL6J_5594439
</td>
<td style="text-align:left;">
Gm35280
</td>
<td style="text-align:left;">
NCBI_Gene:102638803
</td>
<td style="text-align:left;">
MGI:5594439
</td>
<td style="text-align:left;">
lncRNA gene
</td>
<td style="text-align:left;">
predicted gene, 35280
</td>
</tr>
<tr>
<td style="text-align:left;">
6
</td>
<td style="text-align:left;">
gene
</td>
<td style="text-align:right;">
47.22750
</td>
<td style="text-align:right;">
47.23058
</td>
<td style="text-align:left;">
positive
</td>
<td style="text-align:left;">
MGI_C57BL6J_5594724
</td>
<td style="text-align:left;">
Gm35565
</td>
<td style="text-align:left;">
NCBI_Gene:102639203
</td>
<td style="text-align:left;">
MGI:5594724
</td>
<td style="text-align:left;">
lncRNA gene
</td>
<td style="text-align:left;">
predicted gene, 35565
</td>
</tr>
<tr>
<td style="text-align:left;">
6
</td>
<td style="text-align:left;">
pseudogene
</td>
<td style="text-align:right;">
47.43212
</td>
<td style="text-align:right;">
47.43309
</td>
<td style="text-align:left;">
negative
</td>
<td style="text-align:left;">
MGI_C57BL6J_5010769
</td>
<td style="text-align:left;">
Gm18584
</td>
<td style="text-align:left;">
NCBI_Gene:100417389,ENSEMBL:ENSMUSG00000107853
</td>
<td style="text-align:left;">
MGI:5010769
</td>
<td style="text-align:left;">
pseudogene
</td>
<td style="text-align:left;">
predicted gene, 18584
</td>
</tr>
<tr>
<td style="text-align:left;">
6
</td>
<td style="text-align:left;">
gene
</td>
<td style="text-align:right;">
47.45340
</td>
<td style="text-align:right;">
47.52614
</td>
<td style="text-align:left;">
positive
</td>
<td style="text-align:left;">
MGI_C57BL6J_1349658
</td>
<td style="text-align:left;">
Cul1
</td>
<td style="text-align:left;">
NCBI_Gene:26965,ENSEMBL:ENSMUSG00000029686
</td>
<td style="text-align:left;">
MGI:1349658
</td>
<td style="text-align:left;">
protein coding gene
</td>
<td style="text-align:left;">
cullin 1
</td>
</tr>
<tr>
<td style="text-align:left;">
6
</td>
<td style="text-align:left;">
gene
</td>
<td style="text-align:right;">
47.45378
</td>
<td style="text-align:right;">
47.45432
</td>
<td style="text-align:left;">
negative
</td>
<td style="text-align:left;">
MGI_C57BL6J_2445106
</td>
<td style="text-align:left;">
A930035D04Rik
</td>
<td style="text-align:left;">
ENSEMBL:ENSMUSG00000052241
</td>
<td style="text-align:left;">
MGI:2445106
</td>
<td style="text-align:left;">
lncRNA gene
</td>
<td style="text-align:left;">
RIKEN cDNA A930035D04 gene
</td>
</tr>
<tr>
<td style="text-align:left;">
6
</td>
<td style="text-align:left;">
gene
</td>
<td style="text-align:right;">
47.53014
</td>
<td style="text-align:right;">
47.59534
</td>
<td style="text-align:left;">
negative
</td>
<td style="text-align:left;">
MGI_C57BL6J_107940
</td>
<td style="text-align:left;">
Ezh2
</td>
<td style="text-align:left;">
NCBI_Gene:14056,ENSEMBL:ENSMUSG00000029687
</td>
<td style="text-align:left;">
MGI:107940
</td>
<td style="text-align:left;">
protein coding gene
</td>
<td style="text-align:left;">
enhancer of zeste 2 polycomb repressive complex 2 subunit
</td>
</tr>
<tr>
<td style="text-align:left;">
6
</td>
<td style="text-align:left;">
gene
</td>
<td style="text-align:right;">
47.54667
</td>
<td style="text-align:right;">
47.55170
</td>
<td style="text-align:left;">
positive
</td>
<td style="text-align:left;">
MGI_C57BL6J_5477119
</td>
<td style="text-align:left;">
Gm26625
</td>
<td style="text-align:left;">
ENSEMBL:ENSMUSG00000097138
</td>
<td style="text-align:left;">
MGI:5477119
</td>
<td style="text-align:left;">
lncRNA gene
</td>
<td style="text-align:left;">
predicted gene, 26625
</td>
</tr>
<tr>
<td style="text-align:left;">
6
</td>
<td style="text-align:left;">
pseudogene
</td>
<td style="text-align:right;">
47.58522
</td>
<td style="text-align:right;">
47.58541
</td>
<td style="text-align:left;">
positive
</td>
<td style="text-align:left;">
MGI_C57BL6J_5690674
</td>
<td style="text-align:left;">
Gm44282
</td>
<td style="text-align:left;">
ENSEMBL:ENSMUSG00000107628
</td>
<td style="text-align:left;">
MGI:5690674
</td>
<td style="text-align:left;">
pseudogene
</td>
<td style="text-align:left;">
predicted gene, 44282
</td>
</tr>
<tr>
<td style="text-align:left;">
6
</td>
<td style="text-align:left;">
gene
</td>
<td style="text-align:right;">
47.59519
</td>
<td style="text-align:right;">
47.59753
</td>
<td style="text-align:left;">
positive
</td>
<td style="text-align:left;">
MGI_C57BL6J_2442048
</td>
<td style="text-align:left;">
F830010H11Rik
</td>
<td style="text-align:left;">
NA
</td>
<td style="text-align:left;">
NA
</td>
<td style="text-align:left;">
unclassified gene
</td>
<td style="text-align:left;">
RIKEN cDNA F830010H11 gene
</td>
</tr>
<tr>
<td style="text-align:left;">
6
</td>
<td style="text-align:left;">
gene
</td>
<td style="text-align:right;">
47.63530
</td>
<td style="text-align:right;">
47.63688
</td>
<td style="text-align:left;">
positive
</td>
<td style="text-align:left;">
MGI_C57BL6J_5690533
</td>
<td style="text-align:left;">
Gm44141
</td>
<td style="text-align:left;">
ENSEMBL:ENSMUSG00000108180
</td>
<td style="text-align:left;">
MGI:5690533
</td>
<td style="text-align:left;">
unclassified gene
</td>
<td style="text-align:left;">
predicted gene, 44141
</td>
</tr>
<tr>
<td style="text-align:left;">
6
</td>
<td style="text-align:left;">
gene
</td>
<td style="text-align:right;">
47.63714
</td>
<td style="text-align:right;">
47.63722
</td>
<td style="text-align:left;">
positive
</td>
<td style="text-align:left;">
MGI_C57BL6J_5455060
</td>
<td style="text-align:left;">
Gm25283
</td>
<td style="text-align:left;">
ENSEMBL:ENSMUSG00000087897
</td>
<td style="text-align:left;">
MGI:5455060
</td>
<td style="text-align:left;">
unclassified non-coding RNA gene
</td>
<td style="text-align:left;">
predicted gene, 25283
</td>
</tr>
<tr>
<td style="text-align:left;">
6
</td>
<td style="text-align:left;">
gene
</td>
<td style="text-align:right;">
47.64461
</td>
<td style="text-align:right;">
47.65185
</td>
<td style="text-align:left;">
negative
</td>
<td style="text-align:left;">
MGI_C57BL6J_5623256
</td>
<td style="text-align:left;">
Gm40371
</td>
<td style="text-align:left;">
NCBI_Gene:105244835
</td>
<td style="text-align:left;">
MGI:5623256
</td>
<td style="text-align:left;">
lncRNA gene
</td>
<td style="text-align:left;">
predicted gene, 40371
</td>
</tr>
<tr>
<td style="text-align:left;">
6
</td>
<td style="text-align:left;">
gene
</td>
<td style="text-align:right;">
47.65213
</td>
<td style="text-align:right;">
47.65612
</td>
<td style="text-align:left;">
positive
</td>
<td style="text-align:left;">
MGI_C57BL6J_5623254
</td>
<td style="text-align:left;">
Gm40369
</td>
<td style="text-align:left;">
NCBI_Gene:105244833
</td>
<td style="text-align:left;">
MGI:5623254
</td>
<td style="text-align:left;">
protein coding gene
</td>
<td style="text-align:left;">
predicted gene, 40369
</td>
</tr>
<tr>
<td style="text-align:left;">
6
</td>
<td style="text-align:left;">
gene
</td>
<td style="text-align:right;">
47.65492
</td>
<td style="text-align:right;">
47.65509
</td>
<td style="text-align:left;">
negative
</td>
<td style="text-align:left;">
MGI_C57BL6J_97946
</td>
<td style="text-align:left;">
Rn4.5s
</td>
<td style="text-align:left;">
NCBI_Gene:19799
</td>
<td style="text-align:left;">
MGI:97946
</td>
<td style="text-align:left;">
rRNA gene
</td>
<td style="text-align:left;">
4.5S RNA
</td>
</tr>
<tr>
<td style="text-align:left;">
6
</td>
<td style="text-align:left;">
gene
</td>
<td style="text-align:right;">
47.65637
</td>
<td style="text-align:right;">
47.66043
</td>
<td style="text-align:left;">
positive
</td>
<td style="text-align:left;">
MGI_C57BL6J_5623253
</td>
<td style="text-align:left;">
Gm40368
</td>
<td style="text-align:left;">
NCBI_Gene:105244832
</td>
<td style="text-align:left;">
MGI:5623253
</td>
<td style="text-align:left;">
protein coding gene
</td>
<td style="text-align:left;">
predicted gene, 40368
</td>
</tr>
<tr>
<td style="text-align:left;">
6
</td>
<td style="text-align:left;">
gene
</td>
<td style="text-align:right;">
47.66068
</td>
<td style="text-align:right;">
47.66479
</td>
<td style="text-align:left;">
positive
</td>
<td style="text-align:left;">
MGI_C57BL6J_5623252
</td>
<td style="text-align:left;">
Gm40367
</td>
<td style="text-align:left;">
NCBI_Gene:105244831
</td>
<td style="text-align:left;">
MGI:5623252
</td>
<td style="text-align:left;">
protein coding gene
</td>
<td style="text-align:left;">
predicted gene, 40367
</td>
</tr>
<tr>
<td style="text-align:left;">
6
</td>
<td style="text-align:left;">
gene
</td>
<td style="text-align:right;">
47.66504
</td>
<td style="text-align:right;">
47.66912
</td>
<td style="text-align:left;">
positive
</td>
<td style="text-align:left;">
MGI_C57BL6J_5595663
</td>
<td style="text-align:left;">
Gm36504
</td>
<td style="text-align:left;">
NCBI_Gene:102640444
</td>
<td style="text-align:left;">
MGI:5595663
</td>
<td style="text-align:left;">
protein coding gene
</td>
<td style="text-align:left;">
predicted gene, 36504
</td>
</tr>
<tr>
<td style="text-align:left;">
6
</td>
<td style="text-align:left;">
gene
</td>
<td style="text-align:right;">
47.66937
</td>
<td style="text-align:right;">
47.67345
</td>
<td style="text-align:left;">
positive
</td>
<td style="text-align:left;">
MGI_C57BL6J_5623250
</td>
<td style="text-align:left;">
Gm40365
</td>
<td style="text-align:left;">
NCBI_Gene:105244829
</td>
<td style="text-align:left;">
MGI:5623250
</td>
<td style="text-align:left;">
protein coding gene
</td>
<td style="text-align:left;">
predicted gene, 40365
</td>
</tr>
<tr>
<td style="text-align:left;">
6
</td>
<td style="text-align:left;">
gene
</td>
<td style="text-align:right;">
47.67369
</td>
<td style="text-align:right;">
47.67777
</td>
<td style="text-align:left;">
positive
</td>
<td style="text-align:left;">
MGI_C57BL6J_5595486
</td>
<td style="text-align:left;">
Gm36327
</td>
<td style="text-align:left;">
NCBI_Gene:102640203
</td>
<td style="text-align:left;">
MGI:5595486
</td>
<td style="text-align:left;">
protein coding gene
</td>
<td style="text-align:left;">
predicted gene, 36327
</td>
</tr>
<tr>
<td style="text-align:left;">
6
</td>
<td style="text-align:left;">
gene
</td>
<td style="text-align:right;">
47.67802
</td>
<td style="text-align:right;">
47.68209
</td>
<td style="text-align:left;">
positive
</td>
<td style="text-align:left;">
MGI_C57BL6J_5623249
</td>
<td style="text-align:left;">
Gm40364
</td>
<td style="text-align:left;">
NCBI_Gene:105244828
</td>
<td style="text-align:left;">
MGI:5623249
</td>
<td style="text-align:left;">
protein coding gene
</td>
<td style="text-align:left;">
predicted gene, 40364
</td>
</tr>
<tr>
<td style="text-align:left;">
6
</td>
<td style="text-align:left;">
gene
</td>
<td style="text-align:right;">
47.68234
</td>
<td style="text-align:right;">
47.73833
</td>
<td style="text-align:left;">
positive
</td>
<td style="text-align:left;">
MGI_C57BL6J_5623248
</td>
<td style="text-align:left;">
Gm40363
</td>
<td style="text-align:left;">
NCBI_Gene:105244826
</td>
<td style="text-align:left;">
MGI:5623248
</td>
<td style="text-align:left;">
protein coding gene
</td>
<td style="text-align:left;">
predicted gene, 40363
</td>
</tr>
<tr>
<td style="text-align:left;">
6
</td>
<td style="text-align:left;">
gene
</td>
<td style="text-align:right;">
47.76221
</td>
<td style="text-align:right;">
47.77309
</td>
<td style="text-align:left;">
positive
</td>
<td style="text-align:left;">
MGI_C57BL6J_5595832
</td>
<td style="text-align:left;">
Gm36673
</td>
<td style="text-align:left;">
NCBI_Gene:102640659
</td>
<td style="text-align:left;">
MGI:5595832
</td>
<td style="text-align:left;">
lncRNA gene
</td>
<td style="text-align:left;">
predicted gene, 36673
</td>
</tr>
<tr>
<td style="text-align:left;">
6
</td>
<td style="text-align:left;">
gene
</td>
<td style="text-align:right;">
47.78162
</td>
<td style="text-align:right;">
47.78173
</td>
<td style="text-align:left;">
positive
</td>
<td style="text-align:left;">
MGI_C57BL6J_107304
</td>
<td style="text-align:left;">
Rny3
</td>
<td style="text-align:left;">
NCBI_Gene:19874,ENSEMBL:ENSMUSG00000064945
</td>
<td style="text-align:left;">
MGI:107304
</td>
<td style="text-align:left;">
scRNA gene
</td>
<td style="text-align:left;">
RNA, Y3 small cytoplasmic (associated with Ro protein)
</td>
</tr>
<tr>
<td style="text-align:left;">
6
</td>
<td style="text-align:left;">
pseudogene
</td>
<td style="text-align:right;">
47.78596
</td>
<td style="text-align:right;">
47.78637
</td>
<td style="text-align:left;">
negative
</td>
<td style="text-align:left;">
MGI_C57BL6J_3648986
</td>
<td style="text-align:left;">
Rpl31-ps7
</td>
<td style="text-align:left;">
NCBI_Gene:384379,ENSEMBL:ENSMUSG00000097255
</td>
<td style="text-align:left;">
MGI:3648986
</td>
<td style="text-align:left;">
pseudogene
</td>
<td style="text-align:left;">
ribosomal protein L31, pseudogene 7
</td>
</tr>
<tr>
<td style="text-align:left;">
6
</td>
<td style="text-align:left;">
gene
</td>
<td style="text-align:right;">
47.78807
</td>
<td style="text-align:right;">
47.78818
</td>
<td style="text-align:left;">
negative
</td>
<td style="text-align:left;">
MGI_C57BL6J_97995
</td>
<td style="text-align:left;">
Rny1
</td>
<td style="text-align:left;">
NCBI_Gene:19872,ENSEMBL:ENSMUSG00000065701
</td>
<td style="text-align:left;">
MGI:97995
</td>
<td style="text-align:left;">
scRNA gene
</td>
<td style="text-align:left;">
RNA, Y1 small cytoplasmic, Ro-associated
</td>
</tr>
<tr>
<td style="text-align:left;">
6
</td>
<td style="text-align:left;">
gene
</td>
<td style="text-align:right;">
47.79609
</td>
<td style="text-align:right;">
47.81351
</td>
<td style="text-align:left;">
negative
</td>
<td style="text-align:left;">
MGI_C57BL6J_104864
</td>
<td style="text-align:left;">
Pdia4
</td>
<td style="text-align:left;">
NCBI_Gene:12304,ENSEMBL:ENSMUSG00000025823
</td>
<td style="text-align:left;">
MGI:104864
</td>
<td style="text-align:left;">
protein coding gene
</td>
<td style="text-align:left;">
protein disulfide isomerase associated 4
</td>
</tr>
<tr>
<td style="text-align:left;">
6
</td>
<td style="text-align:left;">
gene
</td>
<td style="text-align:right;">
47.80358
</td>
<td style="text-align:right;">
47.80365
</td>
<td style="text-align:left;">
negative
</td>
<td style="text-align:left;">
MGI_C57BL6J_3629672
</td>
<td style="text-align:left;">
Mir704
</td>
<td style="text-align:left;">
miRBase:MI0004688,NCBI_Gene:735289,ENSEMBL:ENSMUSG00000076060
</td>
<td style="text-align:left;">
MGI:3629672
</td>
<td style="text-align:left;">
miRNA gene
</td>
<td style="text-align:left;">
microRNA 704
</td>
</tr>
<tr>
<td style="text-align:left;">
6
</td>
<td style="text-align:left;">
pseudogene
</td>
<td style="text-align:right;">
47.81526
</td>
<td style="text-align:right;">
47.81569
</td>
<td style="text-align:left;">
negative
</td>
<td style="text-align:left;">
MGI_C57BL6J_3781346
</td>
<td style="text-align:left;">
Rpl35a-ps7
</td>
<td style="text-align:left;">
NCBI_Gene:100041154,ENSEMBL:ENSMUSG00000094557
</td>
<td style="text-align:left;">
MGI:3781346
</td>
<td style="text-align:left;">
pseudogene
</td>
<td style="text-align:left;">
ribosomal protein L35A, pseudogene 7
</td>
</tr>
<tr>
<td style="text-align:left;">
6
</td>
<td style="text-align:left;">
gene
</td>
<td style="text-align:right;">
47.81926
</td>
<td style="text-align:right;">
47.83556
</td>
<td style="text-align:left;">
negative
</td>
<td style="text-align:left;">
MGI_C57BL6J_3026883
</td>
<td style="text-align:left;">
Zfp786
</td>
<td style="text-align:left;">
NCBI_Gene:330301,ENSEMBL:ENSMUSG00000051499
</td>
<td style="text-align:left;">
MGI:3026883
</td>
<td style="text-align:left;">
protein coding gene
</td>
<td style="text-align:left;">
zinc finger protein 786
</td>
</tr>
<tr>
<td style="text-align:left;">
6
</td>
<td style="text-align:left;">
gene
</td>
<td style="text-align:right;">
47.83566
</td>
<td style="text-align:right;">
47.87354
</td>
<td style="text-align:left;">
positive
</td>
<td style="text-align:left;">
MGI_C57BL6J_1917856
</td>
<td style="text-align:left;">
Zfp398
</td>
<td style="text-align:left;">
NCBI_Gene:272347,ENSEMBL:ENSMUSG00000062519
</td>
<td style="text-align:left;">
MGI:1917856
</td>
<td style="text-align:left;">
protein coding gene
</td>
<td style="text-align:left;">
zinc finger protein 398
</td>
</tr>
<tr>
<td style="text-align:left;">
6
</td>
<td style="text-align:left;">
pseudogene
</td>
<td style="text-align:right;">
47.84774
</td>
<td style="text-align:right;">
47.84814
</td>
<td style="text-align:left;">
positive
</td>
<td style="text-align:left;">
MGI_C57BL6J_5690433
</td>
<td style="text-align:left;">
Gm44041
</td>
<td style="text-align:left;">
ENSEMBL:ENSMUSG00000107604
</td>
<td style="text-align:left;">
MGI:5690433
</td>
<td style="text-align:left;">
pseudogene
</td>
<td style="text-align:left;">
predicted gene, 44041
</td>
</tr>
<tr>
<td style="text-align:left;">
6
</td>
<td style="text-align:left;">
gene
</td>
<td style="text-align:right;">
47.87720
</td>
<td style="text-align:right;">
47.90848
</td>
<td style="text-align:left;">
positive
</td>
<td style="text-align:left;">
MGI_C57BL6J_2141413
</td>
<td style="text-align:left;">
Zfp282
</td>
<td style="text-align:left;">
NCBI_Gene:101095,ENSEMBL:ENSMUSG00000025821
</td>
<td style="text-align:left;">
MGI:2141413
</td>
<td style="text-align:left;">
protein coding gene
</td>
<td style="text-align:left;">
zinc finger protein 282
</td>
</tr>
<tr>
<td style="text-align:left;">
6
</td>
<td style="text-align:left;">
gene
</td>
<td style="text-align:right;">
47.92048
</td>
<td style="text-align:right;">
47.93264
</td>
<td style="text-align:left;">
positive
</td>
<td style="text-align:left;">
MGI_C57BL6J_2682609
</td>
<td style="text-align:left;">
Zfp212
</td>
<td style="text-align:left;">
NCBI_Gene:232784,ENSEMBL:ENSMUSG00000052763
</td>
<td style="text-align:left;">
MGI:2682609
</td>
<td style="text-align:left;">
protein coding gene
</td>
<td style="text-align:left;">
Zinc finger protein 212
</td>
</tr>
<tr>
<td style="text-align:left;">
6
</td>
<td style="text-align:left;">
pseudogene
</td>
<td style="text-align:right;">
47.94317
</td>
<td style="text-align:right;">
47.95591
</td>
<td style="text-align:left;">
positive
</td>
<td style="text-align:left;">
MGI_C57BL6J_3040704
</td>
<td style="text-align:left;">
Zfp783
</td>
<td style="text-align:left;">
NCBI_Gene:232785,ENSEMBL:ENSMUSG00000072653
</td>
<td style="text-align:left;">
MGI:3040704
</td>
<td style="text-align:left;">
pseudogene
</td>
<td style="text-align:left;">
zinc finger protein 783
</td>
</tr>
<tr>
<td style="text-align:left;">
6
</td>
<td style="text-align:left;">
gene
</td>
<td style="text-align:right;">
47.95339
</td>
<td style="text-align:right;">
47.96530
</td>
<td style="text-align:left;">
positive
</td>
<td style="text-align:left;">
MGI_C57BL6J_2141515
</td>
<td style="text-align:left;">
Zfp956
</td>
<td style="text-align:left;">
NCBI_Gene:101197,ENSEMBL:ENSMUSG00000045466
</td>
<td style="text-align:left;">
MGI:2141515
</td>
<td style="text-align:left;">
protein coding gene
</td>
<td style="text-align:left;">
zinc finger protein 956
</td>
</tr>
<tr>
<td style="text-align:left;">
6
</td>
<td style="text-align:left;">
pseudogene
</td>
<td style="text-align:right;">
47.97699
</td>
<td style="text-align:right;">
47.97790
</td>
<td style="text-align:left;">
negative
</td>
<td style="text-align:left;">
MGI_C57BL6J_3646734
</td>
<td style="text-align:left;">
Gm6443
</td>
<td style="text-align:left;">
NCBI_Gene:623601,ENSEMBL:ENSMUSG00000107824
</td>
<td style="text-align:left;">
MGI:3646734
</td>
<td style="text-align:left;">
pseudogene
</td>
<td style="text-align:left;">
predicted gene 6443
</td>
</tr>
<tr>
<td style="text-align:left;">
6
</td>
<td style="text-align:left;">
gene
</td>
<td style="text-align:right;">
47.97900
</td>
<td style="text-align:right;">
47.97907
</td>
<td style="text-align:left;">
positive
</td>
<td style="text-align:left;">
MGI_C57BL6J_4413824
</td>
<td style="text-align:left;">
n-TCgca57
</td>
<td style="text-align:left;">
NCBI_Gene:102467643
</td>
<td style="text-align:left;">
MGI:4413824
</td>
<td style="text-align:left;">
tRNA gene
</td>
<td style="text-align:left;">
nuclear encoded tRNA cysteine 57 (anticodon GCA)
</td>
</tr>
<tr>
<td style="text-align:left;">
6
</td>
<td style="text-align:left;">
gene
</td>
<td style="text-align:right;">
48.01766
</td>
<td style="text-align:right;">
48.05000
</td>
<td style="text-align:left;">
negative
</td>
<td style="text-align:left;">
MGI_C57BL6J_1919556
</td>
<td style="text-align:left;">
Zfp777
</td>
<td style="text-align:left;">
NCBI_Gene:72306,ENSEMBL:ENSMUSG00000071477
</td>
<td style="text-align:left;">
MGI:1919556
</td>
<td style="text-align:left;">
protein coding gene
</td>
<td style="text-align:left;">
zinc finger protein 777
</td>
</tr>
<tr>
<td style="text-align:left;">
6
</td>
<td style="text-align:left;">
gene
</td>
<td style="text-align:right;">
48.04555
</td>
<td style="text-align:right;">
48.04595
</td>
<td style="text-align:left;">
negative
</td>
<td style="text-align:left;">
MGI_C57BL6J_5804853
</td>
<td style="text-align:left;">
Gm45738
</td>
<td style="text-align:left;">
ENSEMBL:ENSMUSG00000108101
</td>
<td style="text-align:left;">
MGI:5804853
</td>
<td style="text-align:left;">
lncRNA gene
</td>
<td style="text-align:left;">
predicted gene 45738
</td>
</tr>
<tr>
<td style="text-align:left;">
6
</td>
<td style="text-align:left;">
gene
</td>
<td style="text-align:right;">
48.04870
</td>
<td style="text-align:right;">
48.04879
</td>
<td style="text-align:left;">
positive
</td>
<td style="text-align:left;">
MGI_C57BL6J_5454340
</td>
<td style="text-align:left;">
Gm24563
</td>
<td style="text-align:left;">
ENSEMBL:ENSMUSG00000092928
</td>
<td style="text-align:left;">
MGI:5454340
</td>
<td style="text-align:left;">
miRNA gene
</td>
<td style="text-align:left;">
predicted gene, 24563
</td>
</tr>
<tr>
<td style="text-align:left;">
6
</td>
<td style="text-align:left;">
gene
</td>
<td style="text-align:right;">
48.06239
</td>
<td style="text-align:right;">
48.08669
</td>
<td style="text-align:left;">
negative
</td>
<td style="text-align:left;">
MGI_C57BL6J_1916478
</td>
<td style="text-align:left;">
Zfp746
</td>
<td style="text-align:left;">
NCBI_Gene:69228,ENSEMBL:ENSMUSG00000057691
</td>
<td style="text-align:left;">
MGI:1916478
</td>
<td style="text-align:left;">
protein coding gene
</td>
<td style="text-align:left;">
zinc finger protein 746
</td>
</tr>
<tr>
<td style="text-align:left;">
6
</td>
<td style="text-align:left;">
gene
</td>
<td style="text-align:right;">
48.06595
</td>
<td style="text-align:right;">
48.06605
</td>
<td style="text-align:left;">
positive
</td>
<td style="text-align:left;">
MGI_C57BL6J_3629706
</td>
<td style="text-align:left;">
Gt(pU21)138Imeg
</td>
<td style="text-align:left;">
NA
</td>
<td style="text-align:left;">
NA
</td>
<td style="text-align:left;">
unclassified gene
</td>
<td style="text-align:left;">
gene trap 138%2c Institute of Molecular Embryology and Genetics
</td>
</tr>
<tr>
<td style="text-align:left;">
6
</td>
<td style="text-align:left;">
gene
</td>
<td style="text-align:right;">
48.12866
</td>
<td style="text-align:right;">
48.39541
</td>
<td style="text-align:left;">
negative
</td>
<td style="text-align:left;">
MGI_C57BL6J_4439554
</td>
<td style="text-align:left;">
Gm16630
</td>
<td style="text-align:left;">
NCBI_Gene:100503622,ENSEMBL:ENSMUSG00000097434
</td>
<td style="text-align:left;">
MGI:4439554
</td>
<td style="text-align:left;">
lncRNA gene
</td>
<td style="text-align:left;">
predicted gene, 16630
</td>
</tr>
<tr>
<td style="text-align:left;">
6
</td>
<td style="text-align:left;">
gene
</td>
<td style="text-align:right;">
48.13047
</td>
<td style="text-align:right;">
48.14608
</td>
<td style="text-align:left;">
positive
</td>
<td style="text-align:left;">
MGI_C57BL6J_5589594
</td>
<td style="text-align:left;">
Gm30435
</td>
<td style="text-align:left;">
NCBI_Gene:102632336
</td>
<td style="text-align:left;">
MGI:5589594
</td>
<td style="text-align:left;">
lncRNA gene
</td>
<td style="text-align:left;">
predicted gene, 30435
</td>
</tr>
<tr>
<td style="text-align:left;">
6
</td>
<td style="text-align:left;">
gene
</td>
<td style="text-align:right;">
48.13394
</td>
<td style="text-align:right;">
48.13401
</td>
<td style="text-align:left;">
positive
</td>
<td style="text-align:left;">
MGI_C57BL6J_4413804
</td>
<td style="text-align:left;">
n-TCgca37
</td>
<td style="text-align:left;">
NCBI_Gene:102467630
</td>
<td style="text-align:left;">
MGI:4413804
</td>
<td style="text-align:left;">
tRNA gene
</td>
<td style="text-align:left;">
nuclear encoded tRNA cysteine 37 (anticodon GCA)
</td>
</tr>
<tr>
<td style="text-align:left;">
6
</td>
<td style="text-align:left;">
gene
</td>
<td style="text-align:right;">
48.13951
</td>
<td style="text-align:right;">
48.13958
</td>
<td style="text-align:left;">
negative
</td>
<td style="text-align:left;">
MGI_C57BL6J_4413777
</td>
<td style="text-align:left;">
n-TCgca10
</td>
<td style="text-align:left;">
NCBI_Gene:102467491
</td>
<td style="text-align:left;">
MGI:4413777
</td>
<td style="text-align:left;">
tRNA gene
</td>
<td style="text-align:left;">
nuclear encoded tRNA cysteine 10 (anticodon GCA)
</td>
</tr>
<tr>
<td style="text-align:left;">
6
</td>
<td style="text-align:left;">
gene
</td>
<td style="text-align:right;">
48.14048
</td>
<td style="text-align:right;">
48.14055
</td>
<td style="text-align:left;">
negative
</td>
<td style="text-align:left;">
MGI_C57BL6J_4413771
</td>
<td style="text-align:left;">
n-TCgca4
</td>
<td style="text-align:left;">
NCBI_Gene:102467488
</td>
<td style="text-align:left;">
MGI:4413771
</td>
<td style="text-align:left;">
tRNA gene
</td>
<td style="text-align:left;">
nuclear encoded tRNA cysteine 4 (anticodon GCA)
</td>
</tr>
<tr>
<td style="text-align:left;">
6
</td>
<td style="text-align:left;">
gene
</td>
<td style="text-align:right;">
48.14336
</td>
<td style="text-align:right;">
48.14343
</td>
<td style="text-align:left;">
positive
</td>
<td style="text-align:left;">
MGI_C57BL6J_4413780
</td>
<td style="text-align:left;">
n-TCgca13
</td>
<td style="text-align:left;">
NCBI_Gene:102467492
</td>
<td style="text-align:left;">
MGI:4413780
</td>
<td style="text-align:left;">
tRNA gene
</td>
<td style="text-align:left;">
nuclear encoded tRNA cysteine 13 (anticodon GCA)
</td>
</tr>
<tr>
<td style="text-align:left;">
6
</td>
<td style="text-align:left;">
gene
</td>
<td style="text-align:right;">
48.14493
</td>
<td style="text-align:right;">
48.14500
</td>
<td style="text-align:left;">
negative
</td>
<td style="text-align:left;">
MGI_C57BL6J_4413803
</td>
<td style="text-align:left;">
n-TCgca36
</td>
<td style="text-align:left;">
NCBI_Gene:102467282
</td>
<td style="text-align:left;">
MGI:4413803
</td>
<td style="text-align:left;">
tRNA gene
</td>
<td style="text-align:left;">
nuclear encoded tRNA cysteine 36 (anticodon GCA)
</td>
</tr>
<tr>
<td style="text-align:left;">
6
</td>
<td style="text-align:left;">
gene
</td>
<td style="text-align:right;">
48.15471
</td>
<td style="text-align:right;">
48.15478
</td>
<td style="text-align:left;">
positive
</td>
<td style="text-align:left;">
MGI_C57BL6J_4413805
</td>
<td style="text-align:left;">
n-TCgca38
</td>
<td style="text-align:left;">
NCBI_Gene:102467502
</td>
<td style="text-align:left;">
MGI:4413805
</td>
<td style="text-align:left;">
tRNA gene
</td>
<td style="text-align:left;">
nuclear encoded tRNA cysteine 38 (anticodon GCA)
</td>
</tr>
<tr>
<td style="text-align:left;">
6
</td>
<td style="text-align:left;">
gene
</td>
<td style="text-align:right;">
48.15628
</td>
<td style="text-align:right;">
48.15635
</td>
<td style="text-align:left;">
negative
</td>
<td style="text-align:left;">
MGI_C57BL6J_4413802
</td>
<td style="text-align:left;">
n-TCgca35
</td>
<td style="text-align:left;">
NCBI_Gene:102467501
</td>
<td style="text-align:left;">
MGI:4413802
</td>
<td style="text-align:left;">
tRNA gene
</td>
<td style="text-align:left;">
nuclear encoded tRNA cysteine 35 (anticodon GCA)
</td>
</tr>
<tr>
<td style="text-align:left;">
6
</td>
<td style="text-align:left;">
gene
</td>
<td style="text-align:right;">
48.16018
</td>
<td style="text-align:right;">
48.16025
</td>
<td style="text-align:left;">
positive
</td>
<td style="text-align:left;">
MGI_C57BL6J_4413806
</td>
<td style="text-align:left;">
n-TCgca39
</td>
<td style="text-align:left;">
NCBI_Gene:102467642
</td>
<td style="text-align:left;">
MGI:4413806
</td>
<td style="text-align:left;">
tRNA gene
</td>
<td style="text-align:left;">
nuclear encoded tRNA cysteine 39 (anticodon GCA)
</td>
</tr>
<tr>
<td style="text-align:left;">
6
</td>
<td style="text-align:left;">
gene
</td>
<td style="text-align:right;">
48.16410
</td>
<td style="text-align:right;">
48.16417
</td>
<td style="text-align:left;">
positive
</td>
<td style="text-align:left;">
MGI_C57BL6J_4413807
</td>
<td style="text-align:left;">
n-TCgca40
</td>
<td style="text-align:left;">
NCBI_Gene:109705
</td>
<td style="text-align:left;">
MGI:4413807
</td>
<td style="text-align:left;">
tRNA gene
</td>
<td style="text-align:left;">
nuclear encoded tRNA cysteine 40 (anticodon GCA)
</td>
</tr>
<tr>
<td style="text-align:left;">
6
</td>
<td style="text-align:left;">
gene
</td>
<td style="text-align:right;">
48.18632
</td>
<td style="text-align:right;">
48.18639
</td>
<td style="text-align:left;">
positive
</td>
<td style="text-align:left;">
MGI_C57BL6J_4413775
</td>
<td style="text-align:left;">
n-TCgca8
</td>
<td style="text-align:left;">
NCBI_Gene:102467490
</td>
<td style="text-align:left;">
MGI:4413775
</td>
<td style="text-align:left;">
tRNA gene
</td>
<td style="text-align:left;">
nuclear encoded tRNA cysteine 8 (anticodon GCA)
</td>
</tr>
<tr>
<td style="text-align:left;">
6
</td>
<td style="text-align:left;">
gene
</td>
<td style="text-align:right;">
48.19703
</td>
<td style="text-align:right;">
48.19710
</td>
<td style="text-align:left;">
positive
</td>
<td style="text-align:left;">
MGI_C57BL6J_4413808
</td>
<td style="text-align:left;">
n-TCgca41
</td>
<td style="text-align:left;">
NCBI_Gene:102467283
</td>
<td style="text-align:left;">
MGI:4413808
</td>
<td style="text-align:left;">
tRNA gene
</td>
<td style="text-align:left;">
nuclear encoded tRNA cysteine 41 (anticodon GCA)
</td>
</tr>
<tr>
<td style="text-align:left;">
6
</td>
<td style="text-align:left;">
gene
</td>
<td style="text-align:right;">
48.20507
</td>
<td style="text-align:right;">
48.20514
</td>
<td style="text-align:left;">
positive
</td>
<td style="text-align:left;">
MGI_C57BL6J_4413768
</td>
<td style="text-align:left;">
n-TCgca1
</td>
<td style="text-align:left;">
NCBI_Gene:102467487
</td>
<td style="text-align:left;">
MGI:4413768
</td>
<td style="text-align:left;">
tRNA gene
</td>
<td style="text-align:left;">
nuclear encoded tRNA cysteine 1 (anticodon GCA)
</td>
</tr>
<tr>
<td style="text-align:left;">
6
</td>
<td style="text-align:left;">
gene
</td>
<td style="text-align:right;">
48.20745
</td>
<td style="text-align:right;">
48.20752
</td>
<td style="text-align:left;">
negative
</td>
<td style="text-align:left;">
MGI_C57BL6J_4413801
</td>
<td style="text-align:left;">
n-TCgca34
</td>
<td style="text-align:left;">
NCBI_Gene:102467281
</td>
<td style="text-align:left;">
MGI:4413801
</td>
<td style="text-align:left;">
tRNA gene
</td>
<td style="text-align:left;">
nuclear encoded tRNA cysteine 34 (anticodon GCA)
</td>
</tr>
<tr>
<td style="text-align:left;">
6
</td>
<td style="text-align:left;">
gene
</td>
<td style="text-align:right;">
48.20984
</td>
<td style="text-align:right;">
48.20991
</td>
<td style="text-align:left;">
negative
</td>
<td style="text-align:left;">
MGI_C57BL6J_4413816
</td>
<td style="text-align:left;">
n-TCgca49
</td>
<td style="text-align:left;">
NCBI_Gene:102467286
</td>
<td style="text-align:left;">
MGI:4413816
</td>
<td style="text-align:left;">
tRNA gene
</td>
<td style="text-align:left;">
nuclear encoded tRNA cysteine 49 (anticodon GCA)
</td>
</tr>
<tr>
<td style="text-align:left;">
6
</td>
<td style="text-align:left;">
gene
</td>
<td style="text-align:right;">
48.22218
</td>
<td style="text-align:right;">
48.24739
</td>
<td style="text-align:left;">
positive
</td>
<td style="text-align:left;">
MGI_C57BL6J_5621687
</td>
<td style="text-align:left;">
Gm38802
</td>
<td style="text-align:left;">
NCBI_Gene:105242646
</td>
<td style="text-align:left;">
MGI:5621687
</td>
<td style="text-align:left;">
lncRNA gene
</td>
<td style="text-align:left;">
predicted gene, 38802
</td>
</tr>
<tr>
<td style="text-align:left;">
6
</td>
<td style="text-align:left;">
gene
</td>
<td style="text-align:right;">
48.22309
</td>
<td style="text-align:right;">
48.22317
</td>
<td style="text-align:left;">
negative
</td>
<td style="text-align:left;">
MGI_C57BL6J_4413800
</td>
<td style="text-align:left;">
n-TCgca33
</td>
<td style="text-align:left;">
NCBI_Gene:102467500
</td>
<td style="text-align:left;">
MGI:4413800
</td>
<td style="text-align:left;">
tRNA gene
</td>
<td style="text-align:left;">
nuclear encoded tRNA cysteine 33 (anticodon GCA)
</td>
</tr>
<tr>
<td style="text-align:left;">
6
</td>
<td style="text-align:left;">
gene
</td>
<td style="text-align:right;">
48.22433
</td>
<td style="text-align:right;">
48.22440
</td>
<td style="text-align:left;">
negative
</td>
<td style="text-align:left;">
MGI_C57BL6J_4413815
</td>
<td style="text-align:left;">
n-TCgca48
</td>
<td style="text-align:left;">
NCBI_Gene:102467426
</td>
<td style="text-align:left;">
MGI:4413815
</td>
<td style="text-align:left;">
tRNA gene
</td>
<td style="text-align:left;">
nuclear encoded tRNA cysteine 48 (anticodon GCA)
</td>
</tr>
<tr>
<td style="text-align:left;">
6
</td>
<td style="text-align:left;">
gene
</td>
<td style="text-align:right;">
48.24432
</td>
<td style="text-align:right;">
48.24439
</td>
<td style="text-align:left;">
negative
</td>
<td style="text-align:left;">
MGI_C57BL6J_4413799
</td>
<td style="text-align:left;">
n-TCgca32
</td>
<td style="text-align:left;">
NCBI_Gene:102467280
</td>
<td style="text-align:left;">
MGI:4413799
</td>
<td style="text-align:left;">
tRNA gene
</td>
<td style="text-align:left;">
nuclear encoded tRNA cysteine 32 (anticodon GCA)
</td>
</tr>
<tr>
<td style="text-align:left;">
6
</td>
<td style="text-align:left;">
gene
</td>
<td style="text-align:right;">
48.25210
</td>
<td style="text-align:right;">
48.25217
</td>
<td style="text-align:left;">
negative
</td>
<td style="text-align:left;">
MGI_C57BL6J_4413798
</td>
<td style="text-align:left;">
n-TCgca31
</td>
<td style="text-align:left;">
NCBI_Gene:102467499
</td>
<td style="text-align:left;">
MGI:4413798
</td>
<td style="text-align:left;">
tRNA gene
</td>
<td style="text-align:left;">
nuclear encoded tRNA cysteine 31 (anticodon GCA)
</td>
</tr>
<tr>
<td style="text-align:left;">
6
</td>
<td style="text-align:left;">
gene
</td>
<td style="text-align:right;">
48.25308
</td>
<td style="text-align:right;">
48.25315
</td>
<td style="text-align:left;">
negative
</td>
<td style="text-align:left;">
MGI_C57BL6J_4413782
</td>
<td style="text-align:left;">
n-TCgca15
</td>
<td style="text-align:left;">
NCBI_Gene:102467412
</td>
<td style="text-align:left;">
MGI:4413782
</td>
<td style="text-align:left;">
tRNA gene
</td>
<td style="text-align:left;">
nuclear encoded tRNA cysteine 15 (anticodon GCA)
</td>
</tr>
<tr>
<td style="text-align:left;">
6
</td>
<td style="text-align:left;">
gene
</td>
<td style="text-align:right;">
48.26172
</td>
<td style="text-align:right;">
48.26179
</td>
<td style="text-align:left;">
negative
</td>
<td style="text-align:left;">
MGI_C57BL6J_4413797
</td>
<td style="text-align:left;">
n-TCgca30
</td>
<td style="text-align:left;">
NCBI_Gene:102467425
</td>
<td style="text-align:left;">
MGI:4413797
</td>
<td style="text-align:left;">
tRNA gene
</td>
<td style="text-align:left;">
nuclear encoded tRNA cysteine 30 (anticodon GCA)
</td>
</tr>
<tr>
<td style="text-align:left;">
6
</td>
<td style="text-align:left;">
gene
</td>
<td style="text-align:right;">
48.26846
</td>
<td style="text-align:right;">
48.26853
</td>
<td style="text-align:left;">
negative
</td>
<td style="text-align:left;">
MGI_C57BL6J_4413796
</td>
<td style="text-align:left;">
n-TCgca29
</td>
<td style="text-align:left;">
NCBI_Gene:102467279
</td>
<td style="text-align:left;">
MGI:4413796
</td>
<td style="text-align:left;">
tRNA gene
</td>
<td style="text-align:left;">
nuclear encoded tRNA cysteine 29 (anticodon GCA)
</td>
</tr>
<tr>
<td style="text-align:left;">
6
</td>
<td style="text-align:left;">
gene
</td>
<td style="text-align:right;">
48.27193
</td>
<td style="text-align:right;">
48.27200
</td>
<td style="text-align:left;">
positive
</td>
<td style="text-align:left;">
MGI_C57BL6J_4413809
</td>
<td style="text-align:left;">
n-TCgca42
</td>
<td style="text-align:left;">
NCBI_Gene:102467503
</td>
<td style="text-align:left;">
MGI:4413809
</td>
<td style="text-align:left;">
tRNA gene
</td>
<td style="text-align:left;">
nuclear encoded tRNA cysteine 42 (anticodon GCA)
</td>
</tr>
<tr>
<td style="text-align:left;">
6
</td>
<td style="text-align:left;">
gene
</td>
<td style="text-align:right;">
48.27368
</td>
<td style="text-align:right;">
48.27375
</td>
<td style="text-align:left;">
positive
</td>
<td style="text-align:left;">
MGI_C57BL6J_4413810
</td>
<td style="text-align:left;">
n-TCgca43
</td>
<td style="text-align:left;">
NCBI_Gene:102467504
</td>
<td style="text-align:left;">
MGI:4413810
</td>
<td style="text-align:left;">
tRNA gene
</td>
<td style="text-align:left;">
nuclear encoded tRNA cysteine 43 (anticodon GCA)
</td>
</tr>
<tr>
<td style="text-align:left;">
6
</td>
<td style="text-align:left;">
gene
</td>
<td style="text-align:right;">
48.27524
</td>
<td style="text-align:right;">
48.33816
</td>
<td style="text-align:left;">
positive
</td>
<td style="text-align:left;">
MGI_C57BL6J_5589521
</td>
<td style="text-align:left;">
Gm30362
</td>
<td style="text-align:left;">
NCBI_Gene:102632227
</td>
<td style="text-align:left;">
MGI:5589521
</td>
<td style="text-align:left;">
lncRNA gene
</td>
<td style="text-align:left;">
predicted gene, 30362
</td>
</tr>
<tr>
<td style="text-align:left;">
6
</td>
<td style="text-align:left;">
gene
</td>
<td style="text-align:right;">
48.27879
</td>
<td style="text-align:right;">
48.27886
</td>
<td style="text-align:left;">
negative
</td>
<td style="text-align:left;">
MGI_C57BL6J_4413770
</td>
<td style="text-align:left;">
n-TCgca3
</td>
<td style="text-align:left;">
NCBI_Gene:102467640
</td>
<td style="text-align:left;">
MGI:4413770
</td>
<td style="text-align:left;">
tRNA gene
</td>
<td style="text-align:left;">
nuclear encoded tRNA cysteine 3 (anticodon GCA)
</td>
</tr>
<tr>
<td style="text-align:left;">
6
</td>
<td style="text-align:left;">
gene
</td>
<td style="text-align:right;">
48.27977
</td>
<td style="text-align:right;">
48.27984
</td>
<td style="text-align:left;">
negative
</td>
<td style="text-align:left;">
MGI_C57BL6J_4413795
</td>
<td style="text-align:left;">
n-TCgca28
</td>
<td style="text-align:left;">
NCBI_Gene:102467498
</td>
<td style="text-align:left;">
MGI:4413795
</td>
<td style="text-align:left;">
tRNA gene
</td>
<td style="text-align:left;">
nuclear encoded tRNA cysteine 28 (anticodon GCA)
</td>
</tr>
<tr>
<td style="text-align:left;">
6
</td>
<td style="text-align:left;">
gene
</td>
<td style="text-align:right;">
48.28888
</td>
<td style="text-align:right;">
48.28895
</td>
<td style="text-align:left;">
negative
</td>
<td style="text-align:left;">
MGI_C57BL6J_4413779
</td>
<td style="text-align:left;">
n-TCgca12
</td>
<td style="text-align:left;">
NCBI_Gene:102467424
</td>
<td style="text-align:left;">
MGI:4413779
</td>
<td style="text-align:left;">
tRNA gene
</td>
<td style="text-align:left;">
nuclear encoded tRNA cysteine 12 (anticodon GCA)
</td>
</tr>
<tr>
<td style="text-align:left;">
6
</td>
<td style="text-align:left;">
gene
</td>
<td style="text-align:right;">
48.28923
</td>
<td style="text-align:right;">
48.28943
</td>
<td style="text-align:left;">
positive
</td>
<td style="text-align:left;">
MGI_C57BL6J_5451899
</td>
<td style="text-align:left;">
Gm22122
</td>
<td style="text-align:left;">
ENSEMBL:ENSMUSG00000092888
</td>
<td style="text-align:left;">
MGI:5451899
</td>
<td style="text-align:left;">
snRNA gene
</td>
<td style="text-align:left;">
predicted gene, 22122
</td>
</tr>
<tr>
<td style="text-align:left;">
6
</td>
<td style="text-align:left;">
gene
</td>
<td style="text-align:right;">
48.29301
</td>
<td style="text-align:right;">
48.29308
</td>
<td style="text-align:left;">
negative
</td>
<td style="text-align:left;">
MGI_C57BL6J_4413817
</td>
<td style="text-align:left;">
n-TCgca50
</td>
<td style="text-align:left;">
NCBI_Gene:102467287
</td>
<td style="text-align:left;">
MGI:4413817
</td>
<td style="text-align:left;">
tRNA gene
</td>
<td style="text-align:left;">
nuclear encoded tRNA cysteine 50 (anticodon GCA)
</td>
</tr>
<tr>
<td style="text-align:left;">
6
</td>
<td style="text-align:left;">
gene
</td>
<td style="text-align:right;">
48.29743
</td>
<td style="text-align:right;">
48.29750
</td>
<td style="text-align:left;">
positive
</td>
<td style="text-align:left;">
MGI_C57BL6J_4413811
</td>
<td style="text-align:left;">
n-Tcgca44
</td>
<td style="text-align:left;">
NCBI_Gene:102467284
</td>
<td style="text-align:left;">
MGI:4413811
</td>
<td style="text-align:left;">
tRNA gene
</td>
<td style="text-align:left;">
nuclear encoded tRNA cysteine 44 (anticodon GCA)
</td>
</tr>
<tr>
<td style="text-align:left;">
6
</td>
<td style="text-align:left;">
gene
</td>
<td style="text-align:right;">
48.29895
</td>
<td style="text-align:right;">
48.29902
</td>
<td style="text-align:left;">
negative
</td>
<td style="text-align:left;">
MGI_C57BL6J_4413794
</td>
<td style="text-align:left;">
n-TCgca27
</td>
<td style="text-align:left;">
NCBI_Gene:102467278
</td>
<td style="text-align:left;">
MGI:4413794
</td>
<td style="text-align:left;">
tRNA gene
</td>
<td style="text-align:left;">
nuclear encoded tRNA cysteine 27 (anticodon GCA)
</td>
</tr>
<tr>
<td style="text-align:left;">
6
</td>
<td style="text-align:left;">
gene
</td>
<td style="text-align:right;">
48.30246
</td>
<td style="text-align:right;">
48.30253
</td>
<td style="text-align:left;">
positive
</td>
<td style="text-align:left;">
MGI_C57BL6J_4413773
</td>
<td style="text-align:left;">
n-TCgca6
</td>
<td style="text-align:left;">
NCBI_Gene:102467489
</td>
<td style="text-align:left;">
MGI:4413773
</td>
<td style="text-align:left;">
tRNA gene
</td>
<td style="text-align:left;">
nuclear encoded tRNA cysteine 6 (anticodon GCA)
</td>
</tr>
<tr>
<td style="text-align:left;">
6
</td>
<td style="text-align:left;">
gene
</td>
<td style="text-align:right;">
48.30347
</td>
<td style="text-align:right;">
48.30354
</td>
<td style="text-align:left;">
positive
</td>
<td style="text-align:left;">
MGI_C57BL6J_4413784
</td>
<td style="text-align:left;">
n-TCgca17
</td>
<td style="text-align:left;">
NCBI_Gene:102467273
</td>
<td style="text-align:left;">
MGI:4413784
</td>
<td style="text-align:left;">
tRNA gene
</td>
<td style="text-align:left;">
nuclear encoded tRNA cysteine 17 (anticodon GCA)
</td>
</tr>
<tr>
<td style="text-align:left;">
6
</td>
<td style="text-align:left;">
gene
</td>
<td style="text-align:right;">
48.30546
</td>
<td style="text-align:right;">
48.30553
</td>
<td style="text-align:left;">
negative
</td>
<td style="text-align:left;">
MGI_C57BL6J_4413774
</td>
<td style="text-align:left;">
n-TCgca7
</td>
<td style="text-align:left;">
NCBI_Gene:102467269
</td>
<td style="text-align:left;">
MGI:4413774
</td>
<td style="text-align:left;">
tRNA gene
</td>
<td style="text-align:left;">
nuclear encoded tRNA cysteine 7 (anticodon GCA)
</td>
</tr>
<tr>
<td style="text-align:left;">
6
</td>
<td style="text-align:left;">
gene
</td>
<td style="text-align:right;">
48.30873
</td>
<td style="text-align:right;">
48.30880
</td>
<td style="text-align:left;">
negative
</td>
<td style="text-align:left;">
MGI_C57BL6J_4413783
</td>
<td style="text-align:left;">
n-TCgca16
</td>
<td style="text-align:left;">
NCBI_Gene:102467493
</td>
<td style="text-align:left;">
MGI:4413783
</td>
<td style="text-align:left;">
tRNA gene
</td>
<td style="text-align:left;">
nuclear encoded tRNA cysteine 16 (anticodon GCA)
</td>
</tr>
<tr>
<td style="text-align:left;">
6
</td>
<td style="text-align:left;">
gene
</td>
<td style="text-align:right;">
48.30971
</td>
<td style="text-align:right;">
48.30978
</td>
<td style="text-align:left;">
negative
</td>
<td style="text-align:left;">
MGI_C57BL6J_4413793
</td>
<td style="text-align:left;">
n-TCgca26
</td>
<td style="text-align:left;">
NCBI_Gene:102467497
</td>
<td style="text-align:left;">
MGI:4413793
</td>
<td style="text-align:left;">
tRNA gene
</td>
<td style="text-align:left;">
nuclear encoded tRNA cysteine 26 (anticodon GCA)
</td>
</tr>
<tr>
<td style="text-align:left;">
6
</td>
<td style="text-align:left;">
gene
</td>
<td style="text-align:right;">
48.31150
</td>
<td style="text-align:right;">
48.31157
</td>
<td style="text-align:left;">
positive
</td>
<td style="text-align:left;">
MGI_C57BL6J_4413776
</td>
<td style="text-align:left;">
n-TCgca9
</td>
<td style="text-align:left;">
NCBI_Gene:102467270
</td>
<td style="text-align:left;">
MGI:4413776
</td>
<td style="text-align:left;">
tRNA gene
</td>
<td style="text-align:left;">
nuclear encoded tRNA cysteine 9 (anticodon GCA)
</td>
</tr>
<tr>
<td style="text-align:left;">
6
</td>
<td style="text-align:left;">
gene
</td>
<td style="text-align:right;">
48.31302
</td>
<td style="text-align:right;">
48.31309
</td>
<td style="text-align:left;">
negative
</td>
<td style="text-align:left;">
MGI_C57BL6J_4413792
</td>
<td style="text-align:left;">
n-TCgca25
</td>
<td style="text-align:left;">
NCBI_Gene:102467277
</td>
<td style="text-align:left;">
MGI:4413792
</td>
<td style="text-align:left;">
tRNA gene
</td>
<td style="text-align:left;">
nuclear encoded tRNA cysteine 25 (anticodon GCA)
</td>
</tr>
<tr>
<td style="text-align:left;">
6
</td>
<td style="text-align:left;">
gene
</td>
<td style="text-align:right;">
48.31654
</td>
<td style="text-align:right;">
48.31661
</td>
<td style="text-align:left;">
positive
</td>
<td style="text-align:left;">
MGI_C57BL6J_4413769
</td>
<td style="text-align:left;">
n-TCgca2
</td>
<td style="text-align:left;">
NCBI_Gene:102467267
</td>
<td style="text-align:left;">
MGI:4413769
</td>
<td style="text-align:left;">
tRNA gene
</td>
<td style="text-align:left;">
nuclear encoded tRNA cysteine 2 (anticodon GCA)
</td>
</tr>
<tr>
<td style="text-align:left;">
6
</td>
<td style="text-align:left;">
gene
</td>
<td style="text-align:right;">
48.31752
</td>
<td style="text-align:right;">
48.31759
</td>
<td style="text-align:left;">
positive
</td>
<td style="text-align:left;">
MGI_C57BL6J_4413812
</td>
<td style="text-align:left;">
n-TCgca45
</td>
<td style="text-align:left;">
NCBI_Gene:102467505
</td>
<td style="text-align:left;">
MGI:4413812
</td>
<td style="text-align:left;">
tRNA gene
</td>
<td style="text-align:left;">
nuclear encoded tRNA cysteine 45 (anticodon GCA)
</td>
</tr>
<tr>
<td style="text-align:left;">
6
</td>
<td style="text-align:left;">
gene
</td>
<td style="text-align:right;">
48.33284
</td>
<td style="text-align:right;">
48.33291
</td>
<td style="text-align:left;">
negative
</td>
<td style="text-align:left;">
MGI_C57BL6J_4413791
</td>
<td style="text-align:left;">
n-TCgca24
</td>
<td style="text-align:left;">
NCBI_Gene:102467276
</td>
<td style="text-align:left;">
MGI:4413791
</td>
<td style="text-align:left;">
tRNA gene
</td>
<td style="text-align:left;">
nuclear encoded tRNA cysteine 24 (anticodon GCA)
</td>
</tr>
<tr>
<td style="text-align:left;">
6
</td>
<td style="text-align:left;">
gene
</td>
<td style="text-align:right;">
48.33611
</td>
<td style="text-align:right;">
48.33618
</td>
<td style="text-align:left;">
negative
</td>
<td style="text-align:left;">
MGI_C57BL6J_4413790
</td>
<td style="text-align:left;">
n-TCgca23
</td>
<td style="text-align:left;">
NCBI_Gene:102467496
</td>
<td style="text-align:left;">
MGI:4413790
</td>
<td style="text-align:left;">
tRNA gene
</td>
<td style="text-align:left;">
nuclear encoded tRNA cysteine 23 (anticodon GCA)
</td>
</tr>
<tr>
<td style="text-align:left;">
6
</td>
<td style="text-align:left;">
gene
</td>
<td style="text-align:right;">
48.33709
</td>
<td style="text-align:right;">
48.33716
</td>
<td style="text-align:left;">
negative
</td>
<td style="text-align:left;">
MGI_C57BL6J_4413789
</td>
<td style="text-align:left;">
n-TCgca22
</td>
<td style="text-align:left;">
NCBI_Gene:102467275
</td>
<td style="text-align:left;">
MGI:4413789
</td>
<td style="text-align:left;">
tRNA gene
</td>
<td style="text-align:left;">
nuclear encoded tRNA cysteine 22 (anticodon GCA)
</td>
</tr>
<tr>
<td style="text-align:left;">
6
</td>
<td style="text-align:left;">
gene
</td>
<td style="text-align:right;">
48.34061
</td>
<td style="text-align:right;">
48.34068
</td>
<td style="text-align:left;">
positive
</td>
<td style="text-align:left;">
MGI_C57BL6J_4413781
</td>
<td style="text-align:left;">
n-TCgca14
</td>
<td style="text-align:left;">
NCBI_Gene:102467272
</td>
<td style="text-align:left;">
MGI:4413781
</td>
<td style="text-align:left;">
tRNA gene
</td>
<td style="text-align:left;">
nuclear encoded tRNA cysteine 14 (anticodon GCA)
</td>
</tr>
<tr>
<td style="text-align:left;">
6
</td>
<td style="text-align:left;">
gene
</td>
<td style="text-align:right;">
48.34159
</td>
<td style="text-align:right;">
48.34166
</td>
<td style="text-align:left;">
positive
</td>
<td style="text-align:left;">
MGI_C57BL6J_4413772
</td>
<td style="text-align:left;">
n-TCgca5
</td>
<td style="text-align:left;">
NCBI_Gene:102467268
</td>
<td style="text-align:left;">
MGI:4413772
</td>
<td style="text-align:left;">
tRNA gene
</td>
<td style="text-align:left;">
nuclear encoded tRNA cysteine 5 (anticodon GCA)
</td>
</tr>
<tr>
<td style="text-align:left;">
6
</td>
<td style="text-align:left;">
gene
</td>
<td style="text-align:right;">
48.34976
</td>
<td style="text-align:right;">
48.34983
</td>
<td style="text-align:left;">
negative
</td>
<td style="text-align:left;">
MGI_C57BL6J_4413788
</td>
<td style="text-align:left;">
n-TCgca21
</td>
<td style="text-align:left;">
NCBI_Gene:102467641
</td>
<td style="text-align:left;">
MGI:4413788
</td>
<td style="text-align:left;">
tRNA gene
</td>
<td style="text-align:left;">
nuclear encoded tRNA cysteine 21 (anticodon GCA)
</td>
</tr>
<tr>
<td style="text-align:left;">
6
</td>
<td style="text-align:left;">
gene
</td>
<td style="text-align:right;">
48.35402
</td>
<td style="text-align:right;">
48.35409
</td>
<td style="text-align:left;">
negative
</td>
<td style="text-align:left;">
MGI_C57BL6J_4413787
</td>
<td style="text-align:left;">
n-TCgca20
</td>
<td style="text-align:left;">
NCBI_Gene:102467495
</td>
<td style="text-align:left;">
MGI:4413787
</td>
<td style="text-align:left;">
tRNA gene
</td>
<td style="text-align:left;">
nuclear encoded tRNA cysteine 20 (anticodon GCA)
</td>
</tr>
<tr>
<td style="text-align:left;">
6
</td>
<td style="text-align:left;">
gene
</td>
<td style="text-align:right;">
48.35585
</td>
<td style="text-align:right;">
48.35593
</td>
<td style="text-align:left;">
positive
</td>
<td style="text-align:left;">
MGI_C57BL6J_4413813
</td>
<td style="text-align:left;">
n-TCgca46
</td>
<td style="text-align:left;">
NCBI_Gene:102467285
</td>
<td style="text-align:left;">
MGI:4413813
</td>
<td style="text-align:left;">
tRNA gene
</td>
<td style="text-align:left;">
nuclear encoded tRNA cysteine 46 (anticodon GCA)
</td>
</tr>
<tr>
<td style="text-align:left;">
6
</td>
<td style="text-align:left;">
gene
</td>
<td style="text-align:right;">
48.35685
</td>
<td style="text-align:right;">
48.35692
</td>
<td style="text-align:left;">
positive
</td>
<td style="text-align:left;">
MGI_C57BL6J_4413778
</td>
<td style="text-align:left;">
n-TCgca11
</td>
<td style="text-align:left;">
NCBI_Gene:102467271
</td>
<td style="text-align:left;">
MGI:4413778
</td>
<td style="text-align:left;">
tRNA gene
</td>
<td style="text-align:left;">
nuclear encoded tRNA cysteine 11 (anticodon GCA)
</td>
</tr>
<tr>
<td style="text-align:left;">
6
</td>
<td style="text-align:left;">
gene
</td>
<td style="text-align:right;">
48.36353
</td>
<td style="text-align:right;">
48.36360
</td>
<td style="text-align:left;">
negative
</td>
<td style="text-align:left;">
MGI_C57BL6J_4413786
</td>
<td style="text-align:left;">
n-TCgca19
</td>
<td style="text-align:left;">
NCBI_Gene:102467274
</td>
<td style="text-align:left;">
MGI:4413786
</td>
<td style="text-align:left;">
tRNA gene
</td>
<td style="text-align:left;">
nuclear encoded tRNA cysteine 19 (anticodon GCA)
</td>
</tr>
<tr>
<td style="text-align:left;">
6
</td>
<td style="text-align:left;">
gene
</td>
<td style="text-align:right;">
48.37414
</td>
<td style="text-align:right;">
48.39541
</td>
<td style="text-align:left;">
negative
</td>
<td style="text-align:left;">
MGI_C57BL6J_5753272
</td>
<td style="text-align:left;">
Gm44696
</td>
<td style="text-align:left;">
ENSEMBL:ENSMUSG00000107875
</td>
<td style="text-align:left;">
MGI:5753272
</td>
<td style="text-align:left;">
lncRNA gene
</td>
<td style="text-align:left;">
predicted gene 44696
</td>
</tr>
<tr>
<td style="text-align:left;">
6
</td>
<td style="text-align:left;">
pseudogene
</td>
<td style="text-align:right;">
48.38116
</td>
<td style="text-align:right;">
48.38207
</td>
<td style="text-align:left;">
negative
</td>
<td style="text-align:left;">
MGI_C57BL6J_3645672
</td>
<td style="text-align:left;">
Gm7880
</td>
<td style="text-align:left;">
NCBI_Gene:665990,ENSEMBL:ENSMUSG00000108275
</td>
<td style="text-align:left;">
MGI:3645672
</td>
<td style="text-align:left;">
pseudogene
</td>
<td style="text-align:left;">
predicted gene 7880
</td>
</tr>
<tr>
<td style="text-align:left;">
6
</td>
<td style="text-align:left;">
gene
</td>
<td style="text-align:right;">
48.39559
</td>
<td style="text-align:right;">
48.41985
</td>
<td style="text-align:left;">
positive
</td>
<td style="text-align:left;">
MGI_C57BL6J_1925077
</td>
<td style="text-align:left;">
Krba1
</td>
<td style="text-align:left;">
NCBI_Gene:77827,ENSEMBL:ENSMUSG00000042810
</td>
<td style="text-align:left;">
MGI:1925077
</td>
<td style="text-align:left;">
protein coding gene
</td>
<td style="text-align:left;">
KRAB-A domain containing 1
</td>
</tr>
<tr>
<td style="text-align:left;">
6
</td>
<td style="text-align:left;">
gene
</td>
<td style="text-align:right;">
48.39618
</td>
<td style="text-align:right;">
48.39672
</td>
<td style="text-align:left;">
negative
</td>
<td style="text-align:left;">
MGI_C57BL6J_1922119
</td>
<td style="text-align:left;">
4930447G04Rik
</td>
<td style="text-align:left;">
NA
</td>
<td style="text-align:left;">
NA
</td>
<td style="text-align:left;">
unclassified gene
</td>
<td style="text-align:left;">
RIKEN cDNA 4930447G04 gene
</td>
</tr>
<tr>
<td style="text-align:left;">
6
</td>
<td style="text-align:left;">
gene
</td>
<td style="text-align:right;">
48.42077
</td>
<td style="text-align:right;">
48.42084
</td>
<td style="text-align:left;">
positive
</td>
<td style="text-align:left;">
MGI_C57BL6J_5454102
</td>
<td style="text-align:left;">
Gm24325
</td>
<td style="text-align:left;">
ENSEMBL:ENSMUSG00000093085
</td>
<td style="text-align:left;">
MGI:5454102
</td>
<td style="text-align:left;">
miRNA gene
</td>
<td style="text-align:left;">
predicted gene, 24325
</td>
</tr>
<tr>
<td style="text-align:left;">
6
</td>
<td style="text-align:left;">
gene
</td>
<td style="text-align:right;">
48.42236
</td>
<td style="text-align:right;">
48.42756
</td>
<td style="text-align:left;">
positive
</td>
<td style="text-align:left;">
MGI_C57BL6J_5621688
</td>
<td style="text-align:left;">
Gm38803
</td>
<td style="text-align:left;">
NCBI_Gene:105242647
</td>
<td style="text-align:left;">
MGI:5621688
</td>
<td style="text-align:left;">
lncRNA gene
</td>
<td style="text-align:left;">
predicted gene, 38803
</td>
</tr>
<tr>
<td style="text-align:left;">
6
</td>
<td style="text-align:left;">
gene
</td>
<td style="text-align:right;">
48.42769
</td>
<td style="text-align:right;">
48.44603
</td>
<td style="text-align:left;">
negative
</td>
<td style="text-align:left;">
MGI_C57BL6J_1916160
</td>
<td style="text-align:left;">
Zfp467
</td>
<td style="text-align:left;">
NCBI_Gene:68910,ENSEMBL:ENSMUSG00000068551
</td>
<td style="text-align:left;">
MGI:1916160
</td>
<td style="text-align:left;">
protein coding gene
</td>
<td style="text-align:left;">
zinc finger protein 467
</td>
</tr>
<tr>
<td style="text-align:left;">
6
</td>
<td style="text-align:left;">
gene
</td>
<td style="text-align:right;">
48.44623
</td>
<td style="text-align:right;">
48.50126
</td>
<td style="text-align:left;">
positive
</td>
<td style="text-align:left;">
MGI_C57BL6J_2674311
</td>
<td style="text-align:left;">
Sspo
</td>
<td style="text-align:left;">
NCBI_Gene:243369,ENSEMBL:ENSMUSG00000029797
</td>
<td style="text-align:left;">
MGI:2674311
</td>
<td style="text-align:left;">
protein coding gene
</td>
<td style="text-align:left;">
SCO-spondin
</td>
</tr>
<tr>
<td style="text-align:left;">
6
</td>
<td style="text-align:left;">
pseudogene
</td>
<td style="text-align:right;">
48.50434
</td>
<td style="text-align:right;">
48.53483
</td>
<td style="text-align:left;">
positive
</td>
<td style="text-align:left;">
MGI_C57BL6J_1889827
</td>
<td style="text-align:left;">
Zfp862-ps
</td>
<td style="text-align:left;">
NCBI_Gene:58894,ENSEMBL:ENSMUSG00000107476
</td>
<td style="text-align:left;">
MGI:1889827
</td>
<td style="text-align:left;">
pseudogene
</td>
<td style="text-align:left;">
zinc finger protein 862, pseudogene
</td>
</tr>
<tr>
<td style="text-align:left;">
6
</td>
<td style="text-align:left;">
pseudogene
</td>
<td style="text-align:right;">
48.52346
</td>
<td style="text-align:right;">
48.52388
</td>
<td style="text-align:left;">
positive
</td>
<td style="text-align:left;">
MGI_C57BL6J_3704266
</td>
<td style="text-align:left;">
Gm10243
</td>
<td style="text-align:left;">
NCBI_Gene:100302576,ENSEMBL:ENSMUSG00000095526
</td>
<td style="text-align:left;">
MGI:3704266
</td>
<td style="text-align:left;">
pseudogene
</td>
<td style="text-align:left;">
predicted gene 10243
</td>
</tr>
<tr>
<td style="text-align:left;">
6
</td>
<td style="text-align:left;">
pseudogene
</td>
<td style="text-align:right;">
48.52392
</td>
<td style="text-align:right;">
48.52513
</td>
<td style="text-align:left;">
negative
</td>
<td style="text-align:left;">
MGI_C57BL6J_3648251
</td>
<td style="text-align:left;">
Gm7887
</td>
<td style="text-align:left;">
NCBI_Gene:666005,ENSEMBL:ENSMUSG00000044211
</td>
<td style="text-align:left;">
MGI:3648251
</td>
<td style="text-align:left;">
pseudogene
</td>
<td style="text-align:left;">
predicted gene 7887
</td>
</tr>
<tr>
<td style="text-align:left;">
6
</td>
<td style="text-align:left;">
gene
</td>
<td style="text-align:right;">
48.53658
</td>
<td style="text-align:right;">
48.53747
</td>
<td style="text-align:left;">
negative
</td>
<td style="text-align:left;">
MGI_C57BL6J_1916682
</td>
<td style="text-align:left;">
1700026J14Rik
</td>
<td style="text-align:left;">
NCBI_Gene:69432,ENSEMBL:ENSMUSG00000097716
</td>
<td style="text-align:left;">
MGI:1916682
</td>
<td style="text-align:left;">
lncRNA gene
</td>
<td style="text-align:left;">
RIKEN cDNA 1700026J14 gene
</td>
</tr>
<tr>
<td style="text-align:left;">
6
</td>
<td style="text-align:left;">
gene
</td>
<td style="text-align:right;">
48.53757
</td>
<td style="text-align:right;">
48.54180
</td>
<td style="text-align:left;">
positive
</td>
<td style="text-align:left;">
MGI_C57BL6J_1923502
</td>
<td style="text-align:left;">
Atp6v0e2
</td>
<td style="text-align:left;">
NCBI_Gene:76252,ENSEMBL:ENSMUSG00000039347
</td>
<td style="text-align:left;">
MGI:1923502
</td>
<td style="text-align:left;">
protein coding gene
</td>
<td style="text-align:left;">
ATPase, H+ transporting, lysosomal V0 subunit E2
</td>
</tr>
<tr>
<td style="text-align:left;">
6
</td>
<td style="text-align:left;">
gene
</td>
<td style="text-align:right;">
48.55478
</td>
<td style="text-align:right;">
48.57072
</td>
<td style="text-align:left;">
positive
</td>
<td style="text-align:left;">
MGI_C57BL6J_2652848
</td>
<td style="text-align:left;">
Lrrc61
</td>
<td style="text-align:left;">
NCBI_Gene:243371,ENSEMBL:ENSMUSG00000073096
</td>
<td style="text-align:left;">
MGI:2652848
</td>
<td style="text-align:left;">
protein coding gene
</td>
<td style="text-align:left;">
leucine rich repeat containing 61
</td>
</tr>
<tr>
<td style="text-align:left;">
6
</td>
<td style="text-align:left;">
gene
</td>
<td style="text-align:right;">
48.56970
</td>
<td style="text-align:right;">
48.57285
</td>
<td style="text-align:left;">
negative
</td>
<td style="text-align:left;">
MGI_C57BL6J_1918910
</td>
<td style="text-align:left;">
Rarres2
</td>
<td style="text-align:left;">
NCBI_Gene:71660,ENSEMBL:ENSMUSG00000009281
</td>
<td style="text-align:left;">
MGI:1918910
</td>
<td style="text-align:left;">
protein coding gene
</td>
<td style="text-align:left;">
retinoic acid receptor responder (tazarotene induced) 2
</td>
</tr>
<tr>
<td style="text-align:left;">
6
</td>
<td style="text-align:left;">
gene
</td>
<td style="text-align:right;">
48.58944
</td>
<td style="text-align:right;">
48.59058
</td>
<td style="text-align:left;">
positive
</td>
<td style="text-align:left;">
MGI_C57BL6J_3645688
</td>
<td style="text-align:left;">
Gm5111
</td>
<td style="text-align:left;">
NCBI_Gene:330305,ENSEMBL:ENSMUSG00000052730
</td>
<td style="text-align:left;">
MGI:3645688
</td>
<td style="text-align:left;">
protein coding gene
</td>
<td style="text-align:left;">
predicted gene 5111
</td>
</tr>
<tr>
<td style="text-align:left;">
6
</td>
<td style="text-align:left;">
gene
</td>
<td style="text-align:right;">
48.58961
</td>
<td style="text-align:right;">
48.61325
</td>
<td style="text-align:left;">
positive
</td>
<td style="text-align:left;">
MGI_C57BL6J_5753597
</td>
<td style="text-align:left;">
Gm45021
</td>
<td style="text-align:left;">
ENSEMBL:ENSMUSG00000107588
</td>
<td style="text-align:left;">
MGI:5753597
</td>
<td style="text-align:left;">
protein coding gene
</td>
<td style="text-align:left;">
predicted gene 45021
</td>
</tr>
<tr>
<td style="text-align:left;">
6
</td>
<td style="text-align:left;">
gene
</td>
<td style="text-align:right;">
48.59388
</td>
<td style="text-align:right;">
48.59908
</td>
<td style="text-align:left;">
positive
</td>
<td style="text-align:left;">
MGI_C57BL6J_1889817
</td>
<td style="text-align:left;">
Repin1
</td>
<td style="text-align:left;">
NCBI_Gene:58887,ENSEMBL:ENSMUSG00000052751
</td>
<td style="text-align:left;">
MGI:1889817
</td>
<td style="text-align:left;">
protein coding gene
</td>
<td style="text-align:left;">
replication initiator 1
</td>
</tr>
<tr>
<td style="text-align:left;">
6
</td>
<td style="text-align:left;">
gene
</td>
<td style="text-align:right;">
48.59394
</td>
<td style="text-align:right;">
48.61928
</td>
<td style="text-align:left;">
positive
</td>
<td style="text-align:left;">
MGI_C57BL6J_5753541
</td>
<td style="text-align:left;">
Gm44965
</td>
<td style="text-align:left;">
ENSEMBL:ENSMUSG00000107789
</td>
<td style="text-align:left;">
MGI:5753541
</td>
<td style="text-align:left;">
protein coding gene
</td>
<td style="text-align:left;">
predicted gene 44965
</td>
</tr>
<tr>
<td style="text-align:left;">
6
</td>
<td style="text-align:left;">
gene
</td>
<td style="text-align:right;">
48.60200
</td>
<td style="text-align:right;">
48.62323
</td>
<td style="text-align:left;">
positive
</td>
<td style="text-align:left;">
MGI_C57BL6J_2683557
</td>
<td style="text-align:left;">
Zfp775
</td>
<td style="text-align:left;">
NCBI_Gene:243372,ENSEMBL:ENSMUSG00000007216
</td>
<td style="text-align:left;">
MGI:2683557
</td>
<td style="text-align:left;">
protein coding gene
</td>
<td style="text-align:left;">
zinc finger protein 775
</td>
</tr>
<tr>
<td style="text-align:left;">
6
</td>
<td style="text-align:left;">
gene
</td>
<td style="text-align:right;">
48.61127
</td>
<td style="text-align:right;">
48.62236
</td>
<td style="text-align:left;">
negative
</td>
<td style="text-align:left;">
MGI_C57BL6J_5621689
</td>
<td style="text-align:left;">
Gm38804
</td>
<td style="text-align:left;">
NCBI_Gene:105242648,ENSEMBL:ENSMUSG00000107640
</td>
<td style="text-align:left;">
MGI:5621689
</td>
<td style="text-align:left;">
lncRNA gene
</td>
<td style="text-align:left;">
predicted gene, 38804
</td>
</tr>
<tr>
<td style="text-align:left;">
6
</td>
<td style="text-align:left;">
gene
</td>
<td style="text-align:right;">
48.62676
</td>
<td style="text-align:right;">
48.63369
</td>
<td style="text-align:left;">
positive
</td>
<td style="text-align:left;">
MGI_C57BL6J_2141510
</td>
<td style="text-align:left;">
AI854703
</td>
<td style="text-align:left;">
NCBI_Gene:243373,ENSEMBL:ENSMUSG00000053297
</td>
<td style="text-align:left;">
MGI:2141510
</td>
<td style="text-align:left;">
protein coding gene
</td>
<td style="text-align:left;">
expressed sequence AI854703
</td>
</tr>
<tr>
<td style="text-align:left;">
6
</td>
<td style="text-align:left;">
gene
</td>
<td style="text-align:right;">
48.64719
</td>
<td style="text-align:right;">
48.66087
</td>
<td style="text-align:left;">
positive
</td>
<td style="text-align:left;">
MGI_C57BL6J_2685303
</td>
<td style="text-align:left;">
Gimap8
</td>
<td style="text-align:left;">
NCBI_Gene:243374,ENSEMBL:ENSMUSG00000064262
</td>
<td style="text-align:left;">
MGI:2685303
</td>
<td style="text-align:left;">
protein coding gene
</td>
<td style="text-align:left;">
GTPase, IMAP family member 8
</td>
</tr>
<tr>
<td style="text-align:left;">
6
</td>
<td style="text-align:left;">
pseudogene
</td>
<td style="text-align:right;">
48.66540
</td>
<td style="text-align:right;">
48.66586
</td>
<td style="text-align:left;">
negative
</td>
<td style="text-align:left;">
MGI_C57BL6J_5690654
</td>
<td style="text-align:left;">
Gm44262
</td>
<td style="text-align:left;">
ENSEMBL:ENSMUSG00000108117
</td>
<td style="text-align:left;">
MGI:5690654
</td>
<td style="text-align:left;">
pseudogene
</td>
<td style="text-align:left;">
predicted gene, 44262
</td>
</tr>
<tr>
<td style="text-align:left;">
6
</td>
<td style="text-align:left;">
gene
</td>
<td style="text-align:right;">
48.67583
</td>
<td style="text-align:right;">
48.67694
</td>
<td style="text-align:left;">
negative
</td>
<td style="text-align:left;">
MGI_C57BL6J_5826595
</td>
<td style="text-align:left;">
Gm46958
</td>
<td style="text-align:left;">
NCBI_Gene:108169139
</td>
<td style="text-align:left;">
MGI:5826595
</td>
<td style="text-align:left;">
lncRNA gene
</td>
<td style="text-align:left;">
predicted gene, 46958
</td>
</tr>
<tr>
<td style="text-align:left;">
6
</td>
<td style="text-align:left;">
gene
</td>
<td style="text-align:right;">
48.67612
</td>
<td style="text-align:right;">
48.67912
</td>
<td style="text-align:left;">
positive
</td>
<td style="text-align:left;">
MGI_C57BL6J_3511744
</td>
<td style="text-align:left;">
Gimap9
</td>
<td style="text-align:left;">
NCBI_Gene:317758,ENSEMBL:ENSMUSG00000051124
</td>
<td style="text-align:left;">
MGI:3511744
</td>
<td style="text-align:left;">
protein coding gene
</td>
<td style="text-align:left;">
GTPase, IMAP family member 9
</td>
</tr>
<tr>
<td style="text-align:left;">
6
</td>
<td style="text-align:left;">
gene
</td>
<td style="text-align:right;">
48.68455
</td>
<td style="text-align:right;">
48.69206
</td>
<td style="text-align:left;">
positive
</td>
<td style="text-align:left;">
MGI_C57BL6J_1349656
</td>
<td style="text-align:left;">
Gimap4
</td>
<td style="text-align:left;">
NCBI_Gene:107526,ENSEMBL:ENSMUSG00000054435
</td>
<td style="text-align:left;">
MGI:1349656
</td>
<td style="text-align:left;">
protein coding gene
</td>
<td style="text-align:left;">
GTPase, IMAP family member 4
</td>
</tr>
<tr>
<td style="text-align:left;">
6
</td>
<td style="text-align:left;">
gene
</td>
<td style="text-align:right;">
48.70158
</td>
<td style="text-align:right;">
48.70824
</td>
<td style="text-align:left;">
negative
</td>
<td style="text-align:left;">
MGI_C57BL6J_1918876
</td>
<td style="text-align:left;">
Gimap6
</td>
<td style="text-align:left;">
NCBI_Gene:231931,ENSEMBL:ENSMUSG00000047867
</td>
<td style="text-align:left;">
MGI:1918876
</td>
<td style="text-align:left;">
protein coding gene
</td>
<td style="text-align:left;">
GTPase, IMAP family member 6
</td>
</tr>
<tr>
<td style="text-align:left;">
6
</td>
<td style="text-align:left;">
gene
</td>
<td style="text-align:right;">
48.71861
</td>
<td style="text-align:right;">
48.75264
</td>
<td style="text-align:left;">
positive
</td>
<td style="text-align:left;">
MGI_C57BL6J_5547789
</td>
<td style="text-align:left;">
Gm28053
</td>
<td style="text-align:left;">
ENSEMBL:ENSMUSG00000098715
</td>
<td style="text-align:left;">
MGI:5547789
</td>
<td style="text-align:left;">
protein coding gene
</td>
<td style="text-align:left;">
predicted gene, 28053
</td>
</tr>
<tr>
<td style="text-align:left;">
6
</td>
<td style="text-align:left;">
gene
</td>
<td style="text-align:right;">
48.71862
</td>
<td style="text-align:right;">
48.72464
</td>
<td style="text-align:left;">
positive
</td>
<td style="text-align:left;">
MGI_C57BL6J_1349657
</td>
<td style="text-align:left;">
Gimap7
</td>
<td style="text-align:left;">
NCBI_Gene:231932,ENSEMBL:ENSMUSG00000043931
</td>
<td style="text-align:left;">
MGI:1349657
</td>
<td style="text-align:left;">
protein coding gene
</td>
<td style="text-align:left;">
GTPase, IMAP family member 7
</td>
</tr>
<tr>
<td style="text-align:left;">
6
</td>
<td style="text-align:left;">
gene
</td>
<td style="text-align:right;">
48.72928
</td>
<td style="text-align:right;">
48.74144
</td>
<td style="text-align:left;">
negative
</td>
<td style="text-align:left;">
MGI_C57BL6J_3781523
</td>
<td style="text-align:left;">
Gimap1os
</td>
<td style="text-align:left;">
NCBI_Gene:100041450,ENSEMBL:ENSMUSG00000044867
</td>
<td style="text-align:left;">
MGI:3781523
</td>
<td style="text-align:left;">
lncRNA gene
</td>
<td style="text-align:left;">
GTPase, IMAP family member 1, opposite strand
</td>
</tr>
<tr>
<td style="text-align:left;">
6
</td>
<td style="text-align:left;">
gene
</td>
<td style="text-align:right;">
48.72955
</td>
<td style="text-align:right;">
48.73367
</td>
<td style="text-align:left;">
positive
</td>
<td style="text-align:left;">
MGI_C57BL6J_1889832
</td>
<td style="text-align:left;">
A130009E19Rik
</td>
<td style="text-align:left;">
NA
</td>
<td style="text-align:left;">
NA
</td>
<td style="text-align:left;">
unclassified gene
</td>
<td style="text-align:left;">
RIKEN cDNA A130009E19 gene
</td>
</tr>
<tr>
<td style="text-align:left;">
6
</td>
<td style="text-align:left;">
gene
</td>
<td style="text-align:right;">
48.73905
</td>
<td style="text-align:right;">
48.74379
</td>
<td style="text-align:left;">
positive
</td>
<td style="text-align:left;">
MGI_C57BL6J_109368
</td>
<td style="text-align:left;">
Gimap1
</td>
<td style="text-align:left;">
NCBI_Gene:16205,ENSEMBL:ENSMUSG00000090019
</td>
<td style="text-align:left;">
MGI:109368
</td>
<td style="text-align:left;">
protein coding gene
</td>
<td style="text-align:left;">
GTPase, IMAP family member 1
</td>
</tr>
<tr>
<td style="text-align:left;">
6
</td>
<td style="text-align:left;">
gene
</td>
<td style="text-align:right;">
48.74614
</td>
<td style="text-align:right;">
48.75421
</td>
<td style="text-align:left;">
positive
</td>
<td style="text-align:left;">
MGI_C57BL6J_2442232
</td>
<td style="text-align:left;">
Gimap5
</td>
<td style="text-align:left;">
NCBI_Gene:317757,ENSEMBL:ENSMUSG00000043505
</td>
<td style="text-align:left;">
MGI:2442232
</td>
<td style="text-align:left;">
protein coding gene
</td>
<td style="text-align:left;">
GTPase, IMAP family member 5
</td>
</tr>
<tr>
<td style="text-align:left;">
6
</td>
<td style="text-align:left;">
gene
</td>
<td style="text-align:right;">
48.75423
</td>
<td style="text-align:right;">
48.75599
</td>
<td style="text-align:left;">
negative
</td>
<td style="text-align:left;">
MGI_C57BL6J_5690618
</td>
<td style="text-align:left;">
Gm44226
</td>
<td style="text-align:left;">
ENSEMBL:ENSMUSG00000108298
</td>
<td style="text-align:left;">
MGI:5690618
</td>
<td style="text-align:left;">
lncRNA gene
</td>
<td style="text-align:left;">
predicted gene, 44226
</td>
</tr>
<tr>
<td style="text-align:left;">
6
</td>
<td style="text-align:left;">
gene
</td>
<td style="text-align:right;">
48.75534
</td>
<td style="text-align:right;">
48.75637
</td>
<td style="text-align:left;">
positive
</td>
<td style="text-align:left;">
MGI_C57BL6J_1921153
</td>
<td style="text-align:left;">
4833403J16Rik
</td>
<td style="text-align:left;">
ENSEMBL:ENSMUSG00000107481
</td>
<td style="text-align:left;">
MGI:1921153
</td>
<td style="text-align:left;">
lncRNA gene
</td>
<td style="text-align:left;">
RIKEN cDNA 4833403J16 gene
</td>
</tr>
<tr>
<td style="text-align:left;">
6
</td>
<td style="text-align:left;">
gene
</td>
<td style="text-align:right;">
48.76446
</td>
<td style="text-align:right;">
48.77085
</td>
<td style="text-align:left;">
negative
</td>
<td style="text-align:left;">
MGI_C57BL6J_1932723
</td>
<td style="text-align:left;">
Gimap3
</td>
<td style="text-align:left;">
NCBI_Gene:83408,ENSEMBL:ENSMUSG00000039264
</td>
<td style="text-align:left;">
MGI:1932723
</td>
<td style="text-align:left;">
protein coding gene
</td>
<td style="text-align:left;">
GTPase, IMAP family member 3
</td>
</tr>
<tr>
<td style="text-align:left;">
6
</td>
<td style="text-align:left;">
gene
</td>
<td style="text-align:right;">
48.79100
</td>
<td style="text-align:right;">
48.79378
</td>
<td style="text-align:left;">
negative
</td>
<td style="text-align:left;">
MGI_C57BL6J_5589873
</td>
<td style="text-align:left;">
Gm30714
</td>
<td style="text-align:left;">
NCBI_Gene:102632718
</td>
<td style="text-align:left;">
MGI:5589873
</td>
<td style="text-align:left;">
lncRNA gene
</td>
<td style="text-align:left;">
predicted gene, 30714
</td>
</tr>
<tr>
<td style="text-align:left;">
6
</td>
<td style="text-align:left;">
gene
</td>
<td style="text-align:right;">
48.79809
</td>
<td style="text-align:right;">
48.79885
</td>
<td style="text-align:left;">
negative
</td>
<td style="text-align:left;">
MGI_C57BL6J_5589940
</td>
<td style="text-align:left;">
Gm30781
</td>
<td style="text-align:left;">
NCBI_Gene:102632803,ENSEMBL:ENSMUSG00000108130
</td>
<td style="text-align:left;">
MGI:5589940
</td>
<td style="text-align:left;">
lncRNA gene
</td>
<td style="text-align:left;">
predicted gene, 30781
</td>
</tr>
<tr>
<td style="text-align:left;">
6
</td>
<td style="text-align:left;">
pseudogene
</td>
<td style="text-align:right;">
48.81902
</td>
<td style="text-align:right;">
48.82047
</td>
<td style="text-align:left;">
positive
</td>
<td style="text-align:left;">
MGI_C57BL6J_5010019
</td>
<td style="text-align:left;">
Gm17834
</td>
<td style="text-align:left;">
NCBI_Gene:100415951,ENSEMBL:ENSMUSG00000107574
</td>
<td style="text-align:left;">
MGI:5010019
</td>
<td style="text-align:left;">
pseudogene
</td>
<td style="text-align:left;">
predicted gene, 17834
</td>
</tr>
<tr>
<td style="text-align:left;">
6
</td>
<td style="text-align:left;">
gene
</td>
<td style="text-align:right;">
48.83381
</td>
<td style="text-align:right;">
48.84595
</td>
<td style="text-align:left;">
negative
</td>
<td style="text-align:left;">
MGI_C57BL6J_1916348
</td>
<td style="text-align:left;">
Tmem176b
</td>
<td style="text-align:left;">
NCBI_Gene:65963,ENSEMBL:ENSMUSG00000029810
</td>
<td style="text-align:left;">
MGI:1916348
</td>
<td style="text-align:left;">
protein coding gene
</td>
<td style="text-align:left;">
transmembrane protein 176B
</td>
</tr>
<tr>
<td style="text-align:left;">
6
</td>
<td style="text-align:left;">
gene
</td>
<td style="text-align:right;">
48.84092
</td>
<td style="text-align:right;">
48.84707
</td>
<td style="text-align:left;">
positive
</td>
<td style="text-align:left;">
MGI_C57BL6J_1913308
</td>
<td style="text-align:left;">
Tmem176a
</td>
<td style="text-align:left;">
NCBI_Gene:66058,ENSEMBL:ENSMUSG00000023367
</td>
<td style="text-align:left;">
MGI:1913308
</td>
<td style="text-align:left;">
protein coding gene
</td>
<td style="text-align:left;">
transmembrane protein 176A
</td>
</tr>
<tr>
<td style="text-align:left;">
6
</td>
<td style="text-align:left;">
gene
</td>
<td style="text-align:right;">
48.84766
</td>
<td style="text-align:right;">
48.90306
</td>
<td style="text-align:left;">
negative
</td>
<td style="text-align:left;">
MGI_C57BL6J_5753340
</td>
<td style="text-align:left;">
Gm44764
</td>
<td style="text-align:left;">
NCBI_Gene:108169140,ENSEMBL:ENSMUSG00000107597
</td>
<td style="text-align:left;">
MGI:5753340
</td>
<td style="text-align:left;">
lncRNA gene
</td>
<td style="text-align:left;">
predicted gene 44764
</td>
</tr>
<tr>
<td style="text-align:left;">
6
</td>
<td style="text-align:left;">
gene
</td>
<td style="text-align:right;">
48.85990
</td>
<td style="text-align:right;">
48.86608
</td>
<td style="text-align:left;">
positive
</td>
<td style="text-align:left;">
MGI_C57BL6J_3643446
</td>
<td style="text-align:left;">
Gm7932
</td>
<td style="text-align:left;">
NCBI_Gene:666105,ENSEMBL:ENSMUSG00000108084
</td>
<td style="text-align:left;">
MGI:3643446
</td>
<td style="text-align:left;">
lncRNA gene
</td>
<td style="text-align:left;">
predicted gene 7932
</td>
</tr>
<tr>
<td style="text-align:left;">
6
</td>
<td style="text-align:left;">
gene
</td>
<td style="text-align:right;">
48.87290
</td>
<td style="text-align:right;">
48.90919
</td>
<td style="text-align:left;">
positive
</td>
<td style="text-align:left;">
MGI_C57BL6J_1923757
</td>
<td style="text-align:left;">
Aoc1
</td>
<td style="text-align:left;">
NCBI_Gene:76507,ENSEMBL:ENSMUSG00000029811
</td>
<td style="text-align:left;">
MGI:1923757
</td>
<td style="text-align:left;">
protein coding gene
</td>
<td style="text-align:left;">
amine oxidase, copper-containing 1
</td>
</tr>
<tr>
<td style="text-align:left;">
6
</td>
<td style="text-align:left;">
gene
</td>
<td style="text-align:right;">
48.92990
</td>
<td style="text-align:right;">
48.93369
</td>
<td style="text-align:left;">
positive
</td>
<td style="text-align:left;">
MGI_C57BL6J_1917011
</td>
<td style="text-align:left;">
Doxl1
</td>
<td style="text-align:left;">
NCBI_Gene:69761,ENSEMBL:ENSMUSG00000029813
</td>
<td style="text-align:left;">
MGI:1917011
</td>
<td style="text-align:left;">
protein coding gene
</td>
<td style="text-align:left;">
diamine oxidase-like protein 1
</td>
</tr>
<tr>
<td style="text-align:left;">
6
</td>
<td style="text-align:left;">
gene
</td>
<td style="text-align:right;">
48.93479
</td>
<td style="text-align:right;">
48.93492
</td>
<td style="text-align:left;">
positive
</td>
<td style="text-align:left;">
MGI_C57BL6J_5454125
</td>
<td style="text-align:left;">
Gm24348
</td>
<td style="text-align:left;">
ENSEMBL:ENSMUSG00000089335
</td>
<td style="text-align:left;">
MGI:5454125
</td>
<td style="text-align:left;">
snoRNA gene
</td>
<td style="text-align:left;">
predicted gene, 24348
</td>
</tr>
<tr>
<td style="text-align:left;">
6
</td>
<td style="text-align:left;">
pseudogene
</td>
<td style="text-align:right;">
48.93975
</td>
<td style="text-align:right;">
48.94405
</td>
<td style="text-align:left;">
negative
</td>
<td style="text-align:left;">
MGI_C57BL6J_3642310
</td>
<td style="text-align:left;">
Gm10242
</td>
<td style="text-align:left;">
NCBI_Gene:100040589,ENSEMBL:ENSMUSG00000068537
</td>
<td style="text-align:left;">
MGI:3642310
</td>
<td style="text-align:left;">
pseudogene
</td>
<td style="text-align:left;">
predicted gene 10242
</td>
</tr>
<tr>
<td style="text-align:left;">
6
</td>
<td style="text-align:left;">
pseudogene
</td>
<td style="text-align:right;">
48.95626
</td>
<td style="text-align:right;">
48.95690
</td>
<td style="text-align:left;">
positive
</td>
<td style="text-align:left;">
MGI_C57BL6J_5010723
</td>
<td style="text-align:left;">
Gm18538
</td>
<td style="text-align:left;">
NCBI_Gene:100417335,ENSEMBL:ENSMUSG00000107943
</td>
<td style="text-align:left;">
MGI:5010723
</td>
<td style="text-align:left;">
pseudogene
</td>
<td style="text-align:left;">
predicted gene, 18538
</td>
</tr>
<tr>
<td style="text-align:left;">
6
</td>
<td style="text-align:left;">
pseudogene
</td>
<td style="text-align:right;">
48.96279
</td>
<td style="text-align:right;">
48.96417
</td>
<td style="text-align:left;">
positive
</td>
<td style="text-align:left;">
MGI_C57BL6J_5011357
</td>
<td style="text-align:left;">
Gm19172
</td>
<td style="text-align:left;">
NCBI_Gene:100418382,ENSEMBL:ENSMUSG00000107692
</td>
<td style="text-align:left;">
MGI:5011357
</td>
<td style="text-align:left;">
pseudogene
</td>
<td style="text-align:left;">
predicted gene, 19172
</td>
</tr>
<tr>
<td style="text-align:left;">
6
</td>
<td style="text-align:left;">
gene
</td>
<td style="text-align:right;">
48.97481
</td>
<td style="text-align:right;">
48.97875
</td>
<td style="text-align:left;">
positive
</td>
<td style="text-align:left;">
MGI_C57BL6J_3618290
</td>
<td style="text-align:left;">
Doxl2
</td>
<td style="text-align:left;">
NCBI_Gene:243376,ENSEMBL:ENSMUSG00000068536
</td>
<td style="text-align:left;">
MGI:3618290
</td>
<td style="text-align:left;">
protein coding gene
</td>
<td style="text-align:left;">
diamine oxidase-like protein 2
</td>
</tr>
<tr>
<td style="text-align:left;">
6
</td>
<td style="text-align:left;">
pseudogene
</td>
<td style="text-align:right;">
48.98110
</td>
<td style="text-align:right;">
48.98221
</td>
<td style="text-align:left;">
positive
</td>
<td style="text-align:left;">
MGI_C57BL6J_5010724
</td>
<td style="text-align:left;">
Gm18539
</td>
<td style="text-align:left;">
NCBI_Gene:100417336,ENSEMBL:ENSMUSG00000108017
</td>
<td style="text-align:left;">
MGI:5010724
</td>
<td style="text-align:left;">
pseudogene
</td>
<td style="text-align:left;">
predicted gene, 18539
</td>
</tr>
<tr>
<td style="text-align:left;">
6
</td>
<td style="text-align:left;">
gene
</td>
<td style="text-align:right;">
48.98662
</td>
<td style="text-align:right;">
48.99962
</td>
<td style="text-align:left;">
negative
</td>
<td style="text-align:left;">
MGI_C57BL6J_1922597
</td>
<td style="text-align:left;">
4930563H07Rik
</td>
<td style="text-align:left;">
NCBI_Gene:75347,ENSEMBL:ENSMUSG00000107656
</td>
<td style="text-align:left;">
MGI:1922597
</td>
<td style="text-align:left;">
lncRNA gene
</td>
<td style="text-align:left;">
RIKEN cDNA 4930563H07 gene
</td>
</tr>
<tr>
<td style="text-align:left;">
6
</td>
<td style="text-align:left;">
gene
</td>
<td style="text-align:right;">
48.98686
</td>
<td style="text-align:right;">
48.99173
</td>
<td style="text-align:left;">
positive
</td>
<td style="text-align:left;">
MGI_C57BL6J_2682321
</td>
<td style="text-align:left;">
Svs1
</td>
<td style="text-align:left;">
NCBI_Gene:243377,ENSEMBL:ENSMUSG00000039215
</td>
<td style="text-align:left;">
MGI:2682321
</td>
<td style="text-align:left;">
protein coding gene
</td>
<td style="text-align:left;">
seminal vesicle secretory protein 1
</td>
</tr>
<tr>
<td style="text-align:left;">
6
</td>
<td style="text-align:left;">
pseudogene
</td>
<td style="text-align:right;">
48.99521
</td>
<td style="text-align:right;">
48.99765
</td>
<td style="text-align:left;">
positive
</td>
<td style="text-align:left;">
MGI_C57BL6J_5011358
</td>
<td style="text-align:left;">
Gm19173
</td>
<td style="text-align:left;">
NCBI_Gene:100418383,ENSEMBL:ENSMUSG00000107457
</td>
<td style="text-align:left;">
MGI:5011358
</td>
<td style="text-align:left;">
pseudogene
</td>
<td style="text-align:left;">
predicted gene, 19173
</td>
</tr>
<tr>
<td style="text-align:left;">
6
</td>
<td style="text-align:left;">
gene
</td>
<td style="text-align:right;">
49.03652
</td>
<td style="text-align:right;">
49.07093
</td>
<td style="text-align:left;">
positive
</td>
<td style="text-align:left;">
MGI_C57BL6J_1934765
</td>
<td style="text-align:left;">
Gpnmb
</td>
<td style="text-align:left;">
NCBI_Gene:93695,ENSEMBL:ENSMUSG00000029816
</td>
<td style="text-align:left;">
MGI:1934765
</td>
<td style="text-align:left;">
protein coding gene
</td>
<td style="text-align:left;">
glycoprotein (transmembrane) nmb
</td>
</tr>
<tr>
<td style="text-align:left;">
6
</td>
<td style="text-align:left;">
pseudogene
</td>
<td style="text-align:right;">
49.07162
</td>
<td style="text-align:right;">
49.07216
</td>
<td style="text-align:left;">
positive
</td>
<td style="text-align:left;">
MGI_C57BL6J_5011438
</td>
<td style="text-align:left;">
Gm19253
</td>
<td style="text-align:left;">
NCBI_Gene:100462958,ENSEMBL:ENSMUSG00000107758
</td>
<td style="text-align:left;">
MGI:5011438
</td>
<td style="text-align:left;">
pseudogene
</td>
<td style="text-align:left;">
predicted gene, 19253
</td>
</tr>
<tr>
<td style="text-align:left;">
6
</td>
<td style="text-align:left;">
gene
</td>
<td style="text-align:right;">
49.07379
</td>
<td style="text-align:right;">
49.13914
</td>
<td style="text-align:left;">
positive
</td>
<td style="text-align:left;">
MGI_C57BL6J_1922843
</td>
<td style="text-align:left;">
Malsu1
</td>
<td style="text-align:left;">
NCBI_Gene:75593,ENSEMBL:ENSMUSG00000029815
</td>
<td style="text-align:left;">
MGI:1922843
</td>
<td style="text-align:left;">
protein coding gene
</td>
<td style="text-align:left;">
mitochondrial assembly of ribosomal large subunit 1
</td>
</tr>
<tr>
<td style="text-align:left;">
6
</td>
<td style="text-align:left;">
pseudogene
</td>
<td style="text-align:right;">
49.07995
</td>
<td style="text-align:right;">
49.08117
</td>
<td style="text-align:left;">
negative
</td>
<td style="text-align:left;">
MGI_C57BL6J_3645227
</td>
<td style="text-align:left;">
Gm7890
</td>
<td style="text-align:left;">
NCBI_Gene:666011,ENSEMBL:ENSMUSG00000107990
</td>
<td style="text-align:left;">
MGI:3645227
</td>
<td style="text-align:left;">
pseudogene
</td>
<td style="text-align:left;">
predicted gene 7890
</td>
</tr>
<tr>
<td style="text-align:left;">
6
</td>
<td style="text-align:left;">
gene
</td>
<td style="text-align:right;">
49.08522
</td>
<td style="text-align:right;">
49.21501
</td>
<td style="text-align:left;">
negative
</td>
<td style="text-align:left;">
MGI_C57BL6J_1890359
</td>
<td style="text-align:left;">
Igf2bp3
</td>
<td style="text-align:left;">
NCBI_Gene:140488,ENSEMBL:ENSMUSG00000029814
</td>
<td style="text-align:left;">
MGI:1890359
</td>
<td style="text-align:left;">
protein coding gene
</td>
<td style="text-align:left;">
insulin-like growth factor 2 mRNA binding protein 3
</td>
</tr>
<tr>
<td style="text-align:left;">
6
</td>
<td style="text-align:left;">
pseudogene
</td>
<td style="text-align:right;">
49.17267
</td>
<td style="text-align:right;">
49.17352
</td>
<td style="text-align:left;">
positive
</td>
<td style="text-align:left;">
MGI_C57BL6J_5010195
</td>
<td style="text-align:left;">
Gm18010
</td>
<td style="text-align:left;">
NCBI_Gene:100416268,ENSEMBL:ENSMUSG00000107727
</td>
<td style="text-align:left;">
MGI:5010195
</td>
<td style="text-align:left;">
pseudogene
</td>
<td style="text-align:left;">
predicted gene, 18010
</td>
</tr>
<tr>
<td style="text-align:left;">
6
</td>
<td style="text-align:left;">
gene
</td>
<td style="text-align:right;">
49.20581
</td>
<td style="text-align:right;">
49.21248
</td>
<td style="text-align:left;">
negative
</td>
<td style="text-align:left;">
MGI_C57BL6J_2442568
</td>
<td style="text-align:left;">
D030074P21Rik
</td>
<td style="text-align:left;">
NCBI_Gene:102642435
</td>
<td style="text-align:left;">
MGI:2442568
</td>
<td style="text-align:left;">
lncRNA gene
</td>
<td style="text-align:left;">
RIKEN cDNA D030074P21 gene
</td>
</tr>
<tr>
<td style="text-align:left;">
6
</td>
<td style="text-align:left;">
gene
</td>
<td style="text-align:right;">
49.24392
</td>
<td style="text-align:right;">
49.26425
</td>
<td style="text-align:left;">
negative
</td>
<td style="text-align:left;">
MGI_C57BL6J_1933972
</td>
<td style="text-align:left;">
Tra2a
</td>
<td style="text-align:left;">
NCBI_Gene:101214,ENSEMBL:ENSMUSG00000029817
</td>
<td style="text-align:left;">
MGI:1933972
</td>
<td style="text-align:left;">
protein coding gene
</td>
<td style="text-align:left;">
transformer 2 alpha
</td>
</tr>
<tr>
<td style="text-align:left;">
6
</td>
<td style="text-align:left;">
gene
</td>
<td style="text-align:right;">
49.25485
</td>
<td style="text-align:right;">
49.25646
</td>
<td style="text-align:left;">
negative
</td>
<td style="text-align:left;">
MGI_C57BL6J_2441851
</td>
<td style="text-align:left;">
B230214O09Rik
</td>
<td style="text-align:left;">
NA
</td>
<td style="text-align:left;">
NA
</td>
<td style="text-align:left;">
unclassified non-coding RNA gene
</td>
<td style="text-align:left;">
RIKEN cDNA B230214O09 gene
</td>
</tr>
<tr>
<td style="text-align:left;">
6
</td>
<td style="text-align:left;">
gene
</td>
<td style="text-align:right;">
49.25485
</td>
<td style="text-align:right;">
49.25646
</td>
<td style="text-align:left;">
positive
</td>
<td style="text-align:left;">
MGI_C57BL6J_5690372
</td>
<td style="text-align:left;">
Gm43980
</td>
<td style="text-align:left;">
ENSEMBL:ENSMUSG00000107962
</td>
<td style="text-align:left;">
MGI:5690372
</td>
<td style="text-align:left;">
unclassified gene
</td>
<td style="text-align:left;">
predicted gene, 43980
</td>
</tr>
<tr>
<td style="text-align:left;">
6
</td>
<td style="text-align:left;">
pseudogene
</td>
<td style="text-align:right;">
49.28655
</td>
<td style="text-align:right;">
49.28729
</td>
<td style="text-align:left;">
positive
</td>
<td style="text-align:left;">
MGI_C57BL6J_5010196
</td>
<td style="text-align:left;">
Gm18011
</td>
<td style="text-align:left;">
NCBI_Gene:100416269,ENSEMBL:ENSMUSG00000107916
</td>
<td style="text-align:left;">
MGI:5010196
</td>
<td style="text-align:left;">
pseudogene
</td>
<td style="text-align:left;">
predicted gene, 18011
</td>
</tr>
<tr>
<td style="text-align:left;">
6
</td>
<td style="text-align:left;">
gene
</td>
<td style="text-align:right;">
49.31927
</td>
<td style="text-align:right;">
49.34159
</td>
<td style="text-align:left;">
positive
</td>
<td style="text-align:left;">
MGI_C57BL6J_1889376
</td>
<td style="text-align:left;">
Ccdc126
</td>
<td style="text-align:left;">
NCBI_Gene:57895,ENSEMBL:ENSMUSG00000050786
</td>
<td style="text-align:left;">
MGI:1889376
</td>
<td style="text-align:left;">
protein coding gene
</td>
<td style="text-align:left;">
coiled-coil domain containing 126
</td>
</tr>
<tr>
<td style="text-align:left;">
6
</td>
<td style="text-align:left;">
gene
</td>
<td style="text-align:right;">
49.36722
</td>
<td style="text-align:right;">
49.39054
</td>
<td style="text-align:left;">
positive
</td>
<td style="text-align:left;">
MGI_C57BL6J_2442161
</td>
<td style="text-align:left;">
Fam221a
</td>
<td style="text-align:left;">
NCBI_Gene:231946,ENSEMBL:ENSMUSG00000047115
</td>
<td style="text-align:left;">
MGI:2442161
</td>
<td style="text-align:left;">
protein coding gene
</td>
<td style="text-align:left;">
family with sequence similarity 221, member A
</td>
</tr>
<tr>
<td style="text-align:left;">
6
</td>
<td style="text-align:left;">
gene
</td>
<td style="text-align:right;">
49.39548
</td>
<td style="text-align:right;">
49.46951
</td>
<td style="text-align:left;">
positive
</td>
<td style="text-align:left;">
MGI_C57BL6J_1924735
</td>
<td style="text-align:left;">
Stk31
</td>
<td style="text-align:left;">
NCBI_Gene:77485,ENSEMBL:ENSMUSG00000023403
</td>
<td style="text-align:left;">
MGI:1924735
</td>
<td style="text-align:left;">
protein coding gene
</td>
<td style="text-align:left;">
serine threonine kinase 31
</td>
</tr>
<tr>
<td style="text-align:left;">
6
</td>
<td style="text-align:left;">
pseudogene
</td>
<td style="text-align:right;">
49.42375
</td>
<td style="text-align:right;">
49.42497
</td>
<td style="text-align:left;">
negative
</td>
<td style="text-align:left;">
MGI_C57BL6J_4937887
</td>
<td style="text-align:left;">
Gm17060
</td>
<td style="text-align:left;">
NCBI_Gene:100416089,ENSEMBL:ENSMUSG00000090624
</td>
<td style="text-align:left;">
MGI:4937887
</td>
<td style="text-align:left;">
pseudogene
</td>
<td style="text-align:left;">
predicted gene 17060
</td>
</tr>
<tr>
<td style="text-align:left;">
6
</td>
<td style="text-align:left;">
pseudogene
</td>
<td style="text-align:right;">
49.54298
</td>
<td style="text-align:right;">
49.56317
</td>
<td style="text-align:left;">
negative
</td>
<td style="text-align:left;">
MGI_C57BL6J_3644734
</td>
<td style="text-align:left;">
Gm5305
</td>
<td style="text-align:left;">
NCBI_Gene:102633308,ENSEMBL:ENSMUSG00000108266
</td>
<td style="text-align:left;">
MGI:3644734
</td>
<td style="text-align:left;">
pseudogene
</td>
<td style="text-align:left;">
predicted gene 5305
</td>
</tr>
<tr>
<td style="text-align:left;">
6
</td>
<td style="text-align:left;">
pseudogene
</td>
<td style="text-align:right;">
49.59624
</td>
<td style="text-align:right;">
49.59976
</td>
<td style="text-align:left;">
positive
</td>
<td style="text-align:left;">
MGI_C57BL6J_3645840
</td>
<td style="text-align:left;">
Rpl19-ps8
</td>
<td style="text-align:left;">
NCBI_Gene:666210
</td>
<td style="text-align:left;">
MGI:3645840
</td>
<td style="text-align:left;">
pseudogene
</td>
<td style="text-align:left;">
ribosomal protein L19, pseudogene 8
</td>
</tr>
<tr>
<td style="text-align:left;">
6
</td>
<td style="text-align:left;">
gene
</td>
<td style="text-align:right;">
49.63363
</td>
<td style="text-align:right;">
49.67845
</td>
<td style="text-align:left;">
positive
</td>
<td style="text-align:left;">
MGI_C57BL6J_5590385
</td>
<td style="text-align:left;">
Gm31226
</td>
<td style="text-align:left;">
NCBI_Gene:102633391
</td>
<td style="text-align:left;">
MGI:5590385
</td>
<td style="text-align:left;">
lncRNA gene
</td>
<td style="text-align:left;">
predicted gene, 31226
</td>
</tr>
<tr>
<td style="text-align:left;">
6
</td>
<td style="text-align:left;">
pseudogene
</td>
<td style="text-align:right;">
49.72725
</td>
<td style="text-align:right;">
49.72752
</td>
<td style="text-align:left;">
negative
</td>
<td style="text-align:left;">
MGI_C57BL6J_5690611
</td>
<td style="text-align:left;">
Gm44219
</td>
<td style="text-align:left;">
ENSEMBL:ENSMUSG00000107982
</td>
<td style="text-align:left;">
MGI:5690611
</td>
<td style="text-align:left;">
pseudogene
</td>
<td style="text-align:left;">
predicted gene, 44219
</td>
</tr>
<tr>
<td style="text-align:left;">
6
</td>
<td style="text-align:left;">
gene
</td>
<td style="text-align:right;">
49.82271
</td>
<td style="text-align:right;">
49.82951
</td>
<td style="text-align:left;">
positive
</td>
<td style="text-align:left;">
MGI_C57BL6J_97374
</td>
<td style="text-align:left;">
Npy
</td>
<td style="text-align:left;">
NCBI_Gene:109648,ENSEMBL:ENSMUSG00000029819
</td>
<td style="text-align:left;">
MGI:97374
</td>
<td style="text-align:left;">
protein coding gene
</td>
<td style="text-align:left;">
neuropeptide Y
</td>
</tr>
<tr>
<td style="text-align:left;">
6
</td>
<td style="text-align:left;">
pseudogene
</td>
<td style="text-align:right;">
49.85800
</td>
<td style="text-align:right;">
49.85885
</td>
<td style="text-align:left;">
negative
</td>
<td style="text-align:left;">
MGI_C57BL6J_5010192
</td>
<td style="text-align:left;">
Gm18007
</td>
<td style="text-align:left;">
NCBI_Gene:100416264,ENSEMBL:ENSMUSG00000107664
</td>
<td style="text-align:left;">
MGI:5010192
</td>
<td style="text-align:left;">
pseudogene
</td>
<td style="text-align:left;">
predicted gene, 18007
</td>
</tr>
<tr>
<td style="text-align:left;">
6
</td>
<td style="text-align:left;">
gene
</td>
<td style="text-align:right;">
50.05119
</td>
<td style="text-align:right;">
50.10763
</td>
<td style="text-align:left;">
positive
</td>
<td style="text-align:left;">
MGI_C57BL6J_3781631
</td>
<td style="text-align:left;">
Gm3455
</td>
<td style="text-align:left;">
ENSEMBL:ENSMUSG00000108071
</td>
<td style="text-align:left;">
MGI:3781631
</td>
<td style="text-align:left;">
lncRNA gene
</td>
<td style="text-align:left;">
predicted gene 3455
</td>
</tr>
<tr>
<td style="text-align:left;">
6
</td>
<td style="text-align:left;">
gene
</td>
<td style="text-align:right;">
50.10766
</td>
<td style="text-align:right;">
50.11016
</td>
<td style="text-align:left;">
negative
</td>
<td style="text-align:left;">
MGI_C57BL6J_5621690
</td>
<td style="text-align:left;">
Gm38805
</td>
<td style="text-align:left;">
NCBI_Gene:105242649
</td>
<td style="text-align:left;">
MGI:5621690
</td>
<td style="text-align:left;">
lncRNA gene
</td>
<td style="text-align:left;">
predicted gene, 38805
</td>
</tr>
<tr>
<td style="text-align:left;">
6
</td>
<td style="text-align:left;">
gene
</td>
<td style="text-align:right;">
50.11024
</td>
<td style="text-align:right;">
50.19894
</td>
<td style="text-align:left;">
positive
</td>
<td style="text-align:left;">
MGI_C57BL6J_1927340
</td>
<td style="text-align:left;">
Mpp6
</td>
<td style="text-align:left;">
NCBI_Gene:56524,ENSEMBL:ENSMUSG00000038388
</td>
<td style="text-align:left;">
MGI:1927340
</td>
<td style="text-align:left;">
protein coding gene
</td>
<td style="text-align:left;">
membrane protein, palmitoylated 6 (MAGUK p55 subfamily member 6)
</td>
</tr>
<tr>
<td style="text-align:left;">
6
</td>
<td style="text-align:left;">
gene
</td>
<td style="text-align:right;">
50.18889
</td>
<td style="text-align:right;">
50.26386
</td>
<td style="text-align:left;">
negative
</td>
<td style="text-align:left;">
MGI_C57BL6J_1889850
</td>
<td style="text-align:left;">
Gsdme
</td>
<td style="text-align:left;">
NCBI_Gene:54722,ENSEMBL:ENSMUSG00000029821
</td>
<td style="text-align:left;">
MGI:1889850
</td>
<td style="text-align:left;">
protein coding gene
</td>
<td style="text-align:left;">
gasdermin E
</td>
</tr>
<tr>
<td style="text-align:left;">
6
</td>
<td style="text-align:left;">
gene
</td>
<td style="text-align:right;">
50.29333
</td>
<td style="text-align:right;">
50.45635
</td>
<td style="text-align:left;">
negative
</td>
<td style="text-align:left;">
MGI_C57BL6J_1918970
</td>
<td style="text-align:left;">
Osbpl3
</td>
<td style="text-align:left;">
NCBI_Gene:71720,ENSEMBL:ENSMUSG00000029822
</td>
<td style="text-align:left;">
MGI:1918970
</td>
<td style="text-align:left;">
protein coding gene
</td>
<td style="text-align:left;">
oxysterol binding protein-like 3
</td>
</tr>
<tr>
<td style="text-align:left;">
6
</td>
<td style="text-align:left;">
gene
</td>
<td style="text-align:right;">
50.49955
</td>
<td style="text-align:right;">
50.50572
</td>
<td style="text-align:left;">
negative
</td>
<td style="text-align:left;">
MGI_C57BL6J_5623257
</td>
<td style="text-align:left;">
Gm40372
</td>
<td style="text-align:left;">
NCBI_Gene:105244836
</td>
<td style="text-align:left;">
MGI:5623257
</td>
<td style="text-align:left;">
lncRNA gene
</td>
<td style="text-align:left;">
predicted gene, 40372
</td>
</tr>
<tr>
<td style="text-align:left;">
6
</td>
<td style="text-align:left;">
gene
</td>
<td style="text-align:right;">
50.50156
</td>
<td style="text-align:right;">
50.53802
</td>
<td style="text-align:left;">
positive
</td>
<td style="text-align:left;">
MGI_C57BL6J_5590438
</td>
<td style="text-align:left;">
Gm31279
</td>
<td style="text-align:left;">
NCBI_Gene:102633461
</td>
<td style="text-align:left;">
MGI:5590438
</td>
<td style="text-align:left;">
lncRNA gene
</td>
<td style="text-align:left;">
predicted gene, 31279
</td>
</tr>
<tr>
<td style="text-align:left;">
6
</td>
<td style="text-align:left;">
gene
</td>
<td style="text-align:right;">
50.53990
</td>
<td style="text-align:right;">
50.54003
</td>
<td style="text-align:left;">
negative
</td>
<td style="text-align:left;">
MGI_C57BL6J_5456066
</td>
<td style="text-align:left;">
Gm26289
</td>
<td style="text-align:left;">
ENSEMBL:ENSMUSG00000077357
</td>
<td style="text-align:left;">
MGI:5456066
</td>
<td style="text-align:left;">
snoRNA gene
</td>
<td style="text-align:left;">
predicted gene, 26289
</td>
</tr>
<tr>
<td style="text-align:left;">
6
</td>
<td style="text-align:left;">
gene
</td>
<td style="text-align:right;">
50.54076
</td>
<td style="text-align:right;">
50.54298
</td>
<td style="text-align:left;">
positive
</td>
<td style="text-align:left;">
MGI_C57BL6J_5621691
</td>
<td style="text-align:left;">
Gm38806
</td>
<td style="text-align:left;">
NCBI_Gene:105242651
</td>
<td style="text-align:left;">
MGI:5621691
</td>
<td style="text-align:left;">
lncRNA gene
</td>
<td style="text-align:left;">
predicted gene, 38806
</td>
</tr>
<tr>
<td style="text-align:left;">
6
</td>
<td style="text-align:left;">
gene
</td>
<td style="text-align:right;">
50.56256
</td>
<td style="text-align:right;">
50.56654
</td>
<td style="text-align:left;">
negative
</td>
<td style="text-align:left;">
MGI_C57BL6J_88578
</td>
<td style="text-align:left;">
Cycs
</td>
<td style="text-align:left;">
NCBI_Gene:13063,ENSEMBL:ENSMUSG00000063694
</td>
<td style="text-align:left;">
MGI:88578
</td>
<td style="text-align:left;">
protein coding gene
</td>
<td style="text-align:left;">
cytochrome c, somatic
</td>
</tr>
<tr>
<td style="text-align:left;">
6
</td>
<td style="text-align:left;">
gene
</td>
<td style="text-align:right;">
50.56664
</td>
<td style="text-align:right;">
50.59486
</td>
<td style="text-align:left;">
positive
</td>
<td style="text-align:left;">
MGI_C57BL6J_1918631
</td>
<td style="text-align:left;">
5430402O13Rik
</td>
<td style="text-align:left;">
NCBI_Gene:73183,ENSEMBL:ENSMUSG00000085776
</td>
<td style="text-align:left;">
MGI:1918631
</td>
<td style="text-align:left;">
lncRNA gene
</td>
<td style="text-align:left;">
RIKEN cDNA 5430402O13 gene
</td>
</tr>
<tr>
<td style="text-align:left;">
6
</td>
<td style="text-align:left;">
gene
</td>
<td style="text-align:right;">
50.57240
</td>
<td style="text-align:right;">
50.57251
</td>
<td style="text-align:left;">
positive
</td>
<td style="text-align:left;">
MGI_C57BL6J_5530758
</td>
<td style="text-align:left;">
Mir6371
</td>
<td style="text-align:left;">
miRBase:MI0021900,NCBI_Gene:102465193,ENSEMBL:ENSMUSG00000099221
</td>
<td style="text-align:left;">
MGI:5530758
</td>
<td style="text-align:left;">
miRNA gene
</td>
<td style="text-align:left;">
microRNA 6371
</td>
</tr>
<tr>
<td style="text-align:left;">
6
</td>
<td style="text-align:left;">
gene
</td>
<td style="text-align:right;">
50.57330
</td>
<td style="text-align:right;">
50.59691
</td>
<td style="text-align:left;">
negative
</td>
<td style="text-align:left;">
MGI_C57BL6J_1918071
</td>
<td style="text-align:left;">
4921507P07Rik
</td>
<td style="text-align:left;">
NCBI_Gene:70821,ENSEMBL:ENSMUSG00000029828
</td>
<td style="text-align:left;">
MGI:1918071
</td>
<td style="text-align:left;">
protein coding gene
</td>
<td style="text-align:left;">
RIKEN cDNA 4921507P07 gene
</td>
</tr>
<tr>
<td style="text-align:left;">
6
</td>
<td style="text-align:left;">
pseudogene
</td>
<td style="text-align:right;">
50.59987
</td>
<td style="text-align:right;">
50.60024
</td>
<td style="text-align:left;">
positive
</td>
<td style="text-align:left;">
MGI_C57BL6J_5690446
</td>
<td style="text-align:left;">
Gm44054
</td>
<td style="text-align:left;">
ENSEMBL:ENSMUSG00000107447
</td>
<td style="text-align:left;">
MGI:5690446
</td>
<td style="text-align:left;">
pseudogene
</td>
<td style="text-align:left;">
predicted gene, 44054
</td>
</tr>
<tr>
<td style="text-align:left;">
6
</td>
<td style="text-align:left;">
gene
</td>
<td style="text-align:right;">
50.60154
</td>
<td style="text-align:right;">
50.60528
</td>
<td style="text-align:left;">
negative
</td>
<td style="text-align:left;">
MGI_C57BL6J_5621692
</td>
<td style="text-align:left;">
Gm38807
</td>
<td style="text-align:left;">
NCBI_Gene:105242652
</td>
<td style="text-align:left;">
MGI:5621692
</td>
<td style="text-align:left;">
lncRNA gene
</td>
<td style="text-align:left;">
predicted gene, 38807
</td>
</tr>
<tr>
<td style="text-align:left;">
6
</td>
<td style="text-align:left;">
pseudogene
</td>
<td style="text-align:right;">
50.60864
</td>
<td style="text-align:right;">
50.61069
</td>
<td style="text-align:left;">
positive
</td>
<td style="text-align:left;">
MGI_C57BL6J_3646036
</td>
<td style="text-align:left;">
Gm4782
</td>
<td style="text-align:left;">
NCBI_Gene:213320,ENSEMBL:ENSMUSG00000107418
</td>
<td style="text-align:left;">
MGI:3646036
</td>
<td style="text-align:left;">
pseudogene
</td>
<td style="text-align:left;">
predicted gene 4782
</td>
</tr>
<tr>
<td style="text-align:left;">
6
</td>
<td style="text-align:left;">
pseudogene
</td>
<td style="text-align:right;">
50.61785
</td>
<td style="text-align:right;">
50.61854
</td>
<td style="text-align:left;">
negative
</td>
<td style="text-align:left;">
MGI_C57BL6J_3644146
</td>
<td style="text-align:left;">
Gm8035
</td>
<td style="text-align:left;">
NCBI_Gene:666302,ENSEMBL:ENSMUSG00000108135
</td>
<td style="text-align:left;">
MGI:3644146
</td>
<td style="text-align:left;">
pseudogene
</td>
<td style="text-align:left;">
predicted gene 8035
</td>
</tr>
<tr>
<td style="text-align:left;">
6
</td>
<td style="text-align:left;">
pseudogene
</td>
<td style="text-align:right;">
50.61984
</td>
<td style="text-align:right;">
50.62188
</td>
<td style="text-align:left;">
negative
</td>
<td style="text-align:left;">
MGI_C57BL6J_3644148
</td>
<td style="text-align:left;">
Gm8038
</td>
<td style="text-align:left;">
NCBI_Gene:666308,ENSEMBL:ENSMUSG00000107821
</td>
<td style="text-align:left;">
MGI:3644148
</td>
<td style="text-align:left;">
pseudogene
</td>
<td style="text-align:left;">
predicted gene 8038
</td>
</tr>
<tr>
<td style="text-align:left;">
6
</td>
<td style="text-align:left;">
pseudogene
</td>
<td style="text-align:right;">
50.62574
</td>
<td style="text-align:right;">
50.62646
</td>
<td style="text-align:left;">
negative
</td>
<td style="text-align:left;">
MGI_C57BL6J_3647078
</td>
<td style="text-align:left;">
Gm8039
</td>
<td style="text-align:left;">
NCBI_Gene:666310,ENSEMBL:ENSMUSG00000108016
</td>
<td style="text-align:left;">
MGI:3647078
</td>
<td style="text-align:left;">
pseudogene
</td>
<td style="text-align:left;">
predicted gene 8039
</td>
</tr>
<tr>
<td style="text-align:left;">
6
</td>
<td style="text-align:left;">
pseudogene
</td>
<td style="text-align:right;">
50.62768
</td>
<td style="text-align:right;">
50.62834
</td>
<td style="text-align:left;">
negative
</td>
<td style="text-align:left;">
MGI_C57BL6J_5010950
</td>
<td style="text-align:left;">
Gm18765
</td>
<td style="text-align:left;">
NCBI_Gene:100417688,ENSEMBL:ENSMUSG00000107740
</td>
<td style="text-align:left;">
MGI:5010950
</td>
<td style="text-align:left;">
pseudogene
</td>
<td style="text-align:left;">
predicted gene, 18765
</td>
</tr>
<tr>
<td style="text-align:left;">
6
</td>
<td style="text-align:left;">
gene
</td>
<td style="text-align:right;">
50.65067
</td>
<td style="text-align:right;">
50.65444
</td>
<td style="text-align:left;">
negative
</td>
<td style="text-align:left;">
MGI_C57BL6J_1926488
</td>
<td style="text-align:left;">
Npvf
</td>
<td style="text-align:left;">
NCBI_Gene:60531,ENSEMBL:ENSMUSG00000029831
</td>
<td style="text-align:left;">
MGI:1926488
</td>
<td style="text-align:left;">
protein coding gene
</td>
<td style="text-align:left;">
neuropeptide VF precursor
</td>
</tr>
<tr>
<td style="text-align:left;">
6
</td>
<td style="text-align:left;">
gene
</td>
<td style="text-align:right;">
50.65985
</td>
<td style="text-align:right;">
50.67948
</td>
<td style="text-align:left;">
negative
</td>
<td style="text-align:left;">
MGI_C57BL6J_5621693
</td>
<td style="text-align:left;">
Gm38808
</td>
<td style="text-align:left;">
NCBI_Gene:105242653
</td>
<td style="text-align:left;">
MGI:5621693
</td>
<td style="text-align:left;">
lncRNA gene
</td>
<td style="text-align:left;">
predicted gene, 38808
</td>
</tr>
<tr>
<td style="text-align:left;">
6
</td>
<td style="text-align:left;">
pseudogene
</td>
<td style="text-align:right;">
50.74668
</td>
<td style="text-align:right;">
50.74781
</td>
<td style="text-align:left;">
negative
</td>
<td style="text-align:left;">
MGI_C57BL6J_3781195
</td>
<td style="text-align:left;">
Gm3017
</td>
<td style="text-align:left;">
NCBI_Gene:100040877,ENSEMBL:ENSMUSG00000107672
</td>
<td style="text-align:left;">
MGI:3781195
</td>
<td style="text-align:left;">
pseudogene
</td>
<td style="text-align:left;">
predicted gene 3017
</td>
</tr>
<tr>
<td style="text-align:left;">
6
</td>
<td style="text-align:left;">
gene
</td>
<td style="text-align:right;">
50.77611
</td>
<td style="text-align:right;">
50.81503
</td>
<td style="text-align:left;">
positive
</td>
<td style="text-align:left;">
MGI_C57BL6J_2443109
</td>
<td style="text-align:left;">
C530044C16Rik
</td>
<td style="text-align:left;">
NCBI_Gene:319981,ENSEMBL:ENSMUSG00000107667
</td>
<td style="text-align:left;">
MGI:2443109
</td>
<td style="text-align:left;">
lncRNA gene
</td>
<td style="text-align:left;">
RIKEN cDNA C530044C16 gene
</td>
</tr>
<tr>
<td style="text-align:left;">
6
</td>
<td style="text-align:left;">
gene
</td>
<td style="text-align:right;">
50.79907
</td>
<td style="text-align:right;">
50.80303
</td>
<td style="text-align:left;">
negative
</td>
<td style="text-align:left;">
MGI_C57BL6J_5690501
</td>
<td style="text-align:left;">
Gm44109
</td>
<td style="text-align:left;">
ENSEMBL:ENSMUSG00000107978
</td>
<td style="text-align:left;">
MGI:5690501
</td>
<td style="text-align:left;">
lncRNA gene
</td>
<td style="text-align:left;">
predicted gene, 44109
</td>
</tr>
<tr>
<td style="text-align:left;">
6
</td>
<td style="text-align:left;">
gene
</td>
<td style="text-align:right;">
50.80427
</td>
<td style="text-align:right;">
50.82229
</td>
<td style="text-align:left;">
negative
</td>
<td style="text-align:left;">
MGI_C57BL6J_5590614
</td>
<td style="text-align:left;">
Gm31455
</td>
<td style="text-align:left;">
NCBI_Gene:102633693
</td>
<td style="text-align:left;">
MGI:5590614
</td>
<td style="text-align:left;">
lncRNA gene
</td>
<td style="text-align:left;">
predicted gene, 31455
</td>
</tr>
<tr>
<td style="text-align:left;">
6
</td>
<td style="text-align:left;">
gene
</td>
<td style="text-align:right;">
50.84671
</td>
<td style="text-align:right;">
50.87808
</td>
<td style="text-align:left;">
negative
</td>
<td style="text-align:left;">
MGI_C57BL6J_3588242
</td>
<td style="text-align:left;">
G930045G22Rik
</td>
<td style="text-align:left;">
ENSEMBL:ENSMUSG00000107993
</td>
<td style="text-align:left;">
MGI:3588242
</td>
<td style="text-align:left;">
lncRNA gene
</td>
<td style="text-align:left;">
RIKEN cDNA G930045G22 gene
</td>
</tr>
<tr>
<td style="text-align:left;">
6
</td>
<td style="text-align:left;">
gene
</td>
<td style="text-align:right;">
50.88089
</td>
<td style="text-align:right;">
50.88262
</td>
<td style="text-align:left;">
negative
</td>
<td style="text-align:left;">
MGI_C57BL6J_5825559
</td>
<td style="text-align:left;">
Gm45922
</td>
<td style="text-align:left;">
NCBI_Gene:105247310
</td>
<td style="text-align:left;">
MGI:5825559
</td>
<td style="text-align:left;">
lncRNA gene
</td>
<td style="text-align:left;">
predicted gene, 45922
</td>
</tr>
<tr>
<td style="text-align:left;">
6
</td>
<td style="text-align:left;">
gene
</td>
<td style="text-align:right;">
50.90046
</td>
<td style="text-align:right;">
50.90898
</td>
<td style="text-align:left;">
negative
</td>
<td style="text-align:left;">
MGI_C57BL6J_5590790
</td>
<td style="text-align:left;">
Gm31631
</td>
<td style="text-align:left;">
NCBI_Gene:102633926
</td>
<td style="text-align:left;">
MGI:5590790
</td>
<td style="text-align:left;">
lncRNA gene
</td>
<td style="text-align:left;">
predicted gene, 31631
</td>
</tr>
<tr>
<td style="text-align:left;">
6
</td>
<td style="text-align:left;">
gene
</td>
<td style="text-align:right;">
50.90640
</td>
<td style="text-align:right;">
50.91074
</td>
<td style="text-align:left;">
positive
</td>
<td style="text-align:left;">
MGI_C57BL6J_5590738
</td>
<td style="text-align:left;">
Gm31579
</td>
<td style="text-align:left;">
NCBI_Gene:102633851,ENSEMBL:ENSMUSG00000107488
</td>
<td style="text-align:left;">
MGI:5590738
</td>
<td style="text-align:left;">
lncRNA gene
</td>
<td style="text-align:left;">
predicted gene, 31579
</td>
</tr>
<tr>
<td style="text-align:left;">
6
</td>
<td style="text-align:left;">
gene
</td>
<td style="text-align:right;">
50.93500
</td>
<td style="text-align:right;">
50.94438
</td>
<td style="text-align:left;">
negative
</td>
<td style="text-align:left;">
MGI_C57BL6J_5590845
</td>
<td style="text-align:left;">
Gm31686
</td>
<td style="text-align:left;">
NCBI_Gene:102633996
</td>
<td style="text-align:left;">
MGI:5590845
</td>
<td style="text-align:left;">
lncRNA gene
</td>
<td style="text-align:left;">
predicted gene, 31686
</td>
</tr>
<tr>
<td style="text-align:left;">
6
</td>
<td style="text-align:left;">
pseudogene
</td>
<td style="text-align:right;">
50.96727
</td>
<td style="text-align:right;">
50.96751
</td>
<td style="text-align:left;">
positive
</td>
<td style="text-align:left;">
MGI_C57BL6J_5690794
</td>
<td style="text-align:left;">
Gm44402
</td>
<td style="text-align:left;">
ENSEMBL:ENSMUSG00000108153
</td>
<td style="text-align:left;">
MGI:5690794
</td>
<td style="text-align:left;">
pseudogene
</td>
<td style="text-align:left;">
predicted gene, 44402
</td>
</tr>
<tr>
<td style="text-align:left;">
6
</td>
<td style="text-align:left;">
gene
</td>
<td style="text-align:right;">
50.97338
</td>
<td style="text-align:right;">
50.99037
</td>
<td style="text-align:left;">
negative
</td>
<td style="text-align:left;">
MGI_C57BL6J_5621694
</td>
<td style="text-align:left;">
Gm38809
</td>
<td style="text-align:left;">
NCBI_Gene:105242654
</td>
<td style="text-align:left;">
MGI:5621694
</td>
<td style="text-align:left;">
lncRNA gene
</td>
<td style="text-align:left;">
predicted gene, 38809
</td>
</tr>
<tr>
<td style="text-align:left;">
6
</td>
<td style="text-align:left;">
gene
</td>
<td style="text-align:right;">
50.98740
</td>
<td style="text-align:right;">
51.06801
</td>
<td style="text-align:left;">
positive
</td>
<td style="text-align:left;">
MGI_C57BL6J_5826596
</td>
<td style="text-align:left;">
Gm46959
</td>
<td style="text-align:left;">
NCBI_Gene:108169141
</td>
<td style="text-align:left;">
MGI:5826596
</td>
<td style="text-align:left;">
lncRNA gene
</td>
<td style="text-align:left;">
predicted gene, 46959
</td>
</tr>
<tr>
<td style="text-align:left;">
6
</td>
<td style="text-align:left;">
gene
</td>
<td style="text-align:right;">
51.03294
</td>
<td style="text-align:right;">
51.06126
</td>
<td style="text-align:left;">
negative
</td>
<td style="text-align:left;">
MGI_C57BL6J_5590905
</td>
<td style="text-align:left;">
Gm31746
</td>
<td style="text-align:left;">
NCBI_Gene:102634074
</td>
<td style="text-align:left;">
MGI:5590905
</td>
<td style="text-align:left;">
lncRNA gene
</td>
<td style="text-align:left;">
predicted gene, 31746
</td>
</tr>
<tr>
<td style="text-align:left;">
6
</td>
<td style="text-align:left;">
gene
</td>
<td style="text-align:right;">
51.09651
</td>
<td style="text-align:right;">
51.10067
</td>
<td style="text-align:left;">
negative
</td>
<td style="text-align:left;">
MGI_C57BL6J_5591349
</td>
<td style="text-align:left;">
Gm32190
</td>
<td style="text-align:left;">
NCBI_Gene:102634656
</td>
<td style="text-align:left;">
MGI:5591349
</td>
<td style="text-align:left;">
lncRNA gene
</td>
<td style="text-align:left;">
predicted gene, 32190
</td>
</tr>
<tr>
<td style="text-align:left;">
6
</td>
<td style="text-align:left;">
gene
</td>
<td style="text-align:right;">
51.12475
</td>
<td style="text-align:right;">
51.16645
</td>
<td style="text-align:left;">
negative
</td>
<td style="text-align:left;">
MGI_C57BL6J_5591487
</td>
<td style="text-align:left;">
Gm32328
</td>
<td style="text-align:left;">
NCBI_Gene:102634838
</td>
<td style="text-align:left;">
MGI:5591487
</td>
<td style="text-align:left;">
lncRNA gene
</td>
<td style="text-align:left;">
predicted gene, 32328
</td>
</tr>
<tr>
<td style="text-align:left;">
6
</td>
<td style="text-align:left;">
gene
</td>
<td style="text-align:right;">
51.14201
</td>
<td style="text-align:right;">
51.14608
</td>
<td style="text-align:left;">
positive
</td>
<td style="text-align:left;">
MGI_C57BL6J_5591405
</td>
<td style="text-align:left;">
Gm32246
</td>
<td style="text-align:left;">
NCBI_Gene:102634734
</td>
<td style="text-align:left;">
MGI:5591405
</td>
<td style="text-align:left;">
lncRNA gene
</td>
<td style="text-align:left;">
predicted gene, 32246
</td>
</tr>
<tr>
<td style="text-align:left;">
6
</td>
<td style="text-align:left;">
gene
</td>
<td style="text-align:right;">
51.15428
</td>
<td style="text-align:right;">
51.15493
</td>
<td style="text-align:left;">
positive
</td>
<td style="text-align:left;">
MGI_C57BL6J_5621695
</td>
<td style="text-align:left;">
Gm38810
</td>
<td style="text-align:left;">
NCBI_Gene:105242655
</td>
<td style="text-align:left;">
MGI:5621695
</td>
<td style="text-align:left;">
lncRNA gene
</td>
<td style="text-align:left;">
predicted gene, 38810
</td>
</tr>
<tr>
<td style="text-align:left;">
6
</td>
<td style="text-align:left;">
gene
</td>
<td style="text-align:right;">
51.20092
</td>
<td style="text-align:right;">
51.20735
</td>
<td style="text-align:left;">
negative
</td>
<td style="text-align:left;">
MGI_C57BL6J_5591562
</td>
<td style="text-align:left;">
Gm32403
</td>
<td style="text-align:left;">
NCBI_Gene:102634943
</td>
<td style="text-align:left;">
MGI:5591562
</td>
<td style="text-align:left;">
lncRNA gene
</td>
<td style="text-align:left;">
predicted gene, 32403
</td>
</tr>
<tr>
<td style="text-align:left;">
6
</td>
<td style="text-align:left;">
gene
</td>
<td style="text-align:right;">
51.22296
</td>
<td style="text-align:right;">
51.28057
</td>
<td style="text-align:left;">
negative
</td>
<td style="text-align:left;">
MGI_C57BL6J_5621706
</td>
<td style="text-align:left;">
Gm38821
</td>
<td style="text-align:left;">
NCBI_Gene:105242668
</td>
<td style="text-align:left;">
MGI:5621706
</td>
<td style="text-align:left;">
protein coding gene
</td>
<td style="text-align:left;">
predicted gene, 38821
</td>
</tr>
<tr>
<td style="text-align:left;">
6
</td>
<td style="text-align:left;">
gene
</td>
<td style="text-align:right;">
51.26981
</td>
<td style="text-align:right;">
51.26991
</td>
<td style="text-align:left;">
negative
</td>
<td style="text-align:left;">
MGI_C57BL6J_2676833
</td>
<td style="text-align:left;">
Mir148a
</td>
<td style="text-align:left;">
miRBase:MI0000550,NCBI_Gene:387166,ENSEMBL:ENSMUSG00000065505
</td>
<td style="text-align:left;">
MGI:2676833
</td>
<td style="text-align:left;">
miRNA gene
</td>
<td style="text-align:left;">
microRNA 148a
</td>
</tr>
<tr>
<td style="text-align:left;">
6
</td>
<td style="text-align:left;">
gene
</td>
<td style="text-align:right;">
51.28353
</td>
<td style="text-align:right;">
51.30864
</td>
<td style="text-align:left;">
negative
</td>
<td style="text-align:left;">
MGI_C57BL6J_5591638
</td>
<td style="text-align:left;">
Gm32479
</td>
<td style="text-align:left;">
NCBI_Gene:102635043,ENSEMBL:ENSMUSG00000107619
</td>
<td style="text-align:left;">
MGI:5591638
</td>
<td style="text-align:left;">
lncRNA gene
</td>
<td style="text-align:left;">
predicted gene, 32479
</td>
</tr>
<tr>
<td style="text-align:left;">
6
</td>
<td style="text-align:left;">
gene
</td>
<td style="text-align:right;">
51.33049
</td>
<td style="text-align:right;">
51.33371
</td>
<td style="text-align:left;">
positive
</td>
<td style="text-align:left;">
MGI_C57BL6J_2679266
</td>
<td style="text-align:left;">
BC022713
</td>
<td style="text-align:left;">
NA
</td>
<td style="text-align:left;">
NA
</td>
<td style="text-align:left;">
unclassified gene
</td>
<td style="text-align:left;">
cDNA sequence BC022713
</td>
</tr>
<tr>
<td style="text-align:left;">
6
</td>
<td style="text-align:left;">
gene
</td>
<td style="text-align:right;">
51.37971
</td>
<td style="text-align:right;">
51.39278
</td>
<td style="text-align:left;">
negative
</td>
<td style="text-align:left;">
MGI_C57BL6J_5477171
</td>
<td style="text-align:left;">
Gm26677
</td>
<td style="text-align:left;">
ENSEMBL:ENSMUSG00000097598
</td>
<td style="text-align:left;">
MGI:5477171
</td>
<td style="text-align:left;">
lncRNA gene
</td>
<td style="text-align:left;">
predicted gene, 26677
</td>
</tr>
<tr>
<td style="text-align:left;">
6
</td>
<td style="text-align:left;">
gene
</td>
<td style="text-align:right;">
51.37971
</td>
<td style="text-align:right;">
51.39279
</td>
<td style="text-align:left;">
positive
</td>
<td style="text-align:left;">
MGI_C57BL6J_3643041
</td>
<td style="text-align:left;">
Gm6559
</td>
<td style="text-align:left;">
NCBI_Gene:625167,ENSEMBL:ENSMUSG00000107660
</td>
<td style="text-align:left;">
MGI:3643041
</td>
<td style="text-align:left;">
lncRNA gene
</td>
<td style="text-align:left;">
predicted gene 6559
</td>
</tr>
<tr>
<td style="text-align:left;">
6
</td>
<td style="text-align:left;">
gene
</td>
<td style="text-align:right;">
51.40097
</td>
<td style="text-align:right;">
51.40222
</td>
<td style="text-align:left;">
negative
</td>
<td style="text-align:left;">
MGI_C57BL6J_1921343
</td>
<td style="text-align:left;">
0610033M10Rik
</td>
<td style="text-align:left;">
ENSEMBL:ENSMUSG00000108236
</td>
<td style="text-align:left;">
MGI:1921343
</td>
<td style="text-align:left;">
lncRNA gene
</td>
<td style="text-align:left;">
RIKEN cDNA 0610033M10 gene
</td>
</tr>
<tr>
<td style="text-align:left;">
6
</td>
<td style="text-align:left;">
gene
</td>
<td style="text-align:right;">
51.43267
</td>
<td style="text-align:right;">
51.45877
</td>
<td style="text-align:left;">
positive
</td>
<td style="text-align:left;">
MGI_C57BL6J_1339958
</td>
<td style="text-align:left;">
Nfe2l3
</td>
<td style="text-align:left;">
NCBI_Gene:18025,ENSEMBL:ENSMUSG00000029832
</td>
<td style="text-align:left;">
MGI:1339958
</td>
<td style="text-align:left;">
protein coding gene
</td>
<td style="text-align:left;">
nuclear factor, erythroid derived 2, like 3
</td>
</tr>
<tr>
<td style="text-align:left;">
6
</td>
<td style="text-align:left;">
gene
</td>
<td style="text-align:right;">
51.46043
</td>
<td style="text-align:right;">
51.47031
</td>
<td style="text-align:left;">
negative
</td>
<td style="text-align:left;">
MGI_C57BL6J_104819
</td>
<td style="text-align:left;">
Hnrnpa2b1
</td>
<td style="text-align:left;">
NCBI_Gene:53379,ENSEMBL:ENSMUSG00000004980
</td>
<td style="text-align:left;">
MGI:104819
</td>
<td style="text-align:left;">
protein coding gene
</td>
<td style="text-align:left;">
heterogeneous nuclear ribonucleoprotein A2/B1
</td>
</tr>
<tr>
<td style="text-align:left;">
6
</td>
<td style="text-align:left;">
gene
</td>
<td style="text-align:right;">
51.47036
</td>
<td style="text-align:right;">
51.48370
</td>
<td style="text-align:left;">
positive
</td>
<td style="text-align:left;">
MGI_C57BL6J_108515
</td>
<td style="text-align:left;">
Cbx3
</td>
<td style="text-align:left;">
NCBI_Gene:12417,ENSEMBL:ENSMUSG00000029836
</td>
<td style="text-align:left;">
MGI:108515
</td>
<td style="text-align:left;">
protein coding gene
</td>
<td style="text-align:left;">
chromobox 3
</td>
</tr>
<tr>
<td style="text-align:left;">
6
</td>
<td style="text-align:left;">
gene
</td>
<td style="text-align:right;">
51.52390
</td>
<td style="text-align:right;">
51.59068
</td>
<td style="text-align:left;">
positive
</td>
<td style="text-align:left;">
MGI_C57BL6J_1919232
</td>
<td style="text-align:left;">
Snx10
</td>
<td style="text-align:left;">
NCBI_Gene:71982,ENSEMBL:ENSMUSG00000038301
</td>
<td style="text-align:left;">
MGI:1919232
</td>
<td style="text-align:left;">
protein coding gene
</td>
<td style="text-align:left;">
sorting nexin 10
</td>
</tr>
<tr>
<td style="text-align:left;">
6
</td>
<td style="text-align:left;">
gene
</td>
<td style="text-align:right;">
51.62163
</td>
<td style="text-align:right;">
51.62171
</td>
<td style="text-align:left;">
positive
</td>
<td style="text-align:left;">
MGI_C57BL6J_5452691
</td>
<td style="text-align:left;">
Gm22914
</td>
<td style="text-align:left;">
ENSEMBL:ENSMUSG00000094528
</td>
<td style="text-align:left;">
MGI:5452691
</td>
<td style="text-align:left;">
miRNA gene
</td>
<td style="text-align:left;">
predicted gene, 22914
</td>
</tr>
<tr>
<td style="text-align:left;">
6
</td>
<td style="text-align:left;">
gene
</td>
<td style="text-align:right;">
51.69431
</td>
<td style="text-align:right;">
51.71108
</td>
<td style="text-align:left;">
negative
</td>
<td style="text-align:left;">
MGI_C57BL6J_5621696
</td>
<td style="text-align:left;">
Gm38811
</td>
<td style="text-align:left;">
NCBI_Gene:105242656,ENSEMBL:ENSMUSG00000108145
</td>
<td style="text-align:left;">
MGI:5621696
</td>
<td style="text-align:left;">
lncRNA gene
</td>
<td style="text-align:left;">
predicted gene, 38811
</td>
</tr>
<tr>
<td style="text-align:left;">
6
</td>
<td style="text-align:left;">
gene
</td>
<td style="text-align:right;">
51.85742
</td>
<td style="text-align:right;">
52.01255
</td>
<td style="text-align:left;">
negative
</td>
<td style="text-align:left;">
MGI_C57BL6J_1889206
</td>
<td style="text-align:left;">
Skap2
</td>
<td style="text-align:left;">
NCBI_Gene:54353,ENSEMBL:ENSMUSG00000059182
</td>
<td style="text-align:left;">
MGI:1889206
</td>
<td style="text-align:left;">
protein coding gene
</td>
<td style="text-align:left;">
src family associated phosphoprotein 2
</td>
</tr>
<tr>
<td style="text-align:left;">
6
</td>
<td style="text-align:left;">
gene
</td>
<td style="text-align:right;">
52.01260
</td>
<td style="text-align:right;">
52.02699
</td>
<td style="text-align:left;">
positive
</td>
<td style="text-align:left;">
MGI_C57BL6J_5690469
</td>
<td style="text-align:left;">
Gm44077
</td>
<td style="text-align:left;">
ENSEMBL:ENSMUSG00000108244
</td>
<td style="text-align:left;">
MGI:5690469
</td>
<td style="text-align:left;">
lncRNA gene
</td>
<td style="text-align:left;">
predicted gene, 44077
</td>
</tr>
<tr>
<td style="text-align:left;">
6
</td>
<td style="text-align:left;">
gene
</td>
<td style="text-align:right;">
52.10295
</td>
<td style="text-align:right;">
52.11368
</td>
<td style="text-align:left;">
positive
</td>
<td style="text-align:left;">
MGI_C57BL6J_3705267
</td>
<td style="text-align:left;">
Halr1
</td>
<td style="text-align:left;">
NCBI_Gene:102635415,ENSEMBL:ENSMUSG00000085412
</td>
<td style="text-align:left;">
MGI:3705267
</td>
<td style="text-align:left;">
lncRNA gene
</td>
<td style="text-align:left;">
Hoxa adjacent long noncoding RNA 1
</td>
</tr>
<tr>
<td style="text-align:left;">
6
</td>
<td style="text-align:left;">
gene
</td>
<td style="text-align:right;">
52.10480
</td>
<td style="text-align:right;">
52.10616
</td>
<td style="text-align:left;">
positive
</td>
<td style="text-align:left;">
MGI_C57BL6J_5636487
</td>
<td style="text-align:left;">
Halr1os1
</td>
<td style="text-align:left;">
NA
</td>
<td style="text-align:left;">
NA
</td>
<td style="text-align:left;">
unclassified non-coding RNA gene
</td>
<td style="text-align:left;">
Hoxa adjacent long noncoding RNA 1%2c opposite strand 1
</td>
</tr>
<tr>
<td style="text-align:left;">
6
</td>
<td style="text-align:left;">
pseudogene
</td>
<td style="text-align:right;">
52.11822
</td>
<td style="text-align:right;">
52.11894
</td>
<td style="text-align:left;">
positive
</td>
<td style="text-align:left;">
MGI_C57BL6J_3646130
</td>
<td style="text-align:left;">
Rps8-ps3
</td>
<td style="text-align:left;">
NCBI_Gene:666391,ENSEMBL:ENSMUSG00000081979
</td>
<td style="text-align:left;">
MGI:3646130
</td>
<td style="text-align:left;">
pseudogene
</td>
<td style="text-align:left;">
ribosomal protein S8, pseudogene 3
</td>
</tr>
<tr>
<td style="text-align:left;">
6
</td>
<td style="text-align:left;">
pseudogene
</td>
<td style="text-align:right;">
52.13178
</td>
<td style="text-align:right;">
52.13205
</td>
<td style="text-align:left;">
negative
</td>
<td style="text-align:left;">
MGI_C57BL6J_3705565
</td>
<td style="text-align:left;">
Rps19-ps14
</td>
<td style="text-align:left;">
ENSEMBL:ENSMUSG00000082781
</td>
<td style="text-align:left;">
MGI:3705565
</td>
<td style="text-align:left;">
pseudogene
</td>
<td style="text-align:left;">
ribosomal protein S19, pseudogene 14
</td>
</tr>
<tr>
<td style="text-align:left;">
6
</td>
<td style="text-align:left;">
gene
</td>
<td style="text-align:right;">
52.15537
</td>
<td style="text-align:right;">
52.15929
</td>
<td style="text-align:left;">
negative
</td>
<td style="text-align:left;">
MGI_C57BL6J_96170
</td>
<td style="text-align:left;">
Hoxa1
</td>
<td style="text-align:left;">
NCBI_Gene:15394,ENSEMBL:ENSMUSG00000029844
</td>
<td style="text-align:left;">
MGI:96170
</td>
<td style="text-align:left;">
protein coding gene
</td>
<td style="text-align:left;">
homeobox A1
</td>
</tr>
<tr>
<td style="text-align:left;">
6
</td>
<td style="text-align:left;">
gene
</td>
<td style="text-align:right;">
52.15690
</td>
<td style="text-align:right;">
52.15853
</td>
<td style="text-align:left;">
positive
</td>
<td style="text-align:left;">
MGI_C57BL6J_5636492
</td>
<td style="text-align:left;">
Hotairm2
</td>
<td style="text-align:left;">
NA
</td>
<td style="text-align:left;">
NA
</td>
<td style="text-align:left;">
unclassified non-coding RNA gene
</td>
<td style="text-align:left;">
Hoxa transcript antisense RNA%2c myeloid-specific 2
</td>
</tr>
<tr>
<td style="text-align:left;">
6
</td>
<td style="text-align:left;">
gene
</td>
<td style="text-align:right;">
52.15690
</td>
<td style="text-align:right;">
52.16212
</td>
<td style="text-align:left;">
positive
</td>
<td style="text-align:left;">
MGI_C57BL6J_3705155
</td>
<td style="text-align:left;">
Hotairm1
</td>
<td style="text-align:left;">
NCBI_Gene:105242658,ENSEMBL:ENSMUSG00000087658
</td>
<td style="text-align:left;">
MGI:3705155
</td>
<td style="text-align:left;">
antisense lncRNA gene
</td>
<td style="text-align:left;">
Hoxa transcript antisense RNA, myeloid-specific 1
</td>
</tr>
<tr>
<td style="text-align:left;">
6
</td>
<td style="text-align:left;">
gene
</td>
<td style="text-align:right;">
52.15843
</td>
<td style="text-align:right;">
52.15856
</td>
<td style="text-align:left;">
positive
</td>
<td style="text-align:left;">
MGI_C57BL6J_5531249
</td>
<td style="text-align:left;">
Gm27867
</td>
<td style="text-align:left;">
ENSEMBL:ENSMUSG00000098537
</td>
<td style="text-align:left;">
MGI:5531249
</td>
<td style="text-align:left;">
unclassified non-coding RNA gene
</td>
<td style="text-align:left;">
predicted gene, 27867
</td>
</tr>
<tr>
<td style="text-align:left;">
6
</td>
<td style="text-align:left;">
gene
</td>
<td style="text-align:right;">
52.16104
</td>
<td style="text-align:right;">
52.16125
</td>
<td style="text-align:left;">
positive
</td>
<td style="text-align:left;">
MGI_C57BL6J_5531243
</td>
<td style="text-align:left;">
Gm27861
</td>
<td style="text-align:left;">
ENSEMBL:ENSMUSG00000099068
</td>
<td style="text-align:left;">
MGI:5531243
</td>
<td style="text-align:left;">
unclassified non-coding RNA gene
</td>
<td style="text-align:left;">
predicted gene, 27861
</td>
</tr>
<tr>
<td style="text-align:left;">
6
</td>
<td style="text-align:left;">
gene
</td>
<td style="text-align:right;">
52.16201
</td>
<td style="text-align:right;">
52.16211
</td>
<td style="text-align:left;">
positive
</td>
<td style="text-align:left;">
MGI_C57BL6J_5531250
</td>
<td style="text-align:left;">
Gm27868
</td>
<td style="text-align:left;">
ENSEMBL:ENSMUSG00000098536
</td>
<td style="text-align:left;">
MGI:5531250
</td>
<td style="text-align:left;">
unclassified non-coding RNA gene
</td>
<td style="text-align:left;">
predicted gene, 27868
</td>
</tr>
<tr>
<td style="text-align:left;">
6
</td>
<td style="text-align:left;">
gene
</td>
<td style="text-align:right;">
52.16213
</td>
<td style="text-align:right;">
52.16227
</td>
<td style="text-align:left;">
positive
</td>
<td style="text-align:left;">
MGI_C57BL6J_5530859
</td>
<td style="text-align:left;">
Gm27477
</td>
<td style="text-align:left;">
ENSEMBL:ENSMUSG00000098749
</td>
<td style="text-align:left;">
MGI:5530859
</td>
<td style="text-align:left;">
unclassified non-coding RNA gene
</td>
<td style="text-align:left;">
predicted gene, 27477
</td>
</tr>
<tr>
<td style="text-align:left;">
6
</td>
<td style="text-align:left;">
gene
</td>
<td style="text-align:right;">
52.16242
</td>
<td style="text-align:right;">
52.16483
</td>
<td style="text-align:left;">
negative
</td>
<td style="text-align:left;">
MGI_C57BL6J_96174
</td>
<td style="text-align:left;">
Hoxa2
</td>
<td style="text-align:left;">
NCBI_Gene:15399,ENSEMBL:ENSMUSG00000014704
</td>
<td style="text-align:left;">
MGI:96174
</td>
<td style="text-align:left;">
protein coding gene
</td>
<td style="text-align:left;">
homeobox A2
</td>
</tr>
<tr>
<td style="text-align:left;">
6
</td>
<td style="text-align:left;">
gene
</td>
<td style="text-align:right;">
52.16336
</td>
<td style="text-align:right;">
52.21334
</td>
<td style="text-align:left;">
negative
</td>
<td style="text-align:left;">
MGI_C57BL6J_5579014
</td>
<td style="text-align:left;">
Gm28308
</td>
<td style="text-align:left;">
ENSEMBL:ENSMUSG00000101298
</td>
<td style="text-align:left;">
MGI:5579014
</td>
<td style="text-align:left;">
protein coding gene
</td>
<td style="text-align:left;">
predicted gene 28308
</td>
</tr>
<tr>
<td style="text-align:left;">
6
</td>
<td style="text-align:left;">
gene
</td>
<td style="text-align:right;">
52.16567
</td>
<td style="text-align:right;">
52.16956
</td>
<td style="text-align:left;">
positive
</td>
<td style="text-align:left;">
MGI_C57BL6J_1913890
</td>
<td style="text-align:left;">
Hoxaas2
</td>
<td style="text-align:left;">
NCBI_Gene:105242657,ENSEMBL:ENSMUSG00000056445
</td>
<td style="text-align:left;">
MGI:1913890
</td>
<td style="text-align:left;">
lncRNA gene
</td>
<td style="text-align:left;">
Hoxa cluster antisense RNA 2
</td>
</tr>
<tr>
<td style="text-align:left;">
6
</td>
<td style="text-align:left;">
gene
</td>
<td style="text-align:right;">
52.16611
</td>
<td style="text-align:right;">
52.16679
</td>
<td style="text-align:left;">
negative
</td>
<td style="text-align:left;">
MGI_C57BL6J_5826597
</td>
<td style="text-align:left;">
Gm46960
</td>
<td style="text-align:left;">
NCBI_Gene:108169142
</td>
<td style="text-align:left;">
MGI:5826597
</td>
<td style="text-align:left;">
lncRNA gene
</td>
<td style="text-align:left;">
predicted gene, 46960
</td>
</tr>
<tr>
<td style="text-align:left;">
6
</td>
<td style="text-align:left;">
gene
</td>
<td style="text-align:right;">
52.16906
</td>
<td style="text-align:right;">
52.21334
</td>
<td style="text-align:left;">
negative
</td>
<td style="text-align:left;">
MGI_C57BL6J_96175
</td>
<td style="text-align:left;">
Hoxa3
</td>
<td style="text-align:left;">
NCBI_Gene:15400,ENSEMBL:ENSMUSG00000079560
</td>
<td style="text-align:left;">
MGI:96175
</td>
<td style="text-align:left;">
protein coding gene
</td>
<td style="text-align:left;">
homeobox A3
</td>
</tr>
<tr>
<td style="text-align:left;">
6
</td>
<td style="text-align:left;">
gene
</td>
<td style="text-align:right;">
52.17285
</td>
<td style="text-align:right;">
52.17406
</td>
<td style="text-align:left;">
positive
</td>
<td style="text-align:left;">
MGI_C57BL6J_3705180
</td>
<td style="text-align:left;">
Gm15050
</td>
<td style="text-align:left;">
NCBI_Gene:100504154,ENSEMBL:ENSMUSG00000087626
</td>
<td style="text-align:left;">
MGI:3705180
</td>
<td style="text-align:left;">
lncRNA gene
</td>
<td style="text-align:left;">
predicted gene 15050
</td>
</tr>
<tr>
<td style="text-align:left;">
6
</td>
<td style="text-align:left;">
gene
</td>
<td style="text-align:right;">
52.17522
</td>
<td style="text-align:right;">
52.21508
</td>
<td style="text-align:left;">
positive
</td>
<td style="text-align:left;">
MGI_C57BL6J_1919878
</td>
<td style="text-align:left;">
Hoxaas3
</td>
<td style="text-align:left;">
NCBI_Gene:72628,ENSEMBL:ENSMUSG00000085696
</td>
<td style="text-align:left;">
MGI:1919878
</td>
<td style="text-align:left;">
antisense lncRNA gene
</td>
<td style="text-align:left;">
Hoxa cluster antisense RNA 3
</td>
</tr>
<tr>
<td style="text-align:left;">
6
</td>
<td style="text-align:left;">
gene
</td>
<td style="text-align:right;">
52.17694
</td>
<td style="text-align:right;">
52.18085
</td>
<td style="text-align:left;">
positive
</td>
<td style="text-align:left;">
MGI_C57BL6J_1924830
</td>
<td style="text-align:left;">
5730596B20Rik
</td>
<td style="text-align:left;">
NCBI_Gene:77580,ENSEMBL:ENSMUSG00000056468
</td>
<td style="text-align:left;">
MGI:1924830
</td>
<td style="text-align:left;">
lncRNA gene
</td>
<td style="text-align:left;">
RIKEN cDNA 5730596B20 gene
</td>
</tr>
<tr>
<td style="text-align:left;">
6
</td>
<td style="text-align:left;">
gene
</td>
<td style="text-align:right;">
52.18311
</td>
<td style="text-align:right;">
52.18993
</td>
<td style="text-align:left;">
positive
</td>
<td style="text-align:left;">
MGI_C57BL6J_5580136
</td>
<td style="text-align:left;">
Gm29430
</td>
<td style="text-align:left;">
ENSEMBL:ENSMUSG00000101451
</td>
<td style="text-align:left;">
MGI:5580136
</td>
<td style="text-align:left;">
lncRNA gene
</td>
<td style="text-align:left;">
predicted gene 29430
</td>
</tr>
<tr>
<td style="text-align:left;">
6
</td>
<td style="text-align:left;">
gene
</td>
<td style="text-align:right;">
52.18967
</td>
<td style="text-align:right;">
52.19175
</td>
<td style="text-align:left;">
negative
</td>
<td style="text-align:left;">
MGI_C57BL6J_96176
</td>
<td style="text-align:left;">
Hoxa4
</td>
<td style="text-align:left;">
NCBI_Gene:15401,ENSEMBL:ENSMUSG00000000942
</td>
<td style="text-align:left;">
MGI:96176
</td>
<td style="text-align:left;">
protein coding gene
</td>
<td style="text-align:left;">
homeobox A4
</td>
</tr>
<tr>
<td style="text-align:left;">
6
</td>
<td style="text-align:left;">
gene
</td>
<td style="text-align:right;">
52.20175
</td>
<td style="text-align:right;">
52.20459
</td>
<td style="text-align:left;">
negative
</td>
<td style="text-align:left;">
MGI_C57BL6J_96177
</td>
<td style="text-align:left;">
Hoxa5
</td>
<td style="text-align:left;">
NCBI_Gene:15402,ENSEMBL:ENSMUSG00000038253
</td>
<td style="text-align:left;">
MGI:96177
</td>
<td style="text-align:left;">
protein coding gene
</td>
<td style="text-align:left;">
homeobox A5
</td>
</tr>
<tr>
<td style="text-align:left;">
6
</td>
<td style="text-align:left;">
gene
</td>
<td style="text-align:right;">
52.20629
</td>
<td style="text-align:right;">
52.20872
</td>
<td style="text-align:left;">
negative
</td>
<td style="text-align:left;">
MGI_C57BL6J_96178
</td>
<td style="text-align:left;">
Hoxa6
</td>
<td style="text-align:left;">
NCBI_Gene:15403,ENSEMBL:ENSMUSG00000043219
</td>
<td style="text-align:left;">
MGI:96178
</td>
<td style="text-align:left;">
protein coding gene
</td>
<td style="text-align:left;">
homeobox A6
</td>
</tr>
<tr>
<td style="text-align:left;">
6
</td>
<td style="text-align:left;">
gene
</td>
<td style="text-align:right;">
52.21449
</td>
<td style="text-align:right;">
52.21529
</td>
<td style="text-align:left;">
negative
</td>
<td style="text-align:left;">
MGI_C57BL6J_5433747
</td>
<td style="text-align:left;">
Mira
</td>
<td style="text-align:left;">
NCBI_Gene:100736249
</td>
<td style="text-align:left;">
MGI:5433747
</td>
<td style="text-align:left;">
lncRNA gene
</td>
<td style="text-align:left;">
mistral long non-coding RNA
</td>
</tr>
<tr>
<td style="text-align:left;">
6
</td>
<td style="text-align:left;">
gene
</td>
<td style="text-align:right;">
52.21449
</td>
<td style="text-align:right;">
52.22185
</td>
<td style="text-align:left;">
negative
</td>
<td style="text-align:left;">
MGI_C57BL6J_96179
</td>
<td style="text-align:left;">
Hoxa7
</td>
<td style="text-align:left;">
NCBI_Gene:15404,ENSEMBL:ENSMUSG00000038236
</td>
<td style="text-align:left;">
MGI:96179
</td>
<td style="text-align:left;">
protein coding gene
</td>
<td style="text-align:left;">
homeobox A7
</td>
</tr>
<tr>
<td style="text-align:left;">
6
</td>
<td style="text-align:left;">
gene
</td>
<td style="text-align:right;">
52.22310
</td>
<td style="text-align:right;">
52.23109
</td>
<td style="text-align:left;">
negative
</td>
<td style="text-align:left;">
MGI_C57BL6J_96180
</td>
<td style="text-align:left;">
Hoxa9
</td>
<td style="text-align:left;">
NCBI_Gene:15405,ENSEMBL:ENSMUSG00000038227
</td>
<td style="text-align:left;">
MGI:96180
</td>
<td style="text-align:left;">
protein coding gene
</td>
<td style="text-align:left;">
homeobox A9
</td>
</tr>
<tr>
<td style="text-align:left;">
6
</td>
<td style="text-align:left;">
gene
</td>
<td style="text-align:right;">
52.23008
</td>
<td style="text-align:right;">
52.23016
</td>
<td style="text-align:left;">
negative
</td>
<td style="text-align:left;">
MGI_C57BL6J_3618741
</td>
<td style="text-align:left;">
Mir196b
</td>
<td style="text-align:left;">
miRBase:MI0001151,NCBI_Gene:723820,ENSEMBL:ENSMUSG00000065443
</td>
<td style="text-align:left;">
MGI:3618741
</td>
<td style="text-align:left;">
miRNA gene
</td>
<td style="text-align:left;">
microRNA 196b
</td>
</tr>
<tr>
<td style="text-align:left;">
6
</td>
<td style="text-align:left;">
gene
</td>
<td style="text-align:right;">
52.23120
</td>
<td style="text-align:right;">
52.24085
</td>
<td style="text-align:left;">
negative
</td>
<td style="text-align:left;">
MGI_C57BL6J_96171
</td>
<td style="text-align:left;">
Hoxa10
</td>
<td style="text-align:left;">
NCBI_Gene:15395,ENSEMBL:ENSMUSG00000000938
</td>
<td style="text-align:left;">
MGI:96171
</td>
<td style="text-align:left;">
protein coding gene
</td>
<td style="text-align:left;">
homeobox A10
</td>
</tr>
<tr>
<td style="text-align:left;">
6
</td>
<td style="text-align:left;">
gene
</td>
<td style="text-align:right;">
52.24211
</td>
<td style="text-align:right;">
52.24581
</td>
<td style="text-align:left;">
negative
</td>
<td style="text-align:left;">
MGI_C57BL6J_96172
</td>
<td style="text-align:left;">
Hoxa11
</td>
<td style="text-align:left;">
NCBI_Gene:15396,ENSEMBL:ENSMUSG00000038210
</td>
<td style="text-align:left;">
MGI:96172
</td>
<td style="text-align:left;">
protein coding gene
</td>
<td style="text-align:left;">
homeobox A11
</td>
</tr>
<tr>
<td style="text-align:left;">
6
</td>
<td style="text-align:left;">
gene
</td>
<td style="text-align:right;">
52.24439
</td>
<td style="text-align:right;">
52.24980
</td>
<td style="text-align:left;">
positive
</td>
<td style="text-align:left;">
MGI_C57BL6J_107208
</td>
<td style="text-align:left;">
Hoxa11os
</td>
<td style="text-align:left;">
NCBI_Gene:15397,ENSEMBL:ENSMUSG00000086427
</td>
<td style="text-align:left;">
MGI:107208
</td>
<td style="text-align:left;">
lncRNA gene
</td>
<td style="text-align:left;">
homeobox A11, opposite strand
</td>
</tr>
<tr>
<td style="text-align:left;">
6
</td>
<td style="text-align:left;">
gene
</td>
<td style="text-align:right;">
52.24601
</td>
<td style="text-align:right;">
52.24610
</td>
<td style="text-align:left;">
positive
</td>
<td style="text-align:left;">
MGI_C57BL6J_5531414
</td>
<td style="text-align:left;">
Gm28032
</td>
<td style="text-align:left;">
ENSEMBL:ENSMUSG00000098717
</td>
<td style="text-align:left;">
MGI:5531414
</td>
<td style="text-align:left;">
unclassified non-coding RNA gene
</td>
<td style="text-align:left;">
predicted gene, 28032
</td>
</tr>
<tr>
<td style="text-align:left;">
6
</td>
<td style="text-align:left;">
gene
</td>
<td style="text-align:right;">
52.24640
</td>
<td style="text-align:right;">
52.24659
</td>
<td style="text-align:left;">
positive
</td>
<td style="text-align:left;">
MGI_C57BL6J_5530874
</td>
<td style="text-align:left;">
Gm27492
</td>
<td style="text-align:left;">
ENSEMBL:ENSMUSG00000098435
</td>
<td style="text-align:left;">
MGI:5530874
</td>
<td style="text-align:left;">
unclassified non-coding RNA gene
</td>
<td style="text-align:left;">
predicted gene, 27492
</td>
</tr>
<tr>
<td style="text-align:left;">
6
</td>
<td style="text-align:left;">
gene
</td>
<td style="text-align:right;">
52.24672
</td>
<td style="text-align:right;">
52.24681
</td>
<td style="text-align:left;">
positive
</td>
<td style="text-align:left;">
MGI_C57BL6J_5531077
</td>
<td style="text-align:left;">
Gm27695
</td>
<td style="text-align:left;">
ENSEMBL:ENSMUSG00000098969
</td>
<td style="text-align:left;">
MGI:5531077
</td>
<td style="text-align:left;">
unclassified non-coding RNA gene
</td>
<td style="text-align:left;">
predicted gene, 27695
</td>
</tr>
<tr>
<td style="text-align:left;">
6
</td>
<td style="text-align:left;">
gene
</td>
<td style="text-align:right;">
52.24742
</td>
<td style="text-align:right;">
52.24765
</td>
<td style="text-align:left;">
positive
</td>
<td style="text-align:left;">
MGI_C57BL6J_5531199
</td>
<td style="text-align:left;">
Gm27817
</td>
<td style="text-align:left;">
ENSEMBL:ENSMUSG00000098683
</td>
<td style="text-align:left;">
MGI:5531199
</td>
<td style="text-align:left;">
unclassified non-coding RNA gene
</td>
<td style="text-align:left;">
predicted gene, 27817
</td>
</tr>
<tr>
<td style="text-align:left;">
6
</td>
<td style="text-align:left;">
gene
</td>
<td style="text-align:right;">
52.24833
</td>
<td style="text-align:right;">
52.24853
</td>
<td style="text-align:left;">
positive
</td>
<td style="text-align:left;">
MGI_C57BL6J_5530836
</td>
<td style="text-align:left;">
Gm27454
</td>
<td style="text-align:left;">
ENSEMBL:ENSMUSG00000099270
</td>
<td style="text-align:left;">
MGI:5530836
</td>
<td style="text-align:left;">
unclassified non-coding RNA gene
</td>
<td style="text-align:left;">
predicted gene, 27454
</td>
</tr>
<tr>
<td style="text-align:left;">
6
</td>
<td style="text-align:left;">
gene
</td>
<td style="text-align:right;">
52.24930
</td>
<td style="text-align:right;">
52.24947
</td>
<td style="text-align:left;">
positive
</td>
<td style="text-align:left;">
MGI_C57BL6J_5530925
</td>
<td style="text-align:left;">
Gm27543
</td>
<td style="text-align:left;">
ENSEMBL:ENSMUSG00000099248
</td>
<td style="text-align:left;">
MGI:5530925
</td>
<td style="text-align:left;">
unclassified non-coding RNA gene
</td>
<td style="text-align:left;">
predicted gene, 27543
</td>
</tr>
<tr>
<td style="text-align:left;">
6
</td>
<td style="text-align:left;">
gene
</td>
<td style="text-align:right;">
52.25622
</td>
<td style="text-align:right;">
52.25746
</td>
<td style="text-align:left;">
negative
</td>
<td style="text-align:left;">
MGI_C57BL6J_1924645
</td>
<td style="text-align:left;">
9530018H14Rik
</td>
<td style="text-align:left;">
ENSEMBL:ENSMUSG00000102373
</td>
<td style="text-align:left;">
MGI:1924645
</td>
<td style="text-align:left;">
unclassified gene
</td>
<td style="text-align:left;">
RIKEN cDNA 9530018H14 gene
</td>
</tr>
<tr>
<td style="text-align:left;">
6
</td>
<td style="text-align:left;">
gene
</td>
<td style="text-align:right;">
52.25769
</td>
<td style="text-align:right;">
52.26088
</td>
<td style="text-align:left;">
negative
</td>
<td style="text-align:left;">
MGI_C57BL6J_96173
</td>
<td style="text-align:left;">
Hoxa13
</td>
<td style="text-align:left;">
NCBI_Gene:15398,ENSEMBL:ENSMUSG00000038203
</td>
<td style="text-align:left;">
MGI:96173
</td>
<td style="text-align:left;">
protein coding gene
</td>
<td style="text-align:left;">
homeobox A13
</td>
</tr>
<tr>
<td style="text-align:left;">
6
</td>
<td style="text-align:left;">
gene
</td>
<td style="text-align:right;">
52.26115
</td>
<td style="text-align:right;">
52.26121
</td>
<td style="text-align:left;">
positive
</td>
<td style="text-align:left;">
MGI_C57BL6J_5530662
</td>
<td style="text-align:left;">
Gm27280
</td>
<td style="text-align:left;">
ENSEMBL:ENSMUSG00000098484
</td>
<td style="text-align:left;">
MGI:5530662
</td>
<td style="text-align:left;">
unclassified non-coding RNA gene
</td>
<td style="text-align:left;">
predicted gene, 27280
</td>
</tr>
<tr>
<td style="text-align:left;">
6
</td>
<td style="text-align:left;">
gene
</td>
<td style="text-align:right;">
52.26259
</td>
<td style="text-align:right;">
52.26297
</td>
<td style="text-align:left;">
positive
</td>
<td style="text-align:left;">
MGI_C57BL6J_5531367
</td>
<td style="text-align:left;">
Gm27985
</td>
<td style="text-align:left;">
ENSEMBL:ENSMUSG00000099173
</td>
<td style="text-align:left;">
MGI:5531367
</td>
<td style="text-align:left;">
unclassified non-coding RNA gene
</td>
<td style="text-align:left;">
predicted gene, 27985
</td>
</tr>
<tr>
<td style="text-align:left;">
6
</td>
<td style="text-align:left;">
gene
</td>
<td style="text-align:right;">
52.26277
</td>
<td style="text-align:right;">
52.26760
</td>
<td style="text-align:left;">
positive
</td>
<td style="text-align:left;">
MGI_C57BL6J_3642509
</td>
<td style="text-align:left;">
Hottip
</td>
<td style="text-align:left;">
NCBI_Gene:791364,ENSEMBL:ENSMUSG00000055408
</td>
<td style="text-align:left;">
MGI:3642509
</td>
<td style="text-align:left;">
lncRNA gene
</td>
<td style="text-align:left;">
Hoxa distal transcript antisense RNA
</td>
</tr>
<tr>
<td style="text-align:left;">
6
</td>
<td style="text-align:left;">
gene
</td>
<td style="text-align:right;">
52.26306
</td>
<td style="text-align:right;">
52.26342
</td>
<td style="text-align:left;">
positive
</td>
<td style="text-align:left;">
MGI_C57BL6J_5530765
</td>
<td style="text-align:left;">
Gm27383
</td>
<td style="text-align:left;">
ENSEMBL:ENSMUSG00000099229
</td>
<td style="text-align:left;">
MGI:5530765
</td>
<td style="text-align:left;">
unclassified non-coding RNA gene
</td>
<td style="text-align:left;">
predicted gene, 27383
</td>
</tr>
<tr>
<td style="text-align:left;">
6
</td>
<td style="text-align:left;">
gene
</td>
<td style="text-align:right;">
52.26658
</td>
<td style="text-align:right;">
52.26675
</td>
<td style="text-align:left;">
positive
</td>
<td style="text-align:left;">
MGI_C57BL6J_5531248
</td>
<td style="text-align:left;">
Gm27866
</td>
<td style="text-align:left;">
ENSEMBL:ENSMUSG00000098538
</td>
<td style="text-align:left;">
MGI:5531248
</td>
<td style="text-align:left;">
unclassified non-coding RNA gene
</td>
<td style="text-align:left;">
predicted gene, 27866
</td>
</tr>
<tr>
<td style="text-align:left;">
6
</td>
<td style="text-align:left;">
gene
</td>
<td style="text-align:right;">
52.29712
</td>
<td style="text-align:right;">
52.29785
</td>
<td style="text-align:left;">
positive
</td>
<td style="text-align:left;">
MGI_C57BL6J_5690686
</td>
<td style="text-align:left;">
Gm44294
</td>
<td style="text-align:left;">
ENSEMBL:ENSMUSG00000107463
</td>
<td style="text-align:left;">
MGI:5690686
</td>
<td style="text-align:left;">
unclassified gene
</td>
<td style="text-align:left;">
predicted gene, 44294
</td>
</tr>
<tr>
<td style="text-align:left;">
6
</td>
<td style="text-align:left;">
gene
</td>
<td style="text-align:right;">
52.30839
</td>
<td style="text-align:right;">
52.31483
</td>
<td style="text-align:left;">
negative
</td>
<td style="text-align:left;">
MGI_C57BL6J_1917843
</td>
<td style="text-align:left;">
Evx1os
</td>
<td style="text-align:left;">
NCBI_Gene:70593,ENSEMBL:ENSMUSG00000086126
</td>
<td style="text-align:left;">
MGI:1917843
</td>
<td style="text-align:left;">
antisense lncRNA gene
</td>
<td style="text-align:left;">
even skipped homeotic gene 1, opposite strand
</td>
</tr>
<tr>
<td style="text-align:left;">
6
</td>
<td style="text-align:left;">
gene
</td>
<td style="text-align:right;">
52.31350
</td>
<td style="text-align:right;">
52.31839
</td>
<td style="text-align:left;">
positive
</td>
<td style="text-align:left;">
MGI_C57BL6J_95461
</td>
<td style="text-align:left;">
Evx1
</td>
<td style="text-align:left;">
NCBI_Gene:14028,ENSEMBL:ENSMUSG00000005503
</td>
<td style="text-align:left;">
MGI:95461
</td>
<td style="text-align:left;">
protein coding gene
</td>
<td style="text-align:left;">
even-skipped homeobox 1
</td>
</tr>
<tr>
<td style="text-align:left;">
6
</td>
<td style="text-align:left;">
pseudogene
</td>
<td style="text-align:right;">
52.34669
</td>
<td style="text-align:right;">
52.34700
</td>
<td style="text-align:left;">
negative
</td>
<td style="text-align:left;">
MGI_C57BL6J_3646863
</td>
<td style="text-align:left;">
Gm8129
</td>
<td style="text-align:left;">
NCBI_Gene:666487,ENSEMBL:ENSMUSG00000059159
</td>
<td style="text-align:left;">
MGI:3646863
</td>
<td style="text-align:left;">
pseudogene
</td>
<td style="text-align:left;">
predicted pseudogene 8129
</td>
</tr>
<tr>
<td style="text-align:left;">
6
</td>
<td style="text-align:left;">
pseudogene
</td>
<td style="text-align:right;">
52.35754
</td>
<td style="text-align:right;">
52.35782
</td>
<td style="text-align:left;">
negative
</td>
<td style="text-align:left;">
MGI_C57BL6J_3705761
</td>
<td style="text-align:left;">
Gm15054
</td>
<td style="text-align:left;">
ENSEMBL:ENSMUSG00000084282
</td>
<td style="text-align:left;">
MGI:3705761
</td>
<td style="text-align:left;">
pseudogene
</td>
<td style="text-align:left;">
predicted gene 15054
</td>
</tr>
<tr>
<td style="text-align:left;">
6
</td>
<td style="text-align:left;">
gene
</td>
<td style="text-align:right;">
52.49245
</td>
<td style="text-align:right;">
52.50009
</td>
<td style="text-align:left;">
positive
</td>
<td style="text-align:left;">
MGI_C57BL6J_1921554
</td>
<td style="text-align:left;">
1700094M24Rik
</td>
<td style="text-align:left;">
NCBI_Gene:74304,ENSEMBL:ENSMUSG00000107895
</td>
<td style="text-align:left;">
MGI:1921554
</td>
<td style="text-align:left;">
lncRNA gene
</td>
<td style="text-align:left;">
RIKEN cDNA 1700094M24 gene
</td>
</tr>
<tr>
<td style="text-align:left;">
6
</td>
<td style="text-align:left;">
gene
</td>
<td style="text-align:right;">
52.50989
</td>
<td style="text-align:right;">
52.51404
</td>
<td style="text-align:left;">
positive
</td>
<td style="text-align:left;">
MGI_C57BL6J_5690420
</td>
<td style="text-align:left;">
Gm44028
</td>
<td style="text-align:left;">
ENSEMBL:ENSMUSG00000108062
</td>
<td style="text-align:left;">
MGI:5690420
</td>
<td style="text-align:left;">
lncRNA gene
</td>
<td style="text-align:left;">
predicted gene, 44028
</td>
</tr>
<tr>
<td style="text-align:left;">
6
</td>
<td style="text-align:left;">
gene
</td>
<td style="text-align:right;">
52.54623
</td>
<td style="text-align:right;">
52.64039
</td>
<td style="text-align:left;">
negative
</td>
<td style="text-align:left;">
MGI_C57BL6J_1889802
</td>
<td style="text-align:left;">
Hibadh
</td>
<td style="text-align:left;">
NCBI_Gene:58875,ENSEMBL:ENSMUSG00000029776
</td>
<td style="text-align:left;">
MGI:1889802
</td>
<td style="text-align:left;">
protein coding gene
</td>
<td style="text-align:left;">
3-hydroxyisobutyrate dehydrogenase
</td>
</tr>
<tr>
<td style="text-align:left;">
6
</td>
<td style="text-align:left;">
gene
</td>
<td style="text-align:right;">
52.56055
</td>
<td style="text-align:right;">
52.56323
</td>
<td style="text-align:left;">
positive
</td>
<td style="text-align:left;">
MGI_C57BL6J_5690837
</td>
<td style="text-align:left;">
Gm44445
</td>
<td style="text-align:left;">
ENSEMBL:ENSMUSG00000107647
</td>
<td style="text-align:left;">
MGI:5690837
</td>
<td style="text-align:left;">
unclassified gene
</td>
<td style="text-align:left;">
predicted gene, 44445
</td>
</tr>
<tr>
<td style="text-align:left;">
6
</td>
<td style="text-align:left;">
gene
</td>
<td style="text-align:right;">
52.56596
</td>
<td style="text-align:right;">
52.57981
</td>
<td style="text-align:left;">
negative
</td>
<td style="text-align:left;">
MGI_C57BL6J_5592206
</td>
<td style="text-align:left;">
Gm33047
</td>
<td style="text-align:left;">
NCBI_Gene:102635799
</td>
<td style="text-align:left;">
MGI:5592206
</td>
<td style="text-align:left;">
lncRNA gene
</td>
<td style="text-align:left;">
predicted gene, 33047
</td>
</tr>
<tr>
<td style="text-align:left;">
6
</td>
<td style="text-align:left;">
gene
</td>
<td style="text-align:right;">
52.59950
</td>
<td style="text-align:right;">
52.60325
</td>
<td style="text-align:left;">
negative
</td>
<td style="text-align:left;">
MGI_C57BL6J_5690826
</td>
<td style="text-align:left;">
Gm44434
</td>
<td style="text-align:left;">
ENSEMBL:ENSMUSG00000107813
</td>
<td style="text-align:left;">
MGI:5690826
</td>
<td style="text-align:left;">
unclassified gene
</td>
<td style="text-align:left;">
predicted gene, 44434
</td>
</tr>
<tr>
<td style="text-align:left;">
6
</td>
<td style="text-align:left;">
gene
</td>
<td style="text-align:right;">
52.63923
</td>
<td style="text-align:right;">
52.63936
</td>
<td style="text-align:left;">
negative
</td>
<td style="text-align:left;">
MGI_C57BL6J_5454132
</td>
<td style="text-align:left;">
Gm24355
</td>
<td style="text-align:left;">
ENSEMBL:ENSMUSG00000064792
</td>
<td style="text-align:left;">
MGI:5454132
</td>
<td style="text-align:left;">
snoRNA gene
</td>
<td style="text-align:left;">
predicted gene, 24355
</td>
</tr>
<tr>
<td style="text-align:left;">
6
</td>
<td style="text-align:left;">
gene
</td>
<td style="text-align:right;">
52.71334
</td>
<td style="text-align:right;">
52.71395
</td>
<td style="text-align:left;">
negative
</td>
<td style="text-align:left;">
MGI_C57BL6J_5690358
</td>
<td style="text-align:left;">
Gm43966
</td>
<td style="text-align:left;">
ENSEMBL:ENSMUSG00000108170
</td>
<td style="text-align:left;">
MGI:5690358
</td>
<td style="text-align:left;">
unclassified gene
</td>
<td style="text-align:left;">
predicted gene, 43966
</td>
</tr>
<tr>
<td style="text-align:left;">
6
</td>
<td style="text-align:left;">
gene
</td>
<td style="text-align:right;">
52.71357
</td>
<td style="text-align:right;">
52.76678
</td>
<td style="text-align:left;">
positive
</td>
<td style="text-align:left;">
MGI_C57BL6J_1289308
</td>
<td style="text-align:left;">
Tax1bp1
</td>
<td style="text-align:left;">
NCBI_Gene:52440,ENSEMBL:ENSMUSG00000004535
</td>
<td style="text-align:left;">
MGI:1289308
</td>
<td style="text-align:left;">
protein coding gene
</td>
<td style="text-align:left;">
Tax1 (human T cell leukemia virus type I) binding protein 1
</td>
</tr>
<tr>
<td style="text-align:left;">
6
</td>
<td style="text-align:left;">
pseudogene
</td>
<td style="text-align:right;">
52.72263
</td>
<td style="text-align:right;">
52.72293
</td>
<td style="text-align:left;">
negative
</td>
<td style="text-align:left;">
MGI_C57BL6J_5690359
</td>
<td style="text-align:left;">
Gm43967
</td>
<td style="text-align:left;">
ENSEMBL:ENSMUSG00000108036
</td>
<td style="text-align:left;">
MGI:5690359
</td>
<td style="text-align:left;">
pseudogene
</td>
<td style="text-align:left;">
predicted gene, 43967
</td>
</tr>
<tr>
<td style="text-align:left;">
6
</td>
<td style="text-align:left;">
gene
</td>
<td style="text-align:right;">
52.76806
</td>
<td style="text-align:right;">
53.06863
</td>
<td style="text-align:left;">
negative
</td>
<td style="text-align:left;">
MGI_C57BL6J_2141450
</td>
<td style="text-align:left;">
Jazf1
</td>
<td style="text-align:left;">
NCBI_Gene:231986,ENSEMBL:ENSMUSG00000063568
</td>
<td style="text-align:left;">
MGI:2141450
</td>
<td style="text-align:left;">
protein coding gene
</td>
<td style="text-align:left;">
JAZF zinc finger 1
</td>
</tr>
<tr>
<td style="text-align:left;">
6
</td>
<td style="text-align:left;">
pseudogene
</td>
<td style="text-align:right;">
52.84311
</td>
<td style="text-align:right;">
52.84370
</td>
<td style="text-align:left;">
negative
</td>
<td style="text-align:left;">
MGI_C57BL6J_3783020
</td>
<td style="text-align:left;">
Gm15572
</td>
<td style="text-align:left;">
ENSEMBL:ENSMUSG00000081872
</td>
<td style="text-align:left;">
MGI:3783020
</td>
<td style="text-align:left;">
pseudogene
</td>
<td style="text-align:left;">
predicted gene 15572
</td>
</tr>
<tr>
<td style="text-align:left;">
6
</td>
<td style="text-align:left;">
gene
</td>
<td style="text-align:right;">
52.96516
</td>
<td style="text-align:right;">
52.97656
</td>
<td style="text-align:left;">
negative
</td>
<td style="text-align:left;">
MGI_C57BL6J_5592378
</td>
<td style="text-align:left;">
Gm33219
</td>
<td style="text-align:left;">
NCBI_Gene:102636035
</td>
<td style="text-align:left;">
MGI:5592378
</td>
<td style="text-align:left;">
lncRNA gene
</td>
<td style="text-align:left;">
predicted gene, 33219
</td>
</tr>
<tr>
<td style="text-align:left;">
6
</td>
<td style="text-align:left;">
gene
</td>
<td style="text-align:right;">
53.01039
</td>
<td style="text-align:right;">
53.01053
</td>
<td style="text-align:left;">
negative
</td>
<td style="text-align:left;">
MGI_C57BL6J_5455992
</td>
<td style="text-align:left;">
Gm26215
</td>
<td style="text-align:left;">
ENSEMBL:ENSMUSG00000089209
</td>
<td style="text-align:left;">
MGI:5455992
</td>
<td style="text-align:left;">
snoRNA gene
</td>
<td style="text-align:left;">
predicted gene, 26215
</td>
</tr>
<tr>
<td style="text-align:left;">
6
</td>
<td style="text-align:left;">
pseudogene
</td>
<td style="text-align:right;">
53.04562
</td>
<td style="text-align:right;">
53.04705
</td>
<td style="text-align:left;">
positive
</td>
<td style="text-align:left;">
MGI_C57BL6J_3783021
</td>
<td style="text-align:left;">
Gm15573
</td>
<td style="text-align:left;">
NCBI_Gene:108169143,ENSEMBL:ENSMUSG00000080759
</td>
<td style="text-align:left;">
MGI:3783021
</td>
<td style="text-align:left;">
pseudogene
</td>
<td style="text-align:left;">
predicted gene 15573
</td>
</tr>
<tr>
<td style="text-align:left;">
6
</td>
<td style="text-align:left;">
gene
</td>
<td style="text-align:right;">
53.06009
</td>
<td style="text-align:right;">
53.06595
</td>
<td style="text-align:left;">
negative
</td>
<td style="text-align:left;">
MGI_C57BL6J_5592256
</td>
<td style="text-align:left;">
Gm33097
</td>
<td style="text-align:left;">
NCBI_Gene:102635867
</td>
<td style="text-align:left;">
MGI:5592256
</td>
<td style="text-align:left;">
lncRNA gene
</td>
<td style="text-align:left;">
predicted gene, 33097
</td>
</tr>
<tr>
<td style="text-align:left;">
6
</td>
<td style="text-align:left;">
pseudogene
</td>
<td style="text-align:right;">
53.17055
</td>
<td style="text-align:right;">
53.17174
</td>
<td style="text-align:left;">
negative
</td>
<td style="text-align:left;">
MGI_C57BL6J_3644270
</td>
<td style="text-align:left;">
Gm8143
</td>
<td style="text-align:left;">
NCBI_Gene:666516,ENSEMBL:ENSMUSG00000108028
</td>
<td style="text-align:left;">
MGI:3644270
</td>
<td style="text-align:left;">
pseudogene
</td>
<td style="text-align:left;">
predicted gene 8143
</td>
</tr>
<tr>
<td style="text-align:left;">
6
</td>
<td style="text-align:left;">
gene
</td>
<td style="text-align:right;">
53.22116
</td>
<td style="text-align:right;">
53.22660
</td>
<td style="text-align:left;">
positive
</td>
<td style="text-align:left;">
MGI_C57BL6J_3779443
</td>
<td style="text-align:left;">
Gm4872
</td>
<td style="text-align:left;">
NCBI_Gene:231989,ENSEMBL:ENSMUSG00000107935
</td>
<td style="text-align:left;">
MGI:3779443
</td>
<td style="text-align:left;">
lncRNA gene
</td>
<td style="text-align:left;">
predicted gene 4872
</td>
</tr>
<tr>
<td style="text-align:left;">
6
</td>
<td style="text-align:left;">
gene
</td>
<td style="text-align:right;">
53.26248
</td>
<td style="text-align:right;">
53.26823
</td>
<td style="text-align:left;">
negative
</td>
<td style="text-align:left;">
MGI_C57BL6J_5592507
</td>
<td style="text-align:left;">
Gm33348
</td>
<td style="text-align:left;">
NCBI_Gene:102636219
</td>
<td style="text-align:left;">
MGI:5592507
</td>
<td style="text-align:left;">
lncRNA gene
</td>
<td style="text-align:left;">
predicted gene, 33348
</td>
</tr>
<tr>
<td style="text-align:left;">
6
</td>
<td style="text-align:left;">
gene
</td>
<td style="text-align:right;">
53.28722
</td>
<td style="text-align:right;">
53.70038
</td>
<td style="text-align:left;">
positive
</td>
<td style="text-align:left;">
MGI_C57BL6J_2443973
</td>
<td style="text-align:left;">
Creb5
</td>
<td style="text-align:left;">
NCBI_Gene:231991,ENSEMBL:ENSMUSG00000053007
</td>
<td style="text-align:left;">
MGI:2443973
</td>
<td style="text-align:left;">
protein coding gene
</td>
<td style="text-align:left;">
cAMP responsive element binding protein 5
</td>
</tr>
<tr>
<td style="text-align:left;">
6
</td>
<td style="text-align:left;">
gene
</td>
<td style="text-align:right;">
53.43397
</td>
<td style="text-align:right;">
53.43414
</td>
<td style="text-align:left;">
positive
</td>
<td style="text-align:left;">
MGI_C57BL6J_5455595
</td>
<td style="text-align:left;">
Gm25818
</td>
<td style="text-align:left;">
ENSEMBL:ENSMUSG00000096147
</td>
<td style="text-align:left;">
MGI:5455595
</td>
<td style="text-align:left;">
snRNA gene
</td>
<td style="text-align:left;">
predicted gene, 25818
</td>
</tr>
<tr>
<td style="text-align:left;">
6
</td>
<td style="text-align:left;">
gene
</td>
<td style="text-align:right;">
53.46951
</td>
<td style="text-align:right;">
53.47175
</td>
<td style="text-align:left;">
negative
</td>
<td style="text-align:left;">
MGI_C57BL6J_5621700
</td>
<td style="text-align:left;">
Gm38815
</td>
<td style="text-align:left;">
NCBI_Gene:105242662
</td>
<td style="text-align:left;">
MGI:5621700
</td>
<td style="text-align:left;">
lncRNA gene
</td>
<td style="text-align:left;">
predicted gene, 38815
</td>
</tr>
<tr>
<td style="text-align:left;">
6
</td>
<td style="text-align:left;">
gene
</td>
<td style="text-align:right;">
53.49354
</td>
<td style="text-align:right;">
53.49828
</td>
<td style="text-align:left;">
positive
</td>
<td style="text-align:left;">
MGI_C57BL6J_5621699
</td>
<td style="text-align:left;">
Gm38814
</td>
<td style="text-align:left;">
NCBI_Gene:105242661
</td>
<td style="text-align:left;">
MGI:5621699
</td>
<td style="text-align:left;">
lncRNA gene
</td>
<td style="text-align:left;">
predicted gene, 38814
</td>
</tr>
<tr>
<td style="text-align:left;">
6
</td>
<td style="text-align:left;">
gene
</td>
<td style="text-align:right;">
53.58135
</td>
<td style="text-align:right;">
53.59288
</td>
<td style="text-align:left;">
negative
</td>
<td style="text-align:left;">
MGI_C57BL6J_5592551
</td>
<td style="text-align:left;">
Gm33392
</td>
<td style="text-align:left;">
NCBI_Gene:102636280
</td>
<td style="text-align:left;">
MGI:5592551
</td>
<td style="text-align:left;">
lncRNA gene
</td>
<td style="text-align:left;">
predicted gene, 33392
</td>
</tr>
<tr>
<td style="text-align:left;">
6
</td>
<td style="text-align:left;">
gene
</td>
<td style="text-align:right;">
53.66777
</td>
<td style="text-align:right;">
53.66998
</td>
<td style="text-align:left;">
positive
</td>
<td style="text-align:left;">
MGI_C57BL6J_5690472
</td>
<td style="text-align:left;">
Gm44080
</td>
<td style="text-align:left;">
ENSEMBL:ENSMUSG00000108004
</td>
<td style="text-align:left;">
MGI:5690472
</td>
<td style="text-align:left;">
unclassified gene
</td>
<td style="text-align:left;">
predicted gene, 44080
</td>
</tr>
<tr>
<td style="text-align:left;">
6
</td>
<td style="text-align:left;">
pseudogene
</td>
<td style="text-align:right;">
53.77736
</td>
<td style="text-align:right;">
53.77818
</td>
<td style="text-align:left;">
positive
</td>
<td style="text-align:left;">
MGI_C57BL6J_3644732
</td>
<td style="text-align:left;">
Gm5306
</td>
<td style="text-align:left;">
NCBI_Gene:384386,ENSEMBL:ENSMUSG00000107984
</td>
<td style="text-align:left;">
MGI:3644732
</td>
<td style="text-align:left;">
pseudogene
</td>
<td style="text-align:left;">
predicted gene 5306
</td>
</tr>
<tr>
<td style="text-align:left;">
6
</td>
<td style="text-align:left;">
gene
</td>
<td style="text-align:right;">
53.81547
</td>
<td style="text-align:right;">
53.82083
</td>
<td style="text-align:left;">
negative
</td>
<td style="text-align:left;">
MGI_C57BL6J_1914123
</td>
<td style="text-align:left;">
Tril
</td>
<td style="text-align:left;">
NCBI_Gene:66873,ENSEMBL:ENSMUSG00000043496
</td>
<td style="text-align:left;">
MGI:1914123
</td>
<td style="text-align:left;">
protein coding gene
</td>
<td style="text-align:left;">
TLR4 interactor with leucine-rich repeats
</td>
</tr>
<tr>
<td style="text-align:left;">
6
</td>
<td style="text-align:left;">
gene
</td>
<td style="text-align:right;">
53.81874
</td>
<td style="text-align:right;">
53.86818
</td>
<td style="text-align:left;">
positive
</td>
<td style="text-align:left;">
MGI_C57BL6J_3704267
</td>
<td style="text-align:left;">
Gm16499
</td>
<td style="text-align:left;">
NCBI_Gene:100049166,ENSEMBL:ENSMUSG00000108255,ENSEMBL:ENSMUSG00000108175
</td>
<td style="text-align:left;">
MGI:3704267
</td>
<td style="text-align:left;">
lncRNA gene
</td>
<td style="text-align:left;">
predicted gene 16499
</td>
</tr>
<tr>
<td style="text-align:left;">
6
</td>
<td style="text-align:left;">
gene
</td>
<td style="text-align:right;">
53.84417
</td>
<td style="text-align:right;">
53.84506
</td>
<td style="text-align:left;">
negative
</td>
<td style="text-align:left;">
MGI_C57BL6J_1923135
</td>
<td style="text-align:left;">
4930566K11Rik
</td>
<td style="text-align:left;">
NA
</td>
<td style="text-align:left;">
NA
</td>
<td style="text-align:left;">
unclassified gene
</td>
<td style="text-align:left;">
RIKEN cDNA 4930566K11 gene
</td>
</tr>
<tr>
<td style="text-align:left;">
6
</td>
<td style="text-align:left;">
gene
</td>
<td style="text-align:right;">
53.87202
</td>
<td style="text-align:right;">
53.87215
</td>
<td style="text-align:left;">
negative
</td>
<td style="text-align:left;">
MGI_C57BL6J_5452687
</td>
<td style="text-align:left;">
Gm22910
</td>
<td style="text-align:left;">
ENSEMBL:ENSMUSG00000094527
</td>
<td style="text-align:left;">
MGI:5452687
</td>
<td style="text-align:left;">
snoRNA gene
</td>
<td style="text-align:left;">
predicted gene, 22910
</td>
</tr>
<tr>
<td style="text-align:left;">
6
</td>
<td style="text-align:left;">
gene
</td>
<td style="text-align:right;">
53.87328
</td>
<td style="text-align:right;">
53.97867
</td>
<td style="text-align:left;">
negative
</td>
<td style="text-align:left;">
MGI_C57BL6J_1918537
</td>
<td style="text-align:left;">
Cpvl
</td>
<td style="text-align:left;">
NCBI_Gene:71287,ENSEMBL:ENSMUSG00000052955
</td>
<td style="text-align:left;">
MGI:1918537
</td>
<td style="text-align:left;">
protein coding gene
</td>
<td style="text-align:left;">
carboxypeptidase, vitellogenic-like
</td>
</tr>
<tr>
<td style="text-align:left;">
6
</td>
<td style="text-align:left;">
gene
</td>
<td style="text-align:right;">
53.88076
</td>
<td style="text-align:right;">
53.88310
</td>
<td style="text-align:left;">
positive
</td>
<td style="text-align:left;">
MGI_C57BL6J_5592875
</td>
<td style="text-align:left;">
Gm33716
</td>
<td style="text-align:left;">
NCBI_Gene:102636723
</td>
<td style="text-align:left;">
MGI:5592875
</td>
<td style="text-align:left;">
lncRNA gene
</td>
<td style="text-align:left;">
predicted gene, 33716
</td>
</tr>
<tr>
<td style="text-align:left;">
6
</td>
<td style="text-align:left;">
gene
</td>
<td style="text-align:right;">
53.97869
</td>
<td style="text-align:right;">
54.01803
</td>
<td style="text-align:left;">
positive
</td>
<td style="text-align:left;">
MGI_C57BL6J_2443531
</td>
<td style="text-align:left;">
4921529L05Rik
</td>
<td style="text-align:left;">
NCBI_Gene:320178,ENSEMBL:ENSMUSG00000108060
</td>
<td style="text-align:left;">
MGI:2443531
</td>
<td style="text-align:left;">
lncRNA gene
</td>
<td style="text-align:left;">
RIKEN cDNA 4921529L05 gene
</td>
</tr>
<tr>
<td style="text-align:left;">
6
</td>
<td style="text-align:left;">
gene
</td>
<td style="text-align:right;">
54.03955
</td>
<td style="text-align:right;">
54.30181
</td>
<td style="text-align:left;">
positive
</td>
<td style="text-align:left;">
MGI_C57BL6J_1917243
</td>
<td style="text-align:left;">
Chn2
</td>
<td style="text-align:left;">
NCBI_Gene:69993,ENSEMBL:ENSMUSG00000004633
</td>
<td style="text-align:left;">
MGI:1917243
</td>
<td style="text-align:left;">
protein coding gene
</td>
<td style="text-align:left;">
chimerin 2
</td>
</tr>
<tr>
<td style="text-align:left;">
6
</td>
<td style="text-align:left;">
gene
</td>
<td style="text-align:right;">
54.13952
</td>
<td style="text-align:right;">
54.15144
</td>
<td style="text-align:left;">
negative
</td>
<td style="text-align:left;">
MGI_C57BL6J_1925866
</td>
<td style="text-align:left;">
9530036M11Rik
</td>
<td style="text-align:left;">
NCBI_Gene:78616,ENSEMBL:ENSMUSG00000087670
</td>
<td style="text-align:left;">
MGI:1925866
</td>
<td style="text-align:left;">
lncRNA gene
</td>
<td style="text-align:left;">
RIKEN cDNA 9530036M11 gene
</td>
</tr>
<tr>
<td style="text-align:left;">
6
</td>
<td style="text-align:left;">
pseudogene
</td>
<td style="text-align:right;">
54.17720
</td>
<td style="text-align:right;">
54.17773
</td>
<td style="text-align:left;">
negative
</td>
<td style="text-align:left;">
MGI_C57BL6J_3782973
</td>
<td style="text-align:left;">
Gm15526
</td>
<td style="text-align:left;">
NCBI_Gene:108169144,ENSEMBL:ENSMUSG00000080205
</td>
<td style="text-align:left;">
MGI:3782973
</td>
<td style="text-align:left;">
pseudogene
</td>
<td style="text-align:left;">
predicted gene 15526
</td>
</tr>
<tr>
<td style="text-align:left;">
6
</td>
<td style="text-align:left;">
gene
</td>
<td style="text-align:right;">
54.24574
</td>
<td style="text-align:right;">
54.27908
</td>
<td style="text-align:left;">
negative
</td>
<td style="text-align:left;">
MGI_C57BL6J_3782974
</td>
<td style="text-align:left;">
Gm15527
</td>
<td style="text-align:left;">
ENSEMBL:ENSMUSG00000086946
</td>
<td style="text-align:left;">
MGI:3782974
</td>
<td style="text-align:left;">
lncRNA gene
</td>
<td style="text-align:left;">
predicted gene 15527
</td>
</tr>
<tr>
<td style="text-align:left;">
6
</td>
<td style="text-align:left;">
gene
</td>
<td style="text-align:right;">
54.26968
</td>
<td style="text-align:right;">
54.43022
</td>
<td style="text-align:left;">
negative
</td>
<td style="text-align:left;">
MGI_C57BL6J_1918824
</td>
<td style="text-align:left;">
9130019P16Rik
</td>
<td style="text-align:left;">
NCBI_Gene:100042056,ENSEMBL:ENSMUSG00000073067
</td>
<td style="text-align:left;">
MGI:1918824
</td>
<td style="text-align:left;">
lncRNA gene
</td>
<td style="text-align:left;">
RIKEN cDNA 9130019P16 gene
</td>
</tr>
<tr>
<td style="text-align:left;">
6
</td>
<td style="text-align:left;">
gene
</td>
<td style="text-align:right;">
54.32583
</td>
<td style="text-align:right;">
54.32812
</td>
<td style="text-align:left;">
negative
</td>
<td style="text-align:left;">
MGI_C57BL6J_1921781
</td>
<td style="text-align:left;">
9030405F24Rik
</td>
<td style="text-align:left;">
NA
</td>
<td style="text-align:left;">
NA
</td>
<td style="text-align:left;">
unclassified gene
</td>
<td style="text-align:left;">
RIKEN cDNA 9030405F24 gene
</td>
</tr>
<tr>
<td style="text-align:left;">
6
</td>
<td style="text-align:left;">
gene
</td>
<td style="text-align:right;">
54.32583
</td>
<td style="text-align:right;">
54.33020
</td>
<td style="text-align:left;">
positive
</td>
<td style="text-align:left;">
MGI_C57BL6J_1925254
</td>
<td style="text-align:left;">
Prr15
</td>
<td style="text-align:left;">
NCBI_Gene:78004,ENSEMBL:ENSMUSG00000045725
</td>
<td style="text-align:left;">
MGI:1925254
</td>
<td style="text-align:left;">
protein coding gene
</td>
<td style="text-align:left;">
proline rich 15
</td>
</tr>
<tr>
<td style="text-align:left;">
6
</td>
<td style="text-align:left;">
pseudogene
</td>
<td style="text-align:right;">
54.32964
</td>
<td style="text-align:right;">
54.34344
</td>
<td style="text-align:left;">
positive
</td>
<td style="text-align:left;">
MGI_C57BL6J_5593328
</td>
<td style="text-align:left;">
Gm34169
</td>
<td style="text-align:left;">
NCBI_Gene:102637330
</td>
<td style="text-align:left;">
MGI:5593328
</td>
<td style="text-align:left;">
pseudogene
</td>
<td style="text-align:left;">
predicted gene, 34169
</td>
</tr>
<tr>
<td style="text-align:left;">
6
</td>
<td style="text-align:left;">
gene
</td>
<td style="text-align:right;">
54.42922
</td>
<td style="text-align:right;">
54.50377
</td>
<td style="text-align:left;">
positive
</td>
<td style="text-align:left;">
MGI_C57BL6J_3044681
</td>
<td style="text-align:left;">
Wipf3
</td>
<td style="text-align:left;">
NCBI_Gene:330319,ENSEMBL:ENSMUSG00000086040
</td>
<td style="text-align:left;">
MGI:3044681
</td>
<td style="text-align:left;">
protein coding gene
</td>
<td style="text-align:left;">
WAS/WASL interacting protein family, member 3
</td>
</tr>
<tr>
<td style="text-align:left;">
6
</td>
<td style="text-align:left;">
gene
</td>
<td style="text-align:right;">
54.46027
</td>
<td style="text-align:right;">
54.47468
</td>
<td style="text-align:left;">
negative
</td>
<td style="text-align:left;">
MGI_C57BL6J_5690418
</td>
<td style="text-align:left;">
Gm44026
</td>
<td style="text-align:left;">
NCBI_Gene:108169145,ENSEMBL:ENSMUSG00000108081
</td>
<td style="text-align:left;">
MGI:5690418
</td>
<td style="text-align:left;">
lncRNA gene
</td>
<td style="text-align:left;">
predicted gene, 44026
</td>
</tr>
<tr>
<td style="text-align:left;">
6
</td>
<td style="text-align:left;">
gene
</td>
<td style="text-align:right;">
54.47479
</td>
<td style="text-align:right;">
54.56652
</td>
<td style="text-align:left;">
negative
</td>
<td style="text-align:left;">
MGI_C57BL6J_1917188
</td>
<td style="text-align:left;">
Scrn1
</td>
<td style="text-align:left;">
NCBI_Gene:69938,ENSEMBL:ENSMUSG00000019124
</td>
<td style="text-align:left;">
MGI:1917188
</td>
<td style="text-align:left;">
protein coding gene
</td>
<td style="text-align:left;">
secernin 1
</td>
</tr>
<tr>
<td style="text-align:left;">
6
</td>
<td style="text-align:left;">
gene
</td>
<td style="text-align:right;">
54.57760
</td>
<td style="text-align:right;">
54.59731
</td>
<td style="text-align:left;">
negative
</td>
<td style="text-align:left;">
MGI_C57BL6J_2387639
</td>
<td style="text-align:left;">
Fkbp14
</td>
<td style="text-align:left;">
NCBI_Gene:231997,ENSEMBL:ENSMUSG00000038074
</td>
<td style="text-align:left;">
MGI:2387639
</td>
<td style="text-align:left;">
protein coding gene
</td>
<td style="text-align:left;">
FK506 binding protein 14
</td>
</tr>
<tr>
<td style="text-align:left;">
6
</td>
<td style="text-align:left;">
gene
</td>
<td style="text-align:right;">
54.59511
</td>
<td style="text-align:right;">
54.64584
</td>
<td style="text-align:left;">
positive
</td>
<td style="text-align:left;">
MGI_C57BL6J_2681164
</td>
<td style="text-align:left;">
Plekha8
</td>
<td style="text-align:left;">
NCBI_Gene:231999,ENSEMBL:ENSMUSG00000005225
</td>
<td style="text-align:left;">
MGI:2681164
</td>
<td style="text-align:left;">
protein coding gene
</td>
<td style="text-align:left;">
pleckstrin homology domain containing, family A (phosphoinositide binding specific) member 8
</td>
</tr>
<tr>
<td style="text-align:left;">
6
</td>
<td style="text-align:left;">
gene
</td>
<td style="text-align:right;">
54.65944
</td>
<td style="text-align:right;">
54.66256
</td>
<td style="text-align:left;">
positive
</td>
<td style="text-align:left;">
MGI_C57BL6J_5593408
</td>
<td style="text-align:left;">
Gm34249
</td>
<td style="text-align:left;">
NCBI_Gene:102637443
</td>
<td style="text-align:left;">
MGI:5593408
</td>
<td style="text-align:left;">
lncRNA gene
</td>
<td style="text-align:left;">
predicted gene, 34249
</td>
</tr>
<tr>
<td style="text-align:left;">
6
</td>
<td style="text-align:left;">
gene
</td>
<td style="text-align:right;">
54.68093
</td>
<td style="text-align:right;">
54.68192
</td>
<td style="text-align:left;">
negative
</td>
<td style="text-align:left;">
MGI_C57BL6J_5579108
</td>
<td style="text-align:left;">
Gm28402
</td>
<td style="text-align:left;">
ENSEMBL:ENSMUSG00000101651
</td>
<td style="text-align:left;">
MGI:5579108
</td>
<td style="text-align:left;">
lncRNA gene
</td>
<td style="text-align:left;">
predicted gene 28402
</td>
</tr>
<tr>
<td style="text-align:left;">
6
</td>
<td style="text-align:left;">
gene
</td>
<td style="text-align:right;">
54.68162
</td>
<td style="text-align:right;">
54.70385
</td>
<td style="text-align:left;">
positive
</td>
<td style="text-align:left;">
MGI_C57BL6J_1915485
</td>
<td style="text-align:left;">
Mturn
</td>
<td style="text-align:left;">
NCBI_Gene:68235,ENSEMBL:ENSMUSG00000038065
</td>
<td style="text-align:left;">
MGI:1915485
</td>
<td style="text-align:left;">
protein coding gene
</td>
<td style="text-align:left;">
maturin, neural progenitor differentiation regulator homolog (Xenopus)
</td>
</tr>
<tr>
<td style="text-align:left;">
6
</td>
<td style="text-align:left;">
gene
</td>
<td style="text-align:right;">
54.69365
</td>
<td style="text-align:right;">
54.69406
</td>
<td style="text-align:left;">
positive
</td>
<td style="text-align:left;">
MGI_C57BL6J_5690400
</td>
<td style="text-align:left;">
Gm44008
</td>
<td style="text-align:left;">
ENSEMBL:ENSMUSG00000108283
</td>
<td style="text-align:left;">
MGI:5690400
</td>
<td style="text-align:left;">
unclassified gene
</td>
<td style="text-align:left;">
predicted gene, 44008
</td>
</tr>
<tr>
<td style="text-align:left;">
6
</td>
<td style="text-align:left;">
gene
</td>
<td style="text-align:right;">
54.81375
</td>
<td style="text-align:right;">
54.81765
</td>
<td style="text-align:left;">
negative
</td>
<td style="text-align:left;">
MGI_C57BL6J_1917699
</td>
<td style="text-align:left;">
2610209C05Rik
</td>
<td style="text-align:left;">
NA
</td>
<td style="text-align:left;">
NA
</td>
<td style="text-align:left;">
unclassified non-coding RNA gene
</td>
<td style="text-align:left;">
RIKEN cDNA 2610209C05 gene
</td>
</tr>
<tr>
<td style="text-align:left;">
6
</td>
<td style="text-align:left;">
gene
</td>
<td style="text-align:right;">
54.81692
</td>
<td style="text-align:right;">
54.89350
</td>
<td style="text-align:left;">
positive
</td>
<td style="text-align:left;">
MGI_C57BL6J_1196246
</td>
<td style="text-align:left;">
Znrf2
</td>
<td style="text-align:left;">
NCBI_Gene:387524,ENSEMBL:ENSMUSG00000058446
</td>
<td style="text-align:left;">
MGI:1196246
</td>
<td style="text-align:left;">
protein coding gene
</td>
<td style="text-align:left;">
zinc and ring finger 2
</td>
</tr>
<tr>
<td style="text-align:left;">
6
</td>
<td style="text-align:left;">
gene
</td>
<td style="text-align:right;">
54.90242
</td>
<td style="text-align:right;">
54.90573
</td>
<td style="text-align:left;">
positive
</td>
<td style="text-align:left;">
MGI_C57BL6J_5690577
</td>
<td style="text-align:left;">
Gm44185
</td>
<td style="text-align:left;">
ENSEMBL:ENSMUSG00000107841
</td>
<td style="text-align:left;">
MGI:5690577
</td>
<td style="text-align:left;">
unclassified gene
</td>
<td style="text-align:left;">
predicted gene, 44185
</td>
</tr>
<tr>
<td style="text-align:left;">
6
</td>
<td style="text-align:left;">
gene
</td>
<td style="text-align:right;">
54.92394
</td>
<td style="text-align:right;">
54.97267
</td>
<td style="text-align:left;">
negative
</td>
<td style="text-align:left;">
MGI_C57BL6J_1341839
</td>
<td style="text-align:left;">
Nod1
</td>
<td style="text-align:left;">
NCBI_Gene:107607,ENSEMBL:ENSMUSG00000038058
</td>
<td style="text-align:left;">
MGI:1341839
</td>
<td style="text-align:left;">
protein coding gene
</td>
<td style="text-align:left;">
nucleotide-binding oligomerization domain containing 1
</td>
</tr>
<tr>
<td style="text-align:left;">
6
</td>
<td style="text-align:left;">
gene
</td>
<td style="text-align:right;">
54.97465
</td>
<td style="text-align:right;">
54.97478
</td>
<td style="text-align:left;">
positive
</td>
<td style="text-align:left;">
MGI_C57BL6J_5454007
</td>
<td style="text-align:left;">
Gm24230
</td>
<td style="text-align:left;">
ENSEMBL:ENSMUSG00000077501
</td>
<td style="text-align:left;">
MGI:5454007
</td>
<td style="text-align:left;">
snoRNA gene
</td>
<td style="text-align:left;">
predicted gene, 24230
</td>
</tr>
<tr>
<td style="text-align:left;">
6
</td>
<td style="text-align:left;">
gene
</td>
<td style="text-align:right;">
54.98258
</td>
<td style="text-align:right;">
54.99296
</td>
<td style="text-align:left;">
negative
</td>
<td style="text-align:left;">
MGI_C57BL6J_95700
</td>
<td style="text-align:left;">
Ggct
</td>
<td style="text-align:left;">
NCBI_Gene:110175,ENSEMBL:ENSMUSG00000002797
</td>
<td style="text-align:left;">
MGI:95700
</td>
<td style="text-align:left;">
protein coding gene
</td>
<td style="text-align:left;">
gamma-glutamyl cyclotransferase
</td>
</tr>
<tr>
<td style="text-align:left;">
6
</td>
<td style="text-align:left;">
gene
</td>
<td style="text-align:right;">
55.00212
</td>
<td style="text-align:right;">
55.00400
</td>
<td style="text-align:left;">
positive
</td>
<td style="text-align:left;">
MGI_C57BL6J_5621424
</td>
<td style="text-align:left;">
Gm38539
</td>
<td style="text-align:left;">
NCBI_Gene:102641179
</td>
<td style="text-align:left;">
MGI:5621424
</td>
<td style="text-align:left;">
lncRNA gene
</td>
<td style="text-align:left;">
predicted gene, 38539
</td>
</tr>
<tr>
<td style="text-align:left;">
6
</td>
<td style="text-align:left;">
gene
</td>
<td style="text-align:right;">
55.03800
</td>
<td style="text-align:right;">
55.07950
</td>
<td style="text-align:left;">
positive
</td>
<td style="text-align:left;">
MGI_C57BL6J_2449057
</td>
<td style="text-align:left;">
Gars
</td>
<td style="text-align:left;">
NCBI_Gene:353172,ENSEMBL:ENSMUSG00000029777
</td>
<td style="text-align:left;">
MGI:2449057
</td>
<td style="text-align:left;">
protein coding gene
</td>
<td style="text-align:left;">
glycyl-tRNA synthetase
</td>
</tr>
<tr>
<td style="text-align:left;">
6
</td>
<td style="text-align:left;">
gene
</td>
<td style="text-align:right;">
55.09005
</td>
<td style="text-align:right;">
55.13302
</td>
<td style="text-align:left;">
negative
</td>
<td style="text-align:left;">
MGI_C57BL6J_894312
</td>
<td style="text-align:left;">
Crhr2
</td>
<td style="text-align:left;">
NCBI_Gene:12922,ENSEMBL:ENSMUSG00000003476
</td>
<td style="text-align:left;">
MGI:894312
</td>
<td style="text-align:left;">
protein coding gene
</td>
<td style="text-align:left;">
corticotropin releasing hormone receptor 2
</td>
</tr>
<tr>
<td style="text-align:left;">
6
</td>
<td style="text-align:left;">
gene
</td>
<td style="text-align:right;">
55.15443
</td>
<td style="text-align:right;">
55.16721
</td>
<td style="text-align:left;">
positive
</td>
<td style="text-align:left;">
MGI_C57BL6J_5593517
</td>
<td style="text-align:left;">
Gm34358
</td>
<td style="text-align:left;">
NCBI_Gene:102637587,ENSEMBL:ENSMUSG00000108285
</td>
<td style="text-align:left;">
MGI:5593517
</td>
<td style="text-align:left;">
lncRNA gene
</td>
<td style="text-align:left;">
predicted gene, 34358
</td>
</tr>
<tr>
<td style="text-align:left;">
6
</td>
<td style="text-align:left;">
gene
</td>
<td style="text-align:right;">
55.17063
</td>
<td style="text-align:right;">
55.17504
</td>
<td style="text-align:left;">
negative
</td>
<td style="text-align:left;">
MGI_C57BL6J_102963
</td>
<td style="text-align:left;">
Inmt
</td>
<td style="text-align:left;">
NCBI_Gene:21743,ENSEMBL:ENSMUSG00000003477
</td>
<td style="text-align:left;">
MGI:102963
</td>
<td style="text-align:left;">
protein coding gene
</td>
<td style="text-align:left;">
indolethylamine N-methyltransferase
</td>
</tr>
<tr>
<td style="text-align:left;">
6
</td>
<td style="text-align:left;">
gene
</td>
<td style="text-align:right;">
55.20334
</td>
<td style="text-align:right;">
55.32022
</td>
<td style="text-align:left;">
positive
</td>
<td style="text-align:left;">
MGI_C57BL6J_3583959
</td>
<td style="text-align:left;">
Mindy4
</td>
<td style="text-align:left;">
NCBI_Gene:330323,ENSEMBL:ENSMUSG00000038022
</td>
<td style="text-align:left;">
MGI:3583959
</td>
<td style="text-align:left;">
protein coding gene
</td>
<td style="text-align:left;">
MINDY lysine 48 deubiquitinase 4
</td>
</tr>
<tr>
<td style="text-align:left;">
6
</td>
<td style="text-align:left;">
gene
</td>
<td style="text-align:right;">
55.29626
</td>
<td style="text-align:right;">
55.30789
</td>
<td style="text-align:left;">
negative
</td>
<td style="text-align:left;">
MGI_C57BL6J_5690465
</td>
<td style="text-align:left;">
Gm44073
</td>
<td style="text-align:left;">
ENSEMBL:ENSMUSG00000107766
</td>
<td style="text-align:left;">
MGI:5690465
</td>
<td style="text-align:left;">
lncRNA gene
</td>
<td style="text-align:left;">
predicted gene, 44073
</td>
</tr>
<tr>
<td style="text-align:left;">
6
</td>
<td style="text-align:left;">
gene
</td>
<td style="text-align:right;">
55.30880
</td>
<td style="text-align:right;">
55.30891
</td>
<td style="text-align:left;">
negative
</td>
<td style="text-align:left;">
MGI_C57BL6J_5455235
</td>
<td style="text-align:left;">
Gm25458
</td>
<td style="text-align:left;">
ENSEMBL:ENSMUSG00000093878
</td>
<td style="text-align:left;">
MGI:5455235
</td>
<td style="text-align:left;">
miRNA gene
</td>
<td style="text-align:left;">
predicted gene, 25458
</td>
</tr>
<tr>
<td style="text-align:left;">
6
</td>
<td style="text-align:left;">
gene
</td>
<td style="text-align:right;">
55.32564
</td>
<td style="text-align:right;">
55.33084
</td>
<td style="text-align:left;">
negative
</td>
<td style="text-align:left;">
MGI_C57BL6J_5621704
</td>
<td style="text-align:left;">
Gm38819
</td>
<td style="text-align:left;">
NCBI_Gene:105242666
</td>
<td style="text-align:left;">
MGI:5621704
</td>
<td style="text-align:left;">
lncRNA gene
</td>
<td style="text-align:left;">
predicted gene, 38819
</td>
</tr>
<tr>
<td style="text-align:left;">
6
</td>
<td style="text-align:left;">
gene
</td>
<td style="text-align:right;">
55.33630
</td>
<td style="text-align:right;">
55.34855
</td>
<td style="text-align:left;">
positive
</td>
<td style="text-align:left;">
MGI_C57BL6J_103201
</td>
<td style="text-align:left;">
Aqp1
</td>
<td style="text-align:left;">
NCBI_Gene:11826,ENSEMBL:ENSMUSG00000004655
</td>
<td style="text-align:left;">
MGI:103201
</td>
<td style="text-align:left;">
protein coding gene
</td>
<td style="text-align:left;">
aquaporin 1
</td>
</tr>
<tr>
<td style="text-align:left;">
6
</td>
<td style="text-align:left;">
gene
</td>
<td style="text-align:right;">
55.37629
</td>
<td style="text-align:right;">
55.38853
</td>
<td style="text-align:left;">
positive
</td>
<td style="text-align:left;">
MGI_C57BL6J_95710
</td>
<td style="text-align:left;">
Ghrhr
</td>
<td style="text-align:left;">
NCBI_Gene:14602,ENSEMBL:ENSMUSG00000004654
</td>
<td style="text-align:left;">
MGI:95710
</td>
<td style="text-align:left;">
protein coding gene
</td>
<td style="text-align:left;">
growth hormone releasing hormone receptor
</td>
</tr>
<tr>
<td style="text-align:left;">
6
</td>
<td style="text-align:left;">
gene
</td>
<td style="text-align:right;">
55.39689
</td>
<td style="text-align:right;">
55.41228
</td>
<td style="text-align:left;">
positive
</td>
<td style="text-align:left;">
MGI_C57BL6J_5439415
</td>
<td style="text-align:left;">
6430584L05Rik
</td>
<td style="text-align:left;">
NCBI_Gene:330324,ENSEMBL:ENSMUSG00000108228
</td>
<td style="text-align:left;">
MGI:5439415
</td>
<td style="text-align:left;">
lncRNA gene
</td>
<td style="text-align:left;">
RIKEN cDNA 6430584L05 gene
</td>
</tr>
<tr>
<td style="text-align:left;">
6
</td>
<td style="text-align:left;">
gene
</td>
<td style="text-align:right;">
55.45150
</td>
<td style="text-align:right;">
55.50145
</td>
<td style="text-align:left;">
positive
</td>
<td style="text-align:left;">
MGI_C57BL6J_108449
</td>
<td style="text-align:left;">
Adcyap1r1
</td>
<td style="text-align:left;">
NCBI_Gene:11517,ENSEMBL:ENSMUSG00000029778
</td>
<td style="text-align:left;">
MGI:108449
</td>
<td style="text-align:left;">
protein coding gene
</td>
<td style="text-align:left;">
adenylate cyclase activating polypeptide 1 receptor 1
</td>
</tr>
<tr>
<td style="text-align:left;">
6
</td>
<td style="text-align:left;">
gene
</td>
<td style="text-align:right;">
55.53351
</td>
<td style="text-align:right;">
55.53361
</td>
<td style="text-align:left;">
positive
</td>
<td style="text-align:left;">
MGI_C57BL6J_5690744
</td>
<td style="text-align:left;">
Gm44352
</td>
<td style="text-align:left;">
ENSEMBL:ENSMUSG00000104746
</td>
<td style="text-align:left;">
MGI:5690744
</td>
<td style="text-align:left;">
miRNA gene
</td>
<td style="text-align:left;">
predicted gene, 44352
</td>
</tr>
<tr>
<td style="text-align:left;">
6
</td>
<td style="text-align:left;">
gene
</td>
<td style="text-align:right;">
55.53704
</td>
<td style="text-align:right;">
55.65178
</td>
<td style="text-align:left;">
positive
</td>
<td style="text-align:left;">
MGI_C57BL6J_5593600
</td>
<td style="text-align:left;">
Gm34441
</td>
<td style="text-align:left;">
NCBI_Gene:102637698
</td>
<td style="text-align:left;">
MGI:5593600
</td>
<td style="text-align:left;">
lncRNA gene
</td>
<td style="text-align:left;">
predicted gene, 34441
</td>
</tr>
<tr>
<td style="text-align:left;">
6
</td>
<td style="text-align:left;">
pseudogene
</td>
<td style="text-align:right;">
55.62508
</td>
<td style="text-align:right;">
55.62561
</td>
<td style="text-align:left;">
positive
</td>
<td style="text-align:left;">
MGI_C57BL6J_3781904
</td>
<td style="text-align:left;">
Gm3729
</td>
<td style="text-align:left;">
NCBI_Gene:100416934
</td>
<td style="text-align:left;">
MGI:3781904
</td>
<td style="text-align:left;">
pseudogene
</td>
<td style="text-align:left;">
predicted gene 3729
</td>
</tr>
<tr>
<td style="text-align:left;">
6
</td>
<td style="text-align:left;">
gene
</td>
<td style="text-align:right;">
55.67782
</td>
<td style="text-align:right;">
55.68126
</td>
<td style="text-align:left;">
negative
</td>
<td style="text-align:left;">
MGI_C57BL6J_106593
</td>
<td style="text-align:left;">
Neurod6
</td>
<td style="text-align:left;">
NCBI_Gene:11922,ENSEMBL:ENSMUSG00000037984
</td>
<td style="text-align:left;">
MGI:106593
</td>
<td style="text-align:left;">
protein coding gene
</td>
<td style="text-align:left;">
neurogenic differentiation 6
</td>
</tr>
<tr>
<td style="text-align:left;">
6
</td>
<td style="text-align:left;">
gene
</td>
<td style="text-align:right;">
55.68457
</td>
<td style="text-align:right;">
55.69457
</td>
<td style="text-align:left;">
positive
</td>
<td style="text-align:left;">
MGI_C57BL6J_3781457
</td>
<td style="text-align:left;">
Gm3279
</td>
<td style="text-align:left;">
NCBI_Gene:100041339,ENSEMBL:ENSMUSG00000108063
</td>
<td style="text-align:left;">
MGI:3781457
</td>
<td style="text-align:left;">
lncRNA gene
</td>
<td style="text-align:left;">
predicted gene 3279
</td>
</tr>
<tr>
<td style="text-align:left;">
6
</td>
<td style="text-align:left;">
gene
</td>
<td style="text-align:right;">
55.70057
</td>
<td style="text-align:right;">
55.70589
</td>
<td style="text-align:left;">
positive
</td>
<td style="text-align:left;">
MGI_C57BL6J_5593835
</td>
<td style="text-align:left;">
Gm34676
</td>
<td style="text-align:left;">
NCBI_Gene:102638001
</td>
<td style="text-align:left;">
MGI:5593835
</td>
<td style="text-align:left;">
lncRNA gene
</td>
<td style="text-align:left;">
predicted gene, 34676
</td>
</tr>
<tr>
<td style="text-align:left;">
6
</td>
<td style="text-align:left;">
gene
</td>
<td style="text-align:right;">
55.78959
</td>
<td style="text-align:right;">
55.80470
</td>
<td style="text-align:left;">
negative
</td>
<td style="text-align:left;">
MGI_C57BL6J_5621705
</td>
<td style="text-align:left;">
Gm38820
</td>
<td style="text-align:left;">
NCBI_Gene:105242667
</td>
<td style="text-align:left;">
MGI:5621705
</td>
<td style="text-align:left;">
lncRNA gene
</td>
<td style="text-align:left;">
predicted gene, 38820
</td>
</tr>
<tr>
<td style="text-align:left;">
6
</td>
<td style="text-align:left;">
gene
</td>
<td style="text-align:right;">
55.80463
</td>
<td style="text-align:right;">
55.97874
</td>
<td style="text-align:left;">
positive
</td>
<td style="text-align:left;">
MGI_C57BL6J_2685304
</td>
<td style="text-align:left;">
Itprid1
</td>
<td style="text-align:left;">
NCBI_Gene:232016,ENSEMBL:ENSMUSG00000037973
</td>
<td style="text-align:left;">
MGI:2685304
</td>
<td style="text-align:left;">
protein coding gene
</td>
<td style="text-align:left;">
ITPR interacting domain containing 1
</td>
</tr>
<tr>
<td style="text-align:left;">
6
</td>
<td style="text-align:left;">
gene
</td>
<td style="text-align:right;">
56.01750
</td>
<td style="text-align:right;">
56.03269
</td>
<td style="text-align:left;">
positive
</td>
<td style="text-align:left;">
MGI_C57BL6J_1333876
</td>
<td style="text-align:left;">
Ppp1r17
</td>
<td style="text-align:left;">
NCBI_Gene:19051,ENSEMBL:ENSMUSG00000002930
</td>
<td style="text-align:left;">
MGI:1333876
</td>
<td style="text-align:left;">
protein coding gene
</td>
<td style="text-align:left;">
protein phosphatase 1, regulatory subunit 17
</td>
</tr>
<tr>
<td style="text-align:left;">
6
</td>
<td style="text-align:left;">
pseudogene
</td>
<td style="text-align:right;">
56.06094
</td>
<td style="text-align:right;">
56.06148
</td>
<td style="text-align:left;">
negative
</td>
<td style="text-align:left;">
MGI_C57BL6J_5010284
</td>
<td style="text-align:left;">
Gm18099
</td>
<td style="text-align:left;">
NCBI_Gene:100416405
</td>
<td style="text-align:left;">
MGI:5010284
</td>
<td style="text-align:left;">
pseudogene
</td>
<td style="text-align:left;">
predicted gene, 18099
</td>
</tr>
<tr>
<td style="text-align:left;">
6
</td>
<td style="text-align:left;">
gene
</td>
<td style="text-align:right;">
56.06980
</td>
<td style="text-align:right;">
56.65251
</td>
<td style="text-align:left;">
negative
</td>
<td style="text-align:left;">
MGI_C57BL6J_108413
</td>
<td style="text-align:left;">
Pde1c
</td>
<td style="text-align:left;">
NCBI_Gene:18575,ENSEMBL:ENSMUSG00000004347
</td>
<td style="text-align:left;">
MGI:108413
</td>
<td style="text-align:left;">
protein coding gene
</td>
<td style="text-align:left;">
phosphodiesterase 1C
</td>
</tr>
<tr>
<td style="text-align:left;">
6
</td>
<td style="text-align:left;">
gene
</td>
<td style="text-align:right;">
56.17293
</td>
<td style="text-align:right;">
56.17303
</td>
<td style="text-align:left;">
negative
</td>
<td style="text-align:left;">
MGI_C57BL6J_5690805
</td>
<td style="text-align:left;">
Gm44413
</td>
<td style="text-align:left;">
ENSEMBL:ENSMUSG00000106152
</td>
<td style="text-align:left;">
MGI:5690805
</td>
<td style="text-align:left;">
miRNA gene
</td>
<td style="text-align:left;">
predicted gene, 44413
</td>
</tr>
<tr>
<td style="text-align:left;">
6
</td>
<td style="text-align:left;">
gene
</td>
<td style="text-align:right;">
56.36971
</td>
<td style="text-align:right;">
56.37194
</td>
<td style="text-align:left;">
positive
</td>
<td style="text-align:left;">
MGI_C57BL6J_5593912
</td>
<td style="text-align:left;">
Gm34753
</td>
<td style="text-align:left;">
NA
</td>
<td style="text-align:left;">
NA
</td>
<td style="text-align:left;">
unclassified non-coding RNA gene
</td>
<td style="text-align:left;">
predicted gene%2c 34753
</td>
</tr>
<tr>
<td style="text-align:left;">
6
</td>
<td style="text-align:left;">
gene
</td>
<td style="text-align:right;">
56.58120
</td>
<td style="text-align:right;">
56.65249
</td>
<td style="text-align:left;">
negative
</td>
<td style="text-align:left;">
MGI_C57BL6J_3644042
</td>
<td style="text-align:left;">
Gm8239
</td>
<td style="text-align:left;">
ENSEMBL:ENSMUSG00000107494
</td>
<td style="text-align:left;">
MGI:3644042
</td>
<td style="text-align:left;">
lncRNA gene
</td>
<td style="text-align:left;">
predicted gene 8239
</td>
</tr>
<tr>
<td style="text-align:left;">
6
</td>
<td style="text-align:left;">
gene
</td>
<td style="text-align:right;">
56.58371
</td>
<td style="text-align:right;">
56.58385
</td>
<td style="text-align:left;">
negative
</td>
<td style="text-align:left;">
MGI_C57BL6J_5454486
</td>
<td style="text-align:left;">
Gm24709
</td>
<td style="text-align:left;">
ENSEMBL:ENSMUSG00000096087
</td>
<td style="text-align:left;">
MGI:5454486
</td>
<td style="text-align:left;">
snoRNA gene
</td>
<td style="text-align:left;">
predicted gene, 24709
</td>
</tr>
<tr>
<td style="text-align:left;">
6
</td>
<td style="text-align:left;">
gene
</td>
<td style="text-align:right;">
56.60082
</td>
<td style="text-align:right;">
56.60095
</td>
<td style="text-align:left;">
negative
</td>
<td style="text-align:left;">
MGI_C57BL6J_5452270
</td>
<td style="text-align:left;">
Gm22493
</td>
<td style="text-align:left;">
ENSEMBL:ENSMUSG00000077682
</td>
<td style="text-align:left;">
MGI:5452270
</td>
<td style="text-align:left;">
snoRNA gene
</td>
<td style="text-align:left;">
predicted gene, 22493
</td>
</tr>
<tr>
<td style="text-align:left;">
6
</td>
<td style="text-align:left;">
gene
</td>
<td style="text-align:right;">
56.70106
</td>
<td style="text-align:right;">
56.70471
</td>
<td style="text-align:left;">
negative
</td>
<td style="text-align:left;">
MGI_C57BL6J_1913623
</td>
<td style="text-align:left;">
Lsm5
</td>
<td style="text-align:left;">
NCBI_Gene:66373,ENSEMBL:ENSMUSG00000091625
</td>
<td style="text-align:left;">
MGI:1913623
</td>
<td style="text-align:left;">
protein coding gene
</td>
<td style="text-align:left;">
LSM5 homolog, U6 small nuclear RNA and mRNA degradation associated
</td>
</tr>
<tr>
<td style="text-align:left;">
6
</td>
<td style="text-align:left;">
gene
</td>
<td style="text-align:right;">
56.71450
</td>
<td style="text-align:right;">
56.71573
</td>
<td style="text-align:left;">
negative
</td>
<td style="text-align:left;">
MGI_C57BL6J_1918466
</td>
<td style="text-align:left;">
4933427C19Rik
</td>
<td style="text-align:left;">
NA
</td>
<td style="text-align:left;">
NA
</td>
<td style="text-align:left;">
unclassified gene
</td>
<td style="text-align:left;">
RIKEN cDNA 4933427C19 gene
</td>
</tr>
<tr>
<td style="text-align:left;">
6
</td>
<td style="text-align:left;">
gene
</td>
<td style="text-align:right;">
56.71490
</td>
<td style="text-align:right;">
56.76191
</td>
<td style="text-align:left;">
positive
</td>
<td style="text-align:left;">
MGI_C57BL6J_1926187
</td>
<td style="text-align:left;">
Avl9
</td>
<td style="text-align:left;">
NCBI_Gene:78937,ENSEMBL:ENSMUSG00000029787
</td>
<td style="text-align:left;">
MGI:1926187
</td>
<td style="text-align:left;">
protein coding gene
</td>
<td style="text-align:left;">
AVL9 cell migration associated
</td>
</tr>
<tr>
<td style="text-align:left;">
6
</td>
<td style="text-align:left;">
gene
</td>
<td style="text-align:right;">
56.77582
</td>
<td style="text-align:right;">
56.77735
</td>
<td style="text-align:left;">
negative
</td>
<td style="text-align:left;">
MGI_C57BL6J_1920419
</td>
<td style="text-align:left;">
3110035G12Rik
</td>
<td style="text-align:left;">
NA
</td>
<td style="text-align:left;">
NA
</td>
<td style="text-align:left;">
unclassified gene
</td>
<td style="text-align:left;">
RIKEN cDNA 3110035G12 gene
</td>
</tr>
<tr>
<td style="text-align:left;">
6
</td>
<td style="text-align:left;">
gene
</td>
<td style="text-align:right;">
56.77752
</td>
<td style="text-align:right;">
56.79827
</td>
<td style="text-align:left;">
negative
</td>
<td style="text-align:left;">
MGI_C57BL6J_2384811
</td>
<td style="text-align:left;">
Kbtbd2
</td>
<td style="text-align:left;">
NCBI_Gene:210973,ENSEMBL:ENSMUSG00000059486
</td>
<td style="text-align:left;">
MGI:2384811
</td>
<td style="text-align:left;">
protein coding gene
</td>
<td style="text-align:left;">
kelch repeat and BTB (POZ) domain containing 2
</td>
</tr>
<tr>
<td style="text-align:left;">
6
</td>
<td style="text-align:left;">
gene
</td>
<td style="text-align:right;">
56.79759
</td>
<td style="text-align:right;">
56.79920
</td>
<td style="text-align:left;">
positive
</td>
<td style="text-align:left;">
MGI_C57BL6J_3641874
</td>
<td style="text-align:left;">
Gm10209
</td>
<td style="text-align:left;">
ENSEMBL:ENSMUSG00000108080
</td>
<td style="text-align:left;">
MGI:3641874
</td>
<td style="text-align:left;">
protein coding gene
</td>
<td style="text-align:left;">
predicted gene 10209
</td>
</tr>
<tr>
<td style="text-align:left;">
6
</td>
<td style="text-align:left;">
gene
</td>
<td style="text-align:right;">
56.83206
</td>
<td style="text-align:right;">
56.87936
</td>
<td style="text-align:left;">
positive
</td>
<td style="text-align:left;">
MGI_C57BL6J_1350921
</td>
<td style="text-align:left;">
Fkbp9
</td>
<td style="text-align:left;">
NCBI_Gene:27055,ENSEMBL:ENSMUSG00000029781
</td>
<td style="text-align:left;">
MGI:1350921
</td>
<td style="text-align:left;">
protein coding gene
</td>
<td style="text-align:left;">
FK506 binding protein 9
</td>
</tr>
<tr>
<td style="text-align:left;">
6
</td>
<td style="text-align:left;">
gene
</td>
<td style="text-align:right;">
56.88240
</td>
<td style="text-align:right;">
56.92393
</td>
<td style="text-align:left;">
negative
</td>
<td style="text-align:left;">
MGI_C57BL6J_1927186
</td>
<td style="text-align:left;">
Nt5c3
</td>
<td style="text-align:left;">
NCBI_Gene:107569,ENSEMBL:ENSMUSG00000029780
</td>
<td style="text-align:left;">
MGI:1927186
</td>
<td style="text-align:left;">
protein coding gene
</td>
<td style="text-align:left;">
5’-nucleotidase, cytosolic III
</td>
</tr>
<tr>
<td style="text-align:left;">
6
</td>
<td style="text-align:left;">
gene
</td>
<td style="text-align:right;">
56.88443
</td>
<td style="text-align:right;">
56.88611
</td>
<td style="text-align:left;">
positive
</td>
<td style="text-align:left;">
MGI_C57BL6J_5609018
</td>
<td style="text-align:left;">
AK013988
</td>
<td style="text-align:left;">
NA
</td>
<td style="text-align:left;">
NA
</td>
<td style="text-align:left;">
unclassified non-coding RNA gene
</td>
<td style="text-align:left;">
expressed sequence AK013988
</td>
</tr>
<tr>
<td style="text-align:left;">
6
</td>
<td style="text-align:left;">
gene
</td>
<td style="text-align:right;">
56.90591
</td>
<td style="text-align:right;">
56.90658
</td>
<td style="text-align:left;">
negative
</td>
<td style="text-align:left;">
MGI_C57BL6J_3781966
</td>
<td style="text-align:left;">
Gm3793
</td>
<td style="text-align:left;">
ENSEMBL:ENSMUSG00000108260
</td>
<td style="text-align:left;">
MGI:3781966
</td>
<td style="text-align:left;">
unclassified gene
</td>
<td style="text-align:left;">
predicted gene 3793
</td>
</tr>
<tr>
<td style="text-align:left;">
6
</td>
<td style="text-align:left;">
gene
</td>
<td style="text-align:right;">
56.92401
</td>
<td style="text-align:right;">
56.95810
</td>
<td style="text-align:left;">
positive
</td>
<td style="text-align:left;">
MGI_C57BL6J_2159457
</td>
<td style="text-align:left;">
Vmn1r4
</td>
<td style="text-align:left;">
NCBI_Gene:171194,ENSEMBL:ENSMUSG00000071893
</td>
<td style="text-align:left;">
MGI:2159457
</td>
<td style="text-align:left;">
protein coding gene
</td>
<td style="text-align:left;">
vomeronasal 1 receptor 4
</td>
</tr>
<tr>
<td style="text-align:left;">
6
</td>
<td style="text-align:left;">
gene
</td>
<td style="text-align:right;">
56.94184
</td>
<td style="text-align:right;">
56.94482
</td>
<td style="text-align:left;">
negative
</td>
<td style="text-align:left;">
MGI_C57BL6J_5690441
</td>
<td style="text-align:left;">
Gm44049
</td>
<td style="text-align:left;">
ENSEMBL:ENSMUSG00000108021
</td>
<td style="text-align:left;">
MGI:5690441
</td>
<td style="text-align:left;">
unclassified gene
</td>
<td style="text-align:left;">
predicted gene, 44049
</td>
</tr>
<tr>
<td style="text-align:left;">
6
</td>
<td style="text-align:left;">
gene
</td>
<td style="text-align:right;">
56.97027
</td>
<td style="text-align:right;">
56.98839
</td>
<td style="text-align:left;">
positive
</td>
<td style="text-align:left;">
MGI_C57BL6J_2159455
</td>
<td style="text-align:left;">
Vmn1r5
</td>
<td style="text-align:left;">
NCBI_Gene:171192,ENSEMBL:ENSMUSG00000090346
</td>
<td style="text-align:left;">
MGI:2159455
</td>
<td style="text-align:left;">
protein coding gene
</td>
<td style="text-align:left;">
vomeronasal 1 receptor 5
</td>
</tr>
<tr>
<td style="text-align:left;">
6
</td>
<td style="text-align:left;">
gene
</td>
<td style="text-align:right;">
56.99724
</td>
<td style="text-align:right;">
57.00982
</td>
<td style="text-align:left;">
positive
</td>
<td style="text-align:left;">
MGI_C57BL6J_2159456
</td>
<td style="text-align:left;">
Vmn1r6
</td>
<td style="text-align:left;">
NCBI_Gene:171193,ENSEMBL:ENSMUSG00000115701
</td>
<td style="text-align:left;">
MGI:2159456
</td>
<td style="text-align:left;">
protein coding gene
</td>
<td style="text-align:left;">
vomeronasal 1 receptor 6
</td>
</tr>
<tr>
<td style="text-align:left;">
6
</td>
<td style="text-align:left;">
gene
</td>
<td style="text-align:right;">
57.02411
</td>
<td style="text-align:right;">
57.02532
</td>
<td style="text-align:left;">
negative
</td>
<td style="text-align:left;">
MGI_C57BL6J_2159467
</td>
<td style="text-align:left;">
Vmn1r7
</td>
<td style="text-align:left;">
NCBI_Gene:434016,ENSEMBL:ENSMUSG00000093696
</td>
<td style="text-align:left;">
MGI:2159467
</td>
<td style="text-align:left;">
protein coding gene
</td>
<td style="text-align:left;">
vomeronasal 1 receptor 7
</td>
</tr>
<tr>
<td style="text-align:left;">
6
</td>
<td style="text-align:left;">
gene
</td>
<td style="text-align:right;">
57.03228
</td>
<td style="text-align:right;">
57.03739
</td>
<td style="text-align:left;">
positive
</td>
<td style="text-align:left;">
MGI_C57BL6J_2159469
</td>
<td style="text-align:left;">
Vmn1r8
</td>
<td style="text-align:left;">
NCBI_Gene:171205,ENSEMBL:ENSMUSG00000061208
</td>
<td style="text-align:left;">
MGI:2159469
</td>
<td style="text-align:left;">
protein coding gene
</td>
<td style="text-align:left;">
vomeronasal 1 receptor 8
</td>
</tr>
<tr>
<td style="text-align:left;">
6
</td>
<td style="text-align:left;">
gene
</td>
<td style="text-align:right;">
57.06679
</td>
<td style="text-align:right;">
57.07491
</td>
<td style="text-align:left;">
positive
</td>
<td style="text-align:left;">
MGI_C57BL6J_2159466
</td>
<td style="text-align:left;">
Vmn1r9
</td>
<td style="text-align:left;">
NCBI_Gene:171203,ENSEMBL:ENSMUSG00000091541
</td>
<td style="text-align:left;">
MGI:2159466
</td>
<td style="text-align:left;">
protein coding gene
</td>
<td style="text-align:left;">
vomeronasal 1 receptor 9
</td>
</tr>
<tr>
<td style="text-align:left;">
6
</td>
<td style="text-align:left;">
gene
</td>
<td style="text-align:right;">
57.10945
</td>
<td style="text-align:right;">
57.11503
</td>
<td style="text-align:left;">
positive
</td>
<td style="text-align:left;">
MGI_C57BL6J_2148522
</td>
<td style="text-align:left;">
Vmn1r10
</td>
<td style="text-align:left;">
NCBI_Gene:113858,ENSEMBL:ENSMUSG00000115181
</td>
<td style="text-align:left;">
MGI:2148522
</td>
<td style="text-align:left;">
protein coding gene
</td>
<td style="text-align:left;">
vomeronasal 1 receptor 10
</td>
</tr>
<tr>
<td style="text-align:left;">
6
</td>
<td style="text-align:left;">
gene
</td>
<td style="text-align:right;">
57.13345
</td>
<td style="text-align:right;">
57.13962
</td>
<td style="text-align:left;">
positive
</td>
<td style="text-align:left;">
MGI_C57BL6J_2148524
</td>
<td style="text-align:left;">
Vmn1r11
</td>
<td style="text-align:left;">
NCBI_Gene:113860,ENSEMBL:ENSMUSG00000115236
</td>
<td style="text-align:left;">
MGI:2148524
</td>
<td style="text-align:left;">
protein coding gene
</td>
<td style="text-align:left;">
vomeronasal 1 receptor 11
</td>
</tr>
<tr>
<td style="text-align:left;">
6
</td>
<td style="text-align:left;">
gene
</td>
<td style="text-align:right;">
57.15692
</td>
<td style="text-align:right;">
57.16533
</td>
<td style="text-align:left;">
positive
</td>
<td style="text-align:left;">
MGI_C57BL6J_3645555
</td>
<td style="text-align:left;">
Vmn1r12
</td>
<td style="text-align:left;">
NCBI_Gene:626397,ENSEMBL:ENSMUSG00000057981
</td>
<td style="text-align:left;">
MGI:3645555
</td>
<td style="text-align:left;">
protein coding gene
</td>
<td style="text-align:left;">
vomeronasal 1 receptor 12
</td>
</tr>
<tr>
<td style="text-align:left;">
6
</td>
<td style="text-align:left;">
pseudogene
</td>
<td style="text-align:right;">
57.18849
</td>
<td style="text-align:right;">
57.18944
</td>
<td style="text-align:left;">
positive
</td>
<td style="text-align:left;">
MGI_C57BL6J_2148525
</td>
<td style="text-align:left;">
Vmn1r-ps8
</td>
<td style="text-align:left;">
NCBI_Gene:113861,ENSEMBL:ENSMUSG00000093728
</td>
<td style="text-align:left;">
MGI:2148525
</td>
<td style="text-align:left;">
pseudogene
</td>
<td style="text-align:left;">
vomeronasal 1 receptor, pseudogene 8
</td>
</tr>
<tr>
<td style="text-align:left;">
6
</td>
<td style="text-align:left;">
gene
</td>
<td style="text-align:right;">
57.20090
</td>
<td style="text-align:right;">
57.21480
</td>
<td style="text-align:left;">
positive
</td>
<td style="text-align:left;">
MGI_C57BL6J_2148526
</td>
<td style="text-align:left;">
Vmn1r13
</td>
<td style="text-align:left;">
NCBI_Gene:113862,ENSEMBL:ENSMUSG00000064259
</td>
<td style="text-align:left;">
MGI:2148526
</td>
<td style="text-align:left;">
protein coding gene
</td>
<td style="text-align:left;">
vomeronasal 1 receptor 13
</td>
</tr>
<tr>
<td style="text-align:left;">
6
</td>
<td style="text-align:left;">
gene
</td>
<td style="text-align:right;">
57.22615
</td>
<td style="text-align:right;">
57.24088
</td>
<td style="text-align:left;">
positive
</td>
<td style="text-align:left;">
MGI_C57BL6J_2148528
</td>
<td style="text-align:left;">
Vmn1r14
</td>
<td style="text-align:left;">
NCBI_Gene:113864,ENSEMBL:ENSMUSG00000114982
</td>
<td style="text-align:left;">
MGI:2148528
</td>
<td style="text-align:left;">
protein coding gene
</td>
<td style="text-align:left;">
vomeronasal 1 receptor 14
</td>
</tr>
<tr>
<td style="text-align:left;">
6
</td>
<td style="text-align:left;">
gene
</td>
<td style="text-align:right;">
57.25795
</td>
<td style="text-align:right;">
57.25997
</td>
<td style="text-align:left;">
positive
</td>
<td style="text-align:left;">
MGI_C57BL6J_2148527
</td>
<td style="text-align:left;">
Vmn1r15
</td>
<td style="text-align:left;">
NCBI_Gene:113863,ENSEMBL:ENSMUSG00000115199
</td>
<td style="text-align:left;">
MGI:2148527
</td>
<td style="text-align:left;">
protein coding gene
</td>
<td style="text-align:left;">
vomeronasal 1 receptor 15
</td>
</tr>
<tr>
<td style="text-align:left;">
6
</td>
<td style="text-align:left;">
pseudogene
</td>
<td style="text-align:right;">
57.27802
</td>
<td style="text-align:right;">
57.27867
</td>
<td style="text-align:left;">
positive
</td>
<td style="text-align:left;">
MGI_C57BL6J_3781521
</td>
<td style="text-align:left;">
Vmn1r-ps9
</td>
<td style="text-align:left;">
NCBI_Gene:100041445,ENSEMBL:ENSMUSG00000093520
</td>
<td style="text-align:left;">
MGI:3781521
</td>
<td style="text-align:left;">
pseudogene
</td>
<td style="text-align:left;">
vomeronasal 1 receptor, pseudogene 9
</td>
</tr>
<tr>
<td style="text-align:left;">
6
</td>
<td style="text-align:left;">
pseudogene
</td>
<td style="text-align:right;">
57.31294
</td>
<td style="text-align:right;">
57.31383
</td>
<td style="text-align:left;">
positive
</td>
<td style="text-align:left;">
MGI_C57BL6J_2148536
</td>
<td style="text-align:left;">
Vmn1r-ps10
</td>
<td style="text-align:left;">
NCBI_Gene:113874,ENSEMBL:ENSMUSG00000093621
</td>
<td style="text-align:left;">
MGI:2148536
</td>
<td style="text-align:left;">
pseudogene
</td>
<td style="text-align:left;">
vomeronasal 1 receptor, pseudogene 10
</td>
</tr>
<tr>
<td style="text-align:left;">
6
</td>
<td style="text-align:left;">
gene
</td>
<td style="text-align:right;">
57.31805
</td>
<td style="text-align:right;">
57.32949
</td>
<td style="text-align:left;">
negative
</td>
<td style="text-align:left;">
MGI_C57BL6J_2159465
</td>
<td style="text-align:left;">
Vmn1r16
</td>
<td style="text-align:left;">
NCBI_Gene:171202,ENSEMBL:ENSMUSG00000115792
</td>
<td style="text-align:left;">
MGI:2159465
</td>
<td style="text-align:left;">
protein coding gene
</td>
<td style="text-align:left;">
vomeronasal 1 receptor 16
</td>
</tr>
<tr>
<td style="text-align:left;">
6
</td>
<td style="text-align:left;">
gene
</td>
<td style="text-align:right;">
57.35924
</td>
<td style="text-align:right;">
57.36373
</td>
<td style="text-align:left;">
negative
</td>
<td style="text-align:left;">
MGI_C57BL6J_2159452
</td>
<td style="text-align:left;">
Vmn1r17
</td>
<td style="text-align:left;">
NCBI_Gene:171189,ENSEMBL:ENSMUSG00000115644
</td>
<td style="text-align:left;">
MGI:2159452
</td>
<td style="text-align:left;">
protein coding gene
</td>
<td style="text-align:left;">
vomeronasal 1 receptor 17
</td>
</tr>
<tr>
<td style="text-align:left;">
6
</td>
<td style="text-align:left;">
gene
</td>
<td style="text-align:right;">
57.37666
</td>
<td style="text-align:right;">
57.39578
</td>
<td style="text-align:left;">
negative
</td>
<td style="text-align:left;">
MGI_C57BL6J_2159462
</td>
<td style="text-align:left;">
Vmn1r18
</td>
<td style="text-align:left;">
NCBI_Gene:171199,ENSEMBL:ENSMUSG00000091382
</td>
<td style="text-align:left;">
MGI:2159462
</td>
<td style="text-align:left;">
protein coding gene
</td>
<td style="text-align:left;">
vomeronasal 1 receptor 18
</td>
</tr>
<tr>
<td style="text-align:left;">
6
</td>
<td style="text-align:left;">
gene
</td>
<td style="text-align:right;">
57.40378
</td>
<td style="text-align:right;">
57.40603
</td>
<td style="text-align:left;">
positive
</td>
<td style="text-align:left;">
MGI_C57BL6J_2159463
</td>
<td style="text-align:left;">
Vmn1r19
</td>
<td style="text-align:left;">
NCBI_Gene:171200,ENSEMBL:ENSMUSG00000115799
</td>
<td style="text-align:left;">
MGI:2159463
</td>
<td style="text-align:left;">
protein coding gene
</td>
<td style="text-align:left;">
vomeronasal 1 receptor 19
</td>
</tr>
<tr>
<td style="text-align:left;">
6
</td>
<td style="text-align:left;">
gene
</td>
<td style="text-align:right;">
57.42893
</td>
<td style="text-align:right;">
57.43526
</td>
<td style="text-align:left;">
positive
</td>
<td style="text-align:left;">
MGI_C57BL6J_3644381
</td>
<td style="text-align:left;">
Vmn1r20
</td>
<td style="text-align:left;">
NCBI_Gene:434017,ENSEMBL:ENSMUSG00000115253
</td>
<td style="text-align:left;">
MGI:3644381
</td>
<td style="text-align:left;">
protein coding gene
</td>
<td style="text-align:left;">
vomeronasal 1 receptor 20
</td>
</tr>
<tr>
<td style="text-align:left;">
6
</td>
<td style="text-align:left;">
pseudogene
</td>
<td style="text-align:right;">
57.44736
</td>
<td style="text-align:right;">
57.44833
</td>
<td style="text-align:left;">
negative
</td>
<td style="text-align:left;">
MGI_C57BL6J_3643544
</td>
<td style="text-align:left;">
Vmn1r-ps11
</td>
<td style="text-align:left;">
NCBI_Gene:666822,ENSEMBL:ENSMUSG00000093398
</td>
<td style="text-align:left;">
MGI:3643544
</td>
<td style="text-align:left;">
pseudogene
</td>
<td style="text-align:left;">
vomeronasal 1 receptor, pseudogene 11
</td>
</tr>
<tr>
<td style="text-align:left;">
6
</td>
<td style="text-align:left;">
gene
</td>
<td style="text-align:right;">
57.46407
</td>
<td style="text-align:right;">
57.49953
</td>
<td style="text-align:left;">
positive
</td>
<td style="text-align:left;">
MGI_C57BL6J_3612486
</td>
<td style="text-align:left;">
4930533I22Rik
</td>
<td style="text-align:left;">
NCBI_Gene:434018,ENSEMBL:ENSMUSG00000093535
</td>
<td style="text-align:left;">
MGI:3612486
</td>
<td style="text-align:left;">
lncRNA gene
</td>
<td style="text-align:left;">
RIKEN cDNA 4930533I22 gene
</td>
</tr>
<tr>
<td style="text-align:left;">
6
</td>
<td style="text-align:left;">
gene
</td>
<td style="text-align:right;">
57.48512
</td>
<td style="text-align:right;">
57.62603
</td>
<td style="text-align:left;">
positive
</td>
<td style="text-align:left;">
MGI_C57BL6J_5753769
</td>
<td style="text-align:left;">
Gm45193
</td>
<td style="text-align:left;">
ENSEMBL:ENSMUSG00000108112
</td>
<td style="text-align:left;">
MGI:5753769
</td>
<td style="text-align:left;">
lncRNA gene
</td>
<td style="text-align:left;">
predicted gene 45193
</td>
</tr>
<tr>
<td style="text-align:left;">
6
</td>
<td style="text-align:left;">
gene
</td>
<td style="text-align:right;">
57.50650
</td>
<td style="text-align:right;">
57.53553
</td>
<td style="text-align:left;">
negative
</td>
<td style="text-align:left;">
MGI_C57BL6J_2442111
</td>
<td style="text-align:left;">
Ppm1k
</td>
<td style="text-align:left;">
NCBI_Gene:243382,ENSEMBL:ENSMUSG00000037826
</td>
<td style="text-align:left;">
MGI:2442111
</td>
<td style="text-align:left;">
protein coding gene
</td>
<td style="text-align:left;">
protein phosphatase 1K (PP2C domain containing)
</td>
</tr>
<tr>
<td style="text-align:left;">
6
</td>
<td style="text-align:left;">
gene
</td>
<td style="text-align:right;">
57.56579
</td>
<td style="text-align:right;">
57.56611
</td>
<td style="text-align:left;">
positive
</td>
<td style="text-align:left;">
MGI_C57BL6J_5531278
</td>
<td style="text-align:left;">
Gm27896
</td>
<td style="text-align:left;">
ENSEMBL:ENSMUSG00000092903
</td>
<td style="text-align:left;">
MGI:5531278
</td>
<td style="text-align:left;">
unclassified non-coding RNA gene
</td>
<td style="text-align:left;">
predicted gene, 27896
</td>
</tr>
<tr>
<td style="text-align:left;">
6
</td>
<td style="text-align:left;">
gene
</td>
<td style="text-align:right;">
57.58099
</td>
<td style="text-align:right;">
57.66514
</td>
<td style="text-align:left;">
positive
</td>
<td style="text-align:left;">
MGI_C57BL6J_1914388
</td>
<td style="text-align:left;">
Herc6
</td>
<td style="text-align:left;">
NCBI_Gene:67138,ENSEMBL:ENSMUSG00000029798
</td>
<td style="text-align:left;">
MGI:1914388
</td>
<td style="text-align:left;">
protein coding gene
</td>
<td style="text-align:left;">
hect domain and RLD 6
</td>
</tr>
<tr>
<td style="text-align:left;">
6
</td>
<td style="text-align:left;">
pseudogene
</td>
<td style="text-align:right;">
57.61697
</td>
<td style="text-align:right;">
57.61731
</td>
<td style="text-align:left;">
negative
</td>
<td style="text-align:left;">
MGI_C57BL6J_5011429
</td>
<td style="text-align:left;">
Gm19244
</td>
<td style="text-align:left;">
NCBI_Gene:100462737,ENSEMBL:ENSMUSG00000107627
</td>
<td style="text-align:left;">
MGI:5011429
</td>
<td style="text-align:left;">
pseudogene
</td>
<td style="text-align:left;">
predicted gene, 19244
</td>
</tr>
<tr>
<td style="text-align:left;">
6
</td>
<td style="text-align:left;">
gene
</td>
<td style="text-align:right;">
57.68474
</td>
<td style="text-align:right;">
57.68758
</td>
<td style="text-align:left;">
negative
</td>
<td style="text-align:left;">
MGI_C57BL6J_5610625
</td>
<td style="text-align:left;">
Gm37397
</td>
<td style="text-align:left;">
ENSEMBL:ENSMUSG00000104204
</td>
<td style="text-align:left;">
MGI:5610625
</td>
<td style="text-align:left;">
unclassified gene
</td>
<td style="text-align:left;">
predicted gene, 37397
</td>
</tr>
<tr>
<td style="text-align:left;">
6
</td>
<td style="text-align:left;">
gene
</td>
<td style="text-align:right;">
57.68474
</td>
<td style="text-align:right;">
57.69208
</td>
<td style="text-align:left;">
negative
</td>
<td style="text-align:left;">
MGI_C57BL6J_1913709
</td>
<td style="text-align:left;">
Pyurf
</td>
<td style="text-align:left;">
NCBI_Gene:66459,ENSEMBL:ENSMUSG00000043162
</td>
<td style="text-align:left;">
MGI:1913709
</td>
<td style="text-align:left;">
protein coding gene
</td>
<td style="text-align:left;">
Pigy upstream reading frame
</td>
</tr>
<tr>
<td style="text-align:left;">
6
</td>
<td style="text-align:left;">
pseudogene
</td>
<td style="text-align:right;">
57.68943
</td>
<td style="text-align:right;">
57.68965
</td>
<td style="text-align:left;">
negative
</td>
<td style="text-align:left;">
MGI_C57BL6J_5477206
</td>
<td style="text-align:left;">
Gm26712
</td>
<td style="text-align:left;">
ENSEMBL:ENSMUSG00000097656
</td>
<td style="text-align:left;">
MGI:5477206
</td>
<td style="text-align:left;">
pseudogene
</td>
<td style="text-align:left;">
predicted gene, 26712
</td>
</tr>
<tr>
<td style="text-align:left;">
6
</td>
<td style="text-align:left;">
gene
</td>
<td style="text-align:right;">
57.70246
</td>
<td style="text-align:right;">
57.73945
</td>
<td style="text-align:left;">
positive
</td>
<td style="text-align:left;">
MGI_C57BL6J_1919085
</td>
<td style="text-align:left;">
Lancl2
</td>
<td style="text-align:left;">
NCBI_Gene:71835,ENSEMBL:ENSMUSG00000062190
</td>
<td style="text-align:left;">
MGI:1919085
</td>
<td style="text-align:left;">
protein coding gene
</td>
<td style="text-align:left;">
LanC (bacterial lantibiotic synthetase component C)-like 2
</td>
</tr>
<tr>
<td style="text-align:left;">
6
</td>
<td style="text-align:left;">
gene
</td>
<td style="text-align:right;">
57.72781
</td>
<td style="text-align:right;">
57.82537
</td>
<td style="text-align:left;">
negative
</td>
<td style="text-align:left;">
MGI_C57BL6J_2141658
</td>
<td style="text-align:left;">
Vopp1
</td>
<td style="text-align:left;">
NCBI_Gene:232023,ENSEMBL:ENSMUSG00000037788
</td>
<td style="text-align:left;">
MGI:2141658
</td>
<td style="text-align:left;">
protein coding gene
</td>
<td style="text-align:left;">
vesicular, overexpressed in cancer, prosurvival protein 1
</td>
</tr>
<tr>
<td style="text-align:left;">
6
</td>
<td style="text-align:left;">
gene
</td>
<td style="text-align:right;">
57.84283
</td>
<td style="text-align:right;">
57.85034
</td>
<td style="text-align:left;">
negative
</td>
<td style="text-align:left;">
MGI_C57BL6J_2159464
</td>
<td style="text-align:left;">
Vmn1r21
</td>
<td style="text-align:left;">
NCBI_Gene:171201,ENSEMBL:ENSMUSG00000115343
</td>
<td style="text-align:left;">
MGI:2159464
</td>
<td style="text-align:left;">
protein coding gene
</td>
<td style="text-align:left;">
vomeronasal 1 receptor 21
</td>
</tr>
<tr>
<td style="text-align:left;">
6
</td>
<td style="text-align:left;">
gene
</td>
<td style="text-align:right;">
57.84559
</td>
<td style="text-align:right;">
57.84831
</td>
<td style="text-align:left;">
positive
</td>
<td style="text-align:left;">
MGI_C57BL6J_5690506
</td>
<td style="text-align:left;">
Gm44114
</td>
<td style="text-align:left;">
ENSEMBL:ENSMUSG00000107431
</td>
<td style="text-align:left;">
MGI:5690506
</td>
<td style="text-align:left;">
unclassified gene
</td>
<td style="text-align:left;">
predicted gene, 44114
</td>
</tr>
<tr>
<td style="text-align:left;">
6
</td>
<td style="text-align:left;">
gene
</td>
<td style="text-align:right;">
57.85982
</td>
<td style="text-align:right;">
57.85995
</td>
<td style="text-align:left;">
negative
</td>
<td style="text-align:left;">
MGI_C57BL6J_5455503
</td>
<td style="text-align:left;">
Gm25726
</td>
<td style="text-align:left;">
ENSEMBL:ENSMUSG00000077717
</td>
<td style="text-align:left;">
MGI:5455503
</td>
<td style="text-align:left;">
snoRNA gene
</td>
<td style="text-align:left;">
predicted gene, 25726
</td>
</tr>
<tr>
<td style="text-align:left;">
6
</td>
<td style="text-align:left;">
pseudogene
</td>
<td style="text-align:right;">
57.87241
</td>
<td style="text-align:right;">
57.87292
</td>
<td style="text-align:left;">
negative
</td>
<td style="text-align:left;">
MGI_C57BL6J_3643065
</td>
<td style="text-align:left;">
Vmn1r-ps12
</td>
<td style="text-align:left;">
NCBI_Gene:626582,ENSEMBL:ENSMUSG00000093530
</td>
<td style="text-align:left;">
MGI:3643065
</td>
<td style="text-align:left;">
pseudogene
</td>
<td style="text-align:left;">
vomeronasal 1 receptor, pseudogene 12
</td>
</tr>
<tr>
<td style="text-align:left;">
6
</td>
<td style="text-align:left;">
gene
</td>
<td style="text-align:right;">
57.89813
</td>
<td style="text-align:right;">
57.90803
</td>
<td style="text-align:left;">
negative
</td>
<td style="text-align:left;">
MGI_C57BL6J_2159459
</td>
<td style="text-align:left;">
Vmn1r22
</td>
<td style="text-align:left;">
NCBI_Gene:171196,ENSEMBL:ENSMUSG00000115091
</td>
<td style="text-align:left;">
MGI:2159459
</td>
<td style="text-align:left;">
protein coding gene
</td>
<td style="text-align:left;">
vomeronasal 1 receptor 22
</td>
</tr>
<tr>
<td style="text-align:left;">
6
</td>
<td style="text-align:left;">
gene
</td>
<td style="text-align:right;">
57.92584
</td>
<td style="text-align:right;">
57.92684
</td>
<td style="text-align:left;">
negative
</td>
<td style="text-align:left;">
MGI_C57BL6J_2159460
</td>
<td style="text-align:left;">
Vmn1r23
</td>
<td style="text-align:left;">
NCBI_Gene:171197,ENSEMBL:ENSMUSG00000093376
</td>
<td style="text-align:left;">
MGI:2159460
</td>
<td style="text-align:left;">
protein coding gene
</td>
<td style="text-align:left;">
vomeronasal 1 receptor 23
</td>
</tr>
<tr>
<td style="text-align:left;">
6
</td>
<td style="text-align:left;">
pseudogene
</td>
<td style="text-align:right;">
57.94968
</td>
<td style="text-align:right;">
57.94997
</td>
<td style="text-align:left;">
negative
</td>
<td style="text-align:left;">
MGI_C57BL6J_5313107
</td>
<td style="text-align:left;">
Gm20660
</td>
<td style="text-align:left;">
ENSEMBL:ENSMUSG00000093541
</td>
<td style="text-align:left;">
MGI:5313107
</td>
<td style="text-align:left;">
pseudogene
</td>
<td style="text-align:left;">
predicted gene 20660
</td>
</tr>
<tr>
<td style="text-align:left;">
6
</td>
<td style="text-align:left;">
pseudogene
</td>
<td style="text-align:right;">
57.95007
</td>
<td style="text-align:right;">
57.95047
</td>
<td style="text-align:left;">
positive
</td>
<td style="text-align:left;">
MGI_C57BL6J_3781560
</td>
<td style="text-align:left;">
Gm3382
</td>
<td style="text-align:left;">
NCBI_Gene:100041514
</td>
<td style="text-align:left;">
MGI:3781560
</td>
<td style="text-align:left;">
pseudogene
</td>
<td style="text-align:left;">
predicted gene 3382
</td>
</tr>
<tr>
<td style="text-align:left;">
6
</td>
<td style="text-align:left;">
gene
</td>
<td style="text-align:right;">
57.95471
</td>
<td style="text-align:right;">
57.96150
</td>
<td style="text-align:left;">
negative
</td>
<td style="text-align:left;">
MGI_C57BL6J_2159454
</td>
<td style="text-align:left;">
Vmn1r24
</td>
<td style="text-align:left;">
NCBI_Gene:171191,ENSEMBL:ENSMUSG00000115507
</td>
<td style="text-align:left;">
MGI:2159454
</td>
<td style="text-align:left;">
protein coding gene
</td>
<td style="text-align:left;">
vomeronasal 1 receptor 24
</td>
</tr>
<tr>
<td style="text-align:left;">
6
</td>
<td style="text-align:left;">
gene
</td>
<td style="text-align:right;">
57.97830
</td>
<td style="text-align:right;">
57.98081
</td>
<td style="text-align:left;">
negative
</td>
<td style="text-align:left;">
MGI_C57BL6J_2148529
</td>
<td style="text-align:left;">
Vmn1r25
</td>
<td style="text-align:left;">
NCBI_Gene:113865,ENSEMBL:ENSMUSG00000115668
</td>
<td style="text-align:left;">
MGI:2148529
</td>
<td style="text-align:left;">
protein coding gene
</td>
<td style="text-align:left;">
vomeronasal 1 receptor 25
</td>
</tr>
<tr>
<td style="text-align:left;">
6
</td>
<td style="text-align:left;">
pseudogene
</td>
<td style="text-align:right;">
57.99299
</td>
<td style="text-align:right;">
57.99358
</td>
<td style="text-align:left;">
negative
</td>
<td style="text-align:left;">
MGI_C57BL6J_3852363
</td>
<td style="text-align:left;">
Vmn1r-ps13
</td>
<td style="text-align:left;">
NCBI_Gene:100312486,ENSEMBL:ENSMUSG00000107678
</td>
<td style="text-align:left;">
MGI:3852363
</td>
<td style="text-align:left;">
pseudogene
</td>
<td style="text-align:left;">
vomeronasal 1 receptor, pseudogene 13
</td>
</tr>
<tr>
<td style="text-align:left;">
6
</td>
<td style="text-align:left;">
gene
</td>
<td style="text-align:right;">
58.00493
</td>
<td style="text-align:right;">
58.01516
</td>
<td style="text-align:left;">
negative
</td>
<td style="text-align:left;">
MGI_C57BL6J_2159453
</td>
<td style="text-align:left;">
Vmn1r26
</td>
<td style="text-align:left;">
NCBI_Gene:171190,ENSEMBL:ENSMUSG00000048697
</td>
<td style="text-align:left;">
MGI:2159453
</td>
<td style="text-align:left;">
protein coding gene
</td>
<td style="text-align:left;">
vomeronasal 1 receptor 26
</td>
</tr>
<tr>
<td style="text-align:left;">
6
</td>
<td style="text-align:left;">
pseudogene
</td>
<td style="text-align:right;">
58.01748
</td>
<td style="text-align:right;">
58.01794
</td>
<td style="text-align:left;">
positive
</td>
<td style="text-align:left;">
MGI_C57BL6J_3647234
</td>
<td style="text-align:left;">
Rps15-ps2
</td>
<td style="text-align:left;">
NCBI_Gene:633683,ENSEMBL:ENSMUSG00000071419
</td>
<td style="text-align:left;">
MGI:3647234
</td>
<td style="text-align:left;">
pseudogene
</td>
<td style="text-align:left;">
ribosomal protein S15, pseudogene 2
</td>
</tr>
<tr>
<td style="text-align:left;">
6
</td>
<td style="text-align:left;">
pseudogene
</td>
<td style="text-align:right;">
58.04645
</td>
<td style="text-align:right;">
58.04990
</td>
<td style="text-align:left;">
negative
</td>
<td style="text-align:left;">
MGI_C57BL6J_3645438
</td>
<td style="text-align:left;">
Vmn1r-ps14
</td>
<td style="text-align:left;">
NCBI_Gene:626649,ENSEMBL:ENSMUSG00000093382
</td>
<td style="text-align:left;">
MGI:3645438
</td>
<td style="text-align:left;">
pseudogene
</td>
<td style="text-align:left;">
vomeronasal 1 receptor, pseudogene 14
</td>
</tr>
<tr>
<td style="text-align:left;">
6
</td>
<td style="text-align:left;">
pseudogene
</td>
<td style="text-align:right;">
58.07360
</td>
<td style="text-align:right;">
58.07913
</td>
<td style="text-align:left;">
negative
</td>
<td style="text-align:left;">
MGI_C57BL6J_3648676
</td>
<td style="text-align:left;">
Vmn1r-ps15
</td>
<td style="text-align:left;">
NCBI_Gene:626659,ENSEMBL:ENSMUSG00000093666
</td>
<td style="text-align:left;">
MGI:3648676
</td>
<td style="text-align:left;">
pseudogene
</td>
<td style="text-align:left;">
vomeronasal 1 receptor, pseudogene 15
</td>
</tr>
<tr>
<td style="text-align:left;">
6
</td>
<td style="text-align:left;">
pseudogene
</td>
<td style="text-align:right;">
58.09108
</td>
<td style="text-align:right;">
58.09267
</td>
<td style="text-align:left;">
positive
</td>
<td style="text-align:left;">
MGI_C57BL6J_3648885
</td>
<td style="text-align:left;">
Gm6695
</td>
<td style="text-align:left;">
NCBI_Gene:626672,ENSEMBL:ENSMUSG00000093560
</td>
<td style="text-align:left;">
MGI:3648885
</td>
<td style="text-align:left;">
pseudogene
</td>
<td style="text-align:left;">
predicted gene 6695
</td>
</tr>
<tr>
<td style="text-align:left;">
6
</td>
<td style="text-align:left;">
pseudogene
</td>
<td style="text-align:right;">
58.10802
</td>
<td style="text-align:right;">
58.10825
</td>
<td style="text-align:left;">
negative
</td>
<td style="text-align:left;">
MGI_C57BL6J_3852364
</td>
<td style="text-align:left;">
Vmn1r-ps16
</td>
<td style="text-align:left;">
NCBI_Gene:100312487,ENSEMBL:ENSMUSG00000107530
</td>
<td style="text-align:left;">
MGI:3852364
</td>
<td style="text-align:left;">
pseudogene
</td>
<td style="text-align:left;">
vomeronasal 1 receptor, pseudogene 16
</td>
</tr>
<tr>
<td style="text-align:left;">
6
</td>
<td style="text-align:left;">
pseudogene
</td>
<td style="text-align:right;">
58.11573
</td>
<td style="text-align:right;">
58.11663
</td>
<td style="text-align:left;">
negative
</td>
<td style="text-align:left;">
MGI_C57BL6J_3643137
</td>
<td style="text-align:left;">
Vmn1r-ps17
</td>
<td style="text-align:left;">
NCBI_Gene:666893,ENSEMBL:ENSMUSG00000093743
</td>
<td style="text-align:left;">
MGI:3643137
</td>
<td style="text-align:left;">
pseudogene
</td>
<td style="text-align:left;">
vomeronasal 1 receptor, pseudogene 17
</td>
</tr>
<tr>
<td style="text-align:left;">
6
</td>
<td style="text-align:left;">
pseudogene
</td>
<td style="text-align:right;">
58.13589
</td>
<td style="text-align:right;">
58.13614
</td>
<td style="text-align:left;">
positive
</td>
<td style="text-align:left;">
MGI_C57BL6J_5313165
</td>
<td style="text-align:left;">
Gm20718
</td>
<td style="text-align:left;">
ENSEMBL:ENSMUSG00000093708
</td>
<td style="text-align:left;">
MGI:5313165
</td>
<td style="text-align:left;">
pseudogene
</td>
<td style="text-align:left;">
predicted gene 20718
</td>
</tr>
<tr>
<td style="text-align:left;">
6
</td>
<td style="text-align:left;">
gene
</td>
<td style="text-align:right;">
58.19232
</td>
<td style="text-align:right;">
58.19476
</td>
<td style="text-align:left;">
positive
</td>
<td style="text-align:left;">
MGI_C57BL6J_5690386
</td>
<td style="text-align:left;">
Gm43994
</td>
<td style="text-align:left;">
ENSEMBL:ENSMUSG00000107521
</td>
<td style="text-align:left;">
MGI:5690386
</td>
<td style="text-align:left;">
lncRNA gene
</td>
<td style="text-align:left;">
predicted gene, 43994
</td>
</tr>
<tr>
<td style="text-align:left;">
6
</td>
<td style="text-align:left;">
pseudogene
</td>
<td style="text-align:right;">
58.20569
</td>
<td style="text-align:right;">
58.20700
</td>
<td style="text-align:left;">
positive
</td>
<td style="text-align:left;">
MGI_C57BL6J_3781590
</td>
<td style="text-align:left;">
Vmn1r-ps18
</td>
<td style="text-align:left;">
NCBI_Gene:100041571,ENSEMBL:ENSMUSG00000093731
</td>
<td style="text-align:left;">
MGI:3781590
</td>
<td style="text-align:left;">
pseudogene
</td>
<td style="text-align:left;">
vomeronasal 1 receptor, pseudogene 18
</td>
</tr>
<tr>
<td style="text-align:left;">
6
</td>
<td style="text-align:left;">
gene
</td>
<td style="text-align:right;">
58.21309
</td>
<td style="text-align:right;">
58.21882
</td>
<td style="text-align:left;">
negative
</td>
<td style="text-align:left;">
MGI_C57BL6J_2159470
</td>
<td style="text-align:left;">
Vmn1r27
</td>
<td style="text-align:left;">
NCBI_Gene:171206,ENSEMBL:ENSMUSG00000071428
</td>
<td style="text-align:left;">
MGI:2159470
</td>
<td style="text-align:left;">
protein coding gene
</td>
<td style="text-align:left;">
vomeronasal 1 receptor 27
</td>
</tr>
<tr>
<td style="text-align:left;">
6
</td>
<td style="text-align:left;">
pseudogene
</td>
<td style="text-align:right;">
58.25662
</td>
<td style="text-align:right;">
58.25755
</td>
<td style="text-align:left;">
negative
</td>
<td style="text-align:left;">
MGI_C57BL6J_3647074
</td>
<td style="text-align:left;">
Vmn1r-ps19
</td>
<td style="text-align:left;">
NCBI_Gene:626724,ENSEMBL:ENSMUSG00000093522
</td>
<td style="text-align:left;">
MGI:3647074
</td>
<td style="text-align:left;">
pseudogene
</td>
<td style="text-align:left;">
vomeronasal 1 receptor, pseudogene 19
</td>
</tr>
<tr>
<td style="text-align:left;">
6
</td>
<td style="text-align:left;">
gene
</td>
<td style="text-align:right;">
58.26204
</td>
<td style="text-align:right;">
58.27564
</td>
<td style="text-align:left;">
positive
</td>
<td style="text-align:left;">
MGI_C57BL6J_2159461
</td>
<td style="text-align:left;">
Vmn1r28
</td>
<td style="text-align:left;">
NCBI_Gene:171198,ENSEMBL:ENSMUSG00000115705
</td>
<td style="text-align:left;">
MGI:2159461
</td>
<td style="text-align:left;">
protein coding gene
</td>
<td style="text-align:left;">
vomeronasal 1 receptor 28
</td>
</tr>
<tr>
<td style="text-align:left;">
6
</td>
<td style="text-align:left;">
gene
</td>
<td style="text-align:right;">
58.30155
</td>
<td style="text-align:right;">
58.31168
</td>
<td style="text-align:left;">
positive
</td>
<td style="text-align:left;">
MGI_C57BL6J_2148523
</td>
<td style="text-align:left;">
Vmn1r29
</td>
<td style="text-align:left;">
NCBI_Gene:113859,ENSEMBL:ENSMUSG00000091734
</td>
<td style="text-align:left;">
MGI:2148523
</td>
<td style="text-align:left;">
protein coding gene
</td>
<td style="text-align:left;">
vomeronasal 1 receptor 29
</td>
</tr>
<tr>
<td style="text-align:left;">
6
</td>
<td style="text-align:left;">
pseudogene
</td>
<td style="text-align:right;">
58.39405
</td>
<td style="text-align:right;">
58.39442
</td>
<td style="text-align:left;">
positive
</td>
<td style="text-align:left;">
MGI_C57BL6J_5313121
</td>
<td style="text-align:left;">
Gm20674
</td>
<td style="text-align:left;">
ENSEMBL:ENSMUSG00000093547
</td>
<td style="text-align:left;">
MGI:5313121
</td>
<td style="text-align:left;">
pseudogene
</td>
<td style="text-align:left;">
predicted gene 20674
</td>
</tr>
<tr>
<td style="text-align:left;">
6
</td>
<td style="text-align:left;">
pseudogene
</td>
<td style="text-align:right;">
58.40641
</td>
<td style="text-align:right;">
58.40736
</td>
<td style="text-align:left;">
negative
</td>
<td style="text-align:left;">
MGI_C57BL6J_3645210
</td>
<td style="text-align:left;">
Vmn1r-ps20
</td>
<td style="text-align:left;">
NCBI_Gene:666928,ENSEMBL:ENSMUSG00000093702
</td>
<td style="text-align:left;">
MGI:3645210
</td>
<td style="text-align:left;">
pseudogene
</td>
<td style="text-align:left;">
vomeronasal 1 receptor, pseudogene 20
</td>
</tr>
<tr>
<td style="text-align:left;">
6
</td>
<td style="text-align:left;">
gene
</td>
<td style="text-align:right;">
58.43416
</td>
<td style="text-align:right;">
58.44362
</td>
<td style="text-align:left;">
negative
</td>
<td style="text-align:left;">
MGI_C57BL6J_2159458
</td>
<td style="text-align:left;">
Vmn1r30
</td>
<td style="text-align:left;">
NCBI_Gene:171195,ENSEMBL:ENSMUSG00000095670
</td>
<td style="text-align:left;">
MGI:2159458
</td>
<td style="text-align:left;">
protein coding gene
</td>
<td style="text-align:left;">
vomeronasal 1 receptor 30
</td>
</tr>
<tr>
<td style="text-align:left;">
6
</td>
<td style="text-align:left;">
pseudogene
</td>
<td style="text-align:right;">
58.43870
</td>
<td style="text-align:right;">
58.43886
</td>
<td style="text-align:left;">
negative
</td>
<td style="text-align:left;">
MGI_C57BL6J_5690408
</td>
<td style="text-align:left;">
Gm44016
</td>
<td style="text-align:left;">
ENSEMBL:ENSMUSG00000107518
</td>
<td style="text-align:left;">
MGI:5690408
</td>
<td style="text-align:left;">
pseudogene
</td>
<td style="text-align:left;">
predicted gene, 44016
</td>
</tr>
<tr>
<td style="text-align:left;">
6
</td>
<td style="text-align:left;">
pseudogene
</td>
<td style="text-align:right;">
58.45507
</td>
<td style="text-align:right;">
58.45599
</td>
<td style="text-align:left;">
negative
</td>
<td style="text-align:left;">
MGI_C57BL6J_3852365
</td>
<td style="text-align:left;">
Vmn1r-ps21
</td>
<td style="text-align:left;">
NCBI_Gene:100312488,ENSEMBL:ENSMUSG00000093461
</td>
<td style="text-align:left;">
MGI:3852365
</td>
<td style="text-align:left;">
pseudogene
</td>
<td style="text-align:left;">
vomeronasal 1 receptor, pseudogene 21
</td>
</tr>
<tr>
<td style="text-align:left;">
6
</td>
<td style="text-align:left;">
gene
</td>
<td style="text-align:right;">
58.45787
</td>
<td style="text-align:right;">
58.53369
</td>
<td style="text-align:left;">
negative
</td>
<td style="text-align:left;">
MGI_C57BL6J_5594236
</td>
<td style="text-align:left;">
Gm35077
</td>
<td style="text-align:left;">
NCBI_Gene:102638540,ENSEMBL:ENSMUSG00000107960
</td>
<td style="text-align:left;">
MGI:5594236
</td>
<td style="text-align:left;">
lncRNA gene
</td>
<td style="text-align:left;">
predicted gene, 35077
</td>
</tr>
<tr>
<td style="text-align:left;">
6
</td>
<td style="text-align:left;">
pseudogene
</td>
<td style="text-align:right;">
58.46111
</td>
<td style="text-align:right;">
58.46154
</td>
<td style="text-align:left;">
positive
</td>
<td style="text-align:left;">
MGI_C57BL6J_5313120
</td>
<td style="text-align:left;">
Gm20673
</td>
<td style="text-align:left;">
ENSEMBL:ENSMUSG00000093747
</td>
<td style="text-align:left;">
MGI:5313120
</td>
<td style="text-align:left;">
pseudogene
</td>
<td style="text-align:left;">
predicted gene 20673
</td>
</tr>
<tr>
<td style="text-align:left;">
6
</td>
<td style="text-align:left;">
gene
</td>
<td style="text-align:right;">
58.47094
</td>
<td style="text-align:right;">
58.47533
</td>
<td style="text-align:left;">
negative
</td>
<td style="text-align:left;">
MGI_C57BL6J_3649162
</td>
<td style="text-align:left;">
Vmn1r31
</td>
<td style="text-align:left;">
NCBI_Gene:626828,ENSEMBL:ENSMUSG00000115404
</td>
<td style="text-align:left;">
MGI:3649162
</td>
<td style="text-align:left;">
protein coding gene
</td>
<td style="text-align:left;">
vomeronasal 1 receptor 31
</td>
</tr>
<tr>
<td style="text-align:left;">
6
</td>
<td style="text-align:left;">
pseudogene
</td>
<td style="text-align:right;">
58.49434
</td>
<td style="text-align:right;">
58.49525
</td>
<td style="text-align:left;">
negative
</td>
<td style="text-align:left;">
MGI_C57BL6J_3647778
</td>
<td style="text-align:left;">
Vmn1r-ps22
</td>
<td style="text-align:left;">
NCBI_Gene:666954,ENSEMBL:ENSMUSG00000093518
</td>
<td style="text-align:left;">
MGI:3647778
</td>
<td style="text-align:left;">
pseudogene
</td>
<td style="text-align:left;">
vomeronasal 1 receptor, pseudogene 22
</td>
</tr>
<tr>
<td style="text-align:left;">
6
</td>
<td style="text-align:left;">
pseudogene
</td>
<td style="text-align:right;">
58.50444
</td>
<td style="text-align:right;">
58.50474
</td>
<td style="text-align:left;">
negative
</td>
<td style="text-align:left;">
MGI_C57BL6J_3852366
</td>
<td style="text-align:left;">
Vmn1r-ps23
</td>
<td style="text-align:left;">
NCBI_Gene:100312489,ENSEMBL:ENSMUSG00000093588
</td>
<td style="text-align:left;">
MGI:3852366
</td>
<td style="text-align:left;">
pseudogene
</td>
<td style="text-align:left;">
vomeronasal 1 receptor, pseudogene 23
</td>
</tr>
<tr>
<td style="text-align:left;">
6
</td>
<td style="text-align:left;">
pseudogene
</td>
<td style="text-align:right;">
58.51227
</td>
<td style="text-align:right;">
58.51253
</td>
<td style="text-align:left;">
positive
</td>
<td style="text-align:left;">
MGI_C57BL6J_5313119
</td>
<td style="text-align:left;">
Gm20672
</td>
<td style="text-align:left;">
ENSEMBL:ENSMUSG00000093741
</td>
<td style="text-align:left;">
MGI:5313119
</td>
<td style="text-align:left;">
pseudogene
</td>
<td style="text-align:left;">
predicted gene 20672
</td>
</tr>
<tr>
<td style="text-align:left;">
6
</td>
<td style="text-align:left;">
gene
</td>
<td style="text-align:right;">
58.53397
</td>
<td style="text-align:right;">
58.58425
</td>
<td style="text-align:left;">
negative
</td>
<td style="text-align:left;">
MGI_C57BL6J_5825580
</td>
<td style="text-align:left;">
Gm45943
</td>
<td style="text-align:left;">
NCBI_Gene:108167332
</td>
<td style="text-align:left;">
MGI:5825580
</td>
<td style="text-align:left;">
protein coding gene
</td>
<td style="text-align:left;">
predicted gene, 45943
</td>
</tr>
<tr>
<td style="text-align:left;">
6
</td>
<td style="text-align:left;">
gene
</td>
<td style="text-align:right;">
58.58449
</td>
<td style="text-align:right;">
58.69568
</td>
<td style="text-align:left;">
positive
</td>
<td style="text-align:left;">
MGI_C57BL6J_1347061
</td>
<td style="text-align:left;">
Abcg2
</td>
<td style="text-align:left;">
NCBI_Gene:26357,ENSEMBL:ENSMUSG00000029802
</td>
<td style="text-align:left;">
MGI:1347061
</td>
<td style="text-align:left;">
protein coding gene
</td>
<td style="text-align:left;">
ATP binding cassette subfamily G member 2 (Junior blood group)
</td>
</tr>
<tr>
<td style="text-align:left;">
6
</td>
<td style="text-align:left;">
gene
</td>
<td style="text-align:right;">
58.58575
</td>
<td style="text-align:right;">
58.58656
</td>
<td style="text-align:left;">
positive
</td>
<td style="text-align:left;">
MGI_C57BL6J_1921131
</td>
<td style="text-align:left;">
4930430M16Rik
</td>
<td style="text-align:left;">
NA
</td>
<td style="text-align:left;">
NA
</td>
<td style="text-align:left;">
unclassified gene
</td>
<td style="text-align:left;">
RIKEN cDNA 4930430M16 gene
</td>
</tr>
<tr>
<td style="text-align:left;">
6
</td>
<td style="text-align:left;">
gene
</td>
<td style="text-align:right;">
58.67971
</td>
<td style="text-align:right;">
58.68064
</td>
<td style="text-align:left;">
negative
</td>
<td style="text-align:left;">
MGI_C57BL6J_1917453
</td>
<td style="text-align:left;">
2310061B05Rik
</td>
<td style="text-align:left;">
NA
</td>
<td style="text-align:left;">
NA
</td>
<td style="text-align:left;">
unclassified gene
</td>
<td style="text-align:left;">
RIKEN cDNA 2310061B05 gene
</td>
</tr>
<tr>
<td style="text-align:left;">
6
</td>
<td style="text-align:left;">
gene
</td>
<td style="text-align:right;">
58.75891
</td>
<td style="text-align:right;">
58.81375
</td>
<td style="text-align:left;">
negative
</td>
<td style="text-align:left;">
MGI_C57BL6J_5621707
</td>
<td style="text-align:left;">
Gm38822
</td>
<td style="text-align:left;">
NCBI_Gene:105242670
</td>
<td style="text-align:left;">
MGI:5621707
</td>
<td style="text-align:left;">
lncRNA gene
</td>
<td style="text-align:left;">
predicted gene, 38822
</td>
</tr>
<tr>
<td style="text-align:left;">
6
</td>
<td style="text-align:left;">
pseudogene
</td>
<td style="text-align:right;">
58.78169
</td>
<td style="text-align:right;">
58.78194
</td>
<td style="text-align:left;">
negative
</td>
<td style="text-align:left;">
MGI_C57BL6J_5690291
</td>
<td style="text-align:left;">
Gm43899
</td>
<td style="text-align:left;">
ENSEMBL:ENSMUSG00000108164
</td>
<td style="text-align:left;">
MGI:5690291
</td>
<td style="text-align:left;">
pseudogene
</td>
<td style="text-align:left;">
predicted gene, 43899
</td>
</tr>
<tr>
<td style="text-align:left;">
6
</td>
<td style="text-align:left;">
gene
</td>
<td style="text-align:right;">
58.83145
</td>
<td style="text-align:right;">
58.92040
</td>
<td style="text-align:left;">
positive
</td>
<td style="text-align:left;">
MGI_C57BL6J_1921248
</td>
<td style="text-align:left;">
Herc3
</td>
<td style="text-align:left;">
NCBI_Gene:73998,ENSEMBL:ENSMUSG00000029804
</td>
<td style="text-align:left;">
MGI:1921248
</td>
<td style="text-align:left;">
protein coding gene
</td>
<td style="text-align:left;">
hect domain and RLD 3
</td>
</tr>
<tr>
<td style="text-align:left;">
6
</td>
<td style="text-align:left;">
gene
</td>
<td style="text-align:right;">
58.90523
</td>
<td style="text-align:right;">
58.90713
</td>
<td style="text-align:left;">
negative
</td>
<td style="text-align:left;">
MGI_C57BL6J_1923555
</td>
<td style="text-align:left;">
Nap1l5
</td>
<td style="text-align:left;">
NCBI_Gene:58243,ENSEMBL:ENSMUSG00000055430
</td>
<td style="text-align:left;">
MGI:1923555
</td>
<td style="text-align:left;">
protein coding gene
</td>
<td style="text-align:left;">
nucleosome assembly protein 1-like 5
</td>
</tr>
<tr>
<td style="text-align:left;">
6
</td>
<td style="text-align:left;">
gene
</td>
<td style="text-align:right;">
58.93209
</td>
<td style="text-align:right;">
59.02455
</td>
<td style="text-align:left;">
negative
</td>
<td style="text-align:left;">
MGI_C57BL6J_1889842
</td>
<td style="text-align:left;">
Fam13a
</td>
<td style="text-align:left;">
NCBI_Gene:58909,ENSEMBL:ENSMUSG00000037709
</td>
<td style="text-align:left;">
MGI:1889842
</td>
<td style="text-align:left;">
protein coding gene
</td>
<td style="text-align:left;">
family with sequence similarity 13, member A
</td>
</tr>
<tr>
<td style="text-align:left;">
6
</td>
<td style="text-align:left;">
gene
</td>
<td style="text-align:right;">
58.96487
</td>
<td style="text-align:right;">
58.97223
</td>
<td style="text-align:left;">
positive
</td>
<td style="text-align:left;">
MGI_C57BL6J_5753636
</td>
<td style="text-align:left;">
Gm45060
</td>
<td style="text-align:left;">
ENSEMBL:ENSMUSG00000107439
</td>
<td style="text-align:left;">
MGI:5753636
</td>
<td style="text-align:left;">
lncRNA gene
</td>
<td style="text-align:left;">
predicted gene 45060
</td>
</tr>
<tr>
<td style="text-align:left;">
6
</td>
<td style="text-align:left;">
gene
</td>
<td style="text-align:right;">
58.96522
</td>
<td style="text-align:right;">
58.99231
</td>
<td style="text-align:left;">
positive
</td>
<td style="text-align:left;">
MGI_C57BL6J_1918734
</td>
<td style="text-align:left;">
8430415O14Rik
</td>
<td style="text-align:left;">
NCBI_Gene:71484
</td>
<td style="text-align:left;">
MGI:1918734
</td>
<td style="text-align:left;">
lncRNA gene
</td>
<td style="text-align:left;">
RIKEN cDNA 8430415O14 gene
</td>
</tr>
<tr>
<td style="text-align:left;">
6
</td>
<td style="text-align:left;">
gene
</td>
<td style="text-align:right;">
58.96565
</td>
<td style="text-align:right;">
59.00208
</td>
<td style="text-align:left;">
positive
</td>
<td style="text-align:left;">
MGI_C57BL6J_3034340
</td>
<td style="text-align:left;">
BB365896
</td>
<td style="text-align:left;">
ENSEMBL:ENSMUSG00000092323
</td>
<td style="text-align:left;">
MGI:3034340
</td>
<td style="text-align:left;">
lncRNA gene
</td>
<td style="text-align:left;">
expressed sequence BB365896
</td>
</tr>
<tr>
<td style="text-align:left;">
6
</td>
<td style="text-align:left;">
gene
</td>
<td style="text-align:right;">
58.97320
</td>
<td style="text-align:right;">
58.97529
</td>
<td style="text-align:left;">
positive
</td>
<td style="text-align:left;">
MGI_C57BL6J_5753637
</td>
<td style="text-align:left;">
Gm45061
</td>
<td style="text-align:left;">
ENSEMBL:ENSMUSG00000108075
</td>
<td style="text-align:left;">
MGI:5753637
</td>
<td style="text-align:left;">
unclassified gene
</td>
<td style="text-align:left;">
predicted gene 45061
</td>
</tr>
<tr>
<td style="text-align:left;">
6
</td>
<td style="text-align:left;">
pseudogene
</td>
<td style="text-align:right;">
59.03689
</td>
<td style="text-align:right;">
59.03938
</td>
<td style="text-align:left;">
positive
</td>
<td style="text-align:left;">
MGI_C57BL6J_5011350
</td>
<td style="text-align:left;">
Gm19165
</td>
<td style="text-align:left;">
NCBI_Gene:100418369,ENSEMBL:ENSMUSG00000107449
</td>
<td style="text-align:left;">
MGI:5011350
</td>
<td style="text-align:left;">
pseudogene
</td>
<td style="text-align:left;">
predicted gene, 19165
</td>
</tr>
<tr>
<td style="text-align:left;">
6
</td>
<td style="text-align:left;">
gene
</td>
<td style="text-align:right;">
59.09925
</td>
<td style="text-align:right;">
59.09938
</td>
<td style="text-align:left;">
positive
</td>
<td style="text-align:left;">
MGI_C57BL6J_5452144
</td>
<td style="text-align:left;">
Gm22367
</td>
<td style="text-align:left;">
ENSEMBL:ENSMUSG00000096371
</td>
<td style="text-align:left;">
MGI:5452144
</td>
<td style="text-align:left;">
snoRNA gene
</td>
<td style="text-align:left;">
predicted gene, 22367
</td>
</tr>
<tr>
<td style="text-align:left;">
6
</td>
<td style="text-align:left;">
pseudogene
</td>
<td style="text-align:right;">
59.19305
</td>
<td style="text-align:right;">
59.19325
</td>
<td style="text-align:left;">
negative
</td>
<td style="text-align:left;">
MGI_C57BL6J_5690297
</td>
<td style="text-align:left;">
Gm43905
</td>
<td style="text-align:left;">
ENSEMBL:ENSMUSG00000107987
</td>
<td style="text-align:left;">
MGI:5690297
</td>
<td style="text-align:left;">
pseudogene
</td>
<td style="text-align:left;">
predicted gene, 43905
</td>
</tr>
<tr>
<td style="text-align:left;">
6
</td>
<td style="text-align:left;">
gene
</td>
<td style="text-align:right;">
59.20887
</td>
<td style="text-align:right;">
59.21203
</td>
<td style="text-align:left;">
positive
</td>
<td style="text-align:left;">
MGI_C57BL6J_1915390
</td>
<td style="text-align:left;">
Tigd2
</td>
<td style="text-align:left;">
NCBI_Gene:68140,ENSEMBL:ENSMUSG00000049232
</td>
<td style="text-align:left;">
MGI:1915390
</td>
<td style="text-align:left;">
protein coding gene
</td>
<td style="text-align:left;">
tigger transposable element derived 2
</td>
</tr>
<tr>
<td style="text-align:left;">
6
</td>
<td style="text-align:left;">
gene
</td>
<td style="text-align:right;">
59.34450
</td>
<td style="text-align:right;">
59.42629
</td>
<td style="text-align:left;">
negative
</td>
<td style="text-align:left;">
MGI_C57BL6J_1924785
</td>
<td style="text-align:left;">
Gprin3
</td>
<td style="text-align:left;">
NCBI_Gene:243385,ENSEMBL:ENSMUSG00000045441
</td>
<td style="text-align:left;">
MGI:1924785
</td>
<td style="text-align:left;">
protein coding gene
</td>
<td style="text-align:left;">
GPRIN family member 3
</td>
</tr>
<tr>
<td style="text-align:left;">
6
</td>
<td style="text-align:left;">
gene
</td>
<td style="text-align:right;">
59.69817
</td>
<td style="text-align:right;">
59.70590
</td>
<td style="text-align:left;">
positive
</td>
<td style="text-align:left;">
MGI_C57BL6J_5594354
</td>
<td style="text-align:left;">
Gm35195
</td>
<td style="text-align:left;">
NCBI_Gene:102638695
</td>
<td style="text-align:left;">
MGI:5594354
</td>
<td style="text-align:left;">
lncRNA gene
</td>
<td style="text-align:left;">
predicted gene, 35195
</td>
</tr>
<tr>
<td style="text-align:left;">
6
</td>
<td style="text-align:left;">
gene
</td>
<td style="text-align:right;">
59.70575
</td>
<td style="text-align:right;">
59.70587
</td>
<td style="text-align:left;">
negative
</td>
<td style="text-align:left;">
MGI_C57BL6J_5452399
</td>
<td style="text-align:left;">
Gm22622
</td>
<td style="text-align:left;">
ENSEMBL:ENSMUSG00000094183
</td>
<td style="text-align:left;">
MGI:5452399
</td>
<td style="text-align:left;">
snoRNA gene
</td>
<td style="text-align:left;">
predicted gene, 22622
</td>
</tr>
<tr>
<td style="text-align:left;">
6
</td>
<td style="text-align:left;">
pseudogene
</td>
<td style="text-align:right;">
59.94233
</td>
<td style="text-align:right;">
59.94553
</td>
<td style="text-align:left;">
positive
</td>
<td style="text-align:left;">
MGI_C57BL6J_5010197
</td>
<td style="text-align:left;">
Gm18012
</td>
<td style="text-align:left;">
NCBI_Gene:100416270,ENSEMBL:ENSMUSG00000108110
</td>
<td style="text-align:left;">
MGI:5010197
</td>
<td style="text-align:left;">
pseudogene
</td>
<td style="text-align:left;">
predicted gene, 18012
</td>
</tr>
<tr>
<td style="text-align:left;">
6
</td>
<td style="text-align:left;">
gene
</td>
<td style="text-align:right;">
60.03197
</td>
<td style="text-align:right;">
60.15004
</td>
<td style="text-align:left;">
positive
</td>
<td style="text-align:left;">
MGI_C57BL6J_3644382
</td>
<td style="text-align:left;">
Gm5570
</td>
<td style="text-align:left;">
NCBI_Gene:434019,ENSEMBL:ENSMUSG00000073045
</td>
<td style="text-align:left;">
MGI:3644382
</td>
<td style="text-align:left;">
lncRNA gene
</td>
<td style="text-align:left;">
predicted gene 5570
</td>
</tr>
<tr>
<td style="text-align:left;">
6
</td>
<td style="text-align:left;">
gene
</td>
<td style="text-align:right;">
60.07106
</td>
<td style="text-align:right;">
60.07119
</td>
<td style="text-align:left;">
negative
</td>
<td style="text-align:left;">
MGI_C57BL6J_5452500
</td>
<td style="text-align:left;">
Gm22723
</td>
<td style="text-align:left;">
ENSEMBL:ENSMUSG00000095176
</td>
<td style="text-align:left;">
MGI:5452500
</td>
<td style="text-align:left;">
snoRNA gene
</td>
<td style="text-align:left;">
predicted gene, 22723
</td>
</tr>
<tr>
<td style="text-align:left;">
6
</td>
<td style="text-align:left;">
gene
</td>
<td style="text-align:right;">
60.17914
</td>
<td style="text-align:right;">
60.18178
</td>
<td style="text-align:left;">
positive
</td>
<td style="text-align:left;">
MGI_C57BL6J_5826622
</td>
<td style="text-align:left;">
Gm46985
</td>
<td style="text-align:left;">
NCBI_Gene:108169190
</td>
<td style="text-align:left;">
MGI:5826622
</td>
<td style="text-align:left;">
protein coding gene
</td>
<td style="text-align:left;">
predicted gene, 46985
</td>
</tr>
<tr>
<td style="text-align:left;">
6
</td>
<td style="text-align:left;">
gene
</td>
<td style="text-align:right;">
60.25079
</td>
<td style="text-align:right;">
60.25091
</td>
<td style="text-align:left;">
positive
</td>
<td style="text-align:left;">
MGI_C57BL6J_4422026
</td>
<td style="text-align:left;">
n-R5s162
</td>
<td style="text-align:left;">
ENSEMBL:ENSMUSG00000064515
</td>
<td style="text-align:left;">
MGI:4422026
</td>
<td style="text-align:left;">
rRNA gene
</td>
<td style="text-align:left;">
nuclear encoded rRNA 5S 162
</td>
</tr>
<tr>
<td style="text-align:left;">
6
</td>
<td style="text-align:left;">
gene
</td>
<td style="text-align:right;">
60.27904
</td>
<td style="text-align:right;">
60.40371
</td>
<td style="text-align:left;">
negative
</td>
<td style="text-align:left;">
MGI_C57BL6J_2443347
</td>
<td style="text-align:left;">
A530053G22Rik
</td>
<td style="text-align:left;">
NCBI_Gene:208079,ENSEMBL:ENSMUSG00000046764
</td>
<td style="text-align:left;">
MGI:2443347
</td>
<td style="text-align:left;">
lncRNA gene
</td>
<td style="text-align:left;">
RIKEN cDNA A530053G22 gene
</td>
</tr>
<tr>
<td style="text-align:left;">
6
</td>
<td style="text-align:left;">
gene
</td>
<td style="text-align:right;">
60.33980
</td>
<td style="text-align:right;">
60.37427
</td>
<td style="text-align:left;">
positive
</td>
<td style="text-align:left;">
MGI_C57BL6J_5594494
</td>
<td style="text-align:left;">
Gm35335
</td>
<td style="text-align:left;">
NCBI_Gene:102638876
</td>
<td style="text-align:left;">
MGI:5594494
</td>
<td style="text-align:left;">
lncRNA gene
</td>
<td style="text-align:left;">
predicted gene, 35335
</td>
</tr>
<tr>
<td style="text-align:left;">
6
</td>
<td style="text-align:left;">
gene
</td>
<td style="text-align:right;">
60.41120
</td>
<td style="text-align:right;">
60.44969
</td>
<td style="text-align:left;">
positive
</td>
<td style="text-align:left;">
MGI_C57BL6J_5621708
</td>
<td style="text-align:left;">
Gm38823
</td>
<td style="text-align:left;">
NCBI_Gene:105242671
</td>
<td style="text-align:left;">
MGI:5621708
</td>
<td style="text-align:left;">
lncRNA gene
</td>
<td style="text-align:left;">
predicted gene, 38823
</td>
</tr>
<tr>
<td style="text-align:left;">
6
</td>
<td style="text-align:left;">
pseudogene
</td>
<td style="text-align:right;">
60.46193
</td>
<td style="text-align:right;">
60.46211
</td>
<td style="text-align:left;">
negative
</td>
<td style="text-align:left;">
MGI_C57BL6J_5690781
</td>
<td style="text-align:left;">
Gm44389
</td>
<td style="text-align:left;">
ENSEMBL:ENSMUSG00000107863
</td>
<td style="text-align:left;">
MGI:5690781
</td>
<td style="text-align:left;">
pseudogene
</td>
<td style="text-align:left;">
predicted gene, 44389
</td>
</tr>
<tr>
<td style="text-align:left;">
6
</td>
<td style="text-align:left;">
gene
</td>
<td style="text-align:right;">
60.48996
</td>
<td style="text-align:right;">
60.49045
</td>
<td style="text-align:left;">
positive
</td>
<td style="text-align:left;">
MGI_C57BL6J_5690802
</td>
<td style="text-align:left;">
Gm44410
</td>
<td style="text-align:left;">
ENSEMBL:ENSMUSG00000108192
</td>
<td style="text-align:left;">
MGI:5690802
</td>
<td style="text-align:left;">
lncRNA gene
</td>
<td style="text-align:left;">
predicted gene, 44410
</td>
</tr>
<tr>
<td style="text-align:left;">
6
</td>
<td style="text-align:left;">
gene
</td>
<td style="text-align:right;">
60.58677
</td>
<td style="text-align:right;">
60.59155
</td>
<td style="text-align:left;">
positive
</td>
<td style="text-align:left;">
MGI_C57BL6J_5594545
</td>
<td style="text-align:left;">
Gm35386
</td>
<td style="text-align:left;">
NCBI_Gene:102638951,ENSEMBL:ENSMUSG00000108003
</td>
<td style="text-align:left;">
MGI:5594545
</td>
<td style="text-align:left;">
lncRNA gene
</td>
<td style="text-align:left;">
predicted gene, 35386
</td>
</tr>
<tr>
<td style="text-align:left;">
6
</td>
<td style="text-align:left;">
gene
</td>
<td style="text-align:right;">
60.73157
</td>
<td style="text-align:right;">
60.82986
</td>
<td style="text-align:left;">
negative
</td>
<td style="text-align:left;">
MGI_C57BL6J_1277151
</td>
<td style="text-align:left;">
Snca
</td>
<td style="text-align:left;">
NCBI_Gene:20617,ENSEMBL:ENSMUSG00000025889
</td>
<td style="text-align:left;">
MGI:1277151
</td>
<td style="text-align:left;">
protein coding gene
</td>
<td style="text-align:left;">
synuclein, alpha
</td>
</tr>
<tr>
<td style="text-align:left;">
6
</td>
<td style="text-align:left;">
pseudogene
</td>
<td style="text-align:right;">
60.76952
</td>
<td style="text-align:right;">
60.77118
</td>
<td style="text-align:left;">
negative
</td>
<td style="text-align:left;">
MGI_C57BL6J_3701602
</td>
<td style="text-align:left;">
Mageb16-ps2
</td>
<td style="text-align:left;">
NCBI_Gene:625863,ENSEMBL:ENSMUSG00000107755
</td>
<td style="text-align:left;">
MGI:3701602
</td>
<td style="text-align:left;">
pseudogene
</td>
<td style="text-align:left;">
melanoma antigen family B, 16, pseudogene 2
</td>
</tr>
<tr>
<td style="text-align:left;">
6
</td>
<td style="text-align:left;">
gene
</td>
<td style="text-align:right;">
60.79184
</td>
<td style="text-align:right;">
60.79587
</td>
<td style="text-align:left;">
negative
</td>
<td style="text-align:left;">
MGI_C57BL6J_5690256
</td>
<td style="text-align:left;">
Gm43864
</td>
<td style="text-align:left;">
ENSEMBL:ENSMUSG00000108168
</td>
<td style="text-align:left;">
MGI:5690256
</td>
<td style="text-align:left;">
unclassified gene
</td>
<td style="text-align:left;">
predicted gene, 43864
</td>
</tr>
<tr>
<td style="text-align:left;">
6
</td>
<td style="text-align:left;">
pseudogene
</td>
<td style="text-align:right;">
60.88244
</td>
<td style="text-align:right;">
60.88296
</td>
<td style="text-align:left;">
negative
</td>
<td style="text-align:left;">
MGI_C57BL6J_5690259
</td>
<td style="text-align:left;">
Gm43867
</td>
<td style="text-align:left;">
ENSEMBL:ENSMUSG00000108262
</td>
<td style="text-align:left;">
MGI:5690259
</td>
<td style="text-align:left;">
pseudogene
</td>
<td style="text-align:left;">
predicted gene, 43867
</td>
</tr>
<tr>
<td style="text-align:left;">
6
</td>
<td style="text-align:left;">
gene
</td>
<td style="text-align:right;">
60.92498
</td>
<td style="text-align:right;">
60.98938
</td>
<td style="text-align:left;">
positive
</td>
<td style="text-align:left;">
MGI_C57BL6J_1918195
</td>
<td style="text-align:left;">
Mmrn1
</td>
<td style="text-align:left;">
NCBI_Gene:70945,ENSEMBL:ENSMUSG00000054641
</td>
<td style="text-align:left;">
MGI:1918195
</td>
<td style="text-align:left;">
protein coding gene
</td>
<td style="text-align:left;">
multimerin 1
</td>
</tr>
<tr>
<td style="text-align:left;">
6
</td>
<td style="text-align:left;">
pseudogene
</td>
<td style="text-align:right;">
61.01685
</td>
<td style="text-align:right;">
61.01774
</td>
<td style="text-align:left;">
negative
</td>
<td style="text-align:left;">
MGI_C57BL6J_5011023
</td>
<td style="text-align:left;">
Gm18838
</td>
<td style="text-align:left;">
NCBI_Gene:100417805,ENSEMBL:ENSMUSG00000108098
</td>
<td style="text-align:left;">
MGI:5011023
</td>
<td style="text-align:left;">
pseudogene
</td>
<td style="text-align:left;">
predicted gene, 18838
</td>
</tr>
<tr>
<td style="text-align:left;">
6
</td>
<td style="text-align:left;">
pseudogene
</td>
<td style="text-align:right;">
61.04473
</td>
<td style="text-align:right;">
61.04499
</td>
<td style="text-align:left;">
negative
</td>
<td style="text-align:left;">
MGI_C57BL6J_5690284
</td>
<td style="text-align:left;">
Gm43892
</td>
<td style="text-align:left;">
ENSEMBL:ENSMUSG00000108299
</td>
<td style="text-align:left;">
MGI:5690284
</td>
<td style="text-align:left;">
pseudogene
</td>
<td style="text-align:left;">
predicted gene, 43892
</td>
</tr>
<tr>
<td style="text-align:left;">
6
</td>
<td style="text-align:left;">
gene
</td>
<td style="text-align:right;">
61.06470
</td>
<td style="text-align:right;">
61.06925
</td>
<td style="text-align:left;">
negative
</td>
<td style="text-align:left;">
MGI_C57BL6J_5690285
</td>
<td style="text-align:left;">
Gm43893
</td>
<td style="text-align:left;">
ENSEMBL:ENSMUSG00000108087
</td>
<td style="text-align:left;">
MGI:5690285
</td>
<td style="text-align:left;">
lncRNA gene
</td>
<td style="text-align:left;">
predicted gene, 43893
</td>
</tr>
<tr>
<td style="text-align:left;">
6
</td>
<td style="text-align:left;">
gene
</td>
<td style="text-align:right;">
61.14722
</td>
<td style="text-align:right;">
61.14791
</td>
<td style="text-align:left;">
negative
</td>
<td style="text-align:left;">
MGI_C57BL6J_5690286
</td>
<td style="text-align:left;">
Gm43894
</td>
<td style="text-align:left;">
ENSEMBL:ENSMUSG00000107652
</td>
<td style="text-align:left;">
MGI:5690286
</td>
<td style="text-align:left;">
lncRNA gene
</td>
<td style="text-align:left;">
predicted gene, 43894
</td>
</tr>
<tr>
<td style="text-align:left;">
6
</td>
<td style="text-align:left;">
gene
</td>
<td style="text-align:right;">
61.17229
</td>
<td style="text-align:right;">
61.18081
</td>
<td style="text-align:left;">
negative
</td>
<td style="text-align:left;">
MGI_C57BL6J_2444999
</td>
<td style="text-align:left;">
A730020E08Rik
</td>
<td style="text-align:left;">
NCBI_Gene:100504262,ENSEMBL:ENSMUSG00000097924
</td>
<td style="text-align:left;">
MGI:2444999
</td>
<td style="text-align:left;">
lncRNA gene
</td>
<td style="text-align:left;">
RIKEN cDNA A730020E08 gene
</td>
</tr>
<tr>
<td style="text-align:left;">
6
</td>
<td style="text-align:left;">
gene
</td>
<td style="text-align:right;">
61.18032
</td>
<td style="text-align:right;">
62.38287
</td>
<td style="text-align:left;">
positive
</td>
<td style="text-align:left;">
MGI_C57BL6J_3045354
</td>
<td style="text-align:left;">
Ccser1
</td>
<td style="text-align:left;">
NCBI_Gene:232035,ENSEMBL:ENSMUSG00000039578
</td>
<td style="text-align:left;">
MGI:3045354
</td>
<td style="text-align:left;">
protein coding gene
</td>
<td style="text-align:left;">
coiled-coil serine rich 1
</td>
</tr>
<tr>
<td style="text-align:left;">
6
</td>
<td style="text-align:left;">
gene
</td>
<td style="text-align:right;">
61.23173
</td>
<td style="text-align:right;">
61.23446
</td>
<td style="text-align:left;">
positive
</td>
<td style="text-align:left;">
MGI_C57BL6J_3026947
</td>
<td style="text-align:left;">
B230204D01Rik
</td>
<td style="text-align:left;">
NA
</td>
<td style="text-align:left;">
NA
</td>
<td style="text-align:left;">
unclassified gene
</td>
<td style="text-align:left;">
RIKEN cDNA B230204D01 gene
</td>
</tr>
<tr>
<td style="text-align:left;">
6
</td>
<td style="text-align:left;">
pseudogene
</td>
<td style="text-align:right;">
61.40061
</td>
<td style="text-align:right;">
61.40103
</td>
<td style="text-align:left;">
positive
</td>
<td style="text-align:left;">
MGI_C57BL6J_5690290
</td>
<td style="text-align:left;">
Gm43898
</td>
<td style="text-align:left;">
ENSEMBL:ENSMUSG00000107757
</td>
<td style="text-align:left;">
MGI:5690290
</td>
<td style="text-align:left;">
pseudogene
</td>
<td style="text-align:left;">
predicted gene, 43898
</td>
</tr>
<tr>
<td style="text-align:left;">
6
</td>
<td style="text-align:left;">
gene
</td>
<td style="text-align:right;">
61.97510
</td>
<td style="text-align:right;">
61.97713
</td>
<td style="text-align:left;">
positive
</td>
<td style="text-align:left;">
MGI_C57BL6J_5690288
</td>
<td style="text-align:left;">
Gm43896
</td>
<td style="text-align:left;">
ENSEMBL:ENSMUSG00000108246
</td>
<td style="text-align:left;">
MGI:5690288
</td>
<td style="text-align:left;">
unclassified gene
</td>
<td style="text-align:left;">
predicted gene, 43896
</td>
</tr>
<tr>
<td style="text-align:left;">
6
</td>
<td style="text-align:left;">
gene
</td>
<td style="text-align:right;">
61.99305
</td>
<td style="text-align:right;">
61.99639
</td>
<td style="text-align:left;">
positive
</td>
<td style="text-align:left;">
MGI_C57BL6J_2443660
</td>
<td style="text-align:left;">
A730075L09Rik
</td>
<td style="text-align:left;">
NA
</td>
<td style="text-align:left;">
NA
</td>
<td style="text-align:left;">
unclassified gene
</td>
<td style="text-align:left;">
RIKEN cDNA A730075L09 gene
</td>
</tr>
</tbody>
</table>
</div>
</div>
<div id="interactive-term" class="section level2">
<h2>Interactive term</h2>
<div id="significance-thresholds-2" class="section level3">
<h3>Significance thresholds</h3>
<pre class="r"><code>operm &lt;- scan1perm(pr.qc, covars[&quot;age.of.onset&quot;], model=&quot;normal&quot;, addcovar=addcovar, intcovar=addcovar, n_perm=1000, perm_Xsp=TRUE, chr_lengths=chr_lengths(gm$gmap))

summary_table&lt;-data.frame(unclass(summary(operm, alpha=c(0.01,  0.05, 0.1))))
names(summary_table) &lt;- c(&quot;autosomes&quot;,&quot;X&quot;)
summary_table$significance.level &lt;- rownames(summary_table)

rownames(summary_table) &lt;- NULL

summary_table[c(3,1:2)] %&gt;%
  kable(escape = F,align = c(&quot;ccc&quot;)) %&gt;%
  kable_styling(&quot;striped&quot;, full_width = T) %&gt;%
  column_spec(1, bold=TRUE)</code></pre>
<table class="table table-striped" style="margin-left: auto; margin-right: auto;">
<thead>
<tr>
<th style="text-align:center;">
significance.level
</th>
<th style="text-align:center;">
autosomes
</th>
<th style="text-align:center;">
X
</th>
</tr>
</thead>
<tbody>
<tr>
<td style="text-align:center;font-weight: bold;">
0.01
</td>
<td style="text-align:center;">
9.466227
</td>
<td style="text-align:center;">
8.928969
</td>
</tr>
<tr>
<td style="text-align:center;font-weight: bold;">
0.05
</td>
<td style="text-align:center;">
7.803570
</td>
<td style="text-align:center;">
7.474475
</td>
</tr>
<tr>
<td style="text-align:center;font-weight: bold;">
0.1
</td>
<td style="text-align:center;">
6.761415
</td>
<td style="text-align:center;">
6.769414
</td>
</tr>
</tbody>
</table>
</div>
<div id="genome-wide-scan-2" class="section level3">
<h3>Genome-wide scan</h3>
<pre class="r"><code>plot_lod&lt;-function(out,map){
  for (i in 1:dim(out)[2]){
#par(mar=c(5.1, 6.1, 1.1, 1.1))
    ymx &lt;- maxlod(out) # overall maximum LOD score
    plot(out, map, lodcolumn=i, col=&quot;slateblue&quot;, ylim=c(0, ymx+0.5))
    #legend(&quot;topright&quot;, lwd=2, colnames(out)[i], bg=&quot;gray90&quot;)
    title(main = paste0(colnames(out)[i], &quot; - ICI-Early vs PBS &amp; ICI-Late (with interactive covariate) [positions in cM]&quot;))
    add_threshold(map,  summary(operm,alpha=0.1), col = &#39;purple&#39;)
    add_threshold(map,  summary(operm, alpha=0.05), col = &#39;red&#39;)
    add_threshold(map,  summary(operm, alpha=0.01), col = &#39;blue&#39;)

  }
}


#print(&quot;with no kinship&quot;)
#outfi &lt;- scan1(pr.qc, covars[&quot;age.of.onset&quot;], Xcovar=Xcovar, addcovar=addcovar, intcovar=addcovar, model=&quot;normal&quot;)

#plot_lod(outfi,gm$gmap)

#outf.new &lt;- outfi-outfa

#plot_lod(outf.new,gm$gmap)

#print(&quot;with normal kinship&quot;)
#outki &lt;- scan1(pr.qc, covars[&quot;age.of.onset&quot;], Xcovar=Xcovar, model=&quot;normal&quot;, addcovar=addcovar, intcovar=addcovar, kinship=kinship)

#plot_lod(outki,gm$gmap)

#outk.new &lt;- outki-outka

#plot_lod(outk.new,gm$gmap)

#print(&quot;with loco kinship&quot;)
#outlki &lt;- scan1(pr.qc, covars[&quot;age.of.onset&quot;], Xcovar=Xcovar, model=&quot;normal&quot;, addcovar=addcovar, intcovar=addcovar, kinship=K)

#plot_lod(outlki,gm$gmap)

#outlk.new &lt;- outlki-outlka

#plot_lod(outlk.new,gm$gmap)

print(&quot;no kinship and no xcovar&quot;)</code></pre>
<p>[1] “no kinship and no xcovar”</p>
<pre class="r"><code>outi &lt;- scan1(pr.qc, covars[&quot;age.of.onset&quot;], model=&quot;normal&quot;,addcovar=addcovar, intcovar=addcovar)

out.new &lt;- outi-outa

plot_lod(out.new,gm$gmap)</code></pre>
<p><img src="figure/4.1.2_qtl.analysis_cont_age_ici-early.vs.pbs.ici-late_snpsqc_pheno.corrected.cleaned_dis_no-xk_5.batches_mis.Rmd/scanonei-1.png" width="912" style="display: block; margin: auto;" /></p>
<pre class="r"><code>out = out.new




print(paste0(&quot;number of samples in analysis = &quot;, as.data.frame(attributes(out)$sample_size)[1,]))</code></pre>
<p>[1] “number of samples in analysis = 138”</p>
</div>
<div id="lod-peaks-2" class="section level3">
<h3>LOD peaks</h3>
<div id="centimorgan-cm-2" class="section level4">
<h4>Centimorgan (cM)</h4>
<pre class="r"><code>peaks&lt;-find_peaks(out, gm$gmap, threshold=summary(operm,alpha=0.1)$A, thresholdX = summary(operm,alpha=0.1)$X, peakdrop=3, drop=1.5)

if(nrow(peaks) &gt;0){
peaks$marker &lt;- find_marker(gm$gmap, chr=peaks$chr,pos=peaks$pos)
names(peaks)[2] &lt;- c(&quot;phenotype&quot;)
peaks &lt;- peaks[-1]

rownames(peaks) &lt;- NULL
print(kable(peaks, escape = F, align = c(&quot;cccccccc&quot;), &quot;html&quot;) 
  %&gt;% kable_styling(&quot;striped&quot;, full_width = T)%&gt;%
  column_spec(1, bold=TRUE)
  )

#plot only peak chromosomes

plot_lod_chr&lt;-function(out,map,chrom){
  for (i in 1:dim(out)[2]){
    #png(filename=paste0(&quot;/Users/chenm/Documents/qtl/Jai/&quot;,colnames(out)[i],  &quot;_lod.png&quot;))
    
    #par(mar=c(5.1, 6.1, 1.1, 1.1))
    ymx &lt;- maxlod(out) # overall maximum LOD score
    plot(out, map, chr = chrom, lodcolumn=i, col=&quot;slateblue&quot;, ylim=c(0, ymx+0.5))
    #legend(&quot;topright&quot;, lwd=2, colnames(out)[i], bg=&quot;gray90&quot;)
    title(main = paste0(colnames(out)[i], &quot; - chr&quot;, chrom, &quot; - ICI-Early vs PBS &amp; ICI-Late (with interactive covariate) [positions in cM]&quot;))
    add_threshold(map,  summary(operm,alpha=0.1), col = &#39;purple&#39;)
    add_threshold(map,  summary(operm, alpha=0.05), col = &#39;red&#39;)
    add_threshold(map,  summary(operm, alpha=0.01), col = &#39;blue&#39;)
    #for (j in 1: dim(summary_table)[1]){
    #  abline(h=summary_table[j, i],col=&quot;red&quot;)
    #  text(x=400, y =summary_table[j, i]+0.12, labels = paste(&quot;p=&quot;, row.names(summary_table)[j]))
    #}
    #dev.off()

  }
}


for(i in unique(peaks$chr)){
#for (i in 1:nrow(peaks)){
  #plot_lod_chr(out,gm$gmap, peaks$chr[i])
  plot_lod_chr(out,gm$gmap, i)
}

} else {
  print(paste0(&quot;There are no peaks that have a LOD that reaches suggestive (p&lt;0.1) level of &quot;,summary(operm,alpha=0.1)$A, &quot; [autosomes]/&quot;,summary(operm,alpha=0.1)$X, &quot; [x-chromosome]&quot;))
}</code></pre>
<p>[1] “There are no peaks that have a LOD that reaches suggestive (p&lt;0.1) level of 6.7614147385478 [autosomes]/6.76941390896939 [x-chromosome]”</p>
</div>
<div id="megabase-mb-2" class="section level4">
<h4>Megabase (MB)</h4>
<pre class="r"><code>peaks_mba&lt;-find_peaks(out, gm$pmap, threshold=summary(operm,alpha=0.1)$A, thresholdX = summary(operm,alpha=0.1)$X, peakdrop=3, drop=1.5)

if(nrow(peaks) &gt;0){
peaks_mba$marker &lt;- find_marker(gm$pmap, chr=peaks_mba$chr,pos=peaks_mba$pos)
names(peaks_mba)[2] &lt;- c(&quot;phenotype&quot;)
peaks_mba &lt;- peaks_mba[-1]

rownames(peaks_mba) &lt;- NULL
print(kable(peaks_mba, escape = F, align = c(&quot;cccccccc&quot;), &quot;html&quot;) 
  %&gt;% kable_styling(&quot;striped&quot;, full_width = T)%&gt;%
  column_spec(1, bold=TRUE)
  )

plot_lod_chr_mb&lt;-function(out,map,chrom){
  for (i in 1:dim(out)[2]){
    #png(filename=paste0(&quot;/Users/chenm/Documents/qtl/Jai/&quot;,colnames(out)[i],  &quot;_lod.png&quot;))
    
    #par(mar=c(5.1, 6.1, 1.1, 1.1))
    ymx &lt;- maxlod(out) # overall maximum LOD score
    plot(out, map, chr = chrom, lodcolumn=i, col=&quot;slateblue&quot;, ylim=c(0, ymx+0.5))
    #legend(&quot;topright&quot;, lwd=2, colnames(out)[i], bg=&quot;gray90&quot;)
    title(main = paste0(colnames(out)[i], &quot; - chr&quot;, chrom, &quot; - ICI-Early vs PBS &amp; ICI-Late (with interactive covariate) [positions in cM]&quot;))
    add_threshold(map,  summary(operm,alpha=0.1), col = &#39;purple&#39;)
    add_threshold(map,  summary(operm, alpha=0.05), col = &#39;red&#39;)
    add_threshold(map,  summary(operm, alpha=0.01), col = &#39;blue&#39;)
    #for (j in 1: dim(summary_table)[1]){
    #  abline(h=summary_table[j, i],col=&quot;red&quot;)
    #  text(x=400, y =summary_table[j, i]+0.12, labels = paste(&quot;p=&quot;, row.names(summary_table)[j]))
    #}
    #dev.off()
  }
}

for(i in unique(peaks_mba$chr)){
#for (i in 1:nrow(peaks_mba)){
  #plot_lod_chr_mb(out,gm$pmap, peaks_mba$chr[i])
  plot_lod_chr_mb(out,gm$pmap,i)
}

} else {
  print(paste0(&quot;There are no peaks that have a LOD that reaches suggestive (p&lt;0.1) level of &quot;,summary(operm,alpha=0.1)$A, &quot; [autosomes]/&quot;,summary(operm,alpha=0.1)$X, &quot; [x-chromosome]&quot;))
}</code></pre>
<p>[1] “There are no peaks that have a LOD that reaches suggestive (p&lt;0.1) level of 6.7614147385478 [autosomes]/6.76941390896939 [x-chromosome]”</p>
</div>
</div>
<div id="qtl-effects-2" class="section level3">
<h3>QTL effects</h3>
<p>For each peak LOD location we give a list of gene</p>
<pre class="r"><code>if(nrow(peaks) &gt;0){
for (i in 1:nrow(peaks)){
#for (i in 1:1){
  #Plot 1
  g &lt;- maxmarg(pr.qc, gm$gmap, chr=peaks$chr[i], pos=peaks$pos[i], return_char=TRUE)
  #png(filename=paste0(&quot;/Users/chenm/Documents/qtl/Jai/&quot;,&quot;qtl_effect_&quot;, i, &quot;.png&quot;))
  #par(mar=c(4.1, 4.1, 1.5, 0.6))
  plot_pxg(g, covars[,peaks$phenotype[i]], ylab=peaks$phenotype[i], sort=FALSE)
  title(main = paste0(&quot;chr: &quot;, chr=peaks$chr[i],&quot;; pos: &quot;, peaks$pos[i], &quot;cM /&quot;,peaks_mba$pos[i],&quot;MB\n(&quot;,peaks$phenotype[i],&quot; )&quot;), line=0.2)
  ##dev.off()

  chr = peaks$chr[i]

# Plot 2
  pr_sub &lt;- pull_genoprobint(pr.qc, gm$gmap, chr, c(peaks$ci_lo[i], peaks$ci_hi[i]))
  #coeff &lt;- scan1coef(pr[,chr], cross$pheno[,peaks$lodcolumn[i]], addcovar = addcovar)
  #coeff &lt;- scan1coef(pr[,chr], cross$pheno[,peaks$lodcolumn[i]], Xcovar=Xcovar)
  #coeff &lt;- scan1coef(pr.qc[,chr], gm$covar[peaks$lodcolumn[i]], model=&quot;binary&quot;)
  #coeff_sub &lt;- scan1coef(pr_sub[,chr], gm$covar[peaks$lodcolumn[i]], model=&quot;binary&quot;)
  blup &lt;- scan1blup(pr.qc[,chr], covars[peaks$phenotype[i]])
  blup_sub &lt;- scan1blup(pr_sub[,chr], covars[peaks$phenotype[i]])

  write.csv(as.data.frame(blup_sub), paste0(&quot;data/ici-early.vs.pbs.ici-late_&quot;,peaks$phenotype[i],&quot;-interactive.covariate_blup_sub_chr-&quot;,chr,&quot;_peak.marker-&quot;,peaks$marker[i],&quot;_lod.drop-1.5_snpsqc_dis_no-x_updated_5.batches_mis.csv&quot;), quote=F)

  #plot_coef(coeff, 
  #     gm$gmap, columns=1:2,
  #     bgcolor=&quot;gray95&quot;, legend=&quot;bottomleft&quot;, 
  #     main = paste(&quot;chr&quot;, chr=peaks$chr[i],&quot;; pos: &quot;, peaks$pos[i], &quot;cM /&quot;,peaks_mba$pos[i],&quot;MB\n(&quot;,peaks$lodcolumn[i],&quot; [scan1coeff; positions in cM] )&quot;)
  #     )

  #plot_coef(coeff_sub, 
  #     gm$gmap, columns=1:2,
  #     bgcolor=&quot;gray95&quot;, legend=&quot;bottomleft&quot;, 
  #     main = paste(&quot;chr&quot;, chr=peaks$chr[i],&quot;; pos: &quot;, peaks$pos[i], &quot;cM /&quot;,peaks_mba$pos[i],&quot;MB\n(&quot;,peaks$lodcolumn[i],&quot;; 1.5 LOD drop interval [scan1coeff; positions in cM] ) &quot;)
  #     )


  plot_coef(blup, 
       gm$gmap, columns=1:2,
       bgcolor=&quot;gray95&quot;, legend=&quot;bottomleft&quot;, 
       main = paste0(&quot;chr: &quot;, chr=peaks$chr[i],&quot;; pos: &quot;, peaks$pos[i], &quot;cM /&quot;,peaks_mba$pos[i],&quot;MB\n(&quot;,peaks$phenotype[i],&quot; [scan1blup; positions in cM] )&quot;)
       )

  plot_coef(blup_sub, 
       gm$gmap, columns=1:2,
       bgcolor=&quot;gray95&quot;, legend=&quot;bottomleft&quot;, 
       main = paste0(&quot;chr: &quot;, chr=peaks$chr[i],&quot;; pos: &quot;, peaks$pos[i], &quot;cM /&quot;,peaks_mba$pos[i],&quot;MB\n(&quot;,peaks$phenotype[i],&quot;; 1.5 LOD drop interval [scan1blup; positions in cM] )&quot;)
       )


 # Plot 3
  #c2effB &lt;- scan1coef(pr.qc[,chr], gm$covar[peaks$lodcolumn[i]], model=&quot;binary&quot;, contrasts=cbind(a=c(-1, 0), d=c(0, -1)))
  #c2effBb &lt;- scan1blup(pr.qc[,chr], gm$covar[peaks$lodcolumn[i]], contrasts=cbind(a=c(-1, 0), d=c(0, -1)))
  ##c2effB &lt;- scan1coef(pr[,chr], cross$pheno[,peaks$lodcolumn[i]], addcovar = addcovar, contrasts=cbind(mu=c(1,1,1), a=c(-1, 0, 1), d=c(0, 1, 0)))
  ##c2effB &lt;- scan1coef(pr[,chr], cross$pheno[,peaks$lodcolumn[i]],Xcovar=Xcovar, contrasts=cbind(mu=c(1,1,1), a=c(-1, 0, 1), d=c(0, 1, 0)))
  #plot(c2effB, gm$gmap[chr], columns=1:2,
  #     bgcolor=&quot;gray95&quot;, legend=&quot;bottomleft&quot;, 
  #     main = paste(&quot;chr&quot;, chr=peaks$chr[i], &quot;pos&quot;, peaks$pos[i], &quot;(&quot;,peaks$lodcolumn[i],&quot;)&quot;)
  #     )
  #plot(c2effBb, gm$gmap[chr], columns=1:2,
  #     bgcolor=&quot;gray95&quot;, legend=&quot;bottomleft&quot;, 
  #     main = paste(&quot;chr&quot;, chr=peaks$chr[i], &quot;pos&quot;, peaks$pos[i], &quot;(&quot;,peaks$lodcolumn[i],&quot;)&quot;)
  #     )
  ##last_coef &lt;- unclass(c2effB)[nrow(c2effB),2:3] # last two coefficients
  ##for(t in seq(along=last_coef))
  ##  axis(side=4, at=last_coef[t], names(last_coef)[t], tick=FALSE)


  #Table 1
  chr = peaks_mba$chr[i]
  start=as.numeric(peaks_mba$ci_lo[i])
  end=as.numeric(peaks_mba$ci_hi[i])

  genesgss = query_genes(chr, start, end)

  write.csv(genesgss, file=paste0(&quot;data/ici-early.vs.pbs.ici-late_&quot;,peaks$phenotype[i],&quot;-interactive.covariate_genes_chr-&quot;,chr,&quot;_peak.marker-&quot;,peaks$marker[i],&quot;_lod.drop-1.5_snpsqc_dis_no-x_updated_5.batches_mis.csv&quot;), quote=F)

  rownames(genesgss) &lt;- NULL
  genesgss$strand_old = genesgss$strand
  genesgss$strand[genesgss$strand==&quot;+&quot;] &lt;- &quot;positive&quot;
  genesgss$strand[genesgss$strand==&quot;-&quot;] &lt;- &quot;negative&quot;

  #genesgss &lt;- 
  #table &lt;- 
  #genesgss[,c(&quot;chr&quot;,&quot;type&quot;,&quot;start&quot;,&quot;stop&quot;,&quot;strand&quot;,&quot;ID&quot;,&quot;Name&quot;,&quot;Dbxref&quot;,&quot;gene_id&quot;,&quot;mgi_type&quot;,&quot;description&quot;)] %&gt;%
  #kable(escape = F,align = c(&quot;ccccccccccc&quot;)) %&gt;%
  #kable_styling(&quot;striped&quot;, full_width = T) #%&gt;% 
  #cat #%&gt;%
  #column_spec(1, bold=TRUE)
#
  #print(kable(genesgss[,c(&quot;chr&quot;,&quot;type&quot;,&quot;start&quot;,&quot;stop&quot;,&quot;strand&quot;,&quot;ID&quot;,&quot;Name&quot;,&quot;Dbxref&quot;,&quot;gene_id&quot;,&quot;mgi_type&quot;,&quot;description&quot;)], escape = F,align = c(&quot;ccccccccccc&quot;)))

  print(kable(genesgss[,c(&quot;chr&quot;,&quot;type&quot;,&quot;start&quot;,&quot;stop&quot;,&quot;strand&quot;,&quot;ID&quot;,&quot;Name&quot;,&quot;Dbxref&quot;,&quot;gene_id&quot;,&quot;mgi_type&quot;,&quot;description&quot;)], &quot;html&quot;) %&gt;% kable_styling(&quot;striped&quot;, full_width = T))

  #table
  

}
} else {
  print(paste0(&quot;There are no peaks that have a LOD that reaches suggestive (p&lt;0.1) level of &quot;,summary(operm,alpha=0.1)$A, &quot; [autosomes]/&quot;,summary(operm,alpha=0.1)$X, &quot; [x-chromosome]&quot;))
}</code></pre>
<p>[1] “There are no peaks that have a LOD that reaches suggestive (p&lt;0.1) level of 6.7614147385478 [autosomes]/6.76941390896939 [x-chromosome]”</p>
<br>
<p>
<button type="button" class="btn btn-default btn-workflowr btn-workflowr-sessioninfo" data-toggle="collapse" data-target="#workflowr-sessioninfo" style="display: block;">
<span class="glyphicon glyphicon-wrench" aria-hidden="true"></span> Session information
</button>
</p>
<div id="workflowr-sessioninfo" class="collapse">
<pre><code>R version 3.6.2 (2019-12-12)
Platform: x86_64-apple-darwin15.6.0 (64-bit)
Running under: macOS Catalina 10.15.7

Matrix products: default
BLAS:   /Library/Frameworks/R.framework/Versions/3.6/Resources/lib/libRblas.0.dylib
LAPACK: /Library/Frameworks/R.framework/Versions/3.6/Resources/lib/libRlapack.dylib

locale:
[1] en_AU.UTF-8/en_AU.UTF-8/en_AU.UTF-8/C/en_AU.UTF-8/en_AU.UTF-8

attached base packages:
[1] stats     graphics  grDevices utils     datasets  methods   base     

other attached packages:
 [1] abind_1.4-5       qtl2_0.22         reshape2_1.4.4    ggplot2_3.3.5    
 [5] tibble_3.1.2      psych_2.0.7       readxl_1.3.1      cluster_2.1.0    
 [9] dplyr_1.0.8       optparse_1.6.6    rhdf5_2.28.1      mclust_5.4.6     
[13] tidyr_1.0.2       data.table_1.14.0 knitr_1.33        kableExtra_1.1.0 
[17] workflowr_1.6.2  

loaded via a namespace (and not attached):
 [1] httr_1.4.1        bit64_4.0.5       viridisLite_0.4.0 assertthat_0.2.1 
 [5] highr_0.9         blob_1.2.1        cellranger_1.1.0  yaml_2.2.1       
 [9] pillar_1.6.1      RSQLite_2.2.7     backports_1.2.1   lattice_0.20-38  
[13] glue_1.4.2        digest_0.6.27     promises_1.1.0    rvest_0.3.5      
[17] colorspace_2.0-2  htmltools_0.5.1.1 httpuv_1.5.2      plyr_1.8.6       
[21] pkgconfig_2.0.3   purrr_0.3.4       scales_1.1.1      webshot_0.5.2    
[25] getopt_1.20.3     later_1.0.0       git2r_0.26.1      generics_0.0.2   
[29] ellipsis_0.3.2    cachem_1.0.5      withr_2.4.2       cli_3.0.0        
[33] mnormt_1.5-7      magrittr_2.0.1    crayon_1.4.1      memoise_2.0.0    
[37] evaluate_0.14     fs_1.4.1          fansi_0.5.0       nlme_3.1-142     
[41] xml2_1.3.1        tools_3.6.2       hms_0.5.3         lifecycle_1.0.1  
[45] stringr_1.4.0     Rhdf5lib_1.6.3    munsell_0.5.0     compiler_3.6.2   
[49] rlang_1.0.2       grid_3.6.2        rstudioapi_0.13   rmarkdown_2.1    
[53] gtable_0.3.0      DBI_1.1.1         R6_2.5.0          fastmap_1.1.0    
[57] bit_4.0.4         utf8_1.2.1        rprojroot_1.3-2   readr_1.3.1      
[61] stringi_1.7.2     parallel_3.6.2    Rcpp_1.0.7        vctrs_0.3.8      
[65] tidyselect_1.1.2  xfun_0.24        </code></pre>
</div>
</div>
</div>
</div>


<!-- Adjust MathJax settings so that all math formulae are shown using
TeX fonts only; see
http://docs.mathjax.org/en/latest/configuration.html.  This will make
the presentation more consistent at the cost of the webpage sometimes
taking slightly longer to load. Note that this only works because the
footer is added to webpages before the MathJax javascript. -->
<script type="text/x-mathjax-config">
  MathJax.Hub.Config({
    "HTML-CSS": { availableFonts: ["TeX"] }
  });
</script>





</div>

<script>

// add bootstrap table styles to pandoc tables
function bootstrapStylePandocTables() {
  $('tr.header').parent('thead').parent('table').addClass('table table-condensed');
}
$(document).ready(function () {
  bootstrapStylePandocTables();
});


</script>

<!-- tabsets -->

<script>
$(document).ready(function () {
  window.buildTabsets("TOC");
});

$(document).ready(function () {
  $('.tabset-dropdown > .nav-tabs > li').click(function () {
    $(this).parent().toggleClass('nav-tabs-open')
  });
});
</script>

<!-- code folding -->


<!-- dynamically load mathjax for compatibility with self-contained -->
<script>
  (function () {
    var script = document.createElement("script");
    script.type = "text/javascript";
    script.src  = "https://mathjax.rstudio.com/latest/MathJax.js?config=TeX-AMS-MML_HTMLorMML";
    document.getElementsByTagName("head")[0].appendChild(script);
  })();
</script>

</body>
</html>
