<!DOCTYPE html>

<html>

<head>

<meta charset="utf-8" />
<meta name="generator" content="pandoc" />
<meta http-equiv="X-UA-Compatible" content="IE=EDGE" />


<meta name="author" content="Belinda Cornes" />

<meta name="date" content="2023-04-16" />

<title>Genotype Frequencies - Myocarditis Yes vs Myocarditis No</title>

<script src="site_libs/header-attrs-2.21.2/header-attrs.js"></script>
<script src="site_libs/jquery-3.6.0/jquery-3.6.0.min.js"></script>
<meta name="viewport" content="width=device-width, initial-scale=1" />
<link href="site_libs/bootstrap-3.3.5/css/spacelab.min.css" rel="stylesheet" />
<script src="site_libs/bootstrap-3.3.5/js/bootstrap.min.js"></script>
<script src="site_libs/bootstrap-3.3.5/shim/html5shiv.min.js"></script>
<script src="site_libs/bootstrap-3.3.5/shim/respond.min.js"></script>
<style>h1 {font-size: 34px;}
       h1.title {font-size: 38px;}
       h2 {font-size: 30px;}
       h3 {font-size: 24px;}
       h4 {font-size: 18px;}
       h5 {font-size: 16px;}
       h6 {font-size: 12px;}
       code {color: inherit; background-color: rgba(0, 0, 0, 0.04);}
       pre:not([class]) { background-color: white }</style>
<script src="site_libs/navigation-1.1/tabsets.js"></script>
<link href="site_libs/highlightjs-9.12.0/textmate.css" rel="stylesheet" />
<script src="site_libs/highlightjs-9.12.0/highlight.js"></script>
<link href="site_libs/font-awesome-6.2.1/css/all.min.css" rel="stylesheet" />
<link href="site_libs/font-awesome-6.2.1/css/v4-shims.min.css" rel="stylesheet" />

<link rel="icon" href="https://github.com/workflowr/workflowr-assets/raw/main/img/reproducible.png">
<!-- Add a small amount of space between sections. -->
<style type="text/css">
div.section {
  padding-top: 12px;
}
</style>



<style type="text/css">
  code{white-space: pre-wrap;}
  span.smallcaps{font-variant: small-caps;}
  span.underline{text-decoration: underline;}
  div.column{display: inline-block; vertical-align: top; width: 50%;}
  div.hanging-indent{margin-left: 1.5em; text-indent: -1.5em;}
  ul.task-list{list-style: none;}
    </style>

<style type="text/css">code{white-space: pre;}</style>
<script type="text/javascript">
if (window.hljs) {
  hljs.configure({languages: []});
  hljs.initHighlightingOnLoad();
  if (document.readyState && document.readyState === "complete") {
    window.setTimeout(function() { hljs.initHighlighting(); }, 0);
  }
}
</script>









<style type = "text/css">
.main-container {
  max-width: 940px;
  margin-left: auto;
  margin-right: auto;
}
img {
  max-width:100%;
}
.tabbed-pane {
  padding-top: 12px;
}
.html-widget {
  margin-bottom: 20px;
}
button.code-folding-btn:focus {
  outline: none;
}
summary {
  display: list-item;
}
details > summary > p:only-child {
  display: inline;
}
pre code {
  padding: 0;
}
</style>


<style type="text/css">
.dropdown-submenu {
  position: relative;
}
.dropdown-submenu>.dropdown-menu {
  top: 0;
  left: 100%;
  margin-top: -6px;
  margin-left: -1px;
  border-radius: 0 6px 6px 6px;
}
.dropdown-submenu:hover>.dropdown-menu {
  display: block;
}
.dropdown-submenu>a:after {
  display: block;
  content: " ";
  float: right;
  width: 0;
  height: 0;
  border-color: transparent;
  border-style: solid;
  border-width: 5px 0 5px 5px;
  border-left-color: #cccccc;
  margin-top: 5px;
  margin-right: -10px;
}
.dropdown-submenu:hover>a:after {
  border-left-color: #adb5bd;
}
.dropdown-submenu.pull-left {
  float: none;
}
.dropdown-submenu.pull-left>.dropdown-menu {
  left: -100%;
  margin-left: 10px;
  border-radius: 6px 0 6px 6px;
}
</style>

<script type="text/javascript">
// manage active state of menu based on current page
$(document).ready(function () {
  // active menu anchor
  href = window.location.pathname
  href = href.substr(href.lastIndexOf('/') + 1)
  if (href === "")
    href = "index.html";
  var menuAnchor = $('a[href="' + href + '"]');

  // mark the anchor link active (and if it's in a dropdown, also mark that active)
  var dropdown = menuAnchor.closest('li.dropdown');
  if (window.bootstrap) { // Bootstrap 4+
    menuAnchor.addClass('active');
    dropdown.find('> .dropdown-toggle').addClass('active');
  } else { // Bootstrap 3
    menuAnchor.parent().addClass('active');
    dropdown.addClass('active');
  }

  // Navbar adjustments
  var navHeight = $(".navbar").first().height() + 15;
  var style = document.createElement('style');
  var pt = "padding-top: " + navHeight + "px; ";
  var mt = "margin-top: -" + navHeight + "px; ";
  var css = "";
  // offset scroll position for anchor links (for fixed navbar)
  for (var i = 1; i <= 6; i++) {
    css += ".section h" + i + "{ " + pt + mt + "}\n";
  }
  style.innerHTML = "body {" + pt + "padding-bottom: 40px; }\n" + css;
  document.head.appendChild(style);
});
</script>

<!-- tabsets -->

<style type="text/css">
.tabset-dropdown > .nav-tabs {
  display: inline-table;
  max-height: 500px;
  min-height: 44px;
  overflow-y: auto;
  border: 1px solid #ddd;
  border-radius: 4px;
}

.tabset-dropdown > .nav-tabs > li.active:before, .tabset-dropdown > .nav-tabs.nav-tabs-open:before {
  content: "\e259";
  font-family: 'Glyphicons Halflings';
  display: inline-block;
  padding: 10px;
  border-right: 1px solid #ddd;
}

.tabset-dropdown > .nav-tabs.nav-tabs-open > li.active:before {
  content: "\e258";
  font-family: 'Glyphicons Halflings';
  border: none;
}

.tabset-dropdown > .nav-tabs > li.active {
  display: block;
}

.tabset-dropdown > .nav-tabs > li > a,
.tabset-dropdown > .nav-tabs > li > a:focus,
.tabset-dropdown > .nav-tabs > li > a:hover {
  border: none;
  display: inline-block;
  border-radius: 4px;
  background-color: transparent;
}

.tabset-dropdown > .nav-tabs.nav-tabs-open > li {
  display: block;
  float: none;
}

.tabset-dropdown > .nav-tabs > li {
  display: none;
}
</style>

<!-- code folding -->




</head>

<body>


<div class="container-fluid main-container">




<div class="navbar navbar-default  navbar-fixed-top" role="navigation">
  <div class="container">
    <div class="navbar-header">
      <button type="button" class="navbar-toggle collapsed" data-toggle="collapse" data-bs-toggle="collapse" data-target="#navbar" data-bs-target="#navbar">
        <span class="icon-bar"></span>
        <span class="icon-bar"></span>
        <span class="icon-bar"></span>
      </button>
      <a class="navbar-brand" href="index.html">Serreze Autoimmune Workflow</a>
    </div>
    <div id="navbar" class="navbar-collapse collapse">
      <ul class="nav navbar-nav">
        <li>
  <a href="index.html">Home</a>
</li>
<li>
  <a href="about.html">About</a>
</li>
<li>
  <a href="license.html">License</a>
</li>
      </ul>
      <ul class="nav navbar-nav navbar-right">
        <li>
  <a href="https://github.com/TheJacksonLaboratory/Serreze-T1D_Workflow">
    <span class="fa fa-github"></span>
     
    Source code
  </a>
</li>
      </ul>
    </div><!--/.nav-collapse -->
  </div><!--/.container -->
</div><!--/.navbar -->

<div id="header">



<h1 class="title toc-ignore">Genotype Frequencies - Myocarditis Yes vs
Myocarditis No</h1>
<h4 class="author">Belinda Cornes</h4>
<h4 class="date">2023-04-16</h4>

</div>

<div id="TOC">
<ul>
<li><a href="#allele-frequencies" id="toc-allele-frequencies">Allele
Frequencies</a>
<ul>
<li><a href="#all-variants-before-snp-qc"
id="toc-all-variants-before-snp-qc">All variants (before snp
qc)</a></li>
<li><a href="#after-snp-qc" id="toc-after-snp-qc">After SNP QC</a></li>
</ul></li>
<li><a href="#with-sample-outliers" id="toc-with-sample-outliers">with
sample outliers</a></li>
<li><a href="#sample-outliers-removed"
id="toc-sample-outliers-removed">sample outliers removed</a></li>
</ul>
</div>

<p>
<button type="button" class="btn btn-default btn-workflowr btn-workflowr-report" data-toggle="collapse" data-target="#workflowr-report">
<span class="glyphicon glyphicon-list" aria-hidden="true"></span>
workflowr <span class="glyphicon glyphicon-exclamation-sign text-danger"
aria-hidden="true"></span>
</button>
</p>
<div id="workflowr-report" class="collapse">
<ul class="nav nav-tabs">
<li class="active">
<a data-toggle="tab" href="#summary">Summary</a>
</li>
<li>
<a data-toggle="tab" href="#checks"> Checks <span
class="glyphicon glyphicon-exclamation-sign text-danger"
aria-hidden="true"></span> </a>
</li>
<li>
<a data-toggle="tab" href="#versions">Past versions</a>
</li>
</ul>
<div class="tab-content">
<div id="summary" class="tab-pane fade in active">
<p>
<strong>Last updated:</strong> 2023-04-16
</p>
<p>
<strong>Checks:</strong> <span
class="glyphicon glyphicon-ok text-success" aria-hidden="true"></span> 5
<span class="glyphicon glyphicon-exclamation-sign text-danger"
aria-hidden="true"></span> 2
</p>
<p>
<strong>Knit directory:</strong> <code>Serreze-T1D_Workflow/</code>
<span class="glyphicon glyphicon-question-sign" aria-hidden="true"
title="This is the local directory in which the code in this file was executed.">
</span>
</p>
<p>
This reproducible <a href="https://rmarkdown.rstudio.com">R Markdown</a>
analysis was created with <a
  href="https://github.com/workflowr/workflowr">workflowr</a> (version
1.7.0). The <em>Checks</em> tab describes the reproducibility checks
that were applied when the results were created. The <em>Past
versions</em> tab lists the development history.
</p>
<hr>
</div>
<div id="checks" class="tab-pane fade">
<div id="workflowr-checks" class="panel-group">
<div class="panel panel-default">
<div class="panel-heading">
<p class="panel-title">
<a data-toggle="collapse" data-parent="#workflowr-checks" href="#strongRMarkdownfilestronguncommittedchanges">
<span class="glyphicon glyphicon-exclamation-sign text-danger"
aria-hidden="true"></span> <strong>R Markdown file:</strong> uncommitted
changes </a>
</p>
</div>
<div id="strongRMarkdownfilestronguncommittedchanges"
class="panel-collapse collapse">
<div class="panel-body">
<p>The R Markdown is untracked by Git. To know which version of the R
Markdown file created these results, youâ€™ll want to first commit it to
the Git repo. If youâ€™re still working on the analysis, you can ignore
this warning. When youâ€™re finished, you can run
<code>wflow_publish</code> to commit the R Markdown file and build the
HTML.</p>
</div>
</div>
</div>
<div class="panel panel-default">
<div class="panel-heading">
<p class="panel-title">
<a data-toggle="collapse" data-parent="#workflowr-checks" href="#strongEnvironmentstrongempty">
<span class="glyphicon glyphicon-ok text-success"
aria-hidden="true"></span> <strong>Environment:</strong> empty </a>
</p>
</div>
<div id="strongEnvironmentstrongempty" class="panel-collapse collapse">
<div class="panel-body">
<p>Great job! The global environment was empty. Objects defined in the
global environment can affect the analysis in your R Markdown file in
unknown ways. For reproduciblity itâ€™s best to always run the code in an
empty environment.</p>
</div>
</div>
</div>
<div class="panel panel-default">
<div class="panel-heading">
<p class="panel-title">
<a data-toggle="collapse" data-parent="#workflowr-checks" href="#strongSeedstrongcodesetseed20220210code">
<span class="glyphicon glyphicon-ok text-success"
aria-hidden="true"></span> <strong>Seed:</strong>
<code>set.seed(20220210)</code> </a>
</p>
</div>
<div id="strongSeedstrongcodesetseed20220210code"
class="panel-collapse collapse">
<div class="panel-body">
<p>The command <code>set.seed(20220210)</code> was run prior to running
the code in the R Markdown file. Setting a seed ensures that any results
that rely on randomness, e.g. subsampling or permutations, are
reproducible.</p>
</div>
</div>
</div>
<div class="panel panel-default">
<div class="panel-heading">
<p class="panel-title">
<a data-toggle="collapse" data-parent="#workflowr-checks" href="#strongSessioninformationstrongrecorded">
<span class="glyphicon glyphicon-ok text-success"
aria-hidden="true"></span> <strong>Session information:</strong>
recorded </a>
</p>
</div>
<div id="strongSessioninformationstrongrecorded"
class="panel-collapse collapse">
<div class="panel-body">
<p>Great job! Recording the operating system, R version, and package
versions is critical for reproducibility.</p>
</div>
</div>
</div>
<div class="panel panel-default">
<div class="panel-heading">
<p class="panel-title">
<a data-toggle="collapse" data-parent="#workflowr-checks" href="#strongCachestrongnone">
<span class="glyphicon glyphicon-ok text-success"
aria-hidden="true"></span> <strong>Cache:</strong> none </a>
</p>
</div>
<div id="strongCachestrongnone" class="panel-collapse collapse">
<div class="panel-body">
<p>Nice! There were no cached chunks for this analysis, so you can be
confident that you successfully produced the results during this
run.</p>
</div>
</div>
</div>
<div class="panel panel-default">
<div class="panel-heading">
<p class="panel-title">
<a data-toggle="collapse" data-parent="#workflowr-checks" href="#strongFilepathsstrongabsolute">
<span class="glyphicon glyphicon-exclamation-sign text-danger"
aria-hidden="true"></span> <strong>File paths:</strong> absolute </a>
</p>
</div>
<div id="strongFilepathsstrongabsolute" class="panel-collapse collapse">
<div class="panel-body">
<p>
Using absolute paths to the files within your workflowr project makes it
difficult for you and others to run your code on a different machine.
Change the absolute path(s) below to the suggested relative path(s) to
make your code more reproducible.
</p>
<table class="table table-condensed table-hover">
<thead>
<tr>
<th style="text-align:left;">
absolute
</th>
<th style="text-align:left;">
relative
</th>
</tr>
</thead>
<tbody>
<tr>
<td style="text-align:left;">
/Users/corneb/Documents/MyJax/CS/Projects/Serreze/qc/workflowr/Serreze-T1D_Workflow
</td>
<td style="text-align:left;">
.
</td>
</tr>
</tbody>
</table>
</div>
</div>
</div>
<div class="panel panel-default">
<div class="panel-heading">
<p class="panel-title">
<a data-toggle="collapse" data-parent="#workflowr-checks" href="#strongRepositoryversionstrongahrefhttpsgithubcomTheJacksonLaboratorySerrezeT1DWorkflowtree51e3845997bafb634294c1746dc74344f9838d41targetblank51e3845a">
<span class="glyphicon glyphicon-ok text-success"
aria-hidden="true"></span> <strong>Repository version:</strong>
<a href="https://github.com/TheJacksonLaboratory/Serreze-T1D_Workflow/tree/51e3845997bafb634294c1746dc74344f9838d41" target="_blank">51e3845</a>
</a>
</p>
</div>
<div
id="strongRepositoryversionstrongahrefhttpsgithubcomTheJacksonLaboratorySerrezeT1DWorkflowtree51e3845997bafb634294c1746dc74344f9838d41targetblank51e3845a"
class="panel-collapse collapse">
<div class="panel-body">
<p>
Great! You are using Git for version control. Tracking code development
and connecting the code version to the results is critical for
reproducibility.
</p>
<p>
The results in this page were generated with repository version
<a href="https://github.com/TheJacksonLaboratory/Serreze-T1D_Workflow/tree/51e3845997bafb634294c1746dc74344f9838d41" target="_blank">51e3845</a>.
See the <em>Past versions</em> tab to see a history of the changes made
to the R Markdown and HTML files.
</p>
<p>
Note that you need to be careful to ensure that all relevant files for
the analysis have been committed to Git prior to generating the results
(you can use <code>wflow_publish</code> or
<code>wflow_git_commit</code>). workflowr only checks the R Markdown
file, but you know if there are other scripts or data files that it
depends on. Below is the status of the Git repository when the results
were generated:
</p>
<pre><code>
Ignored files:
    Ignored:    .DS_Store
    Ignored:    analysis/.DS_Store

Untracked files:
    Untracked:  analysis/0.1.1_preparing.data_bqc_7.batches_myo.R
    Untracked:  analysis/0.1.1_preparing.data_bqc_7.batches_myo.Rmd
    Untracked:  analysis/0.1_samples_batch_20230124.Rmd
    Untracked:  analysis/0.1_samples_batch_20230225.Rmd
    Untracked:  analysis/0.1_samples_batch_20230313.Rmd
    Untracked:  analysis/0.2_haplotype_comparison_bqc_7.batches_myo_minprob.R
    Untracked:  analysis/0.2_haplotype_comparison_bqc_7.batches_myo_minprob.Rmd
    Untracked:  analysis/2.1_sample_bqc_7.batches_myo.R
    Untracked:  analysis/2.1_sample_bqc_7.batches_myo.Rmd
    Untracked:  analysis/2.1_sample_bqc_7.batches_myo.Rmd.R
    Untracked:  analysis/2.2.1_snp_qc_7.batches_myo.R
    Untracked:  analysis/2.2.1_snp_qc_7.batches_myo.Rmd
    Untracked:  analysis/2.2.1_snp_qc_7.batches_myo_mis.R
    Untracked:  analysis/2.2.1_snp_qc_7.batches_myo_mis.Rmd
    Untracked:  analysis/2.4_preparing.data_aqc_7.batches_myo.Rmd
    Untracked:  analysis/2.4_preparing.data_aqc_7.batches_myo_mis.Rmd
    Untracked:  analysis/2.4_preparing.data_aqc_7.batches_myo_misd.R
    Untracked:  analysis/3.1_phenotype.qc_corrected_7.batches_myo.Rmd
    Untracked:  analysis/3.1_phenotype.qc_corrected_7.batches_myo_mis.Rmd
    Untracked:  analysis/4.1.1_qtl.analysis_binary_het-ici-myo-yes.vs.het-ici-myo-no_snpsqc_7.batches_myo.R
    Untracked:  analysis/4.1.1_qtl.analysis_binary_het-ici-myo-yes.vs.het-ici-myo-no_snpsqc_7.batches_myo.Rmd
    Untracked:  analysis/4.1.1_qtl.analysis_binary_het-ici-myo-yes.vs.het-ici-myo-no_snpsqc_7.batches_myo_mis.R
    Untracked:  analysis/4.1.1_qtl.analysis_binary_het-ici-myo-yes.vs.het-ici-myo-no_snpsqc_7.batches_myo_mis.Rmd
    Untracked:  analysis/4.1.1_qtl.analysis_binary_het-ici-myo-yes.vs.het-ici-myo-no_snpsqc_dis_no-x_updated_7.batches_myo.R
    Untracked:  analysis/4.1.1_qtl.analysis_binary_het-ici-myo-yes.vs.het-ici-myo-no_snpsqc_dis_no-x_updated_7.batches_myo.Rmd
    Untracked:  analysis/4.1.1_qtl.analysis_binary_het-ici-myo-yes.vs.het-ici-myo-no_snpsqc_dis_no-x_updated_7.batches_myo_mis.Rmd
    Untracked:  analysis/4.1.1_qtl.analysis_binary_het-ici-myo-yes.vs.het-ici-myo-no_snpsqc_dis_no-x_updated_no-sex_7.batches_myo.Rmd
    Untracked:  analysis/4.1.1_qtl.analysis_binary_het-ici-myo-yes.vs.het-ici-myo-no_snpsqc_dis_no-x_updated_no-sex_7.batches_myo_mis.Rmd
    Untracked:  analysis/4.1.1_qtl.analysis_binary_het-ici-myo-yes.vs.het-ici-myo-no_snpsqc_no-sex_7.batches_myo.R
    Untracked:  analysis/4.1.1_qtl.analysis_binary_het-ici-myo-yes.vs.het-ici-myo-no_snpsqc_no-sex_7.batches_myo.Rmd
    Untracked:  analysis/4.1.1_qtl.analysis_binary_het-ici-myo-yes.vs.het-ici-myo-no_snpsqc_no-sex_7.batches_myo_mis.R
    Untracked:  analysis/4.1.1_qtl.analysis_binary_het-ici-myo-yes.vs.het-ici-myo-no_snpsqc_no-sex_7.batches_myo_mis.Rmd
    Untracked:  analysis/4.1.1_qtl.analysis_binary_het-ici.vs.het-pbs_snpsqc_7.batches_myo.R
    Untracked:  analysis/4.1.1_qtl.analysis_binary_het-ici.vs.het-pbs_snpsqc_7.batches_myo.Rmd
    Untracked:  analysis/4.1.1_qtl.analysis_binary_het-ici.vs.het-pbs_snpsqc_7.batches_myo_mis.R
    Untracked:  analysis/4.1.1_qtl.analysis_binary_het-ici.vs.het-pbs_snpsqc_7.batches_myo_mis.Rmd
    Untracked:  analysis/4.1.1_qtl.analysis_binary_het-ici.vs.het-pbs_snpsqc_dis_no-x_updated_7.batches_myo.R
    Untracked:  analysis/4.1.1_qtl.analysis_binary_het-ici.vs.het-pbs_snpsqc_dis_no-x_updated_7.batches_myo.Rmd
    Untracked:  analysis/4.1.1_qtl.analysis_binary_het-ici.vs.het-pbs_snpsqc_dis_no-x_updated_7.batches_myo_mis.Rmd
    Untracked:  analysis/4.1.1_qtl.analysis_binary_het-ici.vs.het-pbs_snpsqc_dis_no-x_updated_no-sex_7.batches_myo.Rmd
    Untracked:  analysis/4.1.1_qtl.analysis_binary_het-ici.vs.het-pbs_snpsqc_dis_no-x_updated_no-sex_7.batches_myo_mis.Rmd
    Untracked:  analysis/4.1.1_qtl.analysis_binary_het-ici.vs.het-pbs_snpsqc_no-sex_7.batches_myo.R
    Untracked:  analysis/4.1.1_qtl.analysis_binary_het-ici.vs.het-pbs_snpsqc_no-sex_7.batches_myo.Rmd
    Untracked:  analysis/4.1.1_qtl.analysis_binary_het-ici.vs.het-pbs_snpsqc_no-sex_7.batches_myo_mis.R
    Untracked:  analysis/4.1.1_qtl.analysis_binary_het-ici.vs.het-pbs_snpsqc_no-sex_7.batches_myo_mis.Rmd
    Untracked:  analysis/4.1.1_qtl.analysis_binary_ici-myo-yes.vs.ici-myo-no_snpsqc_7.batches_myo.R
    Untracked:  analysis/4.1.1_qtl.analysis_binary_ici-myo-yes.vs.ici-myo-no_snpsqc_7.batches_myo.Rmd
    Untracked:  analysis/4.1.1_qtl.analysis_binary_ici-myo-yes.vs.ici-myo-no_snpsqc_7.batches_myo_mis.R
    Untracked:  analysis/4.1.1_qtl.analysis_binary_ici-myo-yes.vs.ici-myo-no_snpsqc_7.batches_myo_mis.Rmd
    Untracked:  analysis/4.1.1_qtl.analysis_binary_ici-myo-yes.vs.ici-myo-no_snpsqc_dis_no-x_updated_7.batches_myo.R
    Untracked:  analysis/4.1.1_qtl.analysis_binary_ici-myo-yes.vs.ici-myo-no_snpsqc_dis_no-x_updated_7.batches_myo.Rmd
    Untracked:  analysis/4.1.1_qtl.analysis_binary_ici-myo-yes.vs.ici-myo-no_snpsqc_dis_no-x_updated_7.batches_myo_mis.Rmd
    Untracked:  analysis/4.1.1_qtl.analysis_binary_ici-myo-yes.vs.ici-myo-no_snpsqc_dis_no-x_updated_no-sex_7.batches_myo.Rmd
    Untracked:  analysis/4.1.1_qtl.analysis_binary_ici-myo-yes.vs.ici-myo-no_snpsqc_dis_no-x_updated_no-sex_7.batches_myo_mis.Rmd
    Untracked:  analysis/4.1.1_qtl.analysis_binary_ici-myo-yes.vs.ici-myo-no_snpsqc_no-sex_7.batches_myo.R
    Untracked:  analysis/4.1.1_qtl.analysis_binary_ici-myo-yes.vs.ici-myo-no_snpsqc_no-sex_7.batches_myo.Rmd
    Untracked:  analysis/4.1.1_qtl.analysis_binary_ici-myo-yes.vs.ici-myo-no_snpsqc_no-sex_7.batches_myo_mis.R
    Untracked:  analysis/4.1.1_qtl.analysis_binary_ici-myo-yes.vs.ici-myo-no_snpsqc_no-sex_7.batches_myo_mis.Rmd
    Untracked:  analysis/4.1.1_qtl.analysis_binary_ici-sick.vs.ici-eoi_snpsqc_7.batches_myo.R
    Untracked:  analysis/4.1.1_qtl.analysis_binary_ici-sick.vs.ici-eoi_snpsqc_7.batches_myo.Rmd
    Untracked:  analysis/4.1.1_qtl.analysis_binary_ici-sick.vs.ici-eoi_snpsqc_7.batches_myo_mis.R
    Untracked:  analysis/4.1.1_qtl.analysis_binary_ici-sick.vs.ici-eoi_snpsqc_7.batches_myo_mis.Rmd
    Untracked:  analysis/4.1.1_qtl.analysis_binary_ici-sick.vs.ici-eoi_snpsqc_dis_no-x_updated_7.batches_myo.R
    Untracked:  analysis/4.1.1_qtl.analysis_binary_ici-sick.vs.ici-eoi_snpsqc_dis_no-x_updated_7.batches_myo.Rmd
    Untracked:  analysis/4.1.1_qtl.analysis_binary_ici-sick.vs.ici-eoi_snpsqc_dis_no-x_updated_7.batches_myo_mis.Rmd
    Untracked:  analysis/4.1.1_qtl.analysis_binary_ici-sick.vs.ici-eoi_snpsqc_dis_no-x_updated_no-sex_7.batches_myo.Rmd
    Untracked:  analysis/4.1.1_qtl.analysis_binary_ici-sick.vs.ici-eoi_snpsqc_dis_no-x_updated_no-sex_7.batches_myo_mis.Rmd
    Untracked:  analysis/4.1.1_qtl.analysis_binary_ici-sick.vs.ici-eoi_snpsqc_no-sex_7.batches_myo.R
    Untracked:  analysis/4.1.1_qtl.analysis_binary_ici-sick.vs.ici-eoi_snpsqc_no-sex_7.batches_myo.Rmd
    Untracked:  analysis/4.1.1_qtl.analysis_binary_ici-sick.vs.ici-eoi_snpsqc_no-sex_7.batches_myo_mis.R
    Untracked:  analysis/4.1.1_qtl.analysis_binary_ici-sick.vs.ici-eoi_snpsqc_no-sex_7.batches_myo_mis.Rmd
    Untracked:  analysis/4.1.1_qtl.analysis_binary_ici.vs.pbs_snpsqc_7.batches_myo.R
    Untracked:  analysis/4.1.1_qtl.analysis_binary_ici.vs.pbs_snpsqc_7.batches_myo.Rmd
    Untracked:  analysis/4.1.1_qtl.analysis_binary_ici.vs.pbs_snpsqc_7.batches_myo_mis.R
    Untracked:  analysis/4.1.1_qtl.analysis_binary_ici.vs.pbs_snpsqc_7.batches_myo_mis.Rmd
    Untracked:  analysis/4.1.1_qtl.analysis_binary_ici.vs.pbs_snpsqc_dis_no-x_updated_7.batches_myo.R
    Untracked:  analysis/4.1.1_qtl.analysis_binary_ici.vs.pbs_snpsqc_dis_no-x_updated_7.batches_myo.Rmd
    Untracked:  analysis/4.1.1_qtl.analysis_binary_ici.vs.pbs_snpsqc_dis_no-x_updated_7.batches_myo_mis.Rmd
    Untracked:  analysis/4.1.1_qtl.analysis_binary_ici.vs.pbs_snpsqc_dis_no-x_updated_no-sex_7.batches_myo.Rmd
    Untracked:  analysis/4.1.1_qtl.analysis_binary_ici.vs.pbs_snpsqc_dis_no-x_updated_no-sex_7.batches_myo_mis.Rmd
    Untracked:  analysis/4.1.1_qtl.analysis_binary_ici.vs.pbs_snpsqc_no-sex_7.batches_myo.R
    Untracked:  analysis/4.1.1_qtl.analysis_binary_ici.vs.pbs_snpsqc_no-sex_7.batches_myo.Rmd
    Untracked:  analysis/4.1.1_qtl.analysis_binary_ici.vs.pbs_snpsqc_no-sex_7.batches_myo_mis.R
    Untracked:  analysis/4.1.1_qtl.analysis_binary_ici.vs.pbs_snpsqc_no-sex_7.batches_myo_mis.Rmd
    Untracked:  analysis/4.1.1_qtl.analysis_binary_myo-yes.vs.myo-no_snpsqc_7.batches_myo.R
    Untracked:  analysis/4.1.1_qtl.analysis_binary_myo-yes.vs.myo-no_snpsqc_7.batches_myo.Rmd
    Untracked:  analysis/4.1.1_qtl.analysis_binary_myo-yes.vs.myo-no_snpsqc_7.batches_myo_mis.R
    Untracked:  analysis/4.1.1_qtl.analysis_binary_myo-yes.vs.myo-no_snpsqc_7.batches_myo_mis.Rmd
    Untracked:  analysis/4.1.1_qtl.analysis_binary_myo-yes.vs.myo-no_snpsqc_dis_no-x_updated_7.batches_myo.R
    Untracked:  analysis/4.1.1_qtl.analysis_binary_myo-yes.vs.myo-no_snpsqc_dis_no-x_updated_7.batches_myo.Rmd
    Untracked:  analysis/4.1.1_qtl.analysis_binary_myo-yes.vs.myo-no_snpsqc_dis_no-x_updated_7.batches_myo_mis.Rmd
    Untracked:  analysis/4.1.1_qtl.analysis_binary_myo-yes.vs.myo-no_snpsqc_dis_no-x_updated_no-sex_7.batches_myo.Rmd
    Untracked:  analysis/4.1.1_qtl.analysis_binary_myo-yes.vs.myo-no_snpsqc_dis_no-x_updated_no-sex_7.batches_myo_mis.Rmd
    Untracked:  analysis/4.1.1_qtl.analysis_binary_myo-yes.vs.myo-no_snpsqc_no-sex_7.batches_myo.R
    Untracked:  analysis/4.1.1_qtl.analysis_binary_myo-yes.vs.myo-no_snpsqc_no-sex_7.batches_myo.Rmd
    Untracked:  analysis/4.1.1_qtl.analysis_binary_myo-yes.vs.myo-no_snpsqc_no-sex_7.batches_myo_mis.R
    Untracked:  analysis/4.1.1_qtl.analysis_binary_myo-yes.vs.myo-no_snpsqc_no-sex_7.batches_myo_mis.Rmd
    Untracked:  analysis/4.1.1_qtl.analysis_binary_pbs-myo-yes.vs.pbs-myo-no_snpsqc_7.batches_myo.R
    Untracked:  analysis/4.1.1_qtl.analysis_binary_pbs-myo-yes.vs.pbs-myo-no_snpsqc_7.batches_myo.Rmd
    Untracked:  analysis/4.1.1_qtl.analysis_binary_pbs-myo-yes.vs.pbs-myo-no_snpsqc_7.batches_myo_mis.R
    Untracked:  analysis/4.1.1_qtl.analysis_binary_pbs-myo-yes.vs.pbs-myo-no_snpsqc_7.batches_myo_mis.Rmd
    Untracked:  analysis/4.1.1_qtl.analysis_binary_pbs-myo-yes.vs.pbs-myo-no_snpsqc_dis_no-x_updated_7.batches_myo.R
    Untracked:  analysis/4.1.1_qtl.analysis_binary_pbs-myo-yes.vs.pbs-myo-no_snpsqc_dis_no-x_updated_7.batches_myo.Rmd
    Untracked:  analysis/4.1.1_qtl.analysis_binary_pbs-myo-yes.vs.pbs-myo-no_snpsqc_dis_no-x_updated_7.batches_myo_mis.Rmd
    Untracked:  analysis/4.1.1_qtl.analysis_binary_pbs-myo-yes.vs.pbs-myo-no_snpsqc_dis_no-x_updated_no-sex_7.batches_myo.Rmd
    Untracked:  analysis/4.1.1_qtl.analysis_binary_pbs-myo-yes.vs.pbs-myo-no_snpsqc_dis_no-x_updated_no-sex_7.batches_myo_mis.Rmd
    Untracked:  analysis/4.1.1_qtl.analysis_binary_pbs-myo-yes.vs.pbs-myo-no_snpsqc_no-sex_7.batches_myo.R
    Untracked:  analysis/4.1.1_qtl.analysis_binary_pbs-myo-yes.vs.pbs-myo-no_snpsqc_no-sex_7.batches_myo.Rmd
    Untracked:  analysis/4.1.1_qtl.analysis_binary_pbs-myo-yes.vs.pbs-myo-no_snpsqc_no-sex_7.batches_myo_mis.R
    Untracked:  analysis/4.1.1_qtl.analysis_binary_pbs-myo-yes.vs.pbs-myo-no_snpsqc_no-sex_7.batches_myo_mis.Rmd
    Untracked:  analysis/debugging_comparing_old_Rmd/
    Untracked:  analysis/frequency.plot.R
    Untracked:  analysis/genotype.frequencies_het-ici-myo-yes.vs.het-ici-myo-no_7.batches_myo.Rmd
    Untracked:  analysis/genotype.frequencies_het-ici-myo-yes.vs.het-ici-myo-no_7.batches_myo_mis.Rmd
    Untracked:  analysis/genotype.frequencies_het-ici-myo-yes.vs.het-ici-myo-no_freq_7.batches_myo.Rmd
    Untracked:  analysis/genotype.frequencies_het-ici-myo-yes.vs.het-ici-myo-no_freq_7.batches_myo_mis.Rmd
    Untracked:  analysis/genotype.frequencies_het-ici.vs.het-pbs_7.batches_myo.Rmd
    Untracked:  analysis/genotype.frequencies_het-ici.vs.het-pbs_7.batches_myo_mis.Rmd
    Untracked:  analysis/genotype.frequencies_het-ici.vs.het-pbs_freq_7.batches_myo.Rmd
    Untracked:  analysis/genotype.frequencies_het-ici.vs.het-pbs_freq_7.batches_myo_mis.Rmd
    Untracked:  analysis/genotype.frequencies_ici-myo-yes.vs.ici-myo-no_7.batches_myo.Rmd
    Untracked:  analysis/genotype.frequencies_ici-myo-yes.vs.ici-myo-no_7.batches_myo_mis.Rmd
    Untracked:  analysis/genotype.frequencies_ici-myo-yes.vs.ici-myo-no_freq_7.batches_myo.Rmd
    Untracked:  analysis/genotype.frequencies_ici-myo-yes.vs.ici-myo-no_freq_7.batches_myo_mis.Rmd
    Untracked:  analysis/genotype.frequencies_ici-sick.vs.ici-eoi_7.batches_myo.Rmd
    Untracked:  analysis/genotype.frequencies_ici-sick.vs.ici-eoi_7.batches_myo.Rmd.r
    Untracked:  analysis/genotype.frequencies_ici-sick.vs.ici-eoi_7.batches_myo_edit.Rmd
    Untracked:  analysis/genotype.frequencies_ici-sick.vs.ici-eoi_7.batches_myo_mis.Rmd
    Untracked:  analysis/genotype.frequencies_ici-sick.vs.ici-eoi_freq_7.batches_myo.Rmd
    Untracked:  analysis/genotype.frequencies_ici-sick.vs.ici-eoi_freq_7.batches_myo_mis.Rmd
    Untracked:  analysis/genotype.frequencies_ici.vs.pbs_7.batches_myo.Rmd
    Untracked:  analysis/genotype.frequencies_ici.vs.pbs_7.batches_myo_mis.Rmd
    Untracked:  analysis/genotype.frequencies_ici.vs.pbs_freq_7.batches_myo.Rmd
    Untracked:  analysis/genotype.frequencies_ici.vs.pbs_freq_7.batches_myo_mis.Rmd
    Untracked:  analysis/genotype.frequencies_myo-yes.vs.myo-no_7.batches_myo.Rmd
    Untracked:  analysis/genotype.frequencies_myo-yes.vs.myo-no_7.batches_myo_mis.Rmd
    Untracked:  analysis/genotype.frequencies_myo-yes.vs.myo-no_freq_7.batches_myo.Rmd
    Untracked:  analysis/genotype.frequencies_myo-yes.vs.myo-no_freq_7.batches_myo_mis.Rmd
    Untracked:  analysis/genotype.frequencies_pbs-myo-yes.vs.pbs-myo-no_7.batches_myo.Rmd
    Untracked:  analysis/genotype.frequencies_pbs-myo-yes.vs.pbs-myo-no_7.batches_myo_mis.Rmd
    Untracked:  analysis/genotype.frequencies_pbs-myo-yes.vs.pbs-myo-no_freq_7.batches_myo.Rmd
    Untracked:  analysis/genotype.frequencies_pbs-myo-yes.vs.pbs-myo-no_freq_7.batches_myo_mis.Rmd
    Untracked:  analysis/index_7.batches_myo.Rmd
    Untracked:  analysis/power.analysis_het-ici-myo-yes.vs.het-ici-myo-no_7.batches_myo.Rmd
    Untracked:  analysis/power.analysis_het-ici-myo-yes.vs.het-ici-myo-no_7.batches_myo_mis.Rmd
    Untracked:  analysis/power.analysis_het-ici.vs.het-pbs_7.batches_myo.Rmd
    Untracked:  analysis/power.analysis_het-ici.vs.het-pbs_7.batches_myo_mis.Rmd
    Untracked:  analysis/power.analysis_ici-myo-yes.vs.ici-myo-no_7.batches_myo.Rmd
    Untracked:  analysis/power.analysis_ici-myo-yes.vs.ici-myo-no_7.batches_myo_mis.Rmd
    Untracked:  analysis/power.analysis_ici-sick.vs.ici-eoi_7.batches_myo.Rmd
    Untracked:  analysis/power.analysis_ici-sick.vs.ici-eoi_7.batches_myo_mis.Rmd
    Untracked:  analysis/power.analysis_ici.vs.pbs_7.batches_myo.Rmd
    Untracked:  analysis/power.analysis_ici.vs.pbs_7batches_myo_mis.Rmd
    Untracked:  analysis/power.analysis_myo-yes.vs.myo-no_7.batches_myo.Rmd
    Untracked:  analysis/power.analysis_myo-yes.vs.myo-no_7.batches_myo_mis.Rmd
    Untracked:  analysis/power.analysis_pbs-myo-yes.vs.pbs-myo-no_7.batches_myo.Rmd
    Untracked:  analysis/power.analysis_pbs-myo-yes.vs.pbs-myo-no_7.batches_myo_mis.Rmd
    Untracked:  data/GM_covar_4.batches_myo.csv
    Untracked:  data/GM_covar_7.batches_myo.csv
    Untracked:  data/bad_markers_all_4.batches_myo.RData
    Untracked:  data/bad_markers_all_7.batches_myo.RData
    Untracked:  data/covar_corrected.cleaned_het-ici-myo-yes.vs.het-ici-myo-no_7.batches_myo.csv
    Untracked:  data/covar_corrected.cleaned_het-ici-myo-yes.vs.het-ici-myo-no_7.batches_myo_mis.csv
    Untracked:  data/covar_corrected.cleaned_het-ici.vs.het-pbs_7.batches_myo.csv
    Untracked:  data/covar_corrected.cleaned_het-ici.vs.het-pbs_7.batches_myo_mis.csv
    Untracked:  data/covar_corrected.cleaned_ici-myo-yes.vs.ici-myo-no_7.batches_myo.csv
    Untracked:  data/covar_corrected.cleaned_ici-myo-yes.vs.ici-myo-no_7.batches_myo_mis.csv
    Untracked:  data/covar_corrected.cleaned_ici-sick.vs.ici-eoi_7.batches_myo.csv
    Untracked:  data/covar_corrected.cleaned_ici-sick.vs.ici-eoi_7.batches_myo_mis.csv
    Untracked:  data/covar_corrected.cleaned_ici.vs.pbs_7.batches_myo.csv
    Untracked:  data/covar_corrected.cleaned_ici.vs.pbs_7.batches_myo_mis.csv
    Untracked:  data/covar_corrected.cleaned_myo-yes.vs.myo-no_7.batches_myo.csv
    Untracked:  data/covar_corrected.cleaned_myo-yes.vs.myo-no_7.batches_myo_mis.csv
    Untracked:  data/covar_corrected.cleaned_pbs-myo-yes.vs.pbs-myo-no_7.batches_myo.csv
    Untracked:  data/covar_corrected.cleaned_pbs-myo-yes.vs.pbs-myo-no_7.batches_myo_mis.csv
    Untracked:  data/covar_corrected_het-ici-myo-yes.vs.het-ici-myo-no_7.batches_myo.csv
    Untracked:  data/covar_corrected_het-ici-myo-yes.vs.het-ici-myo-no_7.batches_myo_mis.csv
    Untracked:  data/covar_corrected_het-ici.vs.het-pbs_7.batches_myo.csv
    Untracked:  data/covar_corrected_het-ici.vs.het-pbs_7.batches_myo_mis.csv
    Untracked:  data/covar_corrected_ici-myo-yes.vs.ici-myo-no_7.batches_myo.csv
    Untracked:  data/covar_corrected_ici-myo-yes.vs.ici-myo-no_7.batches_myo_mis.csv
    Untracked:  data/covar_corrected_ici-sick.vs.ici-eoi_7.batches_myo.csv
    Untracked:  data/covar_corrected_ici-sick.vs.ici-eoi_7.batches_myo_mis.csv
    Untracked:  data/covar_corrected_ici.vs.pbs_7.batches_myo.csv
    Untracked:  data/covar_corrected_ici.vs.pbs_7.batches_myo_mis.csv
    Untracked:  data/covar_corrected_myo-yes.vs.myo-no_7.batches_myo.csv
    Untracked:  data/covar_corrected_myo-yes.vs.myo-no_7.batches_myo_mis.csv
    Untracked:  data/covar_corrected_pbs-myo-yes.vs.pbs-myo-no_7.batches_myo.csv
    Untracked:  data/covar_corrected_pbs-myo-yes.vs.pbs-myo-no_7.batches_myo_mis.csv
    Untracked:  data/e_4.batches_myo.RData
    Untracked:  data/e_7.batches_myo.RData
    Untracked:  data/e_snpg_samqc_4.batches_myo.RData
    Untracked:  data/e_snpg_samqc_7.batches_myo.RData
    Untracked:  data/errors_ind_4.batches_myo.RData
    Untracked:  data/errors_ind_7.batches_myo.RData
    Untracked:  data/genetic_map_4.batches_myo.csv
    Untracked:  data/genetic_map_7.batches_myo.csv
    Untracked:  data/genotype_errors_marker_4.batches_myo.RData
    Untracked:  data/genotype_errors_marker_7.batches_myo.RData
    Untracked:  data/genotype_freq_marker_4.batches_myo.RData
    Untracked:  data/genotype_freq_marker_7.batches_myo.RData
    Untracked:  data/gm_allqc_4.batches_myo.RData
    Untracked:  data/gm_allqc_4.batches_myo_mis.RData
    Untracked:  data/gm_allqc_7.batches_myo.RData
    Untracked:  data/gm_allqc_7.batches_myo_mis.RData
    Untracked:  data/gm_samqc_4.batches_myo.RData
    Untracked:  data/gm_samqc_7.batches_myo.RData
    Untracked:  data/gm_serreze.BC312.RData
    Untracked:  data/gm_serreze.BC366.RData
    Untracked:  data/gm_serreze_7.batches_myo.RData
    Untracked:  data/gm_serreze_bc_7.batches_myo.RData
    Untracked:  data/het-ici-myo-yes.vs.het-ici-myo-no_gm_qtl_snpsqc_dis_no-x_updated_7.batches_myo_mis.csv
    Untracked:  data/het-ici-myo-yes.vs.het-ici-myo-no_marker.freq_low.geno.freq.removed_geno.ratio_7.batches_myo.csv
    Untracked:  data/het-ici-myo-yes.vs.het-ici-myo-no_marker.freq_low.geno.freq.removed_geno.ratio_7.batches_myo_mis.csv
    Untracked:  data/het-ici-myo-yes.vs.het-ici-myo-no_marker.freq_low.geno.freq.removed_sample.outliers.removed_geno.ratio_7.batches_myo.csv
    Untracked:  data/het-ici-myo-yes.vs.het-ici-myo-no_marker.freq_low.geno.freq.removed_sample.outliers.removed_geno.ratio_7.batches_myo_mis.csv
    Untracked:  data/het-ici-myo-yes.vs.het-ici-myo-no_marker.freq_low.probs.freq.removed_geno.ratio_7.batches_myo.csv
    Untracked:  data/het-ici-myo-yes.vs.het-ici-myo-no_marker.freq_low.probs.freq.removed_geno.ratio_7.batches_myo_mis.csv
    Untracked:  data/het-ici-myo-yes.vs.het-ici-myo-no_marker.freq_low.probs.freq.removed_sample.outliers.removed_geno.ratio_7.batches_myo.csv
    Untracked:  data/het-ici-myo-yes.vs.het-ici-myo-no_marker.freq_low.probs.freq.removed_sample.outliers.removed_geno.ratio_7.batches_myo_mis.csv
    Untracked:  data/het-ici-myo-yes.vs.het-ici-myo-no_sample.genos_marker.freq_low.geno.freq.removed_7.batches_myo.csv
    Untracked:  data/het-ici-myo-yes.vs.het-ici-myo-no_sample.genos_marker.freq_low.geno.freq.removed_7.batches_myo_mis.csv
    Untracked:  data/het-ici-myo-yes.vs.het-ici-myo-no_sample.genos_marker.freq_low.geno.freq.removed_sample.outliers.removed_7.batches_myo.csv
    Untracked:  data/het-ici-myo-yes.vs.het-ici-myo-no_sample.genos_marker.freq_low.geno.freq.removed_sample.outliers.removed_7.batches_myo_mis.csv
    Untracked:  data/het-ici-myo-yes.vs.het-ici-myo-no_sample.genos_marker.freq_low.probs.freq.removed_7.batches_myo.csv
    Untracked:  data/het-ici-myo-yes.vs.het-ici-myo-no_sample.genos_marker.freq_low.probs.freq.removed_7.batches_myo_mis.csv
    Untracked:  data/het-ici-myo-yes.vs.het-ici-myo-no_sample.genos_marker.freq_low.probs.freq.removed_sample.outliers.removed_7.batches_myo.csv
    Untracked:  data/het-ici-myo-yes.vs.het-ici-myo-no_sample.genos_marker.freq_low.probs.freq.removed_sample.outliers.removed_7.batches_myo_mis.csv
    Untracked:  data/het-ici.vs.het-pbs_gm_qtl_snpsqc_dis_no-x_updated_7.batches_myo.csv
    Untracked:  data/het-ici.vs.het-pbs_gm_qtl_snpsqc_dis_no-x_updated_7.batches_myo_mis.csv
    Untracked:  data/het-ici.vs.het-pbs_marker.freq_low.geno.freq.removed_geno.ratio_7.batches_myo.csv
    Untracked:  data/het-ici.vs.het-pbs_marker.freq_low.geno.freq.removed_geno.ratio_7.batches_myo_mis.csv
    Untracked:  data/het-ici.vs.het-pbs_marker.freq_low.geno.freq.removed_sample.outliers.removed_geno.ratio_7.batches_myo.csv
    Untracked:  data/het-ici.vs.het-pbs_marker.freq_low.geno.freq.removed_sample.outliers.removed_geno.ratio_7.batches_myo_mis.csv
    Untracked:  data/het-ici.vs.het-pbs_marker.freq_low.probs.freq.removed_geno.ratio_7.batches_myo.csv
    Untracked:  data/het-ici.vs.het-pbs_marker.freq_low.probs.freq.removed_geno.ratio_7.batches_myo_mis.csv
    Untracked:  data/het-ici.vs.het-pbs_marker.freq_low.probs.freq.removed_sample.outliers.removed_geno.ratio_7.batches_myo.csv
    Untracked:  data/het-ici.vs.het-pbs_marker.freq_low.probs.freq.removed_sample.outliers.removed_geno.ratio_7.batches_myo_mis.csv
    Untracked:  data/het-ici.vs.het-pbs_sample.genos_marker.freq_low.geno.freq.removed_7.batches_myo.csv
    Untracked:  data/het-ici.vs.het-pbs_sample.genos_marker.freq_low.geno.freq.removed_7.batches_myo_mis.csv
    Untracked:  data/het-ici.vs.het-pbs_sample.genos_marker.freq_low.geno.freq.removed_sample.outliers.removed_7.batches_myo.csv
    Untracked:  data/het-ici.vs.het-pbs_sample.genos_marker.freq_low.geno.freq.removed_sample.outliers.removed_7.batches_myo_mis.csv
    Untracked:  data/het-ici.vs.het-pbs_sample.genos_marker.freq_low.probs.freq.removed_7.batches_myo.csv
    Untracked:  data/het-ici.vs.het-pbs_sample.genos_marker.freq_low.probs.freq.removed_7.batches_myo_mis.csv
    Untracked:  data/het-ici.vs.het-pbs_sample.genos_marker.freq_low.probs.freq.removed_sample.outliers.removed_7.batches_myo.csv
    Untracked:  data/het-ici.vs.het-pbs_sample.genos_marker.freq_low.probs.freq.removed_sample.outliers.removed_7.batches_myo_mis.csv
    Untracked:  data/ici-myo-yes.vs.ici-myo-no_gm_qtl_snpsqc_dis_no-x_updated_7.batches_myo.csv
    Untracked:  data/ici-myo-yes.vs.ici-myo-no_gm_qtl_snpsqc_dis_no-x_updated_7.batches_myo_mis.csv
    Untracked:  data/ici-myo-yes.vs.ici-myo-no_marker.freq_low.geno.freq.removed_geno.ratio_7.batches_myo.csv
    Untracked:  data/ici-myo-yes.vs.ici-myo-no_marker.freq_low.geno.freq.removed_geno.ratio_7.batches_myo_mis.csv
    Untracked:  data/ici-myo-yes.vs.ici-myo-no_marker.freq_low.geno.freq.removed_sample.outliers.removed_geno.ratio_7.batches_myo.csv
    Untracked:  data/ici-myo-yes.vs.ici-myo-no_marker.freq_low.geno.freq.removed_sample.outliers.removed_geno.ratio_7.batches_myo_mis.csv
    Untracked:  data/ici-myo-yes.vs.ici-myo-no_marker.freq_low.probs.freq.removed_geno.ratio_7.batches_myo.csv
    Untracked:  data/ici-myo-yes.vs.ici-myo-no_marker.freq_low.probs.freq.removed_geno.ratio_7.batches_myo_mis.csv
    Untracked:  data/ici-myo-yes.vs.ici-myo-no_marker.freq_low.probs.freq.removed_sample.outliers.removed_geno.ratio_7.batches_myo.csv
    Untracked:  data/ici-myo-yes.vs.ici-myo-no_marker.freq_low.probs.freq.removed_sample.outliers.removed_geno.ratio_7.batches_myo_mis.csv
    Untracked:  data/ici-myo-yes.vs.ici-myo-no_sample.genos_marker.freq_low.geno.freq.removed_7.batches_myo.csv
    Untracked:  data/ici-myo-yes.vs.ici-myo-no_sample.genos_marker.freq_low.geno.freq.removed_7.batches_myo_mis.csv
    Untracked:  data/ici-myo-yes.vs.ici-myo-no_sample.genos_marker.freq_low.geno.freq.removed_sample.outliers.removed_7.batches_myo.csv
    Untracked:  data/ici-myo-yes.vs.ici-myo-no_sample.genos_marker.freq_low.geno.freq.removed_sample.outliers.removed_7.batches_myo_mis.csv
    Untracked:  data/ici-myo-yes.vs.ici-myo-no_sample.genos_marker.freq_low.probs.freq.removed_7.batches_myo.csv
    Untracked:  data/ici-myo-yes.vs.ici-myo-no_sample.genos_marker.freq_low.probs.freq.removed_7.batches_myo_mis.csv
    Untracked:  data/ici-myo-yes.vs.ici-myo-no_sample.genos_marker.freq_low.probs.freq.removed_sample.outliers.removed_7.batches_myo.csv
    Untracked:  data/ici-myo-yes.vs.ici-myo-no_sample.genos_marker.freq_low.probs.freq.removed_sample.outliers.removed_7.batches_myo_mis.csv
    Untracked:  data/ici-sick.vs.ici-eoi_blup_sub_chr-10_peak.marker-UNC18343990_lod.drop-1.5_snpsqc_dis_no-x_updated_7.batches_myo.csv
    Untracked:  data/ici-sick.vs.ici-eoi_blup_sub_chr-10_peak.marker-UNC18343990_lod.drop-1.5_snpsqc_dis_no-x_updated_7.batches_myo_mis.csv
    Untracked:  data/ici-sick.vs.ici-eoi_blup_sub_chr-11_peak.marker-UNC20070077_lod.drop-1.5_snpsqc_dis_no-x_updated_7.batches_myo.csv
    Untracked:  data/ici-sick.vs.ici-eoi_blup_sub_chr-11_peak.marker-UNC20070077_lod.drop-1.5_snpsqc_dis_no-x_updated_7.batches_myo_mis.csv
    Untracked:  data/ici-sick.vs.ici-eoi_blup_sub_chr-12_peak.marker-UNC21652584_lod.drop-1.5_snpsqc_dis_no-x_updated_7.batches_myo.csv
    Untracked:  data/ici-sick.vs.ici-eoi_blup_sub_chr-12_peak.marker-UNC21652584_lod.drop-1.5_snpsqc_dis_no-x_updated_7.batches_myo_mis.csv
    Untracked:  data/ici-sick.vs.ici-eoi_blup_sub_chr-13_peak.marker-UNCHS036579_lod.drop-1.5_snpsqc_dis_no-x_updated_7.batches_myo.csv
    Untracked:  data/ici-sick.vs.ici-eoi_blup_sub_chr-13_peak.marker-UNCHS036579_lod.drop-1.5_snpsqc_dis_no-x_updated_7.batches_myo_mis.csv
    Untracked:  data/ici-sick.vs.ici-eoi_blup_sub_chr-14_peak.marker-UNCHS037782_lod.drop-1.5_snpsqc_dis_no-x_updated_7.batches_myo.csv
    Untracked:  data/ici-sick.vs.ici-eoi_blup_sub_chr-14_peak.marker-UNCHS037782_lod.drop-1.5_snpsqc_dis_no-x_updated_7.batches_myo_mis.csv
    Untracked:  data/ici-sick.vs.ici-eoi_blup_sub_chr-15_peak.marker-UNC26069905_lod.drop-1.5_snpsqc_dis_no-x_updated_7.batches_myo.csv
    Untracked:  data/ici-sick.vs.ici-eoi_blup_sub_chr-15_peak.marker-UNCHS041223_lod.drop-1.5_snpsqc_dis_no-x_updated_7.batches_myo_mis.csv
    Untracked:  data/ici-sick.vs.ici-eoi_blup_sub_chr-16_peak.marker-JAX00070117_lod.drop-1.5_snpsqc_dis_no-x_updated_7.batches_myo.csv
    Untracked:  data/ici-sick.vs.ici-eoi_blup_sub_chr-16_peak.marker-JAX00070117_lod.drop-1.5_snpsqc_dis_no-x_updated_7.batches_myo_mis.csv
    Untracked:  data/ici-sick.vs.ici-eoi_blup_sub_chr-17_peak.marker-UNC28542319_lod.drop-1.5_snpsqc_dis_no-x_updated_7.batches_myo_mis.csv
    Untracked:  data/ici-sick.vs.ici-eoi_blup_sub_chr-17_peak.marker-UNCJPD006670_lod.drop-1.5_snpsqc_dis_no-x_updated_7.batches_myo.csv
    Untracked:  data/ici-sick.vs.ici-eoi_blup_sub_chr-18_peak.marker-UNC28776739_lod.drop-1.5_snpsqc_dis_no-x_updated_7.batches_myo.csv
    Untracked:  data/ici-sick.vs.ici-eoi_blup_sub_chr-18_peak.marker-UNCHS045594_lod.drop-1.5_snpsqc_dis_no-x_updated_7.batches_myo_mis.csv
    Untracked:  data/ici-sick.vs.ici-eoi_blup_sub_chr-19_peak.marker-UNC30414168_lod.drop-1.5_snpsqc_dis_no-x_updated_7.batches_myo.csv
    Untracked:  data/ici-sick.vs.ici-eoi_blup_sub_chr-19_peak.marker-UNC30426276_lod.drop-1.5_snpsqc_dis_no-x_updated_7.batches_myo_mis.csv
    Untracked:  data/ici-sick.vs.ici-eoi_blup_sub_chr-1_peak.marker-UNC2031646_lod.drop-1.5_snpsqc_dis_no-x_updated_7.batches_myo.csv
    Untracked:  data/ici-sick.vs.ici-eoi_blup_sub_chr-1_peak.marker-UNCHS003700_lod.drop-1.5_snpsqc_dis_no-x_updated_7.batches_myo_mis.csv
    Untracked:  data/ici-sick.vs.ici-eoi_blup_sub_chr-2_peak.marker-ICR5131_lod.drop-1.5_snpsqc_dis_no-x_updated_7.batches_myo.csv
    Untracked:  data/ici-sick.vs.ici-eoi_blup_sub_chr-2_peak.marker-UNCHS006420_lod.drop-1.5_snpsqc_dis_no-x_updated_7.batches_myo_mis.csv
    Untracked:  data/ici-sick.vs.ici-eoi_blup_sub_chr-3_peak.marker-UNC5667757_lod.drop-1.5_snpsqc_dis_no-x_updated_7.batches_myo.csv
    Untracked:  data/ici-sick.vs.ici-eoi_blup_sub_chr-3_peak.marker-UNC5667757_lod.drop-1.5_snpsqc_dis_no-x_updated_7.batches_myo_mis.csv
    Untracked:  data/ici-sick.vs.ici-eoi_blup_sub_chr-4_peak.marker-UNC6759992_lod.drop-1.5_snpsqc_dis_no-x_updated_7.batches_myo_mis.csv
    Untracked:  data/ici-sick.vs.ici-eoi_blup_sub_chr-4_peak.marker-UNC6765178_lod.drop-1.5_snpsqc_dis_no-x_updated_7.batches_myo.csv
    Untracked:  data/ici-sick.vs.ici-eoi_blup_sub_chr-5_peak.marker-UNC9889957_lod.drop-1.5_snpsqc_dis_no-x_updated_7.batches_myo_mis.csv
    Untracked:  data/ici-sick.vs.ici-eoi_blup_sub_chr-5_peak.marker-UNC9900273_lod.drop-1.5_snpsqc_dis_no-x_updated_7.batches_myo.csv
    Untracked:  data/ici-sick.vs.ici-eoi_blup_sub_chr-6_peak.marker-UNC10800126_lod.drop-1.5_snpsqc_dis_no-x_updated_7.batches_myo_mis.csv
    Untracked:  data/ici-sick.vs.ici-eoi_blup_sub_chr-6_peak.marker-UNC10832076_lod.drop-1.5_snpsqc_dis_no-x_updated_7.batches_myo.csv
    Untracked:  data/ici-sick.vs.ici-eoi_blup_sub_chr-7_peak.marker-UNCHS020903_lod.drop-1.5_snpsqc_dis_no-x_updated_7.batches_myo_mis.csv
    Untracked:  data/ici-sick.vs.ici-eoi_blup_sub_chr-7_peak.marker-UNCHS021163_lod.drop-1.5_snpsqc_dis_no-x_updated_7.batches_myo.csv
    Untracked:  data/ici-sick.vs.ici-eoi_blup_sub_chr-8_peak.marker-UNC15471847_lod.drop-1.5_snpsqc_dis_no-x_updated_7.batches_myo.csv
    Untracked:  data/ici-sick.vs.ici-eoi_blup_sub_chr-8_peak.marker-UNC15548888_lod.drop-1.5_snpsqc_dis_no-x_updated_7.batches_myo_mis.csv
    Untracked:  data/ici-sick.vs.ici-eoi_blup_sub_chr-9_peak.marker-UNC16231874_lod.drop-1.5_snpsqc_dis_no-x_updated_7.batches_myo.csv
    Untracked:  data/ici-sick.vs.ici-eoi_blup_sub_chr-9_peak.marker-UNC16231874_lod.drop-1.5_snpsqc_dis_no-x_updated_7.batches_myo_mis.csv
    Untracked:  data/ici-sick.vs.ici-eoi_blup_sub_chr-X_peak.marker-UNCHS049472_lod.drop-1.5_snpsqc_dis_no-x_updated_7.batches_myo.csv
    Untracked:  data/ici-sick.vs.ici-eoi_blup_sub_chr-X_peak.marker-UNCHS049472_lod.drop-1.5_snpsqc_dis_no-x_updated_7.batches_myo_mis.csv
    Untracked:  data/ici-sick.vs.ici-eoi_genes_chr-10_peak.marker-UNC18343990_lod.drop-1.5_snpsqc_dis_no-x_updated_7.batches_myo.csv
    Untracked:  data/ici-sick.vs.ici-eoi_genes_chr-10_peak.marker-UNC18343990_lod.drop-1.5_snpsqc_dis_no-x_updated_7.batches_myo_mis.csv
    Untracked:  data/ici-sick.vs.ici-eoi_genes_chr-11_peak.marker-UNC20070077_lod.drop-1.5_snpsqc_dis_no-x_updated_7.batches_myo.csv
    Untracked:  data/ici-sick.vs.ici-eoi_genes_chr-11_peak.marker-UNC20070077_lod.drop-1.5_snpsqc_dis_no-x_updated_7.batches_myo_mis.csv
    Untracked:  data/ici-sick.vs.ici-eoi_genes_chr-12_peak.marker-UNC21652584_lod.drop-1.5_snpsqc_dis_no-x_updated_7.batches_myo.csv
    Untracked:  data/ici-sick.vs.ici-eoi_genes_chr-12_peak.marker-UNC21652584_lod.drop-1.5_snpsqc_dis_no-x_updated_7.batches_myo_mis.csv
    Untracked:  data/ici-sick.vs.ici-eoi_genes_chr-13_peak.marker-UNCHS036579_lod.drop-1.5_snpsqc_dis_no-x_updated_7.batches_myo.csv
    Untracked:  data/ici-sick.vs.ici-eoi_genes_chr-13_peak.marker-UNCHS036579_lod.drop-1.5_snpsqc_dis_no-x_updated_7.batches_myo_mis.csv
    Untracked:  data/ici-sick.vs.ici-eoi_genes_chr-14_peak.marker-UNCHS037782_lod.drop-1.5_snpsqc_dis_no-x_updated_7.batches_myo.csv
    Untracked:  data/ici-sick.vs.ici-eoi_genes_chr-14_peak.marker-UNCHS037782_lod.drop-1.5_snpsqc_dis_no-x_updated_7.batches_myo_mis.csv
    Untracked:  data/ici-sick.vs.ici-eoi_genes_chr-15_peak.marker-UNC26069905_lod.drop-1.5_snpsqc_dis_no-x_updated_7.batches_myo.csv
    Untracked:  data/ici-sick.vs.ici-eoi_genes_chr-15_peak.marker-UNCHS041223_lod.drop-1.5_snpsqc_dis_no-x_updated_7.batches_myo_mis.csv
    Untracked:  data/ici-sick.vs.ici-eoi_genes_chr-16_peak.marker-JAX00070117_lod.drop-1.5_snpsqc_dis_no-x_updated_7.batches_myo.csv
    Untracked:  data/ici-sick.vs.ici-eoi_genes_chr-16_peak.marker-JAX00070117_lod.drop-1.5_snpsqc_dis_no-x_updated_7.batches_myo_mis.csv
    Untracked:  data/ici-sick.vs.ici-eoi_genes_chr-17_peak.marker-UNC28542319_lod.drop-1.5_snpsqc_dis_no-x_updated_7.batches_myo_mis.csv
    Untracked:  data/ici-sick.vs.ici-eoi_genes_chr-17_peak.marker-UNCJPD006670_lod.drop-1.5_snpsqc_dis_no-x_updated_7.batches_myo.csv
    Untracked:  data/ici-sick.vs.ici-eoi_genes_chr-18_peak.marker-UNC28776739_lod.drop-1.5_snpsqc_dis_no-x_updated_7.batches_myo.csv
    Untracked:  data/ici-sick.vs.ici-eoi_genes_chr-18_peak.marker-UNCHS045594_lod.drop-1.5_snpsqc_dis_no-x_updated_7.batches_myo_mis.csv
    Untracked:  data/ici-sick.vs.ici-eoi_genes_chr-19_peak.marker-UNC30414168_lod.drop-1.5_snpsqc_dis_no-x_updated_7.batches_myo.csv
    Untracked:  data/ici-sick.vs.ici-eoi_genes_chr-19_peak.marker-UNC30426276_lod.drop-1.5_snpsqc_dis_no-x_updated_7.batches_myo_mis.csv
    Untracked:  data/ici-sick.vs.ici-eoi_genes_chr-1_peak.marker-UNC2031646_lod.drop-1.5_snpsqc_dis_no-x_updated_7.batches_myo.csv
    Untracked:  data/ici-sick.vs.ici-eoi_genes_chr-1_peak.marker-UNCHS003700_lod.drop-1.5_snpsqc_dis_no-x_updated_7.batches_myo_mis.csv
    Untracked:  data/ici-sick.vs.ici-eoi_genes_chr-2_peak.marker-ICR5131_lod.drop-1.5_snpsqc_dis_no-x_updated_7.batches_myo.csv
    Untracked:  data/ici-sick.vs.ici-eoi_genes_chr-2_peak.marker-UNCHS006420_lod.drop-1.5_snpsqc_dis_no-x_updated_7.batches_myo_mis.csv
    Untracked:  data/ici-sick.vs.ici-eoi_genes_chr-3_peak.marker-UNC5667757_lod.drop-1.5_snpsqc_dis_no-x_updated_7.batches_myo.csv
    Untracked:  data/ici-sick.vs.ici-eoi_genes_chr-3_peak.marker-UNC5667757_lod.drop-1.5_snpsqc_dis_no-x_updated_7.batches_myo_mis.csv
    Untracked:  data/ici-sick.vs.ici-eoi_genes_chr-4_peak.marker-UNC6759992_lod.drop-1.5_snpsqc_dis_no-x_updated_7.batches_myo_mis.csv
    Untracked:  data/ici-sick.vs.ici-eoi_genes_chr-4_peak.marker-UNC6765178_lod.drop-1.5_snpsqc_dis_no-x_updated_7.batches_myo.csv
    Untracked:  data/ici-sick.vs.ici-eoi_genes_chr-5_peak.marker-UNC9889957_lod.drop-1.5_snpsqc_dis_no-x_updated_7.batches_myo_mis.csv
    Untracked:  data/ici-sick.vs.ici-eoi_genes_chr-5_peak.marker-UNC9900273_lod.drop-1.5_snpsqc_dis_no-x_updated_7.batches_myo.csv
    Untracked:  data/ici-sick.vs.ici-eoi_genes_chr-6_peak.marker-UNC10800126_lod.drop-1.5_snpsqc_dis_no-x_updated_7.batches_myo_mis.csv
    Untracked:  data/ici-sick.vs.ici-eoi_genes_chr-6_peak.marker-UNC10832076_lod.drop-1.5_snpsqc_dis_no-x_updated_7.batches_myo.csv
    Untracked:  data/ici-sick.vs.ici-eoi_genes_chr-7_peak.marker-UNCHS020903_lod.drop-1.5_snpsqc_dis_no-x_updated_7.batches_myo_mis.csv
    Untracked:  data/ici-sick.vs.ici-eoi_genes_chr-7_peak.marker-UNCHS021163_lod.drop-1.5_snpsqc_dis_no-x_updated_7.batches_myo.csv
    Untracked:  data/ici-sick.vs.ici-eoi_genes_chr-8_peak.marker-UNC15471847_lod.drop-1.5_snpsqc_dis_no-x_updated_7.batches_myo.csv
    Untracked:  data/ici-sick.vs.ici-eoi_genes_chr-8_peak.marker-UNC15548888_lod.drop-1.5_snpsqc_dis_no-x_updated_7.batches_myo_mis.csv
    Untracked:  data/ici-sick.vs.ici-eoi_genes_chr-9_peak.marker-UNC16231874_lod.drop-1.5_snpsqc_dis_no-x_updated_7.batches_myo.csv
    Untracked:  data/ici-sick.vs.ici-eoi_genes_chr-9_peak.marker-UNC16231874_lod.drop-1.5_snpsqc_dis_no-x_updated_7.batches_myo_mis.csv
    Untracked:  data/ici-sick.vs.ici-eoi_genes_chr-X_peak.marker-UNCHS049472_lod.drop-1.5_snpsqc_dis_no-x_updated_7.batches_myo.csv
    Untracked:  data/ici-sick.vs.ici-eoi_genes_chr-X_peak.marker-UNCHS049472_lod.drop-1.5_snpsqc_dis_no-x_updated_7.batches_myo_mis.csv
    Untracked:  data/ici-sick.vs.ici-eoi_gm_qtl_snpsqc_dis_no-x_updated_7.batches_myo.csv
    Untracked:  data/ici-sick.vs.ici-eoi_gm_qtl_snpsqc_dis_no-x_updated_7.batches_myo_mis.csv
    Untracked:  data/ici-sick.vs.ici-eoi_marker.freq_low.geno.freq.removed_geno.ratio_7.batches_myo.csv
    Untracked:  data/ici-sick.vs.ici-eoi_marker.freq_low.geno.freq.removed_geno.ratio_7.batches_myo_mis.csv
    Untracked:  data/ici-sick.vs.ici-eoi_marker.freq_low.geno.freq.removed_sample.outliers.removed_geno.ratio_7.batches_myo.csv
    Untracked:  data/ici-sick.vs.ici-eoi_marker.freq_low.geno.freq.removed_sample.outliers.removed_geno.ratio_7.batches_myo_mis.csv
    Untracked:  data/ici-sick.vs.ici-eoi_marker.freq_low.probs.freq.removed_geno.ratio_7.batches_myo.csv
    Untracked:  data/ici-sick.vs.ici-eoi_marker.freq_low.probs.freq.removed_geno.ratio_7.batches_myo_mis.csv
    Untracked:  data/ici-sick.vs.ici-eoi_marker.freq_low.probs.freq.removed_sample.outliers.removed_geno.ratio_7.batches_myo.csv
    Untracked:  data/ici-sick.vs.ici-eoi_marker.freq_low.probs.freq.removed_sample.outliers.removed_geno.ratio_7.batches_myo_mis.csv
    Untracked:  data/ici-sick.vs.ici-eoi_sample.genos_marker.freq_low.geno.freq.removed_7.batches_myo.csv
    Untracked:  data/ici-sick.vs.ici-eoi_sample.genos_marker.freq_low.geno.freq.removed_7.batches_myo_mis.csv
    Untracked:  data/ici-sick.vs.ici-eoi_sample.genos_marker.freq_low.geno.freq.removed_sample.outliers.removed_7.batches_myo.csv
    Untracked:  data/ici-sick.vs.ici-eoi_sample.genos_marker.freq_low.geno.freq.removed_sample.outliers.removed_7.batches_myo_mis.csv
    Untracked:  data/ici-sick.vs.ici-eoi_sample.genos_marker.freq_low.probs.freq.removed_7.batches_myo.csv
    Untracked:  data/ici-sick.vs.ici-eoi_sample.genos_marker.freq_low.probs.freq.removed_7.batches_myo_mis.csv
    Untracked:  data/ici-sick.vs.ici-eoi_sample.genos_marker.freq_low.probs.freq.removed_sample.outliers.removed_7.batches_myo.csv
    Untracked:  data/ici-sick.vs.ici-eoi_sample.genos_marker.freq_low.probs.freq.removed_sample.outliers.removed_7.batches_myo_mis.csv
    Untracked:  data/ici.vs.pbs_marker.freq_low.geno.freq.removed_geno.ratio_7.batches_myo.csv
    Untracked:  data/ici.vs.pbs_marker.freq_low.geno.freq.removed_geno.ratio_7.batches_myo_mis.csv
    Untracked:  data/ici.vs.pbs_marker.freq_low.geno.freq.removed_sample.outliers.removed_geno.ratio_7.batches_myo.csv
    Untracked:  data/ici.vs.pbs_marker.freq_low.geno.freq.removed_sample.outliers.removed_geno.ratio_7.batches_myo_mis.csv
    Untracked:  data/ici.vs.pbs_marker.freq_low.probs.freq.removed_geno.ratio_7.batches_myo.csv
    Untracked:  data/ici.vs.pbs_marker.freq_low.probs.freq.removed_geno.ratio_7.batches_myo_mis.csv
    Untracked:  data/ici.vs.pbs_marker.freq_low.probs.freq.removed_sample.outliers.removed_geno.ratio_7.batches_myo.csv
    Untracked:  data/ici.vs.pbs_marker.freq_low.probs.freq.removed_sample.outliers.removed_geno.ratio_7.batches_myo_mis.csv
    Untracked:  data/ici.vs.pbs_sample.genos_marker.freq_low.geno.freq.removed_7.batches_myo.csv
    Untracked:  data/ici.vs.pbs_sample.genos_marker.freq_low.geno.freq.removed_7.batches_myo_mis.csv
    Untracked:  data/ici.vs.pbs_sample.genos_marker.freq_low.geno.freq.removed_sample.outliers.removed_7.batches_myo.csv
    Untracked:  data/ici.vs.pbs_sample.genos_marker.freq_low.geno.freq.removed_sample.outliers.removed_7.batches_myo_mis.csv
    Untracked:  data/ici.vs.pbs_sample.genos_marker.freq_low.probs.freq.removed_7.batches_myo.csv
    Untracked:  data/ici.vs.pbs_sample.genos_marker.freq_low.probs.freq.removed_7.batches_myo_mis.csv
    Untracked:  data/ici.vs.pbs_sample.genos_marker.freq_low.probs.freq.removed_sample.outliers.removed_7.batches_myo.csv
    Untracked:  data/ici.vs.pbs_sample.genos_marker.freq_low.probs.freq.removed_sample.outliers.removed_7.batches_myo_mis.csv
    Untracked:  data/mean.differences_age.of.onset_het-ici-myo-yes.vs.het-ici-myo-no_sample.genos_marker.freq_low.geno.freq.removed_7.batches_myo.csv
    Untracked:  data/mean.differences_age.of.onset_het-ici-myo-yes.vs.het-ici-myo-no_sample.genos_marker.freq_low.geno.freq.removed_7.batches_myo_mis.csv
    Untracked:  data/mean.differences_age.of.onset_het-ici-myo-yes.vs.het-ici-myo-no_sample.genos_marker.freq_low.geno.freq.removed_sample.outliers.removed_7.batches_myo.csv
    Untracked:  data/mean.differences_age.of.onset_het-ici-myo-yes.vs.het-ici-myo-no_sample.genos_marker.freq_low.geno.freq.removed_sample.outliers.removed_7.batches_myo_mis.csv
    Untracked:  data/mean.differences_age.of.onset_het-ici.vs.het-pbs_sample.genos_marker.freq_low.geno.freq.removed_7.batches_myo.csv
    Untracked:  data/mean.differences_age.of.onset_het-ici.vs.het-pbs_sample.genos_marker.freq_low.geno.freq.removed_7.batches_myo_mis.csv
    Untracked:  data/mean.differences_age.of.onset_het-ici.vs.het-pbs_sample.genos_marker.freq_low.geno.freq.removed_sample.outliers.removed_7.batches_myo.csv
    Untracked:  data/mean.differences_age.of.onset_het-ici.vs.het-pbs_sample.genos_marker.freq_low.geno.freq.removed_sample.outliers.removed_7.batches_myo_mis.csv
    Untracked:  data/mean.differences_age.of.onset_ici-myo-yes.vs.ici-myo-no_sample.genos_marker.freq_low.geno.freq.removed_7.batches_myo.csv
    Untracked:  data/mean.differences_age.of.onset_ici-myo-yes.vs.ici-myo-no_sample.genos_marker.freq_low.geno.freq.removed_7.batches_myo_mis.csv
    Untracked:  data/mean.differences_age.of.onset_ici-myo-yes.vs.ici-myo-no_sample.genos_marker.freq_low.geno.freq.removed_sample.outliers.removed_7.batches_myo.csv
    Untracked:  data/mean.differences_age.of.onset_ici-myo-yes.vs.ici-myo-no_sample.genos_marker.freq_low.geno.freq.removed_sample.outliers.removed_7.batches_myo_mis.csv
    Untracked:  data/mean.differences_age.of.onset_ici-sick.vs.ici-eoi_sample.genos_marker.freq_low.geno.freq.removed_7.batches_myo.csv
    Untracked:  data/mean.differences_age.of.onset_ici-sick.vs.ici-eoi_sample.genos_marker.freq_low.geno.freq.removed_7.batches_myo_mis.csv
    Untracked:  data/mean.differences_age.of.onset_ici-sick.vs.ici-eoi_sample.genos_marker.freq_low.geno.freq.removed_sample.outliers.removed_7.batches_myo.csv
    Untracked:  data/mean.differences_age.of.onset_ici-sick.vs.ici-eoi_sample.genos_marker.freq_low.geno.freq.removed_sample.outliers.removed_7.batches_myo_mis.csv
    Untracked:  data/mean.differences_age.of.onset_ici.vs.pbs_sample.genos_marker.freq_low.geno.freq.removed_7.batches_myo.csv
    Untracked:  data/mean.differences_age.of.onset_ici.vs.pbs_sample.genos_marker.freq_low.geno.freq.removed_sample.outliers.removed_7.batches_myo.csv
    Untracked:  data/mean.differences_age.of.onset_myo-yes.vs.myo-no_sample.genos_marker.freq_low.geno.freq.removed_7.batches_myo.csv
    Untracked:  data/mean.differences_age.of.onset_myo-yes.vs.myo-no_sample.genos_marker.freq_low.geno.freq.removed_7.batches_myo_mis.csv
    Untracked:  data/mean.differences_age.of.onset_myo-yes.vs.myo-no_sample.genos_marker.freq_low.geno.freq.removed_sample.outliers.removed_7.batches_myo.csv
    Untracked:  data/mean.differences_age.of.onset_myo-yes.vs.myo-no_sample.genos_marker.freq_low.geno.freq.removed_sample.outliers.removed_7.batches_myo_mis.csv
    Untracked:  data/mean.differences_age.of.onset_pbs-myo-yes.vs.pbs-myo-no_sample.genos_marker.freq_low.geno.freq.removed_7.batches_myo.csv
    Untracked:  data/mean.differences_age.of.onset_pbs-myo-yes.vs.pbs-myo-no_sample.genos_marker.freq_low.geno.freq.removed_7.batches_myo_mis.csv
    Untracked:  data/mean.differences_age.of.onset_pbs-myo-yes.vs.pbs-myo-no_sample.genos_marker.freq_low.geno.freq.removed_sample.outliers.removed_7.batches_myo.csv
    Untracked:  data/mean.differences_age.of.onset_pbs-myo-yes.vs.pbs-myo-no_sample.genos_marker.freq_low.geno.freq.removed_sample.outliers.removed_7.batches_myo_mis.csv
    Untracked:  data/mean.differences_group_het-ici-myo-yes.vs.het-ici-myo-no_sample.genos_marker.freq_low.geno.freq.removed_7.batches_myo.csv
    Untracked:  data/mean.differences_group_het-ici-myo-yes.vs.het-ici-myo-no_sample.genos_marker.freq_low.geno.freq.removed_7.batches_myo_mis.csv
    Untracked:  data/mean.differences_group_het-ici-myo-yes.vs.het-ici-myo-no_sample.genos_marker.freq_low.geno.freq.removed_sample.outliers.removed_7.batches_myo.csv
    Untracked:  data/mean.differences_group_het-ici-myo-yes.vs.het-ici-myo-no_sample.genos_marker.freq_low.geno.freq.removed_sample.outliers.removed_7.batches_myo_mis.csv
    Untracked:  data/mean.differences_group_het-ici.vs.het-pbs_sample.genos_marker.freq_low.geno.freq.removed_7.batches_myo.csv
    Untracked:  data/mean.differences_group_het-ici.vs.het-pbs_sample.genos_marker.freq_low.geno.freq.removed_7.batches_myo_mis.csv
    Untracked:  data/mean.differences_group_het-ici.vs.het-pbs_sample.genos_marker.freq_low.geno.freq.removed_sample.outliers.removed_7.batches_myo.csv
    Untracked:  data/mean.differences_group_het-ici.vs.het-pbs_sample.genos_marker.freq_low.geno.freq.removed_sample.outliers.removed_7.batches_myo_mis.csv
    Untracked:  data/mean.differences_group_ici-myo-yes.vs.ici-myo-no_sample.genos_marker.freq_low.geno.freq.removed_7.batches_myo.csv
    Untracked:  data/mean.differences_group_ici-myo-yes.vs.ici-myo-no_sample.genos_marker.freq_low.geno.freq.removed_7.batches_myo_mis.csv
    Untracked:  data/mean.differences_group_ici-myo-yes.vs.ici-myo-no_sample.genos_marker.freq_low.geno.freq.removed_sample.outliers.removed_7.batches_myo.csv
    Untracked:  data/mean.differences_group_ici-myo-yes.vs.ici-myo-no_sample.genos_marker.freq_low.geno.freq.removed_sample.outliers.removed_7.batches_myo_mis.csv
    Untracked:  data/mean.differences_group_ici-sick.vs.ici-eoi_sample.genos_marker.freq_low.geno.freq.removed_7.batches_myo.csv
    Untracked:  data/mean.differences_group_ici-sick.vs.ici-eoi_sample.genos_marker.freq_low.geno.freq.removed_7.batches_myo_mis.csv
    Untracked:  data/mean.differences_group_ici-sick.vs.ici-eoi_sample.genos_marker.freq_low.geno.freq.removed_sample.outliers.removed_7.batches_myo.csv
    Untracked:  data/mean.differences_group_ici-sick.vs.ici-eoi_sample.genos_marker.freq_low.geno.freq.removed_sample.outliers.removed_7.batches_myo_mis.csv
    Untracked:  data/mean.differences_group_ici.vs.pbs_sample.genos_marker.freq_low.geno.freq.removed_7.batches_myo.csv
    Untracked:  data/mean.differences_group_ici.vs.pbs_sample.genos_marker.freq_low.geno.freq.removed_sample.outliers.removed_7.batches_myo.csv
    Untracked:  data/mean.differences_group_myo-yes.vs.myo-no_sample.genos_marker.freq_low.geno.freq.removed_7.batches_myo.csv
    Untracked:  data/mean.differences_group_myo-yes.vs.myo-no_sample.genos_marker.freq_low.geno.freq.removed_7.batches_myo_mis.csv
    Untracked:  data/mean.differences_group_myo-yes.vs.myo-no_sample.genos_marker.freq_low.geno.freq.removed_sample.outliers.removed_7.batches_myo.csv
    Untracked:  data/mean.differences_group_myo-yes.vs.myo-no_sample.genos_marker.freq_low.geno.freq.removed_sample.outliers.removed_7.batches_myo_mis.csv
    Untracked:  data/mean.differences_group_pbs-myo-yes.vs.pbs-myo-no_sample.genos_marker.freq_low.geno.freq.removed_7.batches_myo.csv
    Untracked:  data/mean.differences_group_pbs-myo-yes.vs.pbs-myo-no_sample.genos_marker.freq_low.geno.freq.removed_7.batches_myo_mis.csv
    Untracked:  data/mean.differences_group_pbs-myo-yes.vs.pbs-myo-no_sample.genos_marker.freq_low.geno.freq.removed_sample.outliers.removed_7.batches_myo.csv
    Untracked:  data/mean.differences_group_pbs-myo-yes.vs.pbs-myo-no_sample.genos_marker.freq_low.geno.freq.removed_sample.outliers.removed_7.batches_myo_mis.csv
    Untracked:  data/myo-yes.vs.myo-no_gm_qtl_snpsqc_dis_no-x_updated_7.batches_myo.csv
    Untracked:  data/myo-yes.vs.myo-no_gm_qtl_snpsqc_dis_no-x_updated_7.batches_myo_mis.csv
    Untracked:  data/myo-yes.vs.myo-no_marker.freq_low.geno.freq.removed_geno.ratio_7.batches_myo.csv
    Untracked:  data/myo-yes.vs.myo-no_marker.freq_low.geno.freq.removed_geno.ratio_7.batches_myo_mis.csv
    Untracked:  data/myo-yes.vs.myo-no_marker.freq_low.geno.freq.removed_sample.outliers.removed_geno.ratio_7.batches_myo.csv
    Untracked:  data/myo-yes.vs.myo-no_marker.freq_low.geno.freq.removed_sample.outliers.removed_geno.ratio_7.batches_myo_mis.csv
    Untracked:  data/myo-yes.vs.myo-no_marker.freq_low.probs.freq.removed_geno.ratio_7.batches_myo.csv
    Untracked:  data/myo-yes.vs.myo-no_marker.freq_low.probs.freq.removed_geno.ratio_7.batches_myo_mis.csv
    Untracked:  data/myo-yes.vs.myo-no_marker.freq_low.probs.freq.removed_sample.outliers.removed_geno.ratio_7.batches_myo.csv
    Untracked:  data/myo-yes.vs.myo-no_marker.freq_low.probs.freq.removed_sample.outliers.removed_geno.ratio_7.batches_myo_mis.csv
    Untracked:  data/myo-yes.vs.myo-no_sample.genos_marker.freq_low.geno.freq.removed_7.batches_myo.csv
    Untracked:  data/myo-yes.vs.myo-no_sample.genos_marker.freq_low.geno.freq.removed_7.batches_myo_mis.csv
    Untracked:  data/myo-yes.vs.myo-no_sample.genos_marker.freq_low.geno.freq.removed_sample.outliers.removed_7.batches_myo.csv
    Untracked:  data/myo-yes.vs.myo-no_sample.genos_marker.freq_low.geno.freq.removed_sample.outliers.removed_7.batches_myo_mis.csv
    Untracked:  data/myo-yes.vs.myo-no_sample.genos_marker.freq_low.probs.freq.removed_7.batches_myo.csv
    Untracked:  data/myo-yes.vs.myo-no_sample.genos_marker.freq_low.probs.freq.removed_7.batches_myo_mis.csv
    Untracked:  data/myo-yes.vs.myo-no_sample.genos_marker.freq_low.probs.freq.removed_sample.outliers.removed_7.batches_myo.csv
    Untracked:  data/myo-yes.vs.myo-no_sample.genos_marker.freq_low.probs.freq.removed_sample.outliers.removed_7.batches_myo_mis.csv
    Untracked:  data/pbs-myo-yes.vs.pbs-myo-no_gm_qtl_snpsqc_dis_no-x_updated_7.batches_myo.csv
    Untracked:  data/pbs-myo-yes.vs.pbs-myo-no_gm_qtl_snpsqc_dis_no-x_updated_7.batches_myo_mis.csv
    Untracked:  data/pbs-myo-yes.vs.pbs-myo-no_marker.freq_low.geno.freq.removed_geno.ratio_7.batches_myo.csv
    Untracked:  data/pbs-myo-yes.vs.pbs-myo-no_marker.freq_low.geno.freq.removed_geno.ratio_7.batches_myo_mis.csv
    Untracked:  data/pbs-myo-yes.vs.pbs-myo-no_marker.freq_low.geno.freq.removed_sample.outliers.removed_geno.ratio_7.batches_myo.csv
    Untracked:  data/pbs-myo-yes.vs.pbs-myo-no_marker.freq_low.geno.freq.removed_sample.outliers.removed_geno.ratio_7.batches_myo_mis.csv
    Untracked:  data/pbs-myo-yes.vs.pbs-myo-no_marker.freq_low.probs.freq.removed_geno.ratio_7.batches_myo.csv
    Untracked:  data/pbs-myo-yes.vs.pbs-myo-no_marker.freq_low.probs.freq.removed_geno.ratio_7.batches_myo_mis.csv
    Untracked:  data/pbs-myo-yes.vs.pbs-myo-no_marker.freq_low.probs.freq.removed_sample.outliers.removed_geno.ratio_7.batches_myo.csv
    Untracked:  data/pbs-myo-yes.vs.pbs-myo-no_marker.freq_low.probs.freq.removed_sample.outliers.removed_geno.ratio_7.batches_myo_mis.csv
    Untracked:  data/pbs-myo-yes.vs.pbs-myo-no_sample.genos_marker.freq_low.geno.freq.removed_7.batches_myo.csv
    Untracked:  data/pbs-myo-yes.vs.pbs-myo-no_sample.genos_marker.freq_low.geno.freq.removed_7.batches_myo_mis.csv
    Untracked:  data/pbs-myo-yes.vs.pbs-myo-no_sample.genos_marker.freq_low.geno.freq.removed_sample.outliers.removed_7.batches_myo.csv
    Untracked:  data/pbs-myo-yes.vs.pbs-myo-no_sample.genos_marker.freq_low.geno.freq.removed_sample.outliers.removed_7.batches_myo_mis.csv
    Untracked:  data/pbs-myo-yes.vs.pbs-myo-no_sample.genos_marker.freq_low.probs.freq.removed_7.batches_myo.csv
    Untracked:  data/pbs-myo-yes.vs.pbs-myo-no_sample.genos_marker.freq_low.probs.freq.removed_7.batches_myo_mis.csv
    Untracked:  data/pbs-myo-yes.vs.pbs-myo-no_sample.genos_marker.freq_low.probs.freq.removed_sample.outliers.removed_7.batches_myo.csv
    Untracked:  data/pbs-myo-yes.vs.pbs-myo-no_sample.genos_marker.freq_low.probs.freq.removed_sample.outliers.removed_7.batches_myo_mis.csv
    Untracked:  data/percent_missing_id_4.batches_myo.RData
    Untracked:  data/percent_missing_id_7.batches_myo.RData
    Untracked:  data/percent_missing_marker_4.batches_myo.RData
    Untracked:  data/percent_missing_marker_7.batches_myo.RData
    Untracked:  data/pheno_4.batches_myo.csv
    Untracked:  data/pheno_7.batches_myo.csv
    Untracked:  data/physical_map_4.batches_myo.csv
    Untracked:  data/physical_map_7.batches_myo.csv
    Untracked:  data/probs_36state_bc_7.batches_myo.rds
    Untracked:  data/probs_8state_bc_7.batches_myo.rds
    Untracked:  data/qc_info_bad_sample_4.batches_myo.RData
    Untracked:  data/qc_info_bad_sample_7.batches_myo.RData
    Untracked:  data/sample_geno_AHB_4.batches_myo.csv
    Untracked:  data/sample_geno_AHB_7.batches_myo.csv
    Untracked:  data/sample_geno_AHB_7.batches_myo_orig.id.csv
    Untracked:  data/sample_geno_bc_4.batches_myo.csv
    Untracked:  data/sample_geno_bc_7.batches_myo.csv
    Untracked:  data/sample_geno_bc_7.batches_myo_orig.id.csv
    Untracked:  data/serreze_probs_4.batches_myo.rds
    Untracked:  data/serreze_probs_7.batches_myo.rds
    Untracked:  data/serreze_probs_allqc_4.batches_myo.rds
    Untracked:  data/serreze_probs_allqc_4.batches_myo_mis.rds
    Untracked:  data/serreze_probs_allqc_7.batches_myo.rds
    Untracked:  data/serreze_probs_allqc_7.batches_myo_mis.rds
    Untracked:  data/summary.cg_4.batches_myo.RData
    Untracked:  data/summary.cg_7.batches_myo.RData
    Untracked:  output/Percent_missing_genotype_data_4.batches_myo.pdf
    Untracked:  output/Percent_missing_genotype_data_7.batches_myo.pdf
    Untracked:  output/Percent_missing_genotype_data_per_marker_4.batches_myo.pdf
    Untracked:  output/Percent_missing_genotype_data_per_marker_7.batches_myo.pdf
    Untracked:  output/Proportion_matching_genotypes_before_removal_of_bad_samples_4.batches_myo.pdf
    Untracked:  output/Proportion_matching_genotypes_before_removal_of_bad_samples_7.batches_myo.pdf
    Untracked:  output/genotype_error_marker_4.batches_myo.pdf
    Untracked:  output/genotype_error_marker_7.batches_myo.pdf
    Untracked:  output/genotype_frequency_marker_4.batches_myo.pdf
    Untracked:  output/genotype_frequency_marker_7.batches_myo.pdf

</code></pre>
<p>
Note that any generated files, e.g.Â HTML, png, CSS, etc., are not
included in this status report because it is ok for generated content to
have uncommitted changes.
</p>
</div>
</div>
</div>
</div>
<hr>
</div>
<div id="versions" class="tab-pane fade">
<p>
There are no past versions. Publish this analysis with
<code>wflow_publish()</code> to start tracking its development.
</p>
<hr>
</div>
</div>
</div>
<div id="allele-frequencies" class="section level2">
<h2>Allele Frequencies</h2>
<div id="all-variants-before-snp-qc" class="section level3">
<h3>All variants (before snp qc)</h3>
<p><img src="figure/genotype.frequencies_myo-yes.vs.myo-no_freq_7.batches_myo.Rmd/pbs%20all%20figures-1.png" width="100%" height="100%" style="display: block; margin: auto;" /><img src="figure/genotype.frequencies_myo-yes.vs.myo-no_freq_7.batches_myo.Rmd/pbs%20all%20figures-2.png" width="100%" height="100%" style="display: block; margin: auto;" /></p>
</div>
<div id="after-snp-qc" class="section level3">
<h3>After SNP QC</h3>
<p><img src="figure/genotype.frequencies_myo-yes.vs.myo-no_freq_7.batches_myo.Rmd/pbs%20qc%20figures-1.png" width="100%" height="100%" style="display: block; margin: auto;" /><img src="figure/genotype.frequencies_myo-yes.vs.myo-no_freq_7.batches_myo.Rmd/pbs%20qc%20figures-2.png" width="100%" height="100%" style="display: block; margin: auto;" /></p>
</div>
</div>
<div id="with-sample-outliers" class="section level2">
<h2>with sample outliers</h2>
<pre class="r"><code>load(&quot;data/gm_allqc_7.batches_myo.RData&quot;)

#gm_allqc
gm=gm_allqc
gm</code></pre>
<pre><code>Object of class cross2 (crosstype &quot;bc&quot;)

Total individuals              357
No. genotyped individuals      357
No. phenotyped individuals     357
No. with both geno &amp; pheno     357

No. phenotypes                   1
No. covariates                  11
No. phenotype covariates         0

No. chromosomes                 20
Total markers                32660

No. markers by chr:
   1    2    3    4    5    6    7    8    9   10   11   12   13   14   15   16 
2482 2412 1742 1772 1648 1842 1541 1517 1769 1092 1754 1230 1441 1505 1128  842 
  17   18   19    X 
 655  820  945 4523 </code></pre>
<pre class="r"><code>pr &lt;- readRDS(&quot;data/serreze_probs_allqc_7.batches_myo.rds&quot;)
#pr &lt;- readRDS(&quot;data/serreze_probs.rds&quot;)

geno &lt;- read.csv(&quot;data/sample_geno_bc_7.batches_myo.csv&quot;, as.is=T, check.names=FALSE)
#names(geno) &lt;- gsub(&quot;^X&quot;,&quot;&quot;,names(geno))
#names(geno) &lt;- gsub(&quot;\.&quot;,&quot;-&quot;,names(geno))
#names(geno)[2:ncol(geno)] &lt;- samples$Unique.Sample.ID
rownames(geno) &lt;- geno$marker</code></pre>
<pre class="r"><code>## extracting animals with ici and pbs group status
#miceinfo &lt;- gm$covar[gm$covar$group == &quot;PBS&quot; | gm$covar$group == &quot;ICI&quot;,]
#table(miceinfo$group)
#mice.ids &lt;- rownames(miceinfo)

#gm &lt;- gm[mice.ids]
#gm
#table(gm$covar$group)

covars &lt;- read_csv(&quot;data/covar_corrected_myo-yes.vs.myo-no_7.batches_myo.csv&quot;)
# removing any missing info
#covars &lt;- subset(covars, covars$age.of.onset!=&#39;&#39;)
nrow(covars)</code></pre>
<pre><code>[1] 241</code></pre>
<pre class="r"><code>table(covars$&quot;Myocarditis Status&quot;)</code></pre>
<pre><code>
 NO YES 
 29 212 </code></pre>
<pre class="r"><code>table(covars$&quot;Murine MHC KO Status&quot;)</code></pre>
<pre><code>
HOM 
241 </code></pre>
<pre class="r"><code>table(covars$&quot;Drug Treatment&quot;)</code></pre>
<pre><code>
ICI PBS 
145  96 </code></pre>
<pre class="r"><code>table(covars$&quot;clinical pheno&quot;)</code></pre>
<pre><code>
 EOI SICK 
  95  146 </code></pre>
<pre class="r"><code># keeping only informative mice
gm &lt;- gm[covars$Mouse.ID]
gm</code></pre>
<pre><code>Object of class cross2 (crosstype &quot;bc&quot;)

Total individuals              241
No. genotyped individuals      241
No. phenotyped individuals     241
No. with both geno &amp; pheno     241

No. phenotypes                   1
No. covariates                  11
No. phenotype covariates         0

No. chromosomes                 20
Total markers                32660

No. markers by chr:
   1    2    3    4    5    6    7    8    9   10   11   12   13   14   15   16 
2482 2412 1742 1772 1648 1842 1541 1517 1769 1092 1754 1230 1441 1505 1128  842 
  17   18   19    X 
 655  820  945 4523 </code></pre>
<pre class="r"><code>table(gm$covar$&quot;Myocarditis Status&quot;)</code></pre>
<pre><code>
 NO YES 
 29 212 </code></pre>
<pre class="r"><code>table(gm$covar$&quot;Murine MHC KO Status&quot;)</code></pre>
<pre><code>
HOM 
241 </code></pre>
<pre class="r"><code>table(gm$covar$&quot;Drug Treatment&quot;)</code></pre>
<pre><code>
ICI PBS 
145  96 </code></pre>
<pre class="r"><code>table(gm$covar$&quot;clinical pheno&quot;)</code></pre>
<pre><code>
 EOI SICK 
  95  146 </code></pre>
<pre class="r"><code>pr.qc.ids &lt;- pr
for (i in 1:20){pr.qc.ids[[i]] = pr.qc.ids[[i]][covars$Mouse.ID,,]}

geno &lt;- geno[,covars$Mouse.ID]
geno &lt;- geno[marker_names(gm),]
dim(geno)</code></pre>
<pre><code>[1] 32660   241</code></pre>
<pre class="r"><code>## calculating genotype frequencies
### from geno genotypes
g &lt;- do.call(&quot;cbind&quot;, gm$geno)
gf_mar_geno &lt;- t(apply(g, 2, function(a) table(factor(a, 1:2))/sum(a != 0)))
gn_mar_geno &lt;- t(apply(g, 2, function(a) table(factor(a, 0:2))))
#gf_mar_raw&lt;- gf_mar_raw[gf_mar_raw[,2] != &quot;NaN&quot;,]
colnames(gf_mar_geno) &lt;- c(&quot;freq_AA_geno_table&quot;,&quot;freq_AB_geno_table&quot;)
colnames(gn_mar_geno) &lt;- c(&quot;count_missing_geno_table&quot;,&quot;count_AA_geno_table&quot;,&quot;count_AB_geno_table&quot;)
gfn_mar_geno &lt;- merge(as.data.frame(gn_mar_geno), as.data.frame(gf_mar_geno), by=&quot;row.names&quot;)
rownames(gfn_mar_geno) &lt;- gfn_mar_geno[,1]
gfn_mar_geno &lt;- gfn_mar_geno[-1]

### from raw using table function in R
#genosl &lt;- list()
#for(i in 1:nrow(geno)){
##for(i in 1:3){
#    genoi &lt;- geno[i,]
#    freqf &lt;- table(factor(geno[i,], c(&quot;-&quot;,&quot;AA&quot;,&quot;AB&quot;)))
#    genoi$count_AA_raw_rowSums &lt;- rowSums(genoi == &quot;AA&quot;)
#    genoi$count_AB_raw_rowSums &lt;- rowSums(genoi == &quot;AB&quot;)
#    genoi$count_missing_raw_rowSums &lt;- rowSums(genoi == &quot;-&quot;)
#    freqf &lt;- t(table(factor(geno[i,], c(&quot;-&quot;,&quot;AA&quot;,&quot;AB&quot;))))
#    freqf &lt;- as.data.frame(t(freqf[1,]))
#    rownames(freqf) &lt;- rownames(genoi)
#    colnames(freqf) &lt;- c(&quot;count_missing_raw_table&quot;,&quot;count_AA_raw_table&quot;,&quot;count_AB_raw_table&quot;)
#    genoif &lt;- cbind(freqf,genoi[c(&quot;count_AA_raw_rowSums&quot;,&quot;count_AB_raw_rowSums&quot;,&quot;count_missing_raw_rowSums&quot;)])
#    genosl[[i]] = genoif
#}
#gf_mar_raw &lt;- do.call(&quot;rbind&quot;,genosl)
#gf_mar_raw &lt;- gf_mar_raw[,c(1:3,6,4:5)]
#gf_mar_raw$index &lt;- 1:nrow(gf_mar_raw)

### from probabilities
gf_mar_probs.1 &lt;- calc_geno_freq(pr.qc.ids, by = &quot;marker&quot;, omit_x = FALSE)
#gn_mar_probs &lt;- calc_geno_freq(probs, by = &quot;individual&quot;, omit_x = FALSE)
gf_mar_probs &lt;- rbind(gf_mar_probs.1$A[,1:2], gf_mar_probs.1$X[,1:2])
colnames(gf_mar_probs) &lt;- paste0(&quot;freq_&quot;,colnames(gf_mar_probs),&quot;_probs&quot;)
gf_mar_probs &lt;- as.data.frame(gf_mar_probs)
gf_mar_probs$index &lt;- 1:nrow(gf_mar_probs)

### merging all genotype frequecies for all markers
#gf_mar.1 &lt;- merge(as.data.frame(gf_mar_raw), as.data.frame(gfn_mar_geno), by=&quot;row.names&quot;)
#rownames(gf_mar.1) &lt;- gf_mar.1[,1]
#gf_mar.1 &lt;- gf_mar.1[-1]
#gf_mar &lt;- merge(gf_mar.1,as.data.frame(gf_mar_probs), by=&quot;row.names&quot;)
gf_mar &lt;- merge(as.data.frame(gfn_mar_geno),as.data.frame(gf_mar_probs), by=&quot;row.names&quot;)
rownames(gf_mar) &lt;- gf_mar[,1]
gf_mar &lt;- gf_mar[-1]
gf_mar &lt;- gf_mar[order(gf_mar$index),]
dim(gf_mar)</code></pre>
<pre><code>[1] 32660     8</code></pre>
<pre class="r"><code># Calculating ratio and flagging informative marker
gf_mar$ratio = as.numeric(gf_mar$freq_AA_geno_table)/as.numeric(gf_mar$freq_AB_geno_table)
gf_mar$Include = ifelse(gf_mar$ratio &gt;= 0.90 &amp; gf_mar$ratio &lt;= 1.10, TRUE,FALSE)
table(gf_mar$Include)</code></pre>
<pre><code>
FALSE  TRUE 
20578 12082 </code></pre>
<pre class="r"><code>## filtering out &lt;= 0.05
gf_mar$count.geno &lt;- rowSums(gf_mar[c(&quot;freq_AA_geno_table&quot;,&quot;freq_AB_geno_table&quot;)] &lt;=0.05)
filtered_gf_mar_geno &lt;- gf_mar[gf_mar$count.geno != 1,]
filtered_gf_mar_geno &lt;- filtered_gf_mar_geno[,-which(names(filtered_gf_mar_geno) %in% c(&quot;count.geno&quot;,&quot;index&quot;))]
dim(filtered_gf_mar_geno)</code></pre>
<pre><code>[1] 26491     9</code></pre>
<pre class="r"><code>table(filtered_gf_mar_geno$Include)</code></pre>
<pre><code>
FALSE  TRUE 
14409 12082 </code></pre>
<pre class="r"><code>gf_mar$count.probs &lt;- rowSums(gf_mar[c(&quot;freq_AA_probs&quot;,&quot;freq_AB_probs&quot;)] &lt;=0.05)
filtered_gf_mar_probs &lt;- gf_mar[gf_mar$count.probs != 1,]
filtered_gf_mar_probs &lt;- filtered_gf_mar_probs[,-which(names(filtered_gf_mar_probs) %in% c(&quot;count.geno&quot;,&quot;count.probs&quot;,&quot;index&quot;))]
dim(filtered_gf_mar_probs)</code></pre>
<pre><code>[1] 27421     9</code></pre>
<pre class="r"><code>table(filtered_gf_mar_probs$Include)</code></pre>
<pre><code>
FALSE  TRUE 
15339 12082 </code></pre>
<pre class="r"><code>## merging with sample_genos
filtered_gf_mar_geno_sample &lt;- merge(geno,filtered_gf_mar_geno, by=&quot;row.names&quot;, all.y=T, sort=F)
#filtered_gf_mar_geno_sample &lt;- filtered_gf_mar_geno_sample[order(filtered_gf_mar_geno_sample$index),]     
#filtered_gf_mar_geno_sample &lt;- filtered_gf_mar_geno_sample[,-which(names(filtered_gf_mar_geno_sample) %in% c(&quot;count.geno&quot;,&quot;index&quot;))]
names(filtered_gf_mar_geno_sample)[1] &lt;- c(&quot;marker&quot;)
dim(filtered_gf_mar_geno_sample)</code></pre>
<pre><code>[1] 26491   251</code></pre>
<pre class="r"><code>filtered_gf_mar_probs_sample &lt;- merge(geno,filtered_gf_mar_probs, by=&quot;row.names&quot;, all.y=T, sort=F)
#filtered_gf_mar_probs_sample &lt;- filtered_gf_mar_probs_sample[order(filtered_gf_mar_probs_sample$index),]
#filtered_gf_mar_probs_sample &lt;- filtered_gf_mar_probs_sample[,-which(names(filtered_gf_mar_probs_sample) %in% c(&quot;count.geno&quot;,&quot;count.probs&quot;,&quot;index&quot;))]
names(filtered_gf_mar_probs_sample)[1] &lt;- c(&quot;marker&quot;)
dim(filtered_gf_mar_probs_sample)</code></pre>
<pre><code>[1] 27421   251</code></pre>
<pre class="r"><code>## saving files
write.csv(filtered_gf_mar_geno_sample, &quot;data/myo-yes.vs.myo-no_sample.genos_marker.freq_low.geno.freq.removed_7.batches_myo.csv&quot;, quote=F)
write.csv(filtered_gf_mar_probs_sample, &quot;data/myo-yes.vs.myo-no_sample.genos_marker.freq_low.probs.freq.removed_7.batches_myo.csv&quot;, quote=F)

write.csv(filtered_gf_mar_geno, &quot;data/myo-yes.vs.myo-no_marker.freq_low.geno.freq.removed_geno.ratio_7.batches_myo.csv&quot;, quote=F)
write.csv(filtered_gf_mar_probs, &quot;data/myo-yes.vs.myo-no_marker.freq_low.probs.freq.removed_geno.ratio_7.batches_myo.csv&quot;, quote=F)</code></pre>
<p><img src="figure/genotype.frequencies_myo-yes.vs.myo-no_freq_7.batches_myo.Rmd/pbs%20figures%20dismark-1.png" width="100%" height="100%" style="display: block; margin: auto;" /><img src="figure/genotype.frequencies_myo-yes.vs.myo-no_freq_7.batches_myo.Rmd/pbs%20figures%20dismark-2.png" width="100%" height="100%" style="display: block; margin: auto;" /></p>
</div>
<div id="sample-outliers-removed" class="section level2">
<h2>sample outliers removed</h2>
<pre class="r"><code>load(&quot;data/gm_allqc_7.batches_myo.RData&quot;)

#gm_allqc
gm=gm_allqc
gm</code></pre>
<pre><code>Object of class cross2 (crosstype &quot;bc&quot;)

Total individuals              357
No. genotyped individuals      357
No. phenotyped individuals     357
No. with both geno &amp; pheno     357

No. phenotypes                   1
No. covariates                  11
No. phenotype covariates         0

No. chromosomes                 20
Total markers                32660

No. markers by chr:
   1    2    3    4    5    6    7    8    9   10   11   12   13   14   15   16 
2482 2412 1742 1772 1648 1842 1541 1517 1769 1092 1754 1230 1441 1505 1128  842 
  17   18   19    X 
 655  820  945 4523 </code></pre>
<pre class="r"><code>pr &lt;- readRDS(&quot;data/serreze_probs_allqc_7.batches_myo.rds&quot;)
#pr &lt;- readRDS(&quot;data/serreze_probs.rds&quot;)

geno &lt;- read.csv(&quot;data/sample_geno_bc_7.batches_myo.csv&quot;, as.is=T, check.names=FALSE)
#names(geno) &lt;- gsub(&quot;^X&quot;,&quot;&quot;,names(geno))
#names(geno) &lt;- gsub(&quot;\.&quot;,&quot;-&quot;,names(geno))
#names(geno)[2:ncol(geno)] &lt;- samples$Unique.Sample.ID
rownames(geno) &lt;- geno$marker</code></pre>
<pre class="r"><code>## extracting animals with ici and pbs group status
#miceinfo &lt;- gm$covar[gm$covar$group == &quot;PBS&quot; | gm$covar$group == &quot;ICI&quot;,]
#table(miceinfo$group)
#mice.ids &lt;- rownames(miceinfo)

#gm &lt;- gm[mice.ids]
#gm
#table(gm$covar$group)

covars &lt;- read_csv(&quot;data/covar_corrected.cleaned_myo-yes.vs.myo-no_7.batches_myo.csv&quot;)
# removing any missing info
covars &lt;- subset(covars, covars$out.age.of.onset==&#39;Keep&#39; &amp; covars$out.rz.age==&#39;Keep&#39;)
nrow(covars)</code></pre>
<pre><code>[1] 241</code></pre>
<pre class="r"><code>table(covars$&quot;Myocarditis Status&quot;)</code></pre>
<pre><code>
 NO YES 
 29 212 </code></pre>
<pre class="r"><code>table(covars$&quot;Murine MHC KO Status&quot;)</code></pre>
<pre><code>
HOM 
241 </code></pre>
<pre class="r"><code>table(covars$&quot;Drug Treatment&quot;)</code></pre>
<pre><code>
ICI PBS 
145  96 </code></pre>
<pre class="r"><code>table(covars$&quot;clinical pheno&quot;)</code></pre>
<pre><code>
 EOI SICK 
  95  146 </code></pre>
<pre class="r"><code># keeping only informative mice
gm &lt;- gm[covars$Mouse.ID]
gm</code></pre>
<pre><code>Object of class cross2 (crosstype &quot;bc&quot;)

Total individuals              241
No. genotyped individuals      241
No. phenotyped individuals     241
No. with both geno &amp; pheno     241

No. phenotypes                   1
No. covariates                  11
No. phenotype covariates         0

No. chromosomes                 20
Total markers                32660

No. markers by chr:
   1    2    3    4    5    6    7    8    9   10   11   12   13   14   15   16 
2482 2412 1742 1772 1648 1842 1541 1517 1769 1092 1754 1230 1441 1505 1128  842 
  17   18   19    X 
 655  820  945 4523 </code></pre>
<pre class="r"><code>table(gm$covar$&quot;Myocarditis Status&quot;)</code></pre>
<pre><code>
 NO YES 
 29 212 </code></pre>
<pre class="r"><code>table(gm$covar$&quot;Murine MHC KO Status&quot;)</code></pre>
<pre><code>
HOM 
241 </code></pre>
<pre class="r"><code>table(gm$covar$&quot;Drug Treatment&quot;)</code></pre>
<pre><code>
ICI PBS 
145  96 </code></pre>
<pre class="r"><code>table(gm$covar$&quot;clinical pheno&quot;)</code></pre>
<pre><code>
 EOI SICK 
  95  146 </code></pre>
<pre class="r"><code>pr.qc.ids &lt;- pr
for (i in 1:20){pr.qc.ids[[i]] = pr.qc.ids[[i]][covars$Mouse.ID,,]}

geno &lt;- geno[,covars$Mouse.ID]
geno &lt;- geno[marker_names(gm),]
dim(geno)</code></pre>
<pre><code>[1] 32660   241</code></pre>
<pre class="r"><code>## calculating genotype frequencies
### from geno genotypes
g &lt;- do.call(&quot;cbind&quot;, gm$geno)
gf_mar_geno &lt;- t(apply(g, 2, function(a) table(factor(a, 1:2))/sum(a != 0)))
gn_mar_geno &lt;- t(apply(g, 2, function(a) table(factor(a, 0:2))))
#gf_mar_raw&lt;- gf_mar_raw[gf_mar_raw[,2] != &quot;NaN&quot;,]
colnames(gf_mar_geno) &lt;- c(&quot;freq_AA_geno_table&quot;,&quot;freq_AB_geno_table&quot;)
colnames(gn_mar_geno) &lt;- c(&quot;count_missing_geno_table&quot;,&quot;count_AA_geno_table&quot;,&quot;count_AB_geno_table&quot;)
gfn_mar_geno &lt;- merge(as.data.frame(gn_mar_geno), as.data.frame(gf_mar_geno), by=&quot;row.names&quot;)
rownames(gfn_mar_geno) &lt;- gfn_mar_geno[,1]
gfn_mar_geno &lt;- gfn_mar_geno[-1]

### from raw using table function in R
#genosl &lt;- list()
#for(i in 1:nrow(geno)){
##for(i in 1:3){
#    genoi &lt;- geno[i,]
#    freqf &lt;- table(factor(geno[i,], c(&quot;-&quot;,&quot;AA&quot;,&quot;AB&quot;)))
#    genoi$count_AA_raw_rowSums &lt;- rowSums(genoi == &quot;AA&quot;)
#    genoi$count_AB_raw_rowSums &lt;- rowSums(genoi == &quot;AB&quot;)
#    genoi$count_missing_raw_rowSums &lt;- rowSums(genoi == &quot;-&quot;)
#    freqf &lt;- t(table(factor(geno[i,], c(&quot;-&quot;,&quot;AA&quot;,&quot;AB&quot;))))
#    freqf &lt;- as.data.frame(t(freqf[1,]))
#    rownames(freqf) &lt;- rownames(genoi)
#    colnames(freqf) &lt;- c(&quot;count_missing_raw_table&quot;,&quot;count_AA_raw_table&quot;,&quot;count_AB_raw_table&quot;)
#    genoif &lt;- cbind(freqf,genoi[c(&quot;count_AA_raw_rowSums&quot;,&quot;count_AB_raw_rowSums&quot;,&quot;count_missing_raw_rowSums&quot;)])
#    genosl[[i]] = genoif
#}
#gf_mar_raw &lt;- do.call(&quot;rbind&quot;,genosl)
#gf_mar_raw &lt;- gf_mar_raw[,c(1:3,6,4:5)]
#gf_mar_raw$index &lt;- 1:nrow(gf_mar_raw)

### from probabilities
gf_mar_probs.1 &lt;- calc_geno_freq(pr.qc.ids, by = &quot;marker&quot;, omit_x = FALSE)
#gn_mar_probs &lt;- calc_geno_freq(probs, by = &quot;individual&quot;, omit_x = FALSE)
gf_mar_probs &lt;- rbind(gf_mar_probs.1$A[,1:2], gf_mar_probs.1$X[,1:2])
colnames(gf_mar_probs) &lt;- paste0(&quot;freq_&quot;,colnames(gf_mar_probs),&quot;_probs&quot;)
gf_mar_probs &lt;- as.data.frame(gf_mar_probs)
gf_mar_probs$index &lt;- 1:nrow(gf_mar_probs)

### merging all genotype frequecies for all markers
#gf_mar.1 &lt;- merge(as.data.frame(gf_mar_raw), as.data.frame(gfn_mar_geno), by=&quot;row.names&quot;)
#rownames(gf_mar.1) &lt;- gf_mar.1[,1]
#gf_mar.1 &lt;- gf_mar.1[-1]
#gf_mar &lt;- merge(gf_mar.1,as.data.frame(gf_mar_probs), by=&quot;row.names&quot;)
gf_mar &lt;- merge(as.data.frame(gfn_mar_geno),as.data.frame(gf_mar_probs), by=&quot;row.names&quot;)
rownames(gf_mar) &lt;- gf_mar[,1]
gf_mar &lt;- gf_mar[-1]
gf_mar &lt;- gf_mar[order(gf_mar$index),]
dim(gf_mar)</code></pre>
<pre><code>[1] 32660     8</code></pre>
<pre class="r"><code># Calculating ratio and flagging informative marker
gf_mar$ratio = as.numeric(gf_mar$freq_AA_geno_table)/as.numeric(gf_mar$freq_AB_geno_table)
gf_mar$Include = ifelse(gf_mar$ratio &gt;= 0.90 &amp; gf_mar$ratio &lt;= 1.10, TRUE,FALSE)
table(gf_mar$Include)</code></pre>
<pre><code>
FALSE  TRUE 
20578 12082 </code></pre>
<pre class="r"><code>## filtering out &lt;= 0.05
gf_mar$count.geno &lt;- rowSums(gf_mar[c(&quot;freq_AA_geno_table&quot;,&quot;freq_AB_geno_table&quot;)] &lt;=0.05)
filtered_gf_mar_geno &lt;- gf_mar[gf_mar$count.geno != 1,]
filtered_gf_mar_geno &lt;- filtered_gf_mar_geno[,-which(names(filtered_gf_mar_geno) %in% c(&quot;count.geno&quot;,&quot;index&quot;))]
dim(filtered_gf_mar_geno)</code></pre>
<pre><code>[1] 26491     9</code></pre>
<pre class="r"><code>table(filtered_gf_mar_geno$Include)</code></pre>
<pre><code>
FALSE  TRUE 
14409 12082 </code></pre>
<pre class="r"><code>gf_mar$count.probs &lt;- rowSums(gf_mar[c(&quot;freq_AA_probs&quot;,&quot;freq_AB_probs&quot;)] &lt;=0.05)
filtered_gf_mar_probs &lt;- gf_mar[gf_mar$count.probs != 1,]
filtered_gf_mar_probs &lt;- filtered_gf_mar_probs[,-which(names(filtered_gf_mar_probs) %in% c(&quot;count.geno&quot;,&quot;count.probs&quot;,&quot;index&quot;))]
dim(filtered_gf_mar_probs)</code></pre>
<pre><code>[1] 27421     9</code></pre>
<pre class="r"><code>table(filtered_gf_mar_probs$Include)</code></pre>
<pre><code>
FALSE  TRUE 
15339 12082 </code></pre>
<pre class="r"><code>## merging with sample_genos
filtered_gf_mar_geno_sample &lt;- merge(geno,filtered_gf_mar_geno, by=&quot;row.names&quot;, all.y=T, sort=F)
#filtered_gf_mar_geno_sample &lt;- filtered_gf_mar_geno_sample[order(filtered_gf_mar_geno_sample$index),]     
#filtered_gf_mar_geno_sample &lt;- filtered_gf_mar_geno_sample[,-which(names(filtered_gf_mar_geno_sample) %in% c(&quot;count.geno&quot;,&quot;index&quot;))]
names(filtered_gf_mar_geno_sample)[1] &lt;- c(&quot;marker&quot;)
dim(filtered_gf_mar_geno_sample)</code></pre>
<pre><code>[1] 26491   251</code></pre>
<pre class="r"><code>filtered_gf_mar_probs_sample &lt;- merge(geno,filtered_gf_mar_probs, by=&quot;row.names&quot;, all.y=T, sort=F)
#filtered_gf_mar_probs_sample &lt;- filtered_gf_mar_probs_sample[order(filtered_gf_mar_probs_sample$index),]
#filtered_gf_mar_probs_sample &lt;- filtered_gf_mar_probs_sample[,-which(names(filtered_gf_mar_probs_sample) %in% c(&quot;count.geno&quot;,&quot;count.probs&quot;,&quot;index&quot;))]
names(filtered_gf_mar_probs_sample)[1] &lt;- c(&quot;marker&quot;)
dim(filtered_gf_mar_probs_sample)</code></pre>
<pre><code>[1] 27421   251</code></pre>
<pre class="r"><code>## saving files
write.csv(filtered_gf_mar_geno_sample, &quot;data/myo-yes.vs.myo-no_sample.genos_marker.freq_low.geno.freq.removed_sample.outliers.removed_7.batches_myo.csv&quot;, quote=F)
write.csv(filtered_gf_mar_probs_sample, &quot;data/myo-yes.vs.myo-no_sample.genos_marker.freq_low.probs.freq.removed_sample.outliers.removed_7.batches_myo.csv&quot;, quote=F)

write.csv(filtered_gf_mar_geno, &quot;data/myo-yes.vs.myo-no_marker.freq_low.geno.freq.removed_sample.outliers.removed_geno.ratio_7.batches_myo.csv&quot;, quote=F)
write.csv(filtered_gf_mar_probs, &quot;data/myo-yes.vs.myo-no_marker.freq_low.probs.freq.removed_sample.outliers.removed_geno.ratio_7.batches_myo.csv&quot;, quote=F)</code></pre>
<p><img src="figure/genotype.frequencies_myo-yes.vs.myo-no_freq_7.batches_myo.Rmd/pbs%20figures%20dismark%20out-1.png" width="100%" height="100%" style="display: block; margin: auto;" /><img src="figure/genotype.frequencies_myo-yes.vs.myo-no_freq_7.batches_myo.Rmd/pbs%20figures%20dismark%20out-2.png" width="100%" height="100%" style="display: block; margin: auto;" /></p>
<br>
<p>
<button type="button" class="btn btn-default btn-workflowr btn-workflowr-sessioninfo" data-toggle="collapse" data-target="#workflowr-sessioninfo" style="display: block;">
<span class="glyphicon glyphicon-wrench" aria-hidden="true"></span>
Session information
</button>
</p>
<div id="workflowr-sessioninfo" class="collapse">
<pre><code>R version 4.2.2 (2022-10-31)
Platform: x86_64-apple-darwin17.0 (64-bit)
Running under: macOS Big Sur ... 10.16

Matrix products: default
BLAS:   /Library/Frameworks/R.framework/Versions/4.2/Resources/lib/libRblas.0.dylib
LAPACK: /Library/Frameworks/R.framework/Versions/4.2/Resources/lib/libRlapack.dylib

locale:
[1] en_US.UTF-8/en_US.UTF-8/en_US.UTF-8/C/en_US.UTF-8/en_US.UTF-8

attached base packages:
[1] stats     graphics  grDevices utils     datasets  methods   base     

other attached packages:
 [1] ggdark_0.2.1      ggthemes_4.2.4    cowplot_1.1.1     ggnewscale_0.4.8 
 [5] abind_1.4-5       qtl2_0.30         reshape2_1.4.4    ggplot2_3.4.0    
 [9] tibble_3.1.8      psych_2.2.9       readxl_1.4.1      cluster_2.1.4    
[13] dplyr_1.0.10      optparse_1.7.3    rhdf5_2.40.0      mclust_6.0.0     
[17] tidyr_1.2.1       data.table_1.14.6 knitr_1.42        kableExtra_1.3.4 
[21] workflowr_1.7.0  

loaded via a namespace (and not attached):
 [1] httr_1.4.4         sass_0.4.5         bit64_4.0.5        jsonlite_1.8.4    
 [5] viridisLite_0.4.1  bslib_0.4.2        assertthat_0.2.1   getPass_0.2-2     
 [9] highr_0.10         blob_1.2.3         cellranger_1.1.0   yaml_2.3.7        
[13] pillar_1.8.1       RSQLite_2.2.19     lattice_0.20-45    glue_1.6.2        
[17] digest_0.6.31      promises_1.2.0.1   rvest_1.0.3        colorspace_2.0-3  
[21] htmltools_0.5.5    httpuv_1.6.6       plyr_1.8.8         pkgconfig_2.0.3   
[25] purrr_0.3.5        scales_1.2.1       webshot_0.5.4      processx_3.8.0    
[29] svglite_2.1.0      whisker_0.4.1      getopt_1.20.3      later_1.3.0       
[33] git2r_0.30.1       farver_2.1.1       generics_0.1.3     cachem_1.0.7      
[37] withr_2.5.0        cli_3.6.1          mnormt_2.1.1       magrittr_2.0.3    
[41] memoise_2.0.1      evaluate_0.20      ps_1.7.2           fs_1.6.1          
[45] fansi_1.0.3        nlme_3.1-161       xml2_1.3.3         tools_4.2.2       
[49] lifecycle_1.0.3    stringr_1.5.0      Rhdf5lib_1.18.2    munsell_0.5.0     
[53] callr_3.7.3        compiler_4.2.2     jquerylib_0.1.4    systemfonts_1.0.4 
[57] rlang_1.1.0        grid_4.2.2         rhdf5filters_1.8.0 rstudioapi_0.14   
[61] rmarkdown_2.21.2   gtable_0.3.1       DBI_1.1.3          R6_2.5.1          
[65] bit_4.0.5          fastmap_1.1.1      utf8_1.2.2         rprojroot_2.0.3   
[69] stringi_1.7.12     parallel_4.2.2     Rcpp_1.0.9         vctrs_0.6.1       
[73] tidyselect_1.2.0   xfun_0.38         </code></pre>
</div>
</div>


<!-- Adjust MathJax settings so that all math formulae are shown using
TeX fonts only; see
https://docs.mathjax.org/en/latest/web/configuration.html. This will make
the presentation more consistent at the cost of the webpage sometimes
taking slightly longer to load. Note that this only works because the
footer is added to webpages before the MathJax javascript. -->
<script type="text/x-mathjax-config">
  MathJax.Hub.Config({
    "HTML-CSS": { availableFonts: ["TeX"] }
  });
</script>





</div>

<script>

// add bootstrap table styles to pandoc tables
function bootstrapStylePandocTables() {
  $('tr.odd').parent('tbody').parent('table').addClass('table table-condensed');
}
$(document).ready(function () {
  bootstrapStylePandocTables();
});


</script>

<!-- tabsets -->

<script>
$(document).ready(function () {
  window.buildTabsets("TOC");
});

$(document).ready(function () {
  $('.tabset-dropdown > .nav-tabs > li').click(function () {
    $(this).parent().toggleClass('nav-tabs-open');
  });
});
</script>

<!-- code folding -->


<!-- dynamically load mathjax for compatibility with self-contained -->
<script>
  (function () {
    var script = document.createElement("script");
    script.type = "text/javascript";
    script.src  = "https://mathjax.rstudio.com/latest/MathJax.js?config=TeX-AMS-MML_HTMLorMML";
    document.getElementsByTagName("head")[0].appendChild(script);
  })();
</script>

</body>
</html>
