<!DOCTYPE html>

<html>

<head>

<meta charset="utf-8" />
<meta name="generator" content="pandoc" />
<meta http-equiv="X-UA-Compatible" content="IE=EDGE" />


<meta name="author" content="Belinda Cornes" />

<meta name="date" content="2023-04-17" />

<title>Serreze Workflow - 7 batches/6 groups</title>

<script src="site_libs/header-attrs-2.21.2/header-attrs.js"></script>
<script src="site_libs/jquery-3.6.0/jquery-3.6.0.min.js"></script>
<meta name="viewport" content="width=device-width, initial-scale=1" />
<link href="site_libs/bootstrap-3.3.5/css/spacelab.min.css" rel="stylesheet" />
<script src="site_libs/bootstrap-3.3.5/js/bootstrap.min.js"></script>
<script src="site_libs/bootstrap-3.3.5/shim/html5shiv.min.js"></script>
<script src="site_libs/bootstrap-3.3.5/shim/respond.min.js"></script>
<style>h1 {font-size: 34px;}
       h1.title {font-size: 38px;}
       h2 {font-size: 30px;}
       h3 {font-size: 24px;}
       h4 {font-size: 18px;}
       h5 {font-size: 16px;}
       h6 {font-size: 12px;}
       code {color: inherit; background-color: rgba(0, 0, 0, 0.04);}
       pre:not([class]) { background-color: white }</style>
<script src="site_libs/navigation-1.1/tabsets.js"></script>
<link href="site_libs/highlightjs-9.12.0/textmate.css" rel="stylesheet" />
<script src="site_libs/highlightjs-9.12.0/highlight.js"></script>
<script src="site_libs/kePrint-0.0.1/kePrint.js"></script>
<link href="site_libs/lightable-0.0.1/lightable.css" rel="stylesheet" />
<link href="site_libs/font-awesome-6.2.1/css/all.min.css" rel="stylesheet" />
<link href="site_libs/font-awesome-6.2.1/css/v4-shims.min.css" rel="stylesheet" />

<link rel="icon" href="https://github.com/workflowr/workflowr-assets/raw/main/img/reproducible.png">
<!-- Add a small amount of space between sections. -->
<style type="text/css">
div.section {
  padding-top: 12px;
}
</style>



<style type="text/css">
  code{white-space: pre-wrap;}
  span.smallcaps{font-variant: small-caps;}
  span.underline{text-decoration: underline;}
  div.column{display: inline-block; vertical-align: top; width: 50%;}
  div.hanging-indent{margin-left: 1.5em; text-indent: -1.5em;}
  ul.task-list{list-style: none;}
    </style>

<style type="text/css">code{white-space: pre;}</style>
<script type="text/javascript">
if (window.hljs) {
  hljs.configure({languages: []});
  hljs.initHighlightingOnLoad();
  if (document.readyState && document.readyState === "complete") {
    window.setTimeout(function() { hljs.initHighlighting(); }, 0);
  }
}
</script>









<style type = "text/css">
.main-container {
  max-width: 940px;
  margin-left: auto;
  margin-right: auto;
}
img {
  max-width:100%;
}
.tabbed-pane {
  padding-top: 12px;
}
.html-widget {
  margin-bottom: 20px;
}
button.code-folding-btn:focus {
  outline: none;
}
summary {
  display: list-item;
}
details > summary > p:only-child {
  display: inline;
}
pre code {
  padding: 0;
}
</style>


<style type="text/css">
.dropdown-submenu {
  position: relative;
}
.dropdown-submenu>.dropdown-menu {
  top: 0;
  left: 100%;
  margin-top: -6px;
  margin-left: -1px;
  border-radius: 0 6px 6px 6px;
}
.dropdown-submenu:hover>.dropdown-menu {
  display: block;
}
.dropdown-submenu>a:after {
  display: block;
  content: " ";
  float: right;
  width: 0;
  height: 0;
  border-color: transparent;
  border-style: solid;
  border-width: 5px 0 5px 5px;
  border-left-color: #cccccc;
  margin-top: 5px;
  margin-right: -10px;
}
.dropdown-submenu:hover>a:after {
  border-left-color: #adb5bd;
}
.dropdown-submenu.pull-left {
  float: none;
}
.dropdown-submenu.pull-left>.dropdown-menu {
  left: -100%;
  margin-left: 10px;
  border-radius: 6px 0 6px 6px;
}
</style>

<script type="text/javascript">
// manage active state of menu based on current page
$(document).ready(function () {
  // active menu anchor
  href = window.location.pathname
  href = href.substr(href.lastIndexOf('/') + 1)
  if (href === "")
    href = "index.html";
  var menuAnchor = $('a[href="' + href + '"]');

  // mark the anchor link active (and if it's in a dropdown, also mark that active)
  var dropdown = menuAnchor.closest('li.dropdown');
  if (window.bootstrap) { // Bootstrap 4+
    menuAnchor.addClass('active');
    dropdown.find('> .dropdown-toggle').addClass('active');
  } else { // Bootstrap 3
    menuAnchor.parent().addClass('active');
    dropdown.addClass('active');
  }

  // Navbar adjustments
  var navHeight = $(".navbar").first().height() + 15;
  var style = document.createElement('style');
  var pt = "padding-top: " + navHeight + "px; ";
  var mt = "margin-top: -" + navHeight + "px; ";
  var css = "";
  // offset scroll position for anchor links (for fixed navbar)
  for (var i = 1; i <= 6; i++) {
    css += ".section h" + i + "{ " + pt + mt + "}\n";
  }
  style.innerHTML = "body {" + pt + "padding-bottom: 40px; }\n" + css;
  document.head.appendChild(style);
});
</script>

<!-- tabsets -->

<style type="text/css">
.tabset-dropdown > .nav-tabs {
  display: inline-table;
  max-height: 500px;
  min-height: 44px;
  overflow-y: auto;
  border: 1px solid #ddd;
  border-radius: 4px;
}

.tabset-dropdown > .nav-tabs > li.active:before, .tabset-dropdown > .nav-tabs.nav-tabs-open:before {
  content: "\e259";
  font-family: 'Glyphicons Halflings';
  display: inline-block;
  padding: 10px;
  border-right: 1px solid #ddd;
}

.tabset-dropdown > .nav-tabs.nav-tabs-open > li.active:before {
  content: "\e258";
  font-family: 'Glyphicons Halflings';
  border: none;
}

.tabset-dropdown > .nav-tabs > li.active {
  display: block;
}

.tabset-dropdown > .nav-tabs > li > a,
.tabset-dropdown > .nav-tabs > li > a:focus,
.tabset-dropdown > .nav-tabs > li > a:hover {
  border: none;
  display: inline-block;
  border-radius: 4px;
  background-color: transparent;
}

.tabset-dropdown > .nav-tabs.nav-tabs-open > li {
  display: block;
  float: none;
}

.tabset-dropdown > .nav-tabs > li {
  display: none;
}
</style>

<!-- code folding -->




</head>

<body>


<div class="container-fluid main-container">




<div class="navbar navbar-default  navbar-fixed-top" role="navigation">
  <div class="container">
    <div class="navbar-header">
      <button type="button" class="navbar-toggle collapsed" data-toggle="collapse" data-bs-toggle="collapse" data-target="#navbar" data-bs-target="#navbar">
        <span class="icon-bar"></span>
        <span class="icon-bar"></span>
        <span class="icon-bar"></span>
      </button>
      <a class="navbar-brand" href="index.html">Serreze Autoimmune Workflow</a>
    </div>
    <div id="navbar" class="navbar-collapse collapse">
      <ul class="nav navbar-nav">
        <li>
  <a href="index.html">Home</a>
</li>
<li>
  <a href="about.html">About</a>
</li>
<li>
  <a href="license.html">License</a>
</li>
      </ul>
      <ul class="nav navbar-nav navbar-right">
        <li>
  <a href="https://github.com/TheJacksonLaboratory/Serreze-T1D_Workflow">
    <span class="fa fa-github"></span>
     
    Source code
  </a>
</li>
      </ul>
    </div><!--/.nav-collapse -->
  </div><!--/.container -->
</div><!--/.navbar -->

<div id="header">



<h1 class="title toc-ignore">Serreze Workflow - 7 batches/6 groups</h1>
<h4 class="author">Belinda Cornes</h4>
<h4 class="date">2023-04-17</h4>

</div>

<div id="TOC">
<ul>
<li><a href="#neogen-files-at-a-glance-total-samples-366"
id="toc-neogen-files-at-a-glance-total-samples-366"><strong>Neogen Files
At A Glance</strong> [Total Samples: 366]</a></li>
<li><a href="#haplotype-phasing"
id="toc-haplotype-phasing"><strong>Haplotype Phasing</strong></a></li>
<li><a href="#experiment-design" id="toc-experiment-design"><strong>1.
Experiment Design</strong></a></li>
<li><a href="#genotype-qc" id="toc-genotype-qc"><strong>2. Genotype QC
</strong></a></li>
<li><a href="#phenotype-qc" id="toc-phenotype-qc"><strong>3. Phenotype
QC</strong></a>
<ul>
<li><a
href="#continous-phenotypes-age-of-onset-not-necessary-for-current-analysis"
id="toc-continous-phenotypes-age-of-onset-not-necessary-for-current-analysis">continous
phenotypes (age of onset) [not necessary for current analysis]</a></li>
</ul></li>
<li><a href="#qtl-analysis" id="toc-qtl-analysis"><strong>4. QTL
Analysis</strong></a>
<ul>
<li><a href="#binary-phenotpe" id="toc-binary-phenotpe"><strong>binary
phenotpe</strong></a></li>
</ul></li>
<li><a href="#power-analysis" id="toc-power-analysis"><strong>5. Power
Analysis</strong></a></li>
</ul>
</div>

<p>
<button type="button" class="btn btn-default btn-workflowr btn-workflowr-report" data-toggle="collapse" data-target="#workflowr-report">
<span class="glyphicon glyphicon-list" aria-hidden="true"></span>
workflowr <span class="glyphicon glyphicon-exclamation-sign text-danger"
aria-hidden="true"></span>
</button>
</p>
<div id="workflowr-report" class="collapse">
<ul class="nav nav-tabs">
<li class="active">
<a data-toggle="tab" href="#summary">Summary</a>
</li>
<li>
<a data-toggle="tab" href="#checks"> Checks <span
class="glyphicon glyphicon-exclamation-sign text-danger"
aria-hidden="true"></span> </a>
</li>
<li>
<a data-toggle="tab" href="#versions">Past versions</a>
</li>
</ul>
<div class="tab-content">
<div id="summary" class="tab-pane fade in active">
<p>
<strong>Last updated:</strong> 2023-04-17
</p>
<p>
<strong>Checks:</strong> <span
class="glyphicon glyphicon-ok text-success" aria-hidden="true"></span> 6
<span class="glyphicon glyphicon-exclamation-sign text-danger"
aria-hidden="true"></span> 1
</p>
<p>
<strong>Knit directory:</strong> <code>Serreze-T1D_Workflow/</code>
<span class="glyphicon glyphicon-question-sign" aria-hidden="true"
title="This is the local directory in which the code in this file was executed.">
</span>
</p>
<p>
This reproducible <a href="https://rmarkdown.rstudio.com">R Markdown</a>
analysis was created with <a
  href="https://github.com/workflowr/workflowr">workflowr</a> (version
1.7.0). The <em>Checks</em> tab describes the reproducibility checks
that were applied when the results were created. The <em>Past
versions</em> tab lists the development history.
</p>
<hr>
</div>
<div id="checks" class="tab-pane fade">
<div id="workflowr-checks" class="panel-group">
<div class="panel panel-default">
<div class="panel-heading">
<p class="panel-title">
<a data-toggle="collapse" data-parent="#workflowr-checks" href="#strongRMarkdownfilestronguncommittedchanges">
<span class="glyphicon glyphicon-exclamation-sign text-danger"
aria-hidden="true"></span> <strong>R Markdown file:</strong> uncommitted
changes </a>
</p>
</div>
<div id="strongRMarkdownfilestronguncommittedchanges"
class="panel-collapse collapse">
<div class="panel-body">
<p>The R Markdown file has unstaged changes. To know which version of
the R Markdown file created these results, you’ll want to first commit
it to the Git repo. If you’re still working on the analysis, you can
ignore this warning. When you’re finished, you can run
<code>wflow_publish</code> to commit the R Markdown file and build the
HTML.</p>
</div>
</div>
</div>
<div class="panel panel-default">
<div class="panel-heading">
<p class="panel-title">
<a data-toggle="collapse" data-parent="#workflowr-checks" href="#strongEnvironmentstrongempty">
<span class="glyphicon glyphicon-ok text-success"
aria-hidden="true"></span> <strong>Environment:</strong> empty </a>
</p>
</div>
<div id="strongEnvironmentstrongempty" class="panel-collapse collapse">
<div class="panel-body">
<p>Great job! The global environment was empty. Objects defined in the
global environment can affect the analysis in your R Markdown file in
unknown ways. For reproduciblity it’s best to always run the code in an
empty environment.</p>
</div>
</div>
</div>
<div class="panel panel-default">
<div class="panel-heading">
<p class="panel-title">
<a data-toggle="collapse" data-parent="#workflowr-checks" href="#strongSeedstrongcodesetseed20220210code">
<span class="glyphicon glyphicon-ok text-success"
aria-hidden="true"></span> <strong>Seed:</strong>
<code>set.seed(20220210)</code> </a>
</p>
</div>
<div id="strongSeedstrongcodesetseed20220210code"
class="panel-collapse collapse">
<div class="panel-body">
<p>The command <code>set.seed(20220210)</code> was run prior to running
the code in the R Markdown file. Setting a seed ensures that any results
that rely on randomness, e.g. subsampling or permutations, are
reproducible.</p>
</div>
</div>
</div>
<div class="panel panel-default">
<div class="panel-heading">
<p class="panel-title">
<a data-toggle="collapse" data-parent="#workflowr-checks" href="#strongSessioninformationstrongrecorded">
<span class="glyphicon glyphicon-ok text-success"
aria-hidden="true"></span> <strong>Session information:</strong>
recorded </a>
</p>
</div>
<div id="strongSessioninformationstrongrecorded"
class="panel-collapse collapse">
<div class="panel-body">
<p>Great job! Recording the operating system, R version, and package
versions is critical for reproducibility.</p>
</div>
</div>
</div>
<div class="panel panel-default">
<div class="panel-heading">
<p class="panel-title">
<a data-toggle="collapse" data-parent="#workflowr-checks" href="#strongCachestrongnone">
<span class="glyphicon glyphicon-ok text-success"
aria-hidden="true"></span> <strong>Cache:</strong> none </a>
</p>
</div>
<div id="strongCachestrongnone" class="panel-collapse collapse">
<div class="panel-body">
<p>Nice! There were no cached chunks for this analysis, so you can be
confident that you successfully produced the results during this
run.</p>
</div>
</div>
</div>
<div class="panel panel-default">
<div class="panel-heading">
<p class="panel-title">
<a data-toggle="collapse" data-parent="#workflowr-checks" href="#strongFilepathsstrongrelative">
<span class="glyphicon glyphicon-ok text-success"
aria-hidden="true"></span> <strong>File paths:</strong> relative </a>
</p>
</div>
<div id="strongFilepathsstrongrelative" class="panel-collapse collapse">
<div class="panel-body">
<p>Great job! Using relative paths to the files within your workflowr
project makes it easier to run your code on other machines.</p>
</div>
</div>
</div>
<div class="panel panel-default">
<div class="panel-heading">
<p class="panel-title">
<a data-toggle="collapse" data-parent="#workflowr-checks" href="#strongRepositoryversionstrongahrefhttpsgithubcomTheJacksonLaboratorySerrezeT1DWorkflowtree52c77a491baee18d975ed7b5d07fa2bbee413ec9targetblank52c77a4a">
<span class="glyphicon glyphicon-ok text-success"
aria-hidden="true"></span> <strong>Repository version:</strong>
<a href="https://github.com/TheJacksonLaboratory/Serreze-T1D_Workflow/tree/52c77a491baee18d975ed7b5d07fa2bbee413ec9" target="_blank">52c77a4</a>
</a>
</p>
</div>
<div
id="strongRepositoryversionstrongahrefhttpsgithubcomTheJacksonLaboratorySerrezeT1DWorkflowtree52c77a491baee18d975ed7b5d07fa2bbee413ec9targetblank52c77a4a"
class="panel-collapse collapse">
<div class="panel-body">
<p>
Great! You are using Git for version control. Tracking code development
and connecting the code version to the results is critical for
reproducibility.
</p>
<p>
The results in this page were generated with repository version
<a href="https://github.com/TheJacksonLaboratory/Serreze-T1D_Workflow/tree/52c77a491baee18d975ed7b5d07fa2bbee413ec9" target="_blank">52c77a4</a>.
See the <em>Past versions</em> tab to see a history of the changes made
to the R Markdown and HTML files.
</p>
<p>
Note that you need to be careful to ensure that all relevant files for
the analysis have been committed to Git prior to generating the results
(you can use <code>wflow_publish</code> or
<code>wflow_git_commit</code>). workflowr only checks the R Markdown
file, but you know if there are other scripts or data files that it
depends on. Below is the status of the Git repository when the results
were generated:
</p>
<pre><code>
Ignored files:
    Ignored:    .DS_Store
    Ignored:    analysis/.DS_Store

Untracked files:
    Untracked:  analysis/0.1.1_preparing.data_bqc_7.batches_myo.R
    Untracked:  analysis/0.2_haplotype_comparison_bqc_7.batches_myo_minprob.R
    Untracked:  analysis/2.1_sample_bqc_7.batches_myo.R
    Untracked:  analysis/2.1_sample_bqc_7.batches_myo.Rmd.R
    Untracked:  analysis/2.2.1_snp_qc_7.batches_myo.R
    Untracked:  analysis/2.2.1_snp_qc_7.batches_myo_mis.R
    Untracked:  analysis/2.4_preparing.data_aqc_7.batches_myo_misd.R
    Untracked:  analysis/4.1.1_qtl.analysis_binary_het-ici-myo-yes.vs.het-ici-myo-no_snpsqc_7.batches_myo.R
    Untracked:  analysis/4.1.1_qtl.analysis_binary_het-ici-myo-yes.vs.het-ici-myo-no_snpsqc_7.batches_myo.Rmd
    Untracked:  analysis/4.1.1_qtl.analysis_binary_het-ici-myo-yes.vs.het-ici-myo-no_snpsqc_7.batches_myo_mis.R
    Untracked:  analysis/4.1.1_qtl.analysis_binary_het-ici-myo-yes.vs.het-ici-myo-no_snpsqc_7.batches_myo_mis.Rmd
    Untracked:  analysis/4.1.1_qtl.analysis_binary_het-ici-myo-yes.vs.het-ici-myo-no_snpsqc_dis_no-x_updated_7.batches_myo.R
    Untracked:  analysis/4.1.1_qtl.analysis_binary_het-ici-myo-yes.vs.het-ici-myo-no_snpsqc_dis_no-x_updated_7.batches_myo.Rmd
    Untracked:  analysis/4.1.1_qtl.analysis_binary_het-ici-myo-yes.vs.het-ici-myo-no_snpsqc_dis_no-x_updated_7.batches_myo_mis.Rmd
    Untracked:  analysis/4.1.1_qtl.analysis_binary_het-ici-myo-yes.vs.het-ici-myo-no_snpsqc_dis_no-x_updated_no-sex_7.batches_myo.Rmd
    Untracked:  analysis/4.1.1_qtl.analysis_binary_het-ici-myo-yes.vs.het-ici-myo-no_snpsqc_dis_no-x_updated_no-sex_7.batches_myo_mis.Rmd
    Untracked:  analysis/4.1.1_qtl.analysis_binary_het-ici-myo-yes.vs.het-ici-myo-no_snpsqc_no-sex_7.batches_myo.R
    Untracked:  analysis/4.1.1_qtl.analysis_binary_het-ici-myo-yes.vs.het-ici-myo-no_snpsqc_no-sex_7.batches_myo.Rmd
    Untracked:  analysis/4.1.1_qtl.analysis_binary_het-ici-myo-yes.vs.het-ici-myo-no_snpsqc_no-sex_7.batches_myo_mis.R
    Untracked:  analysis/4.1.1_qtl.analysis_binary_het-ici-myo-yes.vs.het-ici-myo-no_snpsqc_no-sex_7.batches_myo_mis.Rmd
    Untracked:  analysis/4.1.1_qtl.analysis_binary_het-ici.vs.het-pbs_snpsqc_7.batches_myo.R
    Untracked:  analysis/4.1.1_qtl.analysis_binary_het-ici.vs.het-pbs_snpsqc_7.batches_myo.Rmd
    Untracked:  analysis/4.1.1_qtl.analysis_binary_het-ici.vs.het-pbs_snpsqc_7.batches_myo_mis.R
    Untracked:  analysis/4.1.1_qtl.analysis_binary_het-ici.vs.het-pbs_snpsqc_7.batches_myo_mis.Rmd
    Untracked:  analysis/4.1.1_qtl.analysis_binary_het-ici.vs.het-pbs_snpsqc_dis_no-x_updated_7.batches_myo.R
    Untracked:  analysis/4.1.1_qtl.analysis_binary_het-ici.vs.het-pbs_snpsqc_dis_no-x_updated_7.batches_myo.Rmd
    Untracked:  analysis/4.1.1_qtl.analysis_binary_het-ici.vs.het-pbs_snpsqc_dis_no-x_updated_7.batches_myo_mis.Rmd
    Untracked:  analysis/4.1.1_qtl.analysis_binary_het-ici.vs.het-pbs_snpsqc_dis_no-x_updated_no-sex_7.batches_myo.Rmd
    Untracked:  analysis/4.1.1_qtl.analysis_binary_het-ici.vs.het-pbs_snpsqc_dis_no-x_updated_no-sex_7.batches_myo_mis.Rmd
    Untracked:  analysis/4.1.1_qtl.analysis_binary_het-ici.vs.het-pbs_snpsqc_no-sex_7.batches_myo.R
    Untracked:  analysis/4.1.1_qtl.analysis_binary_het-ici.vs.het-pbs_snpsqc_no-sex_7.batches_myo.Rmd
    Untracked:  analysis/4.1.1_qtl.analysis_binary_het-ici.vs.het-pbs_snpsqc_no-sex_7.batches_myo_mis.R
    Untracked:  analysis/4.1.1_qtl.analysis_binary_het-ici.vs.het-pbs_snpsqc_no-sex_7.batches_myo_mis.Rmd
    Untracked:  analysis/4.1.1_qtl.analysis_binary_ici-myo-yes.vs.ici-myo-no_snpsqc_7.batches_myo.R
    Untracked:  analysis/4.1.1_qtl.analysis_binary_ici-myo-yes.vs.ici-myo-no_snpsqc_7.batches_myo.Rmd
    Untracked:  analysis/4.1.1_qtl.analysis_binary_ici-myo-yes.vs.ici-myo-no_snpsqc_7.batches_myo_mis.R
    Untracked:  analysis/4.1.1_qtl.analysis_binary_ici-myo-yes.vs.ici-myo-no_snpsqc_7.batches_myo_mis.Rmd
    Untracked:  analysis/4.1.1_qtl.analysis_binary_ici-myo-yes.vs.ici-myo-no_snpsqc_dis_no-x_updated_7.batches_myo.R
    Untracked:  analysis/4.1.1_qtl.analysis_binary_ici-myo-yes.vs.ici-myo-no_snpsqc_dis_no-x_updated_7.batches_myo.Rmd
    Untracked:  analysis/4.1.1_qtl.analysis_binary_ici-myo-yes.vs.ici-myo-no_snpsqc_dis_no-x_updated_7.batches_myo_mis.Rmd
    Untracked:  analysis/4.1.1_qtl.analysis_binary_ici-myo-yes.vs.ici-myo-no_snpsqc_dis_no-x_updated_no-sex_7.batches_myo.Rmd
    Untracked:  analysis/4.1.1_qtl.analysis_binary_ici-myo-yes.vs.ici-myo-no_snpsqc_dis_no-x_updated_no-sex_7.batches_myo_mis.Rmd
    Untracked:  analysis/4.1.1_qtl.analysis_binary_ici-myo-yes.vs.ici-myo-no_snpsqc_no-sex_7.batches_myo.R
    Untracked:  analysis/4.1.1_qtl.analysis_binary_ici-myo-yes.vs.ici-myo-no_snpsqc_no-sex_7.batches_myo.Rmd
    Untracked:  analysis/4.1.1_qtl.analysis_binary_ici-myo-yes.vs.ici-myo-no_snpsqc_no-sex_7.batches_myo_mis.R
    Untracked:  analysis/4.1.1_qtl.analysis_binary_ici-myo-yes.vs.ici-myo-no_snpsqc_no-sex_7.batches_myo_mis.Rmd
    Untracked:  analysis/4.1.1_qtl.analysis_binary_ici-sick.vs.ici-eoi_snpsqc_7.batches_myo.R
    Untracked:  analysis/4.1.1_qtl.analysis_binary_ici-sick.vs.ici-eoi_snpsqc_7.batches_myo.Rmd
    Untracked:  analysis/4.1.1_qtl.analysis_binary_ici-sick.vs.ici-eoi_snpsqc_7.batches_myo_mis.R
    Untracked:  analysis/4.1.1_qtl.analysis_binary_ici-sick.vs.ici-eoi_snpsqc_7.batches_myo_mis.Rmd
    Untracked:  analysis/4.1.1_qtl.analysis_binary_ici-sick.vs.ici-eoi_snpsqc_dis_no-x_updated_7.batches_myo.R
    Untracked:  analysis/4.1.1_qtl.analysis_binary_ici-sick.vs.ici-eoi_snpsqc_dis_no-x_updated_7.batches_myo.Rmd
    Untracked:  analysis/4.1.1_qtl.analysis_binary_ici-sick.vs.ici-eoi_snpsqc_dis_no-x_updated_7.batches_myo_mis.Rmd
    Untracked:  analysis/4.1.1_qtl.analysis_binary_ici-sick.vs.ici-eoi_snpsqc_dis_no-x_updated_no-sex_7.batches_myo.Rmd
    Untracked:  analysis/4.1.1_qtl.analysis_binary_ici-sick.vs.ici-eoi_snpsqc_dis_no-x_updated_no-sex_7.batches_myo_mis.Rmd
    Untracked:  analysis/4.1.1_qtl.analysis_binary_ici-sick.vs.ici-eoi_snpsqc_no-sex_7.batches_myo.R
    Untracked:  analysis/4.1.1_qtl.analysis_binary_ici-sick.vs.ici-eoi_snpsqc_no-sex_7.batches_myo.Rmd
    Untracked:  analysis/4.1.1_qtl.analysis_binary_ici-sick.vs.ici-eoi_snpsqc_no-sex_7.batches_myo_mis.R
    Untracked:  analysis/4.1.1_qtl.analysis_binary_ici-sick.vs.ici-eoi_snpsqc_no-sex_7.batches_myo_mis.Rmd
    Untracked:  analysis/4.1.1_qtl.analysis_binary_ici.vs.pbs_snpsqc_7.batches_myo.R
    Untracked:  analysis/4.1.1_qtl.analysis_binary_ici.vs.pbs_snpsqc_7.batches_myo.Rmd
    Untracked:  analysis/4.1.1_qtl.analysis_binary_ici.vs.pbs_snpsqc_7.batches_myo_mis.R
    Untracked:  analysis/4.1.1_qtl.analysis_binary_ici.vs.pbs_snpsqc_7.batches_myo_mis.Rmd
    Untracked:  analysis/4.1.1_qtl.analysis_binary_ici.vs.pbs_snpsqc_dis_no-x_updated_7.batches_myo.R
    Untracked:  analysis/4.1.1_qtl.analysis_binary_ici.vs.pbs_snpsqc_dis_no-x_updated_7.batches_myo.Rmd
    Untracked:  analysis/4.1.1_qtl.analysis_binary_ici.vs.pbs_snpsqc_dis_no-x_updated_7.batches_myo_mis.Rmd
    Untracked:  analysis/4.1.1_qtl.analysis_binary_ici.vs.pbs_snpsqc_dis_no-x_updated_no-sex_7.batches_myo.Rmd
    Untracked:  analysis/4.1.1_qtl.analysis_binary_ici.vs.pbs_snpsqc_dis_no-x_updated_no-sex_7.batches_myo_mis.Rmd
    Untracked:  analysis/4.1.1_qtl.analysis_binary_ici.vs.pbs_snpsqc_no-sex_7.batches_myo.R
    Untracked:  analysis/4.1.1_qtl.analysis_binary_ici.vs.pbs_snpsqc_no-sex_7.batches_myo.Rmd
    Untracked:  analysis/4.1.1_qtl.analysis_binary_ici.vs.pbs_snpsqc_no-sex_7.batches_myo_mis.R
    Untracked:  analysis/4.1.1_qtl.analysis_binary_ici.vs.pbs_snpsqc_no-sex_7.batches_myo_mis.Rmd
    Untracked:  analysis/4.1.1_qtl.analysis_binary_myo-yes.vs.myo-no_snpsqc_7.batches_myo.R
    Untracked:  analysis/4.1.1_qtl.analysis_binary_myo-yes.vs.myo-no_snpsqc_7.batches_myo.Rmd
    Untracked:  analysis/4.1.1_qtl.analysis_binary_myo-yes.vs.myo-no_snpsqc_7.batches_myo_mis.R
    Untracked:  analysis/4.1.1_qtl.analysis_binary_myo-yes.vs.myo-no_snpsqc_7.batches_myo_mis.Rmd
    Untracked:  analysis/4.1.1_qtl.analysis_binary_myo-yes.vs.myo-no_snpsqc_dis_no-x_updated_7.batches_myo.R
    Untracked:  analysis/4.1.1_qtl.analysis_binary_myo-yes.vs.myo-no_snpsqc_dis_no-x_updated_7.batches_myo.Rmd
    Untracked:  analysis/4.1.1_qtl.analysis_binary_myo-yes.vs.myo-no_snpsqc_dis_no-x_updated_7.batches_myo_mis.Rmd
    Untracked:  analysis/4.1.1_qtl.analysis_binary_myo-yes.vs.myo-no_snpsqc_dis_no-x_updated_no-sex_7.batches_myo.Rmd
    Untracked:  analysis/4.1.1_qtl.analysis_binary_myo-yes.vs.myo-no_snpsqc_dis_no-x_updated_no-sex_7.batches_myo_mis.Rmd
    Untracked:  analysis/4.1.1_qtl.analysis_binary_myo-yes.vs.myo-no_snpsqc_no-sex_7.batches_myo.R
    Untracked:  analysis/4.1.1_qtl.analysis_binary_myo-yes.vs.myo-no_snpsqc_no-sex_7.batches_myo.Rmd
    Untracked:  analysis/4.1.1_qtl.analysis_binary_myo-yes.vs.myo-no_snpsqc_no-sex_7.batches_myo_mis.R
    Untracked:  analysis/4.1.1_qtl.analysis_binary_myo-yes.vs.myo-no_snpsqc_no-sex_7.batches_myo_mis.Rmd
    Untracked:  analysis/4.1.1_qtl.analysis_binary_pbs-myo-yes.vs.pbs-myo-no_snpsqc_7.batches_myo.R
    Untracked:  analysis/4.1.1_qtl.analysis_binary_pbs-myo-yes.vs.pbs-myo-no_snpsqc_7.batches_myo.Rmd
    Untracked:  analysis/4.1.1_qtl.analysis_binary_pbs-myo-yes.vs.pbs-myo-no_snpsqc_7.batches_myo_mis.R
    Untracked:  analysis/4.1.1_qtl.analysis_binary_pbs-myo-yes.vs.pbs-myo-no_snpsqc_7.batches_myo_mis.Rmd
    Untracked:  analysis/4.1.1_qtl.analysis_binary_pbs-myo-yes.vs.pbs-myo-no_snpsqc_dis_no-x_updated_7.batches_myo.R
    Untracked:  analysis/4.1.1_qtl.analysis_binary_pbs-myo-yes.vs.pbs-myo-no_snpsqc_dis_no-x_updated_7.batches_myo.Rmd
    Untracked:  analysis/4.1.1_qtl.analysis_binary_pbs-myo-yes.vs.pbs-myo-no_snpsqc_dis_no-x_updated_7.batches_myo_mis.Rmd
    Untracked:  analysis/4.1.1_qtl.analysis_binary_pbs-myo-yes.vs.pbs-myo-no_snpsqc_dis_no-x_updated_no-sex_7.batches_myo.Rmd
    Untracked:  analysis/4.1.1_qtl.analysis_binary_pbs-myo-yes.vs.pbs-myo-no_snpsqc_dis_no-x_updated_no-sex_7.batches_myo_mis.Rmd
    Untracked:  analysis/4.1.1_qtl.analysis_binary_pbs-myo-yes.vs.pbs-myo-no_snpsqc_no-sex_7.batches_myo.R
    Untracked:  analysis/4.1.1_qtl.analysis_binary_pbs-myo-yes.vs.pbs-myo-no_snpsqc_no-sex_7.batches_myo.Rmd
    Untracked:  analysis/4.1.1_qtl.analysis_binary_pbs-myo-yes.vs.pbs-myo-no_snpsqc_no-sex_7.batches_myo_mis.R
    Untracked:  analysis/4.1.1_qtl.analysis_binary_pbs-myo-yes.vs.pbs-myo-no_snpsqc_no-sex_7.batches_myo_mis.Rmd
    Untracked:  analysis/debugging_comparing_old_Rmd/
    Untracked:  analysis/frequency.plot.R
    Untracked:  analysis/genotype.frequencies_het-ici-myo-yes.vs.het-ici-myo-no_7.batches_myo.Rmd
    Untracked:  analysis/genotype.frequencies_het-ici-myo-yes.vs.het-ici-myo-no_7.batches_myo_mis.Rmd
    Untracked:  analysis/genotype.frequencies_het-ici.vs.het-pbs_7.batches_myo.Rmd
    Untracked:  analysis/genotype.frequencies_het-ici.vs.het-pbs_7.batches_myo_mis.Rmd
    Untracked:  analysis/genotype.frequencies_ici-myo-yes.vs.ici-myo-no_7.batches_myo.Rmd
    Untracked:  analysis/genotype.frequencies_ici-myo-yes.vs.ici-myo-no_7.batches_myo_mis.Rmd
    Untracked:  analysis/genotype.frequencies_ici-sick.vs.ici-eoi_7.batches_myo.Rmd
    Untracked:  analysis/genotype.frequencies_ici-sick.vs.ici-eoi_7.batches_myo.Rmd.r
    Untracked:  analysis/genotype.frequencies_ici-sick.vs.ici-eoi_7.batches_myo_edit.Rmd
    Untracked:  analysis/genotype.frequencies_ici-sick.vs.ici-eoi_7.batches_myo_mis.Rmd
    Untracked:  analysis/genotype.frequencies_ici.vs.pbs_7.batches_myo.Rmd
    Untracked:  analysis/genotype.frequencies_ici.vs.pbs_7.batches_myo_mis.Rmd
    Untracked:  analysis/genotype.frequencies_myo-yes.vs.myo-no_7.batches_myo.Rmd
    Untracked:  analysis/genotype.frequencies_myo-yes.vs.myo-no_7.batches_myo_mis.Rmd
    Untracked:  analysis/genotype.frequencies_pbs-myo-yes.vs.pbs-myo-no_7.batches_myo.Rmd
    Untracked:  analysis/genotype.frequencies_pbs-myo-yes.vs.pbs-myo-no_7.batches_myo_mis.Rmd
    Untracked:  analysis/power.analysis_ici.vs.pbs_7.batches_myo_mis.Rmd
    Untracked:  data/GM_covar_4.batches_myo.csv
    Untracked:  data/GM_covar_7.batches_myo.csv
    Untracked:  data/bad_markers_all_4.batches_myo.RData
    Untracked:  data/bad_markers_all_7.batches_myo.RData
    Untracked:  data/covar_corrected.cleaned_het-ici-myo-yes.vs.het-ici-myo-no_7.batches_myo.csv
    Untracked:  data/covar_corrected.cleaned_het-ici-myo-yes.vs.het-ici-myo-no_7.batches_myo_mis.csv
    Untracked:  data/covar_corrected.cleaned_het-ici.vs.het-pbs_7.batches_myo.csv
    Untracked:  data/covar_corrected.cleaned_het-ici.vs.het-pbs_7.batches_myo_mis.csv
    Untracked:  data/covar_corrected.cleaned_ici-myo-yes.vs.ici-myo-no_7.batches_myo.csv
    Untracked:  data/covar_corrected.cleaned_ici-myo-yes.vs.ici-myo-no_7.batches_myo_mis.csv
    Untracked:  data/covar_corrected.cleaned_ici-sick.vs.ici-eoi_7.batches_myo.csv
    Untracked:  data/covar_corrected.cleaned_ici-sick.vs.ici-eoi_7.batches_myo_mis.csv
    Untracked:  data/covar_corrected.cleaned_ici.vs.pbs_7.batches_myo.csv
    Untracked:  data/covar_corrected.cleaned_ici.vs.pbs_7.batches_myo_mis.csv
    Untracked:  data/covar_corrected.cleaned_myo-yes.vs.myo-no_7.batches_myo.csv
    Untracked:  data/covar_corrected.cleaned_myo-yes.vs.myo-no_7.batches_myo_mis.csv
    Untracked:  data/covar_corrected.cleaned_pbs-myo-yes.vs.pbs-myo-no_7.batches_myo.csv
    Untracked:  data/covar_corrected.cleaned_pbs-myo-yes.vs.pbs-myo-no_7.batches_myo_mis.csv
    Untracked:  data/covar_corrected_het-ici-myo-yes.vs.het-ici-myo-no_7.batches_myo.csv
    Untracked:  data/covar_corrected_het-ici-myo-yes.vs.het-ici-myo-no_7.batches_myo_mis.csv
    Untracked:  data/covar_corrected_het-ici.vs.het-pbs_7.batches_myo.csv
    Untracked:  data/covar_corrected_het-ici.vs.het-pbs_7.batches_myo_mis.csv
    Untracked:  data/covar_corrected_ici-myo-yes.vs.ici-myo-no_7.batches_myo.csv
    Untracked:  data/covar_corrected_ici-myo-yes.vs.ici-myo-no_7.batches_myo_mis.csv
    Untracked:  data/covar_corrected_ici-sick.vs.ici-eoi_7.batches_myo.csv
    Untracked:  data/covar_corrected_ici-sick.vs.ici-eoi_7.batches_myo_mis.csv
    Untracked:  data/covar_corrected_ici.vs.pbs_7.batches_myo.csv
    Untracked:  data/covar_corrected_ici.vs.pbs_7.batches_myo_mis.csv
    Untracked:  data/covar_corrected_myo-yes.vs.myo-no_7.batches_myo.csv
    Untracked:  data/covar_corrected_myo-yes.vs.myo-no_7.batches_myo_mis.csv
    Untracked:  data/covar_corrected_pbs-myo-yes.vs.pbs-myo-no_7.batches_myo.csv
    Untracked:  data/covar_corrected_pbs-myo-yes.vs.pbs-myo-no_7.batches_myo_mis.csv
    Untracked:  data/e_4.batches_myo.RData
    Untracked:  data/e_7.batches_myo.RData
    Untracked:  data/e_snpg_samqc_4.batches_myo.RData
    Untracked:  data/e_snpg_samqc_7.batches_myo.RData
    Untracked:  data/errors_ind_4.batches_myo.RData
    Untracked:  data/errors_ind_7.batches_myo.RData
    Untracked:  data/genetic_map_4.batches_myo.csv
    Untracked:  data/genetic_map_7.batches_myo.csv
    Untracked:  data/genotype_errors_marker_4.batches_myo.RData
    Untracked:  data/genotype_errors_marker_7.batches_myo.RData
    Untracked:  data/genotype_freq_marker_4.batches_myo.RData
    Untracked:  data/genotype_freq_marker_7.batches_myo.RData
    Untracked:  data/gm_allqc_4.batches_myo.RData
    Untracked:  data/gm_allqc_4.batches_myo_mis.RData
    Untracked:  data/gm_allqc_7.batches_myo.RData
    Untracked:  data/gm_allqc_7.batches_myo_mis.RData
    Untracked:  data/gm_samqc_4.batches_myo.RData
    Untracked:  data/gm_samqc_7.batches_myo.RData
    Untracked:  data/gm_serreze.BC312.RData
    Untracked:  data/gm_serreze.BC366.RData
    Untracked:  data/gm_serreze_7.batches_myo.RData
    Untracked:  data/gm_serreze_bc_7.batches_myo.RData
    Untracked:  data/het-ici-myo-yes.vs.het-ici-myo-no_gm_qtl_snpsqc_dis_no-x_updated_7.batches_myo_mis.csv
    Untracked:  data/het-ici-myo-yes.vs.het-ici-myo-no_marker.freq_low.geno.freq.removed_geno.ratio_7.batches_myo.csv
    Untracked:  data/het-ici-myo-yes.vs.het-ici-myo-no_marker.freq_low.geno.freq.removed_geno.ratio_7.batches_myo_mis.csv
    Untracked:  data/het-ici-myo-yes.vs.het-ici-myo-no_marker.freq_low.geno.freq.removed_sample.outliers.removed_geno.ratio_7.batches_myo.csv
    Untracked:  data/het-ici-myo-yes.vs.het-ici-myo-no_marker.freq_low.geno.freq.removed_sample.outliers.removed_geno.ratio_7.batches_myo_mis.csv
    Untracked:  data/het-ici-myo-yes.vs.het-ici-myo-no_marker.freq_low.probs.freq.removed_geno.ratio_7.batches_myo.csv
    Untracked:  data/het-ici-myo-yes.vs.het-ici-myo-no_marker.freq_low.probs.freq.removed_geno.ratio_7.batches_myo_mis.csv
    Untracked:  data/het-ici-myo-yes.vs.het-ici-myo-no_marker.freq_low.probs.freq.removed_sample.outliers.removed_geno.ratio_7.batches_myo.csv
    Untracked:  data/het-ici-myo-yes.vs.het-ici-myo-no_marker.freq_low.probs.freq.removed_sample.outliers.removed_geno.ratio_7.batches_myo_mis.csv
    Untracked:  data/het-ici-myo-yes.vs.het-ici-myo-no_sample.genos_marker.freq_low.geno.freq.removed_7.batches_myo.csv
    Untracked:  data/het-ici-myo-yes.vs.het-ici-myo-no_sample.genos_marker.freq_low.geno.freq.removed_7.batches_myo_mis.csv
    Untracked:  data/het-ici-myo-yes.vs.het-ici-myo-no_sample.genos_marker.freq_low.geno.freq.removed_sample.outliers.removed_7.batches_myo.csv
    Untracked:  data/het-ici-myo-yes.vs.het-ici-myo-no_sample.genos_marker.freq_low.geno.freq.removed_sample.outliers.removed_7.batches_myo_mis.csv
    Untracked:  data/het-ici-myo-yes.vs.het-ici-myo-no_sample.genos_marker.freq_low.probs.freq.removed_7.batches_myo.csv
    Untracked:  data/het-ici-myo-yes.vs.het-ici-myo-no_sample.genos_marker.freq_low.probs.freq.removed_7.batches_myo_mis.csv
    Untracked:  data/het-ici-myo-yes.vs.het-ici-myo-no_sample.genos_marker.freq_low.probs.freq.removed_sample.outliers.removed_7.batches_myo.csv
    Untracked:  data/het-ici-myo-yes.vs.het-ici-myo-no_sample.genos_marker.freq_low.probs.freq.removed_sample.outliers.removed_7.batches_myo_mis.csv
    Untracked:  data/het-ici.vs.het-pbs_gm_qtl_snpsqc_dis_no-x_updated_7.batches_myo.csv
    Untracked:  data/het-ici.vs.het-pbs_gm_qtl_snpsqc_dis_no-x_updated_7.batches_myo_mis.csv
    Untracked:  data/het-ici.vs.het-pbs_marker.freq_low.geno.freq.removed_geno.ratio_7.batches_myo.csv
    Untracked:  data/het-ici.vs.het-pbs_marker.freq_low.geno.freq.removed_geno.ratio_7.batches_myo_mis.csv
    Untracked:  data/het-ici.vs.het-pbs_marker.freq_low.geno.freq.removed_sample.outliers.removed_geno.ratio_7.batches_myo.csv
    Untracked:  data/het-ici.vs.het-pbs_marker.freq_low.geno.freq.removed_sample.outliers.removed_geno.ratio_7.batches_myo_mis.csv
    Untracked:  data/het-ici.vs.het-pbs_marker.freq_low.probs.freq.removed_geno.ratio_7.batches_myo.csv
    Untracked:  data/het-ici.vs.het-pbs_marker.freq_low.probs.freq.removed_geno.ratio_7.batches_myo_mis.csv
    Untracked:  data/het-ici.vs.het-pbs_marker.freq_low.probs.freq.removed_sample.outliers.removed_geno.ratio_7.batches_myo.csv
    Untracked:  data/het-ici.vs.het-pbs_marker.freq_low.probs.freq.removed_sample.outliers.removed_geno.ratio_7.batches_myo_mis.csv
    Untracked:  data/het-ici.vs.het-pbs_sample.genos_marker.freq_low.geno.freq.removed_7.batches_myo.csv
    Untracked:  data/het-ici.vs.het-pbs_sample.genos_marker.freq_low.geno.freq.removed_7.batches_myo_mis.csv
    Untracked:  data/het-ici.vs.het-pbs_sample.genos_marker.freq_low.geno.freq.removed_sample.outliers.removed_7.batches_myo.csv
    Untracked:  data/het-ici.vs.het-pbs_sample.genos_marker.freq_low.geno.freq.removed_sample.outliers.removed_7.batches_myo_mis.csv
    Untracked:  data/het-ici.vs.het-pbs_sample.genos_marker.freq_low.probs.freq.removed_7.batches_myo.csv
    Untracked:  data/het-ici.vs.het-pbs_sample.genos_marker.freq_low.probs.freq.removed_7.batches_myo_mis.csv
    Untracked:  data/het-ici.vs.het-pbs_sample.genos_marker.freq_low.probs.freq.removed_sample.outliers.removed_7.batches_myo.csv
    Untracked:  data/het-ici.vs.het-pbs_sample.genos_marker.freq_low.probs.freq.removed_sample.outliers.removed_7.batches_myo_mis.csv
    Untracked:  data/ici-myo-yes.vs.ici-myo-no_gm_qtl_snpsqc_dis_no-x_updated_7.batches_myo.csv
    Untracked:  data/ici-myo-yes.vs.ici-myo-no_gm_qtl_snpsqc_dis_no-x_updated_7.batches_myo_mis.csv
    Untracked:  data/ici-myo-yes.vs.ici-myo-no_marker.freq_low.geno.freq.removed_geno.ratio_7.batches_myo.csv
    Untracked:  data/ici-myo-yes.vs.ici-myo-no_marker.freq_low.geno.freq.removed_geno.ratio_7.batches_myo_mis.csv
    Untracked:  data/ici-myo-yes.vs.ici-myo-no_marker.freq_low.geno.freq.removed_sample.outliers.removed_geno.ratio_7.batches_myo.csv
    Untracked:  data/ici-myo-yes.vs.ici-myo-no_marker.freq_low.geno.freq.removed_sample.outliers.removed_geno.ratio_7.batches_myo_mis.csv
    Untracked:  data/ici-myo-yes.vs.ici-myo-no_marker.freq_low.probs.freq.removed_geno.ratio_7.batches_myo.csv
    Untracked:  data/ici-myo-yes.vs.ici-myo-no_marker.freq_low.probs.freq.removed_geno.ratio_7.batches_myo_mis.csv
    Untracked:  data/ici-myo-yes.vs.ici-myo-no_marker.freq_low.probs.freq.removed_sample.outliers.removed_geno.ratio_7.batches_myo.csv
    Untracked:  data/ici-myo-yes.vs.ici-myo-no_marker.freq_low.probs.freq.removed_sample.outliers.removed_geno.ratio_7.batches_myo_mis.csv
    Untracked:  data/ici-myo-yes.vs.ici-myo-no_sample.genos_marker.freq_low.geno.freq.removed_7.batches_myo.csv
    Untracked:  data/ici-myo-yes.vs.ici-myo-no_sample.genos_marker.freq_low.geno.freq.removed_7.batches_myo_mis.csv
    Untracked:  data/ici-myo-yes.vs.ici-myo-no_sample.genos_marker.freq_low.geno.freq.removed_sample.outliers.removed_7.batches_myo.csv
    Untracked:  data/ici-myo-yes.vs.ici-myo-no_sample.genos_marker.freq_low.geno.freq.removed_sample.outliers.removed_7.batches_myo_mis.csv
    Untracked:  data/ici-myo-yes.vs.ici-myo-no_sample.genos_marker.freq_low.probs.freq.removed_7.batches_myo.csv
    Untracked:  data/ici-myo-yes.vs.ici-myo-no_sample.genos_marker.freq_low.probs.freq.removed_7.batches_myo_mis.csv
    Untracked:  data/ici-myo-yes.vs.ici-myo-no_sample.genos_marker.freq_low.probs.freq.removed_sample.outliers.removed_7.batches_myo.csv
    Untracked:  data/ici-myo-yes.vs.ici-myo-no_sample.genos_marker.freq_low.probs.freq.removed_sample.outliers.removed_7.batches_myo_mis.csv
    Untracked:  data/ici-sick.vs.ici-eoi_blup_sub_chr-10_peak.marker-UNC18343990_lod.drop-1.5_snpsqc_dis_no-x_updated_7.batches_myo.csv
    Untracked:  data/ici-sick.vs.ici-eoi_blup_sub_chr-10_peak.marker-UNC18343990_lod.drop-1.5_snpsqc_dis_no-x_updated_7.batches_myo_mis.csv
    Untracked:  data/ici-sick.vs.ici-eoi_blup_sub_chr-11_peak.marker-UNC20070077_lod.drop-1.5_snpsqc_dis_no-x_updated_7.batches_myo.csv
    Untracked:  data/ici-sick.vs.ici-eoi_blup_sub_chr-11_peak.marker-UNC20070077_lod.drop-1.5_snpsqc_dis_no-x_updated_7.batches_myo_mis.csv
    Untracked:  data/ici-sick.vs.ici-eoi_blup_sub_chr-12_peak.marker-UNC21652584_lod.drop-1.5_snpsqc_dis_no-x_updated_7.batches_myo.csv
    Untracked:  data/ici-sick.vs.ici-eoi_blup_sub_chr-12_peak.marker-UNC21652584_lod.drop-1.5_snpsqc_dis_no-x_updated_7.batches_myo_mis.csv
    Untracked:  data/ici-sick.vs.ici-eoi_blup_sub_chr-13_peak.marker-UNCHS036579_lod.drop-1.5_snpsqc_dis_no-x_updated_7.batches_myo.csv
    Untracked:  data/ici-sick.vs.ici-eoi_blup_sub_chr-13_peak.marker-UNCHS036579_lod.drop-1.5_snpsqc_dis_no-x_updated_7.batches_myo_mis.csv
    Untracked:  data/ici-sick.vs.ici-eoi_blup_sub_chr-14_peak.marker-UNCHS037782_lod.drop-1.5_snpsqc_dis_no-x_updated_7.batches_myo.csv
    Untracked:  data/ici-sick.vs.ici-eoi_blup_sub_chr-14_peak.marker-UNCHS037782_lod.drop-1.5_snpsqc_dis_no-x_updated_7.batches_myo_mis.csv
    Untracked:  data/ici-sick.vs.ici-eoi_blup_sub_chr-15_peak.marker-UNC26069905_lod.drop-1.5_snpsqc_dis_no-x_updated_7.batches_myo.csv
    Untracked:  data/ici-sick.vs.ici-eoi_blup_sub_chr-15_peak.marker-UNCHS041223_lod.drop-1.5_snpsqc_dis_no-x_updated_7.batches_myo_mis.csv
    Untracked:  data/ici-sick.vs.ici-eoi_blup_sub_chr-16_peak.marker-JAX00070117_lod.drop-1.5_snpsqc_dis_no-x_updated_7.batches_myo.csv
    Untracked:  data/ici-sick.vs.ici-eoi_blup_sub_chr-16_peak.marker-JAX00070117_lod.drop-1.5_snpsqc_dis_no-x_updated_7.batches_myo_mis.csv
    Untracked:  data/ici-sick.vs.ici-eoi_blup_sub_chr-17_peak.marker-UNC28542319_lod.drop-1.5_snpsqc_dis_no-x_updated_7.batches_myo_mis.csv
    Untracked:  data/ici-sick.vs.ici-eoi_blup_sub_chr-17_peak.marker-UNCJPD006670_lod.drop-1.5_snpsqc_dis_no-x_updated_7.batches_myo.csv
    Untracked:  data/ici-sick.vs.ici-eoi_blup_sub_chr-18_peak.marker-UNC28776739_lod.drop-1.5_snpsqc_dis_no-x_updated_7.batches_myo.csv
    Untracked:  data/ici-sick.vs.ici-eoi_blup_sub_chr-18_peak.marker-UNCHS045594_lod.drop-1.5_snpsqc_dis_no-x_updated_7.batches_myo_mis.csv
    Untracked:  data/ici-sick.vs.ici-eoi_blup_sub_chr-19_peak.marker-UNC30414168_lod.drop-1.5_snpsqc_dis_no-x_updated_7.batches_myo.csv
    Untracked:  data/ici-sick.vs.ici-eoi_blup_sub_chr-19_peak.marker-UNC30426276_lod.drop-1.5_snpsqc_dis_no-x_updated_7.batches_myo_mis.csv
    Untracked:  data/ici-sick.vs.ici-eoi_blup_sub_chr-1_peak.marker-UNC2031646_lod.drop-1.5_snpsqc_dis_no-x_updated_7.batches_myo.csv
    Untracked:  data/ici-sick.vs.ici-eoi_blup_sub_chr-1_peak.marker-UNCHS003700_lod.drop-1.5_snpsqc_dis_no-x_updated_7.batches_myo_mis.csv
    Untracked:  data/ici-sick.vs.ici-eoi_blup_sub_chr-2_peak.marker-ICR5131_lod.drop-1.5_snpsqc_dis_no-x_updated_7.batches_myo.csv
    Untracked:  data/ici-sick.vs.ici-eoi_blup_sub_chr-2_peak.marker-UNCHS006420_lod.drop-1.5_snpsqc_dis_no-x_updated_7.batches_myo_mis.csv
    Untracked:  data/ici-sick.vs.ici-eoi_blup_sub_chr-3_peak.marker-UNC5667757_lod.drop-1.5_snpsqc_dis_no-x_updated_7.batches_myo.csv
    Untracked:  data/ici-sick.vs.ici-eoi_blup_sub_chr-3_peak.marker-UNC5667757_lod.drop-1.5_snpsqc_dis_no-x_updated_7.batches_myo_mis.csv
    Untracked:  data/ici-sick.vs.ici-eoi_blup_sub_chr-4_peak.marker-UNC6759992_lod.drop-1.5_snpsqc_dis_no-x_updated_7.batches_myo_mis.csv
    Untracked:  data/ici-sick.vs.ici-eoi_blup_sub_chr-4_peak.marker-UNC6765178_lod.drop-1.5_snpsqc_dis_no-x_updated_7.batches_myo.csv
    Untracked:  data/ici-sick.vs.ici-eoi_blup_sub_chr-5_peak.marker-UNC9889957_lod.drop-1.5_snpsqc_dis_no-x_updated_7.batches_myo_mis.csv
    Untracked:  data/ici-sick.vs.ici-eoi_blup_sub_chr-5_peak.marker-UNC9900273_lod.drop-1.5_snpsqc_dis_no-x_updated_7.batches_myo.csv
    Untracked:  data/ici-sick.vs.ici-eoi_blup_sub_chr-6_peak.marker-UNC10800126_lod.drop-1.5_snpsqc_dis_no-x_updated_7.batches_myo_mis.csv
    Untracked:  data/ici-sick.vs.ici-eoi_blup_sub_chr-6_peak.marker-UNC10832076_lod.drop-1.5_snpsqc_dis_no-x_updated_7.batches_myo.csv
    Untracked:  data/ici-sick.vs.ici-eoi_blup_sub_chr-7_peak.marker-UNCHS020903_lod.drop-1.5_snpsqc_dis_no-x_updated_7.batches_myo_mis.csv
    Untracked:  data/ici-sick.vs.ici-eoi_blup_sub_chr-7_peak.marker-UNCHS021163_lod.drop-1.5_snpsqc_dis_no-x_updated_7.batches_myo.csv
    Untracked:  data/ici-sick.vs.ici-eoi_blup_sub_chr-8_peak.marker-UNC15471847_lod.drop-1.5_snpsqc_dis_no-x_updated_7.batches_myo.csv
    Untracked:  data/ici-sick.vs.ici-eoi_blup_sub_chr-8_peak.marker-UNC15548888_lod.drop-1.5_snpsqc_dis_no-x_updated_7.batches_myo_mis.csv
    Untracked:  data/ici-sick.vs.ici-eoi_blup_sub_chr-9_peak.marker-UNC16231874_lod.drop-1.5_snpsqc_dis_no-x_updated_7.batches_myo.csv
    Untracked:  data/ici-sick.vs.ici-eoi_blup_sub_chr-9_peak.marker-UNC16231874_lod.drop-1.5_snpsqc_dis_no-x_updated_7.batches_myo_mis.csv
    Untracked:  data/ici-sick.vs.ici-eoi_blup_sub_chr-X_peak.marker-UNCHS049472_lod.drop-1.5_snpsqc_dis_no-x_updated_7.batches_myo.csv
    Untracked:  data/ici-sick.vs.ici-eoi_blup_sub_chr-X_peak.marker-UNCHS049472_lod.drop-1.5_snpsqc_dis_no-x_updated_7.batches_myo_mis.csv
    Untracked:  data/ici-sick.vs.ici-eoi_genes_chr-10_peak.marker-UNC18343990_lod.drop-1.5_snpsqc_dis_no-x_updated_7.batches_myo.csv
    Untracked:  data/ici-sick.vs.ici-eoi_genes_chr-10_peak.marker-UNC18343990_lod.drop-1.5_snpsqc_dis_no-x_updated_7.batches_myo_mis.csv
    Untracked:  data/ici-sick.vs.ici-eoi_genes_chr-11_peak.marker-UNC20070077_lod.drop-1.5_snpsqc_dis_no-x_updated_7.batches_myo.csv
    Untracked:  data/ici-sick.vs.ici-eoi_genes_chr-11_peak.marker-UNC20070077_lod.drop-1.5_snpsqc_dis_no-x_updated_7.batches_myo_mis.csv
    Untracked:  data/ici-sick.vs.ici-eoi_genes_chr-12_peak.marker-UNC21652584_lod.drop-1.5_snpsqc_dis_no-x_updated_7.batches_myo.csv
    Untracked:  data/ici-sick.vs.ici-eoi_genes_chr-12_peak.marker-UNC21652584_lod.drop-1.5_snpsqc_dis_no-x_updated_7.batches_myo_mis.csv
    Untracked:  data/ici-sick.vs.ici-eoi_genes_chr-13_peak.marker-UNCHS036579_lod.drop-1.5_snpsqc_dis_no-x_updated_7.batches_myo.csv
    Untracked:  data/ici-sick.vs.ici-eoi_genes_chr-13_peak.marker-UNCHS036579_lod.drop-1.5_snpsqc_dis_no-x_updated_7.batches_myo_mis.csv
    Untracked:  data/ici-sick.vs.ici-eoi_genes_chr-14_peak.marker-UNCHS037782_lod.drop-1.5_snpsqc_dis_no-x_updated_7.batches_myo.csv
    Untracked:  data/ici-sick.vs.ici-eoi_genes_chr-14_peak.marker-UNCHS037782_lod.drop-1.5_snpsqc_dis_no-x_updated_7.batches_myo_mis.csv
    Untracked:  data/ici-sick.vs.ici-eoi_genes_chr-15_peak.marker-UNC26069905_lod.drop-1.5_snpsqc_dis_no-x_updated_7.batches_myo.csv
    Untracked:  data/ici-sick.vs.ici-eoi_genes_chr-15_peak.marker-UNCHS041223_lod.drop-1.5_snpsqc_dis_no-x_updated_7.batches_myo_mis.csv
    Untracked:  data/ici-sick.vs.ici-eoi_genes_chr-16_peak.marker-JAX00070117_lod.drop-1.5_snpsqc_dis_no-x_updated_7.batches_myo.csv
    Untracked:  data/ici-sick.vs.ici-eoi_genes_chr-16_peak.marker-JAX00070117_lod.drop-1.5_snpsqc_dis_no-x_updated_7.batches_myo_mis.csv
    Untracked:  data/ici-sick.vs.ici-eoi_genes_chr-17_peak.marker-UNC28542319_lod.drop-1.5_snpsqc_dis_no-x_updated_7.batches_myo_mis.csv
    Untracked:  data/ici-sick.vs.ici-eoi_genes_chr-17_peak.marker-UNCJPD006670_lod.drop-1.5_snpsqc_dis_no-x_updated_7.batches_myo.csv
    Untracked:  data/ici-sick.vs.ici-eoi_genes_chr-18_peak.marker-UNC28776739_lod.drop-1.5_snpsqc_dis_no-x_updated_7.batches_myo.csv
    Untracked:  data/ici-sick.vs.ici-eoi_genes_chr-18_peak.marker-UNCHS045594_lod.drop-1.5_snpsqc_dis_no-x_updated_7.batches_myo_mis.csv
    Untracked:  data/ici-sick.vs.ici-eoi_genes_chr-19_peak.marker-UNC30414168_lod.drop-1.5_snpsqc_dis_no-x_updated_7.batches_myo.csv
    Untracked:  data/ici-sick.vs.ici-eoi_genes_chr-19_peak.marker-UNC30426276_lod.drop-1.5_snpsqc_dis_no-x_updated_7.batches_myo_mis.csv
    Untracked:  data/ici-sick.vs.ici-eoi_genes_chr-1_peak.marker-UNC2031646_lod.drop-1.5_snpsqc_dis_no-x_updated_7.batches_myo.csv
    Untracked:  data/ici-sick.vs.ici-eoi_genes_chr-1_peak.marker-UNCHS003700_lod.drop-1.5_snpsqc_dis_no-x_updated_7.batches_myo_mis.csv
    Untracked:  data/ici-sick.vs.ici-eoi_genes_chr-2_peak.marker-ICR5131_lod.drop-1.5_snpsqc_dis_no-x_updated_7.batches_myo.csv
    Untracked:  data/ici-sick.vs.ici-eoi_genes_chr-2_peak.marker-UNCHS006420_lod.drop-1.5_snpsqc_dis_no-x_updated_7.batches_myo_mis.csv
    Untracked:  data/ici-sick.vs.ici-eoi_genes_chr-3_peak.marker-UNC5667757_lod.drop-1.5_snpsqc_dis_no-x_updated_7.batches_myo.csv
    Untracked:  data/ici-sick.vs.ici-eoi_genes_chr-3_peak.marker-UNC5667757_lod.drop-1.5_snpsqc_dis_no-x_updated_7.batches_myo_mis.csv
    Untracked:  data/ici-sick.vs.ici-eoi_genes_chr-4_peak.marker-UNC6759992_lod.drop-1.5_snpsqc_dis_no-x_updated_7.batches_myo_mis.csv
    Untracked:  data/ici-sick.vs.ici-eoi_genes_chr-4_peak.marker-UNC6765178_lod.drop-1.5_snpsqc_dis_no-x_updated_7.batches_myo.csv
    Untracked:  data/ici-sick.vs.ici-eoi_genes_chr-5_peak.marker-UNC9889957_lod.drop-1.5_snpsqc_dis_no-x_updated_7.batches_myo_mis.csv
    Untracked:  data/ici-sick.vs.ici-eoi_genes_chr-5_peak.marker-UNC9900273_lod.drop-1.5_snpsqc_dis_no-x_updated_7.batches_myo.csv
    Untracked:  data/ici-sick.vs.ici-eoi_genes_chr-6_peak.marker-UNC10800126_lod.drop-1.5_snpsqc_dis_no-x_updated_7.batches_myo_mis.csv
    Untracked:  data/ici-sick.vs.ici-eoi_genes_chr-6_peak.marker-UNC10832076_lod.drop-1.5_snpsqc_dis_no-x_updated_7.batches_myo.csv
    Untracked:  data/ici-sick.vs.ici-eoi_genes_chr-7_peak.marker-UNCHS020903_lod.drop-1.5_snpsqc_dis_no-x_updated_7.batches_myo_mis.csv
    Untracked:  data/ici-sick.vs.ici-eoi_genes_chr-7_peak.marker-UNCHS021163_lod.drop-1.5_snpsqc_dis_no-x_updated_7.batches_myo.csv
    Untracked:  data/ici-sick.vs.ici-eoi_genes_chr-8_peak.marker-UNC15471847_lod.drop-1.5_snpsqc_dis_no-x_updated_7.batches_myo.csv
    Untracked:  data/ici-sick.vs.ici-eoi_genes_chr-8_peak.marker-UNC15548888_lod.drop-1.5_snpsqc_dis_no-x_updated_7.batches_myo_mis.csv
    Untracked:  data/ici-sick.vs.ici-eoi_genes_chr-9_peak.marker-UNC16231874_lod.drop-1.5_snpsqc_dis_no-x_updated_7.batches_myo.csv
    Untracked:  data/ici-sick.vs.ici-eoi_genes_chr-9_peak.marker-UNC16231874_lod.drop-1.5_snpsqc_dis_no-x_updated_7.batches_myo_mis.csv
    Untracked:  data/ici-sick.vs.ici-eoi_genes_chr-X_peak.marker-UNCHS049472_lod.drop-1.5_snpsqc_dis_no-x_updated_7.batches_myo.csv
    Untracked:  data/ici-sick.vs.ici-eoi_genes_chr-X_peak.marker-UNCHS049472_lod.drop-1.5_snpsqc_dis_no-x_updated_7.batches_myo_mis.csv
    Untracked:  data/ici-sick.vs.ici-eoi_gm_qtl_snpsqc_dis_no-x_updated_7.batches_myo.csv
    Untracked:  data/ici-sick.vs.ici-eoi_gm_qtl_snpsqc_dis_no-x_updated_7.batches_myo_mis.csv
    Untracked:  data/ici-sick.vs.ici-eoi_marker.freq_low.geno.freq.removed_geno.ratio_7.batches_myo.csv
    Untracked:  data/ici-sick.vs.ici-eoi_marker.freq_low.geno.freq.removed_geno.ratio_7.batches_myo_mis.csv
    Untracked:  data/ici-sick.vs.ici-eoi_marker.freq_low.geno.freq.removed_sample.outliers.removed_geno.ratio_7.batches_myo.csv
    Untracked:  data/ici-sick.vs.ici-eoi_marker.freq_low.geno.freq.removed_sample.outliers.removed_geno.ratio_7.batches_myo_mis.csv
    Untracked:  data/ici-sick.vs.ici-eoi_marker.freq_low.probs.freq.removed_geno.ratio_7.batches_myo.csv
    Untracked:  data/ici-sick.vs.ici-eoi_marker.freq_low.probs.freq.removed_geno.ratio_7.batches_myo_mis.csv
    Untracked:  data/ici-sick.vs.ici-eoi_marker.freq_low.probs.freq.removed_sample.outliers.removed_geno.ratio_7.batches_myo.csv
    Untracked:  data/ici-sick.vs.ici-eoi_marker.freq_low.probs.freq.removed_sample.outliers.removed_geno.ratio_7.batches_myo_mis.csv
    Untracked:  data/ici-sick.vs.ici-eoi_sample.genos_marker.freq_low.geno.freq.removed_7.batches_myo.csv
    Untracked:  data/ici-sick.vs.ici-eoi_sample.genos_marker.freq_low.geno.freq.removed_7.batches_myo_mis.csv
    Untracked:  data/ici-sick.vs.ici-eoi_sample.genos_marker.freq_low.geno.freq.removed_sample.outliers.removed_7.batches_myo.csv
    Untracked:  data/ici-sick.vs.ici-eoi_sample.genos_marker.freq_low.geno.freq.removed_sample.outliers.removed_7.batches_myo_mis.csv
    Untracked:  data/ici-sick.vs.ici-eoi_sample.genos_marker.freq_low.probs.freq.removed_7.batches_myo.csv
    Untracked:  data/ici-sick.vs.ici-eoi_sample.genos_marker.freq_low.probs.freq.removed_7.batches_myo_mis.csv
    Untracked:  data/ici-sick.vs.ici-eoi_sample.genos_marker.freq_low.probs.freq.removed_sample.outliers.removed_7.batches_myo.csv
    Untracked:  data/ici-sick.vs.ici-eoi_sample.genos_marker.freq_low.probs.freq.removed_sample.outliers.removed_7.batches_myo_mis.csv
    Untracked:  data/ici.vs.pbs_marker.freq_low.geno.freq.removed_geno.ratio_7.batches_myo.csv
    Untracked:  data/ici.vs.pbs_marker.freq_low.geno.freq.removed_geno.ratio_7.batches_myo_mis.csv
    Untracked:  data/ici.vs.pbs_marker.freq_low.geno.freq.removed_sample.outliers.removed_geno.ratio_7.batches_myo.csv
    Untracked:  data/ici.vs.pbs_marker.freq_low.geno.freq.removed_sample.outliers.removed_geno.ratio_7.batches_myo_mis.csv
    Untracked:  data/ici.vs.pbs_marker.freq_low.probs.freq.removed_geno.ratio_7.batches_myo.csv
    Untracked:  data/ici.vs.pbs_marker.freq_low.probs.freq.removed_geno.ratio_7.batches_myo_mis.csv
    Untracked:  data/ici.vs.pbs_marker.freq_low.probs.freq.removed_sample.outliers.removed_geno.ratio_7.batches_myo.csv
    Untracked:  data/ici.vs.pbs_marker.freq_low.probs.freq.removed_sample.outliers.removed_geno.ratio_7.batches_myo_mis.csv
    Untracked:  data/ici.vs.pbs_sample.genos_marker.freq_low.geno.freq.removed_7.batches_myo.csv
    Untracked:  data/ici.vs.pbs_sample.genos_marker.freq_low.geno.freq.removed_7.batches_myo_mis.csv
    Untracked:  data/ici.vs.pbs_sample.genos_marker.freq_low.geno.freq.removed_sample.outliers.removed_7.batches_myo.csv
    Untracked:  data/ici.vs.pbs_sample.genos_marker.freq_low.geno.freq.removed_sample.outliers.removed_7.batches_myo_mis.csv
    Untracked:  data/ici.vs.pbs_sample.genos_marker.freq_low.probs.freq.removed_7.batches_myo.csv
    Untracked:  data/ici.vs.pbs_sample.genos_marker.freq_low.probs.freq.removed_7.batches_myo_mis.csv
    Untracked:  data/ici.vs.pbs_sample.genos_marker.freq_low.probs.freq.removed_sample.outliers.removed_7.batches_myo.csv
    Untracked:  data/ici.vs.pbs_sample.genos_marker.freq_low.probs.freq.removed_sample.outliers.removed_7.batches_myo_mis.csv
    Untracked:  data/mean.differences_age.of.onset_het-ici-myo-yes.vs.het-ici-myo-no_sample.genos_marker.freq_low.geno.freq.removed_7.batches_myo.csv
    Untracked:  data/mean.differences_age.of.onset_het-ici-myo-yes.vs.het-ici-myo-no_sample.genos_marker.freq_low.geno.freq.removed_7.batches_myo_mis.csv
    Untracked:  data/mean.differences_age.of.onset_het-ici-myo-yes.vs.het-ici-myo-no_sample.genos_marker.freq_low.geno.freq.removed_sample.outliers.removed_7.batches_myo.csv
    Untracked:  data/mean.differences_age.of.onset_het-ici-myo-yes.vs.het-ici-myo-no_sample.genos_marker.freq_low.geno.freq.removed_sample.outliers.removed_7.batches_myo_mis.csv
    Untracked:  data/mean.differences_age.of.onset_het-ici.vs.het-pbs_sample.genos_marker.freq_low.geno.freq.removed_7.batches_myo.csv
    Untracked:  data/mean.differences_age.of.onset_het-ici.vs.het-pbs_sample.genos_marker.freq_low.geno.freq.removed_7.batches_myo_mis.csv
    Untracked:  data/mean.differences_age.of.onset_het-ici.vs.het-pbs_sample.genos_marker.freq_low.geno.freq.removed_sample.outliers.removed_7.batches_myo.csv
    Untracked:  data/mean.differences_age.of.onset_het-ici.vs.het-pbs_sample.genos_marker.freq_low.geno.freq.removed_sample.outliers.removed_7.batches_myo_mis.csv
    Untracked:  data/mean.differences_age.of.onset_ici-myo-yes.vs.ici-myo-no_sample.genos_marker.freq_low.geno.freq.removed_7.batches_myo.csv
    Untracked:  data/mean.differences_age.of.onset_ici-myo-yes.vs.ici-myo-no_sample.genos_marker.freq_low.geno.freq.removed_7.batches_myo_mis.csv
    Untracked:  data/mean.differences_age.of.onset_ici-myo-yes.vs.ici-myo-no_sample.genos_marker.freq_low.geno.freq.removed_sample.outliers.removed_7.batches_myo.csv
    Untracked:  data/mean.differences_age.of.onset_ici-myo-yes.vs.ici-myo-no_sample.genos_marker.freq_low.geno.freq.removed_sample.outliers.removed_7.batches_myo_mis.csv
    Untracked:  data/mean.differences_age.of.onset_ici-sick.vs.ici-eoi_sample.genos_marker.freq_low.geno.freq.removed_7.batches_myo.csv
    Untracked:  data/mean.differences_age.of.onset_ici-sick.vs.ici-eoi_sample.genos_marker.freq_low.geno.freq.removed_7.batches_myo_mis.csv
    Untracked:  data/mean.differences_age.of.onset_ici-sick.vs.ici-eoi_sample.genos_marker.freq_low.geno.freq.removed_sample.outliers.removed_7.batches_myo.csv
    Untracked:  data/mean.differences_age.of.onset_ici-sick.vs.ici-eoi_sample.genos_marker.freq_low.geno.freq.removed_sample.outliers.removed_7.batches_myo_mis.csv
    Untracked:  data/mean.differences_age.of.onset_ici.vs.pbs_sample.genos_marker.freq_low.geno.freq.removed_7.batches_myo.csv
    Untracked:  data/mean.differences_age.of.onset_ici.vs.pbs_sample.genos_marker.freq_low.geno.freq.removed_sample.outliers.removed_7.batches_myo.csv
    Untracked:  data/mean.differences_age.of.onset_myo-yes.vs.myo-no_sample.genos_marker.freq_low.geno.freq.removed_7.batches_myo.csv
    Untracked:  data/mean.differences_age.of.onset_myo-yes.vs.myo-no_sample.genos_marker.freq_low.geno.freq.removed_7.batches_myo_mis.csv
    Untracked:  data/mean.differences_age.of.onset_myo-yes.vs.myo-no_sample.genos_marker.freq_low.geno.freq.removed_sample.outliers.removed_7.batches_myo.csv
    Untracked:  data/mean.differences_age.of.onset_myo-yes.vs.myo-no_sample.genos_marker.freq_low.geno.freq.removed_sample.outliers.removed_7.batches_myo_mis.csv
    Untracked:  data/mean.differences_age.of.onset_pbs-myo-yes.vs.pbs-myo-no_sample.genos_marker.freq_low.geno.freq.removed_7.batches_myo.csv
    Untracked:  data/mean.differences_age.of.onset_pbs-myo-yes.vs.pbs-myo-no_sample.genos_marker.freq_low.geno.freq.removed_7.batches_myo_mis.csv
    Untracked:  data/mean.differences_age.of.onset_pbs-myo-yes.vs.pbs-myo-no_sample.genos_marker.freq_low.geno.freq.removed_sample.outliers.removed_7.batches_myo.csv
    Untracked:  data/mean.differences_age.of.onset_pbs-myo-yes.vs.pbs-myo-no_sample.genos_marker.freq_low.geno.freq.removed_sample.outliers.removed_7.batches_myo_mis.csv
    Untracked:  data/mean.differences_group_het-ici-myo-yes.vs.het-ici-myo-no_sample.genos_marker.freq_low.geno.freq.removed_7.batches_myo.csv
    Untracked:  data/mean.differences_group_het-ici-myo-yes.vs.het-ici-myo-no_sample.genos_marker.freq_low.geno.freq.removed_7.batches_myo_mis.csv
    Untracked:  data/mean.differences_group_het-ici-myo-yes.vs.het-ici-myo-no_sample.genos_marker.freq_low.geno.freq.removed_sample.outliers.removed_7.batches_myo.csv
    Untracked:  data/mean.differences_group_het-ici-myo-yes.vs.het-ici-myo-no_sample.genos_marker.freq_low.geno.freq.removed_sample.outliers.removed_7.batches_myo_mis.csv
    Untracked:  data/mean.differences_group_het-ici.vs.het-pbs_sample.genos_marker.freq_low.geno.freq.removed_7.batches_myo.csv
    Untracked:  data/mean.differences_group_het-ici.vs.het-pbs_sample.genos_marker.freq_low.geno.freq.removed_7.batches_myo_mis.csv
    Untracked:  data/mean.differences_group_het-ici.vs.het-pbs_sample.genos_marker.freq_low.geno.freq.removed_sample.outliers.removed_7.batches_myo.csv
    Untracked:  data/mean.differences_group_het-ici.vs.het-pbs_sample.genos_marker.freq_low.geno.freq.removed_sample.outliers.removed_7.batches_myo_mis.csv
    Untracked:  data/mean.differences_group_ici-myo-yes.vs.ici-myo-no_sample.genos_marker.freq_low.geno.freq.removed_7.batches_myo.csv
    Untracked:  data/mean.differences_group_ici-myo-yes.vs.ici-myo-no_sample.genos_marker.freq_low.geno.freq.removed_7.batches_myo_mis.csv
    Untracked:  data/mean.differences_group_ici-myo-yes.vs.ici-myo-no_sample.genos_marker.freq_low.geno.freq.removed_sample.outliers.removed_7.batches_myo.csv
    Untracked:  data/mean.differences_group_ici-myo-yes.vs.ici-myo-no_sample.genos_marker.freq_low.geno.freq.removed_sample.outliers.removed_7.batches_myo_mis.csv
    Untracked:  data/mean.differences_group_ici-sick.vs.ici-eoi_sample.genos_marker.freq_low.geno.freq.removed_7.batches_myo.csv
    Untracked:  data/mean.differences_group_ici-sick.vs.ici-eoi_sample.genos_marker.freq_low.geno.freq.removed_7.batches_myo_mis.csv
    Untracked:  data/mean.differences_group_ici-sick.vs.ici-eoi_sample.genos_marker.freq_low.geno.freq.removed_sample.outliers.removed_7.batches_myo.csv
    Untracked:  data/mean.differences_group_ici-sick.vs.ici-eoi_sample.genos_marker.freq_low.geno.freq.removed_sample.outliers.removed_7.batches_myo_mis.csv
    Untracked:  data/mean.differences_group_ici.vs.pbs_sample.genos_marker.freq_low.geno.freq.removed_7.batches_myo.csv
    Untracked:  data/mean.differences_group_ici.vs.pbs_sample.genos_marker.freq_low.geno.freq.removed_sample.outliers.removed_7.batches_myo.csv
    Untracked:  data/mean.differences_group_myo-yes.vs.myo-no_sample.genos_marker.freq_low.geno.freq.removed_7.batches_myo.csv
    Untracked:  data/mean.differences_group_myo-yes.vs.myo-no_sample.genos_marker.freq_low.geno.freq.removed_7.batches_myo_mis.csv
    Untracked:  data/mean.differences_group_myo-yes.vs.myo-no_sample.genos_marker.freq_low.geno.freq.removed_sample.outliers.removed_7.batches_myo.csv
    Untracked:  data/mean.differences_group_myo-yes.vs.myo-no_sample.genos_marker.freq_low.geno.freq.removed_sample.outliers.removed_7.batches_myo_mis.csv
    Untracked:  data/mean.differences_group_pbs-myo-yes.vs.pbs-myo-no_sample.genos_marker.freq_low.geno.freq.removed_7.batches_myo.csv
    Untracked:  data/mean.differences_group_pbs-myo-yes.vs.pbs-myo-no_sample.genos_marker.freq_low.geno.freq.removed_7.batches_myo_mis.csv
    Untracked:  data/mean.differences_group_pbs-myo-yes.vs.pbs-myo-no_sample.genos_marker.freq_low.geno.freq.removed_sample.outliers.removed_7.batches_myo.csv
    Untracked:  data/mean.differences_group_pbs-myo-yes.vs.pbs-myo-no_sample.genos_marker.freq_low.geno.freq.removed_sample.outliers.removed_7.batches_myo_mis.csv
    Untracked:  data/myo-yes.vs.myo-no_gm_qtl_snpsqc_dis_no-x_updated_7.batches_myo.csv
    Untracked:  data/myo-yes.vs.myo-no_gm_qtl_snpsqc_dis_no-x_updated_7.batches_myo_mis.csv
    Untracked:  data/myo-yes.vs.myo-no_marker.freq_low.geno.freq.removed_geno.ratio_7.batches_myo.csv
    Untracked:  data/myo-yes.vs.myo-no_marker.freq_low.geno.freq.removed_geno.ratio_7.batches_myo_mis.csv
    Untracked:  data/myo-yes.vs.myo-no_marker.freq_low.geno.freq.removed_sample.outliers.removed_geno.ratio_7.batches_myo.csv
    Untracked:  data/myo-yes.vs.myo-no_marker.freq_low.geno.freq.removed_sample.outliers.removed_geno.ratio_7.batches_myo_mis.csv
    Untracked:  data/myo-yes.vs.myo-no_marker.freq_low.probs.freq.removed_geno.ratio_7.batches_myo.csv
    Untracked:  data/myo-yes.vs.myo-no_marker.freq_low.probs.freq.removed_geno.ratio_7.batches_myo_mis.csv
    Untracked:  data/myo-yes.vs.myo-no_marker.freq_low.probs.freq.removed_sample.outliers.removed_geno.ratio_7.batches_myo.csv
    Untracked:  data/myo-yes.vs.myo-no_marker.freq_low.probs.freq.removed_sample.outliers.removed_geno.ratio_7.batches_myo_mis.csv
    Untracked:  data/myo-yes.vs.myo-no_sample.genos_marker.freq_low.geno.freq.removed_7.batches_myo.csv
    Untracked:  data/myo-yes.vs.myo-no_sample.genos_marker.freq_low.geno.freq.removed_7.batches_myo_mis.csv
    Untracked:  data/myo-yes.vs.myo-no_sample.genos_marker.freq_low.geno.freq.removed_sample.outliers.removed_7.batches_myo.csv
    Untracked:  data/myo-yes.vs.myo-no_sample.genos_marker.freq_low.geno.freq.removed_sample.outliers.removed_7.batches_myo_mis.csv
    Untracked:  data/myo-yes.vs.myo-no_sample.genos_marker.freq_low.probs.freq.removed_7.batches_myo.csv
    Untracked:  data/myo-yes.vs.myo-no_sample.genos_marker.freq_low.probs.freq.removed_7.batches_myo_mis.csv
    Untracked:  data/myo-yes.vs.myo-no_sample.genos_marker.freq_low.probs.freq.removed_sample.outliers.removed_7.batches_myo.csv
    Untracked:  data/myo-yes.vs.myo-no_sample.genos_marker.freq_low.probs.freq.removed_sample.outliers.removed_7.batches_myo_mis.csv
    Untracked:  data/pbs-myo-yes.vs.pbs-myo-no_gm_qtl_snpsqc_dis_no-x_updated_7.batches_myo.csv
    Untracked:  data/pbs-myo-yes.vs.pbs-myo-no_gm_qtl_snpsqc_dis_no-x_updated_7.batches_myo_mis.csv
    Untracked:  data/pbs-myo-yes.vs.pbs-myo-no_marker.freq_low.geno.freq.removed_geno.ratio_7.batches_myo.csv
    Untracked:  data/pbs-myo-yes.vs.pbs-myo-no_marker.freq_low.geno.freq.removed_geno.ratio_7.batches_myo_mis.csv
    Untracked:  data/pbs-myo-yes.vs.pbs-myo-no_marker.freq_low.geno.freq.removed_sample.outliers.removed_geno.ratio_7.batches_myo.csv
    Untracked:  data/pbs-myo-yes.vs.pbs-myo-no_marker.freq_low.geno.freq.removed_sample.outliers.removed_geno.ratio_7.batches_myo_mis.csv
    Untracked:  data/pbs-myo-yes.vs.pbs-myo-no_marker.freq_low.probs.freq.removed_geno.ratio_7.batches_myo.csv
    Untracked:  data/pbs-myo-yes.vs.pbs-myo-no_marker.freq_low.probs.freq.removed_geno.ratio_7.batches_myo_mis.csv
    Untracked:  data/pbs-myo-yes.vs.pbs-myo-no_marker.freq_low.probs.freq.removed_sample.outliers.removed_geno.ratio_7.batches_myo.csv
    Untracked:  data/pbs-myo-yes.vs.pbs-myo-no_marker.freq_low.probs.freq.removed_sample.outliers.removed_geno.ratio_7.batches_myo_mis.csv
    Untracked:  data/pbs-myo-yes.vs.pbs-myo-no_sample.genos_marker.freq_low.geno.freq.removed_7.batches_myo.csv
    Untracked:  data/pbs-myo-yes.vs.pbs-myo-no_sample.genos_marker.freq_low.geno.freq.removed_7.batches_myo_mis.csv
    Untracked:  data/pbs-myo-yes.vs.pbs-myo-no_sample.genos_marker.freq_low.geno.freq.removed_sample.outliers.removed_7.batches_myo.csv
    Untracked:  data/pbs-myo-yes.vs.pbs-myo-no_sample.genos_marker.freq_low.geno.freq.removed_sample.outliers.removed_7.batches_myo_mis.csv
    Untracked:  data/pbs-myo-yes.vs.pbs-myo-no_sample.genos_marker.freq_low.probs.freq.removed_7.batches_myo.csv
    Untracked:  data/pbs-myo-yes.vs.pbs-myo-no_sample.genos_marker.freq_low.probs.freq.removed_7.batches_myo_mis.csv
    Untracked:  data/pbs-myo-yes.vs.pbs-myo-no_sample.genos_marker.freq_low.probs.freq.removed_sample.outliers.removed_7.batches_myo.csv
    Untracked:  data/pbs-myo-yes.vs.pbs-myo-no_sample.genos_marker.freq_low.probs.freq.removed_sample.outliers.removed_7.batches_myo_mis.csv
    Untracked:  data/percent_missing_id_4.batches_myo.RData
    Untracked:  data/percent_missing_id_7.batches_myo.RData
    Untracked:  data/percent_missing_marker_4.batches_myo.RData
    Untracked:  data/percent_missing_marker_7.batches_myo.RData
    Untracked:  data/pheno_4.batches_myo.csv
    Untracked:  data/pheno_7.batches_myo.csv
    Untracked:  data/physical_map_4.batches_myo.csv
    Untracked:  data/physical_map_7.batches_myo.csv
    Untracked:  data/probs_36state_bc_7.batches_myo.rds
    Untracked:  data/probs_8state_bc_7.batches_myo.rds
    Untracked:  data/qc_info_bad_sample_4.batches_myo.RData
    Untracked:  data/qc_info_bad_sample_7.batches_myo.RData
    Untracked:  data/sample_geno_AHB_4.batches_myo.csv
    Untracked:  data/sample_geno_AHB_7.batches_myo.csv
    Untracked:  data/sample_geno_AHB_7.batches_myo_orig.id.csv
    Untracked:  data/sample_geno_bc_4.batches_myo.csv
    Untracked:  data/sample_geno_bc_7.batches_myo.csv
    Untracked:  data/sample_geno_bc_7.batches_myo_orig.id.csv
    Untracked:  data/serreze_probs_4.batches_myo.rds
    Untracked:  data/serreze_probs_7.batches_myo.rds
    Untracked:  data/serreze_probs_allqc_4.batches_myo.rds
    Untracked:  data/serreze_probs_allqc_4.batches_myo_mis.rds
    Untracked:  data/serreze_probs_allqc_7.batches_myo.rds
    Untracked:  data/serreze_probs_allqc_7.batches_myo_mis.rds
    Untracked:  data/summary.cg_4.batches_myo.RData
    Untracked:  data/summary.cg_7.batches_myo.RData
    Untracked:  output/Percent_missing_genotype_data_4.batches_myo.pdf
    Untracked:  output/Percent_missing_genotype_data_7.batches_myo.pdf
    Untracked:  output/Percent_missing_genotype_data_per_marker_4.batches_myo.pdf
    Untracked:  output/Percent_missing_genotype_data_per_marker_7.batches_myo.pdf
    Untracked:  output/Proportion_matching_genotypes_before_removal_of_bad_samples_4.batches_myo.pdf
    Untracked:  output/Proportion_matching_genotypes_before_removal_of_bad_samples_7.batches_myo.pdf
    Untracked:  output/genotype_error_marker_4.batches_myo.pdf
    Untracked:  output/genotype_error_marker_7.batches_myo.pdf
    Untracked:  output/genotype_frequency_marker_4.batches_myo.pdf
    Untracked:  output/genotype_frequency_marker_7.batches_myo.pdf

Unstaged changes:
    Modified:   analysis/index_7.batches_myo.Rmd

</code></pre>
<p>
Note that any generated files, e.g. HTML, png, CSS, etc., are not
included in this status report because it is ok for generated content to
have uncommitted changes.
</p>
</div>
</div>
</div>
</div>
<hr>
</div>
<div id="versions" class="tab-pane fade">

<p>
These are the previous versions of the repository in which changes were
made to the R Markdown (<code>analysis/index_7.batches_myo.Rmd</code>)
and HTML (<code>docs/index_7.batches_myo.html</code>) files. If you’ve
configured a remote Git repository (see <code>?wflow_git_remote</code>),
click on the hyperlinks in the table below to view the files as they
were in that past version.
</p>
<div class="table-responsive">
<table class="table table-condensed table-hover">
<thead>
<tr>
<th>
File
</th>
<th>
Version
</th>
<th>
Author
</th>
<th>
Date
</th>
<th>
Message
</th>
</tr>
</thead>
<tbody>
<tr>
<td>
html
</td>
<td>
<a href="https://rawcdn.githack.com/TheJacksonLaboratory/Serreze-T1D_Workflow/efe4c014c4b6064bb5970630f71126f9089507ff/docs/index_7.batches_myo.html" target="_blank">efe4c01</a>
</td>
<td>
Belinda Cornes
</td>
<td>
2023-04-17
</td>
<td>
myocarditis html files 7 batches
</td>
</tr>
<tr>
<td>
Rmd
</td>
<td>
<a href="https://github.com/TheJacksonLaboratory/Serreze-T1D_Workflow/blob/732a0a5325727023ddf805f231e45d302e5c49a4/analysis/index_7.batches_myo.Rmd" target="_blank">732a0a5</a>
</td>
<td>
Belinda Cornes
</td>
<td>
2023-04-17
</td>
<td>
myocarditis Rmd files 7 batches
</td>
</tr>
</tbody>
</table>
</div>
<hr>
</div>
</div>
</div>
<div id="neogen-files-at-a-glance-total-samples-366"
class="section level2">
<h2><strong>Neogen Files At A Glance</strong> [Total Samples: 366]</h2>
<ul>
<li>Samples: The_Jackson_Lab_Serreze_MURGIGV01_20220729
<ul>
<li><a href="0.1_samples_batch_20220729.html">Total Samples: 48</a></li>
</ul></li>
<li>Samples: The_Jackson_Lab_Serreze_MURGIGV01_20220826
<ul>
<li><a href="0.1_samples_batch_20220826.html">Total Samples: 12</a></li>
</ul></li>
<li>Samples: The_Jackson_Lab_Serreze_MURGIGV01_20221006
<ul>
<li><a href="0.1_samples_batch_20221006.html">Total Samples: 85</a></li>
</ul></li>
<li>Samples: The_Jackson_Lab_Serreze_MURGIGV01_20221116
<ul>
<li><a href="0.1_samples_batch_20221116.html">Total Samples: 72</a></li>
</ul></li>
<li>Samples: The_Jackson_Lab_Serreze_MURGIGV01_20230124
<ul>
<li><a href="0.1_samples_batch_20230124.html">Total Samples: 36</a></li>
</ul></li>
<li>Samples: The_Jackson_Lab_Serreze_MURGIGV01_20230225
<ul>
<li><a href="0.1_samples_batch_20230225.html">Total Samples: 95</a></li>
</ul></li>
<li>Samples: The_Jackson_Lab_Serreze_MURGIGV01_20230313
<ul>
<li><a href="0.1_samples_batch_20230313.html">Total Samples: 18</a>
[<strong>Note:</strong> Sample <code>D249-PBS-EOI</code>, also appears
in the <a href="0.1_samples_batch_20230225.html">previous</a> group
submitted in Feb 2023]</li>
</ul></li>
<li><a href="0.1.1_preparing.data_bqc_7.batches_myo.html">Preparing
files for QC/Haplotype Phasing</a></li>
</ul>
</div>
<div id="haplotype-phasing" class="section level2">
<h2><strong>Haplotype Phasing</strong></h2>
<ul>
<li><a
href="0.2_haplotype_comparison_bqc_7.batches_myo_minprob.html">minimum
probability set for estimation of genotypes</a></li>
</ul>
</div>
<div id="experiment-design" class="section level2">
<h2><strong>1. Experiment Design</strong></h2>
<table class="table table-striped" style="margin-left: auto; margin-right: auto;">
<thead>
<tr>
<th style="text-align:left;">
</th>
<th style="text-align:left;">
</th>
</tr>
</thead>
<tbody>
<tr>
<td style="text-align:left;width: 5cms; font-weight: bold;">
abbreviations for strains:
</td>
<td style="text-align:left;width: 1cms; ">
AB: het; AA: hom
</td>
</tr>
<tr>
<td style="text-align:left;width: 5cms; font-weight: bold;">
type of cross:
</td>
<td style="text-align:left;width: 1cms; ">
Backcross
</td>
</tr>
<tr>
<td style="text-align:left;width: 5cms; font-weight: bold;">
number of mice phenotyped:
</td>
<td style="text-align:left;width: 1cms; ">
366
</td>
</tr>
<tr>
<td style="text-align:left;width: 5cms; font-weight: bold;">
total number of mice phenotypes:
</td>
<td style="text-align:left;width: 1cms; ">
2 [1 binary/1 quantiative]
</td>
</tr>
<tr>
<td style="text-align:left;width: 5cms; font-weight: bold;">
phenotypes:
</td>
<td style="text-align:left;width: 1cms; ">
myocarditis status/group [binary] &amp; age of onset [continuous]
</td>
</tr>
<tr>
<td style="text-align:left;width: 5cms; font-weight: bold;">
number of mice:
</td>
<td style="text-align:left;width: 1cms; ">
366
</td>
</tr>
<tr>
<td style="text-align:left;width: 5cms; font-weight: bold;">
number of markers:
</td>
<td style="text-align:left;width: 1cms; ">
137302
</td>
</tr>
<tr>
<td style="text-align:left;width: 5cms; font-weight: bold;">
covariates:
</td>
<td style="text-align:left;width: 1cms; ">
sex [F: 344; M: 22] ; and others depending on samples used in analysis
</td>
</tr>
</tbody>
</table>
</div>
<div id="genotype-qc" class="section level2">
<h2><strong>2. Genotype QC </strong></h2>
<ul>
<li>Step 1: <a href="2.1_sample_bqc_7.batches_myo.html">Sample
QC</a></li>
<li>Step 2: <a href="2.2.1_snp_qc_7.batches_myo.html">SNP
QC</a>(removing monomorphic, genotype error &amp; missing markers)</li>
<li>Step 3: <a
href="2.4_preparing.data_aqc_7.batches_myo.html">Preparing Files for QTL
Analysis</a></li>
<li>Step 4: Identifying additional disproportionate markers within each
sample set
<ul>
<li><a
href="genotype.frequencies_ici-sick.vs.ici-eoi_freq_7.batches_myo.html"><code>ICI Sick vs ICI EOI</code></a></li>
<li><a
href="genotype.frequencies_ici.vs.pbs_freq_7.batches_myo.html"><code>ICI vs PBS</code></a></li>
<li><a
href="genotype.frequencies_ici-myo-yes.vs.ici-myo-no_freq_7.batches_myo.html"><code>ICI Myocarditis Yes vs ICI Myocarditis No</code></a></li>
<li><a
href="genotype.frequencies_pbs-myo-yes.vs.pbs-myo-no_freq_7.batches_myo.html"><code>PBS Myocarditis Yes vs PBS Myocarditis No</code></a></li>
<li><a
href="genotype.frequencies_myo-yes.vs.myo-no_freq_7.batches_myo.html"><code>Myocarditis Yes vs Myocarditis No</code></a></li>
<li><a
href="genotype.frequencies_het-ici.vs.het-pbs_freq_7.batches_myo.html"><code>Hetero: ICI vs PBS</code></a></li>
<li><a
href="genotype.frequencies_het-ici-myo-yes.vs.het-ici-myo-no_freq_7.batches_myo.html"><code>Hetero: ICI Myocarditis Yes vs ICI Myocarditis No</code></a></li>
</ul></li>
</ul>
</div>
<div id="phenotype-qc" class="section level2">
<h2><strong>3. Phenotype QC</strong></h2>
<div
id="continous-phenotypes-age-of-onset-not-necessary-for-current-analysis"
class="section level3">
<h3>continous phenotypes (age of onset) [not necessary for current
analysis]</h3>
<ul>
<li>Corrected Phenotypes
<ul>
<li><a href="3.1_phenotype.qc_corrected_7.batches_myo.html">all
comparisons</a></li>
</ul></li>
</ul>
</div>
</div>
<div id="qtl-analysis" class="section level2">
<h2><strong>4. QTL Analysis</strong></h2>
<div id="binary-phenotpe" class="section level3">
<h3><strong>binary phenotpe</strong></h3>
<p>I reran these a couple of different ways as I am not sure why its
difficult to get any results</p>
<div id="analysis-one" class="section level4">
<h4>** ANALYSIS ONE **</h4>
<ol style="list-style-type: decimal">
<li>removed additional low frequency markers as well as disproportionate
markers (identified in Step 2.4) within each sample set</li>
<li>no X covariate (ie. <code>Xcovar</code>); using sex as a
covariate</li>
<li>no kinship included as <code>r/qtl2</code> does not allow for the
inclusion of a kinship matrix in this type of model</li>
<li>For each model run, permutations (1000) were used for each model to
obtain genome-wide LOD significance threshold for p &lt; 0.01, p &lt;
0.05, p &lt; 0.10, respectively, separately for X and automsomes
(A)</li>
</ol>
<ul>
<li><a
href="4.1.1_qtl.analysis_binary_ici-sick.vs.ici-eoi_snpsqc_dis_no-x_updated_7.batches_myo.html"><code>ICI Sick vs ICI EOI</code></a></li>
<li><a
href="4.1.1_qtl.analysis_binary_ici.vs.pbs_snpsqc_dis_no-x_updated_7.batches_myo.html"><code>ICI vs PBS</code></a></li>
<li><a
href="4.1.1_qtl.analysis_binary_ici-myo-yes.vs.ici-myo-no_snpsqc_dis_no-x_updated_7.batches_myo.html"><code>ICI Myocarditis Yes vs ICI Myocarditis No</code></a></li>
<li><a
href="4.1.1_qtl.analysis_binary_pbs-myo-yes.vs.pbs-myo-no_snpsqc_dis_no-x_updated_7.batches_myo.html"><code>PBS Myocarditis Yes vs PBS Myocarditis No</code></a></li>
<li><a
href="4.1.1_qtl.analysis_binary_myo-yes.vs.myo-no_snpsqc_dis_no-x_updated_7.batches_myo.html"><code>Myocarditis Yes vs Myocarditis No</code></a></li>
<li><a
href="4.1.1_qtl.analysis_binary_het-ici.vs.het-pbs_snpsqc_dis_no-x_updated_7.batches_myo.html"><code>Hetero: ICI vs PBS</code></a></li>
<li><a
href="4.1.1_qtl.analysis_binary_het-ici-myo-yes.vs.het-ici-myo-no_snpsqc_dis_no-x_updated_7.batches_myo.html"><code>Hetero: ICI Myocarditis Yes vs ICI Myocarditis No</code></a></li>
</ul>
</div>
<div id="analysis-two" class="section level4">
<h4>** ANALYSIS TWO **</h4>
<p>The same as above but removing sex as a covariate (not sure if
confounding might be an issue)</p>
<ul>
<li><a
href="4.1.1_qtl.analysis_binary_ici-sick.vs.ici-eoi_snpsqc_dis_no-x_updated_no-sex_7.batches_myo.html"><code>ICI Sick vs ICI EOI</code></a></li>
<li><a
href="4.1.1_qtl.analysis_binary_ici.vs.pbs_snpsqc_dis_no-x_updated_no-sex_7.batches_myo.html"><code>ICI vs PBS</code></a></li>
<li><a
href="4.1.1_qtl.analysis_binary_ici-myo-yes.vs.ici-myo-no_snpsqc_dis_no-x_updated_no-sex_7.batches_myo.html"><code>ICI Myocarditis Yes vs ICI Myocarditis No</code></a></li>
<li><a
href="4.1.1_qtl.analysis_binary_pbs-myo-yes.vs.pbs-myo-no_snpsqc_dis_no-x_updated_no-sex_7.batches_myo.html"><code>PBS Myocarditis Yes vs PBS Myocarditis No</code></a></li>
<li><a
href="4.1.1_qtl.analysis_binary_myo-yes.vs.myo-no_snpsqc_dis_no-x_updated_no-sex_7.batches_myo.html"><code>Myocarditis Yes vs Myocarditis No</code></a></li>
<li><a
href="4.1.1_qtl.analysis_binary_het-ici.vs.het-pbs_snpsqc_dis_no-x_updated_no-sex_7.batches_myo.html"><code>Hetero: ICI vs PBS</code></a></li>
<li><a
href="4.1.1_qtl.analysis_binary_het-ici-myo-yes.vs.het-ici-myo-no_snpsqc_dis_no-x_updated_no-sex_7.batches_myo.html"><code>Hetero: ICI Myocarditis Yes vs ICI Myocarditis No</code></a></li>
</ul>
</div>
<div id="analysis-three" class="section level4">
<h4>** ANALYSIS THREE **</h4>
<p>same as analysis one but this time kept disproportionate markers
(removed low frequency markers). That is, kept sex as covariate.</p>
<ul>
<li><a
href="4.1.1_qtl.analysis_binary_ici-sick.vs.ici-eoi_snpsqc_7.batches_myo.html"><code>ICI Sick vs ICI EOI</code></a></li>
<li><a
href="4.1.1_qtl.analysis_binary_ici.vs.pbs_snpsqc_7.batches_myo.html"><code>ICI vs PBS</code></a></li>
<li><a
href="4.1.1_qtl.analysis_binary_ici-myo-yes.vs.ici-myo-no_snpsqc_7.batches_myo.html"><code>ICI Myocarditis Yes vs ICI Myocarditis No</code></a></li>
<li><a
href="4.1.1_qtl.analysis_binary_pbs-myo-yes.vs.pbs-myo-no_snpsqc_7.batches_myo.html"><code>PBS Myocarditis Yes vs PBS Myocarditis No</code></a></li>
<li><a
href="4.1.1_qtl.analysis_binary_myo-yes.vs.myo-no_snpsqc_7.batches_myo.html"><code>Myocarditis Yes vs Myocarditis No</code></a></li>
<li><a
href="4.1.1_qtl.analysis_binary_het-ici.vs.het-pbs_snpsqc_7.batches_myo.html"><code>Hetero: ICI vs PBS</code></a></li>
<li><a
href="4.1.1_qtl.analysis_binary_het-ici-myo-yes.vs.het-ici-myo-no_snpsqc_7.batches_myo.html"><code>Hetero: ICI Myocarditis Yes vs ICI Myocarditis No</code></a></li>
</ul>
</div>
<div id="analysis-four" class="section level4">
<h4>** ANALYSIS FOUR **</h4>
<p>same as analysis three but removed sex.</p>
<ul>
<li><a
href="4.1.1_qtl.analysis_binary_ici-sick.vs.ici-eoi_snpsqc_no-sex_7.batches_myo.html"><code>ICI Sick vs ICI EOI</code></a></li>
<li><a
href="4.1.1_qtl.analysis_binary_ici.vs.pbs_snpsqc_no-sex_7.batches_myo.html"><code>ICI vs PBS</code></a></li>
<li><a
href="4.1.1_qtl.analysis_binary_ici-myo-yes.vs.ici-myo-no_snpsqc_no-sex_7.batches_myo.html"><code>ICI Myocarditis Yes vs ICI Myocarditis No</code></a></li>
<li><a
href="4.1.1_qtl.analysis_binary_pbs-myo-yes.vs.pbs-myo-no_snpsqc_no-sex_7.batches_myo.html"><code>PBS Myocarditis Yes vs PBS Myocarditis No</code></a></li>
<li><a
href="4.1.1_qtl.analysis_binary_myo-yes.vs.myo-no_snpsqc_no-sex_7.batches_myo.html"><code>Myocarditis Yes vs Myocarditis No</code></a></li>
<li><a
href="4.1.1_qtl.analysis_binary_het-ici.vs.het-pbs_snpsqc_no-sex_7.batches_myo.html"><code>Hetero: ICI vs PBS</code></a></li>
<li><a
href="4.1.1_qtl.analysis_binary_het-ici-myo-yes.vs.het-ici-myo-no_snpsqc_no-sex_7.batches_myo.html"><code>Hetero: ICI Myocarditis Yes vs ICI Myocarditis No</code></a></li>
</ul>
</div>
<div id="analysis-five" class="section level4">
<h4>** ANALYSIS FIVE **</h4>
<p>suggested by Vivek P.:</p>
<ol style="list-style-type: decimal">
<li>map binary phenotype (drug treatment - <code>ICI vs. PBS</code>)
<ul>
<li>Murine MHC KO Status &amp; clinical phenotype as an additive
covariates</li>
<li>Qn: is there a genetic difference between drug treatment groups,
regardless of clinical phenotype, KO or Myocarditis Status?</li>
</ul></li>
<li>map binary phenotype (clinical phenotype -
<code>SICK vs. EOI</code>)
<ul>
<li>Murine MHC KO Status &amp; drug treatment as an additive
covariates</li>
<li>Qn: is there a genetic difference between clinical phenotype
samples, regardless of drug treatment, KO or Myocarditis Status?</li>
</ul></li>
<li>map binary phenotype (clinical phenotype - <code>SICK vs. EOI</code>
within each drug treatment)
<ul>
<li>Murine MHC KO Status &amp; drug treatment as an additive
covariates</li>
<li>Qn: is there a genetic difference between clinical phenotype
samples, regardless of KO or Myocarditis Status?</li>
</ul></li>
</ol>
</div>
</div>
</div>
<div id="power-analysis" class="section level2">
<h2><strong>5. Power Analysis</strong></h2>
<p>Using ANALYSIS ONE</p>
<ul>
<li><a
href="power.analysis_ici-sick.vs.ici-eoi_7.batches_myo.html"><code>ICI Sick vs ICI EOI</code></a></li>
<li><a
href="power.analysis_ici.vs.pbs_7.batches_myo.html"><code>ICI vs PBS</code></a></li>
<li><a
href="power.analysis_ici-myo-yes.vs.ici-myo-no_7.batches_myo.html"><code>ICI Myocarditis Yes vs ICI Myocarditis No</code></a></li>
<li><a
href="power.analysis_pbs-myo-yes.vs.pbs-myo-no_7.batches_myo.html"><code>PBS Myocarditis Yes vs PBS Myocarditis No</code></a></li>
<li><a
href="power.analysis_myo-yes.vs.myo-no_7.batches_myo.html"><code>Myocarditis Yes vs Myocarditis No</code></a></li>
<li><a
href="power.analysis_het-ici.vs.het-pbs_7.batches_myo.html"><code>Hetero: ICI vs PBS</code></a></li>
<li><a
href="power.analysis_het-ici-myo-yes.vs.het-ici-myo-no_7.batches_myo.html"><code>Hetero: ICI Myocarditis Yes vs ICI Myocarditis No</code></a></li>
</ul>
<br>
<p>
<button type="button" class="btn btn-default btn-workflowr btn-workflowr-sessioninfo" data-toggle="collapse" data-target="#workflowr-sessioninfo" style="display: block;">
<span class="glyphicon glyphicon-wrench" aria-hidden="true"></span>
Session information
</button>
</p>
<div id="workflowr-sessioninfo" class="collapse">
<pre><code>R version 4.2.2 (2022-10-31)
Platform: x86_64-apple-darwin17.0 (64-bit)
Running under: macOS Big Sur ... 10.16

Matrix products: default
BLAS:   /Library/Frameworks/R.framework/Versions/4.2/Resources/lib/libRblas.0.dylib
LAPACK: /Library/Frameworks/R.framework/Versions/4.2/Resources/lib/libRlapack.dylib

locale:
[1] en_US.UTF-8/en_US.UTF-8/en_US.UTF-8/C/en_US.UTF-8/en_US.UTF-8

attached base packages:
[1] stats     graphics  grDevices utils     datasets  methods   base     

other attached packages:
 [1] ggrepel_0.9.2     qtlcharts_0.16    qtl2_0.30         broman_0.80      
 [5] ggplot2_3.4.0     tibble_3.1.8      readxl_1.4.1      cluster_2.1.4    
 [9] dplyr_1.0.10      optparse_1.7.3    mclust_6.0.0      tidyr_1.2.1      
[13] data.table_1.14.6 knitr_1.42        kableExtra_1.3.4  workflowr_1.7.0  

loaded via a namespace (and not attached):
 [1] Rcpp_1.0.9        svglite_2.1.0     getPass_0.2-2     ps_1.7.2         
 [5] assertthat_0.2.1  rprojroot_2.0.3   digest_0.6.31     utf8_1.2.2       
 [9] R6_2.5.1          cellranger_1.1.0  RSQLite_2.2.19    evaluate_0.20    
[13] httr_1.4.4        pillar_1.8.1      rlang_1.1.0       rstudioapi_0.14  
[17] blob_1.2.3        whisker_0.4.1     callr_3.7.3       jquerylib_0.1.4  
[21] rmarkdown_2.21.2  qtl_1.54          webshot_0.5.4     stringr_1.5.0    
[25] bit_4.0.5         munsell_0.5.0     compiler_4.2.2    httpuv_1.6.6     
[29] xfun_0.38         pkgconfig_2.0.3   systemfonts_1.0.4 htmltools_0.5.5  
[33] tidyselect_1.2.0  fansi_1.0.3       viridisLite_0.4.1 withr_2.5.0      
[37] later_1.3.0       grid_4.2.2        jsonlite_1.8.4    gtable_0.3.1     
[41] lifecycle_1.0.3   DBI_1.1.3         git2r_0.30.1      magrittr_2.0.3   
[45] scales_1.2.1      cli_3.6.1         stringi_1.7.12    cachem_1.0.7     
[49] fs_1.6.1          promises_1.2.0.1  getopt_1.20.3     xml2_1.3.3       
[53] bslib_0.4.2       generics_0.1.3    vctrs_0.6.1       tools_4.2.2      
[57] bit64_4.0.5       glue_1.6.2        purrr_0.3.5       parallel_4.2.2   
[61] processx_3.8.0    fastmap_1.1.1     yaml_2.3.7        colorspace_2.0-3 
[65] rvest_1.0.3       memoise_2.0.1     sass_0.4.5       </code></pre>
</div>
</div>


<!-- Adjust MathJax settings so that all math formulae are shown using
TeX fonts only; see
https://docs.mathjax.org/en/latest/web/configuration.html. This will make
the presentation more consistent at the cost of the webpage sometimes
taking slightly longer to load. Note that this only works because the
footer is added to webpages before the MathJax javascript. -->
<script type="text/x-mathjax-config">
  MathJax.Hub.Config({
    "HTML-CSS": { availableFonts: ["TeX"] }
  });
</script>





</div>

<script>

// add bootstrap table styles to pandoc tables
function bootstrapStylePandocTables() {
  $('tr.odd').parent('tbody').parent('table').addClass('table table-condensed');
}
$(document).ready(function () {
  bootstrapStylePandocTables();
});


</script>

<!-- tabsets -->

<script>
$(document).ready(function () {
  window.buildTabsets("TOC");
});

$(document).ready(function () {
  $('.tabset-dropdown > .nav-tabs > li').click(function () {
    $(this).parent().toggleClass('nav-tabs-open');
  });
});
</script>

<!-- code folding -->


<!-- dynamically load mathjax for compatibility with self-contained -->
<script>
  (function () {
    var script = document.createElement("script");
    script.type = "text/javascript";
    script.src  = "https://mathjax.rstudio.com/latest/MathJax.js?config=TeX-AMS-MML_HTMLorMML";
    document.getElementsByTagName("head")[0].appendChild(script);
  })();
</script>

</body>
</html>
