---
title: "QTL Analysis - rz transformed age of onset (rz.age) [ICI vs PBS] (corrected phenotype with outliers removed if any)"
author: "Belinda Cornes"
date: "`r Sys.Date()`"
output: html_document
---

```{r setup, include=FALSE}
knitr::opts_chunk$set(echo = FALSE,warning=FALSE)

library("kableExtra")
library("knitr")
options(stringsAsFactors = FALSE)
library(data.table) 
library(tidyr)     
library(mclust)     
library(rhdf5)      
library(optparse)
library(dplyr)
library(cluster)
library(readxl)
library(psych)
library(tibble)
library(ggplot2)
library(reshape2)
library(qtl2)
#library(qtl)
library(abind)

setwd("/Users/corneb/Documents/MyJax/CS/Projects/Serreze/qc/workflowr/Serreze-T1D_Workflow")

```

# Loading Data

```{r data, echo=TRUE}

load("data/gm_allqc_5.batches_mis.RData")

#gm_allqc
gm=gm_allqc
gm
#pr <- readRDS("data/serreze_probs_allqc.rds")
#pr <- readRDS("data/serreze_probs.rds")

##extracting animals with ici and pbs group status
miceinfo <- gm$covar[gm$covar$group == "PBS" | gm$covar$group == "ICI",]
table(miceinfo$group)
mice.ids <- rownames(miceinfo)

gm <- gm[mice.ids]
gm
table(gm$covar$group)
gm.full <- gm

covars <- read_csv("data/covar_corrected.cleaned_ici.vs.pbs_5.batches_mis.csv")
#removing any missing info
missing_ids <- covars[which(covars$"rz.age"==""),]$Mouse.ID
length(missing_ids)
missing_ids

if(length(missing_ids) != 0){
gm <- gm[paste0("-",as.character(missing_ids)),]
gm
table(gm$covar$group)

#keeping only mouseids in covars
covars <- covars[gm$covar$id,]
table(covars$group)
}

##removing problmetic marker

gm <- drop_markers(gm, "UNCHS013106")

query_variants <- create_variant_query_func("/Users/corneb/Documents/MyJax/CS/Projects/support.files/qtl2/cc_variants.sqlite")
query_genes <- create_gene_query_func("/Users/corneb/Documents/MyJax/CS/Projects/support.files/qtl2/mouse_genes_mgi.sqlite")


##dropping monomorphic markers within the dataset

g <- do.call("cbind", gm$geno)

gf_mar <- t(apply(g, 2, function(a) table(factor(a, 1:2))/sum(a != 0)))
#gn_mar <- t(apply(g, 2, function(a) table(factor(a, 1:2))))

gf_mar <- gf_mar[gf_mar[,2] != "NaN",]

count <- rowSums(gf_mar <=0.05)
low_freq_df <- merge(as.data.frame(gf_mar),as.data.frame(count), by="row.names",all=T)
low_freq_df[is.na(low_freq_df)] <- ''
low_freq_df <- low_freq_df[low_freq_df$count == 1,]
rownames(low_freq_df) <- low_freq_df$Row.names

low_freq <- find_markerpos(gm, rownames(low_freq_df))
low_freq$id <- rownames(low_freq)

nrow(low_freq)

low_freq_bad <- merge(low_freq,low_freq_df, by="row.names",all=T)
names(low_freq_bad)[1] <- c("marker")

gf_mar <- gf_mar[gf_mar[,2] != "NaN",]
MAF <- apply(gf_mar, 1, function(x) min(x))
MAF <- as.data.frame(MAF)
MAF$index <- 1:nrow(gf_mar)
gf_mar_maf <- merge(gf_mar,as.data.frame(MAF), by="row.names")
gf_mar_maf <- gf_mar_maf[order(gf_mar_maf$index),]

gfmar <- NULL
gfmar$gfmar_mar_0 <- sum(gf_mar_maf$MAF==0)
gfmar$gfmar_mar_1 <- sum(gf_mar_maf$MAF< 0.01)
gfmar$gfmar_mar_5 <- sum(gf_mar_maf$MAF< 0.05)
gfmar$gfmar_mar_10 <- sum(gf_mar_maf$MAF< 0.10)
gfmar$gfmar_mar_15 <- sum(gf_mar_maf$MAF< 0.15)
gfmar$gfmar_mar_25 <- sum(gf_mar_maf$MAF< 0.25)
gfmar$gfmar_mar_50 <- sum(gf_mar_maf$MAF< 0.50)
gfmar$total_snps <- nrow(as.data.frame(gf_mar_maf))

gfmar <- t(as.data.frame(gfmar))
gfmar <- as.data.frame(gfmar)
gfmar$count <- gfmar$V1

gfmar[c(2)] %>%
  kable(escape = F,align = c("ccccccccc"),linesep ="\\hline") %>%
  kable_styling(full_width = F) %>%
  kable_styling("striped", full_width = F)  %>%
  row_spec(8 ,bold=T,color= "white",background = "black")


gm_qc <- drop_markers(gm, low_freq_bad$marker)
gm_qc <- drop_nullmarkers(gm_qc)

gm_qc

## dropping disproportionate markers
dismark <- read.csv("data/ici.vs.pbs_marker.freq_low.geno.freq.removed_geno.ratio_5.batches_mis.csv")
nrow(dismark)
names(dismark)[1] <- c("marker")
dismark <- dismark[!dismark$Include,]
nrow(dismark)

gm_qc_dis <- drop_markers(gm_qc, dismark$marker)
gm_qc_dis <- drop_nullmarkers(gm_qc_dis)

gm = gm_qc_dis
gm

markers <- marker_names(gm)
gmapdf <- read.csv("/Users/corneb/Documents/MyJax/CS/Projects/Serreze/haplotype.reconstruction/output_5.batches/genetic_map.csv")
pmapdf <- read.csv("/Users/corneb/Documents/MyJax/CS/Projects/Serreze/haplotype.reconstruction/output_5.batches/physical_map.csv")
#mapdf <- merge(gmapdf,pmapdf, by=c("marker","chr"), all=T)
#rownames(mapdf) <- mapdf$marker
#mapdf <- mapdf[markers,]
#names(mapdf) <- c('marker','chr','gmapdf','pmapdf')
#mapdfnd <- mapdf[!duplicated(mapdf[c(2:3)]),]

pr.qc <- calc_genoprob(gm)

```


# Preparing necessary info

```{r permutation prelim, fig.height = 6, fig.width = 9.5, fig.align = "center", echo=TRUE, results="asis"}

covars$rz.age <- as.numeric(covars$rz.age)
Xcovar <- get_x_covar(gm)
addcovar = model.matrix(~ICI.vs.PBS, data = covars)[,-1]
#addcovar.i = model.matrix(~ICI.vs.PBS, data = covars)[,-1]

K <- calc_kinship(pr.qc, type = "loco")
#heatmap(K[[1]])
K.overall <- calc_kinship(pr.qc, type = "overall")
#heatmap(K.overall)
kinship <- calc_kinship(pr.qc)
heatmap(kinship)

```

## No additive covariates

### Significance thresholds

```{r permutation, fig.height = 6, fig.width = 9.5, fig.align = "center", echo=TRUE, results="asis"}

operm <- scan1perm(pr.qc, covars["rz.age"], model="normal", n_perm=1000, perm_Xsp=TRUE, chr_lengths=chr_lengths(gm$gmap))

summary_table<-data.frame(unclass(summary(operm, alpha=c(0.01,  0.05, 0.1))))
names(summary_table) <- c("autosomes","X")
summary_table$significance.level <- rownames(summary_table)

rownames(summary_table) <- NULL

summary_table[c(3,1:2)] %>%
  kable(escape = F,align = c("ccc")) %>%
  kable_styling("striped", full_width = T) %>%
  column_spec(1, bold=TRUE)


```

### Genome-wide scan

```{r scanone, fig.height = 6, fig.width = 9.5, fig.align = "center", echo=TRUE, results="asis"}

plot_lod<-function(out,map){
  for (i in 1:dim(out)[2]){
#par(mar=c(5.1, 6.1, 1.1, 1.1))
    ymx <- maxlod(out) # overall maximum LOD score
    plot(out, map, lodcolumn=i, col="slateblue", ylim=c(0, ymx+0.5))
    #legend("topright", lwd=2, colnames(out)[i], bg="gray90")
    title(main = paste0(colnames(out)[i], " - ICI vs. PBS (no covariates) [positions in cM]"))
    add_threshold(map,  summary(operm,alpha=0.1), col = 'purple')
    add_threshold(map,  summary(operm, alpha=0.05), col = 'red')
    add_threshold(map,  summary(operm, alpha=0.01), col = 'blue')

  }
}


print("with no kinship")
outf <- scan1(pr.qc, covars["rz.age"], Xcovar=Xcovar, model="normal")

plot_lod(outf,gm$gmap)

print("with normal kinship")
outk <- scan1(pr.qc, covars["rz.age"], Xcovar=Xcovar, model="normal", kinship=kinship)

plot_lod(outk,gm$gmap)


print("with loco kinship")
outlk <- scan1(pr.qc, covars["rz.age"], Xcovar=Xcovar, model="normal", kinship=K)

plot_lod(outlk,gm$gmap)

print("no kinship and no xcovar")
out <- scan1(pr.qc, covars["rz.age"], model="normal")

plot_lod(out,gm$gmap)


```

### LOD peaks

#### Centimorgan (cM)

```{r LOD peaks, fig.height = 6, fig.width = 9.5, fig.align = "center", echo=TRUE, results="asis"}

peaks<-find_peaks(out, gm$gmap, threshold=summary(operm,alpha=0.05)$A, thresholdX = summary(operm,alpha=0.05)$X, peakdrop=3, drop=1.5)


if(nrow(peaks) >0){
peaks$marker <- find_marker(gm$gmap, chr=peaks$chr,pos=peaks$pos)
names(peaks)[2] <- c("phenotype")
peaks <- peaks[-1]

rownames(peaks) <- NULL
print(kable(peaks, escape = F, align = c("cccccccc"), "html") 
  %>% kable_styling("striped", full_width = T)%>%
  column_spec(1, bold=TRUE)
  )

#plot only peak chromosomes

plot_lod_chr<-function(out,map,chrom){
  for (i in 1:dim(out)[2]){
    #png(filename=paste0("/Users/chenm/Documents/qtl/Jai/",colnames(out)[i],  "_lod.png"))
    
    #par(mar=c(5.1, 6.1, 1.1, 1.1))
    ymx <- maxlod(out) # overall maximum LOD score
    plot(out, map, chr = chrom, lodcolumn=i, col="slateblue", ylim=c(0, ymx+0.5))
    #legend("topright", lwd=2, colnames(out)[i], bg="gray90")
    title(main = paste0(colnames(out)[i], " - chr", chrom, " - ICI vs. PBS (no covariates) [positions in cM]"))
    add_threshold(map,  summary(operm,alpha=0.1), col = 'purple')
    add_threshold(map,  summary(operm, alpha=0.05), col = 'red')
    add_threshold(map,  summary(operm, alpha=0.01), col = 'blue')
    #for (j in 1: dim(summary_table)[1]){
    #  abline(h=summary_table[j, i],col="red")
    #  text(x=400, y =summary_table[j, i]+0.12, labels = paste("p=", row.names(summary_table)[j]))
    #}
    #dev.off()

  }
}


for(i in unique(peaks$chr)){
#for (i in 1:nrow(peaks)){
  #plot_lod_chr(out,gm$gmap, peaks$chr[i])
  plot_lod_chr(out,gm$gmap, i)
}

} else {
  print(paste0("There are no peaks that have a LOD that reaches suggestive (p<0.05) level of ",summary(operm,alpha=0.05)$A, " [autosomes]/",summary(operm,alpha=0.05)$X, " [x-chromosome]"))
}

```

#### Megabase (MB)

```{r LOD peaks_mba, fig.height = 6, fig.width = 9.5, fig.align = "center", echo=TRUE, results="asis"}

peaks_mba<-find_peaks(out, gm$pmap, threshold=summary(operm,alpha=0.05)$A, thresholdX = summary(operm,alpha=0.05)$X, peakdrop=3, drop=1.5)

if(nrow(peaks) >0){
peaks_mba$marker <- find_marker(gm$pmap, chr=peaks_mba$chr,pos=peaks_mba$pos)
names(peaks_mba)[2] <- c("phenotype")
peaks_mba <- peaks_mba[-1]

rownames(peaks_mba) <- NULL
print(kable(peaks_mba, escape = F, align = c("cccccccc"), "html") 
  %>% kable_styling("striped", full_width = T)%>%
  column_spec(1, bold=TRUE)
  )

  plot_lod_chr_mb<-function(out,map,chrom){
  for (i in 1:dim(out)[2]){
    #png(filename=paste0("/Users/chenm/Documents/qtl/Jai/",colnames(out)[i],  "_lod.png"))
    
    #par(mar=c(5.1, 6.1, 1.1, 1.1))
    ymx <- maxlod(out) # overall maximum LOD score
    plot(out, map, chr = chrom, lodcolumn=i, col="slateblue", ylim=c(0, ymx+0.5))
    #legend("topright", lwd=2, colnames(out)[i], bg="gray90")
    title(main = paste0(colnames(out)[i], " - chr", chrom, " - ICI vs. PBS (no covariates) [positions in cM]"))
    add_threshold(map,  summary(operm,alpha=0.1), col = 'purple')
    add_threshold(map,  summary(operm, alpha=0.05), col = 'red')
    add_threshold(map,  summary(operm, alpha=0.01), col = 'blue')
    #for (j in 1: dim(summary_table)[1]){
    #  abline(h=summary_table[j, i],col="red")
    #  text(x=400, y =summary_table[j, i]+0.12, labels = paste("p=", row.names(summary_table)[j]))
    #}
    #dev.off()
  }
}

for(i in unique(peaks_mba$chr)){
#for (i in 1:nrow(peaks_mba)){
  #plot_lod_chr_mb(out,gm$pmap, peaks_mba$chr[i])
  plot_lod_chr_mb(out,gm$pmap,i)
}

} else {
  print(paste0("There are no peaks that have a LOD that reaches suggestive (p<0.05) level of ",summary(operm,alpha=0.05)$A, " [autosomes]/",summary(operm,alpha=0.05)$X, " [x-chromosome]"))
}

```

### QTL effects

For each peak LOD location we give a list of gene

```{r QTL effects, fig.height = 6, fig.width = 9.5, fig.align = "center", echo=TRUE, results="asis"}


if(nrow(peaks) >0){
for (i in 1:nrow(peaks)){
#for (i in 1:1){
  #Plot 1
  g <- maxmarg(pr.qc, gm$gmap, chr=peaks$chr[i], pos=peaks$pos[i], return_char=TRUE)
  #png(filename=paste0("/Users/chenm/Documents/qtl/Jai/","qtl_effect_", i, ".png"))
  #par(mar=c(4.1, 4.1, 1.5, 0.6))
  plot_pxg(g, covars[,peaks$phenotype[i]], ylab=peaks$phenotype[i], sort=FALSE)
  title(main = paste0("chr: ", chr=peaks$chr[i],"; pos: ", peaks$pos[i], "cM /",peaks_mba$pos[i],"MB\n(",peaks$phenotype[i]," )"), line=0.2)
  ##dev.off()

  chr = peaks$chr[i]

# Plot 2
  pr_sub <- pull_genoprobint(pr.qc, gm$gmap, chr, c(peaks$ci_lo[i], peaks$ci_hi[i]))
  #coeff <- scan1coef(pr[,chr], cross$pheno[,peaks$lodcolumn[i]], addcovar = addcovar)
  #coeff <- scan1coef(pr[,chr], cross$pheno[,peaks$lodcolumn[i]], Xcovar=Xcovar)
  #coeff <- scan1coef(pr.qc[,chr], gm$covar[peaks$lodcolumn[i]], model="binary")
  #coeff_sub <- scan1coef(pr_sub[,chr], gm$covar[peaks$lodcolumn[i]], model="binary")
  blup <- scan1blup(pr.qc[,chr], covars[peaks$phenotype[i]])
  blup_sub <- scan1blup(pr_sub[,chr], covars[peaks$phenotype[i]])

  write.csv(as.data.frame(blup_sub), paste0("data/ici.vs.pbs_",peaks$phenotype[i],"-no.covariates_blup_sub_chr-",chr,"_peak.marker-",peaks$marker[i],"_lod.drop-1.5_snpsqc_dis_no-x_updated_5.batches_mis.csv"), quote=F)

  #plot_coef(coeff, 
  #     gm$gmap, columns=1:2,
  #     bgcolor="gray95", legend="bottomleft", 
  #     main = paste("chr", chr=peaks$chr[i],"; pos: ", peaks$pos[i], "cM /",peaks_mba$pos[i],"MB\n(",peaks$lodcolumn[i]," [scan1coeff; positions in cM] )")
  #     )

  #plot_coef(coeff_sub, 
  #     gm$gmap, columns=1:2,
  #     bgcolor="gray95", legend="bottomleft", 
  #     main = paste("chr", chr=peaks$chr[i],"; pos: ", peaks$pos[i], "cM /",peaks_mba$pos[i],"MB\n(",peaks$lodcolumn[i],"; 1.5 LOD drop interval [scan1coeff; positions in cM] ) ")
  #     )


  plot_coef(blup, 
       gm$gmap, columns=1:2,
       bgcolor="gray95", legend="bottomleft", 
       main = paste0("chr: ", chr=peaks$chr[i],"; pos: ", peaks$pos[i], "cM /",peaks_mba$pos[i],"MB\n(",peaks$phenotype[i]," [scan1blup; positions in cM] )")
       )

  plot_coef(blup_sub, 
       gm$gmap, columns=1:2,
       bgcolor="gray95", legend="bottomleft", 
       main = paste0("chr: ", chr=peaks$chr[i],"; pos: ", peaks$pos[i], "cM /",peaks_mba$pos[i],"MB\n(",peaks$phenotype[i],"; 1.5 LOD drop interval [scan1blup; positions in cM] )")
       )


 # Plot 3
  #c2effB <- scan1coef(pr.qc[,chr], gm$covar[peaks$lodcolumn[i]], model="binary", contrasts=cbind(a=c(-1, 0), d=c(0, -1)))
  #c2effBb <- scan1blup(pr.qc[,chr], gm$covar[peaks$lodcolumn[i]], contrasts=cbind(a=c(-1, 0), d=c(0, -1)))
  ##c2effB <- scan1coef(pr[,chr], cross$pheno[,peaks$lodcolumn[i]], addcovar = addcovar, contrasts=cbind(mu=c(1,1,1), a=c(-1, 0, 1), d=c(0, 1, 0)))
  ##c2effB <- scan1coef(pr[,chr], cross$pheno[,peaks$lodcolumn[i]],Xcovar=Xcovar, contrasts=cbind(mu=c(1,1,1), a=c(-1, 0, 1), d=c(0, 1, 0)))
  #plot(c2effB, gm$gmap[chr], columns=1:2,
  #     bgcolor="gray95", legend="bottomleft", 
  #     main = paste("chr", chr=peaks$chr[i], "pos", peaks$pos[i], "(",peaks$lodcolumn[i],")")
  #     )
  #plot(c2effBb, gm$gmap[chr], columns=1:2,
  #     bgcolor="gray95", legend="bottomleft", 
  #     main = paste("chr", chr=peaks$chr[i], "pos", peaks$pos[i], "(",peaks$lodcolumn[i],")")
  #     )
  ##last_coef <- unclass(c2effB)[nrow(c2effB),2:3] # last two coefficients
  ##for(t in seq(along=last_coef))
  ##  axis(side=4, at=last_coef[t], names(last_coef)[t], tick=FALSE)


  #Table 1
  chr = peaks_mba$chr[i]
  start=as.numeric(peaks_mba$ci_lo[i])
  end=as.numeric(peaks_mba$ci_hi[i])

  genesgss = query_genes(chr, start, end)

  write.csv(genesgss, file=paste0("data/ici.vs.pbs_",peaks$phenotype[i],"-no.covariates_genes_chr-",chr,"_peak.marker-",peaks$marker[i],"_lod.drop-1.5_snpsqc_dis_no-x_updated_5.batches_mis.csv"), quote=F)

  rownames(genesgss) <- NULL
  genesgss$strand_old = genesgss$strand
  genesgss$strand[genesgss$strand=="+"] <- "positive"
  genesgss$strand[genesgss$strand=="-"] <- "negative"

  #genesgss <- 
  #table <- 
  #genesgss[,c("chr","type","start","stop","strand","ID","Name","Dbxref","gene_id","mgi_type","description")] %>%
  #kable(escape = F,align = c("ccccccccccc")) %>%
  #kable_styling("striped", full_width = T) #%>% 
  #cat #%>%
  #column_spec(1, bold=TRUE)
#
  #print(kable(genesgss[,c("chr","type","start","stop","strand","ID","Name","Dbxref","gene_id","mgi_type","description")], escape = F,align = c("ccccccccccc")))

  print(kable(genesgss[,c("chr","type","start","stop","strand","ID","Name","Dbxref","gene_id","mgi_type","description")], "html") %>% kable_styling("striped", full_width = T))

  #table
  

}

} else {
  print(paste0("There are no peaks that have a LOD that reaches suggestive (p<0.05) level of ",summary(operm,alpha=0.05)$A, " [autosomes]/",summary(operm,alpha=0.05)$X, " [x-chromosome]"))
}



```



## Additive covariates


### Significance thresholds

```{r permutationa, fig.height = 6, fig.width = 9.5, fig.align = "center", echo=TRUE, results="asis"}

operm <- scan1perm(pr.qc, covars["rz.age"], model="normal", addcovar=addcovar, n_perm=1000, perm_Xsp=TRUE, chr_lengths=chr_lengths(gm$gmap))

summary_table<-data.frame(unclass(summary(operm, alpha=c(0.01,  0.05, 0.1))))
names(summary_table) <- c("autosomes","X")
summary_table$significance.level <- rownames(summary_table)

rownames(summary_table) <- NULL

summary_table[c(3,1:2)] %>%
  kable(escape = F,align = c("ccc")) %>%
  kable_styling("striped", full_width = T) %>%
  column_spec(1, bold=TRUE)

```

### Genome-wide scan

```{r scanonea, fig.height = 6, fig.width = 9.5, fig.align = "center", echo=TRUE, results="asis"}

plot_lod<-function(out,map){
  for (i in 1:dim(out)[2]){
#par(mar=c(5.1, 6.1, 1.1, 1.1))
    ymx <- maxlod(out) # overall maximum LOD score
    plot(out, map, lodcolumn=i, col="slateblue", ylim=c(0, ymx+0.5))
    #legend("topright", lwd=2, colnames(out)[i], bg="gray90")
    title(main = paste0(colnames(out)[i], " - ICI vs. PBS (with additive covariates) [positions in cM]"))
    add_threshold(map,  summary(operm,alpha=0.1), col = 'purple')
    add_threshold(map,  summary(operm, alpha=0.05), col = 'red')
    add_threshold(map,  summary(operm, alpha=0.01), col = 'blue')

  }
}


print("with no kinship")
outfa <- scan1(pr.qc, covars["rz.age"], Xcovar=Xcovar, model="normal", addcovar=addcovar)

plot_lod(outfa,gm$gmap)

print("with normal kinship")
outka <- scan1(pr.qc, covars["rz.age"], Xcovar=Xcovar, model="normal", addcovar=addcovar, kinship=kinship)

plot_lod(outka,gm$gmap)

print("with loco kinship")
outlk <- scan1(pr.qc, covars["rz.age"], Xcovar=Xcovar, model="normal", addcovar=addcovar, kinship=K)

plot_lod(outlk,gm$gmap)

print("no kinship and no xcovar")
outa <- scan1(pr.qc, covars["rz.age"], model="normal",addcovar=addcovar)

plot_lod(outa,gm$gmap)

out = outa


```

### LOD peaks

#### Centimorgan (cM)

```{r LOD peaksa, fig.height = 6, fig.width = 9.5, fig.align = "center", echo=TRUE, results="asis"}

peaks<-find_peaks(out, gm$gmap, threshold=summary(operm,alpha=0.05)$A, thresholdX = summary(operm,alpha=0.05)$X, peakdrop=3, drop=1.5)

if(nrow(peaks) >0){
peaks$marker <- find_marker(gm$gmap, chr=peaks$chr,pos=peaks$pos)
names(peaks)[2] <- c("phenotype")
peaks <- peaks[-1]

rownames(peaks) <- NULL
print(kable(peaks, escape = F, align = c("cccccccc"), "html") 
  %>% kable_styling("striped", full_width = T)%>%
  column_spec(1, bold=TRUE)
  )

#plot only peak chromosomes

plot_lod_chr<-function(out,map,chrom){
  for (i in 1:dim(out)[2]){
    #png(filename=paste0("/Users/chenm/Documents/qtl/Jai/",colnames(out)[i],  "_lod.png"))
    
    #par(mar=c(5.1, 6.1, 1.1, 1.1))
    ymx <- maxlod(out) # overall maximum LOD score
    plot(out, map, chr = chrom, lodcolumn=i, col="slateblue", ylim=c(0, ymx+0.5))
    #legend("topright", lwd=2, colnames(out)[i], bg="gray90")
    title(main = paste0(colnames(out)[i], " - chr", chrom, " - ICI vs. PBS (with additive covariates) [positions in cM]"))
    add_threshold(map,  summary(operm,alpha=0.1), col = 'purple')
    add_threshold(map,  summary(operm, alpha=0.05), col = 'red')
    add_threshold(map,  summary(operm, alpha=0.01), col = 'blue')
    #for (j in 1: dim(summary_table)[1]){
    #  abline(h=summary_table[j, i],col="red")
    #  text(x=400, y =summary_table[j, i]+0.12, labels = paste("p=", row.names(summary_table)[j]))
    #}
    #dev.off()

  }
}


for(i in unique(peaks$chr)){
#for (i in 1:nrow(peaks)){
  #plot_lod_chr(out,gm$gmap, peaks$chr[i])
  plot_lod_chr(out,gm$gmap, i)
}

} else {
  print(paste0("There are no peaks that have a LOD that reaches suggestive (p<0.05) level of ",summary(operm,alpha=0.05)$A, " [autosomes]/",summary(operm,alpha=0.05)$X, " [x-chromosome]"))
}

```

#### Megabase (MB)

```{r LOD peaks_mbaa, fig.height = 6, fig.width = 9.5, fig.align = "center", echo=TRUE, results="asis"}

peaks_mba<-find_peaks(out, gm$pmap, threshold=summary(operm,alpha=0.05)$A, thresholdX = summary(operm,alpha=0.05)$X, peakdrop=3, drop=1.5)

if(nrow(peaks) >0){
peaks_mba$marker <- find_marker(gm$pmap, chr=peaks_mba$chr,pos=peaks_mba$pos)
names(peaks_mba)[2] <- c("phenotype")
peaks_mba <- peaks_mba[-1]

rownames(peaks_mba) <- NULL
print(kable(peaks_mba, escape = F, align = c("cccccccc"), "html") 
  %>% kable_styling("striped", full_width = T)%>%
  column_spec(1, bold=TRUE)
  )

plot_lod_chr_mb<-function(out,map,chrom){
  for (i in 1:dim(out)[2]){
    #png(filename=paste0("/Users/chenm/Documents/qtl/Jai/",colnames(out)[i],  "_lod.png"))
    
    #par(mar=c(5.1, 6.1, 1.1, 1.1))
    ymx <- maxlod(out) # overall maximum LOD score
    plot(out, map, chr = chrom, lodcolumn=i, col="slateblue", ylim=c(0, ymx+0.5))
    #legend("topright", lwd=2, colnames(out)[i], bg="gray90")
    title(main = paste0(colnames(out)[i], " - chr", chrom, " - ICI vs. PBS (with additive covariates) [positions in cM]"))
    add_threshold(map,  summary(operm,alpha=0.1), col = 'purple')
    add_threshold(map,  summary(operm, alpha=0.05), col = 'red')
    add_threshold(map,  summary(operm, alpha=0.01), col = 'blue')
    #for (j in 1: dim(summary_table)[1]){
    #  abline(h=summary_table[j, i],col="red")
    #  text(x=400, y =summary_table[j, i]+0.12, labels = paste("p=", row.names(summary_table)[j]))
    #}
    #dev.off()
  }
}

for(i in unique(peaks_mba$chr)){
#for (i in 1:nrow(peaks_mba)){
  #plot_lod_chr_mb(out,gm$pmap, peaks_mba$chr[i])
  plot_lod_chr_mb(out,gm$pmap,i)
}

} else {
  print(paste0("There are no peaks that have a LOD that reaches suggestive (p<0.05) level of ",summary(operm,alpha=0.05)$A, " [autosomes]/",summary(operm,alpha=0.05)$X, " [x-chromosome]"))
}

```

### QTL effects

For each peak LOD location we give a list of gene

```{r QTL effectsa, fig.height = 6, fig.width = 9.5, fig.align = "center", echo=TRUE, results="asis"}

if(nrow(peaks) >0){
for (i in 1:nrow(peaks)){
#for (i in 1:1){
  #Plot 1
  g <- maxmarg(pr.qc, gm$gmap, chr=peaks$chr[i], pos=peaks$pos[i], return_char=TRUE)
  #png(filename=paste0("/Users/chenm/Documents/qtl/Jai/","qtl_effect_", i, ".png"))
  #par(mar=c(4.1, 4.1, 1.5, 0.6))
  plot_pxg(g, covars[,peaks$phenotype[i]], ylab=peaks$phenotype[i], sort=FALSE)
  title(main = paste0("chr: ", chr=peaks$chr[i],"; pos: ", peaks$pos[i], "cM /",peaks_mba$pos[i],"MB\n(",peaks$phenotype[i]," )"), line=0.2)
  ##dev.off()

  chr = peaks$chr[i]

# Plot 2
  pr_sub <- pull_genoprobint(pr.qc, gm$gmap, chr, c(peaks$ci_lo[i], peaks$ci_hi[i]))
  #coeff <- scan1coef(pr[,chr], cross$pheno[,peaks$lodcolumn[i]], addcovar = addcovar)
  #coeff <- scan1coef(pr[,chr], cross$pheno[,peaks$lodcolumn[i]], Xcovar=Xcovar)
  #coeff <- scan1coef(pr.qc[,chr], gm$covar[peaks$lodcolumn[i]], model="binary")
  #coeff_sub <- scan1coef(pr_sub[,chr], gm$covar[peaks$lodcolumn[i]], model="binary")
  blup <- scan1blup(pr.qc[,chr], covars[peaks$phenotype[i]])
  blup_sub <- scan1blup(pr_sub[,chr], covars[peaks$phenotype[i]])

  write.csv(as.data.frame(blup_sub), paste0("data/ici.vs.pbs_",peaks$phenotype[i],"-additive.covariates_blup_sub_chr-",chr,"_peak.marker-",peaks$marker[i],"_lod.drop-1.5_snpsqc_dis_no-x_updated_5.batches_mis.csv"), quote=F)

  #plot_coef(coeff, 
  #     gm$gmap, columns=1:2,
  #     bgcolor="gray95", legend="bottomleft", 
  #     main = paste("chr", chr=peaks$chr[i],"; pos: ", peaks$pos[i], "cM /",peaks_mba$pos[i],"MB\n(",peaks$lodcolumn[i]," [scan1coeff; positions in cM] )")
  #     )

  #plot_coef(coeff_sub, 
  #     gm$gmap, columns=1:2,
  #     bgcolor="gray95", legend="bottomleft", 
  #     main = paste("chr", chr=peaks$chr[i],"; pos: ", peaks$pos[i], "cM /",peaks_mba$pos[i],"MB\n(",peaks$lodcolumn[i],"; 1.5 LOD drop interval [scan1coeff; positions in cM] ) ")
  #     )


  plot_coef(blup, 
       gm$gmap, columns=1:2,
       bgcolor="gray95", legend="bottomleft", 
       main = paste0("chr: ", chr=peaks$chr[i],"; pos: ", peaks$pos[i], "cM /",peaks_mba$pos[i],"MB\n(",peaks$phenotype[i]," [scan1blup; positions in cM] )")
       )

  plot_coef(blup_sub, 
       gm$gmap, columns=1:2,
       bgcolor="gray95", legend="bottomleft", 
       main = paste0("chr: ", chr=peaks$chr[i],"; pos: ", peaks$pos[i], "cM /",peaks_mba$pos[i],"MB\n(",peaks$phenotype[i],"; 1.5 LOD drop interval [scan1blup; positions in cM] )")
       )


 # Plot 3
  #c2effB <- scan1coef(pr.qc[,chr], gm$covar[peaks$lodcolumn[i]], model="binary", contrasts=cbind(a=c(-1, 0), d=c(0, -1)))
  #c2effBb <- scan1blup(pr.qc[,chr], gm$covar[peaks$lodcolumn[i]], contrasts=cbind(a=c(-1, 0), d=c(0, -1)))
  ##c2effB <- scan1coef(pr[,chr], cross$pheno[,peaks$lodcolumn[i]], addcovar = addcovar, contrasts=cbind(mu=c(1,1,1), a=c(-1, 0, 1), d=c(0, 1, 0)))
  ##c2effB <- scan1coef(pr[,chr], cross$pheno[,peaks$lodcolumn[i]],Xcovar=Xcovar, contrasts=cbind(mu=c(1,1,1), a=c(-1, 0, 1), d=c(0, 1, 0)))
  #plot(c2effB, gm$gmap[chr], columns=1:2,
  #     bgcolor="gray95", legend="bottomleft", 
  #     main = paste("chr", chr=peaks$chr[i], "pos", peaks$pos[i], "(",peaks$lodcolumn[i],")")
  #     )
  #plot(c2effBb, gm$gmap[chr], columns=1:2,
  #     bgcolor="gray95", legend="bottomleft", 
  #     main = paste("chr", chr=peaks$chr[i], "pos", peaks$pos[i], "(",peaks$lodcolumn[i],")")
  #     )
  ##last_coef <- unclass(c2effB)[nrow(c2effB),2:3] # last two coefficients
  ##for(t in seq(along=last_coef))
  ##  axis(side=4, at=last_coef[t], names(last_coef)[t], tick=FALSE)


  #Table 1
  chr = peaks_mba$chr[i]
  start=as.numeric(peaks_mba$ci_lo[i])
  end=as.numeric(peaks_mba$ci_hi[i])

  genesgss = query_genes(chr, start, end)

  write.csv(genesgss, file=paste0("data/ici.vs.pbs_",peaks$phenotype[i],"-additive.covariates_genes_chr-",chr,"_peak.marker-",peaks$marker[i],"_lod.drop-1.5_snpsqc_dis_no-x_updated_5.batches_mis.csv"), quote=F)

  rownames(genesgss) <- NULL
  genesgss$strand_old = genesgss$strand
  genesgss$strand[genesgss$strand=="+"] <- "positive"
  genesgss$strand[genesgss$strand=="-"] <- "negative"

  #genesgss <- 
  #table <- 
  #genesgss[,c("chr","type","start","stop","strand","ID","Name","Dbxref","gene_id","mgi_type","description")] %>%
  #kable(escape = F,align = c("ccccccccccc")) %>%
  #kable_styling("striped", full_width = T) #%>% 
  #cat #%>%
  #column_spec(1, bold=TRUE)
#
  #print(kable(genesgss[,c("chr","type","start","stop","strand","ID","Name","Dbxref","gene_id","mgi_type","description")], escape = F,align = c("ccccccccccc")))

  print(kable(genesgss[,c("chr","type","start","stop","strand","ID","Name","Dbxref","gene_id","mgi_type","description")], "html") %>% kable_styling("striped", full_width = T))

  #table
  

}
} else {
  print(paste0("There are no peaks that have a LOD that reaches suggestive (p<0.05) level of ",summary(operm,alpha=0.05)$A, " [autosomes]/",summary(operm,alpha=0.05)$X, " [x-chromosome]"))
}


```


## Interactive term 



### Significance thresholds

```{r permutationi, fig.height = 6, fig.width = 9.5, fig.align = "center", echo=TRUE, results="asis"}

operm <- scan1perm(pr.qc, covars["rz.age"], model="normal", addcovar=addcovar, intcovar=addcovar, n_perm=1000, perm_Xsp=TRUE, chr_lengths=chr_lengths(gm$gmap))

summary_table<-data.frame(unclass(summary(operm, alpha=c(0.01,  0.05, 0.1))))
names(summary_table) <- c("autosomes","X")
summary_table$significance.level <- rownames(summary_table)

rownames(summary_table) <- NULL

summary_table[c(3,1:2)] %>%
  kable(escape = F,align = c("ccc")) %>%
  kable_styling("striped", full_width = T) %>%
  column_spec(1, bold=TRUE)
```

### Genome-wide scan

```{r scanonei, fig.height = 6, fig.width = 9.5, fig.align = "center", echo=TRUE, results="asis"}
plot_lod<-function(out,map){
  for (i in 1:dim(out)[2]){
#par(mar=c(5.1, 6.1, 1.1, 1.1))
    ymx <- maxlod(out) # overall maximum LOD score
    plot(out, map, lodcolumn=i, col="slateblue", ylim=c(0, ymx+0.5))
    #legend("topright", lwd=2, colnames(out)[i], bg="gray90")
    title(main = paste0(colnames(out)[i], " - ICI vs. PBS (with interactive covariate) [positions in cM]"))
    add_threshold(map,  summary(operm,alpha=0.1), col = 'purple')
    add_threshold(map,  summary(operm, alpha=0.05), col = 'red')
    add_threshold(map,  summary(operm, alpha=0.01), col = 'blue')

  }
}


print("with no kinship")
outfi <- scan1(pr.qc, covars["rz.age"], Xcovar=Xcovar, addcovar=addcovar, intcovar=addcovar, model="normal")

outf.new <- outfi-outfa

plot_lod(outfi,gm$gmap)

print("with normal kinship")
outki <- scan1(pr.qc, covars["rz.age"], Xcovar=Xcovar, model="normal", addcovar=addcovar, intcovar=addcovar, kinship=kinship)

#plot_lod(outki,gm$gmap)

outk.new <- outki-outka

plot_lod(outk.new,gm$gmap)

print("with loco kinship")
outlk <- scan1(pr.qc, covars["rz.age"], Xcovar=Xcovar, model="normal", addcovar=addcovar, intcovar=addcovar, kinship=K)

#plot_lod(outi,gm$gmap)

outlk.new <- outlk-outlk

plot_lod(outlk.new,gm$gmap)

print("no kinship and no xcovar")
outi <- scan1(pr.qc, covars["rz.age"], model="normal",addcovar=addcovar, intcovar=addcovar)

out.new <- outi-outa

plot_lod(out.new,gm$gmap)

out = out.new





```

### LOD peaks

#### Centimorgan (cM)

```{r LOD peaksi, fig.height = 6, fig.width = 9.5, fig.align = "center", echo=TRUE, results="asis"}

peaks<-find_peaks(out, gm$gmap, threshold=summary(operm,alpha=0.05)$A, thresholdX = summary(operm,alpha=0.05)$X, peakdrop=3, drop=1.5)

if(nrow(peaks) >0){
peaks$marker <- find_marker(gm$gmap, chr=peaks$chr,pos=peaks$pos)
names(peaks)[2] <- c("phenotype")
peaks <- peaks[-1]

rownames(peaks) <- NULL
print(kable(peaks, escape = F, align = c("cccccccc"), "html") 
  %>% kable_styling("striped", full_width = T)%>%
  column_spec(1, bold=TRUE)
  )

#plot only peak chromosomes

plot_lod_chr<-function(out,map,chrom){
  for (i in 1:dim(out)[2]){
    #png(filename=paste0("/Users/chenm/Documents/qtl/Jai/",colnames(out)[i],  "_lod.png"))
    
    #par(mar=c(5.1, 6.1, 1.1, 1.1))
    ymx <- maxlod(out) # overall maximum LOD score
    plot(out, map, chr = chrom, lodcolumn=i, col="slateblue", ylim=c(0, ymx+0.5))
    #legend("topright", lwd=2, colnames(out)[i], bg="gray90")
    title(main = paste0(colnames(out)[i], " - chr", chrom, " - ICI vs. PBS (with interactive covariate) [positions in cM]"))
    add_threshold(map,  summary(operm,alpha=0.1), col = 'purple')
    add_threshold(map,  summary(operm, alpha=0.05), col = 'red')
    add_threshold(map,  summary(operm, alpha=0.01), col = 'blue')
    #for (j in 1: dim(summary_table)[1]){
    #  abline(h=summary_table[j, i],col="red")
    #  text(x=400, y =summary_table[j, i]+0.12, labels = paste("p=", row.names(summary_table)[j]))
    #}
    #dev.off()

  }
}


for(i in unique(peaks$chr)){
#for (i in 1:nrow(peaks)){
  #plot_lod_chr(out,gm$gmap, peaks$chr[i])
  plot_lod_chr(out,gm$gmap, i)
}

} else {
  print(paste0("There are no peaks that have a LOD that reaches suggestive (p<0.05) level of ",summary(operm,alpha=0.05)$A, " [autosomes]/",summary(operm,alpha=0.05)$X, " [x-chromosome]"))
}

```

#### Megabase (MB)

```{r LOD peaks_mbai, fig.height = 6, fig.width = 9.5, fig.align = "center", echo=TRUE, results="asis"}

peaks_mba<-find_peaks(out, gm$pmap, threshold=summary(operm,alpha=0.05)$A, thresholdX = summary(operm,alpha=0.05)$X, peakdrop=3, drop=1.5)

if(nrow(peaks) >0){
peaks_mba$marker <- find_marker(gm$pmap, chr=peaks_mba$chr,pos=peaks_mba$pos)
names(peaks_mba)[2] <- c("phenotype")
peaks_mba <- peaks_mba[-1]

rownames(peaks_mba) <- NULL
print(kable(peaks_mba, escape = F, align = c("cccccccc"), "html") 
  %>% kable_styling("striped", full_width = T)%>%
  column_spec(1, bold=TRUE)
  )

plot_lod_chr_mb<-function(out,map,chrom){
  for (i in 1:dim(out)[2]){
    #png(filename=paste0("/Users/chenm/Documents/qtl/Jai/",colnames(out)[i],  "_lod.png"))
    
    #par(mar=c(5.1, 6.1, 1.1, 1.1))
    ymx <- maxlod(out) # overall maximum LOD score
    plot(out, map, chr = chrom, lodcolumn=i, col="slateblue", ylim=c(0, ymx+0.5))
    #legend("topright", lwd=2, colnames(out)[i], bg="gray90")
    title(main = paste0(colnames(out)[i], " - chr", chrom, " - ICI vs. PBS (with interactive covariate) [positions in cM]"))
    add_threshold(map,  summary(operm,alpha=0.1), col = 'purple')
    add_threshold(map,  summary(operm, alpha=0.05), col = 'red')
    add_threshold(map,  summary(operm, alpha=0.01), col = 'blue')
    #for (j in 1: dim(summary_table)[1]){
    #  abline(h=summary_table[j, i],col="red")
    #  text(x=400, y =summary_table[j, i]+0.12, labels = paste("p=", row.names(summary_table)[j]))
    #}
    #dev.off()
  }
}

for(i in unique(peaks_mba$chr)){
#for (i in 1:nrow(peaks_mba)){
  #plot_lod_chr_mb(out,gm$pmap, peaks_mba$chr[i])
  plot_lod_chr_mb(out,gm$pmap,i)
}

} else {
  print(paste0("There are no peaks that have a LOD that reaches suggestive (p<0.05) level of ",summary(operm,alpha=0.05)$A, " [autosomes]/",summary(operm,alpha=0.05)$X, " [x-chromosome]"))
}

```

### QTL effects

For each peak LOD location we give a list of gene

```{r QTL effectsi, fig.height = 6, fig.width = 9.5, fig.align = "center", echo=TRUE, results="asis"}

if(nrow(peaks) >0){
for (i in 1:nrow(peaks)){
#for (i in 1:1){
  #Plot 1
  g <- maxmarg(pr.qc, gm$gmap, chr=peaks$chr[i], pos=peaks$pos[i], return_char=TRUE)
  #png(filename=paste0("/Users/chenm/Documents/qtl/Jai/","qtl_effect_", i, ".png"))
  #par(mar=c(4.1, 4.1, 1.5, 0.6))
  plot_pxg(g, covars[,peaks$phenotype[i]], ylab=peaks$phenotype[i], sort=FALSE)
  title(main = paste0("chr: ", chr=peaks$chr[i],"; pos: ", peaks$pos[i], "cM /",peaks_mba$pos[i],"MB\n(",peaks$phenotype[i]," )"), line=0.2)
  ##dev.off()

  chr = peaks$chr[i]

# Plot 2
  pr_sub <- pull_genoprobint(pr.qc, gm$gmap, chr, c(peaks$ci_lo[i], peaks$ci_hi[i]))
  #coeff <- scan1coef(pr[,chr], cross$pheno[,peaks$lodcolumn[i]], addcovar = addcovar)
  #coeff <- scan1coef(pr[,chr], cross$pheno[,peaks$lodcolumn[i]], Xcovar=Xcovar)
  #coeff <- scan1coef(pr.qc[,chr], gm$covar[peaks$lodcolumn[i]], model="binary")
  #coeff_sub <- scan1coef(pr_sub[,chr], gm$covar[peaks$lodcolumn[i]], model="binary")
  blup <- scan1blup(pr.qc[,chr], covars[peaks$phenotype[i]])
  blup_sub <- scan1blup(pr_sub[,chr], covars[peaks$phenotype[i]])

  write.csv(as.data.frame(blup_sub), paste0("data/ici.vs.pbs_",peaks$phenotype[i],"-interactive.covariate_blup_sub_chr-",chr,"_peak.marker-",peaks$marker[i],"_lod.drop-1.5_snpsqc_dis_no-x_updated_5.batches_mis.csv"), quote=F)

  #plot_coef(coeff, 
  #     gm$gmap, columns=1:2,
  #     bgcolor="gray95", legend="bottomleft", 
  #     main = paste("chr", chr=peaks$chr[i],"; pos: ", peaks$pos[i], "cM /",peaks_mba$pos[i],"MB\n(",peaks$lodcolumn[i]," [scan1coeff; positions in cM] )")
  #     )

  #plot_coef(coeff_sub, 
  #     gm$gmap, columns=1:2,
  #     bgcolor="gray95", legend="bottomleft", 
  #     main = paste("chr", chr=peaks$chr[i],"; pos: ", peaks$pos[i], "cM /",peaks_mba$pos[i],"MB\n(",peaks$lodcolumn[i],"; 1.5 LOD drop interval [scan1coeff; positions in cM] ) ")
  #     )


  plot_coef(blup, 
       gm$gmap, columns=1:2,
       bgcolor="gray95", legend="bottomleft", 
       main = paste0("chr: ", chr=peaks$chr[i],"; pos: ", peaks$pos[i], "cM /",peaks_mba$pos[i],"MB\n(",peaks$phenotype[i]," [scan1blup; positions in cM] )")
       )

  plot_coef(blup_sub, 
       gm$gmap, columns=1:2,
       bgcolor="gray95", legend="bottomleft", 
       main = paste0("chr: ", chr=peaks$chr[i],"; pos: ", peaks$pos[i], "cM /",peaks_mba$pos[i],"MB\n(",peaks$phenotype[i],"; 1.5 LOD drop interval [scan1blup; positions in cM] )")
       )


 # Plot 3
  #c2effB <- scan1coef(pr.qc[,chr], gm$covar[peaks$lodcolumn[i]], model="binary", contrasts=cbind(a=c(-1, 0), d=c(0, -1)))
  #c2effBb <- scan1blup(pr.qc[,chr], gm$covar[peaks$lodcolumn[i]], contrasts=cbind(a=c(-1, 0), d=c(0, -1)))
  ##c2effB <- scan1coef(pr[,chr], cross$pheno[,peaks$lodcolumn[i]], addcovar = addcovar, contrasts=cbind(mu=c(1,1,1), a=c(-1, 0, 1), d=c(0, 1, 0)))
  ##c2effB <- scan1coef(pr[,chr], cross$pheno[,peaks$lodcolumn[i]],Xcovar=Xcovar, contrasts=cbind(mu=c(1,1,1), a=c(-1, 0, 1), d=c(0, 1, 0)))
  #plot(c2effB, gm$gmap[chr], columns=1:2,
  #     bgcolor="gray95", legend="bottomleft", 
  #     main = paste("chr", chr=peaks$chr[i], "pos", peaks$pos[i], "(",peaks$lodcolumn[i],")")
  #     )
  #plot(c2effBb, gm$gmap[chr], columns=1:2,
  #     bgcolor="gray95", legend="bottomleft", 
  #     main = paste("chr", chr=peaks$chr[i], "pos", peaks$pos[i], "(",peaks$lodcolumn[i],")")
  #     )
  ##last_coef <- unclass(c2effB)[nrow(c2effB),2:3] # last two coefficients
  ##for(t in seq(along=last_coef))
  ##  axis(side=4, at=last_coef[t], names(last_coef)[t], tick=FALSE)


  #Table 1
  chr = peaks_mba$chr[i]
  start=as.numeric(peaks_mba$ci_lo[i])
  end=as.numeric(peaks_mba$ci_hi[i])

  genesgss = query_genes(chr, start, end)

  write.csv(genesgss, file=paste0("data/ici.vs.pbs_",peaks$phenotype[i],"-interactive.covariate_genes_chr-",chr,"_peak.marker-",peaks$marker[i],"_lod.drop-1.5_snpsqc_dis_no-x_updated_5.batches_mis.csv"), quote=F)

  rownames(genesgss) <- NULL
  genesgss$strand_old = genesgss$strand
  genesgss$strand[genesgss$strand=="+"] <- "positive"
  genesgss$strand[genesgss$strand=="-"] <- "negative"

  #genesgss <- 
  #table <- 
  #genesgss[,c("chr","type","start","stop","strand","ID","Name","Dbxref","gene_id","mgi_type","description")] %>%
  #kable(escape = F,align = c("ccccccccccc")) %>%
  #kable_styling("striped", full_width = T) #%>% 
  #cat #%>%
  #column_spec(1, bold=TRUE)
#
  #print(kable(genesgss[,c("chr","type","start","stop","strand","ID","Name","Dbxref","gene_id","mgi_type","description")], escape = F,align = c("ccccccccccc")))

  print(kable(genesgss[,c("chr","type","start","stop","strand","ID","Name","Dbxref","gene_id","mgi_type","description")], "html") %>% kable_styling("striped", full_width = T))

  #table
  

}
} else {
  print(paste0("There are no peaks that have a LOD that reaches suggestive (p<0.05) level of ",summary(operm,alpha=0.05)$A, " [autosomes]/",summary(operm,alpha=0.05)$X, " [x-chromosome]"))
}


```