---
title: "Serreze Workflow - 5 batches (additional analysis)"
author: "Belinda Cornes"
date: "2022-04-06"
site: workflowr::wflow_site
output:
  workflowr::wflow_html:
    toc: true
editor_options:
  chunk_output_type: console
---

```{r setup, include=FALSE}
knitr::opts_chunk$set(echo = FALSE,warning=FALSE)

library("kableExtra")
library("knitr")
library(data.table) 
library(tidyr)
library(mclust)
library(optparse)
library(dplyr)
library(cluster)
library(readxl)
library(tibble)
library(ggplot2)
library(broman)
library(qtl2)
library(qtlcharts)
library(ggrepel)
library(dplyr)
library(readxl) 

options(stringsAsFactors = FALSE)

```

### **Neogen Files At A Glance** [Total Samples: 312]
* Samples: The_Jackson_Lab_Serreze_MURGIGV01_20201209
  + [Total Samples: 24](0.1_samples_batch_20201209.html)
* Samples: The_Jackson_Lab_Serreze_MURGIGV01_20210527
  + [Total Samples: 72](0.1_samples_batch_20210527.html)
* Samples: The_Jackson_Lab_Serreze_MURGIGV01_20210909
  + [Total Samples: 72](0.1_samples_batch_20210909.html)
* Samples: The_Jackson_Lab_Serreze_MURGIGV01_20211101
  + [Total Samples: 24](0.1_samples_batch_20211101.html)
* Samples: The_Jackson_Lab_Serreze_MURGIGV01_220220328
  + [Total Samples: 120](0.1_samples_batch_220220328.html)
* [Preparing files for QC/Haplotype Phasing](0.1.1_preparing.data_bqc_5.batches.html)


### **Haplotype Phasing**
* [minimum probability set for estimation of genotypes](0.2_haplotype_comparison_bqc_5.batches_minprob.html)

### **1. Experiment Design**

```{r chapter1, eval=TRUE}

gm <- read_cross2("/Users/corneb/Documents/MyJax/CS/Projects/Serreze/haplotype.reconstruction/output_5.batches/gm_bc.json")
#gm

genos <- read_csv("/Users/corneb/Documents/MyJax/CS/Projects/Serreze/haplotype.reconstruction/output_5.batches/sample_geno_bc.csv")

design = matrix(ncol=8, nrow=1, dimnames = list("",c("abbreviations for strains:","type of cross:","number of mice phenotyped:","total number of mice phenotypes:","phenotypes:","number of mice:","number of markers:","covariates:")))
design[1,1] <- "AB: het; AA: hom"
design[1,2] <- "Backcross"
design[1,3] <- nrow(gm$pheno)
design[1,4] <- "1 [binary]"
design[1,5] <- "diabetic status/group [binary]"
design[1,6] <- ncol(genos)
design[1,7] <- nrow(genos)
design[1,8] <- paste0("sex [F: ", nrow(gm$covar[gm$covar$sex=="Female",]), "; M: ", nrow(gm$covar[gm$covar$sex=="Male",]),"] though not used as all mice are females")

design <- t(design)
#design <- as.data.frame(design)
#design$category <- rownames(design)
#rownames <- NULL

design[] %>%
  kable(escape = F,align = c("lr")) %>%
  kable_styling("striped", full_width = T) %>%
  column_spec(1, width = "5cms", bold=TRUE)  %>%
  column_spec(2, width = "1cms") 

```

### **2. Genotype QC (Part 1)**
* [Step 1: Sample QC](2.1_sample_bqc_5.batches.html)
* Step 2: SNP QC (Monomorphic, Genotype Error & Missing Data)
  + [Removing missing data](2.2.1_snp_qc_5.batches_mis.html)
  + [Removing monomorphic, genotype error & missing Data](2.2.1_snp_qc_5.batches.html)
* Step 3: Preparing Files for QTL Analysis
  + [Using datatset with missing markers removed](2.4_preparing.data_aqc_5.batches_mis.html)
  + [Using datatset with monomorphic, genotype error & missing markers removed](2.4_preparing.data_aqc_5.batches.html)


### **3. Phenotype QC**
* Phenotypes
  + continuous (although not being used in QTL mapping)
    + Correct Phenotypes
      * [Using datatset with missing markers removed](3.1_phenotype.qc_corrected_5.batches_mis.html)
      * [Using datatset with monomorphic, genotype error & missing markers removed](3.1_phenotype.qc_corrected_5.batches.html)


### **4. Genotype QC (Part 2)**
* Step 4: Disproportionate markers per contrast group
  + ICI vs EOI
    * [Using datatset with missing markers removed](genotype.frequencies_ici.vs.eoi_5.batches_mis.html)
    * [Using datatset with monomorphic, genotype error & missing markers removed](genotype.frequencies_ici.vs.eoi_5.batches.html)
  + ICI vs PBS
    * [Using datatset with missing markers removed](genotype.frequencies_ici.vs.pbs_5.batches_mis.html)
    * [Using datatset with monomorphic, genotype error & missing markers removed](genotype.frequencies_ici.vs.pbs_5.batches.html)
  + ICI-Early vs PBS
    * [Using datatset with missing markers removed](genotype.frequencies_ici-early.vs.pbs_5.batches_mis.html)
    * [Using datatset with monomorphic, genotype error & missing markers removed](genotype.frequencies_ici-early.vs.pbs_5.batches.html)


### **5. QTL Analysis**
* binary phenotpe
  + ICI vs EOI
    * No further markers removed
      + [Using datatset with missing markers removed](4.1.1_qtl.analysis_binary_ici.vs.eoi_5.batches_mis.html)
      + [Using datatset with monomorphic, genotype error & missing markers removed](4.1.1_qtl.analysis_binary_ici.vs.eoi_5.batches.html)
    * Further removal of Low Frequency markers in these group samples
      + [Using datatset with missing markers removed](4.1.1_qtl.analysis_binary_ici.vs.eoi_snpsqc_5.batches_mis.html)
      + [Using datatset with monomorphic, genotype error & missing markers removed](4.1.1_qtl.analysis_binary_ici.vs.eoi_snpsqc_5.batches.html)
    * Further removal of disproportionate markers in these group samples
      + [Using datatset with missing markers removed (no xcovar)](4.1.1_qtl.analysis_binary_ici.vs.eoi_snpsqc_dis_no-x_updated_5.batches_mis.html)
      + [Using datatset with monomorphic, genotype error & missing markers removed (no xcovar)](4.1.1_qtl.analysis_binary_ici.vs.eoi_snpsqc_dis_no-x_updated_5.batches.html)
  + ICI vs PBS
    * No further markers removed
      + [Using datatset with missing markers removed](4.1.1_qtl.analysis_binary_ici.vs.pbs_5.batches_mis.html)
      + [Using datatset with monomorphic, genotype error & missing markers removed](4.1.1_qtl.analysis_binary_ici.vs.pbs_5.batches.html)
    * Further removal of Low Frequency markers in these group samples
      + [Using datatset with missing markers removed](4.1.1_qtl.analysis_binary_ici.vs.pbs_snpsqc_5.batches_mis.html)
      + [Using datatset with monomorphic, genotype error & missing markers removed](4.1.1_qtl.analysis_binary_ici.vs.pbs_snpsqc_5.batches.html)
      + [Using datatset with monomorphic, genotype error & missing markers removed with additional peak information](4.1.1_qtl.analysis_binary_ici.vs.pbs_snpsqc_5.batches_peaks.html)
    * Further removal of disproportionate markers in these group samples
      + [Using datatset with missing markers removed (no xcovar)](4.1.1_qtl.analysis_binary_ici.vs.pbs_snpsqc_dis_no-x_updated_5.batches_mis.html)
      + [Using datatset with missing markers removed (no xcovar with additonal peak information)](4.1.1_qtl.analysis_binary_ici.vs.pbs_snpsqc_dis_no-x_updated_5.batches_mis_peaks.html)
      + [Using datatset with monomorphic, genotype error & missing markers removed (no xcovar)](4.1.1_qtl.analysis_binary_ici.vs.pbs_snpsqc_dis_no-x_updated_5.batches.html)
      + [Using datatset with monomorphic, genotype error & missing markers removed (no xcovar additional peak information)](4.1.1_qtl.analysis_binary_ici.vs.pbs_snpsqc_dis_no-x_updated_5.batches_peaks.html)
  + ICI-Early vs PBS
    * No further markers removed
      + [Using datatset with missing markers removed](4.1.1_qtl.analysis_binary_ici-early.vs.pbs_5.batches_mis.html)
      + [Using datatset with monomorphic, genotype error & missing markers removed](4.1.1_qtl.analysis_binary_ici-early.vs.pbs_5.batches.html)
    * Further removal of Low Frequency markers in these group samples
      + [Using datatset with missing markers removed](4.1.1_qtl.analysis_binary_ici-early.vs.pbs_snpsqc_5.batches_mis.html)
      + [Using datatset with monomorphic, genotype error & missing markers removed](4.1.1_qtl.analysis_binary_ici-early.vs.pbs_snpsqc_5.batches.html)
    * Further removal of disproportionate markers in these group samples
      + [Using datatset with missing markers removed (no xcovar)](4.1.1_qtl.analysis_binary_ici-early.vs.pbs_snpsqc_dis_no-x_updated_5.batches_mis.html)
      + [Using datatset with monomorphic, genotype error & missing markers removed (no xcovar)](4.1.1_qtl.analysis_binary_ici-early.vs.pbs_snpsqc_dis_no-x_updated_5.batches.html)
* continious phenotype (all phenotypes used are corrected and all analyses remove outleirs where applicable)
  + ICI vs EOI
    * age-of-onset
      * No further markers removed
        + [Using datatset with missing markers removed](4.1.2_qtl.analysis_cont_age_ici.vs.eoi_pheno.corrected.cleaned_5.batches_mis.html)
        + [Using datatset with monomorphic, genotype error & missing markers removed](4.1.2_qtl.analysis_cont_age_ici.vs.eoi_pheno.corrected.cleaned_5.batches.html)
      * Further removal of Low Frequency markers in these group samples
        + [Using datatset with missing markers removed](4.1.2_qtl.analysis_cont_age_ici.vs.eoi_snpsqc_pheno.corrected.cleaned_5.batches_mis.html)
        + [Using datatset with monomorphic, genotype error & missing markers removed](4.1.2_qtl.analysis_cont_age_ici.vs.eoi_snpsqc_pheno.corrected.cleaned_5.batches.html)
      * Further removal of disproportionate markers in these group samples
        + [Using datatset with missing markers removed (no xcovar or kinship)](4.1.2_qtl.analysis_cont_age_ici.vs.eoi_snpsqc_pheno.corrected.cleaned_dis_no-xk_5.batches_mis.html)
        + [Using datatset with monomorphic, genotype error & missing markers removed (no xcovar or kinship)](4.1.2_qtl.analysis_cont_age_ici.vs.eoi_snpsqc_pheno.corrected.cleaned_dis_no-xk_5.batches.html)
    * ranknorm age-of-onset
      * No further markers removed
        + [Using datatset with missing markers removed](4.1.2_qtl.analysis_cont_rz.age_ici.vs.eoi_pheno.corrected.cleaned_5.batches_mis.html)
        + [Using datatset with monomorphic, genotype error & missing markers removed](4.1.2_qtl.analysis_cont_rz.age_ici.vs.eoi_pheno.corrected.cleaned_5.batches.html)
      * Further removal of Low Frequency markers in these group samples
        + [Using datatset with missing markers removed](4.1.2_qtl.analysis_cont_rz.age_ici.vs.eoi_snpsqc_pheno.corrected.cleaned_5.batches_mis.html)
        + [Using datatset with monomorphic, genotype error & missing markers removed](4.1.2_qtl.analysis_cont_rz.age_ici.vs.eoi_snpsqc_pheno.corrected.cleaned_5.batches.html)
      * Further removal of disproportionate markers in these group samples
        + [Using datatset with missing markers removed (no xcovar or kinship)](4.1.2_qtl.analysis_cont_rz.age_ici.vs.eoi_snpsqc_pheno.corrected.cleaned_dis_no-xk_5.batches_mis.html)
        + [Using datatset with monomorphic, genotype error & missing markers removed (no xcovar or kinship)](4.1.2_qtl.analysis_cont_rz.age_ici.vs.eoi_snpsqc_pheno.corrected.cleaned_dis_no-xk_5.batches.html)
  + ICI vs PBS
    * age-of-onset
      * No further markers removed
        + [Using datatset with missing markers removed](4.1.2_qtl.analysis_cont_age_ici.vs.pbs_pheno.corrected.cleaned_5.batches_mis.html)
        + [Using datatset with monomorphic, genotype error & missing markers removed](4.1.2_qtl.analysis_cont_age_ici.vs.pbs_pheno.corrected.cleaned_5.batches.html)
      * Further removal of Low Frequency markers in these group samples
        + [Using datatset with missing markers removed](4.1.2_qtl.analysis_cont_age_ici.vs.pbs_snpsqc_pheno.corrected.cleaned_5.batches_mis.html)
        + [Using datatset with monomorphic, genotype error & missing markers removed](4.1.2_qtl.analysis_cont_age_ici.vs.pbs_snpsqc_pheno.corrected.cleaned_5.batches.html)
      * Further removal of disproportionate markers in these group samples
        + [Using datatset with missing markers removed (no xcovar or kinship)](4.1.2_qtl.analysis_cont_age_ici.vs.pbs_snpsqc_pheno.corrected.cleaned_dis_no-xk_5.batches_mis.html)
        + [Using datatset with monomorphic, genotype error & missing markers removed (no xcovar or kinship)](4.1.2_qtl.analysis_cont_age_ici.vs.pbs_snpsqc_pheno.corrected.cleaned_dis_no-xk_5.batches.html)
    * ranknorm age-of-onset
      * No further markers removed
        + [Using datatset with missing markers removed](4.1.2_qtl.analysis_cont_rz.age_ici.vs.pbs_pheno.corrected.cleaned_5.batches_mis.html)
        + [Using datatset with monomorphic, genotype error & missing markers removed](4.1.2_qtl.analysis_cont_rz.age_ici.vs.pbs_pheno.corrected.cleaned_5.batches.html)
      * Further removal of Low Frequency markers in these group samples
        + [Using datatset with missing markers removed](4.1.2_qtl.analysis_cont_rz.age_ici.vs.pbs_snpsqc_pheno.corrected.cleaned_5.batches_mis.html)
        + [Using datatset with monomorphic, genotype error & missing markers removed](4.1.2_qtl.analysis_cont_rz.age_ici.vs.pbs_snpsqc_pheno.corrected.cleaned_5.batches.html)
      * Further removal of disproportionate markers in these group samples
        + [Using datatset with missing markers removed (no xcovar or kinship)](4.1.2_qtl.analysis_cont_rz.age_ici.vs.pbs_snpsqc_pheno.corrected.cleaned_dis_no-xk_5.batches_mis.html)
        + [Using datatset with monomorphic, genotype error & missing markers removed (no xcovar or kinship)](4.1.2_qtl.analysis_cont_rz.age_ici.vs.pbs_snpsqc_pheno.corrected.cleaned_dis_no-xk_5.batches.html)
  + ICI-Early vs PBS
    * age-of-onset
      * No further markers removed
        + [Using datatset with missing markers removed](4.1.2_qtl.analysis_cont_age_ici-early.vs.pbs_pheno.corrected.cleaned_5.batches_mis.html)
        + [Using datatset with monomorphic, genotype error & missing markers removed](4.1.2_qtl.analysis_cont_age_ici-early.vs.pbs_pheno.corrected.cleaned_5.batches.html)
      * Further removal of Low Frequency markers in these group samples
        + [Using datatset with missing markers removed](4.1.2_qtl.analysis_cont_age_ici-early.vs.pbs_snpsqc_pheno.corrected.cleaned_5.batches_mis.html)
        + [Using datatset with monomorphic, genotype error & missing markers removed](4.1.2_qtl.analysis_cont_age_ici-early.vs.pbs_snpsqc_pheno.corrected.cleaned_5.batches.html)
      * Further removal of disproportionate markers in these group samples
        + [Using datatset with missing markers removed (no xcovar or kinship)](4.1.2_qtl.analysis_cont_age_ici-early.vs.pbs_snpsqc_pheno.corrected.cleaned_dis_no-xk_5.batches_mis.html)
        + [Using datatset with monomorphic, genotype error & missing markers removed (no xcovar or kinship)](4.1.2_qtl.analysis_cont_age_ici-early.vs.pbs_snpsqc_pheno.corrected.cleaned_dis_no-xk_5.batches.html)
    * ranknorm age-of-onset
      * No further markers removed
        + [Using datatset with missing markers removed](4.1.2_qtl.analysis_cont_rz.age_ici-early.vs.pbs_pheno.corrected.cleaned_5.batches_mis.html)
        + [Using datatset with monomorphic, genotype error & missing markers removed](4.1.2_qtl.analysis_cont_rz.age_ici-early.vs.pbs_pheno.corrected.cleaned_5.batches.html)
      * Further removal of Low Frequency markers in these group samples
        + [Using datatset with missing markers removed](4.1.2_qtl.analysis_cont_rz.age_ici-early.vs.pbs_snpsqc_pheno.corrected.cleaned_5.batches_mis.html)
        + [Using datatset with monomorphic, genotype error & missing markers removed](4.1.2_qtl.analysis_cont_rz.age_ici-early.vs.pbs_snpsqc_pheno.corrected.cleaned_5.batches.html)
      * Further removal of disproportionate markers in these group samples
        + [Using datatset with missing markers removed (no xcovar or kinship)](4.1.2_qtl.analysis_cont_rz.age_ici-early.vs.pbs_snpsqc_pheno.corrected.cleaned_dis_no-xk_5.batches_mis.html)
        + [Using datatset with monomorphic, genotype error & missing markers removed (no xcovar or kinship)](4.1.2_qtl.analysis_cont_rz.age_ici-early.vs.pbs_snpsqc_pheno.corrected.cleaned_dis_no-xk_5.batches.html)
